<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/5/913" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/5/913.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/5/913.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/5/913.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/5/913.atom" />
<meta name="HW.pisa" content="ggg;4/5/913" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)" />
<meta name="DC.Identifier" content="10.1534/g3.113.010165" />
<meta name="DC.Date" content="2014-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Li et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Switchgrass ( Panicum virgatum L.), a warm season, C4, perennial grass, is one of the predominant grass species of the North American tall grass prairies. It is viewed as a high-potential bioenergy feedstock species because it can produce large amounts of lignocellulosic material with relatively few inputs. The objectives of this project were to develop an advanced switchgrass population and use it for the construction of genetic linkage maps and trait characterization. A three-generation, four-founder population was created and a total of 182 progeny of this advanced population were genotyped, including a mixture of self-pollinated and hybrid individuals. The female map integrated both subpopulations and covered 1629 cM of the switchgrass genome, with an average map length of 91 cM per linkage group. The male map of the hybrid progeny covered 1462 cM, with an average map length of 81 cM per linkage group. Average marker density of the female and male maps was 3.9 and 3.5 cM per marker interval, respectively. Based on the parental maps, the genome length of switchgrass was estimated to be 1776 cM and 1596 cM for the female map and male map, respectively. The proportion of the genome within 5 cM of a mapped locus was estimated to be 92% and 93% for the female map and male map, respectively. Thus, the linkage maps have covered most of the switchgrass genome. The assessment of marker transmission ratio distortion found that 26% of the genotyped markers were distorted from either 1:1 or 3:1 ratios expected for segregation of single dose markers in one or both parents, respectively. Several regions affected by transmission ratio distortion were found, with linkage groups Ib-m and VIIIa-f most affected." />
<meta name="DC.Contributor" content="Guifen Li" />
<meta name="DC.Contributor" content="Desalegn D. Serba" />
<meta name="DC.Contributor" content="Malay C. Saha" />
<meta name="DC.Contributor" content="Joseph H. Bouton" />
<meta name="DC.Contributor" content="Christina L. Lanzatella" />
<meta name="DC.Contributor" content="Christian M. Tobias" />
<meta name="article:published_time" content="2014-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Switchgrass (&lt;i&gt;Panicum virgatum&lt;/i&gt; L.), a warm season, C4, perennial grass, is one of the predominant grass species of the North American tall grass prairies. It is viewed as a high-potential bioenergy feedstock species because it can produce large amounts of lignocellulosic material with relatively few inputs. The objectives of this project were to develop an advanced switchgrass population and use it for the construction of genetic linkage maps and trait characterization. A three-generation, four-founder population was created and a total of 182 progeny of this advanced population were genotyped, including a mixture of self-pollinated and hybrid individuals. The female map integrated both subpopulations and covered 1629 cM of the switchgrass genome, with an average map length of 91 cM per linkage group. The male map of the hybrid progeny covered 1462 cM, with an average map length of 81 cM per linkage group. Average marker density of the female and male maps was 3.9 and 3.5 cM per marker interval, respectively. Based on the parental maps, the genome length of switchgrass was estimated to be 1776 cM and 1596 cM for the female map and male map, respectively. The proportion of the genome within 5 cM of a mapped locus was estimated to be 92% and 93% for the female map and male map, respectively. Thus, the linkage maps have covered most of the switchgrass genome. The assessment of marker transmission ratio distortion found that 26% of the genotyped markers were distorted from either 1:1 or 3:1 ratios expected for segregation of single dose markers in one or both parents, respectively. Several regions affected by transmission ratio distortion were found, with linkage groups &lt;i&gt;Ib-m&lt;/i&gt; and &lt;i&gt;VIIIa-f&lt;/i&gt; most affected.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/05/01" />
<meta name="citation_mjid" content="ggg;4/5/913" />
<meta name="citation_id" content="4/5/913" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/5/913" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/5/913.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/5/913.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/5/913.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.010165" />
<meta name="citation_pmid" content="24637352" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="913" />
<meta name="citation_lastpage" content="923" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Guifen Li" />
<meta name="citation_author_institution" content="Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Desalegn D. Serba" />
<meta name="citation_author_institution" content="Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Malay C. Saha" />
<meta name="citation_author_institution" content="Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Joseph H. Bouton" />
<meta name="citation_author_institution" content="Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Christina L. Lanzatella" />
<meta name="citation_author_institution" content="United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710" />
<meta name="citation_author" content="Christian M. Tobias" />
<meta name="citation_author_institution" content="United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710" />
<meta name="citation_author_email" content="christian.tobias@ars.usda.gov" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Agarwala;citation_author=D. L. Applegate;citation_author=D. Maglott;citation_author=G. D. Schuler;citation_author=A. A. Schäffer;citation_title=A Fast and Scalable Radiation Hybrid Map Construction and Integration Strategy;citation_pages=350-364;citation_volume=10;citation_year=2000;citation_issue=3;citation_pmid=10720576;citation_doi=10.1101/gr.10.3.350" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. S. Akkaya;citation_author=R. C. Shoemaker;citation_author=J. E. Specht;citation_author=A. A. Bhagwat;citation_author=P. B. Cregan;citation_title=Integration of Simple Sequence Repeat DNA Markers into a Soybean Linkage Map;citation_pages=1439-1445;citation_volume=35;citation_year=1995;citation_issue=5;citation_doi=10.2135/cropsci1995.0011183X003500050030x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=L. Amsellem;citation_author=C. Dutech;citation_author=N. Billotte;citation_title=Isolation and characterization of polymorphic microsatellite loci in Rubus alceifolius Poir. (Rosaceae), an invasive weed in la Réunion island.;citation_pages=33-35;citation_volume=1;citation_year=2001;citation_doi=10.1046/j.1471-8278 .2000.00013.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. V. Ashley;citation_author=J. A. Wilk;citation_author=S. M. N. Styan;citation_author=K. J. Craft;citation_author=K. L. Jones;citation_title=High variability and disomic segregation of microsatellites in the octoploid Fragaria virginiana Mill. (Rosaceae).;citation_pages=1201-1207;citation_volume=107;citation_year=2003;citation_issue=7;citation_pmid=12908097;citation_doi=10.1007/s00122-003-1370-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=J. N. Barney;citation_author=J. J. Mann;citation_author=G. B. Kyser;citation_author=E. Blumwald;citation_author=A. Van;citation_title=Tolerance of switchgrass to extreme soil moisture stress: ecological implications.;citation_pages=724-732;citation_volume=177;citation_year=2009;citation_doi=10.1016/j.plantsci.2009.09.003" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. L. Bennetzen;citation_author=J. Schmutz;citation_author=H. Wang;citation_author=R. Percifield;citation_author=J. Hawkins;citation_title=Reference genome sequence of the model plant Setaria.;citation_pages=555-561;citation_volume=30;citation_year=2012;citation_issue=6;citation_pmid=22580951;citation_doi=10.1038/nbt.2196" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=P. Broun;citation_author=S. D. Tanksley;citation_title=Characterization and genetic mapping of simple repeat sequences in the tomato genome.;citation_pages=39-49;citation_volume=250;citation_year=1996;citation_issue=1;citation_pmid=8569686;citation_doi=10.1007/BF02191823" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. D. Casler;citation_title=Ecotypic Variation among Switchgrass Populations from the Northern USA;citation_pages=388-398;citation_volume=45;citation_year=2005;citation_issue=1;citation_doi=10.2135/cropsci2005.0388" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. D. Casler;citation_author=K. P. Vogel;citation_author=C. M. Taliaferro;citation_author=R. L. Wynia;citation_title=Latitudinal Adaptation of Switchgrass Populations;citation_pages=293-303;citation_volume=44;citation_year=2004;citation_issue=1;citation_doi=10.2135/cropsci2004.2930" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Chakravarti;citation_author=L. K. Lasher;citation_author=J. E. Reefer;citation_title=A maximum likelihood method for estimating genome length using genetic linkage data.;citation_pages=175-182;citation_volume=128;citation_year=1991;citation_issue=1;citation_pmid=2060775" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. K. Das;citation_author=G. R. Fuentes;citation_author=C. M. Taliaferro;citation_title=Genetic Variability and Trait Relationships in Switchgrass;citation_pages=443-448;citation_volume=44;citation_year=2004;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Asp. Appl. Biol.;citation_author=H. W. Elbersen;citation_author=D. G. Christian;citation_author=N. E. Bassen;citation_author=W. Bacher;citation_author=G. Sauerbeck;citation_title=Switchgrass variety choice in Europe.;citation_pages=21-28;citation_volume=65;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Journal of Environmental Quality;citation_journal_abbrev=J. Environ. Qual.;citation_author=G. K. Evanylo;citation_author=A. O. Abaye;citation_author=C. Dundas;citation_author=C. E. Zipper;citation_author=R. Lemus;citation_title=Herbaceous Vegetation Productivity, Persistence, and Metals Uptake on a Biosolids-Amended Mine Soil;citation_pages=1811-1819;citation_volume=34;citation_year=2005;citation_issue=5;citation_pmid=16151233;citation_doi=10.2134/jeq2004.0329" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Fazio;citation_author=J. E. Staub;citation_author=M. R. Stevens;citation_title=Genetic mapping and QTL analysis of horticultural traits in cucumber ( Cucumis sativus L.) using recombinant inbred lines.;citation_pages=864-874;citation_volume=107;citation_year=2003;citation_issue=5;citation_pmid=12827247;citation_doi=10.1007/s00122-003-1277-1" />
<meta name="citation_reference" content="citation_journal_title=Biomass Bioenergy;citation_author=J. H. Fike;citation_author=D. J. Parrish;citation_author=D. D. Wolf;citation_author=J. A. Balasko;citation_author=J. T. Green;citation_title=Long-term yield potential of switchgrass-for-biofuel systems.;citation_pages=198-206;citation_volume=30;citation_year=2006;citation_doi=10.1016/j.biombioe.2005.10.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Fishman;citation_author=A. J. Kelly;citation_author=E. Morgan;citation_author=J. H. Willis;citation_title=A Genetic Map in the Mimulus guttatus Species Complex Reveals Transmission Ratio Distortion due to Heterospecific Interactions;citation_pages=1701-1716;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=11779808" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=I. Gadish;citation_author=D. Zamir;citation_title=Differential zygotic abortion in an interspecific Lycopersicon cross.;citation_pages=156-159;citation_volume=29;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=A. Garcia-Dorado;citation_author=A. Gallego;citation_title=On the Use of the Classical Tests for Detecting Linkage;citation_pages=143-146;citation_volume=83;citation_year=1992;citation_issue=2;citation_pmid=1578105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. J. Gutierrez-Gonzalez;citation_author=D. F. Garvin;citation_title=Reference genome-directed resolution of homologous and homeologous relationships within and between different oat linkage maps.;citation_pages=178-190;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.01.0004" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. A. Hopkins;citation_author=C. M. Taliaferro;citation_author=C. D. Murphy;citation_author=D. A. Christian;citation_title=Chromosome Number and Nuclear DNA Content of Several Switchgrass Populations;citation_pages=1192-1195;citation_volume=36;citation_year=1996;citation_issue=5;citation_doi=10.2135/cropsci1996.0011183X003600050021x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. J. Hultquist;citation_author=K. P. Vogel;citation_author=D. J. Lee;citation_author=K. Arumuganathan;citation_author=S. Kaeppler;citation_title=Chloroplast DNA and Nuclear DNA Content Variations among Cultivars of Switchgrass, Panicum virgatum L.;citation_pages=1049-1052;citation_volume=36;citation_year=1996;citation_issue=4;citation_doi=10.2135/cropsci1996.0011183X003600040039x" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=S. Jachner;citation_author=K. G. van den Boogaart;citation_author=T. Petzoldt;citation_title=Statistical methods for the qualitative assessment of dynamic models with time delay (R Package qualV).;citation_pages=1-30;citation_volume=22;citation_year=2007;citation_doi=10.18637/jss.v022.i04" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=Y. Kashi;citation_author=D. King;citation_author=M. Soller;citation_title=Simple sequence repeats as a source of quantitative genetic variation.;citation_pages=74-78;citation_volume=13;citation_year=1997;citation_issue=2;citation_pmid=9055609;citation_doi=10.1016/S0168-9525(97)01008-1" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1944;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Li;citation_author=X. Wang;citation_author=Y. Wei;citation_author=E. C. Brummer;citation_title=Prevalence of segregation distortion in diploid alfalfa and its implications for genetics and breeding applications.;citation_pages=667-679;citation_volume=123;citation_year=2011;citation_issue=4;citation_pmid=21625992;citation_doi=10.1007/s00122-011-1617-5" />
<meta name="citation_reference" content="citation_journal_title=Bioenerg. Res.;citation_author=L. Liu;citation_author=Y. Wu;citation_title=Identification of a selfing compatible genotype and mode of inheritance in switchgrass.;citation_pages=662-668;citation_volume=5;citation_year=2012;citation_doi=10.1007/s12155-011-9173-z" />
<meta name="citation_reference" content="citation_author=L. Liu;citation_author=Y. Wu;citation_author=Y. Wang;citation_author=T. Samuels;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Lorieux;citation_author=B. Goffinet;citation_author=X. Perrier;citation_author=G. León;citation_author=C. Lanaud;citation_title=Maximum-likelihood models for mapping genetic markers showing segregation distortion. 1. Backcross populations.;citation_pages=73-80;citation_volume=90;citation_year=1995;citation_pmid=24173786;citation_doi=10.1007/BF00220998" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Lorieux;citation_author=X. Perrier;citation_author=B. Goffinet;citation_author=C. Lanaud;citation_author=G. León;citation_title=Maximum-likelihood models for mapping genetic markers showing segregation distortion. 2. F2 populations.;citation_pages=81-89;citation_volume=90;citation_year=1995;citation_pmid=24173787;citation_doi=10.1007/BF00220999" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Lu;citation_title=Chromosomal regions associated with segregation distortion in maize.;citation_pages=622-628;citation_volume=105;citation_year=2002;citation_issue=4;citation_pmid=12582513;citation_doi=10.1007/s00122-002-0970-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. C. Mangelsdorf;citation_author=D. F. Jones;citation_title=THE EXPRESSION OF MENDELIAN FACTORS IN THE GAMETOPHYTE OF MAIZE;citation_pages=423-455;citation_volume=11;citation_year=1926;citation_issue=5;citation_pmid=17246465" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Martínez-Reyna;citation_author=K. Vogel;citation_title=Incompatibility Systems in Switchgrass;citation_pages=1800-1805;citation_volume=42;citation_year=2002;citation_issue=6;citation_doi=10.2135/cropsci2002.1800" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Missaoui;citation_author=A. Paterson;citation_author=J. Bouton;citation_title=Investigation of genomic organization in switchgrass (Panicum virgatum L.) using DNA markers.;citation_pages=1372-1383;citation_volume=110;citation_year=2005;citation_issue=8;citation_pmid=15841364;citation_doi=10.1007/s00122-005-1935-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J.-H. Mun;citation_author=D.-J. Kim;citation_author=H. K. Choi;citation_author=J. Gish;citation_author=F. Debellé;citation_title=Distribution of microsatellites in the genome of Medicago truncatula: A resource of genetic markers that integrate genetic and physical maps.;citation_volume=172;citation_year=2006;citation_pmid=16489220;citation_doi=10.1534/genetics.105.054791" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. M. Muraya;citation_author=E. Mutegi;citation_author=H. H. Geiger;citation_author=S. M. de Villiers;citation_author=F. Sagnard;citation_title=Wild sorghum from different eco-geographic regions of Kenya display a mixed mating system.;citation_pages=1631-1639;citation_volume=122;citation_year=2011;citation_issue=8;citation_pmid=21360157;citation_doi=10.1007/s00122-011-1560-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Okada;citation_author=C. Lanzatella;citation_author=M. C. Saha;citation_author=J. Bouton;citation_author=R. Wu;citation_title=complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions.;citation_volume=185;citation_year=2010;citation_pmid=20407132;citation_doi=10.1534/genetics.110.113910" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=N. Ramchiary;citation_author=V. D. Nguyen;citation_author=X. Li;citation_author=C. P. Hong;citation_author=V. Dhandapani;citation_title=Genic microsatellite markers in Brassica rapa: Development, characterization, mapping, and their utility in other cultivated and wild brassica relatives.;citation_volume=18;citation_year=2011;citation_pmid=21768136;citation_doi=10.1093/dnares/dsr017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Ren;citation_author=Z. Zhang;citation_author=J. Liu;citation_author=J. E. Staub;citation_author=Y. Han;citation_title=An integrated genetic and cytogenetic map of the cucumber genome.;citation_pages=e5795-e5795;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19495411;citation_doi=10.1371/journal.pone.0005795" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. M. Rick;citation_title=DIFFERENTIAL ZYGOTIC LETHALITY IN A TOMATO SPECIES HYBRID;citation_pages=1497-1507;citation_volume=48;citation_year=1963;citation_issue=11;citation_pmid=17248190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. M. Rick;citation_title=ABORTION OF MALE AND FEMALE GAMETES IN THE TOMATO DETERMINED BY ALLELIC INTERACTION;citation_pages=85-96;citation_volume=53;citation_year=1966;citation_issue=1;citation_pmid=5900610" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Sci.;citation_author=M. A. Sanderson;citation_author=P. R. Adler;citation_author=A. A. Boateng;citation_author=M. D. Casler;citation_author=G. Sarath;citation_title=Switchgrass as a biofuels feedstock in the USA.;citation_pages=1315-1325;citation_volume=86;citation_year=2006;citation_doi=10.4141/P06-136" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Sandler;citation_author=Y. Hiraizum;citation_author=I. Sandler;citation_title=MEIOTIC DRIVE IN NATURAL POPULATIONS OF DROSOPHILA MELANOGASTER. I. THE CYTOGENETIC BASIS OF SEGREGATION-DISTORTION;citation_pages=233-250;citation_volume=44;citation_year=1959;citation_issue=2;citation_pmid=17247821" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Saski;citation_author=Z. Li;citation_author=F. Feltus;citation_author=H. Luo;citation_title=New genomic resources for switchgrass: a BAC library and comparative analysis of homoeologous genomic regions harboring bioenergy traits.;citation_pages=369-369;citation_volume=12;citation_year=2011;citation_pmid=21767393;citation_doi=10.1186/1471-2164-12-369" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. Schuelke;citation_title=An economic method for the fluorescent labeling of PCR fragments.;citation_pages=233-234;citation_volume=18;citation_year=2000;citation_issue=2;citation_pmid=10657137;citation_doi=10.1038/72708" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Res.;citation_author=D. Serba;citation_author=L. Wu;citation_author=G. Daverdin;citation_author=B. A. Bahri;citation_author=X. Wang;citation_title=Linkage maps of lowland and upland tetraploid switchgrass ecotypes.;citation_pages=953-965;citation_volume=6;citation_year=2013;citation_doi=10.1007/s12155-013-9315-6" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Join Map.;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. E. Talbert;citation_author=D. H. Timothy;citation_author=J. C. Burns;citation_author=J. O. Rawlings;citation_author=R. H. Moll;citation_title=Estimates of Genetic Parameters in Switchgrass;citation_pages=725-728;citation_volume=23;citation_year=1983;citation_issue=4;citation_doi=10.2135/cropsci1983.0011183X002300040029x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_title=Statistical genetics of quantitative traits linkage, maps, and QTL;citation_author=R. Wu;citation_author=G. Casella;citation_author=C.-X. Ma;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. Wu;citation_author=C.-X. Ma;citation_author=I. Painter;citation_author=Z.-B. Zeng;citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.;citation_pages=349-363;citation_volume=61;citation_year=2002;citation_issue=3;citation_pmid=12027621;citation_doi=10.1006/tpbi.2002.1577" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Zhang;citation_author=J. Zalapa;citation_author=A. R. Jakubowski;citation_author=D. L. Price;citation_author=A. Acharya;citation_title=Natural hybrids and gene flow between upland and lowland switchgrass.;citation_pages=2626-2641;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.02.0104" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="twitter:description" content="Switchgrass ( Panicum virgatum L.), a warm season, C4, perennial grass, is one of the predominant grass species of the North American tall grass prairies. It is viewed as a high-potential bioenergy feedstock species because it can produce large amounts of lignocellulosic material with relatively few inputs. The objectives of this project were to develop an advanced switchgrass population and use it for the construction of genetic linkage maps and trait characterization. A three-generation, four-founder population was created and a total of 182 progeny of this advanced population were genotyped, including a mixture of self-pollinated and hybrid individuals. The female map integrated both subpopulations and covered 1629 cM of the switchgrass genome, with an average map length of 91 cM per linkage group. The male map of the hybrid progeny covered 1462 cM, with an average map length of 81 cM per linkage group. Average marker density of the female and male maps was 3.9 and 3.5 cM per marker interval, respectively. Based on the parental maps, the genome length of switchgrass was estimated to be 1776 cM and 1596 cM for the female map and male map, respectively. The proportion of the genome within 5 cM of a mapped locus was estimated to be 92% and 93% for the female map and male map, respectively. Thus, the linkage maps have covered most of the switchgrass genome. The assessment of marker transmission ratio distortion found that 26% of the genotyped markers were distorted from either 1:1 or 3:1 ratios expected for segregation of single dose markers in one or both parents, respectively. Several regions affected by transmission ratio distortion were found, with linkage groups Ib-m and VIIIa-f most affected." />
<meta name="og-title" property="og:title" content="Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/5/913" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Switchgrass ( Panicum virgatum L.), a warm season, C4, perennial grass, is one of the predominant grass species of the North American tall grass prairies. It is viewed as a high-potential bioenergy feedstock species because it can produce large amounts of lignocellulosic material with relatively few inputs. The objectives of this project were to develop an advanced switchgrass population and use it for the construction of genetic linkage maps and trait characterization. A three-generation, four-founder population was created and a total of 182 progeny of this advanced population were genotyped, including a mixture of self-pollinated and hybrid individuals. The female map integrated both subpopulations and covered 1629 cM of the switchgrass genome, with an average map length of 91 cM per linkage group. The male map of the hybrid progeny covered 1462 cM, with an average map length of 81 cM per linkage group. Average marker density of the female and male maps was 3.9 and 3.5 cM per marker interval, respectively. Based on the parental maps, the genome length of switchgrass was estimated to be 1776 cM and 1596 cM for the female map and male map, respectively. The proportion of the genome within 5 cM of a mapped locus was estimated to be 92% and 93% for the female map and male map, respectively. Thus, the linkage maps have covered most of the switchgrass genome. The assessment of marker transmission ratio distortion found that 26% of the genotyped markers were distorted from either 1:1 or 3:1 ratios expected for segregation of single dose markers in one or both parents, respectively. Several regions affected by transmission ratio distortion were found, with linkage groups Ib-m and VIIIa-f most affected." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
    <title>Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wzBccp8GobCCC9QG35BRjtcxooHratnbDWuH2A4x5H0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459864","apath":"\/ggg\/4\/5\/913.atom","pisa":"ggg;4\/5\/913","ac":{"\/ggg\/4\/5\/913.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/5\/913.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/5\/913"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/5\/913"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/5\/913.figures-only":true,"\/content\/4\/5\/913":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MEHno\\\\\\\/gPtWPe5vZYsoBRi6L8hHnr1vj8wSS3uALYLHmRrtBAhA7jZEHse\\\\\\\/QTrgDrxLE4TRB94q25Joa3a3QK+an0aElIb2qvdJO+bMGAT+orAI2K4Vq0TT+nDRaaJJe75mzTHgvcSXg5PVYeIu82fJCSVLsZviTfplZKTg9z73MN9lj6xDRG5g4uJRN7JMgw\\\\\\\/XbAG2bnrJf85ZfgIOCO8dosNdDWAm3D3gfmxth9j1PQlsSRbORDBAV3\\\\\\\/jnHSllRQKBzSbeSUZxQ\\\\\\\/I53b1Z0tpbnFlfEAgeRaQ7x8WiJ9OhXIffCc09O+eMAFILuI3T\\\\\\\/pKMDnyUqsPYwCMv\\\\\\\/z0WMD\\\\\\\/mQW02I1lVub9XE4S406c\\\\\\\/x1zRbceZxADiqVoPWIPA2Tvr6f4Jh0QkRNjGOdWgEjgzKrSfmmx0aUNw0qv19zrlHHOWPY2hZqaVFAG5pLOQuRMEZFlK7PjRHIFMmOF4OxBf4LI+rwx02W8UNG1WOP2NZjZJurwQYCrgnyaB35gKeEHt0VV1+cDCFQ\\\\\\\/TFUMrygdHizdKZy+pOPPp+QglQ1rVVCH6DkHUMravpvDaAdiw08yLOdQrcvKr35N0JReWp2LfIY\\\\\\\/QCZvRAR9Y9QjC4c49z06lxMWmJrUOH4F4AKtMqYdIK4O5djJZI5ox5IEBb9V2hpnA5Bf1O3Jf+omA9l6EdXfk3ocYOT0SqY8SlpVyCIN4LK2sacsuyga1DbQcVRMDDt9vyB0c7qKDGDwdZn13zgeFmzqGUAkcd2DKBrGyiYR6VZwveY+EWvOBgNccnXAYLPggezh9BQQduSUyMBppZMm+DspOsrLVXAppK29KTSvtlI6hyzF57g557ouerI+NtHiXUa\\\\\\\/z0myfYBxSzKWeZsi+aYZRU3Ehw9IleVx6iewfWSsCRzBg0vJViaB\\\\\\\/P4hTR5nt5j8E211QdLy6tTM0jsXTTu6sIAgMrAD8qjLz5fSg5ueE4\\\\\\\/ahcNwO410JaDgtOcju57Tq5f823nYSVypzhQmuWjnkEzisqYtGAOAvY9y5NR0Ts9jr8tyukLgPEZo5vQanToL6re8FiJA6p57xyn9yXuD1ukC\\\\\\\/c\\\\\\\/vW71lcFcwd9LLguVdS9gPIu18R7G\\\\\\\/noXyjPCqWPhvmhAS5SdnnjIcfBIElcdm8\\\\\\\/igo5PETsPcTVuSrnc5E7sCJVOYQoXnRoXYhrfVpCfr34wBc6ofvs6cW+rngrv6Z2TjEp57lDuRxvQ0cgMy1nbswbc5PhqVorZeCCYW87BAUNtt4\\\\\\\/6cSLgDkUYi6WbT0ZRWGlnvJN\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZWCfboXymZPon6VvrRbsPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f5c8d66459cbc1ca0d72996a8c39d07e32fb5b3fef42317a4965202f1d872c3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xcpjqsHkxb8F\\\\\\\/AxaFb40ISQmo2mPno+IMrN3oiHgSjxOpWRqzrIlvC4Ze27gRvkCToK7lrs1vPwderTxm3zwjhGTti1rdQ+whTM6in\\\\\\\/wuGmwskfxwLcIb+hHGyC95hIP2LNVsxyYtwXwPokSspXkkDhdIpEtZnUBpEohlu+J2\\\\\\\/ljebtCvsNR5lk9f14NeDJ+hbrLwvDjVXt5VxBDGof+Umxxs7CguOsOA\\\\\\\/wFyNxV1gxL4+e1EOVdPc3vLH4cnDRvnh\\\\\\\/SKALFy0QopUA06VHRJWpLbOPdPSeTes6Q8TMUNc\\\\\\\/HjBdrb5szKiSES3EbQsDZQkptaX+xkEQkkZRRiEBuNmdn09Fh5wVYPXL\\\\\\\/M4wWS3VQ6ALpXjMf1EWYPDVk\\\\\\\/+x7iFbBIXhC2A6cxgAQLdcWVLi2eP3o1KhpWjfY7DuxiokWAuVQEYJB\\\\\\\/Cj5SN7+DmvfpQOWUhUTA7JlIMqi2Csv5nbEsvyj7vSbAnDWf8aEb8oOg8Su81ydRDAV0hXzZIbjbhbTIVJ2AnyzveeKhdfDZINYNMouSVrgOziRNuFMhitNDlQoq3I2mHk8\\\\\\\/cnkBwVy\\\\\\\/UWQEWZ+AFJKKpFHD7rIbuHat7tLs8oS2YAN9A3vhcjQ4s4If1h3Ysrj2j3S+URb78F8GCzjNsl6EsOB9MefJd5f8MZbQbCrZ86UPX1PEemc2KZp5sMR\\\\\\\/v7GXDsAiJueIoZX11VHwbNJ5yOAl5P6AomLni9ls0mg3Ka6IehdDoEb05pWHyb8YCF0A\\\\\\\/FbVEmQUlVSWKiCSwNYkZkjO0Igq2VV7xY6MgXZTVc2\\\\\\\/M76bef1ZAS+tEzfnpguJlfKQ9sk4O8OBDzHMBgP+dVu8tEkiY48NSR6NhR24V2IyaAHdWXu4+LOokOYvghG9fGSAOfW1kH5BpJ8tQJ1BV2atNhVBHFitD4aenL\\\\\\\/\\\\\\\/WzynX5uYg5CSiPahdiGN96jNZA4g+LbpEF6tlebvH2AJTPQ+dmJSiVJZAkRLDixt8Xz9frqU7Ish7GdXmSj7mVjNX\\\\\\\/B6W0QpaaXPSygDOE6QOmJeuqhPbBIxgafW7S1\\\\\\\/x+DqVExPoP2n4U5fPimsHWaXiP2Gj\\\\\\\/fBtjs97taemg2\\\\\\\/yBNUVqKur7qZOTXJSpqRIQ0Ks3HyMNpcf5yPqDKshfCMancrK9+rRniab7mb5Zw4Twuifnglzzx67VTlwRy3djr2u59ovpjt1afoV1qFs6zHBVyrtvtlKXtiPD6ksBPsKlSxpzZcqP8q0Eo\\\\\\\/NKDYKHeahYLyeoBqZ436V6dEfjd\\\\\\\/fsJUt5Eh5BL0uK16O7nG4ocNierfKlCgAi35LFvoNmKOIiHq5y\\\\\\\/UdjPLYF8anTbkgBgz6jgKiKk\\\\\\\/JDLYsW9KTMytjNqHChs2oURnoZBG7GZPLeM+60B7TJ7cffrjVycTajiLlXegkVFojjoZM9xcu+sQjes3VupfBTVULXk7wtT6ng7uz5HiGiWIiD\\\\\\\/JmuaGRiuU3KyFnpGuq\\\\\\\/aMIKXWMcJK\\\\\\\/ue3VIXxLF4SAHvL3oIB1CmZvyLfPFYdm6LaDDl65yLRFFN6+eO9yTVTZVmvatN3wFr1HlOZcZO7o\\\\\\\/HeW4CauSkWo\\\\\\\/FHaLXpBTXXy6Uy\\\\\\\/0hqy68zjQLZq0f3aRn1TLrBz+cgo7P95HuN8\\\\\\\/c0FtTgmzLWZAMSs79Mmldr5k2cIOjRlRG7oo7v95fBFVCmFPku+2Cq\\\\\\\/4JvVCLj5G1jUoJ1KB0pzklLpbnzdfp5UqHv06XMofkTOuElqAB61WltQi+asL1x61lYHhnuc0r4bRnQx1QCvtiABJgq8ksUEVhuovdW2\\\\\\\/g8JhuMuNw7QCnbM6dNbTVSGRhXJtQbY\\\\\\\/tN7EsGZDimLVDZu8jnG6IT4puFYv9hI8tZ\\\\\\\/VYBB7mmImjxrz1JFUjQtN7ImIh\\\\\\\/hSUcfK2LYFAvI9E1MYFcsk7WfkF8LKe9tkZt5yUMp4M5aqFmq2ISFqdn5U7xrir3NNby1SHFKfLVH5hhN5CoXju27zcNSpY++1Tg\\\\\\\/Ho6jt++DwT07P4rbvXJxKy1+1y38O8aY8VJuJGS2pa3S15FSVpeERrwexL\\\\\\\/w65uPgAnpQRN6sDOTKAGOxfQIptp+ye3RLSXgKsbjZ5mSD3cFn8\\\\\\\/O0JLBye9mJ5y9piBBEK7kbxaLcYycWieDi7oiaypwjAg8IQfuqV1H7IeB602xm8n1YS7s\\\\\\\/aBOs6uE5oJFR34WAOW1SaF0fcmfiVjtIirhsbVR9TKaZ3PivKDSJ4bEySpLK7+9O0E73voWxF9\\\\\\\/KccYh+JhM\\\\\\\/NOmoyVp8fEGIPN7evSUqRgWrvLod2M6+DdkZq+Nh4m+\\\\\\\/Exg4BNDe1Y2ndGlb8jKszK0TpwLWxNCQWsagKa5BAeRIO\\\\\\\/JR1YCEVlL\\\\\\\/cVSNv9mqIS75p8gBd5RE+eoTfjD4d824nSKvrHuMP0qPfo1J8uiB1TwvVENjrwlm00yj7VqyJ5O3LOkPKXR42uERLBN18lSBXUWDXcDs3GQdLGfj2nMVJHs96jiByG5P6ianIqiahzYqGT9rAWuTa\\\\\\\/LAiMNRDf3VVCsHgXvNmDVDK6ZJcAvlrZTS064A8mLNSUM+iVDq5+JdCLhbcbMSQTSTTNa2h\\\\\\\/hC0zLJA7c7KWsNEYcVYZvQmkX9+ONOnpICwhw9bccXN38BzIdgGpc3JyMXQqXpqGC40\\\\\\\/7j1n\\\\\\\/aqg3PwA1DCdx4ltgrK0QqPwp\\\\\\\/fsJU0q+RETSl5vmc9gR0VvpLhfY+y47njQ+r57Lf42JIPWUY4yZ0Lf2sRsdXfLKERam5vDj+qfbgiRgFMiIUQZe0gfxOLn92BFLadrXLTWL2k4zpVCvoKOuPsX3LD2Ox7yyfq++kZsPTap7k51nHePJu2P3DL64MGeH6yC3Vvw0zX9HAwX2kgAsxHrll7vIAHXjAyLYI+sz\\\\\\\/A99g8Q+0qSbD4yt2RWRro+knNQXskgjBlO89NIdB+I1MBkQGt5WTuC6euCwshsce\\\\\\\/7eYb1MOxPIUESCZ\\\\\\\/ltxEVi5TNz7X7rGup5lAu8i\\\\\\\/N0Dp+WBK5ruZP+tDTEVjJMo7sbHIxulRuM\\\\\\\/QlYbP0tMrrnla8O5XP6URUcf61eLHLDcSp2K0nT\\\\\\\/5lTZJzaVsX82rnuHpH2w9PLcanviiMVFkiN\\\\\\\/htuisZfnlGxRjAyR\\\\\\\/epXXtAJab7StFSwvbCUliZ0OntsOdGRNj2NLO585i7AyGl3A5IQOEgWrVOcC8yGdrnP0ns7PvyvjV4Zr\\\\\\\/zbl4JmMQWPAl\\\\\\\/GJYj74miSzBTkvA6pv6U2qat8+KhnBBOWrETZ+hfuy9xHlo2dWQ7mAOoGsszxdoqF+3\\\\\\\/MY+a12KRfIoEgBcGPQPmwBHxOq5L2qvWxlEA8mIdYfbJ736gcXRPzIiZ37bH5jdiUc2Q2DoUlPHOJ4Jz2vjkfxoDqIxe\\\\\\\/hidl7ibz6bjT6Ihdo4KVh3hnRkW03JRCtLxyTs22eRh4RV5wwxBDQkf6HpiHVb+dR3JErNeqfh3FUfc4J6TOYVzqxs5+eTinnWp+Jx3ne2FDRyxaK4AQ4MPUMXcUEQYq3whJjMBWXfMEQB6xGpYhdNJYVfjF6kEicyOCfqOwuN9BPKGNbNKaLXfefIzg+yoQ9x9+JG132lUVn7q9wV1KKjt3nBR7H+gr4iACeui+4PjwoxOlQYRmOAhJOJ0dI\\\\\\\/hSrucPErUsxAh8qUEDBXWNRn5PMMXS0D5IUquS0GLzcY3MI6KEELe0dM2QGr3D97W2S\\\\\\\/n2HGdWWgd8yTGUGNht3Jwq8mBh5jQVgbHEys2+QWicTxJng6iFZZ9oOSZX\\\\\\\/M2ixDvOqLS\\\\\\\/RzdhK0ZnfB2vGgF9+uUxB1SmZIFm34bbYy5zIcLmOv6tUVor4k112giVD7p5TLHf7SLihJGeE6daVNM4juW3T+UKWEwt3V3oq7IRv0ZSMBCsHoF6GHVbbUxiotMVaO7AxQF\\\\\\\/1\\\\\\\/n4S9GQUdgnxiQbXjC1AN69tcEuUVF0xR+ZdZrNCUZpgtYwphxwQZs0pYuSFBPeMh0Cz7WmDB\\\\\\\/aac3r0Hh2bNL5i+nFepZoOa5A9f+hcYpEy7t9DrKL2CIAyT1W9yEnYnn90fIvzPEcN9D3Qf10g\\\\\\\/XgNacfcGcogmrV7gMYLk8aiW+q42qSQCSfYIBuZUCqZUruoDQVnSXfnOzCamr11G2iteSkQs6T\\\\\\\/XfvYAQ8Y96BcZXJZG9JQqvD1C7I3WA1WmGYLDOUzPzGiNchwGOlagLM4\\\\\\\/zYV\\\\\\\/swnjXieTN6xpGhLytPT9iZJXMxIfm+W\\\\\\\/r5ofSev9OC\\\\\\\/JB3\\\\\\\/4TEUprKmGD+RW0biL1S3F+PEuVaENfQxNMb009H96rhahdScsWWlTduTxaLYbND+LRPHtLwXtnXwQ4tKxTpp8B73b0OLD9i6TKiFekjTEB8eUJxZjHnuRrGeUGGR8I3xRMORmvW\\\\\\\/z5qfeTkl\\\\\\\/O5sBx3AqgzHTt2gF1xm\\\\\\\/BoG8mp2Y7Z1wx2HUKCg7FXtzZ49SrncsZ0vMtH9QSugFZybZVsWVN5P11QhxKtKi7AZ\\\\\\\/xLN0FfnyJlhZZvEjPxUVDHNbRhUv4cgqi33WzaiZ0xIQ9mUecda7kdOKZA2CiY977F++c23RlBSSCA\\\\\\\/+UsPcHJjzxseZXoLXEO3X\\\\\\\/K5F25Nxgg3VM7Rfb97m9dHLJrfMWmA0hP3HeUro4c\\\\\\\/BLbRDSnFo7\\\\\\\/t8ZFo3dIZ8\\\\\\\/3mT9+6aqDuA9x4MEGzrj6MlqCy1fix3c+2e6rvg29Ft6jeIG2kkpJJXibrgXDhzDvhFzU5B0U5iQ12vOEfdALBbKijQ5mjiM1QGH3ndDqo4Cu66Zboxcr257wHjxbW4ATvJ65fxOd7wOUSmmPB8r5iizo0HD7Pnm6i07KDSVUkf88DKEUo5jCvARREveF\\\\\\\/Q\\\\\\\/Z5i3ZjrheTR9kW4YdY\\\\\\\/gK1\\\\\\\/U1eK69dSF+io\\\\\\\/NRvkuGb03LB5R1seOH9xs27AYJkW\\\\\\\/zIXjF+A1AVv9+1bShyk2uNFSGDe1xaPCpko+VNJT6ir1DUOpResTm7V14ULOmC1cO5wxlAesTqpCXpdf9NWgJ6NSs6SW1HS44\\\\\\\/gmVqHoOFehSa1EnQ62sgmelXE1cwn3rDy\\\\\\\/xGYOOmlSvffbiS+tEjPjYl2efQuFOBnEYDAU5wTUrh\\\\\\\/oRlALQD1IXseZPO\\\\\\\/Z\\\\\\\/Nld509Z\\\\\\\/9Y3Ml5eOsJZq5\\\\\\\/YXHr06mmVrUCbsPCxce3ZmY5CVay6X8GQ1nz9sxyHzhVUNjtFXklpeONyJTmsf5UjvY+d126mz++1u9u5CYA7633SiusdsRmESJjkePsqNBsjJnO+\\\\\\\/c0lr98SIP6x4+gc1q6LAL2qB8Amh5XtatrGDelLvcaayu0c46gLYEb78SGxOJfEy1eT5LwXfSpfRmmdBMhSvMwU7ZjwlILbeafZY0ozEIHI\\\\\\\/xOBSHlkanMvJk1VFC9ZedrKOa12s22csdcBPNKqXA1NFwQ73n+yF5wOH6KwIeUsjONCLcpVVxSpOXl7fzlK+FtZJ9IC2F5oRFmd3gyGnv7pfhCZtDQMm+PuPPzWlV0oS9dg9bPZxinrRnnKX9L0IjcBxXlSakXir2om44ZlHZK9RHw\\\\\\\/fD4Dw9rFOixp6uEU5baq3az+wTdj5vcmobTXmnbt1k0P+dBo+uZvVyJLDzmKfW71DmSAKjJ+6B8ofSvHGyQIgtwV8HgVq6MgfrDjykmYoc1S3vjDB3aap1ZThOGIaSDnfHZUDAOlKrhdUtNb8uLt4l2UgAD1QrzdEttj0q6MEaPh9MSMeOMcZd366i0vrWPlelsyG7ler1Vexv9827En4AbrViToyG+Wu1v9sXjPZjWR\\\\\\\/oJnN8yqVJQikBxn1GrZ6CLfTdOcUPHcgnwKQ1BAWZiN8JAlr+WHPZjTPyVVbm9q9M4n8\\\\\\\/wLt6+2To7VRFppcUMKZfdvpz72vBQvcdSz6n38SoFWhu0d0w41XFmt+yCbyeBtxcUo12K\\\\\\\/PabUwkrHoDzMEdi0uv8mOnzB+\\\\\\\/KJimwTtGwBTZRLgQKkOgX07KjD\\\\\\\/AMN4+A\\\\\\\/BMuawwK2yr0bgpMBuePp\\\\\\\/x4x2802K\\\\\\\/0nFd5T2TqG\\\\\\\/Wu5z7+VC7QF4MfzxfG9gqBaGrEnQFfwOZZP6+vEU5wKSgslOtGWL9gGEAA9soMUcgAQqXVLuMi2argPLFTdhkPDrnBKCDSB8GCOV2kRQS\\\\\\\/3eoFWv0+1eAi1Cssl4DXnGH3rv334+HKSB0OateAyADkIcVaP1Xc4nx9LuWvIyAh85TsEY1MbxDLQ6S80l0YQvPWTDoBI5nF7JUkVyMSOZTg\\\\\\\/pBV6q8pbyAo4oaAF8K9B0YxXSKOFbLGy9C4CKHLeePeIPuD6jaA5eCU8WI3LfDguUL8jL1KEvgxfRn\\\\\\\/dj3pkSjffcWdzEZRbZSzU8BRlktk2QVRY+s6BikYkehsRyETehDUWltBidX6\\\\\\\/PUKTTc0S9MdOcPdNOAiyRoIsEeFzWLeEYfCBm+5Shr0xBGgd1L9qvp\\\\\\\/3tLjmFkdv36PoISHE0byTQixKNWawG6Cq3\\\\\\\/J6k0LSEbCFTKkpN6eGQ5NIWom0vGeTWY9r8Erd+vGYelSCqlN2VTQ9GJMmQSzYh8tR0X8S+w6Dg4R7gjBeGv5+LHuqCnMhFzqKXPKHhxAN655SuCsLbdicbIMiWANWoyLQhNpZ3ZQDs6xx6o5AGIp7vtrTqt6QdUUFpFcOEPiE6L3LOrdAlG08tXwLAPkOZp2HCVFgxulC5icrBSz\\\\\\\/dHhIWXHRNLgwVQfwZ9TAOQ42hd1aKde8f+sruCctyCgeldUXf6JuBCQGNPBm5HN9BzpPo9EtKF0V+BlEZ0LMqDDErcC1zY2LQM8TR2AcQjFRk07NYlB+0z17k3iOymVnq2h3fB+AO0bFzXRbooK+4aLbn2lLTTxunR34ftbe4awNULIckAcWBC1PBZUJUASAoYVhivIaqVyPNRLQg3jwvhysnN31KBhYW3uA6fdlOJDEhiMAYuQwZ8LnGB6w1cRMbfJEYpXkqm9vSdNFz3mW6\\\\\\\/hJm+GQj8vPkF2MKb60iVsaC0eE6vvHizOcG7mmW5ELnqqE5xoaDfvzHfkKGSTy7OcPP9KWJd3yV7m+GfkdnuSSN7s4kRs35nh0iOS4Y3abh7qh3PtZ7RtsdY3QHMv4rgnKDytl4LKz8SEx8D8hFe2gM8uqL1L4QZYpv9Ir1NUGKN0gvY4XMqVDoUWS7g5CRyzaH\\\\\\\/vnK+R96Xh3unwwuZEnw\\\\\\\/ZRWDAcLK4AFWCfRQaLe0rTKETukrJUY\\\\\\\/7SWuzz4C7tE8\\\\\\\/rn\\\\\\\/zKqmkunwa6e6TMTjWI253L755epuJw6FTv1ST5iBcBfG3WQlkYMLY\\\\\\\/48ukSLDUvS472Nk+07vd\\\\\\\/2TATOorGtsIDJbMg5WNXHN6m7glrHPdOvMQex1\\\\\\\/ds2lJITIA1EF3r81BKijP4OwrjVlwFm0fjpGqLesHwL3gEKL+Q0PlFl5fGxS4crvGhJ6fRA\\\\\\\/6Eb7Ue5HhIGtULvO31HNuUfVm47ydziIpLNr8\\\\\\\/rUV+7TwtwaQFmkJ3Ko1kCj7dmsc6TOOdLtJk+vGEl9Pj2eG4DHXRLnIVopbZUKAZ\\\\\\\/\\\\\\\/AtvkcIouTw5NGft0Ti3K75ZLGNxY\\\\\\\/zhC2iLJctAhLEow1obikHNzbsI4XjeHWIgidbvSfXdgcmeCzgbLiBXUxOeNVKYQHIZZkkvPO7T3ILkCktb4VI4RClfuGc8p05srZPs9MLXocZGZPPKTosgqJcr6+LDksx7i0vw11fBWYj0DMFn2U5iAI6Uup8c0+9fdiDqkd+fuACUrXBXFTr8K8hUWKBHwyn36skYdQIp3BSeVt0FT5azB9676R8rArc0guD3AWMcqKa9CnLYb\\\\\\\/89TrEs9UOWBzIqMBiCLRzustykVeece48c3ruXvJJukFG4LIAjIGXec6iwhX3jl5vn65gNg7tiTyYz+egbNg5Bc2rZv0TdwCk87PeR1Vow+sL2UPGnmda7pleS7V1e+7VYVdlL+3f1ei0nb8BfI8BlDwg6hoFeLpD0hr2E0hNDIWwZ+mbiaCKMofj56nK+bxf+AOGMbb8CdVF0y4f\\\\\\\/92pQwAeGs017l+Am7m2\\\\\\\/+PLxHqLghFULCreuqmSvQx69wqkgb+VmGx8sM5H9swu8uLNkh39kVgcymSCh5vGsNimTLYnjf2YmMLIYkfYyn+3hpyJjxzAWxlezwer9yIz2gSTYXrolpbYmvv2gINv\\\\\\\/akXek\\\\\\\/CWNwAcl+ni4TLh3MpVrP5qsy9Q4KL9OnyPMw\\\\\\\/ayB8isFJpDkXXcpWLtwjnrNXyjORikkibMsPKCoFWNez3DCapBOO0ZUSLz27k0CUgOHTmhuRodwxLAJeGBvFiM9mwae\\\\\\\/5qLR0jUr5Pb3c9RJ0Z2BRB7cTJeYNK+yAePorxbDvOAimlyUZIcMYbIfvFwPU0NlHmmFoRl4gtm3\\\\\\\/3W6ZxxnxwnOmat7K3dTg6JONH1j038337x8nNbjPif+FceXV\\\\\\\/q11ZV0wfqIlDg1j\\\\\\\/xfFMGyQwZk6GljKTcYpYCHkFc49ArbVt9mpthcGCmkJjGHMFb92iwqypsI7JOtP7PkKzk7yhFILhz5MWTHEeWfgzs9eBLC6Ul25+eg0ws2gS5KweovwdnTUJBNTZxlIaydCvATHFEbnR0S1yXkTE9sngi7e75P8rBhP8JgrIAuymH1ab2yTO16QMS3l3I2G0rfMrHE\\\\\\\/JdZwnbyzuANRfErpd\\\\\\\/GYGsY1h23XCvveEfaeDlpbwCFpny4I4YHYO3Y+YTFrBU88gsPoac0fWjt0wIsa4\\\\\\\/KQNypiXPvOeYRmcK26sEVEHkkDsAtRfAXSOdS3a6sLffuElQNKOLZeF+akJRI2q013KrP4rJJNkVukoecFXYePYKdAyhjSPTVvXo10LA\\\\\\\/hEhcuqcK6+o7M4Le9EBsaoqwVgm1\\\\\\\/xGyZBSHpglr80fO9RBliA52AhwxZ4cTV8t1FhcQ4dOxhyhRWRxdEBi4YvSRF6W57ztlrLYSV8HjtMWDp6u0EMDbuY03f9GfFrEhSUyg9B2nSCLbI3UZju\\\\\\\/M7JKsG6SGYdtEErrmy0Cr58sJp2Kz2IahwxszsjOyaFOBFf552gQHL6gqk\\\\\\\/Rprio4sCcdY7YGBCs7Cxk2d4cEGliSEjPccAcFkkbSeyyZEhBLPxpE02Tmi4d8gsPNV2Bo\\\\\\\/riHCPLt2PGtVrdPHsLJnr5oofmlASwnbeXckntXasSy\\\\\\\/d5gn6C3Ls83ZFzBbvg\\\\\\\/GV\\\\\\\/Ph6wD4+U0xqzB8ohaZaJ7SJ4xH2FpfzPyb9Yj0f5PDoX3F\\\\\\\/TAU\\\\\\\/Jf65JFAyQ6reLa51ER\\\\\\\/9Jcy0oxlF1RayHuvtHmt4U4m\\\\\\\/fpz1ZHURNXsbMp43hVP8voGTZm6wMNN5RUDHlyKpGTlvoMJC6\\\\\\\/i45ix6ZRt3MRBIER\\\\\\\/7NYtloAZY68p5ixs8mGsPgRIu02eecgPUqjO6zD+g\\\\\\\/H9Xt7CgwW\\\\\\\/ge7RyLwLKq6IbXm2IBB8qO88UBSeXQUUQMOo+FtI2VeSXemOTN\\\\\\\/6E+SUIursk89HhsCBcVeFtSm4qQoAzaHrcUhPRs6d8VCoLV5edBghSr09btopTFaOxN7eWPd7LbOUeBKGyFuufGRG76ShO5v54MpUgf16JmH8\\\\\\\/LKZorpsUw8JCyum20SippINmS2fMaCuz5MXBF0q9Szgy8D9iWuhnZoAKB5kdQXk7RnJnavswrVu24vNUINUGxwMF0jJK8Bqz5OagdhWk3oafFNJ37myMEyCaUcNKLpUAASlTW+b11w7cYGJjqaiN+0wyxbzWjJ2qJBBRMWg+RZTyZ+Y33nHsx880pS105HFU\\\\\\\/6gHcA+LH9F1Iu2jjQXq5lVge0Uu2QKCM\\\\\\\/JhJa9MON4RouGxwEBjVQxQABNRjGLUCJqv4+LAJeAxHbBNQeKSTUINyKxiTKm48WroLnjPk+Kykg6X9l65Mf4jZDPTwNJSBNMMQoixkJILvo0AC4\\\\\\\/oaqMEZ06FYkKikixKs9JxfFuc9s5K\\\\\\\/03t8g\\\\\\\/CS2RerZ4UFl7W0a9PCiHIj4x\\\\\\\/CIA1fVj\\\\\\\/aQkzoIilSJS\\\\\\\/mmx9vMnxNef+S\\\\\\\/AAsrnKjJc+HmbiKwUR8C5UC1qBvXtLBN+C9kZr3T+p+A\\\\\\\/ascTZw\\\\\\\/b\\\\\\\/VVffyzH5Jj3LJFm06INtZgryPUkwN7epqm6qGp+vsglyCros30RztCYqQ\\\\\\\/FG6893imDLs77etDJIQzvqVx+xP0R4mJX2zyUmIh\\\\\\\/RJBQsqn4SMEhznbZRA2apq56ZS\\\\\\\/HotmgMUWWXiBJw7nOWIgJhWuF3\\\\\\\/1WoDMqHNikTGzKZUo4IxtTpGsHkAAMrohboBirmPxEdu9y+ZKe90ktkpkXSdH3ckJMoWXSvqvLWC48xShdJ\\\\\\\/6De0dFFhz2m1\\\\\\\/04chVmqloKPQyvX0RiJGO9gfus5aiBuEQAoWzgbPuzCLx79CePgeZ\\\\\\\/47mpuKGAu\\\\\\\/K9WGkxgNiai3uv\\\\\\\/6qmdrSBfyEl3rL3Am3+Gyv\\\\\\\/MGVXb0flEJAa6moWOOKfPhDyb+09NVTD2DtLr4Pb0EJ3O7vDbPPtPoAQ5zK3301KSeOmMefaBL1hJ0z2EXKV3vQriy\\\\\\\/fKUxjSJurWDKm5lAhgmtMqba6V6BcpZDKCRSRHGJqTRH811UjA4KCw2ha5hhNxW85SbvNWp+vgGbnmnX4tSuZ0UQxvOxfK3hzxZCWDZ6+EzbxsoLd0WdW3SiBxJHjpRsoCYe8pYEtp7Zaaxu0XvTsey5PZf1xLRfWyVPJPRhTLA+hg8IMatIk3tKNxbAE41lx3P2CDziUxoVt9xnHYeHBJbOiDt2lH57U7pMBcOd7A73qlIgmHqGIBaIxdn5HqEHXFZ4I\\\\\\\/rM8hBTidKw4Kr03ANfteq3jdVASLwG8+e3yt2sdTrT1uUxfVZfWew1EK6GpgakYfTR086alvDz14sqJUawFcpuK\\\\\\\/N5Qiznrb2lqjwiOXEcMKC+9b+sDvBXz0vZQLD93bvLb2iG5rvAgHc2Vs4svavk+giJcs791fRgzMu6jjU5z4Ycjwv6x\\\\\\\/3uU39W95ptAjB3pCo43sXh0sZ1rTe94bno8MXqgLE4YMH1KvU4QR+5fYaNVAfFoviDrPwb7+oMydnES4APA+VrON8F\\\\\\\/thX8n41UaFLgwaIG562EAG9s7+R6uv0pTLN4naKltWFUjJ9VUBjhwhQsGDwB13S\\\\\\\/06MMVBCbO41W6bUv6c7eFThlooKW8nOcLi7zAVrzGRxPlRnvjK68Tnp1ZaOkF8lbhN3Vpo7VXcT+DFbWmJeIp5Zi5TzWF\\\\\\\/sJXPv+iW1Zpbsnh1kDoMJcFv998KaF21HMoYz0DtQucOQrH1IUnV\\\\\\\/j1ifhi+nEpWKUOboljq9aF9rOA7yz+Q1eCBv41kjQLEcmIYcwDmwOfcH+0LNRQk98m6bSqGgcynFYo3ruzA2+WXSkgDV\\\\\\\/OLbQN2xduI39wOk7VKDZE3k0WCAChMoYN97bahfZzNmHHVhSTkxS1DDzWklvufS9DwuWKleng+m6SrdKod0scv1RpRpu8BMkgLkuGnxUE80Q8RHipjuSgNzKFkOyslOf8X1Man5rd5YxCoQLhdpn61ClaFmcvUjvnUOcMvCiANPahb0B2zznPmj01tg6BIRIYlu8Cl9J0hEVcdQKoBwU3zRHt4qGGUW3JbTliKIkZS78pOB2lUaecnWnU0nHP+xcnbpt6USOWVo\\\\\\\/CmXvtLLcXKovAtIqNNa3t8Yf2eXl91BJLmSEEHBA069ZjA6KDWuCvUcYhifqAATcGBsV1E3luJ\\\\\\\/PciQ\\\\\\\/HeuI7FflxHT4roMF+G3gKlBNciJJEJQZ8muzzMd2pf8yWTysv+rQ6o56Dd0YHuKoTNLh7tAtdRBIxrtHex\\\\\\\/DYkW4cQdfxOgdJ9YNfJr6T6yXUDRSRGvTFcEVTF1v2mPT3gzZFq+kzWqXP7VCzLybtDCHiq5SPhUQ9s6siEHX5q8+qdi56Jz5\\\\\\\/xQ5MM21GVYN\\\\\\\/4cd0aIYvKMNlHZx5fvvCk5rRcRnetJ3C14fogj7PpOK62umrOcZG70hEprDHCl+tklICYHa7pljkjaNcL\\\\\\\/IR0C\\\\\\\/mIYy7R1aaWY5CAPkeU1c4wqzZCDQfCgKn+Ceba7Hg0ERD89gidEuWjXWKcoQ2vraGJKoSrPEOyuY9M1HjgJU+mCG6ycwZqCe38u07ta12OpjFFVB1VhH0WvnkMJYoych3zboAdq3rLerw3cauePsi0UyK5Tb1mojiSamwPgEMRWJVOB1BhsFrk60u77de95nr2MnFySmjG9tjP34paP2wSmhyZ7z2ALSjSBSPaiXKMuR6tcV6vPBAOHhZWGj7KAbHFoXOAdePQyvd0WApwsIluJcQnvVqWlR918j7PHp2J+JEn1fIMCw7SuNuIIjW0j7nUW+IiEoSziIitFAijj3vLcqAiPz8pVEVkgQuwJsKSCTZE3bt2F3mWn1Or6Wcfzd2FyTRgb6WgyviC+f0m71sYe3tA98+l78jhUyM45r4CLtYnDt4bH28P47CfBgktT8vLVD\\\\\\\/837R4hAbq3Y7gSpJes9XgU2ZKCHzzti0Mvprpkd4fw5OHfxJFjSJrIDRyXrU3NPxDmCPNUu3Juk3PhcnO+jcxLaoX7Cp1txPnnVaBOEH4UymN7bL6ROY62rx1aVy5YaJODgPqcGt8IMSMIhQYWMhjjpfWgRv53Bq1Lq6PqtiVM0Ef8xvGfiK6\\\\\\\/\\\\\\\/u4\\\\\\\/u4hIdI1RmJCXnnZIVCGyPTgXL8azHBDFhvPGuDldYqGJADdzGyvHo8IXCvxH5IGobyKysTIkZKUp6Hm7tMCqYqrMH8qFVojDxdk6U4gzOn+5nSnlbW\\\\\\\/Xj5xhFOW08\\\\\\\/f+rffO9ujkrCnr1aOwxobGeICSDebyvT+ldnN8Fh1MSrttpvKVqrRyBLqVif4xxapJjwUZN+X0D4viidzw4Os8Q199xRiMXGjPoxvTYAIMdxk9gAwIahyRsLcvsOKjnO2vrxMlbuniGG1JImNGWUvFKQTcfWEnzMYX9Y0K+W5dXabEdvUYQRRut4iqdhYVOfSpJ5Plmln8Zze2S9jw1EZZjrUYwUJDZftTT9bZHpb+xBIHrNS3Jrri\\\\\\\/1KfDQPjj4rwibJxo2OCUe5f6BPy1ZqNFEotfjNVj7iBqdhjYNHNmJPCsgb2zb5+4gSsEAzImmxz67TsEVTRssZ7+v2s0ah8EQPfbuwvH207kg4qfW9qzCWge38AugH14ciF7ZvrwZCYP3SWh4JeYn\\\\\\\/Jpmf4o+BV7SYCtcM4zDQTyLUKsg\\\\\\\/fdo7Ax4LXhX4H2jrR+n26s+puHIu1VWOWvi1GGY4Q7V8WeOCv3WX6zRYTgACFLJQlNGcZZ1gynM6Zd+o+V551WBby1TIgU\\\\\\\/TNSMIx\\\\\\\/PBVNEfijm8f9\\\\\\\/1E6WtuDQR7Acw+AOkpfGf6IcCZYcIzyi0VGrEUOGSSXG6gW+E1Bf9VuzQU+hqaUjpNFzldklb3SaPyGO0xo3DGrG6w886\\\\\\\/EzV5lvNszgUQOTyS0WdchW3fmKMzCoXbpz5v5iTlDaSHGudI7fFEPc8zrTE\\\\\\\/tBJ1aUHUgV04LcVr9o\\\\\\\/2hYXLkRc4XCj59HxF1Ul2ysAdAHxlwJ+gcEDC1QdztUwwJzBIX\\\\\\\/XGGNODd\\\\\\\/sCULDFBeKCN4KdUKnKJ840AGIuoYlfeFs5VDKTrV5bV8FOKTjiiN4eNWhkyIH5DVF+Z4KQSbWgG2N1bhDMg+Tlvfb2sFTrZPcyBou5TnZxoF53MG5qqpIZjWX7NzbXkhDdIZX0ZLCkQm5qtcKyHWmUyNAL8ulK20mD8QpEfv5edBOQ4pe7L\\\\\\\/CgqhjaJ+ds3oMZj2AjMbfGevzC3bVGyQSCiZcV8eSfq2Sjq+aKDKyI5WJXljdhl66zJtVvsFUbA+8a\\\\\\\/+tL908NJorc6xXgb0NxeE0F9fesM7zStd7BvB193Q0KsSbgShLSINyIkTE9IEveohbzmr3aQJhJ4B5URaprmNS1hHhxW3oZWu6rTl8+VMYtMA5gyDDnMjU+k\\\\\\\/aqPjeDyqVf4eM1rE7sKG4bK\\\\\\\/9M5A0nELH52hSqrWJVJu5K8ECFh9mOh7IZe3mbB1m0Lbh6UJU0T8\\\\\\\/ig89tHb5u5mK8NCGQKGSdaiskcIxzUREYnMlnKcWThuIQneyORfqaFHkb8X+86HrNMgHGv08351jpaYVNshd4nsP6JTDXT2\\\\\\\/PovvqEm+jwFAavmhs4iQXDCFAwEJ6pHhpZKci9CdwwNwqt6tH96Ey07VKjuqbKHATtnXAPqDeuipEVj8Kubil1RCIGuBrJ6sMEtVM4Lv3KlgSRhLeXurwJPCggWeISuWf8HGpBje\\\\\\\/ETL\\\\\\\/cTY0UlLCrb7fQQsM2aK3c16WPYDuIyEOHAr5EC9k3gj0Q7qMcM4HQhkQ61mx7DL7JMSWh7+0pszcgCdd9EHwEtV57WhYNe\\\\\\\/uFDqWDR2DD7lH\\\\\\\/U8lwTjy2k166931RCLrxO7uvvGCUuFUYmfGTl6J81bCs7WeJlw1nxt\\\\\\\/FullyFOoX6QiLvy7+CRph\\\\\\\/RXSBFRUDYbOPJckkNODPlq98xvTOoBarE22Dj8YKcHmlv5iotcC3ZwRjLizXuJyWvzYXxyvEKECKt0RAv5zKFYVhUbEziZwgcaL+MZbyVf5q4zzS9cSd88C\\\\\\\/usKWTXREiEskL\\\\\\\/M\\\\\\\/xsW4dUdingyvexvp0xQrjemXfFluYct1rPIz8+Q5BlOU2c+ynAQ3wUbnkBq7l9yu9N3Fpkn5r5FsaPqAAh7VltkmFXeUwxTKS+Ji+0fFQAt45FyBwG98GUu03zgAqF1llDPtGORm2uh\\\\\\\/tZLTDLnVlW18FIckIBO2QkWr4lAdxsUIvzG44GD5dsZW7H++vs9RUWH1aPa5k66nAqc7rhcqbCENtH9LPmkghB51ATpD5JBUv\\\\\\\/OPgUxebu0pUa03kifdkKZTdk4U77BTdqemb9AE8m8df+o6ZdvkDpX8PT1eBrhiUxmidED01srfsoRkqqo1Nhc10qKI3DCrgNUg368E3CH5Zs5djDkAdmJj86eH+sXqpnhg\\\\\\\/KpaNJK5UjYL0HQKfdeiojboaroUly1\\\\\\\/aq4j0pQnUxCQ0C4y+uKDzGCYs+FK2AKNqzvVAzvKzWnjvSKh3XsMfvZEwBETuNl8mSOV14ajDgtEIBi4xrEAhNjjeA0G14BV7LduVHZIvgj1ng98fVPuk\\\\\\\/T\\\\\\\/nyq3Vwdf\\\\\\\/eTbGxPv7\\\\\\\/BnPHG04y4HvPFa5AQFMOZnd9Mu+fvwHk0Y8bWdib8N8e93e9dNgtNGW9QtDtOKwM1fdyYWr+m7l\\\\\\\/qHxz7s2NnvTBQYNQofQUg\\\\\\\/pLVBfXmmL05FKg0u9W4l9KU02qnKgqp\\\\\\\/xL\\\\\\\/sI\\\\\\\/q2ukLm3cTde79IrjjHAsT0DpcFLKOuXhQp1kYWJoJ3U5fOY8fN2oglFrK+qJO8IRZ85+hD8Joc9Rs9IKlqHiqqVBhzFNMWNozV\\\\\\\/HHk8AKh+y2YbG9QB8+SYG+6\\\\\\\/b3QjZc3ZMjdiXIJytEQlLFuWs1ORoLJoqAd3bSMkxYREQe0U7TU+Xu1Z0+898iMwS4DFkZx0A98iz2IPsaDWULy2H5vGgLQ\\\\\\\/a\\\\\\\/cmNTKySo6of4tjKg+TNg6En9zCvqls\\\\\\\/dDVMPBkXfrKOV4Wpd+bjYeVKRtMZKnyRetMkA4zgL5zVRLq0\\\\\\\/NqADlCuD0ORdE0sdPMsPrT1Qk95jQgNxyIf\\\\\\\/lsNdT3RWtR+X\\\\\\\/UzEiWFlwqjDu7t8QveyXhMNTb6pg7NsYFWQ+JkarzZaMGNENPS2xG\\\\\\\/lJ24mt2itG\\\\\\\/eR\\\\\\\/53+h9Oxd6vHKiQkCI4+zayNeAZAnVlCM+8YWQcQE2GebusnvboadP6OvXGivZ8yzX20oWPu89atf8pqF72C8XIJIiz+CuwRN8N2N8aK7y83QIBp25mSGchap7WmBg0I9K1qcjhb7rP8oSzjcWeoq9G\\\\\\\/48NaS65xiTI1H9m2c\\\\\\\/DsW7itvvXTz9IW\\\\\\\/jDFGFQYbYCVT+kgZWT89MF20Bzxky8u4jACzPU8CjY4XpSMgo9sCt2R8wB+sBnnvEWxCkpBGh4ZxCAzlkF2mg7UsIEAsLOQsOASGnaEcxTKUwHUmdBiirAI3p3Y9hLPE248zC1ggMQDlozA\\\\\\\/6nQc499AhB7wq+\\\\\\\/GHsqTEhbM2FMa6BvFJSWzfUQDsBq4u9xwOfxlGPbCDljGUYUCKhEd5EDm3qCDUpNloV5OYVOOVUiEvxG10tVyre36o32BjC6c492X24hmdgc\\\\\\\/gCAXSGAfLIqkPGCTuWxwVTJJhr3WfxTiXuP6Zgvi8cdvpiL+RzpjLUAzgpDheFoNiMLwAl8pb9511spGf\\\\\\\/O8I08VYtQqLZ8VM5ZFfk5UuNa34kQBw7DPuJHQuPwUQEsOGZnUpk\\\\\\\/6AtoIkoXJ8UrwBiORS4SlW9XL40hdRED8QbluvgXHpCF7gg1WUqjareYgoGJSG19Shtrbk+Xyij1YcUJQjzUCZqPu1dggbWMRlXDyAsY0OeKoZwaqiPWG9UpAXFuc8uTCsX13OzPb6b2AfuLDVJWzC1858hjckQEsJvY+cPcLmz7WtFg47WlUyf7E34cQN0dKh7Yesp7RTWsUYmwdZDxOhV3p6tr8Pw8sN7LFqngl\\\\\\\/QpXw7+GYDtxkTMS0bc7rzsOQvtdxGgwamu86EefGCf1GgUEUUibaiL75bkxCbItREP0Zx+uDy5filTop0DWhRUB5JgxkuzwSgpDAXN2lHt2nH\\\\\\\/2wy\\\\\\\/ofSPf\\\\\\\/RCrJrXDSmcvSdcG5xngPnuhwaeFBem2NF0O11d08Kz7Lo64nIpo0XqFWeOs\\\\\\\/0IYNnaxlBploiiXWuohjYEKJMh4Sm7QNWnOU9t5TQQiRrQlgslNc0VeryAmpWqIFop9p1tw5MfmuwkIzx+knZ0KyIMOKAt0sj1MAklCmJgw6TLEvpSXCorsfsVlN85GdKV5NtdOpGCcj\\\\\\\/l82H+ngEAai160ZVe99CIVivh9AArjT1O0lt+K98ngfAV\\\\\\\/HQlQ03i+PoS42QGhWYeKfTwWaLAgfpSkhtPS1ebc2FDsBXuuUe8B+vO\\\\\\\/oP30anChrY6wXXqxgqzWHdAGdTKXf\\\\\\\/wl\\\\\\\/+iHfa9AGELZhEBExakpuT+spsuI+8Mq4EnZrEQKRNQSiGhuQ9W1OLTVj5bPu503ZmBNif8+dGN0hd7VOWx5Q\\\\\\\/Ead78bDc9RE8kJvV1weG+BxoD20FYaN7UtZa1sOQMrKGNzNa54g4RIyei6NKqdSmUcKM2uLtwOsrfjQmz8MNMcWHyBxSE7Gz5aHBgiDlryYBIkWtbuj+AIy55Fz+UNQH7OuDyj07yP+5LgEOWMqPa6zHoZGWRHUXmFpnIP\\\\\\\/ZuAs3apsawpzShik10\\\\\\\/kz7Jly\\\\\\\/qkXSEvnXRoc2XB1eDvyyyu5QWOAYJF6ScIdVqjA7fnjHuNcmpyiPbKtrS6f5LW0OVGrunSjXXhEFtNTs8E+NcTHF1fXuLIY09p4MvSeZTLfAV+jeet+43UdG6gCo\\\\\\\/YxEXwjFPmCpZM\\\\\\\/jOorQyj4WF631GfDZrD8pcem53Xm1H55nCx3AVkRM3Df7TPYOvwtPC4ReyuNia+TM1YzjIf2zFN0OZVmnODRnDSYLtdfIzd0RcuFy8aB3rt+N7LdzkPRWlKP9HQttSOX3iKZ512K1eIgcpjzvNuSStoxVn4gEtGOh2v\\\\\\\/\\\\\\\/WkbNohTiKQLlblGjJ8YdigUhBeCvQYNjInmLE7hSP3bpzgONsxrmdKdZUhdk0OJx\\\\\\\/OryPCaJeapylgf9iqJ9Wp014BUS4n0M72UJHsO0BOHhJyQ3EELcNWcnQs1opdvW\\\\\\\/Hqxz4hScxc18S0lv9VhG0oRXJnVr2aVYy+2QQqNM5DFbFLSCpy6s73BUET0g\\\\\\\/oEwt\\\\\\\/7vlqcM54SligrROWkz1sHzCh9dgTfzesvokc4d5tlbYXRADKrT9Ic\\\\\\\/M1hrq4qAR7gCjKeYluCI1l48yIqU0dd1USOXB9tCg+fZ\\\\\\\/VNA7gBJfRIBZPmuCcaNbQ\\\\\\\/mdtktD92oUbJB00aIk18BRNfEXcoUDRtF6uD6fHViWxEO9duuYa99bNnZeArSsNUesKnYP21usWWCKWk3ErRcLWjKU32FDSSbDu0o9zbRheLbkCui8Qds8FpH2r8zkclLgEFpfjkL4f0aZBQUOxcDRU9\\\\\\\/dSXs5PBFQYEfPpT3gP87Qbn0hbfzXeZK88\\\\\\\/8LUoHXm0t9RERPh3EDpsBoXSgaunEKVSIrFsWW8DKm08dE0mzhxo9dv+Y6rkAwwedpUqy0XZwEor\\\\\\\/rKXXgUkzi1CqpRlDQyr6y4uxMtkWoWdDZRQ+h9NOakyUFmZlpDWnModDnRgpMY02VgqO8UGJ+qPihmlKp7JEjBdVotET\\\\\\\/ZL53153E+IBf2S5w6DswwTpbd7cPhD9XGVlZOv+WMJcK3mkod5rRnJStjyLfd6XyMHWolpvcdOfvlJWubvnGhUN\\\\\\\/uu8Gjwn9CscHRnUpW+zwQfiCBx6TuEvVpolXDocZgNRZ+G1bWJUqeV9J4lKw8yOeJvqXjenfK4MHQGGFsjaJ5X5mMZOfCqerXVkADdQalbHW1ikFZV8VIESzZrf7vFS\\\\\\\/Y\\\\\\\/pqm5TRol9YceyxPMRBB9r555fyALoj6sdspkBCIYjpMRFttC3Ja9P2g5hCrV7Xs0oD\\\\\\\/IZFF4oGmvZz90O1YB6AbQ68dP7vB0utCZHnteJT7bIXrsnBIP6nsMAtUrMTGj\\\\\\\/lnbsHe91uDdkrgacf6HqWvNey6rX0Fd0AR\\\\\\\/\\\\\\\/Rvb45XTY+WIYjGB492\\\\\\\/cX7hMML7\\\\\\\/ySr5Ks8485GgDA2SlY1+axhYw\\\\\\\/yM9UKc6EJiqTiU06+UDjNpzCE\\\\\\\/UZAxZM\\\\\\\/vxQhzAgxq31weFVJ3\\\\\\\/2994SiyVk5Al0oN+LcPPOAF41JTshJJp7b2b\\\\\\\/l4gqfPhvSOwaqttKE5dSnyWFWUQzhvMW4Tr2SaO9yK7+qHqR6d4U54EAEbMp9adfNilPJzs\\\\\\\/p49LysN0TbpZBJybOcTJhIW5hs44i6nbCt6VaQDT76ER+wWEmhYdRbnx4ikr7c3hMje0MMIzRWRfEN0xBOBQYV8QK7G+Vvl3uuSmc7tugTV5cHKYphjQUR8hKUwDq3GsrTR78XKyRtAMYeLz69xzVSDXzkTKXyMp165WtJGEgTeX3X9ztHTfaqoAdG6A8Eu\\\\\\\/8FysCrfYaR4oiWQh\\\\\\\/gW+bOGquQwzL3zOqK6yvqRJjg3VPdzd2v0DNWBiKrySLV8VbyS7lhemH0FvXd6jMOXaX+PYkLzAj09fssE5s9AExxtSKQo+EXmx9hu3DtwYaZoGqST3VV9qZZj+M3EeyBofRO1BzplQbQF+b0C7lZsW9HlSklAhg3NHxGUgP3WQGIaBssJ5eFZXPbgzLQ5TeP5xaLroJkdmKW2SJ6VAArOc8gWFsRy+by9usPdUG+0hRc\\\\\\\/ltPoucdq\\\\\\\/L9IUS1hF\\\\\\\/NBvETWuATx00LuHEIc\\\\\\\/A8sp60A5dkDc\\\\\\\/H9T07f1hChHWF3tvkDYKXZWsjqMzIxwdgqjcloAJyFD7wwqR6vKibDSqfmpMvgF7rjZ1qveHBB28AOTr+C46l4dVgKTCM2PF4gMPUXTNOAzskW\\\\\\\/C0jcn7i21icM4D4+7FjXp4D3LYZcYxfq4dHbHftBGiSt0A5y4JrrRQSrlbLwmh6f8sbVIai5iLl+L5ATNpLfmDg0Vgpa+w2TQlwUQ1ydZif3WdskgJ\\\\\\\/UOEclaNMGDtDaCt9oKPRcRrRvWiHC3AiptblgZPeGX+AkxN9OOJLOX7oZsZai5yI9IidpAYkPNls+wgWFGWz2OsUoPMXxxiDO5WDlO41RwU7IpMiUzwfdHtu1JDONMXI+3w+uTQEe2RkgklZtGgkIkxKultDCOa8ksB1bgPVHwFUcfwzursTBDdg048iCIcWFMREUKAGODidvpdwPml9b5I8ouKP1bf\\\\\\\/bhoxkGUT2+6Hg7xJE5HEp3RvFzSMcaOif9097fgrQPsSU85MIUjw9DSmqQfc\\\\\\\/Sbb\\\\\\\/2MbpoukKnRJAyYx3k1Nc7\\\\\\\/UI0P0UjMybzCM7yKZrCWFCGs\\\\\\\/1ZQlkroFr\\\\\\\/1Kw4O1sihPJPyS2DDY2ZU7zsXelPeJGSjsdpbgodWltrheQDuZeIm8BbdqSyJLBLYmWLIJViWBP3mu1lp37r+KmMA1gaSKosGL4QWj+1NPzChY9re581YtvO3B1dLSUxt6TPp8aQyruBws0PmNCBTYzddauWaARQuvzY6aF0G5vOhmg0V688fxLN3Xvz2+8Iazd03vMXM74t8NihqsLrGGBmjukBcxeGj\\\\\\\/b2a5XmRwDdGanbb3g6eE7cLny0JN1dIjZ1jITQW58oFfQ8qModrvnMtSA2AyOs0XxTnLuDSn47r\\\\\\\/geVPlBa54iP4kB9aZuQgqiMiewL5DDw1BxiDYyIVB+7zdjK9Tgq72bRVoHTnyyauWYAjmCIgNwSe+oh1grnDTrOh4341eVoXhgFK5J89RrmzkM6Sj3FRDfhg4klgex0n7Ckq91wX7QwP6rVLUSO4QYnNGFEenwWKu1yyVnMrm5dcuBLNVmWfMxwmUgf7gcvGQmQKLMQEPOTZP3YwXBJjfiTIiUe12QtLNry+0k8HrBOAzL6vWgfING5xzrhUVXl1GC8tN4eWGF++msZ8buS034M3tJ+PDxdETG8eGMA8wFbhzJuWOOYsMaCW0VMX4yGuWixd0aiDZqQlebSHLGqPQ2vWzgMFGVDefbZ\\\\\\\/NhRaqm\\\\\\\/PfllHaXGKCGQwb7kvlf4u3SqGtyiXzGrx3Etf6fBwUxDmHtcYrdibGxlXDl6PkVuBhkko8I7fdc+vgTrQVUctFBLeHRzVqeLKz0tSYtLF0k\\\\\\\/HdA2IRTNe8LCOhsBDp727yKdvqnOI\\\\\\\/05\\\\\\\/VvbRP06J\\\\\\\/\\\\\\\/gDbHvj\\\\\\\/GySpEZSe+XyXjnMhISL2CdaFVLenMHR0LcvPtFhOwPy5fnLTzjs3Vch2oUzX+XujWe7uh4cQ\\\\\\\/G5XvAkJ2TFOFV1Umm3Is6+swuAHMoHOwBhvrSoM+n+2YzAKN85wZgNBP1y+lg0W9lzNOcKYKRZKcbwzc6hdy+LHJ5d60\\\\\\\/QVdeD5T\\\\\\\/6Mf8Tcq8c5x7W\\\\\\\/yHIsmRT+zjN4hUFm20ZmWM5+YV3m7rcrVRuspGru86fIimSkHNw80v1XBIHhxnJFiDLxVgbY6guPlJUOy2PQRt0q4QRCSjnzpFCCSbZH0LaPTr0Rfs0hmpgXdtYmppQrHhDbPuy57wVCAMfvc5lLmlR0vtK\\\\\\\/Y9ffOQ39SczRet0ZHKVeKQjcYAPHt\\\\\\\/4dCvy5THwX1o6Eu9\\\\\\\/lo+Rdg6RNZjxbOf\\\\\\\/vGiNeAk\\\\\\\/KIeLybwKFHtdqjUGTtuZ1Yxz4YclpNxkZZO43juwAOikjAzkDAgq0mYyK88Vnb1KS\\\\\\\/LlwXRNYGDRcnQrEsWIOGz7D+KgmKGlEej81lSkZ1paUWLjm1Xky2AkqqHMg6JrW\\\\\\\/kD+\\\\\\\/pOPuJSRWsUpcwFp7H5Q\\\\\\\/efjJlb27E3wdQxcDGFPVWCEi\\\\\\\/P5UMeIwUAOfZeEj6Ne\\\\\\\/+pSBBbyaVG\\\\\\\/wMnpiSZO1JdYSKcU31wNfdFDaWnDAnySVk8FHteOXGxfmpBm1BJPikhIPJ0YBwIACjDfnXKLfHtS\\\\\\\/agbOT+OiG3JVJp0TDkiPmrF4MA58wv33msc0i7gqCetyjkDDgFPpW9vZopejq3G0RiMi4jKK6dfrJvec1d8V1gUBxJJWVYNndZ4fXBV3C6WYHSeZ5hIBzSUWYTT9zz0AWsgblMiLWbMlPT4OPzXa++fDO5D9TexrbpUC2N+TB+52+Aev6DtaJgu9sN0oG3YbmCpTPXtmmEg6Q6nGAavWLD1KIB42w2t1BX5FePgfcJDajUi9xGXTJ2MiTmQy0DVquMNtRsqdFpxk8vhDraK8t0g4RzSkmaeoOYKaz0Qmy936iKnKwPLf9cJ+3LZQMzA1ChstSemoPw7Utbdy3n4LSiHNp9UeTE8pwAIABB2Nrdd3mW8c\\\\\\\/qCP6QTsaqsqicK4jz6z9C6SZceN0+F+PVmdFgEjPjQIDyTe6gG9vpifnz6EUIql4fFw2fcC96Vz0geV3pJ\\\\\\\/guBNANQfC\\\\\\\/mr7QwYX6XVWbLzoYktpOttkSkkSCIsgUGmcu3VRXlpT9cvPWCzTkYSktHsIGTkmddJdpopp9CaLHAZp5dShmaraqJbujoXMHUcn4xG5suYtGyE0dPNa3yIxyuEwuGrjY4z+ZB6of4MOKxdRrBLIpcX4tAblbBsXOnM5hRJ0a1apJUCR09xAw1\\\\\\\/0it25eZc\\\\\\\/+cO3pW1XoBVWaI\\\\\\\/0qA9s5opKinw1sziCym773Dxos3KKBLrD9pDdOrQMmo4P6qVZGMg+yEDGyv77PIZKHRxFRLNYuWhpp0dydIz9a50Z0wapUbI0Kw469nWBC4vW9ALT7qJkjxCVL7\\\\\\\/dyq0H4MW0B6RE+jE3+dtCZ4uRXCbgo5odZ\\\\\\\/SYBRsYDWwP0Q7ndoC4oSR4LymIZgtJqD+vjK\\\\\\\/5lLzOuxhlQA7rr4xq779J\\\\\\\/SjrIwJSfodm38USWvC\\\\\\\/TnZX8VCP68cYBnDrSBFQi2MSScV61iReF9JazY\\\\\\\/WSb0FQjjQh3Bpp2zmoQqVHTieCFEFs3MTUwrHS\\\\\\\/HM\\\\\\\/aaFlA8kQM3GC9UhnS2E1n5utIw0OsLiBFqYaUp6rBaU9gxr\\\\\\\/I2S3Eykeu8wuK1m\\\\\\\/9M52X7NtnJojNrxs+0w5F6wvxxqyeGWXxskt1SWLlz5WTOr3NT2T1FLH3aCOxcpck4Twz+8tYid3xo5Yki\\\\\\\/fg+31n\\\\\\\/OIq1KIfgBTB+Jqj1cehLgw94g8cAxlCxlTBRprwL67xkqrY5qxD6KdAYGD9KoMCIKf0ginJ7TubwyHutz09qqV9J\\\\\\\/zBO8urd1Gy6nR2zCnUQ6GwCGLTR64gfDMIOCq2ZWS8oAwOtWlm4EgubHVdzTIp5mAoJaiv9+BdMkR1Tt4wyCAUuTul0tCWWyD7t33oYGWltOtbN5zT5kfI6W6+es1RRrpdTjtpi2mB4uEErot6emAcZO1kcMuakmA2hsZKpkB\\\\\\\/OtwqZYtF7vydzjOM7s6U68f8omuYEqHpV81OvvFSZaYbSlz2JGW7tAw5rl0AVK1nSBhNV7fUozWFv\\\\\\\/yRRVLQQXCB+hlin5vMgshcoRaF+r3Gh6YDfoe5sXw6W9Idxo2xEI8fFz1tBi90ezpICyJLvPHKUo7wpoR5iJtoCtI61BGBy2dt9WowS\\\\\\\/CL3323qdEDdrgcXmWJg04jvVw8U6+6la60IeKhoerFLQsSOlstvQ4E5UMPfNGOwuQTUn3UbzRS28UVO9WQ2r9AxFME9stVxmN2ff7b4IgoOhWeydekRtNUnI4EtIMpIMQgMdsj3yXQHt\\\\\\\/iKnKoRP9s02PbGD0F4ChX+RAm2x\\\\\\\/fibw5zzTb4SRCB5kS6TghiIociu+\\\\\\\/H\\\\\\\/pDjWyJjs8Btk0r2ljFY\\\\\\\/bTxy\\\\\\\/XuRsphEbSkw5s8oDhpaZIrUDGJdiyeWPPnirDKZVEEEtHQazirmGom1vMUNcNbd+FMsmtMdD5+gY50\\\\\\\/i6dle3NOOFJJp80nBnouDBJHL8N3FM0e1pMlM\\\\\\\/bU3VJw1t5axkkd81XUX5slFyjFvJJsIYgG0VwD2AmjOACtOxdlOSWohWlozhy2GY7OKNQFzuyV4Khy2+oWR+buaPAu0oDchQipVcpTKBcx2Mr5SwgWUlvUAYcv3mTM2X9GhgBfwJYoo6TmD9vem679Dhvjgq0qNPll4L+R0obi2cJGlZCT9OPhNOEadzj2ujQooNDfnWQJ7cEn+BB3QL0gbYAA4+HTfc6MddUTAZEaYyUltF5PSdYobVfiY+bbbEtN35dFjs5DT6qqftSPA1ytNnicJUxpK8A16i0jjEU0wpEkVvOewfOl8UwVxC6vl2uyYvjDQYBpoxnEo0arj5rK06zGL1lVu9U\\\\\\\/xr6AaDMVfUnpeMDko5ol12XlxryR39Saknkkm8DHSA2OZgpGQQUxIN0ewjnR5ri\\\\\\\/k9mYbeQtjHLfjdFkwIYvsBPzTiiGYewgj0Z8JgOiPe3tnIX76t0ArYCfWp64cEl\\\\\\\/+IOGjH4pVWmxeR5ytcrKW1ytDWttw0OhUU5xK6M5uB\\\\\\\/rFs28G1O4yKpsBdRmL\\\\\\\/MppeYD6EWJ1BKByVvnKIQrLN1PQs90JMmx8c\\\\\\\/dscmOZnIikpAeeLflLMLtFcb9FZd898RtmFrpyNifCk71K2x6OvumsQxZHZI5OVRwoq4fYlsOFV0Rcydu5UWg6ShD69FIY+7u59+hlOtqua+33U6ZngxVq79hYJrHGTMVwW99zP+XIPoVxsWP59FIroj4p3ob\\\\\\\/5i+LcbzL+aOs3aG2nmRZ1C\\\\\\\/uNDd0LYhR\\\\\\\/BWqAgrUzFUYzV1tdKLDabCfBT+\\\\\\\/nzaLRqSB8tmPtDhUCJPERjyhvWc8pvuuJNu4Ty0NBsJz8MEzbQBmrRUOK8FCQNIOVDN2rFZbTBfNsdzL9GmYtv9+egtQo8jVddjFGZehv7l+xmQEOEA+LtTRsyjLTVBjFVdTCVskRGbPFMGcrie67xxtyfmO8bw\\\\\\\/fX2n9++4yeEfFZRg1kDASvHGuMraGTDRcEyrE\\\\\\\/tidI2n\\\\\\\/2IJwxbVuhKhBALTp1fw60lNz09r9OC\\\\\\\/\\\\\\\/QbOsm9U7NSIpWPp9TTXDHKyM1xsyBKRqBUeeR3Vf0GPuChTi+dlkUGAhpk\\\\\\\/aniTFb0R3G45nnRM1+N68natPCvCTOLeNczD1pOIuo+9MmAGaqXviDLQSWTl9jvtrJPcJ9N8LLa0vYo6\\\\\\\/cq8nCsfEa64e+dRkzRUBMzzAQDVyD30u4avAukZbAUmZovHeBFuNda3xI1i5XY976KFwxwx8jlOJ0JIm99eDYN6PYCxFmxcb4CrJNLEg0E0XGutHB7v7fVUtKfMionyA1pvq8ZX43Wnb53ETtkVNTTgDEXQpS4x7q\\\\\\\/crzLu1dDEdTVKTWAMHOQw817cVmWlUXy6PXo+pMtWIRQoO5as\\\\\\\/Jk8SC1brGTzGx2pQiVFSXepPurhHODzaSOENJy0ZeMLMB1z86ocIo9+dn5uquF7KF83LMEnQbYXcJAifUCGhERfTryJ72JRxjXDYRSbtch66rB89mRjGEuOTHGwacomJf+nWuaU+ddfRyOtOzvnq6WW6chRMCcrryXCBEH0KKvrX4kt+S8ktF+GNdO\\\\\\\/9lYGRB72jQ8XppmDRgpjjv+FOfJhoOriCdQC2\\\\\\\/fouULmXSbCX2IZ8nCUUsv8Q9otlPN7jPQOvb\\\\\\\/7NChF2uoi0b2yXxSoxVphaKt7B9LjKuRcUTjuz6cC0OXjwadBuYdIzzjsBgg26z4stVA\\\\\\\/7syHm8Irxqrc96Zwp+elowzOBbYDLnZq3iHMYR6J5d07A52sioiTu3V6mmrQ7eSeaoWyiWU2rZlKCd9n94QsLetGMl\\\\\\\/gzngT24apmxUU+LSUkHPuGih2f+j6SOnKT1TdH2sHh6Zndxidlpb4MFyHlsvEvWXif5QpeCUJMZqnCLAlGSesoR62by1NUE0lOwJB+DN5+RbsfHRAu7lDjxAj1GJNgbY\\\\\\\/5RSX7KEtvEIHRxNYZLdl1lCtp3EWNhKM0suAcstvMddG\\\\\\\/V0Ga\\\\\\\/KJZ3PbhXpNUScYuh7zEgxTbcP1c2pmW4Fp\\\\\\\/Rtmfcwy3DAD37TIMyhFfksGzGvpYQoZZKnhL78V7PjoYVNtDHQg7Srthc58t8v1q3qS+xvTtz0joyFqjNbk3vl+\\\\\\\/wdbcuo9RPdHhJuqIZJrL\\\\\\\/UocvgQKRjS8aS7M\\\\\\\/5Dt8QWrazJfgLh2fQKk9uAvcTVsVsEfpkYEe\\\\\\\/UDeZVo50wtlaE\\\\\\\/lkyTJeksvInrspXxk1JGyfobkxl5Ugj7aDMkF7AnEkRn0vah+sH58lFn+DrgtbTwYaJV2QM41EhlkY6B4xLaNyZc48kV\\\\\\\/TIBPDQCQNel6rCSKxyg8NA7LjS941GzyxZ9jdEmrP4aPo+cx2I4l6L30l63EbVsLp2aJt6FtPCcHDSvtS4bY55zeiMxPKKw4d69P6MihcTKsofu0O\\\\\\\/ODoBSGtlURx9rtOYjGHEj6iIqbqM5SwppnLPqGV\\\\\\\/D5ezbct5uFcnXL\\\\\\\/Decw0AlgY00Y70yoCwUXVJAQNooU1EdkLkKG9QIC0IvGS3GaqTz3TuueeQACqD38hvxzZQNv+JITWTPZ32+UVfWvMc6lRlTtg9YMVpcEFuWcLZBH61YlKVabSgbI+KlCtENiRaYnMY2rA9h3tid0k0AKLeYM8bDaptpw7n8jmSFJdIeuR7kIe1fMLv\\\\\\\/+ZyOM3kLtAitodIphjIFp4BlcVGoPlBvbjAXmLJJPO2tPeW2oIq2Bf77Tp8LYJCi8mBD2KTk0SE4KIFXe\\\\\\\/rsqJGckTUtREq40sb31cPsvxd6wfc4eEaMkvM5Tf2ZKu\\\\\\\/4FaASnOd5kEDAtlKAAi\\\\\\\/Ms9\\\\\\\/e5+vv0qprwRrJTvIYaKoZ055EE1wKWf\\\\\\\/XPzoWlgwf\\\\\\\/MS98ommtku62MRvHHcTpUSncmKmSyQjFR8UfADQZ8k39rN9WSiIwfM7RSQPn4mLZTKaMA00gAIaT1OkhHN1wWIRigTra1spbriHewLd7fCHvMcPki7FqMwaj\\\\\\\/IY5VYJEh5ALR971tjD\\\\\\\/U2KKYpKr0+eU7BGMw1U9ADfDbiZ3M7qMScT5n\\\\\\\/fzdoQldPrgdnhi6vY4WrBJPfjJVXCl\\\\\\\/f66ILOspkRfGSPPX2c60\\\\\\\/IyQd4IE\\\\\\\/akNnV9uz1+x4dNJJtO6EgOBCE5q0lsnsjhiehHCOfxV6EVtxfxc4ai8FiRIRqfKD5Ip0ww6\\\\\\\/zoEP2pYy+vcrmK4c178MJK3p+MQ+ZWnPXBK4y\\\\\\\/jAiquEa69SI+h60NGmrtc8En+SuJ\\\\\\\/\\\\\\\/vdYrjqKzzLTtfpAs5zdvx0AG3GocKmqGpg+A01z32UFYn3c5EjChEwSAQYrFx5Xn9geUywKUI251zKWuVGlsH5S+tlMMYOpbpdWoVPWfC7csdfHHD3HhWZqZWaRTXrIPUtFo\\\\\\\/yYxmBJmqMJsS7Sdkjl4xuBBtiHDBU2J05KFEgC49Xr+UZElIgWzURllC7OzWQx7d+Elwr9o1f2U3xdq4XLwuWX\\\\\\\/iK79UBB8qAeojU5YvMjY7+abljWJ07siuRZ9NKt44q7A5jpaBfvH5bcYjmVd0G5BQGjeW\\\\\\\/M3TyexPrggSLAVmBaepPg0hHjv\\\\\\\/JmQyJyvPSOZT1aVcluyeSWIiEOuX9ImmitTJc3o8wOtCLyPWA8ADcV1WONETiCCxfhfjIA21KVFDlerr5nn1JlncqcO96u9c8bfpJzv1Ys8J7Q7XoxELbmjCWDuFGU5BZT3bMaM28tC5cUmSk5nW8WENk16Klu8UWm7hYJRqge1ZvI4p0aRaHnbXODMrV39p2nfhzgxj\\\\\\\/qozgeCb\\\\\\\/++gVe55egyjOEXB1Q3YSn2EtWwhvUGWm5iPKQzutpZWNa5Owb\\\\\\\/WcL+hKh7WJKg404ipgVClGKdYh9wkB+aGpEKQBjnvgg5Ty5+9QW0d2yBaSoJvctFdrhrIaktmHeW3SJTwrM6bIJsX6R5CNlY0pUlsjkUytCltp02I18UTWNaaacrAitKOBb+f6nU1\\\\\\\/rPvF4fv\\\\\\\/VlHgxKBWojp\\\\\\\/Q361J3gz6C+bXE98dtJaNCE3sRdB7HwXM8KRSnJXEteaLCMOdoM\\\\\\\/sr1yOMyjxzXOVQWgg3eYIx0VVnnivRPgytxF9VuO5mSeG5e+p0Bqe4m5XhXryCJUh\\\\\\\/XXvEH9Fty5BL00iBtugOkszXX\\\\\\\/7jQNdTfFvVL\\\\\\\/ODElAvI8LQscbl0Z8PuWRNGzJ4FXjMbwL0Ih8cmE9vfdvaOOhQwuNxV7YXMIhJxWNdEMFxvOdlzBz8pNAJB8nhaCw\\\\\\\/V1aAU3gWPCTwrAMLjz49nu2870Jle0NYlFomHcdU6pcqoMxSdvGTX0HVzBSlOCqyA9r9ZxgT4ptQoMXa0sRDsHIao3die+Uy5dfPf5yNFvAGkFR7ycfES1547nGGwXJ5r2rvki7WoEZRGFuUFpiJqIiZmwUL53xd7c97tPWhwlnMHgFBY5kJZd1z6J7DWHY5Ze8ETT2brw9JAHCwZ+ZOI1LCVONGDNLZ+OyKfFsNe76G2WYRzcyLIIg7h9IfkbI\\\\\\\/BO+3N2xFlYBtTeZI7\\\\\\\/oWJPWRnSlJ7SytwnXfl2QXH4ecsf2mFgAsy9U0MPI6IZXiyRcR3Naw\\\\\\\/X21HFtu2ecsyAQpTNBM2hc2EhyuxiFmYH7t1ti4SR5okdMa18OSEdTUS0qfsfuozcaIy5cZs7EdFsPD65xcI4EfSNetmsfRwvLKQDSMQm5HO4MIMsM1WHp67ubTEkFlCs0seb+hgRn5pKPVcqLCs75OZ+s0QBx\\\\\\\/z3kmtfteby9rrJ8FvCLOI1Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022n6FCOFKERVZfFzlZ4VKSOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b3c27ea455b42f008264134d9dce201599550c28730ef80a2e7492f2606c9386\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022htWxorCaJkoxWCJ4AOYH0ggPetbVEWCnYvmswgZYhTOS2hmCqdt1uC2DcbDbf+G5W8iodEYcSREN69266KpPPIw6fLuWWsVutJ6pKxOxnRY6uHB1k6y1NNsaqGGEChaAAvlWOfYJCz0B9PYpHbhEgk2fjdhA1CINPrulFQLPVXfoTpqG+eqqRg\\\\\\\/H18vkAEFzI1VM21S+y2pEQcGLNvf8RXIIiWz+KUkR5pROlB45avb9ZPl3fUfbhClXyOOymIxIEEmJOh3QHzs96RhmbVJdxDgqxn0fEhStrJuGqz4\\\\\\\/\\\\\\\/MTA39DIadHp2JaupWgrAc2UMGngr\\\\\\\/kU1lgYoHNWBnTE0bMUfNnU5o5auG84QnrUWCOlkgHT6UyrudPlNnonRBriS+aJuOFs1kTMyv8jR7c3aykUDILjG9q1b7q5AnDR1GygxYUTbRhdjRudmJTst4me+GY7A3BA3gSkXCBcnfQClMD8PJrhfs+KsSZ7S3bBHlp9oLkdioRgA1kV0W+qbm1eeWQPWg4MTLYYbpPeFOf12DyoH5JZSKlF\\\\\\\/eVBm2++hInTJhBNu3PeRx\\\\\\\/VJRq3vec5JxIeoG9kQ7rKLJlQcZuNx3d2CT+q\\\\\\\/sVaZ73pPN9bwBASTbtldMqHrAtlkzDsHNBjZdU9rFpTzhFPzYw+00ncUaTwMDcQhBtTbvTJLE6m0IfrTvp7WkmZHs+j3uidJRNOCerJnuCF\\\\\\\/FMqGUnUO5dLjQgHc1nHQYZxr5YuJFYiDfTIks9h0KzcMay4CVMHsKV2tsGUKVaXtVUFydiaRwthnX9YhjJxdI7MJw6t2EW3CMV\\\\\\\/CnUaxfLy5pNLqVvQ4tuiXzn4JFxOd97wc8KR23FOlRjPb4b6KVB0TG\\\\\\\/r8zpaA8eYSHE\\\\\\\/L6TY8yqHI8KBlMo3OFTJEhUXjD5r6\\\\\\\/EOZXC5T6z8Gyr+U1RwDbZdYaBkvC6ZLx6w7Jsf\\\\\\\/+uabXd+OU+lbLbGs9XmWykX06y5\\\\\\\/t0Z0ymfrn4rMqC5K3sslazOZBf2bUBjD1zw7mUCa6fZWm1Gnudtq\\\\\\\/OBCF\\\\\\\/EYssI\\\\\\\/Mi+faMVoEd3zQ+ZMtbb6dWZ8f8VWYerf1+RKbwKXmC8X4cP6cbyblNUQ04v+iISAlexRE3fH5k1pCdZZ2bMPNlMJoRWHfqmnToYSv4es2XSVDsXBQc7q8tJaEwCWnt\\\\\\\/nb17Si3mXYt\\\\\\\/XfPIwqGEvF4vu3AW695p5qBvnfBETTALVqGzBp97tcGUtQ\\\\\\\/qojj+Pl1tnCrIitm3z+xrs+hSAuhfGmHWfSCX1j6PxlMj2fWYAGqTXIZ5pBPNG9hI6Wv7yVQnTQkx\\\\\\\/Q3hwxOvSdERiNuSEhpwfAM3xPTAgIN\\\\\\\/0yglMQ3n3uDr\\\\\\\/mJIK6cNcfRritMFMkjy0toJOgVQ0EP64YEcac7X8VuRiJ\\\\\\\/RVIDfYItjyNWfISyCdmwIX4WQRSm5WUpbdzNiZRq7XVcUqqUgbFsS5AnBc6+1IbGqngaFEBTAsCDLYpC63Wk83+zDzFNa5N8MvAliR5awyzfOsVs4T6IDEHDVSDeQRtyItHMurqrwlBK9rnMltZMRvyxCfEXk4y2k8SAjyHayQ0JZ4hwcB6DsEHMxgt1zvKy9OxQJnvwj7dVONNT7CeO5STHGnjl+r9mSExL1sH28OhjNhv6lTTHwVgXX4DBIwNJ81yOJTG84sRiXA+FY69Myd3tKj96uhtfhevysseSAuMLQw23yT4Csa06OKBxVwD1xt4lGWsxQek6NRCSsP4ND40YXaM0oiA+z21otByI68rti+VPeg5657ULBHJNGWVpM8uBUWG96\\\\\\\/ylrJ2QYWdovaVsYTnmeSJ66VbWO8gHWRjw9uYC9XIJzbu0sU4wY32NWk0GgOZ\\\\\\\/vaNVBhzLG5Q5vomxf8ljBYXbT2KklPTXZjj8cIH8Ttt9PE9HaQgPWfkUR6znjZDsP\\\\\\\/Z2kFGlOoppM8YP+ptYp8iMDzobj09Eil9AUoDFOYMdVHA4Ll+iMeDGOSkknxlnzelsLTWGLp5lhYED8jYMC68xyCg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xyhvUiQecTGXLSvN7KzMDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022acd6031f9be785cc94dd232940ccc67f9377f284650462b3ed3a8491ab01ee5c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459864 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/913" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--lMb5CuFkDS3h8ZULEyKqjF8gnH0MEUsgtRalQzRmfE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/913" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UAcQXlKT7Klt7TXTbwwkPbaHIlekj-NJTq-Bj3k2R1g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/901" title="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/925" title="Efficient Gene Knock-out and Knock-in with Transgenic Cas9 in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459864" id="node459864" data-pisa="ggg;4/5/913" data-pisa-master="ggg;g3.113.010165" data-apath="/ggg/4/5/913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of <em>Panicum virgatum</em> (L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guifen Li</span>, <span class="highwire-citation-author" data-delta="1">Desalegn D. Serba</span>, <span class="highwire-citation-author" data-delta="2">Malay C. Saha</span>, <span class="highwire-citation-author" data-delta="3">Joseph H. Bouton</span>, <span class="highwire-citation-author" data-delta="4">Christina L. Lanzatella</span> and <span class="highwire-citation-author" data-delta="5">Christian M. Tobias</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">913-923;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.010165 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459864" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guifen Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guifen%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuifen%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Desalegn D. Serba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Desalegn%2BD.%2BSerba%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serba%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADesalegn%2BD.%2BSerba%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Malay C. Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malay%2BC.%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalay%2BC.%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph H. Bouton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BH.%2BBouton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouton%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BH.%2BBouton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christina L. Lanzatella </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BL.%2BLanzatella%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lanzatella%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristina%2BL.%2BLanzatella%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Christian M. Tobias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BM.%2BTobias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tobias%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BM.%2BTobias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.tobias@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">christian.tobias@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/5/913" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459864" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/913.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459864" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/913.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459864" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/5/913.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459864" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Switchgrass (<em>Panicum virgatum</em> L.), a warm season, C4, perennial grass, is one of the predominant grass species of the North American tall grass prairies. It is viewed as a high-potential bioenergy feedstock species because it can produce large amounts of lignocellulosic material with relatively few inputs. The objectives of this project were to develop an advanced switchgrass population and use it for the construction of genetic linkage maps and trait characterization. A three-generation, four-founder population was created and a total of 182 progeny of this advanced population were genotyped, including a mixture of self-pollinated and hybrid individuals. The female map integrated both subpopulations and covered 1629 cM of the switchgrass genome, with an average map length of 91 cM per linkage group. The male map of the hybrid progeny covered 1462 cM, with an average map length of 81 cM per linkage group. Average marker density of the female and male maps was 3.9 and 3.5 cM per marker interval, respectively. Based on the parental maps, the genome length of switchgrass was estimated to be 1776 cM and 1596 cM for the female map and male map, respectively. The proportion of the genome within 5 cM of a mapped locus was estimated to be 92% and 93% for the female map and male map, respectively. Thus, the linkage maps have covered most of the switchgrass genome. The assessment of marker transmission ratio distortion found that 26% of the genotyped markers were distorted from either 1:1 or 3:1 ratios expected for segregation of single dose markers in one or both parents, respectively. Several regions affected by transmission ratio distortion were found, with linkage groups <em>Ib-m</em> and <em>VIIIa-f</em> most affected.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/biomass" class="hw-term hw-article-keyword hw-article-keyword-biomass" rel="nofollow">biomass</a></li><li class="kwd"><a href="/keyword/feedstock" class="hw-term hw-article-keyword hw-article-keyword-feedstock" rel="nofollow">feedstock</a></li><li class="kwd"><a href="/keyword/biofuel" class="hw-term hw-article-keyword hw-article-keyword-biofuel" rel="nofollow">biofuel</a></li><li class="kwd"><a href="/keyword/allopolyploid" class="hw-term hw-article-keyword hw-article-keyword-allopolyploid" rel="nofollow">allopolyploid</a></li></ul><p id="p-4">Switchgrass (<em>Panicum virgatum</em> L.) is a morphologically diverse, warm season, C4, perennial grass and a predominant species of the tall grass prairies of North America (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bouton 2008</a>). It has long been used for forage production and for soil conservation plantings due to its large fibrous root system, and it is now being developed as a bioenergy crop for second-generation biofuel production (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sanderson <em>et al.</em> 2006</a>). Its suitability as a biomass feedstock is due to its high biomass yield potential, relatively simple growth requirements, and wide adaptation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barney <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elbersen <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Evanylo <em>et al.</em> 2005</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fike <em>et al.</em> 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Moser and Vogel, 1995</a>). Efforts to improve yield, feedstock quality, and stress tolerance have recently been initiated and some of these efforts have focused on lowland ecotypes with high yield potential. These are tetraploid (2n = 4x = 36), whereas upland populations are both tetraploids and octoploids (2n = 8x = 72). Upland and lowland ecotypes can be distinguished genetically, based on morphology, and also by ecological preference (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casler 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casler <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Das <em>et al.</em> 2004</a>). Lowland ecotypes are adapted to heavier soils, are tolerant of flooding, and are generally found in the warmer and wetter regions of the southern United States, whereas upland ecotypes are adapted to lighter soils and dry and cold zones of the middle and northern latitudes of the United States (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hopkins <em>et al.</em> 1996</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hultquist <em>et al.</em> 1996</a>). Upland and lowland ecotypes occupy overlapping distributions, with significant gene flow between groups (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhang <em>et al.</em> 2011</a>).</p><p id="p-5">To hasten efforts at efficient breeding, marker-based methods need to be adopted. Hence, the development of genomic resources and analysis of both structured and unstructured populations for genetic dissection of quantitative traits are priority research areas for switchgrass. One main goal of genetic mapping is to identify simply inherited markers that are closely linked to quantitative trait loci (QTL) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jannink and Walsh 2002</a>). A high-density genetic map of switchgrass is essential for effective application of marker-assisted selection (MAS) in breeding to increase the efficiency of selection and for characterization of specific regions in the genome that hold a special interest for map-based cloning. Mapping of molecular markers distributed throughout the genome of a plant species or at least around the gene of interest is a prelude for the application of molecular approaches in breeding switchgrass for improved yield and tolerance to various biological and environmental stresses.</p><p id="p-6">The segregation patterns of markers in controlled biparental crosses of switchgrass are predicted to behave as a full-sib family derived from two heterozygous parents. The segregation may have different forms as the number of alleles or QTL segregating at a given locus may reach up to four and vary across loci in their informativeness (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wu <em>et al.</em> 2002</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">2007</a>). This introduces complexity to mapping and, in addition, linkage phase between QTL, and nearby markers may be unknown, which can introduce serious biases in estimations of QTL size and affect. A practical approach to this is to treat single-dose alleles segregating in each parent as dominant markers for the purposes of mapping in a two-way pseudo-testcross design (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grattapaglia and Sederoff 1994</a>). Single-dose alleles in common to both parents, although less informative, can also be mapped with this approach. This is advantageous when there are insufficient numbers of fully informative markers (<em>ab</em> × <em>cd</em>) to integrate the parental maps. Transmission ratio distortion (TRD), deviation from the expected Mendelian ratio of segregating alleles at a locus (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sandler <em>et al.</em> 1959</a>), may also affect estimates of recombination fraction. This phenomenon has been commonly encountered in mapping populations (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garcia-Dorado and Gallego 1992</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lorieux <em>et al.</em> 1995a</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">b</a>) and is increasingly recognized as a potentially powerful evolutionary force that also affects the construction of genetic linkage maps (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lu <em>et al.</em> 2002</a>).</p><p id="p-7">Previous linkage mapping in switchgrass has been performed on a full-sib population of two lowland individuals (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> 2010</a>), on a lowland × upland full-sib population (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Missaoui <em>et al.</em> 2005</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Serba <em>et al.</em> 2013</a>), and on the selfed progeny of a heterozygous individual (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Liu <em>et al.</em> 2012</a>). Although self incompatibility systems are present in switchgrass (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Martinez-Reyna and Vogel 2002</a>), these incompatibility systems are not 100% effective. Up to 61% of self-fertilized individuals were reported in one controlled cross of a heterozygous northern lowland genotype, which allowed linkage map construction (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu and Wu 2012</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Liu <em>et al.</em> 2012</a>). These studies have consistently demonstrated the nine base chromosomes of switchgrass are inherited in a disomic manner and have clearly demonstrated synteny with related grass species; however, there has not yet been definitive assignment of individual chromosomes to the two subgenomes. Here we describe joint linkage analysis in a cross that produced both self-pollinated and hybrid individuals from a population of four founders derived from the two linkage populations above with two parents and 182 progeny. The structure of this population enables marker phase to be unambiguously determined, particularly for those markers with alleles shared in common by both parents. The objectives of this project were to construct a genetic linkage map from this population with markers distributed throughout the switchgrass genome and to jointly map both self-pollinated and hybrid progeny as a prelude to analysis of loci underlying TRD and the analysis of QTL present in the parents.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping population development</h3><p id="p-8">This mapping population, designated AL-NF, was developed from four founders in three generations and, as such, is unique from previously reported mapping populations in switchgrass. First, selected genotypes from Kanlow (K5) and Alamo (A4) were crossed in Albany, California, to generate K5 (female) × A4 (male) (ALB) population (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> 2010</a>). The F1 pseudo-testcross population was field evaluated in Athens, Georgia, and four distinct genotypes were selected. A second population (NF-UGA) was developed by crossing a lowland selection, AP13, (female) to an upland selection, VS16, (male), at the University of Georgia (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Missaoui <em>et al.</em> 2005</a>). The population was further expanded at the Noble Foundation and 251 F1 progenies have been field evaluated at three locations: Ardmore (34° 11′ 32′′ N and 97° 5′ 21′′ W) and Burneyville (33° 53′ 20′′ N and 97° 16′ 36′′ W), Oklahoma, and Watkinsville (33° 52′ 19′′ N and 83° 24′ 20′′ W), Georgia. Four distinct genotypes were selected. Genotypes selected from both populations were grown in greenhouses at the Noble Foundation and reciprocal crosses were made in the spring of 2008. A panicle from each of the female and male parents of the selected genotypes was put together in a paper bag. To enhance pollen shedding, the bags were manually tapped every morning. The tillers were tied to a supporting bamboo stick until seed maturity. The seeds were harvested from both the parents to obtain cross and reciprocal seed. The cross with the most seed set, <em>e.g.</em>, PV281 (female) (selection from ALB) × NFGA472 (male) (selection from NF-UGA), was selected to develop a three-generation and four-founder population (AL-NF). The crossing scheme for population development is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/913/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Switchgrass AL-NF mapping population development scheme using four founders in three generations. Other crosses were performed but the PV281 &#xD7; NF472 cross resulted in sufficient seeds for further evaluation. AL-NF-self population represents progeny derived from self-pollination of PV281 that were initially identified by limited genotyping. AL-NF population represents hybrid individuals. The number (n) of individuals in each population is indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1341015863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Switchgrass AL-NF mapping population development scheme using four founders in three generations. Other crosses were performed but the PV281 &#xD7; NF472 cross resulted in sufficient seeds for further evaluation. AL-NF-self population represents progeny derived from self-pollination of PV281 that were initially identified by limited genotyping. AL-NF population represents hybrid individuals. The number (&lt;em&gt;n&lt;/em&gt;) of individuals in each population is indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/913/F1.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/5/913/F1.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/913/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/913/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464359" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Switchgrass AL-NF mapping population development scheme using four founders in three generations. Other crosses were performed but the PV281 × NF472 cross resulted in sufficient seeds for further evaluation. AL-NF-self population represents progeny derived from self-pollination of PV281 that were initially identified by limited genotyping. AL-NF population represents hybrid individuals. The number (<em>n</em>) of individuals in each population is indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Seeds collected from the cross were scarified in diluted sulphuric acid and chilled at 4° for 7 d. The seeds were germinated at 24° with a 16-hr photoperiod. Seedlings were grown in a greenhouse at 32° day and 21° night temperatures. The young plants were propagated by splitting the tillers and then set out in the field following an R-256 (251 progeny, two sets of parents and one Alamo check) honeycomb design. The population has been field evaluated for 3 yr at two locations in Oklahoma: Red River Farm (Burneyville) and Ardmore; 188 plants were randomly selected and genotyped for construction of genetic linkage maps.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and quantification</h3><p id="p-11">Fresh tissue from young leaves of greenhouse-grown plants was collected and frozen in liquid nitrogen. The frozen samples were ground in 2-ml tubes with two zinc-plated ultra smooth ball bearings (Daisy Outdoor Products, Rogers, AZ) using a Tissuelyzer (Qiagen Inc., Valencia, CA). The genomic DNA was extracted using DNeasy Plant Mini Kit (Qiagen Inc.) following the manufacturer’s recommendation.</p></div><div id="sec-4" class="subsection"><h3>Marker screening</h3><p id="p-12">Several sequence-based markers have been developed for switchgrass. In this study, 342 pairs of eSSR (expressed sequence tag–simple sequence repeat) primers developed from publicly available EST resources, 512 pairs of gSSR (genomic SSR) primers developed from (GA/CT)n enriched genomic libraries, and 144 pairs of STS primers developed from switchgrass cellulase and transferase gene sequences were screened for polymorphism. Primers were synthesized by Sigma-Aldrich Corporation (Woodlands, TX) with 18 nucleotides of M13 (5′-TGTAAAACGACGGCCAGT-3′) universal primer sequence added onto the 5′ end of the forward primer (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schuelke 2000</a>). The M13 (-21) universal primer sequence labeled with PET, VIC, 6-FAM, or NED fluorescent dyes was synthesized by Sigma-Aldrich (St. Louis, MO). All of the 998 primer pairs were prescreened using a panel of two parents and six randomly selected F1 individual plants. The primers that were polymorphic in two parents and/or segregate in the progeny were used for genotyping the whole population.</p></div><div id="sec-5" class="subsection"><h3>PCR protocol</h3><p id="p-13">Approximately 20 ng template genomic DNA was amplified in a total volume of 10 μl. PCR reactions consisted of 1× PCR buffer, 150 µM dNTPs, 0.25 µM forward primer and 1 µM reverse primer, 1 µM M13 florescent dye, and 0.5 U GoTaq DNA polymerase (Promega, Madison, WI) and brought to volume by adding ddH<sub>2</sub>O. The PCR were performed on a GeneAmp PCR System 9700 thermocycler (Life Technologies, Carlsbad, CA) following a touchdown PCR program. After an initial 3-min denaturation at 95°, the thermal profile consisted of three phases. The first 6 cycles consisted of a 45-sec denaturation step at 94° and 5-min annealing/extension at 68°, which was reduced by 2° per cycle. The next 8 cycles consisted of a 45-sec denaturation at 94°, followed by 2-min annealing step at 58° that was reduced by 1° per cycle, and extension step at 72° for 1 min. The final 25 cycles consisted of a 45-sec denaturation step at 94°, a 2-min annealing step at 50°, and a 1-min extension step at 72°. A final 7-min extension step at 72° was then performed. This method was observed to enrich the correct product over any nonspecific products amplified by the primers.</p><p id="p-14">The amplification products were diluted in formamide and analyzed by capillary electrophoresis with labeled size standards using an ABI3730 and GeneMapper v.3.7 software (Life Technologies). All markers were scored as dominant loci. Markers with more than 15% missing data were removed from analysis.</p></div><div id="sec-6" class="subsection"><h3>Identification of hybrid and self-fertilized populations</h3><p id="p-15">Switchgrass is highly cross-pollinated and known to be self-incompatible. However, varying degrees of self-fertilization have been observed (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Liu and Wu 2012</a>). To identify the selfed plants among the hybrids, the genomic DNA of the four founders, two parents, and 188 progeny were genotyped using 12 polymorphic SSR markers segregating in the male parent. The amplified product was visualized and scored as indicated above. The amplicons of each individual were compared with the two parents. Inbred individuals were identified as those that yielded female-specific but lacked male-specific amplicons.</p></div><div id="sec-7" class="subsection"><h3>Linkage analysis</h3><p id="p-16">The linkage analysis was conducted using polymorphic amplicons referred to henceforth as “markers” that were single-dose (SD) in either parent and expected to segregate 1:1 or were shared by both parents (SD × SD) and expected to segregate 3:1. The SD × SD markers were placed on the map after the marker order created using only SD markers had been fixed. JoinMap 4.0 software (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">VanOoijen 2006</a>) was used following our breeding full-sib family (CP) model or an F2 model for self-fertilized individuals using the regression mapping algorithm (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stam 1993</a>). For the female map, mean recombination fractions and combined likelihoods of odds (LOD) values of markers from self and hybrid groups were used to integrate the separate linkage groups (LG) and produce the final maps. The SD markers were grouped into linkage groups at an independence test LOD threshold of 7.0. The calculations of the linkage map were performed using all pair-wise recombination frequency estimate of less than 0.40 and a LOD score more than 1.0 (ripple value = 1; goodness-of-fit jump threshold = 5; and a triple threshold = 5). The male and female hybrid maps were analyzed following the two-way pseudo-testcross strategy (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Grattapaglia and Sederoff 1994</a>). Ungrouped markers or misgrouped markers were assigned based on their strongest cross link (SCL). If the SCL was more than 7.0, then ungrouped or suspect markers were reassigned for both female and male segregating markers.</p><p id="p-17">Genotyping data from six individuals were excluded from map construction due to either large amounts of missing data or inconsistent genotyping. Map construction for the NF472 map was conducted using marker data from 108 hybrid genotypes, whereas genotypic data from the entire population of 182 (74 self-fertilized and 108 hybrid) individuals were used for the PV281 map. The combined LOD values and mean recombination fractions were used to integrate the two maps using the regression mapping algorithm. The Kosambi mapping function (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kosambi 1944</a>) was used to convert recombination units into genetic distances.</p><p id="p-18">Switchgrass genome length was estimated by the formulae ∑<em>L</em><sub>i</sub>[(<em>k</em><sub>i</sub>+1)/(<em>k</em><sub>i</sub>−1)] as proposed by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chakravarti <em>et al.</em> (1991)</a> and <em>L</em> + (2<em>tL</em>)/<em>n</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fishman <em>et al.</em> 2001</a>), where <em>L</em> is the total length of the genetic map (cM), <em>n = k−t</em> is the number of marker intervals, <em>k</em> is the number of marker loci, <em>k</em><sub>i</sub> is the number of marker loci on the <em>i</em>th linkage group, where <em>i =</em> 1, 2,..., <em>t</em>, and <em>t</em> is the number of linkage groups. With the assumption of random distribution of marker loci in the genome, the proportion of the genome within <em>d</em> cM of a marker locus was estimated to be 1−<em>e</em><sup>-2</sup><em><sup>dk</sup></em><sup>/</sup><em><sup>G</sup></em>, where <em>k</em> is the number of mapped loci and <em>G</em> is the estimated genome length.</p><p id="p-19">For determining homologies between linkage groups and deriving consistent nomenclature of subgenomes with previous linkage analysis (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> 2010</a>), the populations were genotyped with subsets of previously mapped markers. Resolving homeologous relationships and alignment of parental LG was performed by the following: (1) alignment of multiple markers derived from the same primer set present on different LG; (2) assignment of LG based on identity with the founder’s linkage map order determined by <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> (2010)</a>; and (3) placement of SD × SD markers. For those few LG with no markers in common, a sequence similarity-based strategy was used similar to one used with oat (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gutierrez-Gonzalez and Garvin 2011</a>). Thus EST-STS markers were used to identify collinear stretches of foxtail millet (<em>Setaria italica</em>) genome as the criteria for homeologue formation. EST-STS marker sequences were compared using blastn to the foxtail millet genome (<a href="http://www.phytozome.net">www.phytozome.net</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bennetzen <em>et al.</em> 2012</a>) and 1.0×10<sup>−5</sup> E-value threshold was used to determine synteny. When switchgrass marker sequences from ungrouped LG aligned to the same <em>S. italica</em> scaffold, irrespective of the position, these LG were considered homeologous/homologous. Following construction of the LG, map graphics were constructed using MapChart 2.2 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Voorrips 2002</a>).</p></div><div id="sec-8" class="subsection"><h3>Analysis of transmission ratio distortion</h3><p id="p-20">All markers were scored as dominant loci and those with TRD were identified on the basis of significant deviations from 1:1, 3:1, or 5:1 ratios expected based on allele dosage and disomic or tetrasomic inheritance using the χ<sup>2</sup> test (<em>P</em> &lt; 0.05). Markers displaying significant TRD were excluded from initial map construction and placed at their most likely position after the initial mapping step. To assess the extent of multilocus interactions between unlinked loci associated with single-locus segregation ratio distortion, all mapped TRD markers were tested for independence against all markers in other LG in both male and female maps using the two-locus genotypes in the two-by-two contingency χ<sup>2</sup> test. Since comparisons were made among unlinked SD alleles, the significance was evaluated with a correction for 31,871 comparisons of 28 LG with at least one TRD marker with 35 other LG (<em>P</em> &lt; 1.57 × 10<sup>−6</sup>). The distribution of the TRD markers was also assessed in the female and male maps, and when three or more TRD markers were clustered together we considered them a candidate TRD locus (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2011</a>).</p></div><div id="sec-9" class="subsection"><h3>Comparative mapping</h3><p id="p-21">Individual maps across studies were compared based on their correlation coefficients for marker interval distance and for marker order based on their longest common subsequence (LCS) using the qualV package (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jachner <em>et al.</em> 2007</a>). Results were expressed as markers present in the LCS as a fraction of the number of markers in common between groups. EST-SSR markers were compared to v. 2.1 of the <em>Setaria italica</em> genome (<a href="http://www.phytozome.net">www.phytozome.net</a>) using blast.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Population development</h3><p id="p-22">Four F1 genotypes from each of ALB and NF-UGA populations were selected with distinct morphologies and a total of 10 successful crosses and reciprocal crosses were made between them. The highest number of seeds (850) was obtained from the cross PV281 × NFGA472 and the reciprocal cross NFGA472 × PV281 provided the second highest seed (746). Poor seed set was observed in the cross between the parents PV204 and NFGA472, where 25 seeds were obtained when NFGA472 served as the pollen donor and 33 were obtained when PV204 served as the pollen donor. The cross with the highest seed set (PV281 × NFGA472) was developed as the mapping population ALB × NF. In this cross, PV281 was the female parent and NFGA472 was the male parent.</p></div><div id="sec-12" class="subsection"><h3>Markers and their inheritance</h3><p id="p-23">Among the 998 primer pairs, 447 were genotyped on the entire population. These consisted of 158 (46.2%) eSSR, 272 (53.1%) gSSR, and 17 (11.8%) STS primer pairs that produced a total of 1203 amplified size polymorphisms (referred to henceforth as markers) between the two parents (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The average numbers of polymorphic markers per primer set were 2.97, 2.03, and 1.97, respectively, for the gSSR, eSSR, and STS. Of the total markers, 484 segregated in the female (PV281) and 460 segregated in the male (NF472) parent. The remaining 259 markers were monomorphic in the parents but segregated in the progeny population (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This result indicated that there was high molecular marker polymorphism between the two parents.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464273/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464273/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Molecular marker categories used, their amplification, and their polymorphism in the AL-NF mapping population</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464385/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464385/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Segregation of molecular markers genotyped in AL-NF and AL-NF-self populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">In the hybrid population, single-dose markers in either parent are expected to have 1:1 segregation in the hybrid progeny, whereas double-dose markers in only one parent will be expected to segregate 3:1 under strict disomic inheritance or 5:1 under strict tetrasomic inheritance. The χ<sup>2</sup> tests of markers segregating in the hybrid population indicated that 390 of the polymorphic markers present in the female parent and 274 markers in the male parent fit a segregation ratio of 1:1 in hybrid or 3:1 in self-pollinated progeny. In the hybrid population, there were 8 markers that fit a 3:1 ratio and 7 markers that fit a 5:1 ratio among those that segregated in the female parent; however, among those that segregated in the male parent there were 26 markers that fit a 3:1 ratio and 6 markers that fit a 5:1 ratio. The remaining 232 markers (79 in the female and 153 in the male parent) did not fit any expected segregation ratio. Among 259 markers monomorphic in the parents that segregated in the progeny, 138 fit a 3:1 ratio expected of a single dose marker heterozygous in both parents under either disomic or tetrasomic inheritance (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The remaining 90 markers did not fit any expected segregation ratio. In total, 26% of the markers in the hybrids were significantly distorted from expected segregation patterns.</p><p id="p-26">In the AL-NF-self subpopulation, markers affected by TRD were not necessarily identical to those affected in the hybrid subpopulations. Overall, 39% of the mapped markers in this population did not fit an expected 3:1 ratio of these 200 markers; only 42 were also affected by TRD in the hybrid subpopulation (data not shown).</p></div><div id="sec-13" class="subsection"><h3>Construction of parental linkage maps</h3><p id="p-27">The molecular markers used to produce the parental linkage maps and the genome coverage are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The linkage analysis of 527 polymorphic markers for the female parent map produced from both AL-NF and AL-NF-self subpopulations consisted of 375 SD markers and 111 SD × SD markers that included data from 255 different primer pairs. These were formed into 18 linkage groups. Among the 445 mapped markers in the female map, 318, 116, and 10 were gSSR, eSSR, and STS markers, respectively. A total of 82 SD markers remained ungrouped. The female map spanned 1629 cM. Average length of each LG was 91 cM. An average of 25 polymorphic markers per LG with an average density of 3.8 cM per marker was mapped. The range of polymorphic markers mapped was from 13 to 43 per LG. The longest linkage group was <em>Vb-f</em>, with a total length of 131 cM, whereas the shortest was <em>IVa-f</em>, which had a total length of 65.6 cM (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464303/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464303/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of parental genetic linkage maps of the AL-NF-self and hybrid populations</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464335/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464335/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Number of markers, linkage group length, and marker density comparisons between female (PV281) and male (NF472) parental linkage groups</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Consensus linkage map of PV281 and NF472. PV281 linkage groups were constructed by integrating genotypic data from both self-fertilized individuals (n = 74) and PV281 &#xD7; NF472 hybrid individuals (n = 108) (see Materials and Methods). The Roman numeral designation of each homology group (I&#x2013;IX) follows the foxtail millet chromosomes. NF472 linkage groups were constructed based only on hybrid individuals as it served as the male donor in the cross. Individual linkage groups are indicated by vertical bars. Numbers to the left of the bar indicate position in cM, whereas marker names are indicated to the right of the bar. Markers ending in &#x201C;ds&#x201D; are double simplex markers present in both parents. Markers used to align the subgenomes were amplicons of different fragment length detected with the same primer set. These markers are set-off by similar color labeling. The letters (f) and (m) at the end of the linkage group name denote female (PV281) and male (NF472) parental maps, respectively. Severely distorted markers are noted with asterisks: *P = 0.01, **P = 0.001, and ***P = 0.0001. Shaded regions of the individual linkage groups indicate the presence of three or more consecutive markers with TRD. Markers with significant interactions on different linkage groups are indicated by a hash symbol (#)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1341015863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Consensus linkage map of PV281 and NF472. PV281 linkage groups were constructed by integrating genotypic data from both self-fertilized individuals (&lt;em&gt;n&lt;/em&gt; = 74) and PV281 &#xD7; NF472 hybrid individuals (&lt;em&gt;n&lt;/em&gt; = 108) (see &lt;em&gt;Materials and Methods&lt;/em&gt;). The Roman numeral designation of each homology group (I&#x2013;IX) follows the foxtail millet chromosomes. NF472 linkage groups were constructed based only on hybrid individuals as it served as the male donor in the cross. Individual linkage groups are indicated by vertical bars. Numbers to the left of the bar indicate position in cM, whereas marker names are indicated to the right of the bar. Markers ending in &#x201C;ds&#x201D; are double simplex markers present in both parents. Markers used to align the subgenomes were amplicons of different fragment length detected with the same primer set. These markers are set-off by similar color labeling. The letters (&lt;em&gt;f&lt;/em&gt;) and (&lt;em&gt;m&lt;/em&gt;) at the end of the linkage group name denote female (PV281) and male (NF472) parental maps, respectively. Severely distorted markers are noted with asterisks: *&lt;em&gt;P&lt;/em&gt; = 0.01, **&lt;em&gt;P&lt;/em&gt; = 0.001, and ***&lt;em&gt;P&lt;/em&gt; = 0.0001. Shaded regions of the individual linkage groups indicate the presence of three or more consecutive markers with TRD. Markers with significant interactions on different linkage groups are indicated by a hash symbol (#).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-2.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-2.medium.gif" width="400" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464370" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Consensus linkage map of PV281 and NF472. PV281 linkage groups were constructed by integrating genotypic data from both self-fertilized individuals (n = 74) and PV281 &#xD7; NF472 hybrid individuals (n = 108) (see Materials and Methods). The Roman numeral designation of each homology group (I&#x2013;IX) follows the foxtail millet chromosomes. NF472 linkage groups were constructed based only on hybrid individuals as it served as the male donor in the cross. Individual linkage groups are indicated by vertical bars. Numbers to the left of the bar indicate position in cM, whereas marker names are indicated to the right of the bar. Markers ending in &#x201C;ds&#x201D; are double simplex markers present in both parents. Markers used to align the subgenomes were amplicons of different fragment length detected with the same primer set. These markers are set-off by similar color labeling. The letters (f) and (m) at the end of the linkage group name denote female (PV281) and male (NF472) parental maps, respectively. Severely distorted markers are noted with asterisks: *P = 0.01, **P = 0.001, and ***P = 0.0001. Shaded regions of the individual linkage groups indicate the presence of three or more consecutive markers with TRD. Markers with significant interactions on different linkage groups are indicated by a hash symbol (#)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1341015863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Consensus linkage map of PV281 and NF472. PV281 linkage groups were constructed by integrating genotypic data from both self-fertilized individuals (&lt;em&gt;n&lt;/em&gt; = 74) and PV281 &#xD7; NF472 hybrid individuals (&lt;em&gt;n&lt;/em&gt; = 108) (see &lt;em&gt;Materials and Methods&lt;/em&gt;). The Roman numeral designation of each homology group (I&#x2013;IX) follows the foxtail millet chromosomes. NF472 linkage groups were constructed based only on hybrid individuals as it served as the male donor in the cross. Individual linkage groups are indicated by vertical bars. Numbers to the left of the bar indicate position in cM, whereas marker names are indicated to the right of the bar. Markers ending in &#x201C;ds&#x201D; are double simplex markers present in both parents. Markers used to align the subgenomes were amplicons of different fragment length detected with the same primer set. These markers are set-off by similar color labeling. The letters (&lt;em&gt;f&lt;/em&gt;) and (&lt;em&gt;m&lt;/em&gt;) at the end of the linkage group name denote female (PV281) and male (NF472) parental maps, respectively. Severely distorted markers are noted with asterisks: *&lt;em&gt;P&lt;/em&gt; = 0.01, **&lt;em&gt;P&lt;/em&gt; = 0.001, and ***&lt;em&gt;P&lt;/em&gt; = 0.0001. Shaded regions of the individual linkage groups indicate the presence of three or more consecutive markers with TRD. Markers with significant interactions on different linkage groups are indicated by a hash symbol (#).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-3.medium.gif" width="286" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-3.medium.gif" width="286" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/913/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464370" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Consensus linkage map of PV281 and NF472. PV281 linkage groups were constructed by integrating genotypic data from both self-fertilized individuals (<em>n</em> = 74) and PV281 × NF472 hybrid individuals (<em>n</em> = 108) (see <em>Materials and Methods</em>). The Roman numeral designation of each homology group (I–IX) follows the foxtail millet chromosomes. NF472 linkage groups were constructed based only on hybrid individuals as it served as the male donor in the cross. Individual linkage groups are indicated by vertical bars. Numbers to the left of the bar indicate position in cM, whereas marker names are indicated to the right of the bar. Markers ending in “ds” are double simplex markers present in both parents. Markers used to align the subgenomes were amplicons of different fragment length detected with the same primer set. These markers are set-off by similar color labeling. The letters (<em>f</em>) and (<em>m</em>) at the end of the linkage group name denote female (PV281) and male (NF472) parental maps, respectively. Severely distorted markers are noted with asterisks: *<em>P</em> = 0.01, **<em>P</em> = 0.001, and ***<em>P</em> = 0.0001. Shaded regions of the individual linkage groups indicate the presence of three or more consecutive markers with TRD. Markers with significant interactions on different linkage groups are indicated by a hash symbol (#).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Similarly, linkage analysis of 453 polymorphic markers segregating in the male parent map allowed mapping of 315 SD markers and 36 SD × SD markers, and the resulting 18 linkage groups included data from 284 different primer pairs. The numbers of gSSR, eSSR, and STS markers were 306, 102, and 9, respectively. The male map spanned a total length of 1462 cM, with the average length of 81 cM and an average of 23 polymorphic markers per LG. The number of polymorphic markers mapped per LG ranged from 16 to 32, with LG lengths ranging from 57 cM (<em>VIIIb-m</em>) to 109 cM (<em>IXa-m</em>). Marker density ranged from 2.6 to 4.2, with an average of 1.46 mapped markers per primer pair.</p><p id="p-30">The distribution of the three types of markers among the linkage groups was random and no aggregation of genomic or EST markers was observed. The STS markers were mapped in seven LG of both the female and male maps. In the female map, STS markers were placed in LG <em>IIb-f</em>, <em>IVa-f</em>, <em>IVb-f</em>, <em>Va-f</em>, <em>IXa-f</em>, and <em>IXb-f</em>, whereas in the male map they were placed in LG <em>Ib-m</em>, <em>IIb-m</em>, <em>IIIb-m</em>, <em>IVb-m</em>, <em>Va-m</em>, <em>IXa-m</em>, and <em>IXb-m</em>.</p></div><div id="sec-14" class="subsection"><h3>Genome length and coverage</h3><p id="p-31">As estimated from the information of the parental maps, total genome lengths of switchgrass were 1776 cM and 1596 cM for the female map and male map, respectively. Two methods of estimation were used and consistent genome length estimates were obtained using both <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fishman <em>et al.</em> (2001)</a> and <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chakravarti <em>et al.</em> (1991)</a> methods. The proportion of the genome within 5 cM of a marker locus was estimated to be 92% for the female map and 93% for the male map.</p><p id="p-32">The average of two genome length estimates (1686 cM) was taken as the expected genome length of switchgrass. Based on an estimated genome size of ∼1600 Mbp for tetraploid switchgrass (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Saski <em>et al.</em> 2011</a>), the average recombination rate across all LG was approximated at 1.05 cM per Mbp.</p></div><div id="sec-15" class="subsection"><h3>Map integration</h3><p id="p-33">The linkage groups were integrated with one another based on shared SD × SD markers, or SD markers co-amplified by the same primer set that represented either homeologous loci or alternate alleles heterozygous in either both subgenomes or both parents, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Accordingly, nine homology groups of four linkage groups each were formed. All groups were then assigned names consistent with previous nomenclature based on marker alleles shared in common with the linkage map of <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> (2010)</a> and synteny of selected eSSR markers sequences with the foxtail millet (<em>Setaria italica</em>) pseudomolecules (<a href="http://www.phytozome.net">http://www.phytozome.net</a> v7.0). The nine homology groups formed represent the base chromosome number for switchgrass.</p></div><div id="sec-16" class="subsection"><h3>Transmission ratio distortion</h3><p id="p-34">We found that 26% of the genotyped markers were affected by TRD; however, only 17% of the markers included in the final maps were distorted. The rest could not be positioned with certainty. TRD markers that affected the AL-NF (PV281 × NF472) map were compared with the two related switchgrass pseudo-testcross populations that it was developed from: ALB (A4 × K5) and NF-UGA (AP13 × VS16). As compared to the ALB population in which 8.7% of the markers were affected by TRD, the AL-NF and NF-UGA populations showed a higher frequency of TRD markers (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>), with the highest rates (25.1%) found in the NF-UGA population. A total of 143 TRD markers were recorded in the AL-NF population. Of these, 69 segregated in the female and 70 segregated in the male parent, whereas 4 were distorted SD × SD markers that segregated in both parents. In the male map 44% of the SD TRD markers were scored as present in excess, whereas in the female map the percentage of SD TRD markers present in excess was 54%.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464364/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464364/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Number and frequency of molecular markers with transmission ratio distortion in three switchgrass pseudo-testcross mapping populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">Distorter regions consisting of clusters of three or more consecutive TRD markers were observed on LG <em>IIa-f</em>, <em>Vb-f</em>, <em>VIIIa-f</em>, <em>IXa-f</em>, <em>IXb-f</em>, and <em>VIIIa-f</em> in the PV281 map, indicating a biological phenomenon as opposed to genotyping error. Similarly, in the NF472 map LG <em>Ia-m</em>, <em>Ib-m</em>, <em>IIIb-m</em>, <em>and VIIb-m</em> were affected by distorter regions that are indicated in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. In three cases, genotypic information from the founders in these distorter regions allowed determination of which haplotype was present in excess. Regions affected by TRD in LG <em>VIIb-m</em> and <em>IIIb-m</em> were skewed toward the AP13 haplotype, whereas the TRD region on <em>Ia-m</em> was skewed toward the VS16 haplotype. Significant interactions between loci on corresponding LG in the male and female maps in groups <em>IXa-m/f</em> and <em>VIa-m/f</em> between NFSG012-268 on LG <em>IXa-m</em> and the three adjacent markers NFSG238-195_ds, NFSG238-146, and NFSG328-142_ds on LG <em>IXa-f</em> as well as between NFSG007-357 on LG <em>VIa-m</em> and NFSG203-256_ds on LG <em>VIa-f</em> were observed. These interactions are indicated in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>. There were no significant interactions between TRD markers and markers on different homology groups. Comparison of TRD with the related F1 population of <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> (2010)</a> demonstrated that LG <em>Ia-m</em> and <em>Ib-m</em> were affected by distortion in both maps, as was LG <em>VIIb-m</em>.</p></div><div id="sec-17" class="subsection"><h3>Map comparisons</h3><p id="p-36">eSSR, gSSR, and STS primer pairs used in several other switchgrass linkage studies with the ALB population (<a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> 2010</a>), the NF population (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Serba <em>et al.</em> 2013</a>), and the unrelated NL94 population (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Liu <em>et al.</em> 2012</a>) were compared using a string order comparison based on the LCS (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agarwala <em>et al.</em> 2000</a>). This method identified the largest number of sequences or primer sets that had the same relative order between two different linkage groups. The LCS numbers were summed across all LG by subgenome and male or female linkage maps where appropriate and then expressed as a fraction of the total number of SSR shared in common. These results are presented in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. In 75% of the cases it was not possible to compare <em>a</em> and <em>b</em> sub-genomes due to too few SSR shared in common. The percent of SSR in the LCS ranged from 47% to 77%, whereas the number of SSR mapped in common ranged from 3 to 103. Within the AL-NF population the LCS ranged between 0.74 and 0.57, with a maximum of 159 and a minimum of 24 SSR shared in common between subgenomes and male or female linkage maps. The correlation coefficient between AL-NF marker distances intervals was 0.77 for the NF population and 0.40 for the NF94 population.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464331/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464331/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">LCS map comparisons across all linkage groups<sup><em><sup>a</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Similar comparisons to the foxtail millet genome sequence were performed based on the top scoring blast hits of EST-SSR sequences to the foxtail millet genome sequence. The <em>e</em>-values of these results were less than 1 × 10<sup>−40</sup> except for sww2209, which was 2 × 10<sup>−12</sup>. Comparisons were limited to EST-SSR markers and thus there were fewer markers present in the LCS, which contained between 76% and 91% of the total number (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>).</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Population development</h3><p id="p-41">The mapping population was developed in three generations using four switchgrass genotypes selected from four founders. Two of the genotypes, A4 and AP13, were selected from Alamo. The other genotypes, K5 and VS16, were selected from Kanlow and Summer, respectively. Alamo and Kanlow are lowland cultivars that have robust growth and high biomass yield potential, whereas Summer is a northern-adapted upland cultivar with limited biomass yield but cold tolerance.</p><p id="p-42">These populations combine the variability from different founder populations and will be useful for detecting QTL for yield and other relevant traits. Knowledge of the linkage phase is critical for QTL mapping and other applications. In an outcrossing pedigree, the linkage phase of the parents is not known. Genotyping the grandparents allows determination of the linkage phase of the parents using identity by descent and yields improved results where there is limited marker coverage over inferring phase based on independence LOD scores of marker pairs.</p></div><div id="sec-20" class="subsection"><h3>Selfed individuals</h3><p id="p-43">The presence of 74 self-pollinated individuals among the genotyped population requires a slightly modified mapping approach but also provides an opportunity to compare traits potentially impacted by inbreeding depression in related populations with different degrees of heterozygosity. The true rate of outcrossing in this population was not directly determined because differences in fitness at early life stages biased establishment and genotyping of the population. However, those individuals that were genotyped consisted of 59% hybrid and 41% selfed individuals. This indicates that the female parent of the cross (PV281) was effectively self-compatible. There have been several studies examining reproductive systems in switchgrass. These have found that both pre- and post-zygotic barriers to selfing are active, and that bagged inflorescences produce far fewer seeds than open pollinated inflorescences (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Martinez-Reyna and Vogel 2002</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Talbert <em>et al.</em> 1983</a>). Estimates of 0.35% self-compatibility were obtained from 17 upland and lowland tetraploid genotypes by <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Martinez-Reyna and Vogel (2002)</a>.</p><p id="p-44">The discoveries of conditionally compatible genotypes NL94 LYE 16×13 and SL93 7×15 that are self-compatible under controlled conditions but effectively self-incompatible in the field (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Liu <em>et al.</em> 2013</a>) as well as of the high self-compatibility found in PV281 demonstrate the existence of both reproductive modes in switchgrass. However, very little information is available regarding the genetic basis of incompatibility in switchgrass or other polyploid species. Loss of self-incompatibility due to modifiers of major SI loci, nonfunctional incompatibility alleles, or decreased competition from unrelated pollen at the stigma surface or stylar transmitting tissue under controlled conditions may play roles in conditional self-fertility in switchgrass.</p></div><div id="sec-21" class="subsection"><h3>Marker diversity and segregation</h3><p id="p-45">The marker systems utilized in this study were SSR and STS markers. The SSR markers were developed from both genomic and EST sequences. The gSSR markers were developed from (GA/CT)<sub>n</sub> enriched libraries. The eSSR markers were developed from publicly available switchgrass EST sequences. SSR markers have become quite useful in various aspects of molecular genetic studies, including assessment of genetic diversity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amsellem <em>et al.</em> 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ashley <em>et al.</em> 2003</a>), ecological–genetic studies (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Muraya <em>et al.</em> 2011</a>), marker-assisted selection (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fazio <em>et al.</em> 2003</a>), and genetic linkage mapping (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akkaya <em>et al.</em> 1995</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broun and Tanksley 1996</a>). SSR have an additional evolutionary role in creating and maintaining quantitative genetic variation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kashi <em>et al.</em> 1997</a>).</p><p id="p-46">Both gSSR and eSSR markers were used in the construction of this linkage map to cover genic as well as the intergenic regions of switchgrass. The gSSR markers proved to be evenly distributed throughout the genome. Sequence-tagged markers such as eSSR and STS markers have special importance in covering the euchromatic regions of genome (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mun <em>et al.</em> 2006</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ren <em>et al.</em> 2009</a>) and reveal variation in transcribed genes among individuals (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ramchiary <em>et al.</em> 2011</a>). Totals of 136 and 130 eSSR markers were placed on the female map and male map, respectively. In addition, 12 and 9 STS markers were also mapped on the female map and male map, respectively. These STS markers were developed from cellulase and transferase gene sequences and have special importance in mapping the genome region and genes involved in cellulose accumulation in switchgrass. It will be interesting to see co-segregation of any of these markers with QTL associated with biomass quality traits.</p><p id="p-47">Switchgrass is an outcrossing plant; therefore, the parental genotypes are heterozygous. In a cross between such heterozygous parents, many single-dose polymorphic markers will amplify in one parent but will be absent in the other; therefore, they are expected to segregate in a 1:l ratio in their progeny (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Grattapaglia and Sederoff 1994</a>). Other markers are heterozygous in both the parents and expected to segregate 3:1 under disomic inheritance or 5:1 under tetrasomic inheritance models. The majority of the SSR and STS markers assayed in this population were found to follow the expected 1:1 ratio for the SD markers and 3:1 ratio for the SD × SD cross. This agrees with previous findings that switchgrass inheritance is disomic.</p></div><div id="sec-22" class="subsection"><h3>Linkage map and genome coverage</h3><p id="p-48">The maps lengths produced by this study were similar to those reported in the NF population in which the female linkage map was 13% larger than the male map. In this study, the female linkage map was 11% larger than the male linkage map. Similarly, the overall map lengths were comparable, with this genetic linkage map being 90% of the length of that reported by <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Serba <em>et al.</em> (2013)</a>. These map lengths were both significantly smaller than that reported for the NL94 population. The relative lengths of the male and female linkage maps in this study were the opposite of the ALB population in which the male map was found to be relatively shorter. These results suggest that meiosis in the NF and AL-NF populations was behaving in a consistent manner, and that inclusion of increasing percentages of TRD markers in the individual switchgrass linkage studies did not appear to greatly inflate map length, consistent with the theory of <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hackett and Broadfoot (2003)</a>.</p></div><div id="sec-23" class="subsection"><h3>Transmission ratio distortion</h3><p id="p-49">Distorter loci affecting the transmission of closely linked markers are commonly present in mapping populations and can be linked to pre- or post-zygotic mechanisms, including pollen tube competition (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mangelsdorf and Jones 1926</a>), pollen lethals (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rick 1966</a>), preferential fertilization (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gadish and Zamir 1987</a>), and selective elimination of zygotes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rick 1963</a>). Selfish genetic elements are also selected for in the asymmetric female meiosis of most plants and animals as meiotic drive. In the AL-NF population, TRD reached 17.1% in mapped markers. These rates of distortion are similar to that observed in the NF-GA mapping population (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Serba <em>et al.</em> 2013</a>). However, <a id="xref-ref-42-10" class="xref-bibr" href="#ref-42">Okada <em>et al.</em> (2010)</a> observed an SD of only 9% for the single dose markers of genomic and eSSR markers. This lower SD in the ALB population is attributed to the omission of highly distorted markers from the attempted mapping rather than significant biological differences. The distorted markers were distributed throughout the maps. There were several clusters of three or more loci in the female map and male map. In the female map TRD regions were observed on six LG, whereas TRD regions were observed on 11 LG in the male map. Significant interactions between markers in male and female LG <em>VI</em> and <em>IX</em> in the AL-NF population indicate possible post-zygotic interactions. The presence of TRD loci may lead to bias in estimation of the recombination fraction when two or more TRD loci are present on a single chromosome (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lorieux <em>et al.</em> 1995a</a>,<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">b</a>). However, in our study we did not detect this situation, leading us to believe that our maps were not biased due to TRD.</p></div></div><div class="section conclusions" id="sec-24"><h2 class="">Conclusions</h2><p id="p-50">A three-generation and four-founder mapping population was developed in tetraploid switchgrass. We have constructed genetic linkage map using a combination of SSR and STS markers, with marker density close to saturation. The LG were arranged in nine homeologs corresponding to the haploid chromosome number of tetraploid switchgrass. Clustering of TRD markers was observed in some genomic regions. Using parental maps to identify QTL underlying agronomic traits for MAS is in progress. Due to the long establishment time of switchgrass and the need for multiple-year yield data, this approach has high potential to speed selection.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">Our sincere appreciation goes to Yuhong Tang and her group for genotyping and the greenhouse group of the Noble Foundation for greenhouse maintenance of the population. We are very thankful to Brian Motes and his group members for taking care of field experiments. This research was supported with funds provided by the Department of Energy and United States Department of Agriculture. The United States Department of Agriculture, Agricultural Research Service, is an equal opportunity/affirmative action employer and all agency services are available without discrimination. Mention of commercial products and organizations in this manuscript is solely to provide specific information. It does not constitute endorsement by United States Department of Agriculture, Agricultural Research Service, over other products and organizations not mentioned.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> These authors contributed equally to this work.</p></li><li class="fn" id="fn-6"><p id="p-52"><em>Communicating editor: J. A. Birchler</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-12-31"><span class="received-label">Received </span>December 31, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-03-13"><span class="accepted-label">Accepted </span>March 13, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Li <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.1" data-doi="10.1101/gr.10.3.350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Applegate</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maglott</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuler</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A fast and scalable radiation hybrid map construction and integration strategy.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">350</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DAgarwala%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D350%26rft.epage%253D364%26rft.atitle%253DA%2BFast%2Band%2BScalable%2BRadiation%2BHybrid%2BMap%2BConstruction%2Band%2BIntegration%2BStrategy%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.3.350%26rft_id%253Dinfo%253Apmid%252F10720576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzMvMzUwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.2" data-doi="10.2135/cropsci1995.0011183X003500050030x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akkaya</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwat</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Integration of simple sequence repeat DNA markers into a soybean linkage map.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DAkkaya%26rft.auinit1%253DM.%2BS.%26rft.volume%253D35%26rft.issue%253D5%26rft.spage%253D1439%26rft.epage%253D1445%26rft.atitle%253DIntegration%2Bof%2BSimple%2BSequence%2BRepeat%2BDNA%2BMarkers%2Binto%2Ba%2BSoybean%2BLinkage%2BMap%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1995.0011183X003500050030x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1995.0011183X003500050030x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RR62100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.3" data-doi="10.1046/j.1471-8278 .2000.00013.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amsellem</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutech</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billotte</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Isolation and characterization of polymorphic microsatellite loci in <em>Rubus alceifolius</em> Poir. (Rosaceae), an invasive weed in la Réunion island.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D1%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8278%2B.2000.00013.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8278 .2000.00013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.4" data-doi="10.1007/s00122-003-1370-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashley</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilk</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styan</span>  <span class="cit-name-given-names">S. M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craft</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High variability and disomic segregation of microsatellites in the octoploid <em>Fragaria virginiana</em> Mill. (Rosaceae).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAshley%26rft.auinit1%253DM.%2BV.%26rft.volume%253D107%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1207%26rft.atitle%253DHigh%2Bvariability%2Band%2Bdisomic%2Bsegregation%2Bof%2Bmicrosatellites%2Bin%2Bthe%2Boctoploid%2BFragaria%2Bvirginiana%2BMill.%2B%2528Rosaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1370-5%26rft_id%253Dinfo%253Apmid%252F12908097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1370-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12908097&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186401200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.5" data-doi="10.1016/j.plantsci.2009.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barney</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kyser</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blumwald</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite> Deynze, <em>et al</em>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tolerance of switchgrass to extreme soil moisture stress: ecological implications.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D177%26rft.spage%253D724%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2009.09.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2009.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000271552000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.6" data-doi="10.1038/nbt.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percifield</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reference genome sequence of the model plant Setaria.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBennetzen%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D30%26rft.issue%253D6%26rft.spage%253D555%26rft.epage%253D561%26rft.atitle%253DReference%2Bgenome%2Bsequence%2Bof%2Bthe%2Bmodel%2Bplant%2BSetaria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2196%26rft_id%253Dinfo%253Apmid%252F22580951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22580951&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.7" data-doi="10.1007/BF02191823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broun</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Characterization and genetic mapping of simple repeat sequences in the tomato genome.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">250</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DBroun%26rft.auinit1%253DP.%26rft.volume%253D250%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D49%26rft.atitle%253DCharacterization%2Band%2Bgenetic%2Bmapping%2Bof%2Bsimple%2Brepeat%2Bsequences%2Bin%2Bthe%2Btomato%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02191823%26rft_id%253Dinfo%253Apmid%252F8569686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02191823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8569686&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.913.8"><div class="cit-metadata unstructured">Bouton, J., 2008 Improvement of switchgrass as a bioenergy crop, pp. 295–308 in <em>Genetic Improvement of Bioenergy Crops</em>, edited by W. Vermerris. Springer Science + Business Media, LLC, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.9" data-doi="10.2135/cropsci2005.0388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ecotypic variation among switchgrass populations from the northern USA.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D388%26rft.epage%253D398%26rft.atitle%253DEcotypic%2BVariation%2Bamong%2BSwitchgrass%2BPopulations%2Bfrom%2Bthe%2BNorthern%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226435300048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.10" data-doi="10.2135/cropsci2004.2930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wynia</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Latitudinal adaptation of switchgrass populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D293%26rft.epage%253D303%26rft.atitle%253DLatitudinal%2BAdaptation%2Bof%2BSwitchgrass%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.2930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.2930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187633900041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasher</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reefer</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A maximum likelihood method for estimating genome length using genetic linkage data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChakravarti%26rft.auinit1%253DA.%26rft.volume%253D128%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DA%2Bmaximum%2Blikelihood%2Bmethod%2Bfor%2Bestimating%2Bgenome%2Blength%2Busing%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Apmid%252F2060775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI4LzEvMTc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic variability and trait relationships in switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DDas%26rft.auinit1%253DM.%2BK.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D443%26rft.epage%253D448%26rft.atitle%253DGenetic%2BVariability%2Band%2BTrait%2BRelationships%2Bin%2BSwitchgrass%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000189384600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elbersen</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassen</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacher</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauerbeck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Switchgrass variety choice in Europe.</span> <abbr class="cit-jnl-abbrev">Asp. Appl. Biol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsp.%2BAppl.%2BBiol.%26rft.volume%253D65%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.14" data-doi="10.2134/jeq2004.0329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanylo</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abaye</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dundas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zipper</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemus</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Herbaceous vegetation productivity, persistence, and metals uptake on a biosolids-amended mine soil.</span> <abbr class="cit-jnl-abbrev">J. Environ. Qual.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1811</span>–<span class="cit-lpage">1819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEnvironmental%2BQuality%26rft.stitle%253DJ.%2BEnviron.%2BQual.%26rft.aulast%253DEvanylo%26rft.auinit1%253DG.%2BK.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D1811%26rft.epage%253D1819%26rft.atitle%253DHerbaceous%2BVegetation%2BProductivity%252C%2BPersistence%252C%2Band%2BMetals%2BUptake%2Bon%2Ba%2BBiosolids-Amended%2BMine%2BSoil%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fjeq2004.0329%26rft_id%253Dinfo%253Apmid%252F16151233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/jeq2004.0329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16151233&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232174300036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.15" data-doi="10.1007/s00122-003-1277-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fazio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staub</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic mapping and QTL analysis of horticultural traits in cucumber (<em>Cucumis sativus</em> L.) using recombinant inbred lines</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">864</span>–<span class="cit-lpage">874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFazio%26rft.auinit1%253DG.%26rft.volume%253D107%26rft.issue%253D5%26rft.spage%253D864%26rft.epage%253D874%26rft.atitle%253DGenetic%2Bmapping%2Band%2BQTL%2Banalysis%2Bof%2Bhorticultural%2Btraits%2Bin%2Bcucumber%2B%2528%2BCucumis%2Bsativus%2BL.%2529%2Busing%2Brecombinant%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1277-1%26rft_id%253Dinfo%253Apmid%252F12827247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1277-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12827247&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.16" data-doi="10.1016/j.biombioe.2005.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fike</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balasko</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">J. T.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Long-term yield potential of switchgrass-for-biofuel systems.</span> <abbr class="cit-jnl-abbrev">Biomass Bioenergy</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomass%2BBioenergy%26rft.volume%253D30%26rft.spage%253D198%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biombioe.2005.10.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biombioe.2005.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fishman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A Genetic map in the <em>Mimulus guttatus</em> species complex reveals transmission ratio distortion due to heterospecific interactions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1701</span>–<span class="cit-lpage">1716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFishman%26rft.auinit1%253DL.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1701%26rft.epage%253D1716%26rft.atitle%253DA%2BGenetic%2BMap%2Bin%2Bthe%2BMimulus%2Bguttatus%2BSpecies%2BComplex%2BReveals%2BTransmission%2BRatio%2BDistortion%2Bdue%2Bto%2BHeterospecific%2BInteractions%26rft_id%253Dinfo%253Apmid%252F11779808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE3MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gadish</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Differential zygotic abortion in an interspecific Lycopersicon cross.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D29%26rft.spage%253D156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Dorado</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallego</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">On the use of the classical tests for detecting linkage.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DGarcia-Dorado%26rft.auinit1%253DA.%26rft.volume%253D83%26rft.issue%253D2%26rft.spage%253D143%26rft.epage%253D146%26rft.atitle%253DOn%2Bthe%2BUse%2Bof%2Bthe%2BClassical%2BTests%2Bfor%2BDetecting%2BLinkage%26rft_id%253Dinfo%253Apmid%252F1578105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjgzLzIvMTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.21" data-doi="10.3835/plantgenome2011.01.0004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez-Gonzalez</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garvin</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reference genome-directed resolution of homologous and homeologous relationships within and between different oat linkage maps.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D178%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.01.0004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.01.0004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.22" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.23" data-doi="10.2135/cropsci1996.0011183X003600050021x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chromosome number and nuclear DNA content of several switchgrass populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1192</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHopkins%26rft.auinit1%253DA.%2BA.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D1192%26rft.epage%253D1195%26rft.atitle%253DChromosome%2BNumber%2Band%2BNuclear%2BDNA%2BContent%2Bof%2BSeveral%2BSwitchgrass%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600050021x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600050021x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VK30900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.24" data-doi="10.2135/cropsci1996.0011183X003600040039x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hultquist</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arumuganathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chloroplast DNA and nuclear DNA content variations among cultivars of switchgrass, <em>Panicum virgatum</em> L.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHultquist%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D1049%26rft.epage%253D1052%26rft.atitle%253DChloroplast%2BDNA%2Band%2BNuclear%2BDNA%2BContent%2BVariations%2Bamong%2BCultivars%2Bof%2BSwitchgrass%252C%2BPanicum%2Bvirgatum%2BL.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600040039x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600040039x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UV54900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.25" data-doi="10.18637/jss.v022.i04"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jachner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Boogaart</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petzoldt</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Statistical methods for the qualitative assessment of dynamic models with time delay (R Package qualV).</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D22%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v022.i04%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v022.i04&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.913.26"><div class="cit-metadata unstructured">Jannink, J.-L., and B. Walsh, 2002 Association mapping in plant populations, pp. 59–68 in <em>Quantitative Genetics</em>, <em>Genomics and Plant Breeding</em>, edited by M. S. Kang. CABI Publishing, Wallingford, UK.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.27" data-doi="10.1016/S0168-9525(97)01008-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Simple sequence repeats as a source of quantitative genetic variation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKashi%26rft.auinit1%253DY.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D74%26rft.epage%253D78%26rft.atitle%253DSimple%2Bsequence%2Brepeats%2Bas%2Ba%2Bsource%2Bof%2Bquantitative%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252897%252901008-1%26rft_id%253Dinfo%253Apmid%252F9055609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(97)01008-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9055609&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WG90300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.28" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.29" data-doi="10.1007/s00122-011-1617-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prevalence of segregation distortion in diploid alfalfa and its implications for genetics and breeding applications.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">667</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D667%26rft.epage%253D679%26rft.atitle%253DPrevalence%2Bof%2Bsegregation%2Bdistortion%2Bin%2Bdiploid%2Balfalfa%2Band%2Bits%2Bimplications%2Bfor%2Bgenetics%2Band%2Bbreeding%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1617-5%26rft_id%253Dinfo%253Apmid%252F21625992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1617-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21625992&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.30" data-doi="10.1007/s12155-011-9173-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of a selfing compatible genotype and mode of inheritance in switchgrass.</span> <abbr class="cit-jnl-abbrev">Bioenerg. Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioenerg.%2BRes.%26rft.volume%253D5%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-011-9173-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-011-9173-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samuels</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> A high-density simple sequence repeat-based genetic linkage map of switchgrass. G3 2: 357–370.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-4.5.913.32"><div class="cit-metadata unstructured">Liu L., Thames S. L., Y. Wu, 2013 Lowland switchgrass plants in populations set completely outcrossed seeds under field conditions as assessed with SSR markers. BioEnerg. Res. 7: 253–259.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.33" data-doi="10.1007/BF00220998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">León</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanaud</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span><span class="cit-comment">a</span> <span class="cit-article-title">Maximum-likelihood models for mapping genetic markers showing segregation distortion. 1. Backcross populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220998%26rft_id%253Dinfo%253Apmid%252F24173786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24173786&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF11000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.34" data-doi="10.1007/BF00220999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanaud</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">León</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span><span class="cit-comment">b</span> <span class="cit-article-title">Maximum-likelihood models for mapping genetic markers showing segregation distortion. 2. F2 populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220999%26rft_id%253Dinfo%253Apmid%252F24173787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24173787&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF11000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.35" data-doi="10.1007/s00122-002-0970-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, R.-J. Severson, and R. Bernardo, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Chromosomal regions associated with segregation distortion in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLu%26rft.auinit1%253DH.%26rft.volume%253D105%26rft.issue%253D4%26rft.spage%253D622%26rft.epage%253D628%26rft.atitle%253DChromosomal%2Bregions%2Bassociated%2Bwith%2Bsegregation%2Bdistortion%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-0970-9%26rft_id%253Dinfo%253Apmid%252F12582513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-0970-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582513&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178256200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangelsdorf</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1926</span> <span class="cit-article-title">The expression of Mendelian factors in the gametophyte of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMangelsdorf%26rft.auinit1%253DP.%2BC.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D423%26rft.epage%253D455%26rft.atitle%253DTHE%2BEXPRESSION%2BOF%2BMENDELIAN%2BFACTORS%2BIN%2BTHE%2BGAMETOPHYTE%2BOF%2BMAIZE%26rft_id%253Dinfo%253Apmid%252F17246465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxMS81LzQyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.37" data-doi="10.2135/cropsci2002.1800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez-Reyna</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Incompatibility systems in switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1800</span>–<span class="cit-lpage">1805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMartinez-Reyna%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D1800%26rft.epage%253D1805%26rft.atitle%253DIncompatibility%2BSystems%2Bin%2BSwitchgrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.1800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.1800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000181430200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.38" data-doi="10.1007/s00122-005-1935-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Missaoui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Investigation of genomic organization in switchgrass (<em>Panicum virgatum</em> L.) using DNA markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1372</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMissaoui%26rft.auinit1%253DA.%2BM.%26rft.volume%253D110%26rft.issue%253D8%26rft.spage%253D1372%26rft.epage%253D1383%26rft.atitle%253DInvestigation%2Bof%2Bgenomic%2Borganization%2Bin%2Bswitchgrass%2B%2528Panicum%2Bvirgatum%2BL.%2529%2Busing%2BDNA%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-1935-6%26rft_id%253Dinfo%253Apmid%252F15841364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-1935-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15841364&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229718000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.913.39"><div class="cit-metadata unstructured">Moser, L. E., and K. P. Vogel, 1995 Switchgrass, big bluestem, and indiangrass, pp. 409–420 in <em>An Introduction to Grassland Agriculture</em>, edited by R. F. Barnes, J. C. Nelson, M. Collins, and K. J. Moore. Iowa State University Press, Ames, IA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.40" data-doi="10.1534/genetics.105.054791"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mun</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debellé</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Distribution of microsatellites in the genome of <em>Medicago truncatula</em>: A resource of genetic markers that integrate genetic and physical maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2541</span>–<span class="cit-lpage">2555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.054791%26rft_id%253Dinfo%253Apmid%252F16489220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI1NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.41" data-doi="10.1007/s00122-011-1560-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muraya</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutegi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiger</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villiers</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagnard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Wild sorghum from different eco-geographic regions of Kenya display a mixed mating system.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMuraya%26rft.auinit1%253DM.%2BM.%26rft.volume%253D122%26rft.issue%253D8%26rft.spage%253D1631%26rft.epage%253D1639%26rft.atitle%253DWild%2Bsorghum%2Bfrom%2Bdifferent%2Beco-geographic%2Bregions%2Bof%2BKenya%2Bdisplay%2Ba%2Bmixed%2Bmating%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1560-5%26rft_id%253Dinfo%253Apmid%252F21360157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1560-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21360157&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.42" data-doi="10.1534/genetics.110.113910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzatella</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.113910%26rft_id%253Dinfo%253Apmid%252F20407132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvNzQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.43" data-doi="10.1093/dnares/dsr017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramchiary</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhandapani</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genic microsatellite markers in <em>Brassica rapa</em>: Development, characterization, mapping, and their utility in other cultivated and wild brassica relatives.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsr017%26rft_id%253Dinfo%253Apmid%252F21768136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvMzA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.44" data-doi="10.1371/journal.pone.0005795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staub</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An integrated genetic and cytogenetic map of the cucumber genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRen%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De5795%26rft.epage%253De5795%26rft.atitle%253DAn%2Bintegrated%2Bgenetic%2Band%2Bcytogenetic%2Bmap%2Bof%2Bthe%2Bcucumber%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005795%26rft_id%253Dinfo%253Apmid%252F19495411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19495411&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Differential zygotic lethality in a tomato species hybrid.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1497</span>–<span class="cit-lpage">1507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRick%26rft.auinit1%253DC.%2BM.%26rft.volume%253D48%26rft.issue%253D11%26rft.spage%253D1497%26rft.epage%253D1507%26rft.atitle%253DDIFFERENTIAL%2BZYGOTIC%2BLETHALITY%2BIN%2BA%2BTOMATO%2BSPECIES%2BHYBRID%26rft_id%253Dinfo%253Apmid%252F17248190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czoxMDoiNDgvMTEvMTQ5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Abortion of male and female gametes in the tomato determined by allelic interaction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRick%26rft.auinit1%253DC.%2BM.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D96%26rft.atitle%253DABORTION%2BOF%2BMALE%2BAND%2BFEMALE%2BGAMETES%2BIN%2BTHE%2BTOMATO%2BDETERMINED%2BBY%2BALLELIC%2BINTERACTION%26rft_id%253Dinfo%253Apmid%252F5900610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI1My8xLzg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.47" data-doi="10.4141/P06-136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adler</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boateng</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarath</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Switchgrass as a biofuels feedstock in the USA.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1315</span>–<span class="cit-lpage">1325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BSci.%26rft.volume%253D86%26rft.spage%253D1315%26rft_id%253Dinfo%253Adoi%252F10.4141%252FP06-136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4141/P06-136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiraizum</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Meiotic drive in natural populations of <em>Drosophila melanogaster</em>. I. The cytogenetic basis of segregation-distortion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSandler%26rft.auinit1%253DL.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D250%26rft.atitle%253DMEIOTIC%2BDRIVE%2BIN%2BNATURAL%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BMELANOGASTER.%2BI.%2BTHE%2BCYTOGENETIC%2BBASIS%2BOF%2BSEGREGATION-DISTORTION%26rft_id%253Dinfo%253Apmid%252F17247821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0NC8yLzIzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.49" data-doi="10.1186/1471-2164-12-369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feltus</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">New genomic resources for switchgrass: a BAC library and comparative analysis of homoeologous genomic regions harboring bioenergy traits.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSaski%26rft.auinit1%253DC.%2BA.%26rft.volume%253D12%26rft.spage%253D369%26rft.epage%253D369%26rft.atitle%253DNew%2Bgenomic%2Bresources%2Bfor%2Bswitchgrass%253A%2Ba%2BBAC%2Blibrary%2Band%2Bcomparative%2Banalysis%2Bof%2Bhomoeologous%2Bgenomic%2Bregions%2Bharboring%2Bbioenergy%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-369%26rft_id%253Dinfo%253Apmid%252F21767393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21767393&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.50" data-doi="10.1038/72708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuelke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An economic method for the fluorescent labeling of PCR fragments.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchuelke%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D234%26rft.atitle%253DAn%2Beconomic%2Bmethod%2Bfor%2Bthe%2Bfluorescent%2Blabeling%2Bof%2BPCR%2Bfragments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F72708%26rft_id%253Dinfo%253Apmid%252F10657137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/72708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10657137&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085146900036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.51" data-doi="10.1007/s12155-013-9315-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daverdin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahri</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Linkage maps of lowland and upland tetraploid switchgrass ecotypes.</span> <abbr class="cit-jnl-abbrev">BioEnergy Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BRes.%26rft.volume%253D6%26rft.spage%253D953%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-013-9315-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-013-9315-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.52" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linkage maps by means of a new computer package: Join Map.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.53" data-doi="10.2135/cropsci1983.0011183X002300040029x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timothy</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawlings</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Estimates of genetic parameters in switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTalbert%26rft.auinit1%253DL.%2BE.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D728%26rft.atitle%253DEstimates%2Bof%2BGenetic%2BParameters%2Bin%2BSwitchgrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1983.0011183X002300040029x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1983.0011183X002300040029x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RE73500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.913.54"><div class="cit-metadata unstructured">VanOoijen, J. W., 2006 JoinMap v.4: Software for the calculation of genetic linkage maps in experimental populations. Kyazma B. V., Wageningen, Netherlands.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.55" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: Software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.913.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.-X.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source"><em>Statistical genetics of quantitative traits linkage</em>, <em>maps</em>, <em>and QTL</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.57" data-doi="10.1006/tpbi.2002.1577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D363%26rft.atitle%253DSimultaneous%2Bmaximum%2Blikelihood%2Bestimation%2Bof%2Blinkage%2Band%2Blinkage%2Bphases%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1577%26rft_id%253Dinfo%253Apmid%252F12027621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027621&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176540100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.913.58" data-doi="10.2135/cropsci2011.02.0104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Natural hybrids and gene flow between upland and lowland switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2626</span>–<span class="cit-lpage">2641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2626%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.02.0104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.02.0104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/5/913.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/901" title="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/925" title="Efficient Gene Knock-out and Knock-in with Transgenic Cas9 in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/5"><span>Volume 4 Issue 5, May 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/5.cover-source.jpg?itok=AAKWoOtu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/913.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sUnAeHJIiag37aKLIAKAjBoPBRuBrJ214bDUfr5Kqcg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/5/913" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/5/913" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/5/913" class="active" data-icon-position="" data-hide-link-title="0">Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459864" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yPcNxFy_JFkTYQFyYxlD3idnKGqdkpWH_0MCUGzp3Hc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/913.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/5/913" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ms4GQupmeplxHi46d7CrJPZ5Zf_K5cThGeh4Re-BS6I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/5/913" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/913.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459864" id="node459864--2" data-pisa="ggg;4/5/913" data-pisa-master="ggg;g3.113.010165" data-apath="/ggg/4/5/913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of <em>Panicum virgatum</em> (L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guifen Li</span>, <span class="highwire-citation-author" data-delta="1">Desalegn D. Serba</span>, <span class="highwire-citation-author" data-delta="2">Malay C. Saha</span>, <span class="highwire-citation-author" data-delta="3">Joseph H. Bouton</span>, <span class="highwire-citation-author" data-delta="4">Christina L. Lanzatella</span> and <span class="highwire-citation-author" data-delta="5">Christian M. Tobias</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">913-923;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.010165 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459864--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guifen Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guifen%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuifen%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Desalegn D. Serba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Desalegn%2BD.%2BSerba%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serba%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADesalegn%2BD.%2BSerba%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Malay C. Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malay%2BC.%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalay%2BC.%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph H. Bouton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BH.%2BBouton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouton%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BH.%2BBouton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christina L. Lanzatella </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BL.%2BLanzatella%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lanzatella%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristina%2BL.%2BLanzatella%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Christian M. Tobias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BM.%2BTobias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tobias%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BM.%2BTobias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.tobias@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">christian.tobias@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/5/913" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/5/913&amp;title=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/5/913&amp;title=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/5/913&amp;title=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/5/913&amp;text=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/5/913&amp;title=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/5/913&amp;t=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/5/913&amp;title=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/5/913&amp;title=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459864/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459864" id="node459864--3" data-pisa="ggg;4/5/913" data-pisa-master="ggg;g3.113.010165" data-apath="/ggg/4/5/913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of <em>Panicum virgatum</em> (L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guifen Li</span>, <span class="highwire-citation-author" data-delta="1">Desalegn D. Serba</span>, <span class="highwire-citation-author" data-delta="2">Malay C. Saha</span>, <span class="highwire-citation-author" data-delta="3">Joseph H. Bouton</span>, <span class="highwire-citation-author" data-delta="4">Christina L. Lanzatella</span> and <span class="highwire-citation-author" data-delta="5">Christian M. Tobias</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">913-923;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.010165 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459864--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guifen Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guifen%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuifen%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Desalegn D. Serba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Desalegn%2BD.%2BSerba%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serba%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADesalegn%2BD.%2BSerba%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Malay C. Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malay%2BC.%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalay%2BC.%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph H. Bouton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Samuel Roberts Noble Foundation, Forage Improvement Division, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BH.%2BBouton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouton%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BH.%2BBouton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christina L. Lanzatella </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BL.%2BLanzatella%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lanzatella%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristina%2BL.%2BLanzatella%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Christian M. Tobias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture–Agricultural Research Service, Western Regional Research Center, Albany, California 94710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BM.%2BTobias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tobias%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BM.%2BTobias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.tobias@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">christian.tobias@ars.usda.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=913&amp;rft.epage=923&amp;rft.atitle=Genetic%20Linkage%20Mapping%20and%20Transmission%20Ratio%20Distortion%20in%20a%20Three-Generation%20Four-Founder%20Population%20of%20Panicum%20virgatum%20%28L.%29&amp;rft.volume=4&amp;rft.issue=5&amp;rft.date=2014-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Guifen&amp;rft.au=Serba%2C+Desalegn+D.&amp;rft.au=Saha%2C+Malay+C.&amp;rft.au=Bouton%2C+Joseph+H.&amp;rft.au=Lanzatella%2C+Christina+L.&amp;rft.au=Tobias%2C+Christian+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459864/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459864/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459864/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459864/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459864/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459864/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459864/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459864/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459864/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459864/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459864/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459864/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/5/913" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/5/913.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/5/913.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/4/5/913.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
