<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/5/891" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/5/891.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/5/891.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/5/891.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/5/891.atom" />
<meta name="HW.pisa" content="ggg;4/5/891" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment" />
<meta name="DC.Identifier" content="10.1534/g3.114.010942" />
<meta name="DC.Date" content="2014-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Fu. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping by sequencing (GBS) recently has emerged as a promising genomic approach for assessing genetic diversity on a genome-wide scale. However, concerns are not lacking about the uniquely large unbalance in GBS genotype data. Although some genotype imputation has been proposed to infer missing observations, little is known about the reliability of a genetic diversity analysis of GBS data, with up to 90% of observations missing. Here we performed an empirical assessment of accuracy in genetic diversity analysis of highly incomplete single nucleotide polymorphism genotypes with imputations. Three large single-nucleotide polymorphism genotype data sets for corn, wheat, and rice were acquired, and missing data with up to 90% of missing observations were randomly generated and then imputed for missing genotypes with three map-independent imputation methods. Estimating heterozygosity and inbreeding coefficient from original, missing, and imputed data revealed variable patterns of bias from assessed levels of missingness and genotype imputation, but the estimation biases were smaller for missing data without genotype imputation. The estimates of genetic differentiation were rather robust up to 90% of missing observations but became substantially biased when missing genotypes were imputed. The estimates of topology accuracy for four representative samples of interested groups generally were reduced with increased levels of missing genotypes. Probabilistic principal component analysis based imputation performed better in terms of topology accuracy than those analyses of missing data without genotype imputation. These findings are not only significant for understanding the reliability of the genetic diversity analysis with respect to large missing data and genotype imputation but also are instructive for performing a proper genetic diversity analysis of highly incomplete GBS or other genotype data." />
<meta name="DC.Contributor" content="Yong-Bi Fu" />
<meta name="article:published_time" content="2014-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping by sequencing (GBS) recently has emerged as a promising genomic approach for assessing genetic diversity on a genome-wide scale. However, concerns are not lacking about the uniquely large unbalance in GBS genotype data. Although some genotype imputation has been proposed to infer missing observations, little is known about the reliability of a genetic diversity analysis of GBS data, with up to 90% of observations missing. Here we performed an empirical assessment of accuracy in genetic diversity analysis of highly incomplete single nucleotide polymorphism genotypes with imputations. Three large single-nucleotide polymorphism genotype data sets for corn, wheat, and rice were acquired, and missing data with up to 90% of missing observations were randomly generated and then imputed for missing genotypes with three map-independent imputation methods. Estimating heterozygosity and inbreeding coefficient from original, missing, and imputed data revealed variable patterns of bias from assessed levels of missingness and genotype imputation, but the estimation biases were smaller for missing data without genotype imputation. The estimates of genetic differentiation were rather robust up to 90% of missing observations but became substantially biased when missing genotypes were imputed. The estimates of topology accuracy for four representative samples of interested groups generally were reduced with increased levels of missing genotypes. Probabilistic principal component analysis based imputation performed better in terms of topology accuracy than those analyses of missing data without genotype imputation. These findings are not only significant for understanding the reliability of the genetic diversity analysis with respect to large missing data and genotype imputation but also are instructive for performing a proper genetic diversity analysis of highly incomplete GBS or other genotype data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/05/01" />
<meta name="citation_mjid" content="ggg;4/5/891" />
<meta name="citation_id" content="4/5/891" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/5/891" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/5/891.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/5/891.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/5/891.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010942" />
<meta name="citation_pmid" content="24626289" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="891" />
<meta name="citation_lastpage" content="900" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yong-Bi Fu" />
<meta name="citation_author_institution" content="Plant Gene Resources of Canada, Saskatoon Research Centre, Agriculture and Agri-Food Canada, Saskatoon, SK S7N 0X2, Canada" />
<meta name="citation_author_email" content="yong-bi.fu@agr.gc.ca" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Altshuler;citation_author=V. J. Pollara;citation_author=C. R. Cowles;citation_author=W. J. van Etten;citation_author=J. Baldwin;citation_title=An SNP map of the human genome generated by reduced representation shotgun sequencing.;citation_pages=513-516;citation_volume=407;citation_year=2000;citation_issue=6803;citation_pmid=11029002;citation_doi=10.1038/35035083" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forests.;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_title=Multiple imputation of unordered categorical data, in multiple imputation and its application;citation_author=J. R. Carpenter;citation_author=M. G. Kenward;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_journal_title=Frontiers Genet.;citation_author=J. E. Crawford;citation_author=B. P. Lazzaro;citation_title=ASSESSING THE ACCURACY AND POWER OF POPULATION GENETIC INFERENCE FROM LOW-PASS NEXT-GENERATION SEQUENCING DATA.;citation_pages=66;citation_volume=3;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=D. Thureen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=L. Excoffier;citation_author=H. E. L. Lischer;citation_title=Arlequin suite ver 3.5: A new series of programs to perform population genetics analyses under Linux and Windows.;citation_pages=564-567;citation_volume=10;citation_year=2010;citation_doi=10.1111/j.1755-0998.2010.02847.x" />
<meta name="citation_reference" content="citation_title=The Second Report on the State of the World’s Plant Genetic Resources for Food and Agriculture;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=Y. B. Fu;citation_author=G. W. Peterson;citation_title=Genetic diversity analysis with 454 pyrosequencing and genomic reduction confirmed the eastern and western division in the cultivated barley gene pool.;citation_pages=226-237;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0022" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Y. B. Fu;citation_author=G. W. Peterson;citation_title=Developing genomic resources in two Linum species via 454 pyrosequencing and genomic reduction.;citation_pages=492-500;citation_volume=12;citation_year=2012;citation_pmid=22177006;citation_doi=10.1111/j.1755-0998.2011.03100.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=Y. B. Fu;citation_author=B. Cheng;citation_author=G. W. Peterson;citation_title=Genetic diversity analysis of yellow mustard (Sinapis alba L.) germplasm based on genotyping by sequencing.;citation_pages=579-594;citation_volume=61;citation_year=2014;citation_doi=10.1007/s10722-013-0058-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Fumagalli;citation_author=F. G. Vieira;citation_author=T. S. Korneliussen;citation_author=T. Linderoth;citation_author=E. Huerta-Sanchez;citation_title=Quantifying population genetic differentiation from next-generation sequencing data.;citation_volume=195;citation_year=2013;citation_pmid=23979584;citation_doi=10.1534/genetics.113.154740" />
<meta name="citation_reference" content="citation_journal_title=The American Statistician;citation_journal_abbrev=The American Statistician;citation_author=N. J. Horton;citation_author=K. P. Kleinman;citation_title=Much ado about nothing: A comparison of missing data methods and software to fit incomplete data regression models.;citation_pages=79-90;citation_volume=61;citation_year=2007;citation_issue=1;citation_pmid=17401454;citation_doi=10.1198/000313007X172556" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. Iwata;citation_author=J.-L. Jannink;citation_title=Marker Genotype Imputation in a Low-Marker-Density Panel with a High-Marker-Density Reference Panel: Accuracy Evaluation in Barley Breeding Lines;citation_pages=1269-1278;citation_volume=50;citation_year=2010;citation_issue=4;citation_doi=10.2135/cropsci2009.08.0434" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Liaw;citation_author=M. Wiener;citation_title=Classification and regression by randomForest.;citation_pages=18-22;citation_volume=2;citation_year=2002" />
<meta name="citation_reference" content="citation_title=Statistical Analysis With Missing Data;citation_author=R. J. Little;citation_author=D. B. Rubin;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Lynch;citation_title=Estimation of nucleotide diversity, disequilibrium coefficients, and mutation rates from high-coverage genome-sequencing projects.;citation_volume=25;citation_year=2008;citation_pmid=18725384;citation_doi=10.1093/molbev/msn185" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Proc. Assoc. Adv. Anim. Breed. Genet.;citation_author=G. Moser;citation_author=M. S. Khatkar;citation_author=H. W. Raadsma;citation_title=Imputation of missing genotypes in high desnity SNP data.;citation_pages=612-615;citation_volume=18;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Nielsen;citation_author=T. Korneliussen;citation_author=A. Albrechtsen;citation_author=Y. Li;citation_author=J. Wang;citation_title=SNP calling, genotype calling, and sample allele frequency estimation from New-Generation Sequencing data.;citation_pages=e37558-e37558;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22911679;citation_doi=10.1371/journal.pone.0037558" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. Paradis;citation_author=J. Claude;citation_author=K. Strimmer;citation_title=APE: Analyses of Phylogenetics and Evolution in R language;citation_pages=289-290;citation_volume=20;citation_year=2004;citation_issue=2;citation_pmid=14734327;citation_doi=10.1093/bioinformatics/btg412" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Peakall;citation_author=P. E. Smouse;citation_title=GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research—an update.;citation_volume=28;citation_year=2012;citation_pmid=22820204;citation_doi=10.1093/bioinformatics/bts460" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. E. Pool;citation_author=I. Hellmann;citation_author=J. D. Jensen;citation_author=R. Nielsen;citation_title=Population genetic inference from genomic sequence variation.;citation_volume=20;citation_year=2010;citation_pmid=20067940;citation_doi=10.1101/gr.079509.108" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=D. R. Robinson;citation_author=L. R. Foulds;citation_title=Comparison of phylogenetic trees.;citation_pages=131-147;citation_volume=53;citation_year=1981;citation_doi=10.1016/0025-5564(81)90043-2" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J.E. Rutkoski;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=M.E. Sorrells;citation_title=Imputation of Unordered Markers and the Impact on Genomic Selection Accuracy;citation_pages=427-439;citation_volume=3;citation_year=2013;citation_issue=3;citation_pmid=23449944;citation_doi=10.1534/g3.112.005363" />
<meta name="citation_reference" content="citation_journal_title=Univ. Kansas Sci. Bull.;citation_author=R. R. Sokal;citation_author=C. D. Michener;citation_title=A statistical method for evaluating systematic relationships.;citation_pages=1409-1438;citation_volume=38;citation_year=1958" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. L. Tardivel;citation_author=G. Légaré;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Stacklies;citation_author=H. Redestig;citation_author=M. Scholz;citation_author=D. Walther;citation_author=J. Selbig;citation_title=pcaMethods—a bioconductor package providing PCA methods for incomplete data.;citation_volume=23;citation_year=2007;citation_pmid=17344241;citation_doi=10.1093/bioinformatics/btm069" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. J. Stekhoven;citation_author=P. Bühlmann;citation_title=MissForest—nonparametric missing value imputation for mixed-type data.;citation_pages=112-118;citation_volume=28;citation_year=2011;citation_pmid=22039212" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=O. Troyanskaya;citation_author=M. Cantor;citation_author=G. Sherlock;citation_author=P. Brown;citation_author=T. Hastie;citation_title=Missing value estimation methods for DNA microarrays;citation_pages=520-525;citation_volume=17;citation_year=2001;citation_issue=6;citation_pmid=11395428;citation_doi=10.1093/bioinformatics/17.6.520" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. van Heerwaarden;citation_author=M. B. Hufford;citation_author=J. Ross-Ibarra;citation_title=Historical genomics of North American maize.;citation_volume=109;citation_year=2012;citation_pmid=22802642;citation_doi=10.1073/pnas.1209275109" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_title=Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.;citation_pages=5423-5435;citation_volume=93;citation_year=2010;citation_issue=11;citation_pmid=20965358;citation_doi=10.3168/jds.2010-3149" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. J. Wiens;citation_author=J. Tiu;citation_title=Highly incomplete taxa can rescue phylogenetic analyses from the negative impacts of limited taxon sampling.;citation_pages=e42925-e42925;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22900065;citation_doi=10.1371/journal.pone.0042925" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. M. Williams;citation_author=X. Ma;citation_author=A. R. Boyko;citation_author=C. D. Bustamante;citation_author=M. F. Oleksiak;citation_title=Integrative analysis of the heat shock response in Aspergillus fumigatus.;citation_pages=32-32;citation_volume=11;citation_year=2010;citation_pmid=20074381;citation_doi=10.1186/1471-2164-11-32" />
<meta name="citation_reference" content="citation_title=Multivariate Analysis;citation_title=Estimation of principal components and related models by iterative least squares;citation_author=H. Wold;citation_pages=391-420;citation_year=1966" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. Zhao;citation_author=M. Wright;citation_author=J. Kimball;citation_author=G. Eizenga;citation_author=A. McClung;citation_title=Genomic diversity and introgression in O. sativa reveal the impact of domestication and breeding on the rice genome.;citation_pages=e10780-e10780;citation_volume=5;citation_year=2010;citation_issue=5;citation_pmid=20520727;citation_doi=10.1371/journal.pone.0010780" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping by sequencing (GBS) recently has emerged as a promising genomic approach for assessing genetic diversity on a genome-wide scale. However, concerns are not lacking about the uniquely large unbalance in GBS genotype data. Although some genotype imputation has been proposed to infer missing observations, little is known about the reliability of a genetic diversity analysis of GBS data, with up to 90% of observations missing. Here we performed an empirical assessment of accuracy in genetic diversity analysis of highly incomplete single nucleotide polymorphism genotypes with imputations. Three large single-nucleotide polymorphism genotype data sets for corn, wheat, and rice were acquired, and missing data with up to 90% of missing observations were randomly generated and then imputed for missing genotypes with three map-independent imputation methods. Estimating heterozygosity and inbreeding coefficient from original, missing, and imputed data revealed variable patterns of bias from assessed levels of missingness and genotype imputation, but the estimation biases were smaller for missing data without genotype imputation. The estimates of genetic differentiation were rather robust up to 90% of missing observations but became substantially biased when missing genotypes were imputed. The estimates of topology accuracy for four representative samples of interested groups generally were reduced with increased levels of missing genotypes. Probabilistic principal component analysis based imputation performed better in terms of topology accuracy than those analyses of missing data without genotype imputation. These findings are not only significant for understanding the reliability of the genetic diversity analysis with respect to large missing data and genotype imputation but also are instructive for performing a proper genetic diversity analysis of highly incomplete GBS or other genotype data." />
<meta name="og-title" property="og:title" content="Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/5/891" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping by sequencing (GBS) recently has emerged as a promising genomic approach for assessing genetic diversity on a genome-wide scale. However, concerns are not lacking about the uniquely large unbalance in GBS genotype data. Although some genotype imputation has been proposed to infer missing observations, little is known about the reliability of a genetic diversity analysis of GBS data, with up to 90% of observations missing. Here we performed an empirical assessment of accuracy in genetic diversity analysis of highly incomplete single nucleotide polymorphism genotypes with imputations. Three large single-nucleotide polymorphism genotype data sets for corn, wheat, and rice were acquired, and missing data with up to 90% of missing observations were randomly generated and then imputed for missing genotypes with three map-independent imputation methods. Estimating heterozygosity and inbreeding coefficient from original, missing, and imputed data revealed variable patterns of bias from assessed levels of missingness and genotype imputation, but the estimation biases were smaller for missing data without genotype imputation. The estimates of genetic differentiation were rather robust up to 90% of missing observations but became substantially biased when missing genotypes were imputed. The estimates of topology accuracy for four representative samples of interested groups generally were reduced with increased levels of missing genotypes. Probabilistic principal component analysis based imputation performed better in terms of topology accuracy than those analyses of missing data without genotype imputation. These findings are not only significant for understanding the reliability of the genetic diversity analysis with respect to large missing data and genotype imputation but also are instructive for performing a proper genetic diversity analysis of highly incomplete GBS or other genotype data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
    <title>Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"BPdmqvmHz2blEarGTPqUbW9hVhasbHk4ZFAexHeFA40","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459862","apath":"\/ggg\/4\/5\/891.atom","pisa":"ggg;4\/5\/891","ac":{"\/ggg\/4\/5\/891.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/5\/891.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/5\/891"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/5\/891"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/5\/891":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hUY4+GnjoxM\\\\\\\/hqFUPk7i59+rjJwVgyfz+8ZVwsIT8QEe1WcEghrtwo\\\\\\\/gbiD\\\\\\\/x2Ox\\\\\\\/fWR39vjN+VwSo+jIX44H3itOyqVwtmA7AXprOIh85vKdMMc8IeqP\\\\\\\/LsAwQDox\\\\\\\/2En9N9ZBHTCqRQOViDtp+S6I1iqnxOXuBgdP+IsII5XO5ZedHBUkEB4Sb8Id1jvpVliJuWAdtBEpPfI0yN\\\\\\\/GhzmC7Q+77xhgJw0KjgywD5e4B2jDFGZ6AB8S2DjBlPrlWfspmZ0BjPfiUh88IlfQZ7DxwylUAbbMIuxA0hljH0wxveSUnaofoGcrl4qcPQeYogAi1WhpbezBDZGNt+w5b0GDFlJpSdLRNF3Zlpjy7D6xZoixEn\\\\\\\/MfOX4lQDFukvr9CZgqIpJbVqQBF9LAQ+laYxZwZ9FcIccJ2vI5g1IIQygi9LxkKAI7BpbvA37Jq9GRBF8zKLkVyh4jxB8Y3eWJDNrUqVpAN\\\\\\\/Jb1k3lipztZAmGXJNUUVyMMSpOqlV+EgFp4y6CI0oqOJBiUHaC4FVbDuj2\\\\\\\/I1\\\\\\\/Jt6pB2xqsSG9ktxCQTplTHdIzlbLb9nnd8Sutljsa+T4bu7tfQkj8sjAkAe7SbQSzWaYtepyzfvuv56pZUd58L+WC4BV84eL2\\\\\\\/qynBjoxIWWmU+KrxLkpxHqbTXM8qLN1To\\\\\\\/\\\\\\\/etJ9LkqzxJq3qdfRqkPsqx1bGo+PWrUAGf2Knhdl0sUw5RS\\\\\\\/MOeePp6FF4to6I7GjHKUmkVnEjx3YKc2HzSZiaarq00gV2GUgUM1jltmNO5+Bw\\\\\\\/ok4aH3EFqQd8S57fcIKDi1sWri73nacoYLqDtEjwXR0f\\\\\\\/acTTKIbjACLgxESX5ZTLiwlR3S5nVUGDQfbWpYD1sC2AIQ+QQ4VEm0bTZqmuFYv4N7Dz5jLk4LeYcZr1GEJAq8RatZNpnDVuNMiiqq\\\\\\\/39gRnwQngIee3FQSWH\\\\\\\/Q82CPvknFJhN3OXx5CgC7uma6P8ofCcrgYX7SHXPwiGyAkzS4BcpzMoqwIB9zzGsfuGq\\\\\\\/4PnCweGmDefLdjZ94grd8B\\\\\\\/HT09F3cgDfxgxfOTET2FSNtYL68S2SJOlZFK8K8EkpOuQPfPXWdvEP\\\\\\\/ft2Gor64gEXIV1+2iEN0mJV7EkH\\\\\\\/bLuPQKXkdMYwmsE3WaJv7HQ5Xlzt+jVXw9zjwOjYJSMmiHZy36Ai0kIKP3tCA6pJRlD\\\\\\\/PLMND4icFR2lTVvuD3wJwtcxWBBh6XURPldO\\\\\\\/pTUE6GVOoqo3sL+TLUAfwmOX2XU0Ez1AiHUxA4VF+IZE0Ti3m\\\u0022,\\\u0022iv\\\u0022:\\\u0022O06bbgS\\\\\\\/zKgyNclz8CIckQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223148812583e2042938346cb73b8198c755369b3875bb22f06ccbbe4b144c9fc1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UDrizAkoyU95XcezOk+P9QOBrfrVSNmSEfWfQyPhbHTxhRk\\\\\\\/MktjT\\\\\\\/O+OGMf6hY16XsfjX9xgGPmjGQcfijNcAAir6hgt4d7s69ufbgS2\\\\\\\/DsaJbs+4CSYD9fXEF2Q9xAvaEHmnRt3aIsxmQjeqB1mrk4XMcONyCG1lK\\\\\\\/2ooZeqYoVByaCu21w60SGLt2f6T\\\\\\\/K15wcAKEkh5mEH7EalT1yDPnUAiC2qF4\\\\\\\/LNzIP+RFqnNG6njOfDmU9F9ysFmAjMIzoA\\\\\\\/86BohAMuH6bIxNh3JRIR1vcr\\\\\\\/CqQ4sEB8V4ES9s9+dvRPotW1TrrY+bjQPh9Q+vy3WgTjkJk74dCI6XPvK28kIqs+g0o9tPku9c4J50spZlt7uP+qZRMw2ZhZdN15ouScGIRBwSFk7yFI7WuhWT3FBVuHNClJh7xd+3Ss+EWkxlZfR4XlLFDdTmZANcMcgUkpb6+K1Spniumr\\\\\\\/IBV9vQeDSXq9OoyKT5VHy\\\\\\\/TKs4mP+w35IKrEhQgkiTYIfzb894\\\\\\\/C9r5g5pkp\\\\\\\/YjdzfChsMCnZI\\\\\\\/r4Rxpv1xRgypV75gQcpr9RVRSGGGTP1gg+ymd2OB+Eo1bbVedEKcngBbwZe7sRLA4gjbr+S7Lx25xQXst\\\\\\\/ZVl9emLlBFJs88892OGMSKh0Aqzo1Ns5VnBdLlk+StprVpzzasq8AnSRUFjrAJzjaj6ox4YdINnbiGzkfhCtWjtFY\\\\\\\/wqGWJTFsM7olgEva\\\\\\\/wNO3ZJv26sW6XapHKVu3q+vY1mQ+EU\\\\\\\/bdfmI1gvobEEdExllYFlEWEJZjirqLYA2yA6\\\\\\\/ltQPy\\\\\\\/g2+9cRhO+1qdZOLl34tFwl1JiGDw0efoCX7MdCaRtbEUWE2Uincx6mirTCokUuQ9suUhRzJQKQ\\\\\\\/\\\\\\\/vvKVTR8uWA2qroNWrA7RTyKFQ13FExgGNz\\\\\\\/nOyBiWgxOdU09VWF5OJ4RTk+gZwuQ\\\\\\\/XJltjiQOeY39rYi8W9OXer5fIK1U6Dj2Cmc5i1kYu8klYzj2UYF2pjK688z5nle+DF5jy5zH994DZwFv9Qo4yDK5stuzpJT6HozGJ8pJIeQCvbZ2WXauTZXgV927eKE\\\\\\\/46fhVI4PgRA\\\\\\\/oy4Ir4hly\\\\\\\/cvWhKa252rRQkvq4Jd4Fqwc4xJV9u7vEZK1IE+nK6alBslGhZ\\\\\\\/hcsple17SQHzDAsCmsSjRt7NqWWwYR6mUeDrXvlz2TfKVo1vbvm2VdMxbMLryeKWSpeWU4T+TjXBGz1poLyYHPLnMyIb6zRhCvPuwzJnwewA3OT285RNxr8GEjoikMgUjnQVfH8nFrly6hDsM1FazI+CHC7Ai5st6B8yw5ccRlKPL\\\\\\\/ZY3LlQXx4VXwaaRPTskuklaf1orp\\\\\\\/A+CGGU8ne3Qzo3FoQKZcAJzu4roFPk4PI3vP1Kz9zxErtgJdsrxRk8bOrgRbdOiURzW8DhwtwrUzvc5XwF+mShAaQQk25bMYH0yXA2btzBeGFAjzfxr6FAwomh8WOswpUBA\\\\\\\/n5Fe2978M1rMggaDIB8aaBfN6NA2QUGWcUc\\\\\\\/3x6Q9mYh0YOSB5uPHbEAoAlu9UWmtFvKTi6ZmqrG0W8Y1p41L5dhLzymb5w894MZB5ACI7hT6k3lyCwm\\\\\\\/kqj3yk1tYrlrIL4V1RX5B5lO+V8p+O\\\\\\\/oX05NuFmPMzpsxents4WXhMRBgbHvJ4V1FEknPJ\\\\\\\/VxiEEMClztYYRMzfZuFXeVYRlWrkA74GOL1p8ACoySPYhngqHRVeMuiHk06+QH0gLxqEMxMqFhQlj88IDxyfFOJcfT4hTcbLCXDN0+8UuYcenv6FFb6OhDAKdosg8zc9qFFMceySna2+\\\\\\\/p6S7JnyIDGOAAtkEmtTVWNzLdk58jXzdJSzFDF\\\\\\\/r7s8DezE1EUko6aaAggxjqFQcXjdqIIodzfiVMhmKzewbTIiVT7gTTEKu9dwt1V4R5bWHqFeOqKQnfgVliYczeYmhD++j\\\\\\\/W6OiZADxhCdnRWIOMZp2+rc8vkKQnCvwZ6JP7snrR6AcJU5L94h4ZMYsxwsWpYosTBdPB5TwdVfrMkttxTteyBTimk1Yll1w582JrotuNkmobzBcIMW437pPNi7HYcvmsXKw8smcYo7RaYxJifopTQOP4kweZcegsVxovKYpZ3wgOwh95rTSXQDQi8GCOw0ghoeh4S\\\\\\\/nW9yg\\\\\\\/csKl3zbbFoDXuaxnMwiPqX2u+B39GvdmOGByDqCCnNZRiOFb+8lqFPqM8p1B3CasR8VgrhZi8Q7DcYcc3kL4JvS+bn0QsQ+UpHhcCvPokJ3CwKPrJgwCQgLkafAHgpIRzh2qHZOvXZ1jIWp75EV5kH7vML5Hzo5sO5u\\\\\\\/aBr+mlprwnkTDL\\\\\\\/+LJ\\\\\\\/DeTurx56fUk1l+S6MEM9g0vXuj4CUKQCohJAb2XWy3ZEZHSkDDEaJXmBlcersZlB358bdu2jAofN17ru1PXh1bCuTtE6taL0np7Cjs\\\\\\\/CbijOF0uVdbXA9kdRQk7yFMUbVoG0HTFGkN6+7EzLYxYdAI+9\\\\\\\/ShxoKVkqPT5bxL\\\\\\\/VnRmV3tV7OCGd5dciGO8DPGtDOrFzfIM9tQ+rqFv\\\\\\\/F1rU2akrZF+wZPz88APgwdDg+aN\\\\\\\/nCb6axLYHc5xLY8oZoVDVtzsQnS0ZfXggV9S8RIrNL\\\\\\\/ceCCdlXdpaNXFpUrfTyX9GbzZj6qLmyHLuy5K0Z0tRt2VYt8bqkc+DVXfXfGz1UcAkY6A0wSZSuyP0eHik\\\\\\\/XWT3eKMWVoiFTp3+BSGJBblQIzejvToo58A9bQvyCvnhKbnBzh8wKpHBAkWYJ0\\\\\\\/hLZtEz12g8zqeaLv02ZkFsP7nB+nA+xYH44XiZ2vzpMfgaHa7b\\\\\\\/xpMIz7lyARQR\\\\\\\/jDkCEowFGUFrC1husJT9kaAy0NPlTuHI5IjJ0m0PqC7lDyVs3W3zBFRMN5+WJ1xvixJ8zRj+KB1z3qPLEqZcVatsk71c095MqNOY+xZ15d86WFXAhPl6eq4uGRPbZKkHM3cObT6UJQboHwtaMsJyp6G8UlLftWp\\\\\\\/GE8jyp0qIZ1IaPFE6\\\\\\\/5LPDxOjfOZYQD8jfqhfCz4ajKpQ8P1bpbxCiDO5SytMNjc+FfnI8BLIE+\\\\\\\/Ld2anBkzVcR7MfCYFKOs8Dk3XZrEJF7tO9G8pEJ75Qg5emoZaBWTSF18j1kOTEdk89cqSgY2\\\\\\\/lqSTWla4xJ5smML6ccAfjpwd4aOjLywX7HhFsDfAcssn+ajMWU2PptfnKxyfxpN2D22bCD6Qf9lZWuFjS7H1TQ+Ny0ROVXhsyLcEPX9qiL6SJn\\\\\\\/BDoxMSXkv8EAHKzoStZuKVH9YH9raLcIQCuvVt2VG4RQbK2HAmVTsqzhAy1Ttnb1beeuO1tbFHAvZ9Xf5Kbc2BardRFWhPm2DJk8xg0ePVYXJrxE6y1VYzinoXSvCuNCYCyT5OrGegDoRZqLe7czSOGF3MKrGvu+Qj1sWKX7jtqjH6aCEx5kTSM3V8g7UiUEj1A+PbIuxAJLZDalk2CIe+f7TBU5OQ1Rnr+YzcdIbjfzkfmMscvzamNKR15ZCWzI\\\\\\\/aYBWU2pJr1QiD6cfMkb2T6vsen5zMB8ALVXeNNSK05bjK61EtW\\\\\\\/sg++5lqdwzpZm3eTwQuGt\\\\\\\/DO4HV8g4Aq+xx7tZrcKEcz9ddT+he2Tr83HF68Ba3p2wUb2h7LadFIoveegD9yTR8YXop\\\\\\\/EBkH8QW\\\\\\\/jmsxPN+dPsCaskSHRMTvTwdpuwOR8D4cA2u7YEOK6xW7Ya8+d+BbOsDFjYXzNmQe2hBcCDmIZEA2\\\\\\\/Nu4W5\\\\\\\/\\\\\\\/EdeRLZQ0GEZWeoLFMGzdfvljdTi5GN5lpY9DdsrOJYZbDUnm8vTjBRh5HZePSPW124PlQMdHpkG6T\\\\\\\/uPgnlOJ8mzbnK57+HnIcTCZ6d24jCiTm6wm27rFWJAhJh+zn9841DsbeSFASkoV52VmdJXOCzfT3Q0kDGQqz5LxN17TXPyHqW8Bi366P+bO2AXtq2aAO8b18foNQBZA1dgvaXb7GsushUJ33J3LXjJGwzF9DDiH9sBg4h200j8Fj\\\\\\\/SHVHbVCe\\\\\\\/Q2lRYWmV3UP8XePA5kwRkPHWOjicivwH4EiAO95Ee8hAjyl956LJb501U7xVHLLBF0mjvRQFMHEzBLMt\\\\\\\/obHjYZodZUgY6PY7aNhsHmWTf3GT9jj9chNYpeFQgPba6NxMp2cQoriGJEVj\\\\\\\/0VH42\\\\\\\/pIJaE4sAHnus+ahFwGSZAIzgD8xB6MyQ\\\\\\\/wxNklQca2yhcQJyNNdekbRfsIgyAkj1jUOJpZaAdEZjq7h\\\\\\\/3UUq3ks0jjUl0qcmGA3fwhGZ+7qLAqd7a\\\\\\\/EEt1Xs2CHCd2n6eC\\\\\\\/LkBaLXbZL\\\\\\\/j48wICVbmAT2T35Fr10cvJBwaLfJ5lXOfXWotyrWPTmPyEmaaTeub\\\\\\\/+ojri2nYZbNlDieGNAfx494mdTD1bS6yjIErYpuLyP06fmDcD+QNlZQ+5ztvpbzb1PJbrtu4Ye5kgscMsgpNnqAXIyZFAvTdYiq2r\\\\\\\/N8NIP26QtdVtLLERiEKX56Ltrul+sq2lnUTk1hODuavMx15M8MNtZPpE8Lm2GzG4K4NNkbdYZaYJ5EsZl2p8OGvWVeypWV1Uek6H2RL45\\\\\\\/WViaqkp6nEzsR5VA+IPy21Y0EM36WYB\\\\\\\/l+wypOI2ZsFZTiZAu1v7\\\\\\\/O90b30feZXrOOiP9ZhQgZq0iiVzr6ASWQTqulhq9CdLRHd0TqvjMFaa8MnV7D3eQB0FgmDVfM7Q+RFf9my63fkz6YJAHu2kljBG+msVrgCn6OsEmxbkZuVL1P82TkwfHwxR8KosL9BvxJVnq679JgdfqWRxRNUkHkxBMNLnArB+Vf416tJsnTQK3RSb8B2wG6EhTEovNriquu6Gw84B+f7zLER\\\\\\\/9Kr73UYPRagwK1Q1QLtOsSmEnFLzqjMJjB82ksZhPS7tBi+EDFwguKj15k\\\\\\\/f8xWAuGo6WZ96mNuy4RJn1NCE69IcjW6vo9Zagv2RY7sgn0pg8sDEtwPvwvKJmuQAXGwVKRh5nmj\\\\\\\/FQOs4UEJ9SL28pToqNozn6h1NG84q2iVKrx\\\\\\\/qusmkIhkPopN+\\\\\\\/HlB89xbcZCoFXgGFILWfK+zIeRYFAKNjKeBgaxwECc8fiijSRuT2HW54pxq5A7kC4TlAMSJmYHMn+EDDgBl0VVXcsbrx4JMP7v1C3Ye7zXfdMgwChieBmbg0CQ\\\\\\\/tfPiw0zY2iQmMHNoKn1g0tgpUwjdRZSzaGILSi9bu+6ZH1OdXhUxr\\\\\\\/DP2kWJUr\\\\\\\/W4NmI5nLDrCN5BXLCAVYcniz\\\\\\\/cjZZKdAzQcM4vNMvBZ5XZrVAgtwPwBFve\\\\\\\/ixE+3bjpch6u5SB+K1HKQJwjxTQYwKfR0aFvyJFvP2vvkjy1iuoAyeuJbG2tSoZd8VuJt0spW+ifm1mhmjc43R5s70zr5b2R8fbz0cM1M8mxpMfM9df2zGHU90RNYWqMSrUA\\\\\\\/mT0IkLpM9647wksGYbgFzcfMUYLh3l8x9Hz4cl6TRxDt7kcuz8XZsEsjHvDvnN++UNix1AAgX2LdbbLAmYIioBnDsG3e81K8iGT6yQYuetxGzq8i92wocOV2HrKO3+aCA260KFaVsPJDXjNRo9xLChlF2X6wGW2ddKI\\\\\\\/KzBbJZHH4iOg5hbKMNWQgss8J0k067Le8h8zaPV4OW+8YOr7PxdUdEPliZUyXS05dbNvQQ29ybwufNoF8btCENu7OJmfEEGHsuiHfx0EXa7DuA9Quv6iaCvJQj2gCe5DdPKxUlR4bLODuAqZzmmNLccB0o4TKb4X\\\\\\\/o8IFRd5LqDuVjv0fSAj\\\\\\\/0GE+OPYptilyJL29BEiyVJgE\\\\\\\/HwVnwMNxAXR+E64xsbBNIA7OfOasxrlJTaAWxAsuy9elBTo5EgbPtv17ftMjnP1BeopS8cmwm058YWr7853oe5b5otBFaF1ufhofBJdbeadFZUfNWiB\\\\\\\/nrC\\\\\\\/JOtACivmx6LtScU1fzh20b06LbQNmL13BOKwxqmMP6KiTcK0RRbMO9ZLxB0W67w5IMQQ8rF6rUIcEAU6loChE3dPVZY6UHN\\\\\\\/EZUqxLpiuBIbVFrHNZT0QxR0Eu0TvCe7v\\\\\\\/W9lodMA69SiyDrrosZYF08Hr5xWzeCJI\\\\\\\/f64jm0eQ9XO5MtV8bf6gS7Q\\\\\\\/RW+RNKrJH9tv2hkVA\\\\\\\/jcHYLlRDmHKV5o8nZWb+dLk7HmEaXTHxAHueAHPgcq3j6yPQ9syMgtb0YNxgPRTthKbaA21g\\\\\\\/8fvIWQwnhdLjjqbAceJHIeQGbcc8RoyCFyANmQzkayhc\\\\\\\/6lwiNfYLLF1e22yfLXNz7uq191vczhK3\\\\\\\/buZjsowqzL4YBUiDY63UlHbHbFeoaTShs5ml4Secs3BXTLbbx9eKJYUcd+ReiV8DC670XEz+TEhB54vf1+UpujdGkvMo2vvLVGpckLl4glM\\\\\\\/sKONTFhBGYtvuYG37ZPzRou81K2TL7TDyM+CXwkCBvUvW33sZ2BOdIK8StBZanGKe4C271r1N1v1b4BI18p5NFoWGr1Fm1umUWgEFMWe5z6UaoP9lFBTyWAjsQXMhPNpSxKmPnQlmfamDhx9i5ddSRPI4v9NGc5wk2Z5GLxkwDU+GWxRv3lZ7rtOzfc89r+nF\\\\\\\/NQBDxRsWCfOAgT9rR12ZWj3IOjYwQOzhcmp5Z3qj\\\\\\\/POzjtL4Qrl0Bq7\\\\\\\/r8nI0x6Tzx44qBxl8ePmO6l5bq+yVk7kSe\\\\\\\/ZWfrZz+XTTHTMWH0ZI79PRShHghMbqTgMpePusF+3fZrsc3l8VM7uRsH+nRdUm\\\\\\\/inXfGAEqQnROqJHcDDiDdCs8ndBLidTGqioZZ8pdfGg5LuMaYM55+ihat8DEYmvgLLMRlvDuEP4hcCh5uEsSp++6NfiBoWyR\\\\\\\/q7r15xzkLGvKv1hsQM2\\\\\\\/dqfZHzsv\\\\\\\/e0Ux4ULlFAPQANK+wUXpVedJh27mU8SoAAG7zlgG+DlYp+4v44KpPlxUtDk2A1k97ygEeZN3cpEfwl\\\\\\\/T3gxEz3Y2ECszFHDjQWbz8W8v+j35Pf4i1bn1zvumURtomz0wdfmn2urlGi\\\\\\\/jjhzktbbbZGa+kayn4BP4BAnldd\\\\\\\/Gqh\\\\\\\/NCh4gKoQVd1jg0V2iODHG5euMJ1sUKkPlLi6Tz8Ertx1M2U6YJvwHlpptv0Ymk20TLIDywOA20RE02nTvrnaa4sbZPoH\\\\\\\/zrMGEnXhV+RgPly1pUcnqs62x7G7QSKLjN\\\\\\\/20hDfsVYA8ZpO5WUcuAYzyLH+KZNcVJhGElO9OG9UgxVQO4Xcb6iMPAWmq9cM\\\\\\\/CT1vtxkSZQdnXKv\\\\\\\/d6lBPBR1+YNST6+lm4eN2Qp2DrySwRdNP52hMkolUCwebw5VCrOa5LaEhfHEF98sb8S3dy85rtsnmmFUHLR5s2XetW4QyzpfqR942GaKcUMnYNTwNu6hshKi\\\\\\\/azbHdrNtQqxyDNvv1EXWh54GLcGHPing89esgbcSfmxHJvTTwen971o+i0GPeaO\\\\\\\/MCEuNgyaB6p4I3ixWlR7SCGrPkpCitkv6B+SVzVjuN1RUHFTHY0kJM6yTuzZnfbrPf0FXpl8qlk+qQ8wm+j7CHVkkVBPkp6zda\\\\\\\/Coo2\\\\\\\/nvCH7QGC2nuCeJaWRVbnFRbqq9C6ozXrJSrtUmCqPM2HZXrCNky9oYWwNsL8VUosCewfjmVGTLfsHQIz\\\\\\\/EJgl\\\\\\\/A2iIjCzEz8ZwpscH521fFFbb7iEiH9Mbg+8d8\\\\\\\/wifGGS+vnDrfkHeVsHLn9ILAG+81aWC1JFxvau6VCoTpbSQZLFHvgOWBclDu+hgKaQCHAt7eAKwCEvsBB8XlL+\\\\\\\/FBS8NzCkx3\\\\\\\/jjn5ozqn2yp9wR7RrFcmzmc71nnHe4DsQNDq1dKnWujTrpw2++gTpvLcxfij6fMtvP0ccBC7cBq9DqvpxXwVzhaaTIGUrx8Oq7pNRc+nktqdlZsuODYiQM+P5kW58vM7TwV00g9CDQE+QIpdbiMf4JXRP4gdY5SQgC6rlb7b270AuOoc5X4tIH\\\\\\\/BOF4uFAUTgcpNv0rpTgOJAVg\\\\\\\/mKm+Yp9NOxUBTw7nC5WFnYGpWaZ5\\\\\\\/b2YCClTblYpfoVd\\\\\\\/PvMhNbZDhRT1\\\\\\\/xQm4+Hk9HFNdj+MlYubr9dJG3py+KylyqXaHCgCBBHcusBurCj+a\\\\\\\/pjZKB+gNj3+0vgdWLPnN+Ijo72MYk+4+XO6LWr2D9G2IxwPPVfsjxSqfCkF6DFUv7qzW2y2sGaoCCtPVzT3d3kL9kDChZRmPpacOcmOMK+bg\\\\\\\/ZyyeXjSLtfyiqhl\\\\\\\/jDKV4b4SEJg3dQxbNl+b3GywEwBRXdw2uILaR80ZLUyRC0jKRWk9OUp+s7gr4Id+fPsLlgQKuxCL1bx6gQXNE72oq7x2F80v6rwbQBMx2oS47fNyRmzkRDv87cV4Kx446ZCsw3rNN5WXl8N+ZrXlYMUemXNLfnxtQtE9eW47Z7jXnvSvN43lKw4ydzh4phJxannjB893uGFEvi8l4p8srkigyoQiZ21jmYe9y6vsmbGWLKwhW2HBGjYrV5ZWqJZgJtqP1gy0lSyyav0JHKj4kG3XB7w+I8aY2437JOkfbWVk2te0HfMb8IebsoCglr\\\\\\\/Za0RJTxiYpcbHrAqlYVj810LxsLOVR09ulj7HiAsqBpa\\\\\\\/YGADZdrBWM3MUXYX8R6hHUifr1CRcXclRlGvcSGdlV4Hd0LDAdvirWY+t8J6MDJglLKmvdXECw6bbS6rOotwdnKo\\\\\\\/J1u2tNjJLHNaSuZBECp0b8qhiNrzd+Ij3f1+lavpWfGO3xfDs7V1NxDmzYE\\\\\\\/zDfBiIn3Pbcu\\\\\\\/S01yjBzYAywTm9mlL9aIY+lM04g5bz0zLV6JKsIYV02h0wui2wwbLCGYENdf06ZVp+tWY1L8+stLjhI6HSpa\\\\\\\/XsFQyHzymmlSPXoipT9UWNc5jmNC3tD+orIQwZaKq3Na3DMC6xELHywckY04T0kIxeTg713Q2vmTbYwoKPfV\\\\\\\/S68uC4mCa8uRQZHBOg5ceZ+z3yc66pj9rIaQDKZqZElCfkfponH2i\\\\\\\/1GS7DWGPFvf7zYna7qCviteRsRnyROm\\\\\\\/R57ZRqACmNwCUgJ0OcgA2bvY42nNG55CuRWR9ei2ofAVP06BE7uk\\\\\\\/3mNjuriOMpzkRPsivlfJA3VoJ4HWhnaqSMXqAgM\\\\\\\/UdOsiPez8xlkGqQo9P93del+4iSd4kiyBsFyht8r4VZVbcnc\\\\\\\/91U9rqaXa5ALqXkg3iE2CwhzCoAMHKXW3eQs4MZ+QnBFdFwZzCLgq9yt0jw6w4UW2gtwBydlaH+pQcjMO7AjoLrXIq7E3AZywCVHU\\\\\\\/dFlghlZtPoEIJ79RMFebl4ICRQUDG13ZXwGPzSkiwIsBDFPx71vpIrFqshoNlDKYl4h9pYBKO6pvp8de2qhTfdzG3MAXb92QJZyz8yK7PM52jUPL09mhGhCHJklnaPwFVSsAeljWs7e1HT2QFRAJW5FY+NNXbivPqHfBQs76wXXhxEN8fo+bsJKoUW6mrwqunqFHg\\\\\\\/G53oxjlRw3eVfzLNb0kvwRQdIlvRFRIIcR32c5JmMIL7\\\\\\\/c8HBAgI4g\\\\\\\/fHhcZr8imSi63Yw3Gqn1vNhs4SnWh+0WozlKcGzRIiPmRtBJ4cHVwAUR4Unmr0NJg54jekDUNlJ+7TWygSKBG9Poj5TfyWzZrjpSvdLWlD3NGbBt6TQkfFT6l8lpVuFBoPJfxk9nYQXe4tfVy3m7JPKcIiitFoRMQ6+o7X7ke1XoFa4kLars3zTwpuojI0LUAOq7dgcnWZ7THj676apFReXN4gkQipW9HG9QzVj8MRmVSU9EAUXApQkYERTV8bKxS4MUU02erkviNTq\\\\\\\/Vsmyr32ls40DN0khkV+O4TcneJjzIQ0ISOcQi1ZOz1GbGl+XT2Kupqwn5WPk9m81YJesiQlL7uIehW+Z1nmtriR13jKyX9bduEHwtYQwArD0d0BZnZLS8Tay9UA0PBsCc0WZK7Upj4DGZ5+IeFXOklhvWtraC0waz24+6qlOzRdP+RpmWsX6qqaoUsvMBYm2XdKJNJ4eXBqD53HNYg7bTI2VwqCeB6jDhGiqe8SZedaGhec2nPnld06pwscrcga2S2v10T8GTo1MiFi0FKoAuI9M9I+QKD6rG9uBe+ALDTrxUHobwwBHoK7JXQQPyVg1At6TMQK9xPOAY2F360OjZe77gYV7CRbWFrnzKaG8W8YaOAXNzVwHNBvGR0sFa+cotRzEtw1PKZ0sVQJgnxeD3PhQ3VUajTrHj+DOXkOpLV01p5775BDukqWJotYR\\\\\\\/t6I5sNectuveTsqrWYmaWBzQ14nuvZRKy9ZRzEnSKNOuxFxwzHM6tg4MLJTpwsa1TQV0nknMfKc02z4+BYyBvDylPZB\\\\\\\/fOvYMz9oZPyVVc5GF1W8xtgDYXGAUF0qv8pIV5dFtJ6PN5h4tZmM+nI5YUvS8EYn7em9GBtFjT411V7fWbtS5VklXvvAhYfejRo3BqwFCkOtvlIaDIxDsqSZ5cfXLY1hbQosBiHPO4UUtn0M\\\\\\\/thY4aMhfM4AtGG3pi\\\\\\\/px7\\\\\\\/GelbTiVJatDCTu\\\\\\\/8Lh0jOqjS8ED6XdZOw+DxVnjNdN4g3JI2pIC1Pc3aqdaxEWQi94\\\\\\\/5avEJHg6Y7FMrj2PlQCCX4rPRiDjJ8rN\\\\\\\/W97T3u8Ih0ssS+eYaVVrPu9RNkW3niZ6ZsjT5kdBpxlDT\\\\\\\/Ctin8ETw+irOvU0SLLfgf8KPznvKU37bwcqPTj0sH5lKIPog9leZmGgq1uOvJegwZ1xojhYVu46jNxCk468K37GpLmBT\\\\\\\/aSE4O+QjwxTkPEUrlV7ra\\\\\\\/YfLkxLhvYNgDj6VK+nRedtcUZ\\\\\\\/V3Pxe7pTknT03qqoBU1HOrInCU2IxkM8SOtzo3BM4Fi9l6BK4l2iyJSW3mZAYmPH8cduJ0U0gxojXMRMinGwAdMoraVsF+W4vibGzj85j7namgtwyBbxjhWL62cAMTNlOP3BeSoyFBCfW5NohjFdWr3Nb8DU0yQlmEPXTQ6W+O+F5xw3eqr4a\\\\\\\/m8n6WnWcoyZaPz79hKnrvu85mwpcfCk6UZoCB1N9CZI\\\\\\\/IEQcRpaBss1YELKkzZia4hrN97\\\\\\\/vJV\\\\\\\/mv1jhTUNftEp\\\\\\\/ayMBrTp7zXnpf7Y+q98z2nr141mJveoLkmWLXrIW\\\\\\\/hEusqD8Gm4lSJ+wtPRxJXSdtGgRbnaHuKRrDt7gl60sa2BYOvOIDxgZx6n3OWSgjjhCEQhCJox1li02YfeiU1+wfxRSLM7ZBfX0X3kzmd4WU8i5h5KVLJKvTfAYpCo2Q2Li0GHvMdrEWnfvd1CRtRMkI\\\\\\\/+v\\\\\\\/vFelXykSMxOYA5p3JzZDTkHtK0hbZcITs0jsnj6cyB16LEPsAD2mrREUvCHyzQs70H9CjKeXt7BS8nnxa5\\\\\\\/AUNIoUHV8QCSLkpthUIggaUEyzcSwHtjl5WJwBhDuFlcxSfGAVAbXsJMRnhfaGjeiZgdgJwbmISIoxULC4umVQ456Yg91wW\\\\\\\/lXGCqkcvYC2EquO8sBCoXG4FuMF1CSZeoylZv3MmYmwXWyWM3d+aAUp4b73bWdHNlRZPqEyey0EKYGAkW+fZQHC1NNpU4neS1NGd7fgxAAMJ5\\\\\\\/l1CA8dRbmooboe+GvpNyoM3DxzZOjYWuo2Jezpv9CWfBqS3pRNXdj9EliskIEOwWd8BFjpqeMgR4SUji8O0o1pz6GZH88qre3TD88W2lEvSL5kG3vfVXWDfTEZsFsTOLf5LgmQkIVK5Cm7R4YtcfdcTDrKOReKtWuk6\\\\\\\/blOwzYF\\\\\\\/sdYYasmZt0nh+L+x5ccmHamxrsrL3LlFlaJC8gA5uzTEMCYHvPLYFcr7NV1W2qLZiQGU79Fz5JbvCZ1BT7TUXDaRbuoUcGAMBNSsVVupXZudrbU3eg2FsFrQ4QceIdd9aByXEBPz+BMFMvg899UYdUFFiKrgUSimQk9mUicoFzYvYP1FQ3THSmAu2rbQ+pq3vVKL4BaPLMRjlLGJ2hIopJPD+ulCD6HsfoT4ZsR17nbIfv3d9zbm1zVlMzJd0fqXDEjW25kinGkbxbN\\\\\\\/vX8rhbCyl+17rAfZszjBDoaG9fHNN0DZI3fgwa8Hbw5rc8XzEpC+i5p6U8CY+hzkPkHCqIZhJlQ3EHtgEp1suWuB5wPgicoYXsLZFtUHS2VHWzkh85TeRCyDZ5+mA1sN3hYEMyNNlBuolffVMNdBo6u4GhJHa5PtuIsHEgPXJF\\\\\\\/UUn6e9QczHJyQQOJ3C5b14Q5AltrOuB2qQxc17eBXjZRSELWeHJ52xW9eLDk5DjV9AZKn7219+xP51DAKD1MupUeMK+RPSCYmqSYl5FKWOMzMQ4\\\\\\\/QsMvNLZwdKyVdwwmAU+870z3NNr4pM9AVe1r5Ldha+11P7s8ilvykAnu7DWG0NTEDs8F9YUWeYw2aTDYDcELyKXqAfxdlEHvdWUY0s+Vvn+zDpQWWR5n7YPAhTZ4l2oRqkMA65DPLJd0q6znFpSN5t1x\\\\\\\/V6rtMTZSmhVOgVeL2eNH8G1WPOvONVNJLSoOekZ8wmz2pWYYtHrif\\\\\\\/wQ1xnx65h+Ze\\\\\\\/owTv5KZN4Gq0qBSxjwdDzev4y+SeHABINFjT+TxF14Wd3P8acKJp52FlcFLKomm6M3RKudg6ppelg1lbJDk9I7qd6JW+TtROu\\\\\\\/yatQR0F4B684gxDKR0jjK2F5kMdehpjWbjIhzVA6zJDECyslGJ8cXjNWGn+zdqsK06tYQVXeUszQsq0Y\\\\\\\/0RJ0D4QVam4YifBRShbXgcLislpstJXqQDbWjvbU4VBD0C2E7zjoY5IBUqeIxDwxYM0QHWUjVvNTTPgBSUl1FiSX8GID6PuwKzkgaOuj74HqBZCIwjFl0LyVoLhxAB18ETBPVMbMhOQ1hohKU+b\\\\\\\/ra\\\\\\\/MVz8NQDUHBQxUO4LVgL2KzzczsBnnHLRQobrYOkGmBQk5e5c1FMkY9sTzZ+rWj2ORzQPvAkc4ynfMynmoWLM0\\\\\\\/28VjVKmWy2Qg62P2FHY58wm6bjAinROPrQvM4ItYe6PWLcbcIqNeerM5FqqsJNjHVOLewLJTEBuHrqQdO6YTRYVIMg+KmmUuypyuMmkdNcN1eo6R456VxgzRv0NLkLYIPwmZdqAR9qH4GL12cdqPWyEVhLlilEb6h41QMqyzJDCY+V2X7jZxjd5U4sLVHo0\\\\\\\/Z5d6snJ6u\\\\\\\/Jhuh5s0fIQa0Vj2Fi\\\\\\\/nSSi0KlWUHgMQQcgpbBF+t6nnSDM1BbGajOD9yYHc8NUh223S6Abf8lIfN\\\\\\\/L5X2H3bMOz+2DvQXp9eIcN5EL3F57YyVrD24xEhOweWLMz\\\\\\\/IVncl0PQMFo7xh98LWd1DthzL69bbWXPZUpEAtnJCpeaGA\\\\\\\/tIhOYo4Fnt9Xmmhn25QWvqxOJ2tVk28+vCko603Upfa7TShUmQmo8WFh\\\\\\\/fr2cZZDAhNrYvj0rCsXdbb60HJxB3ool7htIfCK5BS+TN05LKd0R0Si9yRH+DGJqOaeuRPpD7jTT1B0pjXLFCVv0eTmv96vCvT+Wj8WRyD5r7ii+kTbPGmd7TzgwEU9\\\\\\\/Pr3pd\\\\\\\/2G3vAyvmJpp6X3Adj5WRs50j3NLWqcd5wqY0DnP\\\\\\\/TPU1oWx9DqnJD0pDP\\\\\\\/OqZIQk0BYRWiNekFBLwT0PDhT+fUeKhtTIpUf2bBsPZf8JstPiI4\\\\\\\/yvxg7jkaV8u3pzp\\\\\\\/hhjadYfELwHimJAHSUztIVGLw2L6tVWG59qSlsAJ\\\\\\\/d6lgNxGZ0b\\\\\\\/layGC663qagkOfC9Kxy1wSHLN6DsYMM\\\\\\\/a6MOIv3pk0E4EhzA1EeAjeKkIS68mPUtYQkB9A28WuUnDtug6HFkeCEVaKaL8FbWVn3q\\\\\\\/Ys4bU0Twie3TqJC5B9D68cL5qJRqnGPlH9zSzDHpUu5+SlBKaeNMoecmwR6HBLF1sjnonbdbWtMpiK0FJpT+5AFEVV0lxazCW\\\\\\\/H6peW0\\\\\\\/nEsEmyYGlSUoiy84dekIcOxG\\\\\\\/uEsvISJWT6CTaR3MVNjC\\\\\\\/uQJmtzk1KRTjnEDCaA\\\\\\\/L+yW92SB1nOUTOJfV99wgXIfWwdzbpiXbR7MoT3HPamZK11OniPtgxwMxPYXXAgMktlvjGQUsFO4RocjaW0MFalQk755Y9hV1YOgLL2NufOdHyxx21E1jMGeuLSyqpbnWZxC54OwkX5BCEF6+d33QQrD5Imn0zlnGxPXjqg+WNrhajj8Vj1+YwFKxHmZ49gUSi\\\\\\\/qQLdRQA\\\\\\\/O4MRWBgvWwJhhrnd2zFWWyjTkZ7LYwc45Plp\\\\\\\/uf6dD53b7TCCfVoro27tEV43ms6K+9cd+5lYDWLxPHLqlUnNLHag2DcbT55AQJLc54gQymO0vtS7mO69K5IY3NZ+BvovIpOvUq27QIA\\\\\\\/vH0ClaYnJKKUTFW8TQbza7M7OUB3kEVjZCjdCsTZP8X5yaJqSBfM6K81JQ4rzBFU5xlo7uDSCFahcTPLIA63LZQE1nPU2z4gxQ1GcskxILmVpB2A3PsWAFq83rdlvjvL4byRMieb6NBTT67sAWLGbxmukixCAUltQ2YNUXEdf4fzQ1txc5recAT5AG431g9eQoxREr595xh9ILRW8fjhXr4b1afEp2TaurQPnyd00z85AHt+WsFY8qVh7aBIdNEbMPNXSElFz9v46+K5zrV8AEszmPp0WqRXeG5b1yQWUiD2lNpNeqOEo4mY6JTq9oilDpWUd3JDjQCcqak9eXPav6AXm1bPzSgh9xQbFoX\\\\\\\/lMR07Q0YOXaPf+tnl6jb7a2tGbEv0MyhQ0C2MN\\\\\\\/oRdhLmRrTYLIsKQpkkumPuAcZ4B248v67Ur6i8wmJ0Apc+op1+o6jwTLwCilfK4rhVOMh79FYO21bwTx8ClCQfrDGE34fGA+xIqTr29jxFuWVzkdrULXLWiaWvYAaDK8eDtrMkAzaeMO2ymtLFkrv5neN\\\\\\\/NZtyYvzZjgpm\\\\\\\/oN78eQj4ExfYoGZKh0sL3a0CP2Kx1UiH46UGLCxzxpS+VzK7ZRrqetlEKJV\\\\\\\/BMaw4leURSuM\\\\\\\/aqkcNgEXXpoSNVpW9iV+6WtoxUgvh3ju1q8IRDnRokpq+CvJOjw4C5fAbFUhy5cvQaU7n3rYcpC5fbNkgRuEts2X+NKC37kk5cJe\\\\\\\/H6yZ\\\\\\\/4ABE8GhHb6w8gjWAd3PhbiqL1Ii+O8R890gyzyKgQpiCwGeJYjBIgFN4PiEOcdjPG9pQTF8lEzpGR3BaRNQr8f1nuHn7mlXIqvu8rhaWscvzXBIbwFtsWaa3adIlxEFkn07oIMNkciChH545aEHeNdpJq9a91MeD\\\\\\\/9o7X+5OnQUkj7vEesy+9v0Sn394MP8y9jsPxvgz9bn2qpZiC+Sd7dzpfJhe9SZUHxvNlA7HVp6mH\\\\\\\/07fJtPcguMp4XrQQlLz4nFEPElIr7Tc\\\\\\\/6t6O8OSsCc4\\\\\\\/F8HfcXdHKvTU5AKkgvqGFVawBsd10RcVJiMeKyxSvClZV2MrCMD7MZkypA8YGwJQdfz\\\\\\\/I+mBrOd6JO8KKBpzVNs3Xb6wh69nW2SEpy\\\\\\\/0YBaXqpozDLCXC\\\\\\\/7+B04VbOOgcU4Wh76i1FyxXzXiaRoTQSC\\\\\\\/bdX9R3XdqH2yNcv\\\\\\\/MrLS0fY36pIgxg7ycPmMnDggH5eKSab6yD6a2Fq1zR36FMh4JhWtYIQhGdVUMYWdNkcMZwCfM7ghuwDSw+YiAXRHGkfYj74epJL7jXpcDkNUL5MsM2qpZ8pi2HhyoZYzCQl6n8VyR3Cy6u00Yz7oe0aBQEzXVNxFRg8rSF\\\\\\\/LZhfAcdDLz1oX\\\\\\\/Kdz+p\\\\\\\/hP5iBwZPufBSM3itS3DkKD5CjGoIDvqSoLndpil5mbsTcB1HZUl66iSvUBMuIrn5scZirZQe+YYXE\\\\\\\/3rg0IQeFOJ7xoG6ezwEF5GIOXFJWDI635erpgg\\\\\\\/pfASCVRATIqEhNmGvfwdtJR02SSggDTx7sN0VYhyMLgYBQS8EIsPY6Dm86F+CqCfmH44okrxHrxZdlJOU2jJmnetL9MoRUS2mNhQ0quSVWBqX2w2L7YNSwJ7SJ2mS+MuBEyr8FBTrqkuznv7SG19sT81bcgD0pSEcanLuFovqLSNBiQH6pGCCSbBA3BUvDRIdCNTrPfbxCWOatNCKwqU76fbh\\\\\\\/tI98qXfmuPpEAMIUltxcvCimKO2XESQcdgcNWoQmX4m2I26VoGVIwGf\\\\\\\/voAI5F21CMsc27sWXAha93PTH+BlWAK66TeiQHqLL8e+6f\\\\\\\/KTpQI84PhVh9rLABfsFfx0PIKgMLBARzgFNxbQ4kWOQ\\\\\\\/nYxoADMEqkOGJwo\\\\\\\/TEZjGxlWL8+WAMCbzgpGCjNmd3NXdGyDb9C4O5C7E9c+DwQd\\\\\\\/9xrQmQnKfNawGc\\\\\\\/s73ZO53TGJqlC\\\\\\\/nsKV89Eyf3O0ign2RUZJzeLkau+eiN7l8u1wmRgDp8\\\\\\\/TbLXI5qYgZYRXGDHct5NC0nu4IC\\\\\\\/O9UMrKhE4uYX\\\\\\\/BqRz8S7G3ErkdoYQG12AZ6F5koULCo8J7oOr9ihZXpQFQS\\\\\\\/1JCH2DDG2fVRHo6iijh+TYWPkkg1\\\\\\\/Aoa5ClyZkOU7CDNVbkJ8CbdEPVaJvXH76AFN6njtxdwJkHUeTvj\\\\\\\/3pQX7zHp2fH1d+j2QKGQ1btDRHuHctMjBL28ipg\\\\\\\/yWLuCN40zmmzhD85WbDiQm\\\\\\\/keGnK+pq8wRqeXD638K7T9dbwNJ5CLlHTBahngI+IWDZcYWaQ++PPIcw9jW96OXMjzfys4R972u9LKF8LAG\\\\\\\/sni4t+7RtyQIXygaucff8IQmtTTfrnwzrzyWnyTsNpVaw0+wr2uvYvvEm0ojxRkjc207eNyCL5WGRZF6HKAlfxWRUOqxPyko80MHBJMBeAkL92YOGug0CQua+gsPn6zAQ9EigAlYHQdZiLzSjYGDiJObrwGBI9UTJxToYaonoGxWDxThLWsgnBS1khWue38BimFSjewPwCz\\\\\\\/\\\\\\\/sTaLQkllLyuXOC+zKneUwTRd9TYvGoa1HTPrHHSWf\\\\\\\/zFLVt8fPXGhJhh6HF8whEJPIF6j6xstZLnOXGIwmTTu+SUUi9BnO9ziXrSG5t5BFsN5xwDK+izBucQet\\\\\\\/bIcDgsqLpDKttu30p0QkBXPJx6NBO\\\\\\\/YrnROuU7hKdksGqez0OGDrKIk3qCgxKzJUXUmI96u\\\\\\\/GTNlmjwSK6m3mqQJU5HRjXg2PJiwVPVQJRQesqf9l0wSo6wZFu5ISNEC6fuiXgVzFg5gu0hkiHRUHyVX4ggnEOSkdfAT7WdFHVEgJuiwS0csVhpZ2FZ6HOAxdT1Xs4\\\\\\\/5\\\\\\\/kmBu6zZ3Im7sxwmXLOI3nLI6affe5HqskuJzhmE7IIWJa9tch1aD76bYbyekhKfVym1+CC1dtq+YSB8QKMXE48yRFDePQPugVtlH5NGZgw1BVBvjmkZ9oLfug4+C5haxGjv9WQ92a8BGrMZCc7sY8aLPiNIQg1Ca\\\\\\\/u0ISa2YMyVTevu8s7vI7avhYvdBw5qbpKsJFLKAh9B4iSqpdduM2TUD2DrHrDSG85YUU7kfjwX08MP+nXgHnauTgHpzLsAy6Y2dp6Hi+f9QtSyOUI+wNuxfIKwwDv12xF0gawhKB3oMirLalsKtf8k2WXHnYKmgOZ0OriRui4j8e1D2MmW+zDXw\\\\\\\/wmaNAOpn2ZFZBMBEzPoLrHQbGVqZI96JzrBI7CYyS0i2jc6gyXx1kJDHJIaYeS9BQ\\\\\\\/\\\\\\\/CavbB1zKnFeNL3gLvn\\\\\\\/oRI4Qu2UZeyaaeTDM4jE7xBUODjYbm1EKQkpcYrORqZkoIzVl\\\\\\\/aHKYO7HdnEPUVtue432ZxZowUt8LN8geiu\\\\\\\/QHePObOD0Db+6yH8doGf78Z73w0bvTcstJazSqnGWwYpgaLe2v\\\\\\\/rGjcFk1QBc2gbpi1hJmeGg8i6VZcguzbp2Bv+8u1JKYNeqS3Bdae+7QvYBuRBEw\\\\\\\/IBZM4IVhnsbNd\\\\\\\/TbWJbSA7Yzl6Yy6OAt30bLMUIdI\\\\\\\/kZ3RI4xA0kPP93\\\\\\\/xe96h1LdW44cdhYOGdiBJv7Vyni7+Og9LsRunACU5T\\\\\\\/80dsSk2APzENk2Uv85bcJiK3odBVD0C1s9K6oHpjJuLNmOQWntteDR3zsRhmbOJpO8UBQ+BPUjsvTOZ3E7MjzcoRhf0NgK\\\\\\\/btnPkqwUEKfS83Zoig\\\\\\\/jqjeBwzJjWacKbrwrzgVF0\\\\\\\/CuxmTYHIryfaONMrjpYk1wTCuVkz0+nSzwWJ+bPSe6z91xCJg2AswZiQVUxdyrRR7TQKmZfdVs6M5XaJ8fadzcAM6mXRkDa+f7bIxD9mnEPji69eeYJ\\\\\\\/J9v5Lj3NkY7Va4vuNTVck1XfjM2Ze\\\\\\\/p4zs2lKqqOHOV2Arx0m7AuV\\\\\\\/BP1y1dH7SjW1x5rs0KhsX5Gnikvfvp9W7CG+4wE4oZd0VVT54czk4RUoXVvyzA1Nm\\\\\\\/RrH6ck6yZ1X4WcjdDfzsJOK7Wjch9r9NkkaOWQ039SucaiTn6L1++B+SjneASVNnMKaE1J+SinFa+f9zYTfNj9UUw3+OYqF342MROcUAo2JvG1PSOZ8LYK49+fAEHhL79GFY6mpbVb\\\\\\\/yQCMatbfjM4HPWyvNIKmRmovXByMvwQOGjylUJGNg7Gj1Q7I3W1ZGcLrUWBwwq98OzeoL0Dzc8Sv\\\\\\\/e39nVRlZee5yAomouFKZU5xq+c8iHUQark\\\\\\\/0kTy2jxPSNJKSaUO5NWIz1Rqf76Yur04DMhc6TnkV29dIrzs34VRZmRZ6DkrlTgAukIeNv\\\\\\\/1djB+MoRJTPKMRXMqzkSCA4ywq+v70agXe+b\\\\\\\/9N4roFXkwfOJowGqvLZbOEgvL6edTpHgWosoLFW26ihTxqSHsu5kUrffy5i4oCNEPUTb9QOEx58Kmc5CZVpBSk\\\\\\\/yeuZO00VPMfGiYoIJSGTl2PM+CSvPaAc+Z+Iu\\\\\\\/DKgebELHM\\\\\\\/ZtpqDY8teyTiTBmBPo3De60CtM8LB3l\\\\\\\/lG7Ots3YOeu2jUxAnPqEn\\\\\\\/l0xZlHViQl63\\\\\\\/67pwywbAU+cJGGZVjnCEtbcrARQgMgaxCiYOGY4tHeuwmvibDusw3u4SUmqtaBq1ZyT6FMySesnMp4VlgDA3MLrhudDNXjE0xtDPg+M7GQeSQRXqrc40Z+LJ5I0jxxHumj6tpYuU5NxxMyrfV8gxB3tSG6EqM8V44u3zn5ya4F2WtbhnyI1j5bzjR33jAcd5rcKcxCXQRvG6AbzPLdvQkbmWA4PzGO+H+N4FO0H2YtVRug1OzcS6\\\\\\\/lbiDKlefMMHYlwI\\\\\\\/mGu+dooYicqbqE4+hiIQ7lVMrYRbl8MEeauuG7xpYWOqw3EC5EKCxsKvSs14TdYhlwODFNBkp\\\\\\\/89RoH\\\\\\\/GYX65BmKj+Aa+ppj3Vkv6wYkU8PNBGqKhzkGPSmlIgpV8H2+Yl6mMbx\\\\\\\/3SVdJSw56a+41DQZE0xMXjXEf\\\\\\\/\\\\\\\/L\\\\\\\/u2ixx1FmWXsgFVhmJi\\\\\\\/OEFFg12jKot0vogTjSI55wEIFBgB7X+mjiCl8l6y2iUK8\\\\\\\/aZZEWNI8t8zStoBuwKKnq8esmTRxKxqhZXMchXvJ45TJgnwiKdF61MQ1Blh8qhNtxUH\\\\\\\/ZLF2wRIg2TwV1i7IJACPLtwmeSwn3eFZY6hpP9BwP1I1lvkFXNOcjenjawcIXCC1Q9BUTCskUR6MRiK+PCoQHe2BiFrLBMXW8GUA2CUBLfx16z2K91Y6Y0\\\\\\\/ZGulPDg8u\\\\\\\/WQVJXvhMDZziYPACyQSs+ekthEzNXwPEZKuhcDVsTfQhh+p6b3nD6m7gZ0peKhZZb856+u8TycOyQndQA2U\\\\\\\/zPphj\\\\\\\/Rj8JOrbfW+VEAGY44+87boJr9GaAKE0PZE4ZT4K3BSkYtYOvAkofGKiuMOrAKookbqTr0Fmd6eGMUswMNCIdzSktpSuyxc6uzjzVwMR0BQHnd+u\\\\\\\/J\\\\\\\/Ljyv3Wo5Y2DzETz1J3rPsmyOpyJGaSRmxwvMDsosF6gN0ke5eRavygDjGUNcwXcZHoDwr80H66\\\\\\\/5\\\\\\\/N2oePszpuRKJVkGobEtsJPpxqJ3fLigVwM8\\\\\\\/3xqH3\\\\\\\/HR3ryad7sbEQzaVnCpeWZnPWSboNwbWMJIjPW6AE6ngU0bQDXCOuqCWlBsEYIaCcJOK4PolqqefEfk\\\\\\\/+pf\\\\\\\/liFN1t4MOO+\\\\\\\/REBeJTyZ7bplEapxuVY1F1ApDP2KVGGJl6ELb5lIyjw88jOf9DQtG\\\\\\\/yog\\\\\\\/5y6sCU1VczP498hp8HpjUIlirdBi5aXCPNFVjFTNvcaZRNh5Rbpti6mudYjZ1K8wClWFwzhCWxpG\\\\\\\/ArLfKNZQ7a+9Dmhr62YjNR93R38wb17PjnLa8PIIeT9s7cNo55gTZXBLL\\\\\\\/ZX9LbFvqwI7apBNKG8TzPwg4YcQHkil6TaklqHyQR0d38jBHmeh\\\\\\\/N+Uz6uxT3pCbXFo\\\\\\\/SRo0kUsxMjEVJyXS04\\\\\\\/3qAhNLEnItUYpfRvVDGC1z5khFMgtTAoosa4Hecdj4wf1Ew+iz\\\\\\\/yvVTPkM2UNSdEukKJnYWf7okzQ2J3bsyDKplJ\\\\\\\/qmJ7aDlhkgsscLouJdi4BiEgBrQUGWBmfWqhBGkB9rN367OKipPVLBdjU+N8ZsfYzmOZi\\\\\\\/HiOVvAZgbQYHJK04ED1R9XypT+JVxtVaq8ltkldCaTSsqjotCFW7s8h+XNUD55XJm20UtiRaOyg444Y2X4uQICgELF3GH9IFiGk8GDj9SwVmixbKy3hFULPWdc\\\\\\\/dW6FTwUdQr9w5lrOIx4JxOTHvm5a8zr53spDsGU3fWaSVfBWTIFtSSAFKHj\\\\\\\/oJkrM6eGiLC4\\\\\\\/rQaa7z+bwQM98M3GZXsEZFvmUcFop0V4GBtRujBF2+o\\\\\\\/wxszbS7VViVMbN1mZso1M1FvastMKlk\\\\\\\/riqPUGJhuF3nwg3k3+3xlxmXryepLtE0VOaKkyccGKm2jH2Bg4E\\\\\\\/ocNhMvaN7cjWtjtjPCSMTtGfOER2Qo2T0tRgS3QHdVlZS4mx\\\\\\\/iokpH4jifMSlLhP\\\\\\\/BpTFoOaRcciPEGyeANLZPJHbJkgdDbmlF5MUUc0DRZubvjUEbxYcKs5fZcLXeMfDJTXNIt1Q\\\\\\\/epBXOAQ3B4vM+KkMyWUXNvpIp38zjJ3tSHY7FKGaNKisvjlCfH\\\\\\\/uoEHfc2Qw1kfSfCNt4vPzRv\\\\\\\/H+cFRNyQ9yEqq+eCnxY1MDA+YFIiWXj3xvy7Ep3lmFFG4RSbw4Xqb9pA8PCLK+cnkfm4\\\\\\\/\\\\\\\/0GpetgDcJrvjFlvWCY\\\\\\\/jnP43BZw3BkEAy21v6QWalCvM64zVKqzpNLDYkj8Ualm5uBwbOCWgb+vYj+5zczrXn3YP7yeuNMMCqWz\\\\\\\/FBNFofw4p8ktnl5yxZIIYDKuvQTnMz8aFmaB149tpkBJmNz0iAOXhiVt1hnHfVrNpAVcBa9hP4XdfWx4krH\\\\\\\/QE8NHu2oH3mbaGZL05Uggsr7XVSL8EhUNnIsyDTtpUfot7ZXxF9A\\\\\\\/Flo1nTOHHe8yqnxcVw47odmbvknzVO0eqz5wrnx\\\\\\\/YZWKObEtQNeqeN6uwJf3xXguiLfD\\\\\\\/dBBf+UP0bdiBq8PWG8+0IyMDzHPV38xFl1rsKIAA2\\\\\\\/vUorIlNBhWe+HY2A7b5nDbuzrV4lg948NH4HqtiF9ytxtsZSm8LvyGQgyDOFXGUxjK5PdGUNpGJOoy7kAlf4AG2mzrEGtuQgURGO+os4E+Ee81wHSkRtpclw9+wxqvK7\\\\\\\/wz4fN1qhwvdpFEIkm8VI7\\\\\\\/0MRVwSrf3UmRZR7ZBkG+yYdf\\\\\\\/C0ij9IpLedr8QyPSNgNBK\\\\\\\/NlJj5495S1977Ux0U3DHgr6Xes6P6oNkp6u68PNWvkimIKO8Aj38O5P6xnOFvX5MBY\\\\\\\/2pisDDkKlD5bRLni1c5\\\\\\\/4vYePTDV6\\\\\\\/GBFdZ3rbg9ymE9UZd\\\\\\\/dcoAEq3OF\\\\\\\/SfQC0Q7DFOX+7JO0tzNQEy0KbhsYuOw5rs\\\\\\\/XD3icIPQ0KThE4q8LIarqtdZdmfyD4T8mA4IO8EtmguZnAMXhYbPuxzea++ugn2ciaeBPzTTGHAjSLHqGwKhnQFg285DArpVdAnnaztUlBhsfcUso5\\\\\\\/9LPZQf\\\\\\\/RGWU1tj\\\\\\\/nnFM7vmbC25zaN7DnhRpORjPwBK\\\\\\\/nvJLbxwmXEfUjVnvnff2W2Ltfk6uTKu2kS78KPnFLfUS0DijwUasmfV1ez5hMshLaDg4xr1PH68PIHaFhpub\\\\\\\/HE\\\\\\\/k26WIHgPKsPi\\\\\\\/XDA84Fqyh+hOmw2MtjDnFsik\\\\\\\/DQp3sCMQBZk\\\\\\\/A\\\\\\\/+1ttDyKdHdBpca4AOXRj8z0QnVuc3d7arDZ0Zh70SEMNd4dA4bVbhnAy07i575qN\\\\\\\/zqorbWCL+4CF1EhARz3q7u04ebx+2uxPuhzE811Ti8uNjxFdbGQ4aFXA9i4PoK4MqjUxTzpzm3cctQfa8mq0p0\\\\\\\/iLnvGpTDJlI5QeK34B36tweDtAcZUtXiWsvszhiNY8L8ewv94NFom3SiGLkqVLPMNFs8eP\\\\\\\/KJTi2NvduFh24Y9cNfjJ\\\\\\\/1nUu37\\\\\\\/okzA6++Wsj2skMUPrUWfpIkqWfPRDcWBneJV0QjUB2IxSYYz9kjKypLDquoIEk9X846+0wFT\\\\\\\/RKkLu5PTInhfT0K8W7wkY+JHHofzddiW6njwNMHhb+XSO2X8wi7lglcesTPuTu4CK\\\\\\\/RbQ17zD8pV+4LrINyga7K0eDbxuonmPzz0lUKNu6LHrloR3fbccFwDVTuuQncCpkRr9b1S9698E3qfg7UrBp1ctrNqbUmBf5fjJIibd\\\\\\\/4mbaa9obd61MoQo0oMTiD9F1jN3vMjoUmI+4S1ItcG1Bplwpo+LHrz+bC88e2sShtTtlez5eUdprv2kcRm1xMTV9SdYUgu9ccRqJ+JUTXiTeq81aqZA7EO\\\\\\\/cpYLjXpye1RRTC5IlDmjZvfhwM7Iua0xzOlWnmsLDFDKh0bPYr1MJywpat4mnScSmlFehmwwKnKcsQWy2F7gs1wc6juf\\\\\\\/esN16CPbw\\\\\\\/lDkO4jx7Qth2HMT\\\\\\\/fOScCsQuZM6ME810QrVacw\\\\\\\/i\\\\\\\/iB123jffmkQTs\\\\\\\/0WVnl6goWoMOaTz2V9PBVjFLimyMX0sUWUcyC8gslozT00CPMdVi3f2B6dQHjy\\\\\\\/Yuc2MRGKZ8CoQOwePArnbRW9f81Ue8sqh05WR3lYxa28zdYlbUkN3cqVogaiVLOmvKgdfTIIbtFooryRgzDRvB\\\\\\\/byIcHBwVNSTT04fD8vynCbDpvzZDxaFA6MQ1sataDkFlhrgraTK0PmBANgmBdT+RB5ZTQV5EsZYq9e0F\\\\\\\/Uci2IT6rGKSrUGWBkWiCW7KtKlgVQrTH6XAVTOpaBaQ7+OFtjyFKXtJiwxIL3RnIJ\\\\\\\/5z\\\\\\\/Z7d56FwjPqSQJKpYKoHs\\\\\\\/Lu5z22VeHhtuAStRRLIRs2ZmZCAvI3B9sQS9EOcN9qj6+dFBc0Ul2wD0ImtqP61zntFrF7R98GAfwAbDHdYO7dye4UZ0OzZb5ycMcKVmcjJDEShXRCzTnLwUKCAg6mMbwTUOrhebE\\\\\\\/UI1uOKziK\\\\\\\/JyAoMyZ6oltDrz39h6Pnth57SJemG7uZc6OyKuhy\\\\\\\/ly4fSzvioA4ttbRtuXyQRZr8ahhr0XZpFnPvErktWmt4sD984\\\\\\\/VX0KfqhLWoiQEduXdJeRRJm0WCdS1YcS1K2cE+wC8wARYtDft0l79BMwDeg1u8cIczy9HJMp4LOWGw73UwI786TW\\\\\\\/z05fu+CtoSOMviTvIePFAcUjGEzdj+UJ\\\\\\\/zpoFOwYxamsnoZyXMMEUHQRtsI5Swf+Dbuy\\\\\\\/BJB4grlYcuXi\\\\\\\/VGhdpgLXalciZBZOKCqBHRujC4pFiT+922YNUMmFZ4mdY3OKGT2H3WBGojgzdKp\\\\\\\/bWp\\\\\\\/Rw95P+z\\\\\\\/iyD64X5stcZG4bWAJ7mozdprPkTK23lv1XY8ChtxvNRsBHqPO8Zk3oWvGQ5WbPCa6ZDwAAkjqY4o+kGEpJD+wml9dA39wTuu+b0vwKzlsGA7psjVBI58sefb1JS85OP\\\\\\\/w6v+7sA5eQyq93tpenpuNO8jOeR2TErWslQJe1ZqXKnfuJQ435s8PcYQZWr9xUi7apZ3fFX4VF+glgJZzwAhJhtRO4SB3BfVCY\\\\\\\/S9azsaKsB7fU2hLRVy5Tdk7YgIwUCMORp56JceZDFpJj4HU7K\\\\\\\/syEObvQDCH+qSsmiAaNrA73n\\\\\\\/hH3YP7JExu+Qjj5TXnjxu9LCc9J3WJ58kb29RYc8KmRvX0pNMNEVVfadfDEFruzZRUPUqsuqZ8ZIsEgFM\\\\\\\/\\\\\\\/TUA+3\\\\\\\/ap8OlZwGYO3ktIrHrYW867UInjiIACNbLyQOJmBxSPxqN0Uk5MZgUM+mk9+Eoln9PMgBQNWBDu10vOcf\\\\\\\/BP7y+ERtgFXWrRKcUohAQYasEiVrjxk9K5eFKqCYtj1uKUzSXmoLPPFXgwXdqZtS7L14hS\\\\\\\/KQJ4\\\\\\\/o6V+XP1v0226x98JrvPqcchuvDIbbGpUS0zLhTkr5HK\\\\\\\/BhuNAtT30vYWXoCbKO3usA92UpeqkTAzkVFbqKMOuhZFiwgsm3cRgRT3Mr3q1nHhg5D9Ee6Ceq4P\\\\\\\/PTvRBPjJbRf3tX3QzZMPe3NofjXJNdJpFxVmBL1K7rqfq1lcXNi4Za\\\\\\\/qnDht56Nok4YZhDJ1UI4i12MaXJcdvJj\\\\\\\/7StgrYMr\\\\\\\/VguPCZJeN\\\\\\\/Z\\\\\\\/n3IywUSw4mNyC14zEncQMlR\\\\\\\/i\\\\\\\/auJWljKLaxKFoq\\\\\\\/BPwRyI0qHleIQNBV4bbN\\\\\\\/2Ll0XDZAGP4ASg61yocCIjMNMZhpJ9GARU3lGD19nQmAkVkxqkKECvu+eluhwloMcCDVfeO1lcCfta8BQGa5Yc72J5jW\\\\\\\/PgCQI0YsHPNFFn5k4OYXUVSiy06AZT9iPRDRoySVW4KEjCBctUGmFSDFcmkceJoCo3DFO1O+pBvVE13XN5sBSpTBzIVqqw+3Buoun+f7q1cIpi6GEDrAP7oXZPkEpz6pHwcpbakyskc8svig4iZNuaocSgTn7D1gmrSC4pIIY4Nmb+F6LpgV\\\\\\\/T0GtqX7D5oN+B\\\\\\\/8rDzf2G1aYOe9yX5li5J01OcoqXxR707V3WEQdIE4OMMun7J+oN7E8Q8E0kFt0LeB8ieuqqfdrbwVrvxI2Q0Q9HnCqJKz2wv2n78OdNgI5ZFIYxMdgcNJvQRPeKcAyFk80m76OXtH1AKq8AXzOA4hiWPi2D6XziNRZvU4KEea75DMKa1KMye0wQbUTaSHiZSzuBUxyROLMqmTvanGBvClemxKcwNUs5pcR+i2ZTH57DWObblRL2FIPFRr5xWCoUR\\\\\\\/vlnyvT3D1Q07BsYVTo6K9tPSffgEavVdqDwmUoe9BqW6kmPbzylU\\\\\\\/zyjrn0aLBGEjDpDPPYZaso7GkIPKvqzIQxtj3icmD7R9cq+uQR1GyVaFh6TSq3SlwyaA1K\\\\\\\/B4Ec3OiHZwEs709gt5ZVW4mi16JyALxPCCk9ZexNFnWFoGlKgOk9\\\\\\\/MSAejZpqKU1clPG5qOe0kdS+thVnSOV0eyC3Wr6FOgcOGNACULcjCRSq583ZWX2NGHR59H4yCT5Z0srrIqM6rN0OosbE+bmdVQTdLY0Jty\\\\\\\/Po1v\\\\\\\/vCGrMA4Ln8avUussTxciphmRb3RTz4kIxA6+1fcDBXu9FjyrQCE36e\\\\\\\/1Wfbho+\\\\\\\/NfFqp+i1dyV2TQW+psk\\\\\\\/2VJWQVqh5auZ1pQ1u7OSyR1WntmIrLXyBsHLu\\\\\\\/to5VeSab5HOboZs8TtcT84\\\\\\\/CSYRby20ohYEzAmR2I9qRFjATrW2rcMPSGZYtAckScoKE1GmTQneya\\\\\\\/kiUqVfnJpg3Oce6bM82+qSOb+YDQKKRWVLj3R4jFUkAAAQmpt\\\\\\\/Rjrk+BlkB3PZB5mVKbWcjKCe1m+BmQg044eEVRw5qSofrJUFSkk2CT4eiihitCb8LklYzwPPn7A7vywrISnEWyq5o51WWe8tPzZeIITBhOTUiDlJwNKGD3tpxSLu4EjouWBA2L23g\\\\\\\/0KuupkhZ+lsMc86DbO5IJFDIFrqJibS59zlIlbOD2QmQY1C0FHVuw7LT1ma7gNDMn\\\\\\\/tEBtYNtekeRwLgWAWHdHLIjSiGk3wfkOrYpSh9\\\\\\\/0cCWb1dgSd1vx4cYZ7XE446F8gd+Ndh8KSdJwy6uOhV+U1BB99DKIbjvOGd\\\\\\\/kQwVxuX3\\\\\\\/eagXMmjvk0b1S+Gl4ZTboooj6IomKBtba9\\\\\\\/6W1HNMOUzrJuaGLyTIsR0j7TlHnnnVxGNCvdUg0SHiQocynKPEzpXmWfym4MNO0naXAsjszDerOd4HgfdU7XKcCgDi6WUPbnAcGuAizvi0fWZgg4IqgLyO0JP39kOc9qItEs47BmI1lKMeKexrASmyEBf8GyeVaLolMiBph6Ik+k1vBN2lq5xsoqcRlxKlY+oBLoc5GKNDMDIhb8N3Ubh+F3TkEdmrC+gGsytlt8gtCv7Eks+bWUf9MkH+29E10+5WWXmdWiYzulhh12ItdEizkBjSxITZpk8IsQz3Ai8\\\\\\\/Hq2uRPFEsCl9\\\\\\\/MJxW4GBhP3qJaLkd76pOx8aWPQyCL5bZfsRkA8H\\\\\\\/Zhk40Hje1lFEBIUKDKnDUFuOr1Gy7C\\\\\\\/fXt4jAxyXi6fKtDJwo63z71YQHG4cnMtG8O1C3PL+zSpjWGAJlRhFfbI7DF3x8\\\\\\\/nZnRkUM7Jw3WHHbuXMLpSj5CYbw3+TeXnAdfReG\\\\\\\/Z6d\\\\\\\/P86iaX7dtDb4zQzqzy60skOYc2fJzE0drTwXsqxeI7Wjr03EGlMVs6ZhHWZTqpIkUPc0Up4vFtHC26uiDXeO5ufbl6uBTTsUMl4\\\\\\\/dR6sJ2QdjVhHHNtHDxh9EP3fzjiQziuNjgEzEKfUiuomsMqy2CkseixEMQxJHOe+PLEoVMOTvUedrUa8qUqg2kOeTrqlB7M6scWAvNZD0jRtxdigdGBKLXvZGLRPiVaFuHxgyqOXKdMhUK0tQ2JYXaJV5e3ZOGqZA0Vmbkzp56EK5e76IUstb8aqkxvL83wFRP7yzzdeG60NiswgDurwJJ2JcBulQ2ZIIcReSIcxKi8mGWCptqB\\\\\\\/yX0lGmweGn1gfFSwfkWTZOXY1KIM+Fe6PAJaPh\\\\\\\/0QyrME4g2hPrffBJdM5CwXYr+9HxpwXo0IkDRszJzP1+Sq8QepkkN8tNf3Q0oYooQIEaO2GTCragFt9vTMTGh1+aOA3FmY00jfZDPY57qVVDdsdB6WM6GshDMyKCf0rMTwlkcqdeAasbv7xeR9JXoQbx7oL6twl22FOFRqsRasQzembloiktRNsc5PU6Bk3XG3WDqnvBGFjj2Cja7RtYSk9RhrfaCuoixZA6YKM6CR95me9Yi9BSb5q63xcJhhUAY4x82PKl9\\\\\\\/AELaiOQucZR8amDVDCyx8WnFe2iMVfzfQebDo9mmilZ50TFLGhMZK1oGu6SprbEe7Op4pYZjoTkIpnMKETu5sKg7xzz4bUHFhxcn03Ko84qGiXwfFE4VXtG7MnhWQnycJAaVT2b08mBS4NLp7nmBNUHHCx2C8j3Z4FMG44pANqhrJ4aw6YEqjOsj4kMm1PM53vFOsHtpuOe0Ld7YvC4Z6ocbG7WCeojB1KwYA5nfZHyH1XxV6VgxDijFExHlbUrjLCg+Vs6I0lnridfXzQDW\\\\\\\/ZbfQXHelmvPUhb9SPJBA1Lm+UMkztPiPljK5+VpDPpuEzm0zIHbQ4z\\\\\\\/xELimRw5OEgdxGfBSbgQbKjvtwIcg15cu80W0ltvn5sV771hc6jZ5O5tPmFM7G7Ox2ZJG3vgqUw4a8QPKbOrPd+H+5BXc0zMBUPFa+1fL+h5agBWsuvjjUwBaPcoaKyPUaKsyl92b8xYDr8wD1m908TQJpv5\\\\\\\/8TmCpkpflvLxbv1O1QixNJBBuS+TbyJLCdOXCTrU3H2iczSY3ag2UD2Qgi9HfLYUHhPA\\\\\\\/\\\\\\\/dHpxcUgbyBh1DoHsNsdEbXi0YysAJo+hHhNamrewQSMX\\\\\\\/MzU8VnzipVOgim4A73xNGcMZ0d8GNA+i6eDUCo7P98zQ8bQ9aQ91FBunaCXf5Rj356W4qdSc2o2e0MEVSnZVD3dU\\\\\\\/fCZu5BGv5AHTV2hMN4NAV5Bp8v94MgymLbr5Rm8TRCCcFfEQXIDDH134uJNzs28PxPbrJzgJOdm35DY57XWK0HCTZDJDQx\\\\\\\/4gkBp9Jx+jVJCNnwXh2yzt2rO6mcIG7kmWv8+NchxejFEjuOft4InoCFPkk3vmALpTkmdaPZik6a1nKyxpyY9whi5hsz0OsS0gyM3F4DggmKZtWi3RWkzpSJujIHyssmOYgIHsT5uIssc0MTVp3RXEAwwJ0J+Mzn0hRjP0NKCdf6\\\\\\\/NtacoUkBJnR6xFiajhgbxQ9cY0iUR1bKQeuh\\\\\\\/+Rt5Yyj6d\\\\\\\/+4sypfzGDD5rFVeVLMh8tDRuXGwyC2jY5oUrVgohKbw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tKf8KLFc6gAEF7tkpdeOOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217b015ec8707443a2d6d46cec59789ae1479ac24c810c648306142f67e0b6a01\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224KiVG\\\\\\\/mx\\\\\\\/XDLVnASu6nQNvPr\\\\\\\/Hs2WofpzwZ7r\\\\\\\/WINIen3oDgrnnFfywg9SWLrHeHc7qrXa4mNXpkGeTGFlS+BltdrSDlWXD\\\\\\\/3kAOfGme6iCKs4V3W5e8AIn\\\\\\\/t1CUC36oEbgVnxKFAYJMEBTwU+HHVL+7s07DXSs9x72SD87WnQQZmtUnMlkbM2dU4ajlZGlFZRbgEhrCr\\\\\\\/sihgSP7Gf4vCevWkbFvAIQf9v+mUsmnlT9YEab5LcSEwPuEBTsh5AxK22i10aoIbim8uMx1RM6JV6Qu+CmjfggFF20LDwegraLNKQvAMhcmPq2he2qQxj2FIEpnISd+YiIF9LOAOKyqtYGVLyDY5AMyaoQgoPuDPnkI8AKJX3kz1Em\\\\\\\/1F4LK8bsDEr+4w4AXm\\\\\\\/w4LefVSObZu3RZTjHTFBPwmQyBTCGQXxizhHJjZzC0gxBL\\\\\\\/tK5epJF3zojH4caWrvkvuDcV\\\\\\\/GYpgmPh5KETMK4NoRzZylaauJzdR\\\\\\\/VMDYGuy\\\\\\\/6kuGDW0SZaNzvT+WQCrMIhU7qphSI0jeAdLcHx70Beceah5kHJ\\\\\\\/KCInRBGjt\\\\\\\/NArj7o2+OwOS18\\\\\\\/AE7jHWood2q6IESXdUxm8VoCIZwP9fL0sTpEGLiGtVFk96QDYRmN2O5t177W06IvNoIqLUaETuCIKVMDihXoAeaelGRqHslehvlQYnSgOXSRKhByYGuRbtAZn50SjGkEY4Ej8jy+2HdOCn+fdPtTfvLNIKkq2uxRc3DJCNY\\\\\\\/hlN41l5DVCjfD5DTOdjWeTQw\\\\\\\/WSwieW8rKfSaAiWjsbvfo6QClwa09n3RvoZ9vZ7YLsaVzYymDDHnEvdf9Eu4MrqmVvsfB3YiZOzxLrAdurA3SG0PODTJXjBxpqMFHpwWV2EDDLhUNqD+X4quOiCJCSg1098UQn6BWhHFFXdhNdSQux5V7DF3+MQ\\\\\\\/bubK03SOv2K3LpdfYcD\\\\\\\/xOuK6\\\\\\\/jBKEMbRSXZXW93Z0FAB1paNUnZnFjyHiz1UgGkK3n6KoUyLwaHyV7I4sSgQqSjQfYljETCF1mNhhU8N\\\\\\\/P78sazY4V3+pl6g53F4T+rMacCPTlyiE6eNqxClhwH2DWNXJsTuFZpLV1mFQ1vZsHfauBrcVbBjHtGBePHoFN8WjufRT7lgXOpXcYJj7rQd5Imot5zEw0DpngkoUK9No8gF4cKojVSdA3XXCXMhmwjDZxzh8O8D99FQwZ\\\\\\\/v4o1IR3D+INo5VR4oe+F8BPmK1+VHowirvXibWeK2mRoSj0Ls2iJ8tFfY+Id94nnPJSLDHVudFQrQNR7qm6PJVu7nk7oLoj33GsWWqF19dOw1Ss2sKbLGrEpup\\\\\\\/rU04rh2nWJml01KDp\\\\\\\/l5hjgYIZW9VfDdkuS3G4SUlTqU8klpvAlF+3fYllpCDuIfTRE14rwMgtinB59Rw2qSKuUPi2jmIEo7YjzNRtxcN+z\\\\\\\/icpgIp9hEcmnkrSEn6XICoVEei7QW9nZN0L7iLyH4BJun0x6prmW9s0Fnud+4+lrk8PIWeUlRcRddLuT0hoIl\\\\\\\/nG3RWSdlLdekVbm+dhkM3uWJxF\\\\\\\/Nw\\\\\\\/JgR367JwVBEmUzoSF9hShgr+\\\\\\\/wBfRu3ekejsf9fRBZyIYp9lCcOjG2WzeiLWWeJdWJCPbwKhUNDfrqGZIrUfq0KMm1dce687l50aghkdR8C8t5PN3taqMJ+ktG2+MzRzn7iDAiAJoGwndfYVLw7eKYPHYiB5s0waUzGLAHGy3UXKTvrekh8HM8XE1Qrijuo97du7gQ6f+iE0q+r9KQYJSOkErQkw6YnUYEdlGdO6FCJapBMOemLWBHNu0A6Uvy\\\\\\\/4uJ5sU5FjYST8ASRw5ZdHJwzEIkIn6OuUpHZxVcwc2A\\\\\\\/f5rewrDh16baZf\\\\\\\/w1x5EgNCDdELUYY1nVb4iUoqXpTtEsOPsc2mIWheeQGc9XOLIuHnyZv9YqFKcCZjKtxHgdeiLiTT3tpvVEafpTfEa+iTd5cLDuzzSwLOyALziJMIr0i7lBeWsI8i3779c1JgnxhTQOw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022smOvs09NabibjjCirkBRcw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225802263717273dce0102b8a9d39f92f477cd1c6d4872a414cd79875dcc9f19da\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459862 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/891" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ylPqqmp-HREAccIii6sRPDVR3tE7Fbo0GSuq82aMJ64" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/891" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3c9-yjq45CFn7oWtGE5VsIB4bAVcXcAxTiVPU252Z2A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/885" title="Impact of Genetic Background on Neonatal Lethality of Gga2 Gene-Trap Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/901" title="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459862" id="node459862" data-pisa="ggg;4/5/891" data-pisa-master="ggg;g3.114.010942" data-apath="/ggg/4/5/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yong-Bi Fu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-900;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010942 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459862" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong-Bi Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research Centre, Agriculture and Agri-Food Canada, Saskatoon, SK S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong-Bi%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong-Bi%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yong-bi.fu@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">yong-bi.fu@agr.gc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/5/891" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459862" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459862/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/891.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459862" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459862/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/891.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459862" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459862/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/891.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459862" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459862/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/5/891.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459862" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459862/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genotyping by sequencing (GBS) recently has emerged as a promising genomic approach for assessing genetic diversity on a genome-wide scale. However, concerns are not lacking about the uniquely large unbalance in GBS genotype data. Although some genotype imputation has been proposed to infer missing observations, little is known about the reliability of a genetic diversity analysis of GBS data, with up to 90% of observations missing. Here we performed an empirical assessment of accuracy in genetic diversity analysis of highly incomplete single nucleotide polymorphism genotypes with imputations. Three large single-nucleotide polymorphism genotype data sets for corn, wheat, and rice were acquired, and missing data with up to 90% of missing observations were randomly generated and then imputed for missing genotypes with three map-independent imputation methods. Estimating heterozygosity and inbreeding coefficient from original, missing, and imputed data revealed variable patterns of bias from assessed levels of missingness and genotype imputation, but the estimation biases were smaller for missing data without genotype imputation. The estimates of genetic differentiation were rather robust up to 90% of missing observations but became substantially biased when missing genotypes were imputed. The estimates of topology accuracy for four representative samples of interested groups generally were reduced with increased levels of missing genotypes. Probabilistic principal component analysis based imputation performed better in terms of topology accuracy than those analyses of missing data without genotype imputation. These findings are not only significant for understanding the reliability of the genetic diversity analysis with respect to large missing data and genotype imputation but also are instructive for performing a proper genetic diversity analysis of highly incomplete GBS or other genotype data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">Genetic diversity</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/missing-data" class="hw-term hw-article-keyword hw-article-keyword-missing-data" rel="nofollow">missing data</a></li><li class="kwd"><a href="/keyword/unordered-marker-data" class="hw-term hw-article-keyword hw-article-keyword-unordered-marker-data" rel="nofollow">unordered marker data</a></li></ul><p id="p-4">Genotyping by sequencing (GBS) recently has emerged as a promising genomic approach for exploring genetic diversity and association mapping on a genome-wide scale (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Huang <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fu and Peterson 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Poland and Rife 2012</a>), thanks to the advances in next-generation sequencing technologies (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Metzker 2010</a>). Based on the genome reduction with restriction enzymes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altshuler <em>et al.</em> 2000</a>), the GBS approach does not require a reference genome for single nucleotide polymorphism (SNP) discovery, is a one combined step process of marker discovery and genotyping, and provides a rapid, high-throughput, and cost-effective tool for a genome-wide analysis of genetic diversity for a range of nonmodel species and germplasm sets (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Poland and Rife 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2014</a>). However, one of the unique features associated with GBS is the generation of highly incomplete SNP genotype data (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Williams <em>et al.</em> 2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fu and Peterson 2011</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2014</a>), largely due to low coverage sequencing (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Davey <em>et al.</em> 2011</a>). The incompleteness could be up to 90% of observations missing (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Fu and Peterson 2011</a>) and is considerably larger than in traditional genetic data with 15% or lower level of missing observations. Such a high incompleteness in genotype data will undoubtedly make genetic analyses difficult and less reliable (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pool <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nielsen <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crawford and Lazzaro 2012</a>). Thus, it is desirable to assess the accuracy of various population genetic analyses (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Marchini and Howie 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>), including those for genetic diversity, with respect to incompleteness in GBS genotype data.</p><p id="p-5">Some genetic analyses of SNP data with low levels of missing observations can provide reliable estimations of genetic parameters, but little is known about the impact when the missing level is up to 90%. Traditionally, when the missing level is 15% or lower, it is common to eliminate genetic markers with incomplete observations, which could result in some information loss (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Excoffier and Lischer 2010</a>). To minimize the loss, several imputation methods based on row averages, row medians, and data correlation (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Little and Rubin 1987</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Horton and Kleinman 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carpenter and Kenward 2013</a>) have been applied to infer missing genotypes for a genetic analysis (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Troyanskaya <em>et al.</em> 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Iwata and Jannink 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Weigel <em>et al.</em> 2010</a>). The applications based on the haplotype analysis with a reference genome sequence have been quite successful in recovering missing genotypes to empower many genetic analyses such as genome-wide association mapping (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Marchini and Howie 2010</a>). However, many GBS data are generated from nonmodel organisms without sequenced genomes, and resulting SNP markers are not ordered along a reference genome or genetic maps, making the haplotype-based imputations not feasible for genetic analyses of GBS data (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>).</p><p id="p-6">Efforts to impute unordered SNP genotype data have been made using regression-based methods such as random forest (RF, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Breiman 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stekhoven and Bühlmann 2011</a>) and principal component analysis (PCA)-based tools (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stacklies <em>et al.</em> 2007</a>) with the hope of improving genomic selection (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>). These efforts revealed that RF yielded better performance in genomic selection (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>), but it was much slower than PCA-based imputations. However, little attention has been paid to issues associated with unordered missing SNP data and imputation application for a genetic diversity analysis (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Williams <em>et al.</em> 2010</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2014</a>), as current studies on genotype imputations largely considered only the ordered SNP data (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>). Given the increasing use of GBS to explore genetic diversity in nonmodel species (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Poland and Rife 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sonah <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lu <em>et al.</em> 2013</a>) and in 7.4 millions of <em>ex situ</em> plant germplasm samples conserved in world genebanks (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">FAO 2010</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Fu and Peterson 2011</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">2012</a>), more needs to be learnt about the reliability of a genetic diversity analysis of GBS genotype data (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2014</a>).</p><p id="p-7">As allele frequency distributions in GBS data are unknown and may vary for different species, we conducted an empirical assessment of accuracy in genetic diversity analyses of highly incomplete genotype data with imputations. Specifically, this was done with the acquisition of three large SNP genotype data from published corn, rice and wheat databases. Missing data were randomly generated with up to 90% of its observations and then imputed for missing genotypes with three map-independent imputation methods. The genetic diversity analyses of original, missing and imputed data were performed and the corresponding estimates of diversity parameters were compared. It was our hope that this assessment allowed for a better understanding of bias, if any, in the genetic diversity analysis of highly incomplete GBS genotype data. A biased estimation of diversity parameters could mislead the comparison of genetic diversity, invalidate the analysis of genetic structure, or complicate the inference of genetic relationship.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Original dataset</h3><p id="p-8">Since the ideal GBS data set (<em>i.e.</em>, a complete GBS genotype data without any missing observations) is currently not available, we acquired three large published genotype data sets of genome-wide SNP markers for this assessment (see supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010942/-/DC1/FileS1.csv">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010942/-/DC1/FileS2.csv">File S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010942/-/DC1/FileS3.csv">File S3</a>). The first data set was the original corn genomic data set published by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Van Heerwaarden <em>et al.</em> (2012)</a>, in which 400 diverse corn lines were genotyped with 45,997 polymorphic SNP markers with less than 10% missing data. The second data set was the rice genotype data published by <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhao <em>et al.</em> (2010)</a>, in which 395 diverse rice lines were genotyped by 1311 polymorphic SNP markers. The third one was a subset of the original genomic data published by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cavanagh <em>et al.</em> (2013)</a>, in which a worldwide sample of 2994 accessions of hexaploid wheat were genotyped for 90,000 gene-associated SNP markers. For this assessment, we arbitrarily chose the Leaf Rust Panel of 339 diverse wheat lines that were genotyped by 33,526 polymorphic SNP markers. Although genome sequences for corn and rice have been published, this assessment treated the assayed data sets as unordered SNP genotype data.</p></div><div id="sec-3" class="subsection"><h3>Generation of randomly missing data</h3><p id="p-9">For each original data set, a simulation was performed to generate 100 sets of randomly missing data with a given missing level by a random selection of observations from the original data set as missing without considering existing missing observations, individual lines and markers. The assessed missing levels ranged from 10 to 90% of the total observations with missing values which were compatible with those reported in GBS genotype data (<em>e.g.</em>, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Fu and Peterson 2011</a>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">2012</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2014</a>). This was done for all the three original data sets.</p></div><div id="sec-4" class="subsection"><h3>Imputation methods</h3><p id="p-10">We assessed three imputation methods in this study, as these imputations have been shown to be the most promising tools to recover missing genotypes (<em>e.g.</em>, see <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Moser <em>et al.</em> 2009</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>). The first one is the widely used RF regression procedure (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Breiman 2001</a>), and it is based on all available data to predict the missing values for every locus. The RF procedure has been described in detail for imputing missing genotypes for genomic selection (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>) and implemented in the freely available R package “randomForest” (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liaw and Wiener 2002</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">R Development Core Team 2013</a>). The other two are PCA-based imputation procedures: probabilistic PCA (PP) and nonlinear iterative partial least squares PCA (NI). These two procedures are described in detail and implemented in the freely available R package “pcaMethods” (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Stacklies <em>et al.</em> 2007</a>). The principle behind these PCA-based imputations is that missing values are initially set to the row averages, and singular value decomposition of the SNP matrix is used to create orthogonal principal components. The principal components, which correspond to the largest eigenvalues, are then used to reconstruct the missing SNP genotypes in the SNP matrix. Nonlinear iterative partial least squares PCA uses the algorithm of nonlinear estimation by iterative partial least squares (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wold 1966</a>) for finding the principal components of the SNP matrix. Note that the PCA-based methods generated estimates on a continuous scale, which can be less than 0 or larger than 2. An ad-hoc binning algorithm was used to assign the PCA-based estimates into distinct genotypes, where homozygous genotype 1 was assigned if the estimate was ≤1.5, the other homozygous genotype 3 if the estimate was &gt;2.5, and the heterozygous genotype 2 when the estimate was &gt;1.5 and ≤2.5. These reasonable assignments were obtained after several verification trials of each crop data set, but they were not optimized as it is required for each PCA-based imputation in each crop data set.</p></div><div id="sec-5" class="subsection"><h3>Genetic diversity analysis</h3><p id="p-11">We considered a typical crop genetic diversity analysis with three major diversity components that were aimed to (1) estimate observed and expected heterozygosities (Ho, He) and inbreeding coefficient (F), (2) assess genetic differentiation, and (3) infer genetic relationships among different groups of crop germplasm, respectively. Specifically, estimations of Ho, He, and F were performed following the formulae shown in Box 1.2 of GenAlex Tutorial 1 for co-dominant marker data (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Peakall and Smouse 2012</a>). Genetic differentiation among groups of germplasm was estimated with Phi statistics (Φst) following the formulae shown in Box 2.1 of GenAlex Tutorial 2 for co-dominant marker data (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Peakall and Smouse 2012</a>). The estimations of genetic diversity and differentiation were done with custom R functions specifically written and tested in the R environment (<a href="http://www.r-project.org">http://www.r-project.org</a>) with standard genotype data before use. Genetic relationships among different groups of crop germplasm were inferred by obtaining a distance matrix with a custom R function and clustering them based on the neighbor-joining algorithm with the freely available R package “ape” (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Paradis <em>et al.</em> 2004</a>). The custom R function inputted SNP genotype data for different groups, took into account missing data (if any) to calculate similarity using simple match coefficient (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sokal and Michener 1958</a>), and converted those similarity values into a dissimilarity distance matrix.</p></div><div id="sec-6" class="subsection"><h3>Assessment procedure</h3><p id="p-12">The overall assessment procedure for a crop data set started with the input of an original crop data set, generation of its randomly missing data, and imputation for missing genotypes, followed by the genetic diversity analysis of original, missing, and imputed data. For this assessment, nine custom R scripts were specifically written, each involved with each imputation method (RF, PP, NI) and each diversity component (genetic diversity, differentiation, and relationship), and they are available upon request. More specifically, each R script started with (1) an acquisition of an original crop data set; (2) a function of generating a random missing data with a given missing level; (3) a function of a specific diversity analysis; (4) a loop of generating randomly missing data from the whole data set, sampling for a random subset of 130 lines and 1000 markers, imputing missing genotypes of the sampled subset data following specific imputation method as described previously, performing the specific diversity analysis as mentioned previously of the three subset data (original, missing, and imputed), and accumulation of parameter estimates over simulation runs; and (5) ended with related estimation outputs. Note that the loop functioned for each of the nine levels of missing observations (from 10 to 90%) and the number of simulation runs (=100 in this study). The mean and standard deviation of a diversity estimate over 100 runs were obtained, except for the topology accuracy (see below). These nine R scripts were run for each of the crop data sets.</p><p id="p-13">For each simulation run, a random sampling of 130 lines and 1000 markers from the whole randomly missing data set was made, and their corresponding original genotype data were obtained. This step is required to make the assessment practically feasible, as the original crop data sets are large, requiring a large computing capacity and execution time, particularly for RF imputation (see the <em>Results</em> section <em>Imputation accuracy and execution time</em>). The imputation was made only on the random subset of 130 lines with 1000 markers. Consequently, each simulation run generated a random subset of original, missing (for a given missing level), imputed data for the diversity component analysis. The random subset of 130 lines should reasonably sample the originally defined groups in each crop data set and thus be suitable for the assessment on genetic diversity analysis. In this assessment, we considered the originally defined groups of interest in each crop data set as the known stratifications within each crop. Specifically, 400 corn lines represented four major age groups; 395 rice lines were grouped by <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Zhao <em>et al</em>. (2010)</a> as four of the major subpopulations (<em>indica</em>, <em>aus</em>, <em>tropical japonica</em>, and <em>temperate japonica</em>); and 339 wheat lines represented four source groups of wheat leaf rust.</p><p id="p-14">For this assessment, the estimation of heterozygosity and inbreeding coefficient was made above the group level. The functions for estimating both observed and expected heterozygosities took the missing observations into consideration, as the allele frequency estimation was made only from those data without missing genotypes. For the assessment on genetic relationships, each simulation run after the imputation step had an additional random sampling of one individual sample as a representation for each group from the random subset; the corresponding genotype data for the four representative samples were obtained; and the topology of the genetic relationships among the four representative samples was assessed by following the method of <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wiens and Tiu (2012)</a>. The topology accuracy was determined by counting the number out of 100 missing or imputed data sets with the same neighbor-joining tree of the four representative samples as that from the corresponding original genotype data.</p><p id="p-15">For each diversity estimates, bias was inferred by comparing the estimate from missing or imputed data with that obtained from the original data and calculated, if any, as the percentage of the difference between estimates obtained from missing (or imputed) data and original data over the estimate from the original data.</p><p id="p-16">To assess the impact of sample size on the imputation-based estimation of diversity parameters, we performed extra simulations on the wheat data set with the PP imputation method for four sample size combinations of breeding lines (130, 260) and SNP markers (1000, 10,000). For each sample size combination, each diversity estimate was obtained over 100 simulation runs.</p></div><div id="sec-7" class="subsection"><h3>Imputation accuracy and execution time</h3><p id="p-17">The imputation accuracy on imputed genotypes was estimated as the percentage of the imputed genotypes matched with the corresponding original genotypes over all the random missing SNP genotypes for a given missing level in each simulation run. The mean and standard deviation of the accuracy estimate were obtained over 100 simulation runs. This was done for each crop data set, missing level and imputation method. An extra effort was also made to estimate the imputation accuracy on specific imputed genotypes as the percentage of the specific imputed genotypes matched with the corresponding original major (or frequent AA), heterozygous (Aa), or minor (or less frequent aa) genotypes over all the random missing SNP genotypes. During the executions of the custom R scripts in a Linux server (with version of 2.6.18−238.5.1.e15 GNU/Linux), the execution time for each imputation method was also recorded for comparison, as different imputation algorithms may vary in running time.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Allele distribution</h3><p id="p-18">These three original data sets were selected to represent three different crops (corn, rice, and wheat) with different allele frequency distributions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For each crop data set, the minor alleles were determined and their frequencies across the assayed lines were tabulated. The corn data roughly showed a uniform distribution of minor alleles. The rice data revealed a small excess of greater minor allele frequencies with a maximum around 0.45. The wheat data displayed a large excess of lower minor allele frequencies, particularly with less than 0.1. Note that rice alleles were largely genomic SNPs, whereas corn and wheat alleles were associated with transcribed SNPs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/891/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution of the minor alleles in each SNP genotype data set (A: corn data of 400 lines &#xD7; 45,997 markers, B: rice data of 395 lines &#xD7; 1311 markers, and C: wheat data of 339 lines &#xD7; 33,526 markers)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-170722657" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequency distribution of the minor alleles in each SNP genotype data set (A: corn data of 400 lines &#xD7; 45,997 markers, B: rice data of 395 lines &#xD7; 1311 markers, and C: wheat data of 339 lines &#xD7; 33,526 markers).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/891/F1.medium.gif" width="205" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/5/891/F1.medium.gif" width="205" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/891/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/891/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464288" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Frequency distribution of the minor alleles in each SNP genotype data set (A: corn data of 400 lines × 45,997 markers, B: rice data of 395 lines × 1311 markers, and C: wheat data of 339 lines × 33,526 markers).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Diversity estimates under random missing</h3><p id="p-20">The diversity estimates for the original data and for those data under random missing ranging from 10 to 90% of its observations without genotype imputations are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The observed heterozygosity estimates showed little biases in the corn and rice data sets, but slight upward biases with increased standard deviations in the wheat data set. However, the estimates of expected heterozygosity displayed considerable downward biases (up to 33.2% for corn, 39.1% for rice, and 42% for wheat data with the 90% missing level), whereas their standard deviations were largely stable. Similarly, the estimates of inbreeding coefficient also showed some degree of a downward bias (up to 24.1%) for the corn data with the 90% missing level, along with increased variances, but were largely stable for the rice and wheat data. The estimates of genetic differentiations among four groups present in three data sets were less affected, as the Φst statistics were considerably stable over variable levels of missingness, although increased standard deviations were observed at the higher levels of random missing. The estimates of topology accuracy based on the neighbor-joining tree of four representative samples were decreased with increased levels of missingness but were surprisingly robust in the rice and wheat data sets. For example, the topology accuracy remained 80% or greater when 40% of the wheat genotypes was randomly missing and 51% when the missing level increased to 80%. The estimate for the corn data were 50% or greater when the missing level was 30% or lower. However, the estimate for the rice data were relatively low when compared to the other data sets, but remained 50% or greater when the missing level was increased to 80%.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464393/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464393/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Estimates (with SD) of observed and expected heterozygosity (Ho, He), inbreeding coefficient (F), Phi statistic (Φst), and topology accuracy obtained from 100 runs of simulation on each SNP genotype data set (corn, rice, and wheat) with 0−90% of total observations as random missing without genotype imputation</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Diversity estimates from imputed data</h3><p id="p-22">The imputation-based estimates of observed heterozygosity, expected heterozygosity, and inbreeding coefficient are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. These estimates, compared with those without imputations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), were substantially biased, and the extents of the biases were dependent on the levels of missing data in each crop data set. With increased levels of missing observations, heterozygosity estimates from imputed data were upwardly biased and inbreeding coefficient estimates showed downward biases. The extents of biases for these imputation-based estimations also varied with the crop data sets. When three imputation methods were compared, RF imputation seemed to introduce less bias in diversity estimates than the other two, particularly in the estimates of inbreeding coefficient.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/891/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of observed heterozygosity (Ho), expected heterozygosity (He), inbreeding coefficient (F) over 100 runs of simulation on each single-nucleotide polymorphism genotype data set (A: corn, B: rice, and C: wheat) with 10&#x2013;90% of total observations as random missing and as imputed with an imputation method. RF, random forest; PP, probabilistic PCA; or NI, nonlinear iterative partial least squares PCA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-170722657" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of observed heterozygosity (Ho), expected heterozygosity (He), inbreeding coefficient (F) over 100 runs of simulation on each single-nucleotide polymorphism genotype data set (A: corn, B: rice, and C: wheat) with 10&#x2013;90% of total observations as random missing and as imputed with an imputation method. RF, random forest; PP, probabilistic PCA; or NI, nonlinear iterative partial least squares PCA.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/891/F2.medium.gif" width="186" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/5/891/F2.medium.gif" width="186" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/891/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/891/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Estimates of observed heterozygosity (Ho), expected heterozygosity (He), inbreeding coefficient (F) over 100 runs of simulation on each single-nucleotide polymorphism genotype data set (A: corn, B: rice, and C: wheat) with 10–90% of total observations as random missing and as imputed with an imputation method. RF, random forest; PP, probabilistic PCA; or NI, nonlinear iterative partial least squares PCA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The biases in the imputation-based estimates of genetic differentiation, compared with those without imputations (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), also were substantial for the three crop data sets (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). RF imputation seemed to introduce upward biases, except for the wheat data, whereas NI imputation consistently generated downward biases in all the data sets. However, PP imputation displayed stable levels of bias with increased levels of missingness, except when the missing level was 80% or greater. The extents of the biases observed for the three imputation methods seemed to vary with the assayed data sets.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/891/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of genetic differentiations (or &#x3A6;st statistic) among four groups of 130 individual lines randomly sampled from each SNP genotype data set (A: corn, B: rice, and C: wheat) over 100 runs of simulation with 10&#x2013;90% of total observations as random missing (M, missing data without imputation) and as imputed with an imputation method (RF, random forest; PP, probabilistic principal component analysis [PCA]; or NI, nonlinear iterative partial least squares PCA). Note that the genetic differentiations also were estimated based on the original SNP data (=O for short) for the corresponding 130 individual lines selected for each simulation run." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-170722657" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of genetic differentiations (or &#x3A6;st statistic) among four groups of 130 individual lines randomly sampled from each SNP genotype data set (A: corn, B: rice, and C: wheat) over 100 runs of simulation with 10&#x2013;90% of total observations as random missing (M, missing data without imputation) and as imputed with an imputation method (RF, random forest; PP, probabilistic principal component analysis [PCA]; or NI, nonlinear iterative partial least squares PCA). Note that the genetic differentiations also were estimated based on the original SNP data (=O for short) for the corresponding 130 individual lines selected for each simulation run.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/891/F3.medium.gif" width="205" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/5/891/F3.medium.gif" width="205" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/891/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/891/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464320" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Estimates of genetic differentiations (or Φst statistic) among four groups of 130 individual lines randomly sampled from each SNP genotype data set (A: corn, B: rice, and C: wheat) over 100 runs of simulation with 10–90% of total observations as random missing (M, missing data without imputation) and as imputed with an imputation method (RF, random forest; PP, probabilistic principal component analysis [PCA]; or NI, nonlinear iterative partial least squares PCA). Note that the genetic differentiations also were estimated based on the original SNP data (=O for short) for the corresponding 130 individual lines selected for each simulation run.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The imputation-based estimates of topology accuracy as reflected in four representative samples of interested groups were generally reduced with increased levels of missing genotypes and they were smaller than those from random missing without imputations (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). However, PP imputation seemed to generate greater estimates of topology accuracy than the other two imputations in the three crop data sets and largely than those without imputations in the wheat data set (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The patterns of the topology accuracy estimates also depended on the assayed allelic frequency distributions. For example, the imputations of the wheat genotypes were more effective with relatively greater estimates of topology accuracy than those on the other crop data sets (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/891/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of topology accuracy (%) for neighbor-joining clusters of four samples representing four groups in each SNP genotype data set (A: corn, B: rice, and C: wheat) over 100 runs of simulation with 10&#x2013;90% of total observations as random missing (M, missing data without imputation) and as imputed with an imputation method (RF, random forest; PP, probabilistic principal component analysis [PCA]; or NI, nonlinear iterative partial least squares PCA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-170722657" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of topology accuracy (%) for neighbor-joining clusters of four samples representing four groups in each SNP genotype data set (A: corn, B: rice, and C: wheat) over 100 runs of simulation with 10&#x2013;90% of total observations as random missing (M, missing data without imputation) and as imputed with an imputation method (RF, random forest; PP, probabilistic principal component analysis [PCA]; or NI, nonlinear iterative partial least squares PCA).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/891/F4.medium.gif" width="203" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/5/891/F4.medium.gif" width="203" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/891/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/891/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464336" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Estimates of topology accuracy (%) for neighbor-joining clusters of four samples representing four groups in each SNP genotype data set (A: corn, B: rice, and C: wheat) over 100 runs of simulation with 10–90% of total observations as random missing (M, missing data without imputation) and as imputed with an imputation method (RF, random forest; PP, probabilistic principal component analysis [PCA]; or NI, nonlinear iterative partial least squares PCA).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Enlarging the sample size from 130 to 260 wheat breeding lines and/or from 1000 to 10,000 markers did not appear to affect much the PP imputation-based estimates of the four diversity parameters (He, F, Φst statistic, and topology accuracy) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Overall, the variations in the diversity estimates among the four sample size combinations were relatively small, except for those in the topology accuracy estimates. The marked differences in the estimates of Φst statistic and topology accuracy were observed when 90% of the wheat genotypes were randomly missing.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/891/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of expected heterozygosity (He; A), inbreeding coefficient (Fst; B), Phi statistic (&#x3A6;st; C), and topology accuracy (D) obtained from 100 runs of simulation on the wheat SNP genotype data set with 10&#x2013;90% of total observations as random missing and as imputed by the probabilistic principal component analysis (PP) imputation for four sample size combinations of breeding lines (L = 130, 260) and SNP markers (m = 1000, 10,000)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-170722657" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of expected heterozygosity (He; A), inbreeding coefficient (Fst; B), Phi statistic (&#x3A6;st; C), and topology accuracy (D) obtained from 100 runs of simulation on the wheat SNP genotype data set with 10&#x2013;90% of total observations as random missing and as imputed by the probabilistic principal component analysis (PP) imputation for four sample size combinations of breeding lines (L = 130, 260) and SNP markers (m = 1000, 10,000).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/891/F5.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/5/891/F5.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/891/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/891/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Estimates of expected heterozygosity (He; A), inbreeding coefficient (Fst; B), Phi statistic (Φst; C), and topology accuracy (D) obtained from 100 runs of simulation on the wheat SNP genotype data set with 10–90% of total observations as random missing and as imputed by the probabilistic principal component analysis (PP) imputation for four sample size combinations of breeding lines (L = 130, 260) and SNP markers (m = 1000, 10,000).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Imputation accuracy and execution time</h3><p id="p-30">Another way to assess imputation accuracy is to measure the percentage of the random missing SNP genotypes after imputations in which the imputed genotypes were matched with the corresponding original genotypes. Overall, PP imputation showed the highest estimates of imputation accuracy in the corn and wheat data sets (<em>i.e.</em>, 54.4–50.4% and 71.5–67.0% recovery of the original genotypes for the nine levels of missingness, respectively), whereas RF had the highest matches in the rice data set (74.1–65.1% for the nine levels of missingness) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). More interestingly, PP imputation revealed the greatest recovery of the major (or AA) genotypes, followed by NI and RF imputations. For example, when 10% of the wheat data were randomly missing, PP recovered 67.6% of the original major genotypes, NI 65.5%, and RF 48.4%. In contrast, RF imputation had the greatest recovery of the minor (or aa) genotypes, followed by PP and NI imputations. For example, when 10% of the rice data were randomly missing, RF recovered 23.1% of the original minor genotypes, PP 13.1%, and NI 11.0%.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464309/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464309/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimates of imputation accuracy as the percentage of the imputed genotypes matched with the corresponding original genotypes over all the random missing SNP genotypes over 100 runs of simulation on each SNP genotype data set (corn, rice, and wheat) with 10−90% of total observations as random missing and as imputed by an imputation method (RF, PP, or NI)</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The times for execution of R scripts in the Linux server to assess topology accuracy in the corn data with 100 runs of simulation were recorded as 70.5 hr for RF imputation, 7.08 hr for PP imputation, and 1.1 hr for NI imputation. Some variation in execution time was observed for each imputation method with respect to other diversity estimation and crop data set, but the above trend of variation in execution time remained the same.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-33">The empirical assessment performed here revealed variable patterns of bias in estimation of crop genetic diversity with respect to large missing data and map-independent imputation. Generally, a genetic diversity analysis of a random missing data yielded much smaller biases than that of an imputed data. The estimation of genetic differentiation via Φst statistic was robust up to 90% missing observations but could yield a substantial bias from genotype imputation. The imputation-based estimates of topology accuracy for four representative samples were generally reduced with increased levels of missing genotypes. The probabilistic PCA-based imputation could perform better in topology accuracy than those analyses of missing data without imputation. These findings are significant for understanding the reliability of a genetic diversity analysis with respect to large missing data and map-independent imputation.</p><p id="p-34">The robust estimations of genetic differentiation from nonimputed data with respect to high incompleteness are expected because the algorithm used to calculate Φst statistic takes into account missing observations (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Excoffier and Lischer 2010</a>). For some applications, alleles of frequency less than 0.05 may be excluded, but our analysis included those genotypes of frequencies less than 0.05, as we followed the co-dominant genotype data to estimate Φst statistic. However, our nonimputation-based estimations of heterozygosity and inbreeding coefficient also considered the missing observations but still yielded considerable biases (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), although much smaller than those from genotype imputation (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The reason for this remains unknown. It is possible that the biases were due to either nonrandomness partition by imputation of diversity among four groups or other errors we did not detect from genotype imputation. Our assessment showed that PP imputation outperformed the other two methods both in the recovery of missing genotypes and the inference of genetic relationships. This was somehow surprising, given the favorable assessment done on RF for genomic selection (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2013</a>). This could be explained with the specific finding that PP imputation was mostly effective in recovering the major genotypes, whereas RF was the best in recovering the minor genotypes (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). However, with respect to imputation speed, NI imputation was the fastest, while RF imputation was extremely slow.</p><p id="p-35">This assessment revealed two interesting findings. First, the assessed imputations did not show any superiority in accuracy for these genetic diversity analyses (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>) over those without imputations (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The traditional genetic diversity analyses of highly incomplete genotype data showed less biased in diversity estimates than those analyses of imputed data. Second, the high PP imputation accuracy helped only the genetic relationship inference (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>), but not much the estimations of heterozygosity, inbreeding coefficient and genetic differentiation (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). For example, PP imputation recovered 67% of the original genotypes for up to 90% level of missing data (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), but substantial biases were still observed in the imputation-based estimations of heterozygosity, inbreeding coefficient and genetic differentiation (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). It is possible that the biases were largely due to the loss of the minor genotypes from random missing, as PP imputation was mostly effective in recovering major, not minor, genotypes (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-36">However, our assessment had several limitations. First, our findings may be data specific, as the assessed types of allele frequency distribution may not reflect those present in GBS data and for other organisms. These crop data sets may differ from GBS data in the generation of SNP data and the sampling coverage of a genome, as mentioned above. Deviations from GBS allele frequency distributions may generate patterns of bias different from our findings. Second, although we have examined a wide range of missing data from 10 to 90% compatible with the reported GBS data (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Fu and Peterson 2011</a>; <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2014</a>), our analyses were limited to those assayed imputation methods, essentially only two types of imputation using correlation information. More assessments could be explored with other imputation methods and different types of allele frequency distribution to enlarge the scope of accuracy assessments with respect to high incompleteness. Third, it is difficult to assess topology accuracy for a large number of samples (see <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Robinson and Foulds 1981</a>) and we proceeded with only four samples as <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wiens and Tiu (2012)</a> did for a phylogenetic analysis. These topology accuracy measures provided only a snap-shot, but not necessarily the whole picture, of accuracy in the genetic relationship inferences of 130 samples. Fourth, our assessment considered only the traditional methods commonly used in genetic diversity analysis, not those maximum likelihood or Bayesian methods recently developed for population genetic inferences from NGS data (see <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Nielsen <em>et al.</em> 2011</a>). The latter may help to reduce estimation bias due to low coverage like those reported in GBS data (<em>e.g.</em>, see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fumagalli <em>et al.</em> 2013</a>). Fifth, we considered only one simple missing mechanism: random missing. The real mechanisms for missing observations from GBS may be complicated and the expected patterns of estimation bias could differ from what were reported here. Simulating GBS data for different organisms may help to shed more light on the impact of highly incomplete genotype data on diversity estimation.</p><p id="p-37">Despite these limitations, the findings reported here have some implications for the genetic diversity analysis of large unbalanced SNP genotype data, particularly for those highly incomplete GBS data acquired from nonmodel species. First, the estimation of genetic differentiation via Φst statistic was robust, requiring no imputation for missing genotypes. Second, the estimations of heterozygosity and inbreeding coefficient would become less accurate with larger numbers of missing observations, but the estimation biases were much smaller than those from genotype imputations. Thus, these diversity estimations from genotype imputations should not be encouraged. Third, the probabilistic PCA-based imputation could enhance topology accuracy in genetic relationship inferences, but may not always yield higher topology accuracy than those from missing data without genotype imputation. Fourth, considering those assayed types of allele frequency distribution (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), one could reason that these findings may also be applicable to some highly incomplete GBS genotype data generated from non-crop organisms. Last, the demonstrated estimation biases imply more efforts are needed to (1) minimize genotype incompleteness with more effective GBS protocols (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Sonah <em>et al.</em> 2013</a>); (2) develop more effective map-independent imputation tools for GBS data (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012</a>); (3) integrating a GBS protocol with <em>de novo</em> sequencing to assemble a reference for generating reference-based GBS data for which map-dependent imputation can be applied; or (4) develop more accurate population genetic inferences from highly incomplete GBS data (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lynch 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nielsen <em>et al.</em> 2012</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fumagalli <em>et al.</em> 2013</a>).</p><p id="p-38">In conclusion, these findings indicate that the genetic diversity analysis of highly incomplete GBS genotype data could be reliably performed but requires caution with respect to analysis goal, missing level and imputation application. The findings reported here are also instructive for performing a proper genetic diversity analysis of highly incomplete GBS or other genotype data.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">We thank Jeffrey Ross-Ibarra for the access to corn single nucleotide polymorphism (SNP) genotype data; Eduard Akhunov and Clayton Birkett for the access to wheat SNP genotype data from T3 database; John Wiens, Daniel Stekhoven, and Stef van Buuren for their assistance in the diversity and imputation analyses; and three journal reviewers for their constructive comments on the early version of the manuscript. This research was funded by an Agriculture and Agri-Food Canada A-Base research project to YBF.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010942/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010942/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-40"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-01-07"><span class="received-label">Received </span>January 7, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-03-10"><span class="accepted-label">Accepted </span>March 10, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Fu</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.1" data-doi="10.1038/35035083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollara</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Etten</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An SNP map of the human genome generated by reduced representation shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">407</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D407%26rft.issue%253D6803%26rft.spage%253D513%26rft.epage%253D516%26rft.atitle%253DAn%2BSNP%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome%2Bgenerated%2Bby%2Breduced%2Brepresentation%2Bshotgun%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35035083%26rft_id%253Dinfo%253Apmid%252F11029002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35035083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11029002&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089727400050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.2" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.891.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenward</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Multiple imputation of unordered categorical data, in multiple imputation and its application</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Ltd</span>, <span class="cit-publ-loc">Chichester, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.4" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazzaro</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Assessing the accuracy and power of population genetic inference from low-pass next-generations sequncing data.</span> <abbr class="cit-jnl-abbrev">Frontiers Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2BGenet.%26rft.volume%253D3%26rft.spage%253D66%26rft.atitle%253DASSESSING%2BTHE%2BACCURACY%2BAND%2BPOWER%2BOF%2BPOPULATION%2BGENETIC%2BINFERENCE%2BFROM%2BLOW-PASS%2BNEXT-GENERATION%2BSEQUENCING%2BDATA.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.6" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thureen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.7" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.8" data-doi="10.1111/j.1755-0998.2010.02847.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lischer</span>  <span class="cit-name-given-names">H. E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Arlequin suite ver 3.5: A new series of programs to perform population genetics analyses under Linux and Windows.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D10%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02847.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02847.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.891.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FAO</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">The Second Report on the State of the World’s Plant Genetic Resources for Food and Agriculture</span>. <span class="cit-publ-name">FAO</span>, <span class="cit-publ-loc">Rome, Italy</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.10" data-doi="10.3835/plantgenome2011.08.0022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic diversity analysis with 454 pyrosequencing and genomic reduction confirmed the eastern and western division in the cultivated barley gene pool.</span> <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D4%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.11" data-doi="10.1111/j.1755-0998.2011.03100.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developing genomic resources in two Linum species via 454 pyrosequencing and genomic reduction.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D12%26rft.spage%253D492%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.03100.x%26rft_id%253Dinfo%253Apmid%252F22177006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.03100.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22177006&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.12" data-doi="10.1007/s10722-013-0058-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity analysis of yellow mustard (<em>Sinapis alba</em> L.) germplasm based on genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D61%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-013-0058-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-013-0058-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.13" data-doi="10.1534/genetics.113.154740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linderoth</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sanchez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantifying population genetic differentiation from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.154740%26rft_id%253Dinfo%253Apmid%252F23979584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvOTc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.14" data-doi="10.1198/000313007X172556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinman</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Much ado about nothing: a comparison of missing data methods and software to fit incomplete data regression models.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BStatistician%26rft.stitle%253DThe%2BAmerican%2BStatistician%26rft.aulast%253DHorton%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D90%26rft.atitle%253DMuch%2Bado%2Babout%2Bnothing%253A%2BA%2Bcomparison%2Bof%2Bmissing%2Bdata%2Bmethods%2Band%2Bsoftware%2Bto%2Bfit%2Bincomplete%2Bdata%2Bregression%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1198%252F000313007X172556%26rft_id%253Dinfo%253Apmid%252F17401454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/000313007X172556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17401454&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243781600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.15" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.16" data-doi="10.2135/cropsci2009.08.0434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Marker genotype imputation in a low marker-density panel with a high-marker-density reference panel: accuracy evaluation in barley breeding lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DIwata%26rft.auinit1%253DH.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1269%26rft.epage%253D1278%26rft.atitle%253DMarker%2BGenotype%2BImputation%2Bin%2Ba%2BLow-Marker-Density%2BPanel%2Bwith%2Ba%2BHigh-Marker-Density%2BReference%2BPanel%253A%2BAccuracy%2BEvaluation%2Bin%2BBarley%2BBreeding%2BLines%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.08.0434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.08.0434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000279138200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiener</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Classification and regression by randomForest.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.891.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Statistical Analysis With Missing Data</span>. <span class="cit-publ-name">John Wiley Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.19" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.20" data-doi="10.1093/molbev/msn185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimation of nucleotide diversity, disequilibrium coefficients, and mutation rates from high-coverage genome-sequencing projects.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2409</span>–<span class="cit-lpage">2419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn185%26rft_id%253Dinfo%253Apmid%252F18725384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8yNDA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.21" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.22" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies—the next generation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Imputation of missing genotypes in high desnity SNP data.</span> <abbr class="cit-jnl-abbrev">Proc. Assoc. Adv. Anim. Breed. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BAssoc.%2BAdv.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D18%26rft.spage%253D612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.24" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.25" data-doi="10.1371/journal.pone.0037558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP calling, genotype calling, and sample allele frequency estimation from new-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De37558%26rft.epage%253De37558%26rft.atitle%253DSNP%2Bcalling%252C%2Bgenotype%2Bcalling%252C%2Band%2Bsample%2Ballele%2Bfrequency%2Bestimation%2Bfrom%2BNew-Generation%2BSequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037558%26rft_id%253Dinfo%253Apmid%252F22911679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037558&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22911679&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.26" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">ape: analyses of phylogenetics and evolution in R language.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMC8yLzI4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.27" data-doi="10.1093/bioinformatics/bts460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peakall</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research—an update.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2537</span>–<span class="cit-lpage">2539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts460%26rft_id%253Dinfo%253Apmid%252F22820204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTkvMjUzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.28" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.29" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.30" data-doi="10.1101/gr.079509.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genetic inference from genomic sequence variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.079509.108%26rft_id%253Dinfo%253Apmid%252F20067940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.891.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.32" data-doi="10.1016/0025-5564(81)90043-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Comparison of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D53%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252881%252990043-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(81)90043-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LB66300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-4.5.891.33" data-doi="10.1534/g3.112.005363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation of unordered markers and the impact on genomic selection accuracy.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DRutkoski%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D427%26rft.epage%253D439%26rft.atitle%253DImputation%2Bof%2BUnordered%2BMarkers%2Band%2Bthe%2BImpact%2Bon%2BGenomic%2BSelection%2BAccuracy%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005363%26rft_id%253Dinfo%253Apmid%252F23449944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMy80MjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sokal</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michener</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">A statistical method for evaluating systematic relationships.</span> <abbr class="cit-jnl-abbrev">Univ. Kansas Sci. Bull.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUniv.%2BKansas%2BSci.%2BBull.%26rft.volume%253D38%26rft.spage%253D1409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.35" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Légaré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.36" data-doi="10.1093/bioinformatics/btm069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stacklies</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redestig</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selbig</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">pcaMethods—a bioconductor package providing PCA methods for incomplete data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm069%26rft_id%253Dinfo%253Apmid%252F17344241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy85LzExNjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stekhoven</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MissForest—nonparametric missing value imputation for mixed-type data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D28%26rft.spage%253D112%26rft_id%253Dinfo%253Apmid%252F22039212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22039212&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.38" data-doi="10.1093/bioinformatics/17.6.520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Missing value estimation methods for DNA microarrays.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DTroyanskaya%26rft.auinit1%253DO.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D520%26rft.epage%253D525%26rft.atitle%253DMissing%2Bvalue%2Bestimation%2Bmethods%2Bfor%2BDNA%2Bmicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.6.520%26rft_id%253Dinfo%253Apmid%252F11395428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNy82LzUyMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.39" data-doi="10.1073/pnas.1209275109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Historical genomics of North American maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">12420</span>–<span class="cit-lpage">12425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1209275109%26rft_id%253Dinfo%253Apmid%252F22802642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMxLzEyNDIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.40" data-doi="10.3168/jds.2010-3149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5423</span>–<span class="cit-lpage">5435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWeigel%26rft.auinit1%253DK.%2BA.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5423%26rft.epage%253D5435%26rft.atitle%253DAccuracy%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bderived%2Bfrom%2Bimputed%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenotypes%2Bin%2BJersey%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3149%26rft_id%253Dinfo%253Apmid%252F20965358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965358&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.41" data-doi="10.1371/journal.pone.0042925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiens</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Highly incomplete taxa can rescue phylogenetic analyses from the negative impacts of limited taxon sampling.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e42925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWiens%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De42925%26rft.epage%253De42925%26rft.atitle%253DHighly%2Bincomplete%2Btaxa%2Bcan%2Brescue%2Bphylogenetic%2Banalyses%2Bfrom%2Bthe%2Bnegative%2Bimpacts%2Bof%2Blimited%2Btaxon%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0042925%26rft_id%253Dinfo%253Apmid%252F22900065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0042925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22900065&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.42" data-doi="10.1186/1471-2164-11-32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oleksiak</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SNP identification, verification, and utility for population genetics in a non-model genus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAlbrecht%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.spage%253D32%26rft.epage%253D32%26rft.atitle%253DIntegrative%2Banalysis%2Bof%2Bthe%2Bheat%2Bshock%2Bresponse%2Bin%2BAspergillus%2Bfumigatus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-32%26rft_id%253Dinfo%253Apmid%252F20074381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20074381&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-4.5.891.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Estimation of principal components and related models by iterative least squares</span>, pp. <span class="cit-fpage">391</span>–<span class="cit-lpage">420</span> in <span class="cit-source">Multivariate Analysis</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Krishnaiah</span>  <span class="cit-name-given-names">P. R.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.891.44" data-doi="10.1371/journal.pone.0010780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimball</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic diversity and introgression in <em>O. Sativa</em> reveal the impact of domestication and breeding on the rice genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e10780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De10780%26rft.epage%253De10780%26rft.atitle%253DGenomic%2Bdiversity%2Band%2Bintrogression%2Bin%2BO.%2Bsativa%2Breveal%2Bthe%2Bimpact%2Bof%2Bdomestication%2Band%2Bbreeding%2Bon%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010780%26rft_id%253Dinfo%253Apmid%252F20520727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20520727&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/5/891.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/885" title="Impact of Genetic Background on Neonatal Lethality of Gga2 Gene-Trap Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/901" title="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/5"><span>Volume 4 Issue 5, May 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/5.cover-source.jpg?itok=AAKWoOtu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/891" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZVCjtF1eUGACDZeQagvJNxtAnK7MNc56o9x8LdA-F-o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/5/891" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/5/891" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/5/891" class="active" data-icon-position="" data-hide-link-title="0">Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459862" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Uynwn6G2jQHhymn9oQQk-m70TeTuHlQiirlw7Leevsw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/891.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/5/891" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-SFwzTdCE1Ne5WDKt-aQuO_sf_i2TPSgBmLxatNd6E04" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/5/891" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/891.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459862" id="node459862--2" data-pisa="ggg;4/5/891" data-pisa-master="ggg;g3.114.010942" data-apath="/ggg/4/5/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yong-Bi Fu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-900;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010942 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459862--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong-Bi Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research Centre, Agriculture and Agri-Food Canada, Saskatoon, SK S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong-Bi%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong-Bi%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yong-bi.fu@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">yong-bi.fu@agr.gc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/5/891" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/5/891&amp;title=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/5/891&amp;title=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/5/891&amp;title=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/5/891&amp;text=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/5/891&amp;title=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/5/891&amp;t=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/5/891&amp;title=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/5/891&amp;title=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459862/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459862" id="node459862--3" data-pisa="ggg;4/5/891" data-pisa-master="ggg;g3.114.010942" data-apath="/ggg/4/5/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yong-Bi Fu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-900;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010942 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459862--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong-Bi Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research Centre, Agriculture and Agri-Food Canada, Saskatoon, SK S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong-Bi%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong-Bi%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yong-bi.fu@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">yong-bi.fu@agr.gc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=891&amp;rft.epage=900&amp;rft.atitle=Genetic%20Diversity%20Analysis%20of%20Highly%20Incomplete%20SNP%20Genotype%20Data%20with%20Imputations%3A%20An%20Empirical%20Assessment&amp;rft.volume=4&amp;rft.issue=5&amp;rft.date=2014-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fu%2C+Yong-Bi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459862/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459862/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459862/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459862/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459862/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459862/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459862/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459862/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459862/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459862/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459862/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459862/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/5/891" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/5/891.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/5/891.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/5/891.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/5/891.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
