<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/5/871" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/5/871.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/5/871.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/5/871.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/5/871.atom" />
<meta name="HW.pisa" content="ggg;4/5/871" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms" />
<meta name="DC.Identifier" content="10.1534/g3.114.010249" />
<meta name="DC.Date" content="2014-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Zhao et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Messenger RNA 3′-end formation is an essential posttranscriptional processing step for most eukaryotic genes. Different from plants and animals where AAUAAA and its variants routinely are found as the main poly(A) signal, Chlamydomonas reinhardtii uses UGUAA as the major poly(A) signal. The advance of sequencing technology provides an enormous amount of sequencing data for us to explore the variations of poly(A) signals, alternative polyadenylation (APA), and its relationship with splicing in this algal species. Through genome-wide analysis of poly(A) sites in C. reinhardtii , we identified a large number of poly(A) sites: 21,041 from Sanger expressed sequence tags, 88,184 from 454, and 195,266 from Illumina sequence reads. In comparison with previous collections, more new poly(A) sites are found in coding sequences and intron and intergenic regions by deep-sequencing. Interestingly, G-rich signals are particularly abundant in intron and intergenic regions. The prevalence of different poly(A) signals between coding sequences and a 3′-untranslated region implies potentially different polyadenylation mechanisms. Our data suggest that the APA occurs in about 68% of C. reinhardtii genes. Using Gene Ontolgy analysis, we found most of the APA genes are involved in RNA regulation and metabolic process, protein synthesis, hydrolase, and ligase activities. Moreover, intronic poly(A) sites are more abundant in constitutively spliced introns than retained introns, suggesting an interplay between polyadenylation and splicing. Our results support that APA, as in higher eukaryotes, may play significant roles in increasing transcriptome diversity and gene expression regulation in this algal species. Our datasets also provide useful information for accurate annotation of transcript ends in C. reinhardtii ." />
<meta name="DC.Contributor" content="Zhixin Zhao" />
<meta name="DC.Contributor" content="Xiaohui Wu" />
<meta name="DC.Contributor" content="Praveen Kumar Raj Kumar" />
<meta name="DC.Contributor" content="Min Dong" />
<meta name="DC.Contributor" content="Guoli Ji" />
<meta name="DC.Contributor" content="Qingshun Quinn Li" />
<meta name="DC.Contributor" content="Chun Liang" />
<meta name="article:published_time" content="2014-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Messenger RNA 3′-end formation is an essential posttranscriptional processing step for most eukaryotic genes. Different from plants and animals where AAUAAA and its variants routinely are found as the main poly(A) signal, &lt;i&gt;Chlamydomonas reinhardtii&lt;/i&gt; uses UGUAA as the major poly(A) signal. The advance of sequencing technology provides an enormous amount of sequencing data for us to explore the variations of poly(A) signals, alternative polyadenylation (APA), and its relationship with splicing in this algal species. Through genome-wide analysis of poly(A) sites in &lt;i&gt;C. reinhardtii&lt;/i&gt;, we identified a large number of poly(A) sites: 21,041 from Sanger expressed sequence tags, 88,184 from 454, and 195,266 from Illumina sequence reads. In comparison with previous collections, more new poly(A) sites are found in coding sequences and intron and intergenic regions by deep-sequencing. Interestingly, G-rich signals are particularly abundant in intron and intergenic regions. The prevalence of different poly(A) signals between coding sequences and a 3′-untranslated region implies potentially different polyadenylation mechanisms. Our data suggest that the APA occurs in about 68% of &lt;i&gt;C. reinhardtii&lt;/i&gt; genes. Using Gene Ontolgy analysis, we found most of the APA genes are involved in RNA regulation and metabolic process, protein synthesis, hydrolase, and ligase activities. Moreover, intronic poly(A) sites are more abundant in constitutively spliced introns than retained introns, suggesting an interplay between polyadenylation and splicing. Our results support that APA, as in higher eukaryotes, may play significant roles in increasing transcriptome diversity and gene expression regulation in this algal species. Our datasets also provide useful information for accurate annotation of transcript ends in &lt;i&gt;C. reinhardtii&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/05/01" />
<meta name="citation_mjid" content="ggg;4/5/871" />
<meta name="citation_id" content="4/5/871" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/5/871" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/5/871.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/5/871.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/5/871.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010249" />
<meta name="citation_pmid" content="24626288" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="871" />
<meta name="citation_lastpage" content="883" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhixin Zhao" />
<meta name="citation_author_institution" content="Department of Biology, Miami University, Oxford, Ohio 45056" />
<meta name="citation_author" content="Xiaohui Wu" />
<meta name="citation_author_institution" content="Department of Biology, Miami University, Oxford, Ohio 45056" />
<meta name="citation_author_institution" content="Department of Automation, Xiamen University, Xiamen, 361005, China" />
<meta name="citation_author" content="Praveen Kumar Raj Kumar" />
<meta name="citation_author_institution" content="Department of Biology, Miami University, Oxford, Ohio 45056" />
<meta name="citation_author" content="Min Dong" />
<meta name="citation_author_institution" content="Department of Biology, Miami University, Oxford, Ohio 45056" />
<meta name="citation_author_institution" content="Department of Automation, Xiamen University, Xiamen, 361005, China" />
<meta name="citation_author" content="Guoli Ji" />
<meta name="citation_author_institution" content="Department of Automation, Xiamen University, Xiamen, 361005, China" />
<meta name="citation_author" content="Qingshun Quinn Li" />
<meta name="citation_author_institution" content="Department of Biology, Miami University, Oxford, Ohio 45056" />
<meta name="citation_author_institution" content="Key Laboratory of the Ministry of Education for Coastal and Wetland Ecosystems, and College of the Environment and Ecology, Xiamen University, Xiamen, Fujian, China 361102" />
<meta name="citation_author_institution" content="Rice Research Institute, Fujian Academy of Agricultural Sciences, Fuzhou, Fujian, China 350003" />
<meta name="citation_author_email" content="liangc@miamioh.edu" />
<meta name="citation_author_email" content="liq@miamioh.edu" />
<meta name="citation_author" content="Chun Liang" />
<meta name="citation_author_institution" content="Department of Biology, Miami University, Oxford, Ohio 45056" />
<meta name="citation_author_email" content="liangc@miamioh.edu" />
<meta name="citation_author_email" content="liq@miamioh.edu" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Campbell;citation_author=B. J. Haas;citation_author=J. P. Hamilton;citation_author=S. M. Mount;citation_author=C. R. Buell;citation_title=Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis.;citation_pages=327-327;citation_volume=7;citation_year=2006;citation_pmid=17194304;citation_doi=10.1186/1471-2164-7-327" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=R. Elkon;citation_author=A. P. Ugalde;citation_author=R. Agami;citation_title=Alternative cleavage and polyadenylation: extent, regulation and function.;citation_pages=496-506;citation_volume=14;citation_year=2013;citation_pmid=23774734;citation_doi=10.1038/nrg3482" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=D. C. Di Giammartino;citation_author=K. Nishida;citation_author=J. L. Manley;citation_title=Mechanisms and consequences of alternative polyadenylation.;citation_pages=853-866;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21925375;citation_doi=10.1016/j.molcel.2011.08.017" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. H. Graber;citation_author=C. R. Cantor;citation_author=S. C. Mohr;citation_author=T. F. Smith;citation_title=In silico detection of control signals: mRNA 3'-end-processing sequences in diverse species;citation_pages=14055-14060;citation_volume=96;citation_year=1999;citation_issue=24;citation_pmid=10570197;citation_doi=10.1073/pnas.96.24.14055" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=J. Hu;citation_author=C. S. Lutz;citation_author=J. Wilusz;citation_author=B. Tian;citation_title=Bioinformatic identification of candidate cis-regulatory elements involved in human mRNA polyadenylation.;citation_volume=11;citation_year=2005;citation_pmid=16131587;citation_doi=10.1261/rna.2107305" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. G. Hunt;citation_author=D. Xing;citation_author=Q. Q. Li;citation_title=Plant polyadenylation factors: conservation and variety in the polyadenylation complex in plants.;citation_pages=641-641;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23167306;citation_doi=10.1186/1471-2164-13-641" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=Y. Jin;citation_author=T. Bian;citation_title=Nontemplated nucleotide addition prior to polyadenylation: a comparison of Arabidopsis cDNA and genomic sequences.;citation_volume=10;citation_year=2004;citation_pmid=15388874;citation_doi=10.1261/rna.7610404" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Labadorf;citation_author=A. Link;citation_author=M. F. Rogers;citation_author=J. Thomas;citation_author=A. S. Reddy;citation_title=Genome-wide analysis of alternative splicing in Chlamydomonas reinhardtii.;citation_pages=114-114;citation_volume=11;citation_year=2010;citation_pmid=20163725;citation_doi=10.1186/1471-2164-11-114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Liang;citation_author=Y. Liu;citation_author=L. Liu;citation_author=A. C. Davis;citation_author=Y. Shen;citation_title=Expressed Sequence Tags With cDNA Termini: Previously Overlooked Resources for Gene Annotation and Transcriptome Exploration in Chlamydomonas reinhardtii;citation_pages=83-93;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493042;citation_doi=10.1534/genetics.107.085605" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. C. Loke;citation_author=E. A. Stahlberg;citation_author=D. G. Strenski;citation_author=B. J. Haas;citation_author=P. C. Wood;citation_title=Compilation of mRNA polyadenylation signals in Arabidopsis revealed a new signal element and potential secondary structures.;citation_volume=138;citation_year=2005;citation_pmid=15965016;citation_doi=10.1104/pp.105.060541" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=F. Lopez;citation_author=S. Granjeaud;citation_author=T. Ara;citation_author=B. Ghattas;citation_author=D. Gautheret;citation_title=The disparate nature of “intergenic” polyadenylation sites.;citation_volume=12;citation_year=2006;citation_pmid=16931874;citation_doi=10.1261/rna.136206" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Ma;citation_author=C. Guo;citation_author=Q. Q. Li;citation_title=Role of alternative polyadenylation in epigenetic silencing and antisilencing.;citation_volume=111;citation_year=2014;citation_pmid=24335801;citation_doi=10.1073/pnas.1321025111" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=W. Mages;citation_author=B. Cresnar;citation_author=J. F. Harper;citation_author=M. Bruderlein;citation_author=R. Schmitt;citation_title=Volvox carteri alpha 2- and beta 2-tubulin-encoding genes: regulatory signals and transcription.;citation_pages=47-54;citation_volume=160;citation_year=1995;citation_issue=1;citation_pmid=7628715;citation_doi=10.1016/0378-1119(95)00178-9" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Mangone;citation_author=A. P. Manoharan;citation_author=D. Thierry-Mieg;citation_author=J. Thierry-Mieg;citation_author=T. Han;citation_title=The landscape of C. elegans 3′UTRs.;citation_volume=329;citation_year=2010;citation_pmid=20522740;citation_doi=10.1126/science.1191244" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=E. R. Mardis;citation_title=The impact of next-generation sequencing technology on genetics.;citation_pages=133-141;citation_volume=24;citation_year=2008;citation_issue=3;citation_pmid=18262675;citation_doi=10.1016/j.tig.2007.12.007" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Margulies;citation_author=M. Egholm;citation_author=W. E. Altman;citation_author=S. Attiya;citation_author=J. S. Bader;citation_title=Genome sequencing in microfabricated high-density picolitre reactors.;citation_pages=376-380;citation_volume=437;citation_year=2005;citation_issue=7057;citation_pmid=16056220;citation_doi=10.1038/nature03959" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. S. Merchant;citation_author=S. E. Prochnik;citation_author=O. Vallon;citation_author=E. H. Harris;citation_author=S. J. Karpowicz;citation_title=The Chlamydomonas Genome Reveals the Evolution of Key Animal and Plant Functions;citation_pages=245-250;citation_volume=318;citation_year=2007;citation_issue=5848;citation_pmid=17932292;citation_doi=10.1126/science.1143609" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Miura;citation_author=S. Shenker;citation_author=C. Andreu-Agullo;citation_author=J. O. Westholm;citation_author=E. C. Lai;citation_title=Widespread and extensive lengthening of 3′ UTRs in the mammalian brain.;citation_volume=23;citation_year=2013;citation_pmid=23520388;citation_doi=10.1101/gr.146886.112" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. O. Oyola;citation_author=T. D. Otto;citation_author=Y. Gu;citation_author=G. Maslen;citation_author=M. Manske;citation_title=Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes.;citation_pages=1-1;citation_volume=13;citation_year=2012;citation_pmid=22214261;citation_doi=10.1186/1471-2164-13-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Ozsolak;citation_author=A. R. Platt;citation_author=D. R. Jones;citation_author=J. G. Reifenberger;citation_author=L. E. Sass;citation_title=Direct RNA sequencing.;citation_pages=814-818;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19776739;citation_doi=10.1038/nature08390" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. Ozsolak;citation_author=P. Kapranov;citation_author=S. Foissac;citation_author=S. W. Kim;citation_author=E. Fishilevich;citation_title=Comprehensive polyadenylation site maps in yeast and human reveal pervasive alternative polyadenylation.;citation_pages=1018-1029;citation_volume=143;citation_year=2010;citation_issue=6;citation_pmid=21145465;citation_doi=10.1016/j.cell.2010.11.020" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=H. M. Rothnie;citation_title=Plant mRNA 3'-end formation.;citation_pages=43-61;citation_volume=32;citation_year=1996;citation_issue=1-2;citation_pmid=8980473;citation_doi=10.1007/BF00039376" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=F. Sanger;citation_author=A. R. Coulson;citation_title=A rapid method for determining sequences in DNA by primed synthesis with DNA polymerase.;citation_pages=441-448;citation_volume=94;citation_year=1975;citation_issue=3;citation_pmid=1100841;citation_doi=10.1016/0022-2836(75)90213-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=F. Sanger;citation_author=S. Nicklen;citation_author=A. R. Coulson;citation_title=DNA sequencing with chain-terminating inhibitors;citation_pages=5463-5467;citation_volume=74;citation_year=1977;citation_issue=12;citation_pmid=271968;citation_doi=10.1073/pnas.74.12.5463" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. C. Schuster;citation_title=Next-generation sequencing transforms today's biology.;citation_pages=16-18;citation_volume=5;citation_year=2008;citation_issue=1;citation_pmid=18165802;citation_doi=10.1038/nmeth1156" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Shen;citation_author=Y. Liu;citation_author=L. Liu;citation_author=C. Liang;citation_author=Q. Q. Li;citation_title=Unique Features of Nuclear mRNA Poly(A) Signals and Alternative Polyadenylation in Chlamydomonas reinhardtii;citation_pages=167-176;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493049;citation_doi=10.1534/genetics.108.088971" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Shen;citation_author=G. Ji;citation_author=B. J. Haas;citation_author=X. Wu;citation_author=J. Zheng;citation_title=Genome level analysis of rice mRNA 3′-end processing signals and alternative polyadenylation.;citation_volume=36;citation_year=2008;citation_pmid=18411206;citation_doi=10.1093/nar/gkn158" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Shen;citation_author=R. C. Venu;citation_author=K. Nobuta;citation_author=X. Wu;citation_author=V. Notibala;citation_title=Transcriptome dynamics through alternative polyadenylation in developmental and environmental responses in plants revealed by deep sequencing.;citation_volume=21;citation_year=2011;citation_pmid=21813626;citation_doi=10.1101/gr.114744.110" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. Sherstnev;citation_author=C. Duc;citation_author=C. Cole;citation_author=V. Zacharaki;citation_author=C. Hornyik;citation_title=Direct sequencing of Arabidopsis thaliana RNA reveals patterns of cleavage and polyadenylation.;citation_pages=845-852;citation_volume=19;citation_year=2012;citation_issue=8;citation_pmid=22820990;citation_doi=10.1038/nsmb.2345" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=P. Singh;citation_author=T. L. Alley;citation_author=S. M. Wright;citation_author=S. Kamdar;citation_author=W. Schott;citation_title=Global changes in processing of mRNA 3′ untranslated regions characterize clinically distinct cancer subtypes.;citation_volume=69;citation_year=2009;citation_pmid=19934316;citation_doi=10.1158/0008-5472.CAN-09-2236" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Stanke;citation_author=M. Diekhans;citation_author=R. Baertsch;citation_author=D. Haussler;citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding.;citation_volume=24;citation_year=2008;citation_pmid=18218656;citation_doi=10.1093/bioinformatics/btn013" />
<meta name="citation_reference" content="citation_journal_title=J. Classification;citation_author=G. J. Szekely;citation_author=M. L. Rizzo;citation_title=Hierarchical clustering via joint between-within distances: extending Ward’s minimum variance method.;citation_pages=151-183;citation_volume=22;citation_year=2005;citation_doi=10.1007/s00357-005-0012-9" />
<meta name="citation_reference" content="citation_title=FASEB J.;citation_author=C. P. Thomas;citation_author=J. I. Andrews;citation_author=K. Z. Liu;citation_title=Intronic polyadenylation signal sequences and alternate splicing generate human soluble Flt1 variants and regulate the abundance of soluble Flt1 in the placenta.;citation_volume=21;citation_year=2007;citation_pmid=17615362;citation_doi=10.1096/fj.07-8809com" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. P. Thomas;citation_author=N. S. Raikwar;citation_author=E. A. Kelley;citation_author=K. Z. Liu;citation_title=Alternate processing of Flt1 transcripts is directed by conserved cis-elements within an intronic region of FLT1 that reciprocally regulates splicing and polyadenylation.;citation_volume=38;citation_year=2010;citation_pmid=20385595;citation_doi=10.1093/nar/gkq198" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=P. E. Thomas;citation_author=X. Wu;citation_author=M. Liu;citation_author=B. Gaffney;citation_author=G. Ji;citation_title=Genome-wide control of polyadenylation site choice by CPSF30 in Arabidopsis.;citation_volume=24;citation_year=2012;citation_pmid=23136375;citation_doi=10.1105/tpc.112.096107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. Tian;citation_author=J. Hu;citation_author=H. Zhang;citation_author=C. S. Lutz;citation_title=A large-scale analysis of mRNA polyadenylation of human and mouse genes;citation_pages=201-212;citation_volume=33;citation_year=2005;citation_issue=1;citation_pmid=15647503;citation_doi=10.1093/nar/gki158" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Tian;citation_author=Z. Pan;citation_author=J. Y. Lee;citation_title=Widespread mRNA polyadenylation events in introns indicate dynamic interplay between polyadenylation and splicing.;citation_volume=17;citation_year=2007;citation_pmid=17210931;citation_doi=10.1101/gr.5532707" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. van Helden;citation_author=B. André;citation_author=J. Collado-Vides;citation_title=Extracting regulatory sites from the upstream region of yeast genes by computational analysis of oligonucleotide frequencies.;citation_pages=827-842;citation_volume=281;citation_year=1998;citation_issue=5;citation_pmid=9719638;citation_doi=10.1006/jmbi.1998.1947" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=S. Wodniok;citation_author=A. Simon;citation_author=G. Glockner;citation_author=B. Becker;citation_title=Gain and loss of polyadenylation signals during evolution of green algae.;citation_pages=65-65;citation_volume=7;citation_year=2007;citation_pmid=17442103;citation_doi=10.1186/1471-2148-7-65" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Wu;citation_author=M. Liu;citation_author=B. Downie;citation_author=C. Liang;citation_author=G. Ji;citation_title=Genome-wide landscape of polyadenylation in Arabidopsis provides evidence for extensive alternative polyadenylation.;citation_volume=108;citation_year=2011;citation_pmid=21746925;citation_doi=10.1073/pnas.1019732108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Xia;citation_author=V. MacKay;citation_author=X. Yao;citation_author=J. Wu;citation_author=F. Miura;citation_title=Translation initiation: a regulatory role for poly(A) tracts in front of the AUG codon in Saccharomyces cerevisiae.;citation_volume=189;citation_year=2011;citation_pmid=21840854;citation_doi=10.1534/genetics.111.132068" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=D. Xing;citation_author=Q. Q. Li;citation_title=Alternative polyadenylation and gene expression regulation in plants. Wiley interdisciplinary reviews.;citation_pages=445-458;citation_volume=2;citation_year=2011;citation_pmid=21957029" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=J. Zhao;citation_author=L. Hyman;citation_author=C. Moore;citation_title=Formation of mRNA 3' Ends in Eukaryotes: Mechanism, Regulation, and Interrelationships with Other Steps in mRNA Synthesis;citation_pages=405-445;citation_volume=63;citation_year=1999;citation_issue=2;citation_pmid=10357856" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Q. Zheng;citation_author=X.-J. Wang;citation_title=GOEAST: a web-based software toolkit for Gene Ontology enrichment analysis.;citation_pages=W358363;citation_volume=36;citation_year=2008" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="twitter:description" content="Messenger RNA 3′-end formation is an essential posttranscriptional processing step for most eukaryotic genes. Different from plants and animals where AAUAAA and its variants routinely are found as the main poly(A) signal, Chlamydomonas reinhardtii uses UGUAA as the major poly(A) signal. The advance of sequencing technology provides an enormous amount of sequencing data for us to explore the variations of poly(A) signals, alternative polyadenylation (APA), and its relationship with splicing in this algal species. Through genome-wide analysis of poly(A) sites in C. reinhardtii , we identified a large number of poly(A) sites: 21,041 from Sanger expressed sequence tags, 88,184 from 454, and 195,266 from Illumina sequence reads. In comparison with previous collections, more new poly(A) sites are found in coding sequences and intron and intergenic regions by deep-sequencing. Interestingly, G-rich signals are particularly abundant in intron and intergenic regions. The prevalence of different poly(A) signals between coding sequences and a 3′-untranslated region implies potentially different polyadenylation mechanisms. Our data suggest that the APA occurs in about 68% of C. reinhardtii genes. Using Gene Ontolgy analysis, we found most of the APA genes are involved in RNA regulation and metabolic process, protein synthesis, hydrolase, and ligase activities. Moreover, intronic poly(A) sites are more abundant in constitutively spliced introns than retained introns, suggesting an interplay between polyadenylation and splicing. Our results support that APA, as in higher eukaryotes, may play significant roles in increasing transcriptome diversity and gene expression regulation in this algal species. Our datasets also provide useful information for accurate annotation of transcript ends in C. reinhardtii ." />
<meta name="og-title" property="og:title" content="Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/5/871" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Messenger RNA 3′-end formation is an essential posttranscriptional processing step for most eukaryotic genes. Different from plants and animals where AAUAAA and its variants routinely are found as the main poly(A) signal, Chlamydomonas reinhardtii uses UGUAA as the major poly(A) signal. The advance of sequencing technology provides an enormous amount of sequencing data for us to explore the variations of poly(A) signals, alternative polyadenylation (APA), and its relationship with splicing in this algal species. Through genome-wide analysis of poly(A) sites in C. reinhardtii , we identified a large number of poly(A) sites: 21,041 from Sanger expressed sequence tags, 88,184 from 454, and 195,266 from Illumina sequence reads. In comparison with previous collections, more new poly(A) sites are found in coding sequences and intron and intergenic regions by deep-sequencing. Interestingly, G-rich signals are particularly abundant in intron and intergenic regions. The prevalence of different poly(A) signals between coding sequences and a 3′-untranslated region implies potentially different polyadenylation mechanisms. Our data suggest that the APA occurs in about 68% of C. reinhardtii genes. Using Gene Ontolgy analysis, we found most of the APA genes are involved in RNA regulation and metabolic process, protein synthesis, hydrolase, and ligase activities. Moreover, intronic poly(A) sites are more abundant in constitutively spliced introns than retained introns, suggesting an interplay between polyadenylation and splicing. Our results support that APA, as in higher eukaryotes, may play significant roles in increasing transcriptome diversity and gene expression regulation in this algal species. Our datasets also provide useful information for accurate annotation of transcript ends in C. reinhardtii ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
    <title>Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"v8dn3-KyUIJrRYA3uhGKSke87nWdp_OIet0Az21OET0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459865","apath":"\/ggg\/4\/5\/871.atom","pisa":"ggg;4\/5\/871","ac":{"\/ggg\/4\/5\/871.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/5\/871.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/5\/871"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/5\/871"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/5\/871.figures-only":true,"\/content\/4\/5\/871":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ec7G3hx2QtTPu5UXDdWAUBOqte8eb2rvB8KdASv6lFpRjehyYbF4WQGhVMU\\\\\\\/S7Zq4EnlnRIB5KZVSF1fmmogoXmV6fjNYqt3+vjkxGY98C49TmIGtTzlUGcuSzJmSYGoqyc79G\\\\\\\/jISDTOljXsfbhZ0bJ1fdVC6\\\\\\\/IZDZPItXdJ2E22EIHEbLvSo8zyXznSfsgzrs13x0ZZP5riwf8oyD\\\\\\\/v96CsmBJOkMVL6hDph0\\\\\\\/k+AfZCgvSx0a1Xe9D3Qbwmt9969aJYJ1CO6Bd1pLIRbdYw5V3LfYI31eSBnbnkpqZGOxZIxgxRqUd8fkIPVjsu5Af2lIj2HO\\\\\\\/ARYjAZRsJnMe1QCKCh0sGGsBRPMeSfHpOn7M3HIKQJWQ8ondhYiob9TV1ZJBaUO92I6br9XLScAR6hfXEyB4lM0XGqI2lrfdCJXe3C6a15NYz6mhqy+2gv2ZBzMeLfKOR1TGEckiedo2iSPS29L12MUSVF4OYUiZgP5oNG0Y4NEmD1NKyT6mGFdwn\\\\\\\/0jk3HmIVi1ciCakR6JqC6uafquG3Xry3VW1RLWDUc4P7eeW6nM2jb3NrjSazji81NzhClQ8xXK4V6SNtCa3cnAaDH3mOoTAettyunn3bq3rVkghsWW4M68LOM8g7OHWjZbGGU6v2fmE4qGwtTXodZvfiZSfZwoMD5FKsr4uMOQCK00rpbvMt4B4S61afTPhqyAAZBkVf0YGUK3LQ4BuVIn8+Fw938ieC8fAjzsqpSHgQb4G8ejPJfHaxg2ooPWvQQRRgo4ba01MaCdFN\\\\\\\/Y8e4An2Blb73Xxo68a2NidmsUpsWEZ1mxQdaPsIfluv9bln0osJ1+vNfWWC5a4PGfAXeM+35kJmyydt\\\\\\\/SwDwmLlLCHKdRIVidA02Dm+g3nGrsMCxZk4cQvq7CN93fH00Pi35kW1LMWhsWrUiV4G3zNB9VNeTo6nf9U\\\\\\\/Uz0FbFgKt3gwEqL+vlfcUynCQSFGESdBTjo\\\\\\\/2xxVwNgOvJImsW0X4hP9nIX5i22efN3K8BOljTJ\\\\\\\/76xRqu8Kv1A1lNn0feKZmhVFhWKCD2CXLQiAtF2P9aXWnopteYMWtEfFNZYrh0dfLv7tlG1qWTZeH6GYZ6JTm1Kl5GHkKGiqv4tEvPvyJkhnGMnGWTSnVw9iJ4gz2vI2vY\\\\\\\/LHCBtqZoFL4lh9\\\\\\\/SwrH8XMdwFWUDZxTUpH5pLLNbvadq++qpns3eGKUMRkTSD8zjJ84khJ9DJ6uuU24cgJpE3P+zqHVia7FHV0X0lYB1PJTUyYbrQvrcs3+gji\\\u0022,\\\u0022iv\\\u0022:\\\u00223ODxCgFfoIWXFWeA2m3YNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f9ff3f407cb6e36f62b200ed50a88a8d21ffa5464f7c7925119185dc6507603e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rgbExKBcGkG0EUVCx1LP85eIXogsKZeGn4voJ6As2n7mW+5dIp\\\\\\\/ikH6+Iven2vTBqylfYTisw84vgAr83fiG8p8z3xruRsIoyoLrt1qBdxq9PNQimpfxfazI4gYfS7USJkw5CWZAWu71GsIVTDXZmRFsGqcBUOMicwmj3u77+UrVmeO8Cbvkbv3MT3PiCQ8\\\\\\\/QW6wiyg\\\\\\\/jMBbix6m3EpF9fy2ZB7lZsONdvDm1RW1f02oDHkD1ER9yJ23rUu3VDdC\\\\\\\/n98TUUYIO0lwkv17W902pHqfWYXt3uJdguwc7xi6i8eOU5iqKPfpXb3Y6BStqdPeqWRFFbqQkHAcQnP38g5AbBjP6gvgu1vfjnecp5FqrX+R9ozJXC4AjOueso91iXabPxxULBy8h+buQjTuTSgo4l3aenZQKHPBc5cqyHgnyIfxbPSK4nS2gS1mZRBGjdtQjHyibcz8baRNWbI8Yy\\\\\\\/zohjvjh27DEGzhT4L6wxUb2x57JwaBodCiCCSqGApH59SvaYs9Om8N6gHi+anZIIPZRbA5C4kBxZEp8JVIwcXaaJAb\\\\\\\/fOf+xVVFfCrF2VE27ECLks3XIVEQylMkJ2tS290YlVcslq0yHzlheRblsrfgm0mtuPJcZjSX7wE+Yusa3cYbDik2rUlhxkaOh4pHs97pcD8OPh+AOAh\\\\\\\/tRxi\\\\\\\/dSzIToplVbb6fqp5WQJjdzca8BzEwX4QYYizbw00+E96Rs5OAp6kC911KmeSEIbpQncL1aRjmJWHYK8ARgMWw4oQbE08bVtQ1GktI\\\\\\\/BmchPFbgX2PydYsUF8ol1eK3uSuw4xhcJp1W5nA0rqwOTy3zjzQtbJUXBhfK6E2kXu8ly34eW+yLhaI1Kpk49SwiJBafMHD+KWvXRSw5FRR3CXZ\\\\\\\/ZfoTPXhJPfZwg\\\\\\\/hpIlJI6MznFBZHJbTj1JB1era+S5VZI0W1IUxgbtldHH32oMqnihTg8Fnlyl1g5f9+vhrAreJCYp1MJ9uMSx9\\\\\\\/+QTHerXoAZw\\\\\\\/XeScF\\\\\\\/A+gy309qdyRNedeOTpRAibU\\\\\\\/QI5jqGf5MIIRLiI\\\\\\\/nKzJzANGtG0H+cWdK44NVKGWzNDSefVdm437RO9GuAPo01sbhKDUzSVgRuYeH1D\\\\\\\/G6sPPVS9+V1lzDTthjSh5CcHXBL9RQc\\\\\\\/I+PSrrBXQg\\\\\\\/X4SR\\\\\\\/v2Z5EW4XOoc0NFL9yWhSJzRPfqGaZeLjblxr5BT8D61LrMsgmuPD93qpTJgEBMTzEOVJnOSHQ6SDYKbuinqdq\\\\\\\/laNtN\\\\\\\/qTuv5GP01ABXBmTJHXMV\\\\\\\/teYsu\\\\\\\/Jpnz31Dli4tr3z+mla5ZOj3eneb16SY\\\\\\\/IauWkPTBSMBviFC2YRgngE\\\\\\\/8ckNB3u9db3jL1HjNtnpFxUpb9t5DstROUYpe9rUOwws+G8c4T6UBTnBBSEg2L1iT0qd7MWnm\\\\\\\/pEhgpi+2cH9m7RbsOnRCR4o+4cxVLWlaEoFPHnva6\\\\\\\/qeRepbpmMqB0Kl64fSiIBOepl2QAjrZ9T4OxCwMMvQ1aYSn2jGAITS2tnTF3jnu4t3ANz3YVmxEDRykfQeK2Sj8MvAXVmCVkEnmFCYluKWURv6zLcygPIwdQaEXOuYMYQSnQVRttvipdBRLv6ZvmxQEQq1br0M\\\\\\\/WUu4nYiccuWp0rdSNarWwOZV1Ainjdijs7TZQn4i\\\\\\\/8woBVtbfymXMqzC5AuWWF61\\\\\\\/Qq9e6+ErwR9IxFwp7HtuDs1rKDmDyFGLnVW1d9qPxPl+YjO8R1tnH+RyvjN6ve2KQY+XE+8sYZB1wkt8LsddXFBVhz6YaFoZCa2iLJr6Ml68K1UxUQxrmOQ989WM6OpyWaaMCyP3rhnAhQx2qqqRCZb68dFZime6TCQwOADKW20wL+viAg92fJ3StAY8SIH3HOG9tVhcbyAaff3TGwdMJwNhjnoj4Vfr\\\\\\\/2COFyi1RAoUntYWDSZdNmQsip2ltd5I3McKvThL8vsjSGDO7kePGuyG4ln3Evtrp0lYnISt8uDWwFN+zaBagsYrpAzYx8BTt54ZRe5I6KNxQyHt\\\\\\\/4At9HSxvobeCaJ7HYiz3prUN2tZHImRPeTGt1lssY9UJdXXL2CAsZ7s1fyodpDXzToE2x45f7fA69opdgxgnKGCQN9kW6Y\\\\\\\/vzRi3XUMepcK7WHF7dNcrxdajE8SzHutHZW1eoG+BB6NS\\\\\\\/TzAzvXdUugo2kHVYf1h9WX\\\\\\\/7WQhdWRiSObJUIHIBzFiv2VwLgkjXkVGSquQXkCmqg4M8XTGN3kwdI8ciXkGK9Sw3k9A9EuFsE8JmmhO6fe6dO3fh3SWnfA1brPNLJlumL86j\\\\\\\/gqsujVSRcfVGh0tSnNzb\\\\\\\/F1sudFre72JiJaofcidnG7Vne\\\\\\\/nlqgdRh8qrdfvTClgkcUQfkCD0M7J9edQHm\\\\\\\/rcmytBkfIpCNo4n\\\\\\\/kk9hnaeCHUyjRib6S7Jv8Lw95mHEjM7kBhhJSX3og+2uVasAebTx+HV04xzP7obxAC\\\\\\\/ubeeIlvWhc+PyqnVcmGJYVmSyFsRBh3RVVsS2wlB0\\\\\\\/lMMfwCTl5GZaCkYgR0ORIJcHnUghSWaslW\\\\\\\/JBX9gga3zUrHIDZVztdwSM3RJmai3TbrllYwb0uG9HmzMu4vl1sTWDiFg+9OQ6lryom2PFLztWtFJViVw+ABuQbYIVptyd38OwAyP5GH0iiT\\\\\\\/TsA+GbFCsrh1T2R1AeClBNmQXoA8fuwwOBRFezIaceSMtATXUwXFLpNkNlK3c2KP80EzSc3DDti8G3ofu2GFnv1RP6jnBJ+2RrOYuCr2X+TQzy\\\\\\\/jqcPbhG5q8w2rFHcjzDYbYmgS2XvRu0DkAu9kGGUD8E22VKr1JP08XQ2YquankGTNGMS6b1pm985M\\\\\\\/yiUlnlM8jFmW5suyUHnu0QXR33EHr7i9oa5c+ixx8eBzBhWTm6GVq9XzVnjgqBzqJEjT2yF+D7I5THLMvHsAQxvhWDDg6MxGftt5QbEMOKSXMmTGa\\\\\\\/8sMyx7cNcE8ER7sJLhYss1Bh3IuhEME00fwqPzR6k28bAuiEIFcDRWvcfDe9c55FD+QxaSbKjERtyyIsdiy\\\\\\\/T2xi2oQSUxS5O0qR13OUyzbdOA9GG12dtTV1\\\\\\\/5lIEICiQ50PWUvBid\\\\\\\/\\\\\\\/k8b7xjxomVZzVM5h0+\\\\\\\/RWoXFUsnwey3lkdsufJHZEReisLbQAHZqIV3\\\\\\\/c4T70RuwEQDUsvb9bZO\\\\\\\/5RKpglcXH+IQFHKzb8SlT\\\\\\\/oaLE+02qNDzGCSuuE+7ytSZUGBApQVeWT7MxKvVkHm9y\\\\\\\/JE2bnCt5+orgra7DSzVvbU6ZjMlzMAyuUveBg\\\\\\\/C7KmuTYCor0J5k1kCKzCLWI7LWWHchZ\\\\\\\/T8\\\\\\\/in7V0j3qw261OxoRWKmIxvNcbl5g5Ui4sP2zhQLsBduoTarx2421hUNto+j0YR8WQFTsdPVfsX\\\\\\\/95xAr5Ajy0Wj\\\\\\\/K9e\\\\\\\/ucZLzx1+hEh0+WM0yXKBITBkvuzfHtA06YIpMXgsckBTFIGmxBQ07FaX8iFTz9KscTXLYXUXdyuqBsfrrQc5\\\\\\\/cO2jI7WjXeKr3KFOMkmvy4jlnR38FM349rAGpFPbFRAhsddjhCdwMcPsnCMjut\\\\\\\/uR52dD5dKsImhr+nVOHilP+fLVvExKR+4t9E6i56XCqmKbH2deUrBEk7cVUISAwWa7yqle3ZrhOOUs9fvXGaVcSndYpoiduru+tHAxV39XC6mAvX5mu7RI7mixs7ajB9ouuXymAdY2oJtFe8GkvLRuQ24jnGn\\\\\\\/OlM2eXUWNeA32Cmk3IkPmiBcaL060y5GLLrF\\\\\\\/H70jep+ETuvucnzbLiQCOJ81PUJwe5MwQCIbXD4RsbQh18HOLiG\\\\\\\/N6QPBxEpYSYJjSZNXj9ZD4Zxum8+9Chd2N8oFeHvdW9r0L1dejwN88UraJo2ZvvM9t712EoPYPuBa2xJIDGKOBNl+X6xoEjAZXXM1Krw22d12dazOrDMGU3he+G+ZdLpRhpRXA8p0DdJh56+O4NmqFu22xiRYAWsW+aBtbh+gJ\\\\\\\/4n+HT1LJyfoyJk9bmqw\\\\\\\/JpkeaTdpGqOiEqm7SZdvzTpmsXiHmCz+G\\\\\\\/mTGwAtiYo+BYL8pJgyNJvu3Lmq5mOn5+zOF4YU3o1FBi7p00HUDymYszPM0JP8xr1ESu70MKbJMcCujN6yo0iPg8iO+jfo29UH\\\\\\\/DYo7xJ0zxRGD0x0gFAQWvnwbEPOFk3h1QA6a\\\\\\\/MUDqXFODVJnB\\\\\\\/6kcOv+rHGw8rZh9wAKZWwoX8TomAv+E259ZxS3rsRUgLzOCXjzUANcHMLsDRF5TkpvqUvhrwfnQfoF0HgHlfuOg7rpPdCsso4\\\\\\\/kyQrwjMMuvefHvfVBx5GLdGgrU9dY3rJsteUJuRMj5aMYHN9ha4vLV68rwuYqfTxVVmubvJHGMr0LCVxouU2oskzX3lgzW4T\\\\\\\/5lnaQrKe2ZJnJXBOC\\\\\\\/7Y9ClK6hWwpuJaUST9oPMoxkZkuunvqdf6Zee4nLFGbz6d\\\\\\\/mq2v7rMH3aofyjFRMYwcg0Epap\\\\\\\/klOccT\\\\\\\/Hx48exvLWeXASpLpc+iQwCBOWzjO3jsUCIfJE986x\\\\\\\/XwU9B28at6ysKRC4DzNBvMLW\\\\\\\/XTAixp2Fc89uAbhagVm5x0CSEXs1bOIBnGJoAKy9eg+jP9VIIYLTJzxg+9\\\\\\\/\\\\\\\/XnCKbhgN5TWqLcrNcH17yZm\\\\\\\/sQ3U69viZAWCFs+MXrdqsstbVz\\\\\\\/4O6Z2JPfoKcT0uUtzL74oypYE7qrDuphRKBJjkbl7uI7tNQ0v4zIEBn3bhzxJn8zw6uR56QjsJPIMRsC60uZApQuIayc6FiWjgyRLLZpfjZ+xmTlKkd9necAZ\\\\\\\/0+KdcsvZ7NKiNuYABGYU01z1dTyk9sL93bx1yvFSiku8b5GwFTWEPRpW5bTu3m2K8pdj53Z4kp1lyZB5myG6SRqrF0eToPnw7Ix1MDRmsGZu7ROTckkbmXEkQHHyvPIHKUJm7tSAf4FRYmKmZGBH5kX806fN\\\\\\\/Z4\\\\\\\/DWFw8ChTe3F3Q4kV0DbH9FicpJekyVnT2RhKbiZyw9j8pZR\\\\\\\/JtVhtSLdRkBcOCOn2WFCkhWR1n1mnFCFR2WGAffWIBxYde5pXFvjWR7gsU0YGFEcf\\\\\\\/FYfXZwZKJDihfYusIcyONUgBPrmCazEIso2tdJ\\\\\\\/acKSJgrXLDAtAooezhz6pBpleOOH+X93CB0dgbHBwET3sRQ88Dqy0bnM7EtTeevoxD\\\\\\\/FGEKyP+rlcwXJph0\\\\\\\/NrvmISh4mqlQe2qobT8YMfZONm\\\\\\\/5mbjjqQ0H8BezTSZxh30niZRN9oLgD5rdRmS+WmSb0HvsZhInhwVW5Q7hWqm6Zp6\\\\\\\/bWK8fOPFsLDbcXtNiiaLO0MMBzG5uPlII4EUaUf\\\\\\\/9R6H0Q7CIIXelYSHm1i+ggse3SzdQwZc\\\\\\\/uQ5\\\\\\\/7w1+cBT951YQRtCa\\\\\\\/qCPwJLvSTU0abRIEH5rTntli8yqUspaxWM2fAlu4AzRGLMDTDSCHFJCGZ4Bq4DDWGKwzN7bTBVh7icLqgSlbInkzah3tKdpvPDI7PgKBUV3LWEVi7IOeBR\\\\\\\/A5eFa\\\\\\\/ZJMVDAQLXLvDQEYa7Yf2WXtbq4D5Ci15Y9PDn7cTQdT8VDoaA9rg765xSTpPU9wrjRw+wq+1kw3SWHg0w23Np12w9lMuovVdREOdaRcU4bDyiu5HYxc4ZQAG3DzjNnGtoWCbvIKxwtQI+zIKEBI9q\\\\\\\/uuEt\\\\\\\/u0hPravAaTRIl1WQoTmqJy6fjozXj9FKromyYwetZNkKICI+3kys8\\\\\\\/SoHwg+57FVuYy3YNNrriWq4WQMmTVGFALKUzUhaKP11gBPo8aW5\\\\\\\/HOr\\\\\\\/7J9e0Xf5ImPXkHw1cvhoPluK6nrTIXzybmFHEr2ffMXFHdAryvf24J4vjNBfg3faQBQSYZg3id7UKdGP0JCYtV7ngxGn8gmwmaW9T5XEv4kLuADKFjdTpFwgEW0B4BseaOiP7zuyZbz6MbNuoA\\\\\\\/7fHq0T56PvEgUhN6mMgkkvyn2GF8CWf57kY\\\\\\\/0LJpuFZ8c4wVrNt41xVBbR+0Kg20+mXOJiLdpb4CF7\\\\\\\/DGOd38gySXK+ODUWfIEDM9\\\\\\\/dH6CB74PXh+LKN99fLkkh0mV8PCfl0vgbLsOHor8A12nqmh2PiIkUq2mBcJQfZJCIft\\\\\\\/5YJ2SkF8z\\\\\\\/VxKynyep60mUuErMPZk\\\\\\\/eIvoeAbq8rJtbB28E51SguyN1r4EtEZEStghwAX\\\\\\\/Z4uSzs4vh0k5T3qMYcbLWgt+9g3p3c\\\\\\\/E3PFG4JeN1ICXYf8kDyWEnfpYdS0np5W2FdLIC4yNIxLw+t+qVxeNgeZHTuZE39FvQFZWsuTPU\\\\\\\/d75aAFsRX3ij29yI0wuHHFEAZAhzFF4gegwDYt9LnarpODRBRTIjTAWSkpPCX6JVCiBer40bMaujtjnEOBlvSkCnIGLrfpU42irSxhaWvyxeSHIKnSbf5j4qDfdz56xKRybwKgo98vIUfIRJH4JRek1BS6QQ92uoOyRR0EAn5uL6Kbo5StGmMiRn16wzRsw2ZRGjbJBrmwWuGSLZ4RP8C7ozw3lon4rLRqrMRi5PDBPo5sUBd1AOFmwokyvOa0KXJcjBFSfW63zjsmPXb2K4dbt7LYLMXjKYlzAO1pvyIegpiZgvDfy8LbtuypRO49W+W\\\\\\\/ULPCuoBJwsGHaABJoV2kB04X2bRk2Re850MbqbSCs9M+AkXXhbVOkVfxceiSjw9iQiAQlSdmZzgYqWFFvy3Cw5LfcT1kPhcM74Wva\\\\\\\/3DYsVx72pyAON\\\\\\\/G7ueA323xsJRa036Ldl569zrDBrv+CxAz3OvlMF8KxklsvzlepnYtJ7PILB3sG0kyxWnPuY6KOsuf9b8i0ANVq5hHaVvYktVige3\\\\\\\/gXNl43D8cPJI9sG\\\\\\\/h9t7vscbtU1hJPWuFRd3zAH10Sh7vYOhjdq4Lv89PvZBDUr412LrD7UV3VQmP4snyb3aE3WLHwgTNqrh926AVCmAbZ31R09c8APqTB1RZbG5chLQjgHUsrvDOuWfl\\\\\\\/HCEzidlaW2W6Yb9NEQvoRXUyy2uXUbiy6s990FTC4\\\\\\\/cC9A0LSu0hJ1jHyo5S8SDPdugY797OohEMkULMOpm3tPm+q6uOjcdvH8tEV5DLgPoXnq2IrtJZfyXeizXyrtNq7HQKfmehHhpg99WfCC5Mm8J8\\\\\\\/8hSgNKUdP\\\\\\\/bpxCuy3iSa7WhUXuLGw\\\\\\\/+wiiFtX2OQ5NYjkHsb9yCemjU5ueXOIuBo92y+tvhbQkozn1IPuD9WVyXFISqH7VblpzfmBMS0pcxm5aR9WE5TARBG83QG8DhWpVBIveyaUV7OfSQ2yr\\\\\\\/q9waQoD2c4cocGbxXr+3PHKuc4i+JxoVeE0tcbX\\\\\\\/9D5htUUGfEIpfbqqLE6qv5zKdli6g8RnmKCn7O64d\\\\\\\/KXMjhjPmXCrY8kPcdHXQVqPG48E54mrECR6SPjnjimsqJ24M9iognCYLp7Oq8bM7RfABW7VsOogwjGQRknAtAtRcIAjvQHOL8aQhDvq9VyE5ozQrt5GVbOWAs71oQV+F2z83er8Ijw3mWINchEUE3PnTJdXe\\\\\\\/urbNsCtcPLGFcZltGI6lF+32W6UO59kMiEYNXKElZGddB1pRZRp5+KG1qW4XswbiSRoaZJF4J8PSu0JNl11Dgl6Q34vfBPMEWN75OJJVivbsQIWbL94raTLP046VRj6amxTyZmIWYvLbDgnh0VZhPRGAAkYqMZnib1cFUAKMjGhstlWfJ0GjdLDmwpeAiDYRNT6\\\\\\\/DLWv7BtWH2Mh40IBxo8aSIZ1SbTimTY0NdOPdnEXUuQjltoFRccNHynk7\\\\\\\/aFpbInoIa6OrY9S3Chbt3BbLPGCWawHhJvw8t3geUsgeCHYV34bM6bxwcYaHXjdLYyAPvpVrt211bgaw1ykCgdCuw9jeYvMbLfdwqbCuLwU29WEg9n4q+dS1FFwOhwWnrV6cUhaFtpiKZWBacekUi5tuLSogRYRVTlVSrNH8FkicUq0DKF9jpsGu7NGIQhjQFxA5WRKhdDHNix+fBHN8w5HmBE0fNR0xdNeiVl+hE17Ru\\\\\\\/fKemxV8wDRb6LNbKbpwUhzo4Z1Plvxl6iDcWfzGodfgLQEQUf\\\\\\\/6\\\\\\\/RSB+1WHD35NMCWSy8qdUDc2CGab9mXmW32LiMBq2SUYSYsP0kp18S6iIJBJn8bqUf7oY2gewLVhHYt8NsGJJr8dy0gQZ0XtGbDvMYQRoSExHElWdqn2TQzdO1lv9CPQz4ERQBCd32ikMqbKRaBRLFz4vX7P7yEChLVxZvRTqYnNyY7jK5nmDjrjEYepHRNtL\\\\\\\/0BeGCUnNONoPpSCFuzSdiKULIOu1DVq7o1C96jVAqfq1NfadUJuzKKAM+Ts7vGqfAA6RULScurhNCKBNt7PS3xtzkid6Fj4hT6kXXnqVqE3L7l9eky7IoixcRAcPn+Z9qJfiNuMz1XmRDwwHidPrJZcjOkNzgUIDhi77WvLJQPhGU4mZZ5Sfxz3BX7zxbeNHmlZ6AFvE502QhO9nJ+q6XKDgNoFxr2DmiOV0K5gS4R5T6Z1UN1zlCxCGh6niAHIZHPdwnvACNAnj9AoOwEToPlgvJDfGs1OaJQgdH1CvxSM14H5s6Cs9r+kHyf5FhEt7mEEJNx8SmyBsJQ9nyKJ0BxxhQZdVu++1bR0XuunnIzpngRHsrNmIGU1qbDzADKdwj0+T4nP+Svj\\\\\\\/QSz66wzzB0T4cYy7cXAug4XFpLqA\\\\\\\/f1eFagNkQeEdjjT7iPfmdBh33IjJfDuaZ6Tpz0BmSL+1Aua3yrgmmE\\\\\\\/xBEszD9P75Qob3YviPibHXkRptN66h4DXs2h6xrpWxv3ehH6RiH0yuBXMFyQ0+obhbU56sopdEL\\\\\\\/J9Ipn983XcG3QXFjM4IEJASwudFDmpwDfaLnO8B0QiGf6SGTabaK2AAQMbUmmUrxbzww5rUG\\\\\\\/2cThKiTo+xuvLsm3C\\\\\\\/k5c4NKVMX5agysrI7JKdl88D9Ru+AkBgMMKHRaNbbEGYbMddNW11iQf4VTgHjAJCylnCH7G3dMJSO3rgq79AUy\\\\\\\/srqHASMjaW6CGrIRTHyN4MMTD\\\\\\\/wCo09osXElFhWx6Ns1H4ZDsX05Azvd7UoW7gqpjvl3ig5eTOhPLSjUaeWkcXTGb81WJFCMtnpQeMvpK5nSTUcrIL\\\\\\\/PN+7CQlHn1GibYj+rQSDvX6pfxuy0BKeyXqtzvp8yPcc2A1Dy64LY+G0N1S+e\\\\\\\/RDamhxCL7iBp81NLUxcatMBt8qApNPptj5MJbcbW\\\\\\\/vWr87ZpBVgYo4HQCMnK18d+lfFJW9Adxm+F8V8LQg3yH29ILddVno0SWxjC0K+uDr1N2UTpRXodIjFoOqcyxWVCSJPymc6l9iUh4aBdsy+LOSut4z\\\\\\\/An9J7U0r260rNK6LiOiZxylngnUpB2HIRmmV4uiePgOwy+LhbxXAnniSd92n4IDSNJJMpkXzgqmpR5TlH5fRoSFP\\\\\\\/RNCT2+sKY1d7ixyw6FL96xTth6BP5old\\\\\\\/CgEnabp6bkDjMEl93V6ZlqGpXP61\\\\\\\/H2ddQhmWdr1giGRFLkGdc7h5sjClZt3WgcV\\\\\\\/vxBDsmoi4t5CY8VV13Ww7w8UWtwX0M2VAm8Gq7ZmcV9T5tQfsCKbW8uyY+ZKsOkAGzmlaXrqbmkYnwLLX9Cl2A8cPHIgaD6MJR8\\\\\\\/gHeZuXuy6FrAv+Yq4G2jj5aTOLF7DsXthMObf7QhvV\\\\\\\/SUTnqZn4GmvoXLI9vmqs9VhYT00JP6IIU7oTou3vxVTwLys6uhJSY3sMnCPUgb5d2ZulHkxqV7h2WfV8cLewg76EInzeZaf+MYQKY4xit1n5Wm4OMxny+hoyJwDaO8MIp0fMHH1XoXfTlEz\\\\\\\/3oyuNHj1zkb9KwZ\\\\\\\/nHmcqWBYTObi\\\\\\\/\\\\\\\/uXxz3AMK+ahJvtBvrQdusdr2+It+613stks1RlW\\\\\\\/8\\\\\\\/iqxWNpwakzYeyX8TdasveKzi1jUcbRu7FhK+KA6XULEjJpbTX1UVG051mgyKFT5VV9\\\\\\\/zkpt\\\\\\\/RbfeqWepmZxY\\\\\\\/mZ+paKdh3Vg\\\\\\\/4NrOiAXQfazC216fQjDh3Mi+PwPhZVr7peysG6f4n7OyzLZpo5ZwSg559jMMYyNZRv8zl809yc4QG4D9ZZAOb7ug3PA7cyAWG7iEIJNRbcBwM0h\\\\\\\/jdl0bYKG6zzqMs1txVWo0iB\\\\\\\/0zzbQFvKJqTiyhPSh8+INqD++GKyb2vKwoEgBefU7LEuGBu1gowfFUKlgUk\\\\\\\/qW7KiXvFTA26jjUTCh\\\\\\\/j6SjiqVU4Knk5uIBdQmezapXh82Cu8YBzoHAM6nVPJ1rQ57sOtC6EivPdMLPTLnxIiJeAZmC33BKa3qcaYoxONJRT5hntnTYIH7cqMM4LGuoUrsKEV9L7qdevcT6pt+oEUyxmAw8BIyVRyyCsthopL83k2H+4jK\\\\\\\/4rlbD1Wqk1tI02CSlWzedjC+uPtobyH+CY2QfeRx61bfbPjd1BF6Sq1kl89hlxy1b\\\\\\\/GtcUfg\\\\\\\/VaPdxfLiGF5ZFoNXzHffKgbS0YUIAcXD0pyzX9ScqS23bzQqfXVPchcYzR2QcTMdFl08FGEfKL3OPQTrVGpCUNr5E+yoBHK8q6VPUvQlFkKfg4mdEipArNsz2asOt6kQG7GCo8OelyZ7\\\\\\\/FMpblQwmHSTob4pw7InCpTonWCDPxNSwoWDqvK9157JqBPCtEmC0iA4Y25im3p8LH31POgVFmm0SNZDbRHkLn9F2R3FUMEfF2v1\\\\\\\/0W0v+1tOmOo0L0Nf3y\\\\\\\/la7bXOoZFHSZg1vNy3joMNg02kUr8IwGZ+DvLBq2afE81najPcpW82cbuWv4B2a4\\\\\\\/WoI68PUCV4KTkBxnxegoiVSJHvyqoh7iDgW43BY2MWJ6VSysjyl7J2xClKk3vt+j947z1LqMqJf6ZFTiDEtoDl7dyehzjfbhyoqj+CXebFquejIP7mYE5ZKxjdP+bihheTTxTn+gsvYNpLq9doXiGSG5pX5yyZg8jh1fx4sCnmBEU69n+6n2yGfuK8foGUYeSgTV864CrRoRBnB48g1jEOPLrvPY0\\\\\\\/ljjHN6pOXCRWsq\\\\\\\/I8WxoCkGsMRqZ8GPEtczjXuafqZrDX8rDIJJ9yDnpDhQ9Zb082EhxxP6jsEZ\\\\\\\/yXk0w25Cdy3PvxoHYYq1F\\\\\\\/kbhczsEuH2LmwvgHn84sfA\\\\\\\/bIEJjTO\\\\\\\/tvIkCpMfke7xkkmPV6hYh\\\\\\\/j7Dfu0mHEobmZmZH3E\\\\\\\/AYxqsDaAkYRZRICB35hoIByRH9RBAgnvSMERCWidzmws9jmNAJKnZGZ718dNN6OiuTnvaltJMABE5wiPAxVx7FsHG8DHocyiMq\\\\\\\/61GqEJH++wOaJj6VvCPZcWn6ISk3SiHLV5lIVgNfx4tzxJpN5zn7ASiIyz746GuZ9jO7VVWlBlxvJog8PdSvijKryM6aoIwB0+doe25baz53v3hhVG+OjMYAOxMq6zglrtqiWeaY1IvcY5Skkdo7MgIOndgdor60wNqB1znGCle5B2Y1QcM9vk8NKXjQ2YKonKQo10Y1JMp8PlLoJHZ3t6ct8TQeLqeTdF7MqMisS8FxT1JOX1O3DfJmVqDrNSMxiHPiYKVijBNV8z4pefb4EURuJ8TzbTiu\\\\\\\/UH2\\\\\\\/xE\\\\\\\/\\\\\\\/qkTSRoWYZ9BqzNzLADQXJ7zYtwrdLxHkNlMFRlfkzcMr+jBASWc4UilJ823Tg3Mi06KOuE4RpXj5sVm\\\\\\\/ZIoXTn3feQHvcqB6wsNzjSH\\\\\\\/q\\\\\\\/HcIUX29X5sC5QK28qqn8rRk40KLquGNL38ZJdiVde+fwrwcsNBYw1y6baATWUsrGpcFWDJkljggMKZqwIqkJmqEWHAmF\\\\\\\/M\\\\\\\/4ES+lMXTSMSIpqmyGkJrXTfJOUWqBmGB8RLU\\\\\\\/MYAHSZFZTx8eDpX5mezY2g6LWYv9\\\\\\\/hKfzpf+r6uWAJohO5YW8C2TDNcjNJfrHIkpdc2vPyGZFyeReqvzY4PIRUGHBW7ogjXb+oTej+cIJQ+x8BF5b\\\\\\\/OkMckjnZcr1PFfIt1Wuk7pOMeowQL9sIKh6cfVGM+fcBXAINIzB7tC+razmEwT+vkZtjIPnerGcAeIZPoDEOKQzcnQNQNXTSqGS15sWnv9nqGgCD4M+V8tuVLRPrg\\\\\\\/ubBuDz9Grx1Rol58MEvhAHu9JCGOfvcexglX4TbhHlt9qVjAWuNqSsYLO4r\\\\\\\/FU7GzI1g3fsD5EOC6WAa7u7\\\\\\\/5bVB2n506\\\\\\\/XvoaQDdzQbrdEK2ej66OJ5RR+3Z1GYxjDeGbRiURGdzRwBGEEYxwSCkN2uwSjbsAAq\\\\\\\/Q4Iib1lTLFpP\\\\\\\/qKpeGC8T50ThS8YoEnIRkRuyRBBZkpYfYNnC65+DkQ4ikfQgcnsTU1tlYVE4qMun193u6wPpFUpx9SLvH5BwAJ+eA1Z+qYZBL08xuvqFPXGx15v0ZfFAOLIxouXdRnjCI2Z1cNDMF\\\\\\\/h927CdnPG2tHBbUW8A9d5WUgorwTE5xklJYwb6pJS2eJMMZjrWaJWnG5cLaM2nsP34Q\\\\\\\/gIy7gErlLrIByCRE2Fo8lrRCVkhSzZmHTRNphdTEYQcSi7OOOvNjwoc\\\\\\\/jvtSj+j3OBamLHdfViw35vOq21xDBPOJrPb7DD+vKdg01bSKJBBjhOqNffPUkjHRfjfCR4ruGE6kBcBeokfXjn3mPfiFz3yPCUtStQs7ubqknJGF58DCH6OP+64JoFSXx0lVnOyMiFVj3qEfzB0W1L0f8zoM6MOh730gy0QTIPefvuNlq7iPE2Rlkf0wGRHPg2c\\\\\\\/lfh0gIpBKXEOxAeG3aIYAyfIVV45O\\\\\\\/TVPkDuk0d\\\\\\\/pXvOcFnLrvopTPJnJv1Gd7gB+WeSf+4110gXVrcTUp\\\\\\\/0cOJBwKjWNRS8+DkqYQ9aU4dcXlGeQJkslcSLJ+VHWrYOLabFaP+z4iMKTKWQFNtXA0LJrbTokGKnRRYsO0OTTOJFZtFFa1OqKrz5BBWFBamZOPE5Py4CX9TRiP3na4zqMYG+BwBrgMuOMEziRMw1t2ddxUbK0\\\\\\\/xMx1MMadII540eHztmrYZoroKR+qSla7BKNsYSScnMoa79S68wU5dQubEZSmlEwAoTly3oZmc9UWtU++tL17+6bJp024x6XfuyUFsNJCZKhq4IZ+tRzv3U1wuinMQdGIx1xYgOvNt0AbghaT3c9zAlRl5IYK8bSgfffbdGfvsDHVEOrXFLjOq7EeZrbnCUFWcVaDwEeXrxxyh\\\\\\\/iv5SMrnXXqhaCJez75y3n27ofK0JlhajG1gpMXrmf5axWGK5iFr6pmv2KQAPQmyGPM3fNpwLPsgb+H\\\\\\\/dUHs41rjtapHO0BSURyhVdkubbW9D1XwgXTatdHeDDLHZsup2DnD5NPlFOdAI6PSVJn8kv3cqPo4XanAkDgZmg67jWQZStZMW5WI4YvaPHCdgkGQRlV8jLdaHatF\\\\\\\/dpzlKPFTq\\\\\\\/QhM+CnDRY02lh1hlNPXoGMarspP4OQZHZMxMUX6Q1qYtZ9Iqvkig8y5EaKY5o7mzytwMwttw2o3QjqElaxyq2E8NZ1XqSTHstvL5DC4l50Wmaut3W7XyGd+wBnQwifstcJUSRh4+nOqt3OzNc1AHsZGftELKGlO7uVziZKQvtqJviaxaPBEpmQkjtMkTXV65y07SlzJ391vLGzl6JJvM5OW63BOCeZtYGb4y6zDQHLRAmLmnBfrU1i9h4OG1CdeMXMQofPSE6f8atTHRe+uNio4bs0yOoRyq+2a\\\\\\\/1IJiC35NDfwAH+YwUZvznhm3WOTg3XrpL6EGBE6PDZpjxSO6pzHjy2WejDsL4Q0GTRPfzTcF+D97HWOC9JIdcu54r4SLAMvSUAI+zj7\\\\\\\/1Asq4qEXZtTLn2x9TG6yxdxJSMjCj1JMpL+0rQBlEDLYFaQGrqHSt1uFbszgMsHg+JwLC1URoZ55UoFQpDpR18GfUKv3jdpnaARbF7vS1vLHITu7PgGU6a+EBdsn0YMs3Sydgdlv5SzKFnIDsCKJsMWITciN0q5ZPuBj\\\\\\\/vObZuY06juJ5W2WRpeXxITjPE4zp4QPEE81535jOQ0IL\\\\\\\/cn8O1ag9qK5cIq6j49CBHQq6zoyGrx6UT4yyfvB1jgseUYSRBzw9N1lW9WZK0CXE9cFqkJ93aGwcHarENMjcKWbjZttbM44gkxToqjr1N0tZLrzcFi4UV5KjnawNo9DXselfUhqu+cjhhtKmC\\\\\\\/d0sbLTTjya8CEUvJzyBwR1Oq9PJt7aUeyOqM7qaCNUM2c1YyME4UOLiOoZijCJ10jZLephq0eopGsQv\\\\\\\/c1Jwvkj1LIfc7JN+\\\\\\\/elqDuQcfs+Ihw35Vo0Tgzz8yvhdlj0nqmqf0Gf8bFRmyskaM6L1MQ1+CD2HPJ0MjD4DNPavDBEgTr7BrLBtK8CnAsDb6pxTewZRlRVAIJ8rGfomjlIabiD68SRgN1ivuVQx82Y\\\\\\\/DdSxWieCPzR0x76IZBF5Z9C2OrZzwTQj0WcstL\\\\\\\/Xf6aQ4gmlAvGOkUzE7nUKJWkYWdgtpxdedxzxhCPTtcHzv2KFqqB70QJ99scSYy7RSjEy3gs2bC7xpe0rXGcx10+A7ZYGvY90l22DvFmvGZOcE1a6\\\\\\\/4KC\\\\\\\/4KvsSnWrIDWGTJCTXuPRLhoPTute3qXxxHjzXXxf4iQdBrRqQI6e9RLBvP9L6M+uqgykNffgT32fJykWFrlNekQkqSzpzkBeC3+3hv8Z+NtnQYo0fEtIjXuTs5AwmN0eS+1ySLSJZPndKDT2Ghh9PeUwj540H5NzJ2qhTDTMnVnNUXrMKhKyqwU4mNtyXiM6nJufJE7+tjUrmzZi1DJlqEwqupcnb\\\\\\\/83t52tyGmPcse9j+5RqVN9cV7Am9QkoUdSLsRFIbq38WrfDBawVihwazVDRfbddMAzMlDgxLyk6t7IJn3tI0IqA1h2p6s4KKuabu1\\\\\\\/3UBlJyNqlZuyYCM\\\\\\\/0zbfXRHtFUS1kYrHkcn+I7JqPv7XKsyZKy2NMiBeOhaE14OtCkeHtWNnGB9MwOsjQ0JL2+C0GkmyRR\\\\\\\/AXRLSrgGS5eofnkdJ6fKRDnIgR\\\\\\\/5E\\\\\\\/dj8\\\\\\\/DSzwxwgZWkcKBa5CK43YETaqN+gigDaRG5ee\\\\\\\/SK3XgxmKJPor6ATwwzLVukGoyu69LKEilh0jfzO9xCsJXRBpXrtuM\\\\\\\/ueOgI\\\\\\\/WEQTZMDUtQoHT47P1l4+\\\\\\\/3fzf+vQm2AnSTHO2pD7SP0yhB+xmRHg1uD9x5K6jh6m+e17bLG4N3dhvslDltQqcewObNFAeScRPl1uwgbFmGbOG3MYSiDa6MABUAL79WE1F9FrgOcAcKgUyPOUX02ZWVtTR+n+ah0hwJckw4l7\\\\\\\/9apaeT8S7pbCKbvvPEHO8wc70AGloB38fzuPhMWjl+TMkiLGuDuw8g+J8GujpEy1NNvPEA7Fl3spVYBJ0DEEkij90YqS3c6WyACKQ+qZ8cyHdVPadFCdXUcM8DQP1aLYvqNXkMounj05qj3f41bA2yk2p+1Vbo0tRnLklh8FC7kQmqDuPCljKLiZ1JYWBOsDSKsm7woQHs8yGTZupefSUp\\\\\\\/CahJsEq80LoAr\\\\\\\/ZM5z9ohKXAPlBD8Ynt5TOqOcuOljhwAn9blMCtJKCwodgjCzKowTbuFbqImrk5hw9\\\\\\\/+k2p1Se4TL+iYHPeZ060f\\\\\\\/rF5lSPQZlCVRs7piKsi9z7TirAABp4vbQ4F7vfsJz0bIFkeamY6EWiOsy3Jq5yTM+tx3grTHec7QJ0D6BHW1g1LmggoqO7kaJGhIB+nO2pkcoD+bHlBdhR+omgXhNOTp+6lb\\\\\\\/Z0nZLmjmySFQQhvKyJBuo0m3dXNGL1o7QMV8wSX8SIPD1\\\\\\\/2xcmu12dcoS8Q8baDCaJscNSIIod28NgRWfHghL\\\\\\\/NH6BabG3f1sJwdsSKuxCmK1LZCG8+d3LvOJXdYY\\\\\\\/hfAXIKgvWLq5BMWyqi1CK794OQfFJqd5V4Luam0ZnXjGLoaMnKI4DPt2PM5lwSno1eB9+J99Ki+GlpACEJUtF1f08uF7A4J3mR6zee5YJ8kjxDZuvRUpAitzPO27\\\\\\\/1QecjdHkmEYhzNYYaThgk7KSyXIF85tzdpmdKzILYcFm2X0+BzZW7VK1l5Z\\\\\\\/7h33RV+MUZRwQy4rLXqS4uBGL3BODUiBPPY8qipPU3E5BlwNx5Dj9etNepeU8qKvlM1YHB0Z6Yz68HBJqUq34ye93ETVly6+SzLVGusg6SycfQLpPXiulQ0PifsdI780X+MgrQFhUSRnCRJ39AD13rZN6twxh23z56IQ9XR9KnIcLLzntwJs8HcPUivQDp6w0QtDrEZNtyRgbsRTW\\\\\\\/GXvwL1\\\\\\\/+AK3qw59SMsLLvaXbUhw0Hd610wBKQvFw62zFTFh8Eo9C5dTSxE1ZfhkmiOEw4nn1hHNbJH00nq0c0cISX6Gy15YpEWFvS4zjhhwg0KAeFrtjbEPkIBUAtwYdDGL6gBlT2Ekd6TxiFiwmn8nymiu0G9o3IURUzToeJWpQLjdWgG\\\\\\\/RnjlB8i5axCw3Meg9sOF+TSM\\\\\\\/gcMjIpEAN2flLUH9LAFdF4ETaqBKAkiL9V6+lTezNAdi+HItxbqXuLGrAoIq3pHEe2\\\\\\\/CmU3aMYEPMGJE6irXyWMLfDDszGY8ZuRlWiQ6oDThuIz\\\\\\\/n+tleLP5bkDjlXKasuo6eYhzm66adf8O+BDw0q0n392gfhmlyMQO07bMgSOZ4uTpoGv8U5UmQZIk7dd\\\\\\\/xRm7Ar+ZsTIFpt8BmL1C\\\\\\\/fLmHYFqGLHl3tc2isLuQLN4PDbHntqwHBS9Eu3ne5Z0X4ENUopDugWj2S1bhpgEbaDQK+9oNIK41JLdS44KEYRlW1nCohMH4ABXFZT3WhEgiS7eDNZRRPy5mwf0SO2oSiDxaoKefoeX5M5v\\\\\\\/bfctbtzUypuB9hLBjbvJLj8CWlp6g0uJUNQqpooeWUBMzHejptXFpmGVFj7D2otLJmwazDxEAff+Flm5SHUsgXgjPK0tu96MaPkT\\\\\\\/SsusUV7ZWc1kf4gUm9zWd0gphbeoUFzw87r3RaIWmye\\\\\\\/0HFzlDunkl2RPziiIi5Q4Qo4JGKjqzImZars3tzX+aRThjgnkILe\\\\\\\/JamogFH5R6DgL+N6FEqy9bNUdRmk8ZnxnZcvbQkhFalVVqeZyOsOMCAJ8tTtXIAtNEqpMnIj0JfW8vs3Gtzqxp8siVjaTcdgQelAD+pB8qHJk1FROwnod3VSsNZtwysaVpTJJdLPBDABcS5ibl9nSqdjH9nBM3NZa1nbQlwL9hOa2Z4tpnf8HMAp8SjkFvk8qg6dxOKBj6kHbBeUp\\\\\\\/mQsJlTSp\\\\\\\/\\\\\\\/Q3sT3jhr3kB4\\\\\\\/+Ag30v3jWp2PqTLtVXe7IDL+F6zjwMZl9Eba7T9yXpk2zrIrCQQn9sF1noWn4m26ya3lrx00H3XJ23t9DZTMtwKdexHJ3vB7xf0s9ajOOA\\\\\\\/T1I17QuvI34MeeA4ZiHoZo6A3y+55LeuebSAcfK0jKsyP472QXrGSbu79wxCcblQDG5hsSifEnQHxgC2+7IyYWrDXoWG7RwQb7t+3+0DKMp3qmuKTaekhbaFGVbGBGCXTuNhv\\\\\\\/PqrduoAHRLt6hk3\\\\\\\/70tcRYJ\\\\\\\/y4d7S\\\\\\\/5nNcpl8tjbvt4GITRmrwMGcFZEi9s9m\\\\\\\/JVhH0S+tl5A1PLkdf5GXM+1c3vEdRrNYrhCrtXZewVQGY3cTergxFLVt\\\\\\\/WXRgUK+UlRCwWKk3TZ2xJNbSmFz66H9KRbQDr0oLMNir+PfxCso\\\\\\\/gf2BA8I9Yp\\\\\\\/ATButIGeiOJdnNDjwPULommkSrBeIcKnfYASij4zVUXEjs4\\\\\\\/UQch\\\\\\\/CRjBdAjfv5FBhic\\\\\\\/3F5Jwpmi7j1cDRQuVfEGiUOEnrBQfwxEwprbtJuseOl2257VC05Cxyj6hfVIA9O3C3iCN\\\\\\\/KZWWic1HBQre0Tltehj0v6bGp3mL1se8qr+i8PsRraQy963x8g5VqVKeHfDgHF6sNeJuPgo9UXJmiktjTC4SmvNRwTJaachNm7Kgp4UV8dPVwJ4onL+tGfQMG\\\\\\\/G4xUHc8AK4RY1e4j675\\\\\\\/\\\\\\\/jyxOmmFdNl5IYoeRA1OQUNphpl5x4M9bl5O57km6XaqUfW2x9R9n4oNE0U9M0dJ956Ew63aetMV7mrQdtsxcZqzNXJxvqMBTqF099A4kxMlUkrAtt5jjnTm3HBM1WE6B+A9k63gv6rg2hZUeno+yFB\\\\\\\/2hkgPWoY\\\\\\\/BnMaXFnG\\\\\\\/TmPx48tSmVt2ibdBypq+VNj0ZeLK2E7oGHvydxrOjaf\\\\\\\/CaJKZuTpPMEuGl\\\\\\\/yeu17sxjZ6Gpk7DFkeRvb5OqC0aFJnF4BAJ5OJyGyZ2JlWuC2oJvWyHD64LbT0QYyTOuw1e1oieSKPRPyh4ubJpYCFumVMoOmophQVD2G\\\\\\\/inVtdO3QIrNL\\\\\\\/QqQbbI8KkpIlAdFzmJmhwj10jGFGxFwSANa+C+OrCQN3c9xhy19ziBuOPtjWQfgfUN8aXFk0UigGj4w\\\\\\\/EOZ17n8xT\\\\\\\/sn+MknBMVAH7e5Bh4WBACaYWpzxPWt9IArHAVymEFBfs1\\\\\\\/A0w3lIzTlKnjsH2EDkRXXfSC2Om7D+Rf0jOQOvrOgifilXSPzhpcKulYkiCiZMQJOcu9\\\\\\\/vAbP7FgMb7eqDooAB771YsJNKctCsVdp7w9x8En\\\\\\\/FUR8rZu2e92eu8RMu2Npc5EFutpWkrHdaavr8bG38X11SvQI73Rlf0+pmGzruuFMpBH\\\\\\\/FfNM21uRczNgVpHxfxA8EzAd3EjscDrjJ0gvurqpKtWk1zSY7K152zmvdBxCiR+eRMzqRxneZ8bzOwG2qaF4U4CYB11uMGEt0vBGUSqMBrNcMYkKbS\\\\\\\/q7GhX4ejoWtw4Qxc0s2X6ITOzGxcghO\\\\\\\/JVW7A14oX8AHTEgc93G0ee4Au7YhX\\\\\\\/Non+Ah+prr9TGd9EZMFsgLRxs8W5u53XNW45zoKl71PzHjBtlX0PRi2A\\\\\\\/TUkG5FMfwTayFHkA3FKbC5BRrIeFuRLJa5nkkt6R\\\\\\\/wdFI\\\\\\\/HLokavuH9wtCDbwmqw01Y1gU0L3HHCXbyCOKDTQ9Ysdjbh6R2WkTsN6l4RuvoyE1PhJtmdt5KbpyDKK+3NrWkvC\\\\\\\/nEftLnDy3wz3PJSHv+sCKpFz+uMf4aW9rmPuFigrVHi+E2JRwSIZdJgCIq3qq10CWIx1GpQabSKPtHGiiOrbsJczmOfjzfqFj8Fi1M9BGv5eqns+iwkdhqQY55spdAnWCe\\\\\\\/TNiu01gUvN+rxKVzD0uGkQDyXJMfUtzDLkJ+QBgo5aUZbxlwJvT1A7Bi\\\\\\\/fApRI9Jy5t32rIXPdCMXQ7rQ6DbKmr0wXzmEE+6MtIhSEeAWH03rE0pVhwR7g3CmxpjWpALM2ci\\\\\\\/pzw7R1kxks7b2HszR5n1hhYowbMYAAKoSPlbTZMXtTsDIb2wBUOyt2YI0yk2DgfiIwx07luKPrJSKnYIN0nYm9atjemtONs7i5oP\\\\\\\/eATsEXiNc25D3yFcy0ncaNjUN44DFcNifP7tQ2w0FoDK4Z9zPLyBDOXVq3UW4eqY40jCKza88PbjR9DtIxEuy0u30wfuIyUtlM\\\\\\\/v2eY\\\\\\\/6zOSX0KlbMYM5I2ei19dq\\\\\\\/pt41IRKwxKBm4wI3HKgpKvF1Ayd0sLLTuLvfmtez+Snl+s857NJWriC3IPFQzKLnItG17c4y3uOdlxBX1Ox6qgs7QM1F5hoQb\\\\\\\/CqaNgl23+x6eAhhclI3zlvZy\\\\\\\/ewgx5IfHveWDAQd2Q3P6eF5vk1BZfZjmCK9Ec4p6QNh35ejQBXtr7lQuJs+PFemsNgp7El0HDMUgSKZ+rHeYz5l4+BUajpWMzFW1tUWW9MPekBT1biyr+rHcWOuXti0KYeF4aM4UhuTTKk2EPfJ8tHx7Ae4ct\\\\\\\/OtEhw1wVYW24h5Im4oHaxS87XmmXPWthh4S3l6HG72MHYLxHp8Dfn5iyDDDmYszBRRjAS9j+k36kRsz+fpkZ3VJqtRCuvnuwDb8jumRG8NKyHWaY5WCHGjRxbSXfffYJp7o\\\\\\\/SDQwil3mh2Laltj5NzzaLtghAK3GfQUx1LS\\\\\\\/ZIlIoVRKZisq7OrGZZAsssxdzupCqy8EAgA8jDH+XXdXUAibfmzkM80yS8WYp9y8OpIKz3qYAO6bUZVln1E3SaxYiB+JWbg3mQpD4HBfX4Ra7AeYKthiFCYVtwUGwbtRxCcrrrdsE\\\\\\\/9kTu8l0y\\\\\\\/YudJW61loEL0bFFC6Jx8aXP0epdkEgQHv1tIylRcqdNjRVMPmLsEjylVyEuHlh+dKXL4K\\\\\\\/4inL4r+QHkPQSm8QMBiF0iY+63GY1+ya\\\\\\\/PxdBBqHaLtnZVkePRotZMO072cTB4K3USTctRbNP+L8pMDG9c5UV1IChYm\\\\\\\/M6Bb14Mz+XOt9krwlveApkd4pcfGgCu\\\\\\\/cOeCm+cdaPbS5R1QgT9FyVBv6HgCDiOsJuapc20sr\\\\\\\/G6ebGJGCNPUS8pUQffJdujJCGIVgFRP7XbdaKtYGNEUxEHQBHo4TH6RTtYfZspa5ARsJydUclJHQpQFO6mDLpwYvawXMEbsORXtvd7CvJ2omV4eK6r1jFLggc+\\\\\\\/+NIWluIcF\\\\\\\/GB80R0azqqnfdaglrYzouqn75SKoRx1aXYadTt4IBeyk+T9Js7\\\\\\\/OWeCo6aC8ChfV7XKOYgaSYvQUKWaMj5RLxKAq0ML5Ew9PD6rJ5xcls6Fc5clLkFjUMLtHoNAopRYY5xbTAmGO9zLvPjx6RLdSZwfx2YsMBcIYoKriTdf6\\\\\\\/He6W\\\\\\\/tO+3kC3JPporNxQ4P4IoajOk+4kNRAKgq+WctMboWZEVlw2Nz50QGFAt9K6Qy3o3YY0enL06SLS\\\\\\\/du0nNbTMcnInjHnz4Uh7yq1DbXA1xmKF8dMR1c+z59UpFyHsuEGIa3U91BnltCQZP2Pp+YeEflyqCcTvlTl0OSvW\\\\\\\/dXcSQDcZiXOegmXG2D6IcgqgRHDAFGvJiX1BHxuw9S8GCS77RLAJ\\\\\\\/qkxFbS2qwlzTKNvN4i3mDr54CjUt47Nc2TK1nBkYb5wTyVaLdFcV19SSOLZRdAWBSJOa\\\\\\\/58C8zi5Cvmghg1e2G9DoO0\\\\\\\/pBCWxSG2AGRmg0FZLq5HJ4yqNOiwhj6sqmj1w86G1DEOrKERNOV0WRyVNI\\\\\\\/C+AAww5qzGUFfsK4XhP2OGCzT1qsZZ\\\\\\\/\\\\\\\/CUGgy+pdXaPYHkZFqfOHeg+ma2bw0zDioIT9WGAcwMoXUFxWtNcGPkwI0MA1i1nZ5rDLaiembjin4fraNgTOvD2aegE7zOMbwNPKEaPkaUtWOmoWLYVEyujqZFxcAZpCRLM9XzaJAWS2PzdmObNtaAWyrzlx2UI72iJ79VuzX1cYo\\\\\\\/zrAEQaDLeEml2FXi8CzeKPRpbDT2FDpdVpuFeqhsKXticZEa04ca6dCrxtPzHkWgLXz8+yFMdVA9tlH+MSqcotULhEwAilEF\\\\\\\/mRA0AMPki+FyMl6f7NXHO\\\\\\\/G1VQqJDub7sA25fvKE1oihbW3Fsl7yVfjYEonX4Xi9ySf9B31HE6VKbRkZn3TgNoFz72c4OScky3eHNrUxvuTPgMXxptM3OJnzG\\\\\\\/pGHiTQl1YOzePyd0mkUGPNYpo0Rw7A4pR8HrLhu1V37Np+Ys2kHUDocczQ1CnzTGv7oRDjVBNJAvTtZU9PyqWDRgGaE6RXtRq01KY2xAtdTVWu34Pq\\\\\\\/xYz\\\\\\\/zlR7JGP30BetFL8LxqZfFlOxPbH5rn13YCdl\\\\\\\/Gbq8O7vccJtKsidpZwokVEFYYxVF9Xw4oWpDDiKGmVpPUxLSEnMg7sLuw6YfE8kOWpyjQTiB6s5b57NvVmC7zPzQIjOYoE2eJS7LSk4dEt+d0hry8ysCH0e9BUSExzfvswiyIyLSGeFNa9ZYvxXT9SVsxqOLotJo5tFwNHVgfWItqyuHFj8ae2hMTla\\\\\\\/HZSOmtJSmRsigeP7b9Wh+EY24bGzj51Vv+ZKsW5wY06ubUM3wfe2zvnALHV2Uvsx8bzs1ICowwWqIMLc3BlWomKPKnsggkiP3hTP1m\\\\\\\/BgH4xWA6S6Xb34odfJI28cCy9UAYtxdGp1ofzCPGkx9NXOLx\\\\\\\/d+s59Jo80mpFbNAxkuxLMsKhKQFU7e9qJ60+txIFycxz9izNGDZ90M7ZPNKflRxUaf1vr+EnzhwxyyfZsQYso\\\\\\\/MEjF+uOWdUvk0D3ADYA77gFFRs4WJc43DubtudUOwWWLNRwfv5LmJau8v115XlCiq\\\\\\\/tdmNyYXT4+7IGDvmu06+lLk7whVjyvcl42tBG4HS+fXoJxlfZPsjnilqF8WeLazbNvtKRfIONXjRdYoLMOGtL3I8dytVqe0CGyF6t0IPIorGklPjvN6aVG0lj4\\\\\\\/mn2ixE9ZT77Mo9fCOnpLUQca1n\\\\\\\/YZLAsQZnGaJfT0I9rigxggKhra56qLOZ+m+Zsh9hxDQgIwyQ2q50cCXzPov+owcWPWuzjVla4hki8cbAWbmXlPH9k\\\\\\\/4ZtWaPl5eoffyOOkL73sVXa2MxQ4lf1GTcnwegj11lNOuIA\\\\\\\/W70Xq\\\\\\\/zL\\\\\\\/teEhftOGzR7P5GKX2H62wZrgMB1FC8hffVeHnX48YxrcO8q7UXcwGKk9TCuEcRxNowBO0SfX5Hdf0tLY8+kmwqh0m3m3qAu35jHNrXQoPJymJUru1Kz3lP0ppRnoAhKzKJuSYqwAxA+1QWN6obgWUQ1oFe\\\\\\\/u4khl+kw+ZBroY77\\\\\\\/HgDqDCl36tIL5+KgPkqZo1eattqe8Zg7aScDap+AZJ+1hj7CdfaZtDxaksslY6BotNOWGpTFbEjATAwHwBjJyXCjHSAk166x4Ti5WSxsziwk2VU6q3aenOyBubC0FlWjVbnFM0lQ5e6NcQhcGazUJasZtSUU5RjzXZo\\\\\\\/qLsSaJ8kVVTP2ERJ\\\\\\\/1ZDzAJ9zjtGosEeaJxB7rSOkJxtziybhU2p033aASqjy06Oyg\\\\\\\/RY7lU0Q23pREVnOD2ZkFuvlhkZvxgMr2WUMk3AjwBP3H1vSomzAackeoVlITRytRzIXVg9x79OyJO4SaSnskGhMtIbpJEqRQiVVFBgf4I9lCM+h9bW8eSJX72a4KwlubFdTKYBHU9\\\\\\\/yO3XAvTKhSc7A7SdgGXebpBKIYJyILXC3aei9UWo1yBayIkFgtRI59J7pugkwzfzibZ5KBirZAjkfeAMRWtnWIFLs9A0mqaYPqMp7c0Y21L1p7cBkZypXCX4B8eFnt3u8F62AMpAnI7SLzb0UfX7objdBOtLMsEZeqcXKFsJVqeSzN99swy4XrU5wIU0Z8Sl16msBBhyuaYc7rJzpPZIYdrE1NKslJx2\\\\\\\/CssXESoniFLI4iWRBE90dxA0Incxm4fA1XaaaGK0a34ra4EBDSUd22fS4bkNFS72+5mXudZBXi9kGgy2bLcyK2zr4oNgyKFaVmvIjKAgKhYqATeH6pL33Dkd3UUEBRbSMw5KhtoZQt7tNgyfc0egthxBwCPHXe1SmKb7zDMEkfzVgChAupJH+oOrGTrpKueWzdP9DRG+o0\\\\\\\/qCM2vsZoZEyhekCMPDkdA0XEXp4A\\\\\\\/SQfylKzNapgsqwC45fob0VzhWOiYlb0ZopnoBQWBVVEo4IRcaAGi3fCBTebiBeVzYqP9VuxrJd1e7stR92Us5qyAHXd1SqJmIfFUo9y\\\\\\\/otd6Ndx7DoDwRnrYc0VfBiEplrUVZ7J3Mu1+1m2CSHlyhJZt4Dv+UB5i88sr8evsw0YYb1EUXpIHt\\\\\\\/5PdPFjiYKp8jsGFCUyTnq+ryBq82ptoGWxrBsV4gK92oGxcjM3294NhbCKt4uoTVSCeMdjEWdwpu1C9n2NQnDSVpjEOz0xsX7LgIpAfbHewRbr1Dl1AUHK\\\\\\\/ylFaYyQocvJf\\\\\\\/Sc+A9u3Ll9H2a6OOXUjW9WQR0I62gvTIVsqSHVEeJoYR2XRnLx4ACjm+skY+1Ryo7qPZp5in+Df2bpW2fxIYjI6WjtN7BACNUCCHAnMj9jy7PxFcYEnxCQcy\\\\\\\/yX5CHGXC4Cp4h\\\\\\\/ERX4YP0GpeNjE0TGZyD0N\\\\\\\/PMQ34akovZrjc7eSO2PlxYpOH6Uz+rnIJUbOsghoi3fnP6GNAiZDVo\\\\\\\/P8b8v1DOCFZKXQaaIVQEYg4V8TQI\\\\\\\/sdqE6uqOxeaXBLl+ugtQWNu5CV9ayuGt1DjSsa5wFbKYJwnF6eQAEWJb719b9PVILrry8JiGCGN3cVS0AHV4r\\\\\\\/\\\\\\\/tctYer+opFVD4vKUuXUSSk1JeIyuVwNmuJUoyUdHg57sHeKcK+MKx8WKmxzBU647hdS2u6PWHaoa6Xc5FfAE+8mNRF6MQHv6ZsJwyjLEyKs\\\\\\\/WDctgfsb+Htzmh7vBb+uUm0r3XTq3p1AKYmLv2bXDWMtYEsUNj+oV36L8DgSH2rC9roOoaDHDaKPrchdOhn01Hd3a5e4yH4dc0YkBH1ETAxf88UG5xZzkd4uTRK\\\\\\\/1ldQFdHpxMP1jp8XzdGBSU0YMkCaPqJxRq\\\\\\\/oL\\\\\\\/hw2DqyTSOaSFsnNNcmadWnmLPej6hU9PKKLeItQL7DKRcz9StArUZYhun9KRgqlaRW5l\\\\\\\/wRV9TvLdU1CCapNHL2IVx4i0G2BQM+z\\\\\\\/5GE3ID53pym4dnfV7RxENujd54UHbLQHXwW2738ugc3U0\\\\\\\/59GoHtMWzMZJMWTzcFcokIxOOYm+4UyMO\\\\\\\/JhmbRwvVArghOYnjUIRfGjg9ghiwuUhQOj2qAB\\\\\\\/UyTXzPQBncvnbYdEUPXEUSzPE7CGRWDgfLXwDPRYcdwR9Npv1oitApUgo1Xtp3xTsoJyL2z7Mlej\\\\\\\/aJG3ENWJe8BY\\\\\\\/f1hDYCLhrAoeJKHY6tLRkGS3Y2SP0YyiQEstxnL+8RYRtHWz2n4WQGr5pQRRe1GmoIvOnGSTNeom8bNvdahVJRCYkaLBL25jTFymHFUd7b0VGG16riR7NZsI5gSuob040w6OPOKbYBFvHikdYCvOU0ollzlNZ3KLBFBgu+qrbpn\\\\\\\/ccozFSo6XMwiIVot6plZm8FhRG1883arBad5urDawjrtQ1gtyXHoZ07EQPAEF\\\\\\\/Wo1Pi83brn617Fgigt9VDWrealvKId98X5hzpnbTmAh0+gsFHEvuyy+FA8msRL3\\\\\\\/XbSPrd\\\\\\\/AxuuUKkyiQsly\\\\\\\/Xm0fPPOZblFtN4i45zOGGgpvRBi6VCHdEZcrii+jZ9uvFsnoYQUpsqbXVoUNo4Kpm+OZ2uhHOcFONrfU2dbdkimgIuhQKSK\\\\\\\/0vEuQG\\\\\\\/oFdn4JINP4ERLU3wTQSiWYqhniUUZYHZw0XI+FECN2O4ALwwW1bQb9jdUNT9fbj67y8peHi5qAUh7pg+FqZ7ETRpmC2dUFL\\\\\\\/28TnMUVZR1oOyxU+b0FjsriMPWPLZfzXtNhAcnbkkhnE8c9tNteyz6Z5HWI7D27TjNRBDBgTfV1sqY5nBtVCzr2EVM3nLyPucFVn7hpe1E81TLDATNRi1YAbLosD3sK5GnL4PEbKxfj\\\\\\\/G1qX+7LPd7q73r2xB8x0ScJVoc0L98pR2bIwnIJFCzd9kSdZfcfwuxGDjqQpdq6ADAx6z1fO9CV80bQevtrFQrh2i9AONqes3qjdOKPYu\\\\\\\/Y5YI0kImtsq\\\\\\\/PoCSObdw0urL70zdf6UyA65fFKyzEpVA7+7GATZfu+VmnkoizlzWZigTSLYVhjfwgoom8qEjDTjMVFAPYC0bJjMORgw87elcGQAunIvR5VWS0lQVKQI6Nw7hIvW3KEb4s6bSkGNsIMHsuBIXLrIV3iM5IZdYX5TvR9EYNRLkljODOzXGpeVxHqiszJPxQ0nMxyQcJuMDewO0owoqbeUyPCZaae6r2vicafENl8ZzmMvN7d2echCSfrL8i2iUVKGXbBkswfFsGNgCUfqUSOVr2iFi+SPKlH5ggP6bVcd4QocYpGp9sDXCev+nGLg7DHtDnnICjWdK0Rynha4yeQBjjZ1NOcAqTd9IobqEzD9TacpaGSqRfrC60c3JhNZ5RkpbsX6ITDTIUHn2dJEzREbMEuouA12RYSFZb6nKubqNJgJLo+GVjz1ykYuouAxPy8u2Z5iYyPw1saCuBgYD6zNYRbtEsEFTO1OaUb7PPGC86dKKfAR8lRVVNSO2tFqn0q6R1ADMLde9X2T\\\\\\\/JYLcoXHCZLOTyh5Lhw972NJDRC3dcV7cw+M+hQmN\\\\\\\/pxoFWNVBPaTmnnTMm9oTLDPkYFIinh8wES4a3o\\\\\\\/Hicc6KyBTC4lYiZcqlGrYGcwzmZCONn4tsdXUfMy+IRjt1EW\\\\\\\/cpZf\\\\\\\/kiribOVHg3b7122NqVfHpjxknBCEUwYvLbxCOauK1Qgww1HvGlIHbfWe607PSKWbHpO6LCpA6cCfBidRj33Z3EzoNusNA46gtEUxxxxEsHpie0vMM1lAjHrGBpW+ZPrdxzh6kEeDQHyZgBuIW8kRimlS6tKbRCoFqLdwFqatjctaCfBU+9I\\\\\\\/NTe1B7pwoF11+wdNoIg2X9Izsi6+mHT+CtDMublS5KXwbdT3MgXElIeMzDMMBr\\\\\\\/kTmfoCjYJ6q\\\\\\\/Pt7kALXBENwDHPXQbw8mjoQuKz2ScL5SFSYgK1\\\\\\\/d\\\\\\\/MG5qnThRR1t1Mc\\\\\\\/aSvqsw8\\\\\\\/zLYpsNEhza3EKyRPUCfZ4I+eBq0JuRnkL81Ptz6mH8qKrFgx\\\\\\\/ULmzuJIVtoj82g8itPMfGmmJKeZ1Kfbcr0HgMjhoarPvywQb7m+ZCnhGdTl\\\\\\\/EZOVhZEGORiKNasxjx+BSdtSJLvNmhss82ra6F2nXzMMjTqiYHxTBqBM807ZTBucjUSBJ3ynO6YpsmeB1312AtVu3Z6Hx5NMQOkbTDtVhqSezx1a25y8KcEam+RNR5JUrtPMlyaC4oDGrel6AelCqvppocOeCYCqM1DPop0iUCnleiUs6fYBU9m9Oy+wAHh99IwtPl1gvr59sBfGse4co56OFGTqsZfmjXX5\\\\\\\/MNTu6BXTkgecuaKlHcoKr5hlqIb0PdUGlec0EkhlwbkiYQrD\\\\\\\/aI01NIn2QfcfELtuknUHmMc1g6co4Im\\\\\\\/c9fD\\\\\\\/cWyet89kJq0zQDbjGZ2jPUo0LjdsAvTjgt6qhpOVtzGZjOqdSeXSo+3MUfyIB8ig\\\\\\\/fapAUVeXkgpbDxOSxpTlXhykNmACTq7Jm1xUbyk1rUXBter7C5yXFniCcfePr7hHoXXnw+Q8PADee5jjXNo0KuVKKHi9toQxD0j1nWoKNsGFORrJsiFNZOawJ6mTzjDvOgbFM4XDX0Lwm3F+5midaGTcClw7u6pyf49WjzCes6TTPjeCui2+Hrm4oMKQD9++qQC8OTyRTh4IirCA8k77LsSKj2ZxEgNG2PQum\\\\\\\/UyrmerNXry4yNjFN6o1VIesULUTr1cqB71fuYpk3\\\\\\\/oZQU35fr5v5zmMFpOIcr62KbP6AtI07e9BXytz+HqNX3NiyvJH90RvZmJYTm5dd+ze24S37KVuDOTos5XYlltsN0saEDaZobBpw0RqVEPA44I64JTBaHxtjnI+srgoXwIdVaN1ilslTA2ouPspwwLf5lo2QPKcPcUhQqLd1BBZKkUwqTQ9kcAonIs4FTDTw5kWvdK7l0A6IRxDZ+RxwVpqi5B0Pt+oMPBPE3aJJK77n76QTWG0KR+Z48CIGigYF8jTjevFs1FYSrhWSI48DXZzG0gHHJpF7iWtbCdYa3S6BAG8yPzi+qKZ5Omck8rdJir6dJax6CaxhmFu5V1dGV4+KlbS22W76PHCa0uYHkKw74InZW8SOtGzb3H5j0z9ltXmv1+2h4P1c7tlDRbkB5H9\\\\\\\/tCe+INc3AD8HIcTGqjJSexC4UrhyUzwBmCU3P3JuLSYcZd2jHFWjrBsXkA5BJ26Oj8jje0J2v3YVFPeL8g3mS8RnFZJJR0aOCfxImJoZRuXVi5zYqsOZ4AXrZIoQuFs0cLjWO0SgqBiEithQSU64dRp0ZTp8XoPIA3YzkE43gmolX4nDy2EKx+755P\\\\\\\/FNOI3kblVMbMELCuozOoVfeq8+SXvpRtnLrk7fUOFLrp+fttb1lNEG2hMPk9HgXOCyV+QYGqbSLBrqU46\\\\\\\/99IIKbVZvWEWfllvQkrQR\\\\\\\/fYAj9tb6Xau3407Yn3K2jPPSVDlKDiDK9QKl3BfiooFwkNRks3qsXFkuw3OuK6eCTd9ZfkBFVJ0Ne9yVjzrkr+KolbDkdRLMwiGiE4lsxhQE6V0KOSv3tBGNl5gt1RxDXE\\\\\\\/UpituQGj280rn6+osxHK7PD5k6s3puIx\\\\\\\/RU+ILydRgH6ZRE8fY9S3M9yeUGG49PkmetS0+NdzY4VgpPA7Ubv3Amrrz5XQWsnPITlZPV4wxI5nS0SwN2B62iZfHmUI+JYbyetlFf5OqA0ZKyveErrzGxRpxA==\\\u0022,\\\u0022iv\\\u0022:\\\u00220ODa2zhX4HbtHSO5PrO2Ww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002279ded829a0c130c37da6184202dfc191e272bda734067781c2949e16af4b5faa\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aVQ0Z0fI7KO5yl\\\\\\\/Y396o1N4kk0gHIWKG+ZzaRuHq6QL4iN3aflkzDDZGyKYIdoa+Y18321974ZGuREAjfCxlRJK2b26tMPA5E4xS102OXZjx4W4g6GJJA3tSq6JJrcVrhzUsr8\\\\\\\/HOtwQzGfhKZSZprOdJdXbyi11QwwCJgZ\\\\\\\/18p85ZmFOzaeU8CM7oG6cKKMZ7Whcv1GEdlE4\\\\\\\/nScxll5lN09AjTd2tRZ\\\\\\\/PMvFewd9fwnhK0h+fUIa9xOqBI7KfDsIZWe\\\\\\\/qppOzOy8eNH31qSteTldgcFTHl0AZ3+eBw0HXKb+KUfaw8Z1a6Ev9Eq9LypeFzdzrupypzr+z8Q1gT\\\\\\\/JA21f4TXLdWIBPCqZg6QIRkj4FT7xKy2upM3tjZAwlCjNr7yjZg81J0jK0HV78\\\\\\\/xjGEuEgQWoEu653Mo5xZ0JpyA1fmgNUkuvgQu6B2AvmUMMqpUYrQ7cNZNuWJThao\\\\\\\/cMcazUl8cJlQ4UQc7Ud1GPCXr8jQkPhYFugMj0\\\\\\\/3qILXEXu4eFNYJFZzKDzj2HlVd2ef2RYf+jKggHV4mDuRonrZfsB3nYD4kxB3sGgZADB8cF4DxwuGDd3SeFUxRCzSupxuDT45wudIAg8aZIvIh6Mv2BCOjDgVC9y\\\\\\\/sXXmw\\\\\\\/gmaIMLTr90UrxvtQDQESrZhgvjOVB03i09doupVySYFrojmUyMXpr6wd96gY0kLTvzO1ORfOpb7OjlF8bdgoiqxMnXFABkTCjzpC++Papa8k\\\\\\\/d+65k\\\\\\\/GYVBo+eMv6I5RY4vW4ILFXjJepp8eCkZL6QUKota8s1wDefjTpxCCghUUCUAHc9eoFbBjEghLdvaXm6\\\\\\\/mPfA3FknvyUpjDcBLEqHvyTFKgUlgKLjPNbjhwqikcrqt6awGv44b96+Mw28ZhYdCSOzmkp+vxcEOrOy9vvu3s45nlHGTpM\\\\\\\/5oUw30N9hblMfrcfOVf\\\\\\\/AngaQvdwzxSgEJvn0tcaFfkDu7wdPgmcjiMW3Ssk7NVn6P1\\\\\\\/p7Msij5u9wv8YXEqCjM9iprt50\\\\\\\/xGdn4EvkvOPDG0jPqBO3bLTgEVHACFeGldbA3WyQbSIoF5zjw7kCNorPtpmS\\\\\\\/5nY+DW0mRrK0JsXNkLcrhu3o750\\\\\\\/4+ThyysJXjaa944gPTXlJASbrpK4nru1H6vOr4k8TZZR5aPG572suLetpY64nRIse0FU3Y4lwBHfh0o8wrm4IoYMRUcrlmfmS45iYkvzUWcmeo9n1kVqSehM8EVc8gXX3EIc95vMqLrKaVhmUJpDpAp3OfUP6SD8pABwncH2tV2CA\\\\\\\/gHdgIO9kabyKXvYESOV9Ib5E67m2X71aDppCuOLSkzKpvy58SUII32XHbzzuQ7DAeN8jgsZGhrxZnX5k3ZzTc1VIvIv5Sci6s2spRAH52P+XbwNcraHPoHsG+hN1JgfHAdNgtsgadWy9YlNccvTlKFrHIYUjhjvBRY7YITDPtouH5VIegUR3mYrMb+adTcdz1GT7mx1BtHo6Ry\\\\\\\/ySwtbG\\\\\\\/eoi16VuiZZKAK5abwOEsvbJwNFr\\\\\\\/89rhw5oytInY6CVIiriTmxoGq6hd7J2ct+0yED15XQMQG\\\\\\\/iMAXpgxBqmwT+gFDsxxArs3tXUIq1JLaCT1nQRqy7T43vu7a1QVdn1ETF9gpfd1vTYv\\\\\\\/1udZKEHWytzrAPoGlLAF83LCnojHzQF6\\\\\\\/XnvfkoUeKcc\\\\\\\/iNcElNYgjgZRI9hEmyq2PRruFU+z2crfMqsjZ77yv\\\\\\\/BaKkVToCIVlHUcOPv+rzSPymYQrFGaR+6tRJbq0V7jnJsyyMeusSCPof4R48n9Z4Fxeru8FbfXW80hE2CCtXLKF+V1tcrzeva0S32W2E2IRkrK\\\\\\\/RE2I5KK5l2KjI56J8wcA\\\\\\\/bn5mW01mFnPDYoRC8S7q4jzueaO2QAf4Hy1j3tH6Coy\\\\\\\/8jgubtzEDoU0CKBWeRNB4\\\\\\\/FniQeShpCS1G8\\\\\\\/+lXN0GDl7di7AU5uTWsaUtm7W9MVoGfj+URJBMMzYKsxcy\\\\\\\/J0IOUedKS2\\\\\\\/Ppr+Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Na6pM+7M2NUyNTX+vRwEKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002271f8b669ca34486db864848b26decc04acddfd08c2c31472449a511f475a0403\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459865 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/871" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i99UNP_wkB84ScYze6sVyuUwEP9D8on4sUkmeUCEgGc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/871" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EsoSa0Ukq7DGMSCyitQxSZXX-DO5r_DiisI7jRvM2-k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/861" title="Genomics of CpG Methylation in Developing and Developed Zebrafish" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/885" title="Impact of Genetic Background on Neonatal Lethality of Gga2 Gene-Trap Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459865" id="node459865" data-pisa="ggg;4/5/871" data-pisa-master="ggg;g3.114.010249" data-apath="/ggg/4/5/871.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bioinformatics Analysis of Alternative Polyadenylation in Green Alga <em>Chlamydomonas reinhardtii</em> Using Transcriptome Sequences from Three Different Sequencing Platforms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhixin Zhao</span>, <span class="highwire-citation-author" data-delta="1">Xiaohui Wu</span>, <span class="highwire-citation-author" data-delta="2">Praveen Kumar Raj Kumar</span>, <span class="highwire-citation-author" data-delta="3">Min Dong</span>, <span class="highwire-citation-author" data-delta="4">Guoli Ji</span>, <span class="highwire-citation-author" data-delta="5">Qingshun Quinn Li</span> and <span class="highwire-citation-author" data-delta="6">Chun Liang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">871-883;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010249 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459865" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhixin Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhixin%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhixin%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaohui Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaohui%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaohui%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Praveen Kumar Raj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Praveen%2BKumar%2BRaj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APraveen%2BKumar%2BRaj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Min Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Guoli Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guoli%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuoli%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qingshun Quinn Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Key Laboratory of the Ministry of Education for Coastal and Wetland Ecosystems, and College of the Environment and Ecology, Xiamen University, Xiamen, Fujian, China 361102</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Research Institute, Fujian Academy of Agricultural Sciences, Fuzhou, Fujian, China 350003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingshun%2BQuinn%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20QQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQingshun%2BQuinn%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangc@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liangc@miamioh.edu</a>
<a href="mailto:liq@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liq@miamioh.edu</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chun Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChun%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangc@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liangc@miamioh.edu</a>
<a href="mailto:liq@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liq@miamioh.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/5/871" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459865" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459865/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/871.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459865" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459865/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/871.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459865" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459865/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/871.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459865" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459865/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/5/871.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459865" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459865/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Messenger RNA 3′-end formation is an essential posttranscriptional processing step for most eukaryotic genes. Different from plants and animals where AAUAAA and its variants routinely are found as the main poly(A) signal, <em>Chlamydomonas reinhardtii</em> uses UGUAA as the major poly(A) signal. The advance of sequencing technology provides an enormous amount of sequencing data for us to explore the variations of poly(A) signals, alternative polyadenylation (APA), and its relationship with splicing in this algal species. Through genome-wide analysis of poly(A) sites in <em>C. reinhardtii</em>, we identified a large number of poly(A) sites: 21,041 from Sanger expressed sequence tags, 88,184 from 454, and 195,266 from Illumina sequence reads. In comparison with previous collections, more new poly(A) sites are found in coding sequences and intron and intergenic regions by deep-sequencing. Interestingly, G-rich signals are particularly abundant in intron and intergenic regions. The prevalence of different poly(A) signals between coding sequences and a 3′-untranslated region implies potentially different polyadenylation mechanisms. Our data suggest that the APA occurs in about 68% of <em>C. reinhardtii</em> genes. Using Gene Ontolgy analysis, we found most of the APA genes are involved in RNA regulation and metabolic process, protein synthesis, hydrolase, and ligase activities. Moreover, intronic poly(A) sites are more abundant in constitutively spliced introns than retained introns, suggesting an interplay between polyadenylation and splicing. Our results support that APA, as in higher eukaryotes, may play significant roles in increasing transcriptome diversity and gene expression regulation in this algal species. Our datasets also provide useful information for accurate annotation of transcript ends in <em>C. reinhardtii</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polya-signals" class="hw-term hw-article-keyword hw-article-keyword-polya-signals" rel="nofollow">poly(A) signals</a></li><li class="kwd"><a href="/keyword/alternative-polyadenylation" class="hw-term hw-article-keyword hw-article-keyword-alternative-polyadenylation" rel="nofollow">alternative polyadenylation</a></li><li class="kwd"><a href="/keyword/alternative-splicing" class="hw-term hw-article-keyword hw-article-keyword-alternative-splicing" rel="nofollow">alternative splicing</a></li><li class="kwd"><a href="/keyword/intron-retention" class="hw-term hw-article-keyword hw-article-keyword-intron-retention" rel="nofollow">intron retention</a></li><li class="kwd"><a href="/keyword/sequencing-platforms" class="hw-term hw-article-keyword hw-article-keyword-sequencing-platforms" rel="nofollow">sequencing platforms</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/chlamydomonas-reinhardtii" class="hw-term hw-article-keyword hw-article-keyword-chlamydomonas-reinhardtii" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Chlamydomonas reinhardtii</em></a></li></ul><p id="p-4">Messenger RNA (mRNA) 3′-end formation, including cleavage and polyadenylation, is a crucial step in posttranscriptional processing of eukaryotic mRNAs. Polyadenylation plays essential roles for eukaryotes, including protection of mature mRNAs from unregulated degradation and recognition by mRNA cytoplasm export machinery and by translational apparatus. Moreover, the formation of 3′-end is indispensable for transcription termination (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Rothnie 1996</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhao <em>et al.</em> 1999</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hunt <em>et al.</em> 2012</a>).</p><p id="p-5">As a sequenced model organism, unicellular green alga <em>Chlamydomonas reinhardtii</em> is widely used in cell, molecular, and evolutionary biology research and is also used in biopharmaceutical and hydrogen production. <em>C. reinhardtii</em> diverged from the streptophytes more than a billion years ago and shared lots of plant-animal common ancestor genes with animals (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Merchant <em>et al.</em> 2007</a>). Therefore, the poly(A) study in <em>C. reinhardtii</em> will enhance our understanding of polyadenylation in eukaryotes. Apparently different from most higher eukaryotes, in which AAUAAA is believed to be the canonical poly(A) signal (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Graber <em>et al.</em> 1999</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tian <em>et al.</em> 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mangone <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>), it was reported that UGUAA, a poly(A) signal found in the near upstream element region (NUE), plays vital functions in alpha 2- and beta 2-tubulin−encoding genes in green algae <em>Volvoxcarteri carteri</em> and <em>C. reinhardtii</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Mages <em>et al.</em> 1995</a>). A bioinformatics analysis using 10,508 cDNA/expressed sequence tag (EST) data in <em>C. reinhardtii</em> revealed that ~50% of the sequences contain the UGUAA signal within 50 nt upstream from the cleavage sites (CS) or poly(A) site (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wodniok <em>et al.</em> 2007</a>). <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> (2008a)</a> also reported that UGUAA is the most dominant poly(A) signal (~52% of 16,952 EST sequences) in the NUE region in <em>C. reinhardtii</em>.</p><p id="p-6">In eukaryotes, many genes have more than one poly(A) site, and such distinctive sites would generate multiple mRNA isoforms with different sequence lengths and 3′-ends from the same genes. This phenomenon is called alternative polyadenylation (APA), which has been proved to play a crucial role in gene expression regulation. Examples of such regulation include cancer cell division control, neuronal cell development, immune responses in animals (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Elkon <em>et al.</em> 2013</a>), amino acid metabolism, self-incompatibility, RNA processing, flower time control (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Xing and Li 2011</a>), fungal disease resistance, and epigenome modifications in plants (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ma <em>et al.</em> 2014</a>). In yeast <em>Saccharomyces cerevisiae</em>, APA also has been found to play a role in gene regulation through antisense transcripts (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ozsolak <em>et al.</em> 2010</a>). However, information about the extent of APA is very limited in algae, except some work in <em>C. reinhardtii</em> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>). This hampers the study of the potential roles of APA in algal gene expression.</p><p id="p-7">Based on Sanger EST/cDNA data, the extent of APA in <em>C. reinhardtii</em> was shown to be up to 33% (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>). With significant increase of available transcriptomics data, one would expect that additional APA may be found. Indeed, the occurrence of APA is influenced by sequencing depth (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shen <em>et al.</em> 2011</a>). For example, the extent of APA in rice is ~47% and 82% supported by using digital gene expression based protocols of the massively parallel signature sequencing and the Illumina sequencing-by-synthesis, respectively (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Shen <em>et al.</em> 2011</a>). Interestingly, alternative mRNA processing, particular APA, would lead to transcript isoform abundance variation and can be used as powerful molecular biomarkers in cancer research and diagnosis (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Singh <em>et al.</em> 2009</a>).</p><p id="p-8">Not only APA can produce multiple isoforms from the same genes, but alternative splicing also can generate different transcripts from the same genes. Research shows that introns with poly(A) sites in human have weaker 5′ splice site and larger intron size comparing with the introns without a poly(A) site (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2007</a>). A similar relationship between polyadenylation and splicing in introns is also detected in Arabidopsis (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>). Moreover, intronic poly(A) signals are shown to reciprocally regulate splicing and polyadenylation and control sFlt1 (shorter Fms-like tyrosine kinase-1) gene expression (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Thomas <em>et al.</em> 2007</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2010</a>).</p><p id="p-9">The chain-termination DNA sequencing method, also known as Sanger sequencing developed in 1975, was the dominant sequencing technology (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Sanger and Coulson 1975</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Sanger <em>et al.</em> 1977</a>). Following the need for deciphering the human genome, capillary electrophoresis technology was developed and integrated with the chain-terminating fluorescence dye method in the late 1980s. Now ultra-high-throughput sequencing technologies, often referred to as next-generation sequencing (NGS), are rapidly replacing Sanger sequencing because they are revolutionizing the sequencing depths and coverage with much reduced cost (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Schuster 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mardis 2008</a>). The first NGS technology, Roche/454 FLX Pyrosequencing using sequencing-by-synthesis technology, was developed by 454 Life Sciences and Roche Applied Science in 2005 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Margulies <em>et al.</em> 2005</a>). Afterward, RNA-Seq (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2009</a>) and direct RNA sequencing (DRS) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ozsolak <em>et al.</em> 2009</a>) technologies provided deeper transcriptomics data. The massive data accumulated over the last few years provide us with a great opportunity to examine the issues of polyadenylation and splicing in many eukaryotic species.</p><p id="p-10">Although the poly(A) signal UGUAA has been found computationally and verified experimentally in <em>C. reinhardtii</em>, its variations in different genic regions (<em>i.e.</em>, 5′-untranslated region [UTR], 3′-UTR, CDS, and intron) and intergenic regions have not been systematically studied and compared. In this study, we compared the poly(A) sites obtained by different sequencing technologies and found that deep-sequencing (454 and Illumina sequencing) revealed much more previously unannotated poly(A) sites. In particular, a greater proportion of these new poly(A) sites are located in CDS, intron and intergenic regions. With the greater depth and coverage of the poly(A) site data, we now begin to realize the potential of APA in algal gene expression regulation. Moreover, our study suggested that polyadenylation is more likely to occur in constitutively spliced introns than retained introns in <em>C. reinhardtii</em>. Our datasets not only can be used for annotating gene/transcript ends accurately, but also provide useful information for further investigation of the regulatory role of APA in this model organism.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data sources and polyadenylation site definition</h3><p id="p-11">Sanger-based EST sequences were collected from both JGI (US Department of Energy Joint Genome Institute) and The National Center for Biotechnology Information (NCBI) GenBank (<a href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a>). Our collaborator (Dr. Olivier Vallon from Institut de Biologie Physico-Chimmique, Paris, France) provided us with 454 data in <em>C. reinhardtii</em>. In addition, more publically available 454 data and the most of Illumina RNA-Seq data were collected and downloaded from DNAnexus (<a href="http://sra.dnanexus.com/">http://sra.dnanexus.com/</a>), a robust web portal that provides biologist-friendly search and browsing interfaces for NCBI Sequence Read Archive. For EST data, when available, raw EST trace files were used to identify poly(A)/(T) tails by identification of cDNA termini (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Liang <em>et al.</em> 2008</a>). Detailed information about data sources, poly(A) sites obtained, genome sequences, and gene annotation is listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464292/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464292/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The data sources and numbers of poly(A) sites from three sequencing platforms</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">For ESTs, the minimum length of valid poly(A) tails was set to be 10 nt. The maximum adenines that were integral part of poly(A) tails but also mapped to the genome were set to be 4 nt. The maximum nontemplated nucleotide addition (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Jin and Bian 2004</a>) between the poly(A) tails and poly(A) sites [<em>e.g.</em>, the nucleotides between the mapped end of the cDNA and the start of its poly(A) tail] was allowed to be 3 nt. All ESTs with identified poly(A) sites were mapped to the reference genome sequences with GMAP (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wu and Watanabe 2005</a>). The EST-to-genome mapping results were then analyzed and filtered for valid genomic hits by the use of a similar protocol as described previously (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Liang <em>et al.</em> 2008</a>). For 454 and Illumina data, the poly(A) sites were identified using our in-house tool SCOPE++ (<a href="http://code.google.com/p/scopeplusplus/">http://code.google.com/p/scopeplusplus/</a>), with at least 15 nt in length and 95% identity in purity, and then the sequence reads with a valid poly(A) tail were mapped to the genome by the use of Helicos Heliosphere (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Ozsolak <em>et al.</em> 2010</a>). Because poly(A) tails detected in ESTs or RNA-Seq reads were posttranscriptional, they should not be mapped to the genome except in the case in which internal priming was likely to occur. The internal priming was defined as that there were at least 6−7 consecutive adenines (As) from 10 nt-window in −10 to +10 region around poly(A) sites in genomic sequences (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Tian <em>et al.</em> 2005</a>). Therefore, individual genomic poly(A) sites were finally determined through the sequence-to-genome mapping results. The poly(A) sites that might be due to internal priming were filtered out. All individual poly(A) sites were organized to generate a list of nonredundant, unique poly(A) sites with distinctive genomic coordinates.</p><p id="p-13">It is well-known that polyadenylation complex appears to be “sloppy” when cleaving pre-mRNA before polyadenylation and therefore generates microheterogeneity in poly(A) sites (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Tian <em>et al.</em> 2005</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Liang <em>et al.</em> 2008</a>). To circumvent this, iterative clustering of adjacent unique poly(A) sites was performed in each chromosome. We developed an algorithm based on density theory and restraint theory in data mining to cluster poly(A) sites. In our algorithm, we used the advanced Ward’s minimum variance method (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Szekely and Rizzo 2005</a>), a popular hierarchical clustering method. To obtain high-quality poly(A) site clusters (PACs), all individual unique poly(A) sites with the support of at least three ESTs or RNA-Seq reads were used for clustering. These PACs were used to extract 100 nt upstream and 50 nt downstream sequences from the reference genome for further analyses [each poly(A) site is defined as −1 position in genomic coordinates (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>)]. The final PAC sites and relevant sequences extracted from the genome were saved in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Poly(A) signal variation in different parts of the genome</h3><p id="p-14">In this study, the gene annotation (v4.3) from Phytozome (<a href="http://www.phytozome.net">http://www.phytozome.net</a>) was used to determine the regions of 3′-UTR, 5′-UTR, CDS, intron, and intergenic regions in <em>C. reinhardtii</em> genome. Furthermore, 3′-UTR regions were expanded to overcome potential incomplete annotation (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>), so we can investigate the distribution of poly(A) sites in the immediate downstream of 3′-UTR region (namely the intergenic region) more accurately.</p><p id="p-15">A new program, SignalSleuth2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FileS2.zip">File S2</a>), an improved version of SignalSleuth (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Loke <em>et al.</em> 2005</a>), was developed to perform exhaustive search of highly frequent signals within a defined region for the given sequence data. This program offers new functionalities including multiple scanning modes and Position-Specific Scoring Matrix. It also provides a signal distance optional parameter (-gap) if there is more than one signal in the sequence, called distribution scanning mode. Such mode can provide signal distribution in the given region, avoid missing counts for a specific motif that appears multiple times in that region, and prevent over-counting of overlapping motifs. For example, ATATATAT would represent once for ATATAT and TATATA motif search when the gap was set to be two. If we do not consider the distance (namely gap = 0), we called it an overlapping scanning mode. In addition, if there was more than one signal in a sequence, the program would choose the signal closest to the poly(A) site, this was called frequency scanning mode. For each mode, the program also provided Position-Specific Scoring Matrix results as one type of outputs. To check the statistic significance, we used Z-score to inspect the significance of the signals from Regulatory Sequence Analysis Tools (RSAT) based on Markov chain models (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">van Helden <em>et al.</em> 1998</a>).</p></div><div id="sec-4" class="subsection"><h3>APA analysis</h3><p id="p-16">Within a PAC, the unique poly(A) site that had the most sequence read support was treated as the representative site of this PAC. Therefore, PACs represented polyadenylation sites without microheterogeneity. Thus, APA genes were defined as those having at least two PACs, and non-APA genes should have only one PAC [which was also called constitutive poly(A) site]. For APA genes with multiple PACs, the PAC with 75% or more supporting reads was categorized as a strong poly(A) site, and other sites in the same gene were called weak poly(A) sites. If no strong site was detected in a APA gene, then all of the poly(A) sites were classified as median poly(A) sites (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Hu <em>et al.</em> 2005</a>). To investigate Gene Ontology (GO) functions for high-quality APA genes (<em>i.e.</em>, genes with at least five PACs), the GO annotation (v4.0) file of <em>C. reinhardtii</em> was downloaded from JGI. Then GOEAST (Gene Ontology Enrichment Analysis Software Toolkit) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zheng and Wang 2008</a>) was used to determine the enriched GO terms and their significance (p-value).</p></div><div id="sec-5" class="subsection"><h3>The splicing events in <em>C. reinhardtii</em></h3><p id="p-17">A total of more than 7 million <em>C. reinhardtii</em> cDNA sequences of both Sanger and NGS technologies were used to collect splicing events (P. K. Raj Kumar, O. Vallon and C. Liang). An automated version of PASA (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Campbell <em>et al.</em> 2006</a>) integrated with GMAP (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wu and Watanabe 2005</a>) was used to deduce alternatively spliced isoforms. Seqclean (<a href="http://compbio.dfci.harvard.edu/tgi/software/">http://compbio.dfci.harvard.edu/tgi/software/</a>) was used to trim off adapters, links, and poly(A)/(T) tails from raw sequence reads. Then, clean cDNAs reads were mapped to AUGUSTUS u10.2 gene annotation for <em>C. reinhardtii</em> with predicted protein mRNAs (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Stanke <em>et al.</em> 2008</a>) to define the mapped cDNAs as protein coding transcript fragments and the unmapped as non-coding transcript fragments. The results of protein coding and non-coding transcript fragments were separately used in PASA to deduce alternative splicing isoforms. All of the constitutively spliced introns (<em>i.e.</em>, introns that were spliced in all the isoforms) and retained introns (<em>i.e.</em>, introns that were retained in some isoforms but spliced in others) were extracted from the PASA database using Perl scripts, and the genes containing introns were then categorized into four different groups based on coding/noncoding and constitutive/retained introns (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464358/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464358/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of PACs in different gene types</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Poly(A) site collection and distribution in the <em>C. reinhardtii</em> genome</h3><p id="p-19">To study the extent of APA in the <em>C. reinhardtii</em> genome, we systematically acquired raw transcriptome sequence data from various sources (detailed in the section <em>Materials and Methods</em>). From 23,535,153 ESTs and RNA-Seq reads with poly(A) tails, a total of 256,771 unique poly(A) sites and 97,479 PACs were obtained after removing redundancy and potential internal priming candidates (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-20">On the basis of <em>C. reinhardtii</em> genome annotation v4.3, we found that ~44% and 40% of unique poly(A) sites distributed in 3′-UTR and intergenic regions, respectively. The unique poly(A) sites in 5′-UTR, CDS and intron regions were approximately 1%~10% (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). To circumvent potential genome annotation errors in 3′-UTR, the distribution of the intergenic poly(A) sites within 1000 nt to the 3′-UTR end of all genes was investigated (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1">Supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FileS1.zip">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FileS2.zip">File S2</a>). It was found that there were many sites close to the annotated 3′-UTR ends. For example, there were 54% of the poly(A) sites located within 50 nt of the annotated 3′-UTR ends (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS1.pdf">Figure S1</a>). So, empirically 50 nt was used to expand 3′-UTR length to avoid annotation errors. This means that any poly(A) site located within 50 nt from the end of the annotated 3′-UTR will be treated as a poly(A) site in 3'-UTR of that gene.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/871/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of poly(A) sites in the genic and intergenic regions in different datasets. (A) All unique poly(A) sites from Sanger expressed sequence tags (ESTs), 454, and Illumna before 50-nt extension. (B) All unique poly(A) sites from ESTs, 454, and Illumna after 50-nt extension. (C) Poly(A) site clusters (PACs) after 50-nt extension from EST data. (D) PACs after 50-nt extension from 454 data. (E) PACs after 50-nt extension from Illumina data. (F) PACs after 50-nt extension from all three combined data. UTR, untranslated region; CDS, coding sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-343433280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The distribution of poly(A) sites in the genic and intergenic regions in different datasets. (A) All unique poly(A) sites from Sanger expressed sequence tags (ESTs), 454, and Illumna before 50-nt extension. (B) All unique poly(A) sites from ESTs, 454, and Illumna after 50-nt extension. (C) Poly(A) site clusters (PACs) after 50-nt extension from EST data. (D) PACs after 50-nt extension from 454 data. (E) PACs after 50-nt extension from Illumina data. (F) PACs after 50-nt extension from all three combined data. UTR, untranslated region; CDS, coding sequences.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/871/F1.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/5/871/F1.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/871/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/871/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464339" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">The distribution of poly(A) sites in the genic and intergenic regions in different datasets. (A) All unique poly(A) sites from Sanger expressed sequence tags (ESTs), 454, and Illumna before 50-nt extension. (B) All unique poly(A) sites from ESTs, 454, and Illumna after 50-nt extension. (C) Poly(A) site clusters (PACs) after 50-nt extension from EST data. (D) PACs after 50-nt extension from 454 data. (E) PACs after 50-nt extension from Illumina data. (F) PACs after 50-nt extension from all three combined data. UTR, untranslated region; CDS, coding sequences.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Single-nucleotide profiles and poly(A) signal variation</h3><p id="p-22">Single-nucleotide profiles around cleavage or poly(A) sites have been demonstrated to be a signature of poly(A) signals (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Loke <em>et al.</em> 2005</a>; <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">b</a>). In general, there are three different groups of poly(A) signals in plants: the poly(A) site and its surrounding sequences called the cleavage element, the sequences from the NUE region of the poly(A) site, and the far upstream element (FUE) region from the poly(A) site (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Loke <em>et al.</em> 2005</a>). To analyze the poly(A) signals, 100 nt upstream and 50 nt downstream sequences for each PAC were extracted from the <em>C. reinhardtii</em> genomic sequences. We first examined the single-nucleotide profiles and mainly focused on the poly(A) signals in NUE and then FUE, because NUE is the most conserved region where the canonical poly(A) signal and its variants exist, and FUE also contained a group of nucleotide biased signals (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Loke <em>et al.</em> 2005</a>; <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>,<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">b</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Xing and Li 2011</a>).</p><p id="p-23">The single nucleotide profile from all PAC data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) showed that the NUE region (around −28 to −5) had a high A-peak (60.39%) at the poly(A) site (−1 position), with a high G content (33.72% in average) in the whole investigated region (−100 to +50). Then each dataset from different sequencing platforms was specifically investigated, and it was shown that the difference is caused by Illumina data, because only the profile from Illumina has a high G content (34.24% in average) in the whole region (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The single nucleotide profiles from ESTs and 454 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C and D</a>) were much similar to each other with normal G content (30.09% and 29.67% in average) in the whole regions, especially in the NUE regions (25.26% and 25.79% in average). Such a feature is much like what was described previously by <a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> (2008a)</a>. Regarding the NUE regions from the four profiles shown in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, it was clear that there were high U- and A-peaks and C- and G-troughs in <em>C. reinhardtii</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/871/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The single-nucleotide profiles from different datasets. (A) All combined poly(A) site clusters (including ESTs, 454 and Illumina). (B) Illumina data. (C) Expressed sequence tag data EST data. (D) 454 data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-343433280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The single-nucleotide profiles from different datasets. (A) All combined poly(A) site clusters (including ESTs, 454 and Illumina). (B) Illumina data. (C) Expressed sequence tag data EST data. (D) 454 data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/871/F2.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/5/871/F2.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/871/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/871/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464280" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">The single-nucleotide profiles from different datasets. (A) All combined poly(A) site clusters (including ESTs, 454 and Illumina). (B) Illumina data. (C) Expressed sequence tag data EST data. (D) 454 data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Similar results were also obtained from the pentamer signals in the NUE regions among different datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS2.pdf">Figure S2</a>). The signal profiles from combined poly(A) site data show that GGGGG has the highest frequency (28.66%), but it cannot be significantly differentiated from the background noises due to a low Z-score (its Z-score was too low to be on the list generated by RSAT; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS2.pdf">Figure S2</a>A). Pentamer UGUAA was found to be a significant (Z-score = 32.88 by using a third-order Markov model) and highly frequent (13.39%) signal in the NUE region, and G-rich pentamers GGUGG and GUGGG were also detected with significantly high frequencies and Z-scores (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a> for details). Similar to all combined data, UGUAA and G-rich pentamers (<em>e.g.</em>, GGUGG and GUGGG) also were detected with high frequencies and Z-scores in the Illumina dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS2.pdf">Figure S2</a>B and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>). However, the pentamers from ESTs and 454 were similar to the results of <a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> (2008a)</a>, considering the UGUAA (&gt;30%) and UGUAA-related motifs were dominant pentamers whereas G-rich pentamers are not found in those data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS2.pdf">Figure S2</a>, C and D, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>). Such results indicated that Illumina data may be biased with overexpressed G nucleotide in sequencing or reveal a new polyadenylation feature in <em>C. reinhardtii</em>.</p><p id="p-26">Among the FUE regions (around −100 to −28), G and C nucleotides had greater content than A and U nucleotides in the single nucleotide profiles (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). Consistently, the top detected pentamers were also GC-rich signals, especially in all combined and Illumina data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS3.pdf">Figure S3</a>, A and B, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS2.pdf">Table S2</a>). Although pentamer GUGUG always had the greatest frequencies (&gt;21%) in all the datasets, yet it was not significant considering the Z-score (“None” means Z-score was too low to be on the list generated by RSAT). In the four datasets, the most frequent pentamers with significance were G-rich signals (<em>e.g.</em>, GGUGG, GUGCG, and GCGUG), especially in all combined and Illumina data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS2.pdf">Table S2</a>); EST and 454 data had similar profiles and signals and no any single dominant signal (<em>e.g.</em>, UGUAA in NUE) was found in FUE, which was also shown in other studies (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Loke <em>et al.</em> 2005</a>; <a id="xref-ref-26-10" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>).</p></div><div id="sec-9" class="subsection"><h3>Poly(A) sites in different genic and intergenic regions</h3><p id="p-27">It was demonstrated that APA sites located in different regions of the gene body may use different kinds of poly(A) signals (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>). To understand the variations of polyadenylation, the single-nucleotide profiles and poly(A) signals were investigated with PAC data from the three different sequencing platforms as well as the combined one.</p><p id="p-28">As shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, 5′-UTRs always have similar proportion of unique poly(A) sites in all four PAC datasets including ESTs (0.89%, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>), 454 (1.77%, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>), Illumina (1.51%, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1E</a>), and combined data (1.61%, <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1F</a>). In <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS4.pdf">Figure S4</a>, it can be seen that NUE regions (−28 to −5) of 5′-UTR poly(A) sites have a clear U-peak followed by an A-peak, and the peaks are obvious in EST and 454 data (&gt;40%), whereas Illumina data do not show such high peaks (~33%) in comparison with EST and 454 data. Poly(A) sites (−1 position) always possess a high A-peak (~50%) in all three datasets. In term of poly(A) signals, UGUAA is the most frequent signal in the NUE regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS4.pdf">Figure S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>). UGUAA has an obviously greater frequency than other pentamers, especially in EST and 454 data. Because the collected poly(A) site number is too low in 5′-UTRs (~1%), RSAT cannot give the Z-score for the signals from all three datasets.</p><p id="p-29">3′-UTRs always have the greatest poly(A) site numbers (<em>i.e.</em>, 38.44% in Illumina and &gt;67% in EST and 454 PAC datasets, see <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>) in all the genic and intergenic regions. From the single nucleotide profiles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS5.pdf">Figure S5</a>), there are clearly an U-peak (&gt;32%) following by an A-peak (&gt;34%) in their NUE regions and a high A-peak (&gt;49%) at poly(A) sites in all three datasets. In the NUE regions, the canonical pentamer UGUAA shows the greatest frequency (&gt;24%) and significance (Z-score &gt; 13 with order-3 Markov model) in all three datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>). Other UGUAA-like pentamers (<em>e.g.</em>, CUGUA and UGCAA) are also detected with relatively high frequencies (&gt;7%) and significance (Z-score &gt; 9).</p><p id="p-30">The frequency of PACs in CDS varies dramatically from 0.46% (ESTs) to 8.77% (Illumina) (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>). Different from 5′-UTRs and 3′-UTRs, the single-nucleotide profiles in CDS have no clear U-peak and A-peak detected in the relevant NUE regions (−28 to −5) in all three datasets; and at poly(A) sites (−1 position) the A-peaks are only found in 454 and Illumina datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS6.pdf">Figure S6</a>). For the pentamers in the NUE region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>), UGUAA still has the greatest frequency (19.61%) in ESTs (Z-score cannot be given by RSAT because of small dataset). Yet in the 454 dataset, UGUAA still can be found but with low frequency (5.50%). The Illumina dataset does not have UGUAA in the top 50 ranked pentamers but other signals (<em>e.g.</em>, GGCGG and GCGGG) are found in the NUE region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-31">ESTs and 454 have similarly low percentage (0.66% and 1.23%) of poly(A) sites located in introns, yet there are more poly(A) sites detected in introns in Illumina data (14.98%) (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>). The single-nucleotide profiles and signals in the NUE regions of intronic poly(A) sites are also similar between EST and 454 datasets, there is a clear U-peak followed by an A-peak and UGUAA (~35%) is dominant in NUE regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS7.pdf">Figure S7</a>). In Illumina dataset, however, the single-nucleotide profile has dramatically high G content (47.53% in average) and no clear U-peak and A-peak is detected; UGUAA is not detected but G-rich pentamers (<em>e.g.</em>, GGUGG) are found with high frequencies (47.80%) and significance (Z-score = 22.33) in the NUE regions of intronic poly(A) sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-32">The percentages of PACs in intergenic regions increase from ESTs, 454 to Illumina (13.84%, 27.18%, and 36.31%, respectively, <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>). For intergenic poly(A) sites, the single-nucleotide profiles and pentamer signals are similar in ESTs and 454; an U-peak is followed by an A-peak in the NUE regions and a dominant A-peak is also found at poly(A) site (−1 position); UGUAA has the highest frequency (34.05% and 21.12%) and significance (Z-score = 4.98 and 10.46) in these two datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS8.pdf">Figure S8</a>, A and B). In Illumina data, the single nucleotide profile always has high G content (33.17% in average) except at poly(A) sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS8.pdf">Figure S8</a>C), which is similar to those intronic poly(A) sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS7.pdf">Figure S7</a>C). There is a weak U-peak (25.84%) followed by an A-peak (23.63%) in the NUE region, and UGUAA is not dominant (8.57% in frequency) but G-rich signals (<em>e.g.</em>, GUGGG, GGUGG, and GAGGG) are found with high significance (Z-score &gt; 9) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-33">Our results imply that different platforms (with different sequencing depths and coverage) might pick up different groups of transcripts that vary in poly(A) site selection. Although independent confirmations are needed, <em>e.g.</em>, authenticity of the poly(A) sites identified in each platform, one would assume deeper sequencing approach like Illumina might generate better coverage of poly(A) sites.</p></div><div id="sec-10" class="subsection"><h3>Introns and CDS with PACs are larger in sizes</h3><p id="p-34">Previous reports indicated that poly(A) sites tend to be located in longer introns in plants (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>) and animals (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2007</a>). With available larger datasets, we were able to analyze the similar issue in <em>C. reinhardtii</em>. Indeed, we found that the introns with PACs are much longer than those without an internal PAC. There were 13,476 PACs located in 11,780 introns from 5,764 genes. Three control groups (NPA1, NPA2 and NPA3) with the same intron counts (11,780) but without a PAC were randomly selected as control datasets. We found that the average length of introns with and without PACs are 451 nt (median = 345) and 258 nt (median = 217), respectively. Apparently, the mean length of introns with PACs are significantly larger than those without PACs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, Wilcoxon tests, p-value &lt; 2.2e-16).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/871/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The length difference of intron and coding sequences (CDS) with and without poly(A) site clusters. (A) Intron. (B) CDS. PA: poly(A) sites; NPA1: control group 1 without poly(A) sites; NPA2: control group 2 without poly(A) sites; NPA3: control group 3 without poly(A) sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-343433280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The length difference of intron and coding sequences (CDS) with and without poly(A) site clusters. (A) Intron. (B) CDS. PA: poly(A) sites; NPA1: control group 1 without poly(A) sites; NPA2: control group 2 without poly(A) sites; NPA3: control group 3 without poly(A) sites.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/871/F3.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/5/871/F3.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/871/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/871/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464378" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">The length difference of intron and coding sequences (CDS) with and without poly(A) site clusters. (A) Intron. (B) CDS. PA: poly(A) sites; NPA1: control group 1 without poly(A) sites; NPA2: control group 2 without poly(A) sites; NPA3: control group 3 without poly(A) sites.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Similarly, the CDS regions with PACs are also longer than those without a PAC. In our data analysis, 8102 PACs were found in 6953 CDS regions of 4937 genes. Similarly to the analysis in introns, in three randomly selected control groups of CDS regions (without PACs), the average length of CDS with and without PACs are 513 nt (median = 224) and 235 nt (median = 128), respectively. Statistically, the sizes of CDS with PACs are significantly larger than those without PACs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, Wilcoxon tests, p-value &lt; 2.2e-16). Our results suggest a similar overarching phenomenon where longer introns and CDSs are better targets for the polyadenylation machinery.</p></div><div id="sec-11" class="subsection"><h3>The preference of polyadenylation on spliced genes in <em>C. reinhardtii</em></h3><p id="p-37">Identification of a large number of intronic poly(A) sites allowed us to examine the distribution of them in different kinds of introns: constitutive (spliced in all transcripts) and retained introns (not spliced in some transcripts). There are 13.82% (13,476) PACs located in intron regions in the combined dataset (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1F</a>), such sites are referred to as intronic PACs. Intronic PACs based on the categories such as constitutive/retained introns from protein-coding and non-coding genes are shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. There were 139,859 (=134,708 + 5151) introns from coding genes and 2666 (=2550 + 116) introns from potential noncoding genes. Intronic PACs are almost equal percentage-wise between the introns of coding and noncoding genes (compare mean value of 5.63% and 5.16%, respectively) in average, although constitutively spliced introns are over two times of retained introns (compare mean value of 7.47% over 3.32%, respectively) in average. The results may indicate that polyadenylation machinery in <em>C. reinhardtii</em> has no preference on introns of coding over noncoding genes, but strongly prefers constitutive than retained introns, influencing the selection of splicing sites and gene expression in RNA posttrancriptional regulation.</p></div><div id="sec-12" class="subsection"><h3>Association of poly(A) signal UGUAA usage with transcript levels</h3><p id="p-38">To further investigate the potential function of APA in gene expression regulation, we examined the relationship between the level of transcription and the use of particular poly(A) sites. Based on the definition of four different poly(A) site categories (<em>i.e.</em>, constitutive, strong, median and weak poly(A) sites, see the section <em>Materials and Methods</em> for definition), the numbers and percentages of the four different PAC sites and corresponding genes are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. In comparison with median poly(A) sites, there are less constitutive and strong poly(A) sites [2747 (4.93%) and 3653 (6.56%), respectively], and the corresponding genes are also less [2747 (16.05%) and 3653 (21.35%), respectively]. Because weak poly(A) sites are located in the same genes with strong sites, so it has the same gene number (3653, 21.35%) with strong sites. Most PACs are median poly(A) sites (39,044, 70.07%) and nearly half of all annotated genes (7946, 46.43%) possess median sites.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464329/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464329/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of PACs in different categories</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">To study whether different site categories have distinct nucleotide composition and poly(A) signals, the single nucleotide profiles (−50 to +25) and signals in the NUE regions (−28 to −5) are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. It is obvious that strong poly(A) sites have high U-peak (42.70%) and A-peak (45.00%) in the NUE region and dominant A-peak at poly(A) sites in the single nucleotide profile, and UGUAA has the highest frequency (46.45%) and significance (Z-score = 8.04 with order-3 Markov model) among the pentamers of the NUE region (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Then constitutive poly(A) sites have similar features as strong sites but with high G-content (34.67% in average) in the single nucleotide profile and G-rich signals in the NUE region (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). Both weak and median poly(A) sites have similar features: high G-content (34.89% and 36.62% respectively) in the single nucleotide profiles and low UGUAA frequencies, especially in weak sites where UGUAA is not detected in the top 50 frequent pentamers (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, C and D</a>). This result is indicative of the clear association of the best poly(A) signal UGUAA and transcript levels, another evident reflecting the significance of the dominant signal in <em>C. reinhardtii</em>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/871/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The single nucleotide profiles (&#x2212;50 to +25) and putative poly(A) signals in near upstream element (NUE) region (&#x2212;28 to &#x2212;5) from different poly(A) site categories. (A) Constitutive poly(A) sites. (B) Strong poly(A) sites. (C) Weak poly(A) sites. (D) Median poly(A) sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-343433280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The single nucleotide profiles (&#x2212;50 to +25) and putative poly(A) signals in near upstream element (NUE) region (&#x2212;28 to &#x2212;5) from different poly(A) site categories. (A) Constitutive poly(A) sites. (B) Strong poly(A) sites. (C) Weak poly(A) sites. (D) Median poly(A) sites.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/871/F4.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/5/871/F4.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/871/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/871/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464388" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">The single nucleotide profiles (−50 to +25) and putative poly(A) signals in near upstream element (NUE) region (−28 to −5) from different poly(A) site categories. (A) Constitutive poly(A) sites. (B) Strong poly(A) sites. (C) Weak poly(A) sites. (D) Median poly(A) sites.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>APA extent variation among different datasets</h3><p id="p-42">To examine whether different sequencing platforms generated different number of APA events, we tallied the number of genes with PACs from different datasets (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The APA extent varies dramatically based on different datasets (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). It was found that the APA extent was up to 67.78% of genes according to the total PACs in <em>C. reinhardtii</em>. Yet only 7.87% of genes had APA based on the EST data; following the increase of datasets, the APA extent was dramatically increased up to 63.46% in the Illumina dataset. Statistical analysis showed that there was significant relationship between APA extent and the size of poly(A) datasets (Pearson correlation coefficient: <em>r</em> = 0.995, <em>P</em> = 0.005). Clearly, there were less genes with multiple PACs, especially for high quality group (≥5 PACs), in small datasets. For example, there were only 0.11% and 1.80% genes with at least five PACs based on 6770 and 17,828 PACs in EST and 454 datasets, respectively (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, C and D</a>). In Illumina dataset, in contrast, 23.05% genes had at least five PACs, considering that there are 50,664 PACs in total.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/871/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The gene numbers with different PACs among different datasets. (A) All poly(A) data (including Sanger expressed sequence tags (ESTs), 454, and Illumina). (B) Illumina data. (C) EST data. (D) 454 data. PACs, Poly(A) site clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-343433280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The gene numbers with different PACs among different datasets. (A) All poly(A) data (including Sanger expressed sequence tags (ESTs), 454, and Illumina). (B) Illumina data. (C) EST data. (D) 454 data. PACs, Poly(A) site clusters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/871/F5.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/5/871/F5.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/871/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/871/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464356" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">The gene numbers with different PACs among different datasets. (A) All poly(A) data (including Sanger expressed sequence tags (ESTs), 454, and Illumina). (B) Illumina data. (C) EST data. (D) 454 data. PACs, Poly(A) site clusters.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464291/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464291/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">APA extent variation among the four datasets</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">It is of great interest for us to answer the question on the potential biological functions of APA, particularly for those genes rich in APA (<em>i.e.</em>, the genes with at least five PACs). We performed GO analysis using the GO annotation v4.0 from JGI and GOEAST (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Zheng and Wang 2008</a>). The results of the highly significant GO functions are listed in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. The result shows that the significant GO terms involve molecular functions and biological processes, mainly including receptor activities, noncoding RNA processing, protein synthesis, hydrolase, and ligase activities. As shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>, we also conducted GO analysis for those genes without any PAC. In comparison with <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>, it is obviously that non-PAC genes mainly involve in amino acid activation and protein synthesis process, whereas high-quality APA genes could be related with RNA regulation, RNA processing, and signal response (receptors), among others. Further research on these gene categories may reveal detailed functionalities of these APA events.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464340/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464340/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">The most significant GO functions in high-quality APA genes with five or more PACs</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464330/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464330/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">The most significant GO functions in non-PAC genes</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-48">Deep sequencing technologies provided us with unprecedented genome and transcriptome information. With a much larger collection of poly(A) site data in the model organism <em>C. reinhardtii</em>, we are able to perform an in-depth analysis of the distribution of polyadenylation sites, polyadenylation signals and their relationship with splicing and gene expression levels. Our results provide further information about this model algal organism in terms of its RNA processing profiles and the potentials of APA in regulating gene expression.</p><div id="sec-15" class="subsection"><h3>Poly(A) site distribution and polyadenylation profile variation in the genome and different datasets</h3><p id="p-49">According to current genome annotation, there are ~38% of PACs located in intergenic regions. This may be a result of inaccurate or incomplete annotation from insufficient numbers of ESTs/cDNA and reads used for the gene annotation. After expanding the 3′-end of genome annotation by 50 nt, the unique 3′-UTR poly(A) sites increase to 55.67%, whereas there still are 28.48% sites located within intergenic regions. These intergenic poly(A) sites might be from unannotated genes or indicate the presence of some novel polyadenylated transcripts (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lopez <em>et al.</em> 2006</a>; <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>). Furthermore, &gt;500 bp extension (relative to reference 3′-UTRs) was identified in long intergenic noncoding RNAs in mouse and human brain tissues (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Miura <em>et al.</em> 2013</a>). Such notable difference may be due to the biological differences between green algae and mammals (<em>e.g.</em>, mRNA abundance and gene compactness).</p><p id="p-50">From poly(A) site distribution, which is similar to the results of <a id="xref-ref-26-11" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> (2008a)</a> in which they used 16,952 <em>in silico</em>-verified poly(A) sites from EST sequences (<a id="xref-ref-26-12" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>), it is shown that ESTs have much more similar distribution features to 454 than Illumina in the whole genome among the three datasets from different sequencing platforms (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1</a>). This may be caused by the difference of deep-sequencing reads considering that the reads in 454 are much longer (&gt;400 bp) than Illumina (~50 bp). Moreover, deep-sequencing (454 and Illumina) can reveal much more poly(A) sites than Sanger sequencing. Interestingly, 5′-UTRs always have low site abundance (~1%) across the three datasets, which may suggest that 5′-UTRs are more stable in transcription and/or translation and may be less involved in APA events. Such result is consistent with the hypothesis that shorter pre-AUG poly(A) tract in 5′-UTRs can bind to translation initiation factors to enhance translation initiation, while a long tract (≥12) will bind to Pab1p (a poly(A) binding protein) resulting in repression of translation in yeast (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Xia <em>et al.</em> 2011</a>).</p><p id="p-51">The variations of single nucleotide profiles and poly(A) signals are clearly different among different genic and intergenic regions. 3′-UTRs have the most dominant UGUAA signal, which is also consistent with the previous papers (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wodniok <em>et al.</em> 2007</a>; <a id="xref-ref-26-13" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> 2008a</a>) in <em>C. reinhardtii</em>. No clear NUE region or significant poly(A) signal is detected in CDS poly(A) sites in all the three datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1/FigureS6.pdf">Figure S6</a>), and similar result also was found in Arabidopsis (<a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>). In Illumina data, the single-nucleotide profiles and signals in intron and intergenic regions have much greater G content than those in EST and 454 data, and such feature could be caused by extremely biased base composition in Illumina sequencing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Oyola <em>et al.</em> 2012</a>), because the genome of <em>C. reinhardtii</em> is strongly GC-biased considering that the GC content is up to 63.45%, which is significantly greater than other multicellular organisms (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Merchant <em>et al.</em> 2007</a>). Through DRS technology, novel poly(A) signals TTTTTTTTT and AAWAAA (W = A/T) were found around NUE regions in human (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Ozsolak <em>et al.</em> 2010</a>). Such results may be also caused by sequencing bias because human genome is AT-rich. Another possible explanation is internal priming, and it has been reported that internal priming would cause artifacts within protein-coding regions by reverse transcriptase when using an oligo(dT)-based primers in Arabidopsis DRS sequencing (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sherstnev <em>et al.</em> 2012</a>). The different single nucleotide profiles and poly(A) signals among different genic regions in <em>C. reinhardtii</em> indicate that polyadenylation mechanism may be diversified in eukaryotes, and more deep-sequencing data (beyond 3′-UTRs) are needed to give further analysis in other genic regions beyond 3′-UTRs.</p></div><div id="sec-16" class="subsection"><h3>The influence of polyadenylation on splicing events in <em>C. reinhardtii</em></h3><p id="p-52">In our study, CDS and introns with poly(A) sites are longer than those without poly(A) sites, and similar results are also found in intronic poly(A) sites in Arabidopsis (<a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>) and human (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2007</a>). Although a weak 5′ splice site (5′ss) is detected in introns with poly(A) sites in Arabidopsis (<a id="xref-ref-42-10" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>) and human (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2007</a>), no significant difference in the strength of 5′ ss signal was detected between the introns with poly(A) sites and without poly(A) sites in <em>C. reinhardtii</em> in our study (data not shown). Therefore, it seems that polyadenylation may not prefer weak 5′ss, and UGUAA-dominated species (<em>e.g.</em>, <em>C. reinhardtii</em>) may have distinct splice site selection in polyadenylation from AAUAAA-dominated species (<em>e.g.</em>, Arabidopsis and human).</p><p id="p-53">Although intron retention made up half of the alternative splicing events (305 of 611) in <em>C. reinhardtii</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Labadorf <em>et al.</em> 2010</a>), our study shows that constitutively spliced introns are over two times more enriched with poly(A) sites than that of retained introns, suggesting constitutively spliced introns have stronger preference to polyadenylation than retained introns. So it is possible that the preference of polyadenylation to constitutive splicing may be the result of interplay between polyadenylation and splicing. Another explanation could be that constitutively spliced introns are more likely than retained introns to contain noncoding RNAs, which are stabilized by polyadenylation after excision by splicing.</p></div><div id="sec-17" class="subsection"><h3>Alternative polyadenylation in <em>C. reinhardtii</em></h3><p id="p-54">It was found that the extent of APA in <em>C. reinhardtii</em> is up to 68% based on our combined data. We used a stringent criterion (each site with at least three ESTs/reads support) and new annotation file (v4.3) to filter the unique poly(A) site and obtain PACs that can appropriately define high-quality APA events. Interestingly, the APA is only found in 8% of genes based on our EST data (from 21,041 unique poly(A) sites); whereas <a id="xref-ref-26-14" class="xref-bibr" href="#ref-26">Shen <em>et al.</em> (2008a)</a> reported up to 33% APA based on 16,952 poly(A) sites from EST data and genome annotation v3.1. It is clear that the extent of APA depends on the size of dataset and accuracy of genome annotation, and similar results were also obtained in Arabidposis and rice through deep-sequencing (<a id="xref-ref-42-11" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Shen <em>et al.</em> 2011</a>).</p><p id="p-55">In our study, more poly(A) sites were detected in CDS and intron regions, especially in deep-sequencing data (454 and Illumina); and there was no clear NUE region and significant poly(A) signal in CDS poly(A) sites. As known, APA located in intron/CDS regions <em>vs.</em> 3′-UTRs would result in dramatically different mRNA isoforms. APAs from intron/CDS and 3′-UTR may involve in different regulation mechanisms (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Di Giammartino <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Thomas <em>et al.</em> 2012</a>). It was mentioned that poly(A) tag sequencing results from protein-coding regions (CDS/intron regions) may be dubious because of internal priming (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Sherstnev <em>et al.</em> 2012</a>), so more poly(A) sites detected in CDS and intron regions, especially in Illumina data, may not reflect the real number of poly(A) sites. It is equally possible that the differences of poly(A) signal in CDS/intron and 3′-UTR relate to different APA mechanisms (<a id="xref-ref-42-12" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Thomas <em>et al.</em> 2012</a>).</p><p id="p-56">Although functions and potential mechanisms of APA have been reported in plants and animals (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Ozsolak <em>et al.</em> 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Singh <em>et al.</em> 2009</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Thomas <em>et al.</em> 2010</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Mangone <em>et al.</em> 2010</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Ozsolak <em>et al.</em> 2010</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Xing and Li 2011</a>; <a id="xref-ref-42-13" class="xref-bibr" href="#ref-42">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Shen <em>et al.</em> 2011</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Di Giammartino <em>et al.</em> 2011</a>), the verified biological functions and mechanism of APA genes in <em>C. reinhardtii</em> have not been reported. Our <em>in silico</em> analysis of APA genes indicates that receptor genes and noncoding RNA as well as protein metabolism are impacted by the APA in <em>C. reinhardtii</em>. However, non-PAC genes are less involved in RNA regulation but in protein synthesis process. The significance of these bioinformatics findings remains to be verified by wet-lab experiments in the near future. The results presented here would, without a doubt, benefit further research in the contribution of APA in gene expression of this and related organisms.</p><p id="p-57">Polyadenylation is an essential step in eukaryotic mRNA posttranscriptional process, yet model organism green alga <em>C. reinhardtii</em> has an apparently different poly(A) signal (UGUAA) from most eukaryotes (<em>e.g.</em>, Arabidopsis and human). In this study, through genome-wide analysis from three different sequencing data (Sanger ESTs, 454, and Illumina), it is shown that deep-sequencing detected more new poly(A) sites, especially in CDS, intron, and intergenic regions; moreover, Illumina data revealed overexpressed G content around poly(A) sites, which may indicate unknown feature in polyadenylation or be contaminated in sequencing. The prevalence of different poly(A) signals between CDS and 3′-UTR implies potentially different mechanisms of polyadenylation. Our data suggest that the APA occurs in approximately 68% of <em>C. reinhardtii</em> genes. Moreover, intronic poly(A) sites are more abundant in constitutively spliced introns than retained introns, suggesting an interplay between polyadenylation and splicing. Our results support that APA, as in higher eukaryotes, may play significant roles in increasing transcriptome diversity and gene expression regulation in this green alga species. Our datasets also provide information for accurate annotation of transcript ends in the genome level.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">C.L. and Q.Q.L. managed and coordinated the project. Z.Z. carried out data collection, data analysis, and manuscript writing. X.W. participated in data analysis and developed SignalSleuth2 program. P.K.R.K. provided the splicing data. M.D. implemented poly(A) site clustering. All authors participated in manuscript writing and editing. This project was supported by a NIH AREA grant (1R15GM94732-1 A1 to C.L. and Q.Q.L.). This project is also partially funded by the National Natural Science Foundation of China (nos. 61174161 and 61201358), the Natural Science Foundation of Fujian Province of China (no. 2012J01154), the specialized Research Fund for the Doctoral Program of Higher Education of China (no. 20130121130004 and no. 20120121120038), the Fundamental Research Funds for the Central Universities in China (Xiamen University: nos. 2011121047, 201112G018, 201212G005, and 2013121025), and the Fundamental Research Fund for the university student Creative and Entrepreneurship training program in China (Xiamen University: no. XDDC201210384063).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010249/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-59"><em>Communicating editor: J. Hesselberth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-01-03"><span class="received-label">Received </span>January 3, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-03-08"><span class="accepted-label">Accepted </span>March 8, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Zhao <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.1" data-doi="10.1186/1471-2164-7-327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCampbell%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.spage%253D327%26rft.epage%253D327%26rft.atitle%253DComprehensive%2Banalysis%2Bof%2Balternative%2Bsplicing%2Bin%2Brice%2Band%2Bcomparative%2Banalyses%2Bwith%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-327%26rft_id%253Dinfo%253Apmid%252F17194304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194304&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.2" data-doi="10.1038/nrg3482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elkon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ugalde</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agami</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Alternative cleavage and polyadenylation: extent, regulation and function.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D496%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3482%26rft_id%253Dinfo%253Apmid%252F23774734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23774734&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.3" data-doi="10.1016/j.molcel.2011.08.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Giammartino</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manley</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mechanisms and consequences of alternative polyadenylation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDi%2BGiammartino%26rft.auinit1%253DD.%2BC.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D853%26rft.epage%253D866%26rft.atitle%253DMechanisms%2Band%2Bconsequences%2Bof%2Balternative%2Bpolyadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.08.017%26rft_id%253Dinfo%253Apmid%252F21925375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925375&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295309800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.4" data-doi="10.1073/pnas.96.24.14055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graber</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohr</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">In silico detection of control signals: mRNA 3′-end-processing sequences in diverse species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">14055</span>–<span class="cit-lpage">14060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGraber%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D96%26rft.issue%253D24%26rft.spage%253D14055%26rft.epage%253D14060%26rft.atitle%253DIn%2Bsilico%2Bdetection%2Bof%2Bcontrol%2Bsignals%253A%2BmRNA%2B3%2527-end-processing%2Bsequences%2Bin%2Bdiverse%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.24.14055%26rft_id%253Dinfo%253Apmid%252F10570197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjQvMTQwNTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.5" data-doi="10.1261/rna.2107305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilusz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bioinformatic identification of candidate cis-regulatory elements involved in human mRNA polyadenylation.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1485</span>–<span class="cit-lpage">1493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2107305%26rft_id%253Dinfo%253Apmid%252F16131587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMC8xNDg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.6" data-doi="10.1186/1471-2164-13-641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Plant polyadenylation factors: conservation and variety in the polyadenylation complex in plants.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHunt%26rft.auinit1%253DA.%2BG.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D641%26rft.epage%253D641%26rft.atitle%253DPlant%2Bpolyadenylation%2Bfactors%253A%2Bconservation%2Band%2Bvariety%2Bin%2Bthe%2Bpolyadenylation%2Bcomplex%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-641%26rft_id%253Dinfo%253Apmid%252F23167306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23167306&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.7" data-doi="10.1261/rna.7610404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nontemplated nucleotide addition prior to polyadenylation: a comparison of <em>Arabidopsis</em> cDNA and genomic sequences.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1695</span>–<span class="cit-lpage">1697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.7610404%26rft_id%253Dinfo%253Apmid%252F15388874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMS8xNjk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.8" data-doi="10.1186/1471-2164-11-114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labadorf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Link</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of alternative splicing in <em>Chlamydomonas reinhardtii</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLabadorf%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.spage%253D114%26rft.epage%253D114%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Balternative%2Bsplicing%2Bin%2BChlamydomonas%2Breinhardtii.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-114%26rft_id%253Dinfo%253Apmid%252F20163725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20163725&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.9" data-doi="10.1534/genetics.107.085605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Expressed sequence tags with cDNA termini: previously overlooked resources for gene annotation and transcriptome exploration in <em>Chlamydomonas reinhardtii</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiang%26rft.auinit1%253DC.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D93%26rft.atitle%253DExpressed%2BSequence%2BTags%2BWith%2BcDNA%2BTermini%253A%2BPreviously%2BOverlooked%2BResources%2Bfor%2BGene%2BAnnotation%2Band%2BTranscriptome%2BExploration%2Bin%2BChlamydomonas%2Breinhardtii%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085605%26rft_id%253Dinfo%253Apmid%252F18493042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc5LzEvODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.10" data-doi="10.1104/pp.105.060541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loke</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahlberg</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strenski</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Compilation of mRNA polyadenylation signals in Arabidopsis revealed a new signal element and potential secondary structures.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.060541%26rft_id%253Dinfo%253Apmid%252F15965016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM4LzMvMTQ1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.11" data-doi="10.1261/rna.136206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granjeaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghattas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautheret</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The disparate nature of “intergenic” polyadenylation sites.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1794</span>–<span class="cit-lpage">1801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.136206%26rft_id%253Dinfo%253Apmid%252F16931874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNzk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.12" data-doi="10.1073/pnas.1321025111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Role of alternative polyadenylation in epigenetic silencing and antisilencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321025111%26rft_id%253Dinfo%253Apmid%252F24335801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiIxMTEvMS85IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.13" data-doi="10.1016/0378-1119(95)00178-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mages</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresnar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruderlein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Volvox carteri alpha 2- and beta 2-tubulin-encoding genes: regulatory signals and transcription.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMages%26rft.auinit1%253DW.%26rft.volume%253D160%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D54%26rft.atitle%253DVolvox%2Bcarteri%2Balpha%2B2-%2Band%2Bbeta%2B2-tubulin-encoding%2Bgenes%253A%2Bregulatory%2Bsignals%2Band%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252895%252900178-9%26rft_id%253Dinfo%253Apmid%252F7628715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(95)00178-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7628715&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RL01700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.14" data-doi="10.1126/science.1191244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangone</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoharan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The landscape of <em>C. elegans</em> 3′UTRs.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1191244%26rft_id%253Dinfo%253Apmid%252F20522740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5MC80MzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.15" data-doi="10.1016/j.tig.2007.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The impact of next-generation sequencing technology on genetics</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMardis%26rft.auinit1%253DE.%2BR.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D133%26rft.epage%253D141%26rft.atitle%253DThe%2Bimpact%2Bof%2Bnext-generation%2Bsequencing%2Btechnology%2Bon%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.12.007%26rft_id%253Dinfo%253Apmid%252F18262675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18262675&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254058200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.16" data-doi="10.1038/nature03959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margulies</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egholm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attiya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome sequencing in microfabricated high-density picolitre reactors.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMargulies%26rft.auinit1%253DM.%26rft.volume%253D437%26rft.issue%253D7057%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DGenome%2Bsequencing%2Bin%2Bmicrofabricated%2Bhigh-density%2Bpicolitre%2Breactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03959%26rft_id%253Dinfo%253Apmid%252F16056220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16056220&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231849100045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.17" data-doi="10.1126/science.1143609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpowicz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Chlamydomonas genome reveals the evolution of key animal and plant functions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMerchant%26rft.auinit1%253DS.%2BS.%26rft.volume%253D318%26rft.issue%253D5848%26rft.spage%253D245%26rft.epage%253D250%26rft.atitle%253DThe%2BChlamydomonas%2BGenome%2BReveals%2Bthe%2BEvolution%2Bof%2BKey%2BAnimal%2Band%2BPlant%2BFunctions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143609%26rft_id%253Dinfo%253Apmid%252F17932292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OC8yNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.18" data-doi="10.1101/gr.146886.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shenker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreu-Agullo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westholm</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Widespread and extensive lengthening of 3′ UTRs in the mammalian brain.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.146886.112%26rft_id%253Dinfo%253Apmid%252F23520388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzUvODEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.19" data-doi="10.1186/1471-2164-13-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oyola</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maslen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manske</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DOyola%26rft.auinit1%253DS.%2BO.%26rft.volume%253D13%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DOptimizing%2BIllumina%2Bnext-generation%2Bsequencing%2Blibrary%2Bpreparation%2Bfor%2Bextremely%2BAT-biased%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-1%26rft_id%253Dinfo%253Apmid%252F22214261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22214261&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.20" data-doi="10.1038/nature08390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozsolak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reifenberger</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sass</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Direct RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">814</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOzsolak%26rft.auinit1%253DF.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D814%26rft.epage%253D818%26rft.atitle%253DDirect%2BRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08390%26rft_id%253Dinfo%253Apmid%252F19776739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19776739&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.21" data-doi="10.1016/j.cell.2010.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozsolak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapranov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foissac</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishilevich</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comprehensive polyadenylation site maps in yeast and human reveal pervasive alternative polyadenylation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1018</span>–<span class="cit-lpage">1029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOzsolak%26rft.auinit1%253DF.%26rft.volume%253D143%26rft.issue%253D6%26rft.spage%253D1018%26rft.epage%253D1029%26rft.atitle%253DComprehensive%2Bpolyadenylation%2Bsite%2Bmaps%2Bin%2Byeast%2Band%2Bhuman%2Breveal%2Bpervasive%2Balternative%2Bpolyadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.11.020%26rft_id%253Dinfo%253Apmid%252F21145465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21145465&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285182600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.22" data-doi="10.1007/BF00039376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothnie</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Plant mRNA 3′-end formation.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DRothnie%26rft.auinit1%253DH.%2BM.%26rft.volume%253D32%26rft.issue%253D1-2%26rft.spage%253D43%26rft.epage%253D61%26rft.atitle%253DPlant%2BmRNA%2B3%2527-end%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00039376%26rft_id%253Dinfo%253Apmid%252F8980473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00039376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8980473&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VY81900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.23" data-doi="10.1016/0022-2836(75)90213-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">A rapid method for determining sequences in DNA by primed synthesis with DNA polymerase.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSanger%26rft.auinit1%253DF.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D441%26rft.epage%253D448%26rft.atitle%253DA%2Brapid%2Bmethod%2Bfor%2Bdetermining%2Bsequences%2Bin%2BDNA%2Bby%2Bprimed%2Bsynthesis%2Bwith%2BDNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252875%252990213-2%26rft_id%253Dinfo%253Apmid%252F1100841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(75)90213-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1100841&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF21300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.24" data-doi="10.1073/pnas.74.12.5463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicklen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">DNA sequencing with chain-terminating inhibitors.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">5463</span>–<span class="cit-lpage">5467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSanger%26rft.auinit1%253DF.%26rft.volume%253D74%26rft.issue%253D12%26rft.spage%253D5463%26rft.epage%253D5467%26rft.atitle%253DDNA%2Bsequencing%2Bwith%2Bchain-terminating%2Binhibitors%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.12.5463%26rft_id%253Dinfo%253Apmid%252F271968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzQvMTIvNTQ2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.25" data-doi="10.1038/nmeth1156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation sequencing transforms today’s biology.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchuster%26rft.auinit1%253DS.%2BC.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D18%26rft.atitle%253DNext-generation%2Bsequencing%2Btransforms%2Btoday%2527s%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1156%26rft_id%253Dinfo%253Apmid%252F18165802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18165802&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252120600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.26" data-doi="10.1534/genetics.108.088971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">Unique features of nuclear mRNA poly(A) signals and alternative polyadenylation in <em>Chlamydomonas reinhardtii</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShen%26rft.auinit1%253DY.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D176%26rft.atitle%253DUnique%2BFeatures%2Bof%2BNuclear%2BmRNA%2BPoly%2528A%2529%2BSignals%2Band%2BAlternative%2BPolyadenylation%2Bin%2BChlamydomonas%2Breinhardtii%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088971%26rft_id%253Dinfo%253Apmid%252F18493049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvMTY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.27" data-doi="10.1093/nar/gkn158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome level analysis of rice mRNA 3′-end processing signals and alternative polyadenylation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">3150</span>–<span class="cit-lpage">3161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn158%26rft_id%253Dinfo%253Apmid%252F18411206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM2LzkvMzE1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.28" data-doi="10.1101/gr.114744.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venu</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nobuta</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Notibala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptome dynamics through alternative polyadenylation in developmental and environmental responses in plants revealed by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1478</span>–<span class="cit-lpage">1486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114744.110%26rft_id%253Dinfo%253Apmid%252F21813626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMTQ3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.29" data-doi="10.1038/nsmb.2345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherstnev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duc</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zacharaki</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornyik</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Direct sequencing of Arabidopsis thaliana RNA reveals patterns of cleavage and polyadenylation.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSherstnev%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D845%26rft.epage%253D852%26rft.atitle%253DDirect%2Bsequencing%2Bof%2BArabidopsis%2Bthaliana%2BRNA%2Breveals%2Bpatterns%2Bof%2Bcleavage%2Band%2Bpolyadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2345%26rft_id%253Dinfo%253Apmid%252F22820990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820990&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.30" data-doi="10.1158/0008-5472.CAN-09-2236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alley</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamdar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schott</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Global changes in processing of mRNA 3′ untranslated regions characterize clinically distinct cancer subtypes.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">9422</span>–<span class="cit-lpage">9430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-09-2236%26rft_id%253Dinfo%253Apmid%252F19934316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2OS8yNC85NDIyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.31" data-doi="10.1093/bioinformatics/btn013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekhans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baertsch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haussler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn013%26rft_id%253Dinfo%253Apmid%252F18218656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNC81LzYzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.32" data-doi="10.1007/s00357-005-0012-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szekely</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzo</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Hierarchical clustering via joint between-within distances: extending Ward’s minimum variance method.</span> <abbr class="cit-jnl-abbrev">J. Classification</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClassification%26rft.volume%253D22%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00357-005-0012-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00357-005-0012-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-4.5.871.33" data-doi="10.1096/fj.07-8809com"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Intronic polyadenylation signal sequences and alternate splicing generate human soluble Flt1 variants and regulate the abundance of soluble Flt1 in the placenta.</span> <span class="cit-source">FASEB J.</span> <span class="cit-vol">21</span>: <span class="cit-fpage">3885</span>–<span class="cit-lpage">3895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.07-8809com%26rft_id%253Dinfo%253Apmid%252F17615362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xNC8zODg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.34" data-doi="10.1093/nar/gkq198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raikwar</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Alternate processing of Flt1 transcripts is directed by conserved cis-elements within an intronic region of FLT1 that reciprocally regulates splicing and polyadenylation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">5130</span>–<span class="cit-lpage">5140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq198%26rft_id%253Dinfo%253Apmid%252F20385595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNS81MTMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.35" data-doi="10.1105/tpc.112.096107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaffney</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide control of polyadenylation site choice by CPSF30 in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">4376</span>–<span class="cit-lpage">4388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.096107%26rft_id%253Dinfo%253Apmid%252F23136375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMS80Mzc2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.36" data-doi="10.1093/nar/gki158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A large-scale analysis of mRNA polyadenylation of human and mouse genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTian%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D201%26rft.epage%253D212%26rft.atitle%253DA%2Blarge-scale%2Banalysis%2Bof%2BmRNA%2Bpolyadenylation%2Bof%2Bhuman%2Band%2Bmouse%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki158%26rft_id%253Dinfo%253Apmid%252F15647503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMzLzEvMjAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.37" data-doi="10.1101/gr.5532707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Widespread mRNA polyadenylation events in introns indicate dynamic interplay between polyadenylation and splicing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5532707%26rft_id%253Dinfo%253Apmid%252F17210931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMTU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.38" data-doi="10.1006/jmbi.1998.1947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Helden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">André</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collado-Vides</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Extracting regulatory sites from the upstream region of yeast genes by computational analysis of oligonucleotide frequencies.</span> <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253Dvan%2BHelden%26rft.auinit1%253DJ.%26rft.volume%253D281%26rft.issue%253D5%26rft.spage%253D827%26rft.epage%253D842%26rft.atitle%253DExtracting%2Bregulatory%2Bsites%2Bfrom%2Bthe%2Bupstream%2Bregion%2Bof%2Byeast%2Bgenes%2Bby%2Bcomputational%2Banalysis%2Bof%2Boligonucleotide%2Bfrequencies.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.1947%26rft_id%253Dinfo%253Apmid%252F9719638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.1947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9719638&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075771200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.39" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.40" data-doi="10.1186/1471-2148-7-65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wodniok</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glockner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gain and loss of polyadenylation signals during evolution of green algae.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DWodniok%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.spage%253D65%26rft.epage%253D65%26rft.atitle%253DGain%2Band%2Bloss%2Bof%2Bpolyadenylation%2Bsignals%2Bduring%2Bevolution%2Bof%2Bgreen%2Balgae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-65%26rft_id%253Dinfo%253Apmid%252F17442103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17442103&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.41" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.42" data-doi="10.1073/pnas.1019732108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Downie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide landscape of polyadenylation in Arabidopsis provides evidence for extensive alternative polyadenylation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">12533</span>–<span class="cit-lpage">12538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019732108%26rft_id%253Dinfo%253Apmid%252F21746925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzMwLzEyNTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.43" data-doi="10.1534/genetics.111.132068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Translation initiation: a regulatory role for poly(A) tracts in front of the AUG codon in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132068%26rft_id%253Dinfo%253Apmid%252F21840854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNDY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81Lzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Alternative polyadenylation and gene expression regulation in plants. Wiley interdisciplinary reviews.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.volume%253D2%26rft.spage%253D445%26rft_id%253Dinfo%253Apmid%252F21957029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21957029&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Formation of mRNA 3′ ends in eukaryotes: mechanism, regulation, and interrelationships with other steps in mRNA synthesis. Microbiology and molecular biology reviews</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DZhao%26rft.auinit1%253DJ.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D405%26rft.epage%253D445%26rft.atitle%253DFormation%2Bof%2BmRNA%2B3%2527%2BEnds%2Bin%2BEukaryotes%253A%2BMechanism%252C%2BRegulation%252C%2Band%2BInterrelationships%2Bwith%2BOther%2BSteps%2Bin%2BmRNA%2BSynthesis%26rft_id%253Dinfo%253Apmid%252F10357856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2My8yLzQwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.871.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">GOEAST: a web-based software toolkit for Gene Ontology enrichment analysis.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">W358–363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D36%26rft.spage%253DW358363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/5/871.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/861" title="Genomics of CpG Methylation in Developing and Developed Zebrafish" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/885" title="Impact of Genetic Background on Neonatal Lethality of Gga2 Gene-Trap Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/5"><span>Volume 4 Issue 5, May 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/5.cover-source.jpg?itok=AAKWoOtu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/871.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Iv13l-2yx_jnDzDYl0PX2D3Wvr7tA7JwF6a31eYY-Zk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/5/871" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/5/871" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/5/871" class="active" data-icon-position="" data-hide-link-title="0">Bioinformatics Analysis of Alternative Polyadenylation in Green Alga Chlamydomonas reinhardtii Using Transcriptome Sequences from Three Different Sequencing Platforms</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459865" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aUmJ1blr9-OW2Yt8mobb4h8xckKaCmt0fuJfclQHe88" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/871.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/5/871" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jrkA-5oNnWZtb6Wzi7_8rVCYiYgDmnpxNdugQCRPiZ8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/5/871" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/871.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459865" id="node459865--2" data-pisa="ggg;4/5/871" data-pisa-master="ggg;g3.114.010249" data-apath="/ggg/4/5/871.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bioinformatics Analysis of Alternative Polyadenylation in Green Alga <em>Chlamydomonas reinhardtii</em> Using Transcriptome Sequences from Three Different Sequencing Platforms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhixin Zhao</span>, <span class="highwire-citation-author" data-delta="1">Xiaohui Wu</span>, <span class="highwire-citation-author" data-delta="2">Praveen Kumar Raj Kumar</span>, <span class="highwire-citation-author" data-delta="3">Min Dong</span>, <span class="highwire-citation-author" data-delta="4">Guoli Ji</span>, <span class="highwire-citation-author" data-delta="5">Qingshun Quinn Li</span> and <span class="highwire-citation-author" data-delta="6">Chun Liang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">871-883;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010249 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459865--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhixin Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhixin%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhixin%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaohui Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaohui%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaohui%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Praveen Kumar Raj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Praveen%2BKumar%2BRaj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APraveen%2BKumar%2BRaj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Min Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Guoli Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guoli%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuoli%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qingshun Quinn Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Key Laboratory of the Ministry of Education for Coastal and Wetland Ecosystems, and College of the Environment and Ecology, Xiamen University, Xiamen, Fujian, China 361102</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Research Institute, Fujian Academy of Agricultural Sciences, Fuzhou, Fujian, China 350003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingshun%2BQuinn%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20QQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQingshun%2BQuinn%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangc@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liangc@miamioh.edu</a>
<a href="mailto:liq@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liq@miamioh.edu</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chun Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChun%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangc@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liangc@miamioh.edu</a>
<a href="mailto:liq@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liq@miamioh.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/5/871" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/5/871&amp;title=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/5/871&amp;title=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/5/871&amp;title=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/5/871&amp;text=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/5/871&amp;title=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/5/871&amp;t=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/5/871&amp;title=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/5/871&amp;title=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459865/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459865" id="node459865--3" data-pisa="ggg;4/5/871" data-pisa-master="ggg;g3.114.010249" data-apath="/ggg/4/5/871.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bioinformatics Analysis of Alternative Polyadenylation in Green Alga <em>Chlamydomonas reinhardtii</em> Using Transcriptome Sequences from Three Different Sequencing Platforms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhixin Zhao</span>, <span class="highwire-citation-author" data-delta="1">Xiaohui Wu</span>, <span class="highwire-citation-author" data-delta="2">Praveen Kumar Raj Kumar</span>, <span class="highwire-citation-author" data-delta="3">Min Dong</span>, <span class="highwire-citation-author" data-delta="4">Guoli Ji</span>, <span class="highwire-citation-author" data-delta="5">Qingshun Quinn Li</span> and <span class="highwire-citation-author" data-delta="6">Chun Liang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">871-883;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010249 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459865--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhixin Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhixin%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhixin%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaohui Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaohui%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaohui%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Praveen Kumar Raj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Praveen%2BKumar%2BRaj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APraveen%2BKumar%2BRaj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Min Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Guoli Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Automation, Xiamen University, Xiamen, 361005, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guoli%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuoli%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qingshun Quinn Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Key Laboratory of the Ministry of Education for Coastal and Wetland Ecosystems, and College of the Environment and Ecology, Xiamen University, Xiamen, Fujian, China 361102</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Research Institute, Fujian Academy of Agricultural Sciences, Fuzhou, Fujian, China 350003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingshun%2BQuinn%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20QQ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQingshun%2BQuinn%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangc@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liangc@miamioh.edu</a>
<a href="mailto:liq@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liq@miamioh.edu</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chun Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Miami University, Oxford, Ohio 45056</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChun%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liangc@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liangc@miamioh.edu</a>
<a href="mailto:liq@miamioh.edu" class="" data-icon-position="" data-hide-link-title="0">liq@miamioh.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=871&amp;rft.epage=883&amp;rft.atitle=Bioinformatics%20Analysis%20of%20Alternative%20Polyadenylation%20in%20Green%20Alga%20Chlamydomonas%20reinhardtii%20Using%20Transcriptome%20Sequences%20from%20Three%20Different%20Sequencing%20Platforms&amp;rft.volume=4&amp;rft.issue=5&amp;rft.date=2014-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhao%2C+Zhixin&amp;rft.au=Wu%2C+Xiaohui&amp;rft.au=Kumar%2C+Praveen+Kumar+Raj&amp;rft.au=Dong%2C+Min&amp;rft.au=Ji%2C+Guoli&amp;rft.au=Li%2C+Qingshun+Quinn&amp;rft.au=Liang%2C+Chun"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459865/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459865/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459865/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459865/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459865/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459865/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459865/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459865/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459865/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459865/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459865/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459865/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/5/871" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/5/871.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/5/871.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/5/871.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/5/871.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
