<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/5/901" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/5/901.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/5/901.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/5/901.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/5/901/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/5/901.atom" />
<meta name="HW.pisa" content="ggg;4/5/901" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" />
<meta name="DC.Identifier" content="10.1534/g3.113.007633" />
<meta name="DC.Date" content="2014-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Shringarpure and Xing. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Population stratification is an important task in genetic analyses. It provides information about the ancestry of individuals and can be an important confounder in genome-wide association studies. Public genotyping projects have made a large number of datasets available for study. However, practical constraints dictate that of a geographical/ethnic population, only a small number of individuals are genotyped. The resulting data are a sample from the entire population. If the distribution of sample sizes is not representative of the populations being sampled, the accuracy of population stratification analyses of the data could be affected. We attempt to understand the effect of biased sampling on the accuracy of population structure analysis and individual ancestry recovery. We examined two commonly used methods for analyses of such datasets, ADMIXTURE and EIGENSOFT, and found that the accuracy of recovery of population structure is affected to a large extent by the sample used for analysis and how representative it is of the underlying populations. Using simulated data and real genotype data from cattle, we show that sample selection bias can affect the results of population structure analyses. We develop a mathematical framework for sample selection bias in models for population structure and also proposed a correction for sample selection bias using auxiliary information about the sample. We demonstrate that such a correction is effective in practice using simulated and real data." />
<meta name="DC.Contributor" content="Suyash Shringarpure" />
<meta name="DC.Contributor" content="Eric P. Xing" />
<meta name="article:published_time" content="2014-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Population stratification is an important task in genetic analyses. It provides information about the ancestry of individuals and can be an important confounder in genome-wide association studies. Public genotyping projects have made a large number of datasets available for study. However, practical constraints dictate that of a geographical/ethnic population, only a small number of individuals are genotyped. The resulting data are a sample from the entire population. If the distribution of sample sizes is not representative of the populations being sampled, the accuracy of population stratification analyses of the data could be affected. We attempt to understand the effect of biased sampling on the accuracy of population structure analysis and individual ancestry recovery. We examined two commonly used methods for analyses of such datasets, ADMIXTURE and EIGENSOFT, and found that the accuracy of recovery of population structure is affected to a large extent by the sample used for analysis and how representative it is of the underlying populations. Using simulated data and real genotype data from cattle, we show that sample selection bias can affect the results of population structure analyses. We develop a mathematical framework for sample selection bias in models for population structure and also proposed a correction for sample selection bias using auxiliary information about the sample. We demonstrate that such a correction is effective in practice using simulated and real data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/05/01" />
<meta name="citation_mjid" content="ggg;4/5/901" />
<meta name="citation_id" content="4/5/901" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/5/901" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/5/901.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/5/901.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/5/901.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007633" />
<meta name="citation_pmid" content="24637351" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="901" />
<meta name="citation_lastpage" content="911" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Suyash Shringarpure" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, California 94305" />
<meta name="citation_author" content="Eric P. Xing" />
<meta name="citation_author_institution" content="School of Computer Science, Carnegie Mellon University, Pittsburgh, Pennsylvania 15213" />
<meta name="citation_author_email" content="epxing@cs.cmu.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Baran;citation_author=B. Pasaniuc;citation_author=S. Sankararaman;citation_author=D. G. Torgerson;citation_author=C. Gignoux;citation_title=Fast and accurate inference of local ancestry in Latino populations.;citation_volume=28;citation_year=2012;citation_pmid=22495753;citation_doi=10.1093/bioinformatics/bts144" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. L. Cavalli-Sforza;citation_title=The Human Genome Diversity Project: past, present and future.;citation_pages=333-340;citation_volume=6;citation_year=2005;citation_issue=4;citation_pmid=15803201;citation_doi=10.1038/nrg1596" />
<meta name="citation_reference" content="citation_journal_title=Algorithmic Learning Theory;citation_author=C. Cortes;citation_author=M. Mohri;citation_author=M. Riley;citation_author=A. Rostamizadeh;citation_title=Sample selection bias correction theory.;citation_pages=16;citation_volume=5254;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. E. Engelhardt;citation_author=M. Stephens;citation_title=Analysis of population structure: a unifying framework and novel methods based on sparse factor analysis.;citation_pages=12;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. A. Gibbs;citation_title=The International HapMap Project.;citation_pages=789-796;citation_volume=426;citation_year=2003;citation_issue=6968;citation_pmid=14685227;citation_doi=10.1038/nature02168" />
<meta name="citation_reference" content="citation_journal_title=Econometrica;citation_author=J. J. Heckman;citation_title=Sample selection bias as a specification error.;citation_pages=153-161;citation_volume=47;citation_year=1979;citation_doi=10.2307/1912352" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. J. Hoggart;citation_author=E. J. Parra;citation_author=M. D. Shriver;citation_author=C. Bonilla;citation_author=R. A. Kittles;citation_title=Control of confounding of genetic associations in stratified populations.;citation_pages=1492-1504;citation_volume=72;citation_year=2003;citation_issue=6;citation_pmid=12817591;citation_doi=10.1086/375613" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Huelsenbeck;citation_author=P. Andolfatto;citation_title=Inference of population structure under a Dirichlet process prior.;citation_volume=175;citation_year=2007;citation_pmid=17237522;citation_doi=10.1534/genetics.106.061317" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. Kaeuffer;citation_author=D. Réale;citation_author=D. W. Coltman;citation_author=D. Pontier;citation_title=Detecting population structure using STRUCTURE software: effect of background linkage disequilibrium.;citation_pages=374-380;citation_volume=99;citation_year=2007;citation_issue=4;citation_pmid=17622269;citation_doi=10.1038/sj.hdy.6801010" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. J. McTavish;citation_author=J. E. Decker;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=D. M. Hillis;citation_title=New World cattle show ancestry from multiple independent domestication events.;citation_volume=110;citation_year=2013;citation_pmid=23530234;citation_doi=10.1073/pnas.1303367110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. McVean;citation_title=A genealogical interpretation of principal components analysis.;citation_pages=e1000686-e1000686;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19834557;citation_doi=10.1371/journal.pgen.1000686" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. R. Nelson;citation_author=K. Bryc;citation_author=K. S. King;citation_author=A. Indap;citation_author=A. R. Boyko;citation_title=The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research.;citation_pages=347-358;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760391;citation_doi=10.1016/j.ajhg.2008.08.005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Novembre;citation_author=M. Stephens;citation_title=Interpreting principal component analyses of spatial population genetic variation.;citation_pages=646-649;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18425127;citation_doi=10.1038/ng.139" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Novembre;citation_author=T. Johnson;citation_author=K. Bryc;citation_author=Z. Kutalik;citation_author=A. R. Boyko;citation_title=Genes mirror geography within Europe.;citation_pages=98-101;citation_volume=456;citation_year=2008;citation_issue=7218;citation_pmid=18758442;citation_doi=10.1038/nature07331" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Appl.;citation_author=S. J. Phillips;citation_author=M. Dudík;citation_author=J. Elith;citation_author=C. H. Graham;citation_author=A. Lehmann;citation_title=Sample selection bias and presence-only distribution models: implications for background and pseudo-absence data.;citation_pages=181-197;citation_volume=19;citation_year=2009;citation_doi=10.1890/07-2153.1" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. L. Price;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=K. C. Barnes;citation_author=N. Rafaels;citation_title=Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.;citation_pages=e1000519-e1000519;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543370;citation_doi=10.1371/journal.pgen.1000519" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. A. Rosenberg;citation_author=J. K. Pritchard;citation_author=J. L. Weber;citation_author=H. M. Cann;citation_author=K. K. Kidd;citation_title=Genetic Structure of Human Populations;citation_pages=2381-2385;citation_volume=298;citation_year=2002;citation_issue=5602;citation_pmid=12493913;citation_doi=10.1126/science.1078311" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Sankararaman;citation_author=G. Kimmel;citation_author=E. Halperin;citation_author=M. I. Jordan;citation_title=On the inference of ancestries in admixed populations.;citation_volume=18;citation_year=2008;citation_pmid=18353809;citation_doi=10.1101/gr.072751.107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Shringarpure;citation_author=E. P. Xing;citation_title=mStruct: Inference of Population Structure in Light of Both Genetic Admixing and Allele Mutations.;citation_volume=182;citation_year=2009;citation_pmid=19363128;citation_doi=10.1534/genetics.108.100222" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=H. Tang;citation_author=J. Peng;citation_author=P. Wang;citation_author=N. J. Risch;citation_title=Estimation of individual admixture: analytical and study design considerations.;citation_pages=289-301;citation_volume=28;citation_year=2005;citation_issue=4;citation_pmid=15712363;citation_doi=10.1002/gepi.20064" />
<meta name="citation_reference" content="citation_journal_title=J. Hum. Resour.;citation_author=F. Vella;citation_title=Estimating models with sample selection bias: a survey.;citation_pages=127-169;citation_volume=33;citation_year=1998;citation_doi=10.2307/146317" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. D. Wall;citation_author=R. Jiang;citation_author=C. Gignoux;citation_author=G. K. Chen;citation_author=C. Eng;citation_title=Genetic variation in Native Americans, inferred from Latino SNP and resequencing data.;citation_volume=28;citation_year=2011;citation_pmid=21368315;citation_doi=10.1093/molbev/msr049" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
<meta name="twitter:description" content="Population stratification is an important task in genetic analyses. It provides information about the ancestry of individuals and can be an important confounder in genome-wide association studies. Public genotyping projects have made a large number of datasets available for study. However, practical constraints dictate that of a geographical/ethnic population, only a small number of individuals are genotyped. The resulting data are a sample from the entire population. If the distribution of sample sizes is not representative of the populations being sampled, the accuracy of population stratification analyses of the data could be affected. We attempt to understand the effect of biased sampling on the accuracy of population structure analysis and individual ancestry recovery. We examined two commonly used methods for analyses of such datasets, ADMIXTURE and EIGENSOFT, and found that the accuracy of recovery of population structure is affected to a large extent by the sample used for analysis and how representative it is of the underlying populations. Using simulated data and real genotype data from cattle, we show that sample selection bias can affect the results of population structure analyses. We develop a mathematical framework for sample selection bias in models for population structure and also proposed a correction for sample selection bias using auxiliary information about the sample. We demonstrate that such a correction is effective in practice using simulated and real data." />
<meta name="og-title" property="og:title" content="Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/5/901" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Population stratification is an important task in genetic analyses. It provides information about the ancestry of individuals and can be an important confounder in genome-wide association studies. Public genotyping projects have made a large number of datasets available for study. However, practical constraints dictate that of a geographical/ethnic population, only a small number of individuals are genotyped. The resulting data are a sample from the entire population. If the distribution of sample sizes is not representative of the populations being sampled, the accuracy of population stratification analyses of the data could be affected. We attempt to understand the effect of biased sampling on the accuracy of population structure analysis and individual ancestry recovery. We examined two commonly used methods for analyses of such datasets, ADMIXTURE and EIGENSOFT, and found that the accuracy of recovery of population structure is affected to a large extent by the sample used for analysis and how representative it is of the underlying populations. Using simulated data and real genotype data from cattle, we show that sample selection bias can affect the results of population structure analyses. We develop a mathematical framework for sample selection bias in models for population structure and also proposed a correction for sample selection bias using auxiliary information about the sample. We demonstrate that such a correction is effective in practice using simulated and real data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5.cover-source.jpg" />
    <title>Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"u_gOi4-eKog_XWLd0Nai7alKEQMtdDYcbAUpXSxuak8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459867","apath":"\/ggg\/4\/5\/901.atom","pisa":"ggg;4\/5\/901","ac":{"\/ggg\/4\/5\/901.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/5\/901.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/5\/901"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/5\/901"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/5\/901":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hRS10qcDWU4AzD5jvN0TlhcHkwofZT8Mu8H86aTHArUGfX\\\\\\\/2Rf0eCufuvJ+niBCxkYdOvDwpeO8\\\\\\\/5iVCABc88LB6KoFeA204y7oYIX7xDU2cnmWLWzcJrG16rYJuu9M03nQxyf5qCKBcQNaxbJAIJ6bWfzvE51o0V7EzG09CJbX7JXZ3W5FhIMfTGsMj2eBtyOLpfAqsgDpe0NJWWDfcW+I+pnO3S5AE4NiztNlRBSQeom+l56LWOk3rTUesU6DgiqG8sTSD\\\\\\\/JgfbzbxpQe0+LS73FCK8+pYYbeYNfA\\\\\\\/SgFCdKg1tSvGBL1mALk8ZElknpWwxO11CAeUk2JKFVXyooI0fvJTgnRd8howE4lIcjt8hXGX0ucmeDfA\\\\\\\/bKhFqD2H\\\\\\\/vLzxjYiiK\\\\\\\/CsGP\\\\\\\/AaPR7rsZGdfmswnSdpkjmXHPSGq1riidO2Uo772YaanNfL4FL20yaSnbTK25HoU+8uhwUYxoPuG2y4exU2O3kAdOv2RjLhF8tgNnTo00jwFyeM\\\\\\\/iVecPyEC3\\\\\\\/dn706eVNhlrFBMIbuMvlwXorb6IMcwxLQOqyjKtYFWqqmj9TKlZVdqYFv+7+JyJF8qd7M4WxRMauzgdcL\\\\\\\/xx3oCDhxaIDkQAOq+oPkLB7GFbEbZLh7r\\\\\\\/Nev\\\\\\\/ONEDeHXis+15\\\\\\\/LcdFJT73O1zxBVbEq8RU2jcKruAN7cN9wTedK6m0MCHZiJ6X9E3jqKOq3xaEbIjB5SHeDSqEZ7tlR+iPp55+d9EzoIJSfCHkAASQH3axi3i\\\\\\\/Q\\\\\\\/6M1dunnJjNAV6qcDIEBw0tY\\\\\\\/CSP3AQw2lqSLbr5bBdyAY8o96ye+d7GVTfm+dBN1hEujNbMmwmy+e6LY6MQohpcdx\\\\\\\/l3hNqb4GXZlOTBRQV2aF3eXHRlFXC3NnsNB1sAE8JJQI73UBOCERKPUkcKdyKvcw65NrIw39pZlcc2V6Gvp6M7sEW8LyJoG22OefxwONf+IDl7wZZ+LDZzETH882BRBd6Ge4l17vvc4OOYF4WXBKqw5Oz1CGu5E34xiK9CIVrIIV9oP0m5krte+o8AAAWXeU1dXDwrQPzcsAsTfL6IPmQBizHVKAjJoL\\\\\\\/6uBvvHznXUEfhbBTOnx\\\\\\\/7HF02U2K\\\\\\\/JAi5AQoj3v2lzvhc7DXUYi7pts3QNnDXoMAhHD4dJyR8fqlg6IrW97Bo+vPRvwS+Dy8Lqr\\\\\\\/D1Dd6yJQ5F5HxXCIbRJhRghx37XU8eML2x3ZLEcDH13+dw66BCRNpkTW+b\\\\\\\/NANpoCfGIFcIMVj0C69S\\\\\\\/ZAxzBwgMGIf66EiCkPRV\\\u0022,\\\u0022iv\\\u0022:\\\u0022AswpznmRW0ZUXGa5wpeLnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b19011894a191047494b06bac9cb782f5c15a3b8792d606779bd6fa246b3d4ce\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P+ojsblAhkNlCVzcjfb28w1VHvpQDU9ncedDqxYgjIWwdhj5YvE2HKkrnz3MHFXAt8yeAJ7qxq3ZLIgefNaNiolXUtBt1HDYVY29uBZZrbc127qJlY3GNkstB9FyWkyfIsRWH3shbwSdc9B1DyyfKUQEfjQTWsyS0YFOoOY4gDbYsy4gWg0QBVZ32SmzAmf0Hf8DLNQQdd4uWzRTa9fDmNfojoGYfjWJsO4xSlbL3XxyKQD8tPUaex2NY4n7\\\\\\\/+LGGVcOzikhZisUXacgUDbiLgqU6p7e1HZBijakTs8TmA1XAB2qDVjPZSwsYxqfLMS\\\\\\\/txCzY1\\\\\\\/x6L3wlZI\\\\\\\/EgrN69oiChhG2RPqDAIK2um+US8fkdTm2Wnex2juB\\\\\\\/p62lX0mOIkTnoH1QSl33R6VH30NzJq5EQSzwwP5sPmJC4xKxrcqmkAbFwpu+P3n+eQ8Emi7++qOKHTu2nEOSRmUPqqA7ZKRn\\\\\\\/Loy5Nk4gAqlrlFe\\\\\\\/ZgeDoQtb1N75ZyycERVV+aYZavF\\\\\\\/Dc9DTXIvZHBK5cwEosDKf84RRgSTkkkX22AdxC3jRCOwDqySjTjK2lNbu3jfZvSZoyGJ8D5EYrnM07uYUeAD7Jfn39e4IbUS3HyR7RFH2b8GfKdmIMt+KePCylD5UgCtvBrxR+OVI0wPrzFs7dRh+NkpSWcKjddRJCWkPTvKykOghXUg6qAPT+K\\\\\\\/o987XpUBv+nKDsaVfIWbUJdAcsx1DgkpKV5pAwzPLB9VLitw1MrmDxfdDLh26hVhyevKzUIImuJEw70m9+VOVJsvL0YB0qlpccjDC77OoL\\\\\\\/EYvtHin8kj9oY2h17BvN4tmQ51OuKFaO8pU3pE+uWHqPuXNYKrWOitqLY7Hff0cGDx7z1cei+3y7ZNLKNJJOPU7y8Q2QgN8RmK9fuu7QT0Y+Qvp71gjRNU+IQ7EFNCjlP+AbP\\\\\\\/D9liXmDe2VWFC30FLKwj4O8CJ7iDLSNpJaqamhJSld04wzNDj\\\\\\\/giVboj3HQF0giewUGJLaHuoO50cSu9IHMJHnD3UrvIwOIT2BnucW09FotVQUDTU7+ITTr9LG+eeuzTvQhTj334W3BdjuoJiCWytNf2FaZkEQd1VcHpdUfteCmJcb2hKwidFmLHxyP23om2jh3NiiceEUqV\\\\\\\/LhWG59QxACpJG50663JlWhd+N2CbHptesV2zFNizYhJPfE1n1D99DRBXy1gtx4UHsPnmr3cWMtdb2YJMCCycpFVpu5rowQNVPUM2OcqM1bb96WldyvIVmGyAXphHQByO8aV9koJkV2mJgBYzGzxS5snFYlvnH7tlvHUXj2LGmmDY0ufiLJfyFX\\\\\\\/89WK7CDD+q63zYXm1FYcOmvGu+WP9q\\\\\\\/xVKWLZ4ntidQSD3ahdZjQYA3EYJuLIBUpS795rmqI4JzCwVuDBD0pJAGdpLRcUAQ9g8IrCXjt7NzscTYLKSNnCExuOIu+AA\\\\\\\/NcT5uu2Pj4MYGT6V+mw8moe+WQ+yNnBKI3S\\\\\\\/54YX3k95\\\\\\\/zf1AY\\\\\\\/af11XNsBOlyH16g5k2qS2tMmU7ZZCaIOQ9NOwepFMYsSTaVlGlXfQJQ2waOWKVnZXlmvDemN6+ThhLbK2tFMQT4Z2OsrQJKoVaTTcLqCbOUQ7A5ithG0EodDFJpI9mOhGf3NPSCUNpiWiUndIbAwRNdBzd8hwA0d6hLdjDIKxbTGy7c8zlZTROQZzowtvC6\\\\\\\/zx\\\\\\\/CqhAN0VvgF\\\\\\\/tX7qB8qFQQu7yMiXNXmdFBRtsyEn7tsm5pGYbiAQ5MKJ4lq3uUQq0s4AjbT45x4sXuqhD0LFIjA5GwFfBm4SdPJlP6cyb5mLsKUuOuQW7yXsBA93p9uy+E4EUf1iqCuI+oKdL8QHl\\\\\\\/Uns25RhTHZaSPJdPdMOpl\\\\\\\/wP12mouh7+9lZxeAFb2gu4nsuZDifcoaf8NDB4HG0iOfm8RsmvDPy4E1W9MHz+1yEq5PLXhSKEGpsqjxocemsTTrJ9dwqcq+eeohvtSAukwiJr3AS\\\\\\\/8ccqDnivnir2bzLikztvN8SyWriavLUWmXzI7+JO3DVcWlj6XcnvTiDBrCgs3p2mmqdUfR+XEuLoyFyYJ25PXQogK3g+BJrGUVg7bG+Yr1OP43\\\\\\\/O0\\\\\\\/ZiDES0M+Z3dp1DiXm7O7XO2G8MHRHjYGHsnPHT8A5bV7KzkJ5qDiCc7Gxu8rOHua66b\\\\\\\/MKPcHN0IUzGvnhRwjY2ToOM0WURhn7HnIF2jXgOadxOaUEPKEzSPuhcm2pupmX7bXrrGdliY4R3Yiwo+sduZt0+jVHqO4gVZ7IL7UORM+pXy7FoPBAPawpIu6ipWeQkhJLhwpIAxGXpQKsEZk\\\\\\\/LGEvtfoECovkWzX2lUZtO4mVPp70hBXon75PTHKf\\\\\\\/bJOUMNLy284yWjqTqq6RNTG2G9jHW9+M\\\\\\\/\\\\\\\/iphA+869mhuoNjwzmmaEZgV8CwSBL6at1XjkE6LIlGOjPnDR7woJK7LwQG5YcWviYIJhN2UWE+h\\\\\\\/pSUxJaGsbM6GRMCUjxkk31xEGtXxZ5fbHiqdV25OCrNTFGJRO+d5c1J5xBF64ef9BsekaHNxnOx1PgyCG2zjl5FLL2hS9uHzSe1cmBrvKa2FNV3GtNba0in1s0BCdRLeox0HVYaA5h3uGydgsqaz663pTcmy1rpwvk97GrO0S7BqIKNZawHM3S\\\\\\\/9W3a9GUYpYeONAVUN79IWywoGz6wbFJdKTcrb9NTMuxItsI88J9SfUjEbiXKVmUr7Z+uZydtLBARA6tYw7TWupIq37ItX3zb+owPrYsi6Gb\\\\\\\/VaDWp7g8IQisektfLRQuUKzyFqXRcNI1Ro8Y3Zwjy163N2LWZHhRofw24sjsrl\\\\\\\/t07D0\\\\\\\/HFhTi+cf0zp7YAZEsLouCsIfEur5lOkISiSe9G7r9CVsPQAfoo2ss9DkgWyiSGlNQ1PbvJnzXZFzVRgnFyhCFhzPCrf9SB8RrxaEuKc+ztoCMHZ9Y02KADX\\\\\\\/ieD28lRlZV5KAyttliYE4DFILTUI+z1c+kbxQIx3EYXyZxtdUSBAoY41f64F6VkubNLyP3M6QBb7k7mjZDdt862i9NraLg+VywVPhT+90TCCIWK3vU5pJfWsOwpTMvvz9km\\\\\\\/T5rBgwUruyOeIerMMTqARJcepc+pQ0cLUQs48M5onwvQ53gd3y29KRTdaIWbR99jfUeGgGb35hldoiQ9czqNFmkw0ktcQYR1RgQzmC1zNR\\\\\\\/CXbE53DnjWXdSNhTSQ0+OMrFI1wbq7Sc0YYYPNRNVyPpsGA39qo3yJ4aTheqGyMeZ+NOJhqmF\\\\\\\/7D1yiHhLohNLnpU\\\\\\\/rKiOhxJL\\\\\\\/kRvqFjpwFqi2epxWby5W8y5XS9DGIZN2vhMRUs5haGXtCPqPby3tMKc9zUdI+blxcfEiBs3LuyD42X0c3MeUqG619LvL7GeNus3\\\\\\\/\\\\\\\/zkjJKhVh94RC6k8uJSvIb5UGoD7Y3OmsmqX4XD15tL5NOQg5xvcr9L57ucKzIe8LeKaBDw3+YyKUWaOvT1VZ7Qa\\\\\\\/8ky+YDRsG\\\\\\\/yGuW6e2L2Mp2hWhxmgQNk1lvLRgi4\\\\\\\/2znIDArLuPx3Rd9dVDNVLaNP07d2pnMYgWK2HHfLFDdn5xVwr7WhHRFn6NpwWobV2lAzRa+z\\\\\\\/4IFZjh9bduwuSAANxF403iQN3e6nXCcJ6rD8vu6r9qgp58\\\\\\\/iS\\\\\\\/thm86\\\\\\\/respiER5M6POKtF66rXFWUXjCpONbyfufKwvzrR8Ybr5Hb\\\\\\\/MBkbOYKRdQeYXpdJhEvifuqdMW8\\\\\\\/zsKcJbIXfkz7\\\\\\\/BznKhbMglryoJFCObGTQS0vKJtj3RtSY1j6w733u1cJ9so7\\\\\\\/DOwF7D+ZfHWH1y5pGZmo8+fb13uWHbxI2mJT9jCTEQCVf2p15ExKxM5LJr4u4d3tkOiIBxumpURI2FGH9FONUnNmPehKZMRf2ygfYuAhzL8BAUuUojSSaXg64tJ+0QHXjLX\\\\\\\/gpTPuUEOAVF0PXRAWO3T2GyHeBiJu2wpqzSvpG+a7214\\\\\\\/w5xNNzB+5AudreFW51dnEDOVWWNEcxg61RoWifdVhW+DOrCJ0b7rl6JyXPdXuu77yyxIV7SHuG+YvAllPDwBua\\\\\\\/tSW15dYGT3Wg7zfi++dLNDJZ9y1sl3YyALYG0PURMYxYLgjSfOzT86GXZCq7aMKMVjC1XsrC+yNEsRzEfiLxU+d2inCYO5tQ6NBzs5RrPEbjR+rtbC+PV08KQ64C8idR26fotDtJgoehnJBLyr+exeM3qSBDZ6Dd6iV2AxzUIBWKBHoId+pY9f5O\\\\\\\/BtuZMjQZciZ704PEk4is7FY938JeolwXIo55msC9xFTH\\\\\\\/bIlkS1dZjUJLvXPxRyygw9Hy9CncX72a3NRm2Vqf28pkGcKfM3jFDKVftBMYhK2qnWubUJ5+d\\\\\\\/KQkulktpj9Tn0J9c5\\\\\\\/l2XN31jol0p3ZKi2etG9BU3wFpe8ZKIiuplQ5gnOKyifL0LdldTvL6MsNeBA2gW4SOdfsFW7134QWhEyn72hj\\\\\\\/PWROn9sxkWs34eAq45K+21u3gf4JGv2BZRsPDHSqF7Bfx7igelb7G\\\\\\\/yCn17aPACxRw7dcmntubJOnq15xydqGsVBsQJS2S9pj9hO2wZdAG7g8rymEDy9Y3V9CfAISlzTDHGYl9HoZMJeJHsCzefvOY\\\\\\\/SCG8wf9fFcOjhu2fwaF1DjUPDfg2gWYrvrVx2SRyu8BPSPVNHyifCo25OccbJJK7vMbEGcTuLeTWWZmmCRsMiR5c37Gj5tuN3qYU+ZhSK8Q49iseZZR0mhEYYnU59ma4fgOI38U8AaGmavanmtdZRSjoi4TE+H9TzCu9hJahy1y7uPYLH9teH\\\\\\\/sJjsCLSO0abQEUqz+s74xi+ZEcaLMOvXUFbI4p+iVXyRXIyAkuyrG1esZyXLLn4sUGH\\\\\\\/zQrS\\\\\\\/ZJRMJpaeXH12WKetTCOy2I0IC1V2kQjiludElxK9y7Pcz\\\\\\\/adqS0Uw1aoDIR4YfG19rq8id\\\\\\\/wxyeJ5P0Y7PqiSP+OOg6X6uidM7WfuhZcQQpWNI5rErbahjcU2mqHw7+TmVS5Fc8kQm5D1awrT3lAVwDORCP1LBcVWKRjRjHq4pb\\\\\\\/APFrLsSmE79jDtCKtX3Q+8gw7JSDMfjVk96v8SJoVgaTcs5BHbYVeF8NvnNweed4zH05IAti5HvkCooYbFuvCx9IcTFHWbhB0U47dQjPDs5p3E0AXyDZURnVuUhhDVQk5jcTRRJbOWrE8mhQ3u2PBCaABNG43TztWm8RNLgzEEskeR\\\\\\\/iQwx03uGUDXSqvRWKMEoIBjUh9sVCxcNeVLM8c6\\\\\\\/8xf4m89KYZvIbWoX9m729CBep8055mXkvZreVpbB6B1PHFpSEUfzXsk69x+il68gSHEt+MVPYzLuE++GOIG1RUosjaAWXniXAJAK6Ks0fqw+Mk4fUQ6ULGaXBXSYNHVZTRfwaetiURkECXh2mWuTxLb3uuZm+OOkJeVwCNr31dQy+XP1WJ0eUDm2Kb2JnDufrA0XQx0KIgMhYeFA4j9QFcywQNM4EZAjI1yiqPNEPfU7e3jXy6m+Kl55mWHG\\\\\\\/z6G77HVNB5OJwlYiUp0k2JyR9c1u3A65042hlyouu675GUujU1pSUhGMhr83OLio7RSO21ZBV4\\\\\\\/KWBq8PKugkZmzSOEVcdR0uVgjBcVEc2t8xHqyWW1bD3HRwhSUEV\\\\\\\/wahtG9X3zx3EVEDnDX+xL4lmMyoWOS1aktbVA+DhcA2F+EJXTJD7xaW0tvPv\\\\\\\/Y+29zp5ewSMbMq\\\\\\\/pCPd28EuDlw2APOsSgBHm41f1fsV0VipRJUtA8Gc82OZCImkPDclkvX6gwDRct+xdOso0BSyyO+kYUk8VF51JxLm73hyXz4jlxb8vqExPA1Kyoe5mdmBmrxNgfpz2iPBMC+gSl8f81YLJl4vFqyXyugBP8eYzj55lU\\\\\\\/CCiWetKKwM1t97ZHqtkYX7ArDzOY8B64yM4WcQHbkC1EuusqR4sLjeVGoKfc1g+ecFUsEamUGnwat3hd3Ra1xy8x0dZ4NOW\\\\\\\/kzJy3Y4BmZvhu5sHacSlaQQpRvRSej1YyULg69TutCcUklY1xSzAPoaSnbU1o6shBZO\\\\\\\/VW9JRNZQYTeI3MFhYpdMFAZVGUyECGyY4e\\\\\\\/0Z0qCrktCoJ819EH8ZADqXoUkeUTk8zE1tA2J+jcP4JrEUMHE40Uu2c0lTPLkCSBdOMVu+lg+KJfVcDrxMs+\\\\\\\/KxenESXfHVRR3MmgmTnlGutnOT2TAcc4w0VlseZkM7fEBlowqdOB7zxF904c\\\\\\\/J705anr3PF8ZBGyZEtul7pXFJCSqbgZEb5dpVBTdhn7\\\\\\\/y1Q0bSFNz6tVX7Wo3ROcVAXeCg95uhAJ6Gck6yzsli1ceADjRMpbLAXuf2WbNMM\\\\\\\/OLG1+qJfgfDGpFM7mPmfLV3v7MsqnY2C44sgH845WzsgVmpjDB++Gu27z+gfMMGYE5ssWI3AdNJX5f0y86gOqwxhRjDSPiullOwQk+71pQhnonbRnWrNOJi7VTvqySqIvj8b8nppOmJQgE\\\\\\\/mI8asbjIHWs43JMACyMETzI6mVuF5jixAeSWAUI3A7W6zLesEktMYBKhFl7\\\\\\\/rRIjkjuIrbfVTvo\\\\\\\/gM8ec9XnRCJslK3VxB1+ZDPhEmqyzhWO9zNyGHTUBTOWb4ZGQ18qq1rOK2VX\\\\\\\/q9V1zbHdmxwHmoYz9e9PBgPoFVEZp\\\\\\\/9ZPTgC4p07FbX+Arp6Xqqxgzkh3Q4ruUUORpdJ8dq1h7dYt8aZ3aHtFgYRgcs99JmYNllYsNxx+LW9ZP3CekzO6+4HPRFBahh9hsNAYmuKtEC+d4fX4+urvohSeS\\\\\\\/oyWl8hxf0v1vlNK6qkGYrYhSskRw+947lImbKM69qeSXu+Y9QRKME6BX610YAT\\\\\\\/Xoya1+vJKA5Z82ml\\\\\\\/EfjDKYsPrggpphIbXZsDQJXv0OuPGdzWe9fKywn716zMXQ1EfaoPAYbaDgtXgD+uivh\\\\\\\/UtFFc4EU75Dbgo6bVLoC+\\\\\\\/dDDDes0eclYqgZwMdWbOpyT+dkglBb7jQGW0mdO9g+FRx\\\\\\\/jHcLOZFk+30G\\\\\\\/+s90pRz8WaNmSDK4dlZufrSmChoO\\\\\\\/jUwR+QmRHwe+3tcQ6oKsINE9tilhBOWieOcoT8n5ESFJ++b3gl0OrsLV24tcnbZUijvim59i+reu9IWAsRNTTASusA4Wf1ylKo\\\\\\\/o3JVJdEq\\\\\\\/kAPhTzXefxQZwRf3VHYEyxa3nEvF0mQ\\\\\\\/oJ5Y\\\\\\\/MdASA9C7Qu6PNZEdHcEMjUiLvlipdOo+ETl6mCAloyRLkrSJQPVvdBpJb8adrm5TaC\\\\\\\/s8F883qj5cPLBHXE9lAdnbvevuX7GlZgq1zpDbdQl9vACCtUAVcEFIXoemvYuAhGwBb5+iodGK9OtqmFLHqnw2XgNoW6atOGsnZ5U4CaVnXQzU89RkkFXVa2wpQQ3KEdmRKerowEa1scfdcqNf3w2WhEwplTM4DSPnYwpyCxEK7KVgsgk9faZgv6PZfH6N87gGzKXpoldxdipD5ZqjQj5VxtWfV2FqCpBljBGdQcEHMHvsFIZW7sx\\\\\\\/wHHGWgQx\\\\\\\/SoEjEZ5lkFyiFgEE+cEUe3gMx6hnjXRpyNclThMU8Lxs\\\\\\\/q85AlCVpieGJ2KUJB+fFGkIuvOkkpLN72ohMoGpZ6oc\\\\\\\/Q6RVdPDjOFm\\\\\\\/uF79mUdi6b021lpn2uafHnDs72cRqKHSRW+VIoZ2KO6gI8UgOGIn12FYuWV\\\\\\\/nkEp7d1HdRVJlBI3\\\\\\\/KisAd\\\\\\\/HCpevZl9zFNfnWakicUkobVAe6FpewkkEwGiGuans9B5b1FsB5F2Su7MRHTNUUg5mmk5GORyC2mp02gVao10Wno6htbPTIoS5Penf5768cvG2iMa6pg9Nvr0WpjH8x2UbPn0oYiYaeLYKLiIZqEU9eulDO395BdFcnCWcVskeUUAGXYkWvyxjL2M2AqyjgTHfY0Mq0sy6Egx\\\\\\\/KWToAxPTkR+06p5uH4v1bgZAwqPpiACAHpovNPY\\\\\\\/+22tfP61LvEjOMeQHiWMGJL\\\\\\\/7BrpQdMmTI5nUh4P0d1Q7r\\\\\\\/sW5vu0jUNfhMnxX5GD\\\\\\\/5o\\\\\\\/\\\\\\\/EksT8KeTYj+e6ehKmCKiFBpJMTCpZGlE\\\\\\\/9irQHcryU5wZLdH68e3\\\\\\\/aOEYRxzUOyeYxK8t0VwuOCzLGKIry50AyKhGuMEUQt0QI50KcxgM0RY9w19tdgTxI6a9sZT08Y2xiE+g5sqhR03KVXBgfkdVxkqchW+iQDLbSVpIAdGVm\\\\\\\/P9o8+YJhe6gEolrAHY0O21k97cdheOVIZPmDMcDH3yiJ9cxW4DNAsKnAWfOKsD79jljr7OEC\\\\\\\/oEOzghzGPSYpRHTF1aiQ3Jtcb\\\\\\\/TNir8i+Kafn25ZHO0T28ThkFAwvZ4qL0ifuuiFK92wKvE7j\\\\\\\/LirmPObTHRxRhwaIKJcANUx2CHpxLVohaQswG2KfwAwwkBv0Pt6cQaZnEHjIRD95zzPmLN2xJmkF9PYgHSPxZlFV8bTkGY5TUhlhuWBE8XSYMd3YaE04TeaKW8Z3e1x9O7FOHDZreLLXemsaJK+wnb8nKUTBw3sdwABNqOZiiRarN5FmD05p+V46O2C\\\\\\\/O2LtJPdeTdpYfwaJGfnl8jXRM6HnaCCorq3N9UXKhNHm0KJIHjZ8M0mn05FuqGfqSbLqJr8JnmsQB2CVaKW3tL1dSFpnEwY5y6HGt7VK348J9qVoWBmVz2CuvQ6zeKhsZ4LHpFSm4btz0m74AwI9gPxVE+mbymPHsEkur8exp8oBmabagdhU3Wx9PHfWD\\\\\\\/o\\\\\\\/bocx6012m66RugufLWGdwyUw42nTIzkxww4PBk2ObgtIX0S9JJ3lGp0mwM\\\\\\\/Fk4ci86Pe8MGnX7eZBJMxxU2dk9Dp1jU7ucVFey45NEJvHSO8aMR4Wg+KfEbHXYlVo5gOBn+opy\\\\\\\/fxSmCOlVGNaLnxmFwWFaFsncPCq7Jamj5zWQNSdUVrh7wcbl2siNLLP0GZWA3tXgCjy5C9lZITA+csMGxY97fNVrIfBfI1e3rwfkveqokMjhzdrLYzq2IXkRiH9WBYKtiH4kYufJFuaSbudWJAom1uF7Cat5Ih6dND2lsctMFqNj8qhdoO0mx\\\\\\\/953MZXv9gdkj6YlrN3AsYmpwWWLsSu0NhPNkhadjBllbGf9f4OdpZi7GpFcpd2+z6irmmF1hbnRoyfXKDMkaGOo9BIuUOahleZ2FbyS59vHdtxk5NOoAcnk+GE4eR0iLTNWbaD\\\\\\\/ho+WFtW8ZLxOxvs9KCElyrG6VpnR7Rt2IXSe3mpUY5gVASMbasCEq1Rj\\\\\\\/QQh5MSDr24Qb7U2WjUoM+wf\\\\\\\/ulsocg24srEP0EHSJqrdZ\\\\\\\/xR5SUcuxMdCaDRbw3rC0jnRqJnenViv+iKPIOHbOsUcdsJfsNzX7pJASW2Ac2NvyGUvSgKN0yxkdzNl\\\\\\\/ZbgGYy4XCaHP2qpjcuRjSCdd2RgXDxcMq7XcOzyFhhzkbQ8KEq7JERvGWLMOnsUDynX9jOSAmItIGHvpnkMi4i5+\\\\\\\/sgTt4ys8tw7Ix3QbYzsHqGtElrZD3rzh\\\\\\\/lLcSTHjW5rP3t7FSOXjwQXkYEBVxY5wzS7Jij94pHuhwcnbM9RhIfil8OCQtJyZ2QYQgmFzLmDX6\\\\\\\/7CEV8KNGuVcVdQJJCFtlNVJi0WLDkQ7YYJFOIddWeFJlMdxZPmri5wzJo01TyUHuqsfoiaw5igrX6tYfgtRRycvmqFRTdAGHiRUr2ednk1mlujfvntcoSppaMOT+ptzeAy3e+QaaWG+G\\\\\\\/9obk1h33M3pJf0dz0cs0kvIZVuyctjs7RdV0F2IGZi8egqXAamQVknZ6Ft9DQQbIMwtvPFIjq2Y\\\\\\\/vTkfPMNAMP1eATak6xclr1nWdBBkO+ipgHJLhGBeiI\\\\\\\/XsPHZetj6lENQmRUwzr4AA7BjwovXj0OBPPlmvCRgtry8g8WGOUm8dsGXl5Ly001D\\\\\\\/DupdaVA\\\\\\\/8SAcsDqVfez6162NqZdkAApJI5F9OSrLYJpO6e5orqLEo8TMUn7LcytNHmV535z7PNqdMsIyE58Pkf1k8X7E41sFdYiZP3Ln5+Qb38bcXKg5LZnA95Jl\\\\\\\/Hy7EIl5S3bl2n\\\\\\\/nJ8Y3jYmjI+SfmnegXptQbDAeO2vRmyZpQRFOut2RXIUz9\\\\\\\/Uc5rRROZ+wvnOnkHx3P8ZURiavEf\\\\\\\/KG9EvS4sSGdYWPNuad1B8MgfowkzSKCcyuJbCEiNFSdFubaCeruXTWAmUuxACMUTZ60rI2iZt7fM+noN6rhSMPew1pUOAZ4RvB9jT5DzmOSZ20tEjNtDZaZHsO5SKRN89OFjDkGfy0wfR+3TufhH+B5uBIHaM\\\\\\\/Ah2RaL6msWU4Nyus4TrqYah9D8A7XG6I68M9xkX7dn6WRbcNrldSWo5aMxBNYDUvTYsokDSEk4ebfe9RxDKojBhPGL4jy3y7dIrP8gecxdRHJb7Xa6EUum9bd\\\\\\\/mqrx4o9hZSC3ACWppCF9+LSfnYhuBXuVbjQi7wo8\\\\\\\/YK49lP0OoES0\\\\\\\/xpbm\\\\\\\/CZ5cz3iNoIzF8kZZwccbde2whe6yhPgIyDmZwzuBjTyHZfolky\\\\\\\/xKKKFGINFb3p5rJaRMLBdxhk6jlGshuU\\\\\\\/wmq6KVdZK+M4ozqz4bbAzwT7+Ve7JfMSW6RVyy67CChekuqbzoZ0hpx6FzyUZg9NFT+ydOAuAlC0Kgg6WD0HojMLeIvrBSpKcuIjybPIZfXPIHDjiLOG9O7lfXpe8F3Jdl0QY840PG0cn+5eQxlcLCZIBh9w10URW\\\\\\\/LdtFgrJlZNmNpXDO6bllR+HSs4tXlyZNZmHu0ptPDnhlcrzXu6Psf7qtUgECcEjhpp8ftl73QHc\\\\\\\/dGhkL0RHhRs4HD8HTd6keaImUZc8V5wUH4KX\\\\\\\/qv+NfxYHO\\\\\\\/QtFvzfx9lKMQbolP5GBsLZ9izakWBAuijW77VcnB6Xldc7TbJ9SilW3zalX2IbKlRyhQfNS\\\\\\\/KqapaKie5CIek+Ea+Yh3kANaoO\\\\\\\/IILA0xyxmo3U3zUAm9T8IDcQEKh5\\\\\\\/mjji5z\\\\\\\/wTGdJH1FItBvtMeotb6pVTpyk7tvfec8VbsNSizR2XHiKeithetOk8NSsejLtg+GDnN0GUTEtf+oHa50hTKxLN4iB31Uy0ea+imsZoWFD2JUcnrsjXwO9JbL8Er8aHgo\\\\\\\/+NDjUI4SZRX1\\\\\\\/aT8L8VveVFzgzfF3unVHR9CV6Uj84c9UhYjklY\\\\\\\/uZVauGwcYKNP5RyBXzRHs+bY7zl6alHSd9c78Cfkw9d3cdUslWlOyYApubyg6i\\\\\\\/CwgDPYjnPpXDnYYv5wOf5wkZ82S24SyO8MqfDcUrGhGih1vcl9FChpxPY8mOu5me1sDNBmSyT6ppLjB78jYAjt9Rfj8qJB9XElY0vG5sVANUTJhPBdUI+bAo\\\\\\\/LsW7vgQY\\\\\\\/y0\\\\\\\/YE34pJpdYG\\\\\\\/WtbkBzcBVXnL4UlFbkVoJGz0uFlIpb4Lozd8LrsTIuFGU59fHtRQ4MvaLVUSgO8bgq\\\\\\\/swFmBtgo3nCy8Q8hE5bnWcoUU2lVATI4aqzyiNqzSHclXtZzJeqln6ThduVSPwX5Xq9wSrOlLLNzzPzn41C+GaftPEU80vGsmaE47WMN+N8djBhf9Thxx7JNF2+gIAhtbaEaz1A\\\\\\\/+rJJTc3vSsusiJ2VZTLjiIN4Exyu3EUV3du4MY8d\\\\\\\/ainSC1sMkOROuER+M1U7wAiuTx5tn\\\\\\\/3XF75muRw6ptdztE+KROqd4w\\\\\\\/rmUgOSo9TlLr1SO5rfBUMj9\\\\\\\/yLZrbPKl4VqG6Q+ZJhqFceLpNp1LXXqgeTssyu\\\\\\\/O9Bg7K2tL2cVUZiuHDRCEVIRyE1CdWsmH5L1Gi7ZNuATgovFfutU2v2zItM+FjfG1KAo5r9qSq0CysVI\\\\\\\/vad6GoNI6pCHW+ihnB+gKBr3y+Iy6aRH\\\\\\\/o3hGOMid5\\\\\\\/ToRui0AO+lHAXxgkCA9qcUijWDPCLFUq5BLNJwBiKCm+aK6qJQsjuf5EZiq9FYqUocF3yCUwm1WSbepoeI\\\\\\\/oZ9pPsaWaopE75ZLozP89Uf8FKO3ica0i0a7gYI+0Dhb9+yxwSjBMvINCQLkF1loEfjcs9O2xvmuvCWWTkah7gN18kCbv5YNmYt\\\\\\\/LMxcLVfhS\\\\\\\/+gcHQrvQCfkPzGU10KFzdQCSu3Y1vqshDq7VuZpx6VWPioTragSTZQJMA3sp+U46VoVL1jUXRKY+SE0NM6IoBY4\\\\\\\/Iitx3hL0Fhazb7lviZa8MRrppZaB94MfSRs5o1Y28moQZMLjW1BvzZi3qXvYGLpHubGpOJPgu+kCn1yP45JzU6PDicFGuGQT+a17Y6ttSos9pTvjR54KQjLwJHVAJseu0Df4IhIgMz89KqyAE\\\\\\\/59\\\\\\\/QqbYT15fo+gm4tCTpKhHq6aa+Y3ILXscFCJDc6QtPVyTuQ8zbTNM26kQVVo2B+bXHKSeFK\\\\\\\/0tvehW0RxpE8t8RvS4r2IpL3sSGY3tJsXXAzcpcqzxMThb0dogJa\\\\\\\/gKulWGSrq+TcRihVLLk1mRzhXGpR2nkEnAkJD4CrkPWP8fCMclqmhlAgwQeCpZc9+hbx4gDid4ECNotBdqF5vJ294gAKMeWpEFwzQAcY+VlYyGo++WWPiQSIUXlMbm1KuR74Rayuy67olzroYmN1YvX0f5n8zX5HtiQh7Wu3FzdcM5gWNRyE30zQ++OqSrtjmksXMmtIE4rK8rgx3F8cjA6bWf87X\\\\\\\/Rm1tnzu+2847eM8jk4bK6fK0C+NeG1xaeaAiihzhIWe7D77SSf+P\\\\\\\/R5L2ztinVwEcSz2fD7Y4WaYiiGsmC4fGyjuN0bo02O6kBw6Ad8jbXUudTcx3h5YoNohBEQYDggd8CCmHkKpXz09x08hppEkMRJAQs9\\\\\\\/ivwrxS7j\\\\\\\/ycA3FGvnw+J4UWC894cjHBJWibiEIsqzBGsXDJOxHPFRrvviEBhEwFHjj3OcZxabk1HVyDduOaI6UFCH7MTZrF0a+oQPMPFPLyRKnXjSK0JF\\\\\\\/JPhmhaSKiMTYo4NvA+Z7FRYTSczcbrHc692aqSYKnP0+jAjvpKHNEWym98VGYj7Mg93cjIYF+n5Eut3QDsXzrLo73+YQ7SdbZhMtDc9udUJgGvQNFVF8GASzvuHTAoTPrhC1NyXLTnCelCu2sXlkNWxnzHk1ownexpJPYNP50NqXC5sPrjwRCH0HmekbR7UbnBQkEZ0U+rA\\\\\\\/7UMzVsv1wYjGAvnO71WM58\\\\\\\/drhZABTzwzAq4Gg1Kx+4c+XuGUFec3hLHX8HfRjGpC33UfkvBAhk6vcX\\\\\\\/U5VegpHgdJE9MZ8hxUr191lqZxacYEA6ZGaMrxhl0zLkno+QRs\\\\\\\/M9NMg7yW4qF4ZvYoaKM2JCIRm9yRREpBYqXTu7p8LPo5vtrHd6V3He997cF7GOL14jyIKQWm6B7hgq882HxuJWnUMG1TqP7PuRaZGNL0\\\\\\\/Mx2gfepubYzcryj0\\\\\\\/tq82vJsqtAxWDEF9CJk0Lw7zn+Iis4jRmU193+2i7XrFvFi+UteEiTm1\\\\\\\/OR2p5FR+WpiULFUllCTeGWyscSEfoiDOXvpRCX24YguQjJXouQqPlY0tFuK3GKgD8ETC9CEnH2uf4X72b9QExUHShje\\\\\\\/u3om8Vw\\\\\\\/anveXnosZi8GLPwHzsn+XPFBBD70Jg9J2lsh1e\\\\\\\/QRzkJlQHjlcJZNbuSXO9oORtgOaDUDgKsM8kAIZxNRZ7WIOCwOtZZXMNhPRLgOTPJwBL8+0KxF0TF8LEB536VoYIk\\\\\\\/soRGimhMowxaY2sAJj2y0tP5J4s+HyJjzheaEMegtgxkn\\\\\\\/A9p4QicTc3bx9TUccDNrywrPjWA2js2hUVc\\\\\\\/t90TcIjmlHTIznWIAm3BL9KkousLqWgak3TAtFdkCnfZz1OvLYe3dvdeRcBdupxhpNHBJj6n6X9encIN\\\\\\\/+g2rvCSLVec4o7XjeYCGSJ7ujWYUkZe8nRq5WNCfX8ECYcmTuvTLAfQn+wK45xFvhhGw\\\\\\\/4qKxXMMqlge8kEvxQHAYaVJs1EjtnSQF0EXkEKs0\\\\\\\/1tgJ4OVBxiBZZq+QuVO3wJM\\\\\\\/cU3lePrO8us6A9GHiPa2ebQGvCwvMlFDZpFinGqkbwmZhUoh0uh4wUmED+ziIOWxL+T382BVugHKZSk0wPZunM\\\\\\\/9rsTva\\\\\\\/2mn+SAG0d\\\\\\\/bGFAUsxApJVAVaTMXwlpHdVonebXtrzK3+bOA19lccQQvescwU6bhHc+AyXw\\\\\\\/mASg3Vt6JG3H3rMQxvgfNm9rIoA5F72BUPPpzTDpsW+xjobuKtcfMCkdFgOoZnvvFfbtdpkT6\\\\\\\/hFDGhhB3BmRyweCuqT2vzUGhCOswT94zfc0TLmByinr3XR2\\\\\\\/zJ2lCImVchh22l2\\\\\\\/\\\\\\\/CRAXLsew7xpPuYiiWSCPmHdaQzqeTSOl0oON8Q7BMBf1l9qmHVD7LiFU+gAa8VA+klPxTVVPknlNyP6GKqMQZT8GurroKwbgI+IiXWdWRNE72K0AGyXWPPG6TlvqYm7GgKIfU\\\\\\\/4MNRFeEIRY+hx6aZY12cfZ1pZs9mfiiIettxSJajY2CX+pojxeYUE8noFQCswdehI8SHNBuyA91lmECZvYzui3ImdpwnNxxejIptlzlkdZ3dARVvu0Zs6je+\\\\\\\/4Yv2Y+aaJul6mxpPoFMvoR3itgU5GSA3BKtS+s5BPYtl1iUaCeQImPFnZZygN7oMDq3qudtuCPcrYJKX0vd90iSrBoIbUaLW5GWLOXsTkqNxqCI+h6hwuSRyKeN3K2INwyQHY105YrH8accH\\\\\\\/IunD6AC89xDBpx7B8Q\\\\\\\/J3CcF++raUODXmdA+1wmfzUFBgrKglgV+kYr4FYniFZ6LR7Y8sEau\\\\\\\/+qXLfctjYhzEGaaocZPPScjoqsWzm9g\\\\\\\/dohG2o\\\\\\\/dA3OpB7t1h1hq9eZTpXOIM0bhqaGuGJokJW8VMGMKZchzTU7Hv7I+DzSl4R0FK3vOI7iWRltqa0CubQqNn\\\\\\\/Tnq0fdSHkdsx4jTh94JwSr4A0Ld8eGdmTy\\\\\\\/Y0WoFVNe7lvZdEka2qi7z3CaLrEQnnOLUGMDdAcFc3dowQWaVmdwCVm3eRtFctOgo8hc+D98Mr2linNB\\\\\\\/k\\\\\\\/AEmHxRxtc+98UP65s7X7W06Ks03iUf6TOysw79PNDb4QcGH22mQVyoAeO6bWNL5Nd71djxVNyyf96x4brNRcHGM7VFAUXZa5lq1t\\\\\\\/8DblFbA6NkjI2WLkUyqc8oorsnZXNtIxesDZAql1WkBQhES5g5Qr4amzb3Z\\\\\\\/udyEfRWT2VhqQOLV1gTvDSd++EKfC0h5m7cqdWv2WYSoh4rz4T3OePj7vMEZnKv5sHB0JGEHBmLvtPuCVsInqja5q95q1eHRXe0B+IBboGxgBKQiFfXRzpXAJi7LCIFVsliJj4lpLTjuQk35QfTt06h5foGrHWDsguyJUNNqXNMnCUYoFoWNrogdJK98GgDSN\\\\\\\/K\\\\\\\/BpjdShJwI9E3CqnHPeSg4LhJ7mkZSckLIjw1seoXIaXt2UbaH3WDRrK+063A005Yt3ffPbAP5Kc+RfwU8CbOrHiV5OdD9JZs+yhsRIqyMsuNppuGanrvxzDUbNMHYsmR5wkFOGjXHsx+8VIgd6tRAC6OP5uhwNdj+j5EN9enZUn2qEml5GqPNsaEnZqXR0N7OTgDraBSE84aJY7t5TYYdID6NV05pPGSJfPGVSO4GxHYxwQSiQeGjdSyCyt5WQ6way9y\\\\\\\/vGeR2KvPaxmlapC6+XZEDoS0j+CgBp1WDEBmcnxfFYK2ODUgYffjmnoBWPT6ZaBlJH+b8hB0OwWpXiyyt8tUfhPP8kU+hq2wEyXqc\\\\\\\/n1WOL7ltMrX0G2RPYQxvN+6WvX88OdnT4aT8EMdUPWshmGrsyC8fQfSrwJCREw4XuPl0MvfIccj+XSg8YxX+by\\\\\\\/0yZeCyM5TCXhfg2PrTNaAoraGTYwrsuI6nnM6Z2mhvObN7XY0vyBJ7akbbK8RMPXTl2zj+dfCloXoiaeAkoUD2WwYgvkH8SrothF8v\\\\\\\/kq4iadVabmycxQgDg2ict341p6k3tRBXDd0OxUUAAAc7TK5+IR13VKXGfsFvmz1SeVqBRWZEXvclqCLoyCu48hWXm\\\\\\\/C4vfcPeSAdQEgn5X7U5ytOzJmayLFKqLa\\\\\\\/QJXjogdxbqFiFq0Q\\\\\\\/dQuln9KbtR6C3AiodyQ\\\\\\\/I8KKKasaHyH69xaf\\\\\\\/VgTc52Xi\\\\\\\/MNMdvzgzdvs5OsvDX1XdD\\\\\\\/JnCq0g0Ocp1Z\\\\\\\/Rp26QYmBwsTXLwJjXdkc\\\\\\\/0bnYMu+fk9353hZEj+2ieCa0uirBEUbQvK1pkWDG82GVcGHso6B6U19\\\\\\\/Tet7\\\\\\\/Zhf0TmxZHkZE66tdHIbg08kVnTvEXbi1Nvlc8QWAjDu+RKxM1RshFSSkjPl1Qg0KxE5nAz4zEE5OjcYjF1oDxLGiKelkNUk6IkEPN5yFp7Hv9ZhUINbGIqOTt9Sz2j3wua59\\\\\\\/9N3FhQ2JhrizcXfX+t3tTUhmYcr4u325c1EEaW++ANduBvtaiQpzicckdY7VZoa\\\\\\\/3SmFdIdG\\\\\\\/tm6YA23C0SR2WTL\\\\\\\/Zu1i33Air58+c39Kp1k85jmS8jwSIHlUM2VedI+sO5ItAb5y5L+UOg4q8TO4aJ9CNA3+czTobOqfAI741ErVT2e9Jya48ZcOsUGpllEP1aWJJhyitxY4RADQiav1Fm5YATNc4VWOzPg1l1FSmAi6zMDTrH+y71hzPDwrAsV8BfIvT22nfxKOpY5tZZXwtJcZuaRq6qi\\\\\\\/zgNPXMkR4klqxWVM9rx7hhTBuPKbS16PazCTzMG7X2zwiTC7GJW79KFMEpwnueNs8ZR28dGbDu2e+4aFOm\\\\\\\/dE89UIfrtvDz2WhRxJH42Drw+LZ2pWosY8HK0S7sd0MG6uxMLrK4UE5v3Un0\\\\\\\/h1PGTP\\\\\\\/hjfvOpaoDe3Bg4sl\\\\\\\/k7ihfAnGM76HZOrLQgPN2+zibKEQSNpnRQiy0KC3BDZq2XoCugkO35H8ZbsLB\\\\\\\/QsnBi7N25iRWkWnLCHGKB1qlAvOHjrIYAxcn4ql9Le2goOMR1Xu0RwZT+ri3vpGm1Zw\\\\\\\/y4ZSwN9BWEC5o8pHYOi9AHxZ8HNo8RdGS0+6hVJVKPQfT\\\\\\\/VzbxuPe+04Lwz2+4r+VAPLTYuY7MGdrJk219mn3l3oKgkkqJrl2srUYWeREPWxCfOJL4JdMttX8yFEEe79hTllthOpIWyYm0WHfNf8OkukFlpuO450qggPRkS4hJyABwnfkSDzosS5yQ5\\\\\\\/3jm9f1\\\\\\\/uG35jXll5XtDX4kxD9li4oPiyFzXemH1Rc9vdWSyfCGVmb4V8bHbhgZoVpQ1j6tnrlSlNjJX643\\\\\\\/BxoLfgNx+12Ew7vVxOE5bgbX4LXHIhMhCpibgyBsD+klAoai1DmL3aiZhK5rmHGri7rVVHvVzM6\\\\\\\/iPL+PnlCWHdYq06jWZ3qz+Tz\\\\\\\/0aQRtq8H2dxD33Z1N0kO4l+tCaG1A\\\\\\\/IX1Y+2DMSf15f4xVCyxMk6p3J0f3MZl0hobZ80ZEx83R1nUCu2G5Kexj5+4YE1ilsCxdzI\\\\\\\/XFIsRgBzwMVQ0rdAoDfAm\\\\\\\/8SF0uTTLUSEyoDyJWggf2RjqJnVcpuAocIO6uR5g+5Ljxbd4ivjB5+rxfuzKxa8rL3RPaaQca+jLkBiidMROESDfuOh1yDCM\\\\\\\/AH08XIobjnyIZhfnoq7h0BqbqxXz3cwd6Mn56d8XF9WzGeh\\\\\\\/2KAB2Zz3I1dD6o+9UxsEk6y9\\\\\\\/Fslclb9ZYf0BChaM5n49RBetV+ohjQqxWqgR37U1BIRg\\\\\\\/4vvCo7RisVsU76M6yMbXmN2uTCcyWUyCzdbzyhDIOdFx5Y0oDVggjBpwbxSKPNr5SZGy3HyqXXHyY06pQ6Unrbtdp5lF1Epve9IlIeSVzr1sGCOQTMJ+bvtDJT3il+z2gnxmvjtF26avE0b5bbHV0A1GbMqXdq8oDBucpcJqfFoCUEyI4HoeqhcwzFbMjRW5nupZu+MbkUda9Vn5P77meQYQQWoN6Tgnb3zZIgEWWSpftDQ5biD3xWrE\\\\\\\/WmdKVF+lYEf1YqQ15KyeW5lMDmsqmdqcj8R8DWc+PrGb0E6wgIz6Ihb+lbVIlhbAera7zBGBuDnPN5TwXD5nkVu3lPUFdoT18i+io5YnYkbt7sSfprdgyjXmJLn+MLyCjv1\\\\\\\/37V+zT8beX3B+zpCKx801hcWXl81pNDdC+DIqXRzsP55EJwBu3WQZVrtmYfRWpudPTRfG1U301kdr3DL5zAkk5w+p5CFuR2PVFns2rEV8H9bFFVBybJAfsa+p2xK6O0sUIOLKQqxumYMTIhl5dDq8uyC3mspnMKjbfdkZ5J6eDcyAQf0bb+GpfrgkKTKFaxlwt3tg32zl9\\\\\\\/fnIAd\\\\\\\/v\\\\\\\/1jdNxEXMetwObaVm9OhK\\\\\\\/E\\\\\\\/wPNfWyiNNkVwFTY+5V1qeq3Im7KFCAsw6x\\\\\\\/CDroTC+fLhOuK9gBohB87Wq9FKMQdvFHj6TGO23\\\\\\\/HmjXlNmWODSRRXzMorR\\\\\\\/Qcn1mpQS+vSxHmgVylmwCkyzw3Od07CUJ2glfvb5yv8Nq1cB3LBiEiQjigexVXljBLZe22Dvr2UT510CQXaM4pT4TRsMuE3IqL6exVb1JrjoJcEakGvCRud\\\\\\\/05DpSDk41iyn0xSRETcAUkqWQHf\\\\\\\/IFd076rNQmB8MwMFKIPYlcwiz0dYPrVWgnAOui4RAmC4GFpv4wdrS20dgw9SBH1CAgphXAVHDQAxLZewYb8K\\\\\\\/JTE01VJF4kmc\\\\\\\/816\\\\\\\/0vpf\\\\\\\/QdDswGfzZAG4++7\\\\\\\/4oBJFil6GKPUu4YLH25xl3G0UjFultbQWUH55R245vnXoa3JvgY+8RBxR9aMBLKDatCdtMIt6u8vOzKYTqdX7XszlHTqRO9tFXLkDrBGjWiF1dNbVpXDeHgKX2ts0s+dRfdrw6nJ\\\\\\\/WZSWRGWo319sS2k6PV\\\\\\\/G7ca+OZrLIwO73ZhNbjCDhjceaQj3icr4FBDjbV9l6AyUX27aObgARf0+jQslDXBjaMRlp4uXkQVY+VIBR76QCIULyew64cKbfmgINui\\\\\\\/Gi6bE3u+IFOliS5SPF6P7PV+VhcvDP3Ec8oWEiBJdEOgttoFXXJnSVwm03kx7zOeLNaunH9d\\\\\\\/h9rYCCwZzPfWYuTMLeJbb3h4msJvNCc2yB0\\\\\\\/TRXF0tFmJgqElsuBnb\\\\\\\/qt9oDIQ7mD1F1JaNRlgBmD0psFh13pUdU5l2n++Cj00VK+3ObIwMTFcqz+o0o305UD+TnHpWEpvzQxjLtbLyjVgrWA4ULoPegWh50CTNwaE+xBJQKPnxmSZivxqmhXsdJQ\\\\\\\/qdxTn8wM4Qv0auAulGTYosISEtw3zaHz2DOrDmGixWPKkDIs0QKxAKeiECv\\\\\\\/ITBvB2w5QSWCEtNAVGQwljIM52nge\\\\\\\/DOf7yAJfogNo56ZSFipwtedq1tqo5bj29sOjw+jaYtbqtpFdgML\\\\\\\/tL6rd2Fkln6b1omhheERpr\\\\\\\/mk\\\\\\\/d+Q6WGrh46kVP2GIOex9m4aBaLaDN0XZRqC75aWw4De8Zolc8X2qgE+nqsFKGUvaI6tQrLjwCQQVz8\\\\\\\/8vGOa2ljr8fDYbnyQn+grB4gzmCX+oY8Uc+bCegLQM8k1ZVjj4VwDioxuedLEbHcBqVs5oKgxHt075kgTNKKuMgIOrCcRbIaXnWHNL+FTDNxxb6+7o1Kea1DQiqfUN3OGGxI4odfz1XFfxyLVSnWJrTpM+vn0CImc2ksDK9kgWBNI4k+WcpSVJyG6GypvhDzT3kUof9guMB+fbybVTlb5FDhqH7P42BKEfdTjMNDmwGPI2gg79q95uTjmQ7BKdG81u+zJ0HFq\\\\\\\/1yk4EkKmamcOUn2GD36by0yc3dDO9L0Ki4rTY1\\\\\\\/IEN0V4bAAcC506eBwEm9jV4pPEkvzvBWft3ADHVV\\\\\\\/txdB0+LEM+jaGVua\\\\\\\/ngVxQq25auc1IezhJ3I7x3FqD1AkXH3\\\\\\\/wc00u4YootywvpfNPVfHZw4tsxQiWhkoW0qFZ6mmYctqW9+O3W3wnkAoS3E9hlVYuAsRg1cTYFFD07NesvsABrXAZH6LupU8dkL\\\\\\\/eMTZa4ljH6e9IIGpEj6e+VWnjH0gKioCTMIXlj0dj+9hduHtJRLPeMCvoHv3kHGrUoqLsLIEXl9\\\\\\\/If\\\\\\\/ztgmawCEOckMtZDtf50JKHILkK1DCKotb598hgbuzmhlA+EjdOr4bInq3iCZgqe8BKWiXL8akSx\\\\\\\/RzesRW2\\\\\\\/jG6l+euWg9gNe85r5\\\\\\\/2Iun4HzSN70QocDJB6qZ+Se2jZXLGN5qK0gC\\\\\\\/bT9qYSjNYI1c6NcO4aJNHTG\\\\\\\/ZH6AdaKg68HYf1JBUp6RrSx21eOF9k8xfAv6C8A9SZSocwr+3FCfRnkhpzjJCPtWIWbSY\\\\\\\/HarJpbEPMT5lTnpnYyQIplpfaRnuy0\\\\\\\/J2KFEbd1TbHmyHbvhu1XY\\\\\\\/aKdCV+vJYLWnCrPyNjmuyk5xfL9QEMxY9rPk+hWBeRZ8e\\\\\\\/O4higtWee1U2cJwgR5wpobsft3HDy\\\\\\\/dH2EygGe7Lttg1w60IuaA8PjH2WFlyJ1YkEsH65JZef\\\\\\\/uooK3o0wPlJn5xPTR3p2tCeRKVTYPGzbxFrjce5E5f+xEdeeUjh2wmKu6xzVzNk3jDSL5iT44HNZJRPJx57kh2KhPEwmASrBkk\\\\\\\/9hrppL36u+d2D9aawUGWARH0Z+34bbQlW4JeMeYqb5kAdItSoA9nXl0S83cxhz9AoICj9dUgAWWycO2RONKAuVmelbD63L+QOER3giWeRCf\\\\\\\/itrWGKOV6WjymTfkvAtHUITJqjl8+SLFEt7VT8TfruRikcPWwoZTQPHs+vHzFmRMVAP+fPiFeY\\\\\\\/vhwbR\\\\\\\/0Y0WDkP7RrOa+QoBFWRvgJuz5Vl86V0kRg1W90OhgPfYY39Ap8GQkHNIeXEMVB+7\\\\\\\/IupBcPn91ReE59l\\\\\\\/qwi4N0cNsfzxIzsmu7B2SbDc4BmaOJD8AYyDPhIhnISW10ZxuOxb8v37Nryd57KsmqEsWaHokM55TdKc2wyeU8C5aLPp0cKEypwbKn6u9Q6BuaAMCXH0i+Kc2hqez9CFbLnkgn\\\\\\\/k7nio3tcOdgFfhzZGX0MwlYlFC2wf+Li2vksH\\\\\\\/YDriDRfwIqa1k3H\\\\\\\/Can3fNvrrVTNrKXCXwmD+78HJcEKM6YddodYO37Y5p4E2NXBmVLnnF\\\\\\\/6MGEg1S+cVpCA7AI3A+GmArXTzlsD5Zk6fQR69asQtkTjRVGx3hZ00cQ1eBSAZUkmzMXSX5XFr4M70GgMfQGovZ+iXTN9xAmOKrmsPdfe4Ok0OP4jb4BvudoeC2wmAkfQWbQzTOSRBTa5AoEMsbC42M9EnXnLZGOeeqSXgAIrl2tV9NJ7xsgG8xU2LDcF6+bUED8B12hShhtn8KMUa5j+rZWd29Zs2x5CbBU1CGlcXL4zuJONxHw5MuSps0O1lyU0DJMpLcWllorYQ3rnTQ9kux60dQPfLLEVd8bHzT6VmUdXI4DVA5QwKOIrlOBaS83GyfSNc89KAi73XKX1bdo8s5YnBM0gYkORWH73zOudCHwx8sh\\\\\\\/GuxQVVDwzDSQBfhS2iEptWKmm0koyOTZo+vNgHrx61Mv9lwzmZh8WFWyVMC+rtUJF2oL12Ob3mIn0FTjHIanOqNUkWIxZynLjqsJ1KsYNYeGPmeXHj0mc4uhPp5ZvCBYFfeX7LLAjQhhzNp4QEOUGweEbIoQDjG90aLLDmuNvHaD\\\\\\\/AIhIkxo4jUAXaRpgtxfGAzWynbfc6XzOnhOe2Te1ZVXbPQAlP4OPhTYCFdhFtr3I2AkWxGRoLT6GeNvETk4Froa7bMLeS87pnMQ6iJZfkjRduO8PnDq4mO2K9mFILZA2KkzmYz0iqM1Drutr7C91OLBoz6uk9TEsOvaP2KNJyCC9AJ1OEi7vjspUXjcFox7yrdlnbdPYYiVaFeNI7gkum3LerZ2HQrPD5P+6tQF1VsNDkcZS52UdDPxJo7Ze47xIXIIiOXRb5u0ONbMds7bGFOFjl5FuKl+EgdaM8TaJSjIvt7rLIvwWj6BT9szHVduTfQlSusiosVXqBZx9D93NOOcWGiIBz4dWJaqN0hnosMfshxvLei2aPuuvvted1OQMOoHPezoe329PevJCgUegeRsgJw96RjQzD5h8M+VuOHe1e24eUF4paEVNjaqm7+OUHt7hIKfeybwDa50ex2omUSSpMshzIQMAhNFPUuSL1jYW9L5VySoXvijcy\\\\\\\/OGnO6oYHg85G7dfNX+hSL3I7ow2C5ndq8RHoXnGeX4uONXi0X6kSLu1F1sfCWv0dORwiLeBI2ZD0SpMUPI8SgODEn6VpR2njIXYvkObMGEgvJ6u2UjY0dGIzMFeqG6CwyAAsynJjIaNiE61yXhif0Z+QvzbGaWrAdRSO95H\\\\\\\/VIsi9xlb0Z+smH6RDmpN9BMD3qQBBfKYx0vv16JBwTkNyRykL36WCTkFiZqkn55YttCZ2ihpJ3NUmgkJW4LbNeLs+6EuR95b+Fm3L1udxDwFSxMAXHybNXVamHjiN9ao9lywW238IkbTgsEnZayfmcHWPa+zZffJ6zt68otCFJutC9gJGFQNG9sx7QVNvJ5VnOz1NCN2Kuolne+zAUhqTlge2cLegCOYOdSil4iJhgj9yvOj4uj+CjsAs6W8+YMsvvx0x6Nxz+FsDGgbO0o7+uT4Wrkfhg3I1Q+wZ7GEsiocWvLCCxwzXvdWkU88vBoxRNYwhxTvzjj1oYq0gOV3GmLmeqWB7BkCgr1U4xuE3NKZYHaIowzeDKHPKDBWoUhp0O3BJ6F0jmH\\\\\\\/RggckgnRYL2KWJa3UmH9V1xo2OEejjuIGnMzh\\\\\\\/p3wym0LJgGKWFwM13i3IcmSOq+bMUK1DoJVjCmYVvF3bzAUnZ4T6B6zHvpLCXZIV5Kf6sfTDcLRsiN\\\\\\\/jlIKs6EsDQC82s\\\\\\\/LB7\\\\\\\/j4MlHCvpBxUL9bgPtsPIFvF1YT2Jlo+0WQqiOZrpVPNhDMoso23Z+k9X+TxjwmC4rM3luWdhyz1heHmJ7ds0tRvJOE4jcW9mtP0Dzv7AVnhMkJApCwOsQlx1Kf0WWKDCDxjq2WAA5g+K3L9dno2BNCV8+wG7094P8TReKNhiRSXlGWg7Bj0IuT\\\\\\\/K5MoJwv2\\\\\\\/t\\\\\\\/hnb3usqe+Jb1KeQcKS+3JDJFWlDjShGWQs0LvvDmlCN51CvDBSYDtlm8tm\\\\\\\/PzB+PSXiBQbYJkWv2Xs\\\\\\\/R5G\\\\\\\/1n79LszanqvZuHfBUjRsx97O5V6\\\\\\\/9BKFMTrQ4+zPhp6l15MVH0\\\\\\\/fx2oKE1XrZTVApB0uZo5iErP8cXodfmy9CumujUVybdWT06zBmqMVRYugdeVkBxOq0qSrP2OYYK8b8I5UNNWeNOp2vQCweVLOvJdeVVkjb6VXKrjE763sNjXWGi+BucbxQUFtORq6QaeBnu3KCsvNeq99L4viBj\\\\\\\/g+IExIWZtco+fESRH1Y+CxxTUcMiT9rcqbzG8V6kPvhAKqU7wF4oc68vc4euXXD\\\\\\\/WUS\\\\\\\/3zSjB9is+U2XvACFrwSe7qa25q12V2majUtYUtGmC2bvDn47pw9c+7jJD8kamWU5DSKXVu974MHvCZ7DoKPwKfPMSBtql8MxrKe6IPSlZnRWXNx70H2XjKkzb\\\\\\\/gjhUj7chE5DMSbfBOfP+lMBjvskYky1R4\\\\\\\/fGKD9sJRyowFoVXzTLUN\\\\\\\/3MG2E+bvP5ya9OerYeAfrypGKMcZjpIxiTAVOIE8XFEPu1lWSeJXsdL0yjmrLp3PIqIy7vTBFk3A7bXl\\\\\\\/3E8kiNxb+zsYPDLzlzZxpvTCru48vjIPbed+wVyNODkDDiQps5S3+IsTW4+Jy5Vw7jUwk9NhJ3pA9sjvLZ\\\\\\\/6nOJPZCGgy62LLbEiDKdslMvJXb2vDwhmUtT++Udyx\\\\\\\/J\\\\\\\/IXdARng1OGiEtCDcI2Q2tCw9XGlu6\\\\\\\/AxdFFix1ddAn9HWZnRhlUSXTfNYe89qt0EkPMOUz9Sy1rA7V5WuxGYXvQprDe22UvKJE2zZ+91OZf+bYRiMaMFINzvZbkFfDE9VhLbvrq6jr5+6b1EzqyaT9wZe\\\\\\\/rhEdog9syYc3d1TUZKoydx90IxI91VFWXTmPjJ2kztSg4H63V4HQWDGMqe+PNcWlpAcS6yFvzpfFTa6eW2vtjKWPnMmod9fbO3SXnXmCTBkkvQN1PB3BXND0dVvtWZlo80d7I5S1Lo4vH9lugjMeCApva9Ps9S2rz9qUW7HaLyYpn4cVWmhs92jl\\\\\\\/NO044d\\\\\\\/1n6m7zyXO5iOLZSrUGXeoz0OHo4BTkeDHPOl9G7wPdAN1FAphV672U0KQD05\\\\\\\/oQpAgabcny0GVAZ0lHPzf8ONfW7pvB7yMREM+LJHUnUyMZW2qHhCTb8WoOiy2yGFeeerJcx7j4Egh9vO7bHFsZGJjvWillQSzZQ4TBCi4LKSTajdvSyJVKyZSM1wpgLeiLKQwpPQQP8ZwRkdbTgkZZ3HPMLzSutwDupPQihhkt\\\\\\\/t5lHOxepvbCoe46VEs7RCOWKAaM5CxJ3RD2nLBz1fo6w1jE99Kqrv4AEF0E+n1s1yv01ctsJavtcVe2tIfyd9gMPdkqVw4ufFwhaI7u+gJSdFcjB0Vbn6KdYXUzHFdZi6iICdwu37q6KaYhQU5kgup3qRWiLwltgMmNyphwvvefvbet\\\\\\\/0yebSrJWQgD+uUWrIXF2JCjDn8XN4etungIRzX1b6UPcTHccvNvSzUjHpS1E33s1ehpcm91yaw+5KVBO1yYhA7e+6v2t6+FGe2F4PO8b1TlwfniuPQHBzOl+MVeWwXHh\\\\\\\/i1\\\\\\\/ZLjWkJoXdi60MLFB\\\\\\\/lrUMKtFSUD85tYoiupiZzqM1h+wHpYEpy6BYSQlDhcAhYoMbW6X5OG6JTYiZOFaOBvu4dA6fTXTfEBagasSaPN+0iE\\\\\\\/2fbG7xaqDybajo2d1yGJ4TzWph+0rdEBOdUjicUAEmVfCvKNeegnr8hG0XSkTvJL9L+NX+sNrHAzsw9jXeiZI6lEn7OZZrqP3tPGXqLjvkK9P9n7YEUh9T+VfteLbV+rgvCg+sTsfO2SDM\\\\\\\/1\\\\\\\/olbzn4wBOfwi7WsYIy3cirDthfRNfQbzDMcXf2JRk97qyfmNs7Fdh\\\\\\\/ooEuCkOkDOjHfYYtceqlr7JCQ4P77PpMlGr0kKep4cY5Two7ryOBYhLJ3qgsR5PFAXxE1zHZA0iFAydKFWtAwj4oZEhYraSmwktp7wi7bv+Q77t0VsBXtzIqNgsoP8yvzLyxRxJQ7NT08w5Zz4vfWMI9bpZazGLz8UbUM3sC8a3jrYKcrNUA3UATE+BNfIMb\\\\\\\/07kMckrgavJyGgktTpdhv85sBNF6lfgfbCwzQMG2hepkKw5M47SiHD0sXdXARWBXUopVkCbey3ZNcLJnAIjML96o\\\\\\\/cc4Xtkgexl7Fx8lEN5XlAEsL54KaOLaydQGoX8uWsfTe61BkYCgHtkgS+LRFaWrIXolVMbob0yx26JRqTCy\\\\\\\/5jK1eiuedILsFpUdYxWLZWi+j2k8KlGuLaEhV5ZZ\\\\\\\/wXb88xW9jPlENW4dVVpux8OWQD6OyVRd10mdYvglXdBubo2rbaNJOIn\\\\\\\/PYjUJLc5IDE\\\\\\\/h\\\\\\\/PlA4gh3WJesEaHjNDMNtpCm1UeLcSGKz72gcrhLJ1RfXW9mqWKQe48HHN1MqZNoZzEceADZCV0Eli2f2ZtBuXq7\\\\\\\/94oOuT9S\\\\\\\/oqhfdm3LeksRNduJFRDJosGvPDy\\\\\\\/MORn3KjQNYOEI4VneTmD\\\\\\\/\\\\\\\/ajHqXLTHuipT\\\\\\\/zUpcperIFfAQIkSD9P\\\\\\\/9XD4z+0erHG7\\\\\\\/cqP1aBJu\\\\\\\/qFR90u37o+5UEPLbMKHvhuK2kjz6L8gR9HmRuTwjtuM0LCb4HvuZE9ux55X1CQvAeLt0hnUmMFPF71e8LHHFpN9nwDwsDhukQLkNIpWaXntIPXpamQsGvrg2NS7bLE9Xp1rUIGpQYduyEo4Ign\\\\\\\/Cq5ocDeVDjL7JEGNZ0c\\\\\\\/gutI3pWNo5n1Qfqd\\\\\\\/HBctWZ186EiuQsWBVTvfQqU3aUkNhrN4ic8DIq8B4PAA3BORAOf5jCvUvZludB1cXKSv8gd3nNo0o1\\\\\\\/5v4k9kQQyz+7QesLT2gOsZ9ZXzxLRP9hmEHsNC+z73+IHxkgGEblXSJckei3sWRFOO+N7z727JobHGalB7dBV4GyLM7UzbAY6o27WwsXAamCy9MA7Q6QZ9VNxyP2lm4vW2SunrkOpUKhpRWjReI1MCnoW\\\\\\\/Kyu7i4UYMBDrNDTFGXMq0nJ9Cwp0KGc+JYkOsWGcNn9a9p22Juj9dhMWg6kerQJtTR58ILdnIqRSnksdoDdIqe2dcKikT5eMNQFM7CKtiIzXuETqlzLdnuWvRITQI77T5JD89gdojseFHU6QEdRjfC6Z3qXymFoqnd4G4u8yQissB00xCQKDSQvuE5kgw57taj8yHcnIlfzU384lapmjOHMVD3Hp4Y8rg0Q2eInaVFjJjVQVO9yn045uwy9SLgiVmrWnXLF89GMn7Zom+4kMH0sGums3RyWsyP4eyVSNmjeDVrYgrAmp8KNKO\\\\\\\/gjuXm7QAU508YdbYsQ7BHdG6C68Ss9ZwXOPzYPkfO8\\\\\\\/jotMjoLn3zz1s6M1A46hjdiElB0dYRfW9g6kMLufXZanLMSgCKcTX4Ice2QC0VE9n7xsXRVg3O51GDugVTsmkCkc0XWabck5l+M1qgaGAqOzTokHsNsaXv0b2TZ8Jf\\\\\\\/IXMYa9sPZq02rwQrSDbShEpQjNWaOlPXSK2eAu77zTZ1bdMMh81L1dUT\\\\\\\/VockQFtUOF5lNb99AA5yAca0z0ZK27FzAAVUBeMTBgS03z2uudJoByxPq2Zx4awYDpcgCIli11ntzGQqhuUxZTgX+3s\\\\\\\/KIelOli\\\\\\\/lRipX1Qak2a\\\\\\\/gUU8Q\\\\\\\/elBpgIXQi1XdID6Ni2chCbilke8NjXH4IAIyd\\\\\\\/5QA7KbrXLuuZIqvcmrvg4fWtJzUxv1U\\\\\\\/rt1SoGF81DmdK4b7Ttsn7GS0h9JHM3UXOePVFlAeAQPw42mF9YYcv017WH7qUOYoCaxy\\\\\\\/TImmuvdSd\\\\\\\/PZLEtWj3YphHPYXu699L+JwSwwERJN43Ttr+giTj1TK8QGcGnDnnAyCepSB9jvrHKcuMedbfW+CeTQDGP+1a28KSxvCvfhkyB8zYG\\\\\\\/j1KEqQkVPFaS19bnwPSMk7R6rAYH2l1TOCv5xQwayJmNgCVIkBQ0R72QuLVU6vEtERTVCe8ksAemMDp2XpkHyMnkwJSzeaSbd4EtnCC\\\\\\\/0BSiO+YScHwi0bv48Uxinui7cGRGjeof4m14dMmuZHF6ofsBmwTQ7YKDQ5KQ6kQs+xTT7oBiRBJd+Dd+aVaYuPNYsRrpkKR328SIC\\\\\\\/VkgYMGk4h1Me6ogjfS5f5n6FulrTXLhckE+QgAQD8u5e5Zh9Bdrlkh0P7+IWLUAlhXIW4cU4\\\\\\\/H8mV6mZruBTvXK6sifSS2336WMfD+ayR7VnNOdrOWxnToQLfWW\\\\\\\/KsyUPX4wNvag7Pf75WufXXENSvAtVK04RtwUXuqepjmiDytGYdmmVGf1mIpeqpoWOcm79dz8TeDf9SrR76RF6YuGk4WMXxLqiG7pDoTJ9HPSrYzITaAjyGyfILi47vbsbBdZbUZW6D63SKnhcejqVo8lS81V4nOkyt6OtXmycUiErvvel2NLxZplQPMNyGqW9\\\\\\\/o24ToTXAKzl\\\\\\\/IZkQplIz1CAtbyTI5xYZ+TcZ4jyDSGHoxMGz0a4G9+zMi+UKYpoO3NlUHs7NKFY42Gr5Px7OK6sniYTHFGVp3ykroCsApod6QgxM49dRFf2gqA7VnvpCdVSVqQZE+jjCHoj3jeA+2lMp\\\\\\\/+64k41b5sWtgqPzoTIlMn6zKTQhOn+mGFzPb\\\\\\\/YSKncqUyr9v21dy0M0p1mqtz37bo\\\\\\\/f1+dfzsAn\\\\\\\/G\\\\\\\/+gsYvGP3TPU+Ph7VzX66dIsqUVgShG+tuzFL3WlHkwN98VEGnAP8i7nhvjbvnaTcVc3Jx2MEiQSVfU5B+MdEPuj9MS9NMWlVCJJ8W73Z0owRtfrqNmznsVGKBRQa4SCGiZkay\\\\\\\/c0omjkO0VmxSJqae9MGS6\\\\\\\/q3SqizhGdR\\\\\\\/qoY352nKIMUmI49L5CsyWydpyirCSY4iwaG4ezadG60B4yQu6JZddVJb4X3WFTN04cFELpCBXZdXVhHqDjBeo8Re9QyipI4F\\\\\\\/VvvM\\\\\\\/BQcafSa83flTfLdr1Yz7SM\\\\\\\/68MfDogiHkyL8uVgnYVPUAqxLDZ917dpMYRjP71Gb4s4ap6VSojOlqN5A4xVhAVgPronhmzhgQh5TVQQaDuaSRsOc0UY+YL7fc\\\\\\\/tmJ8kp\\\\\\\/bebGjTX2JfUCFUjPj2LoD68QEhXoeqZg1QiY5DIAbrq+gsXZtvacBnuDSbHkK1ezRQ\\\\\\\/PYDXGWa14kWOuh\\\\\\\/7wenLxsP2bm0zDRfLfV\\\\\\\/NKWOHCfaQ=\\\u0022,\\\u0022iv\\\u0022:\\\u0022xVIti3lbGCIjlihmVHucdQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd2027c67a6b6dbc0a45b1bd003b27b68aa1dc30812a98853d95d06b1128dfac\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kYNJeSuqfWTEsaukJlwuQq46qnNqivrOwX+\\\\\\\/eu8UqiJcfbRUDyckqwWs1EGmZBcc2lIfu3bT\\\\\\\/AgDoF8hJFUFths0BeQEH0bZBiZyD7FDvmq5oBZswMY96u8X8UFDxtkUxPZ53Ql\\\\\\\/biMwDObNkxK6kwanJnmUBphtIQVLNGipVyQslPtcUecMa5G27AhvgiCuR0SsXgvjY7PbFuUwL6nYuA7l3mYrDQs1wMKxU4FBXGOjxyEW2rTDS7cdw0unzwOmm7YeK5wRCVk54QFmfehbLkXOM+5GvFHNPL6B+qd\\\\\\\/k+xpdK6z8GedBcrhUfeM0MZycTwGGyWmpsKVPfwUYrBBf5EnOT1lxOzmldsb1eNSrZ8tUR+IzSyakvs13Aaph4a86wyYVfK172K03MTtXVMomKgdK+n25SkZDbKPa\\\\\\\/1YGzJBLzEhi6CRxXObf3ezdGPElzbnpYGHsI4Av1e0A4hVwWSWsMLFZdatr97VMv6gxyz78\\\\\\\/Sa9CkwFuuMJ0JDjz6nCRlqGwoCe0DBwLhsemXG5dvBRdYqZTNupLs58COVJE6tf\\\\\\\/ZiFV7VI\\\\\\\/qKSANGbjJHirk6AaxCu2TQzQSlFFqveqyQyQ8wNqhx84xRBjxEbtTgEGs+2MNFnm7xwzouN7NIB7D8jjw3nszDGX2Cmn50\\\\\\\/ROLQ42WfXdyW6qP74DOqS3XvYF1T\\\\\\\/3\\\\\\\/GLz1bT93PxKxCibc4jCdDYTkEki6x7yYnGwbdjLWsHueRhZ0YGWDDwfWX8xGgrAifD5QZ9xEDEKcellmWKpw2962XF5OrED7DKA9LvZoNcd1BCn0FHeNzMekSn7rvHJ4KiShVSVXB2zPucDi2A6b1leuSEplWbQmJ1WnyFRONIj+x1QDs68hAquxi\\\\\\\/Tqv6PbAkw9+BLqMWt0PA7I0w5DLCJkV7AJnqgYW0UlHBue\\\\\\\/ay3ZOjV4641fLIcHtlUvgoArS+ycudeZkJc41qWFC\\\\\\\/12+tanPUbfhwupwy7UzCQXVHRls2ueZ8Pwc8Z6UnoBF4SdC62aor1JEIiyceVdm9+Qd\\\\\\\/WTMNYoPeTnT8EELzYwevLdDLHKtgCQJ7T2hLjufbtHrMz0cI456MTZvH3HCLSCP8t+bvnfZcDZELNubqn75\\\\\\\/R+FSSHmWocNLfObOmDU15tRl0qqYlSS9nHGUsZRf6fMe55M9p9j\\\\\\\/T2Uv45JFvWStPtiJCAwz6GjlciSPJi33InHZdpEFsIkTroh5sPwLy9+VOV9I+mzoOzJgU0jWKmnbv31ujl+yduLPGaYoqddZenvEbppwpUnP6ArR\\\\\\\/ctLKwfHvXFq0\\\\\\\/X5w6qAfDeYjgUnyTJVWx0AgSm7wlLk6JktA57cMT+bZ8Euwhr45EDe2qWenqWH2Ie0mPS6eCOkReCnPz2F9aD032xJeQZOQwyXZy0\\\\\\\/t8HkwkjlMFCQgNmghRMFzI4si\\\\\\\/OPtnKHeqgA\\\\\\\/aHIuwECw3yPmJ5BgsE4dx\\\\\\\/ne4JEJf9rY9CsGzqQveuBv\\\\\\\/yQxdfsmmlfXivgd\\\\\\\/hje9o8YN9Hoqthes3uYw9OVbCrntOQaZ3R8GfS4iL4kqZJtOssQ6z2lSCmTbjtWCdmj+UPaox0hnyeFVWAu\\\\\\\/AnqRR2ZwvO0kYlJltM9zzrv7jfir2if1AIDc3a5JX7RHto04kO6QN+qmbPcyGO+2KTnvSjDsg+YNcEi5dS4PTyVAEzAqN4xmDKXi6bU4wix\\\\\\\/GglYb1slbNqIaq1soHcnDtwWUwaJ4GkdOrPsjsGxisNqHr0FG7tEJAEcPwnRtdDM63eNigY1KR6r8xtaWyG2XFeUCyYDeYl1VPB2PyM0SSNyLKR2KJO7+QE+CZ1Y9GtzH3zoMsxgSJzxtLqFbD42E1PrHVVUP3AJYpmvuUW+1vSeCXmS48A\\\\\\\/Y\\\\\\\/JTBgiyw592k1RlRg48gyjwc5hEE\\\\\\\/i6ltOjIybVGp7zjmqHB\\\\\\\/6OUW2lb+d62xVIaw0f0wR\\\\\\\/7QAKGiYg4ScQENWYghxEkGF9dnwIYGa\\\\\\\/T0nW+k80I4mSA4TBFm8M4DWkL1esZN2CLmBcK3kxsOS4JzdmhlJx53uk7XRQTdpcC2v6ENSNUiTbha\\\\\\\/VCHMLN9ySa4ADH2GGMZY88P5jT2jd2Xgjbqw0OouEqy0nAwKZF7dTMFur0Sn\\\\\\\/VKpQBsYrvbhYaWRkV5ysg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/u6KpKj\\\\\\\/F9zMiB2bUzvlTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022918a9bb97a3c4c7508b37f6ceb4c769b22a350a6e88d30a20a3b393161ddc567\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459867 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/901" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vuqHApLrm7nyR4ZTuGg4IWLS_Am0Ar9RN-LEThL3xds" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/901" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cygoavOf_jwMLO1oT0F2wEKJdM3XoWe67hs-AqOl3dU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/891" title="Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/913" title="Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459867" id="node459867" data-pisa="ggg;4/5/901" data-pisa-master="ggg;g3.113.007633" data-apath="/ggg/4/5/901.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Suyash Shringarpure</span> and <span class="highwire-citation-author" data-delta="1">Eric P. Xing</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">901-911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007633 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459867" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Suyash Shringarpure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suyash%2BShringarpure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shringarpure%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuyash%2BShringarpure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eric P. Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computer Science, Carnegie Mellon University, Pittsburgh, Pennsylvania 15213</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BP.%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEric%2BP.%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:epxing@cs.cmu.edu" class="" data-icon-position="" data-hide-link-title="0">epxing@cs.cmu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/5/901" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459867" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459867/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/901.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459867" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459867/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/901.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459867" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459867/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/5/901.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459867" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459867/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/5/901.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459867" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459867/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Population stratification is an important task in genetic analyses. It provides information about the ancestry of individuals and can be an important confounder in genome-wide association studies. Public genotyping projects have made a large number of datasets available for study. However, practical constraints dictate that of a geographical/ethnic population, only a small number of individuals are genotyped. The resulting data are a sample from the entire population. If the distribution of sample sizes is not representative of the populations being sampled, the accuracy of population stratification analyses of the data could be affected. We attempt to understand the effect of biased sampling on the accuracy of population structure analysis and individual ancestry recovery. We examined two commonly used methods for analyses of such datasets, ADMIXTURE and EIGENSOFT, and found that the accuracy of recovery of population structure is affected to a large extent by the sample used for analysis and how representative it is of the underlying populations. Using simulated data and real genotype data from cattle, we show that sample selection bias can affect the results of population structure analyses. We develop a mathematical framework for sample selection bias in models for population structure and also proposed a correction for sample selection bias using auxiliary information about the sample. We demonstrate that such a correction is effective in practice using simulated and real data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/admixture-analysis" class="hw-term hw-article-keyword hw-article-keyword-admixture-analysis" rel="nofollow">admixture analysis</a></li><li class="kwd"><a href="/keyword/global-ancestry" class="hw-term hw-article-keyword hw-article-keyword-global-ancestry" rel="nofollow">global ancestry</a></li><li class="kwd"><a href="/keyword/biased-sampling" class="hw-term hw-article-keyword hw-article-keyword-biased-sampling" rel="nofollow">biased sampling</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li></ul><p id="p-4">Population stratification is an important task in genetic analyses. It provides information about the genetic ancestry of individuals and evolutionary history of populations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rosenberg <em>et al.</em> 2002</a>) and can be used to correct for confounding effects in genetic association studies (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Price <em>et al.</em> 2006</a>). A large number of human genetic datasets such as the HAPMAP (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gibbs 2003</a>), Human Genome Diversity Project (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cavalli-Sforza 2005</a>) along with a smaller number from other organisms are available for study. Datasets that sample a number of individuals from a specific region also have been analyzed to look for evidence of population stratification. These datasets contain individuals from geographically and ethnically diverse populations. Due to practical constraints, only a small number of individuals from each population are genotyped, and the resulting data are a sample from the entire population. This often means that the sample selected for analysis is a biased sample from the underlying populations. This problem is also encountered when multiple datasets are combined to detect population structure analysis with better resolution.</p><p id="p-5">We hypothesize that if the distribution of sample sizes is not representative of the populations being sampled, the accuracy of population stratification analyses of the data could be affected because a fundamental assumption of statistical learning algorithms is that the sample available for analysis is representative of the entire population distribution. Although most algorithms are robust to minor violations of this assumption, sampling bias in the case of genetic datasets may be too large for algorithms to accurately recover stratification.</p><p id="p-6">In this work, we develop a mathematical framework for modeling sample selection bias in genotype data. Our experiments on simulated data show that accuracy of population stratification and recovery of individual ancestry are affected to a large extent by the sampling bias in the data collection process. Both likelihood-based methods and eigenanalysis show sensitivity to the effects of sampling bias. We show that sample selection bias can affect population structure analysis of genotype data from cattle. We also propose a mathematical framework to correct for sample selection bias in ancestry inference reduce its effects on ancestry estimates. We show how such a correction can be implemented in practice and demonstrate its effectiveness on simulated and real data.</p><div class="section other1" id="sec-1"><h2 class="">Related work</h2><p id="p-7">We briefly examine methods that can be used for population structure analysis and the factors that affect their accuracy. We also examine related work on addressing the problem of sample selection bias in different contexts.</p><div id="sec-2" class="subsection"><h3>Methods of population structure analysis</h3><p id="p-8">A variety of methods have been developed for detecting population structure. The two main classes of methods used for detecting population structure are model-based methods and eigenanalysis. Model-based methods use an explicit admixture model of how the population sample was formed from its ancestral populations. The STRUCTURE model by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pritchard <em>et al.</em> (2000)</a> was one of the early methods of this class that is commonly used. Extensions to the STRUCTURE method have been proposed to account for other observed evolutionary processes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Falush <em>et al.</em> 2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huelsenbeck and Andolfatto 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Shringarpure and Xing 2009</a>). The frappe method by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tang <em>et al.</em> (2005)</a> and the ADMIXTURE method by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> (2009)</a> are alternative ways of solving the optimization problem underlying the STRUCTURE model. They allow us to efficiently analyze datasets of large size.</p><p id="p-9">The eigenanalysis methods proposed by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Price <em>et al.</em> (2006)</a> and <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Patterson <em>et al.</em> (2006)</a> project genetic data from individuals into a low-dimensional space formed from the eigenvectors of the genetic sample. Since they do not assume a specific model of population evolution, they can be used in a variety of evolutionary scenarios.</p><p id="p-10"><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Engelhardt and Stephens (2010)</a> showed that the model-based approach and the eigenanalysis-based approach to stratification could both be interpreted as different ways of factorizing the genotype matrix of the given data, which suggests that both methods are related despite their apparent differences.</p></div><div id="sec-3" class="subsection"><h3>Factors affecting the accuracy of stratification</h3><p id="p-11">A number of factors are known to affect the accuracy of population stratification and individual ancestry recovery. In one of the early works on model-based methods for population stratification, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Pritchard <em>et al.</em> (2000)</a> showed that the number of loci available for analysis had a significant effect on the recovery of individual ancestry using STRUCTURE. <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kaeuffer <em>et al.</em> (2007)</a> studied the effect of linkage disequilibrium on recovery of population structure using simulated data. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">McVean (2009)</a> suggested an interpretation of the eigenanalysis method that is the basis of the EIGENSOFT method in terms of the coalescence times of individuals. They also examined the performance of eigenanalysis in diverse demographic scenarios. In the following, we discuss the problem of sample selection bias and some related work about the effect of biased sampling on accuracy of population stratification.</p></div><div id="sec-4" class="subsection"><h3>Sample selection bias</h3><p id="p-12">A common assumption of statistical algorithms is that the available sample is representative of the underlying population. In reality, this assumption may not always be true. Sample selection bias is any systematic difference between the sample and the population. It affects the internal validity of an analysis by leading to inaccurate estimation of relationships between variables. It also can affect the external validity of an analysis because the results from a biased sample may not generalize to the population.</p><p id="p-13">The problem of sample selection bias was first widely studied in econometrics, where it appeared as a bias among survey responders. <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Heckman (1979)</a> provided a method of addressing this problem in linear regression models by estimating the probability of an individual being included in the sample. Sample selection bias also has been addressed in the statistics and machine learning literature by attempts to understand its effect on classifiers and how estimation and prediction can be made correctly in the presence of sampling bias (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Vella 1998</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Zadrozny 2004</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Davidson and Zadrozny 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cortes <em>et al.</em> 2008</a>). <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Zadrozny (2004)</a> discusses the properties of learning algorithms and the effect of sample selection bias on their accuracy. It also outlines a possible way of correcting for sample selection bias, provided we know the bias. Sample selection bias is also studied in ecology when one is trying to model species distributions using presence-only data (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Phillips <em>et al.</em> 2009</a>).</p><p id="p-14">In population genetics, sample selection bias could be a serious problem because the estimates of ancestry obtained from stratification analyses often are used to make inferences about the genetic ancestry of the sampled individuals. The inferred individual ancestries also are used as input in correcting for stratification in association studies [for instance, in ADMIXMAP (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hoggart <em>et al.</em> 2003</a>)]. <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Pritchard <em>et al.</em> (2000)</a> suggest that detecting stratification is difficult unless a significant number of unmixed individuals from each ancestral (or pseudo-ancestral) population is present in the sample. This observation was verified by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Tang <em>et al.</em> (2005)</a> through experiments on a small number of simulated datasets. To our knowledge, there has not been a systematic study of the effect of sample selection bias on the accuracy of population structure recovery and individual ancestry recovery.</p><p id="p-15">We propose to study the effect of sample selection bias on the accuracy of population stratification and individual ancestry recovery using a model-based approach (ADMIXTURE) and an eigenanalysis-based approach (EIGENSOFT). Since the analysis of <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">McVean (2009)</a> provides guidelines on the effect of sampling bias on stratification accuracy using eigenanalysis, we will focus our attention mainly on probabilistic models such as ADMIXTURE.</p></div></div><div class="section other2" id="sec-5"><h2 class="">A mathematical framework for sample selection bias</h2><p id="p-16">We will consider the problem of studying genotype data using a probabilistic model. Ordinarily, for probabilistic modeling, we would assume that we have genotypes (<em>g</em>) drawn independently from a distribution <em>D</em> (with domain <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-1.gif"/></noscript></span></span>) over the feature space G. We assume that our points (<em>g</em>, <em>u</em>, <em>s</em>) are drawn independently from a distribution <em>D</em> over <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-2.gif"/></noscript></span></span> × <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-3.gif"/></noscript></span></span> × <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-4.gif"/></noscript></span></span> where <em>G</em> is the space of genotypes, <em>U</em> are some auxiliary features of the data that are not of direct interest for modeling and <em>S</em> is a binary space. The variable <em>s</em> controls the selection of points (1 means the point is selected and is observed in our sample, 0 means the point is not selected). Our observed sample contains only points that have <em>s</em> = 1. We will refer to this as the selected sample and refer to its distribution as <em>D</em>′.</p><p id="p-17">We consider the setting where <em>s</em> is independent of <em>g</em> given <em>u</em>, that is <em>P</em>(<em>s</em>|<em>g</em>, <em>u</em>) = <em>P</em>(<em>s</em>|<em>u</em>). This setting, where the selection is controlled by features different from the genotype we want to model, arises frequently in real applications. In population genetics, whether an individual is included in a genotyping study often depends on factors such as geographical location.</p><div id="sec-6" class="subsection"><h3>Sample selection bias correction</h3><p id="p-18">It is easy to see that if <em>g</em> is independent of <em>u</em> in the previous setting, then sample selection bias has no effect and the probability of <em>g</em> in the selected sample is the same as probability of <em>g</em> under <em>D</em> (asymptotically). If <em>g</em> and <em>u</em> are not independent, then we can write using Bayes rule:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>which can be rewritten as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <em>D</em>′ represents the distribution of the selected sample. Since the term <em>P</em>(<em>s</em> = 1) is constant with respect to (<em>g</em>, <em>u</em>), we can say that<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-8.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where <em>c</em> is a constant that need not be evaluated for tasks such as learning model parameters. Therefore, to model <em>P<sub>D</sub></em>(<em>g</em>, <em>u</em>) accurately to a multiplicative constant, we can follow the procedure below:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-19"><span class="list-label">1. </span>Compute <em>P<sub>D</sub></em><sub>′</sub>(<em>g</em>, <em>u</em>) using a model learned on the selected sample.</p></li><li id="list-item-2"><p id="p-20"><span class="list-label">2. </span>Apply a correction using the term <em>P</em>(<em>s</em> = 1|<em>u</em>). This can be done in two ways:</p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-21"><span class="list-label">(a) </span>If we know the selection procedure, we know <em>P</em>(<em>s</em> = 1|<em>u</em>) and can directly use it in the model.</p></li><li id="list-item-4"><p id="p-22"><span class="list-label">(b) </span>If the selection procedure is not known, but we have access to large number of points for which we know (<em>u</em>, <em>s</em>), but not <em>g</em>, we can estimate <em>P</em>(<em>s</em> = 1|<em>u</em>). In the population genetics example, this would correspond to knowing the language or geographical region of an individual and whether or not they could have been included in the study (since genotyping individuals to find <em>g</em> for a large number of individuals would be expensive).</p></li></ul></li></ul><p id="p-23">However, this analysis, which can accurately correct for sample selection bias in the described setting, requires a model of both <em>g</em> and <em>u</em>. In most applications, we are interested in only modeling <em>g</em> and not <em>u</em>. For instance, although there is interest in modeling the distribution of genotypes, distributions of language or geography are not of direct interest in genetics. Therefore, we consider a similar analysis in the case where we only model <em>P</em>(<em>g</em>) and attempt to derive a correction for sample selection bias.</p></div><div id="sec-7" class="subsection"><h3>Approximate correction</h3><p id="p-24">We consider the case when we only want to model <em>P</em>(<em>g</em>). Proceeding in a similar way as before, we can write<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-9.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>which can be restated as</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-10.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span><p id="p-25">A correction for sample selection bias could therefore be found if we could estimate <em>P</em>(<em>s</em> = 1|<em>g</em>). However, <em>g</em> is typically high-dimensional—in genetics applications, <em>g</em> may have dimensions from 1000 to 1,000,000. <em>P</em>(<em>s</em> = 1|<em>g</em>) is therefore hard to estimate from the small selected sample. We propose that since <em>g</em> and <em>u</em> are dependent and <em>u</em> typically has much lower dimensionality than <em>g</em>, we can approximate <em>P</em>(<em>s</em> = 1|<em>g</em>) by <em>P</em>(<em>s</em> = 1|<em>u</em>). We can therefore write the correction for sample selection bias as</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-11.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span><p id="p-26">with the quality of the approximation varying as a function of the dependence between <em>g</em> and <em>u</em>. In practice, we find that the approximate correction method is adequate for most applications, since probabilistic models are often robust to some differences between the true distribution of the data and the distribution of the selected sample.</p><p id="p-27">It is important to note that even if the selection is determined in reality by the <em>u</em> variables only, the correction proposed in Equation 7 is only an approximate correction. The exact correction would require computing the term <em>P</em>(<em>s</em> = 1|<em>g</em>) which can be written as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-12.gif"/></noscript></span></span> The second term is a distribution conditioned on <em>g</em> and is hard to specify due to the high dimensionality of <em>g</em>.</p><div id="sec-8" class="subsection"><h4>Implementing correction in learning:</h4><p id="p-28">Although applying the proposed correction for accurate probability modeling only requires an extra multiplication step, implementing the correction in learning models consistent with the true distribution is more complex. This problem is well-studied as cost-sensitive learning. A discussion of the ways in which the correction can be applied to classifiers can be found in <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Zadrozny <em>et al.</em> (2003)</a>. In this work, we will use sampling with replacement to implement the correction. To perform sampling with replacement, we sample points in the selected sample (with replacement) with probability proportional to their correction factor (1/<em>p</em>(<em>s</em> = 1|<em>u</em>)). If the selected sample contains <em>N</em> points, the probability of inclusion for the <em>i<sup>th</sup></em> point is given as <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-13.gif"/></noscript></span></span>. Since we sample with replacement, our corrected sample can include non-unique points from the selected sample.</p></div></div></div><div class="section methods" id="sec-9"><h2 class="">Methods</h2><p id="p-29">We will demonstrate the effects of sample selection bias on the accuracy of ancestry recovery by using experiments on simulated and real data. We will also show how the approximate correction for sample selection bias is effective in practice.</p><div id="sec-10" class="subsection"><h3>Simulation experiments</h3><p id="p-30">To examine the recovery of individual ancestry, we simulated data from a two-population admixture scenario using populations from HapMap Phase III as the ancestral populations. We used 50 unrelated individuals each from the CEU (Utah residents with ancestry from northern and western Europe) and YRI (Yoruba in Ibadan, Nigeria) populations as the two ancestral populations. Using a forward simulator (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007633/-/DC1/FileS1.pdf">File S1</a>), we simulated a 50-50 admixture in a single generation followed by six generations of random mating to create a simulated population of 800 diploid individuals. For our experiments, we used a 102-Mb region from chromosome 1 containing 52,040 single-nucleotide polymorphisms (SNPs). The recombination rate was set to be 10<sup>−8</sup> per site per generation and the mutation rate was set to be 10<sup>−8</sup> per site per generation. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the proportion of YRI ancestry among the individuals in the simulated population. From the figure, we can see that most individuals are admixed, and therefore we refer to this simulated population as admixed. For our ancestry inference experiments, we also included the remaining CEU-unrelated individuals (38 in total) and the remaining YRI-unrelated individuals (50 in total) in the analysis as proxies for the ancestral populations. In our experiments, we will refer to the CEU and YRI individuals as unmixed individuals.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of YRI ancestry among individuals in the simulated population, six generations after admixture." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of YRI ancestry among individuals in the simulated population, six generations after admixture.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F1.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/5/901/F1.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464392" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Proportion of YRI ancestry among individuals in the simulated population, six generations after admixture.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We use the proportion of YRI ancestry, denoted as <em>θ<sub>i</sub></em> to represent the ancestry of the <em>i<sup>th</sup></em> individual. The population used for our experiments therefore contained the following three groups:</p><ul class="list-simple " id="list-3"><li id="list-item-5"><p id="p-33"><span class="list-label">1. </span>100 CEU individuals obtained by duplicating (and in some cases triplicating) 38 unrelated CEU individuals, <em>θ<sub>i</sub></em> = 0</p></li><li id="list-item-6"><p id="p-34"><span class="list-label">2. </span>100 YRI individuals obtained by duplicating 50 unrelated YRI individuals, <em>θ<sub>i</sub></em> = 1</p></li><li id="list-item-7"><p id="p-35"><span class="list-label">3. </span>800 admixed individuals, with <em>θ<sub>i</sub></em> ∈ (0, 1)</p></li></ul><p id="p-36">To study the effects of sampling bias, we sample individuals from the dataset with replacement to generate a dataset of size x + y + z, where x is the number of CEU individuals, z is the number of YRI individuals, and y is the number of admixed individuals. By varying x, y, and z, we can generate smaller datasets with different kinds of bias and deviations from the original dataset. We choose x and z from {10,30,50,100} and y from {10,30,100,200,400,700}. Thus, the smallest possible dataset is {10,10,10} (30 individuals), and the largest possible dataset is {100,700,100} (900 individuals). We will use <em>S<sub>xyz</sub></em> to refer to the dataset {<em>x</em>, <em>y</em>, <em>z</em>}.</p><p id="p-37">In this case, <em>g</em> comprises the 52,040-dimensional genotypes of the individuals and <em>u</em> comprises the group memberships of each individual according to their ancestry (<em>u</em> ∈ {1, 2, 3} with the three groups as defined earlier). In general, the probability distribution of <em>u</em> may not be known. Therefore, as a first guess, we will assume that <em>u</em> has a uniform distribution, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-14.gif"/></noscript></span></span> for any value of <em>u</em>. This assumption means that an individual (regardless of inclusion in the study sample) is equally likely to be from the CEU, YRI, or the admixed population. By design, <em>s</em> depends only on <em>u</em>, and we can write the probability distribution P(<em>s</em> = 1|<em>u</em>) as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-15.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-16.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-17.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>which we can simplify to write<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-18.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-19.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-20.gif"/></noscript></span> <span class="disp-formula-label">(13)</span></span>where <em>C</em> is a constant given by <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-21.gif"/></noscript></span></span>.</p></div><div id="sec-11" class="subsection"><h3>Evaluation measure</h3><p id="p-38">A fair evaluation of the results for both ADMIXTURE and EIGENSOFT is difficult to achieve because the individual ancestries produced by ADMIXTURE and EIGENSOFT are different in nature. With <em>K</em> ancestral populations, an individual ancestry vector produced by ADMIXTURE has the form {<em>q</em><sub>1</sub>, …, <em>q<sub>k</sub></em>} such that <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/5/901/embed/mml-math-22.gif"/></noscript></span></span>. Thus, it has only <em>K</em> − 1 independent components. An equivalent representation of ancestry can be produced by projecting an individual on the first <em>K</em> − 1 eigenvectors produced by EIGENSOFT. The ancestry vectors that we store as the true ancestry when generating the simulation data have the same form as those produced by ADMIXTURE.</p><p id="p-39">We use squared correlation between the true ancestry and inferred ancestry as the measure for evaluating the accuracy of ancestry inference. For the dataset <em>S<sub>xyz</sub></em>, let Θ = {<em>θ</em><sub>1</sub>, …, <em>θ<sub>x</sub></em><sub>+</sub><em><sub>y</sub></em><sub>+</sub><em><sub>z</sub></em>} denote the true proportion of YRI ancestry of the individuals. Let <em>Q</em><sub>1</sub> = {<em>q</em><sub>1,1</sub>,…,<em>q</em><sub>1,</sub><em><sub>x</sub></em><sub>+</sub><em><sub>y</sub></em><sub>+</sub><em><sub>z</sub></em>} denote the ancestry proportions in cluster 1 inferred by ADMIXTURE for dataset <em>S<sub>xyz</sub></em> with <em>K</em> = 2, where the second subscript indexes the individuals in the dataset (and therefore we have that <em>Q</em><sub>2</sub> = {<em>q</em><sub>2,1</sub>,…,<em>q</em><sub>2,</sub><em><sub>x</sub></em><sub>+</sub><em><sub>y</sub></em><sub>+</sub><em><sub>z</sub></em>} = {1 − <em>q</em><sub>1,1</sub>, …, 1 − <em>q</em><sub>1,</sub><em><sub>x</sub></em><sub>+</sub><em><sub>y</sub></em><sub>+</sub><em><sub>z</sub></em>}. Then the accuracy metric for ADMIXTURE is <em>correlation</em>(Θ, <em>Q</em><sub>1</sub>)<sup>2</sup> (which is equal to <em>correlation</em>(Θ, <em>Q</em><sub>2</sub>)<sup>2</sup>). The corresponding accuracy measure can be constructed for EIGENSOFT by replacing <em>Q</em><sub>1</sub> by the projections of the individual genotypes on the first eigenvector of the <em>S<sub>xyz</sub></em> genotype matrix. For robustness, we report the mean and standard error of the squared correlation over 30 datasets for each parameter setting.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Sample selection bias in a simulated CEU-YRI admixture</h3><p id="p-40">We examined the effect of biased sampling of individuals by constructing subsets of the whole dataset from the CEU-YRI admixture described earlier and measuring the squared correlation between the true and inferred ancestry (<a href="http://www.g3journal.org/content/suppl/2014/03/14/g3.113.007633.DC1/FileS2.tgz">File S2</a>). <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> shows the results of this analysis with the ADMIXTURE software with six subplots. Within each subplot, the number of admixed individuals in the sample remains constant and the number of unmixed individuals from the two ancestral populations (CEU and YRI) is varied. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> shows the results of an identical analysis of the same datasets with the EIGENSOFT software.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Squared correlation between the true individual ancestry and the individual ancestry inferred using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The different levelplots are drawn for different number of admixed individuals in the dataset. The X and Y axes of the plots are logarithmic in scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Squared correlation between the true individual ancestry and the individual ancestry inferred using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The different levelplots are drawn for different number of admixed individuals in the dataset. The X and Y axes of the plots are logarithmic in scale.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F2.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/5/901/F2.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464260" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Squared correlation between the true individual ancestry and the individual ancestry inferred using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The different levelplots are drawn for different number of admixed individuals in the dataset. The X and Y axes of the plots are logarithmic in scale.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Overall, ADMIXTURE recovered individual ancestry better than EIGENSOFT. Previous work has shown that the accuracy of individual ancestry recovery is a function of the <em>F<sub>ST</sub></em> differentiation between the ancestral populations. The results we obtain may vary for datasets with different <em>F<sub>ST</sub></em> values between the ancestral populations and quantifying this effect will require more study.</p><p id="p-43"><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> shows that the EIGENSOFT results exhibit considerable irregular variation within subplots. This is likely to be a result of our sampling scheme for creating biased datasets, which allows individuals to be present in a dataset more than once. This means that the resulting genotype matrix can have lower rank than expected, resulting in lower accuracy of ancestry inference using EIGENSOFT. This, however, does not cause any difficulties in the ADMIXTURE analysis, as seen by the regular patterns in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a> and would be expected based on the likelihood model underlying ADMIXTURE. We will therefore only use the results in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a> to draw broad conclusions about the behavior of EIGENSOFT and not make specific inferences or recommendations.</p><p id="p-44">In the scenarios in which we have few unmixed individuals from both ancestral populations, <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2, A and B</a> show that the accuracy of individual ancestry recovery drops noticeably. This effect is evident in the subplots with 200, 400, and 700 admixed individuals. In all subplots in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>, the results show high accuracy when we have a large number of unmixed individuals from at least one ancestral population. <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Pritchard <em>et al.</em> (2000)</a> and <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Tang <em>et al.</em> (2005)</a> have previously noted that a significant number of unmixed individuals from each ancestral population is required for accurate recovery of stratification. An initial examination of the results suggests that may be sufficient to have a large number (around 50−100) of unmixed individuals from just one of the two ancestral populations to be able to correctly resolve stratification.</p><p id="p-45">We note that this guideline, which is relevant when the number of admixed individuals is large, does not apply if the dataset contains few admixed individuals and few unmixed individuals. When there are few admixed individuals, both methods perform well (relative to their average performance) even with as few as 10 unmixed individuals in the dataset. Mantel tests reveal that the high correlations obtained with few admixed individuals are statistically significant (<em>P</em> &lt; 10<sup>−3</sup>) in all cases.</p><p id="p-46">To examine the effect of the number of unmixed individuals on the ancestry recovery in more detail, we looked at the subset of the generated subsamples which had the same number of unmixed individuals from both ancestral populations, <em>i.e.</em>, subsets of the form <em>S<sub>xyz</sub></em> where the number of unmixed individuals x∈{10,30,50,100} and the number of admixed individuals y∈{10,30,100,200,400,700}. For each value of <em>x</em> (the number of unmixed individuals from each ancestral population present in the dataset), we observed the effect of varying <em>y</em> (the number of admixed individuals present in the dataset) on the ancestry recovery. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> shows the results for ADMIXTURE, and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> shows the results for EIGENSOFT. When the number of unmixed individuals is large, the methods recover ancestry well, and the number of admixed individuals has no effect on accuracy. However, when the number of unmixed individuals in the sample is small, adding more admixed individuals to the sample reduces the accuracy of the ancestry recovery for both ADMIXTURE and EIGENSOFT. In <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, A and B</a>, we see a threshold effect due to the number of unmixed individuals when the number of unmixed individuals changes from 30 to 100.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of adding more admixed individuals to the dataset on the correlation measure of accuracy when using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The error bars indicate standard error." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of adding more admixed individuals to the dataset on the correlation measure of accuracy when using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The error bars indicate standard error.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F3.medium.gif" width="359" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/5/901/F3.medium.gif" width="359" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464300" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-47" class="first-child">Effect of adding more admixed individuals to the dataset on the correlation measure of accuracy when using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The error bars indicate standard error.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">The high accuracy of ancestry recovery when there are few admixed and few unmixed individuals suggests that earlier hypotheses about the requirement of a large number of unmixed individuals for accurate ancestry recovery may be an incomplete explanation. The results in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, along with the likelihood model underlying ADMIXTURE suggest that the effect on accuracy may depend on the ratio of the number of admixed individuals to unmixed individuals from each population in the sample. For notational convenience, we will refer to this ratio as <em>τ<sub>sample</sub></em> = <em>y</em>/<em>x</em>.</p><p id="p-49">To examine this hypothesis, we replot the data used for <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a> by examining the correlation measure as a function of the ratio of admixed individuals to unmixed individuals in the sample. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a> show the results for this visualization for ADMIXTURE and EIGENSOFT, respectively. From the figure, we can see that the effect of sample selection bias can be understood using <em>τ<sub>sample</sub></em>. The accuracy of ancestry recovery is high while the value of <em>τ<sub>sample</sub></em> is less than 5 (approximately) and drops as this ratio increases. This behavior is independent of the exact number of unmixed individuals in the dataset and can be observed for both ADMIXTURE and EIGENSOFT.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of the ratio of number admixed individuals to unmixed individuals in the dataset (&#x3C4;sample) on the correlation measure of accuracy using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The error bars indicate standard error. The X-axis is logarithmic in scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of the ratio of number admixed individuals to unmixed individuals in the dataset (&lt;em&gt;&#x3C4;&lt;sub&gt;sample&lt;/sub&gt;&lt;/em&gt;) on the correlation measure of accuracy using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The error bars indicate standard error. The X-axis is logarithmic in scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F4.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/5/901/F4.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464298" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-50" class="first-child">Effect of the ratio of number admixed individuals to unmixed individuals in the dataset (<em>τ<sub>sample</sub></em>) on the correlation measure of accuracy using (A) ADMIXTURE with K = 2 and (B) EIGENSOFT with the top eigenvalue. The error bars indicate standard error. The X-axis is logarithmic in scale.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">In our experiments, we observe that individual ancestry can be recovered perfectly as long as <em>τ<sub>sample</sub></em> &lt; 5. Decreasing the number of admixed individuals has no adverse effect on the accuracy of ancestry recovery. The effects of sample selection bias on the accuracy of ancestry recovery in a two-population admixture scenario using ADMIXTURE can thus be explained in two scenarios: (i) when <em>τ<sub>sample</sub></em> &lt; 5, sample selection bias has no effect on the accuracy of individual ancestry recovery and (ii) when <em>τ<sub>sample</sub></em> &gt; 5, the accuracy of individual ancestry measured using the correlation measure decreases with <em>τ<sub>sample</sub></em>.</p><div id="sec-14" class="subsection"><h4>Correction of sample selection bias in simulated data:</h4><p id="p-52">We examined three methods of correcting sample selection bias in the simulated data. The methods and their results are described herein.</p><div id="sec-15" class="subsection"><h5>Supervision:</h5><p id="p-53">ADMIXTURE can be run in supervised mode, where individuals of known ancestry can be specified in advance to belong to exactly one of <em>K</em> populations being inferred. This supervision is partial since only nonadmixed individuals can be part of the supervised input provided to the algorithm. In our experiments, we include supervision by assigning the individuals of known YRI and CEU ancestry to the two ancestral populations. We find that supervision produces an improvement that is statistically significant but practically insignificant (&lt; 0.0001% improvement in squared correlation, <em>P</em> &lt; 2.2 × 10<sup>−16</sup> in a one-sided paired <em>t</em>-test).</p></div><div id="sec-16" class="subsection"><h5>Including more SNPs:</h5><p id="p-54">Our analysis of the CEU-YRI admixture used 52,040 SNPs in a 102-Mb region. We found that including more SNPs in the ADMIXTURE analysis reduced the effects of sample selection bias for identical sample sizes and samples. When 116,415 SNPs are included in the ADMIXTURE analysis, we observe no effects of sample selection bias on the accuracy of ancestry inference. Although we do not thin SNPs in our analysis, thinning the original set of 52,040 SNPs by an <em>r</em><sup>2</sup> threshold of 0.1 produces a set of 35,500 SNPs. Reanalyzing the data with this subset of SNPs produces a small loss in accuracy (&lt;0.5% in squared correlation, <em>P</em> &lt; 2.2 × 10<sup>−16</sup> in a one-sided paired <em>t</em>-test) when compared with the original set of 52,040 SNPs. Thus, while adding independent SNPs improves accuracy of ancestry inference, adding or removing linked SNPs has little effect on accuracy.</p></div><div id="sec-17" class="subsection"><h5>Resampling correction:</h5><p id="p-55">We implemented the resampling correction in the section <em>Approximate correction</em> by using Equations 11−13. Since we do not have an informative prior expectation of the true numbers of admixed individuals or the unmixed individuals, we used the uninformative initial estimate of all groups of individuals having equal population sizes. We found that in more than 99% of the corrected datasets, the squared correlation between true and inferred ancestry was larger than 0.95 (mean = 0.98, standard deviation = 0.01). Thus the resampling correction is effective for correcting sample selection bias.</p></div></div></div><div id="sec-18" class="subsection"><h3>Impact of effective population size on sample selection bias</h3><p id="p-56">In the previous simulation, we used a correction based on an assumption of similar population sizes for the different groups. In general, the census (observed) population size and the effective population size of a population need not be equal due to demographic events that the population may have undergone. We attempt to demonstrate that the effect of biased sampling is dependent on the effective population size and not only on the census population size.</p><p id="p-57">We simulated two populations according to the demographic scenario shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> using the coalescent simulator <em>ms</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hudson 2002</a>). The founding population and the populations resulting from the split 1000 generations before present were set to have identical sizes (<em>N</em><sub>0</sub> = 10,000). Population 2 then undergoes an instantaneous contraction 795 generations after the split, with its population size being reduced to <em>αN</em><sub>0</sub> The contraction factor <em>α</em> was varied across datasets, taking values of 0.99 (nearly no contraction), 0.5 (moderate contraction), or 0.1 (strong contraction). Two-hundred generations after the contraction, population 2 instantaneously grows back in size to <em>N</em><sub>0</sub>. The simulation was continued for five generations after the recovery, after which 50 diploid individuals were sampled from each population. Each simulation used a mutation rate and recombination rate of 10<sup>−8</sup> per bp per generation. In each simulation, 200 independent sites of 100 kb were generated, producing between 47,585 and 58,024 SNPs per simulation. For each value of <em>α</em> (from {0.1,0.5,0.99}), 30 independent datasets were generated to evaluate the robustness of results (<a href="http://www.g3journal.org/content/suppl/2014/03/14/g3.113.007633.DC1/FileS3.tgz">File S3</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A two-population demographic scenario: Populations 1 and 2 (of size N0 = 104) are formed from a single population of the same size 1000 generations before present time. Population 2 then undergoes a bottleneck for 200 generations followed by a instantaneous recovery to its original population size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A two-population demographic scenario: Populations 1 and 2 (of size &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; = 10&lt;sup&gt;4&lt;/sup&gt;) are formed from a single population of the same size 1000 generations before present time. Population 2 then undergoes a bottleneck for 200 generations followed by a instantaneous recovery to its original population size.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F5.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/5/901/F5.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464367" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-58" class="first-child">A two-population demographic scenario: Populations 1 and 2 (of size <em>N</em><sub>0</sub> = 10<sup>4</sup>) are formed from a single population of the same size 1000 generations before present time. Population 2 then undergoes a bottleneck for 200 generations followed by a instantaneous recovery to its original population size.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">The two populations from a single simulation were pooled together to form a 50:50 admixture in a single generation to create 50 admixed individuals. This was followed by five generations of random mating in the resulting admixed population, using the simulator described in the <em>Introduction</em> section. We then generated a sample for study by including the 50 individuals from population 1, 50 admixed individuals, and a variable number of individuals (<em>x</em>) from population 2 (<em>x</em> ∈ {5, 10, 20, 30, 40, 50}). ADMIXTURE was used with <em>K</em> = 2 to infer the ancestry proportions for each individual and the squared correlation coefficient (as described in the section <em>Evaluation measure</em>) for the dataset was used a measure of accuracy.</p><p id="p-60"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the results of the ancestry inference, with one curve for each value of the contraction factor <em>α</em>. As expected from the previous experiments, as the number of individuals from population 2 in the dataset increases, the accuracy of ancestry inference increases, which is demonstrated by each curve. We also observe that the curves for the three different contraction factors have different accuracies before they converge to a accuracy of almost 1.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Impact of effective population size on the correlation measure of accuracy of ancestry inference using ADMIXTURE with K = 2. Each curve is drawn for a single value of the contraction factor &#x3B1; and shows the mean and standard error of the correlation measure over 30 datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Impact of effective population size on the correlation measure of accuracy of ancestry inference using ADMIXTURE with K = 2. Each curve is drawn for a single value of the contraction factor &lt;em&gt;&#x3B1;&lt;/em&gt; and shows the mean and standard error of the correlation measure over 30 datasets.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F6.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/5/901/F6.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464381" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-61" class="first-child">Impact of effective population size on the correlation measure of accuracy of ancestry inference using ADMIXTURE with K = 2. Each curve is drawn for a single value of the contraction factor <em>α</em> and shows the mean and standard error of the correlation measure over 30 datasets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">By our experimental design, the two ancestral populations have identical census population sizes for each simulation and the census size for population 2 is identical across simulations. However, due to the bottleneck, the effective population size of population 2 is reduced, with the amount of reduction dependent to the strength of the contraction (the calculations for reduction in effective population size for population 2 can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007633/-/DC1/FileS1.pdf">File S1</a>). <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> thus demonstrates that effective population size, and not just census population size, has an impact on how ancestry inference is affected by sampling bias. From the figure, we can see that for the strongest contraction (<em>α</em> = 0.1, producing the maximum reduction in the effective population size for population 2 to 0.36<em>N</em><sub>0</sub>), the accuracy of ancestry inference is high even with only 5 individuals from population 2. Since effective population size is a measure of the genetic diversity of a population, this suggests that the reduced genetic diversity produced by the strong contraction can be adequately captured even with a small number of individuals. For the same number of individuals from population 2, across the curves, we see that the accuracy is smallest for the weakest contraction (<em>α</em> = 0.99) and largest for the strongest contraction (<em>α</em> = 0.1). Thus, even when census population sizes are identical, variation in effective population size affects the impact of sampling bias accuracy of ancestry inference. This suggests that sampling bias can be more accurately captured by defining it in terms of effective population size rather than census population size.</p><div id="sec-19" class="subsection"><h4>Correction of bias:</h4><p id="p-63">Using the estimate for the effective population size of population 2 from <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007633/-/DC1/FileS1.pdf">File S1</a>, we can implement a resampling correction as before. We assume that the effective population size for the admixed population is identical to that for population 1, <em>i.e.</em>, <em>N</em><sub>0</sub> = 10<sup>4</sup>. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows the change in accuracy after application of the resampling correction (relative to the accuracies in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Change in correlation measure from Figure 6 after the resampling correction, taking into account the impact of the contraction on effective population size. Each curve is drawn for a single value of the contraction factor &#x3B1; and shows the mean and standard error of the change in correlation measure over 30 datasets. The regions of the curves in bold show regions where accuracy in the uncorrected results (from Figure 6) is less than 0.95." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Change in correlation measure from Figure 6 after the resampling correction, taking into account the impact of the contraction on effective population size. Each curve is drawn for a single value of the contraction factor &lt;em&gt;&#x3B1;&lt;/em&gt; and shows the mean and standard error of the change in correlation measure over 30 datasets. The regions of the curves in bold show regions where accuracy in the uncorrected results (from Figure 6) is less than 0.95.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F7.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/5/901/F7.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-64" class="first-child">Change in correlation measure from <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> after the resampling correction, taking into account the impact of the contraction on effective population size. Each curve is drawn for a single value of the contraction factor <em>α</em> and shows the mean and standard error of the change in correlation measure over 30 datasets. The regions of the curves in bold show regions where accuracy in the uncorrected results (from <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>) is less than 0.95.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">From the figure, we can see that while the accuracy for the original dataset is less than 0.95, the resampling correction produces an improvement in the results, denoted by the portions of the curves in bold. However, beyond those points, as the datasets acquire an excess of the unmixed individuals relative to the admixed individuals, we see that the uncorrected accuracy is high (larger than 0.95) and the resampling correction decreases the accuracy of ancestry inference. Thus a resampling correction is unnecessary when there is an excess of unmixed individuals relative to their expected number from effective population size calculations.</p></div></div><div id="sec-20" class="subsection"><h3>Sample selection bias in cattle genotype data</h3><p id="p-66">To demonstrate the effects of sample selection bias on a real genetic dataset, we used the data from (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">McTavish <em>et al.</em>, 2013a</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">b</a>), who genotyped 1461 individuals belonging to cattle breeds across the world at 47,506 SNPs. We focus on admixture between African and European cattle breeds as seen in their New World descendants. For their experiments with STRUCTURE, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">McTavish <em>et al.</em> (2013b)</a> used a subset of 1814 SNPs that was common to all their genotyping chips. Our data therefore consists of genotype data at 1814 SNPs from 40 New World individuals (Texas Longhorns), 100 European individuals (Limousin from Southern Europe), and 100 African individuals (N’Dama and N’DamaXBoran).</p><p id="p-67">We created multiple subsets from the original data by including all the 40 admixed New World individuals and a variable number (<em>x</em>) of unmixed individuals from the European and African populations (with the same number from each population). The number of unmixed individuals chosen was increased gradually from 10 to 100 (<em>x</em> ∈ {10, 20, 30, 40, 50, 60, 80, 100}). For each value of <em>x</em>, we generated 30 datasets for statistical robustness. ADMIXTURE was used with <em>K</em> = 2 to infer ancestry proportions for all individuals. The squared correlation coefficient for the ancestry proportions of the admixed individuals was used as a measure of accuracy, with the ancestry proportions inferred on the full data used as a truth set.</p><p id="p-68"><a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> shows the results of the ancestry inference as a function of the number of unmixed individuals (<em>x</em>). We see that as the number of unmixed individuals increases, the accuracy of ancestry inference improves. When the number of unmixed individuals is less than 40, the squared correlation measure is less than 0.9.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/5/901/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in accuracy of ancestry inference with changing number of unmixed individuals. The red curve shows accuracy without correction and the blue curve shows the accuracy when the correction is applied." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678842993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variation in accuracy of ancestry inference with changing number of unmixed individuals. The red curve shows accuracy without correction and the blue curve shows the accuracy when the correction is applied.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/5/901/F8.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/5/901/F8.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/5/901/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/5/901/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464387" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-69" class="first-child">Variation in accuracy of ancestry inference with changing number of unmixed individuals. The red curve shows accuracy without correction and the blue curve shows the accuracy when the correction is applied.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-21" class="subsection"><h4>Correction of bias:</h4><p id="p-70">For the cattle data, effective population sizes for the three populations are unknown. We assume that all three populations have equal effective population size. The correction is implemented using resampling as before. The blue curve in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a> shows the effect of the correction on accuracy. From the figure, we can see that when the number of admixed individuals is larger than the number of unmixed individuals (<em>τ<sub>sample</sub></em> ≥ 1 as defined previously), the correction improves accuracy but causes a small drop in accuracy when there are many unmixed individuals.</p></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-71">Biased sampling of individuals is a result of practical constraints on the study design of genotyping/sequencing projects. In this aspect, sample selection bias is similar to bias due to SNP ascertainment. It can also occur as a result of combining multiple genomic datasets from different sources. For instance, the overlap of three genotyping chips required <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">McTavish <em>et al.</em> (2013b)</a> to use only 1814 SNPs for their STRUCTURE analysis.</p><p id="p-72">In most stratification analyses, the recovered ancestry proportions are used to make inferences about the ancestry of the sample and the genetic relationships of similarity/differences between the study populations. Ancestry proportion estimates are also used in association studies to account for effects of stratification. Therefore, it is essential to have accurate recovery of ancestry. Our experiments suggest that sample selection bias can be a problem in accurate population stratification and recovering individual ancestry. Our simulations show that unlike observations from previous studies, the accuracy of individual ancestry recovery is not dependent only on the number of unmixed individuals present in the sample. We observe a threshold effect where the accuracy of ancestry inference is affected by sample selection bias depending on the ratio of admixed individuals to unmixed individuals in the sample. Our simulations suggest that the accuracy of ancestry inference is affected when this ratio, <em>τ<sub>sample</sub></em>, is less than 5. For real genotype data from cattle, we observed that a ratio of <em>τ<sub>sample</sub></em> larger than 1 was sufficient to affect ancestry estimates.</p><p id="p-73">Our experiments do not directly demonstrate an effect of <em>F<sub>ST</sub></em> on sample selection bias. However, using the 1814 SNPs for the genotype data, we were able to observe effects of sample selection bias on African-European admixture in cattle (<em>F<sub>ST</sub></em> ≈ 0.15) but unable to observe any effect of sample selection bias on European-Indian admixture (<em>F<sub>ST</sub></em> ≈ 0.28, results not shown here). This suggests that <em>F<sub>ST</sub></em> has an effect on sample selection bias, with well-differentiated populations being easy to separate even with biased sampling. The simulations and real data analysis had relatively large <em>F<sub>ST</sub></em> between the two ancestral populations (≈ 0.1 for the simulations, ≈ 0.15 for the cattle genotype data) but vastly different number of SNPs (52,040 for the simulation but only 1814 for the cattle genotype data). The degradation in performance of ancestry inference in the two cases was considerably different, suggesting that the effect of sample selection bias also depends on the number of loci available. This was also validated by the observation that doubling the number of SNPs in simulation eliminated the effects of sample selection bias completely.</p><p id="p-74">We also showed through simulations that sample selection bias depends on the effective population size rather than simply the census population size of the populations studied. This suggests that sampling for genotyping/sequencing projects can be improved (in terms of diversity of sampling and cost) by taking into account independent knowledge of the demographic history of the populations being sampled.</p><p id="p-75">Although our analyses use two specific methods (ADMIXTURE and EIGENSOFT), the effects we observe are a feature of the assumptions underlying both methods rather than the specific implementations. ADMIXTURE is a representative of the likelihood-based models that assume (a) admixture between ancestral populations and (b) that modern individual genomes are mixture of contributions from different ancestral populations. This is the model underlying STRUCTURE and frappe and to a large extent the extensions mentioned earlier. Likelihood-based methods are susceptible to effects of sample selection bias since each individual is given equal weight in the sample a priori. This is a result of the fundamental assumption of learning methods that the sample observed is representative of the underlying population distribution. Eigenanalysis, which also weighs each individual equally apriori, also suffers from a similar problem as the likelihood-based method. The sensitivity of eigenanalysis to sample size variation and outliers is well-known and is also reported by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">McVean (2009)</a> and <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Novembre and Stephens (2008)</a>.</p><p id="p-76">ADMIXTURE and EIGENSOFT are unsupervised methods for performing ancestry inference, <em>i.e.</em>, they do not leverage information about individuals of known ancestry to estimate their respective model parameters. ADMIXTURE supports a mode (which is referred to as a supervised mode in the documentation) in which some individuals can be “labeled”, <em>i.e.</em>, assigned to have ancestry from only a single cluster. However, all the individuals, labeled or otherwise, are used to learn the allele frequency parameters of the ADMIXTURE model. In machine learning literature, this mode of learning is commonly called “semi-supervised” learning, wherein a combination of labeled and unlabeled data is used to learn model parameters. In simulations, we observed that this mode of ancestry inference is also affected by sample selection bias.</p><p id="p-77">A different class of methods can estimate local (subcontinental) ancestry from dense genotype data (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Sankararaman <em>et al.</em> 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Price <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Wall <em>et al.</em> 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baran <em>et al.</em> 2012</a>). A common theme of all of these methods is the use of external sources of genotype data as “reference panels” (referred to as “training data” in machine learning literature), which are used to estimate ancestral allele frequencies or ancestral haplotypes. The estimated parameters (frequencies or haplotypes) are then used to infer the ancestry of the individuals in the study sample based on their genotypes (referred to as “test data” in machine learning literature). Therefore, these methods can be said to be performing supervised learning. Supervised learning using external sources of data for estimating model parameters is robust to biased sampling in the study sample but is sensitive to the reference panels used for training (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Zadrozny <em>et al.</em> 2003</a>).</p><p id="p-78">We proposed a resampling correction for sample selection bias using a mathematical framework modeling the sampling process. The proposed correction requires knowledge of some auxiliary information about the selection criteria that is correlated with the genotypes (bias is present only if selection is dependent on the genotypes, directly or indirectly). For genetic datasets, geography provides one such criterion that is easy to acquire during data collection. Using this information, we proposed a correction that is easy to implement. We showed using simulation experiments that such a correction is effective in practice and leads to more accurate results. In terms of computational effort, ADMIXTURE has a linear dependence on the number of individuals. The resampling correction will therefore require time proportional to the number of individuals in the resampled set. However, a better implementation of the resampling correction for ADMIXTURE would be to be weigh the likelihood contribution from each individual in the original sample by its resampling correction factor (1/<em>p</em>(<em>s</em> = 1|<em>u</em>)). This would allow the correction to be implemented without any increase in computational complexity of the ADMIXTURE algorithm. The resampling correction implemented naively can cause problems for EIGENSOFT due to the possibility of colinearity resulting from duplication of individuals, as evident in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>. A weighted eigenanalysis would avoid this problem without affecting the efficiency of the analysis.</p><p id="p-79">An alternative method for correcting sample selection bias is to add more markers to the analysis, as observed from the simulation. Adding more SNPs to the analysis is an appealing solution because it does not require any further assumptions that any statistical correction may require, with the exception of the assumption of no linkage between SNPs. However, this may not always be possible if the study dataset is constructed by intersecting multiple datasets containing the individuals of interest genotyped on different platforms.</p><p id="p-80">Our simulation experiments used simple two-population admixture scenarios to examine the effects of sample selection bias on the accuracy of stratification. In one set of experiments, we used two populations (CEU and YRI) that were easily separable and examined the admixed population resulting after a single recent admixture event. In reality, the demographic processes underlying the evolution of populations are much more complicated. In such scenarios, it is reasonable to expect that the stratification problem may be harder to resolve and would suffer from the effects of sample selection bias more severely.</p><p id="p-81">A limitation of our proposed correction is the requirement for external knowledge of the selection criteria for samples. In our experiments, we either knew the nature of the bias (by design in the simulation experiments) or assumed that it was known. In general, the accuracy of the correction method proposed will strongly depend on the relationship between the selection criteria and the genotypes. Corrections factors therefore may be dataset-specific. An alternative future direction for correcting sample selection bias would be to develop models of population structure that can also model the auxiliary factors such as geography or language that may determine the selection process responsible for the biased sampling. There may be occasions where the nature of the sampling bias is partially known without having sufficient information to allow a correction to be developed. For instance, analyses of the POPRES dataset (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Nelson <em>et al.</em> 2008</a>) often select individuals who are known to have all four grandparents of the same ancestry (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Novembre <em>et al.</em> 2008</a>). In this case, the criteria for selection can be stated explicitly but a mathematical formulation of the bias is hard to obtain. In such scenarios, we recommend subsampling/resampling datasets to examine the robustness of ancestry inference results to variation in sampling.</p><p id="p-82">In summary, our experiments suggest that sample selection bias affects ancestry inference and depends on factors such as effective population size, genetic differentiation between populations, number of loci available, and the ratio of admixed samples to unmixed samples. For analyses of human genotype data, hundreds of thousands of SNPs are typically available. Therefore, we do not expect sample selection bias to be a problem for ancestry inference in human data except when the source populations for admixture are not well-differentiated. Local ancestry methods which can make use of external genotype data also provide a way of avoiding the effects of sample selection bias. We expect that sample selection bias will be of interest in ancestry analysis for model organisms, where availability of external genotype data are limited and most genotyping chips have relatively fewer loci. When the sampling process is relatively well-understood, a mathematical correction as we have proposed will help to reduce the effects of sample selection bias. In our experiments, a rough guideline of using approximately identical number of admixed and unmixed individuals reduces the effects of biased sampling. When the sampling process cannot be modeled accurately, examining the robustness of ancestry estimates to sampling may help avoid inaccurate ancestry estimation.</p></div><div class="section" id="sec-23"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-83">This research was supported in part by NSF Grant DBI-0546594 and NIH Grant R01GM087694.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007633/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007633/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-84"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-05-18"><span class="received-label">Received </span>May 18, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-03-10"><span class="accepted-label">Accepted </span>March 10, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Shringarpure and Xing</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.2" data-doi="10.1093/bioinformatics/bts144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate inference of local ancestry in Latino populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1359</span>–<span class="cit-lpage">1367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts144%26rft_id%253Dinfo%253Apmid%252F22495753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTAvMTM1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.3" data-doi="10.1038/nrg1596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The Human Genome Diversity Project: past, present and future.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCavalli-Sforza%26rft.auinit1%253DL.%2BL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D333%26rft.epage%253D340%26rft.atitle%253DThe%2BHuman%2BGenome%2BDiversity%2BProject%253A%2Bpast%252C%2Bpresent%2Band%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1596%26rft_id%253Dinfo%253Apmid%252F15803201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15803201&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228093000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rostamizadeh</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sample selection bias correction theory.</span> <abbr class="cit-jnl-abbrev">Algorithmic Learning Theory</abbr> <span class="cit-vol">5254</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlgorithmic%2BLearning%2BTheory%26rft.volume%253D5254%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.5.901.5"><div class="cit-metadata unstructured">Davidson, I., and B. Zadrozny, 2005 An improved categorization of classifiers sensitivity on sample selection bias, pp. 605–608 in <em>Fifth IEEE International Conference on Data Mining ICDM05</em>. Available at: <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=%26arnumber=1565737%26queryText%3DAn+improved+categorization+of+classifiers+sensitivity+on+sample+selection+bias">http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&amp;arnumber=1565737&amp;queryText%3DAn+improved+categorization+of+classifiers+sensitivity+on+sample+selection+bias</a>. Accessed March 26, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of population structure: a unifying framework and novel methods based on sparse factor analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvOTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.8" data-doi="10.1038/nature02168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The International HapMap Project.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D426%26rft.issue%253D6968%26rft.spage%253D789%26rft.epage%253D796%26rft.atitle%253DThe%2BInternational%2BHapMap%2BProject.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02168%26rft_id%253Dinfo%253Apmid%252F14685227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14685227&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187342000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.9" data-doi="10.2307/1912352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heckman</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Sample selection bias as a specification error.</span> <abbr class="cit-jnl-abbrev">Econometrica</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEconometrica%26rft.volume%253D47%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1912352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1912352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GH66400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.10" data-doi="10.1086/375613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoggart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriver</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonilla</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittles</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Control of confounding of genetic associations in stratified populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHoggart%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D72%26rft.issue%253D6%26rft.spage%253D1492%26rft.epage%253D1504%26rft.atitle%253DControl%2Bof%2Bconfounding%2Bof%2Bgenetic%2Bassociations%2Bin%2Bstratified%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F375613%26rft_id%253Dinfo%253Apmid%252F12817591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/375613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12817591&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183275500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.11" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a Wright-Fisher neutral model of genetic variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOC8yLzMzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.12" data-doi="10.1534/genetics.106.061317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inference of population structure under a Dirichlet process prior.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1787</span>–<span class="cit-lpage">1802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061317%26rft_id%253Dinfo%253Apmid%252F17237522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE3ODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzUvOTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.13" data-doi="10.1038/sj.hdy.6801010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaeuffer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Réale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontier</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Detecting population structure using STRUCTURE software: effect of background linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DKaeuffer%26rft.auinit1%253DR.%26rft.volume%253D99%26rft.issue%253D4%26rft.spage%253D374%26rft.epage%253D380%26rft.atitle%253DDetecting%2Bpopulation%2Bstructure%2Busing%2BSTRUCTURE%2Bsoftware%253A%2Beffect%2Bof%2Bbackground%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6801010%26rft_id%253Dinfo%253Apmid%252F17622269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6801010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17622269&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249720900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-4.5.901.14"><div class="cit-metadata unstructured">McTavish, E. J., J. E. Decker, R. D. Schnabel, J. F. Taylor, and D. M. Hillis, 2013a Data from: New World cattle show ancestry from multiple independent domestication events. Available at: <a href="http://datadryad.org/resource/doi:10.5061/dryad.42tr0">http://datadryad.org/resource/doi:10.5061/dryad.42tr0</a>. Accessed March 26, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.15" data-doi="10.1073/pnas.1303367110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McTavish</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillis</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">New World cattle show ancestry from multiple independent domestication events.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E1398</span>–<span class="cit-lpage">E1406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1303367110%26rft_id%253Dinfo%253Apmid%252F23530234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzE1L0UxMzk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.16" data-doi="10.1371/journal.pgen.1000686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genealogical interpretation of principal components analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000686%26rft.epage%253De1000686%26rft.atitle%253DA%2Bgenealogical%2Binterpretation%2Bof%2Bprincipal%2Bcomponents%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000686%26rft_id%253Dinfo%253Apmid%252F19834557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834557&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.17" data-doi="10.1016/j.ajhg.2008.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DNelson%26rft.auinit1%253DM.%2BR.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D347%26rft.epage%253D358%26rft.atitle%253DThe%2BPopulation%2BReference%2BSample%252C%2BPOPRES%253A%2Ba%2Bresource%2Bfor%2Bpopulation%252C%2Bdisease%252C%2Band%2Bpharmacological%2Bgenetics%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.005%26rft_id%253Dinfo%253Apmid%252F18760391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760391&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.18" data-doi="10.1038/ng.139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Interpreting principal component analyses of spatial population genetic variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">646</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D646%26rft.epage%253D649%26rft.atitle%253DInterpreting%2Bprincipal%2Bcomponent%2Banalyses%2Bof%2Bspatial%2Bpopulation%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.139%26rft_id%253Dinfo%253Apmid%252F18425127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18425127&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.19" data-doi="10.1038/nature07331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutalik</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genes mirror geography within Europe.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%26rft.volume%253D456%26rft.issue%253D7218%26rft.spage%253D98%26rft.epage%253D101%26rft.atitle%253DGenes%2Bmirror%2Bgeography%2Bwithin%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07331%26rft_id%253Dinfo%253Apmid%252F18758442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18758442&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260674000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.20" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.21" data-doi="10.1890/07-2153.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudík</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sample selection bias and presence-only distribution models: implications for background and pseudo-absence data.</span> <abbr class="cit-jnl-abbrev">Ecol. Appl.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BAppl.%26rft.volume%253D19%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1890%252F07-2153.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/07-2153.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263516200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.22" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.23" data-doi="10.1371/journal.pgen.1000519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafaels</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000519%26rft.epage%253De1000519%26rft.atitle%253DSensitive%2Bdetection%2Bof%2Bchromosomal%2Bsegments%2Bof%2Bdistinct%2Bancestry%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000519%26rft_id%253Dinfo%253Apmid%252F19543370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543370&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.24" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure from multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.25" data-doi="10.1126/science.1078311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic Structure of Human Populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">2381</span>–<span class="cit-lpage">2385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D298%26rft.issue%253D5602%26rft.spage%253D2381%26rft.epage%253D2385%26rft.atitle%253DGenetic%2BStructure%2Bof%2BHuman%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1078311%26rft_id%253Dinfo%253Apmid%252F12493913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTYwMi8yMzgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.26" data-doi="10.1101/gr.072751.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">On the inference of ancestries in admixed populations.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">668</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.072751.107%26rft_id%253Dinfo%253Apmid%252F18353809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzQvNjY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.27" data-doi="10.1534/genetics.108.100222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shringarpure</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">mStruct: Inference of Population Structure in Light of Both Genetic Admixing and Allele Mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100222%26rft_id%253Dinfo%253Apmid%252F19363128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzIvNTc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC81LzkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.28" data-doi="10.1002/gepi.20064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risch</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of individual admixture: analytical and study design considerations.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DEstimation%2Bof%2Bindividual%2Badmixture%253A%2Banalytical%2Band%2Bstudy%2Bdesign%2Bconsiderations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20064%26rft_id%253Dinfo%253Apmid%252F15712363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15712363&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F5%2F901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228573700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.29" data-doi="10.2307/146317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vella</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Estimating models with sample selection bias: a survey.</span> <abbr class="cit-jnl-abbrev">J. Hum. Resour.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHum.%2BResour.%26rft.volume%253D33%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.2307%252F146317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/146317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000072285700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.5.901.30" data-doi="10.1093/molbev/msr049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic variation in Native Americans, inferred from Latino SNP and resequencing data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2231</span>–<span class="cit-lpage">2237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr049%26rft_id%253Dinfo%253Apmid%252F21368315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzgvMjIzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNS85MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-other" id="cit-4.5.901.31"><div class="cit-metadata unstructured">Zadrozny, B. 2004 Learning and evaluating classifiers under sample selection bias, pp. 114 in <em>Twenty-First International Conference on Machine Learning ICML 04</em>. ACM, New York, NY.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.5.901.32"><div class="cit-metadata unstructured">Zadrozny, B., J. Langford, and N. Abe, 2003 Cost-sensitive learning by cost-proportionate example weighting, pp. 435–442 in <em>Third IEEE International Conference on Data Mining</em>, <em>ICDM 2003</em>, Available at: <a href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=%26arnumber=1250950%26url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D1250950">http://ieeexplore.ieee.org/xpl/login.jsp?tp=&amp;arnumber=1250950&amp;url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D1250950</a>. Accessed March 27, 2014.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/5/901.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/5/891" title="Genetic Diversity Analysis of Highly Incomplete SNP Genotype Data with Imputations: An Empirical Assessment" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/5/913" title="Genetic Linkage Mapping and Transmission Ratio Distortion in a Three-Generation Four-Founder Population of Panicum virgatum (L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/5"><span>Volume 4 Issue 5, May 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/5.cover-source.jpg?itok=AAKWoOtu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/5/901" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U6xW3a5mC_WpKP_X4mSls0zf3zNO0hFlFGtGVLfeDoI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/5/901" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/5/901" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/5/901" class="active" data-icon-position="" data-hide-link-title="0">Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459867" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CO2qSoTWP0U724gxY0Xu4tXAW9q-u7BpAMicnInEV5A" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/901.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/5/901" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kW0qn7clOQmLacGJqgfVqZ46Jm_rCPN-a6tJR_OATRw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/5/901" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/5/901.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459867" id="node459867--2" data-pisa="ggg;4/5/901" data-pisa-master="ggg;g3.113.007633" data-apath="/ggg/4/5/901.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Suyash Shringarpure</span> and <span class="highwire-citation-author" data-delta="1">Eric P. Xing</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">901-911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007633 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459867--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Suyash Shringarpure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suyash%2BShringarpure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shringarpure%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuyash%2BShringarpure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eric P. Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computer Science, Carnegie Mellon University, Pittsburgh, Pennsylvania 15213</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BP.%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEric%2BP.%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:epxing@cs.cmu.edu" class="" data-icon-position="" data-hide-link-title="0">epxing@cs.cmu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/5/901" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/5/901&amp;title=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/5/901&amp;title=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/5/901&amp;title=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/5/901&amp;text=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/5/901&amp;title=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/5/901&amp;t=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/5/901&amp;title=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/5/901&amp;title=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459867/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459867" id="node459867--3" data-pisa="ggg;4/5/901" data-pisa-master="ggg;g3.113.007633" data-apath="/ggg/4/5/901.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effects of Sample Selection Bias on the Accuracy of Population Structure and Ancestry Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Suyash Shringarpure</span> and <span class="highwire-citation-author" data-delta="1">Eric P. Xing</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">901-911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007633 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459867--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Suyash Shringarpure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suyash%2BShringarpure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shringarpure%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuyash%2BShringarpure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eric P. Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Computer Science, Carnegie Mellon University, Pittsburgh, Pennsylvania 15213</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BP.%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEric%2BP.%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:epxing@cs.cmu.edu" class="" data-icon-position="" data-hide-link-title="0">epxing@cs.cmu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=901&amp;rft.epage=911&amp;rft.atitle=Effects%20of%20Sample%20Selection%20Bias%20on%20the%20Accuracy%20of%20Population%20Structure%20and%20Ancestry%20Inference&amp;rft.volume=4&amp;rft.issue=5&amp;rft.date=2014-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shringarpure%2C+Suyash&amp;rft.au=Xing%2C+Eric+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459867/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459867/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459867/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459867/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459867/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459867/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459867/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459867/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459867/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459867/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459867/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459867/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/5/901" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Related work</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">A mathematical framework for sample selection bias</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/5/901.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/5/901.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/5/901.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/5/901.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
