<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/2/359" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/2/359.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/2/359.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/2/359.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/2/359.atom" />
<meta name="HW.pisa" content="ggg;3/2/359" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant Centaurea solstitialis" />
<meta name="DC.Identifier" content="10.1534/g3.112.003871" />
<meta name="DC.Date" content="2013-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Dlugosch et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transcriptome sequences are becoming more broadly available for multiple individuals of the same species, providing opportunities to derive population genomic information from these datasets. Using the 454 Life Science Genome Sequencer FLX and FLX-Titanium next-generation platforms, we generated 11−430 Mbp of sequence for normalized cDNA for 40 wild genotypes of the invasive plant Centaurea solstitialis , yellow starthistle, from across its worldwide distribution. We examined the impact of sequencing effort on transcriptome recovery and overlap among individuals. To do this, we developed two novel publicly available software pipelines: SnoWhite for read cleaning before assembly, and AllelePipe for clustering of loci and allele identification in assembled datasets with or without a reference genome. AllelePipe is designed specifically for cases in which read depth information is not appropriate or available to assist with disentangling closely related paralogs from allelic variation, as in transcriptome or previously assembled libraries. We find that modest applications of sequencing effort recover most of the novel sequences present in the transcriptome of this species, including single-copy loci and a representative distribution of functional groups. In contrast, the coverage of variable sites, observation of heterozygosity, and overlap among different libraries are all highly dependent on sequencing effort. Nevertheless, the information gained from overlapping regions was informative regarding coarse population structure and variation across our small number of population samples, providing the first genetic evidence in support of hypothesized invasion scenarios." />
<meta name="DC.Contributor" content="Katrina M. Dlugosch" />
<meta name="DC.Contributor" content="Zhao Lai" />
<meta name="DC.Contributor" content="Aurélie Bonin" />
<meta name="DC.Contributor" content="José Hierro" />
<meta name="DC.Contributor" content="Loren H. Rieseberg" />
<meta name="article:published_time" content="2013-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant Centaurea solstitialis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transcriptome sequences are becoming more broadly available for multiple individuals of the same species, providing opportunities to derive population genomic information from these datasets. Using the 454 Life Science Genome Sequencer FLX and FLX-Titanium next-generation platforms, we generated 11−430 Mbp of sequence for normalized cDNA for 40 wild genotypes of the invasive plant &lt;i&gt;Centaurea solstitialis&lt;/i&gt;, yellow starthistle, from across its worldwide distribution. We examined the impact of sequencing effort on transcriptome recovery and overlap among individuals. To do this, we developed two novel publicly available software pipelines: SnoWhite for read cleaning before assembly, and AllelePipe for clustering of loci and allele identification in assembled datasets with or without a reference genome. AllelePipe is designed specifically for cases in which read depth information is not appropriate or available to assist with disentangling closely related paralogs from allelic variation, as in transcriptome or previously assembled libraries. We find that modest applications of sequencing effort recover most of the novel sequences present in the transcriptome of this species, including single-copy loci and a representative distribution of functional groups. In contrast, the coverage of variable sites, observation of heterozygosity, and overlap among different libraries are all highly dependent on sequencing effort. Nevertheless, the information gained from overlapping regions was informative regarding coarse population structure and variation across our small number of population samples, providing the first genetic evidence in support of hypothesized invasion scenarios.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/02/01" />
<meta name="citation_mjid" content="ggg;3/2/359" />
<meta name="citation_id" content="3/2/359" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/2/359" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/2/359.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/2/359.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/2/359.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003871" />
<meta name="citation_pmid" content="23390612" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="359" />
<meta name="citation_lastpage" content="367" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Katrina M. Dlugosch" />
<meta name="citation_author_institution" content="Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author_email" content="kdlugosch@email.arizona.edu" />
<meta name="citation_author" content="Zhao Lai" />
<meta name="citation_author_institution" content="Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Aurélie Bonin" />
<meta name="citation_author_institution" content="Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="José Hierro" />
<meta name="citation_author_institution" content="Facultad de Ciencias Exactas y Naturales, INCITAP (CONICET-Universidad Nacional de La Pampa), AR-6300, Santa Rosa, La Pampa, Argentina" />
<meta name="citation_author" content="Loren H. Rieseberg" />
<meta name="citation_author_institution" content="Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada" />
<meta name="citation_author_institution" content="Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=F. Angeloni;citation_author=C. A. M. Wagemaker;citation_author=M. S. M. Jetten;citation_author=H. J. M. Op den Camp;citation_author=E. M. Jassen-Megens;citation_title=De novo transcriptome characterization and development of genomic tools for Scabiosa columbaria L. using next-generation sequencing techniques;citation_pages=662-674;citation_volume=11;citation_year=2011;citation_doi=10.1111/j.1755-0998.2011.02990.x" />
<meta name="citation_reference" content="citation_title=Molecular Markers, Natural History, and Evolution;citation_author=J. C. Avise;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=W. B. Barbazuk;citation_author=S. J. Emrich;citation_author=H. D. Chen;citation_author=L. Li;citation_author=P. S. Schnable;citation_title=SNP discovery via 454 transcriptome sequencing.;citation_pages=910-918;citation_volume=51;citation_year=2007;citation_issue=5;citation_pmid=17662031;citation_doi=10.1111/j.1365-313X.2007.03193.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. B. Barbazuk;citation_author=Y. Fu;citation_author=K. M. McGinnis;citation_title=Genome-wide analyses of alternative splicing in plants: opportunities and challenges;citation_volume=18;citation_year=2008;citation_pmid=18669480;citation_doi=10.1101/gr.053678.106" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. S. Barker;citation_author=N. C. Kane;citation_author=M. Matvienko;citation_author=A. Kozik;citation_author=W. Michelmore;citation_title=Multiple paleopolyploidizations during the evolution of the Compositae reveal parallel patterns of duplicate gene retention after millions of years;citation_volume=25;citation_year=2008;citation_pmid=18728074;citation_doi=10.1093/molbev/msn187" />
<meta name="citation_reference" content="citation_journal_title=Evol. Bioinf.;citation_author=M. S. Barker;citation_author=K. M. Dlugosch;citation_author=L. Dinh;citation_author=R. S. Challa;citation_author=N. C. Kane;citation_title=EvoPipes.net: bioinformatic tools for ecological and evolutionary genomics;citation_pages=143-149;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Plant Genome Diversity;citation_title=Duplication and turnover in plant genomes;citation_author=M. S. Barker;citation_author=G. J. Baute;citation_author=S.-L. Liu;citation_pages=155-169;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. S. Barreto;citation_author=G. W. Moy;citation_author=R. S. Burton;citation_title=Interpopulation patterns of divergence and selection across the transcriptome of the copepod Tigriopus californicus.;citation_pages=560-572;citation_volume=20;citation_year=2011;citation_issue=3;citation_pmid=21199025;citation_doi=10.1111/j.1365-294X.2010.04963.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Beldade;citation_author=S. Rudd;citation_author=J. D. Gruber;citation_author=A. D. Long;citation_title=A wing expressed sequence tag resource for Bicyclus anynana butterflies, an evo-devo model.;citation_pages=130-130;citation_volume=7;citation_year=2006;citation_pmid=16737530;citation_doi=10.1186/1471-2164-7-130" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=D. R. Bentley;citation_title=Whole-genome re-sequencing.;citation_pages=545-552;citation_volume=16;citation_year=2006;citation_issue=6;citation_pmid=17055251;citation_doi=10.1016/j.gde.2006.10.009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Bergelson;citation_author=M. Kreitman;citation_author=E. A. Stahl;citation_author=D. Tian;citation_title=Evolutionary Dynamics of Plant R-Genes;citation_pages=2281-2285;citation_volume=292;citation_year=2001;citation_issue=5525;citation_pmid=11423651;citation_doi=10.1126/science.1061337" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Bouck;citation_author=T. Vision;citation_title=The molecular ecologist's guide to expressed sequence tags.;citation_pages=907-924;citation_volume=16;citation_year=2007;citation_issue=5;citation_pmid=17305850;citation_doi=10.1111/j.1365-294X.2006.03195.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=S. Brancheva;citation_author=J. Greilhuber;citation_title=Genome size in Bulgarian Centaurea s.l. (Asteraceae);citation_pages=95-117;citation_volume=257;citation_year=2006;citation_doi=10.1007/s00606-005-0384-7" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_title=Molecular population genomics: a short history.;citation_pages=397-411;citation_volume=92;citation_year=2010;citation_issue=5-6;citation_pmid=21429271;citation_doi=10.1017/S0016672310000522" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Chevreux;citation_author=T. Pfisterer;citation_author=B. Drescher;citation_author=A. J. Driesel;citation_author=W. E. G. Muller;citation_title=Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs;citation_volume=14;citation_year=2004;citation_pmid=15140833;citation_doi=10.1101/gr.1917404" />
<meta name="citation_reference" content="citation_journal_title=Curr. Prot. Mol. Biol.;citation_author=D. C. Christodoulou;citation_author=J. M. Gorham;citation_author=D. S. Herman;citation_author=J. G. Seidman;citation_title=CONSTRUCTION OF NORMALIZED RNA-SEQ LIBRARIES FOR NEXT-GENERATION SEQUENCING USING THE CRAB DUPLEX-SPECIFIC NUCLEASE;citation_pages=4.12.1-4.12.11;citation_volume=94;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=H. Dempewolf;citation_author=N. C. Kane;citation_author=K. L. Ostevik;citation_author=M. Geleta;citation_author=M. S. Barker;citation_title=Establishing genomic tools and resources for Guizotia abyssinica (L.f.) Cass.—the development of a library of expressed sequence tags, microsatellite loci, and the sequencing of its chloroplast genome;citation_pages=1048-1058;citation_volume=10;citation_year=2010;citation_doi=10.1111/j.1755-0998.2010.02859.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. P. Demuth;citation_author=T. De Bie;citation_author=J. E. Stajich;citation_author=N. Cristianini;citation_author=M. W. Hahn;citation_title=The evolution of mammalian gene families.;citation_pages=e85-e85;citation_volume=1;citation_year=2006;citation_pmid=17183716;citation_doi=10.1371/journal.pone.0000085" />
<meta name="citation_reference" content="citation_title=Methods in Molecular Biology Series: Data Production and Analysis in Population Genomics;citation_title=Allele identification in assembled genomic sequence datasets;citation_author=K. M. Dlugosch;citation_author=A. Bonin;citation_pages=197-211;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. M. Dlugosch;citation_author=I. M. Parker;citation_title=Founding events in species invasions: genetic variation, adaptive evolution, and the role of multiple introductions.;citation_pages=431-449;citation_volume=17;citation_year=2008;citation_issue=1;citation_pmid=17908213;citation_doi=10.1111/j.1365-294X.2007.03538.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Evolution of genes and genomes on the Drosophila phylogeny.;citation_pages=203-218;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994087;citation_doi=10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=J. M. Duarte;citation_author=P. K. Wall;citation_author=P. P. Edger;citation_author=L. L. Landherr;citation_author=H. Ma;citation_title=Identification of shared single copy nuclear genes in Arabidopsis, Populus, Vitis and Oryza and their phylogenetic utility across various taxonomic levels.;citation_pages=61-61;citation_volume=10;citation_year=2010;citation_pmid=20181251;citation_doi=10.1186/1471-2148-10-61" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. M. Durbin;citation_author=D. Altshuler;citation_author=G. R. Abecasis;citation_author=D. R. Bentley;citation_author=A. Chakravarti;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. R. Elmer;citation_author=S. Fan;citation_author=H. M. Gunter;citation_author=J. C. Jones;citation_author=S. Boekhoff;citation_title=Carbon source accounting for fish using combined DNA and stable isotope analyses in a regulated lowland river weir pool.;citation_pages=197-212;citation_volume=19;citation_year=2010;citation_issue=1;citation_pmid=19912537" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. J. Emerson;citation_author=C. R. Merz;citation_author=J. M. Catchen;citation_author=P. A. Hohenlohe;citation_author=W. A. Cresko;citation_title=Resolving postglacial phylogeography using high-throughput sequencing;citation_volume=107;citation_year=2010;citation_pmid=20798348;citation_doi=10.1073/pnas.1006538107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=California Exotic Pest Plant Council Symp. Proc.;citation_author=J. Gerlach;citation_title=How the west was lost: reconstructing the invasion dynamics of yellow starthistle and other plant invaders of western rangelands and natural areas;citation_pages=67-72;citation_volume=3;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. W. Hahn;citation_author=T. De Bie;citation_author=J. E. Stajich;citation_author=C. Nguyen;citation_author=N. Cristianini;citation_title=Estimating the tempo and mode of gene family evolution from comparative genomic data;citation_pages=1153-1160;citation_volume=15;citation_year=2005;citation_issue=8;citation_pmid=16077014;citation_doi=10.1101/gr.3567505" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. W. Hahn;citation_author=M. V. Han;citation_author=S.-G. Han;citation_title=Gene family evolution across 12 Drosophila genomes.;citation_pages=e197-e197;citation_volume=3;citation_year=2007;citation_issue=11;citation_pmid=17997610;citation_doi=10.1371/journal.pgen.0030197" />
<meta name="citation_reference" content="citation_title=Plant Population Genetics, Breeding and Germplasm Resources;citation_title=Allozyme diversity in plant species;citation_author=J. L. Hamrick;citation_author=M. J. Godt;citation_pages=43-63;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. M. Hancock;citation_author=B. Brachi;citation_author=N. Faure;citation_author=M. W. Horton;citation_author=L. B. Jarymowycz;citation_title=Adaptation to Climate Across the Arabidopsis thaliana Genome;citation_pages=83-86;citation_volume=334;citation_year=2011;citation_issue=6052;citation_pmid=21980108;citation_doi=10.1126/science.1209244" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=C. B. Heiser;citation_author=T. W. Whitaker;citation_title=Chromosome number, polyploidy, and growth habit in California weeds.;citation_pages=179-186;citation_volume=35;citation_year=1948;citation_issue=3;citation_pmid=18909963;citation_doi=10.2307/2438241" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=Plant and Animal Genome Conference;citation_author=A. Kozik;citation_author=M. Matvienko;citation_author=I. Kozik;citation_author=H. Van Leeuwen;citation_author=A. Van Deynze;citation_title=EUKARYOTIC ULTRA CONSERVED ORTHOLOGS AND ESTIMATION OF GENE CAPTURE IN EST LIBRARIES;citation_pages=P6;citation_volume=XVI;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=Z. Lai;citation_author=N. C. Kane;citation_author=A. Kozik;citation_author=K. Hodgins;citation_author=K. M. Dlugosch;citation_title=Genomics of Compositae weeds: EST libraries, microarrays, and evidence of introgression;citation_volume=99;citation_year=2012;citation_pmid=22058181;citation_doi=10.3732/ajb.1100313" />
<meta name="citation_reference" content="citation_journal_title=Bioinfomatics;citation_author=T. Lassmann;citation_author=Y. Hayashizaki;citation_author=C. O. Daub;citation_title=TagDust—a program to eliminate artifacts from next generation sequencing data;citation_volume=25;citation_year=2009;citation_pmid=19737799;citation_doi=10.1093/bioinformatics/btp527" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. A. Lawlor;citation_author=F. E. Ward;citation_author=P. D. Ennis;citation_author=A. P. Jackson;citation_author=P. Parham;citation_title=HLA-A and B polymorphisms predate the divergence of humans and chimpanzees.;citation_pages=268-271;citation_volume=335;citation_year=1988;citation_issue=6187;citation_pmid=3412487;citation_doi=10.1038/335268a0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. C. Lewontin;citation_author=J. L. Hubby;citation_title=A molecular approach to the study of genic heterozygosity in natural populations. II. Amount of variation and degree of heterozygosity in natural populations of Drosophila pseudoobscura;citation_pages=595-609;citation_volume=54;citation_year=1966;citation_pmid=5968643" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores;citation_volume=18;citation_year=2008;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. H. Li;citation_author=L. A. Sadler;citation_title=Low nucleotide diversity in man.;citation_pages=513-523;citation_volume=129;citation_year=1991;citation_issue=2;citation_pmid=1743489" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Weed Sci.;citation_author=D. M. Maddox;citation_author=A. Mayfield;citation_author=N. H. Poritz;citation_title=Distribution of yellow starthistle (Centaurea solstitialis) and Russian knapweed (Centaurea repens);citation_pages=315-327;citation_volume=33;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Margulies;citation_author=M. Egholm;citation_author=W. E. Altman;citation_author=S. Attiya;citation_author=J. S. Bader;citation_title=Genome sequencing in microfabricated high-density picolitre reactors.;citation_pages=376-380;citation_volume=437;citation_year=2005;citation_issue=7057;citation_pmid=16056220;citation_doi=10.1038/nature03959" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. I. McCarthy;citation_author=G. R. Abecasis;citation_author=L. R. Cardon;citation_author=D. B. Goldstein;citation_author=J. Little;citation_title=Genome-wide association studies for complex traits: consensus, uncertainty and challenges.;citation_pages=356-369;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18398418;citation_doi=10.1038/nrg2344" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Meyer;citation_author=G. V. Aglyamova;citation_author=S. Wang;citation_author=J. Buchanan-Carter;citation_author=D. Abrego;citation_title=Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx.;citation_pages=219-219;citation_volume=10;citation_year=2009;citation_pmid=19435504;citation_doi=10.1186/1471-2164-10-219" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. A. Morin;citation_author=G. Luikart;citation_author=R. K. Wayne;citation_title=SNPs in ecology, evolution and conservation;citation_pages=208-216;citation_volume=19;citation_year=2004;citation_doi=10.1016/j.tree.2004.01.009" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Intraspecific nuclear DNA variation in Drosophila;citation_pages=261-277;citation_volume=13;citation_year=1996;citation_issue=1;citation_pmid=8583899;citation_doi=10.1093/oxfordjournals.molbev.a025563" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=Z. Ning;citation_author=A. J. Cox;citation_author=J. C. Mullikin;citation_title=SSAHA: A Fast Search Method for Large DNA Databases;citation_pages=1725-1729;citation_volume=11;citation_year=2001;citation_issue=10;citation_pmid=11591649;citation_doi=10.1101/gr.194201" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Ossowski;citation_author=K. Schneeberger;citation_author=R. M. Clark;citation_author=C. Lanz;citation_author=N. Warthmann;citation_title=Sequencing of natural strains of Arabidopsis thaliana with short reads;citation_volume=18;citation_year=2008;citation_pmid=18818371;citation_doi=10.1101/gr.080200.108" />
<meta name="citation_reference" content="citation_journal_title=Turk. J. Biol.;citation_author=M. Ozturk;citation_author=E. Martin;citation_author=M. Dinc;citation_author=A. Duran;citation_author=A. Ozdemir;citation_title=A cytogenetical study on some plant taxa in Nizip region (Aksaray, Turkey);citation_pages=35-44;citation_volume=33;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=P. J. Prentis;citation_author=J. R. U. Wilson;citation_author=E. E. Dormontt;citation_author=D. M. Richardson;citation_author=A. J. Lowe;citation_title=Adaptive evolution in invasive species.;citation_pages=288-294;citation_volume=13;citation_year=2008;citation_issue=6;citation_pmid=18467157;citation_doi=10.1016/j.tplants.2008.03.004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnell;citation_title=Inference of population structure using multilocus genotype data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. Renaut;citation_author=A. W. Nolte;citation_author=L. Bernatchez;citation_title=Mining transcriptome sequences towards identifying adaptive single nucleotide polymorphisms in lake whitefish species pairs (Coregonus spp. Salmonidae);citation_pages=115-131;citation_volume=19;citation_year=2010;citation_doi=10.1111/j.1365-294X.2009.04477.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=N. A. Rosenberg;citation_title=Distruct: a program for the graphical display of population structure;citation_pages=137-138;citation_volume=4;citation_year=2004;citation_doi=10.1046/j.1471-8286.2003.00566.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Sebat;citation_author=B. Lakshmi;citation_author=J. Troge;citation_author=J. Alexander;citation_author=J. Young;citation_title=Large-Scale Copy Number Polymorphism in the Human Genome;citation_pages=525-528;citation_volume=305;citation_year=2004;citation_issue=5683;citation_pmid=15273396;citation_doi=10.1126/science.1098918" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=J. E. Seeb;citation_author=C. E. Pascal;citation_author=E. D. Grau;citation_author=L. W. Seeb;citation_author=W. D. Templin;citation_title=Transcriptome sequencing and high-resolution melt analysis advance single nucleotide polymorphism discovery in duplicated salmonids;citation_pages=335-348;citation_volume=11;citation_year=2011;citation_doi=10.1111/j.1755-0998.2010.02936.x" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Bot.;citation_author=M. Sun;citation_title=Population genetic structure of yellow starthistle (Centaurea solstitialis), a colonizing weed in the western United States;citation_pages=1470-1478;citation_volume=75;citation_year=1997;citation_doi=10.1139/b97-861" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. Sun;citation_author=K. Ritland;citation_title=Mating system of yellow starthistle (Centaurea solstitialis), a successful colonizer in North America;citation_pages=225-232;citation_volume=80;citation_year=1998;citation_doi=10.1046/j.1365-2540.1998.00290.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Ueno;citation_author=G. Le Provost;citation_author=V. Léger;citation_author=C. Klopp;citation_author=C. Noirot;citation_title=Bioinformatic analysis of ESTs collected by Sanger and pyrosequencing methods for a keystone forest tree species: oak.;citation_pages=650-650;citation_volume=11;citation_year=2010;citation_pmid=21092232;citation_doi=10.1186/1471-2164-11-650" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Vos;citation_author=R. Hogers;citation_author=M. Bleeker;citation_author=M. Reijans;citation_author=T. van de Lee;citation_title=AFLP: a new technique for DNA fingerprinting;citation_pages=4407-4414;citation_volume=23;citation_year=1995;citation_issue=21;citation_pmid=7501463;citation_doi=10.1093/nar/23.21.4407" />
<meta name="citation_reference" content="citation_title=Coalescent Theory: An introduction;citation_author=J. Wakeley;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=C. W. Wheat;citation_title=Rapidly developing functional genomics in ecological model systems via 454 transcriptome sequencing.;citation_pages=433-451;citation_volume=138;citation_year=2010;citation_issue=4;citation_pmid=18931921;citation_doi=10.1007/s10709-008-9326-y" />
<meta name="citation_reference" content="citation_journal_title=Weed Sci.;citation_author=T. Widmer;citation_author=F. Guermache;citation_author=M. Dolgovskaia;citation_author=S. Reznick;citation_title=Enhanced growth and seed properties in introduced vs. native populations of yellow starthistle (Centaurea solstitialis);citation_pages=465-473;citation_volume=55;citation_year=2007;citation_doi=10.1614/WS-06-211R.1" />
<meta name="citation_reference" content="citation_title=Biology and Biological Control of Starthistle;citation_author=L. M. Wilson;citation_author=C. Jette;citation_author=J. Connett;citation_author=J. McCaffrey;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=C. Zakas;citation_author=N. Schult;citation_author=D. McHugh;citation_author=K. L. Jones;citation_author=J. P. Wares;citation_title=Transcriptome analysis and SNP development can resolve population differentiation of Streblospio benedicti, a developmentally dimorphic marine annelid;citation_pages=e3161;citation_volume=7;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Zayed;citation_author=C. W. Whitfield;citation_title=A genome-wide signature of positive selection in ancient and recent invasive expansions of the honey bee Apis mellifera;citation_volume=105;citation_year=2008;citation_pmid=18299560;citation_doi=10.1073/pnas.0800107105" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinf.;citation_author=Y. Zheng;citation_author=L. Zhao;citation_author=J. Gao;citation_author=Z. Fei;citation_title=iAssembler: a package for de novo assembly of Roche-454/Sanger transcriptome sequences;citation_pages=453;citation_volume=12;citation_year=2011;citation_doi=10.1186/1471-2105-12-453" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. A. Zhulidov;citation_author=E. A. Bogdanova;citation_author=A. S. Shcheglov;citation_author=L. L. Vagner;citation_author=G. L. Khaspekov;citation_title=Simple cDNA normalization using kamchatka crab duplex-specific nuclease;citation_pages=e37-e37;citation_volume=32;citation_year=2004;citation_issue=3;citation_pmid=14973331;citation_doi=10.1093/nar/gnh031" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant Centaurea solstitialis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="twitter:description" content="Transcriptome sequences are becoming more broadly available for multiple individuals of the same species, providing opportunities to derive population genomic information from these datasets. Using the 454 Life Science Genome Sequencer FLX and FLX-Titanium next-generation platforms, we generated 11−430 Mbp of sequence for normalized cDNA for 40 wild genotypes of the invasive plant Centaurea solstitialis , yellow starthistle, from across its worldwide distribution. We examined the impact of sequencing effort on transcriptome recovery and overlap among individuals. To do this, we developed two novel publicly available software pipelines: SnoWhite for read cleaning before assembly, and AllelePipe for clustering of loci and allele identification in assembled datasets with or without a reference genome. AllelePipe is designed specifically for cases in which read depth information is not appropriate or available to assist with disentangling closely related paralogs from allelic variation, as in transcriptome or previously assembled libraries. We find that modest applications of sequencing effort recover most of the novel sequences present in the transcriptome of this species, including single-copy loci and a representative distribution of functional groups. In contrast, the coverage of variable sites, observation of heterozygosity, and overlap among different libraries are all highly dependent on sequencing effort. Nevertheless, the information gained from overlapping regions was informative regarding coarse population structure and variation across our small number of population samples, providing the first genetic evidence in support of hypothesized invasion scenarios." />
<meta name="og-title" property="og:title" content="Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant Centaurea solstitialis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/2/359" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transcriptome sequences are becoming more broadly available for multiple individuals of the same species, providing opportunities to derive population genomic information from these datasets. Using the 454 Life Science Genome Sequencer FLX and FLX-Titanium next-generation platforms, we generated 11−430 Mbp of sequence for normalized cDNA for 40 wild genotypes of the invasive plant Centaurea solstitialis , yellow starthistle, from across its worldwide distribution. We examined the impact of sequencing effort on transcriptome recovery and overlap among individuals. To do this, we developed two novel publicly available software pipelines: SnoWhite for read cleaning before assembly, and AllelePipe for clustering of loci and allele identification in assembled datasets with or without a reference genome. AllelePipe is designed specifically for cases in which read depth information is not appropriate or available to assist with disentangling closely related paralogs from allelic variation, as in transcriptome or previously assembled libraries. We find that modest applications of sequencing effort recover most of the novel sequences present in the transcriptome of this species, including single-copy loci and a representative distribution of functional groups. In contrast, the coverage of variable sites, observation of heterozygosity, and overlap among different libraries are all highly dependent on sequencing effort. Nevertheless, the information gained from overlapping regions was informative regarding coarse population structure and variation across our small number of population samples, providing the first genetic evidence in support of hypothesized invasion scenarios." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
    <title>Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant Centaurea solstitialis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8axUJHaBkrm0xYv2lDQBMJSECG73D5y5BonBPti9M90","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459409","apath":"\/ggg\/3\/2\/359.atom","pisa":"ggg;3\/2\/359","ac":{"\/ggg\/3\/2\/359.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/2\/359.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/2\/359"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/2\/359"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/2\/359":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022byIR4QKUWI9xmkiBlziSQXdhGOVpLaV0Ii4sGzsIPoez+VnFB+jDFy7ALJhUSmtlW\\\\\\\/hIgRBoZ6bhyAbDaV4Lqqyqxh1qLRUobHm1\\\\\\\/6OXnIT3mvpZkquJRxZNvZar+4V2GvhBBVraiqiVU0J6Q+NlWlooWxrT6LxjuLXa0veT9Y38Sawn+UF2Z6eg2\\\\\\\/SXfeAGF3Knm7uQKxUWAvt1zH2h91aLBEsxmcXEKk8s8HjdELMEDoLWhloQMnp2t\\\\\\\/D\\\\\\\/tMlhAFInVeuHOCY7cjAhUItk5hh2lPE9LUCsDS6ybYYLzqBfE1yLWUD6yw2znJbGFnQ+PYAqNUbiezfpfk8LCPqBedD0Y2BXiODRvNGKcmfbMCO1B+Db9\\\\\\\/n4rpcIZiPIJMDkMNbkZM15iY1F886h30Xfq9vLEaUiKJ+bs5fW\\\\\\\/Ew42R0JZ2m5K7O6VehuvFmwpWCDhMVtZjzoC50Dtu\\\\\\\/9nt71LdDTg\\\\\\\/fRfO7vOnZGoJk3ShjPqwAvUA4mAb\\\\\\\/MAtOXVtYKf0EozuwsRPqVR4hTlPSR\\\\\\\/DlyadwDfhKLQj1p4TZIm3LG2antwi0SZYHTqHx9T1tka2ld6xQNKn\\\\\\\/rLV2qqwiYoqjhorkuFNPw5sVro17JK\\\\\\\/D5t\\\\\\\/36BEOCy2nJIik0x162I3IwoccxgPHp8V5AUANJxR7+n3NjLALUKQJt1CW3xqh8gJ39\\\\\\\/WQyoEs7bCtEKjMItLRRizpJahnKD+FrU5W6uQGlxJypd5bL6qyzq5AI1h7KSi0\\\\\\\/7nNoukBuPXNZgO3+s52mTRA2MwIZTxcLJ64zqkkADaNjda2RA\\\\\\\/8cS6s9l4iFSPNPq+xj0SeU1Iebl2vXXTAeu3vzjfxz1U0g32ypYpesHOuDUgZnjIe\\\\\\\/kGcYAq4iNFylXT2AGGbGQxMxhT8WCnfH7E55Jz4uFNdcibl\\\\\\\/VYjz1\\\\\\\/aHBziJnQiFB\\\\\\\/eNfHjLUNRtL1OSHBIQ74UZA13cIZ5s0zqaNvGImeLDywcR\\\\\\\/Zq5LFtj4JFDyAi6CxFsZ4H3DL469XjIOWVhA+vn+mFzFQwwZyQhvH2+DDisl0nmwQVSsAjgtnPc3Vc2pVGyUB+QBLhRs\\\\\\\/yg41ESNRstpEHyFkR9qB4tJkzCCvjD8xMgvRk2Ot+MMWcgbrakW3kfsaUZsMwFnyVpFY5DUvJT\\\\\\\/PhMoJh5+0h+DjQgnQLO6591jM9Mipvvo0mY4IrzOxGxULPsgaCzgB2z+MIOgqvBkTa4H375fWL0tIZIkv3iD2UjhK1RGdqzq6bQfhXOAUU1+lSMWcmhHZ2v2iUOfqio++CJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022FasT2k0+EGqzBACKCQR+yQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd3647fd4b03ee2f22b608a441d1044ecfd3ed3859ee6e7cd2dd49090b537e2f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022p1Wm3KGPwuFY62aOcSMVy9YLj\\\\\\\/xszFi3RWd\\\\\\\/qo1I32uC6msE5ip4N3ERpFd5EoaiYGylCpvxrnRF1sAd+Q1MgIvmi+9gg6ij59g3+qrHpY54nu7CLnkRObb\\\\\\\/cMYqld+\\\\\\\/ciFKhsr7\\\\\\\/1H038Z6y5YkIn7qXn13\\\\\\\/rqfqQoW5ox0r2tWvbdGq+aSW0Ux\\\\\\\/kdK6uBd8ysj\\\\\\\/r\\\\\\\/xN7\\\\\\\/Gwdi8MuzHbbcz8kXr\\\\\\\/z+6+pgBC0+I5hTtPNYPPeWbN5xTRiDnn70LaYomqavDWFwFs2o8erf9TivWigcM+uArQZ\\\\\\\/5dpVTcdqXdxaH7eYNxGoSDGrqEp4Z3iZEsTQc4du9VOXW6Denb5W4ZICMG753gfADce\\\\\\\/0mBPgHeetzYouBbsfHWSoKZ0xP0X6YL1\\\\\\\/KBRvNtsLZSWfB994LIjEerXmOyAhyWc1n822eMfSRnbdH1cVgEXIDUyaYWgBj91ysEbWt56tlg58vBSkQtkh51rD0EPTM4aRDXRtYTQaA2K9FYxA+txBlXS\\\\\\\/DnG8IgAskcHLMTNbdWdt4FQIJqdwnD+lYtv+dZAumsUeAElKX3TfrmRzgpELuRMi4YUEqPafFGfmhGygSGHzKrUSZ68GpQW9ZTGyvy2e\\\\\\\/uCddB5T3LWu1efm4teQUa5xcPsiKMvvzCKuTINXYiyJg\\\\\\\/cHTFcz7MWQeqU8N6BSLU91J\\\\\\\/l0FpixRN7jrsJf8RNv8c7l\\\\\\\/5dDG3Tp\\\\\\\/lWXfRI2gYkO04g\\\\\\\/x6sxyPpO7t\\\\\\\/xYUY\\\\\\\/M3KDc3U3wmVATD4OPA+lRckZtUuZqG2YDXtCgEcU4hK7HvtuCB96LAeaMT1QHGY2NN1jJLunyB4GYk7hEY6HKiVWOO0fNgCC7U7IZEtG+KTVLGxG8clIALvnsjbkuSJgeMHElAYoCR5rt\\\\\\\/mBxdhkCrmlsDNXclKQ6Z4\\\\\\\/\\\\\\\/HmqibO19Rx4Km3YhDR8bHt+uLbcbriQ\\\\\\\/sKqRlgg\\\\\\\/HeBu3CiO4FFZ8ZdZXvndE1n1eRCrRYMY5rZHs1fNBaV+jAiqtT4nvTPJhJsi9gzrK7hz7x6LsDd5BaQQQqhnQQa2nPZ\\\\\\\/gGRFy\\\\\\\/7T5PR2Jxw0M4t7uqtcYur2JbPPFl1SpgUJNU5c1KGYFJKlAptjn8RgbKxj1nNZGAyqviqllYaHb\\\\\\\/Q2lyHujokeRIpBbOlztOYWGpaDcFtYRwrM2ZVupQljaDOUlD2fmDhoeDmmEhYNifZrQEkHyl2Hbqt4IPo+SZb62yBAqES0sTUE7yTABnWlzJiY1GVVWS4wTIfngYz9oDrHmg2GiCEKF05AziMBH7W\\\\\\\/ZkS+N\\\\\\\/OOjEUmfnfvWD10cwqC\\\\\\\/KhBa9Mnm0N0Zcaw0FyIrzGKAm7eyzy0RpDbyMfgYMorkVt3887pPcpTU+jO7X02RLOLZXoL07lNrQW2mdX7ZsPUc+sp75s8qLDgSPKSU3NrUOQjdj7ZSO+iuKtLqBUcKRmGB56ZbzzOUPRy41rmOeRK2bektk\\\\\\\/G1HUOo\\\\\\\/NzgWukwXKV+92lvzH0YnUryoN+r2598XygM8TCpi7QDXDftuH89L5gNAo5QvAN+b7ocvJj20t56N4rpxiBX33nxT\\\\\\\/ukfMc35jbgChngdQBSiW\\\\\\\/qGWqIq5WKnB3Xf0MLQZ3sGJUzUD3ip7ce0qHIGTBSBL3Y2GPmp7b+Xz113SCDH4VBC2PCVciaiZGocJRv565d75iiBrl+DV0d16dsq8zBFTGSe5vX2ouWA7t4AmoBqqY+RrbA8qJiP+5pSIgYnmaptsOK1jkCowRsGXmMMFpK9O+i\\\\\\\/tFYLOUYdtD9AYTQu\\\\\\\/zUSh+aS944MuMyQ2IsXw0KQh1sOQWrx4U4s4pl00KC+dJaUaO2EuUcgv0wUNhzWTQRukJHXhjgkHqEcIc2sOfxoJ9giPNdU87vQUwVQQiwzgkQTXHQ9Dz2fmu7USDi0VHC6kmcFC\\\\\\\/wcVGTuUKGn+KOB9jfVDOAvhy4oPWO0aK3Q8cG5IkHAkh5eq129DlO5y4Uv7jCEjYSM3xFhp1PtF5MF\\\\\\\/d++QhpCG0XVcK2PlnHerEKakRSVeuGN9dAGTafb1j5JWdr84l+ahjUC8DE0uL3VswIW0rIECUnOqcPoBEuXmnBasJuiB7AeWKgsIE6V7se6jort8ZPlIav2oC4gLJ\\\\\\\/6oeMsnM6uWnZgxcR4hgg\\\\\\\/vjlfDTV6AmO8yxDqAap0R1Zena5SFgsBqkcW1bU4M\\\\\\\/tgql7JbTQvjMIBnpgQySvxjTcCoHt9j4\\\\\\\/vm2fYJeKIUFLsr\\\\\\\/aJV06\\\\\\\/gHs4eXIgiWF0Sj0advzxCZtCLH3FufEtJ7AvMz8lGlTYhrfOeYKAnFgLFdzCb0feCI5Kx00j8m+nnPLRGKOloYt9ESOOwFy2DyrPNI19KmJA3FI4UfheGcYpdgSU5Ks1urQmAlcGiO4mDT7EHZPTAyyfW29ahTSxBjsu8P632wJisHy8PyaD60REEwcwjaGjlS6ADesL\\\\\\\/SQLdXltwNvBUx41cSjNLKMlPbY5OPtjubAkbPyAaUGXKHvZ90yQ5oCYdL7Ic8y+AwrRrmMiWFCAoje\\\\\\\/owh62nDoPEcSbabzgOESzwU6kPvipBYcf6X4j\\\\\\\/ZYAMv8Cei6UJubito+QU27ernbrG3q2xKmLWZmKB4y26FmhRynZJtuXuKqL2FJpivb3zsI5oDS1xpk4UlrSXq7qApRuhTOSaUpOiN71ZFufAJkvGLrr\\\\\\\/AWyWUypv3fBdsMdhe6M\\\\\\\/UOfHfbHRHXc3GbWRirnzDyqaGX+iJ35D2izWTZxN+t68+P6CBJC20qMM4zOnLqoahTnN70FKA\\\\\\\/3HVo5GkaYty0VSgI+ttNsiLrY+Kt9Kdx0yWv2nF3Sy2PM9GXcewfYE9VqwytagT5Dm4QZf31OBZN0uSwzZpJRqSVMZYMvU8xPO78RmrK57bTdiZ3sztHTRwLC+ZfsUwh3eNt2WaASZdN98IwdGXAKactW1oWlxjpryZaEfcsV1gz3izoBXjKw9iYGMii9AB52EwIhvyqpAihot7fsmq2yWpvNhJ+4ylD42xnSDa+\\\\\\\/diJI0btAkcMjv3Kka1FDlEMY488e0pI8VvWiFJSzHiWZtxjrfQoRYJCWbqdya3HnLvMYqNfZOsy7n+xWBhUI1EbSDAFE\\\\\\\/zl0AhaVmAkCsHTzogqOglOnTcHTDx7zfpBqIIHZowRuQXf4ZNOxY+0bvSRavDJ+gyZOg5r0aLxVEVametEdDX+ismpJsbHQIyctemy8Q88jigJGuG9fSd5M45zZgG6BsmjaZMtx86HIC7MlA\\\\\\\/HalP8ouQPJzl51QALwYeSGgQ4d0H1NHv+zq4ZV2uYp+OoTn5+iOLMNyiRQVieVikwOirojpS\\\\\\\/FmY6fR5wkXE136XEdlJKOZ\\\\\\\/VajQjVEvBEDWWabNCJ8ImYUcFry0\\\\\\\/ShA0gDW3Ow4Te+1SggelJqx4mQRx1KSpERwTmKGKq6v4l2W\\\\\\\/2bm9sqx0v74YB3LQYA1YlSqhRr\\\\\\\/EFKCbhS\\\\\\\/3GuIf9agtaRRnKEBevSOQPv2HPaA0BhuGteevLNlUQ\\\\\\\/EJLKebQDG\\\\\\\/hIGx\\\\\\\/gBDsDvgvjOgAzU3XhNmx9hjLHgKcI4rQDUKafI7Iq2HMzOVxb\\\\\\\/mZQDVgYXoY7zXhUWweuciIZYebvkYj11q3P\\\\\\\/vBbjqSPSUInpkvg4r5\\\\\\\/pnIxR13s4v0SgpZT0C\\\\\\\/chk3yCkM+5QJYDqWJtt3AryeMHj5rEexQVj7IuaDaDnFTTkOEo\\\\\\\/OqoXG7t3SaCxMcONlKnqDywCmaJ0x7qi4Y\\\\\\\/+Z5Sjn7lB0GBCYhZvES2hwnqBC4FKbzhrNNmN7O6pVmhDL7sOvTfNu8J0Jm6MchtwW9g+MYYkIQA2SA\\\\\\\/TdtPyFFJKy2EHzc81XaR5BqVVM\\\\\\\/L4SMXUEwcTviRpH2mkIR7LQapJDrBFj8EM6jlN6rcqk7MuGeh1EfjkuICp3ud9ulsL5iFqDrFaN6wvVHV+b2PFT5mYYkYDHzdc7h7CfF7eNtMck+dOy7QxAtmEEIbIe2+gvRLeoblmGN1+nI+SqnRHDruJKSR\\\\\\\/4EHRhI+EBFbUrfEonW10Hagt4s5QiaRHx+Ag7wftsH0HRKwbjj7NvPS0zoqvDRKhtW67w0baChWEyd7f6nXNBKdr7PpD1M7AEmVSHh9tQCv+MhuyWlgETAsdNLCYWWLYKv0fD5QzM\\\\\\\/mIhuLTndpYt1wupLyLcqFCAhLYZPP5TbKtxWREfykoePcuq+KS6F0ylwwdrJCfDkpEaMZnr6hNc8VDQWbq6jPq4ZqO5whMlXP5eAWxOTpeoBICX8cC4ScX5sSEnlqTLZDDWZU7cVFp\\\\\\\/HdB3F6wk6tq8UU8u1PZ+PQ6h2qPYObta6BiyUk6sC0G+UhHygQymPR6d0t9De6akbqvYKFCfv7hZNSHEQ\\\\\\\/KpGZqUqOmJxFX6LjymaE9+6EnO24hagb5gUMA63Hun0yLeGGUGXY9Aa2EJChgH+ZVgvDwfm60R9\\\\\\\/vU7yJvqlqGmFuqbRwoN7mOUFiRzkpMZhbpMXejBXGkJNOu5B0mJH0cL1VHQGiZ9o0saPDP\\\\\\\/NEsLZi+tEuFEAO9W60aOnvNCXhyTYJdeYbLhOAxkEk\\\\\\\/F2KIpCRyihIUBFoJRt3ytj+C+0503aKGXC7chDsozkHyi+Fn6qrSiWLTlB\\\\\\\/PWLMR3AMkoaun5TdVoTyA3oB4U1Kb0ERqgqdN+ZlDgI2lyBfm3uW882TgErWd6jmQBzqyLW4RSCR8BAvugnwF+8NBWtBwiLCfxbfk+cJ9VKyGZ3+2unw1tMIQ04ioeliumX9\\\\\\\/ISqe0Oz0dIgjKAhBjPpu8ESX8JJ7XIJ5LyEuKsohH98bmoDSElWXAspRuUZzY3BBjOylTwtbaWvTuCR4OJsImHjyouVxsPFSBOoEsYj7drq1odJi3HlSMSL\\\\\\\/CKnsTksAG9S\\\\\\\/XolH\\\\\\\/uX\\\\\\\/NkhLyGTiiPBSLIzUMcNMem75iGE1oobG0etyo7gDu9yFatZAvlxsN7X4wp3njfAdI2nh9fy3z3yK7vQraXSScieQmaSjWuqfzdeKYNFSjM7YhbMa0pjNwdS6GNx8I\\\\\\\/F991hhaC7pzdTj6nla53YeoBWfuQtkoRhGri24Hy77luPgIeMRyfwKc9iHXRf0TKEwRA25Ye7VDrfdx\\\\\\\/sd4fp3g45BqDA56MEQd6qSzwtahTjAP5Sxg4fl0WG8f5uxJYx65VpXKY4FhNH\\\\\\\/XCuAlHLkse5REKs7p1wPYEUiH+qZXpiDtOIFpR1axJpYN1xyyGDC98rMXMplITP5WfVd\\\\\\\/SX36OSGm9jGDDlY5cpaHmYyC8QeuoF+IWXkvhy1U7JotEqngOpdFT5\\\\\\\/ik13Cr9HJcWJaKFjkAkCQSt57xj1\\\\\\\/6i65qnKK326UB0YJdSMSTvEHgTtyOFVRP\\\\\\\/1UceFAwGFNdTjpUBszKjQ3jc2cZmiYjVgkJnAAXQUZM13A7epZNHY+fMZu0EDHgVMyyJQK7UKX7QZZfDXBQCtMiirUth1PD12Lb6aN9Qcr72Nt9lHk1yq2VJ2cqAjvJXco5Gjrf7VHcgNHp+vvVLNhMjEL2vLQKVm82jpLqWGQzCpL6nul9J\\\\\\\/NCHXgiX6bxH8OneckF2d34BD7AwVaUJDc8wnzWKErPCsFwMblb4SdjLJhbkY2j9ClEGuGwOwVag\\\\\\\/cDLojRJNIB2vBhst6jmfM3u6yQjEc8Ak4V4taX+OBXjsOZ2KbCCUJ0M8R5i+DW0WqiVb4Kjit8HKghmpsIGwLKe0uLtbeZgAtUzuWsvVeqc4Th7Xcz1FC9rnoPxPNs+yf5iwArr43LMhwYy3\\\\\\\/hQhpDgYfwW1bpju+CzG6XmUc8zZ3XqUhPykEOkVOzylEN4mcI98Etu0Zvozs1FOojPL56EzR3zr9PNIbRvv1aBf1eWgbcsubmA3OjjSnANdiW\\\\\\\/AIaERMUHlJXu3+giPbHE3tQ4rafQfSdtai\\\\\\\/BTlNwcMWSFcLeZSe2OQ2RbslZkfW8ruty6yYiikVLgONd4gzlGTJBjxp7tjATo5YjRJcoChu5NZukFRBC3Yzd0DeqI5jPZpDv8IAR0hXfnjYRR7pVMc99PcZkVNqsH9D5fKCwAPXGbdfOaxy8P2pmlT6Tcau8qVFJxRV1KRE3yI+oIoRbASLGqYHrRkOOjRY2pCp0Dpu72lIyq1pl4hofuOGVKQIm+ZfLDdZKymk\\\\\\\/66\\\\\\\/HF9KA3e60KqQrc3\\\\\\\/GbCedXbZ+R7OMONxMu3Eis\\\\\\\/7DzmMnHGaA5ry6lvNUMuU5\\\\\\\/cGfTHRVuQ2bhMTihx\\\\\\\/\\\\\\\/niCgpY6ZkFsrCbF0GYMuF7Gve5+Jeu6UhrQbbGja3S2vN+VHGNQPnscILiF7vcuR3\\\\\\\/CFfbJCntH7Qw12nTU84kle1LrmEaWzCK9mKX\\\\\\\/08W3\\\\\\\/n5qZ5MvoTUPhP3kjk98ji4PUqhxq90oooMevZXYmJbaMhpYrcRjJL0wxVg55\\\\\\\/f6wV4\\\\\\\/zWPNmIFSLM\\\\\\\/i\\\\\\\/JoH4yVylcUPqyMeFo2Fmbk0x9A03FK7+WtvM\\\\\\\/fGAgmW1fB7My1UCj7evG9CeQV8aSNqpPAZjWhJhZiDZt6YwmxuMMdCEj6j3p542ZqmuPoa+N1yTWw84vFSiGBhrKhfv+z5c3gj+AXPsDMWMEsLfn7WswSRwPO+zDd5J7YVOn\\\\\\\/\\\\\\\/ckfPgoWfpCdq\\\\\\\/B4\\\\\\\/FysEKOm+kEl7UJ3MVthfjOjeEIhdv5r9st0x6weXQ+JovKB\\\\\\\/DyWxrGUXDC2NzzYoKt37KGmjDscuUsXZXeu+t3IN4rkTYiLJbnO0i2jGf8yU\\\\\\\/iAvJYXiyjUUkgaYMJYWYuoVLPu2ZtMJR04nvviSOOhvt0IUQiaJSZ\\\\\\\/DHGe62buaL\\\\\\\/RTQuzlBe4o2CVCScon+LVJRcb8gQg9iPerC0MNFsKdWbDEzrd5HdAvwp1h0O9PaA9PmwQQfHeM5hyrcBMzitDAJd8HOp46NgtNfNqMEk8QaQeYkPL+bP6icc6MTIFp0NdYfYKm8PhQAHJxrTtYKsn96K2M+CcEQs0LxKKGtnY6OVFz4pLmyNNhKzpU9juc2XLXeD8mGZxlSbEtFB5NlhcPS+xUSa\\\\\\\/L8ZMKs+x9s1jSqG5\\\\\\\/+pkfLGAjYwRpStdk+XrqyEjKh0EtaOQCXeW1682q62oofkEaA1xVliqiRKTk+5UwlTdlUhA4oOeGet1jZs\\\\\\\/qEw537Fsq14nV1K1oe6yiYyfNPbrn\\\\\\\/01TR6ZjdWmqxpplD57PYkfRfOi3Ro7wPYJ8GDNA33TTuwks2n1pytoeKP0Eanwcg2jLwShsP8\\\\\\\/Ui8oGKhVD0Qet0gaC9T\\\\\\\/lf3S10NMZ4Uj80MvJxO1WaNHkAWeUUxyYgndrC4N+ysZKgoa9PHmN9IQtYe1\\\\\\\/62I0R0y4l0E9ITpncyhbb5JbH2dqUsSgRxyjZV3aKM3aukHJS7oquWmr+MeJoncPIMGgSMKeHx+pepWHzMV1Zhd+YRx8Tq6fp8ZZTodazAjzMWQqhS6pihNQwQUDgmkjQcEjAwYHsAdhWVhIUlYAVVv9EqiRq+7XkFsAfzKNcQfoVVAriXS7YpZHLOIZrgsoA7i6pBaV8TiWiBuEEFK9Cwxp3bYnzDNGT3SIm8AZhQk\\\\\\\/\\\\\\\/9hNRJEMn0O+6jbkqv\\\\\\\/cxDYGGKV3InvnBIabMwHT\\\\\\\/B9SukcgSUMqqZrGT6kyiABU5PMwhP3f8nl+rM5\\\\\\\/OLizHZx95Wuf8vKywB3nj+NZjoQIUPLq76Kf+5sOGQHNnLSxQ5zDQK27ijxMhjxpb\\\\\\\/EEDPjcawQqHaFe4Y8N5VqSVqcMghGlK5GZGTV6JnyrOO98w4vX5Z2xIOdtx\\\\\\\/jglJDLZ1L7xB4fmhVqNMsndlE+iF10+o3mnoGB1OsCj266EwOqvIX2WR70mxsy5nD\\\\\\\/5QSFxMz9Pe8+1G45KQ9Pvduv8XkWv8QqgEZd\\\\\\\/EaTw8t0MnSdToCDwTU4S40AYxShjWpQz8pp53nE8ZwgnVdwiHAFW+A9\\\\\\\/3mLcstApdHvN+dagEQAkI5rExfvQI13OXBoAG2KL+22wl35UX1jA572Uc2snzMt4fDU19QXv3Yp5WfKK\\\\\\\/HIjHWKHcZM6EZEQ\\\\\\\/XIHaeBWJDsmUZInGjJNUgqVZh27FpYVGubo7o2jZYPvdkDpW15W0M2F\\\\\\\/ILDW5+vbWX2oCAjQ3jZgxyrKLeVXOPzGt5W\\\\\\\/2qhjoNnzZ8Mh2Whbe+GEBn2qqVAJq7xUvJ96zZYzUT3ns11cTFPOW057tIrBYedVdClH9rxt3wezc2mvFUq09PK4siXbFZ8\\\\\\\/NmdTfChC2wu+Iwv0ymoGbaY340GZ6bGvWTNkPS6vBEQ3eO6VeRhzvxIxncVN9Z3wLYLEn7pFu1fcY00TvL0qixX69AGxcfpDRtCjYxlorjdkqauFZ\\\\\\\/SEaKJmY2IoYWHXr536FHps6RSKTbhHO0yVrXsGWxjpOryAFSRySV62snRXjcggHV7VKa1hoA3StluzRk+9YA5ODOwh4JIY5vmbkkABuDg6HTMzUfwd8vb8r\\\\\\\/ANM3lRp1NS2IM0ZtwUUnEjWeRr\\\\\\\/eQKjeICL4eeA+y5o0+Ct6fcshorjoU1+pWMByG8yV7sKfjBFp\\\\\\\/vsKzBDlJTMMNM1R5s72O\\\\\\\/t12IUtTnv7SBFDpUm5FMaZQqJrDHZslxFd2NS1mCeSrsxVePGumM8XlIVLBbl3kwxaR9kQkhT8JVE13vA1fF0TyAUOAD94rQlkGahAg2up1lFBdjrx6bMO5vPSK88e2+FBjSWB1w7PNtsTkJoWhKe0XPqdbeDQCbDSg0lFzNbaIbTA6mfuq+uXTSdn0V3CmCH2Ip+zufQpWC\\\\\\\/TzR629LuUQCDI0swzHa+qTABfi3LTF0PfMtqInV\\\\\\\/4F3pygpBun4QV76rRXW46r4bD2lGY9IuRfOm4ny\\\\\\\/41eIGldQEGlVV5SoX0FBxzBQ8tDZdzruO80ewtX4WZ7LuGjV9fO5hGsztIbr+4Lpi7RdXiVuDPu7q3SpKT3DyO\\\\\\\/FuHVb34NeohNJHtG2mCdzd6vYInbTbPx7WoQmhsH9ra5yn9rVFw0N3AhJ3HyCh3Zd1V9e4p+nRWrSG9xN9nPhSKDAINDLJ94+bV8StT688Qju8TfwMYzPUNUjWbdlV934Rb6Ko8fWmD6Vwl+6sbUb8YaI+Yz+QWbr+afHvF4XH\\\\\\\/\\\\\\\/ke9hzyeN5K9iqwgRsV8nBPaeAivFfT7Qe3PX+ZeYl4ionkbeublw4j8hioNQ7b4DyJ\\\\\\\/eWe4RECjhsTBP42P1LlWunJRP8O9l\\\\\\\/Etp8oGB3sEZu1Amt2XV1ECXTO9guSCgXNP+i\\\\\\\/Jo5VFY0PgwEDm6bLhjGtDzzBwpaOeXX6WCVOtVcPGNedlZeh00HEnh5JiGCsP3dM5CCuDY2bHhNr+I5AxoRKwwCRoyBNWIxK2\\\\\\\/U7Fy45V8ct5zA0oKZfGExhZZZMoQy3oeVyS3m+cTKqySmr8M\\\\\\\/Il0DeIeiT0vprP8b66EHHIrRfzx\\\\\\\/Ohgm0L0ibLWPg+WQrmAK9Q726Tfyktf74J3dxJP1RlJakhtlI4veFXpw5lE7CK5OGPd8LxznMKEW8wQ5g7FR\\\\\\\/oV7Udmpuj3CsS73mv0SukMiRXXwwlJVQfWw5mIkbcjZoS\\\\\\\/8NsmiUb3MA6i1VRc9\\\\\\\/iUIesyZU08+1jOvxM1qCfdYzp49BcrT1yGzPnz5DvhOXapen6Lkypghr7B8esfhHAyLpjRTkFFoxaGuVHzoDUYSUVU5RIM4X6eefhXIgTuNrMCPN2YQBbhzyHBgRWRlS\\\\\\\/yE8QXeCffJmUz5wUVL8vceqO2qE1vDhQ8OEAE0Il94W7fySxu2ls6vEc6X5oqxtWSG8et8VOgROUqIDh9bC8AiLRap77HX8EZK2JgrgUo4guoAsEAoDWpQq4ep8kBR1oMn87Tfz4p6SEYNCgTKHKA+sMhnjF1A8Wxi8EfAiRsfjxnlsgm+p0GEz9A\\\\\\\/JmGLeokOa8cuCrC6\\\\\\\/LSiSDOIMHv3u5dvZgydF1XJWtveeg8leMWznqrrVNAxoxAUHlud+KP6I8v8zjTO0vD0MTm9M2LB7N5y6lpJjwEUFB09rHgjdsyZyOOKHlASg+D+ude+zgwOuXzDK6WGXuyiOpWUQnIJm93NgyIVwr6XYELQE3inQ1pyIwDGk6llCCxp\\\\\\\/ej\\\\\\\/lpLE6OPdfyB1hmbrwyKe4l9wJM4NZDeR9CCq2hMXo2t5+1DqXnsrxaQ21sxq6PPq6W+YjTjgmtHUr4MbKF40U1xesd3\\\\\\\/jbUy+Rw8RaBUgm84xO8oZLBLkp2Mr8r\\\\\\\/XDVtMMJ6yn0QjOWgKm7xEQy2FMuEsCFZCbweY2z0EoO1+7f65N2\\\\\\\/ptcvTH+fcok1aB\\\\\\\/1I1sp8DPmbaU95OCd\\\\\\\/cKf\\\\\\\/zztA0qVaygH75aXZlWm1tFM+SnGXFkb5MkjdlYPkJ8ylYDir1Ed7Z8BleA3DEdnSPjS5at6A07kGeEvFUrsn2fj7pWtLHcQI8nwIhntWoyYVJhKhEXiGYvQeiJzmbG313vMZaqt4rCBePtai9NudadpVuWMNu6M+MU834iQ0gI8TXN1Wpu7JFQThESGLGIvjI0rgq\\\\\\\/a1ICfrMc7IE5kUN73FKgFhRyjAIR+iVdTziQ9Xao4AVY2e9\\\\\\\/CoSwE2g94sOUsRXzwbvJbaxSFb9+sjsJ+FMVNjK4sB2FNHkcCpddHl314BIslB17RdbZ7ObHLi1JfTZuZ4I7AWPANU9jiqplMAoLONL2PziQ3EK09sS+8+fXi29bxERcJfClnoJO3Krf07lS78ugRtl7KTyjR1uXxuqRuD2lZKO6p7jBM4cW8\\\\\\\/s\\\\\\\/K5BzgRd5syK1IzaiNMuAP4jKhf9WS\\\\\\\/OeeH+pXgChmWnzFECNlgU5\\\\\\\/uuAHKX9z0dQEvkNOGfquD75wj0ibE\\\\\\\/yHYnCcq2WJYVcOSfTdl9qOm5w2yDY\\\\\\\/opA6hW5qzA0lTd50FQLA5+MuBG0oK5i0MtJgwRWhVSBMhLs91OxrMu1her0ILOpndU9UAavy2IHs5OZWV1mM1hZaI+dGnGu7LxSRFJsSHzr+kzB0Dvv1r9\\\\\\\/vI8ZxG6aI1P6yUd9fxrPnbCVwviH51U8yu0nKwact0cSmN+1ROyVp9lFBUDUv0oIO0r4ijQ0VGXm6yrRXY\\\\\\\/MwEXZauJ4s6KBB7IcEBupyEWlytdmWOhs4TOAS1rT9Avy015\\\\\\\/Bwx6DX2m6NpenJorsmU8FCWmXUpJp02tXx\\\\\\\/5HT49wxWaj1RMGezufmvZNo6c8Fxawcqfr5OX0x2H+SJfctYEOUVPETREk3x91MKSHnXskcDI754NBwevqANl+AI8aL7mgoPA6XMSrcJcnYVPJQXQy\\\\\\\/9uyyadTjpKxy5DiNfBADcSxORbUmYu5WJKn7iarz0CQMfCVLmUiUMWI4x347m+StoVxpTkK3QVF6vr6IKcUSJlQX\\\\\\\/8v83cdNK5929e5QO0uFszlxV6ZCSQQhFmBIubKTuAnB8UkxSUaYMdyA7RoE3G5z4G2v31uBL48f\\\\\\\/hmapdzg6F4F85sKizp1zayTy8PQiLs45EtU5XHPigL4Zhnu6DNLTigyJKUAoLCvG2yA20dOp6KUMRGGwwWmlp0xU2OE7+n5NHBn8gce+t3a\\\\\\\/gmE41nLQ3l6CJHWjVleRZgcYI7\\\\\\\/r2r+SgWRHAVlSTo1sUszRLvLR4C7pQCHhX88vNSOHSQ1erGZzCYzZ01lXUhBMJYK\\\\\\\/h3NU3lNNtjtdtb\\\\\\\/zRFtGDo3J0VP6cPKL\\\\\\\/kNPI2zQlEfC3iDDeX1kSytmGKS85Nsp0DzSSz13nMbbte0pGgA8TfJ+W7duaMw6rYMoih3OaYAXPvsYQ4D9eRNxMmjCXgzRZn1mn0V2De75dJWqtuMqzCsILucddA+4A4f6c8Rb5OQ4qQpeflxaZSEs8lURMetaLGc12i52+I3o+q0WgblahCGopVZHA7i00J5Z0uOohy3A2\\\\\\\/lR\\\\\\\/ELOA4tYfN1hpK1wncna\\\\\\\/b4RVZfeABFCGTlpzCcks2XjYFXRLAkL+g+IgWJLCwJlEwf2q1G1+b6KMqBmPXKEv3i58GLDO8MjP51JQ4AF9YaRyjAz3e+gvTdoIknJ9PdZSSCrh5FA8W+al+CL8nHoWecTHlG47s769Q+a\\\\\\\/FJXlTsKXEplwxkNHYwwI9PwtT4emdsyj3jhOlnYSUYrSToUl6\\\\\\\/mBLu6wyyKUh00llU2Ao+\\\\\\\/tcscuZn74\\\\\\\/1ESBUSWIyhDpd5Sr2Y9hYk7vCtEjmAQuuOTkY0kOspVrImhXPhfTeQx7y0HrkQZEoIAQWLy8zOK6ua4HeEcdFMn+CL4JPdQJ3Aa2bPLXaGtWG9R2NdaiLqgyIi1TEbR9Hyu5YOFIlUhfB\\\\\\\/uv0xbfOdBmnbggDuTd6RnPJMpPDZz\\\\\\\/kOjNX6Xad9MqBmAKIfD+tQAlC0dIcwOXuQtzTkKwdncPLWlqdVI0Dijav4S46HSs7\\\\\\\/nTUN3tovzAxFS5dTvxUNmuAEoewnzS75BiQkEMkmKuQShWc8jwbHPn8Mvwd2gdm+a1EGkb9EuNfoQ07nv6pOiVuPL+VKfjtPYbjaNC9pMZU9CivtJlYdFnSM7Uv5HGVhk2bNynO97Dk9ACH8sgDWA5mNHGc7JS\\\\\\\/74DkotZJ81Ngor8Jv3RSXdJ\\\\\\\/d75FfssTa37Ve\\\\\\\/n6VQeE3zmg6u\\\\\\\/CjRhaf13pFDFuyjCoXChWBfpeeMD8WrMV+5Gh9\\\\\\\/W\\\\\\\/AtlzSWIh9y7lx4TDLgNxJYikQkpap1sfUPnmH+ySFJoiQOcFG+oqWaos2FicAQZIlgn26hPpS+zEUIR6uQ8VYJsC7HfPfENzQpZT8sMqzs\\\\\\\/q0Rv9SNQ4r1doOJBpqJlVLOWpWg5wqOOnkjy0o372BQDi70Zr7Ga0nhkuW3h4mxV\\\\\\\/dXrXGXU1yckERBSfutdNyCukhxOIBCLgaKGA3a2eChVkHwVLxuNhf7XH13BTgu\\\\\\\/I\\\\\\\/l5jZo2knTsCnk1AicJgZalrdqthi67kYZk+jMHJNLVnYT0bateWp6UPoxvZrD6cz2IQMcnUeTDbPIXY9LeoH5sn0vhSkvyWrdG583hVLpJBS\\\\\\\/t\\\\\\\/A3nIUnNwLWwK8lbWD0B+2Y4WS4ig+Oz9TeDnM8hv3fNhsoO40+bU32dk9\\\\\\\/1vKRBXQl4pOVoPs5o4eSJOqeYE59ZsemBS2BENsggb9veo+\\\\\\\/qnvoeB1R3PU7dxHc1CF92eTyZ6YBiuc0yui6ziQaxFPwTfZCryH2tRmngdShKOFClWYTsVd93pRb0HE9j4xiWw0hAQ2V2dHsJGqKPdSdZns8e36JSTQHL0YdL59y6pIwY9djpDGZo9dhM9rvc+fIzzetKNAw0\\\\\\\/TYi2bHuXsir4UWj60VkSjhiQboKGy29xEzRaRlfmOzk5d459n9wl\\\\\\\/zfyRo6AD7zA6R1Q7AnD3niNbdL9Zz8oboZttqvgW2ioGfwDdqsLFowJWJHJpRmcUSQ02eypWzP4agbrxrp\\\\\\\/8E\\\\\\\/kR1synyIqm\\\\\\\/Hhljx9Bl6uxGTrlDzzB5xcr9s4FICzHDdEfblwyGjYAHM+7DokcMErhstCFQQQOlc0XnFwof2u4IO\\\\\\\/Q2l78FJ5OApMtxT4eiw1LJEsE2vXV0ODlv2tEjq1jesh5LSdK1TKZmCLt+UU\\\\\\\/horDnZThbNoFPN5Gd8D00\\\\\\\/p5oDjgSaLJEmfrp1e\\\\\\\/KrXsQjE0D3OwSS2Ire\\\\\\\/D\\\\\\\/MUUsIAu\\\\\\\/TSNL40gbR3yFXO5M27\\\\\\\/2QrcNGF12soT0p5FuQ7Ak3M4f4UdqwAWLn6bEpcJkb+SMUfw7BgxnZmN2V4d8AUQ0vp37Qm9n7Siiyx8D7phfHa6jZJ9Af4825Qm2\\\\\\\/Q21rt9siNUbEwq9oHwEyCJlUukqX\\\\\\\/HzVuzqk6yk8C+sjZINA661sVqh9QyRMy02jzC31YXdlKOfZmeHSgMIf+RAhdigLLV1qerFv6duZNaoiXSyUa\\\\\\\/7vDRsHC82jANWra4a9YMrdouN+YzpOpixz6r+VSKre1KVL0CdVRMRAchp39yp5acnhR2307ig5kqBUCZuru40KBc\\\\\\\/2Z7yMsPxPOYbTso5RSGquROuEhmfiOWe5+whnexcckzKZFquYesMGvdXjJOaa77LOdFwZQB5IaohHfcdXTVsfwS8Qzt6Ve31f0DQP9R16PsAV3dIbzNtFjo6KxusHKiX3tiECglxLEeKyWMD3eo4830Fc28c3xc604bts9Xlg8GSzm3289uZ08xNPSmNEcBcMhkIQ2QvLvi9JrSx4JL+MoMeU8WvVCvvGOD65l26qcAXiUWqQJCUdLFjLeGhpdG\\\\\\\/OjF7WzCGZFZvgQ0aMUkgFgYon8OYW+wisFDTcuXKVLunMs54ECE0XBs1CGxnR+QBHOLNsx9fURJ9amPlLBgfpNNXb9Cqi3kIvNRbavefKw+y1IXHiOALXHvaXxG2Zc1Ejy1T2um7G5cjXnA9ljuzgY0OwREQc+lDDQGTTQoKBAO7loDCa+70pQ1yx9swuhmcm7nfxwBCzG2TGNTZu63cIQj6+HUvR5HeBxShUWJmrOZSdk5RoGqy+gp\\\\\\\/xaE0zaBWBzqxPVJlu2L0TnLzH+Jj7YPZuiL6VnjPPpa6ImNpNW4F0H4C2dGtoxUzk5pCVVFkg1y84RbaURCqQ8HUUY5gl7f1+dXowa5Fz3likaO\\\\\\\/02PTl4kMs6LrREwu8tZVm0KZJ7TA8Ofdf5dPXs2mnhFbi+2kYUY2LW\\\\\\\/rjWf7WejK1hLeDOrUf\\\\\\\/e1GQ71GDCZD7PV3gsUEJxsgeD7TfHJ3IjcSYrAQEAJvUTt21bmM5\\\\\\\/du+AvbpOw94swrRke5Y4wcFl9ldlFWorG64cqkhh4+HLHPbPp0wRawTBoDE3Qo47vIM6EMQBlfJjZrdIO5yXfZIlgvDjONDkSvLV8FDkvYbMVcin+pDoLSx+L3D\\\\\\\/OtdHy5RMT7qQeJzqWMMrNjXMdLdjlsg8SBLAuE3TyUV3ZcZKEXUIZEcg8XFoOdLIXKkUiM1Eg7I8yR8pom33vNiwBgmhP6uiGj6HnZiGYbeZ5Y97bQLPKPGVlUf3lpHQzi32EehbM9Sq\\\\\\\/mgE91JP2a58YHGz9GgA6DzlANr+GQ2LeAkgttU+7NQczLbvumVFCJ4f58Tqno+3Zsxa0evplHO6hS\\\\\\\/NZSY\\\\\\\/gIrbZRkTS4yn\\\\\\\/nWYLxzP6K5g9zZlsh+Di9SNgtln7tEc4ohPqK+FuzrMMzg0bR2+mWFY6ydqLB\\\\\\\/e7KpIq4Um0eI2yhZLXQnfz1hr7De4q2wWOxJpQMSasAgTb6b2my2xIyzKtbwxDL8zcqmP2K7Icv1kbELX0rxH2vbWgzWC4aLmfj04U3itdKkYGtf6l6tjIxJLq3PeEPbk3WtfPkeTjHBGCpRDtdFk5zZym6MjDo\\\\\\\/m+aVqj+ymuJ8OHPin8VM2ej7XLjDrCf7m05Qm6Jm5KkWiN82CqdTnvb5uwfm+n2W8SX+MRVZ2gG4rwtWEHyHF4Wfyk4OtLPr3kj94UkAegOH1b2ZC2WcCHMOZCyodY2xiOxgEZ0AZsKaPhxdiRt5iWyUi2rZMyi1cJNm9YFxqZMWwpwtiRmwhxhg4c+nL7yNx2cLX\\\\\\\/w0xNS1waAg70goy7FqK8AzUmfzap84lVkDLD63Kc0PfiAH2lqAOF+6cj30YxbbQNljNrj0W6HrFMYGheMIUlvPzdsAup7R5fOPo6a0g3np7UFlrKtWsR7TSfRBPJ663CbISx3x+CD4RuRIRPy82dfCvSIO2hw0\\\\\\\/+7ZhCB++ykISjEucxK0xX+HAwJRUPclFj8mIT2mImIgFnKWfcszApQ1De13VqKDPiLsiK53x1\\\\\\\/snreKdZrd2OGQ+4b2sSwhuK083iFI3FrtgCnQf+1zcvS183aLU573Yj0q5FtJDmmK\\\\\\\/N5\\\\\\\/IBvBiERKtrLAbugz0X86ou\\\\\\\/ehnv4OMqcPw7nmC478itVsefFsUNASQsmxy7StZj8ozRNZL5pFmJeg3QuuQctJ6SZdrqYjoMMM9++TTLSx6hjhjszux6dwR\\\\\\\/ZNgJ4PVJ53uuHphbFSZ0P+zN0bHvtHEj9dFaztkC0TfysmhEiOiYeGBC\\\\\\\/3qfGvdZhapZAoMVCY+1KsGTfzDsh5nA3bFlCCqy4yvDEqc4OojPaWcjTIhr+d4pdHCjmldwqGvJwBKLy7yHgP0m2aXilHeIU0IAq7XQJQM2TvI2rNsa67nzdD\\\\\\\/9wr0ZwnfXfBLHZkIMn\\\\\\\/asyX0U2SPTuchpePKF8oKlrogfA9NqPTlDIVmdkHfyMvvG07hm8EJv5iVob8StVXL8hBwRHO60socPhH91VE9dJvi8auyb23Pbukx7n2Lpj5fT7ogxxXfu2UZ7OLwewdwbhr0chLH1+j5hV+hLR7YN9H\\\\\\\/R8OmpnBR5uYRPfwR9MCvJGhnyr+q4rxZmpsvhhfXMBAuiOCVoeefYGsILBn5wK00ZK3fu\\\\\\\/BqDHmrjuf+EKvba+G1sC0UJBRLCaJmcC31NwjoiKFTNwHCfJ3iUPPw6pfZdK7mXyEHLhZ8jwttBrsA0vTe3UrGIes\\\\\\\/cHzJdVG0G6UJccCA7o1gfE9BLxxTsZO6ivRATRWPM42Ub+OlvCnA1dAZn4TlGFKNPKJ4DuLg5LljKlMuefFtvEw3uBJb94C8uCk8MXrQZ8KEmjIrhH450kcNS4a2ESg\\\\\\\/tbiqIEx8D11\\\\\\\/2JhwHls\\\\\\\/NYoh1AGSNSCuo6TYlmtTfyBMIf0DlPlEWFTnK8OhRijv3QgAH7qfIxWpg4BUG5Ldf\\\\\\\/w3nAJHcCcp8fJkIgvslip4ZKa6qQl7Jannl73HIfWwO3lkRw9Qr2fA8Dz79UdXEdSsaEU5AfG5TDmweRGmpVUEVkI\\\\\\\/qRTy2V7d8sD2XneXNA0QoPq\\\\\\\/U+A0ihIvBnjpt5SL3Q+JcTWR1ZqkhemwhsQtD7F7TZUmkywYckOKalHVWCc6oo+TpAIntv9C+wbBDjCKk5L\\\\\\\/OmDbAKlGgmEPVkKaova9ID2TXAJqBKeme8h64x7G3g+hJzk1TzvYk0tXcd+WBX4zMS682xa5CZ1\\\\\\\/0E5ecDlFMwTYMaiNvbw0Aa8lGZC0cyPp8gymwbRjA+2\\\\\\\/bzyCwNdc+lyWtHKfBp3e2XrNpFchUnBK8zsSqyEX4ucGePD9OqdQix+K9KpdHN0EQCMxCJeKTuimEsgfHJ\\\\\\\/e6BvWbT+H7PzszKh8yN238mKkKRnQCOFz45HZmi+rvLlG3HTjIBjtc6Nk0ItM0AxnaZ0yIax9RvRDte7ViXOxMHFHNHqigIzU4osg31vmo81OkGu1qy9Xsru58Yc+Lxs9VbbVBJvLLJ0E8LVc4K1Snw3dYvSRWClUY1k8qSgtPLBxfAY9ePtFQ2sW3nzLaocGy4fv5W3VLmayfAM9rKoR0UQF65U4g3NoqjJN6h0BKiIFV9tNkAYugJ4caewBa0Uv6Pj9t5uJCpPr7cxHxxH639NRD\\\\\\\/E\\\\\\\/8LoiTs0jQkW2Q+Sz2E2KaUoXAjbY4K7uVkGR\\\\\\\/vj8k40pnJJDHTQJXWasRR6gdylMM\\\\\\\/lu3N+GkbcqvX7LRvv4HPULeYKyvB9\\\\\\\/TzYJbPW+xm+nQOsERB0ZNmsajbUhjgRu51UEL8INKERpzi24rETYU9juhyhSv6Tku+a3eahpYvjvyn5txOSTAtZBjk+zsKHJG5Y+7xBZtDcQ3aEp9QCVmRmfO4Nk7OPuHrPaAZxoaqmrmdUfoyarKqPffFs+VtpO+D64sDDINrureMi70zMGXePMVgIOGUKmTUyHwYmKw+hhCzQwzF3I+DV6G\\\\\\\/k9BbzKHXYZcNuUWrJvFKCYPpk942TrCkNNbcMQpBsy6Sx6UtFKgdvSmsYr6xwvl5wM3r3GFzQH45v+1Yug9rRfekcmCY0epEz+PmDleihxizKLh8vVtLUvL\\\\\\\/P\\\\\\\/cLiLWQWMYViVCHFekJf+N5lj6U32QkZ42Hi4f+OEHkaheJ97ucafd7lfUiRvTxrLJh9UXfIMrRWfQq3sXDtnSPUkN\\\\\\\/lDH3mr\\\\\\\/etKL1oVD0rPsL0YIBU3kGyLvQK6LMKBXNsbzLTkszAdba4u7eGk1UIRqyo\\\\\\\/e5QP5ZfSq847ECptmVJheLV4w53WDcJf7Xc7lCs3x1c3xzUbhg1GfMRa6EKZaA7xxv+4HoWPhMxZLsRP5a1qEbVdNaopNmIrkZiOIlnfAuk9ujGcMGQV\\\\\\\/YDr6H8QC4QrwOJllG4FNq7MA0Fj8qUBlQjMUuJvn+BH48qFGrt9DRKsrYt+iTZEJTNY1Ubqv30Iv46\\\\\\\/PikiTO6Rj491QRZN6WezDBH36laI4EephIRuKlYz6ZfRFq9aLBF+8PrmXXhyIjzBnNPnzgbUbzSAziAcg+az\\\\\\\/\\\\\\\/t3lW0UdcLWf2HlpbWp7o+faXhBA+HpawicOGRJ8LJh21ScoIx6qaJh8lYQJDVbm2cpUMtbwESN\\\\\\\/Sb8Xcf\\\\\\\/1RRJcFlGFhKv2eQHYJEnh6k5okKf10HQE7z+DcJBniF71PIYe7H3f2jzzel043Ba824PBHg9IoT9jVVxtXihq8+tptD71buNYOXhRpm4TR4m+TEm4J2pjHhl3kOV06LHtvQR4EkqwpKVpQiKh2JMV7Jmmm4QLEBO3HUwyySmBpubIQ+E7vlpKrgB2B9lvRG9R94blsZYe1AnNztKjgypiLsGSrzQDFs9orV6YwuUgRy+B7rv7cvkWRyK9FBVgtDbXTGNF+8R3Ez8Avav8zFBkVDHo3y8EikFUVy8EpWQIczXBGIuwqJN0bcyZ5yleRiB+7VN94RbjO51PSehTLmckJVrbHbKrqh4w37Uvb6g+OVzBu7TaH\\\\\\\/RgQtSWMmnb040NGH3+Avy+smGJii\\\\\\\/6kMQpFDiXcrxABkr\\\\\\\/gKLK2VRTe7BqGwY6AaDtjCxu5FAQ6nap1WuOzfvzfg3lQJfBQq8SdO4WzctJsNuuVfRThpwJNB37Dbxbx8Fz8\\\\\\\/a9FrNyQfkQ6OdLX+op6RwSZiNCSJN2+oiS0UL+e3ZqQjlZeWSZgEPEfFeEklfhZA+0VC9952s3QKSo\\\\\\\/YWhnAFbgn\\\\\\\/S2UwMl4gwqbN3l4zrmAXSB8pELTHNvHbZcIavraxMY+v6kQiIfOmWlLXJPRynq7uCTQ0GFlpRa0Wbr9bMi62Hbql5rAHoiUKFHDDogBMSebMa6K\\\\\\\/kQd\\\\\\\/Vld9R9ko4usZTkLMhdP5Ys5viCXfSQsCeRrWN53+3ecylxaYVyNqeXTviN4CBFgVXChqDrRsv3SWpZTsu7sZHZQJQaeA0TXSYnByiaLPvm7CLuZQ27j5+onGA8Sa1psFwAaoUQ3juCvjaXOswW7fnzntwOsrNXiIGOfsOSQqFDEMqZLKK3O+nZWs\\\\\\\/qXINuG21aRJnSaPbjcqT+9T9YlAj2SS573kgjmwK+Bl13cn5Z\\\\\\\/WrYbUb\\\\\\\/11nHlWMOi18ONJQbnhURyxLi3GCIfrhBLFwbsKSmz4rwDgb1NSwXvyuTTBlYpXvcwWo\\\\\\\/RhLOoACtwf4vHvOA6rXSk10l2Qrx\\\\\\\/QVaDWRcw39hr8gmwmR8Ho2osmdNLVcC5NboDcvJ+H\\\\\\\/6XJdnILBVhuaSsi9uucrnbMLy0yBCrhG+wRTirQJX3wQ2H6NZjyANIxxbmvT+yabQWVNGlABmG8mjZ6+C\\\\\\\/XHqBs1cghtXaFrtlrGdX2urkFCaa9h+ikbNYh1RHTRGUOCdzrJZ0tAsc7FMvh7pkow61w08NciuYeq7abQeGITdCJkR9fiMAUEwSatDPUPpquSDuZ\\\\\\\/3e05UlWRudd\\\\\\\/guzNS2jj6\\\\\\\/u6JuBLbX73uIUoI99fGCqMkr2n6hEizAgn+RDWnTlpVCyX\\\\\\\/rofuy3TUPRYUElzR3eSXXbdDyFJUel\\\\\\\/9Fe+835ul1OqI\\\\\\\/L3EiRILrKhEzxs6lPicrd8O7HogsMTFxryH4yvsU0X1JMIMDQo6dA1n8hoIBhN5tKYWUF+OUd8Hvl21q9GGaQedx9xF4dzLLo30q+UsFmElTcc7oxCEPfmGjp63akWBwawTOWXZivsKO\\\\\\\/SuPSoStWgAYa2B8ZoEiqV156H9vMvwqFDtpftQCzIAzOEWftA6DKtPMpK959i1pAe4lFTze1LxYxjS8QHpLsDFYSsP2nwHgdhwDpcyok8IogXn8YX+kGUPZNBHIT3vZmF3F+e9kCkm57nHLabN5K0uMuZVT1H8hxicTTmH5vFNBcVAxQbhHO9RB0XEP5LutF4E7J8wSFzPzLZoq0clBp6SJ8MeiGLVKPAWpIhdUtecXrz4FL4jNCep+8npOqws0RjldFy8jr2Zs48+rylO7rVhpnVSaRG+T8uGBeVI5FZ5iDql\\\\\\\/hadigRJRxfpWapJqJ1xZAkAhvx720K3E72wqVfAjlGUritpf+rtrLrWc8VJFbtA5aKkZuZeRz5GwFAMQUoSx5VoU\\\\\\\/tc\\\\\\\/UGJvQmoi7ZjZS\\\\\\\/B11YL2gJ3wA3ijn5eg2umtYkGbNWyDpG37NU0Gt+Vk0w\\\\\\\/W\\\\\\\/utdySZodMJPIx0U5gY+JgBPA9DbGmHE8Jokt\\\\\\\/e1MCmA+eAcWcE3jvL8TzSyUYCPK9Uz9WAvTLNN3Ng1pegAZYDNP\\\\\\\/HiqJq87E964dBQ0Mqt3\\\\\\\/edJTyVTdqtVXnD1WuwwYCWi\\\\\\\/q7iL9I+2xOJWRVK9+hTwi1udxXRbrn1IxEEvZUQd2A0bkc3LGVCjor\\\\\\\/NcXoZVyY3GQkOYu9MGofjHWwNfE0sDG\\\\\\\/8GOZwh84KxXFMVKDlGOy8T9LsSTBX+tYssX74EZrP\\\\\\\/S1jXPJQ29BqX6sDT5YfAdRVGyq70+hGIGzCBNec+kCmiVuFGPq1v4s9Tyj0Ad15xdHr4YlNl8AlfcMmZXKUTDHiJFzzMIsvjbcmtYOGOAqqnCajHTxhYjzLiOakcZpMvpzDHHfsh6G5gMqGVLbt+7BULcs3wFPZUWuihaW1lUMyRVwrLLsoEixnQhoyPBtmtBZ89pJrCiO2jGjxKcUXaRUMwpub0JChl7KwZ+NuL3rkSazlkoCtXViy\\\\\\\/wCG0Fuzh2CMizW5JlUCbAD1GXILaqQm3bIkEnWeles1YDHh50+5H7wgYJvJbwMaSoi+VP25C4k8xP+bu2m2zVGy58AhCcMXVtGJJvLRq1iTyeb2\\\\\\\/rm33ki9ryUtcD73SSV75eegJjl9ZqhtKNOuWprZ2FaWp\\\\\\\/+Ytf+j+foSGIZgiL8xuPEsXZ1oSQoPlx10MMvqN\\\\\\\/Gsgq\\\\\\\/lhyFbjoAwT2ejwl2VLuiDql20A6UBZU4riq7oaaggcrE6PmSQUEVBUh8eh3fluHeGlPliC9GO2Jd33\\\\\\\/yiJCfK7rr\\\\\\\/Xxa\\\\\\\/jB+cutj8ZP2Jfr0bsnN3UTYOiUaErdcY8i4EKN8OlTJ2YYw80HEkxw97e4z8T61AGrSNUUDwiPdc45lnOM4dFwygXyBh7zq7WdztKtXgqJTxgomdqBQ4O091Gdv0dMcUrvVMItM6Di6WFpGve4mHfAlhTOo695SVLyIGKLO+2J6dbr6VumkFCSJrYAJ\\\\\\\/LW0OfjYWajYfLmirX1y2kmBAWl\\\\\\\/MitGCimUn2dLD+Ih3nlJrATcCwKT1vl0hBm1UIGFwKY2azuuolgTH1x73e47a8TlOiwx6X8xOu6KOTGAy1Bh7Sma7g7e1WC19GCOdbEKEnopOKRG1x\\\\\\\/A3jwcgrq\\\\\\\/Rz1+uRsamDDoros6Xx61LgBqmt9BAuEflcptS5Ha0Y0N\\\\\\\/xs47YQfARXuipeW8AxDPbRJc5yjQ+GDKCtk\\\\\\\/DRBSmkfL+hcQ2Np3\\\\\\\/6SVgl3+Kd0WVALWv26nH1vVwE9Eg3TTXVrigzaoblAO84MEEdwgqEYzZ9Zui58amQO5om2k+e6fGdg5+0UIGLPPMaiE75jb7lvmR3C8hTeSPYMPg7F5wFqv8TytQzNb62ZVCjeJNVGqGHuUyVtqAT1Wo+OiSgNMY7snglPSQw6bbPFi6kRSAAhHsY2g4PC2GD2NGmiDUIJzelIi00DgNMk3m+ut86Gcg6kCefEDO2pv5m7RaPc88CzYtnoDMRcPK7Hwcqc61T0mJ+4pxruhLW9feceCVJlpRXiI1OAQ8p31T5d2J3SiVyvuUxYterIpCUM77EGP7vLUhUqrwkv0kBuSxMvIYfWA+\\\\\\\/dYRpmRu4BbQh6764WV2VvjsBXFw7Nuvh7wLJmdNAohdPxHfR4xsYWmMpIkmnUxuXVH81AswYeat9LE5cP6JKK6LJlA5YPPzS3GIKCTgBOfn1v5jD\\\\\\\/MjnE3onEFUpZRoIvvFlX26KW+Eh6kh98rv07vngz1qBWRtWhT4O+WezFMnu94s4CSCEmxgKMrGwu+eteYa4lHFRmmYJWfi6DTSsqr5Rv6eyIJtlp75EEiAIfQaLjrFT+yUOK3ngbf+FVpp8BJxkZ2hqmfwN9bFdFll9tYRdnjGVn2jU3oVz6Wkg5GtVwio1XEUMzgZgLVmd+2ilxzTi1K8j3TUGbibe3mINunAtkIYbbx8d6t7fnffhhaT0qkN5M8smzioHksj1A3KJdy6o+YHA3mXuemJbsByUGOpk2Hy6W+or0kuDjo8wZAS0QHwmW4SfhRSfB+ZfRzSlV8LOKoy6kDCC+0VON9JeFhKHUpRcwT90nssiC6J2Pjh\\\\\\\/ortT+FQRaxWW4Bwko1MKyhdZmHu60GVd5pyjaz5QILiD+XD39OrHjI9bWTrl\\\\\\\/TR8p4sjhy\\\\\\\/2sYk7UxwXNhWd2\\\\\\\/3ygHSsgLberQb+fj1Zx5tKNf331ovAplAZP7GnGO6hxob+KRXmZMDqZJ8+xSLEossluf2VdNJdBqe61YqWUhwwZ0qRFWHjr2NhHkTdEdZPjJdtj+K45IPUZcYDT1GgyADs337wY4vuzcDOjMBM8ltW3fvNX0yAa7DCwxfSpYGrSckaB3\\\\\\\/HS62dAcn5JjXJd4whb\\\\\\\/dowD3qwuEiiqayjg5zJq62HogI+OGGLKYPjaI1aGISoCRzbb\\\\\\\/mBRBxecQ5EE4X49z1AG7pfG59Q+zpvi0WbqkgPP30BFiypP0hfcMzs6866E+9LQnzkFm39ulRfnI2HoZ\\\\\\\/GUsK4WwWNxH61ogrfCCyhigXxtObm5Ps3Cu8NJpRe8byj\\\\\\\/3rMISpQhJ2PHso+ANgxyTg\\\\\\\/LKwDkGDkzcOU0\\\\\\\/80kPDFgyG0XmXGFFY+B6hWZ8YaCLZMxFkOa90dWLHlgRyfK0udGRtv5rwOepAw0Q6IobMaALxhScNHHxybkS8pkTkyjugrx0s72g\\\\\\\/rzgOc6RmNVPlS0C4vCjgwa4WN598PV8XQIUSFHl8Coo9aYEP9SxrvZMPv1qAMr6dIfxg6dvcBzDjvTjxOytnM874Cc\\\\\\\/OAX3lDK7U+0Hzw5KobA1DWeKiVngin1zLbrQAH24xZRXZrddVuIrZ\\\\\\\/K2VUFo6Y1lWJwiA3iIuezhD6NJiBtHC1w3Jb9j6bpAbOcplk\\\\\\\/xK8b2TrD9QtFu4j4OlRmjqyEzGwHCnN3iW4ybFCwrTgmzNxzY+NG4IVqCEHsbya6Ry5LBdm+ywAw7O6n+bkaq87lqAW4krI4Yi3+Xe4OnvCh13D1KKwhSQL25tAJ3G3RgSoE8G8V9AVMihoZtvthQO7l53hifWNkjMTXmTSyC+qw7l6uju0WNx3YgrtJQZWdOAWUHlA04QDg0ppu3YPssq\\\\\\\/M6iMSN4oMEIrqMzSC\\\\\\\/I+2hXUKFjNcKgFa\\\\\\\/w\\\\\\\/ksToKNS+qNRMYCdA5a4yJGN7zbuIzISrxMdNf3GIZmwEQHVfLNjk8shhCNeNf4MayLXtUqf1O0QJpcWCNgvzRyyWpxNAuK50PIid4bIs5ekoPc6SyLLm8DRuOJ5zKu2xCzJEDkRDHNopipsS4oqeV3q3guKJ6MbWNqXjYFb9DJGH87pmCBKUjCeff8DQ08F1eDCOHTJMm8Cs8IG\\\\\\\/\\\\\\\/BQCfxsJkMoJGmDEz+W6cBX+1udeojD4hWrOA06OqRzn5cAi9OStdY2zSZtGPexVRSghIVum7ZzPCZmtvlaVGgvMrZpkhA7hz94gKbfaeIm+UMlqJVRHYaWfkpAjFYkUyNZn4CFAl2J+vEAfA5U9Y7agK4rE1MnofoJTI4RpeVesopWTfzo1G3NRff\\\\\\\/sorp97vRsLcRC773gKwn20T5ujY0xgAqAEL6XCpUJuPkGJjNUrrNv2QISKkHOlbJC2vDmzzgzTOVwUoxiw4i3LyRuVXlD5kj7aHC05dR5PZLMjsqJ0tVYYh2pHX2oi9GqGr1mRKJLmDdYh12mjjYHmn7Gl+LTlAqQ\\\\\\\/7uRG5gEvs15E1R8faNxjPVgXMGW9HEVMAnDDeGix8+9avN5N4ao+PCdSUV2wl3mUfuitTg1HGRhB4eJQF00bo\\\\\\\/EDiy3KGJRLMPOAVYkUcstNM+2cH1yrrVpfVasKtwskLmL4z7ZaA6hx7p\\\\\\\/8ZyvfXciIjF5\\\\\\\/jdcI8ltOOSK5gW2XUzAst9wYKmEmE\\\\\\\/K3fnkfdDI8oB48voIm6XJwhJdUL84OldLIN2Ja0eL\\\\\\\/jETCOM3Z0DQ8ui8l1+jo1x3iJpFdbOEMmwpMmm2ndGto8zdlLmemTcCl4gpBTiZI6g2\\\\\\\/zNhCZeaXOvRzH+zrC9zHc7g95OXu\\\\\\\/YUg3si1+Pu17xaJUg0q0Nb1XOMeP73pfypgFkcBY++KQMbnkNBnGSJFvWrTrtKqg6rE4dWWfU\\\\\\\/WTQsJpYFHZ6+fCu1jx45UCStVkjBXAYRORztgJt8jPkrA39n92IbflrPfJfd45zTAGTlhZPT\\\\\\\/2FSY3j\\\\\\\/gZKCpfozThtZuAg60L7RqlejZTaQ\\\\\\\/1Uab+l6RMOkdVpswrToKaWojhn68cTNZs6aTEfWF43qRZ9FclJJ5P\\\\\\\/\\\\\\\/hymBf8AYRx7rMyQ+u9Zdpb56z\\\\\\\/HGsYvpxllNrPoFo5m0tHh8vZXVKDoyIIAf\\\\\\\/32LdSlf9TjVYoAyexs75wXdKbf99jhCCCaykz5xM4DgnACLZgxOeNkCCdXKW\\\\\\\/RnZwCsPqRTedKlF2daanoBwreU1N9PLUGxRNWOF1l5d5OCXyfqMY8f2EOcUykYjZr7a+KWFwo9I6fmCN13qAEVjZizUPO1d5bOOaSk1RPDQgv1Mp7MjaAM7Vm1QEr\\\\\\\/v5liKzKsM+NwST\\\\\\\/MU1DStRVEQWCtc0W3hX5A3pyevw8j\\\\\\\/qKyQG90Q6phz4ls9qxwnSWeQoMxxk5rZY\\\\\\\/Np8U39wcSfgbobAL3vg40uRVyQrI\\\\\\\/QXdoS0vwqqQplERzbrISTBfI9BB3N0eukZsegDPsXxv+fUrFhd7a0tzEFSCVZ1SCit9pXSyrbOomVorrOAeeWDrt6LYS15YC3\\\\\\\/SXoNxzWM9XxlLOUgb4aFV83S48D8Sm8axXL98NSoVw4g+syR8sd+v1\\\\\\\/J2oBTOSw+4ZJMYjRf5BvKdaZvHBKCyZMUZAyqjPZeRpBLwdmRWR323lOYKGjkZYDcsMFFXELIyizgWqz+eBgv3Ql9gmVj3CN0Wo8KHBqYF6IeF4bdk1B+PYiyoD2AlJ+X4WCqOKH9xqfoCAoeJmn8rqRO6xhJwqwIDkmgcyPjL2OiiptlN0\\\\\\\/sltjTQmssH7FXBrKFVLXIBu0pYo2xubOdmajSCGqAFrmEReq\\\\\\\/Zo\\\\\\\/+YrmnmZpbAVek\\\\\\\/j+eRWAUdikV2MHpLT+iG3bMNE\\\\\\\/f7e5IlfyZ+6y3ookoaKjr3joD0pdEpdUXzZ2yVrGkNHnkdfQ9cjDVC9qaBiJu817QRSu9wzXj49K49fCTSYj5VWe1nZ1svxme3L52nrsVad5rgwftJeiuC7+Af7oCkzxmLg8k4YBxZxnxVN7VagEm3GGc3f+IDFnnjHozoyKJr52LwVMHHDpTE8LX1ukbQcQKgu1K8mPd+OzKS7QGx4REoxsgwBJjGshuxiwfSlgdQcLb5bueJziIbOyu87GkcPl3xewNxbTuPD3MgB0q6sSQIksMAUwfe\\\\\\\/4ga6M4AEnJZE5GvDzDkuf4wwM3H00yPBpf6dRCiGWAq3dikbYiGjH\\\\\\\/W5KH5txU2eWrRMV+85mnqikaWfH6ttEhsiy6zhUuVtsWktrF89wVyxMV2XBKwXIh1XFo0WyW8+Dmcf4kQpwGrJkYkGsGP4HF7+AF+sXrQcOeZbmSIGxHdyoaW\\\\\\\/eHZYqw6KoSIjrGQR8FI9haEY4UJj+jhJN6l3pqkzpn2PyuIzVtQhZFv3SLc1ng1LnYcyjKQZZiA8cMgsDlTc+Nmx4yaYeCQ2Wf0xTfLCVsFIP9EHgBlO+zj1QC16m4MMECQt641Zcy8QArv2oxFRK\\\\\\\/ZAY9WBnvKROJAE4MJophSBZkcJOO\\\\\\\/dy83wXlLPREFmhykSNOAgaFyN1RJ8qEtVC7pIRYdLknTo5cv2UPc6k6lcoyMfif77Sqpxk7u5Nhx969FMqY4VxXw52LEvAim+CEv1\\\\\\\/zRAXOh3O8iCLxWPVw164iSv6VvqBWfdxbD4lb5YNanCRIMg1Nzmk3B7SDJAiQqr8wnEpj6K2HXHxUz7kptWXf6i7gMd6g34vRDyqoNPP6yTXF9d+3H1q5Eckix6JgV\\\\\\\/3VMh8PmROgvdbF51w191UAOnRkjeuhia1+CCVRB\\\\\\\/AKaWLJIeprfPzI7nUnWum8F0Yiqnh1DZcjF+uVIFPcoMIRWaQvcFVx3yxMqetVljfdFBsRojT\\\\\\\/jl3piqOupZvQZZVVk8kH\\\\\\\/gyoZuutNEpX0ylDgVwQf7zOF6vw\\\\\\\/jwVE3V4Hy1UxXkie2vDEU4lIlyJyLu2Emq+cKGgcHaA3opEohBu790VMXvny8FyG5S3uGFQUXA88EtHTi6nZii\\\\\\\/7KEcpvrTgB6sblFgDrQZkv40PzJMZsLjGHKEC1\\\\\\\/Gwgwp\\\\\\\/cgdudpN9gF5P\\\\\\\/8eF+KJa7p9dXe9KrILOrBTh1BLtmsS3kPU9II238CD5FJoit35gRWSZFg5J513\\\\\\\/GXwLHEy0oW0HfLzcyOZNZqMBMPDClizmznNtyRhNvXjYGbixyKnSauHJ74dV7VvzdN7UfpSxIsC\\\\\\\/MG528P05KawlFy5mfX+mu6zX42iM4KeOuqKZXsnsrYgSRr8ggDx1IJcTdVd+SN8T8nudfV8G1kJ37nL150xC0ElFzg3lB+bBOxLplcSLH56e8vhCEY2uMEOoVGN\\\\\\\/pyc8EPBlLEWlPOri8lgj\\\\\\\/S0nw1qJtAQ937VbZ2X5woqgLGLoHgNUir7vJhsTxpsueOJiMRQY3INk7e2joUmUNfAoK8GBO3M6u+9e8t\\\\\\\/qB2b1cGouFPLiHlQB0jFjmSWl9xj2Zr\\\\\\\/EI\\\\\\\/jOoJP1F40tC\\\\\\\/PTh1tiddz3GlJkzZ3dyEw5TrUYe2eaCxvAoc\\\\\\\/yFXAOmCalPwi2U2lIS\\\\\\\/RiMK8+RbbGMZDiOxiFL+RUoyV7+gPVnIrSmi6+WbD1yM1F0e3GS43w9tFaJFDpvN6YvBxRwctVmkO8VJrTqj6pyRGGEAZLxaAxBV1+mQ4uT0SPPRjd9YA+pgMtEx2NwmuKdpe4D9JenPU\\\\\\\/vdPLjVw4cQC0J0\\\\\\\/ndeFoNe4TyqIB2i1kl3pXExo7nFGEOutLQaIBSA8ZNq3G\\\\\\\/VNxGWNt8zFRAC96HaCYvCUucVEv17ulC619WVstspzjqlSoS630vKZ\\\\\\\/nQ2s\\\\\\\/m92BnwxnDnPBGpVOiKgXEj\\\\\\\/GrqV140VEpq72JsFHu+OTn6\\\\\\\/8iN2ZOsKopeyZWRt8zYpp6ADp2pNm8Ni4rTPLK9\\\\\\\/xmkSi\\\\\\\/W9uVt3fvHkjxwmzGaz+pUXwn23pM0LlPv5P7wnOeusJV7xvAinqbWa+tzqtWihkdFffd2srsfg0i3HSjRef4GiYaIN4gJv0nihSqDIFcCq4GgHhVxG7t7zuO8DJukdUyrFxDLmzoHx9sTXF2gA412HPWof9GbVKGY9fh5JtEwPVS5J\\\\\\\/mOY\\\\\\\/PLFRUioPjxFEVCuhNRWiSAeSH6wsT8Lsyd+MDt68A8mzM9ebrcykgdWYv4QFog7nrgq4T9xWzQPMeZB8M65AFIbmIlnaG616rL\\\\\\\/GnX8ZF8NNapNY2+WVF\\\\\\\/dR\\\\\\\/FjUjH5fC1t3G9E+CZJCcW3VJhiTbAy\\\\\\\/oDVvLvv3oFeC7DFArCHE0g0FMYNuh\\\\\\\/8pC\\\\\\\/hAt3XV++8N3QQ+KnAw2dwwxfNiT0AdYDhj1XgbI71p1zr1jsxsUFMhrkhXei5MRY4HHcOZKX6CHowfjfkV8mZZgVFumiScgWdIAxka3xR8hY547HB1D8+snAohK3fQdQeazuFwCJm4b\\\\\\\/ly962U6lIv0aGalgM17Dh92QM2dy2KZ\\\\\\\/BtDYbp6vkmG+Y7U+ZOSrjCvfw9P+JglCKJcjtgUKLdXa3SUmQFcZ5z3r3zijXTLYkScxw7\\\\\\\/QW8KRJ7xlcEWPINxD3TjqORFOARBGVCIF\\\\\\\/X+LVWKqPYZKczh3IqASuKg0lW2vYMXKaqn8DcdtCRMYSfihPZ5plZ15ku2aJPjE5+IsP8uEotMDKH0C5YauBUo9xACa1iwrc12FFq9XpJfUaUUuQB2kRIK6P8YsPqTHzOWKO1jg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ly4P04mqvR+l7ngeBeh\\\\\\\/fg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222c1c2b98b830afd9c3f595aa1707d45f3e2e510a1fe883a10d40f754d366bd6e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DOEUcGfCaSMsd06ek46\\\\\\\/4N7xHFPdMKU99ceFeBvGH3nnxnkNmAhtC4BhxCBNCpAghR\\\\\\\/Vcmj6g6Xj\\\\\\\/R7u0hVWHuxYvYu6mRkaZX9bzonAnN\\\\\\\/CDHgcDnHa+iZIZGetfLyy3KLRUkurxD2iEEmPAeQTi0dvbjZkBayyKWEFyCw0mCzNY2nEtrDdP6CdN8i\\\\\\\/nerjzqsmv1A2aSV9r3qHje7mgieU67GpcVWZQ9xHb40yf6A3yFK7xmKJDtkCvO6AQylFpTmWVaiDxfP6F2l56+WONCM4uklRLNxau\\\\\\\/Nz4Gw0krc6dKhkBjT7u\\\\\\\/IwcrW34RX1oi7TbcsYV9gH8b34pCDg0WyQN6MEL6Cl7PyTFrLe\\\\\\\/\\\\\\\/9cpiM\\\\\\\/i7b9CjMeMdCmt2MALjvwM4EC8YA73IcoGFWzil6qxnxM6FZzh1A4D2+494wBpfek0rAq3y3iHPHTTM+LxCz8zeCwRvzqv7qhiZoKPrincKPGGrTCYJ6BIffytmHLC4Gcq3IMaonmgnLCW1hlTjEmlTxS1MU0+UdXYMvWr6OsZinQvUoXfWFPNbzrhbu4fbgU8PXrM+PMlbPSsEZupD68Rd5ZmGk0vnm6XWpswcjzjohkzZDeGPDqK6vX8XIWH73NR1W1kT\\\\\\\/Yv1pKsF5tV2lvQUep6Z2b9qp0pzlUR334\\\\\\\/AZViLSR\\\\\\\/6be4QO4KOhGMm6B+jsSXCNiR4y32TutcXMMlVon9hrBEQmydo\\\\\\\/hThaZ1qn0rtT+bDY1mQlgV68AxF5Ix250YPyfktk\\\\\\\/2kyakX4WdWnYsQyujcrm1cOq4M1Dyg4qQOw3jhBSCYkPZyM+t9C6hAJS2sCgqLoqqwd\\\\\\\/mlbENJRYyOdzAc4tThvEHZi\\\\\\\/mdtPVIDgT1UJcxul1DMLVnhE8aHG9zYiMe6WMLwSNCQHJItIKiSp25zIoi1h\\\\\\\/H2y4XzAHQQlrCKcUKtbWN28AJ3E3sqApxHR0R8lRmArNEYYaCzmEHFz1i96Xmsfxl8HpK7oRI7AJeWAGw4K13tRLzXqC+SQayxqM0bkzsBk\\\\\\\/4T+KpHoooM4IDC4rzceL7qgWkgGjWi5\\\\\\\/rh+emrLRxC5ZQPALfWvoBl64jTmS6lLIK8K+TAEJSBiW2chsoYXJxxHSRR28TySV9fd9Sd9lqTdu7pG2lReBwO1\\\\\\\/d5PrHgmDvd0Kf+dhp6jW1u4zMFl0HwG\\\\\\\/uEA7ZKDwD6LYDy9v1iQ17sMfLEn+htOYXca7UZEfT6pdv259xnnF0RY6o\\\\\\\/J1J1wn48wlxAmV8qNrE5VlCB34eGTMvVOOsjzxUqxQBKtgXGQ5g9ZY0Askr1UPr3HleOavphJfQsdqViFHjfutfgzKkKwUZO\\\\\\\/7qByn0R0ikWXvfkPPilavSk1x3KXf9hc+NNf5M157iYspq7YcHCspQf5gmZmk7zBv3QlKih+rduyBxiRK4LxRWNr1ZeRng41BCII6O8rqSYDcDa\\\\\\\/H1gw4jg7p3JzvU1s1R1+cOLkxejF15LGkDUntQqd7SCkDLuxzrJpsS0rqxm2nU2Pc9IGYXLvNWOh5g\\\\\\\/g+SdK2vATDsFFzJtwlon0wDYtIs7xk2fqcmwAgJsa0kxn6GSFMQ0BhYr+hI4U+MT+dOM+IGVvfMptdJhd8hwAU3HXZPuveKrQbibnvI8tp6gDhCFOE2iBm\\\\\\\/Ygq6UGb+ZC+tNVCpWMCESCDLARQbbiVGAU7x2sahtzDn1oVI6zMMsBXJe7l+CDatlbno0lTyfT2DsTU0C+oTHX2\\\\\\\/+XrjPpJSPsf49XPV5XMvRF2ScUcBuHGZGjkk7lZ7S2bQasWTH14gNz6CqhHbvYlJDlF6NOzPTTI7ITsSoLZQME84mwI+Ib8bS5AMp7iWX4zddR7Neh0sbJIadr9YJeymOedFbD4o3mNAARbHSB6kTWuoiwfeEV9F3uPpq+nRc1gPER1343XwOXoDODpW251RfvR1dzgDgs\\\\\\\/iO6Dtdj45QQWcdYNJNoEWKPDYWke+CErQVpSM54SIEpL2AwG0QXaDJe1VN2qdruF5PILtL8EQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022u4dw5x38K6wUDnpeaXbYYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c5b128987e88acf29ccd6d443d31d33de6564ddab4af45813104300d6f7911bf\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459409 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/359" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BOCRhJNbADlh1WCJX9tQbabLxAt4jw83Lk7YxLVuGWk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/359" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pIDK1uxZRi6rFIiXnsXteMpUetUi7J0CtT5GoRfvysM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/353" title="How to Design a Genetic Mating Scheme: A Basic Training Package for Drosophila Genetics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/369" title="Functional Analyses of Trichoderma reesei LAE1 Reveal Conserved and Contrasting Roles of This Regulator" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459409" id="node459409" data-pisa="ggg;3/2/359" data-pisa-master="ggg;3/2/359" data-apath="/ggg/3/2/359.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant <em>Centaurea solstitialis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katrina M. Dlugosch</span>, <span class="highwire-citation-author" data-delta="1">Zhao Lai</span>, <span class="highwire-citation-author" data-delta="2">Aurélie Bonin</span>, <span class="highwire-citation-author" data-delta="3">José Hierro</span> and <span class="highwire-citation-author" data-delta="4">Loren H. Rieseberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">359-367;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003871 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459409" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrina M. Dlugosch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrina%2BM.%2BDlugosch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dlugosch%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatrina%2BM.%2BDlugosch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdlugosch@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">kdlugosch@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhao Lai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao%2BLai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhao%2BLai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aurélie Bonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BBonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BBonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Hierro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Ciencias Exactas y Naturales, INCITAP (CONICET-Universidad Nacional de La Pampa), AR-6300, Santa Rosa, La Pampa, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BHierro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hierro%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BHierro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Loren H. Rieseberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Loren%2BH.%2BRieseberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rieseberg%20LH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALoren%2BH.%2BRieseberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/2/359" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459409" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459409/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/359.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459409" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459409/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/359.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459409" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459409/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/359.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459409" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459409/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/2/359.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459409" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459409/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Transcriptome sequences are becoming more broadly available for multiple individuals of the same species, providing opportunities to derive population genomic information from these datasets. Using the 454 Life Science Genome Sequencer FLX and FLX-Titanium next-generation platforms, we generated 11−430 Mbp of sequence for normalized cDNA for 40 wild genotypes of the invasive plant <em>Centaurea solstitialis</em>, yellow starthistle, from across its worldwide distribution. We examined the impact of sequencing effort on transcriptome recovery and overlap among individuals. To do this, we developed two novel publicly available software pipelines: SnoWhite for read cleaning before assembly, and AllelePipe for clustering of loci and allele identification in assembled datasets with or without a reference genome. AllelePipe is designed specifically for cases in which read depth information is not appropriate or available to assist with disentangling closely related paralogs from allelic variation, as in transcriptome or previously assembled libraries. We find that modest applications of sequencing effort recover most of the novel sequences present in the transcriptome of this species, including single-copy loci and a representative distribution of functional groups. In contrast, the coverage of variable sites, observation of heterozygosity, and overlap among different libraries are all highly dependent on sequencing effort. Nevertheless, the information gained from overlapping regions was informative regarding coarse population structure and variation across our small number of population samples, providing the first genetic evidence in support of hypothesized invasion scenarios.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/normalized-ests" class="hw-term hw-article-keyword hw-article-keyword-normalized-ests" rel="nofollow">normalized ESTs</a></li><li class="kwd"><a href="/keyword/allele-clustering" class="hw-term hw-article-keyword hw-article-keyword-allele-clustering" rel="nofollow">allele clustering</a></li><li class="kwd"><a href="/keyword/454-gs-flx-titanium" class="hw-term hw-article-keyword hw-article-keyword-454-gs-flx-titanium" rel="nofollow">454 GS FLX Titanium</a></li><li class="kwd"><a href="/keyword/yellow-starthistle" class="hw-term hw-article-keyword hw-article-keyword-yellow-starthistle" rel="nofollow">yellow starthistle</a></li><li class="kwd"><a href="/keyword/invasive-species" class="hw-term hw-article-keyword hw-article-keyword-invasive-species" rel="nofollow">invasive species</a></li></ul><p id="p-6">Almost five decades of molecular genetic research has revealed that an abundance of genetic variation resides in the natural populations of most living organisms (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lewontin and Hubby 1966</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hamrick and Godt 1989</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Morin <em>et al.</em> 2004</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ossowski <em>et al.</em> 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Durbin <em>et al.</em> 2010</a>). This variation is necessarily shaped by the history of mating, demography, dispersal, and adaptation playing out within species and as a result allele frequency distributions in wild populations can provide unique insights into evolution and ecology (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Avise 2004</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wakeley 2008</a>). Most recently, genome-wide studies of allelic variation in natural populations are proving to be particularly powerful for detecting subtle and/or complex aspects of population structure, selection on specific regions of the genome, and associations between allelic and phenotypic variation (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Drosophila 12 Genomes Consortium 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McCarthy <em>et al.</em> 2008</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zayed and Whitfield 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Emerson <em>et al.</em> 2010</a>; Hancock <em>et al.</em> 2010). Genomic approaches were first accessible to population geneticists via targeted amplification of genome-wide markers (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Vos <em>et al.</em> 1995</a>), but the declining cost and increasing length of next-generation sequencing reads are now making bulk sequencing of the genome practical for allele discovery in nonmodel and outbred study subjects (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nielsen <em>et al.</em> 2011</a>).</p><p id="p-7">Sequencing of transcribed genes, the ‘transcriptome,’ is an especially attractive strategy for surveying genomic variation at a relatively low cost because it provides a reduced fraction of the genome that is also rich in information (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bouck and Vision 2007</a>). Coding sequences can be placed into reading frame, aspects of protein variation studied, and their general function inferred from homology to proteins in model organisms (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wheat 2010</a>). Coding regions also may underlie phenotypic variation of interest and the associated protein variants identified as the direct targets of selection (<em>e.g.</em>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Elmer <em>et al.</em> 2010</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Renaut <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Barreto <em>et al.</em> 2011</a>). Simultaneously, a genome-wide panel of transcriptome-derived SNPs should also be largely suitable for standard population genetic analyses that assume neutrality (<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barbazuk <em>et al.</em> 2007</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Seeb <em>et al.</em> 2011</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ueno <em>et al.</em> 2010</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zakas <em>et al.</em> 2012</a>).</p><p id="p-8">Here, we explore the potential for <em>de novo</em> whole transcriptome sequences from many individuals to generate useful genetic polymorphism data across their genomes. We focus on a species of economic and ecological concern, yellow starthistle (<em>Centaurea solstitialis</em> L., Asteraceae). Yellow starthistle (hereafter YST) is an annual herb, native to Eastern Europe and the Caucasus, and hypothesized to be naturalized in Mediterranean Western Europe (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Maddox <em>et al.</em> 1985</a>). Historical records indicate that this species was introduced to South America as a crop contaminant via Spain in the mid-1600s and then to North America (primarily via Chile) in the 1800s (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gerlach 1997</a>). Introduced YST have successfully invaded a wide variety of grass- and shrublands in western North America and temperate South America. In North America, this species has spread to 23 U.S. states and five Canadian provinces (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Maddox <em>et al.</em> 1985</a>), with the largest infestations occurring in California (&gt;15 million acres) and the Pacific Northwest (&gt;3.3 million acres) United States (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wilson <em>et al.</em> 2003</a>).</p><p id="p-9">For transcriptome sequencing, we generated normalized cDNA libraries from actively growing tissues in an effort to sequence as many of the coding regions of the genome as possible. In contrast to direct sequencing of cDNA for quantification of gene expression (<em>i.e.</em>, ‘RNASeq’), normalization reduces the representation of common transcripts for better sequence coverage of both common and rare transcripts (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zhulidov <em>et al.</em> 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Christodoulou <em>et al.</em> 2011</a>). We report on sequences of 40 YST plants: 19 from its invaded range (11 from North America and 8 from South America), 4 from its putative ancient naturalized range (Spain), and 17 from its native range (Eastern Europe/Caucasus). Plants are diploid (2N = 16) throughout these regions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Heiser and Whitaker 1948</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Widmer <em>et al.</em> 2007</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ozturk <em>et al.</em> 2009</a>), with a genome size of 1N = 851 Mbp (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Brancheva and Greilhuber 2006</a>).</p><p id="p-10">We address two key issues using our dataset: (1) the accurate identification of allelic variation in these outbred individuals, and (2) the extent of sequence overlap among transcriptome libraries. YST propagates exclusively by seed and is self-incompatible (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sun and Ritland 1998</a>), and so we expect allelic variation and observed heterozygosity to be relatively high in our populations (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sun 1997</a>). Accurate identification of allelic variants of the same locus is not straightforward, however, because allele sequences vary widely in their divergence (<em>e.g.</em>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lawlor <em>et al.</em> 1988</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li and Sadler 1991</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Moriyama and Powell 1996</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bergelson <em>et al.</em> 2001</a>), overlapping the divergence of closely related but separate loci (paralogs) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lynch and Conery 2000</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sebat <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hahn <em>et al.</em> 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Demuth <em>et al.</em> 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hahn <em>et al.</em> 2007</a>). This means that it is impossible to discriminate among alleles and paralogs by sequence divergence alone. We present a conservative strategy to filter for unique loci by using the putative allelic variation across all individuals under study to identify the minimum number of haplotypes within individuals. This strategy relies on phasing of multi-locus SNPs within individual genes, and so we use a long-read 454 pyrosequencing approach to increase our ability to discriminate accurately among haplotypes. We find that approximately one-half of our putative loci are suitable for population genetic analyses, that allele recovery is far more dependent on sequencing depth than is gene recovery, but that modest transcriptome sampling nevertheless generates thousands of informative markers observed across many individuals. In our study system, these markers indicate a history of admixture and the presence of high genetic variation in introduced populations of YST.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Library preparation</h3><p id="p-11">We prepared cDNA libraries from leaves or whole plants of 8-wk-old seedlings (n = 40), representing the native and introduced range of YST (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Seeds from these populations were reared in a greenhouse at 25° and 16-hr day/8-hr night conditions in a 1:1 mixture of sand and potting soil. RNA extraction followed <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> (2012)</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462437/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462437/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sequencing effort and assembly information for <em>C. solstitialis</em> normalized Roche 454 transcriptome libraries</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">To generate the full-length complementary DNA (cDNA) for the transcriptome analysis, we used a protocol from the Clontech Creator SMART cDNA library construction kit (Clontech Laboratories, Mountain View, CA). This requires an oligo-dT primer that anchors the polyA tail of mRNA to primer the cDNA synthesis process. However, mononucleotide runs reduce sequence quality and quantity due to excessive light production and crosstalk between neighboring cells (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Margulies <em>et al.</em> 2005</a>). To counteract this problem, we used two different approaches to synthesize cDNA. The first approach was to use a “broken chain” short oligo-dT primer (primer sequence: 5′- AAGCAGTGGTATCAACGCAGAGTCGCAGTCGGTACTTTTTTCTTTTTTV-3′, V = A, G, or C) to prime the poly(A) tail of mRNA during first-strand cDNA synthesis (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Meyer <em>et al.</em> 2009</a>). In the second approach we used two different modified oligo-dT primers: one (5′-AAGCAGTGGTATCAACGCAGAGT(T)<sub>4</sub>G(T)<sub>9</sub>C(T)<sub>10</sub>VN-3′) to prime the poly(A) tail of mRNA during first strand cDNA synthesis and another (5′-AAGCAGTGGTATCAACGCAGAGT(T)<sub>4</sub>GTC(T)<sub>4</sub>GTTCTG(T)<sub>3</sub>C(T)<sub>4</sub>VN-3′) to further break down the stretches of poly(A) sequence during second strand cDNA synthesis (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Beldade <em>et al.</em> 2006</a>). Approximately 1.5 µg of total RNA was reverse-transcribed to first-strand cDNA using these methods.</p><p id="p-14">Double-stranded (ds) cDNA synthesis was performed using Phusion polymerase (New England Biolabs, Ipswich, MA) with a hot start of 98° for 30 sec, followed by 18 cycles of 98° for 7 sec, 66° for 20 sec, and 72° for 4 min. The ds-cDNA polymerase chain reaction product was purified using a QIAquick PCR Purification column (QIAGEN). Normalization was performed using a TRIMMER-DIRECT cDNA normalization kit (Evrogen, Moscow, Russia). Approximately 600−1200 ng of purified ds-cDNA was used as the starting amount for normalization. A mixture of 0.25 µL and 0.5 µL of DSN normalization tubes was used for the first and second amplifications.</p><p id="p-15">After normalization, cDNA was fragmented to 500- to 800-bp fragments by sonication or nebulization and size-selected to remove small fragments using AMPure SPRI beads. The fragmented ends were polished and ligated with adaptors (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Meyer <em>et al.</em> 2009</a>). The optimal ligation products were selectively amplified and subjected to two rounds of size selection including gel electrophoresis and AMPure SPRI bead purification (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> 2012</a>).</p></div><div id="sec-3" class="subsection"><h3>Sequencing and assembly</h3><p id="p-16">All cDNA libraries were sequenced on the 454 Life Science Genome Sequencer (Roche Applied Science, Branford, CT), using either FLX or FLX-Titanium chemistry (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Sequencing of each sample was performed by the 454 Life Science Sequencing Center at Roche Applied Science, the Center for Genomics and Bioinformatics at Indiana University, or the Genome Quebec Innovation Centre at McGill University. Primer/adapter and polyA/T sequences were trimmed from the reads using custom Perl trimming scripts and SeqClean (<a href="http://www.tigr.org/tdb/tgi/software/">http://www.tigr.org/tdb/tgi/software/</a>). Sequences composed of primer multimers were removed using TagDust (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lassmann <em>et al.</em> 2009</a>), with a false discovery rate of 0.01. These cleaning steps were combined into a flexible automatic sequence cleaning pipeline ‘SnoWhite’ (v1.1.4), which we have made publicly available at <a href="http://evopipes.net">http://evopipes.net</a> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Barker <em>et al.</em> 2010</a>).</p><p id="p-17">Cleaned reads were assembled into contigs <em>de novo</em> using the assembler MIRA v3.0 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chevreux <em>et al.</em> 2004</a>). MIRA produced identical duplicate contigs in areas with high read depth, and these were merged using additional iterations of both MIRA and CAP3 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Huang and Madan 1999</a>) at 97% minimum similarity, using the pipeline iAssembler v1.3 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zheng <em>et al.</em> 2011</a>). Successful resolution of highly similar alleles and/or paralogs into unique contigs was verified by examining synonymous site divergence among gene family members using the program DupPipe (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barker <em>et al.</em> 2008</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2010</a>). Resolution of highly similar sequences within eukaryotic individuals should yield gene family phylogenetic trees with a characteristic L-shaped distribution of many recent nodes and diminishing numbers of older nodes (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lynch and Conery 2000</a>).</p></div><div id="sec-4" class="subsection"><h3>Allele clustering with AllelePipe</h3><p id="p-18">Allele and single-nucleotide polymorphism (SNP) identification typically rely on mapping reads to a reference genome that represents a haploid set of genes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bentley 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Charlesworth 2010</a>). YST lacks a reference genome, and allelic variation in our samples prevented simple <em>de novo</em> creation of an accurate haploid reference sequence library. To circumvent this problem, we developed a novel software pipeline to cluster putative alleles within and among individuals, available as ‘AllelePipe’ (v1.0.25; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) at <a href="http://evopipes.net">http://evopipes.net</a>. Using the AllelePipe, we assessed similarity among all sequences from all 40 individuals with SSAHA2 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ning <em>et al.</em> 2001</a>), with 95% minimum similarity and 300-bp minimum alignment length between sequences. We also included sequences from a published assembly of a Sanger EST library for a YST genotype from the invasion in central California (Genbank #<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=EH750647&amp;atom=%2Fggg%2F3%2F2%2F359.atom">EH750647</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=EH791053&amp;atom=%2Fggg%2F3%2F2%2F359.atom">EH791053</a>) (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Barker <em>et al.</em> 2008</a>). AllelePipe was used to verify that similar sequences aligned throughout their region of overlap (expected for true alleles), and to cluster groups of similar sequences via single-linkage clustering. Single-linkage clustering generates maximal aggregation of sequences, and will bring together both closely related paralogs and their alleles (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dlugosch and Bonin 2012</a>). Multiple alignments were created for sequences within each cluster and their consensus sequence generated using CAP3. Clusters were discarded from the analysis if they aggregated increasingly dissimilar sequences, preventing a single CAP3 consensus. From the resulting consensus sequences, a genomic ‘pseudo-reference’ FASTA file was generated for the entire dataset, suitable for anchoring contig alignments across individuals. We evaluated the quality of our overall clustering strategy by aligning our consensus sequences with known highly conserved eukaryotic single copy loci, including 357 ultra-conserved orthologs (UCOs, available at <a href="http://compgenomics.ucdavis.edu/compositae_reference.php">http://compgenomics.ucdavis.edu/compositae_reference.php</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kozik <em>et al.</em> 2008</a>), using tblastx comparisons with maximum expectation (e-value) of 0.1 and minimum 30 protein residue alignments (Blast v2.2.24) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>). Only one-to-one matches between UCOs and clusters are expected for properly clustered loci.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/359/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AllelePipe workflow for identifying alleles without a reference genome. Unigenes from all individuals are pooled and clustered by similarity. Clustered sequences are aligned and consensus sequences are generated, providing a pseudo-reference genome. Unigenes from the same and/or different individuals are aligned to the reference, and SNPs are identified. Multilocus SNP information is used to construct a minimum set of haplotypes for each individual, and clusters in which individuals are represented by an excess number of putative alleles are flagged as potential multigene clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1416316437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;AllelePipe workflow for identifying alleles without a reference genome. Unigenes from all individuals are pooled and clustered by similarity. Clustered sequences are aligned and consensus sequences are generated, providing a pseudo-reference genome. Unigenes from the same and/or different individuals are aligned to the reference, and SNPs are identified. Multilocus SNP information is used to construct a minimum set of haplotypes for each individual, and clusters in which individuals are represented by an excess number of putative alleles are flagged as potential multigene clusters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/359/F1.medium.gif" width="418" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/359/F1.medium.gif" width="418" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/359/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/359/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462458" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-19" class="first-child">AllelePipe workflow for identifying alleles without a reference genome. Unigenes from all individuals are pooled and clustered by similarity. Clustered sequences are aligned and consensus sequences are generated, providing a pseudo-reference genome. Unigenes from the same and/or different individuals are aligned to the reference, and SNPs are identified. Multilocus SNP information is used to construct a minimum set of haplotypes for each individual, and clusters in which individuals are represented by an excess number of putative alleles are flagged as potential multigene clusters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Finally, we filtered for (putatively) valid loci by removing those clusters with evidence of more than two alleles (multi-SNP haplotypes across the entire gene region) from any individual in the dataset. Excess alleles suggest that the cluster is not a single locus and instead a group of paralogous loci. This approach leverages the information gained from multiple individuals to infer paralogy, where it is not possible to do so from genomic DNA sequencing depth or alignment to a complete reference genome. Again using the AllelePipe, we identified SNPs for each sequence against the pseudo-reference by using ssahaSNP (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Ning <em>et al.</em> 2001</a>) with minimum 90% similarity for alignment to the pseudo-reference. Singleton SNPs (those seen only once across the dataset) were removed as potential errors, and the number of unique haplotypes for each individual in the cluster were evaluated for evidence of paralogous clustering. Those clusters with no more than two haplotypes per individual were retained, and their SNP variation (from ssahaSNP) retained for further analyses.</p></div><div id="sec-5" class="subsection"><h3>Gene annotation</h3><p id="p-21">Gene Ontology (GO) classifications for our pseudo-reference sequences and individual transcriptome assemblies were obtained through BLASTx searches against the <em>Arabidopsis thaliana</em> protein database from The Arabidopsis Information Resource (release TAIR10_pep_20101214; <a href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>), using an e-value cut off 1*10<sup>−10</sup>. To evaluate the impact of sequencing effort on the gene content of assemblies, we compared the representation of GO categories across libraries using a Chi-squared Contingency test in R (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">R Development Core Team 2010</a>).</p></div><div id="sec-6" class="subsection"><h3>Population genetic analyses</h3><p id="p-22">We examined the impact of both sequencing effort and source region (native <em>vs.</em> invading) on assembled contig numbers, observed SNP positions, observed heterozygosity, and SNP overlap. SNP frequencies and overlap among individuals were quantified using custom Perl scripts. Significance of relationships was tested with linear model fits in R, using the lm function (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">R Development Core Team 2010</a>). Geographic partitioning of genetic variation in the native range was assessed with STRUCTURE v2.3.2.1 for 1-4 subpopulations (K), using 10,000 burnin steps and 10,000 iterations of the models (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Falush <em>et al.</em> 2003</a>). Runs were repeated three to five times at each K to verify convergence. Naturalized and invading samples were then assigned to native sub-populations using STRUCTURE, and results were plotted using Distruct v 1.1 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rosenberg 2004</a>).</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><div id="sec-8" class="subsection"><h3>Transcriptome recovery</h3><p id="p-23">We obtained between 11.4 and 430.7 Mbp of raw sequence per individual as a result of a range of sequencing efforts across our samples (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; NCBI Short Read Archive accession #SRA059334; assembly for AR-13-24 previously published in <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> 2012</a> and available at doi:10.5061/dryad.cm7td/4). Our assemblies generated up to 71,054 unigenes (contigs and singleton reads combined), including up to 48,230 contigs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Gene number in the YST genome is not yet known, but our largest assemblies are at the top of the range of current annotation counts among complete angiosperm genomes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Barker <em>et al.</em> 2012</a>), and somewhat higher than the ~35,000 loci predicted in the genome of the related asterid tomato (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">The Tomato Genome Consortium 2012</a>). Given that our accessions are outbred individuals and allelic variation is expected, these numbers are consistent with a relatively complete view of expressed genes in this species. Indeed we find patterns of saturating gains in unique sequences and coverage of known conserved loci: Both unigene and contig numbers were positively related to sequencing effort, and these relationships were fit closely by logarithmic curves (regressions: unigenes, R<sup>2</sup> = 0.93, <em>P</em> &lt; 0.0001; contigs, R<sup>2</sup> = 0.91, <em>P</em> &lt; 0.0001, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), indicating that recovery of additional unique sequences was associated with exponential increases in sequencing effort. Although the longer-read GSFLX-Titanium chemistry resulted in longer contigs (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), our contig numbers generated from those libraries followed the same logarithmic relationship with sequencing effort predicted by the shorter-read GS-FLX chemistry (nonsignificant interaction of chemistry type and sequencing effort in linear model: <em>P</em> = 0.43; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/003871SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/FigureS1.pdf">Figure S1</a>). The proportion of UCOs recovered reached 87% (311 loci) among the largest libraries (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). In general, gains from additional sequencing were relatively modest above approximately 100 Mb of usable sequence, which is likely to be less than 5x coverage of the YST transcriptome, given our assembly size. This result suggests that representative information can be gained from low coverage datasets, even while additional sequencing depth continues to yield further gene discovery (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> 2012</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/359/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contig numbers in transcriptome libraries of native (triangles), naturalized (dashes), and invading (circles) genotypes, as a function of total sequence effort after cleaning by SnoWhite. A logarithmic fit is shown to variation across all individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1416316437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Contig numbers in transcriptome libraries of native (triangles), naturalized (dashes), and invading (circles) genotypes, as a function of total sequence effort after cleaning by SnoWhite. A logarithmic fit is shown to variation across all individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/359/F2.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/359/F2.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/359/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/359/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462403" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-24" class="first-child">Contig numbers in transcriptome libraries of native (triangles), naturalized (dashes), and invading (circles) genotypes, as a function of total sequence effort after cleaning by SnoWhite. A logarithmic fit is shown to variation across all individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To evaluate our ability to resolve highly similar sequences in our assemblies, we examined frequency distributions of the synonymous site divergence at nodes in gene family trees for each of our datasets. Distributions of divergence times showed expected L-shaped patterns of abundant recent ancestry in each case, consistent with successful resolution of closely-related paralogs and alleles in our assemblies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/FigureS2.pdf">Figure S2</a>). The distributions also accurately reveal an additional small peak at ~Ks = 0.65, which has been shown previously to correspond to an ancient genome duplication event near the base of the family, based upon Sanger sequence data (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Barker <em>et al.</em> 2008</a>). Consistent with those previous analyses, there was no evidence of more recent genome duplication events in our YST individuals.</p></div><div id="sec-9" class="subsection"><h3>Allele clustering and library overlap</h3><p id="p-26">Single-linkage clustering of unigenes across all individuals using AlellePipe generated 43,717 total putative loci with a median consensus length of 811 bp. These aligned to 260 (73%) of the UCOs, and the majority of these UCOs aligned with only one cluster as expected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/FigureS3.pdf">Figure S3</a>). Alignment of the remaining UCOs to multiple clusters could reflect sequence divergence (low similarity with UCOs), duplication of the locus, or failure of the reads to cluster together. Divergence or duplication is possible but relatively unlikely in such highly conserved “single-copy” loci (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Duarte <em>et al.</em> 2010</a>); it is more likely that splitting of these clusters resulted from alternate splicing (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barbazuk <em>et al.</em> 2008</a>) or insufficient overlap for observing similarity among sets of sequences. These latter processes will introduce clusters of sequences that are incomplete and/or redundant but nevertheless accurate variations on a gene region or splice form. Gene annotation of the consensus cluster sequences by similarity to <em>A. thaliana</em> proteins yielded 26,728 matches, 11,268 of which (25.8% of all clusters) were identified as unique, nonredundant annotations. Similar rates of homology to <em>A. thaliana</em> were observed in other transcriptome surveys of asterid plants, using both Sanger and next-generation sequencing platforms (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Barker <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dempewolf <em>et al.</em> 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Angeloni <em>et al.</em> 2011</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> 2012</a>).</p><p id="p-27">After clustering, AllelePipe identified 22,687 polymorphic unigenes that conformed to expectations of no more than two alleles per individual. These loci annotated to 23,896 <em>A. thaliana</em> genes, indicating that most of the inferred multilocus clusters were not those that had successfully annotated, perhaps due to poor consensus formation. For annotated loci, GO representation differed among inferred single loci and all clusters (<em>P</em> &lt; 0.001 for all three categories; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/FigureS4.pdf">Figure S4</a>). Interestingly, single loci included a lower representation of transcriptome factors, and a greater representation of intracellular and plastid-associated components. These patterns contrast with patterns of duplicate retention from paleo genome duplication events in the Compositae family, consistent with the hypothesis that certain complete pathways are retained in duplicate from whole genome duplications due to dosage constraints, while other functions are free to vary in copy number [and the latter would generate our multilocus clusters (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Barker <em>et al.</em> 2008</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">2012</a>) and references therein].</p><p id="p-28">Aligning sequences from the 41 YST datasets against this filtered set of 22,687 pseudo-reference sequences revealed 237,034 polymorphic sites over the total sequence length of 21.2 Mbp, where each SNP variant was observed at least twice across all haplotypes (1 SNP per 89.6 bp across the sample). This low SNP density underscores the need for long-read approaches to accurately recover haplotypes within and among conspecific individuals (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Dlugosch and Bonin 2012</a>; <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> 2012</a>). The proportion of these sites sequenced in each individual increased sharply with sequencing effort (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), with no indication of saturation; the largest libraries covered less than 60% of SNP positions. The number of observed heterozygous positions also showed a strong and linear increase with sequencing effort in both native and invading samples (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The accurate observation of heterozygosity is likely to be particularly important both for revealing population genetic information, and for validating the SNPs themselves (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Seeb <em>et al.</em> 2011</a>). Thus, despite saturating returns of additional unique sequences in the larger libraries, our ability to observe allelic variation at these loci did not plateau, and analyses of this type of variation must take sequencing effort into account.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/359/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coverage of SNP positions identified across the dataset. (A) Proportion of SNPs that were sequenced in each individual, and (B) frequency of observed heterozygous loci among sequenced SNPs within native (triangles), naturalized (dashes), and invading (circles) individuals, relative to total sequencing effort after read cleaning. Linear fits are shown to native (gray line) and invading (black line) individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1416316437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Coverage of SNP positions identified across the dataset. (A) Proportion of SNPs that were sequenced in each individual, and (B) frequency of observed heterozygous loci among sequenced SNPs within native (triangles), naturalized (dashes), and invading (circles) individuals, relative to total sequencing effort after read cleaning. Linear fits are shown to native (gray line) and invading (black line) individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/359/F3.medium.gif" width="282" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/359/F3.medium.gif" width="282" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/359/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/359/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-29" class="first-child">Coverage of SNP positions identified across the dataset. (A) Proportion of SNPs that were sequenced in each individual, and (B) frequency of observed heterozygous loci among sequenced SNPs within native (triangles), naturalized (dashes), and invading (circles) individuals, relative to total sequencing effort after read cleaning. Linear fits are shown to native (gray line) and invading (black line) individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Sequence overlap among samples also was improved by increased sequencing effort. In pairwise comparisons, the proportion of all polymorphic positions shared between individuals increased from &lt;1% to almost 30% in the largest libraries (which shared nearly 70% of the SNP positions observed in any individual library; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The 10 largest libraries included four native, four invading, and two naturalized individuals with more than 200 Mbp of sequencing effort each. Surveys of SNP sites—as identified from across the dataset—within just these greatest coverage libraries revealed that most SNP sites were sequenced in only a few individuals, though 6883 loci were present in all 10 libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/FigureS5.pdf">Figure S5</a>). The number of overlapping SNP positions was significantly greater than expected by chance, given the number of positions observed in each library (K-S Goodness of Fit test: <em>P</em> &lt; 0.01, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1/FigureS5.pdf">Figure S5</a>). Overlapping positions are almost certainly biased toward the most commonly expressed loci, however our GO categorizations indicate that this does not represent a particularly biased subset of genome function. Functional categorization and distribution of the annotations within the three GO categories (biological process, cellular component, and molecular function), did not vary more than a few percent among the individual assemblies for any classification, though this modest variation was statistically significant (<em>P</em> &lt; 0.0001 among libraries within each of the three categories, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). There was a consistent trend toward greater representation of loci of unknown function (although still similar to <em>A. thaliana</em> proteins) in larger libraries within each category (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), indicating greater recovery of less well-studied and presumably more rarely-expressed loci.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/359/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise overlap in observed SNPs. Isoclines reflect the proportion of all SNP positions that are observed in pairwise comparisons of 40 transcriptome libraries, as a function of sequencing effort in both samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1416316437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pairwise overlap in observed SNPs. Isoclines reflect the proportion of all SNP positions that are observed in pairwise comparisons of 40 transcriptome libraries, as a function of sequencing effort in both samples.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/359/F4.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/359/F4.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/359/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/359/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462461" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-31" class="first-child">Pairwise overlap in observed SNPs. Isoclines reflect the proportion of all SNP positions that are observed in pairwise comparisons of 40 transcriptome libraries, as a function of sequencing effort in both samples.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/359/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of GO annotations to A. thaliana as a function of transcriptome sequencing effort." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1416316437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of GO annotations to &lt;em&gt;A. thaliana&lt;/em&gt; as a function of transcriptome sequencing effort.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/359/F5.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/359/F5.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/359/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/359/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462462" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-32" class="first-child">The distribution of GO annotations to <em>A. thaliana</em> as a function of transcriptome sequencing effort.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Novel insights into YST population genetics</h3><p id="p-33">For introduced and invasive species, identifying the sources of their genetic variation can provide important insights into the circumstances that made these introductions so successful (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dlugosch and Parker 2008</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Prentis <em>et al.</em> 2008</a>; <a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Lai <em>et al.</em> 2012</a>). Native individuals assembled slightly higher numbers of both unigenes and contigs (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) than invading individuals, potentially indicating reduced sequence variation in introduced genotypes, but these differences were not significant in either case (analysis of covariance on Ln-transformed data: unigenes, <em>P</em> = 0.06; contigs, <em>P</em> = 0.23). Instead, the identification of allelic variation among our samples suggested different population genetic inferences in our YST transcriptome dataset: Across all inferred SNP positions, invaders had significantly greater heterozygosity than natives (analysis of covariance : F<sub>2,33</sub> = 82.21, <em>P</em> = 0.005, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>) and a greater variance around predicted values based on sequencing effort (linear regressions: invader R<sup>2</sup> = 0.67, native R<sup>2</sup> = 0.96). These patterns are consistent with a known history of multiple, large introductions of YST (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gerlach 1997</a>), which should have established substantial genetic diversity in the invaded range.</p><p id="p-34">With the use of 2568 SNPs that were genotyped in at least five individuals per continent, STRUCTURE modeling was able to detect coarse genetic structure in the native range. Estimates of support for population number (ln Pr[X|K]) peaked at K = 2, and these two subpopulations were partitioned geographically, with one region including individuals from Hungary and Romania in eastern Europe and the other region including individuals from Turkey and the Republic of Georgia in the Caucasus (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Including invading and naturalized individuals together with natives in a single model obscured all genetic structure and supported no subpopulations, even when geographic groups were provided as prior information in the model, a pattern indicative of a large number of admixed individuals in the dataset (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Pritchard <em>et al.</em> 2000</a>). When the two native subpopulations were instead provided as <em>a priori</em> fixed groups, invading and naturalized individuals were both assigned as admixtures of the native subpopulations (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). The evidence for admixture suggested by our coarse population sampling is consistent with a history of multiple introductions, and similar patterns in the Spanish collections provide some of the first support for the hypothesis that western European populations are not native and are themselves naturalized products of past introductions (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Maddox <em>et al.</em> 1985</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/359/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="STRUCTURE populations inferred from SNP variation. Vertical bars show the population assignment (color) for each individual by region. (A) Two major genetic groups (blue and orange) are supported for the native range, based upon genotypes from Hungary (HU), Romania (RO), Turkey (TK) and the Republic of Georgia (GA). (B) Admixture of these sources is suggested for both putatively naturalized genotypes in Spain (SP) and invading genotypes from California (CA) and Argentina (AR), when the two native genetic groups are fixed as potential source populations. SNP frequencies were based upon 2568 positions observed in at least five individuals per continent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1416316437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;STRUCTURE populations inferred from SNP variation. Vertical bars show the population assignment (color) for each individual by region. (A) Two major genetic groups (blue and orange) are supported for the native range, based upon genotypes from Hungary (HU), Romania (RO), Turkey (TK) and the Republic of Georgia (GA). (B) Admixture of these sources is suggested for both putatively naturalized genotypes in Spain (SP) and invading genotypes from California (CA) and Argentina (AR), when the two native genetic groups are fixed as potential source populations. SNP frequencies were based upon 2568 positions observed in at least five individuals per continent.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/359/F6.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/359/F6.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/359/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/359/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462387" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-35" class="first-child">STRUCTURE populations inferred from SNP variation. Vertical bars show the population assignment (color) for each individual by region. (A) Two major genetic groups (blue and orange) are supported for the native range, based upon genotypes from Hungary (HU), Romania (RO), Turkey (TK) and the Republic of Georgia (GA). (B) Admixture of these sources is suggested for both putatively naturalized genotypes in Spain (SP) and invading genotypes from California (CA) and Argentina (AR), when the two native genetic groups are fixed as potential source populations. SNP frequencies were based upon 2568 positions observed in at least five individuals per continent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Inferences regarding introduction scenarios are only as robust as the sampling of the potential source populations (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Dlugosch and Parker 2008</a>), however, and our dataset is not extensive enough to rule out that unsampled source populations—rather than admixture of our observed native populations—have produced the current invasions. Thorough geographic sampling of the native range is essential for correctly identifying the most likely source genotypes. Moreover, dozens of individuals should be sampled from within each population to accurately estimate allele frequencies, and recover well-supported patterns of population structure (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Avise 2004</a>). Transcriptome studies generally have been considered too expensive to be used for this kind of broad population sampling, but as they becomes increasingly cost effective, investment in deeper sampling of individuals promises to be fruitful for robust population genomic studies.</p></div><div id="sec-11" class="subsection"><h3>Conclusions</h3><p id="p-37">By analyzing the realized outcome of a wide range of sequencing efforts, our dataset reveals that allele recovery is far more dependent on sequencing depth than is gene recovery, although both are enhanced by additional sequencing depth. These relationships are of particular concern for diversity comparisons among outbred individuals, which are increasingly taking center stage as genomic sequencing moves outside of its historical focus on inbred lines of traditional model organisms. Modest transcriptome sampling nevertheless generates thousands of informative markers observable across many individuals. This is promising news for further studies of coding variation among individuals, and our ability to combine datasets generated with different methods over time – one of the inherent strengths of sequence-based population genetics. Using our novel bioinformatic pipeline for allele identification, we were able to recover previously hypothesized population features using 40 individuals dispersed across a worldwide distribution, demonstrating the information-rich nature of transcriptome populations datasets.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank Ö. Eren, L. Khetsuriani, A. Diaconu, K. Török, and D. Montesinos for sharing seed collections; B. Boese and the 454 Sequencing Center for discussions about GS FLX sequencing techniques; M.S. Barker for discussions about sequence assembly; M. King for computing support; and two anonymous reviewers for helpful comments on the manuscript. Funding for this study was provided by a Roche Applied Science/454 Life Sciences sequencing prize to Z.L. and L.H.R., and a Natural Sciences and Engineering Research Council of Canada (NSERC) grant #353026 to L.H.R.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003871/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Reads have been submitted to the SRA database at NCBI as series SRA059334.</p></li><li class="fn" id="fn-5"><p id="p-39"><em>Communicating editor: J. Dekker</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-26"><span class="received-label">Received </span>July 26, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-19"><span class="accepted-label">Accepted </span>December 19, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Dlugosch <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.2" data-doi="10.1111/j.1755-0998.2011.02990.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angeloni</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagemaker</span>  <span class="cit-name-given-names">C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jetten</span>  <span class="cit-name-given-names">M. S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Op den Camp</span>  <span class="cit-name-given-names">H. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jassen-Megens</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">De novo transcriptome characterization and development of genomic tools for Scabiosa columbaria L. using next-generation sequencing techniques</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D11%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.02990.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.02990.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avise</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Molecular Markers, Natural History, and Evolution</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.4" data-doi="10.1111/j.1365-313X.2007.03193.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emrich</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">SNP discovery via 454 transcriptome sequencing</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">910</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DBarbazuk%26rft.auinit1%253DW.%2BB.%26rft.volume%253D51%26rft.issue%253D5%26rft.spage%253D910%26rft.epage%253D918%26rft.atitle%253DSNP%2Bdiscovery%2Bvia%2B454%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03193.x%26rft_id%253Dinfo%253Apmid%252F17662031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03193.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17662031&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249424100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.5" data-doi="10.1101/gr.053678.106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide analyses of alternative splicing in plants: opportunities and challenges</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.053678.106%26rft_id%253Dinfo%253Apmid%252F18669480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMTM4MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.6" data-doi="10.1093/molbev/msn187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matvienko</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple paleopolyploidizations during the evolution of the Compositae reveal parallel patterns of duplicate gene retention after millions of years</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2445</span>–<span class="cit-lpage">2455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn187%26rft_id%253Dinfo%253Apmid%252F18728074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8yNDQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dlugosch</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinh</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Challa</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EvoPipes.net: bioinformatic tools for ecological and evolutionary genomics</span>. <abbr class="cit-jnl-abbrev">Evol. Bioinf.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBioinf.%26rft.volume%253D6%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baute</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Duplication and turnover in plant genomes</span>, pp. <span class="cit-fpage">155</span>–<span class="cit-lpage">169</span> in <span class="cit-source">Plant Genome Diversity</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.9" data-doi="10.1111/j.1365-294X.2010.04963.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barreto</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moy</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interpopulation patterns of divergence and selection across the transcriptome of the copepod <em>Tigriopus californicus</em></span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">560</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBarreto%26rft.auinit1%253DF.%2BS.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D560%26rft.epage%253D572%26rft.atitle%253DInterpopulation%2Bpatterns%2Bof%2Bdivergence%2Band%2Bselection%2Bacross%2Bthe%2Btranscriptome%2Bof%2Bthe%2Bcopepod%2BTigriopus%2Bcalifornicus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04963.x%26rft_id%253Dinfo%253Apmid%252F21199025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04963.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21199025&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286254700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.10" data-doi="10.1186/1471-2164-7-130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beldade</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A wing expressed sequence tag resource for <em>Bicyclus anynana</em> butterflies, an evo-devo model</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBeldade%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.spage%253D130%26rft.epage%253D130%26rft.atitle%253DA%2Bwing%2Bexpressed%2Bsequence%2Btag%2Bresource%2Bfor%2BBicyclus%2Banynana%2Bbutterflies%252C%2Ban%2Bevo-devo%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-130%26rft_id%253Dinfo%253Apmid%252F16737530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16737530&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.11" data-doi="10.1016/j.gde.2006.10.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Whole-genome re-sequencing</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DBentley%26rft.auinit1%253DD.%2BR.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D545%26rft.epage%253D552%26rft.atitle%253DWhole-genome%2Bre-sequencing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2006.10.009%26rft_id%253Dinfo%253Apmid%252F17055251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2006.10.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17055251&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242647400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.12" data-doi="10.1126/science.1061337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evolutionary dynamics of plant R-genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">292</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBergelson%26rft.auinit1%253DJ.%26rft.volume%253D292%26rft.issue%253D5525%26rft.spage%253D2281%26rft.epage%253D2285%26rft.atitle%253DEvolutionary%2BDynamics%2Bof%2BPlant%2BR-Genes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1061337%26rft_id%253Dinfo%253Apmid%252F11423651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUyNS8yMjgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.13" data-doi="10.1111/j.1365-294X.2006.03195.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouck</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vision</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The molecular ecologist’s guide to expressed sequence tags</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">907</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBouck%26rft.auinit1%253DA.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D907%26rft.epage%253D924%26rft.atitle%253DThe%2Bmolecular%2Becologist%2527s%2Bguide%2Bto%2Bexpressed%2Bsequence%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.03195.x%26rft_id%253Dinfo%253Apmid%252F17305850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.03195.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305850&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244245300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.14" data-doi="10.1007/s00606-005-0384-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brancheva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greilhuber</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome size in Bulgarian <em>Centaurea</em> s.l. (Asteraceae)</span>. <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">257</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D257%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00606-005-0384-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00606-005-0384-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.15" data-doi="10.1017/S0016672310000522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular population genomics: a short history</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D92%26rft.issue%253D5-6%26rft.spage%253D397%26rft.epage%253D411%26rft.atitle%253DMolecular%2Bpopulation%2Bgenomics%253A%2Ba%2Bshort%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000522%26rft_id%253Dinfo%253Apmid%252F21429271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429271&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.16" data-doi="10.1101/gr.1917404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevreux</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfisterer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drescher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Driesel</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">W. E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1917404%26rft_id%253Dinfo%253Apmid%252F15140833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christodoulou</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorham</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidman</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Construction of normalized RNA-seq libraries for next-generation sequencing using the crab duplex-specific nuclease</span>. <abbr class="cit-jnl-abbrev">Curr. Prot. Mol. Biol.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">4.12.1</span>–<span class="cit-lpage">4.12.11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProt.%2BMol.%2BBiol.%26rft.volume%253D94%26rft.spage%253D4.12.1%26rft.atitle%253DCONSTRUCTION%2BOF%2BNORMALIZED%2BRNA-SEQ%2BLIBRARIES%2BFOR%2BNEXT-GENERATION%2BSEQUENCING%2BUSING%2BTHE%2BCRAB%2BDUPLEX-SPECIFIC%2BNUCLEASE%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.18" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.19" data-doi="10.1111/j.1755-0998.2010.02859.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempewolf</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostevik</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geleta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishing genomic tools and resources for <em>Guizotia abyssinica</em> (L.f.) Cass.—the development of a library of expressed sequence tags, microsatellite loci, and the sequencing of its chloroplast genome</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1048</span>–<span class="cit-lpage">1058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D10%26rft.spage%253D1048%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02859.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02859.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.20" data-doi="10.1371/journal.pone.0000085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demuth</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cristianini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The evolution of mammalian gene families</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDemuth%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D1%26rft.spage%253De85%26rft.epage%253De85%26rft.atitle%253DThe%2Bevolution%2Bof%2Bmammalian%2Bgene%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000085%26rft_id%253Dinfo%253Apmid%252F17183716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17183716&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dlugosch</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allele identification in assembled genomic sequence datasets</span>, pp. <span class="cit-fpage">197</span>–<span class="cit-lpage">211</span> in <span class="cit-source">Methods in Molecular Biology Series: Data Production and Analysis in Population Genomics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Pompanon</span>  <span class="cit-name-given-names">F.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.22" data-doi="10.1111/j.1365-294X.2007.03538.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dlugosch</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Founding events in species invasions: genetic variation, adaptive evolution, and the role of multiple introductions</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDlugosch%26rft.auinit1%253DK.%2BM.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D431%26rft.epage%253D449%26rft.atitle%253DFounding%2Bevents%2Bin%2Bspecies%2Binvasions%253A%2Bgenetic%2Bvariation%252C%2Badaptive%2Bevolution%252C%2Band%2Bthe%2Brole%2Bof%2Bmultiple%2Bintroductions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03538.x%26rft_id%253Dinfo%253Apmid%252F17908213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03538.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17908213&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251740500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.23" data-doi="10.1038/nature06341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Drosophila 12 Genomes Consortium</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of genes and genomes on the <em>Drosophila</em> phylogeny</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DEvolution%2Bof%2Bgenes%2Band%2Bgenomes%2Bon%2Bthe%2BDrosophila%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06341%26rft_id%253Dinfo%253Apmid%252F17994087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994087&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.24" data-doi="10.1186/1471-2148-10-61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landherr</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of shared single copy nuclear genes in <em>Arabidopsis</em>, <em>Populus</em>, <em>Vitis</em> and <em>Oryza</em> and their phylogenetic utility across various taxonomic levels</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDuarte%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D10%26rft.spage%253D61%26rft.epage%253D61%26rft.atitle%253DIdentification%2Bof%2Bshared%2Bsingle%2Bcopy%2Bnuclear%2Bgenes%2Bin%2BArabidopsis%252C%2BPopulus%252C%2BVitis%2Band%2BOryza%2Band%2Btheir%2Bphylogenetic%2Butility%2Bacross%2Bvarious%2Btaxonomic%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-61%26rft_id%253Dinfo%253Apmid%252F20181251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20181251&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.25" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunter</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boekhoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid evolution and selection inferred from the transcriptomes of sympatric crater lake cichlid fishes</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHardy%26rft.auinit1%253DC.%2BM.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D197%26rft.epage%253D212%26rft.atitle%253DCarbon%2Bsource%2Baccounting%2Bfor%2Bfish%2Busing%2Bcombined%2BDNA%2Band%2Bstable%2Bisotope%2Banalyses%2Bin%2Ba%2Bregulated%2Blowland%2Briver%2Bweir%2Bpool.%26rft_id%253Dinfo%253Apmid%252F19912537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19912537&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.27" data-doi="10.1073/pnas.1006538107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merz</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Resolving postglacial phylogeography using high-throughput sequencing</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">16196</span>–<span class="cit-lpage">16200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006538107%26rft_id%253Dinfo%253Apmid%252F20798348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzM3LzE2MTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D164%26rft.spage%253D1567%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12930761&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185248000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerlach</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">How the west was lost: reconstructing the invasion dynamics of yellow starthistle and other plant invaders of western rangelands and natural areas</span>. <abbr class="cit-jnl-abbrev">California Exotic Pest Plant Council Symp. Proc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCalifornia%2BExotic%2BPest%2BPlant%2BCouncil%2BSymp.%2BProc.%26rft.volume%253D3%26rft.spage%253D67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.30" data-doi="10.1101/gr.3567505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cristianini</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimating the tempo and mode of gene family evolution from comparative genomic data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHahn%26rft.auinit1%253DM.%2BW.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D1153%26rft.epage%253D1160%26rft.atitle%253DEstimating%2Bthe%2Btempo%2Band%2Bmode%2Bof%2Bgene%2Bfamily%2Bevolution%2Bfrom%2Bcomparative%2Bgenomic%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3567505%26rft_id%253Dinfo%253Apmid%252F16077014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTE1MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.31" data-doi="10.1371/journal.pgen.0030197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">S.-G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene family evolution across 12 <em>Drosophila</em> genomes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHahn%26rft.auinit1%253DM.%2BW.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De197%26rft.epage%253De197%26rft.atitle%253DGene%2Bfamily%2Bevolution%2Bacross%2B12%2BDrosophila%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030197%26rft_id%253Dinfo%253Apmid%252F17997610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17997610&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamrick</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godt</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Allozyme diversity in plant species</span>, pp. <span class="cit-fpage">43</span>–<span class="cit-lpage">63</span> in <span class="cit-source">Plant Population Genetics, Breeding and Germplasm Resources</span>, edited by<span class="cit-ed"><span class="cit-name-surname">A. H. D .</span></span>, <span class="cit-ed"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kahler</span></span>, <span class="cit-ed"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>. <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.2.359.33" data-doi="10.1126/science.1209244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Adaptation to climate across the <em>Arabidopsis thaliana</em> genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.34" data-doi="10.2307/2438241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heiser</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">Chromosome number, polyploidy, and growth habit in California weeds</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DHeiser%26rft.auinit1%253DC.%2BB.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D179%26rft.epage%253D186%26rft.atitle%253DChromosome%2Bnumber%252C%2Bpolyploidy%252C%2Band%2Bgrowth%2Bhabit%2Bin%2BCalifornia%2Bweeds.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2438241%26rft_id%253Dinfo%253Apmid%252F18909963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2438241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18909963&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1948UE98700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.35" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: a DNA sequence assembly program</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matvienko</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Eukaryotic ultra conserved orthologs and estimation of gene capture in EST libraries</span>. <abbr class="cit-jnl-abbrev">Plant and Animal Genome Conference</abbr> <span class="cit-vol">XVI</span>: <span class="cit-fpage">P6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Band%2BAnimal%2BGenome%2BConference%26rft.volume%253DXVI%26rft.spage%253DP6%26rft.atitle%253DEUKARYOTIC%2BULTRA%2BCONSERVED%2BORTHOLOGS%2BAND%2BESTIMATION%2BOF%2BGENE%2BCAPTURE%2BIN%2BEST%2BLIBRARIES%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.37" data-doi="10.3732/ajb.1100313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodgins</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dlugosch</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomics of Compositae weeds: EST libraries, microarrays, and evidence of introgression</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1100313%26rft_id%253Dinfo%253Apmid%252F22058181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMjA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.38" data-doi="10.1093/bioinformatics/btp527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashizaki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TagDust—a program to eliminate artifacts from next generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinfomatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2839</span>–<span class="cit-lpage">2840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinfomatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp527%26rft_id%253Dinfo%253Apmid%252F19737799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjEvMjgzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.39" data-doi="10.1038/335268a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ennis</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parham</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">HLA-A and B polymorphisms predate the divergence of humans and chimpanzees</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLawlor%26rft.auinit1%253DD.%2BA.%26rft.volume%253D335%26rft.issue%253D6187%26rft.spage%253D268%26rft.epage%253D271%26rft.atitle%253DHLA-A%2Band%2BB%2Bpolymorphisms%2Bpredate%2Bthe%2Bdivergence%2Bof%2Bhumans%2Band%2Bchimpanzees.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F335268a0%26rft_id%253Dinfo%253Apmid%252F3412487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/335268a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3412487&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q047700065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubby</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">A molecular approach to the study of genic heterozygosity in natural populations. II. Amount of variation and degree of heterozygosity in natural populations of <em>Drosophila pseudoobscura</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D54%26rft.spage%253D595%26rft_id%253Dinfo%253Apmid%252F5968643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=5968643&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668175200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.41" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadler</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Low nucleotide diversity in man</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLi%26rft.auinit1%253DW.%2BH.%26rft.volume%253D129%26rft.issue%253D2%26rft.spage%253D513%26rft.epage%253D523%26rft.atitle%253DLow%2Bnucleotide%2Bdiversity%2Bin%2Bman.%26rft_id%253Dinfo%253Apmid%252F1743489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzIvNTEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.43" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maddox</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayfield</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poritz</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Distribution of yellow starthistle (<em>Centaurea solstitialis</em>) and Russian knapweed (<em>Centaurea repens</em>)</span>. <abbr class="cit-jnl-abbrev">Weed Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWeed%2BSci.%26rft.volume%253D33%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.45" data-doi="10.1038/nature03959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margulies</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egholm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attiya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome sequencing in microfabricated high-density picolitre reactors</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMargulies%26rft.auinit1%253DM.%26rft.volume%253D437%26rft.issue%253D7057%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DGenome%2Bsequencing%2Bin%2Bmicrofabricated%2Bhigh-density%2Bpicolitre%2Breactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03959%26rft_id%253Dinfo%253Apmid%252F16056220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16056220&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231849100045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.46" data-doi="10.1038/nrg2344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide association studies for complex traits: consensus, uncertainty and challenges</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMcCarthy%26rft.auinit1%253DM.%2BI.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D369%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcomplex%2Btraits%253A%2Bconsensus%252C%2Buncertainty%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2344%26rft_id%253Dinfo%253Apmid%252F18398418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18398418&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255057300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.47" data-doi="10.1186/1471-2164-10-219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aglyamova</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan-Carter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrego</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMeyer%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D219%26rft.epage%253D219%26rft.atitle%253DSequencing%2Band%2Bde%2Bnovo%2Banalysis%2Bof%2Ba%2Bcoral%2Blarval%2Btranscriptome%2Busing%2B454%2BGSFlx.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-219%26rft_id%253Dinfo%253Apmid%252F19435504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19435504&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.48" data-doi="10.1016/j.tree.2004.01.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth cit-collab">SNP Wokshop Group</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">SNPs in ecology, evolution and conservation</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D19%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2004.01.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2004.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220842400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.49" data-doi="10.1093/oxfordjournals.molbev.a025563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Intraspecific nuclear DNA variation in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMoriyama%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D261%26rft.epage%253D277%26rft.atitle%253DIntraspecific%2Bnuclear%2BDNA%2Bvariation%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025563%26rft_id%253Dinfo%253Apmid%252F8583899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzEvMjYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.50" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.51" data-doi="10.1101/gr.194201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullikin</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">SSAHA: a fast search method for large DNA databases</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNing%26rft.auinit1%253DZ.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D1725%26rft.epage%253D1729%26rft.atitle%253DSSAHA%253A%2BA%2BFast%2BSearch%2BMethod%2Bfor%2BLarge%2BDNA%2BDatabases%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.194201%26rft_id%253Dinfo%253Apmid%252F11591649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMC8xNzI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.52" data-doi="10.1101/gr.080200.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequencing of natural strains of <em>Arabidopsis thaliana</em> with short reads</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2024</span>–<span class="cit-lpage">2033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.080200.108%26rft_id%253Dinfo%253Apmid%252F18818371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8yMDI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozturk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinc</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozdemir</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A cytogenetical study on some plant taxa in Nizip region (Aksaray, Turkey)</span>. <abbr class="cit-jnl-abbrev">Turk. J. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTurk.%2BJ.%2BBiol.%26rft.volume%253D33%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.54" data-doi="10.1016/j.tplants.2008.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prentis</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">J. R. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dormontt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptive evolution in invasive species</span>. <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DPrentis%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D288%26rft.epage%253D294%26rft.atitle%253DAdaptive%2Bevolution%2Bin%2Binvasive%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2008.03.004%26rft_id%253Dinfo%253Apmid%252F18467157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2008.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18467157&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257517800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnell</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.57" data-doi="10.1111/j.1365-294X.2009.04477.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mining transcriptome sequences towards identifying adaptive single nucleotide polymorphisms in lake whitefish species pairs (<em>Coregonus</em> spp. Salmonidae)</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D19%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04477.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04477.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000275645700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.58" data-doi="10.1046/j.1471-8286.2003.00566.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Distruct: a program for the graphical display of population structure</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D4%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2003.00566.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2003.00566.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189159500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.59" data-doi="10.1126/science.1098918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakshmi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troge</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Large-scale copy number polymorphism in the human genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSebat%26rft.auinit1%253DJ.%26rft.volume%253D305%26rft.issue%253D5683%26rft.spage%253D525%26rft.epage%253D528%26rft.atitle%253DLarge-Scale%2BCopy%2BNumber%2BPolymorphism%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1098918%26rft_id%253Dinfo%253Apmid%252F15273396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4My81MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.60" data-doi="10.1111/j.1755-0998.2010.02936.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascal</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grau</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptome sequencing and high-resolution melt analysis advance single nucleotide polymorphism discovery in duplicated salmonids</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D11%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02936.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02936.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.61" data-doi="10.1139/b97-861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Population genetic structure of yellow starthistle (<em>Centaurea solstitialis</em>), a colonizing weed in the western United States</span>. <abbr class="cit-jnl-abbrev">Can. J. Bot.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1470</span>–<span class="cit-lpage">1478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BBot.%26rft.volume%253D75%26rft.spage%253D1470%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fb97-861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/b97-861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.62" data-doi="10.1046/j.1365-2540.1998.00290.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mating system of yellow starthistle (<em>Centaurea solstitialis</em>), a successful colonizer in North America</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D80%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.1998.00290.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.1998.00290.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000072141900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.63" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Tomato Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.64" data-doi="10.1186/1471-2164-11-650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ueno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Provost</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léger</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klopp</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noirot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bioinformatic analysis of ESTs collected by Sanger and pyrosequencing methods for a keystone forest tree species: oak</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DUeno%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D650%26rft.epage%253D650%26rft.atitle%253DBioinformatic%2Banalysis%2Bof%2BESTs%2Bcollected%2Bby%2BSanger%2Band%2Bpyrosequencing%2Bmethods%2Bfor%2Ba%2Bkeystone%2Bforest%2Btree%2Bspecies%253A%2Boak.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-650%26rft_id%253Dinfo%253Apmid%252F21092232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21092232&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.65" data-doi="10.1093/nar/23.21.4407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleeker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reijans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">AFLP: a new technique for DNA fingerprinting</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4407</span>–<span class="cit-lpage">4414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DVos%26rft.auinit1%253DP.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D4407%26rft.epage%253D4414%26rft.atitle%253DAFLP%253A%2Ba%2Bnew%2Btechnique%2Bfor%2BDNA%2Bfingerprinting%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.21.4407%26rft_id%253Dinfo%253Apmid%252F7501463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMS80NDA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Coalescent Theory: An introduction</span>. <span class="cit-publ-name">Roberts &amp; Company Publishers</span>, <span class="cit-publ-loc">Greenwood Village. CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.67" data-doi="10.1007/s10709-008-9326-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheat</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapidly developing functional genomics in ecological model systems via 454 transcriptome sequencing</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DWheat%26rft.auinit1%253DC.%2BW.%26rft.volume%253D138%26rft.issue%253D4%26rft.spage%253D433%26rft.epage%253D451%26rft.atitle%253DRapidly%2Bdeveloping%2Bfunctional%2Bgenomics%2Bin%2Becological%2Bmodel%2Bsystems%2Bvia%2B454%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9326-y%26rft_id%253Dinfo%253Apmid%252F18931921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9326-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18931921&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F359.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275460800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.68" data-doi="10.1614/WS-06-211R.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Widmer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guermache</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolgovskaia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznick</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Enhanced growth and seed properties in introduced <em>vs.</em> native populations of yellow starthistle (<em>Centaurea solstitialis</em>)</span>. <abbr class="cit-jnl-abbrev">Weed Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWeed%2BSci.%26rft.volume%253D55%26rft.spage%253D465%26rft_id%253Dinfo%253Adoi%252F10.1614%252FWS-06-211R.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1614/WS-06-211R.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.359.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jette</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCaffrey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Biology and Biological Control of Starthistle</span>. <span class="cit-publ-name">USDA Forest Service / University of Nebraska Lincoln Faculty Publications</span>, <span class="cit-publ-loc">Lincoln, NB</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schult</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHugh</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wares</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome analysis and SNP development can resolve population differentiation of <em>Streblospio benedicti</em>, a developmentally dimorphic marine annelid</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e3161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De3161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.71" data-doi="10.1073/pnas.0800107105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zayed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitfield</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide signature of positive selection in ancient and recent invasive expansions of the honey bee <em>Apis mellifera</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3421</span>–<span class="cit-lpage">3426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0800107105%26rft_id%253Dinfo%253Apmid%252F18299560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzkvMzQyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.72" data-doi="10.1186/1471-2105-12-453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fei</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">iAssembler: a package for de novo assembly of Roche-454/Sanger transcriptome sequences</span>. <abbr class="cit-jnl-abbrev">BMC Bioinf.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinf.%26rft.volume%253D12%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.359.73" data-doi="10.1093/nar/gnh031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhulidov</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogdanova</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shcheglov</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaspekov</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Simple cDNA normalization using kamchatka crab duplex specific nuclease</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">e37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZhulidov%26rft.auinit1%253DP.%2BA.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253De37%26rft.epage%253De37%26rft.atitle%253DSimple%2BcDNA%2Bnormalization%2Busing%2Bkamchatka%2Bcrab%2Bduplex-specific%2Bnuclease%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnh031%26rft_id%253Dinfo%253Apmid%252F14973331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMyLzMvZTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/2/359.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/353" title="How to Design a Genetic Mating Scheme: A Basic Training Package for Drosophila Genetics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/369" title="Functional Analyses of Trichoderma reesei LAE1 Reveal Conserved and Contrasting Roles of This Regulator" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/2"><span>Volume 3 Issue 2, February 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/2.cover-source.jpg?itok=h3kYZ-FP" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/359" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4eJ4whAuU27UDQUBOoYg1sQCIa8oSaYdmnQynVf8Azk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/2/359" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/2/359" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/2/359" class="active" data-icon-position="" data-hide-link-title="0">Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant Centaurea solstitialis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459409" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7NxgZdTZ5cRKKabbLi2lHCiZ-u1wTrqxZhCI3o5da8A" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/359.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/2/359" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-c_OL05QL1A2sGg97PWTFM59lZ6-TiqoGjW6Db9959ZM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/2/359" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/359.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459409" id="node459409--2" data-pisa="ggg;3/2/359" data-pisa-master="ggg;3/2/359" data-apath="/ggg/3/2/359.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant <em>Centaurea solstitialis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katrina M. Dlugosch</span>, <span class="highwire-citation-author" data-delta="1">Zhao Lai</span>, <span class="highwire-citation-author" data-delta="2">Aurélie Bonin</span>, <span class="highwire-citation-author" data-delta="3">José Hierro</span> and <span class="highwire-citation-author" data-delta="4">Loren H. Rieseberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">359-367;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003871 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459409--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrina M. Dlugosch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrina%2BM.%2BDlugosch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dlugosch%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatrina%2BM.%2BDlugosch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdlugosch@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">kdlugosch@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhao Lai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao%2BLai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhao%2BLai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aurélie Bonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BBonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BBonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Hierro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Ciencias Exactas y Naturales, INCITAP (CONICET-Universidad Nacional de La Pampa), AR-6300, Santa Rosa, La Pampa, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BHierro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hierro%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BHierro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Loren H. Rieseberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Loren%2BH.%2BRieseberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rieseberg%20LH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALoren%2BH.%2BRieseberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/2/359" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/2/359&amp;title=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/2/359&amp;title=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/2/359&amp;title=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/2/359&amp;text=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/2/359&amp;title=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/2/359&amp;t=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/2/359&amp;title=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/2/359&amp;title=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459409/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459409" id="node459409--3" data-pisa="ggg;3/2/359" data-pisa-master="ggg;3/2/359" data-apath="/ggg/3/2/359.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Allele Identification for Transcriptome-Based Population Genomics in the Invasive Plant <em>Centaurea solstitialis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katrina M. Dlugosch</span>, <span class="highwire-citation-author" data-delta="1">Zhao Lai</span>, <span class="highwire-citation-author" data-delta="2">Aurélie Bonin</span>, <span class="highwire-citation-author" data-delta="3">José Hierro</span> and <span class="highwire-citation-author" data-delta="4">Loren H. Rieseberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">359-367;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003871 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459409--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrina M. Dlugosch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrina%2BM.%2BDlugosch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dlugosch%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatrina%2BM.%2BDlugosch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdlugosch@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">kdlugosch@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhao Lai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao%2BLai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhao%2BLai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aurélie Bonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BBonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BBonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Hierro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Ciencias Exactas y Naturales, INCITAP (CONICET-Universidad Nacional de La Pampa), AR-6300, Santa Rosa, La Pampa, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BHierro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hierro%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BHierro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Loren H. Rieseberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Botany, University of British Columbia, Vancouver, BC V6T1Z4 Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Loren%2BH.%2BRieseberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rieseberg%20LH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALoren%2BH.%2BRieseberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=359&amp;rft.epage=367&amp;rft.atitle=Allele%20Identification%20for%20Transcriptome-Based%20Population%20Genomics%20in%20the%20Invasive%20Plant%20Centaurea%20solstitialis&amp;rft.volume=3&amp;rft.issue=2&amp;rft.date=2013-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dlugosch%2C+Katrina+M.&amp;rft.au=Lai%2C+Zhao&amp;rft.au=Bonin%2C+Aur%C3%A9lie&amp;rft.au=Hierro%2C+Jos%C3%A9&amp;rft.au=Rieseberg%2C+Loren+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459409/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459409/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459409/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459409/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459409/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459409/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459409/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459409/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459409/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459409/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459409/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459409/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/2/359" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/2/359.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/2/359.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/2/359.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/2/359.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
