<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/2/205" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/2/205.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/2/205.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/2/205.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/2/205.atom" />
<meta name="HW.pisa" content="ggg;3/2/205" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray" />
<meta name="DC.Identifier" content="10.1534/g3.112.004739" />
<meta name="DC.Date" content="2013-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Desjardins et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Nasonia, a genus of four closely related parasitoid insect species, is a model system for genetic research. Their haplodiploid genetics (haploid males and diploid females) and interfertile species are advantageous for the genetic analysis of complex traits and the genetic basis of species differences. A fine-scale genomic map is an important tool for advancing genetic studies in this system. We developed and used a hybrid genotyping microarray to generate a high-resolution genetic map that covers 79% of the sequenced genome of Nasonia vitripennis . The microarray is based on differential hybridization of species-specific oligos between N. vitripennis and Nasonia giraulti at more than 20,000 markers spanning the Nasonia genome. The map places 729 scaffolds onto the five linkage groups of Nasonia, including locating many smaller scaffolds that would be difficult to map by other means. The microarray was used to characterize 26 segmental introgression lines containing chromosomal regions from one species in the genetic background of another. These segmental introgression lines have been used for rapid screening and mapping of quantitative trait loci involved in species differences. Finally, the microarray is extended to bulk-segregant analysis and genotyping of other Nasonia species combinations. These resources should further expand the usefulness of Nasonia for studies of the genetic basis and architecture of complex traits and speciation." />
<meta name="DC.Contributor" content="Christopher A. Desjardins" />
<meta name="DC.Contributor" content="Jürgen Gadau" />
<meta name="DC.Contributor" content="Jacqueline A. Lopez" />
<meta name="DC.Contributor" content="Oliver Niehuis" />
<meta name="DC.Contributor" content="Amanda R. Avery" />
<meta name="DC.Contributor" content="David W. Loehlin" />
<meta name="DC.Contributor" content="Stephen Richards" />
<meta name="DC.Contributor" content="John K. Colbourne" />
<meta name="DC.Contributor" content="John H. Werren" />
<meta name="article:published_time" content="2013-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Nasonia, a genus of four closely related parasitoid insect species, is a model system for genetic research. Their haplodiploid genetics (haploid males and diploid females) and interfertile species are advantageous for the genetic analysis of complex traits and the genetic basis of species differences. A fine-scale genomic map is an important tool for advancing genetic studies in this system. We developed and used a hybrid genotyping microarray to generate a high-resolution genetic map that covers 79% of the sequenced genome of &lt;i&gt;Nasonia vitripennis&lt;/i&gt;. The microarray is based on differential hybridization of species-specific oligos between &lt;i&gt;N. vitripennis&lt;/i&gt; and &lt;i&gt;Nasonia giraulti&lt;/i&gt; at more than 20,000 markers spanning the Nasonia genome. The map places 729 scaffolds onto the five linkage groups of Nasonia, including locating many smaller scaffolds that would be difficult to map by other means. The microarray was used to characterize 26 segmental introgression lines containing chromosomal regions from one species in the genetic background of another. These segmental introgression lines have been used for rapid screening and mapping of quantitative trait loci involved in species differences. Finally, the microarray is extended to bulk-segregant analysis and genotyping of other Nasonia species combinations. These resources should further expand the usefulness of Nasonia for studies of the genetic basis and architecture of complex traits and speciation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/02/01" />
<meta name="citation_mjid" content="ggg;3/2/205" />
<meta name="citation_id" content="3/2/205" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/2/205" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/2/205.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/2/205.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/2/205.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004739" />
<meta name="citation_pmid" content="23390597" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="205" />
<meta name="citation_lastpage" content="215" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christopher A. Desjardins" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York 14627" />
<meta name="citation_author_institution" content="Broad Institute, Cambridge, Massachusetts 02141" />
<meta name="citation_author_email" content="cdesjard@broadinstitute.org" />
<meta name="citation_author" content="Jürgen Gadau" />
<meta name="citation_author_institution" content="School of Life Sciences, Arizona State University, Tempe, Arizona 85287" />
<meta name="citation_author" content="Jacqueline A. Lopez" />
<meta name="citation_author_institution" content="The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Oliver Niehuis" />
<meta name="citation_author_institution" content="School of Life Sciences, Arizona State University, Tempe, Arizona 85287" />
<meta name="citation_author_institution" content="Center for Molecular Biodiversity Research, Zoological Research Museum Alexander Koenig, 53113 Bonn, Germany" />
<meta name="citation_author" content="Amanda R. Avery" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York 14627" />
<meta name="citation_author" content="David W. Loehlin" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York 14627" />
<meta name="citation_author" content="Stephen Richards" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="John K. Colbourne" />
<meta name="citation_author_institution" content="The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="John H. Werren" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York 14627" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=L. E. Berchowitz;citation_author=G. P. Copenhaver;citation_title=Genetic interference: don't stand so close to me.;citation_pages=91-102;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20885817;citation_doi=10.2174/138920210790886835" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=L. Beukeboom;citation_author=C. Desplan;citation_title=Nasonia.;citation_pages=R860-R860;citation_volume=13;citation_year=2003;citation_issue=22;citation_pmid=14614836;citation_doi=10.1016/j.cub.2003.10.042" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. Breeuwer;citation_author=J. Werren;citation_title=Hybrid breakdown between two haplodiploid species-the role of nuclear and cytoplasmic genes;citation_pages=705-717;citation_volume=49;citation_year=1995;citation_doi=10.2307/2410324" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Desjardins;citation_author=F. Perfectti;citation_author=J. D. Bartos;citation_author=L. S. Enders;citation_author=J. H. Werren;citation_title=The genetic basis of interspecies host preference differences in the model parasitoid Nasonia.;citation_pages=270-277;citation_volume=104;citation_year=2010;citation_issue=3;citation_pmid=20087393;citation_doi=10.1038/hdy.2009.145" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Gadau;citation_author=R. E. Page;citation_author=J. H. Werren;citation_title=Mapping of hybrid incompatibility loci in Nasonia;citation_pages=1731-1741;citation_volume=153;citation_year=1999;citation_pmid=10581280" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Gadau;citation_author=R. E. Page;citation_author=J. H. Werren;citation_title=The genetic basis of the interspecific differences in wing size in Nasonia (Hymenoptera; Pteromalidae): Major quantitative trait loci and epistasis;citation_pages=673-684;citation_volume=161;citation_year=2002;citation_pmid=12072464" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. D. Gibson;citation_author=O. Niehuis;citation_author=B. C. Verrelli;citation_author=J. Gadau;citation_title=Contrasting patterns of selective constraints in nuclear-encoded genes of the oxidative phosphorylation pathway in holometabolous insects and their possible role in hybrid breakdown in Nasonia.;citation_pages=310-317;citation_volume=104;citation_year=2010;citation_issue=3;citation_pmid=20087391;citation_doi=10.1038/hdy.2009.172" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=T. R. Hughes;citation_author=M. Mao;citation_author=A. R. Jones;citation_author=J. Burchard;citation_author=M. J. Marton;citation_title=Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer.;citation_pages=342-347;citation_volume=19;citation_year=2001;citation_issue=4;citation_pmid=11283592;citation_doi=10.1038/86730" />
<meta name="citation_reference" content="citation_title=Deadly Combinations: Hybrid incompatibilities in the Parasitic Wasp Genus Nasonia;citation_author=T. Koevoets;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=T. Koevoets;citation_author=O. Niehuis;citation_author=L. van de Zande;citation_author=L. W. Beukeboom;citation_title=Hybrid incompatibilities in the parasitic wasp genus Nasonia: negative effects of hemizygosity and the identification of transmission ratio distortion loci.;citation_pages=302-311;citation_volume=108;citation_year=2012;citation_issue=3;citation_pmid=21878985;citation_doi=10.1038/hdy.2011.75" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=D. W. Loehlin;citation_author=L. S. Enders;citation_author=J. H. Werren;citation_title=Evolution of sex-specific wing shape at the widerwing locus in four species of Nasonia.;citation_pages=260-269;citation_volume=104;citation_year=2010a;citation_issue=3;citation_pmid=20087390;citation_doi=10.1038/hdy.2009.146" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. W. Loehlin;citation_author=D. C. Oliveira;citation_author=R. Edwards;citation_author=J. D. Giebel;citation_author=M. E. Clark;citation_title=Non-coding changes cause sex-specific wing size differences between closely related species of Nasonia.;citation_pages=e1000821-e1000821;citation_volume=6;citation_year=2010b;citation_issue=1;citation_pmid=20090834;citation_doi=10.1371/journal.pgen.1000821" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. W. Loehlin;citation_author=J. H. Werren;citation_title=Evolution of Shape by Multiple Regulatory Changes to a Growth Gene;citation_pages=943-947;citation_volume=335;citation_year=2012;citation_issue=6071;citation_pmid=22363002;citation_doi=10.1126/science.1215193" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Lopez;citation_author=V. Silventoinen;citation_author=S. Robinson;citation_author=A. Kibria;citation_author=W. Gish;citation_title=WU-Blast2 server at the European Bioinformatics Institute;citation_pages=3795-3798;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824421;citation_doi=10.1093/nar/gkg573" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=J. A. Lynch;citation_author=C. Desplan;citation_title=A method for parental RNA interference in the wasp Nasonia vitripennis.;citation_pages=486-494;citation_volume=1;citation_year=2006;citation_issue=1;citation_pmid=17406271;citation_doi=10.1038/nprot.2006.70" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. I. Mester;citation_author=Y. I. Ronin;citation_author=Y. Hu;citation_author=J. Peng;citation_author=E. Nevo;citation_title=Efficient multipoint mapping: making use of dominant repulsion-phase markers.;citation_pages=1102-1112;citation_volume=107;citation_year=2003;citation_issue=6;citation_pmid=12928774;citation_doi=10.1007/s00122-003-1305-1" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Michelmore;citation_author=I. Paran;citation_author=R. V. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=H. O. Muller;citation_title=The mechanism of crossing-over;citation_pages=193-221;citation_volume=50;citation_year=1916;citation_doi=10.1086/279534" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. Niehuis;citation_author=A. K. Judson;citation_author=J. Gadau;citation_title=Cytonuclear genic incompatibilities cause increased mortality in male F2 hybrids of Nasonia giraulti and N. vitripennis;citation_pages=413-426;citation_volume=178;citation_year=2008;citation_pmid=18202384;citation_doi=10.1534/genetics.107.080523" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Niehuis;citation_author=J. D. Gibson;citation_author=M. S. Rosenberg;citation_author=B. A. Pannebakker;citation_author=T. Koevoets;citation_title=Recombination and its impact on the genome of the haplodiploid parasitoid wasp Nasonia.;citation_pages=e8597-e8597;citation_volume=5;citation_year=2010;citation_issue=1;citation_pmid=20087411;citation_doi=10.1371/journal.pone.0008597" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=O. Niehuis;citation_author=J. Büllesbach;citation_author=A. K. Judson;citation_author=T. Schmitt;citation_author=J. Gadau;citation_title=Genetics of cuticular hydrocarbon differences between males of the parasitoid wasps Nasonia giraulti and Nasonia vitripennis.;citation_pages=61-70;citation_volume=107;citation_year=2011;citation_issue=1;citation_pmid=21179062;citation_doi=10.1038/hdy.2010.157" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. Raychoudhury;citation_author=C. A. Desjardins;citation_author=J. Buellesbach;citation_author=D. W. Loehlin;citation_author=B. K. Grillenberger;citation_title=Behavioral and genetic characteristics of a new species of Nasonia.;citation_pages=278-288;citation_volume=104;citation_year=2010;citation_issue=3;citation_pmid=20087394;citation_doi=10.1038/hdy.2009.147" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=K. B. Rutten;citation_author=C. Pietsch;citation_author=K. Olek;citation_author=M. Neusser;citation_author=L. W. Beukeboom;citation_title=Chromosomal anchoring of linkage groups and identification of wing size QTL using markers and FISH probes derived from microdissected chromosomes in Nasonia (Pteromalidae: Hymenoptera).;citation_pages=126-133;citation_volume=105;citation_year=2004;citation_issue=1;citation_pmid=15218268;citation_doi=10.1159/000078019" />
<meta name="citation_reference" content="citation_title=Genetic Maps: Locus Maps of Complex Genomes;citation_title=Gene map of the parasitic wasp Nasonia vitripennis (=Mormoniella vitripennis);citation_author=G. B. Saul;citation_pages=3.198-193.201;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Z fur induk. Abst. Vererb.;citation_author=A. H. Sturtevant;citation_title=The behavior of chromosomes as studied through linkage;citation_pages=234-297;citation_volume=13;citation_year=1915" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=X. Wang;citation_author=B. Seed;citation_title=Selection of oligonucleotide probes for protein coding sequences;citation_pages=796-802;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724288;citation_doi=10.1093/bioinformatics/btg086" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Protocols;citation_journal_abbrev=Cold Spring Harb Protoc;citation_author=J. H. Werren;citation_author=D. W. Loehlin;citation_title=The Parasitoid Wasp Nasonia: An Emerging Model System with Haploid Male Genetics;citation_pages=pdb.emo134-pdb.emo134;citation_volume=2009;citation_year=2009;citation_issue=10;citation_pmid=20147035;citation_doi=10.1101/pdb.emo134" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Protoc.;citation_title=Larval RNAi in Nasonia (parasitoid wasp);citation_author=J. H. Werren;citation_author=D. W. Loehlin;citation_author=J. D. Giebel;citation_pages=pdb.prot5311;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. H. Werren;citation_author=M. J. Perrot-Minnot;citation_title=An antennapedia mutant in Nasonia vitripennis;citation_pages=319-320;citation_volume=90;citation_year=1999;citation_doi=10.1093/jhered/90.2.319" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. H. Werren;citation_author=S. Richards;citation_author=C. A. Desjardins;citation_author=O. Niehuis;citation_author=J. Gadau;citation_title=Functional and Evolutionary Insights from the Genomes of Three Parasitoid Nasonia Species;citation_pages=343-348;citation_volume=327;citation_year=2010;citation_issue=5963;citation_pmid=20075255;citation_doi=10.1126/science.1178028" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="twitter:description" content="Nasonia, a genus of four closely related parasitoid insect species, is a model system for genetic research. Their haplodiploid genetics (haploid males and diploid females) and interfertile species are advantageous for the genetic analysis of complex traits and the genetic basis of species differences. A fine-scale genomic map is an important tool for advancing genetic studies in this system. We developed and used a hybrid genotyping microarray to generate a high-resolution genetic map that covers 79% of the sequenced genome of Nasonia vitripennis . The microarray is based on differential hybridization of species-specific oligos between N. vitripennis and Nasonia giraulti at more than 20,000 markers spanning the Nasonia genome. The map places 729 scaffolds onto the five linkage groups of Nasonia, including locating many smaller scaffolds that would be difficult to map by other means. The microarray was used to characterize 26 segmental introgression lines containing chromosomal regions from one species in the genetic background of another. These segmental introgression lines have been used for rapid screening and mapping of quantitative trait loci involved in species differences. Finally, the microarray is extended to bulk-segregant analysis and genotyping of other Nasonia species combinations. These resources should further expand the usefulness of Nasonia for studies of the genetic basis and architecture of complex traits and speciation." />
<meta name="og-title" property="og:title" content="Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/2/205" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Nasonia, a genus of four closely related parasitoid insect species, is a model system for genetic research. Their haplodiploid genetics (haploid males and diploid females) and interfertile species are advantageous for the genetic analysis of complex traits and the genetic basis of species differences. A fine-scale genomic map is an important tool for advancing genetic studies in this system. We developed and used a hybrid genotyping microarray to generate a high-resolution genetic map that covers 79% of the sequenced genome of Nasonia vitripennis . The microarray is based on differential hybridization of species-specific oligos between N. vitripennis and Nasonia giraulti at more than 20,000 markers spanning the Nasonia genome. The map places 729 scaffolds onto the five linkage groups of Nasonia, including locating many smaller scaffolds that would be difficult to map by other means. The microarray was used to characterize 26 segmental introgression lines containing chromosomal regions from one species in the genetic background of another. These segmental introgression lines have been used for rapid screening and mapping of quantitative trait loci involved in species differences. Finally, the microarray is extended to bulk-segregant analysis and genotyping of other Nasonia species combinations. These resources should further expand the usefulness of Nasonia for studies of the genetic basis and architecture of complex traits and speciation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
    <title>Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wrh9ocsNw-TRU8X7-bHbQ7PaXlG9nVGYaCfVYntPD6c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459412","apath":"\/ggg\/3\/2\/205.atom","pisa":"ggg;3\/2\/205","ac":{"\/ggg\/3\/2\/205.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/2\/205.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/2\/205"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/2\/205"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/2\/205.full":true,"\/content\/3\/2\/205":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222hpUxYXzPT3uRSvh+kR1yU49TKewqDdnDXb85ig1Y9SgKuYOC5FhYuo+jTt\\\\\\\/sSNaTKy9yqSBFamAHrL7QaXtX9VvtPt5kd0fA3Rh+ko9ufXOU+lqgWE5Ec1f46qBgJyRwYME1qtMZYQVAvEMO9hPbFJ6Kix80SLux2aW+gSQ8Huib9iHwjGOgL4sQlq3hyshTBJRZ61iyuMJSoym9wofvYNjMRFHXXult64oAzI4fkPRUxsLHW6H4KcyRD\\\\\\\/UcOSjGCcQqnitqDFb0FCAOYjHBDy\\\\\\\/8Rb75WzFf4AbZimxUAezawi7ojroEP97EaW5ZgxfBRy\\\\\\\/ROAw1fvbU8oY9HfG1+WGlVXDEHHAOxEemjuWdwjsO\\\\\\\/q1JzDKnEv+SmeHJ5tolxJj4TehrtmZHtfLUObI7dYpJMXMgrd6x0kkoFBZ4RpRomqEsuTyMdVpR6z5EFZIBGuAv3mGUzBtvHD7eU+kIMB94kSPHWF6R+tFA9f\\\\\\\/z7cv0oa+jJaC8yffiLhFDS2AMvtgeNVaXlV5+5YO2gOtgPnNFa4VnLVvOKYoF4BOjY03GxhTPmgxyz3ti1peeFTa\\\\\\\/4fSL6Ccd1tlpsRDZ87bzKCJr+klwUA+ofc2TH+EUG6\\\\\\\/UnJDZTW9B8h1BMuXHwM4IHzKWyhuXJxuoNHWvaPNl\\\\\\\/Z+lYSYfY4TYP1AaKnfmTRO4xxLOLias7HNHaxDr7yCWYP7SXLShvZ5Jt\\\\\\\/rhUZ5r1r2acdvUhzzExMUku+UVhreqQs09hMjIJdCMEmm6B8XIfhICggmoZAVHu9LZyEBnI\\\\\\\/ZfalA1JG0dPg3lbzIL7Nub3ZNe9r6C9y0pUKMyRnHMcavhaRO4NVoiv7gDhc0zlKi4juee4Kq1Lbx1Ja7JLWm4o0RZ2H25pNT+SxMT4Np4Opk3pscdQ9gIlH99L6tEwcWuR2Eij1sNpFwVozkNeLLZ4LErdc\\\\\\\/MWHtQTLw2EQqA7tGv+ddQQPBD0d0boW7b2fHzuCcX+UmwquTNHW2VxG00MmWfLBDldC6EzLIlHa2v4ai\\\\\\\/gy76ErK0Oy54EKjbW6TfngcbQFsfljXrPV9jE97wQZoMEEYUxAJVROWsOeUJ+lcvu33ukJ2rLuv8xZv6DhRXUWnq\\\\\\\/UcMzaHIqnnVSlGyKEFYj1fU6F2V5hRUwHz+udArpXP1\\\\\\\/2sQnVdhU1A0eeWk\\\\\\\/kEAF2ZbX5jf+sFgNFN9mMXQW58NJx8UHwYC\\\\\\\/ZXEueyP5\\\\\\\/6wq2OQ4B85vJMjVUBec\\\\\\\/jZKFen8ds0OgrvX9\\\\\\\/w0x7mfR5UOPpunxKimQp\\\u0022,\\\u0022iv\\\u0022:\\\u00229yVpFjSUFdc\\\\\\\/QUB8ovHDFQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227416e67f984ab9d8976787a2adc713a36d189d942fa96d1aad5e425d00909208\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZuqLymyrLj4vVAuvKbsVngqQLzP7r1G+9nE2MiUne9m7a2at\\\\\\\/zJXeRYoMC7XfR5K1ar0J1AfDpO+HfmLqFI4oSS\\\\\\\/UYsbELmK5OrTvUNEV2f2mZiy1pbgLP25dDlDenvbC5gAFslYX3ocAt+Rx+mPNKSdJwo1fAvBm5di5FqWGxd3rRGcWG4HvqRvV\\\\\\\/19JNznKegGKVLnE7bYR6hl+yYapgcqv6zmnHLpaHDWP713Tjn9FTNRf43qlvszihTAb1gRClMkDuOYFuCxPr0XmjbCJAK0GIsYnUczT1cGyMzhFluFyUzGQvzuKaDX+weU3zTxIrO6CHe1nNx2+kiqjOIS\\\\\\\/GBC3uGNUQeNvfr9C0G00dU07jjRAfUGAKmTIgttiDVYuVltGv7qPPwVUNsLK2x3Jn+EDRaQWt4AQ4etlDlT\\\\\\\/q5OqS2Rl5lQM6otS5OJrJ\\\\\\\/gAj2dr2ftbsvjXxtruH4VpEvpdj73qM4Ur0tCSFn3j+mx\\\\\\\/WJmAPYbkTLf8c5cJxsCMNOWV63BUEdfcmNEAg1VawKUYJtV0z9mqT0sOfRN1ynxRMLqbtXWRc7cRTI5g4JnC91FFWcz0d9sSOSFTf3O9pfoVYTIRZgOHidohwlEo3CrVCzYHeYM3FvKElQtcKekdYWnJSWWDJs65ap0nIvigblh99NVDf5F8hdWH5EGfxNCxAaj4gTZJbTYiXh4TcA2\\\\\\\/JFtqeGnBim0Z0XKhFus4b9cMML\\\\\\\/gHqpY2UGGGz75NvciJ3cWo8qpy8QjhXHw\\\\\\\/A2scutXjzivnMVuue\\\\\\\/a\\\\\\\/LRSelH4qYhgOAF9FXTMdBO++6t3KRGrx9yVEDVlmPKYhS6bFemWy2fOBvFnzeQ7j4Vm9fPWxV2eFz6JsdSgJ1AcrxwLVhMF9FNcBLXhF3gpwzJw3is8qNLtuefOUbkRXkt\\\\\\\/We+Add1BHBFsed1+oq2wgdcIIAhtuFZEvON14lTfdI\\\\\\\/oNuQCorW+FrDnv6PWXEfyPGO0WizaQPBsMd2r9szBRMtzuI0K\\\\\\\/uqUTxFoYrmzXcssaHAEmEEQ1eX3fz4V2fHhQ5kyTn3NwBlZoaiG+ZZAb3U0uVhfFlGBqZe4xGYeRl\\\\\\\/Ky7QtTLsU3TwzJo8MjVzbhSEuvuaAQ1nJ\\\\\\\/z77IC+I6t5cxzeT8idgI1ErvdhjAT3tbyqfMz3ksk\\\\\\\/o6lRROJ9Lu2VVAZwcLVLgGBrM0Ujl5nQoF1xkpHJIe43UmLg3Odcs6k53jThvq7Uu7G72VdfGLBH7GmmbstzNUKSJiirXUUTGoUBRguQwEOqmLUkkoWkUy6iuFQ6Cu0zu\\\\\\\/SV6gOCH4nc1sP0YiipG13SclvEI1yXEPiiaUmmBtJZBmukEk94jIDAO4qitGQXESCIpP\\\\\\\/7c8fWjJW9tkP2rVQxplLxzOUsF7XsWq6XXoGYhr75oEuwFNn6u2lyfsjFW\\\\\\\/IDYm2tlnzoLtVW3IeemR8zCJZkJ2Q487EP6tQWGZb6TAqsMvnS75NlC\\\\\\\/4i+vIGvHpQP1iO3jOrWugQzZF0XkybRuiNlb+Mxv3Pa70iU4sF1enHVK6JwE6A16xRHFfULOM6FuPlmmVoKkrBhtpLfW7oaC1Gu8xGN5hbFASuNaM2SU9ALhR+6Am0Dq\\\\\\\/Nn0Mi4Orf\\\\\\\/HkVXtD8emLEXuWH2WvYVYpsUbk9ICbluf\\\\\\\/5fV8d+lkIkwOSEI9A1hpkYvdGgvbdeL6FP6tKCfDdRzalARrMC916o+2QWpmdtAgWong9uaDnNtteL0TbdZjrY6irSnrdEjdhVGLEUrs6HkCt1aVBQnpkkgp3TH8Ga3imkNtr4DWnewT1N6HF2LBBv9qwSnuv2aQD15lWWyKi3Q4GzfJal\\\\\\\/dtKhuAoZT60bFNF+3kEBqrnF939UvliaxbHZG+VLAEIXljbUVOiwuL1buaJQi9r5Z5tmdf7HnqO1Nskd\\\\\\\/1IkCPomMvmzcKrO\\\\\\\/1n7p5Fk14Fsr\\\\\\\/eRtS1Ig+mh2x0oZoSvcQMegX3\\\\\\\/FJWxhwAfFMcQJEE5C9BHCCaZHKU0z6ZkL3KKzpvJ7CZzFE7ojYsVBVY5n+tVljVdaRztK2Injr3WGHl4U6psDDD2\\\\\\\/G4jZj\\\\\\\/ytZU8DUjCchAYHTaYREP7OQt4+E6wmQLMBbVpEM9s0yIFfj+qXaMKJBZbifEAvWRK5ezF\\\\\\\/y8wXm8AyQwT+9Y\\\\\\\/PtRtAVTMVe6FQH+sa5bKwyc41lqOAyzleUWSDbxGsBJLXC44+QHAagWs7dc\\\\\\\/9NA5+Xt0fEkJn308TiGKVyx2fyV5tH0vD8WeQvdmajlhOX8CP9m1C+RZx6qyW4kMum5zHJ1APNbX7zc22EjJV\\\\\\\/e5U4YYhzC+5gsLUz277eGQSuVooVTY8K3CY5dkU+Qs2bFVEFQ5j7l3iJ4MytyEw8qH6Yt32v3+pm1VxuQIQUPZT3zVriHzf\\\\\\\/mgNJHIGs9R\\\\\\\/h9WbspHVoIP7tgVpw3zVicmat\\\\\\\/ClTDzm38dyDX7M7tfweamEsmKzxbUbf66RDisWyk8YcZV7r7xtFUqKG5GdDWqJqqoJM9VClvTGlRzNm4EPMjOv6eA0SmwS2sMDAAXzLOUdI7E2o5ekBAgtm1ETCpDwl9MyHNRE4115vIhrjwNb5Z19kZ9xLqrcdd8INICbjrc6G87DJ9k3vTC4H\\\\\\\/IyQtvqe08JLVGxfSrKWaa7hCXM9qelqF\\\\\\\/X3hwYNpX4AjRt\\\\\\\/+tmIcbdP2YNrBM9+1w36f6exp6Jq43MYo5ScEgBpxfBFmk\\\\\\\/UC6o8tUKxkkWSbc9PIYNf+WlRnrY6OGoldZ0DytYyp9wjQ\\\\\\\/aWlR9ffjO67eMRwVRgWnMAWI\\\\\\\/ROqFd\\\\\\\/FUbERjAJkv4lexR18MUcRDpa45\\\\\\\/NsZ8PsjeoRZ3bZEZRXnaOQeJHqDbVQ3FN3KVthHxSIv9E8qYFdAU8CQDCJSdLki5X08QR1dyn1d6yGNBEUO9ZqJW5gxuFI8HUnk6AbV3KjvmZ7PulNiOosWIO1NhnHYGwCfwNhwAUQPTCHDdduMdS5bnva+ODc6axgOPx1h2WWnaXg\\\\\\\/UKKbtneG0IwiPGjhzlVEY9OkEMKhOWmu2GU5bOI2aVF8DE8Z3I7KLH3sX7p0Mn8JXYRAllws4xweMB4AHpL883yBQdVs1RNwDMTCsPVdh4fyKWw0sXEtL1WQr+ntnQxIpGAPg\\\\\\\/jnMdaGupgDqVjBf5ETxrd+LOs3k8GtPCzBGPAGjMi5n\\\\\\\/dN5D0l6qruNrVA2XbFQCioKHzw7ShxtfTxBcvAAIvYY9HVdcTGBVh780yYftd4\\\\\\\/csSp2znIU+FNMSbzOJ59X\\\\\\\/ajYI364K6An0sWwui459tEB5c5WsnZMngFF\\\\\\\/0BSH8CR0ypw9pSX2ogrf0ZuPAo6G4rEBc0SXI5SI3o7nHw0ixiNm2GXOtEw36hpyCHIlJm7Og1KsCVWySBg3CPlMMcM\\\\\\\/QMht\\\\\\\/UqU+IaY4KapjQm2hKTkYOuz2mVX\\\\\\\/Cxm+BpVoeFDXfFxecdICyUAHefbHv9B4d6\\\\\\\/tFrsU+MJks9xstx9KSTifoQW7Wfg\\\\\\\/j8lVi6Ulz2t5zTzMkgPnnPltvJ6Qjpxn5VJNSISRKWBTbRh7URC+DMRa9zNiJg4sUSS5\\\\\\\/zARg1rkKDnTbGEWO6amtVn9fh\\\\\\\/++qPPdp\\\\\\\/h0Zn6yNJ8nsIXRoqLPbXSByU84vNnY6ipDy7ZAIWNnrB5w3BVrJPGi5T3EsGMGf7jNmoltBP21BgyBgNq\\\\\\\/P6Vw8e8dpY7giPdFxSTwcIwEQ02Fc6YfImMANDp0LW57iu3cJXVGx\\\\\\\/89Cng50qQvsmnLEGfOSuKcddnpm6dH6TjFUF6shApf6khHL4\\\\\\\/aQSTWrJujHEg3AL6d+edPJfzAq4odWLF1bmmvYmHZcIhHj3wgYo+bbBDViOpgazWggna+\\\\\\\/tUPGnu03SLUo\\\\\\\/CFjs1pjiDKbMdSiusR8Ya9S6mRi5eYdn3+jKwP5PnXV+DYRNsleNI9kZ8PjPFj\\\\\\\/EFsoRdSCTKG\\\\\\\/HKLhAi\\\\\\\/RXtkLrOE\\\\\\\/2pmO5AE7G+vgU8OlEwbRdbvk6TS9VP9RxUy4zgJNKZry\\\\\\\/wJMz9zd8CCxfvQ\\\\\\\/TFwu4mobEvX+G3w+I5T9cNFUQojjG9a\\\\\\\/1PkrdpssuVdN9SZVRKLLXjQbj6jF7mAlpCSvidb7COy8Knsr03M4PfWTS9XNgNu0jgBu9qce8RJVlcuZFHwQmB9UPcqQkOwTCldkPuz\\\\\\\/IG7UVPavMqQ+yhMxtkwbJNYXfLYjQNlZVz7lJI7vpUK\\\\\\\/dRYkNC+VB5o4x9jbLLvnbYsN9Xk2XsZL1hpgRXlJNMUi++S5P+3nY1i6knFdfMkx0TedPK5eOOkdhQ\\\\\\\/N+DGEZfWrFsoq7Mna9RtmsbU2IUNOh4\\\\\\\/lH3iNmKWNnov0bLp9aDTG1hbn0AUGOA3mXiuqNu1NMDZ71dEzBflT0FIWrKCe+MrxOPK0OssfzzFdX\\\\\\\/JL3esO3T8I7K7QxmPsh+q3+KGiAth7I1UDVgHEuTU197hTWoqKi94ZG0nhn9bIZoPD8fdsNm\\\\\\\/2ydVaZogbjSb463NeCFti2Mz3u8F1PvIOnDVDrbAZ\\\\\\\/8Wd285bPD5+4EezMnDBPxOxxwwJwr+fISMYafsNJvorJNZTwuX\\\\\\\/y8OZczZYJ5sy0Xexhr\\\\\\\/WuvXLZiqmb+nSgU9XjzHj\\\\\\\/1rQFK2LXckIZIcxgNYiDjsbR8\\\\\\\/AjHoVzNe5VLMaifogYA020ANjGxg95er1ysJ7\\\\\\\/8mxcNys1Mber7WkYQHwJVp\\\\\\\/6zFsHlnad6r5OhYypOk4JtpqOqWlQBa5F3Ck6R9ufgbPUQXp1H+5dfAR5aG4O1lqO1d\\\\\\\/L3+8pOdg0u5naBgmlhsjlAfXfEWfG6Q3WbUCsfPnG989lE3WbC7m1jzdztQ54JwHS4Dpo+WUQeImSe1TEYJSDrjj2Logb0eha8HSl6be5uZ3GJ9aUYbSeDEqYFXl79LK6Pn5\\\\\\\/SpSiJdmxXNXLnTyyURw+XMNH\\\\\\\/L6xd0603KaOoBp4VxLmDspttHEK9Q46ua8ved0VvTCUytaqR1syw8PEYOFPPdiAKHKNnPE9gFaL\\\\\\\/pAxmQRmDPFE2X\\\\\\\/je48uOkvSrEhcg+bXE72j829eQV4nXC1OT+CVeg\\\\\\\/ih6jAH3S9mwlDtqRhqRYEwN04Rl4g6OzmC72h71feE3WfFCWBqJK2gLpMfalyAlr6kUKG7ZeGQ6Q9xRAuzOapCxnMliqQVp01mpVg8v5o2rZ5WBPtiJFnomkXvqdrSkjNPzw2YyzbcTTPn9hrHtNNqQcBJCowfaXYFWTW7jsEvx1WeoRzJ3eDXS1kZc2AJ3udhtTwkvZDYg4mezY0Wzxmji4NthobgIwlX5hdzDrtCTRNj951BAM12G2xwfUFJG4Jx9sENgggYv6o5WVzMt6+YXMxNlegcOzuVT1MRfzGkwSO0V2o6LJZ5ejCmEPvcXTZbCx5DdyCZDxNW63Oaawef84MYxioFURsZwgV+a9h6ToWODkvl5UKfPXq67dg1lD9YUk3E3cf+\\\\\\\/Wrm8BpX0ap7L4HxZmoTYGcSf8kZYds1SsmJX2ES+43aznnVQAiuALZhe3xm4OWl3ZydxwL1I0k4C22dZ16R3BbOL\\\\\\\/sY33WoCO2rlh7KVlGeeJ\\\\\\\/OQX07eD63VXZgE63oHilyU5EG72O4H47\\\\\\\/ISc1o0JbJzvm6U6biE2a\\\\\\\/UT5JdeXheS6rAJ86EE+S8H1d1O8i\\\\\\\/6SQFRmmxlKD\\\\\\\/6Bhl5zVK57J4HT8yv+VvbvNeqhK8oPhie+khilZOXoEgCmCcpLEgpFOV6BcRqTGAi5UHVGQE9e9+wQmN9+9E6aDw1kmQmUJxosHq33eU6jQlkGp+Fn6uRbAjn7548Ld\\\\\\\/99V+e2lFX4Q63g+mlUNNoAeTPKF+EwOdXCwSHTnQE84WL3\\\\\\\/W9oyhbip60ifOMzv54q\\\\\\\/3OBdBudKwO3OiPCogdRndSGrWMGfJSBLBCUNXEY+BXxSaGCFah6kpWXMpFeP7yJOheVJbxllDCem8eBpurponS8B\\\\\\\/TpSNgN3NU8rHxaQyaic6FkIXFz\\\\\\\/shBjTXU4Tzt4vUyuNvvcOTZfQD+v9XEWhKKfMyHzy5q2Lp\\\\\\\/d9Xlp5V6iFqNGVhOzRjMx6546vJZ\\\\\\\/TV9Us\\\\\\\/fx9yZZjhyTBGvSfSZZYeS3PR+i3UV\\\\\\\/rmocpEJzMh8yNP8dVTb8JFeIoqRwtCnYo79sXphEPkofl+VTUJZDlaNTKDDyikmKH5jEnUqHiLSZ11\\\\\\\/aW2r4h8P2VVvOwt26JTd8vtQClTBgsQFJjIEBmAHlRmNF1K\\\\\\\/zfYezAAqbGUVRRCtoUBAUHghzt7iTatrcf9vssv1JPEpr5FDV71FhNo2rlFl4YqPX33CmhDMLm+\\\\\\\/8c0fwIAeh+ikpqdXpQpCcTRP03dd0WLxpTiL++N5dTOJOIMKuS\\\\\\\/B+1SDBJgaYQV07efxk+vsRKFzg+GzFvR07McP52s4YyU2D3+P4H60ep\\\\\\\/T0d5zMLvF2FonwNXcAsANuen7JFlAM\\\\\\\/O6C7GUYPqUHfNTZ5iP6b96Qar\\\\\\\/XPOZKruofTZqqr4NIR+5C6AM1+yVdbv4BXFl+CQN4BP0Rc+EY3qmUWv4Q9\\\\\\\/4zAskQhbzr7WJQEN7rxO+WlHk0d\\\\\\\/O+tmtcByC8CmnFHL3CGhOI0mrvpgxjyvTXTYoVhVsZRST3zt\\\\\\\/B3wEV8DV34sGXEo8gxiWz6hts4Lv5cBb5jAddHV2VlGHc9lDLnRVx\\\\\\\/NIbZa95MfhyxC\\\\\\\/du9yjkDseJVrxLhwbqwlrmN2aI\\\\\\\/97NkedJB1MMEaSFvIhWOQV5mRt6KsopLp1UiYguYfmHUzyqj+oiznZrKAUj1W2MUaQ88gtOEV7X++mcPCFMo0lRa6GGKzcfIMHuHuSYG\\\\\\\/pK87WWknF7z7rV\\\\\\\/o9uCfvk+6kM1H5z04LueTuM1MU6Q0azxkij3MMzHhF2LLMlRHeJtIe2vKvuCGGjLKX1T\\\\\\\/ic4RCbHX\\\\\\\/HA9gjE45CYMGRBTvGhtJRscyqO1hcjuycNwC2v\\\\\\\/6p1Uj0emJ0dvhH0GytZIgCvw+wlyp1r729ceOFCicV2C\\\\\\\/zxRupl0MQMNMbcDnsUL6N\\\\\\\/CGZWqvF50OqFLDREdNtO8vz5Dal1mZKYvlD5NGqPZMxU6kvawB9poZmgosLudpcnOY37XlF6Qssy5Y4d\\\\\\\/7Tnm3AN5qjEJND72zdRf6H3ZtCfZDJ2Tckr61maIx7X1QYoL\\\\\\\/1UghiEK\\\\\\\/BBrNyl9sBy08Ix865g\\\\\\\/O2TJXH3jpFrHzYu\\\\\\\/3t\\\\\\\/+M5YN\\\\\\\/Pm2FhDIGajDWSwMYLGdpgquJyTzK7ZrA4gIp1Hx\\\\\\\/hIO6V\\\\\\\/4klbJD2GUBaCRHhN\\\\\\\/zu7IP7IWhJEeuChQI+XJTqrJ0\\\\\\\/t0UyeW8G7JDzeeo\\\\\\\/A+gwSIr2QtdFxel2fcIOIq7U3\\\\\\\/1tBqjG67IWQiAvu10GzaVy\\\\\\\/9bxaaER1a39pB8vYab1yZ3LFxUYyyPK4Y0zfuX7bR1dYrWHD3BAlQszvE7tx8wTupveAgbTJX6eTy152d3gCtjvmvaeAmoBkNXXia8Vw4FLJxqdObfmWH6lId6xU\\\\\\\/VEBLzWinVghQocn+Axzq4+xK+FwdO8kMfIwyBNZ4+Ah2Eidq5kiExLpQSPVlZuODl330inAy2O1ZmFVz9Oh4wgalvFvzr+ZDWZVUWuxXzGQix1s+zvaGwsTZJce4Ywtd2aBwux6X5pLUz\\\\\\\/lp2v\\\\\\\/4B1DnYwnhej2I1WKv2e7+\\\\\\\/hKvPL\\\\\\\/7gXBUqbHRBMfhe5U6oX97XF7c9rjfPFD1c5yJ6PwTWRfwpfBvPWQH2OzHQY+Xez5vfFIT8zGPruz\\\\\\\/u43aIGoG3YaDQS2oBNS+Xf86tpeUnwXWzKq7rZJhPmuDRKJsSrgg5O4n2h2ThUvaZz4qQufr0fjHjsoVxVHAntHbZMtCVf3B4Q2pX084X76mvwNLjVvwSiNFLOx8vMZobRCxd8aizPwIECQQy6rwGDdjUDrqW39yCf+H0mOdTVkmnP3SxwOjXBXZttz6SEn\\\\\\\/8VcwttdB0tNXCtCLHZbzvhXdC38YXodEI1UeIB4J5BbE4kukYwcaI3frNjJPfzCA\\\\\\\/DeJ0JQM29cuh2wkvTFqxVelUPXLVX60gjaanqrNvJ2JlcwoHawIesh6wf+t2I81My2magN3JO02EAlREfsEOpEzzigh4bJj89Y2WM1bTZABzpmIvasmuraQvGhCyGM9B4yj0s\\\\\\\/UVxt9CAtOqKfxrwfj6mD2bmhz7E9XcJ8oq5dy54jdSW47wXSRZpctkl6j2i\\\\\\\/kI2kSwrqwOFzeSmnsPRe3N0hmjRakaruROTobR2FaGZ6aM0i3jXrBN1Ouah7mU\\\\\\\/C815kB5GvWz37ABFFm9\\\\\\\/9BR7+FjKC2SlOj1dohdM1ry1UCqziE3lylJr3mFM9JqHYU0NG621LhSdpggh+76FHitkq2gvrYLle68haT5UuaPnKp6hFu8ji+nmBSeAyyJaSs3Q9KcUWbFIBToXIPcRst7a05Kbyxsr\\\\\\\/bCCaL\\\\\\\/jRlUqDm\\\\\\\/rpGVitTjl1rv7ux\\\\\\\/U+nUNBPyDB987LCxSsLdygJ6Ao4fiaSJiP2LadCMuhi3px3Dq5T9kGF2kDbziCXTdhx6JhYp3Yjvhq6uKzsqhorrwzN2dEEM0\\\\\\\/S6iTrzuN0SDRFVsAHU7ea0Wa3Z9tbzAs+Jfh1wZhDoyra7h6EUG3uPXcLw2lC+iQV8qhKrtip2dDwfInHu39tyJLaZjP9+1d32Qq3nKQm3QboMYf0EmK8dE1tungJU4LQ7wbY46G9GziarEfl56LtEcS7dFLtF3UDLfGHZSvn99WEbaTUQIWTdFr7cvFCxOJqDiHExREm27wHrXeJO\\\\\\\/W+YoPSROez14exk\\\\\\\/4U5+DxIIMMtzmRNb2QDu6ZXycciEa2C9GSXGFQ9+lTPh6c89kpmlsCMtq5fmvyicvT8+m4sR0Gd97KfBR5QjycgRh\\\\\\\/maGZR96Z+9NkP8y7HiGE00I07WgikcM4LRM0ugrzbJXXhjeExmki0lHLCOOe7ZVEYVO6bMacsF9i2wCz2TBTN2RjcNZTK64pQ9xznK3+aNsRLansh6qo32h7hJRVX1yKAge5d0gyyrKvJ07xT0uheJlrAQL5hOFYegS3P3f1EC2Bb06Kr1iYTSSjbJd2M33s3a+O3QPNUaFx7pWWOwXpEeS\\\\\\\/tdEzebE6oJQFPdgXmAG6UjZEbl2RGnj6d9+qVFjwF3hXsqGHoPYot0+Mn1ycPq8+XUyWQVWlz7QdDx4A1Mwt+2tGjP7gIA0JLiuelnmlFRWxP1Gi6+mjIruG\\\\\\\/uDnfqRVOPQ1AM0buwEOl26KX7FVNW27tTlOELXP6ieSAjc3MFRCtO0SOjSr2RFjKmYU0oGgdPHQQPHpkH+nqd\\\\\\\/VrmQ959AJgbAIGD5QzC4ML7j0s6VOGhofE69quC5Pw3ekXFG4OPe++4skVGuoMPS6YYwFcDTc3Kt8oo3hhD6MXybGUU6lY0em\\\\\\\/DSjG6JGOb9o\\\\\\\/COj8SygFOaLctfdE6Y1\\\\\\\/MXnRIqhs2ZtMncr6DPgwWb7ElGod5PRyEzBm4zICXeA\\\\\\\/AI1AgjURUfy0VkGaJ+fW9YlXiSpax\\\\\\\/hPc2t8C0AR7nBXyMCfmOq9Pj6Y5jpjxtIgATuzLbalfNeKJs7IO+im5CtCJ73QrW4137fwn+U\\\\\\\/DLaurL3YzgbyKrDcplGaKv\\\\\\\/d3D5Livnp\\\\\\\/IxfBKh+CRCL+s22OuDl55owrV3LDsJlxyxeEkbLBuE9oV8+dbczHRjzPgVEJXJhSKZ\\\\\\\/uQKK4C6U5FcYsNg4t20joCL6X2XQbfVLp5Zs0c3XORpUpso6Il0PVcvuzOwnw5aPugZ4s2\\\\\\\/VPccizEnd2hZAZ+le\\\\\\\/1+ui3yYOrGOGDWwLLaWhLZechbzNaXqmPk6RgDVxQwifldwAf5Eu7Zlr6xBlXUVbnHJNi4cZkRTckuijC26mSoHXfdxOw8rw\\\\\\\/xOwqVlSUdk+nwf611z6xU96C8aeGYFOYmIi9rfbmnm8VE0JNxO1JMlsZnnQEo8AxPLeX3As9yHOvOYN5a\\\\\\\/OKo48i3O+k+NI63HPUXDbYqqdprk7l5Za+y4qb0Ft\\\\\\\/y6NpFKPlDccKwK8gQ5SS5FEBV2WgfFqAQWaRToX5y9RKfI23lG2zLjGXNtZpMs8TUYKnR4yopqakcRPI8Q3YenYu2r2NI4kfdYJX4CZnzueHgr6LcQoOfnl+D2AaYZR5cj0YFmXE\\\\\\\/FxRqIV5PMfhecA7ZyR48ub6gRslI0SVNuNS1CqZZI0N2mIxnFR2BuS0vnNw67TIrueXAefQflpebBECppQFP3DN94rya8b9nZSXi4\\\\\\\/5b5b6JUbAPu1A99BukGnJfhNAErbQqdXVFUmjrOqOUGG0mVOQdgb4fRY\\\\\\\/M1AT1QhjLjPEJV2pSCRY6\\\\\\\/eXt3VI+PLwANZaWv0l4A\\\\\\\/BmLQcVA8ZAYOEq9PX5WyoQqvsQDBkbR3tb3GwUauUEkIx7DVpJQezup6Kw+G5thcah1ZK+M\\\\\\\/eTa6vXCnPQVP6btFVh9VNg\\\\\\\/vV5HrlXlk2SXPfvQwjOZr6VqNCBT9t9f15Q\\\\\\\/J0DOaiQXfqm\\\\\\\/TmjkETAuxn18jRGue2CVZa5+tkZ7DTsE4hx8Zz3MfW\\\\\\\/KZDInL63PxSiu3Nqym8GW57Yy0u47i2x3+SnrV5zFjC8kxjgqTO3sPKGJYwV5TrqpTmkreyfIMi2e8MuVBj08ft1ge5IuAYg\\\\\\\/ShcFzJJBLPUIvjuVsBGY37eItIgP++WsGnW\\\\\\\/9aISZRP4hccDnh1+1zzN+Vqbkth6+GBkWf2OVqCa7OS6Nj8xfmf4FIenPKY9IiTgSEyVGbk8QQACGo4GxE6dsrDvIEEi80RIZeTcxYk0Rvrw5DIlIyp13OkM40gPcHplSw2MrOI2eVEILjLuJYtMVd\\\\\\\/bp0yF6DbUv9rxjcKpLG5zrIFQGK0kBZrZjm7iasSmKTVRdGvwxE+plSAYeGANVtaMbO\\\\\\\/rLQKw3u5SgjGo83y0k\\\\\\\/byjlDZ\\\\\\\/M5bHACJtO22ITxqHbXOx\\\\\\\/g73mFOjWGAW3TIXIFHLDrlGvuoJak7IW1mpG3z0fBmQLyiDpfYrFIrEb6oGTghOjacJSq0WNoqwRqNjeZBffAHf5bi59FYr1Ke0j+tS69zFLErUE1bNkTqWuOg514i7ROqQ8s4\\\\\\\/Bd8Uzhpphrd1K8wlQMI0p8022wrHMoqcrVA25xIErqFlelNbAHbvIntSEkwFwRGEPX+JoGngZC8pg9PBHh924qyom8udv+lwemuoZn5OUYHthhlFfNlvygK4xYTp1Gskj960v5MzbU9a3lG2+PmYVohWgVnk9UHVhwyQDJl1QiqCgp0C2FfJzzNIn8qRIKVBWyd\\\\\\\/GuApHKrNVtozf34VQXf3DSGoretAsIot39NVo88IBehBRrGBOwp56jI\\\\\\\/YE4pwU9sGBZt9wVMoYF1KOnz7Nzt0MNs6WmnFjdO0JuQ9B8qKbr6lf0lJyN1GacNE6AVT07hApue+xjJFSxo0OZkRqb1cBEmRiUF0igzeZO\\\\\\\/5ZNDw0liqvBuj6DiT1kzVNfRNVAoISgxodgqaWSeJfsD6mrVLg2IOpWBDFA8RIbN6vIhTvsy4X3wFz74ta3JyjNE36sB2pOCk\\\\\\\/3PevyPNQBCVepWOmRmlhL4tS+JTq0Yyzl5ycCm6ZEoySOVZDM4fvTBv6++J1bM4EYN0sL6cY2zfFxK\\\\\\\/Rp3ow3QLtvOEnsZdWjIINix0bMEH8G00iGlC4z6eUe5C+fQRb02bx2Lm6abaimLUaDZyhTk4T4yaP\\\\\\\/6vdDh8I7exOijp5dtoR4CpePjMnkju9zh9PHrb63rRlio5yid6hRDI1l01uuk2Pn4LJTFZB4F52Tk1i3by\\\\\\\/Zgw1tD8xU+NlLYfr1mM0yGGa7x+dMBMLM4q42oo9+kujLWHQwO8vMHRgyL2VnlgrGQG9Oo4dgcsZokk2+6KtqSgqSr7QtHb+MzE5RoAkh4l9uAOBDL784OLh2\\\\\\\/y\\\\\\\/y6cwZFdvDX8CtmY7svz9mk5RBsoEFsXcrtzT6GTAPMqyjJSCb63LILjdouNQjRsgWBZjnQAdQpbKamB5qc8rSFsqkZJAEdMOCYN0aEmE6GEScr4Y2Bwcuk41Ru00+vKqOzcZeFcG1XdUFhpYtQUmabbMIi1RSzcU0wgU49xZWgfqy8HMr7i8Y6hmWzgyx1DZzhbHIBXIurKrJr1+4cCA13WgZC3O83RViQsNTpPribqxDfmVugBBeSMGHQKgh\\\\\\\/nuMGbwy63p\\\\\\\/HNH4HDGhfGewOwSwCNb5YjIQq0jfkpTrUAZzPe22B1Dp5nkUozDAwU\\\\\\\/k6gWktnjYlcH46Lp1aWJhvF5oJQxCaqRScPxwHD5JSDN4v2uxkwgBq9dZijHYnWcWeqvkFA\\\\\\\/TpKaLvdPkPahndpYa+lCN6HIxqWxHoaG7H4ycikgn2VyUqUTthB\\\\\\\/JqaKDWE\\\\\\\/0vzSnW+XS6ca5Cc50nieMliYai0D+mWSXNmN\\\\\\\/itmIRycu\\\\\\\/n+LozMwglnxMaCbUUROnWG8H8dHm81f\\\\\\\/fgSFz3z9BSB64wojNulzL2kFKqy2tOT5kgKTPomotgOIu6bEG8vZFSn8KzpT8FkXG29ZX\\\\\\\/\\\\\\\/4+JIyqCnGBbwEie3OWBgpXvkZ3kwXKc8L4sP1rVgGsWH4IFihPwqwGBeDToIjMTnDAKLWrgbHmkIOM6\\\\\\\/B2LWNiO6v5crs43M9hrn39UfTfwvpfOfLTAHgpst8D\\\\\\\/\\\\\\\/s2dc9ActhbbFxVcU9tm4PmFyUmI1s+pE\\\\\\\/gGBJNTQT21rTvS50nlvNfE\\\\\\\/za286VX643Fjq2poyKeDQJWKc\\\\\\\/jerqRzpDL3eN4yyDe2Kbw\\\\\\\/xePj2zSZP5S9RKckZmYD\\\\\\\/K+HnHijd9\\\\\\\/G6DmNHLDcz76j1vpuQSymc8zwIYY7dHDxIoDygfKLM47JOCXmtP5OpNwpPNIHK9mFGIWO5SIV+rCTZ++ghYz3k84VDjHj6l\\\\\\\/0vANV\\\\\\\/h879pZp80XEGJ1q4ZLMF\\\\\\\/VyXIw6SNNEK85Z6ppi280BBmFmoGjeIpW5sQh7fiSxv3kBTJkAilS8JOKwuv\\\\\\\/o9eCuSNqLsRGZ8K9drDAZYo0PGr6aBSO+OyEnfhTDDdIZwslSqdhVZ0Tr8Bui1YgiU0vW2Ssg99GLvHNuwSjRgUqAAknqe5K9opFDP2W3L6doALVFvA45ksPbdyCEO9U5Dc\\\\\\\/sYYfPq4FS4JNzH9Zjb1oh5ut5HenRqlMF\\\\\\\/VYxfLSBcEirBjaToW+MkNWVm9lAlZ6GVuiYu0Vw8gvlzjEg+zhIdeCm4u7LI5s2ug\\\\\\\/iC5CNp+edJCsd2BnxGm+jWq8B+dXLVz2XUHqifMKMm29u7maSnY52cOK2Dwl6FV3O7o+LVy1R5fAYhNANTelS\\\\\\\/e5z9HjQkOxXAe2BWKMC6n\\\\\\\/oNFB5JJ2p57PgnAlXXZMvQNU7PCpK3Kyj2YxKBs1uAK2rX3I\\\\\\\/Uoidnwqp3ZG\\\\\\\/E9iGTd2riGzGrk4cpCAtDLPuoxUTOimUj58okEoXgAZlHRlYb32YWCPdtZkCeMZEk2HBPnAnqqFRF9tI0ZAdzWTIBRCPyNG8ehe42+qi6X6LbRyZs4s2yUOQT8Lz3gNMRtKnzlJoZLK9cjdR2JROxHnSITMLRi5VVmRs9Id9\\\\\\\/aaOr4fPwACD9nhIoSMfo7DiSduICvxB1Vc\\\\\\\/mulG0sBa+dHg6LDucyH7rRw5qv8oMMc0Wi+6AlPlArJ3+uypXkbEX8RBXZQPmngnIjhQmH1WmcKwrLAiD3nTEirE9qiryBlM+HJYAe\\\\\\\/EsJftAzlhm9i8o4fOnqffSLVf5guy+Uz6Ob0H0qDj1Wz4ZCeqfZWWAzTbF1lU\\\\\\\/wudAMK0rOVf4AdL9xpgN22T3Z2GIx+rxjksNkZp4xGVN4yl98rAD2se8onkncFufBwEEbAn01JFiFFbRZZz7rz0MkKqsTtgM2CQcO2UZKMDAS98aesnvaNeWS030wOrYJJOJ\\\\\\\/M2oJu2H8gnIVdIfCecrH+IIF+7KbCq4Xqy940fR6swO+KEO1SYEmYKg7Gpy+ke3nBjXEpLEi3\\\\\\\/P2lp92IPjUsunMVmacS20FaxJMtDJvayA9VJ\\\\\\\/lPsrUlXnXVMf5ciOJ1VPMJBm+kkcj7rJ9FcW4bIIRHLVYktuWQ0ay2Q\\\\\\\/mIZprgtwwi3BevpBN\\\\\\\/\\\\\\\/nmc5GlysC6F\\\\\\\/gVo87AlPy9R4Katl+1sPuCzJDWJDkl1\\\\\\\/OmaxAx\\\\\\\/6L+\\\\\\\/FC2tUuuIAY3EDxV+4JgEGpGtK0ML7zrWSRB2YlGI2kr7zf8ZMCMpLW2qhMG3NTSzWvSUMuBdfx\\\\\\\/\\\\\\\/Fi9T66stoLBkUJeVKGPTNqggepiBHv3FXpqRTLqKq+34sSNwqjMImGdBRzoaCxYgT7JXprOzilZoXTdozGTtqyvJUKuAxc3gI4m7AXzS\\\\\\\/dvF6\\\\\\\/bTdRxetwOdtIY1WVFM0p7VmXO2Up+ayIEMNqRWEN97YdYKObA75pfO5ZZ8wB2FmqSYmdt8JPv\\\\\\\/DYmgDm3BeQZDHuw\\\\\\\/3dw8IarKPCqSp9eAWIBwjrKkZ6v08Qrdv9UMq+bPpmf0+L9Whg8xFYIQsBhu7lQOz1EA5Z68Vv0HAZntONm1maOSEKe7vfDptwEFPTbFfsnwpIAZZxUUsGhiHlbwe2d9Wfy\\\\\\\/cQ2V4NaSk5+arSQWoNqY9r9tsdJx97JEP\\\\\\\/JO4zJtqJU+LQJAPKm98G8EBbD0rUxZIUbHvT6PhLpDJqrxgZv3KZhaLdICjJ9dgasxwBlkv4zk7NVALgP14aggCpISzW9vHaOQp7S2GtshQspAAE6MzTT7XicRwP0s2UY4nRpYCCePghWJZLqiM+xSFohUgOoUlx6ef1xa7QfvW\\\\\\\/hBmOrWUhGcCySzSBH1mW1HzvzrHv70pw6Kf3\\\\\\\/LtVgIoDvoNF06UrTw2qk1lkuLUNG+pX2Ndmg6RGh+UPqLOeCp9peOmJQKb2ZV5Q4aFRxx+IRAvlUobTWKbHgVGRBCo9N7erZ00jDp7Hgvu63FG\\\\\\\/JYZUkoWwfoYkkNS\\\\\\\/okFBX0zmbIKbajYe9bWolJj0dpdjfbMXAH9Hl48DkKkhqerRpzMv+wL+6qjrxh3DZhpAa\\\\\\\/4TB7GIBi9ue+R\\\\\\\/KrjaKa9Ca5K7yAAdPe46yHE8lBALXLVle3j3s91I8lwI8QWYi6dbKmq6jVp8WSzgpxH1blPpfHghrE0FPVolmNH8supDUdrww78lN+LQbaY30OSE\\\\\\\/\\\\\\\/luCj9EbeV4ABSehBc5OF5IM9LUQsSOEy8+2STsF\\\\\\\/0kuwvsn8WxlDjnoe6bPY4ZBSnZYIS2iCS3LyVTygx4uKSzfFbx0SWKlWQ1K6p6ir83W6qKyqMzyU5a6iia0awd7PQYKXDmXSimBQ77S7tRPnWWMsMFfw0EeZu5GqXqBrvg0w0MJu6or2TRlcK1wnEFrqmAs4BjoFGZ3+T886IkmGsCJQT2Dp9MlhC9rF95qDfQR3lu9UCK6EeFVZ437CDjg6Vrad3DgKa3BMrQpSnrHtJjGvlX29VC5bflpSepL8Yldm65ySDDI\\\\\\\/MhSJFdLAej2yeBBUfZ0SB80utO+pcfAvGeo8MZYE42w2wGdqGspcHDQl3L1pC9D\\\\\\\/2X6\\\\\\\/s3zMDWNzytd1LB8MbEjPbZiQ7+eS\\\\\\\/8O+kj5LZKqpfBF2CXSjiAhfo5ZwIs1yblR1SPh1L4+8GkTCLDpQLpwXrD2v91YHczetKQ54PNnAivfUozoyFSdDXAq5flJUSqCA+SorK+C\\\\\\\/Qt9mzyMRJXuVK7yW0+jDsWzQeBus0lWp5fLxXGnV5WQ8+pfxZ48m0GPoiuwsuQlUlyMyvbJseSxrnCEBjczKQ9WTzeJXLg3xGXgJOU3EPvu5S8zu5wS\\\\\\\/MlpDnXsiiiUk0ifvioBoa1uXG9swzudJ5eDiMx8LMjUIXLBhteVcEMtSqOjAYdZAwDVFKhW4XMiK15OcFprBRzVh5+1AgeH08Ka+D31XomtrLs299s2FECTllPrqv+XPshzHM7y8MBNCom7CfSFfWeIKVoMVeYVoa47D6m8xl4z3PRsaQi7h\\\\\\\/s7VderNXBx6QrP1WpbedzURu\\\\\\\/MnmFJ0WMf8OL8iI+hh2yp5uTUDlCCXVRb9O8Ay5CRJqJa8KUY2B5FrroLCaG06pE\\\\\\\/xcuxbgLBsi2shSzzpXYDxI67fIVqIZ3iZhldxzYL2r2ffx09nVZO1E773njME9gmDjEL4vSvVgl3VigYnNIROD4bm3PCRSZOguX2F+DabCyeyIXJrOtKiLSOvW7Hz3E1yzOg3DZFVBpDD8\\\\\\\/yQ+YK6jcfPrppChqBOVZSgy3O351gzwDjgUAAC6SU+SaAyBrAeOUX3mwM3vI3ERm498rzciYZYelaLDAwwhnCkyHcUB2nsy0j\\\\\\\/zkiqFXfGBhqGRbHuzKjPYiQbmsVRAihubKs6O7JahyL68vzsmCXB3+WOEJLpwjYReU+tBmJkmAarotcqhGOe\\\\\\\/TJrshNrIvwmm0wpKufGAaPWSy\\\\\\\/1LS5lSmybLywvwhRGxxOgJgff9w1YVs57F9pgN2CDHzNF1P7MeNqOIAngST5Nwktd5T3jhKQeoL531XmttmUKe0Z73q9qGEO18FPqeRZAqGkHmtPlgLndMnFJtZZebZLv\\\\\\\/AhTrOumhVpgz3WqlmosG0xn3Mue9BT7QKqE++pBRw7MdEZfXFu0cHLxsQ3TFcw6zXKFQHvslQJYTsbIn7w0yosVXDRrTo3urIFPKjjMZx35WjEs74re0juOGmgKSnefrrB75UB9mVE9Zq5GKjRQADvfrsNH5OgYExvXOttuG+7lbWnYGeLlQ439AQUTNaP7R+mrTXzMZQk\\\\\\\/n3PNcHrP4SbFaWsT\\\\\\\/uG3DZ8g+QzCDg0UH2AdeJcTA\\\\\\\/c+Brn80r8BKpmhNo7xM6yR+dx8DN2gZN\\\\\\\/kQGeLrtm4Chr4UdA+ajXyEfeDUG6ohEvbu0DNCNWZP8Y7wRQcBIB7zjpmR7CrhiL1M\\\\\\\/mEIT+0G2z2Mi8DIej\\\\\\\/IlPIB8761lHilzuxBZ8KMTJtQCV0wPHc3GNPRuWCpIHluz79HMvZFkU6tJYyBP4z8zuBXe98jQJXYizZVqldZ\\\\\\\/yX34Et3qC7NmQiVJUKdz7H\\\\\\\/oVDMqCvgxa4mzviaPPoUj3d8uB\\\\\\\/eTV2E11Fo7OeBEBt5Mm1\\\\\\\/Sp0+CHNVsHOEfisiZxBEyGqIxivgvVcQepW+7stOi7kTSudlEQnb9GdJH+4QQZa2k2bMEIzcorjlclTwTzs+l+qQu9MwKlOP4tGtIvk83SCxlosrBWs06o5FTaZQDQTJ0l5AlAOCppdbCqqQj09pr4nJcpWtPc6lhqPbnwpvxOxObgV5A5Ltr0\\\\\\\/uRcFYRa0wR7EUqW79vfamF2iz9VexjwLCFLrHLnlRh0KLjpM9JSRDqWiGMHXgqGZSmVO+SoE6f6QXzO7EyqmMTy3qli6pIg+T23Nk\\\\\\\/N1T5kIu0VlVaaP4PCujOD86Hc6\\\\\\\/sUfV\\\\\\\/6mUcLEgThBM0boSvj7qBkXd9849+Ot7cQD4nk3V5DnpWdbQXzLap0rqbqxLn3DxrqAcj3+n9JT2SVUdXhK5zRytsDCZLeA8goxZcupW\\\\\\\/R3U2xQZu7s7iZxH\\\\\\\/4+I214bVComXEeHlO1RXzq6FtaPUJn4SYyn4NTVaw7TITjutkb4rvlx+hAke4pO\\\\\\\/eWv+EmeWk\\\\\\\/ad\\\\\\\/Y5v4tfe6lHArIaN2ppl6T8g8sUDUjCSKQ\\\\\\\/o4EAtvaFM8LtpZvCNaRNMtexYXudmTLHlHvPq9e\\\\\\\/6DT6ibmD8GiTx8A889DpG+EbvemH9cjsTdSuStCM2Qg0Fu8qtA7iTne+PZRT5Kim1JMtWEb31bekMw\\\\\\\/I7wPtAYD9Ni4HM4\\\\\\\/haJIrGnmrH1zy04T+kjF0TgiCNq99d\\\\\\\/2mhODFqJ4I\\\\\\\/CZfeQ8wmNPOeId9MWUOqEg14rd2YY364C6\\\\\\\/spSabR2ZOD36j9uFvyg\\\\\\\/z8ErZtuqzxcnVsKbgRM\\\\\\\/HcJDHWdVMtAaaz77V9eYWBrhKMKCNN3Wmivre8AT5cNy2bqS3jP42\\\\\\\/wBIGF3oYsBKJdiAwL7tFlpzfCwEF\\\\\\\/grgzh5x5rkK915o+zN68RoKF7Wj0AmB0J22FyabBwzfE4rHKnyl2bzaH5O+oSJYnKN+mLaS4\\\\\\\/G5HWgsfqvJ9jzO8fCTaAtyFrlbcehBrEBNAOwq\\\\\\\/tpKttbIsbMfnCCYK8IrSBCSpozQUwbsrqePjungatr8zFP+f61Obg39EFq\\\\\\\/RcPQuTE0wUnQIfZRSuCAwKXjVhc7QjmjyqQFjWDAs3wwA2B0UsSly7GjGgsEvGL8byYQzXL1PINN9ZWjd7DhgZnFxebtpZF5vOzq0DWLB9cGnD2vObYBhV\\\\\\\/UPW65BZki894o8GDJxXXUZGeroXXqpVDRsZvzhbhEUK5eDTE7MHEn+YlBIJFRuSzsi+A2u0eA2pLn9EVJV9vEyd6\\\\\\\/B3YB23rrCWURVgsCXCuFmukwB9924nzmEMBHgRme3aAYnMkKsEtySZWyaNVPBX76HHJusJqXwyeBYQvW8Ug5axgd7vvQUcSgdui10F1zz\\\\\\\/wwgaDB5LTkjEjZ+MUeBOkDTBO2F0nT2Z9DsTaR2u5l8mAPL7Zz7ShvIOeYkgz\\\\\\\/S\\\\\\\/zos3paMmxS0uupaOsEDa63Xo52nJsjfcWl5XzL0M433RG+mWF6pyzJoi20Ym\\\\\\\/rJvK+za36uwjwWBH\\\\\\\/C1+odK152Nmg1tIh6\\\\\\\/EFbGaC1MWFajkyIMstJFqfz73gNbcprA+lEBYsRmkC+aTpKSBAqw6JdnbuQQ2OksHcesWZj3GSUCMUjlL7GB5oQrRT2DIfHMZjNSSfMWBd+5VJg32AxEPWMT5bNDaWx9vUGrn3wCFPFoEqWP2LZzVC3jiL5HQGtoBrz22LEfq\\\\\\\/qNvWgTEiudulKxOAA9gDOW\\\\\\\/oy2aJ5EnO01vtHcbSaSSM1IP4THm6i8ubiUW599UoAOvWK\\\\\\\/tLtepWyiTW3BiBHRAMXOzSmtHXewHtNoTtOzIQLATN5RGR0L0eFRqu79WBpogR5oIPLbIII1a3xpJXZuKovAkawiE1KKKu9RgfZEMnIIxtpMnj7z8w0JHr9+oI8H\\\\\\\/3x3SMV5A342byB0l5s4svrW96uEMf4EMTEnLRL1zMD5swHur5KEKdWeWorCBQ7RWPyVMZWasVUf28UQ5v8gVO4AuADev1IN5Oi+MfBYoAMjbl90yTk5c7DZat94QT4a0BFJRPclvgCYOVtEUv9wyY\\\\\\\/Dh24VykIHc\\\\\\\/wUnXdMl1U9WtjWNEZSPVWxmlojFkRkDOipQ67GYZzOC5lJ9Kdwt0dTAP6qwh2UkhBhWxqxS89ZRq1EAZTohHmZ359g+ocXztvsDPAARoleHmjpQnGb9jfLXqzF1xglpL+FUN0mIgb3jnPQYD7fBu6cqUxZSgqyB4\\\\\\\/ECQl+SmTDE3IFAnU3\\\\\\\/1uIpJy1S+4xTNnpdXnHer2QGzLTiMHaFXFSkzwtHHYp1deDjiGn1\\\\\\\/uTbBsvBZkN2zSLjcw3f7cpT3813EGiBQZCo+IaD3lEC9Ot5QXf8lSUOOv8+pzTu94sUQOEfnKdh4WWtgWJoKGbmJzK7zob5rUn5RwbJ+t9l2OLsPSC0YPZ4DwOnjhlQLAS0hZYydZu+A84uTNuv5usVqcRVXMFoIsbEnIYombXY1nZq2M5HsFXWBtfOKYxZ62rkNllc\\\\\\\/yAyo4228ED+MBQXc0MOc6uqAFDP7Lib1D+CSWflOEWmhdMm9jHEvov3yHZxobtDrWrSqQnoL\\\\\\\/+3rltYO+qX9ZHeCHrMoqU8Jo12Dhrib8vF+5HaM9MZ9i5EMqNaLOeEjXYk6Cfyb9d6+kJodLNQ5kViXKt5+Q7xQEbGB1z\\\\\\\/SPP3uwDUNcrs++LCZZpZZ8mrmembJGdqnnxoVj8MQpn1iaoMpPD3ENS1MNYMdpcQFiprJNvG\\\\\\\/bsDilGhRH88FumNDy3EPt40ASPlXK9MZm4OjF2XJkzMCvk8ND2BQ4vGF651RGL0qErzKL\\\\\\\/XipkSNU9SJIED732TrgtUCL\\\\\\\/S1RzH4Ci\\\\\\\/cgSsFfz8jywS3KbhJuH\\\\\\\/oycnC0LIzl\\\\\\\/Vht\\\\\\\/g6\\\\\\\/Enwg8hoZ35782LRDDvSuG8W5G56HHU8uQ\\\\\\\/msJ6JR80pB\\\\\\\/vQXfL0ktpU8UT3sXCWJM5ldXRHU7IJ3\\\\\\\/2QQfkdTm6fQE3+wuef+P\\\\\\\/PzVsBQd1CfjQpE3DtEppGTPPNFLBKrQ0kwVZTzoDcWa5G2vj53UOsaxvnJNmvHdOn8GIv3wIu7ggOH3\\\\\\\/mhCCAMzBJQcQz0HLa5lnRntc9SUh8yvof04P2cjRBP03oAwgz6WUcky6R\\\\\\\/DaxoiWmrwqB89bKvq0uKvX5fAel8BIHbG7pVsB2Iq0+op7Tos27rITLCSuB890qPMhsBYvALAWO9WlLXb1clKeC+qXLpRrcUoKi3FrYwNycmZhcoGdvM63NDIdEKCVHnCdo8XVUozk2HQipJPefZV4zHzwEyQOOuk\\\\\\\/Y4+7TRoToSznxo55UdmYegxfZpQjdApdeAfYRswnWleieEBa1+b8H6PTzMYpHkAvs40YJIYrOKDueJ9pB11aGcKGIoyFoYr6WaSHrSF7o3osTRZ3IHgUbkUJuTY9LzGZbp1wXg8cWAtqg6kTwpoOERC6C89nv8o9v2SLx+JISqSkW61nCa362keUeOJt90+rbgm9Dp\\\\\\\/Ws8tAjX9fI7SjwlsLU\\\\\\\/RyONyw2QJOx\\\\\\\/WRFv5fR5TgsgT+xdu3QfJlGeKZLnjFN2lcxQgH4n20iszADZorlyl4knO7gTAVloOj3zzsTbglvNa6LtXwPwxNWCsF2b5n8W94Efm+fSIvFrScDSEIzLH+kJzup4Num6VR5HpRZ3VZ5IzDO65myGnC1eUe0r5CFDM6WNoyRAELJvx3\\\\\\\/Rj9a2oRp+BGarhBjYwJT5Tz2jmXZgX7kUogKrLebwtuhA2Xs71tlsPrZ6MsRl0yHIwhzyVobK4dKhtctn2THFB2FndY8ofLUbsixQ6TPFK2+z+BJ7FA3DyNI6SgVyKEXORRPnonHUgr6ouAvtSotAEZPwMvLFYnIpwyMJ6QIyMP5k2mHabszpmQ5KS72NTO+V\\\\\\\/x\\\\\\\/jIPAoyO+uxOvfwqC7jflMP+VFQTgYRpfvmpdP4V+At9vcgXDSjJ7GqUdtD8Qv2eo8qo4E60bdaBNAo5CY0KhQNhAK1Xz62yNvqeUuwHnQX1glCgykla\\\\\\\/ts5f623Kb+6l2HxJ6SFVoFYMYnWaL0WIzwPkgneeQOKWHL6wBL9N4whh0M8U0UuEjL9oYqki6txUVYRev4R0droL+3+zLB6hIm+MiyT0G2+SNTHd6FarUPIMtJpHITVoPQbYdLLuYSoy6lf1uIH5l0V9Al3gMo+UL5c2GCjtpZcSv4TsK3QQvU18Vq5VwlueB1qxrXVpXHBXR3yovGVuYI+\\\\\\\/t9IHAszDOf0LvStdC8cQOAb4ya+oTceRKjnAx2K47kXTP07pxuZY3vJFDy41oMoWi84Owjj31l2rXoqUqV55i2TPpkq576hqEPz07Uo5exyyp3wewmVmEOuarXRgo31UXx74WIRr4HfupvetV0x5tI1d8YRSSkQY\\\\\\\/IS5fkPzM+s0HBeg7fsXQgbBI2uJJTYIHjtTGHwJHsxccddJBtXMHDKW7\\\\\\\/eLOMk0Nfw1spd8e3D0TZN5Qmsi\\\\\\\/wwQZRObZl8IxADLjINwxfBv7fyaMjXrGldrurO3cx50dRku4ywsqu3fd7aeKLnuWzfWa1n4Ftq6UciQUAdAMrbtWY2t2zTrdd9oZkaH2K2l\\\\\\\/Yhh3wMAlRNjOc0KJtqNwymAgIPYttFmsowHVFbbb\\\\\\\/z41txvDKmMuGHx94Pu2AfO3d29g6OPmUJ\\\\\\\/J2ho3pDDUOmzaZ2j7K92jG3qySHOvW5xBWOA31YBVdi7nj8NFCpzfwXFE1b+TcTUbSzeTOQns+3ggcjg+c193sZzCnNfohLjOde1SnDznhAES9h7VN9zlJL8n+6caaVAscbzMTrHYNki8H3cKxpTEukOQZjLoI258iNT7w2CJCpE81IMdUeacK2W8YX9WCw6oSMerFHQGfMArH\\\\\\\/fi5XLvEExMhAe4H3CMTSK8zeJCLHZ1qaeuq3uQVzyv2Z+yKXaLpj46RWt9cMuTCS4CMUsajQPBB2HX85d2idJm5\\\\\\\/pRQo8nVO6AyLocxhNjnXEocQu0bPwzQdN8+njGALOKveZZujoihn0MhwduhmzhTu\\\\\\\/gNl0rC0M3wjg9PIccqOikyvQPQhzP4dVosRrZYt2DfjmxzL5Kc7vGVxrqFC56ew7yxvsPtVv1x99sw\\\\\\\/9PgyfTGRhv5ojtqqmNC5Y5zod\\\\\\\/ApTMJF3AJ7LXxCbLhEd82TUcgCa9dRJ2zp98RAUWPRhiwaz2uoQ3se8CtDq27kkEzUVO7Tnh222H2q0urOYnJpGByIMGpsaoCcVy\\\\\\\/GexzmC\\\\\\\/d4\\\\\\\/NDJPsdv1Uj9Xjgt2cMwe7dcPlViBiPS569nYRn+DSq7T3TIhUi\\\\\\\/LEWy2NhE5+5mOMtGZFF9HH3KFAJqfCQCn1V9qNOk2rdrkALpiubppBUSSMDST184+8nsaYhCNLCbThZrdoSSrOAlW2jgFPMiydgkiBE\\\\\\\/OQH0oGxeFGwEcZvylp3yB0uuHM8zHLD6ZcIXDWXQrGj7MvWcg1zpIYNIODI+o3D+ichNvct7dZPGTbNtoAWkr00BavreCyF0nV+3VqrgTaMj5S2jaZZpjYMwUY92IghnBulewosX5C1nhKLu4YZwZHjcW8W7B26DCgjhZC0hBh5pJ9iez+AtsR1ykHaKakNEbGYNMGXE\\\\\\\/vEVaJ4jYQejSBciC9BoJPRDewv2sgUlBA5piFyI9kEj8CtcdIq9PLoSuyW9sm\\\\\\\/ec4k4KgT70tca\\\\\\\/ubLsLLa3DYBO0djZtwJBpzyDEYdZKEj0OBs+z+7RAAFtR\\\\\\\/\\\\\\\/92XeIFihopb4Ij7BMri6+wXXzYk3izgkFHaP8imm1G7N4z14B3oD7Un+MsMx\\\\\\\/8xdv34P\\\\\\\/4u4x82dDkhQNyGPY1Z1TRlRHACQA6H6UGtR\\\\\\\/ydrgtxcxUX2w3zC52EARc3+gBijVZly7zkxN2+l+VgCiM8qEbgZeDA8y48l\\\\\\\/CPYXq6nf1oGMGdU7u7VdNyAqnKrZyAcldDM+30xIEhg5EGn0N6Kj6\\\\\\\/3i\\\\\\\/uV3y6cVipC4faJ27lBAGC5VYJz0d+EfxMHXF9SDkaYs16+SMFPro7\\\\\\\/+\\\\\\\/IFHgHQ3bUF2QhnsOpBoxD6HRKJJW7wHEIJ\\\\\\\/5CHYuJJRL6Lerp\\\\\\\/Zwf1g4jg7jvpJb3z\\\\\\\/yavYvLvAcobqB4NeDYlvwf6CMNN6ikZRSP8iz1DSv2VjwOHM4SVy3XkkBvUWu9ZGSJGBB4xzc2DhJldmY6yZyy7X7nSPwWIPR2jSQCkH7J+6zQR7ixmSuRt18PurpOPNpNxUjpxZpog0QIzfS4jx48RKaVRw5xZeDwQKEYNZDCKZHYPeOjFF462aNtH9a5Akjtg7L5v00UcpP8R9z49ej2FgudJHojWIb6SSX1NZeuVjnsSnPJXfXeIwgLR1PTF8NQhNu6LgYQsiTYq8jCCUch0y9BbG0REm0QJz3vod7FEDNR0K5Ef4otI+tabBY4HWGWnZjnfm4mccd7ZnWOD3aYSbFHHZGuryvMHUY1yJi6rxCkREPBHZJQ9zrI2yhC9xTa31FHg2iprAoBJ8UxwctxDb27uAJdqi5TlsHEKIDrojjSVyC2CjyChQfvIJikoSaMrwH2KijzlhW7BjwrO14HGJQiDtTwHao6WnLYDaL5F\\\\\\\/qI7Nlbgm2FveJ6Tf3dDPeHFmVFWH30lOEUqfpv90zb1gXfr9Czh\\\\\\\/JZRrsP43vr+BcfI6t+0kPwb7UciI8Fno\\\\\\\/xXvqRThZjQUa7K5Y3hlGceIrS+wX\\\\\\\/OOvkjU3kRrX5juZMwncZgXnZuwPBn9QSkr60F84l+uRu6nCVYHQ0JKG850gJkkSgznBynXsecLapoYMxlIQnkCEJC2kISb3LGlXShK7ypd1zioJMCdyBJDuRQzEVPnnhTiW1ErcbtoDxo2lZA5GyDAA0YTicMWeOtOz9n3g\\\\\\\/HuEgPmIGOMVYZ8G7L7MrEWAha5QTUTnDJAehMHsR\\\\\\\/DRmnlIS8B0aGnYyGcsDf4hu9HowEIz5mD\\\\\\\/sP1Vr0I5fUKzcBq2JGowBmV9LHRVvDwp\\\\\\\/sW22EOthlfrqBgEjItMK9BlCwfrhHsVXxvliP4HeZwbg\\\\\\\/gwXLeijWFNu99StFYGkhgAkao8trIFpEb8hAxmGK8E7y+7QJtecATzpQrVxo0hE9GtMwRYfvotLdlXujdMKsE6iyyZJ\\\\\\\/6\\\\\\\/t7M56ePAntIjuXENUyVxregDDjkJ+zvrK6Czb8GtfRdbHBqJAY2XbXWDUXsu89Ur0QrJAfN8Wr7+1fm1cnS2j4OmgKFjUvgyk5hnl66hAglb7ym13BQCfCPjgVTp\\\\\\\/aAwvAfgTV5zPkEJpcLmQfQWPutibyO1SYKneOeOdTvlT7uohZf8ggxfurXhG9nbb7uiQiCpxvIGih2O\\\\\\\/EXbaFZMyaxE\\\\\\\/kHU8xflsouqlJj\\\\\\\/B9iCXpTzG0yNdYNrnJD\\\\\\\/ySnSKnq\\\\\\\/lh+bIamEmRi5ptW7XRGRjF+rEfrgU89qkZfaua5vmhpLH+Y2MUWwKkHQMyUjD5eyXl8mXHeDLUUZjDgSmfnX5gM7nH0zYf9ybjZdc8tC7\\\\\\\/Yi\\\\\\\/B3lC48LhcDkxy4luPDU4cL1RNEUv6ZFZ14QUw7G9zP5RdO\\\\\\\/VMqWAhWYQJIcE4fHAcwKy\\\\\\\/RHkGm71nu42mAEhM\\\\\\\/uq1yZ5t74CzCEmJqtXQ6AaOqshKVhSgbN3htiCo\\\\\\\/S1sxJJPhopfN4+RI6pj6q+y0F+KkjtP1WRk+dnUHpuZlIbt5ZZl5KHFbQKTl5itSzq7rdv2HHfX\\\\\\\/v7DxfAdtSNA6M+GCH+7F5gOmjFFu2c5IWZvcultw+ADCq8K9Hk+iKjNxfX41FzgW8eOSy\\\\\\\/nHo1w6VGeftIZo9ZbNyJSZwtcFkVdLhKkMyF5UGXnKJAD\\\\\\\/+rWBgDZd5WGQQPvevUVf3Vya7kek340+TueQBB1sRQ+AHIE2ezMxGivHcsUezoAdlto1wsiQG5ozvvLHV9lOd8zOZt6TTJVoK9By\\\\\\\/Mrp0JhMu36oSNx84FAfyKpfI4H2b\\\\\\\/FOewgJW+U+p1cOtCXHpzN44T\\\\\\\/GceF6uvDkQ+1euL996oxZeli+GCOH0VOO1bPaNkPjquvWGDyuDEulalVzrNuYGMPWVdJHpLnSViXeYpouaQ0BaqjgpDYCLkbbDYym+jwEKsQmptLxUgtL0JdDwSWkWSU7l7SMmktfwjIM6l11OuHYHmwFh2kUzae8\\\\\\\/c+5kSTddrdIJ8ExiOENLgra+pjoGsAtKab+YocGeEr0ZycciLCGubgVigcbEwd6JWLZbtnVHcav5SJ3uwowwdAAc3QgzZ4EpABRVFqzWa04A6v12d9ySUe794hnGGq3FfYp\\\\\\\/vBxFA\\\\\\\/e5nxL8\\\\\\\/aeIeLZKlLqfi5RMHY9c2asWw2MjnaisziFZpFoXZpEAsyIVhlLaTKUrO9dqBMQ7q5BO\\\\\\\/EiFytxTbnvfKgxR0pHOwjBy5FmeEtGYcaz8mVlmRcfQU9beyFKFA7buK9cpWt9WrrZzPMPBACXHeneTBQ3E1mhXKjKjFLxSSQpsFdFZI2EUpdXgpOfni9ZRwwH1SxPoIrkO00lWAKnSF9OoWFu5NbVBJBg8k5kFW5JVgvAuPWLU4FWEFFPHC\\\\\\\/eI3XNAglwXbvZXgjh4j4ekND7epWJ2OVWJSTqnvhXy9zZdj7PAQWgL3\\\\\\\/dBnvgblfznR3jjytDec9WxlUVleqbrEx80RK9Ku6d0xP0sYIPaVsOhQGF9ldYpT7xPa8NClBjv7UNYDO8Iz1T0Dl7sTR51soW5F4FWsNcJFB75IxKbtDwYD6VdFVJQkMCHtlivHTwxhuTDR2FuIX7W4hu6ZDjWhoJyWVk0WNIjJBJ7luX3+wgsvaDkyphbl23pme7gwYFql3vLEfBF5jbqZcAgrbRT9HlbhC+T0Tlt9XJ5vf7BZrLrIMXsTHWQWLqVoOWmpDj8gtoluYPcBItvulag9HDnT1f8MHqVU4gzJZjByfzNPWkAcmlCa4yzK1TBNGNujtXHLSjGUikvq+XtttkPjfVnuWGr+OrEIXdqPMlpdYVgRuz\\\\\\\/Dnkl559ug3K8JBHxuh3vKFoIwna7fgyHXxCPZZFC9EVznM1n5sUHPXkvEmna5KWabMsYdcISO0NH8C5YVMkvj9bUMi8IwA3kM54hlRd8uSa56rYvjIEP+KUpjQ66OZyacpSq9tCNz82efTIHa99M5vhQXStUJ81cHWhChUyid9FtNcZ\\\\\\\/M7M7zJT1c85J+s0UUnhoEme7TBedqk8\\\\\\\/vpa6S8xdNmI7UhhOkju9C5ttyYmP2MXU+Pf5UZbkiQb73hYuigGZceomZYR7WYy3HJgdRCGmqi4fAleL3lWgPrxo831fHkQBN82G7uUSRKm4kJ98RdJKGjW5N0a3Wkizv0JeZDK3EGxAwEDkfDJM9M5Vr6QKfuYe9iedx6zORg1O4S\\\\\\\/JwS2rkIgQCuYarW4jdXVZ0VjAMe1WmKib+K9cyrnF8mNqDHWvUD1Rz0V7A+G3ba6S7uUV\\\\\\\/DR3NYSr8Oos1tFTitYNp4kFQOjb7iOkDrIjCGqhEOhLUr9TAwnL\\\\\\\/r0XK56B4PheeqKdBl+ar532U6JZCuX64uYG9RG66gWizE0RiJ2EjRa4ouMYJ9FB9Eem2+iC0QxFwMR9zc199yNKa6k1M6BdCN65yrNokRktC0+uJ4iAb67XcBzuOGmwr88BNubpzYN3o8ef6\\\\\\\/fcYTOinHUtO3b+v6lRdvqo6oVxVpXdvO64W+S9f2JPY+Jk03YISbbKBXFS4+jwCLkj7XwMS21JqyiybWrQEZiA8urUZyMRedqf8S7IdEzqUOCI2YcgZOtCA8snc4OIb\\\\\\\/jtJoSP8UlPx6r8LOFZ95K5Fq6DyhIeBCPldJ3RaY6sPnjB\\\\\\\/f9aHUymOD6W9Nm+oytDUf9AU\\\\\\\/Q3yk3B0cVhUPikSZX+QkyAgmyYTysI7lOFIunUYVcukfTVQppjez73AKPaCgf1SnU4IFKSvWHd0g4S66IJZm+yfH3PxTSj5daqkQfXEV88Sjixb\\\\\\\/pR4W5D3BJ\\\\\\\/eDTNMF3pyUB\\\\\\\/FBqPON91R\\\\\\\/TtSZcisjp058o9wZbQY8vp792FNa9B9vqVXsPyhBlhHHKBRCldHFfdPt0WK8wSNGbCO9hPo\\\\\\\/S9RbOYMKs2p7ONqQ2Mi5RtASFRLhVYJWRLVitZYZqgX8dxNevjJPxqiyKe1864ZtI5A6sLH6raCVzkNCw9LbY9yXZvTH8RGPILok39SiHX5DWsfY6x6NkEI6AgpGYkhSjMVF4Sz2gYAhsTmdN1eO0GU23y7zePiuKv3NDUmvdqBLhAbejgSlXJSpGg65hTykTOMHpJuAhI8cd63oxOGBaPd5bKnjrFZulQ2UXH0aw9dJNtwN9fGNGyIQOFsjHqq48vcAxj9ONcEDKfK5Bn3DFziJQgO9D7FTqEoDGbHC\\\\\\\/dncaQTygqzsIOeDEa3SFRyHN679QQ4pfH67Ld97fF0DIKnsu2GqssU3EAeCLJ\\\\\\\/CGXnjWhgoq3UsrwG8Z9NC\\\\\\\/hdMQANtEawRDaVsHC613Wx+O6gVud0GC5zD+rMuFk5DwrWUcbIDblr8u2krquQKiGfRdnduGpUvzAXybq6TqlPdC2\\\\\\\/4Vz5E9LMEx\\\\\\\/zCWQlWfjlpLlDg7bLqVvpZrEcnUeaezHinpPAWdNmnmOTvi\\\\\\\/CzpMWGW6OzQpza3WJnM71mlsEJ0P+umjV\\\\\\\/ILLtJZ\\\\\\\/XfrC4YbhFRMSzdYWzhjeZzwmA0FRk\\\\\\\/H9njOjB\\\\\\\/bJVavVbWWEpuVmhezS1BwBIfUpTRCKergU4jgRpGLLgxxqSJ8pIuXbwi58RcILmMUD7bCAve7LvosRIptCHESsibsD\\\\\\\/7+gNe6rVGqIg+aExfA+1z7P3oDV4xn4Wt6MULe5h46D\\\\\\\/I7aCeffDq1EI+11\\\\\\\/\\\\\\\/sh4CoaagJF2ozAqOMQsDdep8sI5AX41vvi56ZTUbLWOCBewA7L1cEddsA0NjTBGJi6YtSBcxuN26ntjwpqnDFYdoEfnFmj+7fjwBbK5lUuOBOBtTbeINuPMKZ\\\\\\\/fq1K+hA3aFgCMbQb\\\\\\\/9dcCi4G1mQVwm4L23CDmEbjIo4cQl9lb3\\\\\\\/\\\\\\\/FRykbf8nb4wI6UXk73eLyZpuYUdjseeDvW8bghwHTJcklgZdXLqtMyW0EFHzvUbQ\\\\\\\/0dP\\\\\\\/ImTk+PCmCkWF43CrhZnzjYVcleVYqz1U\\\\\\\/Ef24qlL+gg0x93srXlYOZN425pS0QvmptTpRA2OBuNrhLDF5fBemePRDNO+649jbyw7jAdzfZcV4kckpQTFjuzrLiZCIrvthXo\\\u0022,\\\u0022iv\\\u0022:\\\u0022zD\\\\\\\/eQ7TR3jjKpra7SwcHMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002252a7209ad92eeb0b74a1d95106fc7277a9647c22011cc8ac053d740e2a93e7bb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mlSmE2hlQtM2IGhiylVnljgZYid1U+T3BdGCAz6rNOlcnJ+Rav9YtgJXaZsPzKgHBfRNcHidWYfOoKPbk1bxfUbgeBZIFy2gM8ThKOonPX364hHdKJMpjaNHydppEVyyqzEQRYXpXavOCBOG2tdtGDTdXnz3htgcrzc7v2Ebja4nb0sYzpoZD\\\\\\\/xiy\\\\\\\/EmXnswt8nLq3qs0Pbthh\\\\\\\/8q4EjVrMdGkRjVD73FS9JE4J0YFCqkMngMKQOiI305cBG0tetKCVs7+EpP\\\\\\\/o\\\\\\\/oHNLz\\\\\\\/\\\\\\\/0Ecead9MMO02xLDDimKJKuMz2Af4uQn38FF8qXC5mbS84D76ut1DdNAoMD7hNM5Fq+9vZUhAcDejE4eBxfQ+\\\\\\\/J5uYl7YkYdjbzj18PL+Xhr7H5Enoc\\\\\\\/ITxx6mcqGQaM+\\\\\\\/yCI4LRjlm0\\\\\\\/u3djtqdtEkeToC5xS7emPdmzPt2zeqMRhHtxNW75AJmPpSlLH0o\\\\\\\/M64xC2y+4ixf2f0SnwVtGqWkgYPLJYiC6SXlTLRtc467e7kyqmEI1SKuAbHhKKSFFI7+6xmHujKTB25bvs2+dKTAV7ZoZRNK2FkBZgrwjmM2pyoZjgmG9pYNsOLevVTnRy7WqKNY1BN57KAaMNM7kGB67QoYHtLauuPH4PpI767tfrRwMfUP6luL\\\\\\\/jXm22JnF\\\\\\\/IXiseBy7CtrUoQvIWepiRH6wHR9c7ysfZPQbKuJTsnX6bGfDkcIL25JDi1asNcXycK4ncBFErXiUxJ\\\\\\\/OrVpVGytBczX8xDdo5LWjfgvsTfgoO09SmSYolNZBPv4lDMISabc7wxziTwUvLTfj1kgNdEjqXVCY+phn5q3VJCfq4xnKDOrBc0D+WPY+eRWHtoNm815fJMPnGxcj5iPOJsUthAK0Xayi0ol9LlBR8rLe9QL2ohUjSBXJM0RXa+fRRyK558p5yVYbVQPTS5QjH2iZxYnM0VsiT4tJCfz2gcBdjrAdOa3clMQwYgIe7IVyfR\\\\\\\/9OikDsOMqCApxEvMMItspJqtVENYUwY+TAdSsLWoshC8YsW11y2Y07wGA0Qau5UmsjXm1eizkyC\\\\\\\/A4Ej\\\\\\\/o6vurphgglk40WNMITzvr6GMvUC5PUy\\\\\\\/eEq+xIBA7gJOcGkFaDUb7icSu2TxP7yrASWoV2Q8SB4wQeoriWA1c1StQvWlHH6WKDUQciyHGid49VbtGlYNZtlnx79gTLFbfLnevbCoWk+ePp2YqEBy4zzHR4u4hxQ+Lo+8mwfRu2UJIWObICiY9rMqxGbFSXqBzwceK0vcQnoCYGTAxUSae3CGkNb6mibI74b0F0fNhDOSu89VKZo9SYomBaIn8SWrHwQzxaR6HIB1QVjt1oGgD\\\\\\\/6Hkrqcb0vNb7kkIAecee23sVP8fRT\\\\\\\/9fj9OnohQ1nstqOAGO6abE4\\\\\\\/aP7Wpa86BGU1fj9rmuJqHzTTbI3Vo9\\\\\\\/CZIzauLncfLHKQRJvLSjtuOofcAiu7rT\\\\\\\/D58PteKLxqcVgdwmAbbGPRdhTpmO7UJggWO5QgKQpJP0vnsRr\\\\\\\/VetwBj8vwAuD8v1RuKlLXtZegAPqImTg2Zvyj1R+7cg5azTYcWQjohUG9U3ILb+CrDm7ocehSA8m1SiMSBpnE7Yh04HirxTa9SYLjvtrerpHTMkM4J1Wer5EQFH9+V3dHW\\\\\\\/bOXniwIfAQicY5uE2APGnqp0Oes1UfLbeVyfcmRmSWwDS\\\\\\\/Us0Odvf4gm5HToOAee8+PlwaE\\\\\\\/Tvjg0E7ItuXV0b6emWW7\\\\\\\/7UqPl3r\\\\\\\/fb\\\\\\\/Y\\\\\\\/sDf6oVMgPk5phgWHm9\\\\\\\/MD32v766SWlruSBSW8CxT5JkKX\\\\\\\/tVTkgwvm0HIpU0t4L8zOqX4vTFt7FYntSw0O8kRA\\\\\\\/S93Jws1WuW3mNeHjyhGdY0Dsudtkpvx7vjF7Z93Zcr0ltlrY7jxxHtXWvS\\\\\\\/yLvmwpvHJ8MiXtOfcfFYVl\\\\\\\/xgd\\\\\\\/xydGP6s9D+4idatuheVMQeQ1oQ3H9T0mB+6Ia+QzSqXA7vUqUUz5uvCDR08z3m2CETYAXXfYl7LQO18UfwE3I0F30dB8tswFV4cbS\\\\\\\/Lh0mwif4=\\\u0022,\\\u0022iv\\\u0022:\\\u00226in2PDytH\\\\\\\/gq4hReVrZAxw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b5095f591b892a6dccc1974603be5adaf41d09cd5efe98ee6842f7ed35581b20\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459412 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/205" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fkOTOeOvN78LstWFHEd-P5PW3TSXeY0eO-g-JPkp-0Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/205" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uRMuLbSY2bjW6n6lWDozZL6DHbG8tG3r5uK34OxVBpI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/197" title="Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/217" title="Genome-Wide Copy Number Variant Analysis in Inbred Chickens Lines With Different Susceptibility to Marek’s Disease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459412" id="node459412" data-pisa="ggg;3/2/205" data-pisa-master="ggg;3/2/205" data-apath="/ggg/3/2/205.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher A. Desjardins</span>, <span class="highwire-citation-author" data-delta="1">Jürgen Gadau</span>, <span class="highwire-citation-author" data-delta="2">Jacqueline A. Lopez</span>, <span class="highwire-citation-author" data-delta="3">Oliver Niehuis</span>, <span class="highwire-citation-author" data-delta="4">Amanda R. Avery</span>, <span class="highwire-citation-author" data-delta="5">David W. Loehlin</span>, <span class="highwire-citation-author" data-delta="6">Stephen Richards</span>, <span class="highwire-citation-author" data-delta="7">John K. Colbourne</span> and <span class="highwire-citation-author" data-delta="8">John H. Werren</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">205-215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004739 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459412" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher A. Desjardins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Broad Institute, Cambridge, Massachusetts 02141</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BDesjardins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desjardins%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BA.%2BDesjardins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cdesjard@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">cdesjard@broadinstitute.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jürgen Gadau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Arizona State University, Tempe, Arizona 85287</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%BCrgen%2BGadau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gadau%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25BCrgen%2BGadau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacqueline A. Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BA.%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BA.%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Oliver Niehuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Arizona State University, Tempe, Arizona 85287</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Molecular Biodiversity Research, Zoological Research Museum Alexander Koenig, 53113 Bonn, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BNiehuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niehuis%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOliver%2BNiehuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Amanda R. Avery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BAvery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avery%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BR.%2BAvery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David W. Loehlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BLoehlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loehlin%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BW.%2BLoehlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">John K. Colbourne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BColbourne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Colbourne%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BColbourne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">John H. Werren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BWerren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Werren%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BWerren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/2/205" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459412" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/205.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459412" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/205.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459412" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/205.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459412" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/2/205.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459412" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">Nasonia, a genus of four closely related parasitoid insect species, is a model system for genetic research. Their haplodiploid genetics (haploid males and diploid females) and interfertile species are advantageous for the genetic analysis of complex traits and the genetic basis of species differences. A fine-scale genomic map is an important tool for advancing genetic studies in this system. We developed and used a hybrid genotyping microarray to generate a high-resolution genetic map that covers 79% of the sequenced genome of <em>Nasonia vitripennis</em>. The microarray is based on differential hybridization of species-specific oligos between <em>N. vitripennis</em> and <em>Nasonia giraulti</em> at more than 20,000 markers spanning the Nasonia genome. The map places 729 scaffolds onto the five linkage groups of Nasonia, including locating many smaller scaffolds that would be difficult to map by other means. The microarray was used to characterize 26 segmental introgression lines containing chromosomal regions from one species in the genetic background of another. These segmental introgression lines have been used for rapid screening and mapping of quantitative trait loci involved in species differences. Finally, the microarray is extended to bulk-segregant analysis and genotyping of other Nasonia species combinations. These resources should further expand the usefulness of Nasonia for studies of the genetic basis and architecture of complex traits and speciation.</p></div><p id="p-7">The parasitic wasp genus Nasonia (Insecta: Hymenoptera) is a model system in genetics, particularly evolutionary genetics, the genetics of complex traits, developmental genetics, and host−parasite interactions (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beukeboom and Desplan 2003</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Werren and Loehlin 2009</a>). Nasonia consists of four closely related species, <em>Nasonia vitripennis</em>, <em>Nasonia giraulti</em>, <em>Nasonia longicornis</em>, and the recently discovered <em>Nasonia oneida</em>, which are all cross-fertile once cured of their endosymbiotic Wolbachia (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Breeuwer and Werren 1995</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Raychoudhury <em>et al.</em> 2010</a>). The presence of interfertile species is advantageous for evolutionary genetic research because it allows movement of genetic regions between species for the identification, mapping, and cloning of quantitative trait loci (QTL) involved in species differences. Several additional features make Nasonia an excellent genetic model, including short (2-wk) generation time, ease of laboratory rearing, systemic RNA interference, and haplodiploid sex determination (females are diploid, whereas males are haploid and derived from unfertilized eggs, facilitating mutation screening) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lynch and Desplan 2006</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Werren and Loehlin 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Werren <em>et al.</em> 2009</a>). The <em>N. vitripennis−N. giraulti</em> species pair has been used widely to identify QTL involved in a diverse array of phenotypes, including wing size (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gadau <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Loehlin <em>et al.</em> 2010a</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">b</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Loehlin and Werren 2012</a>), cuticular hydrocarbons (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Niehuis <em>et al.</em> 2011</a>), hybrid incompatibilities (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Breeuwer and Werren 1995</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Gadau <em>et al.</em> 1999</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gibson <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Koevoets <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Niehuis <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>), and host preference (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Desjardins <em>et al.</em> 2010</a>).</p><p id="p-8">The recent sequencing of the genomes of three Nasonia species (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>) provides a key resource for advancing this new model system. Using the genome sequence data in conjunction with haploid males from hybrid crosses between <em>N. vitripennis</em> and <em>N. giraulti</em>, a genetic map of the Nasonia genome was generated (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> 2010</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>). This approach was facilitated by the high level of single-nucleotide polymorphisms (SNPs) between species (average coding sequence difference between <em>N. vitripennis</em> and <em>N. giraulti</em> is 1%). The mapping population consisted of haploid male embryos from F<sub>1</sub> hybrid females. The advantage of hybrid haploid males is that for any locus, a hybrid male carries only the allele for one of the parental species. With the use of this strategy, 265 scaffolds (covering 64% of the assembled genome) were mapped onto the five chromosomes of Nasonia (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> 2010</a>). The combination of a genome sequence and genetic map has allowed investigators studying Nasonia to do forward genetics efficiently, <em>i.e.</em>, quickly proceed from detection of QTL to positional cloning of QTL and identification of the genetic architecture that underlie phenotypes of interest (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Loehlin <em>et al.</em> 2010b</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Loehlin and Werren 2012</a>).</p><p id="p-9">To further advance Nasonia as a genetic system, a high-resolution genetic map is needed that places more scaffolds onto the linkage map and has a finer scale of resolution. In addition, the availability of a cost-effective tool for high-throughput genotyping and bulk-segregant analysis could help advance studies geared toward investigating the genetic basis of complex adaptive phenotypes, genetic incompatibilities, and species differences. Here, we developed a comparative hybridization genotyping (CGH) microarray that uses clusters of SNPs and insertion-deletions (indels) in a high-density genotyping microarray to differentiate <em>N. vitripennis</em> from <em>N. giraulti</em> sequence at a large number of markers spanning the Nasonia genome.</p><p id="p-10">Additional resources being developed for Nasonia include a set of segmental introgression lines (SILs), which contain specific genomic regions from one species (typically <em>N. giraulti</em>) in the genetic background of another (typically <em>N. vitripennis</em>) (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Werren and Loehlin 2009</a>). These are produced by performing an interspecific cross followed by repeated backcrossing of hybrid females to males of one species. To “hold onto” the introgressed genetic region during the backcrossing process, either visible mutant markers, phenotypic species-differences (such as wing size), or molecular markers are used. Eventually (usually greater than eight generations depending on the size of the introgressed region) the introgression lines are made homozygous for the “foreign” target region within the other species’ genetic background. SILs already have been used for efficient mapping of genes affecting phenotypic differences between Nasonia species (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Loehlin <em>et al.</em> 2010b</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Loehlin and Werren 2012</a>). To further develop Nasonia SIL resources for QTL mapping, here we used the microarray to genotype 26 SILs.</p><p id="p-11">We also tested the applicability of the microarray to bulk segregant analysis, another useful tool for efficiently identifying genomic regions associated with phenotypes of interest (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Michelmore <em>et al.</em> 1991</a>). In bulk segregant analysis, a population is divided into two subsamples based on phenotype, and then allele proportions are estimated for each subsample. It is an alternative to genotyping a large population at the level of individuals, and is an economical way to quickly and efficiently identify genomic regions associated with traits of interest. For example, this approach was used in combination with genotyping to identify regions associated with nuclear-cytoplasmic incompatibility in hybrids of <em>N. vitripennis</em> and <em>N. giraulti</em> (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>). Here we broadly applied the methodology to mapping phenotypes in Nasonia by using the CGH microarray.</p><p id="p-12">The CGH array will be of even greater utility if it can be applied to hybrid crosses involving other Nasonia species, such as <em>N. longicornis</em> and <em>N. oneida</em>, as a number of research groups are currently investigating these species (L. Beukeboom, personal communication; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Koevoets 2012</a>). Although the array was not designed for this purpose, the close phylogenetic relationship of both <em>N. longicornis</em> and <em>N. oneida</em> to <em>N. giraulti</em> suggest that their DNA would be more likely to hybridize to the <em>N. giraulti</em> oligo than the <em>N. vitripennis</em> oligo (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Raychoudhury <em>et al.</em> 2010</a>). We therefore tested how many markers on the microarray can accurately distinguish between <em>N. vitripennis−N. longicornis</em> and <em>N. vitripennis−N. oneida</em> sequences.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods and Materials</h2><div id="sec-2" class="subsection"><h3>Strains used</h3><p id="p-13">Crosses to generate the F<sub>2</sub> mapping population were between the <em>N. vitripennis</em> (ASymCX) and <em>N. giraulti</em> (RV2Xu) strains used in the genome sequencing project (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>). SILs were generated by sequential backcrossing for variable numbers of generations (usually more than eight) and then typically made homozygous. Some lines were subsequently subjected to additional backcrosses to further reduce the size of the introgressed segment. Some SILs have been described in previous studies, whereas others are first reported here and were generated in the Werren laboratory (University of Rochester) as a resource for QTL mapping and cloning (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Typical nomenclature refers to the marker (visible or molecular) used in the introgression with subscript for the species of the allele, followed by capital letters for the genetic background it was introgressed into (<em>e.g.</em>, rdh5<sub>g</sub>V for the <em>rdh5</em> eye color marker wild-type allele from <em>N. giraulti</em> in a <em>N. vitripennis</em> genetic background), with V representing <em>N. vitripennis</em> and G representing <em>N. giraulti</em>. The introgression lines will contain the marker used plus introgressed flanking DNA and in some cases unlinked regions that also were retained during the introgression.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462310/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462310/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptions of SILs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Microarray design</h3><p id="p-15">To identify polymorphisms (SNPs and small indels of 1–50 bp), we used <em>N. giraulti</em> genomic sequence (1X Sanger) aligned to the <em>N. vitripennis</em> genome sequence (<a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>). Based on these alignments, we identified 80-bp segments with clusters of fixed differences between the two genomes. Sequence segments were filtered to identify those with a SNP at the 3′ end of the sequence, at least five polymorphisms in the 3′ most 20 bp, and at most 15 polymorphisms in the entire 80 bp length. A preponderance of SNPs at the 3′ region was preferred because the oligos are attached to the array substrate at the 5′ end, and solution-end polymorphisms have been shown to be more important to differential hybridization than polymorphisms near the substrate end (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Hughes <em>et al.</em> 2001</a>). Individual bases within a single insertion or deletion were each counted as polymorphisms (<em>e.g.</em>, a three-base insertion was counted as three polymorphisms). To increase confidence in polymorphism calls, at least 2× coverage of the <em>N. giraulti</em> sequence was required along the entire 80-bp segment. All SNPs were required to have PHRED quality values ≥50, and no intraspecific polymorphism was allowed. Finally, the 70 bp downstream of the selected sequence was allowed to contain a maximum of four SNPs to ensure proper alignment of the entire region. Downstream sequences with no SNPs were retained as potential control sequences for the array. In addition to polymorphism clusters, we identified moderately sized indels (30–50 bp) in the alignments and selected 80 bp of total sequence for each species including the insertion when present and flanking sequence centered on the middle of the indel.</p><p id="p-16">From the resulting sequence files, we selected 70-bp subsequences to be potential oligos by using Oligopicker 2.3.2 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wang and Seed 2003</a>). Oligo sequences were then trimmed from the 5′ end so that they would require no more than 180 synthesis cycles according to the Roche NimbleGen protocol. We then removed all oligos with multiple matches to the genome or any number of matches to Nasonia mitochondrial DNA as determined by WU-BLAST (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lopez <em>et al.</em> 2003</a>) (e = 1 × 10<sup>−5</sup>, hspsepsmax = 500). From the remaining set, only pairs with both an <em>N. vitripennis</em> and <em>N. giraulti</em> oligo were retained.</p><p id="p-17">The microarray was manufactured by Roche NimbleGen’s custom design service and consists of all probes on each of four subarrays, which can altogether genotype up to eight samples simultaneously using two-color labeling and hybridization. The same microarray was reused three times by stripping the labeled targets after each hybridization and scanning. Data for custom assembly of the CGH microarray and SNP calling of oligos from the array are available at request from the authors.</p></div><div id="sec-4" class="subsection"><h3>DNA preparation</h3><p id="p-18">For mapping, F<sub>2</sub> hybrid haploid male embryos were used to minimize mapping artifacts due to postembryonic mortality of certain species-hybrid genotypes (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Breeuwer and Werren 1995</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Niehuis <em>et al.</em> 2008</a>). The method to isolate and amplify DNA from embryos has been described in detail elsewhere (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Niehuis <em>et al.</em> 2008</a>, <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">2010</a>). For microarray hybridization of DNA from individual adult Nasonia, including SILs, DNA was extracted using the QIAGEN Puregene Gentra protocol for single Drosophila (<a href="http://www.qiagen.com/literature">http://www.qiagen.com/literature</a>). For microarray hybridization of DNA for bulk sample studies, Nasonia were pooled into a single tube and DNA was then extracted using the QIAGEN Puregene Gentra protocol for 1–30 Drosophila, with two Nasonia equaling one Drosophila. All resulting DNA from both individual and bulk samples was subsequently amplified using the GenomiPhi DNA amplification kit (Amersham Biosciences, Piscataway, NJ). DNA was quantified using a Qubit Fluorometer (Life Technologies).</p></div><div id="sec-5" class="subsection"><h3>DNA labeling and hybridization</h3><p id="p-19">DNA was labeled and hybridized following Roche NimbleGen’s User’s Guide: CGH Analysis v.3.0 methods with minor modifications. At least 1.0 µg of dsDNA was fragmented to 1000–2000 bp in length (as determined by gel electrophoresis) by either hydrodynamic shearing (HydroShear, GeneMachines) or sonication (Sonicator 4000, Misonix). Genomic DNA with lengths primarily between 2000 and 10,000 bp based on gel electrophoresis analysis was subject to hydrodynamic shearing (which offers greater fragmentation sensitivity and control), whereas DNA with lengths &gt;10,000 bp was fragmented by means of sonication. Fragmented DNA was labeled by random primer labeling. First, 1.0 µg of fragmented DNA was primed with 1 O.D. Cy-labeled random monomer primer (Trilink Biotechnology) at 98° for 10 min and subsequently chilled at 0°–2° for 15 min. Next, 100 U Klenow Fragment (3′ &gt; 5′ exo-) and 10 mM dNTP mix (to final concentration of 1 mM dNTPs) was added to the primed DNA. The mixture was incubated at 37° in a thermocycler (heated-lid) for 2 hr and then stopped with 0.05 M EDTA (final concentration). Cy-labeled DNA was purified by isopropanol precipitation in the presence of sodium chloride (0.5 M NaCl final concentration). Dual-color hybridization, posthybridization washing, and scanning were performed according to manufacturer’s instructions (Roche NimbleGen’s User’s Guide: CGH Analysis v.3.0). Images were acquired using a GenePix Professional 4200A scanner with the GenePix 6.0 software. The data from these arrays were saved and exported using the software NimbleScan 2.4 (Roche NimbleGen Inc.).</p></div><div id="sec-6" class="subsection"><h3>Calculation of differential hybridization and genotype calling</h3><p id="p-20">To calculate a normalized difference in hybridization of a DNA sample to paired species-specific oligos, a modified version of M and A values from two-color cDNA microarray analyses was used. We calculated this normalized difference in hybridization (“hybridization value”) as (intensity to <em>N. vitripennis</em> oligo minus the intensity to <em>N. giraulti</em> oligo) divided by (intensity to <em>N. vitripennis</em> oligo plus the intensity to <em>N. giraulti</em> oligo), resulting in numbers ranging from −1 to 1. Positive numbers indicate better hybridization to the <em>N. vitripennis</em> oligo than to the <em>N. giraulti</em> oligo, whereas negative numbers indicate the opposite. We then tested the array by hybridizing DNA from seven <em>N. vitripennis</em> (strain ASymCX) and seven <em>N. giraulti</em> (strain RV2Xu) single whole adult females. Both strains were used for the Nasonia genome sequencing project (<a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>). We then calculated mean, median, and SD of the hybridization values for each studied marker (the genomic region represented by an oligo pair) as a quality control measure.</p><p id="p-21">To genotype a marker of a test individual, we calculated the cumulative probability that the marker belonged to each genotype, <em>N. vitripennis</em> and <em>N. giraulti</em>, based on the pure species hybridization value, assuming the distribution of hybridization values for a single marker within a species is normally distributed. Markers were then called as <em>N. vitripennis</em> (V), <em>N. giraulti</em> (G), or ambiguous (U) based on three parameters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/004739SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/FileS1.pdf">File S1</a>). We optimized all parameters relative to precision (the probability that a called genotype is correct) and recall (the probability that an unambiguous genotype is called for a given locus, also referred to as sensitivity), resulting in 99.5% precision at a level of 90.8% recall (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS6.xlsx">Table S6</a>).</p><p id="p-22">In an attempt to further increase precision, we developed a smoothing algorithm based on genotypes of neighboring markers. If the two closest neighbors of a marker both have at least one opposite genotype call and have no similar genotype calls, that marker is converted to ambiguous. If a marker has only one neighbor, and the two markers are called opposite genotypes, both are considered ambiguous. If a marker has no neighbors (<em>e.g.</em>, scaffolds with only one marker), no correction can be made. Optimization of parameters using this algorithm resulted in an increased precision of 99.9% and recall of 97.7% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS6.xlsx">Table S6</a>).</p><p id="p-23">To determine whether the array could successfully distinguish other species pairs, DNA from four <em>N. longicornis</em> and five <em>N. oneida</em> females was individually extracted and hybridized to the array as described previously. Hybridization value statistics were then recalculated for each marker for each comparison, and precision and recall was calculated using the <em>N.vitripennis</em>−<em>N. longicornis</em> SIL (SIL17_ws1_lV; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) in the same manner as the initial parameter optimization.</p></div><div id="sec-7" class="subsection"><h3>Mapping analysis</h3><p id="p-24">To estimate recombination frequencies and generate a linkage map, we genotyped the same 110 F<sub>2</sub> hybrid male embryos previously used to generate a linkage map with fewer markers (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> 2010</a>), plus an additional 27 newly generated F<sub>2</sub> hybrid male embryos. All embryos were derived from virgin F<sub>1</sub> hybrid females generated by an initial hybrid cross between <em>N. vitripennis</em> (ASymCX) females and <em>N. giraulti</em> (RV2Xu) males. The two parental strains also were used for genome sequencing and should therefore provide the best sequence fit to the oligos used on the array (<a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>). F<sub>2</sub> hybrid male embryos were extracted from the fly host as described previously (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Niehuis <em>et al.</em> 2008</a>).</p><p id="p-25">As the construction of a linkage map requires extremely accurate genotype profiles, we opted to use a set of 19,681 markers and parameters with a precision of 99.98%. To avoid an inflation of recombination estimates due to genotyping errors not correctable by smoothing, we excluded all scaffolds represented by only one marker. We genotyped each hybrid for all remaining markers, and then collapsed these loci into a set of marker clusters with unique genotype profiles. Markers that could not be unambiguously placed into a single cluster were subsequently excluded from map construction. This resulted in 498 clusters composed of 15,546 markers. Using the clusters, we generated a preliminary linkage map using the program MultiPoint (<a href="http://www.multiqtl.com">www.multiqtl.com</a>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mester <em>et al.</em> 2003</a>). Clusters used for linkage mapping showed no significant segregation distortion, and none of the genotype profiles used for creating the map was missing a genotype. The initial two-point mapping was performed with the minimum recombination frequency setting of 0.18, which resulted in five linkage groups containing all clusters. The initial marker cluster order within a linkage group was verified using a jackknifing procedure (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Mester <em>et al.</em> 2003</a>).</p><p id="p-26">We further refined the linkage map by removing genotypes that required double recombinants between physically close marker clusters by using the “graphical genotypes” feature in Multipoint, as the probability of a double recombinant over a short physical distance in an individual is extremely low or impossible due to genetic interference (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Muller 1916</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Sturtevant 1915</a>). Once these genotypes were removed, Multipoint was rerun to produce an improved linkage map. In a last step to improve the marker cluster order of the linkage map, we examined the linkage map for potential mis-assembled contigs and scaffolds in the <em>N. vitripennis</em> assembly version 1.0. In cases where a potential assembly error was only supported by a single genotyped sample and by two or fewer consecutive genotypes, we corrected the linkage map to agree with the assembly. In all other cases, the inferred linkage map was considered correct, indicating that the corresponding contig or scaffold is very likely mis-assembled. Finally, we used recombination events in SILs to subdivide clusters and orient some scaffolds within those clusters.</p><p id="p-27">Once the linkage map was complete, we estimated the positions of 4136 previously excluded markers on the linkage map. These included markers on scaffolds represented by a single marker, and markers with genotyping profiles that could not be unambiguously assigned to single cluster. We identified all marker clusters that had the best matches to genotype profiles of the excluded markers, allowing a maximum of three mismatches.</p></div><div id="sec-8" class="subsection"><h3>Bulk segregant analysis</h3><p id="p-28">We tested the applicability of bulk segregant analysis using the microarray on a sample composed of individuals with known genotypes. Using two linked <em>N. vitripennis</em> mutations, <em>st-318</em> (scarlet eyes) and <em>mm</em> (“mickeymouse,” bulging eyes), we backcrossed wild-type <em>N. giraulti</em> (<em>st-318<sup>+g</sup></em>,<em>mm<sup>+g</sup></em>) in a <em>st-318</em>, <em>mm</em> mutant <em>N. vitripennis</em> background for eight generations. The resulting heterozygous females were set as virgins to produce haploid male offspring segregating for the mutant phenotypes. This created a population of individuals whose <em>N. giraulti</em> allele frequency across the region should be correlated with the distance from the visible markers. From these offspring, 24 recombinant males were selected from each recombinant class (<em>st-318<sup>+g</sup></em>,<em>mm</em> and <em>st-318</em>,<em>mm<sup>+g</sup></em>).</p><p id="p-29">We separated the heads and bodies of each individual, and the bodies of each of the two recombinant classes were pooled. DNA was extracted from each pool and subsequently amplified using a GenomiPhi amplification kit (described previously). The two pools were then independently hybridized to the array and genotype frequencies of the samples were calculated based on the relative hybridization to alternate oligos. Assuming the proportion of alleles in a pooled sample of <em>N. vitripennis</em>−<em>N. giraulti</em> hybrids follows a uniform distribution between the median hybridization values of pure <em>N. vitripennis</em> and <em>N. giraulti</em> samples, the proportion of the DNA hybridizing to the allele from each respective species can be estimated for every marker. To test the accuracy of the bulk segregation results, DNA was extracted from individual heads and genotyped at four markers known to be linked to the mutations using polymerase chain reaction (PCR), as described in <a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> (2010)</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>A high-density genotyping microarray</h3><p id="p-30">The designed microarray includes polymorphism-cluster oligo pairs representing 19,708 markers on a total of 933 scaffolds, which cover 255 of the 295 Mb (86%) of the assembled genome. This includes 100% of scaffolds greater than 400 Kb (139/139) and 87% of scaffolds greater than 100 Kb (404/467). Of these 19,708 markers, 1696 also have a control oligo from sequence located immediately downstream of the marker. Additionally, the array includes large-indel oligo pairs representing 843 markers within those same scaffolds, for a total of 20,551 markers. Both scaffold coverage and marker density are significant improvements over the previous single-SNP-based microarray, which covered only 265 scaffolds (64%) of the genome and had 1,255 single-SNP-based probes.</p><p id="p-31">The distribution of markers between coding and non-coding DNA is shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The majority of markers (88%) fall entirely within either non-coding DNA or introns, although 1096 markers fall entirely within exons. Nevertheless, most annotated genes in <em>N. vitripennis</em> are in close proximity to a mapped marker; of the 10,734 genes in the <em>N. vitripennis</em> RefSeq gene set in Genbank, 94% are on a scaffold represented on the microarray, and those genes are on average 7.4 kb from the nearest marker. This means that mapped markers (and associated SNPs) can be used for microdissection of QTL to regions containing one or a few genes.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462376/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462376/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution of oligo pairs between coding and non-coding DNA</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Analysis of hybridization values for markers from control DNA of <em>N. vitripennis</em> and <em>N. giraulti</em> showed largely nonoverlapping distributions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>)—DNA from <em>N. vitripennis</em> hybridized more strongly to the <em>N. vitripennis</em>-specific oligo 97.9% of the time, whereas DNA of <em>N. giraulti</em> hybridized more strongly to the <em>N. giraulti</em>-specific oligo 98.4% of the time. Furthermore, 19,681 of the 20,551 markers had median hybridization values for each species separated by at least 1 SD of each distribution, and 16,511 markers were separated by at least 2 SD. These largely nonoverlapping distributions of <em>N. vitripennis</em> and <em>N. giraulti</em> control DNA predict that we can call genotypes of unknown samples for most markers with high confidence.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/205/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency histograms of hybridization values when hybridized with N. vitripennis (Nv; gray) or N. giraulti (Ng; white) control DNA. The Y-axis shows the number of markers which produced the relevant hybridization values of the 20,551 total markers. Hybridization values &gt; 0 indicate greater hybridization to the N. vitripennis oligo, whereas values &lt; 0 indicate greater hybridization to the N. giraulti oligo. Only a small fraction of markers produce greater hybridization to the alternate species oligo when hybridized with control DNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-473653616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency histograms of hybridization values when hybridized with &lt;em&gt;N. vitripennis&lt;/em&gt; (Nv; gray) or &lt;em&gt;N. giraulti&lt;/em&gt; (Ng; white) control DNA. The Y-axis shows the number of markers which produced the relevant hybridization values of the 20,551 total markers. Hybridization values &gt; 0 indicate greater hybridization to the &lt;em&gt;N. vitripennis&lt;/em&gt; oligo, whereas values &lt; 0 indicate greater hybridization to the &lt;em&gt;N. giraulti&lt;/em&gt; oligo. Only a small fraction of markers produce greater hybridization to the alternate species oligo when hybridized with control DNA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/205/F1.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/205/F1.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/205/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/205/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-33" class="first-child">Frequency histograms of hybridization values when hybridized with <em>N. vitripennis</em> (Nv; gray) or <em>N. giraulti</em> (Ng; white) control DNA. The Y-axis shows the number of markers which produced the relevant hybridization values of the 20,551 total markers. Hybridization values &gt; 0 indicate greater hybridization to the <em>N. vitripennis</em> oligo, whereas values &lt; 0 indicate greater hybridization to the <em>N. giraulti</em> oligo. Only a small fraction of markers produce greater hybridization to the alternate species oligo when hybridized with control DNA.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>A new genetic map for Nasonia</h3><p id="p-34">The genetic map constructed here contains 510 marker clusters mapped to five linkage groups (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), representing the five previously identified chromosomes of Nasonia (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Rutten <em>et al.</em> 2004</a>). Cluster- and marker-level details of the map are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS2.xlsx">Table S2</a>, respectively. With respect to the <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> (2010)</a> map, the current map presented here expands the number of mapped scaffolds, increases the percentage of assembled genome that is mapped, and decreases average distance between clusters (see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> for comparisons between the new and previous map). The 510 marker clusters in the new map almost doubles the 264 marker clusters of the <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> (2010)</a> genetic map. Despite the increase in mapped scaffolds and marker clusters, the recombination size of the new map totals 435.9 cM, similar to the 446.9 cM of the <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> (2010)</a> map. This finding suggests that we are converging on the recombination map size for Nasonia based on the hybrid mapping population.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/205/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic map of Nasonia. Marker cluster IDs are listed above the line to the right of each cluster, and corresponding map distances (in cM) are listed below the line. Recombination rates for each cluster are given in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-473653616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic map of Nasonia. Marker cluster IDs are listed above the line to the right of each cluster, and corresponding map distances (in cM) are listed below the line. Recombination rates for each cluster are given in Table S1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/205/F2.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/205/F2.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/205/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/205/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462411" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-35" class="first-child">Genetic map of Nasonia. Marker cluster IDs are listed above the line to the right of each cluster, and corresponding map distances (in cM) are listed below the line. Recombination rates for each cluster are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS1.xlsx">Table S1</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462383/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462383/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison of new genetic map (2012) with the previous map (<a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> 2010</a>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Markers used to construct the genetic map represent 476 scaffolds (211 more than in the Niehuis <em>et al.</em> map), covering 74% of the assembled genome. Of these 476 scaffolds, 53 are oriented, while the others are placed within a cluster but not oriented because of absence of a recombination event within the scaffold that would allow orientation. We found evidence for 34 scaffolds being mis-assembled in the Nasonia genome assembly version 1.0 (<a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">Werren <em>et al.</em> 2010</a>), and due to the high density of markers on the microarray, we are able to resolve breakpoints in scaffold assemblies to an average size of 52 Kb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS3.xlsx">Table S3</a>).</p><p id="p-38">Apparent in the genetic map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS1.xlsx">Table S1</a>) is that typically one cluster per chromosome contains a large number of scaffolds and that many of these scaffolds are relatively short. These clusters likely correspond to regions containing the centromere and flanking heterochromatic DNA. The many small scaffolds can result from intervening repetitive DNA (<em>e.g.</em>, transposons or tandem repeats), typical of low recombination regions. Such features fail to assemble well in automated genome assemblies. Based on the criterion of a large number of scaffolds mapping to a single cluster, the clusters corresponding to the centromeric regions (and number of scaffolds within each one) for the five chromosomes are 1.056 (116), 2.046 (91), 3.045 (81), 4.065 (64), and 5.044 (43).</p><p id="p-39">Because the total genetic length of the map did not change substantially between the new and previous maps (435.9 cM and 446.9 cM, respectively), the genome-wide recombination rate estimate of 1.4–1.5 cM/Mb made previously (<a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Niehuis <em>et al.</em> 2010</a>) is still accurate. Local recombination rates (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS1.xlsx">Table S1</a>) were negatively correlated with both the number of scaffolds per cluster (Spearman’s rank order coefficient, r = −0.37, <em>P</em> &lt; 0.001) and the number of markers per cluster (Spearman’s rank order coefficient, r = −0.58, <em>P</em> &lt; 0.001). This finding likely reflects the accumulation of repetitive DNA in regions of low recombination as noted previously. Recombination rates on chromosome arms remained similar between the new and the previous map, which is expected as little sequence was added to chromosome arms in the new map. However, local recombination rates of centromeric clusters decreased dramatically, ranging from a 1.5-fold decrease at the chromosome 3 centromere to a 2.4-fold decrease at the chromosome 5 centromere due to the mapping of many smaller previously unmapped scaffolds to the centromeric region. Overall, the local recombination rates at the centromeres on the new map ranged from 0.044 to 0.173 cM/Mb, or 8.1- to 34.1-fold lower than the average recombination rate. Given that most of the remaining sequence that has still not been placed in the new map is repetitive and likely concentrated in the centromeres, the true recombination rate for centromeric regions is probably even lower than these new estimates.</p><p id="p-40">To increase accuracy of the inferred map, certain markers were excluded during map reconstruction (see <em>Materials and Methods</em>). However, we were able to map many of these markers <em>a posteriori</em> to clusters on the linkage map. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS4.xlsx">Table S4</a> shows the results for markers on scaffolds that are not represented on the genetic map, whereas <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/TableS5.xlsx">Table S5</a> shows the corresponding information for markers on scaffolds represented on the genetic map. Of the 4135 unmapped markers, the genotype profile of 3702 matched that of at least one marker cluster within our mismatch threshold. This allows up to map an additional 253 scaffolds representing 4% of the assembled genome, bringing the complete map coverage to 729 scaffolds covering 79% of the assembled genome. Many of the previously unplaced scaffolds added here are short (median length 20.2 kb) and fall within low recombination regions.</p></div><div id="sec-12" class="subsection"><h3>SILs for QTL mapping</h3><p id="p-41">To further develop resources for QTL mapping in Nasonia, we genotyped 26 SILs (summaries of the regions covered by SILs are described in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and depicted graphically in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). In total, the SILs cover 101 Mb (46%) of the mapped genome sequences. These SILs have proven useful in mapping QTL involved in species differences, and for mapping visible mutants generated in <em>N. vitripennis</em> by introgression of the wild-type <em>N. giraulti</em> allele into <em>N. vitripennis</em>. Examples of mapped QTL and visible mutants are shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>. These SILs have been used as starting points for cloning the <em>widerwing</em> locus [<em>wdw</em>, which affects male wing size differences between <em>N. vitripennis</em> and <em>N. giraulti</em> and has been shown to be the growth regulator <em>unpaired-like</em> (<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Loehlin and Werren 2012</a>)], cloning the male wing locus <em>wing-size-1</em> [<em>ws1</em>, shown to be the sex determining gene <em>doublesex</em> (<a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Loehlin <em>et al.</em> 2010b</a>)], in implicating <em>cinnabar</em> as the eye color mutant <em>st-318</em> (see the section <em>Bulk segregant analysis of a red eye mutant</em>), and in mapping the <em>R</em>-locus [(<a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Saul 1990</a>) a major nonrecombining locus containing genes affecting eye color and other phenotypes], to a region flanking the centromere on chromosome 5. These SILs and their extensive genotypic characterization can be used as an important resource for mapping additional QTL and identifying the underlying genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/205/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotype profiles of 26 SILs along the five chromosomes of Nasonia. Each cluster in the linkage map is depicted by a horizontal tick in black (N. giraulti), dark gray (ambiguous), or light gray (N. vitripennis). Clusters were assigned to a species when at least 60% of markers at that cluster were genotyped as that species; in cases in which neither species had at least 60% of markers, the cluster was labeled as ambiguous. Names and background information on the different SILs (a through z) are provided in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-473653616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotype profiles of 26 SILs along the five chromosomes of Nasonia. Each cluster in the linkage map is depicted by a horizontal tick in black (&lt;em&gt;N. giraulti&lt;/em&gt;), dark gray (ambiguous), or light gray (&lt;em&gt;N. vitripennis&lt;/em&gt;). Clusters were assigned to a species when at least 60% of markers at that cluster were genotyped as that species; in cases in which neither species had at least 60% of markers, the cluster was labeled as ambiguous. Names and background information on the different SILs (a through z) are provided in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/205/F3.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/205/F3.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/205/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/205/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462423" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-42" class="first-child">Genotype profiles of 26 SILs along the five chromosomes of Nasonia. Each cluster in the linkage map is depicted by a horizontal tick in black (<em>N. giraulti</em>), dark gray (ambiguous), or light gray (<em>N. vitripennis</em>). Clusters were assigned to a species when at least 60% of markers at that cluster were genotyped as that species; in cases in which neither species had at least 60% of markers, the cluster was labeled as ambiguous. Names and background information on the different SILs (a through z) are provided in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Some of these SILs are particularly helpful in resolving scaffold positions in regions of low recombination. Although recombination events in these regions are rare in the F<sub>2</sub> mapping population, some SILs have been specifically generated in centromeric and flanking regions (<em>e.g.</em>, <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Desjardins <em>et al.</em> 2010</a>; <a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">Loehlin <em>et al.</em> 2010b</a>). Data from the SILs were used to more finely map scaffolds that had previously been assigned to a single cluster due to absence of F<sub>2</sub> hybrids that are recombinant for these scaffolds (see <em>Materials and Methods</em>). From this analysis, we identified 12 additional clusters and were able to refine positions of 82 scaffolds between these clusters, thereby increasing linkage map resolution and accuracy. In three instances, we were also able to orient scaffolds because they crossed the recombination junction of the newly created clusters. Because recombination distances are estimated from the frequency of recombinant F<sub>2</sub> hybrids, these clusters show no recombination between them in the linkage map.</p></div><div id="sec-13" class="subsection"><h3>Bulk segregant analysis of a red eye mutant</h3><p id="p-44">Bulk segregant analysis is a method in which individuals in a population with segregating phenotypes are sorted into pools and then genotyped for associated molecular markers. To test the accuracy of this type of analysis with the microarray, we used a hybrid where wild-type <em>N. giraulti</em> were backcrossed in a <em>N. vitripennis</em> background with linked mutations <em>st-318</em> and <em>mm</em> (which create red and bulging eyes, respectively). Heterozygous females were then set onto hosts as virgins, and haploid male progeny recombinant between the two mutations were sorted into two sets: individuals with the wild-type <em>N. giraulti</em> st-318 phenotype and mutant <em>N. vitripennis</em> mm phenotype (<em>st-318<sup>+g</sup></em>,<em>mm</em>) and individuals with the mutant <em>N. vitripennis</em> st-318 phenotype and wild-type <em>N. giraulti</em> mm phenotype (<em>st-318</em>,<em>mm<sup>+g</sup></em>).</p><p id="p-45">Each set was genotyped by both PCR and the microarray, and the results of are shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. For all PCR markers, the proportion of each species allele in the bulk sample as estimated with the array is roughly equivalent to the proportion of each allele as determined by individual PCR. It should be noted that for these two measures to be theoretically equivalent, we assume that the DNA content of individuals in the bulk sample is independent of their genotype. As we are performing the analysis on adults, we also assume that survival to adulthood in these hybrid males is independent of their genotype at the loci under investigation. Therefore, this method should be used cautiously when the segregating phenotype causes large changes in body size or is associated with differential mortality. Pools should also be of either all male or all female samples to avoid differences due to ploidy levels (haploid males <em>vs.</em> diploid females).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462398/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462398/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Allele composition of bulk segregant pools based on individual PCR and pooled microarray hybridization</span><div class="sb-div caption-clear"></div></div></div><p id="p-48">We next used the microarray-based allele frequency estimates of the <em>st-318<sup>+g</sup></em>,<em>mm</em> recombinants to identify a candidate gene for the st-318 red eye phenotype. These recombinants are all wild-type for <em>st-318</em> and therefore the pool of these individuals should have a near zero percentage of the (mutant) <em>N. vitripennis</em> allele directly around the <em>st-318</em> locus. However, because the wild-type <em>N. giraulti st-318</em> allele was backcrossed repeatedly into a <em>N. vitripennis</em> genetic background, the percentage of the <em>N. vitripennis</em> allele in the pool should rise sharply to near 100% at the edge of the introgressed region between the <em>st-318</em> locus and the end of the chromosome. Additionally, because each individual in the pool has a (mutant) <em>N. vitripennis mm</em> allele, the percentage of the <em>N. vitripennis</em> allele should rise gradually toward 100% along the chromosome approaching the <em>mm</em> locus.</p><p id="p-49">Estimated proportions of <em>N. vitripennis</em> alleles in <em>st-318<sup>+g</sup></em>,<em>mm</em> recombinants for markers on scaffold 17, which contains the region under study, are shown graphically in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. In the <em>st-318<sup>+g</sup></em>,<em>mm</em> recombinants the proportion of the <em>N. vitripennis</em> alleles declines steadily along scaffold 17 until around 1.44 Mb, where the proportion approaches zero before sharply rising to 100% <em>N. vitripennis</em> at 1.49 Mb. As this sharp increase should be near the <em>st-318</em> locus, we examined this region and noted the gene <em>cinnabar</em> at 1.38 Mb. <em>Cinnabar</em> is an eye color gene which is already known to produce a red-eye phenotype after RNAi knockdown (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Werren <em>et al.</em> 2009</a>), and we therefore hypothesize that the st-318 phenotype is the result of a mutation in this gene. Here, hybridization of a single sample to the array has accurately estimated the true proportion of alleles belonging to each genotype along the region of interest, allowing a causal gene to be hypothesized. A similar approach was not employed with <em>mm</em> due to variable expressivity of the mutation and low recombination rate in the region.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/205/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bulk segregant analysis suggesting that the st-318 red-eye phenotype is caused by the gene cinnabar. A mutant N. vitripennis strain (st-318,mm) was crossed with a wild-type N. giraulti introgression of the region (st-318+g,mm+g) in a N. vitripennis genetic background. Twenty-four recombinants with the wild-type N. giraulti st-318 phenotype and mutant N. vitripennis mm phenotype (st-318+g,mm) were collected, pooled, and genotyped. Shown here is the proportion of sample with N. vitripennis genotype for markers across scaffold 17, 50&#x2212;200 Kb in the st-318+g,mm pool. Because all recombinants show the wild-type N. giraulti st-318 phenotype, the proportion of individuals with an N. vitripennis genotype should approach zero at the site of st-318 locus. Gene presence along the same region is shown in black bars below. Cinnabar, a gene known to cause a red-eye phenotype, is indicated in red and is located in the noted region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-473653616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bulk segregant analysis suggesting that the st-318 red-eye phenotype is caused by the gene &lt;em&gt;cinnabar&lt;/em&gt;. A mutant &lt;em&gt;N. vitripennis&lt;/em&gt; strain (&lt;em&gt;st-318&lt;/em&gt;,&lt;em&gt;mm&lt;/em&gt;) was crossed with a wild-type &lt;em&gt;N. giraulti&lt;/em&gt; introgression of the region (&lt;em&gt;st-318&lt;sup&gt;+g&lt;/sup&gt;&lt;/em&gt;,&lt;em&gt;mm&lt;sup&gt;+g&lt;/sup&gt;&lt;/em&gt;) in a &lt;em&gt;N. vitripennis&lt;/em&gt; genetic background. Twenty-four recombinants with the wild-type &lt;em&gt;N. giraulti&lt;/em&gt; st-318 phenotype and mutant &lt;em&gt;N. vitripennis&lt;/em&gt; mm phenotype (&lt;em&gt;st-318&lt;sup&gt;+g&lt;/sup&gt;&lt;/em&gt;,&lt;em&gt;mm&lt;/em&gt;) were collected, pooled, and genotyped. Shown here is the proportion of sample with &lt;em&gt;N. vitripennis&lt;/em&gt; genotype for markers across scaffold 17, 50&#x2212;200 Kb in the &lt;em&gt;st-318&lt;sup&gt;+g&lt;/sup&gt;&lt;/em&gt;,&lt;em&gt;mm&lt;/em&gt; pool. Because all recombinants show the wild-type &lt;em&gt;N. giraulti&lt;/em&gt; st-318 phenotype, the proportion of individuals with &lt;em&gt;an N. vitripennis&lt;/em&gt; genotype should approach zero at the site of &lt;em&gt;st-318&lt;/em&gt; locus. Gene presence along the same region is shown in black bars below. &lt;em&gt;Cinnabar&lt;/em&gt;, a gene known to cause a red-eye phenotype, is indicated in red and is located in the noted region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/205/F4.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/205/F4.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/205/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/205/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462450" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-50" class="first-child">Bulk segregant analysis suggesting that the st-318 red-eye phenotype is caused by the gene <em>cinnabar</em>. A mutant <em>N. vitripennis</em> strain (<em>st-318</em>,<em>mm</em>) was crossed with a wild-type <em>N. giraulti</em> introgression of the region (<em>st-318<sup>+g</sup></em>,<em>mm<sup>+g</sup></em>) in a <em>N. vitripennis</em> genetic background. Twenty-four recombinants with the wild-type <em>N. giraulti</em> st-318 phenotype and mutant <em>N. vitripennis</em> mm phenotype (<em>st-318<sup>+g</sup></em>,<em>mm</em>) were collected, pooled, and genotyped. Shown here is the proportion of sample with <em>N. vitripennis</em> genotype for markers across scaffold 17, 50−200 Kb in the <em>st-318<sup>+g</sup></em>,<em>mm</em> pool. Because all recombinants show the wild-type <em>N. giraulti</em> st-318 phenotype, the proportion of individuals with <em>an N. vitripennis</em> genotype should approach zero at the site of <em>st-318</em> locus. Gene presence along the same region is shown in black bars below. <em>Cinnabar</em>, a gene known to cause a red-eye phenotype, is indicated in red and is located in the noted region.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Extrapolation to other Nasonia species</h3><p id="p-51">We tested the ability of the array to distinguish between other Nasonia species pairs. Similar to the original <em>N. vitripennis</em>−<em>N. giraulti</em> comparisons, hybridization values for <em>N. vitripennis</em>−<em>N. oneida</em> and <em>N. vitripennis</em>−<em>N. longicornis</em> comparisons showed largely non-overlapping distributions with <em>N. vitripennis</em> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). For the <em>N. vitripennis</em>−<em>N. oneida</em> and <em>N. vitripennis</em>−<em>N. longicornis</em> comparisons, 91.4% and 85.5% of markers, respectively, had median hybridization values separated by at least 1 SD. This result correlates with the phylogenetic relationships of these species, <em>i.e.</em>, <em>N. giraulti</em> and <em>N. oneida</em> are more closely related to each other than to <em>N. longicornis</em> (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Raychoudhury <em>et al.</em> 2010</a>). Furthermore, it suggests that although there is some loss of useful markers relative to the original <em>N. vitripennis</em>−<em>N. giraulti</em> comparison, the array can still effectively genotype a large number of markers in <em>N. vitripennis</em>−<em>N. oneida</em> and <em>N. vitripennis</em>−<em>N. longicornis</em> comparisons. To further evaluate the use of the array for genotyping <em>N. vitripennis</em>−<em>N. longicornis</em> hybrids, we calculated precision to be 98.4% and recall to be 91.2% for a SIL from this species pair (SIL17_ws1_lV), demonstrating that much of the array can effectively distinguish between <em>N. vitripennis</em> and <em>N. longicornis</em> DNA. Finally, the array was tested for its ability to distinguish genotypes of hybrids between the two more closely related species <em>N. giraulti</em> and <em>N. longicornis</em>. In this case, only 38.2% of markers had median hybridization values separated by at least at least one standard deviation. Although it may be possible to genotype large stretches of DNA in <em>N. giraulti</em>−<em>N. longicornis</em> hybrids, the array likely is of limited use for genotyping hybrids of Nasonia species pairs that do not involve <em>N. vitripennis</em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/205/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency histograms of hybridization values when hybridized with (A) N. oneida (No; white) vs. N. vitripennis (Nv; gray), and (B) N. longicornis (Nl; white) vs. N. vitripennis (Nv; gray). The Y-axis shows the number of markers that produced the relevant hybridization values of the 20,551 total markers. Hybridization values &gt; 0 indicate greater hybridization to the N. vitripennis oligo, while hybridization values &lt; 0 indicate greater hybridization to the N. giraulti oligo. A large fraction of markers successfully discriminate between the two species pairs, as most markers produced greater hybridization to the correct oligo when hybridized with control DNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-473653616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency histograms of hybridization values when hybridized with (A) &lt;em&gt;N. oneida&lt;/em&gt; (No; white) &lt;em&gt;vs. N. vitripennis&lt;/em&gt; (Nv; gray), and (B) &lt;em&gt;N. longicornis&lt;/em&gt; (Nl; white) &lt;em&gt;vs. N. vitripennis&lt;/em&gt; (Nv; gray). The Y-axis shows the number of markers that produced the relevant hybridization values of the 20,551 total markers. Hybridization values &gt; 0 indicate greater hybridization to the &lt;em&gt;N. vitripennis&lt;/em&gt; oligo, while hybridization values &lt; 0 indicate greater hybridization to the &lt;em&gt;N. giraulti&lt;/em&gt; oligo. A large fraction of markers successfully discriminate between the two species pairs, as most markers produced greater hybridization to the correct oligo when hybridized with control DNA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/205/F5.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/205/F5.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/205/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/205/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462472" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-52" class="first-child">Frequency histograms of hybridization values when hybridized with (A) <em>N. oneida</em> (No; white) <em>vs. N. vitripennis</em> (Nv; gray), and (B) <em>N. longicornis</em> (Nl; white) <em>vs. N. vitripennis</em> (Nv; gray). The Y-axis shows the number of markers that produced the relevant hybridization values of the 20,551 total markers. Hybridization values &gt; 0 indicate greater hybridization to the <em>N. vitripennis</em> oligo, while hybridization values &lt; 0 indicate greater hybridization to the <em>N. giraulti</em> oligo. A large fraction of markers successfully discriminate between the two species pairs, as most markers produced greater hybridization to the correct oligo when hybridized with control DNA.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Conclusions</h3><p id="p-53">A CGH array containing more than 20,000 markers representing SNP-cluster and indel polymorphisms between <em>N. vitripennis</em> and <em>N. giraulti</em> has been successfully used to create a high-resolution linkage map of Nasonia, covering 79% of the assembled genome of <em>N. vitripennis</em>. Errors in the original <em>N. vitripennis</em> assembly have been identified through mapping, and this information can be used to improve the next assembly. The CGH array also provides an inexpensive means of mapping QTL involved in species differences and visible mutants onto the linkage map. The array can be used for bulk-segregant analysis, as was demonstrated by identifying the gene <em>cinnabar</em> as a potential cause of the red eye mutant <em>st-318</em>. Hybrids of <em>N. vitripennis−N. oneida</em> and <em>N. vitripennis−N. longicornis</em> species pairs also can be genotyped using the microarray due to shared fixed marker differences between <em>N. longicornis</em>, <em>N. oneida</em>, and <em>N. giraulti</em>. The genetic map and 26 SILs presented here will further promote Nasonia as a model for studying the genetics of complex traits.</p><p id="p-54">The microarray design, including oligo sequences and SNP information is available by request from the authors. Perl scripts for processing data are available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1/FileS2.zip">File S2</a>, and additional scripts for data analysis are available by request from the authors. Segmental introgression lines reported here can be obtained by contacting J. H. Werren (<span class="em-link"><span class="em-addr">werr{at}mail.rochester.edu</span></span>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank Rachel Edwards for maintenance of SILs, and Bart Pannebakker and Tosca Koevoets for feedback on the output of the microarray analysis Perl scripts. The research was supported by NIH grant to J.H.W., 1R24GM084917 “Genetic and Genomic Tools for the Emerging Model Organism, Nasonia” with subawards to J.K.C. and J.G. This work was also supported in part by the METACyt Initiative of Indiana University, funded in part through a major grant from the Lilly Endowment, Inc. and Indiana 21st Century Research and Technology Fund. O.N. acknowledges the Alexander von Humboldt Foundation for a Feodor Lynen research stipend for postdoctoral researchers.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004739/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-56"><em>Communicating editor: S. Celniker</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-05-10"><span class="received-label">Received </span>May 10, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-11-30"><span class="accepted-label">Accepted </span>November 30, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Desjardins <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.1" data-doi="10.2174/138920210790886835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berchowitz</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic interference: don’t stand so close to me</span>. <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DBerchowitz%26rft.auinit1%253DL.%2BE.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D91%26rft.epage%253D102%26rft.atitle%253DGenetic%2Binterference%253A%2Bdon%2527t%2Bstand%2Bso%2Bclose%2Bto%2Bme.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920210790886835%26rft_id%253Dinfo%253Apmid%252F20885817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920210790886835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20885817&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275150100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.2" data-doi="10.1016/j.cub.2003.10.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beukeboom</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desplan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nasonia</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBeukeboom%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D22%26rft.spage%253DR860%26rft.epage%253DR860%26rft.atitle%253DNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.10.042%26rft_id%253Dinfo%253Apmid%252F14614836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.10.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14614836&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186558100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.3" data-doi="10.2307/2410324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breeuwer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Hybrid breakdown between two haplodiploid species-the role of nuclear and cytoplasmic genes</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D49%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RZ50400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.4" data-doi="10.1038/hdy.2009.145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perfectti</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartos</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enders</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic basis of interspecies host preference differences in the model parasitoid Nasonia</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">270</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DDesjardins%26rft.auinit1%253DC.%2BA.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D270%26rft.epage%253D277%26rft.atitle%253DThe%2Bgenetic%2Bbasis%2Bof%2Binterspecies%2Bhost%2Bpreference%2Bdifferences%2Bin%2Bthe%2Bmodel%2Bparasitoid%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.145%26rft_id%253Dinfo%253Apmid%252F20087393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20087393&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274604200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Mapping of hybrid incompatibility loci in Nasonia</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1731</span>–<span class="cit-lpage">1741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D153%26rft.spage%253D1731%26rft_id%253Dinfo%253Apmid%252F10581280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10581280&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The genetic basis of the interspecific differences in wing size in Nasonia (Hymenoptera; Pteromalidae): Major quantitative trait loci and epistasis</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D161%26rft.spage%253D673%26rft_id%253Dinfo%253Apmid%252F12072464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12072464&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.7" data-doi="10.1038/hdy.2009.172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrelli</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Contrasting patterns of selective constraints in nuclear-encoded genes of the oxidative phosphorylation pathway in holometabolous insects and their possible role in hybrid breakdown in Nasonia</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGibson%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D310%26rft.epage%253D317%26rft.atitle%253DContrasting%2Bpatterns%2Bof%2Bselective%2Bconstraints%2Bin%2Bnuclear-encoded%2Bgenes%2Bof%2Bthe%2Boxidative%2Bphosphorylation%2Bpathway%2Bin%2Bholometabolous%2Binsects%2Band%2Btheir%2Bpossible%2Brole%2Bin%2Bhybrid%2Bbreakdown%2Bin%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.172%26rft_id%253Dinfo%253Apmid%252F20087391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20087391&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274604200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.8" data-doi="10.1038/86730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burchard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marton</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D342%26rft.epage%253D347%26rft.atitle%253DExpression%2Bprofiling%2Busing%2Bmicroarrays%2Bfabricated%2Bby%2Ban%2Bink-jet%2Boligonucleotide%2Bsynthesizer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F86730%26rft_id%253Dinfo%253Apmid%252F11283592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/86730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11283592&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167940700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.205.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koevoets</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Deadly Combinations: Hybrid incompatibilities in the Parasitic Wasp Genus Nasonia</span>. <span class="cit-comment">Ph.D. Thesis</span>, <span class="cit-publ-name">University of Groningen</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.10" data-doi="10.1038/hdy.2011.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koevoets</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Zande</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beukeboom</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid incompatibilities in the parasitic wasp genus Nasonia: negative effects of hemizygosity and the identification of transmission ratio distortion loci</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DKoevoets%26rft.auinit1%253DT.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D302%26rft.epage%253D311%26rft.atitle%253DHybrid%2Bincompatibilities%2Bin%2Bthe%2Bparasitic%2Bwasp%2Bgenus%2BNasonia%253A%2Bnegative%2Beffects%2Bof%2Bhemizygosity%2Band%2Bthe%2Bidentification%2Bof%2Btransmission%2Bratio%2Bdistortion%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.75%26rft_id%253Dinfo%253Apmid%252F21878985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878985&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300596100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.11" data-doi="10.1038/hdy.2009.146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enders</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010a</span> <span class="cit-article-title">Evolution of sex-specific wing shape at the widerwing locus in four species of Nasonia</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DLoehlin%26rft.auinit1%253DD.%2BW.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D260%26rft.epage%253D269%26rft.atitle%253DEvolution%2Bof%2Bsex-specific%2Bwing%2Bshape%2Bat%2Bthe%2Bwiderwing%2Blocus%2Bin%2Bfour%2Bspecies%2Bof%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.146%26rft_id%253Dinfo%253Apmid%252F20087390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20087390&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274604200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.12" data-doi="10.1371/journal.pgen.1000821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giebel</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010b</span> <span class="cit-article-title">Non-coding changes cause sex-specific wing size differences between closely related species of Nasonia</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLoehlin%26rft.auinit1%253DD.%2BW.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000821%26rft.epage%253De1000821%26rft.atitle%253DNon-coding%2Bchanges%2Bcause%2Bsex-specific%2Bwing%2Bsize%2Bdifferences%2Bbetween%2Bclosely%2Brelated%2Bspecies%2Bof%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000821%26rft_id%253Dinfo%253Apmid%252F20090834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20090834&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.13" data-doi="10.1126/science.1215193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of shape by multiple regulatory changes to a growth gene</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLoehlin%26rft.auinit1%253DD.%2BW.%26rft.volume%253D335%26rft.issue%253D6071%26rft.spage%253D943%26rft.epage%253D947%26rft.atitle%253DEvolution%2Bof%2BShape%2Bby%2BMultiple%2BRegulatory%2BChanges%2Bto%2Ba%2BGrowth%2BGene%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1215193%26rft_id%253Dinfo%253Apmid%252F22363002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA3MS85NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.14" data-doi="10.1093/nar/gkg573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silventoinen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kibria</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">WU-Blast2 server at the European Bioinformatics Institute</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3795</span>–<span class="cit-lpage">3798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLopez%26rft.auinit1%253DR.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3795%26rft.epage%253D3798%26rft.atitle%253DWU-Blast2%2Bserver%2Bat%2Bthe%2BEuropean%2BBioinformatics%2BInstitute%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg573%26rft_id%253Dinfo%253Apmid%252F12824421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMy8zNzk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzIwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.15" data-doi="10.1038/nprot.2006.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desplan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A method for parental RNA interference in the wasp <em>Nasonia vitripennis</em></span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DLynch%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D486%26rft.epage%253D494%26rft.atitle%253DA%2Bmethod%2Bfor%2Bparental%2BRNA%2Binterference%2Bin%2Bthe%2Bwasp%2BNasonia%2Bvitripennis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2006.70%26rft_id%253Dinfo%253Apmid%252F17406271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2006.70&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17406271&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251002200069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.16" data-doi="10.1007/s00122-003-1305-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Efficient multipoint mapping: making use of dominant repulsion-phase markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMester%26rft.auinit1%253DD.%2BI.%26rft.volume%253D107%26rft.issue%253D6%26rft.spage%253D1102%26rft.epage%253D1112%26rft.atitle%253DEfficient%2Bmultipoint%2Bmapping%253A%2Bmaking%2Buse%2Bof%2Bdominant%2Brepulsion-phase%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1305-1%26rft_id%253Dinfo%253Apmid%252F12928774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1305-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12928774&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185811300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.17" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8yMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.18" data-doi="10.1086/279534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1916</span> <span class="cit-article-title">The mechanism of crossing-over</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D50%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200875900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.19" data-doi="10.1534/genetics.107.080523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Judson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cytonuclear genic incompatibilities cause increased mortality in male F2 hybrids of <em>Nasonia giraulti</em> and <em>N. vitripennis</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D413%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080523%26rft_id%253Dinfo%253Apmid%252F18202384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.080523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18202384&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252742200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.20" data-doi="10.1371/journal.pone.0008597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pannebakker</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koevoets</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Recombination and its impact on the genome of the haplodiploid parasitoid wasp Nasonia</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e8597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNiehuis%26rft.auinit1%253DO.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De8597%26rft.epage%253De8597%26rft.atitle%253DRecombination%2Band%2Bits%2Bimpact%2Bon%2Bthe%2Bgenome%2Bof%2Bthe%2Bhaplodiploid%2Bparasitoid%2Bwasp%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008597%26rft_id%253Dinfo%253Apmid%252F20087411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008597&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20087411&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.21" data-doi="10.1038/hdy.2010.157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Büllesbach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Judson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetics of cuticular hydrocarbon differences between males of the parasitoid wasps <em>Nasonia giraulti</em> and <em>Nasonia vitripennis</em></span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DNiehuis%26rft.auinit1%253DO.%26rft.volume%253D107%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D70%26rft.atitle%253DGenetics%2Bof%2Bcuticular%2Bhydrocarbon%2Bdifferences%2Bbetween%2Bmales%2Bof%2Bthe%2Bparasitoid%2Bwasps%2BNasonia%2Bgiraulti%2Band%2BNasonia%2Bvitripennis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.157%26rft_id%253Dinfo%253Apmid%252F21179062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179062&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291635500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.22" data-doi="10.1038/hdy.2009.147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raychoudhury</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buellesbach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grillenberger</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Behavioral and genetic characteristics of a new species of Nasonia</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DRaychoudhury%26rft.auinit1%253DR.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D278%26rft.epage%253D288%26rft.atitle%253DBehavioral%2Band%2Bgenetic%2Bcharacteristics%2Bof%2Ba%2Bnew%2Bspecies%2Bof%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.147%26rft_id%253Dinfo%253Apmid%252F20087394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20087394&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274604200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.23" data-doi="10.1159/000078019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutten</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pietsch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neusser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beukeboom</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Chromosomal anchoring of linkage groups and identification of wing size QTL using markers and FISH probes derived from microdissected chromosomes in Nasonia(Pteromalidae: Hymenoptera)</span>. <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DR%25C3%25BCtten%26rft.auinit1%253DK.%2BB.%26rft.volume%253D105%26rft.issue%253D1%26rft.spage%253D126%26rft.epage%253D133%26rft.atitle%253DChromosomal%2Banchoring%2Bof%2Blinkage%2Bgroups%2Band%2Bidentification%2Bof%2Bwing%2Bsize%2BQTL%2Busing%2Bmarkers%2Band%2BFISH%2Bprobes%2Bderived%2Bfrom%2Bmicrodissected%2Bchromosomes%2Bin%2BNasonia%2B%2528Pteromalidae%253A%2BHymenoptera%2529.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000078019%26rft_id%253Dinfo%253Apmid%252F15218268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000078019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15218268&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F205.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-9" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.205.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saul</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Gene map of the parasitic wasp <em>Nasonia vitripennis</em> (=<em>Mormoniella vitripennis</em>)</span>, <span class="cit-comment">pp</span>. <span class="cit-fpage">3.198</span>–<span class="cit-lpage">193.201</span> <span class="cit-comment">in</span> <span class="cit-source">Genetic Maps: Locus Maps of Complex Genomes</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">S. J.</span></span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1915</span> <span class="cit-article-title">The behavior of chromosomes as studied through linkage</span>. <abbr class="cit-jnl-abbrev">Z fur induk. Abst. Vererb.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZ%2Bfur%2Binduk.%2BAbst.%2BVererb.%26rft.volume%253D13%26rft.spage%253D234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.26" data-doi="10.1093/bioinformatics/btg086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seed</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Selection of oligonucleotide probes for protein coding sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D796%26rft.epage%253D802%26rft.atitle%253DSelection%2Bof%2Boligonucleotide%2Bprobes%2Bfor%2Bprotein%2Bcoding%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg086%26rft_id%253Dinfo%253Apmid%252F12724288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzc5NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8yMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.27" data-doi="10.1101/pdb.emo134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The parasitoid wasp Nasonia: an emerging model system with haploid male genetics</span>. <abbr class="cit-jnl-abbrev">Cold Spring Harb. Protoc.</abbr> <span class="cit-vol">2009</span>: <span class="cit-fpage">pdb.emo134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BProtocols%26rft.stitle%253DCold%2BSpring%2BHarb%2BProtoc%26rft.aulast%253DWerren%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D2009%26rft.issue%253D10%26rft.spage%253Dpdb.emo134%26rft.epage%253Dpdb.emo134%26rft.atitle%253DThe%2BParasitoid%2BWasp%2BNasonia%253A%2BAn%2BEmerging%2BModel%2BSystem%2Bwith%2BHaploid%2BMale%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.emo134%26rft_id%253Dinfo%253Apmid%252F20147035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicHJvdG9jb2xzIjtzOjU6InJlc2lkIjtzOjE4OiIyMDA5LzEwL3BkYi5lbW8xMzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loehlin</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giebel</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Larval RNAi in Nasonia (parasitoid wasp)</span>. <abbr class="cit-jnl-abbrev">Cold Spring Harb. Protoc.</abbr> <span class="cit-pub-date">2009</span><strong>:</strong> <span class="cit-fpage">pdb.prot5311</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.2.205.29" data-doi="10.1093/jhered/90.2.319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrot-Minnot</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">An antennapedia mutant in <em>Nasonia vitripennis</em></span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D90%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F90.2.319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/90.2.319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.205.30" data-doi="10.1126/science.1178028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional and evolutionary insights from the genomes of three parasitoid Nasonia species</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWerren%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D327%26rft.issue%253D5963%26rft.spage%253D343%26rft.epage%253D348%26rft.atitle%253DFunctional%2Band%2BEvolutionary%2BInsights%2Bfrom%2Bthe%2BGenomes%2Bof%2BThree%2BParasitoid%2BNasonia%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178028%26rft_id%253Dinfo%253Apmid%252F20075255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2My8zNDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMjA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/2/205.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/197" title="Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/217" title="Genome-Wide Copy Number Variant Analysis in Inbred Chickens Lines With Different Susceptibility to Marek’s Disease" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/2"><span>Volume 3 Issue 2, February 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/2.cover-source.jpg?itok=h3kYZ-FP" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/205.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M_N6aNQVP_ksLPY1x3-IIaUajdR0FTXY7FKRBql0t4M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/2/205" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/2/205" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/2/205" class="active" data-icon-position="" data-hide-link-title="0">Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459412" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-K4QvdIjtldwcLkhbMPva-7QoIYhXOn4yeO6o7X4y4wg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/205.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/2/205" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IIX1QbxwRDDg_PfOHw4ObejX7757UlkhQg9ivct1c2A" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/2/205" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/205.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459412" id="node459412--2" data-pisa="ggg;3/2/205" data-pisa-master="ggg;3/2/205" data-apath="/ggg/3/2/205.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher A. Desjardins</span>, <span class="highwire-citation-author" data-delta="1">Jürgen Gadau</span>, <span class="highwire-citation-author" data-delta="2">Jacqueline A. Lopez</span>, <span class="highwire-citation-author" data-delta="3">Oliver Niehuis</span>, <span class="highwire-citation-author" data-delta="4">Amanda R. Avery</span>, <span class="highwire-citation-author" data-delta="5">David W. Loehlin</span>, <span class="highwire-citation-author" data-delta="6">Stephen Richards</span>, <span class="highwire-citation-author" data-delta="7">John K. Colbourne</span> and <span class="highwire-citation-author" data-delta="8">John H. Werren</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">205-215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004739 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459412--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher A. Desjardins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Broad Institute, Cambridge, Massachusetts 02141</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BDesjardins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desjardins%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BA.%2BDesjardins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cdesjard@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">cdesjard@broadinstitute.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jürgen Gadau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Arizona State University, Tempe, Arizona 85287</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%BCrgen%2BGadau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gadau%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25BCrgen%2BGadau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacqueline A. Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BA.%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BA.%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Oliver Niehuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Arizona State University, Tempe, Arizona 85287</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Molecular Biodiversity Research, Zoological Research Museum Alexander Koenig, 53113 Bonn, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BNiehuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niehuis%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOliver%2BNiehuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Amanda R. Avery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BAvery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avery%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BR.%2BAvery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David W. Loehlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BLoehlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loehlin%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BW.%2BLoehlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">John K. Colbourne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BColbourne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Colbourne%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BColbourne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">John H. Werren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BWerren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Werren%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BWerren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/2/205" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/2/205&amp;title=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/2/205&amp;title=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/2/205&amp;title=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/2/205&amp;text=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/2/205&amp;title=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/2/205&amp;t=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/2/205&amp;title=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/2/205&amp;title=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459412/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459412" id="node459412--3" data-pisa="ggg;3/2/205" data-pisa-master="ggg;3/2/205" data-apath="/ggg/3/2/205.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher A. Desjardins</span>, <span class="highwire-citation-author" data-delta="1">Jürgen Gadau</span>, <span class="highwire-citation-author" data-delta="2">Jacqueline A. Lopez</span>, <span class="highwire-citation-author" data-delta="3">Oliver Niehuis</span>, <span class="highwire-citation-author" data-delta="4">Amanda R. Avery</span>, <span class="highwire-citation-author" data-delta="5">David W. Loehlin</span>, <span class="highwire-citation-author" data-delta="6">Stephen Richards</span>, <span class="highwire-citation-author" data-delta="7">John K. Colbourne</span> and <span class="highwire-citation-author" data-delta="8">John H. Werren</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">205-215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004739 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459412--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher A. Desjardins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Broad Institute, Cambridge, Massachusetts 02141</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BDesjardins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desjardins%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BA.%2BDesjardins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cdesjard@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">cdesjard@broadinstitute.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jürgen Gadau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Arizona State University, Tempe, Arizona 85287</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%BCrgen%2BGadau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gadau%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25BCrgen%2BGadau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacqueline A. Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BA.%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BA.%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Oliver Niehuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Arizona State University, Tempe, Arizona 85287</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Molecular Biodiversity Research, Zoological Research Museum Alexander Koenig, 53113 Bonn, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BNiehuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niehuis%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOliver%2BNiehuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Amanda R. Avery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BAvery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avery%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BR.%2BAvery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David W. Loehlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BLoehlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loehlin%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BW.%2BLoehlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">John K. Colbourne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Center for Genomics and Bioinformatics, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BColbourne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Colbourne%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BColbourne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">John H. Werren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BWerren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Werren%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BWerren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=205&amp;rft.epage=215&amp;rft.atitle=Fine-Scale%20Mapping%20of%20the%20Nasonia%20Genome%20to%20Chromosomes%20Using%20a%20High-Density%20Genotyping%20Microarray&amp;rft.volume=3&amp;rft.issue=2&amp;rft.date=2013-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Desjardins%2C+Christopher+A.&amp;rft.au=Gadau%2C+J%C3%BCrgen&amp;rft.au=Lopez%2C+Jacqueline+A.&amp;rft.au=Niehuis%2C+Oliver&amp;rft.au=Avery%2C+Amanda+R.&amp;rft.au=Loehlin%2C+David+W.&amp;rft.au=Richards%2C+Stephen&amp;rft.au=Colbourne%2C+John+K.&amp;rft.au=Werren%2C+John+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459412/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459412/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459412/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459412/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459412/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459412/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459412/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459412/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459412/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459412/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459412/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459412/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/2/205" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Materials</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/2/205.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/2/205.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/2/205.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/2/205.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
