<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/2/343" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/2/343.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/2/343.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/2/343.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/2/343.atom" />
<meta name="HW.pisa" content="ggg;3/2/343" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.112.003640" />
<meta name="DC.Date" content="2013-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Waern, Snyder. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="To understand the diversity of transcripts in yeast ( Saccharomyces cerevisiae ) we analyzed the transcriptional landscapes for cells grown under 18 different environmental conditions. Each sample was analyzed using RNA-sequencing, and a total of 670,446,084 uniquely mapped reads and 377,263 poly-adenylated end tags were produced. Consistent with previous studies, we find that the majority of yeast genes are expressed under one or more different conditions. By directly comparing the 5′ and 3′ ends of the transcribed regions, we find extensive differences in transcript ends across many conditions, especially those of stationary phase, growth in grape juice, and salt stimulation, suggesting differential choice of transcription start and stop sites is pervasive in yeast. Relative to the exponential growth condition ( i.e. , YPAD), transcripts differing at the 5′ ends and 3′ ends are predicted to differ in their annotated start codon in 21 genes and their annotated stop codon in 63 genes. Many (431) upstream open reading frames (uORFs) are found in alternate 5′ ends and are significantly enriched in transcripts produced during the salt response. Mutational analysis of five genes with uORFs revealed that two sets of uORFs increase the expression of a reporter construct, indicating a role in activation which had not been reported previously, whereas two other uORFs decreased expression. In addition, RNA binding protein motifs are statistically enriched for alternate ends under many conditions. Overall, these results demonstrate enormous diversity of transcript ends, and that this heterogeneity is regulated under different environmental conditions. Moreover, transcript end diversity has important biological implications for the regulation of gene expression. In addition, our data also serve as a valuable resource for the scientific community." />
<meta name="DC.Contributor" content="Karl Waern" />
<meta name="DC.Contributor" content="Michael Snyder" />
<meta name="article:published_time" content="2013-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;To understand the diversity of transcripts in yeast (&lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;) we analyzed the transcriptional landscapes for cells grown under 18 different environmental conditions. Each sample was analyzed using RNA-sequencing, and a total of 670,446,084 uniquely mapped reads and 377,263 poly-adenylated end tags were produced. Consistent with previous studies, we find that the majority of yeast genes are expressed under one or more different conditions. By directly comparing the 5′ and 3′ ends of the transcribed regions, we find extensive differences in transcript ends across many conditions, especially those of stationary phase, growth in grape juice, and salt stimulation, suggesting differential choice of transcription start and stop sites is pervasive in yeast. Relative to the exponential growth condition (&lt;i&gt;i.e.&lt;/i&gt;, YPAD), transcripts differing at the 5′ ends and 3′ ends are predicted to differ in their annotated start codon in 21 genes and their annotated stop codon in 63 genes. Many (431) upstream open reading frames (uORFs) are found in alternate 5′ ends and are significantly enriched in transcripts produced during the salt response. Mutational analysis of five genes with uORFs revealed that two sets of uORFs increase the expression of a reporter construct, indicating a role in activation which had not been reported previously, whereas two other uORFs decreased expression. In addition, RNA binding protein motifs are statistically enriched for alternate ends under many conditions. Overall, these results demonstrate enormous diversity of transcript ends, and that this heterogeneity is regulated under different environmental conditions. Moreover, transcript end diversity has important biological implications for the regulation of gene expression. In addition, our data also serve as a valuable resource for the scientific community.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/02/01" />
<meta name="citation_mjid" content="ggg;3/2/343" />
<meta name="citation_id" content="3/2/343" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/2/343" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/2/343.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/2/343.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/2/343.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003640" />
<meta name="citation_pmid" content="23390610" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="343" />
<meta name="citation_lastpage" content="352" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Karl Waern" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, Stanford, California 94305" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520" />
<meta name="citation_author" content="Michael Snyder" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, Stanford, California 94305" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520" />
<meta name="citation_author_email" content="mpsnyder@stanford.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Boyle;citation_author=S. Weng;citation_author=J. Gollub;citation_author=H. Jin;citation_author=D. Botstein;citation_title=GO:TermFinder—open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes;citation_volume=20;citation_year=2004;citation_pmid=15297299;citation_doi=10.1093/bioinformatics/bth456" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. A. Brar;citation_author=M. Yassour;citation_author=N. Friedman;citation_author=A. Regev;citation_author=N. T. Ingolia;citation_title=High-resolution view of the yeast meiotic program revealed by ribosome profiling;citation_volume=335;citation_year=2012;citation_pmid=22194413;citation_doi=10.1126/science.1215110" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. Burns;citation_author=B. Grimwade;citation_author=P. B. Ross-Macdonald;citation_author=E. Y. Choi;citation_author=K. Finberg;citation_title=Large-scale analysis of gene expression, protein localization, and gene disruption in Saccharomyces cerevisiae.;citation_pages=1087-1105;citation_volume=8;citation_year=1994;citation_issue=9;citation_pmid=7926789;citation_doi=10.1101/gad.8.9.1087" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Carlson;citation_author=D. Botstein;citation_title=Two differentially regulated mRNAs with different 5' ends encode secreted with intracellular forms of yeast invertase.;citation_pages=145-154;citation_volume=28;citation_year=1982;citation_issue=1;citation_pmid=7039847;citation_doi=10.1016/0092-8674(82)90384-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. T. Cheah;citation_author=A. Wachter;citation_author=N. Sudarsan;citation_author=R. R. Breaker;citation_title=Control of alternative RNA splicing and gene expression by eukaryotic riboswitches.;citation_pages=497-500;citation_volume=447;citation_year=2007;citation_issue=7143;citation_pmid=17468745;citation_doi=10.1038/nature05769" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. M. Cherry;citation_author=C. Adler;citation_author=C. Ball;citation_author=S. A. Chervitz;citation_author=S. S. Dwight;citation_title=SGD: Saccharomyces Genome Database;citation_volume=26;citation_year=1998;citation_pmid=9399804;citation_doi=10.1093/nar/26.1.73" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Chu;citation_author=D. J. Barnes;citation_author=T. von der Haar;citation_title=The role of tRNA and ribosome competition in coupling the expression of different mRNAs in Saccharomyces cerevisiae;citation_volume=39;citation_year=2011;citation_pmid=21558172;citation_doi=10.1093/nar/gkr300" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Cliften;citation_author=P. Sudarsanam;citation_author=A. Desikan;citation_author=L. Fulton;citation_author=B. Fulton;citation_title=Finding functional features in Saccharomyces genomes by phylogenetic footprinting;citation_volume=301;citation_year=2003;citation_pmid=12775844;citation_doi=10.1126/science.1084337" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=P. S. R. Coelho;citation_author=A. C. Bryan;citation_author=A. Kumar;citation_author=G. S. Shadel;citation_author=M. Snyder;citation_title=A novel mitochondrial protein, Tar1p, is encoded on the antisense strand of the nuclear 25S rDNA;citation_pages=656-664;citation_volume=16;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Cvijović;citation_author=D. Dalevi;citation_author=E. Bilsland;citation_author=G. J. L. Kemp;citation_author=P. Sunnerhagen;citation_title=Identification of putative regulatory upstream ORFs in the yeast genome using heuristics and evolutionary conservation.;citation_pages=295-295;citation_volume=8;citation_year=2007;citation_pmid=17686169;citation_doi=10.1186/1471-2105-8-295" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=L. David;citation_author=W. Huber;citation_author=M. Granovskaia;citation_author=J. Toedling;citation_author=C. J. Palm;citation_title=A high-resolution map of transcription in the yeast genome;citation_pages=5320-5325;citation_volume=103;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. A. Frischmeyer;citation_author=A. van Hoof;citation_author=K. O’Donnell;citation_author=A. L. Guerrerio;citation_author=R. Parker;citation_title=An mRNA Surveillance Mechanism That Eliminates Transcripts Lacking Termination Codons;citation_pages=2258-2261;citation_volume=295;citation_year=2002;citation_issue=5563;citation_pmid=11910109;citation_doi=10.1126/science.1067338" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=A. Gaba;citation_author=Z. Wang;citation_author=T. Krishnamoorthy;citation_author=A. G. Hinnebusch;citation_author=M. S. Sachs;citation_title=Physical evidence for distinct mechanisms of translational control by upstream open reading frames;citation_pages=6453-6463;citation_volume=20;citation_year=2001;citation_issue=22;citation_pmid=11707416;citation_doi=10.1093/emboj/20.22.6453" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=N. L. Garneau;citation_author=J. Wilusz;citation_author=C. J. Wilusz;citation_title=The highways and byways of mRNA decay.;citation_pages=113-126;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17245413;citation_doi=10.1038/nrm2104" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. Barrell;citation_author=H. Bussey;citation_author=R. Davis;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Guittaut;citation_author=S. Charpentier;citation_author=T. Normand;citation_author=M. Dubois;citation_author=J. Raimond;citation_title=Identification of an internal gene to the human Galectin-3 gene with two different overlapping reading frames that do not encode Galectin-3;citation_volume=276;citation_year=2001;citation_pmid=11160123;citation_doi=10.1074/jbc.M002523200" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=H. M. Hood;citation_author=D. E. Neafsey;citation_author=J. Galagan;citation_author=S. Sachs Matthew;citation_title=Evolutionary roles of upstream open reading frames in mediating gene regulation in fungi.;citation_pages=385-409;citation_volume=63;citation_year=2009;citation_pmid=19514854;citation_doi=10.1146/annurev.micro.62.081307.162835" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=K. S. Kim Guisbert;citation_author=Y. Zhang;citation_author=J. Flatow;citation_author=S. Hurtado;citation_author=J. P. Staley;citation_title=Meiosis-induced alterations in transcript architecture and noncoding RNA expression in S. cerevisiae;citation_volume=18;citation_year=2012;citation_pmid=22539527;citation_doi=10.1261/rna.030510.111" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol.;citation_author=V. Kochetov;citation_title=Alternative translation start sites and their significance for eukaryotic proteomes;citation_pages=705-712;citation_volume=40;citation_year=2006;citation_doi=10.1134/S0026893306050049" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Kumar;citation_author=P. M. Harrison;citation_author=K. H. Cheung;citation_author=N. Lan;citation_author=N. Echols;citation_title=An integrated approach for finding overlooked genes in yeast.;citation_pages=58-63;citation_volume=20;citation_year=2002;citation_issue=1;citation_pmid=11753363;citation_doi=10.1038/nbt0102-58" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J.-R. Landry;citation_author=D. L. Mager;citation_author=B. T. Wilhelm;citation_title=Complex controls: the role of alternative promoters in mammalian genomes.;citation_pages=640-648;citation_volume=19;citation_year=2003;citation_issue=11;citation_pmid=14585616;citation_doi=10.1016/j.tig.2003.09.014" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=G. L. Law;citation_author=K. S. Bickel;citation_author=V. L. MacKay;citation_author=D. R. Morris;citation_title=The undertranslated transcriptome reveals widespread translational silencing by alternative 5' transcript leaders.;citation_pages=R111-R111;citation_volume=6;citation_year=2005;citation_issue=13;citation_pmid=16420678;citation_doi=10.1186/gb-2005-6-13-r111" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Lawless;citation_author=R. D. Pearson;citation_author=J. N. Selley;citation_author=J. B. Smirnova;citation_author=C. M. Grant;citation_title=Upstream sequence elements direct post-transcriptional regulation of gene expression under stress conditions in yeast.;citation_pages=7-7;citation_volume=10;citation_year=2009;citation_pmid=19128476;citation_doi=10.1186/1471-2164-10-7" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=M. V. Lee;citation_author=S. E. Topper;citation_author=S. L. Hubler;citation_author=J. Hose;citation_author=C. D. Wenger;citation_title=A dynamic model of proteome changes reveals new roles for transcript alteration in yeast;citation_pages=514-514;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21772262;citation_doi=10.1038/msb.2011.48" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Z. Levin;citation_author=M. Yassour;citation_author=X. Adiconis;citation_author=C. Nusbaum;citation_author=D. A. Thompson;citation_title=Comprehensive comparative analysis of strand-specific RNA sequencing methods.;citation_pages=709-715;citation_volume=7;citation_year=2010;citation_issue=9;citation_pmid=20711195;citation_doi=10.1038/nmeth.1491" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Mangone;citation_author=A. P. Manoharan;citation_author=D. Thierry-Mieg;citation_author=J. Thierry-Mieg;citation_author=T. Han;citation_title=The landscape of C. elegans 3′UTRs;citation_volume=329;citation_year=2010;citation_pmid=20522740;citation_doi=10.1126/science.1191244" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=D. McNabb;citation_author=R. Reed;citation_title=Dual Luciferase Assay System for Rapid Assessment of Gene Expression in Saccharomyces cerevisiae;citation_pages=1539-1549;citation_volume=4;citation_year=2005;citation_issue=9;citation_pmid=16151247;citation_doi=10.1128/EC.4.9.1539-1549.2005" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol;citation_author=F. Mignone;citation_author=C. Gissi;citation_author=S. Liuni;citation_author=G. Pesole;citation_title=Untranslated regions of mRNAs;citation_pages=REVIEWS0004.1-0004.10;citation_volume=3;citation_year=2002;citation_doi=10.1186/gb-2002-3-3-reviews0004" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Miura;citation_author=N. Kawaguchi;citation_author=J. Sese;citation_author=A. Toyoda;citation_author=M. Hattori;citation_title=A large-scale full-length cDNA analysis to explore the budding yeast transcriptome;citation_volume=103;citation_year=2006;citation_pmid=17101987;citation_doi=10.1073/pnas.0605645103" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. P. Mueller;citation_author=A. G. Hinnebusch;citation_title=Multiple upstream AUG codons mediate translational control of GCN4.;citation_pages=201-207;citation_volume=45;citation_year=1986;citation_issue=2;citation_pmid=3516411;citation_doi=10.1016/0092-8674(86)90384-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=U. Nagalakshmi;citation_author=Z. Wang;citation_author=K. Waern;citation_author=C. Shou;citation_author=D. Raha;citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing;citation_volume=320;citation_year=2008;citation_pmid=18451266;citation_doi=10.1126/science.1158441" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Mol. Biol.;citation_author=U. Nagalakshmi;citation_author=W. Karl;citation_author=S. Michael;citation_title=RNA-SEQ: A METHOD FOR COMPREHENSIVE TRANSCRIPTOME ANALYSIS;citation_pages=4.11.1-4.11.13;citation_volume=89;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. E. Neafsey;citation_author=J. E. Galagan;citation_title=Dual modes of natural selection on upstream open reading frames;citation_volume=24;citation_year=2007;citation_pmid=17494029;citation_doi=10.1093/molbev/msm093" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. Ozsolak;citation_author=P. Kapranov;citation_author=S. Foissac;citation_author=S. W. Kim;citation_author=E. Fishilevich;citation_title=Comprehensive polyadenylation site maps in yeast and human reveal pervasive alternative polyadenylation.;citation_pages=1018-1029;citation_volume=143;citation_year=2010;citation_issue=6;citation_pmid=21145465;citation_doi=10.1016/j.cell.2010.11.020" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Parkhomchuk;citation_author=T. Borodina;citation_author=V. Amstislavskiy;citation_author=M. Banaru;citation_author=L. Hallen;citation_title=Transcriptome analysis by strand-specific sequencing of complementary DNA;citation_volume=37;citation_year=2009;citation_pmid=19620212;citation_doi=10.1093/nar/gkp596" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. Perepelitsa-Belancio;citation_author=P. Deininger;citation_title=RNA truncation by premature polyadenylation attenuates human mobile element activity.;citation_pages=363-366;citation_volume=35;citation_year=2003;citation_issue=4;citation_pmid=14625551;citation_doi=10.1038/ng1269" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Perocchi;citation_author=Z. Xu;citation_author=S. Clauder-Münster;citation_author=M. Steinmetz Lars;citation_title=Antisense artifacts in transcriptome microarray experiments are resolved by actinomycin D;citation_volume=35;citation_year=2007;citation_pmid=17897965;citation_doi=10.1093/nar/gkm683" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. N. Petersen;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions.;citation_pages=785-786;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21959131;citation_doi=10.1038/nmeth.1701" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. P. Riordan;citation_author=D. Herschlag;citation_author=P. O. Brown;citation_title=Identification of RNA recognition elements in the Saccharomyces cerevisiae transcriptome;citation_volume=39;citation_year=2011;citation_pmid=20959291;citation_doi=10.1093/nar/gkq920" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Ross-Macdonald;citation_author=P. S. Coelho;citation_author=T. Roemer;citation_author=S. Agarwal;citation_author=A. Kumar;citation_title=Large-scale analysis of the yeast genome by transposon tagging and gene disruption.;citation_pages=413-418;citation_volume=402;citation_year=1999;citation_issue=6760;citation_pmid=10586881;citation_doi=10.1038/46558" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Selpi;citation_author=C. H. Bryant;citation_author=G. J. Kemp;citation_author=J. Sarv;citation_author=E. Kristiansson;citation_title=Predicting functional upstream open reading frames in Saccharomyces cerevisiae.;citation_pages=451-451;citation_volume=10;citation_year=2009;citation_pmid=20042076;citation_doi=10.1186/1471-2105-10-451" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Shendure;citation_author=H. Ji;citation_title=Next-generation DNA sequencing.;citation_pages=1135-1145;citation_volume=26;citation_year=2008;citation_issue=10;citation_pmid=18846087;citation_doi=10.1038/nbt1486" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=The international journal of biochemistry &amp; cell biology;citation_journal_abbrev=Int J Biochem Cell Biol;citation_author=A. W. van der Velden;citation_author=A. A. Thomas;citation_title=The role of the 5' untranslated region of an mRNA in translation regulation during development.;citation_pages=87-106;citation_volume=31;citation_year=1999;citation_issue=1;citation_pmid=10216946;citation_doi=10.1016/S1357-2725(98)00134-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. Vilela;citation_author=J. McCarthy;citation_title=Regulation of fungal gene expression via short open reading frames in the mRNA 5'untranslated region.;citation_pages=859-867;citation_volume=49;citation_year=2003;citation_issue=4;citation_pmid=12890013;citation_doi=10.1046/j.1365-2958.2003.03622.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Vilela;citation_author=B. Linz;citation_author=C. Rodrigues-Pousada;citation_author=J. McCarthy;citation_title=The yeast transcription factor genes YAP1 and YAP2 are subject to differential control at the levels of both translation and mRNA stability;citation_volume=26;citation_year=1998;citation_pmid=9469820;citation_doi=10.1093/nar/26.5.1150" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=K. Waern;citation_author=U. Nagalakshmi;citation_author=M. Snyder;citation_title=RNA sequencing.;citation_pages=125-132;citation_volume=759;citation_year=2011;citation_pmid=21863485;citation_doi=10.1007/978-1-61779-173-4_8" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. Xu;citation_author=W. Wei;citation_author=J. Gagneur;citation_author=F. Perocchi;citation_author=S. Clauder-Münster;citation_title=Bidirectional promoters generate pervasive transcription in yeast.;citation_pages=1033-1037;citation_volume=457;citation_year=2009;citation_issue=7232;citation_pmid=19169243;citation_doi=10.1038/nature07728" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=O. K. Yoon;citation_author=R. B. Brem;citation_title=Noncanonical transcript forms in yeast and their regulation during environmental stress;citation_volume=16;citation_year=2010;citation_pmid=20421314;citation_doi=10.1261/rna.2038810" />
<meta name="citation_reference" content="citation_journal_title=Curr. Op. Genet.;citation_author=Z. Zhang;citation_author=F. Dietrich;citation_title=Identification and characterization of upstream open reading frames (uORF) in the 5′ untranslated regions (UTR) of genes in Saccharomyces cerevisiae;citation_pages=77-87;citation_volume=48;citation_year=2005;citation_doi=10.1007/s00294-005-0001-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="twitter:description" content="To understand the diversity of transcripts in yeast ( Saccharomyces cerevisiae ) we analyzed the transcriptional landscapes for cells grown under 18 different environmental conditions. Each sample was analyzed using RNA-sequencing, and a total of 670,446,084 uniquely mapped reads and 377,263 poly-adenylated end tags were produced. Consistent with previous studies, we find that the majority of yeast genes are expressed under one or more different conditions. By directly comparing the 5′ and 3′ ends of the transcribed regions, we find extensive differences in transcript ends across many conditions, especially those of stationary phase, growth in grape juice, and salt stimulation, suggesting differential choice of transcription start and stop sites is pervasive in yeast. Relative to the exponential growth condition ( i.e. , YPAD), transcripts differing at the 5′ ends and 3′ ends are predicted to differ in their annotated start codon in 21 genes and their annotated stop codon in 63 genes. Many (431) upstream open reading frames (uORFs) are found in alternate 5′ ends and are significantly enriched in transcripts produced during the salt response. Mutational analysis of five genes with uORFs revealed that two sets of uORFs increase the expression of a reporter construct, indicating a role in activation which had not been reported previously, whereas two other uORFs decreased expression. In addition, RNA binding protein motifs are statistically enriched for alternate ends under many conditions. Overall, these results demonstrate enormous diversity of transcript ends, and that this heterogeneity is regulated under different environmental conditions. Moreover, transcript end diversity has important biological implications for the regulation of gene expression. In addition, our data also serve as a valuable resource for the scientific community." />
<meta name="og-title" property="og:title" content="Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/2/343" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="To understand the diversity of transcripts in yeast ( Saccharomyces cerevisiae ) we analyzed the transcriptional landscapes for cells grown under 18 different environmental conditions. Each sample was analyzed using RNA-sequencing, and a total of 670,446,084 uniquely mapped reads and 377,263 poly-adenylated end tags were produced. Consistent with previous studies, we find that the majority of yeast genes are expressed under one or more different conditions. By directly comparing the 5′ and 3′ ends of the transcribed regions, we find extensive differences in transcript ends across many conditions, especially those of stationary phase, growth in grape juice, and salt stimulation, suggesting differential choice of transcription start and stop sites is pervasive in yeast. Relative to the exponential growth condition ( i.e. , YPAD), transcripts differing at the 5′ ends and 3′ ends are predicted to differ in their annotated start codon in 21 genes and their annotated stop codon in 63 genes. Many (431) upstream open reading frames (uORFs) are found in alternate 5′ ends and are significantly enriched in transcripts produced during the salt response. Mutational analysis of five genes with uORFs revealed that two sets of uORFs increase the expression of a reporter construct, indicating a role in activation which had not been reported previously, whereas two other uORFs decreased expression. In addition, RNA binding protein motifs are statistically enriched for alternate ends under many conditions. Overall, these results demonstrate enormous diversity of transcript ends, and that this heterogeneity is regulated under different environmental conditions. Moreover, transcript end diversity has important biological implications for the regulation of gene expression. In addition, our data also serve as a valuable resource for the scientific community." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
    <title>Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"T2pU6hUyKYO6OKFHrf8NkO5D-aQyadVdc4LOSrk6Nb8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459413","apath":"\/ggg\/3\/2\/343.atom","pisa":"ggg;3\/2\/343","ac":{"\/ggg\/3\/2\/343.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/2\/343.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/2\/343"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/2\/343"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/2\/343.full":true,"\/content\/3\/2\/343":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022o\\\\\\\/TPJLWhK2CVV9dHHpwd\\\\\\\/cNs\\\\\\\/BlcKvvEyYRpFi4IFs5PHzObD+Yw1whtw1KbVq9Ggtstskev\\\\\\\/QbH1TRfkGUOqIZJQHVC1pidYrrzULPAJ2IduIu+OrbKWq6lvG+aCSoQ80BKz5QbCGHwjDE0zI8PAsy1Np3+A2BGVpGl8g3bKAyvIngxGCNmtJvi8817jw4pfZ6LcIczGXTM\\\\\\\/dFpFsRCC4ojaG+Kb49i5h5DZLQ6fkxOv2oD4pqmOgOQ0BHGT4zRx+wL0eSHSK83yIb19V3aiql1ioICltpnWP3GpO5VAE9Br9G2LTXGPSGvHifl5MqRRxFbyBHC7M2xb+83+2lgX6UyEG\\\\\\\/nbbwZVGGEC5bQbdINojOxvken9RWUDfk96YBsHBj+d4OngNtrTb5VjXSqhwh4vmenHogw75As5CFZhCmK\\\\\\\/J3JOnYo3jcAUvkqKZLDaJwYtQwkOG0jNBqcilfN6y6nCwKKxOMSLEq85BMEMUAJL5wXwOXf6qBVnGpyYooGU7QkvCxZaHMDTvbtx9gVj91fQuwjEEsu89hK9gedMp6vTx8nbQ173TEPWcwa9HVNGgWY\\\\\\\/iIDPsTxfSOTqhW+BAHkD3Ga3lOPKnyU3ddv01lwLTpXNEu+mT+MWuQQ4WY56eQY3EJrKACKGNDKqbFTAWnMr9isa1HqPksKRJ0XGX8qmaZ4YJNPWHzWJk2hGhWwd+mivDyl2OBG6dfGjAT4bhd8b2n55FvFdUoPqLsT9Nk2AasgOpqA44NCeOsNjLZLUAZZAE3wNThOWMRUUfxBASjLYKfhtpBqGjAbVOAayX2pWBd\\\\\\\/DbRcKspvCroZFlEPMnpktolDfjF\\\\\\\/mc9wb+C8U79ihHCJ9dJc4RVUo0qKvWnxfdb1d1XM4H20J2OP\\\\\\\/8ZQZpidH5hoX6Y6oMtiqdfOfdAKAoHuQgVo1nJooH5gzhHNiv8wJskINAHaWg+hkXqAPMXkjzAV4QmCAna2KxgyW6avSHxkcshtziZ8dqFafDZhyLTknIUOMcPpehVLXp3ZGz7todu2TnnDoqyascaiw+tj4sId1zsCk8p\\\\\\\/kx0AKzpcg5XdGutDiR82ZyM1aBysJZIJm72Vs\\\\\\\/zhe17qr3pnVMAkCv1L1mBoX972PUw5wsOtvEvmLvUOKuFpmJBXHq3eO1rqHM8VMDYrKe9IoB\\\\\\\/v6A8tORw76WUOoX\\\\\\\/pKuyOMkgJdps2ih36RL5pbG3cu0G5Hy59zK075k3t5FA+sAAsVx4eRWim+236bwembLANzpfBAbOJC+Y0LM3ZJldpKCB7\\\u0022,\\\u0022iv\\\u0022:\\\u0022UtY3BWqhLKBD2\\\\\\\/re0AOUvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225a4f3dc8d3370d5c1cc9837e491710595b7853e2d643b66a227698b8c58b413d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221cpMWnByEMuicmN7bEMacSWUdqAPSLSTWsRHM1Vt\\\\\\\/BWCVCDvCV4Z7hoDZ9Gmy7tlku1wS\\\\\\\/k4Rdvad71w7vYNOfS8k7XcuICpttz0GN3sTjlN6kL6AwndEvn\\\\\\\/ofI1r4C6wzHFJ7NgS\\\\\\\/qoMnMXsfnHGPBzt2cN73zufg5tT9MJMY7Wwg\\\\\\\/ujNjKcbtg5OGWTTtkW6t9u7B1eXUqUXpaL8ZtEKIGikKi4z5iGrbd\\\\\\\/k505o04\\\\\\\/IfcazK\\\\\\\/kjvXB7tQusbCLZSunO5NjVOhryNGa9PPEnSTKRXARiQb9f4\\\\\\\/821c9na7HniDdkUs2joWaN23PFCxjvKizqvfLtfghPSAy0WaI\\\\\\\/n0lK24WMR7Ck4kzcrfHkfrj5sihT4JpCV304IvO\\\\\\\/qvRQCKDpWql+2PWEpq+zrRsK2GA0pZcwqrmBtEQlz9Q1RAbAVn\\\\\\\/l096OmRgeDpjf2B9F8wqGcFM5WmHtAKZ6zZ64Yev73EGQECaftnOAcZGkO6NOOLi7u5IW2BTS5h5tBwL8Zg+ptTRVZNX+xJdBZ+ciPEDr3poHNIP38jMRgfXp9KcbvI7wPmtMJ3aPYcCYIzzMvpjeNl0bt8Ptg\\\\\\\/\\\\\\\/tzBhqqsAQAItqJk6EJopFQ3jDhmBtrc6gHQkeV2U9JyDf4ty73eppZb5mCpRbg31LvZEMCKkTS8wPsbHbe+EBTHMBBySYtTXjQ+wTzxCh1rHC4guwvCO2y0c0E8sMmcgz7htXg1uyC9hmzmbTy8m5RPAMa+HIUXdrEU0xINi++xvOb1ZKtzpd4T7lt8tG8M3T5ygKdHktDPDbKcl8GArf57QDjfrmfhTSuMMgV1dKkm\\\\\\\/7Z\\\\\\\/gGR9AIq\\\\\\\/05\\\\\\\/7gJ5JXjcEi0qLo\\\\\\\/XNZ\\\\\\\/aGqrZ\\\\\\\/BSbIkE9YhOMaWfBjvZi6GgKoAm4nKZSLq4atFd9wsqeq2OkzDwUMOtkr3NRoZaCiC3+q4aKbcbWL2ZvDvn0tmHc8U09WB1wu7wsfIv5yJg3RZFXj2j7y2Nc8syyym5T0Io4I5n1SE+QEcAfb\\\\\\\/zO9rAhhGYIvkFF241l5V9JtY4dBdYkL2B42cTe1ldnDBjv7+vjMCBOgdPsjWhPq04H9stcYv6JO70o4yeHPS\\\\\\\/TU5t8lQqOrjjdCXbVCTjze48RSX5eQxsk2ZMvVl\\\\\\\/z\\\\\\\/CnhMYaLuTW58\\\\\\\/I8bui9M4yeKhsmQmlsFi8VqRSw1O5G9GFqYrfsDDrWbtMwmMbcnC3y9lV8yAmqRzAq0FhsVmW5VCnkwTl\\\\\\\/\\\\\\\/CSEtcmtiYdeTn8ItDR4VrHfMj+v3dk5kIOAFRF1f6msQnWfVWioQW+xDH8X9lgI6X7Gz1BAf2JCweqQ3tN4JQNPd+QIvhFHl6I0EsLWGL2isJlQFFFSZ7pFMxTTzu5e5iOPN3etvNv+M35tCa4XnqTRofIe8disPA8r0YVX0rFfkgT18+AVRk7X\\\\\\\/Axg7a6ksaMvMFKTCLLgT6SHp6bS7cIn925affHYOxspeVcDnmA16N1eXyLab8QVLoiqeV+5Yo2GrFpSTTiAbwu50c5X7n14p2EUnFO5qGiXQ1635Yf0eANRZh0sVkjzi1ad02UkBohLf0tNLe5Gjb0eeIZmtvYKn1Y56NPp2hugnL2ih\\\\\\\/HRaBf4l4u7BrALraYyYzmC4KpX4CP4Qt0jaSVR1lGDOA\\\\\\\/YvKKKq3dhs+g1ITlPbXPK7Feib\\\\\\\/a82zRVgtb\\\\\\\/WQ7uu3IMT7UUkZDJpdwvjacO4DkE+H9OreVnTIaVN2+r9io1ap32BhgcLFBFef9UOWIjLDHF00o9uYscTHCdcpZg4hX\\\\\\\/4OenRVHXJtWzwLoAXDtaJyYctXr5A+uu5Vh0fhA32mCyvWE6lOdTkyzDwRKdia7Bkj4CGbFDjW39j34dN2U6T0JSewQERMowqHI\\\\\\\/RCDKs8fePG25KjmaxA8DBxwvtY+JQTJHJDeOVeBCIlsb2Kh49ctDLRniOBJpd3rHPjFwXkReio0u09yEuDsgh8DXNpkJTV\\\\\\\/w4\\\\\\\/F9AbWRuHSJz\\\\\\\/5RTfWp+1Zt7W6iSYp4vxa6zOKw3+t\\\\\\\/YAiQ7eAJ683gaybASTBhQ2DvUzstCKLBEMSv35hhLQ0sh\\\\\\\/oaJzfYcg4aXh\\\\\\\/lnAUMgskx+BjghUg5QRTItg7fI55LOiwVISj+6kq4aX9BSGdY59P0hGF7DyQ9i39mXcQmCH4KmTMck4JaW1xnTtvezEpDQDfJ5HhM6US5fx\\\\\\\/z+m9aGPi8dWJkWd6psXx\\\\\\\/gGDntHHDhjAi\\\\\\\/2t1pGIIDYlJpEV+\\\\\\\/78R1yLFIKY8TsnkLVuEorQwVsxsnvvFaToMJcGN+gxlHc9Ap4zjjbbJAEHPM9UREIDltBpBwaeewCpTL2zbEEYBqN9B\\\\\\\/yCYSXMFmkDqyhG3xSQP\\\\\\\/\\\\\\\/+ZgNBbjT4neqffcRmKqwSLkTlW6Eu0zDDzsDwp2fEfzUpAkewwfZfeip0hYtf1tohVrg\\\\\\\/ahwTtSAgcKNJl1+fUf4AY7CvBhrFukRPvCUPkPaD8AyfSP+Yi4WQN\\\\\\\/evYw3jheJB6KrVBFkySR2U3hPakOS7O6i3n79h+VoboWGcoCJUw\\\\\\\/7YJzwA79xb5I6q0uAEKeISR5+tdgEL5QeBkmWY+IpYnw+ib1\\\\\\\/MT15uStogTq1lbDENuCgjsT0utx6S9Xb93iL1+1KYWnb5AEM+ADUqG3mcziW8XrqjeNQTyhK4iAUP1Bwh9XRw+v5ssrLUqDu5\\\\\\\/DX9kuhLJjCF3+BtN5LbIWettvrLYXUtSUMNN9gVuxQdcwu00bMqM6B62AhB\\\\\\\/SBTadurPFFmY2Y+h44VbFfcf8qyESJjR1jftYJ4YbBwfzoVlRUK8fcb6gSkUYzmkE9MFZLpBJjS3ZAYh8R2VaHp+2Wk6D\\\\\\\/SXlhtjybJH0\\\\\\\/aK31+QRnXaNd77KT36019CY3Gr9l2m0Bw+vU27r\\\\\\\/BWmMfcXvVCwVNPdq5Gx8U\\\\\\\/6ghqdWgOAOGkXSAslo46Fnm19YxhQ15Aolh\\\\\\\/OVFVQ+1gOtsmqEi8WCQvziryNHNrU7o7FKtOLcmTd3OvvjuXXmc9MnfXU84ct9x\\\\\\\/nNoU0s\\\\\\\/aTjGruZj9iv8LDnFcQK349NRvvhwxDdkwVLsAmcPbg+9aOFUcdco3foc7AXrE6Q60bAns72hXwinp8Qvnq+\\\\\\\/S18FFtynGPeTEm4PxpDSKKwPM+IxRvlyF7dPaiseUaMiY63i7OK5fA227ureUPzOLjzIk2O\\\\\\\/Bldrm1RD4soOkZO+aLTgn7Ar48G48l6EpwqTJcjYaKVfb1KpcaBWto+1YjfZSNberxUT5lIjbXnWeTGSpW6yrK9+hDMWbhLwhsajUK2g\\\\\\\/PjDfvgb89M3CMgG6g3fpJYtt6rt+DNvLo5TuS9Xq4x6fR3euMaNb9GzMrVHw6PUvdchHRyGL\\\\\\\/rK4q2ni3+gR5pM+acdHCzWAyxfhoSe6bz4lrNuguAMzY87\\\\\\\/vniUv98k04mYbf4o4PyCzeKG+t6Xw6iaNzl8+Zymgg2GR6uv+eCi05cse9Mx+T8KHwPYf7jDZ2W189kS87ezup1aM33cwm8BWudYtnG9WYiDfk2Hs\\\\\\\/cRmIxnG0iv9n6VT57vCHDvrA2cZy0JhOXzUsRIML9BNrGuUj5o4ZNsBoAfY9kHteTiEf20rMNk3wxBNYVG8D24XUiW2eAum7FnX8CJE76vbjwrjT6oje0GMYSOwMCybIGYCqdtW5ug53VwWErj9OQ67uXE7fzfFxHgUqCJ0kS5l6ngtsunGV6eSPrm+QlMTaqTYruwDUXKByCXDcJ8wFcs8ijFJvCiavehpP0wXC1e2hVv9ZE1bVts3FVxPfsr7kJO9L46CHUBRDEJPrV\\\\\\\/W5w0bFUpBeJDpZaL+dp+Gw\\\\\\\/eA7lllz5hx6fHPKzYnTbsmDOQMDu3ibaPmIyjlSVUprb8PmSUvvJ+ULKjzF4uaMtg8\\\\\\\/1qZ87EMAEf8vlWG52ODZVHty2AYdXbHZinUOiPJKbKtOCmYfv1xg5QEp5bemmtqoQzFTCo8NItUf8Ncu0K4yPaYhXWOR1bWfJcd6qm63EEsYtjOJ61fc++bIJABW7jamDtwLBWOqM5uWoK5TOcyxvd8mTRR0GLo73\\\\\\\/hCU+HwO6\\\\\\\/d3WS7QKxTfvHj8X7iHvZdirGNlpc+Rn3GRRZuuM2STz7\\\\\\\/\\\\\\\/79nhiNJtocYloBcn7efZCOW+X3mBHW0CIeJWBbhe50WydwrHigCLW3aoXi8S\\\\\\\/LY2h895+VvffORx3\\\\\\\/mWuNsB60U9KK5+56qtxv4REsg7JgPQbib\\\\\\\/MnfQA4JTiZmy2WeAItS4aUspFDnYSnCv2kOr7f0j33DpyzkTZ7hrCIcB0AXLK6bPRNo\\\\\\\/vA86+1T61nzHEHfSWHTZ0lrb6Zb\\\\\\\/lg2o91O7hGp8CVAkjqTllA1jxtr\\\\\\\/HwS+fKl6F2Jb0UblAZLrB4i7pfBzorJ4FMUF3JY6TgWTigbCyK2z3s1Hbc4Cg2cCaTnB6BQMsxe+x7IUhnbVcJz9rzXVKpAtvdRt9dq0EwtDhSZqW7pR8cG1EUuyIO72lwGNfh6ihEHdszGz+SDK+Fp6wxC2Od2XggBojSSRnVvJTB6jCgiJm0llmsdmxxoVtRrbYQAzCqZMgo4PoIeZHAYMGtlhK6\\\\\\\/3FvzoSJxknaHnEFMr3xyedak2SkDMCyTjcLf\\\\\\\/YHgCKDgsSaWRb77fKBew2TZ+m8CWcGMAN+oQboS3dX9\\\\\\\/E\\\\\\\/0eHMg9sZbOX6sXqd13M4Lm+TWw5RBFnlDI+mm\\\\\\\/u4ONk083MS7KVclRM0yZ5USUzn5v7kLgRBQxgvItsHYssYlZCJIDnlQx7aS6Pq8wpGDi5eUTGDIWLb68Cu3BLaGCCm5Wg9w8dMz4sokGJhga1mTVY5Lp8NY19ckS1frK2JPOqXKJ3iTpSAkcUQ04Wl4Qy7KisCH+dTK90Sfl3UaV3xqX4lUdx+pWtPqcFduq7pCMY8wlzEh6FWQseRe\\\\\\\/cWjkN0IzEaJrPzT6gX2JrJAm+QDQdKoFibXV9Fl7N0BviyxCdUylR\\\\\\\/lr7hb2Z56WI\\\\\\\/U3mFvlr+FFxoePYh6wObcCxygLzQUc2HYjYqWeL6024Vc4jXSybVnA1NL1EgxyG3I7s+LQgUlYG9xbjbltbuiB3YM9spRi6dK1+7S4DKH9mWu1VcbOz9DMBApLDN9NEKedmNlUvXzds8xSJFSIpc1TnRrDuAF9czQYmIeWzu7r0Vjfb0Tpu3rfsNlRBpqQjorIDUxN88GfQFzNvA6o4uQyJXKMY4erztG4uL1Hue7WOobIyr2jQIju6DYddsWULul0Wv2sNL2G67\\\\\\\/1HQgxWsX1fdqNkqfcKPV\\\\\\\/gaGbv4kDpRXQYeW4pXHG3s8kVNjwyfHk5XsIb4U3dM5jJlNqq30qP4kOiHpABpja+8+HLmQnGUVfsSPix88EsgoY4Qvxy9tir4lNpbvur4Zvbb\\\\\\\/jedwfgtg1EIAiuiQ4cu62iysNoAafUJ7KJ6oMtdDq1h0307LpEgWgthxbCj1Fv\\\\\\\/WGZXHXXc6RqkIYnmfvoqxIzk0Qf3+ZxwS+yAjSXs4YyZFmqUPBUAOS0aBpul7cTScncX\\\\\\\/x48Y1H8xc2FHcJ3SMNwYt50cBtifD552Ya\\\\\\\/R326rzwlZwXYNZ9t6RMujwmLqUfFbPUQAO5c7dJFlmRZIb9A3ouS86yiyk3GO68D8oSGBCW3iiOaMLD4arN316wDY0vjxa9MKclkeBsEfnxmsPIvqYRKq9jOEpAhQaQumHZRmxo7q48mLFtvKEI0gmObcblBsJR7gxmVJQ9RpfkUzS2SMqDHucpjf2X1sDzuDzDv8eq2bbKRH3seRQhKShuyEV4Fr0F3zXfT\\\\\\\/nBjaVMJW7mJgmt6rP1cn6Ej7s\\\\\\\/nFmxg0+bSMHNXvht1IWsWpmZq5estwlNc0r8R2zLf4uyCG\\\\\\\/IhO4oeJbxyGgrkUnaJdU5H9CjEg7\\\\\\\/MJOg\\\\\\\/DgaGqYqLmVUsid38xc+66wlLrNISZ3POZJAhnmEwFlb2nEP0I+2XflQ1TZrg9dkOa1DMtM3w9paa6DATXedJQCOaZ3f7iHzODWwviR4IgqI2b0aixPxLqgbGD89HtCMSceCMGlHZNMIN79ltx2FVhFEL8MCnYGnLB\\\\\\\/0pgBP2n8WxqNYMhE1lC6locfRYR8CQKRhf0ga+SR88o9eINUX7E\\\\\\\/tT\\\\\\\/vBwggYD6OZe6PHzzxJ+aI1dQyVV4\\\\\\\/lEfg8vAJZjcuDfvGypJ+NwRwwQ1CR5Yk6spRQV9wdvWPd60NxdbrvbymEKX8g1DW3GDj81wonL7Wamy0tP8EZs5d\\\\\\\/kiO\\\\\\\/YYEfPyjfE0ynGC3+ANbpDEfn7DOP1dsR2BPVQy0sNQfReMktu8XA0Er3UiLwa5UTdmwriHfaokiGOowVcxF\\\\\\\/74sWP\\\\\\\/HBjECRIMxt4QquffzbZ6B8HBlvsjL1VLuCJqMB6yOCf3fpYrWTPp1sFrauXnlepd40wZ2cjOp2yZEUzL+hKBBs566oTfehjBgHcQUQ7E8vG6FvMAj5vEKP8oAv4bof4GqYj3juyUyz75ppnfbJdMka1ABHUbzAhnQmqjHO+0Vl3sQGahFisAID3\\\\\\\/KlQOo4qmL0YRU\\\\\\\/J0rVQICQ0LPvvMO3VUlQAX5iMM4n05TzknFQfwBSm5hj8YCxMF2XKSL7NEcWtKwfUVhQ\\\\\\\/WyEO3\\\\\\\/F9ZJZwXDCEPCSCr47NBDl0WAJ2BQ6tbuqyaSpeb4TCyD+fBSNkorzm5kQ2X87KnP\\\\\\\/UZn7RTRMhKWBFbsik0s9BYqnSZ1zPpcSv99mQU8ooQdnhL8Mc273ej8PiYcvpWYyxx6tBU\\\\\\\/8D6tBmMGdJSWfw+dhWNuFpfG8vROkwIrlJHlfobNdaOIbIK8i+z\\\\\\\/2lRO1UIidNpr\\\\\\\/BgS8ignNsgAuvSZYsX3MUD\\\\\\\/b29uQnaFCFK8EzIPfVcMu0j2V\\\\\\\/hg5e+3QJC05FHES6EcCRtbX++Y08MWxKJACiNWvV5nOG\\\\\\\/1QINJrbGqDryF0+KvW3OLxup3JxmejUfcZeg5d8QavpQOTNH6plh4x8GFZCZi9gHiL7dnKREpJ4dCJCAfSTrrDlS66lh5qGj4rt9Z9yX6DznlSWBtEVk5nPRW8deJGKft2x3rR\\\\\\\/VD742X9fQwaRhrNQW3UDIa11FYC4oIkajwRBcKkd\\\\\\\/B0J8PjHctg9\\\\\\\/NgGUBQduCC\\\\\\\/T\\\\\\\/XQ6WjY6pLRLA\\\\\\\/XBrF7DA9O\\\\\\\/Iqc83sduGslw71+aGPhsVm3RphCU6EkV8lkomKRauS3XqS0Ud6+ni2ZHx8vsevmpdAWVdHLH\\\\\\\/BdXDTypm4A\\\\\\\/nljPuVAM+5dVmGr5GMLDgrY67a73SG2tTIrOb2Atx4SIzl381qw\\\\\\\/qgYoRPl7lEnn7KDRKPzbPBpkbBOpbRmJwW8YzIlh7oR3jaFbGJNRGZmKa9bMAn5CUHxBa2yvZ1ZcqnmLUI4BqhC339zMp+IrU4\\\\\\\/qH6ewM18dnL+nDKOJcfgpGd8iX7R98h+RQRwJ3l\\\\\\\/Q3JtCKU87chPke\\\\\\\/nsWrdo1utJwassxERzS71zAw4YLSQa6p9Ts8NWQvz1O8HgviqUK0vCS6O5K5yCLOkn0cw41Ti8gnrixc4P472FQvGPXEsFCVV6JEeeHtwe0Csh+Mjz1fXs3MQ7kING+LUOYLPtkawi1V0yAyrGeqz4SnvX2CU\\\\\\\/PZZvcXXI2LK1ObbBdq8Nb9RGrOFkWEOLu4hHbdtYFznsfm1qQAbMNM5rjNrbEMnoU0KyEAzuB4AdUrXfHnI\\\\\\\/w\\\\\\\/xqSVi1OqktwyvBtVB8KwizFyWDwpyZypzA168riKQm8izUZZdAqYBP1U\\\\\\\/zW234Efjl6hRlnacgKYCMmv7K4MDIlZBZHLyxXe\\\\\\\/03W1v9ooA0fvIyjqX+eDwD6nYAuHTrJZUEY9gNC4RIRPgwS0TaQek0YNUZTPa3dOhf6Ld3WQI0Pomc6pn9SFXswxaTwoKeyq0ZYML716RIaR27SQwMmPdatBozgOFZd+7an9K3gLTSGXbDZDRpmZ9px3nFQCgVgN1C6CeqdKZDfB7gRQfegfjwYjxnI+plnYg8T10H1QdLlH9GSgaHH23yuEds48fAttYBlvHf\\\\\\\/Hst5GKQYhqcDb18iEwmDQ1Npg6up5GTwGvU+shb\\\\\\\/NL98vBm1jqY3DO5Cgnf4wCw+PqLWedw97j7zQ2Xp5b7GSYPurZ2desG8iSFpL\\\\\\\/I24Mjs4kKxSQ8sHb3SufP54Gvw\\\\\\\/v1TXGYcnp\\\\\\\/WAK6DFKv3FQtyFE7rvFoPQMbnL5DUMRvZ7wP3KhbURlMcrAwf7N\\\\\\\/2fs3v\\\\\\\/I2HDqWOqClpi9+u4+Jzlzy3KR3I0A2QIL8+dL4nIon0zIR\\\\\\\/Def\\\\\\\/40WywWTWIHEA52xVUdRXzkgYuZUyc4tfH0\\\\\\\/LJbrkLroxemOmsDKG5g3BWUZT+FJ1uhupFWcdJEB3phaok1f2erPmE5M5jDb2Y8YUUSaL\\\\\\\/oJUgFNz2N0fVWBrwafGV6HFBarPl0uIqUyJ4p+aDTPCk\\\\\\\/RnazgalM6Oid58bXRSl8h3FZxUxIxVeALzHO1DzGL33JZFBPFAklxXnLCfbJjdfIgrezqzO9KAEEqZ+AGyzD9BWnkHnvhQ0a4LiHXg1ZTlS50If2JEWy+B1bolh\\\\\\\/jpXLLUnZjoP29Tj8ql7H53YcuFYPZBWN9pXnbPuq82jq+3v0WfTk0YTaqk2DVDOIJMAr9F8x2oFjG\\\\\\\/NysEAnnm+XcYDYYT3E85t\\\\\\\/Xrs+EYnE\\\\\\\/A1GUeNFI3HveoI2w9K\\\\\\\/21ZZ8i4zPGi\\\\\\\/uHbfUd5zwztZLrzUMeJUaUKe18bCMuJxf9qA4NHSkEWjYz5kpi6mebRDoqn7UEpM0y1bHfUzgk1Rb04yMzFcfGVhjTcnpVH3li4rv2r5E9L+psJINlFzqTd5eMp4XZkwWPVwKpItczwtSPhplBwzg0oS3uMfXtexljfNjFsReyVT3c0rwkGE7D4bilxbNyu4WhrQZXiWZ+tmaG82yztetZ+Un6Se9T4QgOhD+k4b3J5kpI3x\\\\\\\/vP1rXB9f1ZhFzABtGsnjdoSeTf0Lpy8xkR6UuQjgFTyww16ltd9ASyPqceUeLBjP1rn\\\\\\\/X0wbCZnLgKX8RlqpCozSrElPRKCU\\\\\\\/sPzrlFJA4HL9zBCHB5Mnq62DSrODG8Udd31g78m0ryBgEylJelvyBA2Fc8KEyi1wIbytDfPaft5mLFjD\\\\\\\/Q+KCjA3Vem+WtB5Fzc38ISwMUTzCJz+oWXUAntd4vjSgvWrTfnDA5ObXGECgMV65oVbt1T8ImRmTyaUN6vIwnlnokjIe4w2eWP7+05f6K71n5I6oy5tBwt4zynEnHSAIItfFup1pcC5cBJ+DuYqgg7rU991BTWMaW06EziWR1utxp1pQ2V78uLmePnScEQA+49j29p8YO3AwaJDBCyQoxqTSY3n5fYhz+ROXgweTNgFOjGmoCvqtKenl1iacbfmfcnPm7MIPhAqup7M++VTX2IuqscAsCA+DO5iacHCdqmdDo6i06OZP2EtsB2hEktMfBqPzdeRAm6tfaLt6Js1ZMpat2zR+bz\\\\\\\/4+Mmv5bJexnr4CdaeyLUOFSWCCCoXLdTBTW\\\\\\\/jl3UtEGL1n1MEBxkcDlLC8YqUx5iFOGIeDIFLxk3nYT0pSlj+vwgTqgkjydS8KimXHKFRzu+jv4yoVETSVTsnp2NWUtI\\\\\\\/dFHeg0eTymDV9xJxQzPMYZBzbb35oFy++0yBc824iWatoOMPvFqije5sVHPNfBmeBWKuLEUlYRG5OoK97whvtPtyYmqfPzvsIco17hTfCUFMG2ZGpSWszVP8D2fpecKYcJ13agrAF\\\\\\\/PwPFQEtV2s9dP2ViS1hicptI26nEMMoshJ28qok6xH18cdc4BCIHU8uVWyE1HE1GW97uncaiUfYMQTdaknJY1lDo413by23FzWc6\\\\\\\/2kyWTR3uFanphN261Y5rEt2AtjURDr2x2dhzGDfCX0y2WoDDpgSXf+9rIoymaICoI6oGpcPksbbhBPluIju24somtHyTmEjFyiIJKIvO3AqoW46\\\\\\\/7was478ZfVldU2JzsQFNzzbRVfzmCSh\\\\\\\/gvJdOegXVX2tHgbFIxq9xhPmucgKupom8tK+i2kRV69X0UZMxtXsW8Tln\\\\\\\/+LeL2yJ\\\\\\\/dr0mZN8ACoh\\\\\\\/Thb1RHiduLzrmXQPPVVRfhfbs5Rr0P+LLsvLlYKQRVMRxu9V0W+nILxtmHjPpeRcY2nlAXehXTcPJH3J0t1s5AVgxtgoAnMMOaty1IOPm78pUJ2+MJ\\\\\\\/1LtuM4cAqXrm5UhA\\\\\\\/xukG1H0YXUuKEySV+Wy9HjxsBWVtaMTtLf4KnmO2avULJokP74tOOpyJmEuLWtn3mQIr8lLrZjtNiXwMNNFA1F59HmQQYinP1WGX4ZeDBRQvREGw+mv\\\\\\\/fweOfocqQFtU7rZnikVWRzbVJwHNqt52XyzOXD93d3VqPOWGXh1hFlUBIcrhj779nFTaZY6QAtJuvcr53x5Bw6bv0Gxi6BNcYTXqn7suSk4ImRk\\\\\\\/W+hi12B2xixGNw+JQaLPY6IH6LHaeJod+ou6mNcXCdndODK4v2romJYdSxopAjqFHNI\\\\\\\/\\\\\\\/q1lexEMfmRfJ4jrKQFdJQdi1atSAaIOUZn0PvdAX4WA2yYamA88qhGF2b7sm2\\\\\\\/D9lUg4oEVFZsybQ5NzxRXUrt4Gw+hUX12oVFmxSXkq3wQ6E8kg9Pg\\\\\\\/Y3F06e7qFUMcHMD45k\\\\\\\/QfEc0M1k4SDOrcQxgZTELncUwhTj\\\\\\\/xuEDoumxRLTIo8PPg\\\\\\\/2K3m3B2PU4Sez\\\\\\\/99aHMEiFaZipWYT4x4TkEonZSN4IPIPXA\\\\\\\/s3eV\\\\\\\/qVL\\\\\\\/avQMI4zUt+VJGfDduDZlDOA7dArtJFAAbjp4\\\\\\\/pPsiWaQJHFwPfWIBkKHogV9Gjc1g02\\\\\\\/kidaDCG40fs\\\\\\\/PqYkAlTbYXwjTMZSvuPKEornTJ44+f15AcKqkUdZZbDjtxA9LvyKvS\\\\\\\/wIDEjzR6fDBFaYVUDSGzvCcZnj6qWCckpTRnN63i+Qc8va8QlFSDGE6nMnjMoYaN2vUxyFQC5ZTYGfJjRAe9i+ICtTiw5WWBQdTknvOyjeAq7vMjXf+Y86OPHarKXxKf4d95Exrx+3Bpdj9iYXXXUP8kWORB8jMqMq\\\\\\\/cmceqXJvkwu85+CwLQIssXTXin8bXGcjY+gB3c5jQBiz\\\\\\\/X+zIIUxHwrSPP2plg9z6Rbn2G1QbyJz27Ezr5f8xiUH4m3TrUNaS9oWQ6mSUeFRsdi3JOZ6tNin7H40zg3ZP5WhlA8revCv\\\\\\\/9rLd\\\\\\\/zcEW2\\\\\\\/n2AKS62kG2hvSaFlQD9DICZh60SThC1QieZm71Cd+96SE+yoQ66hJUpx4XPuzq0VM8G0fojXC7mpRuCyQWfrQXndT5I+EYdFckyW\\\\\\\/kj31w40QlF4yqaz7yvY9Z9hCN1DRkqINbIIdwIN0rJCNJJdLvPXKgvseIuTFNKahQXhdWKchSNm3Jbydx7\\\\\\\/iXxOTOzuLSTHuHamg5u+wPzaHkHnBICNt0FrJkxmwkgX2pMijUmu9i2+pFXbqEWMkruk0H\\\\\\\/WoCIjDq\\\\\\\/K7s2h\\\\\\\/kU1xVRKnFHzMTXr3cTCcFSPNkAkrM8Nrkifu9FDq0kO\\\\\\\/HMuu4LRECoi08hUYlzGm\\\\\\\/oR6McED+jXoDYnfv+qps1ANDGO4bzyfUTnRo7Y7wV\\\\\\\/MJDqpkSsfIT9KQ4i9q+7Lzhprm4CyqEYjGU2M8Bf8ELp8uflzDmqYa67zXuZPqEyD56LPHYz+sz\\\\\\\/+MIFRnTAv2E2eLygeR9Y7VqMbU4xxmHVIhuWeXx4n7ifzJXbm4XTi6fz8MK485WUt0AG32gYjf8okpS+KuZlhYFszDmMHXCQ6RZkSr\\\\\\\/Gdu2eYRye56PtkbPd3VnvWAb26IbsrD6OwEHQUSZNlhzsvmZml6RBfQDJCWVYaveCa0e4ygZtyYqNde14X+YTcmdYDJFcMdxCySiTQKJwXg\\\\\\\/0ucbR+u5iDGEO8tNCdq0L7Z80svhiHWf83oc3G7ln\\\\\\\/BkKAantPhonIMNBCRlGWe+a4N5dWP6L8angHakIFK2vBbZ+trlYaf8djq9UllYsyXbhhMADY+Vu+bopiHrUIQYDjEa6iOVbsXAEhicN1DoqAlUEgUt094JTw6eU+KMG\\\\\\\/x0htp1szBMxYX+jwckuUjQEH47pL8sbUjsD39+BWEFHsVAsFG+uhX2H90tPAZNljjkCOOFBP4ToBm5W+ct3NQo3yX0q16+UOXRYbycrS5NoDmqPBpMdE6MaqRazmIiejo6\\\\\\\/DCmtw8iU5OA+b5QMXbYDR+drIDlcaw7c4AlxtROHJ9cpPV134b3gBDuYNktMYigtbgOzlP2\\\\\\\/a+OfTyxkHiYHqDjbqAObJYQsSEGm\\\\\\\/BSR1LaaS0eaTY9GRqtlmXoRKVR5LlpVFAysO2Vl4DFq5BP8Nf+z8znjJg0l9YS3Vqgv2YWuSFEe6PpJPlYMUapmlzd7Qeo89fUlBbOxVJk5vwXui+DXSLpN6c9WZNf1XMtFvdaMYyMy9cc0CERV9mrgRp1fusj4deyHzttU9XhFOcs+ZkdxJ+yxRvHcTB2Jeb7K5mItdCMQlV7Qfb5zuP2d1Sj\\\\\\\/lDipCH\\\\\\\/39zJOZWiYHWLTsa4j3ssoAQQ+\\\\\\\/cX7SgvGUDKoUHuAB+huDF35Jj6YfD8FuhZQlfxGi203Jlsvi\\\\\\\/6RhdZcS7iOjQ7zTW8QmudExw3xlRGgxZy++Oa1GAZs0e5w\\\\\\\/AC0pwGlHR+uzfbEMjETEcb8EK\\\\\\\/0cyk1APUs454pwVTWon3sPVTHqELNh4qspPKn86Vd6Akgv5uPRMV9A931gjKNB0R8PTXYVeUA7a9\\\\\\\/9sCFiyxv+0j94Kz+BWb4OgT2huC+obT+NIaPM\\\\\\\/gfoLSGhHv00NdLzaBNa31ErQYKJAhhAMtRJroqt85dDrithw3SaVsAaDyHGFBP2oewseOEQJssy7L9Qzk4JZggLtxVEYucWmN6IWXHdSqQKX4H1pbujyLQ7SDBgNlrrtOAOTxCejtoDU11Ab1+ZtOM7HnDeGaZJhSodoWSm5T1nmIe5vQUTEVpUTJhROTy3AnewnBgrC\\\\\\\/sBNh5F67B5LvM\\\\\\\/P96XOe0w3AyNfFtchDN0VG3Ut5uAWCgh8sPD8wEuTYohttctw8SuOiq6KBqDb1SUI+V3Mgw0nbG5Zny+vXE+m2Kz4dA2dKU8m4MWJMafOJJBPUBxEp1i6DWF2rezW8ru8pr4FAxg9jp5O1+mGOjd5+R\\\\\\\/R\\\\\\\/fkS1gc+A1oj0x6BosH+ezCNfeaUWG7De9UkJAlMctUD2\\\\\\\/Hl9ZiJ3RzEL9BD6JIuCN4+QQxsaUB7fPg52CEK6fPb2zeHXEuFp86YnNUuQWOnw1VtX9hnJo8L3MHyGMptBwsWcxfM2OwbcFkuzvIyJs4x7wsBKCL2u3y9O+zjiMo3Dy62WqvUjSb2fNx\\\\\\\/lX55Fb7Cibbqv84d\\\\\\\/dN0A9bPaFU26LQ7OCGLkX6zy8q0CAcW7sCugplycWichjJSkigY0\\\\\\\/ia1e8IBgoz9TDEUMmrOtVpBhN3+skuYBmBOR9hBozK2Met6yR5aVR0jxMQiK8gbo0QE9+6fCqnKAvyISR4gbxdTtZKdg4j95xvBn4R9pmhy8JKKPvroprkkFhJbl6KO+PASHIfXY1tFiSHBV9U3rGD5M944AmYzr64SXxEEBP1IL5ioexcVqmHMZEocmvTIblAY8aG2fNa5278Q\\\\\\\/LwS6RJ2hrx4TGdi85O705AjpGcqb9EWxN4jfM0mnS+9DzpVJjxe5NHmWuf8wN7ysT7v\\\\\\\/G2a8YTa2UMWoTHjE+gyJriuKaS9fRPwrQqYJMvy\\\\\\\/vm+SR7fDzklcSeR9JsnKV4\\\\\\\/81HUd7m9TB8pPuf7do6e6ciGerklUoZLb9kpLirTSGDZLDzvFRO4QAHxr7WEls2tYwzaG\\\\\\\/52e4nIOG08NvUP5vMkv44qCUs8WqneU3jxQAAocy8NcOau2SZUdjonzvvNrY9mYWgxfgK5fl8+4C309+vXIyAr2XFE1fmIpnRpufbi+bEXGT9PCjCep2P55u3+fnSnPCcV68TmjtJpyeoU2Y5Wzv53lmMEQEs0BDQa8smztVK8kWKd2A7ayCYWFZWLW\\\\\\\/TtPRJDe6OFniUZgxTfPgKuR4J8olschMmg6MNCNgqi+kroyPWqnVEJPVK3+ME+gWb0sstXqoeVfqLAiXgn3eOxav1QXnKjJ9WNFjpDr2OPkdmUb3r\\\\\\\/2\\\\\\\/ua6CBIxxWMod5cnu1lRFI9g9SR8AVEJghx+6Y6BEUDjMMmOlcWXvxLXju1X0R\\\\\\\/o5Oo1Z6k+sEpxTQRBPb5IZdEom\\\\\\\/P7CcI3VAcQagAM+KEmEmUdKANPiC3kM97pVdqlAkysGg9qzhQADjwh+Wr05rMFQpJLEyFXZaqfwbTNQiC6DhlRm+mxZPjYpMEfmQ9uEBtUct9fv7sI90vxhV6TqciqtwNJjcQpV2JqVUAmRVbTkRlvBj8idbHxqZvkiM+KO5Pux\\\\\\\/aRp6BYOt3iHHTIj8u8KNwpQLKDWiw6xrMWHBNWBmpGTMJhbBATf6XlIJwKX1uP7jfU5\\\\\\\/Ua3B3vB700IjLVD41Olnk7rmqYa13vw1XnRi+pnAjvWMBOlkOBamg8Hm7ksA0mVninmOa2Ek8lX7c5m8VuRnOSWHcDurVO2RoR\\\\\\\/yZ9ytHnGBAksnYvDlSNJ8SW6+VygQeLrPMdxH0FBCkGXooUPb0wGadMxt6w\\\\\\\/dBBsi0cInpa76VAsc2q1jNPkug2vd4nXSJojmfpODalvqeTaIZKhdi2Ul6aLuNEPcUVujvlJ5YXPutWABM0+TVdfN17taYA8K7ntiSx8utrpFAzOvEZj6R5YrMIU\\\\\\\/qAYWBWAHxDoFQld9YZQWlq9LqplbxULZKNK3dvfCm8bSXh6QP\\\\\\\/oh41at7OOL2csFl6K\\\\\\\/XR8JKW82YrxqIzfJfiY130fRX1kHH\\\\\\\/5QZXx0BPcWecZFgPMsqRW4eE5ueFh4bJsGLr2gFCuVCBzA+JWbGWM3z3rMw1EiFNisrzGnFV\\\\\\\/CZrSyq87ZqWkprCOb+nEdxn42ZnoeDyVE5WE8FifMtqaD0S14XoTu9DwYfaNXVOfJc6MIbUGYPxeiaZyWVuUmKQO\\\\\\\/TQkbpBEptd5dEMCp4usLV2ZYoRgd1UzWSnj1ti+vgcHnGVFaWNwZVSaoBnK7XEuoE\\\\\\\/Z9C+eKcd3GqZqNITcay7pouI\\\\\\\/Hb3cR3S9uWOcfvQ0Gt+SA2kSV25p5DbMhGadFXXYTM6K0cQOhMbEWlG1h\\\\\\\/WBNm0ta3U\\\\\\\/cAgfkQJ4dJEMbc9+n0fmdFI39gys1tiV4pKa3WAsyRlBSSgJCHuoa\\\\\\\/AYy6\\\\\\\/ryp\\\\\\\/T5i\\\\\\\/j\\\\\\\/snQ5KjEv22awJkCgTjcwOd2WNsw3MVmljhiXcv6XYR7bhD4dZcUbNWqT9RnxLfGkG7t6qHshoiV3A9EZq9KQ5SOT0klXKGMf9GtRM647kWNY3j0dxCB8ABTjz06WdsahTAA3wkbEhjRxQCeLiQVK8VL04b60H092XuyzZmqH0rR\\\\\\\/pVPEIIxFOh07XrsHKKxu9Uf6guXzJCOQtFG85ebkNK6VBRf0qQb4ZUoQjSjUVuk8jeksNp4ByBjbHGH1jAIufrBg6EbGfvvvBE88ockLZD7QQKzvTmpjr6qa4b+nHCKIe07DhKkC5Boyc3QqsPaziawnOJ8p66PoXwfQCuTVJm4LHieYX\\\\\\\/FcCXyWF\\\\\\\/T76J9bBmtviCzZDXHHkZFul0njLIW1ND6djETmmb5nEqZVIg\\\\\\\/6p8VZpeGjMF+0fW03DiKMlFdDIo91CwSNyAkHyV6LWkwVsLNoHzFu2UYbepkkvV1e3ndUMmTIaFGoFzyxC3mW9SrKuFZBq1QMeHid\\\\\\\/AmSBAV5IVF+9Ie8FSWqFFm7qRXtnuduBZMlO2zKkYwWusDjMV9s4aiQHr1Ef4naGsT7I\\\\\\\/fNWfUn5Ancs3EH1dbgkPYTOWMknwzR4E2AFMDMS5jt7sPzhhRNrxht8Y+36JoLBGjlBtcaNWYjyNRKlgGKHpus8P5g5ZLjAxUImmIJMGOv9ib6+IJx2llxFg5p3OwAQg3GFjbbxzAu7kRJ484Kq6xlcxOPjdGYB38bPTd2p3XU9VPJtIUDGDKDd1a0R41e7\\\\\\\/jzk+kcPfsg0Sr1K6UmVNTlDNzTKVn6FyDj3Yy2igM105NIz\\\\\\\/ERspvdnbGnBRwjUAriuPheT8eYm2BBpaSekV5GO1FH+K8kUBt2VQlnZStw1StDDWc3Ge\\\\\\\/HMVgUp2ffla+XmNYNW4hIKN1\\\\\\\/ulMXWqWvu8q7aeWROIZro0pD6VAhfEd\\\\\\\/QFEVjJPTh0TeKCPI9WTHUkhgAG+51Sfnn\\\\\\\/jQbOlQ\\\\\\\/IC8oReR7ypw3ErOD5MoxkLPKmk0blGV7m5Q+s56RKhjMYvogZsy3val6nsmuC\\\\\\\/zSeYvSV\\\\\\\/n+lIMVV8dmwzaltAm8FYsKBGGJOA1SWam1y3lv1F714xSRSn5x4ViDGUvPHNzIrOj5yTf2i\\\\\\\/oFX4o7wuOO7BZsqZW\\\\\\\/9prg79FI2uE+1frV7g3NHISg1tECVtYPX3q4hkhTALnogWCgSYU0xoUllZ5c1K8rAuSF\\\\\\\/8LwlXrMZ3kT0MTvzamrewITWNL7Ba9TOni+XhIQdF+2WjZDDVkj28T1ss6WweeuIRoJS9LldSJLfidC9dZSN190Zz0iKmfrZbiMjo2ymgPvBtvS3FW4pjoxJRBEZuc37GNI5LILJ+HTk2ZoEe\\\\\\\/gPifTn\\\\\\\/9ycyH\\\\\\\/1\\\\\\\/i13Wczh4GiCjBVOzs0mDYWLSahUXn9VY0DmhC\\\\\\\/873JvpfACCFX0CZIQZGDZGpFbtBh\\\\\\\/ra7Q80JFzGuiQWaTNJv8p5ojC5h1llLrj\\\\\\\/Ic6TmwjM5M\\\\\\\/fphSJkc\\\\\\\/68TnO\\\\\\\/XKP4hJHdx8eR5UtWgC77k1W4rWFgbWwSmtthr9hVQ6RX0ZEI8hrYcnX4z8eh616p6fxfF4YzHkNei9yI3LyNJZwJM5wX7ZKdYmleaYkIh540qrcPDLPT7L7PRvqLdoYukU3qExM1J3MhD7tlTbg2bOJ\\\\\\\/vphkkyfjz0z0VBHDd0zBK2qDi7rB8Dt0vjO\\\\\\\/0jj3OA9HefEpMrYfUzpI\\\\\\\/aXuVb4GPN2Ekbbsvbe3hEM\\\\\\\/RGgO6XwRUPP4+\\\\\\\/0cGLEYnKvVyBCVBFVe3a7QhH1b0nqmHN9iKHEjEruYKBhqsTjCXgo97+9yTctVXo59+qu+aGj+Qb+e8AoNRJEao+i7HL3k4dqHdwA3ZV9mha8Rdf1262qhQElSNym3mmmqNVwHl4Ripu3JvHUBr\\\\\\\/Gyz4lpBdFRtJi2sQASn88q5dKcoPbA8nsmBzdD6wAb\\\\\\\/aXTnHtEzonSwATUxutet\\\\\\\/\\\\\\\/3aiang9Ez845ocRoVH2fAsn3+kwRWY7aKQNHPoqPnC8wyWxqX7xqLbPo6l9GWRCcRvn9yvaVjLeRAKRjIbtruut1hhxZjONAqL8pXTzvxtCjd1ckgMBhvsyeFOrEtfU9e0VXsCkSl3Q7JTlX32YFvfU9KgN8tqNenZYY72SK2dpOVIWHjy4i\\\\\\\/PjAOG0NvB28fA77JgBxsO4foBhduv4mmMC+W1zkm7gU5jhPsLYjvp9s+O9gJ1XP\\\\\\\/VYOsy5khBMrmFupzJdda2YlAk00YYNUWCfniJhVumubS9V4fMpH8L6BqjBdzonoooXdFqBjz3Av\\\\\\\/XO9pI8jAo9uQ3QQJ05vfZrTdYZD3W0SKKesn3QTQiWWRNVS8g8szArNlJ3PWsm\\\\\\\/fPhYqUG88286mcY0ZdubHAlR5SSsrTKQ\\\\\\\/2rEmZ361\\\\\\\/gwFdeY5GgG1t\\\\\\\/JLlwy2oBvMDLB2CBCzKu5BI2J2Gl\\\\\\\/8pAlHdFMfQVAGznOxieQR+qdAUpTyiSq1t+hPw9I0tl7Uya2u7KeKrkP+f8+IMrNZL\\\\\\\/hpn9sIWrWLgUrIrftMUd9kxkbGV8XozGZVjFtKxmxyA77aVQO\\\\\\\/+Fc7XJHDiRM7U5ApL8b9e8P\\\\\\\/c9rGdt4DQO9ALOFofDdmBffj2AHijLBjB2mBgdgJcmYnkuqZzn\\\\\\\/HKZl1\\\\\\\/iqX40mgJuisdv7EX2M9DYLpEvYla4Df3DRDUkxRraiz\\\\\\\/pLF8E0t4hDAa3XSXbnJmhvY+VeewvCOVEv15Rktxq3oIXceno69oCypaxT3QeUA0oRL\\\\\\\/na7cWg2umu72vmw9jz5dsz4VwAF32jkXNajLyudTU2pPcrq7ts7X3se+Y2ptDuQxqWNNaoRZg1WGk2yHmL25Q46nfVpg2FbGz2dJEnpRpqPNuBqMTEyl0mg8s3LGU47YtgnxXiOhKLWjRu4LEJifbmOFeBC7yBTk5upPLgcjd+Wm+b09g7MZkUBqYEqfyfHwqH0AC\\\\\\\/8R171Usb8uAK1cDTSLsJ\\\\\\\/GHz5nqnfO8Uo7JS1ddXzPx8Wa86QsPjglFtEpnqD84423N8pvtbEkvAc8vGomyaa97xpPrffC2EGWX8p2liUEzyEKDHGFiqWI\\\\\\\/PMkVcGJGGqvg6lPewYGNGenTjEpYKVd\\\\\\\/UrzW77VNEY3mK4hnF+Zxk6a6cyRzst26WXs6bd7zT+bb\\\\\\\/IMYstt4gln\\\\\\\/1Jrz792krVRnNo0A7+ZqvpLFVCMgsNNF3zfHQq\\\\\\\/xgmTTI7nRPkiwKt03tXvhH027zAabUcYxmg0DUtRdaZGzBVqjcKTFjpo58ZLl0M1zDcGdmmf7sC2ovtjTqbZv0i1Rr8YfTV15kL7aHe0RSF3VJSWAu050kWn50dBbyD7u6xsl4obZqEj2wM2NXHptlTXVj4cmoByb845XX+cPYBx5LkriIhfcHT9Vh9WceLf68BdhHUJfOQrL0RnZPQFwOvvo17Q26sJ4BOZaU+e3LsFo2aZPkA6DURJRFULGzPzy+kWWyh9nEIM0AFLtCp0f2j4phrRjThCCFl6wLCCMb\\\\\\\/+Y2h56zOuJCOrUQ6vhnrsNyMyhlS0KopXBYlwYQTDQ9a3NNVMMEwrevT\\\\\\\/9rwSQlhkJmdygTx+Z1cIqph2QIXnHatao1w6zpVlNQZBHxWEXLtyATvrUExp6e\\\\\\\/bpfSJohvgTwPIesZmSbHmjzCVMA+EGfJVEkOV9jLGW84qZTKuNKklBOUIfu\\\\\\\/h0eZZJ+KVZuQdLQJ2Xmw7N3LNxefTdUFh0dTAOMPrTDJvlsPsZfjY+RDmzLS6nZfCCTxFGxAgM7fm7EmIMdPAO\\\\\\\/sV3rHDbsxGmH+ai15Egk96L7M8w1REdveCa3fTJmVUK39C+Sb9\\\\\\\/SPoh0ZPo5yxAiHNwyIufeEbEFm+TusaHB4p6\\\\\\\/VeA+Uwll6BEMcpgSWwo\\\\\\\/sCCoooSeBERmSmzA0zVNMvgT1L5QxqIAABmZecg0JP3EwjkQtNUPAiMrQOpzSazUeTH+uLhgIQtMRsr43i7MTKUiF\\\\\\\/iqqZGMh3n8yY0lgOwD4ep2V7Lt6qzW4tVS+KFGvNkWbVZuIbEnWHQdbG1x0XTzJ058iE7nRIwayUZK0S0ipiBS+JK7nO93lyEAU2Z0jz7SD\\\\\\\/5UJOgHbtI08fU7MTNCjFPX3cW2QAy3NQoXZaNnDphtkQIuFJBHAOdcPupspWCNNagvCE74caOILfdYinIZcBp2dQzgDGryRk+oXOPZkoW+\\\\\\\/EEPqBhqnpJ1dX93tYsMMs\\\\\\\/ciEPWnWuVxAQdeGKqyv+MIR8KO9BPGBqXRptcqzElpwNZ2I66+n2023la+U8QV\\\\\\\/AwtO1muHgV4m+WS0wwNE9Pje\\\\\\\/oCrizUtm4GJc\\\\\\\/O+tJQVfn\\\\\\\/XhxW6QjJ3QCqHyKhot+hpNXL\\\\\\\/aklzNmh4r79+XmHFu2KuLD9+6jbkt7qp2\\\\\\\/sNRhRDFOnC+UhDj9Xgv4DSe9n\\\\\\\/B3EASuxKWhLswcQznbnCzmqbrpK2miy0a295\\\\\\\/g9JMZqlJ8uu4W9yquPUM4faMVduv0RrlrEpzLcdpkBnNwyjwYIoJk\\\\\\\/1DKIque6CPQvTQPOkRcWoZtIK9b2K0ZoQsoJznJq5LCO8vd1HjjonnSzSXyRbMl7PXDuieVnYT\\\\\\\/7mf+FAlE8UrWGMC5xH+N\\\\\\\/h08hDQos0SOpygHly8l4mrtxAMNPCQjD7Uu2shLyYiwfxWqp\\\\\\\/ET1J0xEf9K6eVSzGMwt7E0VhZ6VzOL8eq2O4+FfEtehSymKqeXkS2CiR9unQGFomLZE8F3owEWCGWr7q+EqBsKfo0SO4J6H3AGnUWnrS2wbyx+FZlX7gV1+y74iXvPmYXrX4VfxBE8oBRzPegi3T4rmzjMDqiL9FlkOYgtuqt3M1M9udsKQaMEtEsCx6dTf1R8R4DcpovHfdXW0sm8d04FiU4Xoiep+a4QnS3fp5IdsVitI6dAV+BTqqQhMn3mPGtESic0EQR6I8pXxFpIcNiwJytNhEbWq\\\\\\\/cjb9O4HXmN4B5OXa4ZFvwXmKrR2JPsWtjdO7CdJ7LME2C6gVwYCtkiYL09CwEWGQuwuy73XRf1MXAIQdhlyaXAA9aEshbLgOhiVBZogStQhmK6mskccs2V8xhqROKaDHh8k4gpvwmmtwhGWVT7roKQf\\\\\\\/2DTg2oG56+L3A66MYGkkLkhw5EM+GvWPKTEw5WJlz8SRBuuYEBtEMjcRgoe8Fi9Bpd\\\\\\\/s6K0pXjksdHuwj8EtEiyEqckxMMpl2o1xyddvWtC9q3dmsMmqoZ4XZWYn9Uz\\\\\\\/4dmKAlQl9KX0sOidFGc09LN7rSlj51ibXYC2RZ7KGekVFk1H6sVF0KX35vnUf\\\\\\\/r3Scjr2P4TNT2jnIsOLzhvfnh6czob28Qpu10YzW9yRvOPE10azyFTcyfRq9fFRIIMu0nUeTZ8sZ7Mpbyeh2P0gNGBaVRLJOUSVlE0+DM6126rT0C+z8JoLDjhsEwO+M5RMxcv\\\\\\\/X+dSMYOx47s91M66LXmDPms92ndUFG2yfwfGYYr9wV73goJ50ucVFw0Nk3nB98FrVSSkoKcF+CzbHFYvuxtmzgtmSgAloIMwrrHyzREH8R0COEFly6xNeTZ84Pxa7DgDur0OGFbNcJTWy1+64CNS5CnZXmerZ4Xu2DnyDQEiKTg118w+T0geFuEAXz7SLpwgjAjjOtwX5SsMV\\\\\\\/pi0fByojhCnw17TxBC2q9UcvO7z+6wjstXvj8my7M+LVTMysXnurIE8a+aZq2W8WbDEVsHLjJeWT\\\\\\\/E9utrQHhQG7GtP8i9+SlKMex8TA8UyfqmyOtSs+wnz2kjRy2HVPJEZ06mN5rPkj64WEF8zWgUKegkTHU1ImrdON4rbIY8Q2alLUl+nQR4Ggf3DIiMdQ\\\\\\\/b0dhgbUrKyvGncsijYPVM3ATTvoAcuAhR5GR\\\\\\\/qkFA1O+YNY26yQNb4l8g9FvmKGDqZO9CVoz\\\\\\\/uLHjmKXrkFi\\\\\\\/llvuirAH\\\\\\\/3+Goo8UB8nx6xQY4FTAZmgVgiFDqPYsoA7X4LF14tx5vap2M3crcnhdc3vT0m6+ZE\\\\\\\/zQa72X3Wsu7x5Q4db6E9x\\\\\\\/LV87UCsrPZFGGlt8OacjBVnoqTPDzFIeB4vi4aaNC4q7Y5QXI\\\\\\\/QC5cpWcM9eb+JtaMpnqGD8wS91cbVsrgrEcRKCkVGNiZr7Nm+u+Oqxs4oZtX8VAmc6KJQOJawt8BAl9J\\\\\\\/A8jM5dBdJ8vJOqBGwx8xOo+SyHySlQfxQQq+uu4EOJn8RGBh2a7V72d8BnZzEYOFFE6EL+DirTwJ1i\\\\\\\/rjcG6Cuci+WwrNxJBqeLqm9hKokrZmPEAHQnbSpbM4sBWqywOFSoVkqy1FXz9b8lXCFdKZ8W3eO6Lsc8mNj9GEiomYX0PKnlE1Y1PtdJDZsIRmbFINYoDSjtk4SWY3GQDutbHTLH5MfiDICi6ccv1ZSkOHsR5MrfSghg2wecosEpptCyrE8u0kQCFISw\\\\\\\/ILZYUGpbpqfw93lH4eP5\\\\\\\/h8yGg7\\\\\\\/9qzkOBAQ6RCh9OAa7iqJowJQVqWZwIaVyfoVH8\\\\\\\/fHbHAb\\\\\\\/pQfg4lcI6b76u0+dIav6ns7eYxlw09akRFpSXwAe3vmuXc9ROZur5XfhA7ZdI+AMVJ\\\\\\\/4NNT5nI0Ft9hORQ\\\\\\\/tQ+QuynP12LsSVXWfs0UNvy2\\\\\\\/Io54TC6MseAVifHeUXMwp5ojepWpMEkco2rRW+mjerC+KuHGHxaURpDESNGI3QfiDSuze+lkWoI7E\\\\\\\/UHTinRvfw4dyWfCTfOHM2TtrXZlTKu7U7B0vNk\\\\\\\/F4Y8mtl\\\\\\\/39FwNuuNdgvmLtBP8q5RZxSpeCVzDgxgB3ZBcJ6L7NjsUxKN7KiWI4L+DqHz\\\\\\\/RTAKiNhekLr+9HT94R1qwNod8C\\\\\\\/nXMo6lFf849ygwsaIMWaYHdnIY5ABIn\\\\\\\/vYSM206GSbEY7xAaa7wUKIwwWIW+wbSO01GwUaNRM3kqMinhEabnXoPMZ6J+vMQtdogeNKif4\\\\\\\/JDHKwLTLNpYZufvNiY8ueotpd8I7gH0EeBWGnOkj0Vtr+qex7vqI1YH\\\\\\\/9fUSOW\\\\\\\/QrjNrrulCZzWARqe\\\\\\\/GgSJneTAhBMdJLfyKBMoQMbJUPIJYhtSULB4imPDmtYJE3mWgn73CsKqQTh1kDZ\\\\\\\/NQTZGApx5DwStx\\\\\\\/\\\\\\\/5SXY7hhqxxmhLnW8sr5UQsj5l0MDjwnQ2Nas\\\\\\\/Yo4+LDsrSWwlrYnSJNvEYJt6dZ0DCkmoQfISNxi749xTg3kFfIjd7pzCu0jSiiPB2r63dcU37zu3r5iOR3gunJd7HR9ui5Rj\\\\\\\/rppSa42fA3oivAM+YMtau3lrQOSaI95LzqKjekS\\\\\\\/qbA7DjHemHXw9lNWfz8axV7lWVrP5F0lYwI6DYRLtHHSihTt\\\\\\\/AzK29ep+ZogXpTa2ZDkLG4X+vIiQfHZq8aGfs86+yxBoA18A3JFPUniSSQVRh4QyiA0iNaFNPG8CJrYJ7RWB3sTC4lTz48gVOodAfOQ7UOrnlbWpu5rsR3GnNMKvJQHGP5ga9W7RIsgIeoQn5Rv\\\\\\\/DVkgQSvEFOYknvoCX+tvfQXqWq6Uujrq8jEZ11aCGs0E+wE+T2RGuCJxSWbA90GKb9VgCKKigVM5\\\\\\\/RgDHcD9YGjAWYLqi6M+vFgcDjMGqRYoUyeaM+0kpHfQrg3rTlT7aDawF7C51o2bmIYFyyrJMturlv5j0Fi9E2FF8pMVb1\\\\\\\/2l36X5lD7Sys3Zprz33jQg4g+hUv6k\\\\\\\/BX0UjkxImlxPKXWXkpKTQgFA819JANS\\\\\\\/B37ps0cZ\\\\\\\/9FqTw+5bTrzehrQYpSEgZqacT8ZmgPUnzAxQPIyJAEPVPTtA1BYQKrhzuMuwj8FMSWoZuubGXWHFfzrS9jtrNkEwdYKIIXK0f+2MAJvuIfVKXiKCX0JPOzUA2nmPka\\\\\\\/tG6tbr+Dw1jLadpGHRV5PIndKk7C4RZyN9Ru37X70gzwBRgskOL9KorXZ4olOFR7rItsL5uGHdAofvpLgnqFYNvHNPHIBzGtk4eebIjWFGwKfAsJIOx7ZYGMctHRAWNDjNNS3T1ggcYE5fT5Rz3EzH+CITurWfrk4WBK3t1Nukw7mGZH\\\\\\\/6q\\\\\\\/LjY5U3LuhZFLHoafKj2\\\\\\\/Sl6bvo2SHJY+lYkccXoTMHFCFMV0GqCo6kY09HP9thEmLGH9Oav6wXJ1oY8r0oI5sydvhiSrTPFnsAdeChlPZREztiBTK9HXQgupZImYdjYhWj5L4g1OeAxdBD625o6Qd5vxxtEgKhVLSUpI3ZvXS4U4sYXspiccjmxgJidmhC8Ad6edBSuUPMDFMxvJiId2afe9iXMGTpmwPY04ezBD8Q0njAHmSAXhAN6QKo6vc+7p+mtupD0IrGfnG4EuJI9+aiAkobOsmQw0xEDPD3GZxIXJ4P32fQv28je9xoXinIm\\\\\\\/HVUDcxqWEusy2IvYBtfYyigxVvR8RIo7ufi8Pg\\\\\\\/CSl5LJOEMbI7yfmrophzVortcebmeviIThPW+lYc\\\\\\\/5i54Ag1+Nrq9L06ZS5BfI4oZMrGbZpDwVSfWZdGLkqA3WKfwoTr3Xc5KjgM7v7Yfg+kPOjRzQHLzztqAt0IkOFNdh4bRSuWEnv0UAnQs\\\\\\\/AGznLQ1EY\\\\\\\/QG5DgGf9+953e6Wx8T7jjeuqCAM910KOrIUxsvLlhMKPJchEYPrzmz\\\\\\\/PMCdudpIZ\\\\\\\/6\\\\\\\/5BByuvr0pH7XkKZCVO1ieFdZOp20UTKdL\\\\\\\/SL3qQ0Ftsi5YzwplST77gd1koQMnyvAZvUMgVBiDIMy7proI1ldEkI0tE8dwONpkA\\\\\\\/ECWPLOonyR1o1eX\\\\\\\/ZrbnfjS15CUlFiUg6aRlOH8ns6zSfUMPQut4zenclHWtj5VoWSqF0ytf8iiYH5DDImNqQu4dNhshH+eCUfrcH\\\\\\\/kAqaIE2keiKOB9Px1krCUIU2wYY64FPoGdWX5NnERpswGgyl8LJbeVR0\\\\\\\/UQ0YyZ4Pb2upf0V2P0u7hRi\\\\\\\/Hu+8h3XmkBiUW87IaNO9AFVzF2fIVFMsWnEOFqmiMhz0lfT1TYOEn7XxlxM86rXBIt71BzzEMJqn6fJXCmWABTy+mbXQsstcGgSBhXPFoWG492xkZHoSQ3Q3TI1LOImmTVdPxAI5mFxuAXeS4refj7dQFhdRdm85WF8sbfrpK4WpowN6DUpguZKRfidTFQ7qrmUiAq5DOCL4jz4R5NUJuuKlEv7oc3bA2Ln4mZAfBwYVNc\\\\\\\/SdEZNU9J6WvaSD4xtheWT1mYTb9IR+DMcnOJ5aAvGEv6z7V0t7vEqCjM9XRaoR2wueI8uaylPp3Dr35fX1BZ5vqnabaYej9rUSF25iSQu93v+nu0peGQ3oyN+OQ2YDIEwYUeoC9nGlXdSjXFHXx0AzfN1KTfNVGTL3xuoi9aJV76Mc\\\\\\\/F\\\\\\\/9cE11RuQaDHpVBWIAck9FiP1bt2vd+mM0T0lA5k2qx7mVYyVLdw8W7otlNVaOTVOwe1dEhW4lOFMqhDI6Grm\\\\\\\/vpEcuCt0Lg\\\\\\\/vE\\\\\\\/+s0a5djOLiACegg3Zx\\\\\\\/H0jI+yHTr1ypCcjyB94P8D8GuPHjSV6D8u\\\\\\\/6qghgrwensAC3eHk8amJbLU9WzeSgBNaGf\\\\\\\/ghBubM5iDxI0kjNC2adS1PNPe7UPmN2bmOIPpC5GfyN0Iyp9PdDzf49FY4lydMm5829JxMYH\\\\\\\/OaeXtu8Ce8HuLRt8OXggBiTAqpEQdrP05v+j7sh5rxc2l8IMVCzXR49jJpUaznsD+RumGN5QhvwoBhvMyHdaZVJPWK4rFdp1NIxhRlRjk2G\\\\\\\/vgqOcqKyyvIJi3N\\\\\\\/YN5C\\\\\\\/uZuP9jJrZ2M2nkBiJk4ZqPnE+rhn2RVa3Pr0UIt2ptVqoiERnSQRoFWu4KT3wR5u49Uqnr0ZEG5boo7Nvmr\\\\\\\/DkLBopjfySLAQa38AQAUpujI2XfJBmFJtI1ChS3xfIWWlt76yyb9TDUQflIexM6Lo4jBhEr1gRIFU9Pjs1pe\\\\\\\/4UgDYl3ipfmYLOhgZ9JlFwntuaejgNJkxr2dA9DiycJL\\\\\\\/JTJ60LMcoyrbg67otu3AcPvonravGfNsXIXZOmzqSZTB\\\\\\\/SQXG5JqIdFzBmq8PieoAp9oIuDcjV0QGv\\\\\\\/YZel3A4RR7q7JJh5z9R9d8g5j6jI84YI+rNNUKbWSkJ1HVStnvtQTsbuMuhNRM57P2a4vxF7Op+gaBlp3IZlOh1EyWcPmB6OBaYk\\\\\\\/kHt6sDHhG45E+\\\\\\\/aV9EQRlyu8eRD7yTx71d6ENSYSslSHkcsW9LAxD4IMwQUVpaqwjf2tE0aC8K6oKVyZj0GuXD2juDfavAwgJCGZsnWLd4qj+GoPMEM\\\\\\\/Q5sH73A4zJfSAqT3\\\\\\\/Wu6sgbUbbTPnjqjfi3qbBPi7EF71xW6ThjwWGUHifMwCCGFbpn4QcN5R\\\\\\\/Pdj8Grhwd0Bop1EMDdURtX7YgjQNTxabVsJdSgkeOJQyGS9xQNwyzF8nt7tzwmra96NUurLQUhp2ycFJEWdVJakoFc6fa\\\\\\\/WfRFcn0gKOrnSDhASgV2Q+ZJlzdzOHwJ86GaCwvdRIwBL5sQd4qYigr2jw8v5UCmEIs3CsLZIBqWA8hTU2Wh8ydp+TGIfYnvMT7xcsxFZt9paz0o0iJIoXYZb29QCssBjpBvVfToY6Xcy6i2lI0m+PeRscJgEFl43d0tBDDEo\\\\\\\/6m+vvmtEKJ6rqbHv1X5hGCZagIEz7CKBmd3Wm9OB0\\\\\\\/KuqvCR+hpnz7yjmR8f5H9W6AO93CJLjBa+MToSCqsy9u9FUq+9lQ74XicmqXAoC2JhohQd2IBeP4oywA4MX3inGuDCOAuCIBhvHHlwxKTDjr+mg03fzvWWazMCFlyPizILwj5FXuhwYnyrhB8mBPeOHwQGkcZ71IFJ35TVVPnQWXuIFLoZpb53CPkbO+HkI0BpiNzvRk8Vdp7M6qzNP7zU5e4bg5YfxYIXYZeFuMoK7mUaxs6AuAMtz7cNaJR3N5Xn1GaBIhWn0VWYnSS5npH9jxBu\\\\\\\/nr0CMZp+TpA3jol3LExEbh9ksaohtMcfqVNfoGf7H8yHpPYHxAUanKjH+dIKtZKtMhSIR7B6xJ5s6VltW2KFRXg9iqs8R7IOCgaOAA7lKDdzOM1JKTJxq6SLGtSH+zprvyOc0lXxrvG0BOBW2loMUsAPZhxRkfsBsBWcDsbQtFUTbkSoHG8+U+uyiNQ87WoEfyK1Nnd8JDxenrm7i\\\\\\\/rHmpBdhY1KYAx9PQnDCLt9Ny9LyYWh9WrQvEh5DK79DHItAyH742yI4xWGsem3xEFuQNLK+sRGBtKaEGnfFJ1GPxAhMwnjffsYe+258qtVh\\\\\\\/hwCFbkh84cnO91QfeW3RRHJozQMb\\\\\\\/\\\\\\\/ITlOgmdcx6HdrvvIds1+ULRcnSod\\\\\\\/VXn+oGX5xmszGi+6z7IDGixQHyun56CBNzP7m1LoP6bP0luxwPpuN6GHR\\\\\\\/3R+KX1t5h1ru9\\\\\\\/tDYQ0niTM6bEcll3T2jjebwyi7EAclOoZKrGf\\\\\\\/J8H6fi7nKB4a355t7opRxyBldoTxHlp3kqsaZz7mpvGghdfO+hr7lZxZq7otMoGTOA75QgkMvG0jhcm54BanEW9uUui4EGMyzX488ITsiwubhiHo64LjXtELe7R20Aw03QRfkNRPvnuTNM0tbKlOzWbvK6WhS41J1V8+b2sNw2PJXMxWSjdvzLQ+VDv6YL31JXFoBNxQVig7ki7SekYv9XDLjJYGKC19zJtqcXoCF5Fg9c3pb9EVfD3aHae7ampxpNP\\\\\\\/yAdyokU2vVkPWdAQQr4iSecTNxmzIEV3DN0qHe\\\\\\\/lsmoty\\\\\\\/YVZ9jQHXEWiFRavIMMt58SNWi+OfDpOxFfJgfxeoC8DsgYYUqg2bwIMLqai\\\\\\\/OIeQje413PJmwYJacsNgM1\\\\\\\/XBP6hlPztUOxDNNgYMlyl3SOoMLeOchgAMFsTYRj8HdxK\\\\\\\/4gbVzJd5Ue3da2bXeQ\\\\\\\/Cm3F0EUIo7aPdKJ\\\\\\\/Amo3YoODP\\\\\\\/vrUQTIkC0uLNidYZdKkTeSaD5XagfAZAQ7FarYMN\\\\\\\/wWLDw5MHjxH1XZCp2aKd9NWE811onRg92S\\\\\\\/5VVoHCFzrx1m6lkwIFAZN+F6RwqUI3\\\\\\\/QTiiFusKHBU8B2wCaL5kSHrmOatYPp1KiDkiu9Y78Pm+8FtjH3KSRcPU6pEk12u84GpZR3IW+Q7IrvCcdVmRW1kZxglLx8W97BKuH+Fr7orUG1K0uFjOl03MkiEc2vJQCRxcimMaAvQuaPTQK3JheHXS8Slw0XDBIb7NNXkWTcn5vl0prWTr7JS3eSr2XyN0Pk2kDal0Jv1hKZoXp0qKbiJYS5Nq9T7rSBzyezAgnDTdpX3wcY3NknyiXAsga\\\\\\\/\\\\\\\/F9Bk7GtKqKTuiFKBxAMWlvoB1TcWqejqi\\\\\\\/cq1uF87gdY47lZd85+q57qqnnSqvXtvCLMlIfIO1sVJJ6a6DRJQFheTD6KjO20Kz0PZ5Ki5JSzkkdJvKvghx4di4XClAaXSSa9CdRJxbCYlzna1+wAJCi2fAqfcTRW2fEwZhPsNlrwzdMrGMcu4SBrf7PSJgUJgy+ncwRiQDd4K0ZGbCPoX66kT87aCuYiNa+zhlVzUTDjhykLT\\\\\\\/IT1A3bTMpc1Ro1O3Vgr+JpOAa7LCf4DbjwkOUb+5Xa3EYLQvoFoRZgqMLy8knpsYyF6\\\\\\\/vKc3cA8T1zcmdpJe93xKGwYzSQMaFIiYgD9JLvEQAV11fXqxJqSHIpDV0Catq5r5S2cIoZ\\\\\\\/zhSpSxev5YVr+j48Rrh1gKMyNgJ53OI0Sxg7IRpc54wU0zoUgv6na36xk4HzkHDtei+c9rpB6fyhJGgA1cbRsZjV+OJBnXKLBu6p38pvsRCIREcoLjYnv47hyTy+nXqodZbgCrM+zb74HqGE2UlfzcjJzC60aC9QFaIB8wi\\\\\\\/kuhxQgrIgPHnQA+Az1QqVeuG8x+bfZzNsLF+USkb1+FAfhJXfMjnp9vmQUTGcTuRntF4iV9CrsHmFH1yw4tHlfclOO019GeMgKvmhe78h7PMOMl5ZPNxJNXpt9LU\\\\\\\/mwgKOB\\\\\\\/7BRi0Z2eZaltMAj5hf+1SQwChhu4l7+zxdxFI+Qk8qx3zI7arqufOe8r\\\u0022,\\\u0022iv\\\u0022:\\\u00220xC7EzcxKgCw6yk1EV1irA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022211b021a8a181f6cfe2b81a2d2badb152cb117e74012cf21755a22c2d6cf928e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tO+WvGEDgmNk3iEW36suKy2ztFuk\\\\\\\/1TMoCHcGRApGjIx2CbUvxUFdF+Isn+cYICCwEe8B4jm5x+zZ+yGYdkeNxRH91+32k4wiMZ1VhwC6OiGeZU9K+Jk+fhAYcrnYSapxOIHbP0AB4OBk0e9aRmYFstTTNCwwfb2sZ39FA9KY6WcvPjoaEYRiDPegZwMXIms\\\\\\\/AZxn1+51TeFGD\\\\\\\/hF9kqo0awPsyiKqlZjsRw+f6+xUDn4JtpoS3v14cLhW9SU9y4RaE7Je4IgBerqDlp3uk9NZ0u6AWHAu87CfmQ+UncdpLTcfvTrxVEgAxN\\\\\\\/ymjRvXXwSPCBv5VX3tXJ0VGSsU5DTAAQPJ4ZGzKNXCi\\\\\\\/ewd\\\\\\\/D9XeIH1UUz54CvXlJDf1vBZUWydaXZsKNlBGR20RHWf\\\\\\\/eCNrR5pa+eyUbluJnB5bcOYxgL\\\\\\\/bb2+Hm+VV31mFkuUpzZtxT0EHc47O3J9JGn20cpoGcg8smoSoE4VbCzSLt+MjkJo3KYYnMZgJlpzA0P3bYux26fFD0qz\\\\\\\/z0AgwHdReDWshWYkdLAmGpxjfE1o2JJGyqLe0eoexaKjV+gMug7k9v3miHRbXCct83jrYXpkZO6EYQmSHEKGDpFoU\\\\\\\/4EXcFVCYOzQfJWW\\\\\\\/WNOKXSNNPQq4SGzp36hcCYeQOF8qHj4MbFgGIX7AurBiytZ9uh5OzgXORX0SEEACSMqAxAkBqG7L21ZDT2wjFRca2kODfEJ2WBaL7Up6x+mOwqA8LlHy3+FFphgHhq9\\\\\\\/WzXkB30wvo9qR3qgN\\\\\\\/kmIlrToRfHDg2w+60XKOo\\\\\\\/+zOqgXa\\\\\\\/JqXThXufSmQYF71C234dMEYPY2HlRxVrc7gVSNjVCxCBOY\\\\\\\/jppIx9JWfMXIFaKHmmTLu6GvkOIWx9bUgf7sgQh9FIFxD3e8q+0V3YPC1XjPEDGZIYEiyLj1Nrn9AYiRnXRITQk4Vc96jB2iAfVMbFN\\\\\\\/rMrC5yFXiFIEfVoKEWg8un26H9NpvUclaHzfHBaaqT7BUwmV03lxacJB0nMPe+A3qVzkl7ZhkCqrQDmB4XQEnUdPx5Z2loyR3YJLWL+4XgnKljNuwJxH5TZoShKfqpFK8+AZiW089uPaXJC725yfqhR8lD7nvjl0gaN9Xh4SN21M7jmBOUg9IsvkJJNOCG50zukmgG6Fz61kAsUUW\\\\\\\/54YZ8sqPfFsdbINubAlS8y8wbkuIXRqTZmE+FRg7wtu9WZi\\\\\\\/tXbywzOpOc3CdhsNonvKElikyaRbcUtmOvzlk0lyhNgdvht2GBrph6dFJOtFoiyCmcAVYWo4liQdI\\\\\\\/VbR4Q+5kEghQ0rH0S3XUxami2VmusarOv9QNBG\\\\\\\/8bu86dGzkcnoQFTY1YhBl6lHogCFPFDJnH+IT+ZoGGQ36TwE2y7IdWFL9fGZZg51Ij0Kv2OR+J7wIvHPMsX2vJrc+ChGBSRenoU8XAQNhetRmeYiTz3uMPFWKrNHu8y+H0DKq5RsyATZzLWGk3VWqEib+KyI8FlqVElDGXOCNAEg4FjmxA9ND7\\\\\\\/15Qj\\\\\\\/zIQ\\\\\\\/7NYX0DrnQKqQM5dgnh+fqm5GXUGfZRbtTB79Sp6tqt8pYzxwkah3zbfafpunx7dXiGpvn\\\\\\\/766S32nLlboYfGXfRo\\\\\\\/SBwTa9xjGYBjdv+Jthn33pI6v1wEJf74AJ47OZE6DV38mwUVPbqR3qMd\\\\\\\/3YYnhNGh1l+VQcOuNyvx86JYlZHN8psU0QNN9CYICI42ZX8139cSKXoY4z1nLtbwjxMNm6Tgpgrmak+cVoltwdPfRStkY3+uHd2Lwyvlb6ltLd+bljpDvbt5IMyAJvi7IhwPkwhkyLMonuAOnpEj1nkw8B1xDHW9Qu7nGTCnvb4qFitEHmPw7h2c7763TALAd8HZRXZ3A0yNOb3c1T32aBoLjrmqT5gpY408Tvovmm0nG0FTHnElwzcuZbHYAxA6IHDP1yhQRtzxN7a4E206JbN8\\\\\\\/VijanhPvvAATO1VYN14pfG5NQn0iVIjh+s2DtWzcJPeccckXqxm1VokxIV9OBYi65hqjekzNUBk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022oel83N1UEkvW8OvofsYDwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f7dbff58eb040353db6c3570e7bce2029702467423bc3d7624159a2891728b53\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459413 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/343" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UHS1BG9AYR1W6q0-KtAZc2erYrX4vQyrnXrb50tgW4U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/343" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aPhuzzw3p15vB9HsQeiU10y1SF0Z7_SQjUrFH_6VDps" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/333" title="A High Load of Non-neutral Amino-Acid Polymorphisms Explains High Protein Diversity Despite Moderate Effective Population Size in a Marine Bivalve With Sweepstakes Reproduction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/353" title="How to Design a Genetic Mating Scheme: A Basic Training Package for Drosophila Genetics" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459413" id="node459413" data-pisa="ggg;3/2/343" data-pisa-master="ggg;3/2/343" data-apath="/ggg/3/2/343.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Karl Waern</span> and <span class="highwire-citation-author" data-delta="1">Michael Snyder</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">343-352;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003640 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459413" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl Waern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, Stanford, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BWaern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waern%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl%2BWaern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Snyder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, Stanford, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSnyder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snyder%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BSnyder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mpsnyder@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">mpsnyder@stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/2/343" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459413" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459413/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/343.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459413" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459413/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/343.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459413" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459413/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/343.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459413" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459413/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/2/343.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459413" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459413/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">To understand the diversity of transcripts in yeast (<em>Saccharomyces cerevisiae</em>) we analyzed the transcriptional landscapes for cells grown under 18 different environmental conditions. Each sample was analyzed using RNA-sequencing, and a total of 670,446,084 uniquely mapped reads and 377,263 poly-adenylated end tags were produced. Consistent with previous studies, we find that the majority of yeast genes are expressed under one or more different conditions. By directly comparing the 5′ and 3′ ends of the transcribed regions, we find extensive differences in transcript ends across many conditions, especially those of stationary phase, growth in grape juice, and salt stimulation, suggesting differential choice of transcription start and stop sites is pervasive in yeast. Relative to the exponential growth condition (<em>i.e.</em>, YPAD), transcripts differing at the 5′ ends and 3′ ends are predicted to differ in their annotated start codon in 21 genes and their annotated stop codon in 63 genes. Many (431) upstream open reading frames (uORFs) are found in alternate 5′ ends and are significantly enriched in transcripts produced during the salt response. Mutational analysis of five genes with uORFs revealed that two sets of uORFs increase the expression of a reporter construct, indicating a role in activation which had not been reported previously, whereas two other uORFs decreased expression. In addition, RNA binding protein motifs are statistically enriched for alternate ends under many conditions. Overall, these results demonstrate enormous diversity of transcript ends, and that this heterogeneity is regulated under different environmental conditions. Moreover, transcript end diversity has important biological implications for the regulation of gene expression. In addition, our data also serve as a valuable resource for the scientific community.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/rna-sequencing" class="hw-term hw-article-keyword hw-article-keyword-rna-sequencing" rel="nofollow">RNA-sequencing</a></li><li class="kwd"><a href="/keyword/environmental-conditions" class="hw-term hw-article-keyword hw-article-keyword-environmental-conditions" rel="nofollow">environmental conditions</a></li><li class="kwd"><a href="/keyword/utrs" class="hw-term hw-article-keyword hw-article-keyword-utrs" rel="nofollow">UTRs</a></li></ul><p id="p-4">Regulation of gene expression occurs on many levels, both transcriptionally and posttranscriptionally. Measuring RNA levels as a proxy for protein levels and protein function is inadequate. In yeast, RNA levels and protein levels do not perfectly correlate, instead having an <em>r<sup>2</sup></em> value of 0.77 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lee <em>et al.</em> 2011</a>). Thereby, there must be features of messenger RNAs (mRNAs) or the RNA-handling machinery that affects mRNA levels, translational efficiency, and/or protein turnover.</p><p id="p-5">It has been known for some time that untranslated regions (UTRs) play a significant role in the regulation of gene expression (<em>e.g.</em>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">van der Velden and Thomas 1999</a>). Sequences in UTRs may control translation, subcellular localization, and transcript stability through a variety of mechanisms (<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mignone <em>et al.</em> 2002</a>). The UTR of a transcript often changes through the use of differential transcription start sites [<em>e.g.</em>, (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Law <em>et al.</em> 2005</a>). These differential transcript ends—referred to as differential ends in this article—may thus have a role in posttranscriptional regulation of gene expression.</p><p id="p-6">The yeast <em>Saccharomyces cerevisiae</em> provides an ideal organism in which to study this phenomenon. A wealth of molecular, genetic, and genomic information exists for <em>S. cerevisiae</em>, much of it assembled and conveniently available at the Saccharomyces Genome Database (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cherry <em>et al.</em> 1998</a>). Although the complete genome sequence has been known since 1996 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goffeau <em>et al.</em> 1996</a>), information on the exact length of mRNAs has historically been scarce. Several investigators have attempted to provide this information by using cDNA sequencing (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Miura <em>et al.</em> 2006</a>), microarrays (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">David <em>et al.</em> 2006</a>), Illumina-based RNA sequencing (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>), Illumina-based 3′ RNA-Seq (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yoon 2010</a>), and Helicos-based RNA sequencing for 3′ ends (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ozsolak <em>et al.</em> 2010</a>). Furthermore, previous studies in which authors examined alternative UTRs in yeast have only considered cells growing in one condition, with the exception of the studies by Miura <em>et al.</em> and Yoon <em>et al.</em>, whereby analysis was performed under two or three different conditions, respectively (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Miura <em>et al.</em> 2006</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yoon 2010</a>). Miura <em>et al.</em> found that most genes had multiple 5′ and 3′ ends, but they did not examine how these were regulated under different incubation conditions in the same strain. A large-scale systematic analysis of differential transcript ends under varying environmental conditions and stressors has thus not previously been performed. Our study examined not only which and how many transcripts have alternate forms but also how differential ends vary under a large set of different conditions and how these differential ends affect gene expression.</p><p id="p-7">There has long been evidence that alternative 5′ UTR sequences in mRNAs can have biological relevance by affecting protein synthesis or protein function. Previous authors have observed that transcripts in yeast may vary in length and result in (1) a relative increase in protein level (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Law <em>et al.</em> 2005</a>), (2) a decrease in protein level (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Law <em>et al.</em> 2005</a>), (3) secretion of a particular protein (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carlson and Botstein 1982</a>), and (4) an entirely different protein from an overlapping out-of-frame open reading fame (ORF) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guittaut <em>et al.</em> 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Coelho <em>et al.</em> 2002</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em> 2002</a>). Existing knowledge on alternative UTRs is compiled in several reviews, such as (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">van der Velden and Thomas 1999</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Mignone <em>et al.</em> 2002</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Landry <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kochetov 2006</a>).</p><p id="p-8">RNA sequencing (RNA-Seq) provides a powerful tool to study alternative UTRs. Previous studies (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Yoon 2010</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ozsolak <em>et al.</em> 2010</a>) have used RNA-Seq to study the yeast transcriptome in a few conditions and identified many genes with multiple 3′ transcript boundaries. RNA-Seq provides much better resolution than microarrays (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2009</a>), and differential transcript ends have been identified in RNA-Seq data by examining expression drop-offs (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>); in addition, poly-adenylation in eukaryotic transcripts can be identified in RNA-Seq data as “end tags,” runs of adenines that do not map back to the genome that lie as the ends of “mappable sequences” (see <em>e.g.</em>, <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2009</a>) for an explanation).</p><p id="p-9">This study generated deep RNA-Seq data from yeast incubated under many different environmental conditions and stresses, selected to activate a broad variety of gene expression patterns and regulatory networks. We focused on differential transcript ends and found that alternative UTRs are pervasive in yeast. Importantly, we also found that the uORFs introduced in at least two genes with longer 5′ ends enhance gene expression.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10">The BY4741 strain used in (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>) was used in this study. RNA-Seq was performed using the protocol developed in (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>), further described in (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nagalakshmi <em>et al.</em> 2010</a>) and (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Waern <em>et al.</em> 2011</a>), and using the modifications developed by (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Parkhomchuk <em>et al.</em> 2009</a>) to generate strand-specific reads.</p><p id="p-11">Analysis was performed on custom software developed in-house using BowTie (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Langmead <em>et al.</em> 2009</a>) to map reads to the S288C reference genome available on SGD, downloaded on May 17, 2010. Python, NumPy, SciPy, and matplotlib were used to further process the data. The software’s source code is available (<em>Saccharomyces Genome</em> Database). Of note, to maximize the information gleaned, unmappable reads were trimmed by four bases from the 3′ end and remapping was attempted; this was done iteratively until only 28 bp remained, at which point the read was considered unmappable. This end trimming typically doubled or more the number of mappable reads.</p><p id="p-12">Expression levels were calculated by adding, for each base pair, the number of times that base pair was encountered in the RNA-Seq data. For example, if 14 reads overlapped a particular base pair, that base pair would have an expression level of 14. The expression level of a gene was calculated as the mean expression level across the annotated ORF; these values were quantile normalized for comparisons between multiple conditions.</p><p id="p-13">The mappability of genes was calculated because if a gene is in or near an area of the genome with high homology to another genomic area, it becomes impossible to assign the genomic origin of an RNA-Seq read stemming from one of the homologous areas. Mappability of genes was determined by creating, for the plus and minus strands, simulated 76-mer and 28-mer reads starting at each base pair in the annotated genome and processing these reads in the pipeline. (Note from above that reads may have been 28, 32, 36, Δ, 76-mers.) Perfectly mappable genes would thus have an expression level of 104, as all 76 of the 76-mer reads and all 28 of the 28-mer reads would have intersected every base pair of the gene. Genes were considered mappable if the mean expression level across the gene was 90% or more of that (94 reads).</p><p id="p-14">Calling 3′ end tags was done as in <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>, which is further described in <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2009</a>; three or more nongenomic A bases were required for each tag.</p><p id="p-15">For a full explanation of the end-calling algorithm, refer to the software source code. In summary, for each annotated ORF the log2 expression levels for YPAD exponential growth and the other condition were retrieved and median normalized. The standard deviation of the expression level difference over the annotated ORF was calculated as the measure of signal noise, here called <em>n</em>, since the median-normalized expression of the annotated ORF should have been the same in both conditions. Ends were called as the first region—determined by both a 10-bp and 80-bp sliding window—which was expressed at a level of 3.5 times <em>n</em> or more away from (<em>i.e.</em>, above or below) the expression level in YPAD exponential growth, although not less than a fourfold difference in expression. This cutoff was chosen by manual inspection and represents a conservative cutoff level. Tests were then applied to make sure that the called end was sufficiently expressed, was not an annotated intron, and was in a mappable region of the yeast genome. Each differential end also had to be at least 40 bp long.</p><p id="p-16">RNA-binding protein (RBP) motifs from (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Riordan <em>et al.</em> 2011</a>) were called by using the consensus motifs in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FileS1.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS1.pdf">Table S1</a> as a standard text search algorithm, for example, their motif AAACACAW could be matched to either AAACACAA or AAACACAU; no probabilistic weighting of nucleotide combinations was performed.</p><p id="p-17">Differential expression was determined using the DESeq software package (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anders and Huber 2010</a>). Subsequent Gene Ontology (GO) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashburner <em>et al.</em> 2000</a>) analyses were conducted using the GO::TermFinder software (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boyle <em>et al.</em> 2004</a>).</p><p id="p-18">3′ RACE on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">CDC19</a></em> gene was carried out using the RLM-RACE kit from Ambion, following all instructions therein. The 3′ RACE outer primer had sequence CACCGAAACCGTCGCTGCCT and the 3′ RACE inner primer had sequence TTTTCGAACAAAAGGCCAAG.</p><p id="p-19">Luciferase assays were performed using firefly luciferase with <em>Renilla</em> luciferase as a control, as described previously (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McNabb and Reed 2005</a>). Instead of integrating the luciferase constructs, however, they were used on a plasmid, as described in (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chu <em>et al.</em> 2011</a>). Plasmid inserts were produced by DNA 2.0, and sequences are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS8.xls">Table S8</a>.</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Mapping expressed regions of the yeast genome in 18 different conditions</h3><p id="p-20">In this study, we used strand-specific RNA-Seq (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Parkhomchuk <em>et al.</em> 2009</a>) to analyze the transcriptional landscapes of yeast under 18 different environmental conditions, detailed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Each condition was analyzed using two biological replicates, and one additional technical replicate was performed for exponential growth in YPAD medium. In total, 850,570,328 reads of 76-bp length were sequenced on an Illumina GA IIx [see (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Shendure and Ji 2008</a>) for platform details], of which 670,446,084 reads and 377,263 poly-adenylated end tags (a total of 41,663,340,382 bases) were uniquely mapped to the yeast genome using Bowtie (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Langmead <em>et al.</em> 2009</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462409/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462409/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The 18 conditions under which RNA-Seq was performed</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">The number of uniquely mapped reads per condition ranged from 28 to 55 million (hydroxyurea and sorbitol, respectively), and the biological replicates showed high correlation. The Pearson <em>r</em> values of gene expression levels, calculated as described in the Methods section, ranged from 0.981 to 0.999 (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006108">alpha factor</a> and high calcium, respectively). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the number of reads sequenced for each condition, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS1.pdf">Table S1</a> shows the Pearson correlations (<em>r</em> values) of the biological replicates for a given condition.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/343/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequencing done for each condition. Shown is the number of sequence tags generated for each condition (i.e., two biological replicates, pooled). Of the reads labeled as &#x201C;unaligned,&#x201D; some still provide useful information as poly-A tags." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392347016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequencing done for each condition. Shown is the number of sequence tags generated for each condition (&lt;em&gt;i.e.&lt;/em&gt;, two biological replicates, pooled). Of the reads labeled as &#x201C;unaligned,&#x201D; some still provide useful information as poly-A tags.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/343/F1.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/343/F1.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/343/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/343/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-23" class="first-child">Sequencing done for each condition. Shown is the number of sequence tags generated for each condition (<em>i.e.</em>, two biological replicates, pooled). Of the reads labeled as “unaligned,” some still provide useful information as poly-A tags.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Approximately 93% of the yeast genome can be uniquely mapped with 76-bp reads. Most of the yeast genome is expressed, consistent with previous studies (<em>e.g.</em>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">David <em>et al.</em> 2006</a>; <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>), and the data in this study demonstrate that approximately 80% (range 77–87%) of the base pairs in the yeast genome are expressed from one or both strands (<em>i.e.</em>, as a percentage of the 12-Mb genome) at a depth of five sequencing reads of coverage or more per base (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FigureS1.pdf">Figure S1</a>). At the same level of coverage, approximately 45% (range 40–51%) of the base pairs are expressed as a percentage of the 24-Mb genome, <em>i.e.</em>, separating out the Watson and Crick strands. Although we attempted to minimize potential artifacts from second-strand synthesis from reverse transcriptase, it is still possible that some of the observed transcription might be due to reverse transcription. The RNA-Seq method used in this study did not yield lower levels of antisense transcription when actinomycin D was used (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Parkhomchuk <em>et al.</em> 2009</a>), unlike that found in previous studies (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Perocchi <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Levin <em>et al.</em> 2010</a>). Examination of annotated genes, however, revealed that transcription from one strand was accompanied by only a fraction of that amount of transcription occurring on the opposite strand. The median gene’s ratio of sense to anti-sense transcription was 5.1 to 6.4 log-two power (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FigureS3.pdf">Figure S3</a>). These results indicate that most of the yeast genome is truly expressed under different conditions and that some true antisense transcription is present, as expected (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xu <em>et al.</em> 2009</a>).</p></div><div id="sec-4" class="subsection"><h3>Most yeast genes are differentially expressed</h3><p id="p-25">We analyzed gene expression across the 18 conditions. Of 6058 mappable genes (<em>i.e.</em>, not in repetitive regions), we found that 5958 genes were expressed in at least one condition (with a threshold of each ORF base detected five or more times on average); 100 genes fell below this threshold, and 85 of these were classified as “dubious.” Interestingly, 611 dubious genes are expressed at or above our threshold.</p><p id="p-26">Analysis of differential expression for the 5362 genes which are both mappable and nondubious was done using DESeq (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Anders and Huber 2010</a>). Using a binomial test (<em>P</em> ≤ 0.01), the majority (5245; 97.8%) was differentially expressed relative to exponential growth in YPAD media in at least one condition. Adding a criterion of 1.5-fold enrichment reduced this number to 4800 (89.5%), and this figure further reduces to 3421 (63.8%) when a 2-fold threshold is used. Interestingly, only 117 genes never differ in their overall RNA levels when we use the 1.5-efold enrichment threshold, listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS9.xls">Table S9</a>. These are enriched for the GO categories “biological process,” “intron homing,” “cellular component,” “molecular function,” and “endonuclease activity” (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ashburner <em>et al.</em> 2000</a>) at a <em>P</em>-value of 0.05 (corrected) against a background set of mappable genes; the GO::TermFinder software suite (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Boyle <em>et al.</em> 2004</a>) was used to calculate enrichment.</p><p id="p-27">GO-enrichment analysis of the genes that are differentially expressed using the binomial test revealed largely expected results, <em>e.g.</em>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006108">alpha factor</a>−stimulated cells had as their top five categories “conjugation,” “conjugation with cellular fusion,” “sexual reproduction,” “interaction between organisms,” and “response to pheromone.” Similarly, the top five categories for Congo red stimulation, which damages cell walls, were “cell wall organization and biogenesis,” “external encapsulating structure organization and biogenesis,” “cell morphogenesis,” “morphogenesis,” and “anatomical structure development”; one of the top categories for heat shocked cells was “protein folding” (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS2.xls">Table S2</a> for complete results).</p><p id="p-28">Interestingly, cells grown in commercial grape juice had as their top GO category “vitamin metabolism,” and the 41 genes in this category contained many vitamin biosynthetic genes related to thiamine, nicotinamide adenine dinucleotide/nicotinamide adenine dinucleotide phosphate, flavin, and carnitine, and are aimed at rapidly processing sugar. Two genes among these, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003030"><em>PYC1</em></a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000422"><em>PYC2</em></a>, are associated with lactic acidosis when their human homolog is mutated [gene annotations taken from the Saccharomyces Genome Database (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cherry <em>et al.</em> 1998</a>)].</p></div><div id="sec-5" class="subsection"><h3>Extensive differences in 5′ ends in stress conditions</h3><p id="p-29">We were particularly interested in examining differential transcript end diversity in yeast. A custom software pipeline was developed for the identification of differentially expressed transcript ends. This pipeline compared signal tracks for the exponential growth condition to every other condition, one at a time, and called differential ends when the difference in signal between the two conditions exceeded a stringent threshold calculated for each gene based on internal signal noise (see <em>Materials and Methods</em>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FigureS4.pdf">Figure S4</a>). As transcriptional landscapes were directly compared, the term “topology comparisons” was used to describe this approach. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS3.xls">Table S3</a> contains the complete list of alternate ends.</p><p id="p-30">Extensive differences were observed in 5′ and 3′ ends, and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> summarizes the number of genes with differential ends. Relative to cells grown in rich medium, we find 613 cases of longer 5′ ends and 598 instances of shorter 5′ ends across the conditions, affecting 471 and 364 unique ORFs, respectively. For some conditions (<em>e.g.</em>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006108">alpha factor</a>) cells exhibit minimal use of alternative 5′ ends, and, of those that do, the differential 5′ ends most often are shorter relative to cells grown in rich medium, that is, the transcription start site is closer to the start codon for translation for the nonrich medium condition.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/343/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of differential ends. Shown is the number of genes with differential longer and shorter ends on either the 5&#x2032; or 3&#x2032; side. All comparisons were with the exponential growth condition." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392347016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of differential ends. Shown is the number of genes with differential longer and shorter ends on either the 5&#x2032; or 3&#x2032; side. All comparisons were with the exponential growth condition.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/343/F2.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/343/F2.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/343/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/343/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-31" class="first-child">Number of differential ends. Shown is the number of genes with differential longer and shorter ends on either the 5′ or 3′ side. All comparisons were with the exponential growth condition.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Two conditions, salt stimulation and stationary phase, show an abundance of transcripts with increased 5′ UTR size, with more than 200 genes each expressing a longer variant of its transcript relative to growth in rich medium. Of 15 transcripts with an alternative 5′ UTR longer than 300 base pairs, seven were found in stationary phase and four under salt stimulation. The longest alternative 5′ UTR, for the gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003818">CDC8</a></em>, was 853 base pairs longer in cells in stationary phase.</p><p id="p-33">One interesting effect is when a differential transcript is shortened to the point that the annotated start codon is not transcribed. This is observed in 21 instances and 18 unique genes across 10 conditions; the affected genes are <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004660">AAC1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004769">ATG16</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001078">BRL1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003382">CCM1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003527">COS6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001158">COX23</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002924">EMI2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001359">FYV10</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003025">GEP7</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006012">GLR1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004990">LAP2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006105">MRN1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001545">MSN4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004509">PRP39</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000277">RDH54</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001418">UBP7</a></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS1.pdf">Table S1</a>). Of these, only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a></em> has been characterized previously as a gene that encodes two transcript forms that produce proteins with differential amino termini (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Carlson and Botstein 1982</a>).</p></div><div id="sec-6" class="subsection"><h3>Upstream ORFs lie in many differential 5′ ends</h3><p id="p-34">Examining differential 5′ ends identified across all conditions revealed a total of 614 instances of uORFs in the differential transcribed region (see <em>Materials and Methods</em>). These affect 446 transcripts and a total of 543 unique uORFs (47 of which consist of solely a start and stop codon, with no intervening codon). These results are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS4.xlsx">Table S4</a>.</p><p id="p-35">We next compared the number of uORFs found in differential ends to the number expected by chance in differential ends of the same sizes, using random sampling of promoter sequences within 1000 bp of annotated start codons in the yeast genome. We found that for cells incubated in the presence of high salt, transcripts were statistically enriched for uORFs in the differential 5′ ends (z-value &gt; 3). Interestingly, there was no statistical enrichment of uORFs in shortened 5′ UTRs in salt, but only in longer 5′ UTRs. Of the 222 genes with a longer alternative 5′ UTR in the salt response, 104 had one or more uORFs whereas a total of 148 uORFs were introduced in the longer-form UTR, where only 107 (std. dev.: 9.7) would have been expected by chance alone (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS5.xlsx">Table S5</a>). Overall, these results indicate that the differential 5′ ends, particularly long ones, are found under cell growth in extreme conditions such as high salt.</p></div><div id="sec-7" class="subsection"><h3>Diverse roles of differential uORFs in salt response</h3><p id="p-36">To further analyze the role of differential uORFs in gene expression, we selected five genes with uORFs for further study, and fused their upstream sequences from the gene’s start codon to the nearest neighboring gene upstream of a firefly luciferase reporter. A <em>Renilla</em> luciferase gene fused to a constitutive promoter was used as a loading control for these same cells (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">McNabb and Reed 2005</a>; plasmid pTH650 described in <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chu <em>et al.</em> 2011</a>). A version of the upstream sequence in which the “in register uORFs” had been removed also was produced based on sequential AUGs, where the AUG start codon was changed to UUG (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> for an explanation). In each case, the longest uORFs and the first accessible uORFs were removed. Results of the luciferase assays are presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/343/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="uORFs knockouts. Illustration of the manner in which uORFs were mutated. The upper track shows expression, the dotted line under exponential growth in YPAD, and the dashed line under salt stimulation conditions. The lower track shows the main ORF in solid black and the knocked-out uORFs in light gray. Out-of-register ORFs, which were not knocked out, are shown in dark gray. (A) ATP18; (B) GID7. In the case of GID7, the shorter of those two out-of-register ORFs is conserved across Saccharomyces paradoxus, Saccharomyces mikatae, and Saccharomyces bayanus (see sequence data from Cliften et al. 2003). Please also note that the exact insert placed into the plasmid ran from the far left of each panel to the beginning of the main ORF; the latter was replaced with luciferase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392347016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;uORFs knockouts. Illustration of the manner in which uORFs were mutated. The upper track shows expression, the dotted line under exponential growth in YPAD, and the dashed line under salt stimulation conditions. The lower track shows the main ORF in solid black and the knocked-out uORFs in light gray. Out-of-register ORFs, which were not knocked out, are shown in dark gray. (A) &lt;em&gt;ATP18&lt;/em&gt;; (B) &lt;em&gt;GID7&lt;/em&gt;. In the case of &lt;em&gt;GID7&lt;/em&gt;, the shorter of those two out-of-register ORFs is conserved across &lt;em&gt;Saccharomyces paradoxus&lt;/em&gt;, &lt;em&gt;Saccharomyces mikatae&lt;/em&gt;, and &lt;em&gt;Saccharomyces bayanus&lt;/em&gt; (see sequence data from Cliften &lt;em&gt;et al.&lt;/em&gt; 2003). Please also note that the exact insert placed into the plasmid ran from the far left of each panel to the beginning of the main ORF; the latter was replaced with luciferase.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/343/F3.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/343/F3.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/343/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/343/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462456" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-37" class="first-child">uORFs knockouts. Illustration of the manner in which uORFs were mutated. The upper track shows expression, the dotted line under exponential growth in YPAD, and the dashed line under salt stimulation conditions. The lower track shows the main ORF in solid black and the knocked-out uORFs in light gray. Out-of-register ORFs, which were not knocked out, are shown in dark gray. (A) <em>ATP18</em>; (B) <em>GID7</em>. In the case of <em>GID7</em>, the shorter of those two out-of-register ORFs is conserved across <em>Saccharomyces paradoxus</em>, <em>Saccharomyces mikatae</em>, and <em>Saccharomyces bayanus</em> (see sequence data from <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cliften <em>et al.</em> 2003</a>). Please also note that the exact insert placed into the plasmid ran from the far left of each panel to the beginning of the main ORF; the latter was replaced with luciferase.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/343/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Luciferase analysis of uORFs. Luciferase assays confirming the relevance of uORFs in four genes with longer alternative 5&#x2032; UTRs in the salt stress condition. (A) ATP18; (B) HEM3; (C) GID7; and (D) TID3. The left two bars show results in SC&#x2013;ura media and the right two bars show results in SC&#x2013;ura media with 1 M salt for 1 hr. White bars are the results using the leader sequences, and gray bars have had uORF start codons mutated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392347016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Luciferase analysis of uORFs. Luciferase assays confirming the relevance of uORFs in four genes with longer alternative 5&#x2032; UTRs in the salt stress condition. (A) ATP18; (B) HEM3; (C) GID7; and (D) TID3. The left two bars show results in SC&#x2013;ura media and the right two bars show results in SC&#x2013;ura media with 1 M salt for 1 hr. White bars are the results using the leader sequences, and gray bars have had uORF start codons mutated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/343/F4.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/343/F4.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/343/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/343/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462454" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-38" class="first-child">Luciferase analysis of uORFs. Luciferase assays confirming the relevance of uORFs in four genes with longer alternative 5′ UTRs in the salt stress condition. (A) ATP18; (B) HEM3; (C) GID7; and (D) TID3. The left two bars show results in SC–ura media and the right two bars show results in SC–ura media with 1 M salt for 1 hr. White bars are the results using the leader sequences, and gray bars have had uORF start codons mutated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Of note, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002364">HEM3</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007247">ATP18</a></em> were expressed less in salt relative to YPAD according to the RNA-Seq data (0.6× and 0.5×, respectively), and the uORFs were found to play a repressive role, <em>i.e.</em>, the mutated, uORF-less construct was expressed greater. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000544">GID7</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001406">TID3</a></em>, however, were expressed greater in salt (2.5× more in both cases), and the plasmids with uORF knockouts express less reporter indicating that the uORFs for these genes played an activating role in protein expressed. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000971">RPH1</a></em> also was examined, although mutations analyses revealed no change indicating that its uORFs did not appear to play a significant role controlling gene expression.</p><p id="p-40">As a further control that the uORFs themselves were responsible for the translation changes and not a condition-specific splicing event (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheah <em>et al.</em> 2007</a>). TopHat (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Trapnell <em>et al.</em> 2009</a>) was used to analyze the transcripts for splicing events. No evidence of alternative splicing was found. Thus, we conclude that most uORFs affect gene expression; for some genes they activate gene expression and in other instances they repress gene expression</p></div><div id="sec-8" class="subsection"><h3>Extensive differences in mRNA 3′ ends</h3><p id="p-41">Analysis of 3′ ends revealed a number of genes with differential 3′ ends in each of the various conditions as determined by topology comparisons (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Relative to exponential growth in YPAD, there were 451 instances of longer 3′ ends, and 1991 instances of shorter 3′ ends across all conditions, representing 328 and 993 unique ORFs, respectively. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS3.xls">Table S3</a> contains the complete list of altered 3′ lengths. Yeast grown in grape juice was clearly distinct, with 764 genes having a shorter transcript. Interestingly, one gene, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003753">REE1</a></em>, was shortened in every condition relative to rich medium.</p><p id="p-42">In addition to topology comparisons, analysis of 377,263 poly-adenylated end tags in our data set can be used to analyze 3′ ends. These are sequencing reads with a nongenomic run of A bases at the ends of mappable sequences are presumed to be a poly-adenylation event and thus mark the end of a transcript. These reads are recovered from the population of reads that did not initially map to the yeast genome, as the poly-adenylated tail will prevent alignment. Determining differential ends using this approach from our sequencing method is sensitive to thresholding and vulnerable to noise from low sampling levels; with 377,263 such reads across 18 conditions and a 6000 gene organism, only 3.5 reads per gene would be expected in each condition, assuming a uniform distribution of gene expression levels. In addition, poly-adenylated tags may be hampered by artifacts due to oligo-dT primer ligation to A-rich genomic sequences. Topology comparisons are thus relied upon as the main approach in this study.</p><p id="p-43">Ozsolak <em>et al.</em> (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Ozsolak <em>et al.</em> 2010</a>) used poly-adenylation to call transcript ends because their protocol only sequenced poly-adenylated regions of the transcriptome and does not suffer from undersampling. They found 91,891 poly-adenylation sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FileS1.pdf">File S1</a>), and reported that 72.1% of yeast genes have at least two poly-adenylation sites further than 50 bp apart.</p><p id="p-44">Pooling the results for all conditions in our RNA-Seq data, we found that 171 genes had more than 10 end tags. Of those, 128 (74.9%) had two or more discrete poly-adenylation start sites, indicating variability in 3′ end structure. Among our interesting results, we found one gene, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">CDC19</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">PYK1</a></em> (pyruvate kinase), which has an alternative 3′ end (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Rapid amplification of cDNA ends polymerase chain reaction (RACE PCR) confirmed the existence of three distinct ends, one of which is visible only as an expression drop, another only from poly-adenylation tags, and a third visible from both. In a study of 3′ UTRs in <em>Caenorhabditis elegans</em> using polyA capture, 3′ RACE, cDNA sequencing, and RNA-Seq, 27% of called ends were only called using one technique (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mangone <em>et al.</em> 2010</a>); thus, the use of multiple methods for calling ends is generally useful for 3′ end mapping.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/343/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pyruvate kinase differential end validation. (A) The top track shows the expression profile for CDC19/PYK1 (pyruvate kinase) under exponential growth in YPAD and grape juice conditions. The middle track shows the number of sequenced end tags from the study as a whole. The bottom track shows the annotated ORF. The numbers 1, 2, and 3 show Sanger sequencing results from the 3&#x2032; RACE bands in part B. (B) Gel showing the results of 3&#x2032; RACE PCR for CDC19. The ladder shows the 600- and 500-bp markers. The center lane is the result for RNA from the exponential growth condition, and the right lane is from grape juice. The bands labeled 1, 2, and 3 were sequenced and correspond to the marked locations in part A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392347016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pyruvate kinase differential end validation. (A) The top track shows the expression profile for &lt;em&gt;CDC19&lt;/em&gt;/&lt;em&gt;PYK1&lt;/em&gt; (pyruvate kinase) under exponential growth in YPAD and grape juice conditions. The middle track shows the number of sequenced end tags from the study as a whole. The bottom track shows the annotated ORF. The numbers 1, 2, and 3 show Sanger sequencing results from the 3&#x2032; RACE bands in part B. (B) Gel showing the results of 3&#x2032; RACE PCR for &lt;em&gt;CDC19&lt;/em&gt;. The ladder shows the 600- and 500-bp markers. The center lane is the result for RNA from the exponential growth condition, and the right lane is from grape juice. The bands labeled 1, 2, and 3 were sequenced and correspond to the marked locations in part A.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/343/F5.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/343/F5.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/343/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/343/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462323" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-45" class="first-child">Pyruvate kinase differential end validation. (A) The top track shows the expression profile for <em>CDC19</em>/<em>PYK1</em> (pyruvate kinase) under exponential growth in YPAD and grape juice conditions. The middle track shows the number of sequenced end tags from the study as a whole. The bottom track shows the annotated ORF. The numbers 1, 2, and 3 show Sanger sequencing results from the 3′ RACE bands in part B. (B) Gel showing the results of 3′ RACE PCR for <em>CDC19</em>. The ladder shows the 600- and 500-bp markers. The center lane is the result for RNA from the exponential growth condition, and the right lane is from grape juice. The bands labeled 1, 2, and 3 were sequenced and correspond to the marked locations in part A.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Although the effects were often less dramatic than those seen at the 5′ end, expression at the 3′ end can decrease to levels where the main stop codon is not part of the dominant form of the transcript, or not present at all in the data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FigureS2.pdf">Figure S2</a>B shows the 3′ end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001563">VMA5</a></em> gene, which does not appear to have a stop codon at all in synthetic complete media. Under exponential growth, there are 32 poly-adenylated end tags near <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001563">VMA5</a></em> between genomic co-ordinates 285,831 and 285,837 on chromosome XI, although there is clearly some expression past that point, which would expose the annotated stop codon in a minority of transcripts.</p><p id="p-47">In another example, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002844">PPZ2</a></em>, which encodes a phosphatase involved in regulating cell cycle progression and shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/FigureS2.pdf">Figure S2</a>A, the transcript level for the gene as a whole increases by 2.2-fold in the shift from exponential growth to stationary phase, but the level of the differential end, called algorithmically from genomic co-ordinates 1,336,923 to 1,336,988 on chromosome IV, increases by 15.8-fold (after we adjusted for sequencing library size). A total of 97 other instances of a shortened 3′ end were found using topology comparisons, represented in 63 unique ORFs in examples from all 17 non-exponential growth conditions. They are listed in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS6.xls">Table S6</a>. Thus, many yeast transcripts may lack a translation termination codon.</p><p id="p-48">The differential ends called by topology comparisons were examined for RBP motifs using the motifs published in (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Riordan <em>et al.</em> 2011</a>). Most conditions were statistically enriched for RBP motifs among the 3′ sequences that were shortened relative to rich YPD medium, with the exception of the oxidative stress and sorbitol conditions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1/TableS7.xlsx">Table S7</a>). However, only five conditions were enriched for RBPs among the lengthened 3′ sequences (grape juice, oxidative stress, heat shock, sorbitol, and stationary phase). Statistical enrichment was calculated, similarly to that of uORFs, based on random sampling from the yeast genome of sequences of equal length to the differential ends, and a z-value cutoff greater than 3. Thus, transcripts with differential ends are likely to affect differences in RNA binding proteins and presumably gene regulation.</p></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><div id="sec-10" class="subsection"><h3>Prematurely terminated transcripts</h3><p id="p-49">One particularly surprising finding of our study is that 63 ORFs were found which, under at least one condition, showed a 3′ end that likely terminated before the annotated stop codon. These transcripts would be left without a stop codon, potentially triggering a nonstop decay of the mRNA (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Frischmeyer <em>et al.</em> 2002</a>) and/or production of a polylysine tail on the protein that is produced. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002844">PPZ2</a></em>, whose transcript exhibits this property, is a serine/threonine phosphatase involved in regulating cell-cycle progression, so its appearance in the stationary phase condition is of interest. The phenomenon of premature poly-adenylation has been noted before and described as an attenuating regulator of human mobile elements (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Perepelitsa-Belancio and Deininger 2003</a>), and the nonstop decay pathway has been described in yeast (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Frischmeyer <em>et al.</em> 2002</a>). Although the pathway is understood, there is little understanding of how prevalent it is (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garneau <em>et al.</em> 2007</a>). This study indicates the particular genes that may be controlled in this fashion and that the phenomenon made be more widespread that previously appreciated.</p></div><div id="sec-11" class="subsection"><h3>5′ ORF truncations</h3><p id="p-50">Of the 18 genes found to have 5′ ORF truncations, <em>i.e.</em>, the dominant form of the transcript did not contain the annotated start codon, one, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a></em>, has been extensively characterized previously. For <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a></em> the long form contains the signal sequence that causes the sucrose invertase to be secreted (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Carlson and Botstein 1982</a>). Carlson and Botstein found that glucose-repressed cells produce a longer form of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a></em>, which is consistent with the results of our study, in which the RNA-Seq data show a longer transcript present in media using glycerol as the carbon source. In addition to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424"><em>SUC2</em></a>, 17 other genes produce shorter proteins. Only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a></em>, however, is predicted to encode a protein that contains a signal sequence that is missing in the shorter transcript [performed using SignalP 4.0 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Petersen <em>et al.</em> 2011</a>)]. The alternative amino terminal sequence in these 17 instances may affect protein function in other ways.</p></div><div id="sec-12" class="subsection"><h3>RBP motifs</h3><p id="p-51">In 2442 differential 3′ ends (1991 shortened and 451 lengthened) across 17 non-YPAD exponential growth conditions, 6420 RBP motifs were present (motifs from <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Riordan <em>et al.</em> 2011</a>). RBP motifs are, to date, quite degenerate in the method they are defined, and this likely leads to many false positives. Nevertheless, statistical enrichment was more often seen in shortened 3′ ends, suggesting that cells may remove the layer of control offered by RBPs under stress conditions.</p></div><div id="sec-13" class="subsection"><h3>uORFs</h3><p id="p-52">Upstream ORFs are a known regulatory feature. The availability of genomic sequences for many organisms, yeast since 1996 (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Goffeau <em>et al.</em> 1996</a>), has allowed for a theoretical search for uORFs, and a large abundance has been found. Whether they have meaning as a method for translational control depends first and foremost on whether they are transcribed; these data have been harder to produce. Using a random lacZ insertion method, Burns <em>et al.</em> and Ross-Macdonald <em>et al.</em> revealed extensive translation of short and out of frame ORF expression through the yeast genome (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burns <em>et al.</em> 1994</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ross-Macdonald <em>et al.</em> 1999</a>). In a recent RNA-Seq study, <a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> (2008)</a> concluded that 6% of genes had uORFs. Estimates vary and range up to 20% (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hood <em>et al.</em> 2009</a>). Our data indicate that many of these lie in differential UTRs.</p><p id="p-53">Two broad mechanisms of uORF action are known (see <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vilela and MCcarthy 2003</a>): control by uORFs modulating posttermination behavior of ribosomes and control modulated by the actual peptide produced by a uORF. The canonical example of the first is <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000735">GCN4</a></em>, characterized in (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mueller and Hinnebusch 1986</a>), and of the second the arginine attenuator peptide found before the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005829">CPA1</a></em> gene (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gaba <em>et al.</em> 2001</a>).</p><p id="p-54">Even if transcribed, whether uORFs have regulatory function further depends on their interaction with the translational machinery. Genes such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000735">GCN4</a></em> (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Mueller and Hinnebusch 1986</a>), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005829">CPA1</a></em> (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gaba <em>et al.</em> 2001</a>), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004466">YAP1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002831">YAP2</a></em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vilela <em>et al.</em> 1998</a>) are well characterized, and further examples do exist. Some, however, show no effect [see (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhang and Dietrich 2005</a>) where not all validate], and some show an effect dependent on RNA processing [see (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Zhang and Dietrich 2005</a>), for an example involving riboswitches in <em>N. crassa</em>]. Most studies to date have used bioinformatic approaches solely and analyze sequence context and/or conservation, which does not offer proof of uORF relevance [see <em>e.g.</em>, (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cvijović <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lawless <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Selpi <em>et al.</em> 2009</a>), all in <em>S. cerevisiae</em>]. In our study, in which we performed luciferase validation experiments, we examine five leader sequences with uORFs, only four of which had a direct impact on luciferase levels (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Zhang and Dietrich 2005</a>). In another study, authors analyzed the ribosome footprint to mRNA ratios in meiosis and concluded that uORFs in differential ends could alter translational efficiency of mRNA. They also reported that in the case of uORFs with an AUG start codon, the translational efficiency and ribosome occupancy showed a “strong negative correlation” (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brar <em>et al.</em> 2012</a>). A subsequent microarray-based study of meiotic yeast confirmed the presence of abundant transcript architecture changes in meiotic and sporulating yeast (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kim Guisbert <em>et al.</em> 2012</a>).</p><p id="p-55">In this study, there was only a statistical enrichment for uORFs relative to promoter sequences in longer 5′ ends in the salt condition. Under no conditions was a statistically significant enrichment of uORFs present in shortened 5′ UTRs, implying that normally used uORFs are seldom removed by alternative promoter choices.</p><p id="p-56">In each of the four cases of a uORF affecting translation reported in (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Zhang and Dietrich 2005</a>), removing the uORFs led to increased luciferase activity. In this study, a new phenomenon is reported: uORFs which increase luciferase activity (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000544">GID7</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001406">TID3</a></em>; see <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). That uORFs might increase translational efficiency has been hypothesized (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Neafsey and Galagan 2007</a>; see also <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hood <em>et al.</em> 2009</a>), but no previous examples have been found or directed demonstrated using mutational analyses. One possibly is that these uORFs have RNA binding sites or ribosome entry sites that attracting ribosomes to those mRNAs.</p><p id="p-57">Overall, this study demonstrates pervasive differential transcript ends occur in yeast and that in a least several cases these differences likely affect gene expression. The mechanisms by which differential ends are selected remains to be determined, and such information will be important to obtain a details understanding of how eukaryotic gene expression is controlled.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">Jennifer Li-Pook-Than and Jennifer E.G. Gallagher provided comments on the manuscript. This research was supported by National Institute of Health grants NIH P50 HG 002357 and NIH/CA077808.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003640/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-59"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-02"><span class="received-label">Received </span>July 2, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-15"><span class="accepted-label">Accepted </span>December 15, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Waern, Snyder</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.1" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.2" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene Ontology: tool for the unification of biology</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.3" data-doi="10.1093/bioinformatics/bth456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GO:TermFinder—open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3710</span>–<span class="cit-lpage">3715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth456%26rft_id%253Dinfo%253Apmid%252F15297299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTgvMzcxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.4" data-doi="10.1126/science.1215110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>. <span class="cit-article-title">High-resolution view of the yeast meiotic program revealed by ribosome profiling</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1215110%26rft_id%253Dinfo%253Apmid%252F22194413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2OC81NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.5" data-doi="10.1101/gad.8.9.1087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwade</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Macdonald</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Large-scale analysis of gene expression, protein localization, and gene disruption in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBurns%26rft.auinit1%253DN%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D1087%26rft.epage%253D1105%26rft.atitle%253DLarge-scale%2Banalysis%2Bof%2Bgene%2Bexpression%252C%2Bprotein%2Blocalization%252C%2Band%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.9.1087%26rft_id%253Dinfo%253Apmid%252F7926789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiOC85LzEwODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.6" data-doi="10.1016/0092-8674(82)90384-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Two differentially regulated mRNAs with different 5′ ends encode secreted with intracellular forms of yeast invertase</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarlson%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D154%26rft.atitle%253DTwo%2Bdifferentially%2Bregulated%2BmRNAs%2Bwith%2Bdifferent%2B5%2527%2Bends%2Bencode%2Bsecreted%2Bwith%2Bintracellular%2Bforms%2Bof%2Byeast%2Binvertase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252882%252990384-1%26rft_id%253Dinfo%253Apmid%252F7039847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(82)90384-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7039847&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982MY42100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.7" data-doi="10.1038/nature05769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheah</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudarsan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breaker</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Control of alternative RNA splicing and gene expression by eukaryotic riboswitches</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCheah%26rft.auinit1%253DM.%2BT.%26rft.volume%253D447%26rft.issue%253D7143%26rft.spage%253D497%26rft.epage%253D500%26rft.atitle%253DControl%2Bof%2Balternative%2BRNA%2Bsplicing%2Band%2Bgene%2Bexpression%2Bby%2Beukaryotic%2Briboswitches.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05769%26rft_id%253Dinfo%253Apmid%252F17468745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05769&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17468745&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246693100052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.8" data-doi="10.1093/nar/26.1.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chervitz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwight</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">SGD: Saccharomyces Genome Database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.1.73%26rft_id%253Dinfo%253Apmid%252F9399804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.9" data-doi="10.1093/nar/gkr300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von der Haar</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The role of tRNA and ribosome competition in coupling the expression of different mRNAs in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">6705</span>–<span class="cit-lpage">6714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr300%26rft_id%253Dinfo%253Apmid%252F21558172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xNS82NzA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.10" data-doi="10.1126/science.1084337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cliften</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudarsanam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desikan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Finding functional features in <em>Saccharomyces genomes</em> by phylogenetic footprinting</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1084337%26rft_id%253Dinfo%253Apmid%252F12775844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMDEvNTYyOS83MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coelho</span>  <span class="cit-name-given-names">P. S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadel</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A novel mitochondrial protein, Tar1p, is encoded on the antisense strand of the nuclear 25S rDNA</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft.volume%253D16%26rft.spage%253D656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.12" data-doi="10.1186/1471-2105-8-295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cvijović</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalevi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilsland</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">G. J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunnerhagen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification of putative regulatory upstream ORFs in the yeast genome using heuristics and evolutionary conservation</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCvijovic%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.spage%253D295%26rft.epage%253D295%26rft.atitle%253DIdentification%2Bof%2Bputative%2Bregulatory%2Bupstream%2BORFs%2Bin%2Bthe%2Byeast%2Bgenome%2Busing%2Bheuristics%2Band%2Bevolutionary%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-295%26rft_id%253Dinfo%253Apmid%252F17686169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-295&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17686169&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granovskaia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toedling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palm</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution map of transcription in the yeast genome</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">5320</span>–<span class="cit-lpage">5325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D103%26rft.spage%253D5320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.14" data-doi="10.1126/science.1067338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frischmeyer</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Hoof</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerrerio</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An mRNA surveillance mechanism that eliminates transcripts lacking termination codons</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">295</span>: <span class="cit-fpage">2258</span>–<span class="cit-lpage">2261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFrischmeyer%26rft.auinit1%253DP.%2BA.%26rft.volume%253D295%26rft.issue%253D5563%26rft.spage%253D2258%26rft.epage%253D2261%26rft.atitle%253DAn%2BmRNA%2BSurveillance%2BMechanism%2BThat%2BEliminates%2BTranscripts%2BLacking%2BTermination%2BCodons%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1067338%26rft_id%253Dinfo%253Apmid%252F11910109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTUvNTU2My8yMjU4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.15" data-doi="10.1093/emboj/20.22.6453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaba</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnamoorthy</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinnebusch</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Physical evidence for distinct mechanisms of translational control by upstream open reading frames</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">6453</span>–<span class="cit-lpage">6463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DGaba%26rft.auinit1%253DA.%26rft.volume%253D20%26rft.issue%253D22%26rft.spage%253D6453%26rft.epage%253D6463%26rft.atitle%253DPhysical%2Bevidence%2Bfor%2Bdistinct%2Bmechanisms%2Bof%2Btranslational%2Bcontrol%2Bby%2Bupstream%2Bopen%2Breading%2Bframes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.22.6453%26rft_id%253Dinfo%253Apmid%252F11707416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMjIvNjQ1MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.16" data-doi="10.1038/nrm2104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garneau</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilusz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilusz</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The highways and byways of mRNA decay</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DGarneau%26rft.auinit1%253DN.%2BL.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D113%26rft.epage%253D126%26rft.atitle%253DThe%2Bhighways%2Band%2Bbyways%2Bof%2BmRNA%2Bdecay.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2104%26rft_id%253Dinfo%253Apmid%252F17245413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17245413&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.17" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.18" data-doi="10.1074/jbc.M002523200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guittaut</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normand</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubois</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raimond</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of an internal gene to the human Galectin-3 gene with two different overlapping reading frames that do not encode Galectin-3</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">2652</span>–<span class="cit-lpage">2657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M002523200%26rft_id%253Dinfo%253Apmid%252F11160123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzYvNC8yNjUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.19" data-doi="10.1146/annurev.micro.62.081307.162835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hood</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neafsey</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs Matthew</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolutionary roles of upstream open reading frames in mediating gene regulation in fungi</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DHood%26rft.auinit1%253DH.%2BM.%26rft.volume%253D63%26rft.spage%253D385%26rft.epage%253D409%26rft.atitle%253DEvolutionary%2Broles%2Bof%2Bupstream%2Bopen%2Breading%2Bframes%2Bin%2Bmediating%2Bgene%2Bregulation%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.62.081307.162835%26rft_id%253Dinfo%253Apmid%252F19514854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.62.081307.162835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19514854&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270910300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.20" data-doi="10.1261/rna.030510.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim Guisbert</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flatow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurtado</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staley</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiosis-induced alterations in transcript architecture and noncoding RNA expression in <em>S. cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1142</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.030510.111%26rft_id%253Dinfo%253Apmid%252F22539527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE4LzYvMTE0MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.21" data-doi="10.1134/S0026893306050049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kochetov</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Alternative translation start sites and their significance for eukaryotic proteomes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%26rft.volume%253D40%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.1134%252FS0026893306050049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1134/S0026893306050049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.22" data-doi="10.1038/nbt0102-58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echols</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Serial analysis of gene expression</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKumar%26rft.auinit1%253DA.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D58%26rft.epage%253D63%26rft.atitle%253DAn%2Bintegrated%2Bapproach%2Bfor%2Bfinding%2Boverlooked%2Bgenes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0102-58%26rft_id%253Dinfo%253Apmid%252F11753363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0102-58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11753363&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173031600037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.23" data-doi="10.1016/j.tig.2003.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">J.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mager</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">B. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Complex controls: the role of alternative promoters in mammalian genomes</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLandry%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D640%26rft.epage%253D648%26rft.atitle%253DComplex%2Bcontrols%253A%2Bthe%2Brole%2Bof%2Balternative%2Bpromoters%2Bin%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2003.09.014%26rft_id%253Dinfo%253Apmid%252F14585616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2003.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14585616&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186530600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.24" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.25" data-doi="10.1186/gb-2005-6-13-r111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickel</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The undertranslated transcriptome reveals widespread translational silencing by alternative 5′ transcript leaders</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLaw%26rft.auinit1%253DG.%2BL.%26rft.volume%253D6%26rft.issue%253D13%26rft.spage%253DR111%26rft.epage%253DR111%26rft.atitle%253DThe%2Bundertranslated%2Btranscriptome%2Breveals%2Bwidespread%2Btranslational%2Bsilencing%2Bby%2Balternative%2B5%2527%2Btranscript%2Bleaders.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-13-r111%26rft_id%253Dinfo%253Apmid%252F16420678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-13-r111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16420678&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.26" data-doi="10.1186/1471-2164-10-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selley</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smirnova</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Upstream sequence elements direct post-transcriptional regulation of gene expression under stress conditions in yeast</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLawless%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DUpstream%2Bsequence%2Belements%2Bdirect%2Bpost-transcriptional%2Bregulation%2Bof%2Bgene%2Bexpression%2Bunder%2Bstress%2Bconditions%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-7%26rft_id%253Dinfo%253Apmid%252F19128476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19128476&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.27" data-doi="10.1038/msb.2011.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topper</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubler</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hose</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dynamic model of proteome changes reveals new roles for transcript alteration in yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DM.%2BV.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D514%26rft.epage%253D514%26rft.atitle%253DA%2Bdynamic%2Bmodel%2Bof%2Bproteome%2Bchanges%2Breveals%2Bnew%2Broles%2Bfor%2Btranscript%2Balteration%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.48%26rft_id%253Dinfo%253Apmid%252F21772262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.28" data-doi="10.1038/nmeth.1491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adiconis</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comprehensive comparative analysis of strand-specific RNA sequencing methods</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLevin%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D709%26rft.epage%253D715%26rft.atitle%253DComprehensive%2Bcomparative%2Banalysis%2Bof%2Bstrand-specific%2BRNA%2Bsequencing%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1491%26rft_id%253Dinfo%253Apmid%252F20711195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20711195&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281429200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.29" data-doi="10.1126/science.1191244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangone</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoharan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The landscape of <em>C. elegans</em> 3′UTRs</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1191244%26rft_id%253Dinfo%253Apmid%252F20522740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5MC80MzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.30" data-doi="10.1128/EC.4.9.1539-1549.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McNabb</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Dual luciferase assay system for rapid assessment of gene expression in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1539</span>–<span class="cit-lpage">1549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DMcNabb%26rft.auinit1%253DD.%2BS.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1539%26rft.epage%253D1549%26rft.atitle%253DDual%2BLuciferase%2BAssay%2BSystem%2Bfor%2BRapid%2BAssessment%2Bof%2BGene%2BExpression%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.9.1539-1549.2005%26rft_id%253Dinfo%253Apmid%252F16151247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI0LzkvMTUzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.31" data-doi="10.1186/gb-2002-3-3-reviews0004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mignone</span>  <span class="cit-name-given-names">F</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gissi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liuni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pesole</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Untranslated regions of mRNAs</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">REVIEWS0004.1</span>–<span class="cit-lpage">0004.10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D3%26rft.spage%253DREVIEWS0004.1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2002-3-3-reviews0004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2002-3-3-reviews0004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.32" data-doi="10.1073/pnas.0605645103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sese</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A large-scale full-length cDNA analysis to explore the budding yeast transcriptome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">17846</span>–<span class="cit-lpage">17851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605645103%26rft_id%253Dinfo%253Apmid%252F17101987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ3LzE3ODQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.33" data-doi="10.1016/0092-8674(86)90384-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinnebusch</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Multiple upstream AUG codons mediate translational control of GCN4</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMueller%26rft.auinit1%253DP.%2BP.%26rft.volume%253D45%26rft.issue%253D2%26rft.spage%253D201%26rft.epage%253D207%26rft.atitle%253DMultiple%2Bupstream%2BAUG%2Bcodons%2Bmediate%2Btranslational%2Bcontrol%2Bof%2BGCN4.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990384-3%26rft_id%253Dinfo%253Apmid%252F3516411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90384-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3516411&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986C097000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.34" data-doi="10.1126/science.1158441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>. <span class="cit-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1344</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158441%26rft_id%253Dinfo%253Apmid%252F18451266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karl</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RNA-Seq: a method for comprehensive transcriptome analysis</span>. <abbr class="cit-jnl-abbrev">Curr. Protoc. Mol. Biol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">4.11.1</span>–<span class="cit-lpage">4.11.13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMol.%2BBiol.%26rft.volume%253D89%26rft.spage%253D4.11.1%26rft.atitle%253DRNA-SEQ%253A%2BA%2BMETHOD%2BFOR%2BCOMPREHENSIVE%2BTRANSCRIPTOME%2BANALYSIS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.36" data-doi="10.1093/molbev/msm093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neafsey</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dual modes of natural selection on upstream open reading frames</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1744</span>–<span class="cit-lpage">1751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm093%26rft_id%253Dinfo%253Apmid%252F17494029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTc0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.37" data-doi="10.1016/j.cell.2010.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozsolak</span>  <span class="cit-name-given-names">F</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapranov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foissac</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishilevich</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comprehensive polyadenylation site maps in yeast and human reveal pervasive alternative polyadenylation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1018</span>–<span class="cit-lpage">1029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOzsolak%26rft.auinit1%253DF.%26rft.volume%253D143%26rft.issue%253D6%26rft.spage%253D1018%26rft.epage%253D1029%26rft.atitle%253DComprehensive%2Bpolyadenylation%2Bsite%2Bmaps%2Bin%2Byeast%2Band%2Bhuman%2Breveal%2Bpervasive%2Balternative%2Bpolyadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.11.020%26rft_id%253Dinfo%253Apmid%252F21145465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21145465&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285182600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.38" data-doi="10.1093/nar/gkp596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkhomchuk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amstislavskiy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banaru</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcriptome analysis by strand-specific sequencing of complementary DNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp596%26rft_id%253Dinfo%253Apmid%252F19620212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8xOC9lMTIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.39" data-doi="10.1038/ng1269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perepelitsa-Belancio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deininger</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">RNA truncation by premature polyadenylation attenuates human mobile element activity</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPerepelitsa-Belancio%26rft.auinit1%253DV.%26rft.volume%253D35%26rft.issue%253D4%26rft.spage%253D363%26rft.epage%253D366%26rft.atitle%253DRNA%2Btruncation%2Bby%2Bpremature%2Bpolyadenylation%2Battenuates%2Bhuman%2Bmobile%2Belement%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1269%26rft_id%253Dinfo%253Apmid%252F14625551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14625551&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186791000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.40" data-doi="10.1093/nar/gkm683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perocchi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clauder-Münster</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz Lars</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Antisense artifacts in transcriptome microarray experiments are resolved by actinomycin D</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">e128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm683%26rft_id%253Dinfo%253Apmid%252F17897965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNS8xOS9lMTI4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzM0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.41" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.42" data-doi="10.1093/nar/gkq920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riordan</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of RNA recognition elements in the <em>Saccharomyces cerevisiae</em> transcriptome</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1501</span>–<span class="cit-lpage">1509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq920%26rft_id%253Dinfo%253Apmid%252F20959291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzQvMTUwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.43" data-doi="10.1038/46558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Macdonald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coelho</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roemer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Large-scale analysis of the yeast genome by transposon tagging and gene disruption</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">402</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRoss-Macdonald%26rft.auinit1%253DP.%26rft.volume%253D402%26rft.issue%253D6760%26rft.spage%253D413%26rft.epage%253D418%26rft.atitle%253DLarge-scale%2Banalysis%2Bof%2Bthe%2Byeast%2Bgenome%2Bby%2Btransposon%2Btagging%2Band%2Bgene%2Bdisruption.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F46558%26rft_id%253Dinfo%253Apmid%252F10586881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/46558&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10586881&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083913600057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.44" data-doi="10.1186/1471-2105-10-451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selpi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarv</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristiansson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting functional upstream open reading frames in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSelpi%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D451%26rft.epage%253D451%26rft.atitle%253DPredicting%2Bfunctional%2Bupstream%2Bopen%2Breading%2Bframes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-451%26rft_id%253Dinfo%253Apmid%252F20042076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20042076&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.45" data-doi="10.1038/nbt1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation DNA sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D1135%26rft.epage%253D1145%26rft.atitle%253DNext-generation%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1486%26rft_id%253Dinfo%253Apmid%252F18846087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846087&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259926000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.46" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.47" data-doi="10.1016/S1357-2725(98)00134-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Velden</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The role of the 5′ untranslated region of an mRNA in translation regulation during development</span>. <abbr class="cit-jnl-abbrev">Int J Biochem Cell Biol</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Binternational%2Bjournal%2Bof%2Bbiochemistry%2B%2526%2Bcell%2Bbiology%26rft.stitle%253DInt%2BJ%2BBiochem%2BCell%2BBiol%26rft.aulast%253Dvan%2Bder%2BVelden%26rft.auinit1%253DA.%2BW.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D106%26rft.atitle%253DThe%2Brole%2Bof%2Bthe%2B5%2527%2Buntranslated%2Bregion%2Bof%2Ban%2BmRNA%2Bin%2Btranslation%2Bregulation%2Bduring%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1357-2725%252898%252900134-4%26rft_id%253Dinfo%253Apmid%252F10216946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1357-2725(98)00134-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10216946&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078999700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.48" data-doi="10.1046/j.1365-2958.2003.03622.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vilela</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Regulation of fungal gene expression via short open reading frames in the mRNA 5′untranslated region</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DVilela%26rft.auinit1%253DC.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D859%26rft.epage%253D867%26rft.atitle%253DRegulation%2Bof%2Bfungal%2Bgene%2Bexpression%2Bvia%2Bshort%2Bopen%2Breading%2Bframes%2Bin%2Bthe%2BmRNA%2B5%2527untranslated%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03622.x%26rft_id%253Dinfo%253Apmid%252F12890013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03622.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12890013&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184936200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.49" data-doi="10.1093/nar/26.5.1150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vilela</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigues-Pousada</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The yeast transcription factor genes YAP1 and YAP2 are subject to differential control at the levels of both translation and mRNA stability</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1150</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.5.1150%26rft_id%253Dinfo%253Apmid%252F9469820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI2LzUvMTE1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.50" data-doi="10.1007/978-1-61779-173-4_8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA sequencing</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">759</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DWaern%26rft.auinit1%253DK.%26rft.volume%253D759%26rft.spage%253D125%26rft.epage%253D132%26rft.atitle%253DRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-173-4_8%26rft_id%253Dinfo%253Apmid%252F21863485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-173-4_8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21863485&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.51" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.52" data-doi="10.1038/nature07728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagneur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perocchi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clauder-Münster</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bidirectional promoters generate pervasive transcription in yeast</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DZ.%26rft.volume%253D457%26rft.issue%253D7232%26rft.spage%253D1033%26rft.epage%253D1037%26rft.atitle%253DBidirectional%2Bpromoters%2Bgenerate%2Bpervasive%2Btranscription%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07728%26rft_id%253Dinfo%253Apmid%252F19169243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19169243&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F343.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263425400047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.53" data-doi="10.1261/rna.2038810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">O. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Noncanonical transcript forms in yeast and their regulation during environmental stress</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1256</span>–<span class="cit-lpage">1267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2038810%26rft_id%253Dinfo%253Apmid%252F20421314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE2LzYvMTI1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8zNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.343.54" data-doi="10.1007/s00294-005-0001-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification and characterization of upstream open reading frames (uORF) in the 5′ untranslated regions (UTR) of genes in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Curr. Op. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOp.%2BGenet.%26rft.volume%253D48%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-005-0001-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-005-0001-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/2/343.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/333" title="A High Load of Non-neutral Amino-Acid Polymorphisms Explains High Protein Diversity Despite Moderate Effective Population Size in a Marine Bivalve With Sweepstakes Reproduction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/353" title="How to Design a Genetic Mating Scheme: A Basic Training Package for Drosophila Genetics" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/2"><span>Volume 3 Issue 2, February 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/2.cover-source.jpg?itok=h3kYZ-FP" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/343.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2UH_zn8tSTty0exafq13gpfqMKU8frGZmcQtFl2LGd0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/2/343" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/2/343" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/2/343" class="active" data-icon-position="" data-hide-link-title="0">Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459413" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Jziv5dBb2Zw4TFT9AsmIfziw6z3OAcY4la4IEHuJtAo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/343.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/2/343" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-94mD_doGKgSlV8AGSw4kzCE8h6Hfg3UbHb5bJivqa60" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/2/343" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/343.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459413" id="node459413--2" data-pisa="ggg;3/2/343" data-pisa-master="ggg;3/2/343" data-apath="/ggg/3/2/343.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Karl Waern</span> and <span class="highwire-citation-author" data-delta="1">Michael Snyder</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">343-352;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003640 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459413--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl Waern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, Stanford, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BWaern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waern%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl%2BWaern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Snyder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, Stanford, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSnyder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snyder%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BSnyder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mpsnyder@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">mpsnyder@stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/2/343" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/2/343&amp;title=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/2/343&amp;title=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/2/343&amp;title=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/2/343&amp;text=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/2/343&amp;title=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/2/343&amp;t=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/2/343&amp;title=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/2/343&amp;title=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459413/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459413" id="node459413--3" data-pisa="ggg;3/2/343" data-pisa-master="ggg;3/2/343" data-apath="/ggg/3/2/343.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Transcript Diversity and Novel Upstream Open Reading Frame Regulation in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Karl Waern</span> and <span class="highwire-citation-author" data-delta="1">Michael Snyder</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">343-352;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003640 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459413--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl Waern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, Stanford, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BWaern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waern%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl%2BWaern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Snyder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, Stanford, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSnyder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snyder%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BSnyder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mpsnyder@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">mpsnyder@stanford.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=343&amp;rft.epage=352&amp;rft.atitle=Extensive%20Transcript%20Diversity%20and%20Novel%20Upstream%20Open%20Reading%20Frame%20Regulation%20in%20Yeast&amp;rft.volume=3&amp;rft.issue=2&amp;rft.date=2013-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Waern%2C+Karl&amp;rft.au=Snyder%2C+Michael"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459413/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459413/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459413/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459413/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459413/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459413/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459413/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459413/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459413/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459413/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459413/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459413/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/2/343" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/2/343.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/2/343.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/2/343.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/2/343.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
