<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/2/197" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/2/197.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/2/197.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/2/197.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/2/197/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/2/197.atom" />
<meta name="HW.pisa" content="ggg;3/2/197" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups" />
<meta name="DC.Identifier" content="10.1534/g3.112.004630" />
<meta name="DC.Date" content="2013-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Technow et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Northern corn leaf blight (NCLB), a severe fungal disease causing yield losses worldwide, is most effectively controlled by resistant varieties. Genomic prediction could greatly aid resistance breeding efforts. However, the development of accurate prediction models requires large training sets of genotyped and phenotyped individuals. Maize hybrid breeding is based on distinct heterotic groups that maximize heterosis (the dent and flint groups in Central Europe). The resulting allocation of resources to parallel breeding programs challenges the establishment of sufficiently sized training sets within groups. Therefore, using training sets combining both heterotic groups might be a possibility of increasing training set sizes and thereby prediction accuracies. The objectives of our study were to assess the prospect of genomic prediction of NCLB resistance in maize and the benefit of a training set that combines two heterotic groups. Our data comprised 100 dent and 97 flint lines, phenotyped for NCLB resistance per se and genotyped with high-density single-nucleotide polymorphism marker data. A genomic BLUP model was used to predict genotypic values. Prediction accuracies reached a maximum of 0.706 (dent) and 0.690 (flint), and there was a strong positive response to increases in training set size. The use of combined training sets led to significantly greater prediction accuracies for both heterotic groups. Our results encourage the application of genomic prediction in NCLB-resistance breeding programs and the use of combined training sets." />
<meta name="DC.Contributor" content="Frank Technow" />
<meta name="DC.Contributor" content="Anna Bürger" />
<meta name="DC.Contributor" content="Albrecht E. Melchinger" />
<meta name="article:published_time" content="2013-02-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Northern corn leaf blight (NCLB), a severe fungal disease causing yield losses worldwide, is most effectively controlled by resistant varieties. Genomic prediction could greatly aid resistance breeding efforts. However, the development of accurate prediction models requires large training sets of genotyped and phenotyped individuals. Maize hybrid breeding is based on distinct heterotic groups that maximize heterosis (the dent and flint groups in Central Europe). The resulting allocation of resources to parallel breeding programs challenges the establishment of sufficiently sized training sets within groups. Therefore, using training sets combining both heterotic groups might be a possibility of increasing training set sizes and thereby prediction accuracies. The objectives of our study were to assess the prospect of genomic prediction of NCLB resistance in maize and the benefit of a training set that combines two heterotic groups. Our data comprised 100 dent and 97 flint lines, phenotyped for NCLB resistance &lt;i&gt;per se&lt;/i&gt; and genotyped with high-density single-nucleotide polymorphism marker data. A genomic BLUP model was used to predict genotypic values. Prediction accuracies reached a maximum of 0.706 (dent) and 0.690 (flint), and there was a strong positive response to increases in training set size. The use of combined training sets led to significantly greater prediction accuracies for both heterotic groups. Our results encourage the application of genomic prediction in NCLB-resistance breeding programs and the use of combined training sets.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/02/01" />
<meta name="citation_mjid" content="ggg;3/2/197" />
<meta name="citation_id" content="3/2/197" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/2/197" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/2/197.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/2/197.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/2/197.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004630" />
<meta name="citation_pmid" content="23390596" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="197" />
<meta name="citation_lastpage" content="203" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Frank Technow" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany" />
<meta name="citation_author_email" content="Frank.Technow@uni-hohenheim.de" />
<meta name="citation_author" content="Anna Bürger" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany" />
<meta name="citation_author" content="Albrecht E. Melchinger" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H.-J. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=M. L. Carson;citation_title=Inheritance of latent period length in maize infected with Exserohilum turcicum;citation_pages=581-585;citation_volume=79;citation_year=1995;citation_doi=10.1094/PD-79-0581" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=B. C. Y. Collard;citation_author=D. J. Mackill;citation_title=Marker-assisted selection: an approach for precision plant breeding in the twenty-first century;citation_pages=557-572;citation_volume=363;citation_year=2008;citation_issue=1491;citation_pmid=17715053;citation_doi=10.1098/rstb.2007.2170" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=H. D. Daetwyler;citation_author=J. M. Hickey;citation_author=J. M. Henshall;citation_author=S. Dominik;citation_author=B. Gredler;citation_title=Accuracy of estimated genomic breeding values for wool and meat traits in a multi-breed sheep population;citation_pages=1004-1010;citation_volume=50;citation_year=2010;citation_doi=10.1071/AN10096" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=A. Dingerdissen;citation_author=H. Geiger;citation_author=M. Lee;citation_author=A. Scheckert;citation_author=H. Welz;citation_title=Interval mapping of genes for quantitative resistance of maize to Setosphaeria turcica, cause of northern leaf blight, in a tropical environment;citation_pages=143-156;citation_volume=2;citation_year=1996;citation_doi=10.1007/BF00441429" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. a. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Erbe;citation_author=B. J. Hayes;citation_author=L. K. Matukumalli;citation_author=S. Goswami;citation_author=P. J. Bowman;citation_title=Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.;citation_pages=4114-4129;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720968;citation_doi=10.3168/jds.2011-5019" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_title=Correlated response to selection;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_pages=320-322;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. Guzman;citation_author=K. Lamkey;citation_title=Effective Population Size and Genetic Variability in the BS11 Maize Population;citation_pages=338-346;citation_volume=40;citation_year=2000;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. D. Hadfield;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. P. Kärkkäinen;citation_author=M. J. Sillanpää;citation_title=Back to basics for bayesian model building in genomic selection;citation_volume=191;citation_year=2012;citation_pmid=22554888;citation_doi=10.1534/genetics.112.139014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice;citation_pages=611-618;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for Fusarium head blight resistance in six-sow barley;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Martin;citation_author=T. Miedaner;citation_author=B. S. Dhillon;citation_author=U. Ufermann;citation_author=B. Kessel;citation_title=Colocalization of QTL for gibberella ear rot resistance and low mycotoxin contamination in early european maize;citation_pages=1935-1945;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.11.0664" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. M. Messmer;citation_author=A. E. Melchinger;citation_author=J. Boppenmaier;citation_author=E. Brunklaus-Jung;citation_author=R. G. Herrmann;citation_title=Relationship among early European maize inbreds: I. Genetic diversity among Flint and Dent lines revealed by RFLPs;citation_pages=1301-1309;citation_volume=32;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=C. Nadeau;citation_author=Y. Bengio;citation_title=Inference for the generalization error;citation_pages=239-281;citation_volume=52;citation_year=2003;citation_doi=10.1023/A:1024068626366" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=J. M. Perkins;citation_author=W. L. Pederson;citation_title=Disease development and yield losses associated with northern corn leaf blight on corn;citation_pages=940-943;citation_volume=71;citation_year=1987;citation_doi=10.1094/PD-71-0940" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. a. Poland;citation_author=P. J. Bradbury;citation_author=E. S. Buckler;citation_author=R. J. Nelson;citation_title=Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize;citation_volume=108;citation_year=2011;citation_pmid=21482771;citation_doi=10.1073/pnas.1010894108" />
<meta name="citation_reference" content="citation_author=R Development Core Team" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=A. D. Raymundo;citation_author=A. L. Hooker;citation_title=Measuring the relationship between northern corn leaf blight and yield losses;citation_pages=325-327;citation_volume=65;citation_year=1981;citation_doi=10.1094/PD-65-325" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Rebourg;citation_author=M. Chastanet;citation_author=B. Gouesnard;citation_author=C. Welcker;citation_author=P. Dubreuil;citation_title=Maize introduction into Europe: the history reviewed in the light of molecular data.;citation_pages=895-903;citation_volume=106;citation_year=2003;citation_issue=5;citation_pmid=12647065" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen;citation_author=J. Rutkoski;citation_author=J. Benson;citation_author=Y. Jia;citation_author=G. Brown-Guedira;citation_author=J.-L. Jannink;citation_title=Evaluation of genomic prediction methods for fusarium head blight resistance in wheat;citation_pages=51-61;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.02.0001" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Van Inghelandt;citation_author=A. E. Melchinger;citation_author=J.-P. Martinant;citation_author=B. Stich;citation_title=Genome-wide association mapping of flowering time and northern corn leaf blight (Setosphaeria turcica) resistance in a vast commercial maize germplasm set.;citation_pages=56-56;citation_volume=12;citation_year=2012;citation_pmid=22545925;citation_doi=10.1186/1471-2229-12-56" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_title=Accuracy of genomic breeding values in multi-breed beef cattle populations derived from deregressed breeding values and phenotypes;citation_author=K. L. Weber;citation_author=R. M. Thallman;citation_author=J. W. Keele;citation_author=W. M. Snelling;citation_author=G. L. Bennett;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=R. J. Wisser;citation_author=P. J. Balint-Kurti;citation_author=R. J. Nelson;citation_title=The genetic architecture of disease resistance in maize: a synthesis of published studies.;citation_pages=120-129;citation_volume=96;citation_year=2006;citation_issue=2;citation_pmid=18943914;citation_doi=10.1094/PHYTO-96-0120" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. Würschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations;citation_pages=769-776;citation_volume=124;citation_year=2011;citation_pmid=22075809" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="twitter:description" content="Northern corn leaf blight (NCLB), a severe fungal disease causing yield losses worldwide, is most effectively controlled by resistant varieties. Genomic prediction could greatly aid resistance breeding efforts. However, the development of accurate prediction models requires large training sets of genotyped and phenotyped individuals. Maize hybrid breeding is based on distinct heterotic groups that maximize heterosis (the dent and flint groups in Central Europe). The resulting allocation of resources to parallel breeding programs challenges the establishment of sufficiently sized training sets within groups. Therefore, using training sets combining both heterotic groups might be a possibility of increasing training set sizes and thereby prediction accuracies. The objectives of our study were to assess the prospect of genomic prediction of NCLB resistance in maize and the benefit of a training set that combines two heterotic groups. Our data comprised 100 dent and 97 flint lines, phenotyped for NCLB resistance per se and genotyped with high-density single-nucleotide polymorphism marker data. A genomic BLUP model was used to predict genotypic values. Prediction accuracies reached a maximum of 0.706 (dent) and 0.690 (flint), and there was a strong positive response to increases in training set size. The use of combined training sets led to significantly greater prediction accuracies for both heterotic groups. Our results encourage the application of genomic prediction in NCLB-resistance breeding programs and the use of combined training sets." />
<meta name="og-title" property="og:title" content="Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/2/197" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Northern corn leaf blight (NCLB), a severe fungal disease causing yield losses worldwide, is most effectively controlled by resistant varieties. Genomic prediction could greatly aid resistance breeding efforts. However, the development of accurate prediction models requires large training sets of genotyped and phenotyped individuals. Maize hybrid breeding is based on distinct heterotic groups that maximize heterosis (the dent and flint groups in Central Europe). The resulting allocation of resources to parallel breeding programs challenges the establishment of sufficiently sized training sets within groups. Therefore, using training sets combining both heterotic groups might be a possibility of increasing training set sizes and thereby prediction accuracies. The objectives of our study were to assess the prospect of genomic prediction of NCLB resistance in maize and the benefit of a training set that combines two heterotic groups. Our data comprised 100 dent and 97 flint lines, phenotyped for NCLB resistance per se and genotyped with high-density single-nucleotide polymorphism marker data. A genomic BLUP model was used to predict genotypic values. Prediction accuracies reached a maximum of 0.706 (dent) and 0.690 (flint), and there was a strong positive response to increases in training set size. The use of combined training sets led to significantly greater prediction accuracies for both heterotic groups. Our results encourage the application of genomic prediction in NCLB-resistance breeding programs and the use of combined training sets." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
    <title>Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xCJXLDUeXhXTudUGtULLS5huHuaXqvYpIFjp6mUgDNc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459419","apath":"\/ggg\/3\/2\/197.atom","pisa":"ggg;3\/2\/197","ac":{"\/ggg\/3\/2\/197.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/2\/197.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/2\/197"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/2\/197"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/2\/197":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022McplcZDB4GhdUIuwZexvwfAlYkXoKJzVvAJWTz9BVi+Zlh3567EdKZtLrvO2DzeYJ676t9+36rb1kIq7DMYE0eOepT1LZLUD1vqZCd5aDQyzFUbUD04jyNSenw\\\\\\\/CDmXY\\\\\\\/cC4HcOyc4+YcTpqMV++W7vIM+MPKDxwxt2Z9\\\\\\\/GipT0CUgmQ7gXrQ5t3F5nNywY5+jpLTs7OCOwI9wjcV6t2+iBoUrHrapxinjtCz9Xu3hx923f5tVZbqrsCbbut\\\\\\\/yjLQm9g9lFss80GcfTqgrtXmTNd5S5AIBOuX8byj\\\\\\\/Z2KTsZtroJYcJDs5Mvs91dgYB7kkPh0r278NgnSp9UfQbYJ\\\\\\\/us9H+OWiqGBfgHrmI8NX+XLWDUNJTgRdfpqC1nsl09bCz7aZGo9OyidUspqcsAYqtbHooooV2bJbN4MTbfGP5kOXuF2HV1X6XAZcVwkLvj6S1k7373NiU1lGIy5BdeOAT8Pg54ss7M+cM7FljQCNB2CUvDzSu9TQsT+QdbhYxzkf\\\\\\\/PDgIagZZF0+5e4+Ddfj\\\\\\\/luac59bH5W2xaIu7OHmv8IzFQkRoF8FT60izU7NG69D2jHHSyr2fIvGKEdeaOYavpsdFiBANTddet62sVZUwPcVWnNyNLQSoC8S3p2DH+Su1LPuc9wJc1DZKZaXG7J+5b7kETymh7Ie1Vq1nUUexuQe\\\\\\\/+6w6C2ZAG6UFplpyPszRgMxCPZnesS3z72ntWjiLpt+tbhAAHxS7DEkpITc4RGUl4ooOg7dla9bewNrZUHmIaKZCcfflpiKSp\\\\\\\/tBqdyKrPKzwtB0Yk2zwkIxkBDhKKQ307uQx1MoVSI7SDpvRtZBjP219q71HDL6NLZhVR6EkWkJ3wbEG\\\\\\\/inSRIkJMpGATiGXzBFv3v7YWRDZiq7g7j8gvbAQPWCp4ThqwuRyL4vTEti5FbWp8NfrceRacYvw\\\\\\\/p3OD5mc3RslnTe2BvMiZ6waht2RHgwR1L423x79H3iSZo9Eq5ssOjxhlGtbV3dKD944HSILyoWQMADRBrgppQWFsaftKbT9ox1RvuUCMO2tZb5qUfcl+VywNcE83n8FTCYFcgMRFgU0zPQ5DBGYsXmL3YmcEXyT96NCI\\\\\\\/4KyIEXesITHNp3wZk9B3toFZbUXSKAkxs7OCjP23h17VTYIB9t3I5Uj7FLpj1gHF2lAb\\\\\\\/x8wkZyXAsbX9vdPnP1MyBeyW0EadP+fcto4S791rW1AJXSvZjnH9eCjhsDFX11Sp+wQXoZCQDmvOZhCjUHOQFJlsrAA42278Y\\\\\\\/lumyaBi\\\u0022,\\\u0022iv\\\u0022:\\\u0022om\\\\\\\/I9O+px46+OlqDdTlqDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f9c738d22c562836cd14526c5aed10d8fb4b2751282013975db0997832d056bc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022038xfZ7w69mRE47hjWeX8dGiZYTmdggbzvVbORBfNiRNtKk60JcH\\\\\\\/84s+dvmSO7PS\\\\\\\/uDhIB759v2RnR5ckj58JwtAV+ojN6Qa45+FmVMffK0LuuFQ71t14RIz\\\\\\\/+OYUzK2bAaH82JiVm2v5U3FEOBC39P3GfGahT+kb45D7Ye7oUJLv2nzww6nKnAe9HSy9WBYBDzL0u0kKQVWPr9V5sl7NUGYV5DqkGRIA8Pnb3D7OJPxRJDZwRBInVjsYnQ9RdCuzU8DR6g6htiC78Ow5M+Iw+ZJxV4AzmbYIINO\\\\\\\/HSX9kSq5VcnykN6nBWiNc17hjWVeLxE+5BX6oqch2CxqUVimn2nJhMwqJbilCy0jf6QM\\\\\\\/eXTpB86tYNwySw8Etu1D7fqDuon7y+PYevb9tueT0aEs4X9A8+VBVv\\\\\\\/vmAQESUn7bC4Ebb9c1E6Zq5rLm1k5cYjclXAdwOcUl1siU3tbH4OZ2FsUswjeGjpt5woFceP7CT5pfFDuXdoa\\\\\\\/gBB8UFZc+882pRL\\\\\\\/z2Zi19u0oydoQBwNFt4ysUAYqystKtH4y57F+tI\\\\\\\/cGhSEMY6frcKQCPKNIaAzGSIgSPdWWqHMo07Y0z84KPzlSroZTxvIPeFS+i6Chf68RMUtlU5nkSthvCIcizj0wD4HMT74T4zJ0RqSiXNAO4\\\\\\\/ynSzeYWw\\\\\\\/Y58napWbPTriaLCRceUBkC4Ny4GAwdO4LE6RLUsE59HAwe0\\\\\\\/R+WFf\\\\\\\/i6vk4bkg+gXHxBWfXmvC2E71aqG+qTTjuo1GP+4+e43uumfyyf9Hm0RfFobd\\\\\\\/Nr0cBQYjBE+f4WMeLhDgOctZjvhm3y4wd7BSZ1Bapho561oq7lTFWggFeThxK8aVE6rj+aYKatMTISAEjwdbzPbsTwSQAMqYgMEM4yZr4rPPTlAw4Fob2HJj15ckgA+JYhped2TEQeMnPe46BagXot6C75J8+HRbN72UsEM3Lq\\\\\\\/Sxf0uI3EooRCIh8JvRhm+Y8YUUgiWDw2yrqj+1yNa9yJJgrtKlal+3zJA\\\\\\\/cEim2eCnO+UcB3tkMOvVp301Ph455dselhMJWylICZVum7OHrf4uhhr2CB0XKxaYoxxX3GsMER7+uH4GTnmlsxC7o43qZ7bPHYi88PR7tH0pQtvP0MS8WGKOpkF3ZeCE2qTDWX83AM85VKLWEyju2qubgfxlnZmcssULwF5YX76yubXeUmHvQofLGQCcJWxuyE2NWvIYdaZZy57hCPb7zZyAbI0AauMHPvsqB5ishoic8JpLJQ8rJIgwuNFFyqVWvrzKlaaW6y4IwLL5g5Mk\\\\\\\/iAynjZwOkQo6ClBSbFyOxFx9s6TODyR+j3REqOUyk\\\\\\\/OU5ccj3ul4ppYSWOMUpFbMQO6hjM1hBoHvOdnYnHWap54ycKEhdF58flo1F\\\\\\\/gxuZVSf6gtSTqwLvXAoFhqEddv9uu9+fQEECk3YLIv0FPwSt5KuQ8U5u63Yv\\\\\\\/yBHGmx7ICEOlo5b4sBWgB2BVIZZsMELCGw3jNJwPp+2blEIfPQShVblhB+yv7hTUgy8Hfjou5xgoWtr5cNvnv\\\\\\\/A\\\\\\\/i1SLSxHZb47dm57kbBXA5dDGUZKj0WRWAM1JIYcx4+B+XmfdfI4E\\\\\\\/NXTrOyHPbCJUFQwHHhYKCIWLm\\\\\\\/IEYzwNdaisCLveSul6Hm0GbkUVYU+wkpBSJeOqEgbAVBH9Z8WSErbkYal9MkPXY2OXmtv0M0iVP16Ew0fPKLTxkwsXXnjws7G+rmHgj\\\\\\\/J0oC6J5balDTkTMIzh4r8vNzs22548dNrvOaSsyQfWvDTJR3bP0S6kxBLXcdUMS1nNG8wH54eXCG9hQgSB0O\\\\\\\/3baL0BZinEaV2XBDabloJ4nFX0\\\\\\\/G3kfPwZnYM5oDDbHHeat4rWdfFlWqmIg9RnExsihGHbt+6DXvGuTZvI9hB5u\\\\\\\/u3d75O7uIbit\\\\\\\/vlyliGmQorE4FrID1AH3aH0NtTyMat4qLBdkZuycXcJvJj8O7AEGA5n5cvKmZc+8T90hKxaceOUj8GRaVZbzV6bptTOzBuqEUamEdWdN\\\\\\\/MoRPLLJNOagKSyMeH3mf7AVaPt+U5iGNb\\\\\\\/qPQfzdVH67x7x+NAEe9KyK1r+cawztKRdWx0s3ku76WPwFG6lzcUCYnVQH+ut5jrZ2\\\\\\\/nu24Cbc9ZyGD1jGVx4zxGaU7GfKenIVAhHbqVKwv28lbSq4I6sZKvc7WdgwQJ0sqa88i2O4gMpBQLmBHH5Kk5FLndcr3grwC2aRMpp6a2DWG1y1E9lIJfYMVybg5r0E9EhvxRmROlPlyukxl4T+2uykC6byFPUtfQRcuaTp45xGWJ9yfU1vaJ\\\\\\\/MXyWbRCXoBUIaQhDehnGvHttttQ9D7ZhxfalNCKGcrvLSoA5aw\\\\\\\/Dw0WiK6kjuw4GZorQqQVBTPDhX1utTf3A30eS0YYh7zWIvpq56edEQ5ChgGlQywQYY0re64ZioWGGuYwNHEa0EOGx3LbK9qvhi9CcJ\\\\\\\/dA1jhSZcNcLb7I8hXqME8Cs+5j5fxXS1y8zjVA9icwlpr5go3BYFSDPcJsS8Km3jw012P7DWlswdMMuNXE43maR9dI7BbJY0myyHb3vNcVHfrziv6zK1l1bAhxjBm8RmZ3QstwkAgD7jsrQfZgssGRp\\\\\\\/akKbMmEHH4o\\\\\\\/00wi3zgeAgRwF+bydxQr+OA+6aJ72d8ECm5hhjI6jOuE1ISQx2aJbuVoKSjTS79i4dIV\\\\\\\/HjGHjThodnD8RRSVgYtABCcNF+1CzdWAfbv\\\\\\\/04KRjz7CSaXIc+3\\\\\\\/nSkWn\\\\\\\/4JoYqLxssu345YSUyE89w9epKCvnw71ps8kn0Np6TXR7i402xNyWKB1rSGgB8pqt45Mm2PIWm0y5sFJVp\\\\\\\/mpLw+B5sWDIvqKwV8Xr6\\\\\\\/Yhz2v3WOx2Jr1aKgKPRXL5ecxkEBuuYIbeOBspw3MTfHGPG8m0eavKFw8E0+cF\\\\\\\/njfdeZA2HSqzS7X8AA2wf27g8f752GVPG3mjFO+V82NgooEPDatXo5IGUHR3\\\\\\\/jpN\\\\\\\/dWgo8mpvgeEl0i+iBhmba+y50hyaQqo90\\\\\\\/bqjhxX4gqSTccX9hgIuYEnvfna8xH6KQfaFhwoFRfN1mN3jPYbKzEuZqdlihh88A\\\\\\\/\\\\\\\/ySG1eUeXt\\\\\\\/qV02VhK3wJ5HMdkVnaqqBbjt2eXjGCPCbGO7LXSkXg1rRa2pF3yr1GL6PCwMIwXSYGjNseM\\\\\\\/5EM0d51FwidKL2M2iJxZjCP7smu4tz92oqaDY3gHEHwB+epOaZpdhq1RgTDqckRl\\\\\\\/72cv0z9MavJIBFurWRdTd8BK3dpeYJ6WUKDyPb8NEg1LSBNidY17SU7mq6vKPt3SA5Lju2B1BK7vAkm+q21TkraziDVL7r+CHyhKCGUxUwjvouk9S4wrh++EFLmxwUGxdwv6TaIpMwX+T1YAOonb354LiTACp7ecIvJkpWgBBiHUe7C2mRID4KyP2\\\\\\\/CvrG5aRWkNpKgi8+rpKJxzqGtGJrw+k4PGuMyGaDtWjnOYe3CzcyE4h7KE4hxk3nYXtifkJWtaboOonGNAIAV\\\\\\\/lMT6H+7vMh\\\\\\\/dBOXEkrx5tWWYthijyaIurqyOE4+y6AbARnA\\\\\\\/rCaqfm4bs9Snv5B9fILEyijrJyBcUurZbc84S29nTjnIz+\\\\\\\/lh\\\\\\\/NVaId5CueGDbMQejXWTE4R3RMbeCO3XyxW3cgvIgsmD2HGzKAiOF37FiTnrR1KR4gI2FGgz4ZGR3vPOJUrRa4ljJHc3lVhbCSJ1D8bxHVeVXH2adoBfLO6oJbSTzvd1WUum+KIuX\\\\\\\/v+OsVforbJX6EDtx7hO33qVaar10kXFsS3VZ8DWUDWiyRWJG4fBaN8xDAWPboPdjmT8r5+gjQSFbQ3qtqBwh8iAh3WVqlBkLc+WytuEpAQKWb8yxeikiIzc7n1X9qLr37eNvB9vQuSlkrkDkSM4iwSKbF\\\\\\\/NFEuGtDBFopQE2n2dukO5O+L3etING62TwAjAVU6NsQa7DzL9s8daq83l40uHHh476UM+7CdDv11ckvkLUiZoEXQTpj7OJzV6zjiF5jXmj4GbUhAloKCvzHcRJ8ViO19fY\\\\\\\/boebKsMInOm2Y6NBIDNOryUpQS+Tm94Jv80CH4PhKI0oUyHP5mmMiDEmHi97dxUVQnEtLZFoBlYM5iAJeSlPH3HqpWZtXv4\\\\\\\/Mdc\\\\\\\/TnD6n+2CouSVLVZNKt0qUJoOgcg0W4BDeitQELJZDyWBDzr\\\\\\\/1gzwsQymHG5\\\\\\\/89SfRw1\\\\\\\/91buAsv7UMuQsCZigUDkspmyjhW7oDNn656hYBv23hkGgUlIrWJEGer12SHuXFFs0WlCQqubCNpro6roC2GObsZtVam2kCdMaNuO8SN3loOAWr5Ik\\\\\\\/65cHxRcu4BO6YPOOI927j0V8bdgbnXyykRUX8luqcVZrOI4xaN1r73bDVABJvp5FT9JPXkkvYKho7TT3cte6al26Ye8ZPSz+4BWwjw4IJY5p1jkNbxs7W\\\\\\\/PwV\\\\\\\/L5UrcWSMZTT4wENqEy0E+6cgyx84FGWpV2FheRsYczcnHXrF3xFakjP1WslfBUZIcH4PJlwYn0ODl4xO7h5b7FUeVF3OAcO\\\\\\\/Ilwkul03ZdY+9GE2vRuM8IxZj+FUE9LVNtXWqO9ZDdASoa2WXc5aBRTxX5MktYiFVCUNGy9oRZLqDgRL9980AmHOEeLddq4KZoaL9P9fDcQB0h8ooCl+9jAzLDounUdd40AVmaVT91Ztwsm7LxIoezzkF0MTVumqZP\\\\\\\/JVXLlv7tr37r63V7m9bWDvtT066DZ2KKNNfxBlA5y\\\\\\\/g1LrB57HxjTTC2gM+b3GfjHm\\\\\\\/xeHPIVhyQmyegcVTrhNozSf7gKkxibuoqu+TvxMV4vxNSamKylYM2kdWhkL0RwuuIkp9Cq8N50Gq5+4LCKlT7TjUgrrPrJC7+2vY04jOe1CFnLSuy9nsivzUoGTwdvLmaMPWjDtUhsMYWAfujhVc6WkimlbFbg5k\\\\\\\/FCFuLgmmsDDjHBLbHFPRMucWnFu959I7sHU6O1Ys81oRLQ+HSTSrKzfY1NeFhzTZfpiaIkmEs\\\\\\\/MRCrSxQSEoR\\\\\\\/bz0kT0NwV5W5FEy6wIjRy04blEkeaVCkdFaIiQptw+KOJRlZceWTkvB9EK9OqE0JyCwew8KShRIm3jV6EqSEb5gF0qbGrAaFmS5b7wqx64ZzeaqgZrnbybUBUhCS7iawfl9XyIhMJDaGH8InG3JCGNTbzkH8OjOe8atTu79GRaDzjhw8h4M4Y+MTjX5+Dx67MnObLy4pVlIWLDPE4DgG\\\\\\\/+CmQRmgX28YLPrebkCYGtEP\\\\\\\/1gjCESKgiDndK1G7ZTEC+24FFET9n8pfucNhrIij2sm6ZVn+0QA2KZWK6V\\\\\\\/r4TQ87pMzrqQPjB8VGz2OxJQ\\\\\\\/G42Cz4c7\\\\\\\/Zbt5dvWyosG3Ppi4Rc9Xg76f9mgnElkIkE9veE3oBPJPtgcAup87TqicfujIBpGxey45hxHm74fqwleNOGsGpRd0XKXdTFqoF\\\\\\\/OuO+HKRMV7\\\\\\\/xCe9MsurFQECmdmSE5IB9aHgbDMBLETr0q8\\\\\\\/ad8xc+KWeGA2PsejLmCOfJNnENHUnN5hKtFVWN\\\\\\\/3HEt+kvamEbyUglTX9sxfQ3THbcInvJjv8TQpNNHYC+g7iUOWdVi8y7mPmEEIQBsV7LASfijd7SjTuhPIwKuCHAv9uGGnQX78RLg5ra+rxu5pcQHi8Vi0X0e1TkV039mNm5LVZNl1n\\\\\\\/LK++RncsnXyENa4OYHgmWMGinquNsJTT7T44AIRDELyLfb+P5OqdNgrU0Nss58bSWfN7PCxtIka3CQOxbBuOSLIS2f\\\\\\\/0JLZqs3ykQfvvCgPEiUEJ6R\\\\\\\/UY72kIpvX01\\\\\\\/b6vDTxjupdRt1Do1MSa3j3pJ3KBzNbrNTDCS+XtVy3VTu6pLaltaRHzAONgXlphD6sX26uRZgKkp44\\\\\\\/9YZfOpD3tDfx9Ye7R8XVfLSM2V4QsRSgaBAxZtwVuUtDZPraPcrGl4OBLgWOFv0eFqzk6nguduUh0BsJ7pYkuXhmRAFuXduny8azLKhoqpJ9mi6JRVvg4Gxi\\\\\\\/WQ4WGSqvEB2YKNlBRomHuC5qz32l3Ldc3RiXry\\\\\\\/SSyqLFyggDQffSqI\\\\\\\/BcX\\\\\\\/YvWgV20gaZrDoxs6Vr1Wto9H6dlFr+IrANHp7kpxxjobcGtBLwwSTpJnxlVePv9WIhFIQ\\\\\\\/qPwW1qOvQRm+pMA3ypllh1+WfxmipomBkFHud4k7r6qZAvjj6Eentp6CB6k\\\\\\\/C4S1TaBTzNHiS05FYvTv2uZIUzjoic8PXzPJm0aCiB9hpzAnDQOssUKMXBLydrlE7ecTzq7CSbOeeCBgORCNvkhEO0fQnhlnP1coEWjoG1ohr184dIGO0\\\\\\\/CuKNCmSZns4pABpyvkNVCmFCp\\\\\\\/kk9BR1+KnQLsPjQdiyHF26fV87s4KMS+cN9Q06bKAIX+Zty+bkzi+N+okjuqVTLIKTCpa8tgDnBYGSRg6Es+yy9+KZWe6bVZSBfYjIisDTy2M9Lk1vEyGCE0sheR0IwlFprGUCiPrIsxntz4RgRC++1YFtY2fdJsYo1ZYnQFGPDJHSl3bqunTCzCwsH4U3MUSN9KisTT+C73Yq1oQwfCCZA22m9i7fQzUpo7SNiB1KbAa6xiIjsr313WRc0c4wd5GS9jPCZRZqWPh+tm\\\\\\\/mF+WXPqaKxIGnSd7Hq\\\\\\\/IPCq7dpNZXa+zbbJQB284N40gMmE26uGBXwyHdfZq83uCt9+c7yHyzwiFARW8YHEDx72IRRzBTNU+WFEIpkjWGvMwDn8BLFCqE3w\\\\\\\/gcIt9WKRQQ6SYjP1snDKUX2BZeahsJYLoBIl\\\\\\\/3ngLxnUF0vx9MiQm3aXapKDHgSt2vsvz+kZSvqVuNSVoJWDumL4exuUhDp++VA0k2SA4WEnIhZX0HtVU2c5+sxzbs35gJKGJs2H21si6ICp8wJSvfHbmvs+5KSNCL84LGsMtCMxlojq7CH99D77CgimDWRWK98rWAuJWXHF8R4ElGODUyl9CU08kPi4Rjcv1mkGM\\\\\\\/4cQoTkQNW+FQtSgyZKh3nZeHfdkyAksqxFESg4C3HGIsvwa+uAIVc88tnIZnkrcRHmgJdb\\\\\\\/VpeqAcxwnmBqvRkP4H+uTwxmt3D3Jmb7yzUzjBxTJvKGNWkDznf+hPUatHgqoqzGcoY8FuqkIhlqLUoOxVOU6jxoWVdGLkj1\\\\\\\/L9BHOfETCCw5CYcXijSzySdE\\\\\\\/KR1K9hbOyyZthLAOZPwa1BjzvGvb+3L7IUgyjl9SRzERw8seynRV9ejdER7bzivcToZNrQG7YwuEFcy6Pav3ZlE8aBdZie9fZbjNNOZsAnblhsKfWH+VkK6rfBDmYKcC+Pt3KZkizm8ri7zVxMqSTgI4TNybE1pIHUnUHSu9cmzhY89iYt\\\\\\\/M56ascd0hTMfNpwXAFpNWII4Qw\\\\\\\/nI40bdBzTEf2psEcrmVlW8s029ZX+D47BlKLfMp0bdgaffbYwR46W+RQyK6mpHQMiiCQDIf2T5kVPsrPplPjPVDdV6iaXVjNa+9x\\\\\\\/ESnSzs2oSVQy0JUb1YJPVGp9WIg\\\\\\\/o9+SFjN9HFcLd2NGe8ejVa3xAJoq2H94xpuUiTOfCpRaWoHd98ls4KLKnS3tv8hdGkZjXQbSZb33\\\\\\\/AXBk8EpL5QLyYmQt\\\\\\\/gLMW95Lbu4xaU5fOOLLNv71KWC84A+1xF0gVO0WmLGZpK84SsImXwoDJLQjMhP3KouX2EcfHRlYZ0ZbhzfMfIXWel4gzZIjyIKlNCgmMDProRe3z5Uuh2s\\\\\\\/WAhh8V+FASBnJe4tp3Mi+HYCDVwXYRkL9S2r6lQ5Ywfc\\\\\\\/E7bnL\\\\\\\/yua12BfUOYAsXkJkQp7aa\\\\\\\/8i2IOXrbj4ij87MWIUnhQ6rJ1YtIYCkyzq4X9A3ondV1yo9Au3Ev4wkF4qliMy2Ig12HLtjifRP4kLg\\\\\\\/todBf8kYyCX5Z3zBqpyFXIS5KEEJHd+o+qpKiEOlr3uZqxeTInM\\\\\\\/9PeEvNrVHMrNxiOOU9itDIxHNVQGYEkpNp7O78r3JzUbUQhPHaNei\\\\\\\/RGwnthCxO8JyxjwT+XR+NPpL5qYIZoX4NdUxMtjtOnAYVCsj5RB5WKsXeOvxNcj6WcU\\\\\\\/6Wp1IWEsE11E+7eB8CyP\\\\\\\/RSBjH0Eh6efMSw\\\\\\\/W7tFm4czojn+D5jphqgc8d+tqRGKyOZXBXp4SHwLYlvoNLWxK5tHGy0N1YMgt+02qLMMyaL0G1ICKNPR+pWrpVlYUSKcuYSwsnFO6MzNVeefM5JWqvx\\\\\\\/ak2YZPhAPtO5pdXyHipDCK3BBak\\\\\\\/msChYPQtVQTLYixVCwv2p6fLHW24YrkRi0jLQ8VSyLYTrng\\\\\\\/g2e4XkhQI3QZarc\\\\\\\/hVAeS\\\\\\\/yh\\\\\\\/LEusUs1z+SawKzT5+kaWBu0Zhfo2RnnOMEF\\\\\\\/rYWYVSJXDftqgdCbEeVsMZJFiSZILSNml5UercQwXTyJkiivZnKIF3SsYm+qFxFmH2vRAJYXhoa+EWZ5gEr0RFoi0+I6koSzYmUF7K8HD2WcMpx6Ht1F651puwQ6aWpJARmoCDppWfseb\\\\\\\/jslhPcezJ5mkDjThQFhYRVfuiAUWOVfzB0nZ5HVxSAwMSV4OhWhPXL06BJi8J1QWMJZ\\\\\\\/KuruAmfrIl\\\\\\\/v5mYn\\\\\\\/4sZBhh6m8cLvgY5ZKNjwyTfWe++Nl+RfTLnn\\\\\\\/6Vxr6fijIU3\\\\\\\/iikLFM7Uo2hpJfj\\\\\\\/PEZJA0N4CJRpqjNNNvUZPW8olwukoCVQ8bu3Y3XKF6G8WiCBBb\\\\\\\/WeDH484C9xwHsIa6iQAwnQAdurDgNPTb0dwHcLlLzFLGykHk5BfjLkHSV3YZ+4r67il2v+V6UdzASXdadsN\\\\\\\/ft+mU1mSqsNB3kg9MDL4vVGm0gCK9ytjyRjmEvR+gU6rckFF1xiUHadkbZ\\\\\\\/9M1qvxcq\\\\\\\/+Bjft9e8wNFjyFAXBC8jwsaJ0rvp20jq\\\\\\\/T\\\\\\\/L0i2bKArmD+ebGfEFiQkvnvQyl2pSEjErBbR2tHDndoWJXOHX2pHoHo7eZKunmefwLJxBxalFYQ1Kg4MIeQtO\\\\\\\/tQ5AvrfbxPvlAkGPD\\\\\\\/kdUjBgXN8RM0PFg\\\\\\\/4OGpOw9mfinKdmfSJsFukLHiKOzRFETSi4LBz636xHirYzwvyfm36c3IcWI\\\\\\\/Xx+7qZhnblsfolhKHM\\\\\\\/a9nCPFA48e2MgTJ2qOQroWyCGUjpixOec7n4El8lYnx9F34Gd6ldgcBu+Fp1d3b9wZj6TnZAJcgcQrDAjZD41cwmYk05Yt6gSQ4QcK2sFd4DgoDOubBELMVBlzulFoIM49NEMzLJJOXLVNzGw3flw1n8WUfAj0Az9e2HXTEtjTZ2V80UmKuw7TNv1DG5ctRX7j8\\\\\\\/RrKix14rQomFWWxR+tUl1PXfiPn1TRXVS\\\\\\\/tLd\\\\\\\/oV+smJ1igY+Pe6i1ogK1jjFn0cUAywE1OhJpS1voZJAv4D4KVQ8Mh0CuGK8GkaW1dF7cIB50suPHX\\\\\\\/9\\\\\\\/1ewu7FcpJAHVbPU74oqHisYQG3mwAVSwJSIMEnCB1S4EZYc9i0z7Brr33tSxh75KgmjCEMnB4hPAFLamJ8BhYz1QynJPf6bnwGFHAnGVXokQljtE2OdK6qdvKVHhRN\\\\\\\/tA\\\\\\\/iyw967TfOopIpec56Mal00Dn9+J8oLYjqgLdL+p158AjZUVNW\\\\\\\/p3L1B0C24c+53wI8nDDQNcwbL5PB1bUJiMffwhSMPRg8FcslJydYIzNPzjcn0L7Z\\\\\\\/5\\\\\\\/RxCje6xDQ69OqGKFZtHyVA8tPoBnmJa5lASbLTDySDA75EdrxlKmeeYQoWKyizZ1M91VFHrLE0+9j4jFWLoVjKpCUqhgg9i69Y7dzu06\\\\\\\/96lM8ApTUvjwaTCAc\\\\\\\/gInKaCtVHHCNxMNaiJeoF0ZLnOOD\\\\\\\/j6PTJEsDEYnllVuV0qYHzNq6rt4mlt4lbENoXjAurwc\\\\\\\/VMydqrFLGmkTQnaZ\\\\\\\/Abit89kgcOPtDzz5UAP8mNzSdrcMdYRjDMSQQqwqzGa\\\\\\\/NVEhpQo45EUCeDfWDOfWzpUeTf4VPuoQk39BUdym9ZlSw81zi0SjnsI6pnubvuFtHKmkeUN6Ppgs7fH\\\\\\\/uqetmrKUeS2EkI9H+JxwBT3r\\\\\\\/WOFmJuj1sU5zqauvGLfECIo7cBjEQ5tJWnLNA1zamhlSrYcgGVw5MEcZ\\\\\\\/1+HHY8pW\\\\\\\/2x5EXvUYSl6gXh6Y0suRNX372ULZ1vBJM7299yKc5E\\\\\\\/d2HkNI4GO4GDYpuMuETba4EyNkxArIeIQUX1Cs73FLv\\\\\\\/FClKF7uuxePdWr\\\\\\\/bkIdVuiz9qDU+GfEHwrkEqfP5qlYhC85V1M0aDChcSfNfizclVvZIaCDx9WdnfuSkoD6jtg6nv\\\\\\\/PN8\\\\\\\/MDdDNDFVQC3DAvVqzrxh1Ek7RvfA7q6FJLW52vEBqQcZfngnr3Iu9djFnOuPTq\\\\\\\/UeOeo0HMldBeOZcnB1OgB6+3VqyhwN5SEA0dxg+cIMVbhi5DYsxH2ArVo3BWNl6cN1xFRF4pqa2k5lajZq0aiMuAI4g18KWpvD19ZHrQiwufkASD6mNTZ0dGL0UtxMlogvQ1bYiCgb14J\\\\\\\/olrsYwhG5w9xFZsIdQRh17V0bPBsSqKt2Fh\\\\\\\/CjjtekJ3wkehNq7w0d3JDsrNCmuVlheVOXj8xhu1B15wTYqTLM36XRsIVe5\\\\\\\/nCJBNHDhK0Ay+YGGSkP\\\\\\\/PIGwBHIEwjpTJQUmOCqC4+GHcAPdHQaBnk38US+ZGwaTrJ4ZByyI2V7fb4y5Pf0j\\\\\\\/Vv7hn\\\\\\\/MSfv5lqQrilo7aNnj34FaXfhH+KMI\\\\\\\/xAwt+OksnGQylJdbbU84C5fGWr9W\\\\\\\/JVEYDWUHOqMwQ9rwjvPNHTu8ZnM4rQLvctUXrj40WWgRzqz2U5Dd6+Y8233ZgGHKY25bB6LzO\\\\\\\/VlcJuiqS+QKGmxzYEPqAuMwMIdLm8fLhVnW\\\\\\\/vfMHVm+hmRkpEscJtloQSYPiZ2gG99OaoeOm60ynxuUYMsS12tQWSXocZweCoxxq6KtvdEO2n6p6Jd1elG1XsKUkx245TE60eFcw3vV8Nvi8luw3YyxnYzJqRscxzfWQRkPZrZpg+Y70Jeglhy3LG0\\\\\\\/PLwo\\\\\\\/8kJ5WQzDeGDIMzIGSrPaU9kQrQbpfsAFzwSwZ\\\\\\\/0UejWy61ebX2rF55ocWWIV9atPJtVzZ+O0NTBiO+hSpevvBe72356ozRRzlHT4CWTmOleNxyM22WNfueQ3x0nk1hBrxwhA04zVuwfTabJFiVDTlt9U9T10jixQ2A5DW363DoIE5NGgzWTaZmA2pUN5C\\\\\\\/uXZxfkoTrNAwm+jK1sLmLFt4DGOrjTi0MWVGFRleyk93\\\\\\\/zOnzmrouXPoVPnHIWMbo2ZLQoATVCHERhDIANEV4KmzLgT9KGHoM6x2qvtbBoeKIDCY\\\\\\\/blIKxJ9Vr6ds2cEV092bumSr6GYos4Yq9lH6WBsrQ35uhS6iNcRyw9qOo6O6Obk3it34hJzkZ7a2\\\\\\\/oe6RGPIevfFNXkGR5iiXoThxBAT2x9FuKr0ZJf0IXyxp8HZxcA\\\\\\\/uN1P+bZz28bGXj42lhclGpjrHtiBT3yB5qb8GE1YSKUjW5B6YFDKPI3k3xFCxYoe\\\\\\\/tsQWw3l50QEzCorx4py0PWjY6slW\\\\\\\/64+IrVSGEV7gder29OvEPVeD8od8D8QliW3mUQ+vTMu8MBogmKo143eSzTiAXKXalQJTdzf5qO1mrppjyjrNO0EqI8z8O6qtVG3vNIUveNRJBYb\\\\\\\/9tTAc7FqYZZNqPmeoIWjQJhWjMSEGBcwSadnj3xiuPt8LCtO+H0QCqzAlQPq5w5Q1mvDRhotunw3tZapdrdO8\\\\\\\/RIa3fnm76nyh5bza1N9kl2y8ckQv9iouLp0vkwPiiCCPC4k5OkQrXByekZ3WNOJVJ0G3rYHlGsA07HDK1CvDX3AqNkLrq+4SDq3+fPtJptr3kDUwOMpWbWdeNPzGDXfLzxGfKBWuZ8PojpjgL0arTtiGfTraYJrXpER8YnEMu+gJUx5bXr0KgVHWB0z2fEwML9Sy9xfq47tNf6i0M2ZcvU6YATbWz7fdYoBZB9UCTI7advEfsq6lEbfPLKzQjIIi\\\\\\\/PDLM4NSF7EtGIPIje\\\\\\\/g0RoLBA9YscKJJyS5kLSzSu4OEI0s627yi9iscur1O31RMDKbg4H6yuq0qx6GW41bNXqgUIrtDQ8RxJFUKNSwKFZ+WqwtO8IOwgTlo2hj1RScgiuvdsUsEZumystd8RpkW56b4XKH5OMXHMuIzTk1SYtPChawdghVnM2KXCUATJgEu5H5GtJQxIAJEQ+zphDbr+n1UuU6LtZ+PXevU9kLVCkHZ1oqi2\\\\\\\/mk7pfAsfQ57EwWO6CBmQdtApmSppSW4cTzwxEJd\\\\\\\/+Qdvj7VTHiDZ2ridzEnHgqZ7yTtwOSzi7mLJH+Or0BoW7M\\\\\\\/wwMTxojXZ8xOemYi08fzgctsHJ9c6jVWxZVjL\\\\\\\/Q1eKz1PQYDYCVE6jDk6O94hDuZaQDr9ly1cNm19ios9jorbcGHm4xAYjdnuQ3S89NTEG+dYjvqhOeCk1vogQyCcf9bFNx64JtGJHQmdi8Qd5Awbe6H5cvN8hcOTtr3dI8s4ounEgrlF+wUM8cwo5PmqAHDLv774EHBs5b7z2i\\\\\\\/1xvmIHdLt+g\\\\\\\/EkK9BuSjeCTizEjUqyibiAojqe2X6CH8PGRFN72l4ckBTFhTFAFSMYK1SSSAw5DzO3KkWhrRPCHZClqzopczKq5XtIeA070V7+FCbaaWxfuGXY7YO6VB1hPv8uuZE2pXJ1MsR06HdpAoZp6EsWYSUD4VBWew5guvuDTSzHEAcIPebwau3CDZSAafwLAOD6ZTjLcdtw4AvAg6ZMAELpjFvIYOCPCHQphf4sLB4tdmOLiIoIdxJzeErZmk9Hbu71E\\\\\\\/ezvUZ8dSewB\\\\\\\/LMBom4IKgxx+x5WXGjVmS58ZZWPT9alQPDGuxKRyS3Y+Z6nzhON317lsngqJ8HyH36zmEewKi8nBwSPVh4PmkeLQ2UWHXr9GvfR3397ql2Ozh4lDTGuc8zfXRmrQjTYpkT2zNZUL1Z7nT7bvQqeFzOP+1ynqQv7UoOON7hBJ0WG\\\\\\\/\\\\\\\/abHA8tN6xE6lTsvk5SZItGlPhofBALZDEgsO\\\\\\\/EW6YHC9lBe9ui8HEomLM7UHDe2Bx6jLkxWabricCBcTwXUHEXIHH6eJVh8SruqeuE48EMUWpHq8vmg\\\\\\\/jpuwTOHFO81a526dXEhzdrnFZVPFBkUU1Vq6oGoJ2Hi7nTUfPvrEIu3ljTbtDjD18EmHnyV8mWeVMIG1Fno4DE\\\\\\\/Wy0J+QgkLuZI0fT4XKAheTyim5HEYdg\\\\\\\/5lEkd6WKmPd54MtpkH1RoZONYbIJA6YNcIS5XTFvQDi2FRwsYjH8FASYzeER9unlqZQpP26oQNy7A6r81\\\\\\\/WLvizpe2mfi5yc5DSy4GjGg79Fq39bpGIp+MedZsEyj50wBYNHyb1K0GVVi50tQwen0jtBgOne20peI+nhXsTqwEgx\\\\\\\/40HlF03glD16b6F\\\\\\\/0VPempaD19\\\\\\\/xFSybxdKpcuNc+N4621a0V7dEAXs9na0Cq8q1VmMuz2rjQslMl6J1Ddwez7mK7+w91xZ7lKzNuOwxQ2Suy9f5duUy84Fn3mQZYound1EXtsGd2zDHpCnP0FT4iSmuXzjOG98WjlXIxm8IYX0wZTCCOB8Hd0C6lfWhHL\\\\\\\/Bo4LHvhoDqNqY4nrDj\\\\\\\/8qbDSUiP6XD5t4HOY8XZWJvUHgM3ODKM06Mr3sBBczHX8Vnv7S\\\\\\\/1cASUyci2a0QFlJJDa2l9kOtzT3xPqFNpsdj2PjrbroNsCWg3Q+bv9o9atoL7w6Fh1WldLQNyiItV7bUvRP7hLGdceBKStAbHazV1NfX8JKWr7nE1pdlxg51rJZ9c5lDnuhBnsN46a+RgrKHKcZGk8rglMqhmjpxUAoZ0EZF\\\\\\\/XsuZjB8IQRieAOLai2LA1rlZIzjOFQs7RrxUlnSnwVZFGFjjZwk6eWysKC6D2mst11IRZo0g95XVLyqTyK75lJRToLNnumhNMlVz2COxy19A8wDplbfL4ot\\\\\\\/LEqIYNxvl+hnYMQwLFsCtm5HTvmctUu6wI1bw0oP7wYemIGKNH6GTiF28\\\\\\\/7cmbbS3nLugJB8Kxo9QJzkX9urisoZAjw9XI3D4+AiI4uUbDzHfLE3iZH5XU5Dbux7bRyfZ6vYrBYAEDoS\\\\\\\/cBuwg7P6SJrhuqS6BEMl\\\\\\\/1OVeNrv\\\\\\\/D\\\\\\\/E86en5OhcOcCRo3vPV\\\\\\\/kyng+NyrAUSrIXd58a9n166Q\\\\\\\/Pr4lm5DSQ\\\\\\\/whHLPb0FJsadx0UWtYrHiHdAqFC0rkRqXYGUmMUiZpMmOJkIdEYc0zxTD709P2LRySnAbcS\\\\\\\/TJPgmCJB8cRmCTXTEY\\\\\\\/bAIkyxwifrMQ9Ot9xzAXBoXwyJg\\\\\\\/esEELCb3uvas8dv3WhbXaB7vxLCfj7OwAFVPtNMJ5NhF42D4cpptSLP2sOagSCQQcaKvamiJZ7yA3+51yV\\\\\\\/vx0+Rk\\\\\\\/ek28wSlm+13h\\\\\\\/3Jw1mx8mcdy\\\\\\\/yhSISucXhaAvSCxKJlehFYJ9KkN4IJpfA8iSXA\\\\\\\/DF0XtBfa\\\\\\\/sot0gZDuD26cAyVYvMZpqjI\\\\\\\/tj2gC71gZ7kjaVKXZ0cQuC3HzCD8UCYXCf5I2SYhdfBvqlWVft4F3YPkYO9PwOBXaJ+fjvuqsmM+ROmKknhVJeoKUL2GUPiqI+lwyXnof7gmyMcnYOEXhUq9Zy9BNn4VoX+XR6GQLP59eCeIpXZhFWW6S8yfMJ6qEOKbfwMIY\\\\\\\/W6wgLn57irKdzKMwzZaR30Gzdzn\\\\\\\/RjC1hdxgHAPPwjzhOAp0A7oBaltJyOkW6GlKWfPl7XLsqmBfw4wkGH5BUqTf1L5xz7N+pdE9RDXrttp5eJzcSBTqy2FvdpzsffQYiZ05w+daBj8ygnyqiv4I1OpPWrTnCptLzEfbgP\\\\\\\/rt02riYDne73NA09COJATIaQ8zZTD4Frm765GB5380aDx04GAZtuOAonxrNgJbZSr\\\\\\\/wuwSbbESMJKDwVpI7Omtp0SgF9Pe5BFR5znTwKAVm5maGDvbhAGv4A5lyxdHrUbvtN45vcJ1dDwt2sN4z0RpC6IVSvH90hJfB0+rbsEJxXpU1BXix9mrO7zpUqNsPKHwFkUfR\\\\\\\/sabDtd4pgOPg7HpQ776Zg9PhiqlUgEr+d0AcCpyl009MYRD6TFyaPZu6NNBKcwuju5zg82obXNHM7EEoyvY5YWZr8rmc5uFUZB2K+R+NRkz4tm0+0GUxjDDVJ9PXIIbUdkV1E97xZtSgms0D54DUA7zUETbyUttojgpidK9S0A8HFC2xhryND3mpwecwP\\\\\\\/eiE+hjkqE\\\\\\\/Rpg2UO\\\\\\\/FqM1porj7iiA8b+POCO5YlvaBP4lpzKDnEwxRzNsYLUQhrfCUzIqLdKUKkiK97uK61LRNnQO8a0SXkyPuL7IQL72ndChwCqnL183dwwy4bd5aY8MF3oNdvcR\\\\\\\/\\\\\\\/B2ZP\\\\\\\/t3RWwcbT2N\\\\\\\/vwCksnzUbWtpKP+NJze7o6u4fZs9b8qrjIjF9Gbdzt0U\\\\\\\/gmkUC3fE9vBrq0SWFAR8qD3NRHuafCl6lGboxso0u2SB7SRQHxHmkgZimQpuAQTKUf3zOlr4D+8ye4\\\\\\\/4J78\\\\\\\/d8NzCtQ\\\\\\\/MIsJx8ccS14DmNOtTIEXsOSvcIl0iWtJTYlERqqLIUz\\\\\\\/o2K498Y1qNII8pHBOdn2KBTHXtRXKJ+1sUGQW7qTcQUVNp5aekPHkikC6SyrdNJWJfXYrMPT1leQ72vypGgiEuTXTi3\\\\\\\/9q4XVVxYq\\\\\\\/+aPS5xrWnDFTQoIfwquc\\\\\\\/thoKvnJCJW1uZUZzJLKApHpFpUygO3BfasQZVeaoreVddva\\\\\\\/ouzhVti4UBT0cPM7hXXj49HT9nCzBTC\\\\\\\/7EUUxsFLolsSTm\\\\\\\/+uZBMCkMdcSNAAcJG8gOW\\\\\\\/3i28lkFwZsCTceo4ObbY6lQsgDGa2pY4ZJeTngk9OCVWSI9tr1dEvXCWCS1\\\\\\\/KrNw24g0B8lLjhbajLSP2+i6XgU4WCBCGTyEgdGfMtMNsrV\\\\\\\/48LJruMCf4E2su7rI7ZtNQyCh3SugfDSRxvO4K0rbnJdlI33BULlA7Anuyj64\\\\\\\/7bRsRP8N8VHWKiQsNxFlNc0e9S4RRGMkjOp75TsJVqUkIzOJ5viFY57BLRV6ICXsYae+JVI0RzeRnf5eXAzjNZILT5TJw8lAytwLoy3I+PKFDFu\\\\\\\/WsxtuT7wz6D05N\\\\\\\/gzXKkUaGfcK1lnS7LQu6GyM1dBCR5u2sD5H8lEKYd2vQT8JlQyxh+I63GqHcaYbnk8fM6\\\\\\\/YKfcy3l0XyuBffeUsDdM6GxkTNd2ZTtlQ297aLZp65NfeJgLO08a0OTusrlkWB1lcgx2YKs7UycIOmTNBFtIrYD2C1xl\\\\\\\/\\\\\\\/UR0EqU9q3V9LpEbsDCUktSBHnjXjnYPAFW7RnVts5enfqfPnZZVd1deHLrLl8T4qIPsugz7T7uYmSYTOhWy8UqrRAelQDjCry91PoSbRpaELWLNKvhvI0GsOb1uD5V+CsNqEH2TqdGKdJw2Un6MH7fgvrObstQ2kcyrE7198aYtoLqBZqUMXFOZ7iv2hQyYVQdaztbAb4DCIvA9dk50DiND3CupNROfewWKmzQCWOJyIK519QW+yH\\\\\\\/sutdhIAD4lwAssw75E\\\\\\\/+NzhdUdx78hRYRttPZWtHMTvA6xnTdgrjJnoGmB+tJ\\\\\\\/JnRPrUjecu7qkGmnfo0EbyjPbvd0ATDMNsc8lzMBbO8bCXKO80q7QjsNts5S28VF2aQSrrNlRyaPo6\\\\\\\/hOfDCXJYW3FzKNjReZzWFQjHOgn4jyXnXh+FhQWfIroQAFPtKJYQdszcf2ODWgM9ro8jAazCXaTEeGwz+nja95BWsq3pbMT93s8vZ3ijr3iNL140S9WZPlMED2lfBhkdr\\\\\\\/WShxdNktB8tCaBVOeqfbkgjFtOYvoSRMygyxE5wsgcKTYhEnS70ufScg3hJFmJBjJMAvZTHW1P1puAUDSnX9bBHUmV1cYM04DcGSjqsmpzPaR9qBOZbSt2i30\\\\\\\/i1agyM03wke4wF+o3Uec8rc0Tpwm8JkifGe1l1\\\\\\\/nmP6UqHRgIvovGU9GOF\\\\\\\/nwoAPc2uNhJLMREWSPRW4699sZURiRqX2juVi0EpdYiWQ8t5k\\\\\\\/eO0s+myFtEU9e9mkIrEQW9KL3Vuo30Uf6w3njESB\\\\\\\/E+oWqhruEcotYLzm9puMfGdUIz6hSq1b+VVWGig+yGeA2QTjntQFPpYXk7Uuh9rPj0QDFn04gbpjzrURYFwadK01Zb6vETtz0pkAgiaR\\\\\\\/aZl31XusZWtV3Tj4vvUQmndTjB2m\\\\\\\/ExOT3HcXgJZxc6\\\\\\\/4nJdS4Z9jvlEyuZg8TCF+JQaYRNCCOHm\\\\\\\/qe60hhu1\\\\\\\/ltkdrhwz+SMcb3qAKW9fuEjo7b3R64QaqF5OgamRKCr9X2GwoPJEhOgl2ETi5uW+O0kdFJmFQN2WNaeDKy7S+lAcGITqwZ2TSEEe7ZtjY+BHHHOk3xRxqecbS8+PqnAoWsnDcxzN4c\\\\\\\/pdp7Mr5iHyX9MiE9fFVHWHvBmzeWnMQJRmnGDr+\\\\\\\/oNmoB49tdPpknDEG\\\\\\\/ISHU3xXMg8HfTEBN0SBH71L8MqtRnTxW8c5xjXx4MJAX+Nt9dhxEbwDLB2gv94slEFNq1vuHsK5IsYCVbP\\\\\\\/5mCgT2+tpm4E\\\\\\\/DYqmV4YfSWl3Kah+WQRpQ6U+yIIF8TubEDC8gUuRMi2cOShfKf3ObCTLhmrux8DkFkcZbSv3sIObhaHoKxmqw53fOaXKw2mSj4+Rcquej8AYdMzmvJDIGox6zjqMDeEr3cKPD\\\\\\\/ovI\\\\\\\/yvEz3\\\\\\\/jJOvEXJzTed6eCsl2KBmqukx3O2oXyeWoQUkUtLIoL2XrCUTj7s+w3z6Fx1s7iz\\\\\\\/pXsZ\\\\\\\/6KZNGSE3w3\\\\\\\/mB64\\\\\\\/w5skvCQXnW+UUorpUPXoFXMmZxWXmKV\\\\\\\/F53ZynW0szJJzXA7fP0TVRRmVMswBFyMx65s+2S7opdYCF8lGB0Gn24L6DixEL34RaTZ9cynXZFF+9kB\\\\\\\/Qw57RW1xJk7f4WtIU0CKy+9c8ctSgmX+yhV8QE7rh\\\\\\\/gzfJwbCfq1cBm0EHo9KhF5mdagMorzl0RMlD2OyMjI6cBLfrAWSDcvZr3CWqbGWm42G9cXLMhe4kVSPMIp7Ne99hI898tMoSIKT56ALgNpEd8P3r0GQh7ItDgcB2Qwl8EdRvm1ly4JFUWfpMuyn9L0R+BOA5yj2wOqCYY2483QG6QRTFE29zcN4JY84E49hMBZZTeAfpdyg\\\\\\\/fNHFTTQluu3xM1Uo40\\\\\\\/E8idBgrPQMtv\\\\\\\/2I7I3Y16yxtbjchOAaKhG8\\\\\\\/tmo+cAYGwhal5AhObyxFWKBJA6uvnsF722F0cKMMjY0s6z8FB0Qv99rEweQmhNJKo6f+eaA1AJFRb6AdFb6yb1yVajhc9bYA6qiLzB12gU+ii1vEi5l2geYDNJU9Qdh+q6OTgmjDZfQv\\\\\\\/6ghAyilrIQFdJAKoIhtirSSm+UEptEax03mlUB6xP7I9ipm2UCy13okxJIG1AUl6DyBQfcsIOABOFnEa5M2w8ny2JmcUNHkf3WftjQbd8GBIalDxuF3GcU8Rdwun3PWmjj34Lu7WbD9UK\\\\\\\/OvzuqwZYKY\\\\\\\/uEmUCW9vGGj6QWVMVbZDRY1apseVBu77rEoOU3F1OSHGo+3SDVu5Ec1L9N1M1taHq3PkBq\\\\\\\/rdLeKR8BEetQrgNFY9Sqh9NOdqyVeyMVb83tUHYyEf8vwTCa9QBIkRLnMl9P9\\\\\\\/ZQIca5oIAHWD10rU0J3QwhxGxp8HIfzrHoR9J6Z\\\\\\\/2l\\\\\\\/8M0ByZ\\\\\\\/zUeWzmUpLRm++XxPHRm29hoxeCLWIIKWrk8addk1Xx0bDM5p3zvlw6Lsj5KWESRWm7+gxER1uQTDbiJF\\\\\\\/3MQBkkLWvcyDkDRNxX5mLn3PR064IHAbhQDHMwpH+c03xnwpRAsEIK28g5ZkXRD2pF6fcqVxWWy8Z6W05KxBSL8V1mCLgoNUkuzcYFTBqIihaI8EWP3EOQT1dMHP+wzFM0AniFf63YoMRLW8juQ4Me0v9P2O+JATIFjd1jy6QDLEkYGyVHXUD2eeNkHjZWUL0cWEOUZ\\\\\\\/f1I9xPhnEYDKhyd0Qn8FvcLJ3j\\\\\\\/j+j6Nbud9n0yx8h9JipkfI7S7enrWnP44ZMgxC2aEq59lTRTkaGpUc2Awfgnet1TnAxAWqx4TX7CuoqNdniiGNTaCvNyKkJWdYzL9NqDy1geiZp4AZXqVLjV91cWVu3wHmIucHERAtgqwbF5aA5nxC2+ZuGQ7X9XrxyUO4Nla021tmTw141h3sr8tmCB1LsLHctN4lqfd0Z7YoaPON+YZSJs9JXfmuPyeRwlMy9qatzxprIXXgOmEq4+KGSnD82v87u9zS2hUoY8IWxwbS3wD\\\\\\\/W\\\\\\\/k4tmPl8sLfgL\\\\\\\/vdOY\\\\\\\/pVk6g\\\\\\\/QOSUc4B476N9dlDf22ZqupyqkaBwzPa862rLcjE3pbQB\\\\\\\/MdQYAGJElvxS81qOI9MffzJyyyIds7EeJz1G6mV9b++LitmtwR3taMSVmzS\\\\\\\/az72QnuJP3vdWdvvkqJqy3Ix\\\\\\\/ODNHTl4nxIZufEYdlexCiiHuRjbJQhbpnrwIWUQIPuWGe5plUHb+n0RN3mVacNBHvn7YwrKrXpxgcx6+AAvCookeE6QQplzTfK7h9fHiS0U8oMzoFLrtjqFUTu1kLouuSG9cOYn+dAFHoFuRGB3lH1fNCu78wReqliOB4j78nHVE5SAll5rdj9X6YepsaHvfoQg6jkuz6k8KlHMazUS9\\\\\\\/gv\\\\\\\/m2cJu52zBGLen8nWuB0hV9NLiz183mDSPqRbqziPYBKvc\\\\\\\/tjv5E5Qm2Pgq3be6y\\\\\\\/xpfL6\\\\\\\/GW54wyCH5jnHk3+zh86+TyF8xM93Q7DzhJvMvwKlG\\\\\\\/m8iFlhZvwNcS18Q4onPRAGtX+w2+TjpfOquY1IqQxqIZ1PECgfbnw9g8GR01vXrT7gREFSB19k\\\\\\\/doaPAJJb\\\\\\\/ghggIiVO9DxM5+6puH4sMYVgzJO2mtxeDQwBXRwDZvgqeWh+woYYu1xnTKgKGPSzReCu1OuNWeIYI\\\\\\\/1LZOovX\\\\\\\/FkRTIUPXeiKmNgw6uISN6UsRnslUiUfWi4H6Ff24JgKaNuqzRd1eNeYx9o\\\\\\\/yUxJrg\\\\\\\/\\\\\\\/V+afAsTFRzypi6XuGfF8uUbKorvY\\\\\\\/n\\\\\\\/STtiIuGUMH99\\\\\\\/VMHbtt56ZzLgRlrDorONJwQD2LiHHCwsmiHwBiXqk3jCwv5YUy\\\\\\\/BXaaNfDq3ieEYDKMUNK4xSUhifJUpnX0vmXW0b32fwYk23Y4wMRHrnwq7p03mG95gmaLCkLKGiGoGXNCZBVVpCUphFeT2NJ\\\\\\\/s8lCwFdQ94y9MNFEBk5cOqW3\\\\\\\/Ybajf6SUQgNnv7Rol6GuaECeLyM1fE2j05tZ9ppMbGDtUFGgZsHQaxn75Fa6CpJbdzttAYIOex1Fs5YSkOWWBw1mSYdDtRbo6YmUnJPqpg8Mnd7wDHuM2Lu8zjrfhDJH\\\\\\\/xXjKmJOC5JUXr+xX27YYO2\\\\\\\/lmYhu+iFpfTNEdHTyJiCdCyNt4ffKrcbLmMPxrgrtuHNdrb+Hq4PNVzjqwp+UtAw0CQKcqkyLCCOUj\\\\\\\/ElBO+J0s1RXMLuWm1SkRnJ2dAq1m2KMUyGMx0i9U9Di5mE0oQVzpJruWg0B2QdVEPyT+ee625YB2yojshyn3cFWhQTt+20WZBEfU226FJT9axPk\\\\\\\/6epm0PtWxZpVIBIohhgjxFiIJQiBN+HNRk8UqFzqdvVBbwvJXuT8PbVaPj4ZaiK5wtX7fzjAQUIGJuPqpTDx+7vMK5OeZvtTqbXq1ZiKYGgWq9A\\\\\\\/7Pk\\\\\\\/ubPuLzJ541v\\\\\\\/UVmK3YoNcn2CzPc5k6atvpTJOuI0QLDR1jHYeI7FfuyTCYlOJ+GscTdVJia0ym\\\\\\\/jUizHapz6uzeoASysmpK78tbWvTU3ODWhnCtQWxQqJO0fck+FKHnwPJLU8GEBHvguOtuvz8T2+7QCq5xWErkOlDyzMqOkgrsndsrY1wDDmEL6t6jzkCAWAG0xVpdme+DlKlKkg02iJTnHMI\\\\\\\/nXDg7tj8Co8pYBElmuWhCxPyWRplFmwGnN4US1TI+bqa73OUpGcJ9w2\\\\\\\/AA6Tj1qzTFIICM7YnB0Ua1daBcKYJaPyQj+ghOIF6JPjihM7bn5E0WPtZ\\\\\\\/r71yNgupuODUlMjbUgMgwRKFOqLMGBXOspOt1nZzO67UiAiFhn6E1Riozpog82A667LqpWw221uZRRb+jgsPE+6IfUwLvju1\\\\\\\/EaJjxiJDI4h7pVRkSJRyKMcu+vv+6mHN22ycvTyBQ8W+llFHD2xhiikP25uO1Kdd19e49DIJ5n97JgMfB4xhOEDQt1rZflus6kTOj\\\\\\\/EBiBibQxeugGgBKCJ8XyHMhtncaiGTgml5fpuqDpQwtQHzzI4SGGENsQAymT1ZDwsYVisf8nLoNHg8MfhirOcue5V+wGMcFq3TY8qV9k+K3L34vnPqSlVnGDASDvFYWGC6IRVkCYxkTCzjYnLDpY97BRoKrm6JBbAPr+G4AELg058bh7gXKyfxxfkR3TuxX6OGx19AuHcK34ZGkdZ64t+IXnB67NrV3dpgpKhoovDgBRjyATojVutY0DlpjKZi7KxNufFQ+TOS2c0muFNJUvUpq\\\\\\\/CdXzOSAREgbESjMtyuDyPGM6xv\\\\\\\/UdInLykHZrABYwH2XR6B9Ta2xlNSDdzk3TJpHwD3AppdwOvy1CxjanLUT5VXCQyTKC7gptcnM+KYHp+7Y35pXhlNZYDfGXEf7V0c42e+QYxe44b247MwEj5WEVuSIw0moZUL\\\\\\\/e9RxkQH3gDvxs8\\\\\\\/bWD9T8+joRXnte3L0H0GzHuPDG94tldrWnsBK13LjAtVzA1sSDjUZ0Kin3NaFtFutvpQS5W7lVkjE\\\\\\\/E3bmXHjfCn0PYkvtok6GNYBN+5p14VfAzxcWfRkyxeMpks8prpnl2z9zzxDi\\\\\\\/RQtY5OT1+aP\\\\\\\/1cOuSzCLNeelhYP6QqsyISd2IjqgJwx1oMCjMM3bwSBpSZtyrfv667xy65iHfxFV4Avyg9CBk1h0hW\\\\\\\/qVAoLhcxDHVLZuSth+xaK8LErZwSkZ8wGT1eQDwrFuKplmptSKbH1jPsAm+ffLPU1L5ZbPNEtlTSRPAxxRC3RqAnBSb334hX\\\\\\\/ygTlRBTcvmNuPhdYoH4+xb8dMEYGm\\\\\\\/oxwCT+EM7pt79R+4eJT6u0DEApEXxSEE1dPwtI+sg0d2A1wwjiGedCp+BuhwLb2y2hpn6xPvlD\\\\\\\/xgOfA90sdfSNpyYYZRWFOEKdbIbbUVLLWoBwZD3IR6ZQkjzm1z52db\\\\\\\/25iAik6QDggOjbJBD8ENgEwbNIMFA+6\\\\\\\/YkoTLJgzQYmLUwz8VLjbVGKQFS5WgreGZDMiri92IR0MN7miNmKGJNaIlYJgQqGRy9+3PdEtHm11TLZTX9t1cJ1VhZACW9SqbZKgbinR2dE5V8vE2JK3Pt2o0snWZIsjUUkfgtzjidFXKDX54qy1bZoboFsda47ZYhCELFs3tbHGeNO7ccKZnK2\\\\\\\/5t7TTABmwLZ6A2rW5OU\\\\\\\/mn9MXAhZuvvQfAKher6+Cy5W5AKV0jPohOv5+NqBh2cInSPJy97XLzsurDINjnhJ2G9GBaKndI\\\\\\\/Fs+FJVgJM20GD5epuLWB3EFp6U8Jbr\\\\\\\/5+4Agvy\\\\\\\/7zx5T5CVScDTlgWE8XUrpJqW8yMIpmL6CCX+QayGNt+mkC\\\\\\\/Eldd\\\\\\\/yAULuWjfCKDAkF+5rFRk4r2Z\\\\\\\/SnsgBwFtbUVQjKE1w8KqIkoNiKvH5VZ4X+xnTUgiSYvX2YtKv81OHaGixYkxJzpQqdyz1q4wjIoCvtc3hw6RveV8wFTBN14GRGz1o9VKuVgcogJy1kZh2VzKBd2Np6CwYAmcewOd5Ybmn\\\\\\\/j\\\\\\\/imwDcl0+umc0ombCYMR8C1pNguyMg4P1tKU+ar5C+iIUHZGeTh8vZxCP\\\\\\\/G9KNvu0pG8uVCXRwyvzM2sMhuPfX8UXJUla402IVriprd4GMecFVrOQvMngXDTj6yYgNrbz9Xfdzk9hAOP4geqqDalf3u2aYKf01ew5wGlIvZIHKPvULdhetxc9YQqSrxrOlNLuapFY7reyNSGNRIklLH4WAkOe2eLxQX6wa3wex5G3t+B62EBg1+x\\\\\\\/ryKF9jH1CKvYxMEYJnTnlY3ffwaTxAowwe5lHJi+utdYjFUn3eDC26IByUfTtFrBjxZ\\\\\\\/bnHWpIksdOKAcTYVsYZiniE9F+pwg+7y0fAi2XtkEAABmoL9tGX2O\\\\\\\/aAT91ZCRh7aPXCoqTn\\\\\\\/6xZOFzHXeZc85WZuP8Z5WW70fnAGHE0yHNu3OCG1zVbQxEwLR23BfYFx\\\\\\\/yXT3tBFq+ZRDeL2K5B++BkwDDMQQFNFposgbuKoWUzjv06UWxVqepR\\\\\\\/qz7kGCWTY4qMRPP24AXz+43xIe5axZGSe+2VcweV3ip5c6kmsBIQENistMhoVHpKaAB4fLigFyfAF5CSOCyl58xqA6\\\\\\\/MNZCvsGUUKSUCakcH688fV5l10MJTHDZ\\\\\\\/G8yxwD5xBQ4bwSf7eFxuKqeONCbB\\\\\\\/9V3nNirc7GP3fdY2VJFcBHBckRzP3fiB9MfTh1UdBrBdb1ed\\\\\\\/ZAWSzZnEeGSe9WJSW3DQbzBSiX0Da\\\\\\\/e+LBHAFXbDQJTsmGKcsj2H0E4HdV5\\\\\\\/jnjCzm4NxOIkY7Fafv\\\\\\\/CFBz3LK4Smn6IJSfVPv1zVm\\\\\\\/NX0xjRZIDk4vYmmYLHYebCeM5sY9Wv3q5VJwy1om2lO2fwC5YsAPIYa3CW6rztObNEDZ3laiwqa8XXl6OR5rInYwfu4t9QB2q\\\\\\\/6ZWSAo9V2Op8aPiw\\\\\\\/ov5dGqIMmRkvblOVc3YlAPVgDPOvUlzoZV9DOfe\\\\\\\/sm0CbFVZZt1dFU+mzRiI+FUk79LrdAE1lXFhTd5qPyBkm9dAnCdHk7WeJRkIJgOZie+SW7RK4RBhW7dB5RYwvYOebmmbZq1BxwErm2FXrP+06sE15lWAzGS\\\\\\\/25ilsEJqezbrnt7uNqbQXdXLo6pOG1RGfBg5JcNeK3INAm8YO\\\\\\\/xd\\\\\\\/zX4abo7acgNYlYTUALh91gzCBEf5gJQUujsw4IN5rBEcOp+PC8V67P12oIV4M34MvOObblC78RfZkEzOlWEd2NxvxByEQM3QMWDyWODAJY18+6ZKSYm5SSYxFnCqYl\\\\\\\/g7Si+2g0QQW+bqSzM\\\\\\\/fNt4PiuCskbNNnTuB1xGTnVHUChLhkru2oCfUZtiwJ7dt8kEBWXLws7K237wcAkNcO4RcHT\\\\\\\/fV0VGGNV96S1ufRGw4IcBrHm6CxO4XMZeC7BKr+xq8v81MuO5dSd3dfqtC\\\\\\\/qrW4VKIg9m9xCQA6gAI8bxV8D1cellHZoCrXeQ8YdVcj+Pjzt6hx5CDYgl2UkGwMDaM85FQS0wlpkfKDRB9pxnX8DC5pAJ0ut4ZEA7k\\\\\\\/HOH67EgMwKdE3NbeGXgaAxOpXqBZVGsUYXnJN6ue9dRCxpC0xOrvzLelZEc7GLiAtp+fktnYORoVfaGTk5B2gMNO+yYOWWK1fX7Iv4PoA946mzDj1NjbEDJjsURPx64hE9ukPC2Ni3G34B\\\\\\\/Y\\\\\\\/VMi9DgkbMCPRezNu4ogO7FRwUhBy+OEidJ2GTzkGdghEYc4gZp8fsuWnSzSIEJll3EhPQB+rHBnQlyzCp6+XGNBgrZr2IxnGwqoZnu5DzEjIojeQtOJDJ+aeh0n\\\\\\\/x6LFEQ2Uw4MhRx4pEvYzs1UiWonPsTuRvU5LAmqMsmE151jB33vQ54k0xkGEC+CwIRHZ+ho8v7NoFTKCtLE+QoZV9j84bX3vp6RububGcgwXSAyfMZmGD+XHHO4yUfPw7bszATxLshtVMXHuP0t26A2EuZslU1n0h7NP7gKzeyJ7JuHHso8AkVkuwBcgCMlRtqmq3c+9Xcg7RnPikFvJ55eWLp1XbHXnYm+jaDrDmp6VeOKzmsBfLP2u\\\\\\\/B5xxyBBy\\\\\\\/+ap1ew3nv+AwBwkAN9Chl5RLwTAnhuBi2ZNQyTPYNUHg5bcw4AOsSlN3OYVk\\\\\\\/ruEAIcwYOzUi3UEwPxkFRmVK+Z1SrOxtXW9koXwuKsUGODEwb8RF+fUvsXqsswE89QTG5oP7tCxHnU+r1RDl359qa6L8pE54mgY0LGzdNZHmOHX59Pu5wp6uL\\\\\\\/Cbs++kE2GbVLMpLJSEWNloF+J8T3xR8cZyUFqCfYvg\\\\\\\/juohFHU5o3K\\\\\\\/HnmFTpQh76GE1PT7F4AK2vQMqGvyNdgasVv4+k3Mp3zTPNwHSjlj41FTtWh0B2a9nQT4XE\\\\\\\/M7MiGCAe\\\\\\\/47AbAtz4YD2jjKWpszqK5gDHP3n1rCKv5T3czvR6q2wepMYRTTznLKWO+pA9USPOF6HqDhr0bPtc6HX1DsRNYSYYbmLsBV2p1Eua5IMFoRm9Md5bP8tcILq4Mkr4UF9NB0UicFKPUnudqCYFirNPs+baN\\\\\\\/8ToFqPp8n5++WfXakR2x\\\\\\\/nOjQGhk2moUrrphkqHClN3aj6PKFeSO8VGPHV3Dy9XOi2\\\\\\\/HAlM5SjKAk3Le8H3+UwlPN5EyFErOEff4nShjNiZDlJy3gjz\\\\\\\/5wdBk6Whh71y4DQJNAmMEsP8SJum8fJ9kwlCmEFlupqlDib0\\\\\\\/ROSrrkTzYy+wGwVKnlcpCPcgC+kX2H6rjfCFdbCCJaxIj4yTlvfsm6o4TE1+CkoGW25K\\\\\\\/xJ24ZB6YDqIBNflq0Aqp\\\\\\\/7rCDABwZH3JQRuIr4jE5Ale0uRTL\\\\\\\/A9jEfHeK96OCzqVS+WgNubOuqF7lKjS0LkwjlECBkiT3WGxDkWQbdhSodtlUnBQrLVN3rETZq9aL70NUV7ZAjJQwcyENGabiG7gIUyLuallduDFBSViLT9BXEzHlwEI64hqzBZkAa5ssBZwftEUIlcS1JzUNhXA\\\\\\\/Zt\\\\\\\/QVth+N8U0ePJpFZ3cOzb1QctaqBKEwAIHdbGhLNps3knyYbPZxRK4VIGjTMTzAO0LphvRFtArjz3YwDYgIFaWiHDAYA3kVLBcRTixcP5BpUXmrXXB\\\\\\\/WAiBOegLmDqmDaVQyRMnfcXbQinarjSJfGD4cQktCNal2ZiIYlBNOX9t1ZIBILaRWf3hnZJ79DYYZBaLnrjDFEyhCLklPa6tfmwKk6pPkDc1dkdv6HsY6yl86HToad2B6VnCSsnDdNUcbgVOGlBFUd\\\\\\\/ErgXu8iuejMo4niMfpYIjtfNtQHyaB+Tt9EXXssqFhwcV+ZJq9vOWReoPIqolCIVwkrnwzGn\\\\\\\/0Trek85ApNSjL+lDFTCY8VpA44ZbzDRDstubgvE9e0yjUXw3\\\\\\\/o5HAHenduc7mglOHO8b\\\\\\\/P4k8Mqkudwt4uiLdcSLKAh+UNVXlu2AEMj5mKg3EHi4z9ardvJQVZuiR8kDYY9f7aCuRpbznh4rzCP\\\\\\\/82mlYUbTb4YDL3uatdbtIiXTdyLBntSCKHqAxTvBzwQt90QRzjvb77L4iBoJDU\\\\\\\/2\\\\\\\/TO\\\\\\\/\\\\\\\/6Vak4bqZSLXNH40MlsDayVbHIoc7DTD80qwiRI+MAq32+31am6cHxa2ASc2uupds7Jvx9ftXKsnkvswtHaXNf\\\\\\\/r7eu9jpE+xBPyjYkqeA0ZFBYrf6kRMY1gz\\\\\\\/XsP89x+p58nvT8ayerQg6lcPjtoxlfx2jkXx96xUSFvCyw9kS6\\\\\\\/0q5yH62M4zK1Hm7aV9NKiqxuVRsyTY2TsrzB43HiDfmqd2G3uo3Ju\\\\\\\/c1TKHgnolbbLYo2K0vcYeXjuxP4wHubT3hXnBLB5Tp5SxLbgKy+XT2aaLmIg1cVlN8Ke1a+TLufqg2SB5Th0M3iU0vN+4rV9Ov6EDf2bru6pM2urahbFxVNCAKhJa37AF2TFCnvof9aEM5cRQzWKCam9nAspLmBGIRvtW0O1fZgU1+n64Wmxus2D9cmmtN2gjDnbddUeBY1wfuvRvZz1s6oz4EwWKct0JUoFXq7ACbqqZ8V8m5n2B0uN1Y0gU\\\\\\\/7GHmGj9TnRe3y2hpafOxtB4+MOxPNoZiSqpdrONaPmtfslemaS2QupOzPdXenPz1xqhMfKHzLBX0yBqsRwl98w2+0kkEcjMx0sIfTYt5c1bpW6iTI2fmkurFTjz+jzm2NxmtonCxU\\\\\\\/wVPHfhpkdX1fFJyeGvcGyp3\\\\\\\/gLwe70zh3C\\\\\\\/0UF+iC+vWanrZhf6Q\\\\\\\/IR2QFX\\\\\\\/64zn5TqhLwiLYOAELDDiVtM6mnaTzRFU\\\\\\\/9w8GV\\\\\\\/sveZ\\\\\\\/qL4Jw7W\\\\\\\/O\\\\\\\/oQPH6Ks+1lQoJ1YXauqmmVG0gz4pUQmnEGUhZbateO49CviiKZ6N7eY4nb38JqYyZ0m4vsNr+YyvvsLe\\\\\\\/Cqv2uagugCuINIhJlAo+\\\\\\\/5CCWBkqi2e2S8lxcgJ79FCIGsW7o2X\\\\\\\/Li\\\\\\\/3vDKfYv6gt6y8AUEJhBQ4fN1+w5hY8PFqRdaRln+Wbznbkfvw4i9Rwjqt1SKjPljavNqjUZzE1+KqYuxum2lA4R0zDsJiMPfNmMaHx8ikohqcoLadxAHE1uLvzNXFKO7nrPVxOkBgxawtimTLQ7oX+\\\\\\\/+bs0iNwoiI89vooNRxglT6cEzdQaiZ1el+fVuZ8r7gC0OWCmmuXf62BFzK7wDlGA3HKr5WwT7awLhu1rVmaMm3j3QIKNapjWQW\\\\\\\/nJsZ9o9ZG1bWcfqph+JwI4BLwbjrw+7evnPQUEIyHs1z3TOPJcxqnZu644rxG+KB0bmRRkDSyRJWy0kjXSQQJKrGYYij8gmeefbOrbsWQgdILNG6NNxr6uaeVaNlIPqBiT35Mxk\\\\\\\/U4I9DO+eKF7hb8w8Gzb8bVTRH2p2rcBZ7PMVp7AaEoQoCyAxfGQWFNKiHaREDfjku7gNFwt14D9ckPa7ZkzTuRKmccu1pXO2fDKAVCcxwRecM\\\\\\\/yLtG\\\\\\\/ZrXIjX1lGR0MKxi230\\\\\\\/ZcX5M8x1H9RGjLGtQLx6vBIBktOlu4DQpPjHPwbnKVJX6mNzbJ1YoUtIIH+MLWuzBseBs5wEWuApJeTyPLl6j1RKn2MpfL3e7CrGQZKWU3aQU4NUXaM+UORjf0O81hl\\\\\\\/ngdvAmv0hHL5zJ9WDBCLeeCyGQVNdZwUM0Qsa8cKbkZGja7B6z8uXopkbSIta2Zj3CAg6BXYYilK5IKH9jJKJ1a6PUspf+fcTHf129z+W1BOktYOrkF5MV6AzH2596HZPTTXmFVT7B1f\\\\\\\/drQHla4XEGpXqPBAa9pgXlb5Y9pc3ACF5RHBHBLWt2y9Zpa7PatLWRCKbewhrss6Y04KvZbOr9J34cQj+d+Xon+dDZ\\\\\\\/4m8RR\\\\\\\/iXaoaEt0hUUnAcwmRcj0sGPGMjthL0uVipTZFTNzrw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cNywJImXG6DthrT\\\\\\\/beFz5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cc6ef29235e5f64e7b8f1bcd2b13a48d276b4cdd2caaaf7d2f79847a816752c4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oICNOrWyrmvYJ7NkibO25RTdy53BWXB+iCR6wOPr1D4BXcsZjAlePSUyWSAcniiU3AOCX9eA3V6EzL7iV1EbDcGVEJrMq763qqgnGtTK8m3Mus8zHqZ4NUi8Fevksg7amqq8S4KjOBzILKjapRn7sRNjUxoJAZVtcyeUYLFgvnkSZEHFM78qcTBvDrSLflR4eJtSltH2tVQkAWdRywO529vrFZMJLTN1aiZxnO3Sp7\\\\\\\/fSoD8gGhuIRWu8BhdPjVaOStqHLGPz5aNSU0QayFuum7rlM4OKb4EIfLZmcwIZzPMu\\\\\\\/g1w42rKW48+nAfhmcTDovQyavyNqxmpL8QUNf6NTwylTCV2ffLL6oYup2uAvEq\\\\\\\/nZh0kEw9GfzkJDGCzzmguORyGBzreH4jYc4oUDamC0Eo+\\\\\\\/3z4jlwp2wQ2HI9XQeTwel+R\\\\\\\/n7j77JGGTTxoEDkQD0X55jGzqa2aKKAhqXLP1kp1RZf0hxWnrKdhb5H6KEOpCtiJxZhB\\\\\\\/E4bOKTYVw5\\\\\\\/4gRZTwKHJH37yHw1CvgRXW99444zJ5fL0YUMZDJuUPwixShINqbur\\\\\\\/4mc35hyCO9CDkWsB\\\\\\\/qmjdVUvLJe7E7HtgJwa38nZCNBXMVhssHazvwwIWoBO+C5Oey51V5j7TucsbmS7ptcqzigIkYRmnk+kntmHAJMpiddreJkFvFBrBtVjkOW+\\\\\\\/veb+BdvwF9zM5jx8mqNOO75608rMm5OS+RqCm8yVlvdG8YMlC29JutkQAkSuSf+egxksBQERHSo6WA1mkqWHUbvAH0I8m0uW5w7PV84SQ+dVZE3mIiDa8H\\\\\\\/1QMj\\\\\\\/Y1AOHOwPsbkErylB5AAEGKqyzyboYFnBdw8K29CLtU4MjY9tzu410ryNONVg5KgZz1JwYVNf1vIecfky548VEa5peLv8A5vo36bECCqYhWKcfafec2JU01krbGAJJf36c6dM+fIdW395eNmUTx80ZUfHsBZ6TVzdk13zTjJuAaRMr4vsONaPJ8FJhMvDhENO2ZGpjHooYyBeje1WsI2g7Icn2gqZTIA43kQNJihW+u9WvCluDDw+17mnGoELfWxVEM0oJKCYdrm4LALtd8Uvvf\\\\\\\/mXxNSCGoZOCcka860XMq+WTYpzu+TED2US3+DhqJiecu+gMCN12HTgATNXEwSR1TazkGBMUMKp7TPmZ+3uJUAW3UwxSdN7OXJXARRduKgVzXmyL9oz32bHzGh\\\\\\\/47c68JB1vlwG9Vpa9w0HSyIUX259GUZsDSxgdzvkmmT36PXngdmAzP38LnNlp9P9R110boqvO3+fbKnBvbnPy9GvMzDBBokhiSBz9hJldZqbSv6N4QWlHEZksxThqhCN55KT2Lj4JwW9vquTyTfH99QlSjAmCbgOgPp1Di5k7cY7euixnTKwrIUAuWkVcEJTUjO2yv3xw0H+Panszgg4o2G71hyNFVoPrAmH6g8AnJPMc\\\\\\\/E7OVdDYskxe\\\\\\\/4EaTMhrKi74zk53H2U5LcNiotomzgtN5jVvDm\\\\\\\/kNGhWNUTMpW7wF6kNJpJwRHCmZPXgMm4lnmb1ovsJftvU4GD\\\\\\\/dHLgR65yIYryHMZSbyJmbuAAMRotcnfw8PHdxjYsheKxzbXizxtnSvPMZsx2pie5qhCjEOE64GvSM1\\\\\\\/VDEoFlhxFFCGMDszPHCggBVI\\\\\\\/CEuOAg7npw5YNsj9J+D5skufmqRNsRYvQI1XuHxOxKhOUvXsRt5gGs4WVgLDaWVH9bqHktWTn0lMzpM8uJS7IH5AXc+g6X7GCLp489FQ0701mjBfOttztz03FZTx0hNJZ6896NLKImJlafLdAyL7NRZhC5DzWZp4lu1ZwQC81ix0n4Tv8F+8D\\\\\\\/ixVbIguFm\\\\\\\/5YS13n6YL+FMWqYE8Qiswx76RvckH+F16OWdCLDPOVa+SGVLDPLixSmmn1HSRWH0yDy\\\\\\\/d6UcBl0dl5qkZjNDiUAzURgARa5dfEFjScyMYwRquhwX+nkhWTnG1sCiMaChlG0as2WYorJM8eGpSppFHUZ8RQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QhYsOvccOA3t+rn7nt9q3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022937717fd243cab2fe63a60399b412802d94c9311e796d244dd04f5e0952d9ce3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459419 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/197" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IOPNfX7E6vs5yVsAyrX7cUOTImuyxG_S9gxCY4dZWKQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/197" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lWQamEYf2W6lod0KaUBE4L2XFLbPmty-ePjP04Xmrl8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/181" title="Redundant Canonical and Noncanonical Caenorhabditis elegans p21-Activated Kinase Signaling Governs Distal Tip Cell Migrations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/205" title="Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459419" id="node459419" data-pisa="ggg;3/2/197" data-pisa-master="ggg;3/2/197" data-apath="/ggg/3/2/197.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frank Technow</span>, <span class="highwire-citation-author" data-delta="1">Anna Bürger</span> and <span class="highwire-citation-author" data-delta="2">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">197-203;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004630 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459419" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Frank.Technow@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">Frank.Technow@uni-hohenheim.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna Bürger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BB%C3%BCrger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%BCrger%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BB%25C3%25BCrger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/2/197" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459419" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/197.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459419" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/197.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459419" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/197.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459419" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/2/197.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459419" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Northern corn leaf blight (NCLB), a severe fungal disease causing yield losses worldwide, is most effectively controlled by resistant varieties. Genomic prediction could greatly aid resistance breeding efforts. However, the development of accurate prediction models requires large training sets of genotyped and phenotyped individuals. Maize hybrid breeding is based on distinct heterotic groups that maximize heterosis (the dent and flint groups in Central Europe). The resulting allocation of resources to parallel breeding programs challenges the establishment of sufficiently sized training sets within groups. Therefore, using training sets combining both heterotic groups might be a possibility of increasing training set sizes and thereby prediction accuracies. The objectives of our study were to assess the prospect of genomic prediction of NCLB resistance in maize and the benefit of a training set that combines two heterotic groups. Our data comprised 100 dent and 97 flint lines, phenotyped for NCLB resistance <em>per se</em> and genotyped with high-density single-nucleotide polymorphism marker data. A genomic BLUP model was used to predict genotypic values. Prediction accuracies reached a maximum of 0.706 (dent) and 0.690 (flint), and there was a strong positive response to increases in training set size. The use of combined training sets led to significantly greater prediction accuracies for both heterotic groups. Our results encourage the application of genomic prediction in NCLB-resistance breeding programs and the use of combined training sets.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li><li class="kwd"><a href="/keyword/disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-disease-resistance" rel="nofollow">disease resistance</a></li><li class="kwd"><a href="/keyword/northern-corn-leaf-blight" class="hw-term hw-article-keyword hw-article-keyword-northern-corn-leaf-blight" rel="nofollow">northern corn leaf blight</a></li><li class="kwd"><a href="/keyword/heterotic-groups" class="hw-term hw-article-keyword hw-article-keyword-heterotic-groups" rel="nofollow">heterotic groups</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li></ul><p id="p-5">Northern corn leaf blight (NCLB), caused by the pathogen <em>Setosphaeria turcica</em> (anamorph <em>Exserohilum turcicum</em>), is a serious threat to maize (<em>Zea mays</em> L.) cultivation worldwide, reportedly causing yield losses of more than 50% (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Raymundo and Hooker 1981</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Perkins and Pederson 1987</a>). NCLB can be efficiently controlled through cultivation of resistant varieties (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dingerdissen <em>et al.</em> 1996</a>), giving breeding for NCLB resistance a high priority for disease control. Today’s availability of high-density molecular marker data greatly facilitates molecular resistance breeding approaches (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Collard and Mackill 2008</a>) and the understanding of the genetic architecture of resistance traits. Previous studies on resistance to NCLB point to a complex genetic architecture with many quantitative trait loci (QTL) distributed throughout the genome (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Van Inghelandt <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wisser <em>et al.</em> 2006</a>). For instance, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> (2011)</a> identified 29 QTL for NCLB resistance, each with a small effect. This might hamper the application of traditional marker assisted breeding approaches.</p><p id="p-6">Genomic prediction, developed in dairy cattle breeding, uses all available marker data of a genotyped and phenotyped training set for building a prediction model without an intermediate QTL detection step (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Meuwissen <em>et al.</em> 2001</a>). Subsequently, this model is used to predict genotypic values of nonphenotyped individuals for which only marker data are available. The major advantage of genomic prediction is that all polymorphisms affecting a trait are modeled, regardless of effect size, making it a potentially powerful approach for a complex trait like NCLB resistance.</p><p id="p-7">Initial studies on genomic prediction applied to maize showed promising results with highly accurate predictions for traits like dry matter yield and plant height (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>). Although no results are available for genomic prediction of disease resistance in maize, it has been successfully applied to predict resistance to <em>Fusarium</em> head blight (FHB) in barley (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lorenz <em>et al.</em> 2012</a>) and wheat (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rutkoski <em>et al.</em> 2012</a>).</p><p id="p-8">In dairy cattle breeding, genomic prediction is now applied routinely for large breeding populations like Holstein Friesian (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> 2009b</a>). However, its application to small breeds seems to be more challenging, mainly because difficulties of assembling large-enough training sets. To make the advantages of genomic prediction available for small breeds as well, the possibility of combined, multibreed training set were studied by several authors (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Weber <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de Roos <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> 2009a</a>). These authors found this approach to have the potential of increasing the prediction accuracies for small breeds. The approach of using a training set that combines different groups also has been studied in a plant breeding context for genomic prediction of oat (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asoro <em>et al.</em> 2011</a>) and barley (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lorenz <em>et al.</em> 2012</a>). The results of these studies, however, were inconclusive.</p><p id="p-9">In maize breeding, the two parental lines of a hybrid are taken from genetically distinct heterotic groups (dent and flint in Central Europe) for maximum exploitation of heterosis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Messmer <em>et al.</em> 1993</a>). For resistance traits with mainly additive gene action, as applies to NCLB resistance (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carson 1995</a>), both parents of a hybrid should have good resistance. The improvement of the resistance level of the inbred lines within each heterotic group requires allocating the available resources to parallel breeding programs. This makes it more challenging to establish a sufficiently sized training set within each heterotic group. Therefore, enlarging the training set via combination of data from both heterotic groups also would be an interesting approach for genomic prediction in maize breeding. The objectives of this study were to (1) assess the prospects of genomic prediction of NCLB resistance in maize and (2) compare the prediction accuracy of separate training sets for each heterotic group <em>vs.</em> combining both heterotic groups in a single training set.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material and phenotypic evaluation</h3><p id="p-10">Our genetic material consisted of 100 dent and 97 flint maize inbred lines, representing the breeding program of the University of Hohenheim. More detailed information on the history of this breeding program is given by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Technow <em>et al.</em> (2012)</a>. All lines were evaluated for NCLB resistance <em>per se</em> in the trial stations Bingen (Rhineland-Palatinate, Germany) and Pocking (Bavaria, Germany) in 2010. Plants were grown in single row plots, laid out in a 20 × 10 alpha-design with two replications at each location. <em>E. turcicum</em> inoculum was artificially applied using pathogen extract of naturally infected leafs collected at each location in 2009. NCLB severity was visually rated for each plot according to the lesion spot development in the middle-to-upper part leaves on a scale from 1 (susceptible) to 9 (resistant). NCLB severity ratings were adjusted for effects pertaining to the environment and field design. The dent heterotic group had a phenotypic mean of 3.28 (range, 0.98−6.45), the flint heterotic group had a phenotypic mean of 3.77 (range, 1.29−5.84). The heritability (<em>H</em><sup>2</sup>) on an entry mean basis was 0.76 for dent and 0.64 for flint. The adjusted entry means, computed as best linear unbiased estimates by using a mixed model with genotypes treated as fixed effects, are provided in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004630/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Genomic data</h3><p id="p-11">All inbred lines were genotyped with the Illumina single-nucleotide polymorphism (SNP) chip MaizeSNP50 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ganal <em>et al.</em> 2011</a>) containing 57,841 SNPs. Markers with more than 5% missing data within any heterotic group were removed. Because the inbred lines were in very advanced selfing generations (&gt; S<sub>6</sub>), heterozygous marker genotypes also were treated as genotyping errors and considered as missing. “BEAGLE” software (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2009</a>), version 3.3.1, was used to impute all remaining missing marker genotypes, resulting in 37,908 SNP markers available for further analysis.</p><p id="p-12">Second-order natural smoothing spline regression was used to visualize the linkage disequilibrium (LD, calculated as <em>r</em><sup>2</sup>) as a function of the physical distance Δ in Mbp between markers on the same chromosome. This was done separately for the set of dent lines, flint lines, and across both sets. For LD calculations within heterotic groups, all markers with a minor allele frequency (MAF) &gt; 0.05 within the corresponding group and for LD calculation across groups all markers with a MAF &gt; 0.05 within both heterotic groups were included.</p><p id="p-13">The linkage phase persistence across heterotic groups was computed following <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Technow <em>et al.</em> (2012)</a>. First, all marker pairs were binned according to the physical distance Δ in 100 discrete bins of 0.05 Mbp width. Second, the proportion of marker pairs, with identical linkage phase within both heterotic groups, was calculated for each bin. Again, second-order natural smoothing spline regression was used to visualize this proportion as a function of the distance between the center values of the bins.</p><p id="p-14">A principal component analysis, based on the full 37,908 SNP marker profiles of the inbred lines, was used to investigate the genetic distinction of the dent and flint heterotic groups.</p></div><div id="sec-4" class="subsection"><h3>Prediction approaches and their validation</h3><p id="p-15">We investigated the potential of the following three prediction approaches (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>): (1) the “within” prediction approach, where lines used for fitting the model (training set) and lines to be predicted (prediction set) belonged to the same heterotic group; (2) the “across” prediction approach, where lines in the training set belonged to another heterotic group than lines in the predicting set and; (3) the “combined” prediction approach, where lines of both heterotic groups were combined in a training set to predict either lines in a flint or dent prediction set. The dent and flint training sets comprised a random sample of <em>N<sub>t</sub></em> = 75 (25, 50) lines from the corresponding group. The remaining dent and flint lines then made up the prediction sets. The training sets for the “combined” prediction approach were created by merging the training sets from both heterotic groups (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The “prediction accuracy” was calculated by dividing the correlation between the predicted genotypic values and observed phenotypic values by <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-1.gif"/></noscript></span></span>, following common practice (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Legarra <em>et al.</em> 2008</a>). The process of generating training and prediction sets was repeated 100 times for all three levels of <em>N<sub>t</sub></em> in the manner described. All prediction approaches were applied to the same random splits of the data set into training and prediction set, and a paired <em>t</em>-test was used to determine the significance of differences in prediction accuracy observed between the “combined” and “within” prediction approaches. Because the training and prediction sets produced over the replications are always drawn from the same data set, the replications are not independent. To account for this, we used the correction method proposed by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nadeau and Bengio (2003)</a>. Their adjustment is the stronger, the larger the size of the prediction set (<em>N<sub>p</sub></em>) is relative to <em>N<sub>t</sub></em>, because the larger <em>N<sub>p</sub></em>/<em>N<sub>t</sub></em>, the more overlapping the sampled prediction sets will be. Therefore, the test is very conservative for <em>N<sub>t</sub></em> = 25 and <em>N<sub>t</sub></em> = 50.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/197/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic illustration of the investigated prediction approaches: &#x201C;within&#x201D; prediction approach (full line), &#x201C;across&#x201D; prediction approach (dotted line) and &#x201C;combined&#x201D; prediction approach (dashed line). Nt corresponds to the training set size and Np to the size of the prediction set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-269170619" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic illustration of the investigated prediction approaches: &#x201C;within&#x201D; prediction approach (full line), &#x201C;across&#x201D; prediction approach (dotted line) and &#x201C;combined&#x201D; prediction approach (dashed line). &lt;em&gt;N&lt;sub&gt;t&lt;/sub&gt;&lt;/em&gt; corresponds to the training set size and &lt;em&gt;N&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; to the size of the prediction set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/197/F1.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/197/F1.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/197/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/197/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462352" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-16" class="first-child">Schematic illustration of the investigated prediction approaches: “within” prediction approach (full line), “across” prediction approach (dotted line) and “combined” prediction approach (dashed line). <em>N<sub>t</sub></em> corresponds to the training set size and <em>N<sub>p</sub></em> to the size of the prediction set.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Prediction model</h3><p id="p-17">The Bayesian version of genomic best linear unbiased prediction (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kärkkäinen and Sillanpää 2012</a>) was used to predicted genotypic values as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-2.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>μ<sub>i</sub></em> denotes the linear predictor, <em>y<sub>i</sub></em> is the scaled and centered phenotypic entry mean of inbred line <em>i</em> and <em>u<sub>i</sub></em> its total genetic value. The Gaussian density function is denoted by <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-5.gif"/></noscript></span></span> refers to the residual variance. The design row vector <strong><em>X</em></strong><em><sub>i</sub></em> codes for the fixed effects in <strong><em>β</em></strong>. Depending on the prediction approach, these were either the heterotic group effects (“combined” prediction approach) or a single intercept (“within” and ’across’ prediction approaches). We used a uniform, improper prior for <strong><em>β</em></strong>. The prior for <em>u<sub>i</sub></em> was <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-6.gif"/></noscript></span></span>, where <span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-7.gif"/></noscript></span> denotes the multivariate-Gaussian density function, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-8.gif"/></noscript></span></span> the polygenic variance, and <strong><em>A</em></strong> the realized additive relationship matrix. The latter was computed from the marker data according to Method 1 of <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">VanRaden (2008)</a>. Finally, the priors for the variance components <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-10.gif"/></noscript></span></span> were uninformative scaled inverse χ<sup>2</sup> distributions with scale factor equal to 1/2 and degree of freedom parameter equal to 2.</p><p id="p-18">A single Gibbs-sampling chain run for 50,000 iterations was used for sampling from the marginal posterior distributions of the parameters involved. The first 20,000 iterations of the chain were discarded as burn-in, and only every 20<sup>th</sup> sample of the remaining iterations was stored. The posterior means of <strong><em>β</em></strong> and <em>u<sub>i</sub></em> were used to predict the genotypic values. The R (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">R Development Core Team 2011</a>) package “MCMCglmm” (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hadfield 2010</a>) was used for Gibbs-sampling.</p></div><div id="sec-6" class="subsection"><h3>Computation of realized additive relationship matrix</h3><p id="p-19">Only markers informative for a given prediction approach were considered for computing <strong><em>A</em></strong>. Consequently, markers had to segregate (always meaning MAF &gt; 0.05) in at least one heterotic group for the “combined” prediction approach, in both heterotic groups for the “across” prediction approach and in the corresponding heterotic group for the “within” prediction approach. Because the markers were distributed unevenly across the genome, the number of markers was reduced to a density of one marker per Mbp, with a distance of approximately 1 Mbp between adjacent markers, to ensure equal weighing of all genomic regions when computing <strong><em>A</em></strong>. This resulted in a total of 1724 (“combined”), 1513 (“across”), and 1638 (“within”) markers finally used. These marker data sets are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004630/-/DC1/FileS1.zip">File S1</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>LD and genetic relationship between lines</h3><p id="p-20">LD between markers with Δ less than 0.5 Mbp was at very high levels of greater than 0.30 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). It then decreased but still amounted to ≈0.25 within heterotic groups and ≈0.20 across at Δ = 1.0 Mbp. Beyond Δ = 1.0 Mbp, LD continued to decrease slightly but remained considerably greater than a value of 0.10 for the whole range of Δ values considered. In general, the LD within the group of dent lines was slightly greater compared with the group of flint lines, whereas the LD across the set of dent and flint lines was lowest (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/197/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) LD (calculated as r2) as a function of physical distance (&#x394;) in Mbp between markers on the same chromosome for the group of dent lines (full line), flint lines (dashed line), and across both heterotic groups (dotted-dashed line). (B) Proportion of markers with equal linkage phase across heterotic groups as a function of &#x394; in Mbp between markers on the same chromosome. The horizontal gray line indicates the value 0.5. LD calculations within heterotic groups included all markers with MAF &gt; 0.05 within this group; LD calculation across groups included all markers with MAF &gt; 0.05 within both heterotic groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-269170619" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) LD (calculated as &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) as a function of physical distance (&#x394;) in Mbp between markers on the same chromosome for the group of dent lines (full line), flint lines (dashed line), and across both heterotic groups (dotted-dashed line). (B) Proportion of markers with equal linkage phase across heterotic groups as a function of &#x394; in Mbp between markers on the same chromosome. The horizontal gray line indicates the value 0.5. LD calculations within heterotic groups included all markers with MAF &gt; 0.05 within this group; LD calculation across groups included all markers with MAF &gt; 0.05 within both heterotic groups.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/197/F2.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/197/F2.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/197/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/197/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462448" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-21" class="first-child">(A) LD (calculated as <em>r</em><sup>2</sup>) as a function of physical distance (Δ) in Mbp between markers on the same chromosome for the group of dent lines (full line), flint lines (dashed line), and across both heterotic groups (dotted-dashed line). (B) Proportion of markers with equal linkage phase across heterotic groups as a function of Δ in Mbp between markers on the same chromosome. The horizontal gray line indicates the value 0.5. LD calculations within heterotic groups included all markers with MAF &gt; 0.05 within this group; LD calculation across groups included all markers with MAF &gt; 0.05 within both heterotic groups.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The proportion of marker pairs with the same linkage phase in both heterotic groups showed trends similar to the LD (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). It reached a maximum of ≈ 0.75 for marker pairs with a distance Δ in Mbp close to zero and then decreased rather quickly. However, at Δ = 1.0 Mbp it still remained just below 0.6. Afterward, it decreased slowly toward the value 0.5 but nonetheless remained slightly above this value over the whole range of Δ values considered.</p><p id="p-23">The mean pairwise relationship coefficient, from the <strong><em>A</em></strong> matrix computed for the “combined” prediction approach, between dent lines was 0.46 with standard deviation of 0.38, and between flint lines 0.49 with standard deviation of 0.32. Between the dent and flint lines, the mean was −0.49 with standard deviation of 0.18 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The principal component analysis showed a very clear genetic distinction of the dent and flint heterotic groups (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/197/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Density histograms of pairwise relationship coefficients between dent lines (A), flint lines (B) and between dent and flint lines (C). Values are elements of the realized additive relationship matrix A as computed for the &#x201C;combined&#x201D; prediction approach." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-269170619" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Density histograms of pairwise relationship coefficients between dent lines (A), flint lines (B) and between dent and flint lines (C). Values are elements of the realized additive relationship matrix &lt;strong&gt;&lt;em&gt;A&lt;/em&gt;&lt;/strong&gt; as computed for the &#x201C;combined&#x201D; prediction approach.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/197/F3.medium.gif" width="252" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/197/F3.medium.gif" width="252" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/197/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/197/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462365" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-24" class="first-child">Density histograms of pairwise relationship coefficients between dent lines (A), flint lines (B) and between dent and flint lines (C). Values are elements of the realized additive relationship matrix <strong><em>A</em></strong> as computed for the “combined” prediction approach.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/197/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of principal component (PC) 1 vs. PC 2 scores based on 37,908 SNP markers of all 100 dent lines (red dots) and 97 flint lines (blue squares)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-269170619" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of principal component (PC) 1 &lt;em&gt;vs.&lt;/em&gt; PC 2 scores based on 37,908 SNP markers of all 100 dent lines (red dots) and 97 flint lines (blue squares).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/197/F4.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/197/F4.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/197/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/197/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462432" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-25" class="first-child">Plot of principal component (PC) 1 <em>vs.</em> PC 2 scores based on 37,908 SNP markers of all 100 dent lines (red dots) and 97 flint lines (blue squares).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Prediction accuracy</h3><p id="p-26">Overall, prediction accuracies increased with increasing <em>N<sub>t</sub></em>. For example, the prediction accuracy of dent (flint) lines increased from 0.366 (0.389) at <em>N<sub>t</sub></em> = 25 to 0.589 (0.576) at <em>N<sub>t</sub></em> = 50 to 0.706 (0.690) at <em>N<sub>t</sub></em> = 75 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, combined training sets). The “combined” prediction approach resulted in greater prediction accuracies than those of the “within” prediction approach for all levels of <em>N<sub>t</sub></em>. Thereby, the largest differences were observed at <em>N<sub>t</sub></em> = 75 with 0.065 for dent lines and 0.082 for flint lines (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). These differences were also statistically significant (<em>P</em> &lt; 0.05).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462298/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462298/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average and SD of prediction accuracies over the 100 replications of the validation procedure for northern corn leaf blight resistance based on a Bayesian GBLUP model using either pure dent, pure flint, or combined training sets of size <em>N<sub>t</sub></em> to predict either the dent or flint lines</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">Prediction accuracies of the “across” prediction approach were relatively low in both cases, whereby prediction of flint lines using a dent training set was more accurate than vice versa.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><div id="sec-11" class="subsection"><h3>Merit of selection based on genomic prediction</h3><p id="p-29">Successful adoption of genomic prediction approaches to plant breeding programs depends on their advantage over traditional selection methods. For quantifying this potential advantage, genomic prediction can be viewed as an indirect selection method. The merit of indirect selection per unit time, relative to the merit of direct selection, can be described as the indirect selection response (<em>CR<sub>X</sub></em>) divided by the direct selection response (<em>R<sub>X</sub></em>). Following <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Falconer and Mackay (1996)</a>, this ratio can be calculated as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>i<sub>Y</sub></em> is the selection intensity applied on the indirect trait and <em>i<sub>X</sub></em> the selection intensity on the target trait, <em>L<sub>Y</sub></em> and <em>L<sub>X</sub></em> are the cycle lengths of indirect and direct selection, respectively, <em>H<sub>X</sub></em> is the square root of the heritability of the target trait, and <em>H<sub>Y</sub></em> the square root of the heritability of the indirect trait. The latter is assumed to be 1 in the case of genomic data. The genetic correlation between the target and indirect trait is denoted by <em>r<sub>A</sub></em> and corresponds to the prediction accuracy in our context. A ratio greater than 1 indicates superiority of indirect selection over direct selection. Assuming equal selection intensities, we can arrange equation (2) to the inequality<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>It describes the merit of indirect selection as a function of the selection cycle lengths. Accordingly, indirect selection is superior to direct selection when <em>L<sub>Y</sub></em> is shorter than a certain fraction of <em>L<sub>X</sub></em>, which depends on <em>H</em><sup>2</sup> of the target trait and the accuracy of genomic predictions.</p><p id="p-30">Using our <em>H</em><sup>2</sup> estimates and the accuracies observed for the “combined” prediction approach at <em>N<sub>t</sub></em> = 75, selection for NCLB resistance based on genomic predictions would already be superior to phenotypic selection when <em>L<sub>Y</sub></em> is less than 81% (dent) or 86% (flint) of <em>L<sub>X</sub></em>. These are promising numbers, given that other authors found genome based breeding programs to require less than 50% of the time as traditional programs (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em> 2010</a>).</p><p id="p-31">Equation (3) assumed that <em>i<sub>Y</sub></em> = <em>i<sub>X</sub></em>. However, after sufficiently sized training sets are established, which requires phenotypic as well as genotypic data, <em>i<sub>Y</sub></em> can be raised almost arbitrarily by genotyping large numbers of individuals. When novel techniques such as genotyping-by-sequencing are used, this could be done at very competitive costs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> 2011</a>). Thus, in the near future, it is likely that <em>i<sub>Y</sub></em> &gt; <em>i<sub>X</sub></em>, which would add to the advantage of selection based on genomic predictions.</p><p id="p-32">The potential advantage of selection based on genomic predictions was also pointed out by other authors. For example, in simulation studies, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em> (2010)</a> and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo and Yu (2007)</a> found genomic breeding programs to be clearly superior over traditional recurrent selection programs for complex traits in maize.</p><p id="p-33">Our results and conclusions are based on rather low training set sizes <em>N<sub>t</sub></em>. Other studies in crop species too reported high prediction accuracies with surprisingly low <em>N<sub>t</sub></em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Lorenz <em>et al.</em> 2012</a>). One likely explanation for this are the commonly low effective population sizes <em>N<sub>e</sub></em> and consequently low effective number of independent chromosome segments <em>M<sub>e</sub></em>, found in plant breeding populations (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> 2012</a>). Following <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> (2010b)</a>, the expected prediction accuracy can be calculated as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <em>M<sub>e</sub></em> = 2<em>N<sub>e</sub>G</em>/<em>log</em>(4<em>N<sub>e</sub>G</em>) and <em>G</em> is the genome length in Morgan. From equation (4), it can be seen that low <em>M<sub>e</sub></em> combined with high <em>H</em><sup>2</sup> can lead to a high expected prediction accuracy at low <em>N<sub>t</sub></em>. For example, with <em>N<sub>e</sub></em> = 25, which is at the upper end of the range postulated in populations of maize inbred lines (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guzman and Lamkey 2000</a>), and a genome length of 16.34 Morgan (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Martin <em>et al.</em> 2011</a>), the expected prediction accuracy at <em>N<sub>t</sub></em> = 25 is 0.38 (dent) and 0.35 (flint) and 0.58 (dent) and 0.55 (flint) at <em>N<sub>t</sub></em> = 75. These values agree well with our results. Nevertheless, <em>N<sub>t</sub></em> will likely be greater in routine applications by plant breeders. This is expected to increase the prediction accuracy, and thereby the merit of selection based on genomic predictions, even further.</p></div><div id="sec-12" class="subsection"><h3>Merit of combined training sets</h3><p id="p-34">In accordance with other studies on genomic prediction in crops (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Asoro <em>et al.</em> 2011</a>) and livestock (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Weber <em>et al.</em> 2012</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Daetwyler <em>et al.</em> 2010a</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de Roos <em>et al.</em> 2009</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> 2009a</a>), we observed an increase in prediction accuracy when using a combined training set as compared with using training sets comprising lines from one heterotic group only.</p><p id="p-35">Interestingly, this was already the case at a comparatively low marker density of 1 Mbp<sup>–1</sup> or about 1600–1700 markers. However, there is consensus among the aforementioned authors that very high marker densities are required to take advantage of combined training sets. This is to ensure consistent linkage phases between QTL and markers across groups, a necessary condition for the combined prediction approach to work. However, despite several centuries of separation of dent and flint (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rebourg <em>et al.</em> 2003</a>), our results showed that there is still consistent LD across the heterotic groups, even for markers at greater distances (<em>i.e.</em>, the proportion of marker pairs with equal linkage phase was considerably greater than 0.5, the value representing independence). Further, we did not find that using higher marker densities led to an increase in the prediction accuracy (results not shown). Similar results on the required marker density for prediction purposes in elite germplasm of maize were reported by <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> (2012)</a>, who found that the prediction accuracy did not increase markedly when increasing the number of markers beyond 2500.</p><p id="p-36">Genomic best linear unbiased prediction, the prediction method used by us, uses marker data merely for estimating the realized relationship between individuals. Marker effects based methods (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kärkkäinen and Sillanpää 2012</a>) might be able to capitalize more on higher marker densities (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> 2012</a>). However, using a ‘BayesB’ type algorithm (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Meuwissen <em>et al.</em> 2001</a>) in the implementation employed by <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Technow <em>et al.</em> (2012)</a> did not yield greater prediction accuracies (results not shown). Likely, this was because such algorithms require much larger training set sizes to overcome the added complexity of the model due to greater dimensionality and redundancy of the predictor set.</p><p id="p-37"><a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Lorenz <em>et al.</em> (2012)</a> studied the potential of combined training sets for predicting resistance to FHB and related deoxynivalenol toxin (DON) production in barley. They did not find that using a combined training set of 200 individuals from two groups increased the prediction accuracy over using just 100 individuals from either group for predicting individuals from the same group. However, their populations seemed to be rather unresponsive to increases in training set size <em>N<sub>t</sub></em>. For example, even doubling <em>N<sub>t</sub></em> from 100 to 200 for prediction within a group just barely increased the prediction accuracy (4% for FHB and 10% for DON). Therefore, combining 100 + 100 individuals from different groups should not be expected to yield much improvement either. Focusing on prediction accuracies within heterotic groups, we found that increasing <em>N<sub>t</sub></em> from 25 to 50 increased prediction accuracies by 64% (dent) and 46% (flint). In contrast, the same increase in <em>N<sub>t</sub></em> for the populations of <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Lorenz <em>et al.</em> (2012)</a>, increased their prediction accuracies just by 17% and 26% for FHB and DON respectively. Further, we observed a relative increase in accuracy from <em>N<sub>t</sub></em> = 50 to <em>N<sub>t</sub></em> = 75 that was considerably greater than the relative increase <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Lorenz <em>et al.</em> (2012)</a> observed from <em>N<sub>t</sub></em> = 50 to <em>N<sub>t</sub></em> = 100. Thus, responsiveness to <em>N<sub>t</sub></em>, which may be a function of the effective population size, seems to be a key requirement for an advantage of combined training sets.</p><p id="p-38">Compared with the increase in prediction accuracy when increasing <em>N<sub>t</sub></em> by adding individuals from the same group (<em>e.g.</em>, moving from <em>N<sub>t</sub></em> = 25 to <em>N<sub>t</sub></em> = 50 within groups), the increase in prediction accuracy was only marginal, when the same increase in <em>N<sub>t</sub></em> was achieved by adding individuals from the other group. This was because the information added in the latter case was much lower than in the former, as is exemplified in the low linkage phase consistency between the heterotic groups. It is reasonable to assume, however, that the increase would have been more pronounced when the groups were less genetically distant then our dent and flint groups.</p><p id="p-39">Nonetheless, under a fixed budget that has to be allocated to all heterotic groups, increasing <em>N<sub>t</sub></em> within one group can only be achieved by decreasing it in another. This would necessarily lead to differential selection progress, which is undesirable when the heterotic groups are developed reciprocally. Therefore, combining training sets is still worthwhile, since the gain in prediction accuracy obtained is essentially cost neutral and does not lead to a negligence of the other group.</p><p id="p-40">Balancing the large increase in prediction accuracy when moving from <em>N<sub>t</sub></em> to 2<em>N<sub>t</sub></em> within one group and the goal of even development of both groups; however, is possible with an alternating selection scheme. Here, the full phenotyping capacity would be applied to one group in one cycle and to the other group in the next cycle and so on. Thus, always one group would be selected based on a training set of size 2<em>N<sub>t</sub></em> from the same group and one based on across group predictions. Following <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Falconer and Mackay (1996)</a> and assuming constant selection intensities, heritabilities and genetic variances across cycles, the aforementioned alternating scheme would lead to a greater selection gain over two cycles than a scheme based on the combined prediction approach, when <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-14.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-15.gif"/></noscript></span></span> denotes the within group prediction accuracy at 2<em>N<sub>t</sub></em>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-16.gif"/></noscript></span></span>the across group prediction accuracy when the training set size in the other group is 2<em>N<sub>t</sub></em> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/197/embed/mml-math-17.gif"/></noscript></span></span> the prediction accuracy from the combined training sets, when each group has a training set size of <em>N<sub>t</sub></em>). From our results for <em>N<sub>t</sub></em> = 25 at least, however, the alternating scheme would be inferior for both groups. For the alternating scheme to succeed, the across group prediction accuracy needs to be greater than observed by us. As is the case for the gain from combined training sets, the across group prediction accuracy will likely be the greater the lower the genetic distance between groups is.</p><p id="p-41">In conclusion, our results encourage the application of genomic prediction in a NCLB resistance breeding program. Furthermore, combining maize heterotic groups into a single training set is a promising approach for increasing the prediction accuracy of NCLB resistance.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">We are indebted to the staff of Limagrain Europe at the experimental stations in Bingen and Pocking for conducting the field experiments underlying this study. This research was funded by the German Federal Ministry of Education and Research (BMBF) within the AgroClustEr Synbreed—Synergistic plant and animal breeding (FKZ: 0315528d).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004630/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004630/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-43"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-10-08"><span class="received-label">Received </span>October 8, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-11-29"><span class="accepted-label">Accepted </span>November 29, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Technow <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.2" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats</span>. <abbr class="cit-jnl-abbrev">Plant Gen</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.3" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.4" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals</span>. <abbr class="cit-jnl-abbrev">J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHum.%2BGenet.%26rft.volume%253D84%26rft.spage%253D210%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.5" data-doi="10.1094/PD-79-0581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Inheritance of latent period length in maize infected with <em>Exserohilum turcicum</em></span>. <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D79%26rft.spage%253D581%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPD-79-0581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PD-79-0581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.6" data-doi="10.1098/rstb.2007.2170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B. C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackill</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Marker-assisted selection: an approach for precision plant breeding in the twenty-first century</span>. <abbr class="cit-jnl-abbrev">Philos T Roy Soc B</abbr> <span class="cit-vol">363</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCollard%26rft.auinit1%253DB.%2BC.Y%26rft.volume%253D363%26rft.issue%253D1491%26rft.spage%253D557%26rft.epage%253D572%26rft.atitle%253DMarker-assisted%2Bselection%253A%2Ban%2Bapproach%2Bfor%2Bprecision%2Bplant%2Bbreeding%2Bin%2Bthe%2Btwenty-first%2Bcentury%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2007.2170%26rft_id%253Dinfo%253Apmid%252F17715053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNjMvMTQ5MS81NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.7" data-doi="10.1071/AN10096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henshall</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gredler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of estimated genomic breeding values for wool and meat traits in a multi-breed sheep population</span>. <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D50%26rft.spage%253D1004%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN10096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN10096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.8" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.9" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.10" data-doi="10.1007/BF00441429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dingerdissen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheckert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welz</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Interval mapping of genes for quantitative resistance of maize to Setosphaeria turcica, cause of northern leaf blight, in a tropical environment</span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D2%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00441429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00441429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UX42800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.11" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.12" data-doi="10.3168/jds.2011-5019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goswami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4114</span>–<span class="cit-lpage">4129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DErbe%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4114%26rft.epage%253D4129%26rft.atitle%253DImproving%2Baccuracy%2Bof%2Bgenomic%2Bpredictions%2Bwithin%2Band%2Bbetween%2Bdairy%2Bcattle%2Bbreeds%2Bwith%2Bimputed%2Bhigh-density%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5019%26rft_id%253Dinfo%253Apmid%252F22720968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720968&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305363100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.197.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Correlated response to selection</span>, <span class="cit-comment">pp</span>. <span class="cit-fpage">320</span>–<span class="cit-lpage">322</span> <span class="cit-comment">in</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-edition">Ed. 4.</span> <span class="cit-publ-name">Longmans Green</span>, <span class="cit-publ-loc">Harlow, Essex, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.14" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Effective population size and genetic variability in the BS11 maize population</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGuzman%26rft.auinit1%253DP.%2BS.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D338%26rft.epage%253D346%26rft.atitle%253DEffective%2BPopulation%2BSize%2Band%2BGenetic%2BVariability%2Bin%2Bthe%2BBS11%2BMaize%2BPopulation%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000089410300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.16" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MCMC Methods for multi-response generalized linear mixed models: The MCMCglmm R Package</span>. <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.17" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.18" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Invited review: genomic selection in dairy cattle: progress and challenges</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.19" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.20" data-doi="10.1534/genetics.112.139014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kärkkäinen</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Back to basics for bayesian model building in genomic selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139014%26rft_id%253Dinfo%253Apmid%252F22554888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.21" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D180%26rft.spage%253D611%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.088575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18757934&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259758500049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.22" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for Fusarium head blight resistance in six-sow barley</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.23" data-doi="10.2135/cropsci2010.11.0664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miedaner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ufermann</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Colocalization of QTL for gibberella ear rot resistance and low mycotoxin contamination in early european maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1935</span>–<span class="cit-lpage">1945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D1935%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.11.0664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.11.0664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messmer</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boppenmaier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunklaus-Jung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Relationship among early European maize inbreds: I. Genetic diversity among Flint and Dent lines revealed by RFLPs</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D32%26rft.spage%253D1301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.26" data-doi="10.1023/A:1024068626366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengio</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference for the generalization error</span>. <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D52%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1024068626366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1024068626366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.27" data-doi="10.1094/PD-71-0940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pederson</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Disease development and yield losses associated with northern corn leaf blight on corn</span>. <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">940</span>–<span class="cit-lpage">943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D71%26rft.spage%253D940%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPD-71-0940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PD-71-0940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.28" data-doi="10.1073/pnas.1010894108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">6893</span>–<span class="cit-lpage">6898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010894108%26rft_id%253Dinfo%253Apmid%252F21482771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzY4OTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-other" id="cit-3.2.197.29"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-comment">2011 <em>R: A Language and Environment for Statistical Computing</em>. Available at: <a href="http://www.r-project.org/">http://www.r-project.org/</a>. Accessed: December 11, 2012</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.30" data-doi="10.1094/PD-65-325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raymundo</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooker</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Measuring the relationship between northern corn leaf blight and yield losses</span>. <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D65%26rft.spage%253D325%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPD-65-325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PD-65-325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebourg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chastanet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouesnard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welcker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubreuil</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Maize introduction into Europe: the history reviewed in the light of molecular data</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRebourg%26rft.auinit1%253DC.%26rft.volume%253D106%26rft.issue%253D5%26rft.spage%253D895%26rft.epage%253D903%26rft.atitle%253DMaize%2Bintroduction%2Binto%2BEurope%253A%2Bthe%2Bhistory%2Breviewed%2Bin%2Bthe%2Blight%2Bof%2Bmolecular%2Bdata.%26rft_id%253Dinfo%253Apmid%252F12647065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12647065&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182200900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.32" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.33" data-doi="10.3835/plantgenome2012.02.0001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown-Guedira</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genomic prediction methods for fusarium head blight resistance in wheat</span>. <abbr class="cit-jnl-abbrev">Plant Gen</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen%26rft.volume%253D5%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.02.0001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.02.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.34" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.35" data-doi="10.1186/1471-2229-12-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Inghelandt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinant</span>  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping of flowering time and northern corn leaf blight (<em>Setosphaeria turcica</em>) resistance in a vast commercial maize germplasm set</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BInghelandt%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bof%2Bflowering%2Btime%2Band%2Bnorthern%2Bcorn%2Bleaf%2Bblight%2B%2528Setosphaeria%2Bturcica%2529%2Bresistance%2Bin%2Ba%2Bvast%2Bcommercial%2Bmaize%2Bgermplasm%2Bset.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-56%26rft_id%253Dinfo%253Apmid%252F22545925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22545925&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.36" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thallman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keele</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snelling</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed beef cattle populations derived from deregressed breeding values and phenotypes</span>. <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-comment">10.2527/jas.2011–4586</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.38" data-doi="10.1094/PHYTO-96-0120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genetic architecture of disease resistance in maize: a synthesis of published studies</span>. <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DWisser%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D96%26rft.issue%253D2%26rft.spage%253D120%26rft.epage%253D129%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Bdisease%2Bresistance%2Bin%2Bmaize%253A%2Ba%2Bsynthesis%2Bof%2Bpublished%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-96-0120%26rft_id%253Dinfo%253Apmid%252F18943914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-96-0120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18943914&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234911500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.197.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D124%26rft.spage%253D769%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F197.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/2/197.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/181" title="Redundant Canonical and Noncanonical Caenorhabditis elegans p21-Activated Kinase Signaling Governs Distal Tip Cell Migrations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/205" title="Fine-Scale Mapping of the Nasonia Genome to Chromosomes Using a High-Density Genotyping Microarray" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/2"><span>Volume 3 Issue 2, February 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/2.cover-source.jpg?itok=h3kYZ-FP" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/197" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nzKrIu1aP5PJQ77Cydz9ND5MpcUWR9JA6hX3yo3lEDY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/2/197" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/2/197" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/2/197" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459419" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S3oTuJJmJf9zYXZ59wPYCDdiPYSwo6xo2A5e0Cn-u5s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/197.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/2/197" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-XgojaEPKaBQ2Dzqt65yskfiqaiEnNz1oKOeFKPITFoA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/2/197" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/197.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459419" id="node459419--2" data-pisa="ggg;3/2/197" data-pisa-master="ggg;3/2/197" data-apath="/ggg/3/2/197.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frank Technow</span>, <span class="highwire-citation-author" data-delta="1">Anna Bürger</span> and <span class="highwire-citation-author" data-delta="2">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">197-203;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004630 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459419--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Frank.Technow@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">Frank.Technow@uni-hohenheim.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna Bürger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BB%C3%BCrger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%BCrger%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BB%25C3%25BCrger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/2/197" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/2/197&amp;title=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/2/197&amp;title=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/2/197&amp;title=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/2/197&amp;text=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/2/197&amp;title=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/2/197&amp;t=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/2/197&amp;title=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/2/197&amp;title=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459419/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459419" id="node459419--3" data-pisa="ggg;3/2/197" data-pisa-master="ggg;3/2/197" data-apath="/ggg/3/2/197.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frank Technow</span>, <span class="highwire-citation-author" data-delta="1">Anna Bürger</span> and <span class="highwire-citation-author" data-delta="2">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">197-203;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004630 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459419--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Frank.Technow@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">Frank.Technow@uni-hohenheim.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna Bürger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BB%C3%BCrger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C3%BCrger%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BB%25C3%25BCrger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Plant Breeding, Seed Sciences and Population Genetics, University of Hohenheim, Stuttgart 70599, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=197&amp;rft.epage=203&amp;rft.atitle=Genomic%20Prediction%20of%20Northern%20Corn%20Leaf%20Blight%20Resistance%20in%20Maize%20with%20Combined%20or%20Separated%20Training%20Sets%20for%20Heterotic%20Groups&amp;rft.volume=3&amp;rft.issue=2&amp;rft.date=2013-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Technow%2C+Frank&amp;rft.au=B%C3%BCrger%2C+Anna&amp;rft.au=Melchinger%2C+Albrecht+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459419/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459419/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459419/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459419/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459419/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459419/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459419/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459419/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459419/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459419/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459419/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459419/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/2/197" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/2/197.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/2/197.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/2/197.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/2/197.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
