<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/2/263" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/2/263.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/2/263.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/2/263.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/2/263/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/2/263.atom" />
<meta name="HW.pisa" content="ggg;3/2/263" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations" />
<meta name="DC.Identifier" content="10.1534/g3.112.005066" />
<meta name="DC.Date" content="2013-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Guo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Most of previous empirical studies with genome-wide prediction were focused on within-environment prediction based on a single-environment (SE) model. In this study, we evaluated accuracy improvements of across-environment prediction by using genetic and residual covariance across correlated environments. Predictions with a multienvironment (ME) model were evaluated for two corn polygenic leaf structure traits, leaf length and leaf width, based on within-population (WP) and across-population (AP) experiments using a large maize nested association mapping data set consisting of 25 populations of recombinant inbred-lines. To make our study more applicable to plant breeding, two cross-validation schemes were used by evaluating accuracies of (CV1) predicting unobserved phenotypes of untested lines and (CV2) predicting unobserved phenotypes of lines that have been evaluated in some environments but not others. We concluded that (1) genome-wide prediction provided greater prediction accuracies than traditional quantitative trait loci-based prediction in both WP and AP and provided more advantages over quantitative trait loci -based prediction for WP than for AP. (2) Prediction accuracy with ME was significantly greater than that attained by SE in CV1 and CV2, and gains with ME over SE were greater in CV2 than in CV1. These gains were also greater in WP than in AP in both CV1 and CV2. (3) Gains with ME over SE attributed to genetic correlation between environments, with little effect from residual correlation. Impacts of marker density on predictions also were investigated in this study." />
<meta name="DC.Contributor" content="Zhigang Guo" />
<meta name="DC.Contributor" content="Dominic M. Tucker" />
<meta name="DC.Contributor" content="Daolong Wang" />
<meta name="DC.Contributor" content="Christopher J. Basten" />
<meta name="DC.Contributor" content="Elhan Ersoz" />
<meta name="DC.Contributor" content="William H. Briggs" />
<meta name="DC.Contributor" content="Jianwei Lu" />
<meta name="DC.Contributor" content="Min Li" />
<meta name="DC.Contributor" content="Gilles Gay" />
<meta name="article:published_time" content="2013-02-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most of previous empirical studies with genome-wide prediction were focused on within-environment prediction based on a single-environment (SE) model. In this study, we evaluated accuracy improvements of across-environment prediction by using genetic and residual covariance across correlated environments. Predictions with a multienvironment (ME) model were evaluated for two corn polygenic leaf structure traits, leaf length and leaf width, based on within-population (WP) and across-population (AP) experiments using a large maize nested association mapping data set consisting of 25 populations of recombinant inbred-lines. To make our study more applicable to plant breeding, two cross-validation schemes were used by evaluating accuracies of (CV1) predicting unobserved phenotypes of untested lines and (CV2) predicting unobserved phenotypes of lines that have been evaluated in some environments but not others. We concluded that (1) genome-wide prediction provided greater prediction accuracies than traditional quantitative trait loci-based prediction in both WP and AP and provided more advantages over quantitative trait loci -based prediction for WP than for AP. (2) Prediction accuracy with ME was significantly greater than that attained by SE in CV1 and CV2, and gains with ME over SE were greater in CV2 than in CV1. These gains were also greater in WP than in AP in both CV1 and CV2. (3) Gains with ME over SE attributed to genetic correlation between environments, with little effect from residual correlation. Impacts of marker density on predictions also were investigated in this study.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/02/01" />
<meta name="citation_mjid" content="ggg;3/2/263" />
<meta name="citation_id" content="3/2/263" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/2/263" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/2/263.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/2/263.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/2/263.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005066" />
<meta name="citation_pmid" content="23390602" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="263" />
<meta name="citation_lastpage" content="272" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhigang Guo" />
<meta name="citation_author_institution" content="Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709" />
<meta name="citation_author_email" content="zhigang.guo@syngenta.com" />
<meta name="citation_author" content="Dominic M. Tucker" />
<meta name="citation_author_institution" content="Syngenta, Inc., Clinton, Illinois 61727" />
<meta name="citation_author" content="Daolong Wang" />
<meta name="citation_author_institution" content="Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709" />
<meta name="citation_author" content="Christopher J. Basten" />
<meta name="citation_author_institution" content="Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709" />
<meta name="citation_author" content="Elhan Ersoz" />
<meta name="citation_author_institution" content="Syngenta, Inc., Stanton, Minnesota 55018" />
<meta name="citation_author" content="William H. Briggs" />
<meta name="citation_author_institution" content="Syngenta Seeds B.V., 1601 BK Enkhuizen, The Netherlands" />
<meta name="citation_author" content="Jianwei Lu" />
<meta name="citation_author_institution" content="Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709" />
<meta name="citation_author" content="Min Li" />
<meta name="citation_author_institution" content="Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709" />
<meta name="citation_author" content="Gilles Gay" />
<meta name="citation_author_institution" content="Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709" />
<meta name="citation_reference" content="citation_title=Molecular Dissection of Complex Traits;citation_title=QTL analysis: power, precision and accuracy;citation_author=W. D. Beavis;citation_pages=145-162;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Bernardo;citation_title=Molecular markers and selection for complex traits in plants: learning from the last 20 years;citation_pages=1649-1664;citation_volume=48;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide Selection for Rapid Introgression of Exotic Germplasm in Maize;citation_pages=419-425;citation_volume=49;citation_year=2009;citation_issue=2;citation_doi=10.2135/cropsci2008.08.0452" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Blanc;citation_author=A. Charcosset;citation_author=B. Mangin;citation_author=A. Gallais;citation_author=L. Moreau;citation_title=Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.;citation_pages=206-224;citation_volume=113;citation_year=2006;citation_issue=2;citation_pmid=16791688;citation_doi=10.1007/s00122-006-0287-1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. L. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Predictions of genetic values of quantitative traits in plant breeding using pedigree and molecular markers;citation_pages=713-724;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree;citation_pages=375-385;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R.W. Doerge;citation_author=G. A. Churchill;citation_title=Permutation tests for multiple loci affecting a quantitative character;citation_pages=285-294;citation_volume=142;citation_year=1996;citation_pmid=8770605" />
<meta name="citation_reference" content="citation_title=Analysis of Molecular Marker Data;citation_title=RFLPs for rapid recurrent selection;citation_author=M. Edwards;citation_author=L. Johnson;citation_pages=33-40;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=Z. Guo;citation_author=J. C. Nelson;citation_title=Multiple-trait quantitative trait locus mapping with incomplete phenotypic data.;citation_pages=82-82;citation_volume=9;citation_year=2008;citation_pmid=19061502" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Guo;citation_author=D. Tucker;citation_author=J. Lu;citation_author=V. Kishore;citation_author=G. Gay;citation_title=Evaluation of genome-wide selection efficiency in maize nested association mapping populations.;citation_pages=261-275;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21938474;citation_doi=10.1007/s00122-011-1702-9" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. Haley;citation_author=S. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Hayashi;citation_author=H. Iwata;citation_title=The B chromosomes of the African cichlid fish Haplochromis obliquidens harbour 18S rRNA gene copies.;citation_pages=1-1;citation_volume=11;citation_year=2010;citation_pmid=20051104" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. Hayes;citation_author=P. Bowman;citation_author=A. Chamberlain;citation_author=M. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Hayes;citation_author=P. Visscher;citation_author=M. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=J. L. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy for grain quality traits in biparental wheat populations;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Brief Funct. Genomics;citation_author=J. L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. C. Jansen;citation_title=Interval mapping of multiple quantitative trait loci;citation_pages=205-211;citation_volume=135;citation_year=1993;citation_pmid=8224820" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Jiang;citation_author=Z. B. Zeng;citation_title=Multiple trait analysis of genetic mapping for quantitative trait loci;citation_pages=1111-1127;citation_volume=140;citation_year=1995;citation_pmid=7672582" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Lande;citation_author=R. Thompson;citation_title=Efficiency of marker-assisted selection in the improvement of quantitative traits;citation_pages=743-756;citation_volume=124;citation_year=1990;citation_pmid=1968875" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. H. Lee;citation_author=J. H. J. van der Werf;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Predicting unobserved phenotypes for complex traits from whole-genome SNP data.;citation_pages=e1000231-e1000231;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18949033;citation_doi=10.1371/journal.pgen.1000231" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granie;citation_author=E. Manfredi;citation_author=J. M. Elsen;citation_title=Performance of genomic selection in mice;citation_pages=611-618;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. H. Li;citation_author=G. Y. Ye;citation_author=J. K. Wang;citation_title=A modified algorithm for the improvement of composite interval mapping;citation_pages=361-374;citation_volume=175;citation_year=2007;citation_pmid=17110476" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetics;citation_author=T. Luan;citation_author=J. A. Woolliams;citation_author=S. Lien;citation_author=M. Kent;citation_author=M. Svendsen;citation_title=The accuracy of Genomic Selection in Norwegian red cattle assessed by cross-validation.;citation_pages=1119-1126;citation_volume=183;citation_year=2009;citation_issue=3;citation_pmid=19704013;citation_doi=10.1534/genetics.109.107391" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Makowsky;citation_author=N. M. Pajewski;citation_author=Y. C. Klimentidis;citation_author=A. I. Vazquez;citation_author=C. W. Duarte;citation_title=Beyond missing heritability: prediction of complex traits.;citation_pages=e1002051-e1002051;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552331;citation_doi=10.1371/journal.pgen.1002051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. H. Meuwissen;citation_author=T. R. Solberg;citation_author=R. Shepherd;citation_author=J. A. Woolliams;citation_title=A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.;citation_pages=2-2;citation_volume=41;citation_year=2009;citation_pmid=19284681;citation_doi=10.1186/1297-9686-41-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. Moser;citation_author=B. Tier;citation_author=R. E. Crump;citation_author=M. S. Khatkar;citation_author=H. W. Raadsma;citation_title=A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.;citation_pages=56-56;citation_volume=41;citation_year=2009;citation_pmid=20043835;citation_doi=10.1186/1297-9686-41-56" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=F. D. N. Mujibi;citation_author=J. D. Nkumah;citation_author=O. N. Durunna;citation_author=P. Stothard;citation_author=J. Mah;citation_title=Accuracy of genomic breeding values for residual feed intake in crossbred beef cattle;citation_pages=3353-3361;citation_volume=89;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. Nakaya;citation_author=S. N. Isobe;citation_title=Will genomic selection be a practical method for plant breeding?;citation_volume=110;citation_year=2012;citation_pmid=22645117;citation_doi=10.1093/aob/mcs109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=U. Ober;citation_author=M. Erbe;citation_author=N. Y. Long;citation_author=E. Porcu;citation_author=M. Schlather;citation_title=Predicting genetic values: a kernel-based best linear unbiased prediction with genomic data;citation_pages=695-708;citation_volume=188;citation_year=2011;citation_pmid=21515573;citation_doi=10.1534/genetics.111.128694" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. P. Piepho;citation_title=Ridge Regression and Extensions for Genomewide Selection in Maize;citation_pages=1165-1176;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.10.0595" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Piyasatian;citation_author=R. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection for marker-assisted improvement in line crosses.;citation_pages=665-674;citation_volume=115;citation_year=2007;citation_issue=5;citation_pmid=17673971;citation_doi=10.1007/s00122-007-0597-y" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=M. M. Rolf;citation_author=J. F. Taylor;citation_author=R. D. Schnabel;citation_author=S. Mckay;citation_author=M. McClure;citation_title=Impact of reduced marker set estimation of genomic relationship matrices on genomic selection for feed efficiency in Angus cattle.;citation_pages=24-24;citation_volume=11;citation_year=2010;citation_pmid=20403185" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_author=C. C. Schön;citation_title=Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples;citation_pages=1839-1849;citation_volume=154;citation_year=2000;citation_pmid=10866652" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Vattikuti;citation_author=J. Guo;citation_author=C. C. Chow;citation_title=Heritability and genetic correlations explained by common SNPs for metabolic syndrome traits.;citation_pages=e1002637-e1002637;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22479213;citation_doi=10.1371/journal.pgen.1002637" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.;citation_pages=5942-5949;citation_volume=93;citation_year=2010;citation_issue=12;citation_pmid=21094768;citation_doi=10.3168/jds.2010-3335" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. L. Wang;citation_author=J. Zhu;citation_author=Z. K. Li;citation_author=A. H. Paterson;citation_title=Mapping QTLs with epistatic effects and QTL × environment interactions by mixed linear model approaches;citation_pages=1255-1264;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051331" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. Wolc;citation_author=C. Stricker;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. E. Fulton;citation_title=Breeding value prediction for production traits in layer chickens using pedigree or genomic relationships in a reduced animal model.;citation_pages=5-5;citation_volume=43;citation_year=2011;citation_pmid=21255418;citation_doi=10.1186/1297-9686-43-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Estimating polygenic effects using markers of the entire genome;citation_pages=789-801;citation_volume=163;citation_year=2003;citation_pmid=12618414" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=N. Yi;citation_author=S. Xu;citation_title=Linkage analysis of quantitative trait loci in multiple line crosses;citation_pages=217-230;citation_volume=114;citation_year=2002;citation_pmid=12206360;citation_doi=10.1023/A:1016296225065" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=Z. B. Zeng;citation_title=Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.;citation_pages=10972-10976;citation_volume=90;citation_year=1993;citation_issue=23;citation_pmid=8248199;citation_doi=10.1073/pnas.90.23.10972" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. B. Zeng;citation_title=Precision mapping of quantitative trait loci;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. S. Zhao;citation_author=M. Gowda;citation_author=W. X. Liu;citation_author=T. Würschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations.;citation_pages=769-776;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22075809;citation_doi=10.1007/s00122-011-1745-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Q. Zhong;citation_author=J. C. M. Dekkers;citation_author=R. L. Fernando;citation_author=J. L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study;citation_pages=355-364;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="twitter:description" content="Most of previous empirical studies with genome-wide prediction were focused on within-environment prediction based on a single-environment (SE) model. In this study, we evaluated accuracy improvements of across-environment prediction by using genetic and residual covariance across correlated environments. Predictions with a multienvironment (ME) model were evaluated for two corn polygenic leaf structure traits, leaf length and leaf width, based on within-population (WP) and across-population (AP) experiments using a large maize nested association mapping data set consisting of 25 populations of recombinant inbred-lines. To make our study more applicable to plant breeding, two cross-validation schemes were used by evaluating accuracies of (CV1) predicting unobserved phenotypes of untested lines and (CV2) predicting unobserved phenotypes of lines that have been evaluated in some environments but not others. We concluded that (1) genome-wide prediction provided greater prediction accuracies than traditional quantitative trait loci-based prediction in both WP and AP and provided more advantages over quantitative trait loci -based prediction for WP than for AP. (2) Prediction accuracy with ME was significantly greater than that attained by SE in CV1 and CV2, and gains with ME over SE were greater in CV2 than in CV1. These gains were also greater in WP than in AP in both CV1 and CV2. (3) Gains with ME over SE attributed to genetic correlation between environments, with little effect from residual correlation. Impacts of marker density on predictions also were investigated in this study." />
<meta name="og-title" property="og:title" content="Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/2/263" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Most of previous empirical studies with genome-wide prediction were focused on within-environment prediction based on a single-environment (SE) model. In this study, we evaluated accuracy improvements of across-environment prediction by using genetic and residual covariance across correlated environments. Predictions with a multienvironment (ME) model were evaluated for two corn polygenic leaf structure traits, leaf length and leaf width, based on within-population (WP) and across-population (AP) experiments using a large maize nested association mapping data set consisting of 25 populations of recombinant inbred-lines. To make our study more applicable to plant breeding, two cross-validation schemes were used by evaluating accuracies of (CV1) predicting unobserved phenotypes of untested lines and (CV2) predicting unobserved phenotypes of lines that have been evaluated in some environments but not others. We concluded that (1) genome-wide prediction provided greater prediction accuracies than traditional quantitative trait loci-based prediction in both WP and AP and provided more advantages over quantitative trait loci -based prediction for WP than for AP. (2) Prediction accuracy with ME was significantly greater than that attained by SE in CV1 and CV2, and gains with ME over SE were greater in CV2 than in CV1. These gains were also greater in WP than in AP in both CV1 and CV2. (3) Gains with ME over SE attributed to genetic correlation between environments, with little effect from residual correlation. Impacts of marker density on predictions also were investigated in this study." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
    <title>Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jryk0K1xyAMb1wTh8fy0UKw9T5CqYVcC7Qo3WWaSaXo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459399","apath":"\/ggg\/3\/2\/263.atom","pisa":"ggg;3\/2\/263","ac":{"\/ggg\/3\/2\/263.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/2\/263.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/2\/263"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/2\/263"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/2\/263.supplemental":true,"\/content\/3\/2\/263":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n5dYKwFI+EFbVPjNccFoGQwLv3MWqV4pkvWFWDQm5nPazkG3bzmGPVBemO8Qc3p3OD\\\\\\\/AU18EZLkT8spTfuVGVgPYP+0C9E91FfuF+nBVw6Waruut6AdVAziiWr9q\\\\\\\/Mb5Xz0H2Ejwzzg9GWeAEd\\\\\\\/68ATlrGS0oWgpi1AWy00lHstZLcVem9cIcVAlEE4Xv0pqiSTUbWCB14nTSpDR67taosb7xGwI8rNUxdWYpyaMokWjBeQI3RZpkQSDosD8ryzu0AAvxnMRiZ7rLdkyApkqPiDtmkh00jyAsoIV2FywWwi9MtryGy2GDn+r2gkPEm9\\\\\\\/zsQOnLHEcPhoSKo8\\\\\\\/ycZZPwknAjGFf94Y8kbQGRbybf8c7A6VwTBcdxVBXxmgmV4UHuRxCWtGfgvbi7fGkR0KAiow\\\\\\\/ZZN\\\\\\\/JywuCE\\\\\\\/Gwhb8h9K7Lro4leMGC\\\\\\\/Wou01IMmfcr7jyMUKbWTzNFWfmLToHOUXwCvskK\\\\\\\/h8UdeA7X7UALg6bvRM5jmzV\\\\\\\/muC55wI9SnLfxOg5kyIgMGfKR6PV4OgyyDedrOuq3Y1la+QE2sWusDqVZ1gv3SCQ9tE+PDJTZE\\\\\\\/iPJPu9kNZxM6nNUwEgqIS2bkjA2GjEzPYAO8bn0w8hHlOXj7a0qRCdLwn2\\\\\\\/0DDEioa0TuwS3\\\\\\\/\\\\\\\/loHl\\\\\\\/xSEc3TYxiE325ugIsyC5+0xFE964XGNLdvAmUY+ECJB0jht1RrTekQ83Wawg4YBLTU3IFuF+7MLIa696xPhdcB1L81QiPfbkdTSHfDA1OFI5ibpVJ2Lh647iQNtCBwy+e7IrxHx2oH7Dn+VEAOSaQ9y8JuzkAEbPHat5G5CL4pVv5EQtJj7H9yKCOIyW0A7DebMnafqpOO0B\\\\\\\/ORXoNGJ\\\\\\\/88r2\\\\\\\/JRcRPNBsjYJZ3S5LnyIK+E8Wgvg\\\\\\\/Zjbb3CqTMI7Z7aZ5oHjTM2jdQMUTJ+XOkewfiGq18IPjqxp4Kboreg6+l6\\\\\\\/O+QJIZ7Xd+dYfKBosmToZMIf8vHC8pC8h+uGWKCSRXFkWaBvP4bTu4SCl\\\\\\\/2dSctYTTcLzNTLVzWMmDHqGAYHSxOD26PYKekCofLJF4nwlq36qBVOc2PJWVh3Ow1Uz\\\\\\\/NLBn40XJPCgGi7VJk3LB78DENn7HHEOzcxv2c6e0eIvD4+d0v8VKc1Q4+ufIH+DMedCckxc9d6Q8Adeu8aFOgOoIZDWR+riM1aeIg3JSEWcEqKdtaEyAVelaCtZFr7vfJxdg4IzSCtUSuTlnadY8VvG4CjpswxL4oetAKwZxDeawg3GXzNfykn50hFIKtDA\\\u0022,\\\u0022iv\\\u0022:\\\u0022UFABLknLL0nOgQp3osatMA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b93371de17617e41af808cc023b5a03cea95aee5336222e7638316757bdfa783\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hxPgKbTxBkznQ5JIkeQ9zJnkX1Axwnz1JIYhwdQguV1usCG8uP57U9JmYgghJWV4GEpIC1Q3b+1f4HhRb8TggdTw+bH1Xs9Jox2HmCYaUf8vB+KNvyccduKae4\\\\\\\/hAfTpaQUgar+11fHl5FpfCuJovTcymCsppmX7mhI0NBXijtLdseEnmgFXoqGG85LN24YSl+Mo+2KxhUm2MEJApe0QKs83FYirfJ5yPNDy4zy+VYNgEs8Cm0XrlUkVO4hEjDTskaRSCYLsp+Ntfalpp4xOzqE9AGilQ8CndSdEciqSVTG8MwRvucNvh5rabJ3DPwEBT4q3v7RAaKepI2PAidShMdacg5xhemsscun0DZVDd74LXGVEc8QElr\\\\\\\/c5\\\\\\\/qM5MskRAR24G01au2bkWe7jadGOvnPC\\\\\\\/sSrbbxxNZDRWV6U+px9szZ3+jkKGVQywL3Ip9Wq8XLrVNFlt2iKQU0v3FGPX\\\\\\\/FO\\\\\\\/jgG5PSO312TY4xH5oIFBlKwQEchIYG44ham5n4gIN3r8XwJ+gbEHNRhiJC08yjhYHW+0Aujjwct92BLKcS0KVmPRPVg9yEO+zsLAlxiuPxY46lagrD1udxKoEtHWWF0MccFyPaH0yQXJJkm510Hzp+ZhlsbNuVLEH5nAlcuGuVLIBW4FwAFErguQ8QfuHlwKwGliqN04ycyq+5g1YzBNgJlUmCWu6pp5qu4VtNrsQtpkYsQ3PeeLYYyiYoMWjbqK4v+3LFxCqng9lkOZpK81Alfxif9dLua36YcUMjOD3R1mChhPZUeVLHfddN8JHAzMUWba2Ivvzw\\\\\\\/8SYGgIDVAZLdlDYCXLwcWstK4KTjAnrSE4iDkgHZJ4ZIWCZ+U3baKNDotcWgXiFpI6kksfm25bI9SiRferqRLtLrxyQyoAiOws7XGCkhNP6BMMQM+W4NNu19AdCSEuVBif+NDZdIg\\\\\\\/tgNIr0AXUfooMavVhdd\\\\\\\/dwMje8kEhML\\\\\\\/yoYJh5xRFqLLhvf2zm7nHPWgZlbNeAvO9\\\\\\\/sx3I7RuFWpcnyNuWJuPOOCx5+DxNQmqKgZ3FFHfHVqESnmvk5piPsTRhCdthmS5nU0esZ88qK2QCD1JmOw\\\\\\\/AYpw5YERYDGJhXeNbMMYlMZ9bQE9JV6fOs5xHNAMkAqGWxjdhMLKyWfcGJZ0Ul7n\\\\\\\/Tc\\\\\\\/yFU+6usmsvTY0\\\\\\\/McfEM\\\\\\\/BDKdbd84n8vjz\\\\\\\/CcuS+U8PPx4QhB0NEmxgew2tuhTa9rk+7Sz3PMeYR2nxwdOR\\\\\\\/XYo4zC0DWWlLmZlyfcNBQJLyLq+IWgkGcKf4+rxDCv5xaonKOdO3iHV+\\\\\\\/C0KeEadnIhT4uNT05eBLwIc\\\\\\\/60AC1ZpCHuhHB2W2wf0IbhpFac4Iy1UpTtupOJmrzYrCFOkXBm4qA62M6R9e\\\\\\\/wbjiWATQ5daQHrk6\\\\\\\/Cr7rntO6jLTK8x\\\\\\\/qSh0P2iVHRH3Leb5pltRNNjYVq7g\\\\\\\/3pt42saDXwCo6gcXznK4M0gdSev1lIA6Cm0KvHdflr5HD50HvSz3bLleQjNWI1ZHqOi9fVWgqn5FYMneVoz6gFcZj68\\\\\\\/3k01OY6r3EgJ4PKoJRW1Id55TM4UmD2ypEppggdeO0XUMwpfmaExOFwfc4LOHH60RwNcFIGZi0ihN7AeTV7Jyn0dUSYzOg4MQIzeD4nMYTXJEoY0WolENooui5dMrMWBI2z4cOmgjxrI32693yH78YAOlKzLjRCF2\\\\\\\/ggWSswsr6hcCRuc2jUdoJecic9lEg3TXfin\\\\\\\/ODhvuIclZzKsTGzDhNk41p7EWxWl2q53Vat4xcnR7yFLcFXjQ+dmGxBS1TKQaCw5PWDOkUw9azu+JPvXcqhBo5QGS5A6k6v2EKE8TpI7NyhRCvOk2Gv57uYOXLbopMEn1yUP0Aw9Hznfxnj4a7jkwAUmVozD9dWxQZZNFFr1EJZRZadDxSUYKalWcIRDpwzTFQUAkwOY\\\\\\\/9CP29eYlWmpxAn3OXPpXd1a8TQM7cJoKIRtifHS5rNOaEWZsc6z1BSD9A+7rh4miAj46wcZQRTzdrnSfx7+3bMSvG7QYlnTkzXJYOwRAd5S55vDGGBCqkqR3yif9Uycz3s1\\\\\\\/tjrDTCeFV9nBAGUJKtq+3BTy25Jp8wDKgOd6yB4XY3KaSoAyW8lUEqG7gNqc6Y87QHmXAiitimd\\\\\\\/eif0H2QtBIDioV\\\\\\\/vUL9uitU2UJASNz3XsAq2r7Xo0zFIgTdRppddalHhxVCKKu+t\\\\\\\/19OWUULfAi3oznB1TlzIaR5vMG1nYtWK+4MjkplLmh14f5svxebPIPn2HIfqCkiCENCoO3VI9M5s9ULniHXLasOMl8b8EpUmV8SGcRY3UCT8UYWwF0Erz0TsOM8eByG7yoOaeLLmGy3Tje2JOwdLx4ESUU7aLPZJiZdg28W5p4ymMxnn1JCPzndni4ESctGh+shXiVUgj841ldTvWs4wJEzSiZmpSVcG2JZRNtQAhg62hB7voRHJ+VLuTIjUJjFOF8pveyGZ\\\\\\\/QH3ioiRytgfEcCyE73F5VhhB5xrBxq5oYda2kGrh9ACyVhe5HaVpZd5B\\\\\\\/Sk2v+Alf2W+23oVe94OJD0NuXyFjvtPh\\\\\\\/1RLkCqBS1cioBeHeEI9WA99h8vn9XaKokN04k6VnAYpAhmvRk0c9xdoXAJ0eQNBVGe7r+lQeUfCm7xAnxACBqvO6GjJp7APuN4L3cbB\\\\\\\/rRo1SMh8ZkLSFp7q29tAq34pVgFzhzg30ukqUI2bIARs1hdYYsBun6an9EyRqLjXZ2NkWjBYkm0Dwpgji2jWG8Krgzc3obJ7v2r+TxdqQm\\\\\\\/B5FRrRF\\\\\\\/Cj2OKRKAXwq6TjxYputxa5SoiyQGwQn\\\\\\\/jmQJW1Z4rz3p376hDWIVKjg\\\\\\\/RGjm9mrmFmLhZhaesmqaGVWyRJOZciValPNcWUgLlhCypoQ3BNdhtnRtkJgRF4951qcmbjK6phauggw+ZP5SRcoVFd2hLkWAEd\\\\\\\/dZcFZNhgs3YXt7iymlPXJUzRyKphMGPATS1MhAWLkchsU96ktcBKIKZxq4N7FBRxjViIkkAwo+F87uhwszJzyo+bgSsA9\\\\\\\/vBv5EXlFxNlMzRYS2o\\\\\\\/8DRIZI6S6fGwH5KOcDXHDFeghkTIFsmuAMAeW+VgUy\\\\\\\/jLznEg5tovrEGB6kveKy7D0R5aR8gNESeoGzr38S3mbfIB+V011CtWFpSbxwr3D8RdHNSyy03lRg6Ie0RT1mv6bhDn5Fyf3IjoTRgM5po5lMKZ6npRk+ZanE8p8Liadyo7w6KQKupeUzT1IH2gpQB3+k97e6itE+4xO3IO8HXBRpLQT335UDqVcwzTHgnpppXqU+tiT\\\\\\\/ngzYXLbjTsW4UkgytxdMsKApmg9EUG9bSw5Be8brn4e0vmuYAODVzoPg16C8E9iBteK7JlLyl3+3QheM1rQ1wwDH3cIa5S14KP4LvRfPCrNgHGf35oMzpqh0yW2fRuiHqPA80X\\\\\\\/\\\\\\\/8IV+sr5Efu40kZul39iY1HP4ZkW7CwL2kA6vQA2nUsAKU28ZGkbpEiGy4U9d66a2yDGG7n3FgUlkF2TVQg6YnB5GVzbgsYuFJaEDooYfQKRCXhZiCacL1gworENFm9e3KlOqZotFdv8emduC\\\\\\\/p+\\\\\\\/0kZzvA3EfZCUJGZwvv6ujWgUv2Ir85c8ju1tu0furzt1E87HxzQZ\\\\\\\/i0WsMUWYKwexg1xyzdi+XG+2do6yICObWwSKIdHx02MMnFvg6bcZboDSk2RiRwZPnZ3sraCtj0pgaeyiDQQdSOiY5xaS60YI\\\\\\\/BliROzrzAB8L9j9NXaLrT7OHPI0NsqDNSJ7PJxD9QAJz8MaCpVp5FwAl70NTUzwZdAFOb9mIhwBebwJQ9g+NIk6pGLHh6XysKisIi+xk2cQu4c1x4vq\\\\\\\/9lwI27POW+\\\\\\\/Ve0Xu3auJXDEHR69Ac4uasJnZJFhSdpP3\\\\\\\/fb58jCrTcZO2ciWCg94\\\\\\\/EfIlgV2XpGE8+h1Vk\\\\\\\/gHBOn1cWUqzIbUkUpjiUqesvqh7m8x0D1+E3w2Vq6cEFOlAqoHZ2HBN0m8M8JyIglDqk+hwP5oR8Nl7FjeEFCprbezSW3sWBz\\\\\\\/vOrLT6sic59Ov8Nc2D0Bx8aGZMVj\\\\\\\/XlOeUsLbToJ5X84nuf7cDNhppm6kQdmZDUHdQkTK1R+Fj2rT5q9qKIYn22pF5XRuRa1J4fkMDqm3oTmZjNtXWNuUG5yeCE8f3w2fhC8IfQlhMu1bFZWqloiSBZRcpH+Q6uWSn\\\\\\\/q9aAdIDO40YRvG6MpOxhGZom2rJNW4NOlHFKvRVJPXfao7hJB+f0bH4KU27oxlA5qjK5eLFgkEWoX\\\\\\\/HeyzMLai10oVOk6PVjaW85dyE4wOGtC3cypj6ucsAQZWp5JqOc1NmTdIa8Fb3FQcjDUtfsSze6qLZaBVdUAKMbVSIMllEFqgPaIeeZ5IvOhNJqExx9SwgfiCFYpu218yPBOodkFbM\\\\\\\/cj8HeXzvocKbBQnvd7f7QXN7Xls2oNnp4NpxidX52okdWLqjJ6yanTNP3KgAEN7WrXtoAnry9r9Mfaan\\\\\\\/ttv7dcZwk+Ubp\\\\\\\/wbsP8ICgvX14aZrCCl1jvdIFRF+ZRI4Wtj1yv\\\\\\\/KI7BG5qhKfJ16+oW8Iy39IQ8ojQ3zWlxHGDtc7bkL6cQekeUqJYrznTyCcXJmS83KvWO7wuxO\\\\\\\/AMi6\\\\\\\/0SsXnKZED1SBQfaCAohjRuDtte71FAqjg77AR++4XtIv1ElO5raviq2LFpcxpm8wEhlaWW7PICOlgtNNaet36odljPF0RhMK0Eten3evVTs18iObKA9WAqMvc3P5UOJcpZfrXQMd7rizc5fqSLDfQuU0VpLHXuUKv74ZkO2kLvBgbWQYPvX0aviW4Ph9dbvW2\\\\\\\/\\\\\\\/ZuEO17xDt3jqJf1plZBNYWga9Dfsuc0UAI2pSGmPTLbS9cBNp7cG94wb8eC5VQDLSb7PzKJNQheKpoCcjXjFXfHFsAfwwn38n2dKgriFIlLyYDV5JUCaNL2PtI1ONJnNb+cslV7zpcIo4bQmwz5tQ7BLexTp0m6RViIDTkI9HLAdkW55qx1+EEOAYTdadvbNvwt2BntBJ0KT4DF0riSQ8uOtflq3UKiXgZD38e3pYEyoLQGHx2Cz5Zt8iYA814Bf6Lypu91jlI6zJ4uY8ycgEWARiYMyiEJJketx7kFp2y6Jecz7T4Tsw1ygH1G9FR4tsHn+hbMl95UYGk1yTomAYiygFpbCjTrKXt00cV+ZznC9bZfckz1wKD0NTrnWzceEaBXbXM9a+8VVazkK6dLXGwO\\\\\\\/TRMqdNsCz0Hk4Sr4zj5V6HdH4GolGmEfRnmpg7+1Fx6M9OZFYZ\\\\\\\/93D+5v0rYmz\\\\\\\/1zF+rMarpGDVMMGXlGTgqdlO4naEEunVe5kGWgKt\\\\\\\/XKVJqSEQ4OFJ3fcbcgi\\\\\\\/cCscL2pqG+bKBTRyuEsxh6oUwVzbfZi6iDLMt6Lio4ahytTE5BF3kMTkgfcTQ\\\\\\\/\\\\\\\/lOxgobs4iCdIW2KQ1yl880BHNerOgi6LNJcAE6Gwp0NKhcWh7OE+3r\\\\\\\/vZwc3qOX8uDgVTrrtIRNiVB+JtnmB+lQbTak58o4HAUQZia\\\\\\\/Xsxb5beA3YZ9gNTdRQzpKyMtNyEg4Gl0Pg\\\\\\\/ld6aPmD3fcjrlD5EF1ldxn7WL9qAFXp2jc87VLR1unZ324dKbzULn028CX7+duiMWnTzw6XHxG7rwOHH3XpBt2ceQFPiEcEWnUVsidUlBrYAsnw9lYMOBiRgHnKEumYOyz6U7nEy4ItdHocIXlV+rtQJIIaehHKtAKYAk2GCK7U4nGF9I1yM\\\\\\\/9oqeWYN5rsZMBuKxQsrpcT9EFhVKoFNpfBQmMXOx0TQgRTbBx5\\\\\\\/rLGdZnrVHd+b+7JycaGBfjngIwFQFICGUeHNWi0sbVs\\\\\\\/dhkKdMQPAQZsKiQUZXYhZD52w52C3Ym3XLsXHl73+EuxiGtDFktVq2CJAosKmMCBhcHgt+MLBrAhi9nnQPwTq\\\\\\\/fOFzYD9MS\\\\\\\/OkfTqIdTB\\\\\\\/bYcnVMX\\\\\\\/M2YgwCBdirPa0PSDvP1HGS48Zc0YZWZ4Kt0S7JrltULEJkMsCTiR7fcwdv6rby2bqmsd8XLnI6UNQ45CVDiCcGiS+oWC5NgfmWLWX5SqWUKCV73rp6MSSClUwQl9qOKO5rIGVwGB1uI1\\\\\\\/VSVfZXD6q24I6zqU\\\\\\\/iiKFVDvK\\\\\\\/uP00L1MTv0qffdZ\\\\\\\/yPKYYerRRq851Gifmlxo3liOffvgSLd9K0btABXR4ebIv6AerKnw2N9rSBwkVa0v6zdk7MNAYfQTYmJQtyfkJXR8n9dZclbPxR5Q5mkYT2XRlNOa2uvG\\\\\\\/J9f4lxhoMF6QuEHLDmSuyURUIsaHOQQ0\\\\\\\/DePWFFlXxyCgOliSHp1H9ctHigclbkNOaRJAGuygwSHvJ+lE3bL+7ZsGdAFSAdQhWwwPBrFg1Nn5qHlXMAcH4Nbqoiyr4oGyP20\\\\\\\/XIYqc\\\\\\\/K020SPldPwqTO1GK5EZXvEHc3HLy7tMf\\\\\\\/KmVKP2GCf7dLsfFlQg1RRljfc5Wvv\\\\\\\/NtmM60rdqUSkgyeLdaKZVu0UGrtXQlb0Af8eu\\\\\\\/rexUtCQoBNv4cB2zZtKkjZTxbgWv3bFDzqQayN0uYrUOEc6pDPK0O\\\\\\\/n61CtoctFItRIiIgYoPda0Fj5avDcIaXYzfUsbeUQtUibYqza8Bwh99dcB6GvFfZ5UrZDH6XNPmxtQtHHQG5cLUCT2vaA8suvmyed0D06uauwS7sZ1zxO9UZdYy2t40ebL3sQ9ABGeXPO2kUzZHseQvGrV3RPZBTzQ3AVWHRGjyarocDbOXba1uCYUomTDRTNmNPzlphqWvQA3BxRtgyKLo1sqcpqy0MT2NdWhUB7Y7HH7MlVGv+I5VOZu+vSdbm89BbBMHAWNg3R+23f81SOyw4YUZS3HeX39XqYiXU0zUdfesFr5e\\\\\\\/pVUNm4irKEG8LRk2BDftEVJfuQK3DN6XtfYrMuMvFLn8yCvW6hB\\\\\\\/G5BZ7l7i7Fh+XA5IQZl512nq+ZDYtHqrOHawgL\\\\\\\/vHNOEx3OovXxpf5xCLSKdYdgXRWkVjMNn2Q29odKCJhdYoLG5oMi6evYim1XhL89deV5vL+TS3IPMZraXLNgPm5ObF3k4gjtlFA+8yfjW5PItFiW0yorQlkKzqx2miNq+Gn+dGFqh7ChzURbqAXU58tliLcxUbJnWs\\\\\\\/oUdXl1AKT0HEWBiPNPzwXUd5Kcy92s3y4TUUIWz0WwOguNWp4XIooz56qGxDaauHL3WL+kECWH5Qp64H0bNIOu7Hkbp\\\\\\\/mIbRZVIgQVcpkFEArojO8V\\\\\\\/SOJk8toK49\\\\\\\/kP+DwrEN99XSx0Y4ldWMF4mIhTqzfp8E2+sdPcIAvr0VlKMKR7jfny6lZIgvxMDmrkdwC9HJzinPfi8GU2e4ycKRSLM52\\\\\\\/kPb0zE1p612Rh\\\\\\\/nsfSAnry3YqxtavKO796oSTsNfMRdpwvA6GJwDSKcKMAczvY+\\\\\\\/MpAr\\\\\\\/CuMnYgBC0pUKfzwjUGDB+J1iII3QJnc5Isi87pU47uXVVzaQhf8hUv3XZQoUC7REJYfHypkSVv3DJ4jlQB8M1mNJ47xyrAoVqKdpI4csPpXyuzsabBLEb+K8G+Dj09XNMCOm7QhEDr+qPcsoGdgIkez+oiKC99utfqVAH2gXqatHiJFND6yasi1TcBbhGSg7wE3FWXwi9QLud+dQiOH3LAxDzqcFaKcnWsmxdZ1zdJbtMoYR8rtqHQZUms\\\\\\\/ReqwzID1OBwById0hr6NT8XIEe6KgQpMqCaUt38GOJeWbl3IQXTTZGm7P0NJFLGOt1evJKh1Cvp0fSTRabjT8O60kl\\\\\\\/h4o3ZkhluXgKP\\\\\\\/RfdN7SrnJUQtQAMjSvPWfbnYjwWxOZxZN5Z5Vw2IKQY8\\\\\\\/KuLbTpWxmflUA4eKXpKLjx5U9DfZSrgNhXNyL+93fMb8gvagsfKLWLB4\\\\\\\/3tUOp+byKliJh2xvETvZRydTjCBWFU6OMp3stKzN8LkkT1MUGGv1xrsyUzOdN9fktFXnd70KUbdlPhIL\\\\\\\/kOoH31tvRpJhRYXMwohry5GVPc7wdIis+q9opgDCkJG69r+lz7ZJqsOhQXj\\\\\\\/I7Y5lxcCo+z6hfBsWHHBlnhyCscf4d+VoXRMz+KGE\\\\\\\/yuCwroUCRipYRMWihPcUXQ2LM\\\\\\\/tYRkZblP\\\\\\\/wFU5CyyFK0Y8mMVvJqInk7TE2M7mHPs0LvyQDRZfVIxdMRI0LXT+cJlLVgqiNBugNmKbemrqVkFgJJIL4KLkvmD4zBFcI7kikq1us4IGbCWvkbk3WUJQV1tu+s6zO2SbGHfK60t79GJmuMBkaWIdpZrpBInFMczwKAIAJ1ytm0eEF1f3P0YSWWneRTTNyUX7ITDCWbdw3fEyWaBh3JH\\\\\\\/K7+0Y+UYbSTgP56kpXs+ZvFwSr74zjw7dtyuoOu3DTxIuitkovYQftvtE8zD259wQ86gh\\\\\\\/O7idFh2mMoKNCRV7KhJRecoHm9chAGCU60fsnHqUe6gnbTzZ9sfkOgAiPlC4SUs3SLzJaQrUnjzrvlh3VTEh4sTotJth5E\\\\\\\/m3PHIgVxSgkpoNWEvdVam1ai1MmzQ+ucAOS+g0nPOY73qKViOJYyBULUKDoSElIT0GhWwlbDFputhE8xRmO8GuW8WLXHQYKAG9i5m2dyPX+X9pzAdMBB0RIej9AFfItMeml4iF\\\\\\\/9bRsaY\\\\\\\/rV5h3HuQp46aEI6sz+2H7vorcBJrLlOZg8pccwn7Fg619lZpY4vuqnvEcotpeN3CXOPrEt5tiROGEe0EM3KGJZBTgcGyMh0pWobzrZ89uUpEFxRJ57IFVbZVcci6bvIuqU3ILwZ7YQKTeahAv0SJmiK0U7hV2EEURe8umct+GOCVK6ueUr4d4pwMHjQMRGk0S62n+\\\\\\\/AMQRnuj9EEh9RIGygruDK9lP+v8kCL\\\\\\\/MBKVM3E8Vx\\\\\\\/OC8o5ON+cTVD2EwD4vSsgSrl1MJ1mENVYWjdu2kKVyyUv5kiSrE6tifWMdy8diznKLaewQ++vh5uIPtVfVqpIfoqz8GzQs5TudJ1LY2EpwuBRkidzAUpje8Qop110Xkcxfa\\\\\\\/ajgMUGXrDTC\\\\\\\/5LoH7G3I8OGDwLdnPRNeqKiPEfHyEkTxAvQs+RM5GnSyBrDFmvTkq5d6p1ratFG5TqQeqNZY8UxC\\\\\\\/IPlj05e4DSpC0AvjqIUjiVTyMv2RkI3WjK6Gi0Us7Wa9ThDStu539G5ZYVgXFHtDbDsSnGK\\\\\\\/Wz2PT7hkt\\\\\\\/I3Tpmg1TZNvqNYJE6yUCWGC\\\\\\\/W+zMYXY1qZ9evTcbSfCea0mq4Mfrulv7ZNUhaXMcetNoh0kbyOSCNMDe5t+EzbmwJ25Ux6efPHgUr2PZNEMNAyDoiaW\\\\\\\/r1f3bTHDdqZidmUV3ttgINOAVYnloqIhTXsdzyidSdaifm5cGtY1peFKppxR8YWKimL13LCV9JuZDG4EVq4l48\\\\\\\/USisuGwiGONq1Q+os8o1LYLdnNix7o0LkUXYzb0xmsAo0PzWzdg4CBuOnClpiPZTcHebj+lhqZehdrFhxGndZAEb4joswnglm\\\\\\\/icT+YM6+JVAjvU9OwVGQjX5eWeBUI5O+NGwf7N0d6L4\\\\\\\/jpez78Vw3ndI8E49YHpRl2EdQxhGFUhL5IgHFQsvuz\\\\\\\/gSk3Ap0dtqTDgcwXUcXg\\\\\\\/AQXyAFqzZY+YIGdjPUEeNBQZz5mywyRHpFg\\\\\\\/jzFgghph6nO+5AbmQVYgwz4IxZVKFAZ0JbagIwrH8TpT6X4mTF\\\\\\\/saF0NxJukF8+YIeglsyz4UVpuI376xZo\\\\\\\/RGQjq3JJvb3nReoMBZ8kexfJqsfmkJp8DNewh+6rkbbQKExTvC+ce\\\\\\\/FZ67KMK11NqSq6qm7fnUiYGDxOB8oBP90Uiy08xTTt6GaDB8BvbfzcH5xCR1Lpc7yqTOHWveWezLRic1bkNHrfPS1JkcyK+NjAJTlVheXxXccZmp+2M\\\\\\\/SR8gImDqY2rA5pGeDCXR49v6UbLQk0QeIQTEvH4tNkthqVHLMfBnFHrxvFzIJ7\\\\\\\/8qkOjtue1u+8dku4vpXJTtdyZsOkd2ZdCKIruKMhLtxlPscDWQeXJqrv\\\\\\\/CmHUTfpFeV358bZgRQpjqZg0k0oEKP+vAqm\\\\\\\/SJ5X4iw4S9lfhvPhWztj2ginOEWN5AefT4VS6Ij88ke6CQVb9\\\\\\\/7VZJTTkw1I5JjWEmlfPOuxC+2IvS\\\\\\\/3pv89Go8o61uxrk5eMn0jQ9s45mGkXKLxdjKiCVQNtJx+J63368E4a\\\\\\\/CjMGjmSezfj2bmMutSflQ7T32flLFeRllOAPnPRFSHiHzqKlan\\\\\\\/lK61\\\\\\\/ADGTjitQvHY4UPR3\\\\\\\/gb10A2ykHOpN4R9PRNpHMkFqd6j5mEi1Q\\\\\\\/HGenm2ib42zj1Tb2WHzb+beSM1JzScxEk83P9XEymbSGT4uH3E\\\\\\\/0zE0R2IlRLRaPMxi7lET6oP+3uWWuIcZp43II8VVASqSj6QUn7zySTLpkv+kJTeoqsckjYgw5WEBcya\\\\\\\/pUQb4XU4+nyug2SZ95ZgmB4d6Jb32ysn\\\\\\\/NOmFtscTMIq\\\\\\\/Yg8gmTZSZfJeF6f2i8P1gbdixJqJXm0HhAqbYgq72nvr6y2ZHKdE+ifAM3Gs7lrF9ovvYXT7TOi8m16W9Xf2FoWMbaQcRadIrqFn1rLDF0El\\\\\\\/q7dSvr9NMcHwEIubqn\\\\\\\/LlJ9nWorBfB5iPPttH08N882Jzdx6DMDikbVSHJQUm9LuDnpFHLcil95NvOf6DY8zZYzthH8c4gqx62TT6TKqPdK8R2AMumysZBEMaaZYNwKjJSUB5IdzKw\\\\\\\/0thCKKv\\\\\\\/2dHAiF8evNU54XC\\\\\\\/VP7wh6vlBGd4C+5WWGXk2QFNFUgD6ydeAzKbzCN6BJazd2tuuOnik3X5nOQtkRRM1TBYtUT7EqV1Nn\\\\\\\/c3X1TIq9Qxqi\\\\\\\/wN+M05L7LkBdchvkreHu9rC0Dp6eIoscqLdGq+ZyLLLkDz\\\\\\\/jK92ty\\\\\\\/Q3\\\\\\\/MrrKs1XNk\\\\\\\/avAXdyZ+tA0hzpewo5kqEn8WoMZrFN6V7MFklyav15TWwNZRrUKuu692z2dLzyMii8j9vb5x71ZQ6M\\\\\\\/nx0ptumSBQLJTjxmZuIc\\\\\\\/hh0jDtkGrz2PIn\\\\\\\/Xoz5uhE7hjjamWmDaujA0jZxSQ3f+51dYVrVg8Qh\\\\\\\/JTbpAqYJPZ7BZ+B\\\\\\\/BgNp78bWPENvEg2Aw9SVwAWo9otElkbnsmN8gDOg4Nwg3m2bBWIhClrLRO1myEdvhSG1q0f5rG\\\\\\\/27vO\\\\\\\/ea2XfEK49vemrAJLuY0ndZivM\\\\\\\/bw7JOVKxKjlDRvCrLtnpRjWx1qTIKbWSe1tSmGnpEmS+F6yT+\\\\\\\/KhBUU3zoFdYhnvmWD07LfVDS9BvZl4j9cOD9xT9D3m2TuiG1o7sbHs0Q+WBMas3JvsPH4+9KiY2C6MPVE5kS1N42D2q+7BlRRi5EXkUElI5gmxvO1cuD82axd0OrGTsjN971EKvY8KZbk+qYvzQkfpASO7XFmDLE\\\\\\\/z8FDcXywb4u3Rop6V2TJ5\\\\\\\/zjKNJdBZc7Usoa4u3bnzkbctSK3Q0WrlBU1iF6wkDQu\\\\\\\/3xbgTIxtxLPrQtWnhiEx6KfuLx6CRrxkP77Sc0cuixED8q\\\\\\\/XmN0J7f6hfwzLfMWZjWWetX0N7yySnuEAD9zkeD2+f3MmGuRhSruUGtw6010OVsrZZRLStfp\\\\\\\/wf+z4bCXSZccefqV6fvPZowp89IRA7FpThs\\\\\\\/GhgieNlgDAizzOdRHohpPqzKpfzW6fzZdDN6juI50UoSOk9QZR+W7LANq+Qw3UOtJzaPICK5aZW74NZSpXIkOf0Cp2jJbSAyNFQLFI9Tcnw0vlYzCDmZeRtsGnm3q\\\\\\\/fZDX5BdTzjRGXJdVgFntApN3rtKOmAJiybrkuY9tpuZJotg2pHOD3utiLy09MYTvl77c9dr8YA\\\\\\\/XuS3kxV1VsczvwBYzLrEW7yQ5MZiYvTDHPEZ2kInqW1N1iAowqgsVhtC9mI1IT+Vrc39Mfoz15DhovB0EGyBPZ\\\\\\\/qlBCicIFXjzVEXaKvFJXKCwMTtIGu6BAKXieKuCVIkPs0B5Orw6aqMw0JNKMjA9Car0SW2TCC48qJB5MgFLsWYPizqsrEJI7xNX1QHtzN9n16jbkD296IkKIiulla5Xrwzu2dpZn\\\\\\\/90znDdxXYZA6OmmaJ5+WDtHQxBsMny\\\\\\\/451rIzdhTI2dLhwCaQJtn2CHs71RW9u2gNUEf2yIoefxB21tcme2\\\\\\\/+3oi8VVSOvHSDRSHjho7iGd6U+MngSBK2jVZcDRuL+kyviLussBlKg76LZ1fEwOt1bpiOJlHpKQQ1YUugkrLaX67XVa9QVdda2O5IC5Y+jkY9+vSYdY49sRmP68R9r7BDoNaKf5F1dmZWV\\\\\\\/sAumGIVpVYeWOEUU5kB2uIEe3HFkz6DrtXxL\\\\\\\/F0UgzN88M87hyWkKPEt2Bd7jhkRct0G\\\\\\\/HkrJi8YdAuxlYdxYPWzdPbX8phWe7eq0\\\\\\\/o1ifGkzhS9xz02EsWatheTI7ROh\\\\\\\/NZzcY9D+8gE8Cwo1Q24NlaiAsXY1bjykOYsGfxe3lOgxNcMIYhJhGHdly209VQyNzkXJApXCi9YoqZG6YergAFTHEsU8nieb5lYhGjQINYt1VZ\\\\\\\/fU9KbiPvl5aRjt\\\\\\\/RqXv\\\\\\\/4YJXPVkAYFjHgJcs7f8J4rhdiLC1VLjOvhgn1qxpoC\\\\\\\/4IfI89d+vSunjChu2mV2+XtKeiv3VOfyvx1cQfKTsS6QwO9kyEZsvlI37wHmSs\\\\\\\/vkXsrEWKeTbsoCANFfnPR1Ujg74wmK3ffiO4Oqv72JPTOWPKb+Biiwfr0744ThTe+whM7ZSXgGAHCs09LSju3nie9wPJM6cbVq322Xz64PU1ng9WsCDEjCl5hOWT51ck8PgIayyaHSC\\\\\\\/yt2JyUzoO3L3CN8OnshEN0\\\\\\\/ubuQZF6tHIORI2tLlRkv7Aw7oAUGcEt7lQXNel1QJH7hLwjg18JIiJNDHRojuqXQdmCWgtuIoaibrrJhdPk1PvdJk5V91EvB+HDCxf5sFXC1cDQ5hEgv1M4IBnH8Z\\\\\\\/SErDZkscJKoaTpbGHJLUPCWI9IkifPaY0oN020zsvAHtGp4k7qn\\\\\\\/qKfkKmWiulz008Fh\\\\\\\/rfGBwoXRF\\\\\\\/AO2ChJlF0zKP6Fr\\\\\\\/Ep0xJXv76TzQt+4K7U\\\\\\\/rNqNZAZwHi3YjVruFdyz\\\\\\\/E6FveXTbRNv28E5EeCF2YDldiUBo\\\\\\\/883k9Knqx7CcbXZcjfpJ4Y12ce\\\\\\\/cFtNaYestfOIejxLyNBwAvjG40H3s1ACMaU6t2yB+VrSF3D+PPUwuHExZmIu40k4YI2cwRzOJBEPyFHqeDL4O385yH85e23mmuIFhCylRAkfWeZYHZeSZjmXc5xhOI48xBdvFMZzDDhQtLEKVsgiR5\\\\\\\/FBibGf76qX2OOaaxz0K8WeaL9oG7Uq8XOMw1uEVLpcGxz+92XjaylACfzJlf2l4akKYT2rHK8Eus0jlx3lqLe97GRrMRBpqE3UaVlGDJbi\\\\\\\/gr0BmvxTJKWAb9LxNH5pHqGAHB986mBb83ubKUusDyxfnWMsmOL0U07uvxF4U5vJMO2pKheeDY9kkdtpRAHSkRQWTslHBwoZ5mfhfRwUhSgUXlr65rF5rU0wOhiTLQNB31OOQQLdGxMH3JFblS7da\\\\\\\/m7Plqh6pUHXm7clpogNH5Bp48drvE5f++EHdymmVGiCFIJlnSBb6\\\\\\\/z0K42iyemBcvV3gtNjT75dnC4SQa85YsdAu5Tc2lWRpDibBYUVYnB0OtPjnHxmaEHEBO0\\\\\\\/lgKdNM4Gy7nO06MfPg\\\\\\\/sxL6xXIPsIYvI9llrjYM0VwUacK2NsvGTGU9d8e6Vvc4wCnxJ3h8uM\\\\\\\/6kGlYHPsWL5JznMwleoNKbQYkw8rHTJVIzN2UDM4WFBNEePBRthwbSeOHgHM+a4MwqGaAw4TTpuTOTztV0c4J7VK3BS8ft0gh4rt6FW6CqEQBTVuuBeiKeyHDFRbLJd9fso8N+WGLp4kSqYVh0LW\\\\\\\/nFDz2mXtg4SRuXzTFvboOLwhoMQW+R2mFH5JtqmzTj0jk5I7YU0LsuXAWYXt2qvPH1eC\\\\\\\/KSFyAdT4ad+Ors0KRXREipkAgTLWFk+chF2th07XHmcE0EDlRkJvZigJBSsQSwQ1IxkjZ5h+2mD0X\\\\\\\/HJqkvzRgH1B2+ItJ8lYkHo4QlJ7DJPTxb2fKv1wvbI9KDCYZeGX18\\\\\\\/8wctmyZeO\\\\\\\/Ehhx5mTXDIPa22QZxoOrTBe+CVx9v8EeEaSVtQpSnioe4tlFpa+JbBz62AS6dT\\\\\\\/9CRV1vGcQDgm7SuWKTasr9cpzd2l0IOMnT2dECIvVMxO442VM6aXQfug5Z8hQmtJn9dskmJCyfFDWsU82XfDvf2fgrHU6GH\\\\\\\/owvrKmY688QjUDuq6QIt0jvtvMx+B497UXsgVISv+3XnE8BtKA3pb\\\\\\\/x4IeVXiRDpIEhL5e1GvCPFRIbqtRHbkKlkAqEAm33k3kvALOlUBZp+fsThnkUwrJVXeyGVskuF8TZKB4Nk8iRuAXEFGCGzUZYsfy69d9GAbQC\\\\\\\/iXomafzA6In23FrovJvPb8jd+I62c2TanUJ0sI83lx+nU560QvyPp9MLzpt4bzJndOqTLcKjEl7gNpels\\\\\\\/D1jXuJckqIrB738x5GUK\\\\\\\/liu3ok8phik1OhvgOztuwPPgKib29d0LNC+Isw4cJ2sIwP3m7XrTbOW38f4FZjfv4XBnsfjg8VxfG4vMT8T9fC5\\\\\\\/mpfVAcnek4CkAg7dUcOPfIPiuC+c3pmIMfINFAVwaDRtN0pWWQqZiczVYlYmUYs04jkevN6RWLYu2fr0D+m1v6hWdfEZX4xz\\\\\\\/F\\\\\\\/Eu\\\\\\\/d2s\\\\\\\/VlNvTAFvghEgEaf52MNQ2YR10TkTtQkqRNPzKS+kLei6yuxinh\\\\\\\/Y1gMmogWfZcV3uDpNGMj2\\\\\\\/oMSHyVZwwPin5mM\\\\\\\/RHLrKRsH2f3zhx66QMnnKckyc0TbXvHcAFgGlW7dCgSI2H43OV8My9Wg6RWmAwivkZA\\\\\\\/wnFgis+NLq6Lm9MreEZD6sBgt1ds1T15ztBP1ceVcceR1l18mp\\\\\\\/dUO6jE89uYCXXKiJp4qcLPta\\\\\\\/JSKRkPLGcRudX5Q38o\\\\\\\/f9rBDtC8975YX+0I5ViE1hW+9GA62zgbfzGyNH3HfTgl+cW7XOcEG4H8lWcNNAUrJmMx7DjyayH2NLsPhvXWCxHZ+wV0Y1YQusxGVz0x5lQ92LcD45cuFHLhpJFBUp4oKxg4FRwX6coqHBMLj+uzs3th4u2SK2zfZbtBfiFH\\\\\\\/9bPEjZNa1HO61yDuUbl5YbrkkyC0xiwqIAbcGQ9Sx8S1R8yMNizZ\\\\\\\/UHlC3LNbfzf9OZUve2dxzYudUrvteEpHlJE2xaGjffX8su6X1ugKK\\\\\\\/ldIfLQpp92o3RwxUegLfIazvoZ3pWaHq6AjTYBfvwgb1UWpgUyxU\\\\\\\/hgsnq7+pfjO\\\\\\\/K1zvlNvfMwTnzGgY6inejxfaWJIjSPQ\\\\\\\/OSBMGg6YeHmSWbCqZMNbJRPP2LRmBYaR0fV6Wn8ngn8K3QZOc3y7r6InASnmwFf1KNX8kGA5oB2xYipbmkTmCiC+pIJHBdAeH8VAdDjtmcgjYPR\\\\\\\/g3jof2nuAlC13Nm98R0b\\\\\\\/HTuWSitsIeZc4N6wy3YXG+pd0SSiyZqBN2\\\\\\\/l\\\\\\\/gZU1PKh9v9GthutIXFXBM+fafH+rOfW1sZWlG7KkjR1QAvpGVAuC9rTe8n+vjfE8+XUosDQk1mQi0GkLhFWsCWbQomnvAakvBHv8BXzBLjxYaHf1ORZpBon0b\\\\\\\/HntP8Zc9d+4jbN7PWNnb9iJQvOudumojNbgV0zeQ4JESj7K+tcTFWHALMZE6P2RvJfdXUBV\\\\\\\/lL\\\\\\\/VsMnmlz5RE7HA227UBMYN+wsiGdv1330ybIWhmWdUc9QE2j5rtIy4r\\\\\\\/xOZscSvjkoosysmR9ZiVaKyB4MDohtL4pWtxtIV2mGu7NtzAs8lHnOizx1V37hU63fr1lbBYxtxls4QXCOzuH7qsJt9AGf53ZjrSiUv0zg3nADz+yNwdfdC5oB+Z1w9g8HG\\\\\\\/lIO7TS1LptySBEbXas6r6S6xtizL0leaTHUEUX2x2EJK27taP1cUsvFJ\\\\\\\/YO21DZeh9yGhdjElnuy8FVEu6RFoB+KVzfinLHQm+1uUpZymEF9GabtA1yyCarOjR18PcO88Hrdzf6XyCL9opOd24LeNoO+dQZqeJrvlKovegPFGs\\\\\\\/ztKzjz7DRsGJhXoYtcHXEJdXuN0W4jT8wq1VDzRdtiuRghfBxrzokIebpryhPJswe1UpFHqt45LskGqsSTPxnkJPjYAfmSkBqQrxM7m71MX6q0ALfMuv8dcWcNXqvK5OAxPbVgM3Vex4K5L6Rb0W59zzOSD9O9FRYJ7t27fcPY+8\\\\\\\/gwZ14sdhDrr\\\\\\\/X85bCABal8XtGs+zhFiGmy4f0dK8GReUfmM8hbXNDPdrX20t47uiZUGhylQMGTT5+NXJ1EL0pr1zbe1A6O4H9Q3NXP6rQHeFsZxFgiSkeZi+LiUlVmU+CBk++ibXCkYyGc0hiIhRU72MpoieGL5HGITb0K4zqbLLqvriV+1vgxGl3SftcOIzkjqCAj+z7fLtcYQymE56PmFIlADgl5lvTWS5bqVwrjljyta0bnJKLsN96OAACvNGDq1nKTtBNlNh2QZjd6p81LZoE7xsE+cLQqqzXz5oBrtJs296aeA35eEfLfGJ+diLQRAVbkuKRJBnBy8mQywspwXu4sjPD9lziBux5\\\\\\\/qsK0Xb+ARc3NauuHC5jMJnAlw0dlUtgqXKh5YRd84EhKmdTvX15Wyrd9Ng4EUFga3Ciq4fYL1LwRfuJThpdY5L86SDU1u4EKfb2qgBQk1ADQN0\\\\\\\/u7sCwGQet68m\\\\\\\/m8kexm4Ko1ual1EtlJSAW0\\\\\\\/2pZQuCIFe8BDCiO02Xg4EenU43wyZULtLTgFpUA3pRSOwm\\\\\\\/fg8v+mz08EJdxPMAY9cdigMUrqULrOoaqY8h4lTjjMnWWG1i4NkpDVHNg2yHxlP6UsNTEe155LKQ1AB7Ba2CED+oflXX7do1xMPAy3D8jcZ2DCmrly7P3Uhjk8sT8Uj9FUWJ8gnebS9iYYThD8ew0vzLXnJ6TEVAamNjFjYwrNpFICjOvPQt5564DcZD3zHljaCdSSI3ALPikXIXA+Ce69MMqzJX3FvAPClSoYHLAFuhu9Va1p3di2qULbpWge7y1QIGDplZSr5+d+DLjFy1zJt729vxbV+svFTFHQ7XYS8eKRdBAO+uGi5pLgzovu4uAwqWmsQ7yM9qhI1iiqh9CcYud4KdFOIddUr208mPBoZY070uSsx\\\\\\\/pj6Bq5kljUXQpevZbExMXXCt2p9cPps8Rn3KTPxE0S1+VcxKOjx3AbEl3iCqF6ag\\\\\\\/lp5VQphuZPlSbR2EsjHjBnidyGoEcEI4Hf2PrC\\\\\\\/5dKCP0wLkqKmcuaAWbADqdstkJcoPKWDjNEzBhp8kG7JuRC2jG7KCqu1ziruQuh06EkUIBoDbFTzeV7iqT9jM3imeji\\\\\\\/T7n9M85rLMyf4H2BiyfVsArSwxAI9tZ+RBhqMU73keu3cgY3guQVeHmeuk2M9X6j3N+9OzJa01Rdf6VXZWmQPEtZ0XDygcsa1g1u0Fa0h2yE1NG0RDrPJ6+pzcg4ISwJvvkMLoJKK+nWTT04nHtjTSE5lyg8K8PN0sQNm7TAyU0mEj4qKw4U0HjJ4rece8IAVwV38FJ2e+SQMF6GMoSBcbqyBl3JrRdS3fqIZCbCn\\\\\\\/TppFERiISHYOdij8uj51pz+rYHeQkdT9QLfBu4hzNd7+cJD3NVdhHk35USJROlX\\\\\\\/1tfjzvT+bW6+tpXX3K5NIx9Z2lkvIlaux7MMCqn8\\\\\\\/G98d6XianrcZWzgcS1+ZfWwtDz6FQuOSAmtUkLssrqp\\\\\\\/UJZGzpdULdfVArtFgZX+iUap8iu57XLQ8tLaWLjuVyYxS72Henb4ijCUmmP3k5lOVGdmCosgBDEOxHwPMa8f9gpPdeKvTLRs6it+XZkDO3gXmpJEwhw\\\\\\\/5psy0cOSaJWhkDJ8V+wp2nNDOsd4ybyLEmcBseLl6feTShvqQkwrz1a6qAZF3PR1U31+FYFJsVgeFHZv6FClLZDcAF5nlc9Z5mzMWVf1MKHVny\\\\\\\/ZFAFjn\\\\\\\/byw7WxpYhNDxZraQU63PMFxfBhLVb2ovSGsFyZ5SoSvgOTxEix7e+VmUHXiR6opiq0uMgWy8a2z0Zgzq7uKYDr6Aq+dLM79E7im6QvbwTcg2UwCXYn5S1Y9VC5qn76G+LIpc2a\\\\\\\/IWIilUYacWQBAApAtqm7CmQs\\\\\\\/qfgIcqM9zeW56OlZWClulxkXYELPb2ps4W\\\\\\\/TPMtMbmeY1cP4KsX1\\\\\\\/626bJfKlmCN+MkShyuAcQb8lwgus1VrNybFUX2C\\\\\\\/WLFH+pKYMD3DHurBPpuLzmyJe9KSHFuSQpg4MA153cdglc6Fp1CzDizGIg6Ak3NP4GKQE1tNQfxTZ+WkmIHhWNDsS8e3zcC0F7JyBcK4IBKTxCPxdtx\\\\\\\/dF2j\\\\\\\/YdShVb5PzvOCJhEoQDsFGnemRNuTfqn9fl7\\\\\\\/irVTGQrygsE6iwiNGnLYnr5awtXXJ689V6hSCBbgyytfJOfiPUWeRZOk6Xxm4\\\\\\\/Qiqj6kd20l+lScMKvXD47uXKxej0qspxkZn+Jyt7lBWghSClvnD07z3ub25s2mkSz7SoRU5U7i2NCHSMRj6lUyba8UZ9HEBBRLg9T0qyXopdwMgCUA2LkB9Y2kO4hZpRnbGq7fGBP1dRlacAbSmv8h4HXFYumBpxFbkWNyl\\\\\\\/pob88GD85L8GvJ\\\\\\\/HT1GkimdT3trjXd\\\\\\\/nIhRCZWS881sIRMV7iU5xrTrnreS8Jindk6WWMWzk7mQJvhAqaqiidwmQwwzY70AB88GVal0+3WFRz2HpDjGgMQEzmecU60CZJAn1hmmz\\\\\\\/b8sKmg7tIVWqMZVO577Lgn8SHK54DNiEj29XGHm2RXDpabYU0kdlb3LwsASlQEgJ67O4zXWDJ5aTK9hTBd4PAejs3GE90sC25dQFE0+BqlJRG2lI+Jqlg9OaOR5swe7uZHrfB3r7AGj2ZzLl+fm7UwkAQSPIjKBgPQxtxSPUvEfn00xxqNNTaAg4RrToFy9gqwbRMoV9Q8Bst+5IHBc1qaWxplz\\\\\\\/FKrgq4JqEGLn9slgmqZG286nOs0n0blzwHR4IxCaC9k6ScR6QGWWRNhSnUB3jqDuZSHQJd+uHJnwkPbhQEiYSmM\\\\\\\/dy2DAw6l0jcfW\\\\\\\/EcgLsyCX1E6kpaQUc1O1PQYsms0NkYYaMfspuhWCc\\\\\\\/avRazOyJKjwjFuLQMvGaCaTDrz6zw00eV14sQr4bg0GTDbxz21he77WONzf1IaV2ErF0Im6Tf6y+Ubn4XVIUXBure7jYa9\\\\\\\/7Uw5PScHK05cbCQbt870GFLbGEPWlnrK\\\\\\\/QLHmI+lkBWwe8qlQVEbbR1TBpyA7Jnv31mP\\\\\\\/bcEg8O1z5GIMcwOLYckelMwFkOz\\\\\\\/iovF+BMLZMzLcci2l5XYx0QVmNPGCbl\\\\\\\/fiOKGkdxBlJtFCqxBVqmf1eTvxJICwaMGTCPvhQl2CKbdeJetidQz\\\\\\\/XJGWt5cANNkOmNJUdjQNhKECRwaT\\\\\\\/It0dsh2Fb9lQp+0Alvooj9vCbpe319wBqSpfqfg2gRqNHSV2dGGSd\\\\\\\/44ZuNj3Jmq21WGRn1Oj2\\\\\\\/woa0oMtBCPUjV6aU2llm29LzXpvwGr+ArddX2pjj0yviKVfBXaOEwArEKCqIIbvajV\\\\\\\/oSxrjgKuPm4FYzZtJnVuKzuqUoaU7whuTkSKTOXM+zqCTMkoViPPgo0nPMR1xl8lsQfTNpShT2YjFd5Qnn68TiKcVJFFBv6JdquL7KmkkSyPxmxG0W0ybqnCrciQxBL23WFawYQG27wdr01GmSw\\\\\\\/RpL9c7rcTLinduUchit7AKeAmDCaZgxTekKhzgI\\\\\\\/51cFXnVFJszVmoVZVAS1awCqdbQzuEQFT7WwpoNQ0tVvAXR6dF+FihaPk6fxTTqubF6+ACrBL+F2QMn0jTICzEwFS9YZXKp8kxiBnCW+ObAUcgkjZwrJHDIAElpu3bNO9YVwZrou\\\\\\\/boeePnCCT2twCvLkTIQSQUbkhZXiHgpHxbqVJQcfDtlh47m82irtlrg0Q3FBoM2qKNY5qXiHbQNETtdz3FrLNnbzJuPXdIVn1H4GEB7sVoI66ae10uxIDX4z6tszJDcCL1SYPTMuG3izQMgTThBkbsZzPWQiYgGo3LyzUIrySl2f+eDBKrgxNl001tQm1GS+sPl\\\\\\\/P3QOzkqHwzdGeZg0K+cXOUKvz8e0NeZ2Im25swBBQT0dULNSOPrBrQpOu3oZcRFkzZxFkNpaoQk8p2woiHsSAJwo22CLsD36zrsrGro+x5uz56uHtw7BsXUsW8BKizsnW6cs9mYSP3tjW03gbtmn4mu2S8XCHjJYVQ5jJLgSv3d\\\\\\\/fe2ffNsLDKXiRnSu\\\\\\\/LImsU18LE8aAV4np4s+JfeNbJEfMEcpFSx9aChhj+vS34qjdUM+3rdMC8RsjFwZ8IVRusDGqVn5ZY8HA8sKee0ytFcNR697Q74JYdCLn+Ft+fwQH1LVXcjn5bSDb3HAPqJEbxJ4oagcV6Zdtan5WwkFiksoIsn2cAoGtwuEkD\\\\\\\/DrSDqFNtdCJ5I45WNxhts212Hmu4d5z93iTxqW8dRt5OerFx8AvZPtq3kvcPsorLfdhBq\\\\\\\/ujh4fhZFY1\\\\\\\/s3v6qNyaopW7F0PTnaouOg8dN95JjNDqbl619lf7X3TkJDwwBhj6+csxzlLT0yCaPHWAMPWOU08iWOGhVCxXWaGtHSzvsmjXOoo788d1t3qCgadlH2oN5NFXQ5PtwFHKCB2kJbZMTVAY37SZLfQRebN7n6E2zuEwrGItWQnZVACfOk1g9yhSjmJXp\\\\\\\/beHGMw2nDnpsHDfaR7QCXkHnWd98yRZt3KeY0Hl1oZcwZE78Rq5yFlWHDb9gawIAbnqi8JHVcxx2enF7dopvuumhNFG7hBk0V2U4gZlip23YDczVxBKf5Wv6TIjLAdX9jO1A\\\\\\\/MffBw8t44leuLIKnTHTiUVFNzEdlZhltpA6sykJYBKYB1+q9+VHla2hOzdp7vpYbVPilD1vHl\\\\\\\/cQPGp5dtPxiau6SfoVzVQqKUyxs0ZKOclZIXyO9bwDLxLGDHoBSo8FHR5jGAy4N\\\\\\\/kwzPCPSay53zJDeBXd1HxUDqjK8bMI5IakMaDK9vDhlxBeoV7HFpNJ81CtOo4xXr6vG39gwnhKBosi0XR6nLntJiym3SuEnQ3Lxu5DlH89+By7o2dIOo+4JWkB\\\\\\\/tXlZ64HmPlXMoE3yMctALGizWwraYhbu16lwibztp+eZkBHIorcjRcA7aLEsZCqDUNs1HYbQc1gjbx4NHVqO9gImzPB0d2pUxwI9FFoClJDGnQ6BxXNcJBW2uVn44dtFtZXI3LwqpN9Varuj44mGECyIB1A5VMQb4ttO8Exmaix5bbyV72QuwB58AQRH0gvqz8Fr0ZyjwMsKGErcp0saA5UZTGjW4NywRvfK5B32U1tt1FlUcBnyzCHFc9hBPJHYKYFVWJ2sM9k3Ote5jiDEx5rKZQ4RtDOfiOpRhbYrkP5z1FTh9h1AfYfA3RIzXRKJ5i19Zg1Huzbrj9i+EFjz32O8wKhv93ubWsdGffuRKTRyUfZKI25nWANWwb52mFwCxZafDkHiCgxFErXYzMWouebyW6hIaPl2yCeg0ise+D70RLqVILjIMj65xc0WAJBriGvFAp6hXOmMBS7QrnZChcrrBmT83IzFw0GH1+o4Vuuqua3kmcnzEbAneFFB2zobTGl38Wlj\\\\\\\/GmvyP5kQrcVjbRv9niXQWXWZ75UT7eheozEQ2pLRCbkUkCCnUAYE9KgAe5mLFV4hGXQAgH4k2bNZh0kl0GaE6UehJS0CtMwSY+Wco7ITMLLfk6lUHUnQVWIB5WouOrcR+tKdOaOotuHojjs1Svj4iJT6W6J6zdOO\\\\\\\/06zX358GU+zBIjDe2lgJmdwliLK+9+nuAtZTxf0ByWch3DBgJIWnZbN9Bxt3wmls1gQc1VLBdIQB3WWZIN1mkC0qJNhRDpQEs7ahv4xzAXS7oFYSnaGvU2elagnsTNzjjXj4HgolTjDlHWD0uNSYEFZLyc\\\\\\\/Ugz7m2ukPH2xQhLlTC8OyxZ19P5WOrRLPjtGoqnveyNLgwjQzrHp8mLMZGeX2U9XjN3rPIYsOrfIhsMg31dEIiJdIFTXuGcRXVQj80xc65wxcprQ+b5L4L1IBMxh8aJLAMDbDubPVaWcUqAnWP4Yrs6S\\\\\\\/FeEW5vj3vsymQJkulrVHlxq4giGHzfL17KbA6RWhU+mE10RShc4+BP5sTr7Ae81JvyQ08qz2eXrvIHK\\\\\\\/KEZ5utxDx5CnUOonX8\\\\\\\/KGyy9yhKo0R2Mh7PemdqtVhbRWGFdMI4My52dL+vLWDi\\\\\\\/i5mzTNK2eMkefG3C8U9TZCdV\\\\\\\/ZATJYrK8GQ+Ekz6ZCfKYkR8LM43A6MUGjhP6HW51h9GTZZ\\\\\\\/GIdsA7MHWX2ItlO\\\\\\\/89kUxZ\\\\\\\/U6VrrBEsIdIJyPrQjTqdJQxZf49xYys2BEA1ENIWMx6l8QaXFk3ImhxhTHsHNfscKlsVwxks618jMqOpoOmgEMiGOKk+lCIjsHHXYoeR0O0lq3C29SIIu\\\\\\\/Q8PMBXkHAM+BDVVx2nPoQynDlHo7uJJIF+E4vsfzMOGCBQKknsqwRVWG7hd5pRkXUEWjatIATb4IHMjMK6DHyPHWG4fg10Vmafxnk6x0UGjtMelGVhx\\\\\\\/ESAqRF4sXVoU1ChsVMpjn7xew0l2FUGx63bgfbpTqNzXvpX3YxbGj4N3fVplraOki+o5TN4Hbat3LWbThGF2s8VRcVXE7dygi1sF0uizbbIFsGK81xPiVx9fyf54Y+tkix1WlCJ2Jdce6OYWSkQ7n2lq6jV4KdGzwP+2CiRCUHQVUo7OhT+KCnlzXbf3+NHbSHnXJ8w3wOp7Lb\\\\\\\/UdJoN0nnT33hJ5gnhWaM7KMhXt8GzHXcrJeDN2684MXw74Dlm1DeTiaB1qc+kqxKOhfzns+IbPpRjvhdTQY+vI\\\\\\\/WZ0VVnpbTZcw5OxRGVYo3TgW55k3tCaVC\\\\\\\/X26J0K70CanIHNMD6PPFCW5BRaY9M4cHPQGvsAvWKJqJ6eSCxD2+aMX0\\\\\\\/rTnz6553OF99sqNXwAH2is9XK7UT\\\\\\\/v1XdNj2wRagdwwqZN0ZX+0VUGmJzZ9iuqblr5Zl7jQvGhy7EewuVfIEArTpT7Z7\\\\\\\/acCepBieu\\\\\\\/4AxdEgkEwdatWs2cZNk1zJKR5luKTc5jyPqKo7xdl3u0+efd4TQEo3OYaoU09MMq1cNSLAM+gUSG3NJUo7X01klhFZcJuMUNkft7p1lGVKyy6z++WjbQIzFMByKjljN8FGaA9DBIO0h7aMzBdAwf9VIFt9v9ax84eYEJ9yNRDY+IVtpAQyif1j6h4+RY24qhexC4CZ9P7+egtLl8T37KCSmcI8UoC3RrwL+gjZHEiGv0cITfZnc2A6wifC1yuJcHcShF2AvJmAOce6ZjMMftSUYdFhN2+lPb09d32+fSk+Y+mUAldULAxzke6XJdWFBzRoE97HEz+yqdj94Qkr0lRivj\\\\\\\/dxbHsiYjeVYntmrKNzVgO3KKvXxeKlMyqK\\\\\\\/DbiACki3gKBbkaMpTYcDFqo5N5AQY4ZO6FplZ0QitL+waZX2Sf7NafH0M7i6uhxH4muNHQXlt\\\\\\\/NcfPN3sRcqVZt\\\\\\\/m\\\\\\\/5akeuIsoUb1x\\\\\\\/O0apDDdEqWCccn8tV8msaF4sW\\\\\\\/gcDZI0\\\\\\\/lUyUkG43bh4WByvDcUZVoDhjmfUI+IqR1P3\\\\\\\/tA3eBtr9hWtiAALApnbN67g906sV7I5ldN4inJuNC2GEU4zkTPr6B4KPFUyf\\\\\\\/Dyb9CH0qDZITSTIYxtJeCoqy1Og1rWn9EQwKJNzngS87N7WuUbAiSCdPhg\\\\\\\/e+epqpKJYEDpax7EIh6lsKliPtfsg5LYv0\\\\\\\/nn0njnHQer5H9rhqP3cr2K5KwIi0F+pk4ZrIsACXATFhT+8rjAoIp3BzZXsf81fQmvBgNc3tvyZdavksX4FIcH4wCUmQH4W+C11lp1lMLqU9zeVWmmoqKf0Qz+34mLl5U+52Au8+kngwmCG\\\\\\\/Kp7FtwL3qSPwQaw9igR2oHJW0A\\\\\\\/dBjljPhP6dU6qJoE93\\\\\\\/AwrNQayv3EppXDm2ZbiZzBXMYtymDyZPsyGlbbwUvmBc4Li6BJpEuB2ZTJeV+6p2887v4JpXavnCCDjm20Xn\\\\\\\/\\\\\\\/DzeHjVCkcYdo8sDJkebsEGZS3GERjiZmeXXw9fcnFo+NfUuELLsgCypMUxjnI3CQQvopzK1bVt26bTpeeC1\\\\\\\/lBRjXSn1UEedZg8E6nrIYBBXuSqCUuYfp\\\\\\\/FijuP+mkOZlaryPzP\\\\\\\/UKfNsdLoHQSNAe90ZipwDFjmIl2oOPbJ9txz5enqbZN8D9P3MJQIhvUhY1L01CpVCUBvHIJqgQ3ltSJ85g+SkUAfw7K1fK\\\\\\\/7WVaQHHs2bhldIHUSrWS4AhaYwkSIbPvmytwiBo5nq+X7f12rnILAYFeDt\\\\\\\/L3wNECATnQASuSOt+ahb\\\\\\\/YNvcO4hiqgww2a15dwyFXsI9AiHTCgkGopoKpPibOee3cga6h0+XbqJkrHdL8Cef19iGoyrRar5KAfcy5gOL\\\\\\\/XseVYGTGkW\\\\\\\/ThsMLMCAUrWZ1mHObRQ8IOZ2OLAQbRykS3VCt9so5BVjXc3jRSLIP+JyNWk7N2YIyoo6WsNtbXd56eOobRBfmITYCxWRg7y0WhwDmi+Ybz5gVB8yv2dSnikZvoE\\\\\\\/EHPYrIPosqBSu8S6x7IVz7aA0eznJ9A7pElI7PlyeVQAjHExOlQobVKrgXhMBKotAnO851FQa0ikhmY4iL6\\\\\\\/ydquFU96Tt2y6+hEibq27tGUnHCWLtCi5MeiJnfKubdoAoWdAuUVk11KgJqkLtrzG3lL8CRTI65hU7p0x1ZCK65vu7YASCTblkKhoz27u9ZvrRUCtyg1s0VGCAvbQKZZst89O69lghOkj+yppOjJ1GCtCI2EiRMRAxldLSYsowmUkgbuVcpVaMmJoSFPSN\\\\\\\/ufN1XUssWj5JVFOMXFhJ4vF2nNnCGWGNQNGFPpkOVLcTtpITsIYTkqStPXNx5vfpj58r0qWTs4Pr6mlMXUWk63rW7NC7GOlAtRQjEkhjYick\\\\\\\/6jq0eJvMUWv1CddekPGHWRNOmRpio5b9Whr483DEWpV0CRxz+dYoxNuU+4u90\\\\\\\/N1fjM+MLReN+6Z+sLMGknMr8vSLER8DUKJjePB77wF2LoNjVG9Y0UhzOaE807xql9Y5QjrC9KRkLWYaxIxoL+YZx1qkIs30kKxWOtklKIOEJcVZxwlleCYqmbwuZWSlccsxhYUbPyPGWk9nhetapk1HtjELevdeV8OzdthknTynlnc33ivaHfmWaQmD6yyQ6E2uw9q9Rv82BHYSc5TnkQyaxMgyl43+RmRJ+d4Eva6wZbtSu30gLeElMFtCmnDnoD01JsRHdTwd5jd3A5VMFMdSGvUDLGfIgzfP51v\\\\\\\/gfxiYWjF8LPVV\\\\\\\/up9vs7bboeSJVXgGL+TarprO2lE8OWvdBVoidlh3lDRsker6biKpi17gpmWjC0LO\\\\\\\/t587DedMkEmDrYrbMOiy5NjAM5fE0plSUQmimkbP\\\\\\\/iUD4akHFcd3dMZGjtX\\\\\\\/8NBDLLuqOClj+nbm5IhQSiafBgfV81NrTZiC5od2U8x\\\\\\\/Ms8NG9jaP\\\\\\\/7+voBcqcW5oDXMuXFlvHbRTWrUtUFAt5DrmJGOuDrJIPZN\\\\\\\/4bPymbWAkMZO4LDwkfDJGVF\\\\\\\/P71nMyD5bfCgAQ+1sUWBdIwEn\\\\\\\/BZAbgn\\\\\\\/AH5vzgLxQssKV3eziRbj6BroxvKrpWf+VYlCIsM4xN2M+ih5Cv0T3IPLJ0a2pMOEJVI3\\\\\\\/a9UbsfBnDc1T9CtYKfcs9AoEJEEeZviAklYRKS9zGsU4Kl25KJJpZ0HP+j3JgtGgbC6z72RtjoXrr6OvBxmnafMW6kWkGNMwG29b9gQAVK\\\\\\\/OjJRSeZYMooxCRjKz9ZQrppc3QsJm8CDzFpjMVsZ6gQgK7wK2HGlvYZJc8cETbh+3FE\\\\\\\/izxSqDxUUsK3S8vGSIjRiK6Xkoqspst9l\\\\\\\/o1ZzKLhlmaHmYd76N1Xn2M1YoFwo6OiYiLBKlIHq4hm8tVWJ6AxXlqgZpBa7LgYBiOsp3o2we1N+Q9+2QQ9rm\\\\\\\/QWByb9KeKRNBRMpeGkG5cLyEiF1yB\\\\\\\/oZCKJMqGVBm3QmVrZeXOTAvQ1+7HDw4P\\\\\\\/Ru1P02gZsoUbB20g1sh6DyHyDilpmanVM5JRWm0HVB7WR0uaOnRE\\\\\\\/O49KfArWNOFtQOpKPKK6EqcztH5XaM60R32MGHjNqyfQTy73bMNxew\\\\\\\/d8UCRBJgWIfBW6tByD4xKnK2WIQxpM6e+cwwwFKs5A280ck0auIt3rqgxlZBf0XPToSMFYwViobRKPlZ2pl8n06lF06ZPl7DLVZdLlx3gDOvNVdtOxJrgGfn1t0yt7nmm9hvY7iiQ5\\\\\\\/ofN4ltj16kbNOTTsS3Z5rlf8J6E2QsMtntTdJMRxL0lQDhTjjhx\\\\\\\/7lMOUSfVejtVi+TQ0pOJdmNzbS5Q8+02fiEL7hLFDRnn2XDWN\\\\\\\/3UDIoqZRLCqscnEb1sfPq3C1k+cN915f3+C12jKEVK8SvVC6UBu3fERP9U7k5mEqVEfC82Q3r62s0OVRrL6nprQvEsFbK39Bz1rhtiwXz9uN8S\\\\\\\/WaoTYeFiNe3\\\\\\\/Px0SMqYyfDg+F0v11sNeUmigDH+ZIJSri82qA3A1iCgpCci3fK2iCyWH3NTiPFECe6iBUCiQOhyb5eawhSe2jX1cCQ+417sxc3s2qs8z8qKUBQUFBb2S+bKH+BQBUQd1Pq7CnfaYJyXGTJwIHOV6LFlzoxUrlttWzAtcznlAh6q+3tGwb8qqO1Z4BaQp\\\\\\\/PLIeJYwRHz9m5zYbIb8iSuaN54FPPHOnkD\\\\\\\/LsHwLLKGzFM3pBXgw0YiAO7+NA2NpEBiXsy0BWpCK6gQhLlY2fLh9G3p\\\\\\\/C6nOmN5sPbcQ0lqpQSX09HhoOI9Zm97qry1Mi5ykMT5l3ad+8k9Yh6Di6OlbEEpc9AGnUIBgjdTwIpcx17Imi0gFEquL+68Z8XolBCyEHPn0UCaS5aCkTHTpsEIwCIMU3z5HRjaCThhjYRFWyZfLCyR9oZ5oO6A4tKa2Mm0xiRAIDeoZne4WqVa\\\\\\\/kIAPaNsk5jo2eoUw5mxtdOKDSsq5V0W0vnl3w\\\\\\\/BqOgY7XJcQ94kZXEQG8zsBZqh9aZ8bipg9oFLu4D30FPG8B330Mq3wggAL+jyyt7mSrmBHVDleCvQEwIaBcq628PGPNNcmNKXzcby01EDvRAMGE4W1ziM\\\\\\\/AESx8RGa0D8INnuCeiGUf1twbECQjfDl+kUP4GNSLUbeEq\\\\\\\/D1ebmhb99Gwyh84wDX8\\\\\\\/bHOQQ56rWYvaeley9Q3bqpB5KaHIyxSqs2LX\\\\\\\/HqliiqETsyRZllPo7wli0D0IkwtwaHJ8pmZjeCza4iydmhB\\\\\\\/cvSEUtDAentqZQHmRvcXdiw1bnNFs\\\\\\\/mMTG146Ys6DoYbJkPkBoUytFNzG46fAJFRoW1ahoIahxx5t4DuhQ+JjlmcwdpLBVjZln75s0ciwt7+dfxTDvsk5eAGpLFZ8JAHs82u+ENc1PJqX8gQFunE38emPNCes9U38XnFwnTRlrLdcZj7CL6B5g1xT5myWjOo0ldRmvKogfLHpEo8Z3yvRVVhIzGq2cYMKftHP11zF8QAZ50yuLR2ei+oNiQB7YH37aeQGj5GB3Pczs\\\\\\\/wBHxEYzWxGfxycgGz9dJYNmY1RVoGMyn9YBOaStAbAaWcw48EUa+x8m2BfwIESK9cw3Sp88I6Vy1VK4oLkwLqSyrG6UPLRGz+RE6UzpGMIpDSG+lFUATkt\\\\\\\/gjB6AgJTwutBqkM6UDG7PUKfdKUMMt8A69PP7\\\\\\\/s4v8W9h8cv\\\\\\\/E3UV3HilgcZbTpaPTtPRG81r06o39yc1FZHdHKakkKGW+BHlT71L9nz8CPkXu301rtnCPkrxmcKGM+5JpXXr1LAvryXGo9O9rjBbGbKqtsd5tqSpui1n0KwBCUqXxRudZqayMPSahgpi\\\\\\\/NcGTyK2TFLhGOT3TwA4LZdQ2hRAWYv7bP8NPTPrGsHxuefdJqIlZuVPaRUEDz+MGXT4WUmZ9ARjiLi4JMAQhv+ydIw2nvOEZN+VA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022w68U7OKuYDXccJrEB1+zQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca4bafbc8267c712852f7d987c2a1ad46906ce1b2b6a43ca0da11f947b643312\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MyvF5UMFLwH8dk1\\\\\\\/WgZcZvn831QMR6tAGK+V7UpCUWquR02\\\\\\\/\\\\\\\/KGgD01Kh7jp+ikW6YLah15jtZdgzHEIcPcLZR0BUNiZm+pVf+A0NXCb9UPyJZjfe0hs+o1Xc4I2jZ3OM2fz3UmFGAQrzsrZH2GRiyojQYUBPun3WayQWdc0FcpzUS5nn2r+54TIycYzrP7CvPOMuiOjf3Y9AzDNxTNRhwjmkKdFk2c8f0\\\\\\\/PHKeBVinXxFyA3vVwlJenBjQpwu3lH3fGLA2m\\\\\\\/An6cSMLKPblZ3sDyKIO3vjyPJ\\\\\\\/xfZXqiDRcKBUIKEQ8V630N24QUHHJQCtccF4QcIrW\\\\\\\/w0kT9WTofzZHEZ+e3caWXr6ltJ8ohYXPHydtiQQWGdlbFFupVg7YudUJZscHZ60ypUiFtA8tGcPydmP6ZH7Mo9KUhGBCTql2fqHPc3DW92dgdDcwa4GF6RuWWcDdWKFCVA+vrjmXNzBpPYUROo+1sVe6x1NqNVomNpB58Y5ap4RBK8nc5auQzNriXISFdHI9GcyrLcjvkjxwlzg6KAPvCEI4dBICJOE2QFTZAr6lKmosWrOiFymq4V1bpfb+KhqV7PPSoPJyPl2qDR9cGXKc1i7hEKfA9lLJ8+MxAw2G+k8Z0kRXt6huiOCr0DxYSF0WpN6PFET5DcC7aAWF60TgBUolR+LGrhvqveltNrfij8CPxIz610P2WbFpyaTLkwz+sYsWrr7wTVVX\\\\\\\/28vJX4l0B2bkVsLj4LCiOIxpZgTeUByZAm2WT6bYZDNsE\\\\\\\/BUA075q\\\\\\\/EcGo8HWShoi\\\\\\\/NIepU1gxu6RGDhVYE4UHfJOcP6dKen2HKirJ3oq2wrTitSzLw9sQ0rOWAn3q+2idaEcht3xSD+9HHUop3bZ9w+xASNEbCiVo8V6dZtg5siWfdGpnMC\\\\\\\/tZjU31sY\\\\\\\/k91j\\\\\\\/ZErONeYpqbBtzsQ3+R0t6u8RAsd9obVrHxaU37E6XThmTS49ecujp17+TTfcSvq+dHrJcfEgqlGJ4HB5nMxRyxQ8cdQJcGO23AjI\\\\\\\/mVFMKzF3gy3XwxcMmk3Dez9OQ+91ZRN0vvu7fuZMCROC5odyOkiMlMGONOXcQSz7\\\\\\\/kBubN2ZZSP9w4+odwySQqdZ1DDUQQTQKpDpqYwsLRbEB8\\\\\\\/jAqZ3GjmxVsidnENP9EqlvotYCS+AT6\\\\\\\/uf9wGH4RgKH0slLmdS\\\\\\\/GyAFLEF1rqUJbc6cXCSHThQVej+cVfZb0psWJblqZdn1ELM2u\\\\\\\/dt43gp5Oxm8PFTc6O5w6+MNAEbtcU7i9Nb6zqwFbdmuxl+j1XD\\\\\\\/1dyk6Pq7dTSC+yy5z17RTTWuS2qgv5Yv8MEh6MYNdr0jRfHMDU1MKUpXPBFde9kbUomgdnhMT22UdeB8qS+vfDvwnzednyftkkbQQKAbz4gea0H8S8CLTVi0XcbEp2W6X77t\\\\\\\/s8BxEt07+T0vyFgE6pCQYPVCTAaEbPu8r1ItCM\\\\\\\/eLOqhzZL301McELjdIWx1j4O14aa0qdg0GmFekxvJACIRiB7V44CzDX+fckuwrgIm8DHti9JpJ5tKlMKaF23N+l8Rv883qBe2fYeYvbtaN4qjyGDXcKtesi0w6oKHbgbxzrbFyjmalr9AyFhJG54DWKoRKSOtvFYri2ygvsDqqutwVcSILZX6Ndt5cs9Nt8BmcZ59J+fiP7PCCayvdGNhrMfi8Rc38ES1oU1+uUhgtocezsYbXNR\\\\\\\/\\\\\\\/E8VZid6wrhw0p2yCLoWm3EdXpu3Dcl7xiQJzzx1Ip0vV14q0wXkGiNccJOLxqbvkC4U6VCteVZVPqhgg7HS9l2OSr2ZYazmILcAK70KLnN8IRfFn8jmJ8UyHKKAjKq37T6q0JjAU9rleH3opjn6HvojQzJhovQQ5SOEYj4ZpsIiV1EWyMxRZqbjH\\\\\\\/JqTvOn9NP+gsBvXCnYnlXjtZbUTUz6mJ1+Gb3DRJqdheXFuKI0VboMxl8tbvL7OUVYQfjPw9Ajdq\\\\\\\/cK8cNnquBCmLm3s2NnPGI1jo2TFJj10zQN1\\\\\\\/g==\\\u0022,\\\u0022iv\\\u0022:\\\u00221wqes5Wx5k38MkhIwcu8VQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002248af5a05064362c60474020a5dbd7b68ef22e19bdcb42aee806ddad53b5e7e1f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459399 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/263" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3uzssvNVCEY6LWYtqKkq6frGAZ-O9N0B8lqMibAPm3Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YUEi3wtp1jDeVq4JSSl75DT_hMIKtxUlm388zUChs-Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/251" title="Origin and Evolution of Dishevelled" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/273" title="Saccharomyces cerevisiae Genetics Predicts Candidate Therapeutic Genetic Interactions at the Mammalian Replication Fork" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459399" id="node459399" data-pisa="ggg;3/2/263" data-pisa-master="ggg;3/2/263" data-apath="/ggg/3/2/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhigang Guo</span>, <span class="highwire-citation-author" data-delta="1">Dominic M. Tucker</span>, <span class="highwire-citation-author" data-delta="2">Daolong Wang</span>, <span class="highwire-citation-author" data-delta="3">Christopher J. Basten</span>, <span class="highwire-citation-author" data-delta="4">Elhan Ersoz</span>, <span class="highwire-citation-author" data-delta="5">William H. Briggs</span>, <span class="highwire-citation-author" data-delta="6">Jianwei Lu</span>, <span class="highwire-citation-author" data-delta="7">Min Li</span> and <span class="highwire-citation-author" data-delta="8">Gilles Gay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-272;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005066 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459399" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhigang Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhigang%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhigang%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhigang.guo@syngenta.com" class="" data-icon-position="" data-hide-link-title="0">zhigang.guo@syngenta.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominic M. Tucker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Syngenta, Inc., Clinton, Illinois 61727</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominic%2BM.%2BTucker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tucker%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominic%2BM.%2BTucker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daolong Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daolong%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaolong%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christopher J. Basten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BBasten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basten%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BBasten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Elhan Ersoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Syngenta, Inc., Stanton, Minnesota 55018</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elhan%2BErsoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ersoz%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElhan%2BErsoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">William H. Briggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Syngenta Seeds B.V., 1601 BK Enkhuizen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BH.%2BBriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Briggs%20WH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BH.%2BBriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jianwei Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianwei%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianwei%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Min Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gilles Gay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilles%2BGay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gay%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilles%2BGay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/2/263" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459399" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459399/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/263.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459399" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459399/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/263.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459399" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459399/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/263.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459399" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459399/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/2/263.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459399" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459399/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Most of previous empirical studies with genome-wide prediction were focused on within-environment prediction based on a single-environment (SE) model. In this study, we evaluated accuracy improvements of across-environment prediction by using genetic and residual covariance across correlated environments. Predictions with a multienvironment (ME) model were evaluated for two corn polygenic leaf structure traits, leaf length and leaf width, based on within-population (WP) and across-population (AP) experiments using a large maize nested association mapping data set consisting of 25 populations of recombinant inbred-lines. To make our study more applicable to plant breeding, two cross-validation schemes were used by evaluating accuracies of (CV1) predicting unobserved phenotypes of untested lines and (CV2) predicting unobserved phenotypes of lines that have been evaluated in some environments but not others. We concluded that (1) genome-wide prediction provided greater prediction accuracies than traditional quantitative trait loci-based prediction in both WP and AP and provided more advantages over quantitative trait loci -based prediction for WP than for AP. (2) Prediction accuracy with ME was significantly greater than that attained by SE in CV1 and CV2, and gains with ME over SE were greater in CV2 than in CV1. These gains were also greater in WP than in AP in both CV1 and CV2. (3) Gains with ME over SE attributed to genetic correlation between environments, with little effect from residual correlation. Impacts of marker density on predictions also were investigated in this study.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/best-linear-unbiased-prediction" class="hw-term hw-article-keyword hw-article-keyword-best-linear-unbiased-prediction" rel="nofollow">best linear unbiased prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/genetic-correlation" class="hw-term hw-article-keyword hw-article-keyword-genetic-correlation" rel="nofollow">genetic correlation</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li><li class="kwd"><a href="/keyword/ridge-regression" class="hw-term hw-article-keyword hw-article-keyword-ridge-regression" rel="nofollow">ridge regression</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li></ul><p id="p-4">Traditional quantitative trait loci (QTL)-based prediction (QP) in marker-assisted selection (MAS) is defined as a two-step process to predict breeding values (BVs) of untested lines for traits of interest based on QTL identified from a typical biparental plant breeding population. In the first step, QTL are identified based on phenotypic and genotypic data via various QTL mapping methods such as stepwise regression (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lande and Thompson 1990</a>), interval mapping (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lander and Botstein 1989</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haley and Knott 1992</a>), composite interval mapping (CIM) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jansen 1993</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zeng 1993</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">1994</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 1999</a>), and inclusive CIM (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2007</a>). In the next step, identified QTL passing a particular significance threshold are included in a single model, and the combined effects of all QTL alleles are estimated simultaneously by either maximum likelihood or multiple linear regression-based methodologies (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edwards and Johnson 1994</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>). These effects are then used in MAS schemes to predict BVs of untested individuals based solely on genotypic data in a subsequent off-season nursery or green house selection (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lorenzana and Bernardo 2009</a>).</p><p id="p-5">The accuracy of QP is contingent on the numbers of QTL identified and their respective estimated effects. Although it is relatively easy to detect QTL with large phenotypic effects, it is difficult to identify QTL with intermediate or small effects (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lander and Botstein 1989</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Xu 2003</a>). This difficulty may be attributed to limited sample sizes of mapping populations and low heritabilities of target traits. Lack of sufficient replication and precision phenotyping also was indicated as possible reasons for low power of QTL mapping. Regardless, the power of QTL discovery determines the prediction accuracy during MAS and is responsible for the concomitant decrease in overall MAS efficiency. It is well known that QTL discovery in biparental mapping populations with a limited number of progeny (&lt; 300) tends to overestimate individual QTL effects, regardless of statistical methodologies for effect assignment (the Beavis effect, see <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Beavis 1994</a>). In severe situations, QTL effects cannot be estimated when genotypes of QTL are highly correlated or when population sizes are too small relative to the total number of putative QTL considered in the model. Therefore, from the perspective of statistical genetics, QP has focused on two key aspects: improving QTL mapping power and increasing the accuracy of QTL effects estimation. Although many statistical approaches have been developed for both purposes, improvement of QP accuracy for quantitative traits controlled by numerous small-effects QTL remains a current problem (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heffner <em>et al.</em> 2009</a>).</p><p id="p-6">Simulation and empirical studies have shown that genome-wide prediction (GWP) (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> 2001</a>) provides improved accuracy in predicting BVs of untested lines over QP by the use of genomic marker trait associations in plants (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Piyasatian <em>et al.</em> 2007</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhong <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zhao <em>et al.</em> 2012</a>), animals (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lee <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Legarra <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2009a</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Luan <em>et al.</em> 2009</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Moser <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rolf <em>et al.</em> 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mujibi <em>et al.</em> 2011</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wolc <em>et al.</em> 2011</a>), and humans (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Makowsky <em>et al.</em> 2011</a>). With GWP, previously unidentified QTL with small effects in QP are captured in generated predictive models. This was shown to lead to significant increases of accuracies of derived predictions. In addition, to reduce the inflation of effect estimation of QTL, effects of each marker are treated as random draws from a common prior density in GWP. In practice, these assumptions of a prior probability distribution are leveraged to limit large fluctuations in marker effect estimates, inducing shrinkage estimates of marker effects. While ridge regression-best linear unbiased prediction (RR-BLUP), BayesA and BayesB approaches originally were developed to improve assessments of QTL effects for GWP strategies (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> 2001</a>), more attention has been directed toward extending and improving these methods to accommodate various types of study populations and to improve predictive ability (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lee <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Meuwissen <em>et al.</em> 2009</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Piepho 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009b</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hayashi and Iwata 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Habier <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ober <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">González-Camacho <em>et al.</em> 2012</a>).</p><p id="p-7">However, the majority of the aforementioned studies apply to a univariate single-environment (SE) prediction model in which phenotypic records or means of a training sample and a validation sample are obtained from the same set of environments by using the same year and location effects, with no consideration of genetic and residual correlation across environments. Recent studies have illustrated that the use of genetic and residual covariance across correlated environments may better the accuracy of across-environment GWP using a multienvironment (ME) model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Calus and Veerkamp 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> 2012</a>). However, it is still not clear what the actual gain in prediction accuracy with ME over SE in typical biparental populations in plant breeding would be and what is the key factor contributing to the gains. Therefore, objectives in this study are threefold: (1) Evaluate the accuracies of across-environment predictions with QP and GWP based on within-population (WP) and across-population (AP) experiments with SE and ME models by using data from a large NAM data set consisting of 25 bi-parental half-sib populations; (2) assess influences of different genetic and residual covariance structures on the prediction accuracy of GWP with different ME models; and (3) conduct an assessment the effects of variation in marker density on GWP with the SE and ME models. To make our study more practical and applicable to plant breeding, two cross-validation schemes were employed to achieve the above objectives by evaluating the accuracies of (CV1) predicting unobserved phenotypes of untested lines and (CV2) predicting unobserved phenotypes of lines that have been evaluated in some environments but not others.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>NAM population</h3><p id="p-8">We retrieved phenotype and genotype data of 4131 recombinant inbred lines (RILs) from maize NAM populations derived from crosses between 25 genetically diverse inbreds and the maize elite parent B73 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) from the Panzea website (<a href="http://www.panzea.org">http://www.panzea.org</a>). A total of 1106 single-nucleotide polymorphism (SNP) markers were used for genotyping each RIL, covering a genetic map of 1439 cM. The coding rules used for genotypic data were identical to those used in the within-environment prediction shown in <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> (2012)</a>. To summarize, for individual populations, genotypes of each SNP of each progeny were coded −1 if both alleles were from a diverse parental line, 1 if from the common parent B73, and 0 otherwise. Based on coding rules, when several SNPs in complete linkage disequilibrium (LD) were available for a chromosomal region, only one SNP was selected. In each NAM population, selected SNPs varied from 785 to 895 whereas the genome coverage ranged from 1371 to 1397 cM. In the current study, we used phenotypic data of each RIL evaluated for leaf structure traits leaf length (LL) and leaf width (LW) from two locations, Aurora (AU), NY, and Champaign-Urbana (CU), IL, over 2 yr (2006 and 2007). A total of four environments were then defined as E1: AU at 2006; E2: CU at 2006; E3: AU at 2007; and E4: CU at 2007. Both LL and LW traits were found to have a genetic architecture affected by a large number of QTL of small effect (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tian <em>et al.</em> 2011</a>). Based on phenotypic information from 2 yr and two locations, the broad sense heritability <em>H</em><sup>2</sup> on a mean basis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/005066SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/FileS1.pdf">File S1</a>) ranged from 0.60 to 0.84 for the LL trait and from 0.60 to 0.82 for the LW trait (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS1.pdf">Table S1</a>). Over 25 NAM populations, the average of <em>H</em><sup>2</sup> was 0.74 for both traits, reflecting a high precision of phenotyping efforts.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462356/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462356/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Population information for the 25 NAM populations used for analysis in the current study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Models for estimation of marker effects</h3><p id="p-10">Two methods, CIM and RR-BLUP, were used to estimate marker effects based on genotypic and phenotypic data in a training data set in CV1 with balanced phenotypic records and CV2 with unbalanced phenotypic records which were discussed in details in next section. Prediction accuracies with CIM and RR-BLUP were used to measure estimates with QP and GWP strategies, respectively. Two models, SE and ME, were applied in both methods based on training sets obtained from WP and AP. In total, there were four models developed for each method: AP-ME, AP-SE, WP-ME, and WP-SE. In the discussion of each method, we start with the most complex model, AP-ME, and then simplify to obtain others.</p><p id="p-11">The AP-ME model with CIM was extended from multi-trait QTL mapping (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jiang and Zeng 1995</a>) to accommodate multipopulation analysis, which can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <strong>Y</strong> = [<strong>y</strong><sub>1</sub>, <strong>y</strong><sub>2</sub>, Δ, <strong>y</strong><em><sub>n</sub></em>]<sup>T</sup> is an <em>n</em> × <em>m</em> matrix of phenotypic data for <em>n</em> lines from <em>p</em> NAM populations and <em>m</em> environments, and <strong>y</strong><sub>1</sub>, <strong>y</strong><sub>2</sub>,Δ,<strong>y</strong><em><sub>n</sub></em> are vectors composed of observations across <em>m</em> environments; <strong>X</strong> is an <em>n</em> × (1 + <em>p</em>) incidence matrix for <em>n</em> lines from <em>p</em> populations representing population structure with the first column ones; <strong>β</strong> is a (1 + <em>p</em>) × <em>m</em> matrix of individual population effects with the first column representing the overall mean; <strong>W</strong> is an <em>n</em> × <em>f</em> matrix of genotypes of cofactor markers; <strong>s</strong> is an <em>f</em> × <em>m</em> matrix of cofactor marker effects with <em>f</em> the number of cofactor markers; <strong>Q</strong> is an <em>n</em> × 1 matrix of QTL genotypes; <strong>α</strong> is an 1 × <em>m</em> matrix of additive effects of a putative QTL at a tested position; and <strong>ε =</strong> [<strong>ε</strong><sub>1</sub>, <strong>ε</strong><sub>2</sub>, <strong>Δ, ε</strong><sub>n</sub>]<sup>T</sup> is a matrix of residuals <strong>ε</strong><em><sub>i</sub></em> (<em>i</em> = 1, 2, Δ, <em>n</em>), which is assumed to be correlated between environments and to follow a multivariate normal distribution with means zero and covariance matrix<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-2.gif"/></noscript></span></span>In this model, the population effects <strong>β</strong>, cofactor effects <strong>s</strong>, and QTL additive effects <strong>α</strong> were treated as fixed effects and residual <strong>ε</strong> as random effects. QTL genotype <strong>Q</strong> was not observed and was replaced with its expected value obtained from the probability distribution of QTL genotypes conditional on the closest flanking markers (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Haley and Knott 1992</a>). Cofactor markers were selected by a modified stepwise selection procedure (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buckler <em>et al.</em> 2009</a>). Extensions of this model to others were straightforward. For WP-ME, model (1) with population structure excluded from <strong>X</strong> became a single-population ME analysis (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jiang and Zeng 1995</a>). AP-SE was similar to model (1) but was reduced to a univariate regression model for SE analysis, which was further simplified into a WP-SE model without population structure (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zeng 1993</a>, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">1994</a>). To deal with unbalanced phenotypic records in a training data set, the modified CIM method developed by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guo and Nelson (2008)</a> was applied to WP-ME and AP-ME in CV2 by iteratively imputing missing phenotypic data conditional on their posterior distributions.</p><p id="p-12">On the basis of the aforementioned models, CIM was performed by scanning whole genomes with a fixed step size of 1 cM based on genotypic and phenotypic data in a training data set at each replicate of CV1 and CV2. A QTL was identified at the position in which the test statistic logarithm of odds (LODs) score assumed its maximum in the region under consideration with a LOD threshold (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Utz <em>et al.</em> 2000</a>). Cofactor selection in CIM requires a significance level that we estimated to be 0.0001 based on permutation tests for AP-SE and AP-ME (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Tian <em>et al.</em> 2011</a>) and relaxed to 0.01 with WP-SE and WP-ME due to the decreased training sample size. Another key requirement was to determine LOD thresholds for identifying QTL. In the current study, LOD thresholds were estimated from 5000 permutation tests based on a genome-wide significance level 0.05 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doerge and Churchill 1996</a>). In WP-SE and AP-SE, phenotypic records for each RIL were randomized within each NAM population, whereas in WP-ME and AP-ME, permutation tests were performed by shuffling phenotypic records for multiple environments at once to preserve their correlation structure. Note that permutation tests were conducted for each model in CV1 and CV2 based on full data sets from each of the 25 NAM populations. Once QTL were identified using empirical LOD thresholds, effects of QTL were estimated by a multivariate multiple regression model with population structure.</p><p id="p-13">The AP-ME model with RR-BLUP was an extension from the model proposed by <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> (2012)</a> as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-3.gif"/></noscript></span></span>where <strong>y</strong><em><sub>i</sub></em> (<em>i</em> = 1, 2,Δ, <em>m</em>) is a vector of phenotypic data for <em>n</em> lines from <em>p</em> NAM populations at the environment <em>i</em>; <em>m</em> is the total number of environments; <strong>X</strong><em><sub>i</sub></em> is an <em>n</em> × (1 + <em>p</em>) incidence matrix for <em>n</em> lines and <em>p</em> populations representing the population structure with the first column ones; <strong>β</strong><em><sub>i</sub></em> is a vector of individual population effects (including the overall mean); <strong>Z</strong><em><sub>i</sub></em> is an <em>n</em> × <em>k</em> matrix of marker genotypes with <em>k</em> the total number of markers; <strong>α</strong><em><sub>i</sub></em> is a vector of marker effects; and <strong>ε</strong><em><sub>i</sub></em> is a vector of residuals. The above model can be rewritten in reduced matrix form as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <strong>Y</strong> = [<strong>y</strong><sub>1</sub><sup>T</sup>, <strong>y</strong><sub>2</sub><sup>T</sup>, Δ, <strong>y</strong><em><sub>m</sub></em><sup>T</sup>]<sup>T</sup>; <strong>β</strong> = [<strong>β</strong><sub>1</sub><sup>T</sup>, <strong>β</strong><sub>2</sub><sup>T</sup>, Δ, <strong>β</strong><em><sub>m</sub></em><sup>T</sup>]<sup>T</sup>; <strong>α</strong> = [<strong>α</strong><sub>1</sub><sup>T</sup>, <strong>α</strong><sub>2</sub><sup>T</sup>, Δ, <strong>α</strong><em><sub>m</sub></em><sup>T</sup>]<sup>T</sup>; and <strong>ε</strong><sup>T</sup> = [<strong>ε</strong><sub>1</sub><sup>T</sup>, <strong>ε</strong><sub>2</sub><sup>T</sup>, Δ, <strong>ε</strong><em><sub>m</sub></em><sup>T</sup>]<sup>T</sup>. In this model, the population effects <strong>β</strong> were treated as fixed effects. The marker effects <strong>α</strong> were treated as random effects following a multivariate normal distribution <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-5.gif"/></noscript></span></span>with <strong>G</strong><sub>0</sub> the additive genetic covariance matrix between <strong>α</strong> and <strong>α</strong>′, <strong>I</strong><em><sub>K</sub></em> a <em>k</em> × <em>k</em> identity matrix, and ⊗ the Kronecker product of matrices. <strong>G</strong><sub>0</sub> can be defined as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-6.gif"/></noscript></span></span>where <em>σ<sub>ii</sub></em><sup>2</sup> is the genetic variance of lines in environment <em>i</em> (<em>i</em> = 1, 2, Δ, <em>m</em>), <em>ρ<sub>ij</sub></em> (<em>i</em> = 1, 2, Δ, <em>m</em>, and <em>j</em> = 1, 2, Δ, <em>m</em>) is the genetic correlation of lines between environment <em>i</em> and <em>j</em>; and <em>ρ<sub>ij</sub></em> = <em>ρ<sub>ji</sub></em>. The genetic covariance matrix of <strong>Y</strong> can be defined as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-7.gif"/></noscript></span></span>where <strong>A</strong> is an <em>n</em> × <em>n</em> genetic relationship matrix, which can be calculated using genome-wide markers as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-8.gif"/></noscript></span></span>where <strong>M</strong> is an <em>n</em> × <em>k</em> matrix of marker genotypes and <em>p<sub>i</sub></em> is the frequency of an allele at locus <em>i</em> (<em>i</em> = 1, 2, Δ, <em>k</em>) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">VanRaden 2008</a>). Note that the matrix <strong>M</strong> is equivalent to <strong>Z</strong><em><sub>i</sub></em> (<em>i</em> = 1, 2, Δ, <em>m</em>) in the CV1 scheme. The residuals <strong>ε</strong> were considered as random effects following a multivariate normal distribution <em>N</em>(0, <strong>R</strong>) where <strong>R</strong> is a residual covariance matrix calculated by<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-9.gif"/></noscript></span></span>where <strong>I</strong><em><sub>N</sub></em> is an <em>n</em> × <em>n</em> identity matrix, and <strong>R</strong><sub>0</sub> is an <em>m</em> × <em>m</em> matrix of the residuals in different environments, written as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-10.gif"/></noscript></span></span>where <em>e<sub>ii</sub></em><sup>2</sup> is the residual variance in environment <em>i</em> (<em>i</em> = 1, 2, Δ, <em>m</em>), <em>r<sub>ij</sub></em> (<em>i</em> = 1, 2, Δ, <em>m</em>; <em>j</em> = 1, 2, Δ, <em>m</em>) is the residual correlation of lines between environment <em>i</em> and <em>j</em>, and <em>r<sub>ij</sub></em> = <em>r<sub>ji</sub></em>. When model (2) was modified into AP-SE, it was reduced to a univariate model, and <strong>G</strong><sub>0</sub> and <strong>R</strong><sub>0</sub> became a special case of the multivariate model with diagonal entries denoting the genetic and residual variances, respectively. In WP-SE and WP-ME, the population structure <strong>X</strong> was reduced to the column vector of ones. These models were applied to estimate marker effects based on training data sets with balanced phenotypic records in CV1. For training sets with unbalanced phenotypic records in CV2, marker genotype matrix <strong>Z</strong><em><sub>i</sub></em> in environment <em>i</em> may not be equal to <strong>Z</strong><em><sub>j</sub></em> in environment <em>j</em>. The genetic covariance between line <em>l</em> and <em>q</em> across environment <em>i</em> and <em>j</em> in <strong>G</strong> was modified into <strong>G</strong><em><sub>ijlq</sub></em> = {<strong>Z</strong><em><sub>il</sub></em><strong>Z</strong><em><sub>jq</sub><sup>T</sup></em>σ<em><sub>ij</sub></em><sup>2</sup>/2∑<em>p<sub>h</sub></em>(1 - <em>p<sub>h</sub></em>)} where <strong>Z</strong><em><sub>il</sub></em> is the genome-wide marker genotype row vector for line <em>l</em> in environment <em>i</em>, <strong>Z</strong><em><sub>jq</sub></em> is the genome-wide marker genotype row vector for line <em>q</em> in environment <em>j</em>, <em>σ<sub>ij</sub></em><sup>2</sup> is the entry of <strong>G</strong><sub>0</sub> at row <em>i</em> and column <em>j</em>, and <em>p<sub>h</sub></em> is the frequency of an allele at locus <em>h</em> (<em>h</em> = 1, 2, Δ, <em>k</em>). The residual covariance was <strong>R</strong><em><sub>ijlq</sub></em> = {<em>e<sub>ij</sub></em><sup>2</sup>} with <em>e<sub>ij</sub></em><sup>2</sup> the element at the <em>i</em>th row and the <em>j</em>th column of <strong>R</strong><sub>0</sub> if line <em>l</em> in environment <em>i</em> and line <em>q</em> in environment <em>j</em> are from the same genotype, and zero otherwise,</p><p id="p-14">A key requirement in model (2) was to estimate the covariance matrix <strong>G</strong><sub>0</sub> and <strong>R</strong><sub>0.</sub> In this study, <strong>G</strong><sub>0</sub> and <strong>R</strong><sub>0</sub> were estimated by the multivariate restricted maximum likelihood (REML) approach proposed by <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vattikuti <em>et al.</em> (2012)</a> based on model (2). Because estimates of variance components from multivariate REML were sensitive to initial values, estimates from univariate and bivariate REML were used as the initial values for the multivariate REML analysis. Although <strong>G</strong><sub>0</sub> and <strong>R</strong><sub>0</sub> were estimated in each NAM population for WP, they were estimated once by combining full data sets from all NAM populations for AP. Once estimation for <strong>G</strong><sub>0</sub> and <strong>R</strong><sub>0</sub> were done, environment-specific marker effects <strong>α</strong> along with the population effects <strong>β</strong> were estimated by solving Henderson’s mixed-model equations (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Henderson 1984</a>).</p></div><div id="sec-4" class="subsection"><h3>Cross-validation</h3><p id="p-15">We used validation methods to estimate the prediction accuracy of each model described previoulsy. Cross-validation is a statistical technique of splitting data into training and validation sets by using the validation set to evaluate prediction ability of the trained model. Two cross-validation schemes, CV1 and CV2 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), were considered in the study based on different prediction purposes relevant to practical breeding problems (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Calus and Veerkamp 2011</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> 2012</a>). In CV1, the goal was to predict unobserved phenotypes of untested lines or newly generated lines based solely on genotypes; and in CV2, missing phenotypes of lines in an environment were predicted with genotypes and phenotypes from other environments. A total of 100 replicates of cross-validation were performed for each scheme, and the validation results were averaged over each replicate.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/263/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of one replicate of cross-validation in CV1 and CV2. White boxes represent observed phenotypic records, and black ones represent missing phenotypic records." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1215232539" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of one replicate of cross-validation in CV1 and CV2. White boxes represent observed phenotypic records, and black ones represent missing phenotypic records.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/263/F1.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/263/F1.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/263/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/263/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462377" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-16" class="first-child">Example of one replicate of cross-validation in CV1 and CV2. White boxes represent observed phenotypic records, and black ones represent missing phenotypic records.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">In each replicate of CV1 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), RILs in each population were randomly split into a training data set comprising 60% of the samples and a validation data set of the remainder (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Legarra <em>et al.</em> 2008</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>). Phenotypic records of each line in the validation set were set to missing for all environments. In CV2 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), a specific proportion (40%) of phenotypic records for each environment independently were set to missing at random. Therefore, phenotypic information was balanced for training sets generated in CV1, and unbalanced in CV2. Lines lacking phenotypic data for all environments were dropped in CV2. Theoretically, the proportion of these lines in each NAM population was (0.40)<sup>4</sup> = 2.56% across four environments. Excluding these lines had little effect on CV2 analysis.</p><p id="p-18">In both CV1 and CV2 described above, training and validation sets were from a single population by the use of WP information. As an alternative practice, AP information can be used for prediction in the absence of WP information (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Legarra <em>et al.</em> 2008</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Zhao <em>et al.</em> 2012</a>), mimicking a framework of using historical data to predict line performances of a cross. In CV1, the AP training set was generated by combining all RILs from the other 24 NAM populations. In CV2, the AP training set was generated with RILs of 25 NAM populations including the test population. In this case, each line in the training set contained unbalanced phenotypic records. It is worthy of mentioning that both WP and AP were used to predict performances of lines from the identical validation set in CV1 and CV2, respectively.</p><p id="p-19">Although training was performed with WP and AP approaches in CV1 and CV2, validation was conducted within each NAM population for each environment. In each replicate, prediction accuracies were measured as the correlation coefficient between estimated BVs and observed phenotypes of lines in the validation set using<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-11.gif"/></noscript></span></span>where <em>ŷ<sub>i</sub></em> is the estimated estimated BV of individual <em>i</em> in the validation sample; <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2/263/embed/mml-math-13.gif"/></noscript></span></span> are environment-specific overall mean and marker effects estimated from a training sample using the methods discussed in the last section; and <strong>z</strong><em><sub>ij</sub></em> is the genotype of marker <em>j</em> for line <em>i</em> in the validation set. Note that only QTL were used in the model with the QP strategy, while all the markers were included for the GWP strategy. The final reported prediction accuracy was in fact the mean of the 100 predictions generated across the replicate runs. Overall accuracies between various models were compared with a pairwise <em>t</em>-test (<em>α</em> = 0.05) based on identical validation sets within each NAM population. Gains in prediction accuracy with one model (<em>e.g.</em>, model <em>A</em>) over another one (<em>e.g.</em>, model <em>B</em>) were calculated using (<em>R<sub>A</sub></em> – <em>R<sub>B</sub></em>) / <em>R<sub>B</sub></em>, where <em>R<sub>A</sub></em> represents prediction accuracy with model <em>A</em> and <em>R<sub>B</sub></em> prediction accuracy with model <em>B</em>.</p></div><div id="sec-5" class="subsection"><h3>Impacts of genetic and residual covariance structure on GWP</h3><p id="p-20">Impacts of genetic and residual covariance on GWP with ME were assessed by comparing four different models: (1) structured genetic and residual covariance (SG-SR); (2) structured genetic and unstructured residual variance (SG-UR); (3) unstructured genetic and structured residual variance (UG-SR); and (4) unstructured genetic and residual covariance (UG-UR). SG-SR was the simplest model among them, in which each of the nondiagonal entries in <strong>G</strong><sub>0</sub> and <strong>R</strong><sub>0</sub> estimated above were set to zero, imposing independence on genetic and residual correlations. This model was equivalent to fitting each SE model separately with GWP. SG-UR imposed independence on genetic correlation by setting non-diagonal entries to zeros in <strong>G</strong><sub>0</sub>, while UG-SR removed residual correlation by setting non-diagonal entries to zeros in <strong>R</strong><sub>0</sub>. UG-UR was completely unstructured and estimated in the previous section. Influences of residual covariance were evaluated by comparing SG-SR with SG-UR, whereas impacts of genetic covariance were assessed by comparing UG-SR with SG-SR. Finally, effects of residual covariance given the genetic covariance were evaluated by comparing UG-UR with UG-SR.</p></div><div id="sec-6" class="subsection"><h3>Effect of marker density on prediction accuracy of GWP</h3><p id="p-21">Effects of marker densities on GWP were tested by setting a conditional genetic distance criterion (<em>c</em>) between two flanking markers to ensure that each chromosome was evenly covered by a set of SNPs. For each NAM population, varying numbers of markers were obtained by selecting values of <em>c</em> (<em>c</em> = 1.6, 5, 10, 15, 20, 25, and 30 cM) without compromising the overall genomic coverage using the method described by <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> (2012)</a>. This was done to observe the influence of using smaller marker sets than all available data from the original NAM study. All the identified polymorphic markers were included in the model at <em>c</em> = 1.6 cM, meaning that the mean marker distance of the entire marker set was ~1.6 cM. Note that a training sample proportion 0.6 was always used when assessing the effect of different marker densities to ensure different densities were applied to the same training and validation sets at each replicate of 100 cross-validations.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-22">Accuracies of predictions for traits LL and LW are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Prediction accuracy in each cell in this table was the average of prediction accuracies over 25 NAM populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS9.pdf">Table S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS10.pdf">Table S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS11.pdf">Table S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS12.pdf">Table S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS13.pdf">Table S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS14.pdf">Table S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS15.pdf">Table S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS16.pdf">Table S16</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS17.pdf">Table S17</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS18.pdf">Table S18</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS19.pdf">Table S19</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS20.pdf">Table S20</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS21.pdf">Table S21</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS22.pdf">Table S22</a>). As discussed previously, QTL for QP were identified using the empirical LOD thresholds estimated by permutation tests (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS2.pdf">Table S2</a>). With GWP, the genetic and residual covariances obtained from multivariate REML for traits LL and LW (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS5.pdf">Table S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS6.pdf">Table S6</a>) were used to estimate environment-specific marker effects with RR-BLUP. We first compared GWP with QP in CV1 and CV2. Overall, GWP gave consistently greater accuracies than QP. Increase in accuracy with GWP over QP was statistically significant in 99% of predictions generated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS9.pdf">Table S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS10.pdf">Table S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS11.pdf">Table S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS12.pdf">Table S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS13.pdf">Table S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS14.pdf">Table S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS15.pdf">Table S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS16.pdf">Table S16</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS17.pdf">Table S17</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS18.pdf">Table S18</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS19.pdf">Table S19</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS20.pdf">Table S20</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS21.pdf">Table S21</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS22.pdf">Table S22</a>). Gains in accuracies with GWP over QP were larger for WP than for AP were attributed to two factors: (1) GWP with WP gave greater accuracies than that with AP; and (2) QP with WP gave lower accuracies than that with AP due to a decreased number of QTL identified in WP. Although AP gave improved accuracies for QP over WP, it still yielded lower predictive power than GWP, which was also lower than the accuracies with GWP in WP.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462426/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462426/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Prediction accuracy with QP and GWP using SE and ME models in CV1 and CV2 for traits LL and LW based on 25 NAM population</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">Afterward we compared prediction accuracies between the SE and ME models for GWP. The ME model provided greater prediction accuracies than the SM model in both CV1 and CV2. Increases in accuracies were significant in most cases (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS9.pdf">Table S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS10.pdf">Table S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS11.pdf">Table S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS12.pdf">Table S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS13.pdf">Table S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS14.pdf">Table S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS15.pdf">Table S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS16.pdf">Table S16</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS17.pdf">Table S17</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS18.pdf">Table S18</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS19.pdf">Table S19</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS20.pdf">Table S20</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS21.pdf">Table S21</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS22.pdf">Table S22</a>). For WP in CV1, across all populations and environments, average gains in accuracies with ME over SE were 8% and 12% for traits LL and LW, respectively. Gains attained a maximum of 48% in some populations in CV1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS7.pdf">Table S7</a>). For WP in CV2, average increases with ME over SE reached 40% and 33% for LL and LW traits, respectively. For AP, gains with ME over SE were reduced to 3% in CV1 and 10–13% in CV2 for LL and LW traits. On average, across all populations, environments, and traits, gains with ME over SE were 10% and 37% for WP, and 3% and 11.5% for AP in CV1 and CV2, respectively. The aforementioned gains were observed based on the average genetic correlation of 0.77 and 0.87 and the average residual correlation of 0.31 and 0.39 for WP and AP, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS5.pdf">Table S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS6.pdf">Table S6</a>). With different training and validation sets, it may not be appropriate to evaluate performances of GWP between CV1 and CV2 on the same ground in this study. However, given similar accuracies with the SE model obtained from CV1 and CV2, comparison between them suggested, but did not prove, that gains with ME over SE were greater in CV2 than that in CV1 at WP and AP.</p><p id="p-29">Prediction accuracies for GWP using ME with different genetic and residual variance structures are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Similar to <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, each of the accuracies shown in this table was the average of prediction accuracies over 25 NAM populations. Detailed results can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS23.pdf">Table S23</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS24.pdf">Table S24</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS25.pdf">Table S25</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS26.pdf">Table S26</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS27.pdf">Table S27</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS28.pdf">Table S28</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS29.pdf">Table S29</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS30.pdf">Table S30</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS31.pdf">Table S31</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS32.pdf">Table S32</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS33.pdf">Table S33</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS34.pdf">Table S34</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS35.pdf">Table S35</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS36.pdf">Table S36</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS37.pdf">Table S37</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1/TableS38.pdf">Table S38</a>. In CV1 and CV2, SG-UR gave prediction accuracies comparable to or less than that attained by SG-SR, suggesting that addition of residual covariance did not benefit predictions, and may have worsened them. In contrast, prediction accuracies with UG-SR were greater than that with SG-SR, suggesting that modeling genetic covariance improved predictions with the ME model. Finally, UG-UR gave comparable prediction accuracies to UG-SR, suggesting that adding residual covariance had little effect on predictions in the presence of the genetic covariance.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462293/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462293/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Prediction accuracies with ME GWP based on SG-SR, SG-UR, UG-SR, and UG-UR for traits LL and LW based on 25 NAM populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">Marker densities of 1.6, 5, 10, 15, 20, 25, and 30 cM were used corresponding to different numbers of markers in each training sample via methods described in the previous section. As expected, prediction accuracies for GWP with SE and ME models improved with increasing marker density in the example using the NAM population B73×CML322 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). These improvements tended to diminish to zero when marker density exceeded a threshold of 10 cM, suggesting that the density selected based on an interval size of 10 cM was sufficient to capture LD between QTL and markers in these bi-parental segregating populations, similar to reports by <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> (2012)</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/263/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy in environment E1 for the trait LL using SE and ME models in GWP with different levels of marker densities. The training sample proportion is 0.6 in NAM population B73&#xD7;CML322. (A) CV1; (B) CV2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1215232539" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy in environment E1 for the trait LL using SE and ME models in GWP with different levels of marker densities. The training sample proportion is 0.6 in NAM population B73&#xD7;CML322. (A) CV1; (B) CV2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/263/F2.medium.gif" width="251" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/263/F2.medium.gif" width="251" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/263/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/263/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462306" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-35" class="first-child">Prediction accuracy in environment E1 for the trait LL using SE and ME models in GWP with different levels of marker densities. The training sample proportion is 0.6 in NAM population B73×CML322. (A) CV1; (B) CV2.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-36">GWP provides a significant predictive advantage over QP in WP and AP, with increased gains in WP compared with AP. For almost all cases in WP, GWP shows significantly greater accuracy than QP in each of the NAM populations. This conclusion is consistent with the results from previous studies (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Zhao <em>et al.</em> 2012</a>). More interestingly, when training sets are changed from WP to AP, estimations with GWP are reduced, similar to previous reports in mice (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Legarra <em>et al.</em> 2008</a>) and maize (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Zhao <em>et al.</em> 2012</a>). The loss of accuracy from WP to AP for GWP may attribute to the differences of genetic backgrounds between training and validation populations. The differences may also affect QP and could reduce prediction accuracy from WP to AP for QP. However, in this study, we found a gain in accuracy with AP over WP. This could be explained by the different genetic basis used for prediction between GWP and QP. Although GWP relies on genetic relationship derived from genome-wide markers, QP uses a small subset of markers and focuses more on LD between these markers and QTL. As a result, QP may be less affected by different genetic backgrounds between training and validation sets than GWP. The potential loss caused by the different genetic backgrounds in QP could be compensated by the gain obtained from AP by improved QTL mapping power by utilizing multiple populations (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yi and Xu 2002</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blanc <em>et al.</em> 2006</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Tian <em>et al.</em> 2011</a>). However, even with increased prediction accuracy with AP, QP still cannot achieve the high prediction accuracies attained by GWP with WP. This finding suggests, in practical breeding, to achieve high prediction accuracies, even with a high cost, it is still worthwhile to phenotype a proportion of lines from a breeding population to predict performances of genotypes generated from the subsequent intercross or backcross generations (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>).</p><p id="p-37">The use of across-environment information improves prediction with GWP. Two main results were obtained from this study. First, modeling covariances between correlated environments with the ME model gives better predictions compared with SE in both the CV1 and CV2 schemes. At an average genetic correlation of 0.77, gains of 10% in CV1 and 37% in CV2 were observed for WP, greater than that of 3% in CV1 and 11.5% in CV2 for AP at a correlation of 0.87. Although the superiority of ME over SE in CV2 can be explained by borrowing information from the same lines across environments (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> 2012</a>), gains in CV1 may likely attribute to more accurate estimates of environment-specific marker effects by utilizing genetic correlation. Gains in CV1 and CV2 also were reported based on a simulation study in animal breeding with the multitrait model, similar to the ME model tested in this study. This gave increases in accuracies of 0.03 to 0.14 over the single-trait prediction model that is similar to the SE model at genetic correlation levels of 0.25 and 0.75 (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Calus and Veerkamp 2011</a>). Furthermore, we found that the gain in CV2 with ME over SE is greater than that in CV1. This conclusion is also consistent with previously reported results (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Calus and Veerkamp 2011</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> 2012</a>). Second, by modeling and comparing different genetic and residual covariance structures, we found that gains with ME over SE are attributed to genetic covariance in CV1 and CV2, with little or negative contribution from the residual covariance. This finding indicates that accurately estimating the genetic covariance structure between correlated environments is critical to improve prediction accuracy with ME models. Although these results were obtained with relatively simple covariance structures from only four environments, more studies need to be conducted to confirm if these gains reflect a general superiority of the ME model with a large number of environments.</p><p id="p-38">Impacts of marker density also were evaluated for GWP with the SE and ME models. Overall, our conclusions were consistent with previous work from within-environment predictions (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Zhao <em>et al.</em> 2012</a>). Results indicate that a marker density of 10 cM, approximately corresponding to 150 markers, should be used when designing a predictive breeding strategy with GWP in a biparental breeding population. Although this result is consistent with findings in the previous studies in plant breeding (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Guo <em>et al.</em> 2012</a>), it has to be kept in mind that conclusions were drawn based on the typical biparental segregating populations with extensive LD caused by limited recombination. In the context of animal and human GWP where the populations studied are generally composed of complicated pedigreed individuals, a greater marker density is required to capture the small LD caused by accumulated historical recombinations. It was reported that at least 80,000 SNPs in human (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Makowsky <em>et al.</em> 2011</a>) and 5000 to 7500 SNPs in animals (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vazquez <em>et al.</em> 2010</a>) are needed to reach a high prediction accuracy with GWP. Therefore, marker densities may be determined mainly by LD structure in different types of breeding populations. Also the goals and resources of a breeding project may affect decisions.</p><p id="p-39">In practical plant breeding, GWP needs to be integrated into the appropriate genome-wide selection schemes where several cycles of intercrossing may be required to increase genetic gains (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jannink <em>et al.</em> 2010</a>). The CV1 scheme discussed in this study will mainly serve this purpose, and BVs of newly generated genotypes will be predicted and used for selection. Though there is no need for QTL identification in GWP, this does not suggest that the QTL ideotype construction strategy deployed in marker-assisted recurrent selection scheme cannot be implemented via GWP (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nakaya and Isobe 2012</a>). At its simplest, the construction of QTL ideotypes may be accounted for by BVs obtained from GWP. For example, these BVs can be used in the QTL introgression process to efficiently pyramid 3 to 30 QTL as demonstrated by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo (2009)</a>. In contrast, the CV2 scheme investigated in the current paper may be used to predict missing phenotypes caused by random missingness for a same set of genotypes evaluated across multiple environments. Overall, when the focus of a complex trait study is migrated from traditional line performances to allelic effect evaluation (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> 2011</a>), prediction of BVs with GWP may be further improved in CV1 and CV2 schemes. To maximize genetic gains in varied breeding programs per unit time and cost, it will be critical to design different genome-wide selection strategies to fully take advantage of the high accuracy of BVs at key stages of marker-assisted breeding.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors thank researchers and institutions who contributed to the Panzea database (<a href="http://www.panzea.org/">http://www.panzea.org/</a>) and also thank the editor and two anonymous reviewers for valuable points to improve this work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005066/-/DC1</a>.</p></li><li class="fn" id="fn-12"><p id="p-41"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-08-18"><span class="received-label">Received </span>August 18, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-12-09"><span class="accepted-label">Accepted </span>December 9, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Guo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.263.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">QTL analysis: power, precision and accuracy</span>, pp. <span class="cit-fpage">145</span>–<span class="cit-lpage">162</span> in <span class="cit-source">Molecular Dissection of Complex Traits</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular markers and selection for complex traits in plants: learning from the last 20 years</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D48%26rft.spage%253D1649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.3" data-doi="10.2135/cropsci2008.08.0452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomewide selection for rapid introgression of exotic germplasm in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D419%26rft.epage%253D425%26rft.atitle%253DGenomewide%2BSelection%2Bfor%2BRapid%2BIntrogression%2Bof%2BExotic%2BGermplasm%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264654000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.4" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.5" data-doi="10.1007/s00122-006-0287-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBlanc%26rft.auinit1%253DG.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D206%26rft.epage%253D224%26rft.atitle%253DConnected%2Bpopulations%2Bfor%2Bdetecting%2Bquantitative%2Btrait%2Bloci%2Band%2Btesting%2Bfor%2Bepistasis%253A%2Ban%2Bapplication%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0287-1%26rft_id%253Dinfo%253Apmid%252F16791688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0287-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16791688&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238503200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.6" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.7" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.8" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.9" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predictions of genetic values of quantitative traits in plant breeding using pedigree and molecular markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D713%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.118521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20813882&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282807400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D375%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.101501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19293140&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.11" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R.W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Permutation tests for multiple loci affecting a quantitative character</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D142%26rft.spage%253D285%26rft_id%253Dinfo%253Apmid%252F8770605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8770605&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TM92400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.2.263.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">RFLPs for rapid recurrent selection</span>, pp. <span class="cit-fpage">33</span>–<span class="cit-lpage">40</span> in <span class="cit-source">Analysis of Molecular Marker Data</span>. <span class="cit-comment">Joint Plant Breeding Symposia Series</span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.14" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple-trait quantitative trait locus mapping with incomplete phenotypic data</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D9%26rft.spage%253D82%26rft.epage%253D82%26rft.atitle%253DMultiple-trait%2Bquantitative%2Btrait%2Blocus%2Bmapping%2Bwith%2Bincomplete%2Bphenotypic%2Bdata.%26rft_id%253Dinfo%253Apmid%252F19061502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19061502&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.16" data-doi="10.1007/s00122-011-1702-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishore</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genome-wide selection efficiency in maize nested association mapping populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D261%26rft.epage%253D275%26rft.atitle%253DEvaluation%2Bof%2Bgenome-wide%2Bselection%2Befficiency%2Bin%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1702-9%26rft_id%253Dinfo%253Apmid%252F21938474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1702-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21938474&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.17" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.18" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EM algorithm for Bayesian estimation of genomic breeding values</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPoletto%26rft.auinit1%253DA.%2BB.%26rft.volume%253D11%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DThe%2BB%2Bchromosomes%2Bof%2Bthe%2BAfrican%2Bcichlid%2Bfish%2BHaplochromis%2Bobliquidens%2Bharbour%2B18S%2BrRNA%2Bgene%2Bcopies.%26rft_id%253Dinfo%253Apmid%252F20051104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20051104&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.20" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Invited review: genomic selection in dairy cattle: progress and challenges</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.21" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.22" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.23" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy for grain quality traits in biparental wheat populations</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.263.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>, <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph, ON</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.25" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice</span>. <abbr class="cit-jnl-abbrev">Brief Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMTY2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Interval mapping of multiple quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D135%26rft.spage%253D205%26rft_id%253Dinfo%253Apmid%252F8224820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8224820&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LW03300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Multiple trait analysis of genetic mapping for quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D140%26rft.spage%253D1111%26rft_id%253Dinfo%253Apmid%252F7672582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7672582&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RE86800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Efficiency of marker-assisted selection in the improvement of quantitative traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D124%26rft.spage%253D743%26rft_id%253Dinfo%253Apmid%252F1968875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1968875&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CQ97500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D121%26rft.spage%253D185%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2563713&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989R676700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.30" data-doi="10.1371/journal.pgen.1000231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Predicting unobserved phenotypes for complex traits from whole-genome SNP data</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BH.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000231%26rft.epage%253De1000231%26rft.atitle%253DPredicting%2Bunobserved%2Bphenotypes%2Bfor%2Bcomplex%2Btraits%2Bfrom%2Bwhole-genome%2BSNP%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000231%26rft_id%253Dinfo%253Apmid%252F18949033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18949033&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.31" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D180%26rft.spage%253D611%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.088575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18757934&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259758500049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A modified algorithm for the improvement of composite interval mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D361%26rft_id%253Dinfo%253Apmid%252F17110476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17110476&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244180300032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.33" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.34" data-doi="10.1534/genetics.109.107391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svendsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The accuracy of genomic selection in Norwegian red cattle assessed by cross-validation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetics%26rft.aulast%253DLuan%26rft.auinit1%253DT.%26rft.volume%253D183%26rft.issue%253D3%26rft.spage%253D1119%26rft.epage%253D1126%26rft.atitle%253DThe%2Baccuracy%2Bof%2BGenomic%2BSelection%2Bin%2BNorwegian%2Bred%2Bcattle%2Bassessed%2Bby%2Bcross-validation.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.107391%26rft_id%253Dinfo%253Apmid%252F19704013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.107391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19704013&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272295800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.35" data-doi="10.1371/journal.pgen.1002051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makowsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pajewski</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Beyond missing heritability: prediction of complex traits</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMakowsky%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002051%26rft.epage%253De1002051%26rft.atitle%253DBeyond%2Bmissing%2Bheritability%253A%2Bprediction%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002051%26rft_id%253Dinfo%253Apmid%252F21552331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552331&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.37" data-doi="10.1186/1297-9686-41-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D41%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bfor%2BBayesB%2Btype%2Bof%2Bprediction%2Bof%2Bgenome-wide%2Bestimates%2Bof%2Bgenetic%2Bvalue.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-2%26rft_id%253Dinfo%253Apmid%252F19284681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284681&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.38" data-doi="10.1186/1297-9686-41-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMoser%26rft.auinit1%253DG.%26rft.volume%253D41%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DA%2Bcomparison%2Bof%2Bfive%2Bmethods%2Bto%2Bpredict%2Bgenomic%2Bbreeding%2Bvalues%2Bof%2Bdairy%2Bbulls%2Bfrom%2Bgenome-wide%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-56%26rft_id%253Dinfo%253Apmid%252F20043835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043835&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mujibi</span>  <span class="cit-name-given-names">F. D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nkumah</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durunna</span>  <span class="cit-name-given-names">O. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stothard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mah</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genomic breeding values for residual feed intake in crossbred beef cattle</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">3353</span>–<span class="cit-lpage">3361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D89%26rft.spage%253D3353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.40" data-doi="10.1093/aob/mcs109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakaya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Will genomic selection be a practical method for plant breeding?</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs109%26rft_id%253Dinfo%253Apmid%252F22645117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMTAvNi8xMzAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.41" data-doi="10.1534/genetics.111.128694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porcu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlather</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Predicting genetic values: a kernel-based best linear unbiased prediction with genomic data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D188%26rft.spage%253D695%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128694%26rft_id%253Dinfo%253Apmid%252F21515573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.111.128694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21515573&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.42" data-doi="10.2135/cropsci2008.10.0595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ridge regression and extensions for genome-wide selection in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1165%26rft.epage%253D1176%26rft.atitle%253DRidge%2BRegression%2Band%2BExtensions%2Bfor%2BGenomewide%2BSelection%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.10.0595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.10.0595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000268109800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.43" data-doi="10.1007/s00122-007-0597-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piyasatian</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection for marker-assisted improvement in line crosses</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPiyasatian%26rft.auinit1%253DN.%26rft.volume%253D115%26rft.issue%253D5%26rft.spage%253D665%26rft.epage%253D674%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bmarker-assisted%2Bimprovement%2Bin%2Bline%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0597-y%26rft_id%253Dinfo%253Apmid%252F17673971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0597-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17673971&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249019700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.44" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rolf</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mckay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClure</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of reduced marker set estimation of genomic relationship matrices on genomic selection for feed efficiency in Angus cattle</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRolf%26rft.auinit1%253DM.%2BM.%26rft.volume%253D11%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DImpact%2Bof%2Breduced%2Bmarker%2Bset%2Bestimation%2Bof%2Bgenomic%2Brelationship%2Bmatrices%2Bon%2Bgenomic%2Bselection%2Bfor%2Bfeed%2Befficiency%2Bin%2BAngus%2Bcattle.%26rft_id%253Dinfo%253Apmid%252F20403185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20403185&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.46" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of leaf structure in the maize nested association mapping population</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1839</span>–<span class="cit-lpage">1849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D154%26rft.spage%253D1839%26rft_id%253Dinfo%253Apmid%252F10866652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10866652&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.48" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.49" data-doi="10.1371/journal.pgen.1002637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Heritability and genetic correlations explained by common SNPs for metabolic syndrome traits</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVattikuti%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002637%26rft.epage%253De1002637%26rft.atitle%253DHeritability%2Band%2Bgenetic%2Bcorrelations%2Bexplained%2Bby%2Bcommon%2BSNPs%2Bfor%2Bmetabolic%2Bsyndrome%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002637%26rft_id%253Dinfo%253Apmid%252F22479213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22479213&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.50" data-doi="10.3168/jds.2010-3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5942</span>–<span class="cit-lpage">5949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVazquez%26rft.auinit1%253DA.%2BI.%26rft.volume%253D93%26rft.issue%253D12%26rft.spage%253D5942%26rft.epage%253D5949%26rft.atitle%253DPredictive%2Bability%2Bof%2Bsubsets%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bwith%2Band%2Bwithout%2Bparent%2Baverage%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3335%26rft_id%253Dinfo%253Apmid%252F21094768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21094768&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.51" data-doi="10.1007/s001220051331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Mapping QTLs with epistatic effects and QTL × environment interactions by mixed linear model approaches</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D99%26rft.spage%253D1255%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084014900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.52" data-doi="10.1186/1297-9686-43-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stricker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arango</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Breeding value prediction for production traits in layer chickens using pedigree or genomic relationships in a reduced animal model</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DWolc%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DBreeding%2Bvalue%2Bprediction%2Bfor%2Bproduction%2Btraits%2Bin%2Blayer%2Bchickens%2Busing%2Bpedigree%2Bor%2Bgenomic%2Brelationships%2Bin%2Ba%2Breduced%2Banimal%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-5%26rft_id%253Dinfo%253Apmid%252F21255418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21255418&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating polygenic effects using markers of the entire genome</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D163%26rft.spage%253D789%26rft_id%253Dinfo%253Apmid%252F12618414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12618414&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181417200030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.54" data-doi="10.1023/A:1016296225065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Linkage analysis of quantitative trait loci in multiple line crosses</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D114%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1016296225065%26rft_id%253Dinfo%253Apmid%252F12206360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1016296225065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12206360&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.55" data-doi="10.1073/pnas.90.23.10972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10972</span>–<span class="cit-lpage">10976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D90%26rft.issue%253D23%26rft.spage%253D10972%26rft.epage%253D10976%26rft.atitle%253DTheoretical%2Bbasis%2Bfor%2Bseparation%2Bof%2Bmultiple%2Blinked%2Bgene%2Beffects%2Bin%2Bmapping%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.23.10972%26rft_id%253Dinfo%253Apmid%252F8248199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjMvMTA5NzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D136%26rft.spage%253D1457%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8013918&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NB82100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.57" data-doi="10.1007/s00122-011-1745-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2BEuropean%2Bmaize%2Belite%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1745-y%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1745-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.263.58" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.098277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19299342&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270213800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/2/263.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/251" title="Origin and Evolution of Dishevelled" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/273" title="Saccharomyces cerevisiae Genetics Predicts Candidate Therapeutic Genetic Interactions at the Mammalian Replication Fork" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/2"><span>Volume 3 Issue 2, February 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/2.cover-source.jpg?itok=h3kYZ-FP" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/263.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xEIjs-TBIzMUbtmi0-UFsn7vK4xtpgNJo7eWmCTZyPQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/2/263" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/2/263" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/2/263" class="active" data-icon-position="" data-hide-link-title="0">Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459399" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TK_XeL_kHEo5AIzsmF7ZOA3n1V3xSk5mcaPQFA221zc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/263.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/2/263" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VlOR4gRAUaqQWf3t01mL0Q01Sc5lPHJmPRpwjou2-2I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/2/263" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/263.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459399" id="node459399--2" data-pisa="ggg;3/2/263" data-pisa-master="ggg;3/2/263" data-apath="/ggg/3/2/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhigang Guo</span>, <span class="highwire-citation-author" data-delta="1">Dominic M. Tucker</span>, <span class="highwire-citation-author" data-delta="2">Daolong Wang</span>, <span class="highwire-citation-author" data-delta="3">Christopher J. Basten</span>, <span class="highwire-citation-author" data-delta="4">Elhan Ersoz</span>, <span class="highwire-citation-author" data-delta="5">William H. Briggs</span>, <span class="highwire-citation-author" data-delta="6">Jianwei Lu</span>, <span class="highwire-citation-author" data-delta="7">Min Li</span> and <span class="highwire-citation-author" data-delta="8">Gilles Gay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-272;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005066 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459399--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhigang Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhigang%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhigang%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhigang.guo@syngenta.com" class="" data-icon-position="" data-hide-link-title="0">zhigang.guo@syngenta.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominic M. Tucker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Syngenta, Inc., Clinton, Illinois 61727</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominic%2BM.%2BTucker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tucker%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominic%2BM.%2BTucker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daolong Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daolong%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaolong%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christopher J. Basten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BBasten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basten%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BBasten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Elhan Ersoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Syngenta, Inc., Stanton, Minnesota 55018</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elhan%2BErsoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ersoz%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElhan%2BErsoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">William H. Briggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Syngenta Seeds B.V., 1601 BK Enkhuizen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BH.%2BBriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Briggs%20WH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BH.%2BBriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jianwei Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianwei%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianwei%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Min Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gilles Gay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilles%2BGay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gay%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilles%2BGay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/2/263" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/2/263&amp;title=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/2/263&amp;title=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/2/263&amp;title=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/2/263&amp;text=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/2/263&amp;title=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/2/263&amp;t=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/2/263&amp;title=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/2/263&amp;title=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459399/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459399" id="node459399--3" data-pisa="ggg;3/2/263" data-pisa-master="ggg;3/2/263" data-apath="/ggg/3/2/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhigang Guo</span>, <span class="highwire-citation-author" data-delta="1">Dominic M. Tucker</span>, <span class="highwire-citation-author" data-delta="2">Daolong Wang</span>, <span class="highwire-citation-author" data-delta="3">Christopher J. Basten</span>, <span class="highwire-citation-author" data-delta="4">Elhan Ersoz</span>, <span class="highwire-citation-author" data-delta="5">William H. Briggs</span>, <span class="highwire-citation-author" data-delta="6">Jianwei Lu</span>, <span class="highwire-citation-author" data-delta="7">Min Li</span> and <span class="highwire-citation-author" data-delta="8">Gilles Gay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-272;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005066 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459399--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhigang Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhigang%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhigang%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhigang.guo@syngenta.com" class="" data-icon-position="" data-hide-link-title="0">zhigang.guo@syngenta.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominic M. Tucker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Syngenta, Inc., Clinton, Illinois 61727</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominic%2BM.%2BTucker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tucker%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominic%2BM.%2BTucker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daolong Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daolong%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaolong%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christopher J. Basten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BBasten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basten%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BBasten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Elhan Ersoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Syngenta, Inc., Stanton, Minnesota 55018</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elhan%2BErsoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ersoz%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElhan%2BErsoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">William H. Briggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Syngenta Seeds B.V., 1601 BK Enkhuizen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BH.%2BBriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Briggs%20WH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BH.%2BBriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jianwei Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianwei%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianwei%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Min Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gilles Gay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Syngenta Biotechnology, Inc., Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilles%2BGay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gay%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilles%2BGay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=263&amp;rft.epage=272&amp;rft.atitle=Accuracy%20of%20Across-Environment%20Genome-Wide%20Prediction%20in%20Maize%20Nested%20Association%20Mapping%20Populations&amp;rft.volume=3&amp;rft.issue=2&amp;rft.date=2013-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Guo%2C+Zhigang&amp;rft.au=Tucker%2C+Dominic+M.&amp;rft.au=Wang%2C+Daolong&amp;rft.au=Basten%2C+Christopher+J.&amp;rft.au=Ersoz%2C+Elhan&amp;rft.au=Briggs%2C+William+H.&amp;rft.au=Lu%2C+Jianwei&amp;rft.au=Li%2C+Min&amp;rft.au=Gay%2C+Gilles"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459399/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459399/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459399/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459399/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459399/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459399/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459399/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459399/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459399/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459399/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459399/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459399/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/2/263" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/2/263.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/2/263.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/2/263.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/2/263.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
