<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/2/167" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/2/167.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/2/167.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/2/167.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/2/167.atom" />
<meta name="HW.pisa" content="ggg;3/2/167" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.112.004531" />
<meta name="DC.Date" content="2013-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Kubicek et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transcription termination by RNA polymerase (Pol) II is an essential but poorly understood process. In eukaryotic nuclei, the 3′ ends of mRNAs are generated by cleavage and polyadenylation, and the same sequence elements that specify that process are required for downstream release of the polymerase from the DNA. Although Pol II is known to bind proteins required for both events, few studies have focused on Pol II mutations as a means to uncover the mechanisms that couple polyadenylation and termination. We performed a genetic screen in the yeast Saccharomyces cerevisiae to isolate mutations in the N-terminal half of [Rpb2][1], the second largest Pol II subunit, that conferred either a decreased or increased response to a well-characterized poly(A) site. Most of the mutant alleles encoded substitutions affecting either surface residues or conserved active site amino acids at positions important for termination by other RNA polymerases. Reverse transcription polymerase chain reaction experiments revealed that transcript cleavage at the poly(A) site was impaired in both classes of increased readthrough mutants. Transcription into downstream sequences beyond where termination normally occurs was also probed. Although most of the tested readthrough mutants showed a reduction in termination concomitant with the reduced poly(A) usage, these processes were uncoupled in at least one mutant strain. Several [rpb2][1] alleles were found to be similar or identical to published mutants associated with defective TFIIF function. Tests of these and additional mutations known to impair [Rpb2][1]−TFIIF interactions revealed similar decreased readthrough phenotypes, suggesting that TFIIF may have a role in 3′ end formation and termination.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677" />
<meta name="DC.Contributor" content="Charles E. Kubicek" />
<meta name="DC.Contributor" content="Robert D. Chisholm" />
<meta name="DC.Contributor" content="Sachiko Takayama" />
<meta name="DC.Contributor" content="Diane K. Hawley" />
<meta name="article:published_time" content="2013-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transcription termination by RNA polymerase (Pol) II is an essential but poorly understood process. In eukaryotic nuclei, the 3′ ends of mRNAs are generated by cleavage and polyadenylation, and the same sequence elements that specify that process are required for downstream release of the polymerase from the DNA. Although Pol II is known to bind proteins required for both events, few studies have focused on Pol II mutations as a means to uncover the mechanisms that couple polyadenylation and termination. We performed a genetic screen in the yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; to isolate mutations in the N-terminal half of Rpb2, the second largest Pol II subunit, that conferred either a decreased or increased response to a well-characterized poly(A) site. Most of the mutant alleles encoded substitutions affecting either surface residues or conserved active site amino acids at positions important for termination by other RNA polymerases. Reverse transcription polymerase chain reaction experiments revealed that transcript cleavage at the poly(A) site was impaired in both classes of increased readthrough mutants. Transcription into downstream sequences beyond where termination normally occurs was also probed. Although most of the tested readthrough mutants showed a reduction in termination concomitant with the reduced poly(A) usage, these processes were uncoupled in at least one mutant strain. Several &lt;i&gt;rpb2&lt;/i&gt; alleles were found to be similar or identical to published mutants associated with defective TFIIF function. Tests of these and additional mutations known to impair Rpb2−TFIIF interactions revealed similar decreased readthrough phenotypes, suggesting that TFIIF may have a role in 3′ end formation and termination.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/02/01" />
<meta name="citation_mjid" content="ggg;3/2/167" />
<meta name="citation_id" content="3/2/167" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/2/167" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/2/167.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/2/167.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/2/167.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004531" />
<meta name="citation_pmid" content="23390594" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="167" />
<meta name="citation_lastpage" content="180" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Charles E. Kubicek" />
<meta name="citation_author_institution" content="Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229" />
<meta name="citation_author" content="Robert D. Chisholm" />
<meta name="citation_author_institution" content="Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229" />
<meta name="citation_author" content="Sachiko Takayama" />
<meta name="citation_author_institution" content="Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229" />
<meta name="citation_author" content="Diane K. Hawley" />
<meta name="citation_author_institution" content="Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229" />
<meta name="citation_author_email" content="dhawley@uoregon.edu" />
<meta name="citation_reference" content="citation_title=Current Protocols in Molecular Biology;citation_author=F. M. Ausubel;citation_author=R. Brent;citation_author=R. E. Kingston;citation_author=D. D. Moore;citation_author=J. G. Seidman;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. S. Bartlett;citation_author=T. Gaal;citation_author=W. Ross;citation_author=R. L. Gourse;citation_title=RNA polymerase mutants that destabilize RNA polymerase-promoter complexes alter NTP-sensing by rrn P1 promoters.;citation_pages=331-345;citation_volume=279;citation_year=1998;citation_issue=2;citation_pmid=9642041;citation_doi=10.1006/jmbi.1998.1779" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=D. Bentley;citation_title=Rules of engagement: co-transcriptional recruitment of pre-mRNA processing factors.;citation_pages=251-256;citation_volume=17;citation_year=2005;citation_issue=3;citation_pmid=15901493;citation_doi=10.1016/j.ceb.2005.04.006" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=J. D. Boeke;citation_author=F. LaCroute;citation_author=G. R. Fink;citation_title=A positive selection for mutants lacking orotidine-5'-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance.;citation_pages=345-346;citation_volume=197;citation_year=1984;citation_issue=2;citation_pmid=6394957;citation_doi=10.1007/BF00330984" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=M. E. Bucheli;citation_author=S. Buratowski;citation_title=Npl3 is an antagonist of mRNA 3′ end formation by RNA polymerase II;citation_volume=24;citation_year=2005;citation_pmid=15902270;citation_doi=10.1038/sj.emboj.7600687" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=S. Buratowski;citation_title=Connections between mRNA 3' end processing and transcription termination.;citation_pages=257-261;citation_volume=17;citation_year=2005;citation_issue=3;citation_pmid=15901494;citation_doi=10.1016/j.ceb.2005.04.003" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=B. S. Chen;citation_author=M. Hampsey;citation_title=Functional Interaction between TFIIB and the Rpb2 Subunit of RNA Polymerase II: Implications for the Mechanism of Transcription Initiation;citation_pages=3983-3991;citation_volume=24;citation_year=2004;citation_issue=9;citation_pmid=15082791;citation_doi=10.1128/MCB.24.9.3983-3991.2004" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=H. T. Chen;citation_author=L. Warfield;citation_author=S. Hahn;citation_title=The positions of TFIIF and TFIIE in the RNA polymerase II transcription preinitiation complex.;citation_pages=696-703;citation_volume=14;citation_year=2007;citation_issue=8;citation_pmid=17632521;citation_doi=10.1038/nsmb1272" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=Z. Chen;citation_author=A. Jawhari;citation_author=L. Fischer;citation_author=C. Buchen;citation_author=S. Tahir;citation_title=Architecture of the RNA polymerase II-TFIIF complex revealed by cross-linking and mass spectrometry.;citation_pages=717-726;citation_volume=29;citation_year=2010;citation_issue=4;citation_pmid=20094031;citation_doi=10.1038/emboj.2009.401" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Connelly;citation_author=J. L. Manley;citation_title=A functional mRNA polyadenylation signal is required for transcription termination by RNA polymerase II.;citation_pages=440-452;citation_volume=2;citation_year=1988;citation_issue=4;citation_pmid=2836265;citation_doi=10.1101/gad.2.4.440" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Cramer;citation_author=D. A. Bushnell;citation_author=R. D. Kornberg;citation_title=Structural basis of transcription: RNA polymerase II at 2.8 Ångstroms resolution;citation_volume=292;citation_year=2001;citation_pmid=11313498;citation_doi=10.1126/science.1059493" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Cui;citation_author=C. L. Denis;citation_title=In Vivo Evidence that Defects in the Transcriptional Elongation Factors RPB2, TFIIS, and SPT5 Enhance Upstream Poly(A) Site Utilization;citation_pages=7887-7901;citation_volume=23;citation_year=2003;citation_issue=21;citation_pmid=14560031;citation_doi=10.1128/MCB.23.21.7887-7901.2003" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. Dengl;citation_author=P. Cramer;citation_title=Torpedo nuclease Rat1 is insufficient to terminate RNA polymerase II in vitro;citation_volume=284;citation_year=2009;citation_pmid=19535338;citation_doi=10.1074/jbc.M109.013847" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. Desmoucelles;citation_author=B. Pinson;citation_author=C. Saint-Marc;citation_author=B. Daignan-Fornier;citation_title=Screening the yeast “disruptome” for mutants affecting resistance to the immunosuppressive drug, mycophenolic acid;citation_volume=277;citation_year=2002;citation_pmid=12016207;citation_doi=10.1074/jbc.M111433200" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=J. Eichner;citation_author=H. T. Chen;citation_author=L. Warfield;citation_author=S. Hahn;citation_title=Position of the general transcription factor TFIIF within the RNA polymerase II transcription preinitiation complex.;citation_pages=706-716;citation_volume=29;citation_year=2010;citation_issue=4;citation_pmid=20033062;citation_doi=10.1038/emboj.2009.386" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. F. Fisher;citation_author=C. Yanofsky;citation_title=Mutations of the beta subunit of RNA polymerase alter both transcription pausing and transcription termination in the trp operon leader region in vitro.;citation_pages=8146-8150;citation_volume=258;citation_year=1983;citation_issue=13;citation_pmid=6345540" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Freire-Picos;citation_author=S. Krishnamurthy;citation_author=Z. Sun;citation_author=M. Hampsey;citation_title=Evidence that the Tfg1/Tfg2 dimer interface of TFIIF lies near the active center of the RNA polymerase II initiation complex;citation_pages=5045-5052;citation_volume=33;citation_year=2005;citation_issue=16;citation_pmid=16147988;citation_doi=10.1093/nar/gki825" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Ghazy;citation_author=S. Brodie;citation_author=M. Ammerman;citation_author=L. Ziegler;citation_author=A. Ponticelli;citation_title=Amino Acid Substitutions in Yeast TFIIF Confer Upstream Shifts in Transcription Initiation and Altered Interaction with RNA Polymerase II;citation_pages=10975-10985;citation_volume=24;citation_year=2004;citation_issue=24;citation_pmid=15572698;citation_doi=10.1128/MCB.24.24.10975-10985.2004" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. S. Gilmour;citation_author=R. Fan;citation_title=Derailing the locomotive: transcription termination;citation_volume=283;citation_year=2008;citation_pmid=17998201;citation_doi=10.1074/jbc.R700032200" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Gnatt;citation_author=P. Cramer;citation_author=J. Fu;citation_author=D. A. Bushnell;citation_author=R. D. Kornberg;citation_title=Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 Å resolution;citation_volume=292;citation_year=2001;citation_pmid=11313499;citation_doi=10.1126/science.1059495" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. Gromak;citation_author=S. West;citation_author=N. J. Proudfoot;citation_title=Pause Sites Promote Transcriptional Termination of Mammalian RNA Polymerase II;citation_pages=3986-3996;citation_volume=26;citation_year=2006;citation_issue=10;citation_pmid=16648491;citation_doi=10.1128/MCB.26.10.3986-3996.2006" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=I. Gusarov;citation_author=E. Nudler;citation_title=The mechanism of intrinsic transcription termination.;citation_pages=495-504;citation_volume=3;citation_year=1999;citation_issue=4;citation_pmid=10230402;citation_doi=10.1016/S1097-2765(00)80477-3" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. Hekmatpanah;citation_author=R. Young;citation_title=Mutations in a conserved region of RNA polymerase II influence the accuracy of mRNA start site selection.;citation_pages=5781-5791;citation_volume=11;citation_year=1991;citation_issue=11;citation_pmid=1922077;citation_doi=10.1128/MCB.11.11.5781" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=L. E. Hyman;citation_author=S. H. Seiler;citation_author=J. Whoriskey;citation_author=C. L. Moore;citation_title=Point mutations upstream of the yeast ADH2 poly(A) site significantly reduce the efficiency of 3'-end formation.;citation_pages=2004-2012;citation_volume=11;citation_year=1991;citation_issue=4;citation_pmid=2005893" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Y. Ip;citation_author=D. Schmidt;citation_author=Q. Pan;citation_author=A. K. Ramani;citation_author=A. G. Fraser;citation_title=Global impact of RNA polymerase II elongation inhibition on alternative splicing regulation;citation_volume=21;citation_year=2011;citation_pmid=21163941;citation_doi=10.1101/gr.111070.110" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=P. James;citation_author=S. Whelen;citation_author=B. D. Hall;citation_title=The RET1 gene of yeast encodes the second-largest subunit of RNA polymerase III. Structural analysis of the wild-type and ret1-1 mutant alleles;citation_pages=5616-5624;citation_volume=266;citation_year=1991;citation_issue=9;citation_pmid=2005101" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=D. Jin;citation_author=W. Walter;citation_author=C. Gross;citation_title=Characterization of the termination phenotypes of rifampicin-resistant mutants.;citation_pages=245-253;citation_volume=202;citation_year=1988;citation_issue=2;citation_pmid=3050123;citation_doi=10.1016/0022-2836(88)90455-X" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. Kaplan;citation_author=M. J. Holland;citation_author=F. Winston;citation_title=Interaction between transcription elongation factors and mRNA 3′-end formation at the Saccharomyces cerevisiae GAL10–GAL7 locus;citation_volume=280;citation_year=2005;citation_pmid=15531585;citation_doi=10.1074/jbc.M411108200" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Kaplan;citation_author=H. Jin;citation_author=I. Zhang;citation_author=A. Belyanin;citation_title=Dissection of Pol II trigger loop function and Pol II activity-dependent control of start site selection in vivo.;citation_pages=e1002627-e1002627;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22511879;citation_doi=10.1371/journal.pgen.1002627" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kim;citation_author=N. J. Krogan;citation_author=L. Vasiljeva;citation_author=O. J. Rando;citation_author=E. Nedea;citation_title=The yeast Rat1 exonuclease promotes transcription termination by RNA polymerase II.;citation_pages=517-522;citation_volume=432;citation_year=2004;citation_issue=7016;citation_pmid=15565157;citation_doi=10.1038/nature03041" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. L. Kireeva;citation_author=C. Domecq;citation_author=B. Coulombe;citation_author=Z. F. Burton;citation_author=M. Kashlev;citation_title=Interaction of RNA polymerase II fork loop 2 with downstream non-template DNA regulates transcription elongation;citation_volume=286;citation_year=2011;citation_pmid=21730074;citation_doi=10.1074/jbc.M111.260844" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=N. Komissarova;citation_author=J. Becker;citation_author=S. Solter;citation_author=M. Kireeva;citation_author=M. Kashlev;citation_title=Shortening of RNA:DNA hybrid in the elongation complex of RNA polymerase is a prerequisite for transcription termination.;citation_pages=1151-1162;citation_volume=10;citation_year=2002;citation_issue=5;citation_pmid=12453422;citation_doi=10.1016/S1097-2765(02)00738-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. Korzheva;citation_author=A. Mustaev;citation_author=M. Kozlov;citation_author=A. Malhotra;citation_author=V. Nikiforov;citation_title=A Structural Model of Transcription Elongation;citation_pages=619-625;citation_volume=289;citation_year=2000;citation_issue=5479;citation_pmid=10915625;citation_doi=10.1126/science.289.5479.619" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. Krogan;citation_author=M. Kim;citation_author=S. Ahn;citation_author=G. Zhong;citation_author=M. Kobor;citation_title=RNA Polymerase II Elongation Factors of Saccharomyces cerevisiae: a Targeted Proteomics Approach;citation_pages=6979-6992;citation_volume=22;citation_year=2002;citation_issue=20;citation_pmid=12242279;citation_doi=10.1128/MCB.22.20.6979-6992.2002" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=J. N. Kuehner;citation_author=E. L. Pearson;citation_author=C. Moore;citation_title=Unravelling the means to an end: RNA polymerase II transcription termination.;citation_pages=283-294;citation_volume=12;citation_year=2011;citation_issue=5;citation_pmid=21487437;citation_doi=10.1038/nrm3098" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Kuhn;citation_author=S. Geiger;citation_author=S. Baumli;citation_author=M. Gartmann;citation_author=J. Gerber;citation_title=Functional architecture of RNA polymerase I.;citation_pages=1260-1272;citation_volume=131;citation_year=2007;citation_issue=7;citation_pmid=18160037;citation_doi=10.1016/j.cell.2007.10.051" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Society Transactions;citation_journal_abbrev=Biochm. Soc. Trans.;citation_author=R. Landick;citation_title=The regulatory roles and mechanism of transcriptional pausing.;citation_pages=1062-1066;citation_volume=34;citation_year=2006;citation_issue=Pt 6;citation_pmid=17073751;citation_doi=10.1042/BST0341062" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. Landick;citation_author=J. Stewart;citation_author=D. N. Lee;citation_title=Amino acid changes in conserved regions of the beta-subunit of Escherichia coli RNA polymerase alter transcription pausing and termination.;citation_pages=1623-1636;citation_volume=4;citation_year=1990;citation_issue=9;citation_pmid=2253882;citation_doi=10.1101/gad.4.9.1623" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=E. Landrieux;citation_author=N. Alic;citation_author=C. Ducrot;citation_author=J. Acker;citation_author=M. Riva;citation_title=A subcomplex of RNA polymerase III subunits involved in transcription termination and reinitiation.;citation_pages=118-128;citation_volume=25;citation_year=2006;citation_issue=1;citation_pmid=16362040;citation_doi=10.1038/sj.emboj.7600915" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=W. Lane;citation_author=S. A. Darst;citation_title=Molecular evolution of multisubunit RNA polymerases: structural analysis.;citation_pages=686-704;citation_volume=395;citation_year=2010;citation_issue=4;citation_pmid=19895816;citation_doi=10.1016/j.jmb.2009.10.063" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Logan;citation_author=E. Falck-Pedersen;citation_author=J. E. Darnell;citation_author=T. Shenk;citation_title=A poly(A) addition site and a downstream termination region are required for efficient cessation of transcription by RNA polymerase II in the mouse beta maj-globin gene.;citation_pages=8306-8310;citation_volume=84;citation_year=1987;citation_issue=23;citation_pmid=3479794;citation_doi=10.1073/pnas.84.23.8306" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=W. Luo;citation_author=A. Johnson;citation_author=D. Bentley;citation_title=The role of Rat1 in coupling mRNA 3′-end processing to transcription termination: implications for a unified allosteric–torpedo model;citation_volume=20;citation_year=2006;citation_pmid=16598041;citation_doi=10.1101/gad.1409106" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Magrath;citation_author=L. E. Hyman;citation_title=A mutation in GRS1, a Glycyl-tRNA synthetase, affects 3′-end formation in Saccharomyces cerevisiae;citation_pages=129-141;citation_volume=152;citation_year=1999;citation_pmid=10224248" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. Mayer;citation_author=M. Lidschreiber;citation_author=M. Siebert;citation_author=K. Leike;citation_author=J. Söding;citation_title=Uniform transitions of the general RNA polymerase II transcription complex.;citation_pages=1272-1278;citation_volume=17;citation_year=2010;citation_issue=10;citation_pmid=20818391;citation_doi=10.1038/nsmb.1903" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=M. J. Muñoz;citation_author=M. de la Mata;citation_author=A. R. Kornblihtt;citation_title=The carboxy terminal domain of RNA polymerase II and alternative splicing.;citation_pages=497-504;citation_volume=35;citation_year=2010;citation_issue=9;citation_pmid=20418102;citation_doi=10.1016/j.tibs.2010.03.010" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. M. Peters;citation_author=A. D. Vangeloff;citation_author=R. Landick;citation_title=Bacterial transcription terminators: the RNA 3'-end chronicles.;citation_pages=793-813;citation_volume=412;citation_year=2011;citation_issue=5;citation_pmid=21439297;citation_doi=10.1016/j.jmb.2011.03.036" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. W. Pfaffl;citation_title=A new mathematical model for relative quantification in real-time RT-PCR;citation_pages=e45-e45;citation_volume=29;citation_year=2001;citation_issue=9;citation_pmid=11328886;citation_doi=10.1093/nar/29.9.e45" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=D. K. Pokholok;citation_author=N. M. Hannett;citation_author=R. A. Young;citation_title=Exchange of RNA polymerase II initiation and elongation factors during gene expression in vivo.;citation_pages=799-809;citation_volume=9;citation_year=2002;citation_issue=4;citation_pmid=11983171;citation_doi=10.1016/S1097-2765(02)00502-6" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=W. Powell;citation_author=D. Reines;citation_title=Mutations in the Second Largest Subunit of RNA Polymerase II Cause 6-Azauracil Sensitivity in Yeast and Increased Transcriptional Arrest in Vitro;citation_pages=6866-6873;citation_volume=271;citation_year=1996;citation_issue=12;citation_pmid=8636112;citation_doi=10.1074/jbc.271.12.6866" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. S. Rhee;citation_author=B. F. Pugh;citation_title=Genome-wide structure and organization of eukaryotic pre-initiation complexes.;citation_pages=295-301;citation_volume=483;citation_year=2012;citation_issue=7389;citation_pmid=22258509;citation_doi=10.1038/nature10799" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. Richard;citation_author=J. Manley;citation_title=Transcription termination by nuclear RNA polymerases;citation_pages=1247-1269;citation_volume=23;citation_year=2009;citation_issue=11;citation_pmid=19487567;citation_doi=10.1101/gad.1792809" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. Rogatsky;citation_author=J. C. Wang;citation_author=M. K. Derynck;citation_author=D. F. Nonaka;citation_author=D. B. Khodabakhsh;citation_title=Target-specific utilization of transcriptional regulatory surfaces by the glucocorticoid receptor;citation_volume=100;citation_year=2003;citation_pmid=14617768;citation_doi=10.1073/pnas.2336092100" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Scafe;citation_author=C. Martin;citation_author=M. Nonet;citation_author=S. Podos;citation_author=S. Okamura;citation_title=Conditional mutations occur predominantly in highly conserved residues of RNA polymerase II subunits.;citation_pages=1270-1275;citation_volume=10;citation_year=1990;citation_issue=3;citation_pmid=2406567" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Scafe;citation_author=M. Nonet;citation_author=R. A. Young;citation_title=RNA polymerase II mutants defective in transcription of a subset of genes.;citation_pages=1010-1016;citation_volume=10;citation_year=1990;citation_issue=3;citation_pmid=2406558;citation_doi=10.1128/MCB.10.3.1010" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. A. Shaaban;citation_author=B. M. Krupp;citation_author=B. D. Hall;citation_title=Termination-altering mutations in the second-largest subunit of yeast RNA polymerase III;citation_pages=1467-1478;citation_volume=15;citation_year=1995;citation_issue=3;citation_pmid=7862140" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. A. Shaaban;citation_author=E. V. Bobkova;citation_author=D. M. Chudzik;citation_author=B. D. Hall;citation_title=In vitro analysis of elongation and termination by mutant RNA polymerases with altered termination behavior;citation_pages=6468-6476;citation_volume=16;citation_year=1996;citation_issue=11;citation_pmid=8887675" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. J. Shaw;citation_author=D. Reines;citation_title=Saccharomyces cerevisiae Transcription Elongation Mutants Are Defective in PUR5 Induction in Response to Nucleotide Depletion;citation_pages=7427-7437;citation_volume=20;citation_year=2000;citation_issue=20;citation_pmid=11003640;citation_doi=10.1128/MCB.20.20.7427-7437.2000" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. J. Shaw;citation_author=J. L. Wilson;citation_author=K. T. Smith;citation_author=D. Reines;citation_title=Regulation of an IMP dehydrogenase gene and its overexpression in drug-sensitive transcription elongation mutants of yeast;citation_volume=276;citation_year=2001;citation_pmid=11441018;citation_doi=10.1074/jbc.M105075200" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=A. Shilatifard;citation_author=R. Conaway;citation_author=J. Conaway;citation_title=The RNA polymerase II elongation complex;citation_pages=693-715;citation_volume=72;citation_year=2003;citation_pmid=12676794;citation_doi=10.1146/annurev.biochem.72.121801.161551" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. Steinmetz;citation_author=D. A. Brow;citation_title=Ssu72 Protein Mediates Both Poly(A)-Coupled and Poly(A)-Independent Termination of RNA Polymerase II Transcription;citation_pages=6339-6349;citation_volume=23;citation_year=2003;citation_issue=18;citation_pmid=12944462;citation_doi=10.1128/MCB.23.18.6339-6349.2003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Steinmetz;citation_author=N. K. Conrad;citation_author=D. A. Brow;citation_author=J. L. Corden;citation_title=RNA-binding protein Nrd1 directs poly(A)-independent 3'-end formation of RNA polymerase II transcripts.;citation_pages=327-331;citation_volume=413;citation_year=2001;citation_issue=6853;citation_pmid=11565036;citation_doi=10.1038/35095090" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. J. Steinmetz;citation_author=S. B. H. Ng;citation_author=J. P. Cloute;citation_author=D. A. Brow;citation_title=cis- and trans-Acting Determinants of Transcription Termination by Yeast RNA Polymerase II;citation_pages=2688-2696;citation_volume=26;citation_year=2006;citation_issue=7;citation_pmid=16537912;citation_doi=10.1128/MCB.26.7.2688-2696.2006" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Sweetser;citation_author=M. Nonet;citation_author=R. A. Young;citation_title=Prokaryotic and eukaryotic RNA polymerases have homologous core subunits.;citation_pages=1192-1196;citation_volume=84;citation_year=1987;citation_issue=5;citation_pmid=3547406;citation_doi=10.1073/pnas.84.5.1192" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=P. L. Tavormina;citation_author=R. Landick;citation_author=C. A. Gross;citation_title=Isolation, purification, and in vitro characterization of recessive-lethal-mutant RNA polymerases from Escherichia coli.;citation_pages=5263-5271;citation_volume=178;citation_year=1996;citation_issue=17;citation_pmid=8752347;citation_doi=10.1128/jb.178.17.5263-5271.1996" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=P. L. Tavormina;citation_author=W. S. Reznikoff;citation_author=C. A. Gross;citation_title=Identifying interacting regions in the beta subunit of Escherichia coli RNA polymerase.;citation_pages=213-223;citation_volume=258;citation_year=1996;citation_issue=2;citation_pmid=8627620;citation_doi=10.1006/jmbi.1996.0244" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=B. W. Trautinger;citation_author=R. G. Lloyd;citation_title=Modulation of DNA repair by mutations flanking the DNA channel through RNA polymerase;citation_pages=6944-6953;citation_volume=21;citation_year=2002;citation_issue=24;citation_pmid=12486015;citation_doi=10.1093/emboj/cdf654" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=V. Trinh;citation_author=M.-F. Langelier;citation_author=J. Archambault;citation_author=B. Coulombe;citation_title=Structural Perspective on Mutations Affecting the Function of Multisubunit RNA Polymerases;citation_pages=12-36;citation_volume=70;citation_year=2006;citation_issue=1;citation_pmid=16524917;citation_doi=10.1128/MMBR.70.1.12-36.2006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. G. Vassylyev;citation_author=M. N. Vassylyeva;citation_author=A. Perederina;citation_author=T. H. Tahirov;citation_author=I. Artsimovitch;citation_title=Structural basis for transcription elongation by bacterial RNA polymerase.;citation_pages=157-162;citation_volume=448;citation_year=2007;citation_issue=7150;citation_pmid=17581590;citation_doi=10.1038/nature05932" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. von Hippel;citation_title=An Integrated Model of the Transcription Complex in Elongation, Termination, and Editing;citation_pages=660-665;citation_volume=281;citation_year=1998;citation_issue=5377;citation_pmid=9685251;citation_doi=10.1126/science.281.5377.660" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. Weilbaecher;citation_author=C. Hebron;citation_author=G. Feng;citation_author=R. Landick;citation_title=Termination-altering amino acid substitutions in the beta' subunit of Escherichia coli RNA polymerase identify regions involved in RNA chain elongation.;citation_pages=2913-2927;citation_volume=8;citation_year=1994;citation_issue=23;citation_pmid=7527790;citation_doi=10.1101/gad.8.23.2913" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. West;citation_author=N. Gromak;citation_author=N. J. Proudfoot;citation_title=Human 5' --&gt; 3' exonuclease Xrn2 promotes transcription termination at co-transcriptional cleavage sites.;citation_pages=522-525;citation_volume=432;citation_year=2004;citation_issue=7016;citation_pmid=15565158;citation_doi=10.1038/nature03035" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=C. C. Wu;citation_author=Y. C. Lin;citation_author=H. T. Chen;citation_title=The TFIIF-like Rpc37/53 dimer lies at the center of a protein network to connect TFIIIC, Bdp1, and the RNA polymerase III active center;citation_volume=31;citation_year=2011;citation_pmid=21536656;citation_doi=10.1128/MCB.05151-11" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=J. Zhao;citation_author=L. Hyman;citation_author=C. Moore;citation_title=Formation of mRNA 3' Ends in Eukaryotes: Mechanism, Regulation, and Interrelationships with Other Steps in mRNA Synthesis;citation_pages=405-445;citation_volume=63;citation_year=1999;citation_issue=2;citation_pmid=10357856" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
<meta name="twitter:description" content="Transcription termination by RNA polymerase (Pol) II is an essential but poorly understood process. In eukaryotic nuclei, the 3′ ends of mRNAs are generated by cleavage and polyadenylation, and the same sequence elements that specify that process are required for downstream release of the polymerase from the DNA. Although Pol II is known to bind proteins required for both events, few studies have focused on Pol II mutations as a means to uncover the mechanisms that couple polyadenylation and termination. We performed a genetic screen in the yeast Saccharomyces cerevisiae to isolate mutations in the N-terminal half of [Rpb2][1], the second largest Pol II subunit, that conferred either a decreased or increased response to a well-characterized poly(A) site. Most of the mutant alleles encoded substitutions affecting either surface residues or conserved active site amino acids at positions important for termination by other RNA polymerases. Reverse transcription polymerase chain reaction experiments revealed that transcript cleavage at the poly(A) site was impaired in both classes of increased readthrough mutants. Transcription into downstream sequences beyond where termination normally occurs was also probed. Although most of the tested readthrough mutants showed a reduction in termination concomitant with the reduced poly(A) usage, these processes were uncoupled in at least one mutant strain. Several [rpb2][1] alleles were found to be similar or identical to published mutants associated with defective TFIIF function. Tests of these and additional mutations known to impair [Rpb2][1]−TFIIF interactions revealed similar decreased readthrough phenotypes, suggesting that TFIIF may have a role in 3′ end formation and termination.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677" />
<meta name="og-title" property="og:title" content="RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/2/167" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transcription termination by RNA polymerase (Pol) II is an essential but poorly understood process. In eukaryotic nuclei, the 3′ ends of mRNAs are generated by cleavage and polyadenylation, and the same sequence elements that specify that process are required for downstream release of the polymerase from the DNA. Although Pol II is known to bind proteins required for both events, few studies have focused on Pol II mutations as a means to uncover the mechanisms that couple polyadenylation and termination. We performed a genetic screen in the yeast Saccharomyces cerevisiae to isolate mutations in the N-terminal half of [Rpb2][1], the second largest Pol II subunit, that conferred either a decreased or increased response to a well-characterized poly(A) site. Most of the mutant alleles encoded substitutions affecting either surface residues or conserved active site amino acids at positions important for termination by other RNA polymerases. Reverse transcription polymerase chain reaction experiments revealed that transcript cleavage at the poly(A) site was impaired in both classes of increased readthrough mutants. Transcription into downstream sequences beyond where termination normally occurs was also probed. Although most of the tested readthrough mutants showed a reduction in termination concomitant with the reduced poly(A) usage, these processes were uncoupled in at least one mutant strain. Several [rpb2][1] alleles were found to be similar or identical to published mutants associated with defective TFIIF function. Tests of these and additional mutations known to impair [Rpb2][1]−TFIIF interactions revealed similar decreased readthrough phenotypes, suggesting that TFIIF may have a role in 3′ end formation and termination.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/2.cover-source.jpg" />
    <title>RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fLKAHIlw5qlEwtmMvSHTcfPCaO9SGRamILi8d7k9PVE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459402","apath":"\/ggg\/3\/2\/167.atom","pisa":"ggg;3\/2\/167","ac":{"\/ggg\/3\/2\/167.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/2\/167.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/2\/167"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/2\/167"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/2\/167.figures-only":true,"\/content\/3\/2\/167":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AcmMdaWSn5YolRvwCoWcvAiKpjHHXEvh+5b7UXdXW9MkKwgRgUEgxbnFsrW11WIPsJgFmZ4eCmDUz+K2p33onyn96PlgCI9r8F0jmjwN9mr34xkmVvZvk0hT6Xotg\\\\\\\/dkU7P9crLHnDPIdEV4CxjFRa7PB6zVMiS8XlOCqYjOGOeVnxrZjItyXeiu5HmBQtDaMHh1+4pl7pgHMQ9sVNkP2l2uQO\\\\\\\/MubdGWQvfnbowwP+642bBa\\\\\\\/wVzF6eqdbEfddZy6VxGEhVGY8XKhplMeaMP5ljFhW7lN7Lsjt1ZTj7if\\\\\\\/qn\\\\\\\/tAcletWJXVwqrLekHSaohoVlwL\\\\\\\/qxsbjfh7tebYGT7Dg0YlZtgHy3lIa1gziuWN3RgaCD\\\\\\\/0xDY5BGNw4q7pIHCBcRXnTi4SW90xX2q7awMKPzUo0PuttdMV+4nlpRQDfAf1KhcLX1U8Q+n1sd9rL5JZ9jT9RgmLiduScKVhP\\\\\\\/I\\\\\\\/u8Az06DBRuODmoZyHhB5TUSor1+vVygxJnc96Rau20i271g6aWBcQC5pR+xP9+whDWWIX\\\\\\\/z0LhmW+Lvc0pYulescj2MgIFi+vqyMegQvHsEZtDJxqYTBLNzPWhGqz4ASYIzwNgxTv\\\\\\\/T6sSNuEuLo2Gm404cGtxboMUgJwGvxMXVg+As+wQE9DR1DqBcINaXmW9hwMMb5qN22jSZBafJQe4677k+2935BhtXFYu8Xd5wTkRYxNhO1oLguiCnXm0WIBStJAdIbqrECRM\\\\\\\/7rVy6MIOIsRA5lFA4doLvc8EBDGLk\\\\\\\/KCufqJg+s8opwUkLp3eEhIvxxIq8\\\\\\\/Pae7FRlVsX8MkO13iWWeN9ONHssnKdBDFvrkfMK+KIJoCaamthdacs8KW2lxgmFIXrP9EgOCz2zFBKKxEBwvz3hzmO60rWQq1IREke98EzDGbbL8nGjWXMtObWKCTODf5aVORyAgxbusA6d287Zv8EclTopQdZ9gAh2RuDLOT7sATAGTDMA28cj7uji8xJf+JQ8Q7QBxGpiA4740ujkdnQd5O4r6lmQ9weY4IDJzHPo+czviDMOxqDmbkNVKeBgQZV9vy89p605Bc0UnwMON6jyN1RpQOa8tLJpdCYe9htbSdILk46jPbapVRUC8glszOyT2CF8tqh1FZfwa4wtnNBoJOvk\\\\\\\/WV7Gq7apz4fOio97xcPX3p5s3vTsEN5PlADFJdwn5rlmBdCnpgCtdW3pRHZ5gbBXUfY31DWJFJdlB2ZknSPYdLpLIIBnJyDobvjdu+ZEgDe9HhoRqiM9768AuwCGLTI7k\\\u0022,\\\u0022iv\\\u0022:\\\u0022LYbctuEk7UsF5zF759TUDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223d837623238e02160c2a58f548f9cb300280ba5c2fa247bd4949f72f9ab58973\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002237G0BBu22TuMUV1FQxNzGseIlO0pkNXPOOBEa\\\\\\\/HUmMPelnOda2WUU3QaLfqWd+omfCFvfUhf+JTaTt+nKqA99Jf74YJR7OT4LvTvgo8AYC5Cx3OsAsQZ29\\\\\\\/Lk3MSs3\\\\\\\/\\\\\\\/XTCbY7KbzYFR1rfIe\\\\\\\/9jMeE7ZSY6BaZvo4+4ZBQiMHHVufKgXzzy05MnuUxDlsXrHr13LGW96nDE8oFseEN8bs2uCqNVZfGZ+rYF8sRs\\\\\\\/s9xTTAj+NEO3YHVHpJPF1c9rkndeKwUz0ap5XwLEjMx28D21\\\\\\\/FyCLq2s2lbZWwTeUOwMruyvhBc2w7JpVxFfb1bmnoSqOS\\\\\\\/I7kB15lKogVHu9R7O9BGkmKFVCYspHofA+kX\\\\\\\/s5WPiD5ox28EqTZqMpdd8YB4jutmAGagehMWd9IdMkpz7pZc5KPRO7Hbugl8qMwaeE\\\\\\\/8zTh5BO2ckcu2k5LSyONMsTj8AEUEXjtNwViYY92JYU6wkrbvYmYfP9VgJMgehOBKUob2MQNACEJIK5XvzGkHdjEuv4kkE6WdHAdVKwHS6rqTzRAUUzdkwwHOUPgHNI1eDlPLD4A7Vv7Is6dPB5gDYCcrzqZs30cbgqBKLHEYpWZxZfhco5kIASio\\\\\\\/WDRf\\\\\\\/FtVTFqf2vbmQFXpgVhzMcpPEqTj0OF\\\\\\\/roDEl5DUFFJ6FEBfdWpH5waULmTYU65\\\\\\\/1jT\\\\\\\/5Vv14NgnULSdKgO9uvICTWaAScJoexG8AEl+Y7NNMxG4l4iQeLn710yYrazs4atKC75TDN3sj4GoSkzg7XO1bQrciDu59atlbsCKdwP2kEWrxSjikZ+FW8uaUiefD7N6XitSaCO6v77q7CUzMNPNqdoEhITOXdcnBRq42L5Rwc6TNwxnLzzykZhzroq+JLuSqce4TkeKECBN9fpB\\\\\\\/8wKoc1Bu75TaSgxGi72XBue0blrm7HlHXyPWm\\\\\\\/TXuCbjpoARMgX7311OppbSZUzdgs3CV7VSMIzQ8wHC6zZ32H+6jwIi39yKz\\\\\\\/QtuLpFYdMgxNIsnFjmXUpmK5MSfO\\\\\\\/fov0qjTH9Y\\\\\\\/HT9xNi8cScFZZz2GAyadz30BhkSG8mUcnjM23aZFeR8vrFH06DJSGl\\\\\\\/w3Zlcn2SZJI7QYYeLuziP\\\\\\\/DSPr0i2PNXp2iEaQqFGgh6aiEeLX+zyLKDiaKyVQkEQb4mU3b3G00sMad8HGS9JWwu1hz4NZcUHGAZ4tuSbEPzBYrXb7bzdLwKDxjUm6ecR4IZDRPY+FhhiUuzWAIC7bcyUkQ6Xjt1bSqoY0bO4ai2yR+aYzi+L8RwQ2o2dSDkq5kWgs6mjYGLwg7PlMZ1hi2JeP3wnWb\\\\\\\/zhpjVEcdnqmdfa6QRpjiuocKNhSyASl6JuaI7u8Oim7QvJ5N8+YszxDPSdRvKM0zLl8CmDmy\\\\\\\/hCdNnZu\\\\\\\/SIQFmPX9wS7KuoWARXAhArnKlF0II7L39v5MBgxcY9iG9nVbHqdfoynytb+tm6rQ9WTlSmD7wUsFnJfuLOaOfb\\\\\\\/ahjMjTO3aD+vTWw6VVuejxPNylHKqma9nLzCafo95Ogc7f9TJ5Dhd94EL2rsWv\\\\\\\/+icuGrAPBzd9RkHIWaIj+pYZTy9U5XhH5ybu9dOKDylFVF9aYUgBB5cOyiDbcbqU7+RDQ3Nkucr7bE5mRfxVhwNjpjjVVYMXY9oKJgtCILkaU4b+hZvYdfPm5CCOUv5NFTrotrZRDPSGDo7btplkI0JSR4H+3JQVza00wXNBdkaWYC6pfQvFCUJLVjaUo13kDDBel\\\\\\\/G8oLbp\\\\\\\/n\\\\\\\/FxukxhoJsRVWcO+q8Bh4bB3Kzupgdbg\\\\\\\/78PX8pLmd42wP66MZb+sdlT2psU63Uby3BuIQhorwGX2EFB0gMiGX1ZYYtheXCBNG5KjrLymYVJ54XdENohyY\\\\\\\/pw+KzWJjzxeqegYCo0XT35usExOmHS\\\\\\\/LQquvkS0mefZA8JWKt+3pYZqYehRkns0rzv3H0FkJD8DNLTnL\\\\\\\/z2mpf7dxgMCCO9gCrXeRXYWf8Eytzkh2yc9VuY3aiX8UYYhITadyDuGS6fnoHBuCfHYSsrg3pS9vvd2eYFLPMdBlrysvtcji4uW1gTEUY1XxrMRT3bmLpQVTfRnebcQJl\\\\\\\/84lrEgTD4PX0+YMP7jrdwlpiksng9JagZvKUFLGHpZYCq285UTvbZsas\\\\\\\/gYVv8o2p7PKfUptVKzZDDz1CTYgD0WUlu0NznBwb2AJj\\\\\\\/FlTpTCpWaZ\\\\\\\/\\\\\\\/W\\\\\\\/ITUwQKHSaVLN0tIAfofmjR17aR8obI4JZPFsDDKyQ84P+qn8qaDVCLuBODqF9wlW6lv2+gcb8ejtqWFaVUQhBG33Y7u9TvC2GmRXMCSKV9rD3titlUROu3pHqhPMVotlaLimMVca7mSaS7G1A9QhwnHBkrM7iiZ03HdJ2IVyF4tQKlCPY5FhhVH4CGISYeKorJPPWhYRmX3PfVFXKo3SsPTIhqXFh2+8JxgzBPUZ8PIQNaNYYP3Rb9JAx95obGLVuPlaIWQ22Q8u7Kpj3B0Nippme106bO5JPsCcjoZ6FXYpv1s\\\\\\\/GU42vG77quSDnZuerq7\\\\\\\/hkyP05PqAKxwphlx75+dsJ\\\\\\\/CtJsq\\\\\\\/wDN95t1qbbyUC+6GacFPyRAgZd6Zp4ZfzsNiJekjcro\\\\\\\/zycG\\\\\\\/GlbeJt+vmr248V0GARehx1VQKP+JebSORFCrLHnyxy+M9pdw2Vi7nnqebbRXVjcwLwhY3hvFPhV1s+XHjIIVf\\\\\\\/4mTinUtNt19dHW8+B1\\\\\\\/Q2AHNl\\\\\\\/D1Y5\\\\\\\/dbWsZba+wVqT3DeyMlPn+cQIrreMrFwd4xrCayIihjueXxg4wSmSfPMUEPl8oXF6+NlKsDzg1iqLhbO6QhIypgZc1Z57A17Z5bxqgwoAA2JfsXZcGBOdQS7zUz1c5A6ocysoMapw7raXKNSIhN8Koaj4g3VixuuLqicozXUDhnlYCVqIH6GHNVtfwB9Dt46NDwoMCjmvAwmCmIoIf6k0EQ4tGyma+gWwekE\\\\\\\/33f8OSA8BVtGqelnVHRDKqom47wEs9elhKw9cG+hf+\\\\\\\/wcEuvfo4cmBokFnmyHelAliCUrc5iIo9hbuUNl2+l2dXfxmsrx27+6srLMW2RCA38Wc9VZZHaudWGq1EfaQXfM3Wd+msSyV4IhNmsWGgbbz3RsXCmtAgfuErISPKrjaAMl6ui+j47gdGc\\\\\\\/M0VhoTRwZampvGLt3vkBHF9exc2oSU6ZMh1GMYMcR\\\\\\\/76\\\\\\\/ich4EJGFfLzHpmxVZGkyoNg7v7i1wl5KlFOrHMSOZ52eDfXtzYLfWHcDCpnp9b60+XYyizpPuxA2G9A5Da7s17RYaR4D\\\\\\\/Uorp+5f60OywFnryHdLk4E6qWuiDEavKGXvK2j2Uy5RCwaGJwPf4ogHKzLWbX2MAPLPMyHzWcMJNvRvnx5Z7QAemtlkkWgju7a6JrqXiwjnq9MzNXCTlCoZv55jN7pZcEqzBp1DhaelSrk622444RvHnKGIM+aKJzZscWt+1cdpUyF6qAjdJOl4gEauXqtLUhMaNPXiVhEj3zSfIy0HPBRedmLZddJy0tDlCSCclyYtQR3oMPH3GpyYQ0qfR7Ks3HpLzFnxXAVXaVZfRH4yFCcpKMq1koP+1V\\\\\\\/5rq3amVbhKQB4fk+lGQtpstLVN6GvuPwUJ32KZnT6ur3YBFeeEIqWL5jn1PRqlxQmAYTNnACmKe8sztfLgIgO8boBwY+gFclK03OQKImoovQnjSkHvIV4dZZ\\\\\\\/42CP0VhJeCKJ+rwvbUOBJq8epR6pNZx\\\\\\\/kUUxyfta9J9yFhwo+SEwOYTvLtgLunm5qUSyY4RCE7tkgfvjoBf9Op\\\\\\\/luo1fUhAxSkdAlwGOG3FlkxXIqhczvZwIwlkIrDpMjFwaCTJbnTOa3Y+7b348e6VM1IEdnRyQRFRt3VpfpB+NbT2a8AeRXFyjvxG+aPhEv31X3YCABUfBvTplyJHltKWVQ+GQVkfvvmaRImT9crvMuv+MXZU7sfxuneGCSAKCzKFSScO9qncTeuaK3zp22V8w9zlCTg\\\\\\\/cJLBsfjBbrp5BHdLBp\\\\\\\/qdXHpNNhGr0DdksvmggEmxbg5SOSPiixyS9Q6FolQKpCb189U1xvduodIrksrMxHH98qy9oxxJG9zhpLe+ELAhHcB3s5Jzd6luDnmlnJDerTjehHo0c+Q2S54H64HD++xkh4og28xHNOHqE3NbD5HfYw\\\\\\\/sOw3o1GsMNIaezqwD6vRZAb0e8DRE5SFtt2Jprj+LMuqFyASSLCH32DSmM+e7nYVcyX8Z00ym6eVoNoHrqULQ8pxezXGuwpWY\\\\\\\/Zh2LCufmDqYtDy0\\\\\\\/OQC+PoFT+XprsWj1uSQdOOeracXhzMLHA4E6oDcLhKhoM8PIhNqnr2MbgewVS2f4bxj9kWr8trcdfsrxhAVJkxwpARJQPsGsbOs4Ynljc1Xa5icuTXWggNWkI0Cc5vyxCHzU3seUwUVQ8svt9x2Cc6W4U\\\\\\\/6DQZ36P6OW1cIXYxbiRKFXButo1O18nIPB7HATON7fvW2ys+NmuZc2OQsiVI51OrAizECNgoR9zd2kh\\\\\\\/kgs7iefA6ZJ5PE7tAmylw4kFJt\\\\\\\/0vbR8IZCteMEHkOUlOouzgaygVbC73AA5vp3tMytjZ+fSvYMTje7iMOf\\\\\\\/j9XPc2XilcwvGs4L1KAhCnY\\\\\\\/rz+1TH9dupeX4XLZP++CIOL+sOM52cpujVudPGGwy\\\\\\\/6BSFjy\\\\\\\/gboJsOk3LpcCa2Zip9tBSfE6EB\\\\\\\/riyNv+3MUKVu+EBObsvUNmZ4PuyOPiX\\\\\\\/I\\\\\\\/I7bGrpERAvLfqZ4IdZxWyHz7IT4h0BX\\\\\\\/tcDXjl1obzf2LzepO0\\\\\\\/g\\\\\\\/fCJB+ePcl1XkhCbxJXQdr+aSgtxyuJDcYNJbj3OBfyv08AWsTTfvo5yEi60Ydpm0Jbur7nCjL8eNX9azrvhAoVPNYfAMoCxHqkEj2sVeqHM15bMl59nqjImbWrlH02zWkd5EBEfgtM3h0vW0ZPOuulDSNFLvo0BllGPLMru1buzaivYGInUlv4H71QwAQ18gReZD4ddifpWOJUkF1t6HbplbWG\\\\\\\/GnHMzPPzYd7AbNM3r9IasEpymWqqz6d3hEy679MUcgjqst+ABhnmv2pLHqMEnPNq1zriwv9zeob1D3FqisNzEsLCQYQVJYHLJWvtAc9ZIqL8YfnI9YitZkMyKDvl2mZRn7oYSCf+pakJFe98PwRCw1banaSoS+FsfqYJNTiX8wRp\\\\\\\/Pta7IkU8tLsTj6lDS3HqpXYgkMEr9muNfdkExYxo\\\\\\\/\\\\\\\/YDwtkwgg1jbjLvXnplPioa58lP4VzAMXMLGFdX6vpWer5ilvh880MSDymImOqM6K5ilawBVij+iN3XTKmO\\\\\\\/G8IaRNXmGdZEVCaSpxZc9iY2B5j2wfRMbE4SmoV+CrbXpJTDJl8izMUIAjgGV2w\\\\\\\/pYyoBSyKzdm6PoAAlVKPkbH9Enu1RkgIcwpMz4NZaXGF3fHX\\\\\\\/HS+WbwrBD7KwIpw1FQ1WSQTvhsso1jVdR6IwSbEwHuNOr9htIcw19s7Pr\\\\\\\/ERgz61FDCbPzD2\\\\\\\/0wst\\\\\\\/JufEFn1plPLVO2MHOYpP85Ug6aeX\\\\\\\/prPqO2JsQSs8TSmh1yq0JcwWNokatH6UAtRPfvviDHtXScr9Dn0919Qr9H6kQtlMD+YX2BI0HAsFLOmPWpbr0iekzohTuhZ96DejBxJfMWmC8wIpQPRHwbHFBAekXW7zacekQrfgFgk1SMjIqPWQiPvK3X6RsEM7C7JH\\\\\\\/p\\\\\\\/HkJTLHt13iqptzJwjRbzOqtV7LJ4u9UBsoBQjfl6c2jEH\\\\\\\/F2VCrqwqFOnllVj1GlqrUMYz2TF5k479JeFcekKJVz\\\\\\\/kSeOWZww6H6K1H2wImTfpCNhabLb4D3ilzEHpUG4nDpsfr+1N61Gf8wb4QEAcY51UX\\\\\\\/lPMN9CcsHgdvs6Osrj78ateQjLn0u1JzbuIMdxQ3bgE8vVuEZPm4kKF5a4XpOCiBQkeY7Zc52x6G1+UiBBg7D\\\\\\\/aUo7p8C69mHuxFJT4jBtMG9E6Jq\\\\\\\/m\\\\\\\/kWamK5i\\\\\\\/xO2kWrsu9r93XDoZcfSvqqyNR+oWSFwKTftGMCbYFNtmik2OwmFzsL6o5DGZ1IbEgPKEoT6ZcSSbOBPfHlrSCjlyw7HseJxlkpDlB5wThaHbprzVKfaUehY2syZvY6a8j7cWHnxf9afzSrwhvofdjbt92UuxDbg45lvj9tzsOpen273s42whi19vYJoZSJu\\\\\\\/ve+RnlIHuJrJ1f4p5WYnaVqxABkONJeJjrqc9Caa36lCk\\\\\\\/wifpaySx+KFp8shttsDjnLJRNoUWgXtQUd1Avj8bYYgqe4jcbmyaK3i\\\\\\\/jmRinXc0eC9N2F9PtW4phFq0w1Vf5m\\\\\\\/bmIpWXP7a7gTYqdx2hMqRqINktoJOSHvujhaUhv0f+MbqDfuc8MPuzYPbFrmpZhOuc7xS2wcgFUUNfXYqdn4LXkx1XevidmMnfNR5mVy+FoPZjZX6IuCriRvJGSpUvK94NYYe2KGg5iJDxY8vRutOQOoMKLiladYOXkQxlUoygTlZFtc6h9OkfLps917vTYADtgeBVfaO4U9DuYJskwVyFDukeakkrp6UdsItRKc1ORIk2u5EjdlQ6gWer+6Xmf6uhEJzNJgV9z\\\\\\\/sLxIxgwJcAoR1fqB9O63FRyS+72W5i08cfkiH9znLpqlGNuE0y+woFEWhJ66hP5Xq4vJP6AeY8Mq2AI\\\\\\\/kwDEc\\\\\\\/ipPTYqjxMs1oXoJB0nFHjAg+I\\\\\\\/1osUoQzcEd7Qdv4dZdxh33g1IRGJK+obglqNq\\\\\\\/9h87JWlwuFEwkRflg2BWCEFMl4x\\\\\\\/kbpv+H9UCVxyD9ucpQ7NZjRHmM73ThZCQFRBi7EUk9lXS0Ei8T1nfU9VDOf\\\\\\\/6DeQ9iD1J84yxxWuSGaO1P\\\\\\\/1rHomqBPhMUbaX355MSQ39ZBUjsRTNG7yTsX2AiSlRqnVM5cRs+zG77mfJVCStJ94Uqm6aYZ6YrKVhTMfMGVnpoYaNxnlQQ+tRQQk3xUwNsN08jtSq1hpbdLs+Rq47W\\\\\\\/8YLvUQDaMREEyVa1PPcElvbsrffTKdMBcr20QVQBKI8lR2\\\\\\\/VMKfJMj0s\\\\\\\/9nDJIK6mhUdp+ZEc7mjzQa\\\\\\\/lcH2nkjTqBJmpctYCyUexlu87vxhnfgEmxhpkqYO6MH6oBdZiNxctKm5aADYmH82l1oQKXu5LjX67eU5EJJu+\\\\\\\/IJpoUIwfEq34AhyxBjluRF8PodWPXMWufg+PDz7IrVcJYb1HDlOTPe\\\\\\\/F+zTeLYDwpA0hfgc5aWkQGi\\\\\\\/bW1uyM7ID1pW5GhlropkVomX1ElR6jBxIy7M85EUKjK2\\\\\\\/iP1ILikKxOxRufFLR\\\\\\\/yxxBsKeIVWFMuHSQPPY1J\\\\\\\/J4cnrgfDJnbJAVMEaZhh2mEluEQVyUJnuyoiDsZwMTROAmZAYKxQbIY9BTdaiBvCWK0Pm6OINPilwyfS46w\\\\\\\/mmy5EMN7K\\\\\\\/P4uKL93J5b5KyvdAf0FtxxyVivhi9zj\\\\\\\/jOq\\\\\\\/Qns0ymoLODROKea26LqNKddTe4AY47IIYCqq735cAysnUK2TIGzBGBcZ0gl82kEc\\\\\\\/LNywLaNT6DfxOHu31h\\\\\\\/t3Bo5TxoHYSXMX5OhBqMCsLkrlaKYL9tNaueJ85g+1VqMDWmqz57722AtjuHbfF65Ecdw2Ev5hQTC4eOJMs+xfEQg\\\\\\\/anhwp9fsdB2PV1eeDNbVrGA8412JD7brJnwfMdNixhiHVV9Qkpkzi69nHCdMWUtPdzz9hQABDinPEnWVOGiKWKP3iu+juGTj5mrBaODwY15DnfqV1lwVlLRh+PT6RQOcAUty3Z6PBgMUoqdBuyDAdQGP+EmzJn58MX0wC3KvhgTiIy\\\\\\\/jwTq5UQnNzJrdjxaLbknpiFloQrRrmxn1Dw+oVQ8BrGjluEKQ5I5uz2EfkyrCJpSs0peHKnbY42161iAFQl972gEd6eMhvp22p8HTAFp+kdVlFLdovQl6CH4KY8HIYiMIudIxAX+rHsR1Cn\\\\\\\/6wwWNx6I033tM3ylJMQUxwk+FdwrcgQKp9m92VxcvAJd3Q2Onv8gQeY2zjYcOHpxR0xWio1El3PzbUuPtx9MqDWeo3HGjhmxfH5hl7OusIfm80DkJ5jhOlZEcb0xL7VAraSWQSjHvcIepVfBRgoMZ9uGiPkMWt2P\\\\\\\/BekqsNUH2H8mqM\\\\\\\/A+7oug4ZUojcCvdPAasKuSyCb\\\\\\\/cDqrci+I55dlfc8c2tdr9OsnUp0YC3JDZy4WHn09AjZFfCD4JV4WTaRmmD3aoqKcb5WNSdA4bB5bmj1SO3evexx9OiEKHnoHehMATXlROkhhgGTw2Ebcw57DbbbH5lPcuNz6xNY+IchYprrA9D8hR8rXG9s59U9kn2mHmxdUXPm0XcA8YpDnw1H5o2oBV0FcY1Ids\\\\\\\/VcHwYjtzmEQJiaquv86MpjUN\\\\\\\/5RCGsLf8klhg5aqaThQuyxrRCLlrFvB6Tpa+k++GKWjFRRztCq5xD3MJKQetzE\\\\\\\/AdJrfi9hW2r6j0Npy8idRekYYaltVOtLlnnhrBL9RB7bb7FHvKjuqbW5wiEeoQCf+9zXkWxj8Dee4v81fNR6VGkx+NymjaIjAbGlf0bL0NVSbc258DOhhCxV9gOGRTG3BCU66xGLM6NlDAkzwBWzymKpHwYqbW+6HyjMK0H171KyL0SX0o4bEyNHH+ZM4ivqI7sXPfigLxvn9U4XzRJKvQWlqu8NHt8YG2Ju4lseppqkJ9Bx6NvfQGBgh09CznB2HRt2eNhDx\\\\\\\/gOWoIP1Emm+0UnvljY\\\\\\\/6k9Ill0aOF+4LHIfiaMag5StelMe9q+\\\\\\\/MZxKaSvdu28W22vBs+ZpjvZUG7aTtPACRbG3GNu5nXylsRfgy5xFHtWFGvsgiVYa5yjU\\\\\\\/i0rfjF\\\\\\\/aMPYNyCGKwZ+xycI6kCZMZykAQp0r7weMRjghRrCJ7v72XWuQqmdF3r2TQOJnypCAJN1\\\\\\\/50u15+\\\\\\\/cvI0Km3W+ndPH5LMu\\\\\\\/iMwbMY5802dorS39ifj5OErZKul5\\\\\\\/I4rBZRUfmkznRzxSKPd52LzhQ+pupvUTYJCXQCyel9eet9Mo2hUL\\\\\\\/v4ubYUrxK+XyetGugFzAPnA+h248MDDW6K14+e\\\\\\\/wBFntS0BTnYl+Dk2U0CInYOVXhe8lP+mld1htuEGpOCOAh8\\\\\\\/987Bb8TRReYQ8\\\\\\\/tGJk8Cjg+BMDN\\\\\\\/LGhjsbRvRSuthF05+04+uwnUIaYZtFTLpW9ujWxtjmg1xLHh2E5wlldjX\\\\\\\/uWvga1YKuSu2QAT0n9z\\\\\\\/NHy3NjcZZXC8NxAYXWcTqo4ji2d2r4LUf\\\\\\\/RDijW8ap\\\\\\\/z0NaxzbQ+KcUsnWMxqFCiiwyUPJ3wBTVrslK37PMJ01FeOqBzeX1yCoL2fdHlDDmH7rjpp\\\\\\\/VRBMNdqKsoD6Sv2t52noBp4SWRFCU95hGk4QmTyHiTA5bFBTaGvqxxr4i7Z0QM05zu4U9EmVo7+G92l7g+WgXGm6bn8X+8am49b4GTFugl08e+0czWt5EFQebUaDJ2nAdorAyx1NKG8BzIunZBAlCA6cENGvk8E5YGK52eW7henLtjZZoldhqAUCf8kx5XwE8VlxCiyHusQB5oKQu8MTQ\\\\\\\/bd8l+tCfiuXP1BYG5Jh+l7XSCiR1muBn+9Aa40wLceJ4i93j7MzRkthRzE5SwsqX5PoASO9lYeK6t+xxceimgM+iEFENsUn+yCWoxxt\\\\\\\/yGfTADOL+\\\\\\\/aZRY\\\\\\\/K+6FMmSr\\\\\\\/QHSCvjLgFCJjkz9VAN9kWMEYTTR1toaxKkeecBmkqrqhkV1qPxWh\\\\\\\/DJCoRqa87zaAjJJF2f\\\\\\\/XDz0APrgMX7FrRuHhr8mYz789WAe2sYF03t4H2L7nzjsa6G4CmAAqZo8Gb77w9KW2HuFEB8WAo6Z2IFDaEY9r+mR0DY\\\\\\\/ipYMU9QRbW5LAQgWhD\\\\\\\/0kELPOj3PBszT826HqZaCCa6CWY0nfvKn3bQqBMaoPoQP7+xTSbR9e9DZo7Rm\\\\\\\/pI4LCQacuNJCXe7uszanEf9TI\\\\\\\/1ygVPt9Baqo6LZtBjOnYDN3y3mT2Y5uv38N6Era0RQ1SUK+1vJXBp7la9CfNAjXfISlj53jfcwnHRvKXyW\\\\\\\/zeukZywgs4Oximwup5JjSYMxFN5ZyM3e92OTeoSolko28EUdozsi4u+VGsTQ+uqXdlLjhGW6CLfo4DwNpwqDm7NNv0J3shUifga6u82LrdEQm9zU2itZCSOaQh3\\\\\\\/M0dPt0s3jp98OgAgEGNVu+ta\\\\\\\/rEMkRuAopIiXZBWfSAGrRR1FsWi17vsCswY6myrfLopY7qfoyLg6YQo8UVqYKQtbdT\\\\\\\/B4ZBpIpSaJfpCig15MKzaL+5nyYrf8oI\\\\\\\/XUw5FeXg+j9yAHb5WhdQqesRasuoZvfI9D54DrSGLKZjrO8g1KYqBlSs6vtMxZBAeVy+c6+guCjvGPbdDYprffM\\\\\\\/0woIAb4QnRPkEBbh0dR5qpKJfqeOr5V+skSPbcauAFQGtkPXULjvI\\\\\\\/H7toJ5KWnRgoWlBBcsIkgE+20XPfUKr94LSHDCt5cPOe2bLJ+myrhjtn4Jf7q1sxNq2B+cq3x\\\\\\\/\\\\\\\/M320bcYb4pOo4xYFG2LXJvwg83X3Xh8qSEy\\\\\\\/8Blw9DmFCIwe1h40x2kTbbGJh2LQheVTtalpQsQgjD94MkexIC9JtVysUfXeD+Zqwj3mRz+rWkXmvPHnYZ9Lx1TFT8UXx5z5Rt1xcD1k2PErQCFgfH8Te2MelkdbAy9lW+212B6Buj+6wngkjFIqoPU\\\\\\\/45ylkeL6qCfxrQMn0H6RPfrurNv8mGLbTE9bAO3VBwXMNKI6PXK0Rx0zGI+QnpPCj7wm41jfADfKsGclfgVWavWjq04KXx\\\\\\\/PZBIkw2qUeIHqpv2RMLIRFoV2FrToQpi0ItkZU\\\\\\\/JBkMG5gk+EsyolQh8\\\\\\\/4scbVDpFl3kVH8J7mcOqZWOqZu17NaQKOpQadDloAaqWPFKH4XABui50z2y5uRvnTZCh+jDtQRh1Bp3\\\\\\\/uBbU+201mfFgAR0xxNgx5lQZlKVC0z3fattCR+lm89We8XOjx5AI4xSFXboiuAfovWiWWTuAK06oL65mQ4dLdXqA4CmH+SRjtntp1QcJ58AQKis8vw8OQ+WNNIKfTRLFOOcjj9914EYkbYYiqQewbsTOou15w2\\\\\\\/QuWmUTiS3wUszg+\\\\\\\/NjwkUqGgEJEJ9aRZ+GNMHZs4MpHQsdz5CHmZw+JF1ZTvBhYym\\\\\\\/FS72NGq4eA6hdQYxP2WrvCmjauIgIEAGOaAl+cpWuPXnuqevEzw7EH+6zp971xG7GGXbc6SFtLiEEbGDVzrnLIqLPtdTLeLpc8IEd+bQ\\\\\\\/aZHCPhtWCFiytIba6Vd6\\\\\\\/Dla1bv1eaHmWxCBXYyJMLS+vhuwk\\\\\\\/JujVabUTT6Q9J5FonStHZ1GDeLwHBIOczdlvqZQ7dHAfP7LmNc24JSxs3A1HdtjU25kb5F0RNFF6w4gPqR8nH+CPMgkF95SoAfsycPxpUmYSzA9E87ZYEyReZexQjm3OLrorOQkpd2bMj89AVDJnK8kUaDPNePYDeLofTAhoW5ATIbw4vd7gITb\\\\\\\/8pt\\\\\\\/QNc21KRhoUf6UtUGaLdJ8j+qxEAwGnttgERvKUkx\\\\\\\/BHdyyGoABF5OANt+gf0tEThIJ8E+6Lj7+\\\\\\\/tyAHkELstsLlM0ALMGn1xCuBUpPypUzbwLEhc7jr3ghhG35RTtTMHnnwTw0nwI3uhnyPlDzwvMsUT3DQT8En1NFlblec6s8vsTeeKRXdFmcoEu8KMKn6BVGox25rBYoJngE6UBRjA6bqVomKDloYKOrEF4WBVPolunVyKV5nUE3ed0PixB8AfsnoNHEAmjscO7ex7\\\\\\\/2k3rqfUzRvn0hikOsgGb\\\\\\\/ZHCEKNovlPSqEW\\\\\\\/pcRFOLsobFlaOezEVZQ\\\\\\\/dDFnfh9B+QrCi0x4rtv6qOjmKnj8x8k3XAKp5lylFG2WOS50dWABRBG6TXOivZeYWmEK2yNTcFbNBiE7SNDivIejLwIEDdaB2mzUokWrrZldixOKb+hw9KhzGP66j+BTtyvFGFvk4EakYP0JWQ9KcjbJuimSk5pEsRp6OYDwhc47\\\\\\\/2bL7jebQK4ThUiLK0OiNRh45gS3ekfQ33K+nhTp0EIYEBQpThBI96bLmHuBrph10PwlFIODgVT74vWJAlZz1XUoAtFGmD1u4iFD8vP456A\\\\\\\/vzqWfyh7HrOqGZwwn9CwGLNjbnYQPQsApEo2RT88HLpERfWCleH9iS+lzgZrHhbk\\\\\\\/JrpZKiqUqc5LQsrTuJ9XorLhdcVimfO0Z+Hauaj5urJOT9OGNP36kn+mhCRUzCGL4Q3sisFpkT+iv9W1Ud9olMYHx5z5VcKF6chkkMZxj\\\\\\\/C0ZayByaVVgsmn5q6KukbPiAhbx10RzRcRB1YF\\\\\\\/HbonkbmwwnrKcO\\\\\\\/fBOFUVrvDEH78n045H4duzPkhfRXQVjL5C978tgqWTuGghFP6GreRhp2RvlJIHQXVRXVtJLze0bv5nPqLITphaczCeghyMD1zKhSxuFMV4tXHI4D3v3aHqBCuzXM1\\\\\\\/kmUFLB2AsAmSeSxdK5EuGT2m+4wSGxetfDEC59dXPNutgAHVK2KdwTR59Kf6T+imD8kA5hQm8ROwYpe1+wTkpxwo\\\\\\\/kRpBQTN5kIc+dceDwn79ZjmPR9I7bcvdwfowfVbLi\\\\\\\/MDEuDo3COFfYHpEdmxbeVG+L86PItMnc+91bSj+EkqGwa2z3zDMtibZ4k4MhcnSG2IwgGG\\\\\\\/Ek41f+miaxu6i3QHwAy4+3YFuh4QRARttvUHCKczyArv22Y7TZn5v3c\\\\\\\/uzfEXu9foqOxqJPr3NNTTWRsXvpedwvpMV3XqPjqgpu9gDWcw5qT8sfmTbuTAQKHL1Gtn3GmTDbf9JtU67NhA\\\\\\\/Lh\\\\\\\/M0qd252n9wKm12NFYU77UVS0cexieLid5goqLftLBTTAehKSULro31DIBGUtH5NsyvQGReWLE6vlpbGS72z+NcnaayEdHE\\\\\\\/0R+iATf4ZkPqwFkXrFYZvSFO8AHFdzW2YuLfjRSONfV0ATd3lSnSWMyXYrL\\\\\\\/2cSfvJsZgpPJr71ZTeIYWjx72q1GuBM47GDon0lBSwtlGu5mQnfV0ylsMpxZm5pEdoovMWYmyZqQ45aXahWNO+kuK+YQr\\\\\\\/FchBThuJogWhzplmDfeVEqgLJq\\\\\\\/Jp5qNfwrWGzscMvxTfoOfBP\\\\\\\/57tYfDW7s9LHnppecSlEKurarAIwUrTdRGSlSobqjrwpByF4Nrnco6RPMn1OoO5\\\\\\\/xT9ydtJM3PhssoqoHYUJLZVghLEe5iw6FSHC1jAYzVdIGjvQ4k11OdEIHItci+7CGZqWV325de4XPB0zm6C5I441Nv8Lf6220sfLu0Pqk8OFVKAzJi+uPlSC4ayU1w0XrgkCnJJ43XH7957pNNColhT3UdtF7XwzZQEr0Y7TrNALJMUQP2rZf6dl\\\\\\\/mYPqk0vuoq89z8BWD9dKOjC4SxnKzfy5XPs0LrebUJCvwF3lfC5mdCHbwSr2aQADE617PWc0THwwPT4ODUo\\\\\\\/ykWQNVw5q9LJN7urfs+LspIGul5ApFtRRAGPPVG4WiXwWFfuuRPkeoLDXS67z3O0UynvTIwavpbbwiAAGbwwAgV7hB3SMEwGXJF1GZquOzNxic+ZJHbZlq6z4W+nkgfiy79z3VZt\\\\\\\/KEtGJOE5J0TZpa\\\\\\\/dr8Cua+sZcjw54dmRiqHQ9wzPegHIWNKu+fTiFbjEoFOJkH6ZWE6lvCy+qNsCG1wVZV9cFpSbH4zJRd5HYaipEtAqzovqMdcRs8GfJxRu8N7EypUZpy\\\\\\\/gpx9tHl512c030iLMqZTui52lgDByEY+Hthpx1vPo50c2TTrLWY4W0gEeL8J\\\\\\\/XdaWd+u7LWXgXTD+A2gMLebMt4EdOZ+josZBd+aD9OOduUdRx0Wm50bayIv8KqmuYR2aR75kOryNhuu3AIFGQ3iQN+lK70NZTapaYSgr5tca6uYyLUk23b+GzhL7H0DN4QHW3ITZkTwkODh2+zgwPn7kOF4JLUXxhTsodC48QaPsyi0yStCAn+2JMb75McZ5Nc12MTHw5CZC3VXaWEur5bJ2ZiCjjwHNwIdqJK39wH6UYJY4vNT3WOQxQ4VHUyz+RcuRJVv19k7NEwb8URwcSQiXFdqf+1VOI7HoTi5vhTaq9uXeEtDyo5DeR17Bx\\\\\\\/pPGh10Oylbx4l+VEUlHecWVCZt0sQqmWQQqXNQWgFdTsNd0P8qb1IviS18bQ59GscS5Shramgjb94FrrG+7\\\\\\\/cVAK\\\\\\\/\\\\\\\/N7\\\\\\\/WLK13vv7PtYSSM8oKFyxihgAOWQ3yErj9BGEY2LENHbkN9zrpmCBXkaHz0u2uZC9F5k+aHs40+uPTLisZZTUaMdWytinikcS5W7Uk3\\\\\\\/2Mw2ZCbs4\\\\\\\/aUZP72iXiOHYfFw4uD5BS7iOtUkNX7WSGj7F2d0IY3gR16SsgIrtAEY+LxHeTtyrQV7DUp0hPgypdW3e3re\\\\\\\/Gs4+CJbuK5ZmI9c96JSLpYAAnCgpmIKU1xUwTAxj+qrScixPmBGdYSqSWgH9rMf1UTDGNhoKI5qmsektIdqsRWkRcTbfgIEFHQ3N2pgGjDISKlTuX0NcQ5siEX7k1P\\\\\\\/DF7jQ92P1472L7QRUejYLomuawmMhpFFS3UWPZAUWxvn1ePSmwRojdQiAq5lN6iO7mZQWX9Mxr1r0bztcFm45rqRSi7clgWy2RLJ3F6IkrgzA0DXysmpabUFIWVeb04KGSecmiimfMvGXNn3zyIB1sFqHl0UlKawed+NcZjdsafZ3mApoygSDC4DpiduIlfQTz5dcsWcGCbstMsuIejBiRpESZRvwuQQu28xWO\\\\\\\/qOETL36w8ecgJuc8ubQCzuWYZnIF6\\\\\\\/CSJaal3sBZqOl01kRGuXbPrr6okHxFbdZ5LuTZ7NLbpO+JPSEj3ORilwdnTG9+1nTW1LbeEyKI4JR1xNMAdnDfr3l9EeLnTFcLLkCVi6dbtwdMOdBuLlErGTN1d+wpvymnNwAyhnZrk42KNpK7SFPDceuD1ByvlbSJm\\\\\\\/nHLHIGFg9pR+h8tfQmq5fJpXvXZf25a2Nwex2uFx3A4Gtc1d2Vq3VyBiF521kWf+V6h4B4HXYbtVeg2+QqfSiux\\\\\\\/0zfFT9pZPnGM1rMiwMsGNV1Q\\\\\\\/EfAY6M8zsCXJE5HiC+46MwQwW6tuLYbVH4X3riOrxe03lqz3J04mfTA45paNP9dHp4ImLeTcyJdXR0hhIGHRNbjoebYPuimuVe4qpf7aHlQinlUlXFAlP7b1QxD3fmYGqfcUgHyGyRpFrwYYOAii6M941I8ahJjyE4M\\\\\\\/gT7l44D\\\\\\\/Fpr4og+WDNbnf0JtTkmt5IsHTyOV2yB3UUEudku9XkZwMFrHjqMWwtTdawWf9fypAvgHqwTs5Gez75+To5Gpwmjc8UTktQhKGQGUNLgPI9tfO4e0AHsINRMMwLWvgNGsnaLOau\\\\\\\/cPfdQgst2RgSc0akcrnepCScx1rDLOInJV\\\\\\\/EZHbwVb+LJdVDnCchs4H06I9zfS7z4TRdNN0GFwvQreId97qMvGDjFmYheflEzd1S71xCAfKyKqatNAhbi5IyRF1uuGU+atOFE3N6cBhjQHO\\\\\\\/VJXBJ8njI9qfGu42sIxaUC6SHU8yJdZivAg5DmeYy\\\\\\\/rIADjjHf8eZmMceUhrQZjtAZSImcGW1txmEBfI3W0CNkPER5pHSEzG6Xj7pFH1diCu7FHfYbonXkZ1V95SdNmDUJo1hGyio3EovoIclhuJvYAMxNsYEy8d4JqRZA18EJMYvm7RrQrMozB1wPsbLuZrlEWjWHM+51rsIMzSB3+uDGm9kRS\\\\\\\/TYfWb1dcqkBZcry5vALGnoePC+ORmutl6uQWKcF7mWtu2Cm8c\\\\\\\/wmA0GUQWGHAyvXoFs2UBr2Q8HFfkVauhdLhGQmiNPrqNPPqcfGHark6VolHUK6gNT4EIpwKy6wNuT0Tj2tgrcVwvdsTJj2B2IaA5HSZ0lMGvGZZ2YRUu6srmjKn45BFHd8GmpubEEM7bzdTb21OsrRxny2NWzZkD8JQ4Oo9jtyGvbrtYBW13ZuAh3+h0g73VfuggqeMDawtmdnwojfwrv+M7\\\\\\\/v\\\\\\\/19yl5EBmi5vCh3+Y0XDt+EWdSbiFw7YYPcl+wybqRoxQEtg\\\\\\\/O+p9+h7gZDEJRcva38bA+gkZz3uYLZlho8NmpjEW7xRw\\\\\\\/ySRFpPqCQ2CuE8uVIKQXFLgGkaD7b47oxPF6845mfmyfFY1\\\\\\\/VvOktL0mKqSC3abN2ALTo9q+UWF8SIlRdnZ+or400RNE9kGfOG+XiXXEGJa99BexbVJaiC8gPaZjW6+gTDPUbf748v9s6EjGFn3J+xELc6xfochOBZbBubYRSe5oV8C3uYMK2gfMBsU4EHkCQG5IkK3HdGGLPnvx3hW9tFmSPhfAbkZoLgJg9zASWwNQxnQBMuXJw19kWkiNbFJmiHb\\\\\\\/RFnF2Ws+lsuDrTlaF\\\\\\\/wramNYexSl\\\\\\\/jhL0oy6XTpF6fPY\\\\\\\/zCQvPT6a5GE2081OMCryOD+dLjjHON9grII\\\\\\\/6lAXs1JK7vGic6C6EBLDafqMSxp2agzmcB+DolVU+dutZihT4S4Pb1cNCdXuGKApMl7ZMn4YDwUCLlle2u+icDMg240XRxhvR+smykMoYUfh5qvAICW3vX5R7sxGoPtpzoVz6X\\\\\\\/KDF+zSxHcj97gHLSdWyxldIRwk1zXYaLkwjF\\\\\\\/KWNYALQbep\\\\\\\/A5BTHIXrCudzmKtgexLlnInVh5ExfWrRPgvjn9HtbW8cSPXcGtmc2BYhvnNFyk8MnkPdXWD5lZYULos0Mb5fztPxn+3DO3NavpOScZQpdJlotUQAJatzvUZ8SKUR0A3vMM5lJgdaMBT3jscB+1hiOFhB8C9I1arH+NK6kJ12H\\\\\\\/ZGg8A5JZBZvjT\\\\\\\/G\\\\\\\/iv\\\\\\\/+vqB9PxzubJzE3PfahBUhxDn600xd5kK7UdjR3mFvqL3FskE7Vij5jc6Zc8+uq0BPLdbI27Cxn2zIpGRfbuEjmAUDEnvOF1WLuK+o80YeorM5jvPPSZVc62Fvu9wtkKQicl2WlFEcfWuPYNo2J2XWRXLI5UOkKUgVLzkKiuroRbDilSKm+kplsVrrfBfSJg5NLevCv0cYpqzqwjkhTZvmIr23IOI0a28Q3Jqp2oepOm1aqNmaLm5VEjhEx8uKE9qihUX14\\\\\\\/IoFbAKpzA9KzEzLPtEop2s408IqGBMsNK+pJ\\\\\\\/fva0ZRH668XWTdbyl5enT1yiAhbIZ1dTZxqg253R6dCGDQYskQOKLOD2VyQW0SBzyReKpiWoe9K\\\\\\\/N0dQob3Q5DN0a2vLIUkQ9JHXydwcsCyWlpiJeKkuODEl8hlmEUs0GwOZgn7UIxZDT8dCi+IkQEl1mFJvEAh5gXwM82xpF3YHmAYbYibti3DallSlXA1SY7Aoczx363ZFZeFEgT2Y+69gF8rSqqAtrU\\\\\\\/09COpoZABuV7aiHrkdJK+99kN75C7QPDHGVL0bi\\\\\\\/0CBsBBXvUy2iyjCOSAeExrvazSmRV2PFthgcxBBEAwZ7naGGuVO55kCzpSH\\\\\\\/D9hzv8FQrkBRKF6dMFhcC0+YHwvDW\\\\\\\/QE9ClpJwmVQsbpDIZ+niNkH4mUt4bk\\\\\\\/OOQf5\\\\\\\/9F1Y4CB6RukwlQI4OaWY+ayNmUx3FWkRNIxoo+AeZnkcRuNCQjN3yQv5vnkzcidestJrQ3ZVFEqRh6qXhYX9WAUjwc7I\\\\\\\/E5S6aUsaSGP\\\\\\\/pxw0NaxjIr4t00ONa4Ktb+2vH90LRJqKR8DsclVKalmAB2\\\\\\\/wVHFDJ0kaH9pUaSBmaxBP5GMYW8qSLzzFinxa4BwCQH9PpHR8X9zEh7iayVue7OUT4y13yaEjJgK8012cv4P5LqVlpJc+a1OzL5KCWoATFRlHd4R3pxgsOd2LYc9PN9sqeZrk8wxI81ocI685XioaT7hemMJbdeSSpmh5H22ocuTieHFMuruCMqqqwW5qgtW0+YreukqXTtGQcgTyqEsS07jMod29wCHG\\\\\\\/2JMY5uRjCHcGdUhMtfjSGne6ytxKkD4DrIZ+280DL1e9wrUWdyo96ieajYFhgwyYrxSMrRbwJmEmC\\\\\\\/1Gs2f1d7GB4M+lv44Zkh+WvgJJ85d2rBRaq35EF\\\\\\\/pNfim+HUkyIU8dmq9fPv5rdOCJokp1fqw2pV9yIWf99wfZCIgqL6e8RIiitvFhRqgdhZTCKMwtYF4lGNJG4d1m3Ron5vlDmfLLifgW3bFYMS4ikXB0rkWZ6jbDln+OqmsHM7qAM29I8dnqJN7enTTP0n+A0Xkymb0l5MjFaRZWbcN3ZhyDkdPjnguouJTe4ur2zFCuZimiPVkOwe2ZyXDV1NFJm54AhVS+4y7zzco7P0wIeeGVvy6jVoyba6jY7EhABpvVi14+qxd+ZhP5OzrE6AzhJEESE6Lyeh6TEhAMMQvdjR\\\\\\\/b85h+uA9olefLRup1X2A+ijJsjNWzp4N67G2DRVKUeN0PAfpnYlLnGM3vv8QAVTS7mfVOq383bX3pIZyJAwRiSrqVAXByfWXWkkdXCBf6e62Sr8FhbBrrnsJc31rJS6rAHIPFRMUSHske8uiCF+QuJ5fsaukfo0IM2aTn\\\\\\\/maxnuO4NBGt5qXbKb4yfwyIx\\\\\\\/XRWj7wdHNjmJWPLINayz3Xl0wlJ\\\\\\\/giNz1hPkkJ6UU4TCl8v7ZpUSckfe6O17QTAITz6pRNiWqm16FmQeLVdQmqJcqY+JBV177ekIaDuEowH58CkJt82c7ImI9h87eEUI6AGWOtvILIbnPbJi9N8izoX\\\\\\\/hRXP7Utbzs+haOBRvoefRyjWH\\\\\\\/rw6KV3TZsu2IuQNcrCliAucR9R\\\\\\\/EV0Z7\\\\\\\/KKkyqgPuOSvLF0ceQ0ovH2zqgvfOkS2d4Sy8D8cxyzOEZpn\\\\\\\/nmkNJ1tfb2LcmAD21WAVI\\\\\\\/uqG3vNWT\\\\\\\/kV2xLUjO7Ec+tS2wiEmNLCNwt5CMgm4JfTncG1WyIqt3MIHmVV6Q+HSNyTUWYAcCTKLby85TgKJ7yJkypC5uBJMxmQiSgSUFXLdMvFBW4O+TdCR9KIb3+xZ\\\\\\\/8rXhtIvRG6BmaFuujI5XLhcUhPBCHV4wM5UG6KTuqt3xm4vFefaS8eqWJ79\\\\\\\/4S+rdDUyAj9CZZVzUztknOSEw+uxGPEJnAEYyBVjD0Ptn1nAuKsY4weSBFeodYT1a+2dVTN+ngESdZD7DAyiPHihDX\\\\\\\/RRKbJSs0XJrewUVBdY3zNUuIDsSjmEqs7HcKDqasUwVvn4\\\\\\\/6L0TFdMIHbh4v+wsTAeBm4hYkJ6fiWWPqFXjNLo55NJ\\\\\\\/uaxpN+oBSC8r0zQRPZfJSy6SKjZlPKVXpcDnjq\\\\\\\/xFrZ1BbRXQlNLcAbtSll8CB3G6K+ByER5xVfpBDoB70h3K10ABisOpSSkLwMWBipa6PIG\\\\\\\/hJwcvZ9QJNI0tqQqgNNIvb\\\\\\\/AVQvaI4nLCdUtz5kLk0SnGrq3GiI4hRkhksTp+m7W3lg8PtfmnSypruPUpDq\\\\\\\/0eoieAL35KWshcMVeyHW18ptq\\\\\\\/1AuNJjCOYNGv2mnM1SOpwKtSE+HXPOXDskfpgxGAHQH\\\\\\\/Mogk\\\\\\\/JLWzHDLxce7lpxBqy1clJLwzBKoTdqPCoY0NdVZzmbE\\\\\\\/MEMoEQR9V\\\\\\\/2KdAU18gz57ujhFtAVES9o81u\\\\\\\/CsN1yPd0RH6OD1FR9Dmi2HCCSQrrZzLvnCJKf\\\\\\\/kwmJVcl\\\\\\\/AEYejlXpkjIcFDIFtUBQORhDkixRAN6a\\\\\\\/7FH6DaWdKOaOWnBTXxIEirLFYSWic3W7eQQlK7ogEyLiBoup1PaGUKf\\\\\\\/SXloeWV5DnNZlN+PnatuJoRU\\\\\\\/HJbFQ\\\\\\\/nwnfAFkTPE9fWD3WICw8PYEnRfwyw4KAazIqmgAiMNf31ZGPf1uAY9TE1B\\\\\\\/lqViJNFaNUzxTj2Yqpmt5PFH1PtPo\\\\\\\/qbJ76VL0JNvQqEBNEnMZveiSYseZw1BQw7h\\\\\\\/E2jnsIW0z00ni8EZSSpRc2PF5SIkyJBqVrBq1rkvGBwGSwuLmqqFPoTce0qkzLCw5oGFTuYCwDHlS3fxXNTTc8mFpfO9Dx1uXUPP0qy08fOYTquc5yr\\\\\\\/AqxsZ3AB1Ub29ErL0NC6plRCXgFesvgT2HedZi6m5Jls\\\\\\\/\\\\\\\/2KSjbqn1W1+YcXlhgMwX3ucpRZ+6nusNCfzpcmmCIUcIiMKpeMJcW+M6X9cgbPi4n16xSXkH47956kUjHUBetsnAT5d3bpzATOL0Mz351\\\\\\\/NoWFcJwCKCk1BCq6\\\\\\\/pblFE7Cx6EcAsJve6kKDTkpUT7+5Tr9qtdF1C6SMWsEc3q5ZjcKgsr5Qll+7ezkkamCkZNXHO7qN2oXjW21Hu7C902SIPPwrj+ca9CW7YFpRwwHMjuZE\\\\\\\/GoDyirjdRvL1lKzkHLjujLCT6UQmjJPe6Vol8wDWWpKCoTolRoIA1DGkwHZ\\\\\\\/M3deSaQFCrufNfobYZCH5bqsL6e5Bitir52NTEw7ddUkeEN4UMwdjaEPTeD4A9Abq+\\\\\\\/VxcMHljXyRBnP5jYDj9GX\\\\\\\/ZyYnWCMbWC9bvxnntgX8Vi2huX14Gl6qamD0RvoH95NqpZ\\\\\\\/L1wzcVv57zmz6S1z68lxtwXPlUDY7yDA6IFVXELddK7evfbppn1o7FeVCiHi3oyjEgekVKXPDNPN7Ow2PYilqYaiohBK1QnLTPcLXSXjS9rAivxyfbLiWT2VyuCbjxALslDmcNci0Ia+VKBsyZiAkv5jkxvCZsR8eVBj9y3+MN1yw5VTUmkpYjMKqwckjD5aEwbdtM\\\\\\\/czkNasjb+qVjluU8LnnZGjpyLiVqTP7pH2QxlLFPYK8sFI\\\\\\\/0NMhHkVTKVYJUbImdDwNaS2CIg7yOlIono8phBXt5LfZU73xGEmUqLLsl0oOBkKTbpJiauJrLlJJ18Hc2vNOizGDHIw5IT5vk5MhOPbghBCeAYf+0rgfp5OleAj2FlPQzqf+fk3oYz2DCOdyaX9Pr2Q7+GVPjhVYge2mJzAJhw8T0pWvzTV7+WBXdK4LtSmd8QSNrcUiEQJPyzAxIH0RQcyj6n6Yrb6ffgLfDnFyY9d134YkkxAlN8L4xUJynxajYg\\\\\\\/Opr3gvvxOfpJshDwcIaLqgsCwRQ\\\\\\\/THMpZS7jnr\\\\\\\/qJY1GYmfvJLIJMu8J2isvrURS4yA49A2fnLOKfnr0Onp7zxl6R6spAhsnUpnQRqCTWmsP0Es7Oj9i5np\\\\\\\/M\\\\\\\/\\\\\\\/NTqP7Eb9pZBnkZinPZmPVXQi74ZoGN1u3Adn2+Hbd3iLUQKyGUk0UORwq6aXTrJIdvlmVJ\\\\\\\/KcH6d+tno8LV7denoG6BCAQKsurjq+zjcrhCBbkkyZxymQzZAlIbTZ6iGFWFTDmzOIlvjtycbl8e21SmTAexBNA1n9s03F2vHegFbUgNG+sj6gp+ioRam2O8P2JhkdjgsMCJ9m+feYS4xXCturFwio76tU5CjwhVW0Cghhf8AjYLdcu+\\\\\\\/YsQdsZYJLPHi1WOD1mVJMW2pPtXD0VBpbBOELle1SfISbDPfrFZK\\\\\\\/OCJW41lmV3iJ3Rcbe4UlejVH98c3yRcVffPnK2AuVeik0lv3v\\\\\\\/EuN9Vr+nQAKP5d+TBwxmoWxQZZvTK4ON+tNRwcpGadMZBSBghYvKRViNT3FpL8zrX7TFqys98RwhmztTuv4oO1P9ib1DNKL592CcShjFCEFy+pzxs5j9MhyanRI3IX9HJnHpZxIGSwxfnJe3jSQwd9Gf3r0DE48DsfFO0+UZJry84KmQa9q9NgrG9fxiaFUBOHSPfXvuBdFuroo\\\\\\\/Q8TZkT452Uy2XmcrtM+NSKGM5acDrvfmJOzDcX7sMIWl7mw8QotwBLZ3BqQ6WtRwUSf1bGS5FDa3nUWmo8Bhgg5SO2K5jGvL0IkiMyiQv7wb+AqMRSd52hQwiAWBiO\\\\\\\/LN3QQHytrz22Jarr16fcNlVJvQWtKEzht6bYFuUssyEZzvHgFAztjXrJ4O8bCY6neb6XfbuP2Q9dxD3v\\\\\\\/Kkv+77F+1rIcTZPwVDWUCz9w6dX9cWmtjSjeFaKBsOtNbwypJeqqTeUIzDslPRFKthPbQXg53FU8H9XxmsnbWaz0QScDA1Oq2lDmmn7tWQNu0kYz3Q9ZDPoAGcUeddo4EXJ+Tzps9yrPgqozz5w7lx8p+YY5njTtqiZah8wf1mRO3DP90S76K9hvSGuLV\\\\\\\/+CbN082VzDJ8eM76l0MNnIY5k85JH\\\\\\\/VvUnU48Qz12Ce0nrdehXb8f8qnLhbRot4JcwccpgnzSn3x24C\\\\\\\/1BICYM\\\\\\\/4Tt8JLxChtaDsvAfgWAJfMsiePCyvOqTJGMUBeO8WhbF\\\\\\\/vJtOVdgyZDuBJn6GDF6LyvmHP8RdlGDKoFJaH\\\\\\\/YiLFKtmfxg2aXyWmf0ZRHAaNxKCQHJmpGrDzSwmPFAiCPhe6TpKO+yvqqS3ANMzzQf9Qb++ID3OQyutAMaMcIVIQcf89fdZcKLyYRGlHlFWOQo+0DnPLhHXdAK0SWM9mVEZnXKBq+JEspWaN88NMC1lr1M7oQld2C31npZENY\\\\\\\/6\\\\\\\/WC61kgerzBgzuqheWB0BAzM3Z8qLTye46WzM7xXDnuCJgN87cOEXztDXm481i+Ak0H4I4KvbImZugvlq0mdc\\\\\\\/kRUJYXGCNdYH3ctIDgGOrCcBRx8dfCTjYok24ywCKJGzlV1VGYWqRt1OAuCWSmmNGygq8Eh0GCXF1qJcsq\\\\\\\/icDrCTXs9ZeprewhzN\\\\\\\/+D5Ya6NLW2gEb6KrM+75PwBEJv3sF1CvvUY6QRi8MlxTiaGoSdz1Xn2BUwtsh3WbrqGAXQBeH3WsS+PY2Q9n6\\\\\\\/wCj4TIFNvcVgMLtwVBU0RV0YCnYoIwaIdsVkhANpShzLlA3CGGQbRF5wfz52PDFO5b2sv4XNIwOP3GHKDxvVBxd1JNcC6W5shyNx1RWy\\\\\\\/AkzEFH1Dd\\\\\\\/We0MwBPqeXrvIkh1MlDy0np1031gdH4O5CEyBJdQQH\\\\\\\/XI8AJo0IS4MU+0cd0l0AVEQJc8ocPNdJqUYrfpJzQT0QJ0mq7UUQLhtgqnAi0CbNcSpDRevhru\\\\\\\/zhhcmBbFJi0UO4LjHZDKFyP4LFtCU937DlarR\\\\\\\/AzjGtD8b6IfKBYtWfpAo1QcpDheN85IrHsyPhMS3k7KIURHhvBx8l2qYMsc1EH2szl0oru9YpIVkl7jW1dXp21fz0FlbeefiX2Dha1PNV5fSVSVRRW7d\\\\\\\/sH0GVsbrQZpjI8foP4CMlpRELoWM1gUrG55r0tHS2nyYaL1ATv9zhaoMGTbEGwTVTfs4qSoYS+fERxn4jiTidk+SkVo62feuqY7eY\\\\\\\/hz+ExSDogQ\\\\\\\/tn0Tlw\\\\\\\/Orsofv8Pcdu11sBkY3+HA3zC0LSnt4zTP+Me+5iX6tCIq3RMHy0GVaMhp2n\\\\\\\/O7T6MfgL3DFLNknAyDkM\\\\\\\/GyQNS8XkXht5SZHLJJ7+sIzqvJL4vr8SpHYVgVz3zRsfcU8IcBOxxkZTApa0CVafrtqgpWzdWF+sm7EbB5SRkWWQr4RsrBgA1YBlZb\\\\\\\/4LJR0Pk81zoPOdev2de4fodpyoJXjG0HQXx7Wl5U4yI85Q9JEenFplCc\\\\\\\/cj1v5gVX3fMEyRkWk2BNl44BpAcKYDsDz5oo4QSdjKVm1t7xlS5Ox4SKMDlWFLIbq1UNVkSed1sewAqaQPDOrgLTzM5900f1mKn+giab0rc0x75V8mFOq0AbX1MgbmDGTOmvk\\\\\\\/ZtmT90bl9Xo1sCYa7DyVLoMMccqFyd8btL+JY5t7qdhZ\\\\\\\/UTKxJwcS8E1ZWCxVA+ZfmvaaAVwG6Uo\\\\\\\/VwRArXbIVhKV+ichjJIfC6q7wNK+bbWXEaDaQ0Ch5mzEFiTV2ffVoK\\\\\\\/UuzG6NaBdurKyfpza9EgR4+iyV2NAkxrwSuP3ucHkOAKdOF1Pf+4GVMyHGrsobQCKgZBosF0Qsq4lEsEH6pGujhFn8U\\\\\\\/z\\\\\\\/sciVDE60raDGnpgL9lWltFSjQgD2\\\\\\\/d1ejZQnpm9sVQfwiwvOCRSgTu1crYcSvdPQLtKv6DYjFuo2chLzDo5fyYTKnEjKwD5sk2TvHwJLmBYQRFVcSGh\\\\\\\/eb0O22Im2pYlO\\\\\\\/vADSbQw+oGeGL3LOiZbdrrbIGNTQWdPgavpCAPserp322X9Ojja\\\\\\\/dFW2r+2ZFRfXRo\\\\\\\/BgyD4gZEzQqQGsKW6Ywkg8+NTcabujg9sy0kmkB75yRreRh7nkDV5n4CargHUpiPuqQqVwNxX0oYlfkQm143s0o8U65XbnlxjwmkOoRoJvoqEkLBXLUu3emRFygX1b5cHp1N9fv5x5KA4BOBK8cogo64xu\\\\\\\/pDC+hD9WFsvMKw8VRdciNMNxMNlQcIIjX3CoZ3hSHTL+eBp76WaTVcEWePpiKzeZHzrW6aewmuNm6C53bt5neMsQxe64+X8pXJtfd4m5FUpvE2V6QE5gNT1wLFX4D30b0vsAS4q7cpCq+KOnHiGmpu3XC8KZ4HccBkCIerBZRfRZMjdO9BgDxZ7l8rOyXxw1yKV\\\\\\\/wVT7nXnNGov1VuhziGoR7SDQ3yPT9nJaAjlH8eR2AVbtNNO5eneXC6Fwqqw9fMVDQjKD9M4d\\\\\\\/8LC+oQ7HON24U7AJeVZEn8gIvwOtoinUqaRAqJw4CykPRepSwDac4u8Ctx6INEo7JNfgGTSzZ1+ybCFQiJT9WCQG6wbUjQBsAfW2HEM6GTIokGKuXKnm2Z1yB+EXc2M66rh811buZAA6wOlbSHBS\\\\\\\/PtOXdWGXva7VqhWB9e+d8ToJnmQouGYN\\\\\\\/nciu9iCSxa1xr3uWaF1bXfrNsHUE4B8xJEVTosc61dqKC3fEylc6g+89ACaG5L9G0mZzTDFzoe6E5WNgMmSXq+EiDeLnmQkF\\\\\\\/gEnBlUsZx8fHPDMIgBcKfoow\\\\\\\/kMGdOiOoiWsLcgTqsr7zWX1ZjarpHB3KBrqnNR1dUsh\\\\\\\/dpGmjeODSTmCHxV\\\\\\\/X\\\\\\\/EfuW\\\\\\\/9BG32z4k5eC+DFPXwM9MBMUZVO8tnyKwIS3gN79WHPEcJHTFQgHksiDhwGJuUSokoaoTAKDBs1a9g26kDm+jzvHL\\\\\\\/kUgwpLWglseZDYYPD\\\\\\\/SMnjOkgFJz\\\\\\\/7fh+KkMc\\\\\\\/fvQbMQaBc8igCXsYu3bxLA7t87ifiDY2C72kzXxz7xH984uT0Xfr4dZGTZd\\\\\\\/MvzE8zNm2AJPMm5ee9BH\\\\\\\/aF8LsknUZLwNkv4BSWbBXHVuGHXNUVOaOOwey1lBLmGCrI6DHTJi91Y4LEFAAIYVNI4W0JIltQ\\\\\\\/FcLcObl66WPhV5xUE9tRmtjDr7xhjDr\\\\\\\/VIkPuL+3yosuacn\\\\\\\/EgSEB6j+E\\\\\\\/mJKJn\\\\\\\/CcfTXvYTsRaW7Bvr6DvLYMPzT3EA+luQaOn3PPSN5sGEp8ri\\\\\\\/p6udq7jZcBgBezRWPEuChSek8fPrw4NxmFa9RLuPTY3kf6AmgUFltjBOEVc\\\\\\\/t2C5a4I4Xd\\\\\\\/POXIkPMSC02M4C5R04FjihpvsBVv3AlKjESpVft3WqHoB3J8vnEKEGaq5O5z84r8ZGyx5g1DKheKWju0oKLfuA\\\\\\\/kjq3TNwp6evUhjF8TrCLNePW+o80uj23LHXHF0gRi6gLQT9BSrYiEar0ywcLROuC2b3OgG59h7b8IXi+xjF8v94SVlyAS6F2q273p2xdHYsEhzyJ3sFdLLy2WeYknfetbrn+qWHICzQHPEwhDevx8Gc32h9mc6k18gOXagDh\\\\\\\/x3DDrJeCp5Yo37E0yMaBtYXfmj0j0tqoTQMrCHqT6cMo0sqCu6j0HyEAxQkl5tTapBVF19oTFysGssz3bWC+TglHMZaNjCfcDdBkwzSanrI\\\\\\\/stIGQNiJrb+Bczp5sCKyD507iJD+3bMqEmRdgOMk131JitUA1MlRbImvsq2+wA6p7eclKgrD2PGFDD9NPhmnn3OfDSvoyw83zGgMPOjr4A\\\\\\\/g0VV195J6vDi88Dv\\\\\\\/oVqq1aL8TTUHps6r8pmuzk9rScXCqfKo+iAVbJel90LTd4+POssaS2S9pPC824wJBD+VIIh7L2e6zxuGjaxh+GUy8zUqY1UiHaMhU+cpJpxseZSokb4h+f5hIc1HDcdWKCPB9h59MbVvtQuc27EfxNWND4LbHktuwks3q+kjLbNKhEFFPtJ8pDP6\\\\\\\/KeSYIPLeK12JnPQUrZzAgmFz68LeKZBkYn9NxlDt+QLfzv6GbVX8AHfmVyJWb+duZmC2q8Q\\\\\\\/hBwR\\\\\\\/wFzia2QcjoicEo6hRp4eJRnCfflpW6CS7wenQVKVqd6E2o0g5IJOO7MHFphUDcWliQlhWRg4N0sMPi8nMiMQL8y7f8wnb1U7BRj2kPcOuX\\\\\\\/XRnCxH8SHQTgASBWberTcEjQf0Z2MtjCtgqv+xCxXjkIKg2npdyppdGzgjEQYB9sL9zNRDDNPf6E0HaSb0IYQNm\\\\\\\/72k9HCVirnFlELengYIHIHLaPZsrlHxpFG5r7IxHNahx1bDoEwkO+uL+kskZzB+KJzbX7CT\\\\\\\/DQpRUSCGyrz66NdcEiHZGfRqc2nkzrYT1r0J1HS7P9uWbpMWZxAjA7QQgl9dyaoKiMlCQeBQ0iSh5T5aNTT4F\\\\\\\/yiF1AemVOCt2JYDP2xLF2ZhfuDooWC8pck86jBaFhTL9TmqXDd6+U64qvv2T8fHXTbbB0Ro9UOPDcdEjKe1GbLeov8l0\\\\\\\/zjqqqm4RVq8qtzv0khuAIhiwQns14+jLpWj82L0wB6Gah4Q+i\\\\\\\/W0zlRY1dHyqrb2eJLkcnPgoxMPbSf+bfC8fahbyfNilSUUPyWn2DxuJdhGEXhMPxsG3JdWbjUnSnXYA9Se\\\\\\\/otg3K3fJA8IuoKaQWOV\\\\\\\/fyND5JajU6DGff5++fGj3cunnEaSXKXg\\\\\\\/892LO\\\\\\\/3MLO8NfwRk1UGXEVYc7a2vVKnU1Ry2sccTD4Fou\\\\\\\/Q8EMiWiaApjBVYpwWLORfW6qG3ODm0FnWkBWxb\\\\\\\/ghWey7Af5LehgrMn8rZPneaeCJOqeNF0jhMo3QU9Bj\\\\\\\/5Zt1PzJSN4iBzJ+iSkBvMB7cXPZ+5q\\\\\\\/OndfTWv3WBU85F1ore4ty6yd+XcvbyMBc1b7rx0KmpT+wEFXcQdY2Abxk9d8K\\\\\\\/u5QshjMr0Dy6iYMS8yBeu09LVcGTTsWgtsBPntP+sZ762Pfvp2mHTkBK5KEa1yZsGH3CKOB5KIN2IAi\\\\\\\/CcdN\\\\\\\/Ffpqn+IGBVLUa3G10Qjws8GXmJoac+vNZFQmxBVZs8chiLlUDRLGcguMatQFpL2OMVzNmp8WBBqizM3kE1za4fR245C3eODN4y3WeEEcOVYSCULVsWQHDGc5Vwmmxg7vK9Ga9Z96ag5gw\\\\\\\/iN0oZMoDaysJHGAWLeuMQwDGgSP4lpNSns+6ztjJ6VoEeQBkP4i8xcNHPLLdjSpyLojw2SFxwbQvA9BEvCT1EZj2YylD9PKU7pd017NFCO0+clm9ez7CcVBlY5NTe2vOEP\\\\\\\/Ex9rmP3qzHwOveWvNbjpTM3gKBqZBYuX6o5oQZWbRQPxkPOXFFyNc41W2K4uBOPtVvoEnBJB1Bzq6L5ikx5k\\\\\\\/5eiWYciB+y7P3R+S3nlN\\\\\\\/Vzv1K7pSywST70wrRotfWFeTzPd4znpC+r0DX4RggHND5u01SDM0um756r4TmOmQ8HckMxcZcSlA1kPJPxolCasFIyU+htX46upcXpFVybazuFBqzbH3TckEhub+Iz9ffAcDcqPnZaSE95iQDNhXZAKiFOMmsZRAXhHf5sBQIv1kSCaUj65Nw1FEOvpGUSEEemgaMpQx4vNA6vGISIpG+hV9S9wNItuMGZjYG+PazZwHo8lAT8zG60fdE3E0CXQRhzQOq\\\\\\\/Gpd9Pwl57u6\\\\\\\/w5RiNKv\\\\\\\/KYDfj2vkA2Mx6iqbAJAlgCHLXPOpoe2jtOOTW8u1TilsELT3Qs7H1t\\\\\\\/qYBbRG8K8bkY6qpONeRjhponBFc3TviUlQJ3xS7+cG2RLg64C\\\\\\\/xEeEBRXqhiFeu3\\\\\\\/vk3Ca4282hVSJIELpHuPtmBafBWOa+\\\\\\\/xhfv1alpwBbBnxHHKsTD0pMQc6JwF20oDgJfFkKk9qSNi21sjwE7gmxVvlBGbXQ2mXL9juubN+FDOPOlhYCGxl4p\\\\\\\/zUeV7Kkwo1nbUoRchNBORLpglvbPNoiWhxN9GJUhfzDgI3Xpg2AVLaKZsGc6Q+ybZ\\\\\\\/JAmsJCgilHoD7pmUKoi1MpPG2UsSEqI\\\\\\\/NcznNn+Gw72LiCT3uiv93SWjnDFyYBrJ5IabdgJ2rIXYMzlpWAH4uMd96y5yPqPzOxVpWRL0Cn+rIb5I3+M9GW0ARW1T6yEb8MDCtXQ3\\\\\\\/BARaNR7eXj5dgWG8m479tOPQY2na6lbwlnWOnICD8c+QIE0g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZLiQFSl5XDqvoR2lRbNekg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a42d7742649ca99c4207905f9b2679c723508f9cbfc9de0a165dc72f9d765db6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022phcO6hjuYPJLnjW1XOMTJVsnu4WqP\\\\\\\/cRlLd00p5tN6EUyKq07nidWnKwVBiK4Rg4yexg0wKi1NmtXhR78ZGQ8DJFYdwGEBqjeO5SRcaszeSjPJA\\\\\\\/qiednD5uLUWOGVaPYbnpy2+dX+pPxTREyrMNYM+a+bo5kjVyczy+pqL+uYiV4sHEpZOO+6doEmra3vuhriVegsBuwTcd\\\\\\\/EYZRrku65vbYC+yRKXe49mrJAln7qYQEqNgAMGzClSWe4UF1EyaSGzFfaHFWHjH4q7nutemhsaWYQV1xtoWi4NGzmZHxNSZVv43PC9l4M3WPNlLi2mp3F7xX9sVls3mJYE9d++Aaik4Jk+TlXdXJ6BQq0sIQWCl\\\\\\\/go0\\\\\\\/Zj1KCSYy\\\\\\\/qQkJYJX9CEOolPeGqxJYiohH0cJ0cPMkM70lYC3JRXm0J7rHR\\\\\\\/9JHPMPgV6R9L4sQmfFrlRidDS0nw7lq0mm0e89yCWxRIYyc\\\\\\\/aPuJ2oiQSfO4i6hSBn9sAXaud0562go5moEwdrPfHmCinN7ReDiGdXCAe02UzJ9Pc+WbVH+6irLikXf8SMCIrUanYwnCP5rR7qEN2PPyPug75EmsODlJgwoB\\\\\\\/aflLxhaphas9VBngZH1GG+\\\\\\\/QByLvc5TxEch2IRFb7exKKiqkiePeFAHkwi+gnLpo8n\\\\\\\/kIykDT1uorjah1Nnc8Fm5tv\\\\\\\/SYHQmtMfavQS8I\\\\\\\/PUwmg2b2DEt+m93r0PgEQ+huRocpzvw9KOXsvBvV59F8U6EU45r0zFf2\\\\\\\/p1Yp5TpMnAiGrT\\\\\\\/J9EQBMZBnaggrFApIkyrF5GAUhWY5mUisz9xDgE8OC8p60SiuE44vYSgUx5pzIDTWKPgvMwb38XyUJ4ssU1kaMpDzCv9SyIov3DQMkPFDIKzrEYQwnA7CxRRDRac\\\\\\\/T37KJHP3SACq55so3+dLg3daew98XDXx6+fazQqUx44A1dNOETXtO5vQwjJIH\\\\\\\/VjXlMgIAVzq6UmzZN0kzl+hfAq2BnU\\\\\\\/U35TR9yjds0rgcq6h6eEdTKbFq9yVL1eaOczC8chI4z01B3bwRYzLD4Jz\\\\\\\/yt9j7PONnIXdw3mr7WxP3\\\\\\\/oSowatXYElUU1GI5eGCoWv26nldJSj46jQ2gWzZj\\\\\\\/dA7pB1fi2NGjNSWIvcxMVur6MME32tQKQSW58VxWmSUENRB0kOEcdUFaxk7KEsETrQgRswVrWUx1AII4RUN6FjNZCbSEEETpJ0uWaJPUWwNV0g86p93zQlJPjwO0CfvqT9Yw71fLqJ376nZ1XlkMzXQrdVxdf8KZkB0jXSERBbBhOijWk6nRrvLmMUAols4MG+Libm0c9WFerPBpwR56TAiYIGGSnE2h8tIG7Cbs6RS3wHphdNU++aoPE5wZZqvRqHfx0rAM+7Rn27smpjlcfztF94M2a03AFJZni+WuTbUJxfNRO3dx0PYZEHJXxWdmB+PfhKFIiN1MmRFo4yiwXqHrRAr2D08yHR8FyQWsQivuizqDCd3SiXpQi5+Bgou5Dcq7aEa4tlAgUrMr676PG+B5vtU4D5Hy3oAPHPlj4j\\\\\\\/OLM9fs8ezsWhoKT1HU9iOrm0hIOzvdb0g22uOmSWGjQbdiZfFAifOyn+X1LR6qJLtoSisHEm6MFCS95jWq4XDOHkGLa4qTEdZjAP1nCU9L3NX297akFoEKssNayj2ivyYrciunhAbnyTxjlhfm4n+idLYCfUHXFSYA0D5SYnXia82gSVuKjTNSGKCcTK0o5FzgWjuVsZ+TfjzmWkIqoPVPHytupvpXPkQDPLZ5SzmTo9+1bMWT5EW9dMZ4pXcrBhr86NZrITR+HdDJ5zudLIYe\\\\\\\/4f1RRqYFAIOl8dXhqAHUWo\\\\\\\/GNm2oJemgcgq6e\\\\\\\/sSCrWpzWWSW4poYFL427d0MrrvZz62iFPqxhqAt+n2q+mQtRPjmgK78NvJQ7Gpu0QO9yuSJOZcbPJsCQW\\\\\\\/CIl+2X6CsrE8s1LvcyQi5jF9o+aR1TZioiA748k0hUFoe5m\\\\\\\/4hr\\\\\\\/73d4dY6SAWcJTA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/QS9jxWmnmszhXo3MtXVqQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aef4099ad3853b3b876581d4d4588e4723da73da62b98c66767084e49e40acb5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459402 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/167" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XtkexOvUms3Rgo2WPwU9F7IKULicEWRO65P9H3sqVP8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/167" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-f6QF61zbFTF_Yk6Z0ez19z620dwYQ_NhpQeVHHOAdEI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/153" title="Fosmid-Based Structure-Function Analysis Reveals Functionally Distinct Domains in the Cytoplasmic Domain of Drosophila Crumbs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/181" title="Redundant Canonical and Noncanonical Caenorhabditis elegans p21-Activated Kinase Signaling Governs Distal Tip Cell Migrations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459402" id="node459402" data-pisa="ggg;3/2/167" data-pisa-master="ggg;3/2/167" data-apath="/ggg/3/2/167.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles E. Kubicek</span>, <span class="highwire-citation-author" data-delta="1">Robert D. Chisholm</span>, <span class="highwire-citation-author" data-delta="2">Sachiko Takayama</span> and <span class="highwire-citation-author" data-delta="3">Diane K. Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">167-180;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004531 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459402" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles E. Kubicek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BE.%2BKubicek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kubicek%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BE.%2BKubicek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert D. Chisholm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BD.%2BChisholm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chisholm%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BD.%2BChisholm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sachiko Takayama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sachiko%2BTakayama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takayama%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASachiko%2BTakayama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diane K. Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BK.%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20DK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BK.%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhawley@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">dhawley@uoregon.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/2/167" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459402" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/167.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459402" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/167.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459402" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/2/167.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459402" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/2/167.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459402" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Transcription termination by RNA polymerase (Pol) II is an essential but poorly understood process. In eukaryotic nuclei, the 3′ ends of mRNAs are generated by cleavage and polyadenylation, and the same sequence elements that specify that process are required for downstream release of the polymerase from the DNA. Although Pol II is known to bind proteins required for both events, few studies have focused on Pol II mutations as a means to uncover the mechanisms that couple polyadenylation and termination. We performed a genetic screen in the yeast <em>Saccharomyces cerevisiae</em> to isolate mutations in the N-terminal half of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a>, the second largest Pol II subunit, that conferred either a decreased or increased response to a well-characterized poly(A) site. Most of the mutant alleles encoded substitutions affecting either surface residues or conserved active site amino acids at positions important for termination by other RNA polymerases. Reverse transcription polymerase chain reaction experiments revealed that transcript cleavage at the poly(A) site was impaired in both classes of increased readthrough mutants. Transcription into downstream sequences beyond where termination normally occurs was also probed. Although most of the tested readthrough mutants showed a reduction in termination concomitant with the reduced poly(A) usage, these processes were uncoupled in at least one mutant strain. Several <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles were found to be similar or identical to published mutants associated with defective TFIIF function. Tests of these and additional mutations known to impair <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a>−TFIIF interactions revealed similar decreased readthrough phenotypes, suggesting that TFIIF may have a role in 3′ end formation and termination.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polyadenylation" class="hw-term hw-article-keyword hw-article-keyword-polyadenylation" rel="nofollow">polyadenylation</a></li><li class="kwd"><a href="/keyword/eukaryotic-transcription" class="hw-term hw-article-keyword hw-article-keyword-eukaryotic-transcription" rel="nofollow">eukaryotic transcription</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/rpb2-gene-mutations" class="hw-term hw-article-keyword hw-article-keyword-rpb2-gene-mutations" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">rpb2</em> gene mutations</a></li></ul><p id="p-5">Programmed transcription termination—the dissociation of RNA polymerase (RNAP) from the DNA template and nascent RNA in response to encoded signals—is necessary to confine elongation complexes to a single transcription unit, prevent interference with downstream gene expression, and recycle the polymerases (reviewed in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gilmour and Fan 2008</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Richard and Manley 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Peters <em>et al.</em> 2011</a>). For bacterial RNAPs, transcription termination also is responsible for creating the 3′ ends of mRNAs. In contrast, the 3′ ends of eukaryotic nuclear mRNAs, which are synthesized by RNA polymerase (Pol) II, are primarily generated by internal cleavage of the nascent transcript, followed by the addition of a poly(A) tail.</p><p id="p-6">Investigation of Pol II termination has shown that polyadenylation and termination are functionally coupled and share required proteins and nucleic acid sequences (reviewed in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bentley 2005</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buratowski 2005</a>). Cleavage and poly(A) addition are directed by positioning and efficiency elements located upstream and downstream of the poly(A) site (reviewed in <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zhao <em>et al.</em> 1999</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Richard and Manley 2009</a>). These same nucleic acid sequences also are required for dissociation of Pol II from the template, which occurs at multiple positions that can be hundreds of base pairs downstream of the poly(A) site.</p><p id="p-7">Two general classes of models have been proposed to explain how 3′ end processing signals are transmitted to Pol II to induce termination. The first, the “antiterminator” or “allosteric” model, proposes that the set of accessory proteins bound to Pol II is changed upon passage of the elongation complex through polyadenylation-specifying sequences (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Logan <em>et al.</em> 1987</a>). The second model, often called the “torpedo” mechanism, suggests that cleavage of the transcript generates an unprotected (<em>i.e.</em>, uncapped) 5′ end, which allows entry of a termination protein (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Connelly and Manley 1988</a>).</p><p id="p-8">The two models are not mutually exclusive. Indeed, both have some experimental support, and neither appears sufficient to explain all 3′ end processing and termination events (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Buratowski 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Luo <em>et al.</em> 2006</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Richard and Manley 2009</a>). The torpedo model gained support with the discovery of a 5′-3′ exonuclease important to termination in yeast and mammals (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kim <em>et al.</em> 2004</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">West <em>et al.</em> 2004</a>). However, experiments <em>in vitro</em> have suggested that degradation of the RNA by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005574">Rat1</a>, the exonuclease implicated in termination in yeast, may not be sufficient for disassembly of the ternary elongation complex (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dengl and Cramer 2009</a>).</p><p id="p-9">Regardless of the mechanistic details, the models share the common feature that accessory proteins must associate with the nascent RNA, the RNAP, or both to bring about termination. Consistent with that idea, a number of proteins required for both polyadenylation and termination in yeast bind to the C-terminal domain (CTD) of the largest Pol II subunit, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002299">Rpb1</a> (reviewed in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bentley 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kuehner <em>et al.</em> 2011</a>). The CTD consists of many tandem repeats of the heptapeptide YSPTSPS. Changes in the phosphorylation state of these residues at different stages of the transcription cycle affect the ability of Pol II to associate with other proteins, including various RNA processing factors (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Buratowski 2005</a>). These observations suggest a mechanism for recruitment of proteins required for termination or the loss of proteins required for processivity, as predicted by the antiterminator model and possibly also required as a component of the torpedo mechanism.</p><p id="p-10">Much more mechanistic detail is known about transcription termination by other multisubunit RNAPs. For example, intrinsic termination by <em>Escherichia coli</em> RNAP requires a hairpin structure in the nascent RNA directly upstream of a stretch of uridines (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">von Hippel 1998</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Peters <em>et al.</em> 2011</a>). The hairpin promotes melting of the upstream edge of the weak DNA:RNA hybrid, facilitating dissociation of the remaining rU:dA base pairs and collapse of the transcription bubble (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gusarov and Nudler 1999</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Komissarova <em>et al.</em> 2002</a>). Termination by yeast Pol III appears to be even simpler, requiring a run of multiple adenosines in the template DNA but possibly independent of accessory proteins (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Richard and Manley 2009</a>).</p><p id="p-11">Mutations that increase or decrease the response of <em>E. coli</em> RNAP to intrinsic terminators have been isolated in the <em>rpoB</em> and <em>rpoC</em> genes that encode the two largest subunits, β and β’, respectively (<em>e.g.</em>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Landick <em>et al.</em> 1990</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Weilbaecher <em>et al.</em> 1994</a>; reviewed in <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Trinh <em>et al.</em> 2006</a>). In most cases, the affected residues were in regions of strong sequence homology to other prokaryotic and eukaryotic multisubunit RNAPs, suggesting that some general features of transcription termination are shared among these enzymes, even though the detailed mechanisms vary. Consistent with that idea, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Shaaban <em>et al.</em> 1995</a> isolated termination-altering mutations in the second largest subunit of yeast RNA polymerase III (Pol III) by specifically targeting conserved areas shown to be important for <em>E</em><em>. coli</em> RNAP termination.</p><p id="p-12">In several studies investigators have demonstrated phenotypes consistent with termination defects for mutant alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002299">RPB1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em>, the genes encoding the first and second largest subunits of yeast Pol II. (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cui and Denis 2003</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kaplan <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kaplan <em>et al.</em> 2012</a>). In addition, mutations in the Rbp3 and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005365">Rpb11</a> subunits of yeast Pol II were obtained in an untargeted screen for increased terminator readthrough mutants (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Steinmetz <em>et al.</em> 2006</a>). However, a genetic screen specifically designed to isolate termination-altering mutations of Pol II has not yet been reported. To gain further insight into the role of Pol II in coupling polyadenylation to termination, we conducted such a screen and isolated mutants that showed an aberrant response to a well-characterized polyadenylation-dependent termination signal in <em>Saccharomyces cerevisiae</em>. We targeted the mutations to the upstream half of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> because the N-terminal portion of the Rbp2 subunit contains several regions of high sequence and structural similarity shown to be important for termination in other RNAPs, as well as fairly extensive regions that are conserved in but unique to eukaryotic Pol II enzymes (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sweetser <em>et al.</em> 1987</a>). We describe the identification and initial characterization of 38 mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles that confer either a decreased or increased response to one or more termination sites.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and plasmids</h3><p id="p-13">Standard techniques and media (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ausubel <em>et al.</em> 1988</a>) were used for the yeast strains, which were derivatives of Research Genetics strain BY4742 (<em>MAT</em>α <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ<em>1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a></em>Δ<em>0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a></em>Δ<em>0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em>Δ<em>0</em>). DHY268 (BY4742 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>ΔFA <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a>Δ</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> [pRP212]) was the background strain used for the initial screen and DHY349 (DHY268 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>-100 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">cup1</a>Δ</em>::<em>HYG</em>) for most of the experiments characterizing the mutant phenotypes.</p><p id="p-14">pRP212 and pRP214 are <em>CEN</em>-based plasmids containing a wild-type copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> and a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> marker, respectively [gift from Richard Young, MIT (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Scafe <em>et al.</em> 1990b</a>)]. pRP214BX is a derivative of pRP214 that contains <em>BamH</em>I and <em>Xma</em>I restriction sites engineered into the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> open reading frame by site-directed mutagenesis. The silent mutations altered codons 207-208 (GGT/TCC changed to GGA/TCC) and 578-579 (ACA/AGG changed to ACC/CGG).</p><p id="p-15">pL101Btrp, used to screen for termination-altering mutations, was derived from pL101 [a gift from Linda Hyman, Tulane University (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hyman <em>et al.</em> 1991</a>)]. The <em>rp51-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em>p(A)-<em>lacZ</em> fusion reporter gene on pL101, a 2µ plasmid with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> marker gene, was amplified by polymerase chain reaction (PCR) and transferred to pRS414, a <em>CEN</em>-based plasmid with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> marker. pD16trp, used as a positive control in the termination screen, was similarly modified from D16 (also from Linda Hyman) and is identical to pL101Btrp except that the reporter gene lacks the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> terminator (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hyman <em>et al.</em> 1991</a>).</p><p id="p-16">pGAC-CYC83Ftrp and pGAC-SNR13Ftrp were used to test the extent of readthrough of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminators. These <em>CEN</em>-based plasmids, in which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> copper-resistance gene is used as a reporter for readthrough, were derived from pGAC-CYC83F and pGAC-SNR13F [provided by David Brow and Eric Steinmetz, University of Wisconsin, Madison (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Steinmetz <em>et al.</em> 2001</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Steinmetz and Brow 2003</a>)] by replacing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> marker gene with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em>. These plasmids were introduced into DHY349-derived yeast strains bearing pRP214 (wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em>) or derivatives with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> mutant alleles, and the resulting strains were tested for growth on minimal media containing 150, 175, and 200 µM CuSO<sub>4</sub> (for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> terminator) or 350 and 400 µM CuSO<sub>4</sub> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminator).</p><p id="p-17">For those and other growth tests, fivefold serial dilutions of log-phase cells were spotted onto minimal and/or rich medium and incubated at 30° unless otherwise indicated. The growth was scored relative to isogenic strains containing pRP214 with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> gene. Mycophenolic acid (MPA) sensitivity was tested at 50 µM on minimal media.</p></div><div id="sec-3" class="subsection"><h3>Random mutagenesis and screening strategy</h3><p id="p-18">Random mutations were introduced into the upstream half of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> using PCR with Taq polymerase and the DHO86 and Rpb2xbr primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/004531SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/TableS1.pdf">Table S1</a>). The purified PCR product (300 ng) and 100 ng of <em>BamH</em>I-<em>Xma</em>I−digested pRP214BX were cotransformed into DHY268 harboring pL101Btrp and plated onto glucose minimal media lacking Leucine and Tryptophan (SD-LEU-TRP). Individual <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> transformants were patched to SD-LEU-TRP plates and cured of the wild-type copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> by negative selection on media containing 5-fluoroorotic acid (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boeke <em>et al.</em> 1984</a>). Surviving cells were transferred to synthetic media with galactose to induce expression of the <em>lacZ</em> reporter gene.</p><p id="p-19"><em>lacZ</em> expression was detected using an X-gal colony filter lift procedure. Patches were lifted from the plates with Whatman #5 filter paper (Sigma-Aldrich). The filters were submerged in liquid nitrogen for approximately 10 sec. Thawed filters were placed on a second filter soaked in 2 mL of X-gal Z-buffer (60 mM Na<sub>2</sub>HPO<sub>4</sub>, 40 mM NaH<sub>2</sub>PO<sub>4</sub>, 1 mM MgSO<sub>4,</sub> 10 mM KCl, pH 7.0) with 38 mM β-mercaptoethanol and 400 µg/mL X-gal (Sigma-Aldrich). Color development was monitored until the control strain with the wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> allele exhibited no further color change (generally several hours). The pRP214 derivatives that appeared to confer either increased or decreased terminator readthrough were isolated and reintroduced into yeast. Mutant alleles were sequenced if the change in <em>lacZ</em> expression was recapitulated in the reconstructed strains.</p></div><div id="sec-4" class="subsection"><h3>cDNA analysis</h3><p id="p-20">Cells were grown in rich media to saturation, then diluted to an OD<sub>600</sub> of 0.2 in 5 mL of YPGE (1% BactoYeast extract, 2% BactoPeptone, 2% glycerol, 2% ethanol) and grown to an OD<sub>600</sub> of ~1.0. Total RNA was prepared by the hot acid phenol procedure (web.mit.edu/biomicro/forms/biofabmanual.pdf). Trace DNA contamination was eliminated using the Turbo DNA-free kit (Ambion) according to the manufacturer’s instructions. A 20-µL reaction containing 1 µg of RNA and 2 pmol random 9-mer primers was incubated at 70° for 5 min, then cooled on ice for 5 min. After the addition of deoxynucleoside triphosphates and dithiothreitol (final concentrations of 0.5 mM and 100 mM, respectively) and First-Strand Buffer (Invitrogen), incubation resumed at 42° for 2 min. Moloney murine leukemia virus reverse transcriptase (Invitrogen; 200 units) was added and incubation continued at 42° for 60 min, followed by heat inactivation for 15 min at 70°. The reaction was then incubated with 5 units of RNase H for 20 min at 37° and heat inactivated for 10 min at 65°.</p><p id="p-21">Then, 2.0 µL of each cDNA reaction was used in two separate PCRs with a forward primer (BC117) and a reverse primer, either BC116 or BC130 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/TableS1.pdf">Table S1</a>), at 1 pmol each in a 50-µL reaction containing 500 mM KCl; 100 mM Tris, pH 8.9; 1.0% Triton X-100; 2.5 mM MgCl<sub>2</sub>; 0.2 mM deoxynucleoside triphosphates; and 2.5 µL of Taq DNA polymerase. PCR products were resolved on a 1.2% agarose gel containing ethidium bromide.</p><p id="p-22">In some experiments, specific primers BC118, complementary to the C-terminal portion of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a> open reading frame, and BC133, which anneals about 400 nt downstream of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> poly(A) site, were used for cDNA synthesis instead of random primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/TableS1.pdf">Table S1</a>).</p></div><div id="sec-5" class="subsection"><h3>Quantitative reverse transcription PCR (qRT-PCR)</h3><p id="p-23">RNA isolation and cDNA synthesis with random primers was as described previously. PCRs were performed in an ABI PRISM 7900HT in a total volume of 40 µL for 35 cycles, using the conditions described in (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rogatsky <em>et al.</em> 2003</a>). The primers used are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/TableS1.pdf">Table S1</a>. The generation of specific PCR products was verified by melting curve analysis and gel electrophoresis. Quantification of cDNA species was as described (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pfaffl 2001</a>). <em>P</em> values comparing the results from each strain with the wild-type strain were calculated using the paired <em>t</em>-test (pairing wild-type and mutant reactions in the same 96-well plate). The cDNA levels were analyzed for each mutant strain in at least six independent experiments beginning with growth of cells and RNA isolation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/FileS1.xlsx">File S1</a>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-24">Our screen used a well-characterized reporter construct previously used to identify and characterize <em>cis</em>-acting sequences and <em>trans</em>-acting factors that contribute to polyadenylation and termination in yeast (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hyman <em>et al.</em> 1991</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Magrath and Hyman 1999</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cui and Denis 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bucheli and Buratowski 2005</a>). This construct contains the yeast <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> polyadenylation-dependent terminator in an intron upstream of the <em>E. coli lacZ</em> gene ORF (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Because the response to the poly(A) site is not 100% efficient and must occur before the intron is spliced, yeast colonies with wild-type Pol II make a small amount of β-galactosidase and consequently appear light blue when exposed to X-gal. The desired classes of Pol II mutations that increased or decreased the frequency of readthrough of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> terminator would be expected among mutants with detectably darker blue or white colonies, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/167/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Termination screen reporter and distribution of amino acid substitutions. (A) Schematic of the termination reporter gene construct (not to scale) used in the screen (Hyman et al. 1991). (B) Distribution of amino acid substitutions associated with an increased readthrough (blue) phenotype. The N-terminal portion of Rpb2, in which mutations were introduced, is shown as a bar with different patterned intervals representing the defined structural regions (Cramer et al. 2001). These are: 1, external 1; P, protrusion; L, lobe; F, fork; and X2, external 2. The black lines below this bar indicate named regions of sequence homology among bacterial and eukaryotic RNAPs (Sweetser et al. 1987). The bar graph displays the number of mutations obtained in successive intervals of 20 amino acids. The solid bars represent amino acid substitutions that occurred either alone or in combination with another mutation in the same structural region. The striped portions denote substitutions that occurred in combination with another mutation in a different structural region. (C) Distribution of amino acid substitutions identified in rpb2 alleles with a decreased readthrough (white) phenotype. The bar graph was constructed as in (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1921951585" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Termination screen reporter and distribution of amino acid substitutions. (A) Schematic of the termination reporter gene construct (not to scale) used in the screen (Hyman &lt;em&gt;et al.&lt;/em&gt; 1991). (B) Distribution of amino acid substitutions associated with an increased readthrough (blue) phenotype. The N-terminal portion of Rpb2, in which mutations were introduced, is shown as a bar with different patterned intervals representing the defined structural regions (Cramer &lt;em&gt;et al.&lt;/em&gt; 2001). These are: 1, external 1; P, protrusion; L, lobe; F, fork; and X2, external 2. The black lines below this bar indicate named regions of sequence homology among bacterial and eukaryotic RNAPs (Sweetser &lt;em&gt;et al.&lt;/em&gt; 1987). The bar graph displays the number of mutations obtained in successive intervals of 20 amino acids. The solid bars represent amino acid substitutions that occurred either alone or in combination with another mutation in the same structural region. The striped portions denote substitutions that occurred in combination with another mutation in a different structural region. (C) Distribution of amino acid substitutions identified in &lt;em&gt;rpb2&lt;/em&gt; alleles with a decreased readthrough (white) phenotype. The bar graph was constructed as in (B).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/167/F1.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/167/F1.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/167/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/167/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462468" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-25" class="first-child">Termination screen reporter and distribution of amino acid substitutions. (A) Schematic of the termination reporter gene construct (not to scale) used in the screen (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hyman <em>et al.</em> 1991</a>). (B) Distribution of amino acid substitutions associated with an increased readthrough (blue) phenotype. The N-terminal portion of Rpb2, in which mutations were introduced, is shown as a bar with different patterned intervals representing the defined structural regions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cramer <em>et al.</em> 2001</a>). These are: 1, external 1; P, protrusion; L, lobe; F, fork; and X2, external 2. The black lines below this bar indicate named regions of sequence homology among bacterial and eukaryotic RNAPs (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Sweetser <em>et al.</em> 1987</a>). The bar graph displays the number of mutations obtained in successive intervals of 20 amino acids. The solid bars represent amino acid substitutions that occurred either alone or in combination with another mutation in the same structural region. The striped portions denote substitutions that occurred in combination with another mutation in a different structural region. (C) Distribution of amino acid substitutions identified in <em>rpb2</em> alleles with a decreased readthrough (white) phenotype. The bar graph was constructed as in (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We generated random mutations by using PCR and replaced the wild-type copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> with the mutant alleles via plasmid shuffle in a yeast strain deleted for the chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> locus (Materials and Methods). Among approximately 2000 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains tested, we identified 100 strains with either increased or decreased levels of β-galactosidase relative to wild-type cells. To verify that the mutated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles were responsible for the observed phenotypes, we isolated the plasmids from the candidate strains and reintroduced them into yeast. Upon retesting, 24 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains were confirmed to have an increased expression (blue) phenotype (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), whereas 16 displayed decreased expression (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462469/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462469/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Properties and phenotypes of <em>rpb2</em> increased readthrough (blue) strains</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462422/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462422/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Properties and phenotypes of <em>rpb2</em> decreased readthrough (white) strains</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">All but two of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> blue alleles were unique; E104G was obtained twice (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). One amino acid substitution (Q46R) occurred in two alleles with different second mutations. Construction and analysis of the corresponding single mutants confirmed that the Q46R mutation caused the blue phenotype in both of the isolated alleles (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). One position (V225) was mutated to two different amino acids, but only one of these substitutions conferred a blue phenotype as a single mutation (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462407/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Phenotypes of site-directed <em>rpb2</em> mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">There were 15 unique white mutants; two alleles were the same (Q481R; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Two substitutions (I343T and E368K) were isolated twice, in each case both as a single mutation and also in combination with additional mutations. We also isolated a different substitution at position 368 (E368G).</p><p id="p-45"><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a> shows the locations of the amino acid substitutions with respect to the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a> structural domains defined by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Cramer <em>et al.</em> (2001)</a> from the crystal structure of yeast Pol II. The great majority of the amino acid substitutions found in the blue mutants occurred in three domains: the protrusion, external 2, and the fork (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Indeed, every <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a> variant except one was affected in one or more of those domains, which together comprise only about 55% of the mutagenized area (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). Only four mutations were isolated in the lobe; of those, only one (V225M) was shown to be responsible for the blue phenotype (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">3</a>).</p><p id="p-46">In contrast, more than half of the white mutants contained at least one amino acid substitution in the lobe (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). Relatively few white mutations occurred in either the external 2 or protrusion domains, and all but two of those were accompanied by mutations in the lobe and/or fork domains. Mutations in the fork were associated with both phenotypes. Indeed, mutations at K537 were found in both a blue (K537R) and a white (K537E) allele (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">3</a>). We also found mutations affecting F581 in the external 2 domain in both blue and white alleles. Both F581 mutations were isolated in combination, so we constructed <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles containing the single mutations (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>). The mutation in the white allele, F581S, conferred a blue phenotype, demonstrating that at least one of the other two mutations in that allele was responsible for the white phenotype. The conservative mutation in the blue allele, F581L, conferred a wild-type phenotype with the <em>lacZ</em> reporter, implicating the other external 2 mutation in that allele (H572Y) as the source of the blue phenotype.</p><div id="sec-7" class="subsection"><h3>Tests for fitness and altered function at other terminators</h3><p id="p-47">Only five mutants had discernible growth defects at 30°, and white mutants were much more likely than blue mutants to be heat- or cold-sensitive (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a>). Most of the white mutants also were sensitive to MPA, a drug that inhibits <em>de novo</em> synthesis of GMP. MPA sensitivity in yeast often is associated with mutations that alter Pol II elongation activity and/or transcriptional start site selection (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Powell and Reines 1996</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Shaw and Reines 2000</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Shaw <em>et al.</em> 2001</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Desmoucelles <em>et al.</em> 2002</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kaplan <em>et al.</em> 2012</a>).</p><p id="p-48">We tested the propensity of the Pol II variants to read through two additional terminators <em>in vivo</em> using well-characterized reporter constructs containing the poly(A)-dependent <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> termination sequence and the poly(A)-independent <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> termination sequence (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Steinmetz <em>et al.</em> 2001</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Steinmetz and Brow 2003</a>). The reporter gene is an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a>:<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> fusion transcribed from a strong constitutive promoter (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Termination sequences embedded in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> intron normally prevent the reporter gene from complementing a chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">cup1</a></em> deletion, resulting in copper sensitivity (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Steinmetz <em>et al.</em> 2001</a>). We tested the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains on a range of copper concentrations so that we would be able to identify both increased and decreased sensitivity to copper (Materials and Methods).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/167/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Termination defects at the CYC1 and SNR13 terminators. (A) Schematic representation of the CUP1 reporter constructs used to investigate the generality of the termination defects caused by rpb2 mutations (Steinmetz et al. 2001; Steinmetz and Brow 2003). (B) Representative tests of the relative MPA sensitivity and CUP1 reporter expression of rpb2 yeast strains. The number of days of growth is shown. The CuSO4 concentration was 200 &#xB5;M and 400 &#xB5;M for the constructs containing the CYC1 or SNR13 terminator, respectively. The amino acid changes in the Rpb2 mutants are shown except for the triple mutant N206Y/V225E/R605G (N206Y/ in B). Blue lettering indicates mutations that resulted in a blue phenotype with the rp51-ADH2p(A)-lacZ reporter; black lettering indicates white mutants." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1921951585" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Termination defects at the &lt;em&gt;CYC1&lt;/em&gt; and &lt;em&gt;SNR13&lt;/em&gt; terminators. (A) Schematic representation of the &lt;em&gt;CUP1&lt;/em&gt; reporter constructs used to investigate the generality of the termination defects caused by &lt;em&gt;rpb2&lt;/em&gt; mutations (Steinmetz &lt;em&gt;et al.&lt;/em&gt; 2001; Steinmetz and Brow 2003). (B) Representative tests of the relative MPA sensitivity and &lt;em&gt;CUP1&lt;/em&gt; reporter expression of &lt;em&gt;rpb2&lt;/em&gt; yeast strains. The number of days of growth is shown. The CuSO&lt;sub&gt;4&lt;/sub&gt; concentration was 200 &#xB5;M and 400 &#xB5;M for the constructs containing the &lt;em&gt;CYC1&lt;/em&gt; or &lt;em&gt;SNR13&lt;/em&gt; terminator, respectively. The amino acid changes in the Rpb2 mutants are shown except for the triple mutant N206Y/V225E/R605G (N206Y/ in B). Blue lettering indicates mutations that resulted in a blue phenotype with the &lt;em&gt;rp51-ADH2&lt;/em&gt;p(A)-&lt;em&gt;lacZ reporter&lt;/em&gt;; black lettering indicates white mutants.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/167/F2.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/167/F2.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/167/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/167/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-49" class="first-child">Termination defects at the <em>CYC1</em> and <em>SNR13</em> terminators. (A) Schematic representation of the <em>CUP1</em> reporter constructs used to investigate the generality of the termination defects caused by <em>rpb2</em> mutations (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Steinmetz <em>et al.</em> 2001</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Steinmetz and Brow 2003</a>). (B) Representative tests of the relative MPA sensitivity and <em>CUP1</em> reporter expression of <em>rpb2</em> yeast strains. The number of days of growth is shown. The CuSO<sub>4</sub> concentration was 200 µM and 400 µM for the constructs containing the <em>CYC1</em> or <em>SNR13</em> terminator, respectively. The amino acid changes in the Rpb2 mutants are shown except for the triple mutant N206Y/V225E/R605G (N206Y/ in B). Blue lettering indicates mutations that resulted in a blue phenotype with the <em>rp51-ADH2</em>p(A)-<em>lacZ reporter</em>; black lettering indicates white mutants.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">We found that more than two-thirds of the blue <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains grew better on copper than the control strain when the intron contained either the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> or the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminator, consistent with enhanced readthrough of those sites (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). In general, blue strains that were less copper sensitive with one terminator showed a similar response to the other. As expected, most of the white strains grew poorly on copper compared to the wild-type strain, consistent with an enhanced response to both of the terminators (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-51">A few of the blue and white strains appeared wild-type in response to both termination sites, suggesting that their defects were specific to—or more sensitive to—a specific property of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> site. This outcome would not be surprising, as varying responses to different termination sites have also been observed for termination mutants of other RNAPs (<em>e.g.</em>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Landick <em>et al.</em> 1990</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tavormina <em>et al.</em> 1996a</a>). However, we also considered the possibility that the phenotypes of some mutant strains were not directly or solely due to aberrant behavior at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> terminator but might instead reflect a change in the level of induction of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000223">GAL10</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> promoter, the speed or efficiency of RNA splicing, or the ability of the Pol II to elongate through the <em>lacZ</em> gene.</p><p id="p-52">To test whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> terminator was required for the altered <em>lacZ</em> expression, we repeated the β-galactosidase tests using a control plasmid (pD16trp) that was identical to the reporter plasmid used in the screen, except that the intron did not contain a terminator. We tested most of the white mutants and some of the blue mutants, including those that had a wild-type response to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminators. All of the strains turned blue in this assay, and all but four did so at the same time as the strain containing wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">2</a> and data not shown). Three white <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains required about twice as long as the wild-type to turn dark blue with the reporter gene lacking the poly(A) site. However, these same strains had remained totally white after much longer assay times with the original reporter construct, showing that the poly(A) site was required for the white phenotype.</p><p id="p-53">These results support the conclusion that the blue/white phenotypes reflected an aberrant response to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> terminator for some or all of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles. However, since the intronic location of the poly(A) site in the reporter plasmid dictated that cleavage and splicing were mutually exclusive outcomes, a change in the rate of splicing remained a possible explanation, especially for the white mutants. A reduced rate of splicing could increase the time during which the poly(A) site was available for recognition and cleavage, potentially resulting in fewer transcripts that escaped premature polyadenylation (white phenotype).</p></div><div id="sec-8" class="subsection"><h3>Blue mutants show reduced cleavage efficiency at the chromosomal <em>ADH2</em> poly(A) site</h3><p id="p-54">We considered decreased efficiency of RNA cleavage at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> poly(A) site to be the most likely cause of the blue phenotype. Any scenario that allowed cleavage at the poly(A) site would prevent translation of the resulting, uncapped RNA. Therefore, other Pol II behaviors, such as enhanced elongation through <em>lacZ</em> sequences, would increase <em>lacZ</em> expression only from those transcripts that were spliced before cleavage occurred. A change in the rate of RNA splicing was also possible; increased <em>lacZ</em> expression would presumably require a faster rate of splicing to decrease the time available for poly(A) site use.</p><p id="p-55">The intronic location of the poly(A) site in the reporter construct precluded measurement of steady state levels of uncleaved transcripts. To directly examine the extent of RNA cleavage in response to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> poly(A) site, we instead monitored RNA synthesized from the chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> locus (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Total RNA was isolated from the wild-type strain and nine representative blue strains grown in media that induced the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> promoter (see <em>Materials and Methods</em>). cDNAs synthesized using random primers were amplified in two separate PCRs. The upstream PCR primer for both reactions annealed within the ORF. The downstream primer in the PCR1 reaction hybridized upstream of the stop codon, so all of the RNA species of interest would contribute to a product, regardless of whether the RNAs were properly cleaved and/or terminated. The downstream primer in the PCR2 reaction annealed more than 200 nt 3′ of the most distal sites at which polyadenylation-associated cleavage has been shown to occur (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Hyman <em>et al.</em> 1991</a>). Only cDNAs corresponding to uncleaved RNAs from elongation complexes that had escaped the normal termination interval would be amplified in that reaction.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/167/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="cDNA analysis of readthrough at the ADH2 locus. (A) A schematic view of the ADH2 locus and the expected products of the PCR reactions are shown. Total RNA isolated from strains containing the indicated rpb2 alleles was used to synthesize cDNAs from random primers. The cDNAs were then amplified in separate PCR reactions using primers corresponding to PCR products 1 and 2. (B) The products of PCR amplication of the cDNAs were electrophoresed on an agarose gel. The domains that were affected by the mutations are indicated below the gel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1921951585" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;cDNA analysis of readthrough at the &lt;em&gt;ADH2&lt;/em&gt; locus. (A) A schematic view of the &lt;em&gt;ADH2&lt;/em&gt; locus and the expected products of the PCR reactions are shown. Total RNA isolated from strains containing the indicated &lt;em&gt;rpb2&lt;/em&gt; alleles was used to synthesize cDNAs from random primers. The cDNAs were then amplified in separate PCR reactions using primers corresponding to PCR products 1 and 2. (B) The products of PCR amplication of the cDNAs were electrophoresed on an agarose gel. The domains that were affected by the mutations are indicated below the gel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/167/F3.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/167/F3.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/167/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/167/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462420" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-56" class="first-child">cDNA analysis of readthrough at the <em>ADH2</em> locus. (A) A schematic view of the <em>ADH2</em> locus and the expected products of the PCR reactions are shown. Total RNA isolated from strains containing the indicated <em>rpb2</em> alleles was used to synthesize cDNAs from random primers. The cDNAs were then amplified in separate PCR reactions using primers corresponding to PCR products 1 and 2. (B) The products of PCR amplication of the cDNAs were electrophoresed on an agarose gel. The domains that were affected by the mutations are indicated below the gel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">We tested cell lines harboring <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> blue alleles encoding substitutions in each of the different structural regions. Most strains showed a significant increase in the ratio of PCR2 to PCR1 relative to wild-type cells (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). That result both confirmed an increased steady-state level of uncleaved transcripts and also demonstrated that the aberrant behavior did not depend on features of the reporter construct (<em>e.g.</em>, the intron) that were not shared by the chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> gene. The triple mutant N206Y/V225E/R605G was a possible exception, as the PCR2 product was not as enriched relative to PCR1 as was seen for the other mutant stains. That strain also differs from the other blue mutant strains in having a pronounced growth defect (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-58">We repeated these experiments for several mutants using cDNAs synthesized from specific, rather than random, primers to eliminate the possibility that the RNA spanning the poly(A) site arose from an antisense transcript (see <em>Materials and Methods</em>). The method of cDNA priming did not change the qualitative outcome or interpretation of the PCR reactions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-9" class="subsection"><h3>Correlation between poly(A) site cleavage and termination</h3><p id="p-59">The design of primer sets used in the experiment of <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> precluded detection of RNAs that had been cleaved but not terminated or terminated without being cleaved. Therefore, that experiment did not reveal whether any of the mutations had altered the normal coupling between the polyadenylation and termination. We used qRT-PCR to address this issue by measuring separately the amount of uncleaved and readthrough transcripts from the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> gene.</p><p id="p-60">We used the primer sets shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> to monitor three cDNA regions: the ORF, the poly(A) site, and a sequence more than 300 bp downstream of the poly(A) site. In each experiment, we calculated the ratio of poly(A) site or downstream PCR product to the ORF (total RNA) product (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). Measurements of the relative PCR efficiencies indicated that all three primer sets yielded close to the same amount of PCR product (±10%) when used to amplify DNA spanning the entire region (data not shown). Therefore, the numbers on the <em>y</em>-axis are close to true ratios. There were no systematic differences among the wild-type and mutant strains in the amount of PCR fragment corresponding to the ORF, indicating that none of these mutations affected transcription initiation at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> promoter (data not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/167/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="qRT-PCR to assess cleavage and readthrough of the ADH2 terminator. (A) ADH2 cDNAs synthesized using random primers were analyzed with three sets of primers to amplify the 120-bp regions shown below the gene diagram. (B) Results of qRT-PCR are presented as a ratio of the amount of poly(A) site cDNA to the ORF cDNA product. Various symbols represent different RNA preparations; the same symbol is used for qRT-PCRs performed in the same 96-well plate. Horizontal bars indicate averages of the 6 or more experiments for each strain. P values &#x2264; 0.1 are indicated. (C) Same as in B, except that downstream cDNA is compared with the ORF cDNA. (D) Same as in B, except that the downstream cDNA is compared to the poly(A) site cDNA in each experiment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1921951585" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;qRT-PCR to assess cleavage and readthrough of the &lt;em&gt;ADH2&lt;/em&gt; terminator. (A) &lt;em&gt;ADH2&lt;/em&gt; cDNAs synthesized using random primers were analyzed with three sets of primers to amplify the 120-bp regions shown below the gene diagram. (B) Results of qRT-PCR are presented as a ratio of the amount of poly(A) site cDNA to the ORF cDNA product. Various symbols represent different RNA preparations; the same symbol is used for qRT-PCRs performed in the same 96-well plate. Horizontal bars indicate averages of the 6 or more experiments for each strain. P values &#x2264; 0.1 are indicated. (C) Same as in B, except that downstream cDNA is compared with the ORF cDNA. (D) Same as in B, except that the downstream cDNA is compared to the poly(A) site cDNA in each experiment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/167/F4.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/167/F4.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/167/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/167/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462333" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-61" class="first-child">qRT-PCR to assess cleavage and readthrough of the <em>ADH2</em> terminator. (A) <em>ADH2</em> cDNAs synthesized using random primers were analyzed with three sets of primers to amplify the 120-bp regions shown below the gene diagram. (B) Results of qRT-PCR are presented as a ratio of the amount of poly(A) site cDNA to the ORF cDNA product. Various symbols represent different RNA preparations; the same symbol is used for qRT-PCRs performed in the same 96-well plate. Horizontal bars indicate averages of the 6 or more experiments for each strain. P values ≤ 0.1 are indicated. (C) Same as in B, except that downstream cDNA is compared with the ORF cDNA. (D) Same as in B, except that the downstream cDNA is compared to the poly(A) site cDNA in each experiment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">The steady-state accumulation of uncleaved RNAs is shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>. For the wild-type strain, approximately 0.3% of the transcripts containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> ORF were uncleaved at the poly(A) site. The average amount of poly(A) fragment was slightly increased over the wild type for all of the mutants, although in most cases the difference was just outside what is normally considered statistically significant (<em>P</em> &lt; 0.05). The highest ratio—just greater than twofold when the average value was compared with wild-type—was observed for the S2P/D66N mutant. The modest increases in uncleaved poly(A) site RNA are consistent with expectation, because only one blue mutant (N206Y/V225E/R605G) had a severe fitness defect at 30° (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>). We noted that the distribution of values among different experiments was much greater for some of the mutants than for wild type, possibly because some of the Pol II mutations had stochastic effects (<em>e.g.</em>, by influencing the steady state levels of polyadenylation machinery components).</p><p id="p-63">The relative steady state levels of the downstream fragment are shown in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>. The results of this comparison were more varied, with several mutants appearing similar to wild-type, several with a higher accumulation of this fragment (<em>e.g.</em>, S2P/D66N), and one (N206Y/V225E/R605G) with a significantly reduced accumulation. The result suggested that one or more of the mutations had affected the correlation between uncleaved RNA and readthrough into downstream sequences. To make that direct comparison, we plotted the ratio of downstream fragment to uncleaved poly(A) site fragment (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>). For the wild-type strain, that value was about 0.8, consistent with models suggesting that cleavage is required for termination. For half of the tested mutants, this ratio was statistically the same as for the wild-type strain. For one mutant strain, N206Y/V225E/R605G, we observed a highly significant decrease in the ratio. This result suggested that poly(A) site usage and downstream termination were at least partially uncoupled in this strain, in that the reduced efficiency of poly(A) site usage did not result in increased accumulation of downstream RNA. It is possible that the reduced fitness of this strain is related to this unique behavior.</p><p id="p-64">A smaller—but possibly significant—decrease in the ratio of downstream:uncleaved RNA was noted for three additional mutant strains (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>). Although the <em>P</em> values were greater for these strains than for the triple mutant strain (0.08 and 0.09 compared with 0.001), they are strikingly different from the <em>P</em> values determined for the rest of the tested mutants (&gt;0.8). Interestingly, all of these mutants, including the triple mutant, have substitutions in the external 2 domain.</p></div><div id="sec-10" class="subsection"><h3>Mutations in phylogenetically conserved residues</h3><p id="p-65">The mutagenized portion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">RPB2</a></em> contains four regions (homology blocks A-D) in which the sequences are highly conserved in all multisubunit RNAPs (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). Residues within these conserved regions closely approach nucleic acids in the elongation complex (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). We isolated a number of mutations that altered residues in and around all of the homology blocks except C (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a> shows the mutations located in the fork domain, in and immediately adjacent to homology block D. Many mutations that alter the termination behavior of <em>E. coli</em> RNAP and yeast RNA polymerase III have previously been isolated in the same region (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/167/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Amino acid substitutions in phylogenetically conserved regions. (A) Amino acid sequences are shown for a portion of the fork domain of S. cerevisiae Rpb2 (YII) and the corresponding sequences of the Ret1 and &#x3B2; subunits from S. cerevisiae RNA polymerase III (YIII) and E. coli (Eco), respectively. Shading indicates amino acids that are identical in at least two of the three aligned sequences. The thick line below the sequences indicates residues within this interval that are part of homology block D (Sweetser et al. 1987). rpb2 substitutions identified in this study are shown above the alignment; the dotted line indicates a mutation that has been tested only in combination with an additional substitution. Underlining indicates positions at which termination-altering mutations were isolated for Rpb2 (this study), Ret1 (Shaaban et al. 1995), and the &#x3B2; subunit (Jin et al. 1988; Landick et al. 1990; Tavormina et al. 1996a). Italics with wavy underlining indicate residues mutated in increased readthrough variants, whereas bold-faced type with straight underlining indicate decreased readthrough variants. One fork mutation, affecting E468 in fork loop 1 (Table 2), is not shown. (B) Mutations affecting homology region B. The notation is as in (A). The double-underlined residue E142 in the E. coli sequence was identified as a second site suppressor of an increased termination mutant (Tavormina et al. 1996b). (C) Mutations affecting homology region A. The R120C mutation was originally isolated as the heat- and cold-sensitive rpb2-7 allele (Scafe et al. 1990a). (D) Amino acid sequences are shown for the N-terminal regions with the same notation as in (A). Rpb2 single and double substitutions isolated within this interval are shown above the sequence. The two underlined residues in the E. coli sequence were mutated in a recessive lethal allele of E. coli rpoB associated with enhanced readthrough of some terminators (Tavormina et al. 1996a). The thick line below the sequences shows the region of homology defined by Lane and Darst (2010)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1921951585" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Amino acid substitutions in phylogenetically conserved regions. (A) Amino acid sequences are shown for a portion of the fork domain of &lt;em&gt;S. cerevisiae&lt;/em&gt; Rpb2 (YII) and the corresponding sequences of the Ret1 and &#x3B2; subunits from &lt;em&gt;S. cerevisiae&lt;/em&gt; RNA polymerase III (YIII) and &lt;em&gt;E. coli&lt;/em&gt; (Eco), respectively. Shading indicates amino acids that are identical in at least two of the three aligned sequences. The thick line below the sequences indicates residues within this interval that are part of homology block D (Sweetser &lt;em&gt;et al.&lt;/em&gt; 1987). &lt;em&gt;rpb2&lt;/em&gt; substitutions identified in this study are shown above the alignment; the dotted line indicates a mutation that has been tested only in combination with an additional substitution. Underlining indicates positions at which termination-altering mutations were isolated for Rpb2 (this study), Ret1 (Shaaban &lt;em&gt;et al.&lt;/em&gt; 1995), and the &#x3B2; subunit (Jin &lt;em&gt;et al.&lt;/em&gt; 1988; Landick &lt;em&gt;et al.&lt;/em&gt; 1990; Tavormina &lt;em&gt;et al.&lt;/em&gt; 1996a). Italics with wavy underlining indicate residues mutated in increased readthrough variants, whereas bold-faced type with straight underlining indicate decreased readthrough variants. One fork mutation, affecting E468 in fork loop 1 (Table 2), is not shown. (B) Mutations affecting homology region B. The notation is as in (A). The double-underlined residue E142 in the &lt;em&gt;E. coli&lt;/em&gt; sequence was identified as a second site suppressor of an increased termination mutant (Tavormina &lt;em&gt;et al.&lt;/em&gt; 1996b). (C) Mutations affecting homology region A. The R120C mutation was originally isolated as the heat- and cold-sensitive &lt;em&gt;rpb2-7&lt;/em&gt; allele (Scafe &lt;em&gt;et al.&lt;/em&gt; 1990a). (D) Amino acid sequences are shown for the N-terminal regions with the same notation as in (A). Rpb2 single and double substitutions isolated within this interval are shown above the sequence. The two underlined residues in the &lt;em&gt;E. coli&lt;/em&gt; sequence were mutated in a recessive lethal allele of &lt;em&gt;E. coli rpoB&lt;/em&gt; associated with enhanced readthrough of some terminators (Tavormina &lt;em&gt;et al.&lt;/em&gt; 1996a). The thick line below the sequences shows the region of homology defined by Lane and Darst (2010).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/167/F5.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/167/F5.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/167/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/167/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-66" class="first-child">Amino acid substitutions in phylogenetically conserved regions. (A) Amino acid sequences are shown for a portion of the fork domain of <em>S. cerevisiae</em> Rpb2 (YII) and the corresponding sequences of the Ret1 and β subunits from <em>S. cerevisiae</em> RNA polymerase III (YIII) and <em>E. coli</em> (Eco), respectively. Shading indicates amino acids that are identical in at least two of the three aligned sequences. The thick line below the sequences indicates residues within this interval that are part of homology block D (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Sweetser <em>et al.</em> 1987</a>). <em>rpb2</em> substitutions identified in this study are shown above the alignment; the dotted line indicates a mutation that has been tested only in combination with an additional substitution. Underlining indicates positions at which termination-altering mutations were isolated for Rpb2 (this study), Ret1 (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Shaaban <em>et al.</em> 1995</a>), and the β subunit (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jin <em>et al.</em> 1988</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Landick <em>et al.</em> 1990</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Tavormina <em>et al.</em> 1996a</a>). Italics with wavy underlining indicate residues mutated in increased readthrough variants, whereas bold-faced type with straight underlining indicate decreased readthrough variants. One fork mutation, affecting E468 in fork loop 1 (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>), is not shown. (B) Mutations affecting homology region B. The notation is as in (A). The double-underlined residue E142 in the <em>E. coli</em> sequence was identified as a second site suppressor of an increased termination mutant (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tavormina <em>et al.</em> 1996b</a>). (C) Mutations affecting homology region A. The R120C mutation was originally isolated as the heat- and cold-sensitive <em>rpb2-7</em> allele (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Scafe <em>et al.</em> 1990a</a>). (D) Amino acid sequences are shown for the N-terminal regions with the same notation as in (A). Rpb2 single and double substitutions isolated within this interval are shown above the sequence. The two underlined residues in the <em>E. coli</em> sequence were mutated in a recessive lethal allele of <em>E. coli rpoB</em> associated with enhanced readthrough of some terminators (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Tavormina <em>et al.</em> 1996a</a>). The thick line below the sequences shows the region of homology defined by <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lane and Darst (2010)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/2/167/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Location of mutated residues in the Pol II structure. (A) Mutated residues located close to the DNA:RNA hybrid in the crystal structure of a Pol II elongation complex are shown (carbon, gray; nitrogen, blue; oxygen, red). Homology regions A, B, and D are depicted as teal, orange, and violet ribbons. RNA and DNA are shown in green and red, respectively. The active site Mg++ is depicted as a magenta sphere. All of the mutated residues were associated with blue alleles, except for Q481 (white) and K537 (both blue and white). This figure was created from pdb file 1I6H using PyMOL (DeLano Scientific). (B) The residues of Rbp2 are shown in tan, except for the residues that closely approached TFIIF in the PIC, as determined by Hahn and colleagues (Chen et al. 2007, Eichner et al. 2010), which are colored cyan. The Rpb2 positions indicated in green were found to crosslink to TFIIF (Chen et al. 2007). Surface residues mutated in Rpb2 variants that increased or decreased readthrough of the ADH2 terminator are shown in blue and brown, respectively. Surface residues in Rpb3 and Rpb11 that were identified in a separate study of Pol II termination mutants (Steinmetz et al. 2006) are red. The rest of the Pol II subunits are gray. The view on the right is oriented so that part of the DNA:RNA hybrid in the active site is visible; the RNA is orange and DNA is yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1921951585" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Location of mutated residues in the Pol II structure. (A) Mutated residues located close to the DNA:RNA hybrid in the crystal structure of a Pol II elongation complex are shown (carbon, gray; nitrogen, blue; oxygen, red). Homology regions A, B, and D are depicted as teal, orange, and violet ribbons. RNA and DNA are shown in green and red, respectively. The active site Mg&lt;sup&gt;++&lt;/sup&gt; is depicted as a magenta sphere. All of the mutated residues were associated with blue alleles, except for Q481 (white) and K537 (both blue and white). This figure was created from pdb file 1I6H using PyMOL (DeLano Scientific). (B) The residues of Rbp2 are shown in tan, except for the residues that closely approached TFIIF in the PIC, as determined by Hahn and colleagues (Chen &lt;em&gt;et al.&lt;/em&gt; 2007, Eichner &lt;em&gt;et al.&lt;/em&gt; 2010), which are colored cyan. The Rpb2 positions indicated in green were found to crosslink to TFIIF (Chen &lt;em&gt;et al.&lt;/em&gt; 2007). Surface residues mutated in Rpb2 variants that increased or decreased readthrough of the &lt;em&gt;ADH2&lt;/em&gt; terminator are shown in blue and brown, respectively. Surface residues in Rpb3 and Rpb11 that were identified in a separate study of Pol II termination mutants (Steinmetz &lt;em&gt;et al.&lt;/em&gt; 2006) are red. The rest of the Pol II subunits are gray. The view on the right is oriented so that part of the DNA:RNA hybrid in the active site is visible; the RNA is orange and DNA is yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/2/167/F6.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/2/167/F6.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/2/167/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/2/167/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462451" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-67" class="first-child">Location of mutated residues in the Pol II structure. (A) Mutated residues located close to the DNA:RNA hybrid in the crystal structure of a Pol II elongation complex are shown (carbon, gray; nitrogen, blue; oxygen, red). Homology regions A, B, and D are depicted as teal, orange, and violet ribbons. RNA and DNA are shown in green and red, respectively. The active site Mg<sup>++</sup> is depicted as a magenta sphere. All of the mutated residues were associated with blue alleles, except for Q481 (white) and K537 (both blue and white). This figure was created from pdb file 1I6H using PyMOL (DeLano Scientific). (B) The residues of Rbp2 are shown in tan, except for the residues that closely approached TFIIF in the PIC, as determined by Hahn and colleagues (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2007</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eichner <em>et al.</em> 2010</a>), which are colored cyan. The Rpb2 positions indicated in green were found to crosslink to TFIIF (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2007</a>). Surface residues mutated in Rpb2 variants that increased or decreased readthrough of the <em>ADH2</em> terminator are shown in blue and brown, respectively. Surface residues in Rpb3 and Rpb11 that were identified in a separate study of Pol II termination mutants (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Steinmetz <em>et al.</em> 2006</a>) are red. The rest of the Pol II subunits are gray. The view on the right is oriented so that part of the DNA:RNA hybrid in the active site is visible; the RNA is orange and DNA is yellow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">We also isolated mutations in two residues that are among the most highly conserved positions in homology region B (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5B</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">6A</a>). Both mutations, I205V and N206Y, were recovered in combination with other mutations. To determine whether the B region mutations were responsible for the termination phenotype, we used site-directed mutagenesis to create <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles containing the individual changes. The N206Y mutant had a blue phenotype, like the triply mutant strain, whereas both the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em>-V225E and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em>-R605G strains expressed the <em>lacZ</em> reporter at wild-type levels (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>). The finding that the V225E mutation did not confer a blue phenotype was interesting, since another mutation at the same position, V225M, did (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>). Similarly, the R605G mutation did not, by itself, confer a phenotype with the <em>lacZ</em> reporter, although L603S did (<a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a>). One or both of these mutations had to have contributed to the growth defect of the triple mutant, since that property was not shared by any of the singly mutant strains (<a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>). It is likely, therefore, that one or both of these mutations also enhanced the excess readthrough defect caused by the N206Y mutation.</p><p id="p-69">The I205V mutation was isolated in combination with a second mutation (G127D) that altered a highly conserved residue in homology region A (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5C</a>). Construction and testing of the two single mutants showed that both alleles caused a blue phenotype (<a id="xref-table-wrap-3-11" class="xref-table" href="#T3">Table 3</a>). Besides G127D, only one other yeast <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> region A mutation has been reported, R120C, which was isolated in the Young laboratory in a screen for conditional mutants (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Scafe <em>et al.</em> 1990a</a>). Previous studies of that allele (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a>-7)</em> have been somewhat equivocal but have suggested weak changes in the extent of readthrough of poly(A) sites (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Cui and Denis 2003</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kaplan <em>et al.</em> 2005</a>). In our assay strain, R120C conferred a blue phenotype (<a id="xref-table-wrap-3-12" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-70">Finally, several of the blue strains had mutations affecting residues within a region of highly conserved sequence that was originally noted by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">James <em>et al.</em> 1991</a> and more recently identified in a comparison of more than 1000 bacterial, archaeal, and eukaryotic RNAP subunits (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5D</a>) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lane and Darst 2010</a>). Both S45L and Q46R were isolated in combination with other mutations. We constructed the single mutants and also an additional <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> allele containing the same substitution at the neighboring position (Q47R). Each of these three mutations caused a blue phenotype (<a id="xref-table-wrap-3-13" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-11" class="subsection"><h3>Mutations in the TFIIF binding surface of the Rpb2 lobe cause a white phenotype</h3><p id="p-71">Most of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> mutations altered residues clustered on the surface of Pol II in patches that likely coincide with binding sites for proteins involved in RNA processing and/or termination (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). We have not yet identified the proteins that interact with the presumptive binding sites identified by mutations in the protrusion and external 2 domains of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a>. However, we observed that many of the mutations isolated in the lobe domain corresponded to or were near residues reported to interact with TFIIF, an essential transcription factor with proposed functions in both initiation and elongation (reviewed in <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Shilatifard <em>et al.</em> 2003</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2010</a>).</p><p id="p-72">The Hahn laboratory has identified positions in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a> that crosslink to TFIIF when substituted with the synthetic, cross-linking residue BPA (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2007</a>). Based on that information, they mutated specific residues and assayed the ability of the mutated Pol II to interact with TFIIF when assayed by coimmunoprecipitation (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2007</a>). Two of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a> residues shown in that study to interact with TFIIF, E368 and E371, were mutated in our screen in three alleles that conferred a white phenotype (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). We also isolated mutations that altered residues that were sites of cross-linking to TFIIF (Y57, L74) or next to sites of cross-linking in the primary sequence (A75, E468).</p><p id="p-73">To determine whether alteration of the wild-type interaction between TFIIF and Pol II would cause a phenotype in our termination screen, we tested <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains containing the mutations shown by Chen <em>et al.</em> to affect TFIIF binding <em>in vitro</em> (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). All of those mutations shift transcription start sites upstream of where they occur in the wild-type strain (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2007</a>), a property also reported for yeast with TFIIF subunit mutations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ghazy <em>et al.</em> 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Freire-Picos <em>et al.</em> 2005</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Eichner <em>et al.</em> 2010</a>). Consequently, we tested two other previously reported mutations in the same area of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a> lobe: G369S, which causes a similar start site shift (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen and Hampsey 2004</a>), and G369D, which was isolated in a screen for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> strains with altered transcription initiation start sites (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hekmatpanah and Young 1991</a>). A second mutation isolated in that same screen, E368K, was isolated twice in our study, as well, once in combination and once as a single mutation (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462397/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462397/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Phenotypes of <em>rpb2</em> mutations affecting residues important for TFIIF binding</span><div class="sb-div caption-clear"></div></div></div><p id="p-78">All of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> mutants chosen for testing because of a demonstrated or hypothesized effect on TFIIF interactions had a white phenotype with the <em>lacZ</em> reporter (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). A subset of mutations subjected to additional tests shared other common phenotypes, including MPA sensitivity and severe growth defects on copper in assays with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> reporter constructs containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">CYC1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminators. These properties were also shared by other white strains with mutations in nearby residues of the lobe domain (<em>e.g.</em>, I343T, L361P, and F376S; <a id="xref-table-wrap-2-12" class="xref-table" href="#T2">Table 2</a>). These results suggest that mutations within this cluster of lobe residues confer a similar defect responsible for the decreased readthrough phenotypes. Based on published analyses of some of the mutants, that defect might involve an altered interaction with TFIIF.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-79">The screen reported here proved a successful strategy for isolating <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles that alter the normal response of yeast Pol II to the poly(A)-dependent <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> terminator, resulting in a collection of strains with increased or decreased readthrough phenotypes. Most of the mutant strains appeared to have mild but general termination defects, in that they also displayed similarly aberrant responses to another poly(A)-dependent site (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> terminator), a poly(A)-independent site (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminator), or both.</p><p id="p-80">Analysis of the excess readthrough (blue) mutants verified that the screen had identified Pol II residues that contributed to the efficiency of cleavage at the chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004918">ADH2</a></em> poly(A) site (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Some of the mutations also may have interfered with the normal coupling of cleavage and subsequent termination (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>). The fact that the mutations caused enhanced expression of the <em>lacZ</em> reporter is evidence that they did not also confer elongation or splicing defects, unless those activities were inappropriately enhanced.</p><p id="p-81">In contrast, the decreased readthrough (white) strains could have defects in other transcription-related processes, including splicing and elongation. We were particularly aware of the latter possibility. Despite the wide-spread use of <em>lacZ</em> as a reporter in yeast, there are potential concerns when using a bacterial gene, which might contain cryptic processing sites (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Cui and Denis 2003</a>). In addition, because of the length of the ORF (&gt; 3000 nt), <em>lacZ</em> expression might be especially sensitive to minor changes in Pol II elongation competency. However, we found that all but two of the mutants were indistinguishable from the wild-type strain in the level of expression of the <em>lacZ</em> gene when the reporter construct lacked the poly(A) site (<a id="xref-table-wrap-2-13" class="xref-table" href="#T2">Table 2</a>). Furthermore, all but three of the white strains also showed deficiencies with a different reporter gene, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a>:<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> constructs containing different yeast terminators (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-2-14" class="xref-table" href="#T2">Table 2</a>). In contrast to <em>lacZ</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> is a very short yeast gene with an ORF &lt; 200 nt. Together these results strongly support the conclusion that both the blue and white mutants showed altered termination behaviors. Possible alterations to other properties, such as splicing efficiency and transcription elongation, if they occurred, were not sufficient to elicit the observed phenotypes. However, such altered behaviors might have contributed to the aberrant response to the poly(A) site.</p><p id="p-82">A similar, although untargeted, screen for mutations causing excessive readthrough of Pol II terminators previously identified several mutations in different Pol II subunits, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005365">Rpb11</a>, the yeast homologs of the two alpha subunits of bacterial RNAP. In those experiments, Brow and colleagues used their <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a>:<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> reporter construct containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=snr13">SNR13</a></em> terminator (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>) to isolate spontaneous mutations in protein-encoding genes that conferred copper resistance (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Steinmetz <em>et al.</em> 2006</a>). The mutations altered surface exposed residues on the same side of the polymerase structure as the nearest amino acids mutated in our study but separated from them by more than 60 Å (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). It is likely, therefore, that the two studies have located binding sites for different elongation, termination, or processing factors.</p><div id="sec-13" class="subsection"><h3>Comparison with mutations affecting termination in other systems</h3><p id="p-83">In a previous screen for termination-altering mutations affecting the <em>E. coli</em> RNAP β subunit, the majority of mutations clustered in four regions, corresponding to parts of the lobe, the fork, and the hybrid-binding domain (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Landick <em>et al.</em> 1990</a>). Mutagenesis targeted to the corresponding regions of the yeast Pol III <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005733">Ret1</a> subunit also resulted in termination phenotypes (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Shaaban <em>et al.</em> 1995</a>). The portion of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">Rpb2</a> that was mutagenized in our study contained two of these regions, the lobe and the fork. We isolated mutations in both of these locations (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, B and C</a>). Most striking, all but two of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> alleles that decreased readthrough had mutations affecting the lobe or the fork (<a id="xref-table-wrap-2-15" class="xref-table" href="#T2">Table 2</a>). We also observed fork mutations, but very few lobe mutations, among the increased readthrough mutants (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a> and <a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1</a>). More than half of the fork mutations affected positions that were also mutated in termination-altering variants of either the <em>E. coli</em> β or yeast <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005733">Ret1</a> subunit (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5A</a>). The high degree of sequence and structural conservation of these active site residues suggest that they have a common function in all RNAPs and may contribute to the termination defects in similar ways, despite the different mechanisms of termination used in the three systems.</p><p id="p-84">The fork is composed of a series of loops that closely approach the DNA:RNA hybrid in the active site: fork loop 1, which is not present in bacterial RNAPs; fork loop 2, which is conserved among all multisubunit polymerases; and βD loop II, which was defined for the bacterial enzymes and includes part of the conserved D region (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Korzheva <em>et al.</em> 2000</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gnatt <em>et al.</em> 2001</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Trinh <em>et al.</em> 2006</a>). We isolated mutations in each of these loops (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5A</a>). The mobility of the fork loops and their locations within the active site have suggested various functions during elongation, including maintaining and stabilizing the transcription bubble and promoting substrate binding, catalysis, and translocation (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Trinh <em>et al.</em> 2006</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Vassylyev <em>et al.</em> 2007</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kireeva <em>et al.</em> 2011</a>).</p><p id="p-85">Biochemical analyses of bacterial and Pol III systems <em>in vitro</em> have shown that fork domain substitutions can affect both pausing and the overall rate of elongation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fisher and Yanofsky 1983</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Landick <em>et al.</em> 1990</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Shaaban <em>et al.</em> 1996</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tavormina <em>et al.</em> 1996b</a>). Abnormally long pauses and slow polymerization were generally correlated with increased termination and decreased pause times, whereas fast elongation was associated with decreased termination. The possibility that poly(A) site recognition and cleavage might also be influenced by elongation speed and/or pause duration is consistent with current knowledge of the mechanisms of these processes. Indeed, pausing downstream of the poly(A) site has been suggested to be important for both polyadenylation and subsequent Pol II termination (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gromak <em>et al.</em> 2006</a>).</p><p id="p-86">Overall polymerization rate and/or pausing are thought to contribute to termination by several mechanisms, some of which could be envisioned also to influence the efficiency of poly(A) site recognition and RNA cleavage. In prokaryotic systems, both the response to RNA sequence elements and interactions with accessory proteins are facilitated by polymerase pausing at strategic locations (reviewed in <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Landick 2006</a>). In eukaryotic cells, the binding of 3′ end processing components to the Pol II CTD facilitates the interaction of these proteins with the poly(A) site as it emerges from the RNA exit tunnel (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kuehner <em>et al.</em> 2011</a>). Elongation rate would determine both the length of time the relevant RNA sequences are in close proximity to the polymerase and also the relative timing of synthesis of the separated blocks of RNA sequence needed for assembly of the complete poly(A) processing complex. This sort of kinetic coupling contributes to the efficiency of splicing and the selection of alternative splice sites (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Muñoz <em>et al.</em> 2010</a>). Changes in elongation rate can also change the pattern of gene expression (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ip <em>et al.</em> 2011</a>), which in turn could influence the synthesis and availability of elongation, termination, and processing proteins.</p><p id="p-87">Our initial characterization <em>in vitro</em> of Pol II variants mutated in the fork domain is consistent with the hypothesis that faster elongation speed can contribute to greater readthrough (C. E. Kubicek and D. K. Hawley, unpublished data). However, the relationship may be more complicated than that simple correlation suggests because we have observed that mutations in other Pol II domains that also affect elongation rate <em>in vitro</em> do not always show the expected readthrough phenotype. The variety of observed behaviors suggest that this collection of mutants will be a valuable resource for dissecting the mechanistic relationships between elongation rate, pausing, termination, and RNA processing events.</p><p id="p-88">The finding that numerous lobe mutations were identified in our study as well as in termination screens of bacterial RNAP and yeast Pol III (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Landick <em>et al.</em> 1990</a>, <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Shaaban <em>et al.</em> 1995</a>) was initially somewhat surprising. Unlike the fork domain or the other highly conserved residues mutated in our screen, the sequence of the lobe domain is not universally conserved, with the exception of homology region C, which was not represented by a single mutation in our screen. Phenotypes associated with lobe mutations in bacteria have implied a role for that domain in establishing and maintaining the elongation bubble (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bartlett <em>et al.</em> 1998</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Trautinger and Lloyd 2002)</a>, leading Trinh <em>et al</em>. to propose that the increased termination associated with some lobe mutations may reflect an increased propensity for the elongation bubble to collapse at the terminator (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Trinh <em>et al.</em> 2006</a>).</p><p id="p-89">For both Pol II and Pol III, the termination mutants in the lobe may reflect an altered interaction with another protein. TFIIF is a candidate for that protein in the Pol II system. This conclusion is based on the preponderance of mutations that map to the previously identified TFIIF binding surface and the similar phenotypes of mutants shown to have altered interactions with TFIIF. TFIIF stimulates transcription elongation <em>in vitro</em> and has been assumed also to do so <em>in vivo</em>, although it has been difficult to verify association of TFIIF with active Pol II elongation complexes in yeast (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Krogan <em>et al.</em> 2002</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pokholok <em>et al.</em> 2002</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mayer <em>et al.</em> 2010</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rhee and Pugh 2012</a>).</p><p id="p-90">Recent work in the Pol III system may provide precedent for the hypothesis that TFIIF—or possibly another protein that interacts with the same Pol II surface—has a role in Pol II termination. A subcomplex of two polypeptides considered to be integral Pol III subunits, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001733">Rpc37</a>/53, has been proposed to be the Pol III-specific paralog of TFIIF (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kuhn <em>et al.</em> 2007</a>). Based on crosslinking experiments, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001733">Rpc37</a>/53 associates with the lobe and external 2 domains of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005733">Ret1</a> (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wu <em>et al.</em> 2011</a>) and contributes to termination (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Landrieux <em>et al.</em> 2006</a>). Interestingly, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001733">Rpc37</a>/53 and TFIIF might be expected to elicit opposite effects because the intact Pol III is slower, exhibits longer-duration pausing, and terminates more efficiently than the enzyme lacking <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001733">Rpc37</a>/53 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Landrieux <em>et al.</em> 2006</a>), whereas TFIIF has been shown to increase Pol II elongation rate and decrease pausing (reviewed in <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Shilatifard <em>et al.</em> 2003</a>). All but one of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005733">Ret1</a> lobe mutants with strong termination phenotypes increased readthrough (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Shaaban <em>et al.</em> 1995</a>). One of these Pol III variants was selected for further study and shown to have a faster elongation rate and reduced propensity for pausing <em>in vitro</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Shaaban <em>et al.</em> 1996</a>), consistent with expectations if the mutation caused a decreased association with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001733">Rpc37</a>/53. In contrast, the lobe mutations in our study were found in decreased readthrough strains, which, by analogy, is the phenotype expected if the Pol II mutations disturbed the functional interaction with TFIIF.</p><p id="p-91">Many of the surface substitutions in the protrusion and external 2 domains also altered residues corresponding to or next to positions found to crosslink to TFIIF (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>). Unlike the lobe mutations, the large majority of these mutations conferred a decreased readthrough phenotype. One possible explanation to reconcile these observations is that the TFIIF contacts may differ in elongation complexes and preinitiation complexes (PICs). For example, some protrusion domain contacts observed for the PIC were absent from the isolated Pol-TFIIF complex (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Eichner <em>et al.</em> 2010</a>). Interference with normal protrusion/external 2 domain contacts might impair a function of TFIIF that uniquely occurs at or shortly after initiation, whereas the lobe mutant phenotypes may reflect a downstream function, such as elongation speed and pausing in the vicinity of the poly(A) or termination site. Alternatively, during elongation other proteins may associate with surfaces contacted by TFIIF at the promoter. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005677">rpb2</a></em> mutants described here provide a unique tool for answering these and other questions about the contributions of Pol II and associated proteins to polyadenylation and termination.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-92">We thank Christian Burns for technical assistance in making and testing some of the site-directed mutants. This work was supported by grants from the National Institutes of Health (GM-59644) and the National Science Foundation (0719556).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004531/-/DC1</a></p></li><li class="fn" id="fn-23"><p id="p-93"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-09-26"><span class="received-label">Received </span>September 26, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-11-27"><span class="accepted-label">Accepted </span>November 27, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Kubicek <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-3.2.167.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingston</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidman</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-source">Current Protocols in Molecular Biology</span>, <span class="cit-publ-name">Green Publishing Associates/Wiley Interscience</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.2" data-doi="10.1006/jmbi.1998.1779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaal</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourse</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">RNA polymerase mutants that destabilize RNA polymerase-promoter complexes alter NTP-sensing by <em>rrn</em> P1 promoters</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBartlett%26rft.auinit1%253DM.%2BS.%26rft.volume%253D279%26rft.issue%253D2%26rft.spage%253D331%26rft.epage%253D345%26rft.atitle%253DRNA%2Bpolymerase%2Bmutants%2Bthat%2Bdestabilize%2BRNA%2Bpolymerase-promoter%2Bcomplexes%2Balter%2BNTP-sensing%2Bby%2Brrn%2BP1%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.1779%26rft_id%253Dinfo%253Apmid%252F9642041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.1779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9642041&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074196900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.3" data-doi="10.1016/j.ceb.2005.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rules of engagement: co-transcriptional recruitment of pre-mRNA processing factors</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DBentley%26rft.auinit1%253DD.%2BL.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D251%26rft.epage%253D256%26rft.atitle%253DRules%2Bof%2Bengagement%253A%2Bco-transcriptional%2Brecruitment%2Bof%2Bpre-mRNA%2Bprocessing%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2005.04.006%26rft_id%253Dinfo%253Apmid%252F15901493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2005.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15901493&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229621900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.4" data-doi="10.1007/BF00330984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaCroute</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A positive selection for mutants lacking orotidine-5′-phosphate activity in yeast: 5-fluoro-orotic acid resistance</span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DBoeke%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D197%26rft.issue%253D2%26rft.spage%253D345%26rft.epage%253D346%26rft.atitle%253DA%2Bpositive%2Bselection%2Bfor%2Bmutants%2Blacking%2Borotidine-5%2527-phosphate%2Bdecarboxylase%2Bactivity%2Bin%2Byeast%253A%2B5-fluoro-orotic%2Bacid%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00330984%26rft_id%253Dinfo%253Apmid%252F6394957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00330984&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6394957&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TR93500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.5" data-doi="10.1038/sj.emboj.7600687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bucheli</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Npl3 is an antagonist of mRNA 3′ end formation by RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2150</span>–<span class="cit-lpage">2160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600687%26rft_id%253Dinfo%253Apmid%252F15902270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTIvMjE1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.6" data-doi="10.1016/j.ceb.2005.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Connections between mRNA 3′ end processing and transcription termination</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DBuratowski%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D257%26rft.epage%253D261%26rft.atitle%253DConnections%2Bbetween%2BmRNA%2B3%2527%2Bend%2Bprocessing%2Band%2Btranscription%2Btermination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2005.04.003%26rft_id%253Dinfo%253Apmid%252F15901494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2005.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15901494&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229621900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.7" data-doi="10.1128/MCB.24.9.3983-3991.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampsey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Functional interaction between TFIIB and the Rpb2 subunit of RNA polymerase II: implications for the mechanism of transcription initiation</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3983</span>–<span class="cit-lpage">3991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DChen%26rft.auinit1%253DB.-S.%26rft.volume%253D24%26rft.issue%253D9%26rft.spage%253D3983%26rft.epage%253D3991%26rft.atitle%253DFunctional%2BInteraction%2Bbetween%2BTFIIB%2Band%2Bthe%2BRpb2%2BSubunit%2Bof%2BRNA%2BPolymerase%2BII%253A%2BImplications%2Bfor%2Bthe%2BMechanism%2Bof%2BTranscription%2BInitiation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.9.3983-3991.2004%26rft_id%253Dinfo%253Apmid%252F15082791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI0LzkvMzk4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.8" data-doi="10.1038/nsmb1272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warfield</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The positions of TFIIF and TFIIE in the RNA polymerase II transcription preinitiation complex</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DChen%26rft.auinit1%253DH.%2BT.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D696%26rft.epage%253D703%26rft.atitle%253DThe%2Bpositions%2Bof%2BTFIIF%2Band%2BTFIIE%2Bin%2Bthe%2BRNA%2Bpolymerase%2BII%2Btranscription%2Bpreinitiation%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1272%26rft_id%253Dinfo%253Apmid%252F17632521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17632521&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248555400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.9" data-doi="10.1038/emboj.2009.401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jawhari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tahir</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Architecture of the RNA polymerase II–TFIIF complex revealed by cross-linking and mass spectrometry</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DChen%26rft.auinit1%253DZ.%2BA.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D717%26rft.epage%253D726%26rft.atitle%253DArchitecture%2Bof%2Bthe%2BRNA%2Bpolymerase%2BII-TFIIF%2Bcomplex%2Brevealed%2Bby%2Bcross-linking%2Band%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.401%26rft_id%253Dinfo%253Apmid%252F20094031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2009.401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20094031&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274604600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.10" data-doi="10.1101/gad.2.4.440"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manley</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">A functional mRNA polyadenylation signal is required for transcriptional termination by RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">440</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DConnelly%26rft.auinit1%253DS%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D440%26rft.epage%253D452%26rft.atitle%253DA%2Bfunctional%2BmRNA%2Bpolyadenylation%2Bsignal%2Bis%2Brequired%2Bfor%2Btranscription%2Btermination%2Bby%2BRNA%2Bpolymerase%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2.4.440%26rft_id%253Dinfo%253Apmid%252F2836265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMi80LzQ0MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.11" data-doi="10.1126/science.1059493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cramer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushnell</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornberg</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structural basis of transcription: RNA polymerase II at 2.8 Ångstroms resolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">292</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1059493%26rft_id%253Dinfo%253Apmid%252F11313498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUyMy8xODYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.12" data-doi="10.1128/MCB.23.21.7887-7901.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">In vivo evidence that defects in the transcriptional elongation factors RPB2, TFIIS, and SPT5 enhance upstream poly(A) site utilization</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">7887</span>–<span class="cit-lpage">7901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DCui%26rft.auinit1%253DY.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D7887%26rft.epage%253D7901%26rft.atitle%253DIn%2BVivo%2BEvidence%2Bthat%2BDefects%2Bin%2Bthe%2BTranscriptional%2BElongation%2BFactors%2BRPB2%252C%2BTFIIS%252C%2Band%2BSPT5%2BEnhance%2BUpstream%2BPoly%2528A%2529%2BSite%2BUtilization%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.21.7887-7901.2003%26rft_id%253Dinfo%253Apmid%252F14560031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMS83ODg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.13" data-doi="10.1074/jbc.M109.013847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dengl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cramer</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Torpedo nuclease Rat1 is insufficient to terminate RNA polymerase II in vitro</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">21270</span>–<span class="cit-lpage">21279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.013847%26rft_id%253Dinfo%253Apmid%252F19535338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvMzIvMjEyNzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.14" data-doi="10.1074/jbc.M111433200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desmoucelles</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saint-Marc</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daignan-Fornier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Screening the yeast “disruptome” for mutants affecting resistance to the immunosuppressive drug, mycophenolic acid</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">27036</span>–<span class="cit-lpage">27044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111433200%26rft_id%253Dinfo%253Apmid%252F12016207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMzAvMjcwMzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.15" data-doi="10.1038/emboj.2009.386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eichner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warfield</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Position of the general transcription factor TFIIF within the RNA polymerase II transcription preinitiation complex</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DEichner%26rft.auinit1%253DJ.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D706%26rft.epage%253D716%26rft.atitle%253DPosition%2Bof%2Bthe%2Bgeneral%2Btranscription%2Bfactor%2BTFIIF%2Bwithin%2Bthe%2BRNA%2Bpolymerase%2BII%2Btranscription%2Bpreinitiation%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.386%26rft_id%253Dinfo%253Apmid%252F20033062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2009.386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20033062&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274604600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanofsky</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Mutations of the β subunit of RNA polymerase alter both transcription pausing and transcription termination in the <em>trp</em> operon leader region <em>in vitro</em></span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">258</span>: <span class="cit-fpage">8146</span>–<span class="cit-lpage">8150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DFisher%26rft.auinit1%253DR.%2BF.%26rft.volume%253D258%26rft.issue%253D13%26rft.spage%253D8146%26rft.epage%253D8150%26rft.atitle%253DMutations%2Bof%2Bthe%2Bbeta%2Bsubunit%2Bof%2BRNA%2Bpolymerase%2Balter%2Bboth%2Btranscription%2Bpausing%2Band%2Btranscription%2Btermination%2Bin%2Bthe%2Btrp%2Boperon%2Bleader%2Bregion%2Bin%2Bvitro.%26rft_id%253Dinfo%253Apmid%252F6345540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNTgvMTMvODE0NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.17" data-doi="10.1093/nar/gki825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freire-Picos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnamurthy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampsey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence that the Tfg1/Tfg2 dimer interface of TFIIF lies near the active center of the RNA polymerase II initiation complex</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">5045</span>–<span class="cit-lpage">5052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFreire-Picos%26rft.auinit1%253DM.%2BA.%26rft.volume%253D33%26rft.issue%253D16%26rft.spage%253D5045%26rft.epage%253D5052%26rft.atitle%253DEvidence%2Bthat%2Bthe%2BTfg1%252FTfg2%2Bdimer%2Binterface%2Bof%2BTFIIF%2Blies%2Bnear%2Bthe%2Bactive%2Bcenter%2Bof%2Bthe%2BRNA%2Bpolymerase%2BII%2Binitiation%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki825%26rft_id%253Dinfo%253Apmid%252F16147988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8xNi81MDQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.18" data-doi="10.1128/MCB.24.24.10975-10985.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ammerman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziegler</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponticelli</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Amino acid substitutions in yeast TFIIF confer upstream shifts in transcription initiation and altered interaction with RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">10975</span>–<span class="cit-lpage">10985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGhazy%26rft.auinit1%253DM.%2BA.%26rft.volume%253D24%26rft.issue%253D24%26rft.spage%253D10975%26rft.epage%253D10985%26rft.atitle%253DAmino%2BAcid%2BSubstitutions%2Bin%2BYeast%2BTFIIF%2BConfer%2BUpstream%2BShifts%2Bin%2BTranscription%2BInitiation%2Band%2BAltered%2BInteraction%2Bwith%2BRNA%2BPolymerase%2BII%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.24.10975-10985.2004%26rft_id%253Dinfo%253Apmid%252F15572698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjExOiIyNC8yNC8xMDk3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.19" data-doi="10.1074/jbc.R700032200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Derailing the locomotive: transcription termination</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R700032200%26rft_id%253Dinfo%253Apmid%252F17998201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI4My8yLzY2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.20" data-doi="10.1126/science.1059495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnatt</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cramer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushnell</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornberg</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 Å resolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">292</span>: <span class="cit-fpage">1876</span>–<span class="cit-lpage">1882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1059495%26rft_id%253Dinfo%253Apmid%252F11313499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUyMy8xODc2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.21" data-doi="10.1128/MCB.26.10.3986-3996.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gromak</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proudfoot</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Pause sites promote transcriptional termination of mammalian RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3986</span>–<span class="cit-lpage">3996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGromak%26rft.auinit1%253DN.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D3986%26rft.epage%253D3996%26rft.atitle%253DPause%2BSites%2BPromote%2BTranscriptional%2BTermination%2Bof%2BMammalian%2BRNA%2BPolymerase%2BII%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.26.10.3986-3996.2006%26rft_id%253Dinfo%253Apmid%252F16648491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMC8zOTg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.22" data-doi="10.1016/S1097-2765(00)80477-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusarov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nudler</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The mechanism of intrinsic transcription termination</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGusarov%26rft.auinit1%253DI.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D495%26rft.epage%253D504%26rft.atitle%253DThe%2Bmechanism%2Bof%2Bintrinsic%2Btranscription%2Btermination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980477-3%26rft_id%253Dinfo%253Apmid%252F10230402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80477-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10230402&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079986700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.23" data-doi="10.1128/MCB.11.11.5781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hekmatpanah</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Mutations in a conserved region of RNA polymerase II influence the accuracy of mRNA start site selection</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">5781</span>–<span class="cit-lpage">5791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHekmatpanah%26rft.auinit1%253DD%2BS%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D5781%26rft.epage%253D5791%26rft.atitle%253DMutations%2Bin%2Ba%2Bconserved%2Bregion%2Bof%2BRNA%2Bpolymerase%2BII%2Binfluence%2Bthe%2Baccuracy%2Bof%2BmRNA%2Bstart%2Bsite%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.11.5781%26rft_id%253Dinfo%253Apmid%252F1922077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS81NzgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seiler</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whoriskey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Point mutations upstream of the yeast <em>ADH2</em> poly(A) site significantly reduce the efficiency of 3′-end formation</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2004</span>–<span class="cit-lpage">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHyman%26rft.auinit1%253DL%2BE%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D2004%26rft.epage%253D2012%26rft.atitle%253DPoint%2Bmutations%2Bupstream%2Bof%2Bthe%2Byeast%2BADH2%2Bpoly%2528A%2529%2Bsite%2Bsignificantly%2Breduce%2Bthe%2Befficiency%2Bof%2B3%2527-end%2Bformation.%26rft_id%253Dinfo%253Apmid%252F2005893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzQvMjAwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.25" data-doi="10.1101/gr.111070.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ip</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramani</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Global impact of RNA polymerase II elongation inhibition on alternative splicing regulation</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111070.110%26rft_id%253Dinfo%253Apmid%252F21163941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvMzkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whelen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The <em>RET1</em> gene of yeast encodes the second-largest subunit of RNA polymerase III. Structural analysis of the wild-type and <em>ret1–1</em> mutant alleles</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">5616</span>–<span class="cit-lpage">5624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DJames%26rft.auinit1%253DP.%26rft.volume%253D266%26rft.issue%253D9%26rft.spage%253D5616%26rft.epage%253D5624%26rft.atitle%253DThe%2BRET1%2Bgene%2Bof%2Byeast%2Bencodes%2Bthe%2Bsecond-largest%2Bsubunit%2Bof%2BRNA%2Bpolymerase%2BIII.%2BStructural%2Banalysis%2Bof%2Bthe%2Bwild-type%2Band%2Bret1-1%2Bmutant%2Balleles%26rft_id%253Dinfo%253Apmid%252F2005101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNjYvOS81NjE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.27" data-doi="10.1016/0022-2836(88)90455-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Characterization of the termination phenotypes of rifampicin-resistant mutants</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DJin%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D202%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D253%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Btermination%2Bphenotypes%2Bof%2Brifampicin-resistant%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990455-X%26rft_id%253Dinfo%253Apmid%252F3050123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90455-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3050123&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988P410400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.28" data-doi="10.1074/jbc.M411108200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Interaction between transcription elongation factors and mRNA 3′-end formation at the Saccharomyces cerevisiae GAL10–GAL7 locus</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">922</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M411108200%26rft_id%253Dinfo%253Apmid%252F15531585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI4MC8yLzkxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.29" data-doi="10.1371/journal.pgen.1002627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belyanin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dissection of Pol II trigger loop function and Pol II activity-dependent control of start site selection in vivo</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKaplan%26rft.auinit1%253DC.%2BD.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1002627%26rft.epage%253De1002627%26rft.atitle%253DDissection%2Bof%2BPol%2BII%2Btrigger%2Bloop%2Bfunction%2Band%2BPol%2BII%2Bactivity-dependent%2Bcontrol%2Bof%2Bstart%2Bsite%2Bselection%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002627%26rft_id%253Dinfo%253Apmid%252F22511879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22511879&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.30" data-doi="10.1038/nature03041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasiljeva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rando</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nedea</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The yeast Rat1 exonuclease promotes transcription termination by RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKim%26rft.auinit1%253DM.%26rft.volume%253D432%26rft.issue%253D7016%26rft.spage%253D517%26rft.epage%253D522%26rft.atitle%253DThe%2Byeast%2BRat1%2Bexonuclease%2Bpromotes%2Btranscription%2Btermination%2Bby%2BRNA%2Bpolymerase%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03041%26rft_id%253Dinfo%253Apmid%252F15565157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15565157&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225322100050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.31" data-doi="10.1074/jbc.M111.260844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kireeva</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domecq</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulombe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashlev</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interaction of RNA polymerase II fork loop 2 with downstream non-template DNA regulates transcription elongation</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">30898</span>–<span class="cit-lpage">30910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.260844%26rft_id%253Dinfo%253Apmid%252F21730074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMzUvMzA4OTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.32" data-doi="10.1016/S1097-2765(02)00738-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komissarova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kireeva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashlev</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Shortening of RNA:DNA hybrid in the elongation complex of RNA polymerase is a prerequisite for transcription termination</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DKomissarova%26rft.auinit1%253DN.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D1151%26rft.epage%253D1162%26rft.atitle%253DShortening%2Bof%2BRNA%253ADNA%2Bhybrid%2Bin%2Bthe%2Belongation%2Bcomplex%2Bof%2BRNA%2Bpolymerase%2Bis%2Ba%2Bprerequisite%2Bfor%2Btranscription%2Btermination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900738-4%26rft_id%253Dinfo%253Apmid%252F12453422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00738-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12453422&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179450600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.33" data-doi="10.1126/science.289.5479.619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korzheva</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mustaev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozlov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malhotra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikiforov</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A structural model of transcription elongation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKorzheva%26rft.auinit1%253DN.%26rft.volume%253D289%26rft.issue%253D5479%26rft.spage%253D619%26rft.epage%253D625%26rft.atitle%253DA%2BStructural%2BModel%2Bof%2BTranscription%2BElongation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5479.619%26rft_id%253Dinfo%253Apmid%252F10915625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNTQ3OS82MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.34" data-doi="10.1128/MCB.22.20.6979-6992.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">RNA polymerase II elongation factors of Saccharomyces cerevisiae: a targeted proteomics approach</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">6979</span>–<span class="cit-lpage">6992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKrogan%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D22%26rft.issue%253D20%26rft.spage%253D6979%26rft.epage%253D6992%26rft.atitle%253DRNA%2BPolymerase%2BII%2BElongation%2BFactors%2Bof%2BSaccharomyces%2Bcerevisiae%253A%2Ba%2BTargeted%2BProteomics%2BApproach%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.20.6979-6992.2002%26rft_id%253Dinfo%253Apmid%252F12242279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yMC82OTc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.35" data-doi="10.1038/nrm3098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuehner</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Unravelling the means to an end: RNA polymerase II transcription termination</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DKuehner%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D283%26rft.epage%253D294%26rft.atitle%253DUnravelling%2Bthe%2Bmeans%2Bto%2Ban%2Bend%253A%2BRNA%2Bpolymerase%2BII%2Btranscription%2Btermination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3098%26rft_id%253Dinfo%253Apmid%252F21487437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21487437&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.36" data-doi="10.1016/j.cell.2007.10.051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gartmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional architecture of RNA polymerase I</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1260</span>–<span class="cit-lpage">1272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKuhn%26rft.auinit1%253DC.%2BD.%26rft.volume%253D131%26rft.issue%253D7%26rft.spage%253D1260%26rft.epage%253D1272%26rft.atitle%253DFunctional%2Barchitecture%2Bof%2BRNA%2Bpolymerase%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.051%26rft_id%253Dinfo%253Apmid%252F18160037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18160037&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.37" data-doi="10.1042/BST0341062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The regulatory roles and mechanism of transcriptional pausing</span>. <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1062</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BSociety%2BTransactions%26rft.stitle%253DBiochm.%2BSoc.%2BTrans.%26rft.aulast%253DLandick%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253DPt%2B6%26rft.spage%253D1062%26rft.epage%253D1066%26rft.atitle%253DThe%2Bregulatory%2Broles%2Band%2Bmechanism%2Bof%2Btranscriptional%2Bpausing.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0341062%26rft_id%253Dinfo%253Apmid%252F17073751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1042/BST0341062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17073751&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243013500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.38" data-doi="10.1101/gad.4.9.1623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Amino acid changes in conserved regions of the Beta-subunit of <em>Escherichia coli</em> RNA polymerase alter transcription pausing and termination</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLandick%26rft.auinit1%253DR%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1623%26rft.epage%253D1636%26rft.atitle%253DAmino%2Bacid%2Bchanges%2Bin%2Bconserved%2Bregions%2Bof%2Bthe%2Bbeta-subunit%2Bof%2BEscherichia%2Bcoli%2BRNA%2Bpolymerase%2Balter%2Btranscription%2Bpausing%2Band%2Btermination.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.4.9.1623%26rft_id%253Dinfo%253Apmid%252F2253882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiNC85LzE2MjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.39" data-doi="10.1038/sj.emboj.7600915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landrieux</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alic</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducrot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A subcomplex of RNA polymerase III subunits involved in transcription termination and reinitiation</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DMohammad%26rft.auinit1%253DA.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D118%26rft.epage%253D128%26rft.atitle%253DA%2Bsubcomplex%2Bof%2BRNA%2Bpolymerase%2BIII%2Bsubunits%2Binvolved%2Bin%2Btranscription%2Btermination%2Band%2Breinitiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600915%26rft_id%253Dinfo%253Apmid%252F16362040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7600915&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16362040&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234951900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.40" data-doi="10.1016/j.jmb.2009.10.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darst</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular evolution of multisubunit RNA polymerases: structural analysis</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">395</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLane%26rft.auinit1%253DW.%2BJ.%26rft.volume%253D395%26rft.issue%253D4%26rft.spage%253D686%26rft.epage%253D704%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bmultisubunit%2BRNA%2Bpolymerases%253A%2Bstructural%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.10.063%26rft_id%253Dinfo%253Apmid%252F19895816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.10.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19895816&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274556000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.41" data-doi="10.1073/pnas.84.23.8306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falck-Pedersen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darnell</span>  <span class="cit-name-given-names">J. E.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shenk</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A poly(A) addition site and a downstream termination region are required for efficient cessation of transcription by RNA polymerase II in the mouse β<sup>maj</sup>-globin gene</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">8306</span>–<span class="cit-lpage">8310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLogan%26rft.auinit1%253DJ.%26rft.volume%253D84%26rft.issue%253D23%26rft.spage%253D8306%26rft.epage%253D8310%26rft.atitle%253DA%2Bpoly%2528A%2529%2Baddition%2Bsite%2Band%2Ba%2Bdownstream%2Btermination%2Bregion%2Bare%2Brequired%2Bfor%2Befficient%2Bcessation%2Bof%2Btranscription%2Bby%2BRNA%2Bpolymerase%2BII%2Bin%2Bthe%2Bmouse%2Bbeta%2Bmaj-globin%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.23.8306%26rft_id%253Dinfo%253Apmid%252F3479794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODQvMjMvODMwNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.42" data-doi="10.1101/gad.1409106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The role of Rat1 in coupling mRNA 3′-end processing to transcription termination: implications for a unified allosteric–torpedo model</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">954</span>–<span class="cit-lpage">965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1409106%26rft_id%253Dinfo%253Apmid%252F16598041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjAvOC85NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magrath</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A mutation in <em>GRS1</em>, a Glycyl-tRNA synthetase, affects 3′-end formation in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D152%26rft.spage%253D129%26rft_id%253Dinfo%253Apmid%252F10224248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10224248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.44" data-doi="10.1038/nsmb.1903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lidschreiber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leike</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Söding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Uniform transitions of the general RNA polymerase II transcription complex</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1272</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DMayer%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D1272%26rft.epage%253D1278%26rft.atitle%253DUniform%2Btransitions%2Bof%2Bthe%2Bgeneral%2BRNA%2Bpolymerase%2BII%2Btranscription%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1903%26rft_id%253Dinfo%253Apmid%252F20818391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20818391&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282563600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.45" data-doi="10.1016/j.tibs.2010.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Mata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornblihtt</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The carboxy terminal domain of RNA polymerase II and alternative splicing</span>. <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DMunoz%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D35%26rft.issue%253D9%26rft.spage%253D497%26rft.epage%253D504%26rft.atitle%253DThe%2Bcarboxy%2Bterminal%2Bdomain%2Bof%2BRNA%2Bpolymerase%2BII%2Band%2Balternative%2Bsplicing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2010.03.010%26rft_id%253Dinfo%253Apmid%252F20418102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2010.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20418102&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.46" data-doi="10.1016/j.jmb.2011.03.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vangeloff</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bacterial transcription terminators: the RNA 3′-end chronicles</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">412</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPeters%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D412%26rft.issue%253D5%26rft.spage%253D793%26rft.epage%253D813%26rft.atitle%253DBacterial%2Btranscription%2Bterminators%253A%2Bthe%2BRNA%2B3%2527-end%2Bchronicles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2011.03.036%26rft_id%253Dinfo%253Apmid%252F21439297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2011.03.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21439297&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.47" data-doi="10.1093/nar/29.9.e45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfaffl</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new mathematical model for relative quantification in real-time RT-PCR</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">e45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPfaffl%26rft.auinit1%253DM.%2BW.%26rft.volume%253D29%26rft.issue%253D9%26rft.spage%253De45%26rft.epage%253De45%26rft.atitle%253DA%2Bnew%2Bmathematical%2Bmodel%2Bfor%2Brelative%2Bquantification%2Bin%2Breal-time%2BRT-PCR%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.9.e45%26rft_id%253Dinfo%253Apmid%252F11328886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI5LzkvZTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.48" data-doi="10.1016/S1097-2765(02)00502-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pokholok</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannett</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Exchange of RNA polymerase II initiation and elongation factors during gene expression in vivo</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPokholok%26rft.auinit1%253DD.%2BK.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D809%26rft.atitle%253DExchange%2Bof%2BRNA%2Bpolymerase%2BII%2Binitiation%2Band%2Belongation%2Bfactors%2Bduring%2Bgene%2Bexpression%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900502-6%26rft_id%253Dinfo%253Apmid%252F11983171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00502-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11983171&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175244400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.49" data-doi="10.1074/jbc.271.12.6866"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reines</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mutations in the second largest subunit of RNA polymerase II cause 6-azauracil sensitivity in yeast and increased transcriptional arrest in vitro</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">6866</span>–<span class="cit-lpage">6873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DReines%26rft.auinit1%253DD.%26rft.volume%253D271%26rft.issue%253D12%26rft.spage%253D6866%26rft.epage%253D6873%26rft.atitle%253DMutations%2Bin%2Bthe%2BSecond%2BLargest%2BSubunit%2Bof%2BRNA%2BPolymerase%2BII%2BCause%2B6-Azauracil%2BSensitivity%2Bin%2BYeast%2Band%2BIncreased%2BTranscriptional%2BArrest%2Bin%2BVitro%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.271.12.6866%26rft_id%253Dinfo%253Apmid%252F8636112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzEvMTIvNjg2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.50" data-doi="10.1038/nature10799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhee</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide structure and organization of eukaryotic pre-initiation complexes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">483</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRhee%26rft.auinit1%253DH.%2BS.%26rft.volume%253D483%26rft.issue%253D7389%26rft.spage%253D295%26rft.epage%253D301%26rft.atitle%253DGenome-wide%2Bstructure%2Band%2Borganization%2Bof%2Beukaryotic%2Bpre-initiation%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10799%26rft_id%253Dinfo%253Apmid%252F22258509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22258509&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301481800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.51" data-doi="10.1101/gad.1792809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcription termination by nuclear RNA polymerases</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1247</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRichard%26rft.auinit1%253DP.%26rft.volume%253D23%26rft.issue%253D11%26rft.spage%253D1247%26rft.epage%253D1269%26rft.atitle%253DTranscription%2Btermination%2Bby%2Bnuclear%2BRNA%2Bpolymerases%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1792809%26rft_id%253Dinfo%253Apmid%252F19487567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzExLzEyNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.52" data-doi="10.1073/pnas.2336092100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogatsky</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derynck</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonaka</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khodabakhsh</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Target-specific utilization of transcriptional regulatory surfaces by the glucocorticoid receptor</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">13845</span>–<span class="cit-lpage">13850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2336092100%26rft_id%253Dinfo%253Apmid%252F14617768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI0LzEzODQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scafe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okamura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1990</span><span class="cit-comment">a</span> <span class="cit-article-title">Conditional mutations occur predominantly in highly conserved residues of RNA polymerase II subunits</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1270</span>–<span class="cit-lpage">1275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DScafe%26rft.auinit1%253DC%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D1270%26rft.epage%253D1275%26rft.atitle%253DConditional%2Bmutations%2Boccur%2Bpredominantly%2Bin%2Bhighly%2Bconserved%2Bresidues%2Bof%2BRNA%2Bpolymerase%2BII%2Bsubunits.%26rft_id%253Dinfo%253Apmid%252F2406567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzMvMTI3MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.54" data-doi="10.1128/MCB.10.3.1010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scafe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span><span class="cit-comment">b</span> <span class="cit-article-title">RNA polymerase II mutants defective in transcription of a subset of genes</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1010</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DScafe%26rft.auinit1%253DC%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D1010%26rft.epage%253D1016%26rft.atitle%253DRNA%2Bpolymerase%2BII%2Bmutants%2Bdefective%2Bin%2Btranscription%2Bof%2Ba%2Bsubset%2Bof%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.3.1010%26rft_id%253Dinfo%253Apmid%252F2406558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzMvMTAxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaaban</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krupp</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Termination altering mutations in the second-largest subunit of yeast RNA polymerase III</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1467</span>–<span class="cit-lpage">1478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DShaaban%26rft.auinit1%253DS.%2BA.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1467%26rft.epage%253D1478%26rft.atitle%253DTermination-altering%2Bmutations%2Bin%2Bthe%2Bsecond-largest%2Bsubunit%2Bof%2Byeast%2BRNA%2Bpolymerase%2BIII%26rft_id%253Dinfo%253Apmid%252F7862140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTQ2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaaban</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bobkova</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chudzik</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>In vitro</em> analysis of elongation and termination by mutant RNA polymerases with altered termination behavior</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">6468</span>–<span class="cit-lpage">6476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DShaaban%26rft.auinit1%253DS.%2BA.%26rft.volume%253D16%26rft.issue%253D11%26rft.spage%253D6468%26rft.epage%253D6476%26rft.atitle%253DIn%2Bvitro%2Banalysis%2Bof%2Belongation%2Band%2Btermination%2Bby%2Bmutant%2BRNA%2Bpolymerases%2Bwith%2Baltered%2Btermination%2Bbehavior%26rft_id%253Dinfo%253Apmid%252F8887675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMS82NDY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.57" data-doi="10.1128/MCB.20.20.7427-7437.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reines</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> transcription elongation mutants are defective in PUR5 induction in response to nucleotide depletion</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">7427</span>–<span class="cit-lpage">7437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DShaw%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D20%26rft.issue%253D20%26rft.spage%253D7427%26rft.epage%253D7437%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BTranscription%2BElongation%2BMutants%2BAre%2BDefective%2Bin%2BPUR5%2BInduction%2Bin%2BResponse%2Bto%2BNucleotide%2BDepletion%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.20.7427-7437.2000%26rft_id%253Dinfo%253Apmid%252F11003640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8yMC83NDI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.58" data-doi="10.1074/jbc.M105075200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reines</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Regulation of an IMP dehydrogenase gene and its overexpression in drug-sensitive transcription elongation mutants of yeast</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">32905</span>–<span class="cit-lpage">32916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M105075200%26rft_id%253Dinfo%253Apmid%252F11441018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzYvMzUvMzI5MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.59" data-doi="10.1146/annurev.biochem.72.121801.161551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shilatifard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conaway</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conaway</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The RNA polymerase II elongation complex</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.volume%253D72%26rft.spage%253D693%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.72.121801.161551%26rft_id%253Dinfo%253Apmid%252F12676794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.72.121801.161551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12676794&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185092500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.60" data-doi="10.1128/MCB.23.18.6339-6349.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brow</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Ssu72 protein mediates both poly(A) coupled and poly(A)-independent termination of RNA polymerase II transcription</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">6339</span>–<span class="cit-lpage">6349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSteinmetz%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D23%26rft.issue%253D18%26rft.spage%253D6339%26rft.epage%253D6349%26rft.atitle%253DSsu72%2BProtein%2BMediates%2BBoth%2BPoly%2528A%2529-Coupled%2Band%2BPoly%2528A%2529-Independent%2BTermination%2Bof%2BRNA%2BPolymerase%2BII%2BTranscription%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.18.6339-6349.2003%26rft_id%253Dinfo%253Apmid%252F12944462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xOC82MzM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.61" data-doi="10.1038/35095090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brow</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corden</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">RNA-binding protein Nrd1 directs poly(A)-independent 3′ end formation of RNA polymerase II transcripts</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">413</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D413%26rft.issue%253D6853%26rft.spage%253D327%26rft.epage%253D331%26rft.atitle%253DRNA-binding%2Bprotein%2BNrd1%2Bdirects%2Bpoly%2528A%2529-independent%2B3%2527-end%2Bformation%2Bof%2BRNA%2Bpolymerase%2BII%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35095090%26rft_id%253Dinfo%253Apmid%252F11565036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35095090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11565036&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171040500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.62" data-doi="10.1128/MCB.26.7.2688-2696.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloute</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brow</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">cis- and trans-acting determinants of transcription termination by yeast RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2688</span>–<span class="cit-lpage">2696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSteinmetz%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D26%26rft.issue%253D7%26rft.spage%253D2688%26rft.epage%253D2696%26rft.atitle%253Dcis-%2Band%2Btrans-Acting%2BDeterminants%2Bof%2BTranscription%2BTermination%2Bby%2BYeast%2BRNA%2BPolymerase%2BII%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.26.7.2688-2696.2006%26rft_id%253Dinfo%253Apmid%252F16537912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMjY4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.63" data-doi="10.1073/pnas.84.5.1192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sweetser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Prokaryotic and eukaryotic RNA polymerases have homologous core subunits</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1192</span>–<span class="cit-lpage">1196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSweetser%26rft.auinit1%253DD.%26rft.volume%253D84%26rft.issue%253D5%26rft.spage%253D1192%26rft.epage%253D1196%26rft.atitle%253DProkaryotic%2Band%2Beukaryotic%2BRNA%2Bpolymerases%2Bhave%2Bhomologous%2Bcore%2Bsubunits.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.5.1192%26rft_id%253Dinfo%253Apmid%252F3547406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NC81LzExOTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.64" data-doi="10.1128/jb.178.17.5263-5271.1996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavormina</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">a</span> <span class="cit-article-title">Isolation, purification, and <em>in vitro</em> characterization of recessive-lethal mutant RNA polymerases from <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">5263</span>–<span class="cit-lpage">5271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DTavormina%26rft.auinit1%253DP.%2BL.%26rft.volume%253D178%26rft.issue%253D17%26rft.spage%253D5263%26rft.epage%253D5271%26rft.atitle%253DIsolation%252C%2Bpurification%252C%2Band%2Bin%2Bvitro%2Bcharacterization%2Bof%2Brecessive-lethal-mutant%2BRNA%2Bpolymerases%2Bfrom%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.178.17.5263-5271.1996%26rft_id%253Dinfo%253Apmid%252F8752347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3OC8xNy81MjYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.65" data-doi="10.1006/jmbi.1996.0244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavormina</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">b</span> <span class="cit-article-title">Identifying interacting regions in the β subunit of Escherichia coli RNA polymerase</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">258</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DTavormina%26rft.auinit1%253DP.%2BL.%26rft.volume%253D258%26rft.issue%253D2%26rft.spage%253D213%26rft.epage%253D223%26rft.atitle%253DIdentifying%2Binteracting%2Bregions%2Bin%2Bthe%2Bbeta%2Bsubunit%2Bof%2BEscherichia%2Bcoli%2BRNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0244%26rft_id%253Dinfo%253Apmid%252F8627620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8627620&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.66" data-doi="10.1093/emboj/cdf654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trautinger</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Modulation of DNA repair by mutations flanking the DNA channel through RNA polymerase</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">6944</span>–<span class="cit-lpage">6953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DTrautinger%26rft.auinit1%253DB.%2BW.%26rft.volume%253D21%26rft.issue%253D24%26rft.spage%253D6944%26rft.epage%253D6953%26rft.atitle%253DModulation%2Bof%2BDNA%2Brepair%2Bby%2Bmutations%2Bflanking%2Bthe%2BDNA%2Bchannel%2Bthrough%2BRNA%2Bpolymerase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf654%26rft_id%253Dinfo%253Apmid%252F12486015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/emboj/cdf654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12486015&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179952000032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.67" data-doi="10.1128/MMBR.70.1.12-36.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trinh</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langelier</span>  <span class="cit-name-given-names">M.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archambault</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulombe</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural perspectives on mutations affecting the function of multisubunit RNA polymerases</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DTrinh%26rft.auinit1%253DV.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D36%26rft.atitle%253DStructural%2BPerspective%2Bon%2BMutations%2BAffecting%2Bthe%2BFunction%2Bof%2BMultisubunit%2BRNA%2BPolymerases%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.70.1.12-36.2006%26rft_id%253Dinfo%253Apmid%252F16524917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI3MC8xLzEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.68" data-doi="10.1038/nature05932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vassylyev</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vassylyeva</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perederina</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tahirov</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artsimovitch</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Structural basis for transcription elongation by bacterial RNA polymerase</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVassylyev%26rft.auinit1%253DD.%2BG.%26rft.volume%253D448%26rft.issue%253D7150%26rft.spage%253D157%26rft.epage%253D162%26rft.atitle%253DStructural%2Bbasis%2Bfor%2Btranscription%2Belongation%2Bby%2Bbacterial%2BRNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05932%26rft_id%253Dinfo%253Apmid%252F17581590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17581590&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247934500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.69" data-doi="10.1126/science.281.5377.660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Hippel</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Integrated model of the transcription complex in elongation, termination and editing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253Dvon%2BHippel%26rft.auinit1%253DP.%2BH.%26rft.volume%253D281%26rft.issue%253D5377%26rft.spage%253D660%26rft.epage%253D665%26rft.atitle%253DAn%2BIntegrated%2BModel%2Bof%2Bthe%2BTranscription%2BComplex%2Bin%2BElongation%252C%2BTermination%252C%2Band%2BEditing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.281.5377.660%26rft_id%253Dinfo%253Apmid%252F9685251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNTM3Ny82NjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzIvMTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.70" data-doi="10.1101/gad.8.23.2913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weilbaecher</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebron</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Termination-altering amino acid substitutions in the beta’ subunit of <em>Escherichia coli</em> RNA polymerase identify regions involved in RNA chain elongation</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2913</span>–<span class="cit-lpage">2927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWeilbaecher%26rft.auinit1%253DR%26rft.volume%253D8%26rft.issue%253D23%26rft.spage%253D2913%26rft.epage%253D2927%26rft.atitle%253DTermination-altering%2Bamino%2Bacid%2Bsubstitutions%2Bin%2Bthe%2Bbeta%2527%2Bsubunit%2Bof%2BEscherichia%2Bcoli%2BRNA%2Bpolymerase%2Bidentify%2Bregions%2Binvolved%2Bin%2BRNA%2Bchain%2Belongation.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.23.2913%26rft_id%253Dinfo%253Apmid%252F7527790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOC8yMy8yOTEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.71" data-doi="10.1038/nature03035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gromak</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proudfoot</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Human 5′-3′ exonuclease Xrn2 promotes transcription termination at co-transcriptional cleavage sites</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWest%26rft.auinit1%253DS.%26rft.volume%253D432%26rft.issue%253D7016%26rft.spage%253D522%26rft.epage%253D525%26rft.atitle%253DHuman%2B5%2527%2B--%253E%2B3%2527%2Bexonuclease%2BXrn2%2Bpromotes%2Btranscription%2Btermination%2Bat%2Bco-transcriptional%2Bcleavage%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03035%26rft_id%253Dinfo%253Apmid%252F15565158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15565158&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F2%2F167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225322100051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.72" data-doi="10.1128/MCB.05151-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The TFIIF-like Rpc37/53 dimer lies at the center of a protein network to connect TFIIIC, Bdp1, and the RNA polymerase III active center</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2715</span>–<span class="cit-lpage">2728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.05151-11%26rft_id%253Dinfo%253Apmid%252F21536656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMy8yNzE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy8yLzE2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.2.167.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Formation of mRNA 3′ ends in eukaryotes: mechanism, regulation, and interrelationships with other steps in mRNA synthesis</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DZhao%26rft.auinit1%253DJ.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D405%26rft.epage%253D445%26rft.atitle%253DFormation%2Bof%2BmRNA%2B3%2527%2BEnds%2Bin%2BEukaryotes%253A%2BMechanism%252C%2BRegulation%252C%2Band%2BInterrelationships%2Bwith%2BOther%2BSteps%2Bin%2BmRNA%2BSynthesis%26rft_id%253Dinfo%253Apmid%252F10357856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2My8yLzQwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvMi8xNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/2/167.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/2/153" title="Fosmid-Based Structure-Function Analysis Reveals Functionally Distinct Domains in the Cytoplasmic Domain of Drosophila Crumbs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/2/181" title="Redundant Canonical and Noncanonical Caenorhabditis elegans p21-Activated Kinase Signaling Governs Distal Tip Cell Migrations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/2"><span>Volume 3 Issue 2, February 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/2.cover-source.jpg?itok=h3kYZ-FP" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/2/167.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fapsMRvYsn2zLvKBsLKwHQyTKiVA5xrT8tFpYaLRK4Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/2/167" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/2/167" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/2/167" class="active" data-icon-position="" data-hide-link-title="0">RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459402" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jVcDarHpuHo02pbFq-71xVbMY44s3hvg9JBydQ3FPXI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/167.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/2/167" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bnPwhZhthP2XKY4utDnCl8zJszhvu7kLFMaRBBeV9dE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/2/167" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/2/167.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459402" id="node459402--2" data-pisa="ggg;3/2/167" data-pisa-master="ggg;3/2/167" data-apath="/ggg/3/2/167.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles E. Kubicek</span>, <span class="highwire-citation-author" data-delta="1">Robert D. Chisholm</span>, <span class="highwire-citation-author" data-delta="2">Sachiko Takayama</span> and <span class="highwire-citation-author" data-delta="3">Diane K. Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">167-180;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004531 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459402--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles E. Kubicek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BE.%2BKubicek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kubicek%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BE.%2BKubicek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert D. Chisholm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BD.%2BChisholm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chisholm%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BD.%2BChisholm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sachiko Takayama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sachiko%2BTakayama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takayama%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASachiko%2BTakayama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diane K. Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BK.%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20DK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BK.%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhawley@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">dhawley@uoregon.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/2/167" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/2/167&amp;title=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/2/167&amp;title=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/2/167&amp;title=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/2/167&amp;text=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/2/167&amp;title=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/2/167&amp;t=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/2/167&amp;title=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/2/167&amp;title=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459402/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459402" id="node459402--3" data-pisa="ggg;3/2/167" data-pisa-master="ggg;3/2/167" data-apath="/ggg/3/2/167.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA Polymerase II Mutations Conferring Defects in Poly(A) Site Cleavage and Termination in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles E. Kubicek</span>, <span class="highwire-citation-author" data-delta="1">Robert D. Chisholm</span>, <span class="highwire-citation-author" data-delta="2">Sachiko Takayama</span> and <span class="highwire-citation-author" data-delta="3">Diane K. Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">167-180;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004531 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459402--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles E. Kubicek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BE.%2BKubicek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kubicek%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BE.%2BKubicek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert D. Chisholm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BD.%2BChisholm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chisholm%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BD.%2BChisholm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sachiko Takayama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sachiko%2BTakayama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takayama%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASachiko%2BTakayama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diane K. Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Chemistry and Institute of Molecular Biology, University of Oregon, Eugene, Oregon 97403-1229</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BK.%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20DK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BK.%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhawley@uoregon.edu" class="" data-icon-position="" data-hide-link-title="0">dhawley@uoregon.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=167&amp;rft.epage=180&amp;rft.atitle=RNA%20Polymerase%20II%20Mutations%20Conferring%20Defects%20in%20Poly%28A%29%20Site%20Cleavage%20and%20Termination%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=3&amp;rft.issue=2&amp;rft.date=2013-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kubicek%2C+Charles+E.&amp;rft.au=Chisholm%2C+Robert+D.&amp;rft.au=Takayama%2C+Sachiko&amp;rft.au=Hawley%2C+Diane+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459402/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459402/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459402/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459402/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459402/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459402/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459402/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459402/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459402/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459402/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459402/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459402/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/2/167" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/2/167.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/2/167.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/2/167.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/2/167.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
