<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/11/2065" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/11/2065.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/11/2065.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/11/2065.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/11/2065.atom" />
<meta name="HW.pisa" content="ggg;4/11/2065" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.114.012526" />
<meta name="DC.Date" content="2014-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Devisetty et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.117.043190" />
<meta name="DC.Description" content="The mapping and functional analysis of quantitative traits in Brassica rapa can be greatly improved with the availability of physically positioned, gene-based genetic markers and accurate genome annotation. In this study, deep transcriptome RNA sequencing (RNA-Seq) of Brassica rapa was undertaken with two objectives: SNP detection and improved transcriptome annotation. We performed SNP detection on two varieties that are parents of a mapping population to aid in development of a marker system for this population and subsequent development of high-resolution genetic map. An improved Brassica rapa transcriptome was constructed to detect novel transcripts and to improve the current genome annotation. This is useful for accurate mRNA abundance and detection of expression QTL (eQTLs) in mapping populations. Deep RNA-Seq of two Brassica rapa genotypes—R500 (var. trilocularis, Yellow Sarson) and IMB211 (a rapid cycling variety)—using eight different tissues (root, internode, leaf, petiole, apical meristem, floral meristem, silique, and seedling) grown across three different environments (growth chamber, greenhouse and field) and under two different treatments (simulated sun and simulated shade) generated 2.3 billion high-quality Illumina reads. A total of 330,995 SNPs were identified in transcribed regions between the two genotypes with an average frequency of one SNP in every 200 bases. The deep RNA-Seq reassembled Brassica rapa transcriptome identified 44,239 protein-coding genes. Compared with current gene models of B. rapa , we detected 3537 novel transcripts, 23,754 gene models had structural modifications, and 3655 annotated proteins changed. Gaps in the current genome assembly of B. rapa are highlighted by our identification of 780 unmapped transcripts. All the SNPs, annotations, and predicted transcripts can be viewed at &lt;http://phytonetworks.ucdavis.edu/&gt;." />
<meta name="DC.Contributor" content="Upendra Kumar Devisetty" />
<meta name="DC.Contributor" content="Michael F. Covington" />
<meta name="DC.Contributor" content="An V. Tat" />
<meta name="DC.Contributor" content="Saradadevi Lekkala" />
<meta name="DC.Contributor" content="Julin N. Maloof" />
<meta name="article:published_time" content="2014-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The mapping and functional analysis of quantitative traits in &lt;i&gt;Brassica rapa&lt;/i&gt; can be greatly improved with the availability of physically positioned, gene-based genetic markers and accurate genome annotation. In this study, deep transcriptome RNA sequencing (RNA-Seq) of &lt;i&gt;Brassica rapa&lt;/i&gt; was undertaken with two objectives: SNP detection and improved transcriptome annotation. We performed SNP detection on two varieties that are parents of a mapping population to aid in development of a marker system for this population and subsequent development of high-resolution genetic map. An improved &lt;i&gt;Brassica rapa&lt;/i&gt; transcriptome was constructed to detect novel transcripts and to improve the current genome annotation. This is useful for accurate mRNA abundance and detection of expression QTL (eQTLs) in mapping populations. Deep RNA-Seq of two &lt;i&gt;Brassica rapa&lt;/i&gt; genotypes—R500 (var. trilocularis, Yellow Sarson) and IMB211 (a rapid cycling variety)—using eight different tissues (root, internode, leaf, petiole, apical meristem, floral meristem, silique, and seedling) grown across three different environments (growth chamber, greenhouse and field) and under two different treatments (simulated sun and simulated shade) generated 2.3 billion high-quality Illumina reads. A total of 330,995 SNPs were identified in transcribed regions between the two genotypes with an average frequency of one SNP in every 200 bases. The deep RNA-Seq reassembled &lt;i&gt;Brassica rapa&lt;/i&gt; transcriptome identified 44,239 protein-coding genes. Compared with current gene models of &lt;i&gt;B. rapa&lt;/i&gt;, we detected 3537 novel transcripts, 23,754 gene models had structural modifications, and 3655 annotated proteins changed. Gaps in the current genome assembly of &lt;i&gt;B. rapa&lt;/i&gt; are highlighted by our identification of 780 unmapped transcripts. All the SNPs, annotations, and predicted transcripts can be viewed at http://phytonetworks.ucdavis.edu/.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/11/01" />
<meta name="citation_mjid" content="ggg;4/11/2065" />
<meta name="citation_id" content="4/11/2065" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/11/2065" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/11/2065.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/11/2065.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/11/2065.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012526" />
<meta name="citation_pmid" content="25122667" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2065" />
<meta name="citation_lastpage" content="2078" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Upendra Kumar Devisetty" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Michael F. Covington" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California, Davis, California 95616" />
<meta name="citation_author" content="An V. Tat" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Saradadevi Lekkala" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Julin N. Maloof" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="jnmaloof@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. R. Brent;citation_title=Steady progress and recent breakthroughs in the accuracy of automated genome annotation.;citation_pages=62-73;citation_volume=9;citation_year=2008;citation_issue=1;citation_pmid=18087260;citation_doi=10.1038/nrg2220" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. Burge;citation_author=S. Karlin;citation_title=Prediction of complete gene structures in human genomic DNA.;citation_pages=78-94;citation_volume=268;citation_year=1997;citation_issue=1;citation_pmid=9149143;citation_doi=10.1006/jmbi.1997.0951" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Cheng;citation_author=T. Mandakova;citation_author=J. Wu;citation_author=Q. Xie;citation_author=M. A. Lysak;citation_title=Deciphering the diploid ancestral genome of the Mesohexaploid Brassica rapa.;citation_volume=25;citation_year=2013;citation_pmid=23653472;citation_doi=10.1105/tpc.113.110486" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=F. Cheng;citation_author=J. Wu;citation_author=L. Fang;citation_author=S. Sun;citation_author=B. Liu;citation_title=Biased gene fractionation and dominant gene expression among the subgenomes of Brassica rapa.;citation_pages=e36442-e36442;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22567157;citation_doi=10.1371/journal.pone.0036442" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=A. Ching;citation_author=K. S. Caldwell;citation_author=M. Jung;citation_author=M. Dolan;citation_author=O. S. Smith;citation_title=SNP frequency, haplotype structure and linkage disequilibrium in elite maize inbred lines.;citation_pages=19-19;citation_volume=3;citation_year=2002;citation_pmid=12366868;citation_doi=10.1186/1471-2156-3-19" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Rep.;citation_author=S. Dellaporta;citation_author=J. Wood;citation_author=J. Hicks;citation_title=A plant DNA minipreparation: Version II.;citation_pages=19-21;citation_volume=1;citation_year=1983;citation_doi=10.1007/BF02712670" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=L. A. Dorn;citation_author=T. Mitchell-Olds;citation_title=Genetics of Brassica campestris. 1. Genetic constraints on evolution of life-history characters.;citation_pages=371-379;citation_volume=45;citation_year=1991;citation_doi=10.2307/2409671" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. E. Edwards;citation_author=C. Weinig;citation_title=The quantitative-genetic and QTL architecture of trait integration and modularity in Brassica rapa across simulated seasonal settings.;citation_pages=661-677;citation_volume=106;citation_year=2011;citation_issue=4;citation_pmid=20736971;citation_doi=10.1038/hdy.2010.103" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=D. Edwards;citation_author=J. Batley;citation_title=Plant bioinformatics: from genome to phenome.;citation_pages=232-237;citation_volume=22;citation_year=2004;citation_issue=5;citation_pmid=15109809;citation_doi=10.1016/j.tibtech.2004.03.002" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Garber;citation_author=M. G. Grabherr;citation_author=M. Guttman;citation_author=C. Trapnell;citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq.;citation_pages=469-477;citation_volume=8;citation_year=2011;citation_issue=6;citation_pmid=21623353;citation_doi=10.1038/nmeth.1613" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=B. J. Haas;citation_author=M. C. Zody;citation_title=Advancing RNA-Seq analysis.;citation_pages=421-423;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20458303;citation_doi=10.1038/nbt0510-421" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=S. M. Mount;citation_author=J. R. Wortman;citation_author=R. K. Smith;citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;citation_pages=5654-5666;citation_volume=31;citation_year=2003;citation_issue=19;citation_pmid=14500829;citation_doi=10.1093/nar/gkg770" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Kanehisa;citation_author=S. Goto;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes;citation_pages=27-30;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592173;citation_doi=10.1093/nar/28.1.27" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Koenig;citation_author=J. M. Jimenez-Gomez;citation_author=S. Kimura;citation_author=D. Fulop;citation_author=D. H. Chitwood;citation_title=Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato.;citation_volume=110;citation_year=2013;citation_pmid=23803858;citation_doi=10.1073/pnas.1309606110" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=R. Kumar;citation_author=Y. Ichihashi;citation_author=S. Kimura;citation_author=D. H. Chitwood;citation_author=L. R. Headland;citation_title=A High-Throughput Method for Illumina RNA-Seq Library Preparation.;citation_pages=202-202;citation_volume=3;citation_year=2012;citation_pmid=22973283" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=K. Lai;citation_author=C. Duran;citation_author=P. J. Berkman;citation_author=M. T. Lorenc;citation_author=J. Stiller;citation_title=Single nucleotide polymorphism discovery from wheat next-generation sequence data.;citation_pages=743-749;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22748104;citation_doi=10.1111/j.1467-7652.2012.00718.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. H. Majoros;citation_author=M. Pertea;citation_author=S. L. Salzberg;citation_title=TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders.;citation_volume=20;citation_year=2004;citation_pmid=15145805;citation_doi=10.1093/bioinformatics/bth315" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. Mitchell-Olds;citation_title=Pleiotropy causes long-term genetic constraints on life-history evolution in Brassica rapa.;citation_pages=1849-1858;citation_volume=50;citation_year=1996;citation_doi=10.2307/2410742" />
<meta name="citation_reference" content="citation_journal_title=Opera Bot.;citation_author=S. H. Prakash;citation_title=Taxonomy, cytogenetics and origin of crop Brassicas, a review.;citation_pages=1-57;citation_volume=55;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. D. Pruitt;citation_author=T. Tatusova;citation_author=D. R. Maglott;citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.;citation_volume=35;citation_year=2007;citation_pmid=17130148;citation_doi=10.1093/nar/gkl842" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. Rhind;citation_author=Z. Chen;citation_author=M. Yassour;citation_author=D. A. Thompson;citation_author=B. J. Haas;citation_title=Comparative functional genomics of the fission yeasts.;citation_volume=332;citation_year=2011;citation_pmid=21511999;citation_doi=10.1126/science.1203357" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. A. Salamov;citation_author=V. V. Solovyev;citation_title=Ab initio Gene Finding in Drosophila Genomic DNA;citation_pages=516-522;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779491;citation_doi=10.1101/gr.10.4.516" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. H. Schulz;citation_author=D. R. Zerbino;citation_author=M. Vingron;citation_author=E. Birney;citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels.;citation_volume=28;citation_year=2012;citation_pmid=22368243;citation_doi=10.1093/bioinformatics/bts094" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Shendure;citation_author=H. Ji;citation_title=Next-generation DNA sequencing.;citation_pages=1135-1145;citation_volume=26;citation_year=2008;citation_issue=10;citation_pmid=18846087;citation_doi=10.1038/nbt1486" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Surget-Groba;citation_author=J. I. Montoya-Burgos;citation_title=Optimization of de novo transcriptome assembly from next-generation sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20693479;citation_doi=10.1101/gr.103846.109" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=M. Trick;citation_author=Y. Long;citation_author=J. Meng;citation_author=I. Bancroft;citation_title=Single nucleotide polymorphism (SNP) discovery in the polyploid Brassica napus using Solexa transcriptome sequencing.;citation_pages=334-346;citation_volume=7;citation_year=2009;citation_issue=4;citation_pmid=19207216;citation_doi=10.1111/j.1467-7652.2008.00396.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=H. Nijveen;citation_author=X. Rao;citation_author=T. Bisseling;citation_author=R. Geurts;citation_title=Primer3Plus, an enhanced web interface to Primer3.;citation_pages=W714;citation_volume=35;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Wang;citation_author=H. Wang;citation_author=J. Wang;citation_author=R. Sun;citation_author=J. Wu;citation_title=The genome of the mesopolyploid crop species Brassica rapa.;citation_pages=1035-1039;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21873998;citation_doi=10.1038/ng.919" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Ye;citation_author=L. Fang;citation_author=H. Zheng;citation_author=Y. Zhang;citation_author=J. Chen;citation_title=WEGO: a web tool for plotting GO annotations.;citation_pages=W2937;citation_volume=34;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="twitter:description" content="The mapping and functional analysis of quantitative traits in Brassica rapa can be greatly improved with the availability of physically positioned, gene-based genetic markers and accurate genome annotation. In this study, deep transcriptome RNA sequencing (RNA-Seq) of Brassica rapa was undertaken with two objectives: SNP detection and improved transcriptome annotation. We performed SNP detection on two varieties that are parents of a mapping population to aid in development of a marker system for this population and subsequent development of high-resolution genetic map. An improved Brassica rapa transcriptome was constructed to detect novel transcripts and to improve the current genome annotation. This is useful for accurate mRNA abundance and detection of expression QTL (eQTLs) in mapping populations. Deep RNA-Seq of two Brassica rapa genotypes—R500 (var. trilocularis, Yellow Sarson) and IMB211 (a rapid cycling variety)—using eight different tissues (root, internode, leaf, petiole, apical meristem, floral meristem, silique, and seedling) grown across three different environments (growth chamber, greenhouse and field) and under two different treatments (simulated sun and simulated shade) generated 2.3 billion high-quality Illumina reads. A total of 330,995 SNPs were identified in transcribed regions between the two genotypes with an average frequency of one SNP in every 200 bases. The deep RNA-Seq reassembled Brassica rapa transcriptome identified 44,239 protein-coding genes. Compared with current gene models of B. rapa , we detected 3537 novel transcripts, 23,754 gene models had structural modifications, and 3655 annotated proteins changed. Gaps in the current genome assembly of B. rapa are highlighted by our identification of 780 unmapped transcripts. All the SNPs, annotations, and predicted transcripts can be viewed at &lt;http://phytonetworks.ucdavis.edu/&gt;." />
<meta name="og-title" property="og:title" content="Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/11/2065" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The mapping and functional analysis of quantitative traits in Brassica rapa can be greatly improved with the availability of physically positioned, gene-based genetic markers and accurate genome annotation. In this study, deep transcriptome RNA sequencing (RNA-Seq) of Brassica rapa was undertaken with two objectives: SNP detection and improved transcriptome annotation. We performed SNP detection on two varieties that are parents of a mapping population to aid in development of a marker system for this population and subsequent development of high-resolution genetic map. An improved Brassica rapa transcriptome was constructed to detect novel transcripts and to improve the current genome annotation. This is useful for accurate mRNA abundance and detection of expression QTL (eQTLs) in mapping populations. Deep RNA-Seq of two Brassica rapa genotypes—R500 (var. trilocularis, Yellow Sarson) and IMB211 (a rapid cycling variety)—using eight different tissues (root, internode, leaf, petiole, apical meristem, floral meristem, silique, and seedling) grown across three different environments (growth chamber, greenhouse and field) and under two different treatments (simulated sun and simulated shade) generated 2.3 billion high-quality Illumina reads. A total of 330,995 SNPs were identified in transcribed regions between the two genotypes with an average frequency of one SNP in every 200 bases. The deep RNA-Seq reassembled Brassica rapa transcriptome identified 44,239 protein-coding genes. Compared with current gene models of B. rapa , we detected 3537 novel transcripts, 23,754 gene models had structural modifications, and 3655 annotated proteins changed. Gaps in the current genome assembly of B. rapa are highlighted by our identification of 780 unmapped transcripts. All the SNPs, annotations, and predicted transcripts can be viewed at &lt;http://phytonetworks.ucdavis.edu/&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
    <title>Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EzCFtBmEmiXKKVg2omVMwoSpiwX3olglmuPhg1gSu_k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459723","apath":"\/ggg\/4\/11\/2065.atom","pisa":"ggg;4\/11\/2065","ac":{"\/ggg\/4\/11\/2065.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/11\/2065.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2065"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2065"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/11\/2065.long":true,"\/content\/4\/11\/2065":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qZv38cR0\\\\\\\/1C0utcLsnTL69P7DGE\\\\\\\/koj72YE8RVeWdhnu+82os9Fpy7OpMdv59fhRxwp5U2iXGh0t+P93HPaiuiJZfDdTiHtP8lYFGZbEfBR8VFmH6hupRRoUD22uHDypvJ7fn3pr9TJL8+t3+9Y+OOzIzQrHIHsHBcwJgnXUMxNADqix1f4atqXJM3GywN5aMqosr7\\\\\\\/Oxu0So8Y2xcvJBqF2Z7+YTX3vMFvdaIsnYw8QTR\\\\\\\/EwRfwRi0uiRCBKR1CpoQp90+B9OJpFPjGNroJ8cGPyaJr4ohp++8FipaI7IIvth9aNNm3wYdXJX7nVggXMoaK9K9WqiVY1kWv7aNeVRHkmL4ihdX9MSChXvC3fAMvgGc\\\\\\\/5mChgZvBJn+XuujCa4d0PdYmNX1YFr0Qp1qO1u59vYezu1bdTpKO8HjSOlH\\\\\\\/sLEa3owL26HIQRs1vFashDIFV1YZ9BWMQm38Nod4Oy\\\\\\\/zfMtikQBOyMM42RNh2GdBdTuhEWTF3J6UoPsxWcppKwZK58HOt0yle6sUw\\\\\\\/zHKqhZYfCFUr6vBaBbuBDIsJT6nKP2kiEgDHtubYXZw0h5tzPFZ1nTc1JqfAnbZia99VaBh7XXMPMXqdfsmi6W\\\\\\\/XM0ElK+sAvmNFrXEWBjI2fhh7Tyb\\\\\\\/IXd8bvpfwtorEtcStOAlKLmEcvjtq2bPhZdx\\\\\\\/Ame8XrvOjYDBQVuNZNn0Jcqo2m0Z5\\\\\\\/UuvQKC8M68GjRBzBu9F1P8mu8XjS\\\\\\\/kHU9BwpNQTmBB\\\\\\\/ADHfQnnBj\\\\\\\/u3jBGzABGXlHzNDRprGhyT9syRgi6V30OdkwIM6SWVi8bvsAY7wQu5strQ0OW3HGv9FrYng7jbi7idoCUm4ON3bo6pvusZDPL17u5\\\\\\\/D6tO\\\\\\\/ygzs\\\\\\\/j6A47A2Xh8FhOPWQJ1jVvkKoaCG0GmGgWxbVi46YfsWFpCHI4XRH8pbiqXT6EaRHIag26m8fgLl7aQJ3G426mfBrNjCZOUWI\\\\\\\/8O+wd4EwX6VzmUxF99U747lR6YxxopsxW+q576F4q14I02vf2Tf68baGnIxkqhBXrIuqxWjdTqvenC0N0QIoTw7Z\\\\\\\/QPYMUUulMm4FJM2FPZ\\\\\\\/mM8SFLscUCu37hPONE76vSBznYQOhW4JhNNTZhJ\\\\\\\/IY7b54nGPe20ebkQQRlDrcBahXDX4T55o+C+a31whyJVaXpnYiveZO\\\\\\\/\\\\\\\/dnOUJlpNfm2SPgkeQtIteUH2g7P6aZWwDjWcSkGhAS3yILewmvOpjwksD1OPsAR58BUXALu1f27Lx\\\\\\\/rSARQbB1rlPhAhVNC4A\\\\\\\/EOJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022D434K\\\\\\\/nH2wun4NZ4XGv1Fg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aeafd73fbc48fb53fea635f79f935a9634d96d17f324f7ea2b7c67531b7e76eb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ID37KP5TWnSdah4Zmhm44IVu\\\\\\\/ajOmfkM0dbRRbtyBEzdianUgLaMZOGG7TwJerT+YxblFcyI8UDfVKNJNx7Rw4AsVZIQSdmoj4vF49p9ml\\\\\\\/W3fgNjuPI5VhdqgWYF+KLPzcBaDuGshSGn3kRFfze5z0SA4ZYexGIrtA+qhJVRIL\\\\\\\/G2kq06X8x+Yeo8QsP4Kz66yn+IxDnM2YApVVx8p5s3Lek9iz63b7qcpz\\\\\\\/\\\\\\\/65zAQmGp83i674YRZbXB5lcmLEtyYN2\\\\\\\/udCZwX5U1deylzKiWH7XTOHInjdLSKykzt4aoax6NCaKtjPHzXO+N7K3W804GkuphhbnW99rT4Yllmv8\\\\\\\/IBvo1vcaozuKDmtPCNiXCj+0K3Yhgc6ThBlo\\\\\\\/6tNv8sYVC8K3hrIJ3IHy4\\\\\\\/t1Jq8swMlArRowTlndaE2FP0hUpI63B3nssZBuPcRdJbG+ueR4hWtW8Z+1HlmgWhP3uQDXZF4XggoZH9wVEFm6FJZHQ86j8R7wGkTa3u2CljAWvf+nUiCoT3OniXCr6rXGvKSofp+NU4urYsjQSre9WVDh5DdU24eQ0kWoOoopSt5hzqep2oab9QkWDvJSfQtBekpw09lPB4B4C5hMffBGE9Y+uTiMU8sPjGnkgwDL6HTK1dg5tCsKh8BgAviLHK1fUuIOvAPo9NB+tMZoPjvOw5JzzYLy7NOFMFliwb8IjsRAAa609rhTdEGzZWk\\\\\\\/I6LTxcsXlg6qdM\\\\\\\/zN641lz782rzthrywNy2xPWKYrS1iXtQKOE\\\\\\\/n4ZVeuaAf+itRXwKxQ4VgI4DfDDMAKuxzVZcRFJx7+R9Bm+CN2j0KeMroWfZcSZwZeeHOZMbQfiFcYp3ZJHGe+6p6CyFkDVjQVTYdpbgSC0ZCTGPmyvZOxKVvuqJV4IetcJa\\\\\\\/PWkJhCb6\\\\\\\/qOhQqHFuoPt0ABoJHLQbVYNQ2LcoFuD9ovfdCsTSOwbIEap5gdbr7f9BVmGN1F91WhV8iAtetWmVv0SH6khwHXIneMgg02wvyXw1Y+NBd1rJtgpdKMnuag2qlj+TiDC\\\\\\\/TIhRHoJ2a92ZwNxKYLNWVDm3yI\\\\\\\/74b28VYHE2iiuXOf9apWgwyNDxoywhFDd3jHNVXyDEXoImgWZgNxgxX4feAPHW2VcG7Jq1BxdNjwmjOxgNjv4JdJu0rmuqLbvN+yMIAuMxA38Cp6pH97WmNRpXOfnD4wougqnlOCrSC4\\\\\\\/htc4mWSRjKQMSf1c0O6f1sOJ4pR3T4rXVkQFawnhhfkBK7eKdWuRZdoitZA4vnXrUMTn7nW9BuBcNy8LT8wDWERXqU5t5pKwy7yRK4oIO2BerLABvxaqKIoU8BKIlpeViHvO6+x7Ma8o1vaia1m5P6oxJ\\\\\\\/321hbyu+1HldIou1a2QBSTZpO48Y8Po5TTx\\\\\\\/akNlmwPC4+wBwUCzlCD19r33pkLxGHR58TdrZ8\\\\\\\/QSPjpzn2dRdcPZuY898TITdMgXo4ZPhHae6Ik\\\\\\\/qR+0afqtobU\\\\\\\/D1GhY5JZpMf+Yf4gnRtPHT7A4JEAhFeqY0s94JT3IxH759lO\\\\\\\/2tJyYWp2EYVMrRPZrqa721a75FAoYVSox3MXKiFM93mEvn1p1ue46ruw30k28Mf9WJFbHy0yNPUsBS8rc4yXpeZmhLSno9itx0eLSUPidKFx9q0ZiLOi7Qh5TSfmzajITKZAX5tVMTvAqjvdzsyEwWIo8amrfJDCT8NJFW6P8V5dg5t08fK7c5A7E+1Xj5jliIG\\\\\\\/rRcP6hpl0PXXJCX2Qyc2Fp0TEpdClSP1iANjjhsXmK\\\\\\\/ZYM78aORClfva6B8kdqUZ7ITaERKXH16r5ipLyhR8QipIvE9CrsSR4to+T4XMs9BAm8hSUAqv108zL8jws3aMSPXGDQJRo35v6dL35YgSNkWcyWpOqlHnekKcA9SRkQFDiPUTA6Vs23RzYRXWsAfyuQIZXKAaf4HnCjcojSFa08WhKsbFWKK2wyTYsSiOW7\\\\\\\/YLiYUrkQBfFdAu9nxsIGFxwCd8yNtst2gj\\\\\\\/iqTNND2VLYN\\\\\\\/qdkJ1QBo3Np54bw5Nb1poOYAPF7bCnRo+li3Knjc2sZrJaImXFRr4yWhj4X6Wixgp8a\\\\\\\/Y3ThQ6CzMwAC5LlkHrCLh4uUqSXxFc0KOI5rhN0XtutGTW6plH7yTZMjSnJVA5BTrGQIB3WrdIVFyvrUfD9J1jtU\\\\\\\/n5ZxxvXTXc8FC29ncjOz7ADiIfyKzKtM5rjh\\\\\\\/uyFwr66n7VU04enaJz0JTRHPsiywYDdh9aO5MCZMFU5mQSE1erp0lf5zoGrISy6qgnR95Rv06tJ8Mz9gMAjMOeMuRTfcUmT5uy2GFtqzRqwPY\\\\\\\/zBSsLfWFmxr3NQvzuYC57mnyS7dUwPuq4G35sTfVcAY0LvW0KBtp\\\\\\\/6\\\\\\\/s+StPaxJS11u1NAjhEOQH8IXotCp4EvWldyPvdz64bj8QfZV9l8FnSh0izawBhQqh4occ4OaGVs1KoFa4Qb\\\\\\\/TAf9jplgpb10I67vatRrD68k9UrhiRL0wuN7qcq7+XqF4Ql6y39C4Fmscv\\\\\\\/Jfrtt9TQxXwur7jTL2vj1J88Q3M98IinIpHyM234Fm6tkqO05Lp+52E\\\\\\\/y+BAJB6Mv5jRk9SpUDpaCVoCF6PuDeChjDnCsqX8tGybrtn3bQ37eWqZ32m9BipfjhzGYbOiGoiVW1HqH8Qfq6tzGzo+OJGaCO1j6wKp4GHCuaaPOkWU8Uvc5T2kSY+R8Cy\\\\\\\/4T1xEUHlM4F30yX2EqHrD5u7HvEVZi5kVF8AhEB4GgJ6irhmjEnv6cIIhsrl6TpW006+IaupGQt3Yzp2hj7TTVAhQZZosDDLfmeM44hbeUNLSl9eNNJ8IO4+ZBQrBGJrk1zQUVwom85oIw7gMD4hfk53J\\\\\\\/H4ph2HCdZle3l6YXL5oNXWaGQ+48\\\\\\\/MR7ZfS6kB\\\\\\\/ZhC4PT4wJECJro8LPL+4K18sbb9Ejl+3yhtbw5vjqQab4Gut8LFTEyu\\\\\\\/\\\\\\\/HuWosEKXDaXVvqOR6Bvc3Ozi2hA8ELAWtDUcEfVWBN+Xy769aBtddR00DZBuNafao7MZnjd0VM5rS2OGTyvMzoX0po78qpidctocQlanYrjqCqf8FNHXVQYIfPRsAJo6UlOTmEitMeWJuIdAdaQAo1Iwe3VlJiSkE39CKOBIIb027fqG6vetY4ZgzjOdR1py5f6azlAc96eCnB7J1LiqMlaJ61dHyWB3VoG72Qi1YvGVoLkv4P8OdjX4wh2Zkhy9DII0F2br5qbJA1V+crFx\\\\\\\/ZQMW50VW6ZBf8xaveggaWK0opgjdEG9XBATFlz56Rhx8Sim+lLnBs1fER5909WBz3qb9hQlHEg0auA5C0Y0QdOL3ZgmB4hh\\\\\\\/Lj2qRCoMaXSksvw+HwcJ92qYOOxL6MsBY8GRcxpx4+JHQvNcT1wVwe5K7fulZJcCZ2XI9WO14K7DjcrEEjaHjo+nfB9pChDmAPocZVl8TleY7rUMEsZ0njy9Q6HM4phI23WBiN32NwRBdecqEks\\\\\\\/CmCuiM+LquuTpa3FYL2Nrl2Kx9Rbet0VwEoUJhoiYon5Xu82pN5BFm2mrE0wMoLDLkZanioVsxGFf4E\\\\\\\/WF7yo\\\\\\\/UX5zxZpClYJshlN++7qvoTWcBfCj0qs1\\\\\\\/NOgki7\\\\\\\/BKhkSsBgXFIJoz2oWKqRPXc7HHdQaQ7\\\\\\\/iwWNsgLEfN0fvPM9GkL34B5yFenbxRdklYPfFCRbdwjWcwaLm2QY8y7pplyIaFi0+oHVpLvWTc8NVzM+wYuMsPCTJO5irTkXo9UeHnXsuVLeY8gutlN38ovf3h3N2Ce7IIIEqIeW5RbnNd8diaMRUqoys25r47+yWGraDr9jizWgp3KaFuJs6AAFIPxxEBdOwITM1pzKIKa5jYQTP5VVdZgVZ1HzqAEp1gjuOoXXztePSd8e380R5gayKWNEAoW4eWylsp71k+7mgAjZi0b0jSxrivjJr+jF4yP9qZTyRkOtjF\\\\\\\/DUEpAJSm3lk09qHUlyoSts517ORWLNtfMsJzOH7kr++hBQcBTubhoKXMom2YcmqtEDcutoRp+8PlGwDRxkRYu5nPmx3o4RmaERSRopxclftmQNIX3FNiAeoU1YJ2bJm3UQHdxHHpa6n7VYiOejMb\\\\\\\/eveOYnTclRDN5DjTYa1D\\\\\\\/E70p1pRLLJ6OXqbGh8ZAWaqM+KuBiqKFuActWW\\\\\\\/um6D6lymRbz9DgZryABc2zSp2PMwVGXJtC+kPbOTuEKf84hotlMfUL3GDqcW\\\\\\\/mhh90OVV3\\\\\\\/5SaGQuZo8TqDEiUw\\\\\\\/jh84ufUDntEJwppxyA6cQM3QwwIhHwLC6WFHdYoux9uQ+EklM81ferzsdmJz4jb+3C8QGa13qjLJ+up1XnOvY1n7Ck1tDjIOE2tT04EeOBbhrYwPcsTKb0BrxDCse+WYEriD2TrsG9GsnKPsBbSrpHHhixB7LWfazE3JEqR\\\\\\\/hu7iZ0q7\\\\\\\/zxDwgAqklrRzbOZgvT7miU+yjkObUeOa74fpmNwnnehn4xGqtHHThMc+2m57\\\\\\\/IKFDTeAVxR3GpTcTCRYhPWaVqmqXDQa\\\\\\\/l\\\\\\\/mXG9EWEQWLwArc5KFO9VFADCnvLkSERcfNzGbBff+X6jbuPZriDrxRTuIfhxrGBJ3jmIrlKToK29nfn3ZP\\\\\\\/hB8sSCHdkzb7fOT7EKAqbX3PfgZB2fy2lVN+oqU6\\\\\\\/PLKjU7bU7Wh2z0qJoB95TQpZNFIfwqMBvP7f\\\\\\\/UB4Mq9BxOxPCFNo9YTeUgfOww3bRpCfmiiICK7JPfXlSVFJU3i+G7NWMzQ+vkgb\\\\\\\/arEI9EGS\\\\\\\/yy2YZvNbFpDmpzWH6rkxeVzvPeVn4fI8vQCBCqC\\\\\\\/ogtC1zvd\\\\\\\/2mCI2Rm60Nkgv1uZhy\\\\\\\/cAhmmci+Brcwo23BtOzy+F+1WaAQ+VhG5BN6bqSxL8hxh+MKm0SwGITiVmyXfdpsmFi\\\\\\\/bEnbwxHSKE5yvRuXkR7D2O6J6Dcy9yAAcZtc30KWOcyMP+wrKWA3YxXf1IOn\\\\\\\/GRF9ha4tbA7bI1hMRtuK4HvOtw3EZD\\\\\\\/WTHyTSTQyF6pZv0FNdVdKDAIFZx0+zMrWa7NFgKKMaK+rHHhJ0\\\\\\\/2DiNJnnh0gaC27iU9Ah4hDL+4Vy\\\\\\\/7eJxd3NDJoSsXRhOAtT\\\\\\\/achOwrI\\\\\\\/8wuRhHkxoCgPeB3+Ib\\\\\\\/JY6GpC4Ump8C\\\\\\\/GB3LIdb06jp7\\\\\\\/k63B87ZDT2sA1TCH3mfRrD00BQdB628cQ7h9JWD0I9jGMWK1wvu5liVrde3KN67yS4NsCArN1auciouAV8M8lVM0hgkzMYSTdG9UKNxZfHwuGSzqEly8c3LRy\\\\\\\/4+qytBkzjXXZat+r7dxQ7TNPsdMeyyATwqwDuETSXi5uP\\\\\\\/VLEUny8VMMKswPkaDWr4r3MsOQ2Dj3xarSt8iXM5ez6yyvmDDswM6M5mNWZnsqtEtegAV72IoG2SI9LwflvbJcHezTbeDGki2S4rR6BIh7w1AN56sk9U7UZP1Pe1WYc\\\\\\\/YQV0utUtwYUeHl3z1oBnkH1ClfltQ2KZyc2TEZttgJCsNNR7i1VpsALvGmbn9XNWH5ZJlMawVx1graSe6YewbGFO\\\\\\\/z7EbaCXkYeffT9OmgaYak4LbSOKjwStuKOljW3ybn+cbJxehAKSdjd3RKX8Yq2wRizI4Ye4inrFNEwUemeMIUThNHmbCIFPySR0O4\\\\\\\/qe5T3+W2JnAB4I4XAKsTVPTl5cQlzqYVnnygA9V+1I0IDxjrxCb\\\\\\\/31cO1s0EmkuGcZmiWWSUhJeMlIu9JqIpl30mcXZfkR7AlvAJmHf6uS\\\\\\\/kKu+cJGB5SY\\\\\\\/CTMjmOviXQhqh2vY0f\\\\\\\/hSyDKTCimXisfU8Q2FXSQuaL3B6y5inpwhdB2gTbrAQnbMjQhSuiFxatTbTtQd1i1TZj2+sheAAieWi6H6zBqicpQvzD1kAGFBsW2hArSMopwB0VGAqRKPF+8SxvxFdcGJAIcubsV12ifv2NEwpur6gSfMXc7zmn2243Yz9olQXPt+zdJ+6knqThbizTVlJCp5ic9AyuVMAX9bpeFXhmqC9lPGdt14rvIwJ9TI087U+hmg+qCEyLBGxlgZPJrGUQ72xKvaWVv73MS2YPNT4F9PbuIrX+Q+JVAZxJPAGD2sIuNLAFElJnLWr2fKtRkPVHLGJqx3yfBY5outaPy1rAGEnVUGTGDviWkpHHuYfROLkosdRuIN4sHuSWPuFpVfdXtz6yRkKR2ZM5Po6S50OvsVNtzrrOqf1gv7\\\\\\\/PdZWUCLRGrvLa7kXmHPMg4apCYHs6GwaPQwEs1s4Tl0ag9o0q6mAcRK96raM2WDtZTTQLeGrCjIshLPGmOsYJVft2czh0GFF994t91l6\\\\\\\/Q1jdiBMGvIou4fGzxrHbMp80AIQUmAv4aRbkvTHx1rql8euMy3NqVrqwGgbVE6braBWgL+ExK41saxqypnLViq4vfXVUAo5VZAmxq07e\\\\\\\/s5BhFY\\\\\\\/8nsWjRlavGGWjOqMPCf65gOZaOEJ42i8Qlgmb\\\\\\\/5FQtP1KLkBC01\\\\\\\/WT00iXrjq9x8JQF64Ryr60aTrew1SW7kwwvi14aIlzTQ1CJlK2ssnUbpqVWUqtukiBY7JDzCN5Q2tzffRXWRGMl\\\\\\\/QLWmp+eSEjGNw6SdWVjVYnkWSqWPEl2ovCS7UmJQXINMGrDA4uu5kRNbCZkrxFyDaDXdb\\\\\\\/zZEO5ywT3C8GTUAUbXcrawcNmqE5\\\\\\\/3P2fxtM59eP0uyYsArH+AA9yCoqIfmgz33IjWypNmnzAO4mML2+BXqhXFl1pWZ5Y0r8lfr6ofwJxWhAV7pZj6139CX+oWNy4laU\\\\\\\/MzgyeQCvHMPHHSYYfJTQvVDBDFT9JV1ssLi0IY90SNrUmEfM+m78ZDy4A8ue4VqO6OUsNpntXlPjHmoThQs5Ch2qoNGKWWN4SvuM44CMHs5Of8RJzxg5MVhYb2t7ZfzUurENR2BqWSlBndpipuQqonoepBAaOmPG25enCQbeeRKUb6vMGKEDakVQjbpvAwDVnczHAOYeJGXyJaXknnELhPjBdzm7CQRRHGvwHeJO1hzrOu+Qkv+DYyHq0emmD+9uh5cIbrTd\\\\\\\/Qmto6NZJwCcnMEO1yxH\\\\\\\/DxHbe6T8fFX0XO+Npc3w5MjN\\\\\\\/ZtuXARxqIbqzapv\\\\\\\/KVQSCQwxLMbxLuBPriuqcTfEzhe3KlAwpNWgMI0+vu3c2\\\\\\\/7vbwCRs62FUfxn0PdLh14Uh6kUwJHBM+tZWKBaEMYuRWbyn7UNPZ9+FFWEBkC32pbmAb1SnA8ZkQg6MYX7IjP2eMlEU59TJZk\\\\\\\/3a8wAWQ4Z8H8DOa8T0O8Mhak\\\\\\\/OrzNKAqfPgsbOZjbDKJrOnj94xvbKkXPl3Y4t13L1+4hA38ZExu3TDMLkn3QVZnfdasxmnfs8vgyrF9UCJBQE1cmWvKdJn2DcniXuIKjJYHh4vQ5aP4z7wZ8GebLM8KESI9aYQnL1hbgSs\\\\\\\/IfcD1rIEvhrNr5i+OesW3BFURbPk5m7+GFTOF56bMioY\\\\\\\/gbaiGzRiZ7NIa2xSgjXO+cDIJuDJ8pbihNwSQfnll9zdsc+eDcc5k2xDpNsTijNemhyztv4n9smG8UhE5fRiVmBtC64USvHK\\\\\\\/Jb0maoyR3idWq6iicS3zgbrYP5CyjVWb+FEHXf2Meb+k1vmlq5Z\\\\\\\/ua+Nop7ghTXp+MGDlzVnE+3JOzG2WD0J+ZhMYW6Yh0JqqUFJu86yLnZttYuN2lbVAUV+GX0E7Nrnz\\\\\\\/m89GMyfjBzplBstuHvf\\\\\\\/q3wJQXnRaAtiVxc9z46HsZnlqYcuancfKspuXBxexLkjtmupDmErSqnrITfisWRuXD4nON0QDCdEmOBqbcWCopl5+cfiWqWLySBNfKlGPkAu84TNvJ9mCV\\\\\\\/xOsNIyxKoNAKjEv\\\\\\\/E1aeCsvIGL7n7hUYdFYAjSJEL6xvkgT9RQYAD8+ac8MzGm0Ch23\\\\\\\/UWv8xTQiUobV\\\\\\\/MVLJSvmVQuSG+rDG30qpyIbr4SgwqmZsEgBme9PAN\\\\\\\/0oS6nFnBkqVnPJ+RyNHr5FlozHTBhdGfqc4QZY8YHtqow+czcrUvugV9qqEdkW8kIYREArVry5ktz9zoSQc2vY1DOErQdwolXLIIW9PHLqQwVl5vwiAh\\\\\\\/8nQtYaOO4I+IrcomYRe3YXw3i6f8JOhTOgbOdLcSCJ\\\\\\\/cLdhIfq48iZRtjvSXXB03I7KWPcOmi5R7iW52Dfdw8dDKwLRbu63yqar+JKzBnfbdWl1BHeNkBfn0pTxm9sijlFqzdIJmWY1Rx1ytPK8CDvj9AW7VqyvlF3eJrdM4DnmBzp1s6khOpfUvqiaZDkYjiXNeqjf0W87quWzOktbboEfMITyVleYJfPitVSdmU6WW7On7T0zmL1blj1uEO4O+iEzQ9RRT5MYkYeE6D\\\\\\\/A0N+HhO9Deddo7KDXP+JENkzx0wwJwYwOBOmH0UoJXMzCAi5dXWURD5fSqgw0eF873CJ5UqYCmSDMwIc\\\\\\\/EqfOPSLX1xvlnpiNqVFBII8qVXIPKLttTJi9IWccIEazlNF5Ng6+Q8KvP6UU4UQLBN02qOOYx0weGA8EwOBnqQOYpN0UpVAntusX1LBTqUENrZa6z3vZZVyt8Iiwg\\\\\\\/1a8qmzKBVpkD8ezMLv6v3Vwe+jDTYfwTw1Xry3LoF400M3G5pLI9dpxfX5P9h53rHbb118Ov8UmgWQ9Z411rs7bocNbBXOG8ReKg9hzc3xQZF9S9KTcnr5VOaUfQ6W5DKvH7s\\\\\\\/JddAYn12ZTw3QuY0L5Zj2NRRmJ74BZjPALkFBKtWsZuPwGu1bPMUKuh1hZqrjThigNzKLytayyKnGAEC\\\\\\\/y5LlxgHL2kWGTqZxCoRgxS\\\\\\\/TGx18qjVUxjPne2ZTexZlq3OYNtyNor3AVtN1Any5IdXyETjwEVyj56i4Ueit9JvwuowgkZDfL\\\\\\\/Qziju6ClUMnANI3SSodqbXWyj+4el44RNltg07Cd3pWUyZ0CLcoLcukYt1wnWIKPO7LjZGu3p1yw9Fu7TM7Z1g+dQrko28tZ2CsM5NMi1R1lG3jaFCRKtbNquSerr+PfK1YXZPHWaIvUAGvWrtA7IkmZeu7ROpCIjnxsaNww8FLRFhvfj7Lc\\\\\\\/tLglJfrnNO1Lg7JnqZrrqJDLcIwkzq5z26nRm7FJ5SLpx4hXmnLDs7FgKykpJZbMhKpNkb7ph\\\\\\\/u9A67Ge+QXqvYAsIzfVO3r0VPRvcbBZxXQTNmEb5mODBHi73poQCoY+HAwjGWYsZx\\\\\\\/JH6Yd3Qiov3tgi2G6QX8fVfXtCcZauE\\\\\\\/fSK9fPYQsFpxmjFCFx9o73BT1rZYjvHewzqFUTrP5Ihg96kTxMePtkOknJ471am6fWnQJqY3kHfQQL2ki9DfaRhZ9gJwExaTdKDP4S9Dmg5TbqHV5fKLwUDXZCQcVcPbmGaY9gZJAJcuedtELFWMETcbKc4BbYhiIshtxRorOFDP+drvuh4S9TFDCRsT8OXdJYi8e3a\\\\\\\/1iMVYLAxXwZvKvDzbSdv8KVhAQx2D07CqDkPMTUqdMxGfDJ55wd+5BAjMnz5O9q\\\\\\\/3npfbpM3cqIPpakFONcsYdqSTZ4iQQ972oEYIGcbhW16SYmnPbuWF5yQGSh247876QplCKRLbOPsTg3vgO9bsBaoRRzWoy4mTK+ghUXic5yStVVZAXwqRu7ep2T2B++0JP3dahGcSwtOW53HVWl39GmIHHBxD5\\\\\\\/3XSgfYZEBE0acWirz6se43p3dsV1k+8VYhlmCBWBbu2NIZm8Mq0vMdk4BDjQG2UfTjmKXYyS\\\\\\\/legS3p4dBUfv+IIqsKtF\\\\\\\/7TQxBbQ9UxQR2fIdlZTFkuTiZAYL8gDuyIMKY8ky8VzAO69+FfwHh0gtgIJJ\\\\\\\/\\\\\\\/T3CLp1F04EFkquoPhnS2PpDI6E1407sP+YbiX2vplE1j2IxihoVt\\\\\\\/db0qpzHqRpFLw\\\\\\\/MpG1RYTt5Za3o3dH6AFTNX3PyBVVBxgp46Mzwt6ogucagbiwqjM1MR+7QURO++CGTflKmjZxNC3rjBxTbfTV6HV0IVbZVLNp8VenChyo9qe8OHDO2BCQO7bHvDQRt2QWag6QqOhRhov5UomiVETUaZjJ6n5\\\\\\\/nOITPqlS2LOUb1x\\\\\\\/2pxSDBOhVAcZ6e+zE2f2\\\\\\\/DHNwuJ1PgmtXzAZXbMfCykx\\\\\\\/ZSIRk3LyWDBe2prINpyKYflpygK8j0cVD2IfP8qejiB+qNzjnQqYGZBhEcZ7vU7NamgD\\\\\\\/TyI\\\\\\\/1ukkq7AGtxSGRx+ugn8BjqgtDzZqggrLcaEeZn1TOTbvtEFx4qEU\\\\\\\/JpbRmsr15S2Wm+BE1aZbB24AUTrG2oEvIzsvoVhFVpi6ymKm\\\\\\\/2p5xYWYFZSfTQYC3lMMU+E0ndU8dXfUQV3b7LH2bvmibUdqpDI4jIZVda4tomjXMQg\\\\\\\/pVaxfWALpB+y4+6eUU7L\\\\\\\/7GHlhZgThCcGxX8qIFJ1z72Mcc7kilyrx24OkLbAvJiThOsdQfjWw4hoRasG\\\\\\\/nmUtLWzHCOtcafZvxW97Hk8G8eVyz6wIjHbLLh6HVHG4wJRVWKus+2UMJgZWpHPnzOLzI1juP6Vwiv5lPlKdLedQEY5TWYZ7qb70kfG1Hd79x6EYnnf9xCWBIQ04AVG2s6LH0OYCu1KnETPl5qwsID2f2xC7TFh+KXMsL+cTk\\\\\\\/u0YOGY+Eyu4Uj\\\\\\\/rQm7CKqI1kW\\\\\\\/kGSyOgJHcaa\\\\\\\/ZjWQKugQZa9fFLbH1YFERVhCMZZGTXUjLgQMl6hhduDW4i01RgB8HEm3rzjvl\\\\\\\/RBmmRJnWUcQUKZuwXCZV779XZjRpHAIXS7tZZnwzwKluqCwZ1\\\\\\\/Bqfq01DujzGZ9+pNrfXCDASmeRS5ZuiiZV0XWBTTSgcstQ3mNJCZunL6c0rbwnz0tpgPduHFWmFooCjnlEDqRNiRyCiUG+gZYNuv7bcOSTygMTqNfV1vwOk+knio9\\\\\\\/4mNPEd8wBgvDo7nRFRQaiN0u8dxbyhZBVaSbs+yayfflOzduBNItxWX1KvAyVP0vYKd48yEYVJrpU0r9\\\\\\\/JJE9FDnc\\\\\\\/3r8V8MP8VC2OfQC9cKeUmQjiNbn2bPUg4wqVGjD7PtPS8Vtoh174+UGBx3T8fFFiVs3NFlfft+4s5HULaHFPCgPO0bY5riDFPfeHb3ZP\\\\\\\/xA0Zw9HAsSxXM59M9yq7FcWhlI9jvCeIJ1sdlmoRl5YawqNkXAqs0YBokLHwTSewy8tKgb1YHGEZqfFPL4n0gPvvL+dWaOGyEEDQ5vTFAm3Y1HvPyUY1kGh1FXYqNOJ\\\\\\\/RQ1Y+SdKZQ8\\\\\\\/rt2euorDesi+rjqrYPd3qxE3c18A4fnX8IKOPo6wFSO+nYHPYzARf1hiLnUf14zO4R\\\\\\\/mSnJ\\\\\\\/iBf3d5v9x+0L514GDdaomTR3tq\\\\\\\/ZCMYXan2DQeyXr7l80Abl9ySGWmX0TFX5gNbgAgVHRV+Ec9WVp18i5ZyC65Z2nKslzgTvFgBCBeHnCpp8dmWNx0ufBjbQEV6idRie+TWJc+ILwsIP1UiACDW35S09VQXwJROaMd44c9OWIQAQV77i0ZQg\\\\\\\/4uMqKuKQsARWLXYy\\\\\\\/oKuga\\\\\\\/FkbSGRz0Q1OKpbzMlCnNLHG5BX6OWwQQuxfqFih6FMxdqClmo7zwTfsRpiJYwLLEsNXPP\\\\\\\/v7P4+HqyCXia4mLTLO1m4lBdSRe2r8NJm7nP6ykjCVtkKjXD\\\\\\\/HVaG+5e32YMnrlcSeGZXjbCM4e2SA9wrTy5UHYBAbgeayloZcJIZmPulOmWN3WDUXZ4rculx5SMNNYLPnmaL6wSQG3zYHTt9EmA6YwmeyUZ+0gcyamVdV7BWpPnbGl8pzqZMXL\\\\\\\/CuxyvOn0yyFSAhZJyqZ2IQtj1VyWgwR7bBQeXOe7xqjF8jh2vCjmRshebk8Etg7bVTX7vqo5i\\\\\\\/j0M7vfa22fhe+qAVg9+Txoa9z2jWtc9XlRpTnme3dgS7aVK0kTpg6kfLm4u4jKvfyTBh5eU\\\\\\\/ZRUPYHRGJpcotD\\\\\\\/p89dmLu69\\\\\\\/flcdhASFpPVKwW3rYkyUvNTTIB3Y5NKe\\\\\\\/CSzNOMwqG+bLELgsd5SHqtkBnQtYFAgfvoyHDVdkTAA1EeNYdJBPgzZuffUQvdXVc0rXwXTT+GKWOF9pzNlBThOaa4xznNtpEDim+SfXMWKtmYx5P8l8xckoHbo8ZgVaulNkr8KKHo0MafW1EnvHUxIfYWeoZrQar5kkmetb91bQPELA\\\\\\\/sjMG8nqikwZEKSqPgNZptL5MttPUeYVMhU+AevDEaR9m2XYtxRuRGukkcE5P+xQS55fkBLzZazUjGe+EtZDGC5aLjzJHz26r6\\\\\\\/3TEjSmwt3kdhqE0hw5u3rPUH+0r6iSs3Cx4TZNVIKij2rxjRUtZn4DhehhtxB+lI\\\\\\\/yH8cQFmnnogSKu2ERZW0H0rhM6jxpVAW2HGha6I2O1hXV6QOEtJPBM+f5X5Qi5rUhFluC6GIzWt1I3OpHuqkPXC0+UhtKsVzHpgDDl8KBB78UPtP4aGXoJuQqEFF\\\\\\\/yYFPHu0MHN+f\\\\\\\/xZcBzSp8YjQhFg7vRvktdYlhRHsUKcNCOxvYPsP3Bf87aTVezYeXwxCCHql8DKdY8RFo5Yxw19+GYOnzkoW0IRSTrMOowe0SiieAhbDkT9mmf1+eXQ7J1t7VBdKDTjX5xHaDTsBxfxJTJ7dxDdrAhoEPEGE\\\\\\\/Ppi2hCEOnQmebvJwcHnopGKKGV86xpHrHrAcYn1DUFIFK4yGy8XgIg5sGOQnFkv\\\\\\\/HFBedMqUUCnR4lYNU153SL8isZX+1Z1bQn3H1ZCnLxmbwias9ST+mYbem6vm7ybztucYIJL6bRVWXmHh4yv7qUm+ENEhQ6e+RZJICX+8q3lRiDUSjNlT5JwO5dz7e90Fx3eIclkktO33NsXcjJgaN83rmY4Cq3Sj3DvsP2KvUAic4fLXUA52YMxyssLufdfTQGxKeIDKtek3lHjIW8IWTuvoh1Lmp8vMneKSHdspbjNbXXPTfKD2OQf7NbnRiO6KIcFbtRXC7jGeFjlLaMd1r+PX9i+QF0cVgd9jmVevztIMSykCFBpXA4oM224Z0TRpar4BUoRAVT+KFqHI9JjRwDjXSXcriM559p1rEyjAJNubDalkBF4wF3mUnjxYgkMvJuBmDfFh1y5oPVfam53S87Iiv2hg+dshAMq9DQSI6LOdVYVsdBf944ppGt7py1D7+c8IC6DDBitsj+FoOX7CahQqRQQtYGByyFoWIMwDBENvJqZuo0cBJiLY6KB8OOToZKtFILMGp0ZvHGu3eJi7\\\\\\\/dPLgcjcHQyATOs5iBPGN\\\\\\\/wP4Xg\\\\\\\/pfaeccGOMdSGCDN3vn7+6o0RMpyNuYFXdEg9ijiYEfICgHXRuDt50xumAg3pNjUqNHAMszw66YkE4xCkyw5ZfIt1UptsS60Htz14axg\\\\\\\/ePRMK8+ra\\\\\\\/nkYfAeDtyptmQ0KWUIxbAj3HF8BfM62R09GVtkNfZ+m2DhylAc3le0FluGyxycAujtU\\\\\\\/3wQ97sE1S2Am8MlFhkaud8dbclQIC8OmrKdXKoX6PFtxSiBQndb34D6VUoZTtUT9+DwK+8nQR2EKi0fEmIuYpW1PoyoVk65IC6ksUlvKiVLoTQ7dws\\\\\\\/p41tgXAUpLHT6FHuWgZp+QqX3anv9+rX276r2CQaPhfxnEE7j2s4rszp6EU8\\\\\\\/ia7uPg\\\\\\\/ekfmYCf7V5GZSALzezX2TK+LDau0PMfbGsJgnhDzMN0azHO1wQryu5tAjU0Gy\\\\\\\/j+5YlkvbDVwpWqKn2uQf1+BW+BtvfLylR06tJUJ7DgGs8sSClEQGQwDVhsGQ+fe5CrWoymQFBR1uE3uHacac+Vm1jF+LAStVUjg+alZfkc8YGPs5M8aoQW93lVhkXWp2rHw2DN0QVj7q7Lnovl8NpOuDijAbUgZmPpmrwTqk46YU8gFXWimaYgSO5U\\\\\\\/AYV3jnHGQIWPlzOX9RIsbc7z1xvYnGDDnCeslYCJnKdPVFhRdSVAa+LI6PZ9iDZ7iWV5XmDfn3U8SRjCjOnSU13wS50L6UtKcQwRpBgfUWtqiCjgcTYmpjs2+UUYV3fuvcGg8dbGORyv4bdMaLdwoBw0pPkBQGbpBM0qRcK6ZdwKYGMfhUP+D39DGUtlcA9uOkmn4Ie8x1dPoMmTqGq+H2cSAECyXT44\\\\\\\/\\\\\\\/QUCmrmXNUc5sqKcqrSXrcUsgyg0rMVtOl5pDg8uEtGhvK\\\\\\\/Nv9syQIvT20QQsHJkKmdTdTQHDMIc8orBZgzI+xa0Wf3Q3fLrQV2MI8qi9wGx6MtoVPeUYmzQQHP1SGyl49lXIfRzB2ynZB\\\\\\\/3250QUKMXEe8xyHjuwyjYIM0i\\\\\\\/7+ymtoVQdV4a1X9zLp7mefC04lSyhdavU+ep5wUmHJyvkit6ajT35XxAeDwtFBQ2TwDr5tnU77Cy4VnW0QCIL5TrgBPlZAXENQGeHF+RKNNJqsD8WUN7Hl+V4DYGM0\\\\\\\/ueDi3XzmjNt13myzl+K7LS3iTtJ0Zvt6NCiy\\\\\\\/9IB29uNx8pvJ8Uu6xi8rneJn4F02uQa9nVeUNIr1CIkwpXT7himZuG\\\\\\\/2w3VfISwZSO4RWP5361m4hNT+LvR5Xcainc7tThelNNqk9c8sOelH86JzJ2bY50BP2vNLEtXbm13MCbwFw9vzNEolNELYBIQj4HXr\\\\\\\/rdGPSaeIvDHS6EWhFSebPcDnrlYglDxRep\\\\\\\/R9GL6hqKPYJuSD6M1QGXzmA\\\\\\\/8BDKKSfmprdgrALyXbEY5OBb8GDMIJcDZMrRubgDhjCzq8WWHPD1qGn8u7AuYMQUbg77my1HSy8rv+87iVVfdbFgFNqkqSjSsmXq3t8LIbBJDPsyngO5N+6OfpOnxkmnEoqPQ3W7uOgGOfoEnE7PoJJBfU3bIyojUoshizJ\\\\\\\/jVijLD6+9Vqan4lda8wlTKl41FIOe\\\\\\\/a2VwJUozxyhHh+v4ROVBpUIUkMnrL1FPgrZ34M2wCo2nkfbuPeSAtvs8z3nFvNpKbzGlN4vpwAWzswkliKg4MeFcyZRjUigo76PIj9JrtG3pSHI4OI+xQYf5xUCpIKXTlgBztKlZ\\\\\\\/xNELdfDpDO3YTVMo+BN+sNJCbx8EaIT1+W\\\\\\\/lXwAeau\\\\\\\/pl43UTossSP5lbsBGi04P1o5QoAgzM62zXPeHv45FOMOaB+DVPM6PWosv6tRJXTguFvVBzQsd\\\\\\\/fPhP8\\\\\\\/RHs+r+WhlayGFOp0H3b5r5OZnWcliTykNA9W+Pt9r7smQdKPQ6pckFNIjZy+u3qu1a8nydhw3pizJg9wv+fOgu77K6Uyy07F5zQB0ppzI2W1vZaFCm9XqjtwQyNtGGqliBHo\\\\\\\/O6vMSudU270VJMSILEC4RJJLwDLE3iPBMQlcJlzTbHDVIQtdzK\\\\\\\/4cAhSau2L3\\\\\\\/1HjjlCHRrRT39nAMafExd1NP+iD02mad9PXkfOEHocg01cxr0ICNR16tzyTDSi3jcvjJ7ZYtyIgiEMZVo5tMh4aRXbr8rOM+DTeeuW\\\\\\\/1Kp5X8cU1kk4yuPkmi9g8lISA5\\\\\\\/2ThiDDciSry7Ko9d3NZ2LsonkN8nVDN+rmMV5ME9wzk3c3hlE6Bc3XDSkAvz1lBuRTlUWU12iq+oZ7WDYud4qhaKZcYuaR\\\\\\\/17sBxvBmKJ40BC5jxattAEmvamRcGvd9kXeW+JLw\\\\\\\/K2n97jzmaQ\\\\\\\/6gzvS69AqMsJZ8\\\\\\\/Sc\\\\\\\/DXxbVrRklclqRRkWIyof9VtY\\\\\\\/8V5VgcvWQe6gn3aw5ud5AZzAiAbOMy7\\\\\\\/03JvNsLTGNPzeCrDjovUdmzR7DuMBomx0D1c6xwdyqAwEWHbw6PvOkZa5x4SYBi4YKyasYbEDnE\\\\\\\/OG\\\\\\\/YmkPstUlTcvhMcPDeqpq+\\\\\\\/D0UeqPBlmoCM5pxZJiNrj47Rw0QhDKAWICQT6JUnlzjuKspssafnTfaJkWXC3CjKAykQuUr0h41SXcJuXOILX7jTpUVNlA6eW28SzVhluSlLZPWvRoI7FQj09TZhqZbRDXPrDLmmEu2sjQmHO4Y83BUkLqzDljxKvMz\\\\\\\/RtVlRXOoM7a+VJ0OeNC1Try4yI+q\\\\\\\/O\\\\\\\/1i+llUKNVxtUExdtACKOO\\\\\\\/S35uWi1Z61LG\\\\\\\/PuMMylA17HCxBvCgirNIuwvBDtkDaAHWI64sHeH+VWRm0LJFeIKyHWAb17JsclxMvHk5x09dpSgf2oSdazjVcMj\\\\\\\/gWr0Sr8CItGPYXHhMmPI0QamaWQ7k0LLn48nTc\\\\\\\/WnFxm8bNLWoM88p3l3xQe35SoND9KSISGnzRw6V7QfiNpkBnsOY6MEMqvkLLuz9WXAfoe6mWEZ2VkVHDt+X9vXQAI9jE8q0NILTJ6B3ai7l7pFDbSmFYpbloeV5T0MAfswhW3C1E7oDNr3DD7oujhgu5VevviCJ5ZqoD0Jfk4+9uJ4WbX1l2dTBgwWx9h0VdJ6sOgRUOO\\\\\\\/9UBg0zyRBahgufEhrN1mLspHweuqzw+9Z6Dgxz43hUlhOMNljAtf3WPMuvimXlD9ft2tMJLO9h30eBuKDoERlmKjtGvPImZtYWIoTibQ+bxmXrpbtzE63utEJ+F730LO3EK7fFkRNf489xcVtECYXl5gbsMTjaCX+0rRj0G3bu8kxsZ97uDIZcEOsnJX4UhYSeiGaQ7rzrIAuol+IvEorqSMl0iEb0jFmPJfWw\\\\\\\/0WJbEiPdN9ZI5oVzhVWCTGXkp7T+Bn7j5Ffs+AuWw7OF9b1+6+Lp5A5vMk4WFy+KRWVSpz4laHnelAV9DY5liHOORmCU1ZmdrQpnDw6cW1RuI6v\\\\\\\/onCXlQLGkm4FiF0W2qkfI4BHVmxb5t5GPvAFWvzSig3VzYpjIhmrmRmYDOsZBwTnwicLXpJ62jFxdFk49ek1ORSs+nI1IVAInOyQFxZeyUgdO+vttzFTMio0JcG5xsFV\\\\\\\/OviyN6VUWlxhphn61+iz2K6f1onGLD79YXaX+KWQf02vuaG\\\\\\\/xUV4LWPmlkfqI0NMH9DPXMpyiIlovobC4VYTLRCdoO\\\\\\\/pKuLuMJsfz1HuUppkR1K+CVT0LVYms7\\\\\\\/FazZnqAvmkVe2VsoVHCWscwmE5KEF+Jq9N97Lf1mN0xGGKMlplHhtGTvZG4LzwnK1ulx9aULR6o6ZDXo1gG\\\\\\\/zf9E8kezFOHVeNnx2cXh5YoW2QdcE48RgR6BuU6qNKOCuqnck4LHAD\\\\\\\/eZGuFBVCSVzw6B69h07GrTIsGQ4QW8U\\\\\\\/jGdIrr80Vz7NIfvwBaX70ewPaiV6mDJ+06wiCNhcMJbl9Z0rW\\\\\\\/QuIKMKnF4M410BxoG1Ww4nKfz72xdsHW9jBLShgE3\\\\\\\/Fod51sJIxs\\\\\\\/r7s9bD6dGB9fiop0dzKUufRzA9ZaTgEZq+kJC3rmxp+22EKWMzNrfZ3jH6FAhCJwFERO7yZu6A1Te2hyK2o2exJRiT40GCflTTDPsGXOEjeJHZREMpR5+hA9sw6ctZPocWzjkT26tU+L\\\\\\\/cgFUvBL+DTkjsA7KIuBB7Xjxf9my8Jc5Hmvriq5pv79Uh6ffELMXD9d3BjswPa5NsDZ5MIPSPsaKdjI5MlaMM6PQD6eucOy6kbFlWx8fyjD+EN4JQbUXXEzdTwnFS8CFwyBEX6SJu+\\\\\\\/RkLUv2FVBNHCnn\\\\\\\/L9h1OrcLZdaVgSx2nP9pW1VJMhV0ARqcc\\\\\\\/djQXrefQ\\\\\\\/ZazLy0lduPBPdPV2lE+eEFnPEuV7ORZMYiiWIU5zKz2bcbhQfV37MM8LOxxB9ntKb6i2EW44Znnub6bFCnZhvwfwZRUdbPG97kn\\\\\\\/PQIMYVjfJL+90EYrLA\\\\\\\/OfIWXTWi5kU99gsxIrn32PM9H4vlx4\\\\\\\/LKo1AiilHclmIUT7n8puGpCIHVg20RJwP+CPpRatyoN6N1jZX8AidVQxDlSpHdjCxEfuA9nMNQa8Et5CMJ6tr\\\\\\\/CD267lZZFvo5Lt\\\\\\\/QFQ7omYkqpd6J2ECcIoThMtl97cgBhd42KFg6Jh9QzkgVpFlwulCVSoJxYYSrEv1j9ta2MoVRAvDRhACtV\\\\\\\/1D6HXYxzqnhsHg1UEBvU\\\\\\\/1qWYvdJM9+QeawdgoxvwQIsFogSJEo0tXpk2L5PaH8HE1UlVUeOMpbgYAurtWsutWxQFxOQxU2oe4b5Kg8QNfMjdFSP4gEpNpXpljiXMEBFhnkfeVQjLcwlxGTnuxZb6utCw0\\\\\\\/0hLBApSmy6N9NNigufO\\\\\\\/MkN4NmgodL8tpjI8e4bbERTyD3E9e5iSLg+zzdOsSLMd27S3C4wMIVqNW0BG0HlTwm+4o8DvGKMjmS7YacY4behyFK8hxpZc4oO6k\\\\\\\/QI9NYey02ArUiJPYna0e+QWu4Q7\\\\\\\/XGgNTPKv19olryy5fwpXnz88KpCufR6jS5jNDXbXHTZNc0E0wqkXyIUgo51IsYuTNWV7fazM06SYervFOBauRqaChGqWM8BjQOTouERLPVFfVUKSvCsqb4JVkllLriG9MYg8P91+m3T0YQrdvd7ZptZqeHvVOo9PW1+N2dY5NpJhdDigpLP2yV8cJn6cNgpWCzyyKUm+QfpPopxbvjMb5v9t5O+8Y3Gga0ADuJ4D1JKLy4OrLHkxtEYE8Go06ToEtulm2kGXUaZiGYGMWdXABHkRAN34ojGvp5OK29O9iIJpg7hTuSmRW5UQwXBboZHLh2XVloIl4eND8w8mc1rDbMZlGDr3dJZoqK9mo\\\\\\\/FHAWvzS4P7MAHxdNHxZrFS3zjNHl9r6Ns3XW9nFL2eQZ3PUS3RXLCqhHTocy1MgG+FAiwCcOEa+eCMwbwEsQWLIYfVvIb9G9PfPz6oY5lCjdkcEhIbhFyXWft9OyRnfBCUKV0pov4y3IkMZMT7P05XJPeqP0vU+YRr3Muq0+9gSfU5VnqT2OiV93aXkBwvfZk1VZ6vfMQjHL5NUNIVfg9rHclC07C53cU6rR4j86i0FhOxzfkKWZAJAC2OOjnztTmnAQttXy3Zow87Q2F\\\\\\\/as5i+GaGgZTe48uUiEtaShCtiMfT44TXn2lnsPpHV\\\\\\\/55LPTk1l+THyGgQU7au4G+0uVv7+RISR\\\\\\\/Sejbo9FJ6E1857kZPAbn+GRnfpVOfI\\\\\\\/RxctlJaGo75ErRrpAlRDHXUt31d3G6CbPl02KqHyZCuMqOanTPgzlnSsohMXUsdUcXz8\\\\\\\/y5v7g5MEyg7QNri7weZsIvHEM1rRSbZfIZX4215esvBLEIIGlyJoPlPbrS98u+2XQmAKY0wMF41BNaYCYin5\\\\\\\/IX6adAylG8dKA8EffzMFPv3VlR01tjmeYs2G6XCOoRQcWKRem9ZIf14zIoC+csveQniQXsYvUcGKrNCpKUkwSXwUVfH5HnyNP5icb+ZKj2Byv6yrB6NB6N6cDaQUCTSmjHOUADE01C\\\\\\\/arWLA06AYu+gB01J4C477doZOuHPufnjuzv9FK6YiiQV3wdnXhzcyFHUAc6Hyo8ZAByZumERtQoSl2gFAKCPN24Rkka8CfI1wToQa70sUzcOxnqYlwjMHtVsx2WcwngpouQHxufsZGUZMcGEEjRAqDyu4vyY1qkCEYpsxC1fWm4SYmLXldvABLxvXLUnAqSybl05gSpCeTBQDtEOwMrQTdgZaOUuCw\\\\\\\/WChk+B8ZICZ19e\\\\\\\/\\\\\\\/yMVYAW7rBKdnqNxQfABDupW23EK8Oe4oOmnkwTYVaMcKu66uefQtQl\\\\\\\/yQU39c4GiGh1OOxzTtEWlUUp3LGy8iIaELZUOWL7AGpcOqKJTnfOEe7smMqrDwXd9b8rBkkmnYgV5N3dwefWc\\\\\\\/rQBTuaUNp1gqQOuOP5\\\\\\\/BBUFUD5O\\\\\\\/8t7+KOJcNO8YCyTifmcws1H\\\\\\\/sIIZU9+hF+VnnZ689Ue\\\\\\\/N2\\\\\\\/zwbMUk96CSSwrDMumryc8FgEiGvuW1iniGdqWgX0WOYMzSty3pkTywi0n97DVX0yMYTb+YSiwCVyQpKxdT11qMYOJrTOMYcairQo5UkaD0LqHl5JaBEfnU33NglRI4rGNniphr8iQBCp6RQMWuI\\\\\\\/k+VrnK1Za7SYDfQT4RpbF9ZARPAEdl4OoWakBd\\\\\\\/4qalW9sb6G\\\\\\\/HSVsLeMdWlVSMJtjaVeUhv9gyFWbLQduN4Acx4vgwxDw0AFXF8Cr1XJ5DsNvwGLylgJ4r2Kq2QiwmmyFd7zt6MA8XtkjbrUAUF80WDxGDy4+uyGOR+QHaRz42YAq8YHdAI9yH6QZGgurF+POU6c4vNhx5fOOnRPbY2qOK0WoeXnXPhvuIWyUcqpVFpv9EHWu3707UYzttXFeAxjRAF+QEOp8zYmpATCnKuF20WqeDZZHxsfox5\\\\\\\/WmiGtCZh3EyhiefM+hTR0e8dI0jDl1m2XznAU8zwrmLaeVIX6305useIkPZaylPQFj\\\\\\\/VtMC0Icr8r9zB9tAPeeuRzGyeCF8bwf5YFUvdAkKYn6hZpHH7XFvfEm3SBruId4kTxjrECDl2denPw12PrhhrviPWrFqQC4O8xIE61IXNVv+yVfU+SCYo6jJpDbGpiI19utdL+4qY+w9ihLgpdLCbymOjaJ32xYhXQT9++hzAVu5kkglXx4MreYTpahytB3thpd8rbtFGioMhM5rLH5FkdlrLvWvHctO\\\\\\\/x7lP7pt2FEq2REB5P\\\\\\\/PrLsEXswP+zoIgGLBDNX2hwefnmk8+Wr24CkKRg1Fi5N2x2s3utdXh\\\\\\\/9t+TxijjrHF95hyDDv7OddehuJL05CvVXncbhfN\\\\\\\/2WXwWauLE5+EpXS249x+EeD\\\\\\\/BALU2A4pgr\\\\\\\/tp1qC1TcmgVU7PuByytZ\\\\\\\/M9KzHL7dVDueDNUf7RSZ+TOTchGYfNvufK\\\\\\\/ceGUbhA3HsoO8C7tOI4rA7QdcS98jgxhjxP+LEe38r8xR1es8S8\\\\\\\/pWS4SuS\\\\\\\/pN8NsFYxttpn6bfq7psEbEnynlP\\\\\\\/WPDWSI1LAuoFLSyDNiPeys7MQ+FEHQl\\\\\\\/P9kX3SpMdAzJqQrm8u5EueovfDE49lcJm66\\\\\\\/Ge2OpZMu+sSZyoYZgaImo0QZTF9as9zCwGU\\\\\\\/dQpZsMGow\\\\\\\/pssl\\\\\\\/eVfgOG1gF69KscDE8Hx86Hevc3eiL0dh2XUH2TA7Ls5CzT1O1LK5208y\\\\\\\/uujNG1eDQYz3YdvY9niSHR3wIzuovfoIGb0ls\\\\\\\/DkRnVyDSMPHMsNorIwBdVSPQTmRGMw7als4HbTLw6I8Pj+Pngt4xH3QaOq6qgDq7CovWKMbaAB2MIJb5EyP\\\\\\\/COMKwu7sffo1m0jr\\\\\\\/meK\\\\\\\/FsV4t12Xfxp4u8UY8c27F1\\\\\\\/dZZW8dhkjaQdrYhlfRB532YwLjl2XzTxI2B8IjxLWiL2B+USRMKIsNW3MukIzKhtKfwvAh+pzFHDPS9K3osWryJ9MFnwl\\\\\\\/HubDhLN3K\\\\\\\/Wlifsw6HNLjQfmCYH68aV3o21ZhYcMjbe1pGgPs5No9AfLDhXKwVkBQ2oUv0lxU9VOAlvrhcAhYf2g4ZU6WHL957+tbvbQhyCI8hUFHfUBvem+Q41Xd+pEd2c279yQuISN8DpOd83a4KlIsI+M27LFWeMq8e6JfCXoZJlZB+YhqRuaRYm4TZGSHHtuLHMasT5Rv7szvZdYlpNysjS60mKD1c1KGu3vnoG0IQ4Bwue8C5iIlnzT3qURIXwRkNUlqIrKkhE9s\\\\\\\/UafNFGzBerGkRON3o7ujRlsMD2zZId4U8LffhRUzGX0UjP207vZPk\\\\\\\/Lyp+fgZ20qAQgllTmKC0o+bIg3+\\\\\\\/ME1mqQfEqgdu7s2a\\\\\\\/HVTyD+O\\\\\\\/e+fiSu3zHViyHCncBMBOcjZWvOrhQn7wIJ8kNE0pwb2nLR2BT58hL9U0Zz7fTQUzRy5I\\\\\\\/Ies1ajL36mORhIFv0rSEzTLFqKm\\\\\\\/LQotX+0\\\\\\\/EJ5OmnisLnbp1ZTglHI+FhzkrfE\\\\\\\/3ks4H95mQFGEli9DI37lHPdtjUMy4RTbYJA+YXEEQbFCr683cI9Da3ZyVFXs6o3s3bldMnk8hdM6QpPF855zQ7Q0R6G3bwbRbe0oNpzKUsXNft3WOJ3tnhSt7wGXHQkkskWtToWSA+XHRUHpZn+jVU2pfXg1NtECCW3mK6SRVSr3ZokeBNlH30n\\\\\\\/VuuNBHbubz88y+2vXMCS6eJKovdvWsoqNdLSFSwkP07RvACJ9Py1R8i3R57o+l18xvgQ7jnCvdFeQ3RcwCn7dAPIA1Owaf6nnr9f9SD7b0zqCdGiviUyx64G8VEzFG62obySdIcvYNIEy+doOvEavKOtWzz5W4eBcL\\\\\\\/ojgsf371ElpVzutlrrXHI1xTDbZn5ETfTrzpJXGyGB0crL9jutdnZwRL3xULWVKmj92wau09Vfsg69LfWxAHlEtYgmWIlWjKSf0KxLExo0ipa3RblbOoq015gOErO1J1JKgl2vJNeOlaXalwI+pERIGb+paUXSEjhz6+bmtqL\\\\\\\/6\\\\\\\/LHCN5YEzp2yPMZ1uMGcooQEucEtmB+4cWow6EgXQRkYpVbd1zdt3KHK4hHpJR46EvBkc7I+RU0kCER1xmkE3Zl5FjOs573Zn9yO6lT2S0xEf++BqDg8qQCHqr6oR75v7fsVqXRYwEtIdxJA3r2Rdnv6tSqbrOHhFxYfiKbQHZGjzymf4VwgOk6CqjpxHF\\\\\\\/zXTP9kWsrfsTiPxgKcLbEReX2O6qXjz\\\\\\\/3dcF2qy2d7\\\\\\\/SGgQKvkly+9GnxLvKFKzOWaox1tbgaf7YBNYeJ9Skl4wa1N4tt9QZSqj\\\\\\\/VxyNleUCM4y6WhC8gRdVZHPh0GrlyjpS4DMJ0qutiV7cMhsoWRjl1155kH80qsK2EPew+dmjNyu51UnMk\\\\\\\/0RMw6+g+ctcvO2OboF8oLCMFksHsmFqaeWMMj05j0mvpf+myvAgoeeIPjy\\\\\\\/pM1pRI2JdsZpSaUEjRR2VelYQMQ+bvttFEKY+28x3FE6bN4T2RSk86LBn2gpiWblWKTQ\\\\\\\/GbJ241TOCznkQTQZZfUlPriwjcoqdbuwc1K5qujQguz3eXFJMt+F9epV2ndcp\\\\\\\/nrsuZM8zs6Vv+4C7hW9Fg95GxvvstTAmBuQzYCsK1dSnIg1W2vRmQySLNFN+\\\\\\\/VOl4qkB88VJfAdgecrNwXkCkj7jVOkVTzJjN22v9joU9ui4RZFNRdzTq7x8MatwBXVYxbXIBzg6glW+sGa\\\\\\\/VaMg2hdk1xkKCgeLZdoa70Q22VJxkNj3LiLnexjzLRwkv4XUR3HGcR5w7+jBG+SSn0Zl7x1mkKgvIr1+vJ4LsXqvVzSCBDI9qGclfU\\\\\\\/YfVmQ2QQkxcafzCxxKuRZeBQa\\\\\\\/apnXPxoG+mLIIJ16QDHjosK4KKnK0HD2OlTQJ5tm2taq8aCDpppx1YJETJDGAurjzy5rNoW6N41lBPPbxFweskCjj6D02hEm6lBvVZGhX8CMX385rrMOhQF2OsqpILgu+ZMbOKMNzykS46kknc6WmMIs58xHAsqMS6N67AhUzAZoc3BX04yalfeaLJJA\\\\\\\/n3xQRlNkhSd5xrDsSpNTf2y+K7ESXhVLY2gZiOg\\\\\\\/8AUrzL56L22E4kweCIztdoZy222XVcsIiEGQZuByVqjz8fnE4na+931DDxyrdRrY4rGOGH6z5kdg3tpwvoJQUmNt55Op0QkDdTifO3zNQhgM1JJBCIDjZAaRyzsA03OSc8lzCeOw\\\\\\\/kZbYSVyGEG+TlOodOiYqMIX6IjJojQfp1bf1QYnqkYzqLOG23OlhT\\\\\\\/KlUJd+HE05K0Wuy9iHzVR3nI9O+vcRwij99Y5T6bdchkuYgLZbxMyvenjG7N4Kr\\\\\\\/XK1cC9ZoLA\\\\\\\/Xd41Z+mVd0oT9dKsy+3KsFlRS3W\\\\\\\/1U+8ROfmLVpROpGAPES+QNkhjj6UrhpjhJtmR3gxymglagHPF4m0+w5hCGk7L1GdFHLI8w69UF\\\\\\\/Bam5f1K+F2WppURZnvWdeWU4Dj7HBL+Dl6KCRjNKmpupK4nUqAXMX7NFMRVdzwIxKpzAAjiaN01+Gn0hgnZW5ojSYAQk3\\\\\\\/h2PjRgQoX8ZO3ia185kPQVHB6Er8wuJldihvhDqAq4mgJlzbFibhO\\\\\\\/7r2oWOUev9Pwzz5LzvWakoSSjxA7fJ+0YTK1xEDQGqjYGbJnQUoXUgbUbr\\\\\\\/mN0IqqWn\\\\\\\/bPKkDsKmJl\\\\\\\/2\\\\\\\/GXq5Bba69h45sIQi4r6cli0coIAAcl+UQW9\\\\\\\/vCB4yyTQGZjgEgb5YsRwDWevq8cNl8EhKy7afop5juJq6psbgw8zrFoTzlJFhgUEf4H8Ouozhwci6f7x1DozWR18KY0MASiI00hwB6OadZyc2TKks4DA7GpUMpLHkNbOaid61+D22YuSerVkzjdHE64z\\\\\\\/jbBv5qclsaBwhlE+v7mww20g9rFKij4OORZfO+c7uGrVBpdFQVlAfWcs7qIrVR0WmGPm6G9CrXMhCpzgPdea5nMki6Xl1EETT51vJygdUu\\\\\\\/FqJQDtQdK2z5B7qm6Iyya0R8FKW2jsv+agEb96RK1RLecBRT8cCbnQTzcjBn4cGd6P+s5oBc72ixHcHFiYGuNd48yLcBkRdjUr14xr3LnLPetEFLiomdi39RUhkKGFpB\\\\\\\/PwH5WKBkghD10zRjiATmb1FZCWIRyJiu2emvW2ninvOyz+zF3AJkiFh3N5dzWOcYOwKT17C8BCfgJw3dzM\\\\\\\/JGcHnvSbl53imIGQbRe6Z3KsCGIK2mTMHZ9WaL0XSvkR8WtjX12to\\\\\\\/G7RAgpjRl4k56MBsxRLlCWBxpbXsEVbWhCq5aytbhxh9shyIhsreZab9ep8lI41Ww7p6hN9BQ\\\\\\\/3OpVDQg6WVpkZTjST7Zzurx0UnlyOTWIwG+GH\\\\\\\/WIIHYV1tBPtQFuoyWN69rwQYR\\\\\\\/+opI05R8nyI5W5HibiONX3yJ\\\\\\\/UtCCeB6y\\\\\\\/msZxrdW8CYSp5PQAd7CXleWEMo1HkqzxA\\\\\\\/w5yrahfgTLxsnYHBme04uNuRnAuRFdyaOC9eU6dHzr+95nYUiHkjt2Io7euJiMvoV61rqJW9EZ7DNTjO9RuYo01fmvFAnXNvLxBG+Yp9ChK1Gw9LjUeX7w\\\\\\\/92gh1AULnanmyLzK8vyLVRJz52qXyIhA8EDzW5Bj5+XtAqmnWipAJKWvBEcNdRa2GZUipTKJls7ka3KmZoG6QEX7ddo0SKUjij+CdifNq+ox9FZTU1YL5O0f7Sp13LB7zadDqnwCBbbJtUKIBj6yE4HsaCw8VeC6je45XgamXIlHTcCKS8UvPs134bpe0Y+xcz\\\\\\\/FvjT+KYodqCa+b48HDmEYV6mJabA6aGBU2qsHTCJDAIlCvOiD5A68gRoBnjxaMSj9kofSgg6zOu0tegUdUUHpPdf4Iyp+g+I20pzRWn2cupeMhLtQG\\\\\\\/XFmh5xc\\\\\\\/APCfJMNo0Q3oZQpjh2VdDRMUBfcHzFDHLGnih1x\\\\\\\/1CF7TD9WtdsWJR5yjR1HcGWhR47rkadIlKzf\\\\\\\/M15CLfg5zzVcgp3Bkk6ADlN9GGs+eJoEA3TRbiHlCEgZ1vGFInO2\\\\\\\/UasIalLD2LGTAu6YJzvZqO16WxzZGJru6nJjtPE3GYN491RyoGJOscktE8ib9E9TBTl\\\\\\\/Lj\\\\\\\/xTLGFMRcLLIkBY1COMnfaMkKxQI7lJ72r42uGd6G\\\\\\\/vvQBLh0KcdDYxOjhiiypbEZX5bH2lf9ZcgHM825n8hSIaMcvhfB4M++pWJI0ZMZBrsv5mIRIOQbCROlpMVCgCpaJIndUYK5HRCSJaorExMmnI0WJoVWhGW4rrCyzMX7Xdj7FMNuaA+ZyejA6ouXae2mtKBFQ+DL6ofh6jLzUMcEFEsMRpRsA9TkMYthft3tvXoWfPPFwjW0FF+YBg9fMScbTLkevVhga9v5uRPB3VmNeWHfDPmIqy4XCAtwDiLn91ARou+7x4ia0fcikhBo8SAJC70h8pK4qVhCHuePg5SPmOhvDEZp9D0BC9gcu7vpjdYf6JO7b7t0yUfe7bqy0iVBqVXbknL2ui8oIQzrA3eH5J7LO7GDuyyUEhSjIY3M2IAU7\\\\\\\/nLLzG9eO58FEa5hoDNhHFTei\\\\\\\/X\\\\\\\/7sh4mA6bRKBULs\\\\\\\/juGn2uGIpJvd43TcFuXldFcAp5QdPIuoaKGgMy3WTSRI4A1MYDYsr0F\\\\\\\/U2sb8OhYTSQXb6Tkw6muGDysIvQVVD0s3wsVWaJ+sQ7QOiiYmsf3eGOOe4glIL+roLWiqFzB7NEWFXSLspz6oDzSj6nuTG77UczSpUQp03wqkt5xrexYobf0aTUDps9CE0RsT0FUV9EBwPK4COwS3\\\\\\\/+jZrcsWzIQuVKx80\\\\\\\/O9Khb\\\\\\\/3Jn9\\\\\\\/5Ed3bp2rnD8xVJub4dN0mZRSIkIqte5Yfgul7lb4h3Ut7X576mc5GRcdbTLtDJGKBT6OmLxu8MJiHjSky8VM\\\\\\\/+NpAX4cluW3Q94Y+M9kE6hsP5Pj9dwE0hlDW7knGQSUhxYUg4J5jWLLzro9LZgm5VrdT0E\\\\\\\/K3J8hcBpSml1Rl9w7fIq2k33ax7pt2wkXDFZyEBn6B0Rret\\\\\\\/3lT0bjXKLCrKvhyV5ncCLTwEKWNJqrISOrLJ3P6mcTYJfegNwVl1ilU2dkPj13Fq7GL+6WsTBjSsTN2Yqz2aiXI2prCAncYnNjMOOIWrxgCWpp\\\\\\\/e4k3rZppy19Xy4aGYM770DJMcsP4+TEdD2Dc32NkivW9fFp5bdA4h44Swf4MkNwRdmIFSJc0rZkdgCwMBQC1ZaYf1OIF07V3ynus8Oc+3a5UET1njB8Wk\\\\\\\/aUkYvVME6VAf7sQTPfwZEWKUGqjr46Si+i2DlRXsZlq3QI7l+Q3D49fKcX\\\\\\\/nxesfZlSXXanxLXKVcrky+9IGKzP9pqv5jokv\\\\\\\/2I2CzifLIsGPJ3kxS040fW2bfdcPfHdCjLgJmkyK4AofjX1BV5CTwf6oKnbQDpSmRzgdwj3+oFwxWC3qGbIjwE9PKjjPnCp9b6EqoKTVLnGbePj\\\\\\\/cLeIAzinxmJR\\\\\\\/XgdoOrSiOgReJFemhbthYjrjqFgWuUoBHNa8JP4w+R45Ocjvzx\\\\\\\/IAewQpRIANU6VzIxYjqKqLdOrlNbWd6qht0CjwwQrMAKcpaOtnDV6tSSfjl2ccE4eckmbVfJdFkhOJcAuXIaPnGETVP++Ls349Ew\\\\\\\/sYGraURSZDy4oOh51ABCUmZorvVIb7WU5oFm54\\\\\\\/CIBOvmlX7ZfM+Tp9\\\\\\\/siui8bD9T01H\\\\\\\/DLvpEIVOKj2LVGnDMAxd2DOX0nottF8DW7gqjBXYKooVGLP+DYkubjb0C+cgwfkFA92XSz4kofryXmuWueBjGLBak3FqaHbpp+FfVT4JAH8Z2uCO08DO6SiwEUwHA7CJ9M9+H5rzufAX+7qUaQ2znDnPq0VnRrDResWmoUchRf1x5pkQ8nVjf028krBSFswFH5S3J2FiRWQEIiFFIurbEsQsy8XnDfEvHX5YKGnJd6dwaywG3swmTKNLslqUFiujleot+IZXNli6Jp7K0+kmm43OkLO+9W3+Ju65ls6ni0RG4Bvvhe8L73HthFinB8vv2soEzw7SaS0Tmukjhw18uAxgof3KZGjjrQOHfdS\\\\\\\/lYtTUd4DKYoT3mcc7PlYgh7loKTUJ7fhDLFswPT\\\\\\\/gs2sVvmlZ4+nWQdiVWK3052Rz+I6cQAP35fBDnvo\\\\\\\/9YIeJv+NZZqNBDTzoGktry2ffVUvnUOWI7hYHFhthAGRXGOxRlo3rd8xYJD7kxqzWj0v9JdQ8BxxwmnHSdI+8xzk3ao1OTHJitXUM3U7XjxnU5LeMEpiXyTv5FpB7oypEdDq7tm0LidT2NggVnFCSFPkafHhrtJ+g7x\\\\\\\/2HVg8WhKxEZuuitG2K8QSivxTBlkEkNjIGcKAxNrYVCiHLi0\\\\\\\/8hgQBa8RGtduN0iMmmmGB1KDrIO3bEN+3mXGyXrMbXmTtpV9rB4jPERFpawR7mliQeEwyU7F+laCmf5G+CezbwdA83pHJgkOBu0w9uxZC6NlV9zhRlXTQ6CfKaUeVjUePEQFNw5vL+dFPxI0Hlvs1bpyQ8QgSCk1dJyt6a30L5\\\\\\\/D\\\\\\\/2xxYcIUvMIrKqtcX3vHS5tuqRwgB5iOipkQywsZr4X1ZSKw530htkrGZSt6kKEyAbkw8jjDNw7w90\\\\\\\/mm1ogiH3R9qFAbDjBJG5DR20gv0v3P4xwokroHSXBZr1U21UPcKAr4mfo8LF0hC3WiKP0a4ePElyPeP9921r1DqRdYgA6y1uEBfNXgbt4qifUisOflVfBMXLmzY9+8VOGpZG\\\\\\\/HxO16OskZquOlyKExJyaM9p+THpgaTKnZyZ9ZOif8c2WUcpj9X8hjhZdQv4tXWWuHMXkzBflh1GTJoYQMX+xp3X844eTge3zfCz4ngHZBDTXGceh72RKmQrF4nhAosmXWy3lSInsNmLbjZJn3WxJ0vx6rDWnx31bx4hwXk6k6HIBiYWj5EwQZsiE4PPinj+FU76BGwaljHbMa\\\\\\\/6c56uLaSDHJq\\\\\\\/YW\\\u0022,\\\u0022iv\\\u0022:\\\u0022WmZPmzjFtqUOCYnRNNr5jQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022efee5d279e7a1ecd06c4d5bd01ebeee73375867e62b98aac657f1e2574823326\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229EIC6XtUAr4zYJCtpmvFTt\\\\\\\/mF8xXoNJyLjDuOzrs\\\\\\\/DmpMeZq4YFZB356eHF31KRXmHSKBibRRYr45Ej9\\\\\\\/2Wo5GCAGokbF+0F+ryjq\\\\\\\/9pV7KLTZ30vCQqCXr+4X2tLMHArJZ2VHoSU+th3fkwRxlCnKTgCcqzHPudNZKzPsYVlMy3fHsJucQm0tNijaBdMxxmgumYwgxqJiRqLBtYrvchGRRL4HE8iSIyNYUWhtwJALtbDLpg9fZ0Vjwz711ejkryMv3iX5q4Kt4pV2ocUFzqXbJ8Yl6vl+5Bt1CoZTTuhDAajI1JwJVtRhHbRgLjA1aWoW0+lqpf+nu2U9ArFxRdA9A6OncAx22MuaW2FGVswb22EXM6zfi5bxe9M87uv\\\\\\\/hoJP\\\\\\\/fbw70xa9im2jKPv\\\\\\\/g9pYyuB7RDGJxsqc++rHPVTE+4vvbQ40NuI3qUUZBa2TRAvtbH+zdDnHab\\\\\\\/4MM+uoghRXn8jCN\\\\\\\/k+8jafX3pgpfDJOyutY7fnEeAx3ErB76cWPcXsrNP\\\\\\\/gnbVsYqHyiRwUHxZZmR\\\\\\\/QFDWfthGVrwQ\\\\\\\/PaYUgnf6K5GweN+MQuGq3343GifgrpfgbUJtVzv5rWC5LmAvdy8vVjtPD\\\\\\\/dTEmj5xqBpqnUByO3A\\\\\\\/OkNesPMOBTwc3ab1QXH6It8n2YTjVIl5KCA41wsNJRo4JTBsUCZP1pvTa4RBjJi2mwDvOkfdSGqD2vopnBaU0GqqA\\\\\\\/8A5P3tOrT6EIntmfktn72u3LC78ljl9UUjNfd1PGuATluaddN8biI0DhKPEJrh4EnSCl+g12Hqckf2nQk+a4q4X+Jv1Lor2vc9PMALjfx+kYVDTet+paXZY9V0Wt1ypZZIJ5fZVSVreqEPlZ2WkR5EeNVVqcc3tS3ZwOzVT7aCIG4XmTNmIQibxdZLckCGhE8NTTTh3QJT5DF7vx6Y26hMTnBRBG5YW3Df0kKzoQbxgJtIPVixg9GuCqrmHdToTkLeMPjpvR3SofNOW\\\\\\\/I\\\\\\\/51tpamt+cPh9vFLjl5yn2bJiWmp0DHW\\\\\\\/EArDbi0Fo3e+paf8IHowr5m8eHg9WTbBfnxmjzLPT6qLSih1qkdGy\\\\\\\/tb7mKRJ8ygR0ZBaFTU4nZsuRd4zQaAe9Os8RlwNcnkwWp6IIam\\\\\\\/c2tLVgLCz1RbECONLtXACapdgjv43XS3A9wMbKd237UsNxi1zP4O4vTZiUhuQQbC4pJMajNi5VI2yxauhYCI8mx6H8k\\\\\\\/mbY5rVERX0IG3wcGGZHtq1dU4UWFCexlipDzbbCa888yzv5FDbjZJZ0q2ludYcn2ZaV9CI41ZemJbr1lcyuMRkx3d4BflQbHuvxtpmGbdCIvo1DFaS2XLxlI3RBu2Vkxq8HfZLsem8Cw49XR3mXHlhWnoMPTalPz4aTIehVbCcXQX2amlAtNwROAmhDO29wFvieaLdpcEc+69I++dhSeEVPjJIN0r3sKxtX1ZVhgrlFseCD4RsQyjPJP6DBxt3mmpTBfklxk+v6ZnbEoi0FxyXaSdAdwFVVVOGd0T+zQyildO7W1axwNkOCL2WEdqHNkWl8cHffhlTJ\\\\\\\/Quv9d59zjOMdkHWtVkZwR1wuif1nkxv0v7HJzD9y+C5KcT5s13aDgINjz01T4NQR35\\\\\\\/RVhLaSnQEhKDtQIAStiVc4cuJ9fLsRkSqFhACeL+rXZ\\\\\\\/p3RemsYyJ4BU4LIo9uuZzhF5Fe2fqTWyMaqfKiV1OS19PbZUGJAkVXeP+mY80+\\\\\\\/\\\\\\\/0rPCSVkV2UXPhGiH+5EiAGgr9reSUj43mq4iLEoiIawc2hX\\\\\\\/HLXvWBW3H3ZZ\\\\\\\/8ZUnkygdfpigkVaULFXbEwQtr3FFOs8WPpuh3OlR\\\\\\\/DyGBjKZ04fJpVxN\\\\\\\/6JdE5DmVGXL1ygtPLqoDfmwptSCKoBF6uClkLyB5jUHVrrBrhshplNHT37WRErtP7Sra9KncwCHyapFalH9dGk+6ziVDFZ29uqDERwEwc0ZEeXiA2nOToNikUJzXoo+XZP12q2J5HEB2CcKrvtUhvT\\\\\\\/lesLF9Au4zvwI9OIEKDOWBNX+5Dg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022LIp4B9SfS4CcWWOBfVAWUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002256bfeb483cfb300ca7ae67d27a9c2986d70f53b9738ef0d2aa93d7ba0e608c8c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459723 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2065" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VWMzlH_jcEUNDUMEz8Eigik3UX6qM87yJk34i2Q4g0o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2065" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8Tw7wwwqFIAR8SMDEUfB3js5up4jH-gYKP2n_WZBvJc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2051" title="Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2079" title="The Genetic Architecture of Coordinately Evolving Male Wing Pigmentation and Courtship Behavior in Drosophila elegans and Drosophila gunungcola" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459723" id="node459723" data-pisa="ggg;4/11/2065" data-pisa-master="ggg;g3.114.012526" data-apath="/ggg/4/11/2065.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Polymorphism Identification and Improved Genome Annotation of <em>Brassica rapa</em> Through Deep RNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Upendra Kumar Devisetty</span>, <span class="highwire-citation-author" data-delta="1">Michael F. Covington</span>, <span class="highwire-citation-author" data-delta="2">An V. Tat</span>, <span class="highwire-citation-author" data-delta="3">Saradadevi Lekkala</span> and <span class="highwire-citation-author" data-delta="4">Julin N. Maloof</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2065-2078;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012526 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459723" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Upendra Kumar Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BKumar%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUpendra%2BKumar%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">An V. Tat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=An%2BV.%2BTat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tat%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAn%2BV.%2BTat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Saradadevi Lekkala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saradadevi%2BLekkala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lekkala%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaradadevi%2BLekkala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/11/2065" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459723" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459723/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2065.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459723" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459723/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2065.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459723" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459723/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2065.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459723" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459723/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/11/2065.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459723" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459723/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-72690 first last"><a href="/content/7/7/2405" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - July 01, 2017</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The mapping and functional analysis of quantitative traits in <em>Brassica rapa</em> can be greatly improved with the availability of physically positioned, gene-based genetic markers and accurate genome annotation. In this study, deep transcriptome RNA sequencing (RNA-Seq) of <em>Brassica rapa</em> was undertaken with two objectives: SNP detection and improved transcriptome annotation. We performed SNP detection on two varieties that are parents of a mapping population to aid in development of a marker system for this population and subsequent development of high-resolution genetic map. An improved <em>Brassica rapa</em> transcriptome was constructed to detect novel transcripts and to improve the current genome annotation. This is useful for accurate mRNA abundance and detection of expression QTL (eQTLs) in mapping populations. Deep RNA-Seq of two <em>Brassica rapa</em> genotypes—R500 (var. trilocularis, Yellow Sarson) and IMB211 (a rapid cycling variety)—using eight different tissues (root, internode, leaf, petiole, apical meristem, floral meristem, silique, and seedling) grown across three different environments (growth chamber, greenhouse and field) and under two different treatments (simulated sun and simulated shade) generated 2.3 billion high-quality Illumina reads. A total of 330,995 SNPs were identified in transcribed regions between the two genotypes with an average frequency of one SNP in every 200 bases. The deep RNA-Seq reassembled <em>Brassica rapa</em> transcriptome identified 44,239 protein-coding genes. Compared with current gene models of <em>B. rapa</em>, we detected 3537 novel transcripts, 23,754 gene models had structural modifications, and 3655 annotated proteins changed. Gaps in the current genome assembly of <em>B. rapa</em> are highlighted by our identification of 780 unmapped transcripts. All the SNPs, annotations, and predicted transcripts can be viewed at <a href="http://phytonetworks.ucdavis.edu/">http://phytonetworks.ucdavis.edu/</a>.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/brassica-rapa" class="hw-term hw-article-keyword hw-article-keyword-brassica-rapa" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Brassica rapa</em></a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-Seq</a></li><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/snps" class="hw-term hw-article-keyword hw-article-keyword-snps" rel="nofollow">SNPs</a></li><li class="kwd"><a href="/keyword/genome-annotation" class="hw-term hw-article-keyword hw-article-keyword-genome-annotation" rel="nofollow">genome annotation</a></li></ul><p id="p-4">High-density molecular genetic markers are a valuable resource for assessing genetic diversity both within and between species. They are useful for the construction of high-resolution genetic maps, for genotyping segregating populations, for marker-assisted breeding, and for map-based cloning (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Edwards and Batley 2004</a>). Next-generation sequencing (NGS) has significantly improved discovery and genotyping of such markers, especially single nucleotide polymorphisms (SNPs) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Shendure and Ji 2008</a>). NGS SNP discovery can be focused on coding regions using RNA-Seq, reducing costs compared with whole-genome sequencing (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Trick <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lai <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Koenig <em>et al.</em> 2013</a>). For the current study, we are particularly interested in detecting SNPs between two <em>B. rapa</em> varieties (hereafter referred to as genotypes), R500 (Yellow Sarson variety), and IMB211 (a rapid cycling variety) because these are the parents of a mapping population. The R500 genotype is a seed-oil cultivar grown in India for more than 3000 years (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Prakash 1980</a>). The IMB211 genotype is derived from Wisconsin Fast Plant population and therefore has a rapid generation time (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dorn and Mitchell-Olds 1991</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mitchell-Olds 1996</a>). These genotypes have contrasting life-history strategies; compared with IMB211, R500 flowers late, reaches a large size at flowering, and accumulates more biomass (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Edwards <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edwards and Weinig 2011</a>). Previously, a <em>B. rapa</em> genetic linkage map was constructed using RFLP markers for this population (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Iniguez-Luy <em>et al.</em> 2009</a>). However, this linkage map is of low density and may not provide precise and complete information about the numbers and location of QTL. SNP based markers located in coding regions (cSNPs), such as those defined by RNA-Seq, can be useful for the development of high-resolution genetic map of <em>B. rapa</em>.</p><p id="p-5">In addition to physically positioned gene-based molecular markers, accurate and comprehensive genome annotation (<em>e.g.</em>, gene models) is imperative for functional studies. Genome annotations initially relied primarily on <em>ab initio</em> gene predictions and alignment of reference transcripts of related species [GENSCAN (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burge and Karlin 1997</a>), GlimmerHMM (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Majoros <em>et al.</em> 2004</a>), and Fgenesh (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Salamov and Solovyev 2000</a>)]. However, these strategies often have problems identifying short exons and predicting very long exons, identifying nontranslated exons, and predicting genes that encode noncoding RNAs accurately. Genome annotation based on gene prediction programs that rely on ESTs also have problems. For instance, it has been estimated that most EST+homology–based annotations miss 20% to 40% of transcripts that are novel transcripts or that are transcribed only under highly specific tissue or environmental or treatment conditions (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brent 2008</a>). Currently, the annotation of protein-coding genes in the <em>Brassica rapa</em> (Chiifu) genome mainly relied on ESTs data from databases and <em>in silico</em> gene models (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2011</a>). Updating and improving the current genome annotation of <em>B. rapa</em> is essential for better functional analysis studies and more accurate mRNA abundance estimates.</p><p id="p-6">In this study, we describe our approaches for the generation of coding SNPs (cSNPs) and re-annotation of <em>B. rapa</em> genome using deep RNA-Seq. Using these approaches, we were able to identify 330,995 putative cSNPs between the two <em>B. rapa</em> genotypes. In addition, we were able to detect 3537 previously uncharacterized <em>B. rapa</em> transcripts and updated 23,754 gene models of <em>B. rapa</em> genome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and sample collection</h3><p id="p-7">Two <em>Brassica rapa</em> genotypes—R500 and IMB211—were grown in three different locations for tissue collection for RNA-Seq library preparation. For sample collections from the growth chamber, both genotypes were germinated on 1/2 MS (Sigma) plates kept in dark for 3 d with 10 to 15 seeds per plate. Then, the plates were exposed to continuous light and grown at 22° in a Conviron walk-in growth chamber. The 10-d-old seedlings were transplanted to 3-inch pots and were divided into two groups. One group (simulated sun group) continued to grow under the conditions as described above [high-red to far-red ratio (R/FR), 2.0], and the second group (simulated shade group) was grown under a mixture of cool white and far-red fluorescent lights (low R/FR, 0.2) in a complete randomized design. Both conditions had PAR ranging from 80 to 100 µE. Ambient light conditions were measured using <em>LI-COR</em> radiospectrophotemeter (Li-Cor). The following samples were collected from the growth chamber–grown sun and shade plants. Whole seedlings were collected 10 d after germination. Roots were collected from 10-d-old seedlings. Vegetative meristems were collected from plants when the third leaf reached 1 mm. The stem between the fourth and fifth leaves was collected for internode tissue, and inflorescence meristems were collected when these meristems were fully formed. Leaf samples were collected 28 d after germination, and siliques were collected when they were fully mature. For greenhouse sample collection, both genotypes were germinated and grown as indicated above, and 10 d after germination the seedlings were transferred to a greenhouse, transplanted to soil, and subjected to nondense and dense treatments. For the dense treatment, four plants surrounded each plant in a 6-inch square pot; for the nondense treatment, a single plant was grown per 6-inch pot. Internode, leaf, and silique samples were collected as indicated above. Petioles were collected from first, second, and third mature leaves. Only leaf and silique samples were collected from field-grown <em>B. rapa</em> genotypes. All the collected samples were immediately frozen in liquid nitrogen before storage at −80° until RNA extraction and library preparation. Samples used in this study are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465242/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465242/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of tissue samples collected from <em>B. rapa</em> genotypes R500 and IMB211 across growth chamber, greenhouse, and field conditions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Total RNA extraction, RNA-Seq library preparation, and sequencing</h3><p id="p-9">Total RNA was extracted from two to three biological replicates of different samples of <em>B. rapa</em> genotypes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and purified using RNAeasy Plant Mini Kit (Qiagen). DNaseI (Qiagen) was used to remove any contaminating DNA according to the manufacturer’s instructions. The quality and quantity of the extracted RNA were initially assessed by NanoDrop ND 1000 (NanoDrop technologies). RNA-Seq libraries were prepared using Illumina’s TruSeq v1 RNA sample Preparation kit (RS-930-2002) with a Low-Throughput protocol following manufacturer’s instructions with the following modifications. All the reaction volumes were reduced to half to reduce costs. Custom paired-end barcoded adapters (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kumar <em>et al.</em> 2012</a>) were used instead of Illumina’s RNA indexed adapters to multiplex the samples, 10 cycles of PCR enrichment was performed instead of 15 cycles to reduce amplification bias, and, finally, the libraries were constructed with an insert size of 300 to 350bp. The enriched libraries were then quantified on an Analyst Plate Reader (LJL Biosystems) using SYBR Green I reagent (Invitrogen). Once the concentration of libraries was determined, two pools were made to a final concentration of 20 nM, with one pool consisting of 66 samples collected from growth chamber and another pool consisting of 60 samples collected from greenhouse and field (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Each pool was sequenced on eight lanes (total 16 lanes) of an Illumina Genome Analyzer (GAIIx) as 100-bp paired end reads. The libraries that failed from both the pools were further sequenced on one extra lane (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465203/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465203/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of RNA-Seq data obtained from <em>B. rapa</em> deep transcriptome using Illumina GAIIx sequencing</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Pre-processing of Illumina RNA-Seq raw reads</h3><p id="p-11">The FastX-tool kit software (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>) and custom perl scripts were used to perform pre-processing of Illumina raw reads to ensure the good quality of sequencing reads for downstream analysis. The raw reads were either quality-filtered with fastq_quality_filter with parameters [q 20, p 85] or, in some cases, trimmed first using fastx_trimmer with parameters [f 1,3; l 78] and then quality-filtered with the same parameters as indicated above. Next, reads containing custom adapters were removed using a custom script. The reads were then sorted (de-multiplexed) by their custom barcode sequences using fastx_barcode_splitter with default parameters. The quality control of paired end reads resulted in some reads losing their partner reads. Further processing to extract properly paired reads and unpaired reads (orphan reads) was performed using a custom script. Only paired reads were kept for downstream analyses. Reads were checked for quality before and after quality control with FastQC quality assessment software (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>).</p></div><div id="sec-5" class="subsection"><h3>SNP detection</h3><p id="p-12">SNPs were obtained from the alignments of all available reads of each genotype separately (NCBI SRA accessions are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012526/-/DC1/012526SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS3.pdf">Table S3</a>) to <em>B. rapa</em> genome reference v1.2 data of Chiifu available from BRAD (<a href="http://brassicadb.org/brad/">http://brassicadb.org/brad/</a>) using BWA v0.6.1-r104 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li and Durbin 2009</a>) with parameters [k 1, l 25, n 0.02, e 15, i 10]. All unmapped reads from BWA were mapped to putative splice junctions using TopHat v1.4.1 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Trapnell <em>et al.</em> 2009</a>) with the following parameters [–segment-mismatches 1–max-multihits 1–segment-length 22–butterfly-search–max-intron-length 5000]. The resulting alignment files from Tophat and BWA were merged, and SAMtools v0.1.18 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2009</a>) followed by Picard <strong>(</strong><a href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a><strong>)</strong> were used to filter uniquely mapped reads and remove duplicated reads, respectively. Sequence polymorphisms between R500 and IMB211 were identified using a variant-detection, genotype-scoring, and visualization tool that we developed, SNPtools v0.1.5 (<a href="https://github.com/mfcovington/SNPtools/">https://github.com/mfcovington/SNPtools/</a>). R500 compared with IMB211 variants were deduced by comparing the lists of Chiifu-related variants. For example, an R500:Chiifu SNP would also be an R500:IMB211 SNP if, at the position in question, the IMB211 alignment to Chiifu has sufficient coverage and IMB211 matches the Chiifu reference. To refine the R500:IMB211 polymorphism list, we used the SNPtools noise-reduction feature. The initial set of R500:IMB211 polymorphisms was used by SNPtools to interrogate the R500 and IMB211 alignments at the position of each putative polymorphism. For each of these positions, the resulting genotype files indicate the number of reads matching each allele. For a true R500:IMB211 polymorphism, the majority of reads should match the appropriate allele for both alignments. Putative polymorphisms that went against this expectation were filtered out from the final list. The code that was used to perform each of these steps is available at <a href="https://github.com/MaloofLab/devisetty-g3-2014/">https://github.com/MaloofLab/devisetty-g3-2014/</a>.</p></div><div id="sec-6" class="subsection"><h3>SNP annotation</h3><p id="p-13">The categorization of SNP effects was performed using SnpEff v3.0 (Cingolani <em>et al.</em> 2012). First, <em>B. rapa</em> genome annotation information in GFF3 format was retrieved from BRAD (<em>Brapa_gene</em>_v1.2.gff). This genome annotation provided predicted exon–intron gene structure. The default parameters of snpEff were used to perform the variant effect analysis. Both HTML and text output files were generated from snpEff and were used to perform the SNP annotation on the basis of their structural occurrence in the intergenic, intronic, and exonic regions. SNPs located in the exonic region were further categorized as CDS, 5′-region, and 3′-region. Depending on whether SNPs caused changes in the coding of an amino acid, SNPs in the CDS region of the protein-coding genes were annotated according to functional relevance as synonymous or nonsynonymous mutations.</p></div><div id="sec-7" class="subsection"><h3>Experimental validation of SNPs</h3><p id="p-14">For validation of identified SNPs, 300-base-long sequence fragments (100 of them) containing a predicted SNP were excised from <em>B. rapa</em> genome using a custom script, and the Primer3Plus primer design tool (<a href="http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi/">http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi/</a>) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Untergasser <em>et al.</em> 2007</a>) was used to design primers. The designed primers are listed in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS1.pdf">Table S1</a>. For verification of SNPs, genomic DNA was extracted from 7-d-old seedlings of the two <em>B. rapa</em> genotypes using a modified Dellaporta plant DNA extraction method (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dellaporta <em>et al.</em> 1983</a>). Genomic DNA was diluted to 20 ng/μl and used as a template in a PCR final volume of 20 μl containing 1× Standard buffer (homemade), 200 μM of each dNTP (Promega), 0.25 μM of each primer, 0.1 μl of Taq DNA polymerase (homemade), and 2 μl of template DNA (20 ng/ul). The PCR amplification conditions include a 2-min denaturation step at 94° followed by 35 cycles of PCR (94° for 30 sec; 60° for 30 sec; 72° for 30 sec), with a final extension time of 7 min at 72°. The PCR products were purified using Axygen AxyPrep Mag PCR clean-up kit (MAG-PCR-CL-250) according to manufacturer’s protocol. The purified PCR products were then sequenced by Sanger DNA sequencing method with an ABI 3730 Capillary Electrophoresis Genetic Analyzer at UC Davis. Each of the sequences from both genotypes along with the Chiifu reference genome sequences were aligned and visually inspected by ebioX software (<a href="http://www.ebioinformatics.org/">http://www.ebioinformatics.org/</a>) to verify sequence polymorphisms. In addition to the estimation of true-positive and false-positive SNP rates, we also used these fragments to estimate the false-negative rate. To do this we identified all SNPs present in the Sanger reads in a 150-bp window surrounding the focal SNP (we only used 150 bp out of 300 bp because this region had the most consistently good sequence). For each SNP found in the Sanger reads, we then asked if there was coverage of this position in the RNA-Seq data, and if there was coverage whether we had detected the SNP. To further investigate apparent false-positive SNPs, we chose primer sets to amplify four apparent false-positive SNPs from recombinant inbred lines (RILs) predicted to be segregating for the SNPs in question. RNA-Seq libraries and genomic DNA were as used as template for RILs and parental genotypes, respectively. The amplification conditions and sequencing procedure are the same as above.</p></div><div id="sec-8" class="subsection"><h3><em>K</em>a/<em>K</em>s ratio computation</h3><p id="p-15">For a given transcribed region of the genome, <em>K</em>a denotes the average number of nonsynonymous substitutions per nonsynonymous site. Likewise, <em>K</em>s denotes the average number of synonymous substitutions per synonymous site. To estimate <em>K</em>a/<em>K</em>s ratio for the <em>B. rapa</em> genome, all the SNPs were first substituted into the Chiifu reference genome sequence using vcf2diploid v0.2.6 tool of AlleleSeq pipeline (<a href="http://info.gersteinlab.org/AlleleSeq">http://info.gersteinlab.org/AlleleSeq</a>). This generated separate genomes for R500 and IMB211. The corresponding CDS were then extracted using the gffread tool from Cufflinks package. The <em>K</em>a/<em>K</em>s ratio was computed using KaKs calculator (<a href="https://code.google.com/p/kaks-calculator/downloads/list">https://code.google.com/p/kaks-calculator/downloads/list</a>). Tests were conducted to estimate the evolution of each codon using the “MLWL” method of codon substitution. To look for differences in <em>KaKs</em> ratios among chromosomes, we calculated the difference in median <em>KaKs</em> for each chromosome compared with the rest of the genome. We repeated this for 1000 permuted chromosomes. A chromosome was considered to have a significantly different <em>KaKs</em> ratio if the absolute value of the difference in its <em>KaKs</em> ratio (compared with the rest of the genome) exceeded the absolute value of the differences in 95% of the permuted datasets. In the first version of this test (to asses overall differences in <em>KaKs</em>), genes were randomly assigned to the permuted chromosomes (keeping the number of genes the same as in the original data set). To test for the contribution of differential sub-genome composition per chromosome, we assigned genes to chromosomes in a way that preserved the proportion of LF, MF1, and MF2 genes on each chromosome.</p></div><div id="sec-9" class="subsection"><h3><em>De novo</em> transcriptome assembly methods</h3><p id="p-16">Post-processed paired end reads of <em>Brassica rapa</em> R500 genotype from 10 Illumina GAIIx lanes (NCBI SRA accessions SRR1227842, SRR1228204, SRR1228205, SRR1228206, SRR1228207, SRR1228208, SRR1228209, SRR1228210, SRR1228211, SRR1228212) (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS3.pdf">Table S3</a>) were pooled together and assembled <em>de novo</em> using Velvet v1.2.07 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zerbino and Birney 2008</a>) followed by Oases v0.2.08 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schulz <em>et al.</em> 2012</a>) on XSEDE Lonestar computer cluster (<a href="https://www.xsede.org/tacc-lonestar">https://www.xsede.org/tacc-lonestar</a>) and Trinity r2012-06-08 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garber <em>et al.</em> 2011</a>) on XSEDE Blacklight computer clusters (<a href="http://www.psc.edu/index.php/computing-resources/blacklight">http://www.psc.edu/index.php/computing-resources/blacklight</a>). Because of the highly variable nature of transcriptome coverage (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Surget-Groba and Montoya-Burgos 2010</a>), we chose seven different k-mer sizes (31, 35, 41, 45, 51, 55, and 61) to increase the chances of transcript assembly using Velvet-Oases. In addition, we also tested two merged assemblies with k-mer sizes 27 and 55 using Oases merge. We analyzed various assembly parameters such as total number of transcripts, N50 (N50 is a weighted median statistic indicating that 50% of the entire assembly is contained in contigs equal to or larger than this value), longest transcript length, and average transcript length as a function of k-mer length. To produce a nonredundant set of Oases mRNA sequences rather than include alternative splicing, we used a custom python script (<a href="https://code.google.com/p/oases-to-csv/downloads/list">https://code.google.com/p/oases-to-csv/downloads/list</a>) to choose a single representative transcript based on coverage and sequence length. Next, to assess the quality and completeness of the assembly, we then compared the current Velvet-Oases assembly with <em>B. rapa</em> genome (Brapa_sequence_v1.2.fa), <em>B. rapa</em> CDS (<em>Brassica_rapa</em>_v1.2.cds), and NCBI nr database. Finally, the blast output was fed into custom Perl script to remove any transcripts that hit multiple chromosomes at different locations (chimeras) to generate novel Velvet-Oases transcripts. Trinity was run on the paired-end reads with one k-mer size of 25 using the following parameters: minimum contig length = 200; paired fragment length = 500; CPUs = 16; and a butterfly HeapSpace = 10G. For each of the assemblies, first the splicing transcripts were removed by blasting them to the publically available plant refseq database (<a href="ftp://ftp.ncbi.nlm.nih.gov/refseq/release/plant/plant.1.protein.faa.gz">ftp://ftp.ncbi.nlm.nih.gov/refseq/release/plant/plant.1.protein.faa.gz</a>) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pruitt <em>et al.</em> 2007</a>) and NCBI nr database, followed by chimera removal to generate novel Trinity transcripts.</p></div><div id="sec-10" class="subsection"><h3>Reference-based transcriptome assembly methods</h3><p id="p-17">For reference-based transcriptome assembly, high-quality reads from three Illumina GAIIx lanes (NCBI SRA accessions SRR1227842, SRR1228204 and SRR1238058) (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS3.pdf">Table S3</a>) were first aligned to <em>B. rapa</em> genome data using TopHat v1.4.1 with default parameters. The aligned reads from TopHat were then processed by Cufflinks v2.0.2 to assemble into transcripts. Cufflinks may or may not use existing gene annotation during assembly of transcripts, but rather it constructs a minimum set of transcripts that best describe the reads in the data. The current assembly was performed both with and without the help of reference annotation to capture both novel and native transcripts. We used default parameters for Cufflinks. Once all the transcripts were assembled with Cufflinks, the output GTF file (transcripts.gtf) was then fed to Cuffcompare v2.0.2 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Trapnell <em>et al.</em> 2010</a>) along with the reference GTF annotation file (Brapa_gene_v1.2.gff) downloaded from BRAD. This classified each transcript into different class codes. The classification basically describes the nature of the match to the reference gene annotation (<a href="http://cufflinks.cbcb.umd.edu/manual.html#tmap">http://cufflinks.cbcb.umd.edu/manual.html#tmap</a>). For current assembly purposes, only class code “u” transcripts (unknown intergenic transcripts according to Cufflinks) were considered to generate novel TopHat transcripts using gffread utility in Cufflinks package.</p></div><div id="sec-11" class="subsection"><h3>Annotation of novel transcripts</h3><p id="p-18">To lower the redundancy among novel transcripts resulting from Velvet-Oases, Trinity, and TopHat-Cufflink pipelines, they were first pooled and CAP3 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang and Madan 1999</a>) was used with default parameters. The resulting contigs and singletons from CAP3 were concatenated and used for downstream analysis. For all transcripts, the open reading frames most likely to encode proteins were identified using the transdecoder package (<a href="http://transdecoder.sourceforge.net/">http://transdecoder.sourceforge.net/</a>) with default parameters except for minimum protein length [(m) = 50]. For multiple ORFs from the same transcript, we defined the primary transcript as the one with the maximum number of significant hits to NCBI nr database. To generate annotation, the ORF filtered transcripts were first mapped against <em>B. rapa</em> genome sequence with default parameters using Burrows-Wheeler Aligner (BWASW) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Li and Durbin 2009</a>). To select only those unmapped transcripts that are likely to be real, they were queried against the NCBI nr database using BLASTX with an e-value threshold of 1e−06. Functional annotation of the unmapped transcripts was performed using Blast2GO annotation tool. The mapped transcripts from BWASW output were next converted to a bed file using bamtobed utility of BED Tools (<a href="http://bedtools.readthedocs.org/en/latest/">http://bedtools.readthedocs.org/en/latest/</a>). A custom script was next used to join the exons for each transcript in the bed file. A further filtering step was included to eliminate transcripts that are more than 10 kb because these most likely represent problems with <em>B. rapa</em> assembly. In cases when there were more than one isoform per transcript, the selection of the best isoform was based on maximum length of the transcript. Finally, a fasta file corresponding to the annotation file was generated using the getfasta utility from BED Tools.</p></div><div id="sec-12" class="subsection"><h3>PASA annotation</h3><p id="p-19">The use of PASA (<a href="http://pasa.sourceforge.net/">http://pasa.sourceforge.net/</a>) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Haas <em>et al.</em> 2003</a>) to assemble full-length transcripts based on RNA-Seq data has previously been described (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rhind <em>et al.</em> 2011</a>). For the current analysis, PASA was used to update the existing genome annotation using evidence from <em>de novo</em> RNA-Seq assembly and reference-based RNA-Seq assembly. PASA updates pre-existing protein-coding gene annotations to incorporate the PASA alignment evidence, correcting exon boundaries, adding UTRs, and models for alternative splicing based on the PASA alignment assemblies generated. Default parameters were used and a total of three rounds of PASA annotation were performed. RSEM (<a href="http://trinityrnaseq.sourceforge.net/analysis/abundance_estimation.html">http://trinityrnaseq.sourceforge.net/analysis/abundance_estimation.html</a>) was used to further filter out lowly expressed alternatively spliced transcripts. Finally, a fasta file corresponding to PASA updated annotation was generated using getfasta utility from BED Tools.</p></div><div id="sec-13" class="subsection"><h3>Semi-quantitative RT-PCR validation</h3><p id="p-20">For RT-PCR, total RNA from leaves, internodes, siliques, seedlings, and roots was extracted from <em>B. rapa</em> parental genotype R500 and treated with DNase I. Approximately 1 ug of purified total RNA from each sample was used for first-strand cDNA using oligo-dT<sub>(18)</sub> primer (Sigma) and SuperScriptIII reverse-transcriptase (Invitrogen) according to manufacturer’s instructions. The cDNA was diluted 1 to 50 and equal quantities of first-strand cDNA were used as a template. A total of 70 transcripts (20 for each of Velvet-Oases, Trinity, TopHat-Cufflinks “u” transcripts and 10 for TopHat-Cufflinks “o” transcripts) were randomly selected and primer sets were designed to amplify 300-bp to 400-bp fragment using Primer3Plus tool. For TopHat-Cufflinks “o” transcripts, primers were designed in such a way that one of the primers from each set would anneal to the predicted novel exon. The actin gene (JN120480) was used as internal control. Primers used for RT-PCR validation are given in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS2.pdf">Table S2</a>. RT-PCR amplification of template RNA from different samples was performed using ExTaq kit (TaKaRa) using manufacturer’s protocol. The PCR amplification conditions include a 98° hold for 2 min, followed by a 30 cycles at 98° for 30 sec, 60° for 30 sec, 72° for 30 sec, and a final extension at 72° for 7 min. Semi-quantitative analysis of the RT-PCR amplified products was performed by agarose gel electrophoresis.</p></div><div id="sec-14" class="subsection"><h3><em>In silico</em> RNA-Seq coverage and genome annotation validation</h3><p id="p-21">For <em>in silico</em> RNA-Seq coverage and genome annotation validation, Chiifu public RNA-Seq reads (SRR643621-SRR643628) were first mapped to <em>B. rapa</em> genome reference v1.2 using BWA v0.6.1-r104 with default parameters. All unmapped reads from BWA were next mapped to putative splice junctions using TopHat v1.4.1 with the following parameters (–segment-mismatches 1 –max-multihits 1 –segment-length 22 –butterfly-search–max-intron-length 5000). The resulting alignment files from BWA and Tophat were merged into a single alignment file. Finally, the merged bam files of R500 and Chiifu were loaded onto IGV (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Thorvaldsdottir <em>et al.</em> 2013</a>) along with original and updated genome annotation.</p></div><div id="sec-15" class="subsection"><h3>Gene ontology (GO) annotation and functional classification</h3><p id="p-22">BLAST (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) searches of the gene models were performed against the nonredundant (Nr) protein database at NCBI using a blast cut-off of 1e−03. All BLAST results were saved as XML. Blast2GO v2.5.0 (<a href="http://www.blast2go.org/">http://www.blast2go.org/</a>) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> 2005</a>) was used to assign gene ontology (GO) IDs to the gene models based on the BLASTX output. After blast, GO annotation was performed using an e-value cut-off of 1e−03, an annotation score cut-off of 45, and a GO weight of 5. After obtaining GO annotation for every unigene, WEGO software (<a href="http://wego.genomics.org.cn/cgi-bin/wego/index.pl/">http://wego.genomics.org.cn/cgi-bin/wego/index.pl/</a>) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2006</a>) was then used to simplify the output for producing combined graphs for molecular function, cellular process, and biological process.</p></div><div id="sec-16" class="subsection"><h3>Pathway mapping using KEGG</h3><p id="p-23">To determine gene ortholog assignment and pathway mapping of transcripts, Kyoto Encyclopedia Genes and Genomes (KEGG) mapping was used. The transcripts were initially mapped to KEGG metabolic pathway database by submitting the sequences to the Kyoto Encyclopedia of Genes and Genomes automatic annotation server (KAAS) (<a href="http://www.genome.jp/tools/kaas/">http://www.genome.jp/tools/kaas/</a>) and the single-directional best hit (SBH) method was selected. KAAS annotates every submitted sequence with KEGG ontology (KO) identifiers, which represents an orthology group of genes directly linked to an object in the KEGG pathway and thus incorporates different types of relationships that exist in biological systems.</p></div><div id="sec-17" class="subsection"><h3><em>Brassica rapa</em> UCSC genome browser</h3><p id="p-24">A customized UCSC Genome Browser of <em>Brassica rapa</em> (<a href="http://phytonetworks.ucdavis.edu/">http://phytonetworks.ucdavis.edu/</a>) has been set-up as a community resource that provides an integrated display of annotation data (<em>B. rapa</em> novel transcripts annotation track, <em>B. rapa</em> existing annotation track, <em>B. rapa</em> updated annotation track), data containing our alignment results of <em>B. rapa</em> genotypes R500 and IMB211 with respect to <em>B. rapa</em> genome data v1.2 (R500 <em>vs.</em> IMB211, R500 <em>vs.</em> Chiifu, and IMB211 <em>vs.</em> Chiifu SNP tracks). The VCF files (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS1.gz">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS2.gz">File S2</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS3.gz">File S3</a>) for the SNP tracks were generated using a Perl script (parental-vcf-writer.pl) from <a href="https://github.com/mfcovington/vcf-generator">https://github.com/mfcovington/vcf-generator</a>. This script depends on the V<em>cf</em>.pm Perl module from VCFtools (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Danecek <em>et al.</em> 2011</a>).</p></div><div id="sec-18" class="subsection"><h3>Data deposition</h3><p id="p-25">The reads were submitted to the NCBI sequence read archive (SRA) (<a href="http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi/">http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi/</a>) with accession numbers listed in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS3.pdf">Table S3</a> and all the transcript-derived contigs have been submitted to the NCBI Transcriptome Shotgun Assembly (TSA) (<a href="https://submit.ncbi.nlm.nih.gov/subs/tsa/">https://submit.ncbi.nlm.nih.gov/subs/tsa/</a>). This TSA project has been deposited at DDBJ/EMBL/GenBank under the accession number GBDX00000000. The version described in this article is the first version, GBDX01000000. Because TSA allows submission of fasta sequences only greater than 200 bp, the cDNA of all annotated transcripts of <em>B. rapa</em> were saved in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS4.zip">File S4</a>.</p></div></div><div class="section results-discussion" id="sec-19"><h2 class="">Results and Discussion</h2><div id="sec-20" class="subsection"><h3>Overview of deep RNA-Seq data</h3><p id="p-26">To identify a large number of SNPs and to re-annotate the <em>B. rapa</em> genome, we performed deep RNA sequencing of two <em>Brassica rapa</em> genotypes using several tissues, environmental conditions, and treatments on multiple sequencing lanes (see <em>Materials and Methods</em>). Using Illumina sequencing, we generated 3.35 billion reads with an average read length of 98 bp encompassing approximately 880 GB of data in fastq format. From these 3.35 billion reads, we obtained 2.54 billion quality-filtered and trimmed reads after removal of low-quality reads, adapter reads, and primer reads (see <em>Materials and Methods</em>). The final sequencing results are summarized in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. It is expected that the current data would have enough coverage to detect SNPs in expressed regions between the two genotypes, to detect rare novel transcripts, and to allow re-annotation of the <em>Brassica rapa</em> genome coding genes.</p></div><div id="sec-21" class="subsection"><h3>SNP detection in <em>Brassica rapa</em> using deep RNA-Seq</h3><p id="p-27">For expressed genes, the use of RNA-Seq data for SNP detection can be advantageous because it enriches for expressed regions of the genome. Thus, in addition to providing genotyping SNPs, RNA-Seq–based SNP discovery enriches detection of functionally important SNPs. In the current study, all the reads corresponding to each genotype were pooled separately and used for SNP detection. The final number of reads for <em>B. rapa</em> genotypes (R500 and IMB211) are 1.26 and 1.08 billion reads, respectively. A total of 330,995 putative SNPs (in 18,143 BRAD gene models) were identified from <em>B. rapa</em> transcribed regions. Of these, 639,788 (in 24,966 BRAD gene models) and 595,619 (in 24,704 BRAD gene models) were between R500 and Chiifu and IMB211 and Chiifu, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465187/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465187/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of total number of SNPs detected and annotated to different regions of the genome between Chiifu and two genotypes of <em>B. rapa</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Characterization of detected SNPs</h3><p id="p-28">The distribution and frequency are important considerations of using SNPs as genetic markers. The current study found the SNP rate of uniquely mapped gene models showed a nonuniform distribution across the genome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). This observation could reflect low sequence coverage or segments more closely related because of the breeding history of these two genotypes or the presence of detected and undetected ancestral centromeres in the <em>B. rapa</em> genome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cheng <em>et al.</em> 2013</a>). Among regions with sufficient coverage for SNP detections (coverage of four or more reads, 66 to 77 million bp), the SNP rate was 1 SNP per 200 bp for R500 compared with IMB211, 1 SNP/120 bp for R500 compared with Chiifu, and 1 SNP/123 bp for IMB211 compared with Chiifu. In plants, SNP frequencies vary widely; for example, 1 SNP/124 bp in coding regions among 36 inbred lines of maize (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ching <em>et al.</em> 2002</a>), 1 SNP/72 bp in expressed genes among 13 lines of sugar beet (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schneider <em>et al.</em> 2007</a>), and 1 SNP/2.1 kb to 1 SNP/1.2 kb between two cultivars of <em>B. napus</em>, Tapidor, and Ningyou 7 (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Trick <em>et al.</em> 2009</a>). The SNP frequency observed in <em>Brassica rapa</em> coding regions is therefore within the range of those reported for other plant species. Out of 330,995 putative SNPs that were detected between R500 and IMB211, 202,295 (61%) were annotated in coding regions of <em>B. rapa</em> genome. Of these, 66,327 were nonsynonymous and 135,305 were synonymous changes. As expected, most SNPs occurred in the third nucleotide position of the codon unit (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), suggesting that our SNP calling procedures are working correctly. A comparison of the ratio of <em>K</em>a to <em>K</em>s for all 10 chromosomes indicated that chromosomes two and four had significantly higher <em>KaKs</em> ratios than the rest of the genome, whereas chromosomes five and 10 had significantly lower ratios (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). We hypothesized that this may be because of different proportions of the three sub-genomes of <em>B. rapa</em> because the nonsynonymous substitution rates are known to vary among the sub-genomes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2012</a>). To test this, we calculated the <em>K</em>a<em>K</em>s for the three sub-genomes (see <em>Materials and Methods</em>) and found that the ratio of <em>K</em>a<em>K</em>s of LF (least fragmented) is significantly lower than MF1 and MF2 (more fragmented), similar to previous observations (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2012</a>). We next used a permutation-based approach to determine if the <em>KaKs</em> differences among chromosomes could be explained by sub-genome composition of each chromosome. We generated 1000 permuted genomes where, for each permutation, the proportion of LF, MF1, and MF2 genes on each chromosome matched the original dataset. We then asked how often the <em>KaKs</em> differences of the original chromosomes exceed those in the permuted dataset. The observed increased <em>KaKs</em> for chromosome two exceeded that in the 91% of the permuted datasets, suggesting that sub-genome composition may not entirely explain the high <em>KaKs</em> for this chromosome. In contrast, chromosome four exceeded the permuted <em>KaKs</em> differences only 46% of the time, indicating that its high <em>KaKs</em> rate can be explained by its sub-genome composition. Chromosome five had an unusually low <em>KaKs</em> rate that was lower than 99% of the permuted datasets, strongly suggesting that its low <em>KaKs</em> rate is driven by something other than sub-genome fractionation. Finally, the low <em>KaKs</em> of chromosome 10 was lower than 84% of the permuted datasets. In summary, sub-genome distribution can explain some, but not all, of the chromosomal variance in <em>KaKs</em>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP annotation of B. rapa using snpeff. (A) SNP rate (total number of SNPs/100 bp of gene) across 10 chromosomes. Blue line indicates rolling mean across 25 genes. (B) The distribution of SNPs at different codon positions. (C) KaKs box plot of all chromosomes. Asterisks indicate significance at P &#x2264; 0.05 (permutation testing)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1729262988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP annotation of &lt;em&gt;B. rapa&lt;/em&gt; using snpeff. (A) SNP rate (total number of SNPs/100 bp of gene) across 10 chromosomes. Blue line indicates rolling mean across 25 genes. (B) The distribution of SNPs at different codon positions. (C) &lt;em&gt;K&lt;/em&gt;aKs box plot of all chromosomes. Asterisks indicate significance at &lt;em&gt;P&lt;/em&gt; &#x2264; 0.05 (permutation testing).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2065/F1.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/11/2065/F1.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465285" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">SNP annotation of <em>B. rapa</em> using snpeff. (A) SNP rate (total number of SNPs/100 bp of gene) across 10 chromosomes. Blue line indicates rolling mean across 25 genes. (B) The distribution of SNPs at different codon positions. (C) <em>K</em>aKs box plot of all chromosomes. Asterisks indicate significance at <em>P</em> ≤ 0.05 (permutation testing).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Validation of putative SNPs</h3><p id="p-30">Sources of errors within NGS technologies could propagate to SNP detection and can ultimately result in poorly resolved genotypes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Depristo <em>et al.</em> 2011</a>). To evaluate the performance of our SNP detection pipeline, primers were designed to validate putative SNPs detected between R500 and IMB211 (see <em>Materials and Methods</em>). We found that there were, in total, 202 SNPs (including the 100 SNPs we originally chose to validate) detected by RNA-Seq across these regions. Of these 202, there was no Sanger data available for 32 (failed PCR or sequencing), leaving 170 where we had both RNA-Seq and Sanger data. Among these 170, 158 were confirmed by Sanger sequencing. Considering only those positions with RNA-Seq and Sanger data, our validation rate was 93% (false-positive rate of 7%). Considering all 202 SNPs detected by RNA-Seq the validation rate was 78%, the false-positive rate was 6%, and 16% are unknown due to no Sanger data.</p><p id="p-31">To investigate the basis of the 12 false positives, we re-examined the RNA-Seq data. At these apparent false-positive positions, the RNA-Seq data overwhelmingly support the presence of the SNPs: an average of 99.2% of the reads matched the SNP called with an average coverage of 368 and 263 reads for R500 and IMB211, respectively. We hypothesized that the difference between Sanger and RNA-Seq could be due to polymorphisms still segregating in the parental stocks. To test this idea, we used Sanger sequencing to assay four apparent false positives in RILs derived from these parents (see <em>Materials and Methods</em>). All four of the apparent false positive SNPs were detected in the RILs. Based on this result, we conclude that the apparent false-positive SNPs are generally not due to errors in our SNP detection pipeline but because the parental strains are not fully homogeneous (<em>i.e.</em>, there are some polymorphisms between the IMB211 used for our RNA-Seq and for Sanger validation).</p><p id="p-32">To determine the false-negative rate, we identified all SNPs present in a 150-bp window surrounding the focal SNP in each Sanger reaction (193 SNPs total); 158 of these were called as SNPs from our RNA-Seq data. Of the remaining 35 where no SNP was detected by RNA-Seq, 29 were in regions with RNA-Seq coverage below the threshold required by our SNP detection pipeline (four reads in each genotype). This is probably because the Sanger sequencing was performed on genomic DNA and includes intronic and intergenic regions. The remaining six were not called as SNPs by our pipeline and represent false negatives (false-negative rate of 4% in regions with good RNA-Seq coverage).</p><p id="p-33">One of the objectives of this study is the development of a marker system involving the genotyping of <em>B. rapa</em> genotypes by deep RNA-Seq for use in construction of recombination maps. Because of high validation rate of this dataset, the detected SNPs can now be used as markers to genotype the RIL mapping population generated from these two parents (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Iniguez-Luy <em>et al.</em> 2009</a>).</p></div><div id="sec-24" class="subsection"><h3>R500 transcriptome assembly and re-annotation of <em>B. rapa</em> genome</h3><p id="p-34">The pipeline for R500 transcriptome assembly and re-annotation of <em>B. rapa</em> genome involved three steps. First, transcriptome sequence libraries were assembled and novel transcripts were detected and annotated. Second, the current genome annotation of <em>B. rapa</em> was updated and annotated. Third, both the novel annotated transcripts and updated annotated transcripts were combined to generate a final improved transcriptome assembly and genome annotation of <em>B. rapa</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). For assembling and re-annotation purposes, we used reads from one genotype only (R500) to avoid mis-assemblies that could be caused by the approximately 330,000 SNPs existing between the two genotypes. In addition, we only used a subset of total reads for R500 to reduce computational requirements.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pipeline illustrating the overall transcriptome assembly and annotation of Brassica rapa genotype R500." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1729262988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pipeline illustrating the overall transcriptome assembly and annotation of &lt;em&gt;Brassica rapa&lt;/em&gt; genotype R500.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2065/F2.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/11/2065/F2.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465225" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Pipeline illustrating the overall transcriptome assembly and annotation of <em>Brassica rapa</em> genotype R500.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Transcriptome assembly and annotation of novel transcripts</h3><p id="p-36">Currently, there are two basic approaches for transcriptome assembly using RNA-Seq technologies: <em>de novo</em>, which are sequenced reads that are directly assembled into transcripts without the need of reference, and reference-based, which are transcripts that are assembled by first mapping to a reference genome (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haas and Zody 2010</a>). Both methods have advantages and disadvantages. For example, reference-based methods are computationally less demanding, tolerate sequencing errors, and detect repeats through alignment; however, these methods are dependent on a reference genome, assume that transcripts are collinear with the genome, and mismatched genome alignment or genome assembly errors lead to errors in transcriptome prediction. In contrast, <em>de novo</em> methods are not dependent on a reference genome and therefore can define transcripts not present in the reference as well as defining noncollinear transcripts and <em>trans</em>-spliced transcripts that result from chromosomal rearrangements. However, they will perform poorly on low-expressed genes and can produce chimeras and misassemblies due to repeats. In this study we used a hybrid approach that combines the novel transcripts from both <em>de novo</em>–based and reference-based strategies to ultimately incorporate these data into an updated annotation of <em>B. rapa</em>.</p></div><div id="sec-26" class="subsection"><h3>Overview of assemblers used in this study</h3><p id="p-37">Velvet-Oases and Trinity were selected to <em>de novo</em> assemble the sequence reads. Although both these assemblers use similar de Bruijin graph algorithms, they differ in how they handle sequencing errors, resolve ambiguities, and utilize read pair information. For reference-based transcriptome assembly, we chose TopHat-Cufflinks assembly method.</p></div><div id="sec-27" class="subsection"><h3><em>de novo</em> assembly pipeline to detect novel transcripts</h3><p id="p-38">Because all <em>de novo</em> assemblers are computationally intensive, we used 396,946,216 high-quality reads from 10 lanes for <em>de novo</em> assembly using Velvet-Oases and Trinity (see <em>Materials and Methods</em>). The result from the Velvet-Oases pipeline indicated that in comparison with individual k-mer assemblies, the merged assembly with k-mer size of 55 yielded the best assembly because it had the maximum number of transcripts (601,915), largest average transcript size (1553 bp), largest transcript length (26,060 bp), and highest N50 length (2218 bp) (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). A custom python script was then used to choose a single representative transcript for each gene based on coverage and sequence length (see <em>Materials and Methods</em>). This procedure yielded 43,816 best Velvet-Oases transcripts from 601,915 initial transcripts.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465235/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465235/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary statistics for individual and merged Velvet-Oases assemblies</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">To supplement the Velvet-Oases assembly, we used another <em>de novo</em> assembler, Trinity, which was shown to recover more full-length transcripts across a range of sensitivity levels similar to genome alignment methods (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grabherr <em>et al.</em> 2011</a>). For Trinity, we used the same input reads as for Velvet-Oases. A total of 158,863 transcripts were assembled from Trinity; 61,438 (38.6%) of the assembled transcripts were more than 1000 bp long and 5107 (32.1%) were more than 4000 bp long. The mean transcript length, maximum transcript length, and N50 size of Trinity transcripts were 1112, 22,887, and 1863, respectively. To assess the Trinity assembly, bowtie was used to map the reads back to the Trinity assembly and 97% of the reads mapped to the assembly. To identify a core/reference set of transcripts from the 158,863 Trinity transcripts, we performed BLASTX of the Trinity transcripts against Plant RefSeq database and retained the 98,883 with homology to known plant proteins. The transcripts were then subjected to duplicate removal using a custom awk script, yielding 39,084 transcripts (see <em>Materials and Methods</em>).</p><p id="p-40">After initial assembly with Velvet-Oases and Trinity, to extract novel transcripts not present in the existing <em>B. rapa</em> genome annotation, the transcripts were first blasted against the <em>B. rapa</em> CDS, downloaded from BRAD, using megablast (percent identity of ≥95 and e-value filter of 1e−06). To further determine if the extracted novel transcripts were <em>bona fide B. rapa</em> transcripts, these were blasted against the <em>B. rapa</em> genome using megablast (with the same parameters as above). Those transcripts that had genome matches were then subjected to chimera removal using a custom script. The Velvet-Oases transcripts that did not have a <em>B. rapa</em> genome match (due to either gaps or inaccuracies in the genome assembly or due to contamination) were blasted against the nr database of NCBI (e-value cut-off of 1e−06) and only those that have at least one hit on the nr database were kept. Because Trinity transcripts were already derived from blasting to NCBI Plant RefSeq, no further blasting was performed. Finally, the genome-matched novel transcripts (after removing chimeras) and nongenome-matched novel transcripts (after blasting to nr database) were merged separately for Velvet-Oases and Trinity to obtain assembly specific novel transcripts. The full results for each of the above steps are given in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465183/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465183/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Downstream processing of Velvet-Oases and Trinity transcripts after initial assembly</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Reference-based assembly pipeline to detect novel transcripts</h3><p id="p-41">Because reference-based methods are highly sensitive for transcriptome reconstruction, for the current study we used a total of 157,164,008 reads from three lanes only (see <em>Materials and Methods</em>) to align to <em>B. rapa</em> genome using TopHat. Transcripts were then assembled from the mapped fragments using the Cufflinks assembler, producing a total of 76,640 transcripts. We next used Cuffcompare and found that 49.7% of all the transcripts matched exactly to the annotated exons downloaded from BRAD. The remaining 50.3% transcripts were classified into different classes as shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. These results reflect the potential incompleteness or gaps of the current annotation of the <em>B. rapa</em> as well as the complex nature of transcription and RNA processing. We extracted a total of 6700 unknown intergenic transcripts (Class Code = “u”) as putatively novel transcripts because they occur in intergenic regions without any <em>B. rapa</em> annotation. To find out which of these transcripts corresponded to known genes from other organisms, we ran BLASTP against the nr database. Here, almost half of 6700 transcripts had some nr annotation (3275) while the rest did not (3425).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465259/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465259/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">The number of transcripts assembled with Cufflinks and the percentage they represent in the assembly after Cuffcompare analysis</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>Meta-assembly</h3><p id="p-43">Finally, the novel transcripts identified from <em>de novo</em>–based assembly pipeline (14,758) and reference-based assembly pipeline (6700) were assembled together using CAP3 assembler. Further filtering to remove ambiguous ORF transcripts, very long transcripts and short isoforms resulted in final number of 3537 transcripts (see <em>Materials and Methods</em>). We also found that a total of 1197 transcripts remained unmapped to the genome and, among them, 780 transcripts have plant hits to the NCBI nr database. The remaining 417 transcripts have no hits in the NCBI database. These transcripts along with their gene descriptions have been saved to <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS5.fa.zip">File S5</a> and <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS6.txt">File S6</a>, respectively.</p></div><div id="sec-30" class="subsection"><h3>Comparison of different assemblers</h3><p id="p-44">A comparison of the Velvet-Oases, Trinity, and TopHat-Cufflinks assemblers was performed using standard assembly statistics. The result showed that Velvet-Oases outperformed Trinity and TopHat-Cufflinks for all assembly parameters except for the percentage of annotated transcripts (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). A higher percentage of annotated transcripts are generated by TopHat-Cufflinks even though fewer reads were used, consistent with the benefits of using the reference genome to anchor assemblies. We compared the novel transcripts of the three assemblers and found that there is only a marginal overlap, indicating that each transcriptome assembler has their own strengths (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The possible reasons for the marginal overlap are differences in the specifics of the assembly graphs, determination and weighting of graph edges, approaches to handle sequence errors, and approaches to handle diverse expression level.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465263/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465263/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Comparison of assembly statistics from <em>de novo</em> (Velvet-Oases and Trinity) and reference (TopHat-Cufflinks) assemblers</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams showing unique and shared novel transcripts detected between (A) Velvet-Oases, Trinity, and TopHat-Cufflinks assemblers, (B) de novo (Velvet-Oases and Trinity), and reference-based (TopHat-Cufflinks) pipelines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1729262988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Venn diagrams showing unique and shared novel transcripts detected between (A) Velvet-Oases, Trinity, and TopHat-Cufflinks assemblers, (B) &lt;em&gt;de novo&lt;/em&gt; (Velvet-Oases and Trinity), and reference-based (TopHat-Cufflinks) pipelines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2065/F3.medium.gif" width="249" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/11/2065/F3.medium.gif" width="249" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465214" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Venn diagrams showing unique and shared novel transcripts detected between (A) Velvet-Oases, Trinity, and TopHat-Cufflinks assemblers, (B) <em>de novo</em> (Velvet-Oases and Trinity), and reference-based (TopHat-Cufflinks) pipelines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">The final output (novel transcripts) from the combined <em>de novo</em> assemblies (Velvet-Oases and Trinity) were next compared to the novel transcripts from reference-based assembly (TopHat-Cufflinks) and, as expected, the reference-based novel transcripts have a higher N50 than the <em>de novo</em>–based novel transcripts (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). Interestingly, we found more unique novel transcripts in <em>de novo</em>–based assemblies compared with the reference-based assembly, probably due to the number of input reads (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Together, our assembly results show the importance of combining results from several different algorithms.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465123/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465123/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Comparison of <em>de novo</em>–based and reference-based assembly methods on the final output (novel transcripts detected)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h3>Generating an updated annotation of the <em>B. rapa</em> genome</h3><p id="p-47">The PASA pipeline was used for updating the gene structure annotation of <em>B. rapa</em> genome (see <em>Materials and Methods</em>). A total of 23,754 genes were updated by this approach (<a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 9</a>), of which 293 genes were identified as "fused" (where two or more neighboring genes were fused as one gene).</p><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465195/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465195/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9</span> <span class="caption-title">Comparison of original and updated <em>B. rapa</em> annotations using PASA</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-32" class="subsection"><h3>Comprehensive <em>B. rapa</em> transcriptome and annotation</h3><p id="p-48">The 3537 novel transcripts along with their gene annotations were concatenated with 40,727 <em>B. rapa</em> transcripts (which includes both PASA updated and original transcripts) and their gene annotations. The final annotation set contained a total of 44,264 genes. We found that the final annotated <em>B. rapa</em> gene models were notably improved compared with older annotation based on visual inspection on IGV (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS7.zip">File S7</a>) and also in terms of mapping, for example, the percentage of RNA-Seq reads mapped was 2.5% higher in the updated annotation compared with older annotation. The updated gene annotation file (bed) and updated gene description file have been saved in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS8.txt">File S8</a> and <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS9.txt">File S9</a>, respectively.</p></div><div id="sec-33" class="subsection"><h3>Validation of assembled novel transcripts</h3><p id="p-49">To test the authenticity of <em>de novo</em> and reference assembled novel transcripts, semi-quantitative RT-PCR was performed on 70 different novel transcripts. Agarose gel electrophoresis showed that 46 of the 70 primer pairs obtained a band of the expected size in at least one tissue type (<a id="xref-table-wrap-10-1" class="xref-table" href="#T10">Table 10</a>). Out of the remaining 24 primer pairs that did not have bands, 14 of them had very low expression counts (0–5 mapped reads) in the tissues that were tested by RT-PCR, so they may be authentic but not expressed in the tissue available for testing. The remaining 10 with no bands may be due to failed PCR, bad primers, or poor assemblies because all of them have higher expression counts. Overall, deep RNA-Seq analysis was able to find novel transcripts and novel exons outside the current <em>B. rapa</em> annotation, with the majority being validated by RT-PCR.</p><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465301/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465301/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 10</span> <span class="caption-title">RT-PCR validations of assembled transcripts from different assembly methods</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-34" class="subsection"><h3>Validation of novel and PASA updated transcript RNA-Seq structure and annotation</h3><p id="p-50">Our updated <em>B. rapa</em> transcriptome was generated from strain R500. It is important to ask if the differences between our R500 updated transcriptome and the original Chiifu annotation from BRAD are due to structural differences in the R500 and Chiifu transcriptomes, or if our updated R500 transcriptome represents a valid improvement for Chiifu. To do this, we used an <em>in silico</em> validation procedure, taking advantage of publicly available Chiifu RNA-Seq reads. We randomly chose 60 novel transcripts and 60 PASA updated transcripts and visualized the RNA-Seq coverage for R500 and Chiifu reads, along with the original and updated genome annotation in IGV (see <em>Materials and Methods</em>). There were a total of 12 transcripts where alternative splicing or low coverage precluded full classification; of the 108 transcripts that we could classify, the transcript structure of R500 and Chiifu was the same for 94% of them (<a id="xref-table-wrap-11-1" class="xref-table" href="#T11">Tables 11</a> and <a id="xref-table-wrap-12-1" class="xref-table" href="#T12">12</a>). The <em>de novo</em> transcripts matched the RNA-Seq data for both genotypes 70% of the time (<a id="xref-table-wrap-11-2" class="xref-table" href="#T11">Table 11</a>). There were only three transcripts (6%) where the <em>de novo</em> annotation matched R500 but not Chiifu (<a id="xref-table-wrap-11-3" class="xref-table" href="#T11">Table 11</a>). Focusing on the PASA updated annotations, 50 updated annotations (91%) matched both genotypes (<a id="xref-table-wrap-12-2" class="xref-table" href="#T12">Table 12</a>). In contrast there were only two transcripts (4%) where the original annotation was better than the update for Chiifu (<a id="xref-table-wrap-12-3" class="xref-table" href="#T12">Table 12</a>). In summary, the <em>de novo</em> and PASA updated transcripts represent a significant improvement for both the R500 and Chiifu genotypes. The genes used for <em>in silico</em> validation are given in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/TableS5.pdf">Table S5</a>, and IGV screen shots for both PASA and novel transcript annotations have been saved in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS7.zip">File S7</a> and <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS10.zip">File S10</a>.</p><div id="T11" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465202/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465202/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 11</span> <span class="caption-title">Summary of <em>in silico</em> validation of novel transcripts</span><div class="sb-div caption-clear"></div></div></div><div id="T12" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465234/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465234/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 12</span> <span class="caption-title">Summary of <em>in silico</em> validation of PASA updated transcript annotations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-35" class="subsection"><h3>Functional annotation of revised <em>B. rapa</em> transcriptome</h3><p id="p-53">Functional annotation based on BLASTX and gene ontology allowed the classification of re-annotated transcripts into functional groups. The categorization of all BLASTX results indicate that <em>Arabidopsis thaliana</em>, <em>Vitis vinifera</em>, <em>Populus trichocarpa</em>, <em>Oryza sativa</em>, and <em>Ricinus communis</em> were the top five plant species in terms of number of hits to the revised transcriptome. The annotation step of Blast2GO assigned functions to 32,317 (73%) gene models. A total of 155,618 GO terms were obtained for a total of 44,239 re-annotated gene models. The distribution of the gene models in different GO categories is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. Blast2GO can also be used to integrate other information such as metabolic pathways using KEGG annotation (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kanehisa and Goto 2000</a>). We mapped all the predicted proteins to the reference canonical pathways in KEGG for functional categorization and annotation (see <em>Materials and Methods</em>). Of the 44,264 gene models, 13,203 had one or more KEGG annotations belonging to 321 different KEGG pathways. These GO and KEGG annotations will be helpful to researchers using the updated transcripts, for example, enabling categorization of transcriptional responses by the types of enriched GO or KEGG terms. The GO annotated table for <em>B. rapa</em> is saved in <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.012526.DC1/FileS11.txt">File S11</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histogram of level 2 GO term assignment of B. rapa re-annotated gene models. Results are summarized for three main GO categories: biological process (P), molecular function (F), and cellular component (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1729262988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histogram of level 2 GO term assignment of &lt;em&gt;B. rapa&lt;/em&gt; re-annotated gene models. Results are summarized for three main GO categories: biological process (P), molecular function (F), and cellular component (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2065/F4.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/11/2065/F4.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2065/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465188" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-54" class="first-child">Histogram of level 2 GO term assignment of <em>B. rapa</em> re-annotated gene models. Results are summarized for three main GO categories: biological process (P), molecular function (F), and cellular component (C).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section conclusions" id="sec-36"><h2 class="">Conclusions</h2><p id="p-55">In this study, we have demonstrated that deep RNA-Seq of two genotypes using different tissues, growth conditions, and environments provides enough coverage for the detection of a large number of polymorphisms, discovery of unknown transcripts, and an update of existing annotation. Our results showed that no transcriptome assembler in either <em>de novo</em> assembly category or reference-based category is the best choice for transcriptome assembly and integrating assemblers from both the categories can offer more accurate assembly. We found that the sub-genome distribution can explain some, but not all, of the chromosomal variance in <em>KaKs</em>. Finally, even though there is divergence between R500 and Chiifu in terms of SNPs, we found that this divergence is not high enough to cause significant structural divergence of the transcriptome, as the updated annotation (both novel and PASA) provided a significant improvement to the existing annotation for both genotypes. The development of functional gene-based markers using RNA-Seq between contrasting genotypes in this study will help researchers using the parental strains and derived RILs for QTL and eQTL mapping. Ultimately, they could serve as excellent markers for marker-assisted selection in crop breeding.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We specially thank Brian Haas for helping with PASA installation, troubleshooting, and preliminary analysis of the PASA results. We also thank Cynthia Weinig, Marcus T. Brock, and Matthew J. Rubin for providing the field material. We kindly thank Dan Fulop, Cody Markelz, and Kazunari Nozue for their constructive suggestions and useful comments regarding the manuscript. This work is funded by grant from National Science Foundation (grant number IOS 0923752).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-57">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012526/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012526/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-58"><em>Communicating editor: N. D. Young</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-06-10"><span class="received-label">Received </span>June 10, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-08-08"><span class="accepted-label">Accepted </span>August 8, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Devisetty <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.2" data-doi="10.1038/nrg2220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Steady progress and recent breakthroughs in the accuracy of automated genome annotation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrent%26rft.auinit1%253DM.%2BR.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D73%26rft.atitle%253DSteady%2Bprogress%2Band%2Brecent%2Bbreakthroughs%2Bin%2Bthe%2Baccuracy%2Bof%2Bautomated%2Bgenome%2Bannotation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2220%26rft_id%253Dinfo%253Apmid%252F18087260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18087260&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252065700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.3" data-doi="10.1006/jmbi.1997.0951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Prediction of complete gene structures in human genomic DNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBurge%26rft.auinit1%253DC.%26rft.volume%253D268%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D94%26rft.atitle%253DPrediction%2Bof%2Bcomplete%2Bgene%2Bstructures%2Bin%2Bhuman%2Bgenomic%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.0951%26rft_id%253Dinfo%253Apmid%252F9149143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.0951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9149143&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WV87100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.4" data-doi="10.1105/tpc.113.110486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandakova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deciphering the diploid ancestral genome of the Mesohexaploid Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.110486%26rft_id%253Dinfo%253Apmid%252F23653472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzUvMTU0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.5" data-doi="10.1371/journal.pone.0036442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Biased gene fractionation and dominant gene expression among the subgenomes of Brassica rapa.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCheng%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36442%26rft.epage%253De36442%26rft.atitle%253DBiased%2Bgene%2Bfractionation%2Band%2Bdominant%2Bgene%2Bexpression%2Bamong%2Bthe%2Bsubgenomes%2Bof%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036442%26rft_id%253Dinfo%253Apmid%252F22567157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22567157&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.6" data-doi="10.1186/1471-2156-3-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ching</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">SNP frequency, haplotype structure and linkage disequilibrium in elite maize inbred lines.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DChing%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DSNP%2Bfrequency%252C%2Bhaplotype%2Bstructure%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Belite%2Bmaize%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-3-19%26rft_id%253Dinfo%253Apmid%252F12366868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-3-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12366868&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.11.2065.7" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, L. Wang le, M. Coon, T. Nguyen <em>et al</em>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.8" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.9" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.10" data-doi="10.1007/BF02712670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A plant DNA minipreparation: Version II.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Rep.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BRep.%26rft.volume%253D1%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02712670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02712670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.11" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.12" data-doi="10.2307/2409671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorn</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetics of Brassica campestris. 1. Genetic constraints on evolution of life-history characters.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D45%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FL11400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.13" data-doi="10.1038/hdy.2010.103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinig</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The quantitative-genetic and QTL architecture of trait integration and modularity in Brassica rapa across simulated seasonal settings.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DEdwards%26rft.auinit1%253DC.%2BE.%26rft.volume%253D106%26rft.issue%253D4%26rft.spage%253D661%26rft.epage%253D677%26rft.atitle%253DThe%2Bquantitative-genetic%2Band%2BQTL%2Barchitecture%2Bof%2Btrait%2Bintegration%2Band%2Bmodularity%2Bin%2BBrassica%2Brapa%2Bacross%2Bsimulated%2Bseasonal%2Bsettings.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.103%26rft_id%253Dinfo%253Apmid%252F20736971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20736971&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288703900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-other" id="cit-4.11.2065.14"><div class="cit-metadata unstructured">Edwards, C. E., M. S. Haselhorst, A. M. McKnite, B. E. Ewers, D. G. Williams <em>et al.</em>, 2009 Genotypes of Brassica rapa respond differently to plant-induced variation in air CO2 concentration in growth chambers with standard and enhanced venting. Theor. Appl. Genet. 119<strong>:</strong> 991–1004.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.15" data-doi="10.1016/j.tibtech.2004.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Plant bioinformatics: from genome to phenome.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DEdwards%26rft.auinit1%253DD.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D232%26rft.epage%253D237%26rft.atitle%253DPlant%2Bbioinformatics%253A%2Bfrom%2Bgenome%2Bto%2Bphenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2004.03.002%26rft_id%253Dinfo%253Apmid%252F15109809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2004.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15109809&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221435100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.16" data-doi="10.1038/nmeth.1613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Computational methods for transcriptome annotation and quantification using RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGarber%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D469%26rft.epage%253D477%26rft.atitle%253DComputational%2Bmethods%2Bfor%2Btranscriptome%2Bannotation%2Band%2Bquantification%2Busing%2BRNA-seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1613%26rft_id%253Dinfo%253Apmid%252F21623353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623353&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291031800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.17" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.18" data-doi="10.1038/nbt0510-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Advancing RNA-Seq analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D421%26rft.epage%253D423%26rft.atitle%253DAdvancing%2BRNA-Seq%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0510-421%26rft_id%253Dinfo%253Apmid%252F20458303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0510-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20458303&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.19" data-doi="10.1093/nar/gkg770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. K.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5654</span>–<span class="cit-lpage">5666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D5654%26rft.epage%253D5666%26rft.atitle%253DImproving%2Bthe%2BArabidopsis%2Bgenome%2Bannotation%2Busing%2Bmaximal%2Btranscript%2Balignment%2Bassemblies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg770%26rft_id%253Dinfo%253Apmid%252F14500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xOS81NjU0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.20" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: A DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-other" id="cit-4.11.2065.21"><div class="cit-metadata unstructured">Iniguez-Luy, F. L., L. Lukens, M. W. Farnham, R. M. Amasino and T. C. Osborn, 2009 Development of public immortal mapping populations, molecular markers and linkage maps for rapid cycling Brassica rapa and B. oleracea. Theor. Appl. Genet. 120<strong>:</strong> 31–43.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.22" data-doi="10.1093/nar/28.1.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">KEGG: Kyoto encyclopedia of genes and genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.23" data-doi="10.1073/pnas.1309606110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimenez-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulop</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E2655</span>–<span class="cit-lpage">E2662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1309606110%26rft_id%253Dinfo%253Apmid%252F23803858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI4L0UyNjU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichihashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Headland</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-throughput method for Illumina RNA-Seq library preparation.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DKumar%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.spage%253D202%26rft.epage%253D202%26rft.atitle%253DA%2BHigh-Throughput%2BMethod%2Bfor%2BIllumina%2BRNA-Seq%2BLibrary%2BPreparation.%26rft_id%253Dinfo%253Apmid%252F22973283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22973283&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.25" data-doi="10.1111/j.1467-7652.2012.00718.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berkman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single nucleotide polymorphism discovery from wheat next-generation sequence data.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DLai%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D743%26rft.epage%253D749%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bfrom%2Bwheat%2Bnext-generation%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00718.x%26rft_id%253Dinfo%253Apmid%252F22748104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00718.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748104&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306131400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.26" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.27" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.28" data-doi="10.1093/bioinformatics/bth315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majoros</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2878</span>–<span class="cit-lpage">2879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth315%26rft_id%253Dinfo%253Apmid%252F15145805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTYvMjg3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.29" data-doi="10.2307/2410742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Pleiotropy causes long-term genetic constraints on life-history evolution in Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1849</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D50%26rft.spage%253D1849%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VQ71900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, K, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Taxonomy, cytogenetics and origin of crop Brassicas, a review.</span> <abbr class="cit-jnl-abbrev">Opera Bot.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpera%2BBot.%26rft.volume%253D55%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.31" data-doi="10.1093/nar/gkl842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maglott</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D61</span>–<span class="cit-lpage">D65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl842%26rft_id%253Dinfo%253Apmid%252F17130148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNS9zdXBwbF8xL0Q2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.32" data-doi="10.1126/science.1203357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhind</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative functional genomics of the fission yeasts.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">930</span>–<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203357%26rft_id%253Dinfo%253Apmid%252F21511999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAzMi85MzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.33" data-doi="10.1101/gr.10.4.516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solovyev</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ab initio gene finding in Drosophila genomic DNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSalamov%26rft.auinit1%253DA.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D516%26rft.epage%253D522%26rft.atitle%253DAb%2Binitio%2BGene%2BFinding%2Bin%2BDrosophila%2BGenomic%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.516%26rft_id%253Dinfo%253Apmid%252F10779491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-other" id="cit-4.11.2065.34"><div class="cit-metadata unstructured">Schneider, K., D. Kulosa, T. R. Soerensen, S. Mohring, M. Heine <em>et al.</em>, 2007 Analysis of DNA polymorphisms in sugar beet (Beta vulgaris L.) and development of an SNP-based map of expressed genes. Theor. Appl. Genet. 115<strong>:</strong> 601–615.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.35" data-doi="10.1093/bioinformatics/bts094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vingron</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts094%26rft_id%253Dinfo%253Apmid%252F22368243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyOC84LzEwODYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.36" data-doi="10.1038/nbt1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D1135%26rft.epage%253D1145%26rft.atitle%253DNext-generation%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1486%26rft_id%253Dinfo%253Apmid%252F18846087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846087&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259926000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.37" data-doi="10.1101/gr.103846.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Surget-Groba</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montoya-Burgos</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimization of de novo transcriptome assembly from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1432</span>–<span class="cit-lpage">1440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.103846.109%26rft_id%253Dinfo%253Apmid%252F20693479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8xNDMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.38" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.39" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.40" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.41" data-doi="10.1111/j.1467-7652.2008.00396.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bancroft</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single nucleotide polymorphism (SNP) discovery in the polyploid Brassica napus using Solexa transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DTrick%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D334%26rft.epage%253D346%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2B%2528SNP%2529%2Bdiscovery%2Bin%2Bthe%2Bpolyploid%2BBrassica%2Bnapus%2Busing%2BSolexa%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2008.00396.x%26rft_id%253Dinfo%253Apmid%252F19207216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2008.00396.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19207216&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265147500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijveen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisseling</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Primer3Plus, an enhanced web interface to Primer3.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W71–4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D35%26rft.spage%253DW714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.43" data-doi="10.1038/ng.919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genome of the mesopolyploid crop species Brassica rapa.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D1035%26rft.epage%253D1039%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bmesopolyploid%2Bcrop%2Bspecies%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.919%26rft_id%253Dinfo%253Apmid%252F21873998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21873998&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2065.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">WEGO: a web tool for plotting GO annotations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W293–7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D34%26rft.spage%253DW2937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2065.45" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/11/2065.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2051" title="Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2079" title="The Genetic Architecture of Coordinately Evolving Male Wing Pigmentation and Courtship Behavior in Drosophila elegans and Drosophila gunungcola" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/11"><span>Volume 4 Issue 11, November 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/11.cover-source.jpg?itok=oYWFGPiM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2065.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0mCCINSMoW6utNZy29oxAISvw5mErBJ0wJKzHKXXl6A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/11/2065" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/11/2065" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/11/2065" class="active" data-icon-position="" data-hide-link-title="0">Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459723" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xKY1gFyOI5nWOxvay3b3OJTPOCNtQtyT0acAbBA4op8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2065.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/11/2065" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-5Op9Eq7KwKdoUaJ34ZAu53BrH5ZlN4_R57WlMsf49hQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/11/2065" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2065.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459723" id="node459723--2" data-pisa="ggg;4/11/2065" data-pisa-master="ggg;g3.114.012526" data-apath="/ggg/4/11/2065.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Polymorphism Identification and Improved Genome Annotation of <em>Brassica rapa</em> Through Deep RNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Upendra Kumar Devisetty</span>, <span class="highwire-citation-author" data-delta="1">Michael F. Covington</span>, <span class="highwire-citation-author" data-delta="2">An V. Tat</span>, <span class="highwire-citation-author" data-delta="3">Saradadevi Lekkala</span> and <span class="highwire-citation-author" data-delta="4">Julin N. Maloof</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2065-2078;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012526 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459723--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Upendra Kumar Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BKumar%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUpendra%2BKumar%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">An V. Tat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=An%2BV.%2BTat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tat%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAn%2BV.%2BTat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Saradadevi Lekkala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saradadevi%2BLekkala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lekkala%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaradadevi%2BLekkala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/11/2065" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/11/2065&amp;title=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/11/2065&amp;title=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/11/2065&amp;title=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/11/2065&amp;text=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/11/2065&amp;title=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/11/2065&amp;t=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/11/2065&amp;title=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/11/2065&amp;title=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459723/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459723" id="node459723--3" data-pisa="ggg;4/11/2065" data-pisa-master="ggg;g3.114.012526" data-apath="/ggg/4/11/2065.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Polymorphism Identification and Improved Genome Annotation of <em>Brassica rapa</em> Through Deep RNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Upendra Kumar Devisetty</span>, <span class="highwire-citation-author" data-delta="1">Michael F. Covington</span>, <span class="highwire-citation-author" data-delta="2">An V. Tat</span>, <span class="highwire-citation-author" data-delta="3">Saradadevi Lekkala</span> and <span class="highwire-citation-author" data-delta="4">Julin N. Maloof</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2065-2078;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012526 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459723--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Upendra Kumar Devisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Upendra%2BKumar%2BDevisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Devisetty%20UK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUpendra%2BKumar%2BDevisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">An V. Tat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=An%2BV.%2BTat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tat%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAn%2BV.%2BTat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Saradadevi Lekkala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saradadevi%2BLekkala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lekkala%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaradadevi%2BLekkala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2065&amp;rft.epage=2078&amp;rft.atitle=Polymorphism%20Identification%20and%20Improved%20Genome%20Annotation%20of%20Brassica%20rapa%20Through%20Deep%20RNA%20Sequencing&amp;rft.volume=4&amp;rft.issue=11&amp;rft.date=2014-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Devisetty%2C+Upendra+Kumar&amp;rft.au=Covington%2C+Michael+F.&amp;rft.au=Tat%2C+An+V.&amp;rft.au=Lekkala%2C+Saradadevi&amp;rft.au=Maloof%2C+Julin+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459723/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459723/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459723/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459723/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459723/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459723/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459723/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459723/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459723/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459723/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459723/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459723/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/11/2065" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-36" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/11/2065.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/11/2065.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/11/2065.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/11/2065.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
