<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/11/2259" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/11/2259.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/11/2259.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/11/2259.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/11/2259.atom" />
<meta name="HW.pisa" content="ggg;4/11/2259" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Structures of Naturally Evolved CUP1 Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination" />
<meta name="DC.Identifier" content="10.1534/g3.114.012922" />
<meta name="DC.Date" content="2014-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Zhao et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="An important issue in genome evolution is the mechanism by which tandem duplications are generated from single-copy genes. In the yeast Saccharomyces cerevisiae , most strains contain tandemly duplicated copies of [CUP1][1] , a gene that encodes a copper-binding metallothionein. By screening 101 natural isolates of S. cerevisiae , we identified five different types of [CUP1][1] -containing repeats, as well as strains that only had one copy of [CUP1][1] . A comparison of the DNA sequences of these strains indicates that the [CUP1][1] tandem arrays were generated by unequal nonhomologous recombination events from strains that had one [CUP1][1] gene.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095" />
<meta name="DC.Contributor" content="Ying Zhao" />
<meta name="DC.Contributor" content="Pooja K. Strope" />
<meta name="DC.Contributor" content="Stanislav G. Kozmin" />
<meta name="DC.Contributor" content="John H. McCusker" />
<meta name="DC.Contributor" content="Fred S. Dietrich" />
<meta name="DC.Contributor" content="Robert J. Kokoska" />
<meta name="DC.Contributor" content="Thomas D. Petes" />
<meta name="article:published_time" content="2014-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Structures of Naturally Evolved CUP1 Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;An important issue in genome evolution is the mechanism by which tandem duplications are generated from single-copy genes. In the yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, most strains contain tandemly duplicated copies of &lt;i&gt;CUP1&lt;/i&gt;, a gene that encodes a copper-binding metallothionein. By screening 101 natural isolates of &lt;i&gt;S. cerevisiae&lt;/i&gt;, we identified five different types of &lt;i&gt;CUP1&lt;/i&gt;-containing repeats, as well as strains that only had one copy of &lt;i&gt;CUP1&lt;/i&gt;. A comparison of the DNA sequences of these strains indicates that the &lt;i&gt;CUP1&lt;/i&gt; tandem arrays were generated by unequal nonhomologous recombination events from strains that had one &lt;i&gt;CUP1&lt;/i&gt; gene.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/11/01" />
<meta name="citation_mjid" content="ggg;4/11/2259" />
<meta name="citation_id" content="4/11/2259" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/11/2259" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/11/2259.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/11/2259.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/11/2259.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012922" />
<meta name="citation_pmid" content="25236733" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2259" />
<meta name="citation_lastpage" content="2269" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ying Zhao" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author" content="Pooja K. Strope" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author" content="Stanislav G. Kozmin" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author" content="John H. McCusker" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author" content="Fred S. Dietrich" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author" content="Robert J. Kokoska" />
<meta name="citation_author_institution" content="Physical Sciences Directorate, U. S. Army Research Office, Research Triangle Park, North Carolina 27709" />
<meta name="citation_author" content="Thomas D. Petes" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author_email" content="tom.petes@duke.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. L. Angus-Hill;citation_author=A. Schlichter;citation_author=D. Roberts;citation_author=H. Erdjument-Bromage;citation_author=P. Tempst;citation_title=A Rsc3/Rsc30 zinc cluster dimer reveals novel roles for the chromatin remodeler RSC in gene expression and cell cycle control.;citation_pages=741-751;citation_volume=7;citation_year=2001;citation_issue=4;citation_pmid=11336698;citation_doi=10.1016/S1097-2765(01)00219-2" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=J. D. Boeke;citation_author=J. Trueheart;citation_author=G. Natsoulis;citation_author=G. R. Fink;citation_title=5-Fluoroorotic acid as a selective agent in yeast molecular genetics.;citation_pages=164-175;citation_volume=154;citation_year=1987;citation_pmid=3323810;citation_doi=10.1016/0076-6879(87)54076-9" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. S. tCha;citation_author=N. Kleckner;citation_title=ATR Homolog Mec1 Promotes Fork Progression, Thus Averting Breaks in Replication Slow Zones;citation_pages=602-606;citation_volume=297;citation_year=2002;citation_issue=5581;citation_pmid=12142538;citation_doi=10.1126/science.1071398" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Dorsey;citation_author=C. Peterson;citation_author=K. Bray;citation_author=C. E. Paquin;citation_title=Spontaneous amplification of the ADH4 gene in Saccharomyces cerevisiae.;citation_pages=943-950;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1459445" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=S. R. Engel;citation_author=F. S. Dietrich;citation_author=D. G. Fisk;citation_author=G. Binkley;citation_author=R. Balakrishnan;citation_title=The reference genome sequence of Saccharomyces cerevisiae: then and now.;citation_volume=4;citation_year=2014;citation_pmid=24374639;citation_doi=10.1534/g3.113.008995" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. J. Finn;citation_author=J. J. Li;citation_title=Single-stranded annealing induced by re-initiation of replication origins provides a novel and efficient mechanism for generating copy number expansion via non-allelic homologous recombination.;citation_pages=e1003192;citation_volume=9;citation_year=2013;citation_pmid=23300490;citation_doi=10.1371/journal.pgen.1003192" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Fogel;citation_author=J. W. Welch;citation_title=Tandem gene amplification mediates copper resistance in yeast.;citation_pages=5342-5346;citation_volume=79;citation_year=1982;citation_issue=17;citation_pmid=6291039;citation_doi=10.1073/pnas.79.17.5342" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genet.;citation_author=S. Fogel;citation_author=J. W. Welch;citation_author=G. Cathala;citation_author=M. Karin;citation_title=Gene amplification in yeast: CUP1 copy number regulates copper resistance.;citation_pages=347-355;citation_volume=7;citation_year=1983;citation_pmid=24173415;citation_doi=10.1007/BF00445874" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. M. Green;citation_author=K. J. Finn;citation_author=J. J. Li;citation_title=Loss of DNA Replication Control Is a Potent Inducer of Gene Amplification;citation_pages=943-946;citation_volume=329;citation_year=2010;citation_issue=5994;citation_pmid=20724634;citation_doi=10.1126/science.1190966" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Gresham;citation_author=M. M. Desai;citation_author=C. M. Tucker;citation_author=H. T. Jenq;citation_author=D. A. Pai;citation_title=The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.;citation_pages=e1000303-e1000303;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19079573;citation_doi=10.1371/journal.pgen.1000303" />
<meta name="citation_reference" content="citation_title=Guide to Yeast Genetics and Microbiology;citation_author=C. Guthrie;citation_author=G. R. Fink;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=J. E. Haber;citation_author=E. J. Louis;citation_title=Minisatellite origins in yeast and humans.;citation_pages=132-135;citation_volume=48;citation_year=1998;citation_issue=1;citation_pmid=9503027;citation_doi=10.1006/geno.1997.5153" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. L. Ivanov;citation_author=N. Sugawara;citation_author=J. Fishman-Lobell;citation_author=J. E. Haber;citation_title=Genetic Requirements for the Single-Strand Annealing Pathway of Double-Strand Break Repair in Saccharomyces cerevisiae;citation_pages=693-704;citation_volume=142;citation_year=1996;citation_issue=3;citation_pmid=8849880" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=S. Jäger;citation_author=J. Strayle;citation_author=W. Heinemeyer;citation_author=D. H. Wolf;citation_title=Cic1, an adaptor protein specifically linking the 26S proteasome to its substrate, the SCF component Cdc4;citation_pages=4423-4431;citation_volume=20;citation_year=2001;citation_issue=16;citation_pmid=11500370;citation_doi=10.1093/emboj/20.16.4423" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=M. Jasin;citation_author=R. Rothstein;citation_title=Repair of strand breaks by homologous recombination.;citation_volume=5;citation_year=2013;citation_pmid=24097900;citation_doi=10.1101/cshperspect.a012740" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Johnston;citation_author=S. Andrews;citation_author=R. Brinkman;citation_author=J. Cooper;citation_author=H. Ding;citation_title=Complete nucleotide sequence of Saccharomyces cerevisiae chromosome VIII;citation_pages=2077-2082;citation_volume=265;citation_year=1994;citation_issue=5181;citation_pmid=8091229;citation_doi=10.1126/science.8091229" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Karin;citation_author=R. Najarian;citation_author=A. Haslinger;citation_author=P. Valenzuela;citation_author=J. Welch;citation_title=Primary structure and transcription of an amplified genetic locus: the CUP1 locus of yeast.;citation_pages=337-341;citation_volume=81;citation_year=1984;citation_issue=2;citation_pmid=6364141;citation_doi=10.1073/pnas.81.2.337" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=R. Koszul;citation_author=S. Caburet;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Eucaryotic genome evolution through the spontaneous duplication of large chromosomal segments;citation_pages=234-243;citation_volume=23;citation_year=2004;citation_issue=1;citation_pmid=14685272;citation_doi=10.1038/sj.emboj.7600024" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=D. E. Lea;citation_author=C. A. Coulson;citation_title=The distribution of the numbers of mutants in bacterial populations.;citation_pages=264-285;citation_volume=49;citation_year=1949;citation_pmid=24536673;citation_doi=10.1007/BF02986080" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Lee;citation_author=C. M. Carvalho;citation_author=J. R. Lupski;citation_title=A DNA replication mechanism for generating nonrecurrent rearrangements associated with genomic disorders.;citation_pages=1235-1247;citation_volume=131;citation_year=2007;citation_issue=7;citation_pmid=18160035;citation_doi=10.1016/j.cell.2007.11.037" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. L. McCulley;citation_author=T. D. Petes;citation_title=Chromosome rearrangements and aneuploidy in yeast strains lacking both Tel1p and Mec1p reflect deficiencies in two different mechanisms.;citation_volume=107;citation_year=2010;citation_pmid=20534547;citation_doi=10.1073/pnas.1006281107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. P. McCutcheon;citation_author=S. R. Eddy;citation_title=Computational identification of non-coding RNAs in Saccharomyces cerevisiae by comparative genomics;citation_pages=4119-4128;citation_volume=31;citation_year=2003;citation_issue=14;citation_pmid=12853629;citation_doi=10.1093/nar/gkg438" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. Paeschke;citation_author=J. A. Capra;citation_author=V. A. Zakian;citation_title=DNA replication through G-quadruplex motifs is promoted by the Saccharomyces cerevisiae Pif1 DNA helicase.;citation_pages=678-691;citation_volume=145;citation_year=2011;citation_issue=5;citation_pmid=21620135;citation_doi=10.1016/j.cell.2011.04.015" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=F. Paques;citation_author=J. E. Haber;citation_title=Multiple Pathways of Recombination Induced by Double-Strand Breaks in Saccharomyces cerevisiae;citation_pages=349-404;citation_volume=63;citation_year=1999;citation_issue=2;citation_pmid=10357855" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Payen;citation_author=R. Koszul;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Segmental duplications arise from Pol32-dependent repair of broken forks through two alternative replication-based mechanisms.;citation_pages=e1000175-e1000175;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18773114;citation_doi=10.1371/journal.pgen.1000175" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. D. Petes;citation_title=Unequal meiotic recombination within tandem arrays of yeast ribosomal DNA genes.;citation_pages=765-774;citation_volume=19;citation_year=1980;citation_issue=3;citation_pmid=6988084;citation_doi=10.1016/S0092-8674(80)80052-3" />
<meta name="citation_reference" content="citation_title=Proc. Natl. Acad. Sci. USA.;citation_author=W. Song;citation_author=M. Dominska;citation_author=P. W. Greenwell;citation_author=T. D. Petes;citation_title=Genome-wide high-resolution mapping of chromosome fragile sites in Saccharomyces cerevisiae;citation_volume=111;citation_year=2014;citation_pmid=24799712;citation_doi=10.1073/pnas.1406847111" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. St. Charles;citation_author=T. D. Petes;citation_title=High-resolution mapping of spontaneous mitotic recombination hotspots on the 1.1 Mb arm of yeast chromosome IV.;citation_pages=e1003434;citation_volume=9;citation_year=2013;citation_pmid=23593029;citation_doi=10.1371/journal.pgen.1003434" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. H. Sturtevant;citation_author=T. H. Morgan;citation_title=Reverse mutation of the bar gene correlated with crossing over.;citation_pages=746-747;citation_volume=57;citation_year=1925" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. W. Szostak;citation_author=R. Wu;citation_title=Unequal crossing over in the ribosomal DNA of Saccharomyces cerevisiae.;citation_pages=426-430;citation_volume=284;citation_year=1980;citation_issue=5755;citation_pmid=6987539;citation_doi=10.1038/284426a0" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=K. T. Tamai;citation_author=E. B. Gralla;citation_author=L. M. Ellerby;citation_author=J. S. Valentine;citation_author=D. J. Thiele;citation_title=Yeast and mammalian metallothioneins functionally substitute for yeast copper-zinc superoxide dismutase.;citation_pages=8013-8017;citation_volume=90;citation_year=1993;citation_issue=17;citation_pmid=8367458;citation_doi=10.1073/pnas.90.17.8013" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. J. Thiele;citation_author=D. H. Hamer;citation_title=Tandemly duplicated upstream control sequences mediate copper-induced transcription of the Saccharomyces cerevisiae copper-metallothionein gene.;citation_pages=1158-1163;citation_volume=6;citation_year=1986;citation_issue=4;citation_pmid=3537699" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. J. Thomas;citation_author=R. Rothstein;citation_title=Elevated recombination rates in transcriptionally active DNA.;citation_pages=619-630;citation_volume=56;citation_year=1989;citation_issue=4;citation_pmid=2645056;citation_doi=10.1016/0092-8674(89)90584-9" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. Voineagu;citation_author=V. Narayanan;citation_author=K. S. Lobachev;citation_author=S. M. Mirkin;citation_title=Replication stalling at unstable inverted repeats: interplay between DNA hairpins and fork stabilizing proteins.;citation_volume=105;citation_year=2008;citation_pmid=18632578;citation_doi=10.1073/pnas.0804510105" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Wei;citation_author=J. H. McCusker;citation_author=R. W. Hyman;citation_author=T. Jones;citation_author=Y. Ning;citation_title=Genome sequencing and comparative analysis of Saccharomyces cerevisiae strain YJM789.;citation_volume=104;citation_year=2007;citation_pmid=17652520;citation_doi=10.1073/pnas.0701291104" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. W. Welch;citation_author=S. Fogel;citation_author=G. Cathala;citation_author=M. Karin;citation_title=Industrial yeasts display tandem gene iteration at the CUP1 region.;citation_pages=1353-1361;citation_volume=3;citation_year=1983;citation_issue=8;citation_pmid=6621529;citation_doi=10.1128/MCB.3.8.1353" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Yu;citation_author=M. McVey;citation_title=Synthesis-dependent microhomology-mediated end joining accounts for multiple types of repair junctions.;citation_volume=38;citation_year=2010;citation_pmid=20460465;citation_doi=10.1093/nar/gkq379" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=F. Zhang;citation_author=W. Gu;citation_author=M. E. Hurles;citation_author=J. R. Lupski;citation_title=Copy number variation in human health, disease, and evolution.;citation_pages=451-481;citation_volume=10;citation_year=2009;citation_pmid=19715442;citation_doi=10.1146/annurev.genom.9.081307.164217" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Zhang;citation_author=A. F. Zeidler;citation_author=W. Song;citation_author=C. M. Puccia;citation_author=E. Malc;citation_title=Gene copy-number variation in haploid and diploid strains of the yeast Saccharomyces cerevisiae.;citation_volume=193;citation_year=2013;citation_pmid=23307895;citation_doi=10.1534/genetics.112.146522" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Structures of Naturally Evolved CUP1 Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="twitter:description" content="An important issue in genome evolution is the mechanism by which tandem duplications are generated from single-copy genes. In the yeast Saccharomyces cerevisiae , most strains contain tandemly duplicated copies of [CUP1][1] , a gene that encodes a copper-binding metallothionein. By screening 101 natural isolates of S. cerevisiae , we identified five different types of [CUP1][1] -containing repeats, as well as strains that only had one copy of [CUP1][1] . A comparison of the DNA sequences of these strains indicates that the [CUP1][1] tandem arrays were generated by unequal nonhomologous recombination events from strains that had one [CUP1][1] gene.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095" />
<meta name="og-title" property="og:title" content="Structures of Naturally Evolved CUP1 Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/11/2259" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="An important issue in genome evolution is the mechanism by which tandem duplications are generated from single-copy genes. In the yeast Saccharomyces cerevisiae , most strains contain tandemly duplicated copies of [CUP1][1] , a gene that encodes a copper-binding metallothionein. By screening 101 natural isolates of S. cerevisiae , we identified five different types of [CUP1][1] -containing repeats, as well as strains that only had one copy of [CUP1][1] . A comparison of the DNA sequences of these strains indicates that the [CUP1][1] tandem arrays were generated by unequal nonhomologous recombination events from strains that had one [CUP1][1] gene.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
    <title>Structures of Naturally Evolved CUP1 Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9IEui5outY0Gf-NGDDv6WoGar9Ral8ZdpXV7l0Yj_38","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459702","apath":"\/ggg\/4\/11\/2259.atom","pisa":"ggg;4\/11\/2259","ac":{"\/ggg\/4\/11\/2259.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/11\/2259.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2259"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2259"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/11\/2259":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+XO+Hjd8lrPNgJmZ5kkRDtCc1aRXK\\\\\\\/RC7AvPz\\\\\\\/x7HPuBrXJQYTZAz+U0eLUm6ZyHt1cRVYSlPnFSXuDnl8d7Lkky3RUulx+hwxBtMWMT\\\\\\\/k9p9VOolbIaWSOjnn0qSlGv0Tkveht6WbpRqufeDrN1iUSjTRUYy3xgIdBjz2CfZu5IEC\\\\\\\/1sEs1mNzmQ9wLF+czQwMoAW9CdzsNyXyMryoXPnNF0erzTcb8ANmCaHUvS2S+3FJa5mI5qcnqMOMKFLrQVZI1P91CUmqnNQRvYKTuIs4yBq4gszRbyZFEuateU8qkv8vsEYaNLu\\\\\\\/EJ94+xY1+9jGfftXvIcuAOS5dwVZLziq\\\\\\\/5nUZ9mklN\\\\\\\/kDNIbZ3\\\\\\\/QDbBIPBNjzHEI6FN4SLBMHXucYGKule4D5fyZhF+vShj01sZRcIixLf4pmxSHAULA+0a4M72ynRmoPVvjlM+HMkyKzh0KPteroUkYoucZFJimE0EmbGvXiYGJSus3XtUknnWzu+oqGkTcxdgI8S\\\\\\\/GCeKJiSt7Y\\\\\\\/OTtPCUBft38N0yCTwQFZ7CPEqnew8I9DBwYYoa5+LETDHbTi41sZRmTinyAeleVxgOlYzubcM5mUIGAR5FiNWe9C1CObwMvUrJVYdjWwp4TasG2+\\\\\\\/aVT2g82JxXe2BtE9bEYv9iHrwBXiqzXKdLVSxkF2iigIdg52GbMgVZ6COAw6\\\\\\\/0FBmKDSinNASA57X53xDLvf4t7eyqYTCef79A4J+m43RIgYf2g7cuVnMMivPglox\\\\\\\/8J\\\\\\\/vPmKbDX3MpQxhNBWZVTtPYTi6sji7ksz0+qJebxkPEsXfL9Cs24lcbzIRooQWqdEOIebir+o9cwt3y2oT42ly9PuTwOhwnexk6LQN1QXYBP57IEG2b3Eoi4kVJY7e6vutli13PIK8R6JtKPvGywGodiOvqBXcE41iMgHH7Pr+ROcNuLmMcBSPvQjAWjV31\\\\\\\/F0nSQfi48xVlWSb3FqnD90UnRK8e\\\\\\\/UAzckkLrorVDsldFJ+tSSrPxLHP0aKFtPqf8OpXQLf47VdmXnmlLaDpzcLeM3H7KAdiuDUvxFB1drAdKq7jWKVRHjrR+n\\\\\\\/A56hjZqYgAzxmDw1OM8NogOv1hCRB1aRKyCOpA6mw21QsKCPiV\\\\\\\/OJSAFbEHF5zd3zDjKnntI5i8x1tMnr9PELQQkzlCl6vOI58McgjxxmA1ibzYJS0bA5qDKUXklXRkOPNK9Tlc9lR02Avef9j\\\\\\\/ubtvyAtzaqG5GyWzVfKXMjfChZBf\\\\\\\/voT34XZGgcr8uEqWl2g+9qxulCX\\\u0022,\\\u0022iv\\\u0022:\\\u0022C8hiahkIAyBa4TVRbJvsYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d14d3c4b0ed29865a41b65f154a6a4a4cdc1d790f3db5b4ca51b97cbc6a994e6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cg\\\\\\\/7JgHNKQeQWTNLE5TtC5xcm75cvzi8GluuPQA0ACBTogV2IJRdzj0azYhcN6TamM759E4XhqoaTBPRGxVNcuhxPdGbZCwgwHASsJf1KN\\\\\\\/96uU1gmb2qzdcVzD\\\\\\\/WWRPa5LLP01ZlVrnvXiGEkoc\\\\\\\/MqyXthwK\\\\\\\/Ew+O48yNX5T8kUmhZd5t69vIFkbi72hGt\\\\\\\/Quh5smgr5i+B9a4\\\\\\\/5qhtpSPbStiMUuQQSg0duxzH4WiFwKifT9XcaP7eb+qpnABI7kVDZo3kX2NUffKVlJ+Bb7IGfmODEoyjpcU2hef9WK6SsEO32S5oNwkp2c1BnYnqyVxWbf2L6euA1QFVj39WEZGzOUZwCnAws6YyUiFXebwNx57jPrWWxb43zclxuSsKEmDkTJmMUaCIKOoyGfmbx4nsXEjlOjJ4NeIClxCf2T5VvgcgPCJ2DQQXdn9NxM8ZqKZqy3NgHwGwYAH3kium5T2cxY5PD7wd\\\\\\\/kBNm4kP53bpHLh+7VxV908ICRY+EroX45aQXTPYzLCCp00xCFoLdoDRQvx887aPus\\\\\\\/C7RDu1Hlhzmo28dwu3MdwoOgx2bW3AAo7YYAkIOUrhwPkaXYzE25abJtA6RtB30VrCuTrLFBqVaSd3tO1DYlZZ2EWik6w+90Rl03+8uanHYM9CrKD8c2sT4+TRVucyyDR8C+eMRi1UKvDle3tObFHU2U2k9H57+IdOKyO1J0HeHm6vmzU7f+2OEbfTtUDJrVc1qbpZgJAOAOCVtn\\\\\\\/KLFhLuMZ7FxKRjNHGYZcPTaanxjQGCO+i28iFV+E15h\\\\\\\/uZEjwoMvTLFEuReCqA6KyViEhrUzwOEX7wh+9nhhyDYy+fAzchWGyQLXCu2lMUkcaQ4wb79PLvfAGD5budlVanbiiIcs2E9N0Z41zXW0kEz1Ml+Anv8uQCjU2CLdEYWG\\\\\\\/By9fdK1WEYYUWQiKT\\\\\\\/ld8dOCrpAKi644Pt\\\\\\\/XaGP4ld4CbZ5SlbeP2ub8sZCHyGIY4diMDgFqhXAcn3PenbpYS651br5I8iCIOcNFuqTE+8OizdVCox2ukbp+fmP+P41FM+l5w0LtoXjiwspRhuiDTUhQ374ycwHqgvv8Wjn2XbepGqkXqX395m2oIviCpCmMQ0l7Sfe8iMlpAZLP54MzgAh4ojosLtoY2AUzEQSKi4yiF2KaN09cYvvVjI2N3KS+l20pC3Zd8tYg329cjnVrUsT1thExjpnraPKZhdNpYBtWKjumt+vco6LgcGdYwi2krAwGyJlKP4Eb5hXxwTNsqbQtIV\\\\\\\/Q1F8K2\\\\\\\/FrVmUSdVPHfXDTeTgIOWobT9z8EkupCvubxWn0r8t7fdtVdlih51C1LWhuROdHKvDlxDyNUIsNORlcdUiBUwQFh2DlaEeU7UD+NdT6NggUqLhHAX7\\\\\\\/kUFK1TgLTi4HyGweGGpmpKYatVBo2HqrQJh8Wkd2WUyNlX8qdv0kX4cWlnuURYtgLMLStzZI5839v3m4YYO7W7Qfq\\\\\\\/jtLPZ+mc6vGRoAkxd1VZz\\\\\\\/ml4jkDacXJ9kc\\\\\\\/wzPD4OfSU5gXwVy\\\\\\\/K7mMCPBlDauRIa+HpuPOXjov05ypUK7+WKZSqUGKDWRnv20j7r0+efszWBvR2OqE3PibrPZIP314+E10aQ7rpxPHU3aYuh1jdTtFh6RJs4Uqtpt7f8PdaRdwcH4DkZJW8EO9mIPVd1ll0gr1wzMKaJDH0ICK2+XGswMWUDD\\\\\\\/p2heLS3H1xs3NojbF+RV+Rdky3s5+3dPxSu0T0eAwgvQ0G4ltShHrxGI5VHFLrQc2232XyxwnvtpuZwRxb7U0uAW6V0sMINaPYjmxS7izJdAkveFhi9t+6WO8KJfma\\\\\\\/fRuWHEh+FTEvWiNeh7Z5PhSb+vIT57hIJ7cYyr\\\\\\\/WvTbbhfddnXBKV8fV8C\\\\\\\/qsoV33FnnT+lWgGlJbNfRgLkbREwE1hc6Bq6AosVgsUlmYB4L1yyZIFSSQCJe+Rw8\\\\\\\/b\\\\\\\/zpSDaHW+aFKBg5r6AV+zFaY7BhoJ+fJ9wEAyuliWkKI1cKb3sW29uxsiptgQsvrYjXbXNxJEo7WGdVE5s1eono8CRlB0uMVULOCyA1E3DQaoqZ6Jqmm7\\\\\\\/4TNwEsiz9vdk52JozNEGfg+UBAUQny\\\\\\\/59sY4fymWIzX51XMey1SXBMMfpqwvgG6gd70UWharvbDKFR8rgu1DQy2JYh3zr6yw5r521FPMKx0FW3leKuU2aRzr+vC88Lzx19djL1fjY77W9ljcwqrW4RN7oAmqndbMWj3HSb+dV3hxlwKb9JpNsOX2GyXJfHZW7LOyuU3tmKjvRXcVQo15K+WPDnvzobN8wlB9V3VwZYNTsHoPRxqVmOAhPituoZTLm97lfN8ZP2IfOG9KiPkE6dhB5jKWmkgLc\\\\\\\/2FJEaR\\\\\\\/Ot2nIXXVbXtWL8U9mbIYHLfIgvYyHvI40rxxrGS7ue3HDgv9TJUtk5B0bB+ufAgAhRjXIBBl2wK8z+raGyLDZ\\\\\\\/lazbXcBQIMkSfK9evsEvZimeuyovmoK\\\\\\\/q+2RIxcON5jhw9b+T0YAVbI+Lq7VxpH2dRZjVWuxG+0JCdwbfC3NlIpODRTJhnO7qFREZytHV+sRarNHtcsvTawSx1XMqRFO9bRqwq\\\\\\\/eqj4Q1J47HyoP+g\\\\\\\/8xr4oRlhsO23Kccr5kfitbr7zLvolAvZ5ug1qzV+zbxD1XMzfhJSjMaB+x\\\\\\\/LKyvCzftmPdgmhCpmRM4An23BUm2drkz\\\\\\\/oejsVDQi\\\\\\\/858DwHVXyW\\\\\\\/DirxHSijz4Z37QX+YOniuV7SoSlxxbQrE5hSXfS36nzWzopZ75CJ0HExg9w8AXDWH8LZnHuGPtUdXGErBMHUdngNJBfv+cFsxiY82dT\\\\\\\/BiMKYJ4bGQBbfc1\\\\\\\/F5\\\\\\\/f2wiVal1Uxb0xASCdcjtcg7hFkPHs62fDEym2MayILUTOgcR5DVnu8KP6TBaHnuVbrVFdwz3pfGZHm1\\\\\\\/gnL3PkmQhrTGoSLWAfrrS+YDnPdKftzxwyy69dr14YcYaFRjzIRP4JW6EV9ybUlK7+gVB\\\\\\\/NJK3upFKxSIXFIDra53ZYz4A9iVbY\\\\\\\/UC8P1mtvXZ7h1E3DaBdnU3tUOeqnmgyjGi+HRhDsZ8NigQz577re4mRUyzFwx8dmPJ6L+bh\\\\\\\/cXOCeE+QJqJCmL8LnRMZxp7lw5k29P8mHzgjvYcAO6inNEeT4Pkv2qgeMNH7HYNdxqzvXN3Xi+23YofsOPZlEiEdvA8BEm\\\\\\\/WWUyojlTQSctNtnAbyfZxk+Qfz8FdIU23QpnqY0zQiwQp4\\\\\\\/l0s5i7PgZby4v6YYYxQrztVUkI\\\\\\\/mpnsjZFf9NWM+YWrqwbrLj1fsWgWG\\\\\\\/EXjxvky+stVcD\\\\\\\/XT1T7\\\\\\\/i5emPz97AhfxSvSo+Bv4ugaQom8BYZp8ekdULqHJNbO9BNkzxLydJGYI6zOr1uQt6qf87Ia0UKlS7Ydg+FHSTNBqMSksRs7pJrp0+XZZuzAI7iF04iqYgUXmAwWrQY26Env5dcvGFnXxZPlOdAAQO84JFGpzvOvuH8crF2dgQnBgG0RBZPqdMVHyFd+cnJ1OOErvBQX23t9HVxoE1QNzjh555hlAVogouM8UnLyEqJ9TvwtSHIPNjSDN9Kf3\\\\\\\/78OCET1Cr0tNs+XUKdpbh+FD9u9SKVR8PQ8p6PMHQCX5NX7p5d4gEcXA4mJNBQ1fPkQyKoGsdhkJad8imy9WdVHEbW8QwYIrIoSh5Ts7G3yjGfM\\\\\\\/UrWLOnL0786gPS2MwU55CfBBM1hmP17EGh87DKwdDD37MtQpJ8sgW+R38N4f\\\\\\\/9oKtgJ\\\\\\\/2eyKTZjHBB3Kl2k7L9ipXR+UkN\\\\\\\/BtFStD\\\\\\\/w86xueIXSgXCw7ytMTQkwM41aDGfBBwfEvGGM4C5Rc50W6VGM3SQowOfHtltvHf\\\\\\\/EY5m7l0KgaUaIZqEJnprCWjPBl3DE7MK6i62666lv6HEIa40wqizFtFoze+NCieg0oKvfXxL2JnAiRMSRberbSQZ0cSUalHCJUqlNKyI+26GRIaS8\\\\\\\/FhCtvtFyHLRkMAu+uTKaQfgK\\\\\\\/OMCUwmZRTVQLzsrpG+rsbJSTzapyGFKhjIbIqB169giwlWJf3eKXkmQKFJhXOkrgWRWM09Wd3G+LZD5JwJ3QRjxs1eThgI2QGZATqYDr8W95FnV\\\\\\\/ob7c2D5307bJ24JE0UXvpDOP29WfLfVKAhR8GX5XNMmlQxGCYRcu7swSijgr0iaaFMl\\\\\\\/JtxGJxtnKISF86tRzA2gR3dW+qFeVUSdWFCWtYmrMlSLx18ZO9MI8IoDrDwx6ERJNEPt2q8GNbCIHtVEGLJwVns2euBOFDIcmFVmOq9Rh9rf3sxpTO9IkWgr6oksqQF5HvDSn86jwlNUn2F0qXSO92Tp5LObOoIebxzxOrHMBTsRaXSaKMA6EWkKTyia8cWwFcx4PTex7yJ61QPWThKWOFxfz6rEl\\\\\\\/+MUm4KpQtLUE3CYs2btmW+liXuX+Mshoa8G6TEyYP6vlBey0P\\\\\\\/XZezY8a2tw\\\\\\\/fUCVV4Z\\\\\\\/USGSgSgidabwVqyG7F2hgv0N0JbxdbJfK6AQdBkbCaS2usdHf\\\\\\\/+YN9Cd06KqpqUNOjajRnkL62LpVwEPIoohaXoyPG+y9dI4jb0aPT74Xh3AndIdRprvMBapAC3JO7I1ZIq7ZCVVNH\\\\\\\/w7RvLTeQ8IzJQPCXbyL7Kg60RBuwM7LnxfbVPaGbEnFBqYyHwuE\\\\\\\/8XYhV1MWnkF4wU3LOLwQJ6dPfEXeERg\\\\\\\/AjUU2O\\\\\\\/mfSh6EKBERumFWCwjItadufbpGjTWuogbMYcHHv08ZmdbloU7AckgW9ZdOv\\\\\\\/TAIE1bxCnoj0dbZrTIvOA4wSqdhEhAwdEtgPJXjAmNnFjdgC0igcBtooYxhZCW2Fz\\\\\\\/SdBcUH08oqfc1cLofDGR64gs42Zz2sU8XIHkrfNAE2N4Oqs9U4GNjjoonrL1x6TfixROVS9onLDOFyFEYZ9btVsLxl35JYKNj5SjA8+C71Sws8nw6AYN7Y\\\\\\\/c9MDfSIyJeC6W1YqiirrUkMDEr8OcUFE0RlI4QicUPnV\\\\\\\/JAqy8jFBmoLGdlGscREroQj+tv076Rf20maxKugatDPqOKHkHO+9TuU0S4o2OF9oaO11BHCFg8mmWYyG2Tnk+tAkLuBHlfocUqarShVXMKc0w\\\\\\\/edMfSalRcGkJSWldZzUPD8OaHT5M5ng8VFC8pYSo1stzkzrB\\\\\\\/1uw\\\\\\\/3hAnLAkDiGVpkHvYE1YBLSNW66ZJOZuHZyFFJZt0f6ZcR5xQQBIdUzVGgX6MNLN4ikcmXFmjm5HAmqvSoSBG2nd5KB7BDrsysAdXRbuG3qauw9XqVb4Lv0BnuldRGT\\\\\\\/zFZnuTAjqTCTaQwl5LcU++nh7nQS8mTzTH3etihfLUsaqEiLyS4nXysW5V8R1L6NthG0lniKq8rN+A7AyFhjl4e6fWVJDDtnfiSPQ2SFTx2hbYDr6DrFa1XWLRHIUt3laZ376R1Gmeq6B10mXswYKP1VsUyyhRI5s2QoK2cONihmv8nCIWLbz0pul09atJKjy8QcewGAEkOIcgcloHFYhdaNwBy9viEFI2MLsAEdMSVAPO5\\\\\\\/2OqN1\\\\\\\/O5rTvZG4TNknvWDyaKimB8MZZzUqp26s0r+SWo5I\\\\\\\/K4silJjwYrkpzSBi9C0Zaa\\\\\\\/wsASmPU0nIY98exqdmKc2jGglZkEkxhdaI6qB\\\\\\\/F4+MET9lowm+q4d55J0jStX2TFVAHRO11wKQGu0sfK5Oy68euJrYkDbUXV47cICWAoyPhx4SJfbeiCfl7TVoBlKuxoTFKJYP8YzfBvOJlLQ+5sESx55UtIXWFffLv4D9zCqOeOufMhrUBW\\\\\\\/7Yk2roZ4YzWCFwpc9b5ioTr2gSyb0uxvUi++aKeJoFbt3S+oyO6yiufaPzu992p935Sjb7RuGUZVznF5LQkXsVrq7pOXiPWYCgLyfIy3DdlUeKOQIN7bTx70wj3slNF6SgW8v6lrRy2QsX+GVu\\\\\\\/vxs7GKzy8s8KFkdhfKTQPA+vvtkvKtgJOLuFAs9Z89dFyivC7XqVAr5QDdiKJJ3qL\\\\\\\/6hwMzyQ60hwviWBfbFsA\\\\\\\/Z8bU4oMBrt7yrpGZ0ViYTGg7Ps0QwLyQau43f5B8YgQC8ezoOCO+vnnupsw2Ulx3PCxe7LfVQZZjiJJiRq0FaDWaRWY+fAUEaaQm2UU5ei9k4K0OFDvDAHv3zYCl+GUBOO\\\\\\\/aX7Ud7QohUHQdrbZCMSsOHmn4bTXN7h9sbHFlYosbqEr\\\\\\\/2UvEaTWdZbGS\\\\\\\/vpUr2XYaDqgFyJirCV10CPpFS1hRhO8NRKDB4Z+KyQizp3pqiSIOyj1nu5OVqlbyDH+ulR8v02yKMZvaCUzHf\\\\\\\/VWod+Wm8BpcCNuDdB8wGSwfz9mJCyj\\\\\\\/nnC7L+3BarD2o+Ml0rbQESp2+QUZy9vXt3S0awIUPcj4WMDE2lq45drfTb9aHe4fe3Lsa4u6c9ZIWVkrKJMznNLAbtYyAeU2sBpWZ3svLfTXkrYMVQC9sc7TrqDz8dVNAupbkXtqEAYN2W2DIDox0Yi\\\\\\\/2YRWGJpp8Enkg\\\\\\\/WtxXM5KvgHNVPoa5pt2Q9FIAnT\\\\\\\/t+NeV1tgnsm5VjWKVsHClydOx56xDnbmdkOSVaKqCuFtZB21\\\\\\\/MplXZWqIzAftfrtq7aE+lF1MfF8dopJeRU8hGzJZJ4aybyZFuLeLM3ezO7s5YCBJWXAidohQ7rFiflPxLeqDbTrJsMGG+ylQ+jUE57sYVtP+KABvT+x9RYCFkMIn1hEJOMDQf6ZzkT2LWQPMl+HZJy6wevs2j8HLqV5nqR4MdzsBy2fOWfVXWL5pQWu3MhUK2jkcgqjJKt3btbTXk42kFKtpiI9Qs+Tlt8rD0qpQDimrgb5aSR++Vr9duXx0\\\\\\\/LQJW6w+xK2djK4zsSXV1jBcOVdvBnMSJBaPsmU2iFyHDYpaL6hSYzvRoYNqvd1Tsm0qKxHuvnDQfQxG6lYeJJcoFoqpteXQYkQMTYQWQJnT0FkhAGYkEPVnlKVxKSsi7hBiQACmSfGpNiHhrCoTUxxfttC2GwvR4n\\\\\\\/eZH25dS6UQVXx1BgdrjT6Pr8pdhvnhn5cmCOy8waWh7ofPqqede529D5Jomn6r2v0t7No\\\\\\\/d4e4cPXJAdyrRr9ndu1NAEhMGZowY7UPELuotiw35qslrdqDFgQCl3Rp6QsTmdoVbZER3W\\\\\\\/y+AC+oDa5dVDDQsUpWKe0Pb2RRzkpkTPnl\\\\\\\/2C5KusmDSI+D46vg4PoAxCrj1Wgwz2JxjToMT9GPttwxlS0f0MV\\\\\\\/B2dUY0\\\\\\\/FUrtaq+HxYUwvLYTVWHykK\\\\\\\/zrXlBKRhPWIFNR8gQXd2z6Pf0iehWCEqbIwteAefVOvCNm2iyUEL5IHreOM0WygFWMSMx+EXGLe\\\\\\\/EoMSSaHrZ8omFHHU57Ahx055hicm8eXlNQpFQe2oE+5bxV0706omyBif4dCBhCnvo4u4sjBMde6+8PAwcYXCs12q0ldw56k3KQwwZTHqELSB\\\\\\\/34t4lFSXH3igWmYmuKuaZPMugM\\\\\\\/7zWiUnJz7G\\\\\\\/pb\\\\\\\/Ro84rJbbVuuITKGgY2V20r0NpT4fmXPncLO+79fYx\\\\\\\/5h9WDRHMOIn6gQ42pO2p+ZEQfqKVTuJkbbLKCTpsrJSQm5dtoT13a2eJKs0OdR5ASZh9BERJGP3noqzvK84+7sMDktrG0sGkLY2oqFzlxl+w2Lap4EXTWQvqVZ4OkCVP2kaRbyXESqD\\\\\\\/\\\\\\\/vEwLOE5Ljzk5zyQCj73pBKaYmXPKcXfr\\\\\\\/8wiO4hX324FV7XUkrb5CG9iWbJAXBIOcTF6m2XFpxqa\\\\\\\/tYiwyNbM2wg\\\\\\\/\\\\\\\/P\\\\\\\/jDkaxJ\\\\\\\/iskAr496KEQo7iR8o5c2UZ1ib7D6Ob\\\\\\\/TCb0qLbwsqSFdVRIt8LQ55V4O7hD3\\\\\\\/Aj0d7uu0JVCLwYNkLZMk621NBliljS9vW0Kyv3vybnx4un3NM0Ww7KM8IftrHYU69MgL8c2h\\\\\\\/g8RoO8uRNUWNfU\\\\\\\/rS1FY0aoJQDj4d4VAonTnsqYgDn4arFi+y5iZq1SQzxCMhpCwCknxrj4ZxudSUdmwWPfWVYtvYlylJ57oBCsOr2bBDccwe2p\\\\\\\/wyMSF+S248b7kad+SmWaU\\\\\\\/ZQw6mvK\\\\\\\/jb\\\\\\\/TsOP5ELsH2ivu22lTJvilELvRGmauopwObvi+2TWfB7ZslSNHYyOeoeBkZOEDZhOD1AJaBhY\\\\\\\/lb3hkiuYIfYJjFLEPHtsNLZQsDbBbH048qn8h79keZz5VcsQBXsA1RxEwqNhtbbnURBavHAxo2vr4QlwdHjhEfeM4qjrYesKg7paRTednQ86zovqwE43Rd56kSIcp2zr5dWVkWlH8gv7Q3Gw+Ywrmf62bj+VWcq3X5Wg9DUUdd3SNrl1oyfFkH0+4oM1cyHCog+b5SxE+MWXsPzj8QXRlz9VenhPTxWel48VS81q\\\\\\\/8jqJYUpZNj6axN3KUcst1bF3Yw4hfqTvR3k+v+X9mclygOAZ6bcr4JVDcAwQQTbERD1XKoZYVtIW38f+X78c5NALgXdIwiuhuexixY9mqKG7RAXVPtIMlwwxVnVHVg+FOmELSTfjk8UmXnInPSV9zx6yciiBa\\\\\\\/qvpe334S3erb\\\\\\\/JOzoVHWJAf4oPASwtjXM7WbAsN7i2gaSwfojJi1sgut7lkStIXwpb89ZwIAkDspxarrx2uY111T0yi+6WNyq4JJgdM8a0Dfx8Oer17AHluX5+U0RxL19mGWZK5P2wN2kzcarbxhFRO6vrw2Tm44XEvOnrmBoE\\\\\\\/OcJu4mQK1easocxBqBu+FKYfwoqcYU7u5kNX5AwOcqe7Ld8AJ9mCaUPOov9FTT0MNGN08Cng\\\\\\\/gYITSwHaoldS\\\\\\\/5poO\\\\\\\/n+rtsWywUvY6CAjALCAn8h4fDJdtcZ3YjfKgTnTb8tBXnEjJocg22UHKZ52Oea6ST2gtLwW+i7xvoVH9Yw5WcxKtK0okiqyQe0ZvG2X1tAtrFUr+H4ppr8M\\\\\\\/JKMinClB01j5k8oyvwLB2KJYb+KiHwa1FBOXXbTBlTfnlL0rhR1Cox7ERngbgm73xMPSYGEfxV4Zvm2ehuL6vevAlh4Zo4YD9uYZloFXljZT6KT8gycP6pTwO3ZLrclI0qkJKyCSyA2g4CB2Snr3BnnHUpAP+Tqi2LACUXs4+h7yH73qZ7ETae95RgV+h3rs9L+yK3xmdWtv02yz+j654uoOowazDyeEuJZahGj8mtfAgmbotREq+QMIVNks9e8cRflJK8LrxxElDH5Y+5YxbrQK5tvM2EeSFwf5twetbhoww6m3WQB4rIrSDn+XuxJTCB6C\\\\\\\/Fg+eojPvEYkkd85a6uRPfA3Ml+3eeikchrfPsprWywFux+meTo6Wvqn0s+2DpIcLBPHgGbkmn1d4357hsg7ly8pd7cjeY\\\\\\\/RyHO+OceHP5d9A5be17Pl88iANhICvPhUYUQ6uRiYHer5fK2pFJoU0qGdPhdZB+socryoyJEqQkNfb6xeBpp7UaroY+1eOfm9xLdQBZOM+NMuXcyFBPf67lcFlbgsRfcKGgZ1ktSAqtb4F8e0vbde+yEdRw5on\\\\\\\/6fTsmZEt8sxj+HhnNMji2cyue2G3NsT6NLqEbdf3LIsXOFT\\\\\\\/vcXnLrWRbK677VNBFOqIP7PrCjfhtA5Yz2nbq\\\\\\\/TcyZNCQ2v2KVk97NgJ32Zjis\\\\\\\/GPEHCpezU2\\\\\\\/1XZZaz2pIwAsh3jPId7YOz8Cusuc78omFL8TGHf8hgINAjKFXzTQU+PeEfEz4Oe\\\\\\\/C5JoZfVmArkhOEBZDwBB+xRD5qO9TpE1uuhf+0lXmy69YQXVo+Ph56X+tPJRSuJ+LaZ8WV8UCDB6Y+DujvbUn58JTEfb1qda9rFnWug03e5AKXMr3aUG\\\\\\\/H8qn3HRylOJkV+4GCpCbOWLpZtOhYEaFX\\\\\\\/7Cytj1i6+YeAcOfLvwcXmm3lwv6Dr5SoDzSmWvEVpvoDyGvKZQDBcW6JbHOJ0OYN8+7UTXJOKseo2DxAxCkK0Uq0PHC\\\\\\\/yPiCklGGdMhBuArfqji5M+n8kLv6e3w6neemphrg6xrnJMhxFgBcpA206tC8F88PosGkC969O9iuik8ENYKJKrsmYhVqag6tstLf7UqK4o6ad\\\\\\\/dTAe3iBr5U\\\\\\\/xbTkWMZ+UCWbTR\\\\\\\/nUc0JmaeFPjelY\\\\\\\/XJn8gxwkwyx3Iq\\\\\\\/MAVrxL1OnisruOQ0ONJ3wzDNE7BHqTkE2NLTP6QfjH3w\\\\\\\/3ToLn6cyznOtJptMbT2uAPPQoHd\\\\\\\/nYNvddQyc7u9MN\\\\\\\/Zf7VN4BAhk8LOo46LuJTngpx2dGBTgvzNqfwes2B\\\\\\\/YnLEr7jcOgOClkhHWv5WYKeIl+AUxG8kW10VdBvkZGnJFQ1A5CAKhtZrF1T2YIWEi6835fKjDGG55CqpsiazCyf7+osATG4d3SzNOZ3n7ZAjzqxLNUh2GGwZ8YcxXfpQsVG0vj\\\\\\\/nCSbH7msLA9u0UqU4bE3kfwS5qwnIOMHp2zfD04bZ8b10NzOTkLOkP1W4lu7+3qAwPQrwWCfK9LCjLZiW2UkN9PadQ0ORZRTBMExVxmQVxSQK5ClKtgN59iQVWeuYdEB7ag1vpoaw9Kxrm+00v3bGYt+xH9rJcQKj0\\\\\\\/Wt5sSxeKmoJPYOQQqAbnTeP+oD5NHH3n4dWvXji+T1Rko5oVXbtqd+t+hu+bgndRYxDSYxSuTQHHF82H809sTYFZYSeNQzGp6fRunRKoIz6kGPF9yz6EQ2DFqMM9OCIcBPjnFRHG8bFLZcEyI9My8a1nKO92SzNqqv4vB7MXqKEtNm0A4my3fxofzFMV1wHGeX+VZcprvwRtutsy9TvgWP5RbTlZrAdfB9KhGxU0wcuV8QpbgEqAjE32S2gwGu3DJtZts9+Tuokrr96GrRitAJFTN188v7PHdu5Wh5TwxL1JNb4d7UmzKw7K82QW0VJvHnBHlVUU1IyA28YLQe7IM3J\\\\\\\/Z8DVZ7+AQEdi7b9Ca9qPQgRFgsVmKnawkLSq\\\\\\\/HxsaHjKrTEBwscDciUz9nmf99Uc92VhdtQQYXLw\\\\\\\/tthX25X9T43dvs8bIwad51lzabMb249gOoFcWPLei2Z2hEK4jBMEXmoLpoZybqIpN+xUwzfwM8rdFN1qUL5egcx53h4h3swOLYpTmSKD\\\\\\\/9iJFIdLksae2XJgYmI8jWON+IB76285SHOMxYAJ1KIdyJvlvi44+l2vMvse2qe\\\\\\\/97HUhtHme2TTJJvcB9Is89jhxaGggXT1mqBm8mblaNg8\\\\\\\/rOMj6ME2LJgbHFMHQSBt7KPsLBfbNpKmGUQjSwUgT3ZKFqFvxH5XkXsXdhB5By3s\\\\\\\/aCgqBITVapBHx4t7Uv18z8CJI08Q+AG6tNBOe4iC+qNP9sCVG0zKRyLNiinOo0aEdeEN73EOJoAw9+kRYYLLUWIi8ecIu3jFph0I\\\\\\\/35Ch9asFPF331YVxO84Y8zgCrpEqvwiOQC3js2slWsQXfgV7U94FM63vWlh8VayLND2mXkvTkpPdcgULsMVdAIps0khQka+oTfouvvVrUVlZesGGdLmEkr1OGLVsspsDK8gmHr7MIoES9rWVCvz3PlXQ6DmszW+sSmGRCaLNUqxkAvcbU9TfX50i7b\\\\\\\/XKE6zhbWx3OOUk2vFX+Whi1rJ\\\\\\\/wMOtDUrZcX7lrD5U5z2N6wa5+2bclCooxxo8Xg36moam94I9FTeoA0RKW0mLuIM3Vo1Ez6ogIC81kp0+NXyQbpNIa+ZgjkGS8D16QXFGqP92mFw6oR+8Ag\\\\\\\/sPLPZ8DZMuzlHdGGE9vFwYrW80BjqF6kGgx58MI+liFEeYtkNth3ATC+jNnGbtvrSaZ\\\\\\\/es3LAwVVdnzbiwc\\\\\\\/vVTH3+GfyFP9jNLLbm0ocSYG4MofHE2f3J7nbeSNvtMO6\\\\\\\/3e+5WhI8O7Yv7YhUvi7Stjk7QufSl67FA+mDDoCvB3qfyjDN9ZeMfK87FWyfretgTWtxV5eaTN\\\\\\\/0H4\\\\\\\/F5M+Db5ZCo7AUpeJoYc28VRfkT2KyEDDsx4BhCzFHVwC3eOeUNBscOnNQou\\\\\\\/AlLNijkO284wXqm1wsHlxu9ffyFU177f8Falr8duYyiku9LtEvL8XPYyc6FKi\\\\\\\/hug23HYQfiGuArAuF4G2kl7WNwvRFgzHY4lN\\\\\\\/RsVWyzfJpw69Q9z7bDkuCQiZ9C6zP0YjoIA7TFD010aoRmhiV2IfOmq0KRRP+4yaEvOyzXHae9L+3LsV\\\\\\\/rO6Ld3BjV3IRaZzNwzFXYfeMJPrTxYLOC0c1Q96lBViTG9T4AYb51X35cwGb2LPZuloMXnSMxMq3WQ9GreKSg1ETcCo43Of4U6EhBL0ArEn5bq48opfKW2+64J4MqJue6VVtTg6rql+VTOrtAi2zp08wX+clSrjZWyVAqEl0iYwwEW6SsTi4rVUXgv6D7Jq5\\\\\\\/XN8xaXpcn\\\\\\\/aMFFT+R\\\\\\\/dwIRLRDw54B21byAbUeo026pUurET0RwJk1cQd58ZyPlUkf2Ep7vz6SDoC8TklqwAJTw\\\\\\\/90btG46FYUnlJyc9U2bLVp2cs1zUwqmvwjJ+G0HEM\\\\\\\/59dx8yESZ9pZAINy\\\\\\\/ZmcgOFfs9VaTrSR5OYnpgvkYzivPATgUryOM6e14CHwLpNamCOttntjMhgce0Fo327GEJ84ZAMPJiQArZhg90MGmI9dpp7s4EepcWaKDmVXW5YFbfOAcySKUq6xbO7BDEAVAohXWE+8ecRyvLGcRhXwItBn\\\\\\\/TSN0MmDbHc6NIEg5ZvGLlzHNEkxG+QqYLukNK8MkFm28SxCQSTv97psgS3mFKkjWjUDA9qDPMnaLIDAYXaM2PY6pa0k7qDRScaSj3+SOY4kdHPdtJkdOGzHi9k65d9zrydXg46JTbia83FrlatG0ogLjC02+i1SyorqXeORWExZCrZc2YMdmrxWDtR5sBzBKJVKTzlmycN+CY2zImIcDPmnhUrEcmzrcfsioj2EuiW5UBPmZCxGh\\\\\\\/JIiJrQJQigN5Z\\\\\\\/R+gnWWTvbn0R8gQ5FFlngMAuphRehhRQrtiKvOk34RvLJ+BygbS\\\\\\\/I3yd9jVNV+PuiVa+Z9xDvQZP11zJ\\\\\\\/fENKNkGcf8TUSli+LfY2tORhcYoYBgAI+2epTaScwY6ID1QedGLX4zUi6lCbMiyjafRSK2gZLzutNqnJnGbJao7mICBfwyWpMAtA9U3lc7\\\\\\\/+ySZdyhXuoEEmmcrcBgMWi8I8tbdJGSMuG8Bnhx3vIzZK\\\\\\\/hwFiIfIbcDu\\\\\\\/MxyNVVaeaFKZXQaCIdhCJq5WdCEzFXoPtgnw64bY\\\\\\\/Ar5C6gQ8eg3AC\\\\\\\/XgiZGt61hCRHKZR+f1djq6j9iu7tiJlUGSoM4NKwxDeXpw9Db7BweP4WdBCT977BqPjnW33L8uavNlnS2rrdNsZw29XVASC8P0wKdTsqb2D14LKsybtTRs9Yv0FsG+VmEfjmPNs0mp5kcxjLi14Dpg2V2HFHji028DmFT25mKGuXVi+NsKwfypZ9yHWl2Fptbj+P6\\\\\\\/T9JB0mueZPiS\\\\\\\/CkxbAOAiBhzgxv1Shwkv1QvZfmW+6Q9yVy5XR7pFAXvnzKH8QNE6tQi4ubN2Xb7mNmWezeLgiKNWsKocnhop9DLzQEjbipRoU2X8vlXB5CHpb1iR0zJucMErReYsHJD2TuE+OVm1Op6F36aIy3wWcbGBt\\\\\\\/uhKZIQxgutj2n6JklygdlGhFSwn\\\\\\\/89oHP9NjEK6XyCAmIL5IgYf7PiqPWLsmfUgrUOG0ARKcdkQDeP94NeDN7SUtgUTQ4kXp2dVSA1HoJWxXU+xOY8N7DL06Vv24y\\\\\\\/5ol0uktQndOVQnawZRGl5J6ME+KvHJNyDKQEVc\\\\\\\/JgYkeQgqMU5GaAigCvQcbRsnabytnipl1\\\\\\\/a8+3jaTUR1i2\\\\\\\/ES\\\\\\\/VcKGhLXTI6XsJDBFTnKS1\\\\\\\/CAcxnd3qem5stpnoYy1tmHJ5VTgJVWMnAmQ1RujRANziMefzAmc5Kg3XKoWNFy913hzAPoZnO2a+wGmeDztQ5uVjhchrabyfTuBOihBhV2YDkCmO7fGWDSc70SP9ENrrNX6KZaDBTlMGfymSAWc+I+L4jgTXkQW8p1YA6kl1TZk+ziwtdN6TUXYx2d01hZTiBQijzGq1gfo7WANKVpBNgMpRaHUFhkTMJtiD8zHkKwdueTqVSe6AH1nR31z08q80iy08CHgapYC97mRo7W2bSMvnoGvMCj4vYxJOpHW4HAIzqB380sqdjh\\\\\\\/RUaH2cGhEB5b5NX1jKtCmYXO3BPCgMinPBbKJPEXtOuPd7582YVrJv4KT61c\\\\\\\/ycUKGtSF4Aj51s\\\\\\\/wy8Jiy0fZc2\\\\\\\/Jc1GQZVXx7D06z\\\\\\\/j4mcPBby0NBFGIXG3Kg4V4nXHUef\\\\\\\/+YHe+chz11O4u1vaZcfGlF+Wzy8pGJEKbddu0+jlhU93y5fggR9kLW1bzZ9KAPH2sCDvCvB4A32hLqfpzRf9eUGJENrTLpB\\\\\\\/UZxO+sr1D01Wh5eQDPONCgzURW2f5xuA9ulZW0Kpo5EMK94JfRcGM\\\\\\\/my8Xk8QFLKOXS7IcqzvhzhzbhirXXpuROLQN1nzAtp62klTwV6i32jMVYzzrTS\\\\\\\/YmOPA6zd6Z8OQzNHS4SjWXBRKIle91MPQn2dpBQe1CZX2q3aaLdnZWSbBXj0UatSBM4HKg5Md5dmGEhwlo0mus5V6OZ4eqwnMvKWbZOPB7Pr6ADKQukOX5hBH6uD58weD6w6Nv2Z9ryxYup\\\\\\\/QEELn5mi1cHze7mXz7fJ6UHOVo0cQSnnPMA8IKoE11haeE0VRFMaqUngn6lfv0SQjeypvIbVWWkvy0bp5WbzLHLCmoKpcxi76Vpt7W9XN3I15jAFIxoLZsJ0PmqMPl\\\\\\\/WF5hFUs9dVNQFiW4OiYDONNll4vnj2+tUM1GJMeZVv6YC2Waxr\\\\\\\/z80QpLlFyeLQXckXC7OB3yagoFoTvSme+Kce3ywKdriTeIGqnNz\\\\\\\/Jd3omK4DEUZ3q6I3FV+H9d5wYz+po+w5gH7QX8H1cxOBjvt1+0GD7OP+BM4rstTCUEjPEOljfNMWvob0rslZxFpcj9tFAoga7IK\\\\\\\/H0lWxV8ZuxKLK\\\\\\\/yw6bNxCFFdRh2spVlr+Z474IeLI3lc7WBPDlb2ZzEB6AYqjCSUsetC4v+JofvGye6lvizs8ZC\\\\\\\/vlPoVfbiWL20VVRoN27G7jvcAaD251Juk4h9e7IBcJ\\\\\\\/hYaMwmTn9foGqzq9B++hyzJu6v7gfGSo3epXfL1CuRTiVouLom\\\\\\\/bSwvtCe+\\\\\\\/q4+OZ4X3c2YEB06n9PhBW\\\\\\\/qGD9TRwPD7rH5fUb+SHUJtxHiAlYTd+MX84IyrcVpN4d\\\\\\\/CLCyI9aVONWFtmZrSwrxjFbkClNzzmuxeaNA\\\\\\\/TFNAI9jyPIogHkGVatn2rfAnW2++M0Q39bO1I+10LdCIXsdVDkxElkMhePQzVoGiUxkYxGbNYz4vM21OvfqtiNCpuZavAzE7+rDS9qKEgHS8J898ncBwwLfTPogxCbdhMIAobRcy6sdPm7lTvwpRE0nqVNCS35yu1U2cvvuwfEEAP7sJaB+LtVnNdz7He1CT7ppFDsY85Qo+D46Xp5vKfEardTncxSQWzqSNnAAkww1MOVTZ5zd0B9nZEnxNe2a1nRk1J2DnSxW7e7cekZxnvCLIcQJPXZ3Fc8OFXGpLxWEFEx5k5AAX+Y00mkYgsGeR1ghMG9Cb63ntNAKRiw4h9H7KBh+b97DUbPrfaKMjBZG5QQF0yEB+2Sx\\\\\\\/PSWM9mAM6xWf8IeGJdPb6QucrqUHluioslBg9ZlmDU2mWNRgKcv4h9gqaiaXDR2\\\\\\\/S2cabeohv\\\\\\\/vxwsQ5nd6uKXUZjcewUQ4+GTE4+N8CfE8F\\\\\\\/vEiK5WrOQQMhOYbdSfqESeH4sr7+VmkmduCjqwPUfDkTmsfj8ReBpqzfL4uYP6ByBXu0lqCRg8gXlm2r+3Nomg8oFMKRRZuL7OL1gv3irsyZP1PBxa6gDP\\\\\\\/6+YoYxv3KYk8gbtjq14BB9SMs6bfzT1BOzRzGQK4zb4WJSM+S\\\\\\\/5GhI2VhDE5fA02SNuEwSmKXJhjUGw7X4Ud9EC3BRk7iNuB8qm7DcRoPojINn0ET7biYa6pCJqCAjRW\\\\\\\/4lyjC+8cZRMhBQvKbrrACcNSaGfHGhDpk2TP3XLm3N8OFAL5HJBJsLCBeJnFc4A\\\\\\\/AUe2KZLDMA4nZ58KhUoC525dRmGOYvUnzZamSToPFyJws+5ARUBkPhQx9la\\\\\\\/oYqlqGs5jfto\\\\\\\/ubCM8VAr5toNTGSIgpYKv37ZGnI7i5055Ech3r7JZbohDSqwOP1+hLDK5QBsmZBy3EEgd2ZCsFBQUtsfPRzBG23A6Avh\\\\\\\/WD007uFQ+iiYt51kv4urPGE7Kq9ZzIfqIHV0ZVcTT6WIz\\\\\\\/pmDIZJdidXdAlLNG66ZBQScYgOouSOSb8Otl2bVc6xXPX0PIVydw3xpBGwaZWzG9cs+lcTVhnvWdtkiYrupEdNINo3SnO\\\\\\\/AO5vL4ZpFm2uBq3mV7aG5PvrnLBAi9wLDR5PgPInYj7jJRgh5sov\\\\\\\/x++bDMZsWtFo03L1VttS+JlAM1xFD+bLh7\\\\\\\/WB26Vzv4HXofOBFXzpOjtESO2pMAmWYnTAe0zrZkqQtNvjUEMbdGXROOBj8aTIMatSDFCYfgJPIzmnVOSXY77TnQ8sSYAW8QSVruns0565RNrkTAbuG\\\\\\\/Z8YO67iuTTwxIYnXBJ4Rzhhktn\\\\\\\/WleekseKEXQYlLE+GxzaA3VaZ+39fK1NDD8QYrs8Y1\\\\\\\/IQtU30iNXm53xEMYP++ifWKhgjDS\\\\\\\/ga1X1z5stYB1XMgc1nPBWKDdGSNl0jNMQz7CDY6bqR08GY+\\\\\\\/yGt7HEl7UnwXaSfWBfeBZXDRzPyIcdphmkubFKXuENLKtm2QPYLeFkvQcSJwppwwQIU\\\\\\\/9uyw9iR5fvfkdxDjkyvAbIU62TSq8NNBwbEexxEZt2YBLHyccctU6MHMt\\\\\\\/L2b87C507gUV6VIzWMDuIJzlI0FfmBEp0mIZZqM0yGux\\\\\\\/XOjPy+FR\\\\\\\/y4JfoYW5RbcHixi9WeSSgN7e02LWDdKZ4cByEIllxmSL2vz\\\\\\\/\\\\\\\/sn\\\\\\\/MK6M2fOP0PQQzNDAkr11ZukZcx4D7qODhEF4gfg96lEdsnqjKvx4GoC\\\\\\\/6mFBgcI+i6VixJPZVUTKc9REVT+\\\\\\\/M7jcfFd+v31ZXh70EKvTQNsKrxhOtZvZVATcIPfkwKX0TrcR8I\\\\\\\/GcUYwzinPCoa3RESV42quLvwmrmjPI\\\\\\\/\\\\\\\/RdcpIqk4M17gUe5cBuYTVe+dWJartv7NodkHYRCFcTXWljVcjt120C0PAVLawlkXYfOMv\\\\\\\/ljbTIX8H28ZFK1qGtBjKzlFdXebbISh39IhFe\\\\\\\/eq5QGIepigLNdeXme+4rFNWRZos1NoIESX7l52mnFACJVqXy77U5XvNNQg\\\\\\\/YrOGgoMBt+K8Iv51fULpZMPqx7DpPRaHFB+A27KYV2Rib3T3auJdvWENEU\\\\\\\/GHKEyixiSibVoIb25rvoHoaoyrdHbVbVjvTILdYVaLsZCkVmzJ8tOB81\\\\\\\/iB0TXRDFz0jJphnzwu1xGl10KTB1ywgtfhI6Vvpl9utStNAXeUlluslL2R58q+Gsvwq++dACZKGxX7\\\\\\\/Zew5TwYV1y3G0MGJ8UK1vhBxi36Ot98ZuZyaac8sw7Oe+47xxAbZKGb3Reqe1pJUtqMkLQ2fFL2HnO+K6HNJzs6j+vxDZbukZBUfHenKeytAfmVK50nWakSbo+PFyiTQG6do7s2W3D2ZcKkEs5OPBbi\\\\\\\/m7Pd0+SztlbKPBQ7Dyq9\\\\\\\/oEvjriNPs+ZztdlTi5gYTmK1zv4iheyIFXMn9Wg08c5U34CRXfA6TvEVa0BQapJw8pgV+BfbYGg6TiZ3laaqxkF3N80TLZKz8FLZjVQm7g+hKEACOdxMCKOOnQ+g\\\\\\\/xWKO\\\\\\\/nWyFxS\\\\\\\/KZGsuA5KtmefZ07ZeAPy2wN0K89H7Eky4VB8xNgoHsdI0awlogWLrl11GJhRK5Dp5XnKe2h6ufdDX1omHwYd8rY3ic0zSXVbb3nn91i65SEmvJ7s\\\\\\\/D5EXz4W1ZdGQkuk2YXBR\\\\\\\/axgnp9Wj0t6QiG+JWj+pYFOz0D1VTUpGZcq2RgECDzHyDbjqFmK5OXdQth\\\\\\\/GTg2QQjZbw+RHZOlT3NCbsEAyRo\\\\\\\/egR58dFVpB9mog5QC6CIjxc+j1Stum5c9O62WSGuggAWBGtHZPsdLzYFeETOBd1GLVZhSoNzPPyNF8hr8L\\\\\\\/Pq9r\\\\\\\/S\\\\\\\/rVsNDZB0753Mwrz8DP1+d8SFHNX1CHaLSEVQ6AgqmuMFUqJn3fhQBXFpw3HaYBMeJiA1xZ0Hw4BESmO1tr3yLQLdIgv\\\\\\\/ycPW3PD6rN6qAoH8kLcI5O2QbNrhV7OViq\\\\\\\/yvcMcXbeVl5pqS5MRcX2RNAd0A7nigVAJ3xwWvhBjoYkndXiqa4Ev98\\\\\\\/Mz1zMegRovtMUydKb8QHl5ak1GQq4kAAl8sNVa21TX5D9PLwL1uB\\\\\\\/W87gShgRQnNMXTeiojDkXLQQd3xSbTvXMpt8X+OxAouyW8DrwueyIoCcTgRy1b0zcZt8G8AExl1vd8xRlCEJZv4i13ae00Y1IOg4p5GcDs3WkY9JJD2AYlVDSFcTwZr6k32074jPrHarixyvJoh\\\\\\\/YsXP5PjAE0Arz8kDF8rrJbjqrH8gee6LaqpAmDLmgpkOvkZcL9bW89vuydHsTFsnNmsX\\\\\\\/e7Q9+avYgQJ\\\\\\\/ZkGeGspGr\\\\\\\/8HJDmW956+JLaiBo4bF7lYP1JIOZ+yfmoRaS7vYkK05eIDXDGjYUf3V8EtOxjUG\\\\\\\/vWkTEgvF83lk8ckNQCYWwfRH+iC+Y1tW0expCRgEjj8j++rrkq7d\\\\\\\/KvgamQJrAB0DiVFok7qrpLu83BGkzjLcbTnNORbt\\\\\\\/FSdC6CRkrVPgZXmJCsVwccciyX+NfPmnuN44yT85AWVDO3B+x1P9aKyybWkEWTnWxfaDPrOj+bK+7xIrA4hATYNsp4k06ItKAmJc1urVxMHZ3MXPob1OyW9PDfL3k4\\\\\\\/nP1ZXZzf8lLKZ10H9ASZW58KrbikLmA5vZfGVP992KAVu2PkAe7m\\\\\\\/zbji51RhrCyWr7gDIeYtqe9fSESmrnbM+2GgJ74urRz0IwBxnPFeeMp0kpVg1iYQv\\\\\\\/vpDPAEOcYqtKOJ9I1Y7MsEq332Kd22CkenyCZEDHYSxh8zqT5Y8cCDos7RqTrFLNE+w8dvT+8goQYaycU4EjwNMJXUvAwu\\\\\\\/is17xIIbtHRRT4LeaFrWXzIUWR9g7+3AT8KYO5hTKCozHAnWKJ3akrHhthDvOXZMZ0p9N1z3jyNPTunYUhSWTtf8dPpCSDutcCsIbdSfCDrIEBaBXyPw5fHEeywyPHhLnTbCeGojIofvPbSpd5e\\\\\\\/avTiH3vNxVvhr\\\\\\\/9UJRySAYhvT0RCrER+2R5QNGC9\\\\\\\/\\\\\\\/wVxfcHTLRkSsIb24AygQwU96sEEjl4coeGCZgqFzalinGwX+SGLESZZMwZbHGpLrpetCxJyEIqgwfMdZcEfHSe3cHSsrZiUgYY+1LaHy+GNuN98vxM+OMXv51joIVrTycwosEr7\\\\\\\/2dVOuyNodSEVsZzlanbECf1L\\\\\\\/4zuTapUG1pOENKJBQ6LTb6rTxjbJnCtZst5Vl9a1UkP5Xu8LSdKrQwixSVD7mGPYlVjjq1fPO8ix78lQNHTtqF\\\\\\\/+JzuTCu3CFyXg3pL0fT\\\\\\\/civ0RygFS\\\\\\\/ywQXlhf6DMVEAjAQ5VXTZazEWFs2HQ5xa7qZrSmOf6DSZ7F6zyq6GfXY4dtUE+xLqAHBsr4F3rCaX\\\\\\\/NmxgFlj3xJpZv7D0dhbOJCqbQKMAa2QfoTDT+j114fCEao62q8NTjM46KI2ItA0cHoMTCwcJFkaN+d\\\\\\\/WwWR\\\\\\\/HimHbDjtIGmJj9OMs\\\\\\\/m77+pJWybr+esWrOxVtJDB43MJs6Mi8NnFXXuMmkICKky92ERsO16B9kX2uC3vuknADic6DWSg4BQ\\\\\\\/MPmY\\\\\\\/l\\\\\\\/RLtbncPzJBTARC5M5o5rCKd4pn08kJEXIHBjsX9ZNGvEYE+vVj+KA4UoIsCnrq9DKLubf9tO9bIIBi3Elo0\\\\\\\/djLWQVyqyty6JYLo2+HcHQ+daf7bSn1kiCrcwZDCgjssf\\\\\\\/mkEuvnqwR1dgIT5KC28hZ4vAhJ4RZljmXPrqJgrK6FJi8FZzlu5+oa6vaKxq5GeKrwcyqr1E6HOSrv5iqDPTDi18fAJjciPl9E5wb4wxqHYNMGfVc9y\\\\\\\/1HEu6Iqa4196a\\\\\\\/PSbJfykqPPMXwtTrv\\\\\\\/zryx70t0URHxl33h0Ix7jATIVsi2D29Z5t3CNeeDrjRxIqGwNjwSGEfoejnhK+AQAtlj1hkiObet3n+Xj14Bb\\\\\\\/u91RCo7oWpulk0VKgnTBrJCQjnc3lUyjU4kPjFhJRBzVFdluFwpr0D2f7QgxDr02hUD1OmnhsEWVn3TK41hOAmRo+ZMfCkMm5MK\\\\\\\/et4FWO8qYsPP5KYms1VutFyeVLQMaLwpdUdRZnvMaxI4ePV5wBCEV2jtvGYnLY9fVpWCknTt4RJkkY46pjqAsk41C2yw5Nciip+2izdmsWAhQK2jT9YzsibkIphdNvCy5U\\\\\\\/uoxtzSzZN+w6HlyZfK42wJHAs1YHKRx2SkzPtTeEPE0jIOT99JDBfcMoWBP7habZf8VjdmQke+sMpf8HcJGvvQhyQiNPJ0746MCVS7Ho5U0tdlxBZ862x\\\\\\\/Ra2oBVSlUCL0vzBNbT5B6IYxHGHH8WPGZzjOMkgCi7zdtQ6PZrgk\\\\\\\/0gPHfcZRLmpac6UeorPoxg5XC83K0hoqjdqPMnujvmt90CWnxkVdgMW6DQXxTkmL+DRFwQn1lw1yay2xqrwjz3R\\\\\\\/9tZPFohnQUEhAH89JU\\\\\\\/uFqklu9jX6nJvB6+n6sH2qBW0aJTGInhjcQ1+zKyBD5fTHaE+oRW8Z6rlWBYNnGml9hO5o3HfWckEkHADXlQD5DUXa2rm8dlFDSJvfRUYZ+Z4buTfGAF2\\\\\\\/Q39QpT3od4NY3s9QKsSyKsfrzndjWneY8RO1vfaObF78iUTNyIesU+LzvQrb6skCbgOVjAKpQHvlhYJid32psiJiy6TR0RgIKXG7+ypS7rTQ5kpC7iSPsvnptzNFmLSqV0iqdkegXSZKLffH87KKBQldKuTPP3Hpf6aN1ZTVrQMyxvkRErsInJc\\\\\\\/Qzamvex6KN1vlsvD0eL+iaiZHLblhg93HpCiTtfcmL6j8hqA91GtrzVWECWdGZ0HkB24GgPz0ctJLpCQFGID6g39UQ6O17ZS6r2fkn\\\\\\\/y527fjB3wt\\\\\\\/NuyqYFcwd7IVDyXohMLE2LqPsRSrj2liMsfIetZwbyrXo1GZ+ORNDZ+RiRwHy9KU5nCgl6KUsoN9jTyPCIxl\\\\\\\/xyBlffiFYTqgoFpK+rbSYc\\\\\\\/cKX8SbZSLbbEzXtjKdHSEGAu5WvqNhpnwipfkSCJfuymdDMcXpDfWtXc1+ZJIj89gdrlO5+fwypwrSdYNwh+5+zcC894cKWhANLT6tZpuChPHlsOV2TpTp7cp7f7ud4AESIdQOZQRCavJ5YcoU3P7W3ZTxRWnndzvgGqjz9mAZ5+MfRXww2Nv7brAtG82TS0rXTbBGUPcdQxma\\\\\\\/E7QH6Aea0yfXQybQMEaoYsSc4n01qmkYyZJiNbdGQAZXjeHaUPXKIeun2x5n9kgVFJk7gxUKEN+zv\\\\\\\/uWUqHF9+WIfdcaksgFbobYCQnPF2axTYcsRoPSxHjPeKXAseF1Kzi4Flp0DJG\\\\\\\/M+bTFlPwy14hxBQ3n4ryhmFbvcBus4GbW3VdNFlnFiglXqhnYnm+1Hm8Pqg5VUicvURrEg48cWPXVoGY0Ewswp7BOMaMdLnAV6Q4tPEa\\\\\\\/SgqfWbajonpPwj8RipYTRS4KQPzVNMlog5CbDIC81C8OfEqRSKyet0X8EN1ttITAaKfUt1BBFHtv6+lGQ9+S2dmITAR\\\\\\\/O\\\\\\\/s8qsCcc6kWM5E8rODq3ex9Vun+m6FkpwD4lKl19a0R4YQJB3\\\\\\\/HJ335tHeNrO2e6UomiQr1+l1QfHztbzMotX0W5J644Rn+hXgyD+eTtIi\\\\\\\/gqwouhP8NvuRdmieAkYye6OQrpnl0Evs8pVmCUGGapK6kfzX6E\\\\\\\/\\\\\\\/+66PuMehZQm6Cq4iYZ7xzxvnilt6H7Y3bXAdOCThQ2tOplidtslK832sOxeWj99Gk4bmkc4d5nycXnzuKO\\\\\\\/MBk6if\\\\\\\/9pS9wT5jkKsDJoKbYyab38zLJtfnj268\\\\\\\/iuUrH\\\\\\\/1yoseW+r2ErxlCX3u7qiSKuDghUJ+RON2vteCJz99NlHC+472Ghr\\\\\\\/l0AOI0cQy23ho9S560PEgVRvny94+duAS9Al0mm6Zhn+6CzKA9X+egyTDwqAxevRrxWBIvP6qXlVoCF2O\\\\\\\/joWSVYGWk1UWlYuPQluRUb1NnJNnOE1qF3vV3JEMIdnK5ifMomOWK\\\\\\\/Gusirp0gYOpyq6IQqEnSlNbiDmDURBaIdbgiM9wxqdKEzgHLVXB\\\\\\\/fpjY5MrQBmYwrr5TKnB0EuChItuNxIBLcRG2NKqg459sbjbNrnjpGjAZh1r5IH\\\\\\\/44nLVV8JFRYQ++6Pm0uvKkNjtYwa3u+5tlox3SOhP29Jxy0Bz2vIdHVsL4hjxEOYLhndV+YmHQlPRGfOWmQ+S24vsHhAONhZJqztK65ETGZ1ZJUpDgxxzvBb+1D4HOaJHq00hVnYEOhWrfmlj5dZLoK6HjoQbpCk7T8IpogX3844t+MvUUtzjwimRAdqlUSSDC7eCGOqjfbJBUVfXv7xGbIWrSkj5Wr0IaQzeOVg3FQVRKNZy0beHc0YIV\\\\\\\/IOwnBie8h8GCIf6cyC5tnhrg48C61lq\\\\\\\/SazQHxfd+KUsFpshQfhmyBi0u7HaaFnn\\\\\\\/pjPcUNrI5t2EIYH2Vy9zOanfgy865ZMqdVXE5r0tufzLLTg7iKUCl6Tu5Mg4I82EvXOk7kjCu8oci\\\\\\\/vbNvBzQyYgPMHamvAJe21FCe6sTDnyGGt0VtnUHFDmQJFggToAJV4CsrIl3LUKUKqmEk4ynIlPIrkv7ZacZZvz+6AWgzA5gpdtP54lDKqBwY8z6DW5HYao1JrYnZu4i3lJiIY0gWHA2oPjeBxsiFP7HdfGCkuFFEXRS7G9zy2eBc+v+Sdiz7YpeGLr41lC+R9ttjkDbMqBvYqbo4ST2+vZN0fL7D3NdiEW7PrSmjZeIfqpB61NUDqiRqbs9K98owSmnrnBXOCLKG36IoTRYDr77Xhr0x8FBb4dZ3IHWUlUOYbPVJszKh\\\\\\\/xe6UTskyxW8fcwrRMHLGkiw4c3SoSF0JKFTT+cDzLelVQ9+f4CTkrlbo\\\\\\\/nlneqirDjxUOY3pS3+yLVX72WOhl7syl47dMAK83pVw+oWrH4SAHcPORyn047PVHI7u\\\\\\\/55m3i2EYR+J3nTv\\\\\\\/WEwuHL4GvzW2oLlAXywXQBppOfvB9IEch9bVldptyn9JTYV6bD5JK3fkIiDYR2z7MVW4VDMqnNxXy3RO+kA78emZ3un9qU2Vn8I1Dwu3otfaIsvBwyJRCts0RJ\\\\\\\/8pDVW3UfGqBH4nsiizGDsGCZWu9R8q4A+BFU3O6dK4g\\\\\\\/isYJ1YVQbJE4fWCmDmPMW4rrONL4fGPe3u8tNVCx4nxxosyuNXajQ\\\\\\\/6EVlZIuqPrDMwuITpHzKYNhfLbuK3a98it0t9OX1zzJzLilzIak0r7DI4FGXr8j4x0e\\\\\\\/HKvH1Lu7EJPxTdqmQmMWhqBvagO6oUWPdXnQA\\\\\\\/5k5uRyhE7zpft+paRxbodSRI3eR7EixLfuCK1\\\\\\\/vESpoe6Nu0YBRj9bprWWRIDqKl\\\\\\\/qIsnnbqZOCPd3Vv1RQK1WWHe\\\\\\\/Z1O6cR+7\\\\\\\/3+UYI02EHlvsplxp8L1L+uEwYC+JYIypjO8dSLRRWew76wj9z7pwNTQ2GdU\\\\\\\/oU7zlGWZ9y37WXqCY+TGKZwdHtR+Usq0PI2ZcoWIg65T4Mg64\\\\\\\/HqoVH4\\\\\\\/pAAXO2d9fzt5T4zRKjmnk+VcEwke4w0KehijwdHfFsgnklIkOZRrJ3nYhZTiNFOl3rl05Zs6UYSyUrhpytuZGCZ+AcgHRssUXZ3nGjXbefts7zUpoUymywJUXqUcrJ9rXeDZ18WidxNGOtErJNLxSculRG3pkucn9\\\\\\\/5Wg\\\\\\\/4Cq0HCoVfXzCRLxw3x\\\\\\\/m5crITTq52aLYmu1kLWmi8\\\\\\\/De06edzau3lzHgYmvnMk+CjYUsnZCcXBlbbTi2JNdlqGVt6Eobk\\\\\\\/w4JSYYWTfh6ikeVcGG+7oqOle6HgePYmDjnGTi7fY5wrx3LAX5qaIPrLKMBpto+QBriiNG2nQTtRqk20F0kfx6vSuIqwVkZkxoh4M9JDcIrFRc63rJ0QJVgpm7nWuzGKbleRS4y+hKraPRM2Ldk8MWxswg8YLQ2ilxQuN9Az212p5pq9j\\\\\\\/whJEL2XNVhWgPZK1gThzGcUZbCyiADiggC9DVg7l\\\\\\\/4XMgqdC0B0q+KhJznjKQNkHyULoqW9cIyF16U5Ti4sLNz0TxBBxQvT\\\\\\\/wLMcZHt0srT+wAcTXN9MIUnGgIOYSpsT1gqYRL8AnUIVcvLr5dRIMkCa5\\\\\\\/SMdeML\\\\\\\/\\\\\\\/PhecELUJfdx317L7\\\\\\\/7s6+XK9K\\\\\\\/ruyecKgoz4TdgUlamvH8o\\\\\\\/PBA16naAhMwCXWOu\\\\\\\/52h0OMw0v41X9t0PWRlBdmab1CXVkZi+6BKww1f+2WiF5T+V\\\\\\\/6mw8XeTwU6GSbtmKReiyHSNitk1w2jQShqzdqTEMt9SY\\\\\\\/1cqi5WK4po3xAEfGUL2V1w42FRdgOUia\\\\\\\/O5qrIEUo9zaSI7GSk835gamgAQxOFLQqvYVYyBod0OEPrhrYCtJEQqPlBnqcgugKMLXQKFyhwAbKhtP8v2+lrIt0b7hPfbd1k7hoVpoweTSqVfLVrwkdRyEAhQWNN0KgFGKyzLZGDX3am+AzRmXTho5OyUpjxBjGYyVj9j1HhvCckPE+HFW1KkAmjKVyYucGCMD5GOmR9IQB74NYuAj+3yzn9CgsRhmOCsDkoRBXzPUiO1zsiOHNyrcs6moV\\\\\\\/5qiNPhuGWvRk\\\\\\\/bX\\\\\\\/H2D4kM7Bk9dQ8QirZ2mNbixxWVcr5GjpJbccXWbWxIe52RobWheQfWZNNS7auRJRkhfuzTlEvEpsrx5006U4g9S9pQ\\\\\\\/c\\\\\\\/MPQZ9OP1rGFef4YU8O\\\\\\\/YYtf+dhwZMoQ1JN8JMxfmC8mIY3ode3Wek1X9YLeFhFsHmzH230MeBYJojdDf3a693ET57rqZF7zufzlAwKXKochQxA2PdX6YU5qRINnGjuuEzgWSWfBYBSVDfXNxF6z0O8DM1ngqGKA\\\\\\\/E5VhlLpk+hKw5Hzvg2DHp27IKcLDxS7848KU7VAKC7MaXEGS+OxO5WVms3BrkuZsRFoDCXMyGG0Q+IMqpZvh4yI09gCKjFacxnYcYAvh7sRG0QOROraUPDzneBMI3fOIXDFRwF+d0bvG7cxOA6qwuzSx\\\\\\\/JXi2me6VDDMQlubWNAa0AVVfdeZkpRrOhUewceBePds1VGq7QASeE80o1iEq8stCXSJjDIYbjtRvGIu78o+ehaPyHVGgBYSrzOWoSftDMRCEY0B1Ja0LBhc4TsfjBGMfntPoRxenq7GgKCk1fX1JY3vbOc\\\\\\\/m8ymN3Zm2Lqt6IJyr2+PAWap0yEBfBjaJxnpOc\\\\\\\/SN8RYqyWm1p4RNbcm6C4t5UetioAUzXC8jsHddBF7NGm6mGJ8S2rZUOGeZOaNOLnEcLchQs284LErpQdMRweR2wiv9t9OSp1Q9fJCw9qbrOiYHz5F3YMfkSe9rXv4Ja8X4DVixAZuDS32D9LwJ8iwusmkKdmzQ54vPGHXo041Pf91Rs0d5xHPazeDhdlcTps+P\\\\\\\/JGjh2AmxJG6D2zAHSWA4T5ipJS6pX8IlHfjQxQ1oAHjdrzTyGXrwMtHyGMx3GrySBK1\\\\\\\/6vPfHkVfSapcR3Zh00t+vtc2GwXk6SlxVvL7tbhWriEgBzLLZgCe5rTuC7cVZaTC3Efs0xSmcea2BBQoGvsGS\\\\\\\/eFwc+O4VIj9Tzi\\\\\\\/dpNuKXPwH9OKK7WnuXi95tmiXGIjtI1JxrjoEBckNDUjgfEdD4QNu9dTsp2eLJYIXIF\\\\\\\/imP7xPUXXFtYGGG63SlmAM4\\\\\\\/3XL2lt7\\\\\\\/hthANrd9cea71OSToWbnNMcGprGAs\\\\\\\/Uei49D\\\\\\\/ZpgGKNxDUGjp1MPffRP7+XwPblJZ+1iD8VNptAS8+Ym9y+TfPbqungmFsViv5iXX3tOMUowU29ynml83SmiI03sD0eLiYrOzAAtEVdf2P1AwOQ45Sa2XVl16M69pM15pphIe7i6HUYagGTISVgE8M8hgJm4l6eqCU6J8w32DynOPJegJ5nFuK2z3iWdXyXw7Wc\\\\\\\/Aa9YLRvPwzjo8LYT8zmL1Uf6hOLmmvTkzI2Gmm9KCDZMz2GL6IqzT4\\\\\\\/Ed3zkDnX\\\\\\\/6WFjWx7ik3YJ7+ISm7Irx+PnAGDx5mIGxa7IDbL3mcQqMOLWtIxhruEurrQllW3nBBvVxDx2JygVMmXZL+\\\\\\\/WRlaHiuN+LNhJCXcMDm5uLEkWwqp3xsPFKuG98Vgb4W6GE288wPf8r4khtfa85\\\\\\\/8CtrAMckTNb17LtVA3xqCoWHQPfD8GhckPbBs3zaDWRQWIdmNNnKcr7yP3CVqEfmSowc8B9uo\\\\\\\/ExIXUMUw86XnKmxzz6RaQLN7wSGhBM301+dhcCArFKLtM3WKSai5uH0gi5Rgsw3imo\\\\\\\/0\\\\\\\/NWWdSffeB+gtXUQfLW2sZVAeTYK+ThFsWW+QlDep0wi2m4+xmCwdNAROnVfBo\\\\\\\/ZSRKDvQncpii6PAhbUI95g2E4ws7wtGyhUSZe\\\\\\\/1Ywz8Nzxv\\\\\\\/Jep2w4zhPHyCEQBnzl5eN9IfnpO7zL8twvYhnWr+1mmRaePH+ZNPjdhId97+PUy\\\\\\\/9XVYhzeVzjUDmvsFp6wK\\\\\\\/1ZJYBiBT67qMKJsIZuComekrCa7Kkvc0YWibxaiLABGShuNyt6VVfIiX2d28ZIKSQa4O0mnUqHoocSP6EXh7z9VMR5CpL9msf5rlF6uQ1b3UgbS2rx69bjeYEs0\\\\\\\/jxw0McmnLlFIkwnK00zQj90A\\\\\\\/PIV7xfW+BIV\\\\\\\/DXwCnO+aTNwo9d9zgFIMETED++oZODgMchtuhH+EpBtzO4GfMXNkWRrg+hqgeb3y0r4SkgCp1lGxLHUh+CxbOhVHKi9pTNrUnIU+7xyUTESV1y19HFgjEXzpZ14BpR8cXOMswMQgQjhVmcGuCr2ENpCv9lOoW0O0q9BhxbyQvuiTKMM3PMppBQZVEqV7LL7O8+cXQQDF\\\\\\\/LZODNahtz1wuY+AJZlaztCY0iTta173RzrZwB4EWrsCIq5stMpBQ6kxwbQYX6XKKjlXPyzTCaAbrhKgfaWClUobOw8KwbWyH53DQwD52xjXsV0uWqOMSbUnurlYNnBEIHRE9xnc7AvJpkqWXfAAt0CdS0jt3fcDGh+U+D9yo62CT0gzhxA7QIOJTN0zwyYyc1HPuE7YCr1O13B68Upa0wR5LHSV4eUoFWY2J5GSvF3LyecO9GUy7ehPCoYbkq7mHESwyXfsUrz+rwEQZXbHQiCPDYGHzLYBc2cpCywv9i+CraHULq6gmwT72nN0B+ULXLfT4X5d6NpjIKFvrMhbr90akruO9PuylfZeEn\\\\\\\/bNBEVw+i+QDkETVH6VYKLD7278PZKr5a1lmdvBGCgrS\\\\\\\/jVFYCkg+blNfGl1zBWmyrg6hW\\\\\\\/zb0mX1\\\\\\\/4jdM1XfMDRT8078oWWUF5uy3OEgdjdTt4Yq3hFFA8DAg8UfS2tR1PQEAzZx68sCqvx1IJy8BCqxcrtkASiYPf9ZPl+7pFMvlUE+wwtCkDOzOobE1iaUtOs4gWFdhYA1LKBwz7UoneGte6kzViUQztl2U+TTP9sPLJf9mNykGOegsOI7uRroE3ALzIFusiTPOyoUe6sWfd28BZl2o+PlqSBHcUbbGMtc8Pr4ji4gReelqam++CA21w7\\\\\\\/vGXc9j\\\\\\\/MJ9FfStxIzH4Tc43yyJj4WtF2bH0gbzw26HyvgmfboVbIrpzKsnmrYnJWxo06QxU+bUsCz2dSCoBmtXZIi02bBUgEFKtETfSrylGXBVXKZVzG4\\\\\\\/D1KWCuQpoaaTUFXmSWjRTsygDNXysOSSOmXjsy5DjSRhFzO5R5vO4M+jpHFyik3mdRnsel5qfJEybSwMdPxI4cwfXfmdfi15evjD7c4rFilBadWEkD\\\\\\\/r+02w3fgSbeaLXSzJDTu9JLAApRh+o9MtGPSxIF8biDsbLBfl1EOWYzoSXazd0ZpZHNvCzNDVay7W3CxkPuljh\\\\\\\/h8grcTiuQcWPe3\\\\\\\/yCA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wzHo5JZ7H2+aWLNf4O0p7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef581df0299545136c88a082241cc460f74f10b019c2fd807464c81872d42d75\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YGI03mVmmnk\\\\\\\/vhieoDBAm35OXqcPcTVcYqOhM8A+RE9L+WWMLYvHQ8bk5X\\\\\\\/yBqA8ArPtklXqZ3Rypz2Kq04uZfE1KjNw+M8UemMZV2ENNrv07g84T63bkU7MTnvk2HczFhzk1Br2LROIqOA8CDXwzWyM348OBJFPX90fqDdnaOWcih5AjIyBt7gcYjeLt6b\\\\\\\/lkfI7yblI6+ZmSDRZlkCZy26jDVAtbAyXkQYgOTGCVUyLVCK\\\\\\\/ePS9qhBYZxlIM50D+mYssblxamFtbL40ILoaPqBqeo0Or5j97t7ORuGvfgPN7Ar4cQtd9jrXX3bv4kD2p5DXuYOlf5oXYpLa55fiyU8r6JCajqFpfmNwy2BJdKUCP+3lb6PQ7nXCOwGMIMSgULQ6kx5z+FcVDPqlTunDoCiAsR5+CdJa1rVbo3QFZStfXzRP20lNPOAAehsmP7RsWiPAUqOOHD6nSPxaT94JB+5FIIZGrDn7rDDM4A9t84UGUVQnAtlaqAFX3KQkR8lz87DVhl5WGpT2\\\\\\\/CD85VdCDvq7+jqqa2uDW51TXLavB8syYOmHG8qH9jy+pp\\\\\\\/pogMy8cNRcsDPqbA25NM8SlN1I2sDPEoxQUm7mDQw+EhsR4pLltRroL57LO3uNNtFnRabAh+83zMGwgtzmiD3u9SMwjjxmEFOLNh4SsRS+05WEUMvbXfkginKM0mv5UqYIeAbkT18YyVbv+eFfOkurhjQo4NIFy+fyz+8Dvq7aAj99CncxE\\\\\\\/9YNQUUWkIF6GfS8SxOhh4cKtKsjc7iMDH8EFydEu6Ab4BYlbT9ncw2YBpXYqIRuksI6wPPUPQcoYLtRhaZ1skHd505RlWfaLCh5Gc7yu6Wx84qkvQf\\\\\\\/juQgAIJFPn2QeUVA7Ov5ihRQy7+cr9q8FSRWpBKFWLLJIPctLWeBxPDIjsYlKHVrwecpsCVxqu32M9jNOzYKIARYPj1Z8eJnLPQk5QDkZbDPHgQe6RdyD8wOweQwwTkdPclemNWv99WNYDS\\\\\\\/KHMxOWWajow1PR7Ac58jU9uvB2kK1uuYaLTzVey5tAb9fo4jzYh3LvQ6r2X3Zh9iJijVeDFL7bEQGAvuwit4W+xix8YQPiALm09BA+zZLMq4\\\\\\\/0UZqDJ338Zs+qGH2rumh9O7AgSIBe\\\\\\\/PQpBdaAzIh6DjQz3JBpNG7lOxcyTp6arknWw4Crh7r6SlyIpvZLbcMBxX1ad1cDGY0mj5eP1uj\\\\\\\/bRzjErM9jINZ\\\\\\\/hJnSoD6MtPMvQILTKl8Y4Fj59qkiSdugjPdpI\\\\\\\/X9i82Y7JIez81F+9k6FaUU1r0nRNRluO7304mu3eY\\\\\\\/iRshL6gqPXsJp\\\\\\\/Si3YiduMClSPDFGk6Cn2zsz8LKueIiv4fiQFodwUILrLEJXeC47B7EenpXRTCv66Ac+\\\\\\\/AW4AjL3QM6Mczd25F1a0\\\\\\\/YzR\\\\\\\/Nf2mnargW8UBQbA8Ra0+50eOD9ADuhs9uLikRAMugdNJWm5th1aemSXwJQ6L\\\\\\\/Z+XtKfaGkgmd5iCgPM2gtfE9FhakJbkDAKa1sQzHppdp9yIyE+z+A8cvH1dTwp2sM31SH1GUOykfOmHDsB1rVOgx91GuM3V1o5RV3ZcJUm4Nz\\\\\\\/C6joNP6qjBoCEcraVIkdzQSdYOYdkcXbmqVmREuAzXk0RwIgr8FxgeCBnhnQevreMk1yDlG3jNTaKkjksBzjFSgptWe62ACQLUGGu4Biv8PN1Gj51Z57BRhpnEVzZony08ZsNfidh1NJMtJ8SFg34ZHy9Pp4FX4ZnMR4EDsOGs3VQ\\\\\\\/KZoVrUnu5mn3zQ1eC8rnm+8wO6SDqY6KznJDTQMkxTBSzZjhxqcnceMNwJiyQxrpB+lSfaffgqgaMtWYCInlvj+\\\\\\\/B+buBKZcSjXZ+U0Tc\\\\\\\/bPk1iydryfDqekAxv5ngGD14g2dEscFkMz9Vvk9c6IrLsTLtLBz8xawzFVpF4qQoSehZM\\\\\\\/bD\\\\\\\/tFha\\\\\\\/AaLc0ib19LaquS7U\\\\\\\/VHVplJY87Qfd3nmvFwagtlergXOhLLyTXVg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tp98p8fttICoi8WYV1O5yg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002280b468a86de57565863856007f2dfbf0b7e9588b9b5716501775ffa5bbd4846e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459702 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2259" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cwZT2SiwcGsHV9LU05Z6kGXUEU9V1OFWca6IEJvN0wM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2259" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9fPYMaSpO_2Vr1ax37NyKyx3WWyGi6Bs_fMn2IWa_nU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2247" title="The DExH Box Helicase Domain of Spindle-E Is Necessary for Retrotransposon Silencing and Axial Patterning During Drosophila Oogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2271" title="Flow Cytometry of Microencapsulated Colonies for Genetics Analysis of Filamentous Fungi" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459702" id="node459702" data-pisa="ggg;4/11/2259" data-pisa-master="ggg;g3.114.012922" data-apath="/ggg/4/11/2259.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Structures of Naturally Evolved <em>CUP1</em> Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying Zhao</span>, <span class="highwire-citation-author" data-delta="1">Pooja K. Strope</span>, <span class="highwire-citation-author" data-delta="2">Stanislav G. Kozmin</span>, <span class="highwire-citation-author" data-delta="3">John H. McCusker</span>, <span class="highwire-citation-author" data-delta="4">Fred S. Dietrich</span>, <span class="highwire-citation-author" data-delta="5">Robert J. Kokoska</span> and <span class="highwire-citation-author" data-delta="6">Thomas D. Petes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2259-2269;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012922 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459702" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pooja K. Strope </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pooja%2BK.%2BStrope%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strope%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APooja%2BK.%2BStrope%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stanislav G. Kozmin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stanislav%2BG.%2BKozmin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozmin%20SG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStanislav%2BG.%2BKozmin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John H. McCusker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BMcCusker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCusker%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BMcCusker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fred S. Dietrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BS.%2BDietrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dietrich%20FS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BS.%2BDietrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert J. Kokoska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Physical Sciences Directorate, U. S. Army Research Office, Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BKokoska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kokoska%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BKokoska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas D. Petes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BD.%2BPetes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petes%20TD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BD.%2BPetes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tom.petes@duke.edu" class="" data-icon-position="" data-hide-link-title="0">tom.petes@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/11/2259" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459702" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459702/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2259.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459702" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459702/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2259.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459702" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459702/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2259.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459702" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459702/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/11/2259.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459702" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459702/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">An important issue in genome evolution is the mechanism by which tandem duplications are generated from single-copy genes. In the yeast <em>Saccharomyces cerevisiae</em>, most strains contain tandemly duplicated copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>, a gene that encodes a copper-binding metallothionein. By screening 101 natural isolates of <em>S. cerevisiae</em>, we identified five different types of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>-containing repeats, as well as strains that only had one copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>. A comparison of the DNA sequences of these strains indicates that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> tandem arrays were generated by unequal nonhomologous recombination events from strains that had one <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/copper-resistance" class="hw-term hw-article-keyword hw-article-keyword-copper-resistance" rel="nofollow">copper resistance</a></li><li class="kwd"><a href="/keyword/tandemly-repeated-genes" class="hw-term hw-article-keyword hw-article-keyword-tandemly-repeated-genes" rel="nofollow">tandemly repeated genes</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li></ul><p id="p-3">One common and important source of genomic alterations is copy-number variation (CNV). The term is often used to describe a variant genome that contains deletions or duplications of sequences (100 bp to 1 Mb) relative to a standard genome. The rate of CNV formation in human cells is about 2 × 10<sup>−6</sup> to 10<sup>−4</sup> per generation per locus (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> 2009</a>). In comparison, the rate of point mutations in human cells is about 2 × 10<sup>−8</sup>/bp/generation (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> 2009</a>). Here we will emphasize those CNV events that affect alterations in a pre-existing tandem array or that generate tandem duplications of single-copy genomic sequences.</p><p id="p-4">In a pre-existing tandem array, one mechanism that can alter the number of repeats is unequal crossovers (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), a mechanism first demonstrated in studies of Bar eye <em>in Drosophila</em> almost 100 years ago (Sturtevant and Morgan 1923). As shown in the figure, homologous recombination between misaligned tandem arrays can result in both deletions and duplications of repeats. Unequal crossovers can occur between sister chromatids (as shown) or between homologs. Deletions also can be formed nonreciprocally by several other mechanisms, including “pop-outs” (in which an intrachromatid crossover produces a circular DNA molecule containing one or more repeats and a shorter tandem array), or by single-strand annealing (in which processing of a break within the tandem array followed by annealing of the broken ends deletes one or more repeats) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ivanov <em>et al.</em> 1996</a>). In yeast, unequal crossover events within the tandemly repeated ribosomal RNA genes are very frequent, occurring at a mitotic frequency of &gt;10<sup>−2</sup> per mitotic division (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Szostak and Wu 1980</a>) and &gt;10<sup>−1</sup> per meiotic division (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Petes 1980</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mechanisms for altering or generating tandem gene arrays. Events are depicted as occurring between sister chromatids, and centromeres are shown as ovals or circles. (A) Unequal crossovers between misaligned sister chromatids can generate arrays with greater or fewer repeats per array. Repeats are indicated by small rectangles. (B) Unequal crossovers between flanking repeats (triangles) can produce a duplication of single-copy sequences (dashed line). (C) In this model, a broken end invades a sister chromatid using microhomology. Break-induced replication of the invaded chromosome results in a duplication of the sequence (shown as a triangle) initially present in one copy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mechanisms for altering or generating tandem gene arrays. Events are depicted as occurring between sister chromatids, and centromeres are shown as ovals or circles. (A) Unequal crossovers between misaligned sister chromatids can generate arrays with greater or fewer repeats per array. Repeats are indicated by small rectangles. (B) Unequal crossovers between flanking repeats (triangles) can produce a duplication of single-copy sequences (dashed line). (C) In this model, a broken end invades a sister chromatid using microhomology. Break-induced replication of the invaded chromosome results in a duplication of the sequence (shown as a triangle) initially present in one copy.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F1.medium.gif" width="243" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/11/2259/F1.medium.gif" width="243" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465280" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Mechanisms for altering or generating tandem gene arrays. Events are depicted as occurring between sister chromatids, and centromeres are shown as ovals or circles. (A) Unequal crossovers between misaligned sister chromatids can generate arrays with greater or fewer repeats per array. Repeats are indicated by small rectangles. (B) Unequal crossovers between flanking repeats (triangles) can produce a duplication of single-copy sequences (dashed line). (C) In this model, a broken end invades a sister chromatid using microhomology. Break-induced replication of the invaded chromosome results in a duplication of the sequence (shown as a triangle) initially present in one copy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">Homologous recombination between noncontiguous direct repeats can also result in duplication or deletion of single-copy sequences located between the repeats (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Such events have been detected in mammalian/human genomes (reviewed by <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> 2009</a>) and in yeast (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Koszul <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Finn and Li 2013</a>). Although most of these studies were performed in wild-type strains, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Finn and Li (2013)</a> showed that re-replication resulting from a prematurely initiated second round of DNA synthesis substantially elevated the frequency of this class of nonallelic homologous recombination.</p><p id="p-7">Two other mechanisms that lead to duplications of single-copy genes also have been observed. The first mechanism involves a specialized type of break-induced replication (BIR). As shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, a broken chromosome end invades a nonallelic region of a sister chromatid, copying sequences from the chromatid. This initial invasion involves little or no sequence homology. In yeast, which has small chromosomes (&lt;2 Mb), synthesis could continue to the end of the chromosome. This type of event (termed “microhomology-mediated break-induced replication,” or MMBIR) was demonstrated to be the causal mechanism for some duplications of the yeast ribosomal protein gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005839">RPL20B</a></em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Payen <em>et al.</em> 2008</a>). A related mechanism that sometimes involves multiple invasions and template switches was proposed to account for some classes of CNVs in mammalian cells (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lee <em>et al.</em> 2007</a>).</p><p id="p-8">Because different selective procedures were used to obtain <em>de novo</em> duplications in the yeast experiments described previously, it is difficult to compare the relative rates of duplications by these pathways. In experiments in which duplications of a reporter gene were located between two retrotransposons located about 50 kb apart, duplications were observed at an approximate frequency of 10<sup>−6</sup>−10<sup>−7</sup> (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2013</a>); all of the detected duplications in a haploid strain resulted from unequal crossovers between the flanking Ty1 elements (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2013</a>). This result argues that, at least in this chromosomal context, nonallelic homologous recombination events generate duplications more frequently than MMBIR or related mechanisms. In contrast, duplications of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005839">RPL20B</a></em> gene were generated by nonallelic homologous recombination and MMBIR with approximately equal frequencies (about 5 × 10<sup>−8</sup>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Payen <em>et al.</em> 2008</a>).</p><p id="p-9">In the studies described herein, we characterized the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> locus in a collection of yeast strains isolated from the wild and from clinical specimens. In the yeast strain S288c, which was the first strain sequenced (Saccharomyces Genome Database [SGD]), the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> locus is depicted as having two repeats (shown in red brackets in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The repeat has a complex structure that begins about 150 bp upstream of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028470">RUF5-1</a></em> gene, which encodes a transcript of unknown function (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">McCutcheon and Eddy 2003</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene (encoding a protein of only 61 amino acids; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Karin <em>et al.</em> 1984</a>) is embedded within the <em>RUF5</em> gene, but is transcribed in the opposite orientation (<a href="http://www.yeastgenome.org/cgi-bin/seqTools">http://www.yeastgenome.org/cgi-bin/seqTools</a>). There is a second intergenic region of about 70 bp separating the 3′ end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028470">RUF5-1</a></em> gene from the 3′ end of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001096">YHR054C</a></em>. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001096">YHR054C</a></em> is identical to the terminal 1 kb of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em>. There is also an ARS element within each repeat that overlaps with the 3′ end of <em>RUF5</em>, and extends to the 3′ end of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em>. At the centromere-distal end of the array is a complete copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structures of the CUP1 loci for five types of CUP1 repeats and a single-copy CUP1 locus. The CUP1 gene (blue) is located within a noncoding RNA gene (RUF5, pink) between the genes CIC1 (green), and RSC30 (orange) on chromosome VIII. We show only two copies of each repeat (bracketed in red), although most strains have more than two copies. Thin horizontal arrows show the direction of transcription, and short fat arrows indicate the location of primers used in the analysis. The vertical gray lines are 1 kb apart, and the blue arrows show the position of EcoRI sites relevant to Southern analysis. ARS elements are indicated by purple lines. The lines labeled J1-J5 represent the junction fragments between the end of one repeat and the beginning of the next. Lines 1&#x2212;1 to 5&#x2212;1 represent the leftmost end of the tandem arrays, and lines labeled 1&#x2212;2 to 5&#x2212;2 represent the rightmost end of the array. (A) Type 1 repeats (2.0 kb) found in strains S288c and W303-1A. YHR054C is an open reading frame derived from the 3&#x2032; end of RSC30. (B) Type 2 repeats (1.8 kb) found in strains YJM189 and YJM972. (C) Type 3 repeats (1.2 kb) found in strains YJM693, YJM789, and YJM1549. (D) Type 4 repeats (1.9 kb) found in strains YJM271, and YJM1307. (E) Type 5 repeats (1.6 kb) found in strains YJM456, YJM969, and YJM978. (F) Single-copy CUP1 locus found in the copper-sensitive strain DTY3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structures of the &lt;em&gt;CUP1&lt;/em&gt; loci for five types of &lt;em&gt;CUP1&lt;/em&gt; repeats and a single-copy &lt;em&gt;CUP1&lt;/em&gt; locus. The &lt;em&gt;CUP1&lt;/em&gt; gene (blue) is located within a noncoding RNA gene (&lt;em&gt;RUF5&lt;/em&gt;, pink) between the genes &lt;em&gt;CIC1&lt;/em&gt; (green), and &lt;em&gt;RSC30&lt;/em&gt; (orange) on chromosome VIII. We show only two copies of each repeat (bracketed in red), although most strains have more than two copies. Thin horizontal arrows show the direction of transcription, and short fat arrows indicate the location of primers used in the analysis. The vertical gray lines are 1 kb apart, and the blue arrows show the position of &lt;em&gt;EcoR&lt;/em&gt;I sites relevant to Southern analysis. &lt;em&gt;ARS&lt;/em&gt; elements are indicated by purple lines. The lines labeled J1-J5 represent the junction fragments between the end of one repeat and the beginning of the next. Lines 1&#x2212;1 to 5&#x2212;1 represent the leftmost end of the tandem arrays, and lines labeled 1&#x2212;2 to 5&#x2212;2 represent the rightmost end of the array. (A) Type 1 repeats (2.0 kb) found in strains S288c and W303-1A. &lt;em&gt;YHR054C&lt;/em&gt; is an open reading frame derived from the 3&#x2032; end of &lt;em&gt;RSC30&lt;/em&gt;. (B) Type 2 repeats (1.8 kb) found in strains YJM189 and YJM972. (C) Type 3 repeats (1.2 kb) found in strains YJM693, YJM789, and YJM1549. (D) Type 4 repeats (1.9 kb) found in strains YJM271, and YJM1307. (E) Type 5 repeats (1.6 kb) found in strains YJM456, YJM969, and YJM978. (F) Single-copy &lt;em&gt;CUP1&lt;/em&gt; locus found in the copper-sensitive strain DTY3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F2.medium.gif" width="440" height="351"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/11/2259/F2.medium.gif" width="440" height="351"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-10" class="first-child">Structures of the <em>CUP1</em> loci for five types of <em>CUP1</em> repeats and a single-copy <em>CUP1</em> locus. The <em>CUP1</em> gene (blue) is located within a noncoding RNA gene (<em>RUF5</em>, pink) between the genes <em>CIC1</em> (green), and <em>RSC30</em> (orange) on chromosome VIII. We show only two copies of each repeat (bracketed in red), although most strains have more than two copies. Thin horizontal arrows show the direction of transcription, and short fat arrows indicate the location of primers used in the analysis. The vertical gray lines are 1 kb apart, and the blue arrows show the position of <em>EcoR</em>I sites relevant to Southern analysis. <em>ARS</em> elements are indicated by purple lines. The lines labeled J1-J5 represent the junction fragments between the end of one repeat and the beginning of the next. Lines 1−1 to 5−1 represent the leftmost end of the tandem arrays, and lines labeled 1−2 to 5−2 represent the rightmost end of the array. (A) Type 1 repeats (2.0 kb) found in strains S288c and W303-1A. <em>YHR054C</em> is an open reading frame derived from the 3′ end of <em>RSC30</em>. (B) Type 2 repeats (1.8 kb) found in strains YJM189 and YJM972. (C) Type 3 repeats (1.2 kb) found in strains YJM693, YJM789, and YJM1549. (D) Type 4 repeats (1.9 kb) found in strains YJM271, and YJM1307. (E) Type 5 repeats (1.6 kb) found in strains YJM456, YJM969, and YJM978. (F) Single-copy <em>CUP1</em> locus found in the copper-sensitive strain DTY3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">The repeat length in S288c is 1998 bp, which we will refer to as the Type 1 repeat. Although only two repeats are shown in the genomic sequence of S288c in the SGD, the closely related strain X2180-1A has about 15 repeats (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fogel and Welch 1982</a>). Herein, we find that S288c also has about 15 repeats. A copper-sensitive strain BZ31-1-7Ba had a restriction enzyme digest pattern consistent with the presence of a single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fogel and Welch 1982</a>), and industrial yeast strains were observed that had tandem <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats of about 1.5 and 1.7 kb (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Welch <em>et al.</em> 1983</a>). Neither the single-copy <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeat nor the variant repeats were sequenced in previous studies.</p><p id="p-12">Whole-genome sequencing and assembly recently has been performed for 93 diverse strains of <em>Saccharomyces cerevisiae</em> (P. K. Strope, D. A. Skelly, S. G. Kozmin, G. Mahadevan, E. A. Stone, P. M. Magwene, F. S. Dietrich, and J. H. McCusker, personal communication); this collection, which also includes seven previously sequenced strains, has been termed the “100-genomes strains.” From these sequences and additional sequencing efforts, we defined the structures of five different classes of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats as well as the sequence of a strain (DTY3) with only a single copy <em>of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>. The comparison of these sequences suggests a simple mechanism by which a single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene is duplicated to form a tandem array by unequal non-homologous recombination.</p><p id="p-13">We also compared the sizes of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> arrays in 13 different strains that had at least two copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> and found a size range of between two and eighteen repeats per array. The size of the array roughly correlated with the ability of the strain to grow in high levels of copper. Finally, we showed that a strain with 14 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats reverted to having only one <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene at a rate of about 7.6 × 10<sup>-7</sup>/division.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains</h3><p id="p-14">The genotypes of all strains used in our study are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/012922SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS1.pdf">Table S1</a> and primers used in strain construction, polymerase chain reaction (PCR) analysis, or DNA sequencing are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS2.pdf">Table S2</a>. The genotypes of the haploid strains S288c (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Engel <em>et al.</em> 2014</a>), W303-1A (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Thomas and Rothstein 1989</a>), YJM789 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wei <em>et al.</em> 2007</a>), and DTY3 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Tamai <em>et al.</em> 1993</a>) have been previously described. The diploid strains YJM189, YJM271, YJM456, YJM693, YJM969, YJM972, YJM978, YJM996, YJM1307, and YJM1549 were isolated from the wild (P. K. Strope, D. A. Skelly, S. G. Kozmin, G. Mahadevan, E. A. Stone, P. M. Magwene, F. S. Dietrich, and J. H. McCusker, personal communication); these diploids were generated by a mating type switch and have no polymorphisms other than at the <em>MAT</em> locus. The strain YJM799 is isogenic with YJM789 except for changes introduced by transformation and was obtained from J. McCusker (Duke University). JSC19-1 is also isogenic with YJM789 except for changes introduced by transformation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">St. Charles and Petes 2013</a>).</p><p id="p-15">The strain JSC10-1 (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">St. Charles and Petes 2013</a>) is isogenic with W303-1A except for changes introduced by transformation and has the genotype: <em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>-3,112 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>-11,15 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005654">ade2</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>-100Δ</em>::<em>NAT <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004022">RAD5</a></em>. We generated a PCR fragment containing a wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene by amplifying genomic DNA of the strain S288c with the primers VIII212898::URA3 F and VIII212898::URA3 R. This fragment was transformed into JSC10−1 to generate a derivative (YZ22) that had the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene integrated within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> cluster at position 212898 in the <em>RUF5</em> gene of one of the repeats; this allele is called <em>VIII212898</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>.</p></div><div id="sec-3" class="subsection"><h3>Measurement of the sizes and sequences of <em>CUP1</em> repeats</h3><p id="p-16">Our first estimate of the sizes of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats in different strains was initially based on a PCR analysis. For this analysis, we used primers F1 and R1′ located within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> but oriented in different directions (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). To generate a PCR product, the array needs to contain at least two tandem <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes. The resulting PCR fragments were analyzed by gel electrophoresis. Some of the strains did not generate a PCR fragment with this procedure. We confirmed that these strains had only a single copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> by PCR analysis using the primers VIII211849 and VIII216603 (abbreviated 849 and 603 in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). With these primers, strains that have a single copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> produce a PCR fragment of 2.7 kb.</p><p id="p-17">Sequence analysis of various classes of repeats and their flanking sequences were determined by sequencing PCR fragments with the primers described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/FileS1.pdf">File S1</a>. These sequences are displayed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS9.pdf">Table S9</a>, and <a href="http://www.g3journal.org/content/suppl/2014/09/17/g3.114.012922.DC1/TableS10.pdf">Table S10</a>.</p></div><div id="sec-4" class="subsection"><h3>Analysis of the number of <em>CUP1</em> copies in tandem arrays</h3><p id="p-18">Genomic DNA isolated from different strains was isolated in plugs of low-melt agarose as described previously (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">McCulley and Petes 2010</a>). The samples were treated overnight at 37° with <em>Eco</em>RI. The resulting DNA fragments were separated by CHEF (<em>i.e.</em>, contour-clamped homogeneous electric field) gel electrophoresis (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">McCulley and Petes 2010</a>), followed by transfer of the separated fragments to nylon membranes. The hybridization probes were prepared using digoxygenin-dUTP (Roche); details of the hybridization conditions are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/FileS1.pdf">File S1</a>. Sizes of the tandem arrays were estimated relative to DNA size standard (Bioline DNA Hyperladders I and VI).</p></div><div id="sec-5" class="subsection"><h3>Measurements of copper resistance</h3><p id="p-19">For each strain, about 1000 cells were inoculated into 5 mL of SD complete medium (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Guthrie and Fink 1991</a>) containing levels of copper sulfate that varied between 0 and 2.4 mM, changing in 0.1 mM increments. After 2 d of growth at 30°, we measured the OD<sub>660</sub> of each culture. If the OD<sub>660</sub> of the undiluted culture was &lt;0.1, we scored the concentration as inhibitory to growth of that strain. In the uninhibited cultures, the OD<sub>660</sub> was between 1.8 and 3, the equivalent of about 2−4 × 10<sup>7</sup> cells/mL.</p></div><div id="sec-6" class="subsection"><h3>Measurements of the rate of loss of a <em>URA3</em> gene integrated within the <em>CUP1</em> repeats in YZ22</h3><p id="p-20">As described previously, the haploid strain YZ22 (isogenic with W303-1A) contains a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> array of approximately 14 repeats with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene integrated into the array. As explained in the text, an unequal crossover or various intrachromatid recombination events can result in loss of the insertion. Strains that lose the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> insertion result in derivatives that are Ura<sup>-</sup> and selectable on medium containing 5-fluoro-orotate (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Boeke <em>et al</em>. 1987</a>). We measured the frequency of 5-FOA<sup>R</sup> derivatives and the total number of cells in about 60 independent cultures of YZ22. These frequencies were converted to rates using the method of the median (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lea and Coulson 1949</a>). We screened 135 independent 5-FOA<sup>R</sup> derivatives by PCR analysis with primers VII211849 F and VIII216603 to identify strains that contained only one <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene. Six of the 135 strains had the 2.7 kb fragment expected for single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Identifying five different <em>CUP1</em> repeats</h3><p id="p-21">Until recently, the complete assembled sequence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> array was available only for S288c (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Johnston <em>et al.</em> 1994</a>). In S288c, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats are about 2.0 kb in size. In examining genomic sequences derived from YJM789 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Wei <em>et al.</em> 2007</a>), we noticed that the partial sequence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeat in contig 18 was different from that of S288c. We decided, therefore, to examine the sequence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats and flanking sequences in YJM789 and other yeast strains that had repeats different from S288c. In addition, we sequenced the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene in DTY3, a strain that has only one <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene.</p><p id="p-22">The initial characterization of the repeats was done by PCR analysis using primers located within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene (F1 and R1′, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS2.pdf">Table S2</a>). The location and orientation of the primers are shown in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>. To produce a PCR product, there must be at least two tandem <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> copies. Because these primers are located about 40 bp apart, the PCR product is about 40 bp smaller than the size of the repeats. In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, we show a PCR analysis of the five classes of repeats. Although we show the results for only a few of the strains from the 100-genomes sequencing project, all of these strains were examined by PCR.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of PCR fragments representing various types of CUP1 repeats. Using primers F1 and R1&#x2032;, we PCR-amplified genomic DNA samples from five different yeast strains. Because of the locations of F1 and R1&#x2032; within the CUP1 gene, the resulting fragments are about 40 bp shorter than the repeat length determined by DNA sequencing. Lanes 1 and 7 contain sizing ladders. The strain names, and the types and sizes of the repeats in the other lanes are: lane 2 (S288c, type 1, 2.0 kb); lane 3 (YJM271, type 4, 1.9 kb); lane 4 (YJM189, type 2, 1.8 kb); lane 5 (YJM456, type 5, 1.6 kb); lane 6 (YJM789, type 3, 1.2 kb)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of PCR fragments representing various types of &lt;em&gt;CUP1&lt;/em&gt; repeats. Using primers F1 and R1&#x2032;, we PCR-amplified genomic DNA samples from five different yeast strains. Because of the locations of F1 and R1&#x2032; within the &lt;em&gt;CUP1&lt;/em&gt; gene, the resulting fragments are about 40 bp shorter than the repeat length determined by DNA sequencing. Lanes 1 and 7 contain sizing ladders. The strain names, and the types and sizes of the repeats in the other lanes are: lane 2 (S288c, type 1, 2.0 kb); lane 3 (YJM271, type 4, 1.9 kb); lane 4 (YJM189, type 2, 1.8 kb); lane 5 (YJM456, type 5, 1.6 kb); lane 6 (YJM789, type 3, 1.2 kb).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F3.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/11/2259/F3.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465264" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Analysis of PCR fragments representing various types of <em>CUP1</em> repeats. Using primers F1 and R1′, we PCR-amplified genomic DNA samples from five different yeast strains. Because of the locations of F1 and R1′ within the <em>CUP1</em> gene, the resulting fragments are about 40 bp shorter than the repeat length determined by DNA sequencing. Lanes 1 and 7 contain sizing ladders. The strain names, and the types and sizes of the repeats in the other lanes are: lane 2 (S288c, type 1, 2.0 kb); lane 3 (YJM271, type 4, 1.9 kb); lane 4 (YJM189, type 2, 1.8 kb); lane 5 (YJM456, type 5, 1.6 kb); lane 6 (YJM789, type 3, 1.2 kb).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">These results and those obtained by deep sequencing (P. K. Strope, D. A. Skelly, S. G. Kozmin, G. Mahadevan, E. A. Stone, P. M. Magwene, F. S. Dietrich, and J. H. McCusker, personal communication) indicate that there are at least five types of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats in <em>S. cerevisiae</em>. The approximate sizes of these repeats in kb are: Type 1 (2.0), Type 2 (1.8), Type 3 (1.2), Type 4 (1.9), and Type 5 (1.6). S288c and W303-1A have Type 1 repeats, whereas YJM789 has Type 3 repeats. From the deep-sequencing analysis, of the 66 strains with at least two <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes, 57 had only one type of repeat per array, and 9 had more than one type. Of the “pure” arrays, the number of strains with each type are: Type 1 (8), Type 2 (14), Type 3 (18), Type 4 (4), and Type 5 (13).</p><p id="p-25">About one third of the 100-genome strains examined by PCR failed to produce a product with primers F1 and R1′. Genomic DNA from these strains was re-examined using primers VIII211849 and VIII216603 (labeled as 849 and 603, respectively, in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>) that are located in the genes flanking the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> sequences. In strains that have only one copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>, we expect to see a PCR fragment of about 2.7 kb. A fragment of this size was observed in the 100-genomes strains that failed to generate a PCR product with primers F1 and R1′. We also observed the 2.7-kb fragment in DNA derived from the copper-sensitive strain DTY3. Deep-sequencing analysis also showed that 30 of the 100-genome strains had only one <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene.</p></div><div id="sec-9" class="subsection"><h3>Sequence analysis of <em>CUP1</em> repeats</h3><p id="p-26">The structure of the Type 1 repeats of S288c is shown in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>. The repeat can be defined by the sequence of the junction (labeled “J1” in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>) that has centromere-proximal sequences derived from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> (named <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001096">YHR054C</a></em>) fused to centromere-distal sequences derived from the region located 5′ to <em>RUF5</em>. As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, the junction sequence can be matched nearly perfectly to sequences derived from within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> gene (line above J1, labeled 1−2) on the left side and to sequences from the <em>CIC1-RUF5</em> intergenic region (line below J1, labeled 1−1) on the right side; the locations of the 1−1 and 1−2 sequences are shown in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a>. There are no sequence homologies at the breakpoints of the junctions. As will be explained in detail below, this observation is relevant to the mechanism by which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats were formed.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequences at the breakpoints for five different types of CUP1 repeats. Breakpoint sequences of five different types of CUP1 repeats are shown in comparison to the sequences flanking the repeats. The breakpoints of each repeat are labeled &#x201C;Junctions 1&#x2212;5,&#x201D; and are the middle lines of each comparison. The top lines of each comparison (highlighted in blue) represent sequences from the centromere-distal flanking region, usually including a portion of the RSC30 gene, and the bottom lines are from the centromere-proximal flanking region (highlighted in pink) usually containing a portion of the CIC1-RUF5 intergenic region. For each comparison, the flanking sequences are from the same strain that contained the repeats. The locations of junctions J1-J5, as well as the locations of the flanking sequences, are shown in Figure 2. Sequence matches of the junction sequences to the centromere-distal and centromere-proximal flanking sequences are highlighted in blue and pink, respectively. Bases that are shared homologies of the flanking sequences are shown in boxes. Bases that do not match either flanking sequence or that match both flanking sequences are not highlighted. (A) J1 junction sequences of Type 1 repeats (S288c). As indicated, the T-tract is seven bp longer in the repeat than in the flanking sequences. There is an A/T base pair at the junction that is not derived from either of the flanking sequences. (B) J2 junction sequences (YJM189 and YJM996). (C) J3 junction sequences (YJM789). There is one SNP distinguishing the repeat sequence and that of the centromere-distal flanking sequence (base that is not highlighted within the highlighted region). (D) J4 junction sequences (YJM271, YJM1307). (E) J5 junction sequences (YJM456). The strain YJM769 had the identical sequence as YJM456 except for the presence of one SNP (Table S8 and Table S9)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequences at the breakpoints for five different types of &lt;em&gt;CUP1&lt;/em&gt; repeats. Breakpoint sequences of five different types of &lt;em&gt;CUP1&lt;/em&gt; repeats are shown in comparison to the sequences flanking the repeats. The breakpoints of each repeat are labeled &#x201C;Junctions 1&#x2212;5,&#x201D; and are the middle lines of each comparison. The top lines of each comparison (highlighted in blue) represent sequences from the centromere-distal flanking region, usually including a portion of the &lt;em&gt;RSC30&lt;/em&gt; gene, and the bottom lines are from the centromere-proximal flanking region (highlighted in pink) usually containing a portion of the &lt;em&gt;CIC1-RUF5&lt;/em&gt; intergenic region. For each comparison, the flanking sequences are from the same strain that contained the repeats. The locations of junctions J1-J5, as well as the locations of the flanking sequences, are shown in Figure 2. Sequence matches of the junction sequences to the centromere-distal and centromere-proximal flanking sequences are highlighted in blue and pink, respectively. Bases that are shared homologies of the flanking sequences are shown in boxes. Bases that do not match either flanking sequence or that match both flanking sequences are not highlighted. (A) J1 junction sequences of Type 1 repeats (S288c). As indicated, the T-tract is seven bp longer in the repeat than in the flanking sequences. There is an A/T base pair at the junction that is not derived from either of the flanking sequences. (B) J2 junction sequences (YJM189 and YJM996). (C) J3 junction sequences (YJM789). There is one SNP distinguishing the repeat sequence and that of the centromere-distal flanking sequence (base that is not highlighted within the highlighted region). (D) J4 junction sequences (YJM271, YJM1307). (E) J5 junction sequences (YJM456). The strain YJM769 had the identical sequence as YJM456 except for the presence of one SNP (Table S8 and Table S9).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F4.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/11/2259/F4.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465135" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Sequences at the breakpoints for five different types of <em>CUP1</em> repeats. Breakpoint sequences of five different types of <em>CUP1</em> repeats are shown in comparison to the sequences flanking the repeats. The breakpoints of each repeat are labeled “Junctions 1−5,” and are the middle lines of each comparison. The top lines of each comparison (highlighted in blue) represent sequences from the centromere-distal flanking region, usually including a portion of the <em>RSC30</em> gene, and the bottom lines are from the centromere-proximal flanking region (highlighted in pink) usually containing a portion of the <em>CIC1-RUF5</em> intergenic region. For each comparison, the flanking sequences are from the same strain that contained the repeats. The locations of junctions J1-J5, as well as the locations of the flanking sequences, are shown in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>. Sequence matches of the junction sequences to the centromere-distal and centromere-proximal flanking sequences are highlighted in blue and pink, respectively. Bases that are shared homologies of the flanking sequences are shown in boxes. Bases that do not match either flanking sequence or that match both flanking sequences are not highlighted. (A) J1 junction sequences of Type 1 repeats (S288c). As indicated, the T-tract is seven bp longer in the repeat than in the flanking sequences. There is an A/T base pair at the junction that is not derived from either of the flanking sequences. (B) J2 junction sequences (YJM189 and YJM996). (C) J3 junction sequences (YJM789). There is one SNP distinguishing the repeat sequence and that of the centromere-distal flanking sequence (base that is not highlighted within the highlighted region). (D) J4 junction sequences (YJM271, YJM1307). (E) J5 junction sequences (YJM456). The strain YJM769 had the identical sequence as YJM456 except for the presence of one SNP (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS8.pdf">Table S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS9.pdf">Table S9</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The J1 junction has only two sequence differences that are not predicted from a simple fusion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> and the intergenic region of <em>CIC1-RUF5</em>: an A at the fusion breakpoint and seven T residues inserted into the poly T tract in the region upstream of <em>RUF5</em>. The length of the Type 1 repeat is 1998 bp (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Johnston <em>et al.</em> 1994</a>). It should be mentioned that it can be confusing to use SGD coordinates to describe the repeats, since the sequences within the repeats match to two sets of coordinates, and the coordinates of the flanking sequences are displaced by about two kb (the length of the Type 1 repeat).</p><p id="p-29">We sequenced the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats in eight strains including one Type 2 (YJM189), two Type 3 (YJM789, YJM969), two Type 4 (YJM271, YJM307), two Type 5 (YJM456, YJM996) strains, and one strain (DTY3) with a single copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>. The strain S288c, which has a Type 1 repeat, was used for sequence comparisons by BLAST analysis (<a href="http://www.yeastgenome.org/cgi-bin/seqTools">http://www.yeastgenome.org/cgi-bin/seqTools</a>). For those strains that were also examined by deep sequencing, the sequences were identical to those obtained from the PCR products (P. K. Strope, D. A. Skelly, S. G. Kozmin, G. Mahadevan, E. A. Stone, P. M. Magwene, F. S. Dietrich, and J. H. McCusker, personal communication). The details of the sequence analysis, and the resulting sequences are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS9.pdf">Table S9</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS10.pdf">Table S10</a>. In <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS9.pdf">Table S9</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS10.pdf">Table S10</a>, we included about 600 bp of sequences flanking the repeats, in addition to the sequences of the repeats themselves.</p><p id="p-30">Based on the sequence analysis, the repeats of each type are depicted in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>. In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, B−E, we show the junction sequences (middle line) compared with sequences flanking the repeat near <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> (top line) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001094">CIC1</a></em> (bottom line). As observed for the Type 1 repeat, the Type 2−5 junctions show little (≤ 3 bp) sequence homology at the breakpoints. This observation argues strongly that the mechanism that generated the repeats likely involves nonhomologous end-joining (details in Discussion).</p><p id="p-31">We also compared the sequences of all eight strains described above to the sequence of S288c. There were very few differences, and these differences are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS9.pdf">Table S9</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS10.pdf">Table S10</a>. Two of the strains (YJM789 and DTY3) had a C to T change in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> coding sequence (SGD coordinate 212556); this change does not alter an amino acid. In strains YJM789, YJM456, YJM969, and DTY3, the termination codon of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> was UAA instead of UGA, the termination codon for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> in S288c and the other strains.</p></div><div id="sec-10" class="subsection"><h3>Variation in the number of <em>CUP1</em> repeats in different yeast strains</h3><p id="p-32">Variation in the number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats in tandem arrays of different yeast strains has been observed previously by Southern analysis (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Welch <em>et al.</em> 1983</a>). In addition, in the analysis of the 100-genome strains, estimates of the numbers of repeats per strain were made based on the number of times <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> sequences were present relative to single-copy sequences (P. K. Strope, D. A. Skelly, S. G. Kozmin, G. Mahadevan, E. A. Stone, P. M. Magwene, F. S. Dietrich, and J. H. McCusker, personal communication). We used Southern analysis to determine the number of repeats in 10 strains that were part of the 100-genome strain analysis, as well as three other yeast strains (S288c, W303-1A, and YJM789). For this analysis, genomic DNA was treated with <em>Eco</em>RI which does not have a recognition sequence within the repeats (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a>). Southern analysis of the strains is depicted in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Southern analysis of the CUP1 genes in 13 yeast strains. Genomic DNA from each strain was treated with EcoRI. There are no recognition sites for EcoRI within the CUP1 repeats (Figure 2A). The fragments were separated by gel electrophoresis, transferred to a membrane, and hybridized to a probe containing the CUP1 sequences. The lanes labeled S1 and S2 on the left side of the figure are ethidium bromide-stained fragments representing size standards (Hyperladders VI and I from Bioline). S2 in the gel on the right side of the figure is the same ladder hybridized to a ladder-specific probe. The samples in lanes 1&#x2212;13 are: 1 (YJM189), 2 (YJM271), 3 (YJM456), 4 (YJM693), 5 (YJM969), 6 (YJM972), 7 (YJM978), 8 (YJM996), 9 (YJM1549), 10 (YJM1307), 11 (S288c), 12 (W303-1A), and 13 (YJM789). The fragment sizes in lanes 1&#x2212;13 are in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Southern analysis of the &lt;em&gt;CUP1&lt;/em&gt; genes in 13 yeast strains. Genomic DNA from each strain was treated with &lt;em&gt;Eco&lt;/em&gt;RI. There are no recognition sites for &lt;em&gt;Eco&lt;/em&gt;RI within the &lt;em&gt;CUP1&lt;/em&gt; repeats (Figure 2A). The fragments were separated by gel electrophoresis, transferred to a membrane, and hybridized to a probe containing the &lt;em&gt;CUP1&lt;/em&gt; sequences. The lanes labeled S1 and S2 on the left side of the figure are ethidium bromide-stained fragments representing size standards (Hyperladders VI and I from Bioline). S2 in the gel on the right side of the figure is the same ladder hybridized to a ladder-specific probe. The samples in lanes 1&#x2212;13 are: 1 (YJM189), 2 (YJM271), 3 (YJM456), 4 (YJM693), 5 (YJM969), 6 (YJM972), 7 (YJM978), 8 (YJM996), 9 (YJM1549), 10 (YJM1307), 11 (S288c), 12 (W303-1A), and 13 (YJM789). The fragment sizes in lanes 1&#x2212;13 are in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F5.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/11/2259/F5.medium.gif" width="400" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465193" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Southern analysis of the <em>CUP1</em> genes in 13 yeast strains. Genomic DNA from each strain was treated with <em>Eco</em>RI. There are no recognition sites for <em>Eco</em>RI within the <em>CUP1</em> repeats (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2A</a>). The fragments were separated by gel electrophoresis, transferred to a membrane, and hybridized to a probe containing the <em>CUP1</em> sequences. The lanes labeled S1 and S2 on the left side of the figure are ethidium bromide-stained fragments representing size standards (Hyperladders VI and I from Bioline). S2 in the gel on the right side of the figure is the same ladder hybridized to a ladder-specific probe. The samples in lanes 1−13 are: 1 (YJM189), 2 (YJM271), 3 (YJM456), 4 (YJM693), 5 (YJM969), 6 (YJM972), 7 (YJM978), 8 (YJM996), 9 (YJM1549), 10 (YJM1307), 11 (S288c), 12 (W303-1A), and 13 (YJM789). The fragment sizes in lanes 1−13 are in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Based on the size of observed fragment, the sizes of the repeats, and the location of the flanking <em>Eco</em>RI sites, we calculated the number of repeats in each strain (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This number was based on three independent measurements. In the strains examined, the number of repeats varied between two copies (YJM978) and 18 copies (YJM1549). There is a strong correlation (R<sup>2</sup> = 0.76, <em>P</em> = 0.001) between the copy numbers as determined by Southern analysis and the copy numbers as estimated by coverage in the deep-sequencing experiments (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Although the agreement in the estimates of the number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> copies is reasonably good, for several of the strains, there were significant differences. These differences could reflect limitations of the methods used to estimate copy number or unselected alterations in the number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes in independent isolates of the same strain.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465227/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465227/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Analysis of <em>CUP1</em> gene tandem arrays in 14 yeast strains of <em>Saccharomyces cerevisiae</em></span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between different assays of CUP1 copy-number and between CUP1 copy-number and copper resistance. (A) Correlation between different assays of CUP1 copy-number. From the information in Table 1, we found a strong correlation (R2 = 0.76) between copy-number as estimated by coverage in deep-sequencing experiments (Y-axis) vs. copy-number estimated by Southern analysis (X-axis). (B) Correlation between CUP1 copy-number (Southern analysis) and the minimal concentration of copper that inhibits growth for diploid strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlations between different assays of &lt;em&gt;CUP1&lt;/em&gt; copy-number and between &lt;em&gt;CUP1&lt;/em&gt; copy-number and copper resistance. (A) Correlation between different assays of &lt;em&gt;CUP1&lt;/em&gt; copy-number. From the information in Table 1, we found a strong correlation (R&lt;sup&gt;2&lt;/sup&gt; = 0.76) between copy-number as estimated by coverage in deep-sequencing experiments (Y-axis) &lt;em&gt;vs.&lt;/em&gt; copy-number estimated by Southern analysis (X-axis). (B) Correlation between &lt;em&gt;CUP1&lt;/em&gt; copy-number (Southern analysis) and the minimal concentration of copper that inhibits growth for diploid strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F6.medium.gif" width="264" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/11/2259/F6.medium.gif" width="264" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465159" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Correlations between different assays of <em>CUP1</em> copy-number and between <em>CUP1</em> copy-number and copper resistance. (A) Correlation between different assays of <em>CUP1</em> copy-number. From the information in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, we found a strong correlation (R<sup>2</sup> = 0.76) between copy-number as estimated by coverage in deep-sequencing experiments (Y-axis) <em>vs.</em> copy-number estimated by Southern analysis (X-axis). (B) Correlation between <em>CUP1</em> copy-number (Southern analysis) and the minimal concentration of copper that inhibits growth for diploid strains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Copper resistance of strains with different types and different numbers of <em>CUP1</em> repeats</h3><p id="p-41">To determine whether there is a significant positive correlation between the number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes in the tandem array and the copper resistance of different strains, we measured the minimal concentration of copper that prevented growth (details in the section <em>Materials and Methods</em>). The analysis of copper resistance was done in the same strains that were used for the Southern analysis (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). We determined the correlation between copper resistance and the number of repeats for the ten diploid strains analyzed (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). The correlation is significant (<em>P</em> &lt; 0.02) with R<sup>2</sup> = 0.52. Qualitatively similar results were observed previously (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Fogel <em>et al.</em> 1983</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Welch <em>et al.</em> 1983</a>; P. K. Strope, D. A. Skelly, S. G. Kozmin, G. Mahadevan, E. A. Stone, P. M. Magwene, F. S. Dietrich, and J. H. McCusker, personal communication). When we included the haploid strains (S288c, W303-1A, YJM789, and DTY3) with the diploid strains, the correlation between copper resistance and copy number was higher (R<sup>2</sup> = 0.64; <em>P</em> &lt; 0.001).</p></div><div id="sec-12" class="subsection"><h3>Rate of reduction of a tandem <em>CUP1</em> array to a single copy of <em>CUP1</em></h3><p id="p-42">Although we are primarily concerned with the mechanisms by which single-copy genes become duplicated, a related issue is the mechanism by which a tandem array is reduced to a single-copy gene. We constructed a haploid YZ22 in which a tandem array of approximately 14 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats has an integrated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene. Unequal crossing (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) or several types of homologous recombination events not depicted (intrachromatid “pop-out” recombination, and single-strand annealing) can result in a single-copy <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> locus. Derivatives of YZ22 that lose the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> insertion can be identified by selection on solid medium containing 5-fluoro-orotate (details in Materials and Methods). From measurements of the frequency of 5-FOA<sup>R</sup> derivatives in multiple independent cultures, we calculated the rate of appearance of 5-FOA<sup>R</sup> derivatives to be 1.7 × 10<sup>−5</sup>/division.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of a CUP1 tandem array by unequal crossing-over. We depict the CUP1 array of the haploid strain YZ22 that contains an inserted URA3 gene within the array. An unequal crossover between the terminal CUP1 repeat of one array with opposite terminus of another array located on a sister chromatid results in one cell with an elongated array with two URA3 insertions and another cell with a single CUP1 repeat and no URA3 insertion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss of a &lt;em&gt;CUP1&lt;/em&gt; tandem array by unequal crossing-over. We depict the &lt;em&gt;CUP1&lt;/em&gt; array of the haploid strain YZ22 that contains an inserted &lt;em&gt;URA3&lt;/em&gt; gene within the array. An unequal crossover between the terminal &lt;em&gt;CUP1&lt;/em&gt; repeat of one array with opposite terminus of another array located on a sister chromatid results in one cell with an elongated array with two &lt;em&gt;URA3&lt;/em&gt; insertions and another cell with a single &lt;em&gt;CUP1&lt;/em&gt; repeat and no &lt;em&gt;URA3&lt;/em&gt; insertion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F7.medium.gif" width="402" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/11/2259/F7.medium.gif" width="402" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465244" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-43" class="first-child">Loss of a <em>CUP1</em> tandem array by unequal crossing-over. We depict the <em>CUP1</em> array of the haploid strain YZ22 that contains an inserted <em>URA3</em> gene within the array. An unequal crossover between the terminal <em>CUP1</em> repeat of one array with opposite terminus of another array located on a sister chromatid results in one cell with an elongated array with two <em>URA3</em> insertions and another cell with a single <em>CUP1</em> repeat and no <em>URA3</em> insertion.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">To determine what fraction of these derivatives had reduced the array to a single copy, we examined 135 independent 5-FOA<sup>R</sup> derivatives by PCR using primers from within the flanking <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001094">CIC1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> genes (primers VIII211849 F and VIII216603 R, respectively). In strains with a single copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>, a PCR fragment of 2.7 kb was observed. 6 of the 135 strains had a single copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>. Thus, the approximate rate for reduction of the tandem array of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a> genes to a single copy is 6/135 × 1.7 × 10<sup>−5</sup> or 7.6 × 10<sup>−7</sup>/division.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-45">Tandemly repeated genes are a common feature of eukaryotic genomes. These genes often encode proteins or RNA species (such as ribosomal RNA) that are required in large amounts. As discussed in the <em>Introduction</em>, the homologous recombination mechanisms that alter the number of repeats in a pre-existing tandem array are well characterized, and include unequal crossing-over, intra-chromatid crossovers, gene conversion events, and single-strand annealing. The events that generate a tandem duplication of a single-copy gene are less understood. From our analysis, we argue that this duplication event at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> locus likely reflects the joining of two broken ends by a non-homologous end-joining event.</p><div id="sec-14" class="subsection"><h3>Previous observations of gene duplications in yeast</h3><p id="p-46">In yeast, the number of repeats in the ribosomal RNA gene cluster alters at a high rate, 10<sup>−1</sup>/meiotic division (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Petes 1980</a>) and 10<sup>−2</sup>/mitotic division (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Szostak and Wu 1980</a>). In general, the rates of generating <em>de novo</em> duplications are much less, although the rates are variable depending on the context of the reporter gene and the details of the experimental system. If the reporter gene is flanked by repeated elements, the duplication often occurs by unequal crossovers between these elements. For example, <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> (2013)</a> showed that duplication of a reporter construct located between two Ty elements on chromosome V in a haploid strain was a consequence of unequal crossovers, occurring at a frequency of about 10<sup>−6</sup>−10<sup>−7</sup>. No events in which the duplication was generated by non-homologous end-joining were observed. In strains in which re-replication of an origin was induced, gene duplications of sequences within the re-replicated region were efficiently generated (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Green <em>et al.</em> 2010</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Finn and Li 2013</a>). These events usually involved nonallelic homologous recombination between Ty elements flanking the duplicated region.</p><p id="p-47">Haploid yeast strains with a deletion of the ribosomal protein gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004855">RPL20A</a></em> grow slowly, allowing for selection of fast-growing derivatives that duplicate the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004855">RPL20A</a></em>-related gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005839">RPL20B</a></em> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Koszul <em>et al.</em> 2004</a>). These duplications involved both flanking repeated genes (delta and Ty elements) and microhomologies. The events involving recombination between large (&gt;300 bp) regions of homology were Rad52p-dependent, and all classes of duplications had a requirement for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003804">Pol32p</a> (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Payen <em>et al.</em> 2008</a>). Based on the genetic analysis, <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Payen <em>et al.</em> (2008)</a> argued that many of the Rad52p-independent duplications were generated by microhomology-mediated BIR. The rates of duplications of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005839">RPL20B</a></em> were estimated at between 10<sup>−7</sup>−10<sup>−10</sup>, depending on what correction factor was used to determine the relative growth rates of strains with and without the duplication (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Koszul <em>et al.</em> 2004</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Payen <em>et al.</em> 2008</a>). The <em>de novo <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005839">RPL20B</a></em> repeats were large, varying between 41 and 655 kb (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Koszul <em>et al.</em> 2004</a>). In experiments selecting for duplications of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> gene, only one large (&gt;100 kb) chromosomal duplication was observed, yielding a rate of about 10<sup>−10</sup>/cell division (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dorsey <em>et al.</em> 1992</a>).</p><p id="p-48">Although it is difficult to reach a definitive conclusion based on these data, most of the yeast observations suggest that duplications that arise as a consequence of homologous recombination between flanking repeated sequences occur more frequently than those generated by non-homologous end-joining events. In contrast, many duplications in mammalian cells are produced by nonhomologous end-joining or related mechanisms (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> 2009</a>). These differences are consistent with the relative importance of homologous recombination <em>vs.</em> nonhomologous end-joining in yeast compared with mammals (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jasin and Rothstein 2013</a>).</p></div><div id="sec-15" class="subsection"><h3>Duplications of <em>CUP1</em> in yeast</h3><p id="p-49">The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats are located on the right arm of chromosome VIII, and there are no flanking Ty elements on this arm in S288c. In strains with tandemly repeated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes, <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Fogel <em>et al.</em> (1983)</a> were readily able to select derivatives with longer arrays by growing the strains in high concentrations of copper. They were not, however, able to isolate strains with a <em>de novo</em> duplication of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> from a strain that had a single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> copy. Their results argue that such duplications are infrequent when <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> is in its “normal” location. Our analysis of different types of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats, however, suggest that such duplications occur during evolution.</p><p id="p-50">All five classes of repeats can be explained by the duplication mechanism shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>. We suggest that, during replication of a chromosome with only one copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>, two breaks occur, one centromere-distal and one centromere-proximal to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> locus. The joining of these broken ends by nonhomologous end-joining results in two products, one with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> deletion and one with a tandem duplication. An alternative possibility is that the repeats were generated by a single break in the centromere-distal location, followed by a BIR event in which the centromere-proximal site was invaded. Because most BIR events involve either extensive sequence homology (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Paques and Haber 1999</a>) or 5−20 bp of microhomology (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Payen <em>et al.</em> 2008</a>), and most of the observed breakpoints in our study have very little or no homology, we favor the first alternative.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of a de novo duplication of CUP1 by unequal nonhomologous end-joining. A mechanism for the generation of a two-copy tandem array of CUP1 from a single CUP1 gene (similar to the duplication observed in S288c) is shown. We suggest that, during DNA replication, one fork is broken in the intergenic region between CIC1 and RUF5/CUP1/ARS810. The second fork is broken within RSC30. The nonhomologous end-joining of one broken end to the other (indicated by the dashed line) would produce the duplication. If the other broken ends are also joined, a deletion of RUF5/CUP1/ARS810 would be produced in the sister chromatid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of a &lt;em&gt;de novo&lt;/em&gt; duplication of &lt;em&gt;CUP1&lt;/em&gt; by unequal nonhomologous end-joining. A mechanism for the generation of a two-copy tandem array of &lt;em&gt;CUP1&lt;/em&gt; from a single &lt;em&gt;CUP1&lt;/em&gt; gene (similar to the duplication observed in S288c) is shown. We suggest that, during DNA replication, one fork is broken in the intergenic region between &lt;em&gt;CIC1&lt;/em&gt; and &lt;em&gt;RUF5/CUP1/ARS810&lt;/em&gt;. The second fork is broken within &lt;em&gt;RSC30&lt;/em&gt;. The nonhomologous end-joining of one broken end to the other (indicated by the dashed line) would produce the duplication. If the other broken ends are also joined, a deletion of &lt;em&gt;RUF5/CUP1/ARS810&lt;/em&gt; would be produced in the sister chromatid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F8.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/11/2259/F8.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-51" class="first-child">Generation of a <em>de novo</em> duplication of <em>CUP1</em> by unequal nonhomologous end-joining. A mechanism for the generation of a two-copy tandem array of <em>CUP1</em> from a single <em>CUP1</em> gene (similar to the duplication observed in S288c) is shown. We suggest that, during DNA replication, one fork is broken in the intergenic region between <em>CIC1</em> and <em>RUF5/CUP1/ARS810</em>. The second fork is broken within <em>RSC30</em>. The nonhomologous end-joining of one broken end to the other (indicated by the dashed line) would produce the duplication. If the other broken ends are also joined, a deletion of <em>RUF5/CUP1/ARS810</em> would be produced in the sister chromatid.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">In a previous study, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Haber and Louis (1998)</a> noted that minisatellites in yeast were often flanked by 5−10 bp direct repeats, and suggested that these short repeats could be substrates for replication slippage or unequal crossovers. Although the repeated sequences at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> junctions are three bp or less, it is possible that the initial duplication involved longer direct repeats but sequence alterations accumulated subsequent to the duplication. One argument against this possibility is that the sequences flanking the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats are highly conserved. From <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS8.pdf">Table S8</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1/TableS9.pdf">Table S9</a>, we calculate that the average sequence divergence of the centromere-proximal 600 bp flanking the repeats (relative to the standard SGD sequence) is 0.7%. The centromere-distal sequences are similarly well conserved (0.4% divergence). Finally, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats show only 0.5% sequence divergence.</p><p id="p-53">Although we favor the model in which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats are generated by “classic” NHEJ, this model does not explain the A/T base pair that is present at the junction of the Type 1 repeats. In an analysis of NHEJ in Drosophila, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yu and McVey (2010)</a> observed joining events in which one or more bases were inserted at the junction. Their observations suggested that these insertions were a consequence of pairing interactions between short repeats, followed by limited DNA synthesis prior to the joining event. Although we do not need to invoke this model for most of the repeats, it is possible that the Type 1 repeats were generated by this mechanism (termed “synthesis-dependent microhomology-mediated end joining”). Alternatively, the A/T base pair may represent a mutation produced subsequent to the original NHEJ-generated duplication.</p><p id="p-54">The differences in the location of the breaks generated repeats that vary in size between about 1.2 kb and 2 kb. The centromere-proximal breaks occurred in two locations (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). Most were in the intergenic region between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001094">CIC1</a></em> and <em>RUF5</em>, but one was within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001094">CIC1</a></em> gene. The centromere-distal breaks occurred either between <em>RUF5</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em>, or within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> gene.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Clustered breakpoints of five different types of CUP1 repeats. (A) Location of junction breakpoints in the CIC1 gene and intergenic region for five types of CUP1 repeats. (B) Sequence in the region of three tightly clustered breakpoints in the CIC1-RUF5/CUP1/ARS intergenic region. (C) Location of junction breakpoints in the RSC30 gene for five types of repeats. (D) Sequence in the region of two tightly clustered breakpoints within RSC30." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1954192698" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Clustered breakpoints of five different types of &lt;em&gt;CUP1&lt;/em&gt; repeats. (A) Location of junction breakpoints in the &lt;em&gt;CIC1&lt;/em&gt; gene and intergenic region for five types of &lt;em&gt;CUP1&lt;/em&gt; repeats. (B) Sequence in the region of three tightly clustered breakpoints in the &lt;em&gt;CIC1-RUF5/CUP1/ARS&lt;/em&gt; intergenic region. (C) Location of junction breakpoints in the &lt;em&gt;RSC30&lt;/em&gt; gene for five types of repeats. (D) Sequence in the region of two tightly clustered breakpoints within &lt;em&gt;RSC30&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2259/F9.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/4/11/2259/F9.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2259/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465275" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-55" class="first-child">Clustered breakpoints of five different types of <em>CUP1</em> repeats. (A) Location of junction breakpoints in the <em>CIC1</em> gene and intergenic region for five types of <em>CUP1</em> repeats. (B) Sequence in the region of three tightly clustered breakpoints in the <em>CIC1-RUF5/CUP1/ARS</em> intergenic region. (C) Location of junction breakpoints in the <em>RSC30</em> gene for five types of repeats. (D) Sequence in the region of two tightly clustered breakpoints within <em>RSC30</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">The locations of the presumptive DSBs required to generate the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> duplications could reflect chromosomal sequences that are intrinsically susceptible to breakage (fragile sites). In yeast, DNA sequences with the ability to form secondary structures (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Voineagu <em>et al.</em> 2008</a>) or to perturb the progression of replication forks (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cha and Kleckner 2002</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Paeschke <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Song <em>et al.</em> 2014</a>) are hotspots for chromosome breakage.</p><p id="p-57">The minimal size of the duplication is constrained by the sequences required for optimal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> expression. Aside from the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> coding sequence of about 200 bp, optimal expression of the gene and copper-induced transcription requires about 300 bp of upstream sequences (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Thiele and Hamer 1986</a>). Since the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> transcript is about 500 bp in size (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Karin <em>et al.</em> 1984</a>), the minimal size of a functional <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeat is expected to be less than 600 bp. However, all of the repeats share sequences from the 3′ end of <em>RUF5</em> to the beginning of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> gene, a region of about 920 bp. It is possible that these additional sequences are involved in regulating <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene expression in some environments or enhancing the gene amplification process subsequent to the duplication. It is interesting that none of the duplications include an intact <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001094">CIC1</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> gene. <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Koszul <em>et al.</em> (2004)</a> found that duplications of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005839">RPL20B</a></em> gene often included many flanking genes. One obvious possibility is that yeast cells are intolerant of extra doses of either <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001094">CIC1</a></em> (encoding an essential protein associated with the proteasome; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jäger <em>et al.</em> 2001</a>) or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001098">RSC30</a></em> (encoding a protein involved in chromatin re-modeling; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Angus-Hill <em>et al.</em> 2001</a>).</p></div><div id="sec-16" class="subsection"><h3>Alterations in the number of repeats within the <em>CUP1</em> tandem array</h3><p id="p-58">Several lines of evidence demonstrate that the number of repeats per tandem array alters at high frequency. First, as described previously, <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Fogel <em>et al.</em> (1983)</a> showed that strains with longer arrays of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> could be readily isolated by growing the cells in high levels of copper. Second, isolates of yeast obtained from the wild have different numbers of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes in their arrays (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Welch <em>et al.</em> 1983</a>). In the strains examined in our study, the number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> genes varied from one to eighteen (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Third, in several different mutant backgrounds, the rate of alterations in the number of repeats is very high, greater than 10<sup>−3</sup>/division (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">McCulley and Petes 2010</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Song <em>et al.</em> 2014</a>). Because loss or duplication of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats occur in integral numbers of units, these events are presumably the result of homologous recombination (unequal crossovers, single-strand annealing, or gene conversion).</p><p id="p-59">About 30% of the 100-genome strains had only one copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>. Such strains could represent the progenitor isolates from which the tandem arrays were derived. Alternatively, these isolates could represent strains in which all copies except one were lost as a consequence of homologous recombination. It is important to stress that the process by which a single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeat becomes duplicated is inherently different than the process by which a tandem array becomes deleted to a single copy. If the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene is not associated with repeated flanking elements, the duplication process likely occurs by some type of nonhomologous end-joining or a BIR event involving microhomologies. The latter type of event can occur by homologous recombination between the terminal repeats of the array.</p><p id="p-60">Based on our study and a number of previous studies, we argue that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> tandem arrays arise as a consequence of two mechanisms: 1) a duplication of a single-copy <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> gene by an infrequent nonhomologous recombination event and 2) increases in copy number from the duplication to multiple copies by a frequent homologous recombination event. In environments with high levels of copper, strains with larger numbers of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats will be selected. Our observation that a tandem array of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> repeats can be reduced to a single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em> copy emphasizes the necessity of a mechanism for generating <em>de novo</em> duplications.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">We thank D. Thiel for the yeast strain DTY3, and Y. Yin, S. Andersen, and S. Jinks-Robertson for comments on the manuscript. The research was supported by National Institutes of Health grants GM24110 and GM52319 to T.P. and GM098287 to J.H.M. Y.Z. and P.K.S. were supported by grants from the Army Research Office (SPS #200531), and the National Institutes of Health (T32 AI052080), respectively.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-62">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012922/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-63"><em>Communicating editor: L. M. Steinmetz</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-06-20"><span class="received-label">Received </span>June 20, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-09-11"><span class="accepted-label">Accepted </span>September 11, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Zhao <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.1" data-doi="10.1016/S1097-2765(01)00219-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angus-Hill</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlichter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erdjument-Bromage</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempst</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A Rsc3/Rsc30 zinc cluster dimer reveals novel roles for the chromatin remodeler RSC in gene expression and cell cycle control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DAngus-Hill%26rft.auinit1%253DM.%2BL.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D741%26rft.epage%253D751%26rft.atitle%253DA%2BRsc3%252FRsc30%2Bzinc%2Bcluster%2Bdimer%2Breveals%2Bnovel%2Broles%2Bfor%2Bthe%2Bchromatin%2Bremodeler%2BRSC%2Bin%2Bgene%2Bexpression%2Band%2Bcell%2Bcycle%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900219-2%26rft_id%253Dinfo%253Apmid%252F11336698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00219-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11336698&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168462800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.2" data-doi="10.1016/0076-6879(87)54076-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trueheart</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsoulis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">5-Fluoroorotic acid as a selective agent in yeast molecular genetics.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DBoeke%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D154%26rft.spage%253D164%26rft.epage%253D175%26rft.atitle%253D5-Fluoroorotic%2Bacid%2Bas%2Ba%2Bselective%2Bagent%2Bin%2Byeast%2Bmolecular%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252887%252954076-9%26rft_id%253Dinfo%253Apmid%252F3323810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(87)54076-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3323810&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987M168700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.3" data-doi="10.1126/science.1071398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">tCha</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">ATR homolog Mec1 promotes fork progression, thus averting breaks in replication slow zones.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">602</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCha%26rft.auinit1%253DR.%2BS.%26rft.volume%253D297%26rft.issue%253D5581%26rft.spage%253D602%26rft.epage%253D606%26rft.atitle%253DATR%2BHomolog%2BMec1%2BPromotes%2BFork%2BProgression%252C%2BThus%2BAverting%2BBreaks%2Bin%2BReplication%2BSlow%2BZones%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1071398%26rft_id%253Dinfo%253Apmid%252F12142538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTcvNTU4MS82MDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorsey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paquin</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Spontaneous amplification of the <em>ADH4</em> gene in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDorsey%26rft.auinit1%253DM.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D943%26rft.epage%253D950%26rft.atitle%253DSpontaneous%2Bamplification%2Bof%2Bthe%2BADH4%2Bgene%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F1459445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzQvOTQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2259.5" data-doi="10.1534/g3.113.008995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engel</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisk</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The reference genome sequence of <em>Saccharomyces cerevisiae</em>: then and now.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008995%26rft_id%253Dinfo%253Apmid%252F24374639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMy8zODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.6" data-doi="10.1371/journal.pgen.1003192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single-stranded annealing induced by re-initiation of replication origins provides a novel and efficient mechanism for generating copy number expansion via non-allelic homologous recombination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003192%26rft_id%253Dinfo%253Apmid%252F23300490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300490&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.7" data-doi="10.1073/pnas.79.17.5342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Tandem gene amplification mediates copper resistance in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">5342</span>–<span class="cit-lpage">5346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFogel%26rft.auinit1%253DS.%26rft.volume%253D79%26rft.issue%253D17%26rft.spage%253D5342%26rft.epage%253D5346%26rft.atitle%253DTandem%2Bgene%2Bamplification%2Bmediates%2Bcopper%2Bresistance%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.17.5342%26rft_id%253Dinfo%253Apmid%252F6291039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMTcvNTM0MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-2" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.8" data-doi="10.1007/BF00445874"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cathala</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Gene amplification in yeast: <em>CUP1</em> copy number regulates copper resistance.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenet.%26rft.volume%253D7%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00445874%26rft_id%253Dinfo%253Apmid%252F24173415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00445874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24173415&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.9" data-doi="10.1126/science.1190966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Loss of DNA replication control is a potent inducer of gene amplification.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreen%26rft.auinit1%253DB.%2BM.%26rft.volume%253D329%26rft.issue%253D5994%26rft.spage%253D943%26rft.epage%253D946%26rft.atitle%253DLoss%2Bof%2BDNA%2BReplication%2BControl%2BIs%2Ba%2BPotent%2BInducer%2Bof%2BGene%2BAmplification%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190966%26rft_id%253Dinfo%253Apmid%252F20724634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5NC85NDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.10" data-doi="10.1371/journal.pgen.1000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenq</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGresham%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000303%26rft.epage%253De1000303%26rft.atitle%253DThe%2Brepertoire%2Band%2Bdynamics%2Bof%2Bevolutionary%2Badaptations%2Bto%2Bcontrolled%2Bnutrient-limited%2Benvironments%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000303%26rft_id%253Dinfo%253Apmid%252F19079573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079573&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2259.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source">Guide to Yeast Genetics and Microbiology</span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.12" data-doi="10.1006/geno.1997.5153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Minisatellite origins in yeast and humans.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHaber%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D132%26rft.epage%253D135%26rft.atitle%253DMinisatellite%2Borigins%2Bin%2Byeast%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.1997.5153%26rft_id%253Dinfo%253Apmid%252F9503027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.1997.5153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9503027&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072304800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugawara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishman-Lobell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Genetic requirements for the single-strand annealing pathway of double-strand break repair in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DIvanov%26rft.auinit1%253DE.%2BL.%26rft.volume%253D142%26rft.issue%253D3%26rft.spage%253D693%26rft.epage%253D704%26rft.atitle%253DGenetic%2BRequirements%2Bfor%2Bthe%2BSingle-Strand%2BAnnealing%2BPathway%2Bof%2BDouble-Strand%2BBreak%2BRepair%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F8849880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzMvNjkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.14" data-doi="10.1093/emboj/20.16.4423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jäger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strayle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinemeyer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Cic1, an adaptor protein specifically linking the 26S proteasome to its substrate, the SCF component Cdc4.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4423</span>–<span class="cit-lpage">4431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DJager%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D16%26rft.spage%253D4423%26rft.epage%253D4431%26rft.atitle%253DCic1%252C%2Ban%2Badaptor%2Bprotein%2Bspecifically%2Blinking%2Bthe%2B26S%2Bproteasome%2Bto%2Bits%2Bsubstrate%252C%2Bthe%2BSCF%2Bcomponent%2BCdc4%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.16.4423%26rft_id%253Dinfo%253Apmid%252F11500370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTYvNDQyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.15" data-doi="10.1101/cshperspect.a012740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jasin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Repair of strand breaks by homologous recombination.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a012740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a012740%26rft_id%253Dinfo%253Apmid%252F24097900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjEyOiI1LzExL2EwMTI3NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.16" data-doi="10.1126/science.8091229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Complete nucleotide sequence of <em>Saccharomyces cerevisiae</em> chromosome VIII.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">2077</span>–<span class="cit-lpage">2082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJohnston%26rft.auinit1%253DM%26rft.volume%253D265%26rft.issue%253D5181%26rft.spage%253D2077%26rft.epage%253D2082%26rft.atitle%253DComplete%2Bnucleotide%2Bsequence%2Bof%2BSaccharomyces%2Bcerevisiae%2Bchromosome%2BVIII%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8091229%26rft_id%253Dinfo%253Apmid%252F8091229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjUvNTE4MS8yMDc3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.17" data-doi="10.1073/pnas.81.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haslinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valenzuela</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Primary structure and transcription of an amplified genetic locus: the <em>CUP1</em> locus of yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKarin%26rft.auinit1%253DM.%26rft.volume%253D81%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPrimary%2Bstructure%2Band%2Btranscription%2Bof%2Ban%2Bamplified%2Bgenetic%2Blocus%253A%2Bthe%2BCUP1%2Blocus%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.2.337%26rft_id%253Dinfo%253Apmid%252F6364141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4MS8yLzMzNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.18" data-doi="10.1038/sj.emboj.7600024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caburet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Eucaryotic genome evolution through the spontaneous duplication of large chromosomal segments.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKoszul%26rft.auinit1%253DR.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D234%26rft.epage%253D243%26rft.atitle%253DEucaryotic%2Bgenome%2Bevolution%2Bthrough%2Bthe%2Bspontaneous%2Bduplication%2Bof%2Blarge%2Bchromosomal%2Bsegments%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600024%26rft_id%253Dinfo%253Apmid%252F14685272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMy8xLzIzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.19" data-doi="10.1007/BF02986080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lea</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The distribution of the numbers of mutants in bacterial populations.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D49%26rft.spage%253D264%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02986080%26rft_id%253Dinfo%253Apmid%252F24536673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02986080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24536673&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1949XX17200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.20" data-doi="10.1016/j.cell.2007.11.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A DNA replication mechanism for generating nonrecurrent rearrangements associated with genomic disorders.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLee%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D7%26rft.spage%253D1235%26rft.epage%253D1247%26rft.atitle%253DA%2BDNA%2Breplication%2Bmechanism%2Bfor%2Bgenerating%2Bnonrecurrent%2Brearrangements%2Bassociated%2Bwith%2Bgenomic%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.11.037%26rft_id%253Dinfo%253Apmid%252F18160035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.11.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18160035&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.21" data-doi="10.1073/pnas.1006281107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCulley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Chromosome rearrangements and aneuploidy in yeast strains lacking both Tel1p and Mec1p reflect deficiencies in two different mechanisms.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">11465</span>–<span class="cit-lpage">11470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006281107%26rft_id%253Dinfo%253Apmid%252F20534547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzI1LzExNDY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.22" data-doi="10.1093/nar/gkg438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCutcheon</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Computational identification of non-coding RNAs in <em>Saccharomyces cerevisiae</em> by comparative genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">4119</span>–<span class="cit-lpage">4128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMcCutcheon%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D31%26rft.issue%253D14%26rft.spage%253D4119%26rft.epage%253D4128%26rft.atitle%253DComputational%2Bidentification%2Bof%2Bnon-coding%2BRNAs%2Bin%2BSaccharomyces%2Bcerevisiae%2Bby%2Bcomparative%2Bgenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg438%26rft_id%253Dinfo%253Apmid%252F12853629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xNC80MTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.23" data-doi="10.1016/j.cell.2011.04.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paeschke</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakian</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">DNA replication through G-quadruplex motifs is promoted by the <em>Saccharomyces cerevisiae</em> Pif1 DNA helicase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">678</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPaeschke%26rft.auinit1%253DK.%26rft.volume%253D145%26rft.issue%253D5%26rft.spage%253D678%26rft.epage%253D691%26rft.atitle%253DDNA%2Breplication%2Bthrough%2BG-quadruplex%2Bmotifs%2Bis%2Bpromoted%2Bby%2Bthe%2BSaccharomyces%2Bcerevisiae%2BPif1%2BDNA%2Bhelicase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.04.015%26rft_id%253Dinfo%253Apmid%252F21620135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.04.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21620135&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291018600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paques</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple pathways of recombination induced by double-strand breaks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DPaques%26rft.auinit1%253DF.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D349%26rft.epage%253D404%26rft.atitle%253DMultiple%2BPathways%2Bof%2BRecombination%2BInduced%2Bby%2BDouble-Strand%2BBreaks%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F10357855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2My8yLzM0OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.25" data-doi="10.1371/journal.pgen.1000175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segmental duplications arise from Pol32-dependent repair of broken forks through two alternative replication-based mechanisms.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPayen%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000175%26rft.epage%253De1000175%26rft.atitle%253DSegmental%2Bduplications%2Barise%2Bfrom%2BPol32-dependent%2Brepair%2Bof%2Bbroken%2Bforks%2Bthrough%2Btwo%2Balternative%2Breplication-based%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000175%26rft_id%253Dinfo%253Apmid%252F18773114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18773114&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.26" data-doi="10.1016/S0092-8674(80)80052-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Unequal meiotic recombination within tandem arrays of yeast ribosomal DNA genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPetes%26rft.auinit1%253DT.%2BD.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D765%26rft.epage%253D774%26rft.atitle%253DUnequal%2Bmeiotic%2Brecombination%2Bwithin%2Btandem%2Barrays%2Bof%2Byeast%2Bribosomal%2BDNA%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252880%252980052-3%26rft_id%253Dinfo%253Apmid%252F6988084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(80)80052-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6988084&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JK07200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-4.11.2259.27" data-doi="10.1073/pnas.1406847111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenwell</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide high-resolution mapping of chromosome fragile sites in <em>Saccharomyces cerevisiae</em></span>. <span class="cit-source">Proc. Natl. Acad. Sci. USA.</span> <span class="cit-vol">111</span>: <span class="cit-fpage">E2210</span>–<span class="cit-lpage">E2218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1406847111%26rft_id%253Dinfo%253Apmid%252F24799712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzIxL0UyMjEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.28" data-doi="10.1371/journal.pgen.1003434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Charles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of spontaneous mitotic recombination hotspots on the 1.1 Mb arm of yeast chromosome IV.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003434%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003434%26rft_id%253Dinfo%253Apmid%252F23593029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23593029&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.11.2259.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1925</span> <span class="cit-article-title">Reverse mutation of the bar gene correlated with crossing over.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D57%26rft.spage%253D746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.30" data-doi="10.1038/284426a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Unequal crossing over in the ribosomal DNA of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">426</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSzostak%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D284%26rft.issue%253D5755%26rft.spage%253D426%26rft.epage%253D430%26rft.atitle%253DUnequal%2Bcrossing%2Bover%2Bin%2Bthe%2Bribosomal%2BDNA%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F284426a0%26rft_id%253Dinfo%253Apmid%252F6987539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/284426a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6987539&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JL65400037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.31" data-doi="10.1073/pnas.90.17.8013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamai</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gralla</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellerby</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentine</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiele</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Yeast and mammalian metallothioneins functionally substitute for yeast copper-zinc superoxide dismutase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">8013</span>–<span class="cit-lpage">8017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTamai%26rft.auinit1%253DK.%2BT.%26rft.volume%253D90%26rft.issue%253D17%26rft.spage%253D8013%26rft.epage%253D8017%26rft.atitle%253DYeast%2Band%2Bmammalian%2Bmetallothioneins%2Bfunctionally%2Bsubstitute%2Bfor%2Byeast%2Bcopper-zinc%2Bsuperoxide%2Bdismutase.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.17.8013%26rft_id%253Dinfo%253Apmid%252F8367458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTAvMTcvODAxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjI1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thiele</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamer</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Tandemly duplicated upstream control sequences mediate copper-induced transcription of the <em>Saccharomyces cerevisiae</em> copper-metallothionein gene.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1158</span>–<span class="cit-lpage">1163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DThiele%26rft.auinit1%253DD%2BJ%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D1158%26rft.epage%253D1163%26rft.atitle%253DTandemly%2Bduplicated%2Bupstream%2Bcontrol%2Bsequences%2Bmediate%2Bcopper-induced%2Btranscription%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bcopper-metallothionein%2Bgene.%26rft_id%253Dinfo%253Apmid%252F3537699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjYvNC8xMTU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.33" data-doi="10.1016/0092-8674(89)90584-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Elevated recombination rates in transcriptionally active DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DThomas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D619%26rft.epage%253D630%26rft.atitle%253DElevated%2Brecombination%2Brates%2Bin%2Btranscriptionally%2Bactive%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990584-9%26rft_id%253Dinfo%253Apmid%252F2645056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90584-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2645056&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T405100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.34" data-doi="10.1073/pnas.0804510105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voineagu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narayanan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobachev</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Replication stalling at unstable inverted repeats: interplay between DNA hairpins and fork stabilizing proteins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9936</span>–<span class="cit-lpage">9941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804510105%26rft_id%253Dinfo%253Apmid%252F18632578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI5Lzk5MzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.35" data-doi="10.1073/pnas.0701291104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome sequencing and comparative analysis of <em>Saccharomyces cerevisiae</em> strain YJM789.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">12825</span>–<span class="cit-lpage">12830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701291104%26rft_id%253Dinfo%253Apmid%252F17652520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzMxLzEyODI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.36" data-doi="10.1128/MCB.3.8.1353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cathala</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Industrial yeasts display tandem gene iteration at the <em>CUP1</em> region.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWelch%26rft.auinit1%253DJ%2BW%26rft.volume%253D3%26rft.issue%253D8%26rft.spage%253D1353%26rft.epage%253D1361%26rft.atitle%253DIndustrial%2Byeasts%2Bdisplay%2Btandem%2Bgene%2Biteration%2Bat%2Bthe%2BCUP1%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.3.8.1353%26rft_id%253Dinfo%253Apmid%252F6621529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjMvOC8xMzUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.37" data-doi="10.1093/nar/gkq379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synthesis-dependent microhomology-mediated end joining accounts for multiple types of repair junctions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">5706</span>–<span class="cit-lpage">5717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq379%26rft_id%253Dinfo%253Apmid%252F20460465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNy81NzA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.38" data-doi="10.1146/annurev.genom.9.081307.164217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variation in human health, disease, and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.spage%253D451%26rft.epage%253D481%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bhuman%2Bhealth%252C%2Bdisease%252C%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164217%26rft_id%253Dinfo%253Apmid%252F19715442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715442&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2259.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2259.39" data-doi="10.1534/genetics.112.146522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeidler</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puccia</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malc</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene copy-number variation in haploid and diploid strains of the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146522%26rft_id%253Dinfo%253Apmid%252F23307895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvNzg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/11/2259.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2247" title="The DExH Box Helicase Domain of Spindle-E Is Necessary for Retrotransposon Silencing and Axial Patterning During Drosophila Oogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2271" title="Flow Cytometry of Microencapsulated Colonies for Genetics Analysis of Filamentous Fungi" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/11"><span>Volume 4 Issue 11, November 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/11.cover-source.jpg?itok=oYWFGPiM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2259" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2XWYlNbFoiwQowrItMFX-O-_xT-sdzN5lHkpL1ckf5A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/11/2259" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/11/2259" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/11/2259" class="active" data-icon-position="" data-hide-link-title="0">Structures of Naturally Evolved CUP1 Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459702" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-PZBcnXVlo0KZcUXHZZbMqV4_-eUz21sDbAD0YsY9Sz4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2259.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/11/2259" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6gGlxMpsCUHg6uVUEdX3EQaX3Uu_rz-KnJaEpMjN6pU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/11/2259" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2259.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459702" id="node459702--2" data-pisa="ggg;4/11/2259" data-pisa-master="ggg;g3.114.012922" data-apath="/ggg/4/11/2259.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Structures of Naturally Evolved <em>CUP1</em> Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying Zhao</span>, <span class="highwire-citation-author" data-delta="1">Pooja K. Strope</span>, <span class="highwire-citation-author" data-delta="2">Stanislav G. Kozmin</span>, <span class="highwire-citation-author" data-delta="3">John H. McCusker</span>, <span class="highwire-citation-author" data-delta="4">Fred S. Dietrich</span>, <span class="highwire-citation-author" data-delta="5">Robert J. Kokoska</span> and <span class="highwire-citation-author" data-delta="6">Thomas D. Petes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2259-2269;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012922 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459702--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pooja K. Strope </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pooja%2BK.%2BStrope%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strope%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APooja%2BK.%2BStrope%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stanislav G. Kozmin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stanislav%2BG.%2BKozmin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozmin%20SG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStanislav%2BG.%2BKozmin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John H. McCusker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BMcCusker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCusker%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BMcCusker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fred S. Dietrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BS.%2BDietrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dietrich%20FS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BS.%2BDietrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert J. Kokoska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Physical Sciences Directorate, U. S. Army Research Office, Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BKokoska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kokoska%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BKokoska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas D. Petes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BD.%2BPetes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petes%20TD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BD.%2BPetes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tom.petes@duke.edu" class="" data-icon-position="" data-hide-link-title="0">tom.petes@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/11/2259" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/11/2259&amp;title=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/11/2259&amp;title=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/11/2259&amp;title=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/11/2259&amp;text=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/11/2259&amp;title=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/11/2259&amp;t=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/11/2259&amp;title=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/11/2259&amp;title=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459702/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459702" id="node459702--3" data-pisa="ggg;4/11/2259" data-pisa-master="ggg;g3.114.012922" data-apath="/ggg/4/11/2259.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Structures of Naturally Evolved <em>CUP1</em> Tandem Arrays in Yeast Indicate That These Arrays Are Generated by Unequal Nonhomologous Recombination</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying Zhao</span>, <span class="highwire-citation-author" data-delta="1">Pooja K. Strope</span>, <span class="highwire-citation-author" data-delta="2">Stanislav G. Kozmin</span>, <span class="highwire-citation-author" data-delta="3">John H. McCusker</span>, <span class="highwire-citation-author" data-delta="4">Fred S. Dietrich</span>, <span class="highwire-citation-author" data-delta="5">Robert J. Kokoska</span> and <span class="highwire-citation-author" data-delta="6">Thomas D. Petes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2259-2269;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012922 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459702--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pooja K. Strope </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pooja%2BK.%2BStrope%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strope%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APooja%2BK.%2BStrope%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stanislav G. Kozmin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stanislav%2BG.%2BKozmin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozmin%20SG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStanislav%2BG.%2BKozmin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John H. McCusker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BMcCusker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCusker%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BMcCusker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fred S. Dietrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BS.%2BDietrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dietrich%20FS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BS.%2BDietrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert J. Kokoska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Physical Sciences Directorate, U. S. Army Research Office, Research Triangle Park, North Carolina 27709</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BKokoska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kokoska%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BKokoska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas D. Petes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and University Program in Genetics and Genomics, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BD.%2BPetes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petes%20TD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BD.%2BPetes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tom.petes@duke.edu" class="" data-icon-position="" data-hide-link-title="0">tom.petes@duke.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2259&amp;rft.epage=2269&amp;rft.atitle=Structures%20of%20Naturally%20Evolved%20CUP1%20Tandem%20Arrays%20in%20Yeast%20Indicate%20That%20These%20Arrays%20Are%20Generated%20by%20Unequal%20Nonhomologous%20Recombination&amp;rft.volume=4&amp;rft.issue=11&amp;rft.date=2014-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhao%2C+Ying&amp;rft.au=Strope%2C+Pooja+K.&amp;rft.au=Kozmin%2C+Stanislav+G.&amp;rft.au=McCusker%2C+John+H.&amp;rft.au=Dietrich%2C+Fred+S.&amp;rft.au=Kokoska%2C+Robert+J.&amp;rft.au=Petes%2C+Thomas+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459702/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459702/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459702/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459702/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459702/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459702/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459702/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459702/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459702/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459702/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459702/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459702/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/11/2259" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/11/2259.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/11/2259.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/11/2259.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/11/2259.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
