<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/11/2125" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/11/2125.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/11/2125.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/11/2125.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/11/2125.atom" />
<meta name="HW.pisa" content="ggg;4/11/2125" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture" />
<meta name="DC.Identifier" content="10.1534/g3.114.013888" />
<meta name="DC.Date" content="2014-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Suhandynata et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Quantitative proteomics has been widely used to elucidate many cellular processes. In particular, stable isotope labeling by amino acids in cell culture (SILAC) has been instrumental in improving the quality of data generated from quantitative high-throughput proteomic studies. SILAC uses the cell’s natural metabolic pathways to label proteins with isotopically heavy amino acids. Incorporation of these heavy amino acids effectively labels a cell’s proteome, allowing the comparison of cell cultures treated under different conditions. SILAC has been successfully applied to a variety of model organisms including yeast, fruit flies, plants, and mice to look for kinase substrates as well as protein–protein interactions. In budding yeast, several kinases are known to play critical roles in different aspects of meiosis. Therefore, the use of SILAC to identify potential kinase substrates would be helpful in the understanding the specific mechanisms by which these kinases act. Previously, it has not been possible to use SILAC to quantitatively study the phosphoproteome of meiotic Saccharomyces cerevisiae cells, because yeast cells sporulate inefficiently after pregrowth in standard synthetic medium. In this study we report the development of a synthetic, SILAC-compatible, pre-sporulation medium (RPS) that allows for efficient sporulation of S. cerevisiae SK1 diploids. Pre-growth in RPS supplemented with heavy amino acids efficiently labels the proteome, after which cells proceed relatively synchronously through meiosis, producing highly viable spores. As proof of principle, SILAC experiments were able to identify known targets of the meiosis-specific kinase Mek1." />
<meta name="DC.Contributor" content="Ray Suhandynata" />
<meta name="DC.Contributor" content="Jason Liang" />
<meta name="DC.Contributor" content="Claudio. P. Albuquerque" />
<meta name="DC.Contributor" content="Huilin Zhou" />
<meta name="DC.Contributor" content="Nancy M. Hollingsworth" />
<meta name="article:published_time" content="2014-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Quantitative proteomics has been widely used to elucidate many cellular processes. In particular, stable isotope labeling by amino acids in cell culture (SILAC) has been instrumental in improving the quality of data generated from quantitative high-throughput proteomic studies. SILAC uses the cell’s natural metabolic pathways to label proteins with isotopically heavy amino acids. Incorporation of these heavy amino acids effectively labels a cell’s proteome, allowing the comparison of cell cultures treated under different conditions. SILAC has been successfully applied to a variety of model organisms including yeast, fruit flies, plants, and mice to look for kinase substrates as well as protein–protein interactions. In budding yeast, several kinases are known to play critical roles in different aspects of meiosis. Therefore, the use of SILAC to identify potential kinase substrates would be helpful in the understanding the specific mechanisms by which these kinases act. Previously, it has not been possible to use SILAC to quantitatively study the phosphoproteome of meiotic &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; cells, because yeast cells sporulate inefficiently after pregrowth in standard synthetic medium. In this study we report the development of a synthetic, SILAC-compatible, pre-sporulation medium (RPS) that allows for efficient sporulation of &lt;i&gt;S. cerevisiae&lt;/i&gt; SK1 diploids. Pre-growth in RPS supplemented with heavy amino acids efficiently labels the proteome, after which cells proceed relatively synchronously through meiosis, producing highly viable spores. As proof of principle, SILAC experiments were able to identify known targets of the meiosis-specific kinase Mek1.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/11/01" />
<meta name="citation_mjid" content="ggg;4/11/2125" />
<meta name="citation_id" content="4/11/2125" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/11/2125" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/11/2125.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/11/2125.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/11/2125.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.013888" />
<meta name="citation_pmid" content="25168012" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2125" />
<meta name="citation_lastpage" content="2135" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ray Suhandynata" />
<meta name="citation_author_institution" content="Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215" />
<meta name="citation_author" content="Jason Liang" />
<meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, California 92093" />
<meta name="citation_author" content="Claudio. P. Albuquerque" />
<meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093" />
<meta name="citation_author" content="Huilin Zhou" />
<meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093" />
<meta name="citation_author_institution" content="Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, California 92093" />
<meta name="citation_author" content="Nancy M. Hollingsworth" />
<meta name="citation_author_institution" content="Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215" />
<meta name="citation_author_email" content="Nancy.Hollingsworth@stonybrook.edu" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=C. P. Albuquerque;citation_author=M. B. Smolka;citation_author=S. H. Payne;citation_author=V. Bafna;citation_author=J. Eng;citation_title=A multidimensional chromatography technology for in-depth phosphoproteome analysis.;citation_volume=7;citation_year=2008;citation_pmid=18407956;citation_doi=10.1074/mcp.M700468-MCP200" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Allers;citation_author=M. Lichten;citation_title=Differential timing and control of noncrossover and crossover recombination during meiosis.;citation_pages=47-57;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461701;citation_doi=10.1016/S0092-8674(01)00416-0" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=K. R. Benjamin;citation_author=C. Zhang;citation_author=K. M. Shokat;citation_author=I. Herskowitz;citation_title=Control of landmark events in meiosis by the CDK Cdc28 and the meiosis-specific kinase Ime2.;citation_volume=17;citation_year=2003;citation_pmid=12783856;citation_doi=10.1101/gad.1101503" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_title=RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.;citation_pages=1081-1092;citation_volume=79;citation_year=1994;citation_issue=6;citation_pmid=7528104;citation_doi=10.1016/0092-8674(94)90038-8" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_author=D. Park;citation_author=L. Xu;citation_author=N. Kleckner;citation_title=DMC1: a meiosis-specific yeast homolog of E. coli recA required for recombination, synaptonemal complex formation, and cell cycle progression.;citation_pages=439-456;citation_volume=69;citation_year=1992;citation_issue=3;citation_pmid=1581960;citation_doi=10.1016/0092-8674(92)90446-J" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=B. Blagoev;citation_author=I. Kratchmarova;citation_author=S. E. Ong;citation_author=M. Nielsen;citation_author=L. J. Foster;citation_title=A proteomics strategy to elucidate functional protein-protein interactions applied to EGF signaling.;citation_pages=315-318;citation_volume=21;citation_year=2003;citation_issue=3;citation_pmid=12577067;citation_doi=10.1038/nbt790" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. A. Brar;citation_author=B. M. Kiburz;citation_author=Y. Zhang;citation_author=J. E. Kim;citation_author=F. White;citation_title=Rec8 phosphorylation and recombination promote the step-wise loss of cohesins in meiosis.;citation_pages=532-536;citation_volume=441;citation_year=2006;citation_issue=7092;citation_pmid=16672979;citation_doi=10.1038/nature04794" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. B. Buonomo;citation_author=R. K. Clyne;citation_author=J. Fuchs;citation_author=J. Loidl;citation_author=F. Uhlmann;citation_title=Disjunction of homologous chromosomes in meiosis I depends on proteolytic cleavage of the meiotic cohesin Rec8 by separin.;citation_pages=387-398;citation_volume=103;citation_year=2000;citation_issue=3;citation_pmid=11081626;citation_doi=10.1016/S0092-8674(00)00131-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. L. Callender;citation_author=N. M. Hollingsworth;citation_title=Mek1 suppression of meiotic double-strand break repair is specific to sister chromatids, chromosome autonomous and independent of Rec8 cohesin complexes.;citation_volume=185;citation_year=2010;citation_pmid=20421598;citation_doi=10.1534/genetics.110.117523" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Carballo;citation_author=A. L. Johnson;citation_author=S. G. Sedgwick;citation_author=R. S. Cha;citation_title=Phosphorylation of the axial element protein Hop1 by Mec1/Tel1 ensures meiotic interhomolog recombination.;citation_pages=758-770;citation_volume=132;citation_year=2008;citation_issue=5;citation_pmid=18329363;citation_doi=10.1016/j.cell.2008.01.035" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. H. Chen;citation_author=C. P. Albuquerque;citation_author=J. Liang;citation_author=R. T. Suhandynata;citation_author=H. Zhou;citation_title=A proteome-wide analysis of kinase-substrate network in the DNA damage response.;citation_volume=285;citation_year=2010;citation_pmid=20190278;citation_doi=10.1074/jbc.M110.106989" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=J. E. Elias;citation_author=S. P. Gygi;citation_title=Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry.;citation_pages=207214;citation_volume=4;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=S. B. Ficarro;citation_author=M. L. McCleland;citation_author=P. T. Stukenberg;citation_author=D. J. Burke;citation_author=M. M. Ross;citation_title=Phosphoproteome analysis by mass spectrometry and its application to Saccharomyces cerevisiae.;citation_pages=301-305;citation_volume=20;citation_year=2002;citation_issue=3;citation_pmid=11875433;citation_doi=10.1038/nbt0302-301" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. Govin;citation_author=J. Dorsey;citation_author=J. Gaucher;citation_author=S. Rousseaux;citation_author=S. Khochbin;citation_title=Systematic screen reveals new functional dynamics of histones H3 and H4 during gametogenesis;citation_pages=1772-1786;citation_volume=24;citation_year=2010;citation_issue=16;citation_pmid=20713519;citation_doi=10.1101/gad.1954910" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Grushcow;citation_author=T. M. Holzen;citation_author=K. J. Park;citation_author=T. Weinert;citation_author=M. Lichten;citation_title=Saccharomyces cerevisiae Checkpoint Genes MEC1, RAD17 and RAD24 Are Required for Normal Meiotic Recombination Partner Choice;citation_pages=607-620;citation_volume=153;citation_year=1999;citation_issue=2;citation_pmid=10511543" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. A. Henderson;citation_author=K. Kee;citation_author=S. Maleki;citation_author=P. Santini;citation_author=S. Keeney;citation_title=Cyclin-dependent kinase directly regulates initiation of meiotic recombination.;citation_pages=1321-1332;citation_volume=125;citation_year=2006;citation_issue=7;citation_pmid=16814718;citation_doi=10.1016/j.cell.2006.04.039" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. M. Hollingsworth;citation_author=L. Ponte;citation_author=C. Halsey;citation_title=MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in Saccharomyces cerevisiae but not mismatch repair.;citation_pages=1728-1739;citation_volume=9;citation_year=1995;citation_issue=14;citation_pmid=7622037;citation_doi=10.1101/gad.9.14.1728" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. J. Holt;citation_author=B. B. Tuch;citation_author=J. Villen;citation_author=A. D. Johnson;citation_author=S. P. Gygi;citation_title=Global Analysis of Cdk1 Substrate Phosphorylation Sites Provides Insights into Evolution;citation_pages=1682-1686;citation_volume=325;citation_year=2009;citation_issue=5948;citation_pmid=19779198;citation_doi=10.1126/science.1172867" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Hunter;citation_author=N. Kleckner;citation_title=The single-end invasion: an asymmetric intermediate at the double-strand break to double-holliday junction transition of meiotic recombination.;citation_pages=59-70;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461702;citation_doi=10.1016/S0092-8674(01)00430-5" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=V. L. Katis;citation_author=J. J. Lipp;citation_author=R. Imre;citation_author=A. Bogdanova;citation_author=E. Okaz;citation_title=Rec8 phosphorylation by casein kinase 1 and Cdc7-Dbf4 kinase regulates cohesin cleavage by separase during meiosis.;citation_pages=397-409;citation_volume=18;citation_year=2010;citation_issue=3;citation_pmid=20230747;citation_doi=10.1016/j.devcel.2010.01.014" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. P. Kim;citation_author=B. M. Weiner;citation_author=L. Zhang;citation_author=A. Jordan;citation_author=J. Dekker;citation_title=Sister cohesion and structural axis components mediate homolog bias of meiotic recombination.;citation_pages=924-937;citation_volume=143;citation_year=2010;citation_issue=6;citation_pmid=21145459;citation_doi=10.1016/j.cell.2010.11.015" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. Klein;citation_author=P. Mahr;citation_author=M. Galova;citation_author=S. B. C. Buonomo;citation_author=C. Michaelis;citation_title=A central role for cohesins in sister chromatid cohesion, formation of axial elements, and recombination during yeast meiosis.;citation_pages=91-103;citation_volume=98;citation_year=1999;citation_issue=1;citation_pmid=10412984;citation_doi=10.1016/S0092-8674(00)80609-1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=B. H. Lee;citation_author=A. Amon;citation_title=Role of Polo-like kinase CDC5 in programming meiosis I chromosome segregation.;citation_volume=300;citation_year=2003;citation_pmid=12663816;citation_doi=10.1126/science.1081846" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=H.-C. Lo;citation_author=R. C. Kunz;citation_author=A. Marullo;citation_author=S. P. Gygi;citation_author=N. M. Hollingsworth;citation_title=Cdc7-Dbf4 is a gene-specific regulator of meiotic transcription in yeast.;citation_volume=32;citation_year=2012;citation_pmid=22106412;citation_doi=10.1128/MCB.06032-11" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=H.-C. Lo;citation_author=L. Wan;citation_author=A. Rosebrock;citation_author=B. Futcher;citation_author=N. M. Hollingsworth;citation_title=Cdc7-Dbf4 regulates NDT80 transcription as well as reductional segregation during budding yeast meiosis.;citation_pages=49564967;citation_volume=19;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=H. C. Lo;citation_author=N. M. Hollingsworth;citation_title=Using the semi-synthetic epitope system to identify direct substrates of the meiosis-specific budding yeast kinase, Mek1.;citation_pages=135-149;citation_volume=745;citation_year=2011;citation_pmid=21660693;citation_doi=10.1007/978-1-61779-129-1_9" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. S. Longtine;citation_author=A. McKenzie;citation_author=D. J. Demarini;citation_author=N. G. Shah;citation_author=A. Wach;citation_title=Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.;citation_pages=953-961;citation_volume=14;citation_year=1998;citation_issue=10;citation_pmid=9717241;citation_doi=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Lydall;citation_author=Y. Nikolsky;citation_author=D. K. Bishop;citation_author=T. Weinert;citation_title=A meiotic recombination checkpoint controlled by mitotic checkpoint genes.;citation_pages=840-843;citation_volume=383;citation_year=1996;citation_issue=6603;citation_pmid=8893012;citation_doi=10.1038/383840a0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Matos;citation_author=J. J. Lipp;citation_author=A. Bogdanova;citation_author=S. Guillot;citation_author=E. Okaz;citation_title=Dbf4-dependent CDC7 kinase links DNA replication to the segregation of homologous chromosomes in meiosis I.;citation_pages=662-678;citation_volume=135;citation_year=2008;citation_issue=4;citation_pmid=19013276;citation_doi=10.1016/j.cell.2008.10.026" />
<meta name="citation_reference" content="citation_title=The pathway of arginine breakdown in Saccharomyces cerevisae Biochim Biophys Acta;citation_author=W. J. Middlehoven;citation_pages=650-652;citation_volume=93;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=J. Mok;citation_author=P. M. Kim;citation_author=H. Y. Lam;citation_author=S. Piccirillo;citation_author=X. Zhou;citation_title=Deciphering Protein Kinase Specificity Through Large-Scale Analysis of Yeast Phosphorylation Site Motifs;citation_pages=ra12-ra12;citation_volume=3;citation_year=2010;citation_issue=109;citation_pmid=20159853;citation_doi=10.1126/scisignal.2000482" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=H. Niu;citation_author=X. Li;citation_author=E. Job;citation_author=C. Park;citation_author=D. Moazed;citation_title=Mek1 kinase is regulated to suppress double-strand break repair between sister chromatids during budding yeast meiosis.;citation_volume=27;citation_year=2007;citation_pmid=17526735;citation_doi=10.1128/MCB.00416-07" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=H. Niu;citation_author=L. Wan;citation_author=B. Baumgartner;citation_author=D. Schaefer;citation_author=J. Loidl;citation_title=Partner choice during meiosis is regulated by Hop1-promoted dimerization of Mek1.;citation_volume=16;citation_year=2005;citation_pmid=16221890;citation_doi=10.1091/mbc.E05-05-0465" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Niu;citation_author=L. Wan;citation_author=V. Busygina;citation_author=Y. Kwon;citation_author=J. A. Allen;citation_title=Regulation of meiotic recombination via Mek1-mediated Rad54 phosphorylation.;citation_pages=393-404;citation_volume=36;citation_year=2009;citation_issue=3;citation_pmid=19917248;citation_doi=10.1016/j.molcel.2009.09.029" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=S. E. Ong;citation_author=B. Blagoev;citation_author=I. Kratchmarova;citation_author=D. B. Kristensen;citation_author=H. Steen;citation_title=Stable isotope labeling by amino acids in cell culture, SILAC, as a simple and accurate approach to expression proteomics.;citation_volume=1;citation_year=2002;citation_pmid=12118079;citation_doi=10.1074/mcp.M200025-MCP200" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. Onodera;citation_author=Y. Ohsumi;citation_title=Autophagy is required for maintenance of amino acid levels and protein synthesis under nitrogen starvation.;citation_volume=280;citation_year=2005;citation_pmid=16027116;citation_doi=10.1074/jbc.M506736200" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Padmore;citation_author=L. Cao;citation_author=N. R. Kleckner;citation_title=Temporal comparison of recombination and synaptonemal complex formation during meiosis in S. cerevisiae.;citation_pages=1239-1256;citation_volume=66;citation_year=1991;citation_issue=6;citation_pmid=1913808;citation_doi=10.1016/0092-8674(91)90046-2" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual;citation_author=M. D. Rose;citation_author=F. Winston;citation_author=P. Heiter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Shonn;citation_author=R. McCarroll;citation_author=A. W. Murray;citation_title=Requirement of the Spindle Checkpoint for Proper Chromosome Segregation in Budding Yeast Meiosis;citation_pages=300-303;citation_volume=289;citation_year=2000;citation_issue=5477;citation_pmid=10894778;citation_doi=10.1126/science.289.5477.300" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. Sourirajan;citation_author=M. Lichten;citation_title=Polo-like kinase Cdc5 drives exit from pachytene during budding yeast meiosis;citation_pages=2627-2632;citation_volume=22;citation_year=2008;citation_issue=19;citation_pmid=18832066;citation_doi=10.1101/gad.1711408" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=L. Wan;citation_author=T. de los Santos;citation_author=C. Zhang;citation_author=K. Shokat;citation_author=N. M. Hollingsworth;citation_title=Mek1 kinase activity functions downstream of RED1 in the regulation of meiotic DSB repair in budding yeast.;citation_volume=15;citation_year=2004;citation_pmid=14595109;citation_doi=10.1091/mbc.E03-07-0499" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. Wan;citation_author=H. Niu;citation_author=B. Futcher;citation_author=C. Zhang;citation_author=K. M. Shokat;citation_title=Cdc28-Clb5 (CDK-S) and Cdc7-Dbf4 (DDK) collaborate to initiate meiotic recombination in yeast;citation_pages=386-397;citation_volume=22;citation_year=2008;citation_issue=3;citation_pmid=18245450;citation_doi=10.1101/gad.1626408" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Wan;citation_author=C. Zhang;citation_author=K. M. Shokat;citation_author=N. M. Hollingsworth;citation_title=Chemical inactivation of Cdc7 kinase in budding yeast results in a reversible arrest that allows efficient cell synchronization prior to meiotic recombination.;citation_volume=174;citation_year=2006;citation_pmid=17057233;citation_doi=10.1534/genetics.106.064303" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=L. Xu;citation_author=M. Ajimura;citation_author=R. Padmore;citation_author=C. Klein;citation_author=N. Kleckner;citation_title=NDT80, a meiosis-specific gene required for exit from pachytene in Saccharomyces cerevisiae;citation_pages=6572-6581;citation_volume=15;citation_year=1995;citation_issue=12;citation_pmid=8524222;citation_doi=10.1128/MCB.15.12.6572" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. Xu;citation_author=B. M. Weiner;citation_author=N. Kleckner;citation_title=Meiotic cells monitor the status of the interhomolog recombination complex.;citation_pages=106-118;citation_volume=11;citation_year=1997;citation_issue=1;citation_pmid=9000054;citation_doi=10.1101/gad.11.1.106" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=H. Zhou;citation_author=C. P. Albuquerque;citation_author=J. Liang;citation_author=R. T. Suhandynata;citation_author=S. Weng;citation_title=Quantitative phosphoproteomics: New technologies and applications in the DNA damage response.;citation_pages=3479-3484;citation_volume=9;citation_year=2010;citation_issue=17;citation_pmid=20855976;citation_doi=10.4161/cc.9.17.13152" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=G. S. Zubenko;citation_author=A. P. Mitchell;citation_author=E. W. Jones;citation_title=Septum formation, cell division, and sporulation in mutants of yeast deficient in proteinase B.;citation_pages=2395-2399;citation_volume=76;citation_year=1979;citation_issue=5;citation_pmid=377296;citation_doi=10.1073/pnas.76.5.2395" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="twitter:description" content="Quantitative proteomics has been widely used to elucidate many cellular processes. In particular, stable isotope labeling by amino acids in cell culture (SILAC) has been instrumental in improving the quality of data generated from quantitative high-throughput proteomic studies. SILAC uses the cell’s natural metabolic pathways to label proteins with isotopically heavy amino acids. Incorporation of these heavy amino acids effectively labels a cell’s proteome, allowing the comparison of cell cultures treated under different conditions. SILAC has been successfully applied to a variety of model organisms including yeast, fruit flies, plants, and mice to look for kinase substrates as well as protein–protein interactions. In budding yeast, several kinases are known to play critical roles in different aspects of meiosis. Therefore, the use of SILAC to identify potential kinase substrates would be helpful in the understanding the specific mechanisms by which these kinases act. Previously, it has not been possible to use SILAC to quantitatively study the phosphoproteome of meiotic Saccharomyces cerevisiae cells, because yeast cells sporulate inefficiently after pregrowth in standard synthetic medium. In this study we report the development of a synthetic, SILAC-compatible, pre-sporulation medium (RPS) that allows for efficient sporulation of S. cerevisiae SK1 diploids. Pre-growth in RPS supplemented with heavy amino acids efficiently labels the proteome, after which cells proceed relatively synchronously through meiosis, producing highly viable spores. As proof of principle, SILAC experiments were able to identify known targets of the meiosis-specific kinase Mek1." />
<meta name="og-title" property="og:title" content="A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/11/2125" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Quantitative proteomics has been widely used to elucidate many cellular processes. In particular, stable isotope labeling by amino acids in cell culture (SILAC) has been instrumental in improving the quality of data generated from quantitative high-throughput proteomic studies. SILAC uses the cell’s natural metabolic pathways to label proteins with isotopically heavy amino acids. Incorporation of these heavy amino acids effectively labels a cell’s proteome, allowing the comparison of cell cultures treated under different conditions. SILAC has been successfully applied to a variety of model organisms including yeast, fruit flies, plants, and mice to look for kinase substrates as well as protein–protein interactions. In budding yeast, several kinases are known to play critical roles in different aspects of meiosis. Therefore, the use of SILAC to identify potential kinase substrates would be helpful in the understanding the specific mechanisms by which these kinases act. Previously, it has not been possible to use SILAC to quantitatively study the phosphoproteome of meiotic Saccharomyces cerevisiae cells, because yeast cells sporulate inefficiently after pregrowth in standard synthetic medium. In this study we report the development of a synthetic, SILAC-compatible, pre-sporulation medium (RPS) that allows for efficient sporulation of S. cerevisiae SK1 diploids. Pre-growth in RPS supplemented with heavy amino acids efficiently labels the proteome, after which cells proceed relatively synchronously through meiosis, producing highly viable spores. As proof of principle, SILAC experiments were able to identify known targets of the meiosis-specific kinase Mek1." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
    <title>A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"l0KjUEaLR9-oz5nBqTqz-dZyr7x9VR9X0A9o7ZK1KS8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459703","apath":"\/ggg\/4\/11\/2125.atom","pisa":"ggg;4\/11\/2125","ac":{"\/ggg\/4\/11\/2125.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/11\/2125.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2125"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2125"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/11\/2125.abstract":true,"\/content\/4\/11\/2125":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R9bRZ0z9ViVlKCCf82nXLKrQhevfA5qfqbBNArFS5Pm+cXdIOvYNZL\\\\\\\/qtGc8UZOT0iZz5X3JPEE7UJnrunmNjFNjkdb63a0zGQWbLJq\\\\\\\/hxYRFKTFC3oJ5snEyR9ON80HNlalkReZ8AEPCs6qI2lm4IilGW8GM4qZSljmPDpMVyWbPH2LCFgaVtQV\\\\\\\/7qNbEeE3lMS3P7Kejnn6wMLlNsMC1\\\\\\\/UfUxcQyXbH3VTVrRKYXApbffC5dJbfY1Cmunxb40PH2\\\\\\\/lIDMQvqx+QO8w7gwxa1MnCy2lzNC0XdLolhEqxeVM4SYub6RcsQWQI9x4ISnkxgyh\\\\\\\/07yd1na8P7ZyftjlmBYTS8Tka7xusO6+u+8mLnn\\\\\\\/nVeofqa2FKnK2E65+9CKhY8vUyGjZhjES90dYNss3lrzUr\\\\\\\/1ym5DzQVu4fSlPU0uFsFd+pL5UaSZwrvfvy6rEXxqwziFptVOvcmnWLqAXXaKQgrZwxdd+LOuXqmWrZ6niB+fM9ln990VIxlVaSUbEdeb1YJT8fiUcd+vURAEUfL6b5fRlnJMu3fNA32B13W8AZHUYDfCUl3DPcTSY8C1dthPWdxLdo7JDdpEUWBQrb0yvlM\\\\\\\/atWcpLeEG5k+Vv6CcfS6JcjEE7Ou+4D7YDwfe8vTlI0jTj\\\\\\\/8+QQCgEkbDdWh+Vd14bs10D4J2KS\\\\\\\/kdXwYWEd91IAlftL4dWmkCn5wfVexrh3xTl\\\\\\\/Cx0r4cywY1gWHl+zud069Nhg7XMbjNXwXVP6KBRpcDpirOkFmRKK3rJ6B79HvYMUUxafVMlhtALEzKhCSDMSIyKNBlB48Ooe1Jt8b6Wpj\\\\\\\/ayjj3GPjC47LjTdy8cq+mrEQb2IImWOKDlj6sWVdP3aeoBcPQQ7KDaviYux6QVKhiIjyQ2jTszQXJuQCrl13G86w5u8TGeOP722PHkHQOXPpJTbPdmP0l5lI0BkcAlufAzdF4NHKl97f8O5hHWqz+H438c\\\\\\\/6nuyabPSbuTrZbL7ZqGdzI7Lb\\\\\\\/0Y5mL4DnuiAVKYelaw+rpSEsfqazKbi3xy8ielVDPim\\\\\\\/CIQntc6ycWBpBnj0PEWhy8Mpk5gftJ+Egw+8hvfsmFiiDddYDt5gJY94YpxBbl\\\\\\\/lbu+C9Byuipfd2mfzUSykJlSW6Om66dlA6mcw5h\\\\\\\/f036n6eDdL6DJTTSbVeHixsnGeyvC055Gj9Wm89zTUsManOjQURKFTypQVWsbZnPgpYeO5D4ypLt\\\\\\\/ZiGUIMSG7DNyXievob5xKTQasmRHc6+5zKAJ+u2FiO4nbny\\\\\\\/kmvN\\\u0022,\\\u0022iv\\\u0022:\\\u0022GTr3bJsmApjdp5LSRejNOQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269d8240a8451bede5a96b621d8613e5d7c2abb057e5c13c3a84d4fff5a89aab8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZwatOKH+nhnypMZgubwLBmoVG5OUYw8Dus9aNRb4U3UP29DPkiG5uROW76eKFfqnG5KNjFqjBpvuw8yDDYhBIVBRCu3rsk6UxTWSfN69B1PT5OmtynAEgaImPeZlr4QfK\\\\\\\/DhMoks+Uo33BXplmw\\\\\\\/9huuVyCkovvhlCrGRys17+\\\\\\\/Bn8Ph46qghg\\\\\\\/G6VlfTezA+AKvZsVfs+tjpYrZH0ZCcW8zh6hnSoJxcraUZMzmpDs0Z6q\\\\\\\/KSIHh9LN4vNX8MKMEcrxsiX3NT+LeWz5Wr0PSGD3MWDC3zJGfUPUF\\\\\\\/jremI1+m1dKQAZO0XE8HV0JlDYensTkaP2fxQPw8Hq6bpTPx\\\\\\\/VSXH1D88srFcfb30EpsZWurB6MvM\\\\\\\/2ztEjwRNtbomRcL4Ra0F5vICDXtl8ExXUIM0mW9JvSFuFYCm\\\\\\\/aM+cnNffw5y4mE3wY509yPXfS4E8Tshf5BQrhdSNZS+03o43MqN9JW7cXk2nr1xdkeZ80ion\\\\\\\/0RPqPbEHZ8CUFqLNdZv83jFCn3qyETIW2EFZsKubd1ca87qSDr6BFcLAu+Y1qkTmEOSqs5jqrE+S+rHz+zVKRMvS2EQI8YFtezWzA4k0YbpmwJnZevdYtPbgrwRFloOTYKEWWIQzDgSQdSxaol1vmy7wz9kxJtjY9vQ6BQxqM57ol39eUl6XHAyEpqdF6dgdOXtvLl59ZJjnwqXSALQWFclLpgB7ql0PfkUGy01RnMgKGV8TJttfWJiNJJ9YKAEPASvN2ERzryFq6r5CcVAHs2C3hcz05eBNoXHD9uIqURVyLXBSO1eF5jk++mm9SxNeAcsvkYiwvE26MaS3QMzFSXHCMyeldCWjEoG5LDpUnttBKy3KfcxsMsKjPhDRJvifGSdPxXZT4TQgX3kcd85GvV26bK80BN\\\\\\\/2g5xr26nJT6sNHL8ytvMMh5Fst9zRjts5ImRKxR7dB3lizEUUZiUMDFpEuwSTvfjrPvPo5HegEs4jZ9IOzYD8oyQaRm0jNj7HvqzOkH6dw9r7Hn05YGuOuV5mndFMtgYAiG+L2hzh2SJc3n\\\\\\\/4hOVAwZ5ooQ4FQXaNV1Ro72PiyR62vJf5M2PQUBDUtkl26KvpzCd1umYLZzUbAw5LY\\\\\\\/QZhWokRSowz1+I8q2ticEr4Z+oI68LAdIYJywiDfjCjvQ0eSLN4oYarvSncNjSmdBAzBHVzFJiMEB9ILuyAdffxgt3hzAD8A2t668FJplk7Y2lO8rK1rqpnWvFhfjs5bgLy75fxDPct0gEB70l\\\\\\\/WtTl7EcTsF6nxZanQmTKf2RDlJS1648kclJkkDdGrToHoZcXxorQ0aFTwu\\\\\\\/OtrICJBnsk4Zhgs1C7bbk42YHp8CYfiQ7BYKzEdIOy\\\\\\\/rkLXUp0VosqdeRDAnXrqmW4POv0OYSCAzwKZf8gWkIPhoEYo7jlXq4F7gcFmu1AAeY1xPMv1VVm6Cb4hfxE6avwFCwa6N2FyeZTIhsr6bZALE62AQH2TksYZTfm31faAX621422WengDkd1+2oq3U9pMI4xelClWMtmEAVjbQ9jwpXDCAgECsCNrYe2eW2zQx5z\\\\\\\/I0IZeMgGOJyVGb9R5C0qWa8mcHjbWv2krTCxhv\\\\\\\/iKCtJpuH\\\\\\\/jd9SvDmSaT\\\\\\\/sNA+7cp5C887h1tYQhx8K0+9sKjDBWhnUyLwHZX73doRhjRSPbP6kUmhgBjTN7abDiioVjKkQXIZZ+yqmYteTkGoDWUPtZagp0FeZCGB9SNgrmvpVGE4AnZx5Vz7sNFDKZhRF9+VyqRno\\\\\\\/C6O4V\\\\\\\/5W3cg7ZIBgqIsYremaw1sdAXKXvkgN2mL8ozThZkUwwOoyKmR0T2tbRTR4s6eaxVwISywGBJw3vxIgjIS8pp9BPEknfo+j6b9ray1uhththGqmnBLBTW\\\\\\\/sQsR7GpZDGjByQj\\\\\\\/48nAKRUon3YOVmJniOKFAuBgHwHbwUVTXHjY8EHJuVZQ\\\\\\\/RxVaB3Lx1MeUwWZJ3YljdEHDggnCUb29kaxrQFiS+yyDHYBCx6d+47eyITTjq0jl0jss49kdkk34pXk9r798brnNGRgyxhaoJ+G16JQAbs9OULgbfj40H7aMWKuiIvg7\\\\\\\/n5X4lFAFRpqzBD5HAHyMYdA4fzRgsYLqXN6z8SPD93BqEPKcO04MExkcoleIXtDSNLryVuM23TMDSR0vfZ27I19QHxws2Z6P7DESPpJh16sUkxHEMqiwXeSkWkC414pUqFGfxbJUY2GU0znAp9hJWkeFwZNil5Y\\\\\\\/qxCyx9REpZr4uSUPlpo64WjJ5GV43wP3QLF9qCOxB92Oa\\\\\\\/MvcdhXVtfZFSOhQSKDPKnxmOkEFUWgP\\\\\\\/jJZNbesi1P6KR83oSetgb2dwaiSwReW2cB52+vARqH8shxA0vsm3djHMrADXk1VuxR8jYqZgrUiPCjwjSOD4JnBvZkIPGTkf4Q4IQAj0bbMUXsOpv4LHeluvIVDMjgFGHCGP0302D0wfn5dztnV5PLQFO\\\\\\\/s31Xy0nlhhnaWAySdZzyCa+bnnzMt9p+oZDXFDxkPSu+ufza77SMhsgmlYmEdpQYhKL099wUf+wjnPb49gjNAKHcbwEMv7Yz51\\\\\\\/hFK52JXvm8ajNcp7A3oNnhwZSlVcrE4K2gFr0UloC1cV\\\\\\\/X73l58GB0lkgY1G2VUQsAV0JsBXvho56hXn+z4mbJcMRNl07KEYwg7Hirbbnt5mZwHVfElj3XaO\\\\\\\/xgS\\\\\\\/Z0frqJ9PZCOqeKoCAEdMaOPIUkvJyxi7uSmqk1uItytE33uOIz6XfgukOooR34qlUbCPKMIC6VthHB++FvTq19CkQ7CDY0BED2CBDDxBmCOebqHSVJNzdls+IzRFMvBj2stOM92EKNjIW3flkWMhP7V7hKS3PVXu+CCYz66VmlpV6W3n5ACjTMxIXv6aqZXQKBSlwxDT+aw5jvq0M54kNUjN60JzUQH9gJ5Vk0OamOxy9GUMt8TOxYziCYzXAFBgVrvhhQZKdXSRgXrc8X+1V7QXrmu5ax0RpRRlauDl23wETEqb+83Kx4\\\\\\\/mGjt16TjJNB1pAPdqNfwv8Zd+62JtVuvjDNew92IVjcOOQN5PLd9ut18mov3c176VxiGnJfrcq5pDnnxX+512xLX50ha1B8RB0IMWvhjkEeeKWmrGjJjm51BxaC\\\\\\\/d2BnTtphOhIoiHR5cpAPEaQ4BBPVo01oHfJETQ4YzdnCDEulg4LdM0nDiwoUOCs0l4hceeu2+SXAc\\\\\\\/P3iR5aV9l1RcGZ+2ZpZcXZ5jGJ9oZ1gXgSkY6i3wDES7tnNd7tMu4sL5N3c7BYAkHMcWmOP2NWxD2dcLZzyh6ZIjJHiIKbbSgLLzoW1beZObDyYO6t+ePXsCtL0XguEI5LYEZ6vxwBNsCIijMKOMO9OICVzq\\\\\\\/UNbv8dyifRjdzzqL4ut53dayvW+KEU9UBWatFOg1LrMH7DdxLKg7Hf8fohMRWBmIRtGA7Mpa+es8s0Ey0G65lIUZC+sEcn+uaBocDHX62xCWDTmr4sF45S1sLHulkrQKocR59s545qSYt7zuiahBIm\\\\\\\/c1HPecyEMwGp5MIfwA6UuYK4iCI+gL6pBeZ6qht\\\\\\\/abm2bJzykUttASw+K0zW9oLOtBaGRAQDyWjkO74TPgKkcELci1OzonL86sn2n2Na\\\\\\\/5W03OkSkvp5IsE+R3AyOL13bJnvcilfxZupyUZEDSVDmw5Qf1qaTKffmpRnx47Wogji0H\\\\\\\/ihQ20gfAxMuxNrORDsihv9UNadK5N52IBq4ni5EydhDUKzSrrXyCYAvWYikyjh2lc0NMRfvi7ztXWWA06DlH+exs2W+1wqgHzkuIayNOZ6BSBzIG1fPmkMaGE5qhXXufw8exx2bwFMi2A40GNIWWYJ7KFlUfaLasuW8FShwydszMOFgnUyBY5wbdzJu579a+VBuZ0h4d87SefDwf+pYbl1o\\\\\\\/SUM\\\\\\\/hbGuyXXAQF9C+HObz4W\\\\\\\/PMF0V9MsK\\\\\\\/p\\\\\\\/zf2ffAzbLQVFwwTdHWJPyM7VVYP8ozpm+hg16GTG7MK\\\\\\\/v++ZQhBCmHFiVNqnvYhWdjAFJFEzQwnpaQwB+cjJ9lLmi\\\\\\\/02Sihaza7R5S6rqB93Q7vJ2Zd14lbC4QGfFpkYEEPcKYOBHNieI6IDZIb148RHOqTTVQnmxlnjxC\\\\\\\/DV4tmBPTEU6qQNWX+uUFxByVB0Nk\\\\\\\/VXFUDC+HMqCVGi6X7\\\\\\\/d965nJXmK1geKLBqYP+ZKDsx3X9IgZOy2GMe0hmdC66p76wKcgZp+CUl3Mc8t1I5RmlTXMI1MozuLdisw8itnc6vfJ2SSVDYsM03gpW0tl3UqzOHRk2t6T1BOPM3Hb3R8fD82BKnAzTim9MVQtR1eCswdj9xa258frJ8o\\\\\\\/wa\\\\\\\/lH9IYbWx4+qC0Sh53s4sh0TLwsoTEJmYyH2FBjiFCugvFjjNq2xTMtGNSyMnSK+2h93nxhtY45+VQw1XMdv6yNda8EjMvdr4V8xfgluVZ8C9gAxOMOoXaR4tGIN\\\\\\\/VkIaqlE9iDt8rm4DDa9Oah59xYlBgbf6kf9CorYlAaaepX6bJUb4CaT+AAWKUtoTsZbt9EvEEc4PbLcjvJZqOCKxCikTvch4TgFSXgbkeiMTa7PXlL\\\\\\\/0iFcgXafa7m\\\\\\\/SX7IwPhOX5vRn2ubI9ZQyBk\\\\\\\/CpIFLxMw10u6nJlfILDduSPiIwdmuIDly9PfC7aZ46ZFlP3x4X3SDkPkQVHCnKCHVh\\\\\\\/r1UaJZwwJkZYS+nRytMAeABHaxpMVcBttluArOtnsxuer5NP+Ou2jx7ZrVlir4pZNOesrA\\\\\\\/hEE\\\\\\\/ouOC4UOd47vLAdI92fVTcYL8MrofEa8xuckO2cZccSUyhxuGVvImJKnwvclIPFQKViyGUkBGWlkQyZM4nxTHvTJe7Sfv1sMfMzwe65T0CLRssqW6oncA4ovcOVuVtbYg6bJrvuguc7NmvarW2xEEYibiQyy6R6JCA6D\\\\\\\/3O2uvrKrrg8cJKwfIsKdcuvgyz5ms\\\\\\\/+oE0SNrMpV2H1QYk459h9VPaLs4l7eU\\\\\\\/v9PmEQms8sN47jZpMEJ3HvCaLHX9UjhFF1fuSxRCvY6Por7FrNQibq0RbsGEAl6tSwCITbHVuaVot2cmccLn3sO4L63PbsX0p7DaqEqayGyQA5tUw65IlEzItoUS8pJ5aCTkELvVmScCTr\\\\\\\/ZaucDEk18X0nyrVXP17wK3oSfejr0Ufn1684j7PhRnUNtCqXLzdAAxa+VfzpK2w3lcDXZ6CmlKuElK\\\\\\\/103jBYDUohozb4yijdgsr9ACZPtkhstuUu38TqHiBlCJXs1E4gn4vBcfff9MMU4qVta5hUxOQwN72\\\\\\\/5uZSJAz\\\\\\\/drapWOYGplWN0Lw6whac2I3bSWu76zNNI0YR\\\\\\\/Wn7ivAIOhZrRVxYY7n5oJixo\\\\\\\/yDMuXyvh\\\\\\\/TaM4ODChB1\\\\\\\/Em5+8awOi+Np2k0QA8T\\\\\\\/uINQz\\\\\\\/bxNcjbsKPGxoVkzUo9dsQ7qBqAwL6rrytwR33YOROwszDgi\\\\\\\/wuJNm13bsudiENZ6\\\\\\\/cha09LVTCZDBm6uOAVvaOCYSbqN\\\\\\\/utULBDPHY3MobfAiIc8sr4z4U7wljr7A\\\\\\\/JpegcYfkxT8+QZQHqAwCdx6ymwJRSK3Ej6i+sh4Hh+0bxjFKfTnFPk8PhPs1QLh7fq\\\\\\\/30zXF3JtlnsCqgt1ju+RnGTavzfsn8S4rtUe39nuLDn564LP9ump6QlOZT1oU218BTE\\\\\\\/8F1w4iXRFSOkAGmZtaHOpCI4H+Dp+eaBOEZfK0uXVbe9vfhfobK3j\\\\\\\/Du63SEVJDx4MfY94VsKQ0uyCW4m8WnMEeGRm9+MBFKGGYsQmoR5qXU1jsrUHWIXbts2LULZyHaQ6Pv+puYXFQ0uwrnLy05EnO21ztBkrVsrUCsxy2P7XpcdahfxnaNviVXelLlttYKM4LEw\\\\\\\/gkvz63Z29sseSTV6U+xwjEeM9dWxJ4OC6XpLC24gyUoP9KZ1Kw5zVy4FC\\\\\\\/kqp90KWaS0PJtOSr7uzMv7vLntwcfFwLWKOgUdoEslCzHSzjs2EmHo3jTWX\\\\\\\/sh6zPvCpcB\\\\\\\/Q1qQBJ35Pf7RYQIZwZk1WjggfHeq3ad6SX9p1ujUa6wBhZ9GtYCHue9b3HNeUtb8hkuuhY5j7WSAwswQ4ixbvIxheBEt6QJXmrO6t94lXkRj3NYt582HHcMalAwabz0QGe0V9DSPzSPQ8i1ToeiqW3FodrvvtDU26IKJbh1fbAAhgaTNzJ4uU8sivyaFtFmMi2jOlZnDj0kBmdR8MDmQjsAiH5\\\\\\\/S7LXDJJoXWQ7sFdbRjYT3fTlz6MCAb55G0xUV\\\\\\\/U5\\\\\\\/44lcshFoiqgIaL+MejkTi3KlPfEanoUsFHO+YgWn2MMJh1MX9htcU0tYTZV2jD51n\\\\\\\/7K3d\\\\\\\/vDH47OpMehchxTJpXUvZS2y5jHkcieLkQzI1VswG71oWiS7nSHcmLLchutw0PfNyNPTscT0LG\\\\\\\/zIDct9gCTbU0L0aP87F6i1i8WhXraLQSobZuy3lzabrsnz9CRZF5TvTq+KfL7lOjruxuqnsCjGa3kUb9ic8DI8UPBfvjSBlc+ZqYrNbJcKE3OIClUbIkIqAE0LcAO3XFdOLGAo5BxC2xMINEdq2oMi0cp45Kecph3NTD7XUkl7x2gl0PFJ4jsCUOdFQyLgRgcqLZbKUnHKG6i\\\\\\\/qULJSx9h7VApTO5Jzi5lAvHhwjjRCFXhscWdwcaPAL\\\\\\\/1SsLhxd6aZtSm1DjBrfxXJe6QqiNptbUzJi3aKsiXKmYY7tOAk6P7MutMbQeTNjAegC3IawBcmtnCuze2Z8ZF04PIXGdzZQokIW4KqX7PR+2RFDqR8yZCWi0OmJuKgSnRfVXinbFtEUybTW9qRdbyQusOMqUqgkrNK59blqFlPSfqXe9dpUm9IvE1L6hmzylCgIjQ7lhcxkc234YLjWfr7jh0QiR5qJrM7rc4ClZUt8++YDlJbmuNF6+cfIs3UP4ofUzMxeW7+ervzU6dkngKaqruQStlL298KjNJG2ju8hi7EA1KrvqN141PNDswradUNN1klx2nN66yA7F95GrGZ1BZrA\\\\\\\/RWslz3Jxf3QT6wp40Y\\\\\\\/AWb\\\\\\\/XfFSWc+aO1qzTosE8U95sLdiOlS4Fnm\\\\\\\/nLi82v7T+QOya\\\\\\\/ueYAxO7Ztx6DqehRp56D1bkhf8l5NNkmAJ19Q3fiHH\\\\\\\/sQBh\\\\\\\/FFvfdpQxMGREwh7ab\\\\\\\/GmB7dTl1Qm24lTSHaM1IcoIwh\\\\\\\/MvHRR3LEuxTDtMzgkm3kx7CvyRlZ0yBjIlK6EfFGF4XDyiRaRtICi0LNUIP37CivZo6aL1b1rwSShI0tkIT13Ybmsh\\\\\\\/smI1bWo3Oo8k\\\\\\\/4O2AxaeLthMRl6BbaRb7FzKUPMVUQTyfAeSKhUte4G7DskEQOhwYRx31JEbl9WwcqAPYa1ACnP1wSmRfAX30xMvHbE1vbxT0I4CXMtw1GUe0eb5L1uWvInJu33mV2vC2YjUt7QxksTbHwA7fu1HhqMossBkHUYmuaqOQI\\\\\\\/3jnvRXtTfk2rT8pUWiyk5p6NAXaOVEeUOGRNDdv9mym0vGmySEj1BfsDxZYaQGz84Xoj3Z+QTyE2r4YYOiX8AX2uX1c\\\\\\\/KhdVex7q6FmsbT9zea15PIdzS7\\\\\\\/xU8qLJKesKVyvB\\\\\\\/lB4x0JFscpNlkwk7DTo7mkiMRzmtwLMS\\\\\\\/wOb+TlQGp7cCbgKAnkhcaVUazzu0LhrV3ODU5BZGltXFZMl8+DlV+hs7+JfpeFd2K9WLJo3HEC5JiXCAT5PT0SI9anA51EZc09EJndexTIEFzSWBtVGKcko8hQw6PXBAgc9DhkwVCNyXu0mrLvRLfGDL4OBqmGC8k1ph5HGPGElGgF1KtAYpQsSOfQQrWmI32O92iopQIioxOY0wW4M4GSJHrgOBHWtWN4wNrLI59VYvIRBYu5b+V1Py8054hwhcBzPUt8nikTGKXJjepqAi47\\\\\\\/FwKuqkdcAmgD6dYG\\\\\\\/oWL6cEUABYzlRxZuFmYgeRHmfL2S0vAszXuUCQg064WaoLfoX8\\\\\\\/qM7EcDbPL15gytlLtN+5egCg9rgKpjxaCqNW6M\\\\\\\/TVZU+D3QAcL4uSVTawtNNJUTrg3UZl1c2DJLuN1XLwtYVsj5uXJkpyoUhSSRPdTm08rLiQoy5CsNK7a0n4W6\\\\\\\/sEPVcfPl88kt7XYfvNpa9cC4EslLYm0rGf\\\\\\\/clcppfp9YLjpaiHovi9N7yhEKY1bYfxzLaNKVqpDntHWe0teJ5mDLQX86w2E2sqG8zlHxKNkVMJOcn8hvL8tZkkf2RhfjpEkdpACHNHcP1jcJJmzojyAR5SwGuTIBtnVdehapBX36LVMUYYR7fJSZaH13sgb3HexjJtZkznl\\\\\\\/HWZmwD5R8n2sHob1kk4A4euoMt19Bp1f5auJ1sWNjOcn4FmMwMzYBWvoPFx4FFbWxjcFKTmG3tcTRueltLR6gJ+\\\\\\\/bVhaStMs8b1fLxplc\\\\\\\/rkfIxH9j7BuFug2lLvoOQuQQ6f8gyq9cIXxy2pZ6uJNq5KNjw5A0uArhYD8ICZxHwQOdwz9UZcJp2a3HC3paDeccBhuo7u59CEX8RChXeI+uDZAtEwdZVYulCvhex0CrQ5H5lzQCV0S8esCo7ymg1pMKeQnLO4X6YATWL6r4bZ4dXv1\\\\\\\/0Pv8rNF9l+DmuK+Jss\\\\\\\/D0WSb4+XPlnAYbaMJrEqaZ77GtBKnH014YT9EyH0HRfWilvaPinpvKD40qhUr+pFiZqGOD2k+TDJyY8qZdClp+xsV4z1By0vxV3DNvfd\\\\\\\/VeKhcnpza\\\\\\\/XeMkfKwucNyA\\\\\\\/5A0qe4NJXOkrrmoVi41soelrou33ASixcnvAtxIaqq4lYgbDsegclEIjWEnE5+y\\\\\\\/ZnVhfOuA5f9D0BsvSOwfSOVgHNPnOYKGW1EWC2mtZqSORdJQykVmKK3AzUs5ubFgq8a+5IUweIgVUx9XFZKk2Kjm7vxPLtNXqfsk0a9AK9f4w0cam0TLtcLhntjs45i5zX23cLT3ovvk\\\\\\\/Fim500\\\\\\\/4kWbwpli+Dgzv5d2KMCK4UTCvvhOitkYcbzZSRgLLBd+gnvWictLgewFQ6S76v4ED1tNg33ggdYks4VnN67fJIP9UzjAROa79mVQoGUiFhOBX9eBPeCyII5i6+me0WpCEf4naP1HA4Vb9xwa3WYOoR6QheH9FYePx9O684AYYvQrJaoflxCNr7RnOUvbN8\\\\\\\/WCRNwZmS1vMdqnhnMU4VtQPtWwV30XrfCaFFNkwoHRrV5OLawMseZiXfcskgHe4w9p1ineP2FQN0+2GlK3ogvwGsUzbuUCX4xx7ZTBkzIIWHZ7wIottnsgl+tbnScUC3\\\\\\\/aQK8FgdOskwA09BfX6Yx1UmTZsmc\\\\\\\/83tqeV7A8LAHhiqG4X10SQbU5taEDHR6RIAMi1mkRJkM6W9PSekzfcLDUBoc1RMrgc\\\\\\\/8Ekil\\\\\\\/BkAU34rpImAVmJTV2ttdwOieEDwn07sVNGPGX0gmK69l3HrcKSNyXW77QUV\\\\\\\/cLalv9joRmlgPEvW\\\\\\\/TZqRGKoQB0jlAuljrr23Xw11y49SDwLYMgc7u\\\\\\\/z7l8QHoP5P8+YKsU2oYaCWnppwS+jHJtBlbM4uo2Dr1Yvk7pwMZzUMgqmcUIgoZoxjkCBuAB6hQL8JnszTEEPg\\\\\\\/fFg1lmxJ1zFcFI6bNvc0INtRluT4UNwpJ2N6fXJFBF7XqbxVIKUTgl+ZhWH3Ihr+kqflPe+n0amULBvQmI4z9ojfSwq7NZfhIhybPRBiW2BX+2vCXR3+j3MNltH\\\\\\\/fKY+f6+LD0h9QeTZOAEcWiBuq70VnSqkG1y9re1s5MO9bVoX62wTuxzBj2y1D7Wl5ra2TDBR5DsC3irjeNdluQX7sbdOxdxtkC+S8Vdrv1ujHNzBWnfNM4exSZxQXhpveycr7sMJz6U5gUa95znnGThgUsdBduEkKkkrr8gDPnQremUVKpgehb7hzEbnTqdFc0ed4x0neOYM8dtfhGRUMe25X0wHVsqnWmyPqWKKfFS3uy+ZjAMsfOug8yBr35UPeYmVnlHjUzSlaJTZYUjZTqwFCfCv7XE7\\\\\\\/EqNpiY9qCtXEXBDSNW219Qhelh8arDulK9WgQqrZlNEshXVoKvyroMvccfxaB4SE\\\\\\\/9NqY8OggsAA1tPKQWG4Tf+w3rL5ey7plllMFSjyJ\\\\\\\/fNK7eHr+i0scoq3O71iYxUcTffZ+\\\\\\\/b2X15UpzJXL1hCgczIlgMu+YK7ZtjTCEVAs8CjxTL2+bQ7mt2xLEapoe881HbnpqRfrsrKZpaOoXBHbDv0iraKAqF1M59HbbVE+e9UTAlEe6U3Jr0qWevnr655J6Gi6byItKFmirtck727aJnYZ5nrsvsax8EgtEVgcLAkHIKh2eRzezoS7PVsovHDuMncgVyFot\\\\\\\/gEcJfTD7FFk\\\\\\\/g9p7EdvTFa4I2ZTlXXIWzUDdzKrKKzT82wolQTgycuDOjKWrfvYfraoXfXSL6FV0Mya69IAJ7QXOUwdQIYLtqswjYFOzYzcxKNsMFTac1gxE+2W9tyJOVOOYMnkiNL\\\\\\\/4GVYVWek09lrgD\\\\\\\/AhJmPIcwdm708GLAN2\\\\\\\/tjoYXSz7ACGpXLicKtyNr782KmR+s5zHUQ44abD2rnfPveIwFvtMAul4jchTqsJkWlTDgAg7m0ANPCRDM6VNeClSe5eWmdGkyvIUPR\\\\\\\/6plyQ3mfxDSlXHfjAaNTD7kmq5a8JV4U+wKsoyLTB5t8lce7M1pm6A6rdMcwyY5gwqk8W3ibMcSnZVxHece1axEWW7brKOt2Ovu8yrVj39e9dXX3t45rLbja8TEJqv7JktmtKvpn8\\\\\\\/h1Tup2bOhLWXFBso\\\\\\\/uJ4JDMWCrVPtaeiVUIhEciCLzxNE7DF1rQ2QzcecqpHKK9t+GwWTemzkRhS8kS+6bQHUOIzoW3GObsLLMeatTF1AJKTBuxgPyrrXf3B9v2Nsh11o63GfqFSVOLJfqDDAIPx\\\\\\\/KJGuqOvy\\\\\\\/RhrVZ2mHG5viX2Ow8wA0swE4ZTwQmVGjFn7q7g7oPMT5iFjv4RIR6y+niIJTeBm\\\\\\\/bey9FDqL1rWFUacT3wT873xBZm8l1YqSqm9xyNq6UdkuRfnrrem1MZjyqVtV38RJ\\\\\\\/NcBI\\\\\\\/FjQhQExopZcLd7O+LW2ty9WjbrWSBXlYZ24xATj+umYZsEZQUqiNfTEVdVhJq6N11e0Lyedu2VnsraWX5hnwr0dE0lfrVjJ0BpuN9\\\\\\\/UesPZvjVzljKKDECUuQGoHrE+xe4lFOPkCg9FYwPOnBPrgowpxoxVxaJ6HUY\\\\\\\/KtMTAuv0I9NylYONG\\\\\\\/Ebtq1jO\\\\\\\/aZ2zkMGJLn52t238dHYOtXBExVkufBXRNz\\\\\\\/B1MzCf4lSmxP5JAzYYHNwoXJgph7xT6Yn08QaPn8C\\\\\\\/BnJ0m7hTbEsBAlUEaoVnts0Dhx6EwTA1eSlExwbrTqH2upP2zLWygGzyxTqyddIRBVuIrM101UNvuoad7mFXc6v0P3gfTAwThCcqk7XIpXLPaWo8S\\\\\\\/dZRCPYmwZgQaCr4HwvsvznA1E0Nh6xFjJ1tsZQDsiX6OhX5HNSgnXYYlYlMJgoEMeH4x79jEgpPFRLegWiLVuaAYCCh4mLORxgzdPd9u7Nglg\\\\\\\/T78lsZA9CwyN47A9wiclKAlaSIaa2R6KxCDnjO7SmQtsUgCBw2fnvxOWaXs50MGiv3V1+gKAkbP81TYBmTpdeo07puxqx6bUb8n8zr3rUFoTK\\\\\\\/9puCfduDb3XF+CIf+w3Hw7pSNf1dhHBDvXNHwNW7ptAZxRm28wzvRvUoPXiSLbJ8EbQA2s8SQI8Qoxs6YdeFbiTmS1h+Rh+ZYHN2DCQ5OvOhRRsIppTm0UypsQ2naJc+J7051ccEJS9uL+QBXBK+ULnPwg3QBFa0BDVbzlzBjJNtPyU5bCVK7CJY+gjV50xa7zgWcR+HebnXhL18WdJ5MJLHFvN3EDw9BWVa5clc3rj6ew9o0ddXMlUeh2MzqHJd7XLoXvG1mJ2qhFImoVXhVcMqlq9j2TnWKxoauHIeFEADIgOpLrouk1VkxrgoZUKxcYbPpmyB7kJ5oDq6D4i3Z3HG2lI4aGaQ5I6hwrRw4RfqcKaX54JBy2MmNWOXKQSHbAzaQ9mQ9oRKUs4wPSI46VBtJbKg1CAhQjiz\\\\\\\/TalxULxUx896Gr2NfzdGQ+zLDKWC77XhM\\\\\\\/U5KP+H6D+4\\\\\\\/H7AhOr6EFzkdZEFcduoU+mRBZLshGjWS8PL+6v+jTcuMqUqxLBrGR\\\\\\\/k5\\\\\\\/1EfAQqytRJtiHK2Jmru5LjGepPn4+GqToHzdvfNpD1MO4w9bY\\\\\\\/\\\\\\\/9gK4Nnisxs0Ur1JaAJvdPMcixcdaTy8kwPio9lYqct+KMMfuQmSkIZAMQjNT8MhPkrp7ObjzKJUHM8opFZDr0wrs7YxQ54WCsnGnFB6SBvd0BHmnvF2lXutMpWxU75qMGRQkXxboYDk3VmH\\\\\\\/J5oGygjAKWYjAY0JtLQj0+aYeWx\\\\\\\/cc9WFsms7Qku\\\\\\\/xGtDB5qH7fY46w\\\\\\\/hewqdFtCHI0GvmLEgr3lRm8bY3fIEkV3zwFS6kab0fZcdQO0IALKlLFUAz1Ulb0wUJlzNrOW1wQKjQX2YHSgDN3L2y2Qn4DbcMFJtFTs2NPsYqGVZ0rwYTTl6zLJr2yUkSUu+lJon\\\\\\\/Ij3bVgGho+hYAboL6s0USk6Z8kN31ppi0HzNBLl7S\\\\\\\/78HaQAP2OBXiBMxUvFeUCEBjLEzVRt3XEQKCj6t8YXF66hVdq0JAhoghtA+WiuPFD8vKzwwnougqmVVJzKFQ5lx0mvi01Ehy0Pm59uDCoRhiQ\\\\\\\/Bc4UWYql3XHdsoHVZ1jYRKZ021XgCNvBlnSTNDmIYNfseZSUqlUEygpU6f8IAMQj3iCaTbdiTNz1eswgbXgtt7LdsjI2anIMvVq5j9i\\\\\\\/4iv2tlm+QRyINJ5EgnVVO+1M9F1U\\\\\\\/VFWm7Zy2hYjLyFq\\\\\\\/EJYaR8B2ZBd1mx9bwp53mVJDR7wbWmXdhXIqE2VsIZrlmML8oAF5Ikkj2b0e2Ew7kZaZxaGppc6MQxsULRssT9f50GDG2xMOY0S1U0jAetby0DUyON6ZPZE5mlMTSbe8Ss6km02DnOvNm\\\\\\\/hApzA5hgR45c1theCI9lcN3hMozQ4ii7oIfOpUktjbxJ4677M16OTYWPnop\\\\\\\/kiLclPXFTCtnVgrCAcdvXpbVxZeWOUo+JxC6kCEYnNmDPr7HPuNyghS9Qb\\\\\\\/74gV9d\\\\\\\/oxbvRnnKQNCoYjcMKZUdgGMEUH1yqZMoHHEFkB\\\\\\\/8f166mRIAVxDUqIsTp6tg4OeA18VTeqn\\\\\\\/+9kV0nVKc22D4KXjHd7WQyVYRWjVLiCeRfW2+NCVqVIOkaawAgd2\\\\\\\/7TehIUulw8gyKcxbUJBzpBXYyCBa4EJSzocNbW7i7tGY5DLySAVgVMZDqlV6U\\\\\\\/xHObcFkbBzGmKsNrCk4rDYyHLm7g2Pqp2cGuVOF4ERC\\\\\\\/5LF4zAJSawgmuSgIyO43RqeeceOKPayKaC3kv1pDpa2C0qkvJRblGJOCDr2Xpb9oAkptdNGFBTeFBLXtMqtQXg4y9CyVxmyMYTTukuazJ0O743iuWC\\\\\\\/tdoXxLqa3Ps5M4u\\\\\\\/gitJxsW0RGH4spCC1gPlicrmmAqOP1xPXRmtLbvZPqvD001J1Fqkq8C5gkZ1y2cqhbaAETU9NwidlzUzavQ+uIJl\\\\\\\/TcRr13tn8w1x+M9klF8TVEwQeBUZow7rt7ZgK2rV3XzdsJtJzrxQS1oYQ8ZTVBQnIdD2BIPmtmsspU65pwRmsHclXmS+htdWjiB15oPHkRiruSW22kQjGCJmB45BKTHWA90CcGMMnu3Fio4f7la+5qJ1H\\\\\\\/OrMZ52UoX+ZN\\\\\\\/LhiH7bjL2N4IlWj+y6NjVgnGbnniugth9HJdvmAtagAlVww6gaxjp7Gh2OvNn5VdCrvvKEI1gPA715PrIKGyxBml2pEicj3JFr5qKJzcLuDBL05ciWT\\\\\\\/26nTKIzSzH3PdDCD0EHl0yULgMovo\\\\\\\/2PH9JO8UK91nQw9zSfKHZ2jeYZJxfEUU+SYm11aUET1VF7527aebS\\\\\\\/zOomTi4bl3OkeFxXhCsQM4doRNS0NkGJh3MazOvwVhQERPjSYufgFfwh9FPFFMHsT1psyC9Uqsa8DLzgfzCZkLbacPAJzA8XbnvXJOXlh3N944Jm\\\\\\\/ONzEvBtwa0Fq3QKIcZYk+uyAo4R4zQP4\\\\\\\/YZ6RlGxHB1\\\\\\\/NSpXyUePrSF0S2KMZlDvjXRIxy6UyAhki+LRGS74CsOh9Ep582xe3OUQDToGgxYN3uCDIAwQCcvcfiGcn73i7zuNmmU8opu0NAqnC294b2oplLVW5sW\\\\\\\/Jgtf6Ro\\\\\\\/YpZ\\\\\\\/CY2Jly5IrX0VhCOxo6J6aalqLc36lXbVgvPglLCGFk7wiarcXDcVaEjYgzEZH5kjVLRQ7hhPxBwa4L4\\\\\\\/OL02sb8kbDCtRh2OOn1TLG4Lcc4H4aGG0BQU+WDmtkiRZeKYlfcfCV+Xi\\\\\\\/AM16\\\\\\\/rC4VQQ66pNLYArrXxT6U2ZUMBNJv8UOCbJtle9pBWhxlCN60ZMEt0Dk4Q+wlZRPJTKKN2FZsbKNCuaQZM\\\\\\\/r06le4r345KWhtoD5bMgrhrsW4+uuGhjbas6hkZsHyEDVuSGLOjk+mt3sHYi3SQ9t9UACjcfqLAD3uwyasq42p+MPmK8h3dWj5+pU1sQaqvEbhg4jaYDGnv3\\\\\\\/tW1atymGmtoZWo7\\\\\\\/Ca3arOsBsprbak5jBEsUTGUuiFbPPCL99mjJCkEppM1taFbM5XpNkDiEJQ0QY1yWokPVH2m8a\\\\\\\/fVN4LPsimalxgmUrYqaoOgkmN+f6uS4FoGhPc6GyM6ijGJ0sLj1qX5p10VDpXN1ymnSocSijLNLciiNSnyfDrLjuXWLhhJY\\\\\\\/U9lx8ee8T0H0scqumFZ8fasmo6VkalZhfTy\\\\\\\/GL1QOzzmjKBC+XiUzwuQyVQ3WehQ6Tgdqpyy0gPoNIf7\\\\\\\/od3Jz8h5tgQ461R+5NnbDdpVjnl6zXXfo9RhCKiwkqHK8SqQ2gAElgIiJkrbSVZoltNTEO94cQi8XyEmHXCzE8z9PpwidIZT5yayi4RyIpSinrW3uJDASLkG2gcH2bSnPRWIJ2NbmxWYt7okGGL6LhdRCBbexv2xXv18JzalFq\\\\\\\/vslYlFpYmFnZgbAWq5b0JdJ3KAMKZd+bjg4P5OetVuJh0SZHfX6HGUQ+aFZWQJMDAHrfIj33+WO8kfB0WR6PFvqt2XbKRUhKPKBUYIWJdbybXm+Epf+g4H8JB9o\\\\\\\/iY4hC51bY78pyEqP7MSEOT+GskQPQEcj\\\\\\\/wuItI41PeXhk4TR8lZN338mksXcOtEbMBGBkKL0+SlhoclYV3n8E8Jgx\\\\\\\/FvfjIW6fI7gSvYKrHIhf3zgdUcgSnwUBqzkmTRzqMNHVyoKTKTSyf+Ed2N4ONngEIuHcDK6\\\\\\\/n5XFAhtD4l6gIy1lXAIic2uk1IZcxrafLIlG0Q09F6wQa8pOJER1deiQZug+w7bYqybk+hDEebW+lJKyWbkuLl3ATvhzOeKVQVq1Yo2UQS6ejfUddTPwVN09Tbk9qawrChT9dPlnggY7ozu12uVOZRj27zGqjt5Bn4lhrDuHVprPntlNIc6EyYq6ysXvkSebwbYvVwuICWRXeBzDI3fv7f5nEYnvv98esPFcZFl+rwiXVKs4tjljmr67APl\\\\\\\/U1t2I2NyYoTgLkd\\\\\\\/8RhV5R3uHO0wWQqh4ZaJCmQJa8kgPhT8dTZhRalMYNxUs5JxBpJOPcENwYlmC0BXgA5+5j3U\\\\\\\/wpsuS6\\\\\\\/KCFb8w8ZXF6gnMGEtfXFKjZQwvZCSVJkVl2xMRj3vGpDD55S6hVte8HX3nf3XWvR2m7JdzDkSBstBd8PlkmVo163MXljVNOd5qMv9vDO22c9ZQ3MVPhoPIhxtD4I5\\\\\\\/KHjyiByRSGKpwjckxPPavkA79sSqU\\\\\\\/2TyJz1xYEegzAil99ykl07zrM2vkPpilxLM+yhdG\\\\\\\/LEkcEG+QRwQ94bO2xcwb0LvsQZM0AChqg4aO2MpPqhZhSe6PwWC3bK8HPkQmDqylaaY4dYmsB0zyrYIrrBKpmybNrdoJAn2hOJMTnxpPul6YnWf7sqPj3m36VGye6mqt++VntyhRxz24M0xyFpWTM2LEl3sPPkR2jO8Q5Co2rYcJ7BBRgXchgEShXcSdI0pRalkVjUyXcEDH1JbkZ3u8gcMzcR1kCtTOU64JkTStHdJVIKbGwh65JwG8NUvsvnn\\\\\\\/6RWsSFW67Ft8RSwvLE2Ivp8kNmBRsKL7ubojtgEyCU5pz0j81bG0G24oSUnFvAkEKHxFUxO3zNp5IdNZWL\\\\\\\/RV2pl\\\\\\\/jhGe9xwrivdjeudTxp0v9SZCBdXPRyxbhWCTivV\\\\\\\/tvLs4a7XR9jZHTFP+dquYcVQx14DANQFc79drWuCAixOsL0xUxKFeaQr7zBpUe9LpIkDDPSO\\\\\\\/EcQg+h\\\\\\\/kJXUldmkSDdxFx7i\\\\\\\/cGPKvKDFntsjkqUT\\\\\\\/AkhC6qPTlfn9ZC\\\\\\\/Q8moYRjTNHSVA8ox2GdRC7pi3s4qWh6AYNaYTQ3MndF6Xla7C1\\\\\\\/aEjsWHuvewWEe3nx7IFbXM4eFBREWcGmPWgvSvs0A40fj4m1HH+N3yga96D9PfG5+JLBH8tjlJpwzNhmpYFETglv7ADLJVi7c7LURwFHzdo+9mHF117lON0XYDAQVIhhXAaest5AcD6hwpFTnwMOa8uwUFgrFw7s9J8Gqouoj\\\\\\\/iWCHxsjH05atXN4+u8R4Jg6KxiePcR+wE2rAlwQAFAI9x8I6sOl10AG1CFHi2X0aH75EfPG1Ewiwofoslj97GBfSFV9+47\\\\\\\/xF7kuwlq+NNAj6cIfuQFaygN+rFhOSWmkjI+ec0en+IW3EEwCFBi9tLI72B1aYMQ\\\\\\\/0p6kIB\\\\\\\/2pCrIC8QGUYhSjPqbNpg+WQxejm7ozn8TqzMkwghtPVWEDzS+rSs9rZNmi50xyRje+HBrXI7g+HlGTSzUHzkg\\\\\\\/169XNq4pT5bKQ2meRa7Yy54BVqUCrbqKif7VOz2Jv5DYNIV6A7OGqMC988NABwp9GIfn8NtiRj+ULZkrvlxjgsAQ4ea7peoxzMKCAcuWGTjuIuZ6kkG7koVZ8e+G\\\\\\\/t\\\\\\\/aMQ7aZTLQ2XE6uRNQvoa71olhQqiM7Uic92lMzBntEsQaQb0Lwqbjq1aClKPQM+RMPGsIR18whH3H239CFbylg09AEHgPOv+pmSmV8jBK4C39RgW90ukNEviKjc\\\\\\\/wqumTd3D0iqOck9ImMqxoeS5Ui88S6cM9neUQKi518HWFI0BATYaWKi4Prw+MkT5s+YDDxL7Cwy42o04vUKlgryD0Z2XVdtUXrB\\\\\\\/x3LsaqxGFmYeF3eOPud6QvQnIikQb3eT2V5OEOvz0P73lqtZREjGGGEZvB8rBy61YXFYHizugZcAriHR2\\\\\\\/LiMrBbNMeB3ggS\\\\\\\/JvqQa08Tq0Uhws20kYkq6LwyWilIp9m3J3rLARQnI8VN0cFCjaB8f3aKOa6aYHuA0krdKknkZPVolRKV9jcMY8KkEok8IRzV9JhtUMT3vPLNbG4pQaA4ZQcQaGRrHROYp5EUZRI6KXRomqEipeJlN26aGYuhXR8FAEPWahd83\\\\\\\/4mgJILA\\\\\\\/3vnkpyLi3J5kBr8vM+hrJYcGQyuAuPGdDC\\\\\\\/u5gMEPcnAGEoa0XK+30M5lH7A8v1ywg0oecctDH8eB1M\\\\\\\/kBlmgkPZ31Yy+\\\\\\\/kj\\\\\\\/D8j0r9StEG3m8I3UGRR77TUi7KTSr76HjcdMpZ5ctPELWs5V7J6OVGzLIBXqYrs7WDBIh33gEBcXM52PFY+tOq34FLRDa5fRy18TjJU173oEkZiJP27KkSRz7TR6G0r\\\\\\\/W+RjNVR1pB8TdJNcRbI+9gEMd5Jy+pbCWoopbwYPMBVSNpP\\\\\\\/WilUx29VF\\\\\\\/+fZd31UyQNqxHOudxe\\\\\\\/hw8cSzqYdsRT7XA\\\\\\\/DWiMTKpOkQ6MXKlRB9DqX+g7D41xUavh6n+NE\\\\\\\/yTtnGvizNgzuigyfzlLnFkhhEM14nNgOCwIoH37T30kWij9teZf5IpU\\\\\\\/pdyrtgjoXwFtTWKZiEPOCSGuyL6S8Lqa4nQE3bBPqTc2fylz0L8jE5Ao7qCN0GyyZfTjPnqAD2Jp+LVKLrX8jw5jCbtm7RoR4DS8QhfxgAGWLaVcAdLWCsPwRb\\\\\\\/\\\\\\\/uNOFLFCBhHNDeacw5DcZ4bhsEGdVvfwsiFM3nOejmWNhhr1AyHau4nimxF5bGvCOHvIMksrO1MwEjuqUuWHxhH9BYxnBFGzXhOHS4z1LZNwdzuppIXuwqyCddTEkRlM8bfnf8w1P2ihGq9Q4kFgj90yaLEklZUq++MkleDZJGyEp3JfMlNJeC3gajPgw3U+fIxcbJMcQOYywtFx7pCKqwwT5eUBlgwGaWrSJbtDL49DPVBIc\\\\\\\/3MvkJu0Nep38FOhxYyLwqoRmDWUQQHaylS2wWuWUE9f6cDAAc9898Hos\\\\\\\/X8vo9osKrGQbDpvxNCaZ2D3LY\\\\\\\/VkoM\\\\\\\/2YNidq49I+0pMwO1aPAD95Y1LjvZZ6VeGfn5I3LrHDjUugQpybUdtPMx+YoXBREIo7Nnlh3lJZI5FsJadMSNRK8Ezoor\\\\\\\/Tca6fdQsloIM9aJQ9kaNclim3e14CvHZrRtSn6b\\\\\\\/4dKzVvMDX\\\\\\\/VmFM\\\\\\\/pf8pdPuQA0+rFAMfKiU\\\\\\\/JRAXIJEfh7qEBZfzA\\\\\\\/CLVND+hDT5hzMUlNpLwFh\\\\\\\/SBobHGhnzfFBvxK0L5IGQfb4TLkkNN\\\\\\\/WsluX3UkAZWA\\\\\\\/U4WpWB8g+M3yhuArLWj4W17waAqLvqBP3+iGyUMmkviRM4uQlKxlkeJcAFwDoT0SZSvx8LA7OrL1WfOzAApoKxbqGybhS92hMiFOMD8fVq0pyjbzDHMkJC0SfRLIrldPBXhfLBsaRpo1VNEZtGCZlbQEG+fLyedh5xLfH6uAyn6i3PaT7jUVJyKPZJNm8hVmdDlnhvg5MHOkbrETMG\\\\\\\/RWMNF6KOTG8+s9pzavs4hN62zstghPC5KYRJchCisprM776ZH37NwrbScrUJpjuCWwtph89Mb1br7YeiBfAUtVQgZGJhbnh9ccUcTB1\\\\\\\/cOG\\\\\\\/UrCaBL2Y3ocxeOrLaqlqsJrKucxpGvkhUR4WOKZQ4BZUXN1zu2bEHnBCdng3KnWp\\\\\\\/EwX\\\\\\\/NoR7Ty13eYtvwweEU+ba2db7NV4jlXjyEVb\\\\\\\/8e5QWNNDlLWjBnZMiT8UrTRWiQqkjd7ZnqjEto\\\\\\\/hXDcro84iE+FRYXhgp7t1cil9tueH+a+Ys1lfYO7M4dvKiIH4CcM0PY573idGmWGPnuSDwIMwz2dHFdNSXCSYjZLNQmW+p6SAU0sSljLUkG8fvkgNcHeuOstt6DW9LYi3JAwVIhhd9Ze0QdmOti303mgiK\\\\\\\/RmNOVuJ+++YPXeGC05Ytq9OBV+kGSa22Ba1\\\\\\\/L9Ez1DBXAw98NV9hOpgSXKYGfvoyjupewbfbJ4kwoVgpDkEF7c0ztIqIypsvZuitY7\\\\\\\/2N5loZ0ywIgCdQ+jtfTXzpbyK3sTUmkYFlEbFMvXzR3x8cMH8JLOMCKxTRhurlUovW8n+u7LefpuQs59pKxtjmlYknvR4SHvY\\\\\\\/ueAAjgzDmZZgEOvMLqVINB+3zSeRWf09i\\\\\\\/9\\\\\\\/8JethyZGllhyxMBhg2hT4cGBpGBHB3\\\\\\\/OgdWQKVIZ3TNqdSMdZ\\\\\\\/HeWs+iYgwQJZtIWli8V7BckQEjJA1baKPRSv6RXQm1kseOjr\\\\\\\/h3jz8ZzwUWGauedDPoQs+qtBFv2eVeDNaNHnBnAzEW4k9h9PzBnzyaoyvzH0OVWze0tIpfJD6u9oLM2guBGKzPUNVvJuc8bRfKns\\\\\\\/4F6iJrx6qJa7337PqIyyOOy35QotO3ecnLqJGNlusae65PbPreMAmqedy9D5tDxJPVIp+E4aY05fhJ27v6Im6Fogk8jqHtvK4si8Kj2Xy\\\\\\\/nXXga\\\\\\\/7LMBpXg8mx69WIi5HdaqliU9FJc2oagqfI+9yBTQygwq5GN+7gBae+KwpUqvcOex6L9uyp39ymlqBOKPcGJ5SkQkyNUhgIGqsjZ5gbRRcblu18+ZbG3qJU2yrAtna+nRqBqTALSWzwGjO6V2Zi4uOnsYFnhar+RxZjXGDA0SI3GEv6kBhSia2uOYnFNrOWtwzC2i10+BVFHgwNB3scmNClMtNisaSN3YhKh\\\\\\\/Wryf8AFM1DYDabHmjyV+AxzyI9N\\\\\\\/hM3erouVfP+N8N7hNpu+UWEd4FD8ffizCgE4bpum8fepByDpkhzySyBusBhIGAaPSbNc6atjHeahvnm4TQvJEkH5knuZJBdPQ4DYIYYPGXHAaTa0TluNy46\\\\\\\/BlLF6hsCYwh1xgSn34qr0QQm3k2SR\\\\\\\/JeQsxLDnfUpwCe78Z8BtY0PsN0MG3nF3GEhmlt9z\\\\\\\/g6TPgOrvgUcoXCLy7sNP+5F8hMVLcl5uAtjpIYmWIllB4I5aCi6qfaaVgJs8zF2C3hzIHacz6cYxADWMCvRo6I3cCcxpgI5D+zrYAQGll\\\\\\\/R06Or1UJqfejJDhFSVpmAnL9vsjpbpp8rjNF7EPE2\\\\\\\/93\\\\\\\/TzZZz+1o5FNY8p4qBD3476iLKM3OxQ\\\\\\\/7d1GiCbhX7X+fOTQnxvZ0LDynhGX8z9adGCk6ugZnIJBHEkiZKUNSDeCVCiL0Sr66Slj\\\\\\\/sjsZejT3wrZvv0RyI6mQN09GPmSxMy8AapmIV9\\\\\\\/TKBKvraIksxsRxPXFI5pdTRJb1GMSJIIFIFhfcLFo9CpeAI5KgGDnAP3OTv5GEC\\\\\\\/PsaUe\\\\\\\/ZA8HxSM4s6X1ipl3Uk9lvQ+s\\\\\\\/5U\\\\\\\/rwnQVeYND4WqrkM8LKDOnZuWbqukEveH3dNOzeGBE0u+Nw9zPKmK7RTaksqxnLEWOasd0h6LQvqIZ6m0RU16mWLgLx8weXychkt2X6AZKxSlcj2DBKqWz3y+4MB9FDnaIy9KgZLqIO6w5fO6uYoIq6e4TzeAAovUskL8qVQRm1jmXJdWlbUgQqF7UdtQD3ft4KMI3ljrcY4I1X2fch6JblQ4tPt3OScA4frC2O+wWZHA5TF\\\\\\\/rKOd4bv\\\\\\\/BYsY6CVx9VyJOtOA86lwL2gMczNOqgQAJoexiqqTtAujelNtVYTvaFQ51ODXaFQF7iSD4WEgtrf9vEpyYEazMR7BGTYQWBG9M+Z0s1WWDRgnqrHEFIrjCrWZ\\\\\\\/2UjAIcU15GA+z8Z\\\\\\\/8hZr7zBAcslZZrI4nYPYfx5sBz+8y8Wwr1oLWNFgb0eGrUQoOEt7Cslyij2enHBMcUDBqh4A9o+olr+Zxi55PJRd3f2KNv956qZjcxxnx9TMWn\\\\\\\/7ArrzILEdtFRdh3+x5YHJE4PvyGwaGJYZJCJYEmmEEIRQUHTbdslkbYbnSFtr6z4IX10bClFZOyFrnKFYpOnWzRzDmCwYqAdeJ7G39UqHFU5mdEQYLfveYjqtLz2qElu1fo9P5cc5mbBckoBZMxBbuIe3DDRk9NnLy\\\\\\\/55zWIotnX2sgaJeZegBHCpPXBn7mRLAMSerFG0tor9XZDQwOMjsEAbxHDF0fIWZsANyimM0jeEvnKRzeNse4CWOpt1MzWQ5yGy3by51d\\\\\\\/pEoqIgN4ogSzoaD+PiTKo5Ej7335ZQ20yfITnmlF8T1Ks+pfMYhmwVfIOHbWC9tQlaxCfrk0md0aaUZw7DxT+KRh7zidoCdU93Wz1DmhA3ig6ks8WYakv5Sp0v7j1r6cKVCFOMdt69FC1qOl4ZbPVZg3Tu9XatPTWDWsOY4Ae2+0ceh4fkGc1nwlnPJfgdDQqfQxQzI0YDUHtIa5BwlYGNMGuXj2HAmViCIWX+axNGWg8SJM0M6NYfracHUTxHC+xM9WdSP9ppYJ4nSlf5A99k5IvDQZL\\\\\\\/7lBuv9\\\\\\\/+Lz81OAegSJHNm5xe5A538wi6DgY4LNlFm6Qcmmx8q4qgTIeGHy7wslDnq4bpw\\\\\\\/BHGkAOqLZz20PDtc9JnPXDHEu6MpWY1SkF1nkzfoah4LLWjC52k6HlvNIlcfA2DL4k8XL0tCkhsvFDmpqCBW8+uUml758+6f\\\\\\\/otWMs9qTIKn50S8g129Ye9px0KVWzAcgiUd2s+xTR54eIKZmU4qsUnojHucv5aWtzlUMs9M6pMbOd5Ayics\\\\\\\/Zxv5oM3UyT\\\\\\\/DUVRUFMz779hqXJbxq7vQYw54GmxUcrUD+kMswpvn6s21sZbuEavhc1uMFPzGeoRWlldsi6GwhF3F9Xtq3Cf2bqnQGoZDYdYkSpQG58WVoRcSrruYpN9XWMJnUTNUtsN9DbMcyrAl\\\\\\\/zKgaaGTngCKyfLP7QJ0ykFSKArcyyJwlX7tg4w+wbNCtczsWMvy0p3nVg\\\\\\\/91JAXqi6LTYXoiRBDoSREr4GkBJjInTwsor2OVT\\\\\\\/obZGt1kC0Bx9JR+ukgkgRzxQEPGteSNhWEE25UH7Cvm8NSpI+kSwxbhZbx9pLxkNa+ZC\\\\\\\/Cac05MoQPKf3OJXEOXdDqY+J3o9B23FjC9hd5q0zgU7nObeA4uCHDLqLDIsCBtVr27MUyXiTA8WJ\\\\\\\/fYzn+ndQufaFQ7\\\\\\\/DJi+KlQ\\\\\\\/fPsH+8LhdNa4yxqEivAuA9nO\\\\\\\/\\\\\\\/EA+RsYXplezxYmLqAgpcB8wo4LKwzgIpDlZ5bYChn9izzjUDfrxkrHjRlxOsaOyHIzpUFGv1NUolDf5wCNyfeRrxZubFVtjy3gen8ZDLOc4TA+z87vQeW9T29a0FbT6gh0LaXta6NulXKgJy01rO4xKKSa5n7PA\\\\\\\/kV9CUrUDfUARgYXWyymhV1g0lUFUTZB8NAATIV2lY6kZ3nHYVTXkgqqHvXFoRqOF6QjN9T5Fft19A1NJGTJdxP4z1TADJB9hc3OWV45k0g3Opz2by1LlyBxDoWTISkfFep5Fu8Sfu+7z3um6ADsXTZieN\\\\\\\/RFVrLkeVTRJIgcnHXy\\\\\\\/rnBfuPuF+h7tBbAe3r951exugzfjPSCUe9gWX+\\\\\\\/W5QGbTCZMqol9OPzIq9vrdH8r61pDBBGltcd0df59c3934rq3sEzjOdvsT0VNcdf22oE6t8WIvs8ZzxbUFaVL+0Fgem+CAg1UsBqNacnQ2cMn9Pdv1tgELtGcoBf7RC+yyqHJd\\\\\\\/glrS7cPiqDBijKBW54upS3c0DL4WyhCqFisA25FSXtQnkapag6KOq2dxnN2UiD0AEy9ixBthGdDrnAysZoC4J\\\\\\\/zibkmfnhPeKcE4IlDS+HDBXMPNYlV1U07UCM\\\\\\\/s1dX2ddZnsf3isNIUma4CptFWNfPY1Mto\\\\\\\/2ie0q45GQnvqHGiPH9ZEImClB2P5lva0JWo\\\\\\\/ug1gQxxRi5LRfNBTmFy6x1QzhWtD0Y3xLWX03ADULGM79kid7yIqlFzv7bP2f8faywzEMiJqfzZGuWlUVMt7N8xhkmE8VczxA85xBV98C\\\\\\\/CYRpr3fZ2C2iNOSTAoNr\\\\\\\/NovZgGSsMpkeF7ndRjCO\\\\\\\/IKZiQmTRQU\\\\\\\/0cUU9RQBCsOR6D3\\\\\\\/FixbhJ6\\\\\\\/s98Kbk5JQmtXJQuiNfom3LMZ0Rt0ozfcv383ZsNJXKiqQUKF6nVfOq7l1vkedW9CgblExNvHJFECwXzjWw0EcMRfSsZKIL5l\\\\\\\/XTr5\\\\\\\/vJNCQR8mg9EdXpSICy0p3aj4o1YtccOCQ7\\\\\\\/RSr9WmXsKtQ3vvj7BS3KtOGUl4NR5lUiTO821VfVD\\\\\\\/FFynbF199rhjSE6uKKahNH5HKC\\\\\\\/cDGpKo1TOZZ\\\\\\\/R+FVok9nZnI+UpNWZCr4iwma9JWr4IBCn3pIxOL64Um\\\\\\\/yi+S\\\\\\\/0ZhfaGhXVQynnrp5wfcoRnIDn3yRptA\\\\\\\/vr2HELNf4tYPdCsWKjaTjPqYmMFLNTk5IbjrxEZlkupGcDORMpgyyU1LWjhwMOP7D9IYLyiWIf0\\\\\\\/IxgsMDPGLxcKECnMi1Pleip\\\\\\\/V2QWswTVsHWn5u2i1cYTZiydZ\\\\\\\/2oG0bUX6Yn7QvRyJI0BGe\\\\\\\/H7dF0IIPfE\\\\\\\/Phxws1d05J7NobtH1DDntReCaSmIP3RiRYBKQKlrWA29npIulrqMR8rxf6ikbeJyUVINP8jMcLJ3mlZtMBjkNanrZHaESP4CbcgARENfvOSWGmpD8sKIFIKbf7raK\\\\\\\/pbYuBV\\\\\\\/6+546VPp+nZB3abUyhymP8CIlt1BCCBizNbJaqjE8EnQIF1IFxJ0syL775m+eB2M7xfeseDx9coOs69eOqJR5brpA\\\\\\\/Nuyxgytr27+SlT2+i\\\\\\\/bOBuA1tyizKhx\\\\\\\/Y\\\\\\\/C+MQuDfSWDMr1XxuRK\\\\\\\/DBCCBf6o6D3kZ6f7CyGH1CW7YCmb4PnU8Ih6ukOhDKW0jmM6DrJdJ6Vbs60I3+vNTOObwtjvjBiQdbi9QCOBnv3uRfI+htWxw8TGSTZcO1mK72kSBShNnRdizxjLhVonuorxngA17Ck+tPo2IrZ+SJzUVijc9LD6ZAG79Gyo24xDn9ETfM+lsarvt5lWdkUiMUVQ7rZqK0xwy2oxjrzm24RUlGKVvaGntpq2RzXXK43nhebL+ik7dzwVdAP7EKMMcaPGF0\\\\\\\/ISroNBGaFOAFVhngkBa8nZ5HImKnnoY2LflQbCS4ZRSYIDuXjIOtaOpjIca\\\\\\\/6ZGJ2GR4oISP5NZHejSMzSpFX4EeXKFdsfat4AAJX7EvtKr4XLUtQ\\\\\\\/yOHfFF01k7\\\\\\\/TFZxe10Vx4aG0q59Qff1KECQWTJBqFgjHAndF3Tc5VuOycZ8QuOwZd7pIWVaNPGgTKmrWSUtAHmFRtxx8p\\\\\\\/09TrOhTiKDnwwfeshJhwUS3qVtwACL6x\\\\\\\/s6JSP6BBGfix6gxBWNJAUZLz6nWUQzxfiiuHcIbHqHsMhHFqSjhkWNJes3mvS1cNildZOPyU1oiWEVA49RrxtZP7laXQxBkeOEOguIyszV7wNeR9qYttprM7qWD7MR8W3U640fkbv8gXRX0aPbc5nQv\\\\\\\/qMkGnoJAL4sNE0Ij3n3QXKIoCg\\\\\\\/o5n6+N3GuOqTp7GPHxWMP2BDo2qAtLg\\\\\\\/f9MlmnVj+jrwLN+5Jukhu+FtyDTsCFVVdJxTy2AD54Z2pNSe+zlyTNroAVmPmfe5ig8BGmL9WhEu05lsYivwEfTh11CsQfKNmqvGT5+zoCcu6dPCi31+\\\\\\\/EEdZA1KbGfosn0AQ847SN0DXbABb4hp3\\\\\\\/2k3VavG1x0ME08gyMGmT65RGLRjMMWHIUIkGCp630+sUc7ecSE1F8RMU2XXl0GUP0b0TxXT5DoaFEuU2BAh\\\\\\\/mRZ77wtAkcepAXjqz1Hh8VFLer4xHZB710bUaNR+DMLjZr+BgCmpoUBp+iqMTOECqH5idt+fFBVJzL5kMhrIuv1OLvUW4xOiHYmc7+bLHotXd5oEU8C3FKsgPBmuFs9glRmhRgRI24VhqIkuSGxrpq5iWPGOCtkPINJAF2iaudxf7PPoqBQi+52n70bUUg\\\\\\\/Bwyc24ePsjwErcqZVcFhNFbJqUpnbsvYdJ98rH7BzlAVWe0B7+dp5NUKXXSIdNHExNLFpKw3sZzE4l0wudQVpVu+Ur7PYQsyeKGEFlElb8op3fvE1SnA1xMBExSBMapuKwEL4RonHyyc5Ma4QKZv3YLdXWjDvA5TX5t\\\\\\\/AUKSurH4yOQ6+bXU5TRmwuMakmvLCLpMZ4VDtzy6V+xFfSeAzR76o3z4Dp3tUnCPlpx946NuBPJxqs3aC8itUdaVQenaWf0ZAALu2L+IOFAjJX6aPVbjGJveKqqsu6xK0ngXe7IbLtvpnvh+5FZI4CSAd6RVY14G46ILejbO5AqzU6lSi0AwNt0qRnMldZlatc+ddRnhGOyxtB+orAIqEaSJ2Kn9gBQNDw1CnxulrB72AHFy4y8M1eHM86QqiEg83bI88P1xpgwRtbpL6NvNEdtrrJkKaAdM9WEPGSz01xeY9\\\\\\\/9Avr8fpF7G1prFrOiZ9KU25EN8DonUIHkSE8EVR9BDKwXPZKO\\\\\\\/Rq6SMxvSnNxgeU4JGX9MkbX\\\\\\\/eCMLJ9tbsF\\\\\\\/CWbdDA0UmlZndYa42EUl\\\\\\\/\\\\\\\/dHs0pmm3buduCo9bqLJfwd3faYDxsvnMm\\\\\\\/mLjP1EC+\\\\\\\/6VIb2fIOZH8cZJrfVzfCUoS6yW9PdB\\\\\\\/v\\\\\\\/Fk2Op+f54EXo\\\\\\\/LFGqcsUzDL+iImqFR9cXsw0l\\\\\\\/a6J+q5\\\\\\\/0nD+IZUyX4yPfdnxZ9mu2BDPbKc52ILpSz7650xU2LT0amxsiSODsbtTBUOc73m4qs4fvfS\\\\\\\/dsif8F6ywnNboCUtD6WVCqBFBgPsvh8Svw+7xnortKodlN4fKVmWM4cHodwVVqcx360u1m3aGQkCuXfjAHFQiwEVflHC7EfnU80QCvOv5CnLRTxviUoGt6vdcAMXiJGq6QNyq6nULGqnOs+LDVK18hPJWElTb+SBxRmus82HfV0k2djhjzH0XE25EsN1dTEaVYF9aZFrsRAr4t+rkW9leFd0X2Hc7PePId5tNrbr\\\\\\\/lvJ5NSMVL06Zyq3WkUFe+oa3+3k0Hr0eR3DrwPe94Oz20ow3w0aeAig8zEj1\\\\\\\/\\\\\\\/YJijX3Jm9X2ia8r5Ks6IV+CVAcrZLUYxQlI4BhWEOGXXXRzsz1NiU0ZMsJH5jiwkUWutaxZwSD0ElYn0eXwfs8NEfSt5LyZzU5K8bEa+5IFTgas5c8ndBsvAf\\\\\\\/jcr0yP8vUyR\\\\\\\/PdhhSVvEF\\\\\\\/2j9xBpi0XAO7DNC\\\\\\\/+WZyE0JGtgiscLRqueCv3DnuX6Wma1SBgTEgaIX6TXcOSduZ1V\\\\\\\/etD8KrjY341xM3owyUSPFZ0q5mZQbejr0gO6QZfGEXS+2he6fIAIRXQu2S4W1aNMCtKUn1C2L8uiXsaFn7tSqNgIqFbDRGlXXNNeNkLcJ1zGQNYQdfRAJbEKv42u58aSUG5xb8j2huUr1gFCBLCp0sZpoxGnD2eE74rMGz8WwmHQpu3xophHtzjzYtpPNhqI9i8NABZ8wnAzIwJkBqsOqn9+E9RfvTytVj26pEcdlDnAwKDRCNkLvYurs1RHPwW57I8UW628\\\\\\\/x5nsvJKXlMJnzRAWvPDskPggH7YLgknh1NaIaekQGLV+URJ6vCAmkHK3V1S2+xMDZN+5HQEpRWbVxZSVJLS45PNKLGVsW73O0j6xpj1NMOpHi2F6F0hvrz92pPgZKnUcueR7TnoASWbUF3y8kZHudbvfxcBzhDfgS1dUuBqQ67gbYd2k+tjm9dfcRl88zwZtkGbMUaazNG8wWiyLOAUJYgaIbq3tpBq+hIPzmmBesNcprAO4oMHovQX7v1Cq\\\\\\\/5Y6diNNcfpiNDPIBnDICKT1GhLTlPCMEGWJHemO9LivsDSVMua7nSSfHf+1edEeKq9Io7tIddR2\\\\\\\/9rsZQrFmhuxq4iRJxrpEBDxEhPk4mWxebiLbHrU06v49hf9vK3DhHAZvxHpssE7Yq\\\\\\\/fqA4d0moJFEpcmRSeX5IYVwwlMOnot3U\\\\\\\/GcABomO0CSd9h2D0we+bJw+GKX1NvJlvaPTaesguqRpKI5LEtN2+WYHk\\\\\\\/LvUFxVCSip17DMBxQ66SVbSR9BmFYB3Qhm2WLWJLvX0ed2EPcTZy6+6HD2ES21xV+87IrqE0LdxlnRwbFGHOufRg2jTc4gZiixFtk5X2TZZTrWpTO13ftee3QaaYRvHyvDeqMPG7PDRSr5UEbXCYPDtYvDP8WdoWbyMN\\\\\\\/yQhQagF6gAQrhWglHahFFOfTJ8MmFqyx4APPT3CseMce0psRUPOBGfRNE5x0GLtDq\\\\\\\/lPTrZVbVHbKufJ\\\\\\\/5grk8fH9N0pIqjaddLxKyhTMIYKoIacgBD3+vDRbYHIFcldIvIhHezCsLqrKkQppWF3iLm2JEBoGdEL57dSEhknw+ncgkZyj2qsfrEDLojq+GPBcQAScdAU+bZSvjjr166bDdTqsu4yiUZvBYu9Vq\\\\\\\/taTnEEkhaDRJ\\\\\\\/GBWwcbF7pUBsLlNKvb9MBGbu9pkUaO2UHbu8yNild8P81Lzdbfw5PfE42B5wGJCbexmTfNtjwq7e9P5dcqaF3bAee8ucBy39zfDxltI+huGXXYzxX+SpGCe5UzpOkhl1XSetgf6KMUrK1zG7R7hogjt2kWoWn1gAh5N3au9m2k\\\\\\\/h3pFXuHq81982gqeeUnm1tTmzMEX42vCRFvC8nDmLWkBOY96jCZ5Hm+fGB+b0l7mPhuZf5Fa1Qj9JcUGX8JA7D3PKTWGw4FZ0LTfY1DK7xkuDV6z2djirTges8v447Fko5hUekHCJolcVdMp+3K8XQ9KxycQLofQuMNFIUC8TLEf9ZJLtHChtnIXozBzlIfwL1cKV6\\\\\\\/eZlQlnogOhFRWlTsiv4dtAyd1k50V4gSi0ktZt7jHe4\\\\\\\/bd5Xs7by6lMc5NSPyycpg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tw5y6DCbM3cs0JA7Fhswjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224835b0396959c497e1f47bd6c77143fd4b550ca637fa79ba86c9d8f9c0986ff\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Pnte4mv5MgmoS9CtZqyzaXiCiLtuM856eSfenKGyUkuGVC4yTgiyk\\\\\\\/1NJdqqz9h5PU3kQ6N2im3y3kRpTiHeMnSQ4\\\\\\\/HA7StzImuPMIMtvWLhExnCwjQuxRClOyyCggoVUWliULXnSQm+60SMV5WXtS5VtuaDoKTQo6Zh7Gu8eNgaVELC0tnFn2W1FKQbHrPYYOOMCHsDkepxnWIddH\\\\\\\/1Q+im4LgWXK6toDcp6udpnDZ0koWm64D2FjE03b5S7qcIy4F8NfngY8PY+mwJpoOdOkgJS4NQiVfPyDxy2AggXrZcNe4q3dS6j\\\\\\\/nsFYHA87N3tJ0qHuZNIKb\\\\\\\/stmVEesww2CBYFj0k49qXJSkJ5fYg\\\\\\\/wFdNU1NrdYeaYugz\\\\\\\/GnQrinRFh3dEjYHgugkmjDmt2Gz1pOPj2OEwMB7uIu1+WlaNo+8bWg3CH0gmm3\\\\\\\/f9jzVCTGOH8xdkISlMeIm+61NZaHebTaeM+7yliRE9Xi9cvPtkN+Jo670IJqbHixECe0WUb2jTJ2nLLUoaKME2c2tkKwVETeTgdgdzyGdr8GO1Fp+rnx4iCP2QAm7XbZMhEwlWIcrSkwHBOO8w15pF8FweLykYClHyY3jaaQlKuNfP2jNcYUpPOt1ftjy2D8RzfKPi95vC\\\\\\\/nomTguVPDvTO3NVadSr0lLt4Sb4jaPO+WDoEQ23MUZCg0GESHPFUft2AfYK\\\\\\\/7MZY+mNO0V+6m5QCTYHeeQ6kiokDTo\\\\\\\/5\\\\\\\/HMIAn3et7uKPguFey7Vgo6g131ZCJrzmJGyrvxpoRy21BKZx6uomesoVlMO8pVSHGi2ArhqHLEk2jFYx4xP+9sx40GILMpNez01PeK2t+Y1F2x9JmKF262WIcpB7+FyIKWoLJXBA2ARW6h\\\\\\\/iQ+5PvCE6Cd8wGFwYpFekKzPQ329bNyrZqP9xeU1rmcG\\\\\\\/vo91smuVhWv3C8vphq7MLTbEYeeDoEWg+8rf0UgwnEZFNfvXhoetyfpgljR2xljhBzeE1vri9klLVi6GobKWVPOcdZVr7rwEtfRGk0W3uClwvSInHybxP1gXjqRJTck1vUQMVt0MaVVEYChVyOxttU0B3+4EvL2Q69kV\\\\\\\/mfw09S3qTzCn+8\\\\\\\/y\\\\\\\/KEsT3JWuZdGMoAA6qlAcjlCraEB2LtNctQtPALQHZikeTYho6neVoFJ6U+MbqW9he8F2olZnLdNiAQl6q3GXgZwa7ISiRVtPx9bjeYOBtU4gojG92dWj8+R9T84iWyjOUKbYYkS5MskmXNhwa3jAFYWPlXMLe6p7nW+VD5fmbxqEjSGI7w7YVQp1kbq4i67q0hGTn\\\\\\\/RtT0FkIHLKyoUr0oVnk3oi5SBdrrothmCch0RylzaD2TAjR1zhbDfaS+YGwM+UWWhw8gcdife9kqEKaYSu+nyBzf1FmszU34sRGPaS8UwBOn4b8B0bUpOuzBoaCJK9rvbXi9qfYn8Luuv+RBQe\\\\\\\/dduGIyzUfdqjvVCA+oNMfV0CfepzMf3kSuLPOwQA6Bs\\\\\\\/F2njWGQ3uZyEwn\\\\\\\/nNoydfSAvmTvI76ecrlWZUFFf57cBbThNztSPMbtVZ5EJOrcZPRJauCfb6aih3Py42DDP+NEgdFkscJi9zOI1jXMRdEGKUgTb7F8eyyA8jbKfvKEmgyPCnjLm7JHAHCW6MlXh2XITfItWp1avu3F0D8Hsb715zQHTmem201RrBOcEXGnwsqReVnOddpfxUT1O\\\\\\\/QVFHS4VumZeJnh03eLXgHHZtiUxGWykPp0kaoIWsjWWFKhzgkGR122EkC\\\\\\\/xTSjH+8vuz43tjSDyQ5+9m22QwiyU3\\\\\\\/WjmTCb+BJEP02hlNGVIUrEZH1lYvD9jb93sYas6mMGWRiHkFKt61Wxm\\\\\\\/t6p9mr6lOxUsKs0wmWHimiCm0w\\\\\\\/L1h5BiZzD5ae1GjA+0YoPFyqDERwe0Y0cvQK1OTAGZPd03sVvw1kDVPuLSned4aWcrUtVFzikE6zeilOgh2CEKbnNZb\\\\\\\/63qVfcT\\\\\\\/PPAbqtGTRY\\\\\\\/OEJG0pHEXVEeLIZmw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FMbsHgVFHcadTHVVFch9xQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022232f8972ee05ed4a5fffcc65d4b96e9f119ad395737ed50d921e2ed3ce2bedcb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459703 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2125" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-F_ELTUEPtK1HhjEVJAWlWjqubr1SdZrRaBr8JZN0cCw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2125" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wmh8p9f8zoz92HRBP82_nYDsKvtV_yf_1-smeeFyyN0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2115" title="Characterization of Novel Sorghum brown midrib Mutants from an EMS-Mutagenized Population" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2137" title="Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459703" id="node459703" data-pisa="ggg;4/11/2125" data-pisa-master="ggg;g3.114.013888" data-apath="/ggg/4/11/2125.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ray Suhandynata</span>, <span class="highwire-citation-author" data-delta="1">Jason Liang</span>, <span class="highwire-citation-author" data-delta="2">Claudio. P. Albuquerque</span>, <span class="highwire-citation-author" data-delta="3">Huilin Zhou</span> and <span class="highwire-citation-author" data-delta="4">Nancy M. Hollingsworth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2125-2135;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013888 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459703" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ray Suhandynata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ray%2BSuhandynata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Suhandynata%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARay%2BSuhandynata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Claudio. P. Albuquerque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio.%2BP.%2BAlbuquerque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albuquerque%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudio.%2BP.%2BAlbuquerque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huilin Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huilin%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuilin%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nancy M. Hollingsworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BM.%2BHollingsworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hollingsworth%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANancy%2BM.%2BHollingsworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Nancy.Hollingsworth@stonybrook.edu" class="" data-icon-position="" data-hide-link-title="0">Nancy.Hollingsworth@stonybrook.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/11/2125" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459703" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459703/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2125.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459703" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459703/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2125.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459703" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459703/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2125.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459703" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459703/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/11/2125.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459703" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459703/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Quantitative proteomics has been widely used to elucidate many cellular processes. In particular, stable isotope labeling by amino acids in cell culture (SILAC) has been instrumental in improving the quality of data generated from quantitative high-throughput proteomic studies. SILAC uses the cell’s natural metabolic pathways to label proteins with isotopically heavy amino acids. Incorporation of these heavy amino acids effectively labels a cell’s proteome, allowing the comparison of cell cultures treated under different conditions. SILAC has been successfully applied to a variety of model organisms including yeast, fruit flies, plants, and mice to look for kinase substrates as well as protein–protein interactions. In budding yeast, several kinases are known to play critical roles in different aspects of meiosis. Therefore, the use of SILAC to identify potential kinase substrates would be helpful in the understanding the specific mechanisms by which these kinases act. Previously, it has not been possible to use SILAC to quantitatively study the phosphoproteome of meiotic <em>Saccharomyces cerevisiae</em> cells, because yeast cells sporulate inefficiently after pregrowth in standard synthetic medium. In this study we report the development of a synthetic, SILAC-compatible, pre-sporulation medium (RPS) that allows for efficient sporulation of <em>S. cerevisiae</em> SK1 diploids. Pre-growth in RPS supplemented with heavy amino acids efficiently labels the proteome, after which cells proceed relatively synchronously through meiosis, producing highly viable spores. As proof of principle, SILAC experiments were able to identify known targets of the meiosis-specific kinase Mek1.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/phosphorylation" class="hw-term hw-article-keyword hw-article-keyword-phosphorylation" rel="nofollow">phosphorylation</a></li><li class="kwd"><a href="/keyword/mek1-kinase" class="hw-term hw-article-keyword hw-article-keyword-mek1-kinase" rel="nofollow">Mek1 kinase</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/mass-spectrometry" class="hw-term hw-article-keyword hw-article-keyword-mass-spectrometry" rel="nofollow">mass spectrometry</a></li><li class="kwd"><a href="/keyword/silac" class="hw-term hw-article-keyword hw-article-keyword-silac" rel="nofollow">SILAC</a></li></ul><p id="p-3">Protein phosphorylation is critical for many different aspects of meiotic chromosome behavior in <em>Saccharomyces cerevisiae</em>. Recent studies have shown that conserved kinases such as cyclin-dependent kinase 1, Cdc7-Dbf4, casein kinase 1 (Hrr25), and the polo-like kinase, Cdc5, are involved in a variety of meiosis-specific processes, including the initiation of meiotic recombination, resolution of recombination intermediates, synaptonemal complex disassembly, mono-orientation of homologous pairs of sister chromatids at Meiosis I, and/or cleavage of cohesion at the onset of Anaphase I (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Henderson <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lo <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Matos <em>et al.</em> 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sourirajan and Lichten 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wan <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Katis <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lo <em>et al.</em> 2012</a>). In addition, during meiosis the conserved checkpoint kinases, Mec1 and Tel1, promote recombination between homologs and control the meiotic recombination checkpoint (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Grushcow <em>et al.</em> 1999</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carballo <em>et al.</em> 2008</a>). Finally, the meiosis-specific kinase, Mek1, is required for promoting recombination between homologous chromosomes instead of sister chromatids as well as the meiotic recombination checkpoint (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xu <em>et al.</em> 1997</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wan <em>et al.</em> 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Niu <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kim <em>et al.</em> 2010</a>). Understanding the mechanisms by which these kinases control various meiotic processes requires the identification of their substrates, coupled with functional analyses of mutants that are unable to be phosphorylated.</p><p id="p-4">In vegetative yeast cells, combining stable isotope labeling by amino acids in cell culture (SILAC) with phosphopeptide purification and mass spectrometry (MS) has been successful in identifying putative kinase substrates (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhou <em>et al.</em> 2010</a>) . This approach relies on the natural metabolism of cells to incorporate heavy isotopes of specific amino acids into the proteome. The increased mass of the proteins generated using these “heavy” amino acids makes them isotopically distinct by MS from proteins from control cells (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ong <em>et al.</em> 2002</a>). Therefore, a requirement for SILAC is the ability to grow in synthetic medium so that the presence of either heavy or light amino acids can be controlled. The inefficiency with which <em>S. cerevisiae</em> cells sporulate after pre-growth in standard synthetic medium has precluded the application of SILAC to meiotic cells. We have now developed a synthetic pre-sporulation medium that supports efficient sporulation of <em>S</em>. <em>cerevisiae</em> SK1 diploids, thereby making SILAC experiments in meiotic cells possible. Importantly, analysis of various meiotic landmarks indicates that pre-growth in this synthetic medium results in delayed, but otherwise normal, meiosis.</p><p id="p-5">As proof of principle, our SILAC protocol tested to see if it could identify amino acids on proteins known to be phosphorylated by Mek1, modeled on an approach previously developed for identification of Cdk substrates in vegetative cells (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holt <em>et al.</em> 2009</a>). This strategy involves growing a strain containing an analog-sensitive version of the kinase (<em>mek1-as</em>) in the presence of either light or heavy arginine and lysine, arresting the cells, and then adding inhibitor for a short time to the heavy culture to inactivate the kinase. Proteins from each culture are then isolated, combined together, and digested with trypsin to generate peptides. Using immobilized metal affinity chromatography (IMAC), phosphopeptides are purified and the light-to-heavy ratio of specific peptides is determined (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ficarro <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2010</a>). When phosphates added to proteins by the kinase of interest are removed by phosphatases during the period of kinase inactivation, they cannot be replaced; therefore, these phosphopeptides should be under-represented in the heavy culture, resulting in a ratio of light to heavy phosphopeptides greater than 1. Mek1 is an excellent kinase to use as a test case because its activity is constitutively required to maintain the meiotic prophase arrest conferred by deletion of the meiosis-specific recombinase, <em>DMC1</em> (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wan <em>et al.</em> 2004</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Niu <em>et al.</em> 2005</a>), there is a well-characterized analog-sensitive allele of <em>MEK1</em>, <em>mek1-as</em>, and <em>in vivo</em> targets of Mek1 are already known (Mek1 T327, Rad54-T132, and Histone H3 T11), thereby allowing validation of the approach (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Niu <em>et al.</em> 2007</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Niu <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Govin <em>et al.</em> 2010</a>). All three of these substrates match the consensus sequence for Mek1 phosphorylation (RXXT/S) determined by screening peptide libraries (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mok <em>et al.</em> 2010</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains</h3><p id="p-6">All yeast strains are derived from the efficiently sporulating SK1 background and their genotypes are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The construction of the <em>dmc1∆ mek1-as lys4∆ arg4</em> diploid, NH2092, used in the SILAC experiment took several steps. First, <em>MEK1</em> was deleted from DKB187, which contains <em>dmc1∆</em>::<em>LEU2</em>, using the polymerase chain reaction (PCR)-mediated deletion approach with the <em>kanMX6</em> cassette (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Longtine <em>et al.</em> 1998</a>). The <em>LYS4</em> gene was then similarly deleted using <em>hphMX4</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goldstein and McCusker 1999</a>). All knockouts were confirmed either by PCR or by phenotypic analysis. To introduce the <em>arg4-Nsp</em> allele, DKB187 <em>mek1∆ lys4∆</em> was crossed to S2683 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hollingsworth <em>et al.</em> 1995</a>) and <em>MAT</em><strong>a</strong> and <em>MATα</em> segregants containing <em>dmc1∆</em>::<em>LEU2</em>, <em>mek1∆</em>::<em>kanMX6</em>, <em>lys4∆</em>::<em>hphMX4</em> and <em>arg4-Nsp</em> were selected. These haploids were transformed with the <em>mek1-as URA3</em> plasmid, pJR2 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Callender and Hollingsworth 2010</a>), digested with <em>Rsr</em>II to target integration of the plasmid immediately adjacent to <em>mek1∆</em>::<em>kanMX6</em>. The transformants were then mated to make NH2092.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465149/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465149/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>S. cerevisiae</em> strains</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Media</h3><p id="p-7">YPD, YPA, and Spo medium are described elsewhere (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lo and Hollingsworth 2011</a>). To make RPS medium, a drop-out powder lacking arginine and lysine is first created by combining 5.0 g each adenine-HCl (Sigma #A8751), uracil (Sigma #U0750), tryptophan (Sigma #T0254), histidine-HCl (Sigma #H8125), and methionine (Sigma #M9625), 7.5 g each tyrosine (Sigma #T3754), leucine (Sigma #L8000), isoleucine (Sigma #I2752), valine (Sigma #V0500), threonine (Sigma #T8625), and serine (Sigma #S4500), and 12.0 g phenylalanine (Sigma #P2126) in a blender. The nutrients are mixed together by 5× 1-min pulses and the resulting −Arg −Lys powder is transferred to a sterile bottle. To make a 250× solution of light lysine and arginine (3% lysine, 2% arginine), 3 g L-lysine-HCl (Sigma #L5626) and 2 g <span class="sc">l</span>-arginine-HCl (Sigma #A5131) are dissolved in 100 ml water and the solution is filter-sterilized. The heavy arginine and lysine amino acids contain stable heavy isotopes of both <sup>13</sup>C and <sup>15</sup>N. To make a 250× heavy lysine and arginine stock, 0.3 g L-lysine:2HCl (Cambridge Isotope #CNLM-291-H) and 0.2 g <span class="sc">l</span>-arginine:HCl (Cambridge isotope # CNLM-539-H) are resuspended in 10 ml water and filter-sterilized. Both light and heavy amino acid stocks are stored in the dark at room temperature. The heavy amino acids are expensive and more stable in powder form; therefore, smaller stocks are made and used within 2 months.</p><p id="p-8">RPS medium is made by dissolving 28 g yeast nitrogen base without amino acids, 20 g potassium acetate, and 8 g −Arg −Lys powder in a total volume of 1 L in a 2-liter beaker on a stir plate at room temperature. After filter sterilization using a Stericup 0.22 μM filter apparatus (Millipore), 12.5 ml 40% sterile glucose is added to a final concentration of 0.5%. RPS-L or RPS-H is created by the addition of 1.6 ml 250× light or heavy Lysine/Arginine solution, respectively, to 400 ml RPS medium. RPS medium should be stored in the dark at room temperature. Fresh RPS medium exhibits a light green color. After 3 days, the color begins to get darker, and this correlates with less efficient sporulation. Therefore, RPS medium should be used within 3 days.</p></div><div id="sec-4" class="subsection"><h3>Sporulation conditions</h3><p id="p-9">Sporulation of cells after YPA pregrowth is described elsewhere (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lo and Hollingsworth 2011</a>). The following protocol describes sporulation of cells after pre-growth in either RPS-L or RPS-H. On the first day, a single colony is used to inoculate 2 ml YPD, which is then grown at 30° on a roller between 14 and 24 hr. Part of the colony is patched onto YP glycerol plates to ensure that the colony is not petite (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lo and Hollingsworth 2011</a>). The maximum volume of Spo medium for which efficient sporulation has been observed after RPS pre-growth is 200 ml. To ultimately obtain this volume of sporulating culture, overnight cultures are diluted the next day 1:2000 in RPS-L or RPS-H medium in a 2-liter flask (200 μl into 400 ml RPS-L or H). Aeration is important for both pre-growth in RPS and sporulation, so there should always be a flask volume:culture ratio of at least 5:1. The culture is incubated at 30° in a shaker at 250 rpm until it reaches an optical density (OD<sub>660</sub>) of 1.4 to 1.8. Depending on the strain, this can take from 24 to 48 hr. The cells are pelleted by centrifugation and washed once with sterile water. The cells are then resuspended at a cell density of 3 × 10<sup>7</sup> cells/ml in a 2-liter flask [for a conversion chart of OD to cell density, see (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Lo and Hollingsworth 2011</a>)]. The culture is then incubated in a 30° shaker to allow sporulation. Sporulation efficiency can be monitored by phase contrast microscopy to determine the fraction of cells that form spores. Sporulation efficiency for wild-type strains after pre-growth in either RPS-L or RPS-H should be ∼80%.</p></div><div id="sec-5" class="subsection"><h3>Timecourse analysis</h3><p id="p-10">The method for using flow cytometry to monitor premeiotic DNA replication is described in (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wan <em>et al.</em> 2006</a>). DSB repair and crossover formation were analyzed using the <em>HIS4/LEU2</em> hotspot as described previously (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hunter and Kleckner 2001</a>). This hotspot contains a double strand break (DSB) site flanked by <em>Xho</em>I restriction sites. Diagnostic parental and recombinant <em>Xho</em>I restriction fragments are visualized by Southern blot analysis. DNA was isolated using the Epicentre Yeast DNA extraction kit from Illumina. DNA was digested with <em>Xho</em>I and probed with the 0.6 kb AgeI/<em>Bgl</em>II fragment from pNH90 (from Neil Hunter, University of California, Davis). DSBs and crossovers were quantitated using MultiGauge software with a Fujifilm FLA-7000 phosphoimager. To monitor meiotic progression, nuclei were stained with 4′,6-diamindino-2-phenylindole (DAPI) and examined by fluorescence microscopy.</p></div><div id="sec-6" class="subsection"><h3>SILAC labeling of proteins from a <em>dmc1</em>Δ <em>mek1-as</em> diploid in meiosis</h3><p id="p-11">The <em>dmc1∆ mek1-as</em> diploid, NH2092, was pre-grown in either RPS-L or RPS-H (hereafter referred to as the “light” and “heavy” cultures, respectively) and transferred to Spo medium; 200 ml cells were incubated in a 2-liter flask in a 30° shaker for 10 hr to allow the cells to arrest with unrepaired DSBs. At this time, 21 μl of dimethyl sulfoxide (DMSO) was added to the light culture and 21 μl 10 mM 1-NA-PP1 [4-amino-1-<em>tert</em>-butyl-3-(1′napthyl)pyrazolo[3,4-<em>d</em>]pyrimidine] (Tocris Bioscience) dissolved in DMSO was added to the heavy culture (1 μM final concentration). After 20 min, 7 ml cells from each culture were transferred to a 25-ml flask and returned to the 30° shaker until the next day. Inhibition of Mek1-as allows DSB repair by Rad51, thereby eliminating the signal to the checkpoint and allowing the cells to sporulate (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wan <em>et al.</em> 2004</a>). Therefore, the effectiveness of both the <em>dmc1∆</em> arrest and the Mek1-as inhibition can be determined by analyzing the percent sporulation of the light and heavy cultures. The remaining 193 ml of cells from light and heavy sporulating cultures were then collected by centrifugation and washed once with 40 ml cold sterile water. After the cells were resuspended in cold sterile water, they were transferred to 50-ml conical tubes and the cells were collected again by centrifugation. After pouring off the supernatants, the cell pellets were stored at −80°.</p></div><div id="sec-7" class="subsection"><h3>Preparation of crude chromatin and trypsin digestion</h3><p id="p-12">All the quantities described in this protocol are meant for 1 ml yeast pellet volume, unless otherwise specified. To enrich for chromosome-associated proteins, crude chromatin is prepared. The heavy and light cell frozen pellets are thawed by adding 10 mL of 30° reducing buffer [100 mM Tris, pH 9.4, and 10 mM dithiothreitol (DTT) (DTT is added fresh from a 1-M freezer stock)] and incubating at 30° for 15 min. The cells are pelleted by centrifugation at 3700×<em>g</em> for 5 min at room temperature. The temperature of the centrifuge is then lowered to 4° so that it is ready for the next spin. The volume of the cell pellet is measured and, for every milliliter of cell pellet, 10 ml of 30° spheroplasting solution (50 mM KPO<sub>4</sub>, pH 7.5, 1.0 M Sorbitol and 10 mM DTT) are added. The cell walls are removed to create spheroplasts by adding 100 μl of 25 U/μL Lyticase (Sigma), and then incubating at 30° for 45 min with rotation. To determine the efficiency of spheroplasting, 20 μL of cells are added to 1 mL of 0.1% sodium-dodecyl-sulfate (SDS) and the cells are examined by phase contrast microscopy for lysis. Lysed cells look gray, whereas unlysed cells remain bright and refractile. After &gt;80% spheroplasting is achieved (usually ∼45 min), the spheroplasts are pelleted at 1500×<em>g</em> for 5 min at 4° and washed with 10 mL of ice-cold washing buffer (0.1 M KCl, 50 mM HEPES, pH 7.5, 1.0 M Sorbitol). There is no need to resuspend the pellet during this wash; instead, dislodge the pellet from the bottom of the tube by lightly blowing the edges of the pellet with the wash buffer using a P1000 pipette. After harvesting the cells as before, resuspend the pellet in 1 ml of EB buffer [0.1 M KCl, 50 mM HEPES, pH 7.5, 5 mM EDTA, 50 mM NaF, 5 mM β-glycerol phosphate, 1 mM PMSF, and 2× complete, Mini EDTA-free protease inhibitor cocktail (Roche, made fresh every time)] gently with a 1-ml pipette. To lyse the cells, 50 μl 10% Triton X-100 is added (0.5% final concentration) and the spheroplasts are incubated at 4° with gentle rotation for 15 min. This step can be performed without rotation if the suspension is too thick and mixed periodically with a pipette instead. The lysate is then gently placed on top of a one-sample volume of a 30% sucrose cushion in an ultracentrifuge tube (Beckman #326819) and centrifuged at 30,000×<em>g</em> (SW50 or SW55Ti; Beckman Ultracentrifuge) for 15 min at 4°. While loading the sample, it is important to make sure that the boundary between the sample and the sucrose cushion is completely planar. Any unevenness can lead to the sample breaking through the cushion, leading to failure of fractionation. After centrifugation, the supernatant is checked for a yellow tinge. An absence of yellow tinge suggests a failure to fractionate and that the sample should be discarded. The supernatant is removed and the chromatin pellet is thoroughly resuspended in 2 mL Urea Extraction Buffer (8 M Urea, 100 mM Tris pH 8.0, 300 mM KCl, and 10 mM DTT) and incubated at 37° for 30 min with rotation. The resuspended pellet is then centrifuged as before, but for 15 min at 20°. The supernatant is transferred to a new 50-ml conical tube. The pellet is extracted once more with 1 mL of Urea Extraction Buffer and the supernatants combined. At this point, the pellet should appear almost white due to all proteins being extracted from it. The protein concentration of each chromatin preparation is measured using the BioRad Protein Assay reagent. Equal amounts (ideally 10 mg) of light and heavy crude chromatin preparations are combined in a 50 conical tube; 500 mM Iodoacetamide is added to the crude chromatin to a final concentration of 30 mM and incubated for 10 min at room temperature in the dark. Iodoacetamide alkylates the reduced cysteine residues, thereby preventing the reformation of disulfide bonds. The chromatin preparations are then diluted five-fold with TBS (50 mM Tris-HCl, pH 8.0, 150 mM NaCl) so that the concentration of urea is less than 2 M. To digest the proteins into peptides, trypsin [30 mg TPCK-treated trypsin (Worthington, TRTPCK) in 0.1% acetic acid] is added so that the amount of trypsin is equal to 1/100th the amount of chromatin protein. The chromatin is incubated with the trypsin with rotation for 15 hr at 37°. The resulting peptides are acidified by addition of 10% tri-fluoroacetic acid (TFA) to a final concentration of 0.2% and then centrifuged at 3700×<em>g</em> to remove insoluble material. During this time, C18 columns are prepared by washing sequentially with 1 column volume (column volume is indicated on the side-wall of the column) of methanol, 80% acetonitrile/0.1% acetic acid, 0/1% acetic acid, and finally 0.1% TFA. The peptides are de-salted by loading the supernatant onto a 500 mg Sep-Pak C<sub>18</sub> column (Waters) at a ratio of 10 mg total protein per 1 g of C18 resin, washed with 1 column volume of 0.1% TFA using gravity flow, followed by two column volumes of 0.1% acetic acid. Peptides are eluted using 3 ml 80% acetonitrile, 0.1% acetic acid. To maximize the yield, collect any liquid remaining in the column by placing a rubber bulb on top of the column and pushing the liquid out. The peptide solution is aliquoted into glass cuvettes (National Scientific, C4015-843) and dried under vacuum until the volume is reduced sufficiently to fit into one cuvette (approximately 1 hr). The walls of the cuvettes are washed with 80% acetonitrile, 0.1% acetic acid, and then everything is pooled into one cuvette. This cuvette is then placed back into the speedvac overnight to completely dry the peptides.</p><p id="p-13">To measure the efficiency of heavy amino acid labeling, 1 mg of heavy crude chromatin alone is treated with iodoacetamide and trypsin as described above and analyzed using the mass spectrometer.</p></div><div id="sec-8" class="subsection"><h3>Preparation of columns for immobilized metal affinity chromatography (IMAC)</h3><p id="p-14">Gel loading tips (Fisher #02-707-138) are used to make IMAC columns. A small bunch of glass fibers (Fisher #11-388) is carefully placed inside the top half of the tip and is then carefully maneuvered with a piece of thin wire into the narrow part of the tip. Glass fibers are continuously pushed through the tip until they will not move any further. At this point, excess fibers protruding from the tip are cut off with scissors, flush with the end of the tip. The end of the tip containing the frit is bent and flattened with the back of a forceps to decrease the flow rate. The top part of the tip is trimmed until the diameter is small enough so that a 1-ml syringe can fit snugly, creating an airtight seal. To test the column, 10 µl water is loaded into the tip from the top, and the syringe is attached to the tip with the plunger drawn up all the way. If water flows out when pushed with the syringe, then the column tip is ready to use.</p><p id="p-15">Resin from three silica-nitrilotriacetic acid spin columns (Qiagen, Valencia, CA) is pooled in 50 ml of stripping buffer (5 mM EDTA, pH 8.0, 1 M NaCl) in a 50-ml conical tube and incubated for 1 hr at room temperature with rotation. To collect the resin from the columns, each column is turned upside-down over the 50-ml tube and the frit and dry resin are carefully pushed out using a blunt needle. Gently tap the column to collect resin that is stuck to the column wall. The resin is pelleted by centrifugation at 1500×<em>g</em> and washed with 50 ml water, followed by a 50-mL wash of 0.6% acetic acid. The resin is then incubated with 100 mM FeCl<sub>3</sub> in 0.3% acetic acid overnight at room temperature with rotation. The resin is washed once with 50 ml 0.6% acetic acid, and then twice with 50 ml 0.1% acetic acid. The amount of resin is estimated by comparison with the volume of water in a comparable tube so that the resin can be suspended in 0.1% acetic acid as a 50% (volume/volume) slurry and stored at 4°. The final product should have a yellow tinge.</p><p id="p-16">The appropriate amount of packed resin required for IMAC is calculated (40 μl packed resin/5 mg peptides) and the same amount of water is loaded into the top of the column tip. A 1-ml syringe is attached to the tip and water is pushed very slowly until its lower level touches the glass frit. The syringe is removed immediately to prevent the water from flowing through. The upper level of the water is marked, and then it is forced out with the syringe. The 50% resin slurry is loaded into the tip and slowly pushed with the syringe to pack the resin until the water meniscus touches the resin bed. Once the packed resin reaches the mark for the correct volume, the column is ready. Push out all the excess liquid, but do not allow the resin to dry.</p></div><div id="sec-9" class="subsection"><h3>Purification of phosphopeptides using IMAC</h3><p id="p-17">The method for purifying phosphopeptides using IMAC resin is adapted from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albuquerque <em>et al.</em> (2008)</a>. The dried peptides are resuspended in 100 μl 0.1% acetic acid (10 μg/μl peptide concentration) and spun for 5 min at 2300×<em>g</em> in a microcentrifuge. The supernatant is loaded using a micropipette onto a gel loading tip column with fiber glass as a frit containing the appropriate amount of fresh immobilized metal affinity chromatography (IMAC) resin (usually ∼60 μl; see above). After loading, the IMAC resin is washed twice with 60 μL of wash buffer containing 25% acetonitrile, 100 mM NaCl, and 0.1% acetic acid, once with 60 μL 0.1% acetic acid, and finally with 30 μl water. Loading and washes are performed by slowly pushing the buffer through the column using a syringe at a flow rate of 1 to 2 μl/min. The syringe is attached to the tip with the plunger fully drawn, and then the plunger is pushed from the 1 ml to the 0.9 ml marking. This causes the sample to slowly ooze out of the tip. The flow-through is collected in a glass cuvette that can be freeze-dried and used later in case the first round of IMAC fails to purify any phosphopeptides. The column is allowed to run with close monitoring to make sure that the meniscus of the liquid does not drop below the bed level. Once the meniscus reaches the bed, the syringe is removed and next wash solution is added to the top of the column. The plunger is drawn to 1 ml and the syringe is reattached and used as before. The resin should not be allowed to dry at any point (the meniscus of each loading or wash should be right at the resin surface). Phosphopeptides are eluted into a new glass cuvette using 180 μl 6% NH<sub>4</sub>OH and then dried under vacuum. The dried phosphopeptides can be stored at −20° until ready for MS.</p></div><div id="sec-10" class="subsection"><h3>HILIC (hydrophilic interaction liquid chromatography) fractionation and mass spectrometry</h3><p id="p-18">Phosphopeptides eluted from IMAC were subjected to an offline HILIC fractionation method as previously described (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albuquerque <em>et al.</em> 2008</a>). Fractionated samples were run on an LTQ Orbitrap XL using a 1100 Quad PUMP HPLC system (Agilent, Santa Clara, CA) with Ultimate 3000 autosampler (Dionex, Sunnyvale, CA).</p></div><div id="sec-11" class="subsection"><h3>Data analysis using SEQUEST</h3><p id="p-19">To search tandem mass spectra, a composite database is used containing both the yeast protein database and its decoy database. The use of a reverse protein database allows for the estimation of the false discovery rate (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elias and Gygi 2007</a>). The data are analyzed using the Sorcerer system (SageN, San Jose, CA; SEQUEST) and a semi-tryptic restriction is applied to the search. The parameters used for the search are a parent mass tolerance of 20 ppm, +80.0 Da variable modification of STY (serine, threonine, and tyrosine) due to phosphorylation, and a maximum of two modifications per peptide. After searching, the raw results of SEQEST are then filtered using their provided <em>p</em> value to a 1% false discovery rate (number of decoy database hits/total hits).</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Pre-growth in RPS-L medium allows for normal meiosis and sporulation</h3><p id="p-20">Synthetic dextrose (SD) medium for growth of budding yeast cells usually comprises 0.67% yeast nitrogen base without amino acids and 2% glucose, which can then be supplemented with appropriate amino acids and nucleotides to final concentrations of between 20 and 400 mg/L (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rose <em>et al.</em> 1990</a>). Cells pregrown in SD medium plus nutrients do not sporulate efficiently when transferred to Spo medium (data not shown). Sporulation in SK1 strains is enhanced by pre-growth in a rich acetate medium called YPA (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Padmore <em>et al.</em> 1991</a>). To create a synthetic sporulation medium (RPS) that supports efficient sporulation, the major carbon source in SD was changed from glucose to 2% acetate, similar to YPA. A small amount of glucose (0.5%) is included and is necessary for cells to grow well. In addition, four-times the amount of dropout powder is used compared with conventional SD dropout medium (see <em>Materials and Methods</em> for recipes). “Heavy” versions of lysine and arginine contain stable heavy isotopes of <sup>13</sup>C and <sup>15</sup>N. RPS supplemented with “light” or “heavy” arginine/lysine is indicated as RPS-L and RPS-H, respectively.</p><p id="p-21">Although a slight reduction in sporulation (84% from 95%) was observed in two different wild-type strains pre-grown in RPS-L compared with YPA, spore viability was wild-type (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Timecourse analyses were used to monitor various meiotic parameters. Cells pre-grown in RPS-L exhibit a 2-hr delay in the onset of premeiotic DNA synthesis compared with YPA pre-growth (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Once the cells enter meiosis, however, meiotic parameters such as DSB and crossover formation, as well as meiotic progression, occur with normal timing (taking into account the initial delay) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B–D</a>). These results indicate that the transition from vegetative growth to meiosis takes longer after pre-growth in RPS medium but, having once entered meiosis, cells proceed normally and relatively synchronously to form highly viable spores.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465180/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465180/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Sporulation and spore viability under various conditions after pre-growth in YPA, RPS-L, or RPS-H</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of various meiotic parameters in cultures grown in either YPA or RPS-L prior to sporulation. The wild-type SK1 strain, NH716, was pre-grown in either YPA or RPS-L and transferred to Spo medium at 30&#xB0;, and samples were taken at 2-hr intervals. (A) Flow cytometry analysis of premeiotic DNA synthesis. Numbers indicate hours after transfer to Spo medium. Red numbers with asterisks indicate the time points at which DNA synthesis is complete. (B) DSB and crossover analysis was performed using the HIS4/LEU2 hotspot (Hunter and Kleckner 2001). P1 and P2 indicate parental fragments, CO1 and CO2 indicate crossover fragments, and DSB indicates DSB fragments. This analysis was performed for two different time courses with similar results. (C) Quantitation of the DSBs and COs shown in (B). (D) Meiotic progression was measured by staining cells with DAPI and counting the number of nuclei in each cell (binucleates indicate completion of Meiosis I and tetranucleates indicate completion of Meiosis II). For each time point, 200 cells were examined by fluorescent microscopy. Error bars represent the SD observed for the two independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985128186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of various meiotic parameters in cultures grown in either YPA or RPS-L prior to sporulation. The wild-type SK1 strain, NH716, was pre-grown in either YPA or RPS-L and transferred to Spo medium at 30&#xB0;, and samples were taken at 2-hr intervals. (A) Flow cytometry analysis of premeiotic DNA synthesis. Numbers indicate hours after transfer to Spo medium. Red numbers with asterisks indicate the time points at which DNA synthesis is complete. (B) DSB and crossover analysis was performed using the &lt;em&gt;HIS4/LEU2&lt;/em&gt; hotspot (Hunter and Kleckner 2001). P1 and P2 indicate parental fragments, CO1 and CO2 indicate crossover fragments, and DSB indicates DSB fragments. This analysis was performed for two different time courses with similar results. (C) Quantitation of the DSBs and COs shown in (B). (D) Meiotic progression was measured by staining cells with DAPI and counting the number of nuclei in each cell (binucleates indicate completion of Meiosis I and tetranucleates indicate completion of Meiosis II). For each time point, 200 cells were examined by fluorescent microscopy. Error bars represent the SD observed for the two independent experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2125/F1.medium.gif" width="254" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/11/2125/F1.medium.gif" width="254" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465217" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Comparison of various meiotic parameters in cultures grown in either YPA or RPS-L prior to sporulation. The wild-type SK1 strain, NH716, was pre-grown in either YPA or RPS-L and transferred to Spo medium at 30°, and samples were taken at 2-hr intervals. (A) Flow cytometry analysis of premeiotic DNA synthesis. Numbers indicate hours after transfer to Spo medium. Red numbers with asterisks indicate the time points at which DNA synthesis is complete. (B) DSB and crossover analysis was performed using the <em>HIS4/LEU2</em> hotspot (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hunter and Kleckner 2001</a>). P1 and P2 indicate parental fragments, CO1 and CO2 indicate crossover fragments, and DSB indicates DSB fragments. This analysis was performed for two different time courses with similar results. (C) Quantitation of the DSBs and COs shown in (B). (D) Meiotic progression was measured by staining cells with DAPI and counting the number of nuclei in each cell (binucleates indicate completion of Meiosis I and tetranucleates indicate completion of Meiosis II). For each time point, 200 cells were examined by fluorescent microscopy. Error bars represent the SD observed for the two independent experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Pre-growth in RPS-H efficiently labels proteins both in vegetative cells and after the induction of meiosis</h3><p id="p-26">For SILAC experiments to work, the labeling of proteins with heavy amino acids must be highly efficient. Peptides from RPS-H grown vegetative cultures were analyzed by MS, and 3900 out of 4064 total identified peptides (96%) contained heavy amino acids. One question is whether the fraction of heavy labeled proteins remains high after hours in Spo medium. Meiosis and sporulation are induced by transferring cells into a medium that lacks nitrogen and contains a nonfermentable carbon source such as acetate. The absence of nitrogen means that no new amino acid synthesis can occur. Instead, amino acids used for new protein synthesis are recycled from existing proteins by autophagy, as evidenced by the fact that protease activity is essential for sporulation (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zubenko <em>et al.</em> 1979</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Onodera and Ohsumi 2005</a>). This recycling is an advantage for meiotic SILAC experiments, given that proteins produced during meiosis must be made using heavy amino acids that were present during the pre-growth period. Consistent with this idea, 5443 out of 5611 (97%) of the identified peptides analyzed 10 hr after transfer to Spo medium from RPS-H pre-grown cells still contained heavy amino acids.</p><p id="p-27">Another possible problem is that catabolism of heavy arginine could produce proline and glutamic acid residues with heavy atoms that could skew the quantification of heavy peptides (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Middelhoven 1964</a>). To determine how frequently this happens, the raw data were analyzed for the presence of heavy proline or glutamate and low levels (&lt;1.0%) of these amino acids were observed. Analysis of the most abundant proline and glutamate-containing peptides exhibited L/H ratios not significantly different than 1. Therefore, catabolism of arginine during meiosis is unlikely to significantly affect the quantification of proline/glutamate-containing peptides.</p></div><div id="sec-15" class="subsection"><h3>SILAC, combined with phosphopeptide purification, identifies known substrates of Mek1</h3><p id="p-28">To validate the method, a meiotic SILAC experiment was performed to see if it could identify known substrates of Mek1. The experimental strategy used for the <em>dmc1∆ mek1-as</em> SILAC experiments is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>. In the absence of <em>DMC1</em>, cells arrest in prophase with resected double strand breaks (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bishop <em>et al.</em> 1992</a>) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Rad51 is loaded onto the breaks, but its activity is indirectly inhibited by Mek1 (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bishop 1994</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Niu <em>et al.</em> 2009</a>). Inactivation of Mek1-as in <em>dmc1∆ mek1-as</em> arrested cells allows repair of the breaks using sister chromatids as templates, thereby removing the signal to the meiotic recombination checkpoint and allowing meiotic progression (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Niu <em>et al.</em> 2005</a>) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). An overnight culture of a <em>dmc1∆ mek1-as</em> diploid was diluted into either RPS-L or RPS-H, the cells were grown to the appropriate density, and then those cells were transferred to Spo medium for 10 hr to allow them to arrest in meiotic prophase with unrepaired DSBs. The effectiveness of both the arrest and Mek1 inhibition were assessed by removing small aliquots of cells before and after addition of the Mek1-as inhibitor, 1-NA-PP1, and incubating them at 30° overnight. Although no asci were observed in the absence of inhibitor, indicating a robust arrest, inactivation of Mek1 resulted in high levels of sporulation as expected (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). For the SILAC experiment, cells were harvested 20 min after addition of inhibitor. Because all known substrates of Mek1 are associated with DNA, crude chromatin was isolated to enrich for proteins that are potential targets of the kinase. The chromatin preparations from the heavy and light cultures were combined and treated with trypsin. Trypsin cuts after arginine or lysine to generate peptides. Analysis of a fraction of the total peptides by MS showed that 96% of the total peptides exhibited an L/H ratio of approximately 1, indicating that most proteins were present in equal abundance in the two chromatin preparations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Phosphopeptides were then enriched using IMAC. The peptides were subjected to HILIC fractionation prior to MS analysis, which allows for in-depth mapping of protein phosphorylation sites in a complex sample. HILIC is largely orthogonal to reverse-phase high-pressure liquid chromatography for phosphopeptide separation but does not require a de-salting step before LC-MS/MS analysis, preventing sample loss after fractionation (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Albuquerque <em>et al.</em> 2008</a>). In two different experiments, 16,036 and 16,555 total peptides were identified, of which 15,241 (95%) and 13,959 (84%) were phosphorylated, respectively. The peptides from these two experiments represent 1985 and 2058 unique phosphorylated proteins. Because the purpose of this article is meant to report the method for performing SILAC in meiosis, the entire dataset of identified phosphopeptides will be made available elsewhere.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fate of meiotic DSBs in dmc1&#x2206; mek1-as diploids with or without inhibitor. (A) Experimental flow chart for a SILAC experiment using dmc1&#x2206; mek1-as. (B) Inactivation of Mek1-as allows DSB repair in a dmc1&#x2206; mutant. Two homologs (one red and one blue) are shown after DNA replication to make sister chromatids. Each chromatid is represented as a duplex of DNA. DSBs formed on one of the four chromatids remain unrepaired in the absence of the 1-NA-PP1 inhibitor, thereby triggering the meiotic recombination checkpoint and arresting the cells in meiotic prophase (Bishop et al. 1992; Lydall et al. 1996). Addition of 1-NA-PP1 inactivates Mek1, thereby allowing Rad51 to repair the DSBs using sister chromatids as templates. This repair eliminates the signal to the checkpoint, thereby allowing meiosis to proceed and asci to form (Wan et al. 2004; Niu et al. 2005)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985128186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fate of meiotic DSBs in &lt;em&gt;dmc1&#x2206; mek1-as&lt;/em&gt; diploids with or without inhibitor. (A) Experimental flow chart for a SILAC experiment using &lt;em&gt;dmc1&#x2206; mek1-as&lt;/em&gt;. (B) Inactivation of Mek1-as allows DSB repair in a &lt;em&gt;dmc1&#x2206;&lt;/em&gt; mutant. Two homologs (one red and one blue) are shown after DNA replication to make sister chromatids. Each chromatid is represented as a duplex of DNA. DSBs formed on one of the four chromatids remain unrepaired in the absence of the 1-NA-PP1 inhibitor, thereby triggering the meiotic recombination checkpoint and arresting the cells in meiotic prophase (Bishop &lt;em&gt;et al.&lt;/em&gt; 1992; Lydall &lt;em&gt;et al.&lt;/em&gt; 1996). Addition of 1-NA-PP1 inactivates Mek1, thereby allowing Rad51 to repair the DSBs using sister chromatids as templates. This repair eliminates the signal to the checkpoint, thereby allowing meiosis to proceed and asci to form (Wan &lt;em&gt;et al.&lt;/em&gt; 2004; Niu &lt;em&gt;et al.&lt;/em&gt; 2005).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2125/F2.medium.gif" width="260" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/11/2125/F2.medium.gif" width="260" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465184" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Fate of meiotic DSBs in <em>dmc1∆ mek1-as</em> diploids with or without inhibitor. (A) Experimental flow chart for a SILAC experiment using <em>dmc1∆ mek1-as</em>. (B) Inactivation of Mek1-as allows DSB repair in a <em>dmc1∆</em> mutant. Two homologs (one red and one blue) are shown after DNA replication to make sister chromatids. Each chromatid is represented as a duplex of DNA. DSBs formed on one of the four chromatids remain unrepaired in the absence of the 1-NA-PP1 inhibitor, thereby triggering the meiotic recombination checkpoint and arresting the cells in meiotic prophase (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bishop <em>et al.</em> 1992</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lydall <em>et al.</em> 1996</a>). Addition of 1-NA-PP1 inactivates Mek1, thereby allowing Rad51 to repair the DSBs using sister chromatids as templates. This repair eliminates the signal to the checkpoint, thereby allowing meiosis to proceed and asci to form (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Wan <em>et al.</em> 2004</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Niu <em>et al.</em> 2005</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative abundance of total and phosphorylated peptides from a dmc1&#x2206; mek1-as SILAC experiment. Representative distributions of log2 L/H ratios for total peptides are plotted on the Y-axis, whereas an arbitrary peptide identification number is assigned to each individual peptide, shown on the X-axis. Peptides with L/H ratios greater than two-fold are indicated by the red box, whereas peptides with L/H ratios " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985128186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative abundance of total and phosphorylated peptides from a &lt;em&gt;dmc1&#x2206; mek1-as&lt;/em&gt; SILAC experiment. Representative distributions of log&lt;sub&gt;2&lt;/sub&gt; L/H ratios for total peptides are plotted on the Y-axis, whereas an arbitrary peptide identification number is assigned to each individual peptide, shown on the X-axis. Peptides with L/H ratios greater than two-fold are indicated by the red box, whereas peptides with L/H ratios &lt;1 are indicated by the blue box.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2125/F3.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/11/2125/F3.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465266" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Relative abundance of total and phosphorylated peptides from a <em>dmc1∆ mek1-as</em> SILAC experiment. Representative distributions of log<sub>2</sub> L/H ratios for total peptides are plotted on the Y-axis, whereas an arbitrary peptide identification number is assigned to each individual peptide, shown on the X-axis. Peptides with L/H ratios greater than two-fold are indicated by the red box, whereas peptides with L/H ratios &lt;1 are indicated by the blue box.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">A plot of the L/H ratios of the phosphorylated peptides shows that approximately 80% of these peptides are equally abundant in the light and heavy cultures (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Peptides that exhibit a L/H ratio &gt;2 (indicated by the red box) are potential substrates of Mek1. The data were examined for phosphopeptides from the three known <em>in vivo</em> targets of Mek1: Mek1 threonine 327, Rad54 threonine 132, and histone H3 threonine 11. No peptides for H3 were observed, perhaps because the abundance of arginine and lysine residues in this histone produces very small peptides that are not efficiently detected by MS. In contrast, several phosphopeptides were found for both Mek1 and Rad54 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Mek1 activates itself by autophosphorylation of T327 (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Niu <em>et al.</em> 2007</a>). Two peptides containing this phosphosite were identified exhibiting an average L/H ratio of 12.4 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). In addition, seven peptides were observed that contain phosphorylated T331, which is also located in the Mek1 activation loop and whose phosphorylation is required for wild-type levels of Mek1 activity, exhibiting an average L/H ratio of 4.2 (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Niu <em>et al.</em> 2007</a>). High L/H ratios could occur simply because there is less Mek1 protein in the heavy culture. This potential artifact is ruled out in the case of Mek1 because other phosphopeptides (presumably phosphorylated by other kinases) are present that have L/H ratios ≤1. For example, peptides containing phosphorylated serine 143 are present with an average L/H ratio of 0.72 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), supporting the idea that the large L/H ratios for T327 and T331 are Mek1-dependent.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Location of phosphopeptides on Mek1, Rad54, and Rec8. Sequences in red indicate peptides that were detected by mass spectrometry. In some cases, two peptides overlap. Amino acids indicated in bold with blue color are phosphorylated. Phosphorylated sites that have previously been identified in the literature are bold and green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985128186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Location of phosphopeptides on Mek1, Rad54, and Rec8. Sequences in red indicate peptides that were detected by mass spectrometry. In some cases, two peptides overlap. Amino acids indicated in bold with blue color are phosphorylated. Phosphorylated sites that have previously been identified in the literature are bold and green.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2125/F4.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/11/2125/F4.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465173" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Location of phosphopeptides on Mek1, Rad54, and Rec8. Sequences in red indicate peptides that were detected by mass spectrometry. In some cases, two peptides overlap. Amino acids indicated in bold with blue color are phosphorylated. Phosphorylated sites that have previously been identified in the literature are bold and green.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of phosphorylated peptides and their light:heavy (L/H) ratios for Mek1, Rad54, and Rec8. The average L/H ratios are plotted for peptides from dmc1&#x2206; mek1-as SILAC experiments containing the phosphorylated amino acid indicated on the X-axis. Error bars indicate the SD. A complete list of each peptide and its ratio for the proteins shown here can be found in Supporting Information, Table S1. For those phosphosites identified by more than one peptide, the number of peptides is indicated in parentheses. Green indicates phosphosites that match the Mek1 consensus of arginine in the &#x2212;3 position. Hatch marks indicate in vivo Mek1 targets important either for Mek1 auto-activation (T327, T331) or for inhibiting Rad51&#x2013;Rad54 interaction (T132). For Rec8, red indicates phosphosites previously identified (Brar et al. 2006; Katis et al. 2010). Phosphorylation of these sites promotes separase cleavage (Katis et al. 2010). Lines across each graph indicate a ratio of 1.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-985128186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of phosphorylated peptides and their light:heavy (L/H) ratios for Mek1, Rad54, and Rec8. The average L/H ratios are plotted for peptides from &lt;em&gt;dmc1&#x2206; mek1-as&lt;/em&gt; SILAC experiments containing the phosphorylated amino acid indicated on the X-axis. Error bars indicate the SD. A complete list of each peptide and its ratio for the proteins shown here can be found in Supporting Information, Table S1. For those phosphosites identified by more than one peptide, the number of peptides is indicated in parentheses. Green indicates phosphosites that match the Mek1 consensus of arginine in the &#x2212;3 position. Hatch marks indicate &lt;em&gt;in vivo&lt;/em&gt; Mek1 targets important either for Mek1 auto-activation (T327, T331) or for inhibiting Rad51&#x2013;Rad54 interaction (T132). For Rec8, red indicates phosphosites previously identified (Brar &lt;em&gt;et al.&lt;/em&gt; 2006; Katis &lt;em&gt;et al.&lt;/em&gt; 2010). Phosphorylation of these sites promotes separase cleavage (Katis &lt;em&gt;et al.&lt;/em&gt; 2010). Lines across each graph indicate a ratio of 1.0.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2125/F5.medium.gif" width="236" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/11/2125/F5.medium.gif" width="236" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2125/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465232" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Distribution of phosphorylated peptides and their light:heavy (L/H) ratios for Mek1, Rad54, and Rec8. The average L/H ratios are plotted for peptides from <em>dmc1∆ mek1-as</em> SILAC experiments containing the phosphorylated amino acid indicated on the X-axis. Error bars indicate the SD. A complete list of each peptide and its ratio for the proteins shown here can be found in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013888/-/DC1/TableS1.xlsx">Table S1</a>. For those phosphosites identified by more than one peptide, the number of peptides is indicated in parentheses. Green indicates phosphosites that match the Mek1 consensus of arginine in the −3 position. Hatch marks indicate <em>in vivo</em> Mek1 targets important either for Mek1 auto-activation (T327, T331) or for inhibiting Rad51–Rad54 interaction (T132). For Rec8, red indicates phosphosites previously identified (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brar <em>et al.</em> 2006</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Katis <em>et al.</em> 2010</a>). Phosphorylation of these sites promotes separase cleavage (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Katis <em>et al.</em> 2010</a>). Lines across each graph indicate a ratio of 1.0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Another <em>in vivo</em> target of Mek1 is threonine 132 of Rad54 (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Niu <em>et al.</em> 2009</a>). Rad54 is an accessory factor for Rad51 and the negative charge conferred by phosphorylation makes Rad51–Rad54 complex formation more difficult. A high L/H ratio for a peptide containing phosphorylated T132 of Rad54 was observed, consistent with being a Mek1 target (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). This ratio is not due to differential amounts of Rad54 in the two cultures as other phosphopeptides from Rad54 exhibited L/H ratios ranging from 0.06 to 1.35 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>).</p><p id="p-35">In addition to peptides exhibiting L/H ratios greater than 1, there is also a class of peptides in which the L/H ratio is less than 1 (represented by the blue box in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). These are phosphopeptides that are over-represented in the heavy culture, suggesting that phosphorylation occurs in response to Mek1 inhibition. In fact, inactivation of Mek1 in <em>dmc1∆</em>-arrested cells changes the cellular physiology, as DSB repair can now occur, thereby relieving the signal to the checkpoint and allowing meiotic progression (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Wan <em>et al.</em> 2004</a>; <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Niu <em>et al.</em> 2005</a>) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Although the inhibitor was present for only 20 min, the presence of phosphopeptides with ratios &lt;1 suggests that this is enough time for at least some proteins that are phosphorylated only after strand invasion to be modified. An excellent example of this type of regulation was observed with Rec8. Rec8 is a meiosis-specific subunit of cohesin, the multisubunit complex that holds sister chromatids together (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Klein <em>et al.</em> 1999</a>). The spindle checkpoint monitors the attachment of homologous pairs of sister chromatids to opposite poles of this spindle at Meiosis I (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>). Once all homologous pairs are properly oriented, separase is activated to cleave Rec8 specifically along chromosome arms, thereby allowing reductional segregation (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Buonomo <em>et al.</em> 2000</a>). This cleavage requires phosphorylation of Rec8 by several kinases, including Cdc7-Dbf4 and Hrr25 (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Katis <em>et al.</em> 2010</a>). Our SILAC experiments identified eight phosphosites on Rec8 that are present on peptides that exhibit ratios less than 1 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). Five of these have been previously identified by MS studies of Rec8 enriched from meiotic cells and phosphorylation of these amino acids helps promote separase cleavage (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>) (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brar <em>et al.</em> 2006</a>; <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Katis <em>et al.</em> 2010</a>). Therefore, our MS data suggest that phosphorylation of these sites on Rec8 does not occur until after strand invasion.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-36">SILAC requires cells to be grown in medium containing either light or heavy amino acids so that proteins coming from different cultures can be distinguished by the differences in their mass. Controlling the source of amino acids requires the use of synthetic medium. A problem for using SILAC for studying meiosis is that cells pre-grown in standard synthetic medium do not sporulate well. Our development of a synthetic medium that allows efficient sporulation has enabled the application of SILAC to meiotic yeast cells for the first time. In comparison to YPA pre-growth medium, there is an approximate 2-hr delay in the onset of premeiotic DNA synthesis after pre-growth in RPS-L. This delay may be due to the need to deplete particular nutrients such as glucose before entry into meiosis. Importantly, after pre-growth in RPS-L, meiotic recombination and progression occur relatively synchronously and with high efficiency to produce viable spores.</p><p id="p-37">One way to identify kinase substrates using SILAC is to arrest cells containing an analog-sensitive allele of the kinase, and then to inhibit the kinase in either the heavy or the light culture (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Holt <em>et al.</em> 2009</a>). For meiotic studies, a number of different arrest points are available. In addition to the <em>dmc1∆</em> arrest used in this work, <em>ndt80∆</em> can be used to arrest cells in pachytene when homologous chromosomes are completely synapsed and double Holliday junction intermediates have formed, and <em>pCLB2-CDC20</em> can be used to arrest cells in Metaphase I, after synaptonemal complex disassembly and crossover formation are complete (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Xu <em>et al.</em> 1995</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allers and Lichten 2001</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lee and Amon 2003</a>). In addition, analog-sensitive kinases have been developed for several kinases known to play key roles in meiosis, including <em>CDC7</em>, <em>CDC28</em>, <em>IME2</em>, and <em>HRR25</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Benjamin <em>et al.</em> 2003</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wan <em>et al.</em> 2008</a>; <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Katis <em>et al.</em> 2010</a>). An alternative approach to using analog-sensitive kinase mutants is to induce transcription of a gene encoding a wild-type kinase during an arrest. For example, induction of <em>CDC5</em>, the polo-like kinase in budding yeast, in <em>ndt80∆</em> arrested cells is sufficient for Holliday junction resolution and synaptonemal complex disassembly (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Sourirajan and Lichten 2008</a>). Therefore, by inducing <em>CDC5</em> in <em>ndt80∆</em> cells specifically in the light culture, a high L:H phosphopeptide ratio would be expected for Cdc5 substrates. SILAC can now be used to look at the function of these kinases (and others) at various times during meiosis.</p><p id="p-38">Our method was validated by showing that known targets of Mek1 can be detected with high L:H ratios from <em>dmc1∆</em> arrested cells. One of these peptides contained Mek1 T327, which is present in the Mek1 activation loop and is autophosphorylated <em>in trans</em> to activate the kinase (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Niu <em>et al.</em> 2007</a>). In addition, peptides with L:H ratios &gt;2 were observed for Mek1 T331. These ratios suggest that Mek1 may be the kinase that phosphorylates T331, even though this threonine is not contained within a Mek1 consensus site (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Mok <em>et al.</em> 2010</a>). It should be noted that the experimental approach of inhibiting the kinase for a brief time during an arrest may underestimate the number of kinase substrates. This is because high L:H ratios are predicted for peptides containing phosphorylation sites based on the assumption that phosphatases can remove phosphates from substrates during the time that the kinase is inactivated, which may not always be true. In addition, the data can also be analyzed to determine proteins with L/H ratios &lt;1, which may represent phosphorylation that is happening as a result of kinase inhibition, as was seen for Rec8.</p><p id="p-39">Although our goal is to define kinase substrates in meiosis, SILAC has other applications as well. For example, SILAC can also be used to look at protein–protein interactions (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blagoev <em>et al.</em> 2003</a>). By allowing meiotic proteins to be labeled with heavy amino acids, our protocol opens the door for many different SILAC applications to meiosis.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank Aaron Neiman and Nikhil Bhagwat for comments regarding the manuscript, and Xiangyu Chen for helpful discussions. We are grateful to Doug Bishop and Neil Hunter for a strain and plasmid, respectively. This work was supported by National Institutes of Health grants R01 GM050717 (to N.M.H.) and T32 GM008468 (to R.S.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-41">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013888/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013888/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-42"><em>Communicating editor: M. Smolka</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-06-27"><span class="received-label">Received </span>June 27, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-08-25"><span class="accepted-label">Accepted </span>August 25, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Suhandynata <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.1" data-doi="10.1074/mcp.M700468-MCP200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albuquerque</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolka</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bafna</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multidimensional chromatography technology for in-depth phosphoproteome analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M700468-MCP200%26rft_id%253Dinfo%253Apmid%252F18407956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjg6IjcvNy8xMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.2" data-doi="10.1016/S0092-8674(01)00416-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Differential timing and control of noncrossover and crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D57%26rft.atitle%253DDifferential%2Btiming%2Band%2Bcontrol%2Bof%2Bnoncrossover%2Band%2Bcrossover%2Brecombination%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900416-0%26rft_id%253Dinfo%253Apmid%252F11461701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00416-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461701&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.3" data-doi="10.1101/gad.1101503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamin</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokat</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herskowitz</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Control of landmark events in meiosis by the CDK Cdc28 and the meiosis-specific kinase Ime2.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1524</span>–<span class="cit-lpage">1539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1101503%26rft_id%253Dinfo%253Apmid%252F12783856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzEyLzE1MjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-2" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.4" data-doi="10.1016/0092-8674(94)90038-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D1081%26rft.epage%253D1092%26rft.atitle%253DRecA%2Bhomologs%2BDmc1%2Band%2BRad51%2Binteract%2Bto%2Bform%2Bmultiple%2Bnuclear%2Bcomplexes%2Bprior%2Bto%2Bmeiotic%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990038-8%26rft_id%253Dinfo%253Apmid%252F7528104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90038-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7528104&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY08600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.5" data-doi="10.1016/0092-8674(92)90446-J"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title"><em>DMC1</em>: a meiosis-specific yeast homolog of <em>E. coli recA</em> required for recombination, synaptonemal complex formation and cell cycle progression.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D439%26rft.epage%253D456%26rft.atitle%253DDMC1%253A%2Ba%2Bmeiosis-specific%2Byeast%2Bhomolog%2Bof%2BE.%2Bcoli%2BrecA%2Brequired%2Bfor%2Brecombination%252C%2Bsynaptonemal%2Bcomplex%2Bformation%252C%2Band%2Bcell%2Bcycle%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990446-J%26rft_id%253Dinfo%253Apmid%252F1581960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90446-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1581960&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HT07800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.6" data-doi="10.1038/nbt790"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blagoev</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kratchmarova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A proteomics strategy to elucidate functional protein-protein interactions applied to EGF signaling.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBlagoev%26rft.auinit1%253DB.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D315%26rft.epage%253D318%26rft.atitle%253DA%2Bproteomics%2Bstrategy%2Bto%2Belucidate%2Bfunctional%2Bprotein-protein%2Binteractions%2Bapplied%2Bto%2BEGF%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt790%26rft_id%253Dinfo%253Apmid%252F12577067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt790&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12577067&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181312500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.7" data-doi="10.1038/nature04794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiburz</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Rec8 phosphorylation and recombination promote the step-wise loss of cohesins in meiosis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">441</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBrar%26rft.auinit1%253DG.%2BA.%26rft.volume%253D441%26rft.issue%253D7092%26rft.spage%253D532%26rft.epage%253D536%26rft.atitle%253DRec8%2Bphosphorylation%2Band%2Brecombination%2Bpromote%2Bthe%2Bstep-wise%2Bloss%2Bof%2Bcohesins%2Bin%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04794%26rft_id%253Dinfo%253Apmid%252F16672979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16672979&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237778900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.8" data-doi="10.1016/S0092-8674(00)00131-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buonomo</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clyne</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Disjunction of homologous chromosomes in meiosis I depends on proteolytic cleavage of the meiotic cohesin Rec8 by separin.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBuonomo%26rft.auinit1%253DS.%2BB.%26rft.volume%253D103%26rft.issue%253D3%26rft.spage%253D387%26rft.epage%253D398%26rft.atitle%253DDisjunction%2Bof%2Bhomologous%2Bchromosomes%2Bin%2Bmeiosis%2BI%2Bdepends%2Bon%2Bproteolytic%2Bcleavage%2Bof%2Bthe%2Bmeiotic%2Bcohesin%2BRec8%2Bby%2Bseparin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252900131-8%26rft_id%253Dinfo%253Apmid%252F11081626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)00131-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11081626&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000090144100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.9" data-doi="10.1534/genetics.110.117523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callender</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mek1 suppression of meiotic double-strand break repair is specific to sister chromatids, chromosome autonomous and independent of Rec8 cohesin complexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117523%26rft_id%253Dinfo%253Apmid%252F20421598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvNzcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.10" data-doi="10.1016/j.cell.2008.01.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carballo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedgwick</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phosphorylation of the axial element protein Hop1 by Mec1/Tel1 ensures meiotic interhomolog recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarballo%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D132%26rft.issue%253D5%26rft.spage%253D758%26rft.epage%253D770%26rft.atitle%253DPhosphorylation%2Bof%2Bthe%2Baxial%2Belement%2Bprotein%2BHop1%2Bby%2BMec1%252FTel1%2Bensures%2Bmeiotic%2Binterhomolog%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.01.035%26rft_id%253Dinfo%253Apmid%252F18329363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.01.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18329363&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253818000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.11" data-doi="10.1074/jbc.M110.106989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albuquerque</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suhandynata</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A proteome-wide analysis of kinase-substrate network in the DNA damage response.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">12803</span>–<span class="cit-lpage">12812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.106989%26rft_id%253Dinfo%253Apmid%252F20190278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMTcvMTI4MDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elias</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">207–214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D4%26rft.spage%253D207214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.13" data-doi="10.1038/nbt0302-301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ficarro</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCleland</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stukenberg</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Phosphoproteome analysis by mass spectrometry and its application to <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DFicarro%26rft.auinit1%253DS.%2BB.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D301%26rft.epage%253D305%26rft.atitle%253DPhosphoproteome%2Banalysis%2Bby%2Bmass%2Bspectrometry%2Band%2Bits%2Bapplication%2Bto%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0302-301%26rft_id%253Dinfo%253Apmid%252F11875433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0302-301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11875433&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174142800050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.14" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.15" data-doi="10.1101/gad.1954910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Govin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorsey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaucher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousseaux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khochbin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Systematic screen reveals new functional dynamics of histones H3 and H4 during gametogenesis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1772</span>–<span class="cit-lpage">1786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGovin%26rft.auinit1%253DJ.%26rft.volume%253D24%26rft.issue%253D16%26rft.spage%253D1772%26rft.epage%253D1786%26rft.atitle%253DSystematic%2Bscreen%2Breveals%2Bnew%2Bfunctional%2Bdynamics%2Bof%2Bhistones%2BH3%2Band%2BH4%2Bduring%2Bgametogenesis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1954910%26rft_id%253Dinfo%253Apmid%252F20713519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI0LzE2LzE3NzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grushcow</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holzen</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> checkpoint genes <em>MEC1</em>, <em>RAD17</em> and <em>RAD24</em> are required for normal meiotic recombination partner choice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrushcow%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D607%26rft.epage%253D620%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BCheckpoint%2BGenes%2BMEC1%252C%2BRAD17%2Band%2BRAD24%2BAre%2BRequired%2Bfor%2BNormal%2BMeiotic%2BRecombination%2BPartner%2BChoice%26rft_id%253Dinfo%253Apmid%252F10511543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzIvNjA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.17" data-doi="10.1016/j.cell.2006.04.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kee</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maleki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cyclin-dependent kinase directly regulates initiation of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1321</span>–<span class="cit-lpage">1332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHenderson%26rft.auinit1%253DK.%2BA.%26rft.volume%253D125%26rft.issue%253D7%26rft.spage%253D1321%26rft.epage%253D1332%26rft.atitle%253DCyclin-dependent%2Bkinase%2Bdirectly%2Bregulates%2Binitiation%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.04.039%26rft_id%253Dinfo%253Apmid%252F16814718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.04.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16814718&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239104500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.18" data-doi="10.1101/gad.9.14.1728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponte</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halsey</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title"><em>MSH5</em>, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in <em>Saccharomyces cerevisiae</em> but not mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1728</span>–<span class="cit-lpage">1739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHollingsworth%26rft.auinit1%253DN%2BM%26rft.volume%253D9%26rft.issue%253D14%26rft.spage%253D1728%26rft.epage%253D1739%26rft.atitle%253DMSH5%252C%2Ba%2Bnovel%2BMutS%2Bhomolog%252C%2Bfacilitates%2Bmeiotic%2Breciprocal%2Brecombination%2Bbetween%2Bhomologs%2Bin%2BSaccharomyces%2Bcerevisiae%2Bbut%2Bnot%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.14.1728%26rft_id%253Dinfo%253Apmid%252F7622037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8xNC8xNzI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.19" data-doi="10.1126/science.1172867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuch</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Global analysis of Cdk1 substrate phosphorylation sites provides insights into evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">1682</span>–<span class="cit-lpage">1686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHolt%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D325%26rft.issue%253D5948%26rft.spage%253D1682%26rft.epage%253D1686%26rft.atitle%253DGlobal%2BAnalysis%2Bof%2BCdk1%2BSubstrate%2BPhosphorylation%2BSites%2BProvides%2BInsights%2Binto%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172867%26rft_id%253Dinfo%253Apmid%252F19779198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjUvNTk0OC8xNjgyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.20" data-doi="10.1016/S0092-8674(01)00430-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The single-end invasion: an asymmetric intermediate at the double- strand break to double-holliday junction transition of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHunter%26rft.auinit1%253DN.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D70%26rft.atitle%253DThe%2Bsingle-end%2Binvasion%253A%2Ban%2Basymmetric%2Bintermediate%2Bat%2Bthe%2Bdouble-strand%2Bbreak%2Bto%2Bdouble-holliday%2Bjunction%2Btransition%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900430-5%26rft_id%253Dinfo%253Apmid%252F11461702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00430-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461702&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.21" data-doi="10.1016/j.devcel.2010.01.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katis</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipp</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imre</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogdanova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okaz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rec8 phosphorylation by casein kinase 1 and Cdc7-Dbf4 kinase regulates cohesin cleavage by separase during meiosis.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DKatis%26rft.auinit1%253DV.%2BL.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D397%26rft.epage%253D409%26rft.atitle%253DRec8%2Bphosphorylation%2Bby%2Bcasein%2Bkinase%2B1%2Band%2BCdc7-Dbf4%2Bkinase%2Bregulates%2Bcohesin%2Bcleavage%2Bby%2Bseparase%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2010.01.014%26rft_id%253Dinfo%253Apmid%252F20230747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2010.01.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20230747&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276058400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.22" data-doi="10.1016/j.cell.2010.11.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sister cohesion and structural axis components mediate homolog bias of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">924</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKim%26rft.auinit1%253DK.%2BP.%26rft.volume%253D143%26rft.issue%253D6%26rft.spage%253D924%26rft.epage%253D937%26rft.atitle%253DSister%2Bcohesion%2Band%2Bstructural%2Baxis%2Bcomponents%2Bmediate%2Bhomolog%2Bbias%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.11.015%26rft_id%253Dinfo%253Apmid%252F21145459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21145459&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285182600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.23" data-doi="10.1016/S0092-8674(00)80609-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahr</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buonomo</span>  <span class="cit-name-given-names">S. B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A central role for cohesins in sister chromatid cohesion, formation of axial elements and recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKlein%26rft.auinit1%253DF.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D91%26rft.epage%253D103%26rft.atitle%253DA%2Bcentral%2Brole%2Bfor%2Bcohesins%2Bin%2Bsister%2Bchromatid%2Bcohesion%252C%2Bformation%2Bof%2Baxial%2Belements%252C%2Band%2Brecombination%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980609-1%26rft_id%253Dinfo%253Apmid%252F10412984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80609-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10412984&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081451000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.24" data-doi="10.1126/science.1081846"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Role of Polo-like kinase CDC5 in programming meiosis I chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">482</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1081846%26rft_id%253Dinfo%253Apmid%252F12663816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDAvNTYxOC80ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.25" data-doi="10.1128/MCB.06032-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marullo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cdc7-Dbf4 is a gene-specific regulator of meiotic transcription in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.06032-11%26rft_id%253Dinfo%253Apmid%252F22106412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjMyLzIvNTQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosebrock</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cdc7-Dbf4 regulates <em>NDT80</em> transcription as well as reductional segregation during budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4956–4967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft.volume%253D19%26rft.spage%253D49564967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.27" data-doi="10.1007/978-1-61779-129-1_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the semi-synthetic epitope system to identify direct substrates of the meiosis-specific budding yeast kinase, Mek1.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">745</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DLo%26rft.auinit1%253DH.%2BC.%26rft.volume%253D745%26rft.spage%253D135%26rft.epage%253D149%26rft.atitle%253DUsing%2Bthe%2Bsemi-synthetic%2Bepitope%2Bsystem%2Bto%2Bidentify%2Bdirect%2Bsubstrates%2Bof%2Bthe%2Bmeiosis-specific%2Bbudding%2Byeast%2Bkinase%252C%2BMek1.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-129-1_9%26rft_id%253Dinfo%253Apmid%252F21660693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-129-1_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21660693&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.28" data-doi="10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">A.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demarini</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Additional modules for versatile and economical PCR-based gene deletion and modification in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DLongtine%26rft.auinit1%253DM.%2BS.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D953%26rft.epage%253D961%26rft.atitle%253DAdditional%2Bmodules%2Bfor%2Bversatile%2Band%2Beconomical%2BPCR-based%2Bgene%2Bdeletion%2Band%2Bmodification%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199807%252914%253A10%253C953%253A%253AAID-YEA293%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F9717241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717241&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075005100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.29" data-doi="10.1038/383840a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikolsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A meiotic recombination checkpoint controlled by mitotic checkpoint genes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">383</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLydall%26rft.auinit1%253DD.%26rft.volume%253D383%26rft.issue%253D6603%26rft.spage%253D840%26rft.epage%253D843%26rft.atitle%253DA%2Bmeiotic%2Brecombination%2Bcheckpoint%2Bcontrolled%2Bby%2Bmitotic%2Bcheckpoint%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F383840a0%26rft_id%253Dinfo%253Apmid%252F8893012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/383840a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8893012&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VQ14400068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.30" data-doi="10.1016/j.cell.2008.10.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipp</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogdanova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okaz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dbf4-dependent CDC7 kinase links DNA replication to the segregation of homologous chromosomes in meiosis I.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMatos%26rft.auinit1%253DJ.%26rft.volume%253D135%26rft.issue%253D4%26rft.spage%253D662%26rft.epage%253D678%26rft.atitle%253DDbf4-dependent%2BCDC7%2Bkinase%2Blinks%2BDNA%2Breplication%2Bto%2Bthe%2Bsegregation%2Bof%2Bhomologous%2Bchromosomes%2Bin%2Bmeiosis%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.10.026%26rft_id%253Dinfo%253Apmid%252F19013276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.10.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19013276&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260886900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Middlehoven</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The pathway of arginine breakdown in <em>Saccharomyces cerevisae</em> Biochim Biophys Acta</span> <span class="cit-vol">93</span>: <span class="cit-fpage">650</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.32" data-doi="10.1126/scisignal.2000482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mok</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piccirillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Deciphering protein kinase specificity through large-scale analysis of yeast phosphorylation site motifs.</span> <abbr class="cit-jnl-abbrev">Sci. Signal.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">ra12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DMok%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D109%26rft.spage%253Dra12%26rft.epage%253Dra12%26rft.atitle%253DDeciphering%2BProtein%2BKinase%2BSpecificity%2BThrough%2BLarge-Scale%2BAnalysis%2Bof%2BYeast%2BPhosphorylation%2BSite%2BMotifs%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.2000482%26rft_id%253Dinfo%253Apmid%252F20159853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTA6IjMvMTA5L3JhMTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.33" data-doi="10.1128/MCB.00416-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Job</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moazed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mek1 kinase is regulated to suppress double-strand break repair between sister chromatids during budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">5456</span>–<span class="cit-lpage">5467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00416-07%26rft_id%253Dinfo%253Apmid%252F17526735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xNS81NDU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.34" data-doi="10.1091/mbc.E05-05-0465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Partner choice during meiosis is regulated by Hop1-promoted dimerization of Mek1.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5804</span>–<span class="cit-lpage">5818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E05-05-0465%26rft_id%253Dinfo%253Apmid%252F16221890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMi81ODA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.35" data-doi="10.1016/j.molcel.2009.09.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busygina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulation of meiotic recombination via Mek1-mediated Rad54 phosphorylation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DNiu%26rft.auinit1%253DH.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D393%26rft.epage%253D404%26rft.atitle%253DRegulation%2Bof%2Bmeiotic%2Brecombination%2Bvia%2BMek1-mediated%2BRad54%2Bphosphorylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.09.029%26rft_id%253Dinfo%253Apmid%252F19917248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.09.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19917248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272506200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.36" data-doi="10.1074/mcp.M200025-MCP200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blagoev</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kratchmarova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristensen</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Stable isotope labeling by amino acids in cell culture, SILAC, as a simple and accurate approach to expression proteomics.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M200025-MCP200%26rft_id%253Dinfo%253Apmid%252F12118079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjc6IjEvNS8zNzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.37" data-doi="10.1074/jbc.M506736200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onodera</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohsumi</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Autophagy is required for maintenance of amino acid levels and protein synthesis under nitrogen starvation.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">31582</span>–<span class="cit-lpage">31586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M506736200%26rft_id%253Dinfo%253Apmid%252F16027116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMzYvMzE1ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.38" data-doi="10.1016/0092-8674(91)90046-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Padmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Temporal comparison of recombination and synaptonemal complex formation during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPadmore%26rft.auinit1%253DR.%26rft.volume%253D66%26rft.issue%253D6%26rft.spage%253D1239%26rft.epage%253D1256%26rft.atitle%253DTemporal%2Bcomparison%2Bof%2Brecombination%2Band%2Bsynaptonemal%2Bcomplex%2Bformation%2Bduring%2Bmeiosis%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990046-2%26rft_id%253Dinfo%253Apmid%252F1913808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90046-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1913808&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GG55200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2125.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Methods in Yeast Genetics: A Laboratory Course Manual</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.40" data-doi="10.1126/science.289.5477.300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shonn</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Requirement of the spindle checkpoint for proper chromosome segregation in budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShonn%26rft.auinit1%253DM.%2BA.%26rft.volume%253D289%26rft.issue%253D5477%26rft.spage%253D300%26rft.epage%253D303%26rft.atitle%253DRequirement%2Bof%2Bthe%2BSpindle%2BCheckpoint%2Bfor%2BProper%2BChromosome%2BSegregation%2Bin%2BBudding%2BYeast%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5477.300%26rft_id%253Dinfo%253Apmid%252F10894778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNTQ3Ny8zMDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.41" data-doi="10.1101/gad.1711408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sourirajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Polo-like kinase Cdc5 drives exit from pachytene during budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2627</span>–<span class="cit-lpage">2632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSourirajan%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D19%26rft.spage%253D2627%26rft.epage%253D2632%26rft.atitle%253DPolo-like%2Bkinase%2BCdc5%2Bdrives%2Bexit%2Bfrom%2Bpachytene%2Bduring%2Bbudding%2Byeast%2Bmeiosis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1711408%26rft_id%253Dinfo%253Apmid%252F18832066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIyLzE5LzI2MjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.42" data-doi="10.1091/mbc.E03-07-0499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Santos</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokat</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mek1 kinase activity functions downstream of <em>RED1</em> in the regulation of meiotic DSB repair in budding yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E03-07-0499%26rft_id%253Dinfo%253Apmid%252F14595109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjE1LzEvMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.43" data-doi="10.1101/gad.1626408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokat</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cdc28-Clb5 (CDK-S) and Cdc7-Dbf4 (DDK) collaborate to initiate meiotic recombination in yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">386</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWan%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D386%26rft.epage%253D397%26rft.atitle%253DCdc28-Clb5%2B%2528CDK-S%2529%2Band%2BCdc7-Dbf4%2B%2528DDK%2529%2Bcollaborate%2Bto%2Binitiate%2Bmeiotic%2Brecombination%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1626408%26rft_id%253Dinfo%253Apmid%252F18245450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjIvMy8zODYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.44" data-doi="10.1534/genetics.106.064303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokat</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chemical inactivation of Cdc7 kinase in budding yeast results in a reversible arrest that allows efficient cell synchronization prior to meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1767</span>–<span class="cit-lpage">1774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064303%26rft_id%253Dinfo%253Apmid%252F17057233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzE3NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.45" data-doi="10.1128/MCB.15.12.6572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ajimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title"><em>NDT80</em>, a meiosis-specific gene required for exit from pachytene in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">6572</span>–<span class="cit-lpage">6581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DXu%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D6572%26rft.epage%253D6581%26rft.atitle%253DNDT80%252C%2Ba%2Bmeiosis-specific%2Bgene%2Brequired%2Bfor%2Bexit%2Bfrom%2Bpachytene%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.12.6572%26rft_id%253Dinfo%253Apmid%252F8524222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi82NTcyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.46" data-doi="10.1101/gad.11.1.106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Meiotic cells monitor the status of the interhomolog recombination complex.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DXu%26rft.auinit1%253DL%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D118%26rft.atitle%253DMeiotic%2Bcells%2Bmonitor%2Bthe%2Bstatus%2Bof%2Bthe%2Binterhomolog%2Brecombination%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.1.106%26rft_id%253Dinfo%253Apmid%252F9000054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTEvMS8xMDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.47" data-doi="10.4161/cc.9.17.13152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albuquerque</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suhandynata</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative phosphoproteomics: New technologies and applications in the DNA damage response.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">3479</span>–<span class="cit-lpage">3484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DZhou%26rft.auinit1%253DH.%26rft.volume%253D9%26rft.issue%253D17%26rft.spage%253D3479%26rft.epage%253D3484%26rft.atitle%253DQuantitative%2Bphosphoproteomics%253A%2BNew%2Btechnologies%2Band%2Bapplications%2Bin%2Bthe%2BDNA%2Bdamage%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.9.17.13152%26rft_id%253Dinfo%253Apmid%252F20855976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.9.17.13152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20855976&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2125.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2125.48" data-doi="10.1073/pnas.76.5.2395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zubenko</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">E. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Septum formation, cell division, and sporulation in mutants of yeast deficient in proteinase B.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">2395</span>–<span class="cit-lpage">2399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZubenko%26rft.auinit1%253DG.%2BS.%26rft.volume%253D76%26rft.issue%253D5%26rft.spage%253D2395%26rft.epage%253D2399%26rft.atitle%253DSeptum%2Bformation%252C%2Bcell%2Bdivision%252C%2Band%2Bsporulation%2Bin%2Bmutants%2Bof%2Byeast%2Bdeficient%2Bin%2Bproteinase%2BB.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.5.2395%26rft_id%253Dinfo%253Apmid%252F377296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ni81LzIzOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/11/2125.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2115" title="Characterization of Novel Sorghum brown midrib Mutants from an EMS-Mutagenized Population" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2137" title="Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/11"><span>Volume 4 Issue 11, November 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/11.cover-source.jpg?itok=oYWFGPiM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2125.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gdb6pLgZk7AGSm67gVeeJ6i-EMoFnQajAgLnwnJ-grg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/11/2125" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/11/2125" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/11/2125" class="active" data-icon-position="" data-hide-link-title="0">A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459703" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hkZVNufbMN3jK9zqqTXZHfWSyS9MKzdK6uw17rmYpOw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2125.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/11/2125" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zcw0z8Kkn-1_nHje6AF-LI28o6WBCNk-jCm5YT2bSPQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/11/2125" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2125.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459703" id="node459703--2" data-pisa="ggg;4/11/2125" data-pisa-master="ggg;g3.114.013888" data-apath="/ggg/4/11/2125.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ray Suhandynata</span>, <span class="highwire-citation-author" data-delta="1">Jason Liang</span>, <span class="highwire-citation-author" data-delta="2">Claudio. P. Albuquerque</span>, <span class="highwire-citation-author" data-delta="3">Huilin Zhou</span> and <span class="highwire-citation-author" data-delta="4">Nancy M. Hollingsworth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2125-2135;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013888 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459703--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ray Suhandynata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ray%2BSuhandynata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Suhandynata%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARay%2BSuhandynata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Claudio. P. Albuquerque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio.%2BP.%2BAlbuquerque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albuquerque%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudio.%2BP.%2BAlbuquerque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huilin Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huilin%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuilin%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nancy M. Hollingsworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BM.%2BHollingsworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hollingsworth%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANancy%2BM.%2BHollingsworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Nancy.Hollingsworth@stonybrook.edu" class="" data-icon-position="" data-hide-link-title="0">Nancy.Hollingsworth@stonybrook.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/11/2125" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/11/2125&amp;title=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/11/2125&amp;title=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/11/2125&amp;title=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/11/2125&amp;text=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/11/2125&amp;title=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/11/2125&amp;t=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/11/2125&amp;title=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/11/2125&amp;title=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459703/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459703" id="node459703--3" data-pisa="ggg;4/11/2125" data-pisa-master="ggg;g3.114.013888" data-apath="/ggg/4/11/2125.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ray Suhandynata</span>, <span class="highwire-citation-author" data-delta="1">Jason Liang</span>, <span class="highwire-citation-author" data-delta="2">Claudio. P. Albuquerque</span>, <span class="highwire-citation-author" data-delta="3">Huilin Zhou</span> and <span class="highwire-citation-author" data-delta="4">Nancy M. Hollingsworth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2125-2135;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013888 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459703--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ray Suhandynata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ray%2BSuhandynata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Suhandynata%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARay%2BSuhandynata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Claudio. P. Albuquerque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio.%2BP.%2BAlbuquerque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albuquerque%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudio.%2BP.%2BAlbuquerque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huilin Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Ludwig Institute for Cancer Research, University of California, San Diego, La Jolla, California 92093</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huilin%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuilin%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nancy M. Hollingsworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794-5215</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BM.%2BHollingsworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hollingsworth%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANancy%2BM.%2BHollingsworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Nancy.Hollingsworth@stonybrook.edu" class="" data-icon-position="" data-hide-link-title="0">Nancy.Hollingsworth@stonybrook.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2125&amp;rft.epage=2135&amp;rft.atitle=A%20Method%20for%20Sporulating%20Budding%20Yeast%20Cells%20That%20Allows%20for%20Unbiased%20Identification%20of%20Kinase%20Substrates%20Using%20Stable%20Isotope%20Labeling%20by%20Amino%20Acids%20in%20Cell%20Culture&amp;rft.volume=4&amp;rft.issue=11&amp;rft.date=2014-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Suhandynata%2C+Ray&amp;rft.au=Liang%2C+Jason&amp;rft.au=Albuquerque%2C+Claudio.+P.&amp;rft.au=Zhou%2C+Huilin&amp;rft.au=Hollingsworth%2C+Nancy+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459703/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459703/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459703/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459703/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459703/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459703/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459703/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459703/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459703/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459703/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459703/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459703/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/11/2125" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/11/2125.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/11/2125.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/11/2125.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/11/2125.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
