<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/11/2231" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/11/2231.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/11/2231.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/11/2231.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/11/2231.atom" />
<meta name="HW.pisa" content="ggg;4/11/2231" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Genome of the Chicken DT40 Bursal Lymphoma Cell Line" />
<meta name="DC.Identifier" content="10.1534/g3.114.013482" />
<meta name="DC.Date" content="2014-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Molnar et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The chicken DT40 cell line is a widely used model system in the study of multiple cellular processes due to the efficiency of homologous gene targeting. The cell line was derived from a bursal lymphoma induced by avian leukosis virus infection. In this study we characterized the genome of the cell line using whole genome shotgun sequencing and single nucleotide polymorphism array hybridization. The results indicate that wild-type DT40 has a relatively normal karyotype, except for whole chromosome copy number gains, and no karyotype variability within stocks. In a comparison to two domestic chicken genomes and the Gallus gallus reference genome, we found no unique mutational processes shaping the DT40 genome except for a mild increase in insertion and deletion events, particularly deletions at tandem repeats. We mapped coding sequence mutations that are unique to the DT40 genome; mutations inactivating the PIK3R1 and ATRX genes likely contributed to the oncogenic transformation. In addition to a known avian leukosis virus integration in the MYC gene, we detected further integration sites that are likely to de-regulate gene expression. The new findings support the hypothesis that DT40 is a typical transformed cell line with a relatively intact genome; therefore, it is well-suited to the role of a model system for DNA repair and related processes. The sequence data generated by this study, including a searchable de novo genome assembly and annotated lists of mutated genes, will support future research using this cell line." />
<meta name="DC.Contributor" content="János Molnár" />
<meta name="DC.Contributor" content="Ádám Póti" />
<meta name="DC.Contributor" content="Orsolya Pipek" />
<meta name="DC.Contributor" content="Marcin Krzystanek" />
<meta name="DC.Contributor" content="Nnennaya Kanu" />
<meta name="DC.Contributor" content="Charles Swanton" />
<meta name="DC.Contributor" content="Gábor E. Tusnády" />
<meta name="DC.Contributor" content="Zoltan Szallasi" />
<meta name="DC.Contributor" content="István Csabai" />
<meta name="DC.Contributor" content="Dávid Szüts" />
<meta name="article:published_time" content="2014-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Genome of the Chicken DT40 Bursal Lymphoma Cell Line" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The chicken DT40 cell line is a widely used model system in the study of multiple cellular processes due to the efficiency of homologous gene targeting. The cell line was derived from a bursal lymphoma induced by avian leukosis virus infection. In this study we characterized the genome of the cell line using whole genome shotgun sequencing and single nucleotide polymorphism array hybridization. The results indicate that wild-type DT40 has a relatively normal karyotype, except for whole chromosome copy number gains, and no karyotype variability within stocks. In a comparison to two domestic chicken genomes and the &lt;i&gt;Gallus gallus&lt;/i&gt; reference genome, we found no unique mutational processes shaping the DT40 genome except for a mild increase in insertion and deletion events, particularly deletions at tandem repeats. We mapped coding sequence mutations that are unique to the DT40 genome; mutations inactivating the &lt;i&gt;PIK3R1&lt;/i&gt; and &lt;i&gt;ATRX&lt;/i&gt; genes likely contributed to the oncogenic transformation. In addition to a known avian leukosis virus integration in the &lt;i&gt;MYC&lt;/i&gt; gene, we detected further integration sites that are likely to de-regulate gene expression. The new findings support the hypothesis that DT40 is a typical transformed cell line with a relatively intact genome; therefore, it is well-suited to the role of a model system for DNA repair and related processes. The sequence data generated by this study, including a searchable &lt;i&gt;de novo&lt;/i&gt; genome assembly and annotated lists of mutated genes, will support future research using this cell line.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/11/01" />
<meta name="citation_mjid" content="ggg;4/11/2231" />
<meta name="citation_id" content="4/11/2231" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/11/2231" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/11/2231.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/11/2231.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/11/2231.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.013482" />
<meta name="citation_pmid" content="25227228" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2231" />
<meta name="citation_lastpage" content="2240" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="János Molnár" />
<meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary" />
<meta name="citation_author" content="Ádám Póti" />
<meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary" />
<meta name="citation_author" content="Orsolya Pipek" />
<meta name="citation_author_institution" content="Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary" />
<meta name="citation_author" content="Marcin Krzystanek" />
<meta name="citation_author_institution" content="Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark" />
<meta name="citation_author" content="Nnennaya Kanu" />
<meta name="citation_author_institution" content="Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom" />
<meta name="citation_author" content="Charles Swanton" />
<meta name="citation_author_institution" content="Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom" />
<meta name="citation_author" content="Gábor E. Tusnády" />
<meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary" />
<meta name="citation_author" content="Zoltan Szallasi" />
<meta name="citation_author_institution" content="Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark" />
<meta name="citation_author_institution" content="Children's Hospital Informatics Program at the Harvard-Massachusetts Institutes of Technology Division of Health Sciences and Technology (CHIP@HST), Harvard Medical School, Boston, MA 02115" />
<meta name="citation_author" content="István Csabai" />
<meta name="citation_author_institution" content="Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary" />
<meta name="citation_author" content="Dávid Szüts" />
<meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary" />
<meta name="citation_author_email" content="szuts.david@ttk.mta.hu" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Adey;citation_author=J. N. Burton;citation_author=J. O. Kitzman;citation_author=J. B. Hiatt;citation_author=A. P. Lewis;citation_title=The haplotype-resolved genome and epigenome of the aneuploid HeLa cancer cell line.;citation_pages=207-211;citation_volume=500;citation_year=2013;citation_pmid=23925245;citation_doi=10.1038/nature12064" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. B. Alexandrov;citation_author=S. Nik-Zainal;citation_author=D. C. Wedge;citation_author=S. A. Aparicio;citation_author=S. Behjati;citation_title=Signatures of mutational processes in human cancer.;citation_pages=415-421;citation_volume=500;citation_year=2013;citation_pmid=23945592;citation_doi=10.1038/nature12477" />
<meta name="citation_reference" content="citation_title=Command-line tools for processing biological sequencing data;citation_author=E. Aronesty;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=T. W. Baba;citation_author=E. H. Humphries;citation_title=Differential response to avian leukosis virus infection exhibited by two chicken lines.;citation_pages=181-188;citation_volume=135;citation_year=1984;citation_issue=1;citation_pmid=6328748;citation_doi=10.1016/0042-6822(84)90128-4" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=T. W. Baba;citation_author=B. P. Giroir;citation_author=E. H. Humphries;citation_title=Cell lines derived from avian lymphomas exhibit two distinct phenotypes.;citation_pages=139-151;citation_volume=144;citation_year=1985;citation_issue=1;citation_pmid=2998040;citation_doi=10.1016/0042-6822(85)90312-5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Barretina;citation_author=G. Caponigro;citation_author=N. Stransky;citation_author=K. Venkatesan;citation_author=A. A. Margolin;citation_title=The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.;citation_pages=603-607;citation_volume=483;citation_year=2012;citation_issue=7391;citation_pmid=22460905;citation_doi=10.1038/nature11003" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=S. Boisvert;citation_author=F. Laviolette;citation_author=J. Corbeil;citation_title=Ray: simultaneous assembly of reads from a mix of high-throughput sequencing technologies.;citation_pages=1519-1533;citation_volume=17;citation_year=2010;citation_issue=11;citation_pmid=20958248;citation_doi=10.1089/cmb.2009.0238" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. M. Buerstedde;citation_author=S. Takeda;citation_title=Increased ratio of targeted to random integration after transfection of chicken B cell lines.;citation_pages=179-188;citation_volume=67;citation_year=1991;citation_issue=1;citation_pmid=1913816;citation_doi=10.1016/0092-8674(91)90581-I" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=J. M. Buerstedde;citation_author=C. A. Reynaud;citation_author=E. H. Humphries;citation_author=W. Olson;citation_author=D. L. Ewert;citation_title=Light chain gene conversion continues at high rate in an ALV-induced cell line.;citation_pages=921-927;citation_volume=9;citation_year=1990;citation_pmid=2155784" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Cermak;citation_author=E. L. Doyle;citation_author=M. Christian;citation_author=L. Wang;citation_author=Y. Zhang;citation_title=Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting.;citation_volume=39;citation_year=2011;citation_pmid=21493687;citation_doi=10.1093/nar/gkr218" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=H. Chang;citation_author=M. E. Delany;citation_title=Karyotype stability of the DT40 chicken B cell line: macrochromosome variation and cytogenetic mosaicism.;citation_pages=299-307;citation_volume=12;citation_year=2004;citation_issue=3;citation_pmid=15125643;citation_doi=10.1023/B:CHRO.0000021947.17128.96" />
<meta name="citation_reference" content="citation_journal_title=Trends Biochem. Sci.;citation_author=D. Clynes;citation_author=D. R. Higgs;citation_author=R. J. Gibbons;citation_title=The chromatin remodeller ATRX: a repeat offender in human disease.;citation_pages=461-466;citation_volume=38;citation_year=2013;citation_pmid=23916100;citation_doi=10.1016/j.tibs.2013.06.011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=W. L. Fan;citation_author=C. S. Ng;citation_author=C. F. Chen;citation_author=M. Y. Lu;citation_author=Y. H. Chen;citation_title=Genome-wide patterns of genetic variation in two domestic chickens.;citation_volume=5;citation_year=2013;citation_pmid=23814129;citation_doi=10.1093/gbe/evt097" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Flicek;citation_author=M. R. Amode;citation_author=D. Barrell;citation_author=K. Beal;citation_author=K. Billis;citation_title=Ensembl 2014.;citation_volume=42;citation_year=2014;citation_pmid=24316576;citation_doi=10.1093/nar/gkt1196" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. J. Garnett;citation_author=E. J. Edelman;citation_author=S. J. Heidorn;citation_author=C. D. Greenman;citation_author=A. Dastur;citation_title=Systematic identification of genomic markers of drug sensitivity in cancer cells.;citation_pages=570-575;citation_volume=483;citation_year=2012;citation_issue=7391;citation_pmid=22460902;citation_doi=10.1038/nature11005" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. Genschel;citation_author=S. J. Littman;citation_author=J. T. Drummond;citation_author=P. Modrich;citation_title=Isolation of MutS{beta} from Human Cells and Comparison of the Mismatch Repair Specificities of MutS{beta} and MutS{alpha};citation_pages=19895-19901;citation_volume=273;citation_year=1998;citation_issue=31;citation_pmid=9677427;citation_doi=10.1074/jbc.273.31.19895" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Grantham;citation_title=Amino Acid Difference Formula to Help Explain Protein Evolution;citation_pages=862-864;citation_volume=185;citation_year=1974;citation_issue=4154;citation_pmid=4843792;citation_doi=10.1126/science.185.4154.862" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Groenen;citation_author=H. J. Megens;citation_author=Y. Zare;citation_author=W. C. Warren;citation_author=L. W. Hillier;citation_title=The development and characterization of a 60K SNP chip for chicken.;citation_pages=274-274;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21627800;citation_doi=10.1186/1471-2164-12-274" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. S. Hayward;citation_author=B. G. Neel;citation_author=S. M. Astrin;citation_title=Activation of a cellular onc gene by promoter insertion in ALV-induced lymphoid leukosis.;citation_pages=475-480;citation_volume=290;citation_year=1981;citation_issue=5806;citation_pmid=6261142;citation_doi=10.1038/290475a0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. Hishinuma;citation_author=P. J. DeBona;citation_author=S. Astrin;citation_author=A. M. Skalka;citation_title=Nucleotide sequence of acceptor site and termini of integrated avian endogenous provirus ev1: integration creates a 6 bp repeat of host DNA.;citation_pages=155-164;citation_volume=23;citation_year=1981;citation_issue=1;citation_pmid=6260371;citation_doi=10.1016/0092-8674(81)90280-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. H. Huang;citation_author=D. Mandelker;citation_author=O. Schmidt-Kittler;citation_author=Y. Samuels;citation_author=V. E. Velculescu;citation_title=The Structure of a Human p110{alpha}/p85{alpha} Complex Elucidates the Effects of Oncogenic PI3K{alpha} Mutations;citation_pages=1744-1748;citation_volume=318;citation_year=2007;citation_issue=5857;citation_pmid=18079394;citation_doi=10.1126/science.1150799" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.;citation_pages=695-716;citation_volume=432;citation_year=2004;citation_issue=7018;citation_pmid=15592404;citation_doi=10.1038/nature03154" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. K. Johansson;citation_author=J. Brodd;citation_author=C. Eklöf;citation_author=M. Ferletta;citation_author=G. Hesselager;citation_title=Identification of candidate cancer-causing genes in mouse brain tumors by retroviral tagging.;citation_volume=101;citation_year=2004;citation_pmid=15273287;citation_doi=10.1073/pnas.0402716101" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. Kandoth;citation_author=M. D. McLellan;citation_author=F. Vandin;citation_author=K. Ye;citation_author=B. Niu;citation_title=Mutational landscape and significance across 12 major cancer types.;citation_pages=333-339;citation_volume=502;citation_year=2013;citation_pmid=24132290;citation_doi=10.1038/nature12634" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. J. Law;citation_author=K. M. Lower;citation_author=H. P. Voon;citation_author=J. R. Hughes;citation_author=D. Garrick;citation_title=ATR-X syndrome protein targets tandem repeats and influences allele-specific expression in a size-dependent manner.;citation_pages=367-378;citation_volume=143;citation_year=2010;citation_issue=3;citation_pmid=21029860;citation_doi=10.1016/j.cell.2010.09.023" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=I. Levin;citation_author=L. Santangelo;citation_author=H. Cheng;citation_author=L. B. Crittenden;citation_author=J. B. Dodgson;citation_title=An Autosomal Genetic Linkage Map of the Chicken;citation_pages=79-85;citation_volume=85;citation_year=1994;citation_issue=2;citation_pmid=7910177" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. A. Lovejoy;citation_author=W. Li;citation_author=S. Reisenweber;citation_author=S. Thongthip;citation_author=J. Bruno;citation_title=Loss of ATRX, genome instability, and an altered DNA damage response are hallmarks of the alternative lengthening of telomeres pathway.;citation_pages=e1002772-e1002772;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22829774;citation_doi=10.1371/journal.pgen.1002772" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_title=Rate, molecular spectrum, and consequences of human mutation.;citation_volume=107;citation_year=2010;citation_pmid=20080596;citation_doi=10.1073/pnas.0912629107" />
<meta name="citation_reference" content="citation_journal_title=Sub-cellular biochemistry;citation_journal_abbrev=Subcell Biochem;citation_author=P. E. Neiman;citation_author=J. Burnside;citation_author=K. Elsaesser;citation_author=H. Hwang;citation_author=B. E. Clurman;citation_title=Analysis of gene expression, copy number and palindrome formation with a Dt40 enriched cDNA microarray.;citation_pages=245-256;citation_volume=40;citation_year=2006;citation_pmid=17623909" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=T. H. O’Hare;citation_author=M. E. Delany;citation_title=Molecular and cellular evidence for the alternative lengthening of telomeres (ALT) mechanism in chicken.;citation_pages=65-78;citation_volume=135;citation_year=2011;citation_issue=1;citation_pmid=21822009;citation_doi=10.1159/000330125" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=F. A. Ran;citation_author=P. D. Hsu;citation_author=J. Wright;citation_author=V. Agarwala;citation_author=D. A. Scott;citation_title=Genome engineering using the CRISPR-Cas9 system.;citation_pages=2281-2308;citation_volume=8;citation_year=2013;citation_pmid=24157548;citation_doi=10.1038/nprot.2013.143" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. J. Rubin;citation_author=M. C. Zody;citation_author=J. Eriksson;citation_author=J. R. Meadows;citation_author=E. Sherwood;citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication.;citation_pages=587-591;citation_volume=464;citation_year=2010;citation_issue=7288;citation_pmid=20220755;citation_doi=10.1038/nature08832" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=L. J. Simpson;citation_author=J. E. Sale;citation_title=Rev1 is essential for DNA damage tolerance and non-templated immunoglobulin gene mutation in a vertebrate cell line;citation_pages=1654-1664;citation_volume=22;citation_year=2003;citation_issue=7;citation_pmid=12660171;citation_doi=10.1093/emboj/cdg161" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=J. Smith;citation_author=C. K. Bruley;citation_author=I. R. Paton;citation_author=I. Dunn;citation_author=C. T. Jones;citation_title=Differences in gene density on chicken macrochromosomes and microchromosomes.;citation_pages=96-103;citation_volume=31;citation_year=2000;citation_issue=2;citation_pmid=10782207;citation_doi=10.1046/j.1365-2052.2000.00565.x" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=E. Sonoda;citation_author=M. S. Sasaki;citation_author=J. M. Buerstedde;citation_author=O. Bezzubova;citation_author=A. Shinohara;citation_title=Rad51-deficient vertebrate cells accumulate chromosomal breaks prior to cell death;citation_pages=598-608;citation_volume=17;citation_year=1998;citation_issue=2;citation_pmid=9430650;citation_doi=10.1093/emboj/17.2.598" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=S. E. Swanberg;citation_author=M. E. Delany;citation_title=Dynamics of telomere erosion in transformed and non-transformed avian cells in vitro.;citation_pages=318-325;citation_volume=102;citation_year=2003;citation_issue=1-4;citation_pmid=14970723;citation_doi=10.1159/000075769" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=D. Szuts;citation_author=L. J. Simpson;citation_author=S. Kabani;citation_author=M. Yamazoe;citation_author=J. E. Sale;citation_title=Role for RAD18 in homologous recombination in DT40 cells.;citation_volume=26;citation_year=2006;citation_pmid=16923963;citation_doi=10.1128/MCB.01291-06" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. te Riele;citation_author=E. R. Maandag;citation_author=A. Berns;citation_title=Highly efficient gene targeting in embryonic stem cells through homologous recombination with isogenic DNA constructs.;citation_pages=5128-5132;citation_volume=89;citation_year=1992;citation_issue=11;citation_pmid=1594621;citation_doi=10.1073/pnas.89.11.5128" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=I. A. Vergara;citation_author=C. Frech;citation_author=N. Chen;citation_title=CooVar: Co-occurring variant analyzer.;citation_pages=615-615;citation_volume=5;citation_year=2012;citation_issue=1;citation_pmid=23116482;citation_doi=10.1186/1756-0500-5-615" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. Li;citation_author=H. Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Westaway;citation_author=G. Payne;citation_author=H. E. Varmus;citation_title=Proviral deletions and oncogene base-substitutions in insertionally mutagenized c-myc alleles may contribute to the progression of avian bursal tumors.;citation_pages=843-847;citation_volume=81;citation_year=1984;citation_issue=3;citation_pmid=6322173;citation_doi=10.1073/pnas.81.3.843" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. K. Wong;citation_author=B. Liu;citation_author=J. Wang;citation_author=Y. Zhang;citation_author=X. Yang;citation_title=A genetic variation map for chicken with 2.8 million single-nucleotide polymorphisms.;citation_pages=717-722;citation_volume=432;citation_year=2004;citation_issue=7018;citation_pmid=15592405;citation_doi=10.1038/nature03156" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=Y. Yao;citation_author=J. Charlesworth;citation_author=V. Nair;citation_author=M. Watson;citation_title=MicroRNA expression profiles in avian haemopoietic cells.;citation_pages=153;citation_volume=4;citation_year=2013;citation_pmid=23967013" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. Zuckerkandl;citation_title=On the molecular evolutionary clock.;citation_pages=34-46;citation_volume=26;citation_year=1987;citation_issue=1-2;citation_pmid=3125336;citation_doi=10.1007/BF02111280" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Genome of the Chicken DT40 Bursal Lymphoma Cell Line" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="twitter:description" content="The chicken DT40 cell line is a widely used model system in the study of multiple cellular processes due to the efficiency of homologous gene targeting. The cell line was derived from a bursal lymphoma induced by avian leukosis virus infection. In this study we characterized the genome of the cell line using whole genome shotgun sequencing and single nucleotide polymorphism array hybridization. The results indicate that wild-type DT40 has a relatively normal karyotype, except for whole chromosome copy number gains, and no karyotype variability within stocks. In a comparison to two domestic chicken genomes and the Gallus gallus reference genome, we found no unique mutational processes shaping the DT40 genome except for a mild increase in insertion and deletion events, particularly deletions at tandem repeats. We mapped coding sequence mutations that are unique to the DT40 genome; mutations inactivating the PIK3R1 and ATRX genes likely contributed to the oncogenic transformation. In addition to a known avian leukosis virus integration in the MYC gene, we detected further integration sites that are likely to de-regulate gene expression. The new findings support the hypothesis that DT40 is a typical transformed cell line with a relatively intact genome; therefore, it is well-suited to the role of a model system for DNA repair and related processes. The sequence data generated by this study, including a searchable de novo genome assembly and annotated lists of mutated genes, will support future research using this cell line." />
<meta name="og-title" property="og:title" content="The Genome of the Chicken DT40 Bursal Lymphoma Cell Line" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/11/2231" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The chicken DT40 cell line is a widely used model system in the study of multiple cellular processes due to the efficiency of homologous gene targeting. The cell line was derived from a bursal lymphoma induced by avian leukosis virus infection. In this study we characterized the genome of the cell line using whole genome shotgun sequencing and single nucleotide polymorphism array hybridization. The results indicate that wild-type DT40 has a relatively normal karyotype, except for whole chromosome copy number gains, and no karyotype variability within stocks. In a comparison to two domestic chicken genomes and the Gallus gallus reference genome, we found no unique mutational processes shaping the DT40 genome except for a mild increase in insertion and deletion events, particularly deletions at tandem repeats. We mapped coding sequence mutations that are unique to the DT40 genome; mutations inactivating the PIK3R1 and ATRX genes likely contributed to the oncogenic transformation. In addition to a known avian leukosis virus integration in the MYC gene, we detected further integration sites that are likely to de-regulate gene expression. The new findings support the hypothesis that DT40 is a typical transformed cell line with a relatively intact genome; therefore, it is well-suited to the role of a model system for DNA repair and related processes. The sequence data generated by this study, including a searchable de novo genome assembly and annotated lists of mutated genes, will support future research using this cell line." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
    <title>The Genome of the Chicken DT40 Bursal Lymphoma Cell Line | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jw2HPNi4CNnwxBYDh4Y0Ig8dOXoJz8SV5E9UgBlxdX0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459712","apath":"\/ggg\/4\/11\/2231.atom","pisa":"ggg;4\/11\/2231","ac":{"\/ggg\/4\/11\/2231.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/11\/2231.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2231"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2231"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/11\/2231":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZuyguJqtLW1xU1g2WWlQqgCgVIC4xXbSS6auHC3k+RoTznROEQkcH5NKF\\\\\\\/IM5azzMmfqlco\\\\\\\/nIc5jnHF1TQWI1nYfVZc6mXC4VFc4H0kZR5NilH2M9i0v3lxMZCuzfgq7Z94dUScxQMJ1K\\\\\\\/n7YzfEVYQ7L5N1hwhCPoTm0ivFbeCnRwyqoIypIfzMnPPHbePESOwVsXtBkheYQeSxyWV4pKTRHIiL1NEx8CPnnAn5UqXFFNTDDb7vQ8m2QIpb4dnDbgkY8BLO1uz6R+MWhF153JJN9FkX0wIPvHiSmyI1Af\\\\\\\/bP6GRJuySnPRwlMayeb12Rh0Wk1cy0a62K8\\\\\\\/bqdEB5ssOrIqpTT4uQxdi+k1b05tkRCV7ye8YvgIq2QKd6KSSl8UKHxdOI7NVmcd5fdBbs9eslHGWp6jDpksJAw+4\\\\\\\/CtSTePR5513RQowCu6E+VBLaMgxVj3OSms2ohrq6qfKy9UL0vP5sbF7P3eRpP3mYR1G3c4CHQWv1GfPGCTXgNRX5JEqUB8VH6e3JjoNJ4KBO63ywZZ2Y06+8vNKGFHLKtViaf0hkoopDgw1UiRHBpEdR3soCkxwuITRNlgLbQV\\\\\\\/twOV6kHmplvUY3d6H\\\\\\\/540RzkuaFlA0cMrvsaO22nEFBj3iTETPl0TlsKB6uUTn5VE172v8GKc2ilbEowh2sJmB6pW4suSE4q+FuGDwesGe5He7wrVIu9+LWPwTgvXmULXasrXFMJ3AKawbuIbOAZpuw\\\\\\\/OI72H0wfCMyi5WdQZXwl9djr\\\\\\\/PB5xJ2XET8WHRyYyC0g105wnBeySwdiQrWvS8pzC2lZUZIPFthPOFMF55U9tCd0hQZUa7avnh7uW9HAnGhcCW8oX5Njyt+A0VstRPz+tETcLLh5cYj8t8beGxnGJJV3bTzJHA48haFfjnVmy9jmaKO2TLvOvqGrVtpoQve3QcIfe\\\\\\\/grcdpJ5ATSrfKSL0aEtVSYU0u6xCaACG2APQHNxEvCYMzRa7X70QsxMT1tB0BRmuVobd1MEk6h51Td287AUdAzFMbc88gGUNyccyfLwrMI8ZrR2XxDnYFUFvQ6BmjvFMg\\\\\\\/AqIXxWWcY7p42CynufnwngnUFno3vNfU7W36H5mw+82jJNQS3wD6Zh1ADORN7ofam0mA9ZB3Ev5Vvn\\\\\\\/MklmiUJ8vfudEdZ3IcrRGOLmrMS3WRIEJPbDc3eoteoX+jfuGIkQWIRhvSWGMr4oqnUp4jeuDpATxd6HrFcdZmxVBU1q4OEGyaVro3dbUtGAwtq6jjI1A7ltP0+v\\\u0022,\\\u0022iv\\\u0022:\\\u0022YxEZlRwVvW1frmuFwzSizw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002216929c148d3016d40c3939d4f4e3b755f91e8ddfe4e0c6094a7f203e201150cf\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kIsnvn9wVmI1ulE12sBnUK1QoyVF9WEKEr757KtC7K\\\\\\\/wU1IU6HwaSv+GhIrM3VcErPZvV6HDulHQEQk8LviC0HrmzOzmw6\\\\\\\/wsOyYNTQ+77greMNojLuMrBk3JzmSuRBS1RDmV\\\\\\\/tm5z+tR+Q2nRQENf51ElUqcotqBMcMqg0V74wEGw6xUpscq5bVpPOkMAwTSzp0r+SpaCNWpcOvbedWrmHHkr00jaz5wMHUQ4Fr5iyALpCSplk8qHyFlu+Gn387mf3hk4GVszhI+5Dr+Ki3d50Bohg3LBYBrOSVeY4uGfUmJRcK1NIHb6bMhAXJiDQ9cfF4Y\\\\\\\/FE7pah+kfyY3IPntDzaHLQQHXBfzqO1l04uXex9f4fVlTqtz4eunN6R1TlLu9P9KVGqloKcMTL5sRqJjzTS7nYB0l+SMReDs\\\\\\\/UXnbwUjZNDHDADTkcAqTF1JXSj+gBCjybF3Hvc04XEVUJybnXl3LsnJ8RaZv36wZjU5C6SzrF4T+cOR3\\\\\\\/XCv4XGX0IF7Q0lhMxSg3+88UHwAw14q3TsEwUtd710gExmLEUH8h5ZepVPHN6waikgt13a\\\\\\\/u88dHdvVMORK\\\\\\\/48jr0iAfopLlzy4zKg8J7uionM3cnc4Fzmv6yJai76Vc2UhtWJxD17t1p9apku8ETxVmY9jp8Ayb3RihMNmeA\\\\\\\/9df0DTVf5nSRUSJnSU\\\\\\\/g1r8XWCd8eUUKsSoI3kT699PSfm2gdzxYsBtdfFIzq\\\\\\\/axix8xCpmmBpK6Cg82LRBce\\\\\\\/3AoLLNnQIBVpR8+T5Rgtq6DkKhMDjywnsZ1a9AgtVs9jCPeVVKEoTiluUu9visU5e\\\\\\\/Ls5aV9m66iGdROJb3viGCepLJxPn6aORuXhXmSuHgHyToO72Svd6rb\\\\\\\/ihdU2EZtEvLfqfGUNmuKyAXGGOyhnU301VjiQ5+UCDcjs0tb3q95KrR8svJaoR+xMrTO0x+IxhFfqtJV7BcUTVrKJGL9bbJqnNrdOzNRKuDqQZHDPSooy4hpKPOE30S8KhCcHhkSHqmcHjWxeQ6J4BGz7Hc2MRKceXlh7LHenlzcAZtnarz9gUikpRpx0gcgy7SUepYtNTjHkKC\\\\\\\/zJHigAz0T\\\\\\\/7nQCTjJecuGRd5YaNIMEERSC4npIofm\\\\\\\/lSWVQoUYYp9E\\\\\\\/H65IaKaEvO6SAIg2GT9IwuL9Y7snyUkXiRTWyQE761cN1bsh17\\\\\\\/pMuc40sVzrXvJaXkP89nLlXqRTGuVy9TXtrClOZ+msLJH0TnLub8WSFi9npkJAJji2aov2AP8ZrXyabOmI0yvappqWYYwE6nJ6JMxoeqJUf8j9GGpJLXrCoreVgEmz8UXtzHCLnSSmsz5sxatDa\\\\\\\/YtQE7\\\\\\\/nXsoFRXCbKeRlS\\\\\\\/duaST2Vq38qrnRFMU7+ZUYu2jguSf25LvoM932be3RKccYXsPaPX4odzp2UWLgPoteXkczX9BjKoAL7zbe8UQaiItCwUasVcIaJTz8gADT+aF5Mc+YBfupe3ZGfycEbx3LM8PgrMk2LW4e1\\\\\\\/GV+OuFzkxwW5Cg2vMVQb2+am3z5VgTF09q\\\\\\\/\\\\\\\/frrhH4+2k2a4Xpa1dCx6DIMsKnBWTtmdLGS\\\\\\\/zNqtzXlCSraVudNyBCpP7CwlG7vHE9pH9h6ilqzhBORE5thAkYTT1zOfdV0\\\\\\\/pc7ng948XclYyWX\\\\\\\/g94Cs587wwfdayWd0iO9rUlO9IMykNnszQZUsAb430TV0uRNi96pWT6bmXKTuLuf20QP+WemkP4oxw1F\\\\\\\/9tSLWZhLAj1ceFvm7HJtOB39k9jpg+Pjb5rTZAdiUtnmRuGuDNBUBTgtWM1\\\\\\\/LHNE0zmNaWiRiE0Sz8iQyYmIqaNyMteHvd2iNw3ya5Gy0X3V47X5EJVobeeYXzXJ7pHpwEfhfSlBbLOK2v7gvU5wrUvDZHVI\\\\\\\/5422x53bXd7CpLHgcvB5LGF\\\\\\\/HkweR8+T\\\\\\\/fltnSNrZhd5\\\\\\\/i\\\\\\\/+kcvPWmuiSOplpwVNn5W3m1B\\\\\\\/REWgnDbFl5bWfK0Bw4Y5PSgKQS\\\\\\\/GkZTHarm9YbxcEDeIRWgEgq1sdxeSpm0YWehiwYg2pBZd+3zAo3xgg5CEO52MBcw10lfPbFEmu9heYYZVU+laUsB7VOrh56EWxseXpb2ZD7HafeLYRtDX+jLdR3qA1xhBH+kwW3iyHgZawREqMRg+tpaCaMVqL9CH9DZilr+lEKpW+3FkNjbuu6sSRTLLi3Oj7h5QSDalBm9Hlw1EcoNQyTx63EMOHqlM575gvStXsh2PyGIo1jpJNj9IL4pyjwJ1qIgO5kGcOljRV0\\\\\\\/IX4XfiTTVMCinUH2JbS0L09BWJUHW2Q5nEnRCAolWLWvwiSWK9wh0QkedFC7IPm4BMdXGDJIiF20EBXHgJXE2hnO6n\\\\\\\/l6XSCU4efUAfZ0BWwobCA4Z4RbvW4sdyCHGweWkh\\\\\\\/lgGrfNGqw48KhlzvV99vM1EeQpirZBNONSx32ZUPbhSMtJUN6A\\\\\\\/FJxtQbz7ke7C3WsX2cuMUQ1vuQiFGHZqiQrlUH58wCDR2\\\\\\\/lgJM7RUqwS4Mz++LsEDvQ8sP8yr3fI\\\\\\\/uGKlZIn09RAzxsAoQx+8LdS3egtU0u368KzBxLf7mmNW2WX9M8G5FGLWlsOXKameb5Cfb8lP20g7KhAFKQx1C0g2RlbnhIcf5EJW3\\\\\\\/eXF\\\\\\\/2Kxn+9eHYlaytHHM0BfaMJro6VO\\\\\\\/l\\\\\\\/xgDskcLmLSPwGztAPulT5bgEb+Hx4p37+ps1Ptx5dqXU4tyLrbLur0fkPOjex\\\\\\\/VS4uBZvFH10tHb9U68fItLkeuK7yxMIo5DoiJLq07UMzIy2C7wJGKNOnoiyFhNtG3f3EIUPjYXP3meKvs9awUjfZCcE95c\\\\\\\/EzkbBH4IHcrRUFCT0Z+LbTIksamIwGAjIgf3pJmEXjsit\\\\\\\/\\\\\\\/Qhr4+tBHeiJeGcAFAgoprXRaKbjXODpZ32c1Ig2+d8ub3r4vgi40OTIdiZDfMd5OtA3fBdJ3jKYU3HAxOoUG03i\\\\\\\/d3F8W9r7Ex\\\\\\\/yjyEW8dtuSl5\\\\\\\/lCpFx1\\\\\\\/0DqblSetSwHGEWxvgd2aeAVp8MoYRpeM6E8ouXrx4f3klm9I8Z7W8h3CUNssA1D2uevRlcFiNwUNZdn7f2S6TRZkpgohjke1acVMEWZHU6\\\\\\\/lKi8Sb2dod+ODQo2mgGNCSm\\\\\\\/tvablZJcJg1QBoGZWg8VfZ\\\\\\\/Ms6nDTzW9PusdklyTy3FdPNSsyDosMf5bxTuHWZaF9Q4\\\\\\\/dC48wYMqzmNKY9ssqeK9La6i7cEgTYGHn820kXYgoddvQFopbHI+QatRnf5MdDxE+ENaAlIVeivtEOriYmkhvuwD7fhdTRzSBXEzTvRftaZUDFeJ+\\\\\\\/Tinwl0eDWWH9GZK8kDi3uuRLanYj73ymRsOuiHcuYzXQg\\\\\\\/BJjqyhXpB\\\\\\\/e6Br0g5RPMlGgSIm7kMaZq\\\\\\\/KpRjM6sQ6+qUGYiKLx7AEEpZuwNxG81ZtM4w6TEsgjz99a3JWh5+P\\\\\\\/DKYb\\\\\\\/8Wmbw5Umxu+BAaFy5E0s7MkNAJh4R+gdmgSbh2WdFP42rWHijbLoGgzaRQ90yjgrhFwZUAfXA2OfR8NCRAFVeps6c2hvge46R9ynb3xtHYtifuDhaaTPDcFTvciEpySXw\\\\\\\/sy6esmOcvbZRXgxP+5dFDdL8UnNShplGG\\\\\\\/xLDT0wtdtaqg4Cfk8DYxpGXz9dqChOy9MXQW2N+ZctbxyWLDYAskjOt9Z4Lth4CF0QYCz89O2b\\\\\\\/yKb+f6aBhqUqrrxTvoxQ1FLBYUo6ZwzVXajCPDcjAuKUnqDO7wgzm4TnXb8lMNrKdlpBM0TXzR+jb4tIGqfDxi3URp2UHsOxBe41oddUtgvH9ulsnnN\\\\\\\/+km15Pn0VqHMuJ377VqRsxvEPBwpv8LUhOkVe1t0xq55ZJUbY\\\\\\\/XoVzS8J5xGZT2T+A+ERqBQEuZgzkdDE3y2iKUS7Qp5brZgZLbKOkmJOVs1ZGvX07tjoaQakI2+yeFqRhyK2jfgzzOoa+1we5uryJ6ZddI5snQd6qZJYDyInk3FVqbgzEtBrbxCyMZiMywR84QsgbMGnV8N1UeJfDhXY7r1zkrprN3cZhWU6BI\\\\\\\/PR4zyvP8DZQZTfgiiLx1jLMFI5oyw\\\\\\\/7WxypDnhR6To+ZdzffD3O9JlzfFlVgE65ICWNQ0OrJU6UBVYFda06hMUD\\\\\\\/nVeFZRwF\\\\\\\/Srga8axBrA3ZW0JiqHNz0qXaRpP+ykXX8LhFT5XY39M8Azes4pSgUGXZmwaaL1oq2xMHFqpgjjpvDHmiyB\\\\\\\/3uIrcO3s+AKA8EC4RiAYEJAdNayhJsyK1xLIjsKJ5czMMxwpQAkHSNa2dhsQC4A9\\\\\\\/tN4kWGsZgqsoLbQvblNEsFwYwZjCVmCof7R2ZLf66EuFLb\\\\\\\/+UDT1cgXCQrHBdvagzA7S\\\\\\\/H8y48ZtfB8oV0YxZEvASPB3RS92\\\\\\\/\\\\\\\/\\\\\\\/ss65wwAJqQLISAH65y5mck+YM33pdS8JINAHCnXlg2CqOkDgaGW4g47Seb1WCbCbX12CiJHECXSNM9ck3lI1XoZkDaBUfm8\\\\\\\/zWHazY+gR4lOQA4bTpJTqw9QvgrEN2ZHKod8kj3Yzfyyx+mv\\\\\\\/BTDN96g9TknnmLn0oqeBNLq23He2Xmripg0wCsIfWLEFaS++ms08Ho+aHsOKDFPS1+dvya2XlZHJaF4uLm8rJUEu\\\\\\\/uWeLOJTjgDXb\\\\\\\/LqEF03amokPrIt\\\\\\\/MCl+\\\\\\\/Z417bLBIxM8HH2lZiTUNUUUwjB2t3LaJzLYjLd7ohsTYAHa7mJM3jjyMcySLFNrpV\\\\\\\/ygnkJVkZhFKMjjUoqK7zuHBzxJz2kVVIXZHRPGB7Bdrj8ETEqvm0nMb2V5kWuWYBLN2sCy8lg+q9FiTrB33TceWlag8e6tltwF50oWLxGk5TISDQh+jF+jF9QAeNKKJgvhE\\\\\\\/nx6euBJkL5MTA2ClMNlzyIlHAKprXhUrr7a+czvoh2mSAaGwYehaaZSiFJ+Fw+FRibZCyCb2sDxcifTw\\\\\\\/lO5lUUbHi8dAfFalEjwuKlOHZyAbABvqFpt5uHblTxqGSTaoqXa3Pfdy9e5lHyUw8QKOodwioXMKaTOhD9MUPtwAW1AAxzg2TBI5QdoSkfvvCI7FGeGnzjNEdjp5AfNezSvfBTDSRsW83WPFxruVoIPx8ejMJfuqMKl8Izbs1AELaNwtV88wrkKDkV+QlZYDROlDlb6tD4ybBlpd4I9DVrjEF2wb52cm7DEF2dGdxz8K2lUTmJRPi5+Km1FP3QUzQMRmp2oHlmuFe6ZZVaQqR6tnki4M38Tg2MryHx5dt+UVoKbrVRKoq8XDTERBK8qMnyh+D6+IkH8vWAOX16eOrkoIps8+OnumKAzJWClkbHpJxFVkxv7xt5xUIfcD\\\\\\\/DaTQnYTxVz7JnCLl1mpqatq0fCZXBArheOrTRpcwK9iZga8V8Kb2cJTvJHwjKFVMdWrU8kjZ5\\\\\\\/01GDkIpTZuCo9y51A4AajUszDwxeN+kit8LQmtcxDNWy1JzTkCshBToWqamHQpQOyuC1GLW6B9x9tZN77IGYSdTlQrhy0AnojO0vJh0i0MahkfBIUil0m+kEX1GJt27ehWCk3GFI9KnTiaquUbequhwkqd60s1foj3NF2YFxvPCtAEK5vWjrXThqeu+3cRZOVtvkmSx6VCn2CtqAjChO316xqrlI4x+GA3KvNKJa8aruCxU4nnrj7t+sc1oLJXUzu2rUrvGPr+NtuAgd\\\\\\\/aLlFahPtzjVBFX\\\\\\\/jcGB3mgQLFMvEnpdhIT9OIK\\\\\\\/bQyOOeDNqd+AJuD\\\\\\\/RwlOtGKwzrMmPxTL9S9ZX1Zaz57FhpvDayTxXjdsvoGl8+WpClPKUxmkQ5okhg\\\\\\\/Zscdyi88ei40+EqsAg8leNJ999Ce47zY0Or4gz+aRm6lWzL8HrNC9FnFCQmvxyYjidqkxhut4tJoXwUz4cIOXn7z7Jgl7xyks0pbPc1aULmE+mOTomOaTQdoBENDyOla3fEsiP6+XKu6iwG27CdRj4WoAr4RFEmpELreduqscmXyTpvDXe6dVpvfEEEU5Hj7Qp91x74yhAkx8je15ZQlHG3FzWbgdupMhkZgqqcGG5umJQKhjsKuIn5Aj4FHu3exPUgzjUmjxhAT2IuS1g0ZmA7GYD8Li7FwI25RRx2dyHlN6BKmIzuHjEW5Fl66\\\\\\\/UiI1cUXIukDHWUqZspy4DtAEGY4SxxGIxscQukMhfwKI1FY7bnxX8TsOMLRaCPyzTLD+QrYJqSvoNEtrau\\\\\\\/9txmvApX98YR4MfaJJj5N8i5pvsKhZhu1aoP1elkVOeC5DjtDoAR0hKpqJwdtS41YnDd8PnRcQboAL697PM74cIXdxLbYGWlpIO9aYLhNRnv+jmf6XEEg+5vxQjjQeCRpVGvIaRVZGW2gN3gfUPRuzyBOiS81gw79FyIJydryB95hG361+u8dGmNVlbPA\\\\\\\/MXeDV2T5locy6HR4Ijr5GszQi2odSAEDG555AZwLmOmzDWk+TLtxhbC55Ym5\\\\\\\/qauILyG6bifyna8kzyQtAlpEU5ouItaSx9r3PfKIMW3PvPqyp3n59yJFUZJ7MnAKUOOfTotE3VwVwpcmRQq1eDD\\\\\\\/OVtuFREOKJnqRKivaQrkSRRuoarsR2e+FuC\\\\\\\/Pzv2rFIRVxwEdq29j0e9FD3c07T3WYeKQifxuu\\\\\\\/sNMfAO\\\\\\\/1\\\\\\\/H2GHbcC\\\\\\\/bSWajaf7NaBZNyREwzUUdFowAtahNctM59sRfByuL\\\\\\\/ANCg9VS7R9K3TEajH3x4j3fg9fWSu6+c+nIPzfC5rqJLVEex0SOeTdA5Rb3iFUkDwe1cV0oGGcGmkZXh4iJWRYjSFFK0nY8MrnIsNoSQqqBhvS42NWBpoTuQmXUQsdVwAk\\\\\\\/6XmBEYmfwEyfSg6hTC0B+pyvxtek+ex+1tku0xqdd90iwmdTDXUKfV2vruCDB6C9Vc6BqpFeipBFdtchZ7TkPWlLi+p26JmsRM5qU2S7khBo+nCN1bKadgFTkvOJ4Dyq1cQOIXM5aAft3+FY07hddVWqSFKorRiXUcqtWdIF\\\\\\\/+uYJXXxGtXwzKloqRW3RUW7eRTTYfUhPv\\\\\\\/nDHKq3oCfETW9s4gpWUIQw3yowHDJempbal+ZMu1rk7NH\\\\\\\/bQbOV2q1QQsV64hss2yMCVaWf9jfh7aBex4HOEb7VNtbIoJtW+0C9F2F0OZ6QPI8zhi+JgOut6QXKX8GES2ua01y\\\\\\\/ySpsFAk6IUkVuhhKUUgEOBMBz+t+Amub3l5eA4vm\\\\\\\/PEiHEDzESQLZJ8sVoimK1+6hlXFlem1B7gNIcyDvXMuCn+a+Lekl+er\\\\\\\/lZE60eWAub06r30BwmtMogp0eqm353alyQry38cdm5Cxb6pHujxM0pZkVzQYqirkGMFDFG1V8SnjWb87NNaVkmsekzCAu1WaNfxXo++mq6wFhHE4l4LgLhnEGccLh9Pvv\\\\\\\/9RjdX+Ti2\\\\\\\/TVNg4QYSwhb1mgHo6gffUn19vnffae9oGYG\\\\\\\/k9nYSdSpJPrAE2QDCuRxfYw5VuvaaXMx0bvIKt4966XlCSJr4lDgc3wj5X86\\\\\\\/MiRnoWaD8R9C\\\\\\\/HYvSF08OOzLiPXEDi46DXKDddyxWDzFpoec1E3akaCmsb+n8IwvAR3Ke5k7jgjn1xg2AufSa8CtJperT\\\\\\\/88VWahBA0vo\\\\\\\/26XMGLBaF9B\\\\\\\/PttsBjuqpQDBALQK3dH4QMsxCpd9trCgcruU\\\\\\\/tygiJ49wp7kN8ETYfyGusO8abjzoCk2t2aHIShkFQtDQPNYCl9VB12uyMc1cVOr0a8sPt3YgGdGP0tz4\\\\\\\/2xt5GLIj8sob7nJ2eZm3he8aGzZbvSdTJNcXyz9Z1SwrxX8jtr56V1Z5B60bVsDaVmmvIi4FPl\\\\\\\/PVNzo6SZdgM6\\\\\\\/MDqCAawEfWluUll1mn4Op5kW\\\\\\\/2sTCfzL4fbO\\\\\\\/sTvipJG7CUoVbNDrXwgFmeG25s9oqmA4gtWPWGKc2RjPESuOGkQSsWaLSbkjuYWU7USqSknWs\\\\\\\/YFN3DSLTgF\\\\\\\/OeMf08cDKS1bBMa0WYB1wp3FWc1E56NqnoMmqNFuzbG2zSJizRCqI2e37\\\\\\\/m\\\\\\\/8ikKlAub2hq9WWCGNJU48WmHJ65DhF91yf48EJ1g1W5O9al00n\\\\\\\/xhMR98OBm8D2b7yYQ\\\\\\\/CT\\\\\\\/zAFiWQaB88CGBSvabA2to9wZzRZjpsNB89Wxh8x\\\\\\\/YSuEXafvtUxTHCpNfk0296wWGwbFtiFEoq98fxeN5\\\\\\\/PrtiqEhjz2\\\\\\\/T\\\\\\\/DGVC16lDEyUTaHlXe4UiY0XxRKNEM6QMLNu9osLdAjy30Gy270nrWbXcotEXAxcS3+AH26zOm9aTBtU2tMYRgeMMMbfzeg26cvA03Ck4HirGtdwbIZgJ\\\\\\\/LWJWYmJYGoXEHU71I3\\\\\\\/+Ec+QUTOij1Jt+Gg+2RWsKAd7f0\\\\\\\/NJGb8vH4ee96AM7FigLf3pg7pUwzaAANAnbIUx4xTBgq3gobeBlh\\\\\\\/w4Cpbxgg0WF8TcK149ryk3IxTW7+pa+F1l2yIOwV5N8pU2LYZbKMoCldU0plSZMZUiZJSdSKoNPgJMDdSOxImRt19BC7Nk6k6swhK7aPSmfNaHubK3C1gN9mrt5C7rhqFhXQEmeLOJex4dbzSDD0ERD2WFrfWx15jMU26IGyzwy7ol6pPsxN35VxHq1Y79jQ7XOB9S0ox1fiGYvRQdAXfk3TlN\\\\\\\/x34+rYEgm7y7TDJC99I9SVvuYbAQ+kqjIZk6WKmb32ZFRWsmG533Q0jmNgcPUWnFYSKCqo0rfj2nmldIc9GzaEM+72Usz16YSXAer17SQIMQpiE4mdzV6oGfOGWBZs\\\\\\\/XPZkI6CcOrlQoxZmUGrnhPTjSJJ31TSWbJJ7bRHiegX4OZHUSq\\\\\\\/XO2aXP47UjCQEGGTppf+npMOxP\\\\\\\/rYqr+daKNrTMVd1agMWq7Epsy2kR0LcTZ3vkM3zeC0k5UG4w9wt7x\\\\\\\/FBVCLPoVC3qvwBSBx1gG\\\\\\\/JcyeOkSUTQu803vRUNifW\\\\\\\/TOJLgshJ0v+DHSCgfpqVVP8efxEqQCmcsz\\\\\\\/P8ceo7JqaFBt74RZdYaWHIOV3LdD8fHfu4OxHNFbvipueVfUpcahVbY4VSfoAU+LL3bR4\\\\\\\/4DmB+CcHc8DwlOc\\\\\\\/40aVQShmkS8\\\\\\\/Jh+jnGQPnYqXq8GWCTmOChPzl0AAb5B67F4mpU1yenUyP1pIrp5hKxKiuck2wbOtvUSpSzxiszdsQh7kY9KTbWfkEpH9g2bXfcpE2TfQjRniXu4PCVknr2UEPZ0cL9FHWb8VCR6IJsUVNr6NswzeYWp+4gzpc9fiVNksFD55XQP94T4KszLsSGB7WDhbt6hvqAf1csdFBu76rVZNaVAD5whxkiI+KP9SzUni2zv7hgy0UVyMmSH1r4DKbRW3QeKHWhIj4QZ0tk1h3TLTMzhEc22W9qZfhTvEkYScpEYvjCe20EL+hCPhI8MqGy8y+xU77R7qW+gTIfy4GJ2pJHMXWymf631pdmm0y0M5uPfaMg8M1wiIEyP54kaDXncVoX+B1SsAb2Z9p0S3A5\\\\\\\/+k\\\\\\\/N5YGMti5YkXDlfIRj6PJwU2VjDHaWGgDH8vEtFW8gPWoKm1c+eal5hxI3w9UIWg62zDpZuEm\\\\\\\/H3O2NL+LshP7En41hf9Uobehr+MVpGY1wGzvnX2I0H+oN9acAMI4mUMKVmEDssujx7WLgbg94vPUtCYMjSWjjy0mTkmn6nRTnVqkeW+beVdSjFF6SpnZd4Dnkr7lMnZkzeC8S0hryBj7z+6M\\\\\\\/OiQyMrjSMj6nXCTBL+KOmslqe9KfV7vxLBudvVTUNo8RrqaC+gUceKDXngkwGzkwzVfXAEsXTUHsl0bvVaurt6\\\\\\\/0qoC\\\\\\\/VLBXo+oPqW6Vgu4CmYhGuXdZku28IlCUtDFI\\\\\\\/ArR6RubFnH9I35WNs\\\\\\\/jwxKbC50A9++gqFdFis5DzTaDPw+YM\\\\\\\/0QRbMxeCzX5vu021ufES4G2bl\\\\\\\/L\\\\\\\/rke9YAPKEFzkFE3s2u7pAqdL1TFtRz16tNlZcJcui0R8SuuGdzIcCSeKpEWG75iDEXWlBaGvoN16yFEKpqJIAifXKpYxYNiacjSuC79ZRVsYIeGYmqSrqd\\\\\\\/oDu+RvfLOG6ajYFT2uP\\\\\\\/MA5Mqw0OHPuooO12eenzdw9MO+\\\\\\\/NMCImb2KGOSJNVq1bI2qFdsANLRHpRfA57kXqkT6fh4u2J8v0xzlC\\\\\\\/A0qMBZYODi8T60eNa0FdWEA10STY8i9EArXQo76gEbFGPE2jyJy6elWakSaksYYRrtZjrVGai2qHWuMwAI49yA9+MEIsdBtCcop09dM6x8RRSTnEQ5OGNCRlnzR+yW3JtXUD9sBK5HPaRbC61g2eaYPqz4RFoc4sX83AtaPrsnAegLO1CyW8K6OHs9VBXHR0JFEj2Tl3STKBSIMWlKaim1dXgEa565Cb6gmQREZ6Z81tbIYQN2OQqDhYK++ip5xGhfzj+NwGekhdacvaOJ3dpCsB3Lxfh2Li1TTdgKxA3kqNHV8RH3jnzlbXTmSE0\\\\\\\/ExBIYz5DtG9njXEkKmB5DYbYB8DroHJ\\\\\\\/J4bKLPv7VFG1UbFUpW7sfsMUVQ5Zz3Qf6aGBCz3ADDxjicXehjcPLFXnjS7YxlMSKLvXTCYzje8PasRHjfl1RAmBt\\\\\\\/BpQ79IBBO6CPWWoKFVboK0Pq+scdwLX85ut5+Q7tyu+ZaftWCYfNG1rUSuNcNTMAa0UmGiUNc+oh40KcWRGjWiwX4uhnA6EkMAeuO7Kd98KkvwG5zYsgEtDCBJIvKMO3CMq1bJLH3qimOZRKgSiHYH0wMIlTY8yYACiVpWM99eqQsQHAffHoL5VG8gf9KrUCSjaic\\\\\\\/adXDD0hwHRqZNXQ3QDy+7DooztddUujroDrdiaew9\\\\\\\/xOInnWg0XYYtL0+EOA1Spq2XLq+lQAOgce7YJ\\\\\\\/F+uodmG\\\\\\\/2\\\\\\\/DiuGBLM15n\\\\\\\/g+XlVTzV9m93SFwxZTr97FdihdJQo7eVCS4bwO43nfaxNoulRUsVIjWjb+HYaPfXtjz0Hn34zPUdo736dGBjpkP2znOE8ZRV3SdNS1l3Z59swo53r3toyVC3OG22hSDZIWFWtkmbEZCTIBqdEW++UZOtJk4DbPPGVsBmgCdbhVsU99RAoO6BX06qPh\\\\\\\/sglv2U3OvBOjdDEBE9yYI8VFOMCj3lql0kL00FNXC5MTpDnPClERrVMDeRkljR5+Vt2vYPvz7YCwbQKz4+OmIT7AU\\\\\\\/ZCpVivAkAkf+X0USRvFCD9BbfgS5V\\\\\\\/t5g+gcZOMFpQ3S82wkkKBVk479l6G02vh0a68d3w+Xl2iF9uzj7bbZVp0+xCEJR3hhCAd31BUxWuU\\\\\\\/i8U3nvY2mbrOMHzOEI4Cjo2snSViERwnKuWh5u8EFXJd3wQewCk60S4I4Rl5+Jjk94+EelzLJA2Dqg0zVIgO6tR2IRz\\\\\\\/xTE8L\\\\\\\/aXZ3kvqk1cGkhGNmpmJnJNrY\\\\\\\/mitT2tYUiR4FmSC6N9q0l+iueHOaVdIQ73565hYwFttpKx\\\\\\\/BDjgNRHXKI8BZQvsJKPpJFulSJ1aG6pJmisN9sxGre7SybExfJkScpQSU4SAH4MpTq2Chm7ZDm3WrRFpe7s0CBJFPUxUrM1g23hUXamaQgwCKfvU090bIO7IZVQ7LqDxxkcyCn5SDZt5CU+FfF4DRcCOvukZdMdiA9leIiIUh\\\\\\\/dB+\\\\\\\/kivx684OfwUC\\\\\\\/8MefLrYheyVnCnZ7OLUW4XTmvRIZNqKL5OTKwDboLNK1oQRyOv7Vy04Hebcr+ZtX0sgoXIRHY86SHDDSlHmCjp10g1jOkNqYsqpFHT+rpf9IONM6gWadw1ODIDi8lVkSmH7RYflVxBjJ2Jg0l7\\\\\\\/evOOlFmhW\\\\\\\/JRwCbAzpiMryzBnvK1z\\\\\\\/vwyHsdn3fblZitWtz4IUIRdlVs1E6rMx\\\\\\\/viLkpF1x00odRL8YZK8fqjOBYVgG+pqm9uVU9BKB9EYGrfazRdYliLi++T5Al9v0sWnEToyGZJR1qLJF2VKT4NsxnAOsQDWyie7dzSLnfQ1JkWPDqyMGFUlMTfJ6tARr7RmIAnCKH9pKrq\\\\\\\/TPs3\\\\\\\/xBH+pHQZjruU5vQqMH2RngGvaIADcrqrQ7bik2QslXySgmegx6Q1EnBahzkIfloGlatNAgOVI5OE630BvuWgV2Clfbdux+QmpF9YK3OkR3ynID7TQlJtAgIMvJoBX5v3ibrEIitxLQudIJJioEdLvBk\\\\\\\/F1Xqi9p5ev6VbEMfu4NHeSc9wQEtPgaDX5R8WPil2eA2beeV305oAiN1ChEjZAm0S7z1jtv+DYRtKdpEAVVtx4bpwZA\\\\\\\/b8mMBTQ5umvr5LFHhTrYj5Ce8dpbWKGKRAtrIxxiwU5UbwJMJ1kyLYnHD0rxmcEQChWx3wLM9qCreZyc3T+bOJpwMWztkchgL8NaeT\\\\\\\/MNpr3w0IXoCJTLECoLAI+lA4+jngJzQuDcC1M2Avt1OHDTNy7YLvVO755+c4go6k5mbYawFj8iHmypg\\\\\\\/y4Wb+yyAOVXS7DmC6vkYFBU1mcu2roeFTLzbX5Uj069q2wDtzZpqJwtyYK5r+m2zNgPf\\\\\\\/SYM7z532IAJ0Tw718YiQ45yTidCgPC8bi0bGtyOWMjLGLzeZ\\\\\\\/8F\\\\\\\/ZzTsflR6IZC+G3p2sVqwv9m48tpJezKIEjxbir74n4GEtDR65DKGAYI+jUWZcCdho9dEdZ9X60x+SET7bfc8On64Z3xfSPmZ2Tf5LdMxQqVUULclSR+++JN231In9AMirGJR+yZpHo3+GR0pdei3mnOgWA8gEGHNe4\\\\\\\/Cj006L+C7J\\\\\\\/kYx7ns76mjeh8P10mB3LqLDpigdK8FmbaadMoRjpG\\\\\\\/gcVPN3IDFiVcf3lkPeny5i\\\\\\\/QxGCPqfFe1XAPyYcw+8YqFf1EbMLHbIWZHoQvaUVd7qgl9egK0qVfRio1mdF10c2Qzu+PxG5fI23CPJdmuhtuTnYcQYXzKfMDoDZa2UiJWSYysr6Hax2hydXWHC9BBY2x4GF4zUJqeZVfLMGjdOVbIvUFNzekdm1x28Y9vKQthSiAfKEC5w18JUzWqyPuGhd1s2RoO1iXi58TUd7yw+IsFs5Hfg70tfuR98tNZoorRsnSgVhlujB5MHh6TETQo5YjnnVaD3jHDgpXngLZavzq8YbEJ0BqXPZkI49PwsL72DLsKz3pvyzMUSSi+xRG8Amf8ObwJq+7n3YbcxcLZqTjhzf04xF2dVwwcu5QNNfC3uPOewvJuhzMk+Z+2g5AAqEZYzkdeAC0PzVxN36NuZIR0BXRMATsDUCsz5oNC395MaoKzWvicEscUnsk6iN9QGAwDLkMnIPsohzkkPB5oRxj4CfI01Kt3Jda5swxvCrOXQIeSO\\\\\\\/i2ZDeDcEDm2fwS0cuLYM+OU5dpyP\\\\\\\/7UVPT5Q9x6qPjNZmDLFT\\\\\\\/WZKNMH3kA0oL0fNAaBLWRwN4a2ehmuNQoUWqaizz0TkQ3RxPWh6AQq5+Soxcu+L2rAlKIm8T6MuwpTMV9texSW9VF5lpHWEoHIEdqZ5LYtF5F7wmtwlXl\\\\\\\/dTbOQlpkUVU1uYcE\\\\\\\/tPRwpmAKELTqznYdHvZBix0dQMymzMCBn3XZdmxysk0ogbipabc1krTeLftHJFn+x\\\\\\\/0Q4KqROerDgTIYSPeHZy3l5icJyeRDvFDTt3VKL3Q+QnR\\\\\\\/yC8TpOzfSJ57TVMqrtpsKdpg46RvzjX5EWAYeF+dEtcR9EYs5cUiVreoCFmXmwcgJQw3gmM2B0EZlXKrezrK8CqT2kJeAjgNMsbCiiMJstpKer+yfaV\\\\\\\/knJQq9ht6Eir2dHRPgoMqu\\\\\\\/8YgHKEPq5NhYjPeALxh8vGbtBiDFUcoIf0GkEge3t3H6a4zuZoTnMeLgHaKPSAvGt3xnSbHHjnAr43xim86Sm7lePezVd8+48JSA3Y8BBxd9aJdU9ErnckyYTLkGbiMEvb8lnb9T5Yf9rHz9wW0BcQs5hO4jb8d63HXIAYEno4dpk0fU9mD7xrXN09jfqSlO+Wq0\\\\\\\/L4eHqSG6e2Phn7Em1sO98PEipmd6ZWNzCQAt4tq3FSqFgoohzUfj9qjy2V0xORTwTllMREqJJvvwTV8mtLFPYp3n1R6g0IYZ5wGL0UcdUujxmvsy5xs84bNxpQEgndVGV0+pZSobRMR975w2tLIeuRBZtnuzY3QBZJj16NCG3fi+lNyi2NoMWiyprQFNV4eBTSmZWLhcQaY50u0w7A4yYpDxlIlnTzuYtE31n4ZjCDL3VaHXBvdHZELTYaxfoHr4k2+h1c6wiYV8G+dca5X4z4KyxcYMQnepnNY8qjVmwa0IRVKASqzfBJqhVrKkzHp5PtJqmySavazgoaiURyVuIcWe\\\\\\\/RkE0l+W6Xqqx8JiScOIOi1qTVgWDwZnoiTR6+vmKRj0DFHl3gbJ2BbrKpzfAwWKA50mOJEX3Fbsm4y4wKatBsGEnxxAwSAE3N5vAnM1OFMSY1oaPNZ24\\\\\\\/Z7uiKTw+CCV+szw6Vhj\\\\\\\/4ngZT7pwDBf9W1hPJwuQaCqQp52y4XDwmbinabJal7oQ1AXBaI9baB1bE\\\\\\\/jnR7DecOr\\\\\\\/OAYkdCGjT13Di8ogJYiDgnfEXeDYa6kbg\\\\\\\/UN7su2fKwZa1jBGk3LiEZQYIwd5BYmiltJ6knpuy6J6lz6aNe0VpiuAGLVteeKRh4Xva9P+Euy7dnqZSu5um4qXdjL44MzABxEfLp6eHq4K17rFQsxK3Oo6VxV4QfX1r1I3aY5nkB5qZgld06BpuptjOqr9\\\\\\\/F31gDOQ4oaDjKqBRHF9D9Oczct89hCa8+1UzKoCFTwB2pDZ1cv8Q4A74o9J6xH9Awo+D7Cut6cxpt3PsKk82mqR3htfwauUIktYcgDrqlWO8uCiU8X23LZ5NK6nTBhmOUsasi8dnbpxWAUld5pGmJpantiTrE7KO18erFf45DIQp43KLVj2CMfIG7UYfPVLsCsMWRET9y8N6srWTCRhlbK4EIZBtHSCOU7spGT+Z8FPIya\\\\\\\/4vC9nac55BbbWId+9e+\\\\\\\/Qk5kDgGOi\\\\\\\/x6umsQaCMBUz2xtYBJru5ef1KcCZ5ze65L93MOIbBsS3FlIz1q51T\\\\\\\/rqK5PQMvyFw8Hl+Erxk1+g6bh9th5vs5saK9bOANybuPNda2kSonroKiNnhgmhqrlsMoj43bnfU+Zw+aYHLvd\\\\\\\/YNajN\\\\\\\/LXbe1zzT\\\\\\\/moNEn7S+k3cbA7u7Xq5n+Vg2yYZcTMXPwBJeIYc1XiV3I5tL5Pf0F3kxMjszOox3VymGYK1\\\\\\\/V1iUmGobR\\\\\\\/KvFkk7XcXiEPdA4vVU7t+0BelTmc4A3ZKv0WPu\\\\\\\/le\\\\\\\/rM7kSoKPtsUf1CpR5UTZB3rT0pgYqCM9DSsy0f2lggV7FzMvi8q2NJgzCiNC18YpS5zW8M1KkzQZixAdkBrP+ATS0LVJu9uMoE99rHhFYGkq72Swk+6mHHgGQe1mOvFSDdikvLls\\\\\\\/dTqej7UPkMFbs+RhxHfKS1sH0Yzcx0cwuJ01E4bSPyuc8MSh8sDsWXw2dU0JOq38YsAlVG08+N3\\\\\\\/ufbO\\\\\\\/IiLQx\\\\\\\/jFpwlTkNiNuTzAZ6\\\\\\\/1acqjH\\\\\\\/EM96KR06S9qlOLfvc29KlnWfZroNQMdnbxs\\\\\\\/oo2OmzDMAsQpsvZP94AXYytbGuIAGwS+AUnzUQeFQyC7riTceFF2PXj9\\\\\\\/oO6deUK98YQ3PFoOMSNuR8N\\\\\\\/pIrcGi9OUHb50JwCFd6oFiUZ50NgZ2DJAqYqLKJ1ioZm83O5F7dYMsyu1K6HNHBm4FpU2OFKySu8dNFWBXojTIVbQBDE1cSiLoW5RSUkZpiAZTNPkPTrQW5eVH\\\\\\\/WDbkJkBQBJDuL95MzeEnxTPXjA3bclDklhbgxCYyv7UzdZ9eXLSoXltS4t31Zhb7zyClCR27bo9EVzOvolvCNM1aUVuYyTtnPaZdf5t126TFi8hv3dkyif7tmlH6\\\\\\\/kIy5CvETmaUoVW7lQdP3K7CaWaX+iN1TAWnYuRA29SeVmLKAchiobCrDIujVsZh9mXfp0MMpwVVjJdGAkDK9eG57aiF41c++LDiI34lips1lKybaUWVQspKo+Ltg+HCflKmvFytwHx\\\\\\\/gifQiXcm4vCYSGaDs39NF6jP7VDsXz3cA4a+PnO+Q60EZT8ot436ww5pDzbJ\\\\\\\/twvIAd9BEoD1Ume5ZF05f9+Tz7ds2OFCNihV4Ip+6xajfdcfoi8W0Gy2gUH5YAt0VVOuTHjVJhmhJTcNvORlOV+jvkwKSudjTuEJmLSHTlO1KMrSQ8pnNaLHscS8I4WLmaaVvaKY70GshOeNbALPF70JpKopZY\\\\\\\/UZ\\\\\\\/6Soh2IRdtjcE66VYiVXITQharbXOo69fvjgGoaWPJ\\\\\\\/zwsY2itdVCTbs\\\\\\\/+UOMNxOunM\\\\\\\/plG6B8RIdDzO9XxNUCR7tt7Me+w7hvLdhydSbqlqw0YxHAtJOn7OP5bBzR5AJwWIIdgmMBs2Sea1EnwtFNUtZtXZStnELroc+uV4pNsHtTtS6VFn9nYIl2PlV3Dy+ufGE7dl6smea1D1gtNgeWsZCiagREW4jQ2T2gHV2y6d7ku3UVVUyqILKi8fTAhW6cGstQo6\\\\\\\/5QUPgaEdhKS4E1u1OjPI9X78iFx\\\\\\\/n5c910b9wbEWXmjh71f6Er\\\\\\\/w2ssfseB1lDhPTh2wzOakvZap1RubnKJykfL8oc96oaRvG6\\\\\\\/S9Nsoba3RjnfbhQsh1RN\\\\\\\/1dXkTpn\\\\\\\/sOCmlQfBaw6Jot4vXhLpP+c\\\\\\\/GuzS3YBrpdM5EN2UrURI\\\\\\\/tajHOmDNp2\\\\\\\/i4wNTfaMCAeEM4aUeIopQF0UAiKcfGk7zEVd54BzxYqUhfBHw220zT4YF4KQH\\\\\\\/ZTcJtkKNZ356dgntqDpr50sGc0BtjsgsJWOfAylRQhoa8wDDyb0aQsO2KNnwLuH1SoPHPBhGKUfekL2RZs26ypztIyXjP9ZH2S9qYnCmh9DDHU9BKiRQsdZpkMl6j0mqgZuxYy3JXo4Qy3fhqaMXfrRED+dID1lr7OLaw1qgMYjO0SzrKaF7YBEkE1lcbLpFhDnlw4SyVCIn3dBJm2AVfk8dITRiYVHaQPLNPHSqVwR11pPULxTnOoF7WT24YhK4115nPJGVVRd0L6ivzBBZ\\\\\\\/0ia8+oEldtCrdGQDl+z9Z3O72sM0iRmIGyYaDqo9uwsafcyiidi2REoIaPPTjOVvbZoBPOoRAPLg\\\\\\\/1NOe62SjJ5aYxVKk4+n6aR+L3ow01uD3MKdbahQmvX0gsQVpgKfIsY8o0EK3eejdRfNMH5uuodnrRytypzrNSrceKeewuxVQoSrl4VHPUQdoBzVrfFjidPJMQXcsWeyPVkjSi9U7zE2O5eIPgneLLC6DUlDh6EI1zeNhWf\\\\\\\/gFPMAOq0nOtBOXgm9an\\\\\\\/tSwDcOwHVl7BrroMq4QMjPDJE3Q0c4fKDLLirTukK44T2hsZYFxOvRkk4LLWWJkrY+tX4Cod35kZP\\\\\\\/oSR3MZ2LeAZHUCo5HATPGY3GHup\\\\\\\/IwM27vEvbbZ6NpMyijqaLvWoMBNq7L+McUmubzc3y+QwRp+I\\\\\\\/s1eE9LMGeVTobDPHvnR9FXloEZuD8m7yo4JMPHD5vv5awSDupEcqImITCCU8rKqIMDlp6J34v9HGfZBONVyD7KWgq2jnWFTTQzFEGMpJP\\\\\\\/dcofJ4VrRFAdBQsark0yZuRvcbwGAVSJ0+dvlQqDz7Z1BS0\\\\\\\/fYZp3TQXGw9BBcZ7ZGzVRS2Q94STNlm31DXFJHKJHTxLX7Qp2vbFgHZbyq\\\\\\\/yzA4FPUFlTDs8qgfnBGAYkyxlL6GHWbrW44acO31o15I\\\\\\\/Hh9ukypJ\\\\\\\/J4bnwsxSmF2W5iQcW8EdD1mSt9YvCbJZ83p5vHAYrVpFPCOI8ufqXYO8ckLsMRDal02W4FMBpBc\\\\\\\/RxiocEOu5kVxmx9CTLN3rZkF8lAbytd2JxeO4+o9FSmi4fEfzFMNr7ddN5IF7BLQEodMm+WP2uaiih+ZgbqpskMzJb0UPv1QBZyZd7zChHv0vrTCc88ay6UU63lSwm1InfYXXrqyWp0GnaKuVoQmKtG4e3ETOiBljyf43+tIkhEx3pfuuuyrQoSWqHvJkxzIueT8YyFefvyS2VFyrzGaXYbLb9IgIV\\\\\\\/nWXFDtwE+pVpsHbUOxw5LtOQDpP8B0Q\\\\\\\/wSOAq\\\\\\\/857XJQYuyl9mhlAUYpeteFheBjwUDWB4GGKw526HSfhatTxUNlxSstQ1cJNxJQ9AKGhqp\\\\\\\/olOq\\\\\\\/Bw\\\\\\\/U7ZU0t+DaCCFbTj7x96CzxCPG\\\\\\\/uJBR6I9hYtcEhrL2ym8058BUoeohSFkNCpgrc12CcIMmpK5mxIKeJ+ihaaajg6qKx+Mp85KhRDRQG86RIdPQUon8ZrCV7bBhzlz0rB+Wp9dVm9UhnWaR9VX\\\\\\\/0uNXtzfw4R4no7RnWnyqSN5WIJVvSdVrK9L3eeDOXzvdWOY1I\\\\\\\/dqwWYh0TmFRQSfGNCZLGb1zaBE0KugHuPBpyxiSECO91SF53Sls4O3Jk1DsB88LK4GQTjDNp7KDDIkk9udSseh+JPdKjaOLVnUFdyY9Rj4wecAsPyt2d+KGWdzi7oByZQQNTSmrFmf2Pak8ZQk9gD5\\\\\\\/Y1kUpzXCoZoIB1SK+HK62YH6BggqQVYePK9hs8w4lC1aRfjN5BBUOf1Mm\\\\\\\/8Lqls5+vgVTc4l92VQRyKYQCqvNwWQp1e5KP6aZ0\\\\\\\/gbRbuZqzQbaiT3yZGka+RtzGvsYoWJOi+byKzwyVI6ltPP4HtEurBPtLW3TdDpzLZZK7ej8XDxm6OOkZhn2VLLB07iYBVCwtVc5\\\\\\\/QNpI\\\\\\\/6w1uFEq289UBQnudT0G0NEC7fDTVa\\\\\\\/VN5SGU+SgqUsYl\\\\\\\/ur\\\\\\\/EE91bai03GSNeMlwc0D1lIDOTFWbhI9uQYDAOJsTHfk0h1dV1\\\\\\\/nEfEyXgeiKx3WwDTn3YYGid8RddijjPLpw1XxtbI\\\\\\\/V4bwkpK+bXhv1lyyvOtixX1xe5CpJ\\\\\\\/BBj6imR1nRoFSSPmE0XY9x+ON93Ozam88kBdrZmdSM6x9i\\\\\\\/MJm7u7aRULEvadZj+8aX9\\\\\\\/wfBmoO3CKJqmrNm223kizYM\\\\\\\/1wpW9jsPg5JNS7o\\\\\\\/sTvhIVUizC9ldK+BfZbX4M49cnAItOeOquN4tZFF20rDT6uzoPJNdOYIbIFSlV0TQqD\\\\\\\/K8fewwnmp8Usw8\\\\\\\/w7H1GESsfYzuk5N7QKVJBSnJaO8HIqSFBkF2ehrgV2vQkJYmJNQBKquqWJiIaCGuXlw9oZSDpdRlBK3MdHWBohJ16VAFRiKayhNXBbxGI+0XFF2hFskb++HBNQfo5NWjKvehmW2A6WcTvtglp+3T5rWglAGGYgoH9luWMlJStd9\\\\\\\/VmWfGAg7ieU0V6pZcq1phArwvP9QWpfo3jcujeKjJdtynOprS6r5O4qzSNfSTEDL\\\\\\\/zMX\\\\\\\/qgLj+5ht4y1vNZgmaJCczhJqXovGIo+57kQabYK9aY164mtgn2Lrq2HfmS0EJvVq1ORUfTXig85r+Lk96OzmMPc7f5d1czzmbsoOuamqhWw205FED3i3eUy2mQhI9j3xbbFohzeTA09wSfkeAOe83VfLnRfME8U9CkHHNbGqnbBYWWVJ8lxnjFyV5Mdwy9THSeMQ3f32iw66idkrtQBmMgmmih+WhcPpwBAfXQwECsUwkJwIVRQnh8NryXz9bzVAVS1fZAESJY10Q4v5yWVQmpVtC4bWUXd9LcVLIlzCWEN6Ytu8JOsODUNHev1JdOv8d7JiTgUvZGh5DNfQFqxchoclOwCW1gAL4hvsM3hahxLEMEsaGIip7ZbAGk07jQBCjQsidSaD0kbRcaXoE7spy3jFQhp44bKxopqa+4GcKUUuxLVxo7VpY1ilQyDbKZjEv1PAmD8Sdzc0YGCKLrTUdgEcHkQMf3Otez\\\\\\\/1+WJMZjyD9wZn6y5Q9U5gPo6+2GHoQ6FG1LUYh\\\\\\\/cqv6mN3JNYTVc3bHfnsUAy\\\\\\\/9jtvpgVnfKaQYLUYrl1+Mv1DcgEDvr+IWC0Hca\\\\\\\/flZjdMlTiJsfrcEG3Jg0539gAKIwf7tai9iK2POeVt68ZFzTTohQr90z9UJlon9R19jTY0\\\\\\\/xNdjdfQfpcY6r0U37BMmqycXBNRaeIn9D5NwSXIKEXOxwYHTzcbsNz9fRXbubP3KUyaUeFeTk4j4lfV2pEssyySUwvn\\\\\\\/6Xxp7N7uoxB4czAes9Nu5BRpA9JVwe7ZlcjFWYxFUJE\\\\\\\/fCyiVmI73NEbGEFZGoX\\\\\\\/dGg\\\\\\\/l6JkpPdBpmeQFGen+JMah48it7FrxdmeT9scKu554GDtK5u\\\\\\\/FgtZqycaTcyc3bSwAUudmpQsYetLqcNpzkab+jpSaI1575ZvKVEBq5NjDiKr7NGM5vPNk\\\\\\\/SDXqaLxMTRl80OmO9mAS0JztqHX1VUNkCTq3F+KD47Uy8xN7HYgadRqc5F+X3B619z45jOs9VHOqbugMLQliRB7mK26UXuDlzTtS9XMI7kBb4vT4Uj5wUL+A5OrRy1Q\\\\\\\/Txn9egdOhNRV7HhGXD0LZ5dUH7vF2nqezCY9lYkuTOBWEuA4an2IZkPpkxdO9fAnVipb2P7XGG4kqZT1p3na92HvLficKR+V\\\\\\\/nzIuQH7Ehz12UXbsLBBFMtSkJSB2aAHhaAQmzek9LelDOzaOATjxvaxQhXWhJUf5E7MiLV3Qy0Bd73K3NbZ6r02E4VK41RjKzgtCPxV7sy\\\\\\\/1jNQb0TTetDMQfF6pab3qHg\\\\\\\/bxGD848vb\\\\\\\/NJmnAlnUDWdxnrpQXnoNUTvAh7\\\\\\\/9L2797Bvl+SS2b1jPxBPoA2n4nHbUdatDvSVJrWQOgOL9N6GLaRHnDpdSVOMZlxYVSETMHbRFILzWhruEDIsdncAxXa4p0G1BN7nVYZUeMu8hyrF+l0oMvm42S0N7acwLMeqjyVZWC0ltwIR6fK5JWWYi8Uikdz5CXfQq9x4SNuT9vivbCfr14Nk7vJEJaCHjzSUcSx6lD64FNtdBJPUborEBxulD5S7nrV3qIO4O7hTPqer\\\\\\\/0FI1u1CoNvu2ZYtIs6K\\\\\\\/B0kDWJ309fAMdpwlNYJrKfcnSdsgWRugEqiUIM4LzeRoAJhQCkJ\\\\\\\/ngReJb3ohc0DQwIUoyrHtWEB8LJ4CW242NQXF4RQhCWCxDeDUrm8POiYWxOFvowPtuyQlXNM0OORPHzNhS\\\\\\\/slqcnRq0euLa+NN3yNT8IEFLAB6tBUS4zImIoVXTbpKiHP+vvxjmnJPFe9em3KECczyJrbJmFNyIaUaVJ+8HA+2vB2rgQRo8dQLHZ3VDmeYhSmzPQManV2l3QqflxrxPmZGXMzdQKnmN4lgKkQngVQlG46UYmF9w+2AGAsE03fAxUV3p1SZ88y+fdjw2VDKQrZH4N6xxaSOrDWgkn7peNKTbvDTrh\\\\\\\/L8RWpifrP2bsk4gsQ+PGneEJAZIgPZUnl+g+cNWprsV6SzOP1axMvFPj\\\\\\\/jMHxhd1IWFK6DIu7Xj0OfexAjdJFcOHbw7GUKCMYcFzP0BiX2LVoWC3+8WnO1GmkSY05eZAqPdB\\\\\\\/gbTjMvJrxT8Yb6vmyGLMtHMMupWzo4bXY7CsSxZVS8MVDt+ZMlBoZNlH4kVoOiZ2A0nWynogBxs0FwzLjxv241T2k2MqONwCetEDw8Bc7Enj6H7YAiJ7yqPRm4N7n6QOC8FzTQAFe9Hdb9FSRxjOUdRXYXQNGpvqng6ihF+rCa439Nzwl9Qs4kQjp0fe0oTH+Yy\\\\\\\/PktLS+kjVYfBkHaStc+Q0YUGf5z3w1Vmgp8SeytgJt1lUUS+jvUDACgCaA1Yz7m4ArCruy0cLmU+px0LIDs\\\\\\\/RqPCfpqIxnYxMpttkt1OP+gJrAgstKCb2adM0n2+UjSct\\\\\\\/hixWr\\\\\\\/hJ\\\\\\\/n8\\\\\\\/ezqqgx59rHtpGfs396NwnGRrYMH6g+sv4Bma5QpP0kv1Lu46pO5yGSfFl9Qxoxv\\\\\\\/kJWyqtieDYApZNYd26AgHWHmLJYTLWIhzr6r9X8CIFdmdzn88uA7lDXRt6YuF4NI8OzHfILn4KlEfSU6X98oI3nW60MvUXNx8Bj6ymmmEspQNFy3qASzXkbDiq4NnJPbdR+5umN8ZtYIdEo6M5I2nPrJbY98P4K7m7r6XmNHEK8ZSaOrF8l+XSDthfTLvWXYZ9u6EFpy8d1M9WTwbBelCNaJ4NVoIDyTVk1bceC4RyRS3TMCbbo0RtOYWl3NpBro5sJiZYnAXELt67eYqhbh3kFChwChzWmkrIgDCNeKrd7nFpSfayUR9gTHJzlwnD00M8E9JmAIIFAixDY9ovPU3nGYilBDIN2tg0LC3Ob+ggKCVaHnc6hARtHixAaGyK0aWy98f2QgWvofaElD3QQaGHW7kT22\\\\\\\/cz3u7mrxi2yusRmHdNcCYY2z2t4WTWdGp4Td24K3zZwv9PcJlzQgg+kbuf5uMnQVBuRAu6n3jz9TEoa9KusezblZczQWNpOtir6DxeZHa6RTgiWMpZtjoGd+Kd18JWzbVD\\\\\\\/taKkxOkJDXnX6XeMaH+4aGmGInRBBnjfvhDAdpx6W\\\\\\\/4N8KxiqKqJs3QNAtTxOmfrtG83T7Qrky4lqvKxn\\\\\\\/QV0ksV0aqovqrQ4Ms9WGgkVftTTbxlA3Pes4xD+EiGH9hiyZeqbV8RZRGT5asJcPGLwceEwExrk9hc5c0BvfineV9LD63Zn4G+3q70nZp7iHtoSsNaLlRDzdIBmlzwVFSCLGTGzqBuMHVxF8IStm8\\\\\\\/zy6TAsV8JTc4fobeKn2QJQ4N3OHYDnFc4fNOuQbf3ltVHhFjC2HnL6rjJce\\\\\\\/LOmDVELlcEh\\\\\\\/eNX7GsZcl0ElZwJczyQcrkdFfy9DNBiwp0XAU\\\\\\\/4a0a4WsrVSV4JiC7VSWuetHSqH6rHle4MM3u+B9Sm+CEhqy1bQwK3d\\\\\\\/X32cOVG8QaugRR2RWlZSVLOp3OZelzsb11IyapdCGdu+jbig9WbRxPq266hZIGN6nHkHOy24Gb\\\\\\\/dnJa\\\\\\\/4Pqnp+T16nVEpkiD1zbbsDIW\\\\\\\/u\\\\\\\/lSVNDVPw3j9cxjkYcNzMOJtnn3vtEthSY46VESg1iYGxTO\\\\\\\/8o2yJY3Dui5gOMVd+S7Tdfhqsn2LeS4QpFDDo7pjIgcXoDC7PA7BioldV0cfCHmzKXyy+pwVNE7hA2Fd3VNkc6shknSsKZyv9pSN3yXJREBrpsq1kX5bP63qdB1vOQe0\\\\\\\/GJ\\\\\\\/+8ygozutObXZW3K9aIgnT5SuGRGGKDcq\\\\\\\/0jIm0SzMFmR204\\\\\\\/hkKUfRnDWKh5pDbxztWCOFZAAo269jDFkgbZwrkfpI\\\\\\\/g2PQzCWg9WnSmy60BnCUhwZTq1aRyhoaKkgawyvm89wvHQb\\\\\\\/apq02PsvqAXRCS6hG4\\\\\\\/4sByKji4RpLTIubNlhNWjZ4i7rJalfT9FDLduajnl3h+PFA\\\\\\\/Y6tjfdcqhObWv9ADVyry4HRgn4UdZhzNWa7Nj4wGr4bZKMrCy7LO4\\\\\\\/hN6WiRsUK07iCW2ikYhTqFQZ4jYVQrV+rvcZ09aOljykIo\\\\\\\/FBurZ8nwMfz9OGnxV7ziX+ZOkOHItgtQtP5LLswge2hbFl+7Nj+PnWzSSh56jl0MkFUfWfQmf+mhVvtIqkyDu9hwiwW24VnelwkAEFUKz2AN7H1s0gHO6INtUzRyNIsgQONp2WK4ubE\\\\\\\/D3gO3F5+TKyEe09yf3KBOXroLFMl8nQKRuUQnNM4Zn8g5tF31XiGKRbr4QN9z+rmpeJiny++WNHXuHPc2hXB3fgMjhsPYrONKn7CyVNLcjnHiQES3he+HnSvcu3XJmoohNGXOlWJaJqLYuNx4gqyecNeg1JvUvT5bUFg8hrQda97hgn97Xnw4oJeK4Q2xvXR4wX88qelGcH1JZOELCLYY5S8eBVzzXUk1xdynSHwSuSsoKbE9Jke9DW0Gu49dDsBJGVihD27D9VNiByh+p7dQA7udHrPRA4Lhznadwin8DTkYstV6cyxeqVwSacYpKWlABmtrtljTwBja9mNXgBoEKavXYRSiYGpDsTZXllkcT\\\\\\\/QhSr+N6\\\\\\\/5Rw0wTvUUjMjK2XIYdw1IefAxn+sz2fuBeQxBrzSEPX67WNHLiffKQ0VK1HpDDhK04Xudy71HctDiERkGp1TfzFgLssyGDgWozwkuGVs+I581cRvGU+gOk9ddU1aVgWr9WU1N8fRWOvQIf+d4JsbEYsp6wKqqfRQ2v\\\\\\\/CEsyBXmjlso6Kw6r9fMuJZfj\\\\\\\/Ls42S9b1chzDpHkm0d2FIgVp\\\\\\\/qTMRXacZGGxXJ+86dmsxQ7+6eakCYHkLAmzruCnZqISHu3YJ+Vs2nl0koQSwQA3BwdNelbmrWnk13N0YWZ7CtRxtb0XRvLN00CP97TE5gCga\\\\\\\/6q1c4XxDemBrEI3dzf5Jkq6VerMrg1SMKs46pfJBik8f\\\\\\\/J7pDBlob6gRw4I0lKSNiE7ZyYaYFos\\\\\\\/rYgoG2XRxuYUgV65+6w29MlHc0KXUadqxUI\\\\\\\/vAfI2HDsKrmzw6kkezsPe+crQYsmt\\\\\\\/cepURX3Xlnidre9h\\\\\\\/noKkEPkqxvtl\\\\\\\/ko\\\\\\\/xk4NJRpv+7juJav9mkKnpKC8pLenEwEHFM8e7FOp9bwzV89khO9EMwYkE9n7DCXacplGhuOpWKvYOTvxk7o\\\\\\\/NlfjWCOA\\\\\\\/vTcVeIHiuipqaT2IVvUYHZKxljR4\\\\\\\/ywuEMO2QGm3jUPJR6iyQzeOc4p3hk1h+uTwpTS6vS2SWzOAPnjkPxp9uzbiflekg7i07DEFHlILjB6HbwYBdHE79OEf55iDQjUNgq8gFyxPUMpFJueUgIf1Nw4XXtIEQDJEasnSwXhSjCs0kUAHZDI\\\\\\\/nkAse+pRz74GXYciFQbF5+Az9FschLJtSsqQJQo6cyL5wd\\\\\\\/adE8C0lYx39jiwugJLi09cHwnF3cjS+W6yeEVsWldI0ijHcN92MtGrlRJOAQ6z9L4eDPgTVZkNuqsS4PVgyJvwcJibXNO0+Gt1ZlCr2SlhsjWPTJ+mzzc7IUu0JdGCQ6FE2mCE9i+yUk3zZvRyHxFnLH6sYiQs9S80dFHjyCa5Xp\\\\\\\/BJ5wWeC7gd0fgOsVJrTwVlCenQfkLgoWRqhuLwAhtqGEUJhYf+v2cOHhnhzlS2bRhah0y1wpxtcGa\\\\\\\/aMtuLIpeQ\\\\\\\/aaYotPv5l2+mmysHM\\\\\\\/dkXbO7qkkK9c\\\\\\\/n2hxh9U0G7sm5shjYIVgkXw1afGcP3atODPkkPkaualLDNjG8Yz+AHJBEVzpOpLFKZvDWGAZe0oJJXY3XFnABeRubO19NLttLBTkmPkQmPTw3Vb1eBcyrZNkcGancuagyyiKcSQCJcCsWBVEQaQtUXWnKRFPdzoRyGwVNcoGQOu\\\\\\\/Vk0fR9K+lmaUaBJ1mgA\\\\\\\/iUSPFljbMrQjJJx\\\\\\\/jnWZxbOUP+LGYTNLeOMl9KMzgKFx8Kg6P+i6GuF0taEku2OPuWdrOADmR0VZ6httMw\\\\\\\/qYU9xsEwnE2NDtOOHR\\\\\\\/uHqLacRRrq\\\\\\\/iqGAxi8EPJMab+Hse5EocvutlWzj+mXDHCRfLuuTlv0a7n9BMzEJT51Q80vy8yXytmMLoNamlhc4WoW8cNLu\\\\\\\/EAdJxWgWNFh84xxiZQOh7CXLHjjbJ0dOarcI3gM0BRL6V86U\\\\\\\/5v0a6pD+6z9ZocWi\\\\\\\/1Bkk17SN1zyjW7rizk1RcNjEKmVtWhWsc1E3A+CXE7VozcaOea19cHFtmkHqm\\\\\\\/fdWVVNwqKYYZgUnNT8HmUh\\\\\\\/orxFFVD2S2SQ2ES07EtdvagW6VgfMyMoNCMM3KMcUL73QV6K3FbSvRbugxQaV7B\\\\\\\/Sbal3PMVUQPDUT+jzv+r4vZNvb2Cx+306A8GTkqOT\\\\\\\/mQQoUWEtbiaRP3aYsy7GwaH4sYRvfElQYql+BLfcU3EX+Vd75nQhL1UUGYBVyH6RYFjTxGfRpj0ptpDtPLS+VLd\\\\\\\/IkC5Db4v9ra6t4c7olba+SckMSEcW0rdejT9XcOSRxfA6UhN8LdiA45otQSlISeSY2DQ8DYGPpE00k7YBA2OuZW\\\\\\\/rQOlFoJeGFLuf\\\\\\\/No4pNUg0PSrY22laVBbBvPBg1bImPAO6HNzLihyueo0iGzhNJ6jXZhG+hHSOf++iUjhKwwjDsPfzZYimtEKr7jF2mp8ofqESyvG4Y0K8bqgo3or5zenZouGqbUYVd0pts+Ls09PQCNM0ZlvbuEBH3Bisx3Z4QvKrvN1CzV7F94cW7hg2JPI1qFWwDhToKF1noQFm6Gt\\\\\\\/2mt9nIoA65IkNO\\\\\\\/leT0Lpwgm5NqNqGCb1VdRItUXz0TJHSgCCZH8OMczL5q1h8VAAedUhlknhVcI0hbLZVlx+txvX3OlIcOe+JLo8BbV1fuF2T9ZaanWZMMZ9kibNJXeBYtnrlIYsJjjxmrkH\\\\\\\/aNnbHY\\\\\\\/uKclg9eNgP+pFal3lCpqc1vZhQVHN42DbHXjG8Fisd7UZw6XEWAAPNID4AHQhPLfbmRkzlsO70Cw8kz0Y387L3ezSmmfl5Ev1ar83lo1ole+ISYbdoZ+cadygbtHV7mQF5NVmxXKScNiqzaIh46QblcmkJQ5U6sOfidwBspkvW84iZ\\\\\\\/LSNvZykwa1JIpUN5pOGolJNgVudFuY2F4R64cEYk+kKlejRDm0eEBXu4+qUDlhxi02oIlbLgufaJ7mEq3J8px7AzrpfgrqQUbp3swiA\\\\\\\/xrsBlKu\\\\\\\/wItFhh\\\\\\\/A63Eu4sMbTwht+Wb4XrBgfouNjG5apKiqbk+skiVv1ZcYbFzbbcMHi6WnsKKuyCfgakZ\\\\\\\/On3pAd0ZilK0j7oVg5460+gws+nZuc9BolpPsHc1OfVwJB\\\\\\\/oJdNgPK0KXx\\\\\\\/haIO0iKQ9moawqfvaFc5erZ972qlQB+6qTOFRL5Nv200zfISpim29CvXDA+8tjDnmYzyPLPoZ4YQepxCkKldaphsSE9vm1oHxQS3hTKPSctE2lSUhRZk40Ugy8h7KjGMVO7aiJneSiXxgEtpBviohX5WirE\\\\\\\/GOri4CGGL7T9t4yLfFrCcU8dQ87uF95+mghkUbSmz5fmopQGa8u1rPnwapR7p9YuHSpVtfw59UxO3bwNMAgL1knxwWjmS2vnPWlECTdMhO4Xms+\\\\\\\/o8LkjB9sQbev5wQcHDym54dNqxqF7w2Wv9lbw7ylkbqJ3iHvtBGjDX8GbsSHqmPJzmRW9gA2geO28B+h7cz4KwhNsepYoEAocD7ydufVhD3NjOQFMhx2HRargBktjezbn3o8DJySACM8QGsenZGjpUtGyO4kNNtutQwl2pIjYuAH74c3dYTGKT7wC7i0Y\\\\\\\/DKrcPFNn19HDS0+3C0DgcUYKZ+h+MTHdQc\\\\\\\/rxg1CJxsykuqifgkbJQU4UmuRlUxF+aviVudNGVAjdIE9hIl7cSnHhuhmn12fkbBRVdcQ2IBnUaJyCzDkNjxXJUYfNsPpw+wrxpWWuYEHZENCAUTh99UjnUfzvVqEpmoH8evLZLc6B0rKnt4MIB2+37Ub8vajcI\\\\\\\/kfR9iBoR7\\\\\\\/ecqINeab4kMQa+Vm5YkS6\\\\\\\/iF0ff\\\\\\\/zKfI7QgpBDlAp0zTan3PUXAfIZC7eTpDa1+xdnluAoNeNDAB0Ycn3IJo4XC6qI3lhMPzmPWyaGrrBt7p9tuiWnq8p7lIM5bfkTfZvM7gVR6ZTC2jDP9i5oCMfBEVGPySJLj2kMDM03rjJgwPvXhvUJPJr6hQ\\\\\\\/wqSZAD3MdVwbTL2lQE3qAVVZQBgdl5vdqvDdXL5mFUKwlBwWMtFomH\\\\\\\/78lFMCPQLoOJHxpa23PRQSW3gdyEiVMtb4Zaot1cPmsbxezxlOax3LC5DDWeETVuUThVSNGj+mrSx3QbGeQ3pT3f1E1USl9\\\\\\\/wVQuMiRpwXvt9AawMccNU2GsW\\\\\\\/dAH6i0T0GbK6TG5RZX+xLCN9lvi4DbhcgVj4xxrgbPVl4Dwf7XXqqqPqNRx2L6+5ucozztIml+zklZKElYz7PwKK10lKK3w5auXribVGOxR3P2jWu\\\\\\\/u7iKjsyuY4FrexETrc023T3SdentZTrcrNrCbqtSWqf2Q6clwpSmDE2v7FeV7E8+1HZ3kUkXvVBdBdORETdaQUaW2y3z+V+4xEeiuwg\\\\\\\/UGlq35laLrDTL9KP2vlFi9qKLIjXun0LFj4vE+SR1Y67Nf03bnC2GLbg6PoDZEok2RJPFUMkVFkN0lilLwj1pz0NA2Fc01psXyGSNtUrf8X32WFOvebPMvqHrDbbZoVBcAbSkQnc3I3i2rSxqyqpXaL\\\\\\\/Yw8ENlf9CAvFXIWH9Ut9DFapLxy8bBGixeJUUy\\\\\\\/S26jr0X7o2VnM9pSdZ+xE\\\\\\\/PUavPayIU5c6qIu1xI4X9k3iXZDhc4TeJow0cf1q+IsqZhxPSEA2fwyhFwR++8\\\\\\\/QmEzSI83wU7k6PA==\\\u0022,\\\u0022iv\\\u0022:\\\u00220O6\\\\\\\/Hha1z7Fo76p2+IcUQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225aebc299e55fb43da45f1cc5f2ad564e5f1aaa57f075e7ef3e9f1d5849554664\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022a7CeP40Ts3D+pcgS01LCp3J0d7N4iVWwgPss3Y9UYxMKHz44H3TlnhCKBZPeZo6\\\\\\\/1T++sXfntdSHappD+UTd2CrGGkjhAli9j9dow4HxSvv4qbUSmL1CwTLpP3hlxYHR7NtrmSR\\\\\\\/k8qILdscUciEblLz3iNgp9zGRkHIeC1cpDhJgzrWAjIBSwA7zZPZb6zDvCuLneeX1Y88LzpFVyoRSnOtoMND1xG\\\\\\\/GJwYUQXMjVPiXFeQQPIifZFbxrsUzWvfM60OGC8NN\\\\\\\/oXChl7QGI\\\\\\\/JqQPH\\\\\\\/DzPJYdD1XWmlG0gRCXGe+eYtzUNd3S4MCowT876nw3OXAmJwlJZ7I3kDv4bhaZmnAIHPlG0Xm6+LkKs\\\\\\\/1Ijp1okTfWtGYO2NT6LexZe\\\\\\\/HL692UPmDnvsQ+7Cvk4uiA\\\\\\\/jZqJ2xnuwPcw56feab+py88G7WM6yVbmQl4LwI69zIUl0A38mp3tumak2320HvmH4ajNc8l\\\\\\\/ZLRbJR3zQYgrrlLO\\\\\\\/vKBkN0dl0UVrFOOkOWanJEpVwRIyEmEanTtzFEmIHTr+NKfbcSzabPqZQEAdg6NGmcwWkjwgdwidi8ovzW8VZpP1iS9YFScTIF4DTxki4A32J2NS+XqFjED0ZlKgYL0pMEBL\\\\\\\/cJiYjcVFl3E4VxhMLINIo1LoSIIRI3w9zVoH7JPfoZhBu3KlMk\\\\\\\/VrjjIXaEzjhrLBOoi4QMEypR2fWsb0mxBfxzsLgGK+mlf31wSYyisVEA1IVYmkLlh4xnAzSJG+mJe7EPykBXcR4SuvM71o8L9rErpzGezjv37VMg03fNh94FmVMWUP\\\\\\\/V7NTkKt40W5c5yZcgxLjEXks82+f+ldlhCSEhuehCyd+F9TsyxlgvPy94Fk0wJdVHkNK394phkl8mb+jZQJNxp1egBADdlnPGhOz05nuQlrSFT+hvhYlM6V35syL\\\\\\\/nxECgZP9p4dpjI2JCTTErNnP4OgZYe5C14Z846BodrlnGAZJKwmHFSOKEjlKzcp5izbOqKhn42SVo0RZREyuo0CQrF21kHHz8ETp43Dq3yo27zCoxryyTNXXMjF+0zgLTAX6aiMnehwQeKyeOyvIMzuKtjqlqsrXSu8PsDzQIQetG7scTHpIifKhGk5E83UEnbvSfAWeghRQMUyxVInujmMK7LuGxls\\\\\\\/a\\\\\\\/\\\\\\\/UwU13dPjwEPO6IR5ogFVS+l58tvSncLXzhv4szHjY1FXEIBA\\\\\\\/jb8OUOCY3WjAagHNCXRv89jKT+NyNhBlpMkBPJ6ekvPSLpX7QrMn9fOiHykTRmUsz8SRI4DxObcpzyssRUuBFnonhKWooW8wd8udNerlP4Il3zbqEBKJI68G4ptftrfd780gto2TPPOQ6pi\\\\\\\/62l0BXnY4OAuT0Z7U+vtVozDq1iVUuD3DtmWncxvccs8xmwCMp3dJt3Rm2h3M9y\\\\\\\/D\\\\\\\/lCrCn9Q4\\\\\\\/0CxiJZt926PCGCkRUFIbHDF5xAjf27F+uNdlQnws6rmVuC38iM7Nb60Jy9be3WyAF2UHhf59fZdLPTxVkq59NT3tCiwMds0dbLWLD0SiokHs49TwxWodkIK7vCXNkVlSFg9RFLDJGxMN9ng5jYOs8LLfa+yaPntnfVXQyF4Y3u0Jpts0JF5ppsC7313HKITdwcKmQtXG60FaBacTT+rMyE3gcDkiDQVD4cEQ7MKWrHeoqTJugY9dXid0FHpMb1nziJ4SxA16lJTRmyNrlItmM5hL3rmxupaIeJVui73ypWqcBi07BcL8WstuJHUDX5JGO9cavDQn+xQuJ+qHyXcVBn\\\\\\\/cRLg5s1K0FiFLgIfMWZkCFEZaR17Q0cxiganLvwWdQaX1pkG0bWy26wwsCN5LAC3LYxR4QQUqwFG2SBHH6Zj+quYRT0QYsP3hvUeEJujSfcBGUg5DeS34gBWXnEa07Qm6\\\\\\\/E+yHqwCyWeUHY5GIGOg8738bnq0l5pOtIlzFpa85vlVUaENG\\\\\\\/3qfM779SIoM5KsSMrFQ0YpEc3gALP28z+Z\\\\\\\/11nE5gaZicO2Yv0Er44g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022m8VHXlLAUcSezcQz+o7hrw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220314cf0df45d34c091fa6ba8599273c5f3e3f04f8502b5f12764c352988755b9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459712 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2231" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-I-QgHJR8Hi2orOKgk_YCWK7aRjafLzH_6FwnOSC6WIU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2231" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-h2GiJpdv4yHbR0UIsnHGaNtLZ45cEk1phnPcpMWZdbk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2219" title="High-Resolution Genetic Map for Understanding the Effect of Genome-Wide Recombination Rate on Nucleotide Diversity in Watermelon" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2241" title="Drosophila Protamine-Like Mst35Ba and Mst35Bb Are Required for Proper Sperm Nuclear Morphology but Are Dispensable for Male Fertility" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459712" id="node459712" data-pisa="ggg;4/11/2231" data-pisa-master="ggg;g3.114.013482" data-apath="/ggg/4/11/2231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genome of the Chicken DT40 Bursal Lymphoma Cell Line</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">János Molnár</span>, <span class="highwire-citation-author" data-delta="1">Ádám Póti</span>, <span class="highwire-citation-author" data-delta="2">Orsolya Pipek</span>, <span class="highwire-citation-author" data-delta="3">Marcin Krzystanek</span>, <span class="highwire-citation-author" data-delta="4">Nnennaya Kanu</span>, <span class="highwire-citation-author" data-delta="5">Charles Swanton</span>, <span class="highwire-citation-author" data-delta="6">Gábor E. Tusnády</span>, <span class="highwire-citation-author" data-delta="7">Zoltan Szallasi</span>, <span class="highwire-citation-author" data-delta="8">István Csabai</span> and <span class="highwire-citation-author" data-delta="9">Dávid Szüts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2231-2240;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013482 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459712" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">János Molnár </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A1nos%2BMoln%C3%A1r%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moln%C3%A1r%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A1nos%2BMoln%25C3%25A1r%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ádám Póti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%81d%C3%A1m%2BP%C3%B3ti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%B3ti%20%C3%81&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2581d%25C3%25A1m%2BP%25C3%25B3ti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Orsolya Pipek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Orsolya%2BPipek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pipek%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOrsolya%2BPipek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcin Krzystanek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcin%2BKrzystanek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krzystanek%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcin%2BKrzystanek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nnennaya Kanu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nnennaya%2BKanu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kanu%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANnennaya%2BKanu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Swanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BSwanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swanton%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BSwanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gábor E. Tusnády </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G%C3%A1bor%2BE.%2BTusn%C3%A1dy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tusn%C3%A1dy%20GE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AG%25C3%25A1bor%2BE.%2BTusn%25C3%25A1dy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Zoltan Szallasi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Children's Hospital Informatics Program at the Harvard-Massachusetts Institutes of Technology Division of Health Sciences and Technology (CHIP@HST), Harvard Medical School, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zoltan%2BSzallasi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szallasi%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZoltan%2BSzallasi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">István Csabai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Istv%C3%A1n%2BCsabai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csabai%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIstv%25C3%25A1n%2BCsabai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Dávid Szüts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A1vid%2BSz%C3%BCts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sz%C3%BCts%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AD%25C3%25A1vid%2BSz%25C3%25BCts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:szuts.david@ttk.mta.hu" class="" data-icon-position="" data-hide-link-title="0">szuts.david@ttk.mta.hu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/11/2231" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459712" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459712/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2231.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459712" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459712/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2231.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459712" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459712/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2231.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459712" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459712/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/11/2231.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459712" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459712/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The chicken DT40 cell line is a widely used model system in the study of multiple cellular processes due to the efficiency of homologous gene targeting. The cell line was derived from a bursal lymphoma induced by avian leukosis virus infection. In this study we characterized the genome of the cell line using whole genome shotgun sequencing and single nucleotide polymorphism array hybridization. The results indicate that wild-type DT40 has a relatively normal karyotype, except for whole chromosome copy number gains, and no karyotype variability within stocks. In a comparison to two domestic chicken genomes and the <em>Gallus gallus</em> reference genome, we found no unique mutational processes shaping the DT40 genome except for a mild increase in insertion and deletion events, particularly deletions at tandem repeats. We mapped coding sequence mutations that are unique to the DT40 genome; mutations inactivating the <em>PIK3R1</em> and <em>ATRX</em> genes likely contributed to the oncogenic transformation. In addition to a known avian leukosis virus integration in the <em>MYC</em> gene, we detected further integration sites that are likely to de-regulate gene expression. The new findings support the hypothesis that DT40 is a typical transformed cell line with a relatively intact genome; therefore, it is well-suited to the role of a model system for DNA repair and related processes. The sequence data generated by this study, including a searchable <em>de novo</em> genome assembly and annotated lists of mutated genes, will support future research using this cell line.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dt40" class="hw-term hw-article-keyword hw-article-keyword-dt40" rel="nofollow">DT40</a></li><li class="kwd"><a href="/keyword/gene-targeting" class="hw-term hw-article-keyword hw-article-keyword-gene-targeting" rel="nofollow">gene targeting</a></li><li class="kwd"><a href="/keyword/tumor-suppressor-gene" class="hw-term hw-article-keyword hw-article-keyword-tumor-suppressor-gene" rel="nofollow">tumor suppressor gene</a></li><li class="kwd"><a href="/keyword/chicken-genome" class="hw-term hw-article-keyword hw-article-keyword-chicken-genome" rel="nofollow">chicken genome</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism" rel="nofollow">single nucleotide polymorphism</a></li></ul><p id="p-3">The DT40 cell line has been used as an important model system in the study of DNA repair and immunoglobulin diversification for more than two decades. The cell line is derived from a bursal lymphoma of a female domestic layer chicken infected with avian leukosis virus (ALV) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baba and Humphries 1984</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baba <em>et al.</em> 1985</a>). Efficient homologous gene targeting in DT40 makes gene disruptions and sequence manipulations feasible; therefore, the cell line is uniquely suited to genetic studies (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Buerstedde and Takeda 1991</a>). Gene targeting is preferable to gene silencing in many experimental applications due to the completeness of its effect on protein expression, which is especially important when the genetic interactions of several factors are investigated. The use of sequence-specific nucleases such as TALE nucleases and the CRISPR system has recently presented an alternative method of gene manipulation in cell lines (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cermak <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ran <em>et al.</em> 2013</a>); however, the simplicity of the homology-based targeting coupled to the predictability of the outcome and the traceability of off-target integration will support the future use of DT40 genetics.</p><p id="p-4">The design and interpretation of gene targeting experiments are facilitated by the detailed characterization of the genome of the host cell line. In general, DT40 cells have been reported to display a stable karyotype (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sonoda <em>et al.</em> 1998</a>). Although further karyotype analysis detected variation in the number of macrochromosomes between and within DT40 cultures (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chang and Delany 2004</a>), the relative normality and stability of the karyotype of this cancer cell line are significant advantages for its use in genetic experiments.</p><p id="p-5">The design of gene targeting constructs has been much more rapid since the publication of the draft chicken genome in 2004 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">International Chicken Genome Sequencing Consortium 2004</a>). The latest genome build (Gallus_gallus-4.0) was released in 2011, and it covers 1.072 × 10<sup>9</sup> base pairs on 29 of the 37 autosomes, the sex chromosomes W and Z, and further unplaced contigs. A limitation of using this reference genome for DT40-based research is the poor assembly of the microchromosomes. Several of these are entirely absent from the reference genome, and the higher microchromosomal gene densities (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Smith <em>et al.</em> 2000</a>) increase the probability that genes of interest are on unplaced contigs. A further limitation is the sequence divergence between the reference red jungle fowl and domestic breeds from which DT40 is derived. Indeed, 2.9 million to 4.6 million single nucleotide variations (SNVs) were identified in each of 8 domestic chicken breeds using ABI SOLiD sequencing with 3.32–5.53× coverage, and 6.0 and 5.8 million SNVs were found by Illumina sequencing with 23–25× coverage in two further domestic breeds, the Taiwanese L2 and the Silkie breeds (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rubin <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2013</a>), in comparison with the Gallus gallus reference genome. Isogenic targeting DNA is generally used for homologous targeting as nonisogenic DNA greatly reduces targeting efficiency (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">te Riele <em>et al.</em> 1992</a>). Prior knowledge of the cell line–specific genomic DNA sequence therefore helps the planning and assembly of the gene targeting constructs, especially if homology arms are produced by gene synthesis.</p><p id="p-6">Experiments on cell lines that serve as biological model systems should be interpreted in light of specific properties of the cell line. Exome sequencing of a large number of human cancer cell lines has identified mutations in known cancer genes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barretina <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Garnett <em>et al.</em> 2012</a>). However, these datasets do not contain the sequence of all genes or that of intergenic regions. Full genomic sequence data have been obtained for some cancer cell lines, for example, HeLa, in which coordinated genome sequencing and expression analyses were used to characterize the cell line (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Landry <em>et al.</em> 2013</a>). Similarly, annotated whole genome sequence data will help the future use of DT40 as one of the best-established genetic model systems.</p><p id="p-7">In this study we set out to analyze the genomic sequence of the DT40 cell line at high coverage and identify SNVs and insertions/deletions (indels) in comparison with the <em>Gallus gallus</em> reference genome. We compared the SNV and indel dataset with those of published domestic chicken breeds to determine whether cell line–specific mutagenic processes operate in DT40. The analysis of copy number changes in the sequenced clone and in further wild-type DT40 clones suggests a relative stability of the DT40 karyotype. The raw sequence, a <em>de novo</em> genome assembly, and the results of the sequence analysis are made available as a resource to the DT40 research community.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Cell culture and sample preparation</h3><p id="p-8">DT40 wild-type cell line stocks originally sourced from the Institute of Animal Health (now Pirbright Institute, UK) were obtained from the laboratory of Dr Julian E. Sale, MRC Laboratory of Molecular Biology, Cambridge, UK. Cells were grown at 37° under 5% CO<sub>2</sub> in RPMI-1640 medium supplemented with 7% fetal bovine serum, 3% chicken serum, 50 μM 2-mercaptoethanol, and penicillin/streptomycin. Single cell clones were isolated and grown prior to sample preparation. Genomic DNA for both SNP array analysis and DNA sequencing was prepared using the Gentra Puregene Cell Kit (Qiagen). The sample preparation for the SNP array analysis took place previously, and the cell clones were frozen in 90% fetal bovine serum plus 10% DMSO, stored in liquid nitrogen, and re-thawed prior to the preparation of the DNA sequencing sample.</p></div><div id="sec-3" class="subsection"><h3>DNA sequencing and SNP array analysis</h3><p id="p-9">Libraries for next-generation sequencing were prepared using the NEBNext DNA Library Prep kit (New England Biolabs) with the omission of the PCR amplification step; 100 nt paired-end sequencing was performed on an Illumina HiSeq2000 instrument (Genomic Sequencing and Analysis Facility, The University of Texas at Austin). SNP array hybridization was performed at DNA Landmarks (St-Jean-sur-Richelieu, Quebec, Canada) using a 60,000-sample chicken SNP chip developed by Illumina Inc. for the GWMAS Consortium (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Groenen <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Data analysis</h3><p id="p-10">The reads were aligned to the chicken (<em>Gallus gallus</em>) reference sequence Galgal4.73, which was downloaded from Ensembl (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Flicek <em>et al.</em> 2014</a>). The alignment was made using the Burrows-Wheeler Alignment Tool (BWA, version 0.7.5a-r405) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li and Durbin 2009</a>). The reference sequence was indexed with the BWT-SW algorithm, which is recommended in the case of large genomes. The alignments of paired-end reads were generated with the <em>aln</em> and <em>sampe</em> algorithms. The generated alignment files were examined for general statistics, <em>e.g.</em>, number of mapped reads, coverage, insert-size distribution using the <em>sam-stats</em> program from the <em>ea-utils</em> package, and custom scripts (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aronesty 2011</a>). Short genetic variants were identified with SAMtools (version 0.1.18 r982:295) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2009</a>). In the pipeline, we used the options <em>mpileup-E-D-S-u</em> and <em>bcftools view-bvcg</em>. The variant calling format files were compressed with <em>bgzip</em> and indexed with <em>tabix</em> tools. The detected SNVs and indels were divided into distinct files and analyzed separately. SNVs were filtered with VCFtools applying default filters, except that coverage must be at least 3 and the value of the quality field must be at least 30 (<em>vcf-annotate -f +/Q = 30/d = 3 -H</em>) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Danecek <em>et al.</em> 2011</a>). Short genetic variants (SNVs and indels) were annotated with ANNOVAR and CooVAR annotation tools using the Ensembl73 gene annotation file (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Vergara <em>et al.</em> 2012</a>). In the validation process, our SNV dataset was compared with the known chicken SNPs downloaded from Ensembl using the BEDtools <em>intersectBed</em> program (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>). We identified shared and unique SNP variants between DT40 and the L2 and Silkie breeds using BEDtools <em>multiintersectBed</em> program. Short indels were also identified with VarScan 2 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Koboldt <em>et al.</em> 2012</a>) to confirm the data obtained with SAMtools. To detect LOH regions, we first counted the number of SNVs in 100-kb sequence blocks by VCFtools SNPdensity module, and then we selected those blocks where the number of homozygous SNVs was at least 10-times greater than the number of heterozygous SNVs and the number of homozygous SNVs was more than 50. CNV analysis was performed using R version 3.1.0 with Sequenza package version 2.0.1, available from CRAN, and using a protocol for missing normal sample (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Favero <em>et al.</em> 2014</a>). <em>De novo</em> genome assembly was performed by the Ray genome assembler, with k-mer size 31 and with the recommended options (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boisvert <em>et al.</em> 2010</a>). All calculations and software tools were run on a Supermicro HPC cluster, with 256 CPU cores and 1TB RAM. The data analysis scripts are included as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/013482SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/FileS1.pdf">File S1</a>.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Whole genome sequencing</h3><p id="p-11">A wild-type DT40 stock was chosen for analysis that was originally sourced from the Institute of Animal Health (WT-IAH) and has been extensively used in the Sale laboratory for DNA repair studies (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Simpson and Sale 2003</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Szuts <em>et al.</em> 2006</a>). Genomic DNA was prepared from a single cell clone of this stock and sequenced on an Illumina HiSeq2000 instrument; 6.35×10<sup>9</sup> 100 base pair reads were obtained with paired-end sequencing, of which 92.8% could be aligned to the Galgal4.73 reference genome, and 5.950×10<sup>10</sup> base pairs were mapped over the 1.072×10<sup>9</sup> base pair reference genome at 55× overall mean coverage, with a peak at 52× (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The mean insert size of the pairs was 353 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). A <em>de novo</em> assembly was also performed to aid the search for sequences not present in the reference genome. The assembly covered 9.802×10<sup>8</sup> bp with contigs over 500 bp at an N50 value of 9801 bp, and 9.986×10<sup>8</sup> bp with scaffolds over 500 bp at an N50 value of 28,885 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS1.pdf">Table S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Whole genome sequencing of the DT40 cell line. (A) Histogram of the sequencing coverage across all positions of the Gallus gallus reference genome; 52&#xD7; coverage is the most common. (B) Size distribution of the insert sequences of 100 nucleotide paired-end reads. The value 0 refers to unpaired reads, and the value more than 1000 includes all read pairs separated by more than 1000 base pairs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1978245998" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Whole genome sequencing of the DT40 cell line. (A) Histogram of the sequencing coverage across all positions of the &lt;em&gt;Gallus gallus&lt;/em&gt; reference genome; 52&#xD7; coverage is the most common. (B) Size distribution of the insert sequences of 100 nucleotide paired-end reads. The value 0 refers to unpaired reads, and the value more than 1000 includes all read pairs separated by more than 1000 base pairs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2231/F1.medium.gif" width="308" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/11/2231/F1.medium.gif" width="308" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465269" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Whole genome sequencing of the DT40 cell line. (A) Histogram of the sequencing coverage across all positions of the <em>Gallus gallus</em> reference genome; 52× coverage is the most common. (B) Size distribution of the insert sequences of 100 nucleotide paired-end reads. The value 0 refers to unpaired reads, and the value more than 1000 includes all read pairs separated by more than 1000 base pairs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>SNVs and indels</h3><p id="p-13">We detected 6,251,553 SNVs in the DT40 genome, of which 3,320,786 are homozygous; 68% of these SNVs were already reported in dbSNP, which contains data from two sequencing efforts of domestic chicken breeds (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Rubin <em>et al.</em> 2010</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2013</a>). Because the genome of the original animal the DT40 cell line is derived from is not available, we cannot tell which of these mutations arose in the cell line after transformation. To explore this issue, we compared the DT40 genome with the genomes of two domestic breeds that were obtained with the same sequencing technology with high coverage (23–25×), the Taiwanese L2 and the Silkie breeds (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2013</a>). Subjecting the raw sequence data to the identical analysis, we found SNV numbers in these samples that were very similar to those in DT40. In each case this corresponds to 5.8 to 5.9 SNVs per kilobase, similar to values reported in a number of chicken breeds (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Rubin <em>et al.</em> 2010</a>). The spectrum of mutations is also identical in the three samples (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). CG &gt; TA and TA &gt; CG transitions are the most common, with nearly equal numbers, consistent with C &gt; T transitions arising in each sample and in the reference genome after evolutionary separation. The C &gt; T transitions are the most frequent base change in most organisms (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lynch 2010</a>) and are also the main component of the aging-dependent mutational signature in cancer samples (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexandrov <em>et al.</em> 2013</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465118/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465118/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">All SNVs in DT40 cells and in the L2 and Silkie breeds</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">We found 1,844,901 unique SNVs in the DT40 sample, which is similar to the number of unique SNVs in the two domestic breeds used for comparison (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The mutation spectra of these unique SNV sets are different from the entire set, with an expected asymmetry appearing between CG &gt; TA and TA &gt; CG transitions. The increased frequency of CG &gt; TA mutations compared with TA &gt; CG reflects the fact that these mutations are mainly driven by C &gt; T changes, and in the unique SNV sets we are more likely to detect changes that occurred in that particular sample, whereas in the common SNV set we are more likely to detect changes that occurred in the reference genome. The spectrum of unique SNVs also does not show any DT40-specific features. Taken together, the analysis of SNVs did not uncover any mutational processes specific to the DT40 genome.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465246/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465246/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Unique SNVs in DT40 cells and in the L2 and Silkie breeds</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">We identified 708,892 indels up to 50 bp in the DT40 genome in comparison with the reference genome, which is higher than the numbers found in the L2 and Silkie breeds (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The greater number of indels in DT40 was confirmed using a different analysis tool (see <em>Materials and Methods</em>, data not shown). 62.2% of the indels were homozygous in DT40, a proportion similar to that in the domestic breeds used for comparison (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The greater number of indels in DT40 is more apparent when examining indels found in one of the three samples only (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The ratio of these unique indels to unique SNVs is higher in DT40 (0.130) than in the L2 and Silkie genomes (0.0919 and 0.0864, respectively). Considering molecular clock theory (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zuckerkandl 1987</a>), this observation suggests that processes resulting in increased indel formation were operating in the DT40 genome. However, the proportion of heterozygous unique indels is not higher in DT40 (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). As mutations arising in the cell line would be predominantly heterozygous, this suggests that the extra numbers of indels were not generated after transformation or the isolation of the cell line.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465186/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465186/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">All indels in DT40 cells and in the L2 and Silkie breeds</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465137/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465137/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Unique indels in DT40 cells and in the L2 and Silkie breeds</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>CNV and LOH</h3><p id="p-16">Copy number variations (CNV) are apparent from the sequence coverage of individual chromosomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The sex chromosomes W and Z are present at 26× and 27× coverage, respectively, confirming their monosomic status and the female origin of the cell line. Chromosomes 2 and 24 show much higher coverage than the expected diploid level. A number of small chromosomes deviate from the mean coverage, indicating that overall sequence coverage is not a reliable measure of ploidy for chromosomes less than approximately 5 Mb in length. It is possible that there is bias against microchromosomes in the sample preparation procedure. We also analyzed CNV using the Sequenza package (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Favero <em>et al.</em> 2014</a>), which indicated, based on read depth, that chromosome 2 is trisomic, whereas chromosome 24 is tetrasomic (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Whole chromosome copy number variation. (A) Sequence coverage of chromosomes (Chr) 1&#x2013;28, 32, W, and Z. A line at 53&#xD7; coverage indicates the mean coverage of larger disomic chromosomes 1, 3&#x2013;10. (B) CNV and LOH analysis of the sequencing data using the Sequenza package. The plot shows the mean depth ratio and B allele frequency (BAF) of 1-Mb-sized bins overlapping every 0.5 Mb (black). The blue area represents the interquartile range of the binned data, whereas the red line indicates segments generated from 10 neighboring bins. Chromosomes 1&#x2013;28, 32, W, and Z are lined-up on the X axis. The copy number scale is set according to the best fit of Sequenza&#x2019;s probability model, in which the ploidy for DT40 sample was estimated to be 2.2 n. The LOH regions on chromosome 2 and 20 and whole copy number changes of chromosome 24 and 2 are visible. (C) SNP array hybridization analysis of the sequenced sample. The 60,000 SNPs are lined-up on the x axis in order of genomic occurrence. Chromosome boundaries are marked by dashed lines, and chromosome numbers are shown between the two panels. SNPs in genomic regions unassigned to chromosomes are omitted. Top panel shows signal intensity (LogR ratio, LogR); the increased copy number of chromosomes 2 and 24 is apparent. Bottom panel shows B allele frequency (BAF). Large regions of LOH are marked with arrows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1978245998" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Whole chromosome copy number variation. (A) Sequence coverage of chromosomes (Chr) 1&#x2013;28, 32, W, and Z. A line at 53&#xD7; coverage indicates the mean coverage of larger disomic chromosomes 1, 3&#x2013;10. (B) CNV and LOH analysis of the sequencing data using the Sequenza package. The plot shows the mean depth ratio and B allele frequency (BAF) of 1-Mb-sized bins overlapping every 0.5 Mb (black). The blue area represents the interquartile range of the binned data, whereas the red line indicates segments generated from 10 neighboring bins. Chromosomes 1&#x2013;28, 32, W, and Z are lined-up on the X axis. The copy number scale is set according to the best fit of Sequenza&#x2019;s probability model, in which the ploidy for DT40 sample was estimated to be 2.2 n. The LOH regions on chromosome 2 and 20 and whole copy number changes of chromosome 24 and 2 are visible. (C) SNP array hybridization analysis of the sequenced sample. The 60,000 SNPs are lined-up on the x axis in order of genomic occurrence. Chromosome boundaries are marked by dashed lines, and chromosome numbers are shown between the two panels. SNPs in genomic regions unassigned to chromosomes are omitted. Top panel shows signal intensity (LogR ratio, LogR); the increased copy number of chromosomes 2 and 24 is apparent. Bottom panel shows B allele frequency (BAF). Large regions of LOH are marked with arrows.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2231/F2.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/11/2231/F2.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465125" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Whole chromosome copy number variation. (A) Sequence coverage of chromosomes (Chr) 1–28, 32, W, and Z. A line at 53× coverage indicates the mean coverage of larger disomic chromosomes 1, 3–10. (B) CNV and LOH analysis of the sequencing data using the Sequenza package. The plot shows the mean depth ratio and B allele frequency (BAF) of 1-Mb-sized bins overlapping every 0.5 Mb (black). The blue area represents the interquartile range of the binned data, whereas the red line indicates segments generated from 10 neighboring bins. Chromosomes 1–28, 32, W, and Z are lined-up on the X axis. The copy number scale is set according to the best fit of Sequenza’s probability model, in which the ploidy for DT40 sample was estimated to be 2.2 n. The LOH regions on chromosome 2 and 20 and whole copy number changes of chromosome 24 and 2 are visible. (C) SNP array hybridization analysis of the sequenced sample. The 60,000 SNPs are lined-up on the x axis in order of genomic occurrence. Chromosome boundaries are marked by dashed lines, and chromosome numbers are shown between the two panels. SNPs in genomic regions unassigned to chromosomes are omitted. Top panel shows signal intensity (LogR ratio, LogR); the increased copy number of chromosomes 2 and 24 is apparent. Bottom panel shows B allele frequency (BAF). Large regions of LOH are marked with arrows.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">For an independent measure of CNV, we analyzed the sequenced sample on a 60,000-probe SNP hybridization array (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Groenen <em>et al.</em> 2011</a>). The total signal intensity (plotted as LogR ratio) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) confirms the increased number of chromosomes 2 and 24. A plot of the SNP allele frequencies (B allele frequency) shows two distinct nonhomozygous states at chromosome 2, confirming its trisomic status (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). At chromosome 24, the only intermediate allele frequency is approximately 0.5. Because the sequence coverage of this chromosome is approximately two-fold higher than that of similarly sized disomic chromosomes, this confirms that chromosome 24 is tetrasomic with two allelic variants, and the four copies do not segregate independently. Importantly, we did not see any evidence of large-scale CNV at the sub-chromosomal level.</p><p id="p-19">To assess CNV within and between laboratory strains, a wild-type stock that has been widely used for the analysis of immunoglobulin diversification, Clone 18 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Buerstedde <em>et al.</em> 1990</a>), was included in the analysis. A bulk population plus two single cell clones were analyzed from each stock; one of the WT-IAH single cell clones is the sequenced sample. In WT-CL18, there are two more trisomic chromosomes (chromosome 14 and 20); otherwise, it appears identical to WT-IAH (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/FigureS1.pdf">Figure S1</a>). Importantly, we did not find differences between the bulk sample and two single-cell clones in either stock apart from a partial loss of heterozygosity (LOH) in chromosome 21 of the bulk WT-IAH, which is complete in the two isolated clones (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-20">The SNP array shows multiple genomic regions that lack heterozygous SNPs, most notably a large part of chromosome 2 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). The genome sequence allows a more detailed view of regions of copy number neutral LOH (referred to simply as LOH). We calculated the ratio of heterozygous to homozygous SNVs in 100-kb sequence blocks along each chromosome and detected an average of 322 homozygous and 306 heterozygous SNVs per 100 kb; 26% of the sequence blocks have a heterozygous-to-homozygous (het/hom) ratio less than 0.1, which we classified as LOH. A further 8% contained fewer than 50 homozygous SNVs and were not used for LOH classification. In the L2 and Silkie breed samples, we classified 33% and 30% of the genome as LOH regions, respectively, indicating that there is no overall DT40-specific process resulting in large-scale LOH. The size distribution of the LOH regions is also similar between DT40 and the two domestic breeds (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), although there is a greater incidence of short LOH regions (100–200 kb), possibly indicating higher homologous recombination activity in DT40. The size distribution suggests the presence of a larger number of LOH regions below the 100 kb size, but the SNV density does not allow a reliable detection of smaller regions. The position of the LOH regions is mostly unique in the three investigated samples, as illustrated on a selected chromosome (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The two largest LOH regions in the DT40 genome on chromosomes 2 and 20 were confirmed by the Sequenza analysis (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>). A table of the LOH regions in the DT40 genome at 100-kb resolution is presented as supplementary information (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS2.xls">Table S2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regions of copy-neutral loss of heterozygosity. LOH regions were scored in 100-kb sequence blocks according to criteria detailed in the main text. (A) Size distribution of LOH regions in the DT40, L2, and Silkie genomes. (B) The distribution of LOH regions on chromosome (Chr) 5 of the DT40, L2, and Silkie genomes as illustrated by the ratio of heterozygous to homozygous SNVs in 100-kb sequence blocks. A ratio less than 0.1 was used to define LOH." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1978245998" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Regions of copy-neutral loss of heterozygosity. LOH regions were scored in 100-kb sequence blocks according to criteria detailed in the main text. (A) Size distribution of LOH regions in the DT40, L2, and Silkie genomes. (B) The distribution of LOH regions on chromosome (Chr) 5 of the DT40, L2, and Silkie genomes as illustrated by the ratio of heterozygous to homozygous SNVs in 100-kb sequence blocks. A ratio less than 0.1 was used to define LOH.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2231/F3.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/11/2231/F3.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465278" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Regions of copy-neutral loss of heterozygosity. LOH regions were scored in 100-kb sequence blocks according to criteria detailed in the main text. (A) Size distribution of LOH regions in the DT40, L2, and Silkie genomes. (B) The distribution of LOH regions on chromosome (Chr) 5 of the DT40, L2, and Silkie genomes as illustrated by the ratio of heterozygous to homozygous SNVs in 100-kb sequence blocks. A ratio less than 0.1 was used to define LOH.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Coding sequence changes</h3><p id="p-22">We investigated the mutations affecting expressed genes to gain insight into the molecular properties of DT40. The gene transcripts of the Ensembl <em>Gallus gallus</em> genebuild updated in December 2013 contained 78,329 point mutations in comparison with the reference genome. This corresponds to a divergence rate of 1.825/kb, lower than the genome-wide rate of 5.829/kb, indicative of selection against mutations.</p><p id="p-23">The spectrum of SNVs in coding sequences is very similar to that in the whole genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), suggesting that mutagenic processes affect different genomic regions indiscriminately. We further categorized these mutations using the CooVar program according to their likely disruptive effect on protein structure as based on the Grantham matrix (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grantham 1974</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Vergara <em>et al.</em> 2012</a>). The DT40 genome contains 1251 nonsynonymous mutations classified as radical, the chemically most dissimilar category of amino acid substitution. The number of radical nonsynonymous mutations is not significantly different from the 1285 and 1128 radical nonsynonymous mutations found in the L2 and Silkie breeds, respectively (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Again, DT40 does not contain more differences from the reference genome than the two domestic breeds, making it unlikely that many of these mutations arose in the cell line after its isolation.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coding region sequence alterations. Single nucleotide variations (SNVs) and insertion/deletion mutations (indels) were detected in the DT40 genome in comparison to the Gallus gallus reference genome. (A) Identical SNV spectrum in the coding regions of the DT40 genome and at the whole genome level. The percentage of SNVs in each of the six single nucleotide change categories is shown. (B) Length distribution of indels in the whole genome. The total number of insertions (positive) and deletions (negative) at each length is shown. (C) Length distribution of indels in coding regions, as shown in (B). The relative increase in the frequency of indels of lengths that are multiples of 3 is apparent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1978245998" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Coding region sequence alterations. Single nucleotide variations (SNVs) and insertion/deletion mutations (indels) were detected in the DT40 genome in comparison to the &lt;em&gt;Gallus gallus&lt;/em&gt; reference genome. (A) Identical SNV spectrum in the coding regions of the DT40 genome and at the whole genome level. The percentage of SNVs in each of the six single nucleotide change categories is shown. (B) Length distribution of indels in the whole genome. The total number of insertions (positive) and deletions (negative) at each length is shown. (C) Length distribution of indels in coding regions, as shown in (B). The relative increase in the frequency of indels of lengths that are multiples of 3 is apparent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2231/F4.medium.gif" width="329" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/11/2231/F4.medium.gif" width="329" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465182" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Coding region sequence alterations. Single nucleotide variations (SNVs) and insertion/deletion mutations (indels) were detected in the DT40 genome in comparison to the <em>Gallus gallus</em> reference genome. (A) Identical SNV spectrum in the coding regions of the DT40 genome and at the whole genome level. The percentage of SNVs in each of the six single nucleotide change categories is shown. (B) Length distribution of indels in the whole genome. The total number of insertions (positive) and deletions (negative) at each length is shown. (C) Length distribution of indels in coding regions, as shown in (B). The relative increase in the frequency of indels of lengths that are multiples of 3 is apparent.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465153/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465153/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Mutations affecting protein coding regions in DT40 cells and in the L2 and Silkie breeds</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">We also found 1505 short indels in coding exons in DT40. Most common are one-base insertions and one-base deletions. Their length distribution is similar to that of all genomic indels, but an overall pattern of selection against frameshift-causing indels is apparent (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). Following comparison with the L2 and Silkie genomes, we found 363 homozygous indels unique to DT40 (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). Finally, we detected 175 nonsense mutations in DT40 that give rise to a stop codon and are therefore likely to interfere with protein function (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Further types of mutations (<em>e.g.</em>, in splice sites, stop-loss mutations) may also affect gene function. These were not considered in detail, but their numbers are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS3.pdf">Table S3</a>.</p><p id="p-26">We concentrated on radical nonsynonymous mutations, indels, and stop-gain mutations in our search for genes with impaired function in DT40. We excluded those homozygous mutations that were not unique to DT40 and, taking into account that certain genes contain multiple mutations, we were left with homozygous mutations in 485 genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS4.xls">Table S4</a>). In addition to homozygous sequence changes and homozygous indels, genes may be inactivated by heteroallelic combinations of different mutations. We found 126 genes with a combination of at least two radical nonsynonymous mutations, indels, and/or stop-gain mutations in their coding exons. These are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS4.xls">Table S4</a>. We did not attempt to resolve the haplotype of these combinations, and several may not affect each of the two alleles. The list of mutated genes in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS4.xls">Table S4</a> is annotated with the human orthologs of each gene, plus gene ontology (GO) terms for the chicken and the human version.</p><p id="p-27">We searched the list of potentially defective genes for the presence of genes commonly mutated in major cancer types (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kandoth <em>et al.</em> 2013</a>) and for DNA repair genes in general. We reasoned that such genes may be inactivated specifically in DT40, and therefore may have played a role in the oncogenic process during the development of the cell line or may otherwise specifically contribute to its properties. Most notably, we found mutations in the <em>PIK3R1</em> and <em>ATRX</em> tumor suppressor genes, but not in well-known DNA repair genes.</p><p id="p-28"><em>PIK3R1</em> encodes differentially spliced regulatory subunits of class I phosphatidylinositol 3-kinases. The mutation in this gene is an in-frame deletion (Z:21,510,877–21,510,882), resulting in the deletion of E450 and Y451, which are conserved in all vertebrate homologs in the Uniprot database. The structure of the protein in complex with the PIK3CA catalytic subunit has been solved (RCSB ID: 4L1B). According to this structure, these residues are located in a long coiled-coil region that connects the two SH2 domains of PIK3R1 and is responsible for the connection between the two subunits (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2007</a>). The deletion found in DT40 and similar mutations affecting the coiled coil region can be found in the COSMIC cancer mutation database. Mutations of <em>PIK3R1</em> are present at a low prevalence in multiple types of cancer (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kandoth <em>et al.</em> 2013</a>).</p><p id="p-29"><em>ATRX</em> is a chromatin remodeling factor with various roles at genomic tandem repeat sequences (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Clynes <em>et al.</em> 2013</a>). The mutation at 4:12,803,872–12,803,873 is a two-base deletion from a long stretch of thymidines on one allele and an insertion of two thymidines on the other allele, both of which cause a frameshift that destroys the C-terminal half of the protein. We confirmed these mutations by genomic PCR and DNA sequencing (not shown). <em>ATRX</em> mutations have also been found in different types of cancer; the tumor suppressor gene is mutated in 90% of cancers that maintain their telomeres by the telomerase-independent “alternative lengthening of telomeres” (ALT) pathway (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lovejoy <em>et al.</em> 2012</a>).</p></div><div id="sec-10" class="subsection"><h3>Genome stability</h3><p id="p-30">Because DT40 is widely used to study DNA repair and damage tolerance, we asked if we see signs of genome instability that may be indicative of DNA repair defects. The only aspect by which the DT40 genome showed a noticeable difference from the L2 and Silkie chicken genomes was the number of indels (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Therefore, we asked if there is any evidence of repeat instability in the DT40 genome. We examined all indels that are unique to one of the three genomes (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>) and classified these according to whether they occurred at a repeat sequence. In case of one-nucleotide-long indels, we found no difference, with 79–82% of such deletions and 86–88% of insertions occurring at repeat sequences (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>) However, longer deletions were more often found at repeat sequences in the DT40 genome than in the genome of the two domestic chicken breeds. For example, 28% of 10-base deletions are at repeats in DT40, compared with 16% and 17% in the L2 and Silkie, respectively. The enrichment of repeat-derived deletions covers the approximately 6-bp to 15-bp size range (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). This could potentially result from a defect of mismatch repair, because MutSβ has been shown to repair insertion/deletion loops of this size range (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Genschel <em>et al.</em> 1998</a>). However, we found no verifiable mutations in the genes of mismatch repair factors. There appears to be a single base deletion causing a frameshift in the annotated exon 2 of MSH3 at Z:63,874,870. However, this mutation is in a nonconserved region of the putative chicken MSH3 protein, and EST sequences do not contain the exon; therefore, it is unlikely to be a true coding region (data not shown). We did not observe a similar increase in repeat-derived instability at insertions (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>), and the overall length distribution of indels in DT40 is not unique (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a> and data not shown). Thus, we found no strong evidence of DNA repair defects shaping the DT40 genome, in agreement with the lack of mutations inactivating repair genes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Insertion/deletion mutations at repeat sequences. (A) The percentage of deletions that precisely remove a unit of tandem repeats of at least two repeat units, plotted against the length of the deletion. Error bars are calculated as the square root of the number of events counted divided by the number of events. The inset graph shows the enrichment of the selected deletion events in the DT40 genome relative to the mean of the L2 plus Silkie (S) genomes, plotted against the length of the deletion. (B) The percentage of insertions that precisely duplicate a sequence, creating a tandem repeat, plotted against the length of the insertion. Error bars calculated as in (A). The inset graph shows the enrichment of the selected insertion events in the DT40 genome relative to the mean of the L2 plus Silkie (S) genomes, plotted against the length of the insertion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1978245998" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Insertion/deletion mutations at repeat sequences. (A) The percentage of deletions that precisely remove a unit of tandem repeats of at least two repeat units, plotted against the length of the deletion. Error bars are calculated as the square root of the number of events counted divided by the number of events. The inset graph shows the enrichment of the selected deletion events in the DT40 genome relative to the mean of the L2 plus Silkie (S) genomes, plotted against the length of the deletion. (B) The percentage of insertions that precisely duplicate a sequence, creating a tandem repeat, plotted against the length of the insertion. Error bars calculated as in (A). The inset graph shows the enrichment of the selected insertion events in the DT40 genome relative to the mean of the L2 plus Silkie (S) genomes, plotted against the length of the insertion.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2231/F5.medium.gif" width="318" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/11/2231/F5.medium.gif" width="318" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465239" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Insertion/deletion mutations at repeat sequences. (A) The percentage of deletions that precisely remove a unit of tandem repeats of at least two repeat units, plotted against the length of the deletion. Error bars are calculated as the square root of the number of events counted divided by the number of events. The inset graph shows the enrichment of the selected deletion events in the DT40 genome relative to the mean of the L2 plus Silkie (S) genomes, plotted against the length of the deletion. (B) The percentage of insertions that precisely duplicate a sequence, creating a tandem repeat, plotted against the length of the insertion. Error bars calculated as in (A). The inset graph shows the enrichment of the selected insertion events in the DT40 genome relative to the mean of the L2 plus Silkie (S) genomes, plotted against the length of the insertion.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Viral transformation</h3><p id="p-32">The DT40 cell line was isolated from an ALV-induced bursal lymphoma. We looked for viral insertions in the DT40 genome by performing a blastn alignment with each of the 5385 viral reference genomes present in Entrez Genomes against the <em>de novo</em> DT40 genome assembly. Of the 228 hits to scaffolds of the assembly, we excluded 14 that were short matches mainly to phages on long scaffolds that did not contain more virus sequence. The remaining 214 scaffolds contained sequences that all showed similarities to alpharetroviruses specialized to birds. The prototype of these viruses is the avian leukosis virus RSA (also known as Rous sarcoma virus), which contains three genes (gag-pro, pol, and env) between flanking LTRs. Many of the sequence hits were to the endogenous retrovirus EAV-HP, which were excluded from further analysis. The remaining scaffolds contained only viral sequences, only chicken sequences, or both. There were six scaffolds that contained both avian and viral sequences, and five of these only contained part of an LTR. Matching these scaffolds against the chicken reference genome, we found five viral integration sites. Using the independent method of mapping raw read pairs that contain a viral sequence in one of the reads did not reveal any further viral insertion sites (data not shown).</p><p id="p-33">One of the integration sites was in the <em>MYC</em> (c-myc) locus, as expected. Rearrangement of the genomic <em>MYC</em> locus and increased <em>MYC</em> expression have been observed in the DT40 cell line (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Baba <em>et al.</em> 1985</a>), and ALV and related retroviruses have been shown to activate the cellular <em>MYC</em> proto-oncogene by proviral insertional mutagenesis (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayward <em>et al.</em> 1981</a>). We mapped the viral insertion to the first intron of <em>MYC</em> at the approximate position of Chr2:139,318,028 in a TA dinucleotide repeat (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Because the translation start site is located in the second exon of <em>MYC</em>, promoter activity of the viral long terminal repeat can activate the expression of the entire c-myc protein as reported (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hayward <em>et al.</em> 1981</a>). The <em>MYC</em> gene is on the triploid second chromosome in the region that displays loss of heterozygosity, and the viral insertion is present in all three <em>MYC</em> alleles because there are no reads spanning the insertion site. Three further integration sites were mapped to the <em>SOX5</em>, <em>FAM208B</em>, and <em>SLC13A5</em> genes. <em>SOX5</em> encodes a high mobility group protein with a role in cell differentiation, and it has been isolated as a target of retroviral insertional mutagenesis in mouse brain tumors (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Johansson <em>et al.</em> 2004</a>). The viral insertion, which is not present in the L2 and Silkie genomes, was identified previously in white leghorn chickens as an insertion site for the ubiquitous chicken retrovirus ev1 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hishinuma <em>et al.</em> 1981</a>). The virus integrated in a reverse orientation relative to the <em>SOX5</em> gene at Chr1:66,086,819 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). It is present in both alleles in a 900-kb LOH region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1/TableS2.xls">Table S2</a>). The FAM208B gene product is of unknown structure and function, and the effect of the viral integration at Chr1:1,050,550 in the putative fifth intron of one allele is unclear (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). The integration site in the <em>SLC13A5</em> sodium-citrate cotransporter gene at Chr19:9,782,140 between the fifth and sixth exons is likely to disrupt this 12-transmembrane helix protein (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>). The last identified viral integration site is also present in the reference genome between Chr1:32,561,911–32,568,956, and it has been described as the endogenous retrovirus (ERV) locus first called JFevB (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Levin <em>et al.</em> 1994</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ALV insertions in the DT40 genome. Schematic drawing of ALV integration events in the DT40 genome. (A) MYC gene, (B) SOX5 gene, (C) FAM208B gene, and (D) SLC13A5 gene. The genes are transcribed left to right, with the first exon of each gene marked E1. Noncoding sequences are shown as empty boxes; coding exons are shown as filled boxes. The location and orientation of the integrated ALV copy are shown by a thick arrow above the sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1978245998" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ALV insertions in the DT40 genome. Schematic drawing of ALV integration events in the DT40 genome. (A) MYC gene, (B) SOX5 gene, (C) FAM208B gene, and (D) SLC13A5 gene. The genes are transcribed left to right, with the first exon of each gene marked E1. Noncoding sequences are shown as empty boxes; coding exons are shown as filled boxes. The location and orientation of the integrated ALV copy are shown by a thick arrow above the sequence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2231/F6.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/11/2231/F6.medium.gif" width="356" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2231/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465210" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">ALV insertions in the DT40 genome. Schematic drawing of ALV integration events in the DT40 genome. (A) MYC gene, (B) SOX5 gene, (C) FAM208B gene, and (D) SLC13A5 gene. The genes are transcribed left to right, with the first exon of each gene marked E1. Noncoding sequences are shown as empty boxes; coding exons are shown as filled boxes. The location and orientation of the integrated ALV copy are shown by a thick arrow above the sequence.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-35">The chicken DT40 cell line has been extensively used for genetic studies of DNA repair and related processes. Taking advantage of high-throughput whole genome shotgun sequencing, in this study we characterized the genomic landscape of this cell line. The relatively normal, stable karyotype together with a mutational landscape very similar to that of two domestic chicken genomes suggests the suitability of DT40 as a model for studying normal cellular processes. Additionally, the analysis of viral sequence insertions and gene mutations shed light on the oncogenic transformation of the cell line.</p><p id="p-36">One of the properties that makes DT40 useful for genetic studies is its relatively normal karyotype. We compared two commonly used wild-type lines, and the main abnormality we found was the presence of extra copies of a small number of whole chromosomes. The WT-IAH stock displayed trisomy of chromosome 2 and likely tetrasomy of chromosome 24, whereas the WT-CL18 showed additional trisomy of chromosomes 14 and 20. The analysis of a bulk population and two independent clones did not reveal any copy number variability in either stock, in contrast to an earlier report that found macrochromosomal variation between and mosaicism within wild-type DT40 stocks, with only 40–66% of clones conforming to the modal karyotype concerning the five largest chromosomes (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chang and Delany 2004</a>). The difference may be attributable to different culture conditions or a stabilization of the karyotype, although the limited number of clones in our analysis precludes direct comparison. Our results support the further use of DT40 as a model cell line for genetic studies.</p><p id="p-37">We were fortunate to be able to compare the DT40 sequence to the high-quality whole genome sequence of two domestic chicken breeds obtained with the same sequencing technology (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2013</a>). These breeds may not be the ideal comparison for the DT40 cell line derived from a commercial layer breed with a much larger likely effective population size. Nevertheless, when compared with the <em>Gallus gallus</em> reference genome, the cell line–derived sample and those from the L2 and the Silkie breeds have indistinguishable point mutation rates and mutation spectra. SNP rates in chickens have been reported to be high (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wong <em>et al.</em> 2004</a>). The three samples showed equally high SNV rates relative to the red jungle fowl genome (5.8–5.9 SNV/kb), and the divergence between the samples is also as high as 3.0–3.5 SNV/kb. This is not a consequence of the evolutionary distance between the red jungle fowl and domestic chicken breeds, which elsewhere has been calculated as approximately 6000 years (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Rubin <em>et al.</em> 2010</a>). Instead, this gives further evidence that sequence heterogeneity is high in chicken populations in general. This phenomenon facilitates the use of techniques that rely on sequence diversity, such as CNV determination, but also potentially hinders the use of nonisogenic gene targeting constructs.</p><p id="p-38">The identical spectrum of the unique SNVs in each of the three samples suggests that no specific mutational processes operate in the DT40 cell line. However, it would take a large number of extra mutations to change the mutation spectrum derived from more than 1.5 million SNVs. Therefore, this conclusion could only be drawn with certainty if a DNA sample from the DT40 source animal was available for analysis.</p><p id="p-39">The high density of SNVs allowed a detailed scan of the genome for regions of LOH. Although the analysis did not reveal any DT40-specific effects, it was surprising to find that a very large proportion of the three analyzed chicken genomes had lost heterozygosity, especially because this seemed to have happened independently in the different breeds. Copy-neutral LOH may be a relatively common outcome of DNA damage tolerance processes, and selective breeding of domestic animals could make use of this source of genetic variation, increasing the proportion of the genome that is near-homozygous.</p><p id="p-40">A large number of mutations were found in the coding regions of the DT40 genome. Although the overall numbers of coding region mutations do not bear evidence of DT40-specific mutational processes, it was still worth investigating if any gene mutation could be found that significantly impacted the properties of the cell line. In case of nonsynonymous point mutations, an arbitrary decision had to be made to restrict the analysis to a subset of the 4792 unique homozygous SNVs, even though mutations not classified as “radical” may also critically influence the function of particular proteins. The curated list of mutated genes made available as a supplement may contain mistakes due to the imperfections of the reference genome. For example, the analysis showed a frameshift deletion in the DNA methylase DNMT1, but on closer inspection it is likely that this "frameshift" is the consequence of the wrong annotation of a short intron in the gene sequence (data not shown). The incorrect MSH3 mutation mentioned is also the consequence of incorrect exon/intron calling.</p><p id="p-41">The causes of transformation in DT40 were known to include an insertion of ALV into the c-myc gene. We mapped this insertion to the exact TA repeat location of an ALV insertion in a different bursal lymphoma precisely mapped by <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Westaway <em>et al.</em> (1984)</a>. Another ALV insertion was mapped 330 bp further upstream in the same study; therefore, the TA repeat insertion site is not a unique position for ALV-driven upregulation of c-myc activity. All three copies of the homozygous part of chromosome 2 contain the insertion at the same site, suggesting that both the LOH event and the copy number gain happened after the viral integration event. In addition, we also mapped further copies of ALV and related viruses using the <em>de novo</em> sequence assembly. The DT40 genome is relatively virus-free: only ALV and related chicken endogenous retroviruses were found.</p><p id="p-42">In addition to ALV, we looked for additional causes of transformation among mutated genes and found the <em>PIK3R1</em> and <em>ATRX</em> mutations. The mutation of the PI3 kinase regulatory subunit is expected to contribute to the dysregulation of cell growth, as seen in many cancers. More puzzling is the <em>ATRX</em> mutation, because this is commonly found in telomerase-negative tumors that rely on the ALT pathway for telomere maintenance. However, DT40 shows a high level of telomerase expression (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Swanberg and Delany 2003</a>) and shorter telomere lengths than characteristic of ALT (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">O’Hare and Delany 2011</a>). It is possible that the <em>ATRX</em> mutation contributed to the oncogenic transformation through the effect of <em>ATRX</em> on gene expression near genomic tandem repeats (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Law <em>et al.</em> 2010</a>).</p><p id="p-43">Gene expression changes not detected by sequence analysis may also contribute to transformation. Currently, only a limited DT40 gene expression microarray analysis is available (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Neiman <em>et al.</em> 2006</a>). A recent study also determined the comprehensive microRNA expression profile of the DT40 cell line, finding significant and unique differences in both naïve and induced B cells (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Yao <em>et al.</em> 2013</a>). Such analyses, coupled with whole transcriptome sequencing, can shed further light on the causes of oncogenic transformation.</p><p id="p-44">The DT40 cell line is considered to have a high level of homologous recombination that allows gene disruptions due to the high ratio of homologous to random integration of gene targeting constructs. Although the genome analysis did not reveal any obvious signs of high homologous recombination, the higher levels of short LOH blocks and deletions within repeats could be related to higher recombination activity.</p><p id="p-45">In conclusion, our characterization found all the investigated properties of the DT40 genome to be relatively normal. With its near-normal and mostly stable karyotype, a lack of cell line–specific mutational processes and a lack of inactivating mutations in important DNA repair genes, it seems an excellent choice of cell line for continued research in the areas of DNA repair and related processes. The genome sequence, a blast database of the <em>de novo</em> assembly, and a detailed list of genes mutated in the cell line is made available to the research community. We believe that the availability of whole genome sequence data from a growing range of cell lines will greatly aid the planning and interpretation of cell line–based experiments.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">The authors would like to acknowledge the following funding sources: a Momentum Grant of the Hungarian Academy of Sciences to D.S., a Momentum grant and OTKA grant K104586 to G.E.T. and NFÜ-KMR grant 12-1-2012-0216 to I.C. Z.S is supported by the Breast Cancer Research Foundation. C.S. is a senior Cancer Research UK clinical research fellow and is funded by Cancer Research UK, the Rosetrees Trust, EU FP7 (projects PREDICT and RESPONSIFY, ID:259303), the Prostate Cancer Foundation, the European Research Council and the Breast Cancer Research Foundation. This research is supported by the National Institute for Health Research University College London Hospitals Biomedical Research Centre.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-47">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013482/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-48">Raw sequence data from this article has been deposited with the European Nucleotide Archive under accession number PRJEB6887. The data are available at <a href="http://www.ebi.ac.uk/ena/data/view/PRJEB6887">http://www.ebi.ac.uk/ena/data/view/PRJEB6887</a>.</p></li><li class="fn" id="fn-3"><p id="p-49">A BLAST database of the <em>de novo</em> sequence assembly is available at the authors’ DT40 cell line page (<a href="http://dt40.enzim.ttk.mta.hu">http://dt40.enzim.ttk.mta.hu</a>).</p></li><li class="fn" id="fn-4"><p id="p-50"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-07-30"><span class="received-label">Received </span>July 30, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-09-08"><span class="accepted-label">Accepted </span>September 8, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Molnar <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.1" data-doi="10.1038/nature12064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiatt</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The haplotype-resolved genome and epigenome of the aneuploid HeLa cancer cell line.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12064%26rft_id%253Dinfo%253Apmid%252F23925245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23925245&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322825500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.2" data-doi="10.1038/nature12477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nik-Zainal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wedge</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behjati</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Signatures of mutational processes in human cancer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12477%26rft_id%253Dinfo%253Apmid%252F23945592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23945592&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323316100026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-4.11.2231.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aronesty</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Command-line tools for processing biological sequencing data</span>. <a href="http://code.google.com/p/ea-utils">http://code.google.com/p/ea-utils</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.4" data-doi="10.1016/0042-6822(84)90128-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphries</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Differential response to avian leukosis virus infection exhibited by two chicken lines.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBaba%26rft.auinit1%253DT.%2BW.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D181%26rft.epage%253D188%26rft.atitle%253DDifferential%2Bresponse%2Bto%2Bavian%2Bleukosis%2Bvirus%2Binfection%2Bexhibited%2Bby%2Btwo%2Bchicken%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252884%252990128-4%26rft_id%253Dinfo%253Apmid%252F6328748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(84)90128-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6328748&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.5" data-doi="10.1016/0042-6822(85)90312-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giroir</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphries</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Cell lines derived from avian lymphomas exhibit two distinct phenotypes.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBaba%26rft.auinit1%253DT.%2BW.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D151%26rft.atitle%253DCell%2Blines%2Bderived%2Bfrom%2Bavian%2Blymphomas%2Bexhibit%2Btwo%2Bdistinct%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252885%252990312-5%26rft_id%253Dinfo%253Apmid%252F2998040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(85)90312-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2998040&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AKC0600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.6" data-doi="10.1038/nature11003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barretina</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caponigro</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stransky</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margolin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">483</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarretina%26rft.auinit1%253DJ.%26rft.volume%253D483%26rft.issue%253D7391%26rft.spage%253D603%26rft.epage%253D607%26rft.atitle%253DThe%2BCancer%2BCell%2BLine%2BEncyclopedia%2Benables%2Bpredictive%2Bmodelling%2Bof%2Banticancer%2Bdrug%2Bsensitivity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11003%26rft_id%253Dinfo%253Apmid%252F22460905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22460905&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302006100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.7" data-doi="10.1089/cmb.2009.0238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boisvert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laviolette</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbeil</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ray: simultaneous assembly of reads from a mix of high-throughput sequencing technologies.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBoisvert%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D11%26rft.spage%253D1519%26rft.epage%253D1533%26rft.atitle%253DRay%253A%2Bsimultaneous%2Bassembly%2Bof%2Breads%2Bfrom%2Ba%2Bmix%2Bof%2Bhigh-throughput%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2009.0238%26rft_id%253Dinfo%253Apmid%252F20958248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2009.0238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20958248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.8" data-doi="10.1016/0092-8674(91)90581-I"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerstedde</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Increased ratio of targeted to random integration after transfection of chicken B cell lines.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBuerstedde%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D188%26rft.atitle%253DIncreased%2Bratio%2Bof%2Btargeted%2Bto%2Brandom%2Bintegration%2Bafter%2Btransfection%2Bof%2Bchicken%2BB%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990581-I%26rft_id%253Dinfo%253Apmid%252F1913816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90581-I&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1913816&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GJ32000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerstedde</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynaud</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphries</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewert</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Light chain gene conversion continues at high rate in an ALV-induced cell line.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D9%26rft.spage%253D921%26rft_id%253Dinfo%253Apmid%252F2155784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2155784&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CT96600040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.10" data-doi="10.1093/nar/gkr218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cermak</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr218%26rft_id%253Dinfo%253Apmid%252F21493687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzEyL2U4MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.11" data-doi="10.1023/B:CHRO.0000021947.17128.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Karyotype stability of the DT40 chicken B cell line: macrochromosome variation and cytogenetic mosaicism.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DChang%26rft.auinit1%253DH.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D307%26rft.atitle%253DKaryotype%2Bstability%2Bof%2Bthe%2BDT40%2Bchicken%2BB%2Bcell%2Bline%253A%2Bmacrochromosome%2Bvariation%2Band%2Bcytogenetic%2Bmosaicism.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253ACHRO.0000021947.17128.96%26rft_id%253Dinfo%253Apmid%252F15125643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:CHRO.0000021947.17128.96&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15125643&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.12" data-doi="10.1016/j.tibs.2013.06.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clynes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgs</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The chromatin remodeller ATRX: a repeat offender in human disease.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci.%26rft.volume%253D38%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2013.06.011%26rft_id%253Dinfo%253Apmid%252F23916100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2013.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23916100&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324284100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.13" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.14" data-doi="10.1093/gbe/evt097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide patterns of genetic variation in two domestic chickens.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1376</span>–<span class="cit-lpage">1392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt097%26rft_id%253Dinfo%253Apmid%252F23814129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjUvNy8xMzc2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-4.11.2231.15"><div class="cit-metadata unstructured">Favero, F., T. Joshi, A. M. Marquard, N. J. Birkbak, M. Krzystanek <em>et al.</em>, 2014 Sequenza: allele-specific copy number and mutation profiles from tumor sequencing data. <a href="http://cran.r-project.org/web/packages/sequenza/index.html">http://cran.r-project.org/web/packages/sequenza/index.html</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.16" data-doi="10.1093/nar/gkt1196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amode</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ensembl 2014.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D749</span>–<span class="cit-lpage">D755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1196%26rft_id%253Dinfo%253Apmid%252F24316576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9ENzQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.17" data-doi="10.1038/nature11005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garnett</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edelman</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heidorn</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenman</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dastur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Systematic identification of genomic markers of drug sensitivity in cancer cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">483</span>: <span class="cit-fpage">570</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGarnett%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D483%26rft.issue%253D7391%26rft.spage%253D570%26rft.epage%253D575%26rft.atitle%253DSystematic%2Bidentification%2Bof%2Bgenomic%2Bmarkers%2Bof%2Bdrug%2Bsensitivity%2Bin%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11005%26rft_id%253Dinfo%253Apmid%252F22460902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22460902&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302006100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.18" data-doi="10.1074/jbc.273.31.19895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Genschel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Isolation of MutSbeta from human cells and comparison of the mismatch repair specificities of MutSbeta and MutSalpha.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">19895</span>–<span class="cit-lpage">19901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DGenschel%26rft.auinit1%253DJ.%26rft.volume%253D273%26rft.issue%253D31%26rft.spage%253D19895%26rft.epage%253D19901%26rft.atitle%253DIsolation%2Bof%2BMutS%257Bbeta%257D%2Bfrom%2BHuman%2BCells%2Band%2BComparison%2Bof%2Bthe%2BMismatch%2BRepair%2BSpecificities%2Bof%2BMutS%257Bbeta%257D%2Band%2BMutS%257Balpha%257D%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.31.19895%26rft_id%253Dinfo%253Apmid%252F9677427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvMzEvMTk4OTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.19" data-doi="10.1126/science.185.4154.862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Amino acid difference formula to help explain protein evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">862</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrantham%26rft.auinit1%253DR.%26rft.volume%253D185%26rft.issue%253D4154%26rft.spage%253D862%26rft.epage%253D864%26rft.atitle%253DAmino%2BAcid%2BDifference%2BFormula%2Bto%2BHelp%2BExplain%2BProtein%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.185.4154.862%26rft_id%253Dinfo%253Apmid%252F4843792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxODUvNDE1NC84NjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.20" data-doi="10.1186/1471-2164-12-274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zare</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The development and characterization of a 60K SNP chip for chicken.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGroenen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D274%26rft.epage%253D274%26rft.atitle%253DThe%2Bdevelopment%2Band%2Bcharacterization%2Bof%2Ba%2B60K%2BSNP%2Bchip%2Bfor%2Bchicken.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-274%26rft_id%253Dinfo%253Apmid%252F21627800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21627800&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.21" data-doi="10.1038/290475a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neel</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astrin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Activation of a cellular onc gene by promoter insertion in ALV-induced lymphoid leukosis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHayward%26rft.auinit1%253DW.%2BS.%26rft.volume%253D290%26rft.issue%253D5806%26rft.spage%253D475%26rft.epage%253D480%26rft.atitle%253DActivation%2Bof%2Ba%2Bcellular%2Bonc%2Bgene%2Bby%2Bpromoter%2Binsertion%2Bin%2BALV-induced%2Blymphoid%2Bleukosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F290475a0%26rft_id%253Dinfo%253Apmid%252F6261142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/290475a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6261142&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LK04700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.22" data-doi="10.1016/0092-8674(81)90280-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hishinuma</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeBona</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astrin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skalka</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Nucleotide sequence of acceptor site and termini of integrated avian endogenous provirus ev1: integration creates a 6 bp repeat of host DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHishinuma%26rft.auinit1%253DF.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D155%26rft.epage%253D164%26rft.atitle%253DNucleotide%2Bsequence%2Bof%2Bacceptor%2Bsite%2Band%2Btermini%2Bof%2Bintegrated%2Bavian%2Bendogenous%2Bprovirus%2Bev1%253A%2Bintegration%2Bcreates%2Ba%2B6%2Bbp%2Brepeat%2Bof%2Bhost%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990280-4%26rft_id%253Dinfo%253Apmid%252F6260371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90280-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6260371&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LA47000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.23" data-doi="10.1126/science.1150799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandelker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt-Kittler</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samuels</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velculescu</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The structure of a human p110alpha/p85alpha complex elucidates the effects of oncogenic PI3Kalpha mutations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">1744</span>–<span class="cit-lpage">1748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHuang%26rft.auinit1%253DC.-H.%26rft.volume%253D318%26rft.issue%253D5857%26rft.spage%253D1744%26rft.epage%253D1748%26rft.atitle%253DThe%2BStructure%2Bof%2Ba%2BHuman%2Bp110%257Balpha%257D%252Fp85%257Balpha%257D%2BComplex%2BElucidates%2Bthe%2BEffects%2Bof%2BOncogenic%2BPI3K%257Balpha%257D%2BMutations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150799%26rft_id%253Dinfo%253Apmid%252F18079394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTgvNTg1Ny8xNzQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.24" data-doi="10.1038/nature03154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Chicken Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHillier%26rft.auinit1%253DL.%2BW.%26rft.volume%253D432%26rft.issue%253D7018%26rft.spage%253D695%26rft.epage%253D716%26rft.atitle%253DSequence%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bchicken%2Bgenome%2Bprovide%2Bunique%2Bperspectives%2Bon%2Bvertebrate%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03154%26rft_id%253Dinfo%253Apmid%252F15592404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592404&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225597200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.25" data-doi="10.1073/pnas.0402716101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodd</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eklöf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferletta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesselager</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification of candidate cancer-causing genes in mouse brain tumors by retroviral tagging.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">11334</span>–<span class="cit-lpage">11337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0402716101%26rft_id%253Dinfo%253Apmid%252F15273287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzMxLzExMzM0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.26" data-doi="10.1038/nature12634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kandoth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutational landscape and significance across 12 major cancer types.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">502</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D502%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12634%26rft_id%253Dinfo%253Apmid%252F24132290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24132290&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325706300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.27" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2231.28"><div class="cit-metadata unstructured">Landry, J. J., P. T. Pyl, T. Rausch, T. Zichner, M. M. Tekkedil <em>et al.</em>, 2013 The genomic and transcriptomic landscape of a HeLa cell line. G3 (Bethesda) 3: 1213–1224.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.29" data-doi="10.1016/j.cell.2010.09.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lower</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voon</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ATR-X syndrome protein targets tandem repeats and influences allele-specific expression in a size-dependent manner.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLaw%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D143%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D378%26rft.atitle%253DATR-X%2Bsyndrome%2Bprotein%2Btargets%2Btandem%2Brepeats%2Band%2Binfluences%2Ballele-specific%2Bexpression%2Bin%2Ba%2Bsize-dependent%2Bmanner.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.09.023%26rft_id%253Dinfo%253Apmid%252F21029860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.09.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21029860&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283603900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santangelo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crittenden</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">An autosomal genetic linkage map of the chicken.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DLevin%26rft.auinit1%253DI.%26rft.volume%253D85%26rft.issue%253D2%26rft.spage%253D79%26rft.epage%253D85%26rft.atitle%253DAn%2BAutosomal%2BGenetic%2BLinkage%2BMap%2Bof%2Bthe%2BChicken%26rft_id%253Dinfo%253Apmid%252F7910177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6Ijg1LzIvNzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.31" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.32" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.33" data-doi="10.1371/journal.pgen.1002772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lovejoy</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisenweber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thongthip</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Loss of ATRX, genome instability, and an altered DNA damage response are hallmarks of the alternative lengthening of telomeres pathway.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLovejoy%26rft.auinit1%253DC.%2BA.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002772%26rft.epage%253De1002772%26rft.atitle%253DLoss%2Bof%2BATRX%252C%2Bgenome%2Binstability%252C%2Band%2Ban%2Baltered%2BDNA%2Bdamage%2Bresponse%2Bare%2Bhallmarks%2Bof%2Bthe%2Balternative%2Blengthening%2Bof%2Btelomeres%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002772%26rft_id%253Dinfo%253Apmid%252F22829774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22829774&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.34" data-doi="10.1073/pnas.0912629107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rate, molecular spectrum, and consequences of human mutation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0912629107%26rft_id%253Dinfo%253Apmid%252F20080596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMy85NjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neiman</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnside</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsaesser</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clurman</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of gene expression, copy number and palindrome formation with a Dt40 enriched cDNA microarray.</span> <abbr class="cit-jnl-abbrev">Subcell. Biochem.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSub-cellular%2Bbiochemistry%26rft.stitle%253DSubcell%2BBiochem%26rft.aulast%253DNeiman%26rft.auinit1%253DP.%2BE.%26rft.volume%253D40%26rft.spage%253D245%26rft.epage%253D256%26rft.atitle%253DAnalysis%2Bof%2Bgene%2Bexpression%252C%2Bcopy%2Bnumber%2Band%2Bpalindrome%2Bformation%2Bwith%2Ba%2BDt40%2Benriched%2BcDNA%2Bmicroarray.%26rft_id%253Dinfo%253Apmid%252F17623909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17623909&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.36" data-doi="10.1159/000330125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Hare</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular and cellular evidence for the alternative lengthening of telomeres (ALT) mechanism in chicken.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DO%2527Hare%26rft.auinit1%253DT.%2BH.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D78%26rft.atitle%253DMolecular%2Band%2Bcellular%2Bevidence%2Bfor%2Bthe%2Balternative%2Blengthening%2Bof%2Btelomeres%2B%2528ALT%2529%2Bmechanism%2Bin%2Bchicken.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000330125%26rft_id%253Dinfo%253Apmid%252F21822009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000330125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21822009&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.37" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.38" data-doi="10.1038/nprot.2013.143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering using the CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D2281%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.143%26rft_id%253Dinfo%253Apmid%252F24157548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24157548&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.39" data-doi="10.1038/nature08832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwood</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome resequencing reveals loci under selection during chicken domestication.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRubin%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D464%26rft.issue%253D7288%26rft.spage%253D587%26rft.epage%253D591%26rft.atitle%253DWhole-genome%2Bresequencing%2Breveals%2Bloci%2Bunder%2Bselection%2Bduring%2Bchicken%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08832%26rft_id%253Dinfo%253Apmid%252F20220755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220755&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275974200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.40" data-doi="10.1093/emboj/cdg161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sale</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Rev1 is essential for DNA damage tolerance and non-templated immunoglobulin gene mutation in a vertebrate cell line.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1654</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSimpson%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D1654%26rft.epage%253D1664%26rft.atitle%253DRev1%2Bis%2Bessential%2Bfor%2BDNA%2Bdamage%2Btolerance%2Band%2Bnon-templated%2Bimmunoglobulin%2Bgene%2Bmutation%2Bin%2Ba%2Bvertebrate%2Bcell%2Bline%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdg161%26rft_id%253Dinfo%253Apmid%252F12660171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMi83LzE2NTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.41" data-doi="10.1046/j.1365-2052.2000.00565.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruley</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paton</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Differences in gene density on chicken macrochromosomes and microchromosomes.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D96%26rft.epage%253D103%26rft.atitle%253DDifferences%2Bin%2Bgene%2Bdensity%2Bon%2Bchicken%2Bmacrochromosomes%2Band%2Bmicrochromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2052.2000.00565.x%26rft_id%253Dinfo%253Apmid%252F10782207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2052.2000.00565.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10782207&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086363500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.42" data-doi="10.1093/emboj/17.2.598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonoda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buerstedde</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bezzubova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Rad51-deficient vertebrate cells accumulate chromosomal breaks prior to cell death.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">598</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSonoda%26rft.auinit1%253DE.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D598%26rft.epage%253D608%26rft.atitle%253DRad51-deficient%2Bvertebrate%2Bcells%2Baccumulate%2Bchromosomal%2Bbreaks%2Bprior%2Bto%2Bcell%2Bdeath%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.2.598%26rft_id%253Dinfo%253Apmid%252F9430650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIxNy8yLzU5OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.43" data-doi="10.1159/000075769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swanberg</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dynamics of telomere erosion in transformed and non-transformed avian cells in vitro.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DSwanberg%26rft.auinit1%253DS.%2BE.%26rft.volume%253D102%26rft.issue%253D1-4%26rft.spage%253D318%26rft.epage%253D325%26rft.atitle%253DDynamics%2Bof%2Btelomere%2Berosion%2Bin%2Btransformed%2Band%2Bnon-transformed%2Bavian%2Bcells%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000075769%26rft_id%253Dinfo%253Apmid%252F14970723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000075769&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14970723&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.44" data-doi="10.1128/MCB.01291-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szuts</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kabani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamazoe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sale</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Role for RAD18 in homologous recombination in DT40 cells.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">8032</span>–<span class="cit-lpage">8041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01291-06%26rft_id%253Dinfo%253Apmid%252F16923963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8yMS84MDMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.45" data-doi="10.1073/pnas.89.11.5128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">te Riele</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maandag</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berns</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Highly efficient gene targeting in embryonic stem cells through homologous recombination with isogenic DNA constructs.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">5128</span>–<span class="cit-lpage">5132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253Dte%2BRiele%26rft.auinit1%253DH.%26rft.volume%253D89%26rft.issue%253D11%26rft.spage%253D5128%26rft.epage%253D5132%26rft.atitle%253DHighly%2Befficient%2Bgene%2Btargeting%2Bin%2Bembryonic%2Bstem%2Bcells%2Bthrough%2Bhomologous%2Brecombination%2Bwith%2Bisogenic%2BDNA%2Bconstructs.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.11.5128%26rft_id%253Dinfo%253Apmid%252F1594621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMTEvNTEyOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.46" data-doi="10.1186/1756-0500-5-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frech</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CooVar: co-occurring variant analyzer.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DVergara%26rft.auinit1%253DI.%2BA.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DCooVar%253A%2BCo-occurring%2Bvariant%2Banalyzer.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-615%26rft_id%253Dinfo%253Apmid%252F23116482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23116482&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.47" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNi9lMTY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.48" data-doi="10.1073/pnas.81.3.843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westaway</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varmus</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Proviral deletions and oncogene base-substitutions in insertionally mutagenized c-myc alleles may contribute to the progression of avian bursal tumors.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWestaway%26rft.auinit1%253DD.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D843%26rft.epage%253D847%26rft.atitle%253DProviral%2Bdeletions%2Band%2Boncogene%2Bbase-substitutions%2Bin%2Binsertionally%2Bmutagenized%2Bc-myc%2Balleles%2Bmay%2Bcontribute%2Bto%2Bthe%2Bprogression%2Bof%2Bavian%2Bbursal%2Btumors.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.3.843%26rft_id%253Dinfo%253Apmid%252F6322173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4MS8zLzg0MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.49" data-doi="10.1038/nature03156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genetic variation map for chicken with 2.8 million single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWong%26rft.auinit1%253DG.%2BK.%26rft.volume%253D432%26rft.issue%253D7018%26rft.spage%253D717%26rft.epage%253D722%26rft.atitle%253DA%2Bgenetic%2Bvariation%2Bmap%2Bfor%2Bchicken%2Bwith%2B2.8%2Bmillion%2Bsingle-nucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03156%26rft_id%253Dinfo%253Apmid%252F15592405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592405&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225597200039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nair</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MicroRNA expression profiles in avian haemopoietic cells.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D153%26rft_id%253Dinfo%253Apmid%252F23967013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23967013&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2231.51" data-doi="10.1007/BF02111280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuckerkandl</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">On the molecular evolutionary clock.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DZuckerkandl%26rft.auinit1%253DE.%26rft.volume%253D26%26rft.issue%253D1-2%26rft.spage%253D34%26rft.epage%253D46%26rft.atitle%253DOn%2Bthe%2Bmolecular%2Bevolutionary%2Bclock.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02111280%26rft_id%253Dinfo%253Apmid%252F3125336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02111280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3125336&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K936800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/11/2231.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2219" title="High-Resolution Genetic Map for Understanding the Effect of Genome-Wide Recombination Rate on Nucleotide Diversity in Watermelon" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2241" title="Drosophila Protamine-Like Mst35Ba and Mst35Bb Are Required for Proper Sperm Nuclear Morphology but Are Dispensable for Male Fertility" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/11"><span>Volume 4 Issue 11, November 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/11.cover-source.jpg?itok=oYWFGPiM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2231" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cF9w-wkMo6XLrgcR4ao9TKH4NBau7RPLOXb9_ICoqlc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/11/2231" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/11/2231" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/11/2231" class="active" data-icon-position="" data-hide-link-title="0">The Genome of the Chicken DT40 Bursal Lymphoma Cell Line</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459712" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JTuw4SGBa6zo74F6ezKTbX_PXGSyidv6NoBF6A3sbDM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2231.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/11/2231" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-F2HNHx1A7RxMjG6j07Z-nEx703PkisUI54g-Wm_LY3c" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/11/2231" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2231.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459712" id="node459712--2" data-pisa="ggg;4/11/2231" data-pisa-master="ggg;g3.114.013482" data-apath="/ggg/4/11/2231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genome of the Chicken DT40 Bursal Lymphoma Cell Line</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">János Molnár</span>, <span class="highwire-citation-author" data-delta="1">Ádám Póti</span>, <span class="highwire-citation-author" data-delta="2">Orsolya Pipek</span>, <span class="highwire-citation-author" data-delta="3">Marcin Krzystanek</span>, <span class="highwire-citation-author" data-delta="4">Nnennaya Kanu</span>, <span class="highwire-citation-author" data-delta="5">Charles Swanton</span>, <span class="highwire-citation-author" data-delta="6">Gábor E. Tusnády</span>, <span class="highwire-citation-author" data-delta="7">Zoltan Szallasi</span>, <span class="highwire-citation-author" data-delta="8">István Csabai</span> and <span class="highwire-citation-author" data-delta="9">Dávid Szüts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2231-2240;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013482 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459712--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">János Molnár </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A1nos%2BMoln%C3%A1r%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moln%C3%A1r%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A1nos%2BMoln%25C3%25A1r%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ádám Póti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%81d%C3%A1m%2BP%C3%B3ti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%B3ti%20%C3%81&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2581d%25C3%25A1m%2BP%25C3%25B3ti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Orsolya Pipek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Orsolya%2BPipek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pipek%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOrsolya%2BPipek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcin Krzystanek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcin%2BKrzystanek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krzystanek%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcin%2BKrzystanek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nnennaya Kanu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nnennaya%2BKanu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kanu%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANnennaya%2BKanu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Swanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BSwanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swanton%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BSwanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gábor E. Tusnády </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G%C3%A1bor%2BE.%2BTusn%C3%A1dy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tusn%C3%A1dy%20GE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AG%25C3%25A1bor%2BE.%2BTusn%25C3%25A1dy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Zoltan Szallasi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Children's Hospital Informatics Program at the Harvard-Massachusetts Institutes of Technology Division of Health Sciences and Technology (CHIP@HST), Harvard Medical School, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zoltan%2BSzallasi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szallasi%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZoltan%2BSzallasi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">István Csabai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Istv%C3%A1n%2BCsabai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csabai%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIstv%25C3%25A1n%2BCsabai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Dávid Szüts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A1vid%2BSz%C3%BCts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sz%C3%BCts%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AD%25C3%25A1vid%2BSz%25C3%25BCts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:szuts.david@ttk.mta.hu" class="" data-icon-position="" data-hide-link-title="0">szuts.david@ttk.mta.hu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/11/2231" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/11/2231&amp;title=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/11/2231&amp;title=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/11/2231&amp;title=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/11/2231&amp;text=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/11/2231&amp;title=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/11/2231&amp;t=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/11/2231&amp;title=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/11/2231&amp;title=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459712/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459712" id="node459712--3" data-pisa="ggg;4/11/2231" data-pisa-master="ggg;g3.114.013482" data-apath="/ggg/4/11/2231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genome of the Chicken DT40 Bursal Lymphoma Cell Line</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">János Molnár</span>, <span class="highwire-citation-author" data-delta="1">Ádám Póti</span>, <span class="highwire-citation-author" data-delta="2">Orsolya Pipek</span>, <span class="highwire-citation-author" data-delta="3">Marcin Krzystanek</span>, <span class="highwire-citation-author" data-delta="4">Nnennaya Kanu</span>, <span class="highwire-citation-author" data-delta="5">Charles Swanton</span>, <span class="highwire-citation-author" data-delta="6">Gábor E. Tusnády</span>, <span class="highwire-citation-author" data-delta="7">Zoltan Szallasi</span>, <span class="highwire-citation-author" data-delta="8">István Csabai</span> and <span class="highwire-citation-author" data-delta="9">Dávid Szüts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2231-2240;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013482 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459712--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">János Molnár </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A1nos%2BMoln%C3%A1r%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moln%C3%A1r%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A1nos%2BMoln%25C3%25A1r%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ádám Póti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%81d%C3%A1m%2BP%C3%B3ti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%B3ti%20%C3%81&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2581d%25C3%25A1m%2BP%25C3%25B3ti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Orsolya Pipek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Orsolya%2BPipek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pipek%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOrsolya%2BPipek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcin Krzystanek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcin%2BKrzystanek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krzystanek%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcin%2BKrzystanek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nnennaya Kanu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nnennaya%2BKanu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kanu%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANnennaya%2BKanu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Swanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Cancer Research UK London Research Institute, London, WCA2 3PX, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BSwanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swanton%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BSwanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gábor E. Tusnády </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G%C3%A1bor%2BE.%2BTusn%C3%A1dy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tusn%C3%A1dy%20GE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AG%25C3%25A1bor%2BE.%2BTusn%25C3%25A1dy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Zoltan Szallasi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, Department of Systems Biology, Technical University of Denmark, DK-2800 Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Children's Hospital Informatics Program at the Harvard-Massachusetts Institutes of Technology Division of Health Sciences and Technology (CHIP@HST), Harvard Medical School, Boston, MA 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zoltan%2BSzallasi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szallasi%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZoltan%2BSzallasi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">István Csabai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physics of Complex Systems, Eötvös Loránd University, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Istv%C3%A1n%2BCsabai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csabai%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIstv%25C3%25A1n%2BCsabai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Dávid Szüts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Enzymology, Research Centre for Natural Sciences, Hungarian Academy of Sciences, H-1117 Budapest, Hungary</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A1vid%2BSz%C3%BCts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sz%C3%BCts%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AD%25C3%25A1vid%2BSz%25C3%25BCts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:szuts.david@ttk.mta.hu" class="" data-icon-position="" data-hide-link-title="0">szuts.david@ttk.mta.hu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2231&amp;rft.epage=2240&amp;rft.atitle=The%20Genome%20of%20the%20Chicken%20DT40%20Bursal%20Lymphoma%20Cell%20Line&amp;rft.volume=4&amp;rft.issue=11&amp;rft.date=2014-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Moln%C3%A1r%2C+J%C3%A1nos&amp;rft.au=P%C3%B3ti%2C+%C3%81d%C3%A1m&amp;rft.au=Pipek%2C+Orsolya&amp;rft.au=Krzystanek%2C+Marcin&amp;rft.au=Kanu%2C+Nnennaya&amp;rft.au=Swanton%2C+Charles&amp;rft.au=Tusn%C3%A1dy%2C+G%C3%A1bor+E.&amp;rft.au=Szallasi%2C+Zoltan&amp;rft.au=Csabai%2C+Istv%C3%A1n&amp;rft.au=Sz%C3%BCts%2C+D%C3%A1vid"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459712/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459712/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459712/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459712/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459712/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459712/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459712/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459712/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459712/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459712/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459712/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459712/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/11/2231" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/11/2231.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/11/2231.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/11/2231.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/11/2231.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
