<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/11/2051" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/11/2051.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/11/2051.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/11/2051.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/11/2051/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/11/2051.atom" />
<meta name="HW.pisa" content="ggg;4/11/2051" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies" />
<meta name="DC.Identifier" content="10.1534/g3.114.013565" />
<meta name="DC.Date" content="2014-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Rajagopal et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In eukaryotic cells, histone lysines are frequently acetylated. However, unlike modifications such as methylations, histone acetylation modifications are often considered redundant. As such, the functional roles of distinct histone acetylations are largely unexplored. We previously developed an algorithm RFECS to discover the most informative modifications associated with the classification or prediction of mammalian enhancers. Here, we used this tool to identify the modifications most predictive of promoters, enhancers, and gene bodies. Unexpectedly, we found that histone acetylation alone performs well in distinguishing these unique genomic regions. Further, we found the association of characteristic acetylation patterns with genic regions and association of chromatin state with splicing. Taken together, our work underscores the diverse functional roles of histone acetylation in gene regulation and provides several testable hypotheses to dissect these roles." />
<meta name="DC.Contributor" content="Nisha Rajagopal" />
<meta name="DC.Contributor" content="Jason Ernst" />
<meta name="DC.Contributor" content="Pradipta Ray" />
<meta name="DC.Contributor" content="Jie Wu" />
<meta name="DC.Contributor" content="Michael Zhang" />
<meta name="DC.Contributor" content="Manolis Kellis" />
<meta name="DC.Contributor" content="Bing Ren" />
<meta name="article:published_time" content="2014-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In eukaryotic cells, histone lysines are frequently acetylated. However, unlike modifications such as methylations, histone acetylation modifications are often considered redundant. As such, the functional roles of distinct histone acetylations are largely unexplored. We previously developed an algorithm RFECS to discover the most informative modifications associated with the classification or prediction of mammalian enhancers. Here, we used this tool to identify the modifications most predictive of promoters, enhancers, and gene bodies. Unexpectedly, we found that histone acetylation alone performs well in distinguishing these unique genomic regions. Further, we found the association of characteristic acetylation patterns with genic regions and association of chromatin state with splicing. Taken together, our work underscores the diverse functional roles of histone acetylation in gene regulation and provides several testable hypotheses to dissect these roles.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/11/01" />
<meta name="citation_mjid" content="ggg;4/11/2051" />
<meta name="citation_id" content="4/11/2051" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/11/2051" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/11/2051.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/11/2051.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/11/2051.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.013565" />
<meta name="citation_pmid" content="25122670" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2051" />
<meta name="citation_lastpage" content="2063" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nisha Rajagopal" />
<meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653" />
<meta name="citation_author_institution" content="Bioinformatics and Systems Biology Program, University of California, San Diego, La Jolla, California 92037" />
<meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139" />
<meta name="citation_author" content="Jason Ernst" />
<meta name="citation_author_institution" content="Department of Biological Chemistry, University of California, Los Angeles, Los Angeles, California 90095" />
<meta name="citation_author" content="Pradipta Ray" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080" />
<meta name="citation_author" content="Jie Wu" />
<meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, 1 Bungtown Rd., Cold Spring Harbor, New York 11724" />
<meta name="citation_author_institution" content="Department of Applied Mathematics and Statistics, Stony Brook University, Stony Brook, New York 11794" />
<meta name="citation_author" content="Michael Zhang" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080" />
<meta name="citation_author_institution" content="Bioinformatics Division, Center for Synthetic and Systems Biology, Tsinghua National Laboratory for Information Science and Technology, Tsinghua University, Beijing 100084, China" />
<meta name="citation_author" content="Manolis Kellis" />
<meta name="citation_author_institution" content="Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139" />
<meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142" />
<meta name="citation_author" content="Bing Ren" />
<meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653" />
<meta name="citation_author_institution" content="Department of Cellular and Molecular Medicine, Institute of Genomic Medicine, and Moores Cancer Center, University of California, San Diego School of Medicine, La Jolla, California 92093" />
<meta name="citation_author_email" content="biren@ucsd.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Andersson;citation_author=S. Enroth;citation_author=A. Rada-Iglesias;citation_author=C. Wadelius;citation_author=J. Komorowski;citation_title=Nucleosomes are well positioned in exons and carry characteristic histone modifications.;citation_volume=19;citation_year=2009;citation_pmid=19687145;citation_doi=10.1101/gr.092353.109" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Barski;citation_author=S. Cuddapah;citation_author=K. Cui;citation_author=T. Y. Roh;citation_author=D. E. Schones;citation_title=High-resolution profiling of histone methylations in the human genome.;citation_pages=823-837;citation_volume=129;citation_year=2007;citation_issue=4;citation_pmid=17512414;citation_doi=10.1016/j.cell.2007.05.009" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=T. Bylander;citation_title=Estimating generalization error on two-class datasets using out-of-bag estimates.;citation_pages=287-297;citation_volume=48;citation_year=2002;citation_doi=10.1023/A:1013964023376" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. Cho;citation_author=G. Orphanides;citation_author=X. Sun;citation_author=X. J. Yang;citation_author=V. Ogryzko;citation_title=A Human RNA Polymerase II Complex Containing Factors That Modify Chromatin Structure;citation_pages=5355-5363;citation_volume=18;citation_year=1998;citation_issue=9;citation_pmid=9710619;citation_doi=10.1128/MCB.18.9.5355" />
<meta name="citation_reference" content="citation_title=A Cluster Separation Measure.;citation_pages=224-227;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Molecular medicine (Cambridge, Mass.);citation_journal_abbrev=Mol Med;citation_author=C. A. Dinarello;citation_author=G. Fossati;citation_author=P. Mascagni;citation_title=Histone deacetylase inhibitors for treating a spectrum of diseases not related to cancer.;citation_pages=333-352;citation_volume=17;citation_year=2011;citation_issue=5-6;citation_pmid=21556484;citation_doi=10.1007/s00894-010-0726-4" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Ernst;citation_author=M. Kellis;citation_title=ChromHMM: automating chromatin-state discovery and characterization.;citation_pages=215-216;citation_volume=9;citation_year=2012;citation_issue=3;citation_pmid=22373907;citation_doi=10.1038/nmeth.1906" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res. 40: e77.;citation_title=Genome-wide enhancer prediction from epigenetic signatures using genetic algorithm-optimized support vector machines.;citation_author=M. Fernandez;citation_author=D. Miranda-Saavedra;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. A. Firpi;citation_author=D. Ucar;citation_author=K. Tan;citation_title=Discover regulatory DNA elements using chromatin signatures and artificial neural network.;citation_volume=26;citation_year=2010;citation_pmid=20453004;citation_doi=10.1093/bioinformatics/btq248" />
<meta name="citation_reference" content="citation_journal_abbrev=Epigenetics;citation_author=R. Gatta;citation_author=D. Dolfini;citation_author=F. Zambelli;citation_author=C. Imbriano;citation_author=G. Pavesi;citation_title=An acetylation-mono-ubiquitination switch on lysine 120 of H2B.;citation_pages=630-637;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21739721;citation_doi=10.4161/epi.6.5.15623" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Q. Gunderson;citation_author=E. C. Merkhofer;citation_author=T. L. Johnson;citation_title=Dynamic histone acetylation is critical for cotranscriptional spliceosome assembly and spliceosomal rearrangements.;citation_volume=108;citation_year=2011;citation_pmid=21245291;citation_doi=10.1073/pnas.1011982108" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Guttman;citation_author=I. Amit;citation_author=M. Garber;citation_author=C. French;citation_author=M. F. Lin;citation_title=Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals.;citation_pages=223-227;citation_volume=458;citation_year=2009;citation_issue=7235;citation_pmid=19182780;citation_doi=10.1038/nature07672" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=N. D. Heintzman;citation_author=R. K. Stuart;citation_author=G. Hon;citation_author=Y. Fu;citation_author=C. W. Ching;citation_title=Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome.;citation_pages=311-318;citation_volume=39;citation_year=2007;citation_issue=3;citation_pmid=17277777;citation_doi=10.1038/ng1966" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Hnilicova;citation_author=S. Hozeifi;citation_author=E. Duskova;citation_author=J. Icha;citation_author=T. Tomankova;citation_title=Histone deacetylase activity modulates alternative splicing.;citation_pages=e16727-e16727;citation_volume=6;citation_year=2011;citation_issue=2;citation_pmid=21311748;citation_doi=10.1371/journal.pone.0016727" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=G. Hon;citation_author=B. Ren;citation_author=W. Wang;citation_title=ChromaSig: a probabilistic approach to finding common chromatin signatures in the human genome.;citation_pages=e1000201-e1000201;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18927605;citation_doi=10.1371/journal.pcbi.1000201" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=G. Hon;citation_author=W. Wang;citation_author=B. Ren;citation_title=Discovery and annotation of functional chromatin signatures in the human genome.;citation_pages=e1000566-e1000566;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19918365;citation_doi=10.1371/journal.pcbi.1000566" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=W. W. Hwang;citation_author=S. Venkatasubrahmanyam;citation_author=A. G. Ianculescu;citation_author=A. Tong;citation_author=C. Boone;citation_title=A conserved RING finger protein required for histone H2B monoubiquitination and cell size control.;citation_pages=261-266;citation_volume=11;citation_year=2003;citation_issue=1;citation_pmid=12535538;citation_doi=10.1016/S1097-2765(02)00826-2" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=K. W. Jeong;citation_author=K. Kim;citation_author=A. J. Situ;citation_author=T. S. Ulmer;citation_author=W. An;citation_title=Recognition of enhancer element-specific histone methylation by TIP60 in transcriptional activation.;citation_pages=1358-1365;citation_volume=18;citation_year=2011;citation_issue=12;citation_pmid=22081016;citation_doi=10.1038/nsmb.2153" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Kolasinska-Zwierz;citation_author=T. Down;citation_author=I. Latorre;citation_author=T. Liu;citation_author=X. S. Liu;citation_title=Differential chromatin marking of introns and expressed exons by H3K36me3.;citation_pages=376-381;citation_volume=41;citation_year=2009;citation_issue=3;citation_pmid=19182803;citation_doi=10.1038/ng.322" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=O. D. Lau;citation_author=A. D. Courtney;citation_author=A. Vassilev;citation_author=L. A. Marzilli;citation_author=R. J. Cotter;citation_title=p300/CBP-associated factor histone acetyltransferase processing of a peptide substrate. Kinetic analysis of the catalytic mechanism.;citation_volume=275;citation_year=2000;citation_pmid=10777508;citation_doi=10.1074/jbc.M003219200" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=I. Listerman;citation_author=A. K. Sapra;citation_author=K. M. Neugebauer;citation_title=Cotranscriptional coupling of splicing factor recruitment and precursor messenger RNA splicing in mammalian cells.;citation_pages=815-822;citation_volume=13;citation_year=2006;citation_issue=9;citation_pmid=16921380;citation_doi=10.1038/nsmb1135" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=K. W. Lynch;citation_title=Cotranscriptional splicing regulation: it's not just about speed.;citation_pages=952-953;citation_volume=13;citation_year=2006;citation_issue=11;citation_pmid=17082788;citation_doi=10.1038/nsmb1106-952" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=T. R. Mercer;citation_author=S. L. Edwards;citation_author=M. B. Clark;citation_author=S. J. Neph;citation_author=H. Wang;citation_title=DNase I-hypersensitive exons colocalize with promoters and distal regulatory elements.;citation_pages=852-859;citation_volume=45;citation_year=2013;citation_pmid=23793028;citation_doi=10.1038/ng.2677" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=N. Rajagopal;citation_author=W. Xie;citation_author=Y. Li;citation_author=U. Wagner;citation_author=W. Wang;citation_title=RFECS: a random-forest based algorithm for enhancer identification from chromatin state.;citation_pages=e1002968;citation_volume=9;citation_year=2013;citation_pmid=23526891;citation_doi=10.1371/journal.pcbi.1002968" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=O. Ram;citation_author=A. Goren;citation_author=I. Amit;citation_author=N. Shoresh;citation_author=N. Yosef;citation_title=Combinatorial patterning of chromatin regulators uncovered by genome-wide location analysis in human cells.;citation_pages=1628-1639;citation_volume=147;citation_year=2011;citation_issue=7;citation_pmid=22196736;citation_doi=10.1016/j.cell.2011.09.057" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=C. M. Rivera;citation_author=B. Ren;citation_title=Mapping human epigenomes.;citation_pages=39-55;citation_volume=155;citation_year=2013;citation_pmid=24074860;citation_doi=10.1016/j.cell.2013.09.011" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=N. Spies;citation_author=C. B. Nielsen;citation_author=R. A. Padgett;citation_author=C. B. Burge;citation_title=Biased chromatin signatures around polyadenylation sites and exons.;citation_pages=245-254;citation_volume=36;citation_year=2009;citation_issue=2;citation_pmid=19854133;citation_doi=10.1016/j.molcel.2009.10.008" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Tan;citation_author=H. Luo;citation_author=S. Lee;citation_author=F. Jin;citation_author=J. S. Yang;citation_title=Identification of 67 histone marks and histone lysine crotonylation as a new type of histone modification.;citation_pages=1016-1028;citation_volume=146;citation_year=2011;citation_issue=6;citation_pmid=21925322;citation_doi=10.1016/j.cell.2011.08.008" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Z. Wang;citation_author=C. Zang;citation_author=K. Cui;citation_author=D. E. Schones;citation_author=A. Barski;citation_title=Genome-wide mapping of HATs and HDACs reveals distinct functions in active and inactive genes.;citation_pages=1019-1031;citation_volume=138;citation_year=2009;citation_issue=5;citation_pmid=19698979;citation_doi=10.1016/j.cell.2009.06.049" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Wang;citation_author=C. Zang;citation_author=J. A. Rosenfeld;citation_author=D. E. Schones;citation_author=A. Barski;citation_title=Combinatorial patterns of histone acetylations and methylations in the human genome.;citation_pages=897-903;citation_volume=40;citation_year=2008;citation_issue=7;citation_pmid=18552846;citation_doi=10.1038/ng.154" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=K. J. Won;citation_author=I. Chepelev;citation_author=B. Ren;citation_author=W. Wang;citation_title=Prediction of regulatory elements in mammalian genomes using chromatin signatures.;citation_pages=547-547;citation_volume=9;citation_year=2008;citation_pmid=19094206;citation_doi=10.1186/1471-2105-9-547" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Wu;citation_author=M. Akerman;citation_author=S. Sun;citation_author=W. R. McCombie;citation_author=A. R. Krainer;citation_title=SpliceTrap: a method to quantify alternative splicing under single cellular conditions.;citation_volume=27;citation_year=2011;citation_pmid=21896509;citation_doi=10.1093/bioinformatics/btr508" />
<meta name="citation_reference" content="citation_journal_title=Cell 153: 1134–1148;citation_title=Epigenomic analysis of multilineage differentiation of human embryonic stem cells.;citation_author=W. Xie;citation_author=M. D. Schultz;citation_author=R. Lister;citation_author=Z. Hou;citation_author=N. Rajagopal;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=G. E. Zentner;citation_author=S. Henikoff;citation_title=Regulation of nucleosome dynamics by histone modifications.;citation_pages=259-266;citation_volume=20;citation_year=2013;citation_pmid=23463310;citation_doi=10.1038/nsmb.2470" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhao;citation_author=G. Tavoosidana;citation_author=M. Sjolinder;citation_author=A. Gondor;citation_author=P. Mariano;citation_title=Circular chromosome conformation capture (4C) uncovers extensive networks of epigenetically regulated intra- and interchromosomal interactions.;citation_pages=1341-1347;citation_volume=38;citation_year=2006;citation_issue=11;citation_pmid=17033624;citation_doi=10.1038/ng1891" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. L. Zhou;citation_author=M. N. Hinman;citation_author=V. A. Barron;citation_author=C. Geng;citation_author=G. Zhou;citation_title=Hu proteins regulate alternative splicing by inducing localized histone hyperacetylation in an RNA-dependent manner.;citation_volume=108;citation_year=2011;citation_pmid=21808035;citation_doi=10.1073/pnas.1103344108" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=B. Zhu;citation_author=Y. Zheng;citation_author=A. D. Pham;citation_author=S. S. Mandal;citation_author=H. Erdjument-Bromage;citation_title=Monoubiquitination of human histone H2B: the factors involved and their roles in HOX gene regulation.;citation_pages=601-611;citation_volume=20;citation_year=2005;citation_issue=4;citation_pmid=16307923;citation_doi=10.1016/j.molcel.2005.09.025" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="twitter:description" content="In eukaryotic cells, histone lysines are frequently acetylated. However, unlike modifications such as methylations, histone acetylation modifications are often considered redundant. As such, the functional roles of distinct histone acetylations are largely unexplored. We previously developed an algorithm RFECS to discover the most informative modifications associated with the classification or prediction of mammalian enhancers. Here, we used this tool to identify the modifications most predictive of promoters, enhancers, and gene bodies. Unexpectedly, we found that histone acetylation alone performs well in distinguishing these unique genomic regions. Further, we found the association of characteristic acetylation patterns with genic regions and association of chromatin state with splicing. Taken together, our work underscores the diverse functional roles of histone acetylation in gene regulation and provides several testable hypotheses to dissect these roles." />
<meta name="og-title" property="og:title" content="Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/11/2051" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In eukaryotic cells, histone lysines are frequently acetylated. However, unlike modifications such as methylations, histone acetylation modifications are often considered redundant. As such, the functional roles of distinct histone acetylations are largely unexplored. We previously developed an algorithm RFECS to discover the most informative modifications associated with the classification or prediction of mammalian enhancers. Here, we used this tool to identify the modifications most predictive of promoters, enhancers, and gene bodies. Unexpectedly, we found that histone acetylation alone performs well in distinguishing these unique genomic regions. Further, we found the association of characteristic acetylation patterns with genic regions and association of chromatin state with splicing. Taken together, our work underscores the diverse functional roles of histone acetylation in gene regulation and provides several testable hypotheses to dissect these roles." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
    <title>Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lnBjasyC5RW6ybqCsv4mHuYI-NgQPRDWilsfcWGQGrs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459715","apath":"\/ggg\/4\/11\/2051.atom","pisa":"ggg;4\/11\/2051","ac":{"\/ggg\/4\/11\/2051.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/11\/2051.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2051"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2051"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/11\/2051.full":true,"\/content\/4\/11\/2051":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KIJTS\\\\\\\/84czskrsSNo6uaS0dZag71bJ5jE83jYD+yYfDLzljla0Be36nvLIZJMd3fK4YOsEY93sU\\\\\\\/Q2T8D+A6zun\\\\\\\/8mHE\\\\\\\/9uEu1ItUxSsdMV1oAFw+c7AeYv3VfbFpCkmxNa7k0i5sxpRh2VKkeKGpv35hv77mGc\\\\\\\/r7ImyDAgHYvM1dshMFttHLHj0HP22fLlFGxcDsOmc26QQywmYctg7IEHWKcTpLK3j2VkiRfFi8vo2Z5sgFtDyeC7gppc0XhhjjPSL91USzqf7F1XZUnG5shFqOSTpLjWacZ2hDdzh338Cz0Y4jz\\\\\\\/BVMqlR\\\\\\\/C7ZqoVz2ubGPC88kSmIHSks3P3xbA+UP89O+D2r6MNflYWUXyJ93ciN\\\\\\\/hMZdYx80AGjC1fXjvsceAZXH+NOXqY2dpBOSU4SZzUvRmGwSwcoeLsTzmBWsBEuaPefMK+TZwLmHcvJQg\\\\\\\/0tVGzxSDU0Y3cqW5R7rnWrG6InmBOcYajkeJkBieFKD3cWPt7sZnvyngafu2INmX90o9UPZQhGwi8HSs0KdsraMp8mrI+zLFolfnlogVTwZQPeCAPxLLTAubzf4NOU7coTMLlnzPyIjttagMWy9P+jKgXIVH\\\\\\\/w8odiLh1BZh4lyVJ8PPr1Cb3Sqg7g2P2EnkH9AzU6EkaF+0KKQkqFLt4vX01J2ZyVrLK6vAoguGE\\\\\\\/Vv2Vi3OnlCiU9JXup3IS6JH0P4\\\\\\\/HdcAMOt82ZwZ9Ia5ezLUdlRxhbM3m35pRyQ0s0FPtXWZma9ELi3oZJe0IKCUkrG2H4A+GaSkMqaMtToRCa172ImDzOlyu42spgPuDv\\\\\\\/9MS\\\\\\\/+J5uR91pJC+97QowjlY1i+zlTq7Mh24LkB+G8LfOwUqAk90e85Ly1j+QEiAifwX1PjhonaPV8jBoZoy1mHPs1kNg11pKU1IQr4VmR01yWA88rMaMegoQgnNOOgBV4JTAKKgkxWAKFLFW4y985UEqcwgRRwHDNqZHndmujtOGqRi73onBMxrGtM7t1cFWsO5WE9vwpQ\\\\\\\/cqnBilcN5A2Fe\\\\\\\/ERW0asBNmL6SJKtH4\\\\\\\/9xjecxpUt7yb15Zpo7XXdN1vogRR+0L9tghMWsnP+IW5St3iWuyN8lOung2a75v4nXkiQI2wHSDQlkVAnQ0Z51Qt\\\\\\\/NF23JX+ZT+Vzrx60r3nueyvnW8J0V28X+bbzGihxq\\\\\\\/e4NN6aq6R\\\\\\\/\\\\\\\/Uq65NyUseHgB7DLk\\\\\\\/L2tkRkmX\\\\\\\/UR+JMhEb5SqTw28oBHe\\\\\\\/faQY4D3khYw3nWqtqiZ3XU8TrL6zeaPa1xziuFOr\\\u0022,\\\u0022iv\\\u0022:\\\u00227II8+GzaXgU0tTfMDEQjbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022abaa0c0c78e71d783cc597a5807bdbddd6337044075deb0a93a7c3e2afb0749a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uaMcFgITDxNVsPIBSxM2IYYWuAXbnAiVOWfhBWGY8nLpwGaaJu7CNAawazyK3+kuj4e8zXEvaMiBEvubXkctRwOpp9GxXpWyHJ9txlkUHhDAlaVyF8arRkAL978h+csg7C9t1cpC+7aMfaRzUJu8p5mjru1AMU1nCohwQJDL1Ur0XFC2O1V8R89o9Q1TRzNmlg57o+PMNvhm5w0ZRytau9zydJRXqa\\\\\\\/u8HPNlRhoaLRD\\\\\\\/jXts2dFyns5YFrazfw0Ypam9KOr1tlBG+LZzbMS0994LlAt22NAg3INyClz+IgU99iMSJltiqBbVWNAMLMJ8PWtgFCcnLpKyVfUtfiHXWE90Jo\\\\\\\/w+fhcfFs090Ng3HgJgvQ5DrNyiEcvidAMUDAbgJD+WGV0+s8uIsl\\\\\\\/KbeQgIHuDYjNIQnzkcfBUMW9fVLZIKUKRyG9xyceNegvD+8cvDFaBfQG7q6i8ff6+1If+3Fm2WZLd4FhQNcH8cnXEjUXzrm3sso3i96WXUjTHl5kJ99PfjCPM9J3h3gMkti9PHrVgbIOIG9ByBTPBX1iALYK\\\\\\\/2+fo9Wl\\\\\\\/WmzaxUMjMmvs7cULH\\\\\\\/Mxk6pi9myJtMrHc1DAT58vJhN5uYXFShB0i9\\\\\\\/torom7beFhq736p5lh1ecHUTiIXububSx8ZvK2zEAlcEoDPlwQ6y\\\\\\\/uZxt5Fw\\\\\\\/z3iPDtSOGm8I+04Nqs8JiW15ClgkwlqeBqKvklaAbvty\\\\\\\/2y24UXBg9jZtUBHXxcLNzl36hqh\\\\\\\/eF52fDKZosHhHmIqvR50Ku2wTtzz\\\\\\\/5p9HBEY6KRlzB\\\\\\\/+ugwbeg07OmhMIuydIMmNlqXJFR1Wed2wNIHEv9QMgUSUHXvAlZPEOXD9L2BsoAkExawI2iTFWaiI\\\\\\\/PH2bjY+Ng1HQt3a0kI+tRGQeQMIZMd+fvQvbisVhaMj0vJcoACRgn2NUoGbIMLILADcTp1NJp3752q7VZ\\\\\\\/kAnCXwKyuhgmHwY8QsCMp2UNjJ8j3tZCiV6taOCgLPMD2PYeZY6+GxrfSG+FZA0uDehtNbVojbjhhJ21Y7DxnWFuP\\\\\\\/+1yUxultnNoNAmFstDJrgCiVJLXsr39HbxsVxz3UPbfBfmgbmTaK6mzSOfeQD66vgGX+cMg4eFhwsUVSqZhfFJx9l+jwoRjuXSmWPf8HeCMBQX4JvLWxuj29UAzK7IjvjA9bqBh4kODWAFPzUiEWYk\\\\\\\/PI2tPzP\\\\\\\/VCJzEMq0UwmAnkuNAfRsDOYVsDAMXRws7v\\\\\\\/N92r+2xwCOfTGhk355xXal\\\\\\\/hbBtgjJNuEQ0OI9AcdjjWcjn3v5M3DzSN+F+pWiJ14An6VMuQrbo8mrkWrQjzXVV7hIrw7V44+REqh5z6PRR8EHfFv070Ij+wy2Vf4jtOHhM8HPPR0+n3yMNBERDIpdBP0HUzWKTgYfNjGJdAk+CYNJ8kOTf2CcL1OdkLuk5pkBOvD9FQIExDJPCug\\\\\\\/goH3KQTHKkI\\\\\\\/dtbrycG8gyKymwf79PnI46h\\\\\\\/Jai8mY\\\\\\\/1myuVYtj4ZIC6cah4ahUptjLM8FsFoN8GaN4Zznpoisb0xZbQ2\\\\\\\/LBLHlBAHUp+1rosX4IYLxQ5EZxWVop0LMvUQISTG09clgIVz1Y9Cpr1CV9yi4VvqWy4QiArQeuI9Cm2795z4cStOuTR88QH6aVq4qezngCGYJUDuWSuEB9CESjxPPMkpzOBpjP2Lk7J\\\\\\\/dURqFhaYKAiybTRz1jLOdqftK3e1I3ML0KQjFZOldlnIn9Qt\\\\\\\/tDxu9S3hc58DUUDni7Yc8OzUfMYw9pssDvHDMVVYsYqp+fnyPa2fA5zDBzTMNI1+FFeopRb4N9Q0MPXCias1wzLBZlMJLaalFawtC6aAxI1qPSHO+7\\\\\\\/8fujNC0rLUXZmABLIKFFK\\\\\\\/LOacFbfyhtVvnv+zK0jaknSVXv3CZS1jIc4Lm6MCBozCOWJqccjm+Cc2hxvMHssLJVzo32C6aIL5eZ1SrJd43mGoejjgyR2CgK50oJNE3VAQ7v3udp\\\\\\\/6GXiLTp4U+IGkYPQF6AC+6UmhToDj\\\\\\\/3Gia6N37BIi++LygL98W+4\\\\\\\/fS\\\\\\\/v6noI5gssof1x3wa\\\\\\\/zYG\\\\\\\/FVMvqfn01geUbh7d4Q3KNQ9Vh\\\\\\\/gyWYbbqhCCkutwVxlbPNwpJrFo9xz7UsQHbPXMiLrfhQGgB2bCTovttH8ipuBkuuJP7Wv6N9jfSdC\\\\\\\/28Ez4TPC3XCW1cACVpyWznIj5E\\\\\\\/meEFzTQ\\\\\\\/oYvJB0UV0LszpQZO\\\\\\\/fXMJg0l5uZ8oHk6U8Yoyg0In\\\\\\\/6t4T21oE86FHk+lQYRHQGx02gB7bN5DL5khZlSKFvwn8z5EgkAQFpa5ihZN0+k5tfmuxyKMh2GjtNuPtcjkbbhF8Ink1AC40WOnJLK85NQTjHRoXDo31PwlJTQY2VizSuWmBaUvgYx2BhKvWkSvXXVEIs8WuMEAK0XIgE4Qd8vvQ1bajMoknttcuNqZiZ3+9eY6t5iok65\\\\\\\/mOwpTJfJmWh\\\\\\\/sJOswjceIhXKhUT+FwHKvGhiOO0Cj2ebgYzm4ZKKRaB85fH2E6pp\\\\\\\/8Q9kFyjRPQdq4x86FtlMCQgdA7y52okH6m6hZ2Gecr56slnvZOVQ8qAAEZo0Yyf75VVEoKUpz832Jtgv5QLuRZhiWxZ2JTGNApBoDe7GzStfI6ziYAs7v5MP3R6zQfHAycCMIknOPcyDYK4g7kGxdcwYsZjtHEPg3ofbNne8S8Y7JRfwz0hHT0QPBCb9lM4GN7kdwLzSWMbqpJnpMWYXPJfiCAceUXdYWDifVUrttegUG9fbDtkpttL8TRd4WOdIn6VWX\\\\\\\/WLUElxgwi\\\\\\\/s94N5JB3wlong+y2KJzq24DRZA0+ux2tNomwjUwGs7UX1XxrD05ApyN\\\\\\\/FePOFxZnf6Sbe3bcdBcQiYLvETwlPAd4I\\\\\\\/G43tlh9thHXPfe9+QmrAqTlOMsPu6nOuiKkdjwxMdZl9IhaaZVfPpfOLQvBgNXu4Q9nHdzVOjvaoUt9CAyIU8Jq4ld2yVjluwDt8PJ+zmzgvJNZPGC4xM4+eO7aNQiicG9AJk1mWoUYb6wWsv6\\\\\\\/MqkmicsYBhGIQJ+\\\\\\\/thkLVuJ0NWQ2IGbg1ftiNR+F8DPkO0S+KcpcZ4UhasRgUtwDFlAPlYGkyP9W2MmF6s4tt5FgXpk6Y4+x9gi4m+PoF153BR0E\\\\\\\/NfGeB7NLjyTLLjJ9Tt+B3ZlSO4qyEuXjnvQPTjy3kdv7S\\\\\\\/8jemXzYOxRF0Klp1hy4krTrEjippU\\\\\\\/ooZuALJN6Dfl2CWeuSKD\\\\\\\/S32rjnkCZBT0PbT7Wr6OKvRJPmjYusIasd7mXL9RLZ\\\\\\\/moMBOXS7IFvNUpjln+toIgAhi\\\\\\\/ITndc69UQs9HXNrPWX3qwVMFenTQQU4OWFO0Fx0W4waqzv+hVfT8Tr3f2g+l\\\\\\\/Xp\\\\\\\/p0wGSgN7W8hi8WB+RZL3yykMdIMQX9EG22uHki1c49pQdZFH3\\\\\\\/N+U3GP9diWGyPfyMyFYo54x0Wbn71p8OI2cmcDd+mHdSbYNveEZcEUfzIwl12twyOvyr2pnAp1lEkKwc5chJbM5wv9G5Lik4KSGc8tgBvgGZtP\\\\\\\/1wdeXJZmS755AjykvV1adT55tOL0mfLC3+oZUUFv3GTXcg1UKz+NGTm24rZpm8v\\\\\\\/487E6+5IuGWmK5gtX9Il7ElHwub5qea2jmKw1nSZEtNV3nXUfRCHjG22U7vuuWXbxR3xPBL0eL7BVCUclrmDkofefChDiA0Q0DxpSrFqziqQxN7VOaILfuDN5NFgqG6xz0d\\\\\\\/KlEwS6LSBKaL3zHy7a84ki\\\\\\\/YN6mO0HILW5FqwJ6\\\\\\\/WA8SrT4hf3IaMdA843YWywnV2JG9PnKCi6f8BpPgvVW11C5xaGTffDvCzfVV6qXRMhi9igrcfKPc5OiG7qJpwjvfAGEKYZPMw1MVYl+mxHwjhq3CxzAuqhMDCgYzLeqCa6jn4BLD0szGU3PCaazKYjjQq+JysjbS1WwRSwBd0qYEFxUqEvP9Ltly6jStTs+1\\\\\\\/4\\\\\\\/m3lpr49SbfgdReNQU6qHtiNadI7N1tzPC\\\\\\\/iiyzPTEneoov0neiJIjLWs8AH813wdrYIFQJsKduAb2gOyhCUJC+2GQ4vnXGVj1JVwGV3DUOfxTSw00sVW8RNH1jSJwOwjuvMN+isoLcs4ig\\\\\\\/abOYcitdr+saToLhHvMlLFi0ZLi5Rq4gpdFd94ZALLfaqRnf8AZvFzl6s1RdsDN1tOauP0jhtni7G1Bil37IsGXCLhWf6\\\\\\\/M8BQBLgD\\\\\\\/sjYs3lAYec02JzlYRFuQdJQ0kjgz+1JCPUm2dh5nCC3nrxEeEN5sd1fMxll1aINyEQhcKhwTfIN3JMvL6Es7p6C6LgBfcGEv\\\\\\\/Dml87YgoSFngmc\\\\\\\/UYq+CBDvIo74QPaM\\\\\\\/xQFYqjrUgTkOkjzG+ZU+ip2VNnLfJr95tS+OV\\\\\\\/MJ91b11bguVQ\\\\\\\/WfLkjVOMA3f8AwXl6tRnsj+tJsu0br7rnna+BFgs6500XH2yJyUG7VnJMrcLjMad2xUXVbAml+mDqAlvCkF3kzQuIvrYbZN4tLMbUgiBLjd\\\\\\\/XU9LFV7Kne0kYHOJsavlS07os980V8ynIzEtn\\\\\\\/KtbG5njbHSCNHAGgkgMx8c21J3oKH6jApZSBUtfqK+c1VKkGc94fXQKfiPINioL5G3koFjlcF\\\\\\\/6Q\\\\\\\/I8d9DCLUwd1m3umEIabQ6zxJU1jBQwo1Bfc\\\\\\\/kHPoC6+kM1ny2xlzTHF5ROle7q9xHmb63FhTfOI1GT7Q10aThWVWE1Yjq++1dO9s0kbNiO4BpVBqn+pxqU1lXybbodNWSD\\\\\\\/nrS4uMutno5yEVJR8OK0yuD1WX2bh5cMuqlQmcDaQuzuOp+r+EFAmCUXsmzAnH7UgTGnwgNrkxAv5XfC+Qe+To1Rv3c43Z+EUrrGsI2B0YQOzvHKqPYYIeGpi3jQOFobM0+UwPhb4uA0QOCIVRL67XLdaoPQayvV5DCb2EHvOCL5MZGN0yw+8pHjA5gkoV\\\\\\\/LQepC542YzLaC7TMOG9KfhjEIyFCiiDHH4RYLLB4ya7FI5Mi36GowUeI1Kf9CQiW\\\\\\\/KWPR5oBuxNZSHepuEPh6gE5Oico0LDuv120hRxXLIXlBux0rHNIJuDEUAYk8XJIlterRIOVxkYGW+Occ00MqOq1JYvDPbaGALQzxobMekIkS5Nng7tOX6D25ye6vBBbc0oAac7+fjXswCHC\\\\\\\/2B7MpqWX01tar8gx+TO4VXyF5NSoRmF3q40DFEZMWrB3wH9Cqpy8z5khzDPbJod\\\\\\\/jy\\\\\\\/eCRzTsk88o7rlUusKO99GqNXROEiKg17acglssKyUmAGGsP7d4N+cn\\\\\\\/IpScsmDeuWHoI+TD94Srggz48hAbyqgDVw1WafRqYXJXUGhuB9xzG2PXiS65XJO8LoMEDA7IlJ4xvq8TKuF0+e0X7Im82NbpDD067\\\\\\\/sokKxMuNjQHIWbJwbTPXEkv+kjIP0dXDEQ85W\\\\\\\/EwFVkgcKtWE+RsNVm39sC237I0kOvKPzAqpTgucNGZfgbozKm6vnIvLPUEfS\\\\\\\/oNZHLPY96irXCyVLawA5F7vfF3fxaQ+QzT71gv0uhjTczWglYWSDQemazbHvWKzVm8cz1l4s1B7ObDqES7ktzoFnHSJSeTwYpYdLSLMEY3tqLaUBnUoHdxLQ9\\\\\\\/uDkg6qEmNK71v0ms3NQyyP0Asy88m96lFo4k48rHCmb31WEvJCwm1qVcHai41W30ahCZfbIjUBhw\\\\\\\/CTzcxAB9m7yqfMfIZP3nILzHL39lcChK78aP9VgK10IermFTG8NwFJlozombU0vNTCK+mJHf7xyxHj3tNqyTcRY\\\\\\\/3bOqOcnt27CPMn\\\\\\\/bLA0BB7h3zgwNgOJnti1sugngzL\\\\\\\/GByPUlqvz6CDuFNXPlRWGDzwGiawMaoSGh1MhX2QMFSvivg7tbUBgqJfImYcVsF\\\\\\\/8ZV1VeXkQIRtn\\\\\\\/Hf5bAUPod4zBGonwa18QtH0pbPzYqKBBntBj2kgMfbPfgjs4hLoeSkovfB4GuEx5pvWUI+QfjEPmdk1As+RWcIK+6HCnP7IHz6gY2LNg2HHb4FVTCeWthPl048OsuoUFw7Lc5Qg7dKnP8jFfpiccTClvsBw\\\\\\\/eTINJioq5Nq76u55LBHBSL4iOnLYUncLqvhjxWG\\\\\\\/8qH9qFf\\\\\\\/1+mKoATO0AdLctQfIOM9gvdabCCVrzZQQzbXAdfiIkwmxyfANZK939VDVQb4113OlW+8tg1MayTEZuen5O4S\\\\\\\/L4s7lPNkzdRuXBryi7B9tgJ62J1Cz276V1YvQ5vkozR0wgmmBUwgc1PKF6nXBHcpzDlQmLXIsu4wfy1udgi3HNJUwDzzq3zj7F56Vr3fzEEqd+XfjZrPAvqua8G52bs\\\\\\\/WAVRwuL45bB3tAsgf\\\\\\\/BLmf+OdRPezRj2zxvdmI0JDtVRcBNfMy4CO4gMdhbg\\\\\\\/88nW67PDkzFhMIOv2ydaRv1awKWheL99rmoHamtr79GCrYri49eVy42IKdDBAb3Bi+2TNxErYo7jvg2tO3hDgVkvVzLRYl\\\\\\\/sVRpRk5c11ITqRaN3brivNJnegFKpjY5i2ZZ8IH0xQRphvIyMxppHgQK1eudzKsX+rUgrfSkZkCj8w3x+A28p2V+JGmmNj5hIMQaRlvqBO1c\\\\\\\/8UjMB3Qh8Q2HqiTDGsEcBL76OCv1BMYrYYSt+GoFSH9wy0l1z2mjy442EeXHsyAHiC21FtINq0AlnoZNZkcr+Fcf\\\\\\\/CxiqGfFrRdt5HXk4xdGy15sjUkDHu+mYvbDF7uBRIFxnT\\\\\\\/srR\\\\\\\/q87OFie5s6AhlujBcyymp7mS4oDD8idpFdobEppZ1dnRsXZbFZWFz67AxqMk5v7jZQJHeuslMVULRvx01neBy2eFXAqPREE\\\\\\\/ST4O0DXK08v2OSRDG0Os5jVGabrhRvBWLxjM8KaPPF2rghhDTIwepxkV3DShz7vaAUUuMqWCAGJC68D7qgsoxqpBuXnoeOPd7\\\\\\\/CZmf\\\\\\\/XMeChcVyAiAovKkTW8c7dAx07xlHiwSCggDGJdZDsb8FuMXiPv0Mm5B+LdbCpYCBgtuI6vCfPxKOHnyTmVlW+fweE+5MqDU+3npdaKkMmbGiKGAh7zQOUUHNCKOLXEju7XSNuDuhAQpFMQVFDVXVtd5uUlwfEgN3MMp\\\\\\\/\\\\\\\/cP\\\\\\\/EFI4OVSXI8H+ZcOvlXxMLZ8Rf60WyXjV7uvrAvhMuViBxMqetA1AwfgSOSvZJlQydyKmWu2EkRvZDTX2SsKIBEhTiizIcJk2cjctQpvr0Q7NA9faNXh1hQDaYh6eYRjWT9RI6wWNU7h+NH4rXyxE7n7BeNUKubwrbwWr6KgZH7gkSs52tVDdQqilMTGN9phgPxSDnZWOjwtV4J82hlCcJBuFcR2qGVr5iKVIx94UtR6auzufKsAkj4N\\\\\\\/NOrd8esG\\\\\\\/HDxvioeGKMoL2l\\\\\\\/SqxSy1uNVJF4XVHYyrs5lOclb7myGV2buaGplAMs30WGyY6gnMcnD\\\\\\\/tqn8aW7+wgLglvnEH4OQLRwk6a5GeKC+jxoqTvCYMg4RdIbrNZVqBM1iBCR0L+Gh392pCxeWBETqldzIeOIia9v5klcgFAInwtJG9ghkBtQ9tqgPiznuPJxWlEIS\\\\\\\/Hu4BsR90h3Krxa3I+irDtPkCUkbc6OScT9BYRJRJVFvriNVlDeRDBn6UwCCWzvwPqLEPcDy9tcskhsnyvE10PWlvyyc8zFYtEc0LMKpCLE6Oncpp21ngQinTjNIjyGxOSMquRN1O1e+iNT8wVJ3Vs7\\\\\\\/5ZtJpiO2kXWFUzNA4ND4G4EZ4+Ao9XptZ7Ns2cfHUhXuihvzNwQW5BqdjmmNxw6Y8s5ljlFAKxpgMXlk\\\\\\\/+Ab1sHdkQFWjjNMYFqreW4Ue\\\\\\\/iXIiahZ2RWTfW6zYFGQIx5q976E62xDf3bL1oQ3FH25U6778gQjJ8S14e6ZEsi0JqDtKWNj2kxFHLLyVWwZ+jzhFVkTLH2QtXNDcTq3uihDTyTRKq7HMqgbRH\\\\\\\/szNFr0vwsn2op1FbLtndXpkb718TQDXUgNWhHeHdRXX+PQ2e0jV1+9QR0hAzO8t1I0jfOCr659RUgzvPBGOcGfjW4u6gBnpL6rakbcnb7uc3cFaJVCqVarjxSstNVUSYsCJ7fuK2Yalmlo8Crd4z+e6NRgn7d+M+bkB4s\\\\\\\/xWBCWrlygFQtzr1quTl\\\\\\\/tB\\\\\\\/aUiTAblO5NDFqQ9rmroXytoANGv3FwEfAAuYR\\\\\\\/7oLKU8xkWfBUMIE+vneTYSw7hxSf4FEeYgai7F6Da1eHQryNMOFJQPX3bHZ+jWaUUc+Zu9UQg4waZ\\\\\\\/6\\\\\\\/Jh0D439JLqE5y27u8099WDH+0jl8h4Czj+SXUkQwexQo9jn1cVL9dv9JOdMPJDvXgej46s5Sas48d+\\\\\\\/KaBVMMkE7kwuXdsQsuyGhkYFPX9HmZxtMYlulGw4MctJmjIfU05KxtXWqo\\\\\\\/r87vDikwZJfhtQ6yeBr2ENL1OW8MNUy5VN\\\\\\\/uYeeGDz7sZkwWdWii7GyiCdWrrFlRxnBNmYG3HoVr9Ab9yic6iY3ZDiBglrX97DedgfUdaQ3FwpXenoDPDrsB5sYn0MpaKyq1ACtSJQnIJeFj5Q\\\\\\\/0niW\\\\\\\/vRKsIdw7pxqhttIdqpNXeqr5JXkF+VEevgpGZG3NufTstbGfe1iegKjIclVSyjOyDGXvtJwpChZUo1cutDKmKr5RvTQI+gvjAim68WaY+\\\\\\\/mj2FLBzGNCEfMebnIrw\\\\\\\/dgeEu0\\\\\\\/6Tsv2YUXSGNkPRRjtiCMYaWX4nlUg66LLOTftmoKpgi9FLH7bNhfosJidcf20vilYkvS7+ZjL87Yi4pndKJBWR1dnlj131gJXRKDIfBxCvDqFQ7Kg52sF8dcl2gKyE\\\\\\\/7XLeNPOFG4R4vHoq6GyQiBwfUa7Hy2Sx3TghxrxRVUTwImZaxRpsgQSst5TPN069McuIRJTJbOiYh5AOOI4dFxloO4K8Mq\\\\\\\/PdcJ\\\\\\\/pabbU4rRRP3+d8\\\\\\\/Wey5EyiYGShj2cZasiJMbK8ehhekehHSelzLeqlozILV\\\\\\\/RdVfIP6bNW+ePYGzyI0eu55lsE6wWeQ7GAPJTVZ3AE2kjyvba8to9W0jd4SyXv\\\\\\\/xMPn+D4lWD+U6VE6fqwzAbEqT4X3BneuIbXH5Xi1vmnTsRAN09jU+O5qKMlZ0ZJC1YRnOLab1jxQHz1ZsjwVXdZrzjHbhSaLLc+tgsRKRVmE4LXGZuGleFb2a+MVBBM6bfiLsYO79Vj8eco0jscrt0C9lyiicZrlndMPbdWlZUf7WhRYHLikjftb7pbn5tPiO8zVsDWR8G3uY10b+M0KEZ\\\\\\\/2NaAwtx3APAlNMRyh+zXrVKNotkGLkgMPJ2Kl\\\\\\\/oekWFyDjFWgiPaMs6qf6JotOPtOEw4rNf2DHCEWdY\\\\\\\/L6cf817+T+vMppViSe2xtnynG0DLXUEPAjDmkhpLrLmhRxcmqMNL7MYM1XMu52D8ZNMQfQRjg4lrikfS71PoyVt0zIVjpFEQ65x+FgVkcQ1f0DF8+k9oH8R19WjnKMS\\\\\\\/1YNZSdn1sjUDkskRouap4Nw6c3xa+KruWpYscV\\\\\\\/CdvwmQYyYWakKISa7OFfiCzvgHM8V95Z9sIuKJ8+7f4pQ3kaydfw2mYK25dZEiCGXfiwAJsUUo4BXW2xabhHf3iVRv6+i6MEeaIqrBsX82+xGNekh+wgJFILocj7bQxAAojTh4Ytz4rxEjl7eg9Vo6wiKaSoZStjrvadwwqLemnHFKOHAumId8nzs1nxxx1t4t0DOxVXRvG5RbKEZhypgcAteOsRlLSQi3pp5+g6kndkpGCUWTNn8zOz5u+Zrq09qa4sziNOwjnK5SPfeMIto3v\\\\\\\/Ny5wLtnv88S2lXb+pe0trq8MgMFmwzH4ONpR004qj0PAsMJZ60VHxew2udaVR5grvGebvyeHEmlrsQsj87x7z0v21Le9ANGPhH5J9UZ3AjxObLG3mSjNpJeflf6dwU3gKe3zNaGfdzOHPyULHFKF7JNJMhdteAJKTdFr3XjS1ZPr30e28mEXd2xFCEj0L6vWBEoepPJSiSYMogxfuXW7WsQq7Eya6cAXbOyIS0O2P0MB2Utbp3iyAsdotr0aub4pGr0GMLrWkpGAf8JzqdKFQUYK5hoF5lIgDGxV20abTuBae6d4+gCNnm0JPUZivZX2XQ+G2gwGHJzi8om+jjVyCv1ycCVisAWm9JdgMnA7+FNp1oWk0F7LRbv3faovdhYyufzSD+HJBGptaSAhGBF1ZXq2NdwqEPt9GiAGJZ0f1GKEiJuWUr+kv6xvKe4DsqlTDNLk5JG\\\\\\\/YOIBKiDwuFFXbQIaPc1TNjwsAsmd3JUjn0fZy6r5Os5f96jUq8d7uKpGD0JjtgE5Vn4LJ0oE7x+H1w01Xpg00gOtNpn2bcDQCJ6W7Ny0zK+dWWRgXyzPbUWuRPia+RjJvv+a4Dn6KmF6DQnOVSaIFME1lPo3A3e3Lz4A4ZPDjowcLGQRL\\\\\\\/MGZ1dR6EvyTUgGUyDDjua1H8RKKH+Zh6tk\\\\\\\/Tuof803xR\\\\\\\/pwdIkmkfaWv3rJueBJGBJ7w43zcb0lXW\\\\\\\/7C4EO8YzrRjZNzOhJJ3Ogg30uaXSV+bQ4s2Ov+7MAaT97iteKCvZlptCnGmVVbp3\\\\\\\/a3CN6wEUpvUgTztG8JR10ioon0plNUafeAJVofO2J9JrjkY7UJoVeBS3IaNMPPoOYB9aGau5k+M\\\\\\\/vCBN3kNRKe6HFqnoU6mo9cU6r9TxHaJlCaoAfquQWFRztxdhiqNv0aGh4HvJI73pyFx67eTBTGA9l4vjWqfXz0LhAxPWTmxRL7nJjdPm7v\\\\\\\/+6gNhTRmS\\\\\\\/6uKsuaJlFtHnqKbAK4e1p8OMqIsY1yjyMHzqW2uzh\\\\\\\/39VKoGjUzSwA0KUVpPbM1QSo5dREE0GYHckEBn15GM9DwQXU4dLh3dBhqP5k72kOu5DocoY0wx3CqWHj1vi9yE\\\\\\\/q1JxAgvAZTC9ahMbMuIZ8bz9TGLVyJpox7F+GkZBj2cwTJWauxT\\\\\\\/BqolugZqnIwLS1o+U7JV3LmpZlXNbR+pv1+p3df6a8AEuigsu9yDbJOzcwtlHL1f3u3v2OL4VlRhBDPuP6m5vKf676J0KKbjbeTptKYQ69WZNcwIMn1Z5Ol4zWkvZBazGRp2UMVq1Eq7mSo2V3u3RkIqRfgfjPVsH7euiMOxYi\\\\\\\/jDIHwgObcteBXTAKaaw4dgDKPU3MOMW8O0cO0bAW1BY\\\\\\\/8m01S3fQOPARI7KmOH912ASfSMe8pp\\\\\\\/IXZbuczDJgJ1T77Zd7qDJTh9ydrWvH2rhEit\\\\\\\/+WS1iL4XJdXgLutfS4DchqQ5AN6nag2ULyRkd16oObfcud8AdEUBJrmXGsnVR3tEXsSzCUZ7Nv6YbTqERV2FOYpTZl\\\\\\\/IRvmw15i8brOitM4WgglY9AUsv3pM6vb83jGU2L2AOZXEX\\\\\\\/vXZfD7fjQXk0IxvUe4R0CG4evqWmzFGYJthHSjUfQMQScPaeiW9+TE2gkqhjQJjWo0Wk9rJauJl1BISbwa4oDUdhwoZhjsNIZpF+CXm5tTCg+gkTE6xdnzgUR4wAO9o4E+V097NKASRdKaOt\\\\\\\/qifXUDxT8DsZxWDJJ\\\\\\\/R6EYxpiVOBVQbMl4Z2G6xBkNCG4EWXIpfmW3bZsG8ZnrEVlIqVeowpIpQAtgJaxHj\\\\\\\/vr51OomEavlczumtMIppOaYF5hMn3DG35gQOlXaNKZNIVYa03bk5Vcv40tUOsB1tZAcXexrk9xta9BMrLJx6c13SRivOve3Xh5RX2+SV7TDzaYlpxxKn7lnbxv+3YiRIb1hDM\\\\\\\/F\\\\\\\/97Oa8ZNP2JEcW8foW5WYAvg0zrvIqsDzo4yDuZT323f+t\\\\\\\/3ehUPhYZBuMG51mWadi5NI4bqu37vB14qh1U32vIJ7t\\\\\\\/Zc46WG3U2yaE5G\\\\\\\/0n+l3x083QBl2cELN2myx+IKHKqjNn5+q4Mpmq71B6Z339taKdqS46YU7azHia1xblsx76aP1wfMl3PmuTjJEZKtB1+iO5lsFioKPuH\\\\\\\/fyDAHMKVrRf7t3la9y4BgY9gvNk042UloiVf\\\\\\\/pSesEVq3WowKbKYl09amYQbfjCCMIdzcv+ayU2RiA0sXlyolk+qoRtUocTld3ydzViUgJMNwUBh4bUZ1wextX3FBNPol9AfwyIWWzlnTUYC+\\\\\\\/5IrQsNzLUcSjNyHzdMmvD\\\\\\\/1S5zXAP\\\\\\\/GR02uM1eI2XXGDzqy1Ub7IIBTmc72Oq4VddXtEL4oCMH\\\\\\\/vYWJRYjU1xcFU\\\\\\\/253x\\\\\\\/Bw5goOA512sz9is3jEWPdGNiP9vAh3RbrTLY12fkK6wHq\\\\\\\/TpSZe7xG+GY3xZAwVAYqftUeCclP6XhQC+G2ZfuD0JFYZfLnjbj2vlfgFAAqsdrButYt\\\\\\\/KTp7jr5l0A5ppfbnXfJb88Nogqq\\\\\\\/ChYXREdW6dnJv2AyAABKsx3VAzyDgYoCzKoQWhR3Igb8wsHfogpoCl7UvfXwukP0cFZAMuxK05uYL29v5Y\\\\\\\/SUtjyjxPY0Vo1a7FDcuQ2QIoLK\\\\\\\/jxsM8MaXZs4qQpw1IFyIPqN3WZhx5yXwl8j5cpM8XCPeDDaM3wnaSyiyZ13LJzMNu36srsEVflCfvF7ihj4vqHjdTfGhuRs8pWe936Pmb8vn0hfSZ1ObrBQa2YQBI8jDWAjXZ4wps26XaeNRL4wXkJAJw0MAmBxQvaf+NdVW08R9xqHDtSuYZEJlgDbkur9e\\\\\\\/lX6ApWYQVxzGSDY5r\\\\\\\/gto9pnZiOZ6xPfPH1JmOFBkQoYH21BI5VpGc3Pv9svw9tejO6dU3VOY8fCCXxsyYrU2xBqcL2FAOlQ\\\\\\\/FsglBMV6WpxSScG1zlg3CGLbFjmIJ\\\\\\\/ibAhYTlexv82tCNdrSvGJVFw9FzcO51jIYdBVh8P4vIS80BeGV8LRZcF2EZqA1\\\\\\\/ARTMaVXoc51a5013zh9LtXKuYD5cGDkR0FwYhzUT\\\\\\\/ByD0D8kFvJTsZJrfn5KxiXXTb2mEZq7YVEEb\\\\\\\/ebn7YMOAuZF1yEsDaMUc4EBFNpcjHQdhK1YcJimpejgil7y4z7o9UCxMg6Uop0elXIpcHEB7c3z0tZFc4aakSWZDMlbbvtpAA0fZBarMkkEDMuyTdDSYSB6sFsn1siGqw1kiqi7IbtAdfdAzbMB4qgSJa4L57yoTSbhABHmeIUWKS7oDpySyv8XuwB5RrTjiBoHzAd79wrG\\\\\\\/fLWDYXAp3zI1G8OlncP4lzeWE6uDKUDn6vX+POalcU2vyP0yz\\\\\\\/wKmIFUv6bwlFuDyYip\\\\\\\/g7nIIbGwcrLKmrKbuwpg6b7DCk\\\\\\\/ISRJCF+oMSWGsIjQ13johc3+j6OPcRlmiA5QtcOS4NzGOPixWBmBTkIcbd43EY54EnMkqvvKUUQGDnUJlbg6\\\\\\\/AJZx8M3nQyBYKs72Qpo72GUEELrihdozgztJ9\\\\\\\/fPlP6uyTwiJopKs7Iut+HiFQhx0b9oiRPJ9weefNw\\\\\\\/pVRvIDRgtetyzZXVVcABAr7O3AH2NKm0TX7ZnSFSC4bLwaNgNadldtX\\\\\\\/xCYyx93m0FxDxWuyj6waifWbDtGIVw0\\\\\\\/mVZ1L6EbwfhCsvXB6LuMOc9XoSON5vjStb9YfRaNPSThOhcPOjRjZQ9aifRrIb1zhnXW2u0uwo6n\\\\\\\/vmQ7eYISIlMCVJOANdfKDyvidYwBwR63PCSE\\\\\\\/00I4MzxjXHwAOuuR8knHK+gltjbdjQGPkAC5llvsb\\\\\\\/152NxFT1VgpS4c8odSy4mOCKK8YnaRM3t60x40027zdNSPD9XtBlw7d2MhdMS1souWWTtCLRsFTrMGOvWwo+RiSEFtuINUYPLY1BdhVZ5JoVjaI5CYDs9JH2k24syHvfjaCK3u1omj\\\\\\\/Az9NQTD96mxmDQKBKhBxo1XbMVjH4LozwpXYh9cT31alDJ0O5pt59LiWtt62pmYVSOcWEiTXTRwq6ZcqZbQc9B0UqLvaTsUXTy1QYk7DM\\\\\\\/fnJXlzb7CSdctZcTwgTB+n5UcGggOICL1FIWrE6hdfGQv5Javr06ZEPPTsTzrcqC9Q6FIY2IV9Owvv1ILgXmYFFo0XZVjBdPpRx+BHAXfV8xgHfwjJI\\\\\\\/aUHqxQtRGEtmZGR0S35S++L2im8LtUx6yq9gCAVUfS2Lx78lo914pevRjx5FADOzlIlWYTD5k1ASuDj\\\\\\\/h11chOTDYIdE1tri+Iocmb4Lq9k2v08zIemtPtCGFyDM2YYDJEv0E2rnZ3HlMfReuq5bzOFgKNd4PxU0rAoLbDNUHAP4ufKDL3XeAAA5JgHe0e3xflg9ISpa3zHYtpsyyBgY6x5tAp5wZ3gJfXyKlBz0bRkpqcxrTQBOage2T9w+87mzXVueWy22Kh7LNwU1JwD5H7qD2EPsGySyICmDAZRRXCj+r+GuVMxrGYY66SHPY8U6xXtHpFMQruHEKUhP4kRwoYVSqHZ4InIbGqeZI5a7RkhCN02CUFikFpXb14kYOxLfDs8ZeUVgGlToMWSKl4NuWM018jPhLKXTPqGvXMNAbbc2qMiDVr33xi51N1wjVXM7dxCn5o2YmEQFX+hd9B8yLlwyZNcfrg900srmRCEP9l1EAGDVvN22Vvpc\\\\\\\/0bHl9tRRaWU1dfCZ\\\\\\\/5RY40\\\\\\\/mvfRMIou+UdWKSVglaj2WO322TUsJMhYofZh5o6eQ70pi8T4FY2WQ39JvJORC6BgYYpeK3V\\\\\\\/TzXDckkrB7ei0vABIocRfzGWO4zU3C9eag\\\\\\\/7kvlnd5ZMVGzdGnRhNdbxH+KnZsJvInMj2qxGZiqAJHIZ8OkffkGydi3EysU4nNtHrpSwU5ac9HeyH79\\\\\\\/d5Z6Uq01ISbCjhCXHrXpxJDKOy\\\\\\\/YJsJhEoonO\\\\\\\/\\\\\\\/hsMv+tWdj6S9c\\\\\\\/CDaWIeAac557HwsilU52z\\\\\\\/Y3w2Tc1YRVhjS8\\\\\\\/GQG+k8JPed8KtqgND7+\\\\\\\/8QE0QnFFzmTIVm3ObZ\\\\\\\/c9Qxjon2lJ2bnnhLJ0Z5zj34Fa8QbkdOPOTAZe7lszIbOwAVGpsmmzUXNldw6jLuTFp8IKk9LKc3Y4gHpsGcMjZyML6BM06oBf+IMvjGV2xdrmSLzpnb9lv5Qgn9wDFCboUhHm63+jxiI45R5YS+DL4qsZRoHOQK7unTOaNQ9B3\\\\\\\/qCGCJCZ+NZKk8UrraHQjN3609DoWMh22zOttgUal\\\\\\\/7k3r66cdIxWkzATICbrJSZC6TyhwN3c4LCErY+rE0NrniVUn1gkpafXhhq8NGbURdDxOMcx2ESkyU7uGe8hojmQtPZvvW7HTh1znrzGzVtqg5VTXKwxYufD5Qsl4Z0Ni0F9JJGhA7c3yv8EeYhAq2ve0kNlQJMXdO7Eqd\\\\\\\/d+bI31EBWdBBC6ZIPrUX8nUIKET0dV\\\\\\\/ALsiPRJ3jki+FuU4e1MHFvxT1VA4WWTyr4Yl3cEqPLCfuyrIZHC973kAlX0Dx94S7e9+VMyMWOxanNQ8TMzOQPV9cwrsc39EpHMZEuNYhru\\\\\\\/qJA31\\\\\\\/3dwRSvv8vcFNCnYb0clQHqHCb8pcfE4JuXpdyCg0uJvHBb0\\\\\\\/GXsBBKqd6\\\\\\\/i3W5Eeil0HlfoqZLrj1HrwrllATscbkk8v7BIhcSdY5GNToYVUxaRF3lNpykdadyaOkL84nSsbHtY8pT9nhRqw2ezksOFJ\\\\\\\/HRJIRV2ke9r682NUleCya1y9kR3LRB\\\\\\\/az4BjnzAKFm1bsU0KGPp00scRLlL+fwLbDYqXSnjO4ilg+wWXlEKUiZyoC3AO9aT3iIshDmhfu65hELNXYMKp6L11o2LdHK4BZGOUh4UXQwTBtHo1d0wsJhNmgD9g4XG960gHjop2Smqe0zeGJQSFp2KKn+EFfWhLCn4eKx670tKb5juznoXY+nldts4i1eec9322qJy1My9xUQiFSJPk4VywGVO3j0MoJzZ5XYYxgEb+\\\\\\\/O22hpW8pjGWun4dQtvJ6s2GFW65ptWPG5esIXUBO1twE3CamnMnq+ZZTlFBmX87FktA8yUXmejRHmEq8VC8q2QSGYlGHU+GMaiAK5JOQixZKKcUxugGDtYbiObrSqIJ8mtTU3sGpPUBmLjn\\\\\\\/554t7+epWwhCTBlhn7n3Xs7tR4QFBuyjuu\\\\\\\/llD2qmhr\\\\\\\/dUFKp5Vt+dGi9jytZZfgZwxTdmfOZZYme\\\\\\\/MoDp0jOAm6j6bAai51+kJCfSL5t4UkUGlKweFMpaKaAO0ebL+xsPyqSyatkZkk3wE7jHQS3qrBZ3sJEZbAbRSMgo39\\\\\\\/BNO0AX8v5hzILx8tDFEiPKPoBWA1mWQB2xj3tQHNcgYe+14IKWkaDZmqn465QRo5QRNrwGyipvarYLR3X2nlMTCMX99O9sxHHzGIV9xslO4xRgPR0tBBLNWZsxJmLGO6LzxLU4ImxajXtl1UCEGi\\\\\\\/TbdlD\\\\\\\/xy7Q6fhTC95WuVvnQHdXCBMpU8kJ9IRcbyp77P76602vzRJLJ1zz8tCZApTPzQYR5VEhmMRcPMZ5L6ovIYpU7zNpgihYyoi32gB\\\\\\\/G4s5jKhKwtTDGXAvOaIYNVTmOcLE9Feeywc7jqxJgZ80IyEN0fWHhVndVj4ILWdU0W4C810k5J6fU+NNdMMR4dd9V\\\\\\\/PEQ4Z3NI0+RroqJwhc0TUFoRMKJJy5AK9j\\\\\\\/3FGXxmeVo5G0Zn7MSEMNKb3PiBm8tV+mmk5tm3Fmzm2joNvT\\\\\\\/Hrjv6nCXsBeHEQpnwtdehpKJArb6qCAze+0kcOSvRri90jM0U4cKzt\\\\\\\/w16Vomng+bClbLHffhpuOwyObsi2vSsQNUbAIumn\\\\\\\/7q1KBYbJUMl\\\\\\\/2X20100L9nfqT9mlprXsctIEYAwxtF0PvUToaP\\\\\\\/PzXn8Z8\\\\\\\/bpOpJcc1EdumaEaKa4\\\\\\\/vk9lsEj5Qx72ULmLePfJTvgaGrgy94zUvqJycIGtMNzoUyGEdnFsJSHL7gdqmivdBkAfbK5eaQF9uFPk5f\\\\\\\/lDomNhZXyyNxxc3sLaP5EpD\\\\\\\/BHqfXJ66uul11p9AOO651fYJ2fMQfunD+83CBCihBnv1WOFfZJSllvJFpLOY3yFJUGk8SNqGBKhBxaQql9exfV01hJgw4Vcco4Gh5tjv6+cUUop96rPJ3Y6L+Vhc6SlJLhC0YJyK2KqNMZ7sbjCVX7Rqz1wlhrCGDEHrwt7ox5ijHfCgfv6Y7NGAJwLw+F0QPLcIJ24XPvHamlNz5lwFvcAhvJ5DxX99O8W3zjmBhUm\\\\\\\/E10HQcKKju4Ej5m4pEh0cIiM94EEgzFXRtPcgxGhIH\\\\\\\/+Z6bnr0kAedjwRi8eLwsselWLJ36im3oiMYiJKegiZYGUmthd\\\\\\\/WkAXEOVOwteHrF3dZv8OJLc4tBdvMenPXQSYQFPPc4DThl\\\\\\\/PntXpGmiJyZgc+nqOmQ+0PnLSStfHCInfg65nDut6TkAc63aB8n6yDMYwEFmjyBkJndltktDKNTQwZejYo+sZZFQH\\\\\\\/6AoBm1WYgPxaQsNagfc7yK8F7iqKkEyMJoxUdwDx9sHF\\\\\\\/VTPX6dGhKodNegXpmkKcgKr7pzffe\\\\\\\/AkZQetLu8Ao4bbY8S2RPC9k3ruzXh5qvd6spGyMeQBYVyKCktwf0f7KqKz\\\\\\\/nZCxa8gGJ0Kp1cmWxrG6QsZGkCrsVxRJymFRg6TI3KYJfkxdKV4j2mMNYb4PnHyL8tIz88tjXuE0Bwqa2gbmTC1S3Tw\\\\\\\/cIyNly+F0XLy+BFrlLnEdkjM5ImfAUSHN9wSUVAZZ09OVNimxYb72\\\\\\\/5BLpM0XgDaapnLJQsp2ISM6aNbO8QQ3Z\\\\\\\/n2Rk5cNOzNpux7ob0MK+C8858rdq7F5tSMAzWFdTvtjmzqkfJ6QTolvkFdai5wUpHVwEW7ZAK2fG2xQYa5tdrUkTqqZVMcLxwpKbu6JtoppXSHWxJERBG9anpfNNK38OAZQKy9ZfFaCgPp\\\\\\\/Vw8RStEoleL4kEs+CU8N7NgRaQYcc986GlMxW\\\\\\\/q8PqTwT4KBws4kUHZa3Dsy1r7gSBQy1g7c+L3wVTSHnoDJyKCV1plqy1S7QTR34qgU65JJvcvP2aQMcr3Ch6N2ws1vEB2+lb4MbDscRF1ZVqVmM1vOPIR\\\\\\\/QEfR0ko56lru5awq1Yyz6H0kxGjZizz2+2foEiKYLm1ZiAkATUpjNrUTLF9D8XX9fo5DzwJ9bpEVRHL0\\\\\\\/kYCXRqSZLkCutbfI3m9gHWRCMGTBSWwqfDht8bJ2lYbLztY+Abuuiz3eMoI1emepbDt3gPcpQAmiQqc01tP3l3MPnhqa+e4D4\\\\\\\/RFkbwioxUkshioey4G2Yl8I4elhYe3b7E3r82ULr5q9OTVz+hPPWOchXqiZUC8\\\\\\\/FXdChuJH2Zms+ZdpfRBjUtHlVcg3YfpNTU82MdvKXsYeU7+J0Uy8TpcIj01QViqZQ6CQooEJ84fsr2rNNgOV0ZnOYTOtM9fYPcVU2rEecxPtljgLOVm6t6XWzlmTORrQbOeEV2g56m3oG1YlVzTVsKkPlnrju3G8c9A+HRRhHxiuwFvvtr9SBRjfk6QitEPy7Ma+Emis4ruTV6qsua\\\\\\\/VClY5XYQrjetpuc5oYOFLX+BPEuy5MjAr0SihxHyg09nse7RkEKjVkRpOndnmbeXdgEa6tDsakQmT7okfZoOTJfzNaPFeYTbINm1p77nv44mJOfKLnhvvdXqNp+LygKVYJvs6l03+K2i95ehe6yOMALmp2yovr1GjYcAM0LQB1qtMU9CXn2TV7X7qDBRHldmPd1WMLiwKH5gD4uc4Q3T7Vpssz3DbqQ4FL0YKXb\\\\\\\/yeZXGqQ8IuD0VR2M1bUyJp00P8PtSBH4vAGRVcAXMJzJfaezuaFZXdi0\\\\\\\/GdyehFZ+0URU+68HwYT4eGYIZRKULGE8m4m4iq6V8mB+JoWw9Ju40lpYCpS87KsnY9henZ8+9DU5R5hCfs8jDt+EYlM07C3z2MAOVEdBmFZ+7yywGnHzJhC6Ndig092tfNM1eHf5TDXynIjUYNqtd5BAIzSVlLtfzYHS6bawojayILqMM6jDoFnXlUWTCJoRsWKwVG5CzffTJyeMXnxcnzvgnIDzsA3pdC0x+qkGvBfMNlvNp2RmFKN3srMirx++GdaWV3tV5D9NauPdR8K1W3k+N9vS3TS5irZ9MR5vgTmeOK2llkEE3M\\\\\\\/tYpt393zyRHUrgsVU+9R4VZ8I6KMyitB\\\\\\\/2g7AYKj8CIpaIutDMWbDkDGgHnkycqGILjjFgZXRzSXDh4nh5akMHWhu+D5vpYObWnuCdfdnvrG3gXJy0CDt2n87i2Ddtq4B0CQ7F6s+raViA5GNQLWPdm0+AsmU1nvADZTy1TDcu9w9xvHpNgYWTpZ4eebnZBTrqAEbVL7JRiP5YeEroM9NlXNla+2euGLPg9vHRyEP9ogjz3qr4wp0lR6Hpj3SwAbRlIBKtSH1TGOk3yvKnC8LlkNWkybWBq\\\\\\\/IuszZh1Z7h0fiC0dOkHQlJlqMwE1TnmRNftM+x0CfKSfp1wvhCd2yawY6KIEbXnvY5G6JR7qRIFeui\\\\\\\/timyn\\\\\\\/o6nOFlva0VdDsNCqdUiLEtbgj46qBnXqP+mmpU+dBHQiqb+16UJLBTQqZAxCV92VT7pOl2Dr3FS1pjgyS+RNuFp7yJDRj3aJ51X5UecocVdNrRel2oOLbokoxVNqRdq6YsBvcDRz02BvcHJI69yCGzLDT0T0g7iYt33KbU\\\\\\\/+wp8OwQ3PK\\\\\\\/bSWXJt5ZPnzQhkK3KZ\\\\\\\/bBvapvvaJ876Q2nrhJLp7p5yEOhL\\\\\\\/N7R3MpKeTM+2XULAlp+8cxExqIq6YEwC6nNp2HLaQozcaiwYmz+yeSmR4jbVC3\\\\\\\/e3ghVz+RY8ZMBTHiUQ+7gXj9gU\\\\\\\/\\\\\\\/NcS5dKWE1NIK0r4iqjK+n6kfdPgLFRjxvoAeMoO8FWReeXxGwKXREU1LuC436xXLjWwfW+2O15m890zAHZ9ky4sn37wsIQPczgwT6vB6Jk+jgTYK\\\\\\\/nNRinLp43ICCb8bHQEjo5NL5kZZw+st8Xd7p3HLghoG5aRBKV8Uo27acno3hU66ijvv9CIT9LFs+T3vTgGGDKghcxiIIbxyjmvhdtAJ26PUL8cVS1Q3MbnR1VK4xK\\\\\\\/fFjp76UhoQ3P\\\\\\\/unFajNvnnCm5D9H7bwJs5XOAiLkoncdfhVtHE88nMNqtz951Vdh8MXVTr5buAg7B9PbQfVATBKiNaUOdfr3P7uffq5pJmO7mMHSTD2FkmCyZ7C\\\\\\\/9ltcJ6sfG4BJc\\\\\\\/Fyuqu7gSXylolUPvmLbrt\\\\\\\/Dz+nTrZvleK5YBgQV7nlMmwulYfj27nuPnYAvkr7wmuB1bVMP6Oxo47Ea2ezGAOZhQtKlQIRn4HBeC8GNa9Ppv\\\\\\\/x7PDVQLOeLSQdZrDpR8re9ExeqErvmDW2xn1COdBfevc0+AzdImlaskG9BlmMhk3yHgcTzTAZQhedN5c4MK0rdd7IkSIo\\\\\\\/DHtEpu7hCZwJpVaCL6BydX+pI7Ls\\\\\\\/N5KTBwqsfVSVqjpf6q3c7vund6e8X54lsFUy0ooN\\\\\\\/8Niq2Qi5k427yYbdoLaQUpPoM2KEtnmEgP9+KOMx2AcjFuOSeQadSNxsmwT\\\\\\\/LJB7DTrWms\\\\\\\/GnjkbLSd4N65N0IAkyNi33nMGSW2HGu3FiVDmjUnIcJkICsi\\\\\\\/dvwHQcnWS5+bpgylQK3h\\\\\\\/6i6p+f3XyyvHnEcOqySHRW1TEJoZuqH3AvA8EwjiXPR4LPrM6KJRAH0+h4MaRpE2tKAOK5hfs2AzwyH7KFsRsTlSrL4chg6diR\\\\\\\/c9yOH+CLlsk1Gd7Q7+l0ZQ4k+u2wDhVQ0PyOnHahlqVuhFCE\\\\\\\/lJkdO+kv5SCTNMeux7NV4qPURP9JsbQYWUokh6V1y4xnFWDE4j7mRQCbAFNTytzpQz0EIJtErz\\\\\\\/E1fZ3MTmUBisA1Ug4BaKwdbigwg6EQD8g3CJVbU0QdSpHGEZRh03bb2GJszqeti\\\\\\\/HteuMS9vjtiIlUVbl11n3b\\\\\\\/m3EM54kBiBcfkTXi2mB9iQjNWKxpAb7ah6LVkiIkGAi3n7XKI3KAYt8g\\\\\\\/jqXi0tTKNlDLbWEdSv4rGsAOkFeKwBeGE2w8F2FrWPflwrY+oLK2F4FPiUgvPlY\\\\\\\/gtxC25HxYXctANBiZ7t+IWXIiWFxkajIgt8Y2n1IOIG4aeD43SnDEeTLpBRc6xdLkD2K97lYiiEMnhIn9Wn6DO8pIeSYsrMQ7YHKrD6ARcAHX8sbPYWQ+DWTlk3TuaMvObEYDkTpJl4YrRVNl7vqERgkE7NgDMyBpKBHppN4fdz7\\\\\\\/YFE0gghfl0bA353Kzc4fmYkSXMR73hFSvU5RRLVXjAW7vV440Pwr5YvEAUPPmq9PDaNPNUFk9d7R3MQj7UumeQtZaaAnB1NLolEH38CENlKXPDB\\\\\\\/MgH6dw0I3Hpb07JrYZIjXESkOOPbMuoQh36rwqNCBuq39mo2Oi4+2QZUy2SF8lUR6sMPWnbLaLrNC0t1dLhhnJv5cG7iH4vexXt4OTV38OhcwWfkFr9Cxj1uUNogdH5+Bf9Bl4S0LwcPax3fQxPFTie\\\\\\\/3bWD26NYVyjfy\\\\\\\/X39SFZi\\\\\\\/4mapi11axkxBPtU0G33Vdv3wfHEfyciQgYBL79NoLIKOVVglG2O\\\\\\\/4YSZltqpi24hoK51O2MlMvJTwzy6yNrJrgMIkQpw7kXsIlfUzXQJXdojJmYJp64UeElLRnaTFyij0XqItvmOosZDWleLaYwZFCWv0G0Y\\\\\\\/syLmOhBiAPGFB8btF9rgCs9jqJbJeOp99AIwOkMwJnKvRugIhzWpAZYvN330yfu+OmNH0eow\\\\\\\/+A2sQ8FjjCZNishUy8\\\\\\\/NUgN+HviPGBHRiPKvmUrOBTcWyhNJULrZ7axd6EELyMlahMWCBF3CZgrTUU18QIKnoyH\\\\\\\/DCnEjM7oRuDugUwo1Q7R3XAYdFD+\\\\\\\/Ew1zKajDlD6wzVel1LFY2iC9a6wOf\\\\\\\/cIw36Let9RVwoharJ6xtqkgCZKCujNukhiqCw8\\\\\\\/Se16L0N+HCdgln5lqglN8E9CgMQ2d167Xg0VkI\\\\\\\/mGObHnvQKaB5zSQgNCnhtkHnP\\\\\\\/djsK6CGRtci\\\\\\\/Voc17vBElwLGfUdvqBUh6adS5N6fAzF8pFt9xIh4hAPiuIQwicDbucXEJ7h9cL3g8hGDEGOrnZ50o+km1GgfaXDtNNAClde\\\\\\\/dPuCvBW7fR0FduzElQ9glIjN3Y4pRLL4HYtmM3BmKeSF0vTo6nCfs\\\\\\\/xOvcLI2L2VJZsWEz3bXcpl6qI4hDk4DxGS3W7Zgb2FSWH56XwywWaE3TXGtE1GRcUelOr\\\\\\\/8CMlO6WdssuWStNPBryi91dSOl\\\\\\\/vROM6bxNKM9HkOWxOI1h1XdkfKBaWWykx8hoUM6FsuG1hNq9n9ZwdhXT+QMYl1zsngS7X565h+wsOY+CzroAX4fBZNCVYy9WanZmMK0YUTPTCMpxk6Yh4+tIr+XqQDuRLqIkWu2ZDYP9BzaoE+UOGXKG1+E\\\\\\\/qB\\\\\\\/mBdjyCeKnx\\\\\\\/eq7nxBn8Uucw\\\\\\\/5DkXyxwPXo0WXO2bELe+peTODb+QeBWweHI8pKIXm+Jd1K9H+pkj+Bv+6Gl15IqUNMCOGL64b6xDUPA4RyN5uG\\\\\\\/e4lVmAa6hDBTRQavcwzJf7p6xeW1bzU1ymqN9LALTD\\\\\\\/LVI9\\\\\\\/0qVcSonJLB1+vPeY3+oO0c4fN7rM83\\\\\\\/BWfB7Q9hwDbHfnqYMRH8ZswbZVEzNdqj1KluDANK5eY4m1\\\\\\\/ulky+h8s+XoV6+Iz+Y4pvya2\\\\\\\/6NC6BRu+duEtr\\\\\\\/rHr0z+eiV4AgTiMvKQnC9whGbjxjf840dsQVTBwQkW8+QRF3n58r1Re5jbx\\\\\\\/qTgpy6ItELbCl6+FsEmd\\\\\\\/zylcR7Eli7urOenqkFsCpkcrq8gF1i1OYqpnHVokROERjNm0zYZ257JMyFLkEoisxnEOkIFZfVc0r241MEpVQn++VFTFwUk4Nq+BHoI5oBgmab7axr2Lve3cPpocRROPp\\\\\\\/EHfDjqzeX\\\\\\\/nc41ZFD6FL+hNgUw8bcgSGtDa3bFpTOH5my4mYPAS9XN4UlF1nISUUAUHrfH9CAriztQApLKOJYq2+Jup0chispwlhUOQpM95L7HPkJKd5BbWCA82nAxMQI73Fns2KqBBi\\\\\\\/2xJTWsQkN35oE4O7s9Lom85yGcjpyo\\\\\\\/6ptPjhryKgckgvCTvONUvR3aL9tKit0vw67o64KXBNou\\\\\\\/udrX4mR8nyOaAzce9o8otdJ49b+U2wcDk2trTXhlctyUTLPFwNuNCfo5i4K4C2zVWoukjfe58H2z3E+8RKCsQVLv2\\\\\\\/YfcSzsUAoagiFG4moZtCR20foveiLq9CFseGQIHj5pDMgcLtoHKLp\\\\\\\/ISH+S7sgupV0Ubte\\\\\\\/VnFHqf53\\\\\\\/VwsTkmUJO1lg9sJZmLbYRVvedHpUbZB6WmJHF\\\\\\\/JAG\\\\\\\/XhqeuLG37DP15tXBuI0THIQpgkkzYHI92Q2DvzTosiOpJWTH4jps+ePFDHQN7KEEblJkl0r0CwLjembAggvLRtxf+rUUcDErlbbKVL3Kb97EA5mxauOqcQMO8tsikJakSdq3dvCl8+a5gwSDn75GYov2znM7MjrzWUWXsgF9wEmd4ZSKFbeOh6DQLAKjrg0vsFB7peNPlF0n6s7FLIFYQ78myD8a4ODnNttRLrqL41de4R3T1IjzLKVTEUpTB07cD5X\\\\\\\/5lt5wi9SkPyIW9DZ512rTJr8mXzWwTPe4\\\\\\\/DE8YALs4wEBSKkSUtnyFcHHEXOeYejUrO\\\\\\\/Z2ORd+vMN0gvMR++kTlsEIJb1oauGTZFA54J2ZEWyjse42fhPjW7PrRMQEH\\\\\\\/+fC0N7MKLkE4iurPbuBnkYRb2AxoDQwqab8ZYg+DyQuwp2MWXgdEgKYKtlKIwDxJJlzpjzrHWpijr7WQjfiMgPAwhQ9aEGasFHwBVmTMqGNS1ObtZWinT3g8iMHS7sa0GVLxDm5nlfmsyZNjcR\\\\\\\/1WbfKeQPnkx\\\\\\\/bwT+PDqfHCrr3JLHrq+ZlF4NXBAH1z2nRSjyyRLSawkkIVAlCQUDS\\\\\\\/\\\\\\\/ktYGbZTngxgs02a5Y4TwOFEd\\\\\\\/X1bR2Ma2KXNnusaDLm9zzZQpqsN4eXxchIC5F2Qv3\\\\\\\/X8AaPPYMH4bP66IW6kAvfVFaBBkzaW7UxUtAXETaZIsAKR3dgTmkg0ReDg4S2TO4HDqP9hlQUd2bhInRIaEqkuvaV46uRwyBcQBEANVWqoNxK4C7njaQc6rz58wd\\\\\\\/Xp9AfqerJkbslAIx8a5l+dNzEDrAw\\\\\\\/M4SEMgo447nm8J\\\\\\\/oHQ++7A8+ZZNlTvmwGqA3iTDNoZK4JJnpRZVIR7RfZtOZb5Hct8EdZTb7umDr3qRPAxnxbNC2No3NaPnGeLmqA9dCa2Up5Tu4ox\\\\\\\/r3WhphahAhq7F7mVoPt044VX7vHWhrDj1\\\\\\\/Up\\\\\\\/Fg+FXvM\\\\\\\/OHX+joPxDVV3s4NYGw52T8LbfVq\\\\\\\/GPVlv\\\\\\\/Lbh44sMlOmm0jS1oYsRVmDTr\\\\\\\/\\\\\\\/yPGiKvIv+G2OZAr9ieHhZ86rpmPbdFHEfoCCiZZnWCAGb0JwujxTda3WqVCYFXh0+5G2TCoiExUsi5SIUTWJNCS3L1P+JDFJZc4PsSXKHoBv1ZheZAs2eqzAUG0C7aMRVHFK3Ofe+inXZqHBkLyTWV3QOrefPwSl0p03l7kxeR0n04cPPfCTtf6dZDb1LDOgLY+SZcPaqKDLTHAfGlXzoP5fhSRVOvp8wAioHRC14Q03y1szddESc24gpK+mnkF8LkKIAX2jg2lQZVWtroK1uFLfv3GQfWev+zc3mOw2enrLHRiCiRuI5Z4+C7Xk7\\\\\\\/88oX2scxO\\\\\\\/4U7za2fg58raMdpE2+fviT6ZT3bNANKrFFm2f7AsbB\\\\\\\/xFsWnUVb5t1mGFyW88jkCgT+42smsD+01kiSzU3Pof7RHdBn2MKoR9Ayms1pp5YYK6x7Kq70FoWurFmdR0SOEwCsC8n8nX1lO9Bms8rMV3gXZk70So7o\\\\\\\/u1tdbdHiEQDzw\\\\\\\/dPBbfooB7ljj3XJEc4S7mPw8kWiBlJ1jUKTx8b85fDTiB6wIn8Jd9eLlAkBYe+V6VQ8aHlg5rRoAHmORRYMRICWKfDPCOAxmmi\\\\\\\/s1u837rmYmk5afwWow3jUOmZx14oVncpQWol47YthgAT6bECSggrwRN+XjA00tT0JxFHYh6S56eE3RV32icuwt9R5+ECS+sQtCPBHvPra+wf1\\\\\\\/93BJJ88SPKZNJ4xFt2mHgwZn4bWkuKIJHkMBmZ7vJzXLkTcevX+sGzO4L1kbp1LSNbeWHb5BVSySujL7voaCLceRw2ausrHnrj24RZOyvjyAHjWyleak8pPTpmJPnTkgqt6SRiFsdO0Frq4mRr4TVdb2zkQzx7eZbFhMCr\\\\\\\/gqdbUjwIKxz3ZLmhf9ck+k9\\\\\\\/XjN5Cl5rSwC1SX7Kko\\\\\\\/urFmY\\\\\\\/lvYbzS0TiA1\\\\\\\/TMXiXpzod2O70NXc9tycmTzIPIskjSt0oC62aJ04N9jWACjv0u\\\\\\\/Vgp8NHVdSVFkvsk+R2rsfYp4ThciDXrpxTX4gcHmrts\\\\\\\/Wo3MsgXJ7oEqHkuQolpuJYP\\\\\\\/ije6xcFx5Fo22WmlPJvnUS5UhAjnTI6ly6rykpbtYyL+sF26xUcmv4Yft0L3S3BPrtgD5tGHbshJEI8rv+zT2YqclFRpRSBmIseAhOXqoBZUkvRDJ7iWAlk+GSaqhM0VSfUN+3R6reS7p1U\\\\\\\/kEkkSFT6wsq4cAPrxrSOxEAhbTxUtIRUGo1XmKlj29imZuom9nMJBu3sElT\\\\\\\/cugh54mifnh5Ky0YmvBDo4kJixMVupWZYXQCiYQe+D9LTEBzyHesXcBzsxa7r\\\\\\\/y4XC8r4\\\\\\\/IC5oq7RaWF19wP93KElzw\\\\\\\/RWkCA7U\\\\\\\/qFCFFDzWYPhPSBHara+LohECBk\\\\\\\/QpEKypYuJNNg8yNqebnrUgGZGcCj118P7zVfqHFeF\\\\\\\/8pfZlrXoiUFZDG+NNUmtNee3Wwx1wagBwCgHe2k+m94VptiqS6zWaVMBO3lZ8rlINLeTQp\\\\\\\/m2a0W18EVlEywXwmKELj6DgKkp5mdT8s8zl4fewviK9sXz4cq4h\\\\\\\/XhHIXof0L5FTEP80aXLPqO0WxjafzY4TVMsr7lClGC+xeI6MdaF1jFtti9ynuiBgO5yq72GCzzneD7PI1MOrRJMUw5Iz+ahHSnx3hSUBtvRhNufqZwv3oo6VUtWUPm2ANkNI03ABpMTdsOnYVaz\\\\\\\/LbeLmmgjGBqshHacg\\\\\\\/Axd1vxoH\\\\\\\/oDhKfKfzBHZoJjmzKyR6aU4A+bHxsCsy\\\\\\\/eMw0l4\\\\\\\/ZXFWy+6AMRoTkWc3GS26is1wVp85jHV8QSzsJjQIM1WRYm3lm\\\\\\\/GgdaRRPyAd4x5IUNrtRRu4vxsNenVZNd4Wlan54hJ9jz7GGSXexy1SgP4ycTuA\\\\\\\/QQ1lSqhHnjvzSVsBpinS8UCgWUKAaXcKthWarqAKshjeqikMJBQ\\\\\\\/7WmihU5yjW25J9oOu1n8Pg4pOQnHRNkxuADLWlFnUqR+o8pDaL3wU7e2j8l47STdWXq3tjRYFz+WJalXxQ0+mo85vHY7qT\\\\\\\/Dfa8EMj8wDzLM7V1Pjd9K\\\\\\\/Us1fG\\\\\\\/Em4OiZht5F6SwGX9p1nH5T5Ya1SKWQDzXQkdJ8QRJtlzpK+RpeSNAsHe8H6QP1M2YAAmf\\\\\\\/FYOq1WnVRvWbpu7j1q+D1++3K6fLf1QitmK9Hj59hErtdRLhzlNZ7\\\\\\\/cYJ6povTIei5VKsMpQFl5\\\\\\\/1P6tyDOgaarBqLpb5r0NMJK7\\\\\\\/+b2cMuuxr2m+iDcwOjrQfg4VUZkvddSQvAuYrYPyd6\\\\\\\/A+0yISfY3GOxyqWHsZfzdTqK\\\\\\\/C1\\\\\\\/EbWmqPM8dsomhPRY0Z8Jw1TnMOBP38tQ4TUBrs\\\\\\\/kSYYrE8nbnNNawkreOFasDyhdh3hpB8lBOu+jEvnPlBjuRO9HNdwSRh1OIupkgOsnMdu1B1ZeXaqHNX5poLiI6m4T7finhxyoRj4DEgg4LsBHiRATn8CbsLiWAB\\\\\\\/H1AoQaQmQKo5g1yCcMk4nKCNzcwiTiqj7UP+3C8OAVLjnyDQCCTNi+nYdPPaEA\\\\\\\/Wd1GDIAbMF6hsOYvLrqAxvPjxB\\\\\\\/fMJz\\\\\\\/VzYyJluYQYwYvFJPuU6o9Treq9hjl0Z83eVxrtoHQXjpRW6\\\\\\\/L8yl0\\\\\\\/4UfVKz0bCi8JZh48laj9DJ4eaJKEzEiyVgmwr90xV9ujRO+Pp9MtNHrNn2huaq0e7Dmtq923RYoYP2EuzAnsaVRUCRkHrHPku3aUMecc5v08BpgiESwjyPjal+UbfsTzKU\\\\\\\/FVAl7SX6oYjF367loL8lQ1YXazaWjNDLxWlAuavWvoaswqkLaKloMsuEQtHnS\\\\\\\/I1AIpmmbb\\\\\\\/Pd4WLQr6M9NdowqR3tmFwGEY+N9vFhVtZDD8WCrVWuuMOQWFoJSnE0UoURThOQ\\\\\\\/2R+aCvE94lv3sEfUxNa54i5b7V0VgmHbCcldZB8gOpCG7LNGWAcW0MuO5OoeTt6HBhua7rr4mi6AKt\\\\\\\/gnQpsTrEpEN\\\\\\\/sCi4vmXhoAP2KrRmIHiPTthvyl8LTKAHeTnGf7k1teFxDX\\\\\\\/qly9pQC1xnmFhrekCDB6ZQu9qZkwrtwwP\\\\\\\/Ksi2dnOsnx89GOVB2HlhBTg4bzgAI9cCjHzumybKXExQkCE2RTrGmYMF6WCRg5uU9cxHBS5pZRLv5WgbTEjRA07aTYQX9vbX8H4suKNXMzxl74UJovidmXZecFnu3Ix8aA4Ohw4jnOxfiUBUzEiOPXBJ2eMgNF6c5plAgTpT94DNU29iZOG8qlpR0Q\\\\\\\/+AteKvmHAt2CyeA5s07gRUFINxHzWlz3droxv+60dfVGDoQKioP8wA2jZ6AK7Md1p0MJhum7+IoWt8IwG4LQUbGlzxfzUI0lDjCN8pqeKkYdhRh6h2Cl+iKLS0Jzt+T9swnqFDv59GiBpGXUbXlfA6+QRdwPnxxOc6Lb6XkLp1nlCXJmWGDd10BmqdwVUF+yGPs8toI2KpFPrrz0VdmPXgb\\\\\\\/TaFJDFHbUHVj+GC4YPSUpRs+G8wXgDRU5P+2A8pNOXA0\\\\\\\/+If6Jl\\\\\\\/RFrI+l8Rbbk9eDGS1tEtroE64KtYOUQ4iJZM+TNjfYJhab9dmHzVXMGbIKoR1nMJ0W25nlGIXPYvtAdMcnp4K1+aSWbl\\\\\\\/NjYnKX1\\\\\\\/PqbsXu4Wy6RC4uTjBegYzd\\\\\\\/44QMyVklww3fkpg4KK27p66ejImf1AAq8\\\\\\\/12AEhEF78sNhx7DGFZYiv2C5WQOzsIsaHNvl7WnDWXur+utaLCuOfgIWwwAQkv\\\\\\\/WaZCDj7DQ4lyYxQimWXM+FtkGEP7RIeIJLtSo7jRBwjRQo0lmw4HkdUhu5cH7eanJ7m6tnasvs2DYt3VqOFJTdfeAavjHo58S\\\\\\\/ig8yJBSZDTMb+F8FHHHVyhTjgVFz\\\u0022,\\\u0022iv\\\u0022:\\\u0022isLJgFcpzlG2RFIIBwrtBQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225650c370afe65e20c7db0ef2df2bb025e72dcd713913c01abc71f3d0580b7f65\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v1sXwn2sOMP+m+7HIeY+9u7sFNJza\\\\\\\/BpCG+TRJsAItkbN2IvwLP\\\\\\\/rLcWjsFm8Ae+WAuahGcCRX\\\\\\\/cimfu2rfaIujUWUempIZMG2fgq0r+YNSutgAzv4TfSR4XUvEzgChbJnkiLRcVmwwUIVhpIP7gmfKwX1QP43peKzXn3OIwAFYUs9pCtPmHK8jJo8CUE5CqTnpMAZPUiJ3NkSNlwRq25hyaBQUn\\\\\\\/b4EXXcEPyG6aHa5fLUR1UoljQXiTJvc0WwZNccIULGLS7Ca2qeW61dRL7bKTsekgF4YfoByEqm5XMr0EK2G3RtdSV1uXopd3kEwsPkw6Hn5trSEu\\\\\\\/Pq8AOkG6WPRhyVEZTVCwtgibd1qcoGuvEjbAuue2PBE12HRgyKAK8G6fFbi0IMtu8qpfkgljWJYeZbqdi+l8J5h9NZn\\\\\\\/aICe77w6haR8nlaYRHl+TSS81u1BR\\\\\\\/T9MGH79F8kaNMYP2PWaoUIlesl6EBSkQKg+wiim8fRh7utVLAv6DgcDyUwmxgt+KXyeAhO5nFXEUekccSACxjk+g4q\\\\\\\/au1XmS6MuFMQvWcAuzTIEtfIUAerHpJK3h293\\\\\\\/UdiAF4Yz3G0ktay6JNkTGIFMA9kC\\\\\\\/UTZrc7hz0MvJO32ihkYU6GAuKaMrtapBUufAnsnkFhib2CZBObj2jjl3m0rZTgm3B7VU+KvHmUTbxx9r6r3zOM35rJMF7uIYqLvhJI+94FalOEvj\\\\\\\/ZWbSR+lzuHbtgcahITN4mBJlA3WZXankF1nX5qX\\\\\\\/rSXzQJVB49Kz+juY3G\\\\\\\/yHY3qTfkLq8D6GTKIvpEgv\\\\\\\/Vpii0olEXZK8M2bWdwvt9EiZs3CmtZdGa2ewe6\\\\\\\/w5BpZtnjh4N0NflpAsdyjBZI8y5xTsr6lpsheM\\\\\\\/UFz5fgcETQUIa5bmQK5zjragNFb1iwe+JCwckRy0+FezshS8IbgSeTx3ww2OsEEb7i6WtpVpRJA2hACVRs69IYyCcrVk9NTnIqUIzcJyq1AQFuKABW68\\\\\\\/0QA+8ToD67A6GpYi5IMLw11gAavHhglu+3Jw+QjFU51WprY5fq2ssr26l6lJ0uFwgfe+8U4fEWHmI8TECb36BawUH3gbq6Emie\\\\\\\/IfZHU35+\\\\\\\/x1sj\\\\\\\/4mg1gurdkguGYeewwN9kVswBDuE0QynqLhsIytR+fQJNgqEjuRRnbYSTL30+4pLBYqRbnCJ1jZgE1JbKKrZpc6vErz50Jcen8DFKM2YakzH0+iz5xoMwEajA2GpQbZeXapDBQ6MILiMeJEHJ1b8\\\\\\\/ifqX2rxJeb+n3E\\\\\\\/oXTHvb6BVadrRM6ckaCVwZM5bJ8sip8wQC7RYP2legKHec4c78Gp3EfkBi4pNnHpVvvVZmjSgM4dZT4x8M1BJYY0pZo9waHe3bRY4krUZ+04yFPjoQmBaHWCPTJt0VQcsCNwi4Srw6BzhBRqtpf1Q8Jlu2\\\\\\\/K8V6j2FAshKs1N9K95llplA+hUY\\\\\\\/J2hrIj7\\\\\\\/laE2o1mAKctNNt\\\\\\\/G4KZD0d5oxmZ3\\\\\\\/\\\\\\\/chaME1YRceyALAY6qqM2bkiOm9fuTrJ9Ro1IJvd71QasNlwvPTlSSX7jxQs+lKiXeCZGEOJLUYcW8Y01YBfn6wB9XpFjy+tM7UqqSWsSbFi41fRkmikLQvL8tNgvsZBpl3N4z4v0YjWaror\\\\\\\/\\\\\\\/IGphmGMtIW2teSmUCDN0nbeWjKuAZfZgX15idHj6FurbJcdqSNnUHUTNLHNK9XuF0aApcfquAG7D\\\\\\\/DwgzhWN2YfGQDLi2Rf7KnItLmWCzExiC35Lrt0o+8iwNfTfmQ+oiOtXo9lHQKirx44YRVx79Jb9RxygKYDF8j40Y6qorfDU\\\\\\\/Yqn9U\\\\\\\/nGKYSqk5T3U53XuWkf7VPYUAp6lZvM61PwYtjofRe76h56QcTfju9EbAvhJKrh44oe4VX2i9WO+5SzGSMJKOyMWpYa9OrDzYV+om0zJ3Iw6ZY1t50nHQ18yGExBxSMmSQgtmvVINEj7OjeZY0N5n0lbDQoNVzhmDyTBFQWlGTmMzF36rT0YxmieBSEn5awtvdwB8sFNmfk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022EGrQnMWduDq4pks+Lj84cA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022807448f87db6c4d49adcc3a30623e3de3b2d662e69802ef7a52a5acc3dceb6a4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459715 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2051" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uV2OtJ1xJkph5zSP_yi3zScOEqG8T9e4pRtq2XDOckY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2051" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yCsUO063hly_4_XCMNf6lgWjDbEt2rUXEz5HIgYDgw0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/2049" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2065" title="Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459715" id="node459715" data-pisa="ggg;4/11/2051" data-pisa-master="ggg;g3.114.013565" data-apath="/ggg/4/11/2051.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nisha Rajagopal</span>, <span class="highwire-citation-author" data-delta="1">Jason Ernst</span>, <span class="highwire-citation-author" data-delta="2">Pradipta Ray</span>, <span class="highwire-citation-author" data-delta="3">Jie Wu</span>, <span class="highwire-citation-author" data-delta="4">Michael Zhang</span>, <span class="highwire-citation-author" data-delta="5">Manolis Kellis</span> and <span class="highwire-citation-author" data-delta="6">Bing Ren</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2051-2063;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013565 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459715" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nisha Rajagopal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Systems Biology Program, University of California, San Diego, La Jolla, California 92037</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nisha%2BRajagopal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rajagopal%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANisha%2BRajagopal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Chemistry, University of California, Los Angeles, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pradipta Ray </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pradipta%2BRay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ray%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APradipta%2BRay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jie Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Cold Spring Harbor Laboratory, 1 Bungtown Rd., Cold Spring Harbor, New York 11724</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Applied Mathematics and Statistics, Stony Brook University, Stony Brook, New York 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jie%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJie%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Bioinformatics Division, Center for Synthetic and Systems Biology, Tsinghua National Laboratory for Information Science and Technology, Tsinghua University, Beijing 100084, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Manolis Kellis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manolis%2BKellis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManolis%2BKellis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bing Ren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Cellular and Molecular Medicine, Institute of Genomic Medicine, and Moores Cancer Center, University of California, San Diego School of Medicine, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BRen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ren%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABing%2BRen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:biren@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">biren@ucsd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/11/2051" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459715" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459715/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2051.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459715" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459715/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2051.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459715" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459715/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2051.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459715" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459715/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/11/2051.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459715" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459715/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In eukaryotic cells, histone lysines are frequently acetylated. However, unlike modifications such as methylations, histone acetylation modifications are often considered redundant. As such, the functional roles of distinct histone acetylations are largely unexplored. We previously developed an algorithm RFECS to discover the most informative modifications associated with the classification or prediction of mammalian enhancers. Here, we used this tool to identify the modifications most predictive of promoters, enhancers, and gene bodies. Unexpectedly, we found that histone acetylation alone performs well in distinguishing these unique genomic regions. Further, we found the association of characteristic acetylation patterns with genic regions and association of chromatin state with splicing. Taken together, our work underscores the diverse functional roles of histone acetylation in gene regulation and provides several testable hypotheses to dissect these roles.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/histone-lysine-acetylations" class="hw-term hw-article-keyword hw-article-keyword-histone-lysine-acetylations" rel="nofollow">histone lysine acetylations</a></li><li class="kwd"><a href="/keyword/gene-bodies" class="hw-term hw-article-keyword hw-article-keyword-gene-bodies" rel="nofollow">gene bodies</a></li><li class="kwd"><a href="/keyword/promoters" class="hw-term hw-article-keyword hw-article-keyword-promoters" rel="nofollow">promoters</a></li><li class="kwd"><a href="/keyword/enhancers" class="hw-term hw-article-keyword hw-article-keyword-enhancers" rel="nofollow">enhancers</a></li><li class="kwd"><a href="/keyword/splicing" class="hw-term hw-article-keyword hw-article-keyword-splicing" rel="nofollow">splicing</a></li></ul><p id="p-3">In eukaryotes, DNA is packaged into nucleosomes, each consisting of an octamer of histone proteins that can undergo a large number of post-translational modifications (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tan <em>et al.</em> 2011</a>). Recent advances in high-throughput technologies such as ChIP-seq have led to the discoveries that various regulatory sequences are characterized by distinct patterns of histone modifications, which have increasingly been used as biochemical signatures for annotation of the genome (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rivera and Ren 2013</a>). For instance, combinations of H3K4me1 and H3K4me3 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heintzman <em>et al.</em> 2007</a>) have been exploited for the identification of enhancers and promoters in mammalian genomes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Won <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Firpi <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fernandez and Miranda-Saavedra 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>). Similarly, combination patterns of H3K4me3 and H3K36me3 were used to uncover a large number of long intergenic noncoding (linc) genes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Guttman <em>et al.</em> 2009</a>). Several machine-learning tools have been developed to annotate the histone modification patterns characteristic of various DNA elements (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hon <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ernst and Kellis 2012</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>), but given the large number of histone modifications known to exist, there remains a need for more in-depth analysis of histone combination patterns and their relationships to functional sequences.</p><p id="p-4">Histone acetylations are largely considered markers of activity at regulatory elements such as promoters and enhancers, but because of their tendency to co-occur they have been difficult to elucidate the nonredundant roles of these acetylations (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zentner and Henikoff 2013</a>). Histone acetylations are indirectly targeted in the treatment of diseases such as cancer and HIV by the use of HDAC (histone deacetylase) inhibitors (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dinarello <em>et al.</em> 2011</a>). Understanding the specific role of histone acetylations at different genomic elements has the potential to improve such therapies by increasing the specificity of targeting. Certain lines of evidence have suggested nonredundant roles of histone acetylation such as the fact that HDACs as well as histone acetyl-transferases (HATs) have unique genomic distributions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ram <em>et al.</em> 2011</a>). A previous study found certain acetylations such as H3K9ac to be present at promoters and H4K16ac along gene bodies (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2008</a>). However, the extent to which these acetylations are predictive of particular elements is still unknown.</p><p id="p-5">Although different histone modification patterns have been previously associated with enhancers, promoters, and gene bodies, the discovery of co-transcriptional splicing, the finding that pre-mRNA can be spliced during the process of transcription itself (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Listerman <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lynch 2006</a>), suggested that histone modification patterns could also be indicative of alternative splicing. Subsequently, it was found that exons are marked by well-positioned nucleosomes and elevated levels of certain methylations, in particular H3K36me3 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersson <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hon <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kolasinska-Zwierz <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Spies <em>et al.</em> 2009</a>). Further supporting this notion, changes in acetylation levels were found to affect alternative splicing (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gunderson <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hnilicova <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhou <em>et al.</em> 2011</a>). Here, we explore this subject on a genome-wide scale, describing the extent of association of histone modification with alternative splicing in two distinct mammalian cell types.</p><p id="p-6">In a previous study, we developed a random forest-based method (RFECS) that could effectively identify enhancers genome-wide and determine the most informative set of modifications required for this task (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>). Here, we expand the application of this tool to an extended panel of histone acetylation profiles in two distinct mammalian cell types–human embryonic stem cells, <sup>1</sup>H, and fetal lung fibroblasts, IMR90. Using this approach, we find distinctive patterns of acetylations that are associated with promoters, enhancers, gene bodies, and splice junctions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Datasets and processing</h3><p id="p-7">All datasets used, including 24 modifications in <sup>1</sup>H and IMR90, various sequence-specific transcription factors, and DNase-I hypersensitivity sites, were described previously (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>). In addition, the histone modification datasets in H9 were generated by the Ren laboratory and released as part of the Roadmap Epigenome Project and can be accessed using GSE16256. Data normalization for histone modifications, determination of binding sites of transcription factors, training and prediction using RFECS, correlation clustering, and visualization of chromatin patterns are also as previously described (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>).</p></div><div id="sec-3" class="subsection"><h3>Z-score normalization for comparing enhancers and promoters</h3><p id="p-8">We pooled equal numbers of distal p300 binding sites and known UCSC TSS overlapping DNase-I hypersensitive sites, representing active enhancers and promoters, respectively. We computed average histone modification levels, measured as input-adjusted RPKM (reads per kilobase per million), between −1 and +1 kb around each of these elements. The Z-score normalized profile for each element was calculated against the mean and SD of the histone modification levels of the entire set of pooled elements. Hence, deviations of the mean z-score profile for the TSS class would be positive for TSS-preferred modifications, whereas it would be negative for p300-preferred modifications. This would be the exact mirror image of the values of the mean z-score values for the p300 class.</p></div><div id="sec-4" class="subsection"><h3>Genome-wide prediction of promoters</h3><p id="p-9">To perform supervised prediction of promoters, we created a training set comprising a set of UCSC TSS overlapping DNase-I hypersensitive sites as representative of the active promoter class, and a second class comprising TSS-distal p300 binding sites as well as randomly selected non-p300 regions as background. We used input-adjusted RPKM values of histone modifications (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>) measured in 100-bp bins between −1 and +1 kb around the training set elements as the input features for training this classifier. The RFECS classifier was then used to assign every 100-bp bin in the genome “promoter” or “nonpromoter” class based on a 50% voting percentage, after which promoter peaks were called in a genome-wide fashion as described previously for enhancers (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>). We validated our genome-wide promoter predictions by defining gold standard true positive (TP) and true negative (TN) sets. The former comprises UCSC and Gencode annotated TSS overlapping DNase-I hypersensitivity sites in the particular cell-type, whereas the latter (TN) comprises p300 binding sites, cell-type-specific TFs or DNase-I sites lying within gene desert regions. The TN set was selected to comprise the elements most likely to be mistaken for promoters, due to the enrichment of active modifications. Training and prediction were performed using the RFECS methodology previously applied to prediction of enhancers.</p></div><div id="sec-5" class="subsection"><h3>Computation of variable importance</h3><p id="p-10">We used the out-of-bag measure for variable importance (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bylander 2002</a>) to compute importance of either all modifications or just acetylations for various classification or prediction tasks. Because not all modifications had the same replicates, we permuted replicates of each histone modification to create several different combinations and assessed the variable importance for each of these.</p></div><div id="sec-6" class="subsection"><h3>RNA-seq data processing</h3><p id="p-11">We first mapped the Illumina-generated mRNA fragments (paired end reads) to the exon trio database TXdb, which we have previously built (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2011</a>) using Bowtie version 1 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Langmead <em>et al.</em> 2009</a>) for hits with no more than two mismatches. Our sequence mapping is based on the human genome (hg19 assembly, Genome Reference Consortium GRCh37). The fragments are mapped to TXdb to be able to handle transcriptomic variability that arises from alternative splicing. TXdb represents every known contiguous sequence of exons in the human transcriptome as exonic trios and duos, such that mapping to this database allows us to quantify the splicing pattern in terms of the relative abundance of fragments of the different isoforms in this region, locally.</p><p id="p-12">We ran the splicing analysis tool SpliceTrap version 0.90.5 with default parameters, which uses a Bayesian model to estimate inclusion ratios. SpliceTrap uses an inclusion ratio distribution model (estimated from high-confidence data) to reduce noise in the RNA-Seq data without unnecessarily throwing away evidence from real transcriptomic events. Ultimately, it produces inclusion ratio estimates for all splicing events and classifies all local splicing decisions as constitutively spliced exon (CS), alternative donor site (AD), alternative acceptor site (AA), intronic retention (IR), or alternatively spliced exon (CA).</p><p id="p-13">We chose to use SpliceTrap instead of other RNA-Seq analysis tools due to the fact that the SpliceTrap model is exclusively focused on optimizing a local, exon-centric splicing model (which is also our main focus), and that, in our experience, SpliceTrap produces one of the most robust and consistent estimates of inclusion ratios among the tools we compared (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2011</a>).</p></div><div id="sec-7" class="subsection"><h3>Splice site usage</h3><p id="p-14">We created a measure of splice site usage by using labels associated with each exon–intron boundary to the various categories of splice sites—constitutively spliced exon (CS); alternative donor site (AD); alternative acceptor site (AA); intronic retention (IR); or alternatively spliced exon (CA). Each assignment is accompanied by an inclusion value of the exon with respect to the transcript under consideration. We assigned negative weights to all the cases where inclusion values represent increased inclusion, such as IR, AA (3′ end), AD (5′ end), and we assigned positive weights to the inclusion values that represent decreased inclusion, such as AA (5′ end), AD (3′ end), CA, and CS. The splice site usage value was defined as a weighted mean of the inclusion values, with the weights being the activity of the transcript under consideration. That is, splice site usage for a particular exon–intron boundary is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2051/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2051/embed/mml-math-1.gif"/></noscript></span></span>i is a particular assignment of an exon with respect to a transcript T<sub>j</sub></p><p id="p-15">Incl<sub>i</sub> is the inclusion value of exon–intron boundary in instance i</p><p id="p-16">FPKM<sub>i</sub> is the RNA-seq FPKM value of the transcript i belonging to set T<sub>j</sub></p><p id="p-17">A ={IR, AA (3′ end), AD(5′ end)}</p><p id="p-18">B={AA(5′ end), AD(3′ end), CA, CS}</p><p id="p-19">If there was no assignment for any of the seven cases due to weak coverage in that region, then that term was set to 0.</p></div><div id="sec-8" class="subsection"><h3>Identification of chromatin modification patterns at exon–intron boundaries</h3><p id="p-20">Using SpliceTrap (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2011</a>), we obtained annotations for 286,368 exon–intron boundaries in <sup>1</sup>H and 246,657 such boundaries in IMR90, of which 232,919 boundaries had annotations in both cell types. In each cell type, we randomly selected a subset of 50,000 sites (∼25%) for unsupervised classification because larger number of sites required many more rounds of selection of the number of clusters to filter out the outliers. We performed fast k-means++ algorithm (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arthur and Vassilvitskii 2007</a>) at the exon–intron boundaries using RPKM-normalized histone modification levels in 100-bp bins between −2 and +2 kb around the boundary as features, and we determined the number of clusters using the minimum value of the Davies-Bouldin measure (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Davies 1979</a>). We tested different randomly selected subsets of the data to ensure the results were robust. Further confirmation of the distinctiveness of each of these states was obtained by constructing RFECS classifiers for each cluster against all exon–intron boundaries not assigned to that cluster. We were able to show 100% out-of-bag classification accuracy in <sup>1</sup>H and more than 95% in IMR90 for each cluster as compared with all others. We used these classifiers to assign all the boundaries that had not been used in the clustering to assign them to the appropriate state.</p></div><div id="sec-9" class="subsection"><h3>Significance calculations for transitions of chromatin state at exon–intron boundaries between <sup>1</sup>H and IMR90</h3><p id="p-21">For computing the significance of the transition from cluster i in <sup>1</sup>H to cluster j in IMR90, we used a hyper-geometric distribution. Thus, we modeled the probability by using the following analogies to the standard hyper-geometric distribution framework:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-22">total exon–intron boundaries, N = total population</p></li><li id="list-item-2"><p id="p-23">exon–intron boundaries belonging to cluster i in <sup>1</sup>H, m = elements having desired characteristic</p></li><li id="list-item-3"><p id="p-24">exon–intron boundaries belonging to cluster j in IMR90, n = elements drawn without replacement from the population exon–intron boundaries common to cluster i in <sup>1</sup>H and cluster j in IMR90, x = number of elements drawn from the total population with the desired characteristic</p></li></ul><p id="p-25">In Matlab, the p-value of transition from cluster i in <sup>1</sup>H to cluster j in IMR90 was calculated as:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2051/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2051/embed/mml-math-2.gif"/></noscript></span></span></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Differential histone acetylation patterns at promoters and enhancers</h3><p id="p-26">We previously observed that H3K4me1 and H3K4me3 are the most distinctive marks between promoters and enhancers among a limited set of five histone modifications (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Heintzman <em>et al.</em> 2007</a>). To further define the marks that distinguish these two regulatory elements in genome-wide maps of 24 histone modifications (<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>), we compared active TSSs (TSSs overlapping DNase-I HS sites) with an equal number of enhancers defined by TSS-distal p300 binding. After normalization (<em>Materials and Methods</em>), we observe that the mean histone modification profile of either class separates clearly into TSS-preferred and enhancer-preferred groups (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, positive <em>vs.</em> negative axes). We confirmed that the deviation of most of the histone modifications from a set of elements with randomly shuffled labels is statistically significant for total normalized read counts within −1 to +1 kb of the element (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>; p-value &lt;10<sup>−5</sup> using Wilcoxon test, except for bars marked by black dots). In both <sup>1</sup>H and IMR90 cells, we consistently found that H3K4me1, H2BK20ac, and H2BK120ac are significantly enhancer-preferred, whereas H3K4me3, H3K4me2, H3K9ac, H3K56ac, H4K5ac, and H3K27ac are TSS-preferred (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The histone modification profiles in bin sizes of 100 bp between −1 and +1 kb along these elements are also observed to be different from the random set (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/013565SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS1.pdf">Figure S1</a>, A and B, blue <em>vs.</em> red).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Classification of distal enhancers and promoters. (A) Preference of various histone modifications for either enhancer or promoter using a Z-score normalized score of histone modification levels measured as input-subtracted RPKM (reads per kilobase per million) in 1H (blue bars) and IMR90 (red bars).Modifications with preference for promoters, measured as enrichment on the positive y-axis, in both 1H and IMR90, are shown indicated in red text color on the x-axis label while preference for enhancers or enrichment on the negative y-axis in both cell-types is indicated in blue text color. (B) Classification accuracy achieved using each of the 24 histone modifications individually to separate enhancers from promoters using RFECS in three distinct cell-lines: 1H (blue line), IMR90 (red line), and H9 (green line). H9 is another embryonic stem cell line that was used in this case to see if 1H-specific trends were in fact embryonic stem cell&#x2013;specific. Modifications with the topmost classification accuracy in both 1H and IMR90 are shown in either red or blue text color, depending on whether they have preference for promoters or enhancers in both cell types. (C) Comparison of classification accuracy of acetylations with that of all 24 modifications. (D) Ordering of histone acetylations by their out-of-bag variable importance in classification of enhancers against promoters in 1H. Correlation clustering of histone acetylations at promoters and enhancers in (E) IMR90 and (F) 1H. Acetylations that show similar patterns of co-occurrence in both cell types are indicated in red text color along the axes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1064699205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Classification of distal enhancers and promoters. (A) Preference of various histone modifications for either enhancer or promoter using a Z-score normalized score of histone modification levels measured as input-subtracted RPKM (reads per kilobase per million) in &lt;sup&gt;1&lt;/sup&gt;H (blue bars) and IMR90 (red bars).Modifications with preference for promoters, measured as enrichment on the positive y-axis, in both &lt;sup&gt;1&lt;/sup&gt;H and IMR90, are shown indicated in red text color on the x-axis label while preference for enhancers or enrichment on the negative y-axis in both cell-types is indicated in blue text color. (B) Classification accuracy achieved using each of the 24 histone modifications individually to separate enhancers from promoters using RFECS in three distinct cell-lines: &lt;sup&gt;1&lt;/sup&gt;H (blue line), IMR90 (red line), and H9 (green line). H9 is another embryonic stem cell line that was used in this case to see if &lt;sup&gt;1&lt;/sup&gt;H-specific trends were in fact embryonic stem cell&#x2013;specific. Modifications with the topmost classification accuracy in both &lt;sup&gt;1&lt;/sup&gt;H and IMR90 are shown in either red or blue text color, depending on whether they have preference for promoters or enhancers in both cell types. (C) Comparison of classification accuracy of acetylations with that of all 24 modifications. (D) Ordering of histone acetylations by their out-of-bag variable importance in classification of enhancers against promoters in &lt;sup&gt;1&lt;/sup&gt;H. Correlation clustering of histone acetylations at promoters and enhancers in (E) IMR90 and (F) &lt;sup&gt;1&lt;/sup&gt;H. Acetylations that show similar patterns of co-occurrence in both cell types are indicated in red text color along the axes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2051/F1.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/11/2051/F1.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465112" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Classification of distal enhancers and promoters. (A) Preference of various histone modifications for either enhancer or promoter using a Z-score normalized score of histone modification levels measured as input-subtracted RPKM (reads per kilobase per million) in <sup>1</sup>H (blue bars) and IMR90 (red bars).Modifications with preference for promoters, measured as enrichment on the positive y-axis, in both <sup>1</sup>H and IMR90, are shown indicated in red text color on the x-axis label while preference for enhancers or enrichment on the negative y-axis in both cell-types is indicated in blue text color. (B) Classification accuracy achieved using each of the 24 histone modifications individually to separate enhancers from promoters using RFECS in three distinct cell-lines: <sup>1</sup>H (blue line), IMR90 (red line), and H9 (green line). H9 is another embryonic stem cell line that was used in this case to see if <sup>1</sup>H-specific trends were in fact embryonic stem cell–specific. Modifications with the topmost classification accuracy in both <sup>1</sup>H and IMR90 are shown in either red or blue text color, depending on whether they have preference for promoters or enhancers in both cell types. (C) Comparison of classification accuracy of acetylations with that of all 24 modifications. (D) Ordering of histone acetylations by their out-of-bag variable importance in classification of enhancers against promoters in <sup>1</sup>H. Correlation clustering of histone acetylations at promoters and enhancers in (E) IMR90 and (F) <sup>1</sup>H. Acetylations that show similar patterns of co-occurrence in both cell types are indicated in red text color along the axes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">To assess the importance of each modification in classifying promoters and enhancers, we constructed classifiers using each mark individually. Each classifier was composed of a 20-dimensional vector that was the profile of the histone modification in 100-bp bins between −1 and +1 kb around the element. H3K4me3, followed by H3K4me2 and H3K9ac, showed the highest classification accuracy in both <sup>1</sup>H and IMR90 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>, blue, red). Nearly all modifications showed a classification accuracy of at least 55% (in <sup>1</sup>H) and 75% (in IMR90), which is above the classification accuracy of 50% expected at chance (we verified that classification accuracy on randomly shuffling labels was found to be ∼50%). Clearly, the most significantly TSS-preferred modifications are H3K4me3, H3K4me2, and H3K9ac. For enhancers, H3K4me1 is the most distinctive, followed by H2BK20ac. In addition, we also observed cell-type-specific contributions. To verify if the modifications specific to <sup>1</sup>H are due to the distinct biology of stem cells, we repeated our analysis in H9 human embryonic stem cells and observed trends resembling <sup>1</sup>H (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>, green <em>vs.</em> blue).</p><p id="p-29">We next classified p300 binding sites and TSSs using all 24 marks. Interestingly, H3K4me3 alone achieved the same average accuracy as all 24 modifications in both <sup>1</sup>H (∼94%) and IMR90 (∼95%) . Next, we examined whether histone acetylation alone could classify these two elements (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). The classification accuracy using all 15 acetylations is within 1% of that achieved using all 24 marks. Clearly, acetylations are quite distinctive between the enhancers and promoters.</p><p id="p-30">To identify the specific histone acetylation marks contributing most to the accurate classification of promoters and enhancers, we computed the out-of-bag variable importance (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bylander 2002</a>; <a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>) for each acetylation. For both <sup>1</sup>H and IMR90, the top acetylation mark was H3K9ac (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1D</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS1.pdf">Figure S1</a>C), achieving 85% and 89% classification accuracy, respectively (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, C and D</a>). The next mark in ordering of variable importance of <sup>1</sup>H was H2BK120ac, whereas in the case of IMR90 several marks including H2BK20ac shared the same position (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1D</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS1.pdf">Figure S1</a>C). However, correlation clustering indicates that H2BK20ac and H2BK120ac are highly correlated in both <sup>1</sup>H and IMR90 (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1, E and F</a>), suggesting that these are redundant modifications. Hence, we selected the top two marks as H3K9ac and H2BK120ac and found that this combination achieved a classification accuracy of within 1% of using all 15 acetylations in IMR90, whereas in <sup>1</sup>H, this fell short by ∼3%. Including the next mark in the ordering of <sup>1</sup>H, H3K14ac improved this accuracy by ∼2% (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-31">In summary, we observed that using acetylation marks alone we could accurately separate these promoters from enhancers nearly as well as using all 24 modifications. Our results indicate differential enrichment of specific acetylations at enhancers and TSS. In particular, H3K9ac, H2BK120ac, and H3K14ac appear to be most informative in combination, of which H2BK120ac is enhancer-preferred whereas the other two are TSS-preferred (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1, A and B</a>).</p></div><div id="sec-12" class="subsection"><h3>Histone acetylation patterns accurately predict enhancers and promoters</h3><p id="p-32">Our analysis suggests that histone acetylation patterns are distinct at promoters and enhancers (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1, C and D</a>). Next, we wondered if these acetylations could predict promoters and enhancers genome-wide. As a first step, we applied the RFECS methodology, previously used to predict enhancers (<a id="xref-ref-26-10" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>), to the prediction of promoters genome-wide (<em>Materials and Methods</em>).</p><p id="p-33">Using all 24 histone modifications, our approach can accurately predict promoters with ∼92% true-positive (TP) rate and ∼1.6% false-positive (FP) rate in <sup>1</sup>H, whereas in IMR90 we observed even better performance (TP ∼95%, FP ∼0.3%) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). Using the out-of-bag variable measure, we identified H3K4me3 as the most informative mark required to predict promoters, followed by H3K4me2 and H3K4me1 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS2.pdf">Figure S2</a>, A and B). In terms of the area under the curve (AUC), this minimal set performs comparably with the set of all 24 modifications in both <sup>1</sup>H and IMR90 (AUC<sub>min</sub>/AUC<sub>all</sub> = 0.99) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, red <em>vs.</em> blue). While in <sup>1</sup>H, this set is comparable with using just H3K4me3 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>, black <em>vs.</em> red); in IMR90, the addition of the two marks leads to an improvement of ∼10% in TP rate as compared with H3K4me3 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>, black <em>vs.</em> red).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide prediction of promoters. Receiver operating characteristic (ROC) curves for prediction of promoters in (A) 1H and (B) IMR90 using all 24 modifications (blue), H3K4me3 (black), H3K4me1/2/3 (red), or all 15 acetylations (green). Out-of-bag variable importance for acetylations in making genome-wide prediction of promoters in (C) 1H and (D) IMR90. Modification names indicated in red are the ones that show top-most variable importance in both cell types and are considered candidates for selection in the minimal set. ROC curves for prediction of promoters using various minimal combinations of acetylations in (E) 1H and (F) IMR90, as compared with the prediction using all 15 acetylations (in blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1064699205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide prediction of promoters. Receiver operating characteristic (ROC) curves for prediction of promoters in (A) &lt;sup&gt;1&lt;/sup&gt;H and (B) IMR90 using all 24 modifications (blue), H3K4me3 (black), H3K4me1/2/3 (red), or all 15 acetylations (green). Out-of-bag variable importance for acetylations in making genome-wide prediction of promoters in (C) &lt;sup&gt;1&lt;/sup&gt;H and (D) IMR90. Modification names indicated in red are the ones that show top-most variable importance in both cell types and are considered candidates for selection in the minimal set. ROC curves for prediction of promoters using various minimal combinations of acetylations in (E) &lt;sup&gt;1&lt;/sup&gt;H and (F) IMR90, as compared with the prediction using all 15 acetylations (in blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2051/F2.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/11/2051/F2.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Genome-wide prediction of promoters. Receiver operating characteristic (ROC) curves for prediction of promoters in (A) <sup>1</sup>H and (B) IMR90 using all 24 modifications (blue), H3K4me3 (black), H3K4me1/2/3 (red), or all 15 acetylations (green). Out-of-bag variable importance for acetylations in making genome-wide prediction of promoters in (C) <sup>1</sup>H and (D) IMR90. Modification names indicated in red are the ones that show top-most variable importance in both cell types and are considered candidates for selection in the minimal set. ROC curves for prediction of promoters using various minimal combinations of acetylations in (E) <sup>1</sup>H and (F) IMR90, as compared with the prediction using all 15 acetylations (in blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Next, to assess if acetylation can accurately predict promoters, we repeated our analysis on all 15 histone acetylation marks. For IMR90, overall performance was comparable with using all 24 modifications (AUC<sub>ac</sub>/AUC<sub>all</sub> =0.99) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>, green <em>vs.</em> blue); for <sup>1</sup>H, the TP rate was the same for FP rates beyond 1.3% (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>, green <em>vs.</em> blue). To determine which acetylations are the most informative and whether these are robust across cell types, we computed out-of-bag variable importance for acetylations (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, C and D</a>). H3K9ac is clearly the most informative, whereas the next few marks that are comparable across the two cell types appear to be H2BK120ac, H2AK5ac, and H3K18ac. Several other H2BK-ac also occur among the top ranks in IMR90 (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2D</a>) but are highly correlated with H2BK120ac (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1E</a>).</p><p id="p-36">We then made predictions using just H3K9ac, the top two marks in variable importance for <sup>1</sup>H and IMR90, and also the predicted most informative set of four acetylations. In <sup>1</sup>H, there is a significant difference in the ROC (receiver operating characteristic) curve between H3K9ac and the top two marks, H3K9ac and H2BK120ac, and an equivalent increase on including the next two marks, H2AK5ac and H3K18ac (∼8% increase in TP rate for values of FP &gt; 1%) (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2E</a>, black <em>vs.</em> green <em>vs.</em> red). Even though the performance is not as accurate as using all 15 acetylations, including more marks appears to contribute incrementally to the curves, such as using the top six marks (&lt;2% change in TP for FP &gt; 1%) (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2E</a>, magenta <em>vs.</em> red). In IMR90, there is a significant improvement from using the top two modifications as compared with using H3K9ac alone, with difference in TP ranging between 5% and 20% at the same FP (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2F</a>, black <em>vs.</em> cyan). Beyond this, improvements appear to be more incremental (&lt;2%), such as in using the predicted minimal set of four modifications (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2F</a>, red dotted) or even on including the top eight marks (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2F</a>, magenta).</p><p id="p-37">Applying the RFECS algorithm (<a id="xref-ref-26-11" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>) to enhancers, we compared validation and misclassification rates of prediction using just acetylations with that using all 24 marks or the minimal set of H3K4me1, H3K4me2 (or H3K27ac), and H3K4me3. In <sup>1</sup>H, the validation rate computed based on overlap with known true positives (<a id="xref-ref-26-12" class="xref-bibr" href="#ref-26">Rajagopal <em>et al.</em> 2013</a>) using just acetylations appears to be comparable to the set of three marks, H3K4me1, H3K4me3, and H3K27ac (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS2.pdf">Figure S2</a>C), whereas the misclassification rate appears to be within 1% of that using all 24 modifications (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS2.pdf">Figure S2</a>E). In IMR90, the validation rate using just acetylations is within 3% of that using all 24 modifications (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS2.pdf">Figure S2</a>D, green <em>vs.</em> blue) and a misclassification rate that is within 1% using all 24 modifications (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS2.pdf">Figure S2</a>F, green <em>vs.</em> blue).</p><p id="p-38">Hence, enhancers can also be accurately predicted using just histone acetylation patterns. We computed variable importance for the prediction of genome-wide enhancers using acetylations and discovered H3K9ac, H2BK120/20ac, and H3K14/23ac as the minimal set of acetylations for the prediction of enhancers and confirmed this by comparisons of validation and misclassification rates with performance using all acetylations (data not shown).</p><p id="p-39">In summary, we found acetylations alone to predict genome-wide enhancers as well as promoters quite accurately, indicating that acetylations are not only distinct between the two elements but also predictive. The most informative acetylations in the prediction of promoters were H3K9ac, H2BK120ac, H3K18ac, and H2AK5ac, whereas in the case of enhancers this set was composed of H3K9ac, H2BK120/20ac, and H3K14/23ac.</p></div><div id="sec-13" class="subsection"><h3>Minimal set of modifications to identify active genes</h3><p id="p-40">Several histone modifications have been identified as being enriched in the body of active genes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barski <em>et al.</em> 2007</a>). However, the minimum number of modifications required to achieve an accurate prediction of the active gene body is still an unsolved problem. To this end, we identified active RefSeq genes in the <sup>1</sup>H and IMR90 genomes based on the overlap of their TSS with DNase-I HS sites and RNA-seq above log-value of 2 FPKM. Further, we only considered genic regions lying 2.5 kb away from an annotated TSS. As a true negative set, we identified an equal number of intergenic regions as all those regions not lying within any annotated UCSC, GENCODE, or Refseq gene. We constructed a random forest-based classifier to distinguish these two sets using all 24 histone modifications and observed high sensitivity and specificity at the point of maximum accuracy in both <sup>1</sup>H (sensitivity = 89.56%, specificity = 94.54%, AUC = 0.97) and IMR90 (sensitivity = 96.34%, 1−specificity = 97.09%, AUC = 0.99) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Classification of genic from intergenic regions. ROC curves for classification of genic regions in (A) 1H and (B) IMR90 using various combinations of modifications. Out-of-bag variable importance of all modifications in separating genic from intergenic regions in (C) 1H and (D) IMR90." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1064699205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Classification of genic from intergenic regions. ROC curves for classification of genic regions in (A) &lt;sup&gt;1&lt;/sup&gt;H and (B) IMR90 using various combinations of modifications. Out-of-bag variable importance of all modifications in separating genic from intergenic regions in (C) &lt;sup&gt;1&lt;/sup&gt;H and (D) IMR90.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2051/F3.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/11/2051/F3.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465145" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Classification of genic from intergenic regions. ROC curves for classification of genic regions in (A) <sup>1</sup>H and (B) IMR90 using various combinations of modifications. Out-of-bag variable importance of all modifications in separating genic from intergenic regions in (C) <sup>1</sup>H and (D) IMR90.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">In both <sup>1</sup>H and IMR90, the top two informative marks are H3K36me3 and H3K79me1, which rank well above all other marks (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, C and D</a>). By AUC analysis, the performance of these two marks alone is equivalent to that of all 24 marks in IMR90 (AUC<sub>K36me3,K79me1</sub>/AUC<sub>all</sub> = 100%), although it is somewhat lower in <sup>1</sup>H (AUC<sub>K36me3,K79me1</sub>/AUC<sub>all</sub> = 96%) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, A and B</a>, green). We found that the two marks ranked next that were common to both cell types were H3K27me3 and H3K9me3 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, C and D</a>). These modifications may be important because of their relative depletion in genic regions and enrichment in larger intergenic regions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4D</a>). By including these marks, our classifier achieved almost the same accuracy as all 24 marks in <sup>1</sup>H (<sup>1</sup>H: AUC<sub>top 4</sub>/AUC<sub>all</sub> = 99%) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>, magenta <em>vs.</em> blue). Thus, we conclude that the minimal set of modifications required to predict genes within 1% accuracy of the set of all modifications is between 2 and 4, with H3K36me3 and H3K79me1 being the most informative modifications.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Acetylations within the gene body distal to exon&#x2013;intron boundaries and DNAse-I hypersensitive sites in IMR90. (A) ROC curves showing classification of distal genic regions using all 24 modifications (blue) or only 15 acetylations (green). (B) Out-of-bag variable importance of acetylations in classification of distal genic regions against intergenic regions. (C) Heatmap showing enrichment of acetylations in genic regions as compared with intergenic ones using a Z-score normalized measure. Only certain acetylations show enrichment in a majority of genic regions as compared with intergenic ones, as indicated by the black box, and emphasized by red text color. These modifications are also shown in red in (B) and can be seen to be among the top-most marks for variable importance in separation of genic from intergenic regions. UCSC genome browser snapshot of genes (D) TEAD1, (E) CHRM2, and (F) CALD1, showing enrichment of acetylations as compared with neighboring intergenic regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1064699205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Acetylations within the gene body distal to exon&#x2013;intron boundaries and DNAse-I hypersensitive sites in IMR90. (A) ROC curves showing classification of distal genic regions using all 24 modifications (blue) or only 15 acetylations (green). (B) Out-of-bag variable importance of acetylations in classification of distal genic regions against intergenic regions. (C) Heatmap showing enrichment of acetylations in genic regions as compared with intergenic ones using a Z-score normalized measure. Only certain acetylations show enrichment in a majority of genic regions as compared with intergenic ones, as indicated by the black box, and emphasized by red text color. These modifications are also shown in red in (B) and can be seen to be among the top-most marks for variable importance in separation of genic from intergenic regions. UCSC genome browser snapshot of genes (D) TEAD1, (E) CHRM2, and (F) CALD1, showing enrichment of acetylations as compared with neighboring intergenic regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2051/F4.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/11/2051/F4.medium.gif" width="309" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465220" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Acetylations within the gene body distal to exon–intron boundaries and DNAse-I hypersensitive sites in IMR90. (A) ROC curves showing classification of distal genic regions using all 24 modifications (blue) or only 15 acetylations (green). (B) Out-of-bag variable importance of acetylations in classification of distal genic regions against intergenic regions. (C) Heatmap showing enrichment of acetylations in genic regions as compared with intergenic ones using a Z-score normalized measure. Only certain acetylations show enrichment in a majority of genic regions as compared with intergenic ones, as indicated by the black box, and emphasized by red text color. These modifications are also shown in red in (B) and can be seen to be among the top-most marks for variable importance in separation of genic from intergenic regions. UCSC genome browser snapshot of genes (D) TEAD1, (E) CHRM2, and (F) CALD1, showing enrichment of acetylations as compared with neighboring intergenic regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Acetylations at the gene body</h3><p id="p-44">Next, to assess if gene body acetylation can distinguish genic from nongenic regions, we constructed a supervised classifier using only histone acetylations. Supporting this notion, acetylations show an ROC curve that is well above the line of no discrimination in both <sup>1</sup>H and IMR90 (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, A and B</a>). However, the performance of acetylations is lower (<sup>1</sup>H:AUC<sub>ac</sub>/AUC<sub>all</sub> = 0.85,IMR90: AUC<sub>ac</sub>/AUC<sub>all</sub> =0.92) than that achieved using all 24 marks or even the top four nonacetylation marks (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3, A and B</a>, green <em>vs.</em> blue). For instance, in IMR90, the sensitivity and specificity are 81.24% and 84.94%, respectively, as compared with 95.27% and 97.5% for all 24 marks, at default parameters.</p><p id="p-45">Given the lower proportion of genic regions predicted with acetylations, we asked if this was because of the lower fractions of gene bodies recovered by acetylations or the existence of distinct categories of genes that are either completely acetylated or not, defined based on their predictability using just acetylations. To this end, we examined the distribution of fractions of genes recovered by either case and that using all 24 marks leads to 90–100% recovery of most genes, whereas the fractions recovered by just acetylations appear to be more evenly distributed (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS3.pdf">Figure S3</a>, A and B). The partial recovery of certain genes using acetylations may indicate a bias toward certain elements within the gene. Because previous studies have found associations of acetylations with the splicing of certain genes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gunderson <em>et al.</em> 2011</a>), we tested the hypothesis that acetylations might have a preference for exonic regions or exon–intron boundaries and found this to be true in both <sup>1</sup>H and IMR90 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-46">Although acetylations clearly show a bias toward exonic boundaries, a sizeable fraction of genes (12.7% in <sup>1</sup>H; 16.11% in IMR90) that can be recovered up to &gt;90% using acetylations alone still exists (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS3.pdf">Figure S3</a>, A and B). Distal regulatory elements lying within intronic regions are enriched in acetylations. Because we wanted to see if the gene bodies have a distinct acetylation pattern independent of such intronic enhancers, we selected only those genic regions that are at least 2.5 kb away from a known DNase-I HS or an exon–intron boundary. Now, we calculated the classification rate of these filtered genic <em>vs.</em> intergenic regions using all 24 modifications and just acetylations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS4.pdf">Figure S4</a>A). It can be seen that the recovery using just acetylations is still well above the line of no discrimination (significance stats), with a maximum classification accuracy of ∼70% in <sup>1</sup>H and ∼80% in IMR90 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS4.pdf">Figure S4</a>A, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>).</p><p id="p-47">Because gene body acetylations appeared to be quite discriminative in the case of IMR90, we further examined which acetylations are most enriched within the gene body. H2AK5ac, H3K23ac, H3K14ac, H4K5ac, and H2BK5ac were found to be among the top acetylations in order of variable importance (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>), and also showed enrichment in a majority of genic regions on normalization to intergenic background (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). We selected long genes, such as TEAD1 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>),CHRM2 (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4E</a>), and CALD1 (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4F</a>), that could be classified to more than 90% against an intergenic background. It can be seen that several modifications such as H2AK5ac, H3K14ac, H3K23ac, and H2BK5ac seem to cover a large proportion of the gene as compared with the neighboring intergenic region. Although some of this may be accounted for by the presence of punctate regulatory elements, there are also regions that show diffuse enrichment of the aforementioned acetylations, emphasized in <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4E</a> in the black boxes.</p><p id="p-48">In <sup>1</sup>H, similar analysis yielded a different set of acetylations that were seen to be among the most enriched at gene bodies, with H3K27ac being the top-most in terms of variable importance (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS4.pdf">Figure S4</a>B). On visualizing the enrichment of various histone modifications at genic regions <em>vs.</em> intergenic ones, it does appear that H3K27ac has a ubiquitous but low presence (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS4.pdf">Figure S4</a>C). The enrichment of several acetylations within the gene body can also be at the active gene PTPRJ, which is in sharp contrast to a neighboring intergenic block with H3K9me3 enrichment (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS4.pdf">Figure S4</a>D).</p><p id="p-49">Finally, we examined if acetylations have any functional significance in gene bodies. Gene expression levels were slightly higher at acetylated genes (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS4.pdf">Figure S4</a>, E and F), showing a low but significant Pearson correlation coefficient of 0.2 in <sup>1</sup>H and 0.14 in IMR90. Further, we examined if the genes with higher acetylation had specific associations with functional annotations. In <sup>1</sup>H as well as IMR90, mRNA processing and RNA binding were among the significantly enriched terms (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In addition, each cell-type showed different categories that were enriched such as that of genes involved in regulation of intracellular protein transport in IMR90 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/TableS2.xls">Table S2</a>) or genes involved in mRNA splicing in <sup>1</sup>H (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/TableS1.xls">Table S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465161/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465161/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">GO terms for acetylation-rich genes in <sup>1</sup>H and IMR90</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Histone modification signatures at exon–intron boundaries</h3><p id="p-50">Previous observations of co-transcriptional splicing suggest that specific chromatin signatures may be associated with splicing (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kolasinska-Zwierz <em>et al.</em> 2009</a>). As a preliminary investigation, we chose to analyze the predictive power of the histone modifications under study in predicting exon–intron boundaries from the genic background. Using histone modification profiles (in 100-bp bins) between −2 and +2 kb around the exon–intron boundaries, we were able to classify all known boundaries from genic background with an accuracy of 87% in <sup>1</sup>H (AUC<sub>all</sub> = 0.94) or 85.5% in IMR90 (AUC<sub>all</sub> = 0.93). We then investigated the contribution of each histone modification under study to the prediction. On computing variable importance for each of the histone modifications with respect to the aforementioned classification, we found H3K36me3 followed by H3K79me1 to be the most informative and H3K36me3 alone could classify the boundaries within 3% of the accuracy achieved using all 24 modifications (AUC<sub>k36me3</sub>/AUC<sub>all</sub> ∼96%).</p><p id="p-51">To further investigate the association of histone modifications at exon–introns with function, we identified various splicing events from paired-end RNA-seq in both <sup>1</sup>H and IMR90 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Xie <em>et al.</em> 2013</a>) using SpliceTrap (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2011</a>). The algorithm classified each local splicing decision as being one of constitutively spliced exon (CS), alternative donor site (AD), alternative acceptor site (AA), intronic retention (IR), or alternatively spliced exon (CA) with respect to its flanking exons. Based on the diversity of isoforms of a particular gene, this can cause one exon to be part of multiple alternative splicing events. In each such splicing event, we may characterize the splicing decision in terms of the inclusion ratio, defined as the ratio of quantified expression level of the inclusion isoform divided by the sum of quantified expression levels of both inclusion and exon-skipped isoforms. Further, each exon can also be quantified in terms of the exonic activity measured as FPKM (fragments per kilobase per million mapped reads). We aim to use these two quantifications at the exonic level to tease out correlations between histone modification signals and splicing activity.</p><p id="p-52">Because there is a wide diversity of splicing activity in the transcriptome, the multiple signals associated with an exon–intron boundary may lead to the observation of a convoluted histone modification signal. As a first step toward deconvoluting such putative chromatin modification signals, we discovered all possible chromatin modification patterns at exon–intron junctions using a fast k-means++ algorithm (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arthur and Vassilvitskii 2007</a>) (see <em>Materials and Methods</em>). Six distinct clusters are observed in <sup>1</sup>H (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>), with varying levels of acetylations as well as other gene body marks such as H3K36me3, H3K79me1, and H4K20me1. Each of these clusters were characterized in terms of their distinctiveness from the genic background by classifying the exons assigned to the cluster against the genic background using either all 24 modifications or just acetylations (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS5.pdf">Figure S5</a>, A and C). Overall, state 2 is unclassifiable against background using just acetylations, indicating that the weak acetylation signature is comparable with the gene body, whereas other states were found to be either overenriched (states 1, 5, 6) or underenriched (states 3, 4) for acetylations as compared with the rest of the gene (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS5.pdf">Figure S5</a>, A and C). It is worth noting that only those states with enrichment of acetylations appear to have presence of H3K79me1 as well.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromatin modification patterns at exon&#x2013;intron junctions in 1H and IMR90. (A) Six distinct chromatin modification patterns at exon&#x2013;intron junctions with corresponding levels of exonic activity (panel 2) and splice site retention (panel 3). (B) Four distinct chromatin modification patterns at exon&#x2013;intron junctions with corresponding levels of exonic activity (panel 2) and splice site retention (panel 3). Ranks associated with each of these parameters are shown for the clusters, in 1H as well as in IMR90, based on significant differences in either exonic activity or splice site retention between the clusters, measured using a Wilcoxon test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1064699205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromatin modification patterns at exon&#x2013;intron junctions in &lt;sup&gt;1&lt;/sup&gt;H and IMR90. (A) Six distinct chromatin modification patterns at exon&#x2013;intron junctions with corresponding levels of exonic activity (panel 2) and splice site retention (panel 3). (B) Four distinct chromatin modification patterns at exon&#x2013;intron junctions with corresponding levels of exonic activity (panel 2) and splice site retention (panel 3). Ranks associated with each of these parameters are shown for the clusters, in &lt;sup&gt;1&lt;/sup&gt;H as well as in IMR90, based on significant differences in either exonic activity or splice site retention between the clusters, measured using a Wilcoxon test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2051/F5.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/11/2051/F5.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465281" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-53" class="first-child">Chromatin modification patterns at exon–intron junctions in <sup>1</sup>H and IMR90. (A) Six distinct chromatin modification patterns at exon–intron junctions with corresponding levels of exonic activity (panel 2) and splice site retention (panel 3). (B) Four distinct chromatin modification patterns at exon–intron junctions with corresponding levels of exonic activity (panel 2) and splice site retention (panel 3). Ranks associated with each of these parameters are shown for the clusters, in <sup>1</sup>H as well as in IMR90, based on significant differences in either exonic activity or splice site retention between the clusters, measured using a Wilcoxon test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">In IMR90, however, we observed four distinct chromatin modification patterns (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). In common with <sup>1</sup>H, there is an “enhancer-like” cluster, cluster 1 (cluster 1 in <sup>1</sup>H), and “promoter-like” cluster, cluster 2 (cluster 5 and 6, <sup>1</sup>H), based on enrichment of H3K4me1 and me3, respectively. As in <sup>1</sup>H, these two are significantly enriched in acetylations with respect to genic background, whereas state 4 is significantly depleted (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS5.pdf">Figure S5</a>, B and D).</p><p id="p-55">The learned histone modification states in <sup>1</sup>H cells are ranked in decreasing order of exonic activity based on calculations of statistical significance of the difference of mean RNA-seq FPKM (fragments per kilobase per million) levels between clusters using a Student’s <em>t</em>-test (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, panel 2). In <sup>1</sup>H, there appears to be a positive correlation with the level of H3K36me3, which is apparent as clusters 2 &gt; 3 &gt; 4 that show significantly decreasing trends of activity also have correspondingly decreasing H3K36me3 (spearman correlation for clusters 1 to 4 = 0.59; p-value &lt; 2.2×10<sup>−308</sup>). However, “TSS”-like signatures (clusters 5 and 6) appear to be even more highly active, irrespective of H3K36me3 enrichment. The same trend may be observed in IMR90, where cluster 3 with the lowest enrichment of H3K36me3 also has the lowest activity (spearman correlation for clusters 1, 3, and 4 = 0.47; p-value &lt; 2.2×10<sup>−308</sup>), and “TSS-like” state 2 has the maximum exonic activity (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>, panel 2).</p><p id="p-56">In summary, H3K36me3 can accurately classify most exon–intron junctions from genic background. We identified multiple distinct chromatin states at both <sup>1</sup>H and IMR90 that are associated with varying levels of exonic activity. We found that there was considerable variation in the levels of acetylations at exon–intron boundaries, many of which were either highly enriched or highly depleted in acetylations with respect to the rest of the gene.</p></div><div id="sec-16" class="subsection"><h3>Chromatin modification patterns are predictive of splice-site usage</h3><p id="p-57">As described in the section above, an exon can be part of multiple different splicing events such as constitutively spliced exon (CS), alternative donor site (AD), alternative acceptor site (AA), intronic retention (IR), or alternatively spliced exon (CA) with respect to its flanking exons. A single exon–intron junction can have multiple assignments of inclusion values based on the transcript under consideration. Hence, we further developed a metric to characterize the overall splice site usage for every exon–intron boundary based on an expression-weighted average of its inclusion ratio in all transcripts (<em>Materials and Methods</em>).</p><p id="p-58">Chromatin modification clusters are ranked in decreasing order of retention or increasing order of splice site usage in <sup>1</sup>H using a Wilcoxon test with a p-value cutoff of 10<sup>−5</sup> (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a>, panel 3). A clear trend is observed where the greater the enrichment of acetylations, the stronger the tendency for retention, with clusters 6, 5, and 1 having the maximum tendency for retention (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a>, panel 3). In IMR90 as well, the highly acetylated clusters 2 and 1 showed significantly higher retention of the boundary (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5B</a>, panel 3, ranked I and II based on a p-value cutoff of 10<sup>−5</sup>).</p><p id="p-59">We asked to what extent we could predict retention of exon–intron junctions based on chromatin modifications as input features. We defined the constitutive class of exon–intron boundaries as those that have the maximum possible value of inclusion ratio, 0.999, in all transcripts of which they are a part. We defined two categories of alternatively spliced exon–intron boundaries based on their contribution to splice-site usage: group I class of boundaries comprising IR, AD (5′ end), and AA (3′ end) contribute negatively to splice-site usage, whereas group II class of boundaries comprising CA contributes positively to splice-site usage, as defined above. Using all 24 modifications, we obtained a maximal classification accuracy of ∼70% and AUC of 0.75 for group I exon–intron boundaries in IMR90 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>B, black). Although this is clearly greater than expected at random, we asked if we could further improve the classification accuracy by taking into consideration other factors. For instance, exon–intron boundaries within close proximity of each other may share the same chromatin signature, which would cause difficulty in classification. To verify this, we filtered any retained exon–intron boundary within different distances of the constitutive exon–intron boundaries and found a steady improvement in accuracy of classification with filtering distance (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>B, black to red). Now, if we consider filtering the group I elements for any constitutive exon–intron boundaries, we actually observed a worsening of the performance (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>B, black <em>vs.</em> dotted blue). We obtained the best possible accuracy of classification with an AUC of 0.84 and maximal accuracy of 77.1% by using a filtering distance of 10 kb for determining the set of distal constitutive exon–intron boundaries in IMR90 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, blue). In <sup>1</sup>H, we observed the same trend (data not shown) and obtained a maximal accuracy of 76.5% and AUC of 0.84 for classification of group I exon–intron junctions against distal constitutive ones (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>, blue).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Associations of chromatin modification patterns with splicing. (A&#x2013;D) ROC curves for the classification of alternatively spliced exon&#x2013;intron junctions against constitutively spliced ones using all 24 modifications (blue), 15 acetylations (red), or H3K36me3 (green) for classification of (A, B) group I exon&#x2013;intron junctions comprising intronic retention (IR), alternative 5&#x2032; end usage (AD), and alternative 3&#x2032; end usage (AA) in (A) IMR90 and (B) 1H. Group II exon&#x2013;intron junctions comprising alternatively spliced exon (CA) in (C) IMR90 and (D) 1H. Negative logarithm of the p-value of enrichment of alternatively spliced exons exclusive to 1H (blue) or IMR90 (red) in (E) IMR90 and (F) 1H." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1064699205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Associations of chromatin modification patterns with splicing. (A&#x2013;D) ROC curves for the classification of alternatively spliced exon&#x2013;intron junctions against constitutively spliced ones using all 24 modifications (blue), 15 acetylations (red), or H3K36me3 (green) for classification of (A, B) group I exon&#x2013;intron junctions comprising intronic retention (IR), alternative 5&#x2032; end usage (AD), and alternative 3&#x2032; end usage (AA) in (A) IMR90 and (B) &lt;sup&gt;1&lt;/sup&gt;H. Group II exon&#x2013;intron junctions comprising alternatively spliced exon (CA) in (C) IMR90 and (D) &lt;sup&gt;1&lt;/sup&gt;H. Negative logarithm of the p-value of enrichment of alternatively spliced exons exclusive to &lt;sup&gt;1&lt;/sup&gt;H (blue) or IMR90 (red) in (E) IMR90 and (F) &lt;sup&gt;1&lt;/sup&gt;H.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2051/F6.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/11/2051/F6.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2051/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465213" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-60" class="first-child">Associations of chromatin modification patterns with splicing. (A–D) ROC curves for the classification of alternatively spliced exon–intron junctions against constitutively spliced ones using all 24 modifications (blue), 15 acetylations (red), or H3K36me3 (green) for classification of (A, B) group I exon–intron junctions comprising intronic retention (IR), alternative 5′ end usage (AD), and alternative 3′ end usage (AA) in (A) IMR90 and (B) <sup>1</sup>H. Group II exon–intron junctions comprising alternatively spliced exon (CA) in (C) IMR90 and (D) <sup>1</sup>H. Negative logarithm of the p-value of enrichment of alternatively spliced exons exclusive to <sup>1</sup>H (blue) or IMR90 (red) in (E) IMR90 and (F) <sup>1</sup>H.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">Histone lysine acetylations had been observed to be enriched at clusters with greater degree of retention (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5, A and B</a>). To further explore the relative importance of histone lysine acetylations, we classified the group I exon–intron junctions against the distal constitutive ones and obtained a comparable classification accuracy as using all 24 modifications (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, A and B</a>, blue <em>vs.</em> red, <sup>1</sup>H: AUC<sub>ac</sub>/AUC<sub>all</sub> = 0.96, IMR90: AUC<sub>ac</sub>/AUC<sub>all</sub> = 0.98). Previous studies had shown H3K36me3 to be distinctive between alternatively spliced exons and constitutively spliced ones (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hon <em>et al.</em> 2009</a>). As compared with acetylations, H3K36me3 was able to achieve a much lower accuracy of classification (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6, A and B</a>, blue <em>vs.</em> red, <sup>1</sup>H: AUC<sub>k36</sub>/AUC<sub>all</sub> = 0.88, IMR90: AUC<sub>k36</sub>/AUC<sub>all</sub> = 0.94), indicating the stronger association of group I alternatively spliced exons with acetylation signatures, rather than H3K36me3.</p><p id="p-62">On classification of group II alternatively spliced exons against a constitutive background, we achieved a maximal accuracy of ∼66% and AUC of 0.71 in IMR90 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>B, black). We considered the case of classifying distal alternative <em>vs.</em> constitutive ones and found a steady improvement of classification accuracy on increasing filtering distance for removing nonretained exon–intron boundaries in the vicinity of the group II alternative class (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>B, solid blue to red). At best, we achieved a maximal accuracy of 80% and AUC of 0.88 for classifying distal group II alternative exon–intron boundaries against the constitutive background in IMR90 (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>, blue). In <sup>1</sup>H, we achieved a similarly high level of accuracy of performance with a maximal accuracy of 78% and AUC of 0.87 (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6D</a>, blue). In the case of group II junctions, H3K36me3 was more effective than histone lysine acetylations in classifying alternative boundaries against constitutive ones (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6, C and D</a>, green <em>vs.</em> red), although neither acetylations nor H3K36me3 could achieve comparable performance as that using 24 modifications (<sup>1</sup>H: AUC<sub>k36</sub>/AUC<sub>all</sub> = 0.9, IMR90: AUC<sub>k36</sub>/AUC<sub>all</sub> = 0.89).</p><p id="p-63">Patterns in both cell types were also associated with specific splice variants to see if there were significant associations with these (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>, C and D). Alternative donor sites or 5′ splice sites were enriched in the promoter-like clusters in both cell types as compared with any other state. However, surprisingly, all other splice variants also have a greater tendency to occur proximal to such promoter-like signatures. PLEKH3 is a gene that is predicted by SpliceTrap to have a series of retained exon–intron boundaries in <sup>1</sup>H that are constitutively spliced in IMR90. This gene can be seen to have chromatin state changes associated with alternative splicing and retention (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS7.pdf">Figure S7</a>A). The reverse can be seen in the gene VIM where certain exons that are constitutive in <sup>1</sup>H are retained in IMR90 (<a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS7.pdf">Figure S7</a>B). In both cases, the set of exons undergoing various types of retention, excluding alternative 5′ site usage, are indicated by a black box and can be seen to be covered by the expansion of H3K4me3 signal in the cell type with alternate usage. Another observation to note was that state 4 in <sup>1</sup>H appeared to be preferential for exons with both ends constitutively spliced, whereas states 1, 5, and 6 show preference for other events such as alternative acceptor sites or intronic retention (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5A</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/12/g3.114.013565.DC1/FigureS6.pdf">Figure S6</a>D).</p><p id="p-64">In conclusion, using chromatin modification information, we were able to achieve accuracy as high as 80% for the classification of alternatively spliced exon–intron junctions from a constitutively spliced background. We observed improvement in classification accuracy on considering a constitutive background distal to any retained exon in case of group I exons and by considering an alternative class distal to constitutive exons in case of group II exons. This suggests the effect of proximal chromatin signature on neighboring exons. Retained exon–intron boundaries are highly enriched for histone lysine acetylations, especially intronic retention, alternative 3′ end usage, and alternative 5′ end usage. “Enhancer-like” and “promoter-like” chromatin states that appear to be associated with splice site retention are common to both cell types, of which the latter is the most strongly associated with a variety of splice site variants, not just alternative 5′ sites.</p></div><div id="sec-17" class="subsection"><h3>Dynamics of chromatin modification states at splice sites</h3><p id="p-65">Certain chromatin modification clusters in <sup>1</sup>H appear to be analogous to ones in IMR90 based on the patterns of modifications, such as the “enhancer-like” state 1 (<sup>1</sup>H) with state 1 (IMR90), and the “promoter-like” state 5 and state 6 (<sup>1</sup>H) with state 2 (IMR90) (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5, A and B</a>). However, the other clusters are not so easily comparable in terms of chromatin modifications. In this regard, we examined if particular states in <sup>1</sup>H have a tendency to correspond to ones in IMR90 based on the number of exon–intron junctions that are common to the states in the two cell types. We computed the p-value of transitions between the six states in <sup>1</sup>H to the four states in IMR90 using a hyper-geometric distribution (<em>Materials and Methods</em>) and significant transitions, based on a p-value &lt; 2.2×10<sup>−308</sup>, are enumerated in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. It appears that the chromatin state transitions are in keeping with the overall ranking in terms of splice site usage. For instance, state 2 in <sup>1</sup>H and state 4 in IMR90 show significant transitions even though their chromatin modification patterns do not appear to be the same. However, both these clusters are ranked immediately after the “promoter-like” and “enhancer-like” states in terms of their splice site usage. Such a trend is in keeping with the fact that the change in splice site usage across the two cell types is relatively small. For instance, if we assume any exon junction with splice site usage &lt;0.9 to be called alternative, then only 1.92% of the total exons undergo any change at all in their splice site usage between <sup>1</sup>H and IMR90.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465230/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465230/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Significant chromatin state transitions at exon–intron junctions between <sup>1</sup>H and IMR90</span><div class="sb-div caption-clear"></div></div></div><p id="p-66">We observed that we could obtain a considerably higher accuracy of classification of group II alternatively spliced exons in <sup>1</sup>H if we considered a negative set that was composed of constitutive exons in both <sup>1</sup>H and IMR90, rather than just <sup>1</sup>H with an improvement in maximal accuracy of approximately 4% (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6D</a>, magenta <em>vs.</em> blue). However, there is not much difference in accuracy of classification on using this constitutive background in IMR90 (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6C</a>, magenta <em>vs.</em> blue). This suggests that certain constitutive exons in <sup>1</sup>H may be “pre-marked” for alternative splicing in IMR90. To validate this, we created two sets of junctions based on splice site usage—one that is alternatively spliced in <sup>1</sup>H but not IMR90 and another that is alternatively spliced in IMR90 but not <sup>1</sup>H (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6F</a>,blue <em>vs.</em> red). Both the acetylation-rich clusters 1 and 6 in <sup>1</sup>H (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5A</a>) are significantly enriched for cell-type-specific retained junctions whether it is in <sup>1</sup>H or IMR90 (<a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6F</a>). However, in IMR90, the corresponding acetylation-rich clusters 1 and 2 are not significantly enriched for <sup>1</sup>H-exclusive retention events (<a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6E</a>). Hence, it may be that the states in <sup>1</sup>H are pre-marked for alternative splicing in IMR90 because they are undifferentiated cells that contain the tendency for alternative splicing in future differentiated cells as well. Because IMR90 is a fully differentiated cell type, it does not show similar tendencies.</p><p id="p-67">Overall, it appears that only a small proportion (&lt;2%) of exons undergo alternative splicing changes between <sup>1</sup>H and IMR90. The chromatin modification patterns at exon–intron boundaries changes across <sup>1</sup>H and IMR90 in such a manner to correspond to the splice site usage corresponding to the cluster, rather than the actual enrichment of various modifications. Also, constitutive exon–intron boundaries in <sup>1</sup>H may be pre-marked by an alternative splice site signature for use in later differentiated cell types such as IMR90.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-68">Chromatin modifications distinguishing promoters and enhancers have previously been identified as H3K4me1 and H3K4me3 (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Heintzman <em>et al.</em> 2007</a>). Besides these two, we find that several additional histone modifications, especially histone acetylations, can also reliably distinguish these regulatory elements. In particular, H3K9ac, H3K23ac, and H3K14ac are promoter-preferred, whereas H2BK120ac and H2BK20ac are enhancer-preferred. Overall, histone acetylation is not only distinctive between the two regulatory elements but also informative enough to predict promoters and enhancers genome-wide. These observations potentially lead to several hypotheses regarding differences in mechanisms of functioning of these two regulatory elements. H2BK120 has been shown to have a ubiquitination modification that is present at active promoters and exclusive of H2BK120ac (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gatta <em>et al.</em> 2011</a>). This exclusivity may explain the presence of H2BK120ac at enhancers, and may suggest the lack of H2BK120Ub at these elements. Understanding the dynamics of the H2BK120 acetylase, KAT3 (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gatta <em>et al.</em> 2011</a>), and the H2BK120 ubiquitin ligase, RNF20 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hwang <em>et al.</em> 2003</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhu <em>et al.</em> 2005</a>), may lead to further understanding of differences between enhancers and promoters.</p><p id="p-69">In addition to enhancers and promoters, acetylations were found to be quite informative in delineating gene bodies. It was previously observed that certain acetylations showed elevated levels at the promoter region as well as the transcribed regions of active genes in CD4<sup>+</sup> T-cells (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2008</a>). We wanted to investigate if specific patterns of acetylations were distinctive of the gene body alone by investigating the predictive power of these acetylations in <sup>1</sup>H and IMR90 cell types. We found extensive enrichment of H2AK5ac, H2BK120ac, H3K14ac, and H3K23ac along gene bodies, and acetylations alone can achieve 80% accuracy in predicting gene bodies. Some studies have shown PCAF to be regulating H3K14ac (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lau <em>et al.</em> 2000</a>), also known to be part of an elongation-competent form of RNA-polymerase II (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cho <em>et al.</em> 1998</a>). This factor may be involved in the maintenance of gene body acetylations in IMR90. Tip60 and HDAC6 have also been characterized as being within gene bodies (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2009</a>), the former of which is known to acetylate H2AK5 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jeong <em>et al.</em> 2011</a>). Hence, given the patterns of acetylations within gene bodies and prediction of genes enriched in these, there is a potential to generate hypotheses regarding the combinatorial localization of HATs and HDACs within specific genes.</p><p id="p-70">Acetylations within the gene body are especially enriched near exon–intron junctions of retained exons. We described two groups of such exon–intron junctions—one that comprised events contributing to negative splice site usage such as intronic retention (IR), alternative 3′ end (AA), and alternative 5′ end (AD) usage, and another that contributed to positive splice site usage comprising alternatively spliced exons (CA). We found that both these groups showed significant association with proximal chromatin state but had differential associations with histone lysine acetylations. Histone lysine acetylations were found to be highly discriminative in classifying group I exons against a set of distal constitutive exon. However, H3K36me3 depletion appeared to be more distinctive of the class of group II exons. We observed improvement in classification accuracy on considering a constitutive background distal to any retained exon in case of group I exons and by considering an alternative class distal to constitutive exons in case of group II exons. Possibly, the chromatin signature of these group I exons may be more permissive for allowing constitutive splicing in neighboring exons, whereas the chromatin signature of group II exons may not be as permissive for allowing constitutive splicing and may be more strictly restricted to distal group II exons. In case of constitutive and alternative exons within close proximity to each other, factors other than the chromatin state may play an important role in regulating the splicing. One of these could be the effect of distal regulatory elements interacting with the splice-site junctions (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mercer <em>et al.</em> 2013</a>). The role of such elements in splicing can be further studied using a chromosomal conformation captures technique such as 4C (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhao <em>et al.</em> 2006</a>). It was also observed that many acetylation-rich, constitutive exons in <sup>1</sup>H are alternatively spliced in IMR90. Such a hypothesis may be further tested by including detailed splicing and chromatin formation across many human cell lines, both from early and late lineages.</p><p id="p-71">Hence, we observed patterns of histone acetylations that are specific to promoters, enhancers, and genic regions. Such observations are in keeping with many previous studies regarding the localization of chromatin modifiers at these elements and suggest further testable hypotheses regarding the combinatorial enrichment of potential chromatin modifiers at these regions, which could lead to a better understanding of the mechanism of functioning of enhancers, promoters, and genes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">We thank collaborators in the Epigenome Data Analysis and Coordination Center and NCBI who have processed data used in this project. We thank Ashwinikumar Kulkarni for his help in processing RNA-seq data. We thank Gary Hon for his valuable comments on the manuscript. We acknowledge support from the NIH Roadmap Epigenomics Program (U01 ES017166-01), sponsored by the National Institute on Drug Abuse (NIDA) and the National Institute of Environmental Health Sciences (NIEHS). No disclosures have been declared.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-73"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-08-01"><span class="received-label">Received </span>August 1, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-09-19"><span class="accepted-label">Accepted </span>September 19, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Rajagopal <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.1" data-doi="10.1101/gr.092353.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enroth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rada-Iglesias</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wadelius</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komorowski</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nucleosomes are well positioned in exons and carry characteristic histone modifications.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1732</span>–<span class="cit-lpage">1741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092353.109%26rft_id%253Dinfo%253Apmid%252F19687145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xNzMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMDUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.11.2051.2"><div class="cit-metadata unstructured">Arthur, D., and S. Vassilvitskii, 2007. k-means++: the advantages of careful seeding. In <em>Proceedings of the eighteenth annual ACM-SIAM symposium on Discrete algorithms</em>, pp. 1027–1035. Society for Industrial and Applied Mathematics, New Orleans, Louisiana.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.3" data-doi="10.1016/j.cell.2007.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuddapah</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roh</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schones</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-resolution profiling of histone methylations in the human genome.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBarski%26rft.auinit1%253DA.%26rft.volume%253D129%26rft.issue%253D4%26rft.spage%253D823%26rft.epage%253D837%26rft.atitle%253DHigh-resolution%2Bprofiling%2Bof%2Bhistone%2Bmethylations%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.05.009%26rft_id%253Dinfo%253Apmid%252F17512414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17512414&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246788300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.4" data-doi="10.1023/A:1013964023376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bylander</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimating generalization error on two-class datasets using out-of-bag estimates.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D48%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1013964023376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1013964023376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.5" data-doi="10.1128/MCB.18.9.5355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orphanides</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogryzko</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A human RNA polymerase II complex containing factors that modify chromatin structure.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">5355</span>–<span class="cit-lpage">5363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DCho%26rft.auinit1%253DH.%26rft.volume%253D18%26rft.issue%253D9%26rft.spage%253D5355%26rft.epage%253D5363%26rft.atitle%253DA%2BHuman%2BRNA%2BPolymerase%2BII%2BComplex%2BContaining%2BFactors%2BThat%2BModify%2BChromatin%2BStructure%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.9.5355%26rft_id%253Dinfo%253Apmid%252F9710619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvNTM1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2051.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouldin</span>  <span class="cit-name-given-names">D. W.</span></span>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">A Cluster Separation Measure.</span> In IEEE Trans-47 actions on Pattern Analysis and Machine Intelligence Vol PAMI-1 pp. <span class="cit-fpage">224</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.7" data-doi="10.1007/s00894-010-0726-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinarello</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fossati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascagni</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone deacetylase inhibitors for treating a spectrum of diseases not related to cancer.</span> <abbr class="cit-jnl-abbrev">Mol. Med.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmedicine%2B%2528Cambridge%252C%2BMass.%2529%26rft.stitle%253DMol%2BMed%26rft.aulast%253DDinarello%26rft.auinit1%253DC.%2BA.%26rft.volume%253D17%26rft.issue%253D5-6%26rft.spage%253D333%26rft.epage%253D352%26rft.atitle%253DHistone%2Bdeacetylase%2Binhibitors%2Bfor%2Btreating%2Ba%2Bspectrum%2Bof%2Bdiseases%2Bnot%2Brelated%2Bto%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00894-010-0726-4%26rft_id%253Dinfo%253Apmid%252F21556484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00894-010-0726-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21556484&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.8" data-doi="10.1038/nmeth.1906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ChromHMM: automating chromatin-state discovery and characterization.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DErnst%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D215%26rft.epage%253D216%26rft.atitle%253DChromHMM%253A%2Bautomating%2Bchromatin-state%2Bdiscovery%2Band%2Bcharacterization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1906%26rft_id%253Dinfo%253Apmid%252F22373907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22373907&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300890400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda-Saavedra</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide enhancer prediction from epigenetic signatures using genetic algorithm-optimized support vector machines.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res. 40: e77.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.10" data-doi="10.1093/bioinformatics/btq248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Firpi</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ucar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discover regulatory DNA elements using chromatin signatures and artificial neural network.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1579</span>–<span class="cit-lpage">1586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq248%26rft_id%253Dinfo%253Apmid%252F20453004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTMvMTU3OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.11" data-doi="10.4161/epi.6.5.15623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolfini</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zambelli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imbriano</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavesi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An acetylation-mono-ubiquitination switch on lysine 120 of H2B.</span> <abbr class="cit-jnl-abbrev">Epigenetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">630</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEpigenetics%26rft.aulast%253DGatta%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D630%26rft.epage%253D637%26rft.atitle%253DAn%2Bacetylation-mono-ubiquitination%2Bswitch%2Bon%2Blysine%2B120%2Bof%2BH2B.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fepi.6.5.15623%26rft_id%253Dinfo%253Apmid%252F21739721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/epi.6.5.15623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21739721&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290203600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.12" data-doi="10.1073/pnas.1011982108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunderson</span>  <span class="cit-name-given-names">F. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkhofer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dynamic histone acetylation is critical for cotranscriptional spliceosome assembly and spliceosomal rearrangements.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">2004</span>–<span class="cit-lpage">2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011982108%26rft_id%253Dinfo%253Apmid%252F21245291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzUvMjAwNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.13" data-doi="10.1038/nature07672"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amit</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGuttman%26rft.auinit1%253DM.%26rft.volume%253D458%26rft.issue%253D7235%26rft.spage%253D223%26rft.epage%253D227%26rft.atitle%253DChromatin%2Bsignature%2Breveals%2Bover%2Ba%2Bthousand%2Bhighly%2Bconserved%2Blarge%2Bnon-coding%2BRNAs%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07672%26rft_id%253Dinfo%253Apmid%252F19182780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07672&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182780&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264059700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.14" data-doi="10.1038/ng1966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heintzman</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ching</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHeintzman%26rft.auinit1%253DN.%2BD.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D318%26rft.atitle%253DDistinct%2Band%2Bpredictive%2Bchromatin%2Bsignatures%2Bof%2Btranscriptional%2Bpromoters%2Band%2Benhancers%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1966%26rft_id%253Dinfo%253Apmid%252F17277777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17277777&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244480000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.15" data-doi="10.1371/journal.pone.0016727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hnilicova</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hozeifi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duskova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Icha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomankova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone deacetylase activity modulates alternative splicing.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHnilicova%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De16727%26rft.epage%253De16727%26rft.atitle%253DHistone%2Bdeacetylase%2Bactivity%2Bmodulates%2Balternative%2Bsplicing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016727%26rft_id%253Dinfo%253Apmid%252F21311748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21311748&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.16" data-doi="10.1371/journal.pcbi.1000201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">ChromaSig: a probabilistic approach to finding common chromatin signatures in the human genome.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DHon%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000201%26rft.epage%253De1000201%26rft.atitle%253DChromaSig%253A%2Ba%2Bprobabilistic%2Bapproach%2Bto%2Bfinding%2Bcommon%2Bchromatin%2Bsignatures%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000201%26rft_id%253Dinfo%253Apmid%252F18927605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18927605&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.17" data-doi="10.1371/journal.pcbi.1000566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Discovery and annotation of functional chromatin signatures in the human genome.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DHon%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000566%26rft.epage%253De1000566%26rft.atitle%253DDiscovery%2Band%2Bannotation%2Bof%2Bfunctional%2Bchromatin%2Bsignatures%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000566%26rft_id%253Dinfo%253Apmid%252F19918365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000566&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19918365&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.18" data-doi="10.1016/S1097-2765(02)00826-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatasubrahmanyam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ianculescu</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A conserved RING finger protein required for histone H2B monoubiquitination and cell size control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHwang%26rft.auinit1%253DW.%2BW.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D261%26rft.epage%253D266%26rft.atitle%253DA%2Bconserved%2BRING%2Bfinger%2Bprotein%2Brequired%2Bfor%2Bhistone%2BH2B%2Bmonoubiquitination%2Band%2Bcell%2Bsize%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900826-2%26rft_id%253Dinfo%253Apmid%252F12535538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00826-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12535538&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180848900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.19" data-doi="10.1038/nsmb.2153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Situ</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulmer</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recognition of enhancer element-specific histone methylation by TIP60 in transcriptional activation.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DJeong%26rft.auinit1%253DK.%2BW.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D1358%26rft.epage%253D1365%26rft.atitle%253DRecognition%2Bof%2Benhancer%2Belement-specific%2Bhistone%2Bmethylation%2Bby%2BTIP60%2Bin%2Btranscriptional%2Bactivation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2153%26rft_id%253Dinfo%253Apmid%252F22081016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22081016&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.20" data-doi="10.1038/ng.322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolasinska-Zwierz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Down</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latorre</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Differential chromatin marking of introns and expressed exons by H3K36me3.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKolasinska-Zwierz%26rft.auinit1%253DP.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D376%26rft.epage%253D381%26rft.atitle%253DDifferential%2Bchromatin%2Bmarking%2Bof%2Bintrons%2Band%2Bexpressed%2Bexons%2Bby%2BH3K36me3.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.322%26rft_id%253Dinfo%253Apmid%252F19182803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182803&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263640200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.21" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.22" data-doi="10.1074/jbc.M003219200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtney</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vassilev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzilli</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">p300/CBP-associated factor histone acetyltransferase processing of a peptide substrate. Kinetic analysis of the catalytic mechanism.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">21953</span>–<span class="cit-lpage">21959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M003219200%26rft_id%253Dinfo%253Apmid%252F10777508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvMjkvMjE5NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.23" data-doi="10.1038/nsmb1135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listerman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sapra</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neugebauer</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cotranscriptional coupling of splicing factor recruitment and precursor messenger RNA splicing in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DListerman%26rft.auinit1%253DI.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D815%26rft.epage%253D822%26rft.atitle%253DCotranscriptional%2Bcoupling%2Bof%2Bsplicing%2Bfactor%2Brecruitment%2Band%2Bprecursor%2Bmessenger%2BRNA%2Bsplicing%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1135%26rft_id%253Dinfo%253Apmid%252F16921380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16921380&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241102400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.24" data-doi="10.1038/nsmb1106-952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cotranscriptional splicing regulation: it’s not just about speed.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">952</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DLynch%26rft.auinit1%253DK.%2BW.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D952%26rft.epage%253D953%26rft.atitle%253DCotranscriptional%2Bsplicing%2Bregulation%253A%2Bit%2527s%2Bnot%2Bjust%2Babout%2Bspeed.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1106-952%26rft_id%253Dinfo%253Apmid%252F17082788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1106-952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17082788&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.25" data-doi="10.1038/ng.2677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neph</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNase I-hypersensitive exons colocalize with promoters and distal regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">852</span>–<span class="cit-lpage">859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D852%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2677%26rft_id%253Dinfo%253Apmid%252F23793028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793028&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.26" data-doi="10.1371/journal.pcbi.1002968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rajagopal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RFECS: a random-forest based algorithm for enhancer identification from chromatin state.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1002968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1002968%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002968%26rft_id%253Dinfo%253Apmid%252F23526891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23526891&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23526891&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.27" data-doi="10.1016/j.cell.2011.09.057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goren</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amit</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoresh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yosef</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Combinatorial patterning of chromatin regulators uncovered by genome-wide location analysis in human cells.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1628</span>–<span class="cit-lpage">1639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRam%26rft.auinit1%253DO.%26rft.volume%253D147%26rft.issue%253D7%26rft.spage%253D1628%26rft.epage%253D1639%26rft.atitle%253DCombinatorial%2Bpatterning%2Bof%2Bchromatin%2Bregulators%2Buncovered%2Bby%2Bgenome-wide%2Blocation%2Banalysis%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.09.057%26rft_id%253Dinfo%253Apmid%252F22196736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.09.057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22196736&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298403400025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.28" data-doi="10.1016/j.cell.2013.09.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivera</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping human epigenomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D155%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.09.011%26rft_id%253Dinfo%253Apmid%252F24074860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.09.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24074860&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324916700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.29" data-doi="10.1016/j.molcel.2009.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spies</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padgett</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biased chromatin signatures around polyadenylation sites and exons.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSpies%26rft.auinit1%253DN.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D254%26rft.atitle%253DBiased%2Bchromatin%2Bsignatures%2Baround%2Bpolyadenylation%2Bsites%2Band%2Bexons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.10.008%26rft_id%253Dinfo%253Apmid%252F19854133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854133&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271801900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.30" data-doi="10.1016/j.cell.2011.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of 67 histone marks and histone lysine crotonylation as a new type of histone modification.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1016</span>–<span class="cit-lpage">1028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTan%26rft.auinit1%253DM.%26rft.volume%253D146%26rft.issue%253D6%26rft.spage%253D1016%26rft.epage%253D1028%26rft.atitle%253DIdentification%2Bof%2B67%2Bhistone%2Bmarks%2Band%2Bhistone%2Blysine%2Bcrotonylation%2Bas%2Ba%2Bnew%2Btype%2Bof%2Bhistone%2Bmodification.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.08.008%26rft_id%253Dinfo%253Apmid%252F21925322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925322&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.31" data-doi="10.1016/j.cell.2009.06.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schones</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide mapping of HATs and HDACs reveals distinct functions in active and inactive genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D138%26rft.issue%253D5%26rft.spage%253D1019%26rft.epage%253D1031%26rft.atitle%253DGenome-wide%2Bmapping%2Bof%2BHATs%2Band%2BHDACs%2Breveals%2Bdistinct%2Bfunctions%2Bin%2Bactive%2Band%2Binactive%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.06.049%26rft_id%253Dinfo%253Apmid%252F19698979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.06.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19698979&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269564600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.32" data-doi="10.1038/ng.154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenfeld</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schones</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Combinatorial patterns of histone acetylations and methylations in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D40%26rft.issue%253D7%26rft.spage%253D897%26rft.epage%253D903%26rft.atitle%253DCombinatorial%2Bpatterns%2Bof%2Bhistone%2Bacetylations%2Band%2Bmethylations%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.154%26rft_id%253Dinfo%253Apmid%252F18552846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18552846&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.33" data-doi="10.1186/1471-2105-9-547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Won</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chepelev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Prediction of regulatory elements in mammalian genomes using chromatin signatures.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DWon%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D9%26rft.spage%253D547%26rft.epage%253D547%26rft.atitle%253DPrediction%2Bof%2Bregulatory%2Belements%2Bin%2Bmammalian%2Bgenomes%2Busing%2Bchromatin%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-547%26rft_id%253Dinfo%253Apmid%252F19094206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19094206&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.34" data-doi="10.1093/bioinformatics/btr508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akerman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krainer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SpliceTrap: a method to quantify alternative splicing under single cellular conditions.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3010</span>–<span class="cit-lpage">3016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr508%26rft_id%253Dinfo%253Apmid%252F21896509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMzAxMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajagopal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Epigenomic analysis of multilineage differentiation of human embryonic stem cells.</span> <abbr class="cit-jnl-abbrev">Cell 153: 1134–1148</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.36" data-doi="10.1038/nsmb.2470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zentner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Regulation of nucleosome dynamics by histone modifications.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D20%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2470%26rft_id%253Dinfo%253Apmid%252F23463310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23463310&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.37" data-doi="10.1038/ng1891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavoosidana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sjolinder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gondor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariano</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Circular chromosome conformation capture (4C) uncovers extensive networks of epigenetically regulated intra- and interchromosomal interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DZ.%26rft.volume%253D38%26rft.issue%253D11%26rft.spage%253D1341%26rft.epage%253D1347%26rft.atitle%253DCircular%2Bchromosome%2Bconformation%2Bcapture%2B%25284C%2529%2Buncovers%2Bextensive%2Bnetworks%2Bof%2Bepigenetically%2Bregulated%2Bintra-%2Band%2Binterchromosomal%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1891%26rft_id%253Dinfo%253Apmid%252F17033624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17033624&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241592700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.38" data-doi="10.1073/pnas.1103344108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinman</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barron</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hu proteins regulate alternative splicing by inducing localized histone hyperacetylation in an RNA-dependent manner.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">E627</span>–<span class="cit-lpage">E635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1103344108%26rft_id%253Dinfo%253Apmid%252F21808035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzM2L0U2MjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIwNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2051.39" data-doi="10.1016/j.molcel.2005.09.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandal</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erdjument-Bromage</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Monoubiquitination of human histone H2B: the factors involved and their roles in HOX gene regulation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DZhu%26rft.auinit1%253DB.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D601%26rft.epage%253D611%26rft.atitle%253DMonoubiquitination%2Bof%2Bhuman%2Bhistone%2BH2B%253A%2Bthe%2Bfactors%2Binvolved%2Band%2Btheir%2Broles%2Bin%2BHOX%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.09.025%26rft_id%253Dinfo%253Apmid%252F16307923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.09.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16307923&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233525400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/11/2051.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/2049" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2065" title="Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/11"><span>Volume 4 Issue 11, November 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/11.cover-source.jpg?itok=oYWFGPiM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2051.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XcCMKCRofgkbhDF1ZYHY2jLoW_Yq2RilkqOKCNKuixA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/11/2051" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/11/2051" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/11/2051" class="active" data-icon-position="" data-hide-link-title="0">Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459715" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Dnxhgkd6yabRWZnhQrtA_GMqRTP77Jwef21QjNxlVpk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2051.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/11/2051" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VHodTJhUOzqxxaia7S7hp1DL8w2-43hsIeyvWmG-_kg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/11/2051" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2051.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459715" id="node459715--2" data-pisa="ggg;4/11/2051" data-pisa-master="ggg;g3.114.013565" data-apath="/ggg/4/11/2051.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nisha Rajagopal</span>, <span class="highwire-citation-author" data-delta="1">Jason Ernst</span>, <span class="highwire-citation-author" data-delta="2">Pradipta Ray</span>, <span class="highwire-citation-author" data-delta="3">Jie Wu</span>, <span class="highwire-citation-author" data-delta="4">Michael Zhang</span>, <span class="highwire-citation-author" data-delta="5">Manolis Kellis</span> and <span class="highwire-citation-author" data-delta="6">Bing Ren</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2051-2063;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013565 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459715--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nisha Rajagopal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Systems Biology Program, University of California, San Diego, La Jolla, California 92037</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nisha%2BRajagopal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rajagopal%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANisha%2BRajagopal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Chemistry, University of California, Los Angeles, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pradipta Ray </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pradipta%2BRay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ray%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APradipta%2BRay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jie Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Cold Spring Harbor Laboratory, 1 Bungtown Rd., Cold Spring Harbor, New York 11724</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Applied Mathematics and Statistics, Stony Brook University, Stony Brook, New York 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jie%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJie%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Bioinformatics Division, Center for Synthetic and Systems Biology, Tsinghua National Laboratory for Information Science and Technology, Tsinghua University, Beijing 100084, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Manolis Kellis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manolis%2BKellis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManolis%2BKellis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bing Ren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Cellular and Molecular Medicine, Institute of Genomic Medicine, and Moores Cancer Center, University of California, San Diego School of Medicine, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BRen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ren%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABing%2BRen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:biren@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">biren@ucsd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/11/2051" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/11/2051&amp;title=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/11/2051&amp;title=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/11/2051&amp;title=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/11/2051&amp;text=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/11/2051&amp;title=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/11/2051&amp;t=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/11/2051&amp;title=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/11/2051&amp;title=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459715/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459715" id="node459715--3" data-pisa="ggg;4/11/2051" data-pisa-master="ggg;g3.114.013565" data-apath="/ggg/4/11/2051.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nisha Rajagopal</span>, <span class="highwire-citation-author" data-delta="1">Jason Ernst</span>, <span class="highwire-citation-author" data-delta="2">Pradipta Ray</span>, <span class="highwire-citation-author" data-delta="3">Jie Wu</span>, <span class="highwire-citation-author" data-delta="4">Michael Zhang</span>, <span class="highwire-citation-author" data-delta="5">Manolis Kellis</span> and <span class="highwire-citation-author" data-delta="6">Bing Ren</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2051-2063;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013565 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459715--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nisha Rajagopal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Systems Biology Program, University of California, San Diego, La Jolla, California 92037</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nisha%2BRajagopal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rajagopal%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANisha%2BRajagopal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jason Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Chemistry, University of California, Los Angeles, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pradipta Ray </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pradipta%2BRay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ray%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APradipta%2BRay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jie Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Cold Spring Harbor Laboratory, 1 Bungtown Rd., Cold Spring Harbor, New York 11724</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Applied Mathematics and Statistics, Stony Brook University, Stony Brook, New York 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jie%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJie%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular and Cell Biology, Center for Systems Biology, The University of Texas at Dallas, Richardson, Texas 75080</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Bioinformatics Division, Center for Synthetic and Systems Biology, Tsinghua National Laboratory for Information Science and Technology, Tsinghua University, Beijing 100084, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Manolis Kellis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manolis%2BKellis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManolis%2BKellis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bing Ren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ludwig Institute for Cancer Research, 9500 Gilman Drive, La Jolla, California 92093-0653</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Cellular and Molecular Medicine, Institute of Genomic Medicine, and Moores Cancer Center, University of California, San Diego School of Medicine, La Jolla, California 92093</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BRen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ren%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABing%2BRen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:biren@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">biren@ucsd.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2051&amp;rft.epage=2063&amp;rft.atitle=Distinct%20and%20Predictive%20Histone%20Lysine%20Acetylation%20Patterns%20at%20Promoters%2C%20Enhancers%2C%20and%20Gene%20Bodies&amp;rft.volume=4&amp;rft.issue=11&amp;rft.date=2014-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rajagopal%2C+Nisha&amp;rft.au=Ernst%2C+Jason&amp;rft.au=Ray%2C+Pradipta&amp;rft.au=Wu%2C+Jie&amp;rft.au=Zhang%2C+Michael&amp;rft.au=Kellis%2C+Manolis&amp;rft.au=Ren%2C+Bing"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459715/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459715/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459715/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459715/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459715/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459715/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459715/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459715/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459715/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459715/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459715/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459715/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/11/2051" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/11/2051.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/11/2051.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/11/2051.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/11/2051.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
