<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/11/2137" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/11/2137.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/11/2137.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/11/2137.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/11/2137/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/11/2137.atom" />
<meta name="HW.pisa" content="ggg;4/11/2137" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1534/g3.114.013052" />
<meta name="DC.Date" content="2014-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Thakur et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="RNA interference (RNAi), mediated by the introduction of a specific double-stranded RNA, is a powerful method to investigate gene function. It is widely used in the Caenorhabditis elegans research community. An expanding number of laboratories conduct genome-wide RNAi screens, using standard libraries of bacterial clones each designed to produce a specific double-stranded RNA. Proper interpretation of results from RNAi experiments requires a series of analytical steps, from the verification of the identity of bacterial clones, to the identification of the clones’ potential targets. Despite the popularity of the technique, no user-friendly set of tools allowing these steps to be carried out accurately, automatically, and at a large scale, is currently available. We report here the design and production of Clone Mapper, an online suite of tools specifically adapted to the analysis pipeline typical for RNAi experiments with C. elegans . We show that Clone Mapper overcomes the limitations of existing techniques and provide examples illustrating its potential for the identification of biologically relevant genes. The Clone Mapper tools are freely available via &lt;http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html&gt;." />
<meta name="DC.Contributor" content="Nishant Thakur" />
<meta name="DC.Contributor" content="Nathalie Pujol" />
<meta name="DC.Contributor" content="Laurent Tichit" />
<meta name="DC.Contributor" content="Jonathan J. Ewbank" />
<meta name="article:published_time" content="2014-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;RNA interference (RNAi), mediated by the introduction of a specific double-stranded RNA, is a powerful method to investigate gene function. It is widely used in the &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; research community. An expanding number of laboratories conduct genome-wide RNAi screens, using standard libraries of bacterial clones each designed to produce a specific double-stranded RNA. Proper interpretation of results from RNAi experiments requires a series of analytical steps, from the verification of the identity of bacterial clones, to the identification of the clones’ potential targets. Despite the popularity of the technique, no user-friendly set of tools allowing these steps to be carried out accurately, automatically, and at a large scale, is currently available. We report here the design and production of Clone Mapper, an online suite of tools specifically adapted to the analysis pipeline typical for RNAi experiments with &lt;i&gt;C. elegans&lt;/i&gt;. We show that Clone Mapper overcomes the limitations of existing techniques and provide examples illustrating its potential for the identification of biologically relevant genes. The Clone Mapper tools are freely available via http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/11/01" />
<meta name="citation_mjid" content="ggg;4/11/2137" />
<meta name="citation_id" content="4/11/2137" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/11/2137" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/11/2137.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/11/2137.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/11/2137.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.013052" />
<meta name="citation_pmid" content="25187039" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2137" />
<meta name="citation_lastpage" content="2145" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nishant Thakur" />
<meta name="citation_author_institution" content="Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France" />
<meta name="citation_author_institution" content="INSERM U1104, 13288 Marseille, France" />
<meta name="citation_author_institution" content="CNRS UMR7280, 13288 Marseille, France" />
<meta name="citation_author" content="Nathalie Pujol" />
<meta name="citation_author_institution" content="Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France" />
<meta name="citation_author_institution" content="INSERM U1104, 13288 Marseille, France" />
<meta name="citation_author_institution" content="CNRS UMR7280, 13288 Marseille, France" />
<meta name="citation_author" content="Laurent Tichit" />
<meta name="citation_author_institution" content="Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France" />
<meta name="citation_author" content="Jonathan J. Ewbank" />
<meta name="citation_author_institution" content="Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France" />
<meta name="citation_author_institution" content="Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France" />
<meta name="citation_author_institution" content="CNRS UMR7280, 13288 Marseille, France" />
<meta name="citation_author_email" content="ewbank@ciml.univ-mrs.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1257-6862" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=BMC developmental biology [electronic resource];citation_journal_abbrev=BMC Dev Biol;citation_author=J. Ceron;citation_author=J. F. Rual;citation_author=A. Chandra;citation_author=D. Dupuy;citation_author=M. Vidal;citation_title=Large-scale RNAi screens identify novel genes that interact with the C. elegans retinoblastoma pathway as well as splicing-related components with synMuv B activity.;citation_pages=30-30;citation_volume=7;citation_year=2007;citation_pmid=17417969;citation_doi=10.1186/1471-213X-7-30" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=I. Engelmann;citation_author=A. Griffon;citation_author=L. Tichit;citation_author=F. Montanana-Sanchis;citation_author=G. Wang;citation_title=A comprehensive analysis of gene expression changes provoked by bacterial and fungal infection in C. elegans.;citation_pages=e19055-e19055;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21602919;citation_doi=10.1371/journal.pone.0019055" />
<meta name="citation_reference" content="citation_journal_title=Nat. Cell Biol.;citation_author=B. T. Fievet;citation_author=J. Rodriguez;citation_author=S. Naganathan;citation_author=C. Lee;citation_author=E. Zeiser;citation_title=Systematic genetic interaction screens uncover cell polarity regulators and functional redundancy.;citation_pages=103-112;citation_volume=15;citation_year=2013;citation_pmid=23242217;citation_doi=10.1038/ncb2639" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. I. Gent;citation_author=A. T. Lamm;citation_author=D. M. Pavelec;citation_author=J. M. Maniar;citation_author=P. Parameswaran;citation_title=Distinct phases of siRNA synthesis in an endogenous RNAi pathway in C. elegans soma.;citation_pages=679-689;citation_volume=37;citation_year=2010;citation_issue=5;citation_pmid=20116306;citation_doi=10.1016/j.molcel.2010.01.012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. Grosswendt;citation_author=A. Filipchyk;citation_author=M. Manzano;citation_author=F. Klironomos;citation_author=M. Schilling;citation_title=Unambiguous identification of miRNA: target site interactions by different types of ligation reactions.;citation_pages=1042-1054;citation_volume=54;citation_year=2014;citation_pmid=24857550;citation_doi=10.1016/j.molcel.2014.03.049" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Hu;citation_author=C. Roesel;citation_author=I. Flockhart;citation_author=L. Perkins;citation_author=N. Perrimon;citation_title=UP-TORR: online tool for accurate and up-to-date annotation of RNAi reagents.;citation_volume=195;citation_year=2013;citation_pmid=23792952;citation_doi=10.1534/genetics.113.151340" />
<meta name="citation_reference" content="citation_title=Bioinformatics;citation_title=Sequence comparison tools;citation_author=M. Imelfort;citation_pages=13-37;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=R. S. Kamath;citation_author=J. Ahringer;citation_title=Genome-wide RNAi screening in Caenorhabditis elegans.;citation_pages=313-321;citation_volume=30;citation_year=2003;citation_issue=4;citation_pmid=12828945;citation_doi=10.1016/S1046-2023(03)00050-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. S. Kamath;citation_author=A. G. Fraser;citation_author=Y. Dong;citation_author=G. Poulin;citation_author=R. Durbin;citation_title=Systematic functional analysis of the Caenorhabditis elegans genome using RNAi.;citation_pages=231-237;citation_volume=421;citation_year=2003;citation_issue=6920;citation_pmid=12529635;citation_doi=10.1038/nature01278" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT—the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Montojo;citation_author=K. Zuberi;citation_author=H. Rodriguez;citation_author=F. Kazi;citation_author=G. Wright;citation_title=GeneMANIA Cytoscape plugin: fast gene function predictions on the desktop.;citation_volume=26;citation_year=2010;citation_pmid=20926419;citation_doi=10.1093/bioinformatics/btq562" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. Parrish;citation_author=J. Fleenor;citation_author=S. Xu;citation_author=C. Mello;citation_author=A. Fire;citation_title=Functional anatomy of a dsRNA trigger: differential requirement for the two trigger strands in RNA interference.;citation_pages=1077-1087;citation_volume=6;citation_year=2000;citation_issue=5;citation_pmid=11106747;citation_doi=10.1016/S1097-2765(00)00106-4" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=R. Pukkila-Worley;citation_author=R. L. Feinbaum;citation_author=D. L. McEwan;citation_author=A. L. Conery;citation_author=F. M. Ausubel;citation_title=The evolutionarily conserved mediator subunit MDT-15/MED15 links protective innate immune responses and xenobiotic detoxification.;citation_pages=e1004143;citation_volume=10;citation_year=2014;citation_pmid=24875643;citation_doi=10.1371/journal.ppat.1004143" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. Qu;citation_author=C. Ren;citation_author=Y. Li;citation_author=J. Shi;citation_author=J. Zhang;citation_title=Reliability analysis of the Ahringer Caenorhabditis elegans RNAi feeding library: a guide for genome-wide screens.;citation_pages=170-170;citation_volume=12;citation_year=2011;citation_pmid=21453524;citation_doi=10.1186/1471-2164-12-170" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Reboul;citation_author=P. Vaglio;citation_author=N. Tzellas;citation_author=N. Thierry-Mieg;citation_author=T. Moore;citation_title=Open-reading-frame sequence tags (OSTs) support the existence of at least 17,300 genes in C. elegans.;citation_pages=332-336;citation_volume=27;citation_year=2001;citation_issue=3;citation_pmid=11242119;citation_doi=10.1038/85913" />
<meta name="citation_reference" content="citation_title=Algorithms in Bioinformatics;citation_title=mpscan: fast localisation of multiple reads in genomes;citation_author=E. Rivals;citation_author=L. Salmela;citation_author=P. Kiiskinen;citation_author=P. Kalsi;citation_author=J. Tarhio;citation_pages=246-260;citation_year=2009" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=S. H. Roy;citation_author=D. V. Tobin;citation_author=N. Memar;citation_author=E. Beltz;citation_author=J. Holmen;citation_title=A complex regulatory network coordinating cell cycles during C. elegans development is revealed by a genome-wide RNAi screen.;citation_volume=4;citation_year=2014;citation_pmid=24584095;citation_doi=10.1534/g3.114.010546" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. F. Rual;citation_author=J. Ceron;citation_author=J. Koreth;citation_author=T. Hao;citation_author=A. S. Nicot;citation_title=Toward Improving Caenorhabditis elegans Phenome Mapping With an ORFeome-Based RNAi Library;citation_pages=2162-2168;citation_volume=14;citation_year=2004;citation_issue=10b;citation_pmid=15489339;citation_doi=10.1101/gr.2505604" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. F. Rual;citation_author=N. Klitgord;citation_author=G. Achaz;citation_title=Novel insights into RNAi off-target effects using C. elegans paralogs.;citation_pages=106-106;citation_volume=8;citation_year=2007;citation_pmid=17445269;citation_doi=10.1186/1471-2164-8-106" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=R. Saito;citation_author=M. E. Smoot;citation_author=K. Ono;citation_author=J. Ruscheinski;citation_author=P. L. Wang;citation_title=A travel guide to Cytoscape plugins.;citation_pages=1069-1076;citation_volume=9;citation_year=2012;citation_issue=11;citation_pmid=23132118;citation_doi=10.1038/nmeth.2212" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. E. Smoot;citation_author=K. Ono;citation_author=J. Ruscheinski;citation_author=P. L. Wang;citation_author=T. Ideker;citation_title=Cytoscape 2.8: new features for data integration and network visualization.;citation_volume=27;citation_year=2011;citation_pmid=21149340;citation_doi=10.1093/bioinformatics/btq675" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=B. Squiban;citation_author=J. Belougne;citation_author=J. Ewbank;citation_author=O. Zugasti;citation_title=Quantitative and automated high-throughput genome-wide RNAi screens in C. elegans.;citation_pages=e3448;citation_volume=60;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Timmons;citation_author=A. Fire;citation_title=Specific interference by ingested dsRNA.;citation_pages=854-854;citation_volume=395;citation_year=1998;citation_issue=6705;citation_pmid=9804418;citation_doi=10.1038/27579" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=L. Timmons;citation_author=D. L. Court;citation_author=A. Fire;citation_title=Ingestion of bacterially expressed dsRNAs can produce specific and potent genetic interference in Caenorhabditis elegans.;citation_pages=103-112;citation_volume=263;citation_year=2001;citation_issue=1-2;citation_pmid=11223248;citation_doi=10.1016/S0378-1119(00)00579-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Yook;citation_author=T. W. Harris;citation_author=T. Bieri;citation_author=A. Cabunoc;citation_author=J. Chan;citation_title=WormBase 2012: more genomes, more data, new website.;citation_volume=40;citation_year=2012;citation_pmid=22067452;citation_doi=10.1093/nar/gkr954" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Zhou;citation_author=F. Xu;citation_author=H. Mao;citation_author=J. Ji;citation_author=M. Yin;citation_title=Nuclear RNAi contributes to the silencing of off-target genes and repetitive sequences in Caenorhabditis elegans.;citation_volume=197;citation_year=2014;citation_pmid=24532782;citation_doi=10.1534/genetics.113.159780" />
<meta name="citation_reference" content="citation_journal_title=Nat. Immunol.;citation_author=O. Zugasti;citation_author=N. Bose;citation_author=B. Squiban;citation_author=J. Belougne;citation_author=C. L. Kurz;citation_title=Activation of a G protein-coupled receptor by its endogenous ligand triggers the innate immune response of Caenorhabditis elegans.;citation_pages=833-838;citation_volume=15;citation_year=2014;citation_pmid=25086774;citation_doi=10.1038/ni.2957" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
<meta name="twitter:description" content="RNA interference (RNAi), mediated by the introduction of a specific double-stranded RNA, is a powerful method to investigate gene function. It is widely used in the Caenorhabditis elegans research community. An expanding number of laboratories conduct genome-wide RNAi screens, using standard libraries of bacterial clones each designed to produce a specific double-stranded RNA. Proper interpretation of results from RNAi experiments requires a series of analytical steps, from the verification of the identity of bacterial clones, to the identification of the clones’ potential targets. Despite the popularity of the technique, no user-friendly set of tools allowing these steps to be carried out accurately, automatically, and at a large scale, is currently available. We report here the design and production of Clone Mapper, an online suite of tools specifically adapted to the analysis pipeline typical for RNAi experiments with C. elegans . We show that Clone Mapper overcomes the limitations of existing techniques and provide examples illustrating its potential for the identification of biologically relevant genes. The Clone Mapper tools are freely available via &lt;http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html&gt;." />
<meta name="og-title" property="og:title" content="Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/11/2137" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="RNA interference (RNAi), mediated by the introduction of a specific double-stranded RNA, is a powerful method to investigate gene function. It is widely used in the Caenorhabditis elegans research community. An expanding number of laboratories conduct genome-wide RNAi screens, using standard libraries of bacterial clones each designed to produce a specific double-stranded RNA. Proper interpretation of results from RNAi experiments requires a series of analytical steps, from the verification of the identity of bacterial clones, to the identification of the clones’ potential targets. Despite the popularity of the technique, no user-friendly set of tools allowing these steps to be carried out accurately, automatically, and at a large scale, is currently available. We report here the design and production of Clone Mapper, an online suite of tools specifically adapted to the analysis pipeline typical for RNAi experiments with C. elegans . We show that Clone Mapper overcomes the limitations of existing techniques and provide examples illustrating its potential for the identification of biologically relevant genes. The Clone Mapper tools are freely available via &lt;http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11.cover-source.jpg" />
    <title>Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Of6VQA-dnVnT7dUkR9N_IOG6VjHfcUjeE5NyPn2oqmc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459721","apath":"\/ggg\/4\/11\/2137.atom","pisa":"ggg;4\/11\/2137","ac":{"\/ggg\/4\/11\/2137.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/11\/2137.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2137"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/11\/2137"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/11\/2137.supplemental":true,"\/content\/4\/11\/2137":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mohkWd0oSRla9qc\\\\\\\/qPaEi5rraXllhLFnYKiVvrl8oETRNPU1w0k5E8dVsKs6jT\\\\\\\/\\\\\\\/rbiQfWX1782LH8m\\\\\\\/lYEnwym6BKqM6N8lZySGRxt9A8npR2YtrNbBS3ep2PrfXNa9mSrayTKZRLpUnC4qMG1ZNIVZEmKirtQmXM\\\\\\\/ZZil\\\\\\\/Ebwu+8mccPSHtANT2J6hOekZLiNMUs3TzxTr7z3TvS1je6LixdDcee0qlxJfyrWbpa6jmcCz2mDy2MpQedmFr9bV5PxO\\\\\\\/rGSFf1XfQ3LMDMD8gvC+L8cdBmq2dkdXyRHGmWIwZcnYBaB2lxknumJLiGg4WynB0LRRhCVC1CI0GpcJQ8IhbxKn5lLNDIcM+EEi20g8EffdzY2NF0dcvvzt\\\\\\\/KdrqEsXQgTmNf\\\\\\\/Sjm4s4Ko+Yq3GpPf20uFDN9YQ0+848\\\\\\\/7UPTVzi1B5DoXJywYWB1\\\\\\\/eqdh5ip6\\\\\\\/dqLIydmlG8RCL0OC2rnJJejLa3rQMRUZ+5Spuh0ttvbXWy2pUI7lSV6AkgJMdUcGSiLYjXw\\\\\\\/ZboDm2gRvgTFNeIZHacMMlrHQN1lUwnbgMM01hkLLITjj5nIc0YBvvUcLT4ayvDEXtxaQVRg3HxXFOVpGQ1BrelY8bLa4YeBTbisbxxFvZBpQue6cFTkZCD21vGFqiqVggSlFFtGA\\\\\\\/q5bZgcK9NLTU2j1oASNFG38UgRhcq0go0R7b0dADyOIZ+Rk5AU+\\\\\\\/LNzZmhUdJbO\\\\\\\/1TUHMk6VvDrBIYMj8vNR7rV2QmLvHevLHUVTY6yxqtQPM9UhLSGzIe269WAT2USm9Tc4HIhjmaQYwQ8\\\\\\\/Ycbi7Q9wMyM430QspvyYPBJq1nAo7BZQ80Nhxh8TSFcUly2SAavulFcs0MutbtBJFnLMpOPzzAlay\\\\\\\/P9UhUBJa\\\\\\\/KRLO\\\\\\\/wq1dBNgu5PeAk42LPl5J3NqS25MrysfRsCSBLUZZk9dIDIseDxMBQswxqcMQhydCtZT+l4vBdXs93Kjk8W1GN\\\\\\\/EZfMSmsoUTYQsqVMo8lQxxyxhxDKi8enj1d0Sm2qGBXav6A1V7v0AHeTqi5Sj\\\\\\\/DkX6hqMfL8AtSFiI4k4rxwc+TiAP2KzVggxvyzzupBDIJWshR+xZ2IpBm+a2IswmQWQmEJjCwo1ErTdk2VClSP\\\\\\\/bHga9EneEF4GC8japSjqiEvJO1pvCK5GbC24+17qSSaWhjO8sMxv9y1vk\\\\\\\/6DLDT5MeIxINXToIbtq6IxG8xsyRJJ\\\\\\\/UehpXE7L68byVpkGDbtaiSpiDz69jFFBnD0QTHhLHsAe8U9NU8fJ6\\\u0022,\\\u0022iv\\\u0022:\\\u0022vFYVp4cZDvujCBqZSXLleg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022efb35080f78c27588ada046da819c03ab6190d5d7a5d07e894f979948f07c9e8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TThDw7SqhnSSA0hDZwzVbEyJk7r8z7pUKZ4jHRYn9k4EL67xD6afnoSbH9sE2p70gzm5nBtpUpgniGkjsAVT0skTkTQdI3e18ifzvjKU4zyHGg7jhlZ5pp+mCiWnbIAqU6DXOIjg28gdKrRMiaoz7Ux9yb+fFLyocwCR2SNk0rMR1buxmC79RJ+kOt+CEhfr9PeAtatSvTBHmY5yGnU78kC+FYi\\\\\\\/QQTAxow+wkYHAwkfxl8dZzGfZCOyYiCxiM0EsYJ4lCCWK6ZEFnerGJgKskNdKYylxqjXCom7ntN47SFWUuZZwh\\\\\\\/EftWdAdTx0+ujiRz23ZKBZiyHf0wjOVa7GsZME8h8kphtW1fqIi6bTOmh+ygbkruAcaK+avwiGuBSUZIdISVVRIGFWDZutXSu2\\\\\\\/Iibcrz1pKzNvbW2KseSbTzhX+bklPXNJDhfzGdtq3IXZd92TMD7UjtjQOu5DAm4L1MThc0a1a6nbfrzcJ1+4w+pgg6v7fjCh81I9jUodQ48Cxca8DaD3Hb+5caZVdsx2J3RYgDGJsqHdNwFqEt6m8YyT6uKV1SGpoZvr6t\\\\\\\/do50i4ZaX6Rvop6c3sPm+ZPWqaatSPbWVCM+JeONzDCnHbg3e7gei7YCQnagNb75P1cQYCSXEa1sO28ybKnBCP5q4J\\\\\\\/ajQX+vjNl3BWgpuBxCyLIgmgtCfh6vXalEQ2KuGfI1CgFhLxxXz84D66Zdkdr7ApZveMU7Qr6qOtrB3jpS8ZmEzBnOCOw3WFu6I4L0oPKgIxNPEpW69uoXEC50Ipnsp2g6bK+rxNxBywRGA25lxJLXWKS37Sd7eU6MqJ9t04gxnd2W56NcCG\\\\\\\/X1xTLAXAWX+rKEukq2jcEwAUZKTiyxGlrtqX8sDavg1cR\\\\\\\/jekDDmY93Wi93uloqoYcVoIynh6EFPM57x7fnNBbJ23n8L5hlZfiaWNUl2BkQblPokYSLL++LvPAL55OUabV+HojuEKR5IU4RbHZPjFXjb2J8LshCS1Pmo4lNaIFVgK2YWisEKuCGmgGND78hsLWFPO5jcdQzaypMki7nrV0tMaLPxkC9Vy\\\\\\\/Q5S7TPKcMgIO2V0U37losfnKIQwzLka4kwQShZqKHuHJX+OwiOFI06q+y8q3ybVUHJmqY5BJvC2u1cyF17wMmMUvmpgSbxrZcBdeNUSqf0DKT+2r7X8yODuEpBohxItzc8UG\\\\\\\/5GbjnzweDgeZiZ3SOI13HWsoBomWXvC3HkN2EbhuvOJKH7v+wr5yotilEURzqjkPi0vMDJS85nsTwmqgWhZQnnqsyHs56wqIolUkWWrHyiuLGrYf\\\\\\\/28viwCfBnN4YDOWo07p1gl22a50wQYtch5k07pkqEdYhrIHlPvMvsjJveGBBqsknazn2iixxW+AMWbEqxTrDZ47Axa9DJZGhUjMWQD9txOh82\\\\\\\/NkS8lHmIz6UKmBclC2tpMgfiMs4tetWhdfiwJH6IFGAOlQmILCH0nXb6\\\\\\\/1A5QrHh6fBeX40pzikveiqIWx8nFs0QDxSmAcMd+oOF5A45AIh3x\\\\\\\/MVc36Qp\\\\\\\/a\\\\\\\/fkQc67veBV7hlPYPsUXgWPEm1puXP0VeVrUDH8\\\\\\\/g22TYdiTe3IUQt6DUQvrAdEd8SCuiL4Eeq+VrHLGOCLJT1joFuafQaWJ4MNqddEY46BsPpaeufs5+iTG3x2v9jrloulm5YbgKZNz4SI+gQnyy5\\\\\\\/Zo4ZW6ja4+3Vbi412APv5U+nm3WT5+uIViFI30zG334gC0Yd+9VeVpzIIGXuVKowzD3yr5UXSQRhp3nqMjsCEUAnAynp9bSyYpKibnJ3VWpaqFQW92YpsQgwowuY91tSYW+hTuN4py4u+YQ76E+g\\\\\\\/OrZvnwnOiYD4\\\\\\\/SLOd9sVmHaJih7RKi3a76X6fzo1UeLG5xBkQYQHRnEyMUv1oKc1+S47tsrbVAJLyhQEy62Mq92BdY0+N1dxYTo0fInm6Zk3C6WHjzhQnU\\\\\\\/dwAw0uMe7AsMo+TJPbSrevRo5xMthIk+bQ03XVQEV\\\\\\\/i0GM2076prUgUaCyE2tJ63lMzEirlR286QqWi9\\\\\\\/7AAxlQbGEYXZsbv2qfo0FBeMwb8nXuZdkRfG3Ou1nW+IuUFRRofzXw19zqTgo6dtZToXjA9Pmz4HVqHwXr1NSUUqDJ1D5iVvIP27eOqmup51XPY2JNJUJWUhcHxWtPNW9VsLPubV2Yv8xwc988kapHzxWi7kxfrqrFyL9IKu25oPUcofr3JFuCmNllHPOLcHWlA9GJdkLlPd+1UQrk\\\\\\\/CK\\\\\\\/efCH4jECSMgtIyGirJa7MzQ+S\\\\\\\/exOrl4ELCOFdNTU\\\\\\\/\\\\\\\/BdHcTidEul0pF\\\\\\\/81jIvPmFp7Y3tAs99MI6wb8vhM6A1AMChsdFDEvgez7PzlzGyEril+RZ2MsLQzuLSBmRhKpO8Uto8qRuEIjVJ1SGrLRn3yA45WN1qmkCiBMNOs7t3cmrS+6U8K3\\\\\\\/6JARNISyQM0ag8qa6va5Wl+NC4UATFDnujZAt\\\\\\\/KWq9Nw3X6DylbFjOSBp9XhxG+dI8VZZKF4i2kGzZI6bVHIr7+UUeHN5iH37iCwyCRd4EimBCE+0d1jvWskfqG17XQmmlAbk4H00zNXgMRzqBipZxbBVs1uYuLSCojVUX4Eh\\\\\\\/N1fDzzkr0lqUKzgiHdRiuQb7aq4EWpAUT23DGAhT20Y3aKH9U2BAz79ZHqfm3dNcma8rSUA1LKvoY4E3JxE8eXm7UliX26M\\\\\\\/+g9yCRpN3kqTprrbVikqZ84YHEzYS5AMiLrIamni+m\\\\\\\/KzYi3wGLwhqYIxq0BTpAmpigMHmjNLvunFva6M6jrBybsuhEn0UhZmzGoMKq1oCAAI3jsH4Jhjecz6+C\\\\\\\/0VVgIie92pU4pp\\\\\\\/XT0Itle2u3VRqpOh7lHmIfvkVhD1XoL6m66ZaX\\\\\\\/Eb2U\\\\\\\/UJAbM44X8Uzg5mSI0B+ptbJRYJhubNgR7bZKyGhLyvEdKxsUO4x5K0n2HEs+Gw6bhtCOiEBZP0jYzXBZnzGsYojC9qM\\\\\\\/GkRqqcDy9WsQBjzMWPj\\\\\\\/Sdrp1DCh80Vs\\\\\\\/eZtYk6JlNqBPcpZCOlElSZSwFgv4JJH8pLYdieYcRouD4xKpOrQ0uuFt2PsV50AKfgTJ8Ha7Gqo0CHZjU+vzKt1r9dQeABJiV6CKfLgTmqNefFJT0DUq9U4cYlDExPQcbCX1aviPfZ11OEZ1pdSKPkDAm18M\\\\\\\/iMstiqC6ClBVkFoSXj5gxzP1q3M8kpjuKzmbG6NERAVcrRTnSqndktNehp\\\\\\\/IaguaFDEHHbtYYBkbCfUqCORmn15p2t8iTGiRvT7Reny0KhGIlGdfagZm9wBKbL7hl8g7M3YiT6wHzR7ZT87bRnCFtxeILwC4L2YZriPNUjpqZPO6bARHiPyVGoYDUdRGRzZ\\\\\\\/zSA98ZUCbisfOPjMC5wIXKhPCmfoU4HSNG91mjrutJej6O9zvps0lT2baJDUd8HnbSxpZZmItqSJPjQLb0TkC49h6oDuGlQzl\\\\\\\/C4lLLpKPZRq+gG8SiBD2xQptwdU7ZBHeFUcLsXbrSA6q6PjuN5uPaar\\\\\\\/3qp5gpS4q7Di4i235OUdbI4Vu8GIz0CD3Q2hsaeT1icgbXOQWhxva7xssO4EHQfn9mlLx5sga61BgfDjb1QgemkYJzuJDhdVPCXisE8i5XLHZ0T1xHJxBXtTeaauqzp4T1j9uGFIk1NeV0ob7Hwqsm2sm5yrClJYH3me0z8wkxaQKE\\\\\\\/x892bxOnxbXThEAtfGK9t40EgGt8rjBTbiNBQGmcLiMpSQVBXkLQVETjoy7F5XnvRphNYJOVqdNF+Mxw2yuHys5OK+pS\\\\\\\/HmbFB0g\\\\\\\/nEhrrPe3azBqe6yg01rGXkfdqlu5oT5xrRQ33fRWNHrIzclEUIZtGQuRc1bH4SWPrPuWbHj79kscoeMMLa3i4WkM7bCuVeO+xWki1wW45fJ4CN5p\\\\\\\/nWBKbXUvYyXDZT1SvJpZUNX+BX6wKmA211\\\\\\\/dtcfp2cCi9v0YYl22EDX24wKfMiB8kWYqyfKwymkbw8ipbh7qlGaHvV29il6fqKwhkL38vvt9WRbOLfHoN1YiO\\\\\\\/p4EGo+4pdE2nTRJJSeO3PznDV5kf9nNBEIoaMybZrO8VjRraiCJTU0nyozFuUujnpeR2Z+gUm7kB7ipv9qBNYneKsvxa4UAo9fM7tvgdHaw7\\\\\\\/zFsZrtoF\\\\\\\/kDIRUIf\\\\\\\/Jx3MEp++Z7in7+FXh5b87zHNPU\\\\\\\/4sQTG8imSe3uxllzwsyTpJcBEzOTYG\\\\\\\/RonVNWJLAAUggO04or2kL0nnC97kBTL4ZPqGg8pKwLXZtJdC6sygQLVsE+z5iZS0mG7Rtkp0NLcPMZl212BSlDWm1LF\\\\\\\/X\\\\\\\/\\\\\\\/V+AUgjOtt5ZXNg0upI27BdEei7rzGFWSCMhSknu0o2UtLl8A+yTog8g0uFPqMDxqjyJJ8w9HffeepKRrKU+VTqyxzNVA1AMS2rgRma1gKFFT6FHXgfHIvMN5DI6uIdfpraTpESDiJX1Vix3bg+7FcVZLp+Bqjt8zXQ\\\\\\\/GS0SNczVdAtqdIMi5A3at3oZN4YhUaFBfuuJSWUtPd92u1RwCRaa47W6UszIfettaedwm8tNv5sDqF3qejpUXjcpq\\\\\\\/5JmVkMb6Xg9vxwJsqw7hTkRefVUtel\\\\\\\/3qBxwSYCBBfez53I\\\\\\\/pjr4\\\\\\\/q\\\\\\\/aaorN1bi1bidKZhRxN7hK2dyjCWkiUjAm9bOg0FTUt8r1kg24U2qJb\\\\\\\/3NZoT0hXz0DdN4RO+jCWjPEsuqe29BsgdWMeyCazPQaGsZaVLTgcmb1k5dYvS2FG\\\\\\\/oUX74enLRif3pDQvEPpGJp0FEfbrcxgxDnHpVU0vaHtW6rUpN9YgXCwJK9IOCDs6PmmBXJshuFuqXGA4wgMYYJsTeMDBbtfQuh+tXec46ZqvdxTmCnXUGKtHn\\\\\\\/z7Dt6DkPUovjGPozrthm7OZVw3nJLpJ3OIkQ\\\\\\\/Bx5R5IzOdl48LwZM\\\\\\\/dollMy5GUnM8xRbZ8B8L4H4h6H8jsLixFtENJxKZA+tan1GHII\\\\\\\/1JjqL75pIfcjD67WsoO5z0r8c0T8qrhHdAN2lYOl2Yf9reEY15tftSuSI30MDiAki2jB1K4jx10KRg66fhkNhpn949NM8MU94aMVxf6PrHQTbJ1IRV6hsxbobazstpo1y9\\\\\\\/DnDhUYAh0F+ToK9+MrrmDV0KXtbjudvw6mgnNqD\\\\\\\/uFOrFTAtBzk5p6ZeJ87DUwOeFtKvnJOzFSyLfbSqc\\\\\\\/7Qw3uPvNYc6bCgA1uZi\\\\\\\/VCt4cAKruWn3cS6nn0cA5j9lt136PcLYfpXZVWX5r1BgoPNBqFqG\\\\\\\/Pwj7lTWcU2Fj\\\\\\\/tIjghGYQC7QtBiOiYpYsXG9o6xf7VOL2M2JEIavDVGdafKI7AOIfUqAVj4+MMDmwr7siwyBfeJ\\\\\\\/aHo\\\\\\\/o1swYsYly+YZiAtV+kHXcWXxvxe15na2WRODE5fjmIIyI5W\\\\\\\/\\\\\\\/q59La12nUY1a\\\\\\\/pjwyei+lVQCPxFxmEQWQ4\\\\\\\/NEfw9HG2sIG9FrpxZrW23fyoW8p+F0k\\\\\\\/0VcKIJL1Sw3M\\\\\\\/ogGPN\\\\\\\/PPvazJEcfXWJ3EoTP8f8pmpOKKRt7JrHmtSyyFWVYYjtTm1MvMgASJ13Eer1xY6RTIwIzPDNoZ2XwkeTgZEsLN1awBEQs6MGjKz9zNFYlWEywadexB2yVgJr2E+LR3bT4nTvA9S\\\\\\\/AxkWbgDSbY+f0CpGldBXX1kf9n6mCSsJX07PRaKrL0lMI3Yfa5nZENdRgYhEeCSi6bizrx5zG9kqrYXZjXAHLNtQQttQil36lEXEsd4o0At7w1QdLQvGiLOdBRtxXmOn\\\\\\\/zzCoV8XS4\\\\\\\/cHcwwAxuGCKanwLKWxZQrMAXJUKEUeFjnluuL4Gc+VIhBVDgEk\\\\\\\/iWkwWYFAtpRwrFMqpp+H0Czh24rC\\\\\\\/DDLV0zaXyQUas1oNmJOoQ+7d3PQLE9Hmo9vMFETlTdWOcWryb9IvvuxTKzjGJ54Tf\\\\\\\/fV3sWX1u84ncpZHLnDCCWJHRx5jPaG+86V\\\\\\\/B4V4ZCVJrgeTWPC2JBd9d6nt4CM44bk5Z6KEpTls5GSvPBaOnIyEyIAXTgKiYeK7YW7cyhNGWfhB+hxgRWyJf4QyHyUy7FYGKcYZD3IqAEXmZretIRFjwX+wri3SzfTo4+QjVs6op\\\\\\\/gNkI2laaNu+v+L30kCEVfOb\\\\\\\/qjmgqMrU6InHOAl\\\\\\\/5S58z2J+GWvrmXaJpIf5KO6fSVWy2woLMQnjXDV+Avr1JhSgl1QWLkCLFOSDlcJGCJA0mKgniNhQ9CxB59I5lXrefB8PxMGLv4SqokfNCdI1Y4QBKoOhYdr1I8rnkKnxenuvEgh5K4lGs0qG+G+c9mnwOGHAyhhdo8Bf7Nzl0dPmXMmAzsoIlfxZGdpdqjsIHsn7AEVZm0F4jSjJay+3TBiM0ZYSQFq\\\\\\\/gT5sBNIHutU1707PHX\\\\\\\/bD1GI9nY08pZCAaeVIHvg34dcCqXrW+Jhstd3Idt4e1ULP2FDHbbEW0pLC810yAqJbAjzolNfRkH5FsWOji1b5tZR3KCV9J2IkbxGSdqeWZzk4SnVRX404clVQ08Tra5IHkdBawebl0T+e5Z7uWUhrSGakt2O0vHQGlCSJD6Z2JHyDzMk9A2R+pyOJezaqK9yIBsZQvnwPw+32ZOn4ZRiz1idGDIE0cTK7OSAsO8AGoDoSVbqrO8UdW76U3SDa9xM96LPTBP\\\\\\\/mmc\\\\\\\/RIygV0PCEhyoWXW0tw2nVqWbibU0unEONXKc4tpaNCIqNwBeZ8+N4RyYdWyra2nVCPjMsTgC1dReMky8GE\\\\\\\/cNeYyeHHcTx96XH+GmjSwoOmnst+AMsUYWS\\\\\\\/fJytvV7P4UaSXcNIyY3F0BTffTaaBVFwBWCv5AYa78CQz5b5YEvqhJGg3chnFBU+ChfM5mc2tNpA\\\\\\\/QbzrwWOHJBtVqwbNluXtSo2q6DPX7xTzcJHWcoJjOfrvY3ZITQyseUCxMjv96fMyHMpB3ESZWfNWHOtCBM9ZvLEL3FfkwImO4JxJoNY3FoUIjjlw80kAKHZgaawbCUUhxgFN+IyWmmLEEs01PqCkMW+kg7CZTUTWzFCN8SMm21oSwhBW9gWumU\\\\\\\/b22IIcP5FsROZRQMKxkgrJ86W8u6GzRMuapCgjI8TRZWKQXkZOn8VwA9THQpWSrxhd9tZL+txsx2\\\\\\\/2PfGPZblI+BRfAz5qFusskcJxBEA+2tqW8gNlc7wLEaXZf0z5\\\\\\\/O5jPJVsskQUJ7yCDIfu4ZSYWlWYcXCSSbrpIC4FhC5142Arn8tNXulrhl+muXCN46+LnFNQZfh0hzkeMW0v+hxCmOSnovV\\\\\\\/mdVem7HRmcBRUDySJU6fGBXnskHpNxJBJFtK\\\\\\\/6O+hyqLGdA8sIgiB4AQpU75kyFOWWbzuC5DGwVrxPlVKUVj\\\\\\\/vaN\\\\\\\/HwRDEgbbfiBh8F2g2934Zeug31nQO0q1khgLOUEWocBunAcVhC6y+chPp3UDvuh62qHFYPc9R48+e9\\\\\\\/cucdujc3gpbJXow3Z8eojnQ5AR4TXA5CMg\\\\\\\/5dvbKhUv7K18ubQAYMF2G05LPSCxryS367AlmIXifreTqytZjJvTIPFiNMk50FD\\\\\\\/POaNsTj1t3eBsi+zr1Uu74CdGVfGnwYGh6iFi4x4C24AS7D9U2GH2N0+rrQE1ZusYcB4f80A3ORarN2gjwW6+WD6BjzNBE+sfEC1Sy6f8R6v0cj0I1n\\\\\\\/mQoGyCsLaJXZ8w3J2eDUuhmGzPAIfk8bK1UUaYzrlqWdHp9zbME3lRe6ENny\\\\\\\/U0FgCAseK0tyANZtCOK42ZciUGf172johT1TOf0egGt2xws9NG8+cZLKIpY6NPIeY9cCZg7GAtlinG6OBr9kofVALBBj4G8rMUV+MEJLrUe6E0sV4wflP6XbgUsZYUZZPZI99yuQebiFkXNgcfaZwYR9VGJyoMnKr5tPfm1C95rcJhRBuF0qClwZD48OfJQeXC9ZCPrSF+W6YtHBEcfFC8OdAW3d79H3NbitCXGoFbOes8gQd5ZN7sn81w\\\\\\\/he7fJl4NjmJNd8\\\\\\\/zmNlXR39bO+Q0XHJeBfwMVt8kj8zNDfGky\\\\\\\/SzEOHHZ5wQqoSKnjFwZSGJxG0goADlIabskg5EB3f0PRbzSpOa4GivUPvUgJzpqBQy4tp\\\\\\\/LUBS6VgD2jFntgi3Vzxs3cKq62k5lYmmRvAwtH1u7G4gFIDf1GtzCnfpnr8F7Dnk4igIB13KR\\\\\\\/5\\\\\\\/e22ALd2uAR+8PCBXLRg+pX1RYV1CNrwRWFIj6kJJy1m40lvviEVEtgzF1oOOpNpBWRk6PwQ0a4E7pbyAk\\\\\\\/WBOud55kHR\\\\\\\/8AoNxY9Z58Q4kzX2etOorvEEYW70KmE9fITcNQfeRL+h0aqlN+AZ2U\\\\\\\/bUvhiAWCHc5ZYWZnFN9vaB8kGJDOIDO\\\\\\\/wMjT+xVUn6jJ1KYC6CMspLCkYcnBaHnRynX1ORTNtNLzE3JxdYs15K\\\\\\\/V6RvN24Q9xSaffchnerHYFLF+b8919ZTniX6vNr1ubYGms5LTerKpq+8LYzQn3xnLKfSGHzi4kgmgHz\\\\\\\/tWuTBaYlMU90+e6Fwgo+Bc9SvB93CT598CPO00m8px0u8Bf8cUEgONMGiOoQndBpcwud0ioZkmnhBckGwLusmRJI6ieljyyg9wIN7Lj9fcpxwv\\\\\\\/KJpT37Az9Bxen3edqTJXVBxy0s7r2uwuHigUM55PRRv6\\\\\\\/L5EyJL60IYNxJaLEjY98oxJUiGfQZjWWGCVvFwn66PSLhxswrIai\\\\\\\/yC70mSyvQvyw5\\\\\\\/2uUvzPsT5OCG9dsusStGeQpaxB8AVHHAOUgFInz7uoL2gotOZOwI7Lrd83rx84uqIFiRu4aYRjvgsdtHGBLfCch6nYlSBQngzAeRzcc6UFA9\\\\\\\/lnza7hjrbuGdOiMTM6pQiDeKTf+MQ9FQUG6U6x2DtCfg0JBTOHcVxJLW3nEK2pMW\\\\\\\/1Hj8Csi65XAIKHUX1GrPdOusTfaahkO1Bw2IsbyBjDPXf2aUF3PpDBKHggsguL7BowElp5eGpK8tUN1wlfodPqe9eyeju+utdbCowRUnUe3cMy1jjHrZxZu7DXqCPoS7qgnYfTDU57rQdt+l6gNzSuhvwCY5\\\\\\\/k960s0og9UweIoE9z\\\\\\\/rJYy50GWQ\\\\\\\/\\\\\\\/cHpBoXIt2gfKT3uc0CCdTprucRkmUiHBLrLHvi5A8H7PObf+BvkZg58W4BU270eFGYRvxRlqDfKdDu+i\\\\\\\/Ks5Bls\\\\\\\/22vEVNuvEkhf6Uv+2cG3LDZ6wsthmuKqKSnSJp5T4GmHDZ6tCy7Z4+NI0ObTLm4qKuzQwT4wkxN1ZX8yS3oG+aZFO+pEMvkWfCBx7jMPD1VOwmhZfAcvGlSgEsZ6MKszqhr35uIO1rQ1ebrZsZ4qNakltrkUvVwKynCF7E+qmAs3Gosu9EIEctGHXNhIoocMebBv+PUuKuoIigXHvBUvXJ2hN0fMeFL8\\\\\\\/f\\\\\\\/QJapo0wJr6fVcJVVrKhRLxOy2YVoFYEjW10Wqzs6oOAHDXqxfAtvjt+zkRYmHkGwFfO5DSJDA\\\\\\\/v8vJzoGX4zy2lTPjHXmnes24FZh4+wDyGwESpypE8LbjTnyA\\\\\\\/PCYczffS6JUBaFH\\\\\\\/blo94xAUhoKaXkhGEW5IhD\\\\\\\/HDpCIJ1AW6c\\\\\\\/zxKO24LMEn\\\\\\\/abNsaRnRlMiDSvxfaT\\\\\\\/xeGhpi6FqoeQHqeK4QEavE8q8VT5jReRc2\\\\\\\/oiM11qzvmvVKjk8Ib5ANoJX9HXi7ZQG\\\\\\\/Vyn9GwJh+pZg7uc6fUi+wWn4A\\\\\\\/t2WXYfGVdn\\\\\\\/bvHenPrzXyknfLOAFW0WOieX88pjYxXa\\\\\\\/VoGTZqevpWiSG1FQbF+xCuwG6Au9WP8xwZ32TJQLKdbKUdk\\\\\\\/ahYATxC2aZP62rIGgrm4PxU77zYZCqfmIdNkQgs8ropEKagzKPQwqsUYzj6cb0Xl\\\\\\\/Pw59r01HatsIjcCXo2KNMozHSttoLEHRQLcTbAbo7JdbMHUhsF+SvHss5EDWxRc2aYaBbL70DcPb0iWGYBZEtUW4TnxpefRVbKUR7HYTbzTcEEUsIuuC4ozt76LWNh0tvRy0j5KAbbvwdSUnD3OWB3Cia7w9XVPbHR+Spd6LqE+Jl5iPYPbvbRE2TPHv\\\\\\\/lywEpebVHHU9\\\\\\\/c3LNirbBe6yj3BgBj7DfkzfJcdb5zvxiQYo+LjDW6zeCcfktY8LuPxNyXIpGxm1tN5JbAUsT7EwzpIsW+TusQrWdSOPqBee9Ut8UY\\\\\\\/xY5qWNAHRaJCNkP7ntyd27iztXFug9bQDSwLysgxe5B6uYnRfpBOvSVOSS8Ct6gI44V74Q6iIQY5UXuNX\\\\\\\/znIGA4CPOi8GCsEqCGjXPeaAe4JWp+00uBBptFB4P0xME9vchRFtjZkcJ2klfKppqsgTAsyAtGkNHgSNEO4AIn4OtsSYcp0nZ55jKq8hrAG178hb6NnRnbl++hljNTg+HL7H4QGHyJKr1PrNM2gRLFtEVyfzvhHjqmwgIeoAeQy\\\\\\\/MBPF09lA5jY+2t35q2JWWcKXpkkMikcdizHqXh8Ob\\\\\\\/c46T3JOzKXyCeS+yTKdyT3DAeEDnLIOqid\\\\\\\/HQUK\\\\\\\/vMIlWqQNNlrIaP7Foxy97KED5xg\\\\\\\/ZOzKNlnMpieCcnc73GU7RrwLJWEOhhs4mLihXId\\\\\\\/6NVL7bmuyg7jSXx\\\\\\\/S9fc9KR8IZ50Lpe2vUcEfeqNIW5cDWdYQjbY513L2Ja5DbtjwCWG8feRBl7MCG3SQKrkP9nhaTdhcUcQ4xyH23mCp6mEV4woLDe148f3GBYGnnjakYah0YGE8gKehExGoXQ4RUMcLhmuPVEcfWmtHAl41FUohtSj6E0QZ8HIMtxLur1lwp2g5cLT7Ve2Vyki23wGUOvkiNxd9nS4fH3CIGpGcacJhXjzsWTNqJ4sZBkqhewQju7x1GQouvI29SSVKeUXP1Lqcwtq\\\\\\\/y+zHGSpylpH\\\\\\\/i5zmv8QV3EjqTgbaq4VEBXnYfeMBTelm3yvForDL+2rSXq3YMbnON2F80vStPK1Gz8YL+FP+psbk\\\\\\\/YTyHcnVQqdpIKnUrZ6vkImPewscsHRz41F1r2thIxCU4ZJtN8gUxSYGE\\\\\\\/Ufm0SjwL1ts\\\\\\\/QL1WVlqxE1qxbbehOdSGn0NVS9iSnMZ+7TTOqZ9EOg9ZktWjjakkQZ0UQU7n4Td\\\\\\\/UzuLHvp5Spn7nfb0lSR\\\\\\\/QrJ0GHYWvRUe8OGO8GRaQb8wOMTbGCEAT5R6Pp5P9EQuawCpyrpuK\\\\\\\/Zv9xrBlRyVPMeKwu+wUqtvXYOB+FZdtcPB1ux9VH63nq+wAcx+zf0r2mrrOepCinEsgaDsgYD39aqPZzMrbjZA17j9PO6Dr6RG937HdK7arjhclyhBusBczayW3K36ll5ZHsvDFAl0aNQFGGQCqZsbbI14G8HBo34TJw9T6Oz+A7\\\\\\\/SRXDc3NoBczTQnnOyk8\\\\\\\/d8CDIt6GyPHmRApwGqqRgxTDzM6unPsNma\\\\\\\/Agi6PjYfZdcrZ883o06MsALi2c4HRUBHTptY7fQ43HelddXKPjuua8VpdKWmqHq0ZL0uJu2VPK1j0a40JGzBD69SntgVhMDEw2XPGc5BbSDabSOrnFT\\\\\\\/AZwPd8+H8SW1vLnM4ENaieVV1jSNRxHFImTVki8qs4Sh0MBIdPQDuR7QhETUqs0fpYLVjoTKzhb0ZcHTbgRqaPTGjDeuOZSJefEhgwEpOSvQYOp8Nb+z+0qQfTuuBHdRFtGOVr90gjn8UUwmmvCZu2eo7kiG2q917TzME70Ryoe5eRa41Aua81t5qNLfigCatbe2FekZ6wJZjswDNntlpPigSXcthhlADiPXB7wgLyhcysB00BHWKETpWMjFuGTpPOoRUuBEy4gPP58QyBDI+NatIfTzkXgCPuzPP1a49DNdHIEI3kstk1uEyXWQr7GY6fDV8YGR0TUjAWy4iFnzALtd7baxiDgwVtbo7Q0bJ1LxVKPVSrhByVAFQOkpX36Vx2FYhQC8RkIogLSv7OTRhE2lZ7cDJ35Ze8EXWbu7md1UO5vHtjUGJP1aY4kBOIwVI5h03MjrFw3lN4aiPM3DxyQBsmuO13XMezz\\\\\\\/aSauMvH4XHGXH6SxTqC2HBKq1NG6Wtzf9dj38tJ+QfMiYOL326coNgNk3NrdpLeCVl5fGW8XNgXhifcW4rWMzlvcgPq2P6yaFBX9b9E\\\\\\\/nUSLf0cPkjl6oc3F6L5W7xnw3mIfRYTW6Uc9DvlADrOSR4pU1GxuTEdvuJC7YLIyev9B1wWhswer9x+K9za6iJPmIDE\\\\\\\/Ygjd2Vg4lIK21SogbPnwZy1l+ioICupXW5wkXPfEqrGgN+7Jdo3TQ2pjl5he6cQPt27LLlIfgKlnRalSy3iacIfYWyZ+xIJKHJgtRmGsTnGAXM3Ahg8CfhHqtlBjQX6wcJEjQYUEheMIg5uAoOkCfhtjmnZb29+EeTxbIwJOvYJFnZwmm9xAVPyMRRHwCIoaGXyFJbnf+toRniQ\\\\\\\/K1ZU\\\\\\\/04rSu053BurcHgNtJ33WD88Mh5lq8tES6dyT4dJWwdqbwqvCSgiKQyR4pVkHpb5JC5tZUTHZedfUBFNvmR1dnIiRTkdNF7IztaU2V9N2iucfeDXHb9Jnt2GYpWHo0\\\\\\\/iDd7ZhZ\\\\\\\/ObhzizGF4TtaLHlvKlHUvoI5WR+JvscoSOXijDBoRAmG9ZXj9vDFPkuVXq\\\\\\\/2H4nnzD2Dz5gLzoR3Qp06blNAczUYvUyFPNtUpswfo+h\\\\\\\/TKDtNYn4PFnUh1vSyxrohDA8e\\\\\\\/Ev8PGGa65Esmgv\\\\\\\/eO3dqHYnuPXHQMeEJ6mnvytWXIp27e5chjDTOmbtkZAwuDiPdBDur+oEknbUMcZ4RIZcOq00MWvlt1I4EURUm\\\\\\\/tQ9aiTAYbyQ03iqck1Tr6HefaOGvBp3EkNzVJclLz3Dff2cdq4rFoUdjBivPq+9rsaKcE7ZV0DHJVlANtRg0sqzhtJaW+mjadD0LAZuxFYMOFhrnmDoUnLH1e3KypQU86RT9RhQiRzAYzaAJxNX2J1zyZ4fUai2lpO8eHsQaqpkmvSfeUK7weRW2qMMTjaeC8gQx8moQ4kcfcpV7JnFDCGr5+Y4s+jcWT9OJJIxIvt7TybBnct5Bjgojq8zoIeLckJfljw\\\\\\\/Qt5GNJlj8vzM\\\\\\\/vaLiFIZ4+72w11Nd3l\\\\\\\/7s3CKN2+19ZabNCQ2BBXPvh5liOx2FRhsix7rFmrXiP5cvoWmjWdUNFl+kGC1l5mczc2Y5QuIyCIRe44dybkk34jzIHDzZFHORGEFd8nDUPIZqKnVU371vrGi9MtAd9P\\\\\\\/HfrUkN7d2kQ2yXtW60SiUhUaqI0YV9r\\\\\\\/ZI4JOAwwNGbtws0T7vzVtZqhLrTnSyPiTFln9FSpUPk66wE4mYehcXdNA3gUtlE1ZhCQAVjmKpH5c\\\\\\\/qGs\\\\\\\/hRHbe+bxCPLQcrQes52ssclsbFqyPPEKjW\\\\\\\/B28bpgL9msnCvnOY12xiqrWosi3\\\\\\\/+Yf\\\\\\\/HVW8hlRyeKIwxln9UY1pBsmDD7wthvcqJt2rLFpI7iGJX68CsMi1PgvquI39hfNrufUCq6oYhICkkV46NscvGmpBcMASyMQdXl5cCJ3glkECKwscM1YReFoMA\\\\\\\/gTueMy8j5QIbfppXNG8Blc2XZZN4H8YZyq5yAK9WgqSc7+frwJWoXvRE783izPVmbrlyiGFLUd8kEps9ZZzHCjNN5el7MNDw91JCeMT0iukoa1mpEuffL9vi8RpJf5Q6hAerfCWQdZmPXkZFb2cypeA4OJ\\\\\\\/UC7N4ZMDfTMMFvJpWuqdDEbX5gKrkVns6FUyLvMKckqPK+wfUyY4WazxLUSEJ\\\\\\\/NFB31\\\\\\\/ErBCSqUSEUo6QIKnWFhsy4+61pXslKoCGamNyx3Q6\\\\\\\/+xJBQGDx9tYRJ5Lsn7h8CQ2QPYZvSWzNojlEIuQW+bzHTm+BJwHHC9S+tiB\\\\\\\/B9ew0hraJbprP\\\\\\\/LJBeXgnj9eMVanyjVV4yVbHE8hJ1VMUAH1uC9MOR0yzbK9DNJPlBkea3aWNVdJin+fh\\\\\\\/yhfFCKxk309Q0G7GgU2JkaysSKQ7CcCpPO7\\\\\\\/jRZm7x8hFvTgt+uxwrvUyb1Ia5PeAnDxSsd0McvccqUq0dqix4VdYWY8LaxIIa0X8RAUl1T6L9f3pyrvQUvjrsJ0DDQxrVaycTOGovtRqY1t4gyt3B67pFKavhSFZE5mZJhgRsKulFwsnHbCSsy0loSGvyqAK3NY4JOVDl7HzYnQUUBISxoaTAse65e42yZdptoKPhf8yKSsxumYTEDz65xFeDfMC3i9o1xS69MQui5kjQb\\\\\\\/qZRh2x65iW8qaqKt8Y8wTOMZZWKoiqQ4YfO2N1B\\\\\\\/hl55Qop0+JN+6dGVU1ED1pJXRu5ldsZ9LwvlaURWq7SdWE8q0SePDtzmXw5brYYnNzqs3UGeD8mqbAump+JryOWXHnCvHnAnXgGmNpdNm0V7HjHbMJGYzL2ddmiMLtGSAs8E+uh10p80oEDv5tKx+7iQFjzRES6lcUg02bHFmwnzYUMCL5VhvMwfBzJg0aGZoGbdwv4QlEnGMUOEdmrmDM11ajlU1QVM4Ncd8Ang3IQ9Q7JaGXexTtCGdJuC\\\\\\\/+CA1TOdB\\\\\\\/2JlIa+GrnuBrkB7LIBnJStQgutPfjb7+ZTNQCVuRmX3fyb86SPOA3zBuvhxVT7lzKf0rTKajRn3qZuiMI4w1YdaNx\\\\\\\/9Z9k4Z7S9qst3ot8YRgaN1BN6ckITlnv\\\\\\\/bYPLsxuRwk9BbD85pvWSRdZIzHGzsLlCPIMkaw8GQA08u1tXpwljiJgbP8EhEjpS+kCIru3JIriz1QXmX3OFYcIVX7ABbmPZkkZp7vrb1uxndvwqtwAzegnuu3MFPzEUyx0t11e+BfeDVn5GMtMLWXPMhsH+SryK4Cut1jZg+tVQ8+S9Uk4ZgE8Ev5fmJPy+707EOc7MyYsyS7ItI3OMfJbKFSASbtugBxxO7za4sT0PoZYEgpDJV\\\\\\\/l5X3BIYtApL06N8oCKnN0R7R0Vvtu\\\\\\\/LVvaxO2q09mSyYsJlEJFMWftYeZFyFaIvvqwbO9Nrkgoad\\\\\\\/2qKoaY5qOL6hFkunxQ7onOFI\\\\\\\/Q0de2CgeooN84ueIR\\\\\\\/JFdKQ+5n+RoK7ASiDurbSFbsbRATL1q4ulHhGG1V9LNrNxjdOCzX6vy9S+GFVneHs8uo0MUPJShZ1KcK9KZMan39R5oQgPwqoJH0WxVUj7lRkle4f8M9+z8pVJaV4sI9fJ1Sre+4Sqlpjw7x59iaLrQ520Q+crzXh\\\\\\\/krOHFd+UGzHRHzq4RV7VGozYtNisQM\\\\\\\/9BWUolUXzVaxwHHnpTpkNTqBngiBRxx\\\\\\\/GxLikXPaeFFBone157fffZH1litJy9RWCKXsksygnEmxkhHo0rzv+DgclEqY6DlCGWJo8sbm+8hBJJWKeNkoz3GUz618kamexcDbqT2D7hYRjvhQg2vMKewajEuUBguOOQ9ZjtQXtMqWdMANjtZI7v0zehQ4FwntFouUv8QXWNzdu37NC++uQpDtScPBaAHsehuBVVhPjikc7Wun4r83v\\\\\\\/NZHSN4bcUIHNIuotuprHGm+H3VzdNZQ0+TvNNg+xHTXo79hZRMBonZCGPPRMLE8JNjv2VQbaZaWYvTg5cLTCB59mcFTSrZInTkSgpPQ9IGY8pinL3k6bAe1Z7TnR36lNDBX8Wl6V3OiotjVaRZXykOTCpHqKzIi8goRU8ZTpQy+CBgCgN4RmKMucTly6od9T6ydtVhhzKxpNd2A6qv\\\\\\\/Gpb8bPhZk274EiAJYOdnVldFiCCt9uLiFZt8LzCZV4sZEv3g7OLK0uiTFlEeTdNBhFjXUNS1b9Nr00wtVbW7gesufSm2BtXK0Q3loufSguDPJIuB+RhLP3XUum8SleFXwU+crXuRi++wvJR8Jvpciu0AchnJzP78lAMXT\\\\\\\/5tXB\\\\\\\/LnBquEKbeb4meuBbYsghSeLGep\\\\\\\/Tc3Hc+eA3ff8XMPo4o1COCFGMwSqTCgXOCqu1BnbEOyU9I4XduvsASJNjnqS4DjidG\\\\\\\/OmZsBait1sYan7QZ0ukWcG5xBsTE49COuseMNgF2pMxfOyuVz\\\\\\\/49XQFCFqRIs\\\\\\\/NzZW2IrZpPkhGNsdVt9xsDY4g8cVwPQ3CL0x5EbZLW+o3vRrsjBnYVdmwZZBSvY6L3hNgrGxx60Gc24j1hS6zL5WouJN8psgFULRKq680xJ2fPOstfBP51wA+AvYSPg\\\\\\\/qHEXoEXr42oAemoUghkEUi+XAsfu3CrL0ALt8Ug543dNd4emDi4x1stxlGqRKYF6cEETzOplVTPkWsptHCbkljRMYfnaFShywa6RofN9l09L0bruvmstLyuosMevYp3nUqQygE6U3\\\\\\\/JPKZjFSiAjk2J6UX4gbtPFgzVyCdxZ7Pob90YcIubUtlJkXapjS7MNcA5nb7quzFu+tZIvra4LcdXBz70w3ngjwFZZVTBtrf++tGKxMhASiaXbXRNG2pMIcc5\\\\\\\/GQbuEgvAXCTPJcveXpLPQylGErLMpYOqFQ1DQbC8vfW\\\\\\\/4sGEACsUAzUmVTBFoS\\\\\\\/cpEH\\\\\\\/gtRgEBIG\\\\\\\/1n7H0Aw2dfpQV3hgawqtkAUZNtG5GApuFjmWdq1+7XkNcPAyc1skc5pxu0IAses6vKer2gwNt2yqN3jfCIOuyB5Mj7J1S0y3ipcm7R+hUunuQHJ9I16CUtk53n4Rz6Q9X9wiCrC3iMvzlyb1QDf2+vco+k9PtAJXFVX3MIKCHgX9gIP1UIfN4ugIitSYN+u25P9Wfcxy8LMI18mLdCNkTZXj9CYNw1OgA1QZ1u8oIgXUQ7md78CBY7FgbXuAfKgb6ADKJyX\\\\\\\/jU02n9aRuH5NBCSJhWSffwf2p07+7XkoifO8Uqdj7oNmGaVnGK4\\\\\\\/ZkQnXGo8LLpm0XVQNfr3tAnre7OYGiZBRpNPyy5F081MeNLJMzAsgpoWFISH8HgylD3hDmzrwSCXk4VVNUQzM488jAy5ldwGJH7iabFjxiH37L++iqgPkpnzei15Ya9JLSKW+KbzKmhmZcV\\\\\\\/cV46fUhrTJGRYkv8BDDsFqTp\\\\\\\/q4uM7yGxnn4Y2hIevqdQGx+WPJfi3ipKnvERWdK9l0lo8YKfWnVF+eEK3s1prAb+wqrVf9cKnKH2tN94DKNk+a2VL5Xj908IQqAqSHIBAeUi5mnwALpCqpAhSMtiZZca1Z8TAuIzdM\\\\\\\/eUwDnJrx+2OssY26KVkosBG5T5xa4qQIAZZ71Pm7N9fQXFR+yIrbxRow3ErebwXW5qNqEQS58\\\\\\\/xUeHRCmUT72uQgO5V3jyOiW4iNvErAWzG3CWhXP+cqlTgpQgtkE\\\\\\\/P2h5eK+ivWbvkyAJoHuERSGWYfZHQWvaAJfUGyvUzFR+DUyeYXOPhXXMcn1N+k6DShrAl4lgHicZqz0ssvOwSmxpAM6Ti9GPRozCDtSFYf310tbg8xnrk1qz9PeXLGTUL8oXZM7wmiV6\\\\\\\/aVnenwa8pzsxG45B5oBIUGJVI+rNMUmhnGf7SlXSLl8g1iQurK+bLmXRDp5o\\\\\\\/yPbp8GQkYea77ZBXtSJUyo1QemedF2BNB9kTjj7uhI4xUcIIj7NiEcPBoBH5Ba4qV1q\\\\\\\/rCeZ8PqFNc9eILqiL\\\\\\\/c8LfZglTmanFKNI4S6obqFkrSo22mjXPxnOaZFjYUHjyPszgOcuPyDQk2ZNZrCLHd2nbJKb0TJ7ZodydzaSTai3uFChd\\\\\\\/KDccfgjBzZmVmur42ZPh86VMp6JbMSN0ACOOtpI6aPSwf2oOOCcZydgixwk7KC7KO5XcC4USmJKkJF+3tnrfELU\\\\\\\/ftsw8aLCYnwrLmonyBTHrpqS8pfFoXDCxOR5wO7TsUaiSbzi+oyfD\\\\\\\/RvcWIX6dYtvW1bBDKBqlbLi2x7tVjVlpCaL8f9\\\\\\\/24nkI0fMzIicj0FUM2keSKVAmOi2H3obseHFkG9kA2\\\\\\\/gCynAbDnuql7B87yy3Wvj2oW9mnKksjoYTl7Ds\\\\\\\/E16kapHMR2QuSmL0fLujEffaeKgButxNfzWfMU7lgORAwjQac46WG0\\\\\\\/9u7uBXwz+BQLb3z7\\\\\\\/FQZ5GLDGs\\\\\\\/YNaZDdiSpYm95M1nN66N78nTMAHOSGEpCa\\\\\\\/u0sKc9ObvkRDA9xa2cZXXNj7Y6sn7KfESf7vGZzqsvcjj80+dHwUKR4nlOQ\\\\\\\/f9Y3tgvKM2jBmv2FHdU7sr+Or4uud9KQVGQzaF77IUGFmWL7K6QChgbMJKQfdlCqng5PAIG5FmbbB11qnZr6k2U6mvc1YqCKsEDGljDlmKyCy5OHIo\\\\\\\/fpknXoUvuWh25aOvTZybE1mOijE9+nZf3aYjGKOxlzHWXdXzyuWQdips0n5Jk35Qd33AhS7WJInf4sdy4LN6xDAPDX2BNrTlh+HXaICn1fpXeS26OWsJcRk5mLvUO13RNTi5Zcy5v8fPuZD+yX+p9D4iI4Kw6qziEqM+1rV8wvN3gijGkFoiLcl7GOg+YqX7QdJDgsVnHt0XRyr9VWn0LQsiObA02BVNoAfAOGpjXv0O3YX56GqVxg36scJHdwOMSt5Ll2xiqKcDSAVgiujd1DIaiJsb\\\\\\\/vdv75+53p\\\\\\\/VsbFXdoTBJz83A3J3P3RExiC8itIBit7rz29qDQ+iEEFyFR5CpXpkxIYRpemM5Jpz786oiFR7QOb1uGIU8j8+cu\\\\\\\/EdnA93+8LBqB5q\\\\\\\/Q63i4NYPMiqJ8zfssRR3HAGZ5ADbQKTI+XO6qbag4r3i4jRaLCHelGklu6hFmGrQ1qExJtOTw7BCo6g6TmIxdjRlsW2x7z1yKzEd3hSnhXqiPoo3hKVDf+XkSjcUQ8JM04rEp\\\\\\\/c+0+0fUxvQ9JLK9SFEL2jmV9jLnM91+oM7eQoY8WeQYOGOmA4LeuQs29xfOJ\\\\\\\/i9zw3zYMa0vOxXaaWQO1s7PlafS25uSNrGvXGjo0TzTQ1d\\\\\\\/B9mgR04zDGesxSZeAVQrq1GHblLvznODeoFR8mqSA4snveM\\\\\\\/jcGZcS0HLZtBLDgqQ3wFTqV2AQPspvqnHYzthGw9jgeP9vbIK7X0TIPJAKUVVG24gIXcfM2F1yhkpC04WY5SPGbPs+3+F0HD4\\\\\\\/MTevqH+QgCRvjpPDiBzXauh01u5wIZspxIbUc0EvLXkTC4DgP11liWUSksZhCas2KhroN76eiPqL+VIGYlkAnyO3JSAr8MVA3iRkJ4I3G1YUDEpz6eqFw4W3kWTFibAkpjLwXdT8YwT8nmVHb8FXHhCftgqv4XOUOwja4wzWfNQBg30KgRUMrPKOg7hIuOLLWS7u8RecF5D0V7TQ+w0o0hgDTlUTKWgaUhQbY5Y7lJhcFHNY0F\\\\\\\/hp3ctRyrpPD5t18cVV\\\\\\\/04vrfABtOVWxCfKvIENBkEyCmQIknGPN8Ejiae42nGoLC4In8hApYgBmmryzdTaFsX2OxQbuHBI9NymfgNzftPBBXt7b\\\\\\\/JWfaAgujovRjKbJW9uYQOnneBx1RVV892C+GIZ+N36eJ1FkRVtYQCJ89qLpuUrJL9aiTNeGVGboomt9MVsElZ2+Sch6gHitmob+yeDwvsbVDDL\\\\\\\/iPTUeWP2LUg9LC16emVmMKSOLPoqyZm06LiH9YEXAUm7+Yv5GTX6Ljgm3Dp0Gm31+Pe1UfXAkHFQitf8w0wW1cnWBShUaSXAYimLytSLThQ1tKPVsZlQucjEFfkqvpEM8N3JtF4nxB5hnYFtgg+ltyDVJljogLxCdEeC5fjrq1NkQgb8MKfDZmWid\\\\\\\/blFrfM55OeOFI90mzybPNkGZit+9Sj\\\\\\\/DZUmeuir9U0hvN63Xwb3ecQD2zBspyBxCVM9n+d5JngVxyS1YYUl90nWdxHV6wf7JpvRMMXDHTFx4R04o8kubKmnsGsBAQHDakQrBNcZAHvcNab\\\\\\\/V7Ee\\\\\\\/p3SwlXa8tANlA6LXA3S1TW5UWg0sqHeGgZppUAOfbwD3IAOr6f6pw\\\\\\\/jCOkPhGsuZ8AdqKg1WN0USW7pIPQ7HUM+UrfAVbNdjpQNTxbUz7OUMMbAqYz3Lnqk\\\\\\\/8Sm3BIpqGX8iYTZcELiLHPFk+mana6ouSQ8p9hZWZLeBg3MhdfVfvKlUmDRpDB5bBUKiG+PLxUfvhldTbz1\\\\\\\/n+o0cg5jXYbqbWQth04PilbVju8ZVeiYTEGAn++MDteQoQjZUlV5LQYl9rAKcuVpV0vhgfk60N8mSuyAmRaSyO9CrB\\\\\\\/bB8DX+FpD5q+ptq5KKC3cYFWOuHf6XMoR9xBoRgO6ReDBVw1v6FFK+Y29sqBgKNm4aS+\\\\\\\/XxDK4sBVOdP\\\\\\\/GHKCJJEHCySqU+bK+a0Xdrey9nvAEiVivuuj0IY293s5+BPY+GP\\\\\\\/hNfPXBNlHORMKpspbfZaCk56+Rop215zYZLD7vAMsfQFFo8tabXsEba13uiwEq5RLm\\\\\\\/j48alNH5LzEhnJCwqrW+8JVEKi1FfOCtFXl6USDCcFeKbZjDF7uZLvt+mpM0HwNRvXnXBGi5iw4RQwzj04l2p9teHcJ0+++s44IUq3Rt1ILJFf\\\\\\\/h5S8F8Di+luYDFJk2Viy6m7Z+i4cfD5gkGhX61fe681p\\\\\\\/KESHWzWhT7wUbAw2QZTJyJingWVZllcM9\\\\\\\/n8xMyUhSj9KYs\\\\\\\/WF+iTA29Vn\\\\\\\/783bJyQN4r6NgfWDrkhGERkvASSh5uNGJzCNdyYWVqxo3o6o38hZX+rnr6N7ngRqAIIzQgmrLfjJ4KmxFKE2V8wVLye\\\\\\\/\\\\\\\/L3H6QsXWafkuprt29TxGcMmkUrs8ZcxPB6M+mYjdaJj543UDEWN5jVYtxWUe8uKQLHeKyYraUYjQ6OZgMV8RfhrEZmw6Uuxw98SSS4s79Phvne1tiWPlM0uuaV2pWCvm\\\\\\\/bx7UuO9cxVnMKMOTS7h\\\\\\\/RlXy4ZPq4ERcV0NibuHinLqCY5OzHgn87zpV9CMEEByeG6Hy+n\\\\\\\/8qseV+MPgPEMnvi\\\\\\\/kYPTC3mmadLnCnrEdGHsJpp2XiuyqHFTcrzxjCLzkLrpYRYdl30Yjco1kBIGYjCWQBk7UHJpZzsTjWFDiKPEXvLv62c+pVCz8DCn0I+kaEj+jdWqCYZgbRvSMi7upe9u0V6SUopMcCR\\\\\\\/QYOX5aMavOcPmOTYdYR1YCUacqcWT21s9Xp0tr0SbWFHzx5fJdde2FZu5rwdn\\\\\\\/ft\\\\\\\/esZo5Y\\\\\\\/cUjAojhzvuI7fA\\\\\\\/RQTVecbp5homWciwzmbd6xZ1FwaYHLgCsoHAbSknp6SZkZyk7V\\\\\\\/7nL4dvRuH\\\\\\\/V5fzzYM4EYtE7Ic+APAkd61OpcSMNpaCwBrWAx0+zgr1vSwNwEjvv9AAX\\\\\\\/+pjFWXFaAz1RwYxXJYYWlzdstDidsLGp\\\\\\\/trd4RaSC0l4\\\\\\\/DVrSsgzhQgFCrlxDDC0h1fXe47gzxsnvDqHy6T4Koc5Dg7v1bIr7T5vQpFkkEA0M\\\\\\\/EudzVNTWmXwzk8eL02aoLIeYhacpRcHPvnstNNoc4ct2yorCYJ2RjyOk2zzRL8nMUvZLPU8UYpRm75vG9E5t2E+3+Gph9Mob3tt3bPV7dLQbwxopUzCcjbgtDr1rzi3y0LvN+i2v2Fm6AYDMi3cTiunfMGWNP7NG8l0kCnbOx7iQs5fn8bqeJA22ArYv1rCy0y3S9IW1csxrJe6RPbW\\\\\\\/oLG0VSYfuATuhXUDCRh5HsauvLQzW9taRSyt2u\\\\\\\/FfbnM27x0bUoht\\\\\\\/0mXGpStjeIx9SzEnODdNz89SQe3F5MynHCJN2lXawg6H3e5buUjAeBlKtxQRlxiZAd0g0i\\\\\\\/qJs18qE0qMqMr9KEEHQeR+7nEq6EoGz1TP7yc3YLxRqubck1AJPoyPjy+fM0S4EkXUa7zu\\\\\\\/mxQDpaApdPcvR9Tlvue0X0gRjw7E+aAixkKWm2dF\\\\\\\/jG5zQzyqtUc2MMnpU6h20L8uwapa\\\\\\\/b7MgGPe4mvrxppVgJMKtyEiBWfdCuHFQkz2y5EcAHJa5RAEDpXcP8TyaGqF8flL71mKoDMEp9pxpElFVgR7iVeE4AF9Rd\\\\\\\/\\\\\\\/OjUmbFeN30h5HbAOENAnfRNHGIMtOHLChLXhpm7hsyBY6rHv60gdIodEuugOMf7ODiQzNuJAa7IRXJy7tbJ7LatwL9vO6hVoxJWO7MOjp9bU3pCDRlS3chkSbS29cRqkXUM2TVLgHuzjD26WELkCrGYUUZPQp2i\\\\\\\/Cv8PTabQPiHOX20HqMP4b1ZQFSwf9XZUq7NDe58HEgfY61qcFpTex+3Y\\\\\\\/w16GSX3OmEcDLhq2dOY9aSkymknny6cAUDo7h9aXjdUzVKr\\\\\\\/RL0hk3TNk66Tyr\\\\\\\/BPPYm1qet6pGVmxKZnHa9ndoLjZRnViRajLIrRfS29yPjKUsvhB9+JJJnpnnxf1zIgnzZins4SaIfmNm8LsGhRqR5CMrKuxAYkVyle0i4JoRY5X\\\\\\\/iwximicGe39M+PIqsRcMh1oaCvFhcAwriXFM6Yx1zDMAGn6IcdfBv6EcBRQFBP0nBFPwUHRAYShlUwcgg9RAqvg8Q9DcaEbxdafR\\\\\\\/bY6r3o\\\\\\\/iWD4fsbNvgj0nF16l4qDM2pB4cF2Wy93eMLzllftbp7GByt5hJ+QgXd3JZWcUHsItneeU2LXN\\\\\\\/J7oaWblEKctPLsf6iHwQ912otNhrwgkkbhBDGEBxuqrHArbh4Sb0BzHS1kl4ShjZUTrZS6kkv5CKfxDV0H1XCcPrEZbwaOZ0LFPseaoXdtRcYnx74MkbHajwo3fVprpto9bIKSBSx4rqMtfrAovLD8cOWn8G9ZGfVqlxonEmPfx0f+6jQL84nAOoSPIxe8iXLZ+BYffj75S33\\\\\\\/oXT40HJWaJW9\\\\\\\/NTeSdfHNRGws+V2LJwS4ULrKUO+A1kACZQxMrMD90qyh+ISdwj+FKLchFgnFSJGP6lu5g4hHyioukzx1P6uP48FmI4UFTN92bphOLvdNCeASE8QXzPiDQcH4ROV2eWTJIsTsXb\\\\\\\/4EzA0K7N84G9VoFiFbh7YaR2uugDWCnePOuPNgrPEKEgZykXILI76RbP99QvTyGUPXeT6v\\\\\\\/Z8n7RKxxw7+UjHcb6uEJsWZqHtwmSGZEX53QT2f6Q1of9UbWy1AiPyn1BJ2l40DrjIhd1AXSYdhzAxWszmt59NswLDKCZatUilfpgWhyFZFe46gyYjhtH3ZFqb3zPHEzB9DZ33e1voEk4SI3V\\\\\\\/+i+I9lMnTUgE3N8Uf5gJgAB4VC0JObeaRxOgNWAKvMug06CaBNaVRlv+ivrlWkaGtoOhYMX28E\\\\\\\/kGohjX5N1GMyfeEqnnX7vGYAOdrjr4DwBNCMVxZJH6u1LIM+LYWOkaSoPfzECJk7QdTxn6k2Cwa6Q8eQRDMOxcAiD\\\\\\\/YSglvXxAt3tCB7\\\\\\\/FVXvKaddXhJrrHVh5GO15bRaZuLCQEkIJcykNZOUcrfxkmK8ZKQMJi\\\\\\\/7d4n6NLuXG2Xwibda7OPgi5ubjxcs4+Imaj+7m47qcmr9e2AH0X\\\\\\\/Qc9Jgt\\\\\\\/NYz2kivUhl5VK9bB9XtJV2zmG+0zw6Hm1RsIVciFZR+pEyRuub3YnALe1OJX+pngqGdT4qOcoD98Lpf3+UE+D38bqEMlYNqLaAUVGBdu0n0ETZy0Wu87mCgPRsi4bwSNWTiwx763kB\\\\\\\/Pi5NTd4j06gUpEZ2pd\\\\\\\/clWc\\\\\\\/3Z8aZimdV\\\\\\\/nVjkgvIbw35ck\\\\\\\/s9edYyr4IMF7dSiQKqEQi0iFmkjliMfPWkGh1\\\\\\\/FzbYRVKuKEqej\\\\\\\/E\\\\\\\/3NnnVoZaImnSwx0r9FRjhl62Zh\\\\\\\/IbMjboT01ztleraFftcDwDwbsemUcjbaTXhb7ssL+YuAVbEjejZuFvDOjCzhMLt2oepYFEKz8tgOh6O1J4m4RWFQ7AOuX3hw3ZuySKU0pW3DJthJGBQFWLdKbXw5LuEkbGmnSL1cz+KZ5PFlwZY8tH9O1CvoVBkPSc6wamEtL9\\\\\\\/1\\\\\\\/Ma2shV8+wPC6BACPC3jsjvgWuNll8XkYQFE5j42EzO4a+FgTS+60obWaNuUdPD2lGokK9gBbRJHqAR1IQqTNkp5IeQWcSNWLda8RD\\\\\\\/\\\\\\\/VlakbWW4dzU+jfZ3kjR4Ztq2WgAUU9JKF8OCGGUndzLIfaAvxtv5SqBGVcMIKnM2Mady9a2piRvnkfbipRayAlDmjEXZwi4r9pItP2M+Ky0wVCzEmSMR3cE9AxfXZGb6XPm8hGdsSyuyKMTGQCc\\\\\\\/xo6v0rsNTq+nTeqPX4dbWpRSzsgkbGwvh4ZtEICGQqroYPdpweWNsr8cDp17HMG8Ardegz\\\\\\\/k\\\\\\\/0Bg6MeGikAH\\\\\\\/AzDBdrpWbaSQtJZ0ce8iggPf+lDODBuy7jLb7p023XdWYLUGvu9IdT+cYEhXaHjGY8HXZURcviA1rpuAL\\\\\\\/UqzxHLWwkMDMUOH4QUga9LrVgsGvwcWoVX5SGhuowgpvj4F9DuO9aAW\\\\\\\/3HrbQzSz7t03+cZBzbDDbQqh9NqkFI6610fOzwKS8qf7edjhUlCxXXcy0\\\\\\\/E6cgPXLdBmfug4hhdeC3AlQbSstKAu4CQqlnAZEYLnMZByvl4OwGphGJGNUZY5vIe1nE+POE52MtV3nSlGn3Ce1SRlvCjxkusUG0sILNBpGllOAMlhg\\\\\\\/G3VUdpRtQxqSvz+zLG0u6kptuvoa2e3tnKl1K+FM2cU0KpGwsTgOOSQbOgzdHnllISXG0rdwYQShMtCfGKZrPG4dIehjPeTJRTzpfyn35qkcEVMDWeurMawp1VgAamI1+ZrYHYQpWndE8+S8kt8Msv3ULoWud3KUcqlSxPdoBHOSKzCuif5mnqXvOh0XuFhSVZvFFaj4pSAbofMrMZ1ZdXB4zxoBR9HLTTGPu29c16JXAIyIMyduy7cs6em7yQY2VKRBHNnQOfFbsntHViY68M+sqbaGMn6ww7lIlevOqcNJmpq36OMcII4lHFAWou+UwVl9HhETasVGhGDgSBjvyNhuTXrBk9qPNiVQHsk87lYIpAP3sv9S8r7c7T4TPTd9+n\\\\\\\/TPBgTRUhDqnSK\\\\\\\/PaR02vFvJTAZKDHz9U6KvS+uTgmMM3rJkjBR6meWIVAJ\\\\\\\/K4pFohu\\\\\\\/LermdRA0A5DV0Zefnr9oMbRRKg0+zb17r67fvtoRMA2rPtaLzB3NM67Snt\\\\\\\/k0TRopmt3e5R7\\\\\\\/Ri9Xf7yKNYP6STWcn\\\\\\\/JF3TpTAxt55XIsoC\\\\\\\/ehDUYVvNWr5tBBVR9EdTax\\\\\\\/\\\\\\\/bvUnCx4lx8uC57\\\\\\\/pJPG8wsOWEU6EblYPrNcSiB3JNVxdY\\\\\\\/\\\\\\\/+IpvlC8T1uNYT1mODevsTxT0LYvtc4Jf4k9d3V5jt3vvhIv9uom6\\\\\\\/t4f4Y+KDhHVOdh2K2CxDwHOPpIdyzPG6f0d\\\\\\\/sbbXzbkUeCdsK176cRVcn7IrOUJppQwGvmP3JfG0xecp\\\\\\\/RGMEo9Qrz1WcZZOvx9NubhO2dbeNJsRWfXd5N98lmOuUM+ixXSrSU386qooVVHobv8uFAbmYZeYy6vJ895f\\\\\\\/uX7fvQY76QRD0QaQR8yQVzP9qBesfEyWVPXxum2QM6wJFOV9qnHZtwVrB8Vtqaz2FjB26N01Y2pHFOch613qh\\\\\\\/y6jWumQTOX0GH7vmbF1CfzDJ6IiB4V\\\\\\\/C2vXs3SD2rGfcZL9fLdxuXOjpm3obWk6Yvxm4pcBRBZ4Tmhc4a0+BFEsR3K\\\\\\\/RgSWUljIKPTvrh8iQ4nMgg1RTt0Oa0YXqCIQ92cMIIHgpTJxdAcwaQQkObI6Mmd09gGPfTgPzoLhCYE92gkyEh\\\\\\\/e8TcxBTAqeUOQ6xxps1bLUa5qVixQxJj\\\\\\\/y0e\\\\\\\/Xs4rdA2dIqqw3zi7pL1TaNphLQxRyeHgkugWSUhcNZKQJtKUMgqX+ENvGVUVEGbetsaYlQdqNvuTuZFYyaxXi3\\\\\\\/EnbKigjtN\\\\\\\/GxSEf2cWzYpUaB9LKi+fc\\\\\\\/OPh\\\\\\\/zBdR+YoACTr+maExz7Wqr9vB56BhCtO46m4mLtOGkeobqVut\\\\\\\/tmSd0deJpLSuEviLsR9zEaMglHgTl8U57ZnorHX\\\\\\\/EfCDuHwZ9S\\\\\\\/5pb8xhUL8100iytAQivkntaR5xyybbhEvNPFg1JdfvqYVSHmmZTzfQG+ALXlmzGQADxgmZqyEkZIgyg\\\\\\\/1nHoYljqO07nmaZGoQYgOfiYA9ArFQUczi\\\\\\\/16g8znxTiTRVIWGacWSch1dtWnGZ1hIANUsZpCrgWBAPbpSMKBTGcgvji2Qkheujnp9tRgvze1D\\\\\\\/+PlNei8yVDunkBdvcfeVELKvwpS2bsG0vC6+7Nw4hBYCodBYxQWCO6lCHo8XH7QyOQmWKoUwKdDOMPCAIABwSyFOq3KZRySjHzNfRFWhsWKoFfaxJXu2yDQWTC+ug7s5GJqzi7KJeWiU5CqpkWFjBeaRgF\\\\\\\/hAFDkKy2Gp79R5sZ8hm\\\\\\\/nkzWrDNmsU25IkaARMbWHflSd\\\\\\\/euREGT\\\\\\\/d2\\\\\\\/AfyOzJofS3lCixxYi0osHe3Ogg4amJ4+2ijqPyqSr4517ujMU13XqWiU6vauyr+mrbCHeT8oim6A39KZYbeXYX6xNBdqh7cBmnQmxEXUT8QWY\\\\\\\/VRTIaMcwIA\\\\\\\/lAbUwfWNS+yb5mKv8Hj8HmtIpgfbcKzoGfPxDwFp4i7TW5ab8qcbGD1u+Ac\\\\\\\/UaKjLIsxO9XZr3tmtUY\\\\\\\/3DU7HQ5zXOmF0Dfa\\\\\\\/Xi7pH90BhQkumpJKZ0bemExMiR\\\\\\\/jjdtR8HZsDdQDrniT1VZsEUy5v6RRXEW6Q0ScwWzam34O\\\\\\\/soNaaZbxTTOaDTt+F5OQENVyghkvnc5fSQoObkkaGGinlNG3OZimDq2IiAK9VoTLpaNYhtMwMG24QTPMgoOntgh2iXn+yVjFGzq7P2hQcPFo6uDSRtP+Wwb7t3SGr4PPmA1Hi\\\\\\\/JaQDLdgdcL07493+Lczbupr2v7YPcFoda\\\\\\\/kVfdsmtBcPLzsW7EJBIGdHTqklGIhSOMyfZPyBppfUO2Lrj9G5Zpz85f7r087K1DWzM1vlDJ2c9d1laJN8s0WAal2XiYJTZ59tjKuzhYGqPFYhjQuAkOCjejFfvzHKF3AYrnCg3TB2a2HQHh7LcI0gWnRDweMd4EB4pj7glGGA\\\\\\\/v6RmyuoFExLd4TncYkx8MvDgd+dN9rnZcAwFpf8A7vF\\\\\\\/KyNI\\\\\\\/\\\\\\\/R+isq+hi1zcCBqiip7FrqZjjC88U64Sgd\\\\\\\/XoXsVNLJwaOswbmQOxFZpLOmmLLIDTBcC4dTj2tnKTSXT5D+srh+Z6ngwwayFE0NTg5NtnOUozJjroTd4SdeTNainAR+eJtIWUg3A1LJ2JKHkUnZqevvXiFyikr5lqxZmDJiJOlSUsDNxddBx\\\\\\\/6Z97Pbfm1QLq4WkzDTW4uwtfTBzoMNUxm3A9qD2P96WKnpFqSr8mdZ4nbqLigM0xtJ4PEJBJFk7DLuln0NdBmYkGEvFOIB6kH0Fj3rbmBQg3K2gBqdGBnBEwD0wY2T9+3wO0xKV6ZjsO8oaIaJU5aGMnVkYWaPdsz1rbeW5I7NkzCJ0zBsuicI5Y\\\\\\\/eSA2H2Iiad1hHfcaXA\\\\\\\/wBsIBjv+Z8Loi1CzmOuOQXGL2Su706VnMoMkweDXagzKP8GLxdVGGBpHRZz0wyy1o2sFkYdtoC6KDrMyBJ9xhpX+tR9NpULxWiFVmofuoOgM8yEQVZRa7YkP5JOuUhp6HHGRX6l7P\\\\\\\/Y\\\\\\\/IETMXpgLW0Hk4iI9a4DMeFdbORzWvuO1CGsNVB3X80dyE7cQ41PAAbubEH+B1W0qATPMmNtfarFbdsI24EXodROVRTXEl3hOS0Nn7Me0aQfJrPCLz1FgykLHlkTEcleNkKquiMG1J1uQBQnTCmIMjDnkvsZWFleJzH+vHVtCa5knnqvYyAXaM4l0n9qxHASMblwF9n6gV24xQcHPvXn2f48qDSaLWoMqyub2V2faJxLF8hhyIvQdJImc1WRi2BolJkX6Sr7DEOBCAn0PSCEEDI0RXSN+U4IRtuEhxD7XC3fopIFxQ3asCQJxlu8RYEaHZImxbuzgOd1x8MEVBKi\\\\\\\/2yRdmSzw9SJRWBFQEy3hdv\\\\\\\/yytE1pbF1CRS6BPZX5oRnzfIM6kk7JTLdLZKYRR3xzBTZR+LBu8sVJKHQ1v0Cbln0yGZHTGvqfqYyx7Isnpk5JDwBQgnr+QHTQ38vSD\\\\\\\/dMy1brpvvBXAMF7XJirlk1HSgSZe\\\\\\\/IS8hLk9fRYPG\\\\\\\/\\\\\\\/6S0YSeMO6FJwrrFdveSitEv8w6L2byZTDCgIeFpS2bgu7ubmMH8PiSqNULFLQkFw+5wC+wQHw7cW1nY1a01B+YvALTvcitT7NhrZeCCRm5LJVwTczUIl2FFaVUpIlpnjdA2sn3wPF8JVQYaRkD0Wwr7ru+Jm\\\\\\\/FLCEzRqfQfnvtgqmNRiz98hjSPrtBYugfYy+dJ8Eb8Pm6mP23O0tRI+pkDdmvsbg6jNRE4h3wxrL9AkvlagCTGq6idVM7R97fvfSFabgu0UOC0uybjh0zzLaX71BBqmPMk68STEH8N5bINObR7wbZYXgSyFWOuUyzdg0Nlo+VcDyyGXvtKiBwr7qFzfINd7pKfMD8DpY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NdV5NubdXeVM1w2fcGvtlw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222d68296834a5b0d418541db2df51bc9d6ebcefcb650f3a4ed2387c451dea9813\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221um9QP2xSThIdY7TdBx20U3beHCZbSIytFIKKMcxfydH5h6hRr3FqOf90eECQsRIGCazK\\\\\\\/s1EAYDjIVaTkGIA234J5m3hRz59e2H4a0oh\\\\\\\/99dxLoXz4WDqioFSVTIOQDAc07mPfKObldRRVN4B2B3sKemueVmL6CPAGMF5ixUrrLH6F38CYUtijU4FojMHsqS5o4LyOlQnFw5ZGTsZ67\\\\\\\/mWhQL2LT4+qvQc4VAXy5gKQt8cw7sHc\\\\\\\/9gvtT36zviAFtDTxJoISBkWHN\\\\\\\/oxQK7lknekWIbIB+5+hHP3nVQ94TbzC99hkxXz9aFKS4gwSP5bGktd6RofUbP2MlblxeXVSY0FeEQkKtLLplblRn7YNboxZQbD\\\\\\\/Kb2dkcIz0bfPUBGnBLoS6naf6DWvxCiGdtZeBKtQhqERpnfxZ4loIgpHgAbU8Z61\\\\\\\/8EOkTlMBOJBQcBr\\\\\\\/VlpgJy1EZyno7SfPbPuDV4THMABcehsd1bk3qESRvPODMvITOLRe50no1j4qa7G0X6UdMs439Qx7K7edCXTfPc3Qu040gURQt\\\\\\\/xE9lJ3\\\\\\\/aJVSF2LgMaOy6D8\\\\\\\/GI0rfulqqZFhD+9y4gBYSrDa2XodEz3P8QR\\\\\\\/+xpPNSH4QV2B+emFUIFU9Cytir4jL6PwLRNpqtqtqetxAT9YAiuicas9XGGFXXewC5w5enFZG9KPJPBecUoMeBp7Z\\\\\\\/BwQN8MYUNwvVc2\\\\\\\/iE0NH\\\\\\\/anMdejhvJrdRF1YBu7jINA5IdtGArAdQW+fzV1dH7qca491TZIFG\\\\\\\/waejR+uNu8PewiYzci+KYUag3L3bVEdwSI87oueoljDIegyFe6gCWhYlWBkogbSNr1EkwMIby7dZZPVXFvTP63n2NrSPjF5ytU2MXMnReZ4dA97eNrJahc+2xYIooPqAPx7rEcjjSlFY2CzdxD84NyLnOJDl2+dpbcZsgfhPU7TEfUFGkp4d9EN\\\\\\\/J1OqBkfxJ+kQlc0qX64AQmHo94T\\\\\\\/vZbqGe1FkWzh+k8zJye05j2QzdWcdcA\\\\\\\/a4ZcQOm+aVzsdXlFQrci7ZzKco7tX6+f8N0czuRACVj5K8t9\\\\\\\/5g+BbscPXsTamsr618muiMpfKGprGGKKkFN0T\\\\\\\/s4x18KJmCOga8y3rvGIBlqP7eP2Ktnlg6nUWRzk919Orh9ryFfcQmpiPB9GB0MlzVQo6oC3TURMkmI0JUL5zh860qgKGSBq\\\\\\\/2r5RiVpLr8VDzpXzlbJNeRCdWit4Mn1bwHt9tDn9WCdWhggJ9XtJZCc2ZE2P1Ah5oJnmGDFASwL7zG0dN1uTEdXDwFp6d5L7x51OrmM9t61LtC5\\\\\\\/Qfyrx94QpxwsOKRo2ONi+iwrOcKvTCMs10d+OvsAfxbcbvu66OxgSIwllLuTkVEMlv7zdCgybNRNiPrDpoogRDqMZwLxwVqdNL5LClEi4t6c+bX8DMxoHN10HQIJbxOg\\\\\\\/KPE2\\\\\\\/OiCVbzWLFU8IpqOIfZJFhUFNiuS9rrwYin7IfNdJ6g1IvOq5Ru9b1QSsUjyXCrGAEn5vHBCQZpaShqAW\\\\\\\/hR1uZa85sfbWbr8HEDUd0RPLdcHWpR7xWy\\\\\\\/oaqtaYEloMMHdLs7l17C6W2oxmqYhz+aOPZ6NyDIoAFjeBcWokUm5VmeuPcBTwD2TrbhpUuzZomZedwYwfXE6xolQ9wch8jDhFtcW3Lt\\\\\\\/pH2iWFE\\\\\\\/XQnRcIjk69GzSgm3N+rTUOpJTn2IPMGCtdoXhOjdLI2fS9RUYORw36du\\\\\\\/bXmLclepJUWqGMgBsKCMh7pwzGcpNlRp0j+uG6XT0tgXuKxRvgyOPOqb6vU0XLoWmWMvEmfyDl\\\\\\\/8L6xPbWncoFcXEsMH0wRzQnRGA8Fxux1E7VCbeSG0phiRWB12cD2xq375p\\\\\\\/WDOUfQtbXPab9dIUco+Vgqj2Om4fjcOPUyzicwBb4r6lk1c7ewYFpqiGqgRU5QUXHywvbMHjMRZ2642TT+om1UeUkZXlRRPYCCJlf1yogWAzJZFGGa1iDBPx9VJhgYczhDkC9k46DoS7e+193uqSejJ9z0mhplhJyFYXulmtObAiqVtFDMy78HHiWY5tkfcH2nzp0uDtJup4uO1U4IbmLfBi7RKKYJctPCjN1AWO1FhDh\\\\\\\/9Vt7kMYyuJcznMh6+T+3BO\\\\\\\/5YjLlWfw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pBBo83KAXXpd7Ee3s9L10w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9ae2ee468dde7bc2f7e6da10e52b9d70f6d3d263c4924eef73f749c0ad10f67\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459721 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2137" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C4jAi_8-7VrnQ6Swv5nHj_koQtp6vadGvypOjBr9WEQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2137" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-u2wSAI3pSOq7GR3pLC1iaYMuUmcA253wffBjTAdD8qE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2125" title="A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2147" title="De Novo Transcriptome Sequence Assembly from Coconut Leaves and Seeds with a Focus on Factors Involved in RNA-Directed DNA Methylation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459721" id="node459721" data-pisa="ggg;4/11/2137" data-pisa-master="ggg;g3.114.013052" data-apath="/ggg/4/11/2137.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Clone Mapper: An Online Suite of Tools for RNAi Experiments in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nishant Thakur</span>, <span class="highwire-citation-author" data-delta="1">Nathalie Pujol</span>, <span class="highwire-citation-author" data-delta="2">Laurent Tichit</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1257-6862" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan J. Ewbank</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2137-2145;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013052 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459721" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nishant Thakur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>INSERM U1104, 13288 Marseille, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nishant%2BThakur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thakur%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANishant%2BThakur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nathalie Pujol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>INSERM U1104, 13288 Marseille, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathalie%2BPujol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pujol%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathalie%2BPujol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laurent Tichit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurent%2BTichit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tichit%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurent%2BTichit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan J. Ewbank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BJ.%2BEwbank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ewbank%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BJ.%2BEwbank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1257-6862" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan J. Ewbank</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ewbank@ciml.univ-mrs.fr" class="" data-icon-position="" data-hide-link-title="0">ewbank@ciml.univ-mrs.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/11/2137" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459721" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2137.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459721" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2137.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459721" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/11/2137.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459721" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/11/2137.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459721" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">RNA interference (RNAi), mediated by the introduction of a specific double-stranded RNA, is a powerful method to investigate gene function. It is widely used in the <em>Caenorhabditis elegans</em> research community. An expanding number of laboratories conduct genome-wide RNAi screens, using standard libraries of bacterial clones each designed to produce a specific double-stranded RNA. Proper interpretation of results from RNAi experiments requires a series of analytical steps, from the verification of the identity of bacterial clones, to the identification of the clones’ potential targets. Despite the popularity of the technique, no user-friendly set of tools allowing these steps to be carried out accurately, automatically, and at a large scale, is currently available. We report here the design and production of Clone Mapper, an online suite of tools specifically adapted to the analysis pipeline typical for RNAi experiments with <em>C. elegans</em>. We show that Clone Mapper overcomes the limitations of existing techniques and provide examples illustrating its potential for the identification of biologically relevant genes. The Clone Mapper tools are freely available via <a href="http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html">http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/database" class="hw-term hw-article-keyword hw-article-keyword-database" rel="nofollow">database</a></li><li class="kwd"><a href="/keyword/algorithm" class="hw-term hw-article-keyword hw-article-keyword-algorithm" rel="nofollow">algorithm</a></li><li class="kwd"><a href="/keyword/gene-discovery" class="hw-term hw-article-keyword hw-article-keyword-gene-discovery" rel="nofollow">gene discovery</a></li><li class="kwd"><a href="/keyword/functional-genomics" class="hw-term hw-article-keyword hw-article-keyword-functional-genomics" rel="nofollow">functional genomics</a></li><li class="kwd"><a href="/keyword/mpscan" class="hw-term hw-article-keyword hw-article-keyword-mpscan" rel="nofollow">MPscan</a></li></ul><p id="p-3">RNA interference (RNAi) is a powerful and widely used method to investigate gene function. Researchers using the model nematode <em>Caenorhabditis elegans</em> often use a feeding method for RNAi that involves culturing worms on a bacterial clone expressing a double-stranded RNA (dsRNA) that is intended to target a specific worm gene (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Timmons <em>et al.</em> 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Timmons and Fire 1998</a>). Because worms can be handled robotically, screens can be automated and large numbers of clones tested in parallel (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Squiban <em>et al.</em> 2012</a>). Collections of RNAi clones are available. One made by the Ahringer lab contains polymerase chain reaction (PCR)-amplified fragments of genomic DNA (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kamath <em>et al.</em> 2003</a>), whereas the library made by the Vidal lab (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Rual <em>et al.</em> 2004</a>) was constructed from ORFeome clones, which are derived from cDNA (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Reboul <em>et al.</em> 2001</a>). Part of the strength of the method arises from the fact that knowledge of the sequence of the dsRNA in principle allows the corresponding target gene(s) to be identified.</p><p id="p-4">In common with any large-scale resource, the available bacterial RNAi clone libraries contain errors (<em>e.g.</em>, clone positions inverted on 96-well plates). For the Ahringer library, this error rate is estimated to be approximately 7% (<a href="http://www2.gurdon.cam.ac.uk/~ahringerlab/pages/rnai.html">http://www2.gurdon.cam.ac.uk/~ahringerlab/pages/rnai.html</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Qu <em>et al.</em> 2011</a>). These can be compounded by handling errors during a screen, resulting in error rates as high as 15% (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Pukkila-Worley <em>et al.</em> 2014</a>). This means that clones need to be checked by sequencing to confirm their identity. Interpreting the sequences, to confirm clone identity, can be laborious when dealing with large numbers of clones.</p><p id="p-5">In <em>C. elegans</em> long dsRNAs (often &gt;1 kb) are used, in contrast to the short interfering RNAs (siRNA; typically 19−25 bp long) used in vertebrates. Each dsRNA can thus give rise to a multitude of siRNAs, which complicates target identification. Many published studies have relied on the assignment of targets provided by the community database Wormbase (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Yook <em>et al.</em> 2012</a>). This currently suffers from a number of limitations (Wormbase release WS242). The first is that target identification is based on empirical criteria. The sequence of a “primary target” is at least 95% identical with the clone insert sequence for at least 100 nucleotides (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Fievet <em>et al.</em> 2013</a>); for “secondary targets” the definition is more than 80% identity for greater than 200 nucleotides (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Kamath and Ahringer 2003</a>). These figures are calculated using BLAT (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kent 2002</a>), which is not perfectly adapted to the task for algorithmic reasons (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Imelfort 2009</a>). Further, the target(s) of a given clone are predicted assuming that all RNAi clones contain an insert derived from genomic DNA (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). This assumption is clearly incorrect when applied to Vidal clones generated from intron-containing genes and can lead to overprediction of clone targets (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). At the same time, no secondary targets are predicted for Vidal RNAi clones within Wormbase currently, leading to underprediction of clone targets.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Limitations of current RNAi clone annotation illustrated with edited screen grabs from the Wormbase genome browser (WS242). (A) Wormbase currently reports RNAi clone sequences on the basis of genomic DNA, so that sjj_Y27F2A.h and mv_Y27F2A.h are associated with essentially identical insert sequences. (B) Wormbase consequently erroneously reports intronic genes as cDNA clone targets. In the case shown here, contrary to current Wormbase annotation, inos-1 cannot be a target of mv_C47D12.8. (C) For certain cDNA-derived clones, the genomic positions of oligonucleotide primer pairs, designed on the basis of a historical gene model, do not correspond to a current gene model. For the left-hand ORFeome polymerase chain reaction (PCR) product, mv_B0432.8, the gene model used when the primer pair was designed is shown (B0432.8:wp168), but for the adjacent mv_B0432.9, the model is unavailable. In some cases, as shown here for the clones mv_B0432.8 and mv_B0432.9, the current gene models may require revision since there is conflicting ORF-sequence tag (OST) evidence. The extent of the PCR product predicted by UP-TORR on the basis of mv_B0432.8 primer sequences is indicated by the red rectangle. The reason for this erroneous prediction is not clear." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1859886930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Limitations of current RNAi clone annotation illustrated with edited screen grabs from the Wormbase genome browser (WS242). (A) Wormbase currently reports RNAi clone sequences on the basis of genomic DNA, so that sjj_Y27F2A.h and mv_Y27F2A.h are associated with essentially identical insert sequences. (B) Wormbase consequently erroneously reports intronic genes as cDNA clone targets. In the case shown here, contrary to current Wormbase annotation, &lt;em&gt;inos-1&lt;/em&gt; cannot be a target of mv_C47D12.8. (C) For certain cDNA-derived clones, the genomic positions of oligonucleotide primer pairs, designed on the basis of a historical gene model, do not correspond to a current gene model. For the left-hand ORFeome polymerase chain reaction (PCR) product, mv_B0432.8, the gene model used when the primer pair was designed is shown (B0432.8:wp168), but for the adjacent mv_B0432.9, the model is unavailable. In some cases, as shown here for the clones mv_B0432.8 and mv_B0432.9, the current gene models may require revision since there is conflicting ORF-sequence tag (OST) evidence. The extent of the PCR product predicted by UP-TORR on the basis of mv_B0432.8 primer sequences is indicated by the red rectangle. The reason for this erroneous prediction is not clear.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2137/F1.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/11/2137/F1.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465303" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">Limitations of current RNAi clone annotation illustrated with edited screen grabs from the Wormbase genome browser (WS242). (A) Wormbase currently reports RNAi clone sequences on the basis of genomic DNA, so that sjj_Y27F2A.h and mv_Y27F2A.h are associated with essentially identical insert sequences. (B) Wormbase consequently erroneously reports intronic genes as cDNA clone targets. In the case shown here, contrary to current Wormbase annotation, <em>inos-1</em> cannot be a target of mv_C47D12.8. (C) For certain cDNA-derived clones, the genomic positions of oligonucleotide primer pairs, designed on the basis of a historical gene model, do not correspond to a current gene model. For the left-hand ORFeome polymerase chain reaction (PCR) product, mv_B0432.8, the gene model used when the primer pair was designed is shown (B0432.8:wp168), but for the adjacent mv_B0432.9, the model is unavailable. In some cases, as shown here for the clones mv_B0432.8 and mv_B0432.9, the current gene models may require revision since there is conflicting ORF-sequence tag (OST) evidence. The extent of the PCR product predicted by UP-TORR on the basis of mv_B0432.8 primer sequences is indicated by the red rectangle. The reason for this erroneous prediction is not clear.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">A tool, UP-TORR, has been developed that partially resolves these issues (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Hu <em>et al.</em> 2013</a>). As discussed herein, it too has some drawbacks. UP-TORR is designed for researchers using RNAi in different model systems (human, mouse, <em>Drosophila</em>, <em>C. elegans</em>) and so lacks some basic species-specific functions. For example, the standard <em>C. elegans</em> RNAi clone names (with prefixes “sjj_” and “sjj2_” or “mv_” for the Ahringer or Vidal library clones, respectively) cannot be used as input to UP-TORR. It is also not well adapted to the analysis of large datasets derived from genome-wide screens. We therefore decided to construct a tool specifically for <em>C. elegans</em>, basing target identification on matching fragments of sequence generated <em>in silico</em> from the predicted inserts of RNAi clones. This is part of a collection of tools, called Clone Mapper, that also allow clone verification and sequence retrieval. It is publically available via <a href="http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html">http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data sources</h3><p id="p-8">The reference genome sequence and transcript sequences (WS235 and WS240) were downloaded from <a href="ftp://ftp.wormbase.org/pub/wormbase/species/c_elegans/sequence/">ftp://ftp.wormbase.org/pub/wormbase/species/c_elegans/sequence/</a>. Following the Wormbase convention, transcripts corresponding to coding genes were used for the target library; those corresponding to coding genes and pseudogenes were used for the clone insert library. RNAi reagent information was extracted from the GFF3 file at <a href="ftp://ftp.wormbase.org/pub/wormbase/species/c_elegans/gff/">ftp://ftp.wormbase.org/pub/wormbase/species/c_elegans/gff/</a>. Since the original ORFeome primer sequences were designed (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Reboul <em>et al.</em> 2001</a>), there have been changes in the reference sequence of the <em>C. elegans</em> genome, most recently for release WS235 (see <a href="http://www.wormbase.org/about/wormbase_release_WS235">http://www.wormbase.org/about/wormbase_release_WS235</a>). For some 500 ORFeome products, the original primer sequences no longer match to the genome (K. Howe, personal communication). New (pseudo)-primer sequences designed for these products (incorporating the change present in the WS235 genome sequence) were kindly provided by K. Howe; the relevant file is available on request.</p><p id="p-9">To extract the clone-target gene pairs established by Wormbase (WS235), primary targets were retrieved from <a href="ftp://ftp.wormbase.org/pub/wormbase/species/c_elegans/annotation/pcr_product2gene">ftp://ftp.wormbase.org/pub/wormbase/species/c_elegans/annotation/pcr_product2gene</a>; a list of secondary targets was kindly provided by C. Grove.</p></div><div id="sec-3" class="subsection"><h3>Clone-target identification</h3><p id="p-10">To identify potential targets of RNAi clones, we first generated all possible 21 bp fragments from the predicted sequence of each RNAi clone insert and then we searched for matches between these fragments and transcript sequences (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a>). To rank RNAi clone-target transcript pairs, we calculated a score for each pair using a simple formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2137/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2137/embed/mml-math-1.gif"/></noscript></span></span>The different parameters are defined as follows:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-11">PNO: Possible nonoverlapping segments; maximum number of nonoverlapping segments of length <em>l</em> that can be generated from the clone insert. By default, <em>l</em> = 21 bp.</p></li><li id="list-item-2"><p id="p-12">POS: Possible overlapping segments; maximum number of overlapping segments of length <em>l</em> that can be generated from the clone insert.</p></li><li id="list-item-3"><p id="p-13">MNO: Matched nonoverlapping segments; number of nonoverlapping segments that are found in the targets transcript sequence; with a perfect match MNO = PNO; with no match MNO = 0.</p></li><li id="list-item-4"><p id="p-14">MOS: Matched overlapping segments; number of overlapping segments that are found in the targets transcript sequence; with a perfect match MOS = POS; with no match MOS = 0.</p></li></ul><p id="p-15">In the score, weight is given to the MOS on the assumption that the absolute number of fragments generated from the RNAi clone insert that perfectly match a target transcript influences the probability that the target transcript will be affected. This value is divided by 10 to compensate for the inappropriate weight that would otherwise be assigned to perfect matches of small transcripts to large RNAi clone inserts. The MOS/POS ratio represents the overall sequence similarity between an RNAi clone insert and its target transcript. The more similar they are, the greater the ratio. The MNO/PNO element derives from the assumption that if different siRNAs produced by a clone insert match sequences within the target transcript, then there will be a greater chance of the target transcript being knocked down compared with when siRNAs produced from a single region of a clone insert match only one or a few sequences within the target transcript. The adjusted weight given to the MNO/PNO ratio reflects the assumption that RNAi will be more efficient when siRNAs are generated from multiple nonoverlapping segments that have the potential to target different nonoverlapping regions of a transcript.</p><p id="p-16">The score was given a constant threshold of 100, so that if the calculated score exceeded 100, it was adjusted to 100. The equation for the score can be rearranged to:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2137/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/11/2137/embed/mml-math-2.gif"/></noscript></span></span></div><div id="sec-4" class="subsection"><h3>Software</h3><p id="p-17">For clone mapping, the BLAST program from the National Center for Biotechnology Information (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) was locally installed and run with default parameters. Target mapping used MPScan (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Rivals <em>et al.</em> 2009</a>), with default parameters. For the comparison with published datasets of RNAi screens, when necessary, lists of target genes were updated to WS240 using Wormbase Converter (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Engelmann <em>et al.</em> 2011</a>). Network analysis used the GeneMania plugin (version 2012-08-02-core; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Montojo <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Saito <em>et al.</em> 2012</a>) within Cytoscape (v2.8.1) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Shannon <em>et al.</em> 2003</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Smoot <em>et al.</em> 2011</a>). Programs for the various tools of Clone Mapper were written in Perl and the user interface was developed using HTML, PHP, JavaScript, and MySQL.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Construction of an <em>in silico</em> library of RNAi clones</h3><p id="p-18">Wormbase is the repository for a wealth of genetic, genomic, and bibliographic information. There are, however, some lacunae, such as the fact that the DNA inserts of cDNA-derived RNAi clones are not available. We therefore first constructed libraries of sequences corresponding to the expected inserts of the clones contained within the Ahringer genomic (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Kamath <em>et al.</em> 2003</a>) and Vidal cDNA-derived (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Rual <em>et al.</em> 2004</a>) RNAi collections. For the former, we also included a supplementary set of 3507 clones that recently became available. With the exception of this set, the primers made to amplify clone inserts were designed more than a decade ago. Since then, there have been minor changes in the genome sequence and more extensive changes in gene structure prediction. To correct the former problem, Wormbase calculates pseudo-primer sequences to ensure a perfect alignment between primer and genome sequence (C. Grove, personal communication). Since the Ahringer clones contain genomic inserts, generating insert sequences was relatively straightforward. The relevant coordinates were extracted from the publicly available General Feature Format (gff) file on the Wormbase ftp site and used to retrieve the corresponding genomic sequence for all of the clones. The Vidal RNAi clones are generated from the ORFeome collection. Having extracted the coordinates of the distal end of each mapped oligonucleotide primer pair (kindly provided by K. Howe, Wormbase), we calculated the proximal coordinates using the known length of each primer. The genomic coordinates of each primer were then compared with those of each transcript in an <em>in silico</em> transcript library to identify all transcripts that could potentially be amplified by a given pair of primers (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; see the section <em>Materials and Methods</em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the in silico construction of a library of cDNA-derived RNAi clone inserts. The genomic coordinates of each primer were compared to those of exons in a library of predicted transcripts. For each transcript that could potentially be amplified by a given pair of primers the corresponding sequence was extracted and spliced in silico." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1859886930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation of the &lt;em&gt;in silico&lt;/em&gt; construction of a library of cDNA-derived RNAi clone inserts. The genomic coordinates of each primer were compared to those of exons in a library of predicted transcripts. For each transcript that could potentially be amplified by a given pair of primers the corresponding sequence was extracted and spliced &lt;em&gt;in silico&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2137/F2.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/11/2137/F2.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465201" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Schematic representation of the <em>in silico</em> construction of a library of cDNA-derived RNAi clone inserts. The genomic coordinates of each primer were compared to those of exons in a library of predicted transcripts. For each transcript that could potentially be amplified by a given pair of primers the corresponding sequence was extracted and spliced <em>in silico</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">For close to 15% of the clones in the Vidal library, primer pairs do not match current gene models. In the example shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, the primer pair mv_B0432.8 was designed on the basis of a single gene model that existed until 2003. The predicted exons of this gene were subsequently assigned to 2 genes (<a href="http://www.wormbase.org/db/get?name=B0432.8;class=Gene">B0432.8</a> and <a href="http://www.wormbase.org/db/get?name=B0432.13;class=Gene">B0432.13</a>). The insert sequence of clones like mv_B0432.8 that do not correspond to current gene models cannot thus be readily predicted <em>in silico</em> and we excluded these clones. This resulted in a library of 18,405 transcripts from 13,792 genes, corresponding to 88.2% of all ORFeome clones and 85.9% of the Vidal collection of 11599 RNAi clones. These sequences are available via Clone Mapper (see the section <em>A tool for clone verification</em>).</p></div><div id="sec-7" class="subsection"><h3>A tool for clone verification</h3><p id="p-21">Given the errors that are intrinsic to any large collection of clones, it is indispensable to verify that RNAi clones selected through screens correspond to what they are supposed to be. This is generally done by resequencing and comparing the obtained sequence to the genome of <em>C. elegans</em> and crosschecking the position with that expected for the clone. Checking in this way becomes laborious when one needs to sequence-verify tens or hundreds of clones. We therefore made a BLASTN-based tool to match experimentally determined clone sequences with our <em>in silico</em> clone sequence libraries. It returns an output showing whether the clone is the expected one, and if not what the clone is most likely to be (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This became the first tool in a suite that we have called Clone Mapper and for which we provide a web-based access via <a href="http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html">www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html</a>. The other functionalities are described below.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An example of RNAi clone identification using Clone Mapper. The DNA sequences obtained upon sequencing of 10 RNAi clones, from (Zugasti et al. 2014), were used as input into Clone Mapper. The results obtained, ranked by &#x201C;Aligned region,&#x201D; are shown in this screen-grab. The leftmost column shows the library name of each clone, the next column the name of the clone that best matches the experimentally determined RNAi clone insert sequence. In this example, half the clones appeared to be what was expected; for 3 of 5 of the others, an alternative identity was assigned with high confidence. For the remaining clones only a very short sequence matches a clone in the in silico library. These sequences can be compared directly to the genome of C. elegans by clicking the link in the rightmost column. The exact meaning of the different columns and options is explained in the help document, accessible by clicking the question mark at the top of the screen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1859886930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An example of RNAi clone identification using Clone Mapper. The DNA sequences obtained upon sequencing of 10 RNAi clones, from (Zugasti &lt;em&gt;et al.&lt;/em&gt; 2014), were used as input into Clone Mapper. The results obtained, ranked by &#x201C;Aligned region,&#x201D; are shown in this screen-grab. The leftmost column shows the library name of each clone, the next column the name of the clone that best matches the experimentally determined RNAi clone insert sequence. In this example, half the clones appeared to be what was expected; for 3 of 5 of the others, an alternative identity was assigned with high confidence. For the remaining clones only a very short sequence matches a clone in the &lt;em&gt;in silico&lt;/em&gt; library. These sequences can be compared directly to the genome of &lt;em&gt;C. elegans&lt;/em&gt; by clicking the link in the rightmost column. The exact meaning of the different columns and options is explained in the help document, accessible by clicking the question mark at the top of the screen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2137/F3.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/11/2137/F3.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">An example of RNAi clone identification using Clone Mapper. The DNA sequences obtained upon sequencing of 10 RNAi clones, from (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zugasti <em>et al.</em> 2014</a>), were used as input into Clone Mapper. The results obtained, ranked by “Aligned region,” are shown in this screen-grab. The leftmost column shows the library name of each clone, the next column the name of the clone that best matches the experimentally determined RNAi clone insert sequence. In this example, half the clones appeared to be what was expected; for 3 of 5 of the others, an alternative identity was assigned with high confidence. For the remaining clones only a very short sequence matches a clone in the <em>in silico</em> library. These sequences can be compared directly to the genome of <em>C. elegans</em> by clicking the link in the rightmost column. The exact meaning of the different columns and options is explained in the help document, accessible by clicking the question mark at the top of the screen.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Identifying potential targets of RNAi clones</h3><p id="p-23">Given the shortcomings of current target prediction (see above; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), and given the known molecular basis of RNAi, we next sought to design an alternative approach based on matching short clone-derived sequences against a comprehensive collection of predicted transcript sequences. In <em>C. elegans</em>, dsRNA gives rise to siRNAs of different sizes (19−28 bp); 22 bp is the predominant length, approximately 20% are 21mers, and &lt;10% are shorter than 21 bp (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Gent <em>et al.</em> 2010</a>). In Clone Mapper, therefore, the clone sequence is diced <em>in silico</em> into fragments of a predetermined size. By default Clone Mapper uses 21mers, corresponding to &gt;90% of the <em>in vivo</em> siRNAs. Increasing the oligomer size would restrict the number of potential targets identified, whereas, as discussed below, decreasing the oligomer size would allow more potential targets to be captured, but at the probable expense of increasing the proportion of false-positives. The number of occurrences of each oligomer within each transcript is then counted, and a score (from 0 to 100, with 100 corresponding to a high confidence target) assigned on the basis of a simple formula (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>; see the section <em>Materials and Methods</em>). The method allows the identification of potential targets that would not otherwise be found (<em>e.g.</em>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Basis of the target identification strategy. (A) An example of a target identified by Wormbase but not Clone Mapper. The RNAi clone-transcript pair (sjj_B0222.9 - F15E6.6) displays overall high identity, with blocks of &gt;200 nucleotides with &gt;80% identity, but does not contain a single 21 bp contiguous stretch of identical sequence. (B) The approach implemented in Clone Mapper for defining targets of RNAi clones. The set of possible nonoverlapping 21mer fragments (PNO) are generated starting from the 5&#x2032; end of each predicted clone insert sequence (in black). In the example shown, there are 3 complete (purple, blue, red) and one partial (brown) PNOs. All possible overlapping 21mer segments (POS) are generated and assigned to the corresponding PNO; for simplicity only a selection of POS are shown for each PNO. The library of all transcripts is queried with each POS to identify matched overlapping segments (MOS). In the example shown in the lower part of the panel, a transcript (in bold) with a single difference from the clone insert sequence (*) is shown. The number of PNOs that contain at least one POS that exactly matches a given transcript is counted (MNO; here 3). An example of one matching POS for each PNO is boxed. A score is then calculated (see Materials and Methods). (C) An example of a target identified by Clone Mapper but not Wormbase. Here the RNAi clone insert sequence has multiple stretch of sequence that have perfect matches over more than 21 nt to a target transcript (upper part of the panel; sjj_Y50E8.g and ZK643.8a), but no contiguous region of 100 nt with 95% identity (lower part of the panel; the longest stretch of identity in the selected alignment of one fragment shown here is 31 nt)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1859886930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Basis of the target identification strategy. (A) An example of a target identified by Wormbase but not Clone Mapper. The RNAi clone-transcript pair (sjj_B0222.9 - F15E6.6) displays overall high identity, with blocks of &gt;200 nucleotides with &gt;80% identity, but does not contain a single 21 bp contiguous stretch of identical sequence. (B) The approach implemented in Clone Mapper for defining targets of RNAi clones. The set of possible nonoverlapping 21mer fragments (PNO) are generated starting from the 5&#x2032; end of each predicted clone insert sequence (in black). In the example shown, there are 3 complete (purple, blue, red) and one partial (brown) PNOs. All possible overlapping 21mer segments (POS) are generated and assigned to the corresponding PNO; for simplicity only a selection of POS are shown for each PNO. The library of all transcripts is queried with each POS to identify matched overlapping segments (MOS). In the example shown in the lower part of the panel, a transcript (in bold) with a single difference from the clone insert sequence (*) is shown. The number of PNOs that contain at least one POS that exactly matches a given transcript is counted (MNO; here 3). An example of one matching POS for each PNO is boxed. A score is then calculated (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (C) An example of a target identified by Clone Mapper but not Wormbase. Here the RNAi clone insert sequence has multiple stretch of sequence that have perfect matches over more than 21 nt to a target transcript (upper part of the panel; sjj_Y50E8.g and ZK643.8a), but no contiguous region of 100 nt with 95% identity (lower part of the panel; the longest stretch of identity in the selected alignment of one fragment shown here is 31 nt).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2137/F4.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/11/2137/F4.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465237" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Basis of the target identification strategy. (A) An example of a target identified by Wormbase but not Clone Mapper. The RNAi clone-transcript pair (sjj_B0222.9 - F15E6.6) displays overall high identity, with blocks of &gt;200 nucleotides with &gt;80% identity, but does not contain a single 21 bp contiguous stretch of identical sequence. (B) The approach implemented in Clone Mapper for defining targets of RNAi clones. The set of possible nonoverlapping 21mer fragments (PNO) are generated starting from the 5′ end of each predicted clone insert sequence (in black). In the example shown, there are 3 complete (purple, blue, red) and one partial (brown) PNOs. All possible overlapping 21mer segments (POS) are generated and assigned to the corresponding PNO; for simplicity only a selection of POS are shown for each PNO. The library of all transcripts is queried with each POS to identify matched overlapping segments (MOS). In the example shown in the lower part of the panel, a transcript (in bold) with a single difference from the clone insert sequence (*) is shown. The number of PNOs that contain at least one POS that exactly matches a given transcript is counted (MNO; here 3). An example of one matching POS for each PNO is boxed. A score is then calculated (see <em>Materials and Methods</em>). (C) An example of a target identified by Clone Mapper but not Wormbase. Here the RNAi clone insert sequence has multiple stretch of sequence that have perfect matches over more than 21 nt to a target transcript (upper part of the panel; sjj_Y50E8.g and ZK643.8a), but no contiguous region of 100 nt with 95% identity (lower part of the panel; the longest stretch of identity in the selected alignment of one fragment shown here is 31 nt).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The predicted targets (protein coding genes) for all of the Ahringer and Vidal RNAi clones in our library have been precomputed and can be retrieved by entering a clone name in Clone Mapper. Alternatively, a user can input any sequence and its potential target transcript (protein coding and/or noncoding) will be calculated <em>de novo</em>. Conversely, the identity of clones predicted to target a given gene, or set of genes, can be retrieved by entering the relevant identifiers in the query box under the “Find targets” rubric.</p></div><div id="sec-9" class="subsection"><h3>A comparative analysis of potential targets</h3><p id="p-26">To establish on a genome-wide scale how different the transcript to RNAi clone correspondences obtained with Clone Mapper were from those reported in Wormbase, we conducted a global comparative analysis. We compared the overlap between Wormbase and Clone Mapper predictions across a range of scores for each transcript-RNAi clone pair. With regards the Vidal RNAi clones, even at the greatest scores, Clone Mapper predicted essentially all (98%) of the Wormbase-predicted clone-target pairs (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). The missing fraction all falls into the category of overpredicted (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). On the other hand, 1865 clone-target pairs not reported in Wormbase were found. Relaxing the stringency (decreasing the cut-off score from the maximum of 100) progressively increased this number; using a cut-off score of ≥1, there were 4482, which represents an increase of 30% over the total number of Wormbase predicted clone-target pairs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). For the Ahringer RNAi clones, when the analysis was performed with the maximum cut-off score of 100, 5825 (22.5%) of the Wormbase-predicted clone-target pairs were not found by Clone Mapper, whereas an additional 3552 were found by Clone Mapper alone. In this case, reducing the cut-off score progressively increased both the overlap between the two sets and the number of novel clone-target pairs (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). With a cut-off score of ≥1, there were 2581 and 6539 clone-target pairs specific to Wormbase and Clone Mapper, respectively, with 23266 identified by both. This corresponded respectively to 1518, 3137 and 18664 individual RNAi clones. According to Wormbase annotations, half (49.8%) of the 3137 RNAi clones identified by Clone Mapper as potentially targeting a novel transcript (when using a cut-off score of ≥1) were previously predicted to target a single gene. As discussed below, the choice of cut-off is necessarily arbitrary, but our results, taken together with bioinformatic and experimental investigation of on- and off-target effects (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Rual <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Zhou <em>et al.</em> 2014</a>), suggest that Clone Mapper can identify a substantial number of novel targets.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Target coverage with Clone Mapper. Comparison of the coverage of RNAi clone &#x2013; target pairs for targets identified only with Clone Mapper (red), only by Wormbase (WS235; blue) or both (brown) at different cut-off scores for the Vidal (A) and Ahringer (B) clone collections. (C) Number of protein-coding genes identified by Clone Mapper as potential targets for the Vidal and Ahringer RNAi clones using 2 different scores (1 and the less stringent &gt;0, upper and lower parts of the panel respectively) compared to the total number of predicted protein-coding genes (20540; WS240)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1859886930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Target coverage with Clone Mapper. Comparison of the coverage of RNAi clone &#x2013; target pairs for targets identified only with Clone Mapper (red), only by Wormbase (WS235; blue) or both (brown) at different cut-off scores for the Vidal (A) and Ahringer (B) clone collections. (C) Number of protein-coding genes identified by Clone Mapper as potential targets for the Vidal and Ahringer RNAi clones using 2 different scores (1 and the less stringent &gt;0, upper and lower parts of the panel respectively) compared to the total number of predicted protein-coding genes (20540; WS240).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2137/F5.medium.gif" width="164" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/11/2137/F5.medium.gif" width="164" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465262" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Target coverage with Clone Mapper. Comparison of the coverage of RNAi clone – target pairs for targets identified only with Clone Mapper (red), only by Wormbase (WS235; blue) or both (brown) at different cut-off scores for the Vidal (A) and Ahringer (B) clone collections. (C) Number of protein-coding genes identified by Clone Mapper as potential targets for the Vidal and Ahringer RNAi clones using 2 different scores (1 and the less stringent &gt;0, upper and lower parts of the panel respectively) compared to the total number of predicted protein-coding genes (20540; WS240).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We also calculated the number of protein-coding genes targeted by the combined set of Vidal and Ahringer RNAi clones. Using the arbitrary cut-off score of ≥1, the entire set of clones is predicted to target a total of 19,120 of the 20,540 protein coding genes (93.1%; WS240). This figure only increases marginally, to 19,595 (95.4%), when the cut-off score is reduced to include all targets (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>).</p><p id="p-29">To evaluate the potential impact of these differences in prediction, we compared the list of putative targets in four published data sets with those obtained with Clone Mapper. In the first screen, where just 29 clones were selected (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Pukkila-Worley <em>et al.</em> 2014</a>), Clone Mapper predicted the same targets as published; no novel targets with high scores were identified. In the second specific case (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ceron <em>et al.</em> 2007</a>), 14 of 244 targets were not predicted by Clone Mapper since the insert sequences of the corresponding clones cannot be predicted. On the other hand, Clone Mapper identified 23 new targets with of score &gt;1, 9 of which had a score &gt;50 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013052/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013052/-/DC1/TableS1.xls">Table S1</a>). Similar results were obtained for the two other studies (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Fievet <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Roy <em>et al.</em> 2014</a>) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013052/-/DC1/TableS2.xls">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013052/-/DC1/TableS3.xls">Table S3</a>). In all cases, the novel targets identified with Clone Mapper formed part of a closely linked network (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The interconnectivity of the novel RNAi targets suggests that they may be functionally important for the biological process under study. Such a hypothesis requires direct experimental validation, but the results demonstrate the potential utility of Clone Mapper in gene discovery.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465271/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465271/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Identification of novel target genes with Clone Mapper</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Network analysis of novel RNAi targets. (A) Ceron et al. undertook an RNAi screen to identify genes that interact with the C. elegans retinoblastoma gene lin-35 (Ceron et al. 2007). The list of novel targets identified with Clone Mapper for the RNAi clones selected by Ceron et al. was used as input to GeneMania (black circles), together with lin-35/C32F10.2 (highlighted in yellow) as a seed gene. (B) Fievet et al. performed RNAi screens for C. elegans cell polarity mutants, to generate a polarity network (Fievet et al. 2013). A list of novel targets identified with Clone Mapper for the RNAi clones used by Fievet et al. was used as input to GeneMania (yellow circles), together with the genes corresponding to the 14 mutant strains used in the study (black circles). (C) Roy et al. performed a screen to find components of a regulatory network that promotes developmentally programmed cell-cycle quiescence (Roy et al. 2014). Novel targets identified with Clone Mapper for the RNAi clones used by Roy et al. (yellow), together with common targets (black) were used as input to GeneMania. The networks were trimmed to retain only direct neighbors; unconnected genes are not shown. Genes that are linked within GeneMania but do not appear on the list of RNAi clone targets are shown as gray circles; their size is proportional to the calculated probability score. Networks were displayed in Cytoscape; green edges represent experimentally-determined genetic interactions, pink edges represent experimentally-determined physical interactions for the corresponding proteins, orange and gray edges interactions predicted on the basis of co-expression or literature mining, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1859886930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Network analysis of novel RNAi targets. (A) Ceron &lt;em&gt;et al.&lt;/em&gt; undertook an RNAi screen to identify genes that interact with the &lt;em&gt;C. elegans&lt;/em&gt; retinoblastoma gene &lt;em&gt;lin-35&lt;/em&gt; (Ceron &lt;em&gt;et al.&lt;/em&gt; 2007). The list of novel targets identified with Clone Mapper for the RNAi clones selected by Ceron &lt;em&gt;et al.&lt;/em&gt; was used as input to GeneMania (black circles), together with &lt;em&gt;lin-35&lt;/em&gt;/C32F10.2 (highlighted in yellow) as a seed gene. (B) Fievet &lt;em&gt;et al.&lt;/em&gt; performed RNAi screens for &lt;em&gt;C. elegans&lt;/em&gt; cell polarity mutants, to generate a polarity network (Fievet &lt;em&gt;et al.&lt;/em&gt; 2013). A list of novel targets identified with Clone Mapper for the RNAi clones used by Fievet &lt;em&gt;et al.&lt;/em&gt; was used as input to GeneMania (yellow circles), together with the genes corresponding to the 14 mutant strains used in the study (black circles). (C) Roy &lt;em&gt;et al.&lt;/em&gt; performed a screen to find components of a regulatory network that promotes developmentally programmed cell-cycle quiescence (Roy &lt;em&gt;et al.&lt;/em&gt; 2014). Novel targets identified with Clone Mapper for the RNAi clones used by Roy &lt;em&gt;et al.&lt;/em&gt; (yellow), together with common targets (black) were used as input to GeneMania. The networks were trimmed to retain only direct neighbors; unconnected genes are not shown. Genes that are linked within GeneMania but do not appear on the list of RNAi clone targets are shown as gray circles; their size is proportional to the calculated probability score. Networks were displayed in Cytoscape; green edges represent experimentally-determined genetic interactions, pink edges represent experimentally-determined physical interactions for the corresponding proteins, orange and gray edges interactions predicted on the basis of co-expression or literature mining, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/11/2137/F6.medium.gif" width="150" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/11/2137/F6.medium.gif" width="150" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/11/2137/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465160" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-30" class="first-child">Network analysis of novel RNAi targets. (A) Ceron <em>et al.</em> undertook an RNAi screen to identify genes that interact with the <em>C. elegans</em> retinoblastoma gene <em>lin-35</em> (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Ceron <em>et al.</em> 2007</a>). The list of novel targets identified with Clone Mapper for the RNAi clones selected by Ceron <em>et al.</em> was used as input to GeneMania (black circles), together with <em>lin-35</em>/C32F10.2 (highlighted in yellow) as a seed gene. (B) Fievet <em>et al.</em> performed RNAi screens for <em>C. elegans</em> cell polarity mutants, to generate a polarity network (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Fievet <em>et al.</em> 2013</a>). A list of novel targets identified with Clone Mapper for the RNAi clones used by Fievet <em>et al.</em> was used as input to GeneMania (yellow circles), together with the genes corresponding to the 14 mutant strains used in the study (black circles). (C) Roy <em>et al.</em> performed a screen to find components of a regulatory network that promotes developmentally programmed cell-cycle quiescence (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Roy <em>et al.</em> 2014</a>). Novel targets identified with Clone Mapper for the RNAi clones used by Roy <em>et al.</em> (yellow), together with common targets (black) were used as input to GeneMania. The networks were trimmed to retain only direct neighbors; unconnected genes are not shown. Genes that are linked within GeneMania but do not appear on the list of RNAi clone targets are shown as gray circles; their size is proportional to the calculated probability score. Networks were displayed in Cytoscape; green edges represent experimentally-determined genetic interactions, pink edges represent experimentally-determined physical interactions for the corresponding proteins, orange and gray edges interactions predicted on the basis of co-expression or literature mining, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>A comparison of Clone Mapper with available resources</h3><p id="p-31">Most published reports of RNAi experiments in <em>C. elegans</em> have relied on Wormbase for target identification. As explained previously, Wormbase has several limitations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). It does not include predictions for secondary targets for Vidal RNAi clones, and bases target identification on genomic DNA sequence, which is generally inappropriate for open reading frame−derived clones. This limitation has already been addressed in part by the web-based tool UP-TORR (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Hu <em>et al.</em> 2013</a>) that uses primer sequences to generate <em>in silico</em> a potential clone insert and then identify targets for that insert. UP-TORR, however, does not allow easy bulk clone-target mapping, or the use of the names of the Vidal library clones, for example. Furthermore, the current lower limit for stretches of sequence identity when searching for off-target genes with UP-TORR is 15 bp. This can expand the list of potential hits to an unmanageable size, especially since no score is ascribed to each clone-target pair. Clone Mapper addresses these different issues, and as a species-specific tool has been designed to be as simple and intuitive to use as possible.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465169/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465169/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of tools for RNAi experiments</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-33">With Clone Mapper, we have attempted to satisfy several unmet needs for <em>C. elegans</em> researchers using RNAi. In addition to the central function of identifying potential targets for RNAi clones, it offers tools for clone verification and for the retrieval of RNAi clone and transcript sequences. Clone Mapper complements the tools already available in Wormbase and the web-based tool UP-TORR (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Hu <em>et al.</em> 2013</a>). It can be used in conjunction with Wormbase Converter (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Engelmann <em>et al.</em> 2011</a>) (also available via <a href="http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html">http://www.ciml.univ-mrs.fr/EWBANK_jonathan/software.html</a>) to reanalyze published RNAi datasets. As with any resource, there are certain intrinsic and extrinsic limitations. A total of 1490 Vidal clones that are present in the physical library were purportedly amplified using primers that are not compatible with current gene models. In the example shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>, the mv_B0432.8 primers were used successfully to amplify a cDNA. Sequencing of this PCR product supports the existence of a transcript that spans <a href="http://www.wormbase.org/db/get?name=B0432.8;class=Gene">B0432.8</a> and <a href="http://www.wormbase.org/db/get?name=B0432.13;class=Gene">B0432.13</a>. For a subset of Vidal library clones, it might thus be possible to reconstruct their insert sequences on the basis of OST data, but the OST coverage is incomplete (see for example, <a href="http://www.wormbase.org/species/all/pcr_oligo/mv_B0432.9">mv_B0432.9</a> in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>), and each case would require manual inspection. In common with UP-TORR, we therefore did not attempt to resolve these inconsistencies, nor did we try to evaluate systematically whether the current gene models in question are incorrect.</p><p id="p-34">The ORFeome clones that were used to construct the Vidal RNAi library were generated by amplification of cDNA. Thus, for genes with more than one mRNA isoform, the corresponding clone may contain variants with inserts differing in one or more exon. As a consequence, even when sequence data are available for a given Vidal RNAi clone, one cannot exclude the possibility that multiple different inserts might be present since clones were not always completely sequenced (generally ca. 500 bp from 5′ and 3′ primers) and the prevalence of one splice variant may mask the presence of others (J. Reboul, personal communication).</p><p id="p-35">Although we did not find any inconsistency between the publicly available sequence data and sequence data generated from our in-house library (n &gt; 70; O. Zugasti, unpublished results), to be prudent, when constructing the <em>in silico</em> clone insert library, we assumed that each Vidal RNAi clone did contain inserts corresponding to every possible transcript. If in reality not all isoforms are represented in an RNAi clone, then there will be the potential for some over-prediction of off-target genes. When the clone insert sequence is known, it can be used as the input to Clone Mapper, thus avoiding this problem.</p><p id="p-36">Within Clone Mapper, the length used to search for possible matches between clone insert and target transcript can be defined by the user, with a minimum of 6 bp, so that it can be used to identify potential seed regions for miRNAs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Grosswendt <em>et al.</em> 2014</a>) in complete <em>C. elegans</em> transcripts. It can equally be increased to ensure specificity. The minimal length of sequence identity required to obtain efficient knock-down of green fluorescent protein expression in <em>C</em><em>. elegans</em> has been experimentally determined to be ≥23 bp (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Parrish <em>et al.</em> 2000</a>). It has also been reported that to observe an efficient RNAi effect, the length may vary from 30 to 50 nucleotides (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Rual <em>et al.</em> 2007</a>). We set the default oligomer length at 21 bp since this is the size of a substantial proportion of siRNAs in <em>C. elegans</em> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Gent <em>et al.</em> 2010</a>). Increasing oligomer length will obviously reduce the number of potential targets, whereas decreasing it will broaden the set of potential targets. The different targets are assigned scores that help in the evaluation of whether a transcript is likely to be a high-confidence target. It also permits users to evaluate the consequences of setting different values for these parameters. To be inclusive but selective, one could decrease oligomer length and then set a high cut-off score. There is an element of arbitrariness in choosing oligomer length and cut-off scores, but this reflects a biological reality. The efficiency with which a given transcript is knocked down depends not only on its sequence, but also on the level at which it is expressed, the tissue that it is expressed in, and on the expression of any other transcripts that share sequence with it. Indeed, siRNAs generated from a diced primary target (secondary siRNAs) can knock-down mRNAs that are not a direct target of siRNA derived from an RNAi clone (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Zhou <em>et al.</em> 2014</a>). We did not implement this level of target identification as part of the tool, but users can search for these indirect hits by inputting the sequence of any target transcript into the <em>de novo</em> target prediction utility that is available within Clone Mapper.</p><p id="p-37">The modular architecture of Clone Mapper also allows users to choose the best reagent for specifically knocking down a given gene. The identity of clones predicted to target a given gene, or set of genes, can be retrieved. Then one can check the number of off-target genes predicted for each clone, to identify the most specific clone.</p><p id="p-38">Finally, an <em>in silico</em> reanalysis of selected published RNAi datasets identified new target genes. The demonstration of the functional relevance of these targets is beyond the scope of this study, but these results illustrate Clone Mapper’s potential for gene discovery.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">We thank the bioinformatics platforms of the CIML and the Laboratoire d’Informatique Fondamentale de Marseille for providing computing resources, database, and web servers. We are grateful to system administrators Manuel Bertrand and Kai Poutrain for their support and input. We thank the staff at Wormbase, particularly Todd Harris, Chris Grove, and Kevin Howe for their invaluable help, R. Mako Saito and R. Pukkila-Worley for providing clone lists and Ian Hope and Jérôme Reboul for comments. This work was funded by institutional grants from INSERM and CNRS, and a program grant from the ANR (ANR-12-BSV3-0001-01 to J.J.E.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-40">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013052/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.013052/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-41"><em>Communicating editor: M. C. Zetka</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-07-07"><span class="received-label">Received </span>July 7, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-08-25"><span class="accepted-label">Accepted </span>August 25, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Thakur <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.2" data-doi="10.1186/1471-213X-7-30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ceron</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rual</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Large-scale RNAi screens identify novel genes that interact with the <em>C. elegans</em> retinoblastoma pathway as well as splicing-related components with synMuv B activity.</span> <abbr class="cit-jnl-abbrev">BMC Dev. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bdevelopmental%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BDev%2BBiol%26rft.aulast%253DCeron%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.spage%253D30%26rft.epage%253D30%26rft.atitle%253DLarge-scale%2BRNAi%2Bscreens%2Bidentify%2Bnovel%2Bgenes%2Bthat%2Binteract%2Bwith%2Bthe%2BC.%2Belegans%2Bretinoblastoma%2Bpathway%2Bas%2Bwell%2Bas%2Bsplicing-related%2Bcomponents%2Bwith%2BsynMuv%2BB%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-7-30%26rft_id%253Dinfo%253Apmid%252F17417969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-7-30&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17417969&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.3" data-doi="10.1371/journal.pone.0019055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engelmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tichit</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montanana-Sanchis</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A comprehensive analysis of gene expression changes provoked by bacterial and fungal infection in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEngelmann%26rft.auinit1%253DI.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19055%26rft.epage%253De19055%26rft.atitle%253DA%2Bcomprehensive%2Banalysis%2Bof%2Bgene%2Bexpression%2Bchanges%2Bprovoked%2Bby%2Bbacterial%2Band%2Bfungal%2Binfection%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019055%26rft_id%253Dinfo%253Apmid%252F21602919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21602919&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.4" data-doi="10.1038/ncb2639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fievet</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naganathan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeiser</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Systematic genetic interaction screens uncover cell polarity regulators and functional redundancy.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCell%2BBiol.%26rft.volume%253D15%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2639%26rft_id%253Dinfo%253Apmid%252F23242217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23242217&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312841300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.5" data-doi="10.1016/j.molcel.2010.01.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamm</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavelec</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniar</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parameswaran</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Distinct phases of siRNA synthesis in an endogenous RNAi pathway in <em>C. elegans</em> soma.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGent%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D679%26rft.epage%253D689%26rft.atitle%253DDistinct%2Bphases%2Bof%2BsiRNA%2Bsynthesis%2Bin%2Ban%2Bendogenous%2BRNAi%2Bpathway%2Bin%2BC.%2Belegans%2Bsoma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.01.012%26rft_id%253Dinfo%253Apmid%252F20116306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.01.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20116306&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275702500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.6" data-doi="10.1016/j.molcel.2014.03.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grosswendt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipchyk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manzano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klironomos</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unambiguous identification of miRNA: target site interactions by different types of ligation reactions.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1042</span>–<span class="cit-lpage">1054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D54%26rft.spage%253D1042%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.03.049%26rft_id%253Dinfo%253Apmid%252F24857550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.03.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24857550&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340628000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.7" data-doi="10.1534/genetics.113.151340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roesel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flockhart</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrimon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">UP-TORR: online tool for accurate and up-to-date annotation of RNAi reagents.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151340%26rft_id%253Dinfo%253Apmid%252F23792952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk1LzEvMzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2137.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imelfort</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sequence comparison tools</span>, pp. <span class="cit-fpage">13</span>–<span class="cit-lpage">37</span> in <span class="cit-source">Bioinformatics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-name">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.9" data-doi="10.1016/S1046-2023(03)00050-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahringer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide RNAi screening in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DKamath%26rft.auinit1%253DR.%2BS.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D313%26rft.epage%253D321%26rft.atitle%253DGenome-wide%2BRNAi%2Bscreening%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252803%252900050-1%26rft_id%253Dinfo%253Apmid%252F12828945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(03)00050-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12828945&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184116700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.10" data-doi="10.1038/nature01278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Systematic functional analysis of the <em>Caenorhabditis elegans</em> genome using RNAi.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKamath%26rft.auinit1%253DR.%2BS.%26rft.volume%253D421%26rft.issue%253D6920%26rft.spage%253D231%26rft.epage%253D237%26rft.atitle%253DSystematic%2Bfunctional%2Banalysis%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bgenome%2Busing%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01278%26rft_id%253Dinfo%253Apmid%252F12529635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12529635&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180397600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.11" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT—the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.12" data-doi="10.1093/bioinformatics/btq562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montojo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuberi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GeneMANIA Cytoscape plugin: fast gene function predictions on the desktop.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2927</span>–<span class="cit-lpage">2928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq562%26rft_id%253Dinfo%253Apmid%252F20926419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjIvMjkyNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjEzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.13" data-doi="10.1016/S1097-2765(00)00106-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleenor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mello</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Functional anatomy of a dsRNA trigger. Differential requirement for the two trigger strands in RNA interference.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DParrish%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D1077%26rft.epage%253D1087%26rft.atitle%253DFunctional%2Banatomy%2Bof%2Ba%2BdsRNA%2Btrigger%253A%2Bdifferential%2Brequirement%2Bfor%2Bthe%2Btwo%2Btrigger%2Bstrands%2Bin%2BRNA%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252900106-4%26rft_id%253Dinfo%253Apmid%252F11106747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)00106-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11106747&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165472100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.14" data-doi="10.1371/journal.ppat.1004143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pukkila-Worley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feinbaum</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evolutionarily conserved mediator subunit MDT-15/MED15 links protective innate immune responses and xenobiotic detoxification.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D10%26rft.spage%253De1004143%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004143%26rft_id%253Dinfo%253Apmid%252F24875643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24875643&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.15" data-doi="10.1186/1471-2164-12-170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reliability analysis of the Ahringer <em>Caenorhabditis elegans</em> RNAi feeding library: a guide for genome-wide screens.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQu%26rft.auinit1%253DW.%26rft.volume%253D12%26rft.spage%253D170%26rft.epage%253D170%26rft.atitle%253DReliability%2Banalysis%2Bof%2Bthe%2BAhringer%2BCaenorhabditis%2Belegans%2BRNAi%2Bfeeding%2Blibrary%253A%2Ba%2Bguide%2Bfor%2Bgenome-wide%2Bscreens.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-170%26rft_id%253Dinfo%253Apmid%252F21453524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21453524&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.16" data-doi="10.1038/85913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reboul</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaglio</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzellas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Open-reading-frame sequence tags (OSTs) support the existence of at least 17,300 genes in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DReboul%26rft.auinit1%253DJ.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D332%26rft.epage%253D336%26rft.atitle%253DOpen-reading-frame%2Bsequence%2Btags%2B%2528OSTs%2529%2Bsupport%2Bthe%2Bexistence%2Bof%2Bat%2Bleast%2B17%252C300%2Bgenes%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F85913%26rft_id%253Dinfo%253Apmid%252F11242119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/85913&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11242119&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167304200029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-4.11.2137.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivals</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiiskinen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalsi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarhio</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">mpscan: fast localisation of multiple reads in genomes</span>, pp. <span class="cit-fpage">246</span>–<span class="cit-lpage">260</span> in <span class="cit-source">Algorithms in Bioinformatics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Warnow</span>  <span class="cit-name-given-names">T.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-4.11.2137.18" data-doi="10.1534/g3.114.010546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobin</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Memar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beltz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A complex regulatory network coordinating cell cycles during C. elegans development is revealed by a genome-wide RNAi screen.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010546%26rft_id%253Dinfo%253Apmid%252F24584095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNS83OTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzExLzIxMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.19" data-doi="10.1101/gr.2505604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rual</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceron</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koreth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicot</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Toward improving <em>Caenorhabditis elegans</em> phenome mapping with an ORFeome-based RNAi library.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2162</span>–<span class="cit-lpage">2168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRual%26rft.auinit1%253DJ.-F.%26rft.volume%253D14%26rft.issue%253D10b%26rft.spage%253D2162%26rft.epage%253D2168%26rft.atitle%253DToward%2BImproving%2BCaenorhabditis%2Belegans%2BPhenome%2BMapping%2BWith%2Ban%2BORFeome-Based%2BRNAi%2BLibrary%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2505604%26rft_id%253Dinfo%253Apmid%252F15489339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjExOiIxNC8xMGIvMjE2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjEzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.20" data-doi="10.1186/1471-2164-8-106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rual</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klitgord</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achaz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Novel insights into RNAi off-target effects using <em>C. elegans</em> paralogs.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRual%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D8%26rft.spage%253D106%26rft.epage%253D106%26rft.atitle%253DNovel%2Binsights%2Binto%2BRNAi%2Boff-target%2Beffects%2Busing%2BC.%2Belegans%2Bparalogs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-106%26rft_id%253Dinfo%253Apmid%252F17445269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17445269&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.21" data-doi="10.1038/nmeth.2212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruscheinski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A travel guide to Cytoscape plugins.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSaito%26rft.auinit1%253DR.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D1069%26rft.epage%253D1076%26rft.atitle%253DA%2Btravel%2Bguide%2Bto%2BCytoscape%2Bplugins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2212%26rft_id%253Dinfo%253Apmid%252F23132118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23132118&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310848700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.22" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.23" data-doi="10.1093/bioinformatics/btq675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruscheinski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cytoscape 2.8: new features for data integration and network visualization.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq675%26rft_id%253Dinfo%253Apmid%252F21149340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy8zLzQzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTEvMjEzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Squiban</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belougne</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewbank</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zugasti</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative and automated high-throughput genome-wide RNAi screens in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">e3448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253D60%26rft.spage%253De3448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.25" data-doi="10.1038/27579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timmons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Specific interference by ingested dsRNA.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">395</span>: <span class="cit-fpage">854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTimmons%26rft.auinit1%253DL.%26rft.volume%253D395%26rft.issue%253D6705%26rft.spage%253D854%26rft.epage%253D854%26rft.atitle%253DSpecific%2Binterference%2Bby%2Bingested%2BdsRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F27579%26rft_id%253Dinfo%253Apmid%252F9804418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/27579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9804418&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076713400039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.26" data-doi="10.1016/S0378-1119(00)00579-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timmons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Court</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ingestion of bacterially expressed dsRNAs can produce specific and potent genetic interference in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DTimmons%26rft.auinit1%253DL.%26rft.volume%253D263%26rft.issue%253D1-2%26rft.spage%253D103%26rft.epage%253D112%26rft.atitle%253DIngestion%2Bof%2Bbacterially%2Bexpressed%2BdsRNAs%2Bcan%2Bproduce%2Bspecific%2Band%2Bpotent%2Bgenetic%2Binterference%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900579-5%26rft_id%253Dinfo%253Apmid%252F11223248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00579-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11223248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167352400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.27" data-doi="10.1093/nar/gkr954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yook</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabunoc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">WormBase 2012: more genomes, more data, new website.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D735</span>–<span class="cit-lpage">D741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr954%26rft_id%253Dinfo%253Apmid%252F22067452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENzM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.28" data-doi="10.1534/genetics.113.159780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Nuclear RNAi contributes to the silencing of off-target genes and repetitive sequences in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159780%26rft_id%253Dinfo%253Apmid%252F24532782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvMTIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMS8yMTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.11.2137.29" data-doi="10.1038/ni.2957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zugasti</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bose</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squiban</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belougne</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurz</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Activation of a G protein-coupled receptor by its endogenous ligand triggers the innate immune response of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BImmunol.%26rft.volume%253D15%26rft.spage%253D833%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.2957%26rft_id%253Dinfo%253Apmid%252F25086774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.2957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25086774&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F11%2F2137.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/11/2137.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/11/2125" title="A Method for Sporulating Budding Yeast Cells That Allows for Unbiased Identification of Kinase Substrates Using Stable Isotope Labeling by Amino Acids in Cell Culture" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/11/2147" title="De Novo Transcriptome Sequence Assembly from Coconut Leaves and Seeds with a Focus on Factors Involved in RNA-Directed DNA Methylation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/11"><span>Volume 4 Issue 11, November 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/11.cover-source.jpg?itok=oYWFGPiM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/11/2137.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jKVWsDbde848GIISU011WHhMJsoC43ZjVfarHKdRUfs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/11/2137" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/11/2137" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/11/2137" class="active" data-icon-position="" data-hide-link-title="0">Clone Mapper: An Online Suite of Tools for RNAi Experiments in Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459721" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Lt3FBvcq5Sz4M0jFKS3TMlaxES3uKHjpdfhZZoCgRzg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2137.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/11/2137" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kPeF2AlvobwQ2UHRxiGpfS9aQF9XEYKIHenb87Ra6a8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/11/2137" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/11/2137.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459721" id="node459721--2" data-pisa="ggg;4/11/2137" data-pisa-master="ggg;g3.114.013052" data-apath="/ggg/4/11/2137.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Clone Mapper: An Online Suite of Tools for RNAi Experiments in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nishant Thakur</span>, <span class="highwire-citation-author" data-delta="1">Nathalie Pujol</span>, <span class="highwire-citation-author" data-delta="2">Laurent Tichit</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1257-6862" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan J. Ewbank</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2137-2145;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013052 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459721--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nishant Thakur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>INSERM U1104, 13288 Marseille, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nishant%2BThakur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thakur%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANishant%2BThakur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nathalie Pujol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>INSERM U1104, 13288 Marseille, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathalie%2BPujol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pujol%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathalie%2BPujol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laurent Tichit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurent%2BTichit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tichit%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurent%2BTichit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan J. Ewbank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BJ.%2BEwbank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ewbank%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BJ.%2BEwbank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1257-6862" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan J. Ewbank</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ewbank@ciml.univ-mrs.fr" class="" data-icon-position="" data-hide-link-title="0">ewbank@ciml.univ-mrs.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/11/2137" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/11/2137&amp;title=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/11/2137&amp;title=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/11/2137&amp;title=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/11/2137&amp;text=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/11/2137&amp;title=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/11/2137&amp;t=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/11/2137&amp;title=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/11/2137&amp;title=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459721/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459721" id="node459721--3" data-pisa="ggg;4/11/2137" data-pisa-master="ggg;g3.114.013052" data-apath="/ggg/4/11/2137.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Clone Mapper: An Online Suite of Tools for RNAi Experiments in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nishant Thakur</span>, <span class="highwire-citation-author" data-delta="1">Nathalie Pujol</span>, <span class="highwire-citation-author" data-delta="2">Laurent Tichit</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-1257-6862" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan J. Ewbank</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2137-2145;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.013052 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459721--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nishant Thakur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>INSERM U1104, 13288 Marseille, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nishant%2BThakur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thakur%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANishant%2BThakur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nathalie Pujol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>INSERM U1104, 13288 Marseille, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathalie%2BPujol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pujol%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathalie%2BPujol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Laurent Tichit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurent%2BTichit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tichit%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurent%2BTichit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan J. Ewbank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre d’Immunologie de Marseille-Luminy, UM2 Aix-Marseille Université, Case 906, 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institut de Mathématiques de Marseille, Site Sud, Campus de Luminy, Case 907 13288 Marseille Cedex 9, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CNRS UMR7280, 13288 Marseille, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BJ.%2BEwbank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ewbank%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BJ.%2BEwbank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1257-6862" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan J. Ewbank</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ewbank@ciml.univ-mrs.fr" class="" data-icon-position="" data-hide-link-title="0">ewbank@ciml.univ-mrs.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2137&amp;rft.epage=2145&amp;rft.atitle=Clone%20Mapper%3A%20An%20Online%20Suite%20of%20Tools%20for%20RNAi%20Experiments%20in%20Caenorhabditis%20elegans&amp;rft.volume=4&amp;rft.issue=11&amp;rft.date=2014-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Thakur%2C+Nishant&amp;rft.au=Pujol%2C+Nathalie&amp;rft.au=Tichit%2C+Laurent&amp;rft.au=Ewbank%2C+Jonathan+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459721/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459721/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459721/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459721/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459721/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459721/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459721/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459721/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459721/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459721/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459721/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459721/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/11/2137" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/11/2137.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/11/2137.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/11/2137.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/11/2137.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
