<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/4/1191" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/4/1191.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/4/1191.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/4/1191.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/4/1191.atom" />
<meta name="HW.pisa" content="ggg;7/4/1191" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair" />
<meta name="DC.Identifier" content="10.1534/g3.116.037390" />
<meta name="DC.Date" content="2017-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Zapotoczny and Sekelsky. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DNA double-strand breaks (DSBs) are one of the most deleterious types of lesions to the genome. Synthesis-dependent strand annealing (SDSA) is thought to be a major pathway of DSB repair, but direct tests of this model have only been conducted in budding yeast and Drosophila . To better understand this pathway, we developed an SDSA assay for use in human cells. Our results support the hypothesis that SDSA is an important DSB repair mechanism in human cells. We used siRNA knockdown to assess the roles of a number of helicases suggested to promote SDSA. None of the helicase knockdowns reduced SDSA, but knocking down BLM or RTEL1 increased SDSA. Molecular analysis of repair products suggests that these helicases may prevent long-tract repair synthesis. Since the major alternative to SDSA (repair involving a double-Holliday junction intermediate) can lead to crossovers, we also developed a fluorescent assay that detects crossovers generated during DSB repair. Together, these assays will be useful in investigating features and mechanisms of SDSA and crossover pathways in human cells." />
<meta name="DC.Contributor" content="Grzegorz Zapotoczny" />
<meta name="DC.Contributor" content="Jeff Sekelsky" />
<meta name="article:published_time" content="2017-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DNA double-strand breaks (DSBs) are one of the most deleterious types of lesions to the genome. Synthesis-dependent strand annealing (SDSA) is thought to be a major pathway of DSB repair, but direct tests of this model have only been conducted in budding yeast and &lt;i&gt;Drosophila&lt;/i&gt;. To better understand this pathway, we developed an SDSA assay for use in human cells. Our results support the hypothesis that SDSA is an important DSB repair mechanism in human cells. We used siRNA knockdown to assess the roles of a number of helicases suggested to promote SDSA. None of the helicase knockdowns reduced SDSA, but knocking down BLM or RTEL1 increased SDSA. Molecular analysis of repair products suggests that these helicases may prevent long-tract repair synthesis. Since the major alternative to SDSA (repair involving a double-Holliday junction intermediate) can lead to crossovers, we also developed a fluorescent assay that detects crossovers generated during DSB repair. Together, these assays will be useful in investigating features and mechanisms of SDSA and crossover pathways in human cells.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/04/01" />
<meta name="citation_mjid" content="ggg;7/4/1191" />
<meta name="citation_id" content="7/4/1191" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/4/1191" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/4/1191.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/4/1191.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/4/1191.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.037390" />
<meta name="citation_pmid" content="28179392" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1191" />
<meta name="citation_lastpage" content="1199" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Grzegorz Zapotoczny" />
<meta name="citation_author_institution" content="Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Jeff Sekelsky" />
<meta name="citation_author_institution" content="Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Biology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Integrative Program for Biological and Genome Sciences, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="sekelsky@unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4424-677X" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_author=M. McVey;citation_author=J.J. Sekelsky;citation_title=Drosophila BLM in Double-Strand Break Repair by Synthesis-Dependent Strand Annealing;citation_pages=265-267;citation_volume=299;citation_year=2003;citation_issue=5604;citation_pmid=12522255;citation_doi=10.1126/science.1077198" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Allers;citation_author=M. Lichten;citation_title=Differential timing and control of noncrossover and crossover recombination during meiosis.;citation_pages=47-57;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461701;citation_doi=10.1016/S0092-8674(01)00416-0" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=S. L. Andersen;citation_author=J. Sekelsky;citation_title=Meiotic versus mitotic recombination: two different routes for double-strand break repair: the different functions of meiotic versus mitotic DSB repair are reflected in different pathway usage and different outcomes.;citation_pages=1058-1066;citation_volume=32;citation_year=2010;citation_issue=12;citation_pmid=20967781;citation_doi=10.1002/bies.201000087" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Anglana;citation_author=S. Bacchetti;citation_title=Construction of a recombinant adenovirus for efficient delivery of the I-SceI yeast endonuclease to human cells and its application in the in vivo cleavage of chromosomes to expose new potential telomeres.;citation_volume=27;citation_year=1999;citation_pmid=10518621;citation_doi=10.1093/nar/27.21.4276" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=L. J. Barber;citation_author=J. L. Youds;citation_author=J. D. Ward;citation_author=M. J. Mcilwraith;citation_author=N. J. O’neil;citation_title=RTEL1 maintains genomic stability by suppressing homologous recombination.;citation_pages=261-271;citation_volume=135;citation_year=2008;citation_issue=2;citation_pmid=18957201;citation_doi=10.1016/j.cell.2008.08.016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. M. Byrne;citation_author=L. Ortiz;citation_author=P. Mali;citation_author=J. Aach;citation_author=G. M. Church;citation_title=Multi-kilobase homozygous targeted gene replacement in human induced pluripotent stem cells.;citation_volume=43;citation_year=2015;citation_pmid=25414332;citation_doi=10.1093/nar/gku1246" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. H. Chan;citation_author=A. M. Yu;citation_author=M. McVey;citation_title=Dual roles for DNA polymerase theta in alternative end-joining repair of double-strand breaks in Drosophila.;citation_pages=e1001005-e1001005;citation_volume=6;citation_year=2010;citation_issue=7;citation_pmid=20617203;citation_doi=10.1371/journal.pgen.1001005" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=G. Chandramouly;citation_author=A. Kwok;citation_author=B. Huang;citation_author=N. A. Willis;citation_author=A. Xie;citation_title=BRCA1 and CtIP suppress long-tract gene conversion between sister chromatids.;citation_pages=2404;citation_volume=4;citation_year=2013;citation_pmid=23994874;citation_doi=10.1038/ncomms3404" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. Cong;citation_author=F. A. Ran;citation_author=D. Cox;citation_author=S. Lin;citation_author=R. Barretto;citation_title=Multiplex genome engineering using CRISPR/Cas systems.;citation_volume=339;citation_year=2013;citation_pmid=23287718;citation_doi=10.1126/science.1231143" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Crismani;citation_author=C. Girard;citation_author=N. Froger;citation_author=M. Pradillo;citation_author=J. L. Santos;citation_title=FANCM Limits Meiotic Crossovers;citation_pages=1588-1590;citation_volume=336;citation_year=2012;citation_issue=6088;citation_pmid=22723424;citation_doi=10.1126/science.1220381" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Dayani;citation_author=G. Simchen;citation_author=M. Lichten;citation_title=Meiotic recombination intermediates are resolved with minimal crossover formation during return-to-growth, an analogue of the mitotic cell cycle.;citation_pages=e1002083-e1002083;citation_volume=7;citation_year=2011;citation_issue=5;citation_pmid=21637791;citation_doi=10.1371/journal.pgen.1002083" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. De Muyt;citation_author=L. Jessop;citation_author=E. Kolar;citation_author=A. Sourirajan;citation_author=J. Chen;citation_title=BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.;citation_pages=43-53;citation_volume=46;citation_year=2012;citation_issue=1;citation_pmid=22500736;citation_doi=10.1016/j.molcel.2012.02.020" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. Ding;citation_author=M. Schertzer;citation_author=X. Wu;citation_author=M. Gertsenstein;citation_author=S. Selig;citation_title=Regulation of murine telomere length by Rtel: an essential gene encoding a helicase-like protein.;citation_pages=873-886;citation_volume=117;citation_year=2004;citation_issue=7;citation_pmid=15210109;citation_doi=10.1016/j.cell.2004.05.026" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. A. Forbes;citation_author=D. Beare;citation_author=P. Gunasekaran;citation_author=K. Leung;citation_author=N. Bindal;citation_title=COSMIC: exploring the world’s knowledge of somatic mutations in human cancer.;citation_volume=43;citation_year=2015;citation_pmid=25355519;citation_doi=10.1093/nar/gku1075" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. German;citation_author=S. Schonberg;citation_author=E. Louie;citation_author=R. S. Chaganti;citation_title=Bloom's syndrome. IV. Sister-chromatid exchanges in lymphocytes.;citation_pages=248-255;citation_volume=29;citation_year=1977;citation_issue=3;citation_pmid=868871" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=R. Holliday;citation_title=A mechanism for gene conversion in fungi.;citation_pages=282-304;citation_volume=78;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. K. Kuo;citation_author=S. Mcmahan;citation_author=C. M. Rota;citation_author=K. P. Kohl;citation_author=J. Sekelsky;citation_title=Drosophila FANCM helicase prevents spontaneous mitotic crossovers generated by the MUS81 and SLX1 nucleases.;citation_volume=198;citation_year=2014;citation_pmid=25205745;citation_doi=10.1534/genetics.114.168096" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kurkulos;citation_author=J. M. Weinberg;citation_author=D. Roy;citation_author=S. M. Mount;citation_title=P element-mediated in vivo deletion analysis of white-apricot: deletions between direct repeats are strongly favored.;citation_pages=1001-1011;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005410" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Lorenz;citation_author=F. Osman;citation_author=W. Sun;citation_author=S. Nandi;citation_author=R. Steinacher;citation_title=The Fission Yeast FANCM Ortholog Directs Non-Crossover Recombination During Meiosis;citation_pages=1585-1588;citation_volume=336;citation_year=2012;citation_issue=6088;citation_pmid=22723423;citation_doi=10.1126/science.1220111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Mali;citation_author=L. Yang;citation_author=K. M. Esvelt;citation_author=J. Aach;citation_author=M. Guell;citation_title=RNA-guided human genome engineering via Cas9.;citation_volume=339;citation_year=2013;citation_pmid=23287722;citation_doi=10.1126/science.1232033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. McVey;citation_author=M. Adams;citation_author=E. Staeva-Vieira;citation_author=J. J. Sekelsky;citation_title=Evidence for Multiple Cycles of Strand Invasion During Repair of Double-Strand Gaps in Drosophila;citation_pages=699-705;citation_volume=167;citation_year=2004;citation_issue=2;citation_pmid=15238522;citation_doi=10.1534/genetics.103.025411" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=K. Mitchel;citation_author=H. Zhang;citation_author=C. Welz-Voegele;citation_author=S. Jinks-Robertson;citation_title=Molecular structures of crossover and noncrossover intermediates during gap repair in yeast: implications for recombination.;citation_pages=211-222;citation_volume=38;citation_year=2010;citation_issue=2;citation_pmid=20417600;citation_doi=10.1016/j.molcel.2010.02.028" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Mitchel;citation_author=K. Lehner;citation_author=S. Jinks-Robertson;citation_title=Heteroduplex DNA position defines the roles of the Sgs1, Srs2, and Mph1 helicases in promoting distinct recombination outcomes.;citation_pages=e1003340;citation_volume=9;citation_year=2013;citation_pmid=23516370;citation_doi=10.1371/journal.pgen.1003340" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Miura;citation_author=Y. Yamana;citation_author=T. Usui;citation_author=H. I. Ogawa;citation_author=M. T. Yamamoto;citation_title=Homologous recombination via synthesis-dependent strand annealing in yeast requires the Irc20 and Srs2 DNA helicases.;citation_volume=191;citation_year=2012;citation_pmid=22367032;citation_doi=10.1534/genetics.112.139105" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. Nassif;citation_author=J. Penney;citation_author=S. Pal;citation_author=W. R. Engels;citation_author=G. B. Gloor;citation_title=Efficient copying of nonhomologous sequences from ectopic sites via P-element-induced gap repair.;citation_pages=1613-1625;citation_volume=14;citation_year=1994;citation_issue=3;citation_pmid=8114699;citation_doi=10.1128/MCB.14.3.1613" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=D. Nevozhay;citation_author=T. Zal;citation_author=G. Balazsi;citation_title=Transferring a synthetic gene circuit from yeast to mammalian cells.;citation_pages=1451;citation_volume=4;citation_year=2013;citation_pmid=23385595;citation_doi=10.1038/ncomms2471" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Paliwal;citation_author=R. Kanagaraj;citation_author=A. Sturzenegger;citation_author=K. Burdova;citation_author=P. Janscak;citation_title=Human RECQ5 helicase promotes repair of DNA double-strand breaks by synthesis-dependent strand annealing.;citation_volume=42;citation_year=2014;citation_pmid=24319145;citation_doi=10.1093/nar/gkt1263" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. J. Pierce;citation_author=R. D. Johnson;citation_author=L. H. Thompson;citation_author=M. Jasin;citation_title=XRCC3 promotes homology-directed repair of DNA damage in mammalian cells;citation_pages=2633-2638;citation_volume=13;citation_year=1999;citation_issue=20;citation_pmid=10541549;citation_doi=10.1101/gad.13.20.2633" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. J. Pierce;citation_author=P. Hu;citation_author=M. Han;citation_author=N. Ellis;citation_author=M. Jasin;citation_title=Ku DNA end-binding protein modulates homologous repair of double-strand breaks in mammalian cells;citation_pages=3237-3242;citation_volume=15;citation_year=2001;citation_issue=24;citation_pmid=11751629;citation_doi=10.1101/gad.946401" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=G. Sarek;citation_author=J. B. Vannier;citation_author=S. Panier;citation_author=J. H. Petrini;citation_author=S. J. Boulton;citation_title=TRF2 recruits RTEL1 to telomeres in S phase to promote T-Loop unwinding.;citation_pages=622-635;citation_volume=57;citation_year=2015;citation_pmid=25620558;citation_doi=10.1016/j.molcel.2014.12.024" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Séguéla-Arnaud;citation_author=W. Crismani;citation_author=C. Larcheveque;citation_author=J. Mazel;citation_author=N. Froger;citation_title=Multiple mechanisms limit meiotic crossovers: TOP3α and two BLM homologs antagonize crossovers in parallel to FANCM.;citation_volume=112;citation_year=2015;citation_pmid=25825745;citation_doi=10.1073/pnas.1423107112" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Seki;citation_author=T. Nakagawa;citation_author=T. Seki;citation_author=G. Kato;citation_author=S. Tada;citation_title=Bloom Helicase and DNA Topoisomerase III{alpha} Are Involved in the Dissolution of Sister Chromatids;citation_pages=6299-6307;citation_volume=26;citation_year=2006;citation_issue=16;citation_pmid=16880537;citation_doi=10.1128/MCB.00702-06" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. K. Sharan;citation_author=M. Morimatsu;citation_author=U. Albrecht;citation_author=D. S. Lim;citation_author=E. Regel;citation_title=Embryonic lethality and radiation hypersensitivity mediated by Rad51 in mice lacking Brca2.;citation_pages=804-810;citation_volume=386;citation_year=1997;citation_issue=6627;citation_pmid=9126738;citation_doi=10.1038/386804a0" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=T. R. Singh;citation_author=A. M. Ali;citation_author=V. Busygina;citation_author=S. Raynard;citation_author=Q. Fan;citation_title=BLAP18/RMI2, a novel OB-fold-containing protein, is an essential component of the Bloom helicase-double Holliday junction dissolvasome;citation_pages=2856-2868;citation_volume=22;citation_year=2008;citation_issue=20;citation_pmid=18923083;citation_doi=10.1101/gad.1725108" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=W. Sun;citation_author=S. Nandi;citation_author=F. Osman;citation_author=J. S. Ahn;citation_author=J. Jakovleska;citation_title=The FANCM ortholog Fml1 promotes recombination at stalled replication forks and limits crossing over during DNA double-strand break repair.;citation_pages=118-128;citation_volume=32;citation_year=2008;citation_issue=1;citation_pmid=18851838;citation_doi=10.1016/j.molcel.2008.08.024" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. W. Szostak;citation_author=T. L. Orr-Weaver;citation_author=R. J. Rothstein;citation_author=F. W. Stahl;citation_title=The double-strand-break repair model for recombination.;citation_pages=25-35;citation_volume=33;citation_year=1983;citation_issue=1;citation_pmid=6380756;citation_doi=10.1016/0092-8674(83)90331-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. S. Thaler;citation_author=M. M. Stahl;citation_author=F. W. Stahl;citation_title=Tests of the Double-Strand-Break Repair Model for Red-Mediated Recombination of Phage {Lambda} and Plasmid {Lambda}dv;citation_pages=501-511;citation_volume=116;citation_year=1987;citation_issue=4;citation_pmid=2957271" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=A. J. Van Brabant;citation_author=T. Ye;citation_author=M. Sanz;citation_author=I. J. German;citation_author=N. A. Ellis;citation_title=Binding and melting of D-loops by the Bloom syndrome helicase.;citation_pages=14617-14625;citation_volume=39;citation_year=2000;citation_pmid=11087418;citation_doi=10.1021/bi0018640" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. Verma;citation_author=R. A. Greenberg;citation_title=Noncanonical views of homology-directed DNA repair;citation_pages=1138-1154;citation_volume=30;citation_year=2016;citation_issue=10;citation_pmid=27222516;citation_doi=10.1101/gad.280545.116" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Wu;citation_author=I. D. Hickson;citation_title=The Bloom's syndrome helicase suppresses crossing over during homologous recombination.;citation_pages=870-874;citation_volume=426;citation_year=2003;citation_issue=6968;citation_pmid=14685245;citation_doi=10.1038/nature02253" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Wu;citation_author=C. Z. Bachrati;citation_author=J. Ou;citation_author=C. Xu;citation_author=J. Yin;citation_title=BLAP75/RMI1 promotes the BLM-dependent dissolution of homologous recombination intermediates.;citation_volume=103;citation_year=2006;citation_pmid=16537486;citation_doi=10.1073/pnas.0508295103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=D. W. Wyatt;citation_author=W. Feng;citation_author=M. P. Conlin;citation_author=M. J. Yousefzadeh;citation_author=S. A. Roberts;citation_title=Essential roles for polymerase theta-mediated end joining in the repair of chromosome breaks.;citation_pages=662-673;citation_volume=63;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J Biol Chem;citation_author=K. Xu;citation_author=X. Wu;citation_author=J. D. Tompkins;citation_author=C. Her;citation_title=Assessment of anti-recombination and double-strand break-induced gene conversion in human cells by a chromosomal reporter.;citation_volume=287;citation_year=2012;citation_pmid=22773873;citation_doi=10.1074/jbc.M112.352302" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. L. Youds;citation_author=D. G. Mets;citation_author=M. J. Mcilwraith;citation_author=J. S. Martin;citation_author=J. D. Ward;citation_title=RTEL-1 Enforces Meiotic Crossover Interference and Homeostasis;citation_pages=1254-1258;citation_volume=327;citation_year=2010;citation_issue=5970;citation_pmid=20203049;citation_doi=10.1126/science.1183112" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Z. Zhu;citation_author=W. H. Chung;citation_author=E. Y. Shim;citation_author=S. E. Lee;citation_author=G. Ira;citation_title=Sgs1 helicase and two nucleases Dna2 and Exo1 resect DNA double-strand break ends.;citation_pages=981-994;citation_volume=134;citation_year=2008;citation_issue=6;citation_pmid=18805091;citation_doi=10.1016/j.cell.2008.08.037" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="twitter:description" content="DNA double-strand breaks (DSBs) are one of the most deleterious types of lesions to the genome. Synthesis-dependent strand annealing (SDSA) is thought to be a major pathway of DSB repair, but direct tests of this model have only been conducted in budding yeast and Drosophila . To better understand this pathway, we developed an SDSA assay for use in human cells. Our results support the hypothesis that SDSA is an important DSB repair mechanism in human cells. We used siRNA knockdown to assess the roles of a number of helicases suggested to promote SDSA. None of the helicase knockdowns reduced SDSA, but knocking down BLM or RTEL1 increased SDSA. Molecular analysis of repair products suggests that these helicases may prevent long-tract repair synthesis. Since the major alternative to SDSA (repair involving a double-Holliday junction intermediate) can lead to crossovers, we also developed a fluorescent assay that detects crossovers generated during DSB repair. Together, these assays will be useful in investigating features and mechanisms of SDSA and crossover pathways in human cells." />
<meta name="og-title" property="og:title" content="Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/4/1191" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DNA double-strand breaks (DSBs) are one of the most deleterious types of lesions to the genome. Synthesis-dependent strand annealing (SDSA) is thought to be a major pathway of DSB repair, but direct tests of this model have only been conducted in budding yeast and Drosophila . To better understand this pathway, we developed an SDSA assay for use in human cells. Our results support the hypothesis that SDSA is an important DSB repair mechanism in human cells. We used siRNA knockdown to assess the roles of a number of helicases suggested to promote SDSA. None of the helicase knockdowns reduced SDSA, but knocking down BLM or RTEL1 increased SDSA. Molecular analysis of repair products suggests that these helicases may prevent long-tract repair synthesis. Since the major alternative to SDSA (repair involving a double-Holliday junction intermediate) can lead to crossovers, we also developed a fluorescent assay that detects crossovers generated during DSB repair. Together, these assays will be useful in investigating features and mechanisms of SDSA and crossover pathways in human cells." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
    <title>Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"M01V3zVO4wmkvQrYHyoXZIZCMPuty_oCJf4MCTNPsao","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483825","apath":"\/ggg\/7\/4\/1191.atom","pisa":"ggg;7\/4\/1191","ac":{"\/ggg\/7\/4\/1191.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/4\/1191.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1191"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1191"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/4\/1191":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gdYQwhdGaXbeTOjRIgSRWWIOJwSbznuitho\\\\\\\/tSIvBO6WIfyDj0YPr+utJ47+O78pSQ829cqjz+YtUckyDrjGd1ZR9QdgHEZw0Lwipcn0Ph3hzrsvGq2bh07wgQuiZi8Frwsmjuw81Q1+bcO+DRfo4AYCry1hyuVHC4lZLgL35BsmM85rNd1NduLL0qj1uCJiDWbIplAVOA+uxgvkYKJW73+7jofEZqJeZY2UwUeg72D5ySmCzW1GdB\\\\\\\/YL\\\\\\\/1LfAz7x1C41NRPsdIbctkxPQ0i8F1u7G7jNcv9ONdWOp2DdjXE7nwmrE\\\\\\\/JRoBotkt8ZeFglBo0wkjwAdiwWme7onyMxxtGUjtE83TZp4+YE011J4go1ZoXZQkChoodVN1vd5a1KqlHtgyOHVTbjdcFfzwFAeXfW2qnv7tYgrlnKlAVJvCslpb7\\\\\\\/9nmOMgRIKTj5AY7E6bla98J224tJAKCxRc8dHWcHB2aqAEiCmWfBXQAq6D4x\\\\\\\/kOoK\\\\\\\/eRmD33kPb89woxruRSumz1dl2ydJt897os6AoCx3\\\\\\\/I52FfEbFEwaXZkdkhcKx4y6b+74G9sTexD4d41YTkSQZF8U+mOL3iqHUuUeYa1X1PAGdoBjSdz+NP1oSPFeOEq8ZF3Exn9w39v0lb4ngAL83JeiBtnQiZVoAHVX\\\\\\\/B34rty0E2t6rIcBR4+Uafqx6cSQ334HhdRHPfhaNsl6BgKBrjjcljIBA86fa\\\\\\\/+owspBcTaEbpElb5eTO7jaWOh+YaunGsQDcZwpIB+GnlLAlguxk+lcE8AqlkE5+Mg8THOAwEev6IpJOk7ZgLbHhUl6rHLPdR3DJY954X3wiJ60wzgW7x33qODi6XnJAzY1CcCJ4ExzufcBO35u7cexNEcK+4gQwaIuGdPLHmfQvnL3VDLLQInn9FBcFsHigkogusJ9H0yoc8wdo\\\\\\\/6mYalB88Co9UpsIijqlhnYqTsN50zVZcmNXm4gzzOCUgUeFrXzebxQDL8S3UIDBM+5mMexCxWYTPOxstg+SGhLkODmdMwbIhJuCLQDSbJpeLojs4fYBqWUybTfa41yirBaVEQ3to1qxek\\\\\\\/Hah\\\\\\\/FcDjfLFjoHHNDIqKD+xFScsbqQ70ASEKegiQcByiF6GGtpezqMfiF52cSnxd28ciTTsoIiqn1f5rUgIowt\\\\\\\/7uQBTNFc7tuzXKbhmp+lXZibrqu9RtwCs8dpmfUFrF9ebpKk5KijAacYuBdaTz4EkK1IbOUpHUQq4Hkpqr+hd2TkjI5HLYTgMwPM05\\\\\\\/5woeI7k172qI6Mo\\\u0022,\\\u0022iv\\\u0022:\\\u0022giJs72VuMw+HnrdsFAj5gQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002293eea3ba4a0ab6a4f2f94cec7fcd38bbbfbc464bcf5d1be995428d4e2bc60b93\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UoGQx\\\\\\\/oAasofMLnJxz+KpUdHnPaAENUKA5AdfFsQIt\\\\\\\/E3x85zl4taMjPWK7PTlFA2ew\\\\\\\/\\\\\\\/ePq5WaGew1si5cCPrYgpoUqOX+EylX5RYn9e+bOqipkwZ5nD8OT1K6i9OA9JVAgX4PfhLyKhy2zQKwdnE+6XgiK49hQbSW38oHURJRPCnWFW5A7AN0x19LJqNrDMqHRNWqZ6OH0X8KPMLOru50+sStE3usAFxuMG5RJLJOa8iBL\\\\\\\/8UqpeS\\\\\\\/4qzX5ZVCTzZAWaHVQ+K1vgUfXM5Pi0faXFGeH0\\\\\\\/pg8MBerU\\\\\\\/qNl5yPvkt2gwFBHJTGq43TBuIL88kzeC3eMyclSYWZYChd3QSvyqS420YfnYT0STaY\\\\\\\/imDBG1zeyZcYL2uxl8jKbJk1QMmUxKaj3f+vpVdc9fqd+Ebrx60DXfsUBt6DJbGfT9c9fBhgTAtNidhZsyEllMApF7hdphUnWp0Kyu+MZ9a6DM15\\\\\\\/SQ+mioCjiimXq45g8rvgIBcSe7k6zBLFjNTyLLz72NdpMYb7ocrcx63NdPZGIJPKO6ZxObgwjpai1AR2QjKUPaHKhiiYla4UhzZHwcp8JpyfkH2YBH\\\\\\\/RfnvNxDrzgTFy4yeYUbSeXyxzh1mAV7SFvkb5K\\\\\\\/YHLwqsw9EGvOv+c8sqz1qB29fzXXs6NQEwYaWXTtG0AzzeritlhiiAuArVTDGVzT9aR9Q44VMf5FF6iqC\\\\\\\/U4Oiv27CuRZX+RcVItmtmHVpinaljc7oJSnrrhtNiizDtbZpGIHs8xr\\\\\\\/OOI1bRwd\\\\\\\/OxU1h2DY0q1XQp\\\\\\\/XN7ljX3EwlwN6IfkR\\\\\\\/OW\\\\\\\/IPqDA1HFlFYO2g027XVVN0oidFqbyFUWa+\\\\\\\/jVSJEh8dVe4lYsiYwNRu+z6+SaPS5j\\\\\\\/JcPW6QqKQYdIA4Ta48YFuxlylwI\\\\\\\/kbjxTCKsyn+F4FieED8YAbT8zGJzQT0kJTNwomVKIuIOiivNMbdyAO4mCqfEt+jaUKesu3\\\\\\\/gC4oB3Rg8kf2kQd8\\\\\\\/ZPQ5pRUX2SBs9CCtKvdS5bGz5xGgLXhJcslp4IDqqvzSypuGBiTX4Ff+PHdsDoYuS2tBt51f+xZhcNIyuBfmKU4ONv+dvu1paPzx0mPD7qwUC\\\\\\\/UPETFGpV78diugCM63CKKtI+ER+PRrLK410RyDBKrbAPAC9wcLmaVMldnHdvCS2+shNpPMqF8bjNoT+LZ48Nh6uCQJ3F+2CXvHqh3kJ6dg4cZeUUcFkT37CKq\\\\\\\/Jl8PqRUFZkZiebbr4GWFZ8g8l7imBHVJDwVyUZGI+QxrfkCUaWsJ4bSYnp2PBNOPPtzwMMtrijOt+3J10Rvbnmhj0uMspE+j2vl1cfRbupBB4UDMoo0qXfm\\\\\\\/gg84tkoX7fI\\\\\\\/ygGxQDNqygohdtdpMNLA1keEBv3i0zsZt4WXy7rUAhVtbHL3RNL1vQXYSNmPSWa14hhlkg4wt6gAqQ0XzJqAa6hQSJLk3zy09v0RGTSSamqTrOm73SOsJovjnk00rH1jmo0+dPSdCmJIJThXNuSBgPBlk5P7jNBNBOdB5cLbgeoFY8TAJ7IsxZ0e3UXT6H4b\\\\\\\/W3wr2CDHmBNVI5IaYFEM0fa63L5g95CmhLH3h6gK+CPOI1assAax8G\\\\\\\/U5ps4azjFeYSRIGHQiJcchJ+I9gPYSnbdTQEIDM\\\\\\\/UmK\\\\\\\/PNTAp1cPtrBBpsEIAPI\\\\\\\/e35M9wXU+GY9KHGywCVd4kPfrQMNS5+L7NL+AgsxBvHrfrAhx3q5E4vctHG7cUtpgV9UMEAUbe8UeRvbGP2KYm52hd3V5ey6ZbJN\\\\\\\/nmWeoFMjoqUJXM1lqCwv1lZrVUNNGstas4ANc76Igh9ncw0Hyzt2fAM5yoDFy9UhZ2FUbdOJTLeUM0SmKkpdME7upsIIPQCBikEakAIGc1tUE2YKfraVrR9EzispzoRCyQnnEXvVpeSFVwkKXSiwQdxMf3Sn3W+6u0y73RYWlV29Qb1axPMoCqhlUJjUWdafavXpptkl79Y68VVCqfoX2Vy171sUUMZnfawdIttK+35kcOsJQTuFDUNT7EyVb61UyC7thvybngiwJXvufhOQo9enaqcdzkFHdX8oCDt6tr1LNDaseZC\\\\\\\/7P8kwwiBpuaFL1g3YNVkmiLAjpe9z\\\\\\\/vz+CtTqNiF4y\\\\\\\/SE5n4TAEKhhPOlQYxwl7\\\\\\\/o2zHlcvF+JilP3PyAkkzXYeSIjBgVzXfAwD59h\\\\\\\/RSaDE3g\\\\\\\/DxDtmyuq8N4hHp85yPmM1VIu+Bgf8z1GQv1BchnFOIZDrlBrMLWUSpf6567AmCh5dVznrpPG4vM+TY88jbnUBEqeI6XotC5iI1t4YGGJVAQRR+IT88D7PrF22o5xDcVUvcFtmYP\\\\\\\/8KGEEfvEGm26U\\\\\\\/QR8u9PTryrINmieKAvESiTCVu6e1Gs9M2aLHEMis1OHVquUQ2iQ0rHRDV4jphd53oGyvBkzYGvhy6WQA9HOxu5ly0HB2aZrNoNcNdlCMyxXKeGWRRFi9UaZdfFMFuW5Mw5dh\\\\\\\/aMCJ24b6mbjXPH1ve8\\\\\\\/82kkGnYELY4ovWziJzZUBQBE2Ln2jmJXUtF+Y1TbvVWQTjLZqTpaOD8\\\\\\\/NzBDw77boCblLGKe0KxLIplES\\\\\\\/CwloubDS5Sct5lrGcbzY4ZEwonjLH8qIAfQRWRK7PE7yl3HBjrfxqhfx4xPkaKrc1WlVKvpComWyGWyXwfg0ZEHebH1DSgV3YhFGrwCqMVEKNPoM8OmMfFSEZzxGJnHHhfHnRWvVBu1j14fW7ijlTD+S\\\\\\\/1P1iYPTT1EcfSU8jNgBc7BzoILzTcDxJyTx3S9xpgizefudMIbFQ58aJUEBa5R3UUh0vaMlyUvaJyP67y\\\\\\\/ssrrGTapi5LQLVAzlr3TBUpXMOZ6YeFKuIY59aQy5Fyrqsvy+eEO3KICjz1mvohBxsEqnP3W\\\\\\\/FHWOEQROVwBcbgHy\\\\\\\/KALGsbJEvgyQXZYKM8V7QFwENNCF0ghZUANywqz0IP1fK8jysOCusDUbV669CXbLrk\\\\\\\/9b74He7RsvFlN6H2zPEnSCqrYySVuqe5LjpRn7l\\\\\\\/rlku9cDgWKFM6EUJWRta8qIoYHpPO1KixyARauInPKB1lMntBwtsGxIgYCvVra2OrNRz+HH\\\\\\\/j7zHuubP9hs4Fs8LrVQ03XjVza0NPhmwiWIRkOwy\\\\\\\/fuhnv5hlwOAqTL1lGFPBgd7MJ2gHx9A7tQ9B\\\\\\\/0h2IhGk+vNXnUJLsspMAYTEINQTnj22VgLqy5wOKlC+IAF2diyLR2UFmWeM0bYgNCduugQkaVMW3FooYz+vieVEjMlI2boiP2n6nJCPjAnaYLzRJscOKHyunEkKYR3gEUbmug4XFrORgk9lIHmEQlNAoYzCjuN\\\\\\\/NGhaeakRo9hGcGQdHRVQsQTXvjQWqXeYia3WPSKdYh3IwoYZEiFnqLFg9YTWakwyjccG5k03A+jVzjvnYti7Hzfx0YDNL92EusUOXmoj2eLfpvd7L25FnsBL36\\\\\\\/OwYuchu8\\\\\\\/XwEyM7IqG7wB3t1p5EtGD1\\\\\\\/0DEFhhEBZ2D7dDgdDcPE12B5THyY1OWgV5fEiCGbl8TTgn6EW3cHMbygCbEzceJ841wC9b2vBGk9qUQVjvYqNotYWNJl1lkI8sXtpYx45Gwpoe8FpoRkgXj\\\\\\\/jCDVOGhDTDJ4nkXXBp4P1MS2FohDeV+5iPBy+mnF\\\\\\\/74wmBkqUs8E1pfOqN\\\\\\\/M6QU33\\\\\\\/TqU46zaHki0bYUBB88hGTBGbJzti3zVFH66iI7mKGr7N0eamrtVod5Pehe2JnQvkIWbChPeCqb7nmLWvnf\\\\\\\/WIA4KjwTCjYiQG7LvFwFSpLqoW3EFLAJgJLIeHoXtC5xG9qxvtHkgRNoC5zPwaXPgOOZtrsbZhFTbArJ5TY8oIxkJiYynAOmfh70JsSYHgldZNUokU5vo4Z+2KdqSKN1IGNtTL1V6WdqSEsGfatMrj0eDlVD03cyqZUzwDD1MQ1gASrHGiXYzfnR5CAVKSzhyGkCqrtlZPLOIqGr2htGyisuQAg3ZtwSAeIOcS7nRchm2k5a0COyJZ\\\\\\\/CYWkevaRagJEoS26TtpXwE3\\\\\\\/9TDxlPqVI\\\\\\\/WQagLqAX+WunEFKVqXpbiIsrnpp0f44qfUuELhnfxfQcdnai9VWr\\\\\\\/l0EdnTrdxQrJ6ucg8D1Ny5Vnx1rHV20BOaU\\\\\\\/Fo8aN1zk8BYqhMfpF59SWC6Yp6KRT59P2iCM4Ihsr4tcp0C6DaXhQxSGnyTpDNaTSAF6iWvx3J+NEiM\\\\\\\/BwT93y2Ix1xWiHDXZiTNBHoqgqCM7zk2Fav8Tj3RGfYu+ZTspXWPRlhzg7amRS1CnFxga+EzQ3ClVtW7AxwD2\\\\\\\/LsUV0pZBIRvZpWu5eEqOOsKpKaOx13PTCqXZRcr2QlAgRWqYLhP0acDirz\\\\\\\/bPlgmaFXnygzpx+eewS0gESnyPKfXh50SUcw9NeEJIiDCJVJiagINgHWnQ678lFXHoWUL35VCmAkS\\\\\\\/Su4sBKFy\\\\\\\/g3wS0W+d59xHH1fGz5BP5T5ch9sUDUXVfzujBeZq7ks2eY6Rl6Nujjed6V8DHtSVjGpPxuhV6GmBjaOcauqK8XlXsbV1SQzO4QNwlDClXYk2\\\\\\\/lRV0bLcDlV7XGAiULHwHEt76A2\\\\\\\/xC5w6GTAFOMt5qrMA32Na4OxiNGyAezvgcWZQlXjypXlcEEsZLGa759gFK4bt9MlcZm\\\\\\\/0kgK8lxh+QaA08pATET9LcRiiLEeAARMb\\\\\\\/SUe7ddYk0QH2k\\\\\\\/g+UntQK1Q0ZBgnycOqxurP\\\\\\\/1V\\\\\\\/ZrRBE+PegcY\\\\\\\/6wb2XaKazxBX20RMNj4eQPC5YzYgJoSfpNIJ52t0sR1LNT5hx03iDibsP0t+glHbctwPualRvklBG\\\\\\\/8V6cOF7ILBYKNnktvWWqJbwe3bdwG5nhRLNLgL5qfK1srJZOOWMEkTR\\\\\\\/sMo8DboCYjExuY89IflttYayioFHMAdqkDFxI\\\\\\\/WiTjVLKt6X43LWFlZFIOAYKQ8VrVrKBf\\\\\\\/2gQ3nFZkZabGKKi3cqd8sx\\\\\\\/rl0kHYy8xz948KX2thQhtf7tshH9MpKL3AK597p300IfsQ3kDawfODA1pmPlJEkpmql1R4xQc87MANUea2ND1osNoZRKjq0F2JwmsC3c5GMrhwhuSEfX3qvXf86g1ZLF4IEsI6nokml8eH2vSgecOIR29G9rNh5qK4rdZgLOO8A8A1qwcBuLcj9B\\\\\\\/W0p2qZjS9eV3OC2RDrSZFl4kvvOQuQNlVYoPv6IP9kAuSx66zX7CwYOKiYQp0IjUi0kyUkIg\\\\\\\/VghkpD3GqpfzfjrfedKg\\\\\\\/kvEBS3fZhaB1OxaFyJRFd8lnKcOnVECBKPF3wPr2W9O9qveEhJ\\\\\\\/In6hXczob5\\\\\\\/ir4ttl\\\\\\\/hZIeWjlglazyQHuiBYG17O5ltts7y85bS0xwm3W+WyWuiL2TISaZ6bM5fxDOYh15MFQKUMnT\\\\\\\/e4D38+hEcUuhPAzQKODOcasd6yHbg3pX3DCZcczPNyt8C19V3KrX4iNmtcWQQnAzdly5qXNja5gWw6mKxSgOkfBwJSEcO1Dw+Bmlo8mG1u++6i3sx9lUFEXMmqaPycbTdI8VAA8DOcsvu3BDP0p4idPJjMh+SPUTKIbovEDiebC63Nr4nf3FdLS29a6IJSoY5C5+LdYjT06FUkrqmX5Y8Qsf4ooD\\\\\\\/X9sMdXiI91mtUWBWxAkqkoPMUULj3+XMmYRyz1+yO+4qYNvWTbOJWzT7MKWfD6\\\\\\\/mhDTm1FxTxcPCB89\\\\\\\/jkBeDkyt3Zb6N6rpQ+yZC8QSxsHDIfr1tMpJapL\\\\\\\/52MWX+qhV\\\\\\\/pcpJnxRfy5CSpeR5K1N8FUXB2NqNn38q6lh95v+f94AjsKGvQcki5mmeEQsAqYfPbPw1Pf1KhDhQD5xeCSAZkWTvvdm\\\\\\\/fCDhRUKMRTOYZ23iGPPpJbFKrPsOsMEc9N6ynuSdOH+xG5od4MCdRSjP3ZRf4K1zC1FoeSjK\\\\\\\/v+nl3MfSmwcWJpD9hailpEwMQs\\\\\\\/M4gmsoEqr3Pxj7zCIP3ksJhBNp2VYvJzczxsX0tasS\\\\\\\/MNfhoEJTlpYylwZvagqD9ptAYX8i9lZNmbAht874y1jw3SMaM+upiPmWSwmOAO6ddSkNlzgkn52YGshQjwNAwX7ogIUVUXzTBQCX6nKGari\\\\\\\/6QOKT6Pf\\\\\\\/RG9kKfVfBwCTMI5UM2vk2vqgAjo6nnXhug1CO\\\\\\\/VVzGP\\\\\\\/XcA\\\\\\\/PwaSScL7wB9E4Pxy2A64ZiCthrUVFKRs1yAfIc1du6z32TigoOSJtdrffcqiyrcN7GqW6SKsKK3CYo5LLnp0Bj0Vg1csztT3Q8nsn5E+kuIANczsFMqk9TvmlK4aovtiuBQG9qZiP1rLQaehR3AFQ5PacgdnmttF2lX99b11S0AeN5ctwLv2edufjLFHFtChNbe2AVX9BqkvFUcMuHiI\\\\\\\/RbDx12RTKeWsuAk9mxh3Ur8UpOSHbfNG6GHR\\\\\\\/R6omVCV5mBApdy5h5nNh9cYg4yMaWT2ytUGkR1BLez0rtrL1\\\\\\\/HsyS\\\\\\\/BVt9vzlaB\\\\\\\/sOQpAerjDbPrjvKArATjPjyjL1oX+HQvQJ9opf\\\\\\\/FeBODQHXoM3VdZ4RI23G5CNIeqmrZ7iPs\\\\\\\/VmAWXsSxdXmnu1ArPLGzRPYC9gpN+41rZLOjjIWo81XNEegTK2xhyS0JFBTDVs3oQev7GU0zHMHDHZOTnnbPHlN\\\\\\\/XC2P\\\\\\\/FT5OLNOXYHDa883Nweg2nkxcxofhTSEk9I673yjP0J5pBVPcs2sR362lXz24JN0Ka3VrP5PUXrDhp2AkXiukPUs+j+BxPDccrYitgQneAtwyFpXoRcyNSrQ4QGh3V+TYXIUGkn8VWJvGKsAatLkFuOb1qPqtEZ0GSU41zQsMcYM4BvHLVxXr7WLy5LIGLY8VrU7aD7PWmlxhPefI6DseUgVzE4jIONd2zVp8hAcqXwUyh9HIZdrNoNHts5mVweep3JagaOR7z\\\\\\\/7aMMuPfvrwHPOZClqsbVAr6g+AhZ5li0j4mvoMu2JRkP0\\\\\\\/yVvcbgtKyO5ZSoIu\\\\\\\/QcOzq8dnLA7AR6LWF\\\\\\\/wFzPJAC+Ya97LVuLAuagDFdFRC2uXdYUD2uLay5tYP\\\\\\\/m5Q21NZobzFQFPgLfEY+s0czsfjPEklmT2KJAcnqK79jAE+V0BQDjSQ15Csi8yw0Ua7iZxa0qd7Dl6feUBhZk+Vvn1W4Zi6gBYSwWCGJljLbUIgBWop8c4uQrd8rZobK9kXIr1YsluUaKdL20QVvEQn4L5MbShSkQHFPkAnNvp5MU2xqXbBnj0yjeyCOFaZE\\\\\\\/uxH79kCHyKbndbXTX5Rw2CteQ690TjQNG8xkClbg4Rmsa9n6Yzn5Et0PqQ\\\\\\\/odQfhcK9fwnVJRwRJhmISOJYrDpnG++h8qpYPDKukisCTgIpfhml3reMEzxNasjx2ef30CPPLmDpij84zCV9yWhOhBvwfiQUAvAhbGVbrSUnO7lH4YE5juYVHNGs86sOEtfS8lqCKwF3DCAJRuneTDMie+PGHtXs8Lztaz1CEWAZCjo70W9GODrbM0YNhuJ18AxCQOuVLVzmoyvPwwjq2VbYquoYimWh9RJZEyemGYWVP9GaOjayn0h\\\\\\\/OMdiPtVDaY9uoufWNpv8RwobCIuiW8zrkHwnsUbAJUNiuUPRqiwQbjXkUn2gtOZJtqaT1bWTIk49ETcFdJFEabkJPLSlWiRSjgST8dWEhKQ07M2OeEckEF\\\\\\\/V6rEGqrll45xRBbPm9r1b0fTInflfMIpqPXloxJ28PQdTkzbKuoSEBXoAjKEDPE9w7EFdP7BQ8XFoRpSZH9ovF1fp9UhfA7o5k5UWEDryA5jPQsAAAXSyE9CgMOztghpGtDiH4VlAKpXTShfZ\\\\\\\/rf7Q6V8PndLQzlb9x+DpB8Xaof7TwSJIXpW9uk8\\\\\\\/GJBJF\\\\\\\/bhLbpWMQBcaLDSWjHjwXhN6QnjxzCSHQwLRhv53GtB4ZsSvy1tsRs1xtgWyw23uJY\\\\\\\/0pQovbRYwG6lCgiYCdreQ+Yk6ln22x93xk7LF1vfOra463NUt6kay1U7WuPqGgd8Ar2v676R00xPQF9HYUZBx0y0Rz\\\\\\\/wDeElIGb\\\\\\\/WULf6dt3M4bcrSuo6pNvMo2HciZm8JzYXRtQFTvpd78UIs\\\\\\\/414wW5gBW1\\\\\\\/De9VCHjkV1+Q0oV0WT5m\\\\\\\/S2WTssGTKgTgu8VjoyVoMdpwlnIkC6EzS4pABsmbA9hQUylU6ZWsXL2lySzTy\\\\\\\/KPGojN\\\\\\\/UTVZuebUacLcwfVNElcJ9hnhtyLxg5jbSZ08ZCqy2Px7naG+nYmoa5+jl25JFfMMRS3RBfXbZN9PZXkbJXEf3PZ8mcdeOsWDL0KUoIjMaz+lWsA1tgxlIK\\\\\\\/UkljIeRMoC\\\\\\\/Ui9P\\\\\\\/6D+rg4Dy4FMjWLuFikWZY+lWEhEvaJkounbImI9xIzc+eJV+7n2Gn8ojrb2OlDVdf0bTdIAi\\\\\\\/Z0MLRAMwmAVqfFo802IplLsRLvmveVHlr2yF\\\\\\\/Fiw0ek\\\\\\\/+6XnqgRhhfBiab7gTohEqGTaeF68M+eQvDXoKEucTMKrCj9bOyoXfK1JkbJiz2ohiDuEG5JeEjHjZHQzPbn04Nf4NjXUCsu33sk13Unth49Z4uBmyLcJ6amzHKzOadGEzqDUdfHmKyoWlzfcLxILC9Gut8KSB3VqRbli4KU8iDxkj8EpIfoGbCepq2NB9yh1tMh9d06Ij4+GrDBcGs6Jcau93tsMXYxD5mZV4inpWc6mEvkxgXqVciPxzqY0rttdpQ\\\\\\\/QbQZbQ5q6rEarUrhJVepEiKZ9jaUKIi\\\\\\\/YeGz9uIs0WoLPcfjzjOkYmbJWoa6V37SHJCuL5\\\\\\\/VgbIkLRRR9hCLYfwF3RJBM2kB4xCaMdslMW4kSMDckmAYNngCpFQDks03MyIyvQNEnVGPuzgFvCLjAZB6yiAKIUH27xSqwkONEn2aYgfpwcZ1UIRj\\\\\\\/qcxFERKC+NdnlBc6oud8bfIUCCKbIL4xcJv+0+46g3lXVqf8DdnUoDb5dG3TpjatorWKOmbIjhotSoz3xJz2q+mLzfl\\\\\\\/4sYJTYYpA7xhSuv2ukjLkeyxDrlGmiFgpImLAYjNFedTwWQtTtIG9QYUbm400P5Rfpnky5F\\\\\\\/hONcvzLlZ5X0tfASs+arc1hZeCOV6MkgX9zQHL4xrlh3XmB3lQoEZ\\\\\\\/+F\\\\\\\/3JmjCF0Xe4RLc3ENmbCY1xb7ehVwSK88Y5Pk3\\\\\\\/a3SY\\\\\\\/NRc8oJO854GWyoiHk7EG07miQbxSBpubVUCQXHa9LVjngLu1dO8okG1j3dJBVxTlMLviF6aKbBUkKSLX6fH75Ac9WUBNqKZ7TkyuF3khpHr3trKXH3tGd\\\\\\\/RDIkjh34o7XqrxgcvxL4Ruh1k\\\\\\\/\\\\\\\/dtDvK\\\\\\\/IdkpgSJeUyUlFWpXQHRuvhK2er57dH18qjgjw8RqBzfQkzyiS3mK1naKlwX5WLTep5Yr8M6dTRLVho7lnpNy2LsADKyeGxhzf2mvHmY4QMpAz9dHdSR7bpse\\\\\\\/GtjM4qIMbZd04afGjqaoO2ux\\\\\\\/uiF5DNXeaQZ4csjcMWhSuVLRV8pIEaZmVdJvQwrD+35OvnSarvZm0ZvUnpDE8AsuNuiFAWT\\\\\\\/HXRaB8MrD1JkJjgxNXStUgniIKxUx3BkbEJlixXobi98tKfAg1KeUsdB5ENJ8g\\\\\\\/2HZBJWKd7s6GhQZTvlGZtfv95AeQly1lQ1p1Q417rLPRPaCj29+9IrvHoSXBSGMT4uIOUl6QqllAnnfvzpYFZM7GQjHGbDNb5VVsi7RUUYyEOm5y4MgOLkBphMylMnXEUc1hx1vVycycpyM75yRvAQtJu04d8SVVgZtbhhzzEcDTD0DiNkvFutoJFc6iIy8Btw\\\\\\\/zno6BaHeMRGsoQR6XVIyVMzsEreI0uTLVVUC3x+qLcJTjZ5hzfFFSaVhWmU9D+aFi4kt3FY2RP3z88wrBlOcvQ+dSU6VUgEQkQWMN15uKCldjS6Ee\\\\\\\/cG\\\\\\\/P8v0D9coJsBsPIW1znV8wlCkIqMaBTcjDcfUGpjupDyoD8tVnTKFp9OOhBXSoECI6XTT1zzxvv7y4YP36v3nazwdYe61i2xqpM2QCBv4jE2j1MePIXZ+nmWzdOIpjPA2PHYjcd4q47r+onB\\\\\\\/KsZ4FBgQuTPhQjh6JpQTRYlCIgp6ZjwczwVb6yv8Yha0oUXGMBDK6terDm2xD5HO\\\\\\\/ecQzQstoFi\\\\\\\/2Yvo2Rk5WX\\\\\\\/+7+FpHbCyTg455xAodCOBsf5lt1qf7b+5cO2dNUkXdrJfui1hRjMc4DPdvULumG2c2iVwgfNS22kg72z9z8ps1KyYwknAe9hfB3\\\\\\\/\\\\\\\/7BmsR7rFmSsTKpbApYZ8b6OooFjWt\\\\\\\/M1mFeudwBTrtPwZqFfQ5QE+jt8PXTorEbWL7FQPzUl6QP2ZG467KIbARrDHTuqVJUnf7C8OJpINMc5\\\\\\\/QldcmDOq8\\\\\\\/48FGmBnWntXdsTSw6UhCnNp0pqo0I9LHlInEbQdiqinyAYZWUlRrE3kMbinWWcmlCiW2ccX+3o9nnHKiKP8G0o+Cjrpv\\\\\\\/9KDfNkf2TnNAuVJXK0gsxXyhhe1y7DkfG0Bp8wtJ+1JpfhQ5UjB5mztCLRpsKXcQF7ta+qCWDCZYB28yudDYmx+gz01sfRAnzId4TpHu9yERr3Z2POZ+\\\\\\\/xEaltBrqH7Kr8wIi9+peC6UEv4L6x0tQnvJkk7IPlqE31AeDze7hQgH6s4P2jw2f08\\\\\\\/lVtSs945u63u0KCzh\\\\\\\/v6eKyAy\\\\\\\/EeXpxgQWLvExpMNE6wv97Pp4uRNCTUVk+1\\\\\\\/dX2NG2RM8mXk5MYvSyUO9VznJ\\\\\\\/uMVwRCanbE0lWEPv6Dsv43+hiN7bqsWkzL8HoVkjURwjdzJBh0MU5rzY7g7RNhnDiixoS4MGVJpbyUIFjbj7d2B5AE1rlSqYkcHTiUmfKAqquWNH\\\\\\\/BVjq1s4ENhCpvQr5SYGBcHifFFAmtvotVFkTzf6gOEclaNnPVbShJauLIYeP3S+x4y5krs3E2bLs06VqyW7EPaLqQSfKsn5xRDI94+7GebcizycQ+mQp3Jjq59jI290i6mZVkTCa6jyZOaIB+kixiExdre\\\\\\\/yYzNiaGtSIcO8FWFWxq1dXe0Cb1YgUtO\\\\\\\/V0MoijiBKFhX1ce3hWprwyjzdTBMQudwXCBsFrqpMe85t+hw3MOt44r1wDDLok9KFmWGw1u0QJWC8FoowLR5J73eGb3mZdwBUrXyXLZVWUEVOx0eY8TclGk8hNtpAII2Yq+3d8nIMaQ3sv62apH53TMAGJfTGS4FP9NW+G+KdWlRWnGHg1HHfAL0B3w0lXtZg4t1bWS\\\\\\\/o87B5LW6rQmiXp61s7Hfatu+H88vYzoXTDhwpNDPQ7naA5WRMBHLHto8sPgEJzHRW18LKXHB8EHnDEbUblD2v8RRwoCVElef\\\\\\\/9x3KxonPTz0q+CyVz83uXKzhAynuvU5uu8wkeDQ0oi7aijSAkLZbtRZVy48Ia6+IYdnbcRiBjGyonhXCDtOTnp9d2JXw6W2\\\\\\\/4VbiO6Q\\\\\\\/JuFGNQD8AsuWOxCFkCGdLiNwp94dmQJ3jxBS+dZk3d5ctJEbjHzQ8AXlwsdnAJMSQ6UVY7z8yBBAgZ6cOtcxv8kxHUu91SVN\\\\\\\/EXB+TQLMPd6rlWhTPR66Fjuo\\\\\\\/sIJNY\\\\\\\/yHsug66py\\\\\\\/ufrDGDlnuRw5oAiLM53wTEBZfKzVCZmZm4cwte\\\\\\\/g4fmBa6zuh7\\\\\\\/XVcOK8sV5w+WqY+DvxzTBO22D1WL+2FC2gsF4afL2hpCojmS1cgthmyUspkwLQDbn2JHisuk0AO24h75gDCrV5qqtAyeXGpTWAi925mkW7OLVLLa8+XM1juNwJz87GhHpKBISwwosiuLwOL5TT3Nhevhqeq3ZIxU3+uKfvNTfu7e4Xhg7wiFcTjXvf5biv6w0zMKcg6N8NBoEAe1lP4Y+rQIWoajJmFF5jszP0\\\\\\\/Ltr2HYWy1eFT4tNRxX2eXtSXyjiSjWrk3KUW+kXO4YBT9yctPh7u4MMZm0DOerUheWCQ0jo3fcYGrJF7CvwF1uTV208FA8gNvh1+NkkeIVHMIKf3sz+qF8hl3BIZUuh7\\\\\\\/wMGrD2W8i1si9RkbsqyoXN6BanwOHuzKi7xcm12+Dswr97+3SkHujHKuscCwhvMwUf0M79Asb1lKs5J\\\\\\\/j5E68dgx50mRd\\\\\\\/9AkfK083CW0TJQ5Vuq1PKhu5wbJp3ZCu+F0SqFa4cijaqKkwJWRA86Qb1ZZxfNKLJVNWGD\\\\\\\/BEER0pFCTFqYpLjKJdguIS0mBkq3RdLe4slzwX7b8hA2DGyXYU6T6SyetnVJTorEN32pRP6uSYJRfxlf5EacoIyfWM3LT+jg0tzxj0TGbmYN0PCe2DjWG5q\\\\\\\/PGz6XzVMsfqgzqgI6JHc\\\\\\\/4D6qVc+Nd7s0cY6tpcCAN9k6Er+fgGdNVmE\\\\\\\/icGFiIpGA6uKfNEEVGXC2kVu+mEPMs\\\\\\\/ugPg1fRS6yGtSKisoE9JtOpEQYdRNECvBuJ4fZIaDYTSDmmRHO+XsPN19K2NvsAADydhpwC7DP4z2TxZwfIWtoM3M5n9w8lF6z74VlfiEQloCNrq8VolLV8\\\\\\\/HOzVCaQxkHYtKuNcqwp2okiD2aGjs1vElf8GXiYpqKLciSIJSo0tQNw0auU7AU7nF8Fo30m1Ne7VqacCo5JSPkHyAHSC+9MKHm8FLE2quFqoEnBbZBiCAuPKw6v3NXolmns50DRUMHZ0uQr6LH2NZ4OR1I1DJNMBruvhp+e7C5H81lNt98jB56ZXtGWvwgXqU7dGrPAMBZxo\\\\\\\/+rQdNp9fTej6tOA+5JsNJzVij3u1tesbOgO2cYUo\\\\\\\/5FIRgPHsUOW4ELR\\\\\\\/sPD3vPdt8XaKDbogf7c68qKIGJxI\\\\\\\/igoj4fCtHU\\\\\\\/69Z3+KHAik+cTOpvq+yD1As5YNymh22Mp82FqszTtIYa\\\\\\\/qYXPKGdAy5WUUbA8kYGJLW+9joOvF5dPADYIEx2v1YQyF8vZEfvSabN2OsgznIbkD2mfxynWfuwKwEhVreIzKx05gopWHSaMV7FYBJZr71HFEugk\\\\\\\/3yyWllW\\\\\\\/Hqa70jh8BGXmmZ\\\\\\\/8ssd4N7PGy7ZfCLYsZLbVdw73po6294v4lye45FWVNUFvsVZbhag1dRAwP28pPmvARacLLRABj\\\\\\\/d58kC6foPPZVM8PPRILVP3MNIDOwE9m7WyIeAl9J9CQ5757gsWasP7ZKfEUKyFphrwM62qoyqJnSFbQ9LsuhFSy1RxHIe1iiBfgR6BusyreFb74JJvFYN5pNuXKYOaTvFW58YyVSonf5tidEn7Vv2mHBcFRW0PULvW\\\\\\\/MY8tJQAfu43f2wDZ+0tRgBfZ49deW1wVEFjYG\\\\\\\/WLdgsvd+TgbS5J54S59l8qRtUU8AERXKFG1OTlOTzVbhNphzO0EC1g1XKEgiEsroDiCp+o7dlUcoBtfRRLYTmzBOvBKlud90mXKEFQydqZSlQpfJfZLJjLGB8P6WFFw80jSZpLzytW8hOgf5\\\\\\\/U0vwyAhiBNVx\\\\\\\/1nhZKIZvBD3MlogcP8LEF9SsoQTPMnDPuZu9T7+iUslvfM4usQSVlAwcfo6Pe8mKQylYXyv61qISulzHPu2XPXj9YiQG8y2P\\\\\\\/TpOg+HxPGgUARlcvAl0SByJEvW68vUIlodp8A7em4UmEQXjoVxxba4pImFBITyGRIj5LlnUgfmp5MOQ+VzSV+GEUZ52JcDRuIJM9vvIkHVh99uDKl0I1UYOrsXLqnt03URTeAP0OBRmhPa2lD+5EMRpZ3KcBFJ0P3IgvD3X8dkrYkHrHAhaJEGku3AYjZuCvtCejw19kFx8zz1G6D8g2+aV0IpxYKHgFiVf+naLQDCfV5+7LK70HCwan1reLegNVsMAHRDtfp18VCvRAeQJtKk2siymrs7HWdNAiQfdSXYrJ6rHG3YisRRDfvWJIsf8Qgh8cvxg1D6gAAsecmu2Cpm8rXIDT\\\\\\\/p22apSyn8EdVeNAGBJYvAEgiWh0jLMwUnydXAmA7cYPcpIf+4WUmVFhNNhg6gz3MfGev7FVBDXIzyCaByA3+He4DuS9uldmPZboAcJBOcEWW6V6tyHkxy86nL3vf3OKsN40UeSQ+GmgJbGCBu2sPlY8jtk5a63MAyfDFnKTrW1UzgwB9vSjBVFDsdkgJqpU1\\\\\\\/2vIyiekc\\\\\\\/jBaQJZLM9KsWzS3Sy\\\\\\\/uong3TGSkfIwI97u6XeUjI7cpFAfiZJXLJFesnTZQMpJ0SLhJqKOmFo1iBN7enT3WxNwhFSFNWEi85btd7jRD1wFZCdG5X3wmprHQ4FfQmgv4CkdZu6ZSsXX\\\\\\\/1gyuGt+f4vUfwX\\\\\\\/qmrhhRa3yAAwv8aKIsAxNTDLBAscEOUfDrIq88CmkxjubLqE0YNMBZT5wsQM06gd9VPEnyxHllrXsRPytkQ4UrpCkVX0bWaqqp8Fn8BiczChtSSnELyg3wlMJnLCtREisAgbklGczvi98CQMNATjy8U4MkytSK2TgBwILcGD+PiYUp8448Y+WEKlCQz3V6jUFbjkZ\\\\\\\/6z+wv8CD50jX6A21zAS06uh9cJJfaguxi0m4GMEy0mTbYOFiB6peJAngj9oBecUu9GL4DV4obx6yXpLyVv2yahpov5608+LVvkdDq82dFdul0rtyx+\\\\\\\/eiENApbNHhDnIv82WkHxlYh6veJLBVX9nEbSvKjbiMuJbNPxt4trdYbS8wZnvLJ8VkIpWXq16Zgi6AaepDKgdhGNJFhyOOGv35sxn9mDkM9zxCQdSA0e3UWBXdNtaQ1202oBLw1hx6YSESU4RF73alAH4YYj+EnRtWyVmHqXKnsZRuhKlMFdR28Nd9rVVfgWsOIaNGUUG0PSOhWihzdR16jfO5LjBdZsfbyXy6iH86k7qUWeJ61d2J\\\\\\\/mFJI2J68ucpbmdVvBeQ+XUtcCUwcvH0wAnwNJJWTPeAzbN4fkZe5urnH3izvEtrRcRb26pSBwnv+ZY24QkGvHAD0wmfuy\\\\\\\/znWDeAOWayteLy7cH+gwDf9bJdk9TiRLXof1TmZuHGuh1teX5f5bOyAy0KsUK9g1vLJSpXJqQ6uxa6azV5A1LsKvHcF0ClvMdhL23CZTPb6Soh8PxktcAgqd+bnkg8PDCgq++Cqi7lLR7P0rf8ydpjGpeNS3kRXy92J5aiFRR0tnAaATqACBrDhheB8amBgDymmL+4t1vzL7QNgKYJqZ7Ck2PfU5DqFxDE2RXa4IUhfyNp3h+MX\\\\\\\/fO9XpX8JzU06MXTZJy\\\\\\\/xHs5Mfb2ipf6PFcuB2YiCEsdYbPlabZLeTKyk00tPenZ4iI4JJIUEaa2X+2T2FfNdBmb3rrnoOIHvKAmv7gaFWQ5KhYkUiZiupDneYXiP+LGTV7USgXxGGrUcOaxxrFdJ6n\\\\\\\/mX9HlfEyDxlAe3zQNLZrItU9T62eNkw5RappfNgHyYSnHcf+MrV+VOXj+6sJTMqT5JVBFhxOS7EuvNLJNrCcmCb7bVZuMMzfUX2Irx3UAHGtRUhRLv\\\\\\\/cDz3QsavJrvG53790+5Y1oMT9GBY22BJVURYxxW\\\\\\\/Dl0+44Pavrw98wqfGcvNADU6flnGOO00FzDvqcw8JH7EoYIyMMpw3OKdEpxdJ4Ci45dXvs2LchySCdel6InBP6Zeovzk57a79\\\\\\\/aUxXGw4q3tNcFu\\\\\\\/K736FhIRYvEmmtXtS0xmRduoWVTJb6Z9Y6EiHWb1YTJwQ1mhGHfToPoSm6B4TiNzNmB9UQY9KAnDEE6QxXgWxm2Sl\\\\\\\/dQ\\\\\\\/znmm+yZukuIojF9QeZ5GmzSQ1y7WZyTo7YiunIwM+CycTBoPkIYaWwyGes7P5tvbexKjrxA0NSEVlo4JhurW5FWNy7oeftoQ8bUDILD5N0TKdPFaxjJ12pcX86p+FZfrwV7Xll1fNSJNuvJOTp3C2Ny5gUIxgxsvXxaxRSgbP7stWYlVTqEpE2i6HMp01FExMO+CSxxMGKBo5c6ZnuMaXGJ62uD1LS+ARvJn6hhlrIuokVrgfFJCDigwiRBJxrwFY5nj9uVvqns1tU9xCzv9xT+Y5\\\\\\\/y3NdLD\\\\\\\/YXf9JOXjNqulisK22rV3Frt+W3cuDo4fxRIUfBQr5eTIlGXOLWJGESvQriY8wRAikqhtduT+rkb4yqgx+fb23mGAh\\\\\\\/c+c0fVcwpzaJyYrTZuC89fCInbdBH3PUtRJuVf3ADbL32Tj1+QRDkBxoMbOJCF1c5M2vQzOBZkorTOXgPR5XzMvTDQ+0VhWe6YGsRTbydedwbYA77ja9ICIUkIEXVEFrpYAhpujU45AbwLRwXtVjuRe3vt1oa5w1PJt4rFkjujmwqW0PEGJ47OBJs1o64DRkNbpRMPuN9oW\\\\\\\/TGOy2r3r4i8NwIlYVIF9FauWQ2GHc6kwhqgz7eGUySpzlnPAK20X3qpWRC5C+XMXNEei3xz7udRKp1B3phjQ9CHAj6YFxfmMwUl2x\\\\\\\/TibwZXuyAx2PAi9bvaphxk\\\\\\\/Bigh+iAWIjiRlg7v1wz8rTwpBzm4996GMzTJAtQNFjTo+7Gm0uWSGENjbFB+xd7vfQf2Bd1Afcs26muhTdKH5sPKZwTVhtXZRSGMj4K6gh8tMvohdVA5NZOGBl5yIMe8gKaIx+hf6UlIRvSaIXlwj6KdsFV609k+4yUog03vYLtGJylRpCdpKazGgNKruk+5Kjnb4TfMIML5XNu63X4Uhy\\\\\\\/p5dDS5UFM9e6mWWo2oWS4h\\\\\\\/eyh7RxyncBDtikN01bGWZlOWrbVdN\\\\\\\/5FGUXy8Y7XerGmrsZTCiGtd9GrrSrqkBKHvxYNcJASDm7xvw+5s4H8zf30ynKdhR4IkoGYy+QyYgEmKTdxsylckBzt8dm8qCARhVjT3y87owWcm2224Au9lXCoZ\\\\\\\/WTt9I6Kiddv4Uj7\\\\\\\/o9gYLaop8btUsrWygx5tvWYTmoWql\\\\\\\/O3g9BjH5Km8NiTIYTfeAnTYzTScDRpBoK1pNyleT+ezBKjJO5zotvquFYvdI4qDbMv9XYkMM1DA6S7KRZWMDfgfZr18eokl1sURC7lAWHr1lmrmHyoJhm689TIrKOigIimRzB9D6\\\\\\\/tlav9EPUeOREr5mQteCozhLepJXSmLsCHO\\\\\\\/HB9Yu7d05KdBi5SYuZp0JBaO2c1\\\\\\\/IsF0jJTqlNmQyv+7qsZSnxCXeJnwDeMJbWnz2gHMY6nCgc7jDe\\\\\\\/8hiJtevOqrCqYmkItGKE4leEccnjjfNBci2HqdiFFrqCk6xc5W3DQuvlHyEeBmBFKJPquQGgIG+VthBFU01cdMWbNs6k3nNdJKohCA1\\\\\\\/n4w2oUiOHp6AL31Jj53mI\\\\\\\/E9cXIkROKa1VZ7wmO1XAR806jQm2wWbwWddpxqMAwUTJN6PJjBcNZl7XoMO0y1REF10pqF0ZA9+nqg\\\\\\\/Mjzg49FhaJUq+S3ocx4Vgyh+XmDgVhtejVsk5i8VIO34oHnnCA5R3IGeWEBFtEbbP\\\\\\\/U+JaM0WjM5Mz2G3F2sGNpvWCYpMxEowV\\\\\\\/zv1RClh0QXdGA6i6DoOwRX9GMDieAlqC3gvHZmv\\\\\\\/vfPHAEPiuEjOm17cwcsDnIBKtYmzp1DFaTplDHB40r7ewvFKj1L34q1vohr4nncQp9DhY3Zz3pSU\\\\\\\/wR8spYZnJ\\\\\\\/LtGdVpqM+nMiezlF1G\\\\\\\/0XuNT3mwsOUgvGCvQfBKIFsy7UGhpribdKP8NjHZmB4VOmxxYcm2mFPIye0NAmsgFEj+HjkvkmTkLEDJVpKaebRsHZ1EN0qsAdnxv2HMdMCbAwsDHn06hSDNbhV\\\\\\\/gkTvAQJHM0SCFQlDj8kAimfTSxtvTHkWT9372lD8QdcpdNxgyxJZ3cnK1C7z1y3AExSPJVLBfO2OrgJ+apdubNZP9EQPW12pN0+DJnuq9BQK1aA2HIkEKhI\\\\\\\/iGX5DzTHM\\\\\\\/rQg\\\\\\\/kuxMkafYoqSDuhyc8AYL\\\\\\\/QCUYyICf7eHOPpY4hn5S+qSOs5KFIfPYtotOuf0hbjiS3p4m6\\\\\\\/smM8YlAtpNIRqx6jZFQnIlo1ZKLHTeyrBWxjJrLxNWpjZ9VkupS4UBXeExzvWtr1faw2AqbqOnAGswsrmyWPgoKcB3ORb3NNW7ef\\\\\\\/t1k+tAhcLlPTpCpyx88xOK\\\\\\\/AB99pSt0AOyMBauXIwS9vxhYhkQYRfF6BNf4EmmMYV50Eu7nb7AES0CBjnBYKJVwxqXawiaiLpuWjA9WyAHBM2Pn+Tw33F8vtF5+Ypa4u0hZXjrry4iGpFG3gnjzoWd9KgHTHyFPx\\\\\\\/FAJm7Rfs\\\\\\\/XowbUn+llJF94LmyezoyOgwydgG13Fh0t7d1oWPHzYStbkkjgsDKM03UQR0tEic0WftrR15jS0Y9L5ONmPorlnIgH61r4MxdPTsMskops0PNl9XnqOFuO372bP7ZkhYX\\\\\\\/QqekkHXMeBmM5fyHOjV0+Lpb0S+DYTVp5h7fyxcJOUWyKoIXZsxPFjoWkSJpj99rcrV1oCcLtybQ+zOIdmgpyGIO\\\\\\\/fyjzqY9p22vzncujpolT9BJQyTHAvbJ5kX4iAzEGO7ZvF\\\\\\\/j3W3NLLpih\\\\\\\/aK6WX2cA6fIOg6CJRA5uWsJKVfYGFq19t5DqvTAcfAGRGSXfMX3N2s6R0JN6e2mqOlzn6vQrL5m47AKEagjM24e22LeIulUB0duCm01oCQMP9SjzR1OnVmGWphsV9b1TGOMLImMbkEIv9pWl+lPQXwFfcADoRDoy3PMtu5GilgHpOVM084VJ58GNFTw1tixlEjfEK6UmUHXzV9hFfOsGCAK2XIrnKklqx5\\\\\\\/Jx2bJ43Gu\\\\\\\/C8vix+b3Y1M2K2It3VnZ7pguoPDGtO+hbWZqi5ijpu6402TXalEqAuzcizkXbLwUXrdSX3JGXd\\\\\\\/daPKAc1jl8ftz154NWIA55xkMeRi36pSqwUG0ESAoIUSyWbrRwVMfoiYU34sAnLYmV9CU\\\\\\\/RfCQbVC+fJW3Qo9CKDIGGzG7N60jUOVUA0PqyTKrf2ZpUw2lYBQ6D0U9WfHN+dQUZdC5+tilniNOXSOq1WyXQgSo5YFgdXhtcPd\\\\\\\/dYGWrozN1JA2E2DNBJqmAWbur8IbJiMaph4wJ6JyTI31aKwBnDZgKegWzOg1bILbVJSrCsar2LVTJImgpB4c9XhJPbc1vLnJRupI9gGEWgfIWeMb5oTDpcfXtOPJfdTajl8dIIMx9fIBygqDHpT+cdofA7GcowFJlAuasxR+WwhTWtch6PI6SXGnaig\\\\\\\/yewzOK03HTlzvKJj6X\\\\\\\/3qRXdU7jYFElRMoS1J\\\\\\\/Be9lmEfSWkc01QPpgw3Nk8Chb2nPIbVPcFO1F5wYccLGZaUlLVOK6HI7NE8\\\\\\\/SAajn8QOKzt9I\\\\\\\/gamSCM+w\\\\\\\/D2LqWKsFSiB\\\\\\\/r40hbrHrNbE04ekk9VuAo7rs2GAvA+W9W0B8PNCzHhL32edRfH4jE+govkckNRfoRgPHCfroaeOV9YZTOh22CnWsFaKmTpK8\\\\\\\/62AjbIbtDCenbfk0FQJYpJKkklL9BLJzhxLulcriI3xy0AAD\\\\\\\/QJuj+LFYSSm5KUqknm2\\\\\\\/3kONdLGwh42jRXwnfecFlHymCu0lpzYQxczKSe4LtEMI4lmGjwGHyczOuDm1OwY6Z0uhw8nrvFWk4WIeYjrD2RWp4ZfTKxX6zuu+uSQz2EwwW6\\\\\\\/xnTaIQxNLYH8OYa3\\\\\\\/Wjtcne89tPs0TYSuxdZSrOsV4VR4wEj+BqhkNR2N8Jld0x9Hg4qRrlYj2G0uekmA4gdXKEU7uwDXr1qIHFYGIMjJz2m6mYwtevEpL0kL3D16nPeRJda3aM6Ct9nytqpHw8Uf8K+GEbfM\\\\\\\/qKHAJ2HouCqe9yoyoPJDVIPT2JVvbBFpZzF8NTcUsD3P5WK917z4qnDnX2rO0ykHbGw\\\\\\\/p0bUHe2nBoj2WDXWhGUbvRxigid0+X6fizuAvS37k3Nzi+6m5oZnSuhWqb5pTyiHzKt3GzysPk6hdYxTUbXChrqJDuUwCUAR8kv1wltxEyhrh1tGdeFmKprQtVEYVr0xz3AmR\\\\\\\/k5a+goy0L5bYbX5RdJskrCAqU3oAwG9oLT2N7B7+rWrys6AzaYuMG661ZcXgFnISyf3kp7biXQRtn72ygcZnTgi7qUznmWWqTu2KaCsYlfgL5ibDN6cOf2H\\\\\\\/Pu18aeGJnCxU9E0txpLT6N\\\\\\\/+k9kunjbQzNfBiDg4w2mtMj6wwaMERD52SuFZR\\\\\\\/u3BPIKvVQiKHiJz+VsAfmYl+rN5jLXJfwlIfKqVnyMoEtUFa0HBfdxe6w\\\\\\\/HXq0V3YyEtjoC4pNDb+lyNgzRD2\\\\\\\/MnQl3SKt1\\\\\\\/WIazqdSxjGcZg6ex8kUlDvYzxRRM1DwWKBXZjLJJEEZxFHspe5zBaD8zoTuAqpxHT2LD0t9Q0nhTGHc10iCINWCSHR3AIS+CwQeNvPKDZZNMpqKOzRbUc2UKXZh2+4pjfUoiwb4F\\\\\\\/2dt01T9Jk\\\\\\\/xPCht5F1HVYLmGnqPSLhAk+gCjjHfdoZ0lEmFvkgttEl2WOZTcNlcp8QwzJYGorxD\\\\\\\/WAMASh+RWGqs9wjU+ToaxcJXIJ9NOrN8wuML3\\\\\\\/gSVwjpFCVJbTlT\\\\\\\/jKN9\\\\\\\/Qm2Oziy0NblkJJRcyxClwpGbzzmVFxS2HRbk+nyK1os5T06nZKmUMo\\\\\\\/6TK1QWLVt0YxTZHNP4nR6NX6+7gqIQpuNSHnpmr1oDK3b3\\\\\\\/hKZU1Oqywb1ML\\\\\\\/KPBt7XUHacHH7Hi+nIqWDgLYg0WTed6tPIa56OE7YSdqDLCcSJiSniGQdhMDJbRNJKqmP8Ygkb+GU83sj92Tla3lK6e9T3Mv59Bh5ddbiJm2CQ+NqRIeuU5DI1UuLKN2GLuLzxX8j2rcCD93CiHY9+GLo7gfXJWPrgB3asLGe664TcU1iiL\\\\\\\/QREmqeBYiiuIY7Yk3iZuzTBH0pXCu8q6VvgGIvjNkAxqe3aAZRTSY3yuGcs4ZY+Zfsty4RzF1cBTU+nGyDsr\\\\\\\/qSr4LwrM6hKUSMT+3tKAGzs1P2+zi1Q+9DvLZNrDKFxAgQ\\\\\\\/DcogM0PD4LtebNni3vo7SoHcbFt6RBXck32\\\\\\\/jJEafhHcnjgjtAD7+cNZsh3\\\\\\\/wQWdKGUBTAVyLK3tsV80u48w0DNsRyQxy84HAl9hOlRRQY5gODWTLwatt9DvAdV8NMF0HARaUCP9fAUP71x2A6X0Pub49P5MaAOQUw60KrHXtVrY47thg5zjouVQfPU085Mp5mtjqmg+21q76R1hba4yvCwQdyE5CiHHNNn\\\\\\\/878Az\\\\\\\/SaVIwWI83e7pMw20rWLTm3qTp0XWJdPaMsUIEyy1p0lQYVLIGJrIQN7aTdWUiwcnxF5LOi\\\\\\\/\\\\\\\/sU4VjsDDO6AlxcTPaq48clVBNdHEswn+frexJottn7x7guZXifKmNHXgnLIpXetSW0vN9kWGoo46yCYazvbz8OSmlZwFb\\\\\\\/4i8+38JNhjvvLqMUIzdA71UyjB3K4YeH+EYCX2vqYWYbxnwnpNk2p1Wqgxi1mIri4ys0\\\\\\\/kBuskYcGs4\\\\\\\/7bxp8bwPkjFfGGFICdAx1hnGp3zYsywV9DRHudAeNUdvkQAWinsG63MqPf5oQtMyUho38PXk+fmyEAbis6YEuQwMNCyKONutqUqz2mK9+5eQJEeH5u3mpaT\\\\\\\/rzS5YQxhZPtmc3RH\\\\\\\/7jcNo9YUBGvwiOBs0vyTjHz2irM09eOxDQiftdwZ8vG+YyZqeDqR+4WarnUtfyL1\\\\\\\/Sfm3mZHHggwR+a+YemmFWSgCAUvZn4\\\\\\\/osnIiriCrH4N+QhlhN6mCbGFd2TM6qJt3wsig08XkIA6Idhmz5gP5JLp+mUvsq3tPmQS1yS8YuLT\\\\\\\/xpL1u0cDE0GNV8Nh5I6x96NDCv7+nxuTsxsSp5tjcbnRL\\\\\\\/rFKBaCXXLPP0PEhqu5gPeZ16XYknCF49qjTnW2d2cZ78XMt01\\\\\\\/EykCUwdu5wxp0Pa6WSIl+iygvlcd3HpyqxAkLNiG72RkgfgVN60pBbU6stVWOYRWOf+Nu8p1HLDC5qSfqcyO7bAghsGVvRrifwFuzThtt1iJwSsvS+FCVkTztMqwnEPJz4s+0VnLWObG4gdVJrpOb+gXJFsu\\\\\\\/YMzM62je8N61dI9BodVdfdUjSrv3aEbN0aHaOcu98KvPxisljt\\\\\\\/XozgtobJDHEBNF4\\\\\\\/RfjTda2uAK03pxSR07atvhSCqx22YVQ3mJEhjbZYdp13L3\\\\\\\/4BMc2PPcJ3RRoJxCHR3LPwi9ZzC3lHLhzidHLxLkTiK+JSZH4tFc4GKwdOgeWABOfy4cOCzg82bTm8xpfZSaknVykfPkOIXZAnqf3E62dkgKL4ffqFf1OxkWNSkEpmmtbHfTIzoPmZd6XSvXvAhDYi0SO7wioxN983hPWUvuY4AF2kpJK1qBjhmx9b5Z\\\\\\\/BHRTwvQOkrSqm3ZgbvujnE2BwAa4PWq4PBbSzBwemuLFKELcOq1PcruFtFn2NsIYxRQ2r0ZPZoCPWEmhwpXo1JxCjtdLb1EYpBAt1hEiaaZEZ8hNb0BpAFtwe4Aap0va4l8jlUASSiH0OrmHnVmKDUjrLxv111uCCEDc6nkmIzu7O1L7wqzB4Ul3msDOAAqYC4vVDpr+xPJNK1WOm1DUjucFxDAckccZTK1+e5zWnw4Wv1PsnAeCgnAJDoX7PuQmMO\\\\\\\/TWY\\\\\\\/fIhIvxlcGLfORiFV9rLbiEF5q5bBL2v54jaYXSFV5i8PoW7VpQvgSc8jKkxxW5X4OZYpK9pcsPKiB6FcGSqVvZt36LJfHnIE9E3AOthB9SLqwa3E6Z759z+fRjTl6rZikRG\\\\\\\/+duPdrTwEfVypW1knYL\\\\\\\/FWKV178F6v4f\\\\\\\/jYk1WR4zzObu3EizQnkcSG5Grkl0eOGsgogvNT2rGAwgLn99sxGyz0te8l7C7jYqqnnSj5iotKUrXZ78ealQsdk0hv+WBidE+EVGd8Bw\\\\\\\/FoaZ2QUpGoTrQi2lDxU9zXP3YymyMqjYEv5ZM6Z\\\\\\\/UaTfk7VmiCh1fr6lWqFzacFkIGWJ7JsZ1NlG3\\\\\\\/RYNhYtsdCq099KQL1L8FWzj00KZAXSn3tQPDNoZ+T4SS62q5h7ZVpg8\\\\\\\/Pi\\\\\\\/GNJPTR9r5afqjJMf6QWlFgOgxzb8ORfu+SEOln2WtnWwrZnJxioQxUi1g0Mk91lTFkWjgMMkv96cF4D400FOMVmjZ16rFOFQe5z5u+RFRbw44nkEtSZcF\\\\\\\/M+83Zj7cYg1hP9j3YSFT9QVBJgRP5PvLy3SE4HwTPwegsNLUwYzEDWomzoieWpZsjT01TiWfM80FpWnCLuz34bjRn3UgMswL6YrdovzEQicM6JDUh6w7aMWclFc3271sLoTrA32+41s1QqvySa9uRyeOaeXWJ1v\\\\\\\/qThAOmgvGmWczVZVDdz32uD26BSN+Rgbh7weH5P64UhYIxifuEtwjGX8alfyXSW8woPbG6OoiJFMuybVgFsXzV9ARJ5RP1kAW6KQWxLnaFoIT++jZpsg6H1gb+zl+thTfxijZip8tyhwCodkjopRrlFhoMfHZw6C55ahH\\\\\\\/5i4mHBKE+BQ41HRgGBE4KB4K3kAOe+GIjaOzohiEgVqGvsvXKEHiJ0MdMezNuzV+NqKsdWGk+YeJ7\\\\\\\/G9kCQWHwGIQAofjYKtVJQl\\\\\\\/fGNQaaV8a9Ao7wBBfmCRSrzZ5SZhmli0fHTRUvHQDFTqsgOzDOoZtWti3tzM6XPiXnyRWFZEtX3t4I7fFEJ2hQ3nlR+k+2QHz5Fa7jLsA85P1Q4W50soa3JPxmRe364ylRb1rJV+gm4M79sqNDxKucNl77e0wn1GtmBDJN3hMtNYROsz0FBDP5YdudEtyWvylpa1INczormJHqxay1fkrUcE6XfG38ApoeCAxZrEl2KJeey81FrbRnEOJw0A6aLlh5RGq4KV0ZxAfVCkof1p+4J6DkcorsijbdeR+YiREnIXjWlV5fjnokUKoq0GlJSuYH1ECGtQInMedxiT921id0q72tzqvmNk5Y\\\\\\\/nFWwM1He87AEuzI0eZ64oZgRC9\\\\\\\/GAMQcyaXSaSWdLaR9LfeFsRdwo1iMbnHsqq3EM\\\\\\\/pswqYTQJxkiTxdvOuxgcg3t\\\\\\\/BBiShSILdC52lUjjaxVZd5AN20fhN4fpr5K\\\\\\\/UvM+E8lxsZtXdd54QjjGVyZbVW+BaN6nkoqBcG0Ta3k1pxTT1TGGcQoIWMnufgBWctsfOgjpoZaqyEtS34vrzrESaVfebVfwNxrWvgqJIG\\\\\\\/zGdeQmvdPyvYktn964Xv8BPOm2QD6tNK7QDAh9+Qn0YWTFdBp7ZqlVe1yyQ6gpK\\\\\\\/a5bM+fCI4dRfZOxC+nQFvZK8zHgrlCQek5jLv7ALx7cOjFS0k1BzPmLXDlpF2QVwbUCXMK+dHvpW90UU63s7qS8EPWF7wQsTFTk\\\\\\\/9\\\\\\\/VvN8pdWpW2yYx0JmFv2n+aq0E2Rn1NkSH8KG2VenOV0jCWHBHzexMipdXY+O7nnkdxQc7Bl1EonTdTdkwhRnDPP756nSJkvDY0xDvMwZDjAbz07nrcwA1QxgPik89NFjdUe2KGRE4iK18YPCjBpuZ7yCX1RxA3+5YwN6xeX9f1kcmDgiQC\\\\\\\/Fc6GbWDPJESnrGyRxy6MibQHp\\\\\\\/v\\\\\\\/NwtST\\\\\\\/CaFTBP2YRK4UQV5GCdSvjQnD2n0GTP7WCcDaoRcSTS+970p7PGg1+gt34Ckkt4gdy2mzyntiolQI2dmL1e\\\\\\\/TPiiHdQiTsOhxmjwgeatEuu4rytijEEeXmhSvp42lStLUzPupxpnVReSmIMx52PXqVkDXjvaV8UHqF6BQZlu3wPyTm1zJDxdwDSLPcx1UicMGe4EDG7Bo4JbtYHnR3wJEb0ctTpAblAUv9xeHaEocvf\\\\\\\/5yH4DP9WcXePt15fYpfuINEgwYgcvkHLG1RS1jufvIbcvzyeN3uq6T9Mfhk5XB7nDA6vHPK5pjuVmhqhOmTj7fZi5PsjrO9xlbbSQiDXIxMtvpf26xtqijo0OmYjQBBaQHRJzYM+tYRD3QaDlng6vZ+NMbHmESWCQLBiKL\\\\\\\/E8yUg6LDv0ad5wxADA0zT\\\\\\\/oHq1aPgYeja1X2QiyhKAGOz5lQxUke7fn\\\\\\\/jKn622ILJyCSVFK2jS3bwuNWijAv1ArdIC3\\\\\\\/0V1pekEzvdPXtBsboiXrvAhVln+2lLN0EXUUHwxHK9uwBTkBJsT\\\\\\\/Zdv0Dmk\\\\\\\/22q+k3m2EmV+NjLvPuJ+834QVb\\\\\\\/g35oZuM0d8n+nxNXlgN3btSubEK4GGy5vR4BVQ6YqjJLM2yQ8zpsiE4TATrlxsJS7FDhqNUKRUfy33kA9avdfoJi\\\\\\\/hzn+WizPMbS6o\\\\\\\/LzDyoTgRe2yS7iNMKl8km1QD0MDhz0xZRB\\\\\\\/tu2F\\\\\\\/7\\\\\\\/ZgAqGbCXFLWcgvYor3WUUeV+zhjN7+OWMxIgvbfG+POwzLEDyMpny7tnLYA48b7GFqJcMcekKfej1YH7yQfWgQlGCr9hPqwUGo7X\\\\\\\/pmfwF0r90PnQIGuC8QCZ7f6MD58+fxZUMOYQknsACUMxD788snTxVaHGxc4rrJtq7g5jVGmCJV1xLrQFHeOwgGGQ1QmM2fDXw\\\\\\\/ksFgEAC\\\\\\\/zO0cUVu5lFNJUn5\\\\\\\/yvDJifo2umsx5ANgVhrr8FdFTwLUsxYpL6A2QARjy1jht\\\\\\\/1+jjP56bI74Dis7z4n3RQZUlM+acR2gTqwpUnVG3jpLAZfVra8ilr7gCfVJalYQHEmHmsNfObMh6aOJaIKgwa7NA+sJY7c5nGPJhpYj7J6K9A6TG\\\\\\\/Y7VAjHqY\\\\\\\/lN0ljPU6tC7I6KYL0sa27pb3638tNwfEfCpyUh1Ttzy2IPpS6ORKuFJF12Hn8vUmdvJy+zvL0E3n2kxJOmIew+4sf0FLOJBp5XfuLTNkmK0ITbkqxneQnB97ql400Q1vv6HaSMxmcKzK2925raMogNehMdawDoDKEK9psQq7HunatEC9K2Dx3CRsZb8B2vo8Mv\\\\\\\/nlCbx9WvfwbqTywF+CRCG0bKSKHDhM6LayC\\\\\\\/+rME4wudmLJsCx\\\\\\\/2dwuYXnjx4m\\\\\\\/8mRXyQBh+q+FJXWALwUPx0vCyzzGVfItos8yeKkTVtF3J28IMx37Qjp5NuEV+RYq6JkA0TAPO0zp86GGhujOakec+WikUlnqCgIXeBXcfuZPZPji16hM6q5ak95M6kwGTqKLtJ4BprOytAMTY9m+PRGfHfHDaCZHHWq+P0uTp98GjQ\\\\\\\/rCkPVoHOfV1qeED9XE702kJPi0EPTmcjT8zn7eRsBucP8xmLevlZmK7kgfxWKHPPC1mxMGs8LCorb1VPpkS50TO32cp23RK+tmV6mCY24o0GYoUFFMdw1xBLGk\\\\\\\/qI\\\\\\\/r80BkmWe6ITPrRnRkN9jofzIy5MeC8qfo7W5wUPdjuaSs0qd765KFMp4QFmqbSNZ38neUxIoawZwHsy3pTv91+79khRRezM2+jJ0kB+gKYuzl1RYbntzJvkMJRQPZ3mSg0xWzh73r8\\\\\\\/Z3HMETmVknIcGa\\\\\\\/EYwf3DpN2diwpnSvwvt1GApRXEfLFndblLRK\\\\\\\/EPZ50X6LLhjEHq21zpJOE1VyYt0d3xCwhwissHkTiCGo+vyoK8w1MyBOEIn5zYxlDqJ0KeraLlo4Kyn+DhsEoHMhxAIFxNlSXGi0MWcRutO+xEP\\\\\\\/wLyEnZoYUenNR7VQWZp2IS3z1catcXuqxG3rOepUBGDg1Xo0FJDIxElj31lzaYht5sp0e03\\\\\\\/E\\\\\\\/5YrWQkrhF+mP0WhJD\\\\\\\/O59b+t5NgtF\\\\\\\/ewf1CgjG6KThVawjH9qmjE0qpDaAn1rdE+GC3CQSeZIh6J9jnGpaZRX8gCgtrOLqPSef13ie9p75C0e3ae5VLKS\\\\\\\/bWZvDjSJWj4JdBzISpGGnZjcfXQDVgu7Zc7yKUxEC21t1K8EHQnC9rS\\\\\\\/TfVnJDTqZO0mEhTFlqlAQ2NjPqN5sgpUAsptU4a2AtR0loYn2qhyjb4MXZoBnp53Na4AUiBIa1ns8HcU0Q+ztDs7h0ZORgvxbThmTAELlUG3CxVShzGYur+0IajETxzOIMGJcVaxnfLQFsGmFHeWcfPXwGhKBwAnVeFUsgUmXfWwElt3+Qs91ab5gCuJfak0a02lUqnBBQ\\\\\\\/2dV0ddN2f\\\\\\\/UDqyoua19S05e0vjHITjSbI51qJtJTPmk6BNeB9\\\\\\\/MfprPxNk54WwpPUmuMfIMrSR3LIYj4teW5aHUPYrnCMEtS4+IVmZrFK0TaC+nYt+dSlT7eChH+YHtDN+rbfRUBleopa6ZKbQlyos6EdwXbAh4Dx2QPvbkwXo2UjhzDid6kMRy3ntwCtxPQur1LA\\\\\\\/MZgrRFkjvDMNj+6f6uWsrTCvpPj3W0HUj0BGhmjud97UiRdov\\\\\\\/q1fS7id9p84Hvf\\\\\\\/8GFghbD5V5SXUZCkm+vvgk6otP9Nswy9ShPq1QLyDl3otUi+Fp25UYdiFdm4EMwz7Z6AhxKuIEZb\\\\\\\/OkloSGarEGJ53AcLTFlu7TI+aKYDcL+Mf+z4u2kzIpAzllKFPNs\\\\\\\/Wixapv3a6reEMfJxLWDJZBu8Z1CW5rogwv2gz1yL3QD2P2aosBxVYPb2YaAZQZeShJQhLqTPItA+QP2geaqVhGeVa7oSfdrir+KOCo6bdUd81\\\\\\\/bN13IMmWG+lAAFidcfrBTW9GtX+k+3cDU0NaHZb\\\\\\\/pgbEtP+OcaNBWMDlphoLsDmLZIm2Pap4U4TNkeFE6Jw1Iz\\\\\\\/QuRw3hK95o2wyLWUoFIm0ExmlBvQUv1qTc2qyAa3nyEBJoLLSi3sGIwTPPfaJko4bfapzZUk8jYNQrFXSd8QgNfqcuzUQhi+Gw\\\\\\\/4Z5Qa5Fl8IbtnRd3n0Nb1DZSdqK1OW9pULlX8llQcN9egEEbFZWch81LaQr\\\\\\\/WpWEvwtnsRHvUJK4QVnFm9ebEmtfbf0oI1ft331IIDs\\\\\\\/TQaxe8+iaqo3c8SjD+T5lT0fg5gwRpi3\\\\\\\/GZSVKJEDzK3EkfVjoZrZNIQaWL4RSDfIaujX5Mlw64pZqcpv21jnrRSgyrluj3gm9T7c79ewo3ZLe5jCZ42jbwjq7RKiQsHGSXddZSknWz1tHh9AUFOmMevlhOvOxRBW73e6rYR0AxKtc2VsIdG7op9bQiLJu0MHWcRkLxMJnDcvZPmQUCTcCioq7WrqyPbEiKAYsMTQ8gCGamohC2nIFo7j43Svr94Wap+lOILNGM8Tre0dcwduqEwzK5ph8nb5cb6MTDsKLeBauUQD1oRu+Jd7iS1g+0wrOKTgb\\\\\\\/KqSAXy55aMrrVGt0BTMiL8y9P2N\\\\\\\/qwWvjbzvZp6tr6Ii18mHwF\\\\\\\/L\\\\\\\/v60FzWB\\\\\\\/hv+jFNziVFg8fLQjbPNfeYBBiNb33orDlwZiFwFG37AUHhO68j7QOC2fY8Ojsy+PCz6C\\\\\\\/8yKZMmJRVCm\\\\\\\/3qZQX2NFYNB2hJOp+OM0ux\\\\\\\/T1nPyx441V1PnGJD1ukLMKHA\\\u0022,\\\u0022iv\\\u0022:\\\u0022L2hUNVjnD7nWIRdKKKh7Ag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002263205c2918d6cbd332790d92aa0ecb68cc8318e90a45be73aaf88902ff7ef865\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hPWQAua6hmL6eenRWkA1t3yHDX\\\\\\\/cS8SgrePkXBWmuTG1DHRxbnxXqjY1xp5sNKZbp43MGiYwGZS+xPsWqgRQ8niPub9xRoy1KK2LKQlLCZsYD5gzp3FUxJe3ay4Eog12hEE0CChHZQmrIuxTJr4mnN70MvnImlpfBnsKkeHYyl3nGqT4GJZdgbC+1mlCyOFve76ynXM957pXI\\\\\\\/KEU2i9j7Uy6BGhKS+oyo35ImLx2itstC7Bj1\\\\\\\/OtQg\\\\\\\/PHXO3L3JyDUH+Fd7JZG7UorVU57CUufiXSLz6q1F9NXiwkbLqG26y6ZwLOwMybJAeD33HPxSjqWOL1uJ65sujKlajyQVebNZALx1VQ7CBkbCkxR7Eb3rTVWMfH5JeCI2XYp0nh8LX0wi1nKeLOKzwHjLyp02RbWxOE+xyUXvOYPWvjzcSeiLpBBH81recK7KuOsd2QH4Df7l4grnPb5lF9X5c4kqi9TeA6cValBZZdTAh3bIKKeB8x91+DFcnyFk6wAGZ3tn93l3qDeDJTOaFA2PFzBLqomncsc\\\\\\\/1TQORbCIrlWTFr+xw\\\\\\\/JxPpaamEWC2b1+JrIUhHsOAfWyHLPPCP+MBx9FNpjydhu9h1F9aFTra00k5I4Arz05wLr8sSEjPn7BLhaKZmS6A+yu6\\\\\\\/M\\\\\\\/G9OwcjKJYnL0MAZbb7Vx60gX08orvNdQgnXJK0DX9dlnuLuHW+5m14tk6cqEX9wzDuVFzzzcsyGHsPzf+RUhW10tYkCmlbahtNdJ2V\\\\\\\/UM7Zaldd89WYewqKiLr936GBa7BcSVMe0xQQBN+RhaWbrGv8hO82sIFrllFa5UF2uV02NHXafkffvlWFLu+7\\\\\\\/lCWgdE2GcC7aUE4y1QQvnKo\\\\\\\/X7qu4kMW5h3LS7LwlWpdagPAeS2r\\\\\\\/PdcfbsML3BORG2kXamwppaMKUsgXNk+AL\\\\\\\/zYKzd1xQvb\\\\\\\/pUc0gRNW942Os2Nwf0ZUuOX8iaWyspn1qKZIiNzHcr1YJ8xC41d5MqzFYnBIMHsMOObJCTF2FQxeq+lKoAqDv+S\\\\\\\/4pOhEbCVkP++5Oy08J29tP6OFHed3PiHxU7bXR3VmKcz53\\\\\\\/NrT2mSsFWWQOBYt2T6GU\\\\\\\/Lv2Z1GmHpLh111q7D98gWmbBQvPog4j5MXYqr1Kf0j057VDFY6Ox3ZmZftpgvkN\\\\\\\/P2jgZx6vppfZ3Q3soXH0FFIDMT+BOtKWNnirnWlm9kFV1lo9AjisKCJP8\\\\\\\/V9bKlhQ+FawdPxPSDDw8RohhsEBBRJOFcBgNL8ekDQF4hp0xwK06thTUfn2lZjXz\\\\\\\/+Tv8m\\\\\\\/jRD6S3YhzwqmxFvmE5Eb1bVlH2Rtiasp\\\\\\\/reTUsGWVLh9UP5Lsm91BGLP5Z4kXR3W9V6faO1t9mQsRInfQWO\\\\\\\/SHjlbNDgjYGIXiFz4SaxEEDpwUQtjoVzanvUBkNMlwHBRtzglMyCYlHS7slnr9279f5To84ckzoBDUjY+xXoTiMfxHP0wDvLv5Qs06ymVRe9fT+dohIKNccc7htTV7NTzqqN8EYGjROAJBxbvik331qSzFMuXhHHzKsJujsAqGmgGnGSfz7riSrFKpSwzltYw9zimybMv482ZjGcAxcERQB1lMd22MYV9h\\\\\\\/lttxQCyyo+RDeLNSsJVbkyO+O\\\\\\\/EV9PodrPEXbAlYGmbEvuWo9I2\\\\\\\/R1N9d1kAqV\\\\\\\/OXVxFz2+\\\\\\\/S+8Z+OQd8er+XeUcbLDoR+vgQeZDhade7U7cWU75Tg1yZ+Nummpw9wNKrJkp6vkbPvHmcqc\\\\\\\/POXBHWsrhQVbpEpqFo1fkek2GG\\\\\\\/N+APRq7wJSxD8RYkFQzKWv+XH64NKnlng9edCoBr4yw9tBIxI6YaoD1yEotgbQ7WIzVFYcNG5jzIxU9fTZ4zDk3E+Qoq1WPnCHGmdqGvhixqolOYxym6X7hY\\\\\\\/UKBB5F5NKxAqgUOIRr5b\\\\\\\/Zm+QaxlckkHZNqRVEJiK8krycfnZOR5EiK7efb8bcD9Hy1gQqBWDz9jBpE16REzM1YwO6ixffXb\\\\\\\/5UhOPdxhQGpilvMbQKW7KQmByOBjoP5Y1wGeXk8E=\\\u0022,\\\u0022iv\\\u0022:\\\u0022rCK2nei4oHrTblRym6zUYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227fa4a515f44bdb5b701466255b06885b357c66bf48c50d998916e75da01aee29\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483825 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1191" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8hoHMD_7aaGh-e8KepYLK_zv_tg1HqBrOhksgdX5DTk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1191" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7naJ_EWn3aJChyt3gM7pQB4W-FBLzdOTZIHLs0QjfcQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1177" title="Construction of a High-Density American Cranberry (Vaccinium macrocarpon Ait.) Composite Map Using Genotyping-by-Sequencing for Multi-pedigree Linkage Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1201" title="Unstable Inheritance of 45S rRNA Genes in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483825" id="node483825" data-pisa="ggg;7/4/1191" data-pisa-master="ggg;g3.116.037390" data-apath="/ggg/7/4/1191.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Grzegorz Zapotoczny</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4424-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeff Sekelsky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1191-1199;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037390 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483825" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Grzegorz Zapotoczny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grzegorz%2BZapotoczny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zapotoczny%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrzegorz%2BZapotoczny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeff Sekelsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Integrative Program for Biological and Genome Sciences, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeff%2BSekelsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sekelsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeff%2BSekelsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4424-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeff Sekelsky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sekelsky@unc.edu" class="" data-icon-position="" data-hide-link-title="0">sekelsky@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/4/1191" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483825" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1191.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483825" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1191.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483825" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1191.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483825" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/4/1191.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483825" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">DNA double-strand breaks (DSBs) are one of the most deleterious types of lesions to the genome. Synthesis-dependent strand annealing (SDSA) is thought to be a major pathway of DSB repair, but direct tests of this model have only been conducted in budding yeast and <em>Drosophila</em>. To better understand this pathway, we developed an SDSA assay for use in human cells. Our results support the hypothesis that SDSA is an important DSB repair mechanism in human cells. We used siRNA knockdown to assess the roles of a number of helicases suggested to promote SDSA. None of the helicase knockdowns reduced SDSA, but knocking down BLM or RTEL1 increased SDSA. Molecular analysis of repair products suggests that these helicases may prevent long-tract repair synthesis. Since the major alternative to SDSA (repair involving a double-Holliday junction intermediate) can lead to crossovers, we also developed a fluorescent assay that detects crossovers generated during DSB repair. Together, these assays will be useful in investigating features and mechanisms of SDSA and crossover pathways in human cells.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/double-strand-break-repair" class="hw-term hw-article-keyword hw-article-keyword-double-strand-break-repair" rel="nofollow">double-strand break repair</a></li><li class="kwd"><a href="/keyword/crossing-over" class="hw-term hw-article-keyword hw-article-keyword-crossing-over" rel="nofollow">crossing over</a></li><li class="kwd"><a href="/keyword/synthesis-dependent-strand-annealing" class="hw-term hw-article-keyword hw-article-keyword-synthesis-dependent-strand-annealing" rel="nofollow">synthesis-dependent strand annealing</a></li></ul><p id="p-3">Double-strand breaks (DSBs) are considered to be one of the most detrimental types of DNA damage. There are numerous mechanisms for repairing DSBs, broadly classified into end joining and homology-directed recombination (HDR). Among the latter, the double-strand break repair (DSBR) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) model has been popular since it was proposed &gt;30 yr ago (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Szostak <em>et al.</em> 1983</a>). A hallmark of this model is the double-Holliday junction (dHJ) intermediate, which has two of the four-stranded junctions originally hypothesized by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Holliday (1964)</a>. In DSBR, as in Holliday’s model, specialized nucleases resolve Holliday junctions (HJs) by introducing symmetric nicks; independent resolution of the two HJs results in 50% of repair events having a reciprocal crossover. It has also been proposed that dHJs can be processed without the action of a nuclease if a helicase and topoisomerase migrate the two HJs toward one another and then decatenate the remaining link (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Thaler <em>et al.</em> 1987</a>); this process has been called dissolution to distinguish it from endonucleolytic resolution (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wu and Hickson 2003</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Models of DSB repair by homologous recombination. (A) Blue lines represent two strands of a DNA duplex that has experienced a DSB. HDR begins with resection to expose single-stranded DNA with 3&#x2032; ends (arrows). One of these can undergo strand invasion into a homologous duplex (red) to generate a D-loop; the 3&#x2032; invading end is then extended by synthesis. (B) In SDSA, the nascent strand is dissociated and anneals to the other resected end of the DSB. Completion of SDSA may result in noncrossover gene conversion (red patch, shown after repair of any mismatches). (C) An alternative to SDSA is annealing of the strand displaced by synthesis to the other resected end of the DSB. Additional synthesis can lead a dHJ intermediate. (D) In DSBR, the dHJ is resolved by cutting to generate either crossover or noncrossover products (one of two possible outcomes for each case is shown). (E) The dHJ can also be dissolved by a helicase-topoisomerase complex to generate noncrossover products." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1697172406" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Models of DSB repair by homologous recombination. (A) Blue lines represent two strands of a DNA duplex that has experienced a DSB. HDR begins with resection to expose single-stranded DNA with 3&#x2032; ends (arrows). One of these can undergo strand invasion into a homologous duplex (red) to generate a D-loop; the 3&#x2032; invading end is then extended by synthesis. (B) In SDSA, the nascent strand is dissociated and anneals to the other resected end of the DSB. Completion of SDSA may result in noncrossover gene conversion (red patch, shown after repair of any mismatches). (C) An alternative to SDSA is annealing of the strand displaced by synthesis to the other resected end of the DSB. Additional synthesis can lead a dHJ intermediate. (D) In DSBR, the dHJ is resolved by cutting to generate either crossover or noncrossover products (one of two possible outcomes for each case is shown). (E) The dHJ can also be dissolved by a helicase-topoisomerase complex to generate noncrossover products.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1191/F1.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/4/1191/F1.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483926" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-4" class="first-child">Models of DSB repair by homologous recombination. (A) Blue lines represent two strands of a DNA duplex that has experienced a DSB. HDR begins with resection to expose single-stranded DNA with 3′ ends (arrows). One of these can undergo strand invasion into a homologous duplex (red) to generate a D-loop; the 3′ invading end is then extended by synthesis. (B) In SDSA, the nascent strand is dissociated and anneals to the other resected end of the DSB. Completion of SDSA may result in noncrossover gene conversion (red patch, shown after repair of any mismatches). (C) An alternative to SDSA is annealing of the strand displaced by synthesis to the other resected end of the DSB. Additional synthesis can lead a dHJ intermediate. (D) In DSBR, the dHJ is resolved by cutting to generate either crossover or noncrossover products (one of two possible outcomes for each case is shown). (E) The dHJ can also be dissolved by a helicase-topoisomerase complex to generate noncrossover products.</p><div class="sb-div caption-clear"></div></div></div><p id="p-5">In studies of DNA DSB repair resulting from transposable element excision in <em>Drosophila</em>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nassif <em>et al.</em> (1994)</a> noted that crossovers were infrequent and the two ends of a single DSB could use different repair templates. To explain these results, they proposed the synthesis-dependent strand annealing (SDSA) model (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). In addition to continued use of <em>Drosophila</em> gap-repair assays (<em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kurkulos <em>et al.</em> 1994</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2003</a>), other types of evidence have been interpreted as support for the SDSA model. In <em>Saccharomyces cerevisiae</em> meiotic recombination, gel-based separation and quantification of intermediates and products showed that noncrossovers are made before dHJs appear, suggesting that these noncrossovers are generated by SDSA (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allers and Lichten 2001</a>). In vegetatively growing <em>S. cerevisiae</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mitchel <em>et al.</em> (2010)</a> studied repair of a small gap DSB in cells defective in mismatch repair. Based on the high frequency with which heteroduplex DNA tracts (regions that contain one template strand and one recipient strand) in noncrossover products were restricted to one side of the DSB, they concluded that most noncrossover repair occurred through SDSA. <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Miura <em>et al.</em> (2012)</a> used an <em>S. cerevisiae</em> assay designed specifically to detect SDSA. A plasmid with a DSB was introduced into cells in which templates homologous to the two sides of the DSB were on different chromosomes, eliminating the possibility of a dHJ intermediate. Based on results of these various assays, many researchers now believe SDSA to be the most common mechanism of mitotic DSB repair by HDR (reviewed in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andersen and Sekelsky 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Verma and Greenberg 2016</a>).</p><p id="p-6">In mammalian cells, the direct-repeat GFP (DR-GFP) assay (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pierce <em>et al.</em> 1999</a>) has been an instrumental tool for studying DSB repair by HDR. In this assay, an upstream <em>GFP</em> gene (<em>SceGFP</em>) is disrupted by insertion of an <em>I-Sce</em>I site, and a downstream <em>GFP</em> fragment (<em>iGFP</em>) serves as a template for repair. Gene conversion replaces the region surrounding the <em>I-Sce</em>I site in <em>SceGFP</em>, generating an intact <em>GFP</em> gene. This gene conversion has been suggested to arise through SDSA, but it is not possible to distinguish between SDSA and other noncrossover DSB repair in this assay (see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Xu <em>et al.</em> (2012)</a> developed a novel human cell assay in which gene conversion could be detected simultaneously at the DSB site and at another site ∼1 kbp away. They found that the two were often independent and concluded that SDSA is a major mechanism for DSB repair in human cells, but they also could not exclude DSBR as a possible source.</p><p id="p-7">Development of the CRISPR/Cas9 system for genome engineering (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cong <em>et al.</em> 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mali <em>et al.</em> 2013</a>) provides additional emphasis on the importance of understanding SDSA mechanisms in human cells, as it has been suggested that replacement of multi-kilobase pair fragments after Cas9 cleavage, and probably other HDR events, occurs through SDSA (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Byrne <em>et al.</em> 2015</a>). We therefore designed an assay to detect DSB repair by SDSA in human cells. Here, we describe this assay and show that, as hypothesized, SDSA appears to be an important pathway for HDR in human cells. We report the effects of knocking down various proteins proposed to function during SDSA. We also describe a fluorescence-based assay for detecting crossovers generated during DSB repair. Use of these assays should help to further our understanding of DSB repair pathways used in human cells.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Construction of assay plasmids</h3><p id="p-8">The SDSA assay construct, pGZ-DSB-SDSA, was based on pEF1α-mCherry-C1 vector (catalog no. 631972; Clonetech). A fragment of <em>mCherry</em> was removed by cutting with <em>Nhe</em>I and <em>Hin</em>dIII and inserting annealed oligonucleotides containing an <em>I-Sce</em>I recognition sequence and a part of the <em>mCherry</em> sequence. The product, pEF1a-mCherry-I, had 350 bp of <em>mCherry</em> deleted and replaced with an <em>I-Sce</em>I recognition sequence. In parallel, 5′ and 3′ <em>mCherry</em> fragments, overlapping by 350 bp, were PCR-amplified and cloned into a vector containing a fragment of the <em>copia</em> retrotransposon from <em>D. melanogaster</em>. A fragment of <em>HPRT</em> was cloned out of the DR-GFP construct. This entire module (5′ <em>mCherry</em>–<em>copia</em>–3′ <em>mCherry</em>–<em>HPRT</em>) was PCR-amplified and cloned into the pEF1a-mCherry-I to produce pGZ-DSB-SDSA. The full sequence was deposited in GenBank under accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY447299&amp;atom=%2Fggg%2F7%2F4%2F1191.atom">KY447299</a>.</p><p id="p-9">The crossover assay construct, pGZ-DSB-CO, was based on pHPRT-DRGFP (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pierce <em>et al.</em> 2001</a>) and the intron-containing <em>mCherry</em> gene from pDN-D2irC6kwh (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nevozhay <em>et al.</em> 2013</a>). The <em>iGFP</em> fragment was expanded to include the entire 3′ end of the transcribed region, and this was cloned into the <em>mCherry</em> intron of pDN-D2irC6kwh. This module (<em>mCherry</em> with an intron containing 3′ <em>GFP</em>) was cloned into the pHPRT-DRGFP vector cut with <em>Hin</em>dIII, so that it replaced the <em>iGFP</em> fragment and was in reverse orientation relative to the <em>SceGFP</em> gene. The full sequence of pGZ-DSB-CO was deposited in GenBank under accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY447298&amp;atom=%2Fggg%2F7%2F4%2F1191.atom">KY447298</a>.</p></div><div id="sec-3" class="subsection"><h3>Generation of stably-transfected cell lines</h3><p id="p-10">U2OS and HeLa cells were cultured under normal conditions (DMEM + 10% FBS + pen/strep) for 24 hr until they reached 80% confluency before transfection with either SDSA or crossover assay constructs using a Nucleofector 2b Device (catalog no. AAB-1001; Lonza) and Cell Line Nucleofector Kit V (catalog no. VCA-1003; Lonza). At 1 wk post-transfection, appropriate antibiotics were added to select for the cells with a stably-integrated construct. pGZ-DSB-SDSA assay has a gene for neomycin resistance; cells receiving this construct were treated with 700 μg/ml G418 (catalog no. A1720; Sigma) for 1 wk and then single-cell clones were derived. pGZ-DSB-CO contains a <em>PGK1</em> gene that confers resistance to puromycin; cells receiving this construct were treated with 10 μg/ml puromycin (catalog no. P8833; Sigma) for 1 wk and then single-cell clones were derived. Initial attempts to determine copy number by Southern blot were unsuccessful; however, the analyses described below strongly suggested that the lines we characterized each carried a single insertion or possibly a single tandem array.</p></div><div id="sec-4" class="subsection"><h3>DNA repair assays and flow cytometry</h3><p id="p-11">U2OS cells with pGZ-DSB-SDSA integrated were cultured in 10-cm dishes containing 10 ml of DMEM medium with high glucose (Corning) until split onto six-well plates at a concentration of 5 × 10<sup>4</sup> cells/ml using 0.05% trypsin 0.53 mM EDTA solution (Corning). Upon reaching ∼60% confluency, the cells were treated with an siRNA reaction mixture (90 nmol siRNA and 8 μl lipofectamine 2000 reagent per well; Invitrogen). At 24 hr after transfection, the siRNA reaction mixture was replaced with the fresh culture medium. After 12 hr the cells were split so that knockdown could assessed in one half (see <em>qPCR evaluation of the siRNA knockdown efficiency</em>). The other half was treated with 100 μl <em>I-Sce</em>I–expressing adenovirus (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Anglana and Bacchetti 1999</a>) (previously titrated to a nonlethal concentration). After another 24 hr, the medium was replaced and thus the adenovirus removed. After another 72 hr, the cells were harvested and resuspended in 1× PBS (Corning) supplemented with 2% fetal bovine serum (FBS) and 5 mM EDTA, for flow cytometry acquisition on a BD LSRFortessa, using 488 and 561 nm lasers to detect the mCherry fluorescence.</p><p id="p-12">U2OS cells with pGZ-DSB-CO integrated were cultured and treated under the same conditions. Flow cytometry acquisition was conducted on a BD FACSAriaII, using 388 and 532 nm lasers to detect GFP and mCherry fluorescence.</p></div><div id="sec-5" class="subsection"><h3>U2OS genomic DNA isolation</h3><p id="p-13">Cells were cultured in a 15-cm dish until they reached 100% confluency, then rinsed with 1× PBS and harvested in 0.05% trypsin, 0.53 mM EDTA, by centrifuging for 3 min at 2000 rpm. Cells were washed with PBS and transferred to 1.5-ml microfuge tubes and spun for 10 sec to repellet. PBS was removed and cells were resuspended in TSM (10 mM Tris-HCl, pH 7.4; 140 mM NaCl; 1.5 mM MgCl<sub>2</sub>) with 0.5% NP-40 and incubated on ice for 2–3 min. After pelleting, cells were resuspended in 1 ml nuclei dropping buffer (0.075 M NaCl; 0.024 M EDTA, pH 8.0). The suspension was transferred to a 15-ml tube containing 4 ml nuclei dropping buffer with 1 mg Proteinase K (final Proteinase K concentration = 0.2 mg/ml), and 0.5% SDS. The cells were lysed overnight at 37°. The next day, an equal volume of phenol was added and mixed on an orbital shaker for 2 hr followed by a 5-min spin at 2000 rpm. The aqueous phase was transferred to a clean tube, an equal volume of chloroform was added, and the mix was incubated for 30 min on an orbital shaker. After spinning, the aqueous phase was transferred to a new tube and 0.1 vol 3 M NaOAc was added, followed by 1 vol isopropanol. The DNA was spooled out using a glass Pasteur pipette and resuspended overnight in 1 ml TE buffer (10 mM Tris-HCl, pH 8.0; 1 mM EDTA). The next day, the DNA was precipitated using 0.5 vol 7.5 M NH<sub>4</sub>OAc and 2 vol ethanol. DNA was spooled out and resuspended in 0.5 ml TE-4 buffer (10 mM Tris-HCl, pH 8.0; 0.1 mM EDTA). Samples were stored in 4° until analyzed.</p></div><div id="sec-6" class="subsection"><h3>PCR analysis of the repair events</h3><p id="p-14">DNA from the BRCA2 knockdown repair events was isolated according to the protocol described in <em>U2OS genomic DNA isolation</em>, and used in a PCR reaction to amplify a desired DNA fragment for sequencing or fragment length characterization. A total of 1.5 μl DNA was added to each PCR mixture containing primer sets according to Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>, iProof polymerase (catalog no. 424264; BioRad) and buffer. PCR amplification reaction program was 33 cycles of the following: 20 sec at 98°, 20 sec at 64°, and 20–150 sec at 72°. Products were run on 1–1.5% agarose gels with ethidium bromide before being imaged.</p></div><div id="sec-7" class="subsection"><h3>Western blot of BLM protein in siRNA-treated cells</h3><p id="p-15">Cells treated with siRNA as described in <em>DNA repair assays and flow cytometry</em> were harvested on the third day post-transfection using 0.05% Trypsin, 0.53% mM EDTA solution (Corning). After washing with 1× PBS, the cells were resuspended in a protein sample buffer (Tris-HCl; SDS; glycerol; bromophenol blue; 150 mM DTT) and boiled. A total of 20 μl of the protein sample was loaded on a 7.5% SDS-PAGE gel and the gel was run for 1–2 hr at 100 V. Protein was transferred to a PVDF membrane using a wet transfer method (1.5 hr at 90 V in 4°). The membrane was blocked in PBS with 5% powdered milk and incubated in PBS plus 0.1% Triton-X plus primary antibodies [rabbit anti-BLM (catalog no. 2179; Abcam) at 1:2000 and mouse anti-αTubulin (catalog no. T9026; Sigma) at 1:8000] overnight at 4° on a rocker. The membrane was then washed three times in PBS-T solution. HDRP-conjugated secondary antibodies were added (goat anti-rabbit at 1:5000 and goat anti-mouse at 1:100,000) and the blot was incubated for 1 hr at room temperature. The membrane was washed three times in PBS-T solution and then incubated in an ECL solution (Thermo Fisher Scientific) for chemiluminescence for 2 min. The Western blot image was taken using a BioRad Molecular Imager (ChemiDoc XRS+) or the X-ray film was developed using a developer.</p></div><div id="sec-8" class="subsection"><h3>qPCR evaluation of the siRNA knockdown efficiency</h3><p id="p-16">Cells treated with siRNA as described in <em>DNA repair assays and flow cytometry</em> were harvested on the third day post-transfection using 0.05% Trypsin 0.53% mM EDTA solution (Corning). RNA was extracted using the manufacturer’s protocol for ReliaPrep RNA Cell Miniprep System (Promega). Purified RNA was used as a template to generate the cDNA library with QuantiTect Reverse Transcription Kit (catalog no. 205310; Qiagen). The qPCR mix contained gene-specific DNA primers, cDNA, and the QuantiTect SYBR Green PCR kit (catalog no. A 204141; Qiagen). Amplification and quantification was conducted on a RealTime PCR machine (QuantStudio 6 flex Real Time PCR System).</p></div><div id="sec-9" class="subsection"><h3>Statistical analysis</h3><p id="p-17">Statistical comparisons were performed on the raw data (Tables S2 and S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>) using GraphPad Prism version 6.07 for Windows (GraphPad Software Inc., La Jolla, CA). In the case of BLM knockdown in the SDSA assay, one value (271% of control) was found to be a significant outlier based on the Grubb’s test using GraphPad QuickCalcs online (<a href="https://graphpad.com/quickcalcs/grubbs2/">https://graphpad.com/quickcalcs/grubbs2/</a>), and was excluded from further analysis.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-18">All data are in the figures or the supplemental tables in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>. Cell lines and plasmids are available on request. Sequences of assay constructs were deposited in GenBank under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY447298&amp;atom=%2Fggg%2F7%2F4%2F1191.atom">KY447298</a> (pGZ-DSB-CO) and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY447299&amp;atom=%2Fggg%2F7%2F4%2F1191.atom">KY447299</a> (pGZ-DSB-SDSA).</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>An SDSA assay for human cells</h3><p id="p-19">To study SDSA in human cells, we used an approach conceptually similar to the <em>P</em>{<em>w<sup>a</sup></em>} assay used in <em>Drosophila</em> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">McVey <em>et al.</em> 2004</a>). In this assay, if both ends of the DSB generated by <em>P</em> element excision are extended by synthesis from the sister chromatid, the nascent strands can anneal at repeats inside the <em>P</em>{<em>w<sup>a</sup></em>} element, generating a product that is unique to SDSA and easily distinguishable by phenotype. To mimic this situation in human cells, we built a construct (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) that has an <em>mCherry</em> gene in which a 350-bp segment was replaced with the 18 bp <em>I-Sce</em>I recognition sequence, rendering the gene nonfunctional. When a DSB is generated by <em>I-Sce</em>I (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), HDR can be completed using a downstream repair template. The repair template is split: each half has 800 bp of homology adjacent to the break site plus the 350 bp of deleted <em>mCherry</em> sequence. The two halves are separated by a 3-kbp spacer of unique sequence. Since the 350-bp sequence is on both sides of the spacer, it constitutes a direct repeat. We hypothesized that both ends of the <em>I-Sce</em>I–induced break will invade the side of the template to which they are homologous, either simultaneously or sequentially (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). If synthesis on both sides extends through or far enough into the 350-bp repeat before the nascent strands are dissociated from the template, the overlapping regions can anneal to one another (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Completion of SDSA restores a functional <em>mCherry</em> gene at the upstream location.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An SDSA assay for human cells. (A) Schematic of the assay construct. The mCherry coding sequence is represented with a large arrow, with the yellow box indicating the site at which a 350-bp fragment was removed and replaced with an I-SceI site. The thick red lines designate the promoter and 5&#x2032; and 3&#x2032; untranslated regions. Downstream of mCherry is the neo selectable marker and vector sequences (black arrow and line, respectively). The repair template consists of 800 bp of homology to the left of the DSB site, then the 350-bp deleted fragment (magenta), a 3-kbp spacer (blue), another copy of the 350-bp fragment, and 800 bp of homology to the right end of the DSB site. The lines below are a representation of the same sequences as duplex DNA, for use in subsequent panels. Arrowheads indicate 3&#x2032; ends at the outer edges of the construct in all panels. (B) Two-ended SDSA repair. The DSB is shown as resected and paired with a template. This diagram shows interchromatid repair, where the template on the sister chromatid is used (whether or not it was cut by I-SceI). Intrachromatid repair may also be possible; in this case, the black lines to the right of the DSB would be continuous with those to the left of the template. After strand exchange, synthesis into the 350-bp regions, and dissociation, complementary sequences can anneal. In the example shown here, the left end has been extended through the 350-bp fragment into the spacer; the right end synthesized only midway into the 350-bp fragment. After trimming of the spacer sequence, filling of gaps, and ligation, a restored mCherry is produced, resulting in red fluorescence. (C) One-ended sequential SDSA. This is similar to (B), except only the left end of the DSB has paired with the template. After synthesis and dissociation, the nascent strand can pair with the second half of the template. Additional synthesis can extend the nascent strand to provide complementarity to the other end of the DSB, allowing annealing and completion of SDSA. (D) If one or both ends invade the template and synthesis traverses the entire template, a dHJ can form (top). If the dHJ is dissolved or resolved in the noncrossover orientation, the product shown in the middle is generated. The bottom part of this panel shows the chromosomal product of crossover resolution (there is also an acentric extrachromosomal circle that will be lost). (E) A product produced by initiation of repair by SDSA but completion by end joining. In this case, part of the spacer has been copied into the upstream mCherry (sequences from the template are indicated with green lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1697172406" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An SDSA assay for human cells. (A) Schematic of the assay construct. The &lt;em&gt;mCherry&lt;/em&gt; coding sequence is represented with a large arrow, with the yellow box indicating the site at which a 350-bp fragment was removed and replaced with an &lt;em&gt;I-Sce&lt;/em&gt;I site. The thick red lines designate the promoter and 5&#x2032; and 3&#x2032; untranslated regions. Downstream of &lt;em&gt;mCherry&lt;/em&gt; is the &lt;em&gt;neo&lt;/em&gt; selectable marker and vector sequences (black arrow and line, respectively). The repair template consists of 800 bp of homology to the left of the DSB site, then the 350-bp deleted fragment (magenta), a 3-kbp spacer (blue), another copy of the 350-bp fragment, and 800 bp of homology to the right end of the DSB site. The lines below are a representation of the same sequences as duplex DNA, for use in subsequent panels. Arrowheads indicate 3&#x2032; ends at the outer edges of the construct in all panels. (B) Two-ended SDSA repair. The DSB is shown as resected and paired with a template. This diagram shows interchromatid repair, where the template on the sister chromatid is used (whether or not it was cut by &lt;em&gt;I-Sce&lt;/em&gt;I). Intrachromatid repair may also be possible; in this case, the black lines to the right of the DSB would be continuous with those to the left of the template. After strand exchange, synthesis into the 350-bp regions, and dissociation, complementary sequences can anneal. In the example shown here, the left end has been extended through the 350-bp fragment into the spacer; the right end synthesized only midway into the 350-bp fragment. After trimming of the spacer sequence, filling of gaps, and ligation, a restored &lt;em&gt;mCherry&lt;/em&gt; is produced, resulting in red fluorescence. (C) One-ended sequential SDSA. This is similar to (B), except only the left end of the DSB has paired with the template. After synthesis and dissociation, the nascent strand can pair with the second half of the template. Additional synthesis can extend the nascent strand to provide complementarity to the other end of the DSB, allowing annealing and completion of SDSA. (D) If one or both ends invade the template and synthesis traverses the entire template, a dHJ can form (top). If the dHJ is dissolved or resolved in the noncrossover orientation, the product shown in the middle is generated. The bottom part of this panel shows the chromosomal product of crossover resolution (there is also an acentric extrachromosomal circle that will be lost). (E) A product produced by initiation of repair by SDSA but completion by end joining. In this case, part of the spacer has been copied into the upstream &lt;em&gt;mCherry&lt;/em&gt; (sequences from the template are indicated with green lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1191/F2.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/4/1191/F2.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">An SDSA assay for human cells. (A) Schematic of the assay construct. The <em>mCherry</em> coding sequence is represented with a large arrow, with the yellow box indicating the site at which a 350-bp fragment was removed and replaced with an <em>I-Sce</em>I site. The thick red lines designate the promoter and 5′ and 3′ untranslated regions. Downstream of <em>mCherry</em> is the <em>neo</em> selectable marker and vector sequences (black arrow and line, respectively). The repair template consists of 800 bp of homology to the left of the DSB site, then the 350-bp deleted fragment (magenta), a 3-kbp spacer (blue), another copy of the 350-bp fragment, and 800 bp of homology to the right end of the DSB site. The lines below are a representation of the same sequences as duplex DNA, for use in subsequent panels. Arrowheads indicate 3′ ends at the outer edges of the construct in all panels. (B) Two-ended SDSA repair. The DSB is shown as resected and paired with a template. This diagram shows interchromatid repair, where the template on the sister chromatid is used (whether or not it was cut by <em>I-Sce</em>I). Intrachromatid repair may also be possible; in this case, the black lines to the right of the DSB would be continuous with those to the left of the template. After strand exchange, synthesis into the 350-bp regions, and dissociation, complementary sequences can anneal. In the example shown here, the left end has been extended through the 350-bp fragment into the spacer; the right end synthesized only midway into the 350-bp fragment. After trimming of the spacer sequence, filling of gaps, and ligation, a restored <em>mCherry</em> is produced, resulting in red fluorescence. (C) One-ended sequential SDSA. This is similar to (B), except only the left end of the DSB has paired with the template. After synthesis and dissociation, the nascent strand can pair with the second half of the template. Additional synthesis can extend the nascent strand to provide complementarity to the other end of the DSB, allowing annealing and completion of SDSA. (D) If one or both ends invade the template and synthesis traverses the entire template, a dHJ can form (top). If the dHJ is dissolved or resolved in the noncrossover orientation, the product shown in the middle is generated. The bottom part of this panel shows the chromosomal product of crossover resolution (there is also an acentric extrachromosomal circle that will be lost). (E) A product produced by initiation of repair by SDSA but completion by end joining. In this case, part of the spacer has been copied into the upstream <em>mCherry</em> (sequences from the template are indicated with green lines).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The scenario above requires two-ended SDSA, but sequential one-ended SDSA is also possible (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). If only one end of the break invades the downstream template, is extended by repair synthesis, and is then dissociated from the template, the nascent strand will not be complementary to the other resected end; however, this nascent strand will have homology to the repeat on the other side of the template. A second cycle of strand exchange and repair synthesis using the other repeat could lead to addition of sequences complementary to the other resected DSB end. This would also restore a functional <em>mCherry</em> gene by SDSA.</p><p id="p-22">A functional <em>mCherry</em> gene might also be generated through a combination of SDSA and DSBR. In the sequential SDSA scenario, the second strand exchange event could be processed into a dHJ. The product of dissolution or noncrossover resolution of such a dHJ will be identical to that of SDSA (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>), but if the dHJ is resolved as a crossover, generation of a functional <em>mCherry</em> gene will be accompanied by a deletion of all sequences between the upstream <em>mCherry</em> and the downstream template. Dissolution or noncrossover dHJ resolution in this scenario cannot be distinguished from SDSA, but it should be noted that formation of such a dHJ intermediate still requires at least one cycle of D-loop disassembly—a key step that separates SDSA from DSBR (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-23">Other types of repair that do not generate a functional <em>mCherry</em> are possible. A dHJ can be generated if synthesis extends through one <em>mCherry</em> 350-bp repeat, the entire spacer, and the other 350-bp repeat (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>). Processing of this dHJ would give a product in which the entire template, including the duplicated 350-bp sequences and the spacer, was copied into the upstream <em>mCherry</em> gene. Dissolution or noncrossover resolution would result in two copies of the template (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2D</a>, middle), whereas crossover resolution would delete intervening sequences (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2D</a>, bottom). Nonhomologous end joining (NHEJ) can restore or disrupt the <em>I-Sce</em>I recognition sequence, depending on whether it is precise or imprecise (not depicted). Hybrid repair, in which repair is initiated by HDR but completed by end joining instead of annealing, can give rise to an <em>mCherry</em> in which the 350-bp gap is not completely filled or, if synthesis extends into the spacer, in which part of the spacer is copied into the upstream <em>mCherry</em> gene (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2E</a>).</p><p id="p-24">To generate cell lines with the SDSA repair construct, we transfected both U2OS and HeLa cells with linearized SDSA construct and used G418 to select stably-transfected lines. To induce DSBs, we infected cells with an adenovirus expressing <em>I-Sce</em>I (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Anglana and Bacchetti 1999</a>). We detected mCherry activity by fluorescence microscopy (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and Figure S2A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>) as early as 2 d after viral infection. Stable expression persisted through months of cell culturing. Molecular analysis of genomic DNA from clones derived from single mCherry-positive cells confirmed the absence of the <em>I-Sce</em>I, restoration of a complete <em>mCherry</em> gene, and the presence of an intact repair template (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>). We quantified SDSA repair through flow cytometry (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from the SDSA assay. (A) Cells fluorescing red due to mCherry expression after I-SceI infection. (B) Flow cytometry of cells after I-SceI expression. In this example, 10,000 singlet cells were assayed and 200 were gated as exhibiting red fluorescence. (C) Effects of siRNA knockdown on acquisition of mCherry expression (see Materials and Methods). Fluorescence frequencies from flow cytometry were normalized to NT; raw data are given in Table S2 in File S1. Error bars indicate SD. The ratio paired t-test was used to compare raw values for each siRNA target to its NT control and each double-knockdown to the single knockdown of the corresponding helicase. For helicase single knockdowns, P values were Bonferroni-corrected for multiple (six) comparisons. * P &lt; 0.05, ** P &lt; 0.01, **** P &lt; 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1697172406" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results from the SDSA assay. (A) Cells fluorescing red due to mCherry expression after &lt;em&gt;I-Sce&lt;/em&gt;I infection. (B) Flow cytometry of cells after &lt;em&gt;I-Sce&lt;/em&gt;I expression. In this example, 10,000 singlet cells were assayed and 200 were gated as exhibiting red fluorescence. (C) Effects of siRNA knockdown on acquisition of mCherry expression (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Fluorescence frequencies from flow cytometry were normalized to NT; raw data are given in Table S2 in File S1. Error bars indicate SD. The ratio paired &lt;em&gt;t&lt;/em&gt;-test was used to compare raw values for each siRNA target to its NT control and each double-knockdown to the single knockdown of the corresponding helicase. For helicase single knockdowns, &lt;em&gt;P&lt;/em&gt; values were Bonferroni-corrected for multiple (six) comparisons. * &lt;em&gt;P&lt;/em&gt; &lt; 0.05, ** &lt;em&gt;P&lt;/em&gt; &lt; 0.01, **** &lt;em&gt;P&lt;/em&gt; &lt; 0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1191/F3.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/4/1191/F3.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483871" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Results from the SDSA assay. (A) Cells fluorescing red due to mCherry expression after <em>I-Sce</em>I infection. (B) Flow cytometry of cells after <em>I-Sce</em>I expression. In this example, 10,000 singlet cells were assayed and 200 were gated as exhibiting red fluorescence. (C) Effects of siRNA knockdown on acquisition of mCherry expression (see <em>Materials and Methods</em>). Fluorescence frequencies from flow cytometry were normalized to NT; raw data are given in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>. Error bars indicate SD. The ratio paired <em>t</em>-test was used to compare raw values for each siRNA target to its NT control and each double-knockdown to the single knockdown of the corresponding helicase. For helicase single knockdowns, <em>P</em> values were Bonferroni-corrected for multiple (six) comparisons. * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01, **** <em>P</em> &lt; 0.0001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We obtained mCherry-positive cells in multiple isolates of each cell type, and we selected one U2OS cell isolate for further characterization. Since we intended to use siRNA to knock down proteins that might be required for SDSA, we first knocked down BRCA2 as a positive control. BRCA2 is essential for RAD51-mediated strand exchange and for initiation of HDR (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sharan <em>et al.</em> 1997</a>). Consistent with this function, knocking down BRCA2 resulted in a significant decrease in red-fluorescing cells relative to the nontargeting (NT) siRNA-negative control (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Although there was substantial residual HDR, probably due to incomplete loss of BRCA2 (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>), we conclude that SDSA does occur in human cells and that our assay can be used to study this process.</p></div><div id="sec-13" class="subsection"><h3>An assay for detecting crossovers generated during DSB repair in human cells</h3><p id="p-27">D-loop dissociation is a critical step in the SDSA pathway. If the D-loop is not dissociated, 2nd-end capture may lead to formation of a dHJ intermediate (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). It is thought that most dHJs formed in proliferating cells are processed by dissolution to give noncrossover products, but resolution by nicking can generate crossovers (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1D</a>). To complement our assay for SDSA, we developed an assay that detects crossovers generated after DSB formation (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Our assay is based on the DR-GFP gene conversion assay (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Pierce <em>et al.</em> 2001</a>), which has a <em>GFP</em> gene interrupted by an <em>I-Sce</em>I site (<em>SceGFP</em>), and a downstream <em>GFP</em> fragment (<em>iGFP</em>) that serves as a repair template. We added an <em>mCherry</em> gene with an intron and placed the <em>iGFP</em> fragment (modified to contain the entire 3′ end of <em>GFP</em>) inside the intron. This is in reverse orientation relative to <em>SceGFP</em>, to prevent the possibility of the single-strand annealing pathway. In cells transfected with this construct, mCherry is expressed but GFP is not. After DSB induction with <em>I-Sce</em>I, gene conversion of the <em>I-Sce</em>I site restores GFP expression. However, if gene conversion is accompanied by a crossover, the region between the <em>GFP</em> fragments becomes inverted, resulting in loss of mCherry expression.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An assay to detect crossovers generated during DSB repair. (A) Schematic of the assay. The diagram at the top (i) has the SceGFP gene in solid green, with the yellow box indicating insertion of an I-SceI site. The modified iGFP fragment (with the entire 3&#x2032; end of the gene) is indicated in hatched green. The mCherry gene is shown in magenta, with the intron represented as a dotted line. (ii) Representation of the construct as double-stranded DNA. (iii) After DSB introduction and resection, resected ends can pair with the homologous iGFP template. The example here shows intrachromatid repair. (iv) Gene conversion without a crossover results in replacement of the I-SceI site with GFP sequences, generating a functional GFP gene. mCherry expression is unaffected. (v) If a crossover is generated, the central region is inverted and mCherry expression is lost. Repair with the sister chromatid (interchromatid repair) may also be possible. In that case, a crossover results in a dicentric chromatid and an acentric chromatid; these products will not be recovered, but noncrossover gene conversion will be recovered. (B) A field of cells after I-SceI expression, showing the green (left) and red (middle, magenta) channels, and a merged image (right). Green arrowhead marks a cell that gained GFP expression but lost mCherry expression (a crossover); white arrow indicates a cell that gained GFP expression and retained mCherry expression (noncrossover gene conversion); magenta arrow indicates a cell in which GFP was not converted. (C) Flow cytometry showing gating for GFP and mCherry fluorescence. (D) Increased noncrossover and crossover gene conversion after knocking down BLM. Bars indicate percentage of cells expressing GFP and mCherry (noncrossover gene conversion; upper right quadrant of flow cytometry output) or GFP only (crossovers; lower right quadrant). Error bars are SD based on three different wells per treatment. Raw data are in Table S3 in File S1. *** P = 0.0010, **** P &lt; 0.0001 based on unpaired t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1697172406" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An assay to detect crossovers generated during DSB repair. (A) Schematic of the assay. The diagram at the top (i) has the &lt;em&gt;SceGFP&lt;/em&gt; gene in solid green, with the yellow box indicating insertion of an &lt;em&gt;I-Sce&lt;/em&gt;I site. The modified &lt;em&gt;iGFP&lt;/em&gt; fragment (with the entire 3&#x2032; end of the gene) is indicated in hatched green. The &lt;em&gt;mCherry&lt;/em&gt; gene is shown in magenta, with the intron represented as a dotted line. (ii) Representation of the construct as double-stranded DNA. (iii) After DSB introduction and resection, resected ends can pair with the homologous &lt;em&gt;iGFP&lt;/em&gt; template. The example here shows intrachromatid repair. (iv) Gene conversion without a crossover results in replacement of the &lt;em&gt;I-Sce&lt;/em&gt;I site with &lt;em&gt;GFP&lt;/em&gt; sequences, generating a functional &lt;em&gt;GFP&lt;/em&gt; gene. mCherry expression is unaffected. (v) If a crossover is generated, the central region is inverted and mCherry expression is lost. Repair with the sister chromatid (interchromatid repair) may also be possible. In that case, a crossover results in a dicentric chromatid and an acentric chromatid; these products will not be recovered, but noncrossover gene conversion will be recovered. (B) A field of cells after &lt;em&gt;I-Sce&lt;/em&gt;I expression, showing the green (left) and red (middle, magenta) channels, and a merged image (right). Green arrowhead marks a cell that gained GFP expression but lost mCherry expression (a crossover); white arrow indicates a cell that gained GFP expression and retained mCherry expression (noncrossover gene conversion); magenta arrow indicates a cell in which GFP was not converted. (C) Flow cytometry showing gating for GFP and mCherry fluorescence. (D) Increased noncrossover and crossover gene conversion after knocking down BLM. Bars indicate percentage of cells expressing GFP and mCherry (noncrossover gene conversion; upper right quadrant of flow cytometry output) or GFP only (crossovers; lower right quadrant). Error bars are SD based on three different wells per treatment. Raw data are in Table S3 in File S1. *** &lt;em&gt;P&lt;/em&gt; = 0.0010, **** &lt;em&gt;P&lt;/em&gt; &lt; 0.0001 based on unpaired &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1191/F4.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/4/1191/F4.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484016" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">An assay to detect crossovers generated during DSB repair. (A) Schematic of the assay. The diagram at the top (i) has the <em>SceGFP</em> gene in solid green, with the yellow box indicating insertion of an <em>I-Sce</em>I site. The modified <em>iGFP</em> fragment (with the entire 3′ end of the gene) is indicated in hatched green. The <em>mCherry</em> gene is shown in magenta, with the intron represented as a dotted line. (ii) Representation of the construct as double-stranded DNA. (iii) After DSB introduction and resection, resected ends can pair with the homologous <em>iGFP</em> template. The example here shows intrachromatid repair. (iv) Gene conversion without a crossover results in replacement of the <em>I-Sce</em>I site with <em>GFP</em> sequences, generating a functional <em>GFP</em> gene. mCherry expression is unaffected. (v) If a crossover is generated, the central region is inverted and mCherry expression is lost. Repair with the sister chromatid (interchromatid repair) may also be possible. In that case, a crossover results in a dicentric chromatid and an acentric chromatid; these products will not be recovered, but noncrossover gene conversion will be recovered. (B) A field of cells after <em>I-Sce</em>I expression, showing the green (left) and red (middle, magenta) channels, and a merged image (right). Green arrowhead marks a cell that gained GFP expression but lost mCherry expression (a crossover); white arrow indicates a cell that gained GFP expression and retained mCherry expression (noncrossover gene conversion); magenta arrow indicates a cell in which GFP was not converted. (C) Flow cytometry showing gating for GFP and mCherry fluorescence. (D) Increased noncrossover and crossover gene conversion after knocking down BLM. Bars indicate percentage of cells expressing GFP and mCherry (noncrossover gene conversion; upper right quadrant of flow cytometry output) or GFP only (crossovers; lower right quadrant). Error bars are SD based on three different wells per treatment. Raw data are in Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>. *** <em>P</em> = 0.0010, **** <em>P</em> &lt; 0.0001 based on unpaired <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">We introduced this construct into both U2OS and HeLa cells and selected for stably-transfected lines that expressed mCherry. We tested several U2OS lines for response to <em>I-Sce</em>I introduction. We selected a line that consistently yielded cells that were positive for both GFP and mCherry (noncrossover gene conversions) and cells that were positive for GFP only (crossovers). In our standard protocol (see <em>Materials and Methods</em>), treatment of NT siRNA control cells with <em>I-Sce</em>I resulted in 15.6% of cells gaining GFP expression (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>); 3.8% of these (0.6% of all cells) had lost mCherry expression. This is likely to be an underestimate of the crossover frequency because only intrachromatid crossovers can be recovered, as interchromatid crossovers result in dicentric and acentric products (see <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> legend).</p></div><div id="sec-14" class="subsection"><h3>Knocking down BLM or RTEL1 elevates SDSA frequency</h3><p id="p-30">In the model shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>, SDSA diverges from dHJ pathways when a helicase dissociates the nascent strand from the template. Several helicases have been suggested to perform this step. <em>Drosophila</em> gap repair assays found roles for Blm and Fancm helicases in SDSA (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kuo <em>et al.</em> 2014</a>). The <em>Arabidopsis</em> orthologs of these enzymes promote noncrossover recombination in meiosis, possibly by SDSA (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crismani <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Séguéla-Arnaud <em>et al.</em> 2015</a>). Sgs1, the yeast ortholog of Blm, is required for noncrossover recombination in budding yeast meiosis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">De Muyt <em>et al.</em> 2012</a>), and Sgs1 and Mph1 (the orthologs of Fancm) have been implicated in SDSA in vegetative cells (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mitchel <em>et al.</em> 2013</a>). In <em>Schizosaccharomyces pombe</em>, Fml1, the ortholog of Fancm/Mph1, was suggested to promote SDSA in DSB repair during replication and in meiosis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sun <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lorenz <em>et al.</em> 2012</a>). Yet another helicase, RTEL-1, was hypothesized to disrupt D-loops in <em>Caenorhabditis elegans</em> meiosis (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barber <em>et al.</em> 2008</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Youds <em>et al.</em> 2010</a>). In human cells, RECQ5 is proposed to promote SDSA (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Paliwal <em>et al.</em> 2014</a>). Aside from the experiments in <em>Drosophila</em> and budding yeast, none of the assays performed could distinguish between SDSA and other pathways. Thus, we used our assay to ask whether the orthologs of any of these or related helicases affect SDSA in human cells.</p><p id="p-31">We did not detect any change in the frequency of red-fluorescing cells after knocking down FANCM, RECQ5, WRN, or FBXO18 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>). Knockdown of BLM or RTEL1 significantly altered the frequency of red-fluorescing cells; however, instead of decreasing SDSA as expected, both knockdowns resulted in an increase in red-fluorescing cells (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). BLM has been shown to have several functions in HDR pathways, including in DSB end resection in a pathway redundant with <em>Exo</em>I (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhu <em>et al.</em> 2008</a>) and dHJ dissolution (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wu and Hickson 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wu <em>et al.</em> 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Singh <em>et al.</em> 2008</a>). Likewise, RTEL1, which was initially identified as a telomere length regulator and is responsible for T-loop disruption (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ding <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sarek <em>et al.</em> 2015</a>), can also unwind DNA secondary structures and promote replication fork progression (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barber <em>et al.</em> 2008</a>). To begin to assess how knocking down BLM and RTEL1 might increase SDSA, we knocked down each in combination with BRCA2. In simultaneous knockdowns, red-fluorescing cells were decreased from the frequency observed of BLM and RTEL1 single knockdowns (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2C</a>). The magnitude of the decrease was similar to that of the BRCA2 single knockdown relative to the negative control (44% decrease for BRCA2 relative to NT, 47% for BRCA2 + BLM relative to BLM alone, and 54% for BRCA2 + RTEL1 relative to RTEL1 alone), indicating that BLM and RTEL1 impact SDSA through functions after strand exchange into a homologous template.</p><p id="p-32">We also tested the effects of knocking down BLM in the crossover assay. Transfection with siRNA to knockdown BLM resulted in 25% of cells gaining GFP expression (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>). This is a 60% increase compared to the NT control, similar to the average increase of 59% in the SDSA assay. Crossovers were elevated, with 6.4% of the GFP-positive cells (1.6% of all cells) having also lost mCherry expression. This is consistent with studies showing elevated spontaneous crossing over in cells from Bloom syndrome patients (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">German <em>et al.</em> 1977</a>).</p></div><div id="sec-15" class="subsection"><h3>Knocking down BLM or RTEL1 alters repair outcomes</h3><p id="p-33">To further develop our SDSA assay and gain additional insights into the effects of knocking down BLM and RTEL, we determined the structures of repair events produced in knockdown cells. We analyzed 55 clones derived from single red-fluorescing cells, including 23 from the NT control, 21 from BLM knockdown, 10 from FANCM knockdown, and one from RTEL1 knockdown. All but one had the structure expected of SDSA (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2B</a>). The remaining clone, which came from NT siRNA treatment, had lost <em>neo</em> and the template spacer, and therefore may have arisen from SDSA followed by DSBR with crossover resolution (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1D</a>). These results support our conclusion that cells with restored <em>mCherry</em> utilized SDSA to repair the DSB, perhaps occasionally coupled with use of DSBR.</p><p id="p-34">We also analyzed cells that failed to produce mCherry. In the NT control, all 45 lines examined appeared to be identical to the initial construct (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). We did not measure cleavage efficiency in our assay, but we titrated adenovirus infection to the highest dose that did not cause detectable cell lethality. Delivery of <em>I-Sce</em>I by adenovirus infection of HEK293 cells resulted in 85% of sites being cut (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Anglana and Bacchetti 1999</a>), so it is likely that most or all of the cells with intact <em>I-Sce</em>I sites are likely to result from cleavage followed by precise NHEJ using the 4 nt complementary overhangs left by <em>I-Sce</em>I.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structures of repair events in cells not expressing mCherry. Single cells not exhibiting red fluorescence were grown and then analyzed. (A) The starting construct, and the structure found in the majority of clones in each siRNA treatment group. (B) Copying of the entire template into the upstream mCherry I-SceI site. This might occur through long-tract SDSA (one synthesis cycle or several) or a dHJ intermediate (as in Figure 2D, center). (C) Copying of the entire template with loss of intervening sequences. This is proposed to occur when a dHJ intermediate is resolved in the crossover orientation, as in Figure 2D, bottom. (D) Copying of part of the template into the upstream mCherry site, as predicted from initiation of SDSA but completion of repair by end joining, as in Figure 2E. (E) Retention of the I-SceI site with loss of the neo and ori sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1697172406" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structures of repair events in cells not expressing mCherry. Single cells not exhibiting red fluorescence were grown and then analyzed. (A) The starting construct, and the structure found in the majority of clones in each siRNA treatment group. (B) Copying of the entire template into the upstream &lt;em&gt;mCherry I-Sce&lt;/em&gt;I site. This might occur through long-tract SDSA (one synthesis cycle or several) or a dHJ intermediate (as in Figure 2D, center). (C) Copying of the entire template with loss of intervening sequences. This is proposed to occur when a dHJ intermediate is resolved in the crossover orientation, as in Figure 2D, bottom. (D) Copying of part of the template into the upstream &lt;em&gt;mCherry&lt;/em&gt; site, as predicted from initiation of SDSA but completion of repair by end joining, as in Figure 2E. (E) Retention of the &lt;em&gt;I-Sce&lt;/em&gt;I site with loss of the &lt;em&gt;neo&lt;/em&gt; and &lt;em&gt;ori&lt;/em&gt; sequences.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1191/F5.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/4/1191/F5.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1191/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Structures of repair events in cells not expressing mCherry. Single cells not exhibiting red fluorescence were grown and then analyzed. (A) The starting construct, and the structure found in the majority of clones in each siRNA treatment group. (B) Copying of the entire template into the upstream <em>mCherry I-Sce</em>I site. This might occur through long-tract SDSA (one synthesis cycle or several) or a dHJ intermediate (as in <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2D</a>, center). (C) Copying of the entire template with loss of intervening sequences. This is proposed to occur when a dHJ intermediate is resolved in the crossover orientation, as in <a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2D</a>, bottom. (D) Copying of part of the template into the upstream <em>mCherry</em> site, as predicted from initiation of SDSA but completion of repair by end joining, as in <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2E</a>. (E) Retention of the <em>I-Sce</em>I site with loss of the <em>neo</em> and <em>ori</em> sequences.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The majority of clones from BLM or RTEL1 knockdown cells that did not produce mCherry also had an intact <em>I-Sce</em>I site; however, structures indicating other repair processes were observed in 11 out of 34 of these clones from BLM knockdown (<em>P</em> &lt; 0.0001 compared to NT) and 24 out of 133 of these clones from RTEL1 knockdown cells (<em>P</em> = 0.0007). In four of the BLM knockdown clones and 14 of the RTEL1 knockdown clones, the entire 3-kbp spacer sequence was copied from the repair template into the upstream <em>mCherry</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, B and C</a>). This extensive repair synthesis might occur through multiple cycles of strand exchange, as is believed to occur in <em>Drosophila</em> gap repair by SDSA (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">McVey <em>et al.</em> 2004</a>), or through a single, continuous synthesis event. Among the 17 examples in which the entire spacer was copied, one from each knockdown sample had lost <em>neo</em>, a structure that is most consistent with a dHJ being resolved to give a crossover (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2D</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). The other 15 may have arisen by long-tract SDSA or by dissolution or noncrossover resolution of a dHJ (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2D</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>).</p><p id="p-37">There were additional repair events from knockdown cells that also had evidence of long-tract synthesis. Three events from BLM knockdown and seven from RTEL1 knockdown had a subset of the spacer copied into the upstream <em>mCherry</em> (<a id="xref-fig-2-18" class="xref-fig" href="#F2">Figure 2E</a> and <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>). These are most likely hybrid repair that involved long-tract synthesis followed by end joining. There was one event from the BLM knockdown that had an intact <em>I-Sce</em>I site but had lost <em>neo</em> (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5E</a>). The source of this event and whether it occurred following <em>I-Sce</em>I cleavage is unknown.</p></div><div id="sec-16" class="subsection"><h3>Roles of BLM and RTEL1 in SDSA</h3><p id="p-38">It was surprising that none of the helicase knockdowns led to decreased SDSA, since orthologs of all of these have been suggested to promote SDSA. One possibility is that there is redundancy among two or more of these proteins. This possibility can be addressed through simultaneous knockdowns or use of doubly mutant cells. In the case of BLM and RTEL1, knockdown led to dramatic increases in HDR, both in the frequency of red-fluorescing cells (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a>) and in the fraction of nonfluorescing cells that had evidence for HDR (zero out of 45 for NT compared to 11 out of 34 for BLM knockdown and 24 out of 133 for RTEL1 knockdown). This is consistent with a reported increase in gene conversion in the DR-GFP assay after siRNA knockdown of BLM (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Paliwal <em>et al.</em> 2014</a>).</p><p id="p-39">Molecular analysis of repair events provides some insights into sources of increased HDR. Several repair products from both BLM and RTEL1 knockdowns had extensive synthesis (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). Extensive synthesis has also been reported in another assay when the HDR proteins BRCA1 or CtIP were knocked down, and it was suggested that these proteins prevent long-tract HDR (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chandramouly <em>et al.</em> 2013</a>). BLM and RTEL1 might prevent long-tract HDR through their D-loop disruption activities (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Van Brabant <em>et al.</em> 2000</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Barber <em>et al.</em> 2008</a>). Our assay requires enough synthesis from both ends of the DSB to allow annealing within the 350-bp repeat (<a id="xref-fig-2-19" class="xref-fig" href="#F2">Figure 2B</a>), or sequential strand invasion and synthesis (<a id="xref-fig-2-20" class="xref-fig" href="#F2">Figure 2C</a>). If BLM and RTEL1 normally disrupt D-loops after less than a few hundred nucleotides of synthesis, the nascent strands would not be able to anneal at the 350-bp repeat and we would see reduced SDSA. In this scenario, we might expect the NT sample to have some hybrid repair events with short synthesis tracts (as in <a id="xref-fig-2-21" class="xref-fig" href="#F2">Figure 2E</a> but with only part of the 350-bp repeat on one or both ends and none of the spacer sequence). We did not detect any such products, but it is possible our sample size was not large enough to observe these infrequent events.</p><p id="p-40">Some repair products in knockdown cells had synthesis that spanned the entire 3-kbp spacer (<em>e.g.</em>, <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5B</a>). These could generate dHJ intermediates that would then be resolved or dissolved. In the case of RTEL1 knockdown, only one out of 13 events in which the entire spacer was copied had the structure expected of dHJ crossover resolution (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5C</a>). If crossover and noncrossover resolution occur at equal frequencies, then either SDSA or dHJ dissolution were likely responsible for most of these events. The BTR complex, which contains the BLM helicase, is believed to be the major or sole dissolvase (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Wu and Hickson 2003</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Seki <em>et al.</em> 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dayani <em>et al.</em> 2011</a>). Although there was also only a single event in the BLM knockdown suggestive of dHJ crossover resolution (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5C</a>), there were only three other events in which the entire spacer was copied; it is possible that all three of these came from dHJ resolution that had a noncrossover outcome.</p><p id="p-41">In our assay, SDSA may occur if both ends of the DSB engage with the template. This suggests the possibility that a function of BLM and RTEL1 is to ensure that only one end engages with the repair template. Among the 10 repair events in which only part of the spacer was copied into the upstream <em>mCherry</em> (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5D</a>), all of them appeared to have synthesis from the left end only. This is in contrast to the <em>Drosophila P</em>{<em>w<sup>a</sup></em>} excision assay, where most repair events have several kilobase pairs of synthesis from both ends of the break (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2003</a>). The disparity could arise from the difference between organisms or tissues, different structures of the DSB ends (4 nt 3′ overhangs for <em>I-Sce</em>I; 17 nt 3′ overhangs for <em>P</em> element excision), or distance between the sequence homologous to the left side of the DSB and the sequence homologous to the right side (∼3.5 kbp for this assay but &gt;14 kbp for <em>P</em>{<em>w<sup>a</sup></em>}).</p><p id="p-42">The cases of partial copying of the spacer most likely derive from hybrid repair in which the initial steps of SDSA are executed but dissociation of the nascent strand does not reveal complementary sequences for annealing, so repair is instead completed by DNA polymerase Θ-mediated end joining (also called microhomology-mediated end joining) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chan <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wyatt <em>et al.</em> 2016</a>). As in SDSA, these D-loops must have been disassembled by another helicase than the one knocked down, or by residual helicase present after the knockdown.</p><p id="p-43">Knocking down BLM did lead to elevated crossovers in the crossover assay (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). This result was expected, based on phenotypes like elevated sister chromatid exchange (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">German <em>et al.</em> 1977</a>). However, there was also an overall increase in HDR, as noncrossovers were also elevated. Thus, knocking down BLM resulted in elevated HDR in the DR-GFP assay (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Paliwal <em>et al.</em> 2014</a>), our SDSA assay, and our crossover assay. This might be expected if knockdown affects the cell cycle profile, such that more cells are in S or G2 phases and therefore more likely to repair by HDR instead of NHEJ. We conducted cell cycle profiling of cells in which BLM was knocked down, but did not detect any significant differences in the cell cycle profile compared to the NT control (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-44">The causes of increased HDR when BLM or RTEL1 is knocked down remain unknown. This is an interesting area for future investigation, as understanding this unexpected effect will no doubt reveal important functions of these proteins.</p></div><div id="sec-17" class="subsection"><h3>Concluding remarks</h3><p id="p-45">We have demonstrated that our assays efficiently detect DSB repair by SDSA or that lead to crossovers, and that these assays can be used to study the effects of knocking down or knocking out different repair genes. Strengths of the assays include the ease of identifying the SDSA or crossover outcomes and the ability to investigate other types of repair based on structures of repair products. We did not determine whether the lines we used had only a single copy of the assay construct integrated (see <em>Materials and Methods</em>), but analyses of cells exposed to <em>I-Sce</em>I strongly argue that there was only one insertion location in both cases. In the SDSA assay, an average of 2% of cells acquired red fluorescence in any given experiment. If there were insertions at two different sites repairing independently, we would expect that in the vast majority of cases SDSA would occur in only one of the two insertions. PCR across the <em>I-Sce</em>I site would give two bands, one corresponding to the original construct and a larger band resulting from replacement of the 350-bp fragment. In opposition to this expectation, 55 out of 55 red-fluorescing clones examined had only the larger band. Similarly, crossovers in the crossover assay result in loss of mCherry. If there were several integrations every site would have to lose mCherry simultaneously to be scored as a crossover. It remains possible that one or both constructs integrated in a tandem array in the lines we used. If this happened, then it is likely that all <em>I-Sce</em>I sites were cut, leaving some extrachromosomal fragments but only a single chromosomal repair template. It is unknown what effect the extrachromosomal fragments would have on repair of the chromosomal DSB.</p><p id="p-46">These assays can be modified to tailor their use in addressing specific questions. With the development of CRISPR/Cas9 genome editing (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cong <em>et al.</em> 2013</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Mali <em>et al.</em> 2013</a>), gene knockouts could be done instead of knockdowns, at least for genes that are not essential in the timeframe of these assays. It may be advantageous to use other cell lines for this approach, as U2OS cells have more than two copies of many genes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Forbes <em>et al.</em> 2015</a>). For some questions, it would be informative to incorporate SNP markers into the template so that gene conversion tract properties could be measured at a higher resolution than reported here. Differences between the two 350-bp repeats in the SDSA assay could be used to identify cases of template switching or two-ended invasions. We did not attempt to develop high-throughput sequencing of repair products, but amplification of the entire SDSA module with single-molecule tagging would make it possible to sequence a large number of independent repair events simultaneously. Finally, various distance parameters, such as changes to the amount of synthesis required to reach the repeats (they are immediately adjacent to the DSB in our assay, but &gt;5 kbp into the gap in the <em>P</em>{<em>w<sup>a</sup></em>} assay) or length of the repeats (350 bp in this assay, 275 bp in <em>P</em>{<em>w<sup>a</sup></em>}) could provide insight into the frequency of template switching, the length of a typical synthesis tract, and the ability to repair larger gaps.</p><p id="p-47">Elucidating details of SDSA and crossover repair is important for understanding DSB repair in general, but will also prove vital in future optimization of CRISPR/Cas9 gene replacement or integration strategies that have been hypothesized to occur through SDSA (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Byrne <em>et al.</em> 2015</a>). We believe both assays we describe can be useful in achieving these goals.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank Jan LaRocque for cells and teaching us the DR-GFP assay, Mira Pronobis for U2OS and HeLa cells, and Yangze Gao and Cyrus Vaziri for the <em>I-Sce</em>I adenovirus. We thank Diana Chong for assistance in acquiring confocal images of mCherry-positive cells. We thank Lydia Morris, Talia Hatkevich, and Juan Carvajal Garcia for comments on the manuscript. This work was supported by grants from the National Institutes of General Medical Sciences to J.S. under award numbers 5R01GM099890 and 1R35GM118127. The University of North Carolina (UNC) Flow Cytometry Core Facility is supported in part by P30 CA016086 Cancer Center Core Support Grant to the UNC Lineberger Comprehensive Cancer Center.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037390/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-50"><em>Communicating editor: K. S. McKim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-13"><span class="received-label">Received </span>November 13, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-02-03"><span class="accepted-label">Accepted </span>February 3, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Zapotoczny and Sekelsky</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.1" data-doi="10.1126/science.1077198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekelsky</span>  <span class="cit-name-given-names">J.J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title"><em>Drosophila</em> BLM in double-strand break repair by synthesis-dependent strand annealing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D299%26rft.issue%253D5604%26rft.spage%253D265%26rft.epage%253D267%26rft.atitle%253DDrosophila%2BBLM%2Bin%2BDouble-Strand%2BBreak%2BRepair%2Bby%2BSynthesis-Dependent%2BStrand%2BAnnealing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1077198%26rft_id%253Dinfo%253Apmid%252F12522255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTkvNTYwNC8yNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.2" data-doi="10.1016/S0092-8674(01)00416-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Differential timing and control of noncrossover and crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D57%26rft.atitle%253DDifferential%2Btiming%2Band%2Bcontrol%2Bof%2Bnoncrossover%2Band%2Bcrossover%2Brecombination%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900416-0%26rft_id%253Dinfo%253Apmid%252F11461701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00416-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461701&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.3" data-doi="10.1002/bies.201000087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekelsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Meiotic <em>vs.</em> mitotic recombination: two different routes for double-strand break repair: the different functions of meiotic <em>vs.</em> mitotic DSB repair are reflected in different pathway usage and different outcomes.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1058</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DAndersen%26rft.auinit1%253DS.%2BL.%26rft.volume%253D32%26rft.issue%253D12%26rft.spage%253D1058%26rft.epage%253D1066%26rft.atitle%253DMeiotic%2Bversus%2Bmitotic%2Brecombination%253A%2Btwo%2Bdifferent%2Broutes%2Bfor%2Bdouble-strand%2Bbreak%2Brepair%253A%2Bthe%2Bdifferent%2Bfunctions%2Bof%2Bmeiotic%2Bversus%2Bmitotic%2BDSB%2Brepair%2Bare%2Breflected%2Bin%2Bdifferent%2Bpathway%2Busage%2Band%2Bdifferent%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201000087%26rft_id%253Dinfo%253Apmid%252F20967781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201000087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20967781&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284701000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.4" data-doi="10.1093/nar/27.21.4276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anglana</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacchetti</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Construction of a recombinant adenovirus for efficient delivery of the <em>I-Sce</em>I yeast endonuclease to human cells and its application in the in vivo cleavage of chromosomes to expose new potential telomeres.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">4276</span>–<span class="cit-lpage">4281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.21.4276%26rft_id%253Dinfo%253Apmid%252F10518621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8yMS80Mjc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.5" data-doi="10.1016/j.cell.2008.08.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Youds</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcilwraith</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’neil</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RTEL1 maintains genomic stability by suppressing homologous recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBarber%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D135%26rft.issue%253D2%26rft.spage%253D261%26rft.epage%253D271%26rft.atitle%253DRTEL1%2Bmaintains%2Bgenomic%2Bstability%2Bby%2Bsuppressing%2Bhomologous%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.08.016%26rft_id%253Dinfo%253Apmid%252F18957201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.08.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18957201&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260130400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.6" data-doi="10.1093/nar/gku1246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-kilobase homozygous targeted gene replacement in human induced pluripotent stem cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">e21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1246%26rft_id%253Dinfo%253Apmid%252F25414332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQzLzMvZTIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.7" data-doi="10.1371/journal.pgen.1001005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dual roles for DNA polymerase theta in alternative end-joining repair of double-strand breaks in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChan%26rft.auinit1%253DS.%2BH.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De1001005%26rft.epage%253De1001005%26rft.atitle%253DDual%2Broles%2Bfor%2BDNA%2Bpolymerase%2Btheta%2Bin%2Balternative%2Bend-joining%2Brepair%2Bof%2Bdouble-strand%2Bbreaks%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001005%26rft_id%253Dinfo%253Apmid%252F20617203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20617203&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.8" data-doi="10.1038/ncomms3404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandramouly</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">BRCA1 and CtIP suppress long-tract gene conversion between sister chromatids.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2404%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3404%26rft_id%253Dinfo%253Apmid%252F23994874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23994874&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.9" data-doi="10.1126/science.1231143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barretto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiplex genome engineering using CRISPR/Cas systems.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1231143%26rft_id%253Dinfo%253Apmid%252F23287718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMS84MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.10" data-doi="10.1126/science.1220381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crismani</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pradillo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FANCM limits meiotic crossovers.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">1588</span>–<span class="cit-lpage">1590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCrismani%26rft.auinit1%253DW.%26rft.volume%253D336%26rft.issue%253D6088%26rft.spage%253D1588%26rft.epage%253D1590%26rft.atitle%253DFANCM%2BLimits%2BMeiotic%2BCrossovers%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1220381%26rft_id%253Dinfo%253Apmid%252F22723424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzYvNjA4OC8xNTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.11" data-doi="10.1371/journal.pgen.1002083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dayani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simchen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Meiotic recombination intermediates are resolved with minimal crossover formation during return-to-growth, an analogue of the mitotic cell cycle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDayani%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1002083%26rft.epage%253De1002083%26rft.atitle%253DMeiotic%2Brecombination%2Bintermediates%2Bare%2Bresolved%2Bwith%2Bminimal%2Bcrossover%2Bformation%2Bduring%2Breturn-to-growth%252C%2Ban%2Banalogue%2Bof%2Bthe%2Bmitotic%2Bcell%2Bcycle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002083%26rft_id%253Dinfo%253Apmid%252F21637791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21637791&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.12" data-doi="10.1016/j.molcel.2012.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Muyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourirajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDe%2BMuyt%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DBLM%2Bhelicase%2Bortholog%2BSgs1%2Bis%2Ba%2Bcentral%2Bregulator%2Bof%2Bmeiotic%2Brecombination%2Bintermediate%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.02.020%26rft_id%253Dinfo%253Apmid%252F22500736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500736&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302838000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.13" data-doi="10.1016/j.cell.2004.05.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gertsenstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selig</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Regulation of murine telomere length by <em>Rtel</em>: an essential gene encoding a helicase-like protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDing%26rft.auinit1%253DH.%26rft.volume%253D117%26rft.issue%253D7%26rft.spage%253D873%26rft.epage%253D886%26rft.atitle%253DRegulation%2Bof%2Bmurine%2Btelomere%2Blength%2Bby%2BRtel%253A%2Ban%2Bessential%2Bgene%2Bencoding%2Ba%2Bhelicase-like%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.05.026%26rft_id%253Dinfo%253Apmid%252F15210109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.05.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15210109&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222300200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.14" data-doi="10.1093/nar/gku1075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forbes</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beare</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunasekaran</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bindal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">COSMIC: exploring the world’s knowledge of somatic mutations in human cancer.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D805</span>–<span class="cit-lpage">D811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1075%26rft_id%253Dinfo%253Apmid%252F25355519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My9EMS9EODA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">German</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schonberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louie</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaganti</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Bloom’s syndrome. IV. Sister-chromatid exchanges in lymphocytes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DGerman%26rft.auinit1%253DJ.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D248%26rft.epage%253D255%26rft.atitle%253DBloom%2527s%2Bsyndrome.%2BIV.%2BSister-chromatid%2Bexchanges%2Bin%2Blymphocytes.%26rft_id%253Dinfo%253Apmid%252F868871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=868871&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DG79400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">A mechanism for gene conversion in fungi.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D78%26rft.spage%253D282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.17" data-doi="10.1534/genetics.114.168096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcmahan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rota</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohl</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekelsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>Drosophila</em> FANCM helicase prevents spontaneous mitotic crossovers generated by the MUS81 and SLX1 nucleases.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.168096%26rft_id%253Dinfo%253Apmid%252F25205745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvOTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurkulos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinberg</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title"><em>P</em> element-mediated <em>in vivo</em> deletion analysis of <em>white-apricot</em>: deletions between direct repeats are strongly favored.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKurkulos%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D1001%26rft.epage%253D1011%26rft.atitle%253DP%2Belement-mediated%2Bin%2Bvivo%2Bdeletion%2Banalysis%2Bof%2Bwhite-apricot%253A%2Bdeletions%2Bbetween%2Bdirect%2Brepeats%2Bare%2Bstrongly%2Bfavored.%26rft_id%253Dinfo%253Apmid%252F8005410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi8zLzEwMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.19" data-doi="10.1126/science.1220111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nandi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinacher</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The fission yeast FANCM ortholog directs non-crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">1585</span>–<span class="cit-lpage">1588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLorenz%26rft.auinit1%253DA.%26rft.volume%253D336%26rft.issue%253D6088%26rft.spage%253D1585%26rft.epage%253D1588%26rft.atitle%253DThe%2BFission%2BYeast%2BFANCM%2BOrtholog%2BDirects%2BNon-Crossover%2BRecombination%2BDuring%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1220111%26rft_id%253Dinfo%253Apmid%252F22723423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzYvNjA4OC8xNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.20" data-doi="10.1126/science.1232033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-guided human genome engineering via Cas9.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1232033%26rft_id%253Dinfo%253Apmid%252F23287722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMS84MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.21" data-doi="10.1534/genetics.103.025411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staeva-Vieira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekelsky</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evidence for multiple cycles of strand invasion during repair of double-strand gaps in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVey%26rft.auinit1%253DM.%26rft.volume%253D167%26rft.issue%253D2%26rft.spage%253D699%26rft.epage%253D705%26rft.atitle%253DEvidence%2Bfor%2BMultiple%2BCycles%2Bof%2BStrand%2BInvasion%2BDuring%2BRepair%2Bof%2BDouble-Strand%2BGaps%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.025411%26rft_id%253Dinfo%253Apmid%252F15238522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzIvNjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.22" data-doi="10.1016/j.molcel.2010.02.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welz-Voegele</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular structures of crossover and noncrossover intermediates during gap repair in yeast: implications for recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMitchel%26rft.auinit1%253DK.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D222%26rft.atitle%253DMolecular%2Bstructures%2Bof%2Bcrossover%2Band%2Bnoncrossover%2Bintermediates%2Bduring%2Bgap%2Brepair%2Bin%2Byeast%253A%2Bimplications%2Bfor%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.02.028%26rft_id%253Dinfo%253Apmid%252F20417600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.02.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20417600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277598100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.23" data-doi="10.1371/journal.pgen.1003340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heteroduplex DNA position defines the roles of the Sgs1, Srs2, and Mph1 helicases in promoting distinct recombination outcomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003340%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003340%26rft_id%253Dinfo%253Apmid%252F23516370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23516370&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.24" data-doi="10.1534/genetics.112.139105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamana</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usui</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Homologous recombination via synthesis-dependent strand annealing in yeast requires the Irc20 and Srs2 DNA helicases.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139105%26rft_id%253Dinfo%253Apmid%252F22367032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkxLzEvNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.25" data-doi="10.1128/MCB.14.3.1613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nassif</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penney</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engels</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gloor</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Efficient copying of nonhomologous sequences from ectopic sites via P-element-induced gap repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1613</span>–<span class="cit-lpage">1625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNassif%26rft.auinit1%253DN%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D1613%26rft.epage%253D1625%26rft.atitle%253DEfficient%2Bcopying%2Bof%2Bnonhomologous%2Bsequences%2Bfrom%2Bectopic%2Bsites%2Bvia%2BP-element-induced%2Bgap%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.3.1613%26rft_id%253Dinfo%253Apmid%252F8114699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE0LzMvMTYxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.26" data-doi="10.1038/ncomms2471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nevozhay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zal</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balazsi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transferring a synthetic gene circuit from yeast to mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D1451%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2471%26rft_id%253Dinfo%253Apmid%252F23385595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23385595&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.27" data-doi="10.1093/nar/gkt1263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paliwal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanagaraj</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturzenegger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdova</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janscak</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Human RECQ5 helicase promotes repair of DNA double-strand breaks by synthesis-dependent strand annealing.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2380</span>–<span class="cit-lpage">2390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1263%26rft_id%253Dinfo%253Apmid%252F24319145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzQvMjM4MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.28" data-doi="10.1101/gad.13.20.2633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">XRCC3 promotes homology-directed repair of DNA damage in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPierce%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D13%26rft.issue%253D20%26rft.spage%253D2633%26rft.epage%253D2638%26rft.atitle%253DXRCC3%2Bpromotes%2Bhomology-directed%2Brepair%2Bof%2BDNA%2Bdamage%2Bin%2Bmammalian%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.13.20.2633%26rft_id%253Dinfo%253Apmid%252F10541549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEzLzIwLzI2MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.29" data-doi="10.1101/gad.946401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ku DNA end-binding protein modulates homologous repair of double-strand breaks in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3237</span>–<span class="cit-lpage">3242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPierce%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D15%26rft.issue%253D24%26rft.spage%253D3237%26rft.epage%253D3242%26rft.atitle%253DKu%2BDNA%2Bend-binding%2Bprotein%2Bmodulates%2Bhomologous%2Brepair%2Bof%2Bdouble-strand%2Bbreaks%2Bin%2Bmammalian%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.946401%26rft_id%253Dinfo%253Apmid%252F11751629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzI0LzMyMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.30" data-doi="10.1016/j.molcel.2014.12.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarek</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vannier</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrini</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">TRF2 recruits RTEL1 to telomeres in S phase to promote T-Loop unwinding.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D57%26rft.spage%253D622%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.12.024%26rft_id%253Dinfo%253Apmid%252F25620558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.12.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25620558&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.31" data-doi="10.1073/pnas.1423107112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Séguéla-Arnaud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crismani</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larcheveque</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multiple mechanisms limit meiotic crossovers: TOP3α and two BLM homologs antagonize crossovers in parallel to FANCM.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">4713</span>–<span class="cit-lpage">4718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1423107112%26rft_id%253Dinfo%253Apmid%252F25825745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE1LzQ3MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.32" data-doi="10.1128/MCB.00702-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tada</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bloom helicase and DNA topoisomerase IIIα are involved in the dissolution of sister chromatids.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">6299</span>–<span class="cit-lpage">6307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSeki%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D16%26rft.spage%253D6299%26rft.epage%253D6307%26rft.atitle%253DBloom%2BHelicase%2Band%2BDNA%2BTopoisomerase%2BIII%257Balpha%257D%2BAre%2BInvolved%2Bin%2Bthe%2BDissolution%2Bof%2BSister%2BChromatids%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00702-06%26rft_id%253Dinfo%253Apmid%252F16880537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xNi82Mjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.33" data-doi="10.1038/386804a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharan</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morimatsu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Embryonic lethality and radiation hypersensitivity mediated by Rad51 in mice lacking Brca2.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">386</span>: <span class="cit-fpage">804</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSharan%26rft.auinit1%253DS.%2BK.%26rft.volume%253D386%26rft.issue%253D6627%26rft.spage%253D804%26rft.epage%253D810%26rft.atitle%253DEmbryonic%2Blethality%2Band%2Bradiation%2Bhypersensitivity%2Bmediated%2Bby%2BRad51%2Bin%2Bmice%2Blacking%2BBrca2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F386804a0%26rft_id%253Dinfo%253Apmid%252F9126738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/386804a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9126738&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WV70600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.34" data-doi="10.1101/gad.1725108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busygina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raynard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">BLAP18/RMI2, a novel OB-fold-containing protein, is an essential component of the Bloom helicase-double Holliday junction dissolvasome.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2856</span>–<span class="cit-lpage">2868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSingh%26rft.auinit1%253DT.%2BR.%26rft.volume%253D22%26rft.issue%253D20%26rft.spage%253D2856%26rft.epage%253D2868%26rft.atitle%253DBLAP18%252FRMI2%252C%2Ba%2Bnovel%2BOB-fold-containing%2Bprotein%252C%2Bis%2Ban%2Bessential%2Bcomponent%2Bof%2Bthe%2BBloom%2Bhelicase-double%2BHolliday%2Bjunction%2Bdissolvasome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1725108%26rft_id%253Dinfo%253Apmid%252F18923083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIyLzIwLzI4NTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.35" data-doi="10.1016/j.molcel.2008.08.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nandi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakovleska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The FANCM ortholog Fml1 promotes recombination at stalled replication forks and limits crossing over during DNA double-strand break repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSun%26rft.auinit1%253DW.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D118%26rft.epage%253D128%26rft.atitle%253DThe%2BFANCM%2Bortholog%2BFml1%2Bpromotes%2Brecombination%2Bat%2Bstalled%2Breplication%2Bforks%2Band%2Blimits%2Bcrossing%2Bover%2Bduring%2BDNA%2Bdouble-strand%2Bbreak%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.08.024%26rft_id%253Dinfo%253Apmid%252F18851838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.08.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18851838&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260240200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.36" data-doi="10.1016/0092-8674(83)90331-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr-Weaver</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The double-strand-break repair model for recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSzostak%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D35%26rft.atitle%253DThe%2Bdouble-strand-break%2Brepair%2Bmodel%2Bfor%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252883%252990331-8%26rft_id%253Dinfo%253Apmid%252F6380756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(83)90331-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6380756&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QS06800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thaler</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Tests of the double-strand-break repair model for red-mediated recombination of phage Λ and plasmid Λdv.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThaler%26rft.auinit1%253DD.%2BS.%26rft.volume%253D116%26rft.issue%253D4%26rft.spage%253D501%26rft.epage%253D511%26rft.atitle%253DTests%2Bof%2Bthe%2BDouble-Strand-Break%2BRepair%2BModel%2Bfor%2BRed-Mediated%2BRecombination%2Bof%2BPhage%2B%257BLambda%257D%2Band%2BPlasmid%2B%257BLambda%257Ddv%26rft_id%253Dinfo%253Apmid%252F2957271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE2LzQvNTAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.38" data-doi="10.1021/bi0018640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Brabant</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">German</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Binding and melting of D-loops by the Bloom syndrome helicase.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">14617</span>–<span class="cit-lpage">14625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D39%26rft.spage%253D14617%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi0018640%26rft_id%253Dinfo%253Apmid%252F11087418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi0018640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11087418&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165602400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.39" data-doi="10.1101/gad.280545.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Noncanonical views of homology-directed DNA repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1138</span>–<span class="cit-lpage">1154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DVerma%26rft.auinit1%253DP.%26rft.volume%253D30%26rft.issue%253D10%26rft.spage%253D1138%26rft.epage%253D1154%26rft.atitle%253DNoncanonical%2Bviews%2Bof%2Bhomology-directed%2BDNA%2Brepair%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.280545.116%26rft_id%253Dinfo%253Apmid%252F27222516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjMwLzEwLzExMzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.40" data-doi="10.1038/nature02253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickson</span>  <span class="cit-name-given-names">I. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Bloom’s syndrome helicase suppresses crossing over during homologous recombination.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>: <span class="cit-fpage">870</span>–<span class="cit-lpage">874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWu%26rft.auinit1%253DL.%26rft.volume%253D426%26rft.issue%253D6968%26rft.spage%253D870%26rft.epage%253D874%26rft.atitle%253DThe%2BBloom%2527s%2Bsyndrome%2Bhelicase%2Bsuppresses%2Bcrossing%2Bover%2Bduring%2Bhomologous%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02253%26rft_id%253Dinfo%253Apmid%252F14685245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14685245&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187342000066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.41" data-doi="10.1073/pnas.0508295103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachrati</span>  <span class="cit-name-given-names">C. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">BLAP75/RMI1 promotes the BLM-dependent dissolution of homologous recombination intermediates.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">4068</span>–<span class="cit-lpage">4073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508295103%26rft_id%253Dinfo%253Apmid%252F16537486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzExLzQwNjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wyatt</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conlin</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yousefzadeh</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Essential roles for polymerase theta-mediated end joining in the repair of chromosome breaks.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D63%26rft.spage%253D662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.43" data-doi="10.1074/jbc.M112.352302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tompkins</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Her</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Assessment of anti-recombination and double-strand break-induced gene conversion in human cells by a chromosomal reporter.</span> <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">29543</span>–<span class="cit-lpage">29553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.352302%26rft_id%253Dinfo%253Apmid%252F22773873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMzUvMjk1NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.44" data-doi="10.1126/science.1183112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youds</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mets</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcilwraith</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RTEL-1 enforces meiotic crossover interference and homeostasis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">1254</span>–<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYouds%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D327%26rft.issue%253D5970%26rft.spage%253D1254%26rft.epage%253D1258%26rft.atitle%253DRTEL-1%2BEnforces%2BMeiotic%2BCrossover%2BInterference%2Band%2BHomeostasis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183112%26rft_id%253Dinfo%253Apmid%252F20203049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjcvNTk3MC8xMjU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzExOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1191.45" data-doi="10.1016/j.cell.2008.08.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ira</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sgs1 helicase and two nucleases Dna2 and Exo1 resect DNA double-strand break ends.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">981</span>–<span class="cit-lpage">994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZhu%26rft.auinit1%253DZ.%26rft.volume%253D134%26rft.issue%253D6%26rft.spage%253D981%26rft.epage%253D994%26rft.atitle%253DSgs1%2Bhelicase%2Band%2Btwo%2Bnucleases%2BDna2%2Band%2BExo1%2Bresect%2BDNA%2Bdouble-strand%2Bbreak%2Bends.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.08.037%26rft_id%253Dinfo%253Apmid%252F18805091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.08.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18805091&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259318100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/4/1191.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1177" title="Construction of a High-Density American Cranberry (Vaccinium macrocarpon Ait.) Composite Map Using Genotyping-by-Sequencing for Multi-pedigree Linkage Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1201" title="Unstable Inheritance of 45S rRNA Genes in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/4"><span>Volume 7 Issue 4, April 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/4.cover-source.jpg?itok=KOTpzmYp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1191" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZwBF8TW_Ptx7MS02Vw0gUEoC6Hrs03gwc9zKJlFIThs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/4/1191" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/4/1191" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/4/1191" class="active" data-icon-position="" data-hide-link-title="0">Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483825" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ohwfaCrDFpK2g8w-nrVuE5mYkH23cnZts6Cr5QTmY28" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1191.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/4/1191" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nUuiSrDmcVqmE3LntICHuws8kouoG1cxBDjXUMvDWmk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/4/1191" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1191.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483825" id="node483825--2" data-pisa="ggg;7/4/1191" data-pisa-master="ggg;g3.116.037390" data-apath="/ggg/7/4/1191.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Grzegorz Zapotoczny</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4424-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeff Sekelsky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1191-1199;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037390 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483825--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Grzegorz Zapotoczny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grzegorz%2BZapotoczny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zapotoczny%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrzegorz%2BZapotoczny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeff Sekelsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Integrative Program for Biological and Genome Sciences, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeff%2BSekelsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sekelsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeff%2BSekelsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4424-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeff Sekelsky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sekelsky@unc.edu" class="" data-icon-position="" data-hide-link-title="0">sekelsky@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/4/1191" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/4/1191&amp;title=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/4/1191&amp;title=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/4/1191&amp;title=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/4/1191&amp;text=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/4/1191&amp;title=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/4/1191&amp;t=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/4/1191&amp;title=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/4/1191&amp;title=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483825/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483825" id="node483825--3" data-pisa="ggg;7/4/1191" data-pisa-master="ggg;g3.116.037390" data-apath="/ggg/7/4/1191.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Human Cell Assays for Synthesis-Dependent Strand Annealing and Crossing over During Double-Strand Break Repair</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Grzegorz Zapotoczny</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4424-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeff Sekelsky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1191-1199;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037390 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483825--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Grzegorz Zapotoczny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grzegorz%2BZapotoczny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zapotoczny%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrzegorz%2BZapotoczny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeff Sekelsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Genetics and Molecular Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Integrative Program for Biological and Genome Sciences, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeff%2BSekelsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sekelsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeff%2BSekelsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4424-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeff Sekelsky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sekelsky@unc.edu" class="" data-icon-position="" data-hide-link-title="0">sekelsky@unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1191&amp;rft.epage=1199&amp;rft.atitle=Human%20Cell%20Assays%20for%20Synthesis-Dependent%20Strand%20Annealing%20and%20Crossing%20over%20During%20Double-Strand%20Break%20Repair&amp;rft.volume=7&amp;rft.issue=4&amp;rft.date=2017-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zapotoczny%2C+Grzegorz&amp;rft.au=Sekelsky%2C+Jeff"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483825/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483825/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483825/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483825/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483825/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483825/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483825/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483825/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483825/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483825/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483825/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483825/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/4/1191" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/4/1191.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/4/1191.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/4/1191.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/4/1191.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
