<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/4/1377" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/4/1377.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/4/1377.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/4/1377.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/4/1377/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/4/1377.atom" />
<meta name="HW.pisa" content="ggg;7/4/1377" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" />
<meta name="DC.Identifier" content="10.1534/g3.117.040717" />
<meta name="DC.Date" content="2017-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Tsai et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection uses genome-wide marker information to predict breeding values for traits of economic interest, and is more accurate than pedigree-based methods. The development of high density SNP arrays for Atlantic salmon has enabled genomic selection in selective breeding programs, alongside high-resolution association mapping of the genetic basis of complex traits. However, in sibling testing schemes typical of salmon breeding programs, trait records are available on many thousands of fish with close relationships to the selection candidates. Therefore, routine high density SNP genotyping may be prohibitively expensive. One means to reducing genotyping cost is the use of genotype imputation, where selected key animals ( e.g. , breeding program parents) are genotyped at high density, and the majority of individuals ( e.g. , performance tested fish and selection candidates) are genotyped at much lower density, followed by imputation to high density. The main objectives of the current study were to assess the feasibility and accuracy of genotype imputation in the context of a salmon breeding program. The specific aims were: (i) to measure the accuracy of genotype imputation using medium (25 K) and high (78 K) density mapped SNP panels, by masking varying proportions of the genotypes and assessing the correlation between the imputed genotypes and the true genotypes; and (ii) to assess the efficacy of imputed genotype data in genomic prediction of key performance traits (sea lice resistance and body weight). Imputation accuracies of up to 0.90 were observed using the simple two-generation pedigree dataset, and moderately high accuracy (0.83) was possible even with very low density SNP data (∼250 SNPs). The performance of genomic prediction using imputed genotype data was comparable to using true genotype data, and both were superior to pedigree-based prediction. These results demonstrate that the genotype imputation approach used in this study can provide a cost-effective method for generating robust genome-wide SNP data for genomic prediction in Atlantic salmon. Genotype imputation approaches are likely to form a critical component of cost-efficient genomic selection programs to improve economically important traits in aquaculture." />
<meta name="DC.Contributor" content="Hsin-Yuan Tsai" />
<meta name="DC.Contributor" content="Oswald Matika" />
<meta name="DC.Contributor" content="Stefan McKinnon Edwards" />
<meta name="DC.Contributor" content="Roberto Antolín–Sánchez" />
<meta name="DC.Contributor" content="Alastair Hamilton" />
<meta name="DC.Contributor" content="Derrick R. Guy" />
<meta name="DC.Contributor" content="Alan E. Tinch" />
<meta name="DC.Contributor" content="Karim Gharbi" />
<meta name="DC.Contributor" content="Michael J. Stear" />
<meta name="DC.Contributor" content="John B. Taggart" />
<meta name="DC.Contributor" content="James E. Bron" />
<meta name="DC.Contributor" content="John M. Hickey" />
<meta name="DC.Contributor" content="Ross D. Houston" />
<meta name="article:published_time" content="2017-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection uses genome-wide marker information to predict breeding values for traits of economic interest, and is more accurate than pedigree-based methods. The development of high density SNP arrays for Atlantic salmon has enabled genomic selection in selective breeding programs, alongside high-resolution association mapping of the genetic basis of complex traits. However, in sibling testing schemes typical of salmon breeding programs, trait records are available on many thousands of fish with close relationships to the selection candidates. Therefore, routine high density SNP genotyping may be prohibitively expensive. One means to reducing genotyping cost is the use of genotype imputation, where selected key animals (&lt;i&gt;e.g.&lt;/i&gt;, breeding program parents) are genotyped at high density, and the majority of individuals (&lt;i&gt;e.g.&lt;/i&gt;, performance tested fish and selection candidates) are genotyped at much lower density, followed by imputation to high density. The main objectives of the current study were to assess the feasibility and accuracy of genotype imputation in the context of a salmon breeding program. The specific aims were: (i) to measure the accuracy of genotype imputation using medium (25 K) and high (78 K) density mapped SNP panels, by masking varying proportions of the genotypes and assessing the correlation between the imputed genotypes and the true genotypes; and (ii) to assess the efficacy of imputed genotype data in genomic prediction of key performance traits (sea lice resistance and body weight). Imputation accuracies of up to 0.90 were observed using the simple two-generation pedigree dataset, and moderately high accuracy (0.83) was possible even with very low density SNP data (∼250&amp;nbsp;SNPs). The performance of genomic prediction using imputed genotype data was comparable to using true genotype data, and both were superior to pedigree-based prediction. These results demonstrate that the genotype imputation approach used in this study can provide a cost-effective method for generating robust genome-wide SNP data for genomic prediction in Atlantic salmon. Genotype imputation approaches are likely to form a critical component of cost-efficient genomic selection programs to improve economically important traits in aquaculture.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/04/01" />
<meta name="citation_mjid" content="ggg;7/4/1377" />
<meta name="citation_id" content="7/4/1377" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/4/1377" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/4/1377.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/4/1377.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/4/1377.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040717" />
<meta name="citation_pmid" content="28250015" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1377" />
<meta name="citation_lastpage" content="1383" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hsin-Yuan Tsai" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Oswald Matika" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Stefan McKinnon Edwards" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Roberto Antolín–Sánchez" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Alastair Hamilton" />
<meta name="citation_author_institution" content="Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands" />
<meta name="citation_author" content="Derrick R. Guy" />
<meta name="citation_author_institution" content="Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands" />
<meta name="citation_author" content="Alan E. Tinch" />
<meta name="citation_author_institution" content="Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands" />
<meta name="citation_author" content="Karim Gharbi" />
<meta name="citation_author_institution" content="Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3JT, United Kingdom" />
<meta name="citation_author" content="Michael J. Stear" />
<meta name="citation_author_institution" content="Institute of Biodiversity, Animal Health &amp; Comparative Medicine, University of Glasgow, G61 1QH, United Kingdom" />
<meta name="citation_author" content="John B. Taggart" />
<meta name="citation_author_institution" content="Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom" />
<meta name="citation_author" content="James E. Bron" />
<meta name="citation_author_institution" content="Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom" />
<meta name="citation_author" content="John M. Hickey" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Ross D. Houston" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom" />
<meta name="citation_author_email" content="ross.houston@roslin.ed.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. Antolín;citation_author=C. Nettelblad;citation_author=G. Gorjanc;citation_author=D. Money;citation_author=J. M. Hickey;citation_title=A hybrid method for the imputation of genomic data in livestock populations.;citation_pages=30;citation_volume=49;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=D. P. Berry;citation_author=J. F. Kearney;citation_title=Imputation of genotypes from low- to high-density genotyping platforms and implications for genomic selection.;citation_pages=1162-1169;citation_volume=5;citation_year=2011;citation_issue=8;citation_pmid=22440168;citation_doi=10.1017/S1751731111000309" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype imputation with millions of reference samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_title=Genome wide association study for resistance to Caligus rogercresseyi in Atlantic salmon (Salmo salar L.) using a 50K SNP genotyping array.;citation_author=K. Correa;citation_author=J. P. Lhorente;citation_author=L. Bassini;citation_author=M. E. Lopez;citation_author=A. Di Genova;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Dou;citation_author=X. Li;citation_author=Q. Fu;citation_author=W. Jiao;citation_author=Y. Li;citation_title=Evaluation of the 2b-RAD method for genomic selection in scallop breeding.;citation_pages=19244;citation_volume=6;citation_year=2016;citation_pmid=26754638;citation_doi=10.1038/srep19244" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Druet;citation_author=M. Georges;citation_title=A hidden Markov model combining linkage and linkage disequilibrium information for haplotype reconstruction and quantitative trait locus fine mapping.;citation_volume=184;citation_year=2010;citation_pmid=20008575;citation_doi=10.1534/genetics.109.108431" />
<meta name="citation_reference" content="citation_journal_title=J. R. Soc. Interface;citation_author=K. Gharbi;citation_author=L. Matthews;citation_author=J. Bron;citation_author=R. Roberts;citation_author=A. Tinch;citation_title=The control of sea lice in Atlantic salmon by selective breeding.;citation_pages=0574;citation_volume=12;citation_year=2015" />
<meta name="citation_reference" content="citation_title=ASReml User Guide.;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=B. Gjerde;citation_author=J. Ødegård;citation_author=I. Thorland;citation_title=Estimates of genetic variation in the susceptibility of Atlantic salmon (Salmo salar) to the salmon louse Lepeophtheirus salmonis.;citation_pages=66-72;citation_volume=314;citation_year=2011;citation_doi=10.1016/j.aquaculture.2011.01.026" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection using low-density marker panels.;citation_volume=182;citation_year=2009;citation_pmid=19299339;citation_doi=10.1534/genetics.108.100289" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=H. D. Daetwyler;citation_author=J. W. Kijas;citation_author=J. H. J. der Werf;citation_title=Accuracy of genotype imputation in sheep breeds.;citation_pages=72-80;citation_volume=43;citation_year=2012;citation_issue=1;citation_pmid=22221027" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=R. Babu;citation_author=G. de losCampos;citation_title=Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs.;citation_pages=654-663;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.07.0358" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. M. Hickey;citation_author=B. P. Kinghorn;citation_author=B. Tier;citation_author=J. H. J. van der Werf;citation_author=M. A. Cleveland;citation_title=Optimising multistage dairy cattle breeding schemes including genomic selection using decorrelated or optimum selection indices.;citation_pages=1-1;citation_volume=44;citation_year=2012;citation_pmid=22252172;citation_doi=10.1186/1297-9686-44-1" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. D. Houston;citation_author=J. B. Taggart;citation_author=T. Cézard;citation_author=M. Bekaert;citation_author=N. R. Lowe;citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).;citation_pages=90;citation_volume=15;citation_year=2014;citation_pmid=24524230;citation_doi=10.1186/1471-2164-15-90" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. Hozé;citation_author=M.-N. Fouilloux;citation_author=E. Venot;citation_author=F. Guillaume;citation_author=R. Dassonneville;citation_title=High-density marker imputation accuracy in sixteen French cattle breeds.;citation_pages=1-11;citation_volume=45;citation_year=2013;citation_pmid=23281913;citation_doi=10.1186/1297-9686-45-1" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=J. Kijas;citation_author=N. Elliot;citation_author=P. Kube;citation_author=B. Evans;citation_author=N. Botwright;citation_title=Diversity and linkage disequilibrium in farmed Tasmanian Atlantic salmon.;citation_pages=237-241;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.;citation_pages=816-834;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058334;citation_doi=10.1002/gepi.20533" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=P. Matthew;citation_title=The Atlantic salmon genome provides insights into rediploidization.;citation_pages=200-205;citation_volume=533;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Lillehammer;citation_author=T. H. E. Meuwissen;citation_author=A. K. Sonesson;citation_title=A low-marker density implementation of genomic selection in aquaculture using within-family genomic breeding values.;citation_pages=39;citation_volume=45;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=P. Ma;citation_author=R. F. Brøndum;citation_author=Q. Zhang;citation_author=M. S. Lund;citation_author=G. Su;citation_title=Comparison of different methods for imputing genome-wide marker genotypes in Swedish and Finnish red cattle.;citation_pages=4666-4677;citation_volume=96;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Accelerating improvement of livestock with genomic selection.;citation_pages=221-237;citation_volume=1;citation_year=2013;citation_pmid=25387018;citation_doi=10.1146/annurev-animal-031412-103705" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=N. Moghaddar;citation_author=K. P. Gore;citation_author=H. D. Daetwyler;citation_author=B. J. Hayes;citation_author=J. H. J. van der Werf;citation_title=Accuracy of genotype imputation based on random and selected reference sets in purebred and crossbred sheep populations and its effect on accuracy of genomic prediction.;citation_pages=1-12;citation_volume=47;citation_year=2015;citation_pmid=25595431;citation_doi=10.1186/s12711-014-0081-5" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Ødegård;citation_author=T. H. E. Meuwissen;citation_title=Identity-by-descent genomic selection using selective and sparse genotyping.;citation_pages=3;citation_volume=46;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. Ødegård;citation_author=T. Moen;citation_author=N. Santi;citation_author=S. A. Korsvoll;citation_author=S. Kjøglum;citation_title=Genomic prediction in an admixed population of Atlantic salmon (Salmo salar).;citation_pages=402;citation_volume=5;citation_year=2014;citation_pmid=25484890" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Palaiokostas;citation_author=S. Ferraresso;citation_author=R. Franch;citation_author=R. D. Houston;citation_author=L. Bargelloni;citation_title=Genomic prediction of resistance to Pasteurellosis in gilthead sea bream (Sparus aurata) using 2b-RAD sequencing.;citation_pages=3693-3700;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Y. Palti;citation_author=G. Gao;citation_author=S. Liu;citation_author=M. P. Kent;citation_author=S. Lien;citation_title=The development and characterization of a 57K single nucleotide polymorphism array for rainbow trout.;citation_pages=662-672;citation_volume=15;citation_year=2015;citation_pmid=25294387;citation_doi=10.1111/1755-0998.12337" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H. Pausch;citation_author=B. Aigner;citation_author=R. Emmerling;citation_author=C. Edel;citation_author=K.-U. Götz;citation_title=Imputation of high-density genotypes in the Fleckvieh cattle population.;citation_pages=1-10;citation_volume=45;citation_year=2013;citation_pmid=23281913;citation_doi=10.1186/1297-9686-45-1" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Rev. Aquacult.;citation_author=D. Robledo;citation_author=C. Palaiokostas;citation_author=L. Bargelloni;citation_author=P. Martínez;citation_author=R. D. Houston;citation_title=Applications of genotyping by sequencing in aquaculture breeding and genetics.;citation_year=2017;citation_doi=10.1111/raq.12193" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=D. Segelke;citation_author=J. Chen;citation_author=Z. Liu;citation_author=F. Reinhardt;citation_author=G. Thaller;citation_title=Reliability of genomic prediction for German Holsteins using imputed genotypes from low-density chips.;citation_pages=5403-5411;citation_volume=95;citation_year=2012;citation_issue=9;citation_pmid=22916947" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. K. Sonesson;citation_author=T. H. E. Meuwissen;citation_title=Testing strategies for genomic selection in aquaculture breeding programs.;citation_pages=37-37;citation_volume=41;citation_year=2009;citation_pmid=19566932;citation_doi=10.1186/1297-9686-41-37" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=K. Gharbi;citation_title=Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.;citation_pages=969;citation_volume=16;citation_year=2015;citation_pmid=26582102;citation_doi=10.1186/s12864-015-2117-9" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=J. E. Bron;citation_title=Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.;citation_pages=1-11;citation_volume=48;citation_year=2016;citation_pmid=26743767;citation_doi=10.1186/s12711-015-0181-x" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genet.;citation_author=H. Y. Tsai;citation_author=D. Robledo;citation_author=N. R. Lowe;citation_author=M. Bekaert;citation_author=B. John;citation_title=Construction and annotation of a high density SNP linkage map of the Atlantic salmon (Salmo salar) genome.;citation_pages=2173-2179;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=Y. Uemoto;citation_author=S. Sasaki;citation_author=Y. Sugimoto;citation_author=T. Watanabe;citation_title=Accuracy of high-density genotype imputation in Japanese black cattle.;citation_pages=388-394;citation_volume=46;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=R. L. Vallejo;citation_author=T. D. Leeds;citation_author=B. O. Fragomeni;citation_author=G. Gao;citation_author=A. G. Hernandez;citation_title=Evaluation of genome-enabled selection for bacterial cold water disease resistance using progeny performance data in rainbow trout: insights on genotyping methods and genomic prediction models.;citation_pages=96;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=P. M. VanRaden;citation_author=J. R. O’Connell;citation_author=G. R. Wiggans;citation_author=K. A. Weigel;citation_title=Genomic evaluations with many more genotypes.;citation_pages=10-10;citation_volume=43;citation_year=2011;citation_pmid=21366914;citation_doi=10.1186/1297-9686-43-10" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. M. Yáñez;citation_author=R. D. Houston;citation_author=S. Newman;citation_title=Genetics and genomics of disease resistance in salmonid species.;citation_pages=415;citation_volume=5;citation_year=2014;citation_pmid=25505486" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. M. Yáñez;citation_author=S. Newman;citation_author=R. D. Houston;citation_title=Genomics in aquaculture to better understand species biology and accelerate genetic progress.;citation_pages=128;citation_volume=6;citation_year=2015;citation_pmid=25883603" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. M. Yáñez;citation_author=S. Naswa;citation_author=M. E. López;citation_author=L. Bassini;citation_author=K. Correa;citation_title=Genomewide single nucleotide polymorphism discovery in Atlantic salmon (Salmo salar): validation in wild and farmed American and European populations.;citation_pages=1002-1011;citation_volume=16;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection uses genome-wide marker information to predict breeding values for traits of economic interest, and is more accurate than pedigree-based methods. The development of high density SNP arrays for Atlantic salmon has enabled genomic selection in selective breeding programs, alongside high-resolution association mapping of the genetic basis of complex traits. However, in sibling testing schemes typical of salmon breeding programs, trait records are available on many thousands of fish with close relationships to the selection candidates. Therefore, routine high density SNP genotyping may be prohibitively expensive. One means to reducing genotyping cost is the use of genotype imputation, where selected key animals ( e.g. , breeding program parents) are genotyped at high density, and the majority of individuals ( e.g. , performance tested fish and selection candidates) are genotyped at much lower density, followed by imputation to high density. The main objectives of the current study were to assess the feasibility and accuracy of genotype imputation in the context of a salmon breeding program. The specific aims were: (i) to measure the accuracy of genotype imputation using medium (25 K) and high (78 K) density mapped SNP panels, by masking varying proportions of the genotypes and assessing the correlation between the imputed genotypes and the true genotypes; and (ii) to assess the efficacy of imputed genotype data in genomic prediction of key performance traits (sea lice resistance and body weight). Imputation accuracies of up to 0.90 were observed using the simple two-generation pedigree dataset, and moderately high accuracy (0.83) was possible even with very low density SNP data (∼250 SNPs). The performance of genomic prediction using imputed genotype data was comparable to using true genotype data, and both were superior to pedigree-based prediction. These results demonstrate that the genotype imputation approach used in this study can provide a cost-effective method for generating robust genome-wide SNP data for genomic prediction in Atlantic salmon. Genotype imputation approaches are likely to form a critical component of cost-efficient genomic selection programs to improve economically important traits in aquaculture." />
<meta name="og-title" property="og:title" content="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/4/1377" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection uses genome-wide marker information to predict breeding values for traits of economic interest, and is more accurate than pedigree-based methods. The development of high density SNP arrays for Atlantic salmon has enabled genomic selection in selective breeding programs, alongside high-resolution association mapping of the genetic basis of complex traits. However, in sibling testing schemes typical of salmon breeding programs, trait records are available on many thousands of fish with close relationships to the selection candidates. Therefore, routine high density SNP genotyping may be prohibitively expensive. One means to reducing genotyping cost is the use of genotype imputation, where selected key animals ( e.g. , breeding program parents) are genotyped at high density, and the majority of individuals ( e.g. , performance tested fish and selection candidates) are genotyped at much lower density, followed by imputation to high density. The main objectives of the current study were to assess the feasibility and accuracy of genotype imputation in the context of a salmon breeding program. The specific aims were: (i) to measure the accuracy of genotype imputation using medium (25 K) and high (78 K) density mapped SNP panels, by masking varying proportions of the genotypes and assessing the correlation between the imputed genotypes and the true genotypes; and (ii) to assess the efficacy of imputed genotype data in genomic prediction of key performance traits (sea lice resistance and body weight). Imputation accuracies of up to 0.90 were observed using the simple two-generation pedigree dataset, and moderately high accuracy (0.83) was possible even with very low density SNP data (∼250 SNPs). The performance of genomic prediction using imputed genotype data was comparable to using true genotype data, and both were superior to pedigree-based prediction. These results demonstrate that the genotype imputation approach used in this study can provide a cost-effective method for generating robust genome-wide SNP data for genomic prediction in Atlantic salmon. Genotype imputation approaches are likely to form a critical component of cost-efficient genomic selection programs to improve economically important traits in aquaculture." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
    <title>Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"a4jVbV8zzGKgrdtLkgHGHH89NSWnAmFrIJ3uijY27dI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483837","apath":"\/ggg\/7\/4\/1377.atom","pisa":"ggg;7\/4\/1377","ac":{"\/ggg\/7\/4\/1377.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/4\/1377.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1377"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1377"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/4\/1377.abstract":true,"\/content\/7\/4\/1377":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HZ3ebvl8AIZppiNEu3rp+FoZBxGga4bj5qhRArQcImOSKZ2qXfbuBRBkfJ4W0ohnKn9lwCWWqKKcuNHxDdivO1XGWteIFigJhYxDmvIaAp2Edqypjohdx4qh5spUKCjHrTcHwFUzpZTmMOS8hS8yv+e8m267NFvGqRdtKR1oKxwO3hatVj93LlAPvXyCwvQkY88\\\\\\\/tvnctKIpsZJo0QoXIo4th4dK7PtJZX3bnrnr32+yXM\\\\\\\/ZoymMsbzh1rb\\\\\\\/PEiq0jYKRS5P5lNlMXwp7bZ5w0Mi5vUjhfzxgwkpM371E7xZI+ySKuudkHSW0ZCzlHhV87iPf69qt4129kNsrFdoIb5z1rQdCDeAoXWEWCAOngiE6Wm6dBRIISsfCJ9I1PZcxySpkDBmvYL+aIifHD9MUY8nCwANNMS1\\\\\\\/dzIrD5jFDgwK4ssH8bwkjjl0Um5h9p5Do5tZlg8VlYwx+2Nmi0iDKaxLAkRCkE9LycPegOZaLpvPINfWWEa6ONQeRA9TgyedvwPWFlsywKkNPLKJkPgqtYTVV3eQXb4l0pOaz7aSX\\\\\\\/wB6LMKrZgfIolHsg0\\\\\\\/NL6Ia7OprESmYbj5BSwDPpM8Y4jcp+H8TJ63relA3TN8hSdTNFzXNBx3rGgFOymzSM86IZV1QExoBCDORKYfw+uwWZtgSfk3boQTGlGy9sp2Yng80h57efx5RYEQqSjkGMU+EX8JWW8iPUkOtrGtMs9l9j1YKJHaGnqwPMgmYUY6NFA7Dtt93hW1cLsTGcTx2iBM\\\\\\\/1advbwSOMmmvDjwi9Yy3JbEC1hfvtrGw9PLSUGMkY7+HfBKMs\\\\\\\/haktZvqF1OKqXJVlS5HqB8P1tURIqHK8riJj9tHj4ac1aYMQH3T7VIsa7coSrmVW6tJU9IABY7KC4m89OHCUjsFePEFZXZZraQyO0EytlVhuRf+nYtSKeo6rbBJbkYd4u8\\\\\\\/DDm3C5MsFCDfOKwcvJIkGbaOzABq02S4nTqzlFtW1NessqWx6n\\\\\\\/TBzX3CQarcpuxIQCURGYCRB+NNL09jeOLwV3noRCVVLvgffNNrIPQmYuCZUvrg6Ynky6fwkuBzkCfEVof+sb+oPoJT9esZ4KC99JOvqq5tgfHx5U9KIueJ72Wf7cwQXRTlZ99LZv1IifT+0r+8gpi\\\\\\\/T5FPfvDXmcUsEKFsZeoyssuYjNwNJNTJ3FZ5SOWisgIg\\\\\\\/ib6rYidztnkJUEv7w0TkOPRJZpLqu+jhRsdycF8AxKZfnBFwgRiCAJ8cTXXqPzFuIARJi7Op\\\\\\\/tFqHN4UJNJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022yDDL1URBxVV9EZN5BA7hLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b8c6fbd43a349171d4f54f674b9066c67bc6174e82825396d6069bdb12166e39\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ry7xGLZqYrD6iOPUMpVwNpgDHF9NMhsSrKbea4l9ujcg\\\\\\\/G8TJRCS9BIBJJnaEi5xkU\\\\\\\/FA4Zd0L+lIRDjm3TXwXZ61lv3UdwtZgiwSZAW3V8Q3wDEROv40oZFaZD3Q6bKnlZ2TP4rhEdLEzpJ3lFOiXKrxHlg6rBQu3dQsj2sdWlKQBX9M8zDBTNDMJtZoKw083WhtkG7gZNwNCIXEeTIoF5EyX+uB9+DMVZucvUEFz8n+rGJHAQhMPjp+RI3yv2m027cubzTnma+IxR02vgwf3w\\\\\\\/XH5clTuiwXqQqqRvzCUzQ5UWZJBWkFV1LyWzgiCkThETbCvaTF8OHswwN\\\\\\\/nQ1xpHNAGRGuKp61cSzLquRDZujo1c6Ek\\\\\\\/YTNd1VbPqn4l\\\\\\\/\\\\\\\/sTzZQSbMZ6E1972Pdo+9noIs\\\\\\\/ZLleGd+Apc+2OGmdBqLeVhPO8yolIPwXIjOwrZaYDpRKfKPVkmhZtKXolBRWrjDbXWIuL5aVIVoZTcw1EoGVhr6dQbY0Z6OFl+5xXM\\\\\\\/HcafV5df\\\\\\\/JRCpkKHpiWKwHDIMQ9nGHZ0D1A3sYA+L3WnvBeqckKo6\\\\\\\/GE7shrp+mjbZqNrbRPlcd8hYz\\\\\\\/hQO\\\\\\\/62HY\\\\\\\/jxUoLHytfqmJo1nJ1rpjgGdqN5J73ECz4KCYAdkLikQ1mAydquG0EXIoAdJXmvbUyHfv1iipJa933s8olQTy0VX5L6nCIxVD41ClDOrX6CCVP5CgUSjMUkXrDTWlpACDquUDQCQUS5mrB+2NfX7UwuXSFAdd4MWLEKbIrWqU3VlxzhmhqvSPcr4gnH\\\\\\\/XpAG4ACYHe+mTaoog69ASB6mc9KNcCx\\\\\\\/s0l+eYu2UnTeIxk7XgeRg3o29RjbwXoNKOc\\\\\\\/5fCaGgY4mFxmg04IDfQfYZR+ddHlddJAeAnryBJSYmJDjprEo78Io7tg3EvE1WWyVs7\\\\\\\/4JSESQ\\\\\\\/bLJH9LoORXHss5O5jRLfwoxTrS2qCdOir\\\\\\\/jVMegexdOPXZWoEIHc49OxdD09c7cBLPT7lCEWUcCBkZtId3Rw4NWl8UTrHQaTU6fuHBSSRmHVA3EWUMHWUNKCO0PhnVx4cXewVkcvqd\\\\\\\/059L2ZjI++LSB7LgUQpuas1TXdu+UYK6+JwdM17X4vrDt9NSCsxfkJ12pvEvPzKHaeC3U8OGaiO5RSwMpAIFVnj+s+D0ul81UBoHkqZxWixCfZdOQZsA5RgmZMWhKiJsgd\\\\\\\/nQCvMv7bAcv\\\\\\\/gj+as3vgBIeFUYb4qA8\\\\\\\/H9Tpbq+WZvXurN18Wdsx6CaFzup3JQ1P84FcnndHVh4\\\\\\\/f2Vu3kX7BdlwzbyRmTBx7wimNUNdVjG4\\\\\\\/UywzKgPns3ztCxkwrjkFdruisIEx04owK3OmKxSArvrGqCku0xvi\\\\\\\/gnKQsZ6hWcmkuaHUENPgwU2BqDN1LpDHHhSwssLiqaSuEpP3UirRnpANtUpiU8xckJTSM1\\\\\\\/GBNU1rQKK9L3Qp0s9\\\\\\\/2F9rCKwo9UFWWV5UT+jznoNzqAveocqhjT\\\\\\\/oNjdjmqPGZQCIyJokLF9ljGF3bMiON\\\\\\\/nq07ypJD7juRRSl\\\\\\\/qi\\\\\\\/oeCTJyjSs19mijxiyZfUBbXA14vPZmcmbjLTjLjZe8e+2VNYOiNqgYTDTBtiULjJYg7y\\\\\\\/PRP4jxjdO8h6RIx8cu24E+E1zYnDy56aojVg7P1bSZ94j8MLG+UOptG8Dun+6qcao2dvmrJkzOZ7jMOwrdYXO8w2PzoTa0PFoth3rk1rb0QsYE+TmRWjyZmbrFDYbVDR8v\\\\\\\/hVN9o5hDwiQEKwvWaYtyxTyhtdsCvLC3esjdPkz0tSt8372IvyDRUT1+489yVdRlcBPoqr7WWBUjtBfvnu9COQGnQk5Wxnn4z0xLvjtnfdVDTo2O32PbPDEB0tz5978iOeR\\\\\\\/GOZKi1KX2O2nDXMpfboVduT+0H7GzQn5ngWdiWow\\\\\\\/92qDpVfONW0SgybxpMp4ePQYb10FqGILuLLXvNCLpVp9ZVym\\\\\\\/KqUkUKiRM4nOmfh4KtmIfcmbBEPJibZWiA7b8ERfo28xLzVCE1+v2UIW6qqSF9\\\\\\\/inWqJWqhECIynUQMrJkzKVuqj+zBeBjYgxuJSa4UNP9AoZjps169orMsJfZ8XM2gksy0UqqeK9S1DBiilpMhmRuZLI+UDT4\\\\\\\/KTiV+KuwzA3idayrF\\\\\\\/fVqTC\\\\\\\/Lxi4KJb6EY37dkUAUoG0jx5usEjq0IhF6sUluR7WnY8V9D8LyAVJxKOEp9u6SZkNuFtO\\\\\\\/8EHfUBRPFITS\\\\\\\/dxYgpyRU6sd+Ytbw3Ysbe+s\\\\\\\/ItwmYqwD3ml1WCmR4Rt9OXzBlKNrFKKnGLUjJUZH7SaTrus7mUqrF3R7D9bIFW29RXCPcKWwQnr5MeE4Mb8qlSAlaEnqSi7ZFpCAYUrd+71C1nwDeXeWx3NOLrryOHB2TE25G9bbgRF5B3bunGUSz0KXmjRGSFahjXEXCpPMNMnpxQA\\\\\\\/JudvRxWzYllGpOyuJvVbSqn6yJYhZw\\\\\\\/rcwmfqu6YFgmN8SvCChR4ROheHitjc0JhzCjtjtZF96631D+b+1XYHEmkyfxW20YixSoMXbk4a+oWH69FA\\\\\\\/1MpM\\\\\\\/Itg9T0SW0kDx2nxqgAiUhqayKgRracND24GGtVBzwFB9hIKVam0rh9qPm2xz28v8pBjDgJyxoqcDJmD\\\\\\\/EvrUeevze8JzB5K8l4aUIfyKQvnciA8BH8nwpIMZsU\\\\\\\/iVm08GmXHtJ7ookjNLf7ynXMGFzLtCDdESPI2lYAOQDZ\\\\\\\/jwRt4tnfgRta5KEwZhdSwKMmJmTSNQRabFg66LKWM1u0u59E9c3n1eBqDyv5it9xsbTDBmHlClw\\\\\\\/\\\\\\\/qlMP4pTPpvPAFTzIQ0wohLZeBIbn6nTdVw\\\\\\\/jLQh4JesoRX3lPh+C3Yy0P0xxNvd1GOXp6sRv4E80O\\\\\\\/VdoLQJRZ8Ges3l9QIxQr+o0raxWQD1RcHMbk4L4susGBa1bScoEPZzmCi7HnM5V3ZWrrO+jBqfepglcPzbgZv5Jqgy0E9GDWdtHSWBMV1YzUyAZIeGp7ir\\\\\\\/jCINDSsCzk8HQvEySqoIlMqb\\\\\\\/v7f\\\\\\\/Xaj\\\\\\\/V\\\\\\\/J3umGYlHm\\\\\\\/OeAyn0NoB7Xl8T5uhI1Gu4v2\\\\\\\/OBSWQGFM9ViYYGMiB3OtD\\\\\\\/SFrtLfAOeys+1Fz+H3Osw0l7TdUxRB8QF5QrhxXiZQixaIDgyJ2EZBNCXtRwrEl9HFqFgVfIGL67UVaDXXiaJSAY4gK7R6SfGFnEYUCrvwvHFbRDW3GfkCPfcjlDfsRb1d4QQcGsczsNhHjWtcSlvMqHBWZmQ55Euoje8TSk7lAEdDSSsxza1gUW60Gvbkzen4QD3tLKIzGt8tGtvIAkPUYaTgrxkVYkA6oObbgXwz1d3cWjdtJuwSK3bQ0GA3jNsGqCaj4PWmQjaAsARdW6q+LerEg2EENyjVB5JNDhsP3gOc+snZAVOWY0bhcmjK\\\\\\\/3dZC9By+5JH2PLkOJbXYV0i8f2YG6aMtUg1+XC0rbxoDX9LSZ1mItIEvAd3PQR2DEUGLtrmKB7pZ4BsGKBqx54tZk+cGqORPWgtPHb7P2bvgf0lv9ql2rLOXb9mVw7m4qe0T0P1Mf8qUKJakgxwOl2remWUKIoGKonwl+Nc1rDwPAmcfaWyU3gzjSlkNhXpg1XvdELZFTvOMYSPFJ\\\\\\\/CIkU6UQrkSQcNIImxxqQfukiTpzsWI5Pe\\\\\\\/uA3O6lyMsON3vbId3SKHmiyvwGsULlcwjTOtLn1z6+XVZgG6mrCQsUHz6evcVjJPeWl4QuO2uQQl7nPIrD\\\\\\\/p7DeS5AJ8eUHeamv6UKsbHFky0Clvc1scfE4pPgQbkPh8k29yAAKMYJY7zrWgfepevH5tjcPc5cZjQGmrKa1yxK0l592QAnFCPZOlLvKjfZ9FtS14bwv7iwIWz8NHAAZb9hrKX3spbbwfBukJcx0IcZxVwvz\\\\\\\/S4zXUMWlG831TMzW5wv9AGvtl1Mu4gcySYZkYL23dDHBYhShfyf3BO378tC75+vTAL3\\\\\\\/T5pbXVQNFJPKKCHVGlLJe7KlIERQlPrvo\\\\\\\/kMHK0aH48WEGyD0zdICuo9UcMb6x1x2applJDYno+DqRhFH2vX3oxtPPu2Xtv1cvVrRwgOK8B7K+0ZJYcxPl8VnVrlQmhyoPrePEtE6Ju6w+K5wN+oRk\\\\\\\/Bbd\\\\\\\/PoCypCbd7oaPfjY7\\\\\\\/SQEF54nbKstLv2SPFHIfnNwKgfyGZZvR6E7IAblUoqv6cszR1X4PBYMPp+YbclzPhu4uo6cYTAO2r\\\\\\\/yt+BDnUA0gWBJus6W1v1OmW05w3ZGvVCSlvMt0pyQF5AP0pdC2at1K5TVMjQR2AdDqHEAXiP\\\\\\\/nGljvzhlORcjnhhJa\\\\\\\/fKTWGw8blzSnnfHjqABibITEicqEx7rXO7XX0CtnanADGgQz7DfswNInbKn\\\\\\\/bP7UaK3xVI89VXVdQ3cHK2tkgu5P\\\\\\\/GNFAuogfn+X79rx9Qvbrb5YThAv09gatN+mA03+Froy5YRhw3udJEqZRzhForPmq9r107+mkms1L\\\\\\\/LAzQHSD8K0RB0cTT0hWbSGI\\\\\\\/SXCkKizirxpWGB7E7WQUiNt10nGOKto3KA9aLBN2U+NAYQD6jLoOwRGWWk+L6GYMORy1wBSGB9s4awoRnVoB02S8R8NKGvH73IHo2M6XqmTq30uCcifiEh4InvCX24T\\\\\\\/oADGFUPnR6gFtYnxTAw7MxeJikoH+1I+IJOSR7PhTAL26LG92ADCtlG8ncYl3BuMBI+VUpl6aHwsKhFtEcCkW5zSnXIyXZRXvp\\\\\\\/s0h9PXG85VTksHKBz\\\\\\\/kVeumtDdiyP7Kb5g8pHSl3BV3MKn8B5qyA1BzWuXM0Xuf6iJa3TYaQ15qUXjB9IdBY8wOX5Zkr4xTMx7LDRIC98PsTCweTPPVNv5pwm\\\\\\\/ryWdTcmzVnFxKNj0DJ4PJUwNNX70yiEaThknFRz1lBDsmQ+PywunJu5OmgdToJP+ID9FmTBPO25GoacrgAfFgZCd8k9vzA8Bhldu8CWf8wkAirFHdGDzPz79y+QuOnw6KOlXOrLlce6tyUl\\\\\\\/73KFG9uDHT5ZRsznfQ8om0Drnd7ADreODfwNTywK+mOSgB0cqdr1znwHYQwqo4lEfFQlVgV1t8xfv+IkkeP3kYz7zjljvl5axeKKLoP8rM7iPqpLOmfPgLNKMtvRTWTv+wsBdMiQR6t9fiMtXgkudITw1rtE2wF+iUsPYiEcmq1v7VWBiQVC8ZSDdJ2nFACx9vUymyI9VhOLckRxp9xjdRmJ2wInsrX1y\\\\\\\/woYrDUb2wUaUsoGAw+Arzhe8jMwa\\\\\\\/YOLWwkMwh561m5vqySZW8WE5XI2KgXEtZ3nJXVFin9IknSKuULsK0KjzoE6ujcB3hg3wOnHB0SLhHd18L9VofxY5vJtoI5T2MwI7Kvuk\\\\\\\/9XYIWHfzj3AUXlOrGCmXeC0FTUQNZzxoS\\\\\\\/O23FKwBsmYAOzBziT8eQ12Zl06sbz6tX9yk50PswkeX6gWQYlrnQtJsTb30t7P75zev5KGqDe+\\\\\\\/KxfGJREZSQvf2iWUFjuxziatbJz77CgNWqFrx3jbzonxlHX1urQ1ygWNX15vN8cRNgaJ3fMUvBbXofksCpSc3vtPxhZuayw5OLmrJdMwuHHFk\\\\\\\/ee42rxCvQcwaHKjngiqHrZX9ArkJeSNUj+BL1aTYiGafqVUnakXCWIjMpTK0d32e7SgM5AipYwiI7XQ+fP3SmUmjoIXJcedyjfwP1VN\\\\\\\/O0LMkUqOnPGWbE9rrqDDGTbEB7YlYcF7EYAcT85VWlZGFzbO3iYfJxReR221+tIYi8oun1eFC9DXqIFuqntJqIHQeF96QEFVY9Q3gE610DrZXhaGxlBqtsxLtofkfI+EY8rieMpLU38ztDFMMkdqwC2sk70Zrehh\\\\\\\/0T3UWIyV4pvshIj6nLprQotDJGuOqoj0+TNfx2l8oOxD6I6RURDqCOaJT5AubvwSpKQLJdmTseVvQVSVVLbeN5i\\\\\\\/pDpjpYtSk0QlLCO2kmh31NgJ0QpXumwu6xaTyTHpPo+urUQ3\\\\\\\/Oxpbxf2Jgxh\\\\\\\/KLrKUauvOQDqTINcC0d3SWE8pKGTqt0n8rqHfPE6Nm9o6+J5MpcRBjm8qA7dGF4UzDIwp\\\\\\\/FgJ2NzS1ha4FFUenegh3R89YVvXtyZaDlYIMA137tkFvM5me1DwkhhV9pW7p4NJnxtBwqRSzTEZkijNG4lk8X\\\\\\\/OKYj4SPG0TV6R\\\\\\\/MVFi9K0cJBREmUGUdAGoXJyl20iZivcBYbKwRAKrAZervDEb5W1pciJIx+6RZHWtBj29zdtykzmUWa1+q0l3yPxIkJA3QIAudXat1ncHY0kwNEpuKpOI4zUwAaogVyBOh5oLq2+72pdgMIN094nxMECQIe6KNolGRehl45Qxdas1Nf2lGzQh3DxJSWTsKxRMo8koAsng2ZsjLzZT9M\\\\\\\/DDJKLTt1EurJC\\\\\\\/R33PbAIsDkxvstWvMNeoEjn32KZrn4wlCXvjDQy5GZlkPha7IpiVVP4a7xbOv9Zomf20PISUWQQIwk51aLoBghpKQFw0DD5Uqs5QH52+7v3U\\\\\\\/fCHizcsZ4WOnI1urfHQMJCXiyWTDU4CBoxmHBca716qv96aSOedJZTVV8q9aT+6vMG7ZrXMKo4MysFOeKJPQCl40YWzGBmJ\\\\\\\/5CxpYLgrPgwzj63fwzJ3JUzKAh\\\\\\\/6gZkcourbFej2wgg5k0kNsjzGhZ046bMmJQYGZwATOAxQy61juw2bKZRnSTx\\\\\\\/\\\\\\\/zd0\\\\\\\/iYufpVol+1Unv9e4q3txDTbeBVaOHR8H9OnuESMAzNxFdFYm4iy4vLUI+pAD8NA8S0OEF3oLuwJB0cKclwGLKMA9mxrFDPATCFemHIq587gC58tYdfVuhH1FbRj6wWU8Uv44HZbKEzq6nz8TTk1AD5HZoLVMdRmOALFytptvNQYMlhkb1JcDmyvWmY3NrshX9J72GicCK13Eig7G\\\\\\\/AND5sSukQ\\\\\\\/6enodDWesNVCP5\\\\\\\/p9Yvr82wFTxm9e9J6\\\\\\\/k7qWqd0Z7mllK4O8IkWWvtqYGofw36W\\\\\\\/TLrvTnTkYEFLLR8U0rPBBsZYQGcH21JvF71ycdYBRaItMTYWnzvHT1OCzq+h8nlJZENqhniuEplzMlXaeg3g11o4N21vP6eWl\\\\\\\/K2mBA5M1QdWWzHOiyWY\\\\\\\/X1UXK3S3RpbAqGjfV3hmUUEhoMlfeZOvJSf5Faoe2PuoH\\\\\\\/DytYQ5COV5QhMw8D94UNkF+PzBRH7ZGdkYzkPNAQRHEFNsBTurMswazq+fb9z5bV7Nw3kPiUVIBUX6dZFJCaMNyqjnuLpGrjlaVKMCehxeLG7pnkQsTOgrzEMQe573A6qRia49ryy+Egf9RXKeD8RW+Zet0369oeP53YyU2+xJhmrR3hY2Av6tgM008BOtiC5JC15Kt4\\\\\\\/7e6nUgQh80spHxMXFSEOhpsyZ7jmRtaKL4\\\\\\\/bcY6eia1tqfrM53fkrxB\\\\\\\/fV04BDSyHDvd3ek2h2g9sQpfg3wKjleKbOa1y\\\\\\\/Gtb\\\\\\\/fCWMnjQHXq+PfM\\\\\\\/0sQSgbhjM3XrOs3zFY31OjSE2cKu1xReHs4rq+6+BP\\\\\\\/v48vv9s\\\\\\\/GNCQ5ZZr0AwyM9H6tOqK8EWpOnyfzY04nB6tpgr3VA\\\\\\\/MSIeZy9bier+5M3surL2xTRE7Kc\\\\\\\/YQzpcYGW9g4kyT1pzCcOji+CCGYnQfZAEpemp09xVddt5JgQJjjutw1A4RkkZuQKPyMNtP+pm45pUWVw3z44UUL3PEb93+q1nzyxb7njyVjvFv1bYGG1pbMJ1ys3yL2opfWWV9gissyrNMpfRbBzM7EpmVV5r5FrEUkSZkjJJa4qeWkD7ROcZYLSK\\\\\\\/2MYg1RGimIItJ6\\\\\\\/NOax3g\\\\\\\/xOae8sPhGcmeRrKOraexAZ2TT0vv8L8dZ0upDRiGeXu3GOu2w86PVjaD6rRsqt45kyJn25fx\\\\\\\/JOw14UREgtzB166nqxrrB3pEMzrvT+4U1HexeE7hrRBsSqSVwyUX1TbsrhXLoXs62bptmncJTtwnjBH5aPbVkjfgu6+ffwnRSpLs+Qsqp7SelujTWW+AB+347J\\\\\\\/zqKDo8zyo2QFuuS6Ds2uzkg7rplMm1mQG2f2HR036WssDDL1YhmG1otcOCTEnv9GkdR7QVSuMC2vjlAvazF+Ky9b\\\\\\\/niyQsGasDpSV\\\\\\\/PEKO6DtneRfj66kwpmcP9sIvVRcsdqBU1oio88xRNQcMDp+B4Bz8aUm3nfHnj3O08eZ3CTaqJmnlUnfTpLxeRCCGVDuF4rpHjgcCQmQnlh49HNaCjl6EsjBLMlNnPbr\\\\\\\/swlUXNhMAhd0w0EWDHzHHhGfFFME7zSMPsGGdiUFNcGLxcqR986\\\\\\\/k+CSOMI89gIkDt7SAy1kaNoDnqp+n61QNBMF0g+UXNksDF\\\\\\\/mthfyklDpJtQ89g5sih6pdWKUTXREY9\\\\\\\/g622Ni7VebplI7s5OBE9IHjMudskUofhjfmt+4SibF3COapT3uqauzpLbaxPTuwu8+je2YKXsT\\\\\\\/rZvpGlXUzbtF1oF8qlKZ1J6QwHlJLInsBDIJXUQY\\\\\\\/dTtBnnPq6T3ErFe2E1ahhUSCkyLkXrtl8eCzKXQ5z7Ft9pebLcSBYR18isQZD+mQJMWtfv531vfFjHquPZJ0kEN3EMI0MGf6QaLzy2yfBHenKc7SFIAVPXOYt6AuWdt5zE8sMY4S\\\\\\\/d7ICPGz7IbFHg\\\\\\\/3IJ7ngtxH2DZe0ZRAFeZAo6\\\\\\\/ZwXkrIae22YeVgg5BNrwtOfaXd9tCArjHDQWDqU5WUryLFIUQnne0Upr7fr0lW\\\\\\\/5e6Rpem6CbCqwel0Eu\\\\\\\/XEkCiN1a4+L2tqqLAWm\\\\\\\/vsKdoLqZNDg6lu9lqEEQ2g7WzcitamOQMsF\\\\\\\/7RBR11GHQmh0kevo3W0WybokVrt+UTf076RI7oTQACRTEVIz4p2nwJsOcFjDzZ4f4ASybVJgkjdvuiK4IcEQ43ePuF0dp8gxnYsarkkHkXPk8CMjJu4BZ6VyORqDuGlXzSoZmDin8Cb7T4rs9TTql33l+O\\\\\\\/Fe5J738sRv1Kg1N0epEejpdS4HK4BWA2HEsXnc8RACp40EVrwPQBJgve7sGzn1Odug\\\\\\\/I0uWjncAZwkJpiF5N1\\\\\\\/EjBKV0xfqR3BYVzMO7XGH2TtiTc42JnhhGQpPHy6X+24rA7H2XhXP5SYsQtYnsWD+TEck\\\\\\\/SmUHTEOM+\\\\\\\/EAUC8ol6j2mUHuP2umC2l6BFqn+MAZHoe7dnEKHhcC\\\\\\\/AHZWS2EJf0v06e+gdLmpVBKq8Y0tpea+VGmJ7PKr0MUvP66WQNDO9phDyhd30FcDlVLCYV5ylgYIRgOdarA1GBiQt4ASoVWgSTB79w74M\\\\\\\/9Qs4g\\\\\\\/AWuNr5HCbpgSVerjsk2+2rlI7mvn+6r5M5FMBw4YD+BCnlArpbiBjPPvGXtGhn4RK+635lNsZGbuZzYhI\\\\\\\/jh88z6Gfm8wGz\\\\\\\/Mbz9dZFbdVWN+XFguwHc1+ciGHTNvFACvggniWn9+K66+nqT1hiKApFqKz7ZIda+334gBGn8FFRkf69A7euF5ahYzj5GbA+OmLu0N0FTeHoFd+aovDe9UhkP2yAyM14Jye54UoMT6acte9SCFeS24LtF8iDUnUAIrJqmCg4TbQSru3nxoYEA+T5DXi7HyDo1rgYzklnxND3V9CMWyzqkhLu4ddrQ5zofu5\\\\\\\/If3fydN021FOAjHMg0X56aveHY0RRhgQ11\\\\\\\/ATGiF8Fri\\\\\\\/Z7O7fQLvNz0cVXmEymAIPWyq\\\\\\\/lXitBFJRyO6g4eGdEQgCwKhW2GsSsebxOw+tTzej9MyfkXrUYi36f\\\\\\\/VWsKnuW6WQxG2g0dCVvj5dZ8uOWSHaJvoMRBQ4sw8zGp8LuC6c1OrqqX4kVby4YpUw6Hxr4W31ItihjFCxd2k8ruqImAHhKHOv7E9zy0UyV8FVdg111erQpC53tNovVd6n\\\\\\\/3nsWfck9vk8O1g2\\\\\\\/7NSjh+uYZax8fem6R3Kp1RC7sb7o8kcM08qz\\\\\\\/2IAsd6hVFSi+BNHgXjoe06lG\\\\\\\/NbHLl9CeKk2OchTHtZPW9bx8M+SwSA0lR8hpmKFhQXRiyNZL5v2JzDSXEKrFO65U6KNKchlLLzF5VBEIIkoRrqTtTeSE7mvVqG1fXUC7BV6iCSbfdAUIMUlbkvaHWUg4Kptj25+hTWvKzq9oodxU7u4aSmRTUUhXkbX7R48dXbfwXYBPdiweOIdP+Kklwhf2QBf2IY7xPt29pnxXMXl38ZuWsSjajDJZHlTrVSTcbbscGGuAzGoaBzSCfmKAnyGyUUk3Dnj+Ix8J6wV1zIsTvm4ZEXcF5\\\\\\\/HCxaRR7IjUKeTE0Il6ifg3WBLZQaLqaiiJQDFIHVn5WPXww6zQy4jxRURjPwzSJYJ243honhUfdLYjCzWgMT+p+3uCbQs\\\\\\\/VOyb7xV0LYDXqM9X4dyK1LDkInR7jLZ7yeOoROO4icf\\\\\\\/ECca\\\\\\\/HcqGPBSDyOzCyiANYqqwEhvrLNgwxK9IQV6Gp8msWH4hW5MJ3Lh7trR3MDIfl7Zt4U60fpJi4wAqz+WcMnkDfmPkOhHwd+TbW1avqJinjzuDDifzjm7FKXsRKAS9riYLbo7dKVx8CRYkhOUK02cwBngt42ERjyFizgd7abpn0JojPqOQpUCwjpCWbp7gKlUd0SXv6z3zKN2VKG4RPZs0iAUyJd6kJK7XPwOz3ypTlvw+o6ZmK2tAjF315UC8+Cl+034\\\\\\\/lUT8GdIkLNJ\\\\\\\/OIK6Ks9UdCJjS+l4njUO\\\\\\\/QNby\\\\\\\/WWnzVRyE028LDV7fLIJJv1SQ77r5FvDYqL94mi\\\\\\\/Onx7pdS1dFSgxxRCRldn+4n2gThVYoruOZSTvalm7Xfw9vCG44Wvu00VHT964oQ7RW0faup3SuLkNbPnkZJezhpEmsVAKqB94ImPr\\\\\\\/NtuG7jzDdXbBk6T8yxRKfO\\\\\\\/gLJozuZxXF918FzZwPnjW19iRnlZypgJPDRij5rhyhDcgsMPVubmnINhPg8RRe1T5zC3+U3Q8UHDUBdEGHvvh5Nd9OxJ0iPhi+9ZJ7gWD7Qbejm6j4Y7MYsQtIBB86JMustpUmotQ+pd5y5W5sDviBrTWS3CVw7rt2QGFBnSGHDRASfNmxuxlVxeRnz4fkq9qRXpzyPcG8ukJ7WQCczmmRRuZMvr1BKLK7Py5P3ZXcbMESfPE8Pc1GcdOvyYDCdWKXzNNeuQVXbNQsPP8BZw134J\\\\\\\/L15EJaftdN8OD+vLrTiKhOoEpcuTDygg6adrRdhKa6khSqrUvAdAKH1jcJrZEPwTkZI4mqzQ4XgAnAVA3C\\\\\\\/QyQ76y9xrzfJMMAQsG\\\\\\\/Zm5bOyTi3hDl0yvM0A2FhVtv5sTlllbtZL2M8NDnErRvFkVFWnq2Q0+ousRpM7sTZPzuovSrOn0IfY70Py+5Ts5IYd7iBd6eyik7YKOUYDG+qOF3zLBhc0wInYsb8pV8VCaT0lh0nB1UrtZy7XjMpr9L\\\\\\\/v2lRaUP9Zn4tM\\\\\\\/8wa6CwN00xy7uvkIOWzydv2\\\\\\\/FM+q2J8Em81nwOqM9ZA8X7STgrZF0fvyyG78oksrgjzi4bjBzVg1p7z7UXzIEo0LuBgOgGt\\\\\\\/icgbXrCSBAB28er9QiJ+sCsSMvzOCmFpZieDz89v3xotX3ACy0AdHfSIvNUNgWQ3S5PCxOTD3yooFgxznUAGnQ0CWG43YQ+0ZgHn6Vzl90jk4\\\\\\\/\\\\\\\/o1+aeptYo+UgxLPf1HF5TRS0IRi2+GF7fq0V2iTaP7H1BbwiGyJm009C3ZVzTpRHuSM32JjMzg18f4e2ATCOS\\\\\\\/hst+7iJAo6clftzhkec0u+WAc+rMfOCfvnQGK4ob\\\\\\\/\\\\\\\/2nPMHwm3QdM6rw3NaVulPYpAaJqpX5tNUnmxc2Ph+dm11spmaRcHaVVEeyGzPhEHfixtk1tvPAKShUpeph9SNKz8cby7m8VHC0j\\\\\\\/TwRv3nuY8Q9kGayxLzXz4smA14xvCo8QDODLAF4gbfnO0RNGSJIrHiVogGgZaiQumBbtoYi8lKvs2Ri\\\\\\\/e9kLO20CmbYYIIl2y1iRNsnNEgLuLC1XKy20\\\\\\\/eTh6NsAUer5xTZXBj38RQtocZV44H7alrYt9+BAhJp7n5XtTsvKAmTr1Lkij24n8zZJtEyMhvXwtIUSKII3XdeXNmD0kh\\\\\\\/Z5nup7ixGCJTVxQSpXgYH3B7rL6H77WyR\\\\\\\/aFYEjgoAhlQdxvytFm4Tg+EVQX\\\\\\\/9HZ90pc\\\\\\\/aibo9Kad7c9XR4\\\\\\\/JcA2N2R9o3i2OiERp7Ugc2Q9ENFgtye3qaWNJdL75OxoUFv8t8Qr8qFyKt21\\\\\\\/tHWv+1Fy5u3iNIoWQGRU+zgpipnDh00lkwNImfbTNiNXoc7fZORX214CFqxcJYmhjqXZE0Q6JFxBOmjjU1ZB7tUELJZ7zbQ+YkYd+E\\\\\\\/M4rVpUyKluABv+09+nCvqmV8dBG7bbVPzmEmpyLQbMhQaH5U8xwiKmdT5o23RUquf44SUmLelc09cUdy8CYXkoJQchY8vzKakc4lcEIqyuEoY3uS3AXeYLUHWs9jvIfgGVMVB\\\\\\\/uFFry4qFFFbU23l9WATYOld5hUHyVhuLf8g2iYOspTgwIUZxm67uXVs3Y1vAMLt2x1UBOipTaGmFH62AvoeBnWL4cKeLs6YdONM3KmPgSH+p+6smThyCAmt\\\\\\\/2xsgmTXVkwzSNknDjmziyi\\\\\\\/pOl5iiCUQDIWK+Zimir8I7AvKPmMB0Ig+GfzzpY5lUJxzZSNF3rgoAieuxy+DuTbRxkdzyLkD8Q7Q9nCSSAlWn2OkrBqvfQdT9e1Op\\\\\\\/j8wIvGfecdlSAeK7yG7FpnyUppXPz+A6OKQ5AL+45q1V0kKG9wS95zzxhjjTzVHdepHe+hm4ipejLkafDPwVPNtVJv0PTaHvzqvAL3MSvSATnl3dzQP0BwWhHcKBL+50+nCUU1mxN4iBK9lnlAppDEKgvO+MAtw3kR+QdUzkSaTizBd03gHp5vi74FCQG00P5xSts826nzkqvyR5cJTMJY5xs2mUbsSpj+y8E0lymjSGRMDinR9Xg\\\\\\\/XJNoDrcmNL6k0Y3sms9wWAhTFXXVKHyhsi9zLfuFqLdAsKssrTHufhgVdoSUih6Z\\\\\\\/ppFIEou9nJ\\\\\\\/vG137Ig3rEYNB54kaMD54yl+\\\\\\\/yJ5kSUkssvI\\\\\\\/l5y5wWswNq2v7IWaiC1Hdy+\\\\\\\/F33VgXt5OKrT5FqMAXrqKy4m4QxqNMKhqGMZ7POZE+QyWhc\\\\\\\/WX9Zx9WWdoVANeRGB\\\\\\\/KfyHHKSHHXuEpoEOxbI6EoG4m070ljzH\\\\\\\/2ANlSzlmt2pbVYlbQh5HHD52TpP\\\\\\\/czzPPOw1EESaiG0+HYn6eMOQa5KrgBrrXQQ2Aiue+5gkVRD\\\\\\\/J7ojjS2MiKS1x7taiGBkvYRpZIwwlJspCTLefJlGC3odar\\\\\\\/Ex6hMstzehakvqyGd\\\\\\\/9TqvFAMYwoKc1CXsuAJ\\\\\\\/tyWWRE5bkRMNs7h2T8BnFTPj8M9iWRsBbX7aS9z35AS4Tb046QuH06aBxBjOH7QZyTsDKxFLg5bdUUcqL\\\\\\\/4TLfdC2OZBTIwl93tUMnLG\\\\\\\/SjBHhPAGt0XVbJ7t3VFkiVIk2NumAYSKeYdVv\\\\\\\/rm5PA1Uic4mgIBNdG\\\\\\\/gqMo0Wqn2rCIvYEVZIL+9X5k\\\\\\\/\\\\\\\/ef3GTJzycyKlV\\\\\\\/Qg2jELfwhvdONusdgvswnS+Q2+SPi3B7tAAyb6DSTjxsCNXLSI6BDn0pGwIBr0ijgDCmkxpBQopWDZLcOck2JBM7tMG7osJmU9MXF8DVum\\\\\\\/Pvu+ksOTkC9D4tFUAvy1cqemb9wSgV3zdH+FFVlxcfTbvHAA35nCKJ+btSZ8LZqD1VzKbPAmueSTZK87\\\\\\\/P3TzkD2uL\\\\\\\/ys4b\\\\\\\/FK4LP1XorukVnoUoeIcBaSVobq6hGXL6KsEiGtXXAe1+orYNPnoM8FjcApxn+zMrkp681eAJTEEHFBbOUVE4rIcCsxmT2AmkT73pBsGaSrd0uxhHIOzqIKgNe9HOOCSjL\\\\\\\/xziVHCHxRFUppRmWu0JeWiuJa4OIyTn6gZvW3DbaWQ0bWQXgmXkI0DRwRTVPn7e2pDCnUs8w7d169eaXunH2x91za\\\\\\\/VzWCYMKlxfJbSu1r5Xa4lRi4vYCuncyXgRicoxudQb9dS8U2bU+ilvWkc55zrl\\\\\\\/8jgKKn0bvFN\\\\\\\/eSAoElloO+A77jav1B4wOBokVmuWUBzqvcznRp454xGlJiP19SYqc9aB0V4ylpLGPzaPbpb+kL0v9RkhHb+916ZAWg8AnZYaKqmEQ20sLG3fKBnl5TmefU7jDvuv7l\\\\\\\/QoUDlsl2bMhe1C1C0MJuoM4yG5z2IS4XUw7Z4WkY\\\\\\\/jtCZsmCqFa9ZzjscAm+D+oP87xQnxR93InN8Z6Su\\\\\\\/Q7nt8C50BiXTL5CGK5myOLUzcFHx2AbZR0pJLKtuHxNhu5ZNqy9sRTY9Yu+zimg15r66AVzMbc7+wjnFrixRBBXDSpcl50Ffz7ryKyGiQwHvRVC5TEyEE4rYw3VKJHEbdtjNKmpx4RWp6jytvP2mz2d5qzuia+RVgoXpYRVvfQLjskWQukAz\\\\\\\/UYIMjoNX9iT12kyI6FmQfpUXHXlG6+WuOEsfFxqGyQISQWppTQxK90UdRJDogePMsgkRWD5N76ETISzFLTrU9PjXI4I1i1mqbmUL7w9FUu2K2XuknRoiTUGr3s+TH1KYxW7jnCPQ05YEfjgqgtycQJ9sC0pPYaPJRWBrn3PvGSIU30s9kJue6o9mGd5f\\\\\\\/MVbHE9HQ6uwuvItUnwyP++PHevlO2WQ2s354mqss28urVSzOPSgGQkhAO+jIT2Wgvf7TQ18ZG+wHBjc5vXdE\\\\\\\/Gh4hUjPd2CRcG\\\\\\\/ZwZZdHxjaUxqBmVibn9e9N4OOk4Q7IbE7pzyUB2FxXH5K7VONlshUTks+tos9JJwFiFVL7JiElemm169WvIG6gVaZjsJM\\\\\\\/JgBNDtDLprsCv8tg2OV0WTMqNkggxDvlEy05vEW2EW6487sHFXlell3TyzYuwimgaToplKZpTcdYAF32Vyod7lca2ntHJm1hkXJSMBURz02h80AEI4pH8dPAd1bryQ5sMiaxFczx+7pQ351dRw0z2b6TtoRA6Cq7KijSMop2j53N1vYsHzBM8cmoNrz\\\\\\\/ynILVRMRlWfFEHnO1L8\\\\\\\/KdBYAXHJ1C\\\\\\\/VIXjoX5lEvKLORNt13Mc78YdfgmLxbmzqeOYbbymUaoywOnDdiQmGDjnxo40g+1uY0yxISIBX+yzb2yBUduJCER0Z1d+zuDqWaTk7sPQZguXsXb9\\\\\\\/hS\\\\\\\/FnjYZ6nRfhDCz0ClsqcYArGWvuVZG0NJ7451dJ75p8ry3my8FjsdZKJ552h9KoKozNJarZzA7\\\\\\\/wMcEOf0jxLSlKFrEBZHZLoaULaJ4TPQaI2tGv2p7kcCoNjQvRoIf1yKw75hQceu0PfvlPKNrYmcPothPeI8hjck9Tpup0WCSqdGxykUMUtO\\\\\\\/TeurT6HKW4gEHfFUu0s33PDvaqgG4EvWRvBz0hDPzbfqN7H973B8uL2xv\\\\\\\/jEoo64M24P2nvBG5ejYPFHGRaAOj6ccakLB6zDYGnO+PCbmgO7qsSlWe5rwvC1gq\\\\\\\/EaxYFN2imx9ggJ2cFr1zHv3QDEek4yT3P33kKNwYBfBftBeJaKKzfw5uQVH29kwVUFeAD0q50a6XjM+tx0yI94tz0VJz+nwFgF\\\\\\\/OC3wVcEviq86AWtMXGZ8V4TZMhD+UBu8zxrDWbm5O1SuthAL0U23wvHUQXt60kM\\\\\\\/O3dMdunDDX9HE9nJrPVu3jD6htya48hNjGbYystgAl4oaWSXwl0tougcnPq17sdRT1a3TshZ8zeC1mqf7iKvkEV84F1ew\\\\\\\/zGrTvPZ6jRXgVd2cM0xiUuiQF5xyPC2palkaATSa94PrVXbxoK0CjSAdIYWupfE+xY52LhhQIC+Au0AUYIgTpWJ\\\\\\\/c7tVo9aEvuWhrTrPQ4+Z\\\\\\\/mIGBo+6u\\\\\\\/v2fFFC8n+rIiVzXi9eDHAZYk\\\\\\\/FHKZ20UZoSJwu8uhlO9rorOBNuDizbFvWIhLUbRhnagsWbnz4itD1H4qIDQTfQ+DRg2sRsV1oZXwgewNFxc5csBZ3gF5J064xuXwhEdWKz7SNGlEgnSxt+3WscTszruQ+A9d6Jbus6m68XAQO5uIErthVE8RtfRCQ3az0tarhWiv5rlvfzFk8yF\\\\\\\/D6MM+6hWTK2i7IPsQWF7+YyLZqzI3Emmx0+AwRMaX00ELT\\\\\\\/2VFEhtq1Ivt0foPkrA9TL\\\\\\\/gBt\\\\\\\/XIZOO46C7JHbX6+8eK3tKXFjX9PAOTz7xeh2Bsq7Qwqf4dyr35hDp5crh7+KxsQaoBEP9\\\\\\\/1KLchPbse7iBuIcECx5fA\\\\\\\/\\\\\\\/T0F3M5Hi1FWSRN3KShsS+9ZR8WmUOHLQdZlzHn6XX\\\\\\\/l9d08Gey\\\\\\\/VqjEpaaLH485q0QZt76nq3jWpShf3MRgPfVrvDJKvwYUxxJ1aDOPGIPSe+iKZGWbCK5tyGO7ryJPCVqieHzqp6c8Ea+3C14xcAkUrV3PDGd50Koy2KUG1LOpXqJBUXyAmlZgXUUkA4buRDhnLQPSFmiJWdwr3aEvuJf94RPFvzjSV5s4QWfglL4VD2zKuWxFPZHs9ggPvLmrdDe5LO4kicCGWsyNboEFJqQpApCa\\\\\\\/fyQ\\\\\\\/Xn\\\\\\\/enHQ8bCcWT7LycZL0iMZ2zErqxMFig0kQPxgQuQ99muTC3BtIiL3Or2RVuTI+bMhEuTxUY+moEPi0WPYed0otfHbVtKCvcRTGcbigEFKNj9bY\\\\\\\/svs2MOmvUTTGfd5inuN6kfK6gLfba\\\\\\\/g8o6mKQagOtczcnpqjs4YH25h276\\\\\\\/0fklqLqrIkcHf1LnQfRBoocyeIIhG0oyQPVzYdydpMT4Itk3J7gtvcq+4ZJ8cP+Ub5a0uiomtu2EJ7dWu9ZZBe7RI\\\\\\\/37TEEDaOYtqXGkwlex0EV\\\\\\\/1bunN2gWwyhCDt8PEzewTUg\\\\\\\/EFDPYK9V6If86g2PKCQ3ctCF1fCHh+EwAacQ2eQF0XJzl5tWMVcInh1J5BeY1h2GxjTgjdCnXOj1I+zTjNIFoEYFXYz9gyGWtRXry85wmZNw7l2shR\\\\\\\/NKy12dnxe2JiKnNOQbywl1dquw8DjPAiUAXQ8pk9DWwcQdl\\\\\\\/ZLBnsC\\\\\\\/xTNQI1Tr6QSt45GEBo2k0Dxravtw1HVmIt1Xy6WwTvFnHcLVHlXx+9Ot2wJekR2XrRLZgc+5dlq3FxzEIdRzP\\\\\\\/QMaTivbJoefaXPV2FaniH6SQPL0iGz75VuucHZl0F2ygKgG+cobEM+V6ppEQcRIBzwAXecZJ2CHSGTTYrDwAglqsHd4SpLd5pY14WboyGdy+JXMunPYZsFuB7vz6oWuNABrKfCrn7kbacvAhqfY9g5q70Cp+jVkc1xU9T\\\\\\\/nZU9uloJUqC9yf1z2iq0zCdQ9ewXL5eKRuko37I7BAbH31uJSTmrlBbOkPGVsQXrTa7wSflrIDXm306avcE8P\\\\\\\/v8QriNbdqBGDe94ho5rdTKX1\\\\\\\/WeC6DfLRCZ+L2T6WiXZVWkMfSGX9DRZ0+uRDuvNi\\\\\\\/BHQ5YEUGyG3owQ25dUzR1Rpz3dVa85DgnFlzpl02nYr7heK+rkwtjKoIj1xbpbkbysx4LhB9SaFnhpaAihqxgX25mdB41yvYgXGlsT1MsjAGnhjFiuYXWA1AEsMI8xToiA\\\\\\\/euSxdEDH9jk1GBGzDIDhbwluiZBr+bv30\\\\\\\/QXeq5xowPpYR2cmZG69K53\\\\\\\/06Nqm7gX0dxUcksFPrtYaUaNTe9uZEVTArs2wmwiuUNBUeGUCCr9ERuP+uZv2Hgx1CWLNpWiBot7ztVaRdGS04\\\\\\\/QQkq6PC7X1Blq1rVvIzj+AXtNCfIeyRlmcqfXO3CqRn14buXvK5x1yPYjNlRjHVIufSaRZCg+ABiBzIaSFAlO2XllOAcf45R7S+WeMSi5bmtjGW99Dh9FoDUv2Yc5m+lV+U9Lp7zWYTyvsE38oKTYMDae\\\\\\\/\\\\\\\/\\\\\\\/nMzszQlRi+XrIiuk\\\\\\\/MuFx5lXb1eQVKaDr4tm11MPS\\\\\\\/SG6VjxYu1WdLWIgpGA\\\\\\\/KZ3yy0CF\\\\\\\/CY4kpaQDTTBKY4m2JfxNgO4XMNtEbbarztHiD8DVT4+eAdq3tNz86jJ2JnD+W2\\\\\\\/oaGimgafdIo9e8y\\\\\\\/tJFmfpd+c8VLlD5I2hD+viT3fUiOYLV63Lim7EluVsYuXNzVsBStTACLeLkxPEEoaIG8gODm4Dj+wZ\\\\\\\/Q8ccu\\\\\\\/oP4ZL592dl0lSU1HyFAXVdJ1KrkY203ZiyFX7SyVphRtC1Jwchju8+hPwpfx72rcVOmiwRPalQO7Xo4BbmMhc3BFilkWefWK2tGHAEiYJiqjc8xYm2g\\\\\\\/q\\\\\\\/71ev2HVsi9Uad80UK\\\\\\\/pF0QcUs2CXYNt+JBZosl3KR8KrlGzgp4RwlJDBj3zue\\\\\\\/QS0xsY42k0INnC0xClC0I9F\\\\\\\/2xoKDAQ5iykSDlUqTR9b+pbIpp44\\\\\\\/xW+BEdw0T7PZZjS258IWN3t2qZJkMEsG4\\\\\\\/9KaeSyZHChQK71wxwM0LGJ4GbtP4rcsFpee0vVcAnd30xKmTGWI2g7E1q\\\\\\\/F8xOmZuwbKUKF3ltjvCZ3Lw5pyx6ra5zryscn+iaSPjG7W4cQyoOrPIirH09KJPop8dpw3g8YfMs0MgQ+jzeNs2FguU7pOMeOU6fEZE4FZDm8xZsD0m7c\\\\\\\/YUgLobrWwBxKuaYsVv1EpjjlFaly3doZuYJEfBrC+3cTuXTc4kegiY\\\\\\\/tNE+yGNxL1iucKS8rDJo4vezmdJI6S4n6PqM6rm8d4aR9ODHvg54e+lPHpq7DQODV7xAQDPVCM022c2rSLs+BMKJfOuHYEseMNmcpvgW7o3H0Y50I7mDU4xCqEiroxu8budcICAekuwnBF2qCCn9YZqG\\\\\\\/BnsleRpTTfTs3QskoUpiS2IzlG9WN\\\\\\\/sfGo5ZcDvKo5\\\\\\\/bIlyrktvnnLWU9o2bQakrBOmijibSOZzXIG\\\\\\\/d3EDQ+iRf04pYKOMix9xlXc1G8wZxJvwlNjN6QEpBty10mFQt8vMgzXCop0vabt9C+m\\\\\\\/Y6rc\\\\\\\/6mssFBs68NeICrw\\\\\\\/4ubgtbEPXb0KMs+iyiY+aNLObVnmafIaHqPAcQb+8xEdikM6bAnm83JA\\\\\\\/t9eQCHFHBW\\\\\\\/klxiKij7q3NHFAzxs98Qp3ZGcjRQnthEqlZIrppmiSYhjADxci66NZNcjb4wmhf8wQBeC+Z0l+x1z5ycL6osJpuaxGy9LTFo8ah2IAquhbvOVrQFGE\\\\\\\/qSQgzFWHn46NdCw+Bk4L2wNz5Y3H9U3qTv83I0MBj9azsNOnkd23WYyj3Oe4L7JPTJT78SBprMZE\\\\\\\/1mfyXAMdRlp0P34TUolpUBg8mJH6sT+Uk7+HS0hznYVZe0ZAJ2hyORIKIvBqnlmewLdVq0cbYeBDwJRsDixRiK2LJdNocXuM97lZ9aTEZb5Yi0R1Y8cfiQhCOUvgoiIHbyaOfdqoVgyQtFs\\\\\\\/Yy9S50dwVZw6FltAaI9uOFQQZIPuz\\\\\\\/tuLPte7f2qfsu8Rkandnahbmp3hOU5+MamNsOpvOAxFODN\\\\\\\/U38w4RClwaA\\\\\\\/UbWRVGPc8HKybwddqjXPY1+HF2eQOfisXKS7L00n+lWC3+hnUR0uG0hvtg4moVcgJGUQKw221E4KNx1Hwrb9gCyebNK4hZvwvjM52XqgQJteKPXEChYIDfFSxWBnf9pCX2JOUas8bbieNadmBKFlNp9pyDNZXFI55wx+mSmzuc28l3ftEcG9KYy2oMGCedVy1049hV8X8m\\\\\\\/R5itf2iuS2L2OewJjdt0mKiD59KwIGH2ChDxBTl4TmUwUNJ+9yY8v45ZK9a8h8JmpKmcjtgm0neY1r7soP2vF\\\\\\\/wmAIJo0umY54bNEPOg94iTPNL0sQCOJHSSd+cnoydgkX6oAngKXFsmz4zm8QI4K+vRfK15mbWjqiDXuiSZ+6nFs8XOlrzikTpX4Uz3ge3D+HGDCT+5yrjJkLWGP1Sbs3XP5cFRJgg75X4YP0hiRWltMJsi8FTVfNUCwG6w7Dh5t5YEmgVbMWfyTE7KV93LM7BraQ8uxeeqStdVTPqGc+n5AbaGK5xRNxiJxDqvTTrQUOowhrS6wqg1fLE88k4NLQFMFMp7Y\\\\\\\/DEGPzKzNvAnnUNgj7Iemjo3D8Nr6hdUJ6KK8WClrt2ELculnPnaWiMU3RYqNPO0OrF6G9X4LzlcJ7\\\\\\\/Oi8CWGMUHdjZwDCB+1ynuXeQ4DEQ6d6tfE4hSeP7f+XOKzgKTaQp89B1SFVOrjEQW3GhwJHFRkiuKznAn7DpEvr1LQ\\\\\\\/90TvB8AHZDpyhcHedwd0ecy0b5QnDwoOJVTwyrB0DYvC9+Yj+2YOlzxp4yQlorzDBRgUKHZMToP187Vw6c06SptO4KQJfkpBZadqk3IsgF4COMA0ryggh7zlx0eB7h3hyivmYNqHju\\\\\\\/ocxVudgPkrrIg7pKDiiRSjK\\\\\\\/VVPMxC7LHkj3KxFA44aeGzZWBhr8rlsIbpqmcKe8Ea09bm2eGIKrDONqwuUmJTsRhIPSnK8RtoQIHbg4zSxNWhyJO64nucSNDnbWAUKNGWPlKy8dhqGlOYoeKcqB2kUPuj49beljQ7mHVWmY9rFXPMEkyvoJ6T32WIjubZEwk19rF08+BQjNQciqrolfenRvgnN2+erXJFz3V\\\\\\\/6O7j4yzvNzidCtLGAZnW6nBSVw91tbv3tPPsDKKH\\\\\\\/6dp\\\\\\\/dzOdhSufT2ORyqTSJOHnjAcXaPSJPwZjIL6WhG6MZVGC9I6uT9RtBf9eBtOwJ6Jeqnyzig\\\\\\\/4Gb9W6fm2PMt78WArUkGoG6aQq4ARstD86+Fx\\\\\\\/hwVoag9jFdjy0edg\\\\\\\/bXJqAFCThTGK4lZkkk71lNpuumDFgtpn7m4+E5X\\\\\\\/\\\\\\\/JXSXY+TzTPWxSEewVliXnxTV1ZNQGodpbDxhPswc9Pz07OQxjB9XJOzUTm0PX3Qn+2wvuf6jYJecuUCyuxRUDJhyBetxg6gdi58r7wQfjcctrtYQovVkRFiEy7vYB9zhxoWM3V2hKJ4T6ndvzt4XA9zlY21yU\\\\\\\/5L64mJhjqNIyFXUrry4ePlAJbo7D7bFwDd1tDMEWPDL3VO7jqUw1jzz8HpRCMdchzej1qQXWbjetlkptDjILwXayvSSA1wsapK8Vf8Hj8IEcd21kD8CFFqja6lY7wGl+qeCJOnTT47liJt1RRfmVCdJhyMI9fqk3i5m4Dnno51\\\\\\\/ZnRw8tiar+PbcAWmV6OLDUYrWg8WGb3qK2vJxCmofzMLdylNVVYwlHoH60h23op7AAPeNX2YSnbUtxZ8DSge+Q3cXTS8AOzBS6NTNYqJ2J9Dlersqb5LXLs1m+etbi4S1O+gSonrI1G8gs9j4ItaZPcRYaVtdSAzXCkkM1m98sQ\\\\\\\/YC5AM1St35oRcndT4x4w\\\\\\\/6Oq4937\\\\\\\/2jSgmLTwz95QhUnoxrQVu\\\\\\\/47fCJd\\\\\\\/IiEiCtZD\\\\\\\/LcUBFjWMLiqn2rm8jv5qg0d4iTrv5BlHBan7tvFje2IWRumK9GW8M0tAvMTRwfjgV6zo6AtxuNsp6In+IjJEf7cXEGt5\\\\\\\/9Wko9rsRBQHIgyrIX2jLNYv5r89BSHdMKPzdvSd8RomxH6GvnZ4KhaiQi3319AhWRiLE\\\\\\\/49mGWLIKhk1maSTCUCk4Pj7kGtXTlb7JhjzBFFjYKDd5NqmZ1ZaVLSER\\\\\\\/ibAIr8qaS\\\\\\\/ogfdB+rYaiZCv8FRK84uWl8AUYYVd3+ONxeujURii9NGOD2pvcIuP4lPt4isE9GeG\\\\\\\/BXWAD9RHrUOt7mZmWV1tY8xX3gGwmcyzsha8PwNb+7kssVlS9OxqGuPGtaLlTE5oD1qMKP4mE3uUAvyh1JwVGfXFYj+vv28v6NGVHmgqNOXhC7A2s3hVfKdUgto3vZDX+ygkCkHEC1PhS0tY2Xjmgp+RwjAqzw3R\\\\\\\/Z8xIeNqt00gPB0p3qiMKugCDlWxs1g7H\\\\\\\/3U6OsChtj7HSXc6QYav2d1x1EXllO1dSnJm10jSX8y+bApMzlMn7QDQJeT2Wd64XJbROr50yFS8\\\\\\\/a3RPXTrR7YX0DXaQdvRtamCTGOgikPWYjnQwV5BxWQZ6Wm\\\\\\\/a\\\\\\\/OlmxSzHgd\\\\\\\/Pj+EWWNZIGf0YrhWFvmogiNquLVGQmqIeJNVuONuNygpT5JhcwOzMFaHdQxzjV4r6WSt\\\\\\\/JuGrBXBBWZbOdei6LTVje\\\\\\\/fh1\\\\\\\/XrIdAT+UEMmmKmz8wj43VBYcCcouo5k3PkQ2Hu2sVJUtXRkoykRdFjN4gEYzg834osT8rjcxtWxwef2xKZ6DffTZIDJV2G0PYSxVvTDYlAdRMv5mWwsgHe9aOgWxSQKyTyJMbBrpM4ny+m2DssQ2qmRodZId6UJDqo3SunlhTf6CJa7tf8FkMH1qturztQWQ0WOKDeaIDqw4shNAqBkauzydlclIUzu8pOt7vVKHSNzFJIdzI472TF1osy2xvNOMp2m5m7D+tnSf6mwaFFFyAIj1O5rlY\\\\\\\/TyGHXH3KfFjXEgmHVjPC08+4oTzp2xGrwMC0oS8D5xmCeLrDKbG4HfIsqj5HuHJ5nBeVbQmMfZ9cjVYleRtYoHQ\\\\\\\/Is4ci+pKSHwdx72HwdvkFdQHN508emuGgVkR8wHou4rrghaTPMNUANJ+3w98GtYmyklb8OLvYYYlkgHJvlTlihDsI1c7ABWum9KjTt1jLEAT+Ll1lWZIMtxLyxt99su1B7FTrb3S26s46uyGhzo\\\\\\\/BIwn2InWjiQgO2MxqO56bBc+M5EPUAZskyLBvZKIoo2X8aNXvQNWAt6aOw3j1sTt0TmSfIl5n+qcX0c+nx0y2JEggZ6jI1GrJe8JodIGrPEmzoE2c5BrGzNm4wMUO02D3oO\\\\\\\/JfH9QXXttPP9ed40f1+NCle90BZ5svFdxsUeLAcwXo8cU1jLVxzsT9ESAUMbq7SIjw53b2RWyCd1HD8etWOXD87W1aGa5zWf82+1zQq3NLbKZs4v7BUy7H4G7USDcUzWcH9vWDl30eZI7N0VEB0rUKp3itgJPLaVe05aJ4ppvLTdwWeQgzQxA5a5ptMblqv9ij+iSP8\\\\\\\/Df2SRcAc71e\\\\\\\/gOxNJB2+4Yi7L0XUf+J7tPsbHL2pr1YPxoCTrRTItEcYP3Ss4LTqOL2yWaVUgJF3vNztKrgwumCmpYy95DGG3w82Mpv7Mz6ysJdb2JNVoThx7sz9DONq+wXXKSR5dh7aGVuQ5QwAFalvXLImeG0piKFSyFpm4tnkp45i7mkl3z24yMGhtdgzO042K0JUkS5I+ns888Tk69W5WPLMI77haVgmHYc9k6Dthus0r7WDoNK563\\\\\\\/Up3mIg6coI7lAUjCH1ZfYiySUVXogaMYsXO2fnlbpeevxzHrs1qxVGeZtdhnEI3Buq3KXG\\\\\\\/u0GQylVbD6Qi75ra6wzGykYzwkd8jI6wL\\\\\\\/CXk02rZbWYyPhrjpYBjRVl0vkTyW0SS\\\\\\\/NnSPdPCr7v+5dSpgGxZo5RK3vA3dGjd9YcXK5SWFAkqoHY6cjz5MBuMWtiUMGbNx3HPRrD1bD4evAIYu1g1ke\\\\\\\/zQYuxyP2PTG+V7EGxC2AE24asLs93xE3kfM0xaNNfz7njjI6WLX3JLLcRUxHvyjsIL9eH7tuNjPtZ3IwABPcsKLTf\\\\\\\/LQNF34\\\\\\\/YTFYNv8xZZiSd0nwqNV6lcS2h3cd7w5XoLtWu\\\\\\\/MPvVx5vQ7IyPw+tUbYa2a4pPX\\\\\\\/5+lLO3p42iI322jrUHPA2TPKfgRzmfJ0JHPlzXtntGQnCi4egOxZV0zpOaziUCANr872Hmy\\\\\\\/lWggE0Pa0SFc8ebw0WhUtYAhWxKO8eHI9c+q16y6DmnZjrfzkPo6q72mXy6hPTJ8tA8\\\\\\\/g1+xbZZg+319RxeGjcUfieuPCwFIGSTvNtifW3j01k2UC0hvlDrZ1W3nbV78\\\\\\\/GY9B5vTmnWcwhNL124JuPIgAnUEiizfehVEtfGu+zZh1iZ700spkS0YbdZtcoIt5BAiTBmL\\\\\\\/\\\\\\\/rWRDnx8RRF7t7c6K2LADmOUvzBYHY69FhlpJcvKhSOSmY1lONuBpw9n9Yy8bIU5wiuHg\\\\\\\/QcpovQrwcqHZZ0ihYXEtHN\\\\\\\/XM4VyXiVMLJjZ8NI6dj0qeUlDi10\\\\\\\/RdMbqhn50jfwKUuhh9RPL3QZqst7ynkpBy87GLGZXlX6rnN80+jPi0Fch65kxbtIzVY627bIcbUGR8zXn3euQVrF8znELj1I8Znf7q4Ffy\\\\\\\/uut3XaC48PTyGrcPjk9+p+ZIqjLlJNULkkCfQJgmJLVbzWDy6uDOnpELQkisN6wyMiU\\\\\\\/qYUml+xmA1XNu+scra1akb7EjZ\\\\\\\/+yHobdAuJgVRNZ6q6CtWmRlYjwFAwSHdWGzQaKno2g07qDjm+s62q9111SmnNJIijS9gawcYGV0pkz72WMMH3rdK5sQqvfvc\\\\\\\/8+zT\\\\\\\/b0kuuTTUV0s0zzLp3fFgVhn6gE6N5DhqiAlUKsAHkMSUA5OwwdJ8I0bmTBPx21OlO0hPGg6mnMbPrO2yPfp6xPRTEB7bhRY4a+ewHN1msoa0r7H1JUNnuaJHkQorbpUupYz2\\\\\\\/+66tJdWdvAodRF24\\\\\\\/RLiv0Iay4gH6QaCmZJXdUWnFkUHt3ZfuYHg853aRXK+1TO8uIzrAO0xW4KVO2dIBtRGTjmA5HX2WZUWnYPviJL8LaRKe0lXyySnDB0vuY6shdM+9YWNm2EtgRXjkc281IyxuK5mE0uXCpE+lKDNI39Cfn6obCQD8mJUjEVuSLe4JL0Fw+XEFCKzKkFxrE9NsBF3lN4PZF8g5Dqis3d85M7xHoqjIQCzTcZHBG\\\\\\\/cWEYx3YxRlzMbxDL0PxNKA17OivkqbZyHWnhhujYRtbjNw0YM6EwOrXdP7dVxc6yzbhlaIux1i8BuqqHhxqOnFY7uS4qzniGPJJeBLF6L9IRUy2JQ+44LkkpEIzSo1SfFskS7lbYMjexhTwXMCdLa8HCG20FnqT4PTiRSkK9DCeqp8dWUm1\\\\\\\/ZTw1\\\\\\\/IN7ugpXaWK1FTm9WbKVgDBkCT1JsGJeedTl\\\\\\\/DI5Lz8tC88r54Sf+nKf4TlNokHgRhsiGElef8vu\\\\\\\/XxiB9S9Zoe8tOCbt+igsrgdS384tmssphF95JWH+nC7ArxFQjWr2Gwe9CAZMl\\\\\\\/CUtEuccYg\\\\\\\/A\\\\\\\/YJ0jmatRm2VxFDKv7ppHTZtZXQe\\\\\\\/+\\\\\\\/cnV7bhGeuyUYGgjD7i8ZLBiPBJaT5Kufrh+FY7TDyrYTNar\\\\\\\/2S7hww+M6v2YJa5mWjlc+iP3SjJNIQmLthlNxFxCtYnuK0gJnc1MRRepgJ4LtrDJglAabRr8FglaStcKxX\\\\\\\/pJjwkAZ7oUTa\\\\\\\/7rjf4Cqr8ixyAwYWBa0+xKR+fQqzU8FdBDZK9qdRt8dpC97+oZ\\\\\\\/UGgvppQZi9OLWA9eQxMJj0b5eYXNWEKNPoYG6f\\\\\\\/jnXenzFgbdn+3jsxat0JPpYdjUQ3DvUfEZ6b8EIUTSwG3knli1ubjK\\\\\\\/gm5m2px2gvNWgQyBBWlWDZDtCNCLKmq\\\\\\\/N2ebte6cGuFwa4wbTmClcynavNh\\\\\\\/YMpvBRb2ZCtR7RN\\\\\\\/0Zs8xtBDg7rhol3i8S1b+5GJBeb9aSUdppBviFYuAMcyCCy5ml6madgZb3Ar0XBV\\\\\\\/DzM62TG1R526tO4BiE5aHYVaY34NjSCW9PVlyCHBa61UjqVXZHfxr2zNfAH+2O9bVYq0GcwGO\\\\\\\/W8lGAwh8vjUR5aTwbRLeMngu5WXYOxJVKqsijIu7Ek3aqPStqZ7Xj79b2nGgcxlS1mON9lJvDRXEEH\\\\\\\/TuJ604zHgcWs9nq5oEh9UiOmOvDlBiyETE0VLdDwjP\\\\\\\/mUWPkhmwKttkvhtvs1MfZYaf4FSuMnUUAOsH17XL9Pws5YTe9oSFl\\\\\\\/JF2qAFxScqtWTEogaHOutcgoCCQ8NwLsKGGHDEETJ6Cfy+U+y9oxbRXW6awnuqcclBm\\\\\\\/kgGZG+GHocDddf1u+\\\\\\\/UUqJlFMnxSP1ZJy8ioxpApmHUzzAkq1j0CTRLZUyfopKSYCRCWkqDAORgOmem9\\\\\\\/uVvk1p1bkrTRXQH6Bd1tibFb+409IsaKMo\\\\\\\/P1FOrHwmPKLGWWVTelIrpEQf2Apd3vg3nOfxPJpOH0CEHRTxiZysSyLpz9UZzRCdF+0IeTwrNJNHRBw25Oa9MIbJwkgjzkyapCE4JvfchDBq2SggEEwDmDmoCeNYD826BbzGLlLd3gDYwK0gthK3tXcn5fdiezRUPTKwvKiWqOISznVDabcAvqvEV\\\\\\\/hcKTl9FHk4yoDClnhMbL2VZrZ+lg9GpQbgrg2761rUFnuo++Z5d7NEAIqXqpLR3z7KjCcN+2vJqUfOxrwHn3npyFrl31l5HXi3g70hXYX0MlyaWltCJ1WYBNC54DK2PoTknptGpf5DCg4M+QY4P4RS8spAtv4QQcQstz5q5kUqaHggcvLun\\\\\\\/uP9VnYZL6nUtL6O7\\\\\\\/b+ucMbJ7y6IWlZgmKbGRHX4GxqxFnxu4lX7sEopKwdpXSfKXs+ziNWR+EMifRqBXW2lmu+xoGKffGRq1THnOAtNA8nu9P0SQIn0U7hweugLT8EDNYXRXXQ\\\\\\\/7BnMeQXHCPbMhLmgyeMKfBSnE9L6BD4hAHHAe2rROHc2JiG1bfrvT0yLU3CnxW9R5c8FZHXdtq73tLXg2GH4NxcaRmQB1adMqVwy\\\\\\\/3ZfJ4kW+rOXAzYYaJVQohZfNbRSh+\\\\\\\/Amd\\\\\\\/8\\\\\\\/Jb4VobZAz9EI9XzPxJBXgD\\\\\\\/TzjDeCB7nFdxnIG8RpQ9Ihq9l2cSA8nyHkAcoYbeAd7sJpJIVYAEV6MYFDWFHwP5n5r1H7CQ0IZ88F4NiPrsv7spFV2yctlirj1NtLW0MjsYzJBTue83kRbIdDiTwGu55Euv48ULz6ybqrGZ3oMp1ku2hx5fVd0V2\\\\\\\/KZa75oSP94uoVAyGcSOG8bcdtq3wfXNeeSxQeAsfjB7LSB5pYX5JYMyJukQQSN1qJNDD\\\\\\\/qNPS\\\\\\\/UKPQFuldxtR6mRF8bLm3YGxNJozR4UkhrvCfJvgLkhseYhhOK+\\\\\\\/9EcX8nWcfQvtysxGwgkfEWZHTr619RvCpJajvhFZUgc\\\\\\\/u5f+tsLKcl2xlhQ\\\\\\\/fZ\\\\\\\/kfQXP8ccXYM421DuF0VD4U+Lq0Y5OaNwFos0KXC70LwPObVGc1cUF7BP6r2B6SlI3uDN8Mp3vsrWVzrbvcscB3Hx4mRBqPwkHTiTY1AfKni2zGeqtVgryOLT8ucaTq8pX5\\\\\\\/BRcnxcjHUQQCow0w23p+\\\\\\\/AhegT+yBJDC\\\\\\\/SHb69lZaj0NBY\\\\\\\/JGmC63eM4kY2WEbR4Uj0i3WjvP72O2AcPOKLYhiVj3N6qSAkW82VvMEpkvuKxV5nwPtT6lRL7axfUMyrcaNYRpHUibFWrJwJqbvz+6TWqBUFJ9f3aGvWcOd3JBMdlOYqnQAxBDPJO99DtiPbEIrhmqfjYZBsx\\\\\\\/X+0sFOU1t77SS6q2WOiWF9hpde0Pm7u4je1xIhqitXutIUQkIqH\\\\\\\/TsPNMwvPEFNc2y59WcsX8KRVDlSc0luFFNs2KWMX+yZVwEKI1\\\\\\\/lcIt86\\\\\\\/S9AvzRPLW1X490tglHPYq\\\\\\\/XSMhbs9IzIsbxSFe6X2fUXsNrG7zA0tihzjkdLSsIcyJP0rZItaBMu2MwcIa3QcjrQgb6m0yf6OJ77k8gXNYXhfbD8Bd1I1JgOuWjhWFFYYGrB+xIcINgBVvIn1sNPcdvGNn9H4BVQyPd0QmOB+xStpNyHD9myXb7NjQgI4oOcvLsLzsE0pJLY\\\\\\\/br3Gje1LtCSOl9vB1g48rTxdQPIKm6EQ14T8UF\\\\\\\/ACnolZ5cIpr8eMBf7o\\\\\\\/pdbM1dV2VxQwRKipdvk0tY+9fIHiV\\\\\\\/RosJ\\\\\\\/OKPhYJRsRDzezBexhNsp0lu5opcci7Q+q5eX0IdHMi6e4phza6IL4vrUIYTJcalXwWABEp8kMeJ2kNtktlcY84MDVWXXMzYKrjamECDLzUk64byLZABZZOaKUkPysV3NJAea64VxyDRGFvmmqh5Ro075OTyeb+JF7bK91qgJxIvKkkVwl6JIJg9YFfliFpPLPp7btZkdDjg49XVvEhadyILTyi6J4rkPOhPbQ82Nwwsk72w\\\\\\\/VHDSdBAlEji\\\\\\\/Q+S8jC3jZTbdTwuq4Km0MnmY2KBR0CsBPJNao7VVhhkxvOx5IywrVmWO1DZ2quJ9VmyoAF+BuobljZHV9msFvaGcobKs6Vk8WapZYMBPQlUWcBjvNXgkfneACYSQcf0j78IgGvU7pXneB4A+CTCtFcP2DQ48F14Dsa7Zcnwan4TRsRy5vsVQl9UHt4eKpoulJsxgfZGvPjMfjW6mcHjV0TLuoiiEkaV5LSNUVlQGvnYINAdvhYXgWYBpeCBETjmDYGm\\\\\\\/o1s1eRzMl3dq9T\\\u0022,\\\u0022iv\\\u0022:\\\u0022OGdpQZxWVeYMbaqIK1Z97Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d7de44dc9d4895fb93a60c33a00a62b0cc38cbd7135dc0028a6e6f2a04b0f760\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ROPx3GZS9nDPl6K4PYIJ5wv0S36jTIyBDvaFFWHomtrp\\\\\\\/sgln8va4ahjhAw\\\\\\\/5ge88Rl23jJ+3qPgUK+YJ4qvM1+TBAS7W3Og9JaWt6Ky4k1KkzlKL4srhIsuk8cXK83CIwcA0dHlixh3GVmJUPLXISsNkmAonlqb1wWZEOL0Y7iESgMPamassZqQNkJ7Nh6A\\\\\\\/5tVaOP58+pRusyE61fTSX5B81NRKvRqfnmHUILl6PedP5852GlBuhmbS+ola7LV72n7uwN+PAtXqcJgD75OoE\\\\\\\/mz6hpxm0yZGKrr9nlpOaQNccM6yes33bzBIY4+ltLCSFy0N+05QjQ9lpeVlDQYprxcBsqwxmV9NuWI+\\\\\\\/YfocuxjPk0OGMy5n1hv\\\\\\\/qJvSYbw6GieCzQZ4KmLlBM8ls2IQTrda64JAPlJ42TbVoHrCP3IrWBKLrud2jz6Q5BUV0M0wx65MF4U+t7xbOQhgM2SrNp0PeKzAVDnxfSOytlQitf8AVv8va5t0bbXc4cFZEwN4phxIkIEoNEbDFywL04PpKEV6sMooJ8zJXu3VsNus5CVrhUOI00Je0WCS2S7jADf63JO\\\\\\\/ZfTZdl0uusjXeS+UcQBLFgKyOPtwvJ4z2TFUg85lTUlH8JHjxAWzpo04+uensWBmqV2xkRRdZqQqub42L8hDU0BEyn3XQduyJ5PUPBoOpmhO5CUgNh1oUrfr8RDYrWt6HWHn9Mgw0IGasxqMeTNTRR5GYfkv7BMGSDlI5sjtJ8OIS5mW+kEURUIZFXfs\\\\\\\/j9wqKfgiNSYsFcV4qbb\\\\\\\/Rsgao0uMfagv7GVCB3mG+Gpgz5Pd8mAYzqZSGXXYcW9rP1CFv9\\\\\\\/R\\\\\\\/HO87RPOa+02pwG1F5An8RoJ+rBw9S3ch1+mmtGDgdz\\\\\\\/J7fW1\\\\\\\/0dMc5\\\\\\\/z6bW9ISzwrvZp6NLpqZXOCkgvNEd94FrK38M9CrpT8hW0XoTVU8Dv86r1v7+jetHJw6naG9YAfTHwfVPMjz9XCD6DWu\\\\\\\/eqARpDt2uCdt8jKRWe9H5NhsjAAaG9\\\\\\\/U8dih1wpV9QrAQDJ4SkqkaXxW1Qd91xIbDhnXhmuc7daBML5JiUFSzB7m2YAociqCQ4qkgLfREKZXo1VtIKeGHKJSPl5ZWujmXMUpluN9e4hhly4hRBq003xOFMHQoCptgrFC5kmInIpaxlxbrKx88ECYaYu\\\\\\\/U5W+eOcwFcHAqHXMU2CsA5frnEIST3ydeITEV\\\\\\\/1WgaQuguguzQakil5JMQtzP8qBN6wSZD4YBrEdMLKnnliSFs0BhmPfca08WIuGhihW4m+6dZW74LndRjcR1NLdERXajcSJy+5m5LtBkomNk2LJJ6NdQ8bJv22vzuS69MUC+0SjQPmyRFi8PN95Ro6QX7k1Suv3GR54EpFK7fZS53nHdeqAUNHsVF3sOo2a1kRgkN2Q2Jx6sgBg5F1bWcUHQ1lR\\\\\\\/aLZl15ut85jSBplYLO0CdaZJk4\\\\\\\/98eCbinrgvqCY+6TNdefnykznorm7bBwSVN0Ml5sRUsp1y9OL85u88CYSIS8lmCTmjyTBWwxmvWb0pC9unvvXxdI0zCpUQzTc9xqGWlO6IsoEUsoRAJTQyP0OlaSSZVIxnzQhuixYlcCDll2sHbQDn8gcV2EWVYUp7k3NNiKx67AFPCWT5CVx0iEvYrH04JcidxSVSsmJGr2iyGU5PZWG6Dx56ab77eOPmlNRpITMxuLd4PhmLKcHj2P84nqzDA8tkSd4Aq8jJ5vEVIYgrxGguQhlPsB0T1sDZoNJZUuA8N0AV2soOe\\\\\\\/0DpI3C44jGLlEFDoiZAoDRjZiU+Dg5M8Raq7vWczdgzRLGWm7KnYRGPceMG3NOgWQro+XngbDnjxFKuC+Yu8WRp\\\\\\\/gV86pLQXPm6\\\\\\\/6frt4hR6RUZh8TtAtxTinSmWPcPsLZHb2u3wQqBUENce\\\\\\\/ZK5+9GCpe0qb\\\\\\\/OEuLIeJCV7A7BljZM+d1MZU2fv8YzgHXO2t52NHXvk\\\\\\\/MEerVIfyxvMXMDmcAk4nuOnMx48Ta75ujvamf3Kj7SokBjkjuDgjIQ=\\\u0022,\\\u0022iv\\\u0022:\\\u00227ShK7j7KDtwAh33Qzz+\\\\\\\/nQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002210e6d450413c243021c18294a5752843a43593ec9c33da0ce034a79335ac0772\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483837 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1377" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QjEOPrSNEefqTbbxst6wkod2l4cskTzos0EvgV4xbxE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1377" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-39icv67FEOKEpetAeIE4Y_mRZVMh8l8mHHwWRO4A3Qo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1365" title="A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1385" title="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483837" id="node483837" data-pisa="ggg;7/4/1377" data-pisa-master="ggg;g3.117.040717" data-apath="/ggg/7/4/1377.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hsin-Yuan Tsai</span>, <span class="highwire-citation-author" data-delta="1">Oswald Matika</span>, <span class="highwire-citation-author" data-delta="2">Stefan McKinnon Edwards</span>, <span class="highwire-citation-author" data-delta="3">Roberto Antolín–Sánchez</span>, <span class="highwire-citation-author" data-delta="4">Alastair Hamilton</span>, <span class="highwire-citation-author" data-delta="5">Derrick R. Guy</span>, <span class="highwire-citation-author" data-delta="6">Alan E. Tinch</span>, <span class="highwire-citation-author" data-delta="7">Karim Gharbi</span>, <span class="highwire-citation-author" data-delta="8">Michael J. Stear</span>, <span class="highwire-citation-author" data-delta="9">John B. Taggart</span>, <span class="highwire-citation-author" data-delta="10">James E. Bron</span>, <span class="highwire-citation-author" data-delta="11">John M. Hickey</span> and <span class="highwire-citation-author" data-delta="12">Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1377-1383;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040717 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483837" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hsin-Yuan Tsai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-Yuan%2BTsai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsai%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-Yuan%2BTsai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oswald Matika </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oswald%2BMatika%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matika%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOswald%2BMatika%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stefan McKinnon Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BMcKinnon%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BMcKinnon%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Roberto Antolín–Sánchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BAntol%C3%ADn%E2%80%93S%C3%A1nchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Antol%C3%ADn%E2%80%93S%C3%A1nchez%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BAntol%25C3%25ADn%25E2%2580%2593S%25C3%25A1nchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alastair Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alastair%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlastair%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Derrick R. Guy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derrick%2BR.%2BGuy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guy%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerrick%2BR.%2BGuy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alan E. Tinch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BTinch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinch%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BTinch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Karim Gharbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3JT, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karim%2BGharbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gharbi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarim%2BGharbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael J. Stear </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute of Biodiversity, Animal Health & Comparative Medicine, University of Glasgow, G61 1QH, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BStear%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stear%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BStear%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">John B. Taggart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BB.%2BTaggart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taggart%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BB.%2BTaggart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">James E. Bron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BBron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bron%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BBron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ross.houston@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ross.houston@roslin.ed.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/4/1377" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483837" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483837/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1377.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483837" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483837/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1377.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483837" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483837/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1377.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483837" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483837/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/4/1377.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483837" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483837/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genomic selection uses genome-wide marker information to predict breeding values for traits of economic interest, and is more accurate than pedigree-based methods. The development of high density SNP arrays for Atlantic salmon has enabled genomic selection in selective breeding programs, alongside high-resolution association mapping of the genetic basis of complex traits. However, in sibling testing schemes typical of salmon breeding programs, trait records are available on many thousands of fish with close relationships to the selection candidates. Therefore, routine high density SNP genotyping may be prohibitively expensive. One means to reducing genotyping cost is the use of genotype imputation, where selected key animals (<em>e.g.</em>, breeding program parents) are genotyped at high density, and the majority of individuals (<em>e.g.</em>, performance tested fish and selection candidates) are genotyped at much lower density, followed by imputation to high density. The main objectives of the current study were to assess the feasibility and accuracy of genotype imputation in the context of a salmon breeding program. The specific aims were: (i) to measure the accuracy of genotype imputation using medium (25 K) and high (78 K) density mapped SNP panels, by masking varying proportions of the genotypes and assessing the correlation between the imputed genotypes and the true genotypes; and (ii) to assess the efficacy of imputed genotype data in genomic prediction of key performance traits (sea lice resistance and body weight). Imputation accuracies of up to 0.90 were observed using the simple two-generation pedigree dataset, and moderately high accuracy (0.83) was possible even with very low density SNP data (∼250 SNPs). The performance of genomic prediction using imputed genotype data was comparable to using true genotype data, and both were superior to pedigree-based prediction. These results demonstrate that the genotype imputation approach used in this study can provide a cost-effective method for generating robust genome-wide SNP data for genomic prediction in Atlantic salmon. Genotype imputation approaches are likely to form a critical component of cost-efficient genomic selection programs to improve economically important traits in aquaculture.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/aquaculture" class="hw-term hw-article-keyword hw-article-keyword-aquaculture" rel="nofollow">aquaculture</a></li><li class="kwd"><a href="/keyword/disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-disease-resistance" rel="nofollow">disease resistance</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-5">Modern genetic studies typically require high density genome-wide SNPs for mapping variants underlying complex traits, or predicting breeding values from genotype data. Genomic selection has transformed terrestrial and aquatic animal breeding programs, and relies on capturing accurate realized genetic relationships between animals, and linkage disequilibrium (LD) between SNP markers and causative mutations underlying economically important traits (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2013</a>). However, genotyping the large numbers of individuals required for accurate genomic predictions using high density SNP platforms is expensive, often prohibitively so. In turn, this can limit both the number of phenotyped individuals with high density genotype data in the training set used to derive the genomic prediction equation, and the number of selection candidates that can be evaluated using that equation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Habier <em>et al.</em> 2009</a>). The cost of genotyping is largely dependent on marker density, with low density panels being considerably cheaper than high density ones. Therefore, a targeted high and low density genotyping strategy in pedigreed animals, combined with genotype imputation, is an attractive option to improve the cost-efficiency of high resolution genomic studies, and application of genomic selection in aquaculture breeding programs.</p><p id="p-6">Genotype imputation involves high density genotyping of certain key individuals, while the majority of individuals are screened only for a small subset of these markers (a lower density SNP panel). These genotype data are then used to impute the nongenotyped markers for the individuals genotyped at low density (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hickey <em>et al.</em> 2012a</a>). Imputation approaches have been successfully and widely applied in breeding programs for several livestock and crop species (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> 2012</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hickey <em>et al.</em> 2012a</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pausch <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Moghaddar <em>et al.</em> 2015</a>). The accuracy of imputation is affected by several factors, including population structure, the number of SNPs in the imputation panel, the level of relatedness between reference and test data, effective population size, the inherent accuracy of the method used for imputation, and the degree to which markers are correctly ordered along the genome map (<em>e.g.</em>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> 2012</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Hickey <em>et al.</em> 2012a</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hozé <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Uemoto <em>et al.</em> 2015</a>). The methods applied for genotype imputation can broadly be split into two categories: (i) population approaches such as Beagle (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2016</a>), MaCH (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2010</a>) and IMPUTE2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Howie <em>et al.</em> 2009</a>), which utilize linkage disequilibrium (LD) between markers, and (ii) pedigree-based approaches such as PHASEBOOK (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Druet and Georges 2010</a>), findhap (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">VanRaden <em>et al.</em> 2011</a>), and AlphaImpute (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012b</a>), which harness genetic relationships (pedigree) in addition to LD. The latter approaches are suitable for data originating from typical livestock and aquaculture breeding programs, where large numbers of pedigreed individuals with genotype and phenotype data are routinely available.</p><p id="p-7">While research into imputation methods and their application to breeding programs has been extensive for livestock and crop species, they have not yet been widely tested in aquaculture species (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kijas <em>et al.</em> 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> 2016a</a>). In part, this is due to the previous lack of genomic resources (<em>e.g.</em>, SNP genotyping arrays and reference genome sequences) for many aquaculture species (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yáñez <em>et al.</em> 2014</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2015</a>). In recent years, high density SNP arrays have been developed for several aquatic species, including salmonid species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Houston <em>et al.</em> 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Palti <em>et al.</em> 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yáñez <em>et al.</em> 2016</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lien <em>et al.</em> 2016</a>). These SNP arrays, alongside custom lower density SNP panels, have been successfully applied to enable genomic selection for economically important traits in salmonid breeding programs (<em>e.g.</em>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ødegård <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">2016a</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vallejo <em>et al.</em> 2016</a>). An example target trait is resistance to sea lice, since these parasites are the primary constraint to production and result in enormous economic, welfare, and environmental cost (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gharbi <em>et al.</em> 2015</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> 2016a</a>). Genomic prediction of sea lice resistance improves selection accuracy by 27% compared to traditional pedigree-based approaches, highlighting the utility of this technique in aquaculture breeding (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> 2016a</a>). In parallel, a high quality reference genome assembly has been developed for Atlantic salmon (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lien <em>et al.</em> 2016</a>), and the SNP arrays have been integrated with this recent assembly (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Lien <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tsai <em>et al.</em> 2016b</a>). This combination of tools now facilitates the study and use of genotype imputation approaches to improve genomic selection. The potential of genotype imputation in salmon was highlighted in a recent study by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kijas <em>et al.</em> (2016)</a>, who imputed from low density (0.5–5 K) up to high density (78 K) with high accuracy (0.89–0.97) based on a multi-generation reference population.</p><p id="p-8">The primary goal of the current study was to evaluate the utility of genotype imputation in a population of Atlantic salmon from a commercial breeding program, for which high density genotype information was available on parents and offspring (two generations only). A large proportion of SNP genotypes were masked in the offspring, resulting in “pseudo” low density panels. The correlation between true genotypes and imputed genotypes was then assessed for the masked SNPs under various scenarios. Finally, the imputed SNP data were used in genomic prediction for key economic traits, and prediction accuracy was assessed relative to pedigree-based approaches, and genomic approaches using the full genotype dataset.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals and phenotypes</h3><p id="p-9">The genotype and trait data used in the current study were from 624 Atlantic salmon postsmolts, which was a sample from a specific year group subset of a large commercial breeding program (Landcatch Natural Selection Ltd., UK) hatched in the spring of 2008. The samples comprised 59 nuclear families, derived from 30 sires and 59 dams. At ∼1 yr posthatching, juvenile fish were challenged with sea lice (<em>L. salmonis</em>) copepods as described in <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gharbi <em>et al.</em> (2015)</a> and <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> (2016a)</a>. Briefly, all fish were challenged in a single tank with a dose of 96 copepod larvae per fish, and monitored until lice had moulted into chalimus I (7 d postchallenge), at which stage fish were measured for body weight (grams), and number of lice attached to the fish (lice were identified by stereo-microscopic inspection, Olympus SZ-40). Therefore, the two phenotypes used in the current study were sea lice counts and body weight, as described in <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> (2016a)</a>. Both these traits have been shown in previous studies to be heritable, but with a predominantly polygenic genetic architecture (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ødegård <em>et al.</em> 2014</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">2016a</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Correa <em>et al.</em> 2016</a>). The sea lice count data were transformed to account for a positively skewed distribution, using the approach of <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gjerde <em>et al.</em> (2011)</a>, as described previously for these data (<a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> 2016a</a>).</p><p id="p-10">The pedigrees of the fish were identified using PIT-tagging, and an adipose fin clip of each fish was collected and stored in ethanol for genomic DNA extraction. The challenge experiment was performed by the Marine Environmental Research Laboratory (Machrihanish, UK) under approval of the ethics review committee of the University of Stirling (Stirling, UK), and according to Home Office license requirements. All animals were reared in accordance with relevant national and European Union (EU) legislation concerning health and welfare. Landcatch are accredited participants in the Royal Society for the Prevention of Cruelty to Animals (RSPCA) Freedom Foods Standard, the Scottish Salmon Producers Organization Code of Good Practice, and the EU Code-EFABAR Code of Good Practice for Farm Animal Breeding and Reproduction Organizations.</p></div><div id="sec-3" class="subsection"><h3>SNP marker genotyping</h3><p id="p-11">All samples were genotyped using the Affymetrix Axiom 132 K Atlantic salmon SNP chip developed by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Houston <em>et al.</em> (2014)</a>, as described in <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> (2015)</a>. The quality control measures resulted in the exclusion of SNPs with Mendelian errors, minor allele frequency (MAF) &lt;0.05, and proportion of individuals with missing genotypes &gt;0.03. The MAF of SNPs were calculated using Plink 1.9 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Purcell <em>et al.</em> 2007</a>). SNPs with a known and unique chromosome position on the Atlantic salmon reference genome [GenBank accession GCA_000233375.4, (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Lien <em>et al.</em> 2016</a>)] were retained for analysis. After these filtering steps, 78,362 (78 K) SNPs were retained for the high density SNP panel (hereafter “HD SNP panel”). A subset of these SNPs [25,634 (25 K)] formed part of a second medium density Affymetrix Axiom array described in <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> (2015)</a>, and these formed a medium density SNP panel (hereafter “MD SNP panel”). The details of the SNPs in the MD SNP panel and the HD SNP panel are provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS1.txt">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS2.txt">File S2</a>, respectively. As a result, all parents and offspring samples had genotypes for both SNP panels (the genotype data are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS3.txt">File S3</a>), and these formed the basis of the imputation analyses.</p></div><div id="sec-4" class="subsection"><h3>Genotype imputation analyses</h3><div id="sec-5" class="subsection"><h4>Definition of high and low density SNP panels:</h4><p id="p-12">To test genotype imputation accuracy, a number of test scenarios were established as outlined below and represented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. While all individuals were genotyped for both the HD SNP panel and the MD SNP panel, some individuals had a set proportion of genotypes masked to mimic the use of lower density SNP panels (hereafter “LD SNP panels”) data for these individuals <em>in silico</em>. For the individuals chosen to have LD SNP panel data, two settings determining the content of the LD SNP panel were applied by masking either 90 or 99% of the markers. The remaining SNPs (10 or 1% of all SNPs, respectively) were selected to be evenly spaced throughout the genome, based on physical distance according to the Atlantic salmon reference genome assembly [GenBank accession GCA_000233375.4 (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Lien <em>et al.</em> 2016</a>)]. Therefore, since the LD SNP panels were created based on both the HD SNP panel (78 K SNPs) and the MD SNP panel (25 K SNPs), the LD SNP panels corresponded to SNP densities of ∼7836 SNPs, 784 SNPs, 2563 SNPs, and 256 SNPs, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484042/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484042/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The SNP genotype densities used for the imputation analyses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>Proportion of offspring genotyping for LD SNP panels:</h4><p id="p-14">For all the marker density settings described above, the parents had either HD or MD SNP panel data, and two scenarios were evaluated, where either (i) all offspring had LD SNP panel data, or (ii) 75% of offspring had LD SNP panel data, and the remaining 25% had MD or HD SNP panel data. The latter scenario was applied to measure the impact of including a proportion of offspring with complete genotype information on the phasing and imputation accuracy. The 75% of offspring chosen for LD panel data in scenario (ii) were evenly distributed across all nuclear families in the population.</p></div><div id="sec-7" class="subsection"><h4>Evaluation of genotype imputation accuracy:</h4><p id="p-15">The genotype imputation analyses were performed using the AlphaImpute v1.3.2 software (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012b</a>) following the standard procedures, using the “HMM” option (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Antolín <em>et al.</em> 2017</a>), 10 processor cores, and 5 “InternalIterations.” The “CoreAndTailLengths” and “CoreLengths” were set according to the length of corresponding chromosomes. The imputation accuracy was calculated as the correlation (<em>r</em>) between the allele dosage of the true genotype and the most likely imputed genotype, averaged across all SNPs and all animals.</p></div></div><div id="sec-8" class="subsection"><h3>Genomic prediction accuracy using fivefold cross validation</h3><p id="p-16">Due to the fact that medium SNP densities (between 5 K and 20 K SNPs) are sufficient for achieving maximum genomic prediction accuracy in the current experimental set up (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-39-9" class="xref-bibr" href="#ref-39">2016a</a>), only the MD SNP panel (25 K SNPs) was evaluated for testing genomic prediction using imputed genotypes. Genomic breeding values were estimated using best linear unbiased prediction using the genomic relationship matrix to model the polygenic relationship between the animals (GBLUP) using ASReml 3.0 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gilmour <em>et al.</em> 2014</a>). The following animal model was employed:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-2.gif"/></noscript></span></span> is a vector of observed phenotypes, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-3.gif"/></noscript></span></span> is the vector of fixed effects (sex), <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-4.gif"/></noscript></span></span> is a vector of additive genetic effects distributed as ∼<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-5.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-6.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-7.gif"/></noscript></span></span> is the additive (genetic) variance, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-9.gif"/></noscript></span></span> are the genomic and pedigree relationship matrices, respectively. <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-11.gif"/></noscript></span></span> are the corresponding incidence matrices for fixed and additive effects, respectively, and <strong>e</strong> is a vector of residuals. The genomic relationship matrix was constructed using the method of <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">VanRaden (2008)</a>, and then inverted by applying the standard R function “solve” (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Core Team 2016</a>).</p><p id="p-17">To test the accuracy of genomic and pedigree-based prediction, a cross-validation approach was applied (as described in <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-39-10" class="xref-bibr" href="#ref-39">2016a</a>). Briefly, the individuals with imputed genotypes (progeny) were divided into training (80% individuals) and validation (20% individuals) sets. This process was repeated five times, resulting in nonoverlapping validation sets. The lice count and body weight phenotypes were masked in the five validation sets, and then predicted from the genomic breeding values. The prediction accuracy was measured in the validation sets as the correlation between the genomic breeding values, and the trait values divided by the square root of the heritability<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1377/embed/mml-math-12.gif"/></noscript></span></span> The fivefold cross-validation analyses were performed for each level of genotype masking and imputation. In all cases, the LD SNP panels were imputed to the MD SNP panel (25 K SNPs), and this imputed genotype data set was used as the input for the GBLUP calculations.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-18">The data used in this study are available as supplementary files. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS1.txt">File S1</a> contains details of the SNPs used for the medium density (25 K) SNP platform. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS2.txt">File S2</a> contains details of the SNPs used for the high density (78 K) SNP platform. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS3.txt">File S3</a> contains the family and phenotype data used in the analysis. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1/FileS4.zip">File S4</a> contains the genotype data used in the analysis.</p></div></div><div class="section results-discussion" id="sec-10"><h2 class="">Results and Discussion</h2><div id="sec-11" class="subsection"><h3>Accuracy of imputation</h3><div id="sec-12" class="subsection"><h4>Comparison of high and medium density SNP panels:</h4><p id="p-19">The accuracy of imputation of high density genotypes was assessed as the correlation between the imputed genotypes and the true genotypes in the offspring, where varying proportions of genotypes had been masked. The imputation accuracy ranged from 0.62 to 0.85 for the MD SNP panel (25 K), and from 0.76 to 0.90 for the HD SNP panel (78 K). The higher imputation accuracy based on the HD panel compared to the MD panel may be explained by more accurate resolution of haplotypes, especially for short chromosome segments. Higher imputation accuracy with increased marker density has been shown previously in simulated and experimental populations in livestock (<em>e.g.</em> <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em> 2012</a>) and crops (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Hickey <em>et al.</em> 2012a</a>).</p></div><div id="sec-13" class="subsection"><h4>Comparison with previous studies:</h4><p id="p-20">The imputation accuracies achieved in the present study (ranging from 0.62 to 0.90) were generally lower that that achieved in previous studies in farmed animals and crops (<em>e.g.</em>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012b</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Segelke <em>et al.</em> 2012</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pausch <em>et al.</em> 2013</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Moghaddar <em>et al.</em> 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Uemoto <em>et al.</em> 2015</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kijas <em>et al.</em> 2016</a>). It is possible that the modest sample size of the current study (<em>n</em> = 624) may have been a limiting factor in determining the imputation accuracy. In addition, the lack of genotyped ancestral generations may have impaired the phasing of the parental haplotypes for whole chromosomes. When genotype imputation is employed in livestock populations, multiple generations of ancestral genotyped individuals, and pedigree information are typically available. Likewise, in the study of <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Kijas <em>et al.</em> (2016)</a>, multiple generations of genotyped individuals were available for the Tasmanian salmon breeding population. These genotyped multi-generation pedigrees are more amenable to resolving the phase of whole chromosome haplotypes, and therefore result in more accurate genotype imputation.</p></div><div id="sec-14" class="subsection"><h4>Relationship between MAF and imputation accuracy:</h4><p id="p-21">The relationship between SNP MAF and imputation accuracy was assessed under four scenarios using the MD SNP panel, varying the density of the LD SNP panel (either 90 or 99% of SNPs masked to mimic LD panels), and the proportion of offspring designated as being genotyped for the LD panels (100 or 75%). Under these scenarios, the correlation between true and imputed genotypes increased with higher MAF (and it should be noted that SNPs with MAF &lt;0.05 had already been filtered out prior to this analysis; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This relationship between MAF and imputation accuracy is consistent with previous studies, where accuracy was higher for common variants (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ma <em>et al.</em> 2013</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Pausch <em>et al.</em> 2013</a>). It is anticipated that the imputation accuracy for rare alleles (and rare haplotypes) will improve with increased sample size, due to the increased frequency of observing these alleles, and with a multi-generation pedigree structure amenable to resolving whole chromosome haplotypes. As expected, including a higher number of SNPs in the LD panel (<em>i.e.</em>, 90% masked) resulted in higher imputation accuracy at all MAF (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effect of minor allele frequency on imputation accuracy. The plot shows the imputation accuracy for the MD SNP panel with the two different LD SNP panel densities (90% SNPs masked =&#xA0;2563 SNPs; 99% SNPs masked =&#xA0;256 SNPs), plotted against the minor allele frequency of the SNPs using a local regression fit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1369670945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The effect of minor allele frequency on imputation accuracy. The plot shows the imputation accuracy for the MD SNP panel with the two different LD SNP panel densities (90% SNPs masked =&#xA0;2563 SNPs; 99% SNPs masked =&#xA0;256 SNPs), plotted against the minor allele frequency of the SNPs using a local regression fit.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1377/F1.medium.gif" width="398" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/4/1377/F1.medium.gif" width="398" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483966" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">The effect of minor allele frequency on imputation accuracy. The plot shows the imputation accuracy for the MD SNP panel with the two different LD SNP panel densities (90% SNPs masked = 2563 SNPs; 99% SNPs masked = 256 SNPs), plotted against the minor allele frequency of the SNPs using a local regression fit.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483932/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483932/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of genotype imputation accuracy</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Variation in imputation accuracy across animals:</h4><p id="p-24">The mean imputation accuracy using the HD SNP panel when all offspring were designated as being genotyped for the LD panels was 0.76, increasing to 0.85 when only 75% of the offspring were genotyped at LD (and 25% genotyped at HD). The equivalent figures for the MD SNP panel were 0.62 and 0.76, respectively. However, there was a large degree of variability of imputation accuracy across individuals, demonstrating a negatively skewed distribution (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). While the majority of the offspring with imputed genotypes had accuracy values in the range of 0.7–0.9, there was a proportion with much lower accuracy, which reduced the mean accuracy values. This phenomenon has also been reported in studies of imputation in livestock (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2012b</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Moghaddar <em>et al.</em> 2015</a>), and may arise because certain individual parents have inferior definition of whole chromosome haplotypes to others. Removal of individuals or SNPs with the least accurate imputation values would increase overall average imputation accuracy, but was not performed in the current study.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation of imputation accuracies across individual animals in MD SNP panel. The histograms show bins of imputation accuracy (x-axis), and the number of animals in those bins (y-axis) for the two different LD SNP panel densities (90% SNPs masked =&#xA0;2563 SNPs; 99% SNPs masked = 256&#xA0;SNPs)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1369670945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation of imputation accuracies across individual animals in MD SNP panel. The histograms show bins of imputation accuracy (&lt;em&gt;x&lt;/em&gt;-axis), and the number of animals in those bins (&lt;em&gt;y&lt;/em&gt;-axis) for the two different LD SNP panel densities (90% SNPs masked =&#xA0;2563 SNPs; 99% SNPs masked = 256&#xA0;SNPs).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1377/F2.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/4/1377/F2.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Variation of imputation accuracies across individual animals in MD SNP panel. The histograms show bins of imputation accuracy (<em>x</em>-axis), and the number of animals in those bins (<em>y</em>-axis) for the two different LD SNP panel densities (90% SNPs masked = 2563 SNPs; 99% SNPs masked = 256 SNPs).</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-16" class="subsection"><h3>Accuracy of genomic prediction using imputed data</h3><p id="p-26">The second major aim of the current study was to assess the utility of imputed genotype data for genomic prediction in a commercial salmon breeding program. From a practical standpoint, genotyping parents at medium or high density, combined with offspring at lower density with imputation, has potential for major improvement in cost-effectiveness of genomic selection in aquaculture. For the genomic prediction analyses, only imputed data from the MD SNP panel (∼25 K mapped, ordered SNPs) was tested, based on previous studies which suggested that between 5 and 20 K SNPs is adequate for maximum prediction accuracy in a typical salmon breeding set up (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Ødegård <em>et al.</em> 2014</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-39-11" class="xref-bibr" href="#ref-39">2016a</a>). The imputed genotypes used for genomic prediction were retrieved from the scenario where 75% of offspring were assumed genotyped at LD (and 25% at MD), and the LD SNP panel was created by masking 99% of the MD SNP genotypes (akin to a 256 SNP panel; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The prediction accuracies using imputed genotypes were marginally lower than tests using true genotypes (0.58 <em>vs.</em> 0.60 for lice resistance, 0.67 <em>vs.</em> 0.69 for body weight), but substantially higher than pedigree-based method for both phenotypes (0.48 and 0.58 for lice resistance and body weight, respectively) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Taking the pedigree-based breeding value prediction as the baseline, prediction accuracy was improved by nearly 25% when using 25 K true genotypes, and by 21% when using imputed genotypes for the traits of lice resistance (25 K imputation with 75% LD) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This highlights the potential of imputation for cost-effective genomic prediction for the traits studied, although it is important to note that the value of genotype imputation may vary according to the genetic architecture of the trait of interest.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breeding value prediction accuracies for (A) sea lice resistance and (B) body weight calculated using (i) the pedigree (PBLUP), compared to genomic prediction using (ii) the 256 SNP LD panel only, (iii) the 256 SNP LD panel imputed to 25 K SNPs (with all parents and 25% offspring genotyped at MD SNP panel), and (iv) the true genotypes for the 25 K MD SNP panel. For comparison, the accuracy of breeding value prediction under scenario (iv) is shown by the blue dashed line, and the corresponding accuracy under scenario (i) with the red dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1369670945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Breeding value prediction accuracies for (A) sea lice resistance and (B) body weight calculated using (i) the pedigree (PBLUP), compared to genomic prediction using (ii) the 256 SNP LD panel only, (iii) the 256 SNP LD panel imputed to 25 K SNPs (with all parents and 25% offspring genotyped at MD SNP panel), and (iv) the true genotypes for the 25 K MD SNP panel. For comparison, the accuracy of breeding value prediction under scenario (iv) is shown by the blue dashed line, and the corresponding accuracy under scenario (i) with the red dashed line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1377/F3.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/4/1377/F3.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1377/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483868" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Breeding value prediction accuracies for (A) sea lice resistance and (B) body weight calculated using (i) the pedigree (PBLUP), compared to genomic prediction using (ii) the 256 SNP LD panel only, (iii) the 256 SNP LD panel imputed to 25 K SNPs (with all parents and 25% offspring genotyped at MD SNP panel), and (iv) the true genotypes for the 25 K MD SNP panel. For comparison, the accuracy of breeding value prediction under scenario (iv) is shown by the blue dashed line, and the corresponding accuracy under scenario (i) with the red dashed line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The genomic prediction results are consistent with previous studies of imputation in livestock species, where accuracies using imputed genotypes were slightly lower than those using true genotypes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berry and Kearney 2011</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Segelke <em>et al.</em> 2012</a>). Genomic prediction accuracy using just the LD SNP panel (<em>i.e.</em>, 256 SNPs) was also compared to prediction accuracy using the LD SNP panel plus imputation. For the trait of sea lice resistance, genomic prediction using 256 SNPs was inferior to pedigree-based prediction (accuracy ∼0.40 <em>vs.</em> 0.48), while 256 SNPs with imputation increased the accuracy to 0.58 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). For body weight, a similar profile was observed, where pedigree-based prediction accuracy was 0.58, and increased to 0.68 with 256 SNPs and imputation, <em>vs.</em> 0.70 with the full 25 K true genotypes. Interestingly, genomic prediction accuracy was generally higher for the trait of body weight compared to sea lice resistance. The heritability of body weight was substantially higher than lice resistance (0.50 <em>vs.</em> 0.22; <a id="xref-ref-39-12" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> 2016a</a>), which may be expected to result in increased accuracy of genomic prediction (<em>e.g.</em>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sonesson and Meuwissen 2009</a>).</p><p id="p-29">When considering targeted SNP assay genotyping panels (as opposed to direct genotyping by sequencing approaches; discussed briefly below), there is a nonlinear relationship between SNP panel density and cost per sample. This relationship depends on several factors, including the technology, the company, and the number of genotyped samples. However, in general terms, SNP densities of &lt;∼3000 SNPs can be genotyped most cost-effectively using individual targeted assays, for example using KASP technology (LGC Genomics, UK), or targeted genotyping by sequencing (<em>e.g.</em>, Affymetrix Eureka technology), while SNP densities &gt;∼3000 SNPs can be genotyped most cost-effectively using SNP arrays. The cost per sample of genotyping for a medium density SNP chip is several fold higher than the cost of genotyping for a 256 SNP panel. Assuming an approximate price for the former of £40 per sample, and an approximate price for the latter of £5 per sample, the total cost of the genotyping for genomic prediction using the imputation described herein is ∼60% lower than genotyping all samples at MD. Furthermore, the efficacy of genotype imputation (and therefore genomic prediction using imputed data) is likely to increase as high density genotype data are collected on additional generations, especially for grandparents of the population where imputation is being applied. The current study used SNP array genotyping data as the basis for imputation, but genotyping by sequencing approaches such as RAD-Seq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>) have been applied for genomic selection in aquaculture (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dou <em>et al.</em> 2016</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Vallejo <em>et al.</em> 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Palaiokostas <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Robledo <em>et al.</em> 2017</a>), and the benefits of a combined high and low density genotyping strategy with imputation may also be relevant to these genotyping techniques.</p><p id="p-30">The focus of this study was to test the possibility of using genotype imputation to improve the cost-efficiency of genomic selection in salmon breeding (by reducing genotyping costs). There are a number of other routes to improving cost-efficiency of genomic selection; for example by preselecting candidates for genotyping based on trait or breeding values (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lillehammer <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ødegård and Meuwissen 2014</a>). Another route to improvement of genomic selection in salmon is to increase overall selection accuracy, particularly where trait records are only available on distant relatives of the selection candidates (<a id="xref-ref-39-13" class="xref-bibr" href="#ref-39">Tsai <em>et al.</em> 2016a</a>). Successful achievement of accurate “cross-population” genomic prediction reduces the requirement for yearly testing on close relatives (<em>e.g.</em>, siblings) of selection candidates. Prediction accuracy in this scenario is likely to benefit large sample sizes for the training populations, high marker density (potentially using low-cost sequencing methods), and/or prioritization of putative functional variants in the SNP panel used for prediction. The latter may be enhanced by initiatives such as the Functional Annotation of All Salmonid Genomes (FAASG; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Macqueen <em>et al.</em> 2016</a>).</p></div><div id="sec-17" class="subsection"><h3>Conclusion</h3><p id="p-31">Genotype imputation approaches were tested in a sample of Atlantic salmon from a commercial breeding program, and the efficacy of using imputed genotype data for genomic prediction was evaluated. Using a two-generation design, with parents genotyped at medium or high density, and offspring genotyped at a lower density, imputation accuracy of up to 0.90 was possible. Genomic prediction accuracy using imputed genotype data were comparable to true genotype data with a ∼250 SNP panel used on 75% of the offspring. However, overall improvement in imputation accuracy may be expected by genotyping additional ancestral generations in the pedigree. Genomic prediction accuracies using imputed genotypes were very close to those using true genotypes, for both growth and sea lice resistance traits. Given that low density genotyping is substantially cheaper than medium or high density, imputation approaches may contribute to the widespread and cost-effective generation of genome-wide SNP data for genomic selection in aquaculture breeding programs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">We sincerely thank Bill Roy and Matt Tinsley for assistance with collection of trait data. J.E.B. and J.B.T. were partly supported by the Marine Alliance for Science and Technology pooling initiative, funded by the Scottish Funding Council (grant reference HR09011) and contributing institutions. This research was supported by Biotechnology and Biological Sciences Research Council (BBSRC) grants (BB/N024044/1, BB/H022007/1, BB/M028321/1), and by BBSRC Institute Strategic Funding Grants to The Roslin Institute (BB/J004235/1, BB/J004324/1, BB/J004243/1). The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-33">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040717/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-34"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-19"><span class="received-label">Received </span>November 19, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-02-22"><span class="accepted-label">Accepted </span>February 22, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Tsai <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antolín</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettelblad</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Money</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A hybrid method for the imputation of genomic data in livestock populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.3" data-doi="10.1017/S1751731111000309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Imputation of genotypes from low- to high-density genotyping platforms and implications for genomic selection.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DBerry%26rft.auinit1%253DD.%2BP.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D1162%26rft.epage%253D1169%26rft.atitle%253DImputation%2Bof%2Bgenotypes%2Bfrom%2Blow-%2Bto%2Bhigh-density%2Bgenotyping%2Bplatforms%2Band%2Bimplications%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731111000309%26rft_id%253Dinfo%253Apmid%252F22440168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731111000309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22440168&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.4" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype imputation with millions of reference samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Genova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome wide association study for resistance to <em>Caligus rogercresseyi</em> in Atlantic salmon (<em>Salmo salar L</em>.) using a 50K SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> (in press).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.6" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.7" data-doi="10.1038/srep19244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of the 2b-RAD method for genomic selection in scallop breeding.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">19244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19244%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19244%26rft_id%253Dinfo%253Apmid%252F26754638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26754638&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.8" data-doi="10.1534/genetics.109.108431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georges</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A hidden Markov model combining linkage and linkage disequilibrium information for haplotype reconstruction and quantitative trait locus fine mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.108431%26rft_id%253Dinfo%253Apmid%252F20008575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzMvNzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bron</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The control of sea lice in Atlantic salmon by selective breeding.</span> <abbr class="cit-jnl-abbrev">J. R. Soc. Interface</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">0574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BSoc.%2BInterface%26rft.volume%253D12%26rft.spage%253D0574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.4.1377.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>ASReml User Guide</em>.</span> <span class="cit-publ-name">VSN International Ltd</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.11" data-doi="10.1016/j.aquaculture.2011.01.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorland</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimates of genetic variation in the susceptibility of Atlantic salmon (<em>Salmo salar</em>) to the salmon louse <em>Lepeophtheirus salmonis</em>.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D314%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2011.01.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2011.01.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.12" data-doi="10.1534/genetics.108.100289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection using low-density marker panels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100289%26rft_id%253Dinfo%253Apmid%252F19299339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genotype imputation in sheep breeds.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D72%26rft.epage%253D80%26rft.atitle%253DAccuracy%2Bof%2Bgenotype%2Bimputation%2Bin%2Bsheep%2Bbreeds.%26rft_id%253Dinfo%253Apmid%252F22221027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22221027&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.14" data-doi="10.2135/cropsci2011.07.0358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de losCampos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">654</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D654%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.07.0358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.07.0358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.15" data-doi="10.1186/1297-9686-44-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">A phasing and imputation method for pedigreed populations that results in a single-stage genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DBorner%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DOptimising%2Bmultistage%2Bdairy%2Bcattle%2Bbreeding%2Bschemes%2Bincluding%2Bgenomic%2Bselection%2Busing%2Bdecorrelated%2Bor%2Boptimum%2Bselection%2Bindices.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-1%26rft_id%253Dinfo%253Apmid%252F22252172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252172&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.16" data-doi="10.1186/1471-2164-15-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-90%26rft_id%253Dinfo%253Apmid%252F24524230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24524230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.17" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.18" data-doi="10.1186/1297-9686-45-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hozé</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouilloux</span>  <span class="cit-name-given-names">M.-N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venot</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dassonneville</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-density marker imputation accuracy in sixteen French cattle breeds.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-1%26rft_id%253Dinfo%253Apmid%252F23281913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23281913&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elliot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kube</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botwright</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diversity and linkage disequilibrium in farmed Tasmanian Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D48%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.20" data-doi="10.1002/gepi.20533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D816%26rft.epage%253D834%26rft.atitle%253DMaCH%253A%2Busing%2Bsequence%2Band%2Bgenotype%2Bdata%2Bto%2Bestimate%2Bhaplotypes%2Band%2Bunobserved%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20533%26rft_id%253Dinfo%253Apmid%252F21058334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058334&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284719100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthew</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D533%26rft.spage%253D200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lillehammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A low-marker density implementation of genomic selection in aquaculture using within-family genomic breeding values.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of different methods for imputing genome-wide marker genotypes in Swedish and Finnish red cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4666</span>–<span class="cit-lpage">4677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D4666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-web" id="cit-7.4.1377.24"><div class="cit-metadata unstructured">Macqueen, D. J., C. R. Primmer, R. D. Houston, B. F. Nowak, L. Bernatchez <em>et al.</em>, 2016 Functional Analysis of All Salmonid Genomes (FAASG): an international initiative supporting future salmonid research, conservation and aquaculture. bioRxiv Available at: <a href="https://doi.org/10.1101/095737">https://doi.org/10.1101/095737</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTM3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.26" data-doi="10.1146/annurev-animal-031412-103705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accelerating improvement of livestock with genomic selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D1%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-031412-103705%26rft_id%253Dinfo%253Apmid%252F25387018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-031412-103705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387018&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.27" data-doi="10.1186/s12711-014-0081-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghaddar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of genotype imputation based on random and selected reference sets in purebred and crossbred sheep populations and its effect on accuracy of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0081-5%26rft_id%253Dinfo%253Apmid%252F25595431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0081-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25595431&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identity-by-descent genomic selection using selective and sparse genotyping.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsvoll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjøglum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in an admixed population of Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D402%26rft_id%253Dinfo%253Apmid%252F25484890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25484890&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferraresso</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of resistance to Pasteurellosis in gilthead sea bream (<em>Sparus aurata</em>) using 2b-RAD sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3693</span>–<span class="cit-lpage">3700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D3693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.31" data-doi="10.1111/1755-0998.12337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The development and characterization of a 57K single nucleotide polymorphism array for rainbow trout.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12337%26rft_id%253Dinfo%253Apmid%252F25294387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25294387&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.32" data-doi="10.1186/1297-9686-45-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pausch</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aigner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emmerling</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>  <span class="cit-name-given-names">K.-U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation of high-density genotypes in the Fleckvieh cattle population.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-1%26rft_id%253Dinfo%253Apmid%252F23281913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23281913&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.33" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a toolset for whole-genome association and population-based linkage analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-7.4.1377.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.35" data-doi="10.1111/raq.12193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Applications of genotyping by sequencing in aquaculture breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Rev. Aquacult.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/raq.12193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BAquacult.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fraq.12193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/raq.12193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segelke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reliability of genomic prediction for German Holsteins using imputed genotypes from low density chips.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5403</span>–<span class="cit-lpage">5411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DSegelke%26rft.auinit1%253DD.%26rft.volume%253D95%26rft.issue%253D9%26rft.spage%253D5403%26rft.epage%253D5411%26rft.atitle%253DReliability%2Bof%2Bgenomic%2Bprediction%2Bfor%2BGerman%2BHolsteins%2Busing%2Bimputed%2Bgenotypes%2Bfrom%2Blow-density%2Bchips.%26rft_id%253Dinfo%253Apmid%252F22916947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22916947&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.37" data-doi="10.1186/1297-9686-41-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Testing strategies for genomic selection in aquaculture breeding programs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSonesson%26rft.auinit1%253DA.%2BK.%26rft.volume%253D41%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DTesting%2Bstrategies%2Bfor%2Bgenomic%2Bselection%2Bin%2Baquaculture%2Bbreeding%2Bprograms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-37%26rft_id%253Dinfo%253Apmid%252F19566932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19566932&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.38" data-doi="10.1186/s12864-015-2117-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2117-9%26rft_id%253Dinfo%253Apmid%252F26582102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2117-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26582102&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.39" data-doi="10.1186/s12711-015-0181-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bron</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0181-x%26rft_id%253Dinfo%253Apmid%252F26743767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0181-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26743767&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Construction and annotation of a high density SNP linkage map of the Atlantic salmon (<em>Salmo salar</em>) genome.</span> <abbr class="cit-jnl-abbrev">Genes Genomes Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2173</span>–<span class="cit-lpage">2179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenet.%26rft.volume%253D6%26rft.spage%253D2173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uemoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of high-density genotype imputation in Japanese black cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D46%26rft.spage%253D388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leeds</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragomeni</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of genome-enabled selection for bacterial cold water disease resistance using progeny performance data in rainbow trout: insights on genotyping methods and genomic prediction models.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.43" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.44" data-doi="10.1186/1297-9686-43-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic evaluations with many more genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DVanraden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D43%26rft.spage%253D10%26rft.epage%253D10%26rft.atitle%253DGenomic%2Bevaluations%2Bwith%2Bmany%2Bmore%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-10%26rft_id%253Dinfo%253Apmid%252F21366914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21366914&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics and genomics of disease resistance in salmonid species.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D415%26rft_id%253Dinfo%253Apmid%252F25505486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25505486&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomics in aquaculture to better understand species biology and accelerate genetic progress.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D128%26rft_id%253Dinfo%253Apmid%252F25883603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25883603&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1377.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1377.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naswa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomewide single nucleotide polymorphism discovery in Atlantic salmon (<em>Salmo salar</em>): validation in wild and farmed American and European populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1002</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D1002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/4/1377.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1365" title="A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1385" title="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/4"><span>Volume 7 Issue 4, April 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/4.cover-source.jpg?itok=KOTpzmYp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1377.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jen8QDqH1J43lYC_UVuqy1yvR2TGshANDdr7QqDDmbE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/4/1377" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/4/1377" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/4/1377" class="active" data-icon-position="" data-hide-link-title="0">Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483837" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cfpLp7CU3LOG-c1IPseYkdjT1LxWKpIpNcqKRxbnu7k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1377.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/4/1377" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6dosYAO_j5H85CbdrS1PCsM-o_ARjVhbspYqwYEYY6Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/4/1377" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1377.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483837" id="node483837--2" data-pisa="ggg;7/4/1377" data-pisa-master="ggg;g3.117.040717" data-apath="/ggg/7/4/1377.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hsin-Yuan Tsai</span>, <span class="highwire-citation-author" data-delta="1">Oswald Matika</span>, <span class="highwire-citation-author" data-delta="2">Stefan McKinnon Edwards</span>, <span class="highwire-citation-author" data-delta="3">Roberto Antolín–Sánchez</span>, <span class="highwire-citation-author" data-delta="4">Alastair Hamilton</span>, <span class="highwire-citation-author" data-delta="5">Derrick R. Guy</span>, <span class="highwire-citation-author" data-delta="6">Alan E. Tinch</span>, <span class="highwire-citation-author" data-delta="7">Karim Gharbi</span>, <span class="highwire-citation-author" data-delta="8">Michael J. Stear</span>, <span class="highwire-citation-author" data-delta="9">John B. Taggart</span>, <span class="highwire-citation-author" data-delta="10">James E. Bron</span>, <span class="highwire-citation-author" data-delta="11">John M. Hickey</span> and <span class="highwire-citation-author" data-delta="12">Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1377-1383;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040717 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483837--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hsin-Yuan Tsai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-Yuan%2BTsai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsai%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-Yuan%2BTsai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oswald Matika </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oswald%2BMatika%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matika%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOswald%2BMatika%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stefan McKinnon Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BMcKinnon%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BMcKinnon%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Roberto Antolín–Sánchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BAntol%C3%ADn%E2%80%93S%C3%A1nchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Antol%C3%ADn%E2%80%93S%C3%A1nchez%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BAntol%25C3%25ADn%25E2%2580%2593S%25C3%25A1nchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alastair Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alastair%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlastair%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Derrick R. Guy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derrick%2BR.%2BGuy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guy%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerrick%2BR.%2BGuy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alan E. Tinch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BTinch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinch%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BTinch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Karim Gharbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3JT, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karim%2BGharbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gharbi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarim%2BGharbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael J. Stear </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute of Biodiversity, Animal Health & Comparative Medicine, University of Glasgow, G61 1QH, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BStear%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stear%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BStear%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">John B. Taggart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BB.%2BTaggart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taggart%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BB.%2BTaggart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">James E. Bron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BBron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bron%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BBron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ross.houston@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ross.houston@roslin.ed.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/4/1377" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/4/1377&amp;title=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/4/1377&amp;title=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/4/1377&amp;title=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/4/1377&amp;text=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/4/1377&amp;title=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/4/1377&amp;t=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/4/1377&amp;title=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/4/1377&amp;title=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483837/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483837" id="node483837--3" data-pisa="ggg;7/4/1377" data-pisa-master="ggg;g3.117.040717" data-apath="/ggg/7/4/1377.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hsin-Yuan Tsai</span>, <span class="highwire-citation-author" data-delta="1">Oswald Matika</span>, <span class="highwire-citation-author" data-delta="2">Stefan McKinnon Edwards</span>, <span class="highwire-citation-author" data-delta="3">Roberto Antolín–Sánchez</span>, <span class="highwire-citation-author" data-delta="4">Alastair Hamilton</span>, <span class="highwire-citation-author" data-delta="5">Derrick R. Guy</span>, <span class="highwire-citation-author" data-delta="6">Alan E. Tinch</span>, <span class="highwire-citation-author" data-delta="7">Karim Gharbi</span>, <span class="highwire-citation-author" data-delta="8">Michael J. Stear</span>, <span class="highwire-citation-author" data-delta="9">John B. Taggart</span>, <span class="highwire-citation-author" data-delta="10">James E. Bron</span>, <span class="highwire-citation-author" data-delta="11">John M. Hickey</span> and <span class="highwire-citation-author" data-delta="12">Ross D. Houston</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1377-1383;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040717 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483837--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hsin-Yuan Tsai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-Yuan%2BTsai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsai%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-Yuan%2BTsai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oswald Matika </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oswald%2BMatika%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matika%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOswald%2BMatika%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stefan McKinnon Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BMcKinnon%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BMcKinnon%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Roberto Antolín–Sánchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BAntol%C3%ADn%E2%80%93S%C3%A1nchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Antol%C3%ADn%E2%80%93S%C3%A1nchez%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BAntol%25C3%25ADn%25E2%2580%2593S%25C3%25A1nchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alastair Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alastair%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlastair%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Derrick R. Guy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derrick%2BR.%2BGuy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guy%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerrick%2BR.%2BGuy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alan E. Tinch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Hendrix Genetics Aquaculture BV/ NetherlandsVilla 'de Körver', Spoorstraat 695831 CK BoxmeerThe Netherlands </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BTinch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinch%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BTinch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Karim Gharbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Edinburgh Genomics, Ashworth Laboratories, University of Edinburgh, EH9 3JT, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karim%2BGharbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gharbi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarim%2BGharbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael J. Stear </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute of Biodiversity, Animal Health & Comparative Medicine, University of Glasgow, G61 1QH, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BStear%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stear%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BStear%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">John B. Taggart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BB.%2BTaggart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taggart%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BB.%2BTaggart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">James E. Bron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aquaculture, University of Stirling, FK9 4LA, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BBron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bron%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BBron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian EH25 9RG, United Kingdom </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ross.houston@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ross.houston@roslin.ed.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1377&amp;rft.epage=1383&amp;rft.atitle=Genotype%20Imputation%20To%20Improve%20the%20Cost-Efficiency%20of%20Genomic%20Selection%20in%20Farmed%20Atlantic%20Salmon&amp;rft.volume=7&amp;rft.issue=4&amp;rft.date=2017-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tsai%2C+Hsin-Yuan&amp;rft.au=Matika%2C+Oswald&amp;rft.au=Edwards%2C+Stefan+McKinnon&amp;rft.au=Antol%C3%ADn%E2%80%93S%C3%A1nchez%2C+Roberto&amp;rft.au=Hamilton%2C+Alastair&amp;rft.au=Guy%2C+Derrick+R.&amp;rft.au=Tinch%2C+Alan+E.&amp;rft.au=Gharbi%2C+Karim&amp;rft.au=Stear%2C+Michael+J.&amp;rft.au=Taggart%2C+John+B.&amp;rft.au=Bron%2C+James+E.&amp;rft.au=Hickey%2C+John+M.&amp;rft.au=Houston%2C+Ross+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483837/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483837/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483837/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483837/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483837/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483837/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483837/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483837/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483837/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483837/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483837/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483837/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/4/1377" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/4/1377.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/4/1377.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/4/1377.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/4/1377.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
