<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/4/1365" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/4/1365.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/4/1365.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/4/1365.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/4/1365.atom" />
<meta name="HW.pisa" content="ggg;7/4/1365" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity" />
<meta name="DC.Identifier" content="10.1534/g3.116.038497" />
<meta name="DC.Date" content="2017-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Leitwein et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-density linkage maps are valuable tools for conservation and eco-evolutionary issues. In salmonids, a complex rediploidization process consecutive to an ancient whole genome duplication event makes linkage maps of prime importance for investigating the evolutionary history of chromosome rearrangements. Here, we developed a high-density consensus linkage map for the brown trout ( Salmo trutta ), a socioeconomically important species heavily impacted by human activities. A total of 3977 ddRAD markers were mapped and ordered in 40 linkage groups using sex- and lineage-averaged recombination distances obtained from two family crosses. Performing map comparison between S. trutta and its sister species, S. salar , revealed extensive chromosomal rearrangements. Strikingly, all of the fusion and fission events that occurred after the S. salar / S. trutta speciation happened in the Atlantic salmon branch, whereas the brown trout remained closer to the ancestral chromosome structure. Using the strongly conserved synteny within chromosome arms, we aligned the brown trout linkage map to the Atlantic salmon genome sequence to estimate the local recombination rate in S. trutta at 3721 loci. A significant positive correlation between recombination rate and within-population nucleotide diversity (π) was found, indicating that selection constrains variation at linked neutral sites in brown trout. This new high-density linkage map provides a useful genomic resource for future aquaculture, conservation, and eco-evolutionary studies in brown trout." />
<meta name="DC.Contributor" content="Maeva Leitwein" />
<meta name="DC.Contributor" content="Bruno Guinand" />
<meta name="DC.Contributor" content="Juliette Pouzadoux" />
<meta name="DC.Contributor" content="Erick Desmarais" />
<meta name="DC.Contributor" content="Patrick Berrebi" />
<meta name="DC.Contributor" content="Pierre-Alexandre Gagnaire" />
<meta name="article:published_time" content="2017-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-density linkage maps are valuable tools for conservation and eco-evolutionary issues. In salmonids, a complex rediploidization process consecutive to an ancient whole genome duplication event makes linkage maps of prime importance for investigating the evolutionary history of chromosome rearrangements. Here, we developed a high-density consensus linkage map for the brown trout (&lt;i&gt;Salmo trutta&lt;/i&gt;), a socioeconomically important species heavily impacted by human activities. A total of 3977 ddRAD markers were mapped and ordered in 40 linkage groups using sex- and lineage-averaged recombination distances obtained from two family crosses. Performing map comparison between &lt;i&gt;S. trutta&lt;/i&gt; and its sister species, &lt;i&gt;S. salar&lt;/i&gt;, revealed extensive chromosomal rearrangements. Strikingly, all of the fusion and fission events that occurred after the &lt;i&gt;S. salar&lt;/i&gt;/&lt;i&gt;S. trutta&lt;/i&gt; speciation happened in the Atlantic salmon branch, whereas the brown trout remained closer to the ancestral chromosome structure. Using the strongly conserved synteny within chromosome arms, we aligned the brown trout linkage map to the Atlantic salmon genome sequence to estimate the local recombination rate in &lt;i&gt;S. trutta&lt;/i&gt; at 3721 loci. A significant positive correlation between recombination rate and within-population nucleotide diversity (π) was found, indicating that selection constrains variation at linked neutral sites in brown trout. This new high-density linkage map provides a useful genomic resource for future aquaculture, conservation, and eco-evolutionary studies in brown trout.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/04/01" />
<meta name="citation_mjid" content="ggg;7/4/1365" />
<meta name="citation_id" content="7/4/1365" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/4/1365" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/4/1365.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/4/1365.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/4/1365.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038497" />
<meta name="citation_pmid" content="28235829" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1365" />
<meta name="citation_lastpage" content="1376" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Maeva Leitwein" />
<meta name="citation_author_institution" content="Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author_email" content="maeva.leitwein.pro@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5172-0970" />
<meta name="citation_author" content="Bruno Guinand" />
<meta name="citation_author_institution" content="Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author_institution" content="Département Biologie-Ecologie, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6934-1677" />
<meta name="citation_author" content="Juliette Pouzadoux" />
<meta name="citation_author_institution" content="Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author" content="Erick Desmarais" />
<meta name="citation_author_institution" content="Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author" content="Patrick Berrebi" />
<meta name="citation_author_institution" content="Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9026-4987" />
<meta name="citation_author" content="Pierre-Alexandre Gagnaire" />
<meta name="citation_author_institution" content="Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1908-3235" />
<meta name="citation_reference" content="citation_title=Evolutionary Genetics of Fishes, Monographs in Evolutionary Biology;citation_author=F. W. Allendorf;citation_author=G. H. Thorgaard;citation_title=Tetraploidy and the evolution of salmonid fishes;citation_pages=1-53;citation_volume=Volume 1;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Allendorf;citation_author=R. G. Danzmann;citation_title=Secondary Tetrasomic Segregation of MDH-B and Preferential Pairing of Homeologues in Rainbow Trout;citation_pages=1083-1092;citation_volume=145;citation_year=1997;citation_issue=4;citation_pmid=9093860" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=F. W. Allendorf;citation_author=S. Bassham;citation_author=W. A. Cresko;citation_author=M. T. Limborg;citation_author=L. W. Seeb;citation_title=Effects of crossovers between homeologs on inheritance and population genomics in polyploid-derived salmonid fishes.;citation_volume=106;citation_year=2015;citation_pmid=25838153;citation_doi=10.1093/jhered/esv015" />
<meta name="citation_reference" content="citation_journal_title=Freshw. Biol.;citation_author=A. Almodóvar;citation_author=G. G. Nicola;citation_author=B. Elvira;citation_author=J. L. García-Marin;citation_title=Introgression variability among Iberian brown trout evolutionary significant units: the influence of local management and environmental features.;citation_pages=1175-1187;citation_volume=51;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome Evolution and Meiotic Maps by Massively Parallel DNA Sequencing: Spotted Gar, an Outgroup for the Teleost Genome Duplication;citation_pages=799-808;citation_volume=188;citation_year=2011;citation_issue=4;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. Bernatchez;citation_title=The evolutionary history of brown trout (Salmo trutta L.) inferred from phylogeographic, nested clade, and mismatch analyses of mitochondrial DNA variation.;citation_pages=351-379;citation_volume=55;citation_year=2001;citation_issue=2;citation_pmid=11308093;citation_doi=10.1554/0014-3820(2001)055[0351:TEHOBT]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. Bernatchez;citation_author=A. Osinov;citation_title=Genetic diversity of trout (genus Salmo) from its most eastern native range based on mitochondrial DNA and nuclear gene variation.;citation_pages=285-297;citation_volume=4;citation_year=1995;citation_issue=3;citation_pmid=7663748" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Biol.;citation_author=P. Berrebi;citation_title=Three brown trout Salmo trutta lineages in Corsica described through allozyme variation.;citation_pages=60-73;citation_volume=86;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Biol.;citation_author=P. Berrebi;citation_author=C. Poteaux;citation_author=M. Fissier;citation_author=G. Cattaneo-Berrebi;citation_title=Stocking impact and allozyme diversity in brown trout from Mediterranean southern France.;citation_pages=949-960;citation_volume=56;citation_year=2000;citation_doi=10.1111/j.1095-8649.2000.tb00884.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Berthelot;citation_author=F. Brunet;citation_author=D. Chalopin;citation_author=A. Juanchich;citation_author=M. Bernard;citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.;citation_pages=3657;citation_volume=5;citation_year=2014;citation_pmid=24755649;citation_doi=10.1038/ncomms4657" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. Bohling;citation_author=P. Haffray;citation_author=P. Berrebi;citation_title=Genetic diversity and population structure of domestic brown trout (Salmo trutta) in France.;citation_pages=1-9;citation_volume=462;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=V. Bourret;citation_author=M. P. Kent;citation_author=C. R. Primmer;citation_author=A. Vasemägi;citation_author=S. Karlsson;citation_title=SNP-array reveals genome-wide patterns of geographical and potential adaptive divergence across the natural range of Atlantic salmon (Salmo salar).;citation_pages=532-551;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12003" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Brenna-Hansen;citation_author=J. Li;citation_author=M. P. Kent;citation_author=E. G. Boulding;citation_author=S. Dominik;citation_title=Chromosomal differences between European and North American Atlantic salmon discovered by linkage mapping and supported by fluorescence in situ hybridization analysis.;citation_pages=432-432;citation_volume=13;citation_year=2012;citation_pmid=22928605;citation_doi=10.1186/1471-2164-13-432" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=M. S. Brieuc;citation_author=C. D. Waters;citation_author=J. E. Seeb;citation_author=K. A. Naish;citation_title=A dense linkage map for Chinook salmon (Oncorhynchus tshawytscha) reveals variable chromosomal divergence after an ancestral whole genome duplication event.;citation_pages=447-460;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Does linked selection explain the narrow range of genetic diversity across species?;citation_author=G. Coop;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=R. B. Corbett-Detig;citation_author=D. L. Hartl;citation_author=T. B. Sackton;citation_title=Natural selection constrains neutral diversity across a wide range of species.;citation_pages=e1002112;citation_volume=13;citation_year=2015;citation_pmid=25859758;citation_doi=10.1371/journal.pbio.1002112" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Crête-Lafrenière;citation_author=L. K. Weir;citation_author=L. Bernatchez;citation_title=Framing the salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.;citation_pages=e46662-e46662;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071608;citation_doi=10.1371/journal.pone.0046662" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=M. Dukić;citation_author=D. Berner;citation_author=M. Roesti;citation_author=C. R. Haag;citation_author=D. Ebert;citation_title=A high-density genetic map reveals variation in recombination rate across the genome of Daphnia magna.;citation_pages=137;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. Ekblom;citation_author=J. Galindo;citation_title=Applications of next generation sequencing in molecular ecology of non-model organisms.;citation_pages=1-15;citation_volume=107;citation_year=2011;citation_issue=1;citation_pmid=21139633;citation_doi=10.1038/hdy.2010.152" />
<meta name="citation_reference" content="citation_title=Quantitative Ecology and the Brown Trout;citation_author=J. M. Elliott;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. V. Everett;citation_author=M. R. Miller;citation_author=J. E. Seeb;citation_title=Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.;citation_pages=521-521;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23031582;citation_doi=10.1186/1471-2164-13-521" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P.-A. Gagnaire;citation_author=E. Normandeau;citation_author=S. A. Pavey;citation_author=L. Bernatchez;citation_title=Mapping phenotypic, expression and transmission ratio distortion QTL using RAD markers in the Lake Whitefish (Coregonus clupeaformis).;citation_pages=3036-3048;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12127" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=P.-A. Gagnaire;citation_author=S. A. Pavey;citation_author=E. Normandeau;citation_author=L. Bernatchez;citation_title=The genetic architecture of reproductive isolation during speciation-with-gene-flow in lake whitefish species pairs assessed by RAD sequencing.;citation_pages=2483-2497;citation_volume=67;citation_year=2013;citation_pmid=24033162;citation_doi=10.1111/evo.12075" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Gharbi;citation_author=A. Gautier;citation_author=R. G. Danzmann;citation_author=S. Gharbi;citation_author=T. Sakamoto;citation_title=A linkage map for brown trout (Salmo trutta): chromosome homeologies and comparative genome organization with other salmonid fish.;citation_volume=172;citation_year=2006;citation_pmid=16452148;citation_doi=10.1534/genetics.105.048330" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=M. M. Hansen;citation_author=K. L. D. Mensberg;citation_title=Admixture analysis of stocked brown trout populations using mapped microsatellite DNA markers: indigenous trout persist in introgressed populations.;citation_volume=5;citation_year=2009;citation_pmid=19515653;citation_doi=10.1098/rsbl.2009.0214" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. M. Hansen;citation_author=D. E. Ruzzante;citation_author=E. E. Nielsen;citation_author=K. D. Mensberg;citation_title=Microsatellite and mitochondrial DNA polymorphism reveals life-history dependent interbreeding between hatchery and wild brown trout (Salmo trutta L.).;citation_pages=583-594;citation_volume=9;citation_year=2000;citation_issue=5;citation_pmid=10792701;citation_doi=10.1046/j.1365-294x.2000.00898.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. M. Hansen;citation_author=K. Meier;citation_author=K. L. D. Mensberg;citation_title=Identifying footprints of selection in stocked brown trout populations: a spatio-temporal approach.;citation_pages=1787-1800;citation_volume=19;citation_year=2010;citation_issue=9;citation_pmid=20345684" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=S. E. Hartley;citation_author=M. T. Horne;citation_title=Chromosome relationships in the genus Salmo.;citation_pages=229-237;citation_volume=90;citation_year=1984;citation_issue=3;citation_pmid=6499596" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=B. C. Hecht;citation_author=F. P. Thrower;citation_author=M. C. Hale;citation_author=M. R. Miller;citation_author=K. M. Nichols;citation_title=Genetic architecture of migration-related traits in rainbow and steelhead trout, Oncorhynchus mykiss.;citation_pages=1113-1127;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=P. A. Hohenlohe;citation_author=S. J. Amish;citation_author=J. M. Catchen;citation_author=F. W. Allendorf;citation_author=G. Luikart;citation_title=Next-generation RAD sequencing identifies thousands of SNPs for assessing hybridization between rainbow and westslope cutthroat trout.;citation_pages=117-122;citation_volume=11;citation_year=2011;citation_pmid=21429168;citation_doi=10.1111/j.1755-0998.2010.02967.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. D. Houston;citation_author=J. W. Davey;citation_author=S. C. Bishop;citation_author=N. R. Lowe;citation_author=J. C. Mota-Velasco;citation_title=Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.;citation_pages=244-244;citation_volume=13;citation_year=2012;citation_pmid=22702806;citation_doi=10.1186/1471-2164-13-244" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. D. Houston;citation_author=J. B. Taggart;citation_author=T. Cézard;citation_author=M. Bekaert;citation_author=N. R. Lowe;citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).;citation_pages=90;citation_volume=15;citation_year=2014;citation_pmid=24524230;citation_doi=10.1186/1471-2164-15-90" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=L. F. Jensen;citation_author=M. M. Hansen;citation_author=J. Carlsson;citation_author=V. Loeschcke;citation_author=K. L. D. Mensberg;citation_title=Spatial and temporal genetic differentiation and effective population size of brown trout (Salmo trutta, L.) in small Danish rivers.;citation_pages=615-621;citation_volume=6;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. E. Jorde;citation_author=N. Ryman;citation_title=Demographic Genetics of Brown Trout (Salmo trutta) and Estimation of Effective Population Size From Temporal Change of Allele Frequencies;citation_pages=1369-1381;citation_volume=143;citation_year=1996;citation_issue=3;citation_pmid=8807308" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=W. Kai;citation_author=K. Nomura;citation_author=A. Fujiwara;citation_author=Y. Nakamura;citation_author=M. Yasuike;citation_title=A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (Anguilla japonica) provides insights into genome evolution after the teleost-specific genome duplication.;citation_pages=233;citation_volume=15;citation_year=2014;citation_pmid=24669946;citation_doi=10.1186/1471-2164-15-233" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=M. Kodama;citation_author=M. S. O. Brieuc;citation_author=R. H. Devlin;citation_author=J. J. Hard;citation_author=K. A. Naish;citation_title=Comparative mapping between coho salmon (Oncorhynchus kisutch) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event.;citation_pages=1717-1730;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y.-Y. Kuang;citation_author=X.-H. Zheng;citation_author=C.-Y. Li;citation_author=X.-M. Li;citation_author=D.-C. Cao;citation_title=The genetic map of goldfish (Carassius auratus) provided insights to the divergent genome evolutions in the Cyprinidae family.;citation_pages=34849;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Fish. Res.;citation_author=G. Kumar;citation_author=M. Kocour;citation_title=Applications of next-generation sequencing in fisheries research: a review.;citation_pages=11-22;citation_volume=186;citation_year=2017" />
<meta name="citation_reference" content="citation_author=L. Laikre;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. C. Lamaze;citation_author=C. Sauvage;citation_author=A. Marie;citation_author=D. Garant;citation_author=L. Bernatchez;citation_title=Dynamics of introgressive hybridization assessed by SNP population genomics of coding genes in stocked brook charr (Salvelinus fontinalis).;citation_pages=2877-2895;citation_volume=21;citation_year=2012;citation_issue=12;citation_pmid=22548328;citation_doi=10.1111/j.1365-294X.2012.05579.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. R. Largiader;citation_author=A. Scholl;citation_title=Genetic introgression between native and introduced brown trout Salmo trutta L. populations in the Rhone River Basin.;citation_pages=417-426;citation_volume=5;citation_year=1996;citation_doi=10.1046/j.1365-294X.1996.00099.x" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=W. A. Larson;citation_author=L. W. Seeb;citation_author=M. V. Everett;citation_author=R. K. Waples;citation_author=W. D. Templin;citation_title=Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon (Oncorhynchus tshawytscha).;citation_pages=355-369;citation_volume=7;citation_year=2014;citation_pmid=24665338;citation_doi=10.1111/eva.12128" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=W. A. Larson;citation_author=G. J. McKinney;citation_author=M. T. Limborg;citation_author=M. V. Everett;citation_author=L. W. Seeb;citation_title=Identification of multiple QTL hotspots in sockeye salmon (Oncorhynchus nerka) using genotyping-by-sequencing and a dense linkage map.;citation_volume=107;citation_year=2016;citation_pmid=26712859;citation_doi=10.1093/jhered/esv099" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Biol.;citation_author=M. Leitwein;citation_author=P. A. Gagnaire;citation_author=E. Desmarais;citation_author=S. Guendouz;citation_author=M. Rohmer;citation_title=Genome-wide nucleotide diversity of hatchery-reared Atlantic and Mediterranean strains of brown trout Salmo trutta compared to wild Mediterranean populations.;citation_pages=2717-2734;citation_volume=89;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows–Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=M. P. Kent;citation_title=The Atlantic salmon genome provides insights into rediploidization.;citation_pages=200-205;citation_volume=533;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=M. T. Limborg;citation_author=R. K. Waples;citation_author=J. E. Seeb;citation_author=L. W. Seeb;citation_title=Temporally isolated lineages of pink salmon reveal unique signatures of selection on distinct pools of standing genetic variation.;citation_volume=105;citation_year=2014;citation_pmid=25292170;citation_doi=10.1093/jhered/esu063" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. T. Limborg;citation_author=G. J. McKinney;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Recombination patterns reveal information about centromere location on linkage maps.;citation_pages=655-661;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=M. T. Limborg;citation_author=R. K. Waples;citation_author=F. W. Allendorf;citation_author=J. E. Seeb;citation_title=Linkage mapping reveals strong chiasma interference in sockeye salmon: implications for interpreting genomic data.;citation_pages=2463-2473;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=S. Liu;citation_author=Y. Li;citation_author=Z. Qin;citation_author=X. Geng;citation_author=L. Bao;citation_title=High-density interspecific genetic linkage mapping provides insights into genomic incompatibility between channel catfish and blue catfish.;citation_pages=81-90;citation_volume=47;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. Lond. B Biol. Sci.;citation_author=D. J. Macqueen;citation_author=I. A. Johnston;citation_title=A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.;citation_volume=281;citation_year=2014;citation_pmid=24452024;citation_doi=10.1098/rspb.2013.2881" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=P. Martinez;citation_author=A. Vinas;citation_author=C. Bouza;citation_author=J. Arias;citation_author=R. Amaro;citation_title=Cytogenetical characterization of hatchery stocks and natural populations of sea and brown trout from northwestern Spain.;citation_pages=9-17;citation_volume=66;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=B. May;citation_author=M. E. Delany;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosomal pairing in tetraploid derivative salmonid genomes: II. Wright is still right.;citation_volume=106;citation_year=2015;citation_pmid=26320244;citation_doi=10.1093/jhered/esv056" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. L. McKinney;citation_author=W. Seeb;citation_author=D. Larson;citation_author=M. Gomez-Uchida;citation_author=M. Limborg;citation_title=An integrated linkage map reveals candidate genes underlying adaptive variation in Chinook salmon (Oncorhynchus tshawytscha).;citation_pages=769-783;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=O. M. McMeel;citation_author=E. M. Hoey;citation_author=A. Ferguson;citation_title=Partial nucleotide sequences, and routine typing by polymerase chain reaction-restriction fragment length polymorphism, of the brown trout (Salmo trutta) lactate dehydrogenase, LDH-C1*90 and *100 alleles.;citation_pages=29-34;citation_volume=10;citation_year=2001;citation_issue=1;citation_pmid=11251784" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=W. J. Morrison;citation_title=Nonrandom segregation of two lactate dehydrogenase subunit loci in trout.;citation_pages=193-206;citation_volume=99;citation_year=1970;citation_doi=10.1577/1548-8659(1970)99&lt;193:NSOTLD&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. J. Near;citation_author=R. I. Eytan;citation_author=A. Dornburg;citation_author=K. L. Kuhn;citation_author=J. A. Moore;citation_title=Resolution of ray-finned fish phylogeny and timing of diversification.;citation_volume=109;citation_year=2012;citation_pmid=22869754;citation_doi=10.1073/pnas.1206625109" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=C. M. Nugent;citation_author=A. A. Easton;citation_author=J. D. Norman;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=A SNP based linkage map of the Arctic charr (Salvelinus alpinus) genome provides insights into the diploidization process after whole genome duplication.;citation_volume=7;citation_year=2017;citation_pmid=27986793;citation_doi=10.1534/g3.116.038026" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=S. Ohno;citation_title=The enormous diversity in genome sizes of fish as a reflection of nature’s extensive experiments with gene duplication.;citation_pages=120-130;citation_volume=99;citation_year=1970;citation_doi=10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=S. Palm;citation_author=L. Laikre;citation_author=P. E. Jorde;citation_author=N. Ryman;citation_title=Effective population size and temporal genetic change in stream resident brown trout (Salmo trutta, L.).;citation_pages=249-264;citation_volume=4;citation_year=2003;citation_doi=10.1023/A:1024064913094" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Perrier;citation_author=V. Bourret;citation_author=M. P. Kent;citation_author=L. Bernatchez;citation_title=Parallel and nonparallel genome-wide divergence among replicate population pairs of freshwater and anadromous Atlantic salmon.;citation_pages=5577-5593;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12500" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. Phillips;citation_author=P. Ráb;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2001;citation_issue=1;citation_pmid=11325050;citation_doi=10.1017/S1464793100005613" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Biol.;citation_author=C. Poteaux;citation_author=D. Beaudou;citation_author=P. Berrebi;citation_title=Temporal variations of genetic introgression in stocked brown trout populations.;citation_pages=701-713;citation_volume=53;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. Poteaux;citation_author=F. Bonhomme;citation_author=P. Berrebi;citation_title=Microsatellite polymorphism and genetic impact of restocking in Mediterranean brown trout (Salmo trutta L.).;citation_pages=645-653;citation_volume=82;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. Racimo;citation_author=S. Sankararaman;citation_author=R. Nielsen;citation_author=E. Huerta-Sánchez;citation_title=Evidence for archaic adaptive introgression in humans.;citation_pages=359-371;citation_volume=16;citation_year=2015;citation_pmid=25963373;citation_doi=10.1038/nrg3936" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Rezvoy;citation_author=D. Charif;citation_author=L. Guéguen;citation_author=G. A. Marais;citation_title=MareyMap: an R-based tool with graphical interface for estimating recombination rates.;citation_volume=23;citation_year=2007;citation_pmid=17586550;citation_doi=10.1093/bioinformatics/btm315" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. B. Rondeau;citation_author=D. R. Minkley;citation_author=J. S. Leong;citation_author=A. M. Messmer;citation_author=J. R. Jantzen;citation_title=The genome and linkage map of the northern pike (Esox lucius): conserved synteny revealed between the salmonid sister group and the Neoteleostei.;citation_pages=e102089;citation_volume=9;citation_year=2014;citation_pmid=25069045;citation_doi=10.1371/journal.pone.0102089" />
<meta name="citation_reference" content="citation_journal_title=Biol. Conserv.;citation_author=N. Sanz;citation_author=M. Cortey;citation_author=C. Pla;citation_author=J. L. García-Marín;citation_title=Hatchery introgression blurs ancient hybridization between brown trout (Salmo trutta) lineages as indicated by complementary allozymes and mtDNA markers.;citation_pages=278-289;citation_volume=130;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=C. Sauvage;citation_author=M. Vagner;citation_author=N. Derôme;citation_author=C. Audet;citation_author=L. Bernatchez;citation_title=Coding gene single nucleotide polymorphism mapping and quantitative trait loci detection for physiological reproductive traits in brook charr, Salvelinus fontinalis.;citation_pages=379-392;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=Evol Appl;citation_author=D. Serbezov;citation_author=P. E. Jorde;citation_author=L. Bernatchez;citation_author=E. M. Olsen;citation_author=L. A. Vøllestad;citation_title=Life history and demographic determinants of effective/census size ratios as exemplified by brown trout (Salmo trutta).;citation_pages=607-618;citation_volume=5;citation_year=2012;citation_issue=6;citation_pmid=23028401;citation_doi=10.1111/j.1752-4571.2012.00239.x" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=J. Slate;citation_author=J. Gratten;citation_author=D. Beraldi;citation_author=J. Stapley;citation_author=M. Hale;citation_title=Gene mapping in the wild with SNPs: guidelines and future directions.;citation_pages=97-107;citation_volume=136;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. Stapley;citation_author=J. Reger;citation_author=P. G. D. Feulner;citation_author=C. Smadja;citation_author=J. Galindo;citation_title=Adaptation genomics: the next generation.;citation_pages=705-712;citation_volume=25;citation_year=2010;citation_issue=12;citation_pmid=20952088;citation_doi=10.1016/j.tree.2010.09.002" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. J. G. Sutherland;citation_author=T. Gosselin;citation_author=E. Normandeau;citation_author=M. Lamothe;citation_author=N. Isabel;citation_title=Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps.;citation_pages=3600-3617;citation_volume=8;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Freshwat. Fish;citation_author=J. Thaulow;citation_author=R. Borgstrøm;citation_author=M. Heun;citation_title=Genetic persistence of an initially introduced brown trout (Salmo trutta L.) population despite restocking of foreign conspecifics.;citation_pages=485-497;citation_volume=23;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=H. Y. Tsai;citation_author=A. Hamilton;citation_author=D. R. Guy;citation_author=A. E. Tinch;citation_author=S. C. Bishop;citation_title=The genetic architecture of growth and fillet traits in farmed Atlantic salmon (Salmo salar).;citation_pages=51;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=H. Y. Tsai;citation_author=D. Robledo;citation_author=N. R. Lowe;citation_author=M. Bekaert;citation_author=J. B. Taggart;citation_title=Construction and annotation of a high density SNP linkage map of the Atlantic salmon (Salmo salar) genome.;citation_pages=2173-2179;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_title=JoinMap 4; Software for the Calculation of Genetic Map in Experimental Populations;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=L. Wang;citation_author=Z. Y. Wan;citation_author=B. Bai;citation_author=S. Q. Huang;citation_author=E. Chua;citation_title=Construction of a high-density linkage map and fine mapping of QTL for growth in Asian seabass.;citation_pages=16358;citation_volume=5;citation_year=2015;citation_pmid=26553309;citation_doi=10.1038/srep16358" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. K. Waples;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Linkage mapping with paralogs exposes regions of residual tetrasomic inheritance in chum salmon (Oncorhynchus keta).;citation_pages=17-28;citation_volume=16;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Isozymes;citation_journal_abbrev=Isozymes Curr Top Biol Med Res;citation_author=J. E. Wright;citation_author=K. Johnson;citation_author=A. Hollister;citation_author=B. May;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes.;citation_pages=239-260;citation_volume=10;citation_year=1983;citation_pmid=6354984" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=A. Yano;citation_author=B. Nicol;citation_author=E. Jouanno;citation_author=E. Quillet;citation_author=A. Fostier;citation_title=The sexually dimorphic on the Y-chromosome gene (sdY) is a conserved male-specific Y-chromosome sequence in many salmonids.;citation_pages=486-496;citation_volume=6;citation_year=2013;citation_pmid=23745140;citation_doi=10.1111/eva.12032" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Zhao;citation_author=Y. Zhang;citation_author=P. Ji;citation_author=X. Zhang;citation_author=Z. Zhao;citation_title=A dense genetic linkage map for common carp and its integration with a BAC-based physical map.;citation_pages=e63928;citation_volume=8;citation_year=2013;citation_pmid=23704958;citation_doi=10.1371/journal.pone.0063928" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="twitter:description" content="High-density linkage maps are valuable tools for conservation and eco-evolutionary issues. In salmonids, a complex rediploidization process consecutive to an ancient whole genome duplication event makes linkage maps of prime importance for investigating the evolutionary history of chromosome rearrangements. Here, we developed a high-density consensus linkage map for the brown trout ( Salmo trutta ), a socioeconomically important species heavily impacted by human activities. A total of 3977 ddRAD markers were mapped and ordered in 40 linkage groups using sex- and lineage-averaged recombination distances obtained from two family crosses. Performing map comparison between S. trutta and its sister species, S. salar , revealed extensive chromosomal rearrangements. Strikingly, all of the fusion and fission events that occurred after the S. salar / S. trutta speciation happened in the Atlantic salmon branch, whereas the brown trout remained closer to the ancestral chromosome structure. Using the strongly conserved synteny within chromosome arms, we aligned the brown trout linkage map to the Atlantic salmon genome sequence to estimate the local recombination rate in S. trutta at 3721 loci. A significant positive correlation between recombination rate and within-population nucleotide diversity (π) was found, indicating that selection constrains variation at linked neutral sites in brown trout. This new high-density linkage map provides a useful genomic resource for future aquaculture, conservation, and eco-evolutionary studies in brown trout." />
<meta name="og-title" property="og:title" content="A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/4/1365" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-density linkage maps are valuable tools for conservation and eco-evolutionary issues. In salmonids, a complex rediploidization process consecutive to an ancient whole genome duplication event makes linkage maps of prime importance for investigating the evolutionary history of chromosome rearrangements. Here, we developed a high-density consensus linkage map for the brown trout ( Salmo trutta ), a socioeconomically important species heavily impacted by human activities. A total of 3977 ddRAD markers were mapped and ordered in 40 linkage groups using sex- and lineage-averaged recombination distances obtained from two family crosses. Performing map comparison between S. trutta and its sister species, S. salar , revealed extensive chromosomal rearrangements. Strikingly, all of the fusion and fission events that occurred after the S. salar / S. trutta speciation happened in the Atlantic salmon branch, whereas the brown trout remained closer to the ancestral chromosome structure. Using the strongly conserved synteny within chromosome arms, we aligned the brown trout linkage map to the Atlantic salmon genome sequence to estimate the local recombination rate in S. trutta at 3721 loci. A significant positive correlation between recombination rate and within-population nucleotide diversity (π) was found, indicating that selection constrains variation at linked neutral sites in brown trout. This new high-density linkage map provides a useful genomic resource for future aquaculture, conservation, and eco-evolutionary studies in brown trout." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
    <title>A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"N9CXCY9-PfexnlZiAUcnX_LAz8kzThGGSb4ndum_UE8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483841","apath":"\/ggg\/7\/4\/1365.atom","pisa":"ggg;7\/4\/1365","ac":{"\/ggg\/7\/4\/1365.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/4\/1365.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1365"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1365"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/4\/1365":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222JorPSGp40vNuvheNJWx+zGOwKJFL7L0ttgwVLgxWaP6DGgBdQ86Fz2+Jh1F3MvidRzGJlB7esxAQ29TM\\\\\\\/vHudijm58iQtIREGm2RcxpPhPAagh9PrRqaRXYwMl0iadd0OK+8r+8SToXAq\\\\\\\/SCCrO2R+xGfM4c4fyetCYaiwqE9X17BVNUqtTLmSFX80XESojfmFKwzOPWC1zwJ1Acx1FFxV8VhnF0JqohPBdGKP9+VNrqZyT\\\\\\\/1N8iTA2BYvaOs0im5O8346\\\\\\\/IsCuwI2dlPz2HInIo9BpMVnepDaWRoR3MQfv4yyRFnT\\\\\\\/SXmn2vblo78eAbeeZ1gel7wXj8bffMGcX+Hie4zwyyh6As2UcVI1oqakwQXmBAdfod0fWJFa7aPIERwE0mzZIOemDvSgYsuK5oE9sMafjK2mWqg5FjMOfSF8Mo2vcYGL0YbrH9QjDITUvsmcMrUewVneIzm\\\\\\\/GyeQ2vpHxffUcwURKcra7mwj2vmtRjv1wpRiTJdwnwKB3RJS1DBF1AWYK\\\\\\\/j5YNVtf4moJIDVfS6+en4M87UfgEzskIgeasCsb9GY16EntaoGsAhnwX31SfoRvKTwEuYtv9mfUy3J2gQ0BUjqUCri9TLOB4mCa7HWtYbaUV98uuQ3FW3C\\\\\\\/zsq4JZR3KT3Ag9ArmKIqOTrVQSsU5tiYILdx0pg1W+BDSLhuglUUcfMyyS98HKsI\\\\\\\/AGqfj9JHOzV6ZhiO5hDr8dFAMNXY\\\\\\\/EFcFidk+aGJuvMEvI0KiDoXOBQMZ2zHsS5Z06cWM1rJpqzmYvU9xljQgS3tjg5wkE668E21p5IXDoS9kc6SKYsffUE8FQM3\\\\\\\/GlHfceZkKgWfFIYQ+v1Hq0GYeHdu\\\\\\\/stSPSt\\\\\\\/Je8VxjC5adY3PVCBksXW\\\\\\\/dVBlZyv6D8KeAh6ZuCMnfNkxMNYcmis3BbNFsAgiSpsR4Q5Gz+WdH8pFlRRwvDHmh3UTrYVuTGP\\\\\\\/PWSeft8e4JUmMbCiGFBZFDGuAS7xPubtzdnyumGNDhdFKn2FiU7muA9dW4bmOj+cXdDPj8lYLhcS3oj0kOR61\\\\\\\/LPQXjqGMJh7dnQYudfFA10s3EQKZcPlA1DXSPvoXXf8WeRC9tSPcKLwJqHhMsgElfYDWpuxhf4VfzHcR\\\\\\\/IdZgGRshCwHS2SZzHKKloT+K8EHZqYzobyJebM\\\\\\\/wu9WSS\\\\\\\/K+FjpAAXSRpX3UHQwtKmCtCokJ7G47dp+GYmRfij7+dkjTjKdDAs5ulFMK6vQ8Yp+jFrnMnPo391IcFsoaGP5PMCq4zDxaELTo1SY7a\\\u0022,\\\u0022iv\\\u0022:\\\u0022DmB5nknvCFQHqM847As8wA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228a03d120fd3bd80ae402734d38394364c9dd21a02635d9df78718b5bade0dcd7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RDzM6K\\\\\\\/t1ZEEEVeWJTY8610e4tX2u56g\\\\\\\/xSkxeXl0D5L\\\\\\\/f154U2JBgNskw23g8O6Wqdrp1BpkyfHqCw3q0psZkxiBNVf1tETPt\\\\\\\/pas6bnC4Ppun8vtJM83tM30D3+2vmUEVEMEvWyI9rftaP\\\\\\\/G5Eg4jiIIrEDRamE+jl33fAWKheEE3SAoWmDjtSPyCE6\\\\\\\/HuM0J3cNX599uvrPaTLPo5IKRn7M7q6NWcvkoZd2dKuUHfhCXd93RMzGG4ZquWwSu0Q0+IzUHjlJCkjjjJHB5jctJghNI7qKvQdh6P8V2GisakjSLzaPqCSK1HIPCTT4nTXDAaTSV1fwsBwame8qib3xZIVNi53u8xoYkN9RBj1+U2+4qe+2kAIJHvSlCQnj8e7XNFMVU7Ai1J5U+oZszUoQfDRbz+b1dNedoApjNpDC11+tmwJ1u6AxmK2k9ZKNaFhnWnN63gNhRF4nhMhYcR71viEkfmF\\\\\\\/eUCFOeSah0hETx+JzarkXxGYALj\\\\\\\/Rf15BCJLONSc5wFnky\\\\\\\/34aKHLkpz12fB1xTKFJiMtVLzmCHayfPVKQBlqr0\\\\\\\/rD4f3OJvuJVUYJ\\\\\\\/6WYFP2Kam7\\\\\\\/7bkHf+el+4wf9jr1wrNXo7mcqViyqoJ7v0XavxjIpPDA1Q0F+jtSYi59gaQ\\\\\\\/QvnYSVLEVPYypOCpGk\\\\\\\/KvJJaVMsCFZpd2A+QjTUjUIsUiNN4aueiLHMytHrRg6pMCHAA1DxNYl53M7qk6g5ZXlHgyndBTc1DbxSfflUNG4ndm2OZzrmkbXwGbi5W0\\\\\\\/isn7YbAIG3rdTzAO39aUFZCXZbdd+r5qvstmEPtSKOfzDPXF8kWEbv\\\\\\\/ikjDM5MpJWBNxSPey4+\\\\\\\/cZyeQSNqHmJZTTbHneS9e2kAC+ttn+9ogWmXu6W4kLH4Mk0Nhwdex12lBqm\\\\\\\/s8rBKC44xg5UDfFiYbHzInH5L\\\\\\\/\\\\\\\/RQ1VVS7qfXRWODjxMkF1E0u+B6APDX8NUDZN3mNsuQ1LidCxNqdds4gy2q5dgFFKoHHeRib4g1Ego\\\\\\\/aiQkBraHle66ViMxKtnBOENhbdbTYzyYZfMAFfIHUnbcU+vXKVYHO9qadflEuWFG6wuD+IHMGjSyr5uRvM+jJkdnUZeOAygYIwgR91lHJZ+5N52JwByR2uGqvc6mWeKLokAV6zp0qYvetprZBLmyUCrQc4xHmNKkm\\\\\\\/D352lGoRy7MhVcM8FRl9kqpSko98o1PPhBzOZ4WT3GkArIMCc+pOFRbiVzl0aOHNAYpSWuFKyo8vZ6mb\\\\\\\/jjaPI00ITAeD\\\\\\\/3I40WXFEcj0me7SHP1IsqWE22bOQzd+ZodCsywP+eWBmWzb6EfTy90DniIq\\\\\\\/d4mZrvyrOfj\\\\\\\/Ah9H8J1VqIga8Fh5D5vzhiTg8Nm\\\\\\\/OcNheT3W\\\\\\\/uogeXFKB2WmfSNBdPcIoZt5WURvzw9i99cLt3gt\\\\\\\/YLMWyHlrPQMit6ZE93oC8W2D2c8RujA4+DsFNqcDU\\\\\\\/EbXSAhzbhWpSt9H+oTas6eswykM+AfcF\\\\\\\/RBA0xV1rCFWQWwKPXZiH9yEk3upYDDUyNdNt5Lo5tQqVVZ2t7Ea4J4Dg9aIkZdoiEhSUnsrZl78jHIgHz2jiCZVz4pOvL1TkvEtOlmvV\\\\\\\/vNvERtn\\\\\\\/Ip5LKt9JpttQIyz9OV1NpT1wPuPFfdPsn+FQwForjq5Rt81WsIvwlvGNaQqlgdT+4v2wDRolXVVV4etVqWLlDLrEM5W\\\\\\\/YiwD0JDLa2UGJhz8fRxOd6mC14gA+RBCWWqQru+z+eMGhaJ3RqcwkE8JmziybR8qM9yAidfdNhzeJZzYHVnjzSw8sthyDPnVmL1bCtRmc87jVaZhbMVNCdX4traBnf0kUqTOkfKnZ8Gzj2FCl5HqHSDVO\\\\\\\/RxmLlvSTXDtov9nmYeqYM\\\\\\\/Lnjiv04SE0XsnOcxFJI6UrOJm8YEi+Ade3mYOCcUaKtGag6jpfmvmrs+YZA+TfOc8RaNZhuIoBTRdge\\\\\\\/gaSDUM9m16g6CrYZuHENgEIfeloYbityrwiBfAWYlzvHGM\\\\\\\/hA53SvpP57Pu58\\\\\\\/qtrlEXu10WG9pW\\\\\\\/7Iy3RIUXJV+Qr03ZmWhR7cpNRb7FfkRP7WgBlHYB9buTShyTuY29JySjueOqlCAyRTgRYiukLTXiyPZveJQQqWfvM3xbeZIhrKVk8LYcdMDqoMWRPeH+IuYgCAgby0jw2f17CpEgrLK2AMdIRI7\\\\\\\/5dsG5czU3edGF\\\\\\\/56KBLGoNCHTSBqOSyeqPgJuvrbolk1gwpw\\\\\\\/YOGgfIjjYF7Ea3DzHzJc1Mv0RiFd3mQPdycF6ig4dihvJZInFMEd3pA\\\\\\\/LM+7Rs2AjWeeWZpXkvNOy6DyIvY4yDgcFg9kGZx0+pcL\\\\\\\/D5se5GvQT3m5a3CuJkHu6j826Z2MF2MLqZS21SHxM5bYQ8Oj4TG+m6UJDy\\\\\\\/g5Nsz+VjJiSkn91adFUi76TzK9VIJ01qCFnQ6fbQO\\\\\\\/sgCGLbCYiARLdyDJZILtGZ+3i5OLgs\\\\\\\/bkM5WTO5ISh7QtQ4Rg6vrM\\\\\\\/m++bJBzPioQHZ9dZJpx7jF7yGAlVoyLLhGOokBifQUwo7yzLhOf43J5RP\\\\\\\/4px0Rh\\\\\\\/LZUmvPkHJUL141M3D28JwAW5EjuokqAIABuE\\\\\\\/tyCNOO5lkRrjrhsZhyx4VDBj9kYqRRa4p9zhAN\\\\\\\/8XeCq7aFR91CKYsQoI1o6TMAT4ZtpPSsluqVDyc\\\\\\\/hdC9vr1oCvUXgtyJQ+alpQrUsFiQzwQjR+SH9oCYMXG8TtRGm2PzKJy4v2WRd9hpcDKhaZPuDBhVcoMA\\\\\\\/OfW5LSvOzVEVORRDxGYfoa1+W1EaGjJJoZD+BA66GFxno7qCEmLp33KTOmVvn6Iv+SScBvnJrUyZevjFB5LuhIv\\\\\\\/CzjUCY4jQBcK5\\\\\\\/6G16LP7eEU3XqPw0EwG6+2bMi4SWk14d+Cx0kCmqDWLT3OtpyUDv\\\\\\\/KgRMwQIQ4NXLRifYnNhzZ3DzOSnEAA0vi1jyDi1c9s7dby1XA8awaAfBMIQBAxrOOngGu4xTWJGe7Tv3DOXoDJasBTNwOx\\\\\\\/oqhvA6lX9\\\\\\\/cqqTM85K41LmQvy11+Pb1Dh9+5rSg0JMNfm8luB0r+98WJk++8Ixl4eF35K8PVaqe9GTuZEM90p3wntir4HvcLMkl+oHxHIdEQKuWqsFmDsRqlxz9CALnyDwI2STy\\\\\\\/PKK93qDW90zRiilKhgyB9Ukl8rlyXNNoByVplb307NoUBzO7Hj+jCYcv58nzWEFbMt30HD2XZjwjIVkx\\\\\\\/Wd3RXmzbkUeNmF9H41LTvNmanCdW\\\\\\\/qpaYBCsrVU3BnryoI\\\\\\\/\\\\\\\/wwpym+OSecDRk6f\\\\\\\/DQYqvJWtDdUO0WExkWd8WczmpTrLKw1T6sEG2uH5pM+FqIEgqSnOlFwUyxOn\\\\\\\/rVaHidgmLylYNPpqj\\\\\\\/LBQjbgZPighlkhQMPZIpuYCDaYisOYu4uDrSKrLPjIN5kXDtR3GlueXXIc4wbRKpnowiaItV\\\\\\\/U3c3NVpZMqeERUXhA0qSLYozHM9EMZ7A4l9gToaqA6cMaSKP0xm+GGEFRj1lnsubHsNLubKoMlHrDSK1+8ucVl5P40LjpPmVeqfodN0DTOSo0tNtPkRhYt3m+YYV3w2vanzCLDzTl7cW9iSJ1Yvr45JfcTqpCwm5YR2oHexVUVuBKb7AHP74BryKOQ9kkikqARY5pYrvTFo+HVdsVrtHN1LymtLrjV5vXqy\\\\\\\/FfEOsn9nQg9ZYIXG4syaFpbYzFslM8XAXKCfHpB31YyEpxyDbske6x0R+STElMoxU3+Qqc0ODWlSglGtHQvkw9L5zuKwhEkSQq0BEPa4Gn0BuPqwGBJ3U22hHqkVJ6erEOWuaM8xr1Q4ZrvPwR6kiEO4NIiQu6Ap+5ueAEH8sQrxwUzYJNFQ5rt1LtYUiahdaEdyHmaUKnkQNrXnssiwWehJM1K\\\\\\\/wqPv8WDj3C08cV0KL2BNOnlRjkvkNLn+\\\\\\\/oa1e4Fcnm6d+z5GKQXPtbtqhnVY\\\\\\\/WtoAb9RC0MwW6+K18Tl8GYMPH9Ym+aTZ7\\\\\\\/kgABU0TTpnPQ7AOxoOdnb0vW38M6LIZZigpgSzt3V9RyViUWF3bB1Kh0XEypUIvVZWvLOIcNe867kQdc\\\\\\\/GNs1p8cOaGiEXsxC2ZuvnzepvolMsEd2hTUky47uP6Umttp9rpgfEKyN5ed3PRH0T7rBl+u3tNR4GbGx7XPEJKQyIWJE0CmJcQO8k1hdTa8am4ud3NIoETp44PVOxi7arZYT+CZZnVuVYEEESZOnL4rbQbegA+\\\\\\\/QjKcwB8Au7nWzYTfnObfgr1mjRqUP0DYG7j0JG7XKD2rQEi\\\\\\\/crhC4tMULJGjwuvi9TH3sPSut5tGCauMh2EhtNeYwZjGXedk+IRk\\\\\\\/IaZyG3SetEefdBUB5bnoFIOmL0TbYhPCuWzuo0RCGLEazuRJvU93gC6ihtNc2O\\\\\\\/CX6osM5nULBNC4krESDe2KlvZxPq9W8qKb5ru9lqYjG7Jw3mSdb\\\\\\\/0dTn9xCsz4ZRvMafmL55Q3SG0jTu\\\\\\\/oPY8UdNZ8TV8ixsnl2vhXOTwS0WyQ7\\\\\\\/qH0HdS+OU6B4igSGF\\\\\\\/5hyTkasTMPiEwqLFzHhLegnPUAFRP25rUGgrkLtv81VWDwMQxz\\\\\\\/BHXpETAGrlRW1u5ALQXT\\\\\\\/7NbDKtpHHCf7degK9WxFHB3MuTR8sb+pBcpNBTTWQ+XfY3vzGc\\\\\\\/JJJSSIO96t87xwAT5f\\\\\\\/e8gcO4lm4Cq0V5dcFvGwYx6bO0nGaIpEh7x8kxWSdJQguRcMH1HJ2mrZe1MHQFyqD4mN1\\\\\\\/XGgYkIn9Gk+qy9imYV8GaP91NZyuS391DIE2n5\\\\\\\/naSoE0C+3DHGv6og2V2WvX77BbjyFbNnFDsbc9KWHcfdLwtPxWLgJyToo5IhmnUYy2ciLfUErscNCBnuYTNPDCmmW0Uog+4y4gLKpMaUm6SAlcQt1re7ELs1WEqzBEFBbWBjQDEvMVyNuSXsmyEHyhll+FX6IPzwb68scQzKDIlbBxav1QOokoG+Nv751CVqMX4RGU92aStOJ+\\\\\\\/5uWLM5Ij3nFhsdjfogv5j2WlodG5vdPQFcTQ8+TavPtgjMuPXHdPrEZUx4ZfUMQGG6R76mgeLrYABr8kQw+25LOv\\\\\\\/1vlp7WWsTJyBNm\\\\\\\/leLpOe0LOMVU5KhVyFa5gtuC5Wl5bdy1gxTy9qKyGyjKlChspu8C9IzAsvnZewsm6cG1jGJ9X\\\\\\\/9Plq3uGZ4QQ+mUY19nY9nRdKm3W88QmuWsE26JfmcTN+PTIKcQPn1feBQtnR6wk68jM9MS2PV+OWgZhH\\\\\\\/CBpz46bFfO2fbUfWamdgR4BXYPv4QKBFznUSiVWsLHnsexXwB7u7b0j92YRQwLGMzM7\\\\\\\/s\\\\\\\/FtyAKmBLextsTnLHStcNGfV1gFDa\\\\\\\/4MyjP9HXBqU4XFc40ywoKqhDQJzl4M4xv4TMwpjObIKs+V+uecxHdKuXYP+iMpNiQVQFUeX9QI1Kkrif4nnLUpIIDl8kPMidD78h05rZcW+OOMd4yW7iDRl1cy+l1WPEGpZzIVm4rxxHGBUw1ufLLSu3FiXs48kabiznI9x7mTuHi9q4LVAa9xhn\\\\\\\/FwLfSuSxCfnvBA6wUPorh5tf4QHq06SldGrnoerAPux7E2QOHzYFHB5jHPpOm9QqptIlZTA4XDg4YrW3vJ4ifmkp9y2GxazoGucdSNT5ZnHuo43wyC7XfnW79pU7wtdCxH502KMZEcCL56igDzeBSgss2ejilJsQKMGBmw27KUXk\\\\\\\/M5IGSj2WG7FvGe7vxaIGdeG3E0ik0voRVPkz3JrFgbAsTpk7lYQh4\\\\\\\/qv5l5a+o6BU6JNNQId9\\\\\\\/vzpIlDoNAM8tQVnPmi5ZZjdJLK6S+5jgz4+Mnnw+bAwfkLA1ZEypwBsFyVbZ+CCtxS6VnPSqr6DH0HRi6WPL8p7ty9D7wYKI02Ird6rXNUyKvx2aaudv5rztCQ3hdP31z+pVtoFsi\\\\\\\/vcGnWLyMrPknIr0X5034PCpk5R76QkTQPFwZ3eslH4THvAiYD1VV0MhbI2wco4I0UVG1\\\\\\\/Znnwmwd5l8HyVgIlgoIEzS2I5ZoMpU3mWtSNkOU4maZMDw388\\\\\\\/NySSb22hjWHCREpj4nALOmaCHGcQBmhp8i8tOUNgnjYTEQ3CSSzD6WZ5x+6ERBCSAGqBVTjKilljP9IwzYADkRznwbjejftxxB4XS\\\\\\\/iHvXy6TlQ05dfNQSV7NDPmjOpHGDYKveVYvcdn8EgREp5Fktz5kZnSaVuaR0D9fZUYeAFM2D36\\\\\\\/MuAmZpH+sX6OfUQH+IcZu1ICxIjxX8\\\\\\\/m7aCpIm3Zu3DgB5l5AI0QTNs128JM7SrcAcguTDkisXXg2zBrC9bZnfdk0ZHTFF3sXVsxoKcMB9JWC0VSbzhjH5rYz6HAMMzoA5Lm+Yg9uywhxF7Yjw4pPLBxpa1N0HeKFNw1xrJMjQEZHQzdwUYFwxnGYF1m3UW+oogc07GjJu1n2MHlaMnXMDOuauPbPmBDsGCjl\\\\\\\/04kYkjZTV\\\\\\\/xcajjgtpKR+7yzmD5CfRX5Jsa9z4DYUuRQZ4icSphjT\\\\\\\/uyEHgWOoPyygVvR4AriuCaWnbWGW3QKjM46vtne1SnQfB2Qq0G2LOd6IDjefDSyXNP84yjBQiuaGulNBP\\\\\\\/y1ay+UC9jkLDEvUVp\\\\\\\/yDWbCGTZeBOgx\\\\\\\/7K+k9C6z1ENRd9im9ZrzrTt7Gh9KNOyWxnxhvekfLFhyPMCFb9UJZeX6ftjNwK2Wil2tA5o8dNP\\\\\\\/UnaumNoL08DQVnlEbQbf1qxAlI66XWTSb0kV2+rKeC4pSACljkt3\\\\\\\/\\\\\\\/dGFcduQPDRRtH4SjV9VGPok61gSpXSDuln4tYC7F+rJJj866lMM7uYGz0E6\\\\\\\/afxtzfof1us6cRFHfSxVISZ9tzBJRd4dcOyfKW73h1Wq76qU7z89iBBYohzLgpmVKTV155bxtKb+aS41Mfvj+KVK2aDh4I2QpLsHYHxeA9FVeh44XzVq7BqMMu4xLA15OgMSlwUGQ1cWvfKNLbxv\\\\\\\/y9j4QFipCpixJ79qlR1smoQJ9yccnks9shI1Hoyh047hs8BhlaKp863p43+C6VklJovMgEjl2Dgbw9qoApZzLCoa12mBp1iYn2JWCwemQS9LAlSTGzuWTbE18uKg0ioNoVQSfMSLmrNX7OSszPlp2Qq5GX5lZ7IVwa23HWv4TxUVMvab2xhF+isuS42qtmt6Sixqr2BGBx\\\\\\\/v8JV7mfY7AwQbXTkMbL7zTY0YmLBv4e35y+y2WNLv8TnbHZikXhj8EPK8IbXxRfp67vo7FY\\\\\\\/Dux+LyTJ6epqqYuSaCO6nqrZabSaNQuhMaONYyV\\\\\\\/JAu3x3NbOfXURP280v581BM+pEg5BZ4EswUpgj19LyD91bDeohzDUox4SDdJQCKJtiS1FceWBXq9EMK3srxMZg6Lrq0DFhxcDDLg1780Kp1Q6neqcthm9nppTjmoycYRNg0+OOVXrccnwLK5HhGT\\\\\\\/rlff8aUGBGZSjXO6UV\\\\\\\/jbcQqO4oVpEGSjxUKGevXpUBFPkj49UWE3LUsLkzCe22Xp9pKqbQEfkd1coHkm4NWOGEbfNG8x7KeOimvvmObUn1iqb1ubmtijuviZcRTpAcP064jR9T3NhVRdIt9VZz+myw3e3qOMW2OMSYmQAyNPSw2Tt5UuotkWnQlSojKK2KLrLebpmk6lorEB4Je2ocZckHeBeMjUt2a8riuFUOg1lBeQtrt6C+rOmom\\\\\\\/TEe2YcsFOOhyK9uicpI3kTEzNkLM+7+zRiKan9Ee509okYeZdSh+4gDAXWX\\\\\\\/R1Z0rWV9llHkcaNGVPPXFCFDPLRShwb7TaJLJuqs+\\\\\\\/Qf2BRKMyjuatT62+BaMfIVaJDjeDk0mO44TtajruD5SkX8pEJMM7TKlfGp3hz441kkgT1LNGREp2a2bhggzMcNoowwsccCyZfSn90UUf0EWvflw\\\\\\\/fM1d\\\\\\\/ubhFlBkH9z9NtGKOCMzIJRJbalTt8tvtEadHdLznX\\\\\\\/nlc9tzQQmkpjyCf27\\\\\\\/++8fz+RvmpZaGyWCqJHKxoJxqWykb3vzHUNUP5W6tqKfyt6378RvNBmsd7SjfDHlQ+sF\\\\\\\/yiS3olpb3eVc+9t9Z2A2jsK4txrHflD4b4CaZgkIP8oknMoBw0zvJ5TWlzQ14ZksM6XQX014dujHLkkCHpAMxujNTNIQz6sTAwtQiyQJg\\\\\\\/OOQMPDC\\\\\\\/6s8nRgLMf69JcGiwkwJ54yI0MkLaR5BaVCvI2Ozvn5QyPMoFrB81CcJ2pOP\\\\\\\/sJUVK4\\\\\\\/mPT3YCmxHGugrJiN49m7xGdNBu5H2RHfbQNTAmD3WEnvHw8ekQzJmFGEvbMcB72nEnFgKZBn4Lq69K3h96xnqCTiylr1Vt6vnIRAPERE0b1LPWOH3fRJcJx0L0TMKnaSqBnPpyGY5naTsU8ckNQPhpWh8rEmULkD40nnvAsAPwDTv9+548KyUo0t7KM9SRG4ASH7yxqGp3aGYVU79NCYiZLZ1NnIK7fu9h02HdyyazaOO7FUzdZRcxmwRrqNJAwG4pSItTacoD9Lta\\\\\\\/M9JK4fypVJZh0v+z5CyWDpNP2gq3RJrSXDb1Bp9CQBqKQYPrVBWCP+frJj2D6aCTy51sU1m3RmX+GDR2k+C2aNHVS\\\\\\\/mHgl+kaVplZgWioZzRiMKhGhiUTFbup4dUdVsn4U752xMXcIjsQisShhCirI+xyrir5Nvrc5FwF\\\\\\\/c30UACcnxXQfvuGs1xOIW9CJpitUu16vlz2ibFO0Gc15nJ3su8HBZtafTCM7VcVhJgJt5pmp2z9LCss9SruRQGMdO75jzFbUMuszx+nZ+3tzu8xVmVi+8mgPQDQ2GenL3CNRO0zA89qZ0UULdu0co+NuIqwKZDZXwukjwTd3FFkxFdD9W86gqrBSBU6eIYPHhcZQU3odlxhsdsKBMYzE4HlWB7i2lE1sEwWEXfZQgkj4NlYQqpfyCdxlgU6LQ3DyRQ9SeRX\\\\\\\/IHHFltSuSbVAlQQ62Cgwf4wvyNYuHxMySFYAKTw+MnXQWKaKl+g3mDq94GBh9m\\\\\\\/hSgtgA22SIqz8z3HL787WbuxKHtejsM\\\\\\\/uNo\\\\\\\/jx+YzlNtG0yuuEabElk51EvzGFXpNq8M9enrhKj2wZth0w4BiEs\\\\\\\/Rihi7wtLhnIxzXBz3savRwvuoytzAwsy6vXkQAc5ZsHsm8OEMB97ptZcJHO\\\\\\\/r9cO92mWUDYfDBvQm\\\\\\\/4pl2d1uEaTifMg5D2nO47ReiB9pxlLsdTV1boQs3oYFqs9T7MN3JgzdYR0tjg2DB\\\\\\\/8Dp3L970JE0xkyvXuhLzGMhOZMVk95LpRkUuU73S6RsdvyisfvoDrNW980GCw3GSVOVngjFy9bGBZlKpJnnz4H1uVXkEfTAoFjLXlxI4gtOKfeyiF+U2vrFO\\\\\\\/1LL0j1\\\\\\\/XaZ3lWMPvUQncsjK0VZJCPm\\\\\\\/XKoQDvIc40PQo5NiTn5yjKV5oYnVzNwu7LOUktozdVciBEu2W5kgHF5QpI\\\\\\\/CQMkUhqAvtfpAywKHl4vmtLjQxZ2aKVZWJLaVKUYh1r+EuVtgfV6Om3qzarLIFoMlM8IbKM2lXklwTrcesJMsAo7w2M9d9woK9E2xmxPv\\\\\\\/Y7mDm+QFIwgp\\\\\\\/7firRhmfnByr14cD\\\\\\\/YvTGCvSviqC5UeqGsBBoVWmG45yylW6XnNHynkGdrGs7lq59OC6kZxGahPxEdHhXPiiTnuaFdHJBGaR8h3KqVswZXvzu40eeQXiAg2ZlcAkTQT9jZEU8lbg6CI3+CH9lkUzYE6M2HOHmk9\\\\\\\/Qb1KHA0rWEw09MiPVm5o7w16RZfPUF2ckcndjts\\\\\\\/qvUebH4c4YIYqL6nn30YgNrxfkEHiEAV4gIQWlins+3n8WF5W61Db3LDF5hD\\\\\\\/HZPa0SMhQQB3pAnhl2\\\\\\\/YezXRtzp3mO8Of7kfJ4BkaKyG\\\\\\\/+y4uWQFQ+aCy9L0dhPuIqxL2vA9gT9BWFCOx6\\\\\\\/uWT0k0K74DhQ45\\\\\\\/pMd5Rf50IAP7y\\\\\\\/TnnvdilmOeH9rHBeb4y58P3WIV+zpsk\\\\\\\/KJpBCKU0DYeSkyBIoEEEYdGY6t4v30c\\\\\\\/j6yoqtuBnm54eZpcGbfL06pWrjiqAlbM4iDaNxDCmYBzjDtGA4mZjDAu3n5vjKsUe439u5EhWoDc+DpI9GdvYF5eHKtFwMAVE+GmmyiQj\\\\\\\/OFKGfkEzisUle40LEaPBf2fjZR5Le1n6FdYHfKhnC9fLb9b8Q6hHB\\\\\\\/41P8vmCohxNURPaDMU2KeheaoOIwgXMHIMiU8NayveWJNtYXtTZ4wDA6GfhofpDTiwCEmjH+ChOQxQsv4hMFiDOhU0pz0dzbVB47upTPiCsNpcWNErXx7+I98wfhdejOpZJFyhhfbC1t6FPFddYrcb3UgB4RfMxayNWanBMlNv2AVR3Bi5C2rKwnrnWDmYCa7GfDAM8J2lphmpI5JuY2KsPQuZoW8Fo9732JF2ZIf9QzMk\\\\\\\/UMA5oIawQ67BSxczTyy3v3X6hHEpW9xozqD2lqdbwXphHq5hlyhrL\\\\\\\/l0W0ZC9Swcn3Dm2rQbnjYkoqBJelXMqUg\\\\\\\/nKUIMuCAqbEVHLc\\\\\\\/0z9NcxeP0UccXIlSURYHrIWTwMERp4RV5SHJZw\\\\\\\/iseJXDsu\\\\\\\/MhqH6RxxfQIklS+LWtkyC7PYBIRdtPM9D4YwzYs4XCblrBRwMo+qZ\\\\\\\/VuO4D2N7MDYTARJRWo6h3njEB1zs93bunGOUzRqoq\\\\\\\/V492t5\\\\\\\/bNZJXPnWrZIN+sYOGPvyI1y5ZPKTIYAUbEgcuVMDrHqbujVkzdGxDlTgYIjthGO3yUwXOp7TK2yUyZcnrFBE5fn9eb32vDTTO6unL2RtEM1fHgoBvMqMzrX\\\\\\\/6byjr2hv1wQ4aFQGZXP5vaTwUQqxWNCyw+TZkQMXyHMOWpZ3M6tg9trDdE60c2ybOf4gAMrFPzX6Q2X6M4ZipV6cOBjOtP6l7jU4\\\\\\\/hPTqIC7bucJjL86kJvK1t3HNtRTt9W6f\\\\\\\/VKdNj4qwLgvQpH2EMuFkadI3Q6iIX4UiepE+EieIhCKqQ+ulYIs5BD4dV\\\\\\\/1T1CSi0QFp1sf8hT+tTghckU+B\\\\\\\/RiiVw8LdntjS5k583teqlVQtQpd2YYqi9jgkGxoq4GIaCsEf5NT9zIjX5f8JhFCjjhFxsrSsNnUDxy5EVWDgJyexf5PqBW2Hr6gVbmVxLRKXJ6zTRrXFgHTxLEc3UC29epS8\\\\\\\/ky4Gfz+nX4MOI7EW3TY6++gAf4Gm9MH9Ks\\\\\\\/8ECpa9tvdnbgDOf0NqQxon9\\\\\\\/cHqO2HlW2KL31TSbtXVNzTck6pCTKnVKKpCLnVKgd5FqvLdD+yJtGVA2SL3mqDCpzDEwPAyrAa3pcz4crgYxo68Ll\\\\\\\/rtV6joYFypwnW4dNxrOThQT9x\\\\\\\/9UXTl\\\\\\\/hT33eZHmH7cWWMmp9kt+3CQdjrmQKOQ0\\\\\\\/FuLmc5sOat6Rk1YMBCaQVJjeuNTPTFvVlN9QqSjXl6LdZJd5Vi04AnnLJ1ZDGEEXBCwoHo9DxbDNy7EAQMTHnZ2vkaiW8kgiqSb2QSNA7fYBGi3i9iPo4064pGL6Eko6uzDricoA6bx86UDQqW3UykDxAG4M7PZdlhWFtoh1t7x4AqQ\\\\\\\/HsaeZF8ZA3OqB5iGFSc9dhq2V630TZRZ52V2oBjjQGggAf8eozcEYwq27x2h\\\\\\\/fNIxP83tT77okbNHXr3O\\\\\\\/RHCETrRQCcy\\\\\\\/KSVxIVtRt5VYx9KEjbLzysz2xzKi59rBa8g4TrotaKgfx5UKRUXdX7+XLQOCsfEnbvs\\\\\\\/8ZMTq8bLzbnSm+M8Oe9cv5V8pi+ka1NWBlobB9tExTAld2Rgw1z30Tb5IZf+tl6ypdLQ9lb4EvDQU6XY4xnclMcwUu0EDeYYA5C5aEuBhTD08nQl1GOH9xrzEUr8DFA2DOU+UVtpB292Y8ax\\\\\\\/cMaE9VmvluUZXS3t4mKtmL486VKt+7HdcB7mb5NFx3MMzMPChPA\\\\\\\/8HNNKQXcOdyDjLFp\\\\\\\/MjsKVrMlHPyourJQn9AtEYyHHD+4YFsBacofN1N0sxdM6tL16tdX0B6jhfmcB+foidz5oL1X9GH2+AGowV1a98fiOT+UwmUMkNAkulpfxC5qffa\\\\\\\/GJY3mYn7EiDAM1cFC\\\\\\\/Krtm3Isj8zeTzJxdd8AsR4qlTQ3hxLy3MZZ92GWcI7meHrc+aqMMJq9bc3+tJO7tTapBNeceamAxoJwqjUrTNi7eqCIaR\\\\\\\/HMqHqZu0SELFl7qNw40jZeWetiEcZXUvIp0oFXndXhnPMpyUU3a81pI5RP35Yc7xau8ZeWMmNSCcDX2w6COy8E\\\\\\\/e4VFpSBIdlW\\\\\\\/EP1fEYnC40KN3q7JX07T11oNyfnH6SL8YgsG0BkV2XKAc3i7ba\\\\\\\/kJSfTB+uvQ3VZvanHiloU1arGHwriyit3JbRB+xli42wNxqeXjZ625tWyN2v\\\\\\\/oR6iDv0NE4HHlGnTUmxKAjYvBACoB9zsWQM5z2ceMEPWJh884qP9RbQycGX\\\\\\\/z6m\\\\\\\/RyeFtXcxJhGf5qk6IUO4HsX5mlo\\\\\\\/28CKbkEFOq2hzA65HTqHTRsPk+auQL+KGdIgx\\\\\\\/lLFnmF2ISOp\\\\\\\/GDfPa25lLM+I1ImS3wp0j2A+D9IECOAV+5MURaPuRm5rnR9ZFbcdjMpAvbAeIPAmAI2ttfHCIe0Ud7cpqyW1gMh\\\\\\\/UGpqtcJNt882QfXYbyFepQ+oNHXIBc28g4dvLJy71Q7UgptedpoANl2ikVUynF8YWymrsm0yaxG\\\\\\\/ox7nHkCzbTEGuFDzK129\\\\\\\/rD6njLUB8p09SyNXa989\\\\\\\/DtUJD9huTFQ\\\\\\\/y6czt1f4D8wSwgy\\\\\\\/xgbzOalDFcuCfE6JnYssQmQ3wra\\\\\\\/10xC3OJHIUTtFpKFhR8YuWijISOeA4zsrX0DNwg\\\\\\\/zYMAEyiMwmVB4ky0NviDynNzMmh6cjUBBHSbn6DJ23lAT\\\\\\\/pc1GAtKl1vn01OQu7tntkpkCJBwQpSsoQ4\\\\\\\/uUbQaOj0pSIZd1mziaLiJi\\\\\\\/IAOivRJw98sOUS9tAcwFxzXo20WuVy6gsXv92mBaN7Gv\\\\\\\/ill\\\\\\\/NCyCckUyivpLwpPBdgYQtFqp9DEAr1rOmvK0bSVrv2yyIedqq5DKO+dBqIlfROCIEehhewSLpK+XqPHT\\\\\\\/nWRgIRKc5yx+pD943mc5YND3rL8ipSWByVcTQpWGKRY5jZfwtzoSbnLl79t1LUm\\\\\\\/y3b5kMM+lPHguMIaCgyQExsuoHo6konawvkTSwG5lHcTCIT4nHRVmbOlNa5T1WQNcMkTl+ze0fmTEYZ7xsZxq37inpt4Ti0p4teeF60ieRE3qh1lXw4MWlaSG197AKROl20SXLQI2hl6YDe6IxQ0fHJGqCWBfv0LiTHzPTj+IUF+xB0LNKwNP3ARpg+HS4rG05hgmdOc86VCi13ZqYIILEqVS+jB5G3jT+P3j3gMYHeOd4PZ49IfqWpnKwq7Xo8LacxpnxP3guNIo8sP046UYB+SIv8rbbkePeLuFgUVSsBeFHXXwi\\\\\\\/eppbi\\\\\\\/5xo6xLW9ytE\\\\\\\/D3SGYNmX7rU3\\\\\\\/o6FMPHcuZVyJju8DAY+Ymep4uHoP\\\\\\\/JwoGuks0m08ZxujBRVqXPKhB6HXR44Dr3Y8pO3mD4S01fnm7ovkzqTU32g2uhfx9Tw+rwx2GSP\\\\\\\/fxcbUwerOM7GQLSHeQboQjf4hEZb8HyyW\\\\\\\/RvRk6qp7Nz5OQDwasSV0sZQvOTFcdboXmLekUeNdU1\\\\\\\/1wCf4Ugnp41wYAYtIJgMf6m9Yf0nNhYnBcrfs8oa0eXwbNulZhoUuD0kr7UT+Hi2JPYWSJ+H2R9vqNHlYaAqwck6sfEJYI9Ewrzh+\\\\\\\/pU9O5AWl1lJfXzW+f2fOW2xKstAK3LMDjgSlLn5KAtzkyPTIY8dmrMWwy8js5bnO4F3PFywQexPEnHfYGH4qrowuUJy5h72mSzhpVFdYhMRyRTmNldwDfMBsW3Aylwi6agk\\\\\\\/a\\\\\\\/QhfnD\\\\\\\/0Ijkn\\\\\\\/VKsn34I2Qs6co8bDG5+xyBBkA+GmoUn57wwjgk+ZywWVY5PSxZVXpfMvVLMkLOPwfSEuLRTl6mKFV4RhWm0lbCMKlTB4MFbNpniKo6QRKs+rYi8XFIxM+GHx4nx23y4DsV3w17ClbelfQl+eBiw7hiTGDhTT1r6k+dpbpJXufcNnl+bzFgGhv528KDsybq4DN0m4MnEY6sJCETirFqKaFH+Ar9bYlKKq53hVfjdbAqOsUwScCSNjMj4nb8mtRZRBJKLLTO6bi9KFMVMLK6VWTInsutCVp4lR\\\\\\\/KDpxHTPeqFgQ7JETP4ebSDPX0oosCz4DWMSR7YzksGZv58h54OK0vlP18+DIwkP63\\\\\\\/kTzY11vut01m8J6XKhbL22uBr2\\\\\\\/bmPm5M0KbPogB47iTLV2quFMwXFEU3tcVNiGLNAvoXdqd1jex6rqToUPgVD0HfjII79qZytb5KSGqrojntu+++iHKTkblDyK9Q04riiPXfnQhQVEEBOaQDMFjMqYGUAuusq9oEvCXoIxbLppFiloxNxv7\\\\\\\/Js3FCs54X252PdsMXOvZdL3x46wZ44NX5nWsORRQfjQMlZqyiykr+Xcsaw0gcRC866Q16cCPj1lDvAHMcU1A1AFxrfVM3xpXbiKcnFoagYoOaI79819vr8+oNCQ6DyKyztE22iKUkT7yHzRTnqCvLlfxDP++fBx6vmyOWiwj5Pz1Wau3eVNa79R\\\\\\\/F2OTwa9ZGuQp\\\\\\\/8ImIE6v+tJ1dspW7lqR0exCyyfnNSIRE3PvN2BnN+Bq4nZKRPXlTH8HWd9efZ\\\\\\\/37rrs5WodqZFOkIb1RiW9yMDAy10wrBxsb3sLiivWIk4Fdv0HC9ETtqS8nopls03sfOkxaG4S8Aljy9rvGkXEtAIaagRzx77sE37JFLwmLbCClMprVMBAA+uir8cZ5xIadJIaHg7Au06oJ\\\\\\\/NH8gVDB0KXoOqMDd0n7ck2kU9D6JS+7rSsfU3w4TEuY2vylimq7noikyTZBB1QbdsFji\\\\\\\/jqDapy5Qpb0AM8cen+e\\\\\\\/yIbCIKTApnRY2bu239vNkFMFWOOVvvMgZt0DVbMetmCNTqaL6JOoBWgKpfU1oNrW5ZbzLBppVhmjlQFU017tslG8T85t2DL2W5YlyuoPsGinvUyIn\\\\\\\/Wh3v3z9L8DvgJv3NdDpBz8vHZzLFAYFpU9\\\\\\\/GF1xftjM5qK3WsEVIvonHeV2M6JeRVGm3Z5uGDqnVEmz1Ubyg5RHZ4tSiItKB92QWGX+O2BqbrIl0d0IgNUqiwTAYc1azYJ3BcBGL7ldJvDMR7c3T5fbUuybQY\\\\\\\/NFKvNtyH9H\\\\\\\/NCqYezp7D6ArDTNqmP+5Bybu0B7jt3m1pek9W4X5QiCTl\\\\\\\/m8bEw96h75mKIXM1U3ABaL1u07ntafl7jOgUX0GdV79xnR2e9BvXsBHE29qsKLm1JvXteU0wWAKt93UJ+0hOCAaVEcoQmzG\\\\\\\/EBvjL3Vmic\\\\\\\/6CcfleZm0E8jIKt+wbEu0aKhYOUVJzUMNv9veeZcpoI0UfO\\\\\\\/TKsUEHmm77uLxCbEUz6Ev4JqTsc+Errv8OFAHncRS8pIn076OWXKG\\\\\\\/1PfNZSGvUs3+W\\\\\\\/pzRLpx9VQLpfzfi4gbLihtQPFMjfXDGKYAakiQD+qQSGNIAv61jvPOhDobIPOEzTdhfSnBNxdXJRznJzeVNo4z\\\\\\\/ADXE78qXLXsh\\\\\\\/P8cfAVHOWILfq\\\\\\\/5AlDHLQLrOL9nn1xNEcXKVF\\\\\\\/c7XpSxLRU07UAPhGpHtNrYchfEB0w+7x352Y0OGrPjqVsy6H04a97yFv2IC\\\\\\\/rsmV4gaBx8rmHK54l9gv42RGlKSYKKujkp9\\\\\\\/nVS5kskK1HqcFnYTvrbZc+tYLdHqCAbgsTIzNu0TuCdvod5JZg7Kr8yPfm1yXNYsY7ft5TC0Y4VQNRPRd8cEfCa3f+RaTEafp+qcKZ6P1eVlBOl2QJaFSr\\\\\\\/NCt85sFd6Iuu3Vs9xcxLHOTdfzCjS+7tTSEVVuFXetRL+HqCDQMntOQc6N+ApdEwELW07jcC08Kc1sYS1arasSuKtzRmw4o6Lcs+UZPRkQ3x5M3tcZc0hdLMY1ZsmDqSJ4iyBR0peKYYaQFexHLNnwqyUIcr0NgMxSGjyvVNO3boPUfzdr8gVUV+Qxlfvy3PtPU9irJTOHaYZWZWtC3CAn0eKxMkTUdiMhzcjnMwJWH7ykfpxLDrHOxlMHekRDfAh9OPHf2xDKeb25udJQ9PrSdI61AhuvF2pkvZimYcIs86Xn1jJ4OWMPQjC3Kx2ftJTMof8\\\\\\\/qvo8jwnWHfMM2ksBXSzNlbTdf2nY5GrrrDrEQt0ubZJOrns8Gz567RA7FsUA2SMWP3TKftn4hSy4Q\\\\\\\/MXYsS5xXdhJAboM3tViMq8YxvCgr0gNoXaripHrd75Di48Ix6zalaF6LGO\\\\\\\/pgkFyXZU9waSN4sTAucu4FPSk3dI42NGxyNYoe3r59W\\\\\\\/nLKm0g\\\\\\\/cXPJNMq\\\\\\\/0xAFVmx6sV8qHKaFpD4nuFuQlv9vuMo+h3DSndYlsV1KYaSUFKnrs4Zw0uYbxwvqkqXpQTNCW22GXVIWae09jXLYWoog2Q59onYHDkYswbVotTLHWIuzJXT9+OjeHc7gNZ9n7205bGDC3pcgPzuQt5fDgTg3Qyv42WR0l8CO1a0D8G+Xww2RKY79G1YEZ6jSgGnCP7hGd1ZHmU7U1s46XISuQf0k1WuevMSo2NkObnPBPGVBEuAsYF6I4ElTSwqQm+lON7oC68tKp+2vWPE9WaVk5tIhVeUBgZLJ0dD9FAZtMYVxLsgP5IOxDvjRJdcqKIqEpm\\\\\\\/FoXyXn\\\\\\\/eDN\\\\\\\/xXifjPbBE2csDe3qcE5FcNIelQfIBJU6Zg4wvn2yL5n4RMmgpUIVzCUbdu66OzynZdsdoqDLBZSrjuU\\\\\\\/CrHGpdK2Tko\\\\\\\/lZLn3NWpetTBlxIRpeawrFY\\\\\\\/Z10ou7LKahhFEj1gOCFvd6yjgWbz4kdql4IMys7kF3Kcs3NRCAu9F0QL2eFyB57oq8lEfPqA0kZ3+HpVZctT1yMSnXAWNueNy6Q6I70KJBtrpAgSv2LqR15annanC1ORhCDwNDho596TkjIXJu+2CwUHPW3uh0KKHOPbv\\\\\\\/okW6ouRGm7915gmu5LCibU4GModcySdpML\\\\\\\/nQM0sp5ngvQpLWz5VorAW\\\\\\\/FB9m5DqLXkmPa0A28PxWObDE2JNbybLTqXfHxVQthJws7CFYdXfo55nAaWmcnP\\\\\\\/5+bvNSfTKhunRx0YzJvE\\\\\\\/fatP0BvHhI+g\\\\\\\/nE3+uJ25yiQKqlz4vHtZhHvtFvGRsaTfQrcdqNS\\\\\\\/DYoAhSgC0SkPUe1IeXl4YW6XeNB7M+6fWcamRm8NPm8BjruOcP8K+rKHvpiLgNZw2p\\\\\\\/zbHDgCz56sQkPe\\\\\\\/dpgwkx1\\\\\\\/desLN7mL1ua6Yp+xoFvPGbTOS9CjggzsECotVSmrUmby4xZ5kfFlhP2Loja\\\\\\\/c3HjzNHCIbW+sebk+vk8royfcJ+saAp8EWF\\\\\\\/GuAWt7IVBxkqEAQhkObnjU69jagjPtIyWvpm2joI6ZIBcgiEc7odi5Cf4Mabg83eAPDCaZ2PSrMNho5CZWjj5MkVebJsfIrME+sUDOiC70OH5Ey+TQIlTzSTUJ\\\\\\\/JaoNOynL02YfZlzg0Sg8J4sm\\\\\\\/tdsxgli+djxROtLti\\\\\\\/mNbrFSwwf1x+rIvsYSKOUO8OFurL79VftO1LAXO4ImlZuFgN+0hF3Oh9pEGHEH2XlnSeGVt9X9lEnWp9049YBac8fdWzF\\\\\\\/wzyPZSOiOSbtkndZvRq+HSkWfSJBPhfXpbUMf9mFo8gphn1qszeb228nKAAj2kHCsFo5ooEVXXjcxkMYMyJ2wxR7o+zV\\\\\\\/v49hDSJGX31Gfn\\\\\\\/lqh9Txm0Y++B74nKAEz9c\\\\\\\/Xbm3ud5ONxRGUZaT1rooo3jOvMAQHGlqt3GZ\\\\\\\/UxgWlyhDfg9F3dictykOxJld0650Z63rYs5Q1O3PRfU9NL0Rtl1U9cQfjQs+ltOZ5xo7KQ2VlooC\\\\\\\/DzBW3AV6ID3CpDfMChhBC0g\\\\\\\/xiwzIielBdcPA0Fg7v2dSZgI+Y4tijfBdCZUkW1PPFukVxLmrAuL3VzI1jWc+dqCeLMFT9piApwitA13\\\\\\\/IIj6TaVtB7M5ru0jVH5chDTs7Ouw90TzrflYmCJUHZERDlw1XQx8rmK4Jxf0jB4bq9JA3pX0nuJRNeKOJSsYToDVFrITOiAdSuBikDcw2VFF8+3r\\\\\\\/XqtZ3nofrB3CkNe3MuuHh\\\\\\\/eZbeOyc28Bvxxy8UM5oIbKhM\\\\\\\/PME6ZgWuUWHh7CZBszgwIz\\\\\\\/ABUsgPKMPHrQ8dvNxxU7OAKcl5npF3HI6rZBwtXm\\\\\\\/jPeN9pwdWkof+UVIlkfPuWHErnjHM2st8skTxHZIBI\\\\\\\/rXR5+VDHS9lbhoxKZZiXLtFr\\\\\\\/WTKhc5Z+3fuSss05uyHtIYwkh56rX7T\\\\\\\/ez6R8HcXBZgFGCLsRMSJ+K\\\\\\\/6rRevgDcMgtrETiiE3zrVP8K2z6YXo2uGi7S1ay50ScEqHk2mx07IUI2ycem1RxLyOG9VMvV2dZl1EXmRDOcoUmn+n40zLdqyqsbbeHJur0wBMMeSq38qQttZaPCjULTpHZ85dydHVoSrnz2u57UgjbcLPnNLVj3fmgxHST75FXm3ZnrQA\\\\\\\/xfBOczJBoL+8O1YfmebWTyTwajl96cYTc0eTRuk9T7q4fFIiC7c8xYJCpz3RPYm7Xp+LeM0sGKqmXyziLuCTFEoXQPus5T7WJ4A8MN+jKMlqONgLXZ6BS+FSGrnUK5TYKbWub6gPo0latVEa+Bz8MTyf\\\\\\\/fUWYnVsRBNYtZHuqlBShnD7Q0yEwhVTId91amhIn1hlk2f2bF1iUdIjnbkfuklYiBUL2DnAbAeF2g5gO4SkacRH5JWibT0DOQkMb1GduNR9GCs2BJSR05TxkwB\\\\\\\/2NqK7p7M0fMY4VAo+7GaSNVrNSqhG2TrOZc9+XhaF757fBYYp6hLMtEsnhiVe2AWotuupRc4SZI\\\\\\\/LtugX8ibc\\\\\\\/cggKwUvT\\\\\\\/8zWWIgPIeUHsF1SUWpy7TtSlXJ+V09ZydksnAPOAARh7eAq1FKMWWmXCtM4v+q4Ix7wWYAIVXxBrEL7nIa78XVh7okeL8fLoGkLhsQ6SM5SnhUtfTQ0Z5iZH5Mq1H7DcaQNxnA3LenMEFa4hPdzg2mFP3Tbm5meXok9KND0prYURdsHzXxU\\\\\\\/ZK1quXGiR+OtIGIkUQAhgcaDlHw90\\\\\\\/AnbuJDo9X0p4R+EzxySQkent0nnafOjBBOG0VwwlC0DEhgT4mpZqGr6pa0+L1Uk4tqiZYQlAc8LCsNR++\\\\\\\/y8XHpPQHZZVTZaecgvr0woplGI3t3qkZ8x39TzC9SF\\\\\\\/SPbZTUvJvwkE\\\\\\\/UVEmC4eOFdNGrla0h40A0UfWum6j0z5mjnDi1OV7rx\\\\\\\/V\\\\\\\/UqML9lXNyBQm9j\\\\\\\/2j5stieuLVTlv56mUX2\\\\\\\/v8g45Je6VI3aLBF7kf5v8uJjx5jSYUu5b7S7RU8u0VR72vkpCJ6kTRoRVMLBSSk0cwjEEbe98SNZrt\\\\\\\/+P4pAED69CfSTqFnyrQDHhF9C8GewZ50TliYJxlXny6TFPo2oDn42k7QsuCOWnoNbFxJ4pVnXqHchgwWkXbxliqDifJ1pXjLbgoAQkFVP9j2zqV\\\\\\\/Pp0dK6lq2IwsUPWFj\\\\\\\/zS\\\\\\\/c72UIUSpSpOPLwdtBa+EXEmyHvWhdjCphlnpeyR9r7tiLet9H96qtCAnlLCwvDAJA96UcOdSAVgf\\\\\\\/6cnloOvoYhpuHT4oEoD2e9qQFw2MUrHdjXYQAX3K0YX1\\\\\\\/Vtx1T3iYEfhhAw9h6RnO\\\\\\\/BX\\\\\\\/k+g5eKVvbQ3g9a4+izgMjsE3kDZfAihg6GE4U1iyX9g0QXIHi5Ycvk\\\\\\\/5sBfMDU3Moy3x2zO0DMaHqh3gseJXLP2TrxDF1JWZvi+OR5DoaWQxYSSmCY+JX9Nw0mWgFWPCotWq\\\\\\\/HlG1K2iCOujRhmKAFmShCmQSIWC3K1EHGZB0R4UD5veCzVefE38jxmal1Mi6SVM\\\\\\\/8VaoS5\\\\\\\/2QcjqB0iq1BEJkRxlQ75BQm\\\\\\\/ELvuV1FFYZ\\\\\\\/LELlPhK2F1ss26UjKmX+QZhiTJfBU70D6ZLnYNkhF1F6+ARDUfZkOqItPcic7GcV++cI\\\\\\\/70bnqWgVy1QF4CuQNWmVtO4nzYxstlgGkx\\\\\\\/NiEanQfkRX2DAM4TeO\\\\\\\/TfmWwPfFt8ca+Ox2ZnrTnV\\\\\\\/RgmOTfSIU6FB75SnheJoRCMwXehk1HRmL3cD5XeI1Arbslx3xShKg4gQiF4hipZxMuSgnmqKV0qF4iaDWKPRrA9ypDDj2MnQdcZuzS\\\\\\\/5O60x8g+zb8oTZPKjQxNQtZ0A9pz40ViTJSDzm7lkDwdmpMG0dEtvXDGVpFLsgO3pmcXecfrU\\\\\\\/y4j76EK89etKXrTFV8ZnID2nt8Gi48pcKzWvATYPpJyRQkFxJGlGlvGIPir1uC07EfCBtTSb3IqjJ3ocHMDbaajt6EfMBuJa+VgWz+0p\\\\\\\/U\\\\\\\/8WiMKvqgA7oPbXIpRhTdtXFWnd8fYMBtqNWMQ6Aqrku4ARQSbp3VkexzC\\\\\\\/M5ccG8po+fFnCyM0aAGTQwOhYeYyKhHOGCxlmw1Dkm2N1sTvaGvVsoW7vwjvlqjBKDkwpdZ7smA5u3neEXlj48iLFfmk\\\\\\\/u8bP3YJYI\\\\\\\/fKbFduyoawMls3trd3j\\\\\\\/40+h2yOCqCkuCJFYe1ctBNM8Dk7zqHL6bWS2zqZ4qL1X\\\\\\\/OjpO4Yuk+xuG6mW5xSNSWeFeiSaQ2nkEdmIGuL+xsBq8e00w4n9VLNvo68ldm+FHQ+k+PgUjo0Kc8q2dkAdiQ8N9NfjJ6AL5qT43CjBUxfvbFmU7BFdUP\\\\\\\/Ek0CXGdNAl9\\\\\\\/\\\\\\\/PHMWfP43oeNA0Nk5lCI3muwWqtE7oNuuQIQYAklthkvf9fiZrjO7GbhtW4g4g4iX1LCdnyPD7CNJy+H\\\\\\\/iWSkU+JF1OejvSUCovucvRHbzSPd54M0CbtH+MHVCgTE6yzDAeKEepmX2M3v0bP8nxmL2xKt7+UVSsrNlk85JHA12ksED6VmC6ar9Ak5bDT0exAjuv1C\\\\\\\/g5x51ho2womVSf9C0ZqhLOF9Sskw5hrgPHAFaG0zCGTHriPtU0tgN+miSWC71I2toJvEAbs7x7M9meEG6Wc+ry2qN3feKgescaEZNZP+I01yaRlt4kq7CIWoUDP+8TNmpimE2wXbj2bU19\\\\\\\/rcuKpMWQny\\\\\\\/Lhi6Bupt81jmd\\\\\\\/hm4F8VW1sry1lYBbcA1MBmmHBM7keiECCJh2Q2gHQcHEstdnLGDlAPwIlu90T4dM6ONMnxpG9CPydTGiNfIRCR2eMlKaqZNGkGxqQ0avG8\\\\\\\/fDNxIf7spk2b7QkuePcChXfctZKm4e84I0t\\\\\\\/tiuS\\\\\\\/c6VEO7vzGN7R6cYNnr2MeykXbDEfbYPGVfdP4rXCq4QanPHPKmbEAv\\\\\\\/KxEOyBwD8w6lfNJCWb4prKpwIYz5\\\\\\\/jlKcMtfev0kJI8sttWdnRSRFSOm0+S8mNdsvYHDNIpNeaibAAX7S5XEfOVZ35HGJv84m7LHOgrVVfPm6BoNnssHBhhJ7ac1wso7qrRg3GdcHt2eUqyY0pYXrI5FCkc7\\\\\\\/4o4sQG4tK\\\\\\\/uA97hLZR6lXEy4KEUJqkvhsLIA5YjSQ2Zkfc\\\\\\\/JxwjjuZ1rVTRZQj0wcVrSCch5sMC\\\\\\\/xeKuU+eWAGn32n9WyqvvmB3hVyalIikIJsM5ERct385g2b7BsrqEBQPMN9srdZXHf9NTDjj0b9BDlsR6neckSHqM6vwqllcI98ipUDh48gtq\\\\\\\/APjsN82TqsA7+H78CapjQxONOL66Ts4AOXIGbuZM13UkS0UcCOPQ7pdHX5VttXl9H4D+4NJcvku0VpMB1CczCBcnBxKfgeFtXXDsV0AjXYP+oHQgA5FqGW1o\\\\\\\/LYczMFeRnTHuMOUmksffC1kaa3n5Xyl0HRfLAr0rrHsR3VwBJGd0016NE08q2SFPo2YJCg6RehaFslL3NGq\\\\\\\/HxlQWlT35dljf3dbMyCbGfDb7jd4nAWR8KD4gM71QFwbBccz+HxpCVvs3k8l9TcVzEQhZy83jhFFSxOB+cDZrC4scnb1\\\\\\\/YWzDHE6wBOkOOib6CV1gI\\\\\\\/y+NYXpyKGSSW05dy1xseEFxTP4RXDg9yk2kHdTBcRKV3M\\\\\\\/plQHPJHITedvtEZqMS2ivsS7d3wc\\\\\\\/9J6EoY7dqcsNMW3WrPd7Rbhd\\\\\\\/lqQVKxrpFD5eJUl7mQ9KZiwIMB4Ko6yUX5\\\\\\\/8CmWi8Ir54xUjQvZWDByYObUhmupRpYkif37kDcHudvFkTkBPg0vIPTm0\\\\\\\/EQz8lkFt0cRPuNndrj7XreQwXnIxQcCMShaxkkccsFqE5p6fVw3fDvtyQuR3LP8Ngu0zbI\\\\\\\/nzdAVoYt+lDBwIM711y7Y3NV3gVjIENGAZ9F8ix+BoTp1rM7IaBUpwL7frV033QZBgQ2ZNwtqa\\\\\\\/\\\\\\\/LsUBHksUhs2\\\\\\\/W85\\\\\\\/EzttUMXh8bMoyHlU6PcaqPkJVkGH4IzJ9adzBVqwGM9SY8m7ysb7gNFn+mOHky1iWbOMMYnR0Qq5ZEbdWXUTJ7qV2TZQOv1jhP7o6SMMIxNSJdK85wh3gLwO6+Wp4QCXdbmS+KUwzCx9zbETAmBX5WzDoVQ4o5wVcjheIQTTKSlBwfs93nNLOVXHmFbSAikW5EieYqKwsCi3I5xh6TtGNWpfALZCOznQAcCY6lwRvUxONh5va\\\\\\\/2isnKf9qFDacFA\\\\\\\/BaUVaJbMIswCZ7N5Vj20jwyPShRTdPdwrQM9oi4KbN5kKnuIY4BHrpfFOXhvLiIGptBW9Dv4O679jxrigBKBx0sD10jyQ\\\\\\\/g9W7Vlj1268d1K7dPRASak4rxJPlQvvDKqSjb3JR1mBDnkrwnjfYJW6hMw+YjL0KkP4YW06iyeOfwgI5bCEq0Vn411A3Pc1b0l6kJTAB2C7Yc67P5zvOzB+oXx3I05xG93En368vMwn354+qCxUO9jf93wFDb0EbcLxTmlv+3VL6qkZTbj9YKJszak7JlwtacZsknTUqBJxhNOVoEQtmNYUtZ4lVi98Af9iRkw5NC6za5hLmTqZd\\\\\\\/VJiyY4bpIdSrMy0GrktWshdMoZ\\\\\\\/2qTpnu\\\\\\\/iomPB3l\\\\\\\/gNiZpyYNZXbzktOcpinrvj9ZsWzTndkix0xE+A118SH5YSsDMKbRyVV5JRg92WSX4H1yLxoMH0Bjt97h94sqIOhx4uQe4EtIUwevB1clS2j8w0wcmEUkmR\\\\\\\/ehFAWXejMz19qHlJamMVgLV0pU5IGqdrUHRlPzM86KnXXG9lwSOY5noN9\\\\\\\/JAkt5rnTWtANwiT4vYderD3UxHHHw91xmBZ+UZrK+MgAYfW+nrRvclxqagj6zpA7p5iKsPWfWanbcrx9L0uwnm6T8rjU\\\\\\\/J508p4FLb1bjVLGmbEv21VZaXrdZyoIRe\\\\\\\/e44tF+4k6TM6zK1ItdD9yuhShSrfUgqaGNBm1kAI2NJrQq2z5Iwfgkdd0D9lotMVehvmPYnRx\\\\\\\/yEbnlFjsadrtMfORU36klhmGd4Y\\\\\\\/jNCCran+qLLuFKDla3kmofgSkPs1qSRQFuPxy3epvsOqRvX8z2id8uUj8g0Uo88q3PZAn0mjuuA3USr1TXvkda3qo\\\\\\\/0qCzeFzW1SYgdvVTI4GTso6gEHlMaPT5V51q3olX0ScqGQMgUMO+ECeDLjbgcKmXeQd1bbLEe\\\\\\\/4oSr3rsjqCSrtpvECXc\\\\\\\/7niMu6wZoGXudBp07UdIbN+GYUMey5ju0dE9yqoQjChQbttTYtahH8rqvN\\\\\\\/gPgiAROhlQPhoV32DdBbfHKrABWwwHBN1JcOWTqUVnTfbTH6G3E8mIhEQk8FZ56rrab4QmkI\\\\\\\/CVuK1u1i8kZkJ1MCsr\\\\\\\/hOCwJesjruxB7FXm14KN8OCECvnKzN5eTsf8zzz7qOELtI5y5BgrdokTCLmWBWjnA5KCVoNCUwAUc4wC37IbMSolUYSN74Yh9sVKB0pi6xnyuP3HpBF2pCNdgrrTLUxxpKbCbhFcxUUN3MaAnlbheG83rNl4KhDs\\\\\\\/9HFjf9AlLBhbxVkwK9nK\\\\\\\/D4J\\\\\\\/hwFMmQDl2vbCg1U7CbZhrhecct1w1wlmkgJsXOPBilY\\\\\\\/vWR77XCxxgKANYA8e5x6PFZhAqgJq1jzw9UhaUAWOP5aDecacnnjoedJoR9FQk\\\\\\\/2mnpX6RukZOnEuomqB8ukzyS2xnibDex5eecKYPc\\\\\\\/6j5iR13H5DjbGrxvniAcOmVHokBKAyYYkyxSnS9f9382eFO0yr3AsDZ2Ufpy+g72nDBa8srFAZna8ElIKbEo3SAsMw6bjoljsW2wthiMiLHuzXFT0t1Xp64idJjYurvyrsIY\\\\\\\/biqtgMAvsnBLOUrNjY+qmxUWe149pkoG+zEU6j8LqNxX66UFu2l3cbxIQbEjVrGl\\\\\\\/ewM9fkiSvc3blR7iu+X9FLIcdW43i26lvhBJ2VLt\\\\\\\/mO+9ydsUYbh7O8iloqUjLSlLnT\\\\\\\/\\\\\\\/d1TzZoIMuZg\\\\\\\/mVKZ3jutrm2cGq67rnSCPDdjygGwXurIKWbSMq8CuK\\\\\\\/dymft3AIffKOecbLR+DjSW+K+pO3q796EzKeVTcwFPeW0E28EStftOONam8XQMCAtVwan6dWKL9\\\\\\\/QGWDZgvkqTAzX5yn2BY4dUmPFIWEEo27sHBRfaHC0CA6Cfdt8DnQzeqOKofhg2xCPqsZu2R0itKLYWHTEGDALtnxI+3fLIsSMIIll\\\\\\\/WiQOhQMILAJ7rWa9rYqQQ1\\\\\\\/hJBkHUOWUMNGHZJhxnJ7e7ydePM0kmkANdLPeSWSgfdQDJssOQGO3lKmdnJEM40Nzz9oDWUT5Dk53y67yW\\\\\\\/r8Qm25B+9F\\\\\\\/91JCmwlqsLJwMYyqtXgT2wii7ueTpGm8ipuXsmPqxGWcZcPbwSnA42CGKv90Kdla06Ow6Ais9RB06gRRVO3byvGKJoxD+wClf5e+SWm7CkWIKUxj5\\\\\\\/wENm\\\\\\\/vdmTSYWJR2rMzZeqjaHBmOmLRMyt5GGkDIHJAVHWPUQZ8YmY1INPLTsxxkQ1JMTgKRU63PymJzt0r\\\\\\\/39md+iSsiaVJPtSy4ed\\\\\\\/\\\\\\\/ShffLYqNUnOKIu4HGAAAB8EM\\\\\\\/NZi3whk6id+LxW9XRA7AlfGOY4S905s4n9Zf51LzlBdFCqzURKxa6UvlpVq53Qgrg3YYRmiMv02WHye09vU+Yd3MrB3\\\\\\\/T35+H1XoFsvO03UxsYARX\\\\\\\/j\\\\\\\/+zg\\\\\\\/Ue4eMYcZcI3t+QJ+b4nLokyKxJ\\\\\\\/ma1liMj6J1y9hTRZ1lNu8TFEhN+CGRIEEO6ChwQqF5a\\\\\\\/u6I4as8Vx7zrNtbhlPrG5Rqyb\\\\\\\/NRJ4NyTDedTXLjcwU9wJHK5QmghleAaPC\\\\\\\/hKd5iNIpow6XlUhl96PMWE51jWWEVAIQZfqf+9E\\\\\\\/UBrEm9NV7Xm1v0piGP6filNTER+3V3658Zrqr8F29drN7yWjL\\\\\\\/Ksi7j6+TgUk6\\\\\\\/JNsx\\\\\\\/WLmX\\\\\\\/Gw6lTbsaMyMx5S3uGxRBJe4XkBi9p35zxvcMmDCGI112bIM4nM46K9igsJykOQxpGCK7nbG+gMnl0xg\\\\\\\/PF6h+izYxuh50xWLN3piyecL5ReCdTAlBWCxLndgdDMp3Z8kPbS0CByDitOJ12QDhK1u1JhDabylqHSEIzPiknjOb\\\\\\\/fOSm8SP9u4soQaw7NCda3iebNacbaIEWQPtLh8bovij62KSKqSh8dmMjnmu4b8aSoJYNQPYt9ta6+PqbBPvGoFK2K9ZpIrkfx6IZ75y+J0Fg8ARyr7MHGCYSwaZCyu3oPCaaHMZR+M4hmlf8SqDuYrLvSlqPKPVPn2NvxCTJdMf5OGGO53W\\\\\\\/ZZ6DKMGBXVKN2rzl9V6Qip37sLgkpqdacKfeaaniwE6UnHFctXOzP6tsR95uD0JLInOnVMDF2ndmodV9D3T3VosjFm3XY\\\\\\\/wtp3+iQ1xTJilKPWLfE4tuah0FmnoN+Vf\\\\\\\/j8ciLZdC4bs\\\\\\\/EHW7YF3AUDOlAWvu90DRc3gxVRzS8Pui\\\\\\\/pltHlEg49RrsFPkddrQuR\\\\\\\/bF3sGG4HCd4eyi\\\\\\\/IUGq247lm5pcgqLkdY\\\\\\\/hXwt7lbwIXZwmnJJqQaNuoXfMv8im4MwpUYkk+0k\\\\\\\/PcNSuRnPhXVfN9uPja3G7z8B6uXmAtxfGLxCjfHVfQfTOgEUM\\\\\\\/N3ZPW\\\\\\\/YAQZqbufGefoMn8JAXlQObKTas6rUVL0QSpp0Vt\\\\\\\/MmkNyiklyxp5YUu1tW6iPw50+X0lkw0C8BxDD1oaVCqvvgPKLTGU8iGpmYSJJmQbaHwbouicb0f\\\\\\\/motWVvYhROBwPXfxxANqAewhd93YQ0NCZyWvRCOwAU+NYt9X\\\\\\\/SYtFg4oq2MUh5\\\\\\\/etzLJssFq4tHcFhEZjeYKRxbFsegAIu93jGWa25\\\\\\\/gmmtHxB1CB9ZDJjDTtJTiJ5b3wGf6AKaJCeu\\\\\\\/R8gFZz6jZMEwfWoYvH6s85tkKXHgsYPQeseUKn1eI5F\\\\\\\/OKn5Rp0tAJB4JyjZ0xLlPxJIV9UNJZYfOsnm9bhbnHyC9ADh7aM71m8J+2aDMZNpt89bGmNvsfJk37eFnHpgu0fl8+fIb\\\\\\\/dvBzCt9lMUb+zx00BIY\\\\\\\/PMaRz4PZXt7nR\\\\\\\/JXkOw2RrLsKDxvQUWDs4atH5g42uRlRLEAQniliIHv+Xj1ojBcksC+bKpHzuX4vxusp1nUAk8BhLactDvQLfP+kCqxFehx9ZKMeUxDGVrPqB87LkHfxPtsJ3WzavjZjA5ZTGHi08Hh+YAR1VwMCc96ypeFtTwv4llCUMIr2CVutL1Df1ivWjUWh63KLzbVtVzkWbEO3eVqSnIpYcSsX6i\\\\\\\/Jgb1tQRVc6gJIEMnN0sN1c3EOpqCpEaDCzK\\\\\\\/6WtvXldDuJZzoACASg1DbuhgIPS733WNTuMK4bctdHwW+zuFjcgAMxt7hMb0hkmqpAGI7t4jAknxtEcIvM4HEtLzfvIe6Kq0AI69\\\\\\\/MR6omcaCzGFQPxcFXPCrYGFxA+Vv8eZoBTRQF\\\\\\\/5aS8kf2GECfa7u\\\\\\\/DQcKWSIgbLAwD6oaGVrDR+L2fNVxvZq59ODRA9\\\\\\\/YqyMN76ce1BrAu9NY2BwKettrnTNI8dcFIi\\\\\\\/TybqjHtWkufoUiYCfV1MpqddlQf22G6u9sFyfqzMoIPy9h8Cd1xEZny+vSwRPzjTPbVszKbCUknUIwEt862Q9XlK2djjmGdT6VrKJH6XFTrEMvIg4ryp4NUFQB5dkYxzc3WwpN4\\\\\\\/v1VVS7UvzXWS4TwBpYNFBshN+YXjOaQsIGf+\\\\\\\/fY3LZUFwV3bQj5ec4k8vOdNByiU\\\\\\\/NrdUqZT6n6cI07CpDvXvbWkUnH9hkmypnJWus0QDaFJ30u15lI\\\\\\\/1J6w6RUyJlUSjmkE+qKtEbz9gbzlKxpHTO\\\\\\\/o9iNJjGLGjrDKkOHiQwWeMxTIsMDj8gWCAn+AFCJ+0CkI2kWJJJeqZBMLDyj4z0PaiFknV2hcqbHnLgyfsQxNJ1oeKosFYWstxMJ0Y9GKx1ejThHlPsLzU2P5RRSP1c5iXHCV9uPjMD\\\\\\\/1ADEtUwPqlIyE1CmEoKcmVl0oFe6A7lwfM1jyA3E6oJ4Xp71t1fiDwc0V684SgA8iHmZFoZTlVpa3pu6LnpV68c5izqIN65CpEY1bcsabACUNcH0JYclXbT+Nl9Q2yUtms\\\\\\\/rKQK1TVirzQNxf2aktR+XYtaSB\\\\\\\/aIXYY5KT1oR0+Xm9AShXs6qsFlaQxNYSXGEp2uZFZyCRczIZ72\\\\\\\/cKZQPtiBYCDnoAPM1BSOfvgkSUof8eOhWVIwZvgKuXV7DQjsuQlmSsua00Q1DTdv87udGhwh7Cbl\\\\\\\/9yGJZ2H2lYPCfZRtFQrZPF87SyyUFgidZ6AjFoQs4Ag1cel\\\\\\\/PqdPxbgK3Ra1NK7YKrAmDdq7yfen2zRYek1lZ8ind9ocCB3IHMFFsPsE9Jm+Hl+5IGHw8HLEHc2YgTlqUqemarEqVPWtyfNjc1SgjITngO2DKNMwG1VRzbUMxF18Zp3dCih\\\\\\\/bpkCvCv78asv8gnjt2Frme4ZVHG\\\\\\\/ZvK0Gh9wkdu3GcMw2XuoxDu\\\\\\\/w3T3xb6rZTkAH\\\\\\\/PFq3tmo3wGPWL6QseXdqR\\\\\\\/3QfUGDNJnHE6qa5gjBlUlhRa2BYGuZ19Cpi0B33kHrQsAluXIUbTLxakN0dB9oKMMxde1pA1FGVrRfF62qxC0VjnhCDVSOIeeLtLS9T03p2wxbFvRxMaCghOIBa9B4ovnHE3x35PSnyEVg71XS1hWGabNXR+bHo4efsXW5oP95GAtwtvMzZA6G+hDY6JbTGxuV8QYH7FpWKZwVY5+P1LUNl2joqM7TterOXqgvXTg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/AMHyuxgfMJ3bkZ\\\\\\\/95AzLA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002200c147d0296c1e3403273f1ce74492c561d47c50ea6e491ffe8d45848ae1d4c1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226Q0PXZP4y9DSJH+eDhH6B86q5BxwhwbHMu9QwxFfaT5LWBqBnNA9VggyekTBYMKbjuBVfCT8OFa3V8iYAdmteo\\\\\\\/gr+79bkNHShFh2a9W7q46UChGWerKkNkdOKOdLtcSLA0V+4AALKl4mW5NbTMbUXu4VG9FGQHZMxhYwsiUbrUSDzlHOg42il\\\\\\\/j\\\\\\\/P4wQHhsEAIcyGRozAg2lU+eabCPguNhELF7L6Si8TEomGR56V0s6ZrHkhtuc5kSMciLqvmDq5ycSAq71BST88TEJzjun2owWwnMGzXY5gr8K7Lvz9icynCZxPDriND+UnzUEPpMHGYvAEWWrsRxOKxGqUYTKqCQA5yEVXFFFxi2ezobsCsnZPEJxcI8eOw51NQKGXDKJOSsRTVvsYePB\\\\\\\/M8IQQ3fgQ8CvOzGhrZ+HnR4Hq1SUmSb6JfCR4THqpsVwgeC\\\\\\\/Dt3\\\\\\\/lm+k8Uan38tUk2rPVJ94+z2pDuX3I672ns9WgaVej80kxffaGlUVi9JG3ReQ8XeZIJ2YfzWpG2jrvRojOiNetMefV20m6P4xqMupGPeENRvntbH5IkaQutbvJsdsCWOmJs3CbfiR9sxKx3SJwZtEDolhElTqZMpB6IvscJ5umz8SsBBbVxlftzDyE9bYMwb2AcCiYLuMq5mIXqrt+kuYwaOAqQhpHeD12qj98B6eJHz0lHtPiD3hkplzndTjJ+VfjuUkBHtfcLiRHRe1zyi0e0jRsf198\\\\\\\/VXMZycz4h\\\\\\\/qfb0+67nRlnC1YkCpWEj7S78Fbg89uHqzLPqEuygHFgQAaGge6OsayzflBNO4bFeQlKVJ6A4Tm79+bRItZhJpRLW6q9e2h\\\\\\\/ljqtVaIJhCpSSr3P8BLM0HnNjw95kJ4ucuPK4Vza0zHhwRFIlSKtDUrjvpFBTp0a7Wm\\\\\\\/rdcg1i9TnsOi9k79EwXXg9hh\\\\\\\/fxgHz5CvRK17IIwLlLSh9Rgkje5PmM\\\\\\\/1s0SCitbyJnv8DFN\\\\\\\/a3qcFum6ppEQECXrbhmLhtyyLPItCnxaqv2SnzgrG8hrwUeOoXs6NHZ8qIKX+NPAzmnM+3mrhauWy1mazD82yJifYQtH5eXijczd467sTHl67oWzG3kHFGU5zGA+EFVxEMOeEIHznZlwcIUzafILpLwS836NUMCLkn4gkdA6CnD3dPV+pvPyZ8+44K7RuUcH2tPTWLseLeI23snZT3\\\\\\\/wdW4CwS2i6OH921ftCEiNT6gOzZ4Drwd4KTKeApE3Ww778ktrqH2QN1oGuaDzrMv1Z4\\\\\\\/pIhpYbmDMxMpkXNAuI1KgHpjhB5AidqAi3\\\\\\\/Kt4cSdfG8Nu3ad+z+25faF56fQIqengHlmpTi5t7Lc78HdedmUQTcgUf1H32xGACOsXAVUDRRKvmH2ZIqWYRRtdikchOAW7eZS7LUgI\\\\\\\/SVVynOFLkTUnH\\\\\\\/AOsxnwar\\\\\\\/LR3zTgYK2gZc8D30WjucvOhzKNkFPa31mdawPwNHTTm72MrScaYG0V5bRGN\\\\\\\/coIa6rhqXgY+TdOXwF50c4asMPcbXzj+NFObDwY3ZlUSSmnrG8PfPsV94o\\\\\\\/7FLaU7jXXdPSaoQf0nYxTYCh0FRfvj3WKs69X8iGucy5lgJ77U417ECvkuwRn5BmbLg8SdNPcrBvfbeh\\\\\\\/KbaHT9RvrsByiNip7Atiq\\\\\\\/yNisnGBgLlqQ7KWGA6Eu\\\\\\\/3T3H7aohY0\\\\\\\/IQx4RSn0v2jgoSdog9sQJwai47ea2UmMRK5R54akK2Uq4fmK0QXd19pRJrwq1QmEgHu1WEkGZ\\\\\\\/m\\\\\\\/MIGZnwcj5v22mmcKAGt9FbrWZR+UwtI1vaQVEgxM21naFIemhddiu8G1lNynZdMom4oEncQ8OYfbeYYzd3nIK99urK63cqsXXAjCdL3l1Hzt8hbVD\\\\\\\/UienLSoqImf4SIkwSzIkQ1dRdlWwAdr8Iwm2QNoobKWff88\\\\\\\/EEvNlLuE2wnCiTQ4toeVviVzxgqazZOfTHAPLrFYL7Fy2sNgAaCq5Z+CUA2mNHiIG4uMddBOVv1tVm2oi1i88JaUhew==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XewdI\\\\\\\/Pxc5MJwZHeMltj\\\\\\\/w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a88b7fe54f51bcaead03c4aaa2077133bb985ea24d54ed1c871b4f7a277f260a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483841 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1365" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XMtfx_w6X_JUpFKAUSWH_L5goAxTFWCQI05v5XT6Ixg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-areto6R9r_wIIBHRdIjrkwL_aCjIQo-8r3XVpGPJ85M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1357" title="pnp4a Is the Causal Gene of the Medaka Iridophore Mutant guanineless" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1377" title="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483841" id="node483841" data-pisa="ggg;7/4/1365" data-pisa-master="ggg;g3.116.038497" data-apath="/ggg/7/4/1365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dense Brown Trout (<em>Salmo trutta</em>) Linkage Map Reveals Recent Chromosomal Rearrangements in the <em>Salmo</em> Genus and the Impact of Selection on Linked Neutral Diversity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5172-0970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maeva Leitwein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6934-1677" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bruno Guinand</span>, <span class="highwire-citation-author" data-delta="2">Juliette Pouzadoux</span>, <span class="highwire-citation-author" data-delta="3">Erick Desmarais</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9026-4987" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Berrebi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1908-3235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre-Alexandre Gagnaire</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1365-1376;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038497 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483841" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maeva Leitwein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maeva%2BLeitwein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leitwein%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaeva%2BLeitwein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5172-0970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maeva Leitwein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maeva.leitwein.pro@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maeva.leitwein.pro@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bruno Guinand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Département Biologie-Ecologie, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BGuinand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guinand%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BGuinand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6934-1677" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bruno Guinand</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juliette Pouzadoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliette%2BPouzadoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pouzadoux%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliette%2BPouzadoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Erick Desmarais </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erick%2BDesmarais%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desmarais%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErick%2BDesmarais%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick Berrebi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BBerrebi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berrebi%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BBerrebi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9026-4987" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Berrebi</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Pierre-Alexandre Gagnaire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre-Alexandre%2BGagnaire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gagnaire%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre-Alexandre%2BGagnaire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1908-3235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre-Alexandre Gagnaire</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/4/1365" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483841" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1365.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483841" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1365.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483841" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1365.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483841" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/4/1365.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483841" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-density linkage maps are valuable tools for conservation and eco-evolutionary issues. In salmonids, a complex rediploidization process consecutive to an ancient whole genome duplication event makes linkage maps of prime importance for investigating the evolutionary history of chromosome rearrangements. Here, we developed a high-density consensus linkage map for the brown trout (<em>Salmo trutta</em>), a socioeconomically important species heavily impacted by human activities. A total of 3977 ddRAD markers were mapped and ordered in 40 linkage groups using sex- and lineage-averaged recombination distances obtained from two family crosses. Performing map comparison between <em>S. trutta</em> and its sister species, <em>S. salar</em>, revealed extensive chromosomal rearrangements. Strikingly, all of the fusion and fission events that occurred after the <em>S. salar</em>/<em>S. trutta</em> speciation happened in the Atlantic salmon branch, whereas the brown trout remained closer to the ancestral chromosome structure. Using the strongly conserved synteny within chromosome arms, we aligned the brown trout linkage map to the Atlantic salmon genome sequence to estimate the local recombination rate in <em>S. trutta</em> at 3721 loci. A significant positive correlation between recombination rate and within-population nucleotide diversity (π) was found, indicating that selection constrains variation at linked neutral sites in brown trout. This new high-density linkage map provides a useful genomic resource for future aquaculture, conservation, and eco-evolutionary studies in brown trout.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-mapping" class="hw-term hw-article-keyword hw-article-keyword-linkage-mapping" rel="nofollow">linkage mapping</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/salmo-trutta" class="hw-term hw-article-keyword hw-article-keyword-salmo-trutta" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Salmo trutta</em></a></li><li class="kwd"><a href="/keyword/salmonids" class="hw-term hw-article-keyword hw-article-keyword-salmonids" rel="nofollow">salmonids</a></li><li class="kwd"><a href="/keyword/rad-markers" class="hw-term hw-article-keyword hw-article-keyword-rad-markers" rel="nofollow">RAD markers</a></li><li class="kwd"><a href="/keyword/chromosomal-rearrangements" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-rearrangements" rel="nofollow">chromosomal rearrangements</a></li><li class="kwd"><a href="/keyword/recombination-rate" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate" rel="nofollow">recombination rate</a></li></ul><p id="p-3">A renewed interest for linkage mapping studies has recently occurred thanks to the simplified procedures for generating large genotype datasets in nonmodel organisms (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Slate <em>et al.</em> 2008</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Stapley <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ekblom and Galindo 2011</a>). High-density linkage maps have been particularly developed in teleost fishes because of their high number of species, economic importance for aquaculture (<em>e.g.</em>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Sauvage <em>et al.</em> 2012</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Zhao <em>et al.</em> 2013</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Liu <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kumar and Kocour 2017</a>), phylogenetic position within vertebrates (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Rondeau <em>et al.</em> 2014</a>; <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kuang <em>et al.</em> 2016</a>), and broad interests for eco-evolutionary and conservation/management issues (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Everett <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gagnaire <em>et al.</em> 2013a</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kai <em>et al.</em> 2014</a>).</p><p id="p-4">The salmonid family, which has 11 genera and &gt;60 species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Crête-Lafrenière <em>et al.</em> 2012</a>), is a group of fish in which numerous initiatives have been developed to construct high-density linkage maps (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Limborg <em>et al.</em> 2015b</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Larson <em>et al.</em> 2016</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Tsai <em>et al.</em> 2016</a>). Besides their applications for mapping traits of importance for aquaculture (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Houston <em>et al.</em> 2012</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Sauvage <em>et al.</em> 2012</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Tsai <em>et al.</em> 2015</a>), salmonid linkage maps have catalyzed molecular ecology and genome evolution studies over recent years. Several studies have used linkage map information for addressing eco-evolutionary and related conservation biology issues at the species level or among hybridizing taxa (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lamaze <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bourret <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Perrier <em>et al.</em> 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gagnaire <em>et al.</em> 2013b</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Limborg <em>et al.</em> 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">McKinney <em>et al.</em> 2015</a>). A second category of studies have focused on understanding the evolutionary consequences of the whole genome duplication event (salmonid-specific fourth vertebrate whole genome duplication; WGD-Ss4R) that occurred within the salmonid family ∼60 MYA (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Crête-Lafrenière <em>et al.</em> 2012</a>), especially with regards to the partial rediploidization process, which is still underway and not totally understood yet (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Nugent <em>et al.</em> 2017</a>). A significantly improved understanding of this process has been achieved through increased efforts to map and compare chromosomal rearrangements among species (inversions, translocations, fissions, and fusions), which have helped to reconstruct the evolutionary history of salmonid genome architecture (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>). Furthermore, linkage mapping approaches have also improved the characterization of the genomic regions that still exhibit tetrasomic inheritance patterns (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">May and Delany 2015</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Waples <em>et al.</em> 2016</a>). The partial rediploidization of salmonid genomes is also known to challenge the development of genome-wide marker data sets. Indeed, duplicated loci that mostly reside in tetrasomic regions (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>) are usually underrepresented or even excluded in salmonid genomic studies (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hohenlohe <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hecht <em>et al.</em> 2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gagnaire <em>et al.</em> 2013b</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Larson <em>et al.</em> 2014</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Perrier <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> 2016</a>)</p><p id="p-5">The Eurasian brown trout (<em>Salmo trutta</em> L. 1758), is one of the most widespread freshwater species in the Northern Hemisphere. It presents high levels of phenotypic diversity linked to its complex evolutionary history (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Elliott 1994</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernatchez and Osinov 1995</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernatchez 2001</a>). Natural brown trout populations have been intensely studied using molecular markers (<em>e.g.</em>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hansen <em>et al.</em> 2000</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sanz <em>et al.</em> 2006</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Thaulow <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Berrebi 2015</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> 2016</a>), but previous population genetics and phylogeography studies that usually relied on a limited number of markers did not integrate linkage map information (but see <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hansen and Mensberg 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hansen <em>et al.</em> 2010</a>). The microsatellite linkage map available until now in brown trout (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> 2006</a>) contains ∼300 markers (288 microsatellites and 13 allozymes), distributed over 37 of the 40 linkage groups (LGs) expected in this species (2<em>n</em> = 80, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Martinez <em>et al.</em> 1991</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Phillips and Ráb 2001</a>). In order to follow the tendency toward an increasing number of markers in brown trout genetic studies (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> 2016</a>), a higher density linkage map is needed to increase mapping resolution and provide positional information for genome scans and association studies. The development of a high-density linkage map in brown trout should find at least two direct applications.</p><p id="p-6">First, although the brown trout is the closest relative of the Atlantic salmon (<em>S. salar</em> L. 1758), the two species drastically differ in their number of chromosomes (<em>S. salar</em> 2<em>n</em> = 54–58, Brenna-Hansen <em>et al.</em> 2012; <em>S. trutta</em> 2<em>n</em> = 80, <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Phillips and Ráb 2001</a>), indicative of a high frequency of chromosome rearrangements. For understanding the chromosomal evolution in the <em>Salmo</em> genus, a comparison between the genome architecture of the Atlantic salmon (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>) and the brown trout, using other salmonid linkage maps as outgroups, is needed. The availability of important genomic resources in salmonids coupled with recent methodological advances in linkage map comparisons (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>) provide favorable conditions for new insights into the recent history of chromosomal rearrangement in the <em>Salmo</em> genus.</p><p id="p-7">Second, another important application of a high-density linkage map in brown trout relates to understanding the consequences of human-mediated introductions of foreign and domestic genotypes in natural populations. The brown trout has been domesticated for decades, and wild populations have been heavily impacted by the stocking of genetically domesticated hatchery strains sometimes originating from evolutionary lineages distinct from the recipient populations (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Elliott 1994</a>; Laikre 1999; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Berrebi <em>et al.</em> 2000</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bohling <em>et al.</em> 2016</a>). These introductions have often resulted in the introgression of foreign alleles within wild brown trout populations, a generally negatively perceived phenomenon in conservation and management (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Largiader and Scholl 1996</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Poteaux <em>et al.</em> 1998</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">1999</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hansen <em>et al.</em> 2000</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Almodóvar <em>et al.</em> 2006</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Sanz <em>et al.</em> 2006</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hansen and Mensberg 2009</a>). However, the real evolutionary consequences of hybridization may take multiple facets (<em>e.g.</em>, outbreeding depression, heterosis, adaptive introgression, loss of adaptive variation), the individual effects of which remains largely unknown. In order to evaluate those potential fitness outcomes in brown trout, it would be necessary to combine the power of large SNP data sets (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> 2016</a>) with a high-density linkage map and a detailed knowledge of chromosomal variation in recombination rate to perform comprehensive genome scans for introgression in natural populations.</p><p id="p-8">The specific objectives of this study were: (i) to build a high-density linkage map in brown trout based on restriction site–associated DNA (RAD) markers using reciprocal crosses of individuals belonging to two distinct evolutionary lineages (Atlantic and Mediterranean; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bernatchez 2001</a>); (ii) to characterize chromosome morphologies and centromere positions of metacentric chromosomes using the method recently developed in <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Limborg <em>et al.</em> (2015a)</a>; (iii) to perform map comparisons with the <em>S. salar</em> genome and an outgroup salmonid species to infer the timing of chromosome rearrangements during the recent evolutionary history of the <em>Salmo</em> genus; and (iv) to estimate the genome-wide local variation in recombination rate and assess the consequence of this variation on the genomic landscape of nucleotide diversity in brown trout.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Experimental crosses</h3><p id="p-9">Two F1 hybrid families, each comprising two parents and 150 offspring, were used for the linkage map analysis. Both were obtained by crossing one parent of Atlantic origin with one parent of Mediterranean origin (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bernatchez 2001</a>). The F0 progenitors were either from a domestic Mediterranean strain lineage originally bred in the Babeau hatchery [described in <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bohling <em>et al.</em> (2016)</a>] and a domestic Atlantic strain lineage originating from the Cauterets hatchery and independently reared from the Mediterranean strain at the Babeau hatchery. The Cauterets strain is one of the common, internationally distributed Atlantic strains (comATL category; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Bohling <em>et al.</em> 2016</a>). Crosses were performed in December 2015 at the Babeau hatchery as follows: FAMILY 1, one Atlantic female × one Mediterranean male; and FAMILY 2, one Atlantic male × one Mediterranean female. Progenies were killed in February 2016, after full resorption of the yolk sac (fish size ∼2 cm).</p></div><div id="sec-3" class="subsection"><h3>DNA extraction, library preparation, and sequencing</h3><p id="p-10">Whole genomic DNA was extracted from caudal fin clips for the four parents of the two crosses and from tail clips of 150 offspring of each family, using the commercial Thermo Scientific KingFisher Flex Cell and Tissue DNA kit. Individual DNA concentration was quantified using both a NanoDrop ND-8000 spectrophotometer (Thermo Fisher Scientific) and Qubit 1.0 Fluorometer (Invitrogen, Thermo Fisher Scientific). Double digest RAD sequencing library preparation was performed following <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> (2016)</a>. Briefly, DNA was digested with two restriction enzymes (<em>Eco</em>RI-HF and <em>Msp</em>I) and the digested genomic DNA was then ligated to adapters and barcodes. Ligated samples with unique barcodes were pooled in equimolar proportions and CleanPCR beads (GC Biotech, Alphen aan den Rijn, The Netherlands) were used to select fragments sized between 200 and 700 bp. The size selected fragments were amplified by PCR and tagged with a unique index specifically designed for Illumina sequencing (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Peterson <em>et al.</em> 2012</a>). For progenies, PCR products were pooled in equimolar ratios into four pools. Each pool was sequenced in a single Illumina HiSequation 2500 (<em>i.e.</em>, 75 F1 progenies per lane), producing 125-bp paired-end reads. The four parents were sequenced together in a single lane in order to obtain a higher coverage depth.</p></div><div id="sec-4" class="subsection"><h3>Genotyping pipeline</h3><p id="p-11">Bioinformatic analyses followed the same procedure as in <a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> (2016)</a>. Briefly, raw reads that passed an Illumina purity filter and were of sufficient FastQC quality (&gt; 28) were demultiplexed, cleaned, and trimmed with process_radtags.pl implemented in Stacks v1.35 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Catchen <em>et al.</em> 2013</a>). The Atlantic salmon genome (GenBank accession number: GCA_000233375.4_ICSASG_v2; <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>) was used to determine individual genotypes at RAD markers using a reference mapping approach. First, reads were aligned to the <em>S. salar</em> genome with the BWA-mem program (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Li and Durbin 2010</a>). Alignment result files were processed into pstacks to build loci and call haplotypes for each individual (using <em>m</em> = 3 and <em>α</em> = 0.05). Individuals with a minimum mean coverage depth under 7× were removed, resulting in a data set of 147 offspring in <span class="sc">family</span> 1 and 149 offspring in <span class="sc">family</span> 2. Hereafter, a RAD locus is defined as a sequence of 120 bp, and thus can contain more than one SNP defining different haplotypes. A reference catalog of RAD loci was built with cstacks from the four parents of the two crosses (<span class="sc">family</span> 1 and <span class="sc">family</span> 2). Each individual offspring and parent were then matched against the catalog with sstacks. Then, for each family independently, the genotypes module was executed to identify mappable makers that were genotyped in at least 130 offspring in each family, and with a minimum sequencing depth of five reads per allele. The correction option (-c) in the genotypes module was used for automated correction of false-negative heterozygote calls. The map type “cross CP” for heterogeneously heterozygous populations was selected to export haplotypic genotypes in the JoinMap 4.0 format (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Van Ooijen 2006</a>). For importation into JoinMap, only the loci shared between <span class="sc">family</span> 1 and <span class="sc">family</span> 2 were kept, leaving a total of 7680 mappable loci in both families. </p></div><div id="sec-5" class="subsection"><h3>Linkage mapping</h3><p id="p-12">Both family data sets were filtered into JoinMap 4.0 (<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Van Ooijen 2006</a>) to remove (i) loci presenting with significant segregation distortion, (ii) loci in perfect linkage (<em>i.e.</em>, similarity of loci = 1), and (iii) individuals with high rates of missing genotypes (&gt;25%). After this first filtering step, 5635 and 5031 loci remained in <span class="sc">family</span> 1 and <span class="sc">family</span> 2, respectively. Only those that were shared by the two families were kept, resulting in a subset of 4270 filtered informative loci. A random subset of 4000 loci common to both families were finally selected, which corresponds to the maximum number of loci handled by JoinMap 4.0. We then discarded four offspring in <span class="sc">family</span> 1 and six offspring in <span class="sc">family</span> 2 due to high rates of missing genotypes (&gt;20%). Markers were then grouped independently within each family using the “independence LOD” parameter in JoinMap, with a minimum LOD value of 10. Unassigned markers were second assigned using the strongest cross-linkage option with a LOD threshold of 5. Marker ordering was finally estimated within each group using the regression mapping algorithm and three rounds of ordering. The Kosambi mapping function was used to estimate genetic distances between markers in centimorgan. Finally, loci with undetermined linkage were removed and a consensus map was produced between families using the map integration function after identifying homologous LGs between <span class="sc">family</span> 1 and <span class="sc">family</span> 2.</p></div><div id="sec-6" class="subsection"><h3>Estimation of centromere location and chromosome type</h3><p id="p-13">The method recently developed by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Limborg <em>et al.</em> (2015a)</a> was used to identify chromosome types (acrocentric, metacentric) and to determine the approximate centromere position of metacentric chromosomes. This method uses phased progeny genotypes to detect individual recombination events in comparison with parental haplotype combinations. For each LG, the cumulated number of recombination events between the first marker and increasingly distant markers was computed from both extremities, using each terminal marker as a reference starting point. When covering a full chromosome arm, the recombination frequency (RFm) values are expected to reach a plateau of ∼0.5, thus allowing the determination of chromosome morphology and the approximate centromere location for metacentric chromosomes [see <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Limborg <em>et al.</em> (2015a)</a> for details]. We used the subset of markers that were found heterozygous only in the female parent to phase progeny haplotypes within each family. We only focused on female informative sites because the lower recombination rate in male salmonids makes them less adequate for inferring centromere locations (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Limborg <em>et al.</em> 2015a</a>). Moreover, lower male recombination rates have been reported in brown trout (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> 2006</a>). The phased genotypes data sets were used to estimate RFm for intervals between markers along each LG. Finally, we plotted the RFm calculated from each LG extremity to determine the chromosome type and the approximate location of the centromere for metacentric chromosomes.</p></div><div id="sec-7" class="subsection"><h3>Map comparisons</h3><p id="p-14">The microsatellite linkage map established by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> (2006)</a> was compared and anchored to our new high-density linkage map using the MAPCOMP pipeline (available at <a href="https://github.com/enormandeau/mapcomp/">https://github.com/enormandeau/mapcomp/</a>) (<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>). The microsatellite markers’ primer sequences retrieved from <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> (2006)</a>, as well as the five microsatellites used in <a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> (2016)</a> and the lactate dehydrogenase (<em>LDH-C1</em>) gene from <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">McMeel <em>et al.</em> (2001)</a> were aligned with BWA_aln (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Li and Durbin 2010</a>) to the Atlantic salmon reference genome. Only single hit markers with a quality score (MAPQ) above 10 were retained. The 3977 RAD markers integrated in the high-density <em>S. trutta</em> linkage map (see <em>Results</em>) were also aligned to the <em>S. salar</em> reference genome with BWA_mem. Microsatellite and LDH markers were anchored onto the RAD linkage map by identifying the closest RAD locus located on the same contig or scaffold. Pairing with closely linked RAD markers enabled us to position markers from the previous generation map onto the newly developed linkage map.</p><p id="p-15">An additional comparison was performed between the <em>S. salar</em> high-density linkage map published by <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Tsai <em>et al.</em> (2016)</a> and the <em>S. trutta</em> consensus linkage map developed in this study. We ran the MAPCOMP pipeline (<a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>) to identify blocks of conserved synteny and orthologous chromosome arms between the Atlantic salmon and the brown trout maps, using the Atlantic salmon genome as a reference. Conserved synteny blocks were visualized with the web-based VGSC (Vector Graph toolkit of genome Synteny and Collinearity: <a href="http://bio.njfu.edu.cn/vgsc-web/">http://bio.njfu.edu.cn/vgsc-web/</a>). The brook charr (<em>Salvelinus fontinalis</em>) linkage map published in <a id="xref-ref-78-6" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> (2016)</a> was used as an outgroup to identify chromosome rearrangements that occurred within the <em>Salmo</em> lineage.</p><p id="p-16">In order to estimate the genome-wide averaged net nucleotide divergence between <em>S. trutta</em> and <em>S. salar</em> (<em>d<sub>a</sub></em>), we used the mean nucleotide diversity within <em>S. trutta</em> (<em>π<sub>w</sub></em>, averaged between Atlantic and Mediterranean lineages) (<a id="xref-ref-46-8" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> 2016</a>), and the absolute nucleotide divergence between <em>S. trutta</em> and <em>S. salar</em> (<em>π<sub>b</sub></em>, also called <em>d<sub>xy</sub></em>). The net divergence was estimated as <em>d<sub>a</sub></em> = <em>π<sub>b</sub></em> − <em>π<sub>w</sub></em>, by supposing that the mean nucleotide diversity in <em>S. salar</em> (which is not available) was close to the mean nucleotide diversity estimated in <em>S. trutta</em>. To estimate <em>π<sub>b</sub></em><sub>,</sub> we used the 3977 consensus sequences of the RAD loci integrated in our <em>S. trutta</em> linkage map, and aligned them with BWA_mem to the Atlantic salmon reference genome. Then, the total number of mismatches divided by the total number of mapped nucleotides was used to provide an estimate of <em>π<sub>b</sub></em>.</p></div><div id="sec-8" class="subsection"><h3>Estimation of genome-wide recombination rates</h3><p id="p-17">To estimate local variation in recombination rate across the genome, we compared our newly constructed genetic map of <em>S. trutta</em> to the physical map of <em>S. salar</em> with M<span class="sc">arey</span>M<span class="sc">ap</span> (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rezvoy <em>et al.</em> 2007</a>). In order to reconstruct a collinear reference genome for each brown trout LG, physical positions of brown trout RAD markers on the salmon genome were extracted from the BWA_mem alignment for each block of conserved synteny previously identified with MAPCOMP. The Loess method was used to estimate local recombination rates. It performs a local polynomial regression to fit the relationship between the physical and the genetic distances. The window size, defined as the percentage of the total number of markers to take into account for fitting the local polynomial curve, was set to 0.9 to account for local uncertainties due to the high rate of chromosomal rearrangements between species. A recombination rate equal to the weighted mean recombination rate of the two closest markers on the genetic map was assigned to the markers that were not used for fitting.</p></div><div id="sec-9" class="subsection"><h3>Correlation between recombination rate and nucleotide diversity</h3><p id="p-18">In order to test for a correlation between local recombination rate and nucleotide diversity in brown trout, we used the estimate of nucleotide diversity (<em>π</em>) in the Atlantic lineage (based on 20 individuals). These estimates of nucleotide diversity were taken from <a id="xref-ref-46-9" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> (2016)</a>, in which brown trout populations and hatchery strains of different origins showed highly correlated genome-wide variation patterns in nucleotide diversity. Markers with a recombination rate value &gt;2 cM/Mb or <em>π</em> &lt; 0.008 were considered as outliers and were therefore removed from the analysis. This resulted in a data set containing 3038 markers for which both recombination rate and nucleotide diversity information were available.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-19">Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS1.xlsx">Table S1</a> contains RAD markers IDs, consensus sequences, mapping position on the brown trout linkage map, physical position on the Atlantic salmon genome, and estimation of local recombination rate. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS2.docx">Table S2</a> contains correspondence between the microsatellite and the new RAD linkage map. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS1.pdf">Figure S1</a> shows RFm plots obtained for each female haploid data set. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS2.pdf">Figure S2</a> shows MapComp Oxford plot comparing the brown trout to the Atlantic salmon linkage map, with markers paired through the Atlantic salmon genome. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS3.pdf">Figure S3</a> shows distribution of female and male marker positions along each LG for FAMILY 1. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FileS1.zip">File S1</a> is an archive containing the .loc and .map files from both families. Demultiplexed sequence reads aligned against the Atlantic salmon genome (in Bam file format) are available at NCBI Short Read Archive under accession number PRJNA371687.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Construction of a high-density linkage map in S. trutta</h3><p id="p-20">On average, 35M of filtered paired-end reads were obtained for each of the four parents (two families), and 5M paired-end reads for each of the 300 F1 (150 progenies per family) analyzed in this study. The retained subset of 7680 informative markers common to both families had a mean individual sequencing depth of 112× for the parents and 15× for the progenies, ensuring a good genotype calling quality in both families. Among these, a subset of 4000 markers showing no segregation distortion in both families was retained to generate a consensus map. The resulting integrated map thus provides sex- and lineage-averaged genetic distances, which is an important feature considering the reported differences in recombination rates between male and female in salmonids (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Tsai <em>et al.</em> 2015</a>), and the existence of divergent geographic lineages in brown trout (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bernatchez 2001</a>).</p><p id="p-21">A total of 3977 markers were assigned to 40 <em>S. trutta</em> (BT) LGs on the integrated map (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), corresponding to the expected number of chromosomes in this species (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Phillips and Ráb 2001</a>). The total map length was 1403 cM, and individual LGs ranged from 14.6 cM (BT-03; 47 markers) to 84.9 cM (BT-37; 63 markers). The number of markers per LG ranged from 23 for BT-40 (15.2 cM) to 252 for BT-01 (46.3 cM). Detailed information for each LG including the density of markers per centimorgan is reported in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Additional details concerning RAD markers IDs, consensus sequences, and mapping position on the salmon genome are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS1.xlsx">Table S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The brown trout (Salmo trutta) high-density linkage map. Horizontal black lines within each of the 40 LGs represent mapped RAD markers (n = 3977). The most probable chromosome type determined with the RFm method (Limborg et al. 2015a) is indicated above each LG. Green boxes indicate the approximate position of the centromere for metacentric LGs. Red dots represent the position of 89 additional markers that were transferred from the previous generation linkage map (see Table S2 for details). A, acrocentric; M, metacentric; U, undetermined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439154775" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The brown trout (&lt;em&gt;Salmo trutta&lt;/em&gt;) high-density linkage map. Horizontal black lines within each of the 40 LGs represent mapped RAD markers (&lt;em&gt;n&lt;/em&gt; = 3977). The most probable chromosome type determined with the RFm method (Limborg &lt;em&gt;et al.&lt;/em&gt; 2015a) is indicated above each LG. Green boxes indicate the approximate position of the centromere for metacentric LGs. Red dots represent the position of 89 additional markers that were transferred from the previous generation linkage map (see Table S2 for details). A, acrocentric; M, metacentric; U, undetermined.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1365/F1.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/4/1365/F1.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484002" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">The brown trout (<em>Salmo trutta</em>) high-density linkage map. Horizontal black lines within each of the 40 LGs represent mapped RAD markers (<em>n</em> = 3977). The most probable chromosome type determined with the RFm method (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Limborg <em>et al.</em> 2015a</a>) is indicated above each LG. Green boxes indicate the approximate position of the centromere for metacentric LGs. Red dots represent the position of 89 additional markers that were transferred from the previous generation linkage map (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS2.docx">Table S2</a> for details). A, acrocentric; M, metacentric; U, undetermined.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484018/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484018/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the main features of the brown trout (<em>Salmo trutta</em>) high-density linkage map based on the 3977 mapped markers distributed over the 40 LGs described in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Chromosome morphology and centromere location</h3><p id="p-25">Individual LG RFm plots obtained using estimates of recombination frequencies in progeny were generally similar between <span class="sc">family</span> 1 and <span class="sc">family</span> 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS1.pdf">Figure S1</a>). Two examples of RFm plots obtained in <span class="sc">family</span> 1 are reported in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> for BT-22 and BT-28. Following <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Limborg <em>et al.</em> (2015a)</a>, the pattern observed for BT-22 typically reflects an acrocentric chromosome type (paired straight lines; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), whereas BT-28 illustrates a metacentric pattern (mirrored hockey stick shapes; <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) with the centromere located at ∼70 cM. The total number of probable acrocentric chromosomes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; type A, <em>n</em> = 32) largely exceeded the number of metacentric chromosomes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>; type M, <em>n</em> = 3). The approximate position of the centromere was determined for each of the three metacentric LGs (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>; green boxes). The chromosome type remained undetermined for five LGs (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>; type U) due to low resolution of the RFm plots (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots illustrating the recombination frequency estimates (RFm) for intervals between markers along two different LGs. For each LG, RFm was calculated from both chromosomal extremities (right: red circles; left: blue circles), using each of the two terminal markers as a reference starting point. The RFm plot of (A) LG BT-22 illustrates a classical acrocentric pattern, the centromere position remains undetermined with regard to which LG extremity, while (B) LG BT-28 displays a classical metacentric pattern with a centromere position &#x223C;70 cM. The RFm plots of all brown trout LGs obtained in the two families are illustrated in Figure S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439154775" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plots illustrating the recombination frequency estimates (RFm) for intervals between markers along two different LGs. For each LG, RFm was calculated from both chromosomal extremities (right: red circles; left: blue circles), using each of the two terminal markers as a reference starting point. The RFm plot of (A) LG BT-22 illustrates a classical acrocentric pattern, the centromere position remains undetermined with regard to which LG extremity, while (B) LG BT-28 displays a classical metacentric pattern with a centromere position &#x223C;70 cM. The RFm plots of all brown trout LGs obtained in the two families are illustrated in Figure S1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1365/F2.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/4/1365/F2.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483976" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Plots illustrating the recombination frequency estimates (RFm) for intervals between markers along two different LGs. For each LG, RFm was calculated from both chromosomal extremities (right: red circles; left: blue circles), using each of the two terminal markers as a reference starting point. The RFm plot of (A) LG BT-22 illustrates a classical acrocentric pattern, the centromere position remains undetermined with regard to which LG extremity, while (B) LG BT-28 displays a classical metacentric pattern with a centromere position ∼70 cM. The RFm plots of all brown trout LGs obtained in the two families are illustrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS1.pdf">Figure S1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Linkage map comparisons</h3><p id="p-27">A total of 87 microsatellite markers [82 out of 160 from <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> (2006)</a>, five out of five from <a id="xref-ref-46-10" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> (2016)</a>] and the LDH locus (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">McMeel <em>et al.</em> 2001</a>) were successfully mapped to the Atlantic salmon genome, and anchored to the new brown trout linkage map using their pairing with RAD markers, following the method developed in <a id="xref-ref-78-9" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> (2016)</a> (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>; red dots). The correspondence between the microsatellite linkage map and the new RAD linkage map is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS2.docx">Table S2</a>.</p><p id="p-28">The identification of orthologous chromosomes arms between S. <em>trutta</em> (40 LGs) and <em>S. salar</em> (29 LGs) revealed numerous differences, including fusions and fissions between the two species (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We found 15 one-to-one ortholog pairs (<em>i.e.</em>, one brown trout LG corresponds to a single Atlantic salmon LG, as for example <em>Ssa</em>6 and BT-5, or <em>Ssa</em>28 and BT-14; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), 12 cases where Atlantic salmon chromosomes correspond to either two or three brown trout LGs (<em>e.g.</em>, <em>Ssa</em>1, <em>Ssa</em>2; <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), and only two cases where brown trout LGs correspond to two Atlantic salmon chromosomes (BT-1 and BT-15; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Between-species intrachromosomal inversions were also observed, as for example an inversion flanked by noninverted regions between <em>S. salar Ssa</em>6 and <em>S. trutta</em> BT-5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS2.pdf">Figure S2</a>). Apart from these inter- and intrachromosomal rearrangements, a strongly conserved synteny was observed between the brown trout and the Atlantic salmon genomes. We estimated the average net nucleotide divergence between <em>S. trutta</em> and <em>S. salar</em> to <em>d<sub>a</sub></em> = 0.0187 (with <em>π<sub>b</sub></em> = 0.0228, and <em>π<sub>w</sub></em> = 0.0041).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dual synteny plot showing conserved syntenic blocks and chromosome rearrangements between Salmo salar (Ssa) and Salmo trutta (BT) LGs. Chromosomes arms (p, qa and qb) are specified for S. salar LGs, and the seven pairs of chromosome arms that still exhibit residual tetrasomy in salmon appear in red (Lien et al. 2016)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439154775" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dual synteny plot showing conserved syntenic blocks and chromosome rearrangements between &lt;em&gt;Salmo salar&lt;/em&gt; (&lt;em&gt;Ssa&lt;/em&gt;) and &lt;em&gt;Salmo trutta&lt;/em&gt; (BT) LGs. Chromosomes arms (p, q&lt;sub&gt;a&lt;/sub&gt; and q&lt;sub&gt;b&lt;/sub&gt;) are specified for &lt;em&gt;S. salar&lt;/em&gt; LGs, and the seven pairs of chromosome arms that still exhibit residual tetrasomy in salmon appear in red (Lien &lt;em&gt;et al.&lt;/em&gt; 2016).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1365/F3.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/4/1365/F3.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483860" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Dual synteny plot showing conserved syntenic blocks and chromosome rearrangements between <em>Salmo salar</em> (<em>Ssa</em>) and <em>Salmo trutta</em> (BT) LGs. Chromosomes arms (p, q<sub>a</sub> and q<sub>b</sub>) are specified for <em>S. salar</em> LGs, and the seven pairs of chromosome arms that still exhibit residual tetrasomy in salmon appear in red (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The identification of orthologous genomic regions between the brook charr, the brown trout, and the Atlantic salmon (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) allowed inference of ancestral and derived chromosomes structures in the <em>Salmo</em> genus. We found eight chromosomal rearrangements that occurred in the ancestral lineage of the <em>Salmo</em> genus before speciation between <em>S. trutta</em> and <em>S. salar</em>. This includes five fusion events and three fission events (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). While no species–specific rearrangement was detected in the <em>S. trutta</em> branch, the Atlantic salmon branch contained two fission events and 13 fusion events (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Among these fusions, the one between BT-17 and BT-39 corresponds to <em>Ssa</em>15, which is known to contain the salmonid sex-determining gene <em>sdY</em> in the Atlantic salmon (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Li <em>et al.</em> 2011</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Yano <em>et al.</em> 2013</a>). Since the microsatellite locus <em>OmyRT5TUF</em> was consistently shown to be linked to <em>sdY</em> in the brown trout (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> 2006</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Li <em>et al.</em> 2011</a>), our results indicate that LG BT-39 might contain the sex-determining gene in brown trout (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS2.docx">Table S2</a>), which will need to be investigated further.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of fission and fusion events within the Salmo genus, that were oriented using the brook charr (Salvelinus fontinalis) as an outgroup species. Rearrangements are classified with regards to their occurrence (a) before the S. salar/S. trutta speciation, (b) in the S. salar branch after speciation, or (c) in the S. trutta branch after speciation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439154775" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of fission and fusion events within the &lt;em&gt;Salmo&lt;/em&gt; genus, that were oriented using the brook charr (&lt;em&gt;Salvelinus fontinalis&lt;/em&gt;) as an outgroup species. Rearrangements are classified with regards to their occurrence (a) before the &lt;em&gt;S. salar&lt;/em&gt;/&lt;em&gt;S. trutta&lt;/em&gt; speciation, (b) in the &lt;em&gt;S. salar&lt;/em&gt; branch after speciation, or (c) in the &lt;em&gt;S. trutta&lt;/em&gt; branch after speciation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1365/F4.medium.gif" width="440" height="138"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/4/1365/F4.medium.gif" width="440" height="138"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484046" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Summary of fission and fusion events within the <em>Salmo</em> genus, that were oriented using the brook charr (<em>Salvelinus fontinalis</em>) as an outgroup species. Rearrangements are classified with regards to their occurrence (a) before the <em>S. salar</em>/<em>S. trutta</em> speciation, (b) in the <em>S. salar</em> branch after speciation, or (c) in the <em>S. trutta</em> branch after speciation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Estimations of genome-wide and local recombination rates</h3><p id="p-32">The brown trout genome-wide recombination rate averaged across the 3721 markers anchored on the Atlantic salmon reference genome (RAD markers mapped to ungrouped scaffolds were discarded) was estimated at 0.88 ± 0.55 cM/Mb, and ranged from 0.21 cM/Mb (BT-27) to 4.10 (BT-37) across LGs (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS1.xlsx">Table S1</a>). Chromosomal variation in local recombination rate was observed for most chromosomes (<em>e.g.</em>, BT-9 and BT-23, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). A significant positive correlation between local recombination rate and population nucleotide diversity was detected (<em>R</em><sup>2</sup> = 0.058; <em>p</em>-value &lt; 0.001; <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of local recombination rate (&#x3C1;) in centimorgan per megabase for markers located along two Salmo trutta linkage groups: (A) LG BT-9 and (B) LG BT-23. The average recombination rate (mean &#x3C1;) for each LG is given at the top left side of each panel. Local recombination rate estimates for other S. trutta LGs are reported in Table S1 and the average recombination rate of each LG is provided in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439154775" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimates of local recombination rate (&lt;em&gt;&#x3C1;&lt;/em&gt;) in centimorgan per megabase for markers located along two &lt;em&gt;Salmo trutta&lt;/em&gt; linkage groups: (A) LG BT-9 and (B) LG BT-23. The average recombination rate (mean &lt;em&gt;&#x3C1;&lt;/em&gt;) for each LG is given at the top left side of each panel. Local recombination rate estimates for other &lt;em&gt;S. trutta&lt;/em&gt; LGs are reported in Table S1 and the average recombination rate of each LG is provided in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1365/F5.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/4/1365/F5.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484033" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Estimates of local recombination rate (<em>ρ</em>) in centimorgan per megabase for markers located along two <em>Salmo trutta</em> linkage groups: (A) LG BT-9 and (B) LG BT-23. The average recombination rate (mean <em>ρ</em>) for each LG is given at the top left side of each panel. Local recombination rate estimates for other <em>S. trutta</em> LGs are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/TableS1.xlsx">Table S1</a> and the average recombination rate of each LG is provided in <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide positive correlation between the mean nucleotide diversity in the Atlantic brown trout lineage (&#x3C0;) and the local recombination rate (&#x3C1;, in centimorgan per megabase), based on 3038 SNP markers (R2 = 0.058; p-value &lt; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439154775" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide positive correlation between the mean nucleotide diversity in the Atlantic brown trout lineage (&lt;em&gt;&#x3C0;&lt;/em&gt;) and the local recombination rate (&lt;em&gt;&#x3C1;&lt;/em&gt;, in centimorgan per megabase), based on 3038 SNP markers (&lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.058; &lt;em&gt;p&lt;/em&gt;-value &lt; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1365/F6.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/4/1365/F6.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1365/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Genome-wide positive correlation between the mean nucleotide diversity in the Atlantic brown trout lineage (<em>π</em>) and the local recombination rate (<em>ρ</em>, in centimorgan per megabase), based on 3038 SNP markers (<em>R</em><sup>2</sup> = 0.058; <em>p</em>-value &lt; 0.001).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-35">This study describes the first high-density linkage map in the brown trout <em>S. trutta</em>, which extends the microsatellite linkage map already available in this species (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> 2006</a>). The map covers 1403 cM and encompasses 3977 nondistorted markers distributed across 40 LGs, the centromere location of which was determined for 35 LGs. Results showed consistent chromosome number and morphologies compared to species karyotype (2<em>n</em> = 80, NF = 100 chromosome arms; <a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Phillips and Ráb 2001</a>). High-resolution mapping (average density of 3.15 markers per cM) was insured by the use of two mapping families, each containing 150 offspring, and the number of informative markers in both families was close to the limit over which saturation is expected (<em>i.e.</em>, 4200 markers). The homogeneous distribution of makers across the genome, and the estimation of averaged recombination distances between both evolutionary lineages and sexes provide a highly valuable resource for future genomic, aquaculture, conservation, and eco-evolutionary studies in brown trout. Beyond these general interests, this new linkage map also reveals strikingly different rates of chromosomal rearrangements that happened between the brown trout and the Atlantic salmon since their recent divergence.</p><div id="sec-17" class="subsection"><h3>A new generation RAD linkage map in brown trout</h3><p id="p-36">Comparison between the newly developed linkage map and the previous microsatellite map was made possible by the integration of 82 microsatellite markers from the former generation map (<a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> 2006</a>). This enabled us to identify the correspondence between homologous LGs from the two maps, and to detect cases of LG merging (eight out of 40) and LG splitting (nine out of 40) in the microsatellite map compared to the RAD linkage map. Consistent with the reported overrepresentation of acrocentric compared with metacentric chromosomes in the brook charr (<em>i.e.</em>, eight metacentric and 34 acrocentric, <a id="xref-ref-78-10" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>), we found three metacentric and 32 acrocentric chromosomes in the brown trout. The morphology of five chromosomes could not be determined from the analysis of RFm plots alone. However, BT-11 was likely a metacentric chromosome based on its correspondence with the Atlantic salmon LG <em>Ssa</em>5 (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), and BT-17, BT-27, and BT-37 were likely acrocentric chromosomes due to single correspondences with acrocentric chromosomes of the Atlantic salmon and brook charr linkage maps (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). This represents a total number of chromosome arms ranging from 88 to 90, which is lower but quite close to the 96–104 interval previously estimated in <em>S. trutta</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Martinez <em>et al.</em> 1991</a>; <a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Phillips and Ráb 2001</a>). Our assessment of chromosome type did not perfectly match the results from <a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> (2006)</a>, since the three metacentric chromosomes identified here were found to be either acrocentric or undetermined in their study. However, the metacentric chromosomes BT-2 and BT-5 identified here had more than a single matching LG in the former microsatellite map, which suggests that these homologs represent the two telomeric regions of a larger metacentric LG that has been reconstructed in our study. On the contrary, the three metacentric chromosomes identified in <a id="xref-ref-25-11" class="xref-bibr" href="#ref-25">Gharbi <em>et al.</em> (2006)</a> appeared to be acrocentric using the phase information available from our data set. This could be the consequence of pseudolinkage in the microsatellite map built using males of hybrid origin, in which the preferential pairing of conspecific homeologs followed by alternate disjunction may have generated an excess of nonparental gametes (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Morrison 1970</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>). In our study, the possible effect of pseudolinkage was avoided by using nonhybrid parents with a single lineage ancestry, thus possibly explaining the observed differences with the previous generation map.</p><p id="p-37">Besides pseudolinkage, another broader consequence of past whole genome duplication in salmonids is the existence of duplicated loci that share the same alleles due to homeologous recombination in telomeric regions (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Wright <em>et al.</em> 1983</a>; Allendorf and Danzmann 1997; <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>). These residual tetrasomic regions challenge traditional linkage map reconstruction methods since duplicated loci (isoloci) often show Mendelian segregation distortion, and are therefore frequently excluded from linkage mapping analyses. Alternative approaches can be used to detect homeologous regions in experimental crosses (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Waples <em>et al.</em> 2016</a>). In brown trout, the location of duplicated regions have been preliminarily established from genome-wide patterns of RAD markers density and nucleotide diversity projected on the Atlantic salmon reference genome (<a id="xref-ref-46-11" class="xref-bibr" href="#ref-46">Leitwein <em>et al.</em> 2016</a>). Although our new RAD linkage map is based on nondistorted markers and therefore probably displays a reduced marker density in regions exhibiting tetrasomic inheritance, it does not completely exclude such regions. This is illustrated by the mapping of multiple RAD loci within the telomeric region of the seven pairs of homeolog chromosome arms that retain residual tetrasomy in the Atlantic salmon genome (<a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>) (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Except for three of these 14 regions that received relatively few markers (<em>Ssa</em>3q, <em>Ssa</em>11qa, and <em>Ssa</em>17qa on the salmon map; <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>), the remaining arms were well covered in our RAD linkage map, indicating that the filtering of non-Mendelian markers did not completely exclude the telomeric regions affected by homeologous crossovers. This feature of our RAD linkage map should insure a more even recombination rate estimation between sexes (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Lien <em>et al.</em> 2011</a>), since the commonly observed lower recombination rate in males compared to females in salmonids could be partly due to a poor characterization of telomeric regions in mapping studies (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 2015</a>). In order to evaluate the extent to which male and female informative markers tend to map to distinct regions, we compared their positions along each LG in <span class="sc">family</span> 1 that received the highest number of mapped markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS3.pdf">Figure S3</a>). We observed that on average, informative markers in the male parent were less evenly distributed across LGs and tended to aggregate in regions containing lower densities of female informative markers. Although this might also reflect a poor positioning of male markers due to low recombination rates in males (<a id="xref-ref-78-11" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>), the sex-averaged recombination distances in our consensus linkage map should provide relevant estimates of recombination rate along chromosomes for future eco-evolutionary studies. For instance, understanding genome-wide introgression patterns in nature needs to integrate information from both male and female recombination rate.</p><p id="p-38">Following the same idea, integrating maps between parents of different lineage origins (Atlantic and Mediterranean) allowed us to generate a consensus linkage map in which the potential differences in recombination rates between these two allopatric lineages are smoothened. Our results suggest that no fusion polymorphism occurs between the two brown trout lineages. This is consistent with the lack of species–specific chromosomal rearrangement following speciation with the Atlantic salmon (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), and the stable chromosome number previously observed in brown trout (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hartley and Horne 1984</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Martinez <em>et al.</em> 1991</a>; <a id="xref-ref-67-6" class="xref-bibr" href="#ref-67">Phillips and Ráb 2001</a>). Therefore, the lineage-averaged recombination distances provided here are probably not affected by potential interchromosomal rearrangements within the <em>S. trutta</em> species. The lineage-averaged consensus map will therefore be valuable to future conservation genomic studies, considering the extensive human-mediated admixture between Atlantic and Mediterranean brown trout populations in southern Europe (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Berrebi <em>et al.</em> 2000</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Sanz <em>et al.</em> 2006</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Thaulow <em>et al.</em> 2014</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Berrebi 2015</a>).</p></div><div id="sec-18" class="subsection"><h3>The recent evolution of chromosomal rearrangements in the Salmo genus</h3><p id="p-39">Salmonids have been undergoing a rediploidization process since a whole genome duplication event that occurred ∼60 MYA (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Crête-Lafrenière <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Near <em>et al.</em> 2012</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Macqueen and Johnston 2014</a>; <a id="xref-ref-50-8" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>). This process, also described as functional diploidization by <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Ohno (1970)</a>, was accompanied by numerous chromosomal rearrangements within the salmonid family, making linkage maps of prime importance for studying the complex history of chromosomal evolution in salmonids (<a id="xref-ref-78-12" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>). Several studies have pointed out a particularly high number of chromosomal rearrangements in the lineage leading to the Atlantic salmon, involving mainly chromosome fusions that reduced the number of chromosomes to 29 in this species (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-50-9" class="xref-bibr" href="#ref-50">Lien <em>et al.</em> 2016</a>; <a id="xref-ref-78-13" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>). However, the timing of these rearrangements remains largely unknown due to the lack of a closely related species in previous studies. Because <em>S. trutta</em> is the most closely related species to <em>S. salar</em> among salmonids, the new brown trout RAD linkage map allowed for inferring the phylogenetic positions of the chromosome arm rearrangements that occurred within the <em>Salmo</em> genus. The ancestral state of <em>Salmo</em> chromosome structure was further resolved using the brook charr as an outgroup within salmonids, as well as other salmonid species (<em>e.g.</em>, the lake whitefish <em>Coregonus clupeaformis</em>) when necessary (<a id="xref-ref-78-14" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>). This enabled us to identify five fusion events and three fission events (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>) that occurred in the <em>Salmo</em> ancestral lineage before the Atlantic salmon/brown trout speciation. This is relatively small compared to the total number of chromosomal rearrangements that occurred within the whole <em>Salmo</em> branch. Thus, most of the fission and fusion events that were observed in the Atlantic salmon genome (<a id="xref-ref-78-15" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>) most probably occurred after the divergence between the two <em>Salmo</em> species. The net nucleotide divergence between <em>S. salar</em> and <em>S. trutta</em> was here estimated to 1.87%, which indicates a relatively recent divergence time between these two sister species. Surprisingly, all of the interchromosomal rearrangements that occurred after the salmon/trout speciation happened in <em>S. salar</em>. Indeed, we did not observe any fusion/fission events within the <em>S. trutta</em> branch (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>), whereas 15 rearrangements mostly involving fusions (13 fusions and two fissions) specifically occurred in the Atlantic salmon branch. These Robertsonian translocations explain the increased number of metacentric chromosomes in the Atlantic salmon compared to the brown trout, which were produced by the recent fusion of acrocentric chromosomes (<em>Ssa</em>1 to <em>Ssa</em>4; <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>). However, some fusion events have also likely resulted in the formation of acrocentric chromosomes in the Atlantic salmon (<em>e.g.</em>, <em>Ssa</em>15; <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>), which could partly explain the apparent loss of chromosomes arms in this species (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allendorf and Thorgaard 1984</a>). Finally, intrachromosomic rearrangements also occurred along the <em>S. salar</em> branch, as illustrated by the inversion detected within <em>Ssa</em>6, the homologous LG to BT-5 (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1/FigureS2.pdf">Figure S2</a>). Altogether, our results support the previous views that many chromosomal rearrangements have occurred within the <em>Salmo</em> lineage (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-78-16" class="xref-bibr" href="#ref-78">Sutherland <em>et al.</em> 2016</a>), but also demonstrate that most of them recently happened in the Atlantic salmon branch, while at the same time, <em>S. trutta</em> has retained a chromosomal architecture close to the ancestral form of the <em>Salmo</em> genus. The reasons for such a contrasted evolution of chromosome structure between these two species remain to be investigated.</p></div><div id="sec-19" class="subsection"><h3>Recombination rate variation across the brown trout genome</h3><p id="p-40">Despite extensive chromosomal rearrangements in the <em>S. salar</em> lineage, we could identify highly conserved synteny blocks within orthologous chromosome arms to reconstitute local alignments between the brown trout linkage map and the salmon physical map. One of our main objectives was to use these comparisons between maps to estimate local recombination rate variations along each LG in <em>S. trutta</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The average genome-wide recombination rate was estimated at 0.88 cM/Mb, with strong heterogeneity being found across the genome. Highly recombining regions were observed at the extremities of some LGs (<em>e.g.</em>, BT-9; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>), and genomic regions associated with very low recombination rates were also observed (<em>e.g.</em>, BT-23; <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). In order to provide an indirect assessment of this local recombination rate estimation for future eco-evolutionary studies, we verified that the frequently observed positive correlation between local recombination rate and nucleotide diversity (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Corbett-Detig <em>et al.</em> 2015</a>) also occurs in brown trout. In a comparative study across a wide range of species, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Corbett-Detig <em>et al.</em> (2015)</a> suggested that the strength of this correlation is highly variable among species, being generally stronger in species with large <em>vs.</em> small census population size [but see <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Coop (2016)</a>]. Although brown trout populations are usually considered to have relatively small census sizes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jorde and Ryman 1996</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Palm <em>et al.</em> 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jensen <em>et al.</em> 2005</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Serbezov <em>et al.</em> 2012</a>), a significantly positive correlation was found between local recombination rate and nucleotide diversity using polymorphism data from the Atlantic lineage. Thus, recombination rate variation across the brown trout genome shapes the chromosomal landscape of neutral genetic diversity by modulating the efficiency of selection (both positive and negative) at linked neutral sites.</p><p id="p-41">Variation in local recombination rate across genomic regions not only affects genome-wide variation patterns of polymorphism, but also has a profound influence on a range of evolutionary mechanisms of prime importance in brown trout. This recombination map should therefore have beneficial outcomes for our understanding of inbreeding depression, local adaptation, and the historical demography of natural populations (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Rezvoy <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dukić <em>et al.</em> 2016</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Racimo <em>et al.</em> 2015</a>). It will also be of precious help for understanding genome-wide introgression patterns in natural populations that have been restocked with individuals from a different lineage. Finally, a detailed picture of recombination rate variation in brown trout will help for choosing SNPs for the design of genotyping arrays in future association mapping experiments (<em>e.g.</em>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Bourret <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Houston <em>et al.</em> 2014</a>).</p></div><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-42">A high-density sex- and lineage-averaged consensus linkage map for <em>S. trutta</em> was developed and the morphology (acrocentric or metacentric) of most chromosomes was identified. Interspecies comparisons with the Atlantic salmon allowed identifying genomic regions of highly conserved synteny within chromosome arms and contribute toward a better understanding of recent genome evolution in the genus <em>Salmo</em>. A recent and strong acceleration in the rate of chromosomal rearrangements was detected in the <em>S. salar</em> branch. These results should stimulate further research toward understanding the contrasted evolution of chromosome structure in sister species with &lt;2% net nucleotide divergence. Recombination rate variation was found to influence genome-wide nucleotide diversity patterns in brown trout. These new genomic resources will provide important tools for future genome scans, QTL mapping, and genome-wide association studies in brown trout. It also paves the way to enable a new generation of conservation genomics approaches that will look at the fitness consequences of introgression of foreign alleles into wild populations at the haplotype level.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank E. Ravel and people from the Fédération Départementale de la Pêche de l’Hérault (France) and the Babeau hatchery for their help at several stages of this project. We also thank Y. Anselmetti for his kind advice on bioinformatics computing, M. Limborg for his helpful scripts and advice, and B. Sutherland for his suggestions with MapComp. This project largely benefited from the Montpellier Bioinformatics Biodiversity cluster computing platform. Presequencing steps necessary for the production of SNP data were performed at the Genotyping and Sequencing facility of the LabEx CeMEB (Centre Méditerranéen pour l’Environnement et la Biodiversité, Montpellier), and RAD sequencing was performed at Montpellier GenomiX facility (Montpellier, France; <a href="http://www.mgx.cnrs.fr/">http://www.mgx.cnrs.fr/</a>). M.L. was partly supported by a grant from LabEx CeMEB.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-44">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038497/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-45"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-12-16"><span class="received-label">Received </span>December 16, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-02-22"><span class="accepted-label">Accepted </span>February 22, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Leitwein <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-7.4.1365.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Tetraploidy and the evolution of salmonid fishes</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">53</span> in <span class="cit-source"><em>Evolutionary Genetics of Fishes</em>, <em>Monographs in Evolutionary Biology</em></span>, <span class="cit-vol">Volume 1</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. J.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BGenetics%2Bof%2BFishes%252C%2BMonographs%2Bin%2BEvolutionary%2BBiology%26rft.volume%253D1%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1365.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Secondary Tetrasomic Segregation of MDH-B and Preferential Pairing of Homeologues in Rainbow Trout.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1083%26rft.epage%253D1092%26rft.atitle%253DSecondary%2BTetrasomic%2BSegregation%2Bof%2BMDH-B%2Band%2BPreferential%2BPairing%2Bof%2BHomeologues%2Bin%2BRainbow%2BTrout%26rft_id%253Dinfo%253Apmid%252F9093860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NS80LzEwODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.3" data-doi="10.1093/jhered/esv015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Effects of crossovers between homeologs on inheritance and population genomics in polyploid-derived salmonid fishes.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv015%26rft_id%253Dinfo%253Apmid%252F25838153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNi8zLzIxNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almodóvar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicola</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elvira</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Marin</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Introgression variability among Iberian brown trout evolutionary significant units: the influence of local management and environmental features.</span> <abbr class="cit-jnl-abbrev">Freshw. Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1175</span>–<span class="cit-lpage">1187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFreshw.%2BBiol.%26rft.volume%253D51%26rft.spage%253D1175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.5" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontenot</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution and meiotic maps by massively parallel DNA sequencing: spotted gar, an outgroup for the teleost genome duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAmores%26rft.auinit1%253DA.%26rft.volume%253D188%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D808%26rft.atitle%253DGenome%2BEvolution%2Band%2BMeiotic%2BMaps%2Bby%2BMassively%2BParallel%2BDNA%2BSequencing%253A%2BSpotted%2BGar%252C%2Ban%2BOutgroup%2Bfor%2Bthe%2BTeleost%2BGenome%2BDuplication%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvNzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.6" data-doi="10.1554/0014-3820(2001)055[0351:TEHOBT]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The evolutionary history of brown trout (<em>Salmo trutta</em> L.) inferred from phylogeographic, nested clade, and mismatch analyses of mitochondrial DNA variation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBernatchez%26rft.auinit1%253DL.%26rft.volume%253D55%26rft.issue%253D2%26rft.spage%253D351%26rft.epage%253D379%26rft.atitle%253DThe%2Bevolutionary%2Bhistory%2Bof%2Bbrown%2Btrout%2B%2528Salmo%2Btrutta%2BL.%2529%2Binferred%2Bfrom%2Bphylogeographic%252C%2Bnested%2Bclade%252C%2Band%2Bmismatch%2Banalyses%2Bof%2Bmitochondrial%2BDNA%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B0351%253ATEHOBT%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11308093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[0351:TEHOBT]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11308093&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167552800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osinov</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic diversity of trout (genus <em>Salmo</em>) from its most eastern native range based on mitochondrial DNA and nuclear gene variation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBernatchez%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D297%26rft.atitle%253DGenetic%2Bdiversity%2Bof%2Btrout%2B%2528genus%2BSalmo%2529%2Bfrom%2Bits%2Bmost%2Beastern%2Bnative%2Brange%2Bbased%2Bon%2Bmitochondrial%2BDNA%2Band%2Bnuclear%2Bgene%2Bvariation.%26rft_id%253Dinfo%253Apmid%252F7663748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7663748&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Three brown trout <em>Salmo trutta</em> lineages in Corsica described through allozyme variation.</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BBiol.%26rft.volume%253D86%26rft.spage%253D60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.9" data-doi="10.1111/j.1095-8649.2000.tb00884.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poteaux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fissier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattaneo-Berrebi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Stocking impact and allozyme diversity in brown trout from Mediterranean southern France.</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BBiol.%26rft.volume%253D56%26rft.spage%253D949%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8649.2000.tb00884.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8649.2000.tb00884.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000086046800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.10" data-doi="10.1038/ncomms4657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalopin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juanchich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3657%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4657%26rft_id%253Dinfo%253Apmid%252F24755649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755649&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haffray</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic diversity and population structure of domestic brown trout (<em>Salmo trutta</em>) in France.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D462%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.12" data-doi="10.1111/mec.12003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourret</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Primmer</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasemägi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP-array reveals genome-wide patterns of geographical and potential adaptive divergence across the natural range of Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313726300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1365.13" data-doi="10.1186/1471-2164-13-432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenna-Hansen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulding</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosomal Differences between European and North American Atlantic Salmon Discovered by Linkage Mapping and Supported by Fluorescence in Situ Hybridization Analysis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBrenna-Hansen%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.spage%253D432%26rft.epage%253D432%26rft.atitle%253DChromosomal%2Bdifferences%2Bbetween%2BEuropean%2Band%2BNorth%2BAmerican%2BAtlantic%2Bsalmon%2Bdiscovered%2Bby%2Blinkage%2Bmapping%2Band%2Bsupported%2Bby%2Bfluorescence%2Bin%2Bsitu%2Bhybridization%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-432%26rft_id%253Dinfo%253Apmid%252F22928605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22928605&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A dense linkage map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) reveals variable chromosomal divergence after an ancestral whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.15" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Does linked selection explain the narrow range of genetic diversity across species?</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> Available at: <a href="http://biorxiv.org/content/early/2016/03/07/042598">http://biorxiv.org/content/early/2016/03/07/042598</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.17" data-doi="10.1371/journal.pbio.1002112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sackton</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Natural selection constrains neutral diversity across a wide range of species.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1002112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002112%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002112%26rft_id%253Dinfo%253Apmid%252F25859758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25859758&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.18" data-doi="10.1371/journal.pone.0046662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crête-Lafrenière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Framing the Salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrete-Lafreniere%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46662%26rft.epage%253De46662%26rft.atitle%253DFraming%2Bthe%2Bsalmonidae%2Bfamily%2Bphylogenetic%2Bportrait%253A%2Ba%2Bmore%2Bcomplete%2Bpicture%2Bfrom%2Bincreased%2Btaxon%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046662%26rft_id%253Dinfo%253Apmid%252F23071608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071608&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dukić</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haag</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A high-density genetic map reveals variation in recombination rate across the genome of <em>Daphnia magna</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.20" data-doi="10.1038/hdy.2010.152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekblom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Applications of next generation sequencing in molecular ecology of non-model organisms.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DEkblom%26rft.auinit1%253DR.%26rft.volume%253D107%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D15%26rft.atitle%253DApplications%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bin%2Bmolecular%2Becology%2Bof%2Bnon-model%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.152%26rft_id%253Dinfo%253Apmid%252F21139633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21139633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291635500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-7.4.1365.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source"><em>Quantitative Ecology and the Brown Trout</em></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.22" data-doi="10.1186/1471-2164-13-521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DEverett%26rft.auinit1%253DM.%2BV.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D521%26rft.epage%253D521%26rft.atitle%253DMeiotic%2Bmaps%2Bof%2Bsockeye%2Bsalmon%2Bderived%2Bfrom%2Bmassively%2Bparallel%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-521%26rft_id%253Dinfo%253Apmid%252F23031582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23031582&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.23" data-doi="10.1111/mec.12127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Mapping phenotypic, expression and transmission ratio distortion QTL using RAD markers in the Lake Whitefish (<em>Coregonus clupeaformis</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3036</span>–<span class="cit-lpage">3048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3036%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.24" data-doi="10.1111/evo.12075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">The genetic architecture of reproductive isolation during speciation-with-gene-flow in lake whitefish species pairs assessed by RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">2483</span>–<span class="cit-lpage">2497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D2483%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12075%26rft_id%253Dinfo%253Apmid%252F24033162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24033162&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323828500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.25" data-doi="10.1534/genetics.105.048330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A linkage map for brown trout (<em>Salmo trutta</em>): chromosome homeologies and comparative genome organization with other salmonid fish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2405</span>–<span class="cit-lpage">2419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048330%26rft_id%253Dinfo%253Apmid%252F16452148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI0MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.26" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>) genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.27" data-doi="10.1098/rsbl.2009.0214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mensberg</span>  <span class="cit-name-given-names">K. L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Admixture analysis of stocked brown trout populations using mapped microsatellite DNA markers: indigenous trout persist in introgressed populations.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2009.0214%26rft_id%253Dinfo%253Apmid%252F19515653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NzoiNS81LzY1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.28" data-doi="10.1046/j.1365-294x.2000.00898.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruzzante</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mensberg</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Microsatellite and mitochondrial DNA polymorphism reveals life-history dependent interbreeding between hatchery and wild brown trout (<em>Salmo trutta</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHansen%26rft.auinit1%253DM.%2BM.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D583%26rft.epage%253D594%26rft.atitle%253DMicrosatellite%2Band%2Bmitochondrial%2BDNA%2Bpolymorphism%2Breveals%2Blife-history%2Bdependent%2Binterbreeding%2Bbetween%2Bhatchery%2Band%2Bwild%2Bbrown%2Btrout%2B%2528Salmo%2Btrutta%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.2000.00898.x%26rft_id%253Dinfo%253Apmid%252F10792701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.2000.00898.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10792701&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mensberg</span>  <span class="cit-name-given-names">K. L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identifying footprints of selection in stocked brown trout populations: a spatio-temporal approach.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1787</span>–<span class="cit-lpage">1800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHansen%26rft.auinit1%253DM.%2BM.%26rft.volume%253D19%26rft.issue%253D9%26rft.spage%253D1787%26rft.epage%253D1800%26rft.atitle%253DIdentifying%2Bfootprints%2Bof%2Bselection%2Bin%2Bstocked%2Bbrown%2Btrout%2Bpopulations%253A%2Ba%2Bspatio-temporal%2Bapproach.%26rft_id%253Dinfo%253Apmid%252F20345684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20345684&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartley</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horne</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Chromosome relationships in the genus Salmo.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DHartley%26rft.auinit1%253DS.%2BE.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D229%26rft.epage%253D237%26rft.atitle%253DChromosome%2Brelationships%2Bin%2Bthe%2Bgenus%2BSalmo.%26rft_id%253Dinfo%253Apmid%252F6499596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6499596&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thrower</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of migration-related traits in rainbow and steelhead trout, <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D2%26rft.spage%253D1113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.32" data-doi="10.1111/j.1755-0998.2010.02967.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amish</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation RAD sequencing identifies thousands of SNPs for assessing hybridization between rainbow and westslope cutthroat trout.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02967.x%26rft_id%253Dinfo%253Apmid%252F21429168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02967.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429168&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287485100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.33" data-doi="10.1186/1471-2164-13-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mota-Velasco</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D13%26rft.spage%253D244%26rft.epage%253D244%26rft.atitle%253DCharacterisation%2Bof%2BQTL-linked%2Band%2Bgenome-wide%2Brestriction%2Bsite-associated%2BDNA%2B%2528RAD%2529%2Bmarkers%2Bin%2Bfarmed%2BAtlantic%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-244%26rft_id%253Dinfo%253Apmid%252F22702806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22702806&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.34" data-doi="10.1186/1471-2164-15-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-90%26rft_id%253Dinfo%253Apmid%252F24524230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24524230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeschcke</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mensberg</span>  <span class="cit-name-given-names">K. L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Spatial and temporal genetic differentiation and effective population size of brown trout (<em>Salmo trutta</em>, L.) in small Danish rivers.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D6%26rft.spage%253D615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Demographic genetics of brown trout (<em>Salmo trutta</em>) and estimation of effective population size from temporal change of allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1369</span>–<span class="cit-lpage">1381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJorde%26rft.auinit1%253DP.%2BE.%26rft.volume%253D143%26rft.issue%253D3%26rft.spage%253D1369%26rft.epage%253D1381%26rft.atitle%253DDemographic%2BGenetics%2Bof%2BBrown%2BTrout%2B%2528Salmo%2Btrutta%2529%2Band%2BEstimation%2Bof%2BEffective%2BPopulation%2BSize%2BFrom%2BTemporal%2BChange%2Bof%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F8807308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My8zLzEzNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.37" data-doi="10.1186/1471-2164-15-233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nomura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujiwara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuike</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (<em>Anguilla japonica</em>) provides insights into genome evolution after the teleost-specific genome duplication.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-233%26rft_id%253Dinfo%253Apmid%252F24669946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24669946&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hard</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative mapping between coho salmon (<em>Oncorhynchus kisutch</em>) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D1717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuang</span>  <span class="cit-name-given-names">Y.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">D.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic map of goldfish (<em>Carassius auratus</em>) provided insights to the divergent genome evolutions in the Cyprinidae family.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">34849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D34849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocour</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Applications of next-generation sequencing in fisheries research: a review.</span> <abbr class="cit-jnl-abbrev">Fish. Res.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish.%2BRes.%26rft.volume%253D186%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1365.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laikre</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-comment">Conservation genetic management of brown trout (Salmo trutta) in Europe.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.42" data-doi="10.1111/j.1365-294X.2012.05579.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamaze</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dynamics of introgressive hybridization assessed by SNP population genomics of coding genes in stocked brook charr (<em>Salvelinus fontinalis</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2877</span>–<span class="cit-lpage">2895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLamaze%26rft.auinit1%253DF.%2BC.%26rft.volume%253D21%26rft.issue%253D12%26rft.spage%253D2877%26rft.epage%253D2895%26rft.atitle%253DDynamics%2Bof%2Bintrogressive%2Bhybridization%2Bassessed%2Bby%2BSNP%2Bpopulation%2Bgenomics%2Bof%2Bcoding%2Bgenes%2Bin%2Bstocked%2Bbrook%2Bcharr%2B%2528Salvelinus%2Bfontinalis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05579.x%26rft_id%253Dinfo%253Apmid%252F22548328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05579.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22548328&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304903100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.43" data-doi="10.1046/j.1365-294X.1996.00099.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Largiader</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholl</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Genetic introgression between native and introduced brown trout <em>Salmo trutta</em> L. populations in the Rhone River Basin.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D5%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.1996.00099.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.1996.00099.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.44" data-doi="10.1111/eva.12128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D7%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12128%26rft_id%253Dinfo%253Apmid%252F24665338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24665338&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.45" data-doi="10.1093/jhered/esv099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of multiple QTL hotspots in sockeye salmon (<em>Oncorhynchus nerka</em>) using genotyping-by-sequencing and a dense linkage map.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv099%26rft_id%253Dinfo%253Apmid%252F26712859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNy8yLzEyMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitwein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guendouz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide nucleotide diversity of hatchery-reared Atlantic and Mediterranean strains of brown trout <em>Salmo trutta</em> compared to wild Mediterranean populations.</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2717</span>–<span class="cit-lpage">2734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BBiol.%26rft.volume%253D89%26rft.spage%253D2717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.47" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.48" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-comment">Statistical Framework for SNP Calling, Mutation Discovery, Association Mapping and Population Genetical Parameter Estimation from Sequencing Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMjk4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.49" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Dense SNP-Based Linkage Map for Atlantic Salmon (Salmo Salar) Reveals Extended Chromosome Homeologies and Striking Differences in Sex-Specific Recombination Patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D533%26rft.spage%253D200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.51" data-doi="10.1093/jhered/esu063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Temporally isolated lineages of pink salmon reveal unique signatures of selection on distinct pools of standing genetic variation.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesu063%26rft_id%253Dinfo%253Apmid%252F25292170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNS82LzgzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Recombination patterns reveal information about centromere location on linkage maps.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Linkage mapping reveals strong chiasma interference in sockeye salmon: implications for interpreting genomic data.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2463</span>–<span class="cit-lpage">2473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D2463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-density interspecific genetic linkage mapping provides insights into genomic incompatibility between channel catfish and blue catfish.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D47%26rft.spage%253D81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.55" data-doi="10.1098/rspb.2013.2881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macqueen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20132881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2881%26rft_id%253Dinfo%253Apmid%252F24452024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3NzgvMjAxMzI4ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouza</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Cytogenetical characterization of hatchery stocks and natural populations of sea and brown trout from northwestern Spain.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D66%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.57" data-doi="10.1093/jhered/esv056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage, and chromosomal pairing in tetraploid derivative salmonid genomes: II. Wright is still right.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv056%26rft_id%253Dinfo%253Apmid%252F26320244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNi82Lzc2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Uchida</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An integrated linkage map reveals candidate genes underlying adaptive variation in Chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMeel</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoey</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Partial nucleotide sequences, and routine typing by polymerase chain reaction–restriction fragment length polymorphism, of the brown trout (<em>Salmo trutta</em>) lactate dehydrogenase, LDH-C1*90 and *100 alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMcMeel%26rft.auinit1%253DO.%2BM.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D34%26rft.atitle%253DPartial%2Bnucleotide%2Bsequences%252C%2Band%2Broutine%2Btyping%2Bby%2Bpolymerase%2Bchain%2Breaction-restriction%2Bfragment%2Blength%2Bpolymorphism%252C%2Bof%2Bthe%2Bbrown%2Btrout%2B%2528Salmo%2Btrutta%2529%2Blactate%2Bdehydrogenase%252C%2BLDH-C1%2A90%2Band%2B%2A100%2Balleles.%26rft_id%253Dinfo%253Apmid%252F11251784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11251784&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.60" data-doi="10.1577/1548-8659(1970)99&lt;193:NSOTLD&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Nonrandom segregation of two lactate dehydrogenase subunit loci in trout.</span> <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D99%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1577%252F1548-8659%25281970%252999%253C193%253ANSOTLD%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/1548-8659(1970)99&lt;193:NSOTLD&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.61" data-doi="10.1073/pnas.1206625109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Near</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eytan</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dornburg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Resolution of ray-finned fish phylogeny and timing of diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13698</span>–<span class="cit-lpage">13703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206625109%26rft_id%253Dinfo%253Apmid%252F22869754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM0LzEzNjk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.62" data-doi="10.1534/g3.116.038026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nugent</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Easton</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A SNP based linkage map of the Arctic charr (<em>Salvelinus alpinus</em>) genome provides insights into the diploidization process after whole genome duplication.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.038026%26rft_id%253Dinfo%253Apmid%252F27986793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi81NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.63" data-doi="10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">The enormous diversity in genome sizes of fish as a reflection of nature’s extensive experiments with gene duplication.</span> <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D99%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1577%252F1548-8659%25281970%252999%253C120%253ATEDIGS%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.64" data-doi="10.1023/A:1024064913094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palm</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laikre</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effective population size and temporal genetic change in stream resident brown trout (<em>Salmo trutta</em>, <em>L</em>.).</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D4%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1024064913094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1024064913094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.65" data-doi="10.1111/mec.12500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourret</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Parallel and nonparallel genome-wide divergence among replicate population pairs of freshwater and anadromous Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">5577</span>–<span class="cit-lpage">5593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D5577%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.66" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for <em>de novo</em> SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.67" data-doi="10.1017/S1464793100005613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ráb</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D25%26rft.atitle%253DChromosome%2Bevolution%2Bin%2Bthe%2BSalmonidae%2B%2528Pisces%2529%253A%2Ban%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793100005613%26rft_id%253Dinfo%253Apmid%252F11325050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793100005613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11325050&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poteaux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaudou</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Temporal variations of genetic introgression in stocked brown trout populations.</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BBiol.%26rft.volume%253D53%26rft.spage%253D701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poteaux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Microsatellite polymorphism and genetic impact of restocking in Mediterranean brown trout (<em>Salmo trutta L</em>.).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D82%26rft.spage%253D645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.70" data-doi="10.1038/nrg3936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sánchez</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evidence for archaic adaptive introgression in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3936%26rft_id%253Dinfo%253Apmid%252F25963373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3936&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25963373&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.71" data-doi="10.1093/bioinformatics/btm315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rezvoy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charif</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guéguen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marais</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MareyMap: an R-based tool with graphical interface for estimating recombination rates.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2188</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm315%26rft_id%253Dinfo%253Apmid%252F17586550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTYvMjE4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.72" data-doi="10.1371/journal.pone.0102089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rondeau</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messmer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jantzen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genome and linkage map of the northern pike (<em>Esox lucius</em>): conserved synteny revealed between the salmonid sister group and the Neoteleostei.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e102089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De102089%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0102089%26rft_id%253Dinfo%253Apmid%252F25069045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0102089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25069045&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cortey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pla</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Marín</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hatchery introgression blurs ancient hybridization between brown trout (<em>Salmo trutta</em>) lineages as indicated by complementary allozymes and mtDNA markers.</span> <abbr class="cit-jnl-abbrev">Biol. Conserv.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv.%26rft.volume%253D130%26rft.spage%253D278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derôme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Coding gene single nucleotide polymorphism mapping and quantitative trait loci detection for physiological reproductive traits in brook charr, <em>Salvelinus fontinalis</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D2%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.75" data-doi="10.1111/j.1752-4571.2012.00239.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serbezov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vøllestad</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Life history and demographic determinants of effective/census size ratios as exemplified by brown trout (<em>Salmo trutta</em>).</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEvol%2BAppl%26rft.aulast%253DSerbezov%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D607%26rft.epage%253D618%26rft.atitle%253DLife%2Bhistory%2Band%2Bdemographic%2Bdeterminants%2Bof%2Beffective%252Fcensus%2Bsize%2Bratios%2Bas%2Bexemplified%2Bby%2Bbrown%2Btrout%2B%2528Salmo%2Btrutta%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2012.00239.x%26rft_id%253Dinfo%253Apmid%252F23028401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2012.00239.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028401&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gratten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beraldi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene mapping in the wild with SNPs: guidelines and future directions.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.77" data-doi="10.1016/j.tree.2010.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smadja</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation genomics: the next generation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DStapley%26rft.auinit1%253DJ.%26rft.volume%253D25%26rft.issue%253D12%26rft.spage%253D705%26rft.epage%253D712%26rft.atitle%253DAdaptation%2Bgenomics%253A%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2010.09.002%26rft_id%253Dinfo%253Apmid%252F20952088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2010.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20952088&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284795800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosselin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamothe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3600</span>–<span class="cit-lpage">3617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D8%26rft.spage%253D3600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thaulow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borgstrøm</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heun</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic persistence of an initially introduced brown trout (<em>Salmo trutta L</em>.) population despite restocking of foreign conspecifics.</span> <abbr class="cit-jnl-abbrev">Ecol. Freshwat. Fish</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BFreshwat.%2BFish%26rft.volume%253D23%26rft.spage%253D485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genetic architecture of growth and fillet traits in farmed Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction and annotation of a high density SNP linkage map of the Atlantic salmon (<em>Salmo salar</em>) genome.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2173</span>–<span class="cit-lpage">2179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D2173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-book" id="cit-7.4.1365.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source"><em>JoinMap 4</em>; <em>Software for the Calculation of Genetic Map in Experimental Populations</em></span>. <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.83" data-doi="10.1038/srep16358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of a high-density linkage map and fine mapping of QTL for growth in Asian seabass.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">16358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D16358%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep16358%26rft_id%253Dinfo%253Apmid%252F26553309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep16358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26553309&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Linkage mapping with paralogs exposes regions of residual tetrasomic inheritance in chum salmon (<em>Oncorhynchus keta</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes.</span> <abbr class="cit-jnl-abbrev">Isozymes</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIsozymes%26rft.stitle%253DIsozymes%2BCurr%2BTop%2BBiol%2BMed%2BRes%26rft.aulast%253DWright%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D10%26rft.spage%253D239%26rft.epage%253D260%26rft.atitle%253DMeiotic%2Bmodels%2Bto%2Bexplain%2Bclassical%2Blinkage%252C%2Bpseudolinkage%252C%2Band%2Bchromosome%2Bpairing%2Bin%2Btetraploid%2Bderivative%2Bsalmonid%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F6354984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6354984&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RL29500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.86" data-doi="10.1111/eva.12032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouanno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fostier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The sexually dimorphic on the Y-chromosome gene (sdY) is a conserved male-specific Y-chromosome sequence in many salmonids.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D6%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12032%26rft_id%253Dinfo%253Apmid%252F23745140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23745140&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1365.87" data-doi="10.1371/journal.pone.0063928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A dense genetic linkage map for common carp and its integration with a BAC-based physical map.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e63928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De63928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063928%26rft_id%253Dinfo%253Apmid%252F23704958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23704958&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/4/1365.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1357" title="pnp4a Is the Causal Gene of the Medaka Iridophore Mutant guanineless" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1377" title="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/4"><span>Volume 7 Issue 4, April 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/4.cover-source.jpg?itok=KOTpzmYp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ohu-34sW19z_ltEN6ENn-HyR37PHx5uOP9m1JpJajOI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/4/1365" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/4/1365" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/4/1365" class="active" data-icon-position="" data-hide-link-title="0">A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in the Salmo Genus and the Impact of Selection on Linked Neutral Diversity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483841" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-loX2yKlc2R77VqHhlo7F-Dr5Uo2gUngMxV1GVxoGyf8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1365.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/4/1365" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vlETToLrD3O50Nd4jLucmjxa7gE8MQUVSajpX5On5YU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/4/1365" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1365.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483841" id="node483841--2" data-pisa="ggg;7/4/1365" data-pisa-master="ggg;g3.116.038497" data-apath="/ggg/7/4/1365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dense Brown Trout (<em>Salmo trutta</em>) Linkage Map Reveals Recent Chromosomal Rearrangements in the <em>Salmo</em> Genus and the Impact of Selection on Linked Neutral Diversity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5172-0970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maeva Leitwein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6934-1677" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bruno Guinand</span>, <span class="highwire-citation-author" data-delta="2">Juliette Pouzadoux</span>, <span class="highwire-citation-author" data-delta="3">Erick Desmarais</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9026-4987" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Berrebi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1908-3235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre-Alexandre Gagnaire</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1365-1376;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038497 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483841--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maeva Leitwein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maeva%2BLeitwein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leitwein%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaeva%2BLeitwein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5172-0970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maeva Leitwein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maeva.leitwein.pro@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maeva.leitwein.pro@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bruno Guinand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Département Biologie-Ecologie, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BGuinand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guinand%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BGuinand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6934-1677" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bruno Guinand</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juliette Pouzadoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliette%2BPouzadoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pouzadoux%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliette%2BPouzadoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Erick Desmarais </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erick%2BDesmarais%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desmarais%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErick%2BDesmarais%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick Berrebi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BBerrebi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berrebi%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BBerrebi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9026-4987" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Berrebi</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Pierre-Alexandre Gagnaire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre-Alexandre%2BGagnaire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gagnaire%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre-Alexandre%2BGagnaire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1908-3235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre-Alexandre Gagnaire</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/4/1365" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/4/1365&amp;title=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/4/1365&amp;title=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/4/1365&amp;title=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/4/1365&amp;text=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/4/1365&amp;title=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/4/1365&amp;t=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/4/1365&amp;title=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/4/1365&amp;title=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483841/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483841" id="node483841--3" data-pisa="ggg;7/4/1365" data-pisa-master="ggg;g3.116.038497" data-apath="/ggg/7/4/1365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dense Brown Trout (<em>Salmo trutta</em>) Linkage Map Reveals Recent Chromosomal Rearrangements in the <em>Salmo</em> Genus and the Impact of Selection on Linked Neutral Diversity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5172-0970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maeva Leitwein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6934-1677" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bruno Guinand</span>, <span class="highwire-citation-author" data-delta="2">Juliette Pouzadoux</span>, <span class="highwire-citation-author" data-delta="3">Erick Desmarais</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9026-4987" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Berrebi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1908-3235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre-Alexandre Gagnaire</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1365-1376;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038497 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483841--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maeva Leitwein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maeva%2BLeitwein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leitwein%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaeva%2BLeitwein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5172-0970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maeva Leitwein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maeva.leitwein.pro@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maeva.leitwein.pro@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bruno Guinand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Département Biologie-Ecologie, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BGuinand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guinand%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BGuinand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6934-1677" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bruno Guinand</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juliette Pouzadoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliette%2BPouzadoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pouzadoux%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliette%2BPouzadoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Erick Desmarais </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erick%2BDesmarais%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Desmarais%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErick%2BDesmarais%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patrick Berrebi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BBerrebi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berrebi%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BBerrebi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9026-4987" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Berrebi</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Pierre-Alexandre Gagnaire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut des Sciences de l’Evolution, Unité Mixte de Recherche 5554 CNRS-IRD-EPHE, Université de Montpellier, 34095 Montpellier Cedex 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre-Alexandre%2BGagnaire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gagnaire%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre-Alexandre%2BGagnaire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1908-3235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre-Alexandre Gagnaire</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1365&amp;rft.epage=1376&amp;rft.atitle=A%20Dense%20Brown%20Trout%20%28Salmo%20trutta%29%20Linkage%20Map%20Reveals%20Recent%20Chromosomal%20Rearrangements%20in%20the%20Salmo%20Genus%20and%20the%20Impact%20of%20Selection%20on%20Linked%20Neutral%20Diversity&amp;rft.volume=7&amp;rft.issue=4&amp;rft.date=2017-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Leitwein%2C+Maeva&amp;rft.au=Guinand%2C+Bruno&amp;rft.au=Pouzadoux%2C+Juliette&amp;rft.au=Desmarais%2C+Erick&amp;rft.au=Berrebi%2C+Patrick&amp;rft.au=Gagnaire%2C+Pierre-Alexandre"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483841/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483841/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483841/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483841/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483841/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483841/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483841/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483841/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483841/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483841/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483841/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483841/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/4/1365" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/4/1365.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/4/1365.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/4/1365.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/4/1365.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
