<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/4/1301" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/4/1301.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/4/1301.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/4/1301.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/4/1301.atom" />
<meta name="HW.pisa" content="ggg;7/4/1301" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi" />
<meta name="DC.Identifier" content="10.1534/g3.116.038448" />
<meta name="DC.Date" content="2017-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Dupont and Cox. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Lateral gene transfer (LGT, also known as horizontal gene transfer), an atypical mechanism of transferring genes between species, has almost become the default explanation for genes that display an unexpected composition or phylogeny. Numerous methods of detecting LGT events all rely on two fundamental strategies: primary structure composition or gene tree/species tree comparisons. Discouragingly, the results of these different approaches rarely coincide. With the wealth of genome data now available, detection of laterally transferred genes is increasingly being attempted in large uncurated eukaryotic datasets. However, detection methods depend greatly on the quality of the underlying genomic data, which are typically complex for eukaryotes. Furthermore, given the automated nature of genomic data collection, it is typically impractical to manually verify all protein or gene models, orthology predictions, and multiple sequence alignments, requiring researchers to accept a substantial margin of error in their datasets. Using a test case comprising plant-associated genomes across the fungal kingdom, this study reveals that composition- and phylogeny-based methods have little statistical power to detect laterally transferred genes. In particular, phylogenetic methods reveal extreme levels of topological variation in fungal gene trees, the vast majority of which show departures from the canonical species tree. Therefore, it is inherently challenging to detect LGT events in typical eukaryotic genomes. This finding is in striking contrast to the large number of claims for laterally transferred genes in eukaryotic species that routinely appear in the literature, and questions how many of these proposed examples are statistically well supported." />
<meta name="DC.Contributor" content="Pierre-Yves Dupont" />
<meta name="DC.Contributor" content="Murray P. Cox" />
<meta name="article:published_time" content="2017-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Lateral gene transfer (LGT, also known as horizontal gene transfer), an atypical mechanism of transferring genes between species, has almost become the default explanation for genes that display an unexpected composition or phylogeny. Numerous methods of detecting LGT events all rely on two fundamental strategies: primary structure composition or gene tree/species tree comparisons. Discouragingly, the results of these different approaches rarely coincide. With the wealth of genome data now available, detection of laterally transferred genes is increasingly being attempted in large uncurated eukaryotic datasets. However, detection methods depend greatly on the quality of the underlying genomic data, which are typically complex for eukaryotes. Furthermore, given the automated nature of genomic data collection, it is typically impractical to manually verify all protein or gene models, orthology predictions, and multiple sequence alignments, requiring researchers to accept a substantial margin of error in their datasets. Using a test case comprising plant-associated genomes across the fungal kingdom, this study reveals that composition- and phylogeny-based methods have little statistical power to detect laterally transferred genes. In particular, phylogenetic methods reveal extreme levels of topological variation in fungal gene trees, the vast majority of which show departures from the canonical species tree. Therefore, it is inherently challenging to detect LGT events in typical eukaryotic genomes. This finding is in striking contrast to the large number of claims for laterally transferred genes in eukaryotic species that routinely appear in the literature, and questions how many of these proposed examples are statistically well supported.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/04/01" />
<meta name="citation_mjid" content="ggg;7/4/1301" />
<meta name="citation_id" content="7/4/1301" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/4/1301" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/4/1301.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/4/1301.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/4/1301.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038448" />
<meta name="citation_pmid" content="28235827" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1301" />
<meta name="citation_lastpage" content="1314" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Pierre-Yves Dupont" />
<meta name="citation_author_institution" content="Statistics and Bioinformatics Group, Institute of Fundamental Sciences, Massey University, Palmerston North 4442, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7384-5181" />
<meta name="citation_author" content="Murray P. Cox" />
<meta name="citation_author_email" content="m.p.cox@massey.ac.nz" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1936-0236" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. S. Abby;citation_author=E. Tannier;citation_author=M. Gouy;citation_author=V. Daubin;citation_title=Detecting lateral gene transfers by statistical reconciliation of phylogenetic forests.;citation_pages=324-324;citation_volume=11;citation_year=2010;citation_pmid=20550700;citation_doi=10.1186/1471-2105-11-324" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=U. C. Alsmark;citation_author=T. Sicheritz-Ponten;citation_author=P. G. Foster;citation_author=R. P. Hirt;citation_author=T. M. Embley;citation_title=Horizontal gene transfer in eukaryotic parasites: a case study of Entamoeba histolytica and Trichomonas vaginalis.;citation_pages=489-500;citation_volume=532;citation_year=2009;citation_pmid=19271203;citation_doi=10.1007/978-1-60327-853-9_28" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=K. V. Ambrose;citation_author=A. M. Koppenhofer;citation_author=F. C. Belanger;citation_title=Horizontal gene transfer of a bacterial insect toxin gene into the Epichloë fungal symbionts of grasses.;citation_pages=5562;citation_volume=4;citation_year=2014;citation_pmid=24990771;citation_doi=10.1038/srep05562" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Amselem;citation_author=C. A. Cuomo;citation_author=J. A. van Kan;citation_author=M. Viaud;citation_author=E. P. Benito;citation_title=Genomic analysis of the necrotrophic fungal pathogens Sclerotinia sclerotiorum and Botrytis cinerea.;citation_pages=e1002230-e1002230;citation_volume=7;citation_year=2011;citation_issue=8;citation_pmid=21876677;citation_doi=10.1371/journal.pgen.1002230" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. S. Bansal;citation_author=E. J. Alm;citation_author=M. Kellis;citation_title=Efficient algorithms for the reconciliation problem with gene duplication, horizontal transfer and loss.;citation_volume=28;citation_year=2012;citation_issue=12;citation_pmid=22689773;citation_doi=10.1093/bioinformatics/bts225" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=R. G. Beiko;citation_author=M. A. Ragan;citation_title=Untangling hybrid phylogenetic signals: horizontal gene transfer and artifacts of phylogenetic reconstruction.;citation_pages=241-256;citation_volume=532;citation_year=2009;citation_pmid=19271189;citation_doi=10.1007/978-1-60327-853-9_14" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=D. Berry;citation_author=J. E. Takach;citation_author=C. L. Schardl;citation_author=N. D. Charlton;citation_author=B. Scott;citation_title=Disparate independent genetic events disrupt the secondary metabolism gene perA in certain symbiotic Epichloë species.;citation_volume=81;citation_year=2015;citation_issue=8;citation_pmid=25681180;citation_doi=10.1128/AEM.03721-14" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. R. Brown;citation_title=Ancient horizontal gene transfer.;citation_pages=121-132;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12560809;citation_doi=10.1038/nrg1000" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=M. C. Brundrett;citation_title=Coevolution of roots and mycorrhizas of land plants.;citation_pages=275-304;citation_volume=154;citation_year=2002;citation_issue=2;citation_doi=10.1046/j.1469-8137.2002.00397.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutierrez;citation_author=J. M. Silla-Martinez;citation_author=T. Gabaldon;citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_volume=25;citation_year=2009;citation_issue=15;citation_pmid=19505945;citation_doi=10.1093/bioinformatics/btp348" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=K. Cheeseman;citation_author=J. Ropars;citation_author=P. Renault;citation_author=J. Dupont;citation_author=J. Gouzy;citation_title=Multiple recent horizontal transfers of a large genomic region in cheese making fungi.;citation_pages=2876;citation_volume=5;citation_year=2014;citation_pmid=24407037" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=K. Chen;citation_author=D. Durand;citation_author=M. Farach-Colton;citation_title=NOTUNG: a program for dating gene duplications and optimizing gene family trees.;citation_pages=429-447;citation_volume=7;citation_year=2000;citation_issue=3–4;citation_pmid=11108472;citation_doi=10.1089/106652700750050871" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=M. J. Christensen;citation_author=R. J. Bennett;citation_author=H. A. Ansari;citation_author=H. Koga;citation_author=R. D. Johnson;citation_title=Epichloe endophytes grow by intercalary hyphal extension in elongating grass leaves.;citation_pages=84-93;citation_volume=45;citation_year=2008;citation_issue=2;citation_pmid=17919950;citation_doi=10.1016/j.fgb.2007.07.013" />
<meta name="citation_reference" content="citation_journal_title=IEEET Pattern Anal;citation_author=D. Comaniciu;citation_author=P. Meer;citation_title=Mean shift: a robust approach toward feature space analysis.;citation_pages=603-619;citation_volume=24;citation_year=2002;citation_issue=5;citation_doi=10.1109/34.1000236" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. J. Condon;citation_author=Y. Leng;citation_author=D. Wu;citation_author=K. E. Bushley;citation_author=R. A. Ohm;citation_title=Comparative genome structure, secondary metabolite, and effector coding capacity across Cochliobolus pathogens.;citation_pages=e1003233;citation_volume=9;citation_year=2013;citation_issue=1;citation_pmid=23357949;citation_doi=10.1371/journal.pgen.1003233" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=D. Cortez;citation_author=L. Delaye;citation_author=A. Lazcano;citation_author=A. Becerra;citation_title=Composition-based methods to identify horizontal gene transfer.;citation_pages=215-225;citation_volume=532;citation_year=2009;citation_pmid=19271187;citation_doi=10.1007/978-1-60327-853-9_12" />
<meta name="citation_reference" content="citation_journal_title=In silico biology;citation_journal_abbrev=In Silico Biol;citation_author=D. Q. Cortez;citation_author=A. Lazcano;citation_author=A. Becerra;citation_title=Comparative analysis of methodologies for the detection of horizontally transferred genes: a reassessment of first-order Markov models.;citation_pages=581-592;citation_volume=5;citation_year=2005;citation_issue=5–6;citation_pmid=16610135" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. A. Cuomo;citation_author=U. Guldener;citation_author=J. R. Xu;citation_author=F. Trail;citation_author=B. G. Turgeon;citation_title=The Fusarium graminearum Genome Reveals a Link Between Localized Polymorphism and Pathogen Specialization;citation_pages=1400-1402;citation_volume=317;citation_year=2007;citation_issue=5843;citation_pmid=17823352;citation_doi=10.1126/science.1143708" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=E. G. Danchin;citation_title=Lateral gene transfer in eukaryotes: tip of the iceberg or of the ice cube?;citation_pages=101;citation_volume=14;citation_year=2016;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. A. Dean;citation_author=N. J. Talbot;citation_author=D. J. Ebbole;citation_author=M. L. Farman;citation_author=T. K. Mitchell;citation_title=The genome sequence of the rice blast fungus Magnaporthe grisea.;citation_pages=980-986;citation_volume=434;citation_year=2005;citation_issue=7036;citation_pmid=15846337;citation_doi=10.1038/nature03449" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. J. de Wit;citation_author=A. van der Burgt;citation_author=B. Okmen;citation_author=I. Stergiopoulos;citation_author=K. A. Abd-Elsalam;citation_title=The Genomes of the Fungal Plant Pathogens Cladosporium fulvum and Dothistroma septosporum Reveal Adaptation to Different Hosts and Lifestyles But Also Signatures of Common Ancestry.;citation_pages=e1003088-e1003088;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23209441;citation_doi=10.1371/journal.pgen.1003088" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. Dore;citation_author=M. Perraud;citation_author=C. Dieryckx;citation_author=A. Kohler;citation_author=E. Morin;citation_title=Comparative genomics, proteomics and transcriptomics give new insight into the exoproteome of the basidiomycete Hebeloma cylindrosporum and its involvement in ectomycorrhizal symbiosis.;citation_pages=1169-1187;citation_volume=208;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=P. Y. Dupont;citation_author=C. J. Eaton;citation_author=J. J. Wargent;citation_author=S. Fechtner;citation_author=P. Solomon;citation_title=Fungal endophyte infection of ryegrass reprograms host metabolism and alters development.;citation_pages=1227-1240;citation_volume=208;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Microbe Interact.;citation_author=C. J. Eaton;citation_author=P. Y. Dupont;citation_author=P. Solomon;citation_author=W. Clayton;citation_author=B. Scott;citation_title=A core gene set describes the molecular basis of mutualism and antagonism in Epichloë spp.;citation_pages=218-231;citation_volume=28;citation_year=2015;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=I. Ebersberger;citation_author=R. de Matos Simoes;citation_author=A. Kupczok;citation_author=M. Gube;citation_author=E. Kothe;citation_title=A consistent phylogenetic backbone for the fungi.;citation_volume=29;citation_year=2012;citation_issue=5;citation_pmid=22114356;citation_doi=10.1093/molbev/msr285" />
<meta name="citation_reference" content="citation_author=M. Ester;citation_author=H.-P. Kriegel;citation_author=S. Jörg;citation_author=X. Xu;citation_pages=226-231;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=A. Firrincieli;citation_author=R. Otillar;citation_author=A. Salamov;citation_author=J. Schmutz;citation_author=Z. Khan;citation_title=Genome sequence of the plant growth promoting endophytic yeast Rhodotorula graminis WP1.;citation_pages=978;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=D. A. Fitzpatrick;citation_title=Horizontal gene transfer in fungi.;citation_volume=329;citation_year=2012;citation_issue=1;citation_pmid=22112233;citation_doi=10.1111/j.1574-6968.2011.02465.x" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. A. Fitzpatrick;citation_author=M. E. Logue;citation_author=J. E. Stajich;citation_author=G. Butler;citation_title=A fungal phylogeny based on 42 complete genomes derived from supertree and combined gene analysis.;citation_pages=99-99;citation_volume=6;citation_year=2006;citation_pmid=17121679;citation_doi=10.1186/1471-2148-6-99" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. A. Fitzpatrick;citation_author=M. E. Logue;citation_author=G. Butler;citation_title=Evidence of recent interkingdom horizontal gene transfer between bacteria and Candida parapsilosis.;citation_pages=181-181;citation_volume=8;citation_year=2008;citation_issue=1;citation_pmid=18577206;citation_doi=10.1186/1471-2148-8-181" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Floudas;citation_author=M. Binder;citation_author=R. Riley;citation_author=K. Barry;citation_author=R. A. Blanchette;citation_title=The Paleozoic Origin of Enzymatic Lignin Decomposition Reconstructed from 31 Fungal Genomes;citation_pages=1715-1719;citation_volume=336;citation_year=2012;citation_issue=6089;citation_pmid=22745431;citation_doi=10.1126/science.1221748;citation_georef=2012075372" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=E. Y. Furuya;citation_author=F. D. Lowy;citation_title=Antimicrobial-resistant bacteria in the community setting.;citation_pages=36-45;citation_volume=4;citation_year=2006;citation_issue=1;citation_pmid=16357859;citation_doi=10.1038/nrmicro1325" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=N. Galtier;citation_title=A model of horizontal gene transfer and the bacterial phylogeny problem.;citation_volume=56;citation_year=2007;citation_issue=4;citation_pmid=17661231;citation_doi=10.1080/10635150701546231" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Garcia-Vallve;citation_author=A. Romeu;citation_author=J. Palau;citation_title=Horizontal Gene Transfer in Bacterial and Archaeal Complete Genomes;citation_pages=1719-1725;citation_volume=10;citation_year=2000;citation_issue=11;citation_pmid=11076857;citation_doi=10.1101/gr.130000" />
<meta name="citation_reference" content="citation_journal_title=Fungal Biol.;citation_author=R. Gazis;citation_author=A. Kuo;citation_author=R. Riley;citation_author=K. LaButti;citation_author=A. Lipzen;citation_title=The genome of Xylona heveae provides a window into fungal endophytism.;citation_pages=26-42;citation_volume=120;citation_year=2016;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=J. P. Gogarten;citation_author=J. P. Townsend;citation_title=Horizontal gene transfer, genome innovation and evolution.;citation_pages=679-687;citation_volume=3;citation_year=2005;citation_issue=9;citation_pmid=16138096;citation_doi=10.1038/nrmicro1204" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=G. A. Grelet;citation_author=D. Johnson;citation_author=E. Paterson;citation_author=I. C. Anderson;citation_author=I. J. Alexander;citation_title=Reciprocal carbon and nitrogen transfer between an ericaceous dwarf shrub and fungi isolated from Piceirhiza bicolorata ectomycorrhizas.;citation_pages=359-366;citation_volume=182;citation_year=2009;citation_issue=2;citation_doi=10.1111/j.1469-8137.2009.02813.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. V. Grigoriev;citation_author=R. Nikitin;citation_author=S. Haridas;citation_author=A. Kuo;citation_author=R. Ohm;citation_title=MycoCosm portal: gearing up for 1000 fungal genomes.;citation_volume=42;citation_year=2014;citation_issue=Database issue;citation_pmid=24297253;citation_doi=10.1093/nar/gkt1183" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=J. F. Dufayard;citation_author=V. Lefort;citation_author=M. Anisimova;citation_author=W. Hordijk;citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.;citation_volume=59;citation_year=2010;citation_issue=3;citation_pmid=20525638;citation_doi=10.1093/sysbio/syq010" />
<meta name="citation_reference" content="citation_journal_title=Vet. Pathol.;citation_author=C. Gyles;citation_author=P. Boerlin;citation_title=Horizontally transferred genetic elements and their role in pathogenesis of bacterial disease.;citation_volume=51;citation_year=2014;citation_issue=2;citation_pmid=24318976;citation_doi=10.1177/0300985813511131" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. K. Hane;citation_author=R. G. Lowe;citation_author=P. S. Solomon;citation_author=K. C. Tan;citation_author=C. L. Schoch;citation_title=Dothideomycete plant interactions illuminated by genome sequencing and EST analysis of the wheat pathogen Stagonospora nodorum.;citation_volume=19;citation_year=2007;citation_issue=11;citation_pmid=18024570;citation_doi=10.1105/tpc.107.052829" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. Huang;citation_author=W. Si;citation_author=Q. Deng;citation_author=P. Li;citation_author=S. Yang;citation_title=Rapid evolution of avirulence genes in rice blast fungus Magnaporthe oryzae.;citation_pages=45;citation_volume=15;citation_year=2014;citation_pmid=24725999" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Huerta-Cepas;citation_author=H. Dopazo;citation_author=J. Dopazo;citation_author=T. Gabaldon;citation_title=The human phylome.;citation_pages=R109-R109;citation_volume=8;citation_year=2007;citation_issue=6;citation_pmid=17567924;citation_doi=10.1186/gb-2007-8-8-109" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. Huerta-Cepas;citation_author=J. Dopazo;citation_author=T. Gabaldon;citation_title=ETE: a python Environment for Tree Exploration.;citation_pages=24-24;citation_volume=11;citation_year=2010;citation_pmid=20070885;citation_doi=10.1186/1471-2105-11-24" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Jacox;citation_author=C. Chauve;citation_author=G. J. Szöllősi;citation_author=Y. Ponty;citation_author=C. Scornavacca;citation_title=ecceTERA: comprehensive gene tree-species tree reconciliation using parsimony.;citation_volume=32;citation_year=2016;citation_issue=13;citation_pmid=27153713;citation_doi=10.1093/bioinformatics/btw105" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Y. James;citation_author=F. Kauff;citation_author=C. L. Schoch;citation_author=P. B. Matheny;citation_author=V. Hofstetter;citation_title=Reconstructing the early evolution of Fungi using a six-gene phylogeny.;citation_pages=818-822;citation_volume=443;citation_year=2006;citation_issue=7113;citation_pmid=17051209;citation_doi=10.1038/nature05110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. S. Jaron;citation_author=J. C. Moravec;citation_author=N. Martínková;citation_title=SigHunt: horizontal gene transfer finder optimized for eukaryotic genomes.;citation_volume=30;citation_year=2014;citation_issue=8;citation_pmid=24371153;citation_doi=10.1093/bioinformatics/btt727" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_issue=4;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Katoh;citation_author=K. Misawa;citation_author=K. Kuma;citation_author=T. Miyata;citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;citation_pages=3059-3066;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136088;citation_doi=10.1093/nar/gkf436" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=E. Kay;citation_author=T. M. Vogel;citation_author=F. Bertolla;citation_author=R. Nalin;citation_author=P. Simonet;citation_title=In Situ Transfer of Antibiotic Resistance Genes from Transgenic (Transplastomic) Tobacco Plants to Bacteria;citation_pages=3345-3351;citation_volume=68;citation_year=2002;citation_issue=7;citation_pmid=12089013;citation_doi=10.1128/AEM.68.7.3345-3351.2002" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. J. Keeling;citation_author=J. D. Palmer;citation_title=Horizontal gene transfer in eukaryotic evolution.;citation_pages=605-618;citation_volume=9;citation_year=2008;citation_issue=8;citation_pmid=18591983;citation_doi=10.1038/nrg2386" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=N. Khaldi;citation_author=J. Collemare;citation_author=M.-H. Lebrun;citation_author=K. H. Wolfe;citation_title=Evidence for horizontal transfer of a secondary metabolite gene cluster between fungi.;citation_pages=1-10;citation_volume=9;citation_year=2008;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=S. J. Klosterman;citation_author=K. V. Subbarao;citation_author=S. Kang;citation_author=P. Veronese;citation_author=S. E. Gold;citation_title=Comparative genomics yields insights into niche adaptation of plant vascular wilt pathogens.;citation_pages=e1002137-e1002137;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21829347;citation_doi=10.1371/journal.ppat.1002137" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Kohler;citation_author=A. Kuo;citation_author=L. G. Nagy;citation_author=E. Morin;citation_author=K. W. Barry;citation_title=Convergent losses of decay mechanisms and rapid turnover of symbiosis genes in mycorrhizal mutualists.;citation_pages=410-415;citation_volume=47;citation_year=2015;citation_issue=4;citation_pmid=25706625;citation_doi=10.1038/ng.3223" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=L. B. Koski;citation_author=R. A. Morton;citation_author=G. B. Golding;citation_title=Codon Bias and Base Composition Are Poor Indicators of Horizontally Transferred Genes;citation_pages=404-412;citation_volume=18;citation_year=2001;citation_issue=3;citation_pmid=11230541;citation_doi=10.1093/oxfordjournals.molbev.a003816" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. Ku;citation_author=S. Nelson-Sathi;citation_author=M. Roettger;citation_author=F. L. Sousa;citation_author=P. J. Lockhart;citation_title=Endosymbiotic origin and differential loss of eukaryotic genes.;citation_pages=427-432;citation_volume=524;citation_year=2015;citation_issue=7566;citation_pmid=26287458;citation_doi=10.1038/nature14963" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. G. Lawrence;citation_author=H. Ochman;citation_title=Amelioration of bacterial genomes: rates of change and exchange.;citation_pages=383-397;citation_volume=44;citation_year=1997;citation_issue=4;citation_pmid=9089078;citation_doi=10.1007/PL00006158" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Q. Le;citation_author=O. Gascuel;citation_title=An improved general amino acid replacement matrix.;citation_volume=25;citation_year=2008;citation_issue=7;citation_pmid=18367465;citation_doi=10.1093/molbev/msn067" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Le Quere;citation_author=K. A. Eriksen;citation_author=B. Rajashekar;citation_author=A. Schutzendubel;citation_author=B. Canback;citation_title=Screening for rapidly evolving genes in the ectomycorrhizal fungus Paxillus involutus using cDNA microarrays.;citation_pages=535-550;citation_volume=15;citation_year=2006;citation_issue=2;citation_pmid=16448419;citation_doi=10.1111/j.1365-294X.2005.02796.x" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=H. Liu;citation_author=Y. Fu;citation_author=B. Li;citation_author=X. Yu;citation_author=J. Xie;citation_title=Widespread horizontal gene transfer from circular single-stranded DNA viruses to eukaryotic genomes.;citation_pages=276-276;citation_volume=11;citation_year=2011;citation_pmid=21943216;citation_doi=10.1186/1471-2148-11-276" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=Y. J. Liu;citation_author=M. C. Hodson;citation_author=B. D. Hall;citation_title=Loss of the flagellum happened only once in the fungal lineage: phylogenetic structure of kingdom Fungi inferred from RNA polymerase II subunit genes.;citation_pages=74-74;citation_volume=6;citation_year=2006;citation_pmid=17010206;citation_doi=10.1186/1471-2148-6-74" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L.-J. Ma;citation_author=H. C. van der Does;citation_author=K. A. Borkovich;citation_author=J. J. Coleman;citation_author=M.-J. Daboussi;citation_title=Comparative genomics reveals mobile pathogenicity chromosomes in Fusarium.;citation_pages=367-373;citation_volume=464;citation_year=2010;citation_issue=7287;citation_pmid=20237561;citation_doi=10.1038/nature08850" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=W. P. Maddison;citation_title=Gene trees in species trees.;citation_volume=46;citation_year=1997;citation_issue=3;citation_doi=10.1093/sysbio/46.3.523" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. V. Mallet;citation_author=J. Becq;citation_author=P. Deschavanne;citation_title=Whole genome evaluation of horizontal transfers in the pathogenic fungus Aspergillus fumigatus.;citation_pages=171-171;citation_volume=11;citation_year=2010;citation_pmid=20226043;citation_doi=10.1186/1471-2164-11-171" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Marcet-Houben;citation_author=T. Gabaldón;citation_title=Acquisition of prokaryotic genes by fungal genomes.;citation_pages=5-8;citation_volume=26;citation_year=2010;citation_issue=1;citation_pmid=19969385;citation_doi=10.1016/j.tig.2009.11.007" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Martin;citation_author=A. Aerts;citation_author=D. Ahren;citation_author=A. Brun;citation_author=E. G. Danchin;citation_title=The genome of Laccaria bicolor provides insights into mycorrhizal symbiosis.;citation_pages=88-92;citation_volume=452;citation_year=2008;citation_issue=7183;citation_pmid=18322534;citation_doi=10.1038/nature06556" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Martin;citation_author=A. Kohler;citation_author=C. Murat;citation_author=R. Balestrini;citation_author=P. M. Coutinho;citation_title=Perigord black truffle genome uncovers evolutionary origins and mechanisms of symbiosis.;citation_pages=1033-1038;citation_volume=464;citation_year=2010;citation_issue=7291;citation_pmid=20348908;citation_doi=10.1038/nature08867" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. L. Martínez;citation_title=Antibiotics and Antibiotic Resistance Genes in Natural Environments;citation_pages=365-367;citation_volume=321;citation_year=2008;citation_issue=5887;citation_pmid=18635792;citation_doi=10.1126/science.1159483" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Menigaud;citation_author=L. Mallet;citation_author=G. Picord;citation_author=C. Churlaud;citation_author=A. Borrel;citation_title=GOHTAM: a website for ‘genomic origin of horizontal transfers, alignment and metagenomics.’;citation_volume=28;citation_year=2012;citation_issue=9;citation_pmid=22426345;citation_doi=10.1093/bioinformatics/bts118" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. H. Mesarich;citation_author=J. K. Bowen;citation_author=C. Hamiaux;citation_author=M. D. Templeton;citation_title=Repeat-containing protein effectors of plant-associated organisms.;citation_pages=872;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Monier;citation_author=A. Pagarete;citation_author=C. de Vargas;citation_author=M. J. Allen;citation_author=B. Read;citation_title=Horizontal gene transfer of an entire metabolic pathway between a eukaryotic alga and its DNA virus.;citation_volume=19;citation_year=2009;citation_issue=8;citation_pmid=19451591;citation_doi=10.1101/gr.091686.109" />
<meta name="citation_reference" content="citation_title=Natural Products;citation_title=Evolutionary mechanisms involved in development of fungal secondary metabolite gene clusters;citation_author=G. G. Moore;citation_author=J. Collemare;citation_author=M.-H. Lebrun;citation_author=R. E. Bradshaw;citation_pages=341-356;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Moran;citation_author=D. Fredman;citation_author=P. Szczesny;citation_author=M. Grynberg;citation_author=U. Technau;citation_title=Recurrent horizontal transfer of bacterial toxin genes to eukaryotes.;citation_volume=29;citation_year=2012;citation_issue=9;citation_pmid=22411854;citation_doi=10.1093/molbev/mss089" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Morel;citation_author=L. Sterck;citation_author=D. Swennen;citation_author=M. Marcet-Houben;citation_author=D. Onesime;citation_title=Differential gene retention as an evolutionary mechanism to generate biodiversity and adaptation in yeasts.;citation_pages=11571;citation_volume=5;citation_year=2015;citation_pmid=26108467;citation_doi=10.1038/srep11571" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Nakamura;citation_author=T. Itoh;citation_author=H. Matsuda;citation_author=T. Gojobori;citation_title=Biased biological functions of horizontally transferred genes in prokaryotic genomes.;citation_pages=760-766;citation_volume=36;citation_year=2004;citation_issue=7;citation_pmid=15208628;citation_doi=10.1038/ng1381" />
<meta name="citation_reference" content="citation_author=T. H. Nguyen;citation_author=J.-P. Doyon;citation_author=S. Pointet;citation_author=A.-M. A. Chifolleau;citation_author=V. Ranwez;citation_pages=123-134;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Nikolaidis;citation_author=N. Doran;citation_author=D. J. Cosgrove;citation_title=Plant expansins in bacteria and fungi: evolution by horizontal gene transfer and independent domain fusion.;citation_volume=31;citation_year=2014;citation_issue=2;citation_pmid=24150040;citation_doi=10.1093/molbev/mst206" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. J. O’Connell;citation_author=M. R. Thon;citation_author=S. Hacquard;citation_author=S. G. Amyotte;citation_author=J. Kleemann;citation_title=Lifestyle transitions in plant pathogenic Colletotrichum fungi deciphered by genome and transcriptome analyses.;citation_pages=1060-1065;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22885923;citation_doi=10.1038/ng.2372" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ochman;citation_author=J. G. Lawrence;citation_author=E. A. Groisman;citation_title=Lateral gene transfer and the nature of bacterial innovation.;citation_pages=299-304;citation_volume=405;citation_year=2000;citation_issue=6784;citation_pmid=10830951;citation_doi=10.1038/35012500" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=R. A. Ohm;citation_author=N. Feau;citation_author=B. Henrissat;citation_author=C. L. Schoch;citation_author=B. A. Horwitz;citation_title=Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi.;citation_pages=e1003037;citation_volume=8;citation_year=2012;citation_issue=12;citation_pmid=23236275;citation_doi=10.1371/journal.ppat.1003037" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=Y. Ovadia;citation_author=D. Fielder;citation_author=C. Conow;citation_author=R. Libeskind-Hadas;citation_title=The co phylogeny reconstruction problem is NP-complete.;citation_pages=59-65;citation_volume=18;citation_year=2011;citation_issue=1;citation_pmid=20715926;citation_doi=10.1089/cmb.2009.0240" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=P. Pamilo;citation_author=M. Nei;citation_title=Relationships between gene trees and species trees;citation_pages=568-583;citation_volume=5;citation_year=1988;citation_issue=5;citation_pmid=3193878" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=F. Pedregosa;citation_author=G. Varoquaux;citation_author=A. Gramfort;citation_author=V. Michel;citation_author=B. Thirion;citation_title=Scikit-learn: machine learning in Python.;citation_pages=2825-2830;citation_volume=12;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M. Poptsova;citation_title=Testing phylogenetic methods to identify horizontal gene transfer.;citation_pages=227-240;citation_volume=532;citation_year=2009;citation_pmid=19271188" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=M. S. Poptsova;citation_author=J. P. Gogarten;citation_title=The power of phylogenetic approaches to detect horizontally transferred genes.;citation_pages=45-45;citation_volume=7;citation_year=2007;citation_pmid=17376230;citation_doi=10.1186/1471-2148-7-45" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=M. A. Ragan;citation_title=On surrogate methods for detecting lateral gene transfer.;citation_volume=201;citation_year=2001;citation_issue=2;citation_pmid=11470360;citation_doi=10.1111/j.1574-6968.2001.tb10755.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. Redondo-Nieto;citation_author=M. Barret;citation_author=J. P. Morrisey;citation_author=K. Germaine;citation_author=F. Martinez-Granero;citation_title=Genome Sequence of the Biocontrol Strain Pseudomonas fluorescens F113;citation_pages=1273-1274;citation_volume=194;citation_year=2012;citation_issue=5;citation_pmid=22328765;citation_doi=10.1128/JB.06601-11" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=T. A. Richards;citation_author=J. B. Dacks;citation_author=S. A. Campbell;citation_author=J. L. Blanchard;citation_author=P. G. Foster;citation_title=Evolutionary Origins of the Eukaryotic Shikimate Pathway: Gene Fusions, Horizontal Gene Transfer, and Endosymbiotic Replacements;citation_pages=1517-1531;citation_volume=5;citation_year=2006;citation_issue=9;citation_pmid=16963634;citation_doi=10.1128/EC.00106-06" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=T. A. Richards;citation_author=D. M. Soanes;citation_author=P. G. Foster;citation_author=G. Leonard;citation_author=C. R. Thornton;citation_title=Phylogenomic analysis demonstrates a pattern of rare and ancient horizontal gene transfer between plants and fungi.;citation_volume=21;citation_year=2009;citation_issue=7;citation_pmid=19584142;citation_doi=10.1105/tpc.109.065805" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=D. F. Robinson;citation_author=L. R. Foulds;citation_title=Comparison of phylogenetic trees.;citation_pages=131-147;citation_volume=53;citation_year=1981;citation_issue=1–2;citation_doi=10.1016/0025-5564(81)90043-2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. Rogul;citation_author=Z. McGee;citation_author=R. Wittler;citation_author=S. Falkow;citation_title=Nucleic Acid Homologies of Selected Bacteria, L Forms, and Mycoplasma Species;citation_pages=1200-1204;citation_volume=90;citation_year=1965;citation_issue=5;citation_pmid=5848325" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=U. L. Rosewich;citation_author=H. C. Kistler;citation_title=ROLE OF HORIZONTAL GENE TRANSFER IN THE EVOLUTION OF FUNGI;citation_pages=325-363;citation_volume=38;citation_year=2000;citation_issue=1;citation_pmid=11701846;citation_doi=10.1146/annurev.phyto.38.1.325" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=T. Rouxel;citation_author=J. Grandaubert;citation_author=J. K. Hane;citation_author=C. Hoede;citation_author=A. P. van de Wouw;citation_title=Effector diversification within compartments of the Leptosphaeria maculans genome affected by Repeat-Induced Point mutations.;citation_pages=202-202;citation_volume=2;citation_year=2011;citation_pmid=21326234;citation_doi=10.1038/ncomms1189" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Salichos;citation_author=A. Rokas;citation_title=Evaluating ortholog prediction algorithms in a yeast model clade.;citation_pages=e18755-e18755;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21533202;citation_doi=10.1371/journal.pone.0018755" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. Salichos;citation_author=A. Rokas;citation_title=Inferring ancient divergences requires genes with strong phylogenetic signals.;citation_pages=327-331;citation_volume=497;citation_year=2013;citation_issue=7449;citation_pmid=23657258;citation_doi=10.1038/nature12130" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. W. Sayers;citation_author=T. Barrett;citation_author=D. A. Benson;citation_author=S. H. Bryant;citation_author=K. Canese;citation_title=Database resources of the national center for biotechnology information.;citation_volume=37;citation_year=2009;citation_issue=Database issue;citation_pmid=18940862;citation_doi=10.1093/nar/gkn741" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=C. L. Schardl;citation_title=Epichloe festucae and related mutualistic symbionts of grasses.;citation_pages=69-82;citation_volume=33;citation_year=2001;citation_issue=2;citation_pmid=11456460;citation_doi=10.1006/fgbi.2001.1275" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. L. Schardl;citation_author=C. A. Young;citation_author=U. Hesse;citation_author=S. G. Amyotte;citation_author=K. Andreeva;citation_title=Plant-symbiotic fungi as chemical engineers: multi-genome analysis of the Clavicipitaceae reveals dynamics of alkaloid loci.;citation_pages=e1003323;citation_volume=9;citation_year=2013;citation_issue=2;citation_pmid=23468653;citation_doi=10.1371/journal.pgen.1003323" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Schirawski;citation_author=G. Mannhaupt;citation_author=K. Munch;citation_author=T. Brefort;citation_author=K. Schipper;citation_title=Pathogenicity Determinants in Smut Fungi Revealed by Genome Comparison;citation_pages=1546-1548;citation_volume=330;citation_year=2010;citation_issue=6010;citation_pmid=21148393;citation_doi=10.1126/science.1195330" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. A. Shapiro;citation_title=Mutations caused by the insertion of genetic material into the galactose operon of Escherichia coli.;citation_pages=93-105;citation_volume=40;citation_year=1969;citation_issue=1;citation_pmid=4903362;citation_doi=10.1016/0022-2836(69)90298-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Phytopathol.;citation_author=D. Soanes;citation_author=T. A. Richards;citation_title=Horizontal gene transfer in eukaryotic plant pathogens.;citation_pages=583-614;citation_volume=52;citation_year=2014;citation_pmid=25090479;citation_doi=10.1146/annurev-phyto-102313-050127" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=S. M. Soucy;citation_author=J. Huang;citation_author=J. P. Gogarten;citation_title=Horizontal gene transfer: building the web of life.;citation_pages=472-482;citation_volume=16;citation_year=2015;citation_issue=8;citation_pmid=26184597;citation_doi=10.1038/nrg3962" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. D. Spanu;citation_author=J. C. Abbott;citation_author=J. Amselem;citation_author=T. A. Burgis;citation_author=D. M. Soanes;citation_title=Genome Expansion and Gene Loss in Powdery Mildew Fungi Reveal Tradeoffs in Extreme Parasitism;citation_pages=1543-1546;citation_volume=330;citation_year=2010;citation_issue=6010;citation_pmid=21148392;citation_doi=10.1126/science.1194573" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=G. F. Sprague;citation_title=Genetic exchange between kingdoms.;citation_pages=530-533;citation_volume=1;citation_year=1991;citation_issue=4;citation_pmid=1822285;citation_doi=10.1016/S0959-437X(05)80203-5" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=M. Staats;citation_author=J. A. van Kan;citation_title=Genome Update of Botrytis cinerea Strains B05.10 and T4;citation_pages=1413-1414;citation_volume=11;citation_year=2012;citation_issue=11;citation_pmid=23104368;citation_doi=10.1128/EC.00164-12" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Stolzer;citation_author=H. Lai;citation_author=M. Xu;citation_author=D. Sathaye;citation_author=B. Vernot;citation_title=Inferring duplications, losses, transfers and incomplete lineage sorting with nonbinary species trees.;citation_volume=28;citation_year=2012;citation_issue=18;citation_pmid=22962460;citation_doi=10.1093/bioinformatics/bts386" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=X. Sun;citation_author=Q. Yang;citation_author=X. Xia;citation_title=An improved implementation of effective number of codons (Nc).;citation_volume=30;citation_year=2013;citation_issue=1;citation_pmid=22915832;citation_doi=10.1093/molbev/mss201" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. Syvanen;citation_title=Horizontal Gene Flow: Evidence and Possible Consequences;citation_pages=237-261;citation_volume=28;citation_year=1994;citation_issue=1;citation_pmid=7893125;citation_doi=10.1146/annurev.ge.28.120194.001321" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. B Biol. Sci.;citation_author=G. J. Szöllősi;citation_author=A. A. Davin;citation_author=E. Tannier;citation_author=V. Daubin;citation_author=B. Boussau;citation_title=Genome-scale phylogenetic analysis finds extensive gene transfer among fungi.;citation_volume=370;citation_year=2015;citation_issue=1678;citation_pmid=26323765;citation_doi=10.1098/rstb.2014.0335" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. Tanaka;citation_author=M. J. Christensen;citation_author=D. Takemoto;citation_author=P. Park;citation_author=B. Scott;citation_title=Reactive oxygen species play a role in regulating a fungus-perennial ryegrass mutualistic interaction.;citation_volume=18;citation_year=2006;citation_issue=4;citation_pmid=16517760;citation_doi=10.1105/tpc.105.039263" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=E. Temporini;citation_author=H. Van Etten;citation_title=An analysis of the phylogenetic distribution of the pea pathogenicity genes of Nectria haematococca MPVI supports the hypothesis of their origin by horizontal transfer and uncovers a potentially new pathogen of garden pea: Neocosmospora boniensis.;citation_pages=29-36;citation_volume=46;citation_year=2004;citation_issue=1;citation_pmid=15118835" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=C. P. Terhorst;citation_author=J. T. Lennon;citation_author=J. A. Lau;citation_title=The relative importance of rapid evolution for plant-microbe interactions depends on ecological context.;citation_volume=281;citation_year=2014;citation_pmid=24789894;citation_doi=10.1098/rspb.2014.0028" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=C. Than;citation_author=D. Ruths;citation_author=H. Innan;citation_author=L. Nakhleh;citation_title=Confounding factors in HGT detection: statistical error, coalescent effects, and multiple solutions.;citation_pages=517-535;citation_volume=14;citation_year=2007;citation_issue=4;citation_pmid=17572027;citation_doi=10.1089/cmb.2007.A010" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=M. Toome;citation_author=R. A. Ohm;citation_author=R. W. Riley;citation_author=T. Y. James;citation_author=K. L. Lazarus;citation_title=Genome sequencing provides insight into the reproductive biology, nutritional mode and ploidy of the fern pathogen Mixia osmundae.;citation_pages=554-564;citation_volume=202;citation_year=2014;citation_issue=2;citation_doi=10.1111/nph.12653" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. M. Tyler;citation_author=S. Tripathy;citation_author=X. Zhang;citation_author=P. Dehal;citation_author=R. H. Jiang;citation_title=Phytophthora genome sequences uncover evolutionary origins and mechanisms of pathogenesis.;citation_pages=1261-1266;citation_volume=313;citation_year=2006;citation_issue=5791;citation_pmid=16946064;citation_doi=10.1126/science.1128796" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=J. W. Wägele;citation_author=C. Mayer;citation_title=Visualizing differences in phylogenetic information content of alignments and distinction of three classes of long-branch effects.;citation_pages=147-147;citation_volume=7;citation_year=2007;citation_pmid=17725833;citation_doi=10.1186/1471-2148-7-147" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=H. Wang;citation_author=Z. Xu;citation_author=L. Gao;citation_author=B. Hao;citation_title=A fungal phylogeny based on 82 complete genomes using the composition vector method.;citation_pages=195-195;citation_volume=9;citation_year=2009;citation_pmid=19664262;citation_doi=10.1186/1471-2148-9-195" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Weyenberg;citation_author=P. M. Huggins;citation_author=C. L. Schardl;citation_author=D. K. Howe;citation_author=R. Yoshida;citation_title=kdetrees: non-parametric estimation of phylogenetic tree distributions.;citation_volume=30;citation_year=2014;citation_issue=16;citation_pmid=24764459;citation_doi=10.1093/bioinformatics/btu258" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=C. Whidden;citation_author=N. Zeh;citation_author=R. G. Beiko;citation_title=Supertrees based on the subtree prune-and-regraft distance.;citation_volume=63;citation_year=2014;citation_issue=4;citation_pmid=24695589;citation_doi=10.1093/sysbio/syu023" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. H. Wisecaver;citation_author=J. C. Slot;citation_author=A. Rokas;citation_title=The evolution of fungal metabolic pathways.;citation_pages=e1004816;citation_volume=10;citation_year=2014;citation_issue=12;citation_pmid=25474404;citation_doi=10.1371/journal.pgen.1004816" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=O. Zhaxybayeva;citation_title=Detection and quantitative assessment of horizontal gene transfer.;citation_pages=195-213;citation_volume=532;citation_year=2009;citation_pmid=19271186;citation_doi=10.1007/978-1-60327-853-9_11" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="twitter:description" content="Lateral gene transfer (LGT, also known as horizontal gene transfer), an atypical mechanism of transferring genes between species, has almost become the default explanation for genes that display an unexpected composition or phylogeny. Numerous methods of detecting LGT events all rely on two fundamental strategies: primary structure composition or gene tree/species tree comparisons. Discouragingly, the results of these different approaches rarely coincide. With the wealth of genome data now available, detection of laterally transferred genes is increasingly being attempted in large uncurated eukaryotic datasets. However, detection methods depend greatly on the quality of the underlying genomic data, which are typically complex for eukaryotes. Furthermore, given the automated nature of genomic data collection, it is typically impractical to manually verify all protein or gene models, orthology predictions, and multiple sequence alignments, requiring researchers to accept a substantial margin of error in their datasets. Using a test case comprising plant-associated genomes across the fungal kingdom, this study reveals that composition- and phylogeny-based methods have little statistical power to detect laterally transferred genes. In particular, phylogenetic methods reveal extreme levels of topological variation in fungal gene trees, the vast majority of which show departures from the canonical species tree. Therefore, it is inherently challenging to detect LGT events in typical eukaryotic genomes. This finding is in striking contrast to the large number of claims for laterally transferred genes in eukaryotic species that routinely appear in the literature, and questions how many of these proposed examples are statistically well supported." />
<meta name="og-title" property="og:title" content="Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/4/1301" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Lateral gene transfer (LGT, also known as horizontal gene transfer), an atypical mechanism of transferring genes between species, has almost become the default explanation for genes that display an unexpected composition or phylogeny. Numerous methods of detecting LGT events all rely on two fundamental strategies: primary structure composition or gene tree/species tree comparisons. Discouragingly, the results of these different approaches rarely coincide. With the wealth of genome data now available, detection of laterally transferred genes is increasingly being attempted in large uncurated eukaryotic datasets. However, detection methods depend greatly on the quality of the underlying genomic data, which are typically complex for eukaryotes. Furthermore, given the automated nature of genomic data collection, it is typically impractical to manually verify all protein or gene models, orthology predictions, and multiple sequence alignments, requiring researchers to accept a substantial margin of error in their datasets. Using a test case comprising plant-associated genomes across the fungal kingdom, this study reveals that composition- and phylogeny-based methods have little statistical power to detect laterally transferred genes. In particular, phylogenetic methods reveal extreme levels of topological variation in fungal gene trees, the vast majority of which show departures from the canonical species tree. Therefore, it is inherently challenging to detect LGT events in typical eukaryotic genomes. This finding is in striking contrast to the large number of claims for laterally transferred genes in eukaryotic species that routinely appear in the literature, and questions how many of these proposed examples are statistically well supported." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
    <title>Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__6uPNahEPJnOkpurvaRxri2qBNHoapqvM8y1Wtdb7u_Q__sE1YcwwFB0K8aMYGFpNWEgt3sH5t9Ld9QAF34gSvqPY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"JQr6f4ruPRz_U8JYcClJZdI8j8Jr2efo0VV6xcsn3-w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483844","apath":"\/ggg\/7\/4\/1301.atom","pisa":"ggg;7\/4\/1301","ac":{"\/ggg\/7\/4\/1301.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/4\/1301.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1301"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1301"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/4\/1301":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L1Msiw5D0yWuFMCLg6RNJl06zPYmhJm6gKJoca7XRB4CLVTxdeUlfEuFbnHIgTws+eXPA6YGJjRgJVo+mShtRKm3fUiB6WTfPGWBpG\\\\\\\/7PoLgEtRHJwTpLZlIjTlO7hW9Tgfr8CV3lp0LEpAF+Aot\\\\\\\/xmROAE56lrwSgGHu1yQ8izSwegHp21Y2sx2uBba7pJ8q+dUyk6AvupP6IbxU\\\\\\\/k9\\\\\\\/Y0w66Y0tProecAJnPt+U1G0y38BBBTc3wCxwzUOS\\\\\\\/v72XEBATwA5\\\\\\\/HCEBBEbCcQugwwO9MxfbrNzaqXl3MYTui6eI5es2OphuFo5dliFZJ8UwRRgv3+bGlbQgs2Al5j4wcu5KE6DtSxnFa2Vk8+7tbpjK4tkFp\\\\\\\/\\\\\\\/8FKXW6reHrQRmWkJya0BDyA4LzYwu+UJSpxNGQzgBIRkAr+CQLXxC4kl6bYvWBcKwmhiwl5awRcK6kSkOq2uir5dBx8D1Z1bEOPpnf6LCiiKvQbykvp10U3VQdjuZQMoSbwndbSXwqIcVX7lObThYLxjyocjGoH0qga\\\\\\\/9Tn2bMjPufIKyXFxlCWoUThTHFgTLXByW8kHhHoeEiRR\\\\\\\/v2YUIma5Prg2tvZHHdI2uoKUXO3LKTMEJBHCIRHd5tVGfLV\\\\\\\/\\\\\\\/L6BwYen7VMFAb4z0F3RL8EbpbXXTM7wiSNe4pz0rb+HPwg90+26fT\\\\\\\/n9CNpIW63O1B7c+m2mgio8UhKiv9Rp5rqpOcy5ZyvPtjHVK\\\\\\\/pdhmY4K\\\\\\\/yteEi5EKmVRU6TfOWtdn8OpXrXpCvLr173c3PteJz3FN0VcapLouQVno0di+Izvt65AIjB7s8z0t11loHXravUh5Us2VBmTwsIbdvSS27\\\\\\\/Mhu8ZdPecVnNQ6\\\\\\\/kAiujOi9EdWOJkpA+6WiExjMqYvJr6LTfLbG1JG7DlUQ+SFK+tu0BcASle1qtzT7+eyjUUCIuleVNH5ARK1MuOhRYVgPnSB8BXLAYAnmVw3IDnVLpH1Pa7VpVSpkkH3Cb9N9mMzE7tiWJ0lsxVSsr\\\\\\\/STw2PgT6R0Fp5LdcTNmZC8i9X57nnMdiYK57zgQxMq7HMp0TEP1ksNTxgf9Lc2qN7mjZVm43FU3niMhVn7J5s4BsANung98y25Wnvj+5tiQQJNY1rBjUL6QFx6NZrv1duiazCveoNIPAlmGGucOShs6\\\\\\\/EbhEM3djwr+teBhGB7LbeUJBrBPJ\\\\\\\/KIYdPZ1vofp0NR4KqZ7jxlQsC0HKklHXGUXPQODWL5XYYKv1VmQRdZnVR+CppaNOzILU\\\\\\\/M6cttkY\\\\\\\/XzPrcS0IXQ\\\u0022,\\\u0022iv\\\u0022:\\\u00226YtIhwFH2cZGJCzpC9IFxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228c9af42844ea908043f9fbc61fb11d6669548a783788c41ffc8d811929887f19\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jqZ9UawOnhxbh42Fx1aSq+l768\\\\\\\/Vh7aWwOb0olxsDxWJ4V2ec96WeTkAzxPQG84ApQuPv\\\\\\\/lOmGamUIDisFcGTRUO+HG2dzFxVkOtkVuCUgDcHqveEeVBIR3HHT\\\\\\\/1nYG7VKVf2xT7E2p4DOHUpPXyI+J9tYY0+XTKdX0srZPBCiks13xOgqKF3T7IrJxz1qIhT56BHsJd9QtYANvFQylpdr7+xvYkXQDhhyeFdyNMeWfTinL3C7UiLkkGpzaSagDUolTQvKSctOVseFjLsqzk7uxfjZAke7BZxjVdStf4YGpSYsQV7BDrs6vqq2KgHnKpUAWUZzleiq3Q4a+uAwFpcnCTdwEL2OxKEPWkJvCxwpZFzPhhwIYZ4UrfGGjDRNbsUE9MZwQRqz0hxdRxpo\\\\\\\/h1buuV71ubjfq+UowZskN0ZcjvVvw6tAe1l+OV8bCr1yThWVgXRI4onp1DjPvt9zA2TrDwj0poby03CnzzOTtF8v4ncRY3e8brPIkplHujkGcrplJyxFIg+SQEMGawtONe8Q2Hj5sQyVXCMiHKQSpUsnAlikShxFKwNdM2u+kcpS7SplB9j1aqgda0Gz+foXPf2VJz1ox6AEfmc+SZkD6VYMtTpFQbS6BrMy10SZqnD+mWEu96XI759d0RumvAG\\\\\\\/ad+phR9u+7VAmfJKBXal7WfWb\\\\\\\/PCla5QTSuElMB9n4N36zjCIQrTul6gw58bcdRUNt8w5tDdMuhc7rbc\\\\\\\/6UudMRyJNurJNVI91FgY5oaB6eJ77vGEgiDRhiGx03\\\\\\\/\\\\\\\/g1tA4969A+R7VgxV0KV4t37KkbE8w5DjLyjmEMHvHKKq5w7zKxFmFrSYp6SidgulPxE9kdjq3QJep2Azaxc+6Dayml\\\\\\\/q3KEPPJagTIli2Uu1GwmZD6KBj03dshicxL1XQKkc4mp9N+zjRMKgxpzdGD0ZFiHS6uKZpEvAn4NywcPEyuP7V8YUjfUUF2kJeU4VGJwy45JS75B6Hzwulat2io6w\\\\\\\/bYH\\\\\\\/I6lbYiWuUXzoC\\\\\\\/FuvHKqcvl7zaiiTxZvbYdVK28McFem0rmsc5J\\\\\\\/AFQv1gUwt2L0pO8K2C81eKo3WeWWG+ieFE3NIVx\\\\\\\/WrJD5Cqc8oUF\\\\\\\/UDNUmLuVLOK\\\\\\\/h\\\\\\\/aLmqGPOm6x+yINqbn9v3zYDymq8AUu4yMdwxVIyp83KQHBx9NqsuyZXHbZeRotvL1K3K9QvuN5jIxHZjBvYwiFTjhoiNF\\\\\\\/I+Tb2i6979GX5SO2Fni\\\\\\\/NG3YwQDN4DNpFyoe2OltxzetO2WStLHYR2Os0vcw+2UPLXyDPZwE2dFmC1uaL+9gTlTtzc9GS0a1arSnxxnZR7m0dhQY74SXggUpA9DRJ3cEW9CDcuphvJWmiayqGyuV+hdgOYF6jHbodm4hrApl2UiKCjEOPsKZII3oDKDwqQkIGjK\\\\\\\/qwrSNUC+ZmzTUCQrCJUTwhnYxVcAHoMVGSHgKDtT7xyffin2u3s6Yb\\\\\\\/GLDNOe0Fni+fYlDrNW28fR26h91w\\\\\\\/Evj2yydTOIKA4BBfVXFqjDkgKfcSrrw26pWaQvLGEBzR9XLcSpjt2LTQn7iCPz+cSS74UOGCDI3sZM1L+vhG3JMx6DKDW0O7hxdUrDzUxRxZD\\\\\\\/ew6iGgU5z9tbC44S8r9SqAZyVhrgimmJF\\\\\\\/QAhVUH\\\\\\\/1C3ropt8c8KxXnNFmQLJwcQjTDj1IL+2g1sb6hXM4\\\\\\\/L4A5FfkkV4mE80vhgdUxxE1BXcCIjqMS5FCdatT9M7fvU7LAW36puVHgJ7Kujf3dCDw0zD+\\\\\\\/51RbEQMWMQ+kL+uZDzuMTOfKSKznQWjL3N5+QBP2ALX0\\\\\\\/zngFAlOMLcwSnngvqotmeVGIDeW8\\\\\\\/DGFaz\\\\\\\/hXQwoZeG9rwkY5lCdP8OU2qLp6ZrbneBSKXDpjFGNfnH+Qez+fv96mGYDLUI8u6629lZUQXWROnuPlNIxCHSV3OSffcVHycCKe4Ic2yUGTjcmuqNj2sZ3pmuaRnz7Y2yw6fbBRM+GSrF0EKIumRa\\\\\\\/25shGrxNdj7EpbeVnlZ1FYFt3Ut0CSL64nIYv9ZdA47eXPph0U2hDJnC2V56BGhIWh7HOxIguFh0UQFkoXo2d\\\\\\\/\\\\\\\/qmIavp+mYsxTGJFSWAvgI8ZWX6TwPOc\\\\\\\/XqnGn5avNVNiczrVWoTEi2inVBWfvd52Tmmefk5lYB9BJfeE54czcQGydBYsj5n2NjPyqbT\\\\\\\/66CLf2v0jpHay+Op6HVZC94N+sQVGKyPEp3fXylKmMfHnPXp8Uku8rvY6lz7ZHzWh7Y4gypYlOzmKiOzAtmKnM5iGD+AX9JY+UU2TlneKTZ052AOa1uccZHaKI181r7gniYd6gtkG5+\\\\\\\/oH2KOKSJxUpYMYOR+i9UIOS5ymqSUmQpx7q+b4Ng\\\\\\\/KcHOODwp0u4P376jF+H6V5uMPCphVAIfqKopWA5aE5soev+9OLbPmVvCqOdJfY8P4otl+yyvqiN0GBsX1wy5BdEL0gpHpsZLUK5FtSC5CM63dvmrOrlX3m9wHfWrzIGSmNy+tC5E3kfldhjL7kQh\\\\\\\/uDodwEmYJFWv5VLlXNwIArIoELXYK4i8dTiFc0ixJKu50w+yKvtQ6WsVvqQvGpXiCXpWEeIVyzaAuvnlyyu4mLbKLTCbQAFO1\\\\\\\/ambGyrBx7uwq+JU0Xa+lrTZQ9wgt1B0JTC6QJn\\\\\\\/y6YmiWMwDjxmP19iuaIW5fY1SA7J3wAKUgDDhdInf9r30mpkI\\\\\\\/vICBSbR4wv\\\\\\\/\\\\\\\/KYp+f\\\\\\\/IkXH6Hb9EadRBCY\\\\\\\/Pzq9gtCdZOg+TOy5j\\\\\\\/aeE25bpddN2HaQ1f80Q8YdNa9hr\\\\\\\/Ovmg\\\\\\\/euK8+GK4HMYgTrXp1VhmPoEgC0KU+PRvDlzdHZmguPuoZpH1ClsPzphLIgaraSyGhj7bWiliGmxePa+fC7H3B1rHiFRAvOKahFn6k2YSmd6I1Sh8F3BBR\\\\\\\/W6lE6nm5kNrl00gUVhre3GhhyB1hmAt6IkOnSz2\\\\\\\/5NymYrnBlcNfSM8WVmuyiND4i1jUQvENQ0jVS+Mv1TH8knkPNjIAAXVGY93QdjoKv9U8vtXdTEOU3QJPxvYmYYewEYTP0e1IrMI6B9qR4Xi1Hfu1i5W8PajnAxeACAEwY\\\\\\\/k7Bh8fFap3NOPYaei43QVgEoH+cvYnLAUw4P10thKjY4pXOu2h6Pq9ienp5IZV5s0n\\\\\\\/fFIiEgKZVbEZhoOaP+PNhcTkgkVOHqFTQXMvTdRWkYNzi0QSLoPXDNRSsiPoNJkps3UADDifNkgxSYidONdSdxlroramlPA2j4xDv21SuQCfdZgxlPrZ\\\\\\\/9pdfNzBY\\\\\\\/UyUSqsI7NoT5EVEdPy1VmM210k\\\\\\\/gXZR47sB3MImtEjEJb2knbsJtjc3xjGbZ1jnJkF4X\\\\\\\/YFDBBRwqFPoXWX9d1o7p0aViQyeehr17ywSFvZOFQRhgwVHhcytl4L6QZTXoo5xzGrnc0zeqJqpddYIqhsrmdBq7hcB4e0k6kRvlGqaJcITjNIrW0YajpI9iG3ronecYAXjwIiV6VtTafhDUBwazga63x4iZJJSfcoBSbqINs2AHXSxvSoniiuzH6+wRwkiFFV6AdL1o3a\\\\\\\/1O8cDJLCHjt+PZncVbQgMR2ZZvTRP1k9P3J6snz+qNQfgi3grEkWwGoUgBRdGWZJa5xRQybfpxBMmlIMprokyXjidZ7nj1wF55C5wlawmow08EVP2Rm63hHVr65EO2f7xv\\\\\\\/RRRaXQopeUTdbs6xMkwsCypqvdjQYQkw2HFMBeE61ZvFCn0\\\\\\\/HnKqsq7rr1hdsRpCoshuyvTZdym87Re\\\\\\\/bjMVNMAS9NNDV9LbxUBrvo0c7\\\\\\\/rzfzVhgWphLYCEssnWQe2O0p5vqML1iEU\\\\\\\/8LOPdfwytxnQHfeMqEgrBf3EK6iQILPnEOL\\\\\\\/woudop8cmFijW\\\\\\\/rupC\\\\\\\/g0A\\\\\\\/tbWH3tT6WobIqptgCzT0GRhnn9HoJc5oiZTJnwf357xRJH11WSwHSj7h\\\\\\\/liA0B8Wp532LYT2VYRy+tWhZW\\\\\\\/cWdpI78aFsiMgvonfIT3AdVN\\\\\\\/4ZYlCOshQEGtWCuFP2olLHMQdawhNUXS3UVGUwTvXeH4qMJUKlRDafS53DSZ8xwjzLcERNIc3DsX75y4MrjoAlgEDR4It+eo\\\\\\\/E6M1BwBTj0ijNsHK3js1D+EEuIF2BqOzB4a4kCLhguMXgrUtfyBHKk6vYqR2MDJtY8eLT+5H\\\\\\\/dqhEw7i8dNSrU1+dUYe7gTagRIAzxSAWUg6lGJ\\\\\\\/f6Y\\\\\\\/Tbz20lSXqedJIXZjnkX2ySiW\\\\\\\/NK9b7saBv87HtkNF1MPuCaN8Z6cw7t4wuZoRAHM1hoP25NqDfVGdcWzt2md\\\\\\\/MbhaKiWleJE3NCEajoW9yxPc0Z4X0eToFeJxBdNy7IhjEsg4uEHJWvCplkh8yg5GzsqKaRIK9SGWqU7U2xD972+v2wU4EzS18CPjw0Q5Qfh4Ny9k4oAnE+wCYoNGkFGg0mNMPLBM4fJvXJqHSPZu9bP5ITFpW5kd+X0KlRJhfbVDtlA500q1vhDCMABiVtQ94\\\\\\\/m8GY0eA4DxZG5JVtBcyXOJymNu18oTqTDbpcTcdCau4Mt0LSeuuA4ZtDuwAMZVcn8anEL5t7igTXD3n+PaPhj9KPZzA\\\\\\\/K2EMZuywK4UkWDcoMxZQ7CaXMOxLZVBI8XU1BnuC59MJnNx37uP\\\\\\\/ApIcYjjmxeW\\\\\\\/IH47Xn9l3VFEw7eeXIjQU28jPfxiiEe++POhLjFo\\\\\\\/Gu8pxHaJt8YCKe97vJM\\\\\\\/huR550f2DhNjlU1ROtwl9qY7dy7G7E1jfxi66lqNi9JMNSwI4CASTUnI\\\\\\\/it6TNuB7k+Dpokzk6mpAWycROsSJHfTKdau9j9NkgoJ0m7UbNfx+PC\\\\\\\/YZmfyM7cs7jva2\\\\\\\/mwyd3V94UaZlU601iSDBzaiWBH3W5mnmsmDHzWiK0KOafDgBxoStZCK5HuKQ+YIcsKdPbpEF5XYnt86dfofVuq6GYbfyT4jTum5qhZIjS1XVTIW1lpKZA\\\\\\\/PBJsgj0vFKpwtM2vDHi+f8qRpAvf0\\\\\\\/kV98ChfmoFsTVjnYfWbpYCkjoBCwnqZo6PTqW\\\\\\\/q36QhSziq7c2PkwttzKGGQnMxP\\\\\\\/cbWpz10LmfwBJi1HUEkt\\\\\\\/MYzPWv2HecYBOuWw+IFJMW3eOQE3JQVkJkGb2mqm\\\\\\\/FDfXm21Z9UDIvGGXGNxoIbDgBvzjH2MdDzBK7t52c3C\\\\\\\/kLDgC8V44\\\\\\\/qTK5D9XFeam3bHBlrw0pyLiv0URqUdViXe0+XiMeBgImjirkhfYDr0lzYFmU7Ve4Ru4BdV36sK4iKcMgQut\\\\\\\/d+ZqObANXvA+K+jAeNt2E9c+GAhmg8AtFeKOCAAOGus37LKVkkcuPvkPEBQGliE2ptaDpOISIKL1+nMIUpwQfA0dw\\\\\\\/qEeouRWM\\\\\\\/+yTP8\\\\\\\/TpoWUXTNbsQ1qi5t1WXXAHcvkSmsLzM0bhA4fGBzV0j7BoHuKXQFG4FZXJfH\\\\\\\/vuncqNMJUTedN04q4OEC0tXB2qUttq5EnIwtZCno2c0qpO90deniWTN+GeIvz8MbG0w2ClEZN+fj2EOo7RYJo54Z9fEoitp8LsuA7Hwl2lfDKbTIe3F62irSnVe2Y7fN\\\\\\\/QwupdzeeQsYr1bJyP5WGxVMo6cwSPHAphTCRZixJ80hYE1OpVK3Uxm0GSPr5Q3CBVd9oSuX4PfiRzOWBnrlohbqH0TnXKLSfUyZZHGiX7O0pr+nnMjyrDwv+mZaOkGeNlAhX3sj4zLWPZx3PgBqQ\\\\\\\/h7dypurfXLy9eZaNd2CWL843\\\\\\\/zflnKLJnKPu\\\\\\\/IUebWqOxhDUl9SR7w2L+FEX5\\\\\\\/y3LKEl4Bcin+ZGg4d7SlP6OwOCe2fAJxg32QpSYPXPjC5oecNGC440ibomMVyzhYcuzSusPts1rXHee8m9hON9vOEIJNsq8GHl3u5lbhI72mGKWfE8g6+oVECD2dEF7bKnfs72GrCYQbnFOC57dnMPYPXMfXa4eNkYk3ZwpcqJAqXJ6cdA\\\\\\\/bVGLxBadfMNP1ofunoG2Yaj0\\\\\\\/ubrHy0MkaZ2PN+2l13h3sTdLu924shScXoOwz0ovNxvK80PclsYtrTB1EwTztga5vx9vjkPt05YFSS4IhqQD5a7nsT2pTtD952a4+Zl0jSmGjyp5pVS9xfyuyQq4BqLjVMCU2qUR5VEFbZpdmDj8yduJhOOHHEdhJaM05dmvVIxRxrCeK7uFayCvRkn8DTerq\\\\\\\/\\\\\\\/1Gu\\\\\\\/Grpb1gsyA8uGpL8KmB9BZ+v1VoemgUipwdtRiDBPTPJBnJSGPnDnMsbE2Us0N+8kKczH\\\\\\\/xiEKb7wadsC\\\\\\\/KSByonw9nYlnHXF2hNNlmUxkE9mMXLLD1\\\\\\\/fMWj+GE3DrVwedqVw7xNjR3p+mlXI5RE8Nf60kyd3xQ\\\\\\\/rtCE1D2U\\\\\\\/hxwf8OMgTellgIjXHYkvya12vayc60wUiTuBkcp8n6Nwfv\\\\\\\/apxWprkqGr0zrjRbUWNIhDmFpg7xwJVinxSCLzzjA7oWyHOLdJNhxm5fw\\\\\\\/xVWNErMjjxSkaWrMxNedMzyxKkyvmbrzwDrBOad4PtPsmBWvvVwShgkFLp9wD8TI5mLeJIt0aDuDmnPmRajSohXCMa5RJXezlMcOl\\\\\\\/d5FRek2VkA1oJppCdiCNyk1lT6zvU7DDtFqvaiEcK0XNKI\\\\\\\/oK1JXNtSRJ+MiLa1U3SDtqhaXeymb5DfwYPniC\\\\\\\/fGHXytJKaCdoX9ifpZC3yDuDoPp0BKVgN30Q7CAnFseyOOl6Lpqtz0dUqIsLLcVku+c7edeq+USsfcxmdCV8mHfDTsu3rH+3mCmt1MZmVd2TQRjJJdyvl6OXzLqQm31ETdiPUjjQ5781TN2BT4kUAXw\\\\\\\/w1KGnVnP5hB\\\\\\\/gjz3WlXo4LjQFO2bxEoC6rtd\\\\\\\/uKkeK7Z1zrFRC83N20VI2jLumsuBoHCYw6LAl\\\\\\\/znwwa1VAkHUh5J3j5evGnzKzmKEnry5gYjVPzZZrY8S52LETjaIfreNJt9xQpRqdA2oPvwGHQvIL62ULWyVDD\\\\\\\/kvziAFgpyUOwWYUhJg1Amjm8oSwhH5hrEve+PPd2NemH+BvkyATV4kd9c+mqoCu62ex3Gu7Eo7ppwPI8d6spQ7atXizeBh2Y+GKqOXwSRpJKovpwUAHzajnv7\\\\\\\/X3lyeZB1NYeuOro1kDyTlZp3m9EVto1cWxhWSHEo4xqVMAJnlLkFxDflWbcMbi6qBu3RGdiJGmE4M4sCTxB4ZavX90aLAWmQexIorDySdYgY\\\\\\\/51mlpeLspbBG+3c+BxNkAXbLeSEXi1qQil3ImHry4BV1TawypC5YYvdJnVvghfEh6ue6lQjFKDNhwPY5MsTAriojK7VrdNVJ1Js5ra0ehEVPZ5BrjJJslEk10QLrfYjv8i1cP8QpB\\\\\\\/rtUHuSrhsnLJ3mjMY+NHyHSow50zXBuboLaeMuX49jTxQ\\\\\\\/9JwO\\\\\\\/uMPyYdownM3txv7UBjq0cgdGfDaLWlO\\\\\\\/2z6mgJPT+AY7ASuv8hEHsE7mK8Ct\\\\\\\/hwBBAusoG22AEvkNe6uJE4FUvk1bhf995pQkMTFWl3I8DyHf1Hf4OT71pi5weqdcWbySnYvt8ogXtWR1gpMjwLsFHu8lpkKEQmEv8DdcPsPG2arpy3h6VOhvEZ1aduGG45fzN7mUi9\\\\\\\/vtyYzItnkfMwX7U6YSbeeR2koGxmahsjieYqfWGEW7cxh0L4s+96lOBCRzE1l3fIIDZyWgmNEg0S0DzK+G8+wqV73XYvSx8dlUqlYGKXCnJmZQ0WvFTdJu\\\\\\\/x8Lf2rBIwQUYfZNKbmyvTyDPLItobLS1g1dRzMG37ND3xq2WiFYDI0a7KdAvskcF9i0Qo9BpBJIfZ7Wq76MThI19LerHO7z6wl0PpHRXuPffsET5NwUD8nBsSlxqGFmV5mgs7z5GXjcZHHxEV0U29uwwt\\\\\\\/IY+kkYsT7R23wK4hSS9aOgGnKHEpP7nURkpuF1kr6vX36X0Ka4C7G7zKAjrXSN9ueyey5iATttGCLn6l32Bf+tCRCWfT5PAYzcKHQrqbEzaQ2MgTJIR3ySq1jEKKQQD7Lzkt0mlk95aVu4MWZSy4otHisLTO\\\\\\\/uWEmfzIpic1d1KYws8ZTrWoOC7QgSKt\\\\\\\/+\\\\\\\/Qeyj6mHINnP4VgvLLcNum7OTsAv8GWwY5SEE3k3HWQv+HnbBDj2xJvibm3unsQLDnPfrCLlrKLGT6lwoDpfAP4UTuUhTBUsrJm8+Vy3MdwDZQypwACp36yDqCnzj16Pce4MoIThltR4iZZIMWNjJeGO5jdeVAPFmJ8vqVURG9qqae8Jpj\\\\\\\/KCec6O1cJAnlvMah7+OI0VpqFOK9fnLe8+\\\\\\\/OMhRCLtOhvR1noJQgi+yTBskjHEUxAi2czcz5v9fvRdm9xv54Mvl38hDU1DS+obvBEDUmiLYCwsoQy+taxBKwwRFe0QtiLzXw3Xd7AbEbHVJsnh1ryi7lGU5JEw9pgXXN1nFSG24Ogjn3p4YGrTPx9maSUk+qU6HLt7YWDYE8eR6PSaYEfBHTurt70H1rDSuO5N\\\\\\\/OzxrImATgeL\\\\\\\/VKyz5aG2y9Mw3KKf4tZEUFGtqYQDttqx9CqUUlO0mYT2fi2GssYNDYeyK43l4QifW9xGQWFHTTD7v9UhzTFMmaiHQ7zHBkQsPZSCJJ884i3id5+Xd8d4K9fBEJwOaSrTrEIckzh8HR8KmyWpfjdczYKBHy6NHXrQOMH0RKsb8YpRGLxqnVibN1cFfyUwWQAxDU4FHXkNUEAC1ah1s8NVxc\\\\\\\/KHl0sSqMoYO+CpdqQm7b4KLEVJjYCyvjGFb7HQbCSVUvisYGvv0dX74L96hBDijv\\\\\\\/IMS30o83vay8rsz7GbcZQL8jkE2Mf8eoF1bBkVF4zO0I7TMQdv\\\\\\\/ndTyWna8rLUVWcALRq2LnZzqwzJsB1kpK1l1kBydQv1vT2LK52w6UfRq91hkuolqpM7ra\\\\\\\/Fqu+cgXKKNtHI5bPi7vruml58e3IYEdR3pColdpug6ota\\\\\\\/bSCsBGui\\\\\\\/ffFvLeXJmAbvMwDrJ\\\\\\\/24owLUFyaYEBUCc67i7xSbpvjrwwSllmNqpS9DVAz4FnPLuQGsn94uv\\\\\\\/wlQhB2VN60agAbK+Qhq00YczlqMqK5KoEQ6a26fLnOV7vryh\\\\\\\/e1XNOQ2IPpHqLXKYIvB3w1gHbOcwJ4Z0D3w+LR6dx03uORBXQIzZRBTgvUe78h1pdkCNJjWxhmVbPzplNB+Ur\\\\\\\/NbwKjUrp5Zdm1ODMo3vxkmDhvmY97igdoAw+iUioILHGsnu+Z+GnWddyaoVrWEhLix\\\\\\\/TEzA6TeWf4XW0dU1aIIZkBKC59yvyouz+J7A33wOTN2JRoqmcBqsNZDLe4RpPYuV7vIAJW+4ejN+Rw7XvcPkocVTdRbtDFSVLC4OJ0n4NpJ0rMLOwWedmD+f0QtzuMx263sGc+PchB7CdAFls1o5AXTqZL1EGnuMWQ38zOvCuJ5ROy9J67NxjPr7PvC\\\\\\\/d8w4w0DKn58xybYAQUyZaJSzw6uEzA835MSOej0cNXaj3NJ62xbc2V7WJv63qzcsA5NwY\\\\\\\/L+SDMaEvSaefuokuNN7vpnxgUjD9QWETzaQrMzOlmq2Y58Ul6YIB4cMYYYDCqs5UQ2hFzLafipMl4LSCSkmHEt0DJsVqRhR3m9qaGa0S5mf35GKfDbTF7GULtnqQjGLXg97+Hgltwn6CIgt7DDzTzInsO+1mUifiQOawNU5uRBuy3LGFet59UW4RNxNm06xpsegUISg7vjBMDK92rUtKQXfjpmmlpGVz4KMGAAgZ7vYm77Tzotvjr\\\\\\\/yceeZnE1h4HMMhHmTV8+\\\\\\\/2pwhnGJqAup7uS8hSETwAHKe4V6xfkZyDposTy\\\\\\\/UbTnj0Yjgl06OSvzvG11jYjDSgf4+TjYlYHgENS8lmzGq+zta+nb76c3H7D7xgIeBg6ESJ3IlVsduQXZHQjfUg0gdD1eK4yhQuooXa3uVXoji2YCYnvPvG9qm4ATT8vl1eTri44JDBGDtHNmACm1OJC8raEXx9xvNb1E28aC2BMJ+\\\\\\\/eszDibTORcS5MXmJDiI5xbxTG\\\\\\\/knWmhs4rfv92SQbtODlW8QNbB3691\\\\\\\/c7WebR6GfmyMaQdtlqdg2Ggsjb1QVX2+rzgdC7iIy0xbOilid6r8FLJcpl24WQfvi7bCbTRONV3wYVtgRWT8tHfYGSIA\\\\\\\/zbXi\\\\\\\/dfWwxgMO7oXdMWxxRSPJgXDYkiK0cyo9sMIzgD0QgEcmB14DPYd+HPzo3EuKmV\\\\\\\/m89sF2BEwk66OlzXzE3rb5kEHfGbiSDGvERJK+xIoO49N16yv29HV\\\\\\\/0H2CF2+NNZRmOeajMHKJZ\\\\\\\/nxt6HaIsFx3rD1itwap23qVEEz8vU2U7hc9QPz1M8rlD48B+ONAoDs66Pady\\\\\\\/zA7tjWPJlasYJaOCq1izv6LrJe7vK8I2UrZlA3UJxaiRa4lDi3tECeM5b0Tj+2qUoyzt1zJqnVabDVXxzNdir7dYRCIc59SvTZbiyjt9fszJYLsmIwQqrUta8vQrJxNwpE+Hv8G7IoRpwvVhVZidHIb4F0+3\\\\\\\/x2U6JGv4ZTLAt0+t\\\\\\\/MMdoLzH2ys+L\\\\\\\/w\\\\\\\/dmyHdTVtoJGruQDvA1oIVMR8aanPL+QvRHz\\\\\\\/OqWIguNbZvOR30kLaFdg\\\\\\\/JP+KYw6YdkgCxYQViu+aoCOHOjdoMgvQZb7v5AjlxKZ8mc1RuR2SS73H1O\\\\\\\/KcOtRzN2w3JX6jokjS\\\\\\\/I6nEnWdknklCxmJgoJvoi1fGI8SZFmB1YwnH4Xx4UHZuTfViq3VTD6RRxrqnVgbBbqozDYmpwzGz5VZguGNCVpL404sVGOz56oEdfXOhL99Svw3s2EO\\\\\\\/zhmqA9hItt2tr7QioJbq2CmB4851SQITk\\\\\\\/NCc8pArCukaoPuJSujnQfp4sRS9GWLjfopWMPq9ayX72jwLrPEKxlMol\\\\\\\/mm3FYkNpBYFfhl9onrb2BhNij1nECWfGc4Dfh9Kbk4mADmVmy7oAwJfc5SIKY3TKUqhf9TQIoGrTSMXupNsmwxwL8qVsvVsBppV6o9k651jOIfVUGuSWL74IoizwmspVSSdiNrqdgDjBZq9RZ\\\\\\\/37+c+BMwVZOSGAZe8Ul5s+zrJ08QCRw3471w\\\\\\\/mQOar\\\\\\\/wgYhHx+1SIAjmLF8FY68YpsA7CYUiPEPajdQsPpdLzDqYMiUHCc5D9DWRmfl74Z+YVAr0YOsSC1dXI203XT4uGu9CCzKNxpY\\\\\\\/H0lEqwBwTtq8\\\\\\\/AosjMk6vy0e7AGc38t4GrmZQROtNurcCbr+dAJ0n5Ia8AAoAZUa3p3tzk25lJGC1Mk1P1HXxVHNU4ESNXo6oAlhVXKFydtVJAf6iOyrzRkC34\\\\\\\/huSjla8y8u9UD1FraQFUn1u3zIsPNp5Y68FL7ihOytabHGiiyoK2H4Z\\\\\\\/ANJsAhBKOFw85PkKXu6K\\\\\\\/vxE12u9+GqeCzQT6DV\\\\\\\/U8ql1RsxEtloYtQyDMMYoszCjprF8HRxKFElWX5NBz0ztGMeIYkYN9Iq3UT4qT+H4U9sf3Rc9PxrBFV7z3Z2TsyHHghx8kHRBB78l4SdmVvHqEal4\\\\\\\/O0dib8luD8Pqx1iIOBStqASh7MxFpyfMsjAKSqm3T0BZmgN3rGhz8V4cEu1HZ\\\\\\\/aDdOzwgtKXqajHo6PtSNMRC1o4Ne6Y9idTd4rSCag7f7\\\\\\\/Lu\\\\\\\/SheafzDAtjVP\\\\\\\/NBm5YfC5VvDXjif8UznxuOImVmUoVShlnbW0t97tFgdcMdZV9O9eoH6mR8TqrDzR4hMkkw5Bcl1H7vCXG\\\\\\\/FFjTG0+dLBmygft8ElPcS5I\\\\\\\/NSBpCfCg7gLH2iD\\\\\\\/MF7Y3+7K1SBwTjv2ZoYqoJqt4zpv+jSgve6QIcZijpCkqIhlAPYzg3m3b40OaAoWqL47J7UycgM2g0cKWxcO\\\\\\\/KnOtVnZXZxanGtxS6j3hDlCmDHQnxONMpE8VmZEis528efRQrBjFqUysTH\\\\\\\/UIYgcgySloxqvnuy5RJOX2v5OdLedB9HcTZg7zFIoU9ObxPOyouoJSHJaq1dGsvI0nkjTrgrYpB9RJKVlm7m2QZ\\\\\\\/hJlkaGPY+iJjyw0Bnv1OwpGozd8cs2S125i4ZpdIsRCo+WbllqfooHvONjjyNovzVPwGpmtH\\\\\\\/o3wL0\\\\\\\/\\\\\\\/YigBzlfe2Vv7XbwjSnVH4XYEr4MzOG0XKQB3DosCMRta4lI1qvzYNsKvJKyEtDPTDyl7Z0CZQ5JyXvXErJz4W\\\\\\\/eWpsAiVyk0hUoX0nu0pTrQQxA0MfsBCt7yMsUtE632IZiUQOE6kjgrNLNt5gUmHLfqvO3OWSTtKcUnDUSnoZMov4CDUhDS8AbE1S8wxZHI9E\\\\\\\/TH\\\\\\\/q9yjOCvMSma\\\\\\\/4lz488lXHy0n7Js7reeu2lLySFlMLdrVtZ6cKMBf0uUhrwGyx5X4mEn6tHNZ8JUdNzvx+f97JVVJAP5\\\\\\\/uvo8jcAnzx8bw83eDP2lcCL6K9eqr+bFoPFN0l4NvI3qsn8AedYes7udsHqGF2Ytzb61gtNHLChoUpgeVgyr2JxMNyEcnpekdWucVF53o\\\\\\\/jI3ktTE\\\\\\\/K16YftVu\\\\\\\/F3VyxAE12Vl3RwhtexJQv9PxvVNjW3q0PGHBuhzWA\\\\\\\/+veMbsXzX+Sr3iIGgUlkIimCgJQT2oGqLnZ0+3tioY6cg4DyNswBxTX823x\\\\\\\/1clp8uhnDAg7CHr+1UOnpcDmSS4AgcJxyIQ0Y3vFTExi3\\\\\\\/06++G\\\\\\\/KFSM6M0XkVp5g7DONIGH41jCu9IMRCc9lARBf9xUt3S6ncTOQN+iPdFYdSDFJwmjj1hJj8SK8RtvFLKNrq\\\\\\\/avL6sg34o2iG859\\\\\\\/leJZs44PfjD4etDQdzV8I8lQLh2aie16o2c2MISHY00As5uhaIB9SMJqBR5TaTSQxffx\\\\\\\/zxtTqRHmIqUGetOJbkwLtCuqybOxBeAbsiOGqB\\\\\\\/YzAoS6vhcKv0cTW6NX0DcWqFV8VRgu7YcOIQqHn\\\\\\\/36Ea5pe1xDVdmbxhikzh4yTRXZr6p61iHdc5bChT9gYlm0oB6rteDCtOAKtShelwey\\\\\\\/vBxYo5UvvRo9qqAQYSkXnhg1nYyqZeQyh26uwvoNeJCh0YE1PplUKALb4+MgndNUHF0AwliVySPoGSokyvvYAworqBHyP73t2IF\\\\\\\/6bjdnF1DDH\\\\\\\/EHdx++rZovEzTGTvqKeJ\\\\\\\/hV5KfrIguwxGjyzQ34uAlGqtkN9sQFjqp5Gio6Hh58jpzBu8J6JosK7Ac10lWiry3Idqu1I6AQ6UIar7j9u67jMM4A0lCGD\\\\\\\/JVBmfat51tJWDkqiT3Eyd\\\\\\\/h71yZRWA5Y\\\\\\\/8D3\\\\\\\/hUDSFVPCsmxCDaEcozBoOGoPXykytRC+8oG1su1RTodngnUUnghzm1yPrBO2\\\\\\\/8GmwKxOq+1dOJHKzXgDAsPtVwO3ktK2NvEUDeqN0r\\\\\\\/7CFAtgrBtSynBb6kBpE0cX0mWytfUA1MC0pMKt67H75\\\\\\\/tom6f34+swSk7dQQpx27suDIO3FfkhjSM8rn7NtMJf\\\\\\\/MGjMVJnQadqOyjgEbgDlODo8KmQuzv7B8ojjl5qVAG2Mb7gPau9WB8fUmtPOu2WeVMkCXu+gNsNmmZvp5qVP+XZnmdu7tf8MGSKyPRe5s\\\\\\\/xbaCiSLl8EkjF9oPs3sTQusRDNpw3+NlmIw82oPoU7Sz\\\\\\\/EHKUZN39\\\\\\\/YG0usDx2xjae4lKw5oC7KbK5aQ59apjMOoI+pO2fHgp029i67KquDI5Ezxaz+dor7FBMSezyeR3If9au0Q+bozQ1QyBUVerliYmF2ScCJIgJSTbKxWXHrwhrjZqxW4Pp49qfUy8XzvlHQ62jwMx5UY6002YmZ0NgrBIEsjckux5HsXn7ARXmQxTEN7f0iCCS7hVyJ3YsPWnz8F89MPbcpACCEonMRemBfT1yHEyOwsuFWr8xeqSzlcFCw1maxEKSlOx6tlw3Wa4QZ4IBfVHefpASgavQ+pgQm4AZIqg5fHli++Lshlb0\\\\\\\/I1cBS3mu8D2Lc\\\\\\\/1ETwKSI4qAr\\\\\\\/pZL5tcHTcZDKiGTrUsSkyUFJqfIdbnfjyM5+d119v3EmGd40WfRCm4BR8+GL3lwThrsjTAFOxjcP01tYIIQnZ7rkQQxqs2Velh4ihJHmzGMPa6+Dk2xY2dTvyCG8FBGeZF9FJ7UM+5+egMwDIh1lkIBsNnpJs0gfwYnuJ0bBKpCjRhj9TkJ\\\\\\\/hRpwjYVh6DCbD2Z0Zbcj3JSQQxClCej9p465zVz\\\\\\\/IufrQAUwnGznsyNhvl94UlmLk0GOy1AFJ36DgPr8dAz4jsmfPMkVm82Pm5oyDym7KGe3d6F772ikcbR49adqNSngyC4bNAc0rgCjudlY5SdkBfCnHUqV9pHaM7vh2Gp26tcCT7YJ2l3IINnY6nK3WpRFb9WmpN7W4UUgfW7xT6oOx2DHDQSUF0+Nz+2rwq4XtETWwx4kauV9Izu0m9\\\\\\\/B3Va5biNQqekLPkO7XUA6q1hTzkBOP8cgawhxMyAiWa0jsRqgf1xpDrr5Gkh9NRl+CUDbtcIeGfS3YzPOzajqPVTtkPP649aG+QwPmPBORQJSqeuDoH9X+mcUWE7Mk60seAwJc6FfyryCROvjVZ5xu1E\\\\\\\/sMYSDN20DQGRhmYOvo6ExYtQghzisKzp3uIviF+KwtHXTxoW75GXNElAgvSnPZWj6cpNyyzOuUFmQR8lopsOft\\\\\\\/1Qpj+Xx2BCx5m\\\\\\\/+MDijays8myXEfQyacai3DiLj13l4PNKY+HKbLMARMmxD17ihtjSTDK2uHUKbvHo3EaCppKMRW3yocG6nFB\\\\\\\/HnSnU2uxC3TnB1lYt4\\\\\\\/1ow3xEIS8b7k2Or+x049uinG5QJ8l4iBWLEaBHkY+CXDU59saJTz7VOIShMK5VfuWVyuqJLDjrDZcKikCr7rQZElGxlrlOyVy1Y77dSdHrF9t6mdsTQTu9vPl9CQVuPg5spNqd6h3Dn\\\\\\\/x33I1\\\\\\\/HzSzBnmf9IWgES0UiQw75Jw5hDKceownU4vL\\\\\\\/+YM1Xj5QHmwdfE8LXKtrROKWDC2K66rMcA8IoUndyoxShYQJh5gds9ANgZ728E9TVDz2r6gO09OlRw4TPm0xeKfclYm9q8McTFaFZcrsr\\\\\\\/Fxffs9rVXQQGzHFp9Nqor8Y2QmDjXSXLXuXgAdlUYVAh3am3Uj4DYxqULXu5N3CPY1w+K47o08PapG5ajXVdnjiILx9+mewIFO\\\\\\\/Ah5gK6+JVihLRg+kIR7DNLMUaXH37GKz7KowVVfsndJLT\\\\\\\/QBTExW0m7szggt17ZdhvDqnW3LqWgtOGGMaI0c4\\\\\\\/kyhljeeFqvjwa5\\\\\\\/YRylGMM8SImJQSZWE7rzr3NHqs2o0UcHJnjATXAA9Mf08VEc0welxZvl2Zr7wuZAnTVdnP0t7LK1Lt8UiXRgRKbrKlLvTpzcyOWiKRDvwUZ3d1Q9lNvIBR0V2FY5uG5OY5SJ2\\\\\\\/9tii5gQIfl+RyisKq9W2zi6qodXbJB8ApRrEyEJgiv9deQRJMm8mYUdSlhg0bca4WM1cytbpWdGB4FMlob7UeG1MQ090jN9uxjQT1XDIYwEFXVndI9IyzYXQsUp9rfgqPb4Ct0UUTUs9dp17qKjCGuhNzfk7rsv06MjAJq5OGxKUI8jbdBxKdguK1Bly2K888uF1dfOS7ZVC3X6\\\\\\\/EKrReMAIxHVAa1PEUnAGWfL6p6klrNS5qe4viwDs5VjCqQN96L\\\\\\\/74c0x\\\\\\\/IJ7T3NYHOLDPLPQpMsAHToWTUIP03lVivUT+F0QLgAz1kGgTQ9+L0ZtQ1WnK\\\\\\\/d0jDfg12I6PmV3X7ZJjMDiV2btDtFqz4NmZxFRqQJ9fWnnExTPaDQbqkQfFOjMjB\\\\\\\/B5HtYHsNvxSw4Ki7Oz9PRpVnLOGlmv3ffqKyvIgc996ByZJDq02+YZ94BPHEvIZJrX92pEU4pJpYi1L5MhqVBqHXPeirqr13skAvu1OWPjPQW4WIUbOkji+IB\\\\\\\/QBz4YTyN2m0d+2Hdc5BQJXoF+GWcGu0taFbyNq13Cqk65ewwc4jG6YIjeS1GLmNu0p0UypWTYGdUHxs1YsOE1ArdQ0FJpEGM7SM1lT0hGSQdF81vyjKyoVasMc189Q+NTIKny+3ey9xPZb\\\\\\\/Ns0QIWuGL115Q1dJLpaQ1G88pPRJeWCQQfi23Fr3i5QmYYVtbIfFf4BtUBmr66PlpQDmJ7y4RXN9FSj3Ks9ZZ0x3psmqWbQrbuim9qHMF4fneepp845tTJR9QONlFn7f8DB\\\\\\\/W2xJK34yiMWSlyvLw2id+pZTKd28k\\\\\\\/+OP7n95J00z\\\\\\\/AXI5\\\\\\\/Ovt\\\\\\\/HYjFGGzdUWzIgEniy3xmIYqXXU\\\\\\\/uELSk9tN4Geh7SWdZj7FBCy6snIUkvsPZkU++luuVtfaByTshVdmlse\\\\\\\/yvMGSLRs0GYFRtaaeWa3imhFOL3G8nHdMCcbUPV7hiVhO\\\\\\\/j49VPRSGIyMaOuk5TkfaL2qH0v4ErVoVBi6zqZuF25KdauvjuWoMtBASqmKe3pIs8k+MoqVrahqyOCyAZsAYuLrlKw96ExwEZy\\\\\\\/mDseWJp9Rgl5jt7UHcb+XYNxvpnW1uaGqZYkBYy4dEwsk5OKk6GL7BFlwj5zxbcEZCDN9yBx1OHQRyauLMezj\\\\\\\/OgUc5\\\\\\\/8lhlycf8Yidl3WItm3Xb1AHpuN0DWPmNuxVNXZx6GKn85TxRGWtFZBWnHvBeVh+c4mbdnP4l3k9O1Z5GhLE3d8zWBhHFxrKLA8Rq6AnfaXFcQj6Cc4N5idzjyf4OL21NOxi3gLz9TyJN7A+DlNS1HoM\\\\\\\/2pnkxZrPYXiSrAvSX\\\\\\\/FXiwPhTKzkM8IJcAUjTHncNBnyr1K2ajmjYH+a3CxUc2kRlw4e0606Jzc9oKdv3LEIR35DwxWdb6Rn\\\\\\\/kjHxEDbaox\\\\\\\/I7rzpHNoZB+v2x17yg9RodM4y\\\\\\\/vbEF7tCYzdYI1fmMkEpjO6hyXtL7NkNgi+2WHxhJQaqb1TRq4zQD9MSGlFmT4WTiAD9+YLX7igZeGj7EGbwpN3qTpMBC9hFyhxp7SokSTyHErB3lPr4WK+3vrqt\\\\\\\/Q3uueGxZ5ZVA\\\\\\\/JMhwjBNfJJFMY1WXqIaFWskQ3A00+Jf\\\\\\\/hPwGtGGp6YF1eEDPeaoipGzMoPsfKYPh4FgCbjyGcbj9nlR6QEdSaUUMruBx\\\\\\\/NpMm6DezdiMh9A\\\\\\\/zSb56cZmqd7eKYYsXL73Wm3LwOQe0iJekJ5Ugik8iwP5d3ydsepDT0dSNe3aZVZ\\\\\\\/qiqtSqNueqXWzlbBmOxvf3qFMYIdJ9B5jB4xenl+BL7kLzvtZFBiIFgis4dWNAefVIyzVnYGcei14hgFRE4s6Y4MMkPmZzWxpPqvdXSQGhcSzrgzpVpyuP71RkkfGvUAhqVmq6K\\\\\\\/whDjzOUARTBXDvUlK6wPllJj6s+Yv5v6OyWCpI\\\\\\\/PFEnbjOn+MoBEF2SYfI1dQW19CfuOd6t5gCO73xghU8lk7ZkQf+mKX8Rx8NOijl\\\\\\\/ucqqCwJLEQbHnT9Hm1Drdxk4xSreDIsAZGd56f1aolGE3e\\\\\\\/25GbfLHqQ2IMgPRn4xXKV\\\\\\\/4CyiSvGnscnXHk7QCzyudGMZELtHoLda2tzVRjRjDQhZ+kBVN+hogBvDu0U38iqkUwL0g2zuAuwxmeOusjFI1EPLEik9w9OUrw5LNG6MDNBoMIVzmIUGrLKOnXVHuOXryCGIxzN7kiLPNLU1hdeKIDeOZIUAm7vq06dM\\\\\\\/xl77DgUXgke9zYXsFsLapn6yQcWUqBO6TRm4sEsVZ+UtXAU4U4LfZTWVwz5bfqynLcCc3fuLJI8KaI0+ULt2SMYpILMyOYw2fkky+ZUozlVdZZN2HIZW2g8ohhDkzMcxabn4lghUFAa43BXII1auZt2f4Q+e2YwRT488ypCX60J1scT4SySN0t4+BJZ5aaYp\\\\\\\/Ot\\\\\\\/5M31ZOyJNH3ZdKqXWxaCOqNC9Adw7HJ9ZYLoEA2WqxdRsen2wlxcZ2Bey1bREx102lpR4+7LbMsavWpuJWDWiFPDdl9fB5IOiDR64wacbGO+945G9w1KyfFjPTqbNOb5HZAQKNUvPAc+YAg+czUkCBsJp7QL3YSmfkIDh6Ueu+73cehuffSDPIjEtNCCZ6Ce+P06JzmWjfSas5rzs5LkQA78Py47K2RPpMMw3m\\\\\\\/yaD1bj4uQZFHtXM8DZsMs9dcwnbMyMikpLiIdSTJlv0nOHISRx29mXws09uuMObsxf3gaKlTO9a6wZ\\\\\\\/uBK5ZWlV3sBNqj5O4GZTXPLbTEQEAyHzo3ZDmoo5rMu\\\\\\\/E1aGc4EM7VEWg2b+j5q9KsY7qlpgQavBCdsW8R+spjKOqmMP4gMLfq1jG9nGsj4QSVs4EG6PuS6fiHKrewTeySf0zrv2rdJwwbhhvjtJIPMFvAeyafC+qjGXJmXGTqnUbVWfcZwiy49L2enNE6xG2KVbchLwcGuJkDrQM8QMBVYZvva+uXZVTl1CfN\\\\\\\/wJnWXBw6zNjXyBXMFWsOsNwEk8ZaDp5r5syEVlDESwRaSV91gRU+OCNpJCTtFM1jHAQGUT5jzBlwsFRaBzySBps6QSXI4Qe7xek73DXawgijTYjY\\\\\\\/qmgnq3EcAVD0gQUs3Z3xHyT85wknt1rq5CAKpK8336g6uZ15X3id0uGfS1zrsIsC33hP6P5D1vBihwUG94RmvGz6MhQoEoc55GYpaDqYvEsRnM5LYTfH02c6A9tOdy3uu0ymLAYecqthPhI9zi8TDuZLhOlBhJN0\\\\\\\/pze3DC36b7B\\\\\\\/mbliI\\\\\\\/5gsP+5eOxWSo2qX4cDp7nzRdqW8E7p\\\\\\\/GqN2nmE9Lmj4ledzRUoFbgWqObtsASZnvGFK8ya9ncp30b8SGeNsHTybpqiIzMeRBi8n7Dk7N8s0a1dv5uqK\\\\\\\/q3fTRhXx0VWJbIo2Oc9ctkmhUuKAUUtL9\\\\\\\/OGBZLOYG+OvjCxEjwjlTFg9jYVDzWU0Dcos6kuYL3iBkXtkcLGxan2owFPNtolErC4Io38YaRfe9KJZliVEU\\\\\\\/x01\\\\\\\/GxlYgdQQC6kwBKv0U4FD8hgxXfi2T84u75SnKQI84Vho11nrTYcfxZDdontNW4gRQ6Csu01T5LCfY9nMm6bRINiQ46JtSuEKgF+WpiWT166TdobXOyjeT4Hd2Gg7o0NOiHfjSCSMGdVJe9wU\\\\\\\/dtDGMvUhezEtMylTXVcDd7gui1M86XzQ10zE+FSlixW5ObJW3C1mpojGSmMgK1D2whYQfWuY8GjTBVvNfufEjl7CFeSBlkvjRX\\\\\\\/OXaAk2LXzWtpYr9RlmafjZqZsrpCIK1LQXUYmh9UOMY8aNbUdYQegFH\\\\\\\/6p01DYIXLzpDNH1P1uvOPfiDLVbQh01s2CJoKZlZbUA59d3K6Pak9AdHWd4vfWVuWupf2qRiwbpEVwnOCQzqw1UM9mFXNQBqLz0voZdmsHwtyuaei3KGJdtG0IpDkQt9hkgtC4rEIhxOufsP0ll0HtJ6yvbVfBPbQpP2V\\\\\\\/mxLHCaC9mMX66TwMy7S6k0Qlgf+VYNx8h6s7aB0sGUyRPVY4l3sVUeQvWFhK74+3+pq0r1QXSVruUxVU8t+ji3Pwkrp3J1xQ3f+LQGE6UgypCprSP4R+3ek12HchtV0Ipcy9lYQUtiLVsVgmzkYFuVd8MroOOfQZbgF6sltjrfqZuElj6t8l924zrRTjWu4MSpLEU1cSs0COmoOXM9RfSUThGCmdvSqCC\\\\\\\/lSTXwFur5GR1m7\\\\\\\/GGgPCpXLgegpx4fXIkuGOR7utvQXjdCodRIqnOCihcaITYk0en6s7hQ3JXK3s+230NLFd8QAiMPI0P3moED3945T3RTVehE750lEYji5KVcqfA9iU1clxmHSiCI8+ZlAo8zdHyVu4dKaRR9aSu7S4l80VkOkZgssRSFnb4mA+ydZ\\\\\\\/Z+L\\\\\\\/gdgJCMOGoQNTjEYra1Lc3fc7Sjwi9ARsqd9D2FYjCDaXc0LKSDh8JXVDetAftE2prNREd2bFhnyvIBek0dQ+KypKHQ11v1wgkWKXGkCDFaqU+VCe\\\\\\\/gJ02wE9MuRcWJ8rVjh4inDiDoKF5lheXwNwQkfM7xSwFs5yu+b4uawuWOHEtpGyQ7By2QWg3oGhGdqTErfRL\\\\\\\/xKhORO2xVmJOwPutzsX3KY8ueEdB3hnnw\\\\\\\/kyfHPtaHvsJeOqTzLXGU86mFXAsSYf4RpxwBXYtuVVYVw\\\\\\\/lhb6HVkFYWxR2okiww53hdqPLJfmHnKXegWH6ti4VQOZT\\\\\\\/xvzBM8bgGpCb9z4xyUYylTl7eWKyNeEEExYKJuylF4ZQYS1BDRWBBX\\\\\\\/5fBaY7xkSb5M8YTmuYpB3nptc6y5pN4+shSE08aqUf8gPeOR7GwFc8I3XP4AZrmajEywJlIClrs65hOwXvWo2rqUJzLi+GPqtD3vCMXDXyJZ1XS+ayl6b9ZWO\\\\\\\/aZy6KMWH5IudSG8HYQ1PEDUS6K3ltGphlgME4b0vmTlrjh17L0Ld1ya7+sjzVLh7Bw8qE3ljKEL1hvNh0p56yiqazFlrV\\\\\\\/uxo5MpVpsJKOFMPn0llqBRhgBo6F2UkCCoy5LTXMS8IU9hRs6fmvAcc7uCEhnOH+Gf2pu9ZKZTiiT55kJ8gAy3UjvzE\\\\\\\/YAO+Lif+AgkAdVhc+WUXVKI7xp7Eni+j8QMAyZ3rsKzth+iY5lgMGhzyU2q+lHZGK1yLodG7ZGXzHOdTdWxQvdTnQ1pDu3zOXOE+WxEerZDAIIyw6EqZDgDmfF7JinZ6r9++Dbl\\\\\\\/ZGysZzr9puiSjeiyQV9z0CBVHyXSCvKEsjzx6ey9WgeSGNjYdmCONhHf3dsmnMP6u\\\\\\\/SRTnH4CnkRsjH5Kt3mXC\\\\\\\/T8dZZZgL9xO3M9ki2gRn\\\\\\\/Ct2ssJUtwZt8TAHv1JNHELETvZk414fme4+XXktJjsewLqsmNq7Oz3q7AzhN4P55FBzXvrZW\\\\\\\/9jehRmj9gxpVKYucAgqHBvVSs+n\\\\\\\/GIGnYwEAjRJObG0jj+IFw0vPeVHf1WO7Vfotgl4RnIQl9blvUOMZO0diAdTHirVZChzJSjnaEwqgfkljCOMq0p4mpE3g8qAY\\\\\\\/tp9nS06WXw7nY0jAcj2qBYuDWXfYwMl5LSQVbTG+ryklFrm8aTyiflg8KxA8ersXZqz5HTTkk8lyjJpCX\\\\\\\/TIcSogJSIdAhM0ENpIHMajQ\\\\\\\/t3CSWWwtK+OOPdLhRosngVcGfrgViyiDFN+2aFiPDUNN+vLVdZ13\\\\\\\/P0skWTRzDjYqNZxfQ47TbpQPxXWjDzU2KLviC3CiNxNfEq\\\\\\\/W7PZ4erC5IlQpcwxxOVZZINktnlHsKIdsIhVQDWcy\\\\\\\/0z+qdcMdjXYZfTeHWtuCm3AwiHwRkxXLb6PSPn8XF3nrp2mvt+iKTVFq3EBdvjSwozIA6q3DMhiC5UUlf79CahvrQsVrgJncYoI9\\\\\\\/Q\\\\\\\/vhdMK2EbcGLA8k0pngtLO2oKZQorgmqJDKw3iU+5V7rqryhU\\\\\\\/2waxNZzZZht6PM3oIINi3ku2xvb3eF6n4CHAcsrMlfN23qeu0jKWWCrD7ha4wW7NvaiGyiA+20tbd7pveW2FAgyAydaRwEDM+NNLzC3NLZKvd+ua0kkG1TPgz2e07Sgf1OLGfb+u4JhLPwlCK\\\\\\\/IJphK5k72rFG+jgaA8HvqVQlh0ldOdsWVNx4zgQhbDp2JjHhTHXp8QiFsFPorQTdz\\\\\\\/gAhaRtoNmcSgPfu4M6NRhoOCCmPqLIK5zRvHjvXBw8XpV5bMtTKKmLEmfPk\\\\\\\/GDRJ2h1UzvcJ0gc+ALTYrmO5oEzMRdZm75nqW5vTR6NlTjLnt+DRIUZb+pEoFAeGFa8WdX2JBf026fZH8XgAvxHjEyGZgdIzViRxsHbFewXK6NEzTNIiiTxAiKgDFZV4O+t0PXLC1Hn4tL2npJngH0vHvC0pFznnwj5iKmGZ2fd9sn+EJ1s0TyxWvmHKeGTsnAENA8K5LsOPiWoXlPyLo6mFwNQ3sPuJR4Hgfkw\\\\\\\/RbqZSOZ9GdDiABWKd6xrEf0\\\\\\\/gz4QHnuZ9N46x7WkSP5Cppsato9mU+ye7LwuBfm2jlYvhsIf1h7cAYlPsMZsPXEq20rL3XjvH+YKyfyY6tg3uM+c94GytUqxQHmYXcDov2+B+LBKAMXkKJXGkhtaoBuk5P2AoDtZuEPbgNVVqBhPjNIx257wg89hdw4jGzRsMMVKw4edaN4UME2pMNej49\\\\\\\/87VdS9snWCtcbcFvMVZH9d1Rt\\\\\\\/BtemS4VlneSZj73R1j8rJIWwLC7D\\\\\\\/0hSg0fbc1eyWDNLPnEJwfMKTdvEzw5DX+GhCEqB+Vanbny+Mj5pEf5V0ekuMUI+FuYdh7eY0l1OfXc0DAFZhvQfRszS+WLu0quUeo6\\\\\\\/4jXC+ycQ4UYzSpO2DZrrbJtlzwouCG9afByAKg2Veys7mb0hL6jIMfdfFZIJohCrIqoBZmbZd8H\\\\\\\/bhWn+husSon9ezfQW1cnm7bCQbbSbeTd4M8IzIO9W++j6IWNrql1jCY\\\\\\\/l7pwe79muxM25XQqxniv1FI6EECTTRGyo1R5zokiTLL006\\\\\\\/krTHBOzlUcwj3kT+Y8QX8QbRiUVc\\\\\\\/2gElxMlPvMATAGjrCBAmh6AmN\\\\\\\/at3jchipyaIExI2pQAvYH5IbLBAE4lyw5xwzb7SahQ0VF8V5h40lYivbcF6LWUTF9fx85mLbWZpQNVWTszpwPJQ9XKEeF5Zf+c2Bo+wz+FESRZeL+m2wxvDKqUXMqukOeDr8FMU6lYozqHEad2thIT8Uwk7qtjQLac498gcUnmuANLRmoK816fZaQGM9HTfeGcop4gRfxbKzeCe7VzhwRDdtl\\\\\\\/04ZYCcFwm61jGl35hfgug3DDeKkFSXT3kci\\\\\\\/kRZi\\\\\\\/i24DCjIpJ8Ahswvpo6bMfiZCd0ziEgurm6UK1drTQSwxqJbZod7vj7OYEdD0IDXQrkg+rjInFDI8S1NdCh6CxRl2Z9qA8tLIMEm1JL1d9CeZNSk9feuwXwNl3\\\\\\\/59AwJQXnxlPUNMwzZB8jh6BIhAphnAZmu77flyTOeK6syp3YK8HQYWPMpghQ0GCwfWHK4q\\\\\\\/XAJtN06GunjlCYQpC8hu0VYJYU\\\\\\\/NQ7V4e7To+6j+6dotAgJS43S9iB0sOqkqq66CCCMTs+FOqIkf+J318xyxF6ubcmM7vkpIfPb4tLlIepAhHsa1pmG3yUA5\\\\\\\/SJgwrOZOg0cU6GejhwVp+8WOjIcbTRrs0XcaxJYh8aasFSu4GRZA7uORzVZllMJ1vC6UglnkXvR0GaOjllmkpIXA1f+gg68twTFG3NmzA4E5cZ2Zvcf\\\\\\\/V5Wb9a+3kkLfuabCJ1sXRe7kfWhzaTKimUUDAodw32qjWjrgBOM+DXOFsq3OeD45vQexvxIl47wqyOGY7RS+hT8o+wiCXk6YIk9hyZlZosgD1VOeCYeojg9JPY7T+UGqZnT9rMC9S\\\\\\\/Vvj+zKuoidS2AXubrF1asqD1nZuTURJmyxEJ+yrO1AqdAb9xyH7wzkK5HCMAWw9TeL3N3cTPyf7I6O9PcuEvgJpLBUvykpyk2V20iOD0MfjNSP+7MKTUdJi4fea95C1tFiAv3gkjQjBLA+d5kkF\\\\\\\/IzCt6Amib+aT8gsxy5F5WydazXhmVyb1z0WAyO53xF4EmqBW3WIfg+SXdO59\\\\\\\/KgndsqFCeVsRa3jT1bHaXvmjCmAlezrog5p6r1h5j4KQZo10PXzUOUkf1L58UW3hI7SOodeLmBceVf2cdCY\\\\\\\/cNxmcPMBlvUy14it9FjZzzwasO7289U2MEmpSCbEscEF1EkslhKgDYWjq+bc2Hs9JW0M0h63rjSWGwy2vWMqEhEBJFsp+NyyK9y12lb+Dw1hKAZsgpxvbyhKPvoG4Fll\\\\\\\/Kps3q93GYw9PooUQRdzA64hwfLgWWrhlV9taA3sJVkHytD3FxI830X57wGXoMgs1uN2NLRZAwioOLNXrck74uVNW46WxKHjqLbQAyXF6m6Mn6TpcYTBVq8geES5RKhhLsf+yPZdPJYomjdRK4dzz8EJgyE3HsayVTC2ZXKReBEuN+GXZfrL\\\\\\\/SejJtKuwqESWDWlWP6tFt5aydJZp74QdrA\\\\\\\/o5y2H02RtzhrusKu8IT7DtD3IaC04SsPaLB3hYYFtD+3J0U+k\\\\\\\/BfbZFcQ\\\\\\\/nCUOySwGtA7STbxTMazYINke3TZNOwct7gprga92uEQz+tWYD5bhsy9aMu\\\\\\\/X55M8GaTFi26bs5IlMdnBEt3tKTjdJZPHtCHOrVoYNISwBUuudf2+IgIz2+XKlKpoBhAFXBp3XmjnZy2inBQgSJfH\\\\\\\/ts7RGrX9O0dXi8IvjZUgs2B2l1F5KFwr1Sy819FRPilHAL3isdedywH9+eZxVID9qp3ooV+lIeeRUJljNkdw2k7nadvxo8YAJzO5KiMGFS6ldz6fuYqDxVazcIjRzqEUNGgxC5pv0sjWhgOBhfss3wgFAIGQibO63J9FEw2GccP9K533aEq80k+PG1KwGDDX3kRZTt\\\\\\\/w9Qxufb5zFiuu2dIWV\\\\\\\/UjdwOpc5noDrcBsBfGBQnBXDxpgZ37wvcMoaUlyavJ0KaO09zSXvI7fhScBEkWNPUj7oTssyfwfTWKPTnnesuhUu8\\\\\\\/Z0UV2kRAiK\\\\\\\/S9Z7ibyqHHuHX+Z6AEKvt4xnqHtwpHYX2W1udu+Q+mPNrI55pHWtnDrFjbXaTeJA7JHP3acmJfMqvSSRpK5HqJ\\\\\\\/J346HCvRbT\\\\\\\/q8NFW0ltIlh6dn4+JPqiRj74JGDJWbnq3lSIXycPx6YMDyy8PpM1Zg3Z2UctNfCoC34N8e3Gw8c9selFZ3hDa743JdGlFvAgTsPKdieGbON3JkbXCNUJio5WXCUHhM1G4hWix+uYT0CN18M+aAkoES0vJkM5kL5d9S+15Ja1HMW7ZQxUGlms0Z\\\\\\\/KVRvSM2r+CUcGrRkDO0c+KjRzpWVPqBXLTnLGX9DohjCkO93N16nwoOf+A0ns4iDseE1e7BolCxCVFQvwhwXkH2YMf5Z9hmh4aHwpUON7dNnVRjF1Q8ZnN0uZTWAU0g\\\\\\\/IAhAuyMb17ldZaf7Y4jtoX6DffP5yR0StPTqM+OQs\\\\\\\/j9rD\\\\\\\/2gu2RV9KyZ3g9Ip2xfKVF13ccLECdzSlcQJKVh8xXaso6ovaSKe9QwZAwKKZd\\\\\\\/szMj7s0D0HQNM5tOqqhjqbeCWBzK\\\\\\\/U9VZ42LbjSzWmW7KfXyWdBtWL1EuWjk\\\\\\\/0NhZLV663EvTenDjc6\\\\\\\/EfFvtlDhmet3s\\\\\\\/bb8YymzD4KHpyAe7xq1WUfC2RW\\\\\\\/il2aQFlyQ7jh8bjyD2jqmkGI8HW0wQSUD56IbyK+WV5064vNAqER0pMsYUhJE9oCVX3qkYrFu7w0ypfMzziP56KEf7t0mvfpmbPzMbGkBV3lG8EYue\\\\\\\/LsYOVnLGOTDA4aSx6j9LeNetoA4DXroe0Fwg5wRt\\\\\\\/sxd1gSyPHo5j7Jgp+beUynu8Hci1WhuSPqG4gSwk0daOJTPjMM0kMbmMqkwUJlYBmGnluthq7LXP0N2PNOLL9BMOzLXCyJow2olZC8OESwtCr8Ibrs\\\\\\\/nqZgv2r1tafvF+VPyEfzhUDm85dw7lSWk\\\\\\\/m0+G7AbuazEKyWYBzSVi8rlemM8QN0KIrr\\\\\\\/JVkVUWvHg9MNfKBj4\\\\\\\/Hx6BkoPcn\\\\\\\/TsNh3ZY5KD0JeGP46GMYi6b4C\\\\\\\/\\\\\\\/Anh\\\\\\\/IiLEBSrTCgf80S8bM3e7BJaq9xu9GonkCkGw+xDtCF1x6v4aVRv1wGUSaoXbABQhYqpwqjWc+l1OsgjfaK6+0gY6wwy3dIPdDXYRWCUUaOh5foSP8si+gY2QmOqs2O7YwcejucLrw7D732pQ4N1HQjlkG6BGALxsgAZNtF2FnVmoWgO4W6aCBkcACFV7t+3rWuFqhL0DSh6U6u+PG9nj9JbDnTpHMBqa9naL+vq3Ci7l7nXJgPpHFKWqYkk0wOLC7FpBduicRfGmyQO29mH02UOZWpidXrLdRPzxKfJgsn+M\\\\\\\/4s0inlIfgCknuRd6Sz1AaXA6fScAHZ70+wTUJWQYjqiWmvQGBydIm2GsPbFWoOVblqH9hrSlE5z5PzdRXrC4zcBzzMTmGsxflunXZno\\\\\\\/+oVHXu+\\\\\\\/k+TJ\\\\\\\/j4ZhNrCz0xuBUGksq+lWHqq4pVpYCFsCJeN93E7ICU1GgJum0EsR1sXp8gzZ\\\\\\\/E6+Sd2RhQdUbXWIyx\\\\\\\/\\\\\\\/gzlHvMtZDFkpmHQtTGCcQPibFIKdGY68X+tAhO5GqBiQpce6Q4IQ5HEawFbh657cWtkyxWA3sy2ObIhhKJKMI2ML2sF+MdhRcCD+BUJNj0REQuBPWLWkb52IUtx02J761nT7aqPOhfXmL1tDJgIRB3E8kUCt+X1LsQvLz9RPP1gq\\\\\\\/Rnk8cbDaj1o3yg0v9KliR6UctPyw\\\\\\\/eeQm\\\\\\\/sdKwqSatTT4qgUFJ10vnTVNqBgQrX7ibRIvNeni9TM9Bwtbva08Pd3uTyuLmHWoM15KkDj10vWNGgx90Zxdq3efIM8yTwGAnBgckWyakUIj6S7L2LeKjr457uPghyAeroYRSvk+5Vd4mdtLkOw9aVL+gqNy1rcyzqf569JxZrYwE4cbSjw5dFN0UAxAJDq4yZm6G0MKOT5mdoPjWn20myTX8+MtbGRW2UNoI7QyyvNKL8OeqUDTcXWgmivYr9PupPwvaC1C5gTjjeiN1SI1+imnefidgH82E\\\\\\\/abQIiU04oB4Lnn3\\\\\\\/93Vpnd3l4WYNv3QYtoLuQZDG+l9mGrPLc1f0uYsOfvPy5gWUVwPjEP5qUcyNJIzYkptdyQ8Gpun2aHpCRNkd31sJBJdAau3\\\\\\\/2+zJcUpN40lz4A5NCVWrcLOZphycO4t3arlGVnOhsq621GNIqdsS9K8s\\\\\\\/kp5r\\\\\\\/wVHiMgqdxmO7hp3act60s3NqLzn0K05VylANXJpYEiogV0N3LK73Cwc9x4halLheYBJ6DB+khZZeEEcgmzp9NG1N\\\\\\\/sHwDXuaybjVxHBALkk4BERkFhTf5B\\\\\\\/5XWqY10zoE+2UDYIsiD6OZEbIFQab6BlCd56nIuj4PrCPkFWVCulba0OAo1tdCTkQNrraM6rl7CqQxzXOTZ9wUo8atlgYDri0x5pGY2WyxafVfG1sr158P7WYVa2+Q2CziDdQ7PbP\\\\\\\/e6jVRaN5YJnnXLEruH2HiON3cnG6copC9OjQHKp71EIHtJjC972cJhS7YCXqHwxpjaC9gR8C3mAjcHsnfZMcQLzFwWnEpRXaWVSAjIYHgm4AG9Rv71Ah2dQouVKxfyYFVlMAZCcoGFd+d3iMGg+eBbDliwhZGEi5M+dtZKxqkQEn2BR\\\\\\\/GyHTQ8eY+OOTyKwavzSoZLdVQfwKaQ2HTHDSd0UokYV7227V6tuatIZGXHtwqt0XfBbqFe20yxZvrLGuE1GuLjBXvATaK5AKBGUeoqU2cn2WLvuBbQSNAXkb3\\\\\\\/pnLrXvHkSkZV5u7UNWuBXbQEtCDndgDLkCsMJZ+iYEThela0gPMu9315xQMHnmSyiSAw6rpobziyPrGUCxSVnVYQU5pnCZF8d8YYQRcEkRlEldL9hTO6fPqwPeUIa7FkoLM+pp8f68B8yZ28X3cJdYluVwJepzLl9SKG7u1K3me93Cp2+Mf8ZVdgyZg8KoW6DDLKYAooUDcgbizaPOUYMbJfzpju5Tfr8wjCO\\\\\\\/rUgaUY6ru7Q9kJ85W1kRPAi3GXHp\\\\\\\/F\\\\\\\/rDQEjnFPw+w87WycW91348xa69BCRNWhMg0YNCOVJm6N09e9JZkVFjojZmgxa8dGxrcrdv3NqL26XK9CJHP0iVnqx2Lxly4Z1R3dOOB+SM0tiFtpLq\\\\\\\/G9QbKWU9V4SybadhiuoMa8MuyXytVtfHFWN9EW22nobF+8694W66Bs0r8f3xJ94SETlFQODgQflA\\\\\\\/0CUIpf30Y9AHpgjkikEI\\\\\\\/2SA5ArfjOBAz5Qui6mnLec\\\\\\\/\\\\\\\/8UwWuQeRMyga++on9tG4QhdawprWLqjNpZe9GSvUBfabQNK89dPRm\\\\\\\/VZtwvHXU37R748LIplpg31yi8zMKqvgJ1mwSV4QloqR47YuwQPluh7\\\\\\\/dgoP3zL94AjuPFjBafeJXhTSovLLRyUpjwk7KdWlR9T02U3hBytiEUU3CfyZ3H8N9epodClQZYehV4xOSMvTjH81Q0F4v55u0aegSHGmPafJ0JhKG97F2N2RXVcQJvfpI4apd0\\\\\\\/8nXWS92pbSapd2ZpLQ7l1d\\\\\\\/Cw\\\\\\\/gru4r0IpTWkS5543\\\\\\\/XtXurP\\\\\\\/8eFXaO9qaab5n\\\\\\\/+HmfJmgCUBgyg2Bo9u5dgAne+xYMh2eQkqP0uArCHUhfna5lDPXH\\\\\\\/ww3rbcRq7mI2td+wIBj2I79EfTZCFFSAryoa+3QjjfJyM3A==\\\u0022,\\\u0022iv\\\u0022:\\\u00222adA6uwONlYXEjndTWUkvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002284be9223a1b530ad39dce20f257d575669862f9a8e0d9a01da010ae16d762cc8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pZyd+13XD6Ut\\\\\\\/CTtZdZbv1ks5F6055yIQu2YO+n5WjyArTgEQRyk5umbcpq5AHv+kPHNPk\\\\\\\/1oRsZpb3xUiKIRoOeGGjypt2GN2XJmXjL+CFMzLRynoWeSPknl+rzifX2hntL6jT1uWQz3iQOTdxPnz9Rs05rLmNr92KXMY9M2KIA5fKZSeRJhX1EFQFYNwUvLtElwZwzGShXmkn8TaYkKtky+TiFgsVteFFdfNpLcSAdkbygw4hUOg2XGBOKd\\\\\\\/JdF98MUeaaZ\\\\\\\/QQ0eJdCuXDIIjkPrcS6HMZc+RausYQUlDoewaWjzsu3irUT2R+6HmZS0vDjPF+lQS8TFn6eVDjbsBUjTWJtG5WoNO+NVzdFEMobq7w1SCXb6eIYk85K477am514SXfI0R3c2M\\\\\\\/jKFLR+ttq74MmYvCBeoEqng\\\\\\\/b+UMytubUKf+D6OOJfQSUkwR4yDsMmvF1uR7gQvMny\\\\\\\/f+aj4Cljn4n26Ou6l+vgBtFACz6PuBs837yVTRzqORAVQwFGDhkV5C1yBDNJw91a+TSEGyr+XzlLhY\\\\\\\/OpG34WzrHELg9ggd3wd6dHCZCvAY8kz5zMjbMOEWKuALfTImvrFawUUqJGVbVgvRSlf0lbSUnNJqQxKlxRVtGiusw39wenfqcF8\\\\\\\/8wNdSFyKFQbJfRhsc9CqWDcrF5xxjwVnuaOBZqTQMfUGIYEQbwbslR848CBkE17XoigTtYqrAys4h4oi5ym0vDexorDclRT\\\\\\\/HsR29AC6bFNej5WZaurFWPbO8tq40AQFdupCkf6+Ot0JLKJPBkUBlz5wjFlCk4\\\\\\\/pnV2+oYIz2704VSmRtuXuU4lpwCpsnFiGCakigIIbZOmPCF0pbhx3yF69zhPKQhXFdJzQpj\\\\\\\/4ZiXg1m5ZM9bxZg9ktT2Sx\\\\\\\/UkOLPJJ6AwqnuwyAZBo0UYUqeEt8Qqj\\\\\\\/wEohzRyZj6jSMqeuXkYm\\\\\\\/UF8VXwq8bqVOxS9xEge\\\\\\\/e9dVyMw9iI8Wngg1gd7oUFz+dmNvJR8xCjNARUagUL7mF8fAdHRWQVI2JM6yc5wccSPilyMgrNC6DbhG+dc5W\\\\\\\/hwhII0CpmhzFKc3KA3TZ0wHh1hC7abI3h4WgY1S\\\\\\\/wLnND81milVTL3mvc8vtL1MsMp+unA5U+mQcoKXe5tQTuJJKd3QcGhIwaPme2j5d3\\\\\\\/Ka3DJ3nSlknyUB9ig9DpJu0bIOPG64BiO0cPxg5lO7Es+qK5UKXEtuYC2fgzz3ecoRXF0G1ODGjGlDKlvjZ57Gbg9SK1HRPc8KRs3atP0LtAc7ZbR5rfDFFF3nrxiKRh2ADzo+DDgCdM7p7JkCOTemAYm99bE2J1U4r\\\\\\\/nIHMVk+clYUXAhcp0wUd252hSD0BP0q4H0tTnYUx70ZCQUJ3HD8j55w+XfriRpOmDJnb3mKlm0mh78\\\\\\\/swxMAAU96Qvfq5bNgmnJp9jejQIkzxBqwTEh\\\\\\\/9vDNi95KgzHrcaFVGHcX90OYl1Jx4fQQgYQ2TezRDjylUqeAmtVSEiHr+9oi2p7R43XWRrTV6PkplTp4F43TRMunjFTQMetAc0OcG6AAKrbmJdCOCOXLu1RKiU9sMcPPU1k\\\\\\\/gQpY0G8QcAmInRamaJ7XaI9agkVhc2WsonD6GfBFErfRQN3tTBEa62fF1EFIZn+HH21L5Y\\\\\\\/B0eOZFuZJWa4sjz6BY5htbkkdUXZt9UraEk8d931DoCJbTc9S4aFoY8qcsinwS3\\\\\\\/9+mTeVm7jrLhWSDWHH1yaoGNGAENTvz\\\\\\\/lH2gAjbap1xcZQN25RFh3Y9rCvQgXQQGAYn5DefTK37iVlsebHLT71FL4TWTx71dSRT80yR4xvzsfBn2o8IiPc9UmVEr4snNt5xPwyLCko7UHcs5u+2oHxG8G3367UOwRz5qewVEJGKS0Y7OTox9hN+MSEF8B1ZMBydWehBTzVuqajgvPEslWoZuwkcnHuGdmOR3tFvlyM8oHf6w6tWphkEJaN0YIGHf9LWYpFH+GdZ56tDHNDgXwSqGp\\\\\\\/m\\\\\\\/5t+lepbcpg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ff36\\\\\\\/EsEywNu6pB+I1\\\\\\\/Igg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273f52047d62f0e414a54c99809651c050b6b4a6a6e553c1093c5f95f74409f8c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483844 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1301" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ttJZ6sdkbyM4JtZdbBhdXdN_gdFvaUAnSfUG8yRaxfo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1301" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iOvrp0Bsr-6aQknoNZYyyhT7yPr4N9bRWUnCzW7UXH4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1287" title="Independent Maternal and Fetal Genetic Effects on Midgestational Circulating Levels of Environmental Pollutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1315" title="A Nonsense Variant in the ST14 Gene in Akhal-Teke Horses with Naked Foal Syndrome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483844" id="node483844" data-pisa="ggg;7/4/1301" data-pisa-master="ggg;g3.116.038448" data-apath="/ggg/7/4/1301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7384-5181" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre-Yves Dupont</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1936-0236" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Murray P. Cox</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1301-1314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038448 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483844" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pierre-Yves Dupont </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Statistics and Bioinformatics Group, Institute of Fundamental Sciences, Massey University, Palmerston North 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre-Yves%2BDupont%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dupont%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre-Yves%2BDupont%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7384-5181" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre-Yves Dupont</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Murray P. Cox </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murray%2BP.%2BCox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cox%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMurray%2BP.%2BCox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1936-0236" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Murray P. Cox</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.p.cox@massey.ac.nz" class="" data-icon-position="" data-hide-link-title="0">m.p.cox@massey.ac.nz</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/4/1301" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483844" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483844/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1301.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483844" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483844/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1301.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483844" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483844/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1301.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483844" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483844/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/4/1301.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483844" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483844/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Lateral gene transfer (LGT, also known as horizontal gene transfer), an atypical mechanism of transferring genes between species, has almost become the default explanation for genes that display an unexpected composition or phylogeny. Numerous methods of detecting LGT events all rely on two fundamental strategies: primary structure composition or gene tree/species tree comparisons. Discouragingly, the results of these different approaches rarely coincide. With the wealth of genome data now available, detection of laterally transferred genes is increasingly being attempted in large uncurated eukaryotic datasets. However, detection methods depend greatly on the quality of the underlying genomic data, which are typically complex for eukaryotes. Furthermore, given the automated nature of genomic data collection, it is typically impractical to manually verify all protein or gene models, orthology predictions, and multiple sequence alignments, requiring researchers to accept a substantial margin of error in their datasets. Using a test case comprising plant-associated genomes across the fungal kingdom, this study reveals that composition- and phylogeny-based methods have little statistical power to detect laterally transferred genes. In particular, phylogenetic methods reveal extreme levels of topological variation in fungal gene trees, the vast majority of which show departures from the canonical species tree. Therefore, it is inherently challenging to detect LGT events in typical eukaryotic genomes. This finding is in striking contrast to the large number of claims for laterally transferred genes in eukaryotic species that routinely appear in the literature, and questions how many of these proposed examples are statistically well supported.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/lateral-gene-transfer" class="hw-term hw-article-keyword hw-article-keyword-lateral-gene-transfer" rel="nofollow">lateral gene transfer</a></li><li class="kwd"><a href="/keyword/horizontal-gene-transfer" class="hw-term hw-article-keyword hw-article-keyword-horizontal-gene-transfer" rel="nofollow">horizontal gene transfer</a></li><li class="kwd"><a href="/keyword/automated-detection-methods" class="hw-term hw-article-keyword hw-article-keyword-automated-detection-methods" rel="nofollow">automated detection methods</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/eukaryotes" class="hw-term hw-article-keyword hw-article-keyword-eukaryotes" rel="nofollow">eukaryotes</a></li><li class="kwd"><a href="/keyword/fungi" class="hw-term hw-article-keyword hw-article-keyword-fungi" rel="nofollow">fungi</a></li></ul><p id="p-3">LGT (also known as horizontal gene transfer) is an atypical mechanism of transmitting genes, not from one generation to the next within a single species but rather horizontally between different species (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gogarten and Townsend 2005</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Keeling and Palmer 2008</a>). Although a less common mode of inheritance, this process has been known since as early as 1969, when mutations in the galactose operon of <em>Escherichia coli</em> were found to be caused by the insertion of mobile DNA elements (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Shapiro 1969</a>). Today, substantial evidence has accumulated for LGT in prokaryotes, and LGT has been shown to impose a strong evolutionary force on prokaryotic genomes, including through the transfer of pathogenicity-related genes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Gyles and Boerlin 2014</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kay <em>et al.</em> 2002</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Furuya and Lowy 2006</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Martínez 2008</a>). Importantly, however, other studies suggest that the role of LGT in prokaryotes may be overrated (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Galtier 2007</a>). More recent studies have also revealed evidence for LGT in eukaryotic genomes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alsmark <em>et al.</em> 2009</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Nikolaidis <em>et al.</em> 2014</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Richards <em>et al.</em> 2009</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Soanes and Richards 2014</a>), and in some cases even between species from different kingdoms or domains (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Marcet-Houben and Gabaldón 2010</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Liu <em>et al.</em> 2011</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Monier <em>et al.</em> 2009</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Richards <em>et al.</em> 2006</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brown 2003</a>). This is despite earlier skepticism about the presence or extent of LGT in eukaryotes, likely due to a poor understanding of the molecular mechanisms involved (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Sprague 1991</a>; <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Syvanen 1994</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Keeling and Palmer 2008</a>). Nevertheless, the extent of LGT in eukaryotes is unknown, and questions are increasingly being raised over whether it has been overestimated (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Danchin 2016</a>). LGT in eukaryotes has been particularly well studied in fungi (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Rosewich and Kistler 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fitzpatrick 2012</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Szöllősi <em>et al.</em> 2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em> 2008</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Moran <em>et al.</em> 2012</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ma <em>et al.</em> 2010</a>), with many examples of LGT described between fungi and their plant hosts (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Nikolaidis <em>et al.</em> 2014</a>; <a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Richards <em>et al.</em> 2009</a>; <a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">Soanes and Richards 2014</a>). While most early LGT studies focused on only one or a few genes, typically relying on extensive manual annotation, the wealth of genomic data now available is making whole-genome analyses of potential LGT events possible (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Mallet <em>et al.</em> 2010</a>; <a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Wisecaver <em>et al.</em> 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cheeseman <em>et al.</em> 2014</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ku <em>et al.</em> 2015</a>). These analyses rely on two different classes of methods: composition- and phylogeny-based approaches.</p><p id="p-4">Composition-based methods (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cortez <em>et al.</em> 2009</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Menigaud <em>et al.</em> 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Jaron <em>et al.</em> 2014</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Rogul <em>et al.</em> 1965</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Lawrence and Ochman 1997</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Nakamura <em>et al.</em> 2004</a>; <a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Rosewich and Kistler 2000</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Marcet-Houben and Gabaldón 2010</a>) focus on patterns in the primary structure of genes and genome sequences, and aim to find genes or genomic regions with composition patterns that differ significantly from the rest of the genome. These composition differences are often then attributed to an LGT event. A key advantage of these methods is that they do not require additional information about other organisms outside the study species, including the source species of the LGT event. However, where this extra information is available, the composition of the potentially transferred gene can be compared with the composition of other genomes to infer a putative origin of the laterally transferred gene (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Menigaud <em>et al.</em> 2012</a>). The key disadvantage of these methods is that they are sensitive to changes within the genome caused by common evolutionary features other than LGT, such as repetitive elements, AT- or GC-rich isochores, and pathogenicity or symbiosis islands, all of which display different compositions to the rest of the genome. In contrast, phylogeny-based methods (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Weyenberg <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abby <em>et al.</em> 2010</a>; <a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">Zhaxybayeva 2009</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Poptsova 2009</a>) are generally thought to be more accurate, especially for detecting transfers between closely related species (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Koski <em>et al.</em> 2001</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Poptsova and Gogarten 2007</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Poptsova 2009</a>). However, these methods are much more computationally intensive because they require orthology predictions, sequence alignments, and phylogeny reconstructions. They work by detecting a gene phylogeny that is statistically different from either the species phylogeny and/or the majority of other gene phylogenies. Some use a scoring system to rate phylogenetic events and employ a parsimony analysis to infer LGT <em>vs.</em> gene loss/gene duplication (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bansal <em>et al.</em> 2012</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Stolzer <em>et al.</em> 2012</a>). Phylogeny-based methods are highly dependent on the quality of the gene models, multiple sequence alignments, and phylogeny reconstruction (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beiko and Ragan 2009</a>). Unfortunately, the results of different composition- and phylogeny-based methods generally do not overlap (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Ragan 2001</a>), leading to growing suspicion about the accuracy of these results (<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Than <em>et al.</em> 2007</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Danchin 2016</a>), especially since most LGT events could potentially be explained by other molecular mechanisms (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Morel <em>et al.</em> 2015</a>). However, both classes of methods can at least be used to generate hints about potential LGT cases, which might then be confirmed using extensive manual annotation in further studies (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ambrose <em>et al.</em> 2014</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cheeseman <em>et al.</em> 2014</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brown 2003</a>).</p><p id="p-5">To exchange genetic material, two species must live close by, often in the same environmental niche. Therefore, in this study, we focus on possible exchanges between plant-associated fungi using the well-studied beneficial fungal endophyte <em>Epichloë festucae</em> as a reference point (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dupont <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Eaton <em>et al.</em> 2015</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Schardl <em>et al.</em> 2013</a>). Plant-associated fungi frequently cooccur, have likely physically interacted over evolutionary timescales, and therefore seem like reasonable candidates to have exchanged genes via LGT. Importantly, we use genome-scale data typical of that available today, with all its various faults from automated genome assembly, gene calling, and gene annotation. We compare both composition- and phylogeny-based methods for their ability to detect novel and proposed LGT events in fungi, and reveal new statistical insight into the pitfalls of these methodologies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data description</h3><p id="p-6"><em>E. festucae</em> gene models were downloaded from the University of Kentucky Endophyte Database (<a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Schardl <em>et al.</em> 2013</a>). Version 6 of the EfM3 protein and gene (transcript) models were used. Sequences for all other fungal and oomycete species used in this analysis were downloaded from the JGI MycoCosm database (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Grigoriev <em>et al.</em> 2014</a>). The species were chosen to avoid genus redundancy. Lower quality genomes were excluded based on sequencing read coverage, number of scaffolds, and sizes of the longest scaffolds, with the exception of the endophytes, which were all retained. These species include: symbiotic endophytic fungi [<em>Daldinia eschscholtzii</em>, <em>Rhodotorula graminis</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Firrincieli <em>et al.</em> 2015</a>), <em>Xylona heveae</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gazis <em>et al.</em> 2016</a>)], mycorrhizal fungi [<em>Cenococcum geophilum</em>, <em>Choiromyces venosus</em>, <em>Cortinarius glaucopus</em>, <em>Gyrodon lividus</em>, <em>Hebeloma cylindrosporum</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kohler <em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dore <em>et al.</em> 2015</a>), <em>Laccaria bicolor</em> (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Martin <em>et al.</em> 2008</a>), <em>Meliniomyces bicolor</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Grelet <em>et al.</em> 2009</a>), <em>Oidiodendron maius</em> (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Kohler <em>et al.</em> 2015</a>), <em>Paxillus involutus</em> (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Kohler <em>et al.</em> 2015</a>), <em>Pisolithus tinctorius</em> (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Kohler <em>et al.</em> 2015</a>), <em>Terfezia boudieri</em>, <em>Tuber melanosporum</em> (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Martin <em>et al.</em> 2010</a>), and <em>Wilcoxina mikolae</em>)] and plant pathogens [<em>Bipolaris sorokiniana</em> (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Ohm <em>et al.</em> 2012</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Condon <em>et al.</em> 2013</a>), <em>Blumeria graminis</em> (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Spanu <em>et al.</em> 2010</a>), <em>Botryosphaeria dothidea</em>, <em>Botrytis cinerea</em> (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Staats and van Kan 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Amselem <em>et al.</em> 2011</a>), <em>Cercospora zeae-maydis</em>, <em>Colletotrichum graminicola</em> (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">O’Connell <em>et al.</em> 2012</a>), <em>Cronartium quercuum</em>, <em>Didymella exigua</em>, <em>Fomitiporia mediterranea</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Floudas <em>et al.</em> 2012</a>), <em>Fusarium graminearum</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cuomo <em>et al.</em> 2007</a>), <em>Leptosphaeria maculans</em> (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Rouxel <em>et al.</em> 2011</a>), <em>Magnaporthe oryzae</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dean <em>et al.</em> 2005</a>), <em>Mixia osmundae</em> (<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Toome <em>et al.</em> 2014</a>), <em>Passalora fulva</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">de Wit <em>et al.</em> 2012</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Ohm <em>et al.</em> 2012</a>), <em>Phaeosphaeria nodorum</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hane <em>et al.</em> 2007</a>), <em>Sclerotinia sclerotiorum</em> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Amselem <em>et al.</em> 2011</a>), <em>Setosphaeria turcica</em> (<a id="xref-ref-80-3" class="xref-bibr" href="#ref-80">Ohm <em>et al.</em> 2012</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Condon <em>et al.</em> 2013</a>), <em>Sporisorium reilianum</em> (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Schirawski <em>et al.</em> 2010</a>), <em>Verticillium alfalfae</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Klosterman <em>et al.</em> 2011</a>), and <em>Zopfia rhizophila</em>]. Transcript sequences were downloaded for all species, also from the JGI database. Protein models for <em>Phytophthora sojae</em> (<a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Tyler <em>et al.</em> 2006</a>), an oomycete, were used as outgroups to root the gene trees. <em>Pseudomonas fluorescens</em> sequences were downloaded from the NCBI Gene database (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Redondo-Nieto <em>et al.</em> 2012</a>).</p></div><div id="sec-3" class="subsection"><h3>Orthology prediction</h3><p id="p-7">Gene orthologies were predicted using the Reciprocal Best Blast Hit (RBBH) method because it has been shown to have the lowest false-positive error rate (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Salichos and Rokas 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Huerta-Cepas <em>et al.</em> 2007</a>). Transcripts from the EfM3 gene models were used for <em>E. festucae</em>, the gene models from <em>Ps. fluorescens</em> were downloaded from the NCBI Gene database, and all the other gene models were retrieved from the JGI database. A blastn database was built for each of the comparison species, including <em>E. festucae</em>, using transcript sequences (excluding UTRs). The best blastn hit (<em>E</em>-value ≤ 1 × 10<sup>−5</sup>) was recovered for each gene in each species. Blastn was used to avoid problems with misannotated intron–exon boundaries. By definition, to be accepted as a reciprocal best blast hit, if gene G1 from one species is the closest hit of gene G2 in another species, the same gene G2 must be the closest blast hit of G1.</p></div><div id="sec-4" class="subsection"><h3>Gene composition</h3><p id="p-8">The percentage of GC content (GC) and the percentage of GC on the third base of each codon (GC3) were computed on the coding sequences with a custom Python script. The same script performed the computation of the Codon Adaptation Index (CAI) using the formula of <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Sharp and Li (1987)</a>. The effective number of codons (ENC), a measure of codon usage bias that has received attention more recently (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Sun <em>et al.</em> 2013</a>), was computed using the method of <a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Wright (1990)</a>.</p></div><div id="sec-5" class="subsection"><h3>Phylogenetic analysis pipeline</h3><p id="p-9">For the protein phylogenies, the sequences of each orthology group were aligned using MAFFT v7.058b (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Katoh <em>et al.</em> 2002</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Katoh and Standley 2013</a>) with the set of parameters <em>linsi</em>. Automatic trimming of the alignments was then performed using trimAl v1.2rev59 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Capella-Gutierrez <em>et al.</em> 2009</a>). The phylogeny was built using PhyML v20120412 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Guindon <em>et al.</em> 2010</a>) using the model LG (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Le and Gascuel 2008</a>) with 100 bootstrap replicates. The species tree was reconstructed using the Subtree Prune-and-Regraft distance (SPR) supertree method (v1.2.0) (<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Whidden <em>et al.</em> 2014</a>) using all 1768 genes that had predicted orthology with <em>P. sojae</em>, the oomycete outgroup used to root the tree. A bootstrap test was performed to assess the robustness of the tree. A sample of trees was taken with replacement from the initial gene tree population and a SPR supertree was generated for each sample. This procedure was repeated 500 times. The bootstrap values of each branch correspond to the percentage of trees containing the given branch.</p><p id="p-10">To infer potential LGT events, the maximum likelihood value reported by PhyML for each gene tree was compared to the maximum likelihood value of the phylogeny for the same gene with its topology constrained by the species tree topology.</p></div><div id="sec-6" class="subsection"><h3>Phylogenetic-based LGT prediction methods</h3><p id="p-11">Three different software packages were used, all based on the evaluation of the most parsimonious Deletion Transfer Loss (DTL) scenario: Ranger-DTL v1.0 (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bansal <em>et al.</em> 2012</a>), Notung v2.8.1.7 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2000</a>; <a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Stolzer <em>et al.</em> 2012</a>) and ecceTERA v1.2.4 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Jacox <em>et al.</em> 2016</a>). These methods use scores for each of the three possible events: that is, deletion, transfer, and loss. The default scores were used for deletion and loss, and a range of scores were trialed for transfer, from 3 (the default value) to 30. All three methods were run on all trees containing <em>P. sojae</em> as an outgroup.</p></div><div id="sec-7" class="subsection"><h3>Computation of tree distances</h3><p id="p-12">Only bipartitions with a bootstrap score ≥75% were used for the distance comparison. The tree distances were computed using the Robinson and Foulds (RF) algorithm (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Robinson and Foulds 1981</a>) using the normalized RF distance implemented in the ETE toolkit for Python (v2.3.7) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Huerta-Cepas <em>et al.</em> 2010</a>). To compute phylogenetic distances between two trees, both were rooted using the same outgroup species. The bootstrap cut-off introduced polytomies in a small number of trees, which were resolved using the automatic method provided by the ETE toolkit.</p></div><div id="sec-8" class="subsection"><h3>E. festucae gene clustering</h3><p id="p-13">The transcript sequences of <em>E. festucae</em> were clustered using the Mean Shift algorithm (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Comaniciu and Meer 2002</a>) implemented in the <em>scikit-learn</em> (v0.16.1) library for Python (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Pedregosa <em>et al.</em> 2011</a>) based on their GC, GC3, CAI, ENC, and tetranucleotide frequencies. The bandwidth was automatically determined using the method <em>estimate_bandwidth</em> available in <em>scikit-learn</em>. As the Mean Shift algorithm is sensitive to the shape of the data distributions (<em>i.e.</em>, it was built to discover “blobs”), another clustering algorithm from <em>scikit-learn</em>, DBSCAN (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ester <em>et al.</em> 1996</a>), was also used. This algorithm considers clusters as areas of high density separated by areas of low density.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-14">All data are freely available from public databases. Accession numbers are indicated in Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>. </p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Study system</h3><p id="p-15">While claims of LGT are common in the literature (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Gogarten and Townsend 2005</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Keeling and Palmer 2008</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Shapiro 1969</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Gyles and Boerlin 2014</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Kay <em>et al.</em> 2002</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Furuya and Lowy 2006</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-105-2" class="xref-bibr" href="#ref-105">Sprague 1991</a>; <a id="xref-ref-109-2" class="xref-bibr" href="#ref-109">Syvanen 1994</a>; <a id="xref-ref-92-3" class="xref-bibr" href="#ref-92">Rosewich and Kistler 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Fitzpatrick <em>et al.</em> 2008</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Marcet-Houben and Gabaldón 2010</a>), comprehensive studies suggest that the power of automated methods to detect LGT events remain limited (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Ragan 2001</a>; <a id="xref-ref-114-2" class="xref-bibr" href="#ref-114">Than <em>et al.</em> 2007</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Danchin 2016</a>). This study aims to investigate whether it is possible to predict LGT events accurately in eukaryotic systems, specifically for real genomic data from fungi, given all its associated sources of noise and error. This study focused on protein-coding genes from 37 fungal species (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>) that form associations with plants (20 pathogens, 13 mycorrhiza, and four endophytes), plus one pathogenic oomycete species, <em>P. sojae</em>, used as an outgroup. The aim was to focus on species that live in a common environmental niche (the plant host), thus increasing the possibility that LGT events may have occurred between them. As is typical in modern genomic research, the genes and coding sequences studied here have mostly been automatically predicted, with little to no manual curation. Orthology relationships were determined using the RBBH method, which has been shown to offer a good balance between sensitivity and specificity in contrast to more computationally intensive algorithms (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Salichos and Rokas 2011</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Huerta-Cepas <em>et al.</em> 2007</a>). The species tree presented in this study (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) was reconstructed using the SPR supertree method (<a id="xref-ref-120-2" class="xref-bibr" href="#ref-120">Whidden <em>et al.</em> 2014</a>) from all gene trees that included an outgroup sequence from <em>P. sojae</em> (<em>n</em> = 1768). Importantly, this species tree exactly matches previously published phylogenies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Fitzpatrick <em>et al.</em> 2006</a>; <a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Brown 2003</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">James <em>et al.</em> 2006</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Liu <em>et al.</em> 2006</a>) and is also consistent with the traditional fungal taxonomy available on the NCBI taxonomy database.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cladogram of the plant-associated fungal species considered in this study. This tree represents the phylogenetic relationships between 37 plant-associated fungal species that span the fungal tree of life, together with an oomycete outgroup. It contains seven classes of fungi (Sordariomycetes, Leotimycetes, Dothideomycetes, Xylonomycetes, Pezizomycetes, Agaricomycetes, and Ustilagomycetes) and one Oomycetes species. Subphylum Pucciniomycotina represents three more classes of fungi: Microbotryomycetes (R. graminis), Mixiomycetes (Mi. osmundae), and Pucciniomycetes (C. quercuum). It covers the two main fungal phyla: Basidiomycota and Ascomycota. Our reference species, E. festucae, belongs to the same order (Hypocreales) as its closest neighbor in the tree, F. graminearum. The Oomycete P. sojae is a nonfungal organism used as an outgroup in the phylogeny reconstruction. Pathogenic species are represented in red, mycorrhizas in blue, and endophytes in green. Asterisks indicate the species at laddered evolutionary distances that are compared to E. festucae in Figure 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cladogram of the plant-associated fungal species considered in this study. This tree represents the phylogenetic relationships between 37 plant-associated fungal species that span the fungal tree of life, together with an oomycete outgroup. It contains seven classes of fungi (Sordariomycetes, Leotimycetes, Dothideomycetes, Xylonomycetes, Pezizomycetes, Agaricomycetes, and Ustilagomycetes) and one Oomycetes species. Subphylum Pucciniomycotina represents three more classes of fungi: Microbotryomycetes (&lt;em&gt;R. graminis&lt;/em&gt;), Mixiomycetes (&lt;em&gt;Mi. osmundae&lt;/em&gt;), and Pucciniomycetes (&lt;em&gt;C. quercuum&lt;/em&gt;). It covers the two main fungal phyla: Basidiomycota and Ascomycota. Our reference species, &lt;em&gt;E. festucae&lt;/em&gt;, belongs to the same order (Hypocreales) as its closest neighbor in the tree, &lt;em&gt;F&lt;/em&gt;. &lt;em&gt;graminearum&lt;/em&gt;. The Oomycete &lt;em&gt;P. sojae&lt;/em&gt; is a nonfungal organism used as an outgroup in the phylogeny reconstruction. Pathogenic species are represented in red, mycorrhizas in blue, and endophytes in green. Asterisks indicate the species at laddered evolutionary distances that are compared to &lt;em&gt;E. festucae&lt;/em&gt; in Figure 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F1.medium.gif" width="248" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/4/1301/F1.medium.gif" width="248" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Cladogram of the plant-associated fungal species considered in this study. This tree represents the phylogenetic relationships between 37 plant-associated fungal species that span the fungal tree of life, together with an oomycete outgroup. It contains seven classes of fungi (Sordariomycetes, Leotimycetes, Dothideomycetes, Xylonomycetes, Pezizomycetes, Agaricomycetes, and Ustilagomycetes) and one Oomycetes species. Subphylum Pucciniomycotina represents three more classes of fungi: Microbotryomycetes (<em>R. graminis</em>), Mixiomycetes (<em>Mi. osmundae</em>), and Pucciniomycetes (<em>C. quercuum</em>). It covers the two main fungal phyla: Basidiomycota and Ascomycota. Our reference species, <em>E. festucae</em>, belongs to the same order (Hypocreales) as its closest neighbor in the tree, <em>F</em>. <em>graminearum</em>. The Oomycete <em>P. sojae</em> is a nonfungal organism used as an outgroup in the phylogeny reconstruction. Pathogenic species are represented in red, mycorrhizas in blue, and endophytes in green. Asterisks indicate the species at laddered evolutionary distances that are compared to <em>E. festucae</em> in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">For many analyses, it is useful to view LGT from the perspective of a reference species. In this study, <em>E. festucae</em>, a natural symbiotic endophyte of the cool season grass <em>Festuca rubra</em>, was used (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Schardl 2001</a>). This organism is very well studied and used extensively as a model system to explore fungal–plant interactions (<a id="xref-ref-98-3" class="xref-bibr" href="#ref-98">Schardl <em>et al.</em> 2013</a>; <a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Tanaka <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Christensen <em>et al.</em> 2008</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Dupont <em>et al.</em> 2015</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Eaton <em>et al.</em> 2015</a>). Because <em>E. festucae</em> is the primary model organism in our laboratory, we understand its genes and genome particularly well, which has benefited the interpretation of several analyses presented here. Five other fungal species were chosen as secondary reference points based on their laddered evolutionary distances from <em>E. festucae</em>. From the closest to the furthest, these species are <em>F. graminearum</em>, <em>O. maius</em>, <em>M. oryzae</em>, <em>S. turcica</em>, and <em>L. bicolor</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-12" class="subsection"><h3>Assessment of previously described LGT event in E. festucae</h3><p id="p-18">The <em>fitD</em> gene, which encodes an insect toxin, has been reported to be a putatively laterally transferred gene from <em>Pseudomonas</em> bacteria to <em>Epichloë</em> endophytes (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ambrose <em>et al.</em> 2014</a>). A blast and HMMER search on the UniProtKB database shows that this gene is also present in <em>Phlebiopsis gigantea</em>, a fungus from the Agaricomycetes class that is phylogenetically very distant from <em>E. festucae</em> with a much higher <em>E</em>-value than the <em>Pseudomonas</em> sequences. It is also found in <em>Aspergillus flavus</em> (Eurotiomycetes), in different <em>Metarhizium</em> species (Sordariomycetes), and in <em>Neonectria ditissima</em> (Sordariomycetes), all with significant <em>E</em>-values (&lt; 1 × 10<sup>−12</sup>). It is important to consider that <em>fitD</em> may not have been directly transferred from bacteria into <em>Epichloë</em> due to its presence in many phylogenetically distant fungal clades. It is interesting to note that the closest fungal <em>fitD</em> hit is from <em>Ph. gigantea</em>, which is phylogenetically more distant from <em>E. festucae</em> than the other fungal hits, suggesting that <em>fitD</em> may have been transferred into <em>Epichloë</em> from <em>Ph. gigantea</em> rather than directly from bacteria. Indeed, it is possible that <em>fitD</em> was lost in most fungal species and may not have been laterally transferred at all. Nevertheless, <em>fitD</em> is included in many analyses here as a reference point, as it is the only LGT gene claimed to date in an <em>Epichloë</em> species.</p></div><div id="sec-13" class="subsection"><h3>Composition-based methods</h3><p id="p-19">Composition-based methods are used to find genes whose sequence compositions differ from the rest of the genome. They do not require any knowledge about the phylogeny of the studied organism, and have a long history of use for characterizing potential LGT events (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Rogul <em>et al.</em> 1965</a>), especially in prokaryotes (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Lawrence and Ochman 1997</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Garcia-Vallve <em>et al.</em> 2000</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Nakamura <em>et al.</em> 2004</a>) and more recently in fungi (<a id="xref-ref-92-4" class="xref-bibr" href="#ref-92">Rosewich and Kistler 2000</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Fitzpatrick 2012</a>; <a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Marcet-Houben and Gabaldón 2010</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Jaron <em>et al.</em> 2014</a>). Our study aims to evaluate the efficiency of composition-based methods for detecting LGT in fungal genomes.</p><p id="p-20">Four commonly used metrics were employed to characterize the coding sequences of <em>E. festucae</em>: GC, GC3, the CAI, and the ENC. For GC, GC3, and CAI, metrics can vary between 0 and 1. The values of these four metrics in <em>E. festucae</em> are presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Summary statistics of these metrics are shown in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>. In order to be detected, LGT genes must display a composition value that is significantly different from the rest of the gene set of the organism, which will result in the gene falling into one of the two tails of the distribution curves. The wider the genomic distribution (as for GC3; range = 0.84; Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>), the harder it is to detect an LGT event amid the noise of the distribution. The composition values for genes found only in <em>Epichloë</em> spp. and not in other fungal species are plotted separately in gray (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Genes from species absent from the NCBI nonredundant database also fall into this category. It was important to separate these “orphan” genes from the rest of the gene set as the chance of them being misannotated, given their lack of orthologs, is much higher. However, as the curves for <em>Epichloë</em>-unique and -nonunique genes overlap, it appears that the unique genes do not have clear differences in composition compared to the rest of the genome. LGT events were simulated in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> by comparing composition values for 10 randomly chosen genes from <em>L. bicolor</em> and <em>F. graminearum</em> with the distributions of <em>E. festucae</em> genes. Strikingly, it is not possible to distinguish most of the gene differences, even for genes from <em>L. bicolor</em>, the most distant species. The values for the <em>fitD</em> gene (GC = 0.53, GC3 = 0.49, CAI = 0.79, and ENC = 59) are also not different from the rest of the <em>E. festucae</em> genes. To further investigate the potential ability of these metrics to predict LGT events, outlier genes that fell within the peripheral 2.5% of each side of the distribution curves were analyzed manually. Blast analysis against the nonredundant NCBI database was used to examine homology to genes from other species. In many cases, homologs were identified in other closely related species. When a protein function was available for the homologous sequences, no obvious reason for considering the gene as a potential LGT event could be found (<em>e.g.</em>, many were key housekeeping genes involved in primary metabolism with orthologs in closely related species), although this obviously does not allow us to dismiss the possibility of an LGT event completely.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of CAI, ENC, GC, and GC3 in E. festucae coding sequences. Genes present only in Epichlo&#xEB; species (potentially not real genes or secondary metabolism genes with fast evolution, for example) are shaded gray. All other genes are shown in white. The values for three housekeeping genes (tefA, tuba, and actA), often used for reconstructing phylogenies and employed here as examples of putatively nonlaterally transferred genes, are shown by green bars under each graph. The values for 10 randomly drawn genes from F. graminearum and L. bicolor are represented, respectively, by blue and red bars. All graphs have the same ordinate axes. CG, GC3, and CAI are plotted using the same abscissa axes. actA, actin; CAI, codon adaptation index; ENC, effective number of codons; GC, GC content; GC3, GC content on the third position of the codons; tefA, translational elongation factor 1&#x3B1;; tubA, &#x3B1;-tubulin gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of CAI, ENC, GC, and GC3 in &lt;em&gt;E. festucae&lt;/em&gt; coding sequences. Genes present only in &lt;em&gt;Epichlo&#xEB;&lt;/em&gt; species (potentially not real genes or secondary metabolism genes with fast evolution, for example) are shaded gray. All other genes are shown in white. The values for three housekeeping genes (tefA, tuba, and actA), often used for reconstructing phylogenies and employed here as examples of putatively nonlaterally transferred genes, are shown by green bars under each graph. The values for 10 randomly drawn genes from &lt;em&gt;F. graminearum&lt;/em&gt; and &lt;em&gt;L. bicolor&lt;/em&gt; are represented, respectively, by blue and red bars. All graphs have the same ordinate axes. CG, GC3, and CAI are plotted using the same abscissa axes. actA, actin; CAI, codon adaptation index; ENC, effective number of codons; GC, GC content; GC3, GC content on the third position of the codons; tefA, translational elongation factor 1&#x3B1;; tubA, &#x3B1;-tubulin gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F2.medium.gif" width="140" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/4/1301/F2.medium.gif" width="140" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484045" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Distributions of CAI, ENC, GC, and GC3 in <em>E. festucae</em> coding sequences. Genes present only in <em>Epichloë</em> species (potentially not real genes or secondary metabolism genes with fast evolution, for example) are shaded gray. All other genes are shown in white. The values for three housekeeping genes (tefA, tuba, and actA), often used for reconstructing phylogenies and employed here as examples of putatively nonlaterally transferred genes, are shown by green bars under each graph. The values for 10 randomly drawn genes from <em>F. graminearum</em> and <em>L. bicolor</em> are represented, respectively, by blue and red bars. All graphs have the same ordinate axes. CG, GC3, and CAI are plotted using the same abscissa axes. actA, actin; CAI, codon adaptation index; ENC, effective number of codons; GC, GC content; GC3, GC content on the third position of the codons; tefA, translational elongation factor 1α; tubA, α-tubulin gene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">More recent composition-based methods are based on tetranucleotide frequencies. For example, GOHTAM (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Menigaud <em>et al.</em> 2012</a>) and SigHunt (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Jaron <em>et al.</em> 2014</a>) build tetranucleotide signatures of species by computing the frequencies of all possible combinations of four nucleotides along a sliding window. The signatures are then used to find portions of the genomes that differ significantly from the average signature of the genome. SigHunt was used to extract those tetranucleotides with sufficient information to discriminate the <em>E. festucae</em> genome from its closest neighbor in our phylogenetic tree, <em>F. graminearum</em>. The distribution of the gene content for these tetranucleotides is presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. To simulate LGT events into <em>E. festucae</em>, 10 genes were randomly drawn from <em>F. graminearum</em> and <em>L. bicolor</em>, and overlaid on the distribution of <em>E. festucae</em> genes. Although the tetranucleotides were selected specifically to separate the two closest species, the tetranucleotide distributions clearly overlap between these three species, even with <em>L. bicolor</em>, the most distant species. Using these tetranucleotides, SigHunt identifies regions of the <em>E. festucae</em> genome that differ significantly in composition from the rest. These regions appeared to be picked primarily on their AT-rich status (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>). The genes that were included in the regions detected by SigHunt, which were mostly gene-poor isochores, were manually verified by looking at transcriptomic data (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Eaton <em>et al.</em> 2015</a>) to verify the gene model, and using blast results on the nr database of NCBI to check whether any evidence of LGT from a distant species could be found. No confirmation of LGT could be made and orthologs of all genes identified by SigHunt were found in closely related species. The gene <em>fitD</em> does not fall within a region detected by SigHunt.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequencies of the most informative tetranucleotides selected by SigHunt in E. festucae genes. The most informative tetranucleotides selected by SigHunt are presented on the abscissa axis. The box plots represent the distribution of the number of each tetranucleotide found in E. festucae genes divided by the gene length. Blue dots on the left of each box plot represent the values for 10 genes randomly chosen from the closely related species F. graminearum, the red dots on the right to the values for 10 randomly chosen genes from a distantly related species L. bicolor." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequencies of the most informative tetranucleotides selected by SigHunt in &lt;em&gt;E&lt;/em&gt;. &lt;em&gt;festucae&lt;/em&gt; genes. The most informative tetranucleotides selected by SigHunt are presented on the abscissa axis. The box plots represent the distribution of the number of each tetranucleotide found in &lt;em&gt;E. festucae&lt;/em&gt; genes divided by the gene length. Blue dots on the left of each box plot represent the values for 10 genes randomly chosen from the closely related species &lt;em&gt;F. graminearum&lt;/em&gt;, the red dots on the right to the values for 10 randomly chosen genes from a distantly related species &lt;em&gt;L. bicolor&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F3.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/4/1301/F3.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484038" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Frequencies of the most informative tetranucleotides selected by SigHunt in <em>E</em>. <em>festucae</em> genes. The most informative tetranucleotides selected by SigHunt are presented on the abscissa axis. The box plots represent the distribution of the number of each tetranucleotide found in <em>E. festucae</em> genes divided by the gene length. Blue dots on the left of each box plot represent the values for 10 genes randomly chosen from the closely related species <em>F. graminearum</em>, the red dots on the right to the values for 10 randomly chosen genes from a distantly related species <em>L. bicolor</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">A set of tetranucleotides displaying low variance among all genes in <em>E. festucae</em> was also computed (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>). LGT events might be observable as outliers for these low variance tetranucleotides. For example, it might be expected that <em>fitD</em> (Figure S2C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>) would display quite different values to housekeeping genes such as <em>tefA</em> (Figure S2B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>). However, values for <em>tefA</em> were in fact more different from the genome-wide gene signature than those of most <em>E. festucae</em> genes, including <em>fitD</em>.</p><p id="p-25">To obtain an unbiased classification of the genes based on their composition, a data mining approach on the results of all the composition-based methods described above was performed. Transcript sequences of <em>E. festucae</em> were clustered using the Mean Shift clustering method (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Comaniciu and Meer 2002</a>) based on different measures of nucleotide composition, namely their CAI, ENC, GC, GC3, and trinucleotide (codon) and tetranucleotide frequencies. The automatically predicted bandwidth parameter placed all genes within a single cluster, suggesting that there are no genes whose composition differs significantly from any other genes in this dataset. Using the DBSCAN (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ester <em>et al.</em> 1996</a>) clustering algorithm, the same phenomenon was observed. This indicates that these metrics appear to be insufficient to separate genes into different groups, and it follows that they are therefore incapable of discriminating laterally transferred genes from nonlaterally transferred genes.</p><p id="p-26">Given the apparent failure of composition-based methods to identify laterally transferred genes in the <em>E. festucae</em> genome, values for the metrics used above (GC, GC3, CAI, and ENC) were compared across the five secondary fungal reference genomes indicated by stars on <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, together with the genome of the bacterium <em>Ps. fluorescens</em>, which is a suspected source of the <em>fitD</em> potential LGT. To detect LGT, a method must be able to clearly discriminate between genes of different species. Scatterplots of GC <em>vs.</em> ENC for the five additional fungal reference species, together with <em>E. festucae</em> and <em>Ps. fluorescens</em>, are presented in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. These two metrics were chosen because they show the least overlap of all metrics used (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For <em>E. festucae</em>, the three housekeeping genes (<em>actA</em>, <em>tubA</em>, and <em>tefA</em>) and <em>fitD</em> all fall well within the cloud of points. Additionally, the distributions for the different fungal species, and even more surprisingly <em>Ps. fluorescens</em>, all overlap, indicating that it is not possible to discriminate between genes of these different species using these composition-based metrics. To further test the power of these metrics for species discrimination, sequences with the same size distribution as <em>E. festucae</em> coding sequences, but with random nucleotide content (although retaining the same start and stop codons), were generated and their content compared to the different species (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4G</a>). Once more, the distributions were all found to overlap. Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> summarize the overlap between composition values for the five fungal species, <em>Ps. fluorescens</em>, and the random sequences, with <em>E. festucae</em> values. Aside from the random sequences, the lowest overlap is seen between the GC content of <em>E. festucae</em> and <em>Ps. fluorescens</em>. This is expected since the bacterium <em>Ps. fluorescens</em> is the most phylogenetically distant species used in this study. If the data followed the known species phylogeny, the overlap should decrease from <em>F. graminearum</em>, the most closely related fungal species to <em>E. festucae</em>, to <em>L. bicolor</em>, the least closely related species (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). However, this is not the case (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1/FileS1.docx">File S1</a>). For GC content, the greatest overlap is seen for <em>S. turcica</em>, and none of the metrics show a general decrease with increasing phylogenetic distance. This again questions the statistical power of these composition-based methods for detecting LGT events.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of coding sequence compositions (ENC and GC) for species at laddered evolutionary distances from E. festucae. Scatterplots comparing GC content to ENC. All abscissa and ordinate axes are plotted using the same scales. (A) Comparison of GC and ENC for the reference species E. festucae. Housekeeping genes tubA, actA, and tefA are highlighted in red (actA and tefA points overlap), while the fitD putative laterally transferred gene from a Pseudomonas bacterium is shown in green. The fungal species F. graminearum (B), M. oryzae (C), O. maius (D), S. turcica (E), and L. bicolor (F) show a marked overlap with the E. festucae data. (G) Sequences with the same size as E. festucae coding sequences, starting with ATG and ending with the same stop codon, but otherwise containing random nucleotides are shown. For comparison, the sequence composition of the bacterium Ps. fluorescens is also presented in (H). actA, actin; ENC, effective number of codons; fitD, cytotoxin FitD; GC, GC content; tefA, translational elongation factor 1&#x3B1;; tubA, &#x3B1;-tubulin gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of coding sequence compositions (ENC and GC) for species at laddered evolutionary distances from &lt;em&gt;E. festucae&lt;/em&gt;. Scatterplots comparing GC content to ENC. All abscissa and ordinate axes are plotted using the same scales. (A) Comparison of GC and ENC for the reference species &lt;em&gt;E. festucae&lt;/em&gt;. Housekeeping genes tubA, actA, and tefA are highlighted in red (actA and tefA points overlap), while the fitD putative laterally transferred gene from a &lt;em&gt;Pseudomonas&lt;/em&gt; bacterium is shown in green. The fungal species &lt;em&gt;F. graminearum&lt;/em&gt; (B), &lt;em&gt;M. oryzae&lt;/em&gt; (C), &lt;em&gt;O. maius&lt;/em&gt; (D), &lt;em&gt;S. turcica&lt;/em&gt; (E), and &lt;em&gt;L. bicolor&lt;/em&gt; (F) show a marked overlap with the &lt;em&gt;E. festucae&lt;/em&gt; data. (G) Sequences with the same size as &lt;em&gt;E. festucae&lt;/em&gt; coding sequences, starting with ATG and ending with the same stop codon, but otherwise containing random nucleotides are shown. For comparison, the sequence composition of the bacterium &lt;em&gt;Ps&lt;/em&gt;. &lt;em&gt;fluorescens&lt;/em&gt; is also presented in (H). actA, actin; ENC, effective number of codons; fitD, cytotoxin FitD; GC, GC content; tefA, translational elongation factor 1&#x3B1;; tubA, &#x3B1;-tubulin gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F4.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/4/1301/F4.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484029" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Comparison of coding sequence compositions (ENC and GC) for species at laddered evolutionary distances from <em>E. festucae</em>. Scatterplots comparing GC content to ENC. All abscissa and ordinate axes are plotted using the same scales. (A) Comparison of GC and ENC for the reference species <em>E. festucae</em>. Housekeeping genes tubA, actA, and tefA are highlighted in red (actA and tefA points overlap), while the fitD putative laterally transferred gene from a <em>Pseudomonas</em> bacterium is shown in green. The fungal species <em>F. graminearum</em> (B), <em>M. oryzae</em> (C), <em>O. maius</em> (D), <em>S. turcica</em> (E), and <em>L. bicolor</em> (F) show a marked overlap with the <em>E. festucae</em> data. (G) Sequences with the same size as <em>E. festucae</em> coding sequences, starting with ATG and ending with the same stop codon, but otherwise containing random nucleotides are shown. For comparison, the sequence composition of the bacterium <em>Ps</em>. <em>fluorescens</em> is also presented in (H). actA, actin; ENC, effective number of codons; fitD, cytotoxin FitD; GC, GC content; tefA, translational elongation factor 1α; tubA, α-tubulin gene.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483980/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483980/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Percentage overlap of distributions of gene composition metrics between <em>E. festucae</em> and other species, ordered by phylogenetic distance from <em>E. festucae</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Phylogeny-based methods</h3><p id="p-29">A representative species tree is essential for any LGT detection method based on variation within the phylogeny. In this study, a species phylogeny was built using all gene trees containing a sequence from outgroup <em>P. sojae</em> (1768 trees). An SPR supertree was then built using only bipartitions with a bootstrap value ≥75% to increase the reliability of the final tree topology (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Salichos and Rokas 2013</a>). The generated tree (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) matches previously published fungal phylogenies (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ebersberger <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Fitzpatrick <em>et al.</em> 2006</a>; <a id="xref-ref-118-2" class="xref-bibr" href="#ref-118">Wang <em>et al.</em> 2009</a>) and fits with the traditional fungal classification in the NCBI taxonomy database (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Sayers <em>et al.</em> 2009</a>). The statistical significance of the branches in the species tree was computed as described in the <em>Methods</em>, and the resulting bootstrap values are presented under the branches in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Most branches (88%) have bootstrap values &gt;95%, showing strong support for most branches in the proposed species tree. Lower bootstrap values are observed in some small groups close to the tips of the tree, but all deeper groupings are supported by high bootstrap values.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Statistical support for the internal nodes of the species tree. The taxa in this species tree are the same as those presented in Figure 1 and are given in the same order. The numbers indicated over the branches (green) show the percentage of the bipartitions in the gene trees for the node to the right of the number. The numbers under the branches (red) show the corresponding bootstrap values. All trees were rooted using the oomycete outgroup P. sojae." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Statistical support for the internal nodes of the species tree. The taxa in this species tree are the same as those presented in Figure 1 and are given in the same order. The numbers indicated over the branches (green) show the percentage of the bipartitions in the gene trees for the node to the right of the number. The numbers under the branches (red) show the corresponding bootstrap values. All trees were rooted using the oomycete outgroup &lt;em&gt;P. sojae&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F5.medium.gif" width="240" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/4/1301/F5.medium.gif" width="240" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Statistical support for the internal nodes of the species tree. The taxa in this species tree are the same as those presented in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> and are given in the same order. The numbers indicated over the branches (green) show the percentage of the bipartitions in the gene trees for the node to the right of the number. The numbers under the branches (red) show the corresponding bootstrap values. All trees were rooted using the oomycete outgroup <em>P. sojae</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The SPRSupertrees (<a id="xref-ref-120-3" class="xref-bibr" href="#ref-120">Whidden <em>et al.</em> 2014</a>) software used to rebuild the species tree is also able to detect potential LGT events, but none were identified in this dataset. In contrast, <a id="xref-ref-121-2" class="xref-bibr" href="#ref-121">Wisecaver <em>et al.</em> (2014</a>) claim to have identified hundreds of LGT events in a fungal dataset of similar size. To see if the presence of potential paralogs in our dataset could affect the results of SPRSupertrees, other types of phylogeny-based LGT detection software, including programs that accommodate paralogs, were also applied to the dataset. Notung (<a id="xref-ref-107-3" class="xref-bibr" href="#ref-107">Stolzer <em>et al.</em> 2012</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2000</a>) and Ranger-DTL (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bansal <em>et al.</em> 2012</a>) use a parsimony analysis to infer duplications, losses, and transfers of genes within a phylogenetic tree. Using the default costs recommended by their authors (LGT score = 3), both methods suggested that &gt;99% of gene trees in the dataset contain at least one LGT event, and even when the cost of an LGT event was doubled, &gt;50% of the gene trees were predicted to contain at least one LGT event. These results contrast strongly with the LGT detection results of SPRSupertrees and differ markedly with the general consensus that LGTs are relatively rare events. Published comparisons of the results from Notung and Ranger-DTL show that phylogeny-based methods for detecting LGT events often give high rates of false-positive results (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Poptsova and Gogarten 2007</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Nguyen <em>et al.</em> 2012</a>). The software ecceTERA (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Jacox <em>et al.</em> 2016</a>) includes a gene tree correction method that aims to reduce the number of false-positives. EcceTERA results on trees containing only best reciprocal blast orthologs were compared to trees containing all homologs based on a blast <em>E</em>-value &lt; 1 × 10<sup>−12</sup>, but the results returned were comparable in both cases (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in the number of lateral gene transfer (LGT) events predicted depending on the transfer cost in phylogeny-based LGT prediction methods. The percentage of trees displaying at least one LGT event predicted for transfer costs varying between 3 (the default cost) and 30 is shown for three different methods: Notung, Ranger-DTL, and ecceTERA. Only the gene trees with the best reciprocal blast orthologs were used for Ranger-DTL and Notung. Trees with homologs only, or homologs and orthologs combined (blast E-value &lt; 1 &#xD7; 10&#x2212;12), are shown separately for ecceTERA. The theoretical rare event threshold of 5% is indicated by a black dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation in the number of lateral gene transfer (LGT) events predicted depending on the transfer cost in phylogeny-based LGT prediction methods. The percentage of trees displaying at least one LGT event predicted for transfer costs varying between 3 (the default cost) and 30 is shown for three different methods: Notung, Ranger-DTL, and ecceTERA. Only the gene trees with the best reciprocal blast orthologs were used for Ranger-DTL and Notung. Trees with homologs only, or homologs and orthologs combined (blast &lt;em&gt;E&lt;/em&gt;-value &lt; 1 &#xD7; 10&lt;sup&gt;&#x2212;12&lt;/sup&gt;), are shown separately for ecceTERA. The theoretical rare event threshold of 5% is indicated by a black dashed line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F6.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/4/1301/F6.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483978" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-32" class="first-child">Variation in the number of lateral gene transfer (LGT) events predicted depending on the transfer cost in phylogeny-based LGT prediction methods. The percentage of trees displaying at least one LGT event predicted for transfer costs varying between 3 (the default cost) and 30 is shown for three different methods: Notung, Ranger-DTL, and ecceTERA. Only the gene trees with the best reciprocal blast orthologs were used for Ranger-DTL and Notung. Trees with homologs only, or homologs and orthologs combined (blast <em>E</em>-value &lt; 1 × 10<sup>−12</sup>), are shown separately for ecceTERA. The theoretical rare event threshold of 5% is indicated by a black dashed line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">By definition, an LGT event will create a discrepancy between the gene and species trees. Likelihood-based methods can be used to detect such differences. It is possible to constrain the gene tree topology by the species tree topology, and compute the likelihood of the constrained gene tree by letting PhyML optimize only the branch lengths and the rate parameters, but not the topology. By doing this, the final phylogenetic tree will have the same topology as the species tree, only the branches will have different lengths. In the absence of any LGT event, the gene tree would be expected to be close to the species tree, so constraining the gene tree using the species tree topology should have little impact on its likelihood. However, in the presence of an LGT event from a species included in the tree (or a close relative from outside the species set), the length of the branch carrying the LGT event will significantly increase, resulting in a gene tree with lower likelihood. The likelihoods of the constrained and unconstrained trees, both built using the same model, are then compared. The AIC (Akaike Information Criterion), which can be computed on nonnested models, was then calculated from the two likelihoods to verify whether they are meaningfully different. For this comparison, the degrees of freedom correspond to the number of internal branches on the tree. No significant difference between the constrained and unconstrained gene trees was observed; in other words, no LGT events were again detected. To identify whether this method is really capable of detecting LGTs, computer simulations of LGT events were performed. Specifically, random genes were computationally transferred from <em>L. bicolor</em>, <em>S. turcica</em>, <em>M. oryzae</em>, and <em>O. maius</em> to <em>E. festucae</em>. The gene tree topologies were again constrained using the species tree, and the AIC of the constrained and unconstrained trees computed. Again, no significant differences were found, likely reflecting the high number of degrees of freedom that causes an increase in the significance threshold. In practice, this method, which is conceptually a cornerstone of many phylogenetic-based LGT detection techniques, appears to have low statistical power to detect LGT events.</p><p id="p-34">Even though the SPR supertree was identical to that expected based on the published literature, substantial variation in the topologies of the gene trees was observed relative to the final species tree. This has also been noted by <a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Salichos and Rokas (2013)</a>, and is reflected by the low proportion of species tree bipartitions (nodes of the tree) present in the gene trees (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Indeed, 31% of the nodes in the supertree are present in fewer than half of the gene trees, and only 34% are present in at least two-thirds of the gene trees. To determine how topologically distant the gene trees are from the species tree, the normalized RF topologic distance (RFn) (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Robinson and Foulds 1981</a>) between each gene tree and the species tree was computed. The distribution of these distances is presented in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. Contrary to expectations, most of the distances between the gene trees and the species tree are not close to zero, meaning that some of the gene trees have very different topologies from the species tree. To verify whether the observed distribution might simply be due to randomness, possibly resulting from incorrect orthology calls or problems with small numbers of taxa in some of the gene trees, the leaves of each gene tree were shuffled to create random trees with the same size distribution as the original set of trees (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). As the distributions of the real and random data are different, it is unlikely that the distances of the gene trees to the species tree are solely due to a gene tree size effect, or to random effects, at least across the entire tree. It was particularly surprising to observe that even housekeeping genes like <em>tubA</em> (RFn = 0.33) and <em>tefA</em> (RFn = 0.5) display large topological distances from the species tree.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in the topology of gene trees compared to the species tree. The distribution of the RFns between the species tree and between the species tree and trees of individual E. festucae genes is shown in black. The distribution of the RFn distances between the species tree and randomly generated trees with the same size distribution as the real gene trees is displayed in white. The distance values for the housekeeping genes actA, tubA, and tefA are highlighted by red arrows. The fitD gene cannot be represented on this figure because it is only present in E. festucae. (This putative laterally transferred gene is not found in the other fungi studied here.) Examples of comparisons between gene trees and the species tree are displayed in the boxes at the top part of the figure. The species tree appears to vary only because the subset of species present in the gene tree differs between the three examples. The species tree topology is in fact identical for all analyses. Orange links indicate a change in the location of species in the gene tree relative to the species tree. Blue links indicate conservation with the species tree topology. RFn = 0 corresponds to a perfect match between the species tree and the gene tree, while RFn = 0.5 indicates that half of the nodes in the gene tree do not match the species tree. actA, actin; fitD, cytotoxin FitD; RFn, normalized Robinson and Foulds distances; tefA, translational elongation factor 1&#x3B1;; tubA, &#x3B1;-tubulin gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2033822596" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation in the topology of gene trees compared to the species tree. The distribution of the RFns between the species tree and between the species tree and trees of individual &lt;em&gt;E. festucae&lt;/em&gt; genes is shown in black. The distribution of the RFn distances between the species tree and randomly generated trees with the same size distribution as the real gene trees is displayed in white. The distance values for the housekeeping genes actA, tubA, and tefA are highlighted by red arrows. The fitD gene cannot be represented on this figure because it is only present in &lt;em&gt;E. festucae&lt;/em&gt;. (This putative laterally transferred gene is not found in the other fungi studied here.) Examples of comparisons between gene trees and the species tree are displayed in the boxes at the top part of the figure. The species tree appears to vary only because the subset of species present in the gene tree differs between the three examples. The species tree topology is in fact identical for all analyses. Orange links indicate a change in the location of species in the gene tree relative to the species tree. Blue links indicate conservation with the species tree topology. RFn = 0 corresponds to a perfect match between the species tree and the gene tree, while RFn = 0.5 indicates that half of the nodes in the gene tree do not match the species tree. actA, actin; fitD, cytotoxin FitD; RFn, normalized Robinson and Foulds distances; tefA, translational elongation factor 1&#x3B1;; tubA, &#x3B1;-tubulin gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1301/F7.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/4/1301/F7.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1301/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483890" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child">Variation in the topology of gene trees compared to the species tree. The distribution of the RFns between the species tree and between the species tree and trees of individual <em>E. festucae</em> genes is shown in black. The distribution of the RFn distances between the species tree and randomly generated trees with the same size distribution as the real gene trees is displayed in white. The distance values for the housekeeping genes actA, tubA, and tefA are highlighted by red arrows. The fitD gene cannot be represented on this figure because it is only present in <em>E. festucae</em>. (This putative laterally transferred gene is not found in the other fungi studied here.) Examples of comparisons between gene trees and the species tree are displayed in the boxes at the top part of the figure. The species tree appears to vary only because the subset of species present in the gene tree differs between the three examples. The species tree topology is in fact identical for all analyses. Orange links indicate a change in the location of species in the gene tree relative to the species tree. Blue links indicate conservation with the species tree topology. RFn = 0 corresponds to a perfect match between the species tree and the gene tree, while RFn = 0.5 indicates that half of the nodes in the gene tree do not match the species tree. actA, actin; fitD, cytotoxin FitD; RFn, normalized Robinson and Foulds distances; tefA, translational elongation factor 1α; tubA, α-tubulin gene.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-36">LGT is a “hot topic” in molecular biology (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Soucy <em>et al.</em> 2015</a>) and has almost become a default explanation for genes that do not display an expected phylogenetic distribution or composition. However, identification of LGT events is no trivial exercise (<a id="xref-ref-86-3" class="xref-bibr" href="#ref-86">Ragan 2001</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Morel <em>et al.</em> 2015</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Ovadia <em>et al.</em> 2011</a>) and detection methods are expected to be highly dependent on the quality of the underlying genomic data. Recently, many LGT events in eukaryotic species have been called into question (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Danchin 2016</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Ku <em>et al.</em> 2015</a>).</p><p id="p-37">In this study, we assess the reliability of automated LGT detection methods for eukaryotes using datasets derived from plant-associated fungi, as a counterpoint to the extensive analyses performed on bacterial systems (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Lawrence and Ochman 1997</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Garcia-Vallve <em>et al.</em> 2000</a>; <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Nakamura <em>et al.</em> 2004</a>). Both composition- and phylogeny-based approaches were assessed for their ability to detect both putative and simulated LGT events in a eukaryotic fungal dataset. This included <em>fitD</em>, a gene originally described as laterally transferred from <em>Pseudomonas</em> bacteria into <em>Epichloë</em> endophytes (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ambrose <em>et al.</em> 2014</a>), but as it is also present in many other fungi, its LGT status remains uncertain.</p><p id="p-38">Given the wealth of genomic data now available, it is impractical to manually verify all protein or gene models, orthology predictions, or multiple sequence alignments, requiring researchers to accept some margin of error and level of mistakes in their data. It is difficult to predict <em>a priori</em> the extent to which these problems affect the bioinformatics tools and techniques used in LGT detection. For this reason, our analysis was purposely performed using typical uncurated genomic datasets, so as to mimic as closely as possible the situation encountered by most users of the various LGT prediction tools.</p><p id="p-39">Composition-based methods are, in theory, ideal for predicting LGT events; they do not require any knowledge about the studied organism other than its genomic sequence, and because they do not rely on phylogenetics, they are generally much faster to use. Moreover, no information about other organisms is needed for comparison and knowledge of the source species of the LGT event is not required. However, it is important to acknowledge that, strictly speaking, composition-based methods do not detect LGT events, but rather show genes whose composition differs significantly from the rest of the genome. Nonetheless, LGT is not the only explanation for differences in gene composition, as this can also be caused by common non-LGT genetic processes, such as rapid gene evolution, pseudogenization, or simply being located close to AT-rich isochores. Therefore, not all genes that display unexpected composition patterns necessarily result from LGT events. Composition-based detection methods have been evaluated in bacterial genomes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cortez <em>et al.</em> 2005</a>) by studying the ability of different methods to detect artificial gene transfers to <em>E. coli</em> K12 MG1655 from different proteobacteria and prokaryotes. This <em>E. coli</em> study revealed the highly variable rates of efficacy of these methods, especially for the CAI/GC content metric, with detection of LGT events ranging from 6 to 92%. Here, four gene composition metrics were assessed for their reliability in detecting potential LGT events in fungi: GC, GC3, CAI, and ENC. Surprisingly, none of these metrics were able to separate random sequences from true genes, showing that these methods have no power to detect LGT events in our dataset. Manual analysis of genes at the extremes of the four metric distributions (2.5% tails), where laterally transferred genes would be expected to fall, failed to reveal any obvious signs of LGT based on homology searches with other species. Inclusion of a fifth metric, the more recently developed tetranucleotide frequencies, identifies AT-rich isochores as LGT events, showing that this method cannot be applied to fungal genomes without preliminary filtering of AT-rich regions, repetitive elements, and probably other genomic features like pathogenicity islands. Even the most informative tetranucleotide identified by SigHunt failed to separate closely related <em>F. graminearum</em> and distantly related <em>L. bicolor</em> genes from <em>E. festucae</em> orthologs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-40">A crucial assumption in the use of composition-based methods for LGT detection is that gene composition metrics vary between species, and this variation is presumed to increase with phylogenetic distance. Here, we show that this is not true. GC and ENC displayed the lowest average overlap between the different species, but even when these two metrics were combined in a two-dimensional comparison, no significant differences were observed, even against bacterial genes and random sequences. This inability to separate data derived from different species, or even random data, questions the usefulness of these metrics for LGT detection. These metrics have been widely used for predicting LGT events in prokaryotes (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Gogarten and Townsend 2005</a>; <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Keeling and Palmer 2008</a>; <a id="xref-ref-100-3" class="xref-bibr" href="#ref-100">Shapiro 1969</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Gyles and Boerlin 2014</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Kay <em>et al.</em> 2002</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Furuya and Lowy 2006</a>; <a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Martínez 2008</a>), and this study does not question those findings. Rather, we query the widely-assumed suitability of these metrics for analyzing eukaryotic data. Eukaryote gene structures are more complex than in prokaryotes, with the presence of introns and exons that are often poorly annotated in nonmodel organisms. LGT prediction may be even more complex in fungi because they display rapid evolution, especially in plant-associated fungi (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Le Quere <em>et al.</em> 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brundrett 2002</a>; <a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Terhorst <em>et al.</em> 2014</a>), and particularly for genes involved in pathogenicity or symbiosis. Therefore, it might be expected that laterally transferred genes would evolve rapidly to display a similar composition to the rest of the genome.</p><p id="p-41">This study also evaluated the reliability of phylogeny-based methods for LGT detection, using simulated LGT events into <em>E. festucae</em> as a benchmark for detection. Phylogeny-based methods are generally considered to be more reliable than composition-based methods, especially for detecting transfers between closely related species. However, there are considerable inconsistencies between the results of different detection methods (<a id="xref-ref-86-4" class="xref-bibr" href="#ref-86">Ragan 2001</a>; <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Jaron <em>et al.</em> 2014</a>). Additionally, phylogeny-based methods seem to be very sensitive to the quality of the study data (Wägele and Mayer 2007) and it was unclear how these methods would perform on real uncurated datasets, particularly those representing larger eukaryotic genomes. Phylogeny-based methods work by comparing the topology of the species tree with topologies for gene trees to identify genes that display a significantly different phylogeny, thus hinting at possible LGT events.</p><p id="p-42">The species supertree was built so as to minimize the distance between all gene trees. Due to the nature of the resampling process used to compute the bootstrap scores, each bootstrap replicate tends to be based on a very large dataset (despite the resampling) and thus all bootstrap trees are very similar to the species supertree. Consequently, bootstrap scores are typically close to 100%. This seems overly conservative and it proved more informative to look at the distribution of tree shapes rather than the bootstrap values. To check whether high bootstrap values are an artifact of the large set of gene trees, the frequencies of the species tree bipartitions present across all the gene trees were computed. This provides a measure of the variation in the underlying gene phylogenies. Comparing individual gene tree topologies with the species supertree revealed a surprisingly high level of variation among individual gene trees, with the frequency of bipartition identities ranging from 21 to 82%, with an average of 56%. This means that a given node in the species tree is on average present only in around half of the gene trees. This was unexpected as it is generally thought that individual gene trees mostly follow the same phylogeny as the species tree, although this marked variation between gene trees and the species tree topology has been observed previously in bacteria (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Poptsova and Gogarten 2007</a>) and yeasts (<a id="xref-ref-95-3" class="xref-bibr" href="#ref-95">Salichos and Rokas 2013</a>). It is likely that the low bipartition identities observed are at least in part due to pruning of nodes in the gene trees with bootstrap values &lt;75%. However, this was essential to ensure that only highly supported nodes were used for the phylogeny comparisons. It is also interesting to note that low bootstrap values were always associated with a low bipartition frequency value, but that the reciprocal property was not true: some high bootstrap values were associated with a low bipartition frequency. For example, the node that separates the Xylonomycetes and Dothideomycetes has a bootstrap score of 100%, but a bipartition frequency score of only 39%. The fact that these frequencies are generally low emphasizes just how variable gene tree topologies are. As phylogeny-based LGT methods work by identifying gene phylogenies that differ from that of the species tree, such extreme levels of topological variation between the gene trees and species tree poses a considerable problem for these detection methods, as it then becomes complex to distinguish LGT events from normal topological variation.</p><p id="p-43">Phylogeny-based methods compare gene and species trees by computing a distance value between them. In this study, the high variation between the gene and species trees by definition also impacts the RFn, where a distance of zero represents an identical topology for a given gene tree and the species trees, while a distance of one corresponds to the two trees being as different from each other as possible. It was expected that most of the gene tree topologies would reflect the species tree topology and have distances very close to zero. In this case, the distribution of the topological distances between the species tree and gene trees would be represented by a rapidly decreasing curve with most of the distances close to zero. LGT events should then be represented by gene trees with higher distances. Instead, a bell-shaped distribution was observed, centered at ∼0.5 and with a maximum of 0.9, indicating that most of the gene trees are very different from the species tree. This implies that, although the species tree is the average of all the gene trees, it is by no means the most frequent tree. In fact, only 6% of gene trees exactly match the species tree. The sheer diversity of gene tree topologies may be due to the variable speed of gene evolution resulting from quite different environmental selection pressures or incomplete lineage sorting. It is generally thought that discordant gene and species phylogenies reflect only rare events like LGT or transspecies polymorphism (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Pamilo and Nei 1988</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Maddison 1997</a>), and many putative laterally transferred genes have been identified based on this assumption. However, this study reveals that, at least in fungi, discord between gene and species tree topologies appears to be common and can even be observed routinely in housekeeping genes. We suspect that this large variation in tree topologies is the main reason why the SPRSupertrees software could not detect any LGT events in this dataset, and may also explain the unlikely results of Notung and Ranger-DTL, which predict that nearly all gene trees contain at least one LGT event. These tools apply a score for each type of event (duplications, gene losses, and LGTs), and the software returns the most parsimonious series of events relative to the phylogeny. Without <em>a priori</em> knowledge of the rates at which these events occur, it is challenging to find an appropriate set of scores for the algorithm. In this study, the default scores produce unlikely results. Moreover, missing genes in our gene trees may have important consequences: they should be annotated as losses, but often seem to be classified as transfers. EcceTERA attempts to address this by implementing a probabilistic model to infer true transfer events from false-positives. The relationship between the number of detected LGT events and the transfer cost in Notung, Ranger-DTL, and ecceTERA are inverse-like functions; doubling the default score for a transfer event still results in &gt;50% of the trees displaying an LGT event. Even when the default transfer cost is multiplied by five, LGT events still are still not rare, with &gt;5% of trees (8 to 38% of trees) being called as having at least one LGT event. This illustrates how difficult it can be to choose a transfer cost for these methods and how unreliable the default scores can be.</p><p id="p-44">Often, missing orthologs and discontinuous distributions of homologs are also used to infer potential LGT events (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Ambrose <em>et al.</em> 2014</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Moore <em>et al.</em> 2014</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Fitzpatrick <em>et al.</em> 2008</a>; <a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Temporini and Van Etten 2004</a>), but these assumptions tend to change as more genomes are sequenced and intermediate genes are found (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em> 2008</a>). Finally, many of the tools developed to detect LGT events are based on phylogenetic methods that have been developed for bacteria, but in practice, fungal phylogenies are much harder to reconstruct than bacterial phylogenies due to the presence of cryptic intronic sequences and other sequence complexities. Additionally, the genomes of plant-associated fungi often contain effector genes, which are short and highly variable (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Mesarich <em>et al.</em> 2015</a>), as well as genes involved in the production of secondary metabolites, which tend to evolve rapidly and often display a complex modular structure (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berry <em>et al.</em> 2015</a>). The phylogeny is particularly hard to reconstruct for these genes. Thus, the efficacy of LGT detection tools in fungi, and presumably for other eukaryotes, seems to remain relatively underpowered. In addition to the deficiencies mentioned above, the phylogeny-based LGT detection methods also completely failed to identify either the putative LGT event of <em>fitD</em>, any other potentially novel LGT events, or any computationally simulated LGT events. This leads us to express doubt regarding the overall reliability of these tools for LGT detection in similar eukaryotic datasets. Indeed, this study highlights the key problem of using incompletely annotated genome sequence information for automated LGT prediction. Where orthologs could not be identified in some species, this may not have been due to the absence of these genes in those species, but rather from the sequence simply being missing from the dataset. As missing taxa in gene trees are a potential indicator of LGT (<a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">Temporini and Van Etten 2004</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Fitzpatrick <em>et al.</em> 2008</a>), this problem would also confound LGT detection methods.</p><p id="p-45">Overall, this study reveals concerning insight into the deficiencies of both composition- and phylogeny-based LGT detection methods when used on complex, uncurated, but typical eukaryotic genome sequence data. Neither of the broad approaches was capable of identifying a putative LGT event or gave reliable support for any novel LGT event. This is in striking contrast to the large number of claims that routinely appear in the literature. Additionally, this study exposes the existence of extremely high variation between gene tree and species tree topologies across fungi, showing that this major biological pattern holds beyond the yeasts. An important effect is that this variation will likely confound phylogeny-based LGT prediction methods. This study concludes that LGT remains a challenging process to detect, and questions whether the flood of LGT events appearing in the literature to date are necessarily well supported statistically.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We thank Carla Eaton and anonymous reviewers for thoughtful comments on the manuscript. This research was supported by the Tertiary Education Commission through a Centre of Research Excellence grant from the Bio-Protection Research Centre, and by the Royal Society of New Zealand through a Rutherford Fellowship (RDF-10-MAU-001) and Marsden grant (14-MAU-007) to M.P.C. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-47">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038448/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-48"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-12-13"><span class="received-label">Received </span>December 13, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-02-17"><span class="accepted-label">Accepted </span>February 17, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Dupont and Cox</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.1" data-doi="10.1186/1471-2105-11-324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abby</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tannier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daubin</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detecting lateral gene transfers by statistical reconciliation of phylogenetic forests.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DAbby%26rft.auinit1%253DS.%2BS.%26rft.volume%253D11%26rft.spage%253D324%26rft.epage%253D324%26rft.atitle%253DDetecting%2Blateral%2Bgene%2Btransfers%2Bby%2Bstatistical%2Breconciliation%2Bof%2Bphylogenetic%2Bforests.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-324%26rft_id%253Dinfo%253Apmid%252F20550700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20550700&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.2" data-doi="10.1007/978-1-60327-853-9_28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alsmark</span>  <span class="cit-name-given-names">U. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sicheritz-Ponten</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirt</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Embley</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Horizontal gene transfer in eukaryotic parasites: a case study of <em>Entamoeba histolytica</em> and <em>Trichomonas vaginalis</em>.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">532</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DAlsmark%26rft.auinit1%253DU.%2BC.%26rft.volume%253D532%26rft.spage%253D489%26rft.epage%253D500%26rft.atitle%253DHorizontal%2Bgene%2Btransfer%2Bin%2Beukaryotic%2Bparasites%253A%2Ba%2Bcase%2Bstudy%2Bof%2BEntamoeba%2Bhistolytica%2Band%2BTrichomonas%2Bvaginalis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-853-9_28%26rft_id%253Dinfo%253Apmid%252F19271203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-853-9_28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19271203&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.3" data-doi="10.1038/srep05562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ambrose</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koppenhofer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belanger</span>  <span class="cit-name-given-names">F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Horizontal gene transfer of a bacterial insect toxin gene into the <em>Epichloë</em> fungal symbionts of grasses.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">5562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D5562%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep05562%26rft_id%253Dinfo%253Apmid%252F24990771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep05562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24990771&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.4" data-doi="10.1371/journal.pgen.1002230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amselem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kan</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viaud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benito</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic analysis of the necrotrophic fungal pathogens <em>Sclerotinia sclerotiorum</em> and <em>Botrytis cinerea</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1002230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAmselem%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De1002230%26rft.epage%253De1002230%26rft.atitle%253DGenomic%2Banalysis%2Bof%2Bthe%2Bnecrotrophic%2Bfungal%2Bpathogens%2BSclerotinia%2Bsclerotiorum%2Band%2BBotrytis%2Bcinerea.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002230%26rft_id%253Dinfo%253Apmid%252F21876677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21876677&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.5" data-doi="10.1093/bioinformatics/bts225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alm</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient algorithms for the reconciliation problem with gene duplication, horizontal transfer and loss.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">i283</span>–<span class="cit-lpage">i291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts225%26rft_id%253Dinfo%253Apmid%252F22689773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTIvaTI4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.6" data-doi="10.1007/978-1-60327-853-9_14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beiko</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragan</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Untangling hybrid phylogenetic signals: horizontal gene transfer and artifacts of phylogenetic reconstruction.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">532</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBeiko%26rft.auinit1%253DR.%2BG.%26rft.volume%253D532%26rft.spage%253D241%26rft.epage%253D256%26rft.atitle%253DUntangling%2Bhybrid%2Bphylogenetic%2Bsignals%253A%2Bhorizontal%2Bgene%2Btransfer%2Band%2Bartifacts%2Bof%2Bphylogenetic%2Breconstruction.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-853-9_14%26rft_id%253Dinfo%253Apmid%252F19271189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-853-9_14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19271189&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.7" data-doi="10.1128/AEM.03721-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takach</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schardl</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlton</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Disparate independent genetic events disrupt the secondary metabolism gene <em>perA</em> in certain symbiotic <em>Epichloë</em> species.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">2797</span>–<span class="cit-lpage">2807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.03721-14%26rft_id%253Dinfo%253Apmid%252F25681180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjgxLzgvMjc5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.8" data-doi="10.1038/nrg1000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Ancient horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrown%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D121%26rft.epage%253D132%26rft.atitle%253DAncient%2Bhorizontal%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1000%26rft_id%253Dinfo%253Apmid%252F12560809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12560809&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180711300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.9" data-doi="10.1046/j.1469-8137.2002.00397.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brundrett</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Coevolution of roots and mycorrhizas of land plants.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">154</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">275</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D154%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-8137.2002.00397.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-8137.2002.00397.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000175244900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.10" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutierrez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silla-Martinez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheeseman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ropars</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouzy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple recent horizontal transfers of a large genomic region in cheese making fungi.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D2876%26rft_id%253Dinfo%253Apmid%252F24407037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24407037&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.12" data-doi="10.1089/106652700750050871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farach-Colton</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">NOTUNG: a program for dating gene duplications and optimizing gene family trees.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">3–4</span>): <span class="cit-fpage">429</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DChen%26rft.auinit1%253DK.%26rft.volume%253D7%26rft.issue%253D3-4%26rft.spage%253D429%26rft.epage%253D447%26rft.atitle%253DNOTUNG%253A%2Ba%2Bprogram%2Bfor%2Bdating%2Bgene%2Bduplications%2Band%2Boptimizing%2Bgene%2Bfamily%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F106652700750050871%26rft_id%253Dinfo%253Apmid%252F11108472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/106652700750050871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11108472&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165512600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.13" data-doi="10.1016/j.fgb.2007.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ansari</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koga</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Epichloë</em> endophytes grow by intercalary hyphal extension in elongating grass leaves.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">84</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DChristensen%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D45%26rft.issue%253D2%26rft.spage%253D84%26rft.epage%253D93%26rft.atitle%253DEpichloe%2Bendophytes%2Bgrow%2Bby%2Bintercalary%2Bhyphal%2Bextension%2Bin%2Belongating%2Bgrass%2Bleaves.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2007.07.013%26rft_id%253Dinfo%253Apmid%252F17919950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2007.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17919950&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.14" data-doi="10.1109/34.1000236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comaniciu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meer</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mean shift: a robust approach toward feature space analysis.</span> <abbr class="cit-jnl-abbrev">IEEET Pattern Anal</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">603</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEET%2BPattern%2BAnal%26rft.volume%253D24%26rft.spage%253D603%26rft_id%253Dinfo%253Adoi%252F10.1109%252F34.1000236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/34.1000236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.15" data-doi="10.1371/journal.pgen.1003233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Condon</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushley</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genome structure, secondary metabolite, and effector coding capacity across <em>Cochliobolus</em> pathogens.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1003233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003233%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003233%26rft_id%253Dinfo%253Apmid%252F23357949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23357949&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.16" data-doi="10.1007/978-1-60327-853-9_12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaye</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazcano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becerra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Composition-based methods to identify horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">532</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DCortez%26rft.auinit1%253DD.%26rft.volume%253D532%26rft.spage%253D215%26rft.epage%253D225%26rft.atitle%253DComposition-based%2Bmethods%2Bto%2Bidentify%2Bhorizontal%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-853-9_12%26rft_id%253Dinfo%253Apmid%252F19271187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-853-9_12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19271187&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortez</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazcano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becerra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative analysis of methodologies for the detection of horizontally transferred genes: a reassessment of first-order Markov models.</span> <abbr class="cit-jnl-abbrev">In Silico Biol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">5–6</span>): <span class="cit-fpage">581</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2Bsilico%2Bbiology%26rft.stitle%253DIn%2BSilico%2BBiol%26rft.aulast%253DCortez%26rft.auinit1%253DD.%2BQ.%26rft.volume%253D5%26rft.issue%253D5-6%26rft.spage%253D581%26rft.epage%253D592%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bmethodologies%2Bfor%2Bthe%2Bdetection%2Bof%2Bhorizontally%2Btransferred%2Bgenes%253A%2Ba%2Breassessment%2Bof%2Bfirst-order%2BMarkov%2Bmodels.%26rft_id%253Dinfo%253Apmid%252F16610135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16610135&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.18" data-doi="10.1126/science.1143708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldener</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trail</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turgeon</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Fusarium graminearum</em> genome reveals a link between localized polymorphism and pathogen specialization.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>(<span class="cit-issue">5843</span>): <span class="cit-fpage">1400</span>–<span class="cit-lpage">1402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCuomo%26rft.auinit1%253DC.%2BA.%26rft.volume%253D317%26rft.issue%253D5843%26rft.spage%253D1400%26rft.epage%253D1402%26rft.atitle%253DThe%2BFusarium%2Bgraminearum%2BGenome%2BReveals%2Ba%2BLink%2BBetween%2BLocalized%2BPolymorphism%2Band%2BPathogen%2BSpecialization%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143708%26rft_id%253Dinfo%253Apmid%252F17823352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTcvNTg0My8xNDAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Lateral gene transfer in eukaryotes: tip of the iceberg or of the ice cube?</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D14%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.20" data-doi="10.1038/nature03449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebbole</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The genome sequence of the rice blast fungus <em>Magnaporthe grisea</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">434</span>(<span class="cit-issue">7036</span>): <span class="cit-fpage">980</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDean%26rft.auinit1%253DR.%2BA.%26rft.volume%253D434%26rft.issue%253D7036%26rft.spage%253D980%26rft.epage%253D986%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Brice%2Bblast%2Bfungus%2BMagnaporthe%2Bgrisea.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03449%26rft_id%253Dinfo%253Apmid%252F15846337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15846337&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228524600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.21" data-doi="10.1371/journal.pgen.1003088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Burgt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okmen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stergiopoulos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abd-Elsalam</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomes of the fungal plant pathogens <em>Cladosporium fulvum</em> and <em>Dothistroma septosporum</em> reveal adaptation to different hosts and lifestyles but also signatures of common ancestry.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1003088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253Dde%2BWit%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003088%26rft.epage%253De1003088%26rft.atitle%253DThe%2BGenomes%2Bof%2Bthe%2BFungal%2BPlant%2BPathogens%2BCladosporium%2Bfulvum%2Band%2BDothistroma%2Bseptosporum%2BReveal%2BAdaptation%2Bto%2BDifferent%2BHosts%2Band%2BLifestyles%2BBut%2BAlso%2BSignatures%2Bof%2BCommon%2BAncestry.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003088%26rft_id%253Dinfo%253Apmid%252F23209441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23209441&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dore</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perraud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dieryckx</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative genomics, proteomics and transcriptomics give new insight into the exoproteome of the basidiomycete <em>Hebeloma cylindrosporum</em> and its involvement in ectomycorrhizal symbiosis.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1169</span>–<span class="cit-lpage">1187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D1169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wargent</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fechtner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fungal endophyte infection of ryegrass reprograms host metabolism and alters development.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1227</span>–<span class="cit-lpage">1240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D1227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A core gene set describes the molecular basis of mutualism and antagonism in <em>Epichloë</em> spp.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">218</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BMicrobe%2BInteract.%26rft.volume%253D28%26rft.spage%253D218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.25" data-doi="10.1093/molbev/msr285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ebersberger</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Matos Simoes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupczok</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gube</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kothe</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A consistent phylogenetic backbone for the fungi.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1319</span>–<span class="cit-lpage">1334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr285%26rft_id%253Dinfo%253Apmid%252F22114356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzUvMTMxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.4.1301.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriegel</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jörg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> A density-based algorithm for discovering clusters in large spatial databases with noise, pp. <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span> in <em>Proceedings of the Second International Conference on Knowledge Discovery and Data Mining</em>. <span class="cit-publ-name">AAAI Press</span>, <span class="cit-publ-loc">California</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Firrincieli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otillar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome sequence of the plant growth promoting endophytic yeast <em>Rhodotorula graminis</em> WP1.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D6%26rft.spage%253D978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.28" data-doi="10.1111/j.1574-6968.2011.02465.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Horizontal gene transfer in fungi.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">329</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2011.02465.x%26rft_id%253Dinfo%253Apmid%252F22112233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjc6IjMyOS8xLzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.29" data-doi="10.1186/1471-2148-6-99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Logue</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fungal phylogeny based on 42 complete genomes derived from supertree and combined gene analysis.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DFitzpatrick%26rft.auinit1%253DD.%2BA.%26rft.volume%253D6%26rft.spage%253D99%26rft.epage%253D99%26rft.atitle%253DA%2Bfungal%2Bphylogeny%2Bbased%2Bon%2B42%2Bcomplete%2Bgenomes%2Bderived%2Bfrom%2Bsupertree%2Band%2Bcombined%2Bgene%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-6-99%26rft_id%253Dinfo%253Apmid%252F17121679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-6-99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17121679&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.30" data-doi="10.1186/1471-2148-8-181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Logue</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evidence of recent interkingdom horizontal gene transfer between bacteria and <em>Candida parapsilosis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DFitzpatrick%26rft.auinit1%253DD.%2BA.%26rft.volume%253D8%26rft.spage%253D181%26rft.epage%253D181%26rft.atitle%253DEvidence%2Bof%2Brecent%2Binterkingdom%2Bhorizontal%2Bgene%2Btransfer%2Bbetween%2Bbacteria%2Band%2BCandida%2Bparapsilosis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-181%26rft_id%253Dinfo%253Apmid%252F18577206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18577206&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.31" data-doi="10.1126/science.1221748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Floudas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchette</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Paleozoic origin of enzymatic lignin decomposition reconstructed from 31 fungal genomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>(<span class="cit-issue">6089</span>): <span class="cit-fpage">1715</span>–<span class="cit-lpage">1719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFloudas%26rft.auinit1%253DD.%26rft.volume%253D336%26rft.issue%253D6089%26rft.spage%253D1715%26rft.epage%253D1719%26rft.atitle%253DThe%2BPaleozoic%2BOrigin%2Bof%2BEnzymatic%2BLignin%2BDecomposition%2BReconstructed%2Bfrom%2B31%2BFungal%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1221748%26rft_id%253Dinfo%253Apmid%252F22745431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzYvNjA4OS8xNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.32" data-doi="10.1038/nrmicro1325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furuya</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowy</span>  <span class="cit-name-given-names">F. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Antimicrobial-resistant bacteria in the community setting.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">36</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DFuruya%26rft.auinit1%253DE.%2BY.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D45%26rft.atitle%253DAntimicrobial-resistant%2Bbacteria%2Bin%2Bthe%2Bcommunity%2Bsetting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro1325%26rft_id%253Dinfo%253Apmid%252F16357859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro1325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16357859&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234053200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.33" data-doi="10.1080/10635150701546231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A model of horizontal gene transfer and the bacterial phylogeny problem.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">633</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701546231%26rft_id%253Dinfo%253Apmid%252F17661231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjU2LzQvNjMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.34" data-doi="10.1101/gr.130000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Vallve</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romeu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Horizontal gene transfer in bacterial and archaeal complete genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1719</span>–<span class="cit-lpage">1725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DGarcia-Vallve%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D1719%26rft.epage%253D1725%26rft.atitle%253DHorizontal%2BGene%2BTransfer%2Bin%2BBacterial%2Band%2BArchaeal%2BComplete%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.130000%26rft_id%253Dinfo%253Apmid%252F11076857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMS8xNzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gazis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaButti</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipzen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genome of <em>Xylona heveae</em> provides a window into fungal endophytism.</span> <abbr class="cit-jnl-abbrev">Fungal Biol.</abbr> <span class="cit-vol">120</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">26</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BBiol.%26rft.volume%253D120%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.36" data-doi="10.1038/nrmicro1204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Horizontal gene transfer, genome innovation and evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">679</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DGogarten%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D679%26rft.epage%253D687%26rft.atitle%253DHorizontal%2Bgene%2Btransfer%252C%2Bgenome%2Binnovation%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro1204%26rft_id%253Dinfo%253Apmid%252F16138096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro1204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16138096&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231591100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.37" data-doi="10.1111/j.1469-8137.2009.02813.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grelet</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">I. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reciprocal carbon and nitrogen transfer between an ericaceous dwarf shrub and fungi isolated from <em>Piceirhiza bicolorata</em> ectomycorrhizas.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">182</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">359</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D182%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.02813.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.02813.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264635400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.38" data-doi="10.1093/nar/gkt1183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikitin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haridas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MycoCosm portal: gearing up for 1000 fungal genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D699</span>–<span class="cit-lpage">D704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1183%26rft_id%253Dinfo%253Apmid%252F24297253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9ENjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.39" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjU5LzMvMzA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.40" data-doi="10.1177/0300985813511131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gyles</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerlin</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Horizontally transferred genetic elements and their role in pathogenesis of bacterial disease.</span> <abbr class="cit-jnl-abbrev">Vet. Pathol.</abbr> <span class="cit-vol">51</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">328</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVet.%2BPathol.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0300985813511131%26rft_id%253Dinfo%253Apmid%252F24318976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3B2ZXQiO3M6NToicmVzaWQiO3M6ODoiNTEvMi8zMjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.41" data-doi="10.1105/tpc.107.052829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dothideomycete plant interactions illuminated by genome sequencing and EST analysis of the wheat pathogen <em>Stagonospora nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3347</span>–<span class="cit-lpage">3368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.052829%26rft_id%253Dinfo%253Apmid%252F18024570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8zMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Si</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rapid evolution of avirulence genes in rice blast fungus <em>Magnaporthe oryzae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D45%26rft_id%253Dinfo%253Apmid%252F24725999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24725999&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.43" data-doi="10.1186/gb-2007-8-8-109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopazo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopazo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The human phylome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">R109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHuerta-Cepas%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253DR109%26rft.epage%253DR109%26rft.atitle%253DThe%2Bhuman%2Bphylome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-8-109%26rft_id%253Dinfo%253Apmid%252F17567924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-8-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17567924&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.44" data-doi="10.1186/1471-2105-11-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopazo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ETE: a Python environment for tree exploration.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHuerta-Cepas%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DETE%253A%2Ba%2Bpython%2BEnvironment%2Bfor%2BTree%2BExploration.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-24%26rft_id%253Dinfo%253Apmid%252F20070885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070885&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.45" data-doi="10.1093/bioinformatics/btw105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacox</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauve</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szöllősi</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponty</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scornavacca</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">ecceTERA: comprehensive gene tree-species tree reconciliation using parsimony.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">2056</span>–<span class="cit-lpage">2058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw105%26rft_id%253Dinfo%253Apmid%252F27153713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzIvMTMvMjA1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.46" data-doi="10.1038/nature05110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauff</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matheny</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofstetter</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reconstructing the early evolution of fungi using a six-gene phylogeny.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>(<span class="cit-issue">7113</span>): <span class="cit-fpage">818</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJames%26rft.auinit1%253DT.%2BY.%26rft.volume%253D443%26rft.issue%253D7113%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DReconstructing%2Bthe%2Bearly%2Bevolution%2Bof%2BFungi%2Busing%2Ba%2Bsix-gene%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05110%26rft_id%253Dinfo%253Apmid%252F17051209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17051209&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241362700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.47" data-doi="10.1093/bioinformatics/btt727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaron</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moravec</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínková</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SigHunt: horizontal gene transfer finder optimized for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1081</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt727%26rft_id%253Dinfo%253Apmid%252F24371153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC84LzEwODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.48" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvNzcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.49" data-doi="10.1093/nar/gkf436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">3059</span>–<span class="cit-lpage">3066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DMAFFT%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Brapid%2Bmultiple%2Bsequence%2Balignment%2Bbased%2Bon%2Bfast%2BFourier%2Btransform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf436%26rft_id%253Dinfo%253Apmid%252F12136088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xNC8zMDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.50" data-doi="10.1128/AEM.68.7.3345-3351.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolla</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nalin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">In situ transfer of antibiotic resistance genes from transgenic (transplastomic) tobacco plants to bacteria.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">68</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">3345</span>–<span class="cit-lpage">3351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DKay%26rft.auinit1%253DE.%26rft.volume%253D68%26rft.issue%253D7%26rft.spage%253D3345%26rft.epage%253D3351%26rft.atitle%253DIn%2BSitu%2BTransfer%2Bof%2BAntibiotic%2BResistance%2BGenes%2Bfrom%2BTransgenic%2B%2528Transplastomic%2529%2BTobacco%2BPlants%2Bto%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.68.7.3345-3351.2002%26rft_id%253Dinfo%253Apmid%252F12089013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY4LzcvMzM0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.51" data-doi="10.1038/nrg2386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keeling</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Horizontal gene transfer in eukaryotic evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">605</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKeeling%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D605%26rft.epage%253D618%26rft.atitle%253DHorizontal%2Bgene%2Btransfer%2Bin%2Beukaryotic%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2386%26rft_id%253Dinfo%253Apmid%252F18591983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18591983&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257758400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khaldi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collemare</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebrun</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evidence for horizontal transfer of a secondary metabolite gene cluster between fungi.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.53" data-doi="10.1371/journal.ppat.1002137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klosterman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subbarao</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veronese</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative genomics yields insights into niche adaptation of plant vascular wilt pathogens.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1002137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DKlosterman%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002137%26rft.epage%253De1002137%26rft.atitle%253DComparative%2Bgenomics%2Byields%2Binsights%2Binto%2Bniche%2Badaptation%2Bof%2Bplant%2Bvascular%2Bwilt%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002137%26rft_id%253Dinfo%253Apmid%252F21829347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829347&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.54" data-doi="10.1038/ng.3223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Convergent losses of decay mechanisms and rapid turnover of symbiosis genes in mycorrhizal mutualists.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">410</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D410%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3223%26rft_id%253Dinfo%253Apmid%252F25706625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706625&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.55" data-doi="10.1093/oxfordjournals.molbev.a003816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koski</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golding</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Codon bias and base composition are poor indicators of horizontally transferred genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">404</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DKoski%26rft.auinit1%253DL.%2BB.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D404%26rft.epage%253D412%26rft.atitle%253DCodon%2BBias%2Band%2BBase%2BComposition%2BAre%2BPoor%2BIndicators%2Bof%2BHorizontally%2BTransferred%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003816%26rft_id%253Dinfo%253Apmid%252F11230541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE4LzMvNDA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.56" data-doi="10.1038/nature14963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson-Sathi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roettger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockhart</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Endosymbiotic origin and differential loss of eukaryotic genes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">524</span>(<span class="cit-issue">7566</span>): <span class="cit-fpage">427</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D524%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14963%26rft_id%253Dinfo%253Apmid%252F26287458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26287458&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.57" data-doi="10.1007/PL00006158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Amelioration of bacterial genomes: rates of change and exchange.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">383</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DLawrence%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D383%26rft.epage%253D397%26rft.atitle%253DAmelioration%2Bof%2Bbacterial%2Bgenomes%253A%2Brates%2Bof%2Bchange%2Band%2Bexchange.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006158%26rft_id%253Dinfo%253Apmid%252F9089078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9089078&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR61800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.58" data-doi="10.1093/molbev/msn067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An improved general amino acid replacement matrix.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1307</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn067%26rft_id%253Dinfo%253Apmid%252F18367465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTMwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.59" data-doi="10.1111/j.1365-294X.2005.02796.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Quere</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajashekar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schutzendubel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canback</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Screening for rapidly evolving genes in the ectomycorrhizal fungus <em>Paxillus involutus</em> using cDNA microarrays.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">535</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLE%2BQuere%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D535%26rft.epage%253D550%26rft.atitle%253DScreening%2Bfor%2Brapidly%2Bevolving%2Bgenes%2Bin%2Bthe%2Bectomycorrhizal%2Bfungus%2BPaxillus%2Binvolutus%2Busing%2BcDNA%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02796.x%26rft_id%253Dinfo%253Apmid%252F16448419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02796.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16448419&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.60" data-doi="10.1186/1471-2148-11-276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Widespread horizontal gene transfer from circular single-stranded DNA viruses to eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.spage%253D276%26rft.epage%253D276%26rft.atitle%253DWidespread%2Bhorizontal%2Bgene%2Btransfer%2Bfrom%2Bcircular%2Bsingle-stranded%2BDNA%2Bviruses%2Bto%2Beukaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-276%26rft_id%253Dinfo%253Apmid%252F21943216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21943216&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.61" data-doi="10.1186/1471-2148-6-74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodson</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Loss of the flagellum happened only once in the fungal lineage: phylogenetic structure of Kingdom Fungi inferred from RNA polymerase II subunit genes.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DLiu%26rft.auinit1%253DY.%2BJ.%26rft.volume%253D6%26rft.spage%253D74%26rft.epage%253D74%26rft.atitle%253DLoss%2Bof%2Bthe%2Bflagellum%2Bhappened%2Bonly%2Bonce%2Bin%2Bthe%2Bfungal%2Blineage%253A%2Bphylogenetic%2Bstructure%2Bof%2Bkingdom%2BFungi%2Binferred%2Bfrom%2BRNA%2Bpolymerase%2BII%2Bsubunit%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-6-74%26rft_id%253Dinfo%253Apmid%252F17010206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-6-74&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17010206&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.62" data-doi="10.1038/nature08850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Does</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daboussi</span>  <span class="cit-name-given-names">M.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative genomics reveals mobile pathogenicity chromosomes in <em>Fusarium</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>(<span class="cit-issue">7287</span>): <span class="cit-fpage">367</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMa%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D464%26rft.issue%253D7287%26rft.spage%253D367%26rft.epage%253D373%26rft.atitle%253DComparative%2Bgenomics%2Breveals%2Bmobile%2Bpathogenicity%2Bchromosomes%2Bin%2BFusarium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08850%26rft_id%253Dinfo%253Apmid%252F20237561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20237561&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275657100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.63" data-doi="10.1093/sysbio/46.3.523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maddison</span>  <span class="cit-name-given-names">W. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gene trees in species trees.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">523</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F46.3.523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjQ2LzMvNTIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.64" data-doi="10.1186/1471-2164-11-171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mallet</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becq</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deschavanne</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole genome evaluation of horizontal transfers in the pathogenic fungus <em>Aspergillus fumigatus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMallet%26rft.auinit1%253DL.%2BV.%26rft.volume%253D11%26rft.spage%253D171%26rft.epage%253D171%26rft.atitle%253DWhole%2Bgenome%2Bevaluation%2Bof%2Bhorizontal%2Btransfers%2Bin%2Bthe%2Bpathogenic%2Bfungus%2BAspergillus%2Bfumigatus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-171%26rft_id%253Dinfo%253Apmid%252F20226043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20226043&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.65" data-doi="10.1016/j.tig.2009.11.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Acquisition of prokaryotic genes by fungal genomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMarcet-Houben%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D8%26rft.atitle%253DAcquisition%2Bof%2Bprokaryotic%2Bgenes%2Bby%2Bfungal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.11.007%26rft_id%253Dinfo%253Apmid%252F19969385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.11.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19969385&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273884400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.66" data-doi="10.1038/nature06556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahren</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brun</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genome of <em>Laccaria bicolor</em> provides insights into mycorrhizal symbiosis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>(<span class="cit-issue">7183</span>): <span class="cit-fpage">88</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMartin%26rft.auinit1%253DF.%26rft.volume%253D452%26rft.issue%253D7183%26rft.spage%253D88%26rft.epage%253D92%26rft.atitle%253DThe%2Bgenome%2Bof%2BLaccaria%2Bbicolor%2Bprovides%2Binsights%2Binto%2Bmycorrhizal%2Bsymbiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06556%26rft_id%253Dinfo%253Apmid%252F18322534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18322534&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253671900052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.67" data-doi="10.1038/nature08867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murat</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balestrini</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coutinho</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Perigord black truffle genome uncovers evolutionary origins and mechanisms of symbiosis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>(<span class="cit-issue">7291</span>): <span class="cit-fpage">1033</span>–<span class="cit-lpage">1038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMartin%26rft.auinit1%253DF.%26rft.volume%253D464%26rft.issue%253D7291%26rft.spage%253D1033%26rft.epage%253D1038%26rft.atitle%253DPerigord%2Bblack%2Btruffle%2Bgenome%2Buncovers%2Bevolutionary%2Borigins%2Band%2Bmechanisms%2Bof%2Bsymbiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08867%26rft_id%253Dinfo%253Apmid%252F20348908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20348908&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276635000037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.68" data-doi="10.1126/science.1159483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Antibiotics and antibiotic resistance genes in natural environments.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">321</span>(<span class="cit-issue">5887</span>): <span class="cit-fpage">365</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMartinez%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D321%26rft.issue%253D5887%26rft.spage%253D365%26rft.epage%253D367%26rft.atitle%253DAntibiotics%2Band%2BAntibiotic%2BResistance%2BGenes%2Bin%2BNatural%2BEnvironments%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1159483%26rft_id%253Dinfo%253Apmid%252F18635792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjEvNTg4Ny8zNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.69" data-doi="10.1093/bioinformatics/bts118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menigaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picord</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churlaud</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GOHTAM: a website for ‘genomic origin of horizontal transfers, alignment and metagenomics.’</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1270</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts118%26rft_id%253Dinfo%253Apmid%252F22426345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyOC85LzEyNzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mesarich</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamiaux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templeton</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Repeat-containing protein effectors of plant-associated organisms.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.71" data-doi="10.1101/gr.091686.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagarete</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vargas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Horizontal gene transfer of an entire metabolic pathway between a eukaryotic alga and its DNA virus.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1441</span>–<span class="cit-lpage">1449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091686.109%26rft_id%253Dinfo%253Apmid%252F19451591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzgvMTQ0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-book" id="cit-7.4.1301.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collemare</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebrun</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary mechanisms involved in development of fungal secondary metabolite gene clusters</span>, pp. <span class="cit-fpage">341</span>–<span class="cit-lpage">356</span> in <span class="cit-source">Natural Products</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.73" data-doi="10.1093/molbev/mss089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szczesny</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grynberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technau</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recurrent horizontal transfer of bacterial toxin genes to eukaryotes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">2223</span>–<span class="cit-lpage">2230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss089%26rft_id%253Dinfo%253Apmid%252F22411854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMjIyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.74" data-doi="10.1038/srep11571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterck</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swennen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onesime</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Differential gene retention as an evolutionary mechanism to generate biodiversity and adaptation in yeasts.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">11571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D11571%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep11571%26rft_id%253Dinfo%253Apmid%252F26108467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep11571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26108467&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.75" data-doi="10.1038/ng1381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Biased biological functions of horizontally transferred genes in prokaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">760</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNakamura%26rft.auinit1%253DY.%26rft.volume%253D36%26rft.issue%253D7%26rft.spage%253D760%26rft.epage%253D766%26rft.atitle%253DBiased%2Bbiological%2Bfunctions%2Bof%2Bhorizontally%2Btransferred%2Bgenes%2Bin%2Bprokaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1381%26rft_id%253Dinfo%253Apmid%252F15208628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15208628&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222354100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.4.1301.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyon</span>  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pointet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chifolleau</span>  <span class="cit-name-given-names">A.-M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranwez</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> Accounting for gene tree uncertainties improves gene trees and reconciliation inference, pp. <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span> in WABI’12: Workshop on Algorithms in Bioinformatics, September, 2012, Ljubljana, Slovenia.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.77" data-doi="10.1093/molbev/mst206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nikolaidis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doran</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosgrove</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Plant expansins in bacteria and fungi: evolution by horizontal gene transfer and independent domain fusion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">376</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst206%26rft_id%253Dinfo%253Apmid%252F24150040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzIvMzc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.78" data-doi="10.1038/ng.2372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thon</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hacquard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amyotte</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleemann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lifestyle transitions in plant pathogenic <em>Colletotrichum</em> fungi deciphered by genome and transcriptome analyses.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1060</span>–<span class="cit-lpage">1065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DO%2527Connell%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1060%26rft.epage%253D1065%26rft.atitle%253DLifestyle%2Btransitions%2Bin%2Bplant%2Bpathogenic%2BColletotrichum%2Bfungi%2Bdeciphered%2Bby%2Bgenome%2Band%2Btranscriptome%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2372%26rft_id%253Dinfo%253Apmid%252F22885923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22885923&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.79" data-doi="10.1038/35012500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groisman</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lateral gene transfer and the nature of bacterial innovation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>(<span class="cit-issue">6784</span>): <span class="cit-fpage">299</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOchman%26rft.auinit1%253DH.%26rft.volume%253D405%26rft.issue%253D6784%26rft.spage%253D299%26rft.epage%253D304%26rft.atitle%253DLateral%2Bgene%2Btransfer%2Band%2Bthe%2Bnature%2Bof%2Bbacterial%2Binnovation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35012500%26rft_id%253Dinfo%253Apmid%252F10830951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35012500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10830951&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087085700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.80" data-doi="10.1371/journal.ppat.1003037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feau</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horwitz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen <em>Dothideomycetes</em> fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1003037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D8%26rft.spage%253De1003037%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003037%26rft_id%253Dinfo%253Apmid%252F23236275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236275&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.81" data-doi="10.1089/cmb.2009.0240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ovadia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fielder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conow</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libeskind-Hadas</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The cophylogeny reconstruction problem is NP-complete.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">59</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DOvadia%26rft.auinit1%253DY.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D65%26rft.atitle%253DThe%2Bco%2Bphylogeny%2Breconstruction%2Bproblem%2Bis%2BNP-complete.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2009.0240%26rft_id%253Dinfo%253Apmid%252F20715926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2009.0240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20715926&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pamilo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Relationships between gene trees and species trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">568</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DPamilo%26rft.auinit1%253DP%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D568%26rft.epage%253D583%26rft.atitle%253DRelationships%2Bbetween%2Bgene%2Btrees%2Band%2Bspecies%2Btrees%26rft_id%253Dinfo%253Apmid%252F3193878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjUvNS81NjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varoquaux</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gramfort</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thirion</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn: machine learning in Python.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poptsova</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Testing phylogenetic methods to identify horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">532</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DPoptsova%26rft.auinit1%253DM.%26rft.volume%253D532%26rft.spage%253D227%26rft.epage%253D240%26rft.atitle%253DTesting%2Bphylogenetic%2Bmethods%2Bto%2Bidentify%2Bhorizontal%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Apmid%252F19271188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19271188&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.85" data-doi="10.1186/1471-2148-7-45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poptsova</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The power of phylogenetic approaches to detect horizontally transferred genes.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DPoptsova%26rft.auinit1%253DM.%2BS.%26rft.volume%253D7%26rft.spage%253D45%26rft.epage%253D45%26rft.atitle%253DThe%2Bpower%2Bof%2Bphylogenetic%2Bapproaches%2Bto%2Bdetect%2Bhorizontally%2Btransferred%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-45%26rft_id%253Dinfo%253Apmid%252F17376230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17376230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.86" data-doi="10.1111/j.1574-6968.2001.tb10755.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ragan</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">On surrogate methods for detecting lateral gene transfer.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">201</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">187</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2001.tb10755.x%26rft_id%253Dinfo%253Apmid%252F11470360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjk6IjIwMS8yLzE4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.87" data-doi="10.1128/JB.06601-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redondo-Nieto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barret</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrisey</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Germaine</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Granero</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequence of the biocontrol strain <em>Pseudomonas fluorescens</em> F113.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1273</span>–<span class="cit-lpage">1274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DRedondo-Nieto%26rft.auinit1%253DM.%26rft.volume%253D194%26rft.issue%253D5%26rft.spage%253D1273%26rft.epage%253D1274%26rft.atitle%253DGenome%2BSequence%2Bof%2Bthe%2BBiocontrol%2BStrain%2BPseudomonas%2Bfluorescens%2BF113%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.06601-11%26rft_id%253Dinfo%253Apmid%252F22328765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5NC81LzEyNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.88" data-doi="10.1128/EC.00106-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dacks</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolutionary origins of the eukaryotic shikimate pathway: gene fusions, horizontal gene transfer, and endosymbiotic replacements.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1517</span>–<span class="cit-lpage">1531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DRichards%26rft.auinit1%253DT.%2BA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253D1517%26rft.epage%253D1531%26rft.atitle%253DEvolutionary%2BOrigins%2Bof%2Bthe%2BEukaryotic%2BShikimate%2BPathway%253A%2BGene%2BFusions%252C%2BHorizontal%2BGene%2BTransfer%252C%2Band%2BEndosymbiotic%2BReplacements%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00106-06%26rft_id%253Dinfo%253Apmid%252F16963634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI1LzkvMTUxNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.89" data-doi="10.1105/tpc.109.065805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soanes</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Phylogenomic analysis demonstrates a pattern of rare and ancient horizontal gene transfer between plants and fungi.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1897</span>–<span class="cit-lpage">1911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.065805%26rft_id%253Dinfo%253Apmid%252F19584142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTg5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.90" data-doi="10.1016/0025-5564(81)90043-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Comparison of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">131</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D53%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252881%252990043-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(81)90043-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LB66300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogul</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGee</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falkow</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Nucleic acid homologies of selected bacteria, L forms, and <em>Mycoplasma</em> species.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1200</span>–<span class="cit-lpage">1204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DRogul%26rft.auinit1%253DM.%26rft.volume%253D90%26rft.issue%253D5%26rft.spage%253D1200%26rft.epage%253D1204%26rft.atitle%253DNucleic%2BAcid%2BHomologies%2Bof%2BSelected%2BBacteria%252C%2BL%2BForms%252C%2Band%2BMycoplasma%2BSpecies%26rft_id%253Dinfo%253Apmid%252F5848325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiOTAvNS8xMjAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.92" data-doi="10.1146/annurev.phyto.38.1.325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosewich</span>  <span class="cit-name-given-names">U. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kistler</span>  <span class="cit-name-given-names">H. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Role of horizontal gene transfer in the evolution of fungi.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DRosewich%26rft.auinit1%253DU.%2BL.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D325%26rft.epage%253D363%26rft.atitle%253DROLE%2BOF%2BHORIZONTAL%2BGENE%2BTRANSFER%2BIN%2BTHE%2BEVOLUTION%2BOF%2BFUNGI%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.38.1.325%26rft_id%253Dinfo%253Apmid%252F11701846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.38.1.325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701846&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165322300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.93" data-doi="10.1038/ncomms1189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouxel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandaubert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Wouw</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effector diversification within compartments of the <em>Leptosphaeria maculans</em> genome affected by repeat-induced point mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DRouxel%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.spage%253D202%26rft.epage%253D202%26rft.atitle%253DEffector%2Bdiversification%2Bwithin%2Bcompartments%2Bof%2Bthe%2BLeptosphaeria%2Bmaculans%2Bgenome%2Baffected%2Bby%2BRepeat-Induced%2BPoint%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1189%26rft_id%253Dinfo%253Apmid%252F21326234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21326234&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.94" data-doi="10.1371/journal.pone.0018755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evaluating ortholog prediction algorithms in a yeast model clade.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e18755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSalichos%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De18755%26rft.epage%253De18755%26rft.atitle%253DEvaluating%2Bortholog%2Bprediction%2Balgorithms%2Bin%2Ba%2Byeast%2Bmodel%2Bclade.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018755%26rft_id%253Dinfo%253Apmid%252F21533202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21533202&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.95" data-doi="10.1038/nature12130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring ancient divergences requires genes with strong phylogenetic signals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>(<span class="cit-issue">7449</span>): <span class="cit-fpage">327</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12130%26rft_id%253Dinfo%253Apmid%252F23657258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23657258&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318952000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.96" data-doi="10.1093/nar/gkn741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sayers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canese</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Database resources of the national center for biotechnology information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D5</span>–<span class="cit-lpage">D15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn741%26rft_id%253Dinfo%253Apmid%252F18940862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEzOiIzNy9zdXBwbF8xL0Q1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.97" data-doi="10.1006/fgbi.2001.1275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schardl</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>Epichloë festucae</em> and related mutualistic symbionts of grasses.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DSchardl%26rft.auinit1%253DC.%2BL.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D69%26rft.epage%253D82%26rft.atitle%253DEpichloe%2Bfestucae%2Band%2Brelated%2Bmutualistic%2Bsymbionts%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.2001.1275%26rft_id%253Dinfo%253Apmid%252F11456460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.2001.1275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11456460&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170175200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.98" data-doi="10.1371/journal.pgen.1003323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schardl</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesse</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amyotte</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreeva</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Plant-symbiotic fungi as chemical engineers: multi-genome analysis of the Clavicipitaceae reveals dynamics of alkaloid loci.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1003323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003323%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003323%26rft_id%253Dinfo%253Apmid%252F23468653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23468653&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.99" data-doi="10.1126/science.1195330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schirawski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mannhaupt</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brefort</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipper</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Pathogenicity determinants in smut fungi revealed by genome comparison.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>(<span class="cit-issue">6010</span>): <span class="cit-fpage">1546</span>–<span class="cit-lpage">1548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchirawski%26rft.auinit1%253DJ.%26rft.volume%253D330%26rft.issue%253D6010%26rft.spage%253D1546%26rft.epage%253D1548%26rft.atitle%253DPathogenicity%2BDeterminants%2Bin%2BSmut%2BFungi%2BRevealed%2Bby%2BGenome%2BComparison%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1195330%26rft_id%253Dinfo%253Apmid%252F21148393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMC8xNTQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.100" data-doi="10.1016/0022-2836(69)90298-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Mutations caused by the insertion of genetic material into the galactose operon of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">93</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DShapiro%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D105%26rft.atitle%253DMutations%2Bcaused%2Bby%2Bthe%2Binsertion%2Bof%2Bgenetic%2Bmaterial%2Binto%2Bthe%2Bgalactose%2Boperon%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252869%252990298-8%26rft_id%253Dinfo%253Apmid%252F4903362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(69)90298-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4903362&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1969C796300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.101" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The codon adaptation index—a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.102" data-doi="10.1146/annurev-phyto-102313-050127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soanes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Horizontal gene transfer in eukaryotic plant pathogens.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPhytopathol.%26rft.volume%253D52%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-phyto-102313-050127%26rft_id%253Dinfo%253Apmid%252F25090479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-phyto-102313-050127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25090479&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.103" data-doi="10.1038/nrg3962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soucy</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Horizontal gene transfer: building the web of life.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">472</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D472%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3962%26rft_id%253Dinfo%253Apmid%252F26184597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26184597&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.104" data-doi="10.1126/science.1194573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spanu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amselem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgis</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soanes</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome expansion and gene loss in powdery mildew fungi reveal tradeoffs in extreme parasitism.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>(<span class="cit-issue">6010</span>): <span class="cit-fpage">1543</span>–<span class="cit-lpage">1546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSpanu%26rft.auinit1%253DP.%2BD.%26rft.volume%253D330%26rft.issue%253D6010%26rft.spage%253D1543%26rft.epage%253D1546%26rft.atitle%253DGenome%2BExpansion%2Band%2BGene%2BLoss%2Bin%2BPowdery%2BMildew%2BFungi%2BReveal%2BTradeoffs%2Bin%2BExtreme%2BParasitism%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194573%26rft_id%253Dinfo%253Apmid%252F21148392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMC8xNTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.105" data-doi="10.1016/S0959-437X(05)80203-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sprague</span>  <span class="cit-name-given-names">G. F.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic exchange between kingdoms.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">530</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DSprague%26rft.auinit1%253DG.%2BF.%26rft.volume%253D1%26rft.issue%253D4%26rft.spage%253D530%26rft.epage%253D533%26rft.atitle%253DGenetic%2Bexchange%2Bbetween%2Bkingdoms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252805%252980203-5%26rft_id%253Dinfo%253Apmid%252F1822285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(05)80203-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1822285&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.106" data-doi="10.1128/EC.00164-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staats</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kan</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome update of <em>Botrytis cinerea</em> strains B05.10 and T4.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1413</span>–<span class="cit-lpage">1414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DStaats%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1413%26rft.epage%253D1414%26rft.atitle%253DGenome%2BUpdate%2Bof%2BBotrytis%2Bcinerea%2BStrains%2BB05.10%2Band%2BT4%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00164-12%26rft_id%253Dinfo%253Apmid%252F23104368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czoxMDoiMTEvMTEvMTQxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.107" data-doi="10.1093/bioinformatics/bts386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stolzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sathaye</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vernot</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inferring duplications, losses, transfers and incomplete lineage sorting with nonbinary species trees.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">18</span>): <span class="cit-fpage">i409</span>–<span class="cit-lpage">i415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts386%26rft_id%253Dinfo%253Apmid%252F22962460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvaTQwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.108" data-doi="10.1093/molbev/mss201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved implementation of effective number of codons (<em>N</em>c).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">191</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss201%26rft_id%253Dinfo%253Apmid%252F22915832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzEvMTkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.109" data-doi="10.1146/annurev.ge.28.120194.001321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Syvanen</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Horizontal gene transfer: evidence and possible consequences.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSyvanen%26rft.auinit1%253DM%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D237%26rft.epage%253D261%26rft.atitle%253DHorizontal%2BGene%2BFlow%253A%2BEvidence%2Band%2BPossible%2BConsequences%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.28.120194.001321%26rft_id%253Dinfo%253Apmid%252F7893125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.28.120194.001321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7893125&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY64400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.110" data-doi="10.1098/rstb.2014.0335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szöllősi</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tannier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daubin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boussau</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-scale phylogenetic analysis finds extensive gene transfer among fungi.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. B Biol. Sci.</abbr> <span class="cit-vol">370</span>(<span class="cit-issue">1678</span>): <span class="cit-fpage">20140335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2014.0335%26rft_id%253Dinfo%253Apmid%252F26323765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNzAvMTY3OC8yMDE0MDMzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.111" data-doi="10.1105/tpc.105.039263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takemoto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reactive oxygen species play a role in regulating a fungus-perennial ryegrass mutualistic interaction.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1052</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.105.039263%26rft_id%253Dinfo%253Apmid%252F16517760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE4LzQvMTA1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Temporini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Etten</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An analysis of the pea pathogenicity genes of <em>Nectria haematococca</em> MPVI supports the hypothesis of their origin by horizontal transfer and uncovers a potentially new pathogen of garden pea: <em>Neocosmospora boniensis</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DTemporini%26rft.auinit1%253DE.%2BD.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D36%26rft.atitle%253DAn%2Banalysis%2Bof%2Bthe%2Bphylogenetic%2Bdistribution%2Bof%2Bthe%2Bpea%2Bpathogenicity%2Bgenes%2Bof%2BNectria%2Bhaematococca%2BMPVI%2Bsupports%2Bthe%2Bhypothesis%2Bof%2Btheir%2Borigin%2Bby%2Bhorizontal%2Btransfer%2Band%2Buncovers%2Ba%2Bpotentially%2Bnew%2Bpathogen%2Bof%2Bgarden%2Bpea%253A%2BNeocosmospora%2Bboniensis.%26rft_id%253Dinfo%253Apmid%252F15118835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15118835&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.113" data-doi="10.1098/rspb.2014.0028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terhorst</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The relative importance of rapid evolution for plant-microbe interactions depends on ecological context.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>(1785): <span class="cit-fpage">20140028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.0028%26rft_id%253Dinfo%253Apmid%252F24789894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3ODUvMjAxNDAwMjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.114" data-doi="10.1089/cmb.2007.A010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Than</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruths</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakhleh</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Confounding factors in HGT detection: statistical error, coalescent effects, and multiple solutions.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">517</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DThan%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D517%26rft.epage%253D535%26rft.atitle%253DConfounding%2Bfactors%2Bin%2BHGT%2Bdetection%253A%2Bstatistical%2Berror%252C%2Bcoalescent%2Beffects%252C%2Band%2Bmultiple%2Bsolutions.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2007.A010%26rft_id%253Dinfo%253Apmid%252F17572027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2007.A010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572027&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247526900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.115" data-doi="10.1111/nph.12653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toome</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazarus</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing provides insight into the reproductive biology, nutritional mode and ploidy of the fern pathogen <em>Mixia osmundae</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">202</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">554</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D202%26rft.spage%253D554%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.116" data-doi="10.1126/science.1128796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tripathy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>Phytophthora</em> genome sequences uncover evolutionary origins and mechanisms of pathogenesis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>(<span class="cit-issue">5791</span>): <span class="cit-fpage">1261</span>–<span class="cit-lpage">1266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTyler%26rft.auinit1%253DB.%2BM.%26rft.volume%253D313%26rft.issue%253D5791%26rft.spage%253D1261%26rft.epage%253D1266%26rft.atitle%253DPhytophthora%2Bgenome%2Bsequences%2Buncover%2Bevolutionary%2Borigins%2Band%2Bmechanisms%2Bof%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128796%26rft_id%253Dinfo%253Apmid%252F16946064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5MS8xMjYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1301.117" data-doi="10.1186/1471-2148-7-147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wägele</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Visualizing differences in phylogenetic information content of alignments and distinction of three classes of long-branch effects.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DWaegele%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D7%26rft.spage%253D147%26rft.epage%253D147%26rft.atitle%253DVisualizing%2Bdifferences%2Bin%2Bphylogenetic%2Binformation%2Bcontent%2Bof%2Balignments%2Band%2Bdistinction%2Bof%2Bthree%2Bclasses%2Bof%2Blong-branch%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-147%26rft_id%253Dinfo%253Apmid%252F17725833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17725833&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.118" data-doi="10.1186/1471-2148-9-195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A fungal phylogeny based on 82 complete genomes using the composition vector method.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D9%26rft.spage%253D195%26rft.epage%253D195%26rft.atitle%253DA%2Bfungal%2Bphylogeny%2Bbased%2Bon%2B82%2Bcomplete%2Bgenomes%2Busing%2Bthe%2Bcomposition%2Bvector%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-9-195%26rft_id%253Dinfo%253Apmid%252F19664262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-9-195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19664262&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.119" data-doi="10.1093/bioinformatics/btu258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weyenberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huggins</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schardl</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">kdetrees: non-parametric estimation of phylogenetic tree distributions.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2280</span>–<span class="cit-lpage">2287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu258%26rft_id%253Dinfo%253Apmid%252F24764459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTYvMjI4MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.120" data-doi="10.1093/sysbio/syu023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whidden</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beiko</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Supertrees based on the subtree prune-and-regraft distance.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">63</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">566</span>–<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu023%26rft_id%253Dinfo%253Apmid%252F24695589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjYzLzQvNTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.121" data-doi="10.1371/journal.pgen.1004816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisecaver</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slot</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evolution of fungal metabolic pathways.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1004816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004816%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004816%26rft_id%253Dinfo%253Apmid%252F25474404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25474404&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.122" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The ‘effective number of codons’ used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference  in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1301.123" data-doi="10.1007/978-1-60327-853-9_11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhaxybayeva</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection and quantitative assessment of horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">532</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DZhaxybayeva%26rft.auinit1%253DO.%26rft.volume%253D532%26rft.spage%253D195%26rft.epage%253D213%26rft.atitle%253DDetection%2Band%2Bquantitative%2Bassessment%2Bof%2Bhorizontal%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-853-9_11%26rft_id%253Dinfo%253Apmid%252F19271186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-853-9_11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19271186&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/4/1301.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1287" title="Independent Maternal and Fetal Genetic Effects on Midgestational Circulating Levels of Environmental Pollutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1315" title="A Nonsense Variant in the ST14 Gene in Akhal-Teke Horses with Naked Foal Syndrome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/4"><span>Volume 7 Issue 4, April 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/4.cover-source.jpg?itok=KOTpzmYp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1301" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DO2OZuysi93K0Ag4ql6G43rT4CcxPNSgQT7c9EClNSE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/4/1301" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/4/1301" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/4/1301" class="active" data-icon-position="" data-hide-link-title="0">Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483844" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rDt0k7kUkxbUQI3jm3AYtMJ-Uu4cDk8dxhwVU41bUt8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1301.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/4/1301" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-8oBThy5BrWxERKV3BXcG-qbLel3tOegZ1FFb33MWAL4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/4/1301" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1301.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483844" id="node483844--2" data-pisa="ggg;7/4/1301" data-pisa-master="ggg;g3.116.038448" data-apath="/ggg/7/4/1301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7384-5181" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre-Yves Dupont</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1936-0236" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Murray P. Cox</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1301-1314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038448 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483844--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pierre-Yves Dupont </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Statistics and Bioinformatics Group, Institute of Fundamental Sciences, Massey University, Palmerston North 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre-Yves%2BDupont%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dupont%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre-Yves%2BDupont%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7384-5181" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre-Yves Dupont</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Murray P. Cox </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murray%2BP.%2BCox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cox%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMurray%2BP.%2BCox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1936-0236" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Murray P. Cox</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.p.cox@massey.ac.nz" class="" data-icon-position="" data-hide-link-title="0">m.p.cox@massey.ac.nz</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/4/1301" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/4/1301&amp;title=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/4/1301&amp;title=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/4/1301&amp;title=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/4/1301&amp;text=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/4/1301&amp;title=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/4/1301&amp;t=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/4/1301&amp;title=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/4/1301&amp;title=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483844/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483844" id="node483844--3" data-pisa="ggg;7/4/1301" data-pisa-master="ggg;g3.116.038448" data-apath="/ggg/7/4/1301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Data Quality Impacts Automated Detection of Lateral Gene Transfer in Fungi</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7384-5181" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre-Yves Dupont</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1936-0236" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Murray P. Cox</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1301-1314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038448 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483844--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pierre-Yves Dupont </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Statistics and Bioinformatics Group, Institute of Fundamental Sciences, Massey University, Palmerston North 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre-Yves%2BDupont%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dupont%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre-Yves%2BDupont%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7384-5181" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre-Yves Dupont</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Murray P. Cox </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murray%2BP.%2BCox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cox%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMurray%2BP.%2BCox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1936-0236" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Murray P. Cox</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.p.cox@massey.ac.nz" class="" data-icon-position="" data-hide-link-title="0">m.p.cox@massey.ac.nz</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1301&amp;rft.epage=1314&amp;rft.atitle=Genomic%20Data%20Quality%20Impacts%20Automated%20Detection%20of%20Lateral%20Gene%20Transfer%20in%20Fungi&amp;rft.volume=7&amp;rft.issue=4&amp;rft.date=2017-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dupont%2C+Pierre-Yves&amp;rft.au=Cox%2C+Murray+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483844/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483844/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483844/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483844/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483844/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483844/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483844/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483844/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483844/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483844/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483844/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483844/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/4/1301" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/4/1301.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/4/1301.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/4/1301.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/4/1301.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
