<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/4/1085" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/4/1085.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/4/1085.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/4/1085.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/4/1085/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/4/1085.atom" />
<meta name="HW.pisa" content="ggg;7/4/1085" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in Mimulus" />
<meta name="DC.Identifier" content="10.1534/g3.116.038075" />
<meta name="DC.Date" content="2017-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Stanton et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="While quantitative PCR (qPCR) is widely recognized as being among the most accurate methods for quantifying gene expression, it is highly dependent on the use of reliable, stably expressed reference genes. With the increased availability of high-throughput methods for measuring gene expression, whole-transcriptome approaches may be increasingly utilized for reference gene selection and validation. In this study, RNA-seq was used to identify a set of novel qPCR reference genes and evaluate a panel of traditional “housekeeping” reference genes in two species of the evolutionary model plant genus Mimulus . More broadly, the methods proposed in this study can be used to harness the power of transcriptomes to identify appropriate reference genes for qPCR in any study organism, including emerging and nonmodel systems. We find that RNA-seq accurately estimates gene expression means in comparison to qPCR, and that expression means are robust to moderate environmental and genetic variation. However, measures of expression variability were only in agreement with qPCR for samples obtained from a shared environment. This result, along with transcriptome-wide comparisons, suggests that environmental changes have greater impacts on expression variability than on expression means. We discuss how this issue can be addressed through experimental design, and suggest that the ever-expanding pool of published transcriptomes represents a rich and low-cost resource for developing better reference genes for qPCR." />
<meta name="DC.Contributor" content="Kimmy A. Stanton" />
<meta name="DC.Contributor" content="Patrick P. Edger" />
<meta name="DC.Contributor" content="Joshua R. Puzey" />
<meta name="DC.Contributor" content="Taliesin Kinser" />
<meta name="DC.Contributor" content="Philip Cheng" />
<meta name="DC.Contributor" content="Daniel M. Vernon" />
<meta name="DC.Contributor" content="Nancy R. Forsthoefel" />
<meta name="DC.Contributor" content="Arielle M. Cooley" />
<meta name="article:published_time" content="2017-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in Mimulus" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;While quantitative PCR (qPCR) is widely recognized as being among the most accurate methods for quantifying gene expression, it is highly dependent on the use of reliable, stably expressed reference genes. With the increased availability of high-throughput methods for measuring gene expression, whole-transcriptome approaches may be increasingly utilized for reference gene selection and validation. In this study, RNA-seq was used to identify a set of novel qPCR reference genes and evaluate a panel of traditional “housekeeping” reference genes in two species of the evolutionary model plant genus &lt;i&gt;Mimulus&lt;/i&gt;. More broadly, the methods proposed in this study can be used to harness the power of transcriptomes to identify appropriate reference genes for qPCR in any study organism, including emerging and nonmodel systems. We find that RNA-seq accurately estimates gene expression means in comparison to qPCR, and that expression means are robust to moderate environmental and genetic variation. However, measures of expression variability were only in agreement with qPCR for samples obtained from a shared environment. This result, along with transcriptome-wide comparisons, suggests that environmental changes have greater impacts on expression variability than on expression means. We discuss how this issue can be addressed through experimental design, and suggest that the ever-expanding pool of published transcriptomes represents a rich and low-cost resource for developing better reference genes for qPCR.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/04/01" />
<meta name="citation_mjid" content="ggg;7/4/1085" />
<meta name="citation_id" content="7/4/1085" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/4/1085" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/4/1085.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/4/1085.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/4/1085.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038075" />
<meta name="citation_pmid" content="28258113" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1085" />
<meta name="citation_lastpage" content="1095" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kimmy A. Stanton" />
<meta name="citation_author_institution" content="Department of Biology, Whitman College, Walla Walla, Washington 99362" />
<meta name="citation_author" content="Patrick P. Edger" />
<meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Joshua R. Puzey" />
<meta name="citation_author_institution" content="College of William and Mary, Williamsburg, Virginia 23185" />
<meta name="citation_author" content="Taliesin Kinser" />
<meta name="citation_author_institution" content="College of William and Mary, Williamsburg, Virginia 23185" />
<meta name="citation_author" content="Philip Cheng" />
<meta name="citation_author_institution" content="Department of Biology, Whitman College, Walla Walla, Washington 99362" />
<meta name="citation_author" content="Daniel M. Vernon" />
<meta name="citation_author_institution" content="Department of Biology, Whitman College, Walla Walla, Washington 99362" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4182-3105" />
<meta name="citation_author" content="Nancy R. Forsthoefel" />
<meta name="citation_author_institution" content="Department of Biology, Whitman College, Walla Walla, Washington 99362" />
<meta name="citation_author" content="Arielle M. Cooley" />
<meta name="citation_author_institution" content="Department of Biology, Whitman College, Walla Walla, Washington 99362" />
<meta name="citation_author_email" content="cooleya@whitman.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3730-0288" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=C. L. Andersen;citation_author=J. L. Jensen;citation_author=T. F. Orntoft;citation_title=Normalization of Real-Time Quantitative Reverse Transcription-PCR Data: A Model-Based Variance Estimation Approach to Identify Genes Suited for Normalization, Applied to Bladder and Colon Cancer Data Sets;citation_pages=5245-5250;citation_volume=64;citation_year=2004;citation_issue=15;citation_pmid=15289330;citation_doi=10.1158/0008-5472.CAN-04-0496" />
<meta name="citation_reference" content="citation_journal_title=Phytoneuron;citation_author=W. Barker;citation_author=G. Nesom;citation_author=P. M. Beardsley;citation_author=N. S. Fraga;citation_title=A taxonomic conspectus of Phrymaceae: a narrowed circumscription for Mimulus, new and resurrected genera, and new names and combinations.;citation_pages=1-60;citation_volume=39;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=P. M. Beardsley;citation_author=R. G. Olmstead;citation_title=Redefining Phrymaceae: the placement of Mimulus, tribe Mimuleae, and Phryma;citation_pages=1093-1102;citation_volume=89;citation_year=2002;citation_issue=7;citation_pmid=21665709;citation_doi=10.3732/ajb.89.7.1093" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. M. Brunner;citation_author=I. A. Yakovlev;citation_author=S. H. Strauss;citation_title=Validating internal controls for quantitative plant gene expression studies.;citation_pages=14-14;citation_volume=4;citation_year=2004;citation_pmid=15317655;citation_doi=10.1186/1471-2229-4-14" />
<meta name="citation_reference" content="citation_journal_title=Clin. Chem.;citation_author=S. A. Bustin;citation_author=V. Benes;citation_author=J. A. Garson;citation_author=J. Hellemans;citation_author=J. Huggett;citation_title=The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments.;citation_volume=55;citation_year=2009;citation_issue=4;citation_pmid=19246619;citation_doi=10.1373/clinchem.2008.112797" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. M. Chang;citation_author=S. Q. Shi;citation_author=J. F. Liu;citation_author=T. L. Cheng;citation_author=L. Xue;citation_title=Selection of reference genes for quantitative gene expression studies in Platycladus orientalis (Cupressaceae) Using real-time PCR.;citation_pages=e33278-e33278;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22479379" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. M. Cooley;citation_author=G. Carvallo;citation_author=J. H. Willis;citation_title=Is floral diversification associated with pollinator divergence? Flower shape, flower colour and pollinator preference in Chilean Mimulus.;citation_volume=101;citation_year=2008;citation_issue=5;citation_pmid=18272528;citation_doi=10.1093/aob/mcn014" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=T. Czechowski;citation_author=M. Stitt;citation_author=T. Altmann;citation_author=M. K. Udvardi;citation_author=W. R. Scheible;citation_title=Genome-Wide Identification and Testing of Superior Reference Genes for Transcript Normalization in Arabidopsis;citation_pages=5-17;citation_volume=139;citation_year=2005;citation_issue=1;citation_pmid=16166256;citation_doi=10.1104/pp.105.063743" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=L. A. de Oliveira;citation_author=M. C. Breton;citation_author=F. M. Bastolla;citation_author=S. D. Camargo;citation_author=R. Margis;citation_title=Reference genes for the normalization of gene expression in Eucalyptus species.;citation_volume=53;citation_year=2012;citation_issue=2;citation_pmid=22197885;citation_doi=10.1093/pcp/pcr187" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=K. Dheda;citation_author=J. F. Huggett;citation_author=S. A. Bustin;citation_author=M. A. Johnson;citation_author=G. Rook;citation_title=Validation of housekeeping genes for normalizing RNA expression in real-time PCR.;citation_pages=112-112-4, 116, 119;citation_volume=37;citation_year=2004;citation_issue=1;citation_pmid=15283208" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=K. Dheda;citation_author=J. F. Huggett;citation_author=J. S. Chang;citation_author=L. U. Kim;citation_author=S. A. Bustin;citation_title=The implications of using an inappropriate reference gene for real-time reverse transcription PCR data normalization.;citation_pages=141-143;citation_volume=344;citation_year=2005;citation_issue=1;citation_pmid=16054107;citation_doi=10.1016/j.ab.2005.05.022" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=L. Gutierrez;citation_author=M. Mauriat;citation_author=S. Guenin;citation_author=J. Pelloux;citation_author=J. F. Lefebvre;citation_title=The lack of a systematic validation of reference genes: a serious pitfall undervalued in reverse transcription-polymerase chain reaction (RT-PCR) analysis in plants.;citation_pages=609-618;citation_volume=6;citation_year=2008;citation_issue=6;citation_pmid=18433420;citation_doi=10.1111/j.1467-7652.2008.00346.x" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=B. J. Haas;citation_author=M. C. Zody;citation_title=Advancing RNA-Seq analysis.;citation_pages=421-423;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20458303;citation_doi=10.1038/nbt0510-421" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Hellemans;citation_author=G. Mortier;citation_author=A. De Paepe;citation_author=F. Speleman;citation_author=J. Vandesompele;citation_title=qBase relative quantification framework and software for management and automated analysis of real-time quantitative PCR data.;citation_pages=R19-R19;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17291332;citation_doi=10.1186/gb-2007-8-2-r19" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=U. Hellsten;citation_author=K. M. Wright;citation_author=J. Jenkins;citation_author=S. Q. Shu;citation_author=Y. W. Yuan;citation_title=Fine-scale variation in meiotic recombination in Mimulus inferred from population shotgun sequencing.;citation_volume=110;citation_year=2013;citation_issue=48;citation_pmid=24225854;citation_doi=10.1073/pnas.1319032110" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. Libault;citation_author=S. Thibivilliers;citation_author=D. D. Bilgin;citation_author=O. Radwan;citation_author=M. Benitez;citation_title=Identification of four soybean reference genes for gene expression normalization.;citation_pages=44-54;citation_volume=1;citation_year=2008;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. C. Marioni;citation_author=C. E. Mason;citation_author=S. M. Mane;citation_author=M. Stephens;citation_author=Y. Gilad;citation_title=RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays.;citation_volume=18;citation_year=2008;citation_issue=9;citation_pmid=18550803;citation_doi=10.1101/gr.079558.108" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Madrono;citation_author=B. B. Mukherjee;citation_author=R. K. Vickery;citation_title=Chromosome counts in the section Simiolus of the genus Mimulus (Scrophulariaceae). V. The chromosomal homologies of M. guttatus and its allied species and varieties.;citation_pages=141-172;citation_volume=16;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=U. Nagalakshmi;citation_author=Z. Wang;citation_author=K. Waern;citation_author=C. Shou;citation_author=D. Raha;citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing.;citation_volume=320;citation_year=2008;citation_issue=5881;citation_pmid=18451266;citation_doi=10.1126/science.1158441" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Nookaew;citation_author=M. Papini;citation_author=N. Pornputtapong;citation_author=G. Scalcinati;citation_author=L. Fagerberg;citation_title=A comprehensive comparison of RNA-Seq-based transcriptome analysis from reads to differential gene expression and cross-comparison with microarrays: a case study in Saccharomyces cerevisiae.;citation_volume=40;citation_year=2012;citation_issue=20;citation_pmid=22965124;citation_doi=10.1093/nar/gks804" />
<meta name="citation_reference" content="citation_journal_title=Biotechnology letters;citation_journal_abbrev=Biotechnol Lett;citation_author=M. W. Pfaffl;citation_author=A. Tichopad;citation_author=C. Prgomet;citation_author=T. P. Neuvians;citation_title=Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: BestKeeper--Excel-based tool using pair-wise correlations.;citation_pages=509-515;citation_volume=26;citation_year=2004;citation_issue=6;citation_pmid=15127793;citation_doi=10.1023/B:BILE.0000019559.84305.47" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=G. Robertson;citation_author=J. Schein;citation_author=R. Chiu;citation_author=R. Corbett;citation_author=M. Field;citation_title=De novo assembly and analysis of RNA-seq data.;citation_pages=909-912;citation_volume=7;citation_year=2010;citation_issue=11;citation_pmid=20935650;citation_doi=10.1038/nmeth.1517" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_issue=1;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. G. Scoville;citation_author=L. L. Barnett;citation_author=S. Bodbyl-Roels;citation_author=J. K. Kelly;citation_author=L. C. Hileman;citation_title=Differential regulation of a MYB transcription factor is correlated with transgenerational epigenetic inheritance of trichome density in Mimulus guttatus.;citation_pages=251-263;citation_volume=191;citation_year=2011;citation_issue=1;citation_pmid=21352232;citation_doi=10.1111/j.1469-8137.2011.03656.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_pages=3210-3212;citation_volume=19;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. M. Sobel;citation_author=M. A. Streisfeld;citation_title=Flower color as a model system for studies of plant evo-devo.;citation_pages=1-17;citation_volume=4;citation_year=2013;citation_issue=321;citation_pmid=23346092;citation_doi=10.3389/fpls.2013.00366" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=T. Suzuki;citation_author=P. J. Higgins;citation_author=D. R. Crawford;citation_title=Control selection for RNA quantitation.;citation_pages=332-337;citation_volume=29;citation_year=2000;citation_issue=2;citation_pmid=10948434" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biotechnology;citation_journal_abbrev=Journal of Biotechnology;citation_author=O. Thellin;citation_author=W. Zorzi;citation_author=B. Lakaye;citation_author=B. De Borman;citation_author=B. Coumans;citation_title=Housekeeping genes as internal standards: use and limits.;citation_pages=291-295;citation_volume=75;citation_year=1999;citation_issue=2–3;citation_pmid=10617337;citation_doi=10.1016/S0168-1656(99)00163-7" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. D. Twyford;citation_author=M. A. Streisfeld;citation_author=D. B. Lowry;citation_author=J. Friedman;citation_title=Genomic studies on the nature of species: adaptation and speciation in Mimulus.;citation_pages=2601-2609;citation_volume=24;citation_year=2015;citation_issue=11;citation_doi=10.1111/mec.13190" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. Vallejo‐Marín;citation_author=R. J. Buggs;citation_author=A. M. Cooley;citation_author=J. R. Puzey;citation_title=Speciation by genome duplication: repeated origins and genomic composition of the recently formed allopolyploid species Mimulus peregrinus.;citation_pages=1487-1500;citation_volume=69;citation_year=2015;citation_pmid=25929999;citation_doi=10.1111/evo.12678" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Vandesompele;citation_author=K. De Preter;citation_author=F. Pattyn;citation_author=B. Poppe;citation_author=N. Van Roy;citation_title=Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes.;citation_pages=RESEARCH0034-RESEARCH0034;citation_volume=3;citation_year=2002;citation_issue=7;citation_pmid=12184808;citation_doi=10.1186/gb-2002-3-7-research0034" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. H. Willis;citation_title=The role of genes of large effect on inbreeding depression in Mimulus guttatus.;citation_pages=1678-1691;citation_volume=53;citation_year=1999;citation_issue=6;citation_doi=10.2307/2640431" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=C. A. Wu;citation_author=D. B. Lowry;citation_author=A. M. Cooley;citation_author=K. M. Wright;citation_author=Y. W. Lee;citation_title=Mimulus is an emerging model system for the integration of ecological and genomic studies.;citation_pages=220-230;citation_volume=100;citation_year=2008;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=H. L. Yang;citation_author=J. Liu;citation_author=S. M. Huang;citation_author=T. T. Guo;citation_author=L. B. Deng;citation_title=Selection and evaluation of novel reference genes for quantitative reverse transcription PCR (qRT-PCR) based on genome and transcriptome data in Brassica napus L.;citation_pages=113-122;citation_volume=538;citation_year=2014;citation_issue=1;citation_pmid=24406618;citation_doi=10.1016/j.gene.2013.12.057" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. W. Yuan;citation_author=J. M. Sagawa;citation_author=R. C. Young;citation_author=B. J. Christensen;citation_author=H. D. Bradshaw;citation_title=Genetic dissection of a major anthocyanin QTL contributing to pollinator-mediated reproductive isolation between sister species of Mimulus.;citation_volume=194;citation_year=2013;citation_pmid=23335333;citation_doi=10.1534/genetics.112.146852" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=H. Zhuang;citation_author=Y. Fu;citation_author=W. He;citation_author=L. Wang;citation_author=Y. Wei;citation_title=Selection of appropriate reference genes for quantitative real-time PCR in Oxytropis ochrocephala Bunge using transcriptome datasets under abiotic stress treatments.;citation_pages=475;citation_volume=6;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in Mimulus" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="twitter:description" content="While quantitative PCR (qPCR) is widely recognized as being among the most accurate methods for quantifying gene expression, it is highly dependent on the use of reliable, stably expressed reference genes. With the increased availability of high-throughput methods for measuring gene expression, whole-transcriptome approaches may be increasingly utilized for reference gene selection and validation. In this study, RNA-seq was used to identify a set of novel qPCR reference genes and evaluate a panel of traditional “housekeeping” reference genes in two species of the evolutionary model plant genus Mimulus . More broadly, the methods proposed in this study can be used to harness the power of transcriptomes to identify appropriate reference genes for qPCR in any study organism, including emerging and nonmodel systems. We find that RNA-seq accurately estimates gene expression means in comparison to qPCR, and that expression means are robust to moderate environmental and genetic variation. However, measures of expression variability were only in agreement with qPCR for samples obtained from a shared environment. This result, along with transcriptome-wide comparisons, suggests that environmental changes have greater impacts on expression variability than on expression means. We discuss how this issue can be addressed through experimental design, and suggest that the ever-expanding pool of published transcriptomes represents a rich and low-cost resource for developing better reference genes for qPCR." />
<meta name="og-title" property="og:title" content="A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in Mimulus" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/4/1085" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="While quantitative PCR (qPCR) is widely recognized as being among the most accurate methods for quantifying gene expression, it is highly dependent on the use of reliable, stably expressed reference genes. With the increased availability of high-throughput methods for measuring gene expression, whole-transcriptome approaches may be increasingly utilized for reference gene selection and validation. In this study, RNA-seq was used to identify a set of novel qPCR reference genes and evaluate a panel of traditional “housekeeping” reference genes in two species of the evolutionary model plant genus Mimulus . More broadly, the methods proposed in this study can be used to harness the power of transcriptomes to identify appropriate reference genes for qPCR in any study organism, including emerging and nonmodel systems. We find that RNA-seq accurately estimates gene expression means in comparison to qPCR, and that expression means are robust to moderate environmental and genetic variation. However, measures of expression variability were only in agreement with qPCR for samples obtained from a shared environment. This result, along with transcriptome-wide comparisons, suggests that environmental changes have greater impacts on expression variability than on expression means. We discuss how this issue can be addressed through experimental design, and suggest that the ever-expanding pool of published transcriptomes represents a rich and low-cost resource for developing better reference genes for qPCR." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
    <title>A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in Mimulus | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CCzMRlBQxgfUb-wde_jLdqHiPSiPrHvpUjQkKOlhOds","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483842","apath":"\/ggg\/7\/4\/1085.atom","pisa":"ggg;7\/4\/1085","ac":{"\/ggg\/7\/4\/1085.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/4\/1085.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1085"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1085"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/4\/1085.figures-only":true,"\/content\/7\/4\/1085":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002205k\\\\\\\/E4FcP74AXoo\\\\\\\/JX0k1GFjG2rhVzvqFfr+kroWAOVc6oZlp2pwW0sstPCO8DPTyNL+CGJbT1B7w5fyR0XKWuY4K+znXjbyR\\\\\\\/f4w0cIUwKi0G\\\\\\\/zfJh4DsSMSOFSjB0SboRFOU4x0E2sD06pA+KcL9erJGHsL5xoAhNvKvjroTLt7wOna2xxJi9Thm\\\\\\\/PvtNT3QjQGQA5ILV0tofGWh5XwpAILjGvaqtYHbOJQJwGv8VIczdnffYA+aWpQEIvBYJZRqrczHAMP4svgF1AnL74a5hIACsJHFfEmQzG8JWv1JO1Rt+KdeDFTvomP0TcqscE5R\\\\\\\/BuhRTYIbwI8\\\\\\\/7E3YqSAL5KLs4tCCEZSRX5910z2jCGBhu7VR7QVJl+2swxQVDyNWPiCYtag4h0lIsUtEjQXQ45M\\\\\\\/KgaGd7sCCvc92kfBV8U3p\\\\\\\/oq1+ltxIYLz1c+0YVmdvmftfwI7HrGxGzQGqXCT+l+l9kJWJTnxtcAs60xP\\\\\\\/QV1EBcNQ\\\\\\\/Eg3nKjtA4LRrrfYKl9005SnPigWfi8ONiYyhL915hEBIkI4rfhtHSxd\\\\\\\/uW+4pmJaHp\\\\\\\/qTxT+J4xhcRW27rNz0WHEG04jBO2sRj6qHB\\\\\\\/dpcJymbMEjKwGDwEphC7yz3OV+a1J5PCwyq5jI1Qhkel5LcCMHZOdhFVaRR71e\\\\\\\/hQqoE4dDYq7DE\\\\\\\/rtJIEqGmaX7zYMAHceoT+QrEVNQ\\\\\\\/8Cb9WM2NJ6lYZmGfxpidc+sGXVWj+3OrYSVCjBVY43fu10PkJRRF8z55txb0xsoj2gs+XoIXIqxuKJgbyDgyPaEBs8TngKoswDcqxbKMmTrLDi0ZHqZr1tSEinrVe8LmJf8Bj0TcJavYs5h0oGH4XGWe73w4ORZQ24pz5uFqqJlGJNgLmM3ngc17nc4ndBZJaGDBvWqid\\\\\\\/xHZJEgRLr7DjzQOZRdk6AtTj4e8CSKvFITp\\\\\\\/0hEm6dIZpHL\\\\\\\/Cmsu\\\\\\\/e5QfogAwmo3xG6X1RWFaUXfRPSmvJAil0pxuYoaPO5nf1oHJ7cJXURXpuOduiKaOUL4kpsbdnWFDfQBjb0ku9\\\\\\\/49eH+tx3ednsHJGO+PXsirUonN++\\\\\\\/f7QEBEFLuUHjPnELoyYD0IXvUG2UFvx4UlErThxEY3tk1D2GNudpKhturFUc6iONl+ZswOsmJUyGcIMhUJlw93CTXiQhQy57e0iDyUUMTA\\\\\\\/5uvC1MA0AuluKeqvyopCZjxaqSZdfh40pXtsYC76yu7OYH9OBOejEvWG19MMJM8r0Fq7VSJEc9yA3JKxnHaq3MYwGqsgN\\\u0022,\\\u0022iv\\\u0022:\\\u0022qdeGBRA56B2j7HinsL2q6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223d2ca5ab010e5eb2942a63ef53dcedcc9dd037eb14d98a840aa092d3a23354ee\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bwwOf0DEHicvQJ+VI2CbQJtxFAU3FdsJJlqFV2kfgVh2FFCy4JeNVfnbkaYZ0Oe9bp2rnCnxXVG2m5EpyVfyNj7e8SodPs9JnYckF9gry5DFbDjZrjgZJJMrQk2pVEm6JW\\\\\\\/C0uiYMr2R2FTYIxpb982Kka3uyT7S8etXmydZ8LAuT27zvux1FaF9mbnc+wh6L7BuS8K85kUnl50kh3O\\\\\\\/WZZFwpNLIyiWsepxR37lMe6hPK129LuqtemRjB6WT2K9VmOGintU1kyikHmHBTPxMdZOtR2bhBUh3zxJszeC2xsGw9r7GgF+7Q9CfEhgrc8li34fC53BUDUBMqUjA1FudFnkq28+KZmHYIzO5LKzdrRYHX61GuWtaoC8T3x4z3yKjUgIz11mFjZVME9x477hYLMrL5S821jbMDAhN48UU\\\\\\\/cjgqiuBnj\\\\\\\/UuAuK2aC\\\\\\\/AwYJyYnTa4JkWmpg5rk+PMHLfPNZ23mNNqrdMZeR+aubWst9jr8pbA4Okz9MPFxatsLsrwcyNH7skl7se1G6aMmqhmWxDY0dOFyCRUDPLi1FUFXxoRWWmzoyayyjqoYNNbdAeg0V6CLxTE578Qm68zt9K1TY2HTeH2n22jt9vZ2q\\\\\\\/BFpcGAiZJNvXPa7Qul9e4fMURPj+HF\\\\\\\/d0EufLpRnoE9mh2qYtg0WEgLqsBLVfzHHsHJ\\\\\\\/TR3BY0MIgKiS59rq2Yj9qSy\\\\\\\/QlDJN4J414+v+Ck\\\\\\\/COgrHzFuAZ8CY8+cFP3+Krf9K1P2i8tdiiFWF6HHf4ijZnEpDKg+aRzk1dd3JUjRmNcNruxQb49z7Xta\\\\\\\/wsOpb7sixG\\\\\\\/MZaTnzJrvVcNcMaRLaU4vDZ7lZVGHoCMALm3ClgoCzAhJ6NfZl1APNJ7zprsxXTTvG3cMyT5IFF\\\\\\\/tXciWejt50tpIlMU9tAVIQzM4h7uHZHgQbwYUVgE\\\\\\\/KZl8dnK3sXYPuYeTGHskIjAM3TdPPVWSp2Xgnu0H+agx0GRFhdvG5QjWPypb\\\\\\\/Kp65uEFkLxnU6kwXY6ghtErMLDkPI+L4T3d0UbcxaWPZTo5uz20Rfyar4y0fcOjYW+qyHsdHcJZtHb9jeS88masna70Q31U4ohZgX7j0eSD8T8RSjzyiuDB9jDt82czno4Ug7lP2ezTQXcvKbryAeAaN0VX0W6HKwpwz0C+PYLnekBtkkArLEcpjTlyELpZ8R0mvC2XOpRUxBN+4V3kCy+vaoWcjtgChP3\\\\\\\/c+xQeLij\\\\\\\/i8fqQwSeFBX1GMBleQ1iNNMMDXo5wFIljD3H3LzFlbv7VstBAt4+H0aazMA\\\\\\\/gZDhcMfQczfG4bxvIPJtr\\\\\\\/eupN\\\\\\\/bKdj5MgiGt32eQEq6ROrkdTewXwh1hb2TEhQLFRig6EJS5Uc+KTSYu4pa2s+ubW3kYfzzGvGX23vTvqCs\\\\\\\/iC4g30+Ws5hFwQWR8w+XuVSzYIWaVLUBG8OAes7St3vrRdNrt7i3XD0OIBulmFYQlbdxB3V9HVhd8HOrazX\\\\\\\/UHCN34meGTjF7WxavGkbz2orhfurrLiRXxHxG0ueYFvt1L4JH5jmWc81CBfoKfcLUDZlfrAqJVdvKTvMz6Js0WEP3fP7zwQ1EkaM6mIVQuk+aUZJdtfEsPPF2ZZp5IUfWOzE9oICd+vMo4oJ9idIQT1oF2l\\\\\\\/8T3A6DMBpuWpHYMFbAazljw0CKnXMumvDB793I+9SFZ8Y9+en3aewtthUHHE\\\\\\\/qemf+10V7uIyM1V17w5NLZDbhX1Ao\\\\\\\/iOMy20Cqa1NgVa4c5Grmp7Vy0N41ugRgEbcDbOmnkzFnT855nBFOrfUqWowMsNSOP2sJMlwXCDeE0TqW7XYXbWYaCDtg+92KWBlm+g6CTA1xPXsym29viVqDUpx6TFHg4fruRd7lSRJvfOsqI3PiQFrTgSrlgHHtGJsIQeka0eZZRxnaqoFCFN7X6KoMrIzpVDxueouPXW3iYTbtJUaX1xs3MJhqNKDcQX1KrLkQQN0llc6Ufwg87NpDBZuYvhwjC+4+N\\\\\\\/M+hOUXSLNzJZ2NwsoLtxDAsbxmK8sfIyfIbavhNrWzin5GaHkeYcfBZAynpmJOw3OSErQav2Os4W2oEoW\\\\\\\/agUVh0e\\\\\\\/E3zcHjYwwZJGe2CWI12SfOk6BJFRY21klPKV3NXok1WYmWZZFei44CmW28STiLaGrGeCzDavdFGwA0cweL7j\\\\\\\/S\\\\\\\/TZ1QFSPnGF3Yf7cUVge8timzIcj7n5plAU538sl9jNdpN1Vmwc9JB2RsrNRrF\\\\\\\/azqZXzJM\\\\\\\/yltts0qxjDj3cc54wX9DAszHoBB\\\\\\\/4NZiOEskQgyIDk67f34R5ZnFKzymSqun8A1mLjgQyIGsfFVKFdb1PbdL3RFAHeu0Xv1tF7nGv\\\\\\\/BjwHObDcNEsO0xKX\\\\\\\/Gijd0j6+hV3BJnFPUjHg39SQk\\\\\\\/LS8PD7kN8pvft073bEqwh5Pz05NSkdRKqMKlRaF\\\\\\\/6Hp9qzL+Tcc7UyPe651ovV2SHkX9N65b\\\\\\\/GuIAkmd0pSSLEuRZUtERxK6XyGdA1CE3EIbG9CdDSQVc4aSXxT6VvAeRVmpageaHNAVMkOvC\\\\\\\/NakeSQ+\\\\\\\/UEk9\\\\\\\/CGAJK3+OqqZawH\\\\\\\/5zkNyf8P5TyIfsqbH9oF4N9gOR5kqNz5ny3Jc5BAHwEQBG8HV21rAcke1QvYn700glYAvLwcwHuLziNrSmsdtzSOgBOEa3glzosEl6GvMNb1MvDuFkFZmQRMkR3j8NKLGb0BEIU7WIyUfLwZgm3hTvDcahmk9hzrrjrA4Y1Vft65+sXlMznykDGA2uITNwDUNmYFJ748XhCueFCtObL8\\\\\\\/Fz7lnEVHOC+FP4bwePJ+yJC9jxX\\\\\\\/Z4Ke+0fx+jNprAgRTqVKC8SzBXeu8hjcFSCfas6BWx\\\\\\\/cBbnpXweZclbFTg4Ql3BJSyKSCN3eUxyg\\\\\\\/SVRG0LNZPv+nSp9RAFJHKolY63Jsn8yOvhjYmoJ5somwAFZVIPHcdb2byzQ9BGFM1p1yBAJ\\\\\\\/E+TLE7MFWd4j698qDeOLC28MtDZUrJfEGqxaSoIf3uwapQQlDwgfdwr5Kbr7kLscEUrEdvZ1wY4as1dzsJpYhcP46znOPOErxbRASJ1UvL8xeKXjpve\\\\\\\/VIUzf2Xq\\\\\\\/kF2jv7o5fvvmrGi0DK2EmcQmFWqxJQ477PuyitpJMMUAe4f9YcpVmwQjzT\\\\\\\/5Iu5sh3bIHmo0KCSq6sKsXG15xt6D9KZTnhTi4dVzIBCgeqGrTjRgehTFI+qoUN0HBbe7WslKMpM47dLKaEZgMt1vKeD7cmIsB\\\\\\\/8e+qfzZLwY+qTRNPpxJZSeNCq+uw5P0CzBWIQF+lgf233dMJwE+LPvlLg94EqfniJqORniUbcg5XyhrY1TzAkiMcgm7eFuUuk7MWX+YjIQ9JiAVHDj7si0WGjZUv9b6zQDCFyuxcw3pdvTgeCVcmkbXcwReYCsi+t00Oj5\\\\\\\/0uuOALuDIJNQIEFt3uOicrK46UBl0urqO2q5ZzCAUFIsfNGtqZqnAcfyDQvJwhRHUslPusCwR7fl+juSAzPUl2Dv0YFLmBfPjprOfvsWLOJ2w2w0WzW5x\\\\\\\/MbQsDCi347NPK2W1+870j15JA+nmDVsmSa28tBRDu8QfAvKIldFv+D6Vrf38Lghaqh\\\\\\\/UdqQIHhkMeWtfq+wBxEj6rfkrOK52rIaNgfiGwLxB34Nho+0uopZfbHZymr0fHoF+wO+F6h0VfhsfzO47TyrhXT142vNZpVlcP8D5dZXVcMGDHQVvBMxhT1STEpu4ieGOYdXslGDF94RWeD1AnffFoiJ\\\\\\\/bsZFl83QHLPnm+sajfBEIeq2zxbSX8G+xgOMrOsSjige7MDnT+yelmhLFg4mkqVaTuCzO9D7JII23B8rIW6bT22ueih+ZoCcOFf6DBveNULljp2E\\\\\\\/sAzvYfhUh1WHjsOPoMwGgNXVkjtKrK57sf\\\\\\\/yHbPySrjKfzjAaCaR5IbPAMKzSNyhlks7MY6cjim4PWlrdWYI0RFHuH3eV\\\\\\\/\\\\\\\/g4SBYM26D+Z4cnvoinARP\\\\\\\/2B4Pns4lh\\\\\\\/ZAP2q9+\\\\\\\/ppLVHkrFETj8\\\\\\\/wBXBfOaeB5WT+H6Ne8JxM8pBfHdfKBpiffd97lPsOQ+hiab9n9gM\\\\\\\/FEgfo+jeYDXVPvy9NRAmpo4j7sxTDrigYDlXYdlZaOeumjOUuTOP3XgzQzSaO1caBdYXBwo47VLI6PPlr++W0DMqv\\\\\\\/fWCreFzQfpPU9fcxK9CDTbPfPDQotIVV1u0WaFDwD1L7tWtveleWwsjSFbU40M01SaStY2ozQUu4IkqSsycDv\\\\\\\/EVYV2Sx+uKTQI+5R9zra8hzaqGmCHFfRzJV3+TDwIbPsnGW27Girwzkk\\\\\\\/tvX8aIcULAbfMnhSL\\\\\\\/rSKdBoq5Wc8oxdxjW\\\\\\\/Iwf35Fdv8Dz0R1l4CAKJuKI9V9SXb3QPek2dwPtVViCMjfXtZfxi1uG1SIFfI6BqqhK9q5Tl\\\\\\\/eDYL4zzKBU+JqRshG\\\\\\\/xgZM7ihdrhPEZIG\\\\\\\/VQOxSwG6o1AaARkbKft8GEUiRoNZM35AD+r1LN5yeWtd7mikWtXULOlMBe2rJal\\\\\\\/a7qeV0MpfyqKhB2\\\\\\\/6y03PTkGuGwfnSY\\\\\\\/gtSe6OoEmDvq6U37FSu9vvc4rUAu\\\\\\\/DQTxiz7RwT0+LYZBSKuH\\\\\\\/A+7vn8uUHLbwckXje5gSksXuougZD8PPAP1jAOxvz3tD2iFcmarIOssvaw4HmgjDMt92SBdToZA8KZfGhDWmKkFsCchnPfPcI7MC4LoEwz85Plh+R1v7tdLiyr+AB4AAD0V\\\\\\\/HxbQQPCFYSftJSmJlKnZT4QAWPF8ERR\\\\\\\/jGVs\\\\\\\/vPr7L0YJukIkcuV6QnXMjxFcKHlIsmsLmYaae8TZLAO2O4h1nqOUidjZufaWV1R4t2MNsZGhy2VNyWUnSTjH9tpYz7l4TmxwBcy4V+QLXDxQidJMpKPxGOZcvEQ6ok2mRQvZWtBoHtAxiJnasoqDfzObBosprAvsixRde6bHQVpfpQp6W7sOycUIFOBBdusC19DC3lsrZKbxbkIP1UiFSjqJES5l9VqzR0dRlfQfy9sr7pvTNeEzQUS0v5+2sDWX862bE6iJnNIYwrFXkmxtNiQ63WGJCPScTHaOcvZ9\\\\\\\/zG8rjgO\\\\\\\/YOQnPK4VEZBjVGChIDfxwyFQAEwfMPsDSUHhFlqThnAw6kt4PCuGd+UvYao1aaLgUT4uBC5Z7kFQfaVIfcj1is2MfZ61VBGJxhYkayn2X4ux\\\\\\\/04SDEsp67MVuDgnBeIbwQ85FN7tZAi0IuTEpvBzlz2CM3X+\\\\\\\/jCMybgFLo\\\\\\\/W8jhEEnulAnAHnGYMUhfysiMHhW6i6ktwXGKJGb9FR2uCf1QkQmfrSTUMPgL+P5o9tIjvLLldS6uHo8wSfMpLl+0D+g+eb+aiaz3Hfc4AT21Y5EDwjPXSivQ+HZ5MmYu\\\\\\\/R\\\\\\\/xwsLjnXHeIWPb\\\\\\\/uM7PwtNS\\\\\\\/bpHtsYla6y+itbsLDeaYNaNLicAquLYl6bo2JnLfr+0riaazaw1Hl\\\\\\\/2cOXAOgvTY3DoX8cbFm1TGHEIJm136m2a2jPJ3O5qltHrNyOLB5Mmk5ml9g0hJyhvNGssD8\\\\\\\/CAtQI1ZFESyhPKbud+DNttE2884gwSlWH95suV2wNzfIWv+t0m4vuMUrz1JugnYPNItx5gAV1lHLt5Ab2eIP2c4lpdaZ\\\\\\\/m4lSx355k\\\\\\\/q+E+rvndxwYm6bYo5L8LGT9hqkV6OvO\\\\\\\/i+pZlTrdm7uWW0ELo88HUiXF8JW3RiFHnNiq5Uw3pqfAfrap1rSbmZQsPDq77DUtFC43UfiZAQkSC96aYYHknYWsGRSX+B9dx09Vmq46AjApyYXk2whNeeaecJWDTmgXBdyKDWcFx3YgVNINlJsAHK+lUEfxlk0xR5KU7sOyXP3d+ZXYJkossG56xHY3nSATeg71YmljhUz5KvYGrISjcywSMKkJDyCYfBvApuMi4euQF\\\\\\\/sCcNKkCLkv3wTUcclig3mr3+bH1TgG3WmndK\\\\\\\/A2tf9sOWqk8JdVzYvuVtnMmZAYe3YoSEBkKC9Tj2q9H7tviPmz\\\\\\\/hlRnaNAfvUzJF5KffnWwQWY6w2yMM3y7E8DRjBQvx7reQDTRqhK5VOnAlkhVQN0sWUvCE7tpXMxCPwVZJF4bSqRVdALHypRnlzfRT+lySl3KooI5klQ\\\\\\\/ny9+k+4doVFyWIpIK93gW3SA+Q2ZTY4sgVOXxl5ROII3Y+XkmuSjeXOVdJVT4dIpTZ3SRmYjBgCUOB5jqXve5rqQ6ZCqeLbiwESpRnlBesr7Iad2DDrp5b8FmzlIYfFm6PJQx4W5eZPiNzcEgxlYcWi+eJZ8EI084pLHDq3N6eDcCUF62SKSkKcBIb+F18G3ZSPBrHvLl5mj+jaK+3AQ5KmNNN2lrvIsc4jYmJrkmQnIcRDaLHsZephaPAIlBDpXdMS3txCmQMxDFkSMLHStbQ8D\\\\\\\/ltR6nKvcxNUtCsv+dWzhAYg2kGPquNR+W4KQQNUVL4uJpGF9ryhGe61l1x6C0uLj0R5kCuiPrczwNcSVsc1jaSSKQ+q8QT26baSmddGP3y6uw4H0\\\\\\\/w1DwaNTRv1lkHTLdictln8YY3gZvRRiE7F5wL359b4JUZ8MzubTheOB7libg0PDCwt3x2JmXn3y2fBxr3pR\\\\\\\/eb+PQvvpNsSAFdyhESRu+aRvlMfCLzEZOJttxASRzQ94cmsSyOHreiUgTMmD2JS7HHwxSM+1DvqkbbUT4zIsSpOsSnHj4dOZpnQ7P5S5z4zQ+TH+raiDCtSDw04vcaAI2sQ0wS8b6NQoUYlLkjcAF9E91cEtTssHj17997bEjjZWeKqQ9lw4WUR3spPHJw\\\\\\\/tLrWjqS9QwMJzGh9iOLskvzqcSNIjh6MuAUyv1PVsVuvkmCjY+otXd2CRHEhePcFgAaXL4LHxI4pvsTJVFOadI7m4Fg+zSss5OtP42xf7vg2WL4ytnU4uGv\\\\\\\/TCcuZHnFofpaBigT\\\\\\\/REJVcPMaaALMzme1zyDJYdDQ2Yb0hJMcXDKks4V7TWRxbwczSktYE5mBYqEBmAgibOkzwW17w9pfK4UiEZ4FCMh1kHKjPo9+zL9HmyA\\\\\\\/JdgZhX2EQ1M32j2zaOBLXwfo0xuC\\\\\\\/prruSyS4xcO7tozSHKvC5Y64zGEctrvd4ilOnMIbUkoxgqeOqo1LevrSVHz0Tpj+YnFm4yC99Mk46JtTM9Lx3cdFgqHwUvBGVRG5w7HLwBSIETj7WYwypEwIBh+\\\\\\\/86NM8kOi4bILFDf26U1jzPR8JypGRhhIl5zuLh8t397CXaAwryGZfQ+S9R9AddoxVte73zbcREEM\\\\\\\/lLbk1bEii8lAfkkw3kYdJ+TEfTU1+endOUC+uFAq6QWZ0Z1PtQwViUGmmf8+Xw8kcBxD7l4ChWteLJQ9pqqmsYPTjQSM4z0fnuMYiNAq8oerWpPhsYhxhIaT4pRTHEsADN0A8IZPrUSLc+VgQc4chNMtwLq2tFxgUi0wXGQ4ttlSlWk6EWm2yaOV6xVAgnSkcjTAGsWWyvWlzbsUK+0ycQXVpPct8srhQcGeDaf+qC\\\\\\\/5ON6uHReIm+2uX\\\\\\\/P8bahDsNWwdglk4vqAZR0tU9l1N6eRdnB6e7F2aPYcgof2mKvzZBvYF4eb2n6ShVY3XgwBoQVMp5GxI60GLapN328uTS3NHHvyZBdATv5oEcUAb1kLFQMRz7cw\\\\\\\/FAdKxoIq7vuu+69O8lJtRQEV0ysU39IePth7pq5xb6ueB+uF02VqGMVonzr8ACzB7qVemjQOlRLHxeFO4WeHUf6bwS6DnOutKhWApTmCmQ\\\\\\\/\\\\\\\/YpLQz8L8yxYHwNpaGIIqVWIETaPDdpK3cEsgLMvS97ouRE5hkQiumAhM06slUpfXrGuJ47KnsCvNIggdklIwHP6G7B7mUNLeIuZ0jND3C9PK\\\\\\\/rF20mO601brj6kf\\\\\\\/ROyZLa0FQ8BIPq9m\\\\\\\/2W8fvd+qOMktJ4y6IHVIagQ2teg5tqmp+ENyPiLd9CUR1VvSZboGTRFVjgMlLItckOrh1+EtLF31eGVWO1AwD7DwoGltlNWmXtVJz65TmvzL2LCiPB9YyD+474CwC+C6zidxI0DguqC\\\\\\\/OasKvhaxlPg9Ahg7+Sx9Kn6nkXrBxkR8zzluF\\\\\\\/WYYpZvK46JM4c\\\\\\\/3SZIC0ff9aSmFTecWv+22ZCDLXnbpaE5+gEsH8inM2IqUbehLvVmHNMn1xqfhV+\\\\\\\/zBBm7Rro4lTnLZNmZq63EHr12yzL2AhzESz\\\\\\\/nvQsNncsH4zgsEyZ9FyN+SndKX+d2CengXPQE\\\\\\\/JV0VlaxJecgLkQ629bcfYid+igbTcFd2ne8vWvTX36o6tEtCaNkmbYhj+iJQLZBPDEXKUwdBzPxTDt7qYk8bmQkG\\\\\\\/cVc4z2Pyzd3g4YqckTeg2wXZEZdG2gaOnMNa4nrd4OIXa+YhwzwmbZsqNRTyirWDvTk3mhs\\\\\\\/CfixamDjyvp2u1RwQ6ZsfOVgkBOqwLbS2UWCY3aJ48IFTDhIhe6rgST4fzsgPfapVNi3EvK7E5m5SX5VZYIhApxcMb3elC6eZaUhw1XsT9gL4i2HUQif5KpufOE2ojeWNB7IyKzJ0xI\\\\\\\/OdwHXOxhNQawQxnpIOEgzyDZITuIaomI81dPwKiI+wb0HL3Lbx8MB+JcetV2xW7qFLSLDZPaEVm25Tkxcd4SImqBRSXq\\\\\\\/Y4y6hUVuMKPbw5oCibfeF23mRhyhWLrG3yO+D2oAf4da+B6JJTsgtjPDwYmko5z8o9518viK+D8NeWo4KNcPJR0NCkpVfJ63VhvrWLb36wSM7DqzwbEdHQ9lqvPo3B+mHYNK6ll7N1UFmgFCEF9gZKaBiKeNzeZPsbIVnuGEyaLCAmEXgdlrwWjLLV6UjOs5ukNxIiDB+oVGtm59RwUGS1cSR3tvkh3Rb+wE\\\\\\\/pdoaVY4fmNfBq6Cq7h9uwBqRy\\\\\\\/QkrRURUPBMB\\\\\\\/M0EaZaBUwIyjrIFlIS7SkbaOHYbipGyPbmZsDC+THJpvYsMfp2cx5AdQKZNaH4Zd6HfEse5lfLXILeb7RARXfPgIiru+sj1aeozV7uOHogrbz1Qf51\\\\\\\/kW4GxHSs4LEnI\\\\\\\/thqfMQO0SejT2oq7Q\\\\\\\/Hd395WEFGcbmwosyzeWtELOySNacaNk54m8WOMchV9ApFufeOzX0QBGOPSN3ZRYcBpbiRqCfBo4THm01pIFwFk9mx0oGv4qOb57PiLQbhnU4\\\\\\\/FVq6NuhLzS2t0UWEjDrxxuOLWYZcFA2Mkkpu2mzrPrHTK7Kez0pSJ65Pamrwh9h7b8CRiVxLh20Z3DBu7jmFVPBiwDAvO2CjxY5EtdJbQjsX9YvzvmnnyIg9pBxjxCnTD7u2MMlqiUbXpJl6WFadqXAQR8Vz2nenmXvGrnCTwKTHfCsBSBO2JMvFxF9A67WpFO8UcRwKLc2Mmzr8owStDYo4O+l8hNlsmyi8gwsxr1a2kr+O6YSbAJ1sIXCF9iNJN0nGnw2cndCFinTDFxds5OA\\\\\\\/pAQV051msB5bTN9+cidbN4uCLmtFLXkDM+4JEr5cHfj9Wkh3u4VRji5mRW2sNQNEJpzVCb4JLCadBjIeCg4PPzpmQOlASTSpUMawZDgOQ0zptY11ay\\\\\\\/dj8aJa1nKED+6dHSkCtNm0M0vsBCYmuYHhHzQByW9xR2xB5N3OWrhvBoRhRZPLlxoyc7ys\\\\\\\/mXBQQK9hjnYzFePSprEhmn7rv3imAsAu11TZKTQj044if1cGtXIzFUw+1lg\\\\\\\/gcPR5\\\\\\\/KmMnF\\\\\\\/x2ZIcCs47Ag0LsDTsYLFV\\\\\\\/YVw4Ak\\\\\\\/er4cH9IB7ZfoYHsUZwd3DxyE4PJXJjn9rN8E9yn0bfNT4f855QFISdvOb8Q4a4HSCshcJYQhLOFYDcIgu89ZoeDRhjW5SCnqTenCrTTzUSHcYk67ex1kS4qUuR2Kp1Y3mcx8Px8lWU\\\\\\\/\\\\\\\/7r2Oc3tU+zh9Uj+tLSjAI2h+ayPr74iWQYK4WlyJvWWRWDV3kTxPfBllHh1iAJkdvbXgHcfbDmt8wXPXE1jMFes62jH+X8TWpFvsOSrnQXowfE8S+UUKen1jGG+pcVLzGYktBqtD07AH01K3sOAOZ2uX6qUrgHi4zzX03kFeBAKSP7kePEbw137Kq\\\\\\\/60pD0xv\\\\\\\/VvVanlxjmlFsAf5Bv3A4Tv7Akhc7bJK5DEEU7xKQ9M+qMrCpRz19yrBrCahmh7vCds+QCji7AXMRVbepWLD1cA+DRpJxXp2065rSG\\\\\\\/uI7s7dcsiz0yPgRxV2\\\\\\\/SQZBQTCEI2kLvxWLSS5BQ4azCa1o8SB3EQLRlZPnjnAzpGsvo+SMtY6+hEySPs\\\\\\\/XfhXGDsvK2dMIP\\\\\\\/ZMjjGotpONDQC2zYZCJXxMgYS7KA908INYhbfBGZUb4ldEE1ehHTj9CHyDmb5gOxAgXxVGxuhBN997+d6qpB73phf8GrcHEg\\\\\\\/zqOBCpjDU66mj75LQ83pqVv\\\\\\\/i30k2djT+kCzRMOKpPfoIYnjlBR\\\\\\\/dupU50xeHmeKgkn8KviiKOuvlEUzeXOZeRqACHvUvwuM+agiBrPHURCffqsrAjxiTFW6h9axnsza44X3VCZU61VkDiJwLC7YLZmqpaKFKd615d\\\\\\\/zGii9ngAZgsFovlP8YPitzjj+8+1B7bMrXA5sbgne1O007fROXZnPwdOK\\\\\\\/4obfuQCHgUPMBsiRnmzwQssC3uJ03dxs0QBMY\\\\\\\/AD21SWN87FS1rDdnOORaj3mlL\\\\\\\/nikoNVCfuctoFFGFSil37WpPNxyU2pGnZFk\\\\\\\/NF1+MZ1pN6VShKHMPUwr+MjM3Gb3D4l9W2++BtgZgHuE1M0Wa3NfR7KrUuJbCOc1DYJAApaqOkugiRDyrN3be+RscYEiFDCQ1E3dSSh20ojujuvIg+op2pe2nE\\\\\\\/0ve\\\\\\\/uxZU1nccdiSERhORHLmbq1gNq3+DaVrD213tT5tv3lUYUA8YKTnYueRFzLBD3YZqofjn8H1rTgdyCqy6YULJh4LJmZKasmG07gpkQvtV+ybsw6yxkBQU3Uv9FdjkoDkeP4DtL66GcmgLkhsX1lNh+4pD7seYBegnI\\\\\\\/cz5cerWD1Qj9HQwhOqGiH77OvlKSDl0aVjtBvl5i9ZvM8puzrxjt9hpYoKyZTkmms5xkjF1a9RRLIJuUp69tHrSVxGwvJXeN6koHY5i9CBIWYqoYiSgENGJyL\\\\\\\/3bJit5ZXzHreuSQEk8ylqpn5HWhG+\\\\\\\/0pNKchVJGKCeqRoJVX2b0w4OEYA5viGTnPoiYhGezdo8mE0FQkoiZ6u6TMf6cVGE6u5qiBA0Rl1voi351GP7mNnvgJfWWZ2WxH0YbswT4GMSp\\\\\\\/M9axdN1IgD0vrcUFEhvaB4vtby6fFv2rxMmTBmyCbfkFB7tLEGpjpQ561m6kzbhpu69067SdVtlQGa3pDtQvYt8MUABrzPEgvP7CRjiHC8FHMQbRGmtcEINjj2XHRQ+mfXx6BMDb7fTnPEXINCM7LbyJPs7NBNBBB\\\\\\\/LjfzBLSZXbujJcNPOL8\\\\\\\/v1hD+AJYSpOPmm4TgNOhtAQ44qBGG\\\\\\\/SZOSQ3ytTZtXheuZm98ffprU4L\\\\\\\/+vYiTLtsKNKH0c25j5qNouOgO1yXKTmaO1NnKIg18bCBKzyqHNHfxc\\\\\\\/2Kh6GtzPOoKzemdo9p9GRxpVNIddmsF\\\\\\\/ztI1Ej8H+2dbZSJ8FpBKQRqUHVK6ApNcplTV7eUxFVwURqaTKH+4gAPF\\\\\\\/HST6aF8zQmLQfetlB9Ak5lmsvxeou4XWK0rfsYy8UHBT44KGfNjIn1hk2mEJ6EYDfEjeurtrh72CYXmhLdg7OAgZ84AYJgDCSCxRYCj2cpeFu5BBlbW2Ud8k3xbXxO6fZByp+e8IFc2k+xkYYPQEZbapqwBeWgZ6EZll7qcQ27Zvw7rcjMskKrvUYMevuCqYR3OrjPSc1R\\\\\\\/3pmdPdsK+2R6FoeM22m1Sq9KvEq97jDwRhRlBaT8\\\\\\\/XOouk6w5RaL\\\\\\\/zLSe5kA8rXm3Mz3wvlZJyjCq0\\\\\\\/S08f\\\\\\\/G2kN2GUahzmDp6+vh5mhCz8mn\\\\\\\/5EtepScHrbanlDYATP4LAMEkr89LPDMkY0CJrHLGSLyJQs0vMeV8R8jbEEk+bQXpL9iE7Z9AIVgH0W1rvL+fdr2fppbvoZ\\\\\\\/ohLQuVcf738MgFTuTzQ584KDkNzQygTUNEOnOAqeZ0oNWprMin4sYB1NMRu1W\\\\\\\/wzqVh46nNsDsCj5xKvRWQ1jke6QAkapE4CSZTnugdcS+QLVPxi1pHlO6CF4XtjvCBOI\\\\\\\/UFS+imp5+FWkptDggFSvVvuBPmHftuQ\\\\\\\/JXcrvAHQzK6y6cDT+aJAjX+q9aHeK49iy1HSrX\\\\\\\/coAqpMiag6MI3j4yjale9XfX7r3q9t3czAl+0FMjKoSc+Pb6PXCAO0L4O6hO5peqOflVBJI4TSDjfwH5\\\\\\\/NxvaaCxXdTvxCLBOqZK151TgWdUtq+LrzvljOFvr\\\\\\\/\\\\\\\/txfvGPHwsf5Z2W\\\\\\\/pBqaSKWxtaB9+lki0mRvmoyWLcZCQ1iINtOWL9yr3T6pySux8qJhcEHrTJ4vZxNiI46QfG5TPJyup351Xl3EHpA0rLGI8a0pYTd0ctXSnY+rxJm8irkk7j1z41uUoRsTbWoNuPbItKQktMnk14YPoJv8JfC2L7G12Fnw2KDPYgMwvburjbEQuROaanxwI6HCf6k4oFrpHmH6nlHNGoyBLsIdfyNu+ym1YQrFExUVGdczQn8RFKaNQO3VLrWlYhKOJMvB1vmSN6+ZACe0MUwXHLvK22TCZr0qRDcQoEvO6Y\\\\\\\/UyGr\\\\\\\/CcAaFJGcQW1fDlBO2o4elkJ1\\\\\\\/pQA+PeK+WzoDUB64Znt2LmIXVRZTNwXtrhAtiaH38DLUmpjJIHvbb9j1O8V60WMhra4SZwLzC\\\\\\\/c3e1VMKWVyRaM7GyhD+RfeqQnhwkk2ZbcgXV5eJDKnsXqsbQXp38ezxXccTjKRNKjLvYaMXq9PcClf0jqTcw\\\\\\\/js3mcQXWXC+ufuG7VcQb\\\\\\\/ld6OQTqdOMldoJZGKte6RpjjwQGctcVz9C4JOkywsKLeQZJzU\\\\\\\/IDXjrRE2XYU2199mEVjbEu7zTCDCXeNm8Umk+uhYH36C\\\\\\\/J8VVMwQ7kvqhs\\\\\\\/5DqZ5ZQmUR9wgA8+7HNQJpmkEF7261oKCUb5ERjmxVp7ukjl+SZ4Q\\\\\\\/yguga+D1gYTadXp6rqLU+BJSDenVsAnEW4V4cp7N5vgYuAw5eEwhXKQE2pUnnFo3+Ke2ErE74N87XmP+rStteJZCt0ZNf7WtFB7ZTmsbA1bdbXxNvHSDZEPW3ayEausIwdpHAuQFthFNsx6DuJ0OCBKFoy1tRNLvk68ONkUeTh7IQ5UL2RKhjKf4+GUbUaReoDmXRflWz\\\\\\\/MqGvF0F5h51cq8jblLxWgiJIArXR1mbrdTELwsTAzMTc3Wwp0HCgaD0dUzsFmZdAgxk9rFLv9n9KTEfiF6IqrTEs5QITaXX+RFFPeptZ\\\\\\\/g26gvHl\\\\\\\/DimZpy+9iKabNqf8RnDCBJ77HQIxDb5SOzdlFQWqijBJRisaPxFnPFyT2wVaabbvwEa5U0UxGZTmhPgspU1Aqy\\\\\\\/JhJxz7OtGYQSSmLrT2a9KQspnRBUqoUMuRtv375I20N6CMIbUnAZ+SGY+jtUL90H5S2dYUi7w32gsFxbFnNQwRYEifhf0opFcIxueFTRnjtVKihznxKh8gtLQtF4lN6fGP64jjnU97pAj4XLgXTsjX9TWGdgR6MeciV67SY\\\\\\\/5\\\\\\\/P9z4oLHnQfjlvapdl3n\\\\\\\/aUDOIsZ\\\\\\\/Hu3yrxM2F8CBl0JLirj61TjxgzaBL7AfaG4j0h\\\\\\\/w7Kuxf28rehH6GIkjWR3GToYd5EJucYYr6b7VLz3qOvO5OObGhuY0mhexo+ACjAhpVXKh1D2Mi6oW7FTTjoJDj4Wt7rcMuMublXhzIq5pDDM9bXuDpObaBszxw\\\\\\\/fI\\\\\\\/tMNRjBNE+hBqF\\\\\\\/WwW35976oFtqf+FgHVir8sTunVY42qsc2TmJF+lXDlXK6jQwQLFhh+uJnglltY+ekM+4whxv+8yT+BRZeKZTE4aif5c+RIwmBnoHcqQy+XgJSInRBYAvfQnQk1VVq2eHdot4mBdsOrSZdg9zBYweXkJm6MMRYkq6XFq6ZqsDoI06rBlXjtgYVWYoIjea84V8b8QLmNePwc5mWfGv92Gn\\\\\\\/ulTzKGJdTKEsp1E85j86qL5FKRYYrcVaFUbR1UPkO5O+rdlQDfi3EB3PLUNlPP8j2TT\\\\\\\/QhHak9xCSQAK3C1+48wjABN8KUSOajriw117zr0Oa9ytKISrrXqwX9jOph4t\\\\\\\/VoKUDINC35ClgXsKMsrhvNtTTPMGEr5ENE0epNalLUnXrjyGClyYSOdx3bvMyuJtYBzIImesF2a795w4CMLHMThgGlG3WO17B34a2nEjOa5UIgmaBYPAm2dD1XSb6uPMFN0OW157C4GoPSlVJABmVpJO8CzaQCa7y2Q4i2vcnuV6znbqYTBlcoX73dBiC2vLBOqReaFbDgn116K5mNgi1K8pPX2ejZWGHA7ql61WG\\\\\\\/9USZsHK41NnYMNWsmnFasf9Yijt6MSIRUPkTBqw\\\\\\\/4fcJk8Tnk7vWUHjxN6pS5SWiB\\\\\\\/ND71hEv0GGAGyYP6yeTrSwHSdJ0BIUIUFWsl3mF9Pk+BjNzJuUIz+RCi\\\\\\\/gNm54Fv+7j\\\\\\\/m5AY9ahanlroqo3Ymp1kRbUEmGlTINPW2aqtg+ZPQHd4z+iPuRkEgp8TVxIKz2Lhe1hD2qlz6QOUiEYNQPmKIFM7odS8eqFhEu7J+HTZ3+DVhvTmc8yRIXIDrcCIiUmmZ5eBix7ThvujsE1L+V4pnnwJNJ2eiC9BJnN\\\\\\\/i\\\\\\\/qHts7hO6zsSIHd4YxEXV6Z\\\\\\\/ofFgOHP+8nmKzI9ySYfdPS57s9X5E466tofrOZ04hXtCxHKUTEVGH3IL4Zvv\\\\\\\/lwXFhave9Blpw\\\\\\\/vXeVcu+q6pCRBj0Bbx80nBaAzoM5E6+OkLsJ3k8nqROGlkHvQrwOchZMcecLF5i4qQKsi2kgRBeaGI5jwF\\\\\\\/ndPr796di0v8Ovcul4R+Nq2NT\\\\\\\/t9AsII7Si4zg3rjaygBxiCoQd66X7QsFRxUHhCJQdc+j5P4Pr3GoCgSFKQF0Y2HBtBs6yP0v\\\\\\\/Aqf4ucSj00odEYzhB\\\\\\\/QBUjrPCa67MlTP7sT\\\\\\\/Was0fb6Tn4qiyd1ytqBn1DdX2DPVlFeu\\\\\\\/b1bF5pcblDSpDlf48BSnjx4ShhR+63\\\\\\\/uTqP8NcV7LJTdryJ35R1gRnLqSFYWR5xPLSzcTqii0i50AKYT8+UE1iBFd989B3RfEcgNkjTI39QuJoAYbJt0zHXB3BY5zzGhmJzwNCbbGsA4pN4R7f\\\\\\\/L3ulNrJtbcmvLlh2pwaaAHotBW7yaGOnL09Yr2eqZRSvvafFERJAU3\\\\\\\/2dgAL7tdGmguG9wxPtXnwI\\\\\\\/o8A2lVBhBeiL\\\\\\\/6+VMtGh5DnyKz39z\\\\\\\/AZ+OcUt7hi9XQPmOdvEFKtiQrnSSmQ2LvVZIwHgYe+OYymho8ZYrKe7An0S9PIs8oQxjU5DZ7LboQtHt4UsXgZy7NiueiLbcH36o63AYdnVCYbllQ9ivTiqnKNiTrXl21HUve5aNralmtqzE2wtGrhwWI0Iu9LeCA2WYr6rum\\\\\\\/Y7nyAQP+kgd9bVTTu5+hY5ICirsnnW\\\\\\\/zDI2YifJ+HlQ84W5SRkMoYkeY5A5pogcW1u0MZt30VHpTG7hkbB2ZZCMyJDU9pG4ozMdYrfxyPvnbM1PgbFtpDOoDAFvoyDhN7cL9DZmxA3GjnjmE2XzB5Zgyup4MrgToXtle1Ei53Gg8F9Bie0BdblcLzsLErjGHGFHl8uhoXYAm72KPZzt5VHLAd1CA4MqkEY+yfxUioqv5FcadRyGp4h26I\\\\\\\/AJutT3phHMpBCbzPsx3KHIKy7h86bLjupgekfornNyk6Q9lkGstMVWAgPfZguPrc3UYy\\\\\\\/Jk9iaE72Z2PFUACgcKWwS3ytHWTxBdc42JhlEV7otoOw05GfSBwHTJLStv+53i+zyL\\\\\\\/yHUrWOo4OUZZikRJsrf2g23EFb8rnxe7YhWLrlj8KovXrRLoChgyxPvv1XLiAm3pSLPYLeHybiau+YOzMLjL1ZhWnLCvekuly8+VcfwugSC3q3XdDrHCigdauVbodiQU2nNd9PPqauEtUPeSc4tHMKD2XHe1crV41j\\\\\\\/rq8ZnabI7Hbwl9rBWbXlEGOBzIkDc6rc6YywQCht7BvEIKnl7YDV47mLTV++w99sCWKj20npnc2E+AgGZKK4sis1TsxG+xEpierxr7UPtA1pEAqEoLBepoctf5IHC5JYotrCTAPlhyx59UBtCddnq\\\\\\\/873y1MoxACYo29yCq\\\\\\\/1q1KK6ig4c2SCkEXjSDPRn\\\\\\\/+YgXtmHLY\\\\\\\/r0hUmpmcZc\\\\\\\/Tmr3D4bSt0GwPD0JcHbGsdswcmSSNaM+60E6K344T\\\\\\\/tsMeAJGNSX7InNu621vDSv3FcX\\\\\\\/LuLJ+VtnWVJDz5Lu8\\\\\\\/T7Rm4UWtul6qLPlw9UYAdve5D\\\\\\\/9vu7lIbQrpYChhdHcjKShAzhImyf1vdyqBcf8QyKQrsW163k4CJFXFu+t7BRf9h5SruxkxkuIVFmWhmf501r68IXdXcXbRZP97MKZ91tNmrt71887wBZRGCkZN0g6U7lO8fzYt7vgsBR3XL9JPsGQRH2lnZxjTldtZg6lW8Oa+3txa2yLD4RxCxmLMVGWBNIhdyVrQRnEF6D+VneDFGtnXv654jGfZosnv\\\\\\\/izsu6NCti2oQlx3PR27lIPi4TAFIlW+D2QCfKsbt3u4Hj5bl8nlctw\\\\\\\/YkOKT19uY5f8HaS9ehAqsRwnD8kUI0VONPPHhzTR20Shw2ZO\\\\\\\/VVCoVXEOwAzNbSKT0xltzmuBRlTu7GCQRLhxbmmShh2ckk\\\\\\\/GTXgVI7w0xbfN6Ak6H0Tew\\\\\\\/a4KT21YB\\\\\\\/OzyJQ\\\\\\\/s82aHkqUhC1M6IDuvIFVv+XklTfgHNiGLeW\\\\\\\/Rpt57BVnWKI5TOAepMu8n7JtDQca2wdEXj3L87f47OC9sXG+OdWuoGZ5mW9JBVrOcZQ9tDisOCPaCxwwkNcam+sUnGqyZJci6EGPvbbxJcdfKiwf+Ys6PlSvHISbOd334BSziswJROLeKLThEwj28enlaPaEfmMoeHIW1P9\\\\\\\/+g0vUaQZ\\\\\\\/J2RTXmFATfYJtg4pdF4G3YtYFCFadFxoLn3ORDU0ZC8E36EhkABlrgCRkwvroNN4Fz447WFAPyXArIq5ZTiAvd4oR+CFNgiAZz1cCXbMAM+7v56q6oeWc\\\\\\\/lEP3uYH8bODJKNHhLdZADyLmaAzpi+4A5TOU9JTdVck+MEA6xhF\\\\\\\/mQwN\\\\\\\/Rx4V7MXkplmmxHeNUHNLuNFgiyo+yQRZBj\\\\\\\/B+Q3f+KWbdpxPJRHrYXC3zCXMlN0h2dYTf9CHfzEwNI+jWTYZeDYsuxp2qbPcayBhmHcArD6KRML33kC1E0OlnNdmKEW0X+hVfV3SmIxJygcI5HRYRe\\\\\\\/2cg\\\\\\\/LGyDIx8ONPgmjPJFLmTzdAlwlbkfvye9GF38J9mK4MIkI9W3HCB0z3wMGqdiAJWCDe1BnAJk0\\\\\\\/jIFgsyVjqin+4iHu6ITiki1GrmHXmYFkdMUafg9o57ih+LX7UKjjFFtuQovywzFEIsVEgatCPzwWi6KQR+k2UtPpj3VAYlfvfO+lbhNsn6eDVt8zgOg6yxbVW5TVFNeALAe6aKNpR3UC39tsLXdmyTmQIwA2OWhCCJBvMjBYHE6ZYGRC6XmojWXJL4b77+yopof\\\\\\\/9upHTEA5EuIxlmEkFV83apfwRx4MeyWTrQFnpMLDvzFTW8aV0AhDWa3wWHdnv1GySIGh87ZBMbUs1IdYKiP2z871rznVaH1d3CIuLPWeCI4+BEsxVzGeyr\\\\\\\/fIMnWX9lIBY5Cd6h1HV\\\\\\\/Q0L7+EZEBO251YGj\\\\\\\/9FYhEzLBw7fPfts33uRHowY6txgndJb95790ofqoSXyUX4y5TYJCeRUMO9Av83g\\\\\\\/vgsBhHN3PDltIv0gn0Z3p5rVgQD7xfATjzEStm+8bcYbT5myODWfjjxqeEpoUduOFccyqsT9kexLYDpMZ67AX4ThvvzUBWKT5HdV2qvwiaQOkZbpTnip42G2Er7WE0DVHmhtX6HbroZscjyjzUyE41Rcs4kHXYAL+V2s0yZT5W0CFq699h7cLFILT82ij8p8piHBMRPW4pyL9dyYVTblEsTRzszP2iMnykDcKaKUsqXyQpM68NeyaxY7\\\\\\\/nBTSVkegsWwKPamI9ZhaAXJ9Pc4AqBe98MmGlOywfci5mn2xHNnZba1QSrTzFB8FfZWrAfFC4l2fH5IMuNhnq2iCQh6xUHArwdiY5HE70ZDTcG+YWM1kxSn+V8XEcpnPfG5ohcmHuDcBqRnXATkX++YZDdx17u1oCRe+lCsaoBBerdw7ydb0NMpg7TDoUklNDoRrg8QLNSpJ\\\\\\\/32A7BrGpVL1w\\\\\\\/gUSFh+63VnIUXSxEPcuDRdvR7yT0K95bOYdiDLAOWWDEvCc1NvXdab1ZafU2BVYfEGGTVvj79r4YjW1x7S\\\\\\\/Ej4C2Ig\\\\\\\/eZDry4BikTHXh8lU6N3Xko8KFuNJuTK\\\\\\\/uOpnPjMxTNhKt6j9cC4AUVRk98VUImM0zY6+c6dVVWutGGS9MRwy8fxu5ItHIQKQUG3K8qA6YzE9KX+pPNY60u0vMfJin5aQp41sSnrkFPWRoUDhcRsAfM\\\\\\\/xKnDorDHZunuV5Kw+bFgHR3rjAfhayvchjH0tSN8al9mvTv\\\\\\\/+eenEcofLk6ZTsxnlDGnr2+HH2QsYbxzEhmpEoC7lNbRLDeQwULbpUFxiU6DJzJCwJSrIsjFsveU19fCy1ldnuk7m7VSM8JOQYnK43awkVlo+ogtVIJMV\\\\\\\/XcUad+Nqcla7QCoX4genk4XEOLIvFMFjjxnUACPb6rOqixvAUEaqC9IZGVQSZuiMHiKiZxEgcXqvwOgt1xnoFjbv2S93MDd9btEhTXVZtm3pJDo+lBFEfFF1k0WjLIIZzCVK4RjNUrfzZrKPyBUVug70H8WOR53oiU3wduYI4BjwLrR7Hkb4YX2+K8xvQVMGqqeGsmhb5SHPTtPe1imbzPOLAPUYTOGhTrc+RSTsGbrd2synNlgqEKRx1\\\\\\\/xJKC+Er1MuCy0goyTnHjlMLH02GvoF4bTI3KYsoY0ARd0Ei46OCRgrvXI2YWQSR+6tJSe3Z2cj\\\\\\\/iimwS7PMVOZY4WhsdLmtprnOiYChXGfyN9sLyJWGqnzHXJYQxG0BeVHjv4t7qbfUUoTLVmqwjDKrpLBx0MvlpTTOR\\\\\\\/sYrp8S62l3Z0CRuLr6+RLN+uhz8kBc\\\\\\\/FbsO19a\\\\\\\/wYA3po8aGadrvktPGG1XTBuWJX6iGA4t\\\\\\\/G9v+aWzAPqmIfMIngYmqNK64vWNPuNbOijDyURvzKO8X7DU6AW2GaBUz3DT3dtxGzUyF8ufVuI2tDPKp7hmVoH6s3Cx31RQpjhSGMoNcx82k4Nwt9X5tTc6NnKosA8NUXmgowYsO3HVqjy0hGSXt+UgXgZ23e2NmlurtBNbUpNPn38Q4pTCerCwmixnIuyxi6rlASzzTpDCneUnjP4aIQ9\\\\\\\/BEJoqZXBO8UeF1I8KYplOeFR9VccCjF7ZYzWAK37j\\\\\\\/SO9i\\\\\\\/UnADDTrwftT8w7Gg8N33pLjcJ5QS4dA+HM3oIAJUVCnRMZr6IWJ25X5I4DaE+8m7NLjxew328gJhNNBNmeIBLpiSVZU7knYSdHh\\\\\\\/VZkyfqr42varYZ8ZZuSqD+YSnS+6FVnnlSYH9ZX+rv5atdtGlfNZgLE0ZondjFFkULOYxYsxMreGgXiVsRM8u+pEE4tp03T88b5ZR+uxJA8qZxyydWqqggSznMAS2t4ut5BZElJsWUZn3gjS8h569uMcHuzmQXkdvzQtOxH20OQBwQOw2mx6AyZ4A4PT6vLDmVRcr4NrtWCmZ2N98pM7v8quYVQVeHlUXiCVzL9bYG7OcCHd9h70WwXRGYNJGqKHMtWwbI8WgeKdXY++0CS3XYfqdxyjeTQz+wKoV8N35LAVoyredWpY9BHniCUv\\\\\\\/wuctgjZ1ucqj\\\\\\\/Lvc8IUdPtPj7ihltP4\\\\\\\/pYT04\\\\\\\/ayxBMop7kHoSm1FYSYtAD\\\\\\\/T5xZRuxPYylmFt7YwNJS8eV3lvvQA\\\\\\\/GPP9NSLRMEj7uWnK+D\\\\\\\/gd2KvB7ntnbDgjYjklm\\\\\\\/aYeVwhCSKwxHo7RMrM8MQvUrUn2yO2R1NcUFKouMG9nx8YJ789rYrxDYmP3l8suCg+\\\\\\\/RhPSa1mZlk5Ds7QBomV6WTeopw2W78kvk8MaPGLcIckDEWmx0aFY51beBKJl8oWNbM4YcV706zH\\\\\\\/w2h2a3Qt7CfQbf\\\\\\\/cSMx9utvfi9JRyVF3h\\\\\\\/wxk7vfXc9FrPRFROlh5wOtcr4FzrE6Ppbt+UR9L4F0QFSRfalXR0cRaLqGgBkLEA7wp7aBl7hzSzdaA8lE4fVdjGtJG+xXHgKt0YEk5lOkYHtMyWb87rat3BEbzSJ\\\\\\\/dzH10lNnbtkGKGh0OkPDU63Xecg4GSFIamuNRyd54Kh\\\\\\\/Ezg4UxMlxCYW9Jkla1INohV2XduFxDqjm+A3cd+CUXohve71ZFjHhCkAWzQlhTUU\\\\\\\/W0sxQ1NpiD45\\\\\\\/ZXz3TmB5Dn6OyILyzFytWiBuhrt3noAfF5SxOwvJqvlVnm3KmEieWt3FUhC6qI4KpprfqKNT9gaa\\\\\\\/l7NDmbx8XrLN9A0G9WVVec1\\\\\\\/lBoCthma6YA7FnmmE+WMZZBKu71jX1BY+aNZTTfGuAp2+pKRRUnbtqcWksUlJLipLmFtL5E10HDoyYIXs+OnVl9I5PZNk0ianeHwwcEBeVk7FwI3Eb8FydV2yc4KjVQDbyX517ZNrSWLduyKyyczMCojiLqgz5bhqgf3BxN4URv15PprPJy8nuDv2uFIN+kaMt+GqIopDhsx6\\\\\\\/kEDnUIilQzeaiNnKl6Kjd8K9HJF6XY1RVSKeYNCJkxgks00VR8bGBVs9mqV4vDHNuPY4k9OPYLuweSsABkd64mtkjNxXOipZpyKmJxMKwJ2VmEnLpfMyCBKJ0H5kpqkjtzOKaZE9y58TrugY8J431h8RRZOFobxBrijLFmlMBDnFijvGyHyL9atT8HxgUI2UgSROEoyhuzaCywv8Ujfvynh2kIp\\\\\\\/PLF9V5xfMCkc8rRw4PQBqAFuU76m3smRk+ItivxcBbhuQqZwsS2KvvRZktsnDj9hFCMzC\\\\\\\/DZQ3OcHhLsiezaGIlvkce5Ib\\\\\\\/1f9GToLA0VYlJEcoSPdsIp+6E4s8igvN4noiWe2png9u6TPPahW5n8AWwfo0KQL+6jBPR1ckaHXplLWpK4gL\\\\\\\/p\\\\\\\/XBAuZirxpiKc7KWfYT3R4Qug3DOsONPXW6pki36myIlyZWDXox6jT4iqxAoWd30bulKBodCucKBmQC2gKcaqfCvmXEmSuaHSOtUrSzAMuasB7yC113Jsg4j4j5\\\\\\\/OG3REHykpE3HOG6wm4hqobikht+s6UFk9WjtZDx\\\\\\\/uk7CMpc6mXdystlGpssxS7XqOqZWn9ETp8NqyOIYqapRuJJzkrSR74lASup3k\\\\\\\/1x4+kvbocvxFVynjEr1twsjI4yz\\\\\\\/eWB9k1JPpP5rO6gEi05OrBGnEBfJkdx0uFpE93sbTStnaa4G0EHf+bC6n\\\\\\\/wBjGVRmL7FA1fuFAdGJIAgnDuo9DDcIbptUMl9\\\\\\\/zmHG5gdniv6lGQyxYLsDFVZeZKCoBYsvbRQNaJIEUQi9LT7J4u4iaHy\\\\\\\/Tinr8QiyC9HnKwdQ2mfLmRY9SW9SiD\\\\\\\/Ol4F2wtvtc7a+Si2umNMegVQTXVxqeg2NBpNqReEAIb8+qTOT3Ovaa+KPfh4cuegLjM7xuP0yTMBrQ6FJvVL1NYMzwnJCKUnJFKdv1HVCm69pD9aUxeesSiIvWSkMByJ3jM8eJ0G9i+gHZN2xWCg40oAz\\\\\\\/QYmXm29E0l5PK6afpcoPvDMF\\\\\\\/JaBc5cO4MDXhzNXp4SfLzzopeWEZ+wvG2v9iSlQSfy8Jmc6\\\\\\\/bje1sTScW0gvvOki8FzKwyfh1ZBHbhEfEff+SUN3tvOm9u42yds0+LWbBcumR0V+IrXVCE3EUEQh0SR5BEU7FoEYcU1tA5AtNAeGfelYbv6rCsbTzEVpmoLD6RUoJronBTR9ruySr\\\\\\\/TadtuRnWoUc0O3IXYKsa3fuq6aMEWrd7rLl7WhM0Q7wgwH2BpgzSfnQbjxUJPOFYJdai5F0X5yK4EjEGW0Fyi3h+ftuyiCs3blQMMe5vw0L\\\\\\\/6o2M96RraiBjwm7j\\\\\\\/AFr\\\\\\\/OtOcH7NVjmN2KWiZEVkvx+sshfEbrkUpNYWpZe80nYRpaUsC9tBQhhRcbODNbXNJEYso4Ru5z94SEFNlkdSrZk6OXaaErQsz1VFnfzJEygXJ+HVZ+92wV4b6unosTfjSTpEiIcC2JWBxQ\\\\\\\/VDSyVZCzdk8k1X1wK1V3y3umWWsZU0NxtFYB9r5LEi8tJatDKbyM6NCMBeGuNLqH7wbsBvOj9JMQQ84AiUR8HbXgOvb2n2S\\\\\\\/GIzU0kAPH3SIDCe\\\\\\\/Aq5zSiQNKqbgfVQvZSg5A1uPbW3RVFO0GCjC40Z3rdG8LuXprqCoqcMsIsSNy+a5LqQRiqjqRBZcULthe0ufVbaKzylyxVpBhx28+etnH\\\\\\\/GoPEKC6PGn7iV8vaaMtLDC64xbXhhDos\\\\\\\/fV8eY9bfl78wBLQFNvHhzUZwwodIzkDKhY7zDXcZPxgYOJR28vrxBcmF8iZMwm5KbVC+vXbxQhc1+\\\\\\\/Gix6y0MVM9PjYi5H0gHAXHDmvcwysXJfdc+S5gjdFtWE59HRkj8QUv3jLlK6CFajvlOVjKjeRAIiAp3lKLPYYM8d8RIoiEo\\\\\\\/cvRuIU4w75pIctO05hkNVSgYRLiHK1VuIojFyz8J+I8JHQF6myCHUw7rfEK3vu52vgVBAI\\\\\\\/sd7kijRoHJ0M0UfY0aSyP+cZpmiWEVEfB+wpwa12FxdS5vtHm0GqAmz+B0j\\\\\\\/Z2e+tK4nKyzfd2kUWtw3JkZYPSEZNl0RXJnbIz7nZYCya0r4+k6Ypw4CssCtpmykgCVol1yKXBLpLD3sCarAX\\\\\\\/2dKgLvpxutFfStsad2ziEPTrVyxylOL4AvpedN6B+SJ0tHbwtSVDK6sZXzL+2b8fH3lKXrl7HdaPfToiOPn2KwKXZ46yBIevqVDjTTPaWb8micKG1C+uoNgJepyFj20BULm91qnKRwas+L987zYlVeY+vMMGpyGyRtte4oeaRCrOwtCfSd+XWZBH3bD5fUsVgP020DZPgDDjAcneMkSD5hlWtNiEEx0QXWXrx6NepHJuE9+uqiVjya11TZzGN5Wagl2Dz9cp7r+ALFO\\\\\\\/cOvcCcxcLSkNJ9B89jGWkVlD0Y\\\\\\\/dtmYTLQaCMRw0dJSHxbQlsAHHL\\\\\\\/cCp8l+YK1xxMcv0MrjSlbcNwrJRpdw2ZgPowInp3UFO\\\\\\\/Elj+VhmTXIiItc0lhulNIw2DcmJowAR95VGfXKJpBsr+5UFniR4YiercTNetMgrhW3JboNrQCQGb5X0Q3TWn7F+b2eY5wzD47QsymeCQ25v8fdJ8cn6H0qwdV3VtAcgsqOZdpXangQVRjqNl28iX7RAgR56nmSvnbu9j6lhJo4jkeTFVtAYUtvOKLAhDZH+D+fymT4Cq3TVzUjTtcfMixWT8f9tEmcyW9KIImnG8yuFfiR8JDF2CLno9SVk0f5B\\\\\\\/jTotB0V2z6yoMAvkAdz7ESfzG+\\\\\\\/HMwPpZz8EaMrBX7Vx+1cRLHwTEycgxZBF0UxwRGZPVUWyqB53LVKW5PQZYbjH0+\\\\\\\/yrR61VS103MjIYJkGvpyuIk+IrLA1SyI7ZUDwLVvOgT9g1p6yuLtmoMea7FNpxrnBS8ew31R46UQd9xm\\\\\\\/0NGwEiirDcCKHER5aSJYgTXDZgaqKo9pLbjAW3+Pp4ErxkNKRfrjkzf+WGkr5F6QTSMFLM8JhJZMzZEcEvL4ylSp+coGUOF1WK+\\\\\\\/Ll0VrP4M8w9igDIOczZwXfX8iwgBDQ8y\\\\\\\/lEjUz7PrFWZMuPTFwsizxpRlWN2FczyDKtGzz+Mv6WeTWm+pHr1MI4UJSieNLkG\\\\\\\/NJqwxKoJCO8rkKT4LiS\\\\\\\/eYZqnTHYXLh7OtrtlxqvAaQrBc+9B27DNeYiiwNfNUfGlt9eL6g10er2UtvHMLOJRqL05W5R7EMpOwj\\\\\\\/F0HDaBWHftLho0fkvjfBABPib9sRFj1wWdMqTj6EfLrtQLj9AWg6d5UQW+DZ5mYhNytqwmEUYmEwxXUbiwekBK4aVGMQ3F2mtMO144saXWCIj1H3AlGQBcxpRONNuVhJZCSBwpVQlsHuma9ZFMyR2ob13be4oiHWpszS\\\\\\\/ro0keORAPAihviCAb\\\\\\\/U6mBH0iVZLxonubgtQ6qA4QKuZrCCFArYeqwfaNrqau6mPwdZBMuogkBtG73D46ZkwokMQt\\\\\\\/1cG66KfF6qb0HOcyogULi+cdJ\\\\\\\/ivXFf9UDOy1fpqBa9Tw79IOLIbAkfmSbhRc3E+zQHL2n4EXEjcnaWxoNLcxkgFJkR8wn+vHRQmaxcq86cy2G6E0zJvU3nAmkVuhop5Z5zE\\\\\\\/WZ6vlc5vmXkZPa2kpAQT9\\\\\\\/F\\\\\\\/iR3UGouxNbGXN5IPP0jAoOCcJ0NftgINO3LyvTJTNLeU+RVuS3UG\\\\\\\/AObA4yph5psU2qHX8xCnlh3wqCDTPVf06\\\\\\\/ju8dcqi+iMyjqG1eUF9BCKlr3pijKJNsc4szzbu5OTOImkL0AXJV3f1ZTTdz+iG+HfzUSP1pmD82PC+ZMbLxxt+BIVNT6WeWLr70CXQcW4q6aTSJU0e7cd+g0PNbIBj\\\\\\\/nOjO\\\\\\\/BWaqGc9hh+bU6lhmS6ZYoTOPJP9bI\\\\\\\/sVrIUAiBcX9DqDijy+oMZOhuWzdZ1PxtNDZWCRtqeiKh9etOS3WssGJriWW0WRO7z3qzgnoPdRla5vCGrAJraw2wZIOixkZ8FIxcYPtvt\\\\\\\/1QJN8xOQ0CMH3ryR+nCVYfqvOBzWKY5MtAO+tteHZ2Yd7nqM1XUXyFzWCXcSIwxwgwkxbpi5oi4t\\\\\\\/lFeGol73I5bbWIlYp+\\\\\\\/VsJcBcTxfCdlX4Rq6GPLGJa58WCTjnt7xInguliXi+mjsNV7ctvfOBoJdoJaaltJdmwdhKr8zfc5WyOZU+m8YCvnUXi8YYQmn42SwrN1y3mGWI3mblEmqhWPBhyAjVpc2GR00hcqvfB6jZK+GFkd53nWBywDPCG\\\\\\\/L0wPq9b\\\\\\\/\\\\\\\/HGRePIaQC1jnkr9QvJsZTUPQFrKTgcaBKVGqQp2rcjoO6Iu2xwt9xEffF5KYhSiJctqpmUta5Fn2kFukH9FHiEuT4G4oM\\\\\\\/heQ+4kd\\\\\\\/mwgl2IntWjuRoXWhfEK1F1MrfvTkoQnCcB\\\\\\\/rfOsPGSfgfVeTByjecPx3FDwCjokO8rH5\\\\\\\/9EwMupRzKO854dh52n6kqkjXrNGnTsnjslc3ycXK+FpeszWE97o6cEHmJ1QglpXvz66YRWWheTlcyh55tfujLbbx7rdLpsynJfNg41S7ZqEuYNB3p4sI9UuE5hyeqetn+FxURGTq3lyNZlZcNOwcLGNpT5MK7vyjrzLzXqMeBH3yFtUVOtaO3c15Pjnj7ulwZ3VahE4cInmFgoFmVQsX7OlikMv3TDyyJYHw37IPGFDG7ICVmuYrTl+KhJBkSKsE2Kke6SBguggIyv+8eX3KyT0Yn71\\\\\\\/h+VuTdMNh1wTRtWY\\\\\\\/NmSyrpDEqpYvozJeYQ1QUp\\\\\\\/n70q2ybginmC77dj\\\\\\\/H3dsxrqqVEf36e0UjMsIG+f+ARwEzkpOxe4iDEtMEEubqlGKEF8PJZ\\\\\\\/UUU0pgLWmA5v8SbjO\\\\\\\/0U0eYW7anwaLz\\\\\\\/u3xG+TCGuOO5TeThXG0GaCbH75KbFr5HKD212wA4l0S\\\\\\\/3P\\\\\\\/cI6fgX35HO90udg4He6X1MgCRYU07OBeJxeU6H28CyZaOG5Vvd+wENAAAJkanP1CX1MOUNejY8E0Oo3ZKXNcHNLnfV21foPEjcL8G\\\\\\\/amy43HUsVMhCqBbWvC9laJw1jsTUFL6LsIMl1UkaaTJYJ8z2hnsDzBOGrv4RqudzE1Sl7RrifJ8lYpicBMtHstz0kXiWnWtj3ZnnuaoqCReK78Q4R6ayzaL413Expj\\\\\\\/WFRro05Rz3C5LlycwvYkvJaPo1c+M8u1uLwsauVihURdXOZhMs2WHY21lBj75HQEwcNz1Oweey4CNroQiNojvbXn0kUalKdmXjQNAK3XVN2ZpWw164dSFaYoN5ueR6wFFtc0jgxLzBLXkZtjCQ7lEHTc5nB4cPoNnZ9+coMjw5XLbF\\\\\\\/ZmAXaCzqRVV+BvWJYjU13ZkoX9CgDQcwkcvdDp63IsB6PBk7PvZnJ9rJ\\\\\\\/vWeIDzhAnLeP0xwX74Wch4soS4pl7iPov1N8yb7lWJOz+Pws\\\\\\\/h9FFM2knLNtF0M4vpCkrfI04teGbGLw8UZHHHJxhR7zeHNbBMU5WRD+56mxf+IS0q1NSGh7YS5yA\\\\\\\/TVJz3oe0AaK66CPNDLO\\\\\\\/2dHW9KG114S74imSOBjYKOQXAuFnaK8cGkbtI+vscaJq9FESNCXxonuijEDqexzT75guwLbHayIeJ9wpr4vRyRwE6dHWNgZFZZm5hFHl2rEPL7THnlXbElq35IkNlX2cxaj0CWi+9puC\\\\\\\/xIzHnOf7Xbk4U9TrHU0YOTEZ45o7Y+cElBIcfe+iFwM2Z\\\\\\\/yG1EkNVid1D\\\\\\\/1V1DIbk4rT3Ruw+UcqCsgZTQ\\\\\\\/VWzNczzxPCqjWDFHa4Ygimc5qh+Zqc4XiIB2AkPyKeZ6YJbZ8JXiccmNRbAdmLhLTIcYwbrH+Al9VjLB4QhIs1b6eqRBhA7ipFRVFM1e6ZOrv4sgStyZr4+yaBC2TdjBcU\\\\\\\/afTcPwIZb0ZUZUMjUpOCBd+OE0JOKgc1g4XtDUQcIUT1AFmz8XZo6sCc3flH0VvDBk2C6v24v1BslgrLJ\\\\\\\/ZpuniD9P4AO9TKDSuBgAqCDY7FKAhRFPYbyfs\\\\\\\/NwsIMH2L7QlzJ8GT9k9k21IWZTuemmgBCy2m3h4f4+SUpRdPwtl5k09Sj1ixNjrMoVbvKwyd1hb4kVbxYaRzygPCQd167aWhH8C0nTMErcoOwDoteOYbtV0+mvPU16r2mDTHuGN91pO3Uml6vdohYneZXNdPcBzEgzihDwgdy\\\\\\\/aRfMB5uOM0NK0ridzP95ZOH8fr72GmBXAzw8ui7zu20QMKHY4QycGPO16rOuVUoAbNRfaLCk079HLkvHS6mrkXpcKoziqnSg\\\\\\\/Q7efylPuq2j3ANiONn52FlxFxbRQyY6pmEyxkyNl4ICoE1o1sYWvHqyyDPUp0gX7MeoDFz9Wd\\\\\\\/GjEFceh5DLZHbyN3YliFzfJbYZdNznm8cIOpO3dK8nIfwnjv0tXnnkBy8Nye\\\\\\\/rCHyfSyBq49PrSjJGB10zLnCyTEycT8CVw+rUSTIuZsBpWEtKjvFg1J8BFPu\\\\\\\/133AOWOoNpThC9+6lOIDX31kxxCxSXHnFuzWWQSOmvTHl37w229J0Uj9f8\\\\\\\/\\\\\\\/g1q6j4Jk\\\\\\\/+0RH9RMVkeFXwwAfdKEc6R8eUEeQ9b3HCEDhHdJHxLeJStwSJy3DvoluERpIwctFY93gU6QgRjTI2tWHi2MXIN+C3oM3GEGnfS8u9l5BbRJRYYjSo7kAqFLF+\\\\\\\/66Kifs0B5Hy0oNKis9ECWBxVyw7wENO93MxEM5O4VxlGLbq4fVlq2LNgUPbsXKeEs85AUc49JXfEnufiy+R7es63SpuPzoDR0LFbGL07eGZjop1la7hqhmFfT1KUFPAgLtGzd\\\\\\\/o\\\\\\\/nDUZUKApulaMAVXTwwjUWaZu51BbzueaOZY9D+jQSQQAAyR61nIUBM0V242zZEtg4QrDoigOS5q0p9rRuBfsVU2tRWD9CbDG1MiGYmgxgvSQ77Sqhmel6\\\\\\\/iYcrViBQPxbgUn6UBrRRdMCugXbnor1LKOxuhobhKVpGUEXGlimNTXG9Uk3P8yBtmlLXQxh2yomxZC1e0VqZzj8lBj9wnVdpnNMjYDcNVJ1luFcCcW\\\\\\\/bETVnBLF1rDcFcCADe64JfeQBkwwAx1PrzYqSxFMT01\\\\\\\/DWN9tOL\\\\\\\/p\\\\\\\/1jcleDXK2ISNNqGNInDxbXi\\\\\\\/lDb5FIrEPajpW3\\\\\\\/8tZiRlDJKXmIc+sHVa6LrGVJWjaYDdcv5XxndCzMi3iXeNjfJ6zeUC06jVp20a5h62EmpvyA0oNiaA8MZqfy8hN9\\\\\\\/YKEFUr\\\\\\\/e3hjdlMgqiQVvVKQ9pZbI9\\\\\\\/ARdVCBnUoPDW64U2MxoONkACY7bJu3OGRBca\\\\\\\/lu7pZhoMh9LOowuZ7XYmJbxVsfcl6XOtEzy7GBRkJpJFstnitQB\\\u0022,\\\u0022iv\\\u0022:\\\u0022YRIItciW+bXZS7BqxRIqRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fc42333e05dfbdcdb410270076a1e65c8c710eda18a896ff196a5d171f3f4f18\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022K3hwWe1OISnwg4HJNkBEDEoKrNd+z6nWXYMTJ0yJDjBhr2zBYKjOVclAjJbp+IvCU8RtpZiKqT\\\\\\\/DjZIug+wcEfEc62YBHVK8v5SO4zDC09FRHnapsIXwpYOqprgPGGYryd2kc3VtLOIEc7GPDSNIusIfGzEuagNDFZso5mrr\\\\\\\/OHZCVOUnC9ZmKF7yBG7TzlNQN2nOGIfA88Ezm0Ql2LIs\\\\\\\/jtjloyqynzmS3Xbc1otolp+3\\\\\\\/OjItoZO1kYxZnk4\\\\\\\/L1Ct5ihJ0k56apnFpU4FEuCTMGcb99LlxhH39urZdqWNJko3b2+ZL\\\\\\\/3r8Y7HQK4omepcICYqAwisKpclRVkjWgH\\\\\\\/HSqwsMsGi0w4kOzGVGDPeCPTITBPk3omn5cTcg1vIKPqNTxR\\\\\\\/LnsAcIgHm68v7MdJrnvHP1ZN93BP+osTRUxfZrkAqvIY8kCCVRptp0J+y+8FTJtYSQRvTKzKIujNDSkBEOcNSC8ZWPoYSWHcrg8su30YcmUXrfOSd+oPdP\\\\\\\/hTcFN37qip6eOjJ3NoveLmkGg\\\\\\\/PwZT9xTAAxuMc7db2sDhcJsr136dCybqIHIWGEKsae8wdt1nT71pczQj3yh6kMaXCtm3fdVaqBm+1lEV2kmSMO4oIplvVryI1xPKoaXcqsjNEKFxFCyDJJGQ\\\\\\\/6VleNpxDySKDc9x+dRRbMoHDYK0lTLKV5rJR1NERzXw3qCZNS4\\\\\\\/ziVVwOPv4dAL2tacBswUM7Wy2dsMf0cbfegJWNPkGmCOmG27CEUFI3DYfhsngoVFETh44FSzu7CjEtV9wzGb7MIqzZr7+8Yo+5k5MQ5vb301kOEaX7RhwPcP9PtJ3rz4zthHky00hQ6gaQFJD4P1hAzFrTdETd50LLRw1JsVwwkF3YQbrOmq2xF8k0fa3AWfZ4uOZc2VczJtwlx6NFzN3EnEhUFGQc\\\\\\\/xEJMsMj4a6WUCPLS45g4EeCZczVOTvOE+9e1sV7Gz\\\\\\\/dI2SuGlEVPXdp46J\\\\\\\/uQ8dDgzEXcyqn8YHqna7aSpmdoe\\\\\\\/aGdJdvCSUZfIaizYy477ac5Dw5ufiSdzYvWnuJAHdRO+xPTmplUykNt0XO6Lf3Zg07PzvKDR2Su9fohQIVmEJdAzMLp0TFjDk1YK\\\\\\\/0iy\\\\\\\/kfvlBqv79Wo8xGpZEQcryHjNdCCxD8zlhfTNuanoPce7XnpO1Js3NkPyaURf1fpwREx6Nd7GG2JQMENF\\\\\\\/\\\\\\\/PcuPqbOSLrnwkDyi5b1LoWr03sJwl9MulN8Fktw3lGfux3txh2GQU3J8NzhwnYlnx5VtCKCtoHhGCR7xZeNXc3SPI4XmY4TaWnymVQjzH9vuw\\\\\\\/2UoRCIyDtB9STL87bRNqeb94DFTVw5r4MHu5m0FcjaAMruOtlhjQ7mHP\\\\\\\/PNRmq1jgDK\\\\\\\/nfc7eJYYsJP0ngVJQaHT35mUnUDi9\\\\\\\/JCFyvHj1+OcZDv3aXepE+WjPeH8qUpOwHfQiAboqiOuMhesdQqB0tJwP2PeV84fhEY1rlvoIMWQHjaNv7rVHWIyJwohQxWLrJZJJDXuentWfNneGIux8UFYFvZNis8TKzx2aQEkQnlZbBExntrlMcYKmzXkN8EWs96nIrWH4AG6WsKW86nCTUhb6lwD9\\\\\\\/\\\\\\\/iOCRRfszTm5DseYZuMTQylcFg\\\\\\\/VBSy82awXq+E8l8wcDb0x87wX2eDU6b7Jk0uIOefn9HvKJaxomKCrWk8gGsGIhqBNQNvRZaeyKM8xQaFXTJeYXBQlujNNMIXg2Ujq6FEEgc8vRk8Z\\\\\\\/wy5y4li34jqn25DSte5V7Iq\\\\\\\/vUt3vhDhrjUHhC+WBG2O62+wMrJOk\\\\\\\/cuw1qAFcXZfR+S8pkyUxM4K60ela6iWVd8IuIl27ANsIfRbIP6ioT6UX2mRIfpizjxAeUcpKyOS1LKiGfFQDOOBHMC5Qdo1o1QksQv1EpvRzYzSyAr77Y9Vwz+Xs0iSyyO2Vd\\\\\\\/lGIbYUoJVusGjdcBTDZZBoyJq4Dnr+OOQfDncFIBh03R2pj9sAAgA6l6s4YlmqplW3kbbItSlFdc2uejxd0uHjNRMelTis4\\\\\\\/EVNYAz\\\\\\\/Hq6214WXgkzKD\\\u0022,\\\u0022iv\\\u0022:\\\u0022hfHHb5fvgHh9k1ctDLfu2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002296e99ec694cb3da8bec056a973174e50a39bfc210ea4d8cd5e8ea2a4f514bdb5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483842 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1085" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rf-DUsLSc2ukpmYKw2oMrzI7miztaJUoUpWrZq-T3W4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1085" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lCJoLZpIffw6htRf1yorYfpMBgw5v7mEdvaa9ww_Yd8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1061" title="Identification of Genes in Saccharomyces cerevisiae that Are Haploinsufficient for Overcoming Amino Acid Starvation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1097" title="Triploid Production from Interspecific Crosses of Two Diploid Perennial Helianthus with Diploid Cultivated Sunflower (Helianthus annuus L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483842" id="node483842" data-pisa="ggg;7/4/1085" data-pisa-master="ggg;g3.116.038075" data-apath="/ggg/7/4/1085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in <em>Mimulus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kimmy A. Stanton</span>, <span class="highwire-citation-author" data-delta="1">Patrick P. Edger</span>, <span class="highwire-citation-author" data-delta="2">Joshua R. Puzey</span>, <span class="highwire-citation-author" data-delta="3">Taliesin Kinser</span>, <span class="highwire-citation-author" data-delta="4">Philip Cheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4182-3105" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Vernon</span>, <span class="highwire-citation-author" data-delta="6">Nancy R. Forsthoefel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3730-0288" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arielle M. Cooley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1085-1095;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038075 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483842" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kimmy A. Stanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kimmy%2BA.%2BStanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stanton%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKimmy%2BA.%2BStanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick P. Edger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BP.%2BEdger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edger%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BP.%2BEdger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua R. Puzey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>College of William and Mary, Williamsburg, Virginia 23185</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BR.%2BPuzey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Puzey%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BR.%2BPuzey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Taliesin Kinser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>College of William and Mary, Williamsburg, Virginia 23185</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taliesin%2BKinser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kinser%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaliesin%2BKinser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Philip Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Daniel M. Vernon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BVernon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vernon%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BVernon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4182-3105" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Vernon</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nancy R. Forsthoefel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BR.%2BForsthoefel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forsthoefel%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANancy%2BR.%2BForsthoefel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Arielle M. Cooley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arielle%2BM.%2BCooley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArielle%2BM.%2BCooley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3730-0288" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arielle M. Cooley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cooleya@whitman.edu" class="" data-icon-position="" data-hide-link-title="0">cooleya@whitman.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/4/1085" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483842" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1085.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483842" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1085.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483842" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1085.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483842" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/4/1085.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483842" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483842/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">While quantitative PCR (qPCR) is widely recognized as being among the most accurate methods for quantifying gene expression, it is highly dependent on the use of reliable, stably expressed reference genes. With the increased availability of high-throughput methods for measuring gene expression, whole-transcriptome approaches may be increasingly utilized for reference gene selection and validation. In this study, RNA-seq was used to identify a set of novel qPCR reference genes and evaluate a panel of traditional “housekeeping” reference genes in two species of the evolutionary model plant genus <em>Mimulus</em>. More broadly, the methods proposed in this study can be used to harness the power of transcriptomes to identify appropriate reference genes for qPCR in any study organism, including emerging and nonmodel systems. We find that RNA-seq accurately estimates gene expression means in comparison to qPCR, and that expression means are robust to moderate environmental and genetic variation. However, measures of expression variability were only in agreement with qPCR for samples obtained from a shared environment. This result, along with transcriptome-wide comparisons, suggests that environmental changes have greater impacts on expression variability than on expression means. We discuss how this issue can be addressed through experimental design, and suggest that the ever-expanding pool of published transcriptomes represents a rich and low-cost resource for developing better reference genes for qPCR.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/expression-stability" class="hw-term hw-article-keyword hw-article-keyword-expression-stability" rel="nofollow">expression stability</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mimulus-guttatus" class="hw-term hw-article-keyword hw-article-keyword-mimulus-guttatus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mimulus guttatus</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mimulus-luteus" class="hw-term hw-article-keyword hw-article-keyword-mimulus-luteus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mimulus luteus</em></a></li><li class="kwd"><a href="/keyword/quantitative-rt-pcr" class="hw-term hw-article-keyword hw-article-keyword-quantitative-rt-pcr" rel="nofollow">quantitative RT-PCR</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-seq</a></li></ul><p id="p-5">qPCR is the premier method for quantifying gene expression because of its simplicity, accuracy, and low cost. However, the quantification accuracy of qPCR is dependent on normalization against reference genes to reduce the impact of technical noise and variation in sample preparation. qPCR data normalization is crucial for the reliable quantification of expression levels, so care must be taken to choose a reliable reference gene that has low variation in expression across diverse sample types (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dheda <em>et al.</em> 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gutierrez <em>et al.</em> 2008</a>). Traditionally, high expression housekeeping genes involved in basic cellular functions were used for qPCR normalization based on the assumption that they would be stably expressed (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Thellin <em>et al.</em> 1999</a>). Unfortunately, these traditional housekeeping reference genes, such as ubiquitin-conjugating enzyme (UBC), polyubiquitin (UBQ), β-actin, α- and β-tubulin, and glyceraldehyde 3-phosphate dehydrogenase (GAPDH), can exhibit surprisingly high expression variance in some species, or among different environmental conditions (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brunner <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dheda <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Suzuki <em>et al.</em> 2000</a>).</p><p id="p-6">In efforts to find alternatives to housekeeping genes, high-throughput technologies have been used to survey whole transcriptomes for novel, stably expressed genes. Microarrays have been successfully used for novel reference gene identification in a variety of plants, including <em>Arabidopsis thaliana</em>, <em>Eucalyptus</em>, and soybean (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Libault <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de Oliveira <em>et al.</em> 2012</a>). However, RNA-seq, a potentially more effective high-throughput method, has rarely been employed. RNA-seq has many advantages over microarrays: it does not require an assembled genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Haas and Zody 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Grabherr <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Robertson <em>et al.</em> 2010</a>), it has the power to identify novel transcripts and splice variants (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Trapnell <em>et al.</em> 2010</a>), and it is sensitive enough to quantify transcripts with very low expression levels (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Marioni <em>et al.</em> 2008</a>). In addition, RNA-seq is fast, relatively inexpensive, and shows minimal variation across technical replicates (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Marioni <em>et al.</em> 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mortazavi <em>et al.</em> 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nagalakshmi <em>et al.</em> 2008</a>). For all of these reasons, RNA-seq is an attractive, whole-transcriptome method for the detection of stably expressed genes and the identification of novel reference genes for qPCR normalization. This approach has rarely been used to evaluate potential qPCR reference genes [but see <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> (2012)</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yang <em>et al.</em> (2014)</a>, and <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhuang <em>et al.</em> (2015)</a>].</p><p id="p-7">A potential pitfall of both the microarray and the RNA-seq approach to reference gene selection is that there are no accepted practices for the analysis of expression variability within whole transcriptomes. Many methods for analyzing expression variability from qPCR data have been developed, including geNorm, BestKeeper, and NormFinder (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Vandesompele <em>et al.</em> 2002</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pfaffl <em>et al.</em> 2004</a>), but these programs can only analyze the expression data from a handful of genes at a time and, thus, are not useful for exploring whole transcriptomes. Without an established method for analysis, many diverse methods have been adopted for estimating expression variability within whole transcriptomes, including coefficient of variation (CV) calculations (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> 2005</a>), fold change cut-offs (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Yang <em>et al.</em> 2014</a>), and <em>P</em>-value cut-offs (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Libault <em>et al.</em> 2008</a>). However, no comparison of the different methods is currently available; each of the earlier studies included only a single whole-transcriptome measure of expression variability.</p><p id="p-8">One system in which a transcriptomic approach to reference gene selection has great potential to advance gene expression studies is the monkeyflower genus <em>Mimulus</em> [recently split into genera <em>Mimulus</em> and <em>Erythranthe</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barker <em>et al.</em> 2012</a>)]. <em>Mimulus</em> has become a widely used model for evolutionary genetic studies because of its phenotypic, ecological, and genetic variation, with centers of species diversity in both North and South America (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wu <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sobel and Streisfeld 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beardsley and Olmstead 2002</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Twyford <em>et al.</em> 2015</a>). <em>Mimulus</em> is a powerful system for genetic studies due to the interfertility of diverse species and the availability of genomic resources, including the genome sequence of <em>Mimulus guttatus</em>, <em>M. cardinalis</em>, <em>M. lewisii</em>, and <em>M. luteus</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hellsten <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yuan <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edger <em>et al.</em> 2016</a>). Yet, despite the utility of <em>Mimulus</em> for studying the evolution of genes and gene expression, the only evaluation of qPCR reference genes to date is a nonquantitative assessment of six housekeeping genes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Scoville <em>et al.</em> 2011</a>). A rigorous and quantitative genome-wide analysis of candidate qPCR reference genes is therefore of special utility for advancing evolutionary genetic studies in <em>Mimulus</em>.</p><p id="p-9">In this study, we systematically and quantitatively evaluate a panel of traditional reference genes and screen whole transcriptomes to identify a set of novel reference genes that can be used for qPCR expression studies in <em>Mimulus</em>. We utilize whole-transcriptome RNA-seq libraries from two species: <em>M. guttatus</em>, a North American diploid, and <em>M. luteus</em> var. <em>luteus</em>, a Chilean allotetraploid (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mukherjee and Vickery 1962</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Vallejo‐Marín <em>et al.</em> 2015</a>). We further develop the toolkit for transcriptome-enabled reference gene selection by comparing the utility of two distinct methods—the “CV method” and the “fold change cut-off method”—for identifying novel stably expressed genes from RNA-seq data.</p><p id="p-10">In these two <em>Mimulus</em> species, which differ in their ecology, ploidy, and level of resource development, we find that both the CV and fold change methods identify a similar set of novel reference genes. We propose that these highly stable genes provide a good starting pool of candidate reference genes for qPCR expression studies in <em>Mimulus</em>, and report that some traditional reference genes are also satisfactory according to standard quantitative guidelines for qPCR. In addition, we propose a workflow that incorporates either the CV or the fold change method to screen whole transcriptomes for novel reference genes in other systems. Across environmentally and genetically different plants, we found that gene expression means were relatively similar but expression variability fluctuated dramatically. Based on this finding, we suggest that transcriptomes should either be specific to the samples used for the planned qPCR study or should cover a wide span of biological and environmental diversity, in order for reference genes to be selected with high confidence.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-11">Two batches of each species were grown in separate greenhouses, providing the RNA samples for both RNA-seq and qPCR (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). <em>M. guttatus</em> genotype CG (Dublane, Scotland) and <em>M</em>. <em>luteus</em> var. <em>luteus</em> inbred line EY7 [El Yeso, Chile, see <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cooley <em>et al.</em> (2008)</a>]were grown at Duke University (NC). Transcriptome T1 was produced for <em>M. l. luteus</em> using stem, calyx, and petal tissues from a single individual grown at Duke University (NC). For <em>M. guttatus</em>, stem, calyx, petal, and leaf were sampled from two CG plants grown at Duke University. For transcriptome T2, <em>M. guttatus</em> inbred line IM767 [Iron Mountain, OR, see <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Willis (1999)</a>] and <em>M. l. luteus</em> inbred line EY7 (El Yeso, Chile) were grown at Whitman College (WA). RNA from four tissue types of a single individual of each species grown at Whitman College was sequenced to produce the second set of transcriptomes (T2). Two different leaf samples were collected from <em>M. l. luteus</em> and processed separately (L2A and L2B), to compensate for the lack of a leaf transcriptome in T1. RNA from four tissue types of four individuals (one of which was the same individual used for the T2 transcriptomes) from each species grown at Whitman College was extracted for use in qPCR.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sources of the plant materials that provided RNA for RNA-seq and qPCR. Because no leaf sample was available for M. l. luteus T1, two leaves were collected and sequenced for M. l. luteus T2, to enable leaf-to-leaf comparisons within M. l. luteus. M. guttatus genotype CG was collected in Dublane, Scotland; M. l. luteus highly inbred line EY7 was originally collected from El Yeso, Chile (Cooley et al. 2008); M. guttatus highly inbred line IM767 was originally collected from Iron Mountain, OR (Willis 1999). qPCR, quantitative polymerase chain reaction; RNA-seq, RNA sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-448557103" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sources of the plant materials that provided RNA for RNA-seq and qPCR. Because no leaf sample was available for &lt;em&gt;M. l. luteus&lt;/em&gt; T1, two leaves were collected and sequenced for &lt;em&gt;M. l. luteus&lt;/em&gt; T2, to enable leaf-to-leaf comparisons within &lt;em&gt;M. l. luteus. M. guttatus&lt;/em&gt; genotype CG was collected in Dublane, Scotland; &lt;em&gt;M. l. luteus&lt;/em&gt; highly inbred line EY7 was originally collected from El Yeso, Chile (Cooley &lt;em&gt;et al.&lt;/em&gt; 2008); &lt;em&gt;M. guttatus&lt;/em&gt; highly inbred line IM767 was originally collected from Iron Mountain, OR (Willis 1999). qPCR, quantitative polymerase chain reaction; RNA-seq, RNA sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1085/F1.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/4/1085/F1.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483882" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Sources of the plant materials that provided RNA for RNA-seq and qPCR. Because no leaf sample was available for <em>M. l. luteus</em> T1, two leaves were collected and sequenced for <em>M. l. luteus</em> T2, to enable leaf-to-leaf comparisons within <em>M. l. luteus. M. guttatus</em> genotype CG was collected in Dublane, Scotland; <em>M. l. luteus</em> highly inbred line EY7 was originally collected from El Yeso, Chile (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cooley <em>et al.</em> 2008</a>); <em>M. guttatus</em> highly inbred line IM767 was originally collected from Iron Mountain, OR (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Willis 1999</a>). qPCR, quantitative polymerase chain reaction; RNA-seq, RNA sequencing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">In the Whitman greenhouse, plants were grown with supplemental 14 hr lighting in Miracle-Gro potting soil (N:P:K = 0.21:0.11:0.16, The Scotts Company, Marysville, OH). Plants were maintained on “self-watering” capillary action flats with once-daily top-watering. Greenhouse temperatures, as recorded by a wall sensor, ranged from 16° to 36° daily. Plants were fertilized twice weekly with Miracle-Gro Bloom Booster (N:P:K = 15:30:15). In the Duke greenhouse, plants were grown with supplemental 16 hr lighting with twice-daily watering. Greenhouse temperatures ranged from 12° to 21° daily. Plants were fertilized with Peter’s Professional fertilizer every 2 wk, alternating between general purpose (N:P:K = 20:10:20) and low-phosphorus (N:P:K = 15:0:15) formulas, and fertilized with Jack’s Classic Blossom Booster (JR Peters INC, PA) (N:P:K = 10:30:20) every week to enhance flowering.</p><p id="p-14">Tissue was harvested from young, budding plants, usually between the first and third flower. Four tissue types were collected: young leaf (&lt;2.5 cm) near apical and lateral meristems, whole calyx from unemerged buds, petal (with stamen and pistil removed) from unemerged buds, and stem (∼2.5 cm segments) from newer plant growth. Tissue samples were flash frozen in liquid nitrogen and stored at −80° until the date of RNA extraction.</p></div><div id="sec-3" class="subsection"><h3>Transcriptome preparation and assembly</h3><p id="p-15">Total RNA was isolated from four tissue types (stem, leaf, calyx, and petal) from both <em>M. guttatus</em> and <em>M. l. luteus</em>. At Whitman, the Agilent Plant RNA Isolation Kit (Santa Clara, CA) was used, and at Duke, the Zymo Research Direct-Zol RNA MiniPrep (Irvine, CA) was used, following the manufacturer’s protocol, with on-column DNase I and elution in nuclease free water heated to 65°. RNA concentration and integrity were assessed using a NanoDrop Lite spectrophotometer (Thermo Fisher Scientific, DE) or Qubit fluorometer (Thermo Fisher Scientific).</p><p id="p-16">Whole-transcriptome, RNA-seq libraries were constructed for four tissue types from each of two biological replicates (T1 and T2) for both <em>M. guttatus</em> and <em>M. l. luteus</em> (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). T1 transcriptomes were prepared using the TruSeq RNA kit (Illumina, San Diego, CA) and then sequenced with single-end 100 bp reads using one lane of an Illumina HiSeq-2000 at the University of Missouri DNA core. T2 transcriptomes were prepared using the Kapa Stranded mRNA-Seq kit (Kapa Biosystems, Wilmington, MA) and were sequenced using one lane of an Illumina Hiseq-2500 at the Duke University DNA core.</p><p id="p-17">All Illumina reads were quality filtered using NextGENe v2.3.3.1 (SoftGenetics, State College, PA). Adapter sequences and reads with a median quality score of &lt;22 were removed, reads were trimmed at positions that had three consecutive bases with a quality score of &lt;20, and any trimmed reads with a total length &lt;40 bp were removed. This resulted in ∼87.9% of the reads passing the quality-score filter. Expression levels, in FPKM (fragments per kilobase per million reads), were determined for a total of 25,465 genes in <em>M. guttatus</em> (diploid) and 46,855 genes in <em>M. l. luteus</em> (tetraploid). Quality-filtered reads for each library were aligned to the respective genomes using NextGENe v2.3.3.1. Only uniquely mapped reads were counted, using the following parameters: A. matching Requirement: &gt;40 Bases and &gt;99%, B. Allow Ambiguous Mapping: FALSE, and C. Rigorous Alignment: TRUE. This resulted in the alignment of over 74.4 million reads to the diploid <em>M. guttatus</em> genome and 107.4 million reads to the tetraploid <em>M. l. luteus</em> genome.</p><p id="p-18">Genome completeness of the allotetraploid <em>M. l. luteus</em> in terms of gene content was assessed using BUSCO (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Simão <em>et al.</em> 2015</a>) with the default setting and a set of universal single-copy orthologs. The vast majority of BUSCO groups, 931 of 956 (97.4%), were identified in the <em>M. l. luteus</em> genome assembly, and 837 of those had duplicates. The high percentage of duplicate genes in this analysis indicates that homeologs were not collapsed during the assembly of the genome. This is further supported by comparative genomic analyses of both <em>Mimulus</em> genomes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Edger <em>et al.</em> 2016</a>), revealing a 2:1 genome-wide ratio of <em>M. l. luteus</em> (tetraploid): <em>M. guttatus</em> (diploid) syntenic blocks.</p></div><div id="sec-4" class="subsection"><h3>Analysis of RNA-seq libraries</h3><p id="p-19">Within each species, genes with expression levels lower than five FPKM in any of the eight transcriptomes were excluded from any of the further stability analyses. We reasoned that such low-expression genes would make poor qPCR references due to the difficulties in detecting and quantifying their expression. After their removal, a total of 7225 genes in <em>M. guttatus</em> and 10,755 genes in <em>M. l. luteus</em> were evaluated. Two methods were used for the analysis of expression stability: simple CV calculations and exclusion of differentially expressed genes [fold change method (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Robinson <em>et al.</em> 2010</a>)].</p><div id="sec-5" class="subsection"><h4>For the CV method:</h4><p id="p-20">Calculations for mean expression (mean), SD, and the CV were executed in Microsoft Excel or in R (Pumpkin Helmet, v.3.1.2). CV was calculated as SD/mean. Mean and SD were measured over the four tissue types of both biological replicates (eight samples in total) for each species. We adopted a CV cut-off for stable genes of 0.5, which was the cut-off for stable expression across heterogeneous samples advocated by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hellemans <em>et al.</em> (2007)</a>.</p></div><div id="sec-6" class="subsection"><h4>For the fold change method:</h4><p id="p-21">Log fold change was used to evaluate differential expression in pairwise sample comparisons. Genes with a high fold change (&gt;0.4 in <em>M. guttatus</em> and 0.3 in <em>M. l. luteus</em>) in any pairwise sample comparison were eliminated until a final list of stably expressed genes was obtained (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS1.xlsx">Table S1</a>). The cut-off values used in this study were selected so as to obtain a short list of genes with low variation in expression; the appropriate cut-off value can vary depending on the samples being analyzed and the overall goal of the analysis. The edgeR program (v. 3.12.0) was used to calculate log fold change because the program normalizes expression values by library size for each sample, but any method of fold change calculation can be used. The edgeR program was accessed through Bioconductor and analysis was executed in R.</p></div></div><div id="sec-7" class="subsection"><h3>Gene annotation</h3><p id="p-22">Stably expressed genes were annotated based on the agreement between BLAST results from the NCBI nucleotide database (<a href="http://blast.ncbi.nlm.nih.gov/">http://blast.ncbi.nlm.nih.gov/</a>) and from the annotated <em>M. guttatus</em> v.2 genome in the Phytozome v.10 database (<a href="http://phytozome.jgi.doe.gov/">http://phytozome.jgi.doe.gov/</a>). Traditional reference genes were identified in the RNA-seq datasets in a three-part method. First, known <em>A. thaliana</em> sequences for traditional reference genes 60s ribosomal protein L8 (L8), actin 2/7 (ACT), actin 11 (ACT1), β-tubulin 2 (TUB), ubiquitin 5 (UBQ), UBC 25, peroxin 4 (PEX), GAPDH-C1 (GAP), and EF1-α (EF1) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS2.docx">Table S2</a>) were used in a BLAST search against the <em>M. guttatus</em> v.2 genome in the Phytozome v.10 database in order to identify the appropriate <em>M. guttatus</em> homologs. Once a gene match with the correct annotation was identified in Phytozome, a short (∼20 bp) sequence from the coding region was then used to identify transcripts from the <em>M. guttatus</em> and <em>M. l. luteus</em> RNA-seq libraries. The resulting <em>M. guttatus</em> and <em>M. l. luteus</em> transcripts were used in a BLAST search against the NCBI nucleotide database to ensure that they had been correctly identified.</p></div><div id="sec-8" class="subsection"><h3>qPCR genes</h3><p id="p-23">Eight genes were selected for validation via qPCR (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS3.docx">Table S3</a>). Four traditional reference genes were selected based on both their widespread use in qPCR reference gene literature and on the ease of designing copy-specific primers. The four traditional genes chosen were ACT, GAP, PEX, and UBC. See the above section on <em>Gene annotation</em> for methods of gene identification within the transcriptome. Four additional genes were chosen based on their apparent stability across T1 tissues in both species, but were later found to be unstably expressed across T2 tissues (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS3.docx">Table S3</a>). However, these genes were retained for analysis in order to compare the qPCR and RNA-seq methods. The four genes chosen were mediator of RNA polymerase 12 (MRP), pectin acetylesterase (PAE), receptor-like kinase (RPK), and FYVE zinc-finger transcription factor (ZNF). The <em>M. guttatus</em> GenBank accession numbers for these eight genes, cataloged under <em>Erythranthe guttata</em> (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barker <em>et al.</em> 2012</a>), are: ACT = XM_012974510.1, GAP = XM_012999102.1, PEX = XM_013002418.1, UBC = XM_012995233.1, MRP = XM_012984744.1, PAE = XM_012984356.1, RPK = XM_012985914.1, and ZNF = XM_013000433.1.</p></div><div id="sec-9" class="subsection"><h3>qPCR primer design</h3><p id="p-24">qPCR primers were designed using Primer3 (<a href="http://biotools.umassmed.edu/bioapps/primer3_www.cgi">http://biotools.umassmed.edu/bioapps/primer3_www.cgi</a>) with the following criteria: <em>T</em><sub>m</sub> of 60 ± 3°, PCR amplicon length of 130–250 bp, primer length of 18–25 bp, and GC content of 35–60%. The <em>T</em><sub>m</sub> criterion was relaxed for UBC to 55 ± 3° to enable the discovery of suitable primers. Primers were designed to optimally sit as close to the 3′-end of the transcript as possible and to span an intron, but these criteria were relaxed in an effort to design primers that are homeolog-specific in the allotetraploid <em>M. l. luteus</em>. <em>M. l. luteus</em> primers were aligned with BLAST against the <em>M. l. luteus</em> (Illumina masked v1.1) genome in CoGe (<a href="https://genomevolution.org/CoGe/">https://genomevolution.org/CoGe/</a>) to ensure homeolog and paralog specificity. <em>M. guttatus</em> primers were aligned with BLAST against the <em>M. guttatus</em> genome (JGI hardmasked vV2) in CoGe to ensure copy specificity. Primers were synthesized by Invitrogen (Life Technologies). See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS4.docx">Table S4</a> for the full list of primer pairs.</p><p id="p-25">To verify primer specificity, PCR products were amplified by Taq DNA polymerase in a Mastercycler Nexus (Eppendorf, Germany), gel purified using the E.Z.N.A. kit (Omega Biotek), and Sanger sequenced by Eton Bioscience. Although all primers produced a single band on an agarose gel, the gel extraction step was included to produce cleaner and more concentrated sequencing products. Sequencing confirmed the copy specificity of all primer pairs except for the <em>M. l. luteus</em> RPK and PEX primer pairs, which targeted two and three paralogs, respectively.</p></div><div id="sec-10" class="subsection"><h3>cDNA synthesis</h3><p id="p-26">cDNA was synthesized from 1 μg of total RNA and a mixture of oligo dT and random primers using the Quanta qScript cDNA Synthesis kit (Quanta BioSciences, MD) and following the manufacturer’s protocol. cDNA was stored at 4° and unused RNA was stored at −80°.</p><p id="p-27">Quality controls for cDNA were twofold. First, all RNAs and cDNAs were checked for the absence of genomic DNA contamination using primers that surround an actin intron (5′-CCCAAGGCTAACAGGGAGAA-3′ and 5′-GTGCTGGATTCTGGTGACG-3′). Second, gene expression estimates were obtained from the 3′- <em>vs.</em> 5′-ends of a single gene. A 3′/5′ ratio substantially greater or less than 1 may indicate degradation of the mRNA template, or incomplete processivity of the reverse transcription reaction. The MIQE guidelines (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bustin <em>et al.</em> 2009</a>) suggest a range of 0.2–5.0 for samples to be used in qPCR. The 3′/5′ ratio of the receptor-like protein kinase cDNA was tested for all tissue types in each individual used in this study, using two primer pairs that amplify in the 5′ region (5′-TGGGCTCGAGTATTTTGCTT-3′ and 5′-TGCTTCCTAATCCAAAGATACCA-3′) or the 3′ region (5′-CCTGAGGGTGACAAGACACA-3′ and 5′-ATCAATGGACAAAAGCAGGC-3′) ∼1 kb away from each other. Some 3′/5′ ratios were found to be &gt;5 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS5.docx">Table S5</a>). This could result in an underestimation of expression for genes with primers in the 5′ region of the gene, which includes ACT in both species and the <em>M. guttatus</em> ZNF. The 3′/5′ ratios also had a tissue bias, with all stem cDNA samples and some of the calyx cDNA samples having values &gt;5.</p></div><div id="sec-11" class="subsection"><h3>qPCR conditions</h3><p id="p-28">Comparative qPCR was performed for four biological replicates (all from plants grown at Whitman College, see <em>Plant materials</em>) and three technical replicates for each tissue type (leaf, stem, petal, and calyx) from each of the two species (<em>M. guttatus</em> and <em>M. l. luteus</em>). A total of eight genes were selected for qPCR validation (see section <em>qPCR genes</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS3.docx">Table S3</a>) using the primers listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS4.docx">Table S4</a>. Reactions contained 1 × SYBR Green Master Mix (Brilliant III Ultra-Fast SYBR Kit, Agilent Technologies, CA), 400 nM of primer (except for when amplifying PEX4 from <em>M. l. luteus</em>, where 500 nM of primer was used), 1 μl of 1:500 diluted ROX dye, and 1 μl of cDNA (50 ng/μl), in a final volume of 12.5 μl. PCR reactions were performed in either optical eight-well PCR strips (Agilent Technologies) or optical 96-well plates (Greiner Bio-One, Belgium) using the Stratagene Mx3000P qPCR system (Agilent Technologies). Samples were amplified for 40 cycles of 10 sec at 95° and 20 sec at the appropriate annealing temperature (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS4.docx">Table S4</a>), after an initial denaturation step at 95° for 3 min. An additional dissociation curve was recorded after cycle 40 by heating from 55° to 95° with a ramp speed of 0.01° per second (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/FigureS1.docx">Figure S1</a>). Raw qPCR fluorescence data were collected and analyzed by the default settings of the MxPro software v.4.10 (Agilent Technologies). <em>C</em>q (“quantification cycle,” the cycle in which fluorescence from DNA amplification first exceeds background fluorescence) was determined at a fluorescence threshold of 0.23 for all runs; this fixed threshold was based on the average adaptive threshold of all individual runs. Amplification efficiencies for each primer pair were determined using the <em>C</em>q values obtained from a 1/4 dilution series (1:4, 1:16, 1:64, 1:256, and 1:1024) where <em>E</em> = 10<sup>(1/−slope)</sup>. Efficiency for each primer pair was calculated to be between 83 and 102% using the standard curve method (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS6.docx">Table S6</a>).</p></div><div id="sec-12" class="subsection"><h3>Analysis of qPCR expression data</h3><p id="p-29">Before analysis, the <em>C</em>q values from qPCR were averaged over the three technical replicates, unless the replicates differed by &gt;1 <em>C</em>q. In that case, the outlier technical replicate was removed and <em>C</em>q was averaged over the two remaining technical replicates. These averages were then both calibrator and efficiency normalized using the equation below. GAP amplified from the same sample of <em>M. l. luteus</em> young leaf cDNA acted as the interplate calibrator. Efficiency values for each gene are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS6.docx">Table S6</a>. Relative expression of each gene was calculated as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1085/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1085/embed/mml-math-1.gif"/></noscript></span></span>In order to have a metric of gene stability that could be directly compared to stability estimates from RNA-seq data, the CV was calculated for each gene from the relative qPCR expression data. Calculations for mean expression (mean), SD, and for the CV (CV = SD/mean) were executed in Microsoft Excel. SD and mean were calculated from the relative expression of each of the four tissue types, averaged over the four biological replicates per tissue.</p></div><div id="sec-13" class="subsection"><h3>Statistical analyses</h3><p id="p-30">All statistical tests were run using R software (Pumpkin Helmet, v.3.1.2). Linear models were fitted to obtain <em>t</em>-test results and Pearson’s correlation coefficient.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-31">All transcriptomic expression data are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS7.xlsx">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS8.xlsx">Table S8</a>. Primer sequences are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS4.docx">Table S4</a>. Raw reads from this study are deposited in Dryad (<a href="http://dx.doi.org/10.5061/dryad.84655">http://dx.doi.org/10.5061/dryad.84655</a>) and are further analyzed in <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Edger <em>et al.</em> (2016)</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Identification of novel reference genes for Mimulus</h3><p id="p-32">In order to identify potential qPCR reference genes, we compared two simple methods for evaluating variation in expression across tissue types and growing environments: (a) genes with the lowest overall CV across all tissues from both transcriptome sets (T1 and T2; see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) and (b) exclusion of differentially expressed genes, determined through calculations of fold change, between pairwise comparisons of all tissue samples from both transcriptome sets. We identified 50 genes per species using the CV method and eight genes per species using the fold change method (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS9.xlsx">Table S9</a>) that have the potential to be good candidate reference genes for qPCR studies in <em>Mimulus</em>.</p><p id="p-33">Although CV was not correlated with total expression level (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/FigureS2.docx">Figure S2</a>), we used a minimum expression cut-off of 5 FPKM in order to exclude genes that are expressed at levels too low to be useful for qPCR normalization. The 50 genes with the lowest CV across both biological replicates of each species are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS9.xlsx">Table S9</a>. Genes on this list have CVs &lt;0.14 for <em>M. guttatus</em> and &lt;0.12 for <em>M. l. luteus</em>. Although a 0.50 CV cut-off has previously been recommended for choosing qPCR reference genes (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hellemans <em>et al.</em> 2007</a>), we find that the majority of robustly expressed genes fall under this cut-off (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In <em>M. guttatus</em>, 4106 genes out of 7225 had a CV of &lt;0.50; in <em>M. l. luteus</em>, 6832 genes out of 10,755 were under this cut-off.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of CV for all reliably expressed genes (&gt;5 FPKM in all samples) in (A) M. guttatus and (B) M. l. luteus. The dashed line marks the 0.50 CV cut-off for stably expressed genes and the arrows point to the two traditional reference genes with the lowest variation in expression for each species (Table S2). The portion of the density curves containing the top 200 genes with the lowest CV are shaded black; all genes selected using the CV and fold change method fall within this region. ACT, actin 2/7; CV, coefficient of variation; FPKM, fragments per kilobase per million reads; PEX, peroxin 4; TUB, &#x3B2;-tubulin 2; UBC, ubiquitin-conjugating enzyme." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-448557103" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of CV for all reliably expressed genes (&gt;5 FPKM in all samples) in (A) &lt;em&gt;M. guttatus&lt;/em&gt; and (B) &lt;em&gt;M. l. luteus&lt;/em&gt;. The dashed line marks the 0.50 CV cut-off for stably expressed genes and the arrows point to the two traditional reference genes with the lowest variation in expression for each species (Table S2). The portion of the density curves containing the top 200 genes with the lowest CV are shaded black; all genes selected using the CV and fold change method fall within this region. ACT, actin 2/7; CV, coefficient of variation; FPKM, fragments per kilobase per million reads; PEX, peroxin 4; TUB, &#x3B2;-tubulin 2; UBC, ubiquitin-conjugating enzyme.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1085/F2.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/4/1085/F2.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Distribution of CV for all reliably expressed genes (&gt;5 FPKM in all samples) in (A) <em>M. guttatus</em> and (B) <em>M. l. luteus</em>. The dashed line marks the 0.50 CV cut-off for stably expressed genes and the arrows point to the two traditional reference genes with the lowest variation in expression for each species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS2.docx">Table S2</a>). The portion of the density curves containing the top 200 genes with the lowest CV are shaded black; all genes selected using the CV and fold change method fall within this region. ACT, actin 2/7; CV, coefficient of variation; FPKM, fragments per kilobase per million reads; PEX, peroxin 4; TUB, β-tubulin 2; UBC, ubiquitin-conjugating enzyme.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">For the fold change method, any genes with a log fold change &gt;0.4 in <em>M. guttatus</em> or 0.3 in <em>M. l. luteus</em>, in any pairwise sample comparison, were excluded. Eight <em>M. guttatus</em> and eight <em>M. l. luteus</em> genes were identified in this manner that had low variation in expression across the four tissue types from two biological replicates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS1.xlsx">Table S1</a>). The fold change method was consistent with the CV method; five <em>M. guttatus</em> genes and one <em>M. l. luteus</em> gene identified by the fold change method are also found on the top 50 CV list, and all of the genes identified by the fold change method are listed within the top 200 genes with the lowest CV (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS10.xlsx">Table S10</a>).</p></div><div id="sec-17" class="subsection"><h3>Traditional reference genes in Mimulus</h3><p id="p-36">Since traditional reference genes can be inconsistently expressed in many biological systems (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brunner <em>et al.</em> 2004</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> 2005</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dheda <em>et al.</em> 2004</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Suzuki <em>et al.</em> 2000</a>), we investigated the expression variability of these traditional housekeeping reference genes in <em>Mimulus</em> using both transcriptomics and qPCR. We chose nine common traditional reference genes to analyze from the RNA-seq datasets: L8, ACT, ACT1, TUB, UBQ, UBC, PEX, GAP, and EF1 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS2.docx">Table S2</a>). We then corroborated the expression variability for four of these nine genes (ACT, PEX, UBC, and GAP) using qPCR (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS3.docx">Table S3</a>).</p><p id="p-37">In both <em>M. guttatus</em> and <em>M. l. luteus</em>, there were thousands of expressed genes with lower CVs than the traditional housekeeping genes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS2.docx">Table S2</a>), and none of the traditional housekeeping genes were among the 16 genes identified by the fold change method. Nevertheless, four traditional genes in <em>M. guttatus</em> (GAP, UBC, TUB, and PEX) and four in <em>M. l. luteus</em> (L8, GAP, ACT, and UBC) do have CVs &lt;0.5, suggesting that they could be useful reference genes for qPCR normalization in these species (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression variability estimates for selected traditional reference genes, based on coefficient of variation (CV). Expression variability in M. guttatus (A) and M. l. luteus (B) measured via RNA sequencing (RNA-seq) on both T1 and T2 (left column) or via quantitative polymerase chain reaction (qPCR) on T2 samples only (right column). Gray bars show the calculated CV when all tissue types are included and black bars show the calculated CV when petal tissue is excluded. Genes are ordered from most variable to least variable, with a dash line showing a previously suggested cut-off for usable reference genes at 0.50 CV. For the tetraploid M. l. luteus, CVs reported for the RNA-seq data are the average of both homeologs. The genes tested include 60s ribosomal protein L8 (L8), actin 2/7 (ACT), actin 11 (ACT1), &#x3B2;-tubulin 2 (TUB), ubiquitin 5 (UBQ), ubiquitin conjugating enzyme 25 (UBC), peroxin 4 (PEX), GAPDH-C1 (GAP), and EF1-&#x3B1; (EF1). CV was calculated from fragments per kilobase per million reads values for genes measured via RNA-seq and from relative expression values, calculated by Efficiency&#x394;Cq, where &#x394;Cq = CqCalibrator &#x2013; Cqsample, for genes measured via qPCR." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-448557103" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression variability estimates for selected traditional reference genes, based on coefficient of variation (CV). Expression variability in &lt;em&gt;M. guttatus&lt;/em&gt; (A) and &lt;em&gt;M. l. luteus&lt;/em&gt; (B) measured via RNA sequencing (RNA-seq) on both T1 and T2 (left column) or via quantitative polymerase chain reaction (qPCR) on T2 samples only (right column). Gray bars show the calculated CV when all tissue types are included and black bars show the calculated CV when petal tissue is excluded. Genes are ordered from most variable to least variable, with a dash line showing a previously suggested cut-off for usable reference genes at 0.50 CV. For the tetraploid &lt;em&gt;M. l. luteus&lt;/em&gt;, CVs reported for the RNA-seq data are the average of both homeologs. The genes tested include 60s ribosomal protein L8 (L8), actin 2/7 (ACT), actin 11 (ACT1), &#x3B2;-tubulin 2 (TUB), ubiquitin 5 (UBQ), ubiquitin conjugating enzyme 25 (UBC), peroxin 4 (PEX), GAPDH-C1 (GAP), and EF1-&#x3B1; (EF1). CV was calculated from fragments per kilobase per million reads values for genes measured via RNA-seq and from relative expression values, calculated by Efficiency&lt;sup&gt;&#x394;&lt;/sup&gt;&lt;em&gt;&lt;sup&gt;C&lt;/sup&gt;&lt;/em&gt;&lt;sup&gt;q&lt;/sup&gt;, where &#x394;&lt;em&gt;C&lt;/em&gt;q = &lt;em&gt;C&lt;/em&gt;q&lt;sub&gt;Calibrator&lt;/sub&gt; &#x2013; &lt;em&gt;C&lt;/em&gt;q&lt;sub&gt;sample&lt;/sub&gt;, for genes measured via qPCR.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1085/F3.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/4/1085/F3.medium.gif" width="440" height="263"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483901" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Expression variability estimates for selected traditional reference genes, based on coefficient of variation (CV). Expression variability in <em>M. guttatus</em> (A) and <em>M. l. luteus</em> (B) measured via RNA sequencing (RNA-seq) on both T1 and T2 (left column) or via quantitative polymerase chain reaction (qPCR) on T2 samples only (right column). Gray bars show the calculated CV when all tissue types are included and black bars show the calculated CV when petal tissue is excluded. Genes are ordered from most variable to least variable, with a dash line showing a previously suggested cut-off for usable reference genes at 0.50 CV. For the tetraploid <em>M. l. luteus</em>, CVs reported for the RNA-seq data are the average of both homeologs. The genes tested include 60s ribosomal protein L8 (L8), actin 2/7 (ACT), actin 11 (ACT1), β-tubulin 2 (TUB), ubiquitin 5 (UBQ), ubiquitin conjugating enzyme 25 (UBC), peroxin 4 (PEX), GAPDH-C1 (GAP), and EF1-α (EF1). CV was calculated from fragments per kilobase per million reads values for genes measured via RNA-seq and from relative expression values, calculated by Efficiency<sup>Δ</sup><em><sup>C</sup></em><sup>q</sup>, where Δ<em>C</em>q = <em>C</em>q<sub>Calibrator</sub> – <em>C</em>q<sub>sample</sub>, for genes measured via qPCR.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The follow-up qPCR validation reported much lower expression variability for the tested subset of traditional genes. This is most likely due to a less variable group of plants being measured for qPCR than were measured for RNA-seq (see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Expression variability was even lower when measures from petal tissue were excluded (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), as expression levels for all four tested genes were substantially higher in petal tissue than in the other three tissue types (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/FigureS3.docx">Figure S3</a>). This is only the case for the qPCR data and there is no trend in the RNA-seq data when petal is excluded, even though transcriptome T2 was derived from one of the same RNA samples that was used for qPCR. When all tissues were included in the qPCR variability calculations, we found that GAP had the lowest variation in expression in <em>M. guttatus</em> and PEX was the least variable in <em>M. l. luteus</em>. When petal was excluded, UBC was the least variable traditional reference gene in both species.</p></div><div id="sec-18" class="subsection"><h3>Efficacy of transcriptomics for reference gene selection</h3><p id="p-40">Although environmental condition was not a purposeful manipulation in our study, the different growth histories of the genetically identical plants used for the two <em>M. l. luteus</em> transcriptome sets allowed us to evaluate the robustness of gene expression to moderate environmental variation. This was achieved by comparing both mean expression and expression variability (measured by CV) across the different tissue types between T1 and T2. For comparison, we also evaluated the two <em>M. guttatus</em> transcriptome sets, although the plants used in this comparison were genetically as well as environmentally different (see <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-41">The correlation in CV between T1 and T2 is weaker than the correlation in mean expression for both species, showing a stronger environmental effect on the variance than on the mean (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Additionally, CV estimates were more closely correlated between the replicates of <em>M. l. luteus</em> than between the replicates of <em>M. guttatus</em>, as expected given that the <em>M. l. luteus</em> replicates came from the same highly inbred line of plants while the <em>M. guttatus</em> replicates came from different lineages.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation in expression mean (FPKM) and variability (CV), as measured by RNA-seq, between the biological replicates (T1 and T2) of M. guttatus (A) and M. l. luteus (B). Values for Pearson&#x2019;s correlation coefficient are given above each graph and the line of best fit is shown in black. Genes with mean expression " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-448557103" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlation in expression mean (FPKM) and variability (CV), as measured by RNA-seq, between the biological replicates (T1 and T2) of &lt;em&gt;M. guttatus&lt;/em&gt; (A) and &lt;em&gt;M. l. luteus&lt;/em&gt; (B). Values for Pearson&#x2019;s correlation coefficient are given above each graph and the line of best fit is shown in black. Genes with mean expression &lt;1 FPKM were excluded from the plot. CV, coefficient of variation; FPKM, fragments per kilobase per million reads; RNA-seq, RNA sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1085/F4.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/4/1085/F4.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Correlation in expression mean (FPKM) and variability (CV), as measured by RNA-seq, between the biological replicates (T1 and T2) of <em>M. guttatus</em> (A) and <em>M. l. luteus</em> (B). Values for Pearson’s correlation coefficient are given above each graph and the line of best fit is shown in black. Genes with mean expression &lt;1 FPKM were excluded from the plot. CV, coefficient of variation; FPKM, fragments per kilobase per million reads; RNA-seq, RNA sequencing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The expression data collected via RNA-seq were validated for accuracy using qPCR expression data for a selected group of eight genes, including four traditional reference genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS3.docx">Table S3</a>). Mean expression values measured by RNA-seq transcriptomes T1 and T2 were both in agreement with values found via qPCR (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). In contrast, expression variability estimated by qPCR was significantly correlated with T2 expression variability, but had no significant relationship to T1 expression variability (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). This is most likely due to variation in plant lineage and plant growth conditions, as the T2 transcriptomes and the qPCR data derive from genetically identical plants that were grown in the same greenhouse, while the T1 transcriptomes derive from plants grown in a greenhouse at a separate institution. For <em>M. guttatus</em>, T1 and T2 also differed in the accession used (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). This pattern, particularly for the isogenic <em>M. l. luteus</em> transcriptomes, suggests that environmental factors may have a greater effect on the “noise” in gene expression than on the expression level itself.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of relative gene expression and of expression variability as determined by RNA-seq and qPCR for a sample of four traditional reference genes (closed symbols) and four additional genes that had initially been found to be stably expressed in transcriptome T1 (open symbols) (Table S3). The T2 RNA-seq transcriptomes and the qPCR data were derived from genetically identical plants grown in the same greenhouse, while the T1 RNA-seq transcriptomes were derived from plants grown in a greenhouse at a separate institution (Figure 1). (A) There is a strong correlation in relative expression determined by qPCR and RNA-seq, for both T1 (left panel, r = 0.90, P &lt; 0.001) and T2 (right panel, r = 0.85, P &lt; 0.001). (B) Expression variability (CV) measured via qPCR is correlated with expression variability measured via T2 (right panel, r = 0.74, P = 0.001), but is not correlated to expression variability measured via T1 (left panel, r = &#x2212;0.42, P = 0.104). Expression data from both M. guttatus and M. l. luteus are included together. Relative expression of T1 and T2 is given in FPKM. Relative qPCR expression = Efficiency&#x394;Cq, where &#x394;Cq = CqCalibrator &#x2212; Cqsample. CV, coefficient of variation; FPKM, fragments per kilobase per million reads; qPCR, quantitative polymerase chain reaction; RNA-seq, RNA sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-448557103" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons of relative gene expression and of expression variability as determined by RNA-seq and qPCR for a sample of four traditional reference genes (closed symbols) and four additional genes that had initially been found to be stably expressed in transcriptome T1 (open symbols) (Table S3). The T2 RNA-seq transcriptomes and the qPCR data were derived from genetically identical plants grown in the same greenhouse, while the T1 RNA-seq transcriptomes were derived from plants grown in a greenhouse at a separate institution (Figure 1). (A) There is a strong correlation in relative expression determined by qPCR and RNA-seq, for both T1 (left panel, &lt;em&gt;r&lt;/em&gt; = 0.90, &lt;em&gt;P&lt;/em&gt; &lt; 0.001) and T2 (right panel, &lt;em&gt;r&lt;/em&gt; = 0.85, &lt;em&gt;P&lt;/em&gt; &lt; 0.001). (B) Expression variability (CV) measured via qPCR is correlated with expression variability measured via T2 (right panel, &lt;em&gt;r&lt;/em&gt; = 0.74, &lt;em&gt;P&lt;/em&gt; = 0.001), but is not correlated to expression variability measured via T1 (left panel, &lt;em&gt;r&lt;/em&gt; = &#x2212;0.42, &lt;em&gt;P&lt;/em&gt; = 0.104). Expression data from both &lt;em&gt;M. guttatus&lt;/em&gt; and &lt;em&gt;M. l. luteus&lt;/em&gt; are included together. Relative expression of T1 and T2 is given in FPKM. Relative qPCR expression = Efficiency&lt;sup&gt;&#x394;&lt;/sup&gt;&lt;em&gt;&lt;sup&gt;C&lt;/sup&gt;&lt;/em&gt;&lt;sup&gt;q&lt;/sup&gt;, where &#x394;&lt;em&gt;C&lt;/em&gt;q = &lt;em&gt;C&lt;/em&gt;q&lt;sub&gt;Calibrator&lt;/sub&gt; &#x2212; &lt;em&gt;C&lt;/em&gt;q&lt;sub&gt;sample&lt;/sub&gt;. CV, coefficient of variation; FPKM, fragments per kilobase per million reads; qPCR, quantitative polymerase chain reaction; RNA-seq, RNA sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1085/F5.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/4/1085/F5.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1085/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483994" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Comparisons of relative gene expression and of expression variability as determined by RNA-seq and qPCR for a sample of four traditional reference genes (closed symbols) and four additional genes that had initially been found to be stably expressed in transcriptome T1 (open symbols) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS3.docx">Table S3</a>). The T2 RNA-seq transcriptomes and the qPCR data were derived from genetically identical plants grown in the same greenhouse, while the T1 RNA-seq transcriptomes were derived from plants grown in a greenhouse at a separate institution (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). (A) There is a strong correlation in relative expression determined by qPCR and RNA-seq, for both T1 (left panel, <em>r</em> = 0.90, <em>P</em> &lt; 0.001) and T2 (right panel, <em>r</em> = 0.85, <em>P</em> &lt; 0.001). (B) Expression variability (CV) measured via qPCR is correlated with expression variability measured via T2 (right panel, <em>r</em> = 0.74, <em>P</em> = 0.001), but is not correlated to expression variability measured via T1 (left panel, <em>r</em> = −0.42, <em>P</em> = 0.104). Expression data from both <em>M. guttatus</em> and <em>M. l. luteus</em> are included together. Relative expression of T1 and T2 is given in FPKM. Relative qPCR expression = Efficiency<sup>Δ</sup><em><sup>C</sup></em><sup>q</sup>, where Δ<em>C</em>q = <em>C</em>q<sub>Calibrator</sub> − <em>C</em>q<sub>sample</sub>. CV, coefficient of variation; FPKM, fragments per kilobase per million reads; qPCR, quantitative polymerase chain reaction; RNA-seq, RNA sequencing.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><div id="sec-20" class="subsection"><h3>Identification of novel reference genes for Mimulus</h3><p id="p-45">While RNA-seq has the potential to accurately identify genes with low variation in expression, there is still not a universally accepted method for selecting reference genes from RNA-seq data. Most of the programs that are widely used for reference gene selection, such as geNorm, BestKeeper, and NormFinder, were designed specifically for qPCR data and can only process a handful of genes at a time (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Vandesompele <em>et al.</em> 2002</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2004</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Pfaffl <em>et al.</em> 2004</a>). We explored two different methods for identifying stably expressed genes from whole-transcriptome data: (1) ranking genes based on the CV of expression across different samples (CV method) and (2) excluding unstable genes using a log fold change cut-off value (fold change method). We find that both methods identify many stably expressed genes that have the potential to be novel reference genes for qPCR expression studies in <em>M. guttatus</em> and <em>M. l. luteus</em> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS9.xlsx">Table S9</a>).</p><p id="p-46">Using the CV method, all expressed genes from <em>M. guttatus</em> and <em>M. l. luteus</em> were ranked based on the variability of their expression across different tissue types and growing conditions, and the top 50 genes with the lowest variability were identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS9.xlsx">Table S9</a>). Using the fold change method, we identified eight <em>M. guttatus</em> and eight <em>M. l. luteus</em> genes with low variability in expression across four different tissue types and two biological replicates. No traditional reference genes were identified as being among the top 50 most stably expressed genes by either of our methods. In addition, the novel reference genes we identified had much lower expression variability in our system than any of the most commonly used traditional reference genes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), which highlights the utility of the whole-transcriptome approach to reference gene selection.</p><p id="p-47">The advantage of using either of these methods for reference gene selection is their simplicity in calculation. While the fold change method has the benefit of producing a discrete list of genes with low variation in expression, the CV method has the benefit of quantifying expression variability in a way where genes can be ranked and directly compared. These methods have previously been used in other plant species to select novel reference genes from transcriptomic data (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> 2005</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> 2012</a>), but we are the first to show that these two methods produce comparable results. All of the genes found on the fold change short-list were among the 200 genes with the lowest CV, which corresponds to the top 2–3% most stably expressed transcripts. Ideally, novel reference genes would be selected that score well according to both metrics.</p><p id="p-48">The CVs of the novel reference genes we identified are all &lt;0.20, whereas a previously suggested cut-off for valid reference genes is a CV of 0.50 (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hellemans <em>et al.</em> 2007</a>). It is important to note that using a 0.50 CV cut-off in our system included the majority of expressed genes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), and thus it was not a very discriminatory standard for determining expression variability.</p></div><div id="sec-21" class="subsection"><h3>Traditional reference genes in Mimulus</h3><p id="p-49">Many studies have pointed to the instability of traditional housekeeping reference genes (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Brunner <em>et al.</em> 2004</a>; <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> 2005</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dheda <em>et al.</em> 2004</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Suzuki <em>et al.</em> 2000</a>). We find that some traditional reference genes in <em>Mimulus</em> have the potential to work well for qPCR normalization. Using a whole-transcriptome method, we identified four traditional reference genes that have somewhat low variation in expression (CV &lt; 0.50) in <em>M. guttatus</em> and <em>M. l. luteus</em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Two genes, UBC and GAP, were even identified as stably expressed in both species and could potentially be good universal reference genes for the <em>Mimulus</em> genus. We confirmed our findings for four of these traditional reference genes with qPCR and found that all four (GAP, ACT, UBC, and PEX) could be acceptable as reference genes for both species based on qPCR estimates of expression variability across tissues, although some of the genes were at or slightly above the recommended 0.5 CV cut-off when the relatively divergent petal tissue samples were included (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). However, these traditional reference genes were nowhere near the most stably expressed in the transcriptome as a whole (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), which highlights the opportunity to discover dramatically more stable reference genes using a transcriptome-guided approach.</p><p id="p-50">Despite the widespread use of <em>Mimulus</em> as a model genus for genetics, very few papers have attempted to validate reference genes for use in this genus. As part of a larger study, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Scoville <em>et al.</em> (2011)</a> qualitatively ranked the expression variability of six traditional reference genes in <em>M. guttatus</em> and found that UBQ and EF1 were the most stably expressed. We quantitatively investigated four of these six traditional reference genes in our own study and found that UBQ and EF1 had higher expression variability than other traditional reference genes and that, in both species, the genes' CVs were &gt;0.5 under our study conditions. <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Scoville <em>et al.</em> (2011)</a> tested different lines of <em>M. guttatus</em> and included a wound treatment, which may have resulted in our differing reports of traditional reference gene stabilities. This again highlights the importance of reference gene validation for specific study conditions.</p><p id="p-51">Although we found that some traditional reference genes can be used for qPCR normalization, they are not optimal reference genes; the variability in expression of the traditional reference genes is very high when compared to the variability of all robustly expressed genes (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1/TableS2.docx">Table S2</a>). This indicates that whole-transcriptome approaches, such as RNA-seq, have great potential to discover novel reference genes that are stably expressed in the study system of interest. With the current speed and low cost of RNA-seq, as well as the online availability of multi-tissue and/or multi-environment RNA-seq data sets, we expect that the whole-transcriptome approach will be increasingly useful for reference gene identification and validation.</p></div><div id="sec-22" class="subsection"><h3>Efficacy of transcriptomics for reference gene selection</h3><p id="p-52">RNA-seq has been repeatedly shown to generate accurate measurements of gene expression (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Marioni <em>et al.</em> 2008</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Mortazavi <em>et al.</em> 2008</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Nagalakshmi <em>et al.</em> 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nookaew <em>et al.</em> 2012</a>). We find similar results in <em>Mimulus</em> when comparing the relative expression determined by RNA-seq to the relative expression determined by qPCR for eight selected genes (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). We also find that estimates of expression mean are robust to moderate environmental and genetic variation, but that estimates of expression variability across tissue types are only in agreement when the samples are obtained from a shared environment (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). These results suggest that environmental changes may have a greater impact on expression variability than on expression means.</p><p id="p-53">For the goal of reference gene selection, where expression variability must remain low, this difficulty can be solved in two ways. One approach is to use the same samples for both RNA-seq and the subsequent qPCR analysis, as in <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> (2012)</a> and <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Yang <em>et al.</em> (2014)</a>. This method would be highly accurate, but would be extremely specific to particular study conditions. A second approach would be to evaluate a large variety of genotypes or growth conditions to discover genes that are maximally stable across genetically and environmentally distinct samples, as was done for <em>A. thaliana</em> in <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Czechowski <em>et al.</em> (2005)</a>. This method would allow for the identification of a starting pool of “universally” stable genes.</p></div><div id="sec-23" class="subsection"><h3>Reference gene selection using RNA-seq</h3><p id="p-54">We show, using <em>Mimulus</em> as a case study, that RNA-seq is a promising tool for selecting genes with low gene expression variance that can be used as novel qPCR reference genes. As many research labs regularly use RNA-seq as a first approach to collecting expression data, already completed RNA-seq transcriptomes are a readily available tool that can be used to search for candidate qPCR reference genes in any study system. Although we find that the variance in expression is variable between environmental conditions, we propose that transcriptomes from diverse samples can be pooled in order to identify more universally stable genes. We show that two simple methods for identifying genes with low expression variance, the CV method and the fold change method, both result in comparable evaluations of expression variance. Thus, either of these methods can be used to identify a preliminary set of highly stable candidate reference genes for qPCR experiments.</p></div></div><div class="section" id="sec-24"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank Mario Vallejo-Marín and John Kelly for seeds of <em>M. guttatus</em> CG and IM767 respectively. We thank Paul Whetstone and two anonymous reviewers for helpful comments on the manuscript. Funds for this research were provided by Murdock Life Sciences grant #2013265 to A.M.C.; a Franklin Research Grant to A.M.C.; and Michigan State University AgBioResearch funds to P.P.E.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038075/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: E. Grotewold</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-12-01"><span class="received-label">Received </span>December 1, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-05"><span class="accepted-label">Accepted </span>January 5, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Stanton <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.1" data-doi="10.1158/0008-5472.CAN-04-0496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orntoft</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">64</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">5245</span>–<span class="cit-lpage">5250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DAndersen%26rft.auinit1%253DC.%2BL.%26rft.volume%253D64%26rft.issue%253D15%26rft.spage%253D5245%26rft.epage%253D5250%26rft.atitle%253DNormalization%2Bof%2BReal-Time%2BQuantitative%2BReverse%2BTranscription-PCR%2BData%253A%2BA%2BModel-Based%2BVariance%2BEstimation%2BApproach%2Bto%2BIdentify%2BGenes%2BSuited%2Bfor%2BNormalization%252C%2BApplied%2Bto%2BBladder%2Band%2BColon%2BCancer%2BData%2BSets%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-04-0496%26rft_id%253Dinfo%253Apmid%252F15289330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NC8xNS81MjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesom</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beardsley</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraga</span>  <span class="cit-name-given-names">N. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A taxonomic conspectus of Phrymaceae: a narrowed circumscription for <em>Mimulus</em>, new and resurrected genera, and new names and combinations.</span> <abbr class="cit-jnl-abbrev">Phytoneuron</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytoneuron%26rft.volume%253D39%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.3" data-doi="10.3732/ajb.89.7.1093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beardsley</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Redefining Phrymaceae: the placement of <em>Mimulus</em>, tribe Mimuleae and Phryma.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">89</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1093</span>–<span class="cit-lpage">1102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DBeardsley%26rft.auinit1%253DP.%2BM.%26rft.volume%253D89%26rft.issue%253D7%26rft.spage%253D1093%26rft.epage%253D1102%26rft.atitle%253DRedefining%2BPhrymaceae%253A%2Bthe%2Bplacement%2Bof%2BMimulus%252C%2Btribe%2BMimuleae%252C%2Band%2BPhryma%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.89.7.1093%26rft_id%253Dinfo%253Apmid%252F21665709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjk6Ijg5LzcvMTA5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.4" data-doi="10.1186/1471-2229-4-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakovlev</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Validating internal controls for quantitative plant gene expression studies.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DBrunner%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.spage%253D14%26rft.epage%253D14%26rft.atitle%253DValidating%2Binternal%2Bcontrols%2Bfor%2Bquantitative%2Bplant%2Bgene%2Bexpression%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-4-14%26rft_id%253Dinfo%253Apmid%252F15317655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-4-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15317655&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.5" data-doi="10.1373/clinchem.2008.112797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustin</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellemans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huggett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments.</span> <abbr class="cit-jnl-abbrev">Clin. Chem.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">611</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2008.112797%26rft_id%253Dinfo%253Apmid%252F19246619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6ODoiNTUvNC82MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Selection of reference genes for quantitative gene expression studies in <em>Platycladus orientalis</em> (Cupressaceae) using Real-Time PCR.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e33278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChang%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33278%26rft.epage%253De33278%26rft.atitle%253DSelection%2Bof%2Breference%2Bgenes%2Bfor%2Bquantitative%2Bgene%2Bexpression%2Bstudies%2Bin%2BPlatycladus%2Borientalis%2B%2528Cupressaceae%2529%2BUsing%2Breal-time%2BPCR.%26rft_id%253Dinfo%253Apmid%252F22479379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22479379&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.7" data-doi="10.1093/aob/mcn014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooley</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvallo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Is floral diversification associated with pollinator divergence? Flower shape, flower colour and pollinator preference in Chilean <em>Mimulus</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">101</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">641</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcn014%26rft_id%253Dinfo%253Apmid%252F18272528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjEwMS81LzY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.8" data-doi="10.1104/pp.105.063743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Czechowski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udvardi</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheible</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-wide identification and testing of superior reference genes for transcript normalization in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">139</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DCzechowski%26rft.auinit1%253DT.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D17%26rft.atitle%253DGenome-Wide%2BIdentification%2Band%2BTesting%2Bof%2BSuperior%2BReference%2BGenes%2Bfor%2BTranscript%2BNormalization%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.063743%26rft_id%253Dinfo%253Apmid%252F16166256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo3OiIxMzkvMS81IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.9" data-doi="10.1093/pcp/pcr187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Oliveira</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breton</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastolla</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camargo</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reference genes for the normalization of gene expression in <em>Eucalyptus</em> species.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">405</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcr187%26rft_id%253Dinfo%253Apmid%252F22197885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjg6IjUzLzIvNDA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dheda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huggett</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustin</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rook</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Validation of housekeeping genes for normalizing RNA expression in real-time PCR.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">112</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DDheda%26rft.auinit1%253DK.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D112%26rft.epage%253D112-4%252C%2B116%252C%2B119%26rft.atitle%253DValidation%2Bof%2Bhousekeeping%2Bgenes%2Bfor%2Bnormalizing%2BRNA%2Bexpression%2Bin%2Breal-time%2BPCR.%26rft_id%253Dinfo%253Apmid%252F15283208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15283208&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222669300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.11" data-doi="10.1016/j.ab.2005.05.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dheda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huggett</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">L. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustin</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The implications of using an inappropriate reference gene for real-time reverse transcription PCR data normalization.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">344</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">141</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DDheda%26rft.auinit1%253DK.%26rft.volume%253D344%26rft.issue%253D1%26rft.spage%253D141%26rft.epage%253D143%26rft.atitle%253DThe%2Bimplications%2Bof%2Busing%2Ban%2Binappropriate%2Breference%2Bgene%2Bfor%2Breal-time%2Breverse%2Btranscription%2BPCR%2Bdata%2Bnormalization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ab.2005.05.022%26rft_id%253Dinfo%253Apmid%252F16054107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ab.2005.05.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16054107&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231544500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-7.4.1085.12"><div class="cit-metadata unstructured">Edger, P. P., R. Smith, M. R. McKain, A. M. Cooley, M. Vallejo-Marin <em>et al.</em>, 2016 Subgenome dominance in an interspecific hybrid, synthetic allopolyploid, and a 140 year old naturally established neo-allopolyploid monkeyflower. bioRxiv. Available at: <a href="https://doi.org/10.1101/094797">https://doi.org/10.1101/094797</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.13" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.14" data-doi="10.1111/j.1467-7652.2008.00346.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauriat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guenin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelloux</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefebvre</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The lack of a systematic validation of reference genes: a serious pitfall undervalued in reverse transcription-polymerase chain reaction (RT-PCR) analysis in plants.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">609</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DGutierrez%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253D609%26rft.epage%253D618%26rft.atitle%253DThe%2Black%2Bof%2Ba%2Bsystematic%2Bvalidation%2Bof%2Breference%2Bgenes%253A%2Ba%2Bserious%2Bpitfall%2Bundervalued%2Bin%2Breverse%2Btranscription-polymerase%2Bchain%2Breaction%2B%2528RT-PCR%2529%2Banalysis%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2008.00346.x%26rft_id%253Dinfo%253Apmid%252F18433420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2008.00346.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18433420&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257571000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.15" data-doi="10.1038/nbt0510-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Advancing RNA-Seq analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">421</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D421%26rft.epage%253D423%26rft.atitle%253DAdvancing%2BRNA-Seq%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0510-421%26rft_id%253Dinfo%253Apmid%252F20458303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0510-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20458303&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.16" data-doi="10.1186/gb-2007-8-2-r19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellemans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortier</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Paepe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speleman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandesompele</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">qBase relative quantification framework and software for management and automated analysis of real-time quantitative PCR data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">R19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHellemans%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253DR19%26rft.epage%253DR19%26rft.atitle%253DqBase%2Brelative%2Bquantification%2Bframework%2Band%2Bsoftware%2Bfor%2Bmanagement%2Band%2Bautomated%2Banalysis%2Bof%2Breal-time%2Bquantitative%2BPCR%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-2-r19%26rft_id%253Dinfo%253Apmid%252F17291332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-2-r19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17291332&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.17" data-doi="10.1073/pnas.1319032110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellsten</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fine-scale variation in meiotic recombination in <em>Mimulus</em> inferred from population shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>(<span class="cit-issue">48</span>): <span class="cit-fpage">19478</span>–<span class="cit-lpage">19482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1319032110%26rft_id%253Dinfo%253Apmid%252F24225854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ4LzE5NDc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Libault</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibivilliers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilgin</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radwan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benitez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of four soybean reference genes for gene expression normalization.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">44</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D1%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.19" data-doi="10.1101/gr.079558.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mane</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilad</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1509</span>–<span class="cit-lpage">1517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.079558.108%26rft_id%253Dinfo%253Apmid%252F18550803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMTUwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.20" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vickery</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-article-title">Chromosome counts in the section <em>Simiolus</em> of the genus <em>Mimulus</em> (Scrophulariaceae). V. The chromosomal homologies of <em>M. guttatus</em> and its allied species and varieties.</span> <abbr class="cit-jnl-abbrev">Madrono</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMadrono%26rft.volume%253D16%26rft.spage%253D141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.22" data-doi="10.1126/science.1158441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>(<span class="cit-issue">5881</span>): <span class="cit-fpage">1344</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158441%26rft_id%253Dinfo%253Apmid%252F18451266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.23" data-doi="10.1093/nar/gks804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nookaew</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pornputtapong</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scalcinati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fagerberg</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comprehensive comparison of RNA-Seq-based transcriptome analysis from reads to differential gene expression and cross-comparison with microarrays: a case study in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">10084</span>–<span class="cit-lpage">10097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks804%26rft_id%253Dinfo%253Apmid%252F22965124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC8yMC8xMDA4NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.24" data-doi="10.1023/B:BILE.0000019559.84305.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfaffl</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tichopad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prgomet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuvians</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: BestKeeper – Excel-based tool using pair-wise correlations.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Lett.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">509</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Bletters%26rft.stitle%253DBiotechnol%2BLett%26rft.aulast%253DPfaffl%26rft.auinit1%253DM.%2BW.%26rft.volume%253D26%26rft.issue%253D6%26rft.spage%253D509%26rft.epage%253D515%26rft.atitle%253DDetermination%2Bof%2Bstable%2Bhousekeeping%2Bgenes%252C%2Bdifferentially%2Bregulated%2Btarget%2Bgenes%2Band%2Bsample%2Bintegrity%253A%2BBestKeeper--Excel-based%2Btool%2Busing%2Bpair-wise%2Bcorrelations.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253ABILE.0000019559.84305.47%26rft_id%253Dinfo%253Apmid%252F15127793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:BILE.0000019559.84305.47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15127793&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220217900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.25" data-doi="10.1038/nmeth.1517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>De novo</em> assembly and analysis of RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">909</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DRobertson%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253D909%26rft.epage%253D912%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Band%2Banalysis%2Bof%2BRNA-seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1517%26rft_id%253Dinfo%253Apmid%252F20935650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20935650&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283586600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.26" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzEzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.27" data-doi="10.1111/j.1469-8137.2011.03656.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scoville</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnett</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodbyl-Roels</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hileman</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Differential regulation of a MYB transcription factor is correlated with transgenerational epigenetic inheritance of trichome density in <em>Mimulus guttatus</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">191</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">251</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DScoville%26rft.auinit1%253DA.%2BG.%26rft.volume%253D191%26rft.issue%253D1%26rft.spage%253D251%26rft.epage%253D263%26rft.atitle%253DDifferential%2Bregulation%2Bof%2Ba%2BMYB%2Btranscription%2Bfactor%2Bis%2Bcorrelated%2Bwith%2Btransgenerational%2Bepigenetic%2Binheritance%2Bof%2Btrichome%2Bdensity%2Bin%2BMimulus%2Bguttatus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03656.x%26rft_id%253Dinfo%253Apmid%252F21352232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03656.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21352232&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291229500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D19%26rft.spage%253D3210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.29" data-doi="10.3389/fpls.2013.00366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streisfeld</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Flower color as a model system for studies of plant evo-devo.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">321</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D4%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2013.00366%26rft_id%253Dinfo%253Apmid%252F23346092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2013.00366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23346092&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Control selection for RNA quantitation.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">332</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DSuzuki%26rft.auinit1%253DT.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D332%26rft.epage%253D337%26rft.atitle%253DControl%2Bselection%2Bfor%2BRNA%2Bquantitation.%26rft_id%253Dinfo%253Apmid%252F10948434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10948434&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088679400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.31" data-doi="10.1016/S0168-1656(99)00163-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thellin</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorzi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakaye</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Borman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coumans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Housekeeping genes as internal standards: use and limits.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">75</span>(<span class="cit-issue">2–3</span>): <span class="cit-fpage">291</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiotechnology%26rft.stitle%253DJournal%2Bof%2BBiotechnology%26rft.aulast%253DThellin%26rft.auinit1%253DO.%26rft.volume%253D75%26rft.issue%253D2-3%26rft.spage%253D291%26rft.epage%253D295%26rft.atitle%253DHousekeeping%2Bgenes%2Bas%2Binternal%2Bstandards%253A%2Buse%2Band%2Blimits.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-1656%252899%252900163-7%26rft_id%253Dinfo%253Apmid%252F10617337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-1656(99)00163-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10617337&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083683900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.32" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.33" data-doi="10.1111/mec.13190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Twyford</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streisfeld</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic studies on the nature of species: adaptation and speciation in <em>Mimulus</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2601</span>–<span class="cit-lpage">2609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D2601%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.34" data-doi="10.1111/evo.12678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo‐Marín</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buggs</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooley</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puzey</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Speciation by genome duplication: repeated origins and genomic composition of the recently formed allopolyploid species <em>Mimulus peregrinus</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1487</span>–<span class="cit-lpage">1500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D1487%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12678%26rft_id%253Dinfo%253Apmid%252F25929999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25929999&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.35" data-doi="10.1186/gb-2002-3-7-research0034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vandesompele</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Preter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattyn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poppe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Roy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">RESEARCH0034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DVandesompele%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253DRESEARCH0034%26rft.epage%253DRESEARCH0034%26rft.atitle%253DAccurate%2Bnormalization%2Bof%2Breal-time%2Bquantitative%2BRT-PCR%2Bdata%2Bby%2Bgeometric%2Baveraging%2Bof%2Bmultiple%2Binternal%2Bcontrol%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2002-3-7-research0034%26rft_id%253Dinfo%253Apmid%252F12184808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2002-3-7-research0034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12184808&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.36" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.37" data-doi="10.2307/2640431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The role of genes of large effect on inbreeding depression in <em>Mimulus guttatus</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1678</span>–<span class="cit-lpage">1691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D1678%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084703100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooley</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Mimulus</em> is an emerging model system for the integration of ecological and genomic studies.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">220</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D100%26rft.spage%253D220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.39" data-doi="10.1016/j.gene.2013.12.057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selection and evaluation of novel reference genes for quantitative reverse transcription PCR (qRT-PCR) based on genome and transcriptome data in <em>Brassica napus</em> L.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">538</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">113</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D538%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.12.057%26rft_id%253Dinfo%253Apmid%252F24406618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.12.057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24406618&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.40" data-doi="10.1534/genetics.112.146852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagawa</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic dissection of a major anthocyanin QTL contributing to pollinator-mediated reproductive isolation between sister species of <em>Mimulus</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146852%26rft_id%253Dinfo%253Apmid%252F23335333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzEvMjU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1085.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of appropriate reference genes for quantitative real-time PCR in <em>Oxytropis ochrocephala</em> Bunge using transcriptome datasets under abiotic stress treatments.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/4/1085.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1061" title="Identification of Genes in Saccharomyces cerevisiae that Are Haploinsufficient for Overcoming Amino Acid Starvation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1097" title="Triploid Production from Interspecific Crosses of Two Diploid Perennial Helianthus with Diploid Cultivated Sunflower (Helianthus annuus L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/4"><span>Volume 7 Issue 4, April 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/4.cover-source.jpg?itok=KOTpzmYp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1085.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VsyXFIibNs3CdcJcj4oGp4FbbAhUEWjGHRsaG6pEa5I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/4/1085" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/4/1085" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/4/1085" class="active" data-icon-position="" data-hide-link-title="0">A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in Mimulus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483842" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--23TYmcE3TV6NvVsJ6TXTT3xw0UrVXAtQSjDPE_e8BU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1085.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/4/1085" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-V_bJPuyvUtURNw6HmfFOMW3u_yYBqVI5SOHHxTS1-EE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/4/1085" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1085.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483842" id="node483842--2" data-pisa="ggg;7/4/1085" data-pisa-master="ggg;g3.116.038075" data-apath="/ggg/7/4/1085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in <em>Mimulus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kimmy A. Stanton</span>, <span class="highwire-citation-author" data-delta="1">Patrick P. Edger</span>, <span class="highwire-citation-author" data-delta="2">Joshua R. Puzey</span>, <span class="highwire-citation-author" data-delta="3">Taliesin Kinser</span>, <span class="highwire-citation-author" data-delta="4">Philip Cheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4182-3105" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Vernon</span>, <span class="highwire-citation-author" data-delta="6">Nancy R. Forsthoefel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3730-0288" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arielle M. Cooley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1085-1095;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038075 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483842--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kimmy A. Stanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kimmy%2BA.%2BStanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stanton%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKimmy%2BA.%2BStanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick P. Edger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BP.%2BEdger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edger%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BP.%2BEdger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua R. Puzey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>College of William and Mary, Williamsburg, Virginia 23185</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BR.%2BPuzey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Puzey%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BR.%2BPuzey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Taliesin Kinser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>College of William and Mary, Williamsburg, Virginia 23185</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taliesin%2BKinser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kinser%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaliesin%2BKinser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Philip Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Daniel M. Vernon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BVernon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vernon%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BVernon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4182-3105" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Vernon</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nancy R. Forsthoefel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BR.%2BForsthoefel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forsthoefel%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANancy%2BR.%2BForsthoefel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Arielle M. Cooley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arielle%2BM.%2BCooley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArielle%2BM.%2BCooley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3730-0288" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arielle M. Cooley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cooleya@whitman.edu" class="" data-icon-position="" data-hide-link-title="0">cooleya@whitman.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/4/1085" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/4/1085&amp;title=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/4/1085&amp;title=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/4/1085&amp;title=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/4/1085&amp;text=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/4/1085&amp;title=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/4/1085&amp;t=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/4/1085&amp;title=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/4/1085&amp;title=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483842/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483842" id="node483842--3" data-pisa="ggg;7/4/1085" data-pisa-master="ggg;g3.116.038075" data-apath="/ggg/7/4/1085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Transcriptome Approach to Evaluating Reference Genes for Quantitative Gene Expression Studies: A Case Study in <em>Mimulus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kimmy A. Stanton</span>, <span class="highwire-citation-author" data-delta="1">Patrick P. Edger</span>, <span class="highwire-citation-author" data-delta="2">Joshua R. Puzey</span>, <span class="highwire-citation-author" data-delta="3">Taliesin Kinser</span>, <span class="highwire-citation-author" data-delta="4">Philip Cheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4182-3105" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Vernon</span>, <span class="highwire-citation-author" data-delta="6">Nancy R. Forsthoefel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3730-0288" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arielle M. Cooley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1085-1095;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038075 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483842--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kimmy A. Stanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kimmy%2BA.%2BStanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stanton%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKimmy%2BA.%2BStanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick P. Edger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BP.%2BEdger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edger%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BP.%2BEdger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua R. Puzey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>College of William and Mary, Williamsburg, Virginia 23185</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BR.%2BPuzey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Puzey%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BR.%2BPuzey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Taliesin Kinser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>College of William and Mary, Williamsburg, Virginia 23185</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taliesin%2BKinser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kinser%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaliesin%2BKinser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Philip Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Daniel M. Vernon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BVernon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vernon%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BVernon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4182-3105" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Vernon</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nancy R. Forsthoefel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BR.%2BForsthoefel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forsthoefel%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANancy%2BR.%2BForsthoefel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Arielle M. Cooley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Whitman College, Walla Walla, Washington 99362</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arielle%2BM.%2BCooley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArielle%2BM.%2BCooley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3730-0288" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arielle M. Cooley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cooleya@whitman.edu" class="" data-icon-position="" data-hide-link-title="0">cooleya@whitman.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1085&amp;rft.epage=1095&amp;rft.atitle=A%20Whole-Transcriptome%20Approach%20to%20Evaluating%20Reference%20Genes%20for%20Quantitative%20Gene%20Expression%20Studies%3A%20A%20Case%20Study%20in%20Mimulus&amp;rft.volume=7&amp;rft.issue=4&amp;rft.date=2017-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Stanton%2C+Kimmy+A.&amp;rft.au=Edger%2C+Patrick+P.&amp;rft.au=Puzey%2C+Joshua+R.&amp;rft.au=Kinser%2C+Taliesin&amp;rft.au=Cheng%2C+Philip&amp;rft.au=Vernon%2C+Daniel+M.&amp;rft.au=Forsthoefel%2C+Nancy+R.&amp;rft.au=Cooley%2C+Arielle+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483842/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483842/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483842/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483842/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483842/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483842/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483842/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483842/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483842/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483842/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483842/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483842/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/4/1085" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/4/1085.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/4/1085.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/4/1085.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/4/1085.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
