<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/4/1277" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/4/1277.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/4/1277.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/4/1277.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/4/1277/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/4/1277.atom" />
<meta name="HW.pisa" content="ggg;7/4/1277" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Automated Phenotyping Indicates Pupal Size in Drosophila Is a Highly Heritable Trait with an Apparent Polygenic Basis" />
<meta name="DC.Identifier" content="10.1534/g3.117.039883" />
<meta name="DC.Date" content="2017-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Reeves and Tautz. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The intense focus on studying human height has done more than any other genetic analysis to advance our understanding of the heritability of highly complex phenotypes. Here, we describe in detail the properties of a previously unexplored trait in Drosophila melanogaster that shares many salient properties with human height. The total length of the pupal case varies between 2.8 and 3.9 mm among natural variants, and we report that it is among the most heritable traits reported in this species. We have developed a simple semiautomatic phenotyping system with which a single operator can reliably score &gt;5000 individuals in a day. The precision of the automated system is 0.042 mm (± 0.030 SD). All phenotyped individuals are available to be mated in subsequent generations or uniquely archived for future molecular work. We report both broad sense and narrow sense heritability estimates for two biologically distinct data sets. Narrow sense heritability ( h 2) ranged from 0.44 to 0.50, and broad sense heritability ( H 2) ranged from 0.58 to 0.61. We present results for mapping the trait in 195 recombinant inbred lines, which suggests that there are no loci with &gt;10% effect size in this panel. We propose that pupal size genetics in Drosophila could represent a model complex trait amenable to deep genetic dissection using the automated system described." />
<meta name="DC.Contributor" content="R. Guy Reeves" />
<meta name="DC.Contributor" content="Diethard Tautz" />
<meta name="article:published_time" content="2017-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Automated Phenotyping Indicates Pupal Size in Drosophila Is a Highly Heritable Trait with an Apparent Polygenic Basis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The intense focus on studying human height has done more than any other genetic analysis to advance our understanding of the heritability of highly complex phenotypes. Here, we describe in detail the properties of a previously unexplored trait in &lt;i&gt;Drosophila melanogaster&lt;/i&gt; that shares many salient properties with human height. The total length of the pupal case varies between 2.8 and 3.9&amp;nbsp;mm among natural variants, and we report that it is among the most heritable traits reported in this species. We have developed a simple semiautomatic phenotyping system with which a single operator can reliably score &amp;gt;5000 individuals in a day. The precision of the automated system is 0.042&amp;nbsp;mm (±&amp;nbsp;0.030&amp;nbsp;SD). All phenotyped individuals are available to be mated in subsequent generations or uniquely archived for future molecular work. We report both broad sense and narrow sense heritability estimates for two biologically distinct data sets. Narrow sense heritability (&lt;i&gt;h&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt;) ranged from 0.44 to 0.50, and broad sense heritability (&lt;i&gt;H&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt;) ranged from 0.58 to 0.61. We present results for mapping the trait in 195 recombinant inbred lines, which suggests that there are no loci with &amp;gt;10% effect size in this panel. We propose that pupal size genetics in &lt;i&gt;Drosophila&lt;/i&gt; could represent a model complex trait amenable to deep genetic dissection using the automated system described.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/04/01" />
<meta name="citation_mjid" content="ggg;7/4/1277" />
<meta name="citation_id" content="7/4/1277" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/4/1277" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/4/1277.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/4/1277.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/4/1277.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.039883" />
<meta name="citation_pmid" content="28258111" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1277" />
<meta name="citation_lastpage" content="1286" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="R. Guy Reeves" />
<meta name="citation_author_institution" content="Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany" />
<meta name="citation_author_email" content="reeves@evolbio.mpg.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9454-6175" />
<meta name="citation_author" content="Diethard Tautz" />
<meta name="citation_author_institution" content="Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0460-5344" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=Y. S. Aulchenko;citation_author=M. V. Struchalin;citation_author=N. M. Belonogova;citation_author=T. I. Axenovich;citation_author=M. N. Weedon;citation_title=Predicting human height by Victorian and genomic methods.;citation_pages=1070-1075;citation_volume=17;citation_year=2009;citation_issue=8;citation_pmid=19223933;citation_doi=10.1038/ejhg.2009.5" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Ayroles;citation_author=S. M. Buchanan;citation_author=C. O’Leary;citation_author=K. Skutt-Kakaria;citation_author=J. K. Grenier;citation_title=Behavioral idiosyncrasy reveals genetic control of phenotypic variability.;citation_volume=112;citation_year=2015;citation_pmid=25953335;citation_doi=10.1073/pnas.1503830112" />
<meta name="citation_reference" content="citation_journal_title=Journal of Insect Physiology;citation_journal_abbrev=Journal of Insect Physiology;citation_author=A. B. Barron;citation_title=Anaesthetising Drosophila for behavioural studies.;citation_pages=439-442;citation_volume=46;citation_year=2000;citation_issue=4;citation_pmid=12770207;citation_doi=10.1016/S0022-1910(99)00129-8" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=The infinitesimal model.;citation_author=N. H. Barton;citation_author=A. M. Etheridge;citation_author=A. Véber;citation_pages=039768;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=Y. F. Chan;citation_author=F. C. Jones;citation_author=E. McConnell;citation_author=J. Bryk;citation_author=L. Bünger;citation_author=D. Tautz;citation_title=Parallel selection mapping using artificially selected mice reveals body weight control loci.;citation_pages=794-800;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22445301;citation_doi=10.1016/j.cub.2012.03.011" />
<meta name="citation_reference" content="citation_journal_title=J. Anthropol. Inst. G. B. Irel.;citation_author=F. Galton;citation_title=Regression towards mediocrity in hereditary stature.;citation_pages=246-263;citation_volume=15;citation_year=1886" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_title=One hundred years of statistical developments in animal breeding.;citation_pages=19-56;citation_volume=3;citation_year=2015;citation_pmid=25387231;citation_doi=10.1146/annurev-animal-022114-110733" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=J. A. Hanley;citation_title=“Transmuting” women into men: Galton’s family data on human stature.;citation_pages=237-243;citation_volume=58;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. Houle;citation_author=J. Mezey;citation_author=P. Galpern;citation_author=A. Carter;citation_title=Automated measurement of Drosophila wings.;citation_pages=25-25;citation_volume=3;citation_year=2003;citation_pmid=14670094;citation_doi=10.1186/1471-2148-3-25" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Ramia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila synthetic population resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. G. King;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.;citation_volume=191;citation_year=2012;citation_pmid=22505626;citation_doi=10.1534/genetics.112.138537" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Kofler;citation_author=C. Schlötterer;citation_title=A guide for the design of evolve and resequencing studies.;citation_volume=31;citation_year=2014;citation_pmid=24214537;citation_doi=10.1093/molbev/mst221" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=M. R. Lamprecht;citation_author=D. M. Sabatini;citation_author=A. E. Carpenter;citation_title=CellProfiler: free, versatile software for automated biological image analysis.;citation_pages=71-75;citation_volume=42;citation_year=2007;citation_issue=1;citation_pmid=17269487;citation_doi=10.2144/000112257" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and quantitative traits: using model organisms to study gene–gene interactions.;citation_pages=22-33;citation_volume=15;citation_year=2013;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=V. Medici;citation_author=S. C. Vonesch;citation_author=S. N. Fry;citation_author=E. Hafen;citation_title=The FlyCatwalk : a high-throughput feature-based sorting system for artificial selection in Drosophila.;citation_pages=317-327;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_title=A century of trends in adult human height.;citation_pages=e13410;citation_volume=5;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=C. P. Nelson;citation_author=S. E. Hamby;citation_author=D. Saleheen;citation_author=J. C. Hopewell;citation_author=L. Zeng;citation_title=Genetically determined height and coronary artery disease.;citation_pages=1608-1618;citation_volume=372;citation_year=2015;citation_pmid=25853659;citation_doi=10.1056/NEJMoa1404881" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=S. V. Nuzhdin;citation_author=T. L. Turner;citation_title=Promises and limitations of hitchhiking mapping.;citation_pages=694-699;citation_volume=23;citation_year=2013;citation_pmid=24239053;citation_doi=10.1016/j.gde.2013.10.002" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=T. J. C. Polderman;citation_author=B. Benyamin;citation_author=C. A. de Leeuw;citation_author=P. F. Sullivan;citation_author=A. van Bochoven;citation_title=Meta-analysis of the heritability of human traits based on fifty years of twin studies.;citation_pages=702-709;citation_volume=47;citation_year=2015;citation_pmid=25985137;citation_doi=10.1038/ng.3285" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=K. Rawlik;citation_author=O. Canela-Xandri;citation_author=A. Tenesa;citation_title=Evidence for sex-specific genetic architectures across a spectrum of human complex traits.;citation_pages=166;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. A. Roff;citation_author=T. A. Mousseau;citation_title=Quantitative genetics and fitness lessons from Drosophila.;citation_pages=103-118;citation_volume=58;citation_year=1987;citation_pmid=3818341;citation_doi=10.1038/hdy.1987.15" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. L. Turner;citation_author=A. D. Stewart;citation_author=A. T. Fields;citation_author=W. R. Rice;citation_author=A. M. Tarone;citation_title=Population-based resequencing of experimentally evolved populations reveals the genetic basis of body size variation in Drosophila melanogaster.;citation_pages=e1001336-e1001336;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437274;citation_doi=10.1371/journal.pgen.1001336" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=A. A. Vinkhuyzen;citation_author=N. R. Wray;citation_author=J. Yang;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Estimation and partition of heritability in human populations using whole-genome analysis methods.;citation_pages=75-95;citation_volume=47;citation_year=2013;citation_pmid=23988118;citation_doi=10.1146/annurev-genet-111212-133258" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=P. M. Visscher;citation_author=B. McEvoy;citation_author=J. Yang;citation_title=From Galton to GWAS: quantitative genetics of human height.;citation_pages=371-379;citation_volume=92;citation_year=2010;citation_issue=5-6;citation_pmid=21429269;citation_doi=10.1017/S0016672310000571" />
<meta name="citation_reference" content="citation_journal_title=Archives of Disease in Childhood;citation_journal_abbrev=Arch. Dis. Child.;citation_author=L. D. Voss;citation_author=B. J. Bailey;citation_author=K. Cumming;citation_author=T. J. Wilkin;citation_author=P. R. Betts;citation_title=The reliability of height measurement (the Wessex Growth Study).;citation_pages=1340-1344;citation_volume=65;citation_year=1990;citation_issue=12;citation_pmid=2270942;citation_doi=10.1136/adc.65.12.1340" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Wählby;citation_author=L. Kamentsky;citation_author=Z. H. Liu;citation_author=T. Riklin-Raviv;citation_author=A. L. Conery;citation_title=An image analysis toolbox for high-throughput C. elegans assays.;citation_pages=714-716;citation_volume=9;citation_year=2012;citation_issue=7;citation_pmid=22522656;citation_doi=10.1038/nmeth.1984" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. N. Weedon;citation_author=H. Lango;citation_author=C. M. Lindgren;citation_author=C. Wallace;citation_author=D. M. Evans;citation_title=Genome-wide association analysis identifies 20 loci that influence adult height.;citation_pages=575-583;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391952;citation_doi=10.1038/ng.121" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. R. Wood;citation_author=T. Esko;citation_author=J. Yang;citation_author=S. Vedantam;citation_author=T. H. Pers;citation_title=Defining the role of common variation in the genomic and biological architecture of adult human height.;citation_pages=1173-1186;citation_volume=46;citation_year=2014;citation_pmid=25282103;citation_doi=10.1038/ng.3097" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Automated Phenotyping Indicates Pupal Size in Drosophila Is a Highly Heritable Trait with an Apparent Polygenic Basis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
<meta name="twitter:description" content="The intense focus on studying human height has done more than any other genetic analysis to advance our understanding of the heritability of highly complex phenotypes. Here, we describe in detail the properties of a previously unexplored trait in Drosophila melanogaster that shares many salient properties with human height. The total length of the pupal case varies between 2.8 and 3.9 mm among natural variants, and we report that it is among the most heritable traits reported in this species. We have developed a simple semiautomatic phenotyping system with which a single operator can reliably score &gt;5000 individuals in a day. The precision of the automated system is 0.042 mm (± 0.030 SD). All phenotyped individuals are available to be mated in subsequent generations or uniquely archived for future molecular work. We report both broad sense and narrow sense heritability estimates for two biologically distinct data sets. Narrow sense heritability ( h 2) ranged from 0.44 to 0.50, and broad sense heritability ( H 2) ranged from 0.58 to 0.61. We present results for mapping the trait in 195 recombinant inbred lines, which suggests that there are no loci with &gt;10% effect size in this panel. We propose that pupal size genetics in Drosophila could represent a model complex trait amenable to deep genetic dissection using the automated system described." />
<meta name="og-title" property="og:title" content="Automated Phenotyping Indicates Pupal Size in Drosophila Is a Highly Heritable Trait with an Apparent Polygenic Basis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/4/1277" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The intense focus on studying human height has done more than any other genetic analysis to advance our understanding of the heritability of highly complex phenotypes. Here, we describe in detail the properties of a previously unexplored trait in Drosophila melanogaster that shares many salient properties with human height. The total length of the pupal case varies between 2.8 and 3.9 mm among natural variants, and we report that it is among the most heritable traits reported in this species. We have developed a simple semiautomatic phenotyping system with which a single operator can reliably score &gt;5000 individuals in a day. The precision of the automated system is 0.042 mm (± 0.030 SD). All phenotyped individuals are available to be mated in subsequent generations or uniquely archived for future molecular work. We report both broad sense and narrow sense heritability estimates for two biologically distinct data sets. Narrow sense heritability ( h 2) ranged from 0.44 to 0.50, and broad sense heritability ( H 2) ranged from 0.58 to 0.61. We present results for mapping the trait in 195 recombinant inbred lines, which suggests that there are no loci with &gt;10% effect size in this panel. We propose that pupal size genetics in Drosophila could represent a model complex trait amenable to deep genetic dissection using the automated system described." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4.cover-source.jpg" />
    <title>Automated Phenotyping Indicates Pupal Size in Drosophila Is a Highly Heritable Trait with an Apparent Polygenic Basis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3U7_2ZzSkGvv-XV8yAN-mjQwQBQH5UPoDJFMyQzm668","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483838","apath":"\/ggg\/7\/4\/1277.atom","pisa":"ggg;7\/4\/1277","ac":{"\/ggg\/7\/4\/1277.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/4\/1277.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1277"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/4\/1277"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/4\/1277.abstract":true,"\/content\/7\/4\/1277":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zE2CFvTGEO+KSb7+KIHaamBSURCAFi\\\\\\\/pA7fg1OqVhcUWBjHOPjF4nc74w\\\\\\\/OUi1gE0mfGUToxxpraN61igDYxX86eUSTh785a2WZWjv9J7u4JtnzXeoPtkKyqpA3KRoRisV9apZcCGDpoQ717Cbyp3Ktr1HoMuCNZDVeSS\\\\\\\/37Ab1+ugqC28CXCBJMe1tCLCCI973+CBigQ9QiKMmY27tSoOj00oMtNJESD2a3g+CMjqv57MiyxGFAr+qO7plYyb30vBtjUTKofHVWDVC3V+nv2\\\\\\\/c8RhvrqFVw6+DtbEEu3WdIciH5TwMvUYOGiRWdqL0bFB55JZF87ZF4p0085EEkDYj2OWI7c7ZoGumZnvpbMSXJQvQS8LMwvIOm5puNVE0T4ni31eXcA\\\\\\\/dnXYV9+xVBJKSgfDuYF375LHkfsdpqiCdKfeWfzNm1D9cSPkkSxLqg+2XPrNgjTzRh7foQQSggc2GNrLpJfeT0AwfFh4IfUw+4S5jh00ESgJCljWi0f18ZMXMqjnHTN\\\\\\\/KDP1rG2rBNE5uud9UQ5MTu2+TbG3HhDt0LpmamPiEy0BJc5+qKmaQPw8bAVI7NTAE7JZrDtkyjUdTFeLusIrz4ympMwcI8vnmBQybn3E39H566a6s8BiSm0w32UpruQvN8NxfQcP1\\\\\\\/QEM+Rws+atfuj54ngsOBFJF57vHmqE0WzgGiKNMbj+rHmowGeG8IFV9l\\\\\\\/RFNMiGvz9NtQLj0GZcpVtHEW24QZK7\\\\\\\/Vw9\\\\\\\/QSHZ0BNVoMI5Vxouea5r6D1u0O4H0SXdbRG05Ot6HgU2yZEV9MloRSLHJtJ6I+MxGzNuZ\\\\\\\/b2P3vm4KyBDMBusr8+aMXNaQocUVxzuFzQQVJsrbqQ7MWXkDHcPNwjeu6B9uIJUR0InL4858WtXorYV1bedZzkI2MJiGHGyMVFD6Nh9fWx88vjXOjPp9flnTXeaL2IvHLi3lGuOf6ZzYnJpn3EF83O7yg\\\\\\\/Fz+fvRs7uwqPc0jQXC+kpjPlNHdSfamII3kBObbi5tYZbE0qFQzdtmVSdeS74QyMeGffU26YEDmLrquqvNdQr+P5uuJ7scI\\\\\\\/Ffp3qVuWy9f88tB+tIsM5AJAeokYPFo4ZgSmQ8TpUMVvTRyP\\\\\\\/g2xsChp3njqu5FzQV\\\\\\\/axW2aqAaXwjDk4iTRipPof9x5onGxNMihkTVGI1lpsrhcOefn5CsmMz6ASEb3lVHe6c0kqzpuqo36BBzNbUF9bTlAtHF7GIB9M1jXMixjMjOFU5OYfxjKO5L38XKSSQuTOsTd\\\\\\\/JfQqQnh\\\u0022,\\\u0022iv\\\u0022:\\\u0022bgnEEGX+70zTWUJNIlf5pA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d5243645cf3fa8292d07503dced724ace8f075ee4b289f4df70dd00922623b13\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+\\\\\\\/t9q0E8BbORtLcgT8r+w23WLIf6v9TPF9z3lvCcKosjp2objc6Bczm1ezkVMrGZ9NzmUh6zIFoO5Amkusq1r1o8JFsRsHAMuE30F\\\\\\\/Qw\\\\\\\/8iyuKFRktI8U2aS65xQQTDD5bv\\\\\\\/riaagpJsjTpidE4cAJWnjOp7\\\\\\\/JAtVc1IOk5Qsv0Bu0v4a8N+JCrqg04ZBF6dz7fQ0cz8u2exC+UBvMusuJrh8lVtux9YwURFwLzE0kWv1qWRJNna2TfAEfN985kyAdsA8k8PDiP14KlxpAH8iUzAQYzGlHc9cSiZQj2DRGdPEGp4vTKR2jFQYR30cOfePefqP9GiI9B44s6SNGZ8w5wS6GkFP3FB+SEMSxHFxv\\\\\\\/l5ACJXF8FshYO4cLzdRyUNiCdZK7w2IywafFE81fuQ9a1tqT3GNfj+uJ7hLxFCVLRsfC\\\\\\\/8dzU0656cn3Igf9Q4iPmTuRB\\\\\\\/\\\\\\\/635+jAWoT6oG0SVarrBsgSDxHvCku3y6ZyxZEIKSx1blR3jprhH4KpXUpeQ\\\\\\\/wQLuemxZCb8y1L6LUyW7bjU7hzqNU9xl3rVxHMR5zkQuzV43QktK19YtI1gXR3hVjP2zOKxJo7DNYXuXPX8HjJOA7RA4tDYdS96xysXXnp88aacxqcuQWwUlD72Y1vyk+0oom7UbNrIHIGbD5RhFs6aAQBVm2WhcCaC0qnjqWTj2HjUPtiaw6Ri+7zdffdhE4tHNl259Lr5oE9DVQOY\\\\\\\/5RWiK2S4gQ7ocpQbOQ9oLB\\\\\\\/Q947AJ5ZnoD\\\\\\\/j0BUniUYz8ZxL02MEPt7iwUTGouivXUcMgYfGXo21kmh2\\\\\\\/fZtx80sbM5S7FtLbOWMv9Z6QBlr0dMthW2RpAzs0y44jJPCj6pyrS4vgdizdmMSj5YMKd\\\\\\\/3NjkV2kOkdykQD\\\\\\\/Mt1BFcb3IlKCXR9RDwnh\\\\\\\/juZE06vhtAGIPV0a7FNxaZLlCGArKmSPBomuWfmGIgTofE6Q0S2N+fecIzViTiVaWZHAnMtWCv0UYXK3eBbDLJCZYS8JQH4LOgj8L4cvffgiPJlnFhI+7NN4ecVNbAxRH+lndTe6vJy\\\\\\\/j9+Qb75yMsnC4zDpVuc3pdN2GDbLY6jVBb2nJJAbUTZ72oQqcxEBqeKlrB7Fe5q67gse2i\\\\\\\/KdKyc1zZdpz8SCZ7acvx\\\\\\\/ozryxxIqt8yPP\\\\\\\/T68vofiJ2NTIjfzqWNsPpV\\\\\\\/krdb\\\\\\\/LJpJADu8w0jwBuYs9f1XLFUICo\\\\\\\/fapJPbGcUdIjJJ4Ux91T1i0FhXuW+HbnUO8p1JBcN03vUu9cGuZlu2fPD4e8JCmqyCJFBJXgSgCZWSEhw9OuiANOCU++72G+VRTVPK1iZu4RJV1in5Ej0tqoUfy3ep2cLp495pNptI0gXvZ1PANKCwCfGWiEncyH8Xq23wkmY0QYzYAHkGKPqiormC96apXecGB6X5iF8mpCP\\\\\\\/4MJFmrqQ4lf1r173R5pue+cy7kddIpAMTLnM9oBmrnWRxVRjcT1eHTXWKzFM8KYe8Ag\\\\\\\/MM+TtSwYD7jd2TpX5+Mr5MXiNIZhaJVhBkAdWK\\\\\\\/Y+\\\\\\\/U275R6fBhYw7vOTEzizpKi78ptUOjCWiNWJbAjm8rTwPUZMLtPB21XAjJsl82S+a8WmDjS+VpVfy59jOyRN+CKQa294BwR+Qt9\\\\\\\/of0WT1ytHzKsNEyHWaEeDDWZfWY3Hgcp+oQnRRlKt+DLFyydj4Jo4pWAELcknC1wJppGFnx8828IlmzMBY0dFSGWOWrXFBHZchj0HW8OhcCs2oDuzvJdcPFuZa+OOlrul4LYBPC0gheWvgIRyZgNe472U7OB9V0sCb0giU\\\\\\\/0oZXBpJiPv39CeOczSrd5d6GVhyLNXS23CygpEV2qhppgjio0W3Rjn3p91zPJ5HwNYohGOBSzQmE07Pm1eZnNBPGADQWpW8Ao3WwnhmL63ZNwrn3G+P8kfwBkOa3BQozsECEo942Zp7H9qAHbRXYNXu6\\\\\\\/de6436Jfc16XwGp4HZqRgT5j4OY9sfCy2BJOANHbUgELe8vuUAqP65ajjLyMCtrth5wqV6Du81kWK5k8VOCXefbJ4wOdY\\\\\\\/MQ\\\\\\\/Te5MMwoXLHEfZvCj6rLTDffYW9PLy0kHy4zskbN4PJqzkgpLYMbCfn11I6ywyoJKH3KSpgfVOHbU9IDW18RM4w8pSIHDXOSVvhxvh9DMyjaYU3fjv7IXbAJFaK7oNK6sd+oI9eSsvrCbzB55Rv4COYKZ5TSw7HIq9OjTcOe47qgw8w2OPGCtb\\\\\\\/vl93lkHClGmLvUI01GlfCQlX39Cau4bILVgo3jJ6WzfquGoFLVya8NLis+zWLmjURF3zl9wSw0JLOp7UofyFqkjxB30KYaIb9nmt80ELpU2TLHxFnf+wRGxuvu7zi6S8C8G1oF1boGNh39wNISBCOAH2PP3vDW50DkXbFO53vUSknl6KH9g83bPnkzExEVwd\\\\\\\/N7MRBtFJeEgGtsG8JSQwUpDLem76Hei9ybApzedB+lDyrEzAKMOCvhOGa1iFHCrw4RU2zDnBVI06VC68o69mODzayNrwG+mPspsBXqThCigss2bfBDXeyKt5mlYZN3VViUUu4beaANXI4FqLDd5pzAYvSSMQpY9cBPaq6AsHzglwfP\\\\\\\/NYrOYF54LAny1cEYqXLuzH3SF39\\\\\\\/XgtljBj5zSu5JapBm\\\\\\\/W+EzPH8LXhMiu0GZZC8myULwMj18Sq0gyjCYK1jGhR+\\\\\\\/tRVOnvfEtsE6wpQvTSlw5Ov5nEZ9SbvWDOFs0OdP8Sn\\\\\\\/Idgd4KsBmwrdFwjoBkAE4qI5Qh2AYuJUhvgngUPTQYSusL1CznFZctC3+QZJh+g81bQ8zC3OlAAPLT2u+fktCIUS4OVGVPf1HF741xLrdbNtOVamOnHRdSpL+aNoPW9MHyDExMtKivfIchrryiA3WMffi+xdmOz7GeKA2yBD2ybdHmS3LIlgOpltO8K1VB9fn9cIO8LTC7RYdkGrwWvyh9Xf3oauLhM0TZ2jr1IfSbrrW0bMhHyCMZ7H6xi4UAEtyNPUD0NgnWoj9vp5Q63bGW0ABk2BLnZsdSQqts7E6erocTGFAd0X9loDbakclMzNgzQgzvq5nafDDasHgVNzw1fheUkNK1o5OPFXmR3UAXVP\\\\\\\/JQxpfr9f3FvkJ1RFsnhAZSdkMXj3FEJZcS\\\\\\\/G9OFwGnfY0QnxtLN2lLW2FqC6sqCN1li5SQTSd8KWNwZl89sQsymIgJ6iakZgkFlHjb+SNMEV56cIs61yXCt5KAiEt3HNTjCpZAeEmGsCPMmrm7FIJycgqxCfBw\\\\\\\/W3iX1aoozZTzaWBzdhDRkGufteVWbVUb69FfvsQg1xKUAizIOtHekFB+BqvF8Ci2lpa8Yc8QVkFXMPOHxNkXkLQO0LXWoyK1ENTEdEGefbrhI6bx1IgqgSGOqXUaPUsSSwGi2NI2WvWTqwDiOulJV2q\\\\\\\/V3xHiFmUZnkyH8m9Vp2axNZB03WhUqD7jc6fsmq2ah99I6A8FyRdcI6mh7wpguxyM4n7EOw3fxu5OE7m9MAMxhpHw+yuhhx89YwkaChQrSWft4Vn6DTcYa1d40Swc8vkKeJFlg1HwN\\\\\\\/66D2JjYBxKXmTeWwBSGQvOeM157Zw\\\\\\\/moZu6UQ8jfCiGNQBMItcfh0vsWbryKm4tIYfENtKmOlDohmXZOL1\\\\\\\/8nkReoq0h+2Hw9plIAw3qPJuNeKJFCqt9LdWvrb2uTCxrEMs9efKh3aDxodyok+WyMTldRxb7Uw6rkC4AXb7qCTGPkwFFyZ0wVIiUbce7YmqJEV5MujiQ12ebfWlszrvN0FVZSvPTYdF3zJaJxvfwrCy+EiIGI2US7rJPkPL\\\\\\\/tWU1kNPA7VBKJKaaeymWYee4zB2PjDOc+cRPbvfR\\\\\\\/ixNG7WI4Cc7zH0BQ6acPngxbhDUsbTTL15ZgsOf+18pAL9ZUY0I\\\\\\\/trJ6oqQRInsULJ0Mo+eP4RZEs\\\\\\\/AjRBQUsdiN2lGyRNck0KQf8c\\\\\\\/vYpLOCgUTmwxP8RxagbIbptGNAG7x51E4g+5qTHZpqWDafLHiSBeasr\\\\\\\/RVodFldSNTOVKidP3BbLk63h\\\\\\\/WST8XNXn+lfthPA9dvwrd8bpqPW3QcU4de5XSVGG1RQ6Ug7rVMRsmK6vECvnc20zrsrAHhCmKisRyA5R\\\\\\\/TpuNWXEbEV12tJyQxPwwWGiWqsYd9LUW9vc41FgIsghJVziFzoGdjibbvdKL\\\\\\\/KMfOmDE1Bqb6IEW5P8VQ6zA3B5Zq3bwU\\\\\\\/OQ3LVET3j9JkmilXC93SiV3yduQWATCHUiz4CDDl0NJLY9QFyJLbtW7LKENxGM1hrG+\\\\\\\/GAwwx3Iqra8BUlZtu1YYduSBzJlfZSfFqhAtFnbCNbLqJFZYF74P97jfeMwOVf\\\\\\\/vwfG6zD3OQlOSgVD5LweuBuQKnZO2cl5IPRim2PMFK9FFIQdpSr8Qky\\\\\\\/439G9kDgMl6Wlr0eJl7QVxptWzPsMpP00DjogcNmEEg8DHHeZq0sHf16fsvrozhkow8uVZSwTXLLLLssX2d8LgPU6R6UYRCMH+odPGoHAqx6D2ZtWpiSvutueMGfEHaNK22sOmxSsztJ7Ae0OifWveR4iDVsjcKJZdXJrzavtRIliNIFLFQkNTR0IaW2d+3YPqGnQcKKCOToRUBG6Ymzqe8vFNk3c0H+BbJ9V3U4KtYrCLh75zwEYrbIGx228ZyIbFbHG+wzbIu44x2y\\\\\\\/kWnKe5uuh+3PnzLw3XJHb950I8s8kCTzD298gzD2+8kp5yWUYz+ATZ0U6Nyg8tH\\\\\\\/JKB3fqQe6b5Khs4Cb2dx\\\\\\\/aGbFv0ENkHnjIy1XOwgjFrVOPAKCklnHTtdzcCBeEbsD+TChXBqGPnAwAgnXHS0bDBk8egsOY+etmoXkAaKCsgCanbidl3elfzwGcYmkXZYkjJmqtjSVGBt2YnthE6qmOhQRYfhZ8wWBLJi6bNrg+X2WVbE7ReCldkJanSkAcRhHBMNtzwlwqQWwKGNUsxZjtgFP0JF9bp+j0hlCaYlfONe7xN3KK5Zi8plam1gNZminaaBEwpkeU5CtdatbKbgSHd9QdQfJT6dBeMJIopIzla3R97loiQjJ7H1UI++mQiR0G7AcQb2AJGI2ChCbxDAxFB+NibB4rpBqDyifAjIqjYHVU1A5bDWYRFVJAPSQBFAWwIUQ6H8IWTlQ66FD59yR9RYlirC2DQwq+w+ZRp4td46\\\\\\\/0RWfet3yqeJLI\\\\\\\/oLuvpGnmr441ZevkY7nMMTvFhc3oHpWLgdTl0bLiE2LX5BZB9h63Wj+knuKYBXWFLPyJA7TF0UQwagGrZqGkGG6fwk\\\\\\\/uuEgUnWwu0TMB048+9m66w\\\\\\\/6I1A7Ys7bdPpvOz0IPXb2L264JLANhic8g2M8jGMSwVuNsmGWqiV1nEklmhXktUjxXj0mlILWEizDrU0i9Ev5YOiZypD5zUVKWTPbyG0S2DqteKbxFV7W+2Ulf8byP7tyz2TD+4cY7MGuB2XK8p18OkXjoREqvNQbbZcxUWADbdDAWCWkO2kKb1ytW6BCdpVirVSnX+iG2qb9K\\\\\\\/spttvIsdjybzzxLbO460D6iHvWgBSyE\\\\\\\/sMUX+CD9fVe12Pi9Im4KHpM+WA9JaaW+Vs6mP0Q7rMJY6MXlq0o5DRp13UPkNu8u0eIjeHrYio1I1DGk55Kjz2O9P\\\\\\\/2Ce9UDNr5eU6DPub6+OLASowMkqPuFwu4byPyqhj3BQMrE0Ew6e33WLX+wdIT1PfAKMzqGWGGxntXPae1Ib90rUoOaZ04Wo6viejEY\\\\\\\/2U6fJe897ucc9uhtC9xue4\\\\\\\/XpESXqZpQHWKZB4V+FUmIAQOfTDg5ubSzdxRxwvrLKpGb2Q6FZkO7GBio+Q1PbiAnUV9I6LT32v+bP1MhCw9bcl6jr39onT\\\\\\\/gUU6cb2g6ibO+aiWeF8A5G2w9xHA4ookXtXOw+ofsXVfW7xKW9ehiQT\\\\\\\/ajAq6LgswJDJiqoHf8q5iV52QMYTDZHEaD+0nUP7Bt336A0rjXI3ClYirSTxmVj8jjtdrsHMzWRsX+SSmSV0cFd87qAEC6owMYZ\\\\\\\/izZJEj5ernIyyElEOjWsqh1212BT+O+meRqz79fbyRRKSnz4YYlP4GpztSgHYJmoRRHD4dX16EjSKgvjQeV4x+Zhx3ce5ZZbNvRsrSZ\\\\\\\/SVXZuzc67Puj8bdHa6XCDCglV2Pp2VqMYsd+y6Pu+Cv\\\\\\\/jIk8ilXG5FaPJ1dw\\\\\\\/U95qHHulCFkTR8XK0KgSQ6C6Yph4BuRMduVG3uFCP\\\\\\\/D2VOpgg6LtfdBpB47uFfwnqI6QIdvXK\\\\\\\/yK1cUNQmWRAVP6Ob1+Ry0VA9KQLwQbGJzmZ6wTI5Kf0Wa0dPf2sViSoG3sodX8GCUrbKRlr7UjXC03JVT5Ml+EwTA89eSZaqbWJOr\\\\\\\/HuGkVKSXI3PA3LKHJSsC0kuLGMiw1yEMHuUk7RpmRBvYgTDc0hvyblEk7RG13bI\\\\\\\/MlfS9YpD2bzM4FloU83FE9bHuo5WRtm27OBwHDcuvjQ7\\\\\\\/ENZFak6lIpcVFLCuPlIS4pPnvQeMzMiIapvIwmRbnFtLma3VVYn+5jgWboN0qKJChJVohjgTrNE+dBOKjCA3JtoRTHty\\\\\\\/\\\\\\\/iWiW\\\\\\\/2f2g\\\\\\\/\\\\\\\/VhNIKqAoMZYmKvGUTg1L4hpWsYr2tutyXE7MUPjObLewIAA6HqAda4m7GTd\\\\\\\/+0wyLka+zJ8XVlqvjeB93dxJneKUbgLxBAI\\\\\\\/AjmCvynSRRNqWZAkCgY7gjPtFN48xUuK+svfoEhHJubvFKi+BZGJMAkf73l3u+1k7dAfmP3fqwA3QwFcjv5qreqpRWnV\\\\\\\/eIxzO9Cq0Mn\\\\\\\/4cInxeq6VbwA7Udy6CRsWQIsU6ILMOsgRPMGXNUHpKXjrSfe\\\\\\\/XMZknZdmxeBowDW9g+pQnGtYbbTj7y3UjMNhRh4jFF3rNUUqmCPTyqrWOUA7Ett\\\\\\\/BXeqeqQNXBJETUwn0VGqgWSfwKC3f1qAdjZandN7D16ICcNnZc\\\\\\\/ZeEB20LZ\\\\\\\/K3zKM1UgMNO10eDtyAywkkD2fBSnpZhMtY+ayex8R6PjX8+Uzbg0uB2voLQ48FnlsyWnlIN04PUWLL4f26kivyEjtLc+S\\\\\\\/Gm2aQxY\\\\\\\/M1hMeWZdPHtcBPaSvH6Jsy5APODRNSikMrKgPPKyS58UER\\\\\\\/n5Hsy\\\\\\\/tFDB2RLuLZ3LBkNbSsFKqtmNI2TMdK2HtDtAe78XFn\\\\\\\/fuYBvKlpmGt\\\\\\\/R2yroFdlmsFTXxPet1v6ghFA+9zT5vXSG6IHvHTuRHq7MDEJbUuZ6yQksDiKPEK5tCbEtV6LztEIpAFlTC76fRBVkHI\\\\\\\/nUV1CQeTxjDWvXq3a45N0kI5ebYirC2bTlPk4Rx0ugjKXjyqUH20vuvQZfpoSTufZuFPFVqY3ml\\\\\\\/wXgt9LFYtTa2SEcId+2twsOxZlhZkj5G+cF\\\\\\\/mAsxUw2F3Px+65U++Fe\\\\\\\/+tkRFSaZgWT62uIm67ynKRtURxrFUAe4YVaNWfJd7+iLdo\\\\\\\/xz+n5qKbGp257Klr6Ou1vbAYRcOa2SXJ\\\\\\\/LWt0d6ijLIkL3PMUrqGAajWtzQq7LxKmiSemwuXkYJXrhfzT1NCeorVaLFW+S18tp0SRvbjQPwh5pFEYRuLdpa3Elp0mBwnuWJQbyiq7Ncvl5MVVMCiurZ2+LHs53jBdU4rtHiDlae8XRsk7dAkSf4FJXbYjjG9Y+YkemD\\\\\\\/uH1pKthKnv9T0wjJZ7Ega3bVGwB\\\\\\\/4owC4cxO95ymhJvBKf2vvWs2mt2BNyQXYy3nABJZvpxzxTvKkvsi\\\\\\\/bb3qWt642iNmggfr3n1EzmOLZWqg6C4ZqdLy5U14EAtwa5bix11Ec7p1fGP4mdy3In4+oUtVP\\\\\\\/MvhwbrPQZi6kzIUbfZn8uh0lfRNPIg8o4qvG3JHlN6LYfStWOf8Ou83hUTCOT0dynZ\\\\\\\/IydJb3o55yrLz40xbzeGY30JVpIcSaV\\\\\\\/P6f7qQBa+RjBsCzQwcgVzbBeyoE9L4wUCm6StGsQfLbK4XNVQl9v\\\\\\\/yYRNoxQsXtecrnr93aYnp41JPLOhGNBvOe3yaFIld1ZusRgl5HcARIlr05m+c7jGN8jfpFad7P067InjoKJ0KgMg0nbdT\\\\\\\/ODs+KhKrWalYZB+\\\\\\\/wCQg43Igg+pf5PyKjPGPpUYn1r9W8Sag3oSiRL2jNOe4hCFCqj+LesETL7E9R8AhFLpi6FrrzfBb1SWnQoTPIhmaLdixL+P9CGklQ7vwLPZ7uCLk713cnXHq51V\\\\\\\/NiXbshIyMFkXMblsTEFu8xlQCQWbdph1enXpaZLK6H5AP4ncyIyEBn5M5qIzQYOcZQlZ+imaB8Qi1r6ifQh1Rz5A6H0echJEgpN2DHzn0ZAu0QLsIYyCEpWzkeodoSd1p6jalAHJmV80MWew1LYJI19aKD\\\\\\\/WGJF6rg1M7P3qSr6YCARfYT4+yrJYxQigRWaleoUgdkGpXecI6+s\\\\\\\/6lDgVU0nS2BVTfNgFfoQbIVWKn7jMdIy2mnNhoGBA4Q1zgD2JqtAb2lN7zeSEuwB5V6ba1vz+7Qr9OEr9QHjbMqKyqTZPvSQhc7fGxHGcGuXrWnMX9ezadca5GQrS1+2wVAGdT8pw3onA+ANRZFGWKrp1PgjcrwDqqtMkNTGx\\\\\\\/vo\\\\\\\/rVV9nxnSfrtezzsrKkIqfHMzJ0+MucvidevvRhddrca7DP+R3oLVTSg0EQ5TswjYy07uTx2vQTzKpP+FXedeUsKMnY4iXlIyq0KL+G9sNTPr1KI8IZe2C+OJaPlf0PUFW9LE2JXXAetohSa1mH1Y5+N505f2BbFJZE9DBkYqoaRmztlpvTIuzYwp77SpYivs1Ebnix6sybvliRmRGvx\\\\\\\/nF0A+tSIMS3G4GwMfbWjWGgvhGfcxlJsqCb1YZGdQLrSopfcCQUIUQyoNOZx5edEanFfC1JNvhZauSyQLwctCRp5w4NKts44IrZ\\\\\\\/z7C\\\\\\\/plDvJK\\\\\\\/PPRYVOfVi9h60I0FCcxo+MJRNQEL06hMbBTY65Pi7c7WVDPF93J8BiBLFSRtXBMnGRPlKO8\\\\\\\/jUwjhuNtlMF2AYPvIwU6x\\\\\\\/RZgkmKUr5NOlyEDglq3Ri0n8UjakHAEnthjxqN6JksLKi9eptUK\\\\\\\/WgSki93vpsTh3JXpFUkMnuyAHtM+2HrD4onHVxj\\\\\\\/hqR3tGXWzmpgruim12SvrLXiN1DPCBsX9BwAvPj8JchJZOr4AmVCXwA3e0OaXj5QII2xdYTvcd0OZv+nWsHmDgmvpEqrJoDom4qjeGQqmtbazagIw2QYtvo6fIPJiQgcx3rxGLwuAzebFXOfm2g+30J0qPlne72MVtVElXJ4CMOwf1pI9hJTCOjuW8Khs0gOQlQU4yu1js\\\\\\\/6RSr+c7HGxZND5Pc0FUI64wqM4Tz\\\\\\\/0Pmm5RKSzdxLIf1zmgRM9j36g6eg34\\\\\\\/W36IQ0ZggLdVS2E9QlaXVoOaKdckOGQ\\\\\\\/69D+6RHxqLyUBHYltyPgtRuHBLZckBKd1QBifYEU9XRFeUQTdoreYFcOYJjwfOy7Antz4eYbekdhgkTCkaegpq7He\\\\\\\/hZJdeHm1TbsJFZ1got6hxhf6pQPGGy4stQKc3M2SK7x1gBXr440yFVnRNE8nKYWtYsP\\\\\\\/4D3W252C+7NU45l4lctGHvPt7KAGPWStuzUx3ZLaHpx7nQWKBANdE3P9AoQFSxJtzgz1Fc6tZtp2aqn1VPSVkeJXJQJZXcx2VBvKVGAWdeaiFBPq55RtZrAMnWguZsnNRFGTp9YuMp7HpKUnDhbpyRF3s5BopuEKKRDJmUG0pfCctxUgP+tK0++x7n2Ah4JsawgeE9pdN0rwZ+zsh5V61hrcv97NQ+mFPseTg0IrBvuVukOoxdlP4Jaj9XVApn6F\\\\\\\/RHDAbMtj6LJCki9q3nXyNrzIsxq6TeNZJvgrH3rSZLrAtjzBh77oyHoCvG2MRNY5JMlZKhzREXXeBQ2oyIR+h8euhRbdSalI+cGTXNMGPboKcAOF2K5bV4VHgRUVxuLupqVWQXTF2Q5ZPHPwrJom16fWDMBTrJxHV4+gPQz9\\\\\\\/aFAZNVOhnj8XESt18JIn4LdIPF\\\\\\\/UY\\\\\\\/IOdcWn\\\\\\\/eoh13lNy6dDCiaVc\\\\\\\/qgvsWM9OP3LhIfFPAvAsD3CK0vegcaRipwP8OApIcDY2qRRDWVuLS9\\\\\\\/itwb74BijeAZZeTQl7dfOpHEnHJKKGpekzT6FAUbqrrIRBB2wscaOhgrYhCGafjJstloNcuNctCpLD+gP5iNGBJk+2X3DB3AlwZ9LVtSuFxxaIl9RSISGKCV\\\\\\\/1JU643z0ApH3ntvG6gq0nwaXK\\\\\\\/23TriamikveLlVLe1YUm5NSQqkwKU\\\\\\\/7F07\\\\\\\/oqspPgSBM9txBGTZVHhQY4MhBlZNgjTGGXzFXTDHwy8YbmCebhW\\\\\\\/I1dEoOVb\\\\\\\/bOqrQIVVu3K7rbbbyKj0WLw5jz7SWeNoEESVk8yqCImRCfdrcP+6cgrRRr0gzaaYNNL4BlglzX0+0C0Tdivzh8JHhGASLaPu8pHJPS2VSRQ0mquVHNvyxl0hprFQ9H3KFjC3qbGgo8XvcxXFVtSygPCzkRE1ny8PNJuv4JOoQZj0qrThjF10SX8NfFlfvZCeJpHXRXftad1oRVgEmYSRCtdwecuocWftlQ9PP52HQMrmY0U+fDruykaq2J+kjISogVC3WJPq5RYuMcdHpeZJqm\\\\\\\/rf4AhWH+fY6fi33WMsTPpdpWNYSjaInaRcYs2g0FsP8bSLwMx8gzTFGKlP0hzi6MdKYf\\\\\\\/8kQu9pxh\\\\\\\/pfh41Y\\\\\\\/U7wAz+jKNBAM+9+dOMpaw\\\\\\\/qHFo6wACPgKXjt70vskU6P8Z5BFCYbFs31LshxfrDRArI\\\\\\\/JUcOiRLV34zKGlWKnx\\\\\\\/eLX8fD7dHghu8X7z1LuUed06MmHi8WfgV8zHEEyEcJ6ag5No4b1QDa2WWsqOAEgWvA0CvLGX5Avce+ZY8AJU+kgWqJ+nfqGRCznuem47x9dTlzEVaJkxgwo0O17oN3ODjfLP5+HM2rADoPLowLz76hyVLMif1OOwpJGWf2NyO4nOelWB8Z+uGU3WCE+2I7G+JetkuwMe67EPDN+ae\\\\\\\/2SwwKjWR4Ucpm3CsDszbWkwYxQ8iZY1kDOBaYryizxVsbk0qLoio9oGPwFgW9xe8tl4UElX1vhOqQEIG1sWk6knFGKYJzx7ZPcQHaaWMTFfVQvBYzCi0la6ZqwKricXi7ipAIlt8vgTZYZWXxpmfrGdmhy\\\\\\\/gqBHaJ8Afzc5GHLA6p0psaVJTXXMXeknEp\\\\\\\/FUM+LXnLmCdQpqBluPQmY3m3JigB3ClxrONcSES0rgy\\\\\\\/aFbu0oO82R2FC7zyXMc85yt9V0Gjg5VuUYuoEcP9a5mmT5o9NZqpbWH4+q5\\\\\\\/gQitUWv2XcFBU6gAlzYRrwL1UZMgo5tpaId2bjsOot0Uwj\\\\\\\/aZ8DeC2npxK2vmsR4nhwkL6KL7XtPYBUVXdAeNEHXJW46xAVsjoixVA\\\\\\\/h86BeE7NW9nkWbvaBZ58oLHlRZBZN\\\\\\\/MFSahgD\\\\\\\/eZdCVjdoviP1bRUfby8d26TEbcc7tVe24wugiXioqleWBeTIlA0aRaFSmUhvHQgXlGQt0wPEprdQGyH5V6wlqk9oGgsgl8cBWBBMWalFyid9YpZ0z2jMkwsn5eBupEVe0S6lOuik8DUlCP0uNA4t1OQDEHEo8feZav7TJIW9ZyhmgnGOrSTUawAmNdsPhVD8epQvb5iTcQ127tj2UxCcKOs1OwzsR+SbcMWaJO1RxPj1USyl1e++hCAkJFDGsbt63HqHRwDPG4OB5PY+EFv\\\\\\\/E1Z9VXksLOYPCBRfBkHIVIiSF5nXgE8jr6lNfSIlWQzay+M2HobPrsKXIjOeZaFBIjX0URT+xvvcrfc5xeqO1KdGHftg\\\\\\\/ReMcHpH3odb4X55k6ZkA8lkR8BCUOMRAcLTPKUH9lH3sEnHLX3c0bzy6QEPijE6kCru9xJ4KFUcEFkT0rsEHtTyVQ72i60NEuNJ7LnXLmgkP+4CWl0NSnhStV01IHdTFIIRcnvkyw6a0\\\\\\\/dVeX6ylQa1bOQG8be6g6KBgR5j4h\\\\\\\/N5tBhaov6E\\\\\\\/jmS\\\\\\\/hsFJe+AJn0LxuQPnnIP7YztUBWwhnXqtrcIEw1NDvLFUbrHoFgCfqQtPIkQPWthHyJCSNhWmgBg7odBa7qjuvQPBuOmyIAP6NI4L00nzSTEzYw1k11WY7x73bnFb5tjtbRxqEQulW91KVVpNDeGfvbjy4KoAU59ch98xprjkuH7Rj6lzGmqoVbMy7oIqSJzoK+PoPeAcdWo5Im0NZ29hs69d8YgDs\\\\\\\/lGDwNlIMPlyiG5SSu8vhO9wqI7nAb4KWHoXTIht2oNC1j6sxj1zNZdvz4bH1sUXgvBtG8c8RIgiAejgUJ903pI9nqDUa\\\\\\\/o5w9jF6lOqKaGd4t\\\\\\\/vB\\\\\\\/QHhkpPqo8Q2XS+JqZpTj2WocV0t4euIGCyCJhTwxP+kKkITbjZoUn2F684xgsukRMbF4WBN5iNagzPqr\\\\\\\/u3bEWbbGp7kyKcdbFI+OZHE2Ue66vd7pM91vwyWYu5KIe4kVAs5kK6qsCTzxAWnQQaXzcmbetfW3frociSc\\\\\\\/jGsDNrue\\\\\\\/xXgTNiR8PIHyrCe4vCmjHK75+Dfsx68T28Tkm3R4fHVuKH+L7ixrTkjFGrTroKKY\\\\\\\/gm2fpbhTBWvGwpDl9Ai+HmNsHQcL11QUOket8Wa6QMtRyEgoBI0W5fawdskBi33mMoxgD6n2i8CnK+81RMd\\\\\\\/mCRbb44eo22FuCKRy2a+lftBOJvOvISHvrdWM3q73lEVdf8h2\\\\\\\/VOASGTljjYnlagJ205T7hRsX3lTN6SemCCbtKRfKeeRZI5gjE++ZTlapIGysmBl0aIPLoFzHo2jiBfvl6xGbI9jyV4JPVdAHirO8rZySG2mKTGn91NagHEEt6xJGtkkDh21cfWRFqREtlQ+plDJczS6Z2XtcRLg+MMUi4z6KevwmP9NQggqhJg22pRa+o3GVFSFjzE1ld450ndmcKlGxOxnR7Uslui+rUPpyTKHYuRaGd2pIh82R8ahuiACko8eSf2i6wBS5eQiC4EncmkcUNlT4fC2CTXCwkg5n6mVsqT+I8HBtzIkcS5hbKPzX8y\\\\\\\/66p78oE9D98M2BFrWJYs04Q6FXSyMqct9U27cUU\\\\\\\/IBISsS0qfHsDsTunOwu2q5dx5z\\\\\\\/NqHNkSsKN8e8+h2NrJAGggwXuyqMornNOVXRGMiXXinS8XS8ViwvyW\\\\\\\/NNZVQX06RJ3p0Vhas62BjCKq+hhK69WImAZ794G7K7lCEft9oGYKvZ\\\\\\\/5qmIpBPQ8SXpV5zrqcyqWSraf2xLaf3fdAU26JlJviEDOZIRo2c7e8B6vAADG0W4+S+D1JvmnHXHAcrgMFqD13vALVYoHJ5aAX7U3ga2tzY+KTdX6B\\\\\\\/5BXySGThbnzjg+VQ1cenjvdVGIsp8\\\\\\\/6ZAqWC9U5O65MiIJq0Lik+GNtaLeon2waQWVaI+svzq1DDNqbySSJjMcdTrIG4\\\\\\\/q95wXmIyZT75SMsUi3ebHn7HFRlEifkZWhnl9FjoY8zxu\\\\\\\/q87wa3ymh44mB4W4TTm9HWToUpzT5GMQIdCNAvIWw5pNS9OHqYVY1qei\\\\\\\/YJFSU97ofIMNOSw7XisZwMrCZ7esin+0+FHB67aYpwX74jHg+pVv3Cv\\\\\\\/MGyc5y8xrmR1Xg4u0KMVB96LUxfZbs4tb906jaj5uaveYSOMm9UXl5SlF7a8YJLw0AIhhC6Ioc4Y8lwhP0VRD+qh6EbYVbL8Xnrb1SyWqmwUPlJvdcq8Ap0tcglKMdGVcZeFtb5hpoKqlA\\\\\\\/mxDrjcCyEsmPZRP7lNSZ+CSF2EZOfuZyvqfZewuaKChagva65EYUs7fBCbFi61BIMAhmHuxsTHDgWofijiGLdhyxpA\\\\\\\/emW63AQw3zyzj3sK7DJRcm6ApsNVUfVHUj7ISIn5RaaHyESPHXJmheMGXNFWH2ARdcdDqEMUWtA6\\\\\\\/4KqDwAquv7540RoteeCHLPhj9FArvlAT\\\\\\\/MbCtjVd8LERD3yopfFphkEyk5YipDMTs8dI6E41snLoUUtMKvd3fj3+u83uISIlPr2fJUVCxRqDYL5De3UD4uLoFq1gFuVTlgaWTKIk3Dyovo3OzB7r3My7i67iYlAqQ+nPPH43oWFMqkxYWW+TXFPC6dO7R44rVrq+ZoMMXuaBj6L3yBnujOPTwAlSF4ugwSymS+xkjZRawxnO9B\\\\\\\/oqF6jvvzYmMODmoft+lLuOfKH31EGymqq6sUaSud\\\\\\\/jnsu5U9JdJPYckYBfXn4UNDsOrjVTzROMIt\\\\\\\/pTO8hgnlAic45O4BHAt3KgLUGPBDFobKSugVk7ujipZZlKnCP6NKjcVp3aPzXPK47CzdkwBdn+IaFiq0bWjDoHfrqxzzfeQ2zqvZfY7fLFOwxXNamXyRHfcLf8AgLykUJwoC4DkQFnIXiIpTQ7+qS\\\\\\\/YmJ3CLidTA4y\\\\\\\/HUS8oFFJ9TLgWE+1c4HbD91A2jzbc8o5JcVOGJS9BepahqLdAGkCzTLa5YFqV+bNyy1PTL0a5VTc8uzSiWy1lmjuUptAPH9oUmNVhkcx3Qu4ggvZfFUhW6fhuHXjlJamnaYtv3AYKXrMwtHjFE4YsxUMy7cRodS3E\\\\\\\/A0A+309AQYUCCCx1bbIk3MmFgYkTrpGyZF9v7iVKq1CUilTwhQYh0arB2rVLGnIyb5oSLcZA\\\\\\\/R18aCfgQpGyqDaJzY41dz8QpSyjXpXQqnra3l9M9OFeSEhtEBZzoUFBxhc\\\\\\\/JgAMswoMB3LFbfEClTZHVgabhjBjti3wBoSmtph1ZZCUDG8k5EBv+2dZ6DA2EWAzEbxDIIQg8w8Cqh3Ir6e6bSEvjI9ho0k00S8AAm2YnNKuGXd0SrOS4K8oLVfOcMlRAcT+vWH6XJ9ojD6xnMvHzXPYEaKVbD+7UnNxXNbDliux9cBNFVGb7R1mmmOmEQSw9TmOxZHlMANzDnVozaELz9BJ16P18xBcTlzM7cz+wVc33+D03EUVuvGOJEuOT4cdhloxget1yAG8PQLCmlFgth\\\\\\\/MVtXm\\\\\\\/qQGumNQ6GFQjLxp6VnMYOmYSNsUeQ16h2ePbWK1yg803mrmWvbZbgTRz8aYbv8SGDvOjug1Xsw2DTH958sLaTz+Js5qQcJYAMRDcZszua+Kj+4YhFoXe5un1WQANvlPbHf6eLXY4fFwBlGfoedyKltYN1mqVArErm\\\\\\\/2nqLIJ4b0yWoWUddl\\\\\\\/k7ultVCq3W9UNKL5ll6K0H74XWIQZdX4Ilil3lsd1L0BFsN8SWwQ45xxAFE8cfIA8e8pdLeMaUYZN3aluK061PJFWF88nII+qNzLjGLlSXuSCNHCyY5Mb4BXA8tig6kDwYGcEn81d01KT6DIQPChbAU4auPxfBMO0A1r32jWVtWc5jmHhcUIgin\\\\\\\/Ij+mrwf21CFr\\\\\\\/tTah77yH6kH45l6NLH4CRsc3SeZ1GzKkvcnI+xKDD2H\\\\\\\/LO0LBQBDVAvhdn2f3xb5X73FsJA2YeDygVCezbjvyx446eziqCs\\\\\\\/ki84P\\\\\\\/+Ae0rsJEDFwNBVwn1swRUCzc5td\\\\\\\/3gTKcfJRHTdB8OGCiOigukdtc8Xf2wycpL67wZ7tK7LiCbprv8+y7H+xytnMdx0gvKruRlaiUiDqkL+K9MPn95CA9bhQ4WJwJjo\\\\\\\/FVuxcj7OPK14zP+P80jpnZsNXa6vPOrvFS\\\\\\\/+HinACyo2tmGS5r9bdEk4xvzNGRvMA+39Un9hSA2hZRfgVZgS\\\\\\\/gmJqPDD5n1XVlN9D8k8qJqROHGqVItr467GIMXo1driwrvR4iBAXWQr2HzoUJMwJcuegalhfpz6i8cLjmiCKah+XUWwkVlucLAOgEg3pFS849ZpZza1eCaLhLTvenzZHHcisOaLws2DGqx4BkaJ75vbkrhlv3JM3s+R1JtahCrl0q6dcoYT\\\\\\\/3gH1+qP\\\\\\\/4kiI00wBla5KRS3hq9c+jln3imUtWFew0K3ip6KSBqyEm6JlAQHXEaKIImzxd08pgM08Tf\\\\\\\/2PvYtlSItrA0JHR7Fp7Mfoo6fme77QV11bwgXQ98IE1rdDtiJiVgC8Yiy9smYsEMVnb8CwhERJyxWug4UQqx\\\\\\\/YrGLqlcrt\\\\\\\/1xZYH\\\\\\\/DX+tGf1Y7npZdl\\\\\\\/lWB1tLHd5gy6n824lWf2I9cSRkE\\\\\\\/B\\\\\\\/OZ0G35THsc\\\\\\\/SOkyUEqA2zsAeEUb6G3L97rQegK9FMua1Q16KO71lrqAxxEQDt\\\\\\\/v+PpatlCCrPThfY3cYW4PzV7oCj9kXtSDdegEf1qx+m06d4R\\\\\\\/SQePacPgRDsfQ9ZP97ABhJf6bdfAUPU7S7z3D2C3IH0zpSermnOrLZeUjv2LtPepzoclS9+8M4F9T4LtckHms\\\\\\\/iZuDpy8mjH3wNehp1s2QaSJp7awarS+opEikS3Z+byy+gS9F74PQ7LV\\\\\\\/vDh5fHpVLZZ4tNvfpQQl9YBzmBagaiQ0vPfxTO2ovWAUVWimLuqU1WGUQthZlmd8ltFjKJvhOvidFpBw4WGQPWhUu7be98sk8mM8OzOpKkp6birLTcx+qF\\\\\\\/Rv76KcMSRZuo+KQeh+suSHHrjtkr3V\\\\\\\/V9ueWLXj9IQym9U8FFiezW80biA2ufByzFE16OJjyKqBDHW7CEvg93hT8teIBjVpjfcLJk1gGA8ApdHrEipfACVgxCAgEG5e0fcZyY58YcR3HL36IJuE4Dla6CijBdl4pwQeIsb9D8tmn2nUYNV8Ajm53F+WwKaeWjcTkU+ycuRex06+\\\\\\\/6eilxn4ZeUWq+i5qQTemqHqb3yvhYQLqlc8B\\\\\\\/uTodRC175pk6Gdxs6p3CvYGLP7YxrvzlWu0yVLd1el2AEQiE5ZfErK9fCSiVp22i1mwoC01YW7F1IQqB52UgjF4Ni16HU4E0pWhrY5gNHLArpXs7pB7F070snyH2xhEnyVaG\\\\\\\/ss6GXQZEjDtDP4dX2XiKnZM5sTR2EZ\\\\\\\/uuH7xSmPvD7dQaMfzyR243cu6+W1KUhK\\\\\\\/pxw2i6FODa5\\\\\\\/W6z6VTOoKt6aV2WlKteFjGd6mv3aTeWs1kXDdDuoxnPcSW5ikmvOe7deZRwO7AW3TJ\\\\\\\/ZwlxDnD0DY9wWXeoOr3o9Ms0f45bCcR6LxPS9uYAmRD8q+ye8mVCNPdm\\\\\\\/1zyNhLqwLp13lqeFwLYdqTuTA7poAq7f5kdzUv0y7IAUsBmPB5V6GEQoJxe5710mkdPJuP5b6bNECXsAkGBWfK35ttEDwm+ChPyWdnb37DVPCrjkznWv+e6Kxa+rHojkkfjuO8a3vEIgjWGSDqxwhFjbnTvgBjCkUhfXh9BcEowZOOVnX0U1cCOX9tHtFlXhLOFptSMD9nGRrmbA7cbt+Mkcy7Ik+BF+oW8DK6mofgomu5wf+XKR22stNwzIV9kNlf67QJftFvVuSJoHfwsDbz2jsYUqL1mM7lTvWJdzpxpjOsG29qeAHWQO\\\\\\\/IKjOI7syChm\\\\\\\/z+yGRMUFP6L5ae1LXsaCFdaIA\\\\\\\/v2\\\\\\\/wlfsT2gtGHgJpajVm3qoHyWj8Gpt6aFwUxPW4k\\\\\\\/IGbM77qjWfq\\\\\\\/FfCTP+nxQ+Z6Htam7FC6IXWr+TLLXB4v+yVmw3aLjGPZH57VJPKizmPkV3cqAYK4EzWB786y+qe1UY+JiNxl4nBZznnXMMp6SB6SXtv+SuEjGDuWzGyOEFzTeblq3gnEZkmAlpPZH++r+B\\\\\\\/KNzPGLt1EOqMLmZcG8g5nc8VHW0pRhXkTfQU3CPbs+SbceiybEEGlOnAvLIWElTY7evtXEhQY72qAm9wowhl1uvUul\\\\\\\/muN5wGi6lt4sBJVRlV2Oxm9tFdLlAh0We+quR1\\\\\\\/u5GSoMXMZKG+5wILV0QG2yynNJrHjaMAx1qxpKlYodqJAG8wlVP9nyJ2zAV46dviL7R02x9AxWkO4EbIshxPnEQlwRYSgcFpW\\\\\\\/KKllQwcmek3WLWVbJcRum68NSOEY7TxE2yb4uMTopMjoyDktHbakuyszMrHmFcfAVnbN6MkMs7yH69d8F4Z9pkEQOHb0qTMtErOVMZrUQtP4BRzfvAKgC0G8pVr\\\\\\\/bB+VwwUT6ecqcZMi0L1TMmvboCsm1ldMk5YMrX7GVZdkjR28+4I2c75745K8uodIvOIfRPBGYp+IoxWt0tl5UmVmbgY04lXkyPzJLOuzqKxLTAVCC5lwEDbsZgpMQId7RDu+ZfHRRg+1iWJP5frOyD07TpD8Lk+RP7za5BztUjnq+YsSIAnD41BTvt68QJZFGfxbhfpSQ7O81xOByTcKJcjmjBvjtzmkCAGxP2xa6P+Hm9oyT+ISUAZW7MnzmlmnkRZyBLASHTgmigTfEPM1em8Dat1HfQb5RESrIqQr8Mn5bW7LY3zys7o0oAwL\\\\\\\/6vza8gCvrStm2VZKXk5g\\\\\\\/3VYzRTrGsxzQWphcLQ1hvlS05PjlPmYHIj\\\\\\\/hypXmHIETKe3TO\\\\\\\/ozCT5oERQHfCwyrTCsZfpyVLIxvli\\\\\\\/2D63tdChphldYaGHqw9eLrTwIS3mnb3kcw2JGZ+mtyDnELzXl9WlNQZwMFPgYo7hIexAq0a0Zip5ue8H6K41hrYLzSuQOW+pDhK3PR+qvuvZKrSoDl0Ly4EnWcd2kj1p\\\\\\\/h72HO0gnQVd2klyZkm5gRqfcfVhs7+K+YmVynu7jN41llgKDS25UUcxpzmYr0m7JVBt8VprCvY5LBFeFcdr+WGkgauyTtgBUGebXARJY7S5dEZgHhHvKbgydvnqHNbQZU5VMn2v2oJejTvRX+wHqB3UQK2CoxiMTb6FiIvK3unf7AxTnxb5tZgSX+RZ7bPaWhyRpvKe7OqMLqPP0gW3CxyU8q6WTlfTC9wlq8gnEhy9fqIq5DKtZTxWp\\\\\\\/gEuEXTm2k41L34O\\\\\\\/FMBWAbG8lb0Hs8L6kc\\\\\\\/nV3+yRIuH25LgZjoeKmTH6yJGgSEBR5LY\\\\\\\/CR7lZ\\\\\\\/VxyuIyXLYHQmaA1V+3gkiz0+RgndexgroqyutWGoEOvrOG8nwC7h0DXnb8QpVmaM39V2BwyC52dQOXKXA3KG+zzPDOAbSggxu6uoh2bw7Fz\\\\\\\/ZD5KCH47P6jiUg3Txc8HSLNJPvscHKcMNEBRe\\\\\\\/n\\\\\\\/YG\\\\\\\/NxtueWIOUAMAKK6ajspRN0KazuFCjTTLQWUHi5a1pQ15yMF5iSdmtgifn5bzCfef\\\\\\\/xMOZzhioI7OKmcxlwocN\\\\\\\/0+J5vtBiYy2UaADwQgzBC6lZ+Fn+osjA0Tkewm88DAX5IDQs23gPCWOtssq1Tbl1lMgunRQmfasBQFq1X\\\\\\\/FO2YL\\\\\\\/qSBenpQDM5LcbV0o2QVkl3QXwEk9HWyWAX\\\\\\\/9mQSCeatwTYHqAIqI\\\\\\\/uyYl+lMzl4xB+jlToxbkXAeapwsBSWNnky6bivEVTZ2vBfVGxRJoD3Ui163TjcbdiooQmPP8N0q5OJAH\\\\\\\/pk8A\\\\\\\/AlzPVvCIRrkSMbcrYFB3wecLp1ZFb7XcQfn\\\\\\\/CSFffOwuNu1WzkseKq5jGmp8sF9Y9MUtLYnK+ybJD6ZtemF\\\\\\\/rpzw9KOoNJgnIzkYuobrcbqyshU+HmcEF6432n3KTAKqME0rvCMwdT6PRfxfucawWdnWlv3dSKe90AUd8HVhdm9i22By\\\\\\\/aAbBozf\\\\\\\/Da9wBvjQ8V0KI5LVzFv0UsEBWwyn1MmPC6vhzlV05YhTbiDwoE8IiBOZyI9jEiuI04k9AJSQFixQZhVBGSA8aJOZCN6NsFRu7g2jv7Bi8PwN41qXNU6Cb5+PMLxYKZJeeXnuhXs\\\\\\\/cEsd4QM6msRiaEWyXNS1w7fRBtmx2ExvwrSIC\\\\\\\/LLDJlqptVoRmNM7J5SFItfk86TgJRyULpDL7RTQkaG0AMhDnbTbhOkHAxHsQZ3yPjYM8He8+NjSFgqQIyqrChIYbM7hsoILwRYhsJSaNeM1YVkEd2PjiJJCdozC6uKbxpISDFB5SJSF0waDiMBqBHNzMeStnzT1Zfkec0wYAODIxwljH\\\\\\\/WmGG2hiFzHkqrS8I\\\\\\\/5l1hAy5sNtFu4vFdJz\\\\\\\/LbI4d+dnAbZoDi+0pXfDvdahw\\\\\\\/5\\\\\\\/AV\\\\\\\/LGGUFArgsfDXliVqvSwbEPY3g2iTqtQGzp6X6fvryAQYIaLQS532t9Zgm3OPZ8keIiP6wq1PI5H3\\\\\\\/fousWFYHnMgjTCEtSVfCxp3IDgHbRJHuKi7q5+zs2SugZXE2Zv+VQQ6o1FiNWya4NjdhjrviRmb+lJATr1h1sLLv+WO35scRBEPk9JCmaqeZdSWO7REnlqltBhkAlYLTL12jzqSVixDfjUpvoBLw7Duh5F8KTEMFW4cWHxLj3bKuWcyHeV+qTeCZbQNjS2cu63S31sPkz8++YMdLA8slc9XygS6lMsMggGxN+h5sq84UADFSBV\\\\\\\/vGHBipOE1RBZru1X9mB7fq976wRpRy\\\\\\\/my7tq\\\\\\\/B8nQLX5Hob5toIc7okHyXGI9Dk5NhhHXyQllRGf85rgWsFMFA63TWjlpzdVVDbl+o+gdGgpg5N\\\\\\\/BGLmsVRrYdgHhWIefzRd52BGG2iLAxrPteQDokiH5BChvtcir1Rxgq+7S3MN3F0TWeovKfYNsn8u4ht3hvvWN8nPYzC3z5P3tQbgkY0uGoAf4H2jTodaBuUzKEgUVQDa5QFw9iyXKF6xorB42zkcC25FEYbuLxtbf2Zjrk906P4f\\\\\\\/MxoUDM+rqkjytswLiXGv52rPwVg9YpkiOOQHP7gQcpA\\\\\\\/X8lk5dFpD1iSK2z1VhU\\\\\\\/enT9KZdW8qfEHYeWFfluWDJXZhWZlUS2pkBUR+6Y2fxGJI01S7n2RwtS+eD+6e6tFN7V1EiFJPJU\\\\\\\/QIDAwuT3KegltQhnRJGuCSQK9NNfQ4QsdrIMpPB0CC4ZGmOsQJ1C5d8oCuQsG6OkRUTwD5qp9GMexwauODpZa4BjtJmbDG03zvJRfGy5jbCyEFR6GEIajUZwIdYplnjG0ytwOD9MJOWUOcEIVnbjJfnMmu7w7t1Cb3m7Yu4j6ufFndGYFsBSMW8yTQLjZtd\\\\\\\/lrlzs4jA2W12MdkNS+ZNC90jq1aFfd7uwwoKEBtaA31uV9HPijwVs5bw2biAxOVGzjGvKSKADk6fGHgQjM3ieCBRbUQc8shAMcXyYQVbQxKV7oXN5DlcUGRmPnE7zu3QVj9HlnXtOiPgshbVizkWnCeP0RL5aH6fvoo0bHOJyqfokVerqRY2ysX50fcdQBoKHwCpzkdxmWoxHgfI+PCjnR3yzfuievjzEhSEiujjbUbpfq+56dI0wezX7fcZxI1NLaQVLJGwRvCL3Gx4gquLTyrmumnxkEVCzAQjgUYjRoWfJMYMIQgch2orVhMDONbsNXDOuMpinn57h8YhtLYCF+D707CWxpJmtkRm0upz9jtow3WZCtvQ5AZaFH6z3RDD5dhIcSioo5G0aA+Gc0VVtDCdNt0TjzTWj7xm4D5zpa1qrALN4jjQIYosA0bGtGak\\\\\\\/Vih+FOUQnuaV0qKN403Vaas5dtCMDZyUIyyimph0Kd+P3LsOM69vI2hWSs3puFunObUohsURE3rgbTYHUymZulqBB5XqqTRa7cgabXUypN4xO2z\\\\\\\/X9oXr7D2\\\\\\\/Hdmocoed7jBrh7m1ZG9fB\\\\\\\/CaHBE5YZvaETZpvE\\\\\\\/lZ5Zw2+Vqj6MgUeaZnOyuyB2uUM3pCiDrwd0Uj6H3kV+NVAbkzs13d\\\\\\\/x+RbH103PA\\\\\\\/kCdVJUD7guu635779q\\\\\\\/xYffwpeD4MCaVhKtQNDdI\\\\\\\/oYij3+KJy6wLgMH4ahRIo8M6O\\\\\\\/rlDUuNXGC2Q36zl+T+5Vd4phlYL4UHyY5Rb\\\\\\\/JXAvwqQLu6su579Y6cyJcrSsGQJl1OdDS5Y46Lv3b4DXjDfsH+B\\\\\\\/JiLFTMDAz36yRQ95dWp9GwTvyUbOC\\\\\\\/1LU9nrsY91YHUukFXkve8buYUuoSCMGAahtOz\\\\\\\/zgtACi1hdIDzJbKSFsnNmujPSKQVaAShwxB\\\\\\\/EJuVi7fSET2uuquZ+q1FsHomSf1Xn03MmsRcQACMgaxgRRZcwTGmDx2BVTq\\\\\\\/jXLa1LUL0use0SIi3wr9PrOBKwZNQPRrF8yKuUMJrrX2prQz+RcAxKNkpTQMDJKNjIlEVyS85VZBFCtqiVHfSOPZ7tlPOTnx7e6fIu88qMXr66a+g5lN6a\\\\\\\/r12AIb\\\\\\\/ZmDurnANs2aQcON50HG1SqAi4d4ov+eQGU7YfXj5T4TN8L4O8+yiAbztQUQ+bkEaor0\\\\\\\/TUKOb20r6QuM2f06TaaZSIfGw0eiZKVwoIjyyLqdR7EQTB0w5I497lEW1tnBwCxebNxEsyA1PNLq21kisMOzW\\\\\\\/We\\\\\\\/fdcFpHIUWqZDC6VdbekOHfSexBC4HcSeujE3OYsw4Y5KJXISe8U6ZqCBBMZNOh9bNh3kGpDND+bURSWtPlDDDfokxGO\\\\\\\/sYdrfElvymcDVYIajjEyoapiMg9JG1y64j7kEX1y3n9xJXDvTuGqr\\\\\\\/i4RTwfT59PK7WSc7SgSZ5ioT+3opbeMO8+zu7ouD4L8Gja1YlSdLEhILMg4Y+026Hjn9c\\\\\\\/j6HPzXVOxldPJCuFHHHUDi2E9lSCCOd3N0aEOr4h0rNKmDGmv0jw+hSOl5e16auc63d+cSJj6mIhNE3ed3zd\\\\\\\/bF43Gy\\\\\\\/YmeKwJPOWwmxCodB7yf1yu8RyXuIxUtFkyrGo1u6EqIxK\\\\\\\/XDmrvOWOndxVBKhixoroJtmOuzkMhSYDsym6UiydnmW\\\\\\\/ZZ68aRRoeXQRwD7O7Hc4gdFz7dvJNgsgwXVmVpdYETKFF+w6mDz8XXhpNiCp8IfoauHKS+1Zq9QQqxzLdzTy9n82ARIb33B1CxmQybUA5prIVaAmp1HCF9oilJFkIvajaVyA6iXZwK8pHjzDVIg8AQFVBTYGaqAzRzu4X1d0I8q8vKj948zz9brJqj2us0Yg8rcF\\\\\\\/fNGIBkoYZoqide\\\\\\\/uRH7eBLLmdf0749WQU+r\\\\\\\/tVDf4RaARGAd9lHjkAMHlgRM+EsvibAgT+UguIbw3zKToyRCyvkZHmFXhZpEe0mr\\\\\\\/uV++LlcSIxtZr1yUMlcG8GtBMehG3q2y2KiuG3kVcHkyc23LfpHyrovoZLrR5soxtM0R5nJ5TKjaWvXhko0p6J8\\\\\\\/L93u4cjB1W6dpQVLNOOrxaRNTmUjkoiC274DowHZy4RhHBezfECwIQe9cFXiUVvMjmTOVukiAZiImkYZJ6zPqgpcdf1hnBsKJh9p2SuQOaD+q1zKs8BhNC5Yf+BwDFGaNvesUUC\\\\\\\/0cKkET6BGhoXbVCFgnpGlBalqilmiK2PeRdIaTUqYgSOvKS5TnnNMg6CxhBfbU\\\\\\\/r9YfaXHTi\\\\\\\/6BeIMr\\\\\\\/lGPUW3pzwFAGBc299idirGY4E7Y3RCaNpIX+j81ik8u6CJpj4hOAWDYiAjY6TIIybfY7GuTya\\\\\\\/sCX3jbOM214c0Cc4RBygBeuhdbtUvohgHexCvgI+dXGGuGhQI\\\\\\\/6zkQhd0wFemtJRkw4C8P0pMVEloT2wyQxq1MOvsMejAAeRHVe3ugQmPwzTs6NclCyErPlUrJ0XYrBF0ZIRbITZ884JCbTlhOoNeE7beicGsFqAayAnFtnBihKWhWO\\\\\\\/1wFzBL2EtVwBf3KCLxaL1G8TcOy3\\\\\\\/s7rbHiVOdX1mTGHgUZC+JGkPXkDLFGxgAa+OefCo9wfjntRwG8qSyPATGIGKBvy06syIf7jAthJ8nyvgknpboquTB+LvT14hxESkVBVefTKeW+oLFjQKPIPp93PLsRh6hzzi9dUlooP\\\\\\\/xYdWG0KmRhtkQT0Ms7trMSnTpeZe\\\\\\\/oxZDZl1UHk8KJgiHQXipq8mZbfFBG0v26bZseQSW6wu8OJ5oKDRQfGfgH6PR\\\\\\\/bkOiAjVQ6QvOTnUcEv6L25faCkuo\\\\\\\/9RFzIbRMiCeU8OP0QcdcPuky2c+vYwwO207Zly6FH0rFjv52dAuw3\\\\\\\/\\\\\\\/7foK+\\\\\\\/7yQKgjMxGpRvBA48hpS9mimwaOcnwMo+xHdPLWdae2s4gB\\\\\\\/qNo8ZwfUjSGCMmqexJVYpf6iYQ5ISOouvjG0hkyfKxEPdRasFnxZQSxv\\\\\\\/iFjEd1WwH44OeMu1owrQBbZLMBgfgwOsQvnh5sfeG8ApbnrwiJANyDBFBWkiwgMq8qnYLUYsO9y9LlBvy9ayAlnKGX2IMdKp7nz83cg\\\\\\\/GIK3FclJvQSnQW5vUXgQT0JgUteB1KsB\\\\\\\/CRx7aZ0Rxo7nlJcqdhpDioAG81ycsyCz62C4up6+ztgg\\\\\\\/WSkki2PxlXTgPMJ6pJjPgFZcjSAdJdcmWeizgcvDh8ydeoeuGc7v+UCi6mp\\\\\\\/sL05PLjQNnd7HphVD1ojktM4jHo9FfIhcjuvAJQmMkPJgAjjaiUJ3BbpwlEMMN9+1Ub1kS0lg4b2eVLB2AX0fYIErp3G66PqvdyO1Zditl9P3tI80kNcWXh9WWuBkn6TF900rQgA4BJQgGjiPfzyy300IrRIIXcNz8DAFWEu1bb0gjq4gYEl4FLLJe+nBDMPBCwCxJLQh3\\\\\\\/Ono3a61vVAtUNRHIj8FWYKXLaMPN3kvT\\\\\\\/uwtHL2IqVnElv6sFGjoqY6+fc47Ab97A\\\\\\\/mgljxHGzNzmyWkoBIrW+7oweaqGiD0iJ4zzPGyK2AQR0Xk5MNb+8+gmKQGOf0vzMmNGdt3r582bxp8jRf1ji0A2WK2eN0jRjHczNijZZAY+okffofdGRrOIupCWG4SZkRrd0OWiuZ7\\\\\\\/fmDhSbpWbtF9MM1vfWDXmkOJUZm7i9wYidTQtSnm18btLyx2dhR5Ok3fbyYce598HdPHnZcv+HHqan+wmI6OACjaWlpoEDb+kRYt3zeaTzMsrl9sXS26j6ypH\\\\\\\/rXfmEuInJ9IvO2Dv2F7OcVphLYutsc8qi1MmDwiWrA9J2qbk1TvBhc4qg3qYnlXdfKxxbcMoiKzcUtqKplr3XTj2J4aHf8wTb6cmBKMg\\\\\\\/T0tsUzUVDcHSWqs67a\\\\\\\/qs\\\\\\\/NPlAOBW\\\\\\\/ZG8wYek33Z8ULP3HvKLuS\\\\\\\/V8yUvWpA0JlNqE9lcNeOP0CjVuYsorWyf8C8vrZ+vusnfmbnvgsVYhcLsKdA8XrUK\\\\\\\/4L5BZ6R\\\\\\\/zY5pDvdEaKMBNdyiKie\\\\\\\/kvhFxml3SX2MR6Xcmm\\\\\\\/5RYpbXiei0G+ldSKEElyZfoW3umCXQgj7TgT0CGvCEVHacqDZ0evME9iHNuKhyJCclFlK1bAbXhMOrPn\\\\\\\/X0pyimCUEQvb7mjvybGUkBuAVMiRxES5N6x8ILFhsmHqo0BEXS0c\\\\\\\/OudhFNmEv60kYHR7uDG48IjZS78L2w6ligowzBvPqE9JKcooktT3Hla7p5a3TvU23hQsnkTAhb\\\\\\\/ljSBkXHn04edK9mijiesP7nKRlMWC7P5i\\\\\\\/owCl241bLEB6y4Aj5pPuk+i7FE9tLZVhVJNNCAGP8HWkveztx03KPQJu9+L6+IDdLCi0wNDCmnA+4NDELK4mD2PnkUUsmei39MF5E8756\\\\\\\/9b8m9tGHrbgq\\\\\\\/tc1zWb4YjAsujhIShtIHzh5lFeZzY+dW84e+DLUGIzSeSNR9sqxNFQ3WEN3c2DiF\\\\\\\/yANfgojwEo+xckpjcVH+Z1sNOON+BHD9F3VMa5MF9AJ25bFbWiOXVHF6D37UZRNiGuc4G3XO7VAxYNI6e++PvWt46QHE1jQ+j6Fd0J++IBxkr2Hfjurd552yCSacpAFrXrIpsgEG++NtzerdbnHhW4wxkkO26TpH3aMvd\\\\\\\/VZGhx5CAkg4E7GgNGTslbVKehhJWHe4ITG6C98UWu\\\\\\\/Bkx41WINn1eEnj03BuM8\\\\\\\/MvFNh2SVPWE\\\\\\\/uy5LZmGjFZ0PjCpNX44dL6WN257Qn9W7Okrv2XgrRv23cBcBzwF\\\\\\\/nty4DrQ8taGAkAJAg7zdFqSmxCr3IAR+723V891VEiTx9xChelVwNCqUFUeaIwvajscbM3Pjjt1ioQUvsDHDj5izNZNTQLkxuVKQa3FpBoq9mtPT7NUbUP9QZ2anwreAmIK2p5LuTRn7P1NMNnrIJn9Ow3Rd\\\\\\\/JgtvkV\\\\\\\/qhneVcO2RINCbvUg73Vp8A5KTrBMvPqULJ97CkqdYq89xGo7FlRsa9tHHfY3Wlw1\\\\\\\/sjb2kNO9l0Po1ii9S0b4BusaB54G\\\\\\\/EedCQa845GVmzzB3pd8WM1IaOi7gQaLBQxZVg0zR9zI\\\\\\\/fOwXvAKQqeJp1OEi0\\\\\\\/ggeHXJk4oArOViVlcLQ+r4XEN9e6DHlqNVxZgBSjhIV+RHQ3V0XLstqTNyYoLskIvy\\\\\\\/kIt8I4mcOwfvXxRdD2YbaVuJIeQs3do5flCm\\\\\\\/DgZjvTVi45Aao9SnPGyvRFamXSt5uNXIvLejrg+uyPWlXHzsLncyg1bwkeu1+SDTe9PLdk8Bbe\\\\\\\/hDQsaEqncpDMSf5wHpuIYFUmdu8HNgVKM+1u2s2ZtU1RBrXYwUpaUqrcqQybqgkxduV\\\\\\\/4KV7k+qoYzMDlBZFtOokcqUSbGebksKOgh9oqiBR2YHdZ0tJnAowC6Wx8CEc2gxQ9c+2NqRNDCQ0ND6rw5W3V\\\\\\\/oMTb7B9Mv5WdUfFbxWG2zaoBZ5uNRfpiPuLMqDb1JSwO00tzsI9nLVe01IWyb2osZST1qXsxB3uBxj8inVHOPk7LkjtGpb2qOIKKuHft4qC1t6DuhkPZXqNeSMmfa1ri4eycI5bBNb2MIrbLGfCVOLrn\\\\\\\/uZzK6rdbzXadjl3zCsjLoUzaP20tw15TzYlbske3eBajEVZ+5OQEQgc4pYlV5twCvYMt7C1eLksFX+lWJWkYLK4vg2C06tIonDT5vvPH6PgxoOhFrR3nU2xl3PZWqJ01dGI038Daxhf4A1T1bNiFoIy8aWlDmWFYJv1RKMczRR9Ho6+QDEX7\\\\\\\/ZlH6QPo1V\\\\\\\/qYdTO4TXmgUi+lYKijzF9NNYwxfz6WBSKV\\\\\\\/Uxz0+CyCbajt\\\\\\\/tDC0wO\\\\\\\/ihen6tQQt9Wy7XonGw3B6a2wGUA03zAI3ZG3vDz+m1ViKS+fHfFxggCV6woTvLjvVeUsoXjv1bmqgu2a8GRpZK09+f9oqet\\\\\\\/piP+lFNfOaXE2hrfwFfsz+MRYnF4GIBgyUK9b988nw7W2hyDb9kn30kNN+MwvIWHdXMCCFqD3hBgjFP91\\\\\\\/jdch3XkrAywKFyXKnxEsjZfuYErpGC6VjHqAZFx8bebwlqf9LQw63fjGJP2DqtCWNMIpgClwXrFYm0jN1oNUkO4iR4p0YXAHY1zz5rSMschNEIqB0U56pukBKpDWijui5Db9lOYW3+u1jRMMJ\\\\\\\/jro2h6V87VCf6IwsA2vvU0+so\\\\\\\/YBHqr4DOq295brNat5D5vrBrYwB4WF4GCpjkUch8tlCDwuGC2+OWiySwDoR6Nwq\\\\\\\/YG0MfqBhNWQrl1vJN9M9cKKypo+uWtl1Cy20Wc16bIDUpcQvhkew7ISaCS2p8MCldYlfwmnRI1\\\\\\\/yWWGvM4Y0cCCyObrmsUreDglci97RnkVvoVplQM40aSAOIBTOLbl34l1iRgwaVzBX1kdX+7i89dxzDPDG\\\\\\\/l5pbqmD7SXDTB7+33Da\\\\\\\/oCAc9l7iuXWcN1\\\\\\\/zvc6ffsYZOGRgKrvOqTiopBwjuk6sy8hcZ6RARtZKc6goMvJt\\\\\\\/707Lh\\\\\\\/pIJ80PClS\\\\\\\/nUGp324yJkGPKP3h2kaagi7\\\\\\\/whlOYZUuWSu2s2Hmzr6i0v4w920151GriWBBYwmni7CtpbLMj1Nw2jmqcp8UGFLZQlm+KH\\\\\\\/Jwm4Sg8tPyXCpkZ1CZwPaKtPJIW\\\\\\\/D64P70IYJFAKpp1zWK6yeULOj4X7aWkYjaF22osIco5spzor0ZkT1sZwV3NFWfWjgWmfH6pCvl3av\\\\\\\/ZUt5tUL7YQ4AsPNRVU7OFSkQOno0CVDSrw1Q9iQQJUNco75vrUaPyj6\\\\\\\/I81LJRE+jSjH8uXj5gzOVURhGH41WIHC3OTup\\\\\\\/z9dVvck\\\\\\\/4E8XY\\\\\\\/NOAoCgvqh8peKje7Q+UsAS8BFs3\\\\\\\/yN3Zly1sAEk0nXTijDRn\\\\\\\/+95IRtYZ1G5vnKQjbkyOXmW0LiuqZzJTUN6o394GgQmgQriVm9Yn\\\\\\\/OmfD19yoUgo1WwcvUDNl7Yhpn6PX4PcZHm9Ol2JMLZ9WUc3ecca70g9B+hXUZaSIWoZL0yEVOCfYAs4plI3NvhyHqUkEWTbXWdMUuI\\\\\\\/G6oBORHG1kZGZGtQLgM7f6muiLv2yy6tf3dVJtYQqu2kBH0miwwXNKht1XC2BYcEQeyH2c22ug83NGJ4PczDK0kZMYT5jF1UMQdRECZMhBdk7a63\\\\\\\/QVHx85D9hfzxi8WTg36KmqEF7LHKWTle7Sr1obzcnMJJ74xIp6Qgg56Gvgnp8E\\\\\\\/hL5dytbi3i+wS9mfIbIWy6wY44nDyVDZFogJQiuu064txmjRdyqAZO2nDXq8WrkQtvJDTf\\\u0022,\\\u0022iv\\\u0022:\\\u0022vNA6mIF0ceqF00PRRbVerQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b4ce3d4b7ce3900a130aec51013b8ca27901d949715b07c3afd36bc9d637e5e1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227+UfDUsEokw4h5L+LwSqi7kPCz5VE6dboGkfJ1rDdHcaDBAqCIcLZ1KOpZxbPUGcLP\\\\\\\/abAnLMOooGQpTcCb45dOywZydQPhiklvWaRTaaRL1GIFUXGnI2E2L4KmyUZR+v6NcCb1LfEBF9H8j00X6m2oKhnxfSN6i\\\\\\\/HWe9Mv3\\\\\\\/QJAk4jxri3rshKe4H0\\\\\\\/LdRW3oID+iCyqDi8bICjKkIkQseErODCedkizI1EqCLJfJnrKOgG7bDfm0X90ZhrSK+zBZRnvdp\\\\\\\/ZrYSrvYC9Vrnz8rXCCite6qt7+ewPGWPTkXkZX\\\\\\\/wnpThBc78l8c0\\\\\\\/7\\\\\\\/EakJKXJHQRhFqKZMagRokmK1asfZs6s+GGigxnuHyOtMU7NNs3IgtXBCH\\\\\\\/xnHndk0IDsg5e0KtyaIz28eQqY8ysa8LX5dMuBhoJwFLIHyTgVvbIv5pSfan1rB46MpGNAJMXRytkk2qab021iLyUFINzDQ\\\\\\\/cj4vNEvg1rVcDq9wFon6nHkmhI2+qlpoy+lT+VJgmhgPLrQwH2VAKyH2gcVh5Rk7XcX3dr1cTLGSgwQ8ymuURVCC8AKvTn7btBGb4jxIWwv14Yj3sfEK7SlDO791oKufMI\\\\\\\/oWjz3yyMjVsJsKLMy3sbVyzuigi3c5xVSY4hTU\\\\\\\/JfQdZMOP8VyrxsQsvd+F9MwPO\\\\\\\/7H3kOL0cATW\\\\\\\/YdOMePAV99mcd1P9Il0rTr9WHxi1peTyUQaaJr6QL2B8f+LX\\\\\\\/gqbn59NH7K6FsRPw8TkNwxUF3E75vlqyzcMWkQYViVGPiNQso9OY2RPZ9rAu6hYuvH4vF4pO7FZADSAgCKIU4p4z\\\\\\\/g90pQhdO7D4sG+P0MQGZqFF0m3GNhzYFalB4hO5cy0\\\\\\\/ESdrWeQZeErFTy28yuMcCqUmzE\\\\\\\/1RTSmtQEifD03zfaNWLYLJYg06S4vxnA3sAH1XqftIJn2qTuTKc++lZGN9jjYPCCVCb4LtiqI9Z8docMVKUjtTbfIsuMRetqNx+zwBtQftRour+jkhCOLgGCuCob03wxhYeTv+XElA4QiwAdTt7f1Q\\\\\\\/pAsG+AnCtcPfJVyoFVB+m1TRjXTzzLMuubH0o5jB4jPP6yHk52n0U+r5pTefcZN1P0AXkU9gq3zyeo56c3K7pGIxhPhzueahI9k5aa4dd6RDRARh7LzHugb1L5n6WWblNCJ7tDb+PiLwSEyRxUOUxE+QP7zlwnip46QiBw9XXaOdUu9z4dDDdbIreKLcoyNFedsHRQUIJSx8Xo0IJYSaW5Mf03vlxC0uimuwjyTVXiuxbrFY0AEEjTMSXPFbQGrQfKKPwU3x2+rOTD4C7czqgx2fJzVtXAj5q3gb18pEn1Xei0qCN87NckK7mlo4MS25r8wPkwl\\\\\\\/tdOULu6ztDvgkxnW\\\\\\\/gjKvZGXuIaRKSIr9xI+0JACsowhbvP2u2KFqJAhy9enLk6uLtMw3xlz8ibVV26vZiu7Auwx0NDXI9Xd7Ufi\\\\\\\/h7VGgzlwqUHDi0y2Xmb\\\\\\\/MFmyue98BzlPyLqxJpvocwE40bz5flkM4Py\\\\\\\/\\\\\\\/cB3KEo68k7s835uN+HJUMCaU98d7ukUc35yyTXissGdRz1YicQ9FI9vazkaBeydMsvNeQVY1lvMNLkfRd7pYHQr86d0Nyz+uRLm6RE\\\\\\\/XFcWvVR0B34QYNRIW6HV+6JWst5k3GUH10LrgjDy2ElqySP9WQzoAF4TIV6QuOSFi6SXTfxBdWoXOydzwrj+eLX0p+PGaqS+HCS6xlDnIg7fgn4cY2jJWMw613rLWt\\\\\\\/is+u788Ly+8SQdMXPX0rmfvhhkettoCLVezM7GTtSFMOrPNXSCcnf+I57QAURmbXWrR+J2RYw8qiYJvZaQjPcQlaOmpxcZRkODNBXsCiqb4SDqNi9Ddf+LbkgSXNwOJNtYuiZv401Ijj5HA7chgKitumoBaNcmkcW6JV4Jd2AEwEq8R5HuV8Kf4R3RY6t6ijaBFq81pJzUMb\\\\\\\/vrq56FHQi0BReRdnl4UxiVmtQ507vuE6lQVXHhycvnwOs46P\\\\\\\/gFjKGz3EUVpZkD8v8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022tHg+hcdolCoC0K2S5kPNZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d9e30a633f29ad72c91a2727ea606d9f6a743aa59ffba2ee6974a10ca8a0f6f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483838 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1277" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-F58D9gRafu3GLSmCrbcA7DjY0wjELVbS033UmSZDqW0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1277" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-12gkcSVDMSk4wJzPC4IcLLZiOiFwvutYsDn-vnXLNI4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1267" title="Genetic Adaptation of Schizothoracine Fish to the Phased Uplifting of the Qinghai–Tibetan Plateau" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1287" title="Independent Maternal and Fetal Genetic Effects on Midgestational Circulating Levels of Environmental Pollutants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483838" id="node483838" data-pisa="ggg;7/4/1277" data-pisa-master="ggg;g3.117.039883" data-apath="/ggg/7/4/1277.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Automated Phenotyping Indicates Pupal Size in <em>Drosophila</em> Is a Highly Heritable Trait with an Apparent Polygenic Basis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9454-6175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Guy Reeves</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0460-5344" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diethard Tautz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1277-1286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039883 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483838" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. Guy Reeves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BGuy%2BReeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reeves%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BGuy%2BReeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9454-6175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Guy Reeves</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reeves@evolbio.mpg.de" class="" data-icon-position="" data-hide-link-title="0">reeves@evolbio.mpg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diethard Tautz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diethard%2BTautz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tautz%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiethard%2BTautz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0460-5344" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diethard Tautz</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/4/1277" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483838" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1277.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483838" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1277.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483838" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/4/1277.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483838" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/4/1277.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483838" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The intense focus on studying human height has done more than any other genetic analysis to advance our understanding of the heritability of highly complex phenotypes. Here, we describe in detail the properties of a previously unexplored trait in <em>Drosophila melanogaster</em> that shares many salient properties with human height. The total length of the pupal case varies between 2.8 and 3.9 mm among natural variants, and we report that it is among the most heritable traits reported in this species. We have developed a simple semiautomatic phenotyping system with which a single operator can reliably score &gt;5000 individuals in a day. The precision of the automated system is 0.042 mm (± 0.030 SD). All phenotyped individuals are available to be mated in subsequent generations or uniquely archived for future molecular work. We report both broad sense and narrow sense heritability estimates for two biologically distinct data sets. Narrow sense heritability (<em>h</em><sup>2</sup>) ranged from 0.44 to 0.50, and broad sense heritability (<em>H</em><sup>2</sup>) ranged from 0.58 to 0.61. We present results for mapping the trait in 195 recombinant inbred lines, which suggests that there are no loci with &gt;10% effect size in this panel. We propose that pupal size genetics in <em>Drosophila</em> could represent a model complex trait amenable to deep genetic dissection using the automated system described.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/automated-phenotyping" class="hw-term hw-article-keyword hw-article-keyword-automated-phenotyping" rel="nofollow">automated phenotyping</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-melanogaster" class="hw-term hw-article-keyword hw-article-keyword-drosophila-melanogaster" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila melanogaster</em></a></li><li class="kwd"><a href="/keyword/pupal-length" class="hw-term hw-article-keyword hw-article-keyword-pupal-length" rel="nofollow">pupal length</a></li><li class="kwd"><a href="/keyword/human-height" class="hw-term hw-article-keyword hw-article-keyword-human-height" rel="nofollow">human height</a></li><li class="kwd"><a href="/keyword/complex-trait" class="hw-term hw-article-keyword hw-article-keyword-complex-trait" rel="nofollow">complex trait</a></li></ul><p id="p-3">Elucidating the genetic architecture of a phenotypic trait fundamentally requires that it is, to some degree, heritable. Phenotypic traits with low heritability do not generally require additional explanation as to why quantitative trait loci (QTL) cannot be robustly identified. However, while traits with high heritability increase the power to identify causative loci, the genetic architecture of traits remains a key factor in predicting success in identifying QTL. For example, a highly heritable trait that is dependent on a large number of interacting alleles or loci may still require substantial sample sizes and complex analyses to identify causative loci (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Visscher <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barton <em>et al.</em> 2016</a>). The archetypical example of this is human height, which, by at least two distinct measures of heritability, is among the most heritable of quantitative human traits, <em>h</em><sup>2</sup> ≈ 0.4–0.7 using regression of trios (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Galton 1886</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Hanley 2004</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Visscher <em>et al.</em> 2010</a>) or <em>h</em><sup>2</sup> ≈ 0.7 using twin studies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Polderman <em>et al.</em> 2015</a>). However, the identification of the loci involved, and any interactions among them, has proven far from trivial, and successes to date have been dependent on utilizing very large data sets [up to 250,000 individuals (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wood <em>et al.</em> 2014</a>)]. The current understanding being that of the 180–4000 loci implicated in impacting human height over its typical range all have additive effect sizes of &lt;&lt;1 mm (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Weedon <em>et al.</em> 2008</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Visscher <em>et al.</em> 2010</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Wood <em>et al.</em> 2014</a>). Although studies of human height represent a valuable model with which to test how complex heritable phenotypes can be dissected, the ability to experimentally control allele frequencies and environmental conditions would be a valuable capacity that is not possible in human studies. While this is a possibility in model organisms, there are relatively few morphological traits that have been robustly demonstrated to be both highly heritable and amenable to automated high throughput phenotyping (though see below). Herein, we describe in detail a new quantitative trait—length of the pupal case—in <em>Drosophila melanogaster</em> that places it in the 15–20% most heritable morphological traits described in this species, and is amenable to reliable and automated high-throughput phenotyping (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Roff and Mousseau 1987</a>).</p><p id="p-4">The utility of automated phenotyping systems for a wide variety of organisms has increased greatly over the last 10 yr. While automated morphological phenotyping systems have been developed for <em>Drosophila</em>, as yet only automation analysis of images of wings has been widely employed (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Houle <em>et al.</em> 2003</a>). However, acquiring the wing images requires manual manipulation to position flies individually and so is difficult to scale up. Likewise, a system to measure heartbeat function requires that each fly be manipulated into position. An alternative approach to phenotyping in a selection experiment for gross body size was achieved using a series of graduated sieves (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Turner <em>et al.</em> 2011</a>). More recently, a sophisticated platform called the “fly cat walk” was described that has the capacity to reliably phenotype 700 flies a day for a wide range of morphological traits simultaneously (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Medici <em>et al.</em> 2015</a>). This requires no user manipulation of individual flies and is nondestructive. While the cost of constructing the equipment is not detailed (<a href="https://github.com/IMSB/FlyCatwalk/">https://github.com/IMSB/FlyCatwalk/</a>), it is likely that it would represent a significant investment of time, expertise, and resources for most laboratories.</p><p id="p-5">Our setup uses an inexpensive camera in a light-proof box and the open source image analysis software Cellprofiler, with which a single user can phenotype 5000 pupae in a day. Pupae are photographed <em>in situ</em> on flattened squares of transparent film that lined the entire vertical surface of the vials. We demonstrate how the system can be used to (1) measure panels of recombinant inbred lines (RILs), and (2) generate large numbers of parent offspring trios that are particularly useful in exploring complex traits using artificial selection techniques. Furthermore, the increased throughput facilitates exploring the heritability of family means rather than single individual measurements, which are associated with increased measurement variance. Hence, pupal size could become a model phenotype that will allow deep dissection of its genetic architecture, since the availability of automated phenotyping will allow to screen very large mapping panels, or to design new complex mapping strategies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly stocks</h3><p id="p-6">The automated phenotyping system was applied to two independent datasets.</p><p id="p-7">The first dataset, referred to here as “eight-way” is a collection of 195 RILs that are part of the <em>Drosophila</em> Synthetic Population Resource (<a href="http://wfitch.bio.uci.edu/~dspr/">http://wfitch.bio.uci.edu/∼dspr/</a>). These RILs are all originally derived from a cross between eight global stocks of <em>D. melanog</em>aster, their generation is described in detail in <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">King <em>et al.</em> (2012a</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">b</a>). Narrow sense heritability, <em>h</em><sup>2</sup>, for the eight-way dataset was estimated by measuring the progeny of 67 single pair matings between individuals from 11 different RILs that spanned the full range of phenotype measurements (IDs of crossed stocks are given in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS5.zip">File S5</a>; six of the 67 crosses are duplicates also indicated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS5.zip">File S5</a>). Broad sense heritability estimates <em>H</em><sup>2</sup> of the eight-way dataset were generated by repeated measurements of RILs (IDs given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS5.zip">File S5</a>).</p><p id="p-8">The second dataset, termed “four-way,” was initiated by a cross between two Japanese and two African stocks (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/TableS1.pdf">Table S1</a>). Narrow sense heritability, <em>h</em><sup>2</sup>, was estimated by measuring the progeny of single pair matings from the 2nd to 6th generation where phenotyped parents were selected randomly from different vials to form subsequent generations. The following number of single pairs were measured in each generation [G2, 15 pairs], [G3, 81 pairs], [G4, 78 pairs], [G5, 88 pairs], and [G6, 154 pairs]. No duplicate vials were generated from the same mating pair. Note that, unlike the eight-way dataset, levels of heterozygosity in the parents and their offspring are likely to be similar. <em>H</em><sup>2</sup> estimates of the four-way dataset were generated by measuring 83 RILs established from the four-way cross by generation 37 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS5.zip">File S5</a> for stock IDs). Details of all progenitor stocks of the four-way, and four-way datasets with estimates of their pupal length, are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Image acquisition</h3><p id="p-9">Flies were maintained on standard food dispensed into 28.5 mm diameter, 95 mm height vials (Genesee Scientific). Once the food vials had fully cooled, 10 cm × 10.5 cm squares of overhead projector film were slid into each vial lining their entire vertical wall (nobo, plain paper copier film, 33638237). A more detailed description of the entire procedure and equipment set up is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS1.docx">File S1</a>. A custom printed semitransparent label, including a unique barcode, was affixed to the outside of each vial. Vials were incubated at 24° in a 12 hr light/12 hr dark incubator. Adults were removed after one to two nights in vials (sometimes three to four nights, if fertility appeared to be low, or due to holidays). Generally, by the 10th day after the parents were initially introduced, the majority of offspring in the vials were present as pupae attached to the transparent film; few if any larvae remained in the food. The film from each vial was removed, and placed into a purpose-made plastic frame (this frame can be 3D printed using a file provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS2.zip">File S2</a>) that holds the film flat for photographing. Food from the lower part of the film was scraped away and any larvae, or any white puparium stage (P1), removed. The frame was then photographed using bottom illumination in a light tight box. Batches of the resulting images were then analyzed using the procedure below.</p></div><div id="sec-4" class="subsection"><h3>Automated image analysis</h3><p id="p-10">A Cellprofiler (v2.1.0) pipeline was developed to simultaneously recognize pupae and to measure a variety of attributes, including length. The outputs of all measurements are written to .xls files that can then be viewed or imported into any database program. Cellprofiler is free, open access, software providing a suite of flexible image analysis tools (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lamprecht <em>et al.</em> 2007</a>). In brief, the Cellprofiler pipeline first identifies “primary objects” distinct from the background without restriction on their size (module: identify primary objects). Then, applying a scalable model of pupal shape to all objects, those that are composed of multiple touching pupae are separated into distinct pupae [module: Untangle Worms, (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Wählby <em>et al.</em> 2012</a>)]. The resulting putative pupae are then each shrunk, and then repropagated outwards to more precisely identify the edges of each pupa based on boundary changes in pixel intensity (module: Identify Secondary objects). Finally, pupae are crudely filtered on size attributes and the proximity of neighboring objects to place them in one of the three confidence classes described in the <em>Results</em>. The digital outlines of pupae are overlaid onto a cropped version of the original image to allow users to easily visually assess quality.</p><p id="p-11">Overlaid images and files of measurements were imported in batches into the database program FileMaker (v14, FileMaker Inc.). A unique barcode sticker identifying the film in the image was automatically read by the database. This enabled the image files to be automatically renamed with the barcode as their name for archiving. In addition to pupal measurements, a 1 €cent coin (16.25 mm diameter) present in all images was measured to control for camera changes, and to allow conversion of measurement in pixels to millimeter. Quality filtering of pupae and basic analyses were performed using the database. An annotated copy of the Cellprofiler pipeline is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS3.zip">File S3</a>.</p></div><div id="sec-5" class="subsection"><h3>Estimates of heritability</h3><p id="p-12"><em>H</em><sup>2</sup> was estimated using SPSS version 22 with the “variance components” function, with pupal length as the dependent variable, and RIL name as a random factor. This was done using the Minimum Norm Quadratic Unbiased method (though ANOVA produced identical results to two decimal places). The model for the single factor was <em>y<sub>ij</sub></em> <em>=</em> <em>μ</em> + <em>α<sub>j</sub></em> <em>+e<sub>ij</sub></em>, where <em>y<sub>ij</sub></em> is the <em>j</em>th observation of the <em>i</em>th RIL, <em>μ</em> is the overall mean, <em>α<sub>j</sub></em> is the random factor, and <em>e</em> is the associated error.</p><p id="p-13">Estimates of <em>h</em><sup>2</sup> were made using the “linear regression” function, with pupal length as the dependent variable, and parent-midpoints as an independent variable. All other statistical analyses and graphs were also performed using SPSS or Filemaker. All data for individual pupa are available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS4.zip">File S4</a>, and for vial means as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS5.zip">File S5</a> (fields used in each graph are indicated in the associated “readme” files).</p></div><div id="sec-6" class="subsection"><h3>Human height data</h3><p id="p-14">Estimates of <em>h</em><sup>2</sup> for human height data were made using the same methods as for pupae, using the original data of Francis Galton for 898 individuals, transcribed from his 1880s laboratory notebooks (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Hanley 2004</a>) (available <a href="http://www.math.uah.edu/stat/data/Galton.html">http://www.math.uah.edu/stat/data/Galton.html</a>). The 760 individuals in 123 families where four or more individuals were measured were used for estimates of the <em>h</em><sup>2</sup> heritability of mean family height (females were not transmuted into males).</p></div><div id="sec-7" class="subsection"><h3>Genome scan for loci of large effect impacting pupal length using eight-way RILs</h3><p id="p-15">Scans were performed in R (version 3.2.2) for the 195 DSPR RILs mentioned above (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">King <em>et al.</em> 2012a</a>,<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">b</a>) using the “DSPRscan” command within the DSPRqtl-tools procedures described in <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">King <em>et al.</em> (2012a</a>,<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">b</a>). These utilize one SNP per 10 kb throughout the genome reference (with the exception of the Y chromosome and the mitochondrial genome). Significance LOD score thresholds were estimated using the “DSPRperm” command with 1000 replicates. The data input file is available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS6.txt">File S6</a>. This scan was undertaken to asses whether one or few loci of large effect control pupal length, <em>i.e.</em>, the trait is not complex. Note that, if the trait is controlled by many loci of small or moderate effect size, the scan is too underpowered to realistically identify individual loci. However, the goal of the scan is to show that pupal size is not a relatively simple trait controlled by only a few major effect loci.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-16">All starting fly stocks are available from the sources detailed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/TableS1.pdf">Table S1</a>. RILs are available from the web address given above. Files of raw data are available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS4.zip">File S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS5.zip">File S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS6.txt">File S6</a>. </p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><p id="p-17">The large numbers of individuals and vials measured, made possible through the use of the automated measuring system, provides robust insights into how phenotypic variation is partitioned for pupal length. While the method automatically identifies and measures pupae for multiple different parameters (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS1.docx">File S1</a> for a full list), the length of the pupae was the principle focus of this study. Two distinct sets of biological material were used to define the biological properties of pupal length. The first dataset was all derived from an eight-way cross (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">King <em>et al.</em> 2012a</a>,<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">b</a>); the second was from a four-way cross. None of the founding stocks are shared between the eight-way and the four-way datasets. <em>H</em><sup>2</sup> (reflecting all potentially genetic contributions to trait variance <em>e.g.</em>, additive, epistatic, dominance, maternal, and paternal effects) was estimated through the analysis of repeated measurements of RILs, whereas <em>h</em><sup>2</sup> heritability (reflecting only the impact of additive genetic effects) was measured by regression of midparent against their offspring measurements. Sampling properties of the eight-way and four-way datasets are detailed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> (see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS1.pdf">Figure S1</a> for summary of repeat measurements of RILs).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484026/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484026/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of sampling of eight-way and four-way datasets</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sampling distributions of pupal length for individuals and vials. (A) Distributions for individual pupal lengths: four-way dataset (blue) n&#xA0;=&#xA0;47,843 and eight-way dataset (green) n&#xA0;=&#xA0;86,515. (B) Corresponding distributions for vial mean pupal lengths: four-way dataset (blue) n&#xA0;=&#xA0;799 vials and eight-way dataset (green) n&#xA0;=&#xA0;1251 vials. Only vials where &#x2265;15 pupae were measured by the automated system are considered. A stock possessing the Tb1 mutation, resulting in the well-known tubby pupal phenotype (Bloomington stock 3644), was measured and found to have a mean pupal length of 2.7&#xA0;mm, which is at the lower bounds of the smallest wildtype individuals or vials shown here." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sampling distributions of pupal length for individuals and vials. (A) Distributions for individual pupal lengths: four-way dataset (blue) &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;47,843 and eight-way dataset (green) &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;86,515. (B) Corresponding distributions for vial mean pupal lengths: four-way dataset (blue) &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;799 vials and eight-way dataset (green) &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1251 vials. Only vials where &#x2265;15 pupae were measured by the automated system are considered. A stock possessing the Tb&lt;sup&gt;1&lt;/sup&gt; mutation, resulting in the well-known tubby pupal phenotype (Bloomington stock 3644), was measured and found to have a mean pupal length of 2.7&#xA0;mm, which is at the lower bounds of the smallest wildtype individuals or vials shown here.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F1.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/4/1277/F1.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484054" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Sampling distributions of pupal length for individuals and vials. (A) Distributions for individual pupal lengths: four-way dataset (blue) <em>n</em> = 47,843 and eight-way dataset (green) <em>n</em> = 86,515. (B) Corresponding distributions for vial mean pupal lengths: four-way dataset (blue) <em>n</em> = 799 vials and eight-way dataset (green) <em>n</em> = 1251 vials. Only vials where ≥15 pupae were measured by the automated system are considered. A stock possessing the Tb<sup>1</sup> mutation, resulting in the well-known tubby pupal phenotype (Bloomington stock 3644), was measured and found to have a mean pupal length of 2.7 mm, which is at the lower bounds of the smallest wildtype individuals or vials shown here.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-10" class="subsection"><h3>Performance of automated pupal phenotyping</h3><p>The automated pupal recognition pipeline attempts to identify the external outlines of three classes of objects:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-21"><em>High confidence pupae</em>: conform to a set of expected properties and that have no neighboring pupae within 20 pixels (≈0.7 mm, Cellprofiler module: filter objects)</p></li><li id="list-item-2"><p id="p-22"><em>Medium confidence pupae</em>: conform to the same set of properties but with neighboring objects closer than 20 pixels. This often occurs as larvae select pupation sites touching each other, resulting in a more challenging target for image recognition</p></li><li id="list-item-3"><p id="p-23"><em>Low quality objects</em>: unlikely to be pupae.</p></li></ul><p>Examples of the three classes are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS1.docx">File S1</a> for further details).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visual output of automated phenotyping system. Blue outline = high confidence pupae, red outline = medium confidence pupae, and yellow outline = low quality objects. A small proportion of pupae are entirely missed by the automated system (green arrow). Others are aberrantly measured due to close proximity to other pupae resulting in truncation or extension of their outline. Even those with a high confidence assignment can also appear malformed (orange arrow). They should be manually excluded from analysis to reduce noise (see File S1 for more examples of aberrant and canonical pupae). All objects are numbered in Cellprofiler output files to permit numerical measurements to be easily related to images." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Visual output of automated phenotyping system. Blue outline = high confidence pupae, red outline = medium confidence pupae, and yellow outline = low quality objects. A small proportion of pupae are entirely missed by the automated system (green arrow). Others are aberrantly measured due to close proximity to other pupae resulting in truncation or extension of their outline. Even those with a high confidence assignment can also appear malformed (orange arrow). They should be manually excluded from analysis to reduce noise (see File S1 for more examples of aberrant and canonical pupae). All objects are numbered in Cellprofiler output files to permit numerical measurements to be easily related to images.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F2.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/4/1277/F2.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Visual output of automated phenotyping system. Blue outline = high confidence pupae, red outline = medium confidence pupae, and yellow outline = low quality objects. A small proportion of pupae are entirely missed by the automated system (green arrow). Others are aberrantly measured due to close proximity to other pupae resulting in truncation or extension of their outline. Even those with a high confidence assignment can also appear malformed (orange arrow). They should be manually excluded from analysis to reduce noise (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS1.docx">File S1</a> for more examples of aberrant and canonical pupae). All objects are numbered in Cellprofiler output files to permit numerical measurements to be easily related to images.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Of the pupae retained for analysis, 47 and 53% were high and medium confidence, respectively. The probability of pupae being manually excluded as aberrantly measured was 0.02 for high quality pupae and 0.10 for medium confidence pupae (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Note that not all images were examined manually, as, due to the robustness of the automated estimates, generally only vials with atypically high variance of the mean were examined.</p><p id="p-26">The precision of the automated system was assessed by remeasuring a subset of films after rotating them by 180°, and comparing 516 duplicated measurements of the same pupae (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The difference between the two measurements was an average of 0.043 mm (±0.030 SD). Remeasuring pupae after delays of 15 or 30 hr also generated similar precision estimates, indicating that, once pupae become brown puparium (P2), there is no detectable change in pupal length. Furthermore, the exuvia of eclosed individuals can also be reliably measured (data not shown).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Precision of automated pupal length estimates. Comparison of pupal length measurements of films in the normal orientation compared to where the same film was rotated by 180&#xB0; All pupae isolated from others by up to 15 pixels in any direction were used (n&#xA0;=&#xA0;516 pupae, across 10 vials). Slope&#xA0;=&#xA0;0.94 and R2&#xA0;=&#xA0;0.98. An x&#xA0;=&#xA0;y line is shown for reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Precision of automated pupal length estimates. Comparison of pupal length measurements of films in the normal orientation compared to where the same film was rotated by 180&#xB0; All pupae isolated from others by up to 15 pixels in any direction were used (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;516 pupae, across 10 vials). Slope&#xA0;=&#xA0;0.94 and &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.98. An &lt;em&gt;x&lt;/em&gt;&#xA0;=&#xA0;&lt;em&gt;y&lt;/em&gt; line is shown for reference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F3.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/4/1277/F3.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483963" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Precision of automated pupal length estimates. Comparison of pupal length measurements of films in the normal orientation compared to where the same film was rotated by 180° All pupae isolated from others by up to 15 pixels in any direction were used (<em>n</em> = 516 pupae, across 10 vials). Slope = 0.94 and <em>R</em><sup>2</sup> = 0.98. An <em>x</em> = <em>y</em> line is shown for reference.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>The count of automatically measured pupae is a reasonable proxy for density within vials</h3><p id="p-28">Unsurprisingly, not all pupae were correctly captured by the automated system, and, based on a sample of 148 vials, an average 20% ± 11.7 SD pupae per vial were missed, called as low confidence objects, or manually excluded (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). While there is an increased variance in the proportion of unmeasured pupae in higher density vials (<em>e.g.</em>, where the automated count is &gt;80), even here the automated count provides a reasonable proxy for density of individuals in each vial. This assumes that few, if any, larvae remained on the food surface after film transfer, as was generally the case (due in part to the short period parents remained in the vials). In addition, the small number of pupae removed from films that were obscured by larval food represents a constant proportion across all vials (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS1.docx">File S1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of automated count of pupae per vial vs. manual counting; 148 vials from across the entire range of the automated densities observed were manually recounted. While at higher densities the accuracy of the automated count decreases, it remains a reasonable proxy for vial density across the entire observed range. Slope&#xA0;=&#xA0;0.81 and R2&#xA0;=&#xA0;0.92. An x&#xA0;=&#xA0;y line is shown for reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of automated count of pupae per vial &lt;em&gt;vs.&lt;/em&gt; manual counting; 148 vials from across the entire range of the automated densities observed were manually recounted. While at higher densities the accuracy of the automated count decreases, it remains a reasonable proxy for vial density across the entire observed range. Slope&#xA0;=&#xA0;0.81 and &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.92. An &lt;em&gt;x&lt;/em&gt;&#xA0;=&#xA0;&lt;em&gt;y&lt;/em&gt; line is shown for reference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F4.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/4/1277/F4.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483956" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Comparison of automated count of pupae per vial <em>vs.</em> manual counting; 148 vials from across the entire range of the automated densities observed were manually recounted. While at higher densities the accuracy of the automated count decreases, it remains a reasonable proxy for vial density across the entire observed range. Slope = 0.81 and <em>R</em><sup>2</sup> = 0.92. An <em>x</em> = <em>y</em> line is shown for reference.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Impact of vial density on pupal length</h3><p id="p-30">One common major environmental covariate of many <em>Drosophila</em> traits is density of individuals within the vial in which they develop. Consequently, many <em>Drosophila</em> researchers control for this in experiments by collecting large numbers of zygotes and placing a controlled number in each vial. This is a fairly laborious process to perform routinely, and is complicated considerably where single pair crosses are required.</p><p id="p-31">Here, density was controlled only indirectly through limiting the number of parents used per vial, and restricting the number of nights they remained before being cleared [generally two nights for single pair crosses, and one night for small groups (<em>n</em> = 10–20) of RIL individuals, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FileS1.docx">File S1</a>]. Throughout all experiments, two stocks were continually remeasured to act as controls (stock 335 and 329, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/TableS1.pdf">Table S1</a>). The large number of repeat measurements of these two stocks across a range of densities permits the examination of any relationship between density and pupal length (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Furthermore, in the same way, it is also possible to explore the relationship between density and pupal length using the repeated eight-way and four-way RIL measurements (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS2.pdf">Figure S2</a>). All RILs and control stocks exhibit a uniformly negative relationship between density and pupal length; the slope varies from −0.0006 to −0.0041. This observed variability may reflect either variance in estimating slopes, or also that RILs exhibit different reaction norms. If the mode slope of −0.002 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS2.pdf">Figure S2</a>) is used to correct the mean length of vials (or individuals) to that observed at the mean observed vial density, this can be achieved with the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1277/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/4/1277/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Applying this formula to the 431 vials established as single pair crosses, &gt;99% of them would require a correction of &lt;±0.1 mm, and, of the 1620 vials established from small groups of RIL individuals, 95% would require a correction of &lt;±0.118 mm (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Given the modest number of vials subjected to a large correction, and uncertainly about whether there is a truly universal linear relationship between density and pupal length, none of the measurements presented in the manuscript have been corrected analytically for density. Furthermore, with respect to repeated RIL measurements, reducing any confounding impact of density can be achieved by either experimentally increasing the number of replicate measurements closer to the mean density, or, analytically, by excluding or weighting down vials with extreme density values.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between vial density and pupal length for the two control stocks. Repeated measurements of the same stocks (one short and one long) at different densities permits the slope and correlation of the relationship to be estimated. While there is a high degree of correlation for the longer stock-329 (red circles) (R2&#xA0;=&#xA0;0.48 and P&#xA0;=&#xA0;&gt;0.001), the correlation for the shorter stock-335 (black triangles) is minimal (R2&#xA0;=&#xA0;0.07 and P&#xA0;=&#xA0;0.07)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between vial density and pupal length for the two control stocks. Repeated measurements of the same stocks (one short and one long) at different densities permits the slope and correlation of the relationship to be estimated. While there is a high degree of correlation for the longer stock-329 (red circles) (&lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.48 and &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;&gt;0.001), the correlation for the shorter stock-335 (black triangles) is minimal (&lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.07 and &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.07).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F5.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/4/1277/F5.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483888" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Relationship between vial density and pupal length for the two control stocks. Repeated measurements of the same stocks (one short and one long) at different densities permits the slope and correlation of the relationship to be estimated. While there is a high degree of correlation for the longer stock-329 (red circles) (<em>R</em><sup>2</sup> = 0.48 and <em>P</em> = &gt;0.001), the correlation for the shorter stock-335 (black triangles) is minimal (<em>R</em><sup>2</sup> = 0.07 and <em>P</em> = 0.07).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Magnitude of potential correction of pupal length for vial density, using Equation 1. The more uniform vial densities from single pair matings (A) resulted in all but one vial being within 50 pupae of the mean observed density of 65, corresponding to a maximum correction for density of 0.1&#xA0;mm (n&#xA0;=&#xA0;431 vials). The increased variance of the vial densities resulting from establishing vials from small groups of RIL individuals (B) led to &#x223C;5% of vials being corrected by &gt;0.1&#xA0;mm (n&#xA0;=&#xA0;1620 vials). A calculated mean density (M) of 65 was used for all vials. A slope value of S&#xA0;=&#xA0;&#x2212;0.002 was used, if a steeper slope is used then there would be a corresponding increase in the magnitude of the corrections. All vial with a density of " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Magnitude of potential correction of pupal length for vial density, using Equation 1. The more uniform vial densities from single pair matings (A) resulted in all but one vial being within 50 pupae of the mean observed density of 65, corresponding to a maximum correction for density of 0.1&#xA0;mm (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;431 vials). The increased variance of the vial densities resulting from establishing vials from small groups of RIL individuals (B) led to &#x223C;5% of vials being corrected by &gt;0.1&#xA0;mm (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1620 vials). A calculated mean density (&lt;em&gt;M&lt;/em&gt;) of 65 was used for all vials. A slope value of &lt;em&gt;S&lt;/em&gt;&#xA0;=&#xA0;&#x2212;0.002 was used, if a steeper slope is used then there would be a corresponding increase in the magnitude of the corrections. All vial with a density of &lt;15 were excluded.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F6.medium.gif" width="440" height="191"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/4/1277/F6.medium.gif" width="440" height="191"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483869" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-33" class="first-child">Magnitude of potential correction of pupal length for vial density, using Equation 1. The more uniform vial densities from single pair matings (A) resulted in all but one vial being within 50 pupae of the mean observed density of 65, corresponding to a maximum correction for density of 0.1 mm (<em>n</em> = 431 vials). The increased variance of the vial densities resulting from establishing vials from small groups of RIL individuals (B) led to ∼5% of vials being corrected by &gt;0.1 mm (<em>n</em> = 1620 vials). A calculated mean density (<em>M</em>) of 65 was used for all vials. A slope value of <em>S</em> = −0.002 was used, if a steeper slope is used then there would be a corresponding increase in the magnitude of the corrections. All vial with a density of &lt;15 were excluded.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Variance in the estimates of vial means</h3><p id="p-34">A script was written in Filemaker to explore the extent to which estimates of mean vial pupal length based on subsamples of individuals within a vial deviate from the vial means based on all individuals. This provides insight into at what point vials with low densities generate mean estimates with unacceptably high variance. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> indicates that selecting single individuals to estimate vial means unsurprisingly results in a high degree of variance of up to 0.4 mm. While these estimates are unbiased, this is likely to prove unacceptably high given that the total observed range of pupal lengths is 1.1 and 0.8–0.9 mm within datasets. However, for random subsample sizes equal to a density of ≥15, the variance is greatly reduced to &lt;0.07 mm for 95% of vials (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). On this basis, only vials with &gt;15 measured pupae were included for analysis or presentation throughout this study.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deviation of mean vial pupal length of random subsamples compared to that based on all individuals within a vial. Each vial was resampled selecting only a subsample of the pupal length measurements within a vial to calculate a subsample mean length that was then subtracted from the mean length based on all sampled individuals. Each vial was resampled 100 times per subsample size. Whiskers represent range of 95% of subsample deviations, and circles the mean deviation of 100 replicates. Based on 729 vials, with densities ranging from 51 to 60 measured pupae." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Deviation of mean vial pupal length of random subsamples compared to that based on all individuals within a vial. Each vial was resampled selecting only a subsample of the pupal length measurements within a vial to calculate a subsample mean length that was then subtracted from the mean length based on all sampled individuals. Each vial was resampled 100 times per subsample size. Whiskers represent range of 95% of subsample deviations, and circles the mean deviation of 100 replicates. Based on 729 vials, with densities ranging from 51 to 60 measured pupae.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F7.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/4/1277/F7.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483939" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child">Deviation of mean vial pupal length of random subsamples compared to that based on all individuals within a vial. Each vial was resampled selecting only a subsample of the pupal length measurements within a vial to calculate a subsample mean length that was then subtracted from the mean length based on all sampled individuals. Each vial was resampled 100 times per subsample size. Whiskers represent range of 95% of subsample deviations, and circles the mean deviation of 100 replicates. Based on 729 vials, with densities ranging from 51 to 60 measured pupae.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>The impact of using smaller numbers of vials to estimate RIL mean length</h3><p id="p-36">In measuring panels of RILs, it is useful, for practical reasons, to minimize the number of replicate vials measured for each RIL to estimate an RIL mean with an acceptable degree of variance. Using eight RILs and the two control lines, all of which were measured 12 times or more, it is possible to explore the relationship between RIL means and the number of replicate vial measurements used. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> indicates that six replicated measurements generally result in 95% of RIL estimates being &lt;0.1 mm different from that based on larger numbers of replicate vial measurements. The mean number of replicate vial measurements per RIL in this study was 6.4 ± 2.1 SD (5.8% = the minimum four replicates, 18.0% = five replicates, 61.2% = six replicates, and 15% more than six replicates).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deviation of mean RIL pupal length utilizing random subsamples of vials compared to that based on all measured vials. For all eight RIL, and two control stocks with 12 or more vial replicate measurements, the RIL mean pupal length was recalculated using a random subsample of vials. Each RIL was resampled 100 times per number of vial subsample size used. Whiskers represent range of subsample means, circles represent the overall means. (A) Range of 95% of subsamples; (B) 50% of subsamples. Total number of replicate vial measurements per RIL were as follows: RILs 11,229, 11,257, 11,259, 11,265, and 11,021&#xA0;=&#xA0;12 replicates; 11,210&#xA0;=&#xA0;13 replicates; 11,236&#xA0;=&#xA0;14 replicates; 11,237&#xA0;=&#xA0;15 replicates; 335&#xA0;=&#xA0;31 replicates; and 329&#xA0;=&#xA0;64 replicates. Note all pupae in vials were used to calculate each vial length mean, only the number of vial means in each subsample was varied in estimating length RIL means." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Deviation of mean RIL pupal length utilizing random subsamples of vials compared to that based on all measured vials. For all eight RIL, and two control stocks with 12 or more vial replicate measurements, the RIL mean pupal length was recalculated using a random subsample of vials. Each RIL was resampled 100 times per number of vial subsample size used. Whiskers represent range of subsample means, circles represent the overall means. (A) Range of 95% of subsamples; (B) 50% of subsamples. Total number of replicate vial measurements per RIL were as follows: RILs 11,229, 11,257, 11,259, 11,265, and 11,021&#xA0;=&#xA0;12 replicates; 11,210&#xA0;=&#xA0;13 replicates; 11,236&#xA0;=&#xA0;14 replicates; 11,237&#xA0;=&#xA0;15 replicates; 335&#xA0;=&#xA0;31 replicates; and 329&#xA0;=&#xA0;64 replicates. Note all pupae in vials were used to calculate each vial length mean, only the number of vial means in each subsample was varied in estimating length RIL means.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F8.medium.gif" width="409" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/4/1277/F8.medium.gif" width="409" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-37" class="first-child">Deviation of mean RIL pupal length utilizing random subsamples of vials compared to that based on all measured vials. For all eight RIL, and two control stocks with 12 or more vial replicate measurements, the RIL mean pupal length was recalculated using a random subsample of vials. Each RIL was resampled 100 times per number of vial subsample size used. Whiskers represent range of subsample means, circles represent the overall means. (A) Range of 95% of subsamples; (B) 50% of subsamples. Total number of replicate vial measurements per RIL were as follows: RILs 11,229, 11,257, 11,259, 11,265, and 11,021 = 12 replicates; 11,210 = 13 replicates; 11,236 = 14 replicates; 11,237 = 15 replicates; 335 = 31 replicates; and 329 = 64 replicates. Note all pupae in vials were used to calculate each vial length mean, only the number of vial means in each subsample was varied in estimating length RIL means.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Estimates of h<sup>2</sup></h3><p id="p-38">Estimates of the additive genetic impact on the variance of the trait can be gained from the slope of regressing the parental midpoint [(length of father + length of mother)/2] against the length of the progeny (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Galton 1886</a>). <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> indicates that, despite the eight-way (67 crosses) and four-way datasets (363 crosses) having no overlap in biological material, and having quite distinct sampling properties, they both result in remarkably similar estimates of <em>h</em><sup>2</sup>, when representing all offspring of a cross as a mean. If all offspring are represented individually, then the estimate of heritability is largely unchanged for both datasets (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This robustness in the estimates is shared with human height (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Hanley 2004</a>). Likewise, regressions that use only the father or mother measurements, rather than their midpoint, confirm that paternal and maternal effects are of equal magnitude for both pupal length and human height (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of mean vial length narrow sense heritability h2 for eight-way and four-way data sets. Despite no overlap in the stocks used to initiate both datasets, there is a remarkable similarity in the slope (h2) and the degree of correlation (R2) among them (see Table 2 for all parameter estimates). Green closed circles, eight-way crosses; blue open circles, four-way crosses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimates of mean vial length narrow sense heritability &lt;em&gt;h&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; for eight-way and four-way data sets. Despite no overlap in the stocks used to initiate both datasets, there is a remarkable similarity in the slope (&lt;em&gt;h&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) and the degree of correlation (&lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) among them (see Table 2 for all parameter estimates). Green closed circles, eight-way crosses; blue open circles, four-way crosses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F9.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/4/1277/F9.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483983" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-39" class="first-child">Estimates of mean vial length narrow sense heritability <em>h</em><sup>2</sup> for eight-way and four-way data sets. Despite no overlap in the stocks used to initiate both datasets, there is a remarkable similarity in the slope (<em>h</em><sup>2</sup>) and the degree of correlation (<em>R</em><sup>2</sup>) among them (see <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> for all parameter estimates). Green closed circles, eight-way crosses; blue open circles, four-way crosses.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483987/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483987/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of heritability estimates for pupal length and human height</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Estimates of H<sup>2</sup></h3><p id="p-42">Estimates of all potentially genetic (<em>H</em><sup>2</sup>) impacts on the variance of the trait can be obtained by estimating the proportion of the total variance in mean vial length measurements related to RIL stock. <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> summarizes the results with both the eight-way (195 RILs) and four-way (81 RILs) datasets generating very similar estimates of <em>H</em><sup>2</sup> of 0.58 and 0.61, respectively. The significance of ANOVA tests reflects the strong genetic signal eight-way <em>P</em> = 2.7 × 10<sup>−140</sup> (<em>F</em> = 10.86, Mean Square = 0.093, d.f. = 194, sum of squares = 18.3) four-way <em>P</em> = 8.8 × 10<sup>−50</sup> (<em>F</em> = 9.026, Mean Square = 0.087, d.f. = 80, sum of squares = 6.9).</p></div><div id="sec-17" class="subsection"><h3>Whole genome scan for loci impacting pupal length</h3><p id="p-43">Using 195 RILs from the eight-way dataset, and the DSPR tools, a genome scan for genomic regions associated with mean RIL pupal length was conducted (unweighted mean of replicate vial means). The density of markers used in the analysis is one SNP per 10 kb across almost the entire reference genome (excluding the mtDNA and Y chromosome). No SNPs of significance were identified at <em>α</em> = 0.05; this was also the case if density corrected RIL means were used (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome scan of RIL pupal length in eight-way dataset (195 RILs means uncorrected for vial density). No peaks are significant at a P&#xA0;=&#xA0;0.05 threshold of 7.02 LOD (estimated by permutation)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-653749847" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome scan of RIL pupal length in eight-way dataset (195 RILs means uncorrected for vial density). No peaks are significant at a &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.05 threshold of 7.02 LOD (estimated by permutation).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/4/1277/F10.medium.gif" width="440" height="174"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/7/4/1277/F10.medium.gif" width="440" height="174"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/4/1277/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484030" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-44" class="first-child">Genome scan of RIL pupal length in eight-way dataset (195 RILs means uncorrected for vial density). No peaks are significant at a <em>P</em> = 0.05 threshold of 7.02 LOD (estimated by permutation).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-45">While (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Visscher <em>et al.</em> 2010</a>) stated that “One could argue that height in humans is the equivalent of bristle number in <em>Drosophila</em>, in terms of its role as a model phenotype.” we propose that pupal length is as good, if not a better, analog, based on its biological properties described in detail for the first time here. This is in addition to the capacity to automate phenotyping using a reliable low cost system. Heritability is estimated using two distinct measures, broad sense (<em>H</em><sup>2</sup>) and narrow sense (<em>h</em><sup>2</sup>), for both of two biologically independent datasets. This allows the consistency of heritability estimates to be assessed between two datasets, which likely span most of the range of the trait within <em>D. melanogaster</em>. Furthermore, the contrast between <em>h</em><sup>2</sup> (additive genetic effects only) and <em>H</em><sup>2</sup> (additive + nonadditive effects) permits the magnitude of nonadditive effects to be estimated. The properties of pupal length established in this manner are potentially salient to efforts to develop resource efficient means to dissect genetically complex traits, and similar to those reported for human height (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The key parameter being the high and consistent heritability estimates, which, in part, reflect the reliability of automated measurements where the estimated precision of measurements of 0.04 mm is small relative to the range of the trait as a species (1.1 mm), or within datasets (0.8–0.9 mm). The reliability of pupal length measurements also likely contributes to the normality of the distributions observed at every hierarchical level in the datasets (<em>e.g.</em>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484053/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484053/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Salient properties of pupal length relative to human height</span><div class="sb-div caption-clear"></div></div></div><p id="p-50">The ease of phenotyping large numbers of individuals at a developmental stage where they are obligatorily stationary for 2–3 d has enabled robust insight into the heritability of pupal length at 24°. The very similar estimates (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>) of heritability for two independent datasets provides increased confidence that this trait represents an excellent prospect with which to attempt to identify the genes underlying this complex trait. Furthermore, the similarity between the observed <em>H</em><sup>2</sup> estimates and those expected based on a purely additive model of inheritance (see last row in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>) indicates that epistatic, dominance, and paternal or maternal effects are relatively modest (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Mackay 2013</a>). The observed similarity in the two <em>h</em><sup>2</sup> estimates (and the associated correlation <em>R</em><sup>2</sup>) between the four-way and eight-way datasets (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>) could also be argued to reflect a modest role for dominance. This is because, while the parents and offspring of the four-way crosses have similar levels of heterozygosity, the eight-way parents are inbred RILs while their offspring are heterozygous. Under some forms of dominance effects, both the slope and the correlation of the regression could be reduced in the latter case, which is not the case for pupal length. With respects to maternal and paternal effects, the data presented here (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS3.pdf">Figure S3</a>) provide evidence that any nonadditive paternal or maternal effects are small, as the size of the effect mothers and fathers exert on their offspring is symmetrical [which is also a property of human height (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Hanley 2004</a>)].</p><p id="p-51">It has been detailed above that the impact of density within vials on pupal length is relatively modest (mostly &lt;0.1 mm, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), and can be conveniently controlled for. While other environmental variables were not systematically explored, limited data collected for the two control lines indicates that between 18° and 24° pupal length changes by 0.2–0.4 mm (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS4.pdf">Figure S4</a>), with pupal length being greater at cooler temperatures. The correlation between adult body weight and pupal length was also briefly examined and found to be <em>R</em><sup>2</sup> = 0.6 for both males and females (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1/FigureS5.pdf">Figure S5</a>). Any correlation between adult body length and pupal length was not examined, though it is potentially noteworthy that individuals possessing the Tb<sup>1</sup> mutation resulting in a strikingly short tubby pupa are not readily distinguished from wildtype as adults based on their body length. Given the large numbers of genes impacting human height (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), it is unsurprising that it can be correlated with disparate traits of particular interest (<em>e.g.</em>, cancer risk, cardiovascular disease and longevity [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Nelson <em>et al.</em> 2015</a>; <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=NCD%20Risk%20Factor%20Collaboration%20(NCD-RisC)%5BCorporate%20Author%5D">NCD Risk Factor Collaboration</a> <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">(NCD-RisC) 2016)</a>]. Future examination of traits correlated with pupal length (including their reaction norms) may prove informative with respect to their genetic architecture and that of pupal length.</p><p id="p-52">Currently, the automated system does not attempt to distinguish male pupae from female pupae, but it is likely that it could be extended to generate probabilistic sex assignments based on the fact that male pupae are on average ≈8% smaller than females (data not shown). When human height data are corrected for sexual dimorphism [also an 8% average difference between sexes (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Galton 1886</a>; <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Hanley 2004</a>)], there is a corresponding increase of the heritability estimate by 9% (see Table 1 in <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Hanley (2004)</a>). This implies that analyzing sexed pupal length data will generate heritability estimates that exceed the values reported here for unsexed pupae (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-53">The observation that an average 20% ± 11.7 SD of pupae in a vial are not measured by the automated system (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) is generally not an issue for most experiments (even for select and resequence approaches), as it is often necessary that only a substantial proportion of individuals are measured, which is sufficient to provide a range of parents for the next generation, and to reduce the variance of mean vial estimates (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Furthermore, parameters provided in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a> and <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a> will provide future researchers with the capacity to readily design maximally resource-efficient experimental strategies.</p><p id="p-54">The observation that the genome scan using 195 RILs (eight-way dataset) failed to identify any significant loci impacting pupal length should be considered in light of the fact that, with this modest number of RILs, the power to detect SNPs of 10 or 5% effect size is only ≈0.37 and ≈0.08, respectively [based on simulations incorporating these exact RILs, see <a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9</a> (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">King <em>et al.</em> 2012b</a>)]. Consequently, the results presented here should be viewed as only potentially sufficient to indicate that few, if any, loci of large effect of size are likely to exist for this trait among the RILs used. It is of course conceivable that phenotyping more of the &gt;1600 RILs currently available would identify significant loci (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">King <em>et al.</em> 2012a</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Huang <em>et al.</em> 2014</a>), and there are several in <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a> that are close to the 0.05 significance threshold. Alternatively, an expanding variety of other approaches could be applied <em>e.g.</em>, select and resequence (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Turner <em>et al.</em> 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Nuzhdin and Turner 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kofler and Schlötterer 2014</a>), or comparisons between parallel selected lines (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chan <em>et al.</em> 2012</a>). While this needs to be further explored in the future, it is clear that the automated phenotyping system will be a key to designing mapping strategies that should allow the identification of many low effect size genes for this trait.</p><p id="p-55">The apparatus used to photograph pupae can be rapidly assembled for ∼500 € (including a camera), or may already be present in many laboratories as geldoc systems. Furthermore the necessary software is open source and free, and runs on any standard desktop PC or MAC. The capacity of the automated system to provide a large number of phenotyped individuals provides increased power to examine the genetic architecture of traits by most approaches. This is likely to prove key to identifying genes or alleles influencing mean pupal length and its variance (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Wood <em>et al.</em> 2014</a>). This is in addition to potentially facilitating exploring the poorly understood genetic basis of sexual dimorphism (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rawlik <em>et al.</em> 2016</a>), and loci which impact the variance of traits [rather than their central values (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ayroles <em>et al.</em> 2015</a>)]. In addition, the capacity to select from large numbers of individuals from which to establish future generations has the potential to enhance select and resequence approaches (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Turner <em>et al.</em> 2011</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Nuzhdin and Turner 2013</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kofler and Schlötterer 2014</a>). Furthermore, the large numbers of powerful approaches based on single pair matings developed by animal and plant breeders (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gianola and Rosa 2015</a>) may also become amenable to examine complex traits in <em>Drosophila</em> through the use of this simple, low-cost, system. As with human height, over its typical range the genetic architecture of pupal length is of limited practical interest. However, for the last 130 yr, the former continues to provide key insights into the methods through which complex traits can be best understood, this is in part due to the ease and reliability of human height measurement. The high heritability of pupal length, and the capacity to easily automate phenotyping, combined with the small and well-described genome of <em>D. melanogaster</em> could make pupal length a similarly valuable model trait.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank the following individuals: Stuart Macdonald for his help with the Drosophila Synthetic Population Resource (DSPR) stocks and tools and for maintaining this valuable common resource, Masayoshi Watada for his invaluable advice and access to his personal fly stocks, Predrag Kalajdzic for karyotyping the founder lines of the four-way cross, Oliver Hirsch for help with the development of the semi-transparent barcode stickers, Carolina Wählby for help with extending the use of the “untangle worms” module of Cellprofiler, and Anita Moeller for her excellent technical help and advice with conducting this experiment.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-57">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039883/-/DC1</a>.</p></li><li class="fn" id="fn-9"><p id="p-58"><em>Communicating editor: B. Oliver</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-21"><span class="received-label">Received </span>September 21, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-02-13"><span class="accepted-label">Accepted </span>February 13, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Reeves and Tautz</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1277.1" data-doi="10.1038/ejhg.2009.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struchalin</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belonogova</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axenovich</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting human height by Victorian and genomic methods.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1070</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DAulchenko%26rft.auinit1%253DY.%2BS.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D1070%26rft.epage%253D1075%26rft.atitle%253DPredicting%2Bhuman%2Bheight%2Bby%2BVictorian%2Band%2Bgenomic%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2009.5%26rft_id%253Dinfo%253Apmid%252F19223933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2009.5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19223933&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.2" data-doi="10.1073/pnas.1503830112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skutt-Kakaria</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Behavioral idiosyncrasy reveals genetic control of phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">6706</span>–<span class="cit-lpage">6711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503830112%26rft_id%253Dinfo%253Apmid%252F25953335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIxLzY3MDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzQvMTI3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1277.3" data-doi="10.1016/S0022-1910(99)00129-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barron</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Anaesthetising <em>Drosophila</em> for behavioural studies.</span> <abbr class="cit-jnl-abbrev">J. Insect Physiol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.stitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.aulast%253DBarron%26rft.auinit1%253DA.%2BB.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D439%26rft.epage%253D442%26rft.atitle%253DAnaesthetising%2BDrosophila%2Bfor%2Bbehavioural%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-1910%252899%252900129-8%26rft_id%253Dinfo%253Apmid%252F12770207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-1910(99)00129-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12770207&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085842500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etheridge</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Véber</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The infinitesimal model.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">039768</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.5" data-doi="10.1016/j.cub.2012.03.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McConnell</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bünger</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Parallel Selection Mapping Using Artificially Selected Mice Reveals Body Weight Control Loci.</span> <abbr class="cit-jnl-abbrev">Cur. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">794</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DChan%26rft.auinit1%253DY.%2BF.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D794%26rft.epage%253D800%26rft.atitle%253DParallel%2Bselection%2Bmapping%2Busing%2Bartificially%2Bselected%2Bmice%2Breveals%2Bbody%2Bweight%2Bcontrol%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.03.011%26rft_id%253Dinfo%253Apmid%252F22445301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.03.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22445301&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galton</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1886</span> <span class="cit-article-title">Regression towards mediocrity in hereditary stature.</span> <abbr class="cit-jnl-abbrev">J. Anthropol. Inst. G. B. Irel.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnthropol.%2BInst.%2BG.%2BB.%2BIrel.%26rft.volume%253D15%26rft.spage%253D246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.7" data-doi="10.1146/annurev-animal-022114-110733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">One hundred years of statistical developments in animal breeding.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D3%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-022114-110733%26rft_id%253Dinfo%253Apmid%252F25387231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-022114-110733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387231&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanley</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">“Transmuting” women into men: Galton’s family data on human stature.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D58%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.9" data-doi="10.1186/1471-2148-3-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mezey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galpern</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Automated measurement of <em>Drosophila</em> wings.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DHoule%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAutomated%2Bmeasurement%2Bof%2BDrosophila%2Bwings.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-3-25%26rft_id%253Dinfo%253Apmid%252F14670094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-3-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14670094&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.10" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 <em>Drosophila melanogaster</em> genetic reference panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.11" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic dissection of a model complex trait using the <em>Drosophila</em> synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.12" data-doi="10.1534/genetics.112.138537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Properties and power of the <em>Drosophila</em> synthetic population resource for the routine dissection of complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.138537%26rft_id%253Dinfo%253Apmid%252F22505626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.13" data-doi="10.1093/molbev/mst221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A guide for the design of evolve and resequencing studies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst221%26rft_id%253Dinfo%253Apmid%252F24214537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzIvNDc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy80LzEyNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.14" data-doi="10.2144/000112257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CellProfiler: free, versatile software for automated biological image analysis.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DLamprecht%26rft.auinit1%253DM.%2BR.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D75%26rft.atitle%253DCellProfiler%253A%2Bfree%252C%2Bversatile%2Bsoftware%2Bfor%2Bautomated%2Bbiological%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112257%26rft_id%253Dinfo%253Apmid%252F17269487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17269487&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248948900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-2" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.15" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Epistasis and quantitative traits: using model organisms to study gene–gene interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Medici</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonesch</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The FlyCatwalk : a high-throughput feature-based sorting system for artificial selection in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.17"><div class="cit-metadata"><cite><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=NCD%20Risk%20Factor%20Collaboration%20(NCD-RisC)%5BCorporate%20Author%5D">NCD Risk Factor Collaboration</a> (NCD-RisC), <span class="cit-pub-date">2016</span> <span class="cit-article-title">A century of trends in adult human height.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De13410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.18" data-doi="10.1056/NEJMoa1404881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamby</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saleheen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopewell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetically determined height and coronary artery disease.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">1608</span>–<span class="cit-lpage">1618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D372%26rft.spage%253D1608%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1404881%26rft_id%253Dinfo%253Apmid%252F25853659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1404881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25853659&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.19" data-doi="10.1016/j.gde.2013.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Promises and limitations of hitchhiking mapping.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">694</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D23%26rft.spage%253D694%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2013.10.002%26rft_id%253Dinfo%253Apmid%252F24239053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2013.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24239053&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.20" data-doi="10.1038/ng.3285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polderman</span>  <span class="cit-name-given-names">T. J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leeuw</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Bochoven</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meta-analysis of the heritability of human traits based on fifty years of twin studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">702</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D702%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3285%26rft_id%253Dinfo%253Apmid%252F25985137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25985137&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rawlik</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canela-Xandri</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenesa</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evidence for sex-specific genetic architectures across a spectrum of human complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.22" data-doi="10.1038/hdy.1987.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mousseau</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Quantitative genetics and fitness lessons from <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D58%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1987.15%26rft_id%253Dinfo%253Apmid%252F3818341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1987.15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3818341&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G332500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.23" data-doi="10.1371/journal.pgen.1001336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarone</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population-based resequencing of experimentally evolved populations reveals the genetic basis of body size variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001336%26rft.epage%253De1001336%26rft.atitle%253DPopulation-based%2Bresequencing%2Bof%2Bexperimentally%2Bevolved%2Bpopulations%2Breveals%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bbody%2Bsize%2Bvariation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001336%26rft_id%253Dinfo%253Apmid%252F21437274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437274&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1277.24" data-doi="10.1146/annurev-genet-111212-133258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinkhuyzen</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimation and partition of heritability in human populations using whole-genome analysis methods.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133258%26rft_id%253Dinfo%253Apmid%252F23988118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23988118&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.25" data-doi="10.1017/S0016672310000571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">From Galton to GWAS: quantitative genetics of human height.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D5-6%26rft.spage%253D371%26rft.epage%253D379%26rft.atitle%253DFrom%2BGalton%2Bto%2BGWAS%253A%2Bquantitative%2Bgenetics%2Bof%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000571%26rft_id%253Dinfo%253Apmid%252F21429269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429269&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.4.1277.26" data-doi="10.1136/adc.65.12.1340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voss</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumming</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkin</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betts</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The reliability of height measurement (the Wessex growth study).</span> <abbr class="cit-jnl-abbrev">Arch. Dis. Child.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1340</span>–<span class="cit-lpage">1344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BDisease%2Bin%2BChildhood%26rft.stitle%253DArch.%2BDis.%2BChild.%26rft.aulast%253DVoss%26rft.auinit1%253DL%2BD%26rft.volume%253D65%26rft.issue%253D12%26rft.spage%253D1340%26rft.epage%253D1344%26rft.atitle%253DThe%2Breliability%2Bof%2Bheight%2Bmeasurement%2B%2528the%2BWessex%2BGrowth%2BStudy%2529.%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fadc.65.12.1340%26rft_id%253Dinfo%253Apmid%252F2270942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImFyY2hkaXNjaGlsZCI7czo1OiJyZXNpZCI7czoxMDoiNjUvMTIvMTM0MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNC8xMjc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.27" data-doi="10.1038/nmeth.1984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wählby</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamentsky</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riklin-Raviv</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An image analysis toolbox for high-throughput <em>C. elegans</em> assays.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DWahlby%26rft.auinit1%253DC.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D714%26rft.epage%253D716%26rft.atitle%253DAn%2Bimage%2Banalysis%2Btoolbox%2Bfor%2Bhigh-throughput%2BC.%2Belegans%2Bassays.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1984%26rft_id%253Dinfo%253Apmid%252F22522656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1984&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22522656&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.28" data-doi="10.1038/ng.121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lango</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide association analysis identifies 20 loci that influence adult height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWeedon%26rft.auinit1%253DM.%2BN.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D575%26rft.epage%253D583%26rft.atitle%253DGenome-wide%2Bassociation%2Banalysis%2Bidentifies%2B20%2Bloci%2Bthat%2Binfluence%2Badult%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.121%26rft_id%253Dinfo%253Apmid%252F18391952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391952&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.4.1277.29" data-doi="10.1038/ng.3097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vedantam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pers</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Defining the role of common variation in the genomic and biological architecture of adult human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1173%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3097%26rft_id%253Dinfo%253Apmid%252F25282103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25282103&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F4%2F1277.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/4/1277.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1267" title="Genetic Adaptation of Schizothoracine Fish to the Phased Uplifting of the Qinghai–Tibetan Plateau" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/4/1287" title="Independent Maternal and Fetal Genetic Effects on Midgestational Circulating Levels of Environmental Pollutants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/4"><span>Volume 7 Issue 4, April 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/4.cover-source.jpg?itok=KOTpzmYp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/4/1277.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0G70Qx_hByuJhf3_w-HDOG8fjdN9BXohsvxD1R7aFM8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/4/1277" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/4/1277" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/4/1277" class="active" data-icon-position="" data-hide-link-title="0">Automated Phenotyping Indicates Pupal Size in Drosophila Is a Highly Heritable Trait with an Apparent Polygenic Basis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483838" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nWj_F5fFAgIVSXA0qp-WSF-1fGdfTE-ORjMupcfBNno" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1277.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/4/1277" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IcBEp6VJnG5iIHJcnP2EJ_OUXlwBJzqawxm06PY0Qv8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/4/1277" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/4/1277.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483838" id="node483838--2" data-pisa="ggg;7/4/1277" data-pisa-master="ggg;g3.117.039883" data-apath="/ggg/7/4/1277.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Automated Phenotyping Indicates Pupal Size in <em>Drosophila</em> Is a Highly Heritable Trait with an Apparent Polygenic Basis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9454-6175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Guy Reeves</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0460-5344" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diethard Tautz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1277-1286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039883 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483838--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. Guy Reeves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BGuy%2BReeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reeves%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BGuy%2BReeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9454-6175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Guy Reeves</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reeves@evolbio.mpg.de" class="" data-icon-position="" data-hide-link-title="0">reeves@evolbio.mpg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diethard Tautz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diethard%2BTautz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tautz%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiethard%2BTautz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0460-5344" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diethard Tautz</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/4/1277" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/4/1277&amp;title=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/4/1277&amp;title=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/4/1277&amp;title=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/4/1277&amp;text=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/4/1277&amp;title=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/4/1277&amp;t=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/4/1277&amp;title=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/4/1277&amp;title=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483838/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483838" id="node483838--3" data-pisa="ggg;7/4/1277" data-pisa-master="ggg;g3.117.039883" data-apath="/ggg/7/4/1277.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Automated Phenotyping Indicates Pupal Size in <em>Drosophila</em> Is a Highly Heritable Trait with an Apparent Polygenic Basis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9454-6175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Guy Reeves</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0460-5344" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diethard Tautz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1277-1286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039883 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483838--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">R. Guy Reeves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BGuy%2BReeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reeves%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BGuy%2BReeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9454-6175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Guy Reeves</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reeves@evolbio.mpg.de" class="" data-icon-position="" data-hide-link-title="0">reeves@evolbio.mpg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diethard Tautz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diethard%2BTautz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tautz%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiethard%2BTautz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0460-5344" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diethard Tautz</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1277&amp;rft.epage=1286&amp;rft.atitle=Automated%20Phenotyping%20Indicates%20Pupal%20Size%20in%20Drosophila%20Is%20a%20Highly%20Heritable%20Trait%20with%20an%20Apparent%20Polygenic%20Basis&amp;rft.volume=7&amp;rft.issue=4&amp;rft.date=2017-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Reeves%2C+R.+Guy&amp;rft.au=Tautz%2C+Diethard"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483838/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483838/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483838/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483838/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483838/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483838/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483838/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483838/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483838/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483838/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483838/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483838/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/4/1277" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/4/1277.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/4/1277.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/4/1277.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/4/1277.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
