<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/2/265" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/2/265.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/2/265.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/2/265.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/2/265.atom" />
<meta name="HW.pisa" content="ggg;4/2/265" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multigenic Natural Variation Underlies Caenorhabditis elegans Olfactory Preference for the Bacterial Pathogen Serratia marcescens" />
<meta name="DC.Identifier" content="10.1534/g3.113.008649" />
<meta name="DC.Date" content="2014-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Glater et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The nematode Caenorhabditis elegans can use olfaction to discriminate among different kinds of bacteria, its major food source. We asked how natural genetic variation contributes to choice behavior, focusing on differences in olfactory preference behavior between two wild-type C. elegans strains. The laboratory strain [N2][1] strongly prefers the odor of Serratia marcescens , a soil bacterium that is pathogenic to C. elegans , to the odor of Escherichia coli , a commonly used laboratory food source. The divergent Hawaiian strain [CB4856][2] has a weaker attraction to Serratia than the [N2][1] strain, and this behavioral difference has a complex genetic basis. At least three quantitative trait loci (QTLs) from the [CB4856][2] Hawaii strain (HW) with large effect sizes lead to reduced Serratia preference when introgressed into an [N2][1] genetic background. These loci interact and have epistatic interactions with at least two antagonistic QTLs from HW that increase Serratia preference. The complex genetic architecture of this C. elegans trait is reminiscent of the architecture of mammalian metabolic and behavioral traits.

 [1]: http://www.wormbase.org/db/get?name=N2;class=Strain
 [2]: http://www.wormbase.org/db/get?name=CB4856;class=Strain" />
<meta name="DC.Contributor" content="Elizabeth E. Glater" />
<meta name="DC.Contributor" content="Matthew V. Rockman" />
<meta name="DC.Contributor" content="Cornelia I. Bargmann" />
<meta name="article:published_time" content="2014-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Multigenic Natural Variation Underlies Caenorhabditis elegans Olfactory Preference for the Bacterial Pathogen Serratia marcescens" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The nematode &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; can use olfaction to discriminate among different kinds of bacteria, its major food source. We asked how natural genetic variation contributes to choice behavior, focusing on differences in olfactory preference behavior between two wild-type &lt;i&gt;C. elegans&lt;/i&gt; strains. The laboratory strain N2 strongly prefers the odor of &lt;i&gt;Serratia marcescens&lt;/i&gt;, a soil bacterium that is pathogenic to &lt;i&gt;C. elegans&lt;/i&gt;, to the odor of &lt;i&gt;Escherichia coli&lt;/i&gt;, a commonly used laboratory food source. The divergent Hawaiian strain CB4856 has a weaker attraction to &lt;i&gt;Serratia&lt;/i&gt; than the N2 strain, and this behavioral difference has a complex genetic basis. At least three quantitative trait loci (QTLs) from the CB4856 Hawaii strain (HW) with large effect sizes lead to reduced &lt;i&gt;Serratia&lt;/i&gt; preference when introgressed into an N2 genetic background. These loci interact and have epistatic interactions with at least two antagonistic QTLs from HW that increase &lt;i&gt;Serratia&lt;/i&gt; preference. The complex genetic architecture of this &lt;i&gt;C. elegans&lt;/i&gt; trait is reminiscent of the architecture of mammalian metabolic and behavioral traits.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/02/01" />
<meta name="citation_mjid" content="ggg;4/2/265" />
<meta name="citation_id" content="4/2/265" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/2/265" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/2/265.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/2/265.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/2/265.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008649" />
<meta name="citation_pmid" content="24347628" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="265" />
<meta name="citation_lastpage" content="276" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elizabeth E. Glater" />
<meta name="citation_author_institution" content="Department of Biology, Harvey Mudd College, Claremont, California 91711" />
<meta name="citation_author_email" content="Elizabeth.glater@pomona.edu" />
<meta name="citation_author" content="Matthew V. Rockman" />
<meta name="citation_author_institution" content="Department of Biology and Center for Genomics and Systems Biology, New York University, New York, New York 10003" />
<meta name="citation_author" content="Cornelia I. Bargmann" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Laboratory of Neural Circuits and Behavior, The Rockefeller University, New York, New York 10065" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Altshuler;citation_author=M. J. Daly;citation_author=E. S. Lander;citation_title=Genetic Mapping in Human Disease;citation_pages=881-888;citation_volume=322;citation_year=2008;citation_issue=5903;citation_pmid=18988837;citation_doi=10.1126/science.1156409" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Med.;citation_author=F. H. Bach;citation_author=M. B. Widmer;citation_author=M. L. Bach;citation_author=J. Klein;citation_title=Serologically defined and lymphocyte-defined components of the major histocompatibility complex in the mouse.;citation_volume=136;citation_year=1972;citation_pmid=4404878;citation_doi=10.1084/jem.136.6.1430" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Bendesky;citation_author=C. I. Bargmann;citation_title=Genetic contributions to behavioural diversity at the gene-environment interface.;citation_pages=809-820;citation_volume=12;citation_year=2011;citation_issue=12;citation_pmid=22064512;citation_doi=10.1038/ni.2088" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Bendesky;citation_author=M. Tsunozaki;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_author=C. I. Bargmann;citation_title=Catecholamine receptor polymorphisms affect decision-making in C. elegans.;citation_pages=313-318;citation_volume=472;citation_year=2011;citation_issue=7343;citation_pmid=21412235;citation_doi=10.1038/nature09821" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Bhatnagar;citation_author=A. T. Oler;citation_author=M. E. Rabaglia;citation_author=D. S. Stapleton;citation_author=K. L. Schueler;citation_title=Positional cloning of a type 2 diabetes quantitative trait locus; tomosyn-2, a negative regulator of insulin secretion.;citation_pages=e1002323-e1002323;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=21998599;citation_doi=10.1371/journal.pgen.1002323" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Brenner;citation_title=THE GENETICS OF CAENORHABDITIS ELEGANS;citation_pages=71-94;citation_volume=77;citation_year=1974;citation_issue=1;citation_pmid=4366476;citation_doi=10.1051/medsci/20031967725" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. C. Campbell;citation_author=A. Ranciaro;citation_author=A. Froment;citation_author=J. Hirbo;citation_author=S. Omar;citation_title=Evolution of functionally diverse alleles associated with PTC bitter taste sensitivity in Africa.;citation_volume=29;citation_year=2012;citation_pmid=22130969;citation_doi=10.1093/molbev/msr293" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. W. Davis;citation_author=M. Hammarlund;citation_author=T. Harrach;citation_author=P. Hullett;citation_author=S. Olsen;citation_title=Rapid single nucleotide polymorphism mapping in C. elegans.;citation_pages=118-118;citation_volume=6;citation_year=2005;citation_pmid=16156901;citation_doi=10.1186/1471-2164-6-118" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. de Bono;citation_author=C. I. Bargmann;citation_title=Natural variation in a neuropeptide Y receptor homolog modifies social behavior and food response in C. elegans.;citation_pages=679-689;citation_volume=94;citation_year=1998;citation_issue=5;citation_pmid=9741632;citation_doi=10.1016/S0092-8674(00)81609-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Doroszuk;citation_author=L. B. Snoek;citation_author=E. Fradin;citation_author=J. Riksen;citation_author=J. Kammenga;citation_title=A genome-wide library of CB4856/N2 introgression lines of Caenorhabditis elegans.;citation_volume=37;citation_year=2009;citation_pmid=19542186;citation_doi=10.1093/nar/gkp528" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=A. C. Edwards;citation_author=L. Zwarts;citation_author=A. Yamamoto;citation_author=P. Callaerts;citation_author=T. F. Mackay;citation_title=Mutations in many genes affect aggressive behavior in Drosophila melanogaster.;citation_pages=29-29;citation_volume=7;citation_year=2009;citation_pmid=19519879;citation_doi=10.1186/1741-7007-7-29" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=M. A. Felix;citation_author=F. Duveau;citation_title=Population dynamics and habitat sharing of natural populations of Caenorhabditis elegans and C. briggsae.;citation_pages=59-59;citation_volume=10;citation_year=2012;citation_pmid=22731941;citation_doi=10.1186/1741-7007-10-59" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=Mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Gaertner;citation_author=M. D. Parmenter;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_author=P. C. Phillips;citation_title=More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in Caenorhabditis elegans.;citation_volume=192;citation_year=2012;citation_pmid=23086219;citation_doi=10.1534/genetics.112.142877" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. A. Glauser;citation_author=W. C. Chen;citation_author=R. Agin;citation_author=B. L. Macinnis;citation_author=A. B. Hellman;citation_title=Heat avoidance is regulated by transient receptor potential (TRP) channels and a neuropeptide signaling pathway in Caenorhabditis elegans.;citation_volume=188;citation_year=2011;citation_pmid=21368276;citation_doi=10.1534/genetics.111.127100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Hodgkin;citation_author=T. Doniach;citation_title=Natural Variation and Copulatory Plug Formation in Caenorhabditis elegans;citation_pages=149-164;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. D. Jones;citation_title=The Genetic Basis of Drosophila sechellia's Resistance to a Host Plant Toxin;citation_pages=1899-1908;citation_volume=149;citation_year=1998;citation_issue=4;citation_pmid=9691045" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=C. L. Kurz;citation_author=S. Chauvet;citation_author=E. Andres;citation_author=M. Aurouze;citation_author=I. Vallet;citation_title=Virulence factors of the human opportunistic pathogen Serratia marcescens identified by in vivo screening;citation_pages=1451-1460;citation_volume=22;citation_year=2003;citation_issue=7;citation_pmid=12660152;citation_doi=10.1093/emboj/cdg159" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Human Biology;citation_journal_abbrev=Human Biology;citation_author=H. Li;citation_author=A. J. Pakstis;citation_author=J. R. Kidd;citation_author=K. K. Kidd;citation_title=Selection on the human bitter taste gene, TAS2R16, in Eurasian populations.;citation_pages=363-377;citation_volume=83;citation_year=2011;citation_issue=3;citation_pmid=21740153;citation_doi=10.3378/027.083.0303" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=X. Li;citation_author=W. Li;citation_author=H. Wang;citation_author=J. Cao;citation_author=K. Maehashi;citation_title=Pseudogenization of a sweet-receptor gene accounts for cats' indifference toward sugar.;citation_pages=27-35;citation_volume=1;citation_year=2005;citation_issue=1;citation_pmid=16103917;citation_doi=10.1371/journal.pgen.0010027" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=G. V. Mallo;citation_author=C. L. Kurz;citation_author=C. Couillault;citation_author=N. Pujol;citation_author=S. Granjeaud;citation_title=Inducible antibacterial defense system in C. elegans.;citation_pages=1209-1214;citation_volume=12;citation_year=2002;citation_issue=14;citation_pmid=12176330;citation_doi=10.1016/S0960-9822(02)00928-4" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. S. McBride;citation_title=Rapid evolution of smell and taste receptor genes during host specialization in Drosophila sechellia.;citation_volume=104;citation_year=2007;citation_pmid=17360391;citation_doi=10.1073/pnas.0608424104" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Med.;citation_author=H. O. McDevitt;citation_author=M. L. Tyan;citation_title=Genetic control of the antibody response in inbred mice. Transfer of response by spleen cells and linkage to the major histocompatibility (H-2) locus.;citation_volume=128;citation_year=1968;citation_pmid=5662015;citation_doi=10.1084/jem.128.1.1" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Med.;citation_author=H. O. McDevitt;citation_author=B. D. Deak;citation_author=D. C. Shreffler;citation_author=J. Klein;citation_author=J. H. Stimpfling;citation_title=Genetic control of the immune response. Mapping of the Ir-1 locus.;citation_volume=135;citation_year=1972;citation_pmid=4554451;citation_doi=10.1084/jem.135.6.1259" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=P. T. McGrath;citation_author=M. V. Rockman;citation_author=M. Zimmer;citation_author=H. Jang;citation_author=E. Z. Macosko;citation_title=Quantitative mapping of a digenic behavioral trait implicates globin variation in C. elegans sensory behaviors.;citation_pages=692-699;citation_volume=61;citation_year=2009;citation_pmid=19285466;citation_doi=10.1016/j.neuron.2009.02.012" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Melo;citation_author=G. Ruvkun;citation_title=Inactivation of conserved C. elegans genes engages pathogen- and xenobiotic-associated defenses.;citation_pages=452-466;citation_volume=149;citation_year=2012;citation_issue=2;citation_pmid=22500807;citation_doi=10.1016/j.cell.2012.02.050" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Q. Niu;citation_author=X. Huang;citation_author=L. Zhang;citation_author=J. Xu;citation_author=D. Yang;citation_title=A Trojan horse mechanism of bacterial pathogenesis against nematodes.;citation_volume=107;citation_year=2010;citation_pmid=20733068;citation_doi=10.1073/pnas.1007276107" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. F. Palopoli;citation_author=M. V. Rockman;citation_title=Molecular basis of the copulatory plug polymorphism in Caenorhabditis elegans.;citation_pages=1019-1022;citation_volume=454;citation_year=2008;citation_issue=7207;citation_pmid=18633349;citation_doi=10.1038/nature07171" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Persson;citation_author=E. Gross;citation_author=P. Laurent;citation_author=K. E. Busch;citation_author=H. Bretes;citation_title=Natural variation in a neural globin tunes oxygen sensing in wild Caenorhabditis elegans.;citation_pages=1030-1033;citation_volume=458;citation_year=2009;citation_issue=7241;citation_pmid=19262507;citation_doi=10.1038/nature07820" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Pradel;citation_author=Y. Zhang;citation_author=N. Pujol;citation_author=T. Matsuyama;citation_author=C. I. Bargmann;citation_title=Detection and avoidance of a natural product from the pathogenic bacterium Serratia marcescens by Caenorhabditis elegans.;citation_volume=104;citation_year=2007;citation_pmid=17267603;citation_doi=10.1073/pnas.0610281104" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=N. Pujol;citation_author=E. M. Link;citation_author=L. X. Liu;citation_author=C. L. Kurz;citation_author=G. Alloing;citation_title=A reverse genetic analysis of components of the Toll signaling pathway in Caenorhabditis elegans.;citation_pages=809-821;citation_volume=11;citation_year=2001;citation_issue=11;citation_pmid=11516642;citation_doi=10.1016/S0960-9822(01)00241-X" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=J. P. Rapp;citation_author=B. Joe;citation_title=Use of contiguous congenic strains in analyzing compound QTLs.;citation_volume=44;citation_year=2012;citation_pmid=22108210;citation_doi=10.1152/physiolgenomics.00136.2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=H. M. Robertson;citation_title=Two Large Families of Chemoreceptor Genes in the Nematodes Caenorhabditis elegans and Caenorhabditis briggsae Reveal Extensive Gene Duplication, Diversification, Movement, and Intron Loss;citation_pages=449-463;citation_volume=8;citation_year=1998;citation_issue=5;citation_pmid=9582190" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=H. M. Robertson;citation_title=The Large srh Family of Chemoreceptor Genes in Caenorhabditis Nematodes Reveals Processes of Genome Evolution Involving Large Duplications and Deletions and Intron Gains and Losses;citation_pages=192-203;citation_volume=10;citation_year=2000;citation_issue=2;citation_pmid=10673277;citation_doi=10.1101/gr.10.2.192" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Recombinational landscape and population genomics of Caenorhabditis elegans.;citation_pages=e1000419-e1000419;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19283065;citation_doi=10.1371/journal.pgen.1000419" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=H. Schulenburg;citation_author=J. J. Ewbank;citation_title=Diversity and specificity in the interaction between Caenorhabditis elegans and the pathogen Serratia marcescens.;citation_pages=49-49;citation_volume=4;citation_year=2004;citation_pmid=15555070;citation_doi=10.1186/1471-2148-4-49" />
<meta name="citation_reference" content="citation_journal_title=Parasitology;citation_author=H. Schulenburg;citation_author=S. Muller;citation_title=Natural variation in the response of Caenorhabditis elegans towards Bacillus thuringiensis.;citation_pages=433-443;citation_volume=128;citation_year=2004;citation_pmid=15151149;citation_doi=10.1017/S003118200300461X" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Shao;citation_author=L. C. Burrage;citation_author=D. S. Sinasac;citation_author=A. E. Hill;citation_author=S. R. Ernest;citation_title=Genetic architecture of complex traits: large phenotypic effects and pervasive epistasis.;citation_volume=105;citation_year=2008;citation_pmid=19066216;citation_doi=10.1073/pnas.0810388105" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=H. Shao;citation_author=D. S. Sinasac;citation_author=L. C. Burrage;citation_author=C. A. Hodges;citation_author=P. J. Supelak;citation_title=Analyzing complex traits with congenic strains.;citation_pages=276-286;citation_volume=21;citation_year=2010;citation_issue=5-6;citation_pmid=20524000;citation_doi=10.1007/s00335-010-9267-5" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Biology;citation_journal_abbrev=J. Exp. Biol.;citation_author=B. B. Shtonda;citation_author=L. Avery;citation_title=Dietary choice behavior in Caenorhabditis elegans;citation_pages=89-102;citation_volume=209;citation_year=2006;citation_issue=1;citation_pmid=16354781;citation_doi=10.1242/jeb.01955" />
<meta name="citation_reference" content="citation_journal_title=Transplantation;citation_journal_abbrev=Transplantation;citation_author=G. D. Snell;citation_author=H. P. Bunker;citation_title=HISTOCOMPATIBILITY GENES OF MICE. V. FIVE NEW HISTOCOMPATIBILITY LOCI IDENTIFIED BY CONGENIC RESISTANT LINES ON A C57B 10 BACKGROUND.;citation_pages=235-252;citation_volume=3;citation_year=1965;citation_pmid=14265497;citation_doi=10.1097/00007890-196503000-00011" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=M. C. Stensmyr;citation_title=Drosophila sechellia as a model in chemosensory neuroecology.;citation_pages=468-475;citation_volume=1170;citation_year=2009;citation_pmid=19686179;citation_doi=10.1111/j.1749-6632.2009.03911.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. K. Stewart;citation_author=N. L. Clark;citation_author=G. Merrihew;citation_author=E. M. Galloway;citation_author=J. H. Thomas;citation_title=High genetic diversity in the chemoreceptor superfamily of Caenorhabditis elegans.;citation_volume=169;citation_year=2005;citation_pmid=15520260;citation_doi=10.1534/genetics.104.035329" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. H. Thomas;citation_author=J. L. Kelley;citation_author=H. M. Robertson;citation_author=K. Ly;citation_author=W. J. Swanson;citation_title=Adaptive evolution in the SRZ chemoreceptor families of Caenorhabditis elegans and Caenorhabditis briggsae.;citation_volume=102;citation_year=2005;citation_pmid=15761060;citation_doi=10.1073/pnas.0406469102" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=J. H. Thomas;citation_author=H. M. Robertson;citation_title=The Caenorhabditis chemoreceptor gene families.;citation_pages=42-42;citation_volume=6;citation_year=2008;citation_pmid=18837995;citation_doi=10.1186/1741-7007-6-42" />
<meta name="citation_reference" content="citation_title=Modern applied statistics with S;citation_author=W. N. Venables;citation_author=B. D. Ripley;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Development, growth &amp; differentiation;citation_journal_abbrev=Dev Growth Differ;citation_author=M. E. Viney;citation_author=M. P. Gardner;citation_author=J. A. Jackson;citation_title=Variation in Caenorhabditis elegans dauer larva formation.;citation_pages=389-396;citation_volume=45;citation_year=2003;citation_issue=4;citation_pmid=12950280;citation_doi=10.1046/j.1440-169X.2003.00703.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=R. J. Volkers;citation_author=L. B. Snoek;citation_author=C. J. van Hellenberg Hubar;citation_author=R. Coopman;citation_author=W. Chen;citation_title=Gene-environment and protein degradation signatures characterize genomic and phenotypic diversity in wild Caenorhabditis elegans populations.;citation_pages=1-13;citation_volume=11;citation_year=2013;citation_pmid=23294804;citation_doi=10.1186/1741-7007-11-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Zhang;citation_author=H. Lu;citation_author=C. I. Bargmann;citation_title=Pathogenic bacteria induce aversive olfactory learning in Caenorhabditis elegans.;citation_pages=179-184;citation_volume=438;citation_year=2005;citation_issue=7065;citation_pmid=16281027;citation_doi=10.1038/nature04216" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multigenic Natural Variation Underlies Caenorhabditis elegans Olfactory Preference for the Bacterial Pathogen Serratia marcescens" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
<meta name="twitter:description" content="The nematode Caenorhabditis elegans can use olfaction to discriminate among different kinds of bacteria, its major food source. We asked how natural genetic variation contributes to choice behavior, focusing on differences in olfactory preference behavior between two wild-type C. elegans strains. The laboratory strain [N2][1] strongly prefers the odor of Serratia marcescens , a soil bacterium that is pathogenic to C. elegans , to the odor of Escherichia coli , a commonly used laboratory food source. The divergent Hawaiian strain [CB4856][2] has a weaker attraction to Serratia than the [N2][1] strain, and this behavioral difference has a complex genetic basis. At least three quantitative trait loci (QTLs) from the [CB4856][2] Hawaii strain (HW) with large effect sizes lead to reduced Serratia preference when introgressed into an [N2][1] genetic background. These loci interact and have epistatic interactions with at least two antagonistic QTLs from HW that increase Serratia preference. The complex genetic architecture of this C. elegans trait is reminiscent of the architecture of mammalian metabolic and behavioral traits.

 [1]: http://www.wormbase.org/db/get?name=N2;class=Strain
 [2]: http://www.wormbase.org/db/get?name=CB4856;class=Strain" />
<meta name="og-title" property="og:title" content="Multigenic Natural Variation Underlies Caenorhabditis elegans Olfactory Preference for the Bacterial Pathogen Serratia marcescens" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/2/265" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The nematode Caenorhabditis elegans can use olfaction to discriminate among different kinds of bacteria, its major food source. We asked how natural genetic variation contributes to choice behavior, focusing on differences in olfactory preference behavior between two wild-type C. elegans strains. The laboratory strain [N2][1] strongly prefers the odor of Serratia marcescens , a soil bacterium that is pathogenic to C. elegans , to the odor of Escherichia coli , a commonly used laboratory food source. The divergent Hawaiian strain [CB4856][2] has a weaker attraction to Serratia than the [N2][1] strain, and this behavioral difference has a complex genetic basis. At least three quantitative trait loci (QTLs) from the [CB4856][2] Hawaii strain (HW) with large effect sizes lead to reduced Serratia preference when introgressed into an [N2][1] genetic background. These loci interact and have epistatic interactions with at least two antagonistic QTLs from HW that increase Serratia preference. The complex genetic architecture of this C. elegans trait is reminiscent of the architecture of mammalian metabolic and behavioral traits.

 [1]: http://www.wormbase.org/db/get?name=N2;class=Strain
 [2]: http://www.wormbase.org/db/get?name=CB4856;class=Strain" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
    <title>Multigenic Natural Variation Underlies Caenorhabditis elegans Olfactory Preference for the Bacterial Pathogen Serratia marcescens | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AX8vsMnUcLTzqDo1RjYYNTxIUILhr-ju9pGDQnRP4RY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459640","apath":"\/ggg\/4\/2\/265.atom","pisa":"ggg;4\/2\/265","ac":{"\/ggg\/4\/2\/265.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/2\/265.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/2\/265"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/2\/265"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/2\/265":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hCOhQGYbgMAI2gfztgaNPIwb6mgTJbA40ZYQDtwzXPa8yabrEbPareBrkkvvVJj8+w831gEKt5EdQZvq5+bnMjxe20qwgWPtlsSUuXLYSHzYt87G6oObO4iSxfUhKqr0ElC3agSKzt38zIYj\\\\\\\/LxYH8NGSqtKt4yptmDvrKU72l\\\\\\\/8wUVbrPgrRVQddXqgCXsbW7FIdILwgu7wnAQSkiIq1uHtXYmrAecIthoPXfq9T50t2Fqg7P1G0OayxT5QJWdR1FXVzNFfwluZCBgptRekXOnHLNFUY7qLDD14AEg+0ShKapeHUiVb6m+cHHwjnhinPJrcqLDa8Aj81iXXrGQ\\\\\\\/WHuC9WvWJn7aVQcWq8+u0VQC2LZ57xO8OeKi\\\\\\\/tAT4b8ni7NL5J3Pimnv2t6k4z1NoMe4EnbgarNBp0HEcR9zQRWMN6lH5cJoRKBoYda2EBEeTWC6ja7tsNylhhC0fJumG+rW4K8X+8QrkFGFHFxBqDvv8mggx2QpcW4J6IVB1o9QaMzPUiParHs0i5tITAlt6JCL3AhGl9ESKPaXLSBnZH1Wll0xlB7O5rt4eVO+wzKCMd6u2H+sfA+dy4UkCLdhHYpWA1g5wTCfIALOxypciClJSRGKX2Cpo9CdcHqwuFF6eWVmoRzngh7zMkn1ol+YqZ08E7xZhwRsEr47mQH0i7tiuX8EuPZ9wsU4SHMbhOI+DkZN3X55rJ8OUHBEfk666kgc8R+GT+MqaVeSxPppz5XHrbssgn5QJQc2pnQU2\\\\\\\/Q0bwQx22344kcztx3u1z9XFOZlN4UNdOw3+tdNHMtATSeD7oFeMtvoYgEx4bqgMwl2qpl0I8YFAR\\\\\\\/cV2Np0hLfMh3lFr+biegS7dxsGX\\\\\\\/XJse1cZTwBJlfjqfMBlxSgTLO9wL4dCqi3JTKanShPcal4WUvydDbObcInybIuMT\\\\\\\/gXDwPNVJPeP4lTJDistwpZ9lG9M3svAxUZenlHnzLn+9ysvMxLdZ3xl6x8uujw2gAWAxEf6CDhVbI21cPlQ+9k+XG2n6VIsQsmKtZ65s9EAXHBTGTVYLN53LA+OwrVwX16DuHUh0ABcARztldZZTcjw7Hcj0ct\\\\\\\/0RcjEbwBAkfw10PTCDltEcgg0tQ0C1W9TUviZ1PkJSl5qL3RlbdsiVVtCPqnPk2FT9kgYVIlelRYOCZC0\\\\\\\/r\\\\\\\/0Loyoz4HFd1Qk36w+diZOxZUod39Jc6VC7hqTOwLv20GfRFiU0neM2DwIVGZl5W9WSHh2BnGy5ZlXRv+wzTw62GLgErjuLyGR4o8j\\\u0022,\\\u0022iv\\\u0022:\\\u0022mCvyE9uU4BbfCeCPL9cPfg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e857afc7f053006e44fb5b5515cfc4f64ca7dcb8b5dfaf837b122f4dfa4ffb59\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dEiKPKPrIMhBFy53ajEraJfS3FD8lkrj3e+5hJaAx5xhidn3mKCUL3iRGDY6mf000KuSSN8xzZOK6JRBgo6vtIPjvfQHfOyJYlJz6VvYsoxZGRq4jmjeYiuhoGNcqXMvpI7RgH6ZnTj660lfmVTMEokRn1CLXawd6y2GLg1wWEf8EFv3ahIyUGe0tO9Rxara62CDqVzf\\\\\\\/k4tYzg5IXQ07A6wFi\\\\\\\/Hfrc6t8d\\\\\\\/cL\\\\\\\/l5XRuwbe5nwoOsA34m2ybYrCCGJlrdymnlfF8OeIuNmTV6Ixsd+8ipshE\\\\\\\/au6AiwcKmdaI80t+J0muYDp2GRDuIuK7BTRIAkl0GHItXiuT6i3kxzaYfNm8kgSeYHihFlouabxx9QqXmx2jdngSxT89leCDftmpfQGadCT5ZKIHvn2scOuIU69aQbeP3K3gkzu60JhN2CNVjrcSEKfgtThv0f3JRXzP1g7tBRofWRL7\\\\\\\/9S2q71mu1jHOMkFOpObvozZeWfKP9xTewu9vOm5RPbgo9vH8lodtPiaOPabpu9hN2extbJoAbgxwx+ZyBZaOYTWydHFHnMAPWdweTj+o5JVGblrYkAe616PceD4JwigLEE07OAoKKLROUb3wjNiujEvtwtrkIIm+lXPEgdgCIkx17TzTfT8RyF8hq4jkH2TsFcxj+6XpScFCIN\\\\\\\/b0tfbnB5iN9JCgkrfUgNHN\\\\\\\/JTnT1pyBfoiwv2N7CaqdAvl0Vb+eJrTSTcbkDPImK+obZ1kAuanRrlH\\\\\\\/6KxSwVYIuxQDcQaBzr7f4qEyvq5eopT121sgTIQGItSRA5sFWIPiPrW7+dvt8WSskzhKrda0saT2DPrJGKLOW+9DFfdoCPnhInp3Ar5Uh0TqYTmuDcazEQGGicLnSMf4TnYFsQFq5k0vPnVaTmOoIv48tCg\\\\\\\/VqtyHlXWLfQ9BHxfr4rhABbij39+1cJakmViR9KArRhuPewBsVw0aLI7276qbjMYs\\\\\\\/dWoOUPlILjHrRPxjaAW5zNbYInho2YNliSUHcIH7hg5sWOu++Z9VbW9b3FBifxEDwJ6wMJrYNOvjsc8Vb3tlywSaMv58i2XgfoLMtDBHDiIdgRtrSprvW7Z2sj60bnsdE9Kx\\\\\\\/uIrlC9E70kHB3B9FfUaMzogQmyHXHpMDxc43OxOPa0E2d7XXcMSFJpEayPGy5kZpgIrtjWbJXVVeGMipficj3fpYoxEYwv6FBj4ddc2PWqtgX0jc2\\\\\\\/4NBZPzPYCMP\\\\\\\/rfoJ8LkfMjmxivbq2kV4yAoWWUgzE1U6Xz\\\\\\\/G4n2\\\\\\\/mmTFmzcDhkQBt\\\\\\\/9LS+ItrR+A7MsGdmpxyn2Awp9fBajqKR5yhtm+x22y8GLLoLelWzmh60ZTMayghzbT50znpWtM1arXrh8X0TnZPF1crg2XOQKuRSCN+b8i4eiAn5nTwqoKO54dBAdMG93g4MBrxrdIJb1cNqxsbgn4yCadTlNdYtemIZyaHBJODsiBaVy2B9SyKKEs+7I5WLx5VP0tgjLEKWkj5JASf1ThU7h89mJmwn\\\\\\\/cS5YxGZVOQtXhiu9Lv4zVazLch0B0zPSU\\\\\\\/sTMVQ1WxU+YftT+OMzlkSJzEeps79Z6Y71JwUKgDn9eSsb3CDGvBtWZl3kmQYAYlfvixGuehMN1KMMvmXBNjfEEFWkBQd1m9lBrqRiJzSn4FfsrXFkw1lTLVN53bLNG8upVIW3Z\\\\\\\/rz8FaHtOoS0P0TJ4tLGwXWrXqPWD7y5gDl5WKgZeut4C\\\\\\\/pGIqAcTdeb6ZS76CkKpqkR8yk24N14TRyfbxgRDzQBVgfa7u0wqa8cmc0+SMhQJsYEIbAYFoFk0DTpBu6tS7DSXh8havZ6i2czPGyOBVwhYeifDci0KJbcvkcmQhxeyAifn3GTSxHuNOBvD18SkD\\\\\\\/LrxhaSXdEySM1PTB2Y8CpmrtMVwv1\\\\\\\/sQk9TjhB+P9wAcSAosovYnyEdzqCp9RrU5oWWqS\\\\\\\/sCIC60B5oi5RhhVAB\\\\\\\/Rt5BBrnlwGMG0+uQJiVf9HCaGOlWURQVVrIgESJjIZd95t0FmFb5HxOWGFgwwRyNyVw0l4hRqUqP48GHZPZxSrW3yPL\\\\\\\/EZNqvZdqBSywdukoPP7AgoThdSPn8GwWx1JGSCXYpDZerr3I\\\\\\\/cXqzfafNoMVZ91q8AccSXa80afhkhmwGRBJsGvIR98+TEbhVItW9bBGowMEF6b+Uf0\\\\\\\/aH8W4NqfmmdkwaomFJVB+po5cnrzyyijpWav0txUdyN4ZAOxgmKr3szAtcAHcXlvsEGeAC6Dg1JWKebhR8n3l44YaPBdTT9J5YsuJU4QcY1fwl5j3MAv5flK7lIQosW645qSMIgRLPKdQNZq1mSnJofVBa6WYFkuucae7ClXJoIzLYltB6jwLdHD+xY0afOO4BHGfE5jr1roHBAeiniMyn+lgmuGzmGSFzA+0NEC6KGAN9sLOqbk4kBXF7ajY1z9ySYjTJTb+RqKVeG5pd1npZoeDXv6avIe1wLQe9J9vr96Z5OWWpfH0lHklfLIzsmHGtI+JeComcvZDHxbpSaFGgJFU1ep6yDQip7JX8SK1IcnFFRxQQtcjUoMdS\\\\\\\/akLeDGf6nKDZJJXGg8yeX0nlfIftC0mAiolr+BNQS0lmmhy8GgBvYJtVIOB3lDPCH95lGwlGr8pUCFPlYgZU5a4IXxo5eUY64aITLsnq8+tzEj\\\\\\\/+99QG1NoZHWB3oaSOMWXMD7tVRTyjpiLoWtyKf97xOqOiJpZ7YCMEQZ3KZsmlcoGd\\\\\\\/gtcqwiPEev\\\\\\\/OITrBtC+E3RO8fUfO+w8nf\\\\\\\/OMA5s88PZ5Op3dhdN96dEmHF1yoLRPZBwMlCe+1In3dNCigjw3J\\\\\\\/R2wbe2Kh8hfJqNIrStGSWeJYQkJg0KokvecfmwYTOkUTcg0JdCaOJUNtR0L\\\\\\\/mMH7NdIUfcekpgTkXtzIGNw1uCxm0ElNbJ5YBn+\\\\\\\/xH\\\\\\\/KHqMn9C3Q63\\\\\\\/INONGMFrAveB7emzLrSM\\\\\\\/l+hwAnL4uou3moN2r96MoVqcRVlYWKhEer0qAB1H++4Ruaug7QrMEOew6G2WG\\\\\\\/r+fTMUo2P92PpCpi2HOisON0g\\\\\\\/8xYzsJpQm\\\\\\\/0bvoa8jUCZlCCFp2t1BFV8BH3hwl55ajFvvCSYdJtFQTbxIZ3xdkaVrohU2hlIPXi0dmG7RLytp6KSlSMzEDlRVevyQqhy+qrngGhr2G+MChM\\\\\\\/dpetZjaP1PhWWy53Jf3bpXHUL57gb6nqOkAoyXeCa5IdhdS2mLvKRDKhjh1qxq5BRfLe9QrIbbHTwkLZufWD5mtjm9k5fGp16lInJ4KqQmYtpDHy4n6ecuwXDtJaIx4IGnJhut9G1r\\\\\\\/8UthTVJnWV7QJG8OsdSB8E2P32eoVj96HHs4IhkRYmAYfb+4BD0fA8CyqCm1HbzYWNw0asw3Z7UOOb1V+\\\\\\\/qunyXIycdw8d6r1HzeWu6jAV9JFDHOXCb8ET1XZUN\\\\\\\/JCSV7GbzRHq4ufK1R9A9A21FP9o43BVFFP9Wy9Y8K0hpbwJ\\\\\\\/PEhrBTwvKwseQY0yYIqeZJd7L\\\\\\\/R8H0S6K3RW1mqu\\\\\\\/NNjYMSyoW221PxD\\\\\\\/kGd8oFxV+dg9SSh4zSzywpxGhGRNPXkokEUEGOmb5rRFuBSr4Odrb+yilIPgeXa7wkfqYXCS4CloQditcOxRxvewNDOgpq1h\\\\\\\/EU33czIwixJhrXh7DoMNqfSqgEBjL6QqNKq9YDYZTusujSH9KIO2\\\\\\\/NNbzL25NiaIJtZDKKJ0cYP+rVWbWz7Dwm+YFhGXFseHXjAX\\\\\\\/ninHYTLI9uoUYY26sdwHvYgqmGIHQK2oOLTnvdK969tjD7mo2m0WyBEofRWqBF6BjenQyaygjUwum\\\\\\\/Oj6b4N1p60Gbu4M6PU69GEkw6xw+HQwhHnZ+gd33I2iUFiEhV2IliBXDrvqTtddIu8b5iA43RT7RuxX6xHyCDAaDBTErDH\\\\\\\/WRDNy93keGIo60OnF+WtrFruK8pQFUxxcKJsHeX5\\\\\\\/A6j0IIqIMoWtWQwJVZO4Q4rZD32NEUt5N\\\\\\\/hWQIlCrnsEMG0LBVrcuCLj76DwQoQLws5ybbZGw4FbdL63N+gFIgPzxt2Lk5MD+97IQxFr4yJZaHzeob6+2gH2LrZDykVuM9Rd0OFArQa6Xfr3vtpj2VwRrOIxkW0zcBII5Z7jahrCJB9XwnO4Sf89uOjGHGIvyzIUJQPWw4BxaXhHxF19BDdJZu8mHdiu5v8mYIJfn1z60d56Dd5q4uFmZQauAZvsw0Bn\\\\\\\/sCcL05KXAWqt2zZv8ZkrzNIa7Asi3k3oaf7BM9z4QRVBr7zlYOmGqnN0GandwMRJsTgSdob7iS\\\\\\\/tolLf8eammrgvL\\\\\\\/Za69kxx0uazWxcwDWIDT14ZLiBjwKzvFpW37X5bIgkNNRhz0Z9p57rU70Yti6GZJKOk5UfHqoPdN+voSz3Ae\\\\\\\/BEFmuFovZUwlbyfO7nhLCGE+wWktF4TaHOmXJ46LRjcbQoQbB3dhSS5ZGn\\\\\\\/ZwMWTftRHeZwOYR38cBbg1GqaYmwu\\\\\\\/32oLLypSBJQ+uGfvj+OiyCvCvo3QlLGNlpQKyUPjpFlPcZmM9quJTLbmWjiMe\\\\\\\/QKhffI9PuIjMDEDgSZqxrII2HaDg3tS6sNA0IpolPf5Kl6\\\\\\\/Lw4BCn2mdiSePm\\\\\\\/Wrwvp\\\\\\\/mbXQs2QQhQa576eQ+ASVkQZltlAkuJmrqe80A5nrxdihLsSg65wdKMpsfvhHgaLpzIRmDcSvYXLzvPifa2aj\\\\\\\/M0rE1L816PtN5LfqKE35ErQv6dMM6Konsz1w9lczWTVNXOl4u1sCui7WmuOjrdCNDQocz2vEIs\\\\\\\/4zNP7RwRG1qxm02Vl4cWza6X4UnIwlnTtDo0k9BvASy0LuCSgD8npapVI8s9e8PR4hGx3HcswCd7dDfTV0jNt\\\\\\\/umQFmjeBqabzwlH71JRSY5OdEDRxpLR9FT86ghOEeaOnQk1LyazeyS+YxaoGVw1S4crnMY7w+gKF3nS0SVBrMPMXdSLtnHpSr64CWNvgNw5licife\\\\\\\/XL9U5jBW5LV8gAei5XqTZo0Rxd0ZE3+yrU8bhP3RLfv0voO3\\\\\\\/7uYoYaQvmrLIDRa9skuTLxtWmtHNqt9AcOEEdGBs\\\\\\\/\\\\\\\/R1TxNSIFWnzurfwDZ2iNmu6RmPRjtFxSRzkxDD1LBmh5Xrq+PNvjTzVjL8UVHpoj+uPnNZLH7CrAzkNlhs\\\\\\\/kYUmIrvepqWyMT40pOCtjiFof59gWcaE5hqsK6tbJhN7L43sbCEO4FBOV9sVUElsK2Uoe47a2TmqoziP2Li\\\\\\\/5H\\\\\\\/avtXXVTgDvL5PGQwOFBeMk\\\\\\\/cDbhKf+BvnciMB7gK2bwKizmwBEGjYADrT5YCAkQNKC\\\\\\\/2JEDweWxaeg\\\\\\\/KxK3c6VvUaGB7s\\\\\\\/2bBOUIMVa41k2CsNkiCqxQjpXgAfmnL52hhVybrVxgd7f6jLIiY0RqNNlLFC5UXNQEebl5sFQfxbaQ8WX3h2aEJvWZxHb10OzKFRNWAQnnl\\\\\\\/aHD6S0r9KNv2AL4cOKQdTdwZUIs+viwdpGo+JhGvv7w5lYCPq7IqoRLgrftCcqqZ+rCzSIGcnoWyP9TiUDBHkNZudH4Fo0JT0fkD55ak82fmr+0M7VaHt1VOvnNHojNf8VQGeEtRqWgnxhyxAU38zLZvHI0aXNNOBn8LxiKL++5zWNPZfL5KC0Nst9IetDC8E\\\\\\\/zvg9FZX+UnFqo8v9byxMkSlukVVma7XcrCGPpA\\\\\\\/NikR0mgTMrpDDi8TEJD\\\\\\\/B4KOLSaRro6SyfIYXEDSm3mZZ+3of79EO7EcXtSDLYBSU57844Ct37e72yIfcpqgSRpZnMeq1tcoKV9cNGrY81v1\\\\\\\/q6gWU3W11kdby4Z5JtFi3bT9dycn9s7Px5NeyTPSRB1YV7HRXUeYWJVsXaVrLC5WcAULHyBwGSU3G+knS+iMsKk6PuQ6ifHG+khfFPfeOoaLOQpNHRJzdn4yTcwJch6w0eFrBzfhuorXj4gFitrJFaXd4ofg7\\\\\\\/vx2wTWNQpr76QGlW\\\\\\\/ogmIRkCda11VZUKO9ii8AqdNkQfiITa2krIpFmfIgq+6T\\\\\\\/Lns0AA5+HQqN48Mo3piIG6sYswALPHrwmGmGJ4W8YAVCkr8wn\\\\\\\/Z4OOquoBSnD8EiIUwAIg98KgrG3L+ODbszZKJTjqP\\\\\\\/gRc0eXzKqde9OFquSeQnlifaJoIXxZHSQOrSKdnzrKavL3XT1fgVMaOmydrVlM1AaEpXxIBO24XtQn4cMCKUUZUUF8QZUGcMHxPddLs4XiQesisrJnNx5Kumu1HOD7+5lrXr9AWDzxLsyQfKiN8qsE5+Zoby8bzg4jwu1safFuDLymg+xu3ozS16MpKdhAl8+T7U\\\\\\\/XwRvDhL7ObnG6Xs60S\\\\\\\/pDh3SC9kSMwx6grn6bdSb\\\\\\\/o7C\\\\\\\/K8T+usknsmlCarTGNEfB5WYZ8QXU49KemV27EynQJ4xg\\\\\\\/Y5fHqsTDaxRfVLMs29G2E8BCL0Xilx5stxbQ7hMtT72p5YZ4nTMTBAU\\\\\\\/uY1120fGNhGHdrNtx9cxxVgOLCTSzlhQQwvIr3JgpavWJpMdNYLDYmvKZ1lW4I5r\\\\\\\/5LGLXNHL+SzBCDaUmQKVm8Y0f1QxYGO1tXO6Rb58nI6iPc\\\\\\\/TWv3WcD\\\\\\\/ACcyq30eAF6Q8h170vXPCxu2X9HBXXCflv9s1ZP6y8Wia2lfQt\\\\\\\/qx8fgyIehr\\\\\\\/KG\\\\\\\/8TYIuHllsK0D\\\\\\\/oI0bBndhJrPXSrOgtoaRYXOCEipXzCG4b8Dw3ayqUUGa711w6gv4S5u9mF\\\\\\\/jHKb7s33dHmB62gp2eAb9KOVQ+4xWP8L8MWjY\\\\\\\/A+7Hwk+K\\\\\\\/4+yFX\\\\\\\/Wy8z1jiiRAk1Ptq3BRgBgLtWrgV0wEbtYXEjyFAuukl3pnfpTxYEUWecwLpBZkDUUBNhi4VHYAm65aGv3b0F+pOyf6ydH1Hko5SamnbZ\\\\\\\/RO9dJ5DIdBJ9CTh9VjVHC8o0SUNiYJvUvcwpwAD8i53IZfRgKGiaHGjiS6rYZcGSSxHjsSSXWbW4LlKmeP4kmQcat6bz54dSOvTz3pnd2OLlcfQNkypoTfjUeu0U9r0M2xIqA9iHFaNgOxZQvgyDD\\\\\\\/JLFBCVE8XEvwyvGXPLOmCq43xTmSFl6X+iwpSadnuZWH5AZdtk\\\\\\\/leNYi29zKzveGGgwlfVoALEGV7yJNnLKDOOcfIs5Mg+n0koKnRSx+c9S982CN0xXzFneiPhPuWOmk2PDmdcYZUrVrbNXLs+MxlBa6TV17BwlCZkZapCgOptriTxqvSXP5jtxpJk5fRJbFf8ZGwmkTQjFDswrYpBRYfcEEvn\\\\\\\/42nNomSHvMt1+iVGJ+axBH9QjfqFMVA5rR8sDurvMWHjoadk8Ks6YQ3NpzbDwf0ss3Kq1ccHNyi0NlPVaqUiI1Na\\\\\\\/Mf+wtw997s3JlWVyjQT8obrPwcOcyMKLEsTGarYE3dpEPh76E9CtcEp7IACtkw09z+HMhaTfcozfQS5IOppQ+K2swR3AS5acHH57ClgSOvLpA71vDdOb\\\\\\\/CRoYegC+W2Zv\\\\\\\/j3gewK4ebu7pDDA0gdvRprBY\\\\\\\/34NHIdzqwYfi9ghxWShKm9WdBFyHtxZHky\\\\\\\/9JrFZzrBJVfJdNo3yFoXllyXYtjPnYNqV9oZ9iSs4kw5MraEByrD7YY56FN9UmiS7EE0xScJPW7G9mZp2E6v++zG7bWWJS1a7opLkCRcAqec1VlBkDZ3O1J4hYotdZiqOvlrwEzaqjID\\\\\\\/KThUqUlgEO3ZqkcpsAlG9+uBtFcerSnrI5J1cmKMoI+EJ4uZr5cqFBpyC3ocm8Axb2qlN7EqmfMUod2O3WxpvxU7zHxmOjcFm5jMzVLXcrNls42oS1NeXtRJDq\\\\\\\/WZDYuzqGDniJqRj7oK+5EL5hEq5\\\\\\\/cNwbCeQelTxCO2HIugkh0fuqntrrNHuNVjpdNJvDNJwfubLv5RR+EqvcA4oyN83WTFNhNdDE5iHpZZJmABSki0UFoBwPu3qrGzf9kzFPEhR3cXSzlvNrulEHmGJSPC52u9qvQADt57T8tnCSsemR7LdBUN6ib0uJsCedcAhHAM8YP27f9bMGP6xslyJRTweRl6Z6+kAXSsmflV9NjXSTF43FJGpwdpBTOaVYro6H4O1Crp93LO26pmlG0QvsVG5Vvqy1Rbyy0mzo3EoqWGX3fxB9GJn6UanyMPx2ztUJnlcb5U3KwX30FzUgHVCbmB0AVKt9XalfS2a6UCnQ6IZhwntu8jXkrinvcwhWXpplHJc\\\\\\\/PgHzEV1LtkIjImu4TGeaANU+znjFJnwvzJcO+iZL5rBw\\\\\\\/Qqi2VUpDYZa+BuVKD+McYlc5GVuA8lksGZADMvdUM\\\\\\\/3YAF38SG344z9j7E7xDZmQmPu+GPqDndPgYgcjiyhN3pedqshrPZoK5upJ2dEL\\\\\\\/buNSp6qHAU2QIBbfUh29P1jJFCtVgkclpGnUkT5NUkxr0V3sthuSG+ep4UC3R5A9dfBtH2tBfU7UBUvUDlYvcSPv2Py+wjThLjAVPOoTjQBGaa8rOwRHvHdnGkdQI0xrE3tSStaTjm1RJAWLYeyLsGVCn+a2rUnwdR2pZTgFZfYfk0gVWfSKThPc2PtmK5iLjC6kmZNSCtWFz\\\\\\\/KYJcVpsAbvlnpuiIuL33C4i2uHLfzR4lw6Yir0be\\\\\\\/Alfb2JIlNAUrRVAeumWGdkHfyuGbOl+iQEnHU7TMcPFXr0KgTrawLA1lPyIHHeE3iNcQOCOJteB+X06Vn6Noa7kttkk2IFFEPrEBw0+Ta99YfE2yu0kh25isrvBBOzni7d\\\\\\\/X56ZLK\\\\\\\/uwQje\\\\\\\/SoucA90Lyvd4g\\\\\\\/vLHiK3OAEdZ+tqGh\\\\\\\/8nPXEbWMLfSuEjurJ9iAnkZNe3Df9yR1CZaR+rkPfHyc1LgMM1UFccwPQRlipRJp5ehsOD0h\\\\\\\/UL7J1KT6OqYUel2Xa3xL\\\\\\\/HuF7VzGwf9PJtG1glugL6sktFMnZG1OBtXQPyfgnNIag8hCqQO4MBHnzdAz90R2a1Hw7gN2an3Rp13Sh4mogzCLW8Y0ikk6exH9xPDivH6qR0WcqwadIBRZk7jwEUAoY1gYAAndOOw\\\\\\\/tjl\\\\\\\/RZxJJe\\\\\\\/0V7S7ENHu19JzwF084\\\\\\\/OwJyW\\\\\\\/zv4+YlYgtZKP4g2lqsRd0wVkO7zqgG7GToR+iF2kKgXu3q4lBPodTNkYJLJYEFAsKbD56fNUdJGbSVdQJLH9DSJ\\\\\\\/ZWTlkvHwazyZ4drUeeyk3prU0hHVBeXLvKDWJdoMokX+ezOaRMPaacSBb7woqW70IFANeqIyunJZzC6pTj7njhB8CMZ7qgDtEEU7ZQG7S8hru5KpViAIIchL59BTA6aASxWzHmhRsHVRN80GJ5BTA925kNbxZAw+c8rqA\\\\\\\/uO3xLhJ7I6gzD2GUlgQeERe0Gnf\\\\\\\/3dg\\\\\\\/Zce7q0NtUA1g6GFnYiQsx5M7r1JS25ocIChOVMnLA82IONisnbCvby\\\\\\\/Y99wCRvMEJpAeTsVLi8ww5quoSEewz7HY5WQVMeDw8qbvvz30JJ0DvtkMM7oVaRUX3RwzQ9Kden+S34T2P7Reev0hw4ueu46jN9QmH3Tp93O0364CsMvAa5r2LdweitZDBvMUuAHsn8r\\\\\\\/Sm5iCeC7\\\\\\\/uAHKnoLajsuYvNmiYElIm4ds+I+nnh1nH9pd5NssA3YvOfx2VYedyq12p5T2armuMCoD2gtX1KWO2XYnWF7elUED09k1Z18HRpOx442YRx4wqzPyXZSmj94QA2vIwtN\\\\\\\/QF0LgJegbjcWktOELB5hKQIMIHiipnHSrbWmlZYvopt872hQn7YH9g\\\\\\\/OVJB39Gx5n6Rq8350KJ+L75ni5e1LkJXMJXWBEG6dYRwJDc0u3WgJ3VndMmvFQVURQiVx3ujBafRjSHjON8ufrzzDkB5wIBXgca5\\\\\\\/7Gih8FwIVi7xf52TtN8aY9VN4YQpSh3aNOsZyEsd5EYZnDXWUTnzNpolknLHERMNxOFtfE7tvysA0\\\\\\\/6o0SoDibGEgVKgIRHZoYg23JSqI3DlcB5Lb24krygP8HxRDn0bVMjSlEj7szB79EghAV4ORNY2DvckEKSV0Zk0DGzT0SxYFHkc5Jf\\\\\\\/pDE4TawWTdw12oDwl5a37DlOfjc\\\\\\\/q0yxd2+F83WIcuYMePfrMYPX2QK\\\\\\\/mRjvz9N1YOTBWQNkJZ3lMUNvwWae7vVBQLJ8IjTuw\\\\\\\/kSuiDvgy0N6TGOqFbc4Mpo\\\\\\\/ValyFmGtS4+HD5xuAZ\\\\\\\/QfagD10ep\\\\\\\/w0Kcvkq+5qfk2ND66GDQiFgVAv4Pgg9lLqUBb7HuJ0C7yHBCEpYBCmZbE32grplDOUL59btjCn5tGkh3WvLo5vxg8rDrU67tX0lFiMxM71BgFEX6Voq7j1BQ5pD\\\\\\\/8CTm7yy1z2Xel+\\\\\\\/axcroTh5S19bX3nDGi34R0p16\\\\\\\/fx+LC8XN3Ty3JWOmEmig\\\\\\\/CY\\\\\\\/pzzuqG+P4A1oD4SL3XYdpiaMUHRinMa7nhJlwRoIrWvE4\\\\\\\/qE7xT2AnlgQ6cLyf6\\\\\\\/DvMgCkD1J5UzT6gWn6mokU1N+iCgf8dP\\\\\\\/v6rf+NGGB\\\\\\\/K+kvEx3syD9JvHZqW0wXMLf2PMFEXK57CAW1PNVdFZFoeKlEsKjNY26h6\\\\\\\/0F7D0wUCJkO7a2ckRXVodBVH9kuqDyGFJG29uX6lM63duZwdWLwRzqPV52j+Lr9kEBGVfVGhh2pSAMnSuIxP4Iuw3cqKgg4\\\\\\\/zSMj81DtTVkRXnuAIhSBqzplw3IS9RnWJEKr3O+QS8RJtBWJv8mkWl+lvV33KY9RPu6yhP4A2d9GzW+XhJSYNmMHfOv4jeTfSIkefdWfuE4vj7NY36AYBAdOzhQeJL5BwUL76Yoc9wq392Hfnqj0ivSKXTuVU8BDi3wQ5ZMAJF\\\\\\\/pnmHSkcDxUuVT4DlfCfq6hPy5Pa\\\\\\\/n8\\\\\\\/AybJ3zTJ7C2M7jx8sYJPU4lLxQklxMh37p8RSA7orao8Tose5BJ4RFJTx17RHmB3gKYSy5cpaK5DYjNM4EqauRoh1IUhQb1XkTCxu\\\\\\\/0c2BryUKz14s0l51XPHNNtPdsT1RTP+NvQL5fyO4Uq2InByJmUEhVh8vI6wUs8sgMuScmrnZjaqj2UgNFhCLp0vB2DXltjXmdhS4k+0pcIm\\\\\\\/\\\\\\\/wYAJX\\\\\\\/ph4k6KmrC8MYm4mhbTxPr6PKPK3pLToGlLksiK9Ichp6F6Xa1Qm8eLkqsKt0bF9HOIXW3qSG1CyvKLc0nuOiIsH100Gpt5GAgiqRXhO+Kadf7blpcYbYjT5ImhfD\\\\\\\/TqS8rRyrAoZCZiuFRnJynhcXl2MpuRxNWrGUwSk89f\\\\\\\/t+hXtqL9hK6EONu1xHwHsgpJnWZkIuSm43g+vpjpjhF056piQ0KfC6wmZdyPsEwEgt6s0qT8TmzFbVyeoj\\\\\\\/J5nkAFiY+2zDc7p7QdrWsNAtRLLn45c4xVeHhAD7Sfg36KFdkEoANRE90MoIfwg+57M0jrSLKkGGY9rX7TpkC82F3\\\\\\\/dAFjz69ApZFM+GXCwtArvjtNxuSR9CqohTKg4AoxGvJw0cWC5cynXRwsQWyaAQ+JAXCAPO6Abx9NIEpl57p1JhUsYYH7HBPCgon\\\\\\\/L2vMuhppyB8gfBUi\\\\\\\/4PZPizhSdgr0\\\\\\\/SEJxjJ+W+hpkAO1hcHhrkjUp7yg1ktMe3PR3ChHDjQp\\\\\\\/LTYa7shQxVzw62qyK\\\\\\\/aXWihSAPNIFH2PFqyAh63fOhmRrMY3ow8IMx2gWF9ki+4PAS1Ofx\\\\\\\/UCRRuuQMKLPXopB00iZagOrrm9uMyKbMHCYm3q2iuvannbH1RJhC9N1LPebI+MI61rVlaL3diU4VC9hcN6LkivDFGGOX0TbgCvd1l91z9uy2tKxZ06GL5jIHFn5TS5nk7bKP8T4wcSftZRu92TByATF72NfpEZn5fS9JtE+yuCGFgb3pIyGMC5\\\\\\\/feb30iUD3GzfLJsri298nAQb0fwOmMbYtTtkHRM0T4F+p47gGKzY5oe47cclwwqhADfrzT6YW1sPv+UHJ2zNG5CcQcIwuabUIDV\\\\\\\/CgEcMloc5\\\\\\\/hilvJam4iIrbPQZkV\\\\\\\/8fiwNgg6bUJ9\\\\\\\/2QYu4wcDysg6vAgkuayPQFKP3HQksQZRFjntRUhs7b5atGToNXUmE\\\\\\\/95V152UDrZsGcGHgMZH0fuNZKjhaxqvqdAFXO6uUpI0dhsFWa7qwrFAmEqZgQc6Vws5Czo00VyO8LYlSNrw2vPd9RmT8lRDK4ITso6TApOezYEenIgeafmu3ysXBjCi2vjtQrb8dHU28Y6YKYhKjY7AdmTIV8mYBdHZVrOiNJLBBH9YS2WAS8JcBl6tgSet\\\\\\\/4GTGWMzRy2zINei8ghbDw+FkexVHfy007oViE+YIWNoR8abHFTvXLY5nFoLPBrMnNFBAiWmMN34WBlH73kYAvbR+x8ARwaS8+HwRJB4MTitpRAgrPR8\\\\\\\/Z9mSsrYcuZLDtoa\\\\\\\/VzQR5mTBKZsslp9kMBB20FTAbkz4MtwOcDBWLBCEVdMTnHN7j\\\\\\\/+5IcEpbjBvSiOLvkD1UkNbt\\\\\\\/OgyS+0MNfVx5zL95PP7hCLkXUJ0wKLBlUCPOGHvVKbI1pbM8598kdZSSInWLYXNraYPiwrycrZoinCAQtz\\\\\\\/MBF4nZj3BPKxj01sSYtMFN0SUpQhDtk0EJS06o+CLAqFKTPlBKMVyA7jd6fM5xnD01pstscjkjdiEGfMFC2GBjt\\\\\\\/dvQDja4+D1ZKiEUTh7PIxnHQG4U1VTpg+Z2Hy1MVufvepEvlNIZp4r+zSuetDH+PvNld7BRBtdvqT4VSzMwq1JiLMz89HUHlbYtAAY25bO5qiwkhOKZ1api6KoyJrO8vPDZn01zuPwLRQMI1VqZVCj8tm6x5I6ae\\\\\\\/zlabf79GSCC5k+Z0T1cvjeBwjs3yXJw9BLNVTKAlIYrKDwEvPy4tJABJsQx7grjqwkyvG74PzQxxqv1cC8NSCdR315wajnbLZH\\\\\\\/eBY7n4Qbt7A65yPg0s\\\\\\\/kexfwnb+MakBaXqlSr4clkUtjgYcyt5aMaMX1BBQFIW7\\\\\\\/fLlbfbksLQiCbna4ULp4Y9IcsW1WXX7bGeaO0MWcsg9w3yCEESBEOnoSk6Pwg8jfdUxlQgSLhNkxzHkOKE8vdJXL50YSIItAkEowp\\\\\\\/aic\\\\\\\/1DfdHYSHXRvBiKKIdn1dBPUZ6VPFSGTTQyEbUZF6t0G\\\\\\\/T2p9MFFuj4V7PLR\\\\\\\/p9nbyNKjweLuNOmUbhbjkJodffwZyX2UFzEFsryYR2MXbxnFHOf7SPxLGdeAreuf29U0W\\\\\\\/lueO1ivWaiGje\\\\\\\/S4D4kBEJVC9hMW5CHtKrvD4HifisXgodGkewwt1hNIedb+\\\\\\\/5KFTpYUVmrQdYxbcP58kTlAolIjDO3BHrQPU2ylnTjlLC1rvAZ8iHdSNuvtWOTyqO7zxtfoGM2ntji7kCiSNnAJ7rez3nLllepK8H0ZL42oq143UIsuBI3sBXCVNtUe6a6mQfBDOmH5ig8F74MqTFIhPWtwffC5QvNjMwlhZohTiLzAd9fegJL7HhPf28gLoJSJetoE9N0gIlJyu6TFaxf0P62xp\\\\\\\/cmTJZ1XGkV+qpZRFjNXRdcenvGcF2e7z6JOlV+hMBIEXhTuUsbEKkLQCXLFE6VG4OPtNuUz2LznweVyv7Aot4rH40Bj80xUw5lN7dZG4jRlicc9oqvJNg9J2EaAkIOedAn4qh8PGHV\\\\\\\/omHJLNSa36vrnEBEsYvtZzo6geBhdtnUE4PVHz0PbFmjxJk6i49Y4dWNBQv1ooliJrgmHhjhoYS8cv4OnKsMNzSVmnX71WtTK4\\\\\\\/gK\\\\\\\/2sL1o+eDHp0d1tmCYPSq0+kLK68PWkniL6H3g7cR5a2VwiK+FhwC8SmKCHXViKZS6gwmUWMhYPSlSIZabrD4Io\\\\\\\/vwAXehXRzzYGhCbrGMhgKwgWtZy93UISmBihGlPNgjBCxJRPU4kTKBXC5kzw6nw8w32v6rbLafGqt5ajTE3fYuGOW62gNkwI0OSsNLb9ynmxU39LtZhfI0Rul45cBkTSUfUOuKXlW8kvw4lqVdPURr8kZTMPb3hZWQozOuHIRUp2e1H9YRnbhvgLJP2gWt5lLPjkNiq+0AmMih1e54VRCHYRyXgntSy7ZC6elPW1yd3RlVjV\\\\\\\/K9raDmZUqxPTg0vaIOaXj37YIA3YgAVtYrG+2pGf2kEh0Z6ob7YSwbItgd9Iry+9bndTHytpfE8130iSt6yJKtMMs6wkVq3UkD1LT9Nm3esa9PyFj6WY5FXnqw9neDv7ZngIRy4AbjGFm9\\\\\\\/lRFKRIK\\\\\\\/0SFCaF3HsCo2FpuR9Wlg+ITwT\\\\\\\/B81AAcwpoCRJ4E+98eqVJr\\\\\\\/OGddwfsV5hvXH+IP3kyPL01YBsDQYTXb7h5nyECxpHrH7GU445RexC03xboQPJ8uTEe+p2pB9O9TmRbpcSCSHXOnf20w\\\\\\\/lEeL\\\\\\\/ySAr1OaX3OZvuMBm0\\\\\\\/0HI3qTRDxdDOoLwiP8HVO\\\\\\\/G6N\\\\\\\/ZucNav\\\\\\\/WOW2rkEfo7YSy4pUZGKvfM4V6rqYzXbCul5sMcF+E6z+O9qexK+DN6fHBQktSlq1reLkcZurG1ktD8sAa9PuLzkmYQgOqlVVIINX26iXDX4OIZ0J+cKIavle\\\\\\\/GF7Or3+TPUJ6e+odCOw7qYZt7qtRww3SjGz3mqaNrOu+8XanGRq4Jy8Svwirunz0pl6wuBZJ3d9R8zFuidkdzZbskEa07PgfV1BsbAPsjkvz5+1KNYT7IRfdrNJQB+aQgztkiQVq4vFxZiLWezTNE0o0sb\\\\\\\/VpZchTVDj4Jwaaj7gahPEkmBhvCkhvDmeh6Z3mhVYtztwjCX1CKpEvTw0dK0gcQT\\\\\\\/RrU\\\\\\\/sGJ\\\\\\\/bxcTXNEZDl2nY9RB\\\\\\\/D7EGucJOFBxOqfm84JzQ\\\\\\\/tRqW\\\\\\\/HE1IOkg81JOoZeoCisuiXWSZqhdO5GAb+TTDUMA9Db+TixCJXxfbCHaZ\\\\\\\/HE133WfafALqPwKu1xNmS13T+z0Uk+QU9cy2bePflGUm8apCIY8OQVibQAafg6aCNv0CTi2y4hSofrb5crKosWhFPxGRJqrgb6bX45JdFpX06KCA7KPg\\\\\\\/2tRXVB4jwTPg+rWvBs4FHSyX+xF6RwqWL2lmTPKDQAiQLS3dqAfohPiw2Xgmc7VHvCEFn2\\\\\\\/si68jNGQzztZQEbd7ysx1DElSJ8pHZHlLwPLQu9w8y4L9B5HF+XHnOBTiK9FFNxI2sS+IwWt0DejAIHDU2fmapLERUQpYhl5a041b3SPlZqKDfX4a\\\\\\\/keX\\\\\\\/KxGZEUGmi8nzH6ZpHPGR5uR\\\\\\\/In14iJCfSrbbZvzKR5OPpwy2Pv\\\\\\\/irMiKjuZ4wnPiZ4iO6ljmjCmRbocaG7ToaumWgJlXjySTBTFV12XpwU8LitGqbw4rBdOikFpBFRxmfdc0fQr7EY8FnooWgN+P+uYC6oxaE1LkD6LMOVu7aQmpx2cFQT1zGSGvqMN1bQSwkv4ciM8RrihgqAn1yZXTeCjJ6Vwf6XUJ9lZyds\\\\\\\/5R+SpCvcNjOtgIsGdjRcRyNP0Z77Jfz8ffN853kpTwUTS+CQ8i5yZKE7U4r9I5rwmwl2evRt8s0xCsONWao7+e0uRUlAU9qVybvPIFTH0V2fQMQQE+5SfMo9iyBGvXUjO2jjOwusGJfFQ2PK+dMAq6oXGuXNSNZ0dXLi+DBogNG6rRsLxBf0Jc3X\\\\\\\/xtk05S03llzUm\\\\\\\/dCfYb3JnE83zrj4EOt97FICUjvD1YbY\\\\\\\/\\\\\\\/OybWpN\\\\\\\/5pZXotggqOJQTwKt896ZTWvuuOpoY9IfuCfUeluh9k+xe9mYKzopwFtbD\\\\\\\/dPIy9Ds55tJeLXvxpaj5ua5QVYDBWBwjFtbjQevvZJXbfKWj0suU5zx5OEt2DUkTV4qnEXCZRceMXSuSRK0ielUGczgBMoc\\\\\\\/Io6FolAUl4FzpcZX3Tj9lnVeJHVDxFPE6j3uGUGzh3rVgTEPkCDwAnKF\\\\\\\/OsakRyrXtJsjZ4qINMStFxMOWRmM332ofRy4GPX1QqcDvxExhPOZXmz1956cm1\\\\\\\/bZ\\\\\\\/ywb\\\\\\\/FZOpx8jBYTl5p4HYf5blkRpef7cckmlG8DCEViFQlVdSTMcEHvxnRsifgXSI6Ixe2aCi71jqkoMzseHDxcoq0T1Fxf15uNfR1cM0PKVkLwqhUgCFxkjjMTohhuK8Foy35OK8qTlAKzQT1bMkRtBnfPzcxPy+IflmPT2AkcIHOWCcgICva0WI\\\\\\\/W3HAncjy0cIimRV8fi59o6mE15\\\\\\\/XQaJ1oJlf7B50ZeMOCHK2dD0+teParwFmwI6eX2M5XBGIBVHb\\\\\\\/F7zgLCTTzu4hQoAM1aHkhEKpYviBBOrlobTz09AIOJg7UhH6Y13x5YoD7SigkWmmnm6sWromF8YgE8YzqM5aXAnLwDsxBXQ0HuR0KjhU1aSK3xVsHLAu4YsbJk0tgm0l8ySlZ2wLMI8Dg2qntUZthBJpjKkc0TJOWusCYT0JBQ8muDWB6BFN5uHA4fAlY2GbXxSTCpdRmkRWA8U+25vzK13CGCDKl5l7VX0Yc5+WSX6l0NnD9\\\\\\\/2naIuaJub4XKY6dVK4ESXnMCpKRYNlAs3MUqDfgPoDU18JyQlm8JHp+2tFcQM5uwHUewscoeuBTlZSslHLhD1+jAmhIMs27+7MFjwUw7T2bV9wm1aHOWPHlfntlXfqMjGupKKIYVsYoFzPs5C9qIUJtXeIIO3+QQE8uXbocszGXkikR8TmYoIGdhdMj3v8xKkKJvu3Yl+8zvU1l1vo5LgSAVMGPX7rF\\\\\\\/43LW5oIKHfkjcDAzWn6JrNQqaVFa6bXUhvr6iHZ+wfFxQR4BTdHXpKbLo0yculk8WbxdYfx5V1qRIbffXBgROvHW7g00ZpOlmzq+i3rr+EQ212Aa+eF7pDDoUJTd4VB3VEWUNADZRlBYBdksDEtyIP99+z7ZGjGD\\\\\\\/LQipGH21goe9IBPeV30+4qp41rSZa1RyyGmnyMhsGC9vbTzvgzn\\\\\\\/VdF0cPRcugnhpGGvoFH3kgajvCceCSmUqvC5MWiuaM\\\\\\\/OVKqE3Z+c7V08H71NTmoXMSeZ3iRlIJWf1+t3w6o9QnsCoUgbuMJlTye9cl3SKepKdE3DBTHnosSZWH0+kuM7lqB5oxgZirb9akdd2oj2EqQz6bJxHZI6IpqR1tKrohiy4I\\\\\\\/tpEEV4947jSNLBqG9RmefeN\\\\\\\/WMRrytTwD36uZ5iXPHdKJssUx6bKoy+8tSKz0PzLjgMJxowcsTzw6KBxzRdG\\\\\\\/Nj5ZfGYtE5WIBS4fLXhM3rcJeOQ8jM8ISBxKsO7oGPJm7MLFX7mGk\\\\\\\/Dl\\\\\\\/9aICvtSKRUKM1qqc4aFhcbNXW09QAkpY9ER0qhFsVOw0KUXOXGLyenYknwFClUi5r3uE2pv2CbWMOYk8pEdr4bmQn+0donPrnaRea+kNNnVeFuetJH8oSC2zDggjX0jpyaEca2y4XVo6bVTREFN9bsinuNHZj1U7z\\\\\\\/lyDlnKW8O+rq7yfE\\\\\\\/dDd5Auu\\\\\\\/\\\\\\\/Mb3tGnIWTovqLcvl779hry70D\\\\\\\/tXQBMBlGgeCOZ\\\\\\\/JwHBIrJQA\\\\\\\/t8nIOsCLBfrh5qBUf97Hvyt8huGVDhCKBVd+FhSA5yHMI2fjj7Y73gU8gyLZiSEQxI\\\\\\\/5o2ti7WkDrVW9IF2BYwLbc9uuekBtnVvWmvDCJ8Ug45vG+q2lvUgD4J1qzJ59n2ZPgvYkD+YvlPAbw9wY364HiF\\\\\\\/P1LnLbWUrMv3oSOzUFqemicaQkBfGxnZWwZ5WPWH2NvUuG0jLrbHis7cjPJKjIkapKPDo\\\\\\\/hCcNVz66A4Ka45kD0zce6cJp81sxt9JeHfHkzZ3loitgYmftwraIkc\\\\\\\/YKuAaG8+ObY8TtoDPOUmiKPdbnl4jXPrTuRH\\\\\\\/H6g1nM4w\\\\\\\/rxns2LJFfqCXtQqADZE+B+paTOHrhZ8InAQfwsXby0bOftIW\\\\\\\/scz6KdRUHLJvFcbyfrGtvaljNJWrYzHU0KJdU8H0fZJz+VsiBkVHixTajt0xjU7pJsnX0ExE5mWjQJ+xXFUXc10fU5TBLo8ytItzLsWnMzV6fV0itL5xbNLjiRvX2+kdkqmW3A0lc0TAs6xN1CqqfFLaRwPJ5PfveqxOli56GIY5P4nPaUUK10wjFP48Py0YyDVNjllDn4wv1RVG3klG+WwscrqDg1cmzDo7hAjLPPyRNO0d+naVmEIigUBgZrB10R4q0TCNOg\\\\\\\/HHOXsJZ7cifIlRtboc1Z21PPt13IsgBOq6rfNsvH5usNHd3\\\\\\\/fen+FoWdLYTEtCfcXvtz1xSlUFWtW\\\\\\\/72BtdpONqXN35CaUDTwC0aSpUuuaLIgFzXh7b42ikVfi1wgy2EDX8hUJLPT4lur7ZPgPVTr7VZxYeHjzyFnioqZgVnKcDVIAYStHWtaUIX5vx7N2iNDv9TOcYNoOlpbI7IodCWr5CEyu2MysTY2gdFOi645OAS3mvp4eQvldRM0S4isyiBfL2uoyecxcBWRGuhKAE2p3Cbj+QzzC6iGkGTrXmKel3lLd3zN41\\\\\\\/6Mxe6dctESoCpBU2wyn8+3IJN50\\\\\\\/2Y184ej6dxdUd4zS4c3MZLjb7ViyvJxbu0mubS0+cp2FypP\\\\\\\/eNPf8s31EXtsDsdgq7kw+d+7wB8Q1F3zwaHgRsnt5dVXzCcmx7Tc5Dv+oZ1iJRzi+qEBC7PE83+N5x2hKrdAQfK7f9Ft4WzZfunJ4BBLQz660hbg3EQ1uCIWYXdFWI8mcx3IJsrTOfUHDOeN1AhVSx+gLg7JF4G6exlnZgbRdyN0zbAk0XDtv2zRHHcXfFbrb3p4hMLGRsRdarGEVcZAuHg1Q5f8KiDMjeCPvFqA8Qx0JqPSwyVhuBHbd5ZWIlaGpwcwRKZ2zAdejCj5wnHms2T\\\\\\\/M\\\\\\\/a5V1A\\\\\\\/JKqzAOemrHFNoHKd4Hy7PtBeP7gawm2auMzFv1KVHPq0P0AgGnxGuyhJsREs7IQHi6EdQ0AcaPfR\\\\\\\/ghPt0tal1pUAOeeVKzP4Pr8gBLDJDqX+2\\\\\\\/d+GT7Xa88YQeRbFcMhmBYPOG9jIRhnxkGHwTAiRFq8RT3YqM3zqCwLkL3hns6XOED3D0TqSWoAKUE9hALVNtKCjB7W3hHeaQrU\\\\\\\/o6922LSdx27x1UgWkcAxVJH175P3C6DrFwTwdAnVUcEiC\\\\\\\/zWFJFbcHOlgnueVhGbT6r1QbF8B97OVh8qMYT+dqlYOIXpxOx5l2oIgXlcaIdHp0v5DRaQrgfxn8ZMy9Nmh91GozPf+GC5cVYqidY3HocKvzdJaSwDgqQt7WmWvxzFPkOsFePxUdw9kA1L7UpF9gNk5n9EPgqU68JPr87LHRhkWvB3s+UjWPZgh9qTr1E+9pN3firwBmukRxZ+\\\\\\\/dIdOjKfIfZ3K4P68IW0UsRFzMvtNG\\\\\\\/nwLK9mKNqPfjyPBCMxfv2zJ95urwJBC9z4KwwPgl9W7HoymMzO5mnfRST2zvswp+inlYQYDfWyyuckvTQo6xehYbz37J4zFwi7RaowIHe6XU\\\\\\\/sBly4AG\\\\\\\/yc2bLgNgDtu4XGE\\\\\\\/rZnE2fZpUD2pvL\\\\\\\/yh9i0ofotww2CqbPag+\\\\\\\/vuC4i2U63QF0dmP6xg9Pubwd+lnqRPp9d6u3ZCvgIQiDYDMeQOcWz6yB0jjXxE727IFn11hr4psx9FShPxEJYL+YpY47QhXiZL5qOeDQfZuaYAXGkNl7hIFnrMwaOD7JTpNoVPIXw0qv7Li04qSQmgYdcoJxhFzkeGwkkJ7yNn1uSSLMN6aoVRVXG2dIiRoazm06WBWKH6UExaUeBssIOnFsEyRv+6QeVAmbnksDBRYwb29v4uQ\\\\\\\/6GcdoAqaZ1w4\\\\\\\/3RM3mCInMvHGyOXFkzygRVPrcUVX19nuiB7VhjPTRwOxgYHA04sWQ3fySHS5kVCUugTAHPTedEJfPr7AaV+2oXla2aLjobu2VgqijpT8KzmgEPcpfn18RhNQT90bYatjiRdZ0ERRv5I8B6G3Iar+PeMHHNdFdyprSwgk4JdK561HNKxvVJi95K8\\\\\\\/Im0wpgTKNH1WvyjVzOdfWjU3FL7iUF6RzsShKeed6tNMJZDBe8pVdM4ce3tUgovRzyQLkZB5SZWIUh4ZIg1ERfAhtvFAyKWR74k8wpvABZvX61T5DLctyJVTrpCWjyonf0pOgo0OdKVF3PIEAmyC2TzZa0jiw\\\\\\\/wzpBMe745wmiNNjvLZ+fJrCwWr24mB2Z22rg2pp9Ax9Sqza50yWGjvXN3QaLFp0UznZIUDpJT0lDqRXj+wot0juzQdFOI9AE5Op+dcFp6IMAUsHaun64RIK\\\\\\\/dwBrh\\\\\\\/A0yn3jNeBrDMizYnHR1NUawpX66IK61ZJapRbCLmKuBy17+lIVnwbJGAbA3mJKyQxZrYZF8EQOQZLBtfMEsrAYoCxrv4f51UTR2RsIu01xdwL1IqwyUWU7cC9zb6SDQADbNkbR9PTLEyFM2RGKrw0dAihceknP5pgQWD1J0D3ZykoCrUesSpo2eeoi+YSatm86Z42o6VDvTz0j5cT4Em23lifN2org8dh+Cqh\\\\\\\/ApD9Cqi3KMAb5IL+KrYsy1WRh9h6FI4uCG3EVp9tXSEoIsTeBZYbp\\\\\\\/TZyAyp0wm1N0E4hmVESlUN0PQBh9vfTv2QS3DEFYTETdn74ruVMBR5PBm\\\\\\\/Enk5kASCNTXqFINOBfF\\\\\\\/Pae9Eu8uqCnMgMDNqiGP4Amv3BjN93ECz2iQXTuxf\\\\\\\/4hdfVIuwK9URnmrKw6xVOos+iaZ6afwb4SUO+pXy++U\\\\\\\/8QctKNYXxUO4MdRC8+TO+NmRuuSnsPHEvvO4HnGG95l0DjY8nYQvT0vu5+WZJpRfAlZAd104Pum4wnlCBzeMEsCrPb2e7m+zo\\\\\\\/EpEn48rttQFlIXamIVllbbEhK7CeYusBKWcssGbApEN5IbodNk\\\\\\\/q\\\\\\\/ySQQnWELv\\\\\\\/uZ6q2LIWXIY61qRQpLsoTR4MZ7cdJdPu07Z6wDOgoa8QJaWvbXqzmrboFIRsNYkX12UXEcTPIsAxONefLioCT+Uv6g6hsjZfxsKURbDW3FsPvbybzIUC+C8EOzJSjZXDUeuqy8f0SgUZK5BpTqpvw7xD1m+4mzwbPaF0hyj1gk2RQynw6aL8OcP44JWLlpH1NemkawaOfh+k9dZTmbd+OD5xwAm4FGjPfaMvfbr+ByGfli+km0WVCGYf6abTtQ9t1w3n9mMDT15W1bM+NIlD8clz6JbkKVOpSPe+LCDL4uhuCOt76V\\\\\\\/2oj4e4tR0VYTgPQI4EuqmZI1E7XmtImAZ2JLle+FB9gDBx2w\\\\\\\/7wpNS\\\\\\\/VV2Yjb31PxgDYtt3B\\\\\\\/4LBrPsqZ6TF0fjkumTBaCG3vkZbrhv2ht6e8g8SiYM33QP2MVr+\\\\\\\/ViEJjemqH3D1SCFSPWajzoynzSrpoopPJmiLEUzkyxDcR4iN4oeI0Ym08TF80CVGTpAUEvN2BbfgmetbmGT2m8MG+KWbXAZnrjww7HFonjQPLTYNRwQIRXoH4aqxfufAd1crFkNN4KYG19zfNyIlaoUU1vP160nJdRg3I5TT5h3GMZ+XqSNr+8tkW1D6OAYF3un0wBq80pdJTiPOBstLKLd\\\\\\\/jqC5o3NmJ9WD+GTU5ra2YgOGGS\\\\\\\/eLoZO6736W\\\\\\\/AZVA6vtehdAsLPKYrkjTg9OKXAfuCKCQPr7vblkq3xcFn6X9T4lW0hFBJswGageKzaL4RwG5GqC7fzv9K\\\\\\\/xJrEY9oPlSijyFGRZMtmLoKLsznauCHA2SGwxtbU8H9548xu59Bx+4mhOgSL4PYzJyE3lOFeBHhQ6M2IclEQgPyDFGbR5nNGYDdglhxWoRAdTGmuVjiS+EtPJff+Vfl\\\\\\\/+InNp3ojH2\\\\\\\/2tfe\\\\\\\/f4Wn+vSExIAvyq5YHH3FWxcO81UB6lN5mft2qYbNRewor00\\\\\\\/Qk3Eqg9pRSN7C3lkt\\\\\\\/HRdzrqoSwwzeWCmGkBlj9ZXwAYNpoNmPAnDVBJVOojazXQ4lUnXyi3Chicr2ynH5nyP+IpLoF6fK3nzOJDWkrSNIbZi6PtOdW591pBcU6+Zf1MCy55UZUdP8WWn3JeypnWwP4ZqzIgRPzJFNt4xL+wh\\\\\\\/rp7KtKl4cpjALIN+BEDCIBM93sCPpQY\\\\\\\/vwydkHVVxD09e8s6JsG0J+2QHZ\\\\\\\/d\\\\\\\/+fJCzKhg2xXgIxTd2k2dYniDeXxGpE1eFhhZKm29D0Vd\\\\\\\/mfy+rG3DGrOJtpr02UZ9VELxF9qNbQSsYjlkLxgnvhqzFV5f58NEq2SRkfNmETIVE9i2JmcCfrALKDYebuvd6BFflI5KRDOc9zkhfbft9mCUmGFGiVYGdjnW\\\\\\\/sjSb9O8BojhNYookJMdJqeC6lfiMMMpnbOsSbM2Yg6N2NCXmFBnTHnpIlezGLG3UEJhSba+1lz5rMOy\\\\\\\/wd\\\\\\\/lHDBBf1GwNx0sIRI7I5y5Bo0A\\\\\\\/R2rGgsWjGjK9SRJL\\\\\\\/BcWHyShZtPMDE4EApTN2EZmnx0eK3lPDsNMeppPGaXE1Ick5y08EvIr4x8iWN8td0TnDKmwFY0uOOqAUdtYpeQvssF3J3RO7TRbDOJAOxsch\\\\\\\/6kksmfldv159zqE\\\\\\\/XsuV5tPx5CkVZ8Bl4TLXv7+biw4pp5HOeYLnoouwsh4q0O3kPYglLK5E7TtrXr+n8uGS\\\\\\\/z6tsR3mwmS248tdTX5RKO1cpqhigyiBQ\\\\\\\/F2AxxKZHey8uk1ryoZh1azWKtgKIIVxjmD\\\\\\\/wqhGdTMXNuy8YlTI6r3suprRnyVMoSvdlu2onHZdf\\\\\\\/VFjSTaHUJIi1Lzi89I3rJL\\\\\\\/9k72FZ2aoltkDYuzqBooNobhl+SWch1VjnJTEcXHhsBSke2NGbxsuYW1mOU0LCV1BtMKNXIsvrK\\\\\\\/8rhyoCXwVfWVD6GGk2sf50OX7KMrKTbJ87+uUvGpln5V9YlsqTxhN8RqftNsP0KMvpdquLHxzMC0GmlQjlBuQW0Bm95z54xlOYbKSw8\\\\\\\/kxAZ\\\\\\\/65NmdbY4sSg\\\\\\\/nfGURsV\\\\\\\/nIgng3o1MzWPT7dFjvd20+V1qfBRDJc8lp4WwAW9WYWJemu+dLqvamqfRO122wZ9upcB7DClnPsnXiq93jI7Q7drwKxjkYKxKt9E0dsJEWpdr4Mgc+1eCF4f0XHN\\\\\\\/OlF7EjJ5ve02TS9rSz6vmlsW0NrUkNhpkq8uFNrsCMJV2FfHUrL5tazXeVP20YMzIRWPEbyfkJ+wU4nE2+zxE+gP8mcn60j1L8nKTXWslt6JDNPy24CvSkVwtdF39sHnsHuS5pCDz5v2mmpYTokEbbWlxeBb5HLmvEQFyqWRQAcfYwC\\\\\\\/U8hXTRWbzEiKpdgUvos\\\\\\\/TmNbmFgzVgzcguXi7Bjqi3JU+S4KTbd0zEmtsn85Niov1xShgK2crmj1cZEA14z7393Ue62fVZ6ML+C+qRvHY2anVvpyok5iUJ9KLl5lnUBGyiEoMbuo4HpAFGzu4DwIGq\\\\\\\/ilfWE2DzFyvOHTj8kgznLq58xelT0hrBGeMYz26ULe8OHOJf\\\\\\\/eu9yPjY5yBHcOmP7eUGT8qKNcd9CGsv7TR8J+qvvAVVD\\\\\\\/UM7TuAuSFQp4n9aIZd1BYuYtN60IWw0tjt1w5\\\\\\\/49ciHVZdgOD07M\\\\\\\/Bu\\\\\\\/1+jmcRD1yJPy1l5j3MnOlVyMdjwxCHc4BwcU0mWJSeOq5U27kvlxEN+UMDn7+m7J5KNWDux9kVUVxqXRHDS2MyV6mJhPiPyKdYJIEMhsGxKbcmLSxvuDmcApIDYb5jD93S2rtTp5hy6iuCAybCcuBfVOQQsVWo8+TtZ4AfS45fxfiMdCW1htUOZsH\\\\\\\/r0PbRVp1F77FKmOqzC7zvQXP6B9zxbQ70Yf6ZqLIlIjpVN4RKJbRI4GzGGNkRKVB3S0nb5lEK7SsG6Xau7IMFWTQEOHrMHnOWKKn4Y7woMeghav6TSHKLl6rz2WaKuw3kgiipsWC9UN0TkzarWXoaUr94vj7g8WebkbK1SksEV+RWETIa\\\\\\\/pN0xlg0FnYbe15f9SH31iqaMyPGU\\\\\\\/3Uz8sJOv5Rl+aztmKzm95RhcBE4AhvmSGSf1wv7NGgTKRnqvqUMTD0nWanhzhT23oqDYUpkpvw2\\\\\\\/dMA0XZOHq89fgrswqYjWsXalCHkpjxglP3j6aR9TnPeeImpPJEyH\\\\\\\/TXdV52tc22GovneCzzy\\\\\\\/0bhS3434437yWDuZfxYt\\\\\\\/txckDYkxLgG78QfgmWuS9QTjIOtPJfWStwKD2dTn3AHHkaJp6LJPUgix2ApH\\\\\\\/2cz9dOK0Yeehs4n3XIOO4MWQhBqVmecHHOkX4tU9RBP7JPeRXUzsvuzBae8TRkeDeQ9opx\\\\\\\/6dm58jrYHV3gjqclrzH9zvvvurHy\\\\\\\/jD\\\\\\\/nbwm\\\\\\\/9p6I4B3k2EcPrz30zx22TysYb+Nr7myOC8xuXvrEEVcOJF2frKufV38tn5+n7+8e+XRBYhHoIexcpdDQgLwcQ+IYdq0lAeOshalsd1DoSigv9xjV2caD46zI0hlbtb7AEfW4QNUy0pNRnqiPuZEqU6MFwt\\\\\\\/U\\\\\\\/SN2UdNNu4WOYZgmhmmVzKtpcfuvatRSuEM9zPa5kvs5pwvETBlgiKKiAXlCAqck8thTvOMrt1jrEFYbzqDT9QOl1yqcl1FfUSCGbgvOqF8GnQUmuXXe7AL0VPr2M9MuMT\\\\\\\/99xaBofgH8NVSJjeBSJhoFCyX\\\\\\\/9kCvtFzr7ElzVs+PC+5+Xtp63fMsIpbhnllRsvcX922wbKFFRaQmIT3LSYwNFQv2CeJQBmvOwRhBsZSPCaMjp+c7kxtHv7+HQ\\\\\\\/n3kXKy\\\\\\\/hI32Uu8INXeb5pPBFm2EBuTYAyD6int6m8TPNUr3uRPtT4FkJ16\\\\\\\/6\\\\\\\/IcoeXF6t1pC\\\\\\\/RBENzbMfwhJdJSQi+7PVPaqRKUd0dxlsOpcUB5X8xfw82mBELfFHAUWMaRCzKTzf6NIoi1XiE72COWSyKfSHKPx0H+HXeQZTifGA0bhVHv6IXuSchO0TLpu+DszEsklJcLvaYYnhgL1EI9FX4AOgkkBc4wjHMZhxbzKikYhXb7QGu1vz7Ct2Xr6ViNUSjvOw1ZaKBraJk9smJWay0LSBzUm+\\\\\\\/FZqbnycOQxTW6W71ZFWn+R6jr9AKnMkd5JzrGvjmb6zEQGjre9gIMAaVO5m8XQK51TObF30UnceB9GVxzIwMXvHN\\\\\\\/MeVexxbvdKQiqme\\\\\\\/ZXMEiUw7lJ7gSNu9j6CRpqAFlnKPkia357oJwyRYDYRFPbYX+SJRw7F9RzTmSCQuARjOv3+I2f0fx1eiBkAQ4VWCP1k31TLT0fxzSvUwIBnK2GRX1PsWzoOmKB3sJr7f+w95WC3e75H3ZOXiJ6tjJVOugxsjzOZIYUsiohQEOjbOZGF0rqIFYJpD6Djb1RKVIyHW16753tnKz73PzWMdGMyiIe8koYPzNtH9aILXAC\\\\\\\/G\\\\\\\/xzfe0X6Mh39\\\\\\\/osKdqCAMkSU0RQQlLWgJ\\\\\\\/XbkMiFYHhQ3K44ftCahykFMhUwEdSf290XFBU\\\\\\\/tTELDNw6a9Xi3++L2WoKlck1+3o\\\\\\\/oIxYkiPIhvn2aIXmOlXG6v6yA8z0yhPMD4cb6G8PGWNraYVSkWxi4s6qahm\\\\\\\/SsQ6ff9bxGBqklzG36NbnPXQHGSz3s3vzSHijBX5TInmKK7\\\\\\\/tNWGXE+4\\\\\\\/AE5ZrMYd9heClWwqTMRAxSAVtUKlI6bFcVi8TfUlJumH8r1mi6oVLcy19mw\\\\\\\/9VMHadFQu7nP46phJqQBsefBlvBp6FDzDycVB2C5odrFWAOhkOV0rJKg00ug9mc2pdqOXj9qT0BLjukfGfhYKserbJYp3Xnn62wb9ntN3Er1z3h2l0BNMqCqyns2DvKK51PIY1oaX\\\\\\\/L2U4EI8DOqUOYuGaSA2jhbuyNedrICSt\\\\\\\/FTPtlmy\\\\\\\/EP\\\\\\\/V+v0RpOBgdH8KYZBO9vw0m+EJTcJTSofGDJqepHZqcbL1FkkYIGdQWgkpwhWgIeLrP9XAKY4w52Zpp\\\\\\\/BDiWv7wUvqS1xsly2ChJxtWWeM9D9PSdX3aDy1BR6oS1MS0a3VkAoXGtxbvua+7JgrU8soIA2e6G9vaUcNg17zszV70+wbhK8hg4YW4HU5gQjdbuBSp2iZbsCijQtrK+UiWvwdr3xZIN9\\\\\\\/1d9AQSAdxrZ95UF23yNdwB+XJEuy0sBEZC65reEiTwg3k55fR5yqzcIOJfwd7UE5IbfZxi8y\\\\\\\/AytHni7guDu74qOTGSPU+d\\\\\\\/rAI9HZRTGrwfBlRfTzb7pn8ec4yqQGptTVwP2cLnTLgIyQcOJ+O28tyX1ScN114Hr2JQzCPgtrKmxUdGjvBNq24SXGPbHr91FrqTjcHUoRSX9M06njWKjwcohSGIDS1UKTX2vEkZruZv01GNs\\\\\\\/SZVWMQlghi5c5hZhx1LMdAMEubDuXb\\\\\\\/CMp8iYrMnOX6fq3wHC6LjVwDcEFPbkXZimqo4+AAF0hShG4pjor9YUiY7cSA+3bNN6u7xyyrhvUY6bMCRqHmD\\\\\\\/6zJKiXOPNL8YtTnowZWdMvj035ZrmeawawdtQLSiKszSUdyqgueLMrPzYmVQyj7+z61nwIDd66Ld3P1o3rOTuLtauLLWSdUzku66qY5m0F\\\\\\\/zj0MHVUlxPBADMNN\\\\\\\/mQ2sh\\\\\\\/Z0QVPRbY3hVZ6QJ2r4S86jpDx26xc6\\\\\\\/994i+RkMRmom71Kkt+8VjnBYUBDHNARRPPfaz5e\\\\\\\/5w9ByzAxQUhGa5lPiNDClRatut\\\\\\\/fsUGKcevaJykliX+xAXETO7Bn9tqThL5Zzt\\\\\\\/VRXWBXyo3tPJykia8UcDAQeFu5ZuOrlEf1clKzJhjxzShe5Ae3sQt4Ronwb8TZocTdSJ9BONPT4CtaW5a0sEszPHoUDw1uGumSG7DWTkllpQTEXqgxi4N+UOIBuuISEnDYfJNovq10pfsPuPbosugzAOo2PCFor1PWrZKYpKvwlnPXtwX1wLkJACylO0+wYUDi\\\\\\\/Du5drgVEB14OJ25OdPC\\\\\\\/ITlvfMECnCeoKd3BcmbEnLmlmO8kLuu3Z46GK8RV+24EsxZYEVBpRZvDZ7pb+jmwGU9ZL10iONJ\\\\\\\/CYDlZXlClclnSwRTnFjOPinW4vAiXvgudTVgRd27ZKFSRz0QsWcuKJ51wnZh6sN9mZ+uS5MSyZa2v2GIkJ4NTsXZ+256AUjrFIEUdzclszthHGbc0dYp4G89jo\\\\\\\/EmeSjbAefP5WtBLvZA5vzZP9SqbqVyd7NfQrellmmx9Mtzyj1zsykrrLj\\\\\\\/TeUh+kiOteFvvUy8ZgnUy0S0TghfMRdOuTckHAVqhr+6HNcSjP4zoq7q28fQ2ERbr+OJCSmqRt\\\\\\\/UPkk7aH2qxE6ZWhP0lequWO7oZ73hNsqMerDOEHBRKP\\\\\\\/n1sH28BZ3yNkrcs8pcujYWEpFuyNsJrbfnTjA4hXklkliTXGu6w\\\\\\\/lMt+tiwmTeb4EjDspDdyYB+\\\\\\\/MPNfg9A+jh6C1pjEreLtWhE+8sOKZiTxdaiyhuvpsh+J4yqUgnGE4hV1HeLIzjxZnONoIcdRZuToZXA79\\\\\\\/i58Oq\\\\\\\/KWAy5gGb08OxKdd0DY1EAN+SUQIUy9c\\\\\\\/0jzKkk2usUSQBPLR7hHayqQEXhsOAQntL9nkTMdpHCd8kLxVe03Yclv7xrBYLVIaGkRvlnQ8Y+QcAQdRyL+KVY3WTVPmFU3O4IQBv5NDVjpjPj\\\\\\\/M6Fr9Tovr4+5Rvd\\\\\\\/2i4hF0pd71uTsXp70LAVrAhbWteNiIVhITIcISt8eY5sgdwc3tBPH3C3yjWwnPuPLom0g+5Zrw2T0DzziUYSL4hgp\\\\\\\/SUL9gtbb9E+vGJlFyv77BSBKbf+ORdSbV8y8SXERuqP+R0WRNB4lY8h4JPNR7Onjf6H94HfzJwNABIqaoFxZ\\\\\\\/mA48UcBmlcedTh03ZuMqrwuHELLu4WLzSfX\\\\\\\/lX+gufPsh4lRu9mzkFfHvXeXyYdjaLF1x+fDaypiHo12HNlfBBUw+ew1qszMWsThtB2j2QxEg5S5QL69A+58dLmnB2a2fg0uKIH087Y5nElOJUJCEcVUhsT\\\\\\\/uLblyWzFE7CH2+mHudYKsXd9ul6BWDXWogxjn12wTgjPQYN6cq3luid7Pmsi6vrSWge9FduebU1mbzjqnUY\\\\\\\/w7YedRwHMO\\\\\\\/Mw37KmdcXQC6m3CvECApjG31KFP3biQkLbI0Lt6eA4ymZMgVpCt\\\\\\\/IZ4o7U2oCRPWZNboVIvRBIZgKTBgoKVn4F2IKTFbjfLHhQUAJSrqB+iYC3aojFzei4pTYZEHicj+2HWOomvpik3sIaykphhjH+ohL76RxprIpjT1Z5UrNGYFIMbt6uuQ592XcgU25LJzULetXc0SK1cU1ZImqxpsug2C6OwSn61SK0YvOivElZs7TM03xHghyhAHBQgnQnUI2sCn0VAecOTl86CruajAqaLxUz5kTChY7XzCVrTp\\\\\\\/1S98VVslFkaQZiYRAZPG2\\\\\\\/Tgi\\\u0022,\\\u0022iv\\\u0022:\\\u0022SJXP1cdcmNdolnzH1Oayow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f3e9917204d3f83eae4aae5cfb98488dfa1c95cba17eebcc90f1a94af141dce\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HWFNRX8DnE7G5wf9nEWQDOBwlR6XLIZBBtUAJ8StJottJepwDvLW4LO\\\\\\\/\\\\\\\/CO2cI37E3Bl0aYVbmg9rGNSjIHmgzq+YwNjwHSY84AF0CViaYnXd0PGkSc4SrwLleYiQJNWkzKV371kwj42Me\\\\\\\/z5+LgEPpr1sszUugbxl2BHbZ1yvwcAphdAIuupf7OuaErAY9St9j260N4oXBmCKPn1ysmPicCI5oYSbJXRM+VJJ55aTvz4vDpsRPu+rLfkeJHYEh5Q+07WXV\\\\\\\/2N76HtPk40uCSPJkAA1pBluIyd+2S972TmhHqc44FLNNh7S7S\\\\\\\/xf3xVtdcY6bN0p\\\\\\\/iDJgAQgLgE49QTvB3k1PZrtRB9U6AUap4oILl3RWyZGGZAMDgooLVUDqrkJ4DKmk9mhcHpLdSGt5kSdEaFN8cHrbwmufoA2lAXFIXh+KLCSH37Ct+LbNm7WI3JZNXnixki0Ysp9oPWmb\\\\\\\/tyRmeYiaEzJnSIX49uyxTcU6aHXxpMDausPj5Sghxl4WPfMb+Ms7FLN19QA0Eq\\\\\\\/KPJlkmARaG4DuLnUQr69\\\\\\\/AZQ8RW3ubuu29CqoVnRd5emGGGhrJlJbgTl8GdHp\\\\\\\/ldWwsR9ph3JGU+nTFEybMR+oTl\\\\\\\/i\\\\\\\/MUkAxysa2XuAHAMmkMClMjUH8b1jtvAkOliy3zdgNCJyNd9I3uglrjGcnGWgy1E8MVGO6upd1vI1AXUbU8eyvuZuCF\\\\\\\/zTFSJrqe0Jvke7pQkfmLg+cPhzOF0toX9nTJjFx\\\\\\\/bQgR3Y6y\\\\\\\/EQQhqHK4P44rUMrZBAoUFPO3j7FW8TJe\\\\\\\/WbQvYAtTMc21UEegck6hz\\\\\\\/D2RpRIIoYfdTol\\\\\\\/q9nVMbI4sJtAF+Pu4tc7EQ7ZxErhgWzb2STaTsCzJ2OpdtcLzYAJNuCcJfWDFMe6EIqchx25Cf7RgyfQ2MwNszq96+pcafqyoCWnXW0t\\\\\\\/Y4SursXvVdIsT7hfZS4KAry5Cnb+ADpVR+RP15O0CUxX5+cPpA5Lm7mbJAYRhuSMzKLMkgT87NDL1oJAQeI\\\\\\\/kfG+UZ1ReljEmlcvi1NEnZQW\\\\\\\/9oDegmzo7+gjn56rwL+wy4UKn4eMjTDjfgoMeHTEKcZ67j9fea+WbbwnFyTGUUPRTGWWFSIcW3iWFn97jcOMnnGKPxKMsO6FmIZC5TEwCwUrn7kONzJh0drxj4+H6j9TFlr9XOn+P7fciCHB0YshWwS2X6V9VuNy7QSXjpoUt9PFZodgIiWdavl2lrSZrFOZC9xjNG\\\\\\\/pQEfw\\\\\\\/nJaq72IZmOKJ9e+Utiu\\\\\\\/Ab43GsIGRm0YjLN6lV1bEZ4Y3epQCFGEue+czFVJPjQrQBX5rSuD1+EhDlFhPFeR113ebE76KBm44XOERGU\\\\\\\/oLVYv5wADPp0gAQirbNc3b3Fz+\\\\\\\/puRJiedfdlxzLcMjaWIrFIAoX9tSHxzHm+0WzvlifErf9BZtyymIQmJVcxTU9j4U32vrzkSgXbE+kZxOm1aKhNBJRi0MNTma5qQ3OCJW3J+KFkyWcTQhlLvy7R9r+ByJ1uXlm5AQDQL7uEQEHbpMqE3CpXSE+ICspx9gLDSL\\\\\\\/1A+4I8JZp+VZeEarEgDtnd+Y+Wt67\\\\\\\/oa60odc9ihm\\\\\\\/59C3oSk0pX+E70RZM+gFLDtYkrjnIHQWrHGB8syeulkcuv5QRqTv0tmHFzHPOzTboLFQnzYqZbCF5mgXOSwYUZZw95UWKVrqMurqyEpUbdTbR9XdNqft3itKN4rJ+B1qpA+FSpb+QgkMvogJ3zRbfC8YfObUFdqoBqgwKiXh+GfJSqJRbeOfhcuWGasL+IfUou76sgqNeS4uRjUJuwq+2YZ9XOaIZod\\\\\\\/w63Y4lZ92r+LpX8OZBZYNBo\\\\\\\/hslz5D4rKzxa7CNpFFGKvfZiq1aZxKjN+Ye4njl4I4sqT0c0XoKIviiBnyPXMMReRuO2ZszLPPu8g4c48MzZEhTbA57DXzb0OXIsnV7yOInuzrlYzISZ1GDmsABZRkPlI7PWApr\\\\\\\/Y\\\\\\\/ZJfvwhxNYhBc2mq+VDnFtFsA0pUi5Dv2MSI0SnWFiYd3pDt5J\\\\\\\/nSNUqQ8dy9QPEHKmN\\\u0022,\\\u0022iv\\\u0022:\\\u0022cR6Hmc0ja4HtHKWXHvRKCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002213673239eaa33174ddc8b4ac5938a7c6da87875e97b60655bd7c1826f6802525\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459640 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/265" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ALX-RghWdwxzQZeYXUOR7K-lLm7YxdX29vW0vJr3ppY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/265" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rSFnswAXE_GkGIPOrxYBkFFOI1c7bKobVSCl0rlfaY0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/2/255" title="Genetic Architecture of Contemporary Adaptation to Biotic Invasions: Quantitative Trait Locus Mapping of Beak Reduction in Soapberry Bugs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/2/277" title="Extensive Differences in Gene Expression Between Symbiotic and Aposymbiotic Cnidarians" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459640" id="node459640" data-pisa="ggg;4/2/265" data-pisa-master="ggg;g3.113.008649" data-apath="/ggg/4/2/265.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multigenic Natural Variation Underlies <em>Caenorhabditis elegans</em> Olfactory Preference for the Bacterial Pathogen <em>Serratia marcescens</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elizabeth E. Glater</span>, <span class="highwire-citation-author" data-delta="1">Matthew V. Rockman</span> and <span class="highwire-citation-author" data-delta="2">Cornelia I. Bargmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">265-276;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008649 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459640" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth E. Glater </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Harvey Mudd College, Claremont, California 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BE.%2BGlater%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glater%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BE.%2BGlater%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Elizabeth.glater@pomona.edu" class="" data-icon-position="" data-hide-link-title="0">Elizabeth.glater@pomona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew V. Rockman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and Center for Genomics and Systems Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BV.%2BRockman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rockman%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BV.%2BRockman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cornelia I. Bargmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Howard Hughes Medical Institute, Laboratory of Neural Circuits and Behavior, The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cornelia%2BI.%2BBargmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bargmann%20CI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACornelia%2BI.%2BBargmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/2/265" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459640" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459640/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/265.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459640" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459640/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/265.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459640" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459640/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/265.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459640" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459640/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/2/265.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459640" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459640/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The nematode <em>Caenorhabditis elegans</em> can use olfaction to discriminate among different kinds of bacteria, its major food source. We asked how natural genetic variation contributes to choice behavior, focusing on differences in olfactory preference behavior between two wild-type <em>C. elegans</em> strains. The laboratory strain <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> strongly prefers the odor of <em>Serratia marcescens</em>, a soil bacterium that is pathogenic to <em>C. elegans</em>, to the odor of <em>Escherichia coli</em>, a commonly used laboratory food source. The divergent Hawaiian strain <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> has a weaker attraction to <em>Serratia</em> than the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> strain, and this behavioral difference has a complex genetic basis. At least three quantitative trait loci (QTLs) from the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> Hawaii strain (HW) with large effect sizes lead to reduced <em>Serratia</em> preference when introgressed into an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genetic background. These loci interact and have epistatic interactions with at least two antagonistic QTLs from HW that increase <em>Serratia</em> preference. The complex genetic architecture of this <em>C. elegans</em> trait is reminiscent of the architecture of mammalian metabolic and behavioral traits.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/caenorhabditis-elegans" class="hw-term hw-article-keyword hw-article-keyword-caenorhabditis-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Caenorhabditis elegans</em></a></li><li class="kwd"><a href="/keyword/olfaction" class="hw-term hw-article-keyword hw-article-keyword-olfaction" rel="nofollow">olfaction</a></li><li class="kwd"><a href="/keyword/natural-variation" class="hw-term hw-article-keyword hw-article-keyword-natural-variation" rel="nofollow">natural variation</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/serratia-marcescens" class="hw-term hw-article-keyword hw-article-keyword-serratia-marcescens" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Serratia marcescens</em></a></li></ul><p id="p-5">Individual differences in behavior have genetic and environmental components. The genetic basis of natural variation in behavior is generally understood to be complex, with multiple contributing loci that each explains only a fraction of the variance in a trait (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fisher 1918</a>). Our current understanding of this variation is largely based either on association studies (such as GWAS), in which the effect of each locus is assessed across a variety of genetic backgrounds, or on studies that use recombinant inbred lines constructed from two parent strains, in which many segregating loci are examined in parallel (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altshuler <em>et al.</em> 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lander and Botstein 1989</a>). Both of these approaches screen broadly for variation, but the quantitative assumptions underlying their use are biased toward loci with additive effects that are insensitive to epistatic interactions or genetic background. In both cases, the effect size of each locus is averaged over all tested genetic backgrounds.</p><p id="p-6">In experimental animals, defined genetic regions can be transferred between strains through introgression, holding the genetic background constant. This method has been historically important in immunological studies in mice, in which highly introgressed recombinant inbred lines defined specific immune functions within the major histocompatibility loci (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bach <em>et al.</em> 1972</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McDevitt <em>et al.</em> 1972</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">McDevitt and Tyan 1968</a>). Introgression is particularly valuable when multiple loci interact in unpredictable ways, as can occur in immune responses and metabolism-related traits (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bhatnagar <em>et al.</em> 2011</a>).</p><p id="p-7"><em>Caenorhabditis elegans</em> is an excellent model organism for studying natural variation. It reproduces as a self-fertilizing hermaphrodite with occasional male outcrossing, which facilitates the generation of isogenic strains compared to obligate sexual species. Different strains of <em>C. elegans</em> vary in a wide range of phenotypes, including foraging behavior, oxygen and carbon dioxide preference, susceptibility to pathogenic bacteria, and dauer development (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bendesky and Bargmann 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de Bono and Bargmann 1998</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hodgkin and Doniach 1997</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McGrath <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Persson <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schulenburg and Ewbank 2004</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schulenburg and Muller 2004</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Viney <em>et al.</em> 2003</a>). Loci affecting several of these traits have been identified through quantitative trait loci (QTLs) approaches, but the genetic architecture of many of these traits has not been fully examined. In this study, we examined the multigenic basis of a complex trait, bacterial preference behavior, using an introgression strategy.</p><p id="p-8"><em>C. elegans</em> lives largely in association with human agriculture, where it feeds on a variety of bacteria and fungi associated with rotting fruit and plant matter (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Felix and Duveau 2012</a>). Among these microorganisms, the animal must choose food that is edible, nutritious, and nonpathogenic. Microbiomes are highly diverse, so <em>C. elegans</em> strains isolated from different regions may have adapted to local microbiota. In agreement with this possibility, different strains of <em>C. elegans</em> exhibit innate genetic variation in their interactions with specific bacteria. Wild-type <em>C. elegans</em> vary in their susceptibility to being killed by the bacterial pathogens <em>Bacillus thuringiensis</em> and <em>Serratia marcescens</em> (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Schulenburg and Ewbank 2004</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Schulenburg and Muller 2004</a>), in their behavioral evasion of <em>Bacillus thuringiensis</em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Schulenburg and Muller 2004</a>), and in their ability to distinguish behaviorally among different species of bacteria (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Volkers <em>et al.</em> 2013</a>). In addition to exhibiting strong innate preferences, <em>C. elegans</em> can learn to avoid the odor of specific pathogenic bacteria after infection (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2005</a>) and will migrate away from toxic or inedible bacteria, in part based on olfactory cues (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Melo and Ruvkun 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pradel <em>et al.</em> 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shtonda and Avery 2006</a>). Both innate and learned odor responses are generated by a highly developed olfactory system with thousands of chemoreceptor genes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bargmann 2006</a>).</p><p id="p-9">We examined the neuronal and genetic basis of <em>C. elegans</em> olfactory preference with a choice between the pathogenic bacteria <em>Serratia marcescens</em> and nonpathogenic <em>Escherichia coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a>. <em>S. marcescens</em> is highly attractive to and readily consumed by <em>C. elegans</em>, even though it establishes an intestinal infection that kills the worm after 2 to 3 d (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kurz <em>et al.</em> 2003</a>). Although <em>C. elegans</em> is initially strongly attracted to a patch of <em>Serratia</em> bacteria, the worms will leave the bacteria after several hours through a learned avoidance mediated by the <em><a href="http://www.wormbase.org/db/get?name=tol-1;class=Gene">tol-1</a></em> gene (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pujol <em>et al.</em> 2001</a>). We examined natural variation in <em>Serratia</em> preference between the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol laboratory strain and the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> Hawaii strain (HW) using recombinant inbred lines, chromosome substitution strains, and introgression lines, and we found that multiple QTLs and multiple epistatic interactions influence olfactory preference behavior. The genetic complexity of this <em>C. elegans</em> trait recapitulates the genetic complexity of mammalian behaviors and suggests that introgression will be a valuable approach for finding underlying genes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Nematode growth and strains</h3><p id="p-10">Strains were grown and maintained under standard conditions at 20<strong>°</strong> on nematode growth media (NGM) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brenner 1974</a>). L4 animals were placed on 100-mm NGM plates seeded with <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> ATCC 33694 and their adult progeny were assayed 4 d later. A complete list of <em>C. elegans</em> strains is provided (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/008649SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-3" class="subsection"><h3>Bacterial strains</h3><p id="p-11">Bacterial strains were obtained from the American Type Culture Collection. Strains were <em>Serratia marcescens</em> ATCC 274 and <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> ATCC 33694.</p></div><div id="sec-4" class="subsection"><h3>Bacterial choice assay</h3><p id="p-12">The two-choice bacterial choice assay was modified from the work of <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> (2005)</a>. Briefly, bacteria grown overnight in LB at 26<strong>°</strong> were resuspended at an OD600 of 1.0 for <em>S. marcescens</em> or an OD600 of 10.0 for <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a>, and 25 μl of each bacterial suspension was spotted onto an NGM plate and air-dried for 5 hr at 20<strong>°</strong>. At these OD600 values, both bacteria had approximately the same cellular density: at OD600 of 1.0, <em>S. marcescens</em> yields 2.1 × 10<sup>9</sup> ± 1.5 × 10<sup>9</sup> colony-forming units (cfu) per ml; at OD600 of 10, <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> yields 3.2 × 10<sup>9</sup> ± 1.7 × 10<sup>9</sup> cfu per ml. Adult animals were washed three times in 1.5 ml S-basal buffer and 50–200 animals were placed with glass Pasteur pipette near the center of an NGM plate, equidistant from the two bacteria. Animals were allowed to move freely for 1 hr before being immobilized by 1 μl of 1 M sodium azide (movie of bacterial choice assay, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS2.wmv">File S2</a>). We scored the number of animals on the <em>Serratia</em> lawn and the number of animals on the <em>E. coli</em> lawn. After 1 hr, less than 5% of animals were found outside the bacterial lawn for all strains tested; these animals were not counted. Assays for chromosome substitution strains and introgression strains were repeated at least five times on at least two different days. Assays for recombinant inbred advanced intercross lines (RIAILs) were repeated three to 10 times on at least two different days.</p></div><div id="sec-5" class="subsection"><h3>Generation of introgression strains</h3><p id="p-13">Chromosome IV introgression strains were made by crossing <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> males to hermaphrodites from strain WE5239, which bears the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> (HW) chromosome IV on an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background. The F2 progeny were screened for recombination events by PCR analysis of known chromosome IV polymorphisms between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW (<a href="http://www.wormbase.org">www.wormbase.org</a>) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Davis <em>et al.</em> 2005</a>). F3 self-progeny homozygous for the recombinant chromosomes were identified by PCR genotyping, and homozygous strains were assayed in the bacterial choice assay in subsequent generations. Strains with a behavioral phenotype resembling the HW parent were then crossed to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> males and the process was repeated to generate introgression strains containing smaller regions of HW DNA. Introgression strains were genotyped with SNPs identified in WormBase (<a href="http://www.wormbase.org">www.wormbase.org</a>). The genotypes of these lines can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS4.xlsx">File S4</a>.</p></div><div id="sec-6" class="subsection"><h3>Statistical analysis for determining QTLs</h3><div id="sec-7" class="subsection"><h4>RIAIL analysis:</h4><p id="p-14">Seventy-two RIAILs, each genotyped at 1455 markers (for RIAIL genotypes, see <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rockman and Kruglyak 2009</a>), were phenotyped in three to 10 assays each, and the mean choice indices were analyzed by interval mapping in <em>R/qtl</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman <em>et al.</em> 2003</a>) after they were Box-Cox–transformed to approximate normality (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Venables and Ripley 2002</a>). The genome-wide <em>P</em> of the peak LOD score was estimated by 1000 permutations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Churchill and Doerge 1994</a>). Qualitatively identical results were found with nonparametric interval mapping.</p><p id="p-15">To directly evaluate a contribution from the introgression line-defined QTLs, we used the fitqtl function of <em>R/qtl</em>, which performs an ANOVA to test the significance and to estimate the variance explained for specified QTL in a multiple QTL model. This has the advantage of using imputed genotypes or genotype probabilities at QTL rather than relying on marker class means.</p></div><div id="sec-8" class="subsection"><h4>Introgression line analysis and common segment method:</h4><p id="p-16">The choice index of each introgression line (47 lines), or a subset of these lines, was compared to the choice index of <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> using ANOVA with Dunnett correction for multiple comparisons (<em>P</em> &lt; 0.05). Strains with phenotypes that differ significantly from the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> are likely to contain one or more QTLs. Strains that do not differ significantly from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> likely do not contain a QTL or contain a QTL and an additional suppressor. We attempted to explain the phenotypes of each strain that differs from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> by invoking the fewest necessary QTLs (<em>i.e.</em>, common segments, shared by strains) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shao <em>et al.</em> 2008</a>). Strains that contain those QTLs but are not different from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> are inferred to carry suppressors, whose number we minimize in the same manner. The suppressors are invoked by parsimony and are not subjected to formal hypothesis test (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Shao <em>et al.</em> 2010</a>).</p></div><div id="sec-9" class="subsection"><h4>Introgression line analysis and sequential minimum spanning tree method:</h4><p id="p-17">In the sequential method (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Shao <em>et al.</em> 2010</a>), strains are compared two at a time and significant differences imply that the chromosome segments not shared by the two strains harbor a QTL. To minimize the number of strain comparisons and to maximize localization resolution, the method compares pairs of strains that are most genetically similar to one another. The sequence of comparisons is determined by constructing a minimum spanning tree that connects the strains according to their pairwise similarity. In our implementation, we calculated genetic similarity by estimating the number of base pairs that differ between each strain, assuming that breakpoints are at the midpoints of marker intervals. We used the spantree function of the R package vegan (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Oksanen <em>et al.</em> 2012</a>) to find a minimum spanning tree and we tested for phenotypic differences between pairs of strains adjacent on the tree by <em>t</em> test with Bonferroni correction.</p></div></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Wild-type strains vary in bacterial preference</h3><p id="p-18">Bacterial preferences of <em>C. elegans</em> were evaluated using a bacterial choice assay in which worms migrate to one of two patches of bacteria on opposite sides of an agar plate (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2005</a>). The first approach of the animals over 1–2 hr is dominated by their olfactory preferences for volatile odors released by the bacteria. We examined two strains, <em>S. marcescens</em> ATCC 274 and <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a>. Surprisingly, although <em>S. marcescens</em> is a bacterial pathogen that can kill <em>C. elegans</em>, it was more attractive to the wild-type <em>C. elegans</em> strain <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> than its standard laboratory food source, <em>E. coli</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bacterial choice behavior varies among wild-type C. elegans strains. (A) Cartoon of the bacterial choice assay. Approximately 100 worms are placed on the agar plate between two patches of bacteria, which they can approach by olfactory chemotaxis. (B) Bacterial choice index of six wild strains. HW, strain CB4856. n &#x2265; 6 assays. (C) Choice assays for Serratia vs. E. coli, Serratia vs. LB media, and E. coli vs. LB media conducted in parallel experiments. ***P &lt; 0.001, t test, n &#x2265; 6 assays. S.E.M. represented by error bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bacterial choice behavior varies among wild-type &lt;em&gt;C. elegans&lt;/em&gt; strains. (A) Cartoon of the bacterial choice assay. Approximately 100 worms are placed on the agar plate between two patches of bacteria, which they can approach by olfactory chemotaxis. (B) Bacterial choice index of six wild strains. HW, strain CB4856. n &#x2265; 6 assays. (C) Choice assays for &lt;em&gt;Serratia vs. E. coli&lt;/em&gt;, &lt;em&gt;Serratia vs.&lt;/em&gt; LB media, and &lt;em&gt;E. coli vs.&lt;/em&gt; LB media conducted in parallel experiments. ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, &lt;em&gt;t&lt;/em&gt; test, n &#x2265; 6 assays. S.E.M. represented by error bars.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F1.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/2/265/F1.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464008" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Bacterial choice behavior varies among wild-type <em>C. elegans</em> strains. (A) Cartoon of the bacterial choice assay. Approximately 100 worms are placed on the agar plate between two patches of bacteria, which they can approach by olfactory chemotaxis. (B) Bacterial choice index of six wild strains. HW, strain CB4856. n ≥ 6 assays. (C) Choice assays for <em>Serratia vs. E. coli</em>, <em>Serratia vs.</em> LB media, and <em>E. coli vs.</em> LB media conducted in parallel experiments. ***<em>P</em> &lt; 0.001, <em>t</em> test, n ≥ 6 assays. S.E.M. represented by error bars.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">An animal’s preference for different food sources should vary based on its natural ecology, and recent studies of <em>C. elegans</em> indicate that it is found in human-associated environments with a variety of different bacteria (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Felix and Duveau 2012</a>). We examined bacterial preference behavior in wild-type strains isolated from different environments and found that wild-type strains of <em>C. elegans</em> varied in their preference between <em>S. marcescens</em> ATCC 274 and <em>E. coli</em> <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Among six tested strains, the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> laboratory strain had the strongest preference for <em>Serratia</em> over <em>E. coli</em>, and a highly divergent strain, HW, had the weakest preference for <em>Serratia</em>.</p><p id="p-21">In a choice between bacteria and the bacterial growth media alone (LB), <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> had a significantly stronger preference for <em>Serratia</em> than HW (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). A trend toward an increased HW preference for <em>E. coli</em> over media was not statistically significant (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). These results indicate that the response to <em>Serratia</em> is the main source of genetic variability between the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW strains, although the <em>E. coli</em> response may also contribute to their distinct preferences.</p></div><div id="sec-12" class="subsection"><h3>Segregation of preference behavior in recombinant inbred lines</h3><p id="p-22">To determine the genetic basis of natural variation in bacteria preferences between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> (HW), we first assayed 72 genotyped <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>–<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> RIAILs (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rockman and Kruglyak 2009</a>) in the bacterial choice assay (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). These strains have been genotyped at more than 1000 informative loci and have been used successfully to identify loci affecting a variety of behavioral, developmental, and life history traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bendesky <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gaertner <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">McGrath <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Palopoli <em>et al.</em> 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Seidel <em>et al.</em> 2008</a>). Variance among RIAIL strains accounted for 46.3% of the total variance in bacterial preference across assays, providing an estimate of broad-sense heritability of the trait (F<sub>71,408</sub> = 4.95; <em>P</em> &lt; 10<sup>−15</sup>). The RIAILs varied smoothly in their bacterial preference, suggesting that more than one gene affects bacterial choice (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). In addition, several strains had bacterial preference more extreme than either starting strain, a pattern of transgressive segregation suggesting that <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW each carry alleles that act in both directions, possibly in background-dependent manners (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). Although this pattern suggests that there are multiple segregating loci in the strains, linkage analysis of the RIAILs yielded only a single QTL on chromosome II at genome-wide significance (II:2808858 with LOD = 3.255; genome-wide <em>P</em> = 0.036) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). The HW allele at this QTL decreases behavioral preference for <em>Serratia</em> bacteria.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping of bacterial choice index with N2&#x2013;HW recombinant inbred advanced intercross lines (RIAILs). (A) Bacterial choice index of 72 N2&#x2013;HW RIAILS (black), N2 (red), and HW (blue). RIAIL strain names and choice index are in File S3. (B) Logarithm of odds (LOD scores) along chromosomes for RIAILs shown, with horizontal line denoting threshold for genome-wide significance (P = 0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL mapping of bacterial choice index with N2&#x2013;HW recombinant inbred advanced intercross lines (RIAILs). (A) Bacterial choice index of 72 N2&#x2013;HW RIAILS (black), N2 (red), and HW (blue). RIAIL strain names and choice index are in File S3. (B) Logarithm of odds (LOD scores) along chromosomes for RIAILs shown, with horizontal line denoting threshold for genome-wide significance (&lt;em&gt;P&lt;/em&gt; = 0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F2.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/2/265/F2.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463988" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">QTL mapping of bacterial choice index with N2–HW recombinant inbred advanced intercross lines (RIAILs). (A) Bacterial choice index of 72 N2–HW RIAILS (black), N2 (red), and HW (blue). RIAIL strain names and choice index are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS3.xls">File S3</a>. (B) Logarithm of odds (LOD scores) along chromosomes for RIAILs shown, with horizontal line denoting threshold for genome-wide significance (<em>P</em> = 0.05).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Multiple loci that differ between N2 and HW affect bacterial preference</h3><p id="p-24">To further examine the significance of the QTLs on chromosome II, and to probe the genetic structure of the bacterial preference trait more generally, we next assayed bacterial preference in six <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>–HW chromosome substitution strains (CSSs) in which a single homozygous HW chromosome replaced the corresponding chromosome in an otherwise <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Glauser <em>et al.</em> 2011</a>). The strain bearing HW chromosome II (CSSII) closely resembled <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>, showing no evidence of HW-like bacterial preference (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This result could indicate that the marginally significant QTLs identified using the RIAILs were false-positive, or that the CSSs were false-negative. We tested the bacterial choice behavior of an introgression strain with the HW chromosome II QTL predicted by RIAIL analysis and found that the strain had an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like phenotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FigureS1.pdf">Figure S1</a>). This strain may have an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like phenotype because this region does not contain a QTL or because this region interacts epistatically with other QTL to generate a HW-like phenotype. The latter interpretation would be consistent with a complex genetic architecture for bacterial preference, as suggested by transgressive segregation in the RIAILs.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative trait loci on two chromosomes underlie natural variation in bacterial choice behavior. Bacterial choice behavior of chromosome substitution strains. &#x201C;Genotype&#x201D; shows chromosomes (blue denotes HW DNA; red denotes N2 DNA). ***P &lt; 0.001, **P &lt; 0.01 compared to N2 by ANOVA with Dunnett test, n in parentheses under each bar. CSSIV, CSSV, and CSSIV; CSSV were statistically indistinguishable by t test. S.E.M. represented by error bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative trait loci on two chromosomes underlie natural variation in bacterial choice behavior. Bacterial choice behavior of chromosome substitution strains. &#x201C;Genotype&#x201D; shows chromosomes (blue denotes HW DNA; red denotes N2 DNA). ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01 compared to N2 by ANOVA with Dunnett test, n in parentheses under each bar. CSSIV, CSSV, and CSSIV; CSSV were statistically indistinguishable by &lt;em&gt;t&lt;/em&gt; test. S.E.M. represented by error bars.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F3.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/2/265/F3.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463898" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Quantitative trait loci on two chromosomes underlie natural variation in bacterial choice behavior. Bacterial choice behavior of chromosome substitution strains. “Genotype” shows chromosomes (blue denotes HW DNA; red denotes N2 DNA). ***<em>P</em> &lt; 0.001, **<em>P</em> &lt; 0.01 compared to N2 by ANOVA with Dunnett test, n in parentheses under each bar. CSSIV, CSSV, and CSSIV; CSSV were statistically indistinguishable by <em>t</em> test. S.E.M. represented by error bars.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Examining the other CSS strains provided support for a complex genetic architecture. Strains bearing either HW chromosome IV (CSSIV) or chromosome V (CSSV) had a HW-like phenotype, whereas all other CSS lines had an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like phenotype (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). These results suggest that at least two regions, one each on chromosomes IV and V, contain QTLs for bacterial choice behavior, although neither emerged from the RIAILs.</p><p id="p-27">To assess the interaction between these two chromosomes, we generated a CSS with both chromosome IV and chromosome V from the HW background. The behavior of this strain was statistically indistinguishable from either individual CSS (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Therefore, both chromosome IV and chromosome V bear QTLs that affect bacterial preference, but these QTLs are not additive.</p></div><div id="sec-14" class="subsection"><h3>Mapping QTL on chromosome IV</h3><p id="p-28">Regions of chromosome IV that affected bacterial preference were defined further by making recombinants between CSSIV and <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>. Recombinants resulting from two or three iterative rounds of recombination [(CSSIV × <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) × <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (× <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>)] were genotyped across chromosome IV and tested for preference behavior as homozygotes. All recombinant strains that differed significantly from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> shared an HW region between 2.29 and 4.99 MB, suggesting the presence of a QTL conferring HW-like behavior in this region (QTL1) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). However, these introgression lines provided relatively little power to resolve QTLs, because the HW-derived DNA segments were large and contained relatively few breakpoints.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple QTLs on chromosome IV. QTLs for bacterial choice of introgression lines derived from CSSIV (this article) and from recombinant inbred lines (Doroszuk et al. 2009). Left, Genotypes are shown for various genetic markers (white is N2; black is HW; graded from white to black indicates unknown genomic regions between genotyped SNPs. Right, Bacterial choice behavior of introgression lines. Blue markers indicate lines that differ significantly from N2. ***P &lt; 0.001, **P &lt; 0.01, *P &lt; 0.05, ANOVA with Dunnett, n &#x2265; 5 assays. S.E.M. represented by error bars. Chromosome IV introgression line strain names, choice index, and genotype at additional genetic markers are in File S4. Introgression lines were analyzed by the common segment method to determine QTLs. The inferred locations of QTLs that decrease Serratia preference are indicated by blue vertical lines on the genetic map; antagonistic QTLs that restore N2-like phenotype are indicated by red vertical lines on the genetic map. N2 and CB4856 were tested as well as subset of introgression strains on each day. The N2 and CB4856 data shown are the average data for N2 and CB4856 tested on all days that the introgression strains were tested. (A) Initial set of lines derived from CSSIV. Using the common segment method, recombinant strains that differed significantly from N2 shared an HW region (&#x223C;2.29 to &#x223C;4.99 MB; QTL1). (B) Introgression lines that begin at the left telomere of chromosome IV and are derived from kyIR28. Common segment method reveals QTL2 and QTL3 that confer HW-like behavior and antagonistic QTL5 that confers N2-like behavior. QTL4 and antagonistic QTL6 are supported by only one line (kyIR62) and are indicated with dashed lines. (C) Independent introgression lines (ewIR) from study by Doroszuk et al. (2009). Common segment method identifies QTL7 and QTL8. (D) Analysis of all chromosome IV introgression lines (kyIR and ewIR). All lines were analyzed, but only a subset that includes all lines significantly different from N2 is shown. Analysis of all lines yielded the same QTLs as in subsets, with three exceptions: QTL3 is a smaller region because it is defined by both ewIR and kyIR lines (specifically ewIR46 and kyIR67); antagonistic QTL6 is defined by two lines instead of one (ewIR47 and kyIR62); and QTL8 is only supported by one line, ewIR55. The line ewIR54 is no longer significantly different from N2 when part of a larger data set. Complete explanation of chromosome IV QTLs defined by common segment method appears in File S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multiple QTLs on chromosome IV. QTLs for bacterial choice of introgression lines derived from CSSIV (this article) and from recombinant inbred lines (Doroszuk &lt;em&gt;et al.&lt;/em&gt; 2009). Left, Genotypes are shown for various genetic markers (white is N2; black is HW; graded from white to black indicates unknown genomic regions between genotyped SNPs. Right, Bacterial choice behavior of introgression lines. Blue markers indicate lines that differ significantly from N2. ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01, *&lt;em&gt;P&lt;/em&gt; &lt; 0.05, ANOVA with Dunnett, n &#x2265; 5 assays. S.E.M. represented by error bars. Chromosome IV introgression line strain names, choice index, and genotype at additional genetic markers are in File S4. Introgression lines were analyzed by the common segment method to determine QTLs. The inferred locations of QTLs that decrease &lt;em&gt;Serratia&lt;/em&gt; preference are indicated by blue vertical lines on the genetic map; antagonistic QTLs that restore N2-like phenotype are indicated by red vertical lines on the genetic map. N2 and CB4856 were tested as well as subset of introgression strains on each day. The N2 and CB4856 data shown are the average data for N2 and CB4856 tested on all days that the introgression strains were tested. (A) Initial set of lines derived from CSSIV. Using the common segment method, recombinant strains that differed significantly from N2 shared an HW region (&#x223C;2.29 to &#x223C;4.99 MB; QTL1). (B) Introgression lines that begin at the left telomere of chromosome IV and are derived from &lt;em&gt;kyIR28&lt;/em&gt;. Common segment method reveals QTL2 and QTL3 that confer HW-like behavior and antagonistic QTL5 that confers N2-like behavior. QTL4 and antagonistic QTL6 are supported by only one line (&lt;em&gt;kyIR62&lt;/em&gt;) and are indicated with dashed lines. (C) Independent introgression lines (&lt;em&gt;ewIR&lt;/em&gt;) from study by Doroszuk &lt;em&gt;et al.&lt;/em&gt; (2009). Common segment method identifies QTL7 and QTL8. (D) Analysis of all chromosome IV introgression lines (&lt;em&gt;kyIR&lt;/em&gt; and &lt;em&gt;ewIR&lt;/em&gt;). All lines were analyzed, but only a subset that includes all lines significantly different from N2 is shown. Analysis of all lines yielded the same QTLs as in subsets, with three exceptions: QTL3 is a smaller region because it is defined by both &lt;em&gt;ewIR&lt;/em&gt; and &lt;em&gt;kyIR&lt;/em&gt; lines (specifically &lt;em&gt;ewIR46&lt;/em&gt; and &lt;em&gt;kyIR67&lt;/em&gt;); antagonistic QTL6 is defined by two lines instead of one (&lt;em&gt;ewIR47&lt;/em&gt; and &lt;em&gt;kyIR62&lt;/em&gt;); and QTL8 is only supported by one line, &lt;em&gt;ewIR55&lt;/em&gt;. The line &lt;em&gt;ewIR54&lt;/em&gt; is no longer significantly different from N2 when part of a larger data set. Complete explanation of chromosome IV QTLs defined by common segment method appears in File S6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-4.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-4.medium.gif" width="389" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple QTLs on chromosome IV. QTLs for bacterial choice of introgression lines derived from CSSIV (this article) and from recombinant inbred lines (Doroszuk et al. 2009). Left, Genotypes are shown for various genetic markers (white is N2; black is HW; graded from white to black indicates unknown genomic regions between genotyped SNPs. Right, Bacterial choice behavior of introgression lines. Blue markers indicate lines that differ significantly from N2. ***P &lt; 0.001, **P &lt; 0.01, *P &lt; 0.05, ANOVA with Dunnett, n &#x2265; 5 assays. S.E.M. represented by error bars. Chromosome IV introgression line strain names, choice index, and genotype at additional genetic markers are in File S4. Introgression lines were analyzed by the common segment method to determine QTLs. The inferred locations of QTLs that decrease Serratia preference are indicated by blue vertical lines on the genetic map; antagonistic QTLs that restore N2-like phenotype are indicated by red vertical lines on the genetic map. N2 and CB4856 were tested as well as subset of introgression strains on each day. The N2 and CB4856 data shown are the average data for N2 and CB4856 tested on all days that the introgression strains were tested. (A) Initial set of lines derived from CSSIV. Using the common segment method, recombinant strains that differed significantly from N2 shared an HW region (&#x223C;2.29 to &#x223C;4.99 MB; QTL1). (B) Introgression lines that begin at the left telomere of chromosome IV and are derived from kyIR28. Common segment method reveals QTL2 and QTL3 that confer HW-like behavior and antagonistic QTL5 that confers N2-like behavior. QTL4 and antagonistic QTL6 are supported by only one line (kyIR62) and are indicated with dashed lines. (C) Independent introgression lines (ewIR) from study by Doroszuk et al. (2009). Common segment method identifies QTL7 and QTL8. (D) Analysis of all chromosome IV introgression lines (kyIR and ewIR). All lines were analyzed, but only a subset that includes all lines significantly different from N2 is shown. Analysis of all lines yielded the same QTLs as in subsets, with three exceptions: QTL3 is a smaller region because it is defined by both ewIR and kyIR lines (specifically ewIR46 and kyIR67); antagonistic QTL6 is defined by two lines instead of one (ewIR47 and kyIR62); and QTL8 is only supported by one line, ewIR55. The line ewIR54 is no longer significantly different from N2 when part of a larger data set. Complete explanation of chromosome IV QTLs defined by common segment method appears in File S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multiple QTLs on chromosome IV. QTLs for bacterial choice of introgression lines derived from CSSIV (this article) and from recombinant inbred lines (Doroszuk &lt;em&gt;et al.&lt;/em&gt; 2009). Left, Genotypes are shown for various genetic markers (white is N2; black is HW; graded from white to black indicates unknown genomic regions between genotyped SNPs. Right, Bacterial choice behavior of introgression lines. Blue markers indicate lines that differ significantly from N2. ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001, **&lt;em&gt;P&lt;/em&gt; &lt; 0.01, *&lt;em&gt;P&lt;/em&gt; &lt; 0.05, ANOVA with Dunnett, n &#x2265; 5 assays. S.E.M. represented by error bars. Chromosome IV introgression line strain names, choice index, and genotype at additional genetic markers are in File S4. Introgression lines were analyzed by the common segment method to determine QTLs. The inferred locations of QTLs that decrease &lt;em&gt;Serratia&lt;/em&gt; preference are indicated by blue vertical lines on the genetic map; antagonistic QTLs that restore N2-like phenotype are indicated by red vertical lines on the genetic map. N2 and CB4856 were tested as well as subset of introgression strains on each day. The N2 and CB4856 data shown are the average data for N2 and CB4856 tested on all days that the introgression strains were tested. (A) Initial set of lines derived from CSSIV. Using the common segment method, recombinant strains that differed significantly from N2 shared an HW region (&#x223C;2.29 to &#x223C;4.99 MB; QTL1). (B) Introgression lines that begin at the left telomere of chromosome IV and are derived from &lt;em&gt;kyIR28&lt;/em&gt;. Common segment method reveals QTL2 and QTL3 that confer HW-like behavior and antagonistic QTL5 that confers N2-like behavior. QTL4 and antagonistic QTL6 are supported by only one line (&lt;em&gt;kyIR62&lt;/em&gt;) and are indicated with dashed lines. (C) Independent introgression lines (&lt;em&gt;ewIR&lt;/em&gt;) from study by Doroszuk &lt;em&gt;et al.&lt;/em&gt; (2009). Common segment method identifies QTL7 and QTL8. (D) Analysis of all chromosome IV introgression lines (&lt;em&gt;kyIR&lt;/em&gt; and &lt;em&gt;ewIR&lt;/em&gt;). All lines were analyzed, but only a subset that includes all lines significantly different from N2 is shown. Analysis of all lines yielded the same QTLs as in subsets, with three exceptions: QTL3 is a smaller region because it is defined by both &lt;em&gt;ewIR&lt;/em&gt; and &lt;em&gt;kyIR&lt;/em&gt; lines (specifically &lt;em&gt;ewIR46&lt;/em&gt; and &lt;em&gt;kyIR67&lt;/em&gt;); antagonistic QTL6 is defined by two lines instead of one (&lt;em&gt;ewIR47&lt;/em&gt; and &lt;em&gt;kyIR62&lt;/em&gt;); and QTL8 is only supported by one line, &lt;em&gt;ewIR55&lt;/em&gt;. The line &lt;em&gt;ewIR54&lt;/em&gt; is no longer significantly different from N2 when part of a larger data set. Complete explanation of chromosome IV QTLs defined by common segment method appears in File S6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-5.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-5.medium.gif" width="328" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F4/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Multiple QTLs on chromosome IV. QTLs for bacterial choice of introgression lines derived from CSSIV (this article) and from recombinant inbred lines (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Doroszuk <em>et al.</em> 2009</a>). Left, Genotypes are shown for various genetic markers (white is N2; black is HW; graded from white to black indicates unknown genomic regions between genotyped SNPs. Right, Bacterial choice behavior of introgression lines. Blue markers indicate lines that differ significantly from N2. ***<em>P</em> &lt; 0.001, **<em>P</em> &lt; 0.01, *<em>P</em> &lt; 0.05, ANOVA with Dunnett, n ≥ 5 assays. S.E.M. represented by error bars. Chromosome IV introgression line strain names, choice index, and genotype at additional genetic markers are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS4.xlsx">File S4</a>. Introgression lines were analyzed by the common segment method to determine QTLs. The inferred locations of QTLs that decrease <em>Serratia</em> preference are indicated by blue vertical lines on the genetic map; antagonistic QTLs that restore N2-like phenotype are indicated by red vertical lines on the genetic map. N2 and CB4856 were tested as well as subset of introgression strains on each day. The N2 and CB4856 data shown are the average data for N2 and CB4856 tested on all days that the introgression strains were tested. (A) Initial set of lines derived from CSSIV. Using the common segment method, recombinant strains that differed significantly from N2 shared an HW region (∼2.29 to ∼4.99 MB; QTL1). (B) Introgression lines that begin at the left telomere of chromosome IV and are derived from <em>kyIR28</em>. Common segment method reveals QTL2 and QTL3 that confer HW-like behavior and antagonistic QTL5 that confers N2-like behavior. QTL4 and antagonistic QTL6 are supported by only one line (<em>kyIR62</em>) and are indicated with dashed lines. (C) Independent introgression lines (<em>ewIR</em>) from study by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Doroszuk <em>et al.</em> (2009)</a>. Common segment method identifies QTL7 and QTL8. (D) Analysis of all chromosome IV introgression lines (<em>kyIR</em> and <em>ewIR</em>). All lines were analyzed, but only a subset that includes all lines significantly different from N2 is shown. Analysis of all lines yielded the same QTLs as in subsets, with three exceptions: QTL3 is a smaller region because it is defined by both <em>ewIR</em> and <em>kyIR</em> lines (specifically <em>ewIR46</em> and <em>kyIR67</em>); antagonistic QTL6 is defined by two lines instead of one (<em>ewIR47</em> and <em>kyIR62</em>); and QTL8 is only supported by one line, <em>ewIR55</em>. The line <em>ewIR54</em> is no longer significantly different from N2 when part of a larger data set. Complete explanation of chromosome IV QTLs defined by common segment method appears in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS6.pdf">File S6</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The introgression strain <em>kyIR28</em> resembled CSSIV in the choice index but contained only 5 MB of HW DNA beginning at the left telomere of chromosome IV (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Using <em>kyIR28</em> as a starting point, we generated additional recombinants as a nested set of introgression lines that derived from <em>kyIR28</em> and included HW sequences beginning at the left telomere of chromosome IV. These strains were tested for preference behavior as homozygotes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). Direct inspection of their phenotypes suggested that <em>kyIR28</em> probably contains more than one QTL: two groups of strains within the nested series were HW-like (<em>kyIR76,74</em> and <em>kyIR67,68,75,65</em>), but another group of nested strains were <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like <em>(kyIR69,66,42,71</em>). The simplest explanation for these results is the existence of two QTLs that confer HW-like behavior (QTL2 and QTL3), separated by a third antagonistic QTL from the HW strain that confers <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like behavior (QTL5). Statistical testing of these strains using the “common segment” method as described by <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Shao <em>et al.</em> (2010)</a> using ANOVA with Dunnett correction for multiple testing supported the existence of each of these three QTLs (<em>P</em> &lt; 0.05) (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-31">Statistical testing also provided support for two additional QTLs of opposite signs, one conferring HW-like behavior (QTL4) and one conferring <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like behavior (QTL6). The existence of QTL4 and QTL6 was supported only by a single introgression strain, <em>kyIR62</em>, whereas the existence of QTL2, QTL3, and QTL5 were all supported by multiple strains (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>).</p><p id="p-32">The antagonistic interactions among QTLs in these strains suggest that HW QTL do not uniformly promote HW-like behavior; some regions of HW DNA, including QTL5 and possibly QTL6, favor <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like behavior.</p></div><div id="sec-15" class="subsection"><h3>Chromosome IV QTL defined by independent introgression lines</h3><p id="p-33">It has been suggested that the most powerful way to identify multiple QTLs is to use contiguous congenic strains that tile a chromosome in small segments with minimal overlap (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rapp and Joe 2012</a>). In <em>C. elegans</em>, congenic strains of this design that span the genome have been generated between the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW strains and colleagues (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Doroszuk <em>et al.</em> 2009</a>). We systematically examined the strains that covered chromosome IV to test the power of these strains for identifying QTLs and to ask if congenic strains generated by different approaches would yield similar QTLs.</p><p id="p-34">Two QTLs that confer HW-like behavior were identified from this analysis (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a>). One, QTL7, fell in the same region as QTL1. The second, QTL8, fell on the right arm of the chromosome, in a region that was poorly resolved by breakpoints in the previous set of introgression lines (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>) but was well-resolved in this set (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4C</a>).</p><p id="p-35">Combining all data from all introgression lines into a single dataset yielded results consistent with those from individual strains (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4D</a>), with four to five QTLs favoring HW-like behavior (QTL2, QTL3, QTL4, QTL7, QTL8) and two antagonistic QTLs favoring <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like behavior (QTL5, QTL6). Contrary to the simple expectation that chromosome IV might have one major locus for bacterial preference, the introgression lines defined multiple QTLs, whose numbers increased as the number of informative recombination breakpoints increased.</p><p id="p-36">The common segment method has a long history of use in congenic inbred strains (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Snell and Bunker 1965</a>), but alternative methods for mapping have recently been proposed to be more rigorous. We used the sequential minimum spanning tree method (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Shao <em>et al.</em> 2010</a>) to examine the same set of introgression lines characterized and found that this method identified four QTLs: QTL m2, which overlapped with QTL2; QTL m7, which overlapped with QTL7; antagonistic QTL m6, which overlapped with QTL6; and QTL m8, identified only in the <em>ewIR</em> set, which overlapped with QTL8 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The sequential method uses a very stringent Bonferroni correction for multiple testing; less stringent approaches (<em>e.g.</em>, false discovery rate) suggest the presence of multiple additional QTLs coincident with those found by the common segment method.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of QTLs on chromosome IV. Location of QTLs determined by common segment method (Figure 4) [blue and red (antagonistic QTLs)] and by sequential minimum spanning tree method [green and red (antagonistic QTLs)] showing generally similar locations of QTL (bottom of figure). In the sequential minimum spanning tree method, the introgression lines that differ significantly from each other (t test with Bonferonni correction, P &lt; 0.05) in both subsets of lines and all lines were as follows: kyIR76 and kyIR54, defining QTL m2 (0.79&#x2013;1.03 Mb) and ewIR53 and N2, defining QTL m7 (2.76&#x2013;3.35 Mb). The difference between ewIR58 and ewIR60, defining QTL m8 (10.12&#x2013;12.75 Mb), was significant in ewIR lines, but not all lines combined. The significant difference between ewIR47 and kyIR65, defining antagonistic QTL m6 (&#x223C;2.76 to &#x223C;3.92 Mb), was present only in all lines combined. Complete explanation of chromosome IV QTL defined by sequential MST appears in File S7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of QTLs on chromosome IV. Location of QTLs determined by common segment method (Figure 4) [blue and red (antagonistic QTLs)] and by sequential minimum spanning tree method [green and red (antagonistic QTLs)] showing generally similar locations of QTL (bottom of figure). In the sequential minimum spanning tree method, the introgression lines that differ significantly from each other (&lt;em&gt;t&lt;/em&gt; test with Bonferonni correction, &lt;em&gt;P&lt;/em&gt; &lt; 0.05) in both subsets of lines and all lines were as follows: &lt;em&gt;kyIR76&lt;/em&gt; and &lt;em&gt;kyIR54&lt;/em&gt;, defining QTL m2 (0.79&#x2013;1.03 Mb) and &lt;em&gt;ewIR53&lt;/em&gt; and &lt;em&gt;N2&lt;/em&gt;, defining QTL m7 (2.76&#x2013;3.35 Mb). The difference between &lt;em&gt;ewIR58&lt;/em&gt; and &lt;em&gt;ewIR60&lt;/em&gt;, defining QTL m8 (10.12&#x2013;12.75 Mb), was significant in &lt;em&gt;ewIR&lt;/em&gt; lines, but not all lines combined. The significant difference between &lt;em&gt;ewIR47&lt;/em&gt; and &lt;em&gt;kyIR65&lt;/em&gt;, defining antagonistic QTL m6 (&#x223C;2.76 to &#x223C;3.92 Mb), was present only in all lines combined. Complete explanation of chromosome IV QTL defined by sequential MST appears in File S7.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F5.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/2/265/F5.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463934" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Summary of QTLs on chromosome IV. Location of QTLs determined by common segment method (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>) [blue and red (antagonistic QTLs)] and by sequential minimum spanning tree method [green and red (antagonistic QTLs)] showing generally similar locations of QTL (bottom of figure). In the sequential minimum spanning tree method, the introgression lines that differ significantly from each other (<em>t</em> test with Bonferonni correction, <em>P</em> &lt; 0.05) in both subsets of lines and all lines were as follows: <em>kyIR76</em> and <em>kyIR54</em>, defining QTL m2 (0.79–1.03 Mb) and <em>ewIR53</em> and <em>N2</em>, defining QTL m7 (2.76–3.35 Mb). The difference between <em>ewIR58</em> and <em>ewIR60</em>, defining QTL m8 (10.12–12.75 Mb), was significant in <em>ewIR</em> lines, but not all lines combined. The significant difference between <em>ewIR47</em> and <em>kyIR65</em>, defining antagonistic QTL m6 (∼2.76 to ∼3.92 Mb), was present only in all lines combined. Complete explanation of chromosome IV QTL defined by sequential MST appears in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS7.pdf">File S7</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Initial localization of chromosome V QTL</h3><p id="p-38">To further characterize the inferred QTL or QTLs on chromosome V suggested by the CSSV strain (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>), we examined minimally overlapping congenic strains generated between the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW strains (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Doroszuk <em>et al.</em> 2009</a>). The common segment analysis assuming the smallest possible number of contributing QTLs on chromosome V identified QTL9 (∼10.91 to ∼13.95), defined by the introgression line <em>ewIR71</em> that differed significantly from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). However, the strain <em>ewIR70</em> that included this region and additional sequences had <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like behavioral preference, suggesting that one or more antagonistic QTLs on chromosome V modify the QTL9 preference. The sequential minimum spanning tree method also identified one QTL (QTL m9) in an interval adjacent to QTL9.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="At least one quantitative trait locus on chromosome V. QTLs for bacterial choice of introgression lines derived from recombinant inbred lines (Doroszuk et al. 2009). Data are portrayed as in Figure 4. Chromosome V introgression line strain names and choice index appear in File S5. The common segment analysis of chromosome V identified QTL9 (&#x223C;10.91 to &#x223C;13.95), defined by the introgression line ewIR71 that differed significantly from N2. Antagonistic QTL10 is defined by the strain ewIR70 that includes QTL9 and had an N2-like behavioral preference. The sequential minimum spanning tree method identified QTL m9 because of the significant difference in the choice index between ewIR69 and ewIR70. Additional explanation of chromosome V QTL defined by the common segment method and sequential MST appears in File S8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;At least one quantitative trait locus on chromosome V. QTLs for bacterial choice of introgression lines derived from recombinant inbred lines (Doroszuk &lt;em&gt;et al.&lt;/em&gt; 2009). Data are portrayed as in Figure 4. Chromosome V introgression line strain names and choice index appear in File S5. The common segment analysis of chromosome V identified QTL9 (&#x223C;10.91 to &#x223C;13.95), defined by the introgression line &lt;em&gt;ewIR71&lt;/em&gt; that differed significantly from N2. Antagonistic QTL10 is defined by the strain &lt;em&gt;ewIR70 that&lt;/em&gt; includes QTL9 and had an N2-like behavioral preference. The sequential minimum spanning tree method identified QTL m9 because of the significant difference in the choice index between &lt;em&gt;ewIR69&lt;/em&gt; and &lt;em&gt;ewIR70&lt;/em&gt;. Additional explanation of chromosome V QTL defined by the common segment method and sequential MST appears in File S8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F6.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/2/265/F6.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463974" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-39" class="first-child">At least one quantitative trait locus on chromosome V. QTLs for bacterial choice of introgression lines derived from recombinant inbred lines (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Doroszuk <em>et al.</em> 2009</a>). Data are portrayed as in <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4</a>. Chromosome V introgression line strain names and choice index appear in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS5.xlsx">File S5</a>. The common segment analysis of chromosome V identified QTL9 (∼10.91 to ∼13.95), defined by the introgression line <em>ewIR71</em> that differed significantly from N2. Antagonistic QTL10 is defined by the strain <em>ewIR70 that</em> includes QTL9 and had an N2-like behavioral preference. The sequential minimum spanning tree method identified QTL m9 because of the significant difference in the choice index between <em>ewIR69</em> and <em>ewIR70</em>. Additional explanation of chromosome V QTL defined by the common segment method and sequential MST appears in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FileS8.pdf">File S8</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The full set of suggested QTLs for bacterial preference converged on several similar regions for chromosome IV but were less well-defined for chromosome V (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Together, our mapping data suggest that there are at least four and probably five or six QTLs on chromosome IV and chromosome V that confer HW-like behavior in the HW strain, along with at least two antagonistic loci.</p><p id="p-41">To estimate the effect size of individual QTLs, we examined the behavior of introgression strains that should each contain a unique QTL among those defined here, after further backcrossing onto a common <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-like genetic background. This analysis was possible for QTL2, QTL7, and QTL9, defined by the nonoverlapping introgression lines <em>kyIR76</em>, <em>ewIR53</em>, and <em>ewIR71</em>, respectively. The preference behavior in each strain was strongly HW-like, ranging from 66% to 102% of the preference difference between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The cumulative phenotypic effect of these three QTLs was 264%, exceeding the 100% starting difference between the two parental strains.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/265/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect sizes of individual QTL on an N2 background. Introgression strains that should each contain a unique QTL (or set of QTLs). Two loci are on chromosome IV: QTL2 (kyIR76), &#x223C;0.79 to &#x223C;1.03 Mb, and QTL7 (ewIR53), &#x223C;2.76 to &#x223C;3.35 Mb. One locus is on chromosome V: QTL9 (ewIR71), &#x223C;10.91 to &#x223C;13.95 Mb. Each QTL was backcrossed two additional times onto the N2 background before testing, yielding choice indexes slightly different from the original strains in Figure 4. Percentages above bars indicate the preference behavior of each strain as a percentage of the preference difference between N2 and HW. Horizontal dashed lines indicate choice index for 0% and 100% preference difference between N2 and HW. S.E.M. represented as error bars. At bottom, blue segments indicate approximate location of QTLs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-610021924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect sizes of individual QTL on an N2 background. Introgression strains that should each contain a unique QTL (or set of QTLs). Two loci are on chromosome IV: QTL2 (&lt;em&gt;kyIR76&lt;/em&gt;), &#x223C;0.79 to &#x223C;1.03 Mb, and QTL7 (&lt;em&gt;ewIR53&lt;/em&gt;), &#x223C;2.76 to &#x223C;3.35 Mb. One locus is on chromosome V: QTL9 (&lt;em&gt;ewIR71&lt;/em&gt;), &#x223C;10.91 to &#x223C;13.95 Mb. Each QTL was backcrossed two additional times onto the N2 background before testing, yielding choice indexes slightly different from the original strains in Figure 4. Percentages above bars indicate the preference behavior of each strain as a percentage of the preference difference between N2 and HW. Horizontal dashed lines indicate choice index for 0% and 100% preference difference between N2 and HW. S.E.M. represented as error bars. At bottom, blue segments indicate approximate location of QTLs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/265/F7.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/2/265/F7.medium.gif" width="400" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/265/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/265/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464002" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">Effect sizes of individual QTL on an N2 background. Introgression strains that should each contain a unique QTL (or set of QTLs). Two loci are on chromosome IV: QTL2 (<em>kyIR76</em>), ∼0.79 to ∼1.03 Mb, and QTL7 (<em>ewIR53</em>), ∼2.76 to ∼3.35 Mb. One locus is on chromosome V: QTL9 (<em>ewIR71</em>), ∼10.91 to ∼13.95 Mb. Each QTL was backcrossed two additional times onto the N2 background before testing, yielding choice indexes slightly different from the original strains in <a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4</a>. Percentages above bars indicate the preference behavior of each strain as a percentage of the preference difference between N2 and HW. Horizontal dashed lines indicate choice index for 0% and 100% preference difference between N2 and HW. S.E.M. represented as error bars. At bottom, blue segments indicate approximate location of QTLs.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Preference for <em>Serratia</em> bacteria</h3><p id="p-43">Animals from the HW had a lower preference for <em>Serratia</em> than <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> animals, and four other wild strains had intermediate preferences compared to these two strains. It is surprising that <em>C. elegans</em> has a strong preference for <em>S. marcescens</em>, a pathogenic bacteria that can kill infected animals in a few days (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mallo <em>et al.</em> 2002</a>). This may be an example of a host–pathogen evolutionary arms race in which the pathogen is winning by attracting its host (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Niu <em>et al.</em> 2010</a>) or a fortuitous event in which <em>Serratia</em> odors resemble those of other harmless bacteria. Although there should be a strong selection for avoidance of this odor, the level of complexity of the microbiome may challenge even the considerable genetic capacity of <em>C. elegans</em> for chemosensation.</p></div><div id="sec-19" class="subsection"><h3>Complex genetics of bacterial preference traits</h3><p id="p-44">The analysis of <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>–HW strains suggests the existence of as many as nine QTLs on HW chromosomes IV and V and perhaps one on chromosome II that affect bacterial preference. The location and number of QTLs identified were sensitive to the exact strains and analysis methods that were used, but several different approaches and two independently derived sets of introgressed strains yielded similar locations for most QTLs on chromosome IV (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). QTL alleles on two different HW chromosomes favor HW-like behavioral preferences, whereas additional QTLs have antagonistic effects.</p><p id="p-45">Identifying QTLs for bacterial preference traits proved unexpectedly challenging, but it became more straightforward as smaller regions of HW DNA were successively introgressed onto an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background. The RIAILs with a 50:50 mix of HW and <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> DNA yielded the fewest QTLs (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), the chromosome substitution strains were more informative (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), and the smallest nested introgression strains (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4B</a>) gave the most informative and interpretable results. These results are most simply explained by the relatively complex genetic architecture of the underlying trait and particularly the presence of alleles in the HW strain that suppress the effects of HW alleles at other loci. Our results suggest that this complex trait is most effectively dissected by analyzing small genetic regions in a common strain background, with the knowledge that this approach (and probably any experimentally feasible approach) will reveal only a subset of the QTLs.</p><p id="p-46">Although the bacterial chemotaxis assay uses a scale that is, in principle, able to detect additive factors, the QTLs on chromosomes IV and V appeared not to have additive effects on preference (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). This observation suggests that epistatic interactions among QTLs affect the behavioral preference phenotype, as is seen in many other systems. For example, natural variation in aggressive behavior between two wild-type strains of <em>Drosophila</em> involves at least five QTLs with epistatic interactions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Edwards <em>et al.</em> 2009</a>). In <em>C. elegans</em>, epistatic interactions among multiple loci that vary between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HW strains cause synergistic effects on thermal preference (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gaertner <em>et al.</em> 2012</a>). Multiple epistatic QTLs contribute to natural variation in metabolic, blood, and bone trait differences between two wild-type strains of mice (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Shao <em>et al.</em> 2008</a>). Discovery of the mouse metabolic QTL was facilitated by characterizing nested introgression lines within a chromosome (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Shao <em>et al.</em> 2008</a>), the approach that also succeeded best in our study of <em>C. elegans</em> preference behavior.</p><p id="p-47">Although several groups have successfully identified <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>–HW QTLs using recombinant inbred advanced intercross lines, the analysis of RIAILs did not identify any of the QTLs for behavioral preference defined by the introgression approach, despite having sufficient power to detect QTLs that explain 30% of variance among RIAILs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1/FigureS2.pdf">Figure S2</a>). We confirmed this negative result in the RIAIL lines with a multiple QTL model that includes the QTLs defined in the introgression lines; none of the QTLs defined by introgression lines was significant in any model incorporating some or all of them with or without interactions. Given the apparent prevalence of epistasis among QTLs, this discrepancy is explained by the large number of segregating genotypes compared to the number of tested RIAILs. Only 72 RIAILs were tested, a number that is small relative to the 128 possible genotypes at seven QTLs. With this genetic complexity, even QTLs that individually had large effects on specific backgrounds became undetectable when averaged across backgrounds. These results point out the value of testing smaller, defined genomic regions in introgression lines as a complementary approach to combining loci randomly in conventional RIL analysis.</p></div><div id="sec-20" class="subsection"><h3>Similarities between mammalian and <em>C. elegans</em> complex trait genetics</h3><p id="p-48">This introgression analysis of <em>C. elegans</em> odor preference yielded results strikingly similar to an analysis of mouse metabolic and behavioral traits from 22 introgression lines with BALB/c chromosomes introduced into the C57B6 strain (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Shao <em>et al.</em> 2008</a>). First, the mouse chromosome substitution strains showed that for many traits, the effect size of a single chromosome was at least half of the total difference between the two starting strains. Second, many chromosome substitutions could affect any single metabolic or behavioral trait, so that the total effect sizes added together often represented 600% or more of the difference between the two starting mouse strains. Third, adding together multiple chromosome substitutions did not result in additive effects on the traits, and extensive epistasis often masked or reversed the effect of single chromosomes.</p><p id="p-49">Population genetic analysis is appropriately focused on trait variance; however, from a mechanistic perspective, it is more straightforward to examine each genetic variant in a defined background before reconstructing the entire system. Therefore, applying introgression studies to define genetic factors may be a valuable approach to problems in behavior, metabolism, and other complex traits.</p></div><div id="sec-21" class="subsection"><h3>Food choice behavior evolves rapidly</h3><p id="p-50">Increasing evidence suggests that taste and olfactory preferences are particularly fast-evolving behaviors that coordinate the behavioral and metabolic specializations of a species. For example, over the past 500,000 years, <em>Drosophila sechellia</em> has acquired metabolic specializations for growth on the toxic morinda fruit, in tandem with olfactory preferences for the same fruit (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jones 1998</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McBride 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stensmyr 2009</a>). Over several million years, felines with a carnivorous diet that lacks sugar have accumulated inactivating mutations in the <em>Tas1r3</em> receptor gene, which is required for sweet taste in other mammals (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2005</a>). Strong recent signatures of positive selection on human bitter taste receptor genes suggest that dietary pressures, such as recognizing toxic foods, may also have left their mark on human sensory preference (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Campbell <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2011</a>).</p><p id="p-51">The chemosensory system of <em>C. elegans</em> is rapidly evolving compared to the rest of its genome, suggesting that it is under positive selection (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Robertson 1998</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Robertson 2000</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stewart <em>et al.</em> 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Thomas <em>et al.</em> 2005</a>). Nearly 2000 <em>C. elegans</em> genes encode G-protein-coupled chemoreceptor genes, representing 5% to 10% of all protein-coding genes, and these genes are divergent among <em>C. elegans</em> wild isolates and among <em>Caenorhabditis</em> species (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Stewart <em>et al.</em> 2005</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Thomas and Robertson 2008</a>). We speculate that the spectrum of wholesome and pathogenic bacteria in different environments generates local selective pressures on chemoreceptors and other <em>C. elegans</em> genes and subsequent within-species genetic diversity. This suggestion is consistent with the cosmopolitan lifestyle of <em>C. elegans</em> and its broad dispersion through a variety of human agricultural environments.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We thank Jan Kammenga, Leonid Kruglyak, and Man-Wah Tan for strains; Patrick McGrath and Andres Bendesky for methods and advice; Patrick Liu for technical assistance; all members of the Bargmann laboratory for discussion; and WormBase for N2-HW SNPs. E.G. and C.I.B. designed experiments, E.G. conducted genetic and behavioral experiments, E.G. and M.R. analyzed RIAIL data and introgression line data, and E.G. and C.I.B. interpreted results and wrote the paper. C.I.B. is an Investigator of the Howard Hughes Medical Institute. Research was funded by the Howard Hughes Medical Institute, National Science Foundation Postdoctoral Fellowship #0706753 (to E.G.) and National Institutes of Health grant R01GM089972 (to M.R.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008649/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">1</span> Department of Neuroscience, Pomona College, Claremont, California 91711, USA</p></li><li class="fn" id="fn-3"><p id="p-53"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-07-30"><span class="received-label">Received </span>July 30, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-12-07"><span class="accepted-label">Accepted </span>December 7, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Glater <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.1" data-doi="10.1126/science.1156409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic mapping in human disease.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D322%26rft.issue%253D5903%26rft.spage%253D881%26rft.epage%253D888%26rft.atitle%253DGenetic%2BMapping%2Bin%2BHuman%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1156409%26rft_id%253Dinfo%253Apmid%252F18988837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMy84ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.2" data-doi="10.1084/jem.136.6.1430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Widmer</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Serologically defined and lymphocyte-defined components of the major histocompatibility complex in the mouse.</span> <abbr class="cit-jnl-abbrev">J. Exp. Med.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1430</span>–<span class="cit-lpage">1444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.136.6.1430%26rft_id%253Dinfo%253Apmid%252F4404878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjEwOiIxMzYvNi8xNDMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-4.2.265.3"><div class="cit-metadata unstructured">Bargmann, C. I., 2006 Chemosensation in <em>C. elegans</em> in <em>Wormbook</em>, ed. <em>C. elegans</em> Research Community. Wormbook, doi/10.1895/wormbook.1.123.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.4" data-doi="10.1038/ni.2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic contributions to behavioural diversity at the gene-environment interface.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBendesky%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D809%26rft.epage%253D820%26rft.atitle%253DGenetic%2Bcontributions%2Bto%2Bbehavioural%2Bdiversity%2Bat%2Bthe%2Bgene-environment%2Binterface.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.2088%26rft_id%253Dinfo%253Apmid%252F22064512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22064512&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.5" data-doi="10.1038/nature09821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsunozaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Catecholamine receptor polymorphisms affect decision-making in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">472</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBendesky%26rft.auinit1%253DA.%26rft.volume%253D472%26rft.issue%253D7343%26rft.spage%253D313%26rft.epage%253D318%26rft.atitle%253DCatecholamine%2Breceptor%2Bpolymorphisms%2Baffect%2Bdecision-making%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09821%26rft_id%253Dinfo%253Apmid%252F21412235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21412235&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289724600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.6" data-doi="10.1371/journal.pgen.1002323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatnagar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oler</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabaglia</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapleton</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schueler</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Positional cloning of a type 2 diabetes quantitative trait locus; tomosyn-2, a negative regulator of insulin secretion.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBhatnagar%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002323%26rft.epage%253De1002323%26rft.atitle%253DPositional%2Bcloning%2Bof%2Ba%2Btype%2B2%2Bdiabetes%2Bquantitative%2Btrait%2Blocus%253B%2Btomosyn-2%252C%2Ba%2Bnegative%2Bregulator%2Bof%2Binsulin%2Bsecretion.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002323%26rft_id%253Dinfo%253Apmid%252F21998599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21998599&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.7" data-doi="10.1051/medsci/20031967725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrenner%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D94%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fmedsci%252F20031967725%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzcvMS83MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8yNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.8" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8yNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.9" data-doi="10.1093/molbev/msr293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranciaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froment</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirbo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of functionally diverse alleles associated with PTC bitter taste sensitivity in Africa.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr293%26rft_id%253Dinfo%253Apmid%252F22130969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzQvMTE0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8yNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.11" data-doi="10.1186/1471-2164-6-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammarlund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hullett</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid single nucleotide polymorphism mapping in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDavis%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.spage%253D118%26rft.epage%253D118%26rft.atitle%253DRapid%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmapping%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-6-118%26rft_id%253Dinfo%253Apmid%252F16156901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-6-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16156901&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.12" data-doi="10.1016/S0092-8674(00)81609-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Bono</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Natural variation in a neuropeptide Y receptor homolog modifies social behavior and food response in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253Dde%2BBono%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D5%26rft.spage%253D679%26rft.epage%253D689%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bneuropeptide%2BY%2Breceptor%2Bhomolog%2Bmodifies%2Bsocial%2Bbehavior%2Band%2Bfood%2Bresponse%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981609-8%26rft_id%253Dinfo%253Apmid%252F9741632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81609-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9741632&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075851900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.13" data-doi="10.1093/nar/gkp528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doroszuk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fradin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome-wide library of CB4856/N2 introgression lines of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp528%26rft_id%253Dinfo%253Apmid%252F19542186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8xNi9lMTEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.14" data-doi="10.1186/1741-7007-7-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zwarts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callaerts</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutations in many genes affect aggressive behavior in <em>Drosophila</em> melanogaster.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DEdwards%26rft.auinit1%253DA.%2BC.%26rft.volume%253D7%26rft.spage%253D29%26rft.epage%253D29%26rft.atitle%253DMutations%2Bin%2Bmany%2Bgenes%2Baffect%2Baggressive%2Bbehavior%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-7-29%26rft_id%253Dinfo%253Apmid%252F19519879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-7-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19519879&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.15" data-doi="10.1186/1741-7007-10-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felix</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duveau</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population dynamics and habitat sharing of natural populations of <em>Caenorhabditis elegans</em> and <em>C. briggsae</em>.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DFelix%26rft.auinit1%253DM.%2BA.%26rft.volume%253D10%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DPopulation%2Bdynamics%2Band%2Bhabitat%2Bsharing%2Bof%2Bnatural%2Bpopulations%2Bof%2BCaenorhabditis%2Belegans%2Band%2BC.%2Bbriggsae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-10-59%26rft_id%253Dinfo%253Apmid%252F22731941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-10-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22731941&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.16" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> The correlation between relatives on the supposition of <span class="cit-article-title">Mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.17" data-doi="10.1534/genetics.112.142877"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaertner</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmenter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142877%26rft_id%253Dinfo%253Apmid%252F23086219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.18" data-doi="10.1534/genetics.111.127100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glauser</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macinnis</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellman</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heat avoidance is regulated by transient receptor potential (TRP) channels and a neuropeptide signaling pathway in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127100%26rft_id%253Dinfo%253Apmid%252F21368276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg4LzEvOTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doniach</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Natural variation and copulatory plug formation in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHodgkin%26rft.auinit1%253DJ.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D164%26rft.atitle%253DNatural%2BVariation%2Band%2BCopulatory%2BPlug%2BFormation%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F9136008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMTQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The genetic basis of <em>Drosophila sechellia’s</em> resistance to a host plant toxin.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1899</span>–<span class="cit-lpage">1908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJones%26rft.auinit1%253DC.%2BD.%26rft.volume%253D149%26rft.issue%253D4%26rft.spage%253D1899%26rft.epage%253D1908%26rft.atitle%253DThe%2BGenetic%2BBasis%2Bof%2BDrosophila%2Bsechellia%2527s%2BResistance%2Bto%2Ba%2BHost%2BPlant%2BToxin%26rft_id%253Dinfo%253Apmid%252F9691045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OS80LzE4OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.21" data-doi="10.1093/emboj/cdg159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurz</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauvet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aurouze</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallet</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Virulence factors of the human opportunistic pathogen <em>Serratia marcescens</em> identified by in vivo screening.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKurz%26rft.auinit1%253DC.%2BL.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D1451%26rft.epage%253D1460%26rft.atitle%253DVirulence%2Bfactors%2Bof%2Bthe%2Bhuman%2Bopportunistic%2Bpathogen%2BSerratia%2Bmarcescens%2Bidentified%2Bby%2Bin%2Bvivo%2Bscreening%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdg159%26rft_id%253Dinfo%253Apmid%252F12660152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMi83LzE0NTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.23" data-doi="10.3378/027.083.0303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pakstis</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Selection on the human bitter taste gene, TAS2R16, in Eurasian populations.</span> <abbr class="cit-jnl-abbrev">Hum. Biol.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BBiology%26rft.stitle%253DHuman%2BBiology%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D377%26rft.atitle%253DSelection%2Bon%2Bthe%2Bhuman%2Bbitter%2Btaste%2Bgene%252C%2BTAS2R16%252C%2Bin%2BEurasian%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.3378%252F027.083.0303%26rft_id%253Dinfo%253Apmid%252F21740153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3378/027.083.0303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21740153&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.24" data-doi="10.1371/journal.pgen.0010027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maehashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Pseudogenization of a sweet-receptor gene accounts for cats’ indifference toward sugar.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D35%26rft.atitle%253DPseudogenization%2Bof%2Ba%2Bsweet-receptor%2Bgene%2Baccounts%2Bfor%2Bcats%2527%2Bindifference%2Btoward%2Bsugar.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010027%26rft_id%253Dinfo%253Apmid%252F16103917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103917&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234295900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.25" data-doi="10.1016/S0960-9822(02)00928-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mallo</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurz</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couillault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granjeaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Inducible antibacterial defense system in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DMallo%26rft.auinit1%253DG.%2BV.%26rft.volume%253D12%26rft.issue%253D14%26rft.spage%253D1209%26rft.epage%253D1214%26rft.atitle%253DInducible%2Bantibacterial%2Bdefense%2Bsystem%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252802%252900928-4%26rft_id%253Dinfo%253Apmid%252F12176330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(02)00928-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12176330&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177093800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.26" data-doi="10.1073/pnas.0608424104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McBride</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid evolution of smell and taste receptor genes during host specialization in <em>Drosophila sechellia</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">4996</span>–<span class="cit-lpage">5001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608424104%26rft_id%253Dinfo%253Apmid%252F17360391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEyLzQ5OTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.27" data-doi="10.1084/jem.128.1.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDevitt</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyan</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Genetic control of the antibody response in inbred mice. Transfer of response by spleen cells and linkage to the major histocompatibility (H-2) locus.</span> <abbr class="cit-jnl-abbrev">J. Exp. Med.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.128.1.1%26rft_id%253Dinfo%253Apmid%252F5662015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjc6IjEyOC8xLzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.28" data-doi="10.1084/jem.135.6.1259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDevitt</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deak</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shreffler</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stimpfling</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic control of the immune response. Mapping of the Ir-1 locus.</span> <abbr class="cit-jnl-abbrev">J. Exp. Med.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">1259</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.135.6.1259%26rft_id%253Dinfo%253Apmid%252F4554451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjEwOiIxMzUvNi8xMjU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.29" data-doi="10.1016/j.neuron.2009.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macosko</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative mapping of a digenic behavioral trait implicates globin variation in <em>C. elegans</em> sensory behaviors.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D61%26rft.spage%253D692%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.02.012%26rft_id%253Dinfo%253Apmid%252F19285466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285466&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264366900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.30" data-doi="10.1016/j.cell.2012.02.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruvkun</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inactivation of conserved <em>C. elegans</em> genes engages pathogen- and xenobiotic-associated defenses.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMelo%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D149%26rft.issue%253D2%26rft.spage%253D452%26rft.epage%253D466%26rft.atitle%253DInactivation%2Bof%2Bconserved%2BC.%2Belegans%2Bgenes%2Bengages%2Bpathogen-%2Band%2Bxenobiotic-associated%2Bdefenses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.02.050%26rft_id%253Dinfo%253Apmid%252F22500807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.02.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500807&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302805600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.31" data-doi="10.1073/pnas.1007276107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A Trojan horse mechanism of bacterial pathogenesis against nematodes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">16631</span>–<span class="cit-lpage">16636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1007276107%26rft_id%253Dinfo%253Apmid%252F20733068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzM4LzE2NjMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-4.2.265.32"><div class="cit-metadata unstructured">Oksanen, J., F. G. Blanchet, R. Kindt, P. Legendre, P. R. Minchin <em>et al.</em>, 2012 Community Ecology Package. R package version 2.0–3.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.33" data-doi="10.1038/nature07171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palopoli</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, A. TinMaung, C. Ramsay, S. Curwen <em>et al</em>., <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular basis of the copulatory plug polymorphism in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPalopoli%26rft.auinit1%253DM.%2BF.%26rft.volume%253D454%26rft.issue%253D7207%26rft.spage%253D1019%26rft.epage%253D1022%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bthe%2Bcopulatory%2Bplug%2Bpolymorphism%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07171%26rft_id%253Dinfo%253Apmid%252F18633349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18633349&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258591000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.34" data-doi="10.1038/nature07820"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Persson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busch</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bretes</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Natural variation in a neural globin tunes oxygen sensing in wild <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPersson%26rft.auinit1%253DA.%26rft.volume%253D458%26rft.issue%253D7241%26rft.spage%253D1030%26rft.epage%253D1033%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bneural%2Bglobin%2Btunes%2Boxygen%2Bsensing%2Bin%2Bwild%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07820%26rft_id%253Dinfo%253Apmid%252F19262507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07820&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19262507&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265412900043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.35" data-doi="10.1073/pnas.0610281104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pradel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Detection and avoidance of a natural product from the pathogenic bacterium Serratia marcescens by <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">2295</span>–<span class="cit-lpage">2300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0610281104%26rft_id%253Dinfo%253Apmid%252F17267603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzcvMjI5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8yNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.36" data-doi="10.1016/S0960-9822(01)00241-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Link</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurz</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alloing</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A reverse genetic analysis of components of the Toll signaling pathway in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPujol%26rft.auinit1%253DN.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D809%26rft.epage%253D821%26rft.atitle%253DA%2Breverse%2Bgenetic%2Banalysis%2Bof%2Bcomponents%2Bof%2Bthe%2BToll%2Bsignaling%2Bpathway%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252801%252900241-X%26rft_id%253Dinfo%253Apmid%252F11516642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(01)00241-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11516642&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169200400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.37" data-doi="10.1152/physiolgenomics.00136.2011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joe</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Use of contiguous congenic strains in analyzing compound QTLs.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00136.2011%26rft_id%253Dinfo%253Apmid%252F22108210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo4OiI0NC8yLzExNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8yNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Two large families of chemoreceptor genes in the nematodes <em>Caenorhabditis elegans</em> and <em>Caenorhabditis briggsae</em> reveal extensive gene duplication, diversification, movement, and intron loss.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRobertson%26rft.auinit1%253DH.%2BM.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D449%26rft.epage%253D463%26rft.atitle%253DTwo%2BLarge%2BFamilies%2Bof%2BChemoreceptor%2BGenes%2Bin%2Bthe%2BNematodes%2BCaenorhabditis%2Belegans%2Band%2BCaenorhabditis%2Bbriggsae%2BReveal%2BExtensive%2BGene%2BDuplication%252C%2BDiversification%252C%2BMovement%252C%2Band%2BIntron%25A0Loss%26rft_id%253Dinfo%253Apmid%252F9582190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvNS80NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.39" data-doi="10.1101/gr.10.2.192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The large <em>srh</em> family of chemoreceptor genes in Caenorhabditis nematodes reveals processes of genome evolution involving large duplications and deletions and intron gains and losses.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRobertson%26rft.auinit1%253DH.%2BM.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D192%26rft.epage%253D203%26rft.atitle%253DThe%2BLarge%2Bsrh%2BFamily%2Bof%2BChemoreceptor%2BGenes%2Bin%2BCaenorhabditis%2BNematodes%2BReveals%2BProcesses%2Bof%2BGenome%2BEvolution%2BInvolving%2BLarge%2BDuplications%2Band%2BDeletions%2Band%2BIntron%2BGains%2Band%2BLosses%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.2.192%26rft_id%253Dinfo%253Apmid%252F10673277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzIvMTkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.40" data-doi="10.1371/journal.pgen.1000419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombinational landscape and population genomics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000419%26rft.epage%253De1000419%26rft.atitle%253DRecombinational%2Blandscape%2Band%2Bpopulation%2Bgenomics%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000419%26rft_id%253Dinfo%253Apmid%252F19283065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283065&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.41" data-doi="10.1186/1471-2148-4-49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulenburg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewbank</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Diversity and specificity in the interaction between <em>Caenorhabditis elegans</em> and the pathogen <em>Serratia marcescens</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DSchulenburg%26rft.auinit1%253DH.%26rft.volume%253D4%26rft.spage%253D49%26rft.epage%253D49%26rft.atitle%253DDiversity%2Band%2Bspecificity%2Bin%2Bthe%2Binteraction%2Bbetween%2BCaenorhabditis%2Belegans%2Band%2Bthe%2Bpathogen%2BSerratia%2Bmarcescens.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-4-49%26rft_id%253Dinfo%253Apmid%252F15555070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-4-49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15555070&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.42" data-doi="10.1017/S003118200300461X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulenburg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Natural variation in the response of <em>Caenorhabditis elegans</em> towards <em>Bacillus thuringiensis</em>.</span> <abbr class="cit-jnl-abbrev">Parasitology</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasitology%26rft.volume%253D128%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS003118200300461X%26rft_id%253Dinfo%253Apmid%252F15151149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S003118200300461X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15151149&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221548700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.43" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.44" data-doi="10.1073/pnas.0810388105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burrage</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinasac</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernest</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic architecture of complex traits: large phenotypic effects and pervasive epistasis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">19910</span>–<span class="cit-lpage">19914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810388105%26rft_id%253Dinfo%253Apmid%252F19066216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUwLzE5OTEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.45" data-doi="10.1007/s00335-010-9267-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinasac</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burrage</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Supelak</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analyzing complex traits with congenic strains.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DShao%26rft.auinit1%253DH.%26rft.volume%253D21%26rft.issue%253D5-6%26rft.spage%253D276%26rft.epage%253D286%26rft.atitle%253DAnalyzing%2Bcomplex%2Btraits%2Bwith%2Bcongenic%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-010-9267-5%26rft_id%253Dinfo%253Apmid%252F20524000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-010-9267-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20524000&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279918900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.46" data-doi="10.1242/jeb.01955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shtonda</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avery</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Dietary choice behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DShtonda%26rft.auinit1%253DB.%2BB.%26rft.volume%253D209%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D102%26rft.atitle%253DDietary%2Bchoice%2Bbehavior%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.01955%26rft_id%253Dinfo%253Apmid%252F16354781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjg6IjIwOS8xLzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.47" data-doi="10.1097/00007890-196503000-00011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bunker</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Histocompatibility genes of mice. V. Five new histocompatibility loci identified by congenic resistant lines on a C57b 10 background.</span> <abbr class="cit-jnl-abbrev">Transplantation</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransplantation%26rft.stitle%253DTransplantation%26rft.aulast%253DSnell%26rft.auinit1%253DG.%2BD.%26rft.volume%253D3%26rft.spage%253D235%26rft.epage%253D252%26rft.atitle%253DHISTOCOMPATIBILITY%2BGENES%2BOF%2BMICE.%2BV.%2BFIVE%2BNEW%2BHISTOCOMPATIBILITY%2BLOCI%2BIDENTIFIED%2BBY%2BCONGENIC%2BRESISTANT%2BLINES%2BON%2BA%2BC57B%2B10%2BBACKGROUND.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00007890-196503000-00011%26rft_id%253Dinfo%253Apmid%252F14265497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00007890-196503000-00011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14265497&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.48" data-doi="10.1111/j.1749-6632.2009.03911.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stensmyr</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Drosophila sechellia</em> as a model in chemosensory neuroecology.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1170</span>: <span class="cit-fpage">468</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DStensmyr%26rft.auinit1%253DM.%2BC.%26rft.volume%253D1170%26rft.spage%253D468%26rft.epage%253D475%26rft.atitle%253DDrosophila%2Bsechellia%2Bas%2Ba%2Bmodel%2Bin%2Bchemosensory%2Bneuroecology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2009.03911.x%26rft_id%253Dinfo%253Apmid%252F19686179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2009.03911.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19686179&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270495700084&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.49" data-doi="10.1534/genetics.104.035329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merrihew</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">High genetic diversity in the chemoreceptor superfamily of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1985</span>–<span class="cit-lpage">1996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035329%26rft_id%253Dinfo%253Apmid%252F15520260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.50" data-doi="10.1073/pnas.0406469102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution in the SRZ chemoreceptor families of Caenorhabditis elegans and <em>Caenorhabditis briggsae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4476</span>–<span class="cit-lpage">4481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0406469102%26rft_id%253Dinfo%253Apmid%252F15761060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzEyLzQ0NzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.51" data-doi="10.1186/1741-7007-6-42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The <em>Caenorhabditis</em> chemoreceptor gene families.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DThomas%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D6%26rft.spage%253D42%26rft.epage%253D42%26rft.atitle%253DThe%2BCaenorhabditis%2Bchemoreceptor%2Bgene%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-6-42%26rft_id%253Dinfo%253Apmid%252F18837995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-6-42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18837995&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.265.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venables</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripley</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Modern applied statistics with S</span>, <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.53" data-doi="10.1046/j.1440-169X.2003.00703.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viney</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Variation in <em>Caenorhabditis elegans</em> dauer larva formation.</span> <abbr class="cit-jnl-abbrev">Dev. Growth Differ.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%252C%2Bgrowth%2B%2526%2Bdifferentiation%26rft.stitle%253DDev%2BGrowth%2BDiffer%26rft.aulast%253DViney%26rft.auinit1%253DM.%2BE.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D389%26rft.epage%253D396%26rft.atitle%253DVariation%2Bin%2BCaenorhabditis%2Belegans%2Bdauer%2Blarva%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1440-169X.2003.00703.x%26rft_id%253Dinfo%253Apmid%252F12950280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1440-169X.2003.00703.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12950280&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185196200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.54" data-doi="10.1186/1741-7007-11-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Hellenberg Hubar</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coopman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene-environment and protein degradation signatures characterize genomic and phenotypic diversity in wild <em>Caenorhabditis elegans</em> populations.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-11-1%26rft_id%253Dinfo%253Apmid%252F23294804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-11-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23294804&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.265.55" data-doi="10.1038/nature04216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Pathogenic bacteria induce aversive olfactory learning in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D438%26rft.issue%253D7065%26rft.spage%253D179%26rft.epage%253D184%26rft.atitle%253DPathogenic%2Bbacteria%2Binduce%2Baversive%2Bolfactory%2Blearning%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04216%26rft_id%253Dinfo%253Apmid%252F16281027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16281027&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233133500039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/2/265.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/2/255" title="Genetic Architecture of Contemporary Adaptation to Biotic Invasions: Quantitative Trait Locus Mapping of Beak Reduction in Soapberry Bugs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/2/277" title="Extensive Differences in Gene Expression Between Symbiotic and Aposymbiotic Cnidarians" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/2"><span>Volume 4 Issue 2, February 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/2.cover-source.jpg?itok=nk9yht63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/265" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vcXbOHU64TU0Da7talLiY_CpTrfWrCLiKb3NRVEqe68" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/2/265" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/2/265" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/2/265" class="active" data-icon-position="" data-hide-link-title="0">Multigenic Natural Variation Underlies Caenorhabditis elegans Olfactory Preference for the Bacterial Pathogen Serratia marcescens</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459640" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kkYIklrw_2CQreBFTivWCYYGcz5om8Yjck27d-SVCPQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/2/265.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/2/265" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-iCP-8GBZ6tLjOV18I-3rSEXd_iyv1G0l7Z0E864g0-c" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/2/265" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/2/265.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459640" id="node459640--2" data-pisa="ggg;4/2/265" data-pisa-master="ggg;g3.113.008649" data-apath="/ggg/4/2/265.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multigenic Natural Variation Underlies <em>Caenorhabditis elegans</em> Olfactory Preference for the Bacterial Pathogen <em>Serratia marcescens</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elizabeth E. Glater</span>, <span class="highwire-citation-author" data-delta="1">Matthew V. Rockman</span> and <span class="highwire-citation-author" data-delta="2">Cornelia I. Bargmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">265-276;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008649 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459640--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth E. Glater </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Harvey Mudd College, Claremont, California 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BE.%2BGlater%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glater%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BE.%2BGlater%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Elizabeth.glater@pomona.edu" class="" data-icon-position="" data-hide-link-title="0">Elizabeth.glater@pomona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew V. Rockman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and Center for Genomics and Systems Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BV.%2BRockman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rockman%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BV.%2BRockman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cornelia I. Bargmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Howard Hughes Medical Institute, Laboratory of Neural Circuits and Behavior, The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cornelia%2BI.%2BBargmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bargmann%20CI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACornelia%2BI.%2BBargmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/2/265" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/2/265&amp;title=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/2/265&amp;title=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/2/265&amp;title=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/2/265&amp;text=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/2/265&amp;title=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/2/265&amp;t=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/2/265&amp;title=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/2/265&amp;title=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459640/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459640" id="node459640--3" data-pisa="ggg;4/2/265" data-pisa-master="ggg;g3.113.008649" data-apath="/ggg/4/2/265.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multigenic Natural Variation Underlies <em>Caenorhabditis elegans</em> Olfactory Preference for the Bacterial Pathogen <em>Serratia marcescens</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elizabeth E. Glater</span>, <span class="highwire-citation-author" data-delta="1">Matthew V. Rockman</span> and <span class="highwire-citation-author" data-delta="2">Cornelia I. Bargmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">265-276;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008649 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459640--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth E. Glater </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Harvey Mudd College, Claremont, California 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BE.%2BGlater%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glater%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BE.%2BGlater%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Elizabeth.glater@pomona.edu" class="" data-icon-position="" data-hide-link-title="0">Elizabeth.glater@pomona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew V. Rockman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and Center for Genomics and Systems Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BV.%2BRockman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rockman%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BV.%2BRockman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cornelia I. Bargmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Howard Hughes Medical Institute, Laboratory of Neural Circuits and Behavior, The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cornelia%2BI.%2BBargmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bargmann%20CI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACornelia%2BI.%2BBargmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=265&amp;rft.epage=276&amp;rft.atitle=Multigenic%20Natural%20Variation%20Underlies%20Caenorhabditis%20elegans%20Olfactory%20Preference%20for%20the%20Bacterial%20Pathogen%20Serratia%20marcescens&amp;rft.volume=4&amp;rft.issue=2&amp;rft.date=2014-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Glater%2C+Elizabeth+E.&amp;rft.au=Rockman%2C+Matthew+V.&amp;rft.au=Bargmann%2C+Cornelia+I."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459640/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459640/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459640/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459640/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459640/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459640/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459640/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459640/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459640/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459640/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459640/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459640/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/2/265" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/2/265.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/2/265.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/2/265.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/2/265.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
