<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/2/325" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/2/325.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/2/325.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/2/325.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/2/325/embed/mml-math-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/2/325.atom" />
<meta name="HW.pisa" content="ggg;4/2/325" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision" />
<meta name="DC.Identifier" content="10.1534/g3.113.009845" />
<meta name="DC.Date" content="2014-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Takeuchi et al. . This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Horizontal gene transfer (HGT) is a major factor in the evolution of prokaryotes. An intriguing question is whether HGT is maintained during evolution of prokaryotes owing to its adaptive value or is a byproduct of selection driven by other factors such as consumption of extracellular DNA (eDNA) as a nutrient. One hypothesis posits that HGT can restore genes inactivated by mutations and thereby prevent stochastic, irreversible deterioration of genomes in finite populations known as Muller’s ratchet. To examine this hypothesis, we developed a population genetic model of prokaryotes undergoing HGT via homologous recombination. Analysis of this model indicates that HGT can prevent the operation of Muller’s ratchet even when the source of transferred genes is eDNA that comes from dead cells and on average carries more deleterious mutations than the DNA of recipient live cells. Moreover, if HGT is sufficiently frequent and eDNA diffusion sufficiently rapid, a subdivided population is shown to be more resistant to Muller’s ratchet than an undivided population of an equal overall size. Thus, to maintain genomic information in the face of Muller’s ratchet, it is more advantageous to partition individuals into multiple subpopulations and let them “cross-reference” each other’s genetic information through HGT than to collect all individuals in one population and thereby maximize the efficacy of natural selection. Taken together, the results suggest that HGT could be an important condition for the long-term maintenance of genomic information in prokaryotes through the prevention of Muller’s ratchet." />
<meta name="DC.Contributor" content="Nobuto Takeuchi" />
<meta name="DC.Contributor" content="Kunihiko Kaneko" />
<meta name="DC.Contributor" content="Eugene V. Koonin" />
<meta name="article:published_time" content="2014-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Horizontal gene transfer (HGT) is a major factor in the evolution of prokaryotes. An intriguing question is whether HGT is maintained during evolution of prokaryotes owing to its adaptive value or is a byproduct of selection driven by other factors such as consumption of extracellular DNA (eDNA) as a nutrient. One hypothesis posits that HGT can restore genes inactivated by mutations and thereby prevent stochastic, irreversible deterioration of genomes in finite populations known as Muller’s ratchet. To examine this hypothesis, we developed a population genetic model of prokaryotes undergoing HGT via homologous recombination. Analysis of this model indicates that HGT can prevent the operation of Muller’s ratchet even when the source of transferred genes is eDNA that comes from dead cells and on average carries more deleterious mutations than the DNA of recipient live cells. Moreover, if HGT is sufficiently frequent and eDNA diffusion sufficiently rapid, a subdivided population is shown to be more resistant to Muller’s ratchet than an undivided population of an equal overall size. Thus, to maintain genomic information in the face of Muller’s ratchet, it is more advantageous to partition individuals into multiple subpopulations and let them “cross-reference” each other’s genetic information through HGT than to collect all individuals in one population and thereby maximize the efficacy of natural selection. Taken together, the results suggest that HGT could be an important condition for the long-term maintenance of genomic information in prokaryotes through the prevention of Muller’s ratchet.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/02/01" />
<meta name="citation_mjid" content="ggg;4/2/325" />
<meta name="citation_id" content="4/2/325" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/2/325" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/2/325.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/2/325.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/2/325.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009845" />
<meta name="citation_pmid" content="24347631" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="325" />
<meta name="citation_lastpage" content="339" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nobuto Takeuchi" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894" />
<meta name="citation_author_institution" content="Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan" />
<meta name="citation_author_email" content="takeuchi.nobuto@gmail.com" />
<meta name="citation_author" content="Kunihiko Kaneko" />
<meta name="citation_author_institution" content="Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan" />
<meta name="citation_author" content="Eugene V. Koonin" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894" />
<meta name="citation_reference" content="citation_journal_title=Critical reviews in microbiology;citation_journal_abbrev=Crit Rev Microbiol;citation_author=A. Aertsen;citation_author=C. W. Michiels;citation_title=Stress and how bacteria cope with death and survival.;citation_pages=263-273;citation_volume=30;citation_year=2004;citation_issue=4;citation_pmid=15646400;citation_doi=10.1080/10408410490884757" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. Aravind;citation_author=R. L. Tatusov;citation_author=Y. I. Wolf;citation_author=D. R. Walker;citation_author=E. V. Koonin;citation_title=Evidence for massive gene exchange between archaeal and bacterial hyperthermophiles.;citation_pages=442-444;citation_volume=14;citation_year=1998;citation_issue=11;citation_pmid=9825671;citation_doi=10.1016/S0168-9525(98)01553-4" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=G. Bell;citation_title=Recombination and the immortality of the germ line.;citation_pages=67-82;citation_volume=1;citation_year=1988;citation_doi=10.1046/j.1420-9101.1988.1010067.x" />
<meta name="citation_reference" content="citation_title=The Evolution of Sex;citation_title=Is meiotic recombination an adaptation for repairing DNA, producing genetic variation, or both?;citation_author=H. Bernstein;citation_author=F. A. Hopf;citation_author=R. E. Michod;citation_pages=139-160;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=J. Antimicrob. Chemother.;citation_author=M. T. Blahna;citation_author=C. A. Zalewski;citation_author=J. Reuer;citation_author=G. Kahlmeter;citation_author=B. Foxman;citation_title=The role of horizontal gene transfer in the spread of trimethoprim-sulfamethoxazole resistance among uropathogenic Escherichia coli in Europe and Canada.;citation_volume=57;citation_year=2006;citation_pmid=16464890;citation_doi=10.1093/jac/dkl020" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=M. Brigulla;citation_author=W. Wackernagel;citation_title=Molecular aspects of gene transfer and foreign DNA acquisition in prokaryotes with regard to safety issues.;citation_pages=1027-1041;citation_volume=86;citation_year=2010;citation_issue=4;citation_pmid=20191269;citation_doi=10.1007/s00253-010-2489-3" />
<meta name="citation_reference" content="citation_journal_abbrev=Archaea;citation_author=C. Brochier-Armanet;citation_author=P. Forterre;citation_title=Widespread distribution of archaeal reverse gyrase in thermophilic bacteria suggests a complex history of vertical inheritance and lateral gene transfers.;citation_pages=83-93;citation_volume=2;citation_year=2007;citation_issue=2;citation_pmid=17350929" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Burke;citation_author=P. Steinberg;citation_author=D. Rusch;citation_author=S. Kjelleberg;citation_author=T. Thomas;citation_title=Bacterial community assembly based on functional genes rather than species.;citation_volume=108;citation_year=2011;citation_pmid=21825123;citation_doi=10.1073/pnas.1101591108" />
<meta name="citation_reference" content="citation_journal_title=Soil Biol. Biochem.;citation_author=M. T. Ceccherini;citation_author=J. Ascher;citation_author=G. Pietramellara;citation_author=T. M. Vogel;citation_author=P. Nannipieri;citation_title=Vertical advection of extracellular DNA by water capillarity in soil columns.;citation_pages=158-163;citation_volume=39;citation_year=2007;citation_doi=10.1016/j.soilbio.2006.07.006" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=Rapid fixation of deleterious alleles can be caused by Muller's ratchet.;citation_pages=63-73;citation_volume=70;citation_year=1997;citation_issue=1;citation_pmid=9369098;citation_doi=10.1017/S0016672397002899" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=D. Charlesworth;citation_author=M. T. Morgan;citation_author=B. Charlesworth;citation_title=Mutation accumulation in finite outbreeding and inbreeding populations.;citation_pages=39-56;citation_volume=61;citation_year=1993;citation_doi=10.1017/S0016672300031086" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=J. P. Claverys;citation_author=M. Prudhomme;citation_author=B. Martin;citation_title=Induction of competence regulons as a general response to stress in gram-positive bacteria.;citation_pages=451-475;citation_volume=60;citation_year=2006;citation_pmid=16771651;citation_doi=10.1146/annurev.micro.60.080805.142139" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=J. P. Claverys;citation_author=B. Martin;citation_author=P. Polard;citation_title=The genetic transformation machinery: composition, localization, and mechanism.;citation_volume=33;citation_year=2009;citation_pmid=19228200;citation_doi=10.1111/j.1574-6976.2009.00164.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Combadao;citation_author=P. R. Campos;citation_author=F. Dionisio;citation_author=I. Gordo;citation_title=Small-world networks decrease the speed of Muller's ratchet.;citation_pages=7-18;citation_volume=89;citation_year=2007;citation_issue=1;citation_pmid=17517155;citation_doi=10.1017/S0016672307008658" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. F. Cooper;citation_title=Recombination speeds adaptation by reducing competition between beneficial mutations in populations of Escherichia coli.;citation_pages=e225-e225;citation_volume=5;citation_year=2007;citation_issue=9;citation_pmid=17713986;citation_doi=10.1371/journal.pbio.0050225" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Costechareyre;citation_author=F. Bertolla;citation_author=X. Nesme;citation_title=Homologous recombination in Agrobacterium: potential implications for the genomic species concept in bacteria.;citation_volume=26;citation_year=2009;citation_pmid=18936442;citation_doi=10.1093/molbev/msn236" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Dagan;citation_author=Y. Artzy-Randrup;citation_author=W. Martin;citation_title=Modular networks and cumulative impact of lateral transfer in prokaryote genome evolution.;citation_volume=105;citation_year=2008;citation_pmid=18632554;citation_doi=10.1073/pnas.0800679105" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Dechesne;citation_author=G. Wang;citation_author=G. Gulez;citation_author=D. Or;citation_author=B. F. Smets;citation_title=Hydration-controlled bacterial motility and dispersal on surfaces.;citation_volume=107;citation_year=2010;citation_pmid=20660312;citation_doi=10.1073/pnas.1008392107" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=D. Dubnau;citation_title=DNA UPTAKE IN BACTERIA;citation_pages=217-244;citation_volume=53;citation_year=1999;citation_issue=1;citation_pmid=10547691;citation_doi=10.1146/annurev.micro.53.1.217" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=D. J. Engelmoer;citation_author=D. E. Rozen;citation_title=Competence increases survival during stress in Streptococcus pneumoniae.;citation_pages=3475-3485;citation_volume=65;citation_year=2011;citation_issue=12;citation_pmid=22133219;citation_doi=10.1111/j.1558-5646.2011.01402.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Eppley;citation_author=G. W. Tyson;citation_author=W. M. Getz;citation_author=J. F. Banfield;citation_title=Genetic exchange across a species boundary in the archaeal genus ferroplasma.;citation_volume=177;citation_year=2007;citation_pmid=17603112;citation_doi=10.1534/genetics.107.072892" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. J. Feil;citation_author=E. C. Holmes;citation_author=D. E. Bessen;citation_author=M. S. Chan;citation_author=N. P. Day;citation_title=Recombination within natural populations of pathogenic bacteria: Short-term empirical estimates and long-term phylogenetic consequences;citation_pages=182-187;citation_volume=98;citation_year=2001;citation_issue=1;citation_pmid=11136255;citation_doi=10.1073/pnas.98.1.182" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. J. Feil;citation_author=J. M. Smith;citation_author=M. C. Enright;citation_author=B. G. Spratt;citation_title=Estimating Recombinational Parameters in Streptococcus pneumoniae From Multilocus Sequence Typing Data;citation_pages=1439-1450;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10747043" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. W. Feldman;citation_author=F. B. Christiansen;citation_author=L. D. Brooks;citation_title=Evolution of recombination in a constant environment;citation_pages=4838-4841;citation_volume=77;citation_year=1980;citation_issue=8;citation_pmid=16592864;citation_doi=10.1073/pnas.77.8.4838" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Biol. Fertil. Soils;citation_author=R. C. Foster;citation_title=Microenvironments of soil microorganisms.;citation_pages=189-203;citation_volume=6;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Ecol.;citation_author=E. Gallori;citation_author=M. Bazzicalupo;citation_author=L. Dal Canto;citation_author=R. Fani;citation_author=P. Nannipieri;citation_title=Transformation of Bacillus subtilis by DNA bound on clay in non-sterile soil.;citation_volume=15;citation_year=1994;citation_doi=10.1111/j.1574-6941.1994.tb00236.x" />
<meta name="citation_reference" content="citation_journal_title=Infection, genetics and evolution :  journal of molecular epidemiology and evolutionary genetics in infectious diseases;citation_journal_abbrev=Infect Genet Evol;citation_author=N. Garcia-Russell;citation_author=B. Elrod;citation_author=K. Dominguez;citation_title=Stress-induced prophage DNA replication in Salmonella enterica serovar Typhimurium.;citation_pages=889-895;citation_volume=9;citation_year=2009;citation_issue=5;citation_pmid=19501196;citation_doi=10.1016/j.meegid.2009.05.017" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. P. Gogarten;citation_author=W. F. Doolittle;citation_author=J. G. Lawrence;citation_title=Prokaryotic Evolution in Light of Gene Transfer;citation_pages=2226-2238;citation_volume=19;citation_year=2002;citation_issue=12;citation_pmid=12446813;citation_doi=10.1093/oxfordjournals.molbev.a004046" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Goyal;citation_author=D. J. Balick;citation_author=E. R. Jerison;citation_author=R. A. Neher;citation_author=B. I. Shraiman;citation_title=Dynamic mutation-selection balance as an evolutionary attractor.;citation_volume=191;citation_year=2012;citation_pmid=22661327;citation_doi=10.1534/genetics.112.141291" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Ecol.;citation_author=G. L. Grundmann;citation_title=Spatial scales of soil bacterial diversity--the size of a clone.;citation_volume=48;citation_year=2004;citation_pmid=19712395;citation_doi=10.1016/j.femsec.2004.01.010" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=G. L. Grundmann;citation_author=P. Normand;citation_title=Microscale Diversity of the Genus Nitrobacter in Soil on the Basis of Analysis of Genes Encoding rRNA;citation_pages=4543-4546;citation_volume=66;citation_year=2000;citation_issue=10;citation_pmid=11010914;citation_doi=10.1128/AEM.66.10.4543-4546.2000" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. Haigh;citation_title=The accumulation of deleterious genes in a population--Muller's Ratchet.;citation_pages=251-267;citation_volume=14;citation_year=1978;citation_issue=2;citation_pmid=746491;citation_doi=10.1016/0040-5809(78)90027-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Higgins;citation_author=M. Lynch;citation_title=Metapopulation extinction caused by mutation accumulation.;citation_volume=98;citation_year=2001;citation_pmid=11226343;citation_doi=10.1073/pnas.031358898" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=N. Hulter;citation_author=W. Wackernagel;citation_title=Double illegitimate recombination events integrate DNA segments through two different mechanisms during natural transformation of Acinetobacter baylyi.;citation_pages=984-995;citation_volume=67;citation_year=2008;citation_issue=5;citation_pmid=18194157;citation_doi=10.1111/j.1365-2958.2007.06096.x" />
<meta name="citation_reference" content="citation_journal_title=Research in Microbiology (Paris);citation_journal_abbrev=Research in Microbiology (Paris);citation_author=O. Johnsborg;citation_author=V. Eldholm;citation_author=L. S. Havarstein;citation_title=Natural genetic transformation: prevalence, mechanisms and function.;citation_pages=767-778;citation_volume=158;citation_year=2007;citation_issue=10;citation_pmid=17997281;citation_doi=10.1016/j.resmic.2007.09.004" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=C. Johnston;citation_author=B. Martin;citation_author=C. Granadel;citation_author=P. Polard;citation_author=J. P. Claverys;citation_title=Programmed protection of foreign DNA from restriction allows pathogenicity island exchange during pneumococcal transformation.;citation_pages=e1003178;citation_volume=9;citation_year=2013;citation_pmid=23459610;citation_doi=10.1371/journal.ppat.1003178" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. D. Keightley;citation_author=S. P. Otto;citation_title=Interference among deleterious mutations favours sex and recombination in finite populations.;citation_pages=89-92;citation_volume=443;citation_year=2006;citation_issue=7107;citation_pmid=16957730;citation_doi=10.1038/nature05049" />
<meta name="citation_reference" content="citation_journal_title=Food and chemical toxicology :  an international journal published for the British Industrial Biological Research Association;citation_journal_abbrev=Food Chem Toxicol;citation_author=B. G. Kelly;citation_author=A. Vespermann;citation_author=D. J. Bolton;citation_title=Horizontal gene transfer of virulence determinants in selected bacterial foodborne pathogens.;citation_pages=969-977;citation_volume=47;citation_year=2009;citation_issue=5;citation_pmid=18420327;citation_doi=10.1016/j.fct.2008.02.007" />
<meta name="citation_reference" content="citation_title=Overstepping Biology (“Hamidashi Seibutsugaku”) (in Japanese);citation_author=S.,. Komatsu;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. S. Kondrashov;citation_title=Deleterious mutations and the evolution of sexual reproduction.;citation_pages=435-440;citation_volume=336;citation_year=1988;citation_issue=6198;citation_pmid=3057385;citation_doi=10.1038/336435a0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=A. S. Kondrashov;citation_title=Classification of Hypotheses on the Advantage of Amphimixis;citation_pages=372-387;citation_volume=84;citation_year=1993;citation_issue=5;citation_pmid=8409359" />
<meta name="citation_reference" content="citation_title=The Logic of Chance;citation_author=E. V. Koonin;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=E. V. Koonin;citation_author=K. S. Makarova;citation_author=L. Aravind;citation_title=HORIZONTAL GENE TRANSFER IN PROKARYOTES: Quantification and Classification;citation_pages=709-742;citation_volume=55;citation_year=2001;citation_issue=1;citation_pmid=11544372;citation_doi=10.1146/annurev.micro.55.1.709" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=A. S. Lang;citation_author=J. T. Beatty;citation_title=Importance of widespread gene transfer agent genes in alpha-proteobacteria.;citation_pages=54-62;citation_volume=15;citation_year=2007;citation_issue=2;citation_pmid=17184993;citation_doi=10.1016/j.tim.2006.12.001" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=A. S. Lang;citation_author=O. Zhaxybayeva;citation_author=J. T. Beatty;citation_title=Gene transfer agents: phage-like elements of genetic exchange.;citation_pages=472-482;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22683880;citation_doi=10.1038/nrmicro2802" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. Lerat;citation_author=V. Daubin;citation_author=H. Ochman;citation_author=N. A. Moran;citation_title=Evolutionary origins of genomic repertoires in bacteria.;citation_pages=e130-e130;citation_volume=3;citation_year=2005;citation_issue=5;citation_pmid=15799709;citation_doi=10.1371/journal.pbio.0030130" />
<meta name="citation_reference" content="citation_title=The Evolution of Sex;citation_title=The evolution of sex in bacteria;citation_author=B. R. Levin;citation_pages=194-211;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. R. Levin;citation_author=O. E. Cornejo;citation_title=The population and evolutionary dynamics of homologous gene recombination in bacterial populations.;citation_pages=e1000601-e1000601;citation_volume=5;citation_year=2009;citation_issue=8;citation_pmid=19680442;citation_doi=10.1371/journal.pgen.1000601" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. R. Levin;citation_author=V. Perrot;citation_author=N. Walker;citation_title=Compensatory Mutations, Antibiotic Resistance and the Population Genetics of Adaptive Evolution in Bacteria;citation_pages=985-997;citation_volume=154;citation_year=2000;citation_issue=3;citation_pmid=10757748" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=M. G. Lorenz;citation_author=W. Wackernagel;citation_title=Bacterial gene transfer by natural genetic transformation in the environment.;citation_pages=563-602;citation_volume=58;citation_year=1994;citation_issue=3;citation_pmid=7968924" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. Lynch;citation_author=R. Burger;citation_author=D. Butcher;citation_author=W. Gabriel;citation_title=The Mutational Meltdown in Asexual Populations;citation_pages=339-344;citation_volume=84;citation_year=1993;citation_issue=5;citation_pmid=8409355" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Majewski;citation_author=F. M. Cohan;citation_title=DNA Sequence Similarity Requirements for Interspecific Recombination in Bacillus;citation_pages=1525-1533;citation_volume=153;citation_year=1999;citation_issue=4;citation_pmid=10581263" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Maslov;citation_author=S. Krishna;citation_author=T. Y. Pang;citation_author=K. Sneppen;citation_title=Toolbox model of evolution of prokaryotic metabolic networks and their regulation.;citation_volume=106;citation_year=2009;citation_pmid=19482938;citation_doi=10.1073/pnas.0903206106" />
<meta name="citation_reference" content="citation_title=The Evolution of Sex;citation_author=J. Maynard Smith;citation_year=1978" />
<meta name="citation_reference" content="citation_title=Evolutionary Genetics;citation_author=J. Maynard Smith;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=J. P. McCutcheon;citation_author=N. A. Moran;citation_title=Extreme genome reduction in symbiotic bacteria.;citation_pages=13-26;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=22064560;citation_doi=10.1038/nrmicro2670" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. D. McDaniel;citation_author=E. Young;citation_author=J. Delaney;citation_author=F. Ruhnau;citation_author=K. B. Ritchie;citation_title=High Frequency of Horizontal Gene Transfer in the Oceans;citation_pages=50-50;citation_volume=330;citation_year=2010;citation_issue=6000;citation_pmid=20929803;citation_doi=10.1126/science.1192243" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=V. Merhej;citation_author=D. Raoult;citation_title=Rickettsial evolution in the light of comparative genomics.;citation_pages=379-405;citation_volume=86;citation_year=2011;citation_issue=2;citation_pmid=20716256;citation_doi=10.1111/j.1469-185X.2010.00151.x" />
<meta name="citation_reference" content="citation_title=The Evolution of Sex;citation_author=R. E. Michod;citation_author=B. R. Levin;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. A. Moran;citation_title=Accelerated evolution and Muller's rachet in endosymbiotic bacteria.;citation_pages=2873-2878;citation_volume=93;citation_year=1996;citation_issue=7;citation_pmid=8610134;citation_doi=10.1073/pnas.93.7.2873" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Y. Mulkidjanian;citation_author=E. V. Koonin;citation_author=K. S. Makarova;citation_author=S. L. Mekhedov;citation_author=A. Sorokin;citation_title=The cyanobacterial genome core and the origin of photosynthesis.;citation_volume=103;citation_year=2006;citation_pmid=16924101;citation_doi=10.1073/pnas.0605709103" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=H. J. Muller;citation_title=THE RELATION OF RECOMBINATION TO MUTATIONAL ADVANCE.;citation_pages=2-9;citation_volume=106;citation_year=1964;citation_pmid=14195748;citation_doi=10.1016/0027-5107(64)90047-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. A. Neher;citation_author=B. I. Shraiman;citation_title=Fluctuations of fitness distributions and the rate of Muller’s ratchet.;citation_volume=191;citation_year=2012;citation_pmid=22649084;citation_doi=10.1534/genetics.112.141325" />
<meta name="citation_reference" content="citation_title=Molecular Evolutionary Genetics;citation_author=M. Nei;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Environmental biosafety research;citation_journal_abbrev=Environ Biosafety Res;citation_author=K. M. Nielsen;citation_author=P. J. Johnsen;citation_author=D. Bensasson;citation_author=D. Daffonchio;citation_title=Release and persistence of extracellular DNA in the environment.;citation_pages=37-53;citation_volume=6;citation_year=2007;citation_issue=1-2;citation_pmid=17961479;citation_doi=10.1051/ebr:2007031" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=P. S. Novichkov;citation_author=Y. I. Wolf;citation_author=I. Dubchak;citation_author=E. V. Koonin;citation_title=Trends in prokaryotic evolution revealed by comparison of closely related bacterial and archaeal genomes.;citation_volume=191;citation_year=2009;citation_pmid=18978059;citation_doi=10.1128/JB.01237-08" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ochman;citation_author=J. G. Lawrence;citation_author=E. A. Groisman;citation_title=Lateral gene transfer and the nature of bacterial innovation.;citation_pages=299-304;citation_volume=405;citation_year=2000;citation_issue=6784;citation_pmid=10830951;citation_doi=10.1038/35012500" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. P. Otto;citation_author=T. Lenormand;citation_title=Resolving the paradox of sex and recombination.;citation_pages=252-261;citation_volume=3;citation_year=2002;citation_issue=4;citation_pmid=11967550;citation_doi=10.1038/nrg761" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Pál;citation_author=B. Papp;citation_author=M. J. Lercher;citation_title=Adaptive evolution of bacterial metabolic networks by horizontal gene transfer.;citation_pages=1372-1375;citation_volume=37;citation_year=2005;citation_issue=12;citation_pmid=16311593;citation_doi=10.1038/ng1686" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=P. Pamilo;citation_author=M. Nei;citation_author=W.-H. Li;citation_title=Accumulation of mutations in sexual and asexual populations.;citation_pages=135-146;citation_volume=49;citation_year=1987;citation_issue=2;citation_pmid=3596234;citation_doi=10.1017/S0016672300026938" />
<meta name="citation_reference" content="citation_journal_title=Biol. Fertil. Soils;citation_author=G. Pietramellara;citation_author=J. Ascher;citation_author=F. Borgogni;citation_author=M. T. Ceccherini;citation_author=G. Guerri;citation_title=Extracellular DNA in soil and sediment: fate and ecological relevance.;citation_pages=219-235;citation_volume=45;citation_year=2009;citation_doi=10.1007/s00374-008-0345-8" />
<meta name="citation_reference" content="citation_journal_title=Ecotoxicology and environmental safety;citation_journal_abbrev=Ecotoxicol Environ Saf;citation_author=J. Pote;citation_author=M. Teresa Ceccherini;citation_author=W. Rosselli;citation_author=W. Wildi;citation_author=P. Simonet;citation_title=Leaching and transformability of transgenic DNA in unsaturated soil columns.;citation_pages=67-72;citation_volume=73;citation_year=2010;citation_issue=1;citation_pmid=19828198;citation_doi=10.1016/j.ecoenv.2009.09.009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Prudhomme;citation_author=L. Attaiech;citation_author=G. Sanchez;citation_author=B. Martin;citation_author=J. P. Claverys;citation_title=Antibiotic Stress Induces Genetic Transformability in the Human Pathogen Streptococcus pneumoniae;citation_pages=89-92;citation_volume=313;citation_year=2006;citation_issue=5783;citation_pmid=16825569;citation_doi=10.1126/science.1127912" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. J. Redfield;citation_title=Evolution of bacterial transformation: is sex with dead cells ever better than no sex at all?;citation_pages=213-221;citation_volume=119;citation_year=1988;citation_issue=1;citation_pmid=3396864" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. J. Redfield;citation_title=Do bacteria have sex?;citation_pages=634-639;citation_volume=2;citation_year=2001;citation_issue=8;citation_pmid=11483988;citation_doi=10.1038/35084593" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. J. Redfield;citation_author=M. R. Schrag;citation_author=A. M. Dean;citation_title=The Evolution of Bacterial Transformation: Sex With Poor Relations;citation_pages=27-38;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9135998" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=N. Rius;citation_author=M. C. Fuste;citation_author=C. Guasp;citation_author=J. Lalucat;citation_author=J. G. Loren;citation_title=Clonal Population Structure of Pseudomonas stutzeri, a Species with Exceptional Genetic Diversity;citation_pages=736-744;citation_volume=183;citation_year=2001;citation_issue=2;citation_pmid=11133969;citation_doi=10.1128/JB.183.2.736-744.2001" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=I. M. Rouzine;citation_author=E. Brunet;citation_author=C. O. Wilke;citation_title=The traveling-wave approach to asexual evolution: Muller's ratchet and speed of adaptation.;citation_pages=24-46;citation_volume=73;citation_year=2008;citation_issue=1;citation_pmid=18023832;citation_doi=10.1016/j.tpb.2007.10.004" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. J. Shapiro;citation_author=J. Friedman;citation_author=O. X. Cordero;citation_author=S. P. Preheim;citation_author=S. C. Timberlake;citation_title=Population Genomics of Early Events in the Ecological Differentiation of Bacteria;citation_pages=48-51;citation_volume=336;citation_year=2012;citation_issue=6077;citation_pmid=22491847;citation_doi=10.1126/science.1218198" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=J. Sikorski;citation_author=S. Graupner;citation_author=M. G. Lorenz;citation_author=W. Wackernagel;citation_title=Natural genetic transformation of Pseudomonas stutzeri in a non-sterile soil;citation_pages=569-576;citation_volume=144;citation_year=1998;citation_issue=2;citation_pmid=9493393;citation_doi=10.1099/00221287-144-2-569" />
<meta name="citation_reference" content="citation_title=Gene Transfer in the Environment;citation_title=Gene transfer among bacteria in soil;citation_author=G. Stotzky;citation_pages=165-222;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=C. M. Thomas;citation_author=K. M. Nielsen;citation_title=Mechanisms of, and barriers to, horizontal gene transfer between bacteria.;citation_pages=711-721;citation_volume=3;citation_year=2005;citation_issue=9;citation_pmid=16138099;citation_doi=10.1038/nrmicro1234" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. J. Treangen;citation_author=E. P. Rocha;citation_title=Horizontal transfer, not duplication, drives the expansion of protein families in prokaryotes.;citation_pages=e1001284-e1001284;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21298028;citation_doi=10.1371/journal.pgen.1001284" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=J. Vogel;citation_author=P. Normand;citation_author=J. Thioulouse;citation_author=X. Nesme;citation_author=G. L. Grundmann;citation_title=Relationship between Spatial and Genetic Distance in Agrobacterium spp. in 1 Cubic Centimeter of Soil;citation_pages=1482-1487;citation_volume=69;citation_year=2003;citation_issue=3;citation_pmid=12620832;citation_doi=10.1128/AEM.69.3.1482-1487.2003" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=M. Vos;citation_title=Why do bacteria engage in homologous recombination?;citation_pages=226-232;citation_volume=17;citation_year=2009;citation_issue=6;citation_pmid=19464181;citation_doi=10.1016/j.tim.2009.03.001" />
<meta name="citation_reference" content="citation_journal_abbrev=ISME J;citation_author=M. Vos;citation_author=X. Didelot;citation_title=A comparison of homologous recombination rates in bacteria and archaea.;citation_pages=199-208;citation_volume=3;citation_year=2009;citation_issue=2;citation_pmid=18830278;citation_doi=10.1038/ismej.2008.93" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=M. Vos;citation_author=A. B. Wolf;citation_author=S. J. Jennings;citation_author=G. A. Kowalchuk;citation_title=Micro-scale determinants of bacterial diversity in soil.;citation_volume=37;citation_year=2013;citation_pmid=23550883;citation_doi=10.1111/1574-6976.12023" />
<meta name="citation_reference" content="citation_journal_title=Plant Soil;citation_author=J. Vries;citation_author=W. Wackernagel;citation_title=Microbial horizontal gene transfer and the DNA release from transgenic crop plants.;citation_pages=91-104;citation_volume=266;citation_year=2005;citation_doi=10.1007/s11104-005-4783-x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=V. M. Watt;citation_author=C. J. Ingles;citation_author=M. S. Urdea;citation_author=W. J. Rutter;citation_title=Homology requirements for recombination in Escherichia coli.;citation_pages=4768-4772;citation_volume=82;citation_year=1985;citation_issue=14;citation_pmid=3161076;citation_doi=10.1073/pnas.82.14.4768" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=D. M. Wolf;citation_author=V. V. Vazirani;citation_author=A. P. Arkin;citation_title=Diversity in times of adversity: probabilistic strategies in microbial survival games.;citation_pages=227-253;citation_volume=234;citation_year=2005;citation_issue=2;citation_pmid=15757681;citation_doi=10.1016/j.jtbi.2004.11.020" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. S. Wylie;citation_author=A. D. Trout;citation_author=D. A. Kessler;citation_author=H. Levine;citation_title=Optimal strategy for competence differentiation in bacteria.;citation_pages=e1001108;citation_volume=6;citation_year=2010;citation_pmid=20838595;citation_doi=10.1371/journal.pgen.1001108" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=K. Yahara;citation_author=M. Kawai;citation_author=Y. Furuta;citation_author=N. Takahashi;citation_author=N. Handa;citation_title=Genome-wide survey of mutual homologous recombination in a highly sexual bacterial species.;citation_volume=4;citation_year=2012;citation_pmid=22534164;citation_doi=10.1093/gbe/evs043" />
<meta name="citation_reference" content="citation_title=Advances in Agronomy;citation_title=Microbial distribution in soils: physics and scaling;citation_author=I. M. Young;citation_author=J. W. Crawford;citation_author=N. Nunan;citation_author=W. Otten;citation_author=A. Spiers;citation_pages=81-121;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Zafra;citation_author=M. Lamprecht-Grandio;citation_author=C. G. de Figueras;citation_author=J. E. Gonzalez-Pastor;citation_title=Extracellular DNA Release by Undomesticated Bacillus subtilis Is Regulated by Early Competence.;citation_pages=e48716-e48716;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23133654;citation_doi=10.1371/journal.pone.0048716" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
<meta name="twitter:description" content="Horizontal gene transfer (HGT) is a major factor in the evolution of prokaryotes. An intriguing question is whether HGT is maintained during evolution of prokaryotes owing to its adaptive value or is a byproduct of selection driven by other factors such as consumption of extracellular DNA (eDNA) as a nutrient. One hypothesis posits that HGT can restore genes inactivated by mutations and thereby prevent stochastic, irreversible deterioration of genomes in finite populations known as Muller’s ratchet. To examine this hypothesis, we developed a population genetic model of prokaryotes undergoing HGT via homologous recombination. Analysis of this model indicates that HGT can prevent the operation of Muller’s ratchet even when the source of transferred genes is eDNA that comes from dead cells and on average carries more deleterious mutations than the DNA of recipient live cells. Moreover, if HGT is sufficiently frequent and eDNA diffusion sufficiently rapid, a subdivided population is shown to be more resistant to Muller’s ratchet than an undivided population of an equal overall size. Thus, to maintain genomic information in the face of Muller’s ratchet, it is more advantageous to partition individuals into multiple subpopulations and let them “cross-reference” each other’s genetic information through HGT than to collect all individuals in one population and thereby maximize the efficacy of natural selection. Taken together, the results suggest that HGT could be an important condition for the long-term maintenance of genomic information in prokaryotes through the prevention of Muller’s ratchet." />
<meta name="og-title" property="og:title" content="Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/2/325" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Horizontal gene transfer (HGT) is a major factor in the evolution of prokaryotes. An intriguing question is whether HGT is maintained during evolution of prokaryotes owing to its adaptive value or is a byproduct of selection driven by other factors such as consumption of extracellular DNA (eDNA) as a nutrient. One hypothesis posits that HGT can restore genes inactivated by mutations and thereby prevent stochastic, irreversible deterioration of genomes in finite populations known as Muller’s ratchet. To examine this hypothesis, we developed a population genetic model of prokaryotes undergoing HGT via homologous recombination. Analysis of this model indicates that HGT can prevent the operation of Muller’s ratchet even when the source of transferred genes is eDNA that comes from dead cells and on average carries more deleterious mutations than the DNA of recipient live cells. Moreover, if HGT is sufficiently frequent and eDNA diffusion sufficiently rapid, a subdivided population is shown to be more resistant to Muller’s ratchet than an undivided population of an equal overall size. Thus, to maintain genomic information in the face of Muller’s ratchet, it is more advantageous to partition individuals into multiple subpopulations and let them “cross-reference” each other’s genetic information through HGT than to collect all individuals in one population and thereby maximize the efficacy of natural selection. Taken together, the results suggest that HGT could be an important condition for the long-term maintenance of genomic information in prokaryotes through the prevention of Muller’s ratchet." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
    <title>Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FS7_iA94mkEBkUUZH7jR--NWCMY6otWe1J2MJbqIl4Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459629","apath":"\/ggg\/4\/2\/325.atom","pisa":"ggg;4\/2\/325","ac":{"\/ggg\/4\/2\/325.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/2\/325.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/2\/325"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/2\/325"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/2\/325.supplemental":true,"\/content\/4\/2\/325":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223YNruwNMhh9UmGcrqoiB687tcubYcb14mIWVUsrblbFsQlm4T8Pl6MegRRZGSnqlSgSCWuR1gFd+zw\\\\\\\/Ue0XJUQ6z9iC+vp4kgu0kdOn8KELH9jDsHKFUxpUbUqc3L+N8nPJiEiQ0AyUcTAH08wFK83Y6BVpNOpN4q\\\\\\\/CKM4Dpinx9Hemt7G2s0mUVrmAVs2\\\\\\\/NowYXfFZ+CRoe+TTBCuPIOi6D4zx25FBiP6x17Am62i2UpSDXHeywgLxKl6iPTz49q5BiF2cuSN0KushKy4CMM5GxVNPsiIxFb3dRKe5H5JA1s8YZC3URvfm6JC2ap+Zqh4NEg8oUAqk4tZ2IbIMlBU0LdviGPumdNOAxiqaMYStrhD27\\\\\\\/+WA3+l0tD4TgOd4bz93FF7Vos9tdMfPzXNYFvSFh4y6rXHPMz6F84+hwoBvYg7DfjEbYc+xfHyw55eeyHblAf3M4g5MlvAjBBe+UqMK+IgUGrYRYu9VIWTsO0vnwO2fmbgshWnMOzzre9zSiZLVfm55TsaSGeguklkPDljMrd2boyCn2y+e\\\\\\\/+QXuj8aJkJBDwsMXYrNN5Eb3P+Bp57WEDhg46AtqvZzqy4aNC0805PU+CPSFjxu\\\\\\\/tAGGu+OG0JShZo7MmFKGrvCUjsLQHFsqE+d3DW6TQZp8CxT5BdH85iv\\\\\\\/9RDi7NdSFmU2mito3aKyd6A4oT04iybkOkd4nLQIJbPQSkoWLm8lD+Xp0iEfSQRnsqeyWPpmlDR2GWdosrQBrKn54mBVc1AeLgY9K62VB3QYAG\\\\\\\/pRPecfPoe6NPaIPwhshBM05cTQnPbfAqn1W4A0R7tNfq+MIDSYw9Z8ZWVLl9dQvQe2F56mv\\\\\\\/mFCJ5WXNw13LKCcaqZRytvtQPN5TnuRK8hgJKYFCgj5th2qdVmc7HVVy01C2tWZQAcOr91qszKf5how0rtOPn\\\\\\\/VLm4xkSUylYnR6p9O\\\\\\\/awqSriYQ2dwymhxDe+9Jwmt7zXEp9NzMcMOkVMJCl9ah\\\\\\\/j3cxy3izLz6mCnVFdZ3tAUo6VA8S3ak1a0\\\\\\\/oley1XsLeBoYUWTDN\\\\\\\/04UNlzSG4I25xf+7YuxzeRelzHcMc8ReGYHI6A37FKjw6MM+dn5blrNxPExKYSiu045Nid2dx6BS88FoRQVgR8xeqPwOjHJKuOqlgmAecPD+avf0IQpII4niNrXJpSXR5w3g34Cmi\\\\\\\/fuFCz4Mf+4JyzkmAuvM6mdxHdxUFk0J2Z\\\\\\\/2nsnst\\\\\\\/PTaWoiIf6QE+iOTrwE2sJJ1fpjpUHSSMNm09cD\\\\\\\/6875gpIR\\\u0022,\\\u0022iv\\\u0022:\\\u0022HJB+ZJjpG0YgQxcaPZ8Cog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002260d2b3308cb499b65c5bd2d8aa7ea54b3c20469e38530a256265433089587239\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nxew5rlbfBXVNHG6TApNAV0a+aysOkI4XUJ4rAkGImmrk15UyaKOWLOQG08gJN7dORHYogWyZHd8z2ShpNo1+ox+agAxx9d2dNbdcOVyvy\\\\\\\/arQNu+RMDdptLvWrCJT2Of0Jc+9QFWhCYL2ZiQ1IyZ3zK0zR3BtQ9fFkdOtgn1Dcq9\\\\\\\/qWrJOUGMiVqaUAHoGTYWIp45EWw3jzeD07GN0ah4lv2B1abWhlPsu0jh6cL\\\\\\\/8XkJRiYEKmPsPLQIgQdJ0l8B6P0u5sF5s7LKjUUj+eRzR02rH+bU65QBRoj5G\\\\\\\/3VpL7OmX8JZd8\\\\\\\/BkZHqctGsjaKbfUggCxGrek4s91+9ox+24I1aR\\\\\\\/vLxNsyKsQ6BwM\\\\\\\/xj+9ekblnvBfOiLop7NHHC1W+tD+cvmY2yvcprKJxar6HegpVpwela3I4JnRKH3ESCAf0Fs9TfInaI+XGx4T+sDQsc\\\\\\\/wj9+jACOTzjuc1J1Ar6JpYjIGVCXvj9cV977Ek2x7ZEpWh7AlHLTgOpOFvLcn0M8O04aJz\\\\\\\/IT6UuXT+BXmt01cKGFVeiW1L8tIRfF94vkvrnKv3eWVvD\\\\\\\/KMAgX6dG3XoJLiEaPDCvLSa9YeEoDU1TVxnO778bM93eEeqLP+MBxIxdhbsFzWT7lfmMsVUJzwL3PHB9LXSNYEexSXqFidT3s8R30Pgib9cYSgi8f+gqC5WfkL5mP4b0QuwCaKqmyz2pYh3T2SB6ODN\\\\\\\/YCzfcW77zg3RJhvTQcNC\\\\\\\/CuTF4mG\\\\\\\/30tgx+cme2nx\\\\\\\/7V2nNajvLcNIooKORfGAAyXfMRPIBuahGvyPwYwlp0AOQXDN8wbAWNyVLseECrFShGwMm3G5SoCxlBzIJ3ZFrV\\\\\\\/DP\\\\\\\/xoC3vsYujgB15Du8CjmWozfSU6AjQpuCBV3FzLPUWzQdeM7Vb0FoT\\\\\\\/71NKCpbiVW4X5pyNWpmp+aaSexM8GDKQOvBS5bQoIqkzZmjXxgOhLrgxVugbUfrBDJt47BV2IRrx+d\\\\\\\/NxC6dv+xpZp8S1BA4xPKJMBuVakgamPGAdEHov2HLubiw\\\\\\\/orfWN9UECdiQZ33T5X8FbEOlvKo7swA0QpGJSQyln22116UgNG4Xuz8gfdMpLaat8BgDvJqrEGAgaviG23jnVv2cKQErwljIo4C1+MUKTxj0wly5sVMuldoytZ7YfL9gXbrttdcR5HBSkK6zdDMUENzT0yVHm2v9LKvJk0jepR\\\\\\\/Q2fH+Yo0OPWR3bMSjAQWDWqE+O3KgElsGGyAb3qCzVls3Uf3YzkD+kJqfLRHciz9jCFJo7jGVBtFv3Id+n8dAFvaeGXqz4FzBuENylE4nMT9uers3U5ungxqDDdhV73tEqvcWB5NivtbcDibzK4sr6S5Fp7\\\\\\\/8unqzFHufDPdmIhD+xOdiGMXoZ2RuCNORQ1ULNQOOMufUM0xPP4\\\\\\\/2t1Kbyr\\\\\\\/BgCpC1kBRFod3KzQTSCi5eJ+Jf2e5klqJr+hvFuttMkPJxCeBP8YVsQm7dsQlW606BaHBtPTsix\\\\\\\/PPskm3UNFHHnG6UY\\\\\\\/xeVqY0EwTsB3eT\\\\\\\/bakN2UXt+utS6wVudJ0xX\\\\\\\/2UmEiB0BJXJBhrPaNinWyMbOS14nJJQkSslfyKudVGGp200nHtAb6o5WTh+6gy8m6Lq3ZSJExbxEP\\\\\\\/LQfMH51I0u9BZbaVPs90j6hIvogMLW9NHakcZmLNiR5lKuaWZcTACBufoTmiln5I3pCva2oT5wByGgstlcZ18Dliv+TtK2K1NJdna3vBC2XMC6ojYW13jGsCHuVlMoOhOEhDsfA3YFAKS2hnVfGcm5KWM8C4topwS0bnLjlP28NyA4Ytk0ZoNqMPgpgwxtOeEcdbF\\\\\\\/gaz1On7T6ur5ul8iCdtVRB0htdwvE+H9U4L83Az9uVz7XK1iLY7m1M8+ttOPf6W5shM1qdZIroOsEP\\\\\\\/5IXcxH18wgPsvdWVjMRQDiaxGpYv4TGIiyfRRQ71TsvjmZiVhHEdu0TrBCul7OBFzAY+6mZbN3sw94xV+vplrN9DUc+htTCCqUGXkxEUXhZzQYyHHuW8v+lQR987A3fWNP9TQxdGRslizFXrornS8CCj0wbFsSMgjXCxNo98LZ5jTh7oe7WgsVxim4DcdNh\\\\\\\/U1PqQZFlY3JksBgdR\\\\\\\/oqkHMdQKq+YjzV2t3eBE7+Dnlg\\\\\\\/sHACLA2xOdEz0fVuoFmShEt2tY7gIuco2iu6DenRYrzH2ZxzSwOBWAlNBA2dXbN7kSM7hCSUDfyDWxSOgyjTLWrYb3QoqZoWJDwrIE3tZS8J+r5M4IUurvSML0HJKZVQrTyQ6sTK6aZ9IuRpDk+NzHSjIuOpeyB9JajoD+0sCPK7tWlWq5j7dYzmtQF1dTJ792L8lSZZQWLd4nUfDmD4oUJ2c+5oR6vs3Fv2uozvSAyeBaLgbWFIbfuoqYwEYFMLCG32bUXZDLwmEw4X8VUnkqW2U9BzNhIQp3Jf2vOseWOGszKJv+ELnFjz6JM5r89W7Vx9MjjemipKxsUTZ97KFf6aUMF5ZBoE9jz0NQctbxZD6+caMSAT0NGmSfnNwKHNJBe7bmRdv3VC\\\\\\\/1a8BnzhbiUDiwX7xpf1mWjNIf3nwstzCu2ucUyPES3ADDrrvvzAGp2kPKEuWD+qFUqgqHwtqIlvXqeauNBG1YNjQylZte+YMksIwg2kefPasMcLG2VFmYXEUv\\\\\\\/PXE3xuToyqE78ASvCO4pFTVzN4oFmNC63tu9z\\\\\\\/8ulwQi5Wfa0\\\\\\\/+kQpyYnNvRyT7ZviE0m+122kcqbEjDx8mPFXvtYAz2+NCZO4FbZGbRpohcbM6AuulNRUHORhug0fYtp69fEDXwN61b2cEnxigKKU1v0bSlpgPz831LIlt+i7zrwTbCWs7oXq938wQ\\\\\\\/IUQEaTAPOmNGFKA4D+Jx0eiFN8u3Ltyw5urNInnUnviT2t4u3LZWUbWVj\\\\\\\/t3ar\\\\\\\/+NwvoEXm\\\\\\\/H6q6Bf8EbIfCCTWFBeWM++Iagc5xDGnI+x+r93KteqXiJa9+60BY8ULd0ZvxU\\\\\\\/TJz8pC9nSBIan4\\\\\\\/DA2dXEvt9AZpk0R43hxaUM8jDRN3fV6DZtuH\\\\\\\/rfM4jwSsyu4g+cYQnoH99J90nR4q8OsCiibPmhpXunorTyBU9qa2CexN8e9taf8ZU+mkWKaxCDzQXGUliJL3aAoYTVMXadcQGDPkQJ4tfcH\\\\\\\/wYB0WFoxPJl8e57Qw9VISsVQZhNFyy7OribrfjoyZPyWkbRfKvN654v8abS1aaa+49oUSjkILEeH4+\\\\\\\/9cDhZCQgtRP+xf17a4x9PVlgLtx2uKX33lWOBQybjENv\\\\\\\/u3isHiZ1A7wRGzWt52GNEEb6KQ8qF\\\\\\\/\\\\\\\/01SgrxxHLfCdFwt\\\\\\\/GQTo5k5U+JK7cM7qGZAIKCkyVSpppvdJbcHpgqAs92Biuam6jjMqaJ8mfw\\\\\\\/tA2I3I6NmjOntY4YKlFer6TovSfBEErGPYMCpKmWuyHyFvvqvI9fJIvRqWeInnLhv80cTJn6Kgip7CM7h45WD9o+IU1xjq3AmZZBqm7gpfk+62E235bNaadK1N7glHkD5+Al5hvHy7KOYJV3RUFT9nYrF2DiQ\\\\\\\/Zx+GB2Zz3Olf+qgAXOQJiW\\\\\\\/mZv+wMcXFWjEBIZ4CoLMNRnXM6KRT45jPtzKdHWqvZMY9cpHbOFYfW16q4yuFNrPsmVr2l4\\\\\\\/D\\\\\\\/jCVM8G0inq2VEz0wJGdGAbJ+Cavt5QQS\\\\\\\/U9ocCIRoE\\\\\\\/Z4amIzzr9QgwpkPLJWLuO4fn04Dy0hSzNqU5Z6QPjEGER1CX8k42Z9Z8sPIyIFuSRCcq46BUtLh7cOAqNnvJkxFi0IYBDTeD8u8qx3F\\\\\\\/si8nvz+gGGCReloM4DnmQy4r0J6C1HUcWCxZ46bTHVkSHvyplK7vhxT4NeALNEcSU5EspYD5b47L\\\\\\\/BnDm2LCe2yG0OiNiHPb\\\\\\\/7ICbix0X17KaY8jiVbfZa5e2ezAfy+rLGcgwCzVONESz3JfIWBOqegTTwgtE3JXoPpLefJFYE4ZDF6xCL2x+ERZhyHifOyfWx1j6saby4w0zR0BGrgJMGBnuCnsPoCqxebPxxXvkq6SD+LvVaVB+SUlgwpIgd8cUS12u+HJVPXW0jPsLaCB\\\\\\\/Y8pjQjM7iJZC4NmOxB4CghiaoiUQH9YYxLO6a7fILh2wKpqX5DV2pnUS633tGpevu6WNzdJIUbdL\\\\\\\/20Pm1inWqWPAJzDrra2qiD5wse2dsLvWFfCtDdz\\\\\\\/agoNKThywJZMyyfZV5va23+UkpAeyWy+ZPufeq51PdXLI2cANTiLctKvO7fTWfibu5Qf9Ncdx81lQ9YnQALdO7dNjYmdLPpW0XgAIdfMR6UZ+iT5XLWNP+9Et5k0XbXS5eUf0IYi1nUEg5bY+AaiztiX18bvSZ2QmSIOhnqRCoqoE3VmBLRstycbTh3nLFzJ3vtL7s4fNc75gyXDYcAjZE3DGHFJOMgGN4ZJXdL2hZSdndDHDsRrIy18m4kqL5HFaF5d\\\\\\\/\\\\\\\/aMOSBmBJArN+0kcPxPeL8IOImHyEZKVbwfFkOe4hxbr38AeyBGZTH2sTJveQjo6Dyf8ROLt7dhP3VAO2JdnnAaBp+KzKcbF3SqXLX4dcDoMTlSJ6lgwzGx6wYvZxdoDq8WIX3p0+v2CZtEarcb7LrGz0An0M\\\\\\\/8fp5+mn0R\\\\\\\/71AAtfV\\\\\\\/ZaIjvui1N5xV9WbyjajQscK1byPao6\\\\\\\/Fj1GPL1HT0ZwzOYKsy12YXVXOLaNJ91M097Z4F6YA98WAQ9kimZEufNWHC2QiXA1g+CnUXL9Q5Ahd0IhE\\\\\\\/kWU+Rxmsoa3fxh2LuCzQw579rSeNyjGvKvYsb0nU+7wEyBtOxc4snOg9Q3PruYtbANVtDZl6gCEWEKQIaJBrxXgNZTeN\\\\\\\/t\\\\\\\/qjoGu6w9L+Ylp+2krU1WhXz615jS1y1iXI0SYiEhWPdUyDadHgTzljH49SpqhtfCqD\\\\\\\/XVM7ykMMPj0OzSGaDmnJilBurH7wgLqEkkh2e5TrPA4xVKnFTn03qUL\\\\\\\/EYkcDrBr32eR62p1lggxez4WheV\\\\\\\/PL9wwLS6L1F2peYxfCMOWC4RgoJcvnaEnJ5A3hudOzgSl\\\\\\\/\\\\\\\/3yci4SkeDR\\\\\\\/SAmr4tPJ24sJS7WW9HrJxHvyVh7g9\\\\\\\/xBRgMTOh45XvzZzEcU0fYNAaixzt+XuwC2daUEkkS7IV67X9nwFoSpPvKiqHFsqrsyzZsJVlZe4s9NCD0REeZiupC2ekomZFqmU0oqBJl1Q6c6auMu25zJKWKfsGRvsE7mjyGuMGQPccNwlodeHAvYX6AaGaVCu7RVCvl2lHlHVx0AZr4EldXtasdI8h49Ehqw8G3pmrdmSvUQ46XV6fQvw1AUcE3iL6S7i8RL4OCnjVaQwrOPblfPPk8bUd6FPerOFIHNMrrSX1eMoceHfXc1njPOShQkoW1pB1w1xTq7RmGQ5hgDCFdV0PH\\\\\\\/Z+3i5kw\\\\\\\/m3Kapp1rYnPfqyZ44UmbWp\\\\\\\/VC\\\\\\\/hAKK7ZXhOnQhMvKAVQYKM4xGB77xgZFGrc81mMDHFonHkPjPYEL80KcTI5lyVFen0Pd+jV\\\\\\\/suHaBcNjDOtR5hapnW2NrGTuUfqpVyDp9x0a51OiairEf8PAEXiInqfl76KchfRADcg\\\\\\\/nkqBiqilfQAJk+zJ91sdlln3K10BN4xLw4r8NnDWcbZPaD0+4scvzgPOUPDR2ml6ehfK1M5aY3VuyIDNAmfCxQWMKJ2LvyMiZwEpm4pGfGY\\\\\\\/51MV5eljHFcTYeXCJylqAg58YwAIHqVKYL5qgu\\\\\\\/hGEWAvSq9nZLH2RCyNYJwUTnMuc0JF\\\\\\\/KXILSG58HL5ZlT+J633U71bIKsVRMEMdW10ZCZTeVsYL1VIarxBp7XCNUgzQVdbJP\\\\\\\/I\\\\\\\/5xSgjDmlTXAGgWHkA8El83r8ZUjyysnX5hurV00yl8Lz7aqXVGSscRgTrjFIOwoyFNQkgp26E4n\\\\\\\/x0ntctHUeqre4RTCsKJUd2BG6D4m3CVKKIVN6CF4F4p+U\\\\\\\/Qwq6UaBBUzPd6M29Dej3M5z+RYUZpduCsxz8rsBeunQBhCZndJ5WtTiQ8HgtLFzTMpYQA0j2322PaJhYA8kPUnGbXUOPkUUm6KJrZh+BkExK\\\\\\\/CNjvnV3gWzVjRKAkmyYuI1pZxeIYlYtKD5vMy5PmGlLfs621mGkzWMroLaM3ExTnMxzfR8BvKqB7C\\\\\\\/XnDKqSTN0nxNIdn3voe5XYWya3rE+cdo6\\\\\\\/p5nPSmUMvn3x25ehKtREEb6I46zud5ymwtilL3yYlyjewOqqJNGWcevn191Q\\\\\\\/yfE9GkPU6J18fG7yxHuIGfTFHkjY9z4W8M68ckMtZMilr\\\\\\\/YHS0v6PJfk3FBNInzzImro2RMSVrb0fDHINQLyaWSTPC69xnmwlwTKj5uRgbyp5fr7VGNrPRqaN8EGoxV2YTXFZj\\\\\\\/9Kg5jCKqpjfgYHSHzA4o3C4y3uewZywNg4huDDgbC6S\\\\\\\/zcsrJ+dSMFzoLwiPwqgzhS7SgW0a6ltgruzwMHzB409aOwxSf5zBJ5RO8ld0a90UHwrH+X0wcK8un\\\\\\\/D41SNRY9BmdXOjZvtz7nafl\\\\\\\/aIvEfamqyuE9TsEfwwpO2LojF6+7\\\\\\\/qthN4X+YM7dTv7koH+cfxFw73nDMheVIW4bGMow2cvHRlYVfUph4Nah4qb6BGwTQ\\\\\\\/SH1\\\\\\\/WbJojCifz0Agj\\\\\\\/jnx2CCqMvJLwacE\\\\\\\/XE2VTn57UpHsESI63q5uo9Ka3e38TqN4+FSh5wTASjswDzCr9R2Avdn5\\\\\\\/VlFSl6KGHorQ45QUR9Nb6cXnrPKO+mjpguzchtw6Sr2ZcrEVueqInzCzcFmD6b9ynKVGGuhHbCdBgJtVZEk678Yh+qZ4tx5TjiTY\\\\\\\/mkeTmVECH\\\\\\\/56yH2hJzvqoi7p+mZoJDYSShZRmU\\\\\\\/DxwEX5jH62swt\\\\\\\/oQZ+ZWx0qEWo9pyYxCvsY2iRWukBeZ40ByDMwY0PQ\\\\\\\/0fzf2IMPGJF9RxHyH4HuRPgKvUGEF0q4\\\\\\\/fu7SoxUgxsApmS3O3I3x6876EVP45swcDSaI5HJIdVGWhiT\\\\\\\/Np\\\\\\\/Jc92sQBs5jLwz7l\\\\\\\/oY5RUvxkRIGt2MpIkDF\\\\\\\/f99lhWbi3PPATIsz1DUEmvRhQGqE\\\\\\\/sFUWBAKjExdbbrfOG7f1C23\\\\\\\/\\\\\\\/0uh+qL\\\\\\\/q97nkUpK0u4ThMjdrJZvs5sp5EvdLdizVk8RAeSZY52Cr2N3DG1vnsUe85ZyoOK28vBCEQxfl8E+3eoT3Yt4W8YWhfdH2QEJGTROh5NMMMHQpc77fBUQDq0hQ9ApFZkoOa6bArCWsiErPLYeBWkJkjND46pXYYs5tL3leixI6WLSaZuL+CDvbp\\\\\\\/YGOU77V\\\\\\\/XbZfb\\\\\\\/I\\\\\\\/LTiWaMkTTV5D0NvqPgvhNYYAnZd0DxwPYohchFVPl+efTOlAjEvWbUzX1WyB\\\\\\\/LoTq4HaZoDMcm5LR4bsm7GvTpbJaj89zJhrToSwblKkaE+EDxrt1eJoC5\\\\\\\/oI3OlScUQ2i5jl1Ch8T+2Pkl\\\\\\\/JpZv249spywpBg3uG8G3XLThraEdnkPb0btWVvWNBpBHO55iW4O7DH\\\\\\\/usOD+ExSG42E4qxfM0HHIm23yRsiiYKgmvxy55E8V1YjbZrIMsTKZMgjz1GNz1bmQBXqrC4gK\\\\\\\/hyJLz1l\\\\\\\/2Q5jhoKR2oMGXWNgencjxANJxB5pEb\\\\\\\/dL0DjLwWz0bzorPJKq9WO0AlmeJ0TOyHOuCTNKbbM5S+7U7qoV+7geEvCoTMFsQqurtQ1reROfepFj06Aqucv9hZ\\\\\\\/XNUCckwO7oH056Hh9rRCX3D61yyPpRQ4ephE\\\\\\\/2gHP76MeGYh3fDDv+Y5pbP2Ae5\\\\\\\/9M\\\\\\\/MuTq9TRwccyTN4h6Joy75fDsubvKG4EJS0+53LjxuYckzHlOsL+WRl1BNr6Y7TfDXhzLa06MSLkV66yWVXw4kDKDlHBO8pCXrqJX9URN32Hl5R+16seZcTFmF64MSto7uoTXcVp0FF9h43zOtGjdMfuUHnCULgvfZ0vN9J6Fmz8qyymRiriZkDrQ5Dn071feXEn12uUzNl9Z88Cz1Q\\\\\\\/L2esQiqrXeLFkvWbpqD6yBdh4lElkX\\\\\\\/Ketl3IAD0caRnCIsW9w1NA947sT4ovPmsGrXyOOePcycRnNSxlMpkiCJQLTy8Y6sCTZqp8ciwx+KvJetCzQ6PR2pjsuFajqOdiU7mIUTHl7w2MHFtY7vh3qjyQCLV9AQcqTOal62Tpg04qGsbjqwKB25C098WbtJdbWYFssgfxOxxOLUCihavEg8FteEylZf5a8RrgziV2Bl9DQ35qnHGnsoo9rUpPe8mU9U\\\\\\\/0mDPtzz5KbZNEtHx4P+c70A3zldJPxk62yaEkCk7aOzVLY97e8Sjalkfl8KiHyf61lGdsbWwVVtHpd5COowozCHGQYYBbRPHtlOiNqcnFMZGt0wmPw04VpzAL97+CskkAXAfGGJNutiM8ewGO8tzlJzj8F+N+fp8\\\\\\\/qhVgWaXi71zfuLVMrVGKy1bulXHvwIJsUBMzjAyIWHGk+rsnu9sugc\\\\\\\/IFCAArhK1DN7jgdgnGg7SaTuZ\\\\\\\/tFzV6J+P9lDhxVmP4+NPLuqHPIsXycnPcgV3bpiiuCxAmNg04oA9dbZoNoSQ61EohDlfsjyNY8rbIOJiSN3P8efEBN4PBcvCmEitfBD9a2sxGwNYG+mg1cJ\\\\\\\/mJofv4w75bh5nm4ufIVjNo0JWf8MM2VCH2rQnFrOoagomrnMDhGcCpt7pjQXs0oUdRq6vgTsgIn2MmbA5qlY30bXJG8Ar0qqTns\\\\\\\/6meItgp21L4P7kJYPNH0z+GZdOgjHwTZwJuXgqubOzMTMYO8MIy3sMhGaNwMCqsv1lY+bYx\\\\\\\/FToZoGj3oAbv87zqyQU8qI3Hoq4S+zAb+tPkJYLtK\\\\\\\/RRgVNJd\\\\\\\/2X6dFPR4CU+7diWCoEnJuhNDTwt3HwhcIKV4IwUp1\\\\\\\/qgHtTnigo6fA11p6zimJvwZWdEF0rlJ4qG3NzS2CDUSWIL+BLCHlGSFg13XS5r7iGzY3Lb8\\\\\\\/Q2+V+u4eBy57sWbeHmG1YNcsiD792U9CyV+o7IjmA+bYqGuh2\\\\\\\/6gGjLd7ybcZ2tIV\\\\\\\/33IXICLO2jm\\\\\\\/c8P8ruFRmlpl+5\\\\\\\/+Vq4FZ5T1DJk6BqJlUX4d8aN8sTtv3qZgzPHDgGCUYhPF\\\\\\\/cwnuv0H6dff7bFC1MzhAM78yyegNwHpgAS9UT7l01stpv1\\\\\\\/hyMJzAqMrlosj0H5XyVWXugRPjwuoeR+9H0hV7XVPyxoksdoxShz63cf+lPe\\\\\\\/Y4pIgAQ+M\\\\\\\/CQLX01+yjLIpkW7BdNYjFhvecSoUXf1L+AD6PtPFPJQ1fxk6GYbHSIsENHpCsp\\\\\\\/xrEwpyT1f5n5GKCSPIdQjjyNvgPf4BpXYsaEcfYuwQQuJqYzagQ4O8h3H1lBAY4r30V0t3qjk4jzldugRetDWGfiC7Db+SJ9g8hqXk1mFicjMoK6lC8EqfFObwdcMPnsqRZxmsK\\\\\\\/LGgBjN2nlsXOAL3DN0OCqWp0Sq8yyyMRBVYu3NizVzysZuK8tdQVuXxNzTx8VZsnjLivN1gkfShtYj5U4fG9E1KXs8Yb9H8jCfQJ0x+TgiEayPGw9O242mL0WpvJYBzsIYafzAl3nVN7n67kaZvfm\\\\\\\/OwJ7fCH4pRAqcF\\\\\\\/lUD7msDuvs4jmUJPY2W2NO9aC+LLOXtm5Srr6OGOVrzbczUwV0O7ThqRiaMbG7Oz7QFoLftV8A7gi6LDsL2Y2qCXz5uj097R17kwhWrBscQVa3ip9y+dewvkArdOBj4Ib3Hdl7mpcWhmSBxx2Z2OVS3l4ZjHM1Rrvi8vhBvH873Sxa2X0QJ\\\\\\\/bAHtRHEtMVfemEEKvyOlsVPR0h2\\\\\\\/vXuvjZSL1VWTHnn0W30TC8vFp0wieuxObhAFBYMA9gd9pu9VbcPwV+aueOt0gp4eVgy40sdij0JNdUZ6ojdk0AdtNsGXODXIYUrDZoR7zDXZBYZOMcHWoJ+IwpdnQF2L8YJxcdM2tfPOoEbqLPNx23VGuFDdM2Ya\\\\\\\/lMRDTLsAQis7WwjdZ\\\\\\\/HgTqxqHRZcyL7LhkyFHvP2maFw9e3h05QQsukwM9+qvFiJsY9\\\\\\\/nEYk25cFILQ7XATgHbAQlv2aJfqrJxHTBkBYq8ThCEnF+CZGKOcWBv3Rn6OU06\\\\\\\/svY5rtht+6crwGQLeElD2bgF5nng\\\\\\\/tgM45SwCLCergzLxlSOmBVxmOORfkfBjRR3XojHkvj+hfLNuW0wW9\\\\\\\/X6\\\\\\\/cc0MQScl5BmWEUu+3JES2MtJvJeCl7Ct5ePIKtCQAH4Ox5tvpjZau4xsJ6oUS7x2j0\\\\\\\/N0kgg3b4xBEIdPwYJXcd0F0jIW55ZOgaBFSwnUHTUTOgzb+AwtTNbFhVtXGL6M6nxPARtHlABh\\\\\\\/3NeAu7cHYLoVA0X9fvhIB\\\\\\\/EBKKhLxQ5FvAkvdzGRZhEoODWgReC6CuQdtSr0fNaioJuJIgYg5UJsx5apJ9oa47dgL+Vnc6E6RHH0WKGeUvmUgY3X3YaW43iTMY5WRkl5CzHWv8KTkzMOg+EAzzHdFXA1rm60KbV6mw2tIwC1bzaxZ9MLDX0U1I2gr6BWku3Vr11WeKzVJ2Y2+D5K+ti7seMYTlaanT5v749mZOJxdIBNsJUSxpopozy62Kec3+W+Hj\\\\\\\/9VbU0byJub4KrhMy9au99cquJWSJMmIlfrDqeSm8mXPhZaP92f4lezqpO6zRZtJywVULXVaq7oLgAgOnuaf9P9+5UagluHF2f0tTkxMGUyVLfhH8smmZo0gfGTLFZT+G7HzVBqCZEVJ6dDj6CyxFhz1R0NATib1GiTlIwzMOfvXLQQ8g8dQ59V9NslT+NaWmRTUYche40882wTjV0K0bDzNHh88LirVEhTMwDnxcawT6Q7uly2LCBOTLVIpSPwY6G6lSy3aJ+DKF1Qzh79ipBG\\\\\\\/41ezm2d0fQSBhU2E00wT89lL4uEc05KyyWwFKsXkRg+Jweu50lmgMy7UFqi9l6MJJK0jaNjTY5IEZbISrN22\\\\\\\/qZYtxWJjKu9wrLDoQSBTHJx7XctCqF2KSxcc8cAkHBAGW4sySvp0shm47wz3iJt8uV+YPKvLG6VONbbdq5w5XrOWEyE6C0WMUTLTVPsKfyg\\\\\\\/ml8dX62cEz3deMLWk9Xz3D7NywA6ANdjW4i5MUGJWgGRMkQKkRnkrai7UN2immZ2AAkdjn9HPFKoDF+seIL+ZIglYUGLxtgvlpMOxvcHg18\\\\\\\/TFgjXY7c3oLhZJsDy\\\\\\\/zOJSmG\\\\\\\/rMCNj8oc3H6Ckj6XbTvUscD2drJJHacky7xufSZGoZEQ5\\\\\\\/4GPwbmCEEz2azOlliBUzooXOkva3kQrFMQg32dK9yMceTzCo6EA48WPKw+hRONmNZdUttTt9rCuiSytNl74BkURUWtBa4GANPWiTl8Dsc8cuA0w2bXw42sn8fKE1MDlQNvOOpvNdtLCO95ZKcrpQWkvCLqlhQs1MEo+YmLxUDaDikdXks+0dBsx0GmfKETh7hJI9YFHDRT8W9slgFDW6xBlzA\\\\\\\/krwyLx1q8w2Hya3y5bgEjey8ZkIP\\\\\\\/QjlocpIwmZ2rQvH1YLNmatb9sCoUA2C1hld4ATUBgAr8C3oSiqqosUsFROSZxgpJ7tQM+1rReejj5fO3jCC6pMMZhpCjBTZKs9wG7hVGjkeHWbgt25m2DXuVRhxJCBNwj1yphXcBSIWTHVoXkXz\\\\\\\/WnG37HPTsoKTF59vQS+a96MPzrMTF5DVb\\\\\\\/CcsyE+u39kHU8VSpKk2kLzhM4yj2kz9UCgyczHNcdkmWp5HhLLHJqRzxfRUuUc4PY6GKUEyFHpQ2SUiHPELSKFBT1TIXEnJx9YNHADQqRLZhF+GdiE8On2DtKT37yI4SXxIrN92vj9HDTVGHhHNufGgbG5XHp3S2jmxuK\\\\\\\/Pm52WrnlQyzJCnuZYjVgpOtZDCKNs96PPArjHrIJm4czXEeXJfxYYII+tF64WU3rSAcPa\\\\\\\/mibOgto+rhI0UdddeqFYh+cNq+bTdy47nZSszLqubT\\\\\\\/nMVIkabBYrila58fAor2ZpZvUdvKj\\\\\\\/28oBR9XSui1radtAoFmI\\\\\\\/bSXO5oGRPoU20cLFmlcmlJYDzH5TOduLTIMF+uDQU6o85FypCOisFeM1Hp9VjFiGsA53ZWhCzLFfwjOQ1SF3yrRSl6ffpCZRSS+x1xdM1jriuyrls5JPIS1LnwbZ67fI2w7\\\\\\\/9P9LEAs+xqzx1mHGdTJLz26nknx9ww2OmCeJPrKb2N0RTj82\\\\\\\/NLTGkR3ffpwfTrV5vHyy6PB6\\\\\\\/RpzIe2MEc2GbX6bvYMHJknKUKTG31Tqn\\\\\\\/yyKMWezF11jzHbOcgDGH7pYvoBhe7REiLHQG4rroCv9hb3NgDll9v7bPTTbcC8v86eYGuwnk8rMlGmb4ocq5OSyQufBuuaopQs8lagUiMg6jw8nBbP50N7HySySynLgBpZJUK4\\\\\\\/yc80Mb0zuyGFqejyIbC8hdud3fUB0e9NKwQw59K4WA0YsVI6eIskg04x2T2g51wSz04y36hbA\\\\\\\/JPeyBZ5hq4cu3HLqfQQqGXJrA5PN739\\\\\\\/C+behW\\\\\\\/HVLPSZbHEKAKY5aKMJNNRzpWM43odQEMTppwlkqnjTuG5pVSd8Il\\\\\\\/4U5\\\\\\\/ynG+DFCQFNwTbK8y7s1HyUMVPmIRveT92Rlkxsy+kmYEwo5limOsTLSWCmxnnvpjSzQBQJ0c+eBejobz3i1c1l+PEKdP7Nu5fjmg9j6UaA9VmcJokBPMsbHekIeig8LELJgcaWlWIejegOf83IrcNAewLuc+MwmdCCBZbSPoz3RuV8rOssqG6b5ced1AXEoConZOKWKAfUHzbmdDKg8TTQAmr6dgE4AncQTDBhm+15R2KuKtMLetN4\\\\\\\/wAo3mBYBC8A2ykeHob0f1HU6CxxCEjIcslqtqUeWF0dOcz1\\\\\\\/a+Dx5qYCkTUFAjrfnbsg16GWvc+RCn2i653P9KF7Uj33DQia6cef\\\\\\\/2yEY4EBAH+GdlC3CG9LLh955i5nwzQpY5g9flFy5973JmzJi\\\\\\\/xKxutEAQvnmIxNoFVAxoaajTL0ZWP8Cjq7BAL61ND\\\\\\\/RIOoosbqOwtomPi8+p20DRPusp4fMFrF0AoGtlO+ySH+XNJaxBOAhzx4ctgW4rdl8o9TMGK5JiTABO4RE3E4K2Le1hVVYX3U7evmUjbG8Y7bIYv\\\\\\\/fW\\\\\\\/QrXOf0o94bW9ZMzGlWzzjPoQJU9jRuOzoRNtpTfJ+KG6jqzVQOsIaiXFOoUlXJN8PRnRLHiq8ggra5GIrSthqrABQcgW7QWtlNxycgLMfa8BiSRadhgC0oHtRh3PlGvelfUM9x36Kaww6fIa9rX1Hpkh85RqGVfKbz8WaoRDEYqCMBZFqvISTyHuyimGD3m2RP1SFrG\\\\\\\/J+H6BgmT3UZHoeM8URlmxdfFF6LPsTgDQsv9RsDzQ8kRstenb\\\\\\\/StEOwOfBd2oRBO9yy01e\\\\\\\/ASnXeLxnvAK2qHM8jI\\\\\\\/IljCTiLR0iMSSuwvan039OqAgy5tnaJHUewE9hZo0twmDwbzvxdrI3xTe7eCxVQ6oZihU\\\\\\\/1BGeG+ApxbHP5ftLSPdrrUiKMfygWIKU96qACUb\\\\\\\/5dbk+TEmqtRuSTHlCA4o6t3Hl\\\\\\\/m0L8gxEa2krUqyfTrRBEn+4JUtzgFqFFi\\\\\\\/UGt9sJLvWtD9\\\\\\\/DNoGJ7u6xG5+2PUJsJ1t1fmBAWOegObuKAw3pakamIiTodIvlw5LJO3pAchA8jga3MDusOoBS8Tjng1598eGTZzdZSaOGitrkhzVQnhd769Qp\\\\\\\/XSUbgVygEO7qWBI2siImZs6d9SaXuQLxSo2mw1FDgo3a7R6GjRX+Pk9dsuRZrOwOuy5C94I4\\\\\\\/q4EcT8DpMM4ayM2bzaEq24bIzvTm7dQnzUDysPBepCiX+0kt6sitlhveFucgj9wucLohpAar2ziUDLEUcSUfKWMVcWFmrysyAuY9f3I6tYqN7qw0tQ312BMyF8CzDs9e9+KMUhCQuLlBslzSpeuzNXOWQMEqskHinUiRXAfUIIwbw0lEfCAdbk2SNxS\\\\\\\/axnI2EUOgFcTgVzr2RwWKp+IGU0I9D2Eb+rJbMhYCSHDuWRcParw5AKCPRoiXtV4XiCsWhivoYUpIkgneDi8P0uwLJdZ5amEHLUIWzkeHG9RgaUXpgYPUFGukV5Ja+bmbAdFovgEQFiOO6Vjso6LIVPKPM1GHTss3+8\\\\\\\/kfK\\\\\\\/18pzjsXX9DBtY56+BHhBJOagJmvxlN\\\\\\\/pwFVZlvrtuZgScAS\\\\\\\/IsmSGNSckyOAOZyUz+hTUdgKCYlEQu9DnlGNRdMOYqhA12CMDW02EuC4Ea\\\\\\\/nqaDVHm5Hn+neEO3spbcCTCqKZqgVtgV2vzPRFEA+FvOW0f3wv4aJUSsorj6ZPiC2obgw0Mbu34wgzRygWyL\\\\\\\/iCBYryx3jNiiEKna6XTMp\\\\\\\/Bgv2uq9xmnCiQcmoN+crQVq4zjtFn0rZ5UdiI5N2x8X4Kazv+LHPG+6OqD7SRupTW3GFr9HeMzFY1VfSPDRc8zjU3Y6OxnberCW5YtLPWrnna5JeV78uJW5NYi6FnhZSy0HAfefreICvP80hY9uhhs6fP3n51i0avS9BQEFJcC9exFHTPJBuafcRETkuIG0SVPSfuS9FkfEWnPHqtCk+2H1m5bHtJZoC37pZ+4qJVGwlP6JUeB4zaboJnlr3evL0Y\\\\\\\/h8EVKRvunqvCtumfYzD+3a2uC8MIbo\\\\\\\/Y6Oj5mmH43IgO4TZDHn1GtRsoXc3qPOz97ISKm2qjkJu2\\\\\\\/sopRUZl+LC8EkyseSopUE73\\\\\\\/Cep7JKMwtitPQtfF1jQCEaGoWXgl4IN26NYxE7d0RNjHPEk\\\\\\\/qT3twKIvOvEu1I14scD1nzG8vqAUyOCz2wjp0wuN1jNTR4GRI\\\\\\\/ykmMoQaYrvhcDzqyJ3ey5uub40Pl5NIHtLysMLDUgWR6KjrHk1OC50lcAprNX1cztbsVHRuwfoK5pGDtIbpN2bUEIpINMLeU\\\\\\\/usbIWqNlrQzA3Ymgq1wm9lmnKe48EZOcBdxnRWxyrEx6UfD2MwkBotgv0pLplG8frD8Hy3lIi+7KLhsfKqFd+AAyy0zNoyLnEBBODi7zf2JoL+E9Wv4GsOGTgEIwxWuf0K0YutQ57Z2sfSXnN\\\\\\\/r+3tbvQqDchbYmOszovecd9TvRwIX3plYodLONVesgHLFpxXdXMvOAkR9pbhkcbospb8ryb6NzNR7LVIGLK82kltWc0FQiZnj7qeeu9oXJ1OrAyPyOLcVgsdvOStWeWzW6PnCownSvLNh86fT5S3z\\\\\\\/xOLvJ2UuxVfIwk3iWF5HNpAppOkF3rye2V9CxWr25viCFLkrpRbQt2dHNu3VhfNqDQ3teHMga8tjj1V2\\\\\\\/jbxmByyrMujsTtE7WzyrdEc8aOehysqpvx4H46csTjx39JETh1rlLLWT0IVpT4g9G8inMlSvy2U32W2a9UOild4Vjta\\\\\\\/PFxlz4WQDY9FkZlGHSxTMBiAkLklqVP6x7Mn0UI8Gm6tlOU\\\\\\\/pAfGGGqSzTqe\\\\\\\/UuUgQ0FpD4tpyw0KsXZ5hw+MZ8uPVsMSs7S9dWobZyYiiNsGtt4nCW7s6fLCTEgj4gH4gmXRBWT0eSc+E3ty8vNbVkufulE3sZnsVe+3J405NbF3re1ftRGF\\\\\\\/bWbr1ZWety8BwgEGlYGtlFSs0t+M3ItWKSnUGNa9W9h2nFiD4XhXlwroNBZwt\\\\\\\/\\\\\\\/fY6giGI5F4YY2KC\\\\\\\/EyPuFsIHpVTl4xJ+WgzQUh7R7IOzSH49zH+xvkjjqcWddTVg53lOtkSaM2z5b8TcsjczP6Rd+o2i48JnhF1B55pgVFG0QmL\\\\\\\/xxaz498L7vYx0ASj69wjYCXQvAm5UKnGNolkrdB4iN4+OAz6WfGuobdSZHzhZsSTmf+Bhysrek3amRoEvYjI6UQi\\\\\\\/okuCVDbdXz\\\\\\\/kwL+HZQY73zTGlpy27GjfSurQk1TzIZDORvG1jvCNY0j6L2Q2zXE5X9z8GeVy1yuNRoAhJj5JPodnUUOdmm9YIZqgY1Ys8TuCKI4f3CgBMRxiDyYnNSz0v9XwoW9jOvb9vGnFjj86g6RGQsiAZXE94zuuTM12vox8AD\\\\\\\/wuZP50mk+owTGdhxJ7HjzjfgOR4Bq7zmuqoEYWqjtwYeDSFfe0+ooJr3\\\\\\\/DX4xhxJEFa5wjkZtcOMWBa7BFWqP3rvMVLLPbZ8fSgGQpzNYDlWSUA2BeVWCW8po0CJMCLN87WSF0EjqZzOOJlY6vlqOC9esgoQ5VOGVE1mIrHE0MgfoKN68gnd0BUEelryzYATBXwAUTATYYO3HkOGUqBVsAxF\\\\\\\/\\\\\\\/3sVovoldt1GkfVl+AmePTFquCD73B+nMceORgk88f+mvSZM\\\\\\\/miR8boDaf7P7FaDYNmHzfMwA42c9NkQwfStT\\\\\\\/t0Uwvqfp9T\\\\\\\/TvTkWTjWIgPemQkdjxkpd99IzyNKc8JtA3dz4aBKSTaPAQYuggQuS35A0BfdN2ZHzF9Fd4wRCr2+H+PJ359FLrszx2zopEXy7NAkpF+vZh\\\\\\\/t289aZ8JM0EchkAeLaNxeNV322bQFsNrT6DE5fvTcWKpx4yw\\\\\\\/yMyCK0YLp0TH37WjRE35B4N0LTF4osU2LVGMpgv1AauOewDMsqQ\\\\\\\/VvU6qc7qRi8+cPnPkNAANQyNUHd09g2X4eFgGBYT\\\\\\\/OJybGKijOGlViqxMzpNPccb+8yOn6D\\\\\\\/iQ75W4MaYpVuNqP7IiImh5eDPx\\\\\\\/apVu37IwCopEKhD2ItzzK95NTMPWFrtPjlrykROybENZaE+GbEo3og6HfpLp0WSb\\\\\\\/bUI\\\\\\\/uZLIIkxtONDbvC1+Ruftc\\\\\\\/0A3bCcoYAgt+QXctXZ3oBsBC7bXtr8+8X1IjWgfrGP86lHqY6T6EgcTryf8V8NQItd4Wk8TO7\\\\\\\/HMMD2FrtdTrwPNGVPqAllofSG4LzsA\\\\\\\/8i3iil7PU6qhzKHb\\\\\\\/W9Vh5WLTLaKt5oogUiwD0qUkF5MbEekL3DQy5JJJpNUDk2Aaxp28N2ycvE9ke3OAHyjKT+VpgVrj9CbhVIi\\\\\\\/2NKoVf\\\\\\\/ggJlTWRH0Pri37ieRmpl7mzIG2OcvmgechEQSHGTOOGmLPAcIEPBlI40h+O3zWzL0kr4VUn2ZFWVUDhFLfO3XegRejtThmWilgLlSQtDmHuQqQmkJ5pFM6z0R+hyCrPSLBkLfj6arPiYu6e\\\\\\\/8ZrdSBy8zbX9bhOLZedWznc9gz2x6RSggQ82YoHwIsUZstfKv8GHbbI3VVCl96GYTdVk4yYyJAiOURF4kK4m\\\\\\\/ZhJCUERKEwmRWaoaCVXluY99uf2E3hpcXhaMGD3RjN44Nmk9r6pv3GYXj7Q94B0eAfQUCfgi4Rrh04HAzxnqz2zQFsmDY0droyaGdUUuIOk0jZec1P3SFPXGIblOohTz9Is7Q+Enk+aFp6WGA3pVwbIkc9\\\\\\\/l0XGiFXAut63gkuLKFU6inYtzNZ2TaZCwmdnUXxO5Rl8tKL6eHkYSXskmkWomCuH8aft2wrTnBodYi\\\\\\\/RfltG1jdUghXJvUB4cso3SYdqRktfAaHLnMz7JT0wDSJhal9fTSqLx7eeUNklZA7NH92+\\\\\\\/6I67NQ7H6I44dtpYbeaOhiF+rtj1+hW4JkKzdEkeQK6ZmBB1Sjlj1ejiWVci6Bg96pjtWEY+dVTN1OZ\\\\\\\/UpNcrTCFI6Tb8pk37ofW2I9N3OCUzUbsTnFa2fvAACMQ3inp5qQVsSrWeqpF0JV5NJR\\\\\\\/8u+RkTAowJyMX07YTdM6sVB+6OXG0hQqZU2hVYvKlMeuw\\\\\\\/gbkpu0o6rd6lIyw\\\\\\\/DCK8LsHqlxn+Sc7A4ySeDTXArgLo+wgYOOlUicPFJPNCB47XZNLa8xOBmKB9jtr8k+137pmSR0P44cxT9x6Qwo6wUvIK0VRjJtfh14yrS+iHGOwo3FVs\\\\\\\/l\\\\\\\/otyjku58giVr08Jg7sZTQZAuQ+59kKjnrHLgacmBBS5oIAmcW7emIdyksyW1MdEHU2f77mxdYwKGqZHdURZ3VhqxQjG3Ems7x1EJuks77zJ27+l6O0GvsuGOESKbiPllpD2HE+aljwt+UiFRP2uRV3UBg8aCetDjqlIOtH4\\\\\\\/L06GHz\\\\\\\/bGp1ecAkydPzKoPFP32xtFZ5sh5LohjLAo8\\\\\\\/zwxAu7y2zj4EjUI2yAF5KHsBMcigMVL80uvf86R1IuYhINmaiUQoWN9GHmo6iklw+iYYAw+HfRjq6hrk71WrYiDfg0PwjzJg2XIFxCMPlBKoRFdjlzwr14oATubJYOcZ\\\\\\\/ya\\\\\\\/WsjTObst6S5a2jI9mTzBNlJNypvPmZaYNBIMTELzx3YqaOQIuhb4NhthPzZE1q2oKqW948kNbmOxNBLIsCgI\\\\\\\/DbEHQaspOUHanjRZw9qnqmOEmyVLPeiO8W0chSpKiFa+37BcpVFSGNGLjt31R+M4RFU4UxNEsDporBQ76acSmje2gOuCxx9uQfgNpTWhPlH0iO4vGut5QhW+8JlSKWkfb4LW2rHEjfZ\\\\\\\/J3dcvrf1t4M8ytP36u+8iA6q0iRAFlvo78EP4OYdc4zyHAqGyuDfze87tErciMbjscebCAC\\\\\\\/JRAIfZiP3JT+KfZnx9GiguPrNmZjeCl\\\\\\\/gSg2jeWGNJA8KI89h6w4ehNxJz+eITMZkpV+AivcLqNlibHgjFQAMtQnGIp6YE6yEZIP+jP\\\\\\\/PfoarnXLXyUhRQEUCnOBviYYgBn5OrwfsRC7ee4pQJyTTah\\\\\\\/g7q0xcVvpDzLUDYwIkRDo9j7j\\\\\\\/0XS5eaBSeg4C75x5PVuL+QsHwe2wYTUwfsEzrXZEYEpluR8H541y7oDAiLlW75cp8+TxU5r\\\\\\\/QQesDsX+BDnP0aFHpZ0qVhoaikolq8T4dDP6A8CnVEWK29lRQV0nZuxAtOWFG\\\\\\\/JhxNq0atYb2vros6VdENO5ipFXvyxoui79GrY2H77rjEaDVAh4wbItCltO7YDi3O2usVoRo\\\\\\\/kzm4NDuHpbCwygnAw\\\\\\\/G0AbEgMDSR20P+0aXRd1t4Ejs6wjQwGe+22UTxaCCMKnxmnVcPPI0OIVbLwL\\\\\\\/uCSO6+LbgRMA55KERU5TovKV6oihfC3kOC0UVMCRnHQQQzkRActwhLq3PZVOGm2ZpLBbhH5KYbSnclVlvk3dvwhYwUXdTXNYR\\\\\\\/r7YqJKEeHz41Ja3+FdL2u56uk0CvPnOcFc\\\\\\\/ZmJ43tygfau9IC8BLHfGZLMNwfL7ltZyh898KJDXH6jT50JyFnEVUUBvVwX9Au\\\\\\\/t6dx3D0CvP65WdvYABwok1HadRrJfWn+3Svf5n1tMgzbp7MTkMj+fLgm8xUbepztJTk1ZCxd55xzyEo7UPSgjHe+raTLKdM3EraXq0NoAMd9l2zHekrpPCVX3bWpjBFgKxotfDFuODDR3JkGIcf9fwNK68muqMo9J8FN4+rv3PGXJq22MsZr3gEz7HXjovIMX2b69e3q7QJM2pwK87SrE\\\\\\\/zaLmeR9XqcOK385NFzv87e2vvKL\\\\\\\/sqQBKydVGcva9ZzZkgj\\\\\\\/vbx4S8o0Olv7f\\\\\\\/fXB0YTYL3cc4LaBivxMf0i8CF1A6l7PY5zhRJcO1jcs0uMZ8TFiNbReNKHrJvWSLyXH4Lf0iBjYogx1LjmjvELEZ\\\\\\\/M7AJ5qGqno3IUGddc9GMUmxDzIhlIzQWSp8Hr8ZLsDNYjF25O\\\\\\\/YJnHHTTYgvEAJRtLWftU3QOW1vnvmZqTvKnw65aB8wqMDAAAJHluESz6GGWOVxeElmdYwKsTzcspTU0Ll\\\\\\\/RQ\\\\\\\/iwrrV41dz4pQ0YJFrxrkTYDiHanvQAkiHlOBZ5ae6bkJy8jUBQBL69dzTZaxPVoUp5rTFj4vNaTShv2Q6JS41o0QD0Ib5G3TZh9zbCXU9FeBkn1T8Hteh7H7m3KiqkISRjQ0bwDYlcIF4CVj6jhiizFv5HHvkqZTPR8CCWF1wm+s6IVlmBDSdP\\\\\\\/YscAwRQe4GQlgsCiN592+nCwl99ERJsMSFkX78+x9ogjbbKQGt5gqO30h+aSFYqlFs3b5km1hHF96+tyuU3VaCnWFqZcQMx82I7Rl2lioYLcpBzzgaFDk96uUg5qmoWi4OLG3ce3QTqp9m1qxdLyXy2BNaoVhUNnr+iQaAXIphGayyj7Zzx9ouevMI0vJz\\\\\\\/+SNU1s8veN7Xo\\\\\\\/64S7rKG4XEQE7nm5+jm1+drOKt59J6E\\\\\\\/cNZuixX21U0aL216pzQYvuImpIfyOvNICu2vxGE4Ee8rWr8TBuhRoEeS1T\\\\\\\/GSTLGl6oO20QDYhdbw1IbcgOpPZjV+t9gQ7vEY4leo1omSnIAPnR3kgVxXZuslijEjivjhI9YwqZSlZredhnlanby\\\\\\\/eGLAPMG6IQOOGzkR7dbwo1+qjl9Nokz+J0VLo5JxQPFhrbm47vH5FdHrKFEykIrMVV7N6pJqMV4wH2u4bqmU9ctGSzv5lPtFlwYAA7jc3Iq7ZmVYdbMrYn7bTXcz4dnksUWYu931ekAf1gbasvh5POtEwMlMXjRgJutYRHitnpPhudR0jUELxZfK3kxZxgeIvbW7U\\\\\\\/bZAICJSp2cJ9jgfJPJTQPcS4Nh2j+YWDgsZXPje+Evg8Tp+46+rNhqlkQY9Pw5HMqD4w7+8yi0WqiyETfIXbqd8hFAi0l7L5SC678ZfCqAtIJ+\\\\\\\/+vJzlp2bR1PB7UKdsR1oGY5Gg+gvJFEcvfRBPs4sZ8uU7HPN2u7hTBvY2HuaV4hVmt551H3Og2ZPDI+oMvxnC8o6ib2lEY1Sx49N0Xfu1jGz2y+XroZZEc1ogjDvztS6KDkOfS7ENB+Nb1yDw+groEDFiY\\\\\\\/Xy0C71U9FmlzIBDCa0TxddTX+TsuxcjiiGodXcS2SvDbd8x+74A9AKGFSRpFs5lH79qPDR0B8DmIIrhoZ2x5fS\\\\\\\/p7cPJQADlA7CHOMjyc\\\\\\\/2ej4CqCjQFfGHGcEm3hhT0e2q1qKk1Uhze4HgNP\\\\\\\/I+zS+mBP4WI9E76Uvo6wmKKr5YtyX01lxAQO+zUCtP5jBeTta2RutGqXfxudm1\\\\\\\/dJxeQ\\\\\\\/c6trJb+d+S7rAIeUhfrpwdLNdAr3coZsMHNk\\\\\\\/B+3YgaqgOksZrfZYNiyE\\\\\\\/YfAhmfF0t3h61BllMeuVhjCo2WAOcEmUugaanxasNtNoXqtj4rPkwYR+PdCe0DspW3xtLEVzAM2rCgJGEOKCRHGgkENIa5qo4gQVHPKuRLKN2nSUvOltwrS5ACX2BaEJzlaR\\\\\\\/EDoKa43WhgMTkBmA+0KAG27ZWtCMRP\\\\\\\/zQPZexZncG3gwlHL4BJbYJw8wiyR1DmudIAo2zdYnAAUjlkO7R+IG5Kxuoi1d7ys3RNB8nc5hlkdpEO2YwucbUctZtEYHgrCpxwYni\\\\\\\/iFbE8NuDsn2rBgOqR9mD4JhQRfEZjQnlpi6CcLZeXncR4d6KdJjiK3F+\\\\\\\/sQpibBT5K2TV6PZgriFCbJRcLv9u4iwcT6zEGwo\\\\\\\/ugqGHQrPjvUhRPPXSpbOGJEIhOoCeMmyIVIgywZ+rotjW26g6wyx4WBq3Cg+s2Rmj7QrC8g3FylVVP2mMEXhH3eH05XHxaRyARBW+JMYwOH5o9WxdXC\\\\\\\/0Hg5VH0ry3bWqFVAH4oJ4HjaULITisI+YWnD1t49rAjOh22hukgYt0teG+U9lJ7tF+VJJbudn\\\\\\\/ICS+Bl5XSD3BbD8WxKxpWN4An0JuyUcryn4QpWd9vABTM5B4uaVt624WDA3aaCQu9RJYtcn6C\\\\\\\/PG5uj12QPems2st8u1K56Q6JXmRfDv\\\\\\\/R2\\\\\\\/DB3Q13ih\\\\\\\/u78Y+i8J4Pt37cu34uE\\\\\\\/arEnMNiolfWsWNtGMqo9U2Q3Xyk0ryvqzJQVC8r1Ksy2V3qLNoCuGFZDvWFWVj\\\\\\\/+HsWfCWheUihCyY64IU5+9GgGJfbC+w3uplZqcw6ONHHkb0+S+C1KhA\\\\\\\/AIWMTE0qO8gZ178tp1tEWm2fT7xQKw4ERR9s3oZkVqaSinQr8A81XM1y36E8ThitbPXWNPQCXJ8H\\\\\\\/BiwzbLZp+gYD+ODknraFtpUPMPJdzZNXRP4AE4p\\\\\\\/bn6+J+bpgTyfGKfUwHf7vBBR4jut8x5VLanUVQDeIxJmvJM6xLm7vRYOJu2EiJeKKDbpRQ0egEmZouKLjxhATiw5oIr9tFT6WG7pXD0aOSuCmTXcBHGYT0eXfwoD6lZBJyi+myYOH7RXvHRowz8SA7bzjIlnfML1vq56oR2S1Q8xAl0+4e47YRlEz6I9Iqu1CqK\\\\\\\/0i9M6ncO8e1GNHhCCqcoDEeUP\\\\\\\/we\\\\\\\/kiwIqlXBBfJ756beCXs+JbC9QEniSxzHZfXh6I6tLLMb+TEfmNI25YAnNV\\\\\\\/TqFXF0\\\\\\\/B3GtV2lMHP9LTX6p2DcSgw6Ms8Xa6ImwLGFZe4rG9zs20iMjaq3WsIGAanf4m5hHbyan+IZ58YcG11Nz5SWeY5yhbB2huPumlgwkp5gEbuKkUte\\\\\\\/kMsjBGzrHufOenubqSBgFbaMfHzZm6QTQoJlwSXlMNBElmZJob+JXFqk5gHavGZoVJjhTCxbAUOfzgx4yIs2B8BS3PH6EV26+7UwPo1ko1azfn4Brc2fbYYBmrjpg9sAKuxIQdVjLqPTla2FDwPhpc4VgxcW7lq6zj8b7c+14kqNt3VzMhLH1q1+RAqjceqxqR1NPvSwPpaTkyNbb76gSCOVN81jpZyaRIgerT5NvYUw\\\\\\\/kR\\\\\\\/fyh5oDFQcFTmJWunwYb9sU30M\\\\\\\/PrNSnicJQDrGXcfaKdgsfUF1oWgdCKS2ve0PW3fUV9\\\\\\\/o2zl9Ns3A358FEXmNjYp8rKLQYGqjURjZ6ye2rE5T9XUNGGsFcZ+SnYL6d+lPX3EZG6wGCUiMpEJgiI96+7T3QNeIrW+7sBlyM8fKqP8HZhm+PgVEFiXrp0+wBQ0cpU4brwUZTdnKb50EHKY2LKn8aG2V2bxiFfd+vXN7NgHMFnPtW3jQsoN4y2q8ipz03G5IN89O\\\\\\\/+AfiHQF4viJ9ZQHilIreaf4BCK6I7vScIql8Bp8+bMHyNkY85C\\\\\\\/gIWQqRNkr28RFuczQ4+L\\\\\\\/his2xKhaeYwo0aUheqq7CfTsl1GvEzNN7RMnShq1TWxCU1kUjBS\\\\\\\/xGq0g39KJm6TzkNDwkf8bMZrX7k8a\\\\\\\/0Jb42LhrbdDGc\\\\\\\/8ZyWOcaJ1AEL9+6j3Rq5OZeA3toY7mxP+Aj3XsrsSRwJuK47TNGJxQYsGQzHLDEOTsYBhHmCXSbwpCHlzof4CMk5OSG2VBznNYCPkF4ccADcVm5wdAp\\\\\\\/qjPK4YDdUdHfDQQ\\\\\\\/qTGCEPAK02CIMs\\\\\\\/H\\\\\\\/ING6GU9C63LfTLHGN3VImpyWyTxSaqZBw1rAazQhgdC2FNLCezxUTlJrV+WOz9VGlKvb7ONliZPeulujxB\\\\\\\/1MGD+HJBTvvROjZH5p4i8pbf5kTs+z+6XpI0ynyl6nUj7e5jo+PmWosZghFvKaXWXUq7bx7o0syB8NP0ZFOS0AiErW2STNcxY2VLaXc33aK6R6c\\\\\\\/TEbVVuiMdu5wcQyMDf\\\\\\\/BoVMxzkV2ZYz0JDCKLl3uMZ2DER7NFjMK9Iq5mY92kwkEk6eH+dxwTI+rYZQphiY6SAsEJFboND5Dvbs8mWBG9UN2qlYaZrjDJNOUkS1WUBUJtG2BcirOoYjAkEfmudXWl51h0hdV9vQlu+yFWDN1GsUuQDJh\\\\\\\/0bfFMTdxmiyuOtWcmA4Uc52VhSwHcuNmEyfnfGqQPuqQVy5Qo57ZHTN8b5xDXWHCzCcXyYnxR\\\\\\\/XnPP3ff6wE3owoDYlMKxwzvFVVRU125l2cpBylQC9qRrSjzKMItBJGVPJSqaCRkUOfS5rHDRJtIwwdAvR395rWNRZUGiGOS5g0kelRRg9bmjDF7oXDqdvgr44X2XFzFbQqdsxHi4PqeiFtfjrcEPCaf24BDJU9S7ojbLsT23lp+UX4eynZcUTAtLoYkO3NzT4zB7GR8wZsLGnrAmfrsY5Rzfafz5n1Bdarmk3s3Pn+LNTXleC792xBFrsPxcbc2H\\\\\\\/h0F1Acyq6du6GaM6koNfY+SiG1CuZHFdEhwbE4MAXJraK7fM\\\\\\\/DRejDij24uaOhfvgwHXwg3oKAeHFoAysBJ3T+A5KxZ\\\\\\\/\\\\\\\/6JyxWcFeQBbhq2cWOrqfILlQQ8RcvqsSn97k+ZMvKjAxUKchdXhFH5NjtWI6lcrZI0D1TOHCyDOXGRqfct6gHyPwMcW4O4cv0JB6cgRFyQmIsVo689oMlOqEiQCdh8OBUIkdadB3PP22n+ClzCTnfWmNeTV6YeuaIiKzrVA1WW\\\\\\\/63PfvVGJmzH5y0YSrQZtn6VwXa5VWrFw\\\\\\\/3+JNZkRYSS1Qdi4M0R58yRe9c3+TEbX1ZW7NRbchLfx3QplWjFmaDQ+SZLxpjCSGHHL85NMknvwX6uepti5XxveYLpkj5g3oVjE+Crx0O6FO0gH5gH+WK2BTqSXFsDqxnhgW\\\\\\\/w9DRxbIQWShlwNynRVYQIk5t0pYK1b9rDUsrKyCSNJQXG2UEUAUUeOQ\\\\\\\/4KczZMLTsP5rWb3ZzuD3IrX93bIZsLF+bIin8YznOFMkuzPQ9zAbT38Ap7YebtvXvY4z+C\\\\\\\/SedDOCworE\\\\\\\/IH2TrLr9fu8lb4lqoRbgABPsnD+vIl60WqzO0BrSpirvc+4lizoyqXatZVvhb9dwzrrDpRd8ggYir2NLuMBWOX2oG8VUFEWVA2DgWoxpchwNlMBbJEhJhf\\\\\\\/s\\\\\\\/OejuCgHXL\\\\\\\/miKoLPuza4DUBExavZaemtzNd26\\\\\\\/8KLUFLAeR6VzLt6oKDTMweEiKV\\\\\\\/7AMjVvsds+8pepek4iXkcXzb7j4o9X4iZaqFFf2c6WKZTC\\\\\\\/WL+9bJzop0SK0ETUyxU6h\\\\\\\/ve4FOmqBanAisX2wPfln8p7r+j+j19CXjeIPkbd7MODGVOOui5mL66TXjEKf6rDZV3Vw\\\\\\\/BU0EMkLQxV2HdnM3m51y43o81OVI1oltSnqouF20TjN6HfuWl\\\\\\\/4r6Gerkxc2G0CDWg46loYd3EkQsT23BgVh9EcRWZ7kgDUSHhqdhRNu9dP1nSNPsRZmPvVRjm3nKVPXNa1tN5gxZtBA1Hw0lTJsAydgF0RioL6YNwNkdFeWxuU0y2GP\\\\\\\/YyYL3INIOywwTBjHmCKy6eFBoI8jIEijoH2dmUdXBgpafHLeM1r0tCns0JC82UZhyJKd2gbJX9DfhYuAsitl5urVZ+IHX5rYoxzuKu27KLIG9MemU6rZVHl0FNH3AsoVrb0ONBdH3fWQGZosFh4wWPXdDgxYRdMfYOVKT9nV2cJ4Cz37vytN4dK2bpYu3T1ch7+4DYE1vn\\\\\\\/+mFX7g5XKV4CuCswkKCiNfEwfluISDadoA1nJTUxtnR6tO9odGquGy8qATYyt3mDiP9j7AYrN31hbInlLHi4ooblCH3IY2fZ0DH2Y0fbSNXyOziekfV1XjsjVRl3OQsbXN\\\\\\\/z9ri3zGEL2IjuBA\\\\\\\/6bnqNuycPlG1ezfQUU5wJH8MEP+nxv8dtylNtRsTVf7myPx00bQFAn3CvQMISSPECegHlMEOKWf0Imk\\\\\\\/5mHXIbTR4\\\\\\\/pYyiHfEXW3L6KL14lMC\\\\\\\/jR\\\\\\\/UnzeDAvfuP9YoUd53LAx\\\\\\\/YocYirTLTSIpjlpsh\\\\\\\/GKEq4HDcsLNSTn\\\\\\\/U7jGEsV2vxw96tdpzhuRopPgzn72971iYOGZM8HveORrzTSm20f5NUBEyms9bT+8\\\\\\\/olXpFC23KekaWgcty4ovQIi4ftFvy7tVjU87iiQx\\\\\\\/EcHo3zbc9YPpFoa5\\\\\\\/8t1dhvUIgAFgPYwussCVJgPfcMvEsL2BKfPcquM1hoCTh9b29gGYJrSKFxqfVA\\\\\\\/m\\\\\\\/tdMhDkAYKikaNWACyqSi6zeAEHd6fVJC2UdJna4H+JKk32EQAxO2OqyX0fqX3Q0QDmrehqc8qECCPW5aW\\\\\\\/Boojn\\\\\\\/RS+9VeQ60fMvvXXIYtuCpXdrLtJkSalbhYVVcWL9GMEpM1K5pCgSzZWBDXjSfl9D2ZBXyS2wx82NQf83AT1HFPr2OskdoSG2rLuPmCnukfR+AM8d+NcvRho29N2NgoqDIq4cH4AB23Me4g8F7ChXCSl3XKtWIiTwTqGpySflR7HvINTyQNJdgBIF726HdT7EBstTxaCKFmLNf7uTBWivrxtuDpxbtBidl4P27sxo2rLGS1TnVY0\\\\\\\/ZxcUJvZ0AZqpl9RJsuY7amOlANh8T3314cb7bwVCWdc+1ORUxT\\\\\\\/KAL5Ny6Mkj+jEKAuOuHjrxJzK+cM0zyIMYSP7uxSIv8sOsVex06zK0tfXY21zk5Ou6zniGOzBus8Va4fVGJM5C4eiPN6c3E9xVU9P5IwVvAJEw7dIO7BNzSffxDAP6UcE4qhFdYE+GWTYlKFhXh5749IrSkCG4d4clIarxaCkYSfE9r9KpqokDEmDE55uv3Ul0GL0f\\\\\\\/xJkvwXugJya5tZ4vy7nfWf7zVBJG7bmRBmVfK5Jrop8eKd42aoCuFzHgE\\\\\\\/pAfCnQnAcjbHE0lMmT\\\\\\\/X8zoq1Qc3vz14NQcU\\\\\\\/tywa\\\\\\\/SoMtV1TM5BiIgmB+B7wxcTV3U0mHTGD8V\\\\\\\/bfvUKf06Hiz5Czy6eQWUUIruRE4GMD0T1Bs3KyIS8FCScfcVJzCxVtQkCJrLMnlus716KrbsQjbOm5uY7F+ZGrNlhwZIvbqeiTJdnAsI\\\\\\\/YNgdhv1UL72Op1xMSgndGKRaxoaIrp7rWFRJ50BwLfykAebZf975FGnI+msqqgY4zGg+CsXGr1Z9vBtQ+4SFDyWoF\\\\\\\/Bm0R4a0\\\\\\\/TSQ5pc9\\\\\\\/5xJV8iog+Up\\\\\\\/ObZqsbZCz3HH5tUtQmE17o6qKaQAesInDv5h0gAVYcgoHldsx9kQ0uWU+yydb7DP+uW5FfUGfBZnYusLQGMtec0u9so7TULcyCbjWYuNvwFQCKvNBcFxAbAX1sxeq3zq6nwmj2bkOEWZMMPQn5\\\\\\\/18x+Y0h4kSEe0jamDJkqRD466hvWUxPpgzcQWYexzcI6uBwwSf9EEX86+Qu5nWX9sd6mBXpcEu9U+lqjYb\\\\\\\/ZIQjp31BRJHU4zlNON0IjGoER61Yf2kUsn441NeXCbQawI6Dz2QxXAC9Kkerm58YyzWotlPoZs0HkszT4nYAfTWL33lRKm64eZjokM9fThLF8dbSZPE8R6JTDo\\\\\\\/qDAgMFySeZhaJz5IFFqnl2QYEwk7eQQmm3SLsl4T1KgR76hWvq0zoHAkuzI69duyKO8xGTjVwiD10gcsXAQH5DMZz1AJYo2qF+e7g\\\\\\\/07b70LASZ9s9FuREjvqyu7LQoR5tfR7fplwc57S5P3gJyr83h2oOT5zi4ikokQNE1snrU20R21VSqpiT++CL9XIeigd8p2HYJC+FHcB2YvogEtmxM5r9Ry6KvjbysM5KqPk6pUm9ilQWljKaNOa+Gc05WrJMtYWcIP+h+TFA0N3ekMjAmCkx67TvHko7hxatGcTg10tDiD4z2G1bpKtgMNpPogWrFPEDKzJJmP\\\\\\\/lIUejpK66zpSDIOETEqyaoHSa8RI3HIrT1IKDLu5ulkGhZWuoo71Dd+LsqcdoOKLVszlOcnY0wswUddRfDHbM\\\\\\\/6c8rjCvbQbKqjG1pdkzMwt4AqNBJgJ+YexsZPEz+BCtvPJMncyoM1UPR1JIRxta+Z1cGzEjjA2CaigiZexL3n9wU8bSC+SveH2X486zyYSlqk9bt0Z6JEu1rccygCjsEiXv7wCBKzT2n1uCka2C2UTViI2f6Ava2OSrkeiyCyak680HnlSZmVi4N0Bja8RiyTT15VJhZlHWKPV1BesOCFkB8oRXbYYBAzfPuiw6YKjh\\\\\\\/KHpkAMXcpkr90\\\\\\\/XSg+lrOPndm6N8eXV23yOV14F3O3wj6NYHDvzascxU93R9oq+qh5D88NqavWaQh6zOQwUBmQJWbeNJ9zu3ios\\\\\\\/eS+WhND9w6bZj6ZHX+ABO5BY26mfu1CmdWt0cDj+zEcBYmLdgY57zadjRSwFZO1Tcbbbgskl8x6V1eMEf6g4aNn21XGvH\\\\\\\/gk\\\\\\\/vQIIhoacyWuFAH9lhKASXzF31rTpe2vXAsHyyTvJO+1qCLgVbpzkzRAH3jiptJQUu36lAy6LFrbXdtnF+j73ZQNSMLXTeXMpgb59Z9b1q2Ol6Bx\\\\\\\/MvD0J0eM4kGRr\\\\\\\/CcEQ14LRUVaWtkIy2ynPYIm\\\\\\\/NdOLDDwHwWXE8r4vpX+27EXCjS8MMBjfbXjGGMP4+2jOvagi6YhT+3JBoy\\\\\\\/piF+X8LbT0f6WRI2FETxtoxzqZ18U6guV66sGvUcm3tpCY3R4H6NEmZPBt7zeQaIf9fJAvM83pfnPThk8hKuooVO3NFosO0NxDG+V6hDIDLk8g4NDjVp9Uy7oRuCgEiDSgQscPsQMSPksVs6tk46HGbm6eOK28Ppwz9inLMX2Oz5\\\u0022,\\\u0022iv\\\u0022:\\\u0022abOOzHJdYeoZJN0Eq945XQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022239cd44e3d1e33d09dd35621c87fcef0b3d9b609b8abeb14207273fd4dc1690c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NqHgKdzGHMrVaBp5Gix3Wq1v5BM9xTf1SFiIssQxQz6elTOVv8a8ZBOk8dge3CzKY10tKimdBbWgPr557W77SQPyUc6iE1g7xje5ocGGnf0a28QoR8K0KCFa\\\\\\\/Kj28gpFmqBZ5psKLu+vCuxGl4HX21rbRmcQgXgJJRO+9ZSYiKT\\\\\\\/ndQ6OhDqXiA+PgldT+uNv8UHRyJu2tbBIM2Auz1hDJUHz4AR9N1AXzPR8NXw3eGCojBl\\\\\\\/ScsB0KRKVrR+bceQYxvZkbrvRCc+eNE\\\\\\\/7UJR6gO0Cq4h\\\\\\\/sF9OLHP7ypbP7Mjudt+drMvrdlnrpImqLfaiDLefVtJ\\\\\\\/TZKcWGA93Mtxk+z6Igzo9CH0rEjXeVHcuHQDnrMdonutCqPTfVoM88q+RqjRh4lML\\\\\\\/7SZ5RkzE12v0gf9el59yiLfBl9Or37mejVoViHl5DxZOAOqBBE8aMSD1RHrENPmWPZo0UkOXF4CnbfTA74rNlXieccYcy1Hh6dEdCs2\\\\\\\/AsPQbhmSUYjLj78DIeC\\\\\\\/XuP+ODRiuxB7UAQz7LneLPNcShIIffoIlyYMAQqckeeJhegg5c+WTaRdDfmYc\\\\\\\/pqZ3HiOwE90l1lb6GvpKu2yP7yxKsEnkO5P2nUyNUQd8YvcrdAHR\\\\\\\/9nX+HY5HLakI1YpPvrreaBiqfJk6asRmUl6Y8vutdb4b+Ubf24XCeBaMXHadFHQNm5QjdTM5ZJkWA9w6mpb7UJOZ8wLdiRuFAGnO62me9\\\\\\\/6j\\\\\\\/vu7hj5TOyJSO93OECWkY9MS0xbB2s3umYu5\\\\\\\/6J84q3J\\\\\\\/hU9RY4b8nodkveUClTrcxbqWcPSDG7Hw\\\\\\\/UbdNc2zTbv4NEhtwvwkBIHuCAO1znzygZgM\\\\\\\/riqcnEIEt8OMcywEcoSyzvN4paLxunxr5\\\\\\\/Vajsk24OmLnGn2fkbyX4ADzfgEVqHvGbRhzDndxQurlPw5OCewpYl+OZ1zx8zkEE+CTzF5dfTtfE+AHBAkKmyWyR4LAt2y\\\\\\\/HIMrLBis6x+WJe3fHuiPoNPslm0r1\\\\\\\/O9tS60\\\\\\\/uMcUfaX6pgdz7vNUDXd1vkMxLaVc3t9f8QBRx\\\\\\\/KXR5qrA63dv2wLgBz3vgh6IkggT\\\\\\\/\\\\\\\/uVt0H+Mx7Ws0iSKU7tGQhdI2\\\\\\\/mRuAk82rgHBNaJrFG3C\\\\\\\/3faZgIlHFG3gKZQIdnpANc5X95vAmgnfQdbBeQa966IiiPWpXttlZmUCBkZePkhEB9YLuUEQR2hq4ZStbDBK\\\\\\\/yKYlhA8idfW\\\\\\\/SP1uRGbfCgKPsPRYNL482jqN2EIjNxvnl3vnRWIn6uagCAKLSaibsMgEL5dnEhXCNTnffZyPQp4O9jEDeluo7JHDAk+iOc4zEWrIp2qyE57sS3\\\\\\\/bqiuoVpHid+K8lN7Cl5dVO6dOsdfUCOmgdA4e8MHrECX9GkpsjrgipOU8OTENzxH1x9gqPaKqhJfyekhSVfLnAEUOcOLomDUDhzV+\\\\\\\/dUR2bnf5NQLoUzwkY+Zoyc8Ja\\\\\\\/\\\\\\\/fxQL5Lvbn7Nm\\\\\\\/3mV4wUNyBjzBSLNi+smmULSpsmdDmSrwoFOlnulN1Mja9ERlDlFdO3I1GreVHVND8kAvYenUQzMHVAOcjp9G7hYC9r0HIncRaaCJzY3kK5AvZLzyWxffGgj5ocIQmBGQWRzsYGuUpdIzeY\\\\\\\/LJc8o\\\\\\\/GVymd1A\\\\\\\/4i5vUVRpZRwA2y+iEH95yQM2Y3qg0yXJU1SKwYruw1m0YIcVVd8lzmvz\\\\\\\/98e3sZiGArHSb+qdiQZGTYB8JL7rceM6okSFBHOviFotAbNz3V9OjtPw3TdQCqMqvlLvUmxnakwOgGWiSXmroR\\\\\\\/rii3vtH55IR6garGdsivi8Be6Wy4n2OgUwYcCPCZPnNwonUupoDU3uOXpjV2gKvHBM4F4Y39d9pTy7bjwzcmqykAjiP5p+MjPXgs2yL24WNEbahlvAyNV\\\\\\\/P1tVAh6gLVW5HnTMGsZRejwrDHT9VMBb3rujSadxXDGgALVzXjFeP4D0YFkkMr9c+VBQU+dtaVKkBAmxyFiHcMjmt3vCdqnsBbuBODeNI2sse2Qagv4A36YShhc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022+mp9sobfTBGxeDGVTcqrwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f8f03da55c79ab0d4f4b48ba133ca5a3f6f6b45eb6a10c8056018f39206ad7ce\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459629 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/325" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hdAoRwCIk9gLSFetZDkArF7d6NLOOnImt_RahqASxQQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/325" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G0xDmHUz65XtjcTcHtxkF8lpBgpX4SH2he1BPXgAS9Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/2/315" title="Heterosis Is Prevalent Among Domesticated but not Wild Strains of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/2/341" title="Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459629" id="node459629" data-pisa="ggg;4/2/325" data-pisa-master="ggg;g3.113.009845" data-apath="/ggg/4/2/325.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nobuto Takeuchi</span>, <span class="highwire-citation-author" data-delta="1">Kunihiko Kaneko</span> and <span class="highwire-citation-author" data-delta="2">Eugene V. Koonin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">325-339;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009845 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459629" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nobuto Takeuchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nobuto%2BTakeuchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takeuchi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANobuto%2BTakeuchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:takeuchi.nobuto@gmail.com" class="" data-icon-position="" data-hide-link-title="0">takeuchi.nobuto@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kunihiko Kaneko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kunihiko%2BKaneko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaneko%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKunihiko%2BKaneko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eugene V. Koonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BV.%2BKoonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koonin%20EV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEugene%2BV.%2BKoonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/2/325" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459629" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459629/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/325.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459629" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459629/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/325.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459629" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459629/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/325.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459629" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459629/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/2/325.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459629" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459629/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Horizontal gene transfer (HGT) is a major factor in the evolution of prokaryotes. An intriguing question is whether HGT is maintained during evolution of prokaryotes owing to its adaptive value or is a byproduct of selection driven by other factors such as consumption of extracellular DNA (eDNA) as a nutrient. One hypothesis posits that HGT can restore genes inactivated by mutations and thereby prevent stochastic, irreversible deterioration of genomes in finite populations known as Muller’s ratchet. To examine this hypothesis, we developed a population genetic model of prokaryotes undergoing HGT via homologous recombination. Analysis of this model indicates that HGT can prevent the operation of Muller’s ratchet even when the source of transferred genes is eDNA that comes from dead cells and on average carries more deleterious mutations than the DNA of recipient live cells. Moreover, if HGT is sufficiently frequent and eDNA diffusion sufficiently rapid, a subdivided population is shown to be more resistant to Muller’s ratchet than an undivided population of an equal overall size. Thus, to maintain genomic information in the face of Muller’s ratchet, it is more advantageous to partition individuals into multiple subpopulations and let them “cross-reference” each other’s genetic information through HGT than to collect all individuals in one population and thereby maximize the efficacy of natural selection. Taken together, the results suggest that HGT could be an important condition for the long-term maintenance of genomic information in prokaryotes through the prevention of Muller’s ratchet.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/environmental-dna" class="hw-term hw-article-keyword hw-article-keyword-environmental-dna" rel="nofollow">environmental DNA</a></li><li class="kwd"><a href="/keyword/evolution-transformation" class="hw-term hw-article-keyword hw-article-keyword-evolution-of-transformation" rel="nofollow">evolution of transformation</a></li><li class="kwd"><a href="/keyword/competence" class="hw-term hw-article-keyword hw-article-keyword-competence" rel="nofollow">competence</a></li><li class="kwd"><a href="/keyword/structured-population" class="hw-term hw-article-keyword hw-article-keyword-structured-population" rel="nofollow">structured population</a></li><li class="kwd"><a href="/keyword/soil-bacteria" class="hw-term hw-article-keyword hw-article-keyword-soil-bacteria" rel="nofollow">soil bacteria</a></li></ul><p id="p-4">Evolutionary advantages of recombination have been extensively considered, primarily in relation to the evolution of sex in eukaryotes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kondrashov 1993</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Maynard Smith 1978</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">1998</a>). However, advances in genome sequence analyses have revealed that many prokaryotes, whose mode of reproduction is asexual (<em>i.e.</em>, no meiosis or syngamy), frequently undergo genetic recombination, the process known as horizontal gene transfer (HGT; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Feil <em>et al.</em> 2000</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">2001</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Koonin <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gogarten <em>et al.</em> 2002</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Vos and Didelot 2009</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yahara <em>et al.</em> 2012</a>). Numerous studies indicate that HGT has been instrumental in producing functionally consequential genomic changes throughout the evolution of prokaryotes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lerat <em>et al.</em> 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dagan <em>et al.</em> 2008</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Treangen and Rocha 2011</a>). For example, HGT is implicated in the evolution of various functional systems and specific adaptations such as various metabolic pathways (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Pál <em>et al.</em> 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Maslov <em>et al.</em> 2009</a>), oxygenic photosynthesis (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Mulkidjanian <em>et al.</em> 2006</a>), thermal resistance (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aravind <em>et al.</em> 1998</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brochier-Armanet and Forterre 2007</a>), antibiotic resistance (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blahna <em>et al.</em> 2006</a>), pathogenicity (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kelly <em>et al.</em> 2009</a>), and others. Moreover, it has been shown that HGT can substantially impact microbial community development (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burke <em>et al.</em> 2011</a>) and ecological diversification (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Shapiro <em>et al.</em> 2012</a>). The prevalence and significance of recombination in prokaryotes as well as in eukaryotes point to the crucial importance of recombination for the evolution of life in general (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Komatsu 1980</a>).</p><p id="p-5">HGT in prokaryotes occurs though three basic mechanisms: conjugation, transduction, and transformation (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Levin 1988</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Thomas and Nielsen 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Johnsborg <em>et al.</em> 2007</a>). Conjugation and transduction involve transfer of prokaryotic DNA between cells mediated by infectious agents such as viruses and conjugative plasmids, respectively. Because genes involved in these mechanisms reside in the genomes of mobile genetic elements, transduction and conjugation can be considered side effects of the selfish propagation of these infectious agents (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Redfield 2001</a>).</p><p id="p-6">Transformation, by contrast, is a bacterium-programmed mechanism of HGT that is mediated by natural competence systems encoded in the genomes of many bacteria (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lorenz and Wackernagel 1994</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Claverys <em>et al.</em> 2009</a>). Through transformation, a cell absorbs and integrates extracellular DNA (eDNA) into its chromosome. Although transformation might be viewed as a side effect of metabolic consumption of eDNA (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Redfield 2001</a>), there is increasing evidence, at least in several bacteria, that absorbed eDNA serves as the source of information rather than or at least in addition to being a source of nutrients (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Johnsborg <em>et al.</em> 2007</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Claverys <em>et al.</em> 2009</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Zafra <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Johnston <em>et al.</em> 2013</a>). Such evidence includes, <em>e.g.</em>, the existence of competence-induced proteins (<em>e.g.</em>, DprA in <em>Streptococcus pneumoniae</em>) that protect absorbed eDNA from degradation by intracellular nucleases (for other lines of evidence, see the references cited above).</p><p id="p-7">Besides transformation, there is another mechanism of HGT called gene transfer agents (GTAs), which is encoded in the genomes of many prokaryotes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lang and Beatty 2007</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">McDaniel <em>et al.</em> 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lang <em>et al.</em> 2012</a>). The GTAs are defective virus particles that encapsulate apparently random fragments of the host DNA (rather than the genes encoding the GTA proteins) and mediate the transfer of these fragments into other prokaryotic cells. At face value, the GTAs appear to function as dedicated vehicles for HGT. The existence of these apparently prokaryote-programmed mechanisms of HGT suggests an intriguing possibility that these mechanisms have been maintained by selection in prokaryotes owing to evolutionary advantages offered by HGT—a situation akin to what is currently believed for sexual recombination in eukaryotes (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Michod and Levin 1988</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kondrashov 1993</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Keightley and Otto 2006</a>; see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernstein <em>et al.</em> 1988</a> and <a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Redfield 2001</a> for counteracting views).</p><p id="p-8">What selective advantages could HGT offer to prokaryotes? To address this question, we asked whether an evolutionary advantage of recombination that has been originally suggested for eukaryotes could be extended to the case of HGT in prokaryotes. In particular, we hypothesized that HGT could restore genes that are deleted or inactivated by mutations, thereby preventing the stochastic, irreversible deterioration of genomes in finite populations known as Muller’s ratchet (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Muller 1964</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Felsenstein 1974</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Pamilo <em>et al.</em> 1987</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bell 1988</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Charlesworth <em>et al.</em> 1993</a>; for other types of advantages, see, <em>e.g.</em>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Redfield <em>et al.</em> 1997</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cooper 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Levin and Cornejo 2009</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Vos 2009</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Wylie <em>et al.</em> 2010</a>). This hypothesis is consistent with the observation of extreme genome reduction and accelerated protein evolution in intracellular bacterial parasites and symbionts (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Moran 1996</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Novichkov <em>et al.</em> 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Merhej and Raoult 2011</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">McCutcheon and Moran 2012</a>). Because of continual bottlenecks and sequestration within the host organisms, these bacteria are likely to have small effective population sizes and be effectively isolated from HGT between populations in different hosts. Under these conditions, the impact of Muller’s ratchet increases and could lead to genome reduction, accelerated protein evolution, and possibly eventual extinction (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lynch <em>et al.</em> 1993</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Merhej and Raoult 2011</a>). Therefore, HGT might be an important requirement for the long-term maintenance of genomic information in prokaryotes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Koonin 2011</a>).</p><p id="p-9">However, HGT in prokaryotes differs from eukaryotic recombination both in quantity and in quality. Quantitatively, HGT is relatively less frequent and affects a far smaller portion of chromosomes per recombination event than eukaryotic recombination that is inextricably linked to sexual reproduction (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dubnau 1999</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Feil <em>et al.</em> 2000</a>). Qualitatively, whereas eukaryotic recombination is a direct exchange of DNA sequences between haploid genomes inherited from the two parents, HGT in prokaryotes, in particular, transformation is an indirect exchange of DNA sequences between cells via an eDNA pool (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Redfield 1988</a>). The sources of eDNA can be diverse, but the most common one is likely to be the debris of dead conspecific cells. If the death rate of a cell increases with the number of deleterious mutations in the genome, the genomes of dead cells on average carry more deleterious mutations than those of living cells (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Redfield <em>et al.</em> 1997</a>). Consequently, transformation on average is expected to introduce more deleterious mutations than it would remove. In view of these differences between eukaryotic recombination and HGT in prokaryotes, which potentially nullify the hypothetical benefit of HGT, we sought to assess the net effect of HGT on the operation of Muller’s ratchet by performing stochastic population genetic simulations.</p><p id="p-10">We analyzed a simple evolutionary model of finite prokaryotic populations undergoing HGT. Our model is based on the infinite population model studied by <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Redfield (1988)</a> with several extensions (see the section <em>Materials and Methods</em>). The model was analyzed under the assumption that is most unfavorable to the hypothesized advantage of HGT, namely, that variation in fitness is attributed solely to variation in death rates, and the fittest individuals do not contribute their genomes to the eDNA pool. The results show that, even under this assumption, HGT can prevent the operation of Muller’s ratchet by causing a continual generation of the least-loaded class (<em>i.e.</em>, genomes with the least number of deleterious mutations) from more-loaded classes. Moreover, the prevention of Muller’s ratchet by HGT is substantially more efficient in a subdivided population than in a panmictic population of an equal overall size, provided that the diffusion of eDNA is sufficiently rapid.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-11">Definitions for the symbols used in this article can be found in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463983/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463983/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of symbols</span><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>General description of the model</h3><p id="p-13">The model consists of two components: a population of prokaryotic cells and a pool of eDNA. The population undergoes a standard mutation-selection process (Wright-Fisher model) as well as HGT, through which cells acquire alleles randomly drawn from the eDNA pool. The eDNA pool gains input from the population due to natural cell death and also undergoes spontaneous decay. The details of each of these processes are described in the sections to follow.</p></div><div id="sec-3" class="subsection"><h3>Selection</h3><p id="p-14">The population dynamics was modeled as the discrete-generation Wright-Fisher process with fixed population size <em>N</em>. The genome was modeled as <em>l</em> loci (haploid), each representing a genomic segment that could be replaced by one recombination event (<a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Wylie <em>et al.</em> 2010</a>). The typical lengths of such segments are known to be on the order of 10 kb in several bacteria (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Dubnau 1999</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Feil <em>et al.</em> 2000</a>), and the lengths of bacterial genomes are on the order of 1 Mb. Accordingly, the value of <em>l</em> was set to 100 unless otherwise stated. Each locus was assigned one of the infinite number of possible alleles, each of which was represented by a pair of integers: one integer indicating the locus and the other one indicating the number of deleterious mutations. All deleterious mutations were assumed to cause an equal fitness effect <em>s</em> without epistasis; thus, the relative fitness <em>f</em><sub>i</sub> of a genome with <em>i</em> mutations is <em>f</em><sub>i</sub> = (1 − s)<em><sup>i</sup></em>. This simplified model was adopted so that the effect of HGT on Muller’s ratchet could be assessed without confounding the additional effects caused by epistasis (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Redfield <em>et al.</em> 1997</a>). The speed of Muller’s ratchet is known to depend on the two combinations of parameters, <em>U</em>/<em>s</em> and <em>sN</em> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Neher and Shraiman 2012</a>). Thus, in this study, the value of <em>s</em> was set to 0.01, and the values of <em>U</em> and <em>N</em> (more specifically, <em>sN</em> exp(−<em>U</em>/<em>s</em>) (denoted as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-5.gif"/></noscript></span></span>) and <em>N</em>) were varied to explore the behavior of the model.</p></div><div id="sec-4" class="subsection"><h3>Mutation</h3><p id="p-15">In the model, each genome acquired the Poisson distributed number of new deleterious mutations with the mean <em>U</em> per generation. Beneficial mutations were ignored because we were concerned with the possible deterioration or maintenance of initially well-adapted genomes, for which the frequency of beneficial mutations is likely to be rare (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Maynard Smith 1978</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Charlesworth <em>et al.</em> 1993</a>; cf. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Goyal <em>et al.</em> 2012</a>).</p></div><div id="sec-5" class="subsection"><h3>Horizontal gene transfer</h3><p id="p-16">HGT generally can be categorized into homologous recombination and illegitimate (<em>i.e.</em>, nonhomologous) recombination. Because homologous recombination is much more frequent than nonhomologous recombination (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hulter and Wackernagel 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brigulla and Wackernagel 2010</a>), only the former was included in the model. In each genome, homologous recombination replaced the Poisson distributed number of alleles with mean <em>r</em> per generation per genome with alleles randomly drawn from the eDNA pool (after the replacement the original alleles were discarded).</p></div><div id="sec-6" class="subsection"><h3>Dynamics of the eDNA pool</h3><p id="p-17">The frequency of homologous recombination decreases exponentially with the divergence between DNA sequences (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Watt <em>et al.</em> 1985</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Majewski and Cohan 1999</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Eppley <em>et al.</em> 2007</a>). Thus, the most important source of eDNA for homologous recombination would be the genomes of conspecific cells. For the sake of simplicity, the model assumed that genomes of conspecific cells in the same population were the only source of eDNA.</p><p id="p-18">Prokaryotic cells can release their genomic DNA into the extracellular environment either actively by fratricide, suicide, or DNA secretion, or passively by natural death. However, the model assumed that genomic DNA was released only by natural cell death in order to consider the simplest and most unfavorable condition for the putative advantage of HGT (see the <em>Introduction</em> section).</p><p id="p-19">To incorporate the influx of eDNA as the result of natural cell death, the model assumed that the per-generation probability of an individual with <em>i</em> mutations releasing its genome into the eDNA pool (<em>i.e.</em>, dying) was <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-6.gif"/></noscript></span></span> where <em>n<sub>j</sub></em> is the number of cells with <em>j</em> mutations, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-7.gif"/></noscript></span></span> is the fitness of cells with <em>j</em> mutations normalized by the fitness of the least-loaded class, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-8.gif"/></noscript></span></span> where <em>m</em><sub>LLC</sub> is the number of mutations in the least-loaded class (<em>m</em><sub>LLC</sub> is defined as the integer for which <em>n<sub>m</sub></em> = 0 for <em>m</em> &lt; <em>m</em><sub>LLC</sub> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-9.gif"/></noscript></span></span>).</p><p id="p-20">The aforementioned assumption on ε<em><sub>i</sub></em> was obtained from the following heuristic argument. Let us consider a continuous population dynamics described by <em>dn<sub>i</sub></em>/<em>d<sub>t</sub></em> = <em>bn<sub>i</sub></em> − <em>d<sub>i</sub>n<sub>i</sub></em>, where <em>b</em> and <em>d<sub>i</sub></em> are the birth and death rates, respectively. The absolute fitness per unit time of a cell with <em>i</em> mutations can be expressed as <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-10.gif"/></noscript></span></span>. The relative fitness is expressed as <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-11.gif"/></noscript></span></span> to be consistent with the aforementioned definition of <em>f</em><sub>i</sub> (so that <em>f</em><sub>0</sub> = 1). The fraction (probability) of cells dying per unit time is <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-12.gif"/></noscript></span></span>, which can be expressed as 1 − (1 − <em>D</em><sub>0</sub>)<em>f</em><sub>i</sub>, where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-13.gif"/></noscript></span></span> (<em>D</em><sub>0</sub> is the probability that an individual of the least-loaded class dies). Thus, the normalized contribution to the eDNA pool by a cell with <em>i</em> mutations is <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-14.gif"/></noscript></span></span>. Setting <em>D</em><sub>0</sub> to zero maximizes the number of mutations in the genomes of dead cells (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a>), leading to the expression <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-15.gif"/></noscript></span></span>. This expression is used in the seminal work of Redfield (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Redfield 1988</a>), which considers an infinite population model. In a finite population model, however, the number of mutations in the least-loaded class can increase over time owing to Muller’s ratchet. To keep constant the average number of mutations in the eDNA pool relative to the number of mutations in the least-loaded class (so that the dynamics is stationary over time), the fitness in <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-16.gif"/></noscript></span></span> was normalized by the fitness of the least leaded class <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-17.gif"/></noscript></span></span> to obtain the expression for ε<em><sub>i</sub></em> shown previously. Note that assuming ε<em><sub>i</sub></em> is more unfavorable to the hypothesized advantage of HGT than assuming <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-18.gif"/></noscript></span></span> because <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-19.gif"/></noscript></span></span> is nearly independent of <em>i</em> when <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-20.gif"/></noscript></span></span>.</p><p id="p-21">In addition, the model assumed spontaneous decay of eDNA at rate <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-21.gif"/></noscript></span></span> per cell generation. The eDNA decay rate determines the turnover of the eDNA pool, <em>i.e.</em>, how synchronous the genetic content of the eDNA pool is relative to that of the population (<em>e.g.</em>, when <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-22.gif"/></noscript></span></span>, the entire eDNA pool is replaced with the new input every generation). Although the absolute concentration of eDNA is likely to influence the frequency of HGT (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Levin and Cornejo 2009</a>), in the model this influence was assumed to be absorbed by <em>r</em> so that the value of <em>d</em><sub>eDNA</sub> does not influence the frequency of HGT. To indicate this fact, <em>d</em><sub>eDNA</sub> will be henceforth referred to as the rate of eDNA turnover.</p><p id="p-22">With the aforementioned assumptions, the dynamics of the eDNA pool was simulated as follows. The state of the eDNA pool was described by the vector <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-23.gif"/></noscript></span></span> where <em>e</em><sub>ij</sub> is the copy number of allele <em>j</em> (<em>i.e.</em>, <em>j</em> mutations) from locus <em>i</em> present in the eDNA pool. In each generation, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-24.gif"/></noscript></span></span> reduced by <em>d</em><sub>eDNA</sub><em>e<sub>ij</sub></em> (if <em>d</em><sub>eDNA</sub><em>e<sub>ij</sub></em> &gt; 1, it was normalized to the nearest integer; otherwise, the decrease of <em>e<sub>ij</sub></em> was drawn from a binomial distribution with the number of trials <em>e<sub>ij</sub></em> and success probability <em>d</em><sub>eDNA</sub>). Then, <em>N</em> cells were randomly chosen with replacement from the population of the current generation with the probability ε<em><sub>i</sub></em>. All alleles in the genomes of the chosen cells were added to the eDNA pool, leading to the increase of <em>e<sub>ij</sub></em>.</p></div><div id="sec-7" class="subsection"><h3>Population structure</h3><p id="p-23">The simplest version of the model assumed that the population is genetically isolated from other populations. However, to investigate the effect of genetic exchange between semi-isolated populations, we also considered an extended model in which the population was subdivided into 16 subpopulations of an equal size. The subpopulations were arranged in a two-dimensional square grid (4 × 4 subpopulations) with toroidal boundaries. Each subpopulation exchanged individuals with the four nearest-neighbor subpopulations via random migration. The number of migrants between each pair of neighboring subpopulations was drawn from the binomial distribution with the success probability <em>D</em><sub>pop</sub> and <em>N</em><sub>s</sub>/4 trials where <em>N</em><sub>s</sub> is the size of a subpopulation (<em>N</em><sub>s</sub> = <em>N</em>/16). Each subpopulation had its own pool of eDNA that exchanged alleles with the four nearest-neighbor pools via diffusion. The copy number of each allele diffusing out of a pool was defined as a fraction <em>D</em><sub>eDNA</sub>/4 per direction per generation (if <em>D</em><sub>eDNA</sub><em>e<sub>ij</sub></em>/4 &gt; 1, it was normalized to the nearest integer; otherwise, the numbers of copies diffusing out were drawn from a multinomial distribution with the number of trials <em>e<sub>ij</sub></em> and success probability <em>D</em><sub>eDNA</sub>/4 per direction).</p></div><div id="sec-8" class="subsection"><h3>Organization of the simulation</h3><p id="p-24">The dynamics of the model in one generation was simulated in four steps, in the following order:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-25"><span class="list-label">1. </span>The Wright-Fisher process. In this step, the population of the current generation died and was replaced by that of the next generation.</p></li><li id="list-item-2"><p id="p-26"><span class="list-label">2. </span>Production and decay of the eDNA pools.</p></li><li id="list-item-3"><p id="p-27"><span class="list-label">3. </span>Diffusion of both the microbial populations and eDNA.</p></li><li id="list-item-4"><p id="p-28"><span class="list-label">4. </span>Mutation and HGT.</p></li></ul></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Characterization of Muller’s ratchet</h3><p id="p-29">With HGT disabled, the model is known to have the following family of stationary distributions <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-25.gif"/></noscript></span></span> that are distinguished by the number of deleterious mutations in the least-loaded class <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-26.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-27.gif"/></noscript></span></span>, ⋯, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-28.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-29.gif"/></noscript></span></span> where <em>k</em> is a non-negative integer (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Haigh 1978</a>). A finite-size population cannot stay in any of these distributions indefinitely because of stochasticity. The absence of back mutations implies that <em>m</em><sub>LLC</sub> increases over time, hence the accumulation of deleterious mutations (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Haigh 1978</a>). In general, such accumulation can result from two different processes (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Charlesworth <em>et al.</em> 1993</a>): the fixation of deleterious mutations via genetic drift or the operation of Muller’s ratchet <em>per se</em>, <em>i.e.</em>, the extinction of the least-loaded class as the result of stochastic fluctuations. Muller’s ratchet can be reversed by recombination only if a mutation has not been fixed (unless eDNA turnover is very slow, as described later in this article). Thus, it is beneficial to know through which of these processes the accumulation of mutations occurs in the model. Thus, we first analyzed the model in the absence of HGT (<em>i.e.</em>, <em>r</em> = 0). The results of the analysis essentially confirmed the conclusion of <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charlesworth and Charlesworth (1997)</a> with additional data. Although these results in part reproduce the work of <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Charlesworth and Charlesworth (1997)</a>, we present them here because they serve as the baseline with which to compare the results presented in the later sections.</p><p id="p-30">The simulations show that the tempo and mode of mutation accumulation in the model depend on whether the value of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-30.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-31.gif"/></noscript></span></span>) is much greater than unity or not, as expected from previous work (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Rouzine <em>et al.</em> 2008</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Neher and Shraiman 2012</a>). For <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-32.gif"/></noscript></span></span> (slow accumulation regime), <em>m</em><sub>LLC</sub> and the number of mutations fixed in the population (<em>m</em><sub>fix</sub>) increase over time in a very similar fashion (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), so that the number of segregating mutations in the least-loaded class (<em>m</em><sub>LLC</sub> − <em>m</em><sub>fix</sub>) is at most one. Each increase in <em>m</em><sub>LLC</sub> is followed by an increase in <em>m</em><sub>fix</sub> with a delay of approximately a few thousand generations before the next increase occurs to <em>m</em><sub>LLC</sub> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Thus, the extinction of the least-loaded class (<em>i.e.</em>, a click of Muller’s ratchet) always precedes the fixation of a mutation (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Charlesworth and Charlesworth 1997</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Operation of Muller&#x2019;s ratchet in the model in the absence of HGT (). (A, B, D, E) The number of deleterious mutations is plotted against time: the population average (gray); the number of mutations fixed in the population (black); and the number of mutations in the least-loaded class (red). (C and F) The average gene diversity of the least-loaded class  (green), that of the one but the least-loaded class  (blue), and the eDNA quality  (orange) are plotted against time (see the main text for how these quantities are defined). Parameters were as follows: s = 0.01, N = 105, deDNA = 1, l = 100. In the slow ratchet regime, ,  (A&#x2212;C); in the fast ratchet regime, ,  (D&#x2212;F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Operation of Muller&#x2019;s ratchet in the model in the absence of HGT (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). (A, B, D, E) The number of deleterious mutations is plotted against time: the population average (gray); the number of mutations fixed in the population (black); and the number of mutations in the least-loaded class (red). (C and F) The average gene diversity of the least-loaded class &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green), that of the one but the least-loaded class &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-35&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (blue), and the eDNA quality &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (orange) are plotted against time (see the main text for how these quantities are defined). Parameters were as follows: &lt;em&gt;s&lt;/em&gt; = 0.01, &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt;, &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; = 1, &lt;em&gt;l&lt;/em&gt; = 100. In the slow ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A&#x2212;C); in the fast ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-39&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (D&#x2212;F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F1.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/2/325/F1.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463997" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Operation of Muller’s ratchet in the model in the absence of HGT (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-33.gif"/></noscript></span></span>). (A, B, D, E) The number of deleterious mutations is plotted against time: the population average (gray); the number of mutations fixed in the population (black); and the number of mutations in the least-loaded class (red). (C and F) The average gene diversity of the least-loaded class <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-34.gif"/></noscript></span></span> (green), that of the one but the least-loaded class <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-35.gif"/></noscript></span></span> (blue), and the eDNA quality <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-36.gif"/></noscript></span></span> (orange) are plotted against time (see the main text for how these quantities are defined). Parameters were as follows: <em>s</em> = 0.01, <em>N</em> = 10<sup>5</sup>, <em>d</em><sub>eDNA</sub> = 1, <em>l</em> = 100. In the slow ratchet regime, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-37.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-38.gif"/></noscript></span></span> (A−C); in the fast ratchet regime, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-39.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F1/embed/mml-math-40.gif"/></noscript></span></span> (D−F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The dynamics of mutation accumulation was further analyzed by measuring the average gene diversity (“heterozygosity”) of the least-loaded class (denoted <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-41.gif"/></noscript></span></span>) and that of one but the least-loaded class (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-42.gif"/></noscript></span></span>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Nei 1987</a>). <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-43.gif"/></noscript></span></span>is defined as <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-44.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-45.gif"/></noscript></span></span> is the frequency of allele <em>j</em> in locus <em>i</em> in the least-loaded class (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Nei 1987</a>). <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-46.gif"/></noscript></span></span> is defined in a similar manner. The results show that <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-48.gif"/></noscript></span></span> display the following dynamics. When Muller’s ratchet clicks (<em>i.e.</em>, when the value of <em>m</em><sub>LLC</sub> increases), one but the least-loaded class becomes the new least-loaded class by definition, so that the value of <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-49.gif"/></noscript></span></span> jumps to the value of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-50.gif"/></noscript></span></span> immediately prior to the click of the ratchet (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). This value of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-51.gif"/></noscript></span></span> is always nearly 0.02, indicating that, immediately prior to a click of Muller’s ratchet, one but the least-loaded class consists of <em>l</em> genotypes (one mutation in one of the <em>l</em> loci), whose frequencies roughly equal to each other (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-52.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-53.gif"/></noscript></span></span>). Thus, immediately after the ratchet clicks, the new least-loaded class consists of <em>l</em> genotypes. Because these genotypes have an equal fitness value, the gene diversity <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-54.gif"/></noscript></span></span> decreases to zero over time owing to genetic drift (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Almost simultaneously with <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-55.gif"/></noscript></span></span> hitting zero, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-56.gif"/></noscript></span></span> reaches a steady state value of approximately 0.02 mentioned previously (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>), and the value of <em>m</em><sub>fix</sub> increases, indicating the fixation of a deleterious mutation (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). In summary, the accumulation of mutations occurs as a cycle of three distinct steps (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>):</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic diagram depicting accumulation of mutations in the model. Horizontal bars indicate the genomes of the least-loaded class (genomes of the other classes in the population are not shown). Crosses on the bars indicate deleterious mutations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic diagram depicting accumulation of mutations in the model. Horizontal bars indicate the genomes of the least-loaded class (genomes of the other classes in the population are not shown). Crosses on the bars indicate deleterious mutations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F2.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/2/325/F2.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463927" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Schematic diagram depicting accumulation of mutations in the model. Horizontal bars indicate the genomes of the least-loaded class (genomes of the other classes in the population are not shown). Crosses on the bars indicate deleterious mutations.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-2"><li id="list-item-5"><p id="p-34"><span class="list-label">1. </span>Extinction of the least-loaded class due to stochasticity (<em>i.e.</em>, Muller’s ratchet);</p></li><li id="list-item-6"><p id="p-35"><span class="list-label">2. </span>Gradual decline of the genetic diversity in the new least-loaded class due to genetic drift;</p></li><li id="list-item-7"><p id="p-36"><span class="list-label">3. </span>Eventual fixation of a deleterious mutation in the entire population.</p></li></ul><p id="p-37">For <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-57.gif"/></noscript></span></span>, mutations accumulate more rapidly than when <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-58.gif"/></noscript></span></span> (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1D</a>). The value of <em>m</em><sub>LLC</sub> is incremented multiple times before the value of <em>m</em><sub>fix</sub> reaches the value of <em>m</em><sub>LLC</sub> (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1E</a>). This result indicates that, unlike the case of <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-59.gif"/></noscript></span></span>, cycles of mutation accumulation can overlap with each other in time: a new cycle can begin with a click of Muller’s ratchet (<em>i.e.</em>, step 1 in the previous paragraph) before the previous cycle undergoes the fixation of a mutation (<em>i.e.</em>, step 3). Moreover, the dynamics of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-60.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-61.gif"/></noscript></span></span> are no more synchronized, in that the decrease of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-62.gif"/></noscript></span></span> after a click of Muller’s ratchet is delayed with respect to the decrease of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-63.gif"/></noscript></span></span> (cf. <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Neher and Shraiman 2012</a>). Likewise, the increase of <em>m</em><sub>fix</sub> also is delayed with respect to the decrease of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-64.gif"/></noscript></span></span>. These results indicate that the decline of gene diversity after a click of Muller’s ratchet (<em>i.e.</em>, step 2) is a sequential process, in that gene diversity is consecutively lost in different genotype classes distinguished by the number of mutations (<em>i.e.</em>, first, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-65.gif"/></noscript></span></span> decreases to zero, and then <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-66.gif"/></noscript></span></span> decreases to a steady-state level, and so on). Taken together, the results indicate that the three-step picture of mutation accumulation described in the previous paragraph remains essentially valid in the rapid ratchet regime (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-67.gif"/></noscript></span></span>) as well.</p><p id="p-38">Therefore, Muller’s ratchet is the dominant process leading to the accumulation of mutations in the model with HGT disabled (rather than the fixation by drift of mutations; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Charlesworth and Charlesworth 1997</a>). However, a click of Muller’s ratchet eventually results in the fixation of a mutation, so that the number of fixed mutations (<em>m</em><sub>fix</sub>) far exceeds the number of segregating mutations in the least-loaded class (<em>m</em><sub>LLC</sub> − <em>m</em><sub>fix</sub>). This situation is in stark contrast with the result with a diploid, sexual model that assumes no recombination and weak dominance, which shows that the number of fixed mutations is much smaller than the number of segregating mutations (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Charlesworth <em>et al.</em> 1993</a>). The most probable cause of this discrepancy is the prevention of mutation fixation in diploid populations due to homozygote disadvantage, which is absent in our haploid model. The resulting paucity of segregating mutations in the haploid model seems to work against the putative advantage of HGT; later we show how this limitation might be overcome by population subdivision.</p></div><div id="sec-11" class="subsection"><h3>Effect of HGT on Muller’s ratchet</h3><p id="p-39">We next examined the effect of HGT on Muller’s ratchet. For simplicity, a rapid turnover of eDNA was assumed by setting <em>d</em><sub>eDNA</sub> = 1 (the effect of slow eDNA turnover is presented in the next section). In this case, assuming a very large population, the average number of mutations in the eDNA pool per <em>l</em> loci (<em>i.e.</em>, per genome) can be approximated by <em>U</em>/<em>s</em> + 1, where <em>U</em>/<em>s</em> is the average number of mutations in the population at the steady state with <em>m</em><sub>LLC</sub> = 0 (<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Redfield <em>et al.</em> 1997</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a>). Because the mean number of mutations in the eDNA pool is greater than that in the population, HGT on average introduces more deleterious mutations into the population than it removes and so decreases the steady-state population size of the least-loaded class.</p><p id="p-40">To assess the net effect of HGT on Muller’s ratchet, we measured the rate of mutation accumulation Δ<em>m</em><sub>fix</sub> /Δ<em>t</em> as a function of <em>r</em> (HGT rate) for different combinations of <em>N</em> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-68.gif"/></noscript></span></span> values. The results show that HGT prevents the accumulation of mutations (<em>i.e.</em>, Δ<em>m</em><sub>fix</sub> /Δ<em>t</em> = 0) if its frequency is sufficiently high (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Moreover, the frequency of HGT required to stop Muller’s ratchet decreases with the increase of <em>N</em>, indicating that the preventive effect of HGT increases with <em>N</em>. This trend is seen even in the slow ratchet regime (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-69.gif"/></noscript></span></span>), in which mutation accumulation accelerates with the increase of <em>N</em> in the absence of HGT (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) (note that because <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-70.gif"/></noscript></span></span> and <em>s</em> are fixed, <em>U</em> was set to a greater value for a greater value of <em>N</em>; thus, mutation accumulation can accelerate with the increase of <em>N</em>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effect of HGT on the operation of Muller&#x2019;s ratchet. The rate of mutation accumulation &#x394;mfix/&#x394;t is plotted as a function of the rate of HGT r. The accumulation rate was estimated as mfix/t, where mfix is the number of fixed mutations at the end of a simulation and t is the duration of a simulation (e.g., t &gt;106 for N = 106 and t &gt;109 for N = 2 &#xD7; 103). The error bars indicate standard error of means estimated as . The parameters were as follows: s = 0.01, deDNA = 1, l = 100. In the slow ratchet regime,  (filled circles): N = 106 and  (black), N = 105 and  (red), N = 104 and  (green), N = 2 &#xD7; 103  (blue). In the fast ratchet regime,  (open circles): N = 106 and  (black), N = 105 and  (red), N = 104 and  (green),  and  (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The effect of HGT on the operation of Muller&#x2019;s ratchet. The rate of mutation accumulation &#x394;&lt;em&gt;m&lt;/em&gt;&lt;sub&gt;fix&lt;/sub&gt;/&#x394;&lt;em&gt;t&lt;/em&gt; is plotted as a function of the rate of HGT &lt;em&gt;r&lt;/em&gt;. The accumulation rate was estimated as &lt;em&gt;m&lt;/em&gt;&lt;sub&gt;fix&lt;/sub&gt;/&lt;em&gt;t&lt;/em&gt;, where &lt;em&gt;m&lt;/em&gt;&lt;sub&gt;fix&lt;/sub&gt; is the number of fixed mutations at the end of a simulation and &lt;em&gt;t&lt;/em&gt; is the duration of a simulation (&lt;em&gt;e.g.&lt;/em&gt;, &lt;em&gt;t&lt;/em&gt; &gt;10&lt;sup&gt;6&lt;/sup&gt; for &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;6&lt;/sup&gt; and &lt;em&gt;t&lt;/em&gt; &gt;10&lt;sup&gt;9&lt;/sup&gt; for &lt;em&gt;N&lt;/em&gt; = 2 &#xD7; 10&lt;sup&gt;3&lt;/sup&gt;). The error bars indicate standard error of means estimated as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The parameters were as follows: &lt;em&gt;s&lt;/em&gt; = 0.01, &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; = 1, &lt;em&gt;l&lt;/em&gt; = 100. In the slow ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-72&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (filled circles): &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;6&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (black), &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (red), &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;4&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-75&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green), &lt;em&gt;N&lt;/em&gt; = 2 &#xD7; 10&lt;sup&gt;3&lt;/sup&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-76&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (blue). In the fast ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-77&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (open circles): &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;6&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-78&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (black), &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-79&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (red), &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;4&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-80&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green), &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-81&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-82&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F3.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/2/325/F3.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463903" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">The effect of HGT on the operation of Muller’s ratchet. The rate of mutation accumulation Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> is plotted as a function of the rate of HGT <em>r</em>. The accumulation rate was estimated as <em>m</em><sub>fix</sub>/<em>t</em>, where <em>m</em><sub>fix</sub> is the number of fixed mutations at the end of a simulation and <em>t</em> is the duration of a simulation (<em>e.g.</em>, <em>t</em> &gt;10<sup>6</sup> for <em>N</em> = 10<sup>6</sup> and <em>t</em> &gt;10<sup>9</sup> for <em>N</em> = 2 × 10<sup>3</sup>). The error bars indicate standard error of means estimated as <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-71.gif"/></noscript></span></span>. The parameters were as follows: <em>s</em> = 0.01, <em>d</em><sub>eDNA</sub> = 1, <em>l</em> = 100. In the slow ratchet regime, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-72.gif"/></noscript></span></span> (filled circles): <em>N</em> = 10<sup>6</sup> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-73.gif"/></noscript></span></span> (black), <em>N</em> = 10<sup>5</sup> and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-74.gif"/></noscript></span></span> (red), <em>N</em> = 10<sup>4</sup> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-75.gif"/></noscript></span></span> (green), <em>N</em> = 2 × 10<sup>3</sup> <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-76.gif"/></noscript></span></span> (blue). In the fast ratchet regime, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-77.gif"/></noscript></span></span> (open circles): <em>N</em> = 10<sup>6</sup> and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-78.gif"/></noscript></span></span> (black), <em>N</em> = 10<sup>5</sup> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-79.gif"/></noscript></span></span> (red), <em>N</em> = 10<sup>4</sup> and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-80.gif"/></noscript></span></span> (green), <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-81.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F3/embed/mml-math-82.gif"/></noscript></span></span> (blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">To understand why HGT can prevent Muller’s ratchet despite its average deleterious effect due to the high mutation load of eDNA that comes from dead cells, we considered the following simple Wright-Fisher model. The model assumed two populations: the population of the least-loaded class <em>n</em><sub>0</sub> and that of the other classes <em>n</em><sub>1</sub> (the model is thus equivalent to one-locus, two-allele, haploid model). The total population size N = <em>n</em><sub>0</sub> + <em>n</em><sub>1</sub> was constant. The value of <em>n</em><sub>0</sub> at time <em>t</em> + 1 was drawn from a binomial distribution with <em>N</em> trials and success probability <em>p</em>′ defined as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-83.gif"/></noscript></span></span>where <em>p</em> = <em>n</em><sub>0</sub>/<em>N</em> and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-84.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-85.gif"/></noscript></span></span>) at time <em>t</em>, <em>s</em>′ is the fitness effect of mutation, <em>U</em> is the mutation rate from <em>n</em><sub>0</sub> to <em>n</em><sub>1</sub> (back mutations ignored), <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-86.gif"/></noscript></span></span> is the average fitness, and <em>r</em><sub>01</sub> and <em>r</em><sub>10</sub> are the rates of HGT. For simplicity, HGT was assumed to convert <em>n</em><sub>0</sub> into <em>n</em><sub>1</sub> at rate <em>r</em><sub>01</sub> and, conversely, <em>n</em><sub>1</sub> into <em>n</em><sub>0</sub> at rate <em>r</em><sub>10</sub>. The effect of eDNA was implicitly incorporated into the model by the choice of parameters <em>r</em><sub>01</sub> and <em>r</em><sub>10</sub> as follows. If <em>r</em><sub>01</sub> and <em>r</em><sub>10</sub> satisfy the following condition, HGT decreases the steady-state level of <em>n</em><sub>0</sub> compared with the steady-state level reached in the absence of HGT (<em>i.e.</em>, <em>r</em><sub>01</sub> = <em>r</em><sub>10</sub> = 0):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-87.gif"/></noscript></span></span>(the left-hand side of this inequality is the per-generation conversion of <em>n</em><sub>1</sub> into <em>n</em><sub>0</sub>, whereas the right-hand side is that of <em>n</em><sub>0</sub> into <em>n</em><sub>1</sub>, when <em>n</em><sub>0</sub> and <em>n</em><sub>1</sub> are set to the steady-state level achieved in the absence of HGT).</p><p id="p-43">With this simplified model, we asked whether HGT facilitated or antagonized the survival of <em>n</em><sub>0</sub>. The mean time to extinction of <em>n</em><sub>0</sub>, which is a proxy for Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> in the full model, was measured as a function of <em>r</em><sub>01</sub> and <em>r</em><sub>10</sub> for different combinations of <em>N</em> and <em>U</em>′ values. The results show that HGT increases the mean time to extinction of <em>n</em><sub>0</sub> whether or not HGT decreases the steady state level of <em>n</em><sub>0</sub>, as long as the per-generation production of the least-loaded class by HGT (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-88.gif"/></noscript></span></span>) is on the order of unity or greater (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; note that <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-89.gif"/></noscript></span></span> for the parameters used in the simulations). This result makes intuitive sense: the least-loaded class cannot go extinct if it is continually produced by HGT in every generation; and as long as such production is guaranteed, the steady-state population size of <em>n</em><sub>0</sub> is immaterial to the extinction of <em>n</em><sub>0</sub>. To check whether this explanation also applies to the full model, we investigated the consistency between the two models from a few different angles as described below.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mean time to extinction of the least-loaded class in the simplified Fisher-Wright model. Below the diagonal lines, HGT decreases the steady-state population size of the least-loaded class (i.e., the condition  is fulfilled). The mean time to extinction is normalized by that obtained in the absence of HGT (i.e., ). The parameters were as follows: , and  where . N = 104 and  (A); N = 105 and  (B); N = 106 and  (C). Each data point was obtained as an average of 100 simulation runs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The mean time to extinction of the least-loaded class in the simplified Fisher-Wright model. Below the diagonal lines, HGT decreases the steady-state population size of the least-loaded class (&lt;em&gt;i.e.&lt;/em&gt;, the condition &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-90&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is fulfilled). The mean time to extinction is normalized by that obtained in the absence of HGT (&lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-91&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The parameters were as follows: &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-92&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-93&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-94&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;4&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-95&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A); &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-96&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B); &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;6&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (C). Each data point was obtained as an average of 100 simulation runs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F4.medium.gif" width="440" height="125"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/2/325/F4.medium.gif" width="440" height="125"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463929" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">The mean time to extinction of the least-loaded class in the simplified Fisher-Wright model. Below the diagonal lines, HGT decreases the steady-state population size of the least-loaded class (<em>i.e.</em>, the condition <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-90.gif"/></noscript></span></span> is fulfilled). The mean time to extinction is normalized by that obtained in the absence of HGT (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-91.gif"/></noscript></span></span>). The parameters were as follows: <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-92.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-93.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-94.gif"/></noscript></span></span>. <em>N</em> = 10<sup>4</sup> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-95.gif"/></noscript></span></span> (A); <em>N</em> = 10<sup>5</sup> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-96.gif"/></noscript></span></span> (B); <em>N</em> = 10<sup>6</sup> and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F4/embed/mml-math-97.gif"/></noscript></span></span> (C). Each data point was obtained as an average of 100 simulation runs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">First, the simplified model indicates that the required frequency of HGT to prevent Muller’s ratchet decreases as <em>N</em> increases because increasing <em>N</em> increases the production rate of the least-loaded class (with <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-98.gif"/></noscript></span></span> kept constant). The same result is produced by the full model as already described previously.</p><p id="p-46">Next, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> shows that whether HGT increases the time to extinction depends on the value of <em>r</em><sub>10</sub> much more strongly than on the value of <em>r</em><sub>01</sub> (as seen from the fact that the boundary between yellow and red regions is nearly horizontal). We thus hypothesized that, in the full model, HGT prevents Muller’s ratchet most effectively when the probability of HGT producing the least-loaded class is maximized. This probability can be approximated by the probability <em>P</em><sub>10</sub> that an HGT event transforms one but the least-loaded class (<span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-99.gif"/></noscript></span></span>) into the least-loaded class (<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-100.gif"/></noscript></span></span>), because the contributions from other mutant classes likely involve more than one HGT event. <em>P</em><sub>10</sub> can be calculated as <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-101.gif"/></noscript></span></span> under the assumption that <em>N</em> is very large (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a>). <em>P</em><sub>10</sub> takes the maximum value when <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-102.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-103.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-104.gif"/></noscript></span></span>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a>). The non-monotonic dependency of <em>P</em><sub>10</sub> on <em>l</em> can be intuitively understood as follows: decreasing <em>l</em> can decrease <em>P</em><sub>10</sub> because it increases the probability that an allele randomly drawn from the eDNA pool contains a mutation that is not carried by the least-loaded class (if <em>l</em> = 1, this probability is one). However, increasing <em>l</em> also can decrease <em>P</em><sub>10</sub> because it decreases the probability that HGT occurs to the very locus in which one but the least-loaded class has the mutation not carried by the least-loaded class. Therefore, <em>P</em><sub>10</sub> takes the maximum value at an intermediate value of <em>l</em>. By contrast, the probability <em>P</em><sub>01</sub> that HGT introduces one or more mutations in the least-loaded class is <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-105.gif"/></noscript></span></span> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a>), which is a monotonically decreasing function of <em>l</em> because increasing <em>l</em> decreases the probability that an allele randomly drawn from the eDNA pool contains mutations. Thus, the aforementioned hypothesis can be reformulated as follows: Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> reaches the minimum value when <em>l</em> assumes the value <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-106.gif"/></noscript></span></span>. To examine this hypothesis, we measured Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> as a function of <em>l</em> for various combinations of <em>N</em> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-107.gif"/></noscript></span></span> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The results show that Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> takes the minimum value when the value of <em>l</em> is on the same order of magnitude as <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-108.gif"/></noscript></span></span> (around 10 rather than 100), in accord with the hypothesis.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The normalized rate of mutation accumulation as a function of the number of loci l. The reference rate is set to the rate obtained for l = 100. The value of l that maximizes the probability P10 (that HGT converts one but the least-loaded class to the least-loaded class) is indicated as  in the graph. The parameters and color coding are the same as in Figure 3 (except for l)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The normalized rate of mutation accumulation as a function of the number of loci &lt;em&gt;l&lt;/em&gt;. The reference rate is set to the rate obtained for &lt;em&gt;l&lt;/em&gt; = 100. The value of &lt;em&gt;l&lt;/em&gt; that maximizes the probability &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;10&lt;/sub&gt; (that HGT converts one but the least-loaded class to the least-loaded class) is indicated as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-109&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the graph. The parameters and color coding are the same as in Figure 3 (except for &lt;em&gt;l&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F5.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/2/325/F5.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463972" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">The normalized rate of mutation accumulation as a function of the number of loci <em>l</em>. The reference rate is set to the rate obtained for <em>l</em> = 100. The value of <em>l</em> that maximizes the probability <em>P</em><sub>10</sub> (that HGT converts one but the least-loaded class to the least-loaded class) is indicated as <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F5/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F5/embed/mml-math-109.gif"/></noscript></span></span> in the graph. The parameters and color coding are the same as in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> (except for <em>l</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Finally, the simplified model shows that <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-110.gif"/></noscript></span></span> is sufficient for HGT to prevent the extinction of <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-111.gif"/></noscript></span></span>. To check if a similar condition applies to the full model, we calculated the per-generation production of the least-loaded class by HGT occurring to one but the least-loaded class (<span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-112.gif"/></noscript></span></span>). For <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-113.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-114.gif"/></noscript></span></span> sharply drops at <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-115.gif"/></noscript></span></span> for <em>N</em> = 10<sup>6</sup> and 10<sup>5</sup> (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). The values of <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-116.gif"/></noscript></span></span> in these cases are approximately 0.6 and 0.4 for <em>N</em> = 10<sup>6</sup> and 10<sup>5</sup>, respectively, in qualitative agreement with the expectation that <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-117.gif"/></noscript></span></span> be on the order of unity (note that <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-118.gif"/></noscript></span></span> is expected because the contributions from the more-loaded classes are ignored and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-119.gif"/></noscript></span></span> is a stronger condition than <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-120.gif"/></noscript></span></span>). For <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-121.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-122.gif"/></noscript></span></span> sharply drops at <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-123.gif"/></noscript></span></span> for <em>N</em> = 10<sup>6</sup> and 10<sup>5</sup>. The values of <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-124.gif"/></noscript></span></span> in these cases are approximately 0.8 and 0.6 for <em>N</em> = 10<sup>6</sup> or 10<sup>5</sup>, respectively, again in qualitative agreement with the expectation. (The results were also consistent for smaller values of <em>N</em>, at least in the orders of magnitude.)</p><p id="p-49">Taken together, the consistency of the results indicates that the simplified model captures, at least qualitatively, the mechanism by which HGT prevents Muller’s ratchet in the full model. Namely, HGT can prevent Muller’s ratchet even if it decreases the steady-state population size of the least-loaded class because it can hinder the extinction of the least-loaded class by continually generating it from the more-loaded classes.</p></div><div id="sec-12" class="subsection"><h3>Effect of the eDNA turnover rate on the prevention of Muller’s ratchet by HGT</h3><p id="p-50">We further investigated the effect of the eDNA turnover rate on the ability of HGT to prevent Muller’s ratchet. As described previously, the accumulation of mutations in the model is initiated by a click of Muller’s ratchet followed by the fixation of a mutation. Once a mutation is fixed (<em>i.e.</em>, once a good allele is lost), HGT cannot undo it because the only source of HGT in the model is eDNA derived from the members of the same population. However, slow turnover of eDNA can delay the loss of good alleles from the eDNA pool and thereby might help HGT prevent the accumulation of mutations.</p><p id="p-51">To examine this possibility, we measured <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-125.gif"/></noscript></span></span> as a function of <em>d</em><sub>eDNA</sub> for various combinations of <em>N</em> and <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-126.gif"/></noscript></span></span> values (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The results show that Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> decreases with the decrease of <em>d</em><sub>eDNA</sub>. However, for this effect to be significant, eDNA turnover has to be slower than the population turnover (<em>i.e.</em>, generation time) by more than three orders of magnitude (<em>d</em><sub>eDNA</sub> &lt; 10<sup>−3</sup>), a condition that appears to be unrealistic in natural environments (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Nielsen <em>et al.</em> 2007</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Pietramellara <em>et al.</em> 2009</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of slow eDNA turnover on the prevention of Muller&#x2019;s ratchet by HGT. The rate of mutation accumulation &#x394;mfix/&#x394;t is plotted as a function of the eDNA turnover rate deDNA. The parameters and color coding are the same as in Figure 3 (except for deDNA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of slow eDNA turnover on the prevention of Muller&#x2019;s ratchet by HGT. The rate of mutation accumulation &#x394;&lt;em&gt;m&lt;/em&gt;&lt;sub&gt;fix&lt;/sub&gt;/&#x394;&lt;em&gt;t&lt;/em&gt; is plotted as a function of the eDNA turnover rate &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt;. The parameters and color coding are the same as in Figure 3 (except for &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F6.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/2/325/F6.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463962" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-52" class="first-child">Effect of slow eDNA turnover on the prevention of Muller’s ratchet by HGT. The rate of mutation accumulation Δ<em>m</em><sub>fix</sub>/Δ<em>t</em> is plotted as a function of the eDNA turnover rate <em>d</em><sub>eDNA</sub>. The parameters and color coding are the same as in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> (except for <em>d</em><sub>eDNA</sub>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">To understand why such a severe condition applied, we measured the “quality” of eDNA defined as <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-127.gif"/></noscript></span></span> where <em>x<sub>ki</sub></em> is the fraction of the least-loaded class having allele <em>i</em> (<em>i.e.</em>, <em>i</em> mutations) in locus <em>k</em>, and <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-128.gif"/></noscript></span></span> is the fraction of allele <em>j</em> from locus <em>k</em> in the eDNA pool. <em>q</em><sub>eDNA</sub> indicates the potential of the eDNA pool to remove deleterious mutations in the genomes of the least-loaded class (<em>e.g.</em>, <em>q</em><sub>eDNA</sub> is zero if the eDNA pool contains no allele that can remove mutations in the least-loaded class). <em>q</em><sub>eDNA</sub> was measured in the absence of HGT for various values of <em>d</em><sub>eDNA</sub>. The results show that, in the slow ratchet regime (<span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-129.gif"/></noscript></span></span>), the dynamics of <em>q</em><sub>eDNA</sub> closely follows that of <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-130.gif"/></noscript></span></span> when eDNA turnover is not very slow (<em>d</em><sub>eDNA</sub> ≥ 10<sup>−2</sup>) (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7, A and B</a>). When eDNA turnover is very slow (<em>d</em><sub>eDNA</sub> ≤ 10<sup>−3</sup>), the decay of <em>q</em><sub>eDNA</sub> lags behind the decline of <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-131.gif"/></noscript></span></span> (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7C</a>). This result indicates that eDNA turnover has to be slower than the decline of <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-132.gif"/></noscript></span></span> to delay the decay of <em>q</em><sub>eDNA</sub> significantly. Because the decrease of <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-133.gif"/></noscript></span></span>is attributed to genetic drift as mentioned previously, its timescale is determined by the population size of the least-loaded class, which was approximately 1000 in the simulations for the slow ratchet regime (<span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-134.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-135.gif"/></noscript></span></span>). Thus, <em>d</em><sub>eDNA</sub> must be &lt;10<sup>−3</sup> to cause any significant effect.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of slow eDNA turnover on the quality of eDNA qeDNA . The average gene diversity of the least-loaded class  (green), that of the one but the least-loaded class  (blue), and the quality of eDNA qeDNA (orange) are plotted against time for various values of eDNA turnover rate deDNA. The parameters were as follows: l = 100, s = 0.01, deDNA = 0.1 (A and C), 0.01 (B and E), and 0.001 (C and F). In the slow ratchet regime, , N = 105 and (A&#x2212;C). In the fast ratchet regime, , N = 105, and  (D&#x2212;F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Effect of slow eDNA turnover on the quality of eDNA &lt;em&gt;q&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; . The average gene diversity of the least-loaded class &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green), that of the one but the least-loaded class &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (blue), and the quality of eDNA &lt;em&gt;q&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; (orange) are plotted against time for various values of eDNA turnover rate &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt;. The parameters were as follows: &lt;em&gt;l&lt;/em&gt; = 100, &lt;em&gt;s&lt;/em&gt; = 0.01, &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; = 0.1 (A and C), 0.01 (B and E), and 0.001 (C and F). In the slow ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-139&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;(A&#x2212;C). In the fast ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-140&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt;, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-141&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (D&#x2212;F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F7.medium.gif" width="440" height="396"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/2/325/F7.medium.gif" width="440" height="396"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-54" class="first-child">Effect of slow eDNA turnover on the quality of eDNA <em>q</em><sub>eDNA</sub> . The average gene diversity of the least-loaded class <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-136.gif"/></noscript></span></span> (green), that of the one but the least-loaded class <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-137.gif"/></noscript></span></span> (blue), and the quality of eDNA <em>q</em><sub>eDNA</sub> (orange) are plotted against time for various values of eDNA turnover rate <em>d</em><sub>eDNA</sub>. The parameters were as follows: <em>l</em> = 100, <em>s</em> = 0.01, <em>d</em><sub>eDNA</sub> = 0.1 (A and C), 0.01 (B and E), and 0.001 (C and F). In the slow ratchet regime, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-138.gif"/></noscript></span></span>, <em>N</em> = 10<sup>5</sup> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-139.gif"/></noscript></span></span>(A−C). In the fast ratchet regime, <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-140.gif"/></noscript></span></span>, <em>N</em> = 10<sup>5</sup>, and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F7/embed/mml-math-141.gif"/></noscript></span></span> (D−F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">The aforementioned argument might imply that the effect of slow eDNA turnover could be more significant if the population size of the least-loaded class is small. However, this turns out not to be the case. In the fast ratchet regime (<span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-142.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-143.gif"/></noscript></span></span>), the dynamics of <em>q</em><sub>eDNA</sub> no more closely follows the dynamics of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-144.gif"/></noscript></span></span>, and it still takes more than 500 generations for <em>q</em><sub>eDNA</sub> to decrease to zero when eDNA turnover is rapid (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7, D−F</a>). Therefore, <em>d</em><sub>eDNA</sub> still has to be smaller than 1/500 to have any significant effect (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a> contains a more detailed explanation of the results described in this and the previous paragraphs).</p><p id="p-56">In summary, the fixation of mutations is an evolutionary process that is driven by selection and drift and thus occurs on relatively long timescales compared with the generation time of individuals, so that eDNA turnover has to be commensurately slow to have any appreciable effect.</p></div><div id="sec-13" class="subsection"><h3>Effect of population subdivision on the prevention of Muller’s ratchet by HGT</h3><p id="p-57">Next, we considered population subdivision. On the one hand, population subdivision can accelerate accumulation of mutations because it enhances the effect of genetic drift as shown in previous studies (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Higgins and Lynch 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Combadao <em>et al.</em> 2007</a>). On the other hand, subdivision might help HGT prevent Muller’s ratchet as follows. Let us suppose that the isolation between subpopulations is so strong that the accumulation of mutations in different subpopulations is independent of each other in the absence of HGT. In this case, a mutation must be fixed in every subpopulation independently before being fixed in the entire population. This situation is likely to lead to an increased number of segregating mutations when the entire population is considered (rather than when each subpopulation is considered separately). The increased number of segregating mutations by itself does not affect accumulation of mutations if recombination requires direct contact between individuals as in many animals. HGT, however, does not require such direct contact. If subpopulations share a common eDNA pool because of the rapid transport of DNA molecules between their habitats, increasing the number of segregating mutations is expected to increase the number of mutations that are introduced or removed by HGT events. This should enhance the preventive effect of HGT on Muller’s ratchet because removal of mutations by HGT is more important than introduction of mutations by HGT according to the simplified Wright-Fisher model presented above. The key question is whether this potential advantage of population subdivision can outweigh the disadvantage of population subdivision associated with the enhanced genetic drift.</p><p id="p-58">To address the aforementioned question, we measured <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-145.gif"/></noscript></span></span> as a function of population migration rate (<em>D</em><sub>pop</sub>) in a model incorporating population subdivision (<em>Materials and Methods</em>). <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-146.gif"/></noscript></span></span> was defined as the average of <em>m</em><sub>fix</sub> measured within each subpopulation. For simplicity, all subpopulations were assumed to share a common eDNA pool (<em>i.e.</em>, <em>D</em><sub>eDNA</sub> = ∞; we will consider finite <em>D</em><sub>eDNA</sub> later). The results show that, in the absence of HGT (<em>r</em> = 0), the system becomes more prone to Muller’s ratchet (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-147.gif"/></noscript></span></span> increases) as <em>D</em><sub>pop</sub> decreases (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8, A and B</a>). This outcome is expected, given the disadvantage of population subdivision due to enhanced genetic drift. By contrast, in the presence of HGT (<em>r</em> ≥ 10<sup>−4</sup>), the dependency of <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-148.gif"/></noscript></span></span> on <em>D</em><sub>pop</sub> is non-monotonic: the system initially becomes more prone to Muller’s ratchet as <em>D</em><sub>pop</sub> decreases, but becomes less so as <em>D</em><sub>pop</sub> decreases further, with the position of the maximum <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-149.gif"/></noscript></span></span> along the <em>D</em><sub>pop</sub> axis shifting as a function of <em>r</em> (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8, A and B</a>). The comparison against the curve for <em>r</em> = 0 shows that the reduction in <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-150.gif"/></noscript></span></span> caused by HGT increases as <em>D</em><sub>pop</sub> decreases, indicating that population subdivision increases the effect of HGT. At a high HGT rate (<em>r</em> = 0.01), in the fast ratchet regime (<span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-151.gif"/></noscript></span></span>), the system is more resistant to Muller’s ratchet at lower values of <em>D</em><sub>pop</sub> (≤10<sup>−5</sup>) than at greater values of <em>D</em><sub>pop</sub> (≥10<sup>−1</sup>) for which the system can be considered well-mixed (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>). In the slow ratchet regime (<span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-152.gif"/></noscript></span></span>), however, this situation cannot be observed because <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-153.gif"/></noscript></span></span> falls to zero and so shows no difference between low and high values of <em>D</em><sub>pop</sub> within the accuracy of the simulations (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8A</a>). In addition to the aforementioned results, <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-154.gif"/></noscript></span></span> was measured as a function of <em>D</em><sub>pop</sub> for other combinations of <em>N</em> and <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-155.gif"/></noscript></span></span> values; the results were qualitatively the same as described previously (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1/009845SI.pdf">File S1</a>). Taken together, these results indicate that the advantage of population subdivision can outweigh the associated disadvantage, provided that the frequency of HGT is sufficiently high, and subpopulations share a common eDNA pool.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of population subdivision on the prevention of Muller&#x2019;s ratchet by HGT when eDNA diffusion is infinitely fast (DeDNA = &#x221E;). The rate of mutation accumulation  is plotted as a function of population migration rate Dpop for various values of HGT rate r. The color coding is as follows: r = 0 (black), r = 10&#x2212;4 (red), r = 10&#x2212;3 (green), and r = 10&#x2212;2 (blue). The parameters were as follows: in the slow ratchet regime, , N = 105, and  (A) ; in the fast ratchet regime, , N = 105 , and  (B). The number of subpopulations was  with toroidal boundaries (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Effect of population subdivision on the prevention of Muller&#x2019;s ratchet by HGT when eDNA diffusion is infinitely fast (&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; = &#x221E;). The rate of mutation accumulation &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is plotted as a function of population migration rate &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;pop&lt;/sub&gt; for various values of HGT rate &lt;em&gt;r&lt;/em&gt;. The color coding is as follows: &lt;em&gt;r&lt;/em&gt; = 0 (black), &lt;em&gt;r&lt;/em&gt; = 10&lt;sup&gt;&#x2212;4&lt;/sup&gt; (red), &lt;em&gt;r&lt;/em&gt; = 10&lt;sup&gt;&#x2212;3&lt;/sup&gt; (green), and &lt;em&gt;r&lt;/em&gt; = 10&lt;sup&gt;&#x2212;2&lt;/sup&gt; (blue). The parameters were as follows: in the slow ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt;, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-158&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A) ; in the fast ratchet regime, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-159&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;em&gt;N&lt;/em&gt; = 10&lt;sup&gt;5&lt;/sup&gt; , and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-160&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B). The number of subpopulations was &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with toroidal boundaries (see &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F8.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/2/325/F8.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-59" class="first-child">Effect of population subdivision on the prevention of Muller’s ratchet by HGT when eDNA diffusion is infinitely fast (<em>D</em><sub>eDNA</sub> = ∞). The rate of mutation accumulation <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-156.gif"/></noscript></span></span> is plotted as a function of population migration rate <em>D</em><sub>pop</sub> for various values of HGT rate <em>r</em>. The color coding is as follows: <em>r</em> = 0 (black), <em>r</em> = 10<sup>−4</sup> (red), <em>r</em> = 10<sup>−3</sup> (green), and <em>r</em> = 10<sup>−2</sup> (blue). The parameters were as follows: in the slow ratchet regime, <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-157.gif"/></noscript></span></span>, <em>N</em> = 10<sup>5</sup>, and <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-158.gif"/></noscript></span></span> (A) ; in the fast ratchet regime, <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-159.gif"/></noscript></span></span>, <em>N</em> = 10<sup>5</sup> , and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-160.gif"/></noscript></span></span> (B). The number of subpopulations was <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F8/embed/mml-math-161.gif"/></noscript></span></span> with toroidal boundaries (see <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">To elucidate the mechanism by which population subdivision helps HGT prevent Muller’s ratchet, we first sought to test the expectation that population subdivision increases the number of segregating mutations at the level of the entire population. To this end, we measured the average gene diversity of the least-loaded class within and between subpopulations (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Nei 1987</a>). The average gene diversity within one subpopulation is defined as <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-162.gif"/></noscript></span></span> where <em>x<sub>yki</sub></em> is the frequency of allele <em>i</em> in locus <em>k</em> in the least-loaded class of subpopulation <em>y</em> (the least-loaded class was defined separately for each subpopulation). The average gene diversity within subpopulations <em>H<sub>S</sub></em> is defined as the average <em>H<sub>Sy</sub></em> over all subpopulations (weighted by the population size of the least-loaded class in each subpopulation). The average gene diversity in the entire population <em>H<sub>T</sub></em> is defined similarly with <em>x<sub>yk</sub></em><sub>i</sub> replaced by the frequency of an allele in the least-loaded classes from all subpopulations combined together. The average gene diversity between subpopulations was defined as <em>H<sub>T</sub></em> − <em>H<sub>S</sub></em> (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Nei 1987</a>). <em>H<sub>T</sub></em> and <em>H<sub>S</sub></em> were measured in the absence of HGT. Simulations show that the values of <em>H<sub>T</sub></em> and <em>H<sub>S</sub></em> initially increase and then saturate over time (results not shown). The time averages of <em>H<sub>T</sub></em> and <em>H<sub>S</sub></em> after or near saturation were obtained as a function of <em>D</em><sub>pop</sub> (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>). The result shows that <em>H<sub>T</sub></em> increases as <em>D</em><sub>pop</sub> decreases, whereas <em>H<sub>S</sub></em> remains close to zero for the entire range of <em>D</em><sub>pop</sub>. This result indicates that population subdivision increases the number of loci that are polymorphic between subpopulations, whereas within subpopulations almost all loci remain monomorphic. Therefore, population subdivision increases the number of segregating mutations at the level of the entire population as expected.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of population subdivision on the genetic structure of populations and on the contents of eDNA pools. (A) The average gene diversity within subpopulations (HS) and in the entire population (HT), and the coefficient of gene differentiation GST = (HT &#x2212; HS)/HT are plotted as a function of Dpop in the absence of HGT (r = 0). The plotted values were obtained as the time average after or nearly after the values reached saturation (all time averages were taken from the last  generations of simulations). The error bars denote standard deviation. The parameters were the same as in Figure 8A (filled circles) and Figure 8B (open circles) except that r = 0. (B) The quality qeDNA and toxicity teDNA of eDNA are plotted as a function of Dpop in the absence of HGT (r = 0). The values of qeDNA and teDNA were obtained as the time average after the values reached saturation except for Dpop = 0, for which the values indicate lower bounds (all time averages were taken from the last 2 &#xD7; 105 generations of simulations). The error bars denote standard deviation. The parameters were the same as in Figure 8B except that r = 0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Effect of population subdivision on the genetic structure of populations and on the contents of eDNA pools. (A) The average gene diversity within subpopulations (&lt;em&gt;H&lt;sub&gt;S&lt;/sub&gt;&lt;/em&gt;) and in the entire population (&lt;em&gt;H&lt;sub&gt;T&lt;/sub&gt;&lt;/em&gt;), and the coefficient of gene differentiation G&lt;sub&gt;ST&lt;/sub&gt; = (&lt;em&gt;H&lt;sub&gt;T&lt;/sub&gt;&lt;/em&gt; &#x2212; &lt;em&gt;H&lt;sub&gt;S&lt;/sub&gt;&lt;/em&gt;)/&lt;em&gt;H&lt;sub&gt;T&lt;/sub&gt;&lt;/em&gt; are plotted as a function of &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;pop&lt;/sub&gt; in the absence of HGT (&lt;em&gt;r&lt;/em&gt; = 0). The plotted values were obtained as the time average after or nearly after the values reached saturation (all time averages were taken from the last &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-163&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; generations of simulations). The error bars denote standard deviation. The parameters were the same as in Figure 8A (filled circles) and Figure 8B (open circles) except that &lt;em&gt;r&lt;/em&gt; = 0. (B) The quality &lt;em&gt;q&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; and toxicity &lt;em&gt;t&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; of eDNA are plotted as a function of &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;pop&lt;/sub&gt; in the absence of HGT (&lt;em&gt;r&lt;/em&gt; = 0). The values of &lt;em&gt;q&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; and &lt;em&gt;t&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; were obtained as the time average after the values reached saturation except for &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;pop&lt;/sub&gt; = 0, for which the values indicate lower bounds (all time averages were taken from the last 2 &#xD7; 10&lt;sup&gt;5&lt;/sup&gt; generations of simulations). The error bars denote standard deviation. The parameters were the same as in Figure 8B except that &lt;em&gt;r&lt;/em&gt; = 0.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F9.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/4/2/325/F9.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464012" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-61" class="first-child">Effect of population subdivision on the genetic structure of populations and on the contents of eDNA pools. (A) The average gene diversity within subpopulations (<em>H<sub>S</sub></em>) and in the entire population (<em>H<sub>T</sub></em>), and the coefficient of gene differentiation G<sub>ST</sub> = (<em>H<sub>T</sub></em> − <em>H<sub>S</sub></em>)/<em>H<sub>T</sub></em> are plotted as a function of <em>D</em><sub>pop</sub> in the absence of HGT (<em>r</em> = 0). The plotted values were obtained as the time average after or nearly after the values reached saturation (all time averages were taken from the last <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F9/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F9/embed/mml-math-163.gif"/></noscript></span></span> generations of simulations). The error bars denote standard deviation. The parameters were the same as in <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8A</a> (filled circles) and <a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 8B</a> (open circles) except that <em>r</em> = 0. (B) The quality <em>q</em><sub>eDNA</sub> and toxicity <em>t</em><sub>eDNA</sub> of eDNA are plotted as a function of <em>D</em><sub>pop</sub> in the absence of HGT (<em>r</em> = 0). The values of <em>q</em><sub>eDNA</sub> and <em>t</em><sub>eDNA</sub> were obtained as the time average after the values reached saturation except for <em>D</em><sub>pop</sub> = 0, for which the values indicate lower bounds (all time averages were taken from the last 2 × 10<sup>5</sup> generations of simulations). The error bars denote standard deviation. The parameters were the same as in <a id="xref-fig-8-7" class="xref-fig" href="#F8">Figure 8B</a> except that <em>r</em> = 0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">The increase in the number of segregating mutations is expected to drive an increase in the number of mutations that are introduced or removed by HGT events. To test this expectation, we measured the quality of the eDNA pool defined above (<em>q</em><sub>eDNA</sub>) and the “toxicity” of the eDNA pool defined as <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-164.gif"/></noscript></span></span> (<em>q</em><sub>eDNA</sub> and <em>t</em><sub>eDNA</sub> were calculated for each subpopulation and averaged over subpopulations using the population sizes of the least-loaded classes as weights). <em>t</em><sub>eDNA</sub> indicates the potential of the eDNA pool to introduce deleterious mutations in the genomes of the least-loaded class. Note that the difference <em>t</em><sub>eDNA</sub> − <em>q</em><sub>eDNA</sub> is equal to the average number of mutations introduced (or removed if the value is negative) by one HGT event occurring to an individual of the least-loaded classes. Thus, − <em>q</em><sub>eDNA</sub> can be interpreted as the contribution to the average number of mutations introduced by HGT by those HGT events that decrease the number of mutations, whereas <em>t</em><sub>eDNA</sub> as the contribution by those events that increase the number of mutations. Simulations show that the values of <em>q</em><sub>eDNA</sub> and <em>t</em><sub>eDNA</sub> increase and then saturate over time except for <em>D</em><sub>pop</sub> = 0 (result not shown; for <em>D</em><sub>pop</sub> = 0, saturation is likely to be reached, but probably requires an exceedingly long time). The time averages of <em>q</em><sub>eDNA</sub> and <em>t</em><sub>eDNA</sub> after saturation were obtained as a function of <em>D</em><sub>pop</sub> in the absence of HGT (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>; for <em>D</em><sub>pop</sub> = 0, the values reached toward the end of the simulation are shown, thus indicating a lower bound). The result shows that population subdivision increases the number of mutations removed by an HGT event (<em>q</em><sub>eDNA</sub>) as well as the number of mutations introduced by an HGT event (<em>t</em><sub>eDNA</sub>), thus confirming the expectation (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9B</a>). Moreover, it leaves nearly constant the average (<em>i.e.</em>, net) number of mutations introduced by an HGT event (<em>t</em><sub>eDNA</sub> − <em>q</em><sub>eDNA</sub>). In other words, population subdivision increases the conversion of the least-loaded class by HGT in the direction of both increasing and decreasing the number of mutations without changing the magnitude and direction of the net conversion. According to the simplified Wright-Fisher model described before, removal of mutations by HGT (<em>r</em><sub>10</sub>) has a greater influence on the extinction of the least-loaded class than introduction of mutations by HGT (<em>r</em><sub>01</sub>). Although <em>r</em><sub>10</sub> and <em>r</em><sub>01</sub> are concerned with HGT occurring in different genotypes, they are analogous to <em>q</em><sub>eDNA</sub> and <em>t</em><sub>eDNA</sub>, respectively. This analogy points to the advantage of increasing <em>q</em><sub>eDNA</sub> and <em>t</em><sub>eDNA</sub> with <em>t</em><sub>eDNA</sub> − <em>q</em><sub>eDNA</sub> kept constant to prevent Muller’s ratchet.</p><p id="p-63">Whether population subdivision is advantageous or disadvantageous depends on the combination of parameters. To examine what appears to be the most important aspect of this parameter dependence, we next considered the finite diffusion of eDNA. To this end, the value of <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-165.gif"/></noscript></span></span> was compared between <em>D</em><sub>pop</sub> = 10<sup>−5</sup> (subdivided population) and <em>D</em><sub>pop</sub> = 10<sup>−1</sup> (well-mixed population) for various combinations of <em>d</em><sub>eDNA</sub> and <em>D</em><sub>eDNA</sub> values. As expected, the results show that <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/embed/mml-math-166.gif"/></noscript></span></span> is smaller for <em>D</em><sub>pop</sub> = 10<sup>−5</sup> than for <em>D</em><sub>pop</sub> = 10<sup>−1</sup> when <em>d</em><sub>eDNA</sub> is sufficiently small or <em>D</em><sub>eDNA</sub> is sufficiently large (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>). The values of <em>D</em><sub>eDNA</sub> and <em>d</em><sub>eDNA</sub> demarcating the region of the parameter space for which population subdivision is advantageous do not appear unrealistic, although it is difficult to judge whether the conditions represented by this region are achievable in natural environments (see also <em>Discussion</em>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/325/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of population subdivision on the prevention of Muller&#x2019;s ratchet by HGT when the eDNA diffusion rate is finite. Red squares indicate the combinations of deDNA and DeDNA values for which the rate of mutation accumulation  is lower when a population is subdivided (Dpop = 10&#x2212;5) than when a population is well-mixed (Dpop = 10&#x2212;1). Black circles indicate the parameter region where the opposite was the case. The parameters (other than DeDNA and 2eDNA) were the same as in Figure 8B with ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1830646178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Effect of population subdivision on the prevention of Muller&#x2019;s ratchet by HGT when the eDNA diffusion rate is finite. Red squares indicate the combinations of &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; and &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; values for which the rate of mutation accumulation &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-167&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is lower when a population is subdivided (&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;pop&lt;/sub&gt; = 10&lt;sup&gt;&#x2212;5&lt;/sup&gt;) than when a population is well-mixed (&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;pop&lt;/sub&gt; = 10&lt;sup&gt;&#x2212;1&lt;/sup&gt;). Black circles indicate the parameter region where the opposite was the case. The parameters (other than &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt; and &lt;em&gt;2&lt;/em&gt;&lt;sub&gt;eDNA&lt;/sub&gt;) were the same as in Figure 8B with &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-168&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/325/F10.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/4/2/325/F10.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/325/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/325/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463982" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-64" class="first-child">Effect of population subdivision on the prevention of Muller’s ratchet by HGT when the eDNA diffusion rate is finite. Red squares indicate the combinations of <em>d</em><sub>eDNA</sub> and <em>D</em><sub>eDNA</sub> values for which the rate of mutation accumulation <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F10/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F10/embed/mml-math-167.gif"/></noscript></span></span> is lower when a population is subdivided (<em>D</em><sub>pop</sub> = 10<sup>−5</sup>) than when a population is well-mixed (<em>D</em><sub>pop</sub> = 10<sup>−1</sup>). Black circles indicate the parameter region where the opposite was the case. The parameters (other than <em>D</em><sub>eDNA</sub> and <em>2</em><sub>eDNA</sub>) were the same as in <a id="xref-fig-8-8" class="xref-fig" href="#F8">Figure 8B</a> with <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F10/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/325/F10/embed/mml-math-168.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-65">The results presented here suggest that HGT can prevent the operation of Muller’s ratchet in prokaryotic populations, even if on average HGT introduces more deleterious mutations than it removes. The avoidance of Muller’s ratchet via transformation and recombination with eDNA might appear somewhat paradoxical because on average eDNA is expected to carry more deleterious mutations than the DNA in live prokaryotic cells. Indeed, it has been long recognized that “sex with dead cells” is a dubious proposition (<a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Redfield 1988</a>, <a id="xref-ref-77-4" class="xref-bibr" href="#ref-77">2001</a>; <a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Redfield <em>et al.</em> 1997</a>). However, the modeling results indicate that, even though transformation via eDNA is expected to increase the mean mutation load and hence decrease the average fitness of a population, it nevertheless can stop Muller’s ratchet. This appears to be the case because HGT provides for the chance to eliminate deleterious mutations, leading to the continual restoration of the least-loaded class. In other words, HGT prolongs the persistence of the least-loaded class in the population in the face of stochastic fluctuations, hence the prevention of Muller’s ratchet.</p><p id="p-66">The efficacy of HGT in the prevention of Muller’s ratchet depends on the stability of eDNA, its diffusion rate, and population subdivision. The contribution of eDNA stability (<em>i.e.</em>, slow turnover) is intuitively clear because it delays elimination of high-fitness alleles from the population and so increases the chance that these alleles offset Muller’s ratchet via HGT. However, the simulations show that, for this effect to be substantial, the characteristic lifetime of eDNA should exceed the generation time by orders of magnitude (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), suggesting that eDNA stability alone is not a key factor in the evolution of microbial populations. If, however, a population is subdivided, enhanced eDNA stability facilitates the transport of eDNA across subpopulations and thus contributes to the ability of HGT to prevent Muller’s ratchet. Under these conditions, for the effect to be significant, the characteristic lifetime of eDNA does not have to be exceedingly long, depending on the diffusion rate of eDNA (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a>).</p><p id="p-67">If the transport of eDNA is sufficiently fast, HGT can prevent Muller’s ratchet more efficiently in a subdivided population than in an undivided population. As a result, a subdivided population can be more resistant to Muller’s ratchet than an undivided population of an equal overall size despite the fact that population subdivision accelerates Muller’s ratchet in the absence of HGT owing to the enhanced effect of genetic drift. Put differently, to maintain genomic information in the face of deleterious mutations and genetic drift, it is advantageous to partition individuals into multiple (smaller) subpopulations and let them “cross-reference” each other’s genetic information through recombination rather than collect all individuals in one population and thereby maximize the efficacy of natural selection. Note, however, that although population subdivision can be beneficial, it is not necessary for HGT to prevent Muller’s ratchet.</p><p id="p-68">How do these findings relate to the previous population genetics studies on the advantage of recombination? Previous studies have shown that recombination is beneficial (compared with no recombination) when a population suffers from the reduced efficacy of natural selection because of negative linkage disequilibrium (<em>i.e.</em>, biased associations on chromosomes between beneficial and deleterious alleles) which arises from genetic drift (the Hill-Robertson effect) or synergetic epistasis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hill and Robertson 1966</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Felsenstein 1974</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Feldman <em>et al.</em> 1980</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kondrashov 1988</a>, <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">1993</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Otto and Lenormand 2002</a>). Our present results are consistent with these findings because they show that the reduction in the rate of mutation accumulation caused by HGT is greater in a subdivided population than in an undivided population (<a id="xref-fig-8-9" class="xref-fig" href="#F8">Figure 8, A and B</a>). Indeed, population subdivision causes negative linkage disequilibrium in many pairs of loci in the least-loaded classes at the entire population level because the least-loaded classes in different subpopulations have mutations in different loci (this can be seen from the fact that <em>H<sub>T</sub></em> &gt; 0 and <em>H<sub>S</sub></em> ≈ 0 when <em>D</em><sub>pop</sub> ≈ 0). In addition, the results show that the rate of mutation accumulation itself can be lower in a subdivided population than in an undivided population, despite the disadvantage of population subdivision due to enhanced drift. This conclusion does not immediately follow from the previous findings because the fact that negative linkage disequilibrium increases the benefit of recombination (compared with no recombination) does not necessarily mean that negative linkage disequilibrium is beneficial in preventing the accumulation of mutations in the presence of recombination.</p><p id="p-69">Evolutionary maintenance of genomic information based on population subdivision and recombination hinges on the fact that genetic exchange between cells happens via a shared pool of eDNA. On the one hand, this implies that the indirectness of HGT might actually be an asset rather than a drawback for prokaryotes because HGT provides the possibility of exchange between physically separated subpopulations that accumulate different mutations.</p><p id="p-70">On the other hand, indirect recombination imposes strict conditions under which population subdivision is advantageous in terms of Muller’s ratchet, requiring restricted migration between populations and rapid transport of eDNA molecules. Although at present it is difficult to judge whether such conditions are fulfilled in natural environments, the available data seem to suggest soils as an environment that is conducive to these processes. The distribution of bacteria in natural soils is “patchy,” <em>i.e.</em>, bacteria typically occur as small colonies separated from each other, and their locations are restricted by the availability of soil micro-pores, water, and organic substances (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Foster 1988</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Stotzky 1989</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Grundmann 2004</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Young <em>et al.</em> 2008</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Vos <em>et al.</em> 2013</a>). Moreover, bacterial motility can be highly restricted by water availability (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dechesne <em>et al.</em> 2010</a>). In addition, soil bacteria display high levels of genetic diversity within local populations (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Grundmann 2004</a>). For example, <em>Agrobacterium</em> biovar 1 and <em>Nitrobacter</em>-like bacteria within minute soil samples (&lt;1 mm in diameter) display genetic divergence as great as that between strains sampled from different geographical areas (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Grundmann and Normand 2000</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Vogel <em>et al.</em> 2003</a>, although <em>Agrobacterium</em> biovar 1 is not known to be naturally competent, this bacterium readily undergoes homologous recombination; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Costechareyre <em>et al.</em> 2009</a>). Also, genomes of <em>Pseudomonas stutzeri</em> display extremely high gene diversity (<em>H<sub>T</sub></em> &gt; 0.8) and high frequency of null alleles (&gt;80% of the examined strains fail to exhibit the activity of at least one of the 20 enzymes examined; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Rius <em>et al.</em> 2001</a>). Taken together, these findings suggest that bacterial populations in unsaturated soils are strongly subdivided and genetically heterogeneous within local environments. This population structure is compatible with a situation in which HGT and population subdivision jointly prevent the operation of Muller’s ratchet (if eDNA transport is sufficiently rapid as well).</p><p id="p-71">Moreover, eDNA in soils becomes highly stable against hydrolysis by nucleases when it is absorbed to clay minerals, sand particles, and humic acids without losing the ability to transform competent bacteria (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Vries and Wackernagel 2005</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Nielsen <em>et al.</em> 2007</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Pietramellara <em>et al.</em> 2009</a>). Natural transformation assays indicate that DNA added to nonsterile soils retains transformation ability for 3−15 days (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gallori <em>et al.</em> 1994</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Sikorski <em>et al.</em> 1998</a>). In addition, eDNA can be transported within unsaturated soils through water capillarity or leaching (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ceccherini <em>et al.</em> 2007</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Pote <em>et al.</em> 2010</a>). Although the difference between bacteria and eDNA in terms of transport efficiency within soils remains to be evaluated, the obvious size difference suggests that eDNA is more readily transported than bacteria. Taken together, these findings suggest soil prokaryote systems as a prime candidate for a system in which HGT and population subdivision could be important for the long-term maintenance of genomic information in the face of Muller’s ratchet. Furthermore, it appears plausible that both the molecular machinery of natural transformation and GTAs evolved, at least in part, as adaptations facilitating HGT and overcoming the limitations on HGT imposed by the aforementioned conditions, <em>e.g.</em>, GTAs can increase the stability of DNA emitted from the cell.</p><p id="p-72">Is Muller’s ratchet an important factor of evolution in prokaryotes? Its role can be questioned because many bacteria have large population sizes in which the extent of stochasticity is limited and that are accordingly immune to Muller’s ratchet (but see also the discussion of soil bacteria above in this section). However, bacteria are also known to go through frequent population bottlenecks under stress conditions (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Levin <em>et al.</em> 2000</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aertsen and Michiels 2004</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Wolf <em>et al.</em> 2005</a>), and it is under these conditions that Muller’s ratchet and mechanisms that can overcome it could become relevant. Indeed, competence has been shown to increase stress resistance in bacteria (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Engelmoer and Rozen 2011</a>) and induction of the competence regulons is an important part of bacterial stress response (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Claverys <em>et al.</em> 2006</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Prudhomme <em>et al.</em> 2006</a>). Although the biology of the GTAs is not nearly as well understood as that of competence and transformation, stress-induced prophage replication that is likely to potentiate HGT has been demonstrated (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Garcia-Russell <em>et al.</em> 2009</a>).</p><p id="p-73">To conclude, the results of the modeling study described here indicate that HGT can prevent the operation of Muller’s ratchet, and this effect can be enhanced by population subdivision. Thus, through preventing the action of Muller’s ratchet, HGT could be a necessary condition for the long-term survival of prokaryotic populations. The benefits of HGT for prokaryotes certainly are not limited to the prevention of Muller’s ratchet and additionally include acquisition of novel genes and functions (see, <em>e.g.</em>, <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Levin and Cornejo 2009</a>; <a id="xref-ref-93-3" class="xref-bibr" href="#ref-93">Wylie <em>et al.</em> 2010</a>). Jointly, these advantages could be the driving forces behind the evolution of dedicated mechanisms for HGT.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-74">We thank Paulien Hogeweg, Alexander Lobkovsky, Nen Saito, and Yuri Wolf for helpful discussions. This research was in part supported by the Intramural Research Program of the National Institutes of Health, National Library of Medicine, by the Japan Society for the Promotion of Science Research Fellowship for Japanese Biomedical and Behavioral Researchers at the National Institutes of Health, and by the Japan Society for the Promotion of Science Research Fellowship for Young Scientists.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009845/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-75"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-04"><span class="received-label">Received </span>September 4, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-12-12"><span class="accepted-label">Accepted </span>December 12, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Takeuchi <em>et al.</em> </li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.1" data-doi="10.1080/10408410490884757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aertsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michiels</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Stress and how bacteria cope with death and survival.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Microbiol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2Breviews%2Bin%2Bmicrobiology%26rft.stitle%253DCrit%2BRev%2BMicrobiol%26rft.aulast%253DAertsen%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D263%26rft.epage%253D273%26rft.atitle%253DStress%2Band%2Bhow%2Bbacteria%2Bcope%2Bwith%2Bdeath%2Band%2Bsurvival.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10408410490884757%26rft_id%253Dinfo%253Apmid%252F15646400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10408410490884757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15646400&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.2" data-doi="10.1016/S0168-9525(98)01553-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aravind</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatusov</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Evidence for massive gene exchange between archaeal and bacterial hyperthermophiles.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DAravind%26rft.auinit1%253DL.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D442%26rft.epage%253D444%26rft.atitle%253DEvidence%2Bfor%2Bmassive%2Bgene%2Bexchange%2Bbetween%2Barchaeal%2Band%2Bbacterial%2Bhyperthermophiles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252898%252901553-4%26rft_id%253Dinfo%253Apmid%252F9825671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(98)01553-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9825671&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076946100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.3" data-doi="10.1046/j.1420-9101.1988.1010067.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Recombination and the immortality of the germ line.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D1%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.1988.1010067.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.1988.1010067.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988P869500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopf</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michod</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Is meiotic recombination an adaptation for repairing DNA, producing genetic variation, or both?</span> pp. <span class="cit-fpage">139</span>–<span class="cit-lpage">160</span> in <span class="cit-source">The Evolution of Sex</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Michod</span>  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.5" data-doi="10.1093/jac/dkl020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blahna</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalewski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reuer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahlmeter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foxman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The role of horizontal gene transfer in the spread of trimethoprim-sulfamethoxazole resistance among uropathogenic <em>Escherichia coli</em> in Europe and Canada.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">666</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAntimicrob.%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252Fdkl020%26rft_id%253Dinfo%253Apmid%252F16464890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamFjIjtzOjU6InJlc2lkIjtzOjg6IjU3LzQvNjY2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.6" data-doi="10.1007/s00253-010-2489-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brigulla</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wackernagel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular aspects of gene transfer and foreign DNA acquisition in prokaryotes with regard to safety issues.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DBrigulla%26rft.auinit1%253DM.%26rft.volume%253D86%26rft.issue%253D4%26rft.spage%253D1027%26rft.epage%253D1041%26rft.atitle%253DMolecular%2Baspects%2Bof%2Bgene%2Btransfer%2Band%2Bforeign%2BDNA%2Bacquisition%2Bin%2Bprokaryotes%2Bwith%2Bregard%2Bto%2Bsafety%2Bissues.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-010-2489-3%26rft_id%253Dinfo%253Apmid%252F20191269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-010-2489-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20191269&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brochier-Armanet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forterre</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Widespread distribution of archaeal reverse gyrase in thermophilic bacteria suggests a complex history of vertical inheritance and lateral gene transfers.</span> <abbr class="cit-jnl-abbrev">Archaea</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DArchaea%26rft.aulast%253DBrochier-Armanet%26rft.auinit1%253DC.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D83%26rft.epage%253D93%26rft.atitle%253DWidespread%2Bdistribution%2Bof%2Barchaeal%2Breverse%2Bgyrase%2Bin%2Bthermophilic%2Bbacteria%2Bsuggests%2Ba%2Bcomplex%2Bhistory%2Bof%2Bvertical%2Binheritance%2Band%2Blateral%2Bgene%2Btransfers.%26rft_id%253Dinfo%253Apmid%252F17350929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17350929&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.8" data-doi="10.1073/pnas.1101591108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rusch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjelleberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bacterial community assembly based on functional genes rather than species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">14288</span>–<span class="cit-lpage">14293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1101591108%26rft_id%253Dinfo%253Apmid%252F21825123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM0LzE0Mjg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.9" data-doi="10.1016/j.soilbio.2006.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ceccherini</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ascher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pietramellara</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nannipieri</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Vertical advection of extracellular DNA by water capillarity in soil columns.</span> <abbr class="cit-jnl-abbrev">Soil Biol. Biochem.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoil%2BBiol.%2BBiochem.%26rft.volume%253D39%26rft.spage%253D158%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.soilbio.2006.07.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.soilbio.2006.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.10" data-doi="10.1017/S0016672397002899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Rapid fixation of deleterious alleles can be caused by Muller’s ratchet.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D73%26rft.atitle%253DRapid%2Bfixation%2Bof%2Bdeleterious%2Balleles%2Bcan%2Bbe%2Bcaused%2Bby%2BMuller%2527s%2Bratchet.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002899%26rft_id%253Dinfo%253Apmid%252F9369098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9369098&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YF21000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.11" data-doi="10.1017/S0016672300031086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Mutation accumulation in finite outbreeding and inbreeding populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D61%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300031086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300031086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993KV91600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.12" data-doi="10.1146/annurev.micro.60.080805.142139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claverys</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prudhomme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Induction of competence regulons as a general response to stress in gram-positive bacteria.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DClaverys%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D60%26rft.spage%253D451%26rft.epage%253D475%26rft.atitle%253DInduction%2Bof%2Bcompetence%2Bregulons%2Bas%2Ba%2Bgeneral%2Bresponse%2Bto%2Bstress%2Bin%2Bgram-positive%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.60.080805.142139%26rft_id%253Dinfo%253Apmid%252F16771651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.60.080805.142139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16771651&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.13" data-doi="10.1111/j.1574-6976.2009.00164.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claverys</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polard</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic transformation machinery: composition, localization, and mechanism.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2009.00164.x%26rft_id%253Dinfo%253Apmid%252F19228200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjg6IjMzLzMvNjQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.14" data-doi="10.1017/S0016672307008658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combadao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dionisio</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordo</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Small-world networks decrease the speed of Muller’s ratchet.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCombadao%26rft.auinit1%253DJ.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D18%26rft.atitle%253DSmall-world%2Bnetworks%2Bdecrease%2Bthe%2Bspeed%2Bof%2BMuller%2527s%2Bratchet.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672307008658%26rft_id%253Dinfo%253Apmid%252F17517155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672307008658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17517155&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247188500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.15" data-doi="10.1371/journal.pbio.0050225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination speeds adaptation by reducing competition between beneficial mutations in populations of Escherichia coli.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DCooper%26rft.auinit1%253DT.%2BF.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De225%26rft.epage%253De225%26rft.atitle%253DRecombination%2Bspeeds%2Badaptation%2Bby%2Breducing%2Bcompetition%2Bbetween%2Bbeneficial%2Bmutations%2Bin%2Bpopulations%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050225%26rft_id%253Dinfo%253Apmid%252F17713986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17713986&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.16" data-doi="10.1093/molbev/msn236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costechareyre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolla</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesme</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Homologous recombination in Agrobacterium: potential implications for the genomic species concept in bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn236%26rft_id%253Dinfo%253Apmid%252F18936442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMTY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.17" data-doi="10.1073/pnas.0800679105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dagan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artzy-Randrup</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Modular networks and cumulative impact of lateral transfer in prokaryote genome evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">10039</span>–<span class="cit-lpage">10044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0800679105%26rft_id%253Dinfo%253Apmid%252F18632554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzI5LzEwMDM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.18" data-doi="10.1073/pnas.1008392107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dechesne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gulez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Or</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smets</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hydration-controlled bacterial motility and dispersal on surfaces.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">14369</span>–<span class="cit-lpage">14372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1008392107%26rft_id%253Dinfo%253Apmid%252F20660312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzMyLzE0MzY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.19" data-doi="10.1146/annurev.micro.53.1.217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dubnau</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">DNA uptake in bacteria.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DDubnau%26rft.auinit1%253DD.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D217%26rft.epage%253D244%26rft.atitle%253DDNA%2BUPTAKE%2BIN%2BBACTERIA%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.53.1.217%26rft_id%253Dinfo%253Apmid%252F10547691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.53.1.217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10547691&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083208500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.20" data-doi="10.1111/j.1558-5646.2011.01402.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engelmoer</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Competence increases survival during stress in <em>Streptococcus pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">3475</span>–<span class="cit-lpage">3485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DEngelmoer%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D65%26rft.issue%253D12%26rft.spage%253D3475%26rft.epage%253D3485%26rft.atitle%253DCompetence%2Bincreases%2Bsurvival%2Bduring%2Bstress%2Bin%2BStreptococcus%2Bpneumoniae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01402.x%26rft_id%253Dinfo%253Apmid%252F22133219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01402.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22133219&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297732000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.21" data-doi="10.1534/genetics.107.072892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eppley</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyson</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Getz</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banfield</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic exchange across a species boundary in the archaeal genus ferroplasma.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.072892%26rft_id%253Dinfo%253Apmid%252F17603112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvNDA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.22" data-doi="10.1073/pnas.98.1.182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination within natural populations of pathogenic bacteria: short-term empirical estimates and long-term phylogenetic consequences.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D182%26rft.epage%253D187%26rft.atitle%253DRecombination%2Bwithin%2Bnatural%2Bpopulations%2Bof%2Bpathogenic%2Bbacteria%253A%2BShort-term%2Bempirical%2Bestimates%2Band%2Blong-term%2Bphylogenetic%2Bconsequences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.1.182%26rft_id%253Dinfo%253Apmid%252F11136255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8xLzE4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimating recombinational parameters in <em>Streptococcus pneumoniae</em> from multilocus sequence typing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1439</span>–<span class="cit-lpage">1450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1439%26rft.epage%253D1450%26rft.atitle%253DEstimating%2BRecombinational%2BParameters%2Bin%2BStreptococcus%2Bpneumoniae%2BFrom%2BMultilocus%2BSequence%2BTyping%2BData%26rft_id%253Dinfo%253Apmid%252F10747043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE0MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.24" data-doi="10.1073/pnas.77.8.4838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christiansen</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Evolution of recombination in a constant environment.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">4838</span>–<span class="cit-lpage">4841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFeldman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D77%26rft.issue%253D8%26rft.spage%253D4838%26rft.epage%253D4841%26rft.atitle%253DEvolution%2Bof%2Brecombination%2Bin%2Ba%2Bconstant%2Benvironment%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.8.4838%26rft_id%253Dinfo%253Apmid%252F16592864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ny84LzQ4MzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Microenvironments of soil microorganisms.</span> <abbr class="cit-jnl-abbrev">Biol. Fertil. Soils</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BFertil.%2BSoils%26rft.volume%253D6%26rft.spage%253D189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1988N550800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.27" data-doi="10.1111/j.1574-6941.1994.tb00236.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallori</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazzicalupo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dal Canto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nannipieri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Transformation of <em>Bacillus subtilis</em> by DNA bound on clay in non-sterile soil.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6941.1994.tb00236.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2VjIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xLTIvMTE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.28" data-doi="10.1016/j.meegid.2009.05.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Russell</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elrod</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominguez</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Stress-induced prophage DNA replication in <em>Salmonella enterica</em> serovar Typhimurium.</span> <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DGarcia-Russell%26rft.auinit1%253DN.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D889%26rft.epage%253D895%26rft.atitle%253DStress-induced%2Bprophage%2BDNA%2Breplication%2Bin%2BSalmonella%2Benterica%2Bserovar%2BTyphimurium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2009.05.017%26rft_id%253Dinfo%253Apmid%252F19501196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2009.05.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19501196&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.29" data-doi="10.1093/oxfordjournals.molbev.a004046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doolittle</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Prokaryotic evolution in light of gene transfer.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2226</span>–<span class="cit-lpage">2238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DGogarten%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D2226%26rft.epage%253D2238%26rft.atitle%253DProkaryotic%2BEvolution%2Bin%2BLight%2Bof%2BGene%2BTransfer%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004046%26rft_id%253Dinfo%253Apmid%252F12446813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi8yMjI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.30" data-doi="10.1534/genetics.112.141291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goyal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jerison</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shraiman</span>  <span class="cit-name-given-names">B. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dynamic mutation-selection balance as an evolutionary attractor.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141291%26rft_id%253Dinfo%253Apmid%252F22661327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MS80LzEzMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.31" data-doi="10.1016/j.femsec.2004.01.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grundmann</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Spatial scales of soil bacterial diversity--the size of a clone.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Ecol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.femsec.2004.01.010%26rft_id%253Dinfo%253Apmid%252F19712395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2VjIjtzOjU6InJlc2lkIjtzOjg6IjQ4LzIvMTE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.32" data-doi="10.1128/AEM.66.10.4543-4546.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grundmann</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normand</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Microscale diversity of the genus Nitrobacter in soil on the basis of analysis of genes encoding rRNA.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">4543</span>–<span class="cit-lpage">4546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DGrundmann%26rft.auinit1%253DG.%2BL.%26rft.volume%253D66%26rft.issue%253D10%26rft.spage%253D4543%26rft.epage%253D4546%26rft.atitle%253DMicroscale%2BDiversity%2Bof%2Bthe%2BGenus%2BNitrobacter%2Bin%2BSoil%2Bon%2Bthe%2BBasis%2Bof%2BAnalysis%2Bof%2BGenes%2BEncoding%2BrRNA%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.66.10.4543-4546.2000%26rft_id%253Dinfo%253Apmid%252F11010914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI2Ni8xMC80NTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.33" data-doi="10.1016/0040-5809(78)90027-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">The accumulation of deleterious genes in a population—Muller’s ratchet.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHaigh%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D251%26rft.epage%253D267%26rft.atitle%253DThe%2Baccumulation%2Bof%2Bdeleterious%2Bgenes%2Bin%2Ba%2Bpopulation--Muller%2527s%2BRatchet.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252878%252990027-8%26rft_id%253Dinfo%253Apmid%252F746491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(78)90027-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=746491&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978GA18900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.34" data-doi="10.1073/pnas.031358898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Metapopulation extinction caused by mutation accumulation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">2928</span>–<span class="cit-lpage">2933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.031358898%26rft_id%253Dinfo%253Apmid%252F11226343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC81LzI5MjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.35" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.36" data-doi="10.1111/j.1365-2958.2007.06096.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wackernagel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Double illegitimate recombination events integrate DNA segments through two different mechanisms during natural transformation of <em>Acinetobacter baylyi</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">984</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DHulter%26rft.auinit1%253DN.%26rft.volume%253D67%26rft.issue%253D5%26rft.spage%253D984%26rft.epage%253D995%26rft.atitle%253DDouble%2Billegitimate%2Brecombination%2Bevents%2Bintegrate%2BDNA%2Bsegments%2Bthrough%2Btwo%2Bdifferent%2Bmechanisms%2Bduring%2Bnatural%2Btransformation%2Bof%2BAcinetobacter%2Bbaylyi.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2007.06096.x%26rft_id%253Dinfo%253Apmid%252F18194157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2007.06096.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18194157&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252932000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.37" data-doi="10.1016/j.resmic.2007.09.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsborg</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eldholm</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havarstein</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Natural genetic transformation: prevalence, mechanisms and function.</span> <abbr class="cit-jnl-abbrev">Res. Microbiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.stitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.aulast%253DJohnsborg%26rft.auinit1%253DO.%26rft.volume%253D158%26rft.issue%253D10%26rft.spage%253D767%26rft.epage%253D778%26rft.atitle%253DNatural%2Bgenetic%2Btransformation%253A%2Bprevalence%252C%2Bmechanisms%2Band%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.resmic.2007.09.004%26rft_id%253Dinfo%253Apmid%252F17997281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.resmic.2007.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17997281&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252370300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.38" data-doi="10.1371/journal.ppat.1003178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granadel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claverys</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Programmed protection of foreign DNA from restriction allows pathogenicity island exchange during pneumococcal transformation.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003178%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003178%26rft_id%253Dinfo%253Apmid%252F23459610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23459610&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.39" data-doi="10.1038/nature05049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Interference among deleterious mutations favours sex and recombination in finite populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeightley%26rft.auinit1%253DP.%2BD.%26rft.volume%253D443%26rft.issue%253D7107%26rft.spage%253D89%26rft.epage%253D92%26rft.atitle%253DInterference%2Bamong%2Bdeleterious%2Bmutations%2Bfavours%2Bsex%2Band%2Brecombination%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05049%26rft_id%253Dinfo%253Apmid%252F16957730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16957730&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240313900045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.40" data-doi="10.1016/j.fct.2008.02.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vespermann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Horizontal gene transfer of virulence determinants in selected bacterial foodborne pathogens.</span> <abbr class="cit-jnl-abbrev">Food Chem. Toxicol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2Band%2Bchemical%2Btoxicology%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bpublished%2Bfor%2Bthe%2BBritish%2BIndustrial%2BBiological%2BResearch%2BAssociation%26rft.stitle%253DFood%2BChem%2BToxicol%26rft.aulast%253DKelly%26rft.auinit1%253DB.%2BG.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D969%26rft.epage%253D977%26rft.atitle%253DHorizontal%2Bgene%2Btransfer%2Bof%2Bvirulence%2Bdeterminants%2Bin%2Bselected%2Bbacterial%2Bfoodborne%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fct.2008.02.007%26rft_id%253Dinfo%253Apmid%252F18420327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fct.2008.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18420327&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-other" id="cit-4.2.325.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komatsu</span>  <span class="cit-name-given-names">S.,</span></span></li></ol><cite> <span class="cit-pub-date">1980</span> <span class="cit-source"><em>Overstepping Biology</em> (“<em>Hamidashi Seibutsugaku</em>”) (<em>in Japanese</em>)</span>. <span class="cit-publ-name">Heibonsha</span>, <span class="cit-publ-loc">Tokyo</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.42" data-doi="10.1038/336435a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Deleterious mutations and the evolution of sexual reproduction.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKondrashov%26rft.auinit1%253DA.%2BS.%26rft.volume%253D336%26rft.issue%253D6198%26rft.spage%253D435%26rft.epage%253D440%26rft.atitle%253DDeleterious%2Bmutations%2Band%2Bthe%2Bevolution%2Bof%2Bsexual%2Breproduction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F336435a0%26rft_id%253Dinfo%253Apmid%252F3057385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/336435a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3057385&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R135800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Classification of hypotheses on the advantage of amphimixis.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DKondrashov%26rft.auinit1%253DA.%2BS.%26rft.volume%253D84%26rft.issue%253D5%26rft.spage%253D372%26rft.epage%253D387%26rft.atitle%253DClassification%2Bof%2BHypotheses%2Bon%2Bthe%2BAdvantage%2Bof%2BAmphimixis%26rft_id%253Dinfo%253Apmid%252F8409359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijg0LzUvMzcyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">The Logic of Chance</span>. <span class="cit-publ-name">FT Press Science</span>, <span class="cit-publ-loc">Upper Saddle River, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.45" data-doi="10.1146/annurev.micro.55.1.709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravind</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Horizontal gene transfer in prokaryotes: quantification and classification.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D709%26rft.epage%253D742%26rft.atitle%253DHORIZONTAL%2BGENE%2BTRANSFER%2BIN%2BPROKARYOTES%253A%2BQuantification%2Band%2BClassification%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.55.1.709%26rft_id%253Dinfo%253Apmid%252F11544372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.55.1.709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11544372&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171732600027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.46" data-doi="10.1016/j.tim.2006.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beatty</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Importance of widespread gene transfer agent genes in alpha-proteobacteria.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DLang%26rft.auinit1%253DA.%2BS.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D54%26rft.epage%253D62%26rft.atitle%253DImportance%2Bof%2Bwidespread%2Bgene%2Btransfer%2Bagent%2Bgenes%2Bin%2Balpha-proteobacteria.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2006.12.001%26rft_id%253Dinfo%253Apmid%252F17184993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2006.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17184993&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244535900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.47" data-doi="10.1038/nrmicro2802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhaxybayeva</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beatty</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene transfer agents: phage-like elements of genetic exchange.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">472</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DLang%26rft.auinit1%253DA.%2BS.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D472%26rft.epage%253D482%26rft.atitle%253DGene%2Btransfer%2Bagents%253A%2Bphage-like%2Belements%2Bof%2Bgenetic%2Bexchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2802%26rft_id%253Dinfo%253Apmid%252F22683880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22683880&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.48" data-doi="10.1371/journal.pbio.0030130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lerat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daubin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolutionary origins of genomic repertoires in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLerat%26rft.auinit1%253DE.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De130%26rft.epage%253De130%26rft.atitle%253DEvolutionary%2Borigins%2Bof%2Bgenomic%2Brepertoires%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030130%26rft_id%253Dinfo%253Apmid%252F15799709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15799709&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The evolution of sex in bacteria</span>, pp. <span class="cit-fpage">194</span>–<span class="cit-lpage">211</span> in <span class="cit-source">The Evolution of Sex</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Michod</span>  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.50" data-doi="10.1371/journal.pgen.1000601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornejo</span>  <span class="cit-name-given-names">O. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The population and evolutionary dynamics of homologous gene recombination in bacterial populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLevin%26rft.auinit1%253DB.%2BR.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De1000601%26rft.epage%253De1000601%26rft.atitle%253DThe%2Bpopulation%2Band%2Bevolutionary%2Bdynamics%2Bof%2Bhomologous%2Bgene%2Brecombination%2Bin%2Bbacterial%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000601%26rft_id%253Dinfo%253Apmid%252F19680442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19680442&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrot</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Compensatory mutations, antibiotic resistance and the population genetics of adaptive evolution in bacteria.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLevin%26rft.auinit1%253DB.%2BR.%26rft.volume%253D154%26rft.issue%253D3%26rft.spage%253D985%26rft.epage%253D997%26rft.atitle%253DCompensatory%2BMutations%252C%2BAntibiotic%2BResistance%2Band%2Bthe%2BPopulation%2BGenetics%2Bof%2BAdaptive%2BEvolution%2Bin%2BBacteria%26rft_id%253Dinfo%253Apmid%252F10757748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzMvOTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wackernagel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Bacterial gene transfer by natural genetic transformation in the environment.</span> <abbr class="cit-jnl-abbrev">Microbiol. Rev.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DLorenz%26rft.auinit1%253DM%2BG%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D563%26rft.epage%253D602%26rft.atitle%253DBacterial%2Bgene%2Btransfer%2Bby%2Bnatural%2Bgenetic%2Btransformation%2Bin%2Bthe%2Benvironment.%26rft_id%253Dinfo%253Apmid%252F7968924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1OC8zLzU2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butcher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The mutational meltdown in asexual populations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D84%26rft.issue%253D5%26rft.spage%253D339%26rft.epage%253D344%26rft.atitle%253DThe%2BMutational%2BMeltdown%2Bin%2BAsexual%2BPopulations%26rft_id%253Dinfo%253Apmid%252F8409355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijg0LzUvMzM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majewski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohan</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">DNA sequence similarity requirements for interspecific recombination in Bacillus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1525</span>–<span class="cit-lpage">1533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMajewski%26rft.auinit1%253DJ.%26rft.volume%253D153%26rft.issue%253D4%26rft.spage%253D1525%26rft.epage%253D1533%26rft.atitle%253DDNA%2BSequence%2BSimilarity%2BRequirements%2Bfor%2BInterspecific%2BRecombination%2Bin%2BBacillus%26rft_id%253Dinfo%253Apmid%252F10581263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My80LzE1MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.55" data-doi="10.1073/pnas.0903206106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maslov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sneppen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Toolbox model of evolution of prokaryotic metabolic networks and their regulation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9743</span>–<span class="cit-lpage">9748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0903206106%26rft_id%253Dinfo%253Apmid%252F19482938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzI0Lzk3NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-source">The Evolution of Sex</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Evolutionary Genetics</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.58" data-doi="10.1038/nrmicro2670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCutcheon</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extreme genome reduction in symbiotic bacteria.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DMcCutcheon%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D26%26rft.atitle%253DExtreme%2Bgenome%2Breduction%2Bin%2Bsymbiotic%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2670%26rft_id%253Dinfo%253Apmid%252F22064560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22064560&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.59" data-doi="10.1126/science.1192243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDaniel</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruhnau</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High frequency of horizontal gene transfer in the oceans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcDaniel%26rft.auinit1%253DL.%2BD.%26rft.volume%253D330%26rft.issue%253D6000%26rft.spage%253D50%26rft.epage%253D50%26rft.atitle%253DHigh%2BFrequency%2Bof%2BHorizontal%2BGene%2BTransfer%2Bin%2Bthe%2BOceans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1192243%26rft_id%253Dinfo%253Apmid%252F20929803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzAvNjAwMC81MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.60" data-doi="10.1111/j.1469-185X.2010.00151.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merhej</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raoult</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rickettsial evolution in the light of comparative genomics.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DMerhej%26rft.auinit1%253DV.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D379%26rft.epage%253D405%26rft.atitle%253DRickettsial%2Bevolution%2Bin%2Bthe%2Blight%2Bof%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2010.00151.x%26rft_id%253Dinfo%253Apmid%252F20716256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2010.00151.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20716256&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.61"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Michod</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite> (Editors), <span class="cit-pub-date">1988</span> <span class="cit-source">The Evolution of Sex</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.62" data-doi="10.1073/pnas.93.7.2873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Accelerated evolution and Muller’s ratchet in endosymbiotic bacteria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">2873</span>–<span class="cit-lpage">2878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMoran%26rft.auinit1%253DN.%2BA.%26rft.volume%253D93%26rft.issue%253D7%26rft.spage%253D2873%26rft.epage%253D2878%26rft.atitle%253DAccelerated%2Bevolution%2Band%2BMuller%2527s%2Brachet%2Bin%2Bendosymbiotic%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.7.2873%26rft_id%253Dinfo%253Apmid%252F8610134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5My83LzI4NzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.63" data-doi="10.1073/pnas.0605709103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulkidjanian</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mekhedov</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorokin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The cyanobacterial genome core and the origin of photosynthesis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">13126</span>–<span class="cit-lpage">13131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605709103%26rft_id%253Dinfo%253Apmid%252F16924101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM1LzEzMTI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.64" data-doi="10.1016/0027-5107(64)90047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The relation of recombination to mutational advance.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DMuller%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D106%26rft.spage%253D2%26rft.epage%253D9%26rft.atitle%253DTHE%2BRELATION%2BOF%2BRECOMBINATION%2BTO%2BMUTATIONAL%2BADVANCE.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252864%252990047-8%26rft_id%253Dinfo%253Apmid%252F14195748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(64)90047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14195748&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.65" data-doi="10.1534/genetics.112.141325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shraiman</span>  <span class="cit-name-given-names">B. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fluctuations of fitness distributions and the rate of Muller’s ratchet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141325%26rft_id%253Dinfo%253Apmid%252F22649084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MS80LzEyODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Molecular Evolutionary Genetics</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.67" data-doi="10.1051/ebr:2007031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daffonchio</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Release and persistence of extracellular DNA in the environment.</span> <abbr class="cit-jnl-abbrev">Environ. Biosafety Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Bbiosafety%2Bresearch%26rft.stitle%253DEnviron%2BBiosafety%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DK.%2BM.%26rft.volume%253D6%26rft.issue%253D1-2%26rft.spage%253D37%26rft.epage%253D53%26rft.atitle%253DRelease%2Band%2Bpersistence%2Bof%2Bextracellular%2BDNA%2Bin%2Bthe%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Febr%253A2007031%26rft_id%253Dinfo%253Apmid%252F17961479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/ebr:2007031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17961479&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.68" data-doi="10.1128/JB.01237-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novichkov</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Trends in prokaryotic evolution revealed by comparison of closely related bacterial and archaeal genomes.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01237-08%26rft_id%253Dinfo%253Apmid%252F18978059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiMTkxLzEvNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.69" data-doi="10.1038/35012500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groisman</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lateral gene transfer and the nature of bacterial innovation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOchman%26rft.auinit1%253DH.%26rft.volume%253D405%26rft.issue%253D6784%26rft.spage%253D299%26rft.epage%253D304%26rft.atitle%253DLateral%2Bgene%2Btransfer%2Band%2Bthe%2Bnature%2Bof%2Bbacterial%2Binnovation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35012500%26rft_id%253Dinfo%253Apmid%252F10830951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35012500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10830951&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087085700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.70" data-doi="10.1038/nrg761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Resolving the paradox of sex and recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D252%26rft.epage%253D261%26rft.atitle%253DResolving%2Bthe%2Bparadox%2Bof%2Bsex%2Band%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg761%26rft_id%253Dinfo%253Apmid%252F11967550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11967550&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174739800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.71" data-doi="10.1038/ng1686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pál</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution of bacterial metabolic networks by horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1372</span>–<span class="cit-lpage">1375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DP%25C3%25A1l%26rft.auinit1%253DC.%26rft.volume%253D37%26rft.issue%253D12%26rft.spage%253D1372%26rft.epage%253D1375%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bbacterial%2Bmetabolic%2Bnetworks%2Bby%2Bhorizontal%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1686%26rft_id%253Dinfo%253Apmid%252F16311593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16311593&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233657300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.72" data-doi="10.1017/S0016672300026938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pamilo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Accumulation of mutations in sexual and asexual populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DPamilo%26rft.auinit1%253DP.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D135%26rft.epage%253D146%26rft.atitle%253DAccumulation%2Bof%2Bmutations%2Bin%2Bsexual%2Band%2Basexual%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300026938%26rft_id%253Dinfo%253Apmid%252F3596234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300026938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3596234&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987H523200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.73" data-doi="10.1007/s00374-008-0345-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pietramellara</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ascher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borgogni</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceccherini</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerri</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Extracellular DNA in soil and sediment: fate and ecological relevance.</span> <abbr class="cit-jnl-abbrev">Biol. Fertil. Soils</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BFertil.%2BSoils%26rft.volume%253D45%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00374-008-0345-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00374-008-0345-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000262825700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.74" data-doi="10.1016/j.ecoenv.2009.09.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pote</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teresa Ceccherini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosselli</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Leaching and transformability of transgenic DNA in unsaturated soil columns.</span> <abbr class="cit-jnl-abbrev">Ecotoxicol. Environ. Saf.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcotoxicology%2Band%2Benvironmental%2Bsafety%26rft.stitle%253DEcotoxicol%2BEnviron%2BSaf%26rft.aulast%253DPote%26rft.auinit1%253DJ.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D72%26rft.atitle%253DLeaching%2Band%2Btransformability%2Bof%2Btransgenic%2BDNA%2Bin%2Bunsaturated%2Bsoil%2Bcolumns.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecoenv.2009.09.009%26rft_id%253Dinfo%253Apmid%252F19828198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecoenv.2009.09.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19828198&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.75" data-doi="10.1126/science.1127912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prudhomme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attaiech</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claverys</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Antibiotic stress induces genetic transformability in the human pathogen <em>Streptococcus pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPrudhomme%26rft.auinit1%253DM.%26rft.volume%253D313%26rft.issue%253D5783%26rft.spage%253D89%26rft.epage%253D92%26rft.atitle%253DAntibiotic%2BStress%2BInduces%2BGenetic%2BTransformability%2Bin%2Bthe%2BHuman%2BPathogen%2BStreptococcus%2Bpneumoniae%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1127912%26rft_id%253Dinfo%253Apmid%252F16825569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMTMvNTc4My84OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Evolution of bacterial transformation: is sex with dead cells ever better than no sex at all?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRedfield%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D119%26rft.issue%253D1%26rft.spage%253D213%26rft.epage%253D221%26rft.atitle%253DEvolution%2Bof%2Bbacterial%2Btransformation%253A%2Bis%2Bsex%2Bwith%2Bdead%2Bcells%2Bever%2Bbetter%2Bthan%2Bno%2Bsex%2Bat%2Ball%253F%26rft_id%253Dinfo%253Apmid%252F3396864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE5LzEvMjEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.77" data-doi="10.1038/35084593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Do bacteria have sex?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">634</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRedfield%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D634%26rft.epage%253D639%26rft.atitle%253DDo%2Bbacteria%2Bhave%2Bsex%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35084593%26rft_id%253Dinfo%253Apmid%252F11483988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35084593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11483988&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170332000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The evolution of bacterial transformation: sex with poor relations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRedfield%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D38%26rft.atitle%253DThe%2BEvolution%2Bof%2BBacterial%2BTransformation%253A%2BSex%2BWith%2BPoor%2BRelations%26rft_id%253Dinfo%253Apmid%252F9135998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTQ2LzEvMjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.79" data-doi="10.1128/JB.183.2.736-744.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rius</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuste</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guasp</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalucat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loren</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Clonal population structure of <em>Pseudomonas stutzeri</em>, a species with exceptional genetic diversity.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">736</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DRius%26rft.auinit1%253DN.%26rft.volume%253D183%26rft.issue%253D2%26rft.spage%253D736%26rft.epage%253D744%26rft.atitle%253DClonal%2BPopulation%2BStructure%2Bof%2BPseudomonas%2Bstutzeri%252C%2Ba%2BSpecies%2Bwith%2BExceptional%2BGenetic%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.2.736-744.2001%26rft_id%253Dinfo%253Apmid%252F11133969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTgzLzIvNzM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.80" data-doi="10.1016/j.tpb.2007.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouzine</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The traveling-wave approach to asexual evolution: Muller’s ratchet and speed of adaptation.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DRouzine%26rft.auinit1%253DI.%2BM.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D46%26rft.atitle%253DThe%2Btraveling-wave%2Bapproach%2Bto%2Basexual%2Bevolution%253A%2BMuller%2527s%2Bratchet%2Band%2Bspeed%2Bof%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2007.10.004%26rft_id%253Dinfo%253Apmid%252F18023832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2007.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18023832&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253339100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.81" data-doi="10.1126/science.1218198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordero</span>  <span class="cit-name-given-names">O. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preheim</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timberlake</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of early events in the ecological differentiation of bacteria.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShapiro%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D336%26rft.issue%253D6077%26rft.spage%253D48%26rft.epage%253D51%26rft.atitle%253DPopulation%2BGenomics%2Bof%2BEarly%2BEvents%2Bin%2Bthe%2BEcological%2BDifferentiation%2Bof%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1218198%26rft_id%253Dinfo%253Apmid%252F22491847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzYvNjA3Ny80OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.82" data-doi="10.1099/00221287-144-2-569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graupner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wackernagel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Natural genetic transformation of <em>Pseudomonas stutzeri</em> in a non-sterile soil.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DSikorski%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D2%26rft.spage%253D569%26rft.epage%253D576%26rft.atitle%253DNatural%2Bgenetic%2Btransformation%2Bof%2BPseudomonas%2Bstutzeri%2Bin%2Ba%2Bnon-sterile%2Bsoil%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-144-2-569%26rft_id%253Dinfo%253Apmid%252F9493393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-144-2-569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9493393&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071989700036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stotzky</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Gene transfer among bacteria in soil</span>, pp. <span class="cit-fpage">165</span>–<span class="cit-lpage">222</span> in <span class="cit-source">Gene Transfer in the Environment</span>. edited by <span class="cit-ed"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">S. B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">R. V.</span></span>. <span class="cit-publ-name">McGraw-Hill</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.84" data-doi="10.1038/nrmicro1234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of, and barriers to, horizontal gene transfer between bacteria.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DThomas%26rft.auinit1%253DC.%2BM.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D711%26rft.epage%253D721%26rft.atitle%253DMechanisms%2Bof%252C%2Band%2Bbarriers%2Bto%252C%2Bhorizontal%2Bgene%2Btransfer%2Bbetween%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro1234%26rft_id%253Dinfo%253Apmid%252F16138099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16138099&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231591100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.85" data-doi="10.1371/journal.pgen.1001284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Horizontal transfer, not duplication, drives the expansion of protein families in prokaryotes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTreangen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De1001284%26rft.epage%253De1001284%26rft.atitle%253DHorizontal%2Btransfer%252C%2Bnot%2Bduplication%252C%2Bdrives%2Bthe%2Bexpansion%2Bof%2Bprotein%2Bfamilies%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001284%26rft_id%253Dinfo%253Apmid%252F21298028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21298028&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.86" data-doi="10.1128/AEM.69.3.1482-1487.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normand</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thioulouse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesme</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grundmann</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Relationship between spatial and genetic distance in <em>Agrobacterium</em> spp. in 1 cubic centimeter of soil.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1482</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DVogel%26rft.auinit1%253DJ.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D1482%26rft.epage%253D1487%26rft.atitle%253DRelationship%2Bbetween%2BSpatial%2Band%2BGenetic%2BDistance%2Bin%2BAgrobacterium%2Bspp.%2Bin%2B1%2BCubic%2BCentimeter%2Bof%2BSoil%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.69.3.1482-1487.2003%26rft_id%253Dinfo%253Apmid%252F12620832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY5LzMvMTQ4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.87" data-doi="10.1016/j.tim.2009.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Why do bacteria engage in homologous recombination?</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DVos%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D226%26rft.epage%253D232%26rft.atitle%253DWhy%2Bdo%2Bbacteria%2Bengage%2Bin%2Bhomologous%2Brecombination%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2009.03.001%26rft_id%253Dinfo%253Apmid%252F19464181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2009.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19464181&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267449400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.88" data-doi="10.1038/ismej.2008.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of homologous recombination rates in bacteria and archaea.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253DVos%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D208%26rft.atitle%253DA%2Bcomparison%2Bof%2Bhomologous%2Brecombination%2Brates%2Bin%2Bbacteria%2Band%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2008.93%26rft_id%253Dinfo%253Apmid%252F18830278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2008.93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18830278&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263261800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.89" data-doi="10.1111/1574-6976.12023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jennings</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalchuk</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Micro-scale determinants of bacterial diversity in soil.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1574-6976.12023%26rft_id%253Dinfo%253Apmid%252F23550883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjg6IjM3LzYvOTM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzMyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.90" data-doi="10.1007/s11104-005-4783-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vries</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wackernagel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Microbial horizontal gene transfer and the DNA release from transgenic crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Soil</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSoil%26rft.volume%253D266%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11104-005-4783-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11104-005-4783-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226385500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.91" data-doi="10.1073/pnas.82.14.4768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingles</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urdea</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutter</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Homology requirements for recombination in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">4768</span>–<span class="cit-lpage">4772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWatt%26rft.auinit1%253DV.%2BM.%26rft.volume%253D82%26rft.issue%253D14%26rft.spage%253D4768%26rft.epage%253D4772%26rft.atitle%253DHomology%2Brequirements%2Bfor%2Brecombination%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.82.14.4768%26rft_id%253Dinfo%253Apmid%252F3161076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODIvMTQvNDc2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.92" data-doi="10.1016/j.jtbi.2004.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazirani</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Diversity in times of adversity: probabilistic strategies in microbial survival games.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">234</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DWolf%26rft.auinit1%253DD.%2BM.%26rft.volume%253D234%26rft.issue%253D2%26rft.spage%253D227%26rft.epage%253D253%26rft.atitle%253DDiversity%2Bin%2Btimes%2Bof%2Badversity%253A%2Bprobabilistic%2Bstrategies%2Bin%2Bmicrobial%2Bsurvival%2Bgames.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2004.11.020%26rft_id%253Dinfo%253Apmid%252F15757681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2004.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15757681&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227849300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.93" data-doi="10.1371/journal.pgen.1001108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wylie</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trout</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimal strategy for competence differentiation in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001108%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001108%26rft_id%253Dinfo%253Apmid%252F20838595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838595&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.94" data-doi="10.1093/gbe/evs043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yahara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide survey of mutual homologous recombination in a highly sexual bacterial species.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">628</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs043%26rft_id%253Dinfo%253Apmid%252F22534164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjQvNS82MjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.325.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otten</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiers</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Microbial distribution in soils: physics and scaling</span>, pp. <span class="cit-fpage">81</span>–<span class="cit-lpage">121</span> in <span class="cit-source">Advances in Agronomy</span>. edited by <span class="cit-ed"><span class="cit-name-surname">Donald</span>  <span class="cit-name-given-names">L. S.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.325.96" data-doi="10.1371/journal.pone.0048716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zafra</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht-Grandio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Figueras</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Pastor</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extracellular DNA release by undomesticated <em>Bacillus subtilis</em> is regulated by early competence.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e48716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZafra%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48716%26rft.epage%253De48716%26rft.atitle%253DExtracellular%2BDNA%2BRelease%2Bby%2BUndomesticated%2BBacillus%2Bsubtilis%2BIs%2BRegulated%2Bby%2BEarly%2BCompetence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048716%26rft_id%253Dinfo%253Apmid%252F23133654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23133654&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F325.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/2/325.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/2/315" title="Heterosis Is Prevalent Among Domesticated but not Wild Strains of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/2/341" title="Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/2"><span>Volume 4 Issue 2, February 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/2.cover-source.jpg?itok=nk9yht63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/325.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HtzGytIb6EG9nmD1gPZaC3lGwAInyZIHKsZkbNIhCAU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/2/325" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/2/325" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/2/325" class="active" data-icon-position="" data-hide-link-title="0">Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459629" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9gEaCzKi1m3CxKORZNjTdQZHJ9697FTSHfgIOJr5hoE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/2/325.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/2/325" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-UjOj-U7DJoiOOoWVr5qdyLKYi_3R77SWAjLl61QyvGI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/2/325" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/2/325.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459629" id="node459629--2" data-pisa="ggg;4/2/325" data-pisa-master="ggg;g3.113.009845" data-apath="/ggg/4/2/325.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nobuto Takeuchi</span>, <span class="highwire-citation-author" data-delta="1">Kunihiko Kaneko</span> and <span class="highwire-citation-author" data-delta="2">Eugene V. Koonin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">325-339;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009845 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459629--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nobuto Takeuchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nobuto%2BTakeuchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takeuchi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANobuto%2BTakeuchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:takeuchi.nobuto@gmail.com" class="" data-icon-position="" data-hide-link-title="0">takeuchi.nobuto@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kunihiko Kaneko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kunihiko%2BKaneko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaneko%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKunihiko%2BKaneko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eugene V. Koonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BV.%2BKoonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koonin%20EV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEugene%2BV.%2BKoonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/2/325" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/2/325&amp;title=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/2/325&amp;title=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/2/325&amp;title=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/2/325&amp;text=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/2/325&amp;title=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/2/325&amp;t=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/2/325&amp;title=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/2/325&amp;title=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459629/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459629" id="node459629--3" data-pisa="ggg;4/2/325" data-pisa-master="ggg;g3.113.009845" data-apath="/ggg/4/2/325.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nobuto Takeuchi</span>, <span class="highwire-citation-author" data-delta="1">Kunihiko Kaneko</span> and <span class="highwire-citation-author" data-delta="2">Eugene V. Koonin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">325-339;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009845 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459629--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nobuto Takeuchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nobuto%2BTakeuchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takeuchi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANobuto%2BTakeuchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:takeuchi.nobuto@gmail.com" class="" data-icon-position="" data-hide-link-title="0">takeuchi.nobuto@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kunihiko Kaneko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Basic Science, Graduate School of Arts and Sciences, University of Tokyo, Tokyo 153-8902, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kunihiko%2BKaneko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaneko%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKunihiko%2BKaneko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eugene V. Koonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BV.%2BKoonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koonin%20EV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEugene%2BV.%2BKoonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=325&amp;rft.epage=339&amp;rft.atitle=Horizontal%20Gene%20Transfer%20Can%20Rescue%20Prokaryotes%20from%20Muller%E2%80%99s%20Ratchet%3A%20Benefit%20of%20DNA%20from%20Dead%20Cells%20and%20Population%20Subdivision&amp;rft.volume=4&amp;rft.issue=2&amp;rft.date=2014-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Takeuchi%2C+Nobuto&amp;rft.au=Kaneko%2C+Kunihiko&amp;rft.au=Koonin%2C+Eugene+V."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459629/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459629/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459629/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459629/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459629/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459629/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459629/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459629/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459629/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459629/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459629/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459629/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/2/325" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/2/325.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/2/325.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/2/325.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/2/325.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
