<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/2/341" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/2/341.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/2/341.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/2/341.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/2/341/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/2/341.atom" />
<meta name="HW.pisa" content="ggg;4/2/341" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies" />
<meta name="DC.Identifier" content="10.1534/g3.113.008706" />
<meta name="DC.Date" content="2014-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Ekine et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In animal breeding, the genetic potential of an animal is summarized as its estimated breeding value, which is derived from its own performance as well as the performance of related individuals. Here, we illustrate why estimated breeding values are not suitable as a phenotype for genome-wide association studies. We simulated human-type and pig-type pedigrees with a range of quantitative trait loci (QTL) effects (0.5–3% of phenotypic variance) and heritabilities (0.3−0.8). We analyzed 1000 replicates of each scenario with four models: (a) a full mixed model including a polygenic effect, (b) a regression analysis using the residual of a mixed model as a trait score (so called GRAMMAR approach), (c) a regression analysis using the estimated breeding value as a trait score, and (d) a regression analysis that uses the raw phenotype as a trait score. We show that using breeding values as a trait score gives very high false-positive rates (up 14% in human pedigrees and &gt;60% in pig pedigrees). Simulations based on a real pedigree show that additional generations of pedigree increase the type I error. Including the family relationship as a random effect provides the greatest power to detect QTL while controlling for type I error at the desired level and providing the most accurate estimates of the QTL effect. Both the use of residuals and the use of breeding values result in deflated estimates of the QTL effect. We derive the contributions of QTL effects to the breeding value and residual and show how this affects the estimates." />
<meta name="DC.Contributor" content="Chinyere C. Ekine" />
<meta name="DC.Contributor" content="Suzanne J. Rowe" />
<meta name="DC.Contributor" content="Stephen C. Bishop" />
<meta name="DC.Contributor" content="Dirk-Jan de Koning" />
<meta name="article:published_time" content="2014-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In animal breeding, the genetic potential of an animal is summarized as its estimated breeding value, which is derived from its own performance as well as the performance of related individuals. Here, we illustrate why estimated breeding values are not suitable as a phenotype for genome-wide association studies. We simulated human-type and pig-type pedigrees with a range of quantitative trait loci (QTL) effects (0.5–3% of phenotypic variance) and heritabilities (0.3−0.8). We analyzed 1000 replicates of each scenario with four models: (a) a full mixed model including a polygenic effect, (b) a regression analysis using the residual of a mixed model as a trait score (so called GRAMMAR approach), (c) a regression analysis using the estimated breeding value as a trait score, and (d) a regression analysis that uses the raw phenotype as a trait score. We show that using breeding values as a trait score gives very high false-positive rates (up 14% in human pedigrees and &amp;gt;60% in pig pedigrees). Simulations based on a real pedigree show that additional generations of pedigree increase the type I error. Including the family relationship as a random effect provides the greatest power to detect QTL while controlling for type I error at the desired level and providing the most accurate estimates of the QTL effect. Both the use of residuals and the use of breeding values result in deflated estimates of the QTL effect. We derive the contributions of QTL effects to the breeding value and residual and show how this affects the estimates.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/02/01" />
<meta name="citation_mjid" content="ggg;4/2/341" />
<meta name="citation_id" content="4/2/341" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/2/341" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/2/341.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/2/341.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/2/341.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008706" />
<meta name="citation_pmid" content="24362310" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="341" />
<meta name="citation_lastpage" content="347" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Chinyere C. Ekine" />
<meta name="citation_author_institution" content="The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom" />
<meta name="citation_author" content="Suzanne J. Rowe" />
<meta name="citation_author_institution" content="The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom" />
<meta name="citation_author" content="Stephen C. Bishop" />
<meta name="citation_author_institution" content="The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom" />
<meta name="citation_author" content="Dirk-Jan de Koning" />
<meta name="citation_author_institution" content="The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom" />
<meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden" />
<meta name="citation_author_email" content="dj.de-koning@slu.se" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. S. Aulchenko;citation_author=D. J. De Koning;citation_author=C. Haley;citation_title=GRAMMAR: a fast and simple method for genome-wide pedigree-based quantitative trait loci association analysis.;citation_volume=177;citation_year=2007;citation_pmid=17660554;citation_doi=10.1534/genetics.107.075614" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. S. Aulchenko;citation_author=S. Ripke;citation_author=A. Isaacs;citation_author=C. M. van Duijn;citation_title=GenABEL: an R library for genome-wide association analysis.;citation_volume=23;citation_year=2007;citation_pmid=17384015;citation_doi=10.1093/bioinformatics/btm108" />
<meta name="citation_reference" content="citation_title=PLoS ONE;citation_author=D. Becker;citation_author=K. Wimmers;citation_author=H. Luther;citation_author=A. Hofer;citation_author=T. Leeb;citation_title=A genome-wide association study to Detect QTL for commercially important traits in Swiss large white boars.;citation_pages=e55951;citation_volume=8;citation_year=2013;citation_pmid=23393604;citation_doi=10.1371/journal.pone.0055951" />
<meta name="citation_reference" content="citation_journal_title=Meat Sci.;citation_author=S. Čepica;citation_author=P. Zambonelli;citation_author=F. Weisz;citation_author=M. Bigi;citation_author=A. Knoll;citation_title=Association mapping of quantitative trait loci for carcass and meat quality traits at the central part of chromosome 2 in Italian large white pigs.;citation_pages=368-375;citation_volume=95;citation_year=2013;citation_pmid=23747631;citation_doi=10.1016/j.meatsci.2013.05.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=Human heredity;citation_journal_abbrev=Hum Hered;citation_author=A. W. George;citation_author=E. M. Wijsman;citation_author=E. A. Thompson;citation_title=MCMC multilocus lod scores: application of a new approach.;citation_pages=98-108;citation_volume=59;citation_year=2005;citation_issue=2;citation_pmid=15838179;citation_doi=10.1159/000085224" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0. 1ES;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. E. Johnston;citation_author=J. C. McEwan;citation_author=N. K. Pickering;citation_author=J. W. Kijas;citation_author=D. Beraldi;citation_title=Genome-wide association mapping identifies the genetic basis of discrete and quantitative variation in sexual weaponry in a wild sheep population.;citation_pages=2555-2566;citation_volume=20;citation_year=2011;citation_issue=12;citation_pmid=21651634;citation_doi=10.1111/j.1365-294X.2011.05076.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. R. Svishcheva;citation_author=T. I. Axenovich;citation_author=N. M. Belonogova;citation_author=C. M. van Duijn;citation_author=Y. S. Aulchenko;citation_title=Rapid variance components-based method for whole-genome association analysis.;citation_pages=1166-1170;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22983301;citation_doi=10.1038/ng.2410" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
<meta name="twitter:description" content="In animal breeding, the genetic potential of an animal is summarized as its estimated breeding value, which is derived from its own performance as well as the performance of related individuals. Here, we illustrate why estimated breeding values are not suitable as a phenotype for genome-wide association studies. We simulated human-type and pig-type pedigrees with a range of quantitative trait loci (QTL) effects (0.5–3% of phenotypic variance) and heritabilities (0.3−0.8). We analyzed 1000 replicates of each scenario with four models: (a) a full mixed model including a polygenic effect, (b) a regression analysis using the residual of a mixed model as a trait score (so called GRAMMAR approach), (c) a regression analysis using the estimated breeding value as a trait score, and (d) a regression analysis that uses the raw phenotype as a trait score. We show that using breeding values as a trait score gives very high false-positive rates (up 14% in human pedigrees and &gt;60% in pig pedigrees). Simulations based on a real pedigree show that additional generations of pedigree increase the type I error. Including the family relationship as a random effect provides the greatest power to detect QTL while controlling for type I error at the desired level and providing the most accurate estimates of the QTL effect. Both the use of residuals and the use of breeding values result in deflated estimates of the QTL effect. We derive the contributions of QTL effects to the breeding value and residual and show how this affects the estimates." />
<meta name="og-title" property="og:title" content="Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/2/341" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In animal breeding, the genetic potential of an animal is summarized as its estimated breeding value, which is derived from its own performance as well as the performance of related individuals. Here, we illustrate why estimated breeding values are not suitable as a phenotype for genome-wide association studies. We simulated human-type and pig-type pedigrees with a range of quantitative trait loci (QTL) effects (0.5–3% of phenotypic variance) and heritabilities (0.3−0.8). We analyzed 1000 replicates of each scenario with four models: (a) a full mixed model including a polygenic effect, (b) a regression analysis using the residual of a mixed model as a trait score (so called GRAMMAR approach), (c) a regression analysis using the estimated breeding value as a trait score, and (d) a regression analysis that uses the raw phenotype as a trait score. We show that using breeding values as a trait score gives very high false-positive rates (up 14% in human pedigrees and &gt;60% in pig pedigrees). Simulations based on a real pedigree show that additional generations of pedigree increase the type I error. Including the family relationship as a random effect provides the greatest power to detect QTL while controlling for type I error at the desired level and providing the most accurate estimates of the QTL effect. Both the use of residuals and the use of breeding values result in deflated estimates of the QTL effect. We derive the contributions of QTL effects to the breeding value and residual and show how this affects the estimates." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2.cover-source.jpg" />
    <title>Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"QjyaCc132WA4kM9ZF9shopWnBN0mf8M8B8K-fn-EfLQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459634","apath":"\/ggg\/4\/2\/341.atom","pisa":"ggg;4\/2\/341","ac":{"\/ggg\/4\/2\/341.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/2\/341.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/2\/341"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/2\/341"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/2\/341.long":true,"\/content\/4\/2\/341":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jZgg4Dmjia8JCEme7YjQNK4C1TE8AYJs9PJKhfvDAvsW25+98rGSnmDS9dqIq+i22CkTzWXwkR\\\\\\\/D9PmE8Lrdo35me9rSysuggN\\\\\\\/FGuBBC9jZ+\\\\\\\/yIHtqL3hLEx5gxPoUaTR3voO8hu3\\\\\\\/30zH\\\\\\\/pxgVriwDrgX3nJvGAg2UGiF85Y052F8Ear1v1PcBLadMdxGi7SYuVpOH1rp6DPg7TxSszyBmnE7ZotexeqHhk3DTUWQAwmtzrT9AoeWLb0mTDf2w0ZUorUJwMCSYC\\\\\\\/yeOpy0ItZ5jJZsC9x2xiMn4o7Zlj+FamKj3MSFH5KerEDdSoz03gf5wFiNN509XY1484mpJZhsSKtiJdgfYTF6p280VFtEPaHKCddSDXSt0oy3GdYyQNqlgmg9pLMgU45diOjAEuZtVwoTHkdeSS6Fwsf14CjcwcsxDBFWUhX1S0xR6GrApas+Wgwz+FYuqbh2IsnKmcJ8Hfc2+NCH3OmJWOzsYruKuU5s30ww7aof2il9RZgtu70REk8HNhTZtEPhOoGzNSjjK+0lv\\\\\\\/2IVFTqZAhdVx\\\\\\\/N5nd45+mkCKTzBJNfBLDN444nqgAfgDczH6cV1fehkcPEqyfHaKRxyV72gmHY6G4cHmcWT3LWRDfJRQb+oBXQNLet7wTXinW9toZkTtuy0L2AsqpiK82cv77QvIuZS5vrYSv0GcBK9rMnjaa3M\\\\\\\/HFNzLCFy6bBqaxzEXqXk8I3B6WNlGE\\\\\\\/w1c7DFMzfYPmVx8RQsYiAoplZNlvoViKdYevQt9Yo52rBbpDatm6gxELnbll3T4I2704t6vIgniMFvh\\\\\\\/okmJKVAlcdgHh2Q3PTu+juZ5J+7O+S5YuVyn5ovvx8EoTiOxL4dUj\\\\\\\/FNAAfkNqB2qyqwZ964Bjdur0L9bojFJOi1r9MkGqNANUvZ3XL+m3Z+Kq0nKSreyjSl1deZ1CBdO2YZMznRTWNuvQ0WMGRZWuX9crO2ZbiAeOxPf9zmeco6HlBACtaW+M5TC27yP5+JgxNDgsGN6Us096etDzuOBEIXDaCKkVjxEgtdjtZvNmos1DEXWhDykPAXpuIgRXwAUhf58\\\\\\\/QvLd56113CW32LgDq\\\\\\\/0LQacgRkqSgaIC0DRT6qcCRpKuDvueK+ovexz5vMowC9pvav6lli8jBDaDUPz\\\\\\\/GIfAe5gctQpl5HjxF7Cn1jho86gWz5g7Uc+Zq\\\\\\\/cBsREWBLMvNeZE\\\\\\\/3cMLm3qLrqDkKw67gdEz\\\\\\\/ZXySsfSljB0Kues+zJM07MBO4gIzLrza3Ao4fPrwT6imcQTE6TG\\\u0022,\\\u0022iv\\\u0022:\\\u0022uoDqkVLVdfZLg6DrFQMdXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c5a963e0c66bda4c458cf074ac65f315fb17ee17baee717d14990977f4a85603\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U5QLKWz4Htuj7LcmEbMBPFSCzU8KF7yOWu86vVdn8H28TOpXq9MAgxYq8u\\\\\\\/3ePHxx\\\\\\\/yD6TtOAul\\\\\\\/PGuplR9zqg8Sf9nZX\\\\\\\/h4oLHM5BOofT8+g9k0X5FRyXg5tnn+b\\\\\\\/bAtm3+20tc64dt0b3J+7noAVlt1rgvgXCorNGHpcEAMpjN8LwkOEkFx+1GHCFIVcQSB0odsq9\\\\\\\/OEyCiv6Or79yTLm6CLPu8Qcae9I8pTLdQCte\\\\\\\/lal4dvlaZiknJJlXurZ8C71Vz0L\\\\\\\/6jUYTdQBvb+wzmCYpCf+I+pD\\\\\\\/r3TAh9Q7b4JF108al3zpqFEA55pd\\\\\\\/2\\\\\\\/dopYOWeuNfWOk8p0S8XQKgmyinYzkYpKPLenmoRasMb4OBseLzhsesOYXm\\\\\\\/mJzFx4C6IHchSy8A39HvHQyc9Yn+P70FutZ3c0QilDeHHkcst3qod3\\\\\\\/n2+peuSIo5pUge7Al6Jdb3WOlkMsR1BaZerx1Mpcf4biL0lybCi\\\\\\\/gjr22cpgIh9gBhc1\\\\\\\/1YknHj3SFs\\\\\\\/FUl7yfJPzNJc\\\\\\\/FiJw99SonBEqbd\\\\\\\/c\\\\\\\/cPgh21VJxM58a8n6ERzfHL1ho5oqGRHpTV8dG6PhVR\\\\\\\/vEoAhM42qnO3B+Dol4vGJFRNdnEB3nhP8BE3C61miwpXeU8TKyW13ZdHoHsa180cf\\\\\\\/9wKIdMjnEJdT5DTJdw6m3OcbOaFtUew6oo6brvyqmuzQ80DM0Ylwkc8pBFQr0ytE8DGf5Vt9hslS4FhKIgJV7VOyYHlNWXqNoSMC7VhjvPpol\\\\\\\/iRQ9TQ6zZ76xK6\\\\\\\/xuHd78lLEa+4HfduTPxoNnSq8Zdh+2CeFSXhujh\\\\\\\/fIucXf5ODMD9HYpGXvUqUrdQKisgjsQMFq8CA6KGr3wTzxvHE40I1s9Sgj79sBsK3HgISEHbXSMQIQD36miR8vOCQSa0T4+gR0agRpjgeLvwgAij9C2fZl6RY0g+Wuqezvc02s+QtvSO\\\\\\\/gvVteVtQ3v5Mkw2\\\\\\\/OAI+B4HAH1DAYZrmGMr4ZToi5BsKtE82lrW0TtYbJ96zwC5KRbIJZYsxmOOpeCoPuRVoGP7IJG4RTnL+Uqo+1J3l7E6YaKG3qZ\\\\\\\/Gk08nIxNM69EJ12f9bTHq0xEIJXjTV2t3qLhgIJyCj9MOZUHZvAtmPc2OZ9u+by7Da84by2\\\\\\\/ROfTi3Bt6n0908cgWb7f5kbCA7knZTX7CZo9HrnI3NprSZY9661xvQpyjTa5Nv6AWptNXMXmjPXdgzyg\\\\\\\/ZJmQNQ3prWQpzWoCuBEPCt9CWEsz7nVExqk7HF2ZWUogT7Vz8+TmrB3O2WhJivtES5gpqb9YjtL5PxAvognOo7xlGhuUQ5hK5Ey1SowpvdfuAoAMbMdCIkf7+w3lFC94PbJSHIoxN+i8XDmFPFIGVW0Pgp\\\\\\\/00FyiidYH9MGyhGHOAXA13xu43n+EmaIbJd1XOYgW1M25LsoxIe3pSyCfuvZ5dX95TunIhl2xiAL8IO\\\\\\\/9XLkKgEB7cFiQiDKuFhorrkmH+6ST2xhB\\\\\\\/iqBhzgl16fAEWNsO\\\\\\\/QUxHwQHBlVvVEVvAwhyaiumCzcLHyiNoJsBxurtUBprwC1Gbq74QIfndmRVi+6gsCEY50dibsd7tbUDvZROfgfvfsgIki7f4lu6GX5Y2Q1QH15POgSJ\\\\\\\/YJ0whsOaNcGM0i6nbjo9TvAAsGtxKdmAyz2rir7YGRrtLU1fVzIBBc0XeLoIf+AamqLSHTmVpP6LGC+pbb2mtrHJyef4LhPuMdLzIEOP7dWgxpeArKKteVWYhVZ2Nn2h9IDWIQGeAwUaDQN2b9CD7gtVZ1YORXmSEo0R\\\\\\\/QE7jZAOWJwCe\\\\\\\/r1oX6NgQ3XSYlBAqAm2XaeHDXCznTCP+mm3V+\\\\\\\/T89ahfrsUGHodLfM6f\\\\\\\/NtMlOnrMNGZxZoBkgwlcAXO8unD8fg8Rf+WJvAMMMxN15+RJX9RNICxYHwMmjTdsOHWFYe3apCcbqD1xq0nB\\\\\\\/vbcY17CblICghkGMuQtoGsgQziSw+H2j5EMKLo+645RdH3itCGGG3vHq3Cx0Cs4BAadR9K+UkuY\\\\\\\/OmAUYVOPcfn9TE9UobX9\\\\\\\/4VSwVT+3Ns5zHA\\\\\\\/Naaw9EbsUvKm6PnIafgvjcUezQ7iMHB18LpNJSQwi7ENBl2HPJloO0twSU02WBkKNWFPxYZxz5l1TMXvhlKtZJ4Q3M0ZA4x7XQBOJJtYtndk2+qikjxka8F99nzRseAeGpJVPbcmDhosW8IMURUal0hyavn16S4MejUuYiz0HRNznKvUmRpHzOrnyflb+V5JsnDpYiLFJybw0PpQwOAreLP4jhxpx2xjMEKFSz7wPys0haKvPDrdbZFw6F10ptp4gHG6U71TrEXxl01WT+Rhlv63N3A4ihDjvdSDNnMLT6b59PlG56ACTLr+FnxvVFfSbG3ukbrW3x9nIG+iK2gEmBxEdVY405XUTBbQV0j0ysy87VVZCulwt9EHT2highkLD+QJXX1VZ\\\\\\\/9XWlfcfKQ31jSKUlOctrocTuE1S9O+jyOwnMFdMUMEw+gct8QFXDOjh9lCFdlBWyREqJ4ZYonAz+smPIIg\\\\\\\/JvNAPi0eUqVxJcWIyApoy2i4f6J3kdTpoodJ9owRoEU1yeLLpL7EW8Fbpn6O2EIfzPtF4rseAWo\\\\\\\/nMW2Qh66xymuUVkL8q6aq2ANcuAQVxkTWQU95FO\\\\\\\/fKJcN9pKhWcoF2fYe46qAtIIAvww1XIqWgYB1MJqvc\\\\\\\/fLZyQmp0A5sjya8YmnKmZy4HrYgMYEuQTKsNdGb\\\\\\\/fuEagCm+83n\\\\\\\/mvHAZdYT+LDetcwGTNzSWLade9Nu\\\\\\\/An0Rbdyfv6rFjHxNYEB6Yi\\\\\\\/w2kLRxE4GZB9L9lfxD3csm1Twn9spT5mccKXurWKsDKOv3ufnF8fokSzA5ZxMSjZV5NIS9673IQ6fWS8CffzQ500LGWNY2G2xqWvrtxnQe7BE2T3kFxaZ+ARHO0sTaQ046h+Pzal3mw6+bFhIbLqx7PVsfsEYPMXZ7fFY5fPAt0P2JQ4Pxc5iROYaJX5yigbMR1URNDnDGFYTvld8KttzYBUfi1UGFqnH+sfFr0aF5y2gwde5RYvlrQVTNhXfL6j6BBBwm1id52XzMycVlbo4capH5qiUVooYEq027JtM4Z\\\\\\\/uc5WmFR\\\\\\\/wtvABr\\\\\\\/7pE5Fg2EmNTitBrb2WsxaCmvklfhoQkPE3nFwviZfd1xjDZ6ehRh6Dllv5jLTu5H36Lj437E+FPOkvj5MaCwjZei9XykzTBFkvinK5gMz3R37rOzLUkS03\\\\\\\/QJtyVHQefCILwqpBsfBO7ImReF08gVp9Srz1VwBC1IJgUze18CzjTU9npuy1o6Wqt5Yy1YwG+y1lj1omsP7e1Wq+d\\\\\\\/28je\\\\\\\/Gg8VajMqOP0YPJgWxC39AiQk3tdmdd0t1\\\\\\\/a6Lg71tkJgaLv9laexnUMhlUZPa\\\\\\\/IgKghDp87depDcNZBQcxQy\\\\\\\/2OhRP3o8zplMudAjxw4xzvg01aVw321ht\\\\\\\/bbLzk2+rkfZ76kthXhu2Fiyc4D6sj4rgPCXzl4JZuiP70yMCp18hw6d1ZbWj6mzrG+iXAouiFER\\\\\\\/eqyHRCEzssREkx1QsAJiDoyfOhHVJFgdRNTHudYDw3F5zLeoW6vd+bNRxbsNK9blq3s\\\\\\\/WIH6PjJuBIOeXmqYAt2qd4HiNJ20YVSxKhke8rwkOREjI09qtEC6g8fiAuVeijJy1mVk6hV6e88XdNhD0ZCpCMgZMhkKDzhNDaQOLZ4TByd5PESDjO7FyoV5O1jc20DxouacwnO4+d0sks0KBD\\\\\\\/f6qyhBdsJDWkEuFyyefA3r69V9y3CH8LZLSUYKC1MAOSfzS+tJwMJq5NXRrkM+EyU2FXg3Wo9g8KW+onvXrk+U94PlNvs0bkvMdJ32sZQJV7j9sGmnKkj5VO7OFxfJJBw+\\\\\\\/74r0Hd\\\\\\\/qEXK9FbCi8ey8cfdGdCagUqvUNEKe0g+9ehhTRWI4TeR\\\\\\\/+vLZEWrAkqSEhOsF7\\\\\\\/tyFU+ZPwYb9LMNshSISoX932hp6fFXVcgHZyDS7+XahBcfL1ZuA1GElHlyRbV9E\\\\\\\/K4S14qIDC1J0RFfSw8ZVcKTNPQPPIQH5WRAFVjyVGhu4Ijpbaq+yUN0QtlXQeOkkE6QipwjTU65B4GjznjAAGMR3HO3uQHEhDjdB8UxDrf03CLk\\\\\\\/Yf977mbWTX6Zu+UrWmxkTeo3ScNe1G7KW3xL7z8HgOD7cSmV4Cl58h1ApufgdYkH2qiWZSPfiW9eubOUUyKngcdO8Lk\\\\\\\/PTTTVug\\\\\\\/ABMoU8E6k3yHaMSf0tYnx29yL\\\\\\\/btc+Wqtbcp3eDCV8zCvG7tozh7R1wSK5XmTgavf7RxaP0IcYbHF2qXF01cKTHvKlj7\\\\\\\/Tj+oL20BtLqtdr2r2G46hD\\\\\\\/NO8fXxMLbcDbTUaUeFfu8gAPw\\\\\\\/E4xV\\\\\\\/cYAXNW\\\\\\\/4Q11fH+mQ9tACP9g48y3Sw3DZEbRvicyfm0oMLdw87GQKsnIP4rrQ88RNYeCdvHGNlLwUEXIpfsGlTaq05eKfulMDmAlNO4DdYamNioUarx2xGskuBWvMYflvW\\\\\\\/eGwGNz2+B135j13n4rBEExrLVxDoMAhpOaZEKHGkV\\\\\\\/RaXfmPPY4LtoxfbymhzoNnEXq7xEF2ALJ7SytFAxQAo+bW7OOmhTH8zTwcZSSiT\\\\\\\/AgZvyKqJMIKaP3cwM98z+gWapdw1IOQUKFx0N4K2lFHKq3n7JHijmqN5UKh4SaLsko4MDJoI+dxa1GmXNE3B4Xzl8+pk8pcvqGJMTW4IDFZ7x+Q\\\\\\\/rpSfoHLe817HOoZ8M0Y8TZdum7CS0niO8niL9m2ctqTL1aQE3TZgRSow1rafmJofvU5pxgNEWB5KgEd0sD1Uf3BZs\\\\\\\/Xz5l98EO0qlI+v\\\\\\\/0EW3FYBeg42s+lixX9KeS7yJrR2yoVR4r9pz5OjON2Vks6vwSAVtjqqDr7LA6kwiagz3TqRSmTJMVQ2XPeUAaa32VC9UjEfWY8X6xljJoyMByh1z1xMkgOUBks0oWf5FU05i4fsenrMLStCNfqyuiIvomc+PejlxMH2xAvnpvrnaO9hnzE\\\\\\\/m8JEfwW6ecv4wsUuTor3qlcimq527ohBfKT6WmcJdb8rMFSvEjI7fwnnQgluV+5kPV9wgho7BfuNHq562B6fRcCgq4fG5EROVBvYsI5rGQym2\\\\\\\/lgdCpf6vKXaGSEvY+\\\\\\\/3pd02JZe4RoBEqczOuWd40UzJFYlzdVxxmscAf4aGgx459AYHCOmweHlTSU6pzbsVb0bLzdmfFIy\\\\\\\/3DYGlu8kSXUMhf5E9IHyRSKGHpORXvnryM5W\\\\\\\/AlPXGmb+sbBNOR\\\\\\\/Jk2EL+j9q079Excwb3b\\\\\\\/QqySB9tAaxDPTf3yhGOBOnEoWNOamOUTlDTLToU6BMkIaSQprMicqEv\\\\\\\/9HfdlyWoPe0\\\\\\\/VeWZws4SQFJBGnDRM5YnUCiOLGEhY70ZBzCkwR33bD\\\\\\\/CsU0F\\\\\\\/GVAgW\\\\\\\/PvKc8YxWk\\\\\\\/KCmpKexwtTWb8w2faErNbNnc3K18Zzv\\\\\\\/E6VOCuBR6D3Xt4MCHiZdr\\\\\\\/Fj3gjWnNlzl9okoEQmQpOXpZsQgbe7cmZ4Ax4MroVaE5r3cyFdxyKjRa6W1s6qsJQw8uHJDBi\\\\\\\/MCPrQI7naBe5QY07cy7dv7R868BNd588AAp1xOul17c+aD74vkvXKiz7VrMtucrsvf\\\\\\\/shcBn1ld5LvxdF0Z1V1PwEV+aUsQHhqQU5XWWVcECye5xeCcsAmtrtmcaawQzPvMbBP9eEAoYQNG3kniNwk1BZChcZgHTIlW0kvjYfeReaR9AdqbJ8BIteP3Zjb\\\\\\\/+nCXiefPhsjIRnQMU4aZjAw3uheCSOEgK4Alqxw0dn+JxeFuF+X\\\\\\\/fdi3feYupwlLOfQMbavZAt8738gn3WKkw+qEAkgW+NBxEjZ2ZNarj7gmNKA0cWMbGn7rjJErmgITO\\\\\\\/Mp7uC5UupXxQHjPXv3EXoVrpypvzzaZ8mgCA2E6+CvjJobBKtgjLrgBicGTfBYSD2gOZlYTevqJM+eNfRsQyQ8yWM67yMYNJEx3btWpGv6SRLEqZjdY9MqDpHTGXOzmwRPViSaOOmakrdltIko9x3gpJky1UUhf5UXbUGG8Ot1GWdw4lZ5KGU4lqpk993unjQS40brCo4BT1jdmLJKn955Sl1DazhwYZlDPdzfCqm8feHeYFxQznfkaxGOWFV81zJG36akBhrI8VAUb\\\\\\\/+E9hvzbwhxq4G2ejU9g2NaX7nODTcGRa7LMtrUSUa9kbvRdt23ULoFJfcWVDUZPG9C+yTPqM\\\\\\\/z9JToi+yDRZZ3mB8smLUJkSw\\\\\\\/7tNaJlRDEsCLn\\\\\\\/e0diTLCG+zxIV00\\\\\\\/0JuFaR4UfJ2SWHNOBt4iYnKe1bs29EhpMMDVTvZ0W0Vg3oEZOCtZrHgQ5q0H8AtKXXY6AZhYZnAT6hTLLdDa7oQGZPgRKpy7ChGpmyQg6PEq6D9lnb46eRe6yTIbWt50+kMCkmARs8rdW2NZMAfJlxP7gArxKUKx+0SvAPmviJQF2qgesuvuKpwJ2+b8IVa8opidpOQ9vZJV8u1EZ+du4Fs2Yk4kkn7mQ\\\\\\\/xN\\\\\\\/Je\\\\\\\/glcw7qwH\\\\\\\/T41hYB0OtjiNvqcESWkbdRnp68KmHf6IpmHXwt+blbix9b1ZxB\\\\\\\/Y9mMJ5P3aV58Y2EbqYECzKyuXo0KRLh1k\\\\\\\/qVS49k7JfQYZow3McqZONUfMzqtkDBfV5VvOLGsXecKTuAkVfyc+QADUfIOfrjPUzeKaLSptwBNtVtCyRtPBBqo+TD\\\\\\\/P6rZadIOrMevwy+qZ5s16YnK6V9RhzlBuKz5isvUuV1IS35TpKSQ4VPw8YR30RsdW0a49RnXQe4SMh+fsRfSfdnaxsKBlbRkQOGaF98kNTCGtfGor+PpLTuyv0OLfh8T9FHGHaN0pI4fxZaFavRHBteWVgIAYLgnsrOWBwLFRp29HwP7ycMrGyz2ZDX8w\\\\\\\/XJdSYsO6lsnxpddLtMNxQwdHuMz9ufgzEGZb3f7z13+OJERLuGulOH1HjEjtwGh+uA2qrAwq340t9kNxwSYkin7\\\\\\\/KAJ8QkNAPajg7jP0\\\\\\\/gH2utuShlOQwL6PeQv0+M\\\\\\\/HMSzsj3zgDvwMvGJ1dGuTo4bfy8LO6gjzXKPEDhCJDVvZBigKtpMtoUiYqmUA5gBf8hlS+NWiFszqe3UGkh\\\\\\\/zwSZ3a5nAQsEj0xOgWHo6ur63q46rY25J\\\\\\\/255G0Bs55Li\\\\\\\/R3p0NjvFtPBk51mnuYWnfs+qv1JWkclREvf\\\\\\\/ox+EpPfrLGv08cWZf9\\\\\\\/LOGY2\\\\\\\/g1\\\\\\\/Wm+AO7Ir8S1tDGuiTi8ionWhokubQV1TZRT\\\\\\\/f51+lzQ8Ta+XJdvO97LPaVJTE096kBFli3mXx\\\\\\\/lTxaNQSmQBO4BKpMEGCLIBJejAoVieypTSP6n+DXVgrg79bGrMSZSq2nlu2a\\\\\\\/cPWrxPCOfHHKTYoa2D5GeaD7XfG2PjMyNi0jsv8sDbqUFp4RQVbqA2RegDv+\\\\\\\/1lFTkXGd7ca5IJk2YJG6gZzfSIvwwnhk+0ppcYvpi0aRt5BEZPl\\\\\\\/oUzg6GHHLRjAQS6lYkT1S0+9g1mjMnzelqJZjqPjIZfCsIdIKYvF+mF7FX2myKwqftVEzzOlS2PBws9J3sXb\\\\\\\/CN+EE3RTwDKzyZuBxNYblVdXqOvc7iunCh9x8Sy3qqZluDoHo+hlGbUldalN26o5ys7jsbn5AWc7\\\\\\\/EU35h7HFxJMFLPZyesfmCC1MdSa1zBwNNa9IldKA3Ctjgd1QbdiB6mpAtJkkd8ckAUvZhEB\\\\\\\/t82DtM2RnVDvcvaZencGtl9Oxs0t9k7op7eI6SNaujaxjr1clLPrA6hS8xRQbBdtBCwCd3BBsrmpsnCsV5s2stiinM7cFLfkMdLmwybDpXyUDradeHE\\\\\\\/EmvBBJTW0OTaqtxg5iTLTcuoMNJLM94XUAvl4obB2ylmFNmq7QT5\\\\\\\/4Pz3B2w6nRAFr3Q+7u+6IqxvXdhOFnPXSEZsHG0ANNQubPJeHPP1fiZHmLmIGaX5XEZVNOQ7uIpMjPmNs3ucK5uSQgdE4WZvSTXtpGd5OeAqWvV5Y6hjtFaYLgUEIJx1+4y1\\\\\\\/6aPzfGjkA4c9tBSx\\\\\\\/P9Gr+mH5UHGS1PAyJsUKDbVZu3QXQbtbY3fz6UCZ+RqLKmStbGWBG9q8gMb8oIxWsGvO2BfDNf\\\\\\\/p\\\\\\\/MDELk1LQzhV7pRDBbUJl8Kq+4ssAzo\\\\\\\/uf9+MTfa\\\\\\\/J\\\\\\\/v1uoe6vWRC1+zpinrW+6HHxwroYjATMjjE6fdPGfBoqz8NNAHivOA40bnf8kOMP0J1G3zcje6TcmarZtsuHoRALZmbV0PeoRwoI85DGtgm9KZegmRzSc4ZHdRALXAd+wYJumlYbDJnROsOsETltmB31F8L6FeHSExX\\\\\\\/L\\\\\\\/5bowVFpfHJ6F8KB0PhbefPU\\\\\\\/1hzvtfb4W3Cs\\\\\\\/1g4h1LffMFZMtNuG96QG3xUhBd6iEfch\\\\\\\/5L01xqMLJLWWLO0sEC3\\\\\\\/bF14BCATayxK5N0cdzX\\\\\\\/Fh3MtRUGTrSRi3TC9SaueOVdQu9E0DA3r7Mj0ybdISnz4+KMxjeuYld7PtL6Y7L4zku6kD7c1M\\\\\\\/9F65LzGyRsYlFOTIeISMzc8U3l5fRdYJpKA5BA9zJ8gSVqV4ZT6DYfMHBmgDg+97OSX\\\\\\\/XKdpZE1+pYMrO9VMmF+Jyx8T6Qlvqc2Pl28b1yWC4MIN7\\\\\\\/PaD1pfX+eJh1Xxr2FN2L\\\\\\\/KU46w82bJaY5vWWR6ppvI6EgaXjpfhAik+JU0\\\\\\\/\\\\\\\/6uESvUOkv27db+vTc6qWaXj9tMNzYnZkyOY6bAhks11AiEqFZLQmOfgByjtUVB0KxyRMGkN\\\\\\\/hW0YyssRbz0quChLBGCplqlRFwfhFg92T+zBe0o4thVAlfPfkQgTRLqnGwtENPyUrf8yDBhqJ9ogTv+o9\\\\\\\/9Pkh9ydFxK46ZnRS8AxQfhECRIQLd86Cv90nSQy+65IkFEHOgS\\\\\\\/XYbA2tmPvHKPnZUO6Q7kdqgYf+iGPrxL1HkbdKi9IXEN0G7E39Nu6Muy1VIqe9iwQfjuf0pvs7+1mpuNYNmPbtU8fpIw8b\\\\\\\/alybbfeUJVdjrYvo7Lxc2ozWKq7ya2w4TQ76lhhI6B+suDpDJ7TqcxYCf1v6qceCZHLVewZ0+AZFHRVmDhHkaamtGZvWhKQh+W0ORmTGSD3sMgehIewZ5Q0RvcKwvV2up8uI2yD06F4RfRubIr2DEKjFPH1TsAAMKfWHUc\\\\\\\/RGpU5JXTHzjN4gLoQxRdGRNgo9bb+YXxIFvtdMG07g8\\\\\\\/TLthdQnyzxSX7B4UIVVoLaBw3MbTyG6UguJr\\\\\\\/et2tsTS6BuM0sfSeovHn5Qh9ncwM87irZVI3h6K9IEPrWHt0abUM3nyvI4kzQywqA7TYPpdUYreUeNEmJ6cOBmOAQazq7n\\\\\\\/fQY7LMSfIg9l6PHMgrRZDJXo0qMAc6NmF3A7nW77\\\\\\\/jHaUAeYPC2pZK70UbHCJGt3GiUh3sxo8fUBLj8Q9E00hZTEBbEzav1ZOZu3iQbLImwpPGBeZ+2uBzogxDtjBFuhshS+PT\\\\\\\/DsXjMM\\\\\\\/DoyYXGkPuNMcLOFItHQLcO4pwhyT2Flg4uPLEri1Tmo+PIUeWCWz7pU1mITpSK+V1CQG8NUExwXalJZ\\\\\\\/qKgGPp04gywnpPTf+hpiu9MsvNi4dXJIn516RFoWHZVFkJinWQLaEsAZGvAGAaxkRap1vMT0ISVbfhcF6ZvRwX2PfAJP\\\\\\\/ZO0J9PbiKo6qpSpipe9X40RQxnI3c3CP1qJ\\\\\\\/eUh2CjvbF1QwIHOV+4w+LdjTHKHvdd27Iu\\\\\\\/hXccmoQEpHpvr59NEdh+qIm5kF+jBuZYu+ooNiw4pff8j\\\\\\\/FxT1MMDzo7ImxxrYh946wG7n7KZCfGapOZABOFoYOTNBxA1kHa\\\\\\\/hkpcH8OC89Ul2ESPZjzi5C45WWImJewwrZvFO\\\\\\\/9xzyjQCpnu\\\\\\\/DCNz\\\\\\\/uspeO7BTILyc7V7TuS979f3csDDWB+yckK9aRYq7YTuhPJeGPRWdw3YLH5nv52MCRXn42gZquDMqHTnNyrLUOu1kT0MWyZq9pzT71hgC9xoKIVHngsSOsp9jEAP0hrUQ61uWv2MqBjjiL4qDItBvTRFMbrDrv3+x8I83qW0rJzs3QXdeeyT7tsiSgkL9chn+WMVCVJWCgsW2JdA9nWJYiizLJ+i008gDh3PQxCCIBoE81uGs3Szs6dk2SSQv85hdcXwvlXGsgxBcxChwBBsvx6cXBmgbT0LPlJXkS0HGdg+CaWsPkRgGDrlhJltONwg+sjrMOqMf4pne4yMfMVEcd4LRc4lVyUL5lH+4TtR74oSReJSBO6joIw3lX\\\\\\\/e666vkqrgmqb3R7PEU1h0EJUyTOX27n9an4pnQcRBbzz8khTWC85nxVcV9JFD2pxzVUIZnI0uLKJyfHAqEnYfmjvx9nb2XwsJjySrU+kMKYmIsim7FKHuyD3lT5Abj0Nzzt9yhXJkqhnR4r9GP2JJzkpo3rOj4z+TgVZ0ps4Xfhl43WRyIxub7SS36zakdSC4NRrgckB3hEdA243yIxwGlGR5Xb3hyxjWtlBIbzuexdeEmbO8FyAENaGoYXPdG58p9ESwHgyh51cYamXtE6w1qJg4N1tVWi9MUulsLW4wI\\\\\\\/uG9cwASXyrfPuy5k0zad7tJQ+IzzJNTeqn93MmVOSvq7\\\\\\\/BPwmyxUUbxMciCdBud8wxXnD1T2dUSdBwDV5vjVH241lsR7jzbBv7UPfpc0TDtn8a6Bj4PSAHQ24tfwGSQ4HSvQ3GcbXyCvDl46RcVA3nY+CVger7L6pHxu4WCSenoBYdlxRRIDXMaMwLDYH8ep\\\\\\\/KAt\\\\\\\/qQe3Toi8yREgoydcE1T3etzh9uBmHkSyXKhp\\\\\\\/14O9qV3g1ezeGcDqmUJ+F5onXl4Y7vp8uNffXiHWq3iBkNZ0NoJyHJq9GUR8HkTfePaQBakcWfj7cZfxsSDcFdfSCMeYGd0ryo3LF9v9AFk8Orhz12xKYQ29jycGcxNFfKodvJNH7qGAEsD\\\\\\\/4uSWycn11+qiJk8h+hNrtvLX7KMIDpBo91Y4NkxbJAcx25bjJM6FJYI+zmodLOhoklEYezbT1UYmxj2gAhtPMDide7J1xLilLqlDgen3oM1U89z9W5F9\\\\\\\/OZVanA1zx4FTYcx\\\\\\\/sDbcLK9iUy9ksghRO5D89oTTqzOkao9bJdD8JMIwuBMt+vUQb3YhQ4r2moq7O6SbTt49wvQ7FOkcv0kgnBL7tBSwmiFG9hYPU1Hla7VcdunAXZ5Rk9j9i89P43Y0naf6aa4cwEX\\\\\\\/MpFZ50\\\\\\\/WsYWbr+6B0LI5IMuXI3qxHM5dXZ0jbLTDHqqdztpubmvWKcpnaGqFPJlkOW7a8N\\\\\\\/lD+aqx\\\\\\\/aL\\\\\\\/WMg\\\\\\\/tnBa1+U9PptSQo\\\\\\\/3rfaIhwPfLbRX6i9dPRIo92CueNRsbgvWO\\\\\\\/6czEaDrgAvlq1cbHat4LrOj4DVy04JWaU4NYhklplcsv53pvNNAoHhKPh8i\\\\\\\/HKp\\\\\\\/Hi5yIw+Civ4G1QjonspU5SU3orleVRukCzOHYp\\\\\\\/scc50OjhIotxFnKM6SU5KVu0DND6HdImJ1n16LrSO9GS87M17nOob881HxwBJwyiC4nmsOg++qTPoR0ABzPsqp9IXQ4Gj\\\\\\\/mGa8VyrMTSDUe7RKzazFOZ0mO2Uwu3BPHnP8RRYEF9MvcpQL8hFFT6GT303ZxmE4c5q0TT80zFWZJ5nP76roCoc5L6P8irP8wK2FzAhqhegrU8CVQN8AOBaecJOJJ\\\\\\\/eZBhubUUz2Z5+1sq22asfizOYE9zTvKIns9nv7JqrWK2\\\\\\\/B84GD1NbgKeTfHuwhxmGTQONS\\\\\\\/w0SdeAx9b7zFYa\\\\\\\/dwfaK5dJsz\\\\\\\/019A7ozWgWu1bRu2cn5pP74\\\\\\\/98aN62CB2dBA0W8WMLkPXlWvVAK1XT1T23aAHBCrR2cSme5tifBzllq3tzuoEjKGqg40Ji9rUv+qQIQDFgQPKzkCjfgAZc+pY8iWypdB3\\\\\\\/sn+p5AF55WU5ru+By4IfUKjkclqLUXFI3\\\\\\\/jntHTQ8Dd6prODftzuEIMcoyJ7TSnv2lFJjtKWmDOmSwD2+6kdcu74Mrr2DDRIrM\\\\\\\/W0rCsb90Zalqxz+rCXSShpekj1G\\\\\\\/xB\\\\\\\/wHcs6WphWl0yTqZC\\\\\\\/80odBryCt8yQSXbaJh\\\\\\\/cDpv2QLSdod0Jsyo0acTbf\\\\\\\/ERtyVr4O0YI7c7XlUzocf8nDQfXs44+32DaJjqM0pwEceLpZJRp2HysTa7DrOY0eCrujJWr4mAX0mV9Ei46KRRYAbXkilm1+NZbitB9M8du8KCgV5EX7us6E\\\\\\\/XzmMYo6HHrPduQwYfi5RrASjRYWqn+dnF6gMTcpa3ajFoXikB0JF+JYxmN0BPU\\\\\\\/jUKnCU2+ShqY9pLS5UBqgCjBczUkHN4PJByffT1\\\\\\\/IR70qEkCdi51vtE1gycf3vtfe8rmU5XzxvSXbs9SQMq\\\\\\\/D9pHr5MS3jWq1sFdlf9ZFoxbwrkJVOy6M7EWm4u37X\\\\\\\/uLchDX6QenbXGPjgyi8a0ZoVqCr4U8LsnARqxvPdKRN+KNTyecjY7+EgSFQv020QFPu8ggv6FXqKfBZBCKTRNCAWPP9f1B81mkC7Cf9yM9VDzQYkF1qvR4cfTxp8GK+FDZwiEnwNEd1i0UZT0jFmsUYcqpopDGpfTjdBhLahy+DoUId11vSQL42XOHU2CgFv7+TvXji6ySLMibEgUqxVh8RAIJMhwTNKkpaRpIf1+ZmLJmmSdz1+hS+oHZ1J88jskfntQUjLKX\\\\\\\/mwLWdIPlnbxqAx7Fcsy61vSnsUnTPeVAER3htGe4JfT676Bg7fMHBCE58P4uGNB5y6o3m2Aq9Uqf4WhZ0qE3nbESmhC7F3pc9Yhv5X9H7koTsG02BpKDGVc6ld6zlzXRGoAEGJ5jAKXrwUQ\\\\\\\/nM3wCsd3aaIvDjCf5lUoHe4bJ40+OXswXOslaLsHvCJ2fIngcUdx5fUeaBbml6pBQ2NtMZzKwld3ZA9DdkwfxLF7hQtnUFNc\\\\\\\/\\\\\\\/ow8nCBmaCNzvPHlvPVQyEpwHcv7TA0fTW8v5Bc0jQ4CD4ugBpLPleVV+7cvXsKOP5AOxcmySzdP\\\\\\\/9kK5dnDRA9OPm\\\\\\\/Brg+uGDzUMQd5im2SwMVnm6ujoLbR6FwIqFFAz4LhXGq5fqMSnVb+nw9u2g1eNhgsL3MrzamAjF348AH28wlRLCF7V13OjoQIgc85foOq3gQio3asYZshep2RiVtA04FD1qXgnU12zZHQS8P1bnyGil51N0xrJrjhf\\\\\\\/pJvG6vW4truYobCs82hjEogu08EZ1BnJYrLcqNJ8DIZZZUBf2pXAzVTp0HT2GMQgoijhv9YMiRaY072S8\\\\\\\/UI+RGF2FZS2VETj48l4CK+isX750VA9WWVXJ4DO9CXKa+05RYBl+DGUlMUREFPcNnJ7CbV\\\\\\\/fqKNVS8kgiJ6bHJoqlTwJtwyEeTslz0UdNZszmFpAu+1b6Te16sPEjj6YnWgD0mKhuxX3yq6YURNl9RugDCfFTxBhIJ39UNKszOAMahuVsNrhOQsU13snjJXbVOJEvw786caoo3Z2DTPZ27ZyyfACdlkXgZSTvVWcIM7gMP+8I5+oKou0UaLVrppQLUhvvtYlT6CviCdXRfL641zEeQQJGmHzO9KRka697B0HQ5V9FO6k2PtIHz2BQN6VtNn30gnzESuYXnLETs57CmAEz3jH5xPXbNPkor1of8hgMdJTShzsocMyYwsfU1+XBjUnPluKN1taK+BYQl2ZUr4Jz1XCkOakHZWusk835aw25dxsHsbQLk4msOUy4LA2RZRLdqAanZfi2DRXNn5\\\\\\\/8oWg9c6Exj3A+bXfb1aMMk\\\\\\\/6W8hWunycJi2GPUPE8z5mrRyg2\\\\\\\/9lwMMY7V2scaktefgtiSB3ljH9tMCExyqH+JtCjKF7wcMtIY52pSQScKHfRnT4NKKuT+lLtrXGZgXp0Su+rPQj\\\\\\\/hX4r5dy\\\\\\\/mWkRethsLQkkMhvRPzmd31tP\\\\\\\/srrHs7aUN\\\\\\\/P1hebAbFy6Py22RAakXxo0hNSQG5DlVJy78aM9ck9Dmv80GE\\\\\\\/j0\\\\\\\/nJvjHMsG0gqDgbyI8pg066ANwD9hdQ+7wkRJhxRxvNNqVCjXmu+KYdg2GZC\\\\\\\/CI9aywDy3UFHKbbwpPCuzqcOdY3fTJWb6GmAL2y\\\\\\\/oLzoJ8JpfOiqN8CY\\\\\\\/cGUTB6rHvhjQxMK88uAAv5u0pb8HKMYbBfYVNsELCmXrQcMF+E17Yl\\\\\\\/fam3LxhCp9woY7R+qKLkyPdMw8SecmxhkSkeys87fUHZG1b+mmVdee3kh6bSfj0zSRunF4S0GHerzX7pBQynSXY1gWSPRdY7Acvyx8I2BlKtPyobjG+JVTn\\\\\\\/k8XDg4NRte8wUSSUBBSfcUHRKgJKMJEI4gbdghH3a2SPt\\\\\\\/EwJs6gdZiR7Rl4bg5EiLTGZ0tNwEj4BxaPJ\\\\\\\/AMjHuJyhtIdcrd0TKU+JYO\\\\\\\/5+GZtReLE+QKjc3y3vK3myemb6vSLR6Wae\\\\\\\/CCSo9VCkdFXBkcapobMcfe3ioh\\\\\\\/gunoYgKEAoUZWDdhZkcR8rrKdXGMiqngVYr07H06ydiRwIqEJoKyFUhlgtOAuPG9G9VRYo8ZGQKlcOpaPwkMnIRflrUQGMouYTM2bDaHORHg2fqFFWeCA8GYAq2g9ZpDKMHkRFnhroKBt9HOkNBRtGs84F4bU0+o7qCtbpNtcD46DVVhW3ZMqGsy1WtSnVCsHD2cgySgdKy9fptXVeBJu07Y\\\\\\\/FGvdEJLlY+NG\\\\\\\/fO3G6nqJrhxxIai1wubhj2EIvf9k9jBKtqXfsHewZXhtrIUU5Du1Rjn+oAaif0NGLsLSZIyLVhgfRqUfEZZUkx14G4ZXLJoTzwbj6x3pecZ1hnqUgp06iirVnrtXnx8CeDGnajn4k9d5DSGklMn4z2H4utGcXYwoW0du8d6TBaZxqu9hgOcHo9CGt8bUPB61unoBX\\\\\\\/nP94+nalfgxjJekNX5OkRG86MVlduUBATP7ouSONKAwyUlrjQJHI22DuxkSSA8OdOFK+lVn51pXfBTmDA7T0e3MuhrBbm52IEKn03XZy8\\\\\\\/9eNoAkUM0v3Ta0LEXy1PIOGgwYJVIH9ic2TWAO5a8+2LrUXEFPILk2ePLtJE3bR+xQxZ7viJJrPtUits6NGnTJSGL4Rsv0CICssaEVRNjDeCyFqi\\\\\\\/LdqUo56zebzlsOTv2JKQGZHGXqrSR48Rjo\\\\\\\/LYR+S2PSnzW8x3keqSWQcdivqP3SOp1llxZDSnoeZ0wADiSTSiHUYQRZZ+BDFQqhkNh5HlKDMzFFjkbGSsPwa6KfeW9cXPIKuHHVMnE6dKST\\\\\\\/wL1\\\\\\\/PBEiZOl0wUpHd4Bo5ntyTMODaVfR1XRsRfqlfqaLMOd6EOTUMxHEkerzJ5MuweU7u1CsMYWORb2luDeQw\\\\\\\/v8iz8Sln5xJntHtSy5KJ\\\\\\\/nHMGv45be33pQcJCbuuZdl4sxx4sGt4LuzbQeYdlxVrDTVfEcXwhWPrLxpIizVr+tp+NwRxiGT2OlDbg60k2ZyJwyx+Hnkdim5WrjQnKFb2nXdDbjaSIeBzILh8tOPqTijlRHAbXQZKE3BUBB4W6B8IkkwePPciqP8ge3WBihwH3Px02frEBbPyiXeQmtN5ik49MlsgfKVBpz8x1lfRD8uoQ\\\\\\\/FUD5Lpic1IoRo7eV5120S7LFrfFlkk2iIgyxvfYTBOoL9xKzVT\\\\\\\/LAtNlpa0VOmXMDhhiV\\\\\\\/IuBoBIngjn4XbNRCNeJexxD1BLnkSKd5nee3W\\\\\\\/gblXojfZ5Zt6XwIN2SMnW5V\\\\\\\/YLwKdbQVUeYSb3uO3fajV5C+YbGMqsUkUqlkHkbzbaDTXce71y14ncCau9V+oHkBAsqhNowZrl7Dj0R5Wh2kHMNRxaQZQTu2SLnWJevfHigdLDEA7WkCUMfd\\\\\\\/BfMLgLXcYpt4uieT\\\\\\\/xX4n7kDcQJ8\\\\\\\/LcwFL6PA61rTQgHv4XPmiyW9VLT6z1MqE+3+2gYp8x\\\\\\\/3RGUeFeGsS130oP1NJ3EZ5EGBUrgWZUeREcfRTHYJEtZEsrx3aQXDjp5kwlJQQL2kRLfEbJQr+2mcmiYuW1Dzs2cQqrkCoEmfOVy3dT2D6OfDKz4rp59IPSNNzp1hjUAelydbwu1oluORl\\\\\\\/98yMck2\\\\\\\/nJ3Ev+gWi1tOFmQMlAaV\\\\\\\/kU9ixk7xqGsUaS4sMzjzETgBnJVhh\\\\\\\/3204\\\\\\\/B\\\\\\\/rTU1bmLPYu4GkI25QKZZe3kBIlxHHgOb17GQZF+wVJMHZfC4BRrT3l0P4blGlarK2J3PDGfG5iqcLovnQ3ol7DRUec7Svmy+Y7dGzVNHs0mcW9pTR3n1PprxI+mY5uSxr\\\\\\\/OCqyB0vJ+G3vz07nVpFqUldkCNeD\\\\\\\/klY97VmVwR41Z8ipjieWmvfaIYi5VScNqxe+Bj3j2teINMGG3n\\\\\\\/XSMNmGM03ehOjseRIhhVZd++7nO8wKZu1S30+sZQ6\\\\\\\/+0NCKiwA7kADzTRfTETk3gCLtEB\\\\\\\/CWK7WhOn9jYM\\\\\\\/IqkF+CWS0VY9bulCpOX6YVD7+paRmuVA6\\\\\\\/HVXdrnij42qBa1hf4kdh4OLcl260zbBEJb+3\\\\\\\/pMJ7kkgP\\\\\\\/IgZAw2BTDPDhaTWTuDbvDVefpryy3XJAd3q3D6My5Zbdj29nGYBBDQRHz8zc4bL4Vt4ECJmMevNRO1Bdf2IahRcf1W3DG7R6oJwxegNwZgsuK9sJ2Bgi7iOFMkWSADxyPeNdM0TF3SBEbapO3OA2LB4Tufs5JPK4p110CTRF6x3YUu2o3juTjjtA6GZ8v1OEBI2v5uB97v2iLcTvHcNFZbiO20aDDeKjX2YgFJhLMrNl\\\\\\\/o2CtvUy7cTE3H1GCRHZe4Yx9sRKRCzAH+nc9t8V4KaNuKoAbSLOgDtpl0M5yHFqsOt\\\\\\\/1TySg320jpntHMpE+26ta0\\\\\\\/m5\\\\\\\/9Vbj23ZQXjcT1jVNgySIXcovavyFZJF7yc\\\\\\\/68FgDcoV6nyKeUu2mxysatsLOiUC3fOXx7fp1XujpngpFIEldrTr2SuWnagdVJWtoVcuL\\\\\\\/biW6KTyzQxY2IdwZeFeO5QUkAEipvDAM4J\\\\\\\/ggOOdDaFiL3SPk4Jn+FRT60\\\\\\\/Ed4RYeto3oZOZObe2Ga3cNCQLS4CMvrn3GJ6Vkr+4BGx873m2pxKYzGKlWFg+kJYL\\\\\\\/fcH5AvW4uXThEVsJCR3l8R0UOLWLe1cpnVAWVFLavHTmBTRdSvbbnipRiBi\\\\\\\/Vwa3azOtVcvsovwM74N00ySl8vOxjEA5EZWtz1UaakABxHIGmKwQc6pZyu\\\\\\\/rokPCURrsd3KeiW2jgbdsfPe4IRZSqYz1gByICnfblZI9RAq\\\\\\\/Yw19CaJuTvnGE88tw7ajQ1e1riCExIzGqFEDvSp21JRZKldv0GWpm5SI1GC3KT99MaSKSclokCaaezjLFgNpgfId3aBPf8fP53QXKlntg9hFArP9b+toco+K8tjK\\\\\\\/OXRAnngoMxvVqPGjAyfVaY+KSwm96OIzmKRZ7hcJVesmlmJsr9c+8xueuLKyGL1Uxy8k2VQUoXvoEVZ1C9WAB6mxRt3gqvksLJljjXYoQz5lat8wW6YhQUOAQxmbqKwtZCjTzrkjSxCuvcUD3ojWkdnX1d4jNfZyQprqqNqZeSOjjOP6rOzqH8iGGXR2K0G1dLG9yqZ1VEFG2RuRtjG8CZmHsZGfkOAUVN+FZdHNvWR1rMymW1hg+ucue4ZmWsT15BKGuSEkfRmlz9xcTVztSncbwiP0y9uvj6kJf6wvpX\\\\\\\/U0IjnSfoWFNtSfK4Kr+RmHO2C11ZvaHqtSuqnyk2zH+YkSAgapMZX3C\\\\\\\/9sKbWcjdXl5LnOAfWU\\\\\\\/LtGUb58pqyvrUZqnXgqNEJKXzuj2FScMxGPbUvdYbQ1ek6BuWDbtJ2wsPsW0XJEICi+sqnz8HH5s1XVpTae3xFs9bge3kW+qo99FUzVQzYzMY9ZM8TTKD1j3OAIIzSMawc9KcC7Sn4Ql+xcYCanUMm0X37LMJgUyyIo1Pim6fI3PZOVATvcEVvzgbL4UhCsLjpnTWrGzRCCNbJI\\\\\\\/AmZv44zDJ3dVxilh1Bfmkv9zrnxUBMW9Yaw1aG6huzmecCkU27Fl1WthDrk2amZkZRt5G5gEOHMPkv+LcGRdPdA1YN\\\\\\\/Ts\\\\\\\/3MX5ET+wI+dT30Rps7fzUgpekMpQ3S3O4JIuy9vWOneMSoxzP69h3hi4XrIzlb\\\\\\\/HivZDKGSgihU90S57SeZINLef0ptM80MQZ+ufXW8KQ+ByEU9Bn3TELorJhtkoCrTzwPIq9X1EUV5+O\\\\\\\/EAxTvRCRG3\\\\\\\/2vdXgql4WMEqTJGkx\\\\\\\/ow4z\\\\\\\/xYO0V2jjZiKLDDXHkxjT3MzAS1nWZQDKDG4x\\\\\\\/b8FGeYKi51omxdAxOZf8DWDcbYDw\\\\\\\/2F0VWJaIwbiftj7QRFKM6EGEujHVRaP13f4AD8rOZaGohjJSMfKonJNWVw2db6clT2dCbNuBdaTqfmihJRUIQRNqfkWJM1YLv1Q5De8aW1CXoKMIlPOZ\\\\\\\/9qHDgzjhg\\\\\\\/xbGht\\\\\\\/mnXdJUYMXxEVyyxZPIOAhJUdx7a\\\\\\\/5v+qiWllbcsU6u62OUs+ARfGCWBlymMtG4dTWfetCx2bnVbsok+eGu68jT3BdXOz5UInJl5wsJq2qpBgT+AVkbk0fDmM1VCRlEL8QvlFRtYzpfFuPHk+Kx3zU8pIIYzBIOmKGH7lrEQ1pmVp+iY0t9eBs8n\\\\\\\/Qv7JDvzaAL1xO14KBdBS3HxUhqzwMHHrg5+S72HNTurfKnkJZBBfTVlrRJzOfklYGJYEjeycjIJfkAOZ9TRlRFuIwJwFE6xFulxncGn6L\\\\\\\/v+hd7DdPDTijL9sonGNodJTe9n1bwGoBMXT2lj6fIlpbHl0YBTFFTSxddivT8WZBy9jfp\\\\\\\/3X7GcnUXTZ2TNAwMeP953wPBzfnQTwkfs38MfSAq2xcBgsOpNdpT7gQKPfKvSLgjnfLdYbmWbmR2Mzekwznm9LXrEz+tB4ugOzokXVAZQD8R2ywEpro8zpCTuirALwiYcAHVHQAxPCXzeJ83z+QY4POjUwweXb5U4A82XOA2nN1t7HBUnqhWlYmfdpw+OSXMOlevQOW63LaO2rvvSoKQLokn4IsCDjC3Qipoo3m1eqLMEjJcZq08XUtEjhgLEua2h9TXxdTikkq0t\\\\\\\/MZUu5tGv47pyYng0r3vEf6LthqOK57+la0GMBzxfQyg9\\\\\\\/r14nOA6tyrcr3Hsm355Z2uYqUInypRWfHrvNMCy9Xv+lLc3TabHOOPo2XNPjNXFw5WxF2NxhNcI7ewYM4otQaH1JUNBi9RTx66BSEj2vd0TQY0uPblyaYejxoKDjMYMsTJcCgcf4Ysbq1bGPXOt1\\\\\\\/0wbmTNPqWOX2ca0cAh8PAo5BuxKOel0Om\\\\\\\/1fRtdNa2iQeMaaO+F9xnDKbcV+0Lf4Qy0WNWkJac1Snoy5MgXcuv+EKb3XVjRmhnmq\\\\\\\/BM2NVQE9WaTYigFxFZLhxXXQGdC4zK2ZKUf4jV4MHXs3EMY0zzUPiADJggMKeDrZDnnfvfPgWAl+BJcmkco01474\\\\\\\/966zSlpBTsKnSCAeD1Ca7XEZQXjtYg491j8sSX3kTNqfO+\\\\\\\/OVgXfhv\\\\\\\/9gLLXjbWdwF\\\\\\\/+eLNKZUovXJb6bnomOh6jDqOo7xVcRc7lJd4ub97iqVq0L3kfbIDZfz2XEGkI7Kyuguta2d4uyAOgl+F6PvxIXSX+bQNPMoOWdhsWEXnTCgyrUn3Ju7h4e3kzMHIHlGdLG7Czb5T6G9uOz\\\\\\\/r1GTduaLhY2sfbUlEkGwgZwmwxCxsijEL5F1xs\\\\\\\/ckuYlW9DpkBItUjmRvX1A6TE\\\\\\\/iXBkUh3heWOUVSiK4Xgk1NHD7urZyNGS\\\\\\\/MbHNowDxLE3mRBrs9nw\\\\\\\/4YJVmJ4NRPXGNxMDpg\\\\\\\/QiUgn6SWi+e4zdQM+2+LommU4tnf0qMd2fkQP75QL7SB31qy3uXf4bvP8DntTENUffvzNVgsUvd\\\\\\\/kUmBwnJo9AoTtcaaTeGXunuiW7UyNEv0gtfl3zEDLm6gXaRBGimYeYRuzExQ2srvddVj6p50fKZR+RhYoKg+\\\\\\\/Qj50yJjzEJq3qH4DzmKou+da8mOsSxrG8YB9veXzNHVkAERZjTwFZ0DI+vHVe6Uc5pajyI5h37J6sf2RXnxpGQQWSmbTGOhqHmmFNCJs4xP1A10xAPTfjzM9PQfkAi7UjT\\\\\\\/F+iE9oVBBVtvL8VMubAPL+SuGHuDH6WsoaXAuuWGrl5MD2SKFvWHQ7Db9uNv3nFm9794b6kZc\\\\\\\/3Hn4\\\\\\\/fepvQ4gkyGCvtjOL2U0p4cIOd6rt5kezbWk\\\\\\\/cvJVNhql6UmKrfPedrGYikU0SHsi+SQgqGijOx38ot+g3ZL15JuLYnKgLYN0gdFe+gm7iWqKrZteR+GwDfmwyn1Gei0CmeAsA6\\\\\\\/Ac9\\\\\\\/KWtzTEcpPb7eN2aBk0jqtPDtOUzcH6tCCJS\\\\\\\/4ZXfrqvrcVGJyT0DKXOFVfSRPVw9BZ8\\\\\\\/fBiZeVuQyofq9AGr3aksZ9d6qbwPOOR7FKGuFiGYlAjbXnDA40gjA5fApOrmkOGE9Zqkfgv04rN1sngx7r98ijIPYJaQsT9tPLNuJKPUjpyw3vA1YDHUCqHgfGbtGc8SHxe6t9k1H+YaSZIDPCE00f\\\\\\\/u9Wpuqv\\\\\\\/mNNzCFBqrDK8Npr9IFs4IKfq353jJgk3s8zfCNPlKoVhGi7Q\\\\\\\/7\\\\\\\/8iFprq3hz3wWguLY03rOMabMfRdHgLaUNRFtJAllATXhIEtvbyQkaxrKXpJpelNARJQlbKXXdtbjv3JJnL0N9X1IZrCoAnJSVB6PxET9UOLRNYalbt9zMHDOHU2lqM\\\\\\\/NvtgJgTegqkQTrLVe9bPq9Vwxd3v9ab3hSfSYIjNQE0KCbtDTc04Lh39Tv\\\\\\\/AFHTRK7yhQ4ODFQmddcF0kDcgtAfAE3xGhTOPtYQCZGrEjT+of4RAdYKwrLKvLdKj3Eyn6HedSimclLA\\\\\\\/H1CPSKbrAkIpF\\\\\\\/pnQi2lDu+ozVD4rWV6qjLtpQu\\\\\\\/jaacpYnD+s\\\\\\\/KFrAH2wlRGaS5vT9BDXh6ZZdLRe5s6QXi3sl3VjfqJZsmOReDM5aPNSw+TIgrx8GhxgUfJfBLnQiGZXPHMopKkZEeB+s5Florh\\\\\\\/4wLVnliyoCRxjLe660dv8+l+gd8E2gF7cs6v6zbJfuPd+ikYjZLMab6Pp0PI4DjynlO1fP90E6cAmWNc3+aSVXB+thhwOvb6T9f6HkxpUc1c79zahiVTqsPxPGwaoOF9AL\\\\\\\/zotKUoSIYg2nwbdUV0pUr3FIWUuJNSEnaXwRc5jmG\\\\\\\/0tJc9gAyyACzT6NvSxsQ3MCZAC7eecstFCH2xekaZZCR6qBn27+E0drmymXlcqEdJTOWRBKoaUNSEfEgGeAOwiXnOS6NTzCVxK4qGxF82WgOZHFP4CwF27wy20SE9P7znRtcmD+VkBxLr8Gr4UVsLzV\\\\\\\/cG1weoFMjHGY7ir4A20bnjwwZN9nml\\\\\\\/8l3yVkhIOz79uvSB3dsdr3XQ3XDyHIvl5excLbwH00V3w6iRZk0wV2OvVbcFSZq+i+pcsxCwzU6D9JAsz9Gf4LuyRRXywDZHpm84DKlB\\\\\\\/aW67skLa3SFYIWocXU6YPH3Rj\\\\\\\/yg2x7FIs\\\\\\\/rJmfPfh59kwxeCDbq4by3INAJR13q1AqstlWrNTZ+BWJcxKo59vdvagf81Iy4Rl0dmX8olmiG63Yet3kxv4EqoFMG8kA9WUoQgGcf+dhHQXx3kNd5KKJeSOcdSDMlhdAyCugOymSMnpKRK7kQqZoKsZv8VA5alvPhMjQk3n34TdbkvKRRQP60hw+rsacrFCH1wEcmh7EzPw4Mv5MBAgYUwQrr3MH9IxWDDcWuACGSr1rRrz\\\\\\\/UhnljemQGiHXV71NjCIrwfb72mZpPAV9o\\\\\\\/ituiFl9ugg0fVDVrDsiGlIDThWzR\\\\\\\/7g+i9yrVTeHTVyr7mr31srqc5FyQhG5ChSwv7JJ7DHiuAF15PkovmLAgMnmU2B5eIvZPt1MUOoAl20kREHo8odYfnsSYiryuoaHIFOCcHDHHqi7VU5yO1s54VV8EOyQU6Lg1WokFZj0qDt0RNzDF2flVqW2gTGc8gGwyXe5XmYWHjEf5MKoFJgvviXgqwAz+E7Pfj1JKt5ZLIyRWWJb35Qm8AqErpTzgRGk0ODsbUvxf6GTKbWlXDK8TEzbLDnuvbzqbJny0OKmhl4gZXwFaJqTUR4S+sxyGUWzb0jC3s8n6j0ZdfH4JGe8cYVdtJHDCbzsmqH4mfIq5wKY8P33FevIBBtCZlHE4MraAXSTIekJ4xETvarF1N+oT6OXFYHroE9VQFqSEop0N5V06sLeyf410nSYb03Oup\\\\\\\/4srI3+o5nqjkIsst5PHVPWVLQWD7ZQiR8JLs7gbYsujpLWTUcpgRbOXnzK\\\\\\\/VPlUWu3T5USe9IBNq8QiG446i\\\\\\\/7u92SAVBxAFYRzz9txWaMY+B\\\\\\\/tTJ5EGNJ4qe1L8rj8Mggm9B6vCJ+jBdvnD2zOtmAn5armLlJTuLGC5ZxItU6MNjvy0NNqZXyeFW958wRfM\\\\\\\/lmdjmRBGmopOPWlj8hNvG3qDjtrUDG475oZktGG\\\\\\\/Jasy0zaI7hc40WHBo4LTcljfCPYG1cfUqjrug7k2mdvMeHuU264YoB9UFKVFmKPTuUPdVlHkyoDMq6RhcZJ8SBWsrLpSckhTmQDt+JDoSPMcWJcOVPBfnyIUl3hjlSFRpTK3yvKPNZw9WIY8Yp0WPBQ0ZiPEVJKcOtt9O9W6Iz58WRKRt8zyDMher5XzVCoe9IInMkwa+CFi0dtTbj8Y5Oi9sXo56yEtqe26aJkS6dHXNkNYah3mL7bQmKAXmLbLuNO+4bVk64DFI4qdZst\\\\\\\/i4PKKcGSwWFO3QS2Av9jRoM768NevuZDoPGtRqh7qEtCJlJf8n\\\\\\\/R1sI+UpAcnwgvsLJGSWllCTd\\\\\\\/uupiQh7EzDsk1pG3QLPbSU8IEZABmDgbozhiDiU6K3i5SLLyjUZqoRp2\\\\\\\/yrhyKtRPNsvW5yzhKjE7nGVFTFMCdDmwGK4yLRHESTGX8Z\\\\\\\/mWjhD+iPhKMedxD99uh9txo0c4Uo9qjUiIXsJWD15K7v+TC\\\\\\\/7UIDRhVLQa2Locwx1L4v3k0ehlDvNRxisMWJEnCXZxXUm5uXOL3YyQ5b4GVSNzPxe+u41wnNk54ake1skVISTxcIY9fzt8ChiCxzeAYHsxkajn9JLix7uE0bTIho9a\\\\\\\/Vc7QjJDB+AMiiCVcUGgmBWNWeFCWqaEiFSNNbrlIg56YHhgGCnTHilQz\\\\\\\/OcJa685WAeVshexTGnLLSqExvVrS+dPmwskyJ4xUOF0Z0n2VEC0DsNapPg31wJqm4WdapOQe9aK\\\\\\\/W6w\\\\\\\/6mhP17j5GJurus+1ohtqmGnpz1PPSvFd5rTbkCAeQ\\\\\\\/GJ519L6OH5UO84vCZi996hcLq6cf0h1ZoMuTLtV\\\\\\\/fiFNeETbGqdwPWEnoqZE6daI\\\\\\\/sIiZpCZIt8ZZJPEW7v4JRWScd\\\\\\\/9bR5x1oN6y8aE4iQ3aA2rhIsKWHu56Z560OR3ajzHc8KFTgSWssyfKLjo0Yd1WaeDaWPm9NYzB\\\\\\\/icO9OcWujcGau0y3cGXHrO+o85t5zEigbf+wpoS4vUwVLSH8NaFlL3w5DvppMuim+lqeDMDdGmZFhabpwPQ1e4NMFF4wdoxPWIsfgWL90iJIAf5IBtXdWk1TWdkiO6QlxkVDOt4S0x3pV+KlvufUHmTYOHODhyOCc61JtYfQTC\\\\\\\/865ZbcXLm120v\\\\\\\/vP38Nqrd6CdgZw+UzTju5SW4iqA\\\\\\\/lfwjhG3NVXlEaSB1lJDaCLD2mhCBZo6VNjSmaEz1kk1bobuOstT+piNGu0sVze0zepNqd8Gh0+uAvHw9YUhqJDWNDcDAEmudUKmPqBYqEWsg2RNzrxdkxvG+fy66uP6k37u\\\\\\\/ycrUo3DWvpY1z16yEANbaCBpNXtdntXV61vJ9K+Z5RsO8F9EhallVnzxOXhB5KvtJhue5ZlC1O4Z4zU3zlvLdxWfUUJ+Jid7YfyPNN9IAIEMt4dshYglpjbb4q9QuevdoHABH6iZ7FsD2wzpoqz6Ay1qchuxpMmK76w1oNWVKM7DQka1YWt6DgJ8hij05oTCyU7dHUtieXqXnc6E5S736muS+yDGM7i3OjZ20BHSXOJdI\\\\\\\/TaSj5QbG49eZJp4FdKFBe\\\\\\\/2K1owMoXKACp1yVaQLa1mkmYYSCbwNJMty3RnnO6y2BglBo8jwEMJx9tG6b+geK7DCrQLpsbrPYX9Tsf3eoQMH2uFzmpI\\\\\\\/2PH7t9ennl32fk\\\\\\\/OD4gj4swSgDRMYDq4U2i0fyXp4Exdppc0q8UfNsLk89TndPUX06dajERoz8OI2s+EBdnbV3MZFw7dUVq+Fr3B+dtGWcK5PCdShRgEAAm9gea5qVc3WDCGxsEeg+GwTmFzEHvjOQbgH1KoOgd6+qeAGtYagsCIiP+KqObcreR+CxN8yt6kr+6f36uVVPfol3GpLQL2Sgj8ts8HDw11emmR1HcwU4kqWZ2cxB5WuWBdk30wJpyjqu6saY1ij6cQeRIEvBo\\\\\\\/3MQyiBdlUH9g4H5CJ5l+mdj9q6b0wx1D5O7AjCwIp7eNe5BqETXjKlrvicP+nVF+YL7tJXo8z4tNzVWmI7L29k\\\\\\\/AX97xI3sx7PTTQOR1Owa16k64D3KsRxNhBilYJtkwRpbYOAPIumD62GIRKFMsgC0wHKlDGNMGrzd0LXWD3Yx9RrtrLmODqNGhaa\\\\\\\/7Nx1m9VtCdiGXWKigNL\\\\\\\/ZigeDBcWg6aIqlD54u7m8CQZumA7ce4k4sQvPdkVbZTkLZq4C1asvUDoVQb9DxtXAF4Y8LpKKWvMELbvF1fdte5ybGYWbGmF+1YbRCdXrMKJFI06vffx1zU8oqneu27qa7izChqDvbbt+pn4GJGqvXiqzHQQMpoHwn3asSWEI\\\\\\\/\\\\\\\/DQQAqqByeQT9O5AbWRye4U8+1FbCYPULFFalfgaYvecPejNw0Y7U8W+3KefBrmcd8Gbh3RIXlPkWoGRynkG3KsXWJ6DYHk0CwVib\\\\\\\/4lxDETw8h1jD+rXLLDngND4VZ3+0oIhhbJyhCTcDNZaiejxDrluKAHe\\\\\\\/nPANDQsnKXUrrtLXDdowVOzu02G3mgVGZJNYr3BNk7V5eMhr2rql0khv\\\\\\\/YaUkib5EUbGv2\\\\\\\/dw\\\\\\\/PDJdzHAvx3+YfnYFVxynRaDUfPdlShXLr6+GYKMVlSZX+XSbkdPNr90PjC1Tg9ROPBdXtKRlxHgR9TF2zm+16MAoG8alKs+lnSefhIYVebeQ\\\\\\\/KXyoNfTBq3ViGLTi\\\\\\\/HOQ8wB6Yherx7IwA22Wl091C1nnZ063aYsx+ZQrGTf7q\\\\\\\/Y62Rs2LRRJzs3KwEKIBP568yNjLRSalEvt2ijodGcpfHzp8yXoLeqrYb47QoWJ3mCQsV7B1tskBVY3oDG3hryI\\\\\\\/kVfU+TSSJngNbN0G5AAKjLpocaacUOiqhD97ju5OM6I\\\\\\\/udCktDSjz9CAEms7PcVB\\\\\\\/jiirtN3E5hG8A6oRdo8fk7GXuE70YKeM7JdiNnLUcZAVls9Hyn3NC+VdVMq3vBxDICsS12xOrijMdL8faQyvylE62Gsvmg3zxJkvihijBJKNw+mHsJoEnqBJ7pvwP1xy40G8FSymgZjOb79FipZlbDbYUfgMnFDnE+wTPMbVYjs7NCQKI55iOoD1nk49h5i+Yxpd1km2y78W+URpve50+aBOsmPne\\\\\\\/SxpOA396OOkB1hjn2rs5RIRk3ni0qBv0bh+n7fNyuIuTpKjQArBVe8Ovp87WBxsk4tDPZcTqs+Gw0cGoqQ2rHbAjqfFn2fRV\\\\\\\/cceqhQ2sYUdAVepU6i\\\\\\\/HmK589CJZn9HMNL0EIJFpBpNZ5ra3omIVfeJfNGe+ZUv5Az049RgfxtP9qBNNgL0H507MafHlrn012K2i\\\\\\\/ulyHRCoaxq+BEHTFUzFIVTyZULKOlA1oQZpTCkMWySI2EH9X8WJUnI+aRy4Q24A3oB1Omk1NKFUxXs4d10BAb\\\\\\\/Hbn7b1K4la154W0RtcLpbnoI+R2yWee\\\\\\\/OQVNkOUqXAJDNu\\\\\\\/MBFQ4F0P63uzH8K4oW9CghwBzjbkXTx9L3KoJiC3pIFWKEt\\\\\\\/WI1zg2\\\\\\\/onzpBqG2q9o4doxuL6gS\\\\\\\/9nQcsVk5f6y4RPD77P2zt8HeasNicOXOeVHNj1QNewgSrzf8YCuHJ2Y9HXb639MfmR7oNdTnmM8QrWqkfEjUJ11aNZMjebdA0+fcdohbz4OAZKHRN40vIvyaAfjC9zpzzpcSZ\\\\\\\/gzA0fhtA6XLnPz\\\\\\\/CLITTRaJudmufUdLD4nGFXbAERAp75d5GBEZf+UEwJr82ZVxexphbMPxZ06Q\\\\\\\/onpyAWRjGAQ8xKuZL8oEkXVY81ojoymmEoOxGWHjl3T+tyhuKYLNU0vR6ES9xO4WG2cztZsaEqpSWtrCsNhRxuknpF9CsvuTM7CwvGa\\\\\\\/P78as7CUFtTNvTPfXRswOl1zWeql2mmHUnxpHTWz1U6uLT5KKyQF\\\\\\\/ITLeOJ0CyLBDJGZd0F8WSFcgNE6CM9BhdUVldfVuFry7vBeKndMd1wbfxQNai1l+I4iWkusMIIYfD0mM8K91mHQt2EimB9t4KhoZcLqD6iyoIfqFQfQQy7tcC+pWTuLShiF3FKTXB5cwtdrww1+NvwFBaHeB+V6zSuOlKlDg76Yt4fwKS9vSc6ppFBNtozMCNYEkZ3wX9sTEU\\\\\\\/WfnaXlgslr0ljQauRntLhQjDmp9VhLWHVeWyanvd19Z15yhYHHbZHfX1Nu68rnLMYTbrQnOkddZb3rcralNLykUhEtohSV16LMx3Q8Z6G1YxaikaDNldJLGZk17DgtVF+kePZ6txg+z9ixRQrfDYwbC1gqhNoU9C6UEa0EzputMvcC8YjhB+KhL5cALXn4uJ+dzrgBfWEcT2wDBQ+oemvMq4xj30S3I3zrZI7sRo72oriY5iDeBSvntqay4MzJiYNl3FZ8aG1Hnc5CsGCLm6JT1iKxQDCZCOFjH9PpNCvl0tkROqNZzUyLuBkpEaTX\\\\\\\/W37bL6moGMzFLhZheCJoUPmeUwKrDbTC5re3nD2mITSWdL8CTYmYBDLSszmv0WeI7WKOK\\\\\\\/gD841Y0ofEiYAHQOYqQLHdZxJq4VqE3mYs0VzPKrl0oWdi2vQH9SU3CIrmsrsVnalUsC3W9sO6tyQEzGc4PVt+o6EuOhgMIrM5YbNytJ4unZwxTcJhbDrWQUmQTKt4384Zc7Pb65QLCjpk7\\\\\\\/qihlnJG5XBIGxBIKOrmM2wSoY1jAlj6ZRqRoTkcjfq5Rn5RepB4e1v59ErElst1MiQ3ccGkLmQUgvJKXd5d7yNzr15Rem7sMxqvy+fh2BlKmg6l6x7BzI971\\\\\\\/1W2pZtQFdd6JhlIXddA4qRZpKRNlx6KOi33IBgr\\\\\\\/DqjcWFh66avSbxLqfHGnwff6rXbC\\\\\\\/nEH3X1AeT7BcIvEdHyRxl\\\\\\\/9EXWPbsXVa370Bn0iV6D7JcMDuScthnUcBKfPD+b5P4VbDGxOWXCQI3TvMFfFrJEQZZf+0d9sCGBT181mLgPdNrnUOD2uixQwnRrg7qtAzlqVgxNqoBtCZ86FRG8HoBL\\\\\\\/JdeZzhVMrZImjg5dCPGpScbFP4YBx66WDH8pfg4NrIRohGgcEfoW5uMBIIOrv6bwMdHsxDLLOhtDOM04xB8bBkE5WVDSKy\\\\\\\/f8vbvAuopthVZVUxwGnXiCHBEW2axgL8p1hYJ8LbEX0KZtQjPB06hd+wl76ZoIXXs3L2OhRQnTnNdpebluaV5wiS\\\\\\\/pp7rDQ47iNRQFu7TPI+ahzow7nXjpq0DmbTPneBcgynbLyomC8EfEJnZeLM6gIo34\\\\\\\/k2IaVbvOsMi0QTQ0+1dn5sIICa2FJ324bM5J6ROJ9DcC28TrUebXh8bCBwbA7Dl\\\\\\\/IfHqM3NDNrbqMhXocm\\\\\\\/Gju7pO3m9t0MFXOvWf\\\\\\\/85NWcW\\\\\\\/0C4m5t7XIG8JWMHQBuU49u54y5mjY7+IaDLYvd6norCwEA1BhWYCQCGJ3gkqCrIsIN++vrnrOnjJ2xcldrXwgJp8xhHkyW\\\\\\\/pVI3+7KEozcKgTpSyu4Ck\\\\\\\/aWDOMCUdvF0cgSH0EvV2Yf6cQ6vWztZNs7Z94DYqTHDJNdF7yVPPQIPlV6B1Q9jCaVS4KYr\\\\\\\/OgPd8ESSxdktIQkc83FgZ+xehmVPevNciPYzkyVVOXNgjzPj0ol5fpN4DpTXW2AKehkTrnxNpZKfgARI0cPd6aVSd2kF\\\\\\\/5wuKUvTutWNuExQXedoJXv4k5mKaBt73uG3jO\\\\\\\/KUEPy\\\u0022,\\\u0022iv\\\u0022:\\\u0022lfVOWad4\\\\\\\/W5rMIyPIOn3Ug==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222a341235e1d897d1ee3d1a07ef31a258eedee1cdf07080b986e73a8d2d25cc38\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nPncoSYneACeKIZUtU2uvjAeV8FcZBh3mTXtz6d8f9d+dWDsWeaeQYV1qI4U9Dqg0zfh6w6y9YTpA1ajQM94WRgk\\\\\\\/mNTo+3c1i6bYyoEcoI3iGeUhJO7Qd+UlQFOHqle6gpnEADXR02u668E2o6R5buD9GgfiY2UFnpBX49bdrA6lyI6lEdODE5vgagFvulD4wKe68o0jVtW1ICOPUfRbnncp3p6xlSE9B49ZEFA2NQU+mvhnVXpfFUQsBKyCtWZ8LyvoP80nOxoAogqwZZLPwiT6zq6Pt4Pz8oQtvIAU0tOiYh6rqL3rKENn6Rvbw9Vh6E2lbojpYuebQ3YuA+9p8bSOKdLofneqoUyRkYH1FPDt41NZQ9qNQ7pf\\\\\\\/\\\\\\\/ZEtw\\\\\\\/Edajv2AEQHpXzd9o51Ik1ggrSKfXN5H1m2AFRk\\\\\\\/ZMLQrLslRt5bR6XYvxji9cmzEzKReLfyX5LFd5cgYgKFZD3jhnHWooaG4W\\\\\\\/qxCYNJXg7sdd7pdRiAmnTDuV\\\\\\\/T6UbshPvOIhuokMZ9W4WlaPHdL6ahWhl810nNeFr6t7nxA4Ra6Kw8HPXzqVluIpGAOkci\\\\\\\/qJ2dys5Y+MEzhZKB7OfN1QMA4J5BB+2XN3RArts\\\\\\\/EnNwm9CfmhYSk6N5hcCUDkKU9LZjDNVdNNlk5JMHxBfe24A81O06Fjn0Hn1+9AmtK1a2XaXk1RydKqyF4AGJuvEJA1uclSANR1Y+yHKiCzZRe7CjkFIvm94tQFyiAmLqkM81MnTV4ki\\\\\\\/D86fZ3ac1vqg22ZbTVdUiLC+AiJSq9nePnS0C2CkRpecN7r3O5\\\\\\\/FuG9s+x4F0YFRTN5IBskhoE6C08iUppp3\\\\\\\/QLbHWNKwz5I8G3zgICXJAr+oYre86yoHl+825DyRPs25Wmv7O6k\\\\\\\/1Kj9z0KLGCYCs2+s2lxmhafobIJUnu4z9IXqSQH26pPFkfLD1eP0sPrlejHY5ofUsUKH4\\\\\\\/VriL6CJhw1p05oMyHnlyN5G6ZpJ3pWOnXq9nl8sk\\\\\\\/MnFsuVrc565f+g0VZ6Qjpbel07fyiqV2vPNXtCQsTldxTiGh32Ec2f\\\\\\\/7muib5tLReCWqnUPe10y\\\\\\\/KiZ4Y3IY2E02fbtRlZivG5b7sCmrrAK6ycYjCaBKFq4j+ChWehx06bQR24WGGsAxEmakpHQYvEvDx3boeLIMHat\\\\\\\/6WCxYZn93pNEV3MB1CNLhSzB4oB\\\\\\\/D0KF2qwg1CZpsZGrk9TBKAdi3zrldEYwGwsv3PJDAB5AspVXb6ByGY17GSkQj92veXrSGU5115oQRWdyo1UXSETPtEvbg4J3IZ6mWQLKJB9V7Gpe4alxrrr4IYzLYxZzn5y9nZOipGVhvBp7mgA2EhRWRSDrCywJ4No6pBM9vwwJrRZJS4Q6E5uv8hMorceZe1ng5adJEnPexCB+4EVglYFRiLT\\\\\\\/SZHY83GZgVdh8acjQtzh7NI0iZKAGv2AbugHr5kp6n0OFYNCSg7DfYnVP+VdVBGUjtjDYr8d+IAm4V\\\\\\\/AGkpKXXbjTB5WtT92MfNbteYAGOIbbVlSrkiMr0c0\\\\\\\/NJWoUbmK685RxK85PDbeptFaZdFTvG72EwVbxmMK3IB1b6Fv8OJ\\\\\\\/uPaOr7vii7HDuDiQeR4BVRBYFWebQ28T7MPlpCSBJ5s10Imc7pJbNJ9TBMcqHNg1LbM9TMlNFylBwPsJtEmGK5psgBX4bwEV4Y2SeVakVQ3V7UtpRd+qTF+X0l4xiXpeTO8d6N+7OZEM5Mdiluc50UNkO5+512Fpg8Ti7vnVDz9Ya04QMcdC0haXgGFNfJqYogVtFTs6PKyQKPVXOq+SZ9dGzLeIesf+Cfsft3DUPxnWeIgD8ZulmeloST9gvF2xFdaZ481J3FpyuAlxD06fG8uUGQ\\\\\\\/DvyntAsL9qo8YIAgmhUWCLkdb\\\\\\\/l+PT58qbwEHBv8eMNpP6scC9Jr6yglXrM3hWHq7T1d7k2xcQYPJLtO4LRd3kdchng9xNAs5MWazGsg2M\\\\\\\/XQWJSOoe+PrdSagEFGSHScQqgKBpJmvlco0zhUIjSKoTvPI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022QOT+1JUqhvteGiT4c1nQuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bb00f9164e0730c93f3b69a08287428a5ac21056f32ecea38811c366d3d985ff\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459634 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/341" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FBmIB9tASfvbxzw2TCewsskiBt1mLqwm7WjFtMQnS2Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/341" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZAqQb2dC9UCoUCcP_1NW_f65zmQLl_pdHmnhERzFqog" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/2/325" title="Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/2/349" title="Global Analysis of Serine/Threonine and Tyrosine Protein Phosphatase Catalytic Subunit Genes in Neurospora crassa Reveals Interplay Between Phosphatases and the p38 Mitogen-Activated Protein Kinase" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459634" id="node459634" data-pisa="ggg;4/2/341" data-pisa-master="ggg;g3.113.008706" data-apath="/ggg/4/2/341.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chinyere C. Ekine</span>, <span class="highwire-citation-author" data-delta="1">Suzanne J. Rowe</span>, <span class="highwire-citation-author" data-delta="2">Stephen C. Bishop</span> and <span class="highwire-citation-author" data-delta="3">Dirk-Jan de Koning</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">341-347;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008706 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459634" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chinyere C. Ekine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chinyere%2BC.%2BEkine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ekine%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChinyere%2BC.%2BEkine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Suzanne J. Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BJ.%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuzanne%2BJ.%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen C. Bishop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BC.%2BBishop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bishop%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BC.%2BBishop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dirk-Jan de Koning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk-Jan%2Bde%2BKoning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Koning%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADirk-Jan%2Bde%2BKoning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dj.de-koning@slu.se" class="" data-icon-position="" data-hide-link-title="0">dj.de-koning@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/2/341" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459634" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459634/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/341.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459634" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459634/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/341.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459634" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459634/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/2/341.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459634" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459634/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/2/341.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459634" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459634/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">In animal breeding, the genetic potential of an animal is summarized as its estimated breeding value, which is derived from its own performance as well as the performance of related individuals. Here, we illustrate why estimated breeding values are not suitable as a phenotype for genome-wide association studies. We simulated human-type and pig-type pedigrees with a range of quantitative trait loci (QTL) effects (0.5–3% of phenotypic variance) and heritabilities (0.3−0.8). We analyzed 1000 replicates of each scenario with four models: (a) a full mixed model including a polygenic effect, (b) a regression analysis using the residual of a mixed model as a trait score (so called GRAMMAR approach), (c) a regression analysis using the estimated breeding value as a trait score, and (d) a regression analysis that uses the raw phenotype as a trait score. We show that using breeding values as a trait score gives very high false-positive rates (up 14% in human pedigrees and &gt;60% in pig pedigrees). Simulations based on a real pedigree show that additional generations of pedigree increase the type I error. Including the family relationship as a random effect provides the greatest power to detect QTL while controlling for type I error at the desired level and providing the most accurate estimates of the QTL effect. Both the use of residuals and the use of breeding values result in deflated estimates of the QTL effect. We derive the contributions of QTL effects to the breeding value and residual and show how this affects the estimates.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-association" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association" rel="nofollow">genome-wide association</a></li><li class="kwd"><a href="/keyword/family-structure" class="hw-term hw-article-keyword hw-article-keyword-family-structure" rel="nofollow">family structure</a></li><li class="kwd"><a href="/keyword/type-i-error" class="hw-term hw-article-keyword hw-article-keyword-type-i-error" rel="nofollow">type I error</a></li><li class="kwd"><a href="/keyword/statistical-power" class="hw-term hw-article-keyword hw-article-keyword-statistical-power" rel="nofollow">statistical power</a></li></ul><p id="p-6">Genome-wide association studies (GWAS) are now commonplace in humans, livestock, plants, and model organisms. A commonality among these studies is that genetic links exist between genotyped subjects and these must be accounted for in statistical analyses. Several approaches have been proposed to take account of these genetic structures in GWAS (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aulchenko <em>et al.</em> 2007a</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kang <em>et al.</em> 2008</a>), resulting in a range of statistical tools such as TASSEL (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Zhang <em>et al.</em> 2010</a>), EMMA(X) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kang <em>et al.</em> 2010</a>) and GenABEL (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007b</a>). Somewhat less attention has been paid to the definition of the trait value that is used for the GWAS.</p><p id="p-7">In many situations, a phenotypic trait may be decomposed into an estimated breeding value (EBV) and a residual. The EBV is an estimated measure of the additive genetic merit of an individual (<em>e.g.</em>, animal, plant, tree) for the given trait based on its own performance and/or that of genetically related individuals. In the genome-wide rapid association using mixed model and regression (GRAMMAR) approach, the observed phenotype is analyzed under a mixed model resulting in an EBV and a residual, with the latter being used as the trait value for GWAS (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aulchenko <em>et al.</em> 2007a</a>). Other investigators, however, have used the EBV of the individual as the trait score for GWAS, assuming that it encompasses the best estimate of the genetic merit of an individual (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Johnston <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Becker <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Čepica <em>et al.</em> 2013</a>). As a citation from <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Becker <em>et al.</em> (2013)</a> shows, the EBV is sometimes considered the best estimate of the genetic merit of an individual:</p><div class="section" id="sec-1"><blockquote id="disp-quote-1" class="disp-quote"><p id="p-8">Breeding values have the advantage that they are free of systematic environmental effects on measured phenotypes, as these effects are considered in the statistical model used for the estimation of EBVs. Additionally, they reflect the genetic makeup more accurately because they do not solely rely on own records but include information from all measured relatives.</p></blockquote><p id="p-9">We will show here with a straightforward simulation study that the “information from all measured relatives” is a prime source of false-positive results in GWAS. We note that this insight is neither profound nor novel, but our aim is to provide a clear and concise illustration that using EBV comprising familial information (<em>e.g.</em>, parents, sibs, etc.) can give much greater false-positive rates than ignoring family relationships altogether.</p></div><div class="section materials-methods" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Simulations</h3><p id="p-10">The simulation scheme follows that of (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em>, 2007a</a>) with two simulated family structures (human and pig) and one complex real pedigree (pig in this example). For the human pedigree, we simulated 337 nuclear families of three full-sibs with parents that are not related to each other or any of the other parents. For the pig pedigree, we simulated 10 sires, each mated to 10 dams that had 10 or 11 offspring, resulting in 1010 measured individuals for analysis. For the real pig pedigree, we randomly sampled 1010 last-generation offspring from a total pedigree of 5390 commercial pigs and included either two or five generations of pedigree information. The latter was to test the impact of the depth of pedigree information on performance of the EBV approach. The pedigrees that were the basis for the simulations are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/008706SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/FileS1.xlsx">File S1</a>. Each of the 46 scenarios was simulated in 1000 replicates using the MORGAN genedrop program (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">George <em>et al.</em> 2005</a>). MORGAN genedrop simulates genotypes at marker loci, trait genotypes, and polygenic values contributing to the quantitative traits. Quantitative traits were defined as the sum of the single-nucleotide polymorphism (SNP) effect, the polygenic effects, and a random environmental error. Two SNP genotypes were simulated and analyzed for association: one SNP was not linked with the trait of interest, or any other marker, and used for studying the type I error rate. For studying power, a causal SNP with an additive effect of 4.0 and a minor allele frequency of 0.3 was simulated explaining 0.5, 1, 2, or 3% of the total variation in the trait. The simulated traits had a total heritability of 0.30, 0.40, 0.50, 0.60, and 0.80. The QTL effect and variance due to the QTL were constant throughout the simulations whereas the polygenic variance and the residual variance were scaled to achieve the different QTL contributions and overall heritabilities. An example of the MORGAN genedrop script files that were used for simulation is given as File S2.</p></div><div id="sec-4" class="subsection"><h3>Statistical analyses</h3><p>The simulated data were analyzed using four different approaches:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-12">Measured genotype: The SNP to be tested for association was fitted as a covariate in a polygenic mixed model (1), which accounted for familial relatedness of individuals in the pedigree using the additive genetic relationships among individuals. The SNP effect and polygenic effect were estimated together using this model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span></p><p id="p-13">where <strong>y</strong> is the vector of trait values, μ is the overall mean, a is the additive QTL effect, <strong>u</strong> and <strong>e</strong> are vectors of additive polygenic effects (random), and random residuals, respectively; <strong>u</strong> ~ N(0, <em><strong>A</strong></em>σ<sup>2</sup><sub>a</sub>), where <em><strong>A</strong></em> is the additive genetic relationship matrix based on pedigree information and <strong>e</strong> ~ N(0, <strong>I</strong>σ<sup>2</sup><sub>e</sub>), where <strong>I</strong> is an identity matrix; σ<sup>2</sup><sub>a</sub> and σ<sup>2</sup><sub>e</sub> are the additive genetic and residual error variance, respectively. <strong>w</strong> is a vector of marker genotypes (codes as 0, 1, 2) and <em><strong>Z</strong></em> is an incidence matrix related to polygenic effects.</p></li><li id="list-item-2"><p>GRAMMAR: The GRAMMAR approach consists of two steps (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007a</a>): the first step accounts for the familial dependence among family members and the second step tests the single SNP effect on the remaining variation by analysis of variance.</p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-15">Step 1: For the simulated trait score, we fitted the following mixed model [with the same variable definitions as (1)] without the marker effect:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span></p></li><li id="list-item-4"><p id="p-16">Step 2: Using the estimated residuals from Step 1 as the new quantitative trait (<strong>y</strong>*), the marker genotype effect of each SNP on the trait was tested by linear regression:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span></p></li></ul></li><li id="list-item-5"><p id="p-17">Ignoring family structure (IF): The IF analysis is comparable with the second step of the GRAMMAR analysis. It uses a direct regression of the phenotypic observation (<strong>y</strong>) on the SNP data and does not take account of family relationships.</p></li><li id="list-item-6"><p id="p-18">EBV: Similar to GRAMMAR but in this analysis the EBV from the polygenic model [û, from model (2)] is used as a trait score (<strong>y</strong>*) for the association study (3).</p></li></ol><p id="p-19">All analyses were performed in ASReml (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gilmour <em>et al.</em> 2009</a>). The type I error of each scenario was estimated using the unlinked SNP and a tabulated threshold of <em>F</em> &gt; 3.85 (<em>P</em> &lt; 0.05). The statistical power to detect the causal SNP was estimated using either the tabulated <em>F</em>-threshold of 3.84 or an empirical threshold based on a 5% error rate for the unlinked SNP. In order to facilitate the computational load of 1000 replicates of 46 scenarios, the simulations and analyses were run on the Edinburgh Compute and Data Facility.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Type I error</h3><p id="p-20">The false-positive rates (FPRs) for the four methods are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, averaged across QTL effect size. Because they were estimated on unlinked QTL they were, as expected, observed to be independent of QTL size. The GRAMMAR approach was conservative, whereas the measured genotype approach performed very close to the tabulated threshold. Use of either the EBV or ignoring family relationships resulted in much greater levels of false-positive results. However, the FPR depended on the family structure in the data, with a greater degree of relatedness for the pig scenarios than the human scenarios. Particularly for the pig data, FPR for IF increased with increasing heritability whereas it decreased for EBV. Conversely, GRAMMAR was slightly less conservative for the human data than for the pig data.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463971/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463971/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Type 1 error rate for MG, GRAMMAR, EBV, and IF analysis for the simulated human and pig population structures, averaged across QTL effects for each heritability (h<sup>2</sup>) class</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">The simulations based on the commercial pig pedigree also showed a large type I error when using EBV (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a>), albeit slightly lower than for the simulated pedigree. This was due to the smaller family sizes in the sample from the real pedigree compared to the simulated pedigree. The use of five generations of pedigree in the mixed model (2) gave a greater FPR when we used EBV as the trait score compared to using only two generations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a>). The use of GRAMMAR was more conservative when applying five generations of pedigree compared with two. There was no clear trend in type I error when using the measured genotype approach (1) or ignoring family structure: in some scenarios using five generation of pedigree gave a more conservative type I error, whereas in other scenarios it was more liberal than using two generations of pedigree information (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a>).</p></div><div id="sec-7" class="subsection"><h3>Power</h3><p id="p-23">The full comparisons of statistical power to detect the QTL across all scenarios are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS2.pdf">Table S2</a> for the simulated human and pig pedigrees, and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a> for the real pedigree simulations. Measured genotype analyses (1) usually, but not always, had the greatest power, irrespective of whether empirical or tabulated thresholds were used (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS2.pdf">Table S2</a>). Using EBV or ignoring family relationships had greater power when using tabulated (rather than empirical) thresholds (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS2.pdf">Table S2</a>), but at the cost of high FPRs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). GRAMMAR was conservative when using tabulated thresholds but comparable in power with using measured genotype when applying empirical thresholds (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS2.pdf">Table S2</a>), in agreement with (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007b</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/2/341/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical and tabulated power of detecting a QTL that explains 1% of phenotypic variance in a trait with 40% heritability. MG: measured genotype; tab: tabulated power, emp: empirical power." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1138646895" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Empirical and tabulated power of detecting a QTL that explains 1% of phenotypic variance in a trait with 40% heritability. MG: measured genotype; tab: tabulated power, emp: empirical power.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/2/341/F1.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/2/341/F1.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/2/341/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/2/341/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463987" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Empirical and tabulated power of detecting a QTL that explains 1% of phenotypic variance in a trait with 40% heritability. MG: measured genotype; tab: tabulated power, emp: empirical power.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">For scenarios in which the heritability and QTL effect sizes were low, <em>e.g.</em>, a heritability of 30% and a QTL explaining 1% of phenotypic variance as shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, the human pedigree had greater power than the pig pedigree. However, at greater heritabilities, these differences diminished and for some scenarios the pig pedigree had the higher power (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS2.pdf">Table S2</a>). For the simulations based on the commercial pig pedigree, the measured genotype approach had the highest power regardless of whether the thresholds were tabulated or based on empirical results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a>). The two-generation pedigree generally gave slightly greater power to detect QTL than the five-generation pedigree.</p></div><div id="sec-8" class="subsection"><h3>Estimated effects</h3><p id="p-26">The estimates of the QTL effect and the empirical standard deviation more than 1000 replicates are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Noting that the expected effect size was 4.0, using measured genotype gave the most accurate, and apparently unbiased, estimates of the QTL effect regardless of variance explained by the QTL or overall heritability (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Ignoring family information also gave accurate estimates of the QTL effect, apart from when the proportion of variance explained by the QTL was small in which case, the estimates were slightly inflated (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). When using GRAMMAR or EBV the QTL effects were underestimated dramatically. This confirms earlier observations that GRAMMAR underestimates the QTL effects (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007a</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crooks <em>et al.</em> 2009</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463896/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463896/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean estimates (mean) and empirical standard deviations (SD) of QTL effect for different association analyses across a range of relative QTL effects and heritabilities (h<sup>2</sup>) in simulated human and pig pedigrees</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">A clear trend was apparent for the effect of the heritability on the estimates when using GRAMMAR or EBVs. With increasing heritability, the GRAMMAR estimates were increasingly biased downward while those from the EBV approach became less biased (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Furthermore, in the pig simulations the estimates of using EBV were more severely underestimated with increasing variance explained by the QTL. On the other hand, the estimates in which GRAMMAR was used appeared to be unaffected by the proportion of variance explained by the QTL in both the pig and the human simulations (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). When we looked at individual replicates, it was apparent that the sum of the GRAMMAR and EBV estimates provided an unbiased estimate of the SNP effect. This could provide a quick estimate of the true effect of significant SNPs after GRAMMAR analyses, rather than re-estimating the effect in a full mixed model as suggested previously (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007a</a>). Across all scenarios and analyses, the precision of the estimates increased as the proportion of variance explained by the QTL increased, as shown by the empirical standard errors (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>).</p></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-29">Our simulations have shown clearly that the use of EBV in association studies, incorporating information from relatives from the same or previous generations, can result in several problems, most notably huge increases in the type I error. This finding is attributed to the fact that when an individual’s EBV is estimated with familial information, the estimate is a linear combination of the individual’s phenotype, expressed as a deviation from the family mean, and the family mean itself. Although the individual phenotype captures the within-family segregation of the QTL, the family mean contains information on the QTL allele expressed by other family members. This “contamination” of the EBV by family information can affect both power and the FPR as follows. First, power can be reduced (as shown by results for the empirical thresholds in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS2.pdf">Table S2</a>) as many sibs may have received alternative QTL alleles. This dilutes the SNP effect, and it will have greater impact in situations in which family information makes a greater contribution to the EBV. Second, there is a greater risk of FPRs, as any SNP that differs in frequency between families risks being correlated by chance with the family mean polygenic value for the trait and hence shows a significant association with the EBV. This finding implies that in analyses of real human pedigrees the type I errors are expected to be even more serious that those shown here in the simulated pedigrees as a result of minor ethnic variations from one pedigree to the next.</p><p id="p-30">In the Appendix, we demonstrate how an EBV may be decomposed in individual and family information and into major gene and polygenic (unlinked) effects. With EBVs, it is apparent that the weighting given to the family mean is always greater than that given to the Mendelian sampling term, hence the risks of FPRs and reduced power described previously, with the converse true for residuals. The relative weightings, and the expected value of the regression of phenotype on marker, are slightly complex and depend on the trait heritability, the accuracy with which the family mean is estimated, and the QTL frequency. Derivations are shown in the Appendix for family information estimated solely from sib means; however, the same principles apply to information obtained from other sources. This is seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1/TableS1.pdf">Table S1</a>, where the five-generation pedigree resulted in a greater FPR than the two-generation pedigree; because the family mean was estimated more accurately, a greater weighting was applied to the family mean resulting in a greater FPR.</p><p id="p-31">In species in which the bulk of the information is derived from progeny testing, such as dairy cattle breeding, the contribution to the EBV coming from relatives other than the direct offspring becomes smaller and using EBV will be less detrimental than in the cases considered here. This is because offspring information directly estimates the Mendelian sampling term of the animal being evaluated, and hence assists in estimation of the QTL effect. We acknowledge that for many species, most commonly in dairy cattle where EBV are derived for bulls using a large number of daughter records, EBV for a wide range of traits are routinely available and are a convenient source of information. In these cases, the use of deregressed EBV has been proposed for GWAS and genomic prediction (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Garrick <em>et al.</em> 2009</a>). De-regressed EBV take account of the heterogeneous variances of EBV that are the result of, <em>e.g.</em>, different numbers of daughter records per sire. However, de-regressed EBVs do not remove the component of the EBV coming from information on other relatives. To remove effects from other relatives from the (deregressed) EBV, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Garrick <em>et al.</em> (2009)</a> suggested to adjust for the parent average effect. The use of de-regressed EBVs, adjusted for parent average effects, can be also relevant when the EBV is the result of repeated measurements that are not easily replaced by a single trait score for GWAS.</p><p id="p-32">In summary, when each genotyped individual has its own associated trait score(s), we recommend the use of a measured genotype approach (1) or an approximation using GRAMMAR. Although GRAMMAR was once again shown to be conservative and give an underestimate of the QTL effect, recent developments of the GenABEL software have accounted for this in the GRAMMAR-Lambda module that provides an adjusted test static and a correction for the estimated QTL effect (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Svishcheva <em>et al.</em> 2012</a>). At all costs, naïve usage of EBVs incorporating familial information should be avoided, as use of EBVs will achieve the triple whammy of reducing power, increasing the FPR and misestimating QTL effect sizes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">S.R., S.B., and D.J.K. acknowledge support from Biotechnology and Biological Sciences Research Council through the Institute Program Grant to The Roslin Institute. This work has made use of the resources provided by the Edinburgh Compute and Data Facility (<a href="http://www.ecdf.ed.ac.uk/">http://www.ecdf.ed.ac.uk/</a>).</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-10"><h3>SOURCES OF INFORMATION IN SNP ASSOCIATION ANALYSES</h3></div><div class="section" id="sec-11"><h3>Decomposing the animal genotype</h3><p id="p-33">Assume that the phenotype for individual <em>i</em> is composed of a genotype and a residual (“environmental”) term. Thus:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>Assuming that we are interested in the contribution of a major (additive) gene to the genotype, the individual’s genotype can be decomposed in several equivalent ways. First, the genotype can be decomposed into a polygenic component (uncorrelated with the major gene) and the direct additive effect of the major gene, <em>i.e.</em>:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>Alternatively, the genotype can be decomposed into the family mean (<em>F</em>) plus a Mendelian sampling term (<em>M<sub>i</sub></em>), ignoring the major gene effect for the time being. Thus;<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>Following the logic that the overall genotype can be decomposed into a family mean and a Mendelian sampling term, the same can be done with the polygenic terms, <em>i.e.</em>, <em>F<sub>poly</sub></em> and <em>M<sub>poly</sub></em>. Therefore:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>Clearly, the true major gene effect will contribute to the individual’s deviation from the family mean, hence will be contained in the <em>Mi</em> term in equation 3. However, depending on the allele frequency of the major gene variant, the major gene effect will also contribute to the family mean. It is easily shown that if the major gene is additive with a frequency of the major allele <em>p</em>, and assuming Hardy-Weinberg equilibrium, then the expected value of the major gene contribution to the family mean is (<em>p</em> − <em>q</em>)<em>g</em>.</p></div><div class="section" id="sec-12"><h3>Composition of EBV and residual</h3><p id="p-34">Now consider the situation in which we have phenotypic information on an individual and on <em>n</em> sibs, who have phenotypes <em>Ps</em>. Deeper pedigree is ignored. The EBV will be calculated as follows:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-8.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>The first term of equation (5) contains data potentially leading to false positives, <em>i.e.</em>, the false-positive signal, as any SNP that differs in frequency between families and shows a correlation with the family mean may show up as a significant effect. Depending on allele frequency the family mean will also contain information on the major gene of interest, as well. The second term contains the true major gene effect, <em>i.e.</em>, the true signal. The third term contains noise that is uncorrelated with the major gene effect in individual <em>i</em> that potentially masks the signal.</p><p id="p-35">The residual term is simply the phenotype for the individual minus the EBV:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-9.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>Note that for the EBV, the weighting given to the family mean is always greater than that given to the Mendelian sampling term<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-10.gif"/></noscript></span></span>, hence a much greater risk of false positives as well as a masking of true effects, whereas the converse is true for the Residual.</p></div><div class="section" id="sec-13"><h3>Calculating the weighting factors</h3><p id="p-36">The weighting factors <em>b</em>1 and <em>b</em>2 can easily be calculated from standard selection index theory, where <strong><em>Pb</em></strong> <em>=</em> <strong><em>Gv</em></strong>. In this case with one trait the vector <strong><em>v</em></strong> is simply a unity scalar, <strong><em>P</em></strong> is the phenotypic variance/covariance matrix of the phenotypes, and <strong><em>G</em></strong> is the genetic covariance vector between the information sources and the individual’s true genotype.</p><p id="p-37">If the phenotypic variance is set to one, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-11.gif"/></noscript></span></span>, and using standard expectations for variance and covariances, then <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-12.gif"/></noscript></span></span> and</p><p id="p-38"><span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-13.gif"/></noscript></span></span>, where <em>r</em> is the coefficient of relationship between sibs, <em>e.g.</em>, 0.5 for full sibs. For the full-sib scenario and assuming no common environmental effects, after some algebra and removing the common factor <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-14.gif"/></noscript></span></span>, we obtain the following relative weighting factors: <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-16.gif"/></noscript></span></span>, where * indicates that the terms have been scaled by the common factor. Therefore, the relative weights depend, as you would expect, on the trait heritability and the information available to calculate the family mean (<em>n</em>).</p></div><div class="section" id="sec-14"><h3>Expectations of estimated genotype effects</h3><p id="p-39">Let the major gene effect be indicated by a SNP marker, and the regression of the trait on the allele count (<em>e.g.</em>, −1, 0, 1) for the marker be defined as <em>β</em>.</p><div id="sec-15" class="subsection"><h4>Estimating the genotype as a fixed covariate in mixed model equations:</h4><p id="p-40">Let the estimated <em>g</em> be <em>ĝ</em>. Invoking the BLUE properties of Mixed Model BLUP equations, for fixed effects E(<em>ĝ</em>) = <em>g</em>. Thus, the estimate of g is unbiased.</p></div><div id="sec-16" class="subsection"><h4>Estimating the genotype from regression of phenotype on markers, <em>ignoring families:</em></h4><p id="p-41">Combining equations 1 and 2, we have</p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-17.gif"/></noscript></span></span><p id="p-42">Therefore, the expected value of the regression equation is:</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-18.gif"/></noscript></span></span><p id="p-43">Thus, the estimate of <em>g</em> is again unbiased.</p></div><div id="sec-17" class="subsection"><h4>Estimating the genotype from regression of EBV or residual on markers:</h4><p id="p-44">First note that by definition <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-19.gif"/></noscript></span></span>. Therefore, if <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-20.gif"/></noscript></span></span>is an unbiased estimate of <em>g</em>, then the sum <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-21.gif"/></noscript></span></span>is also an unbiased estimate of <em>g</em>.</p><p id="p-45">Consider <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-22.gif"/></noscript></span></span>. As described previously, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-23.gif"/></noscript></span></span>; therefore, we wish to find the expected value of the regression: <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-24.gif"/></noscript></span></span>. The first term yields a contribution <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-25.gif"/></noscript></span></span> to the expected regression coefficient. The second term is:</p><p id="p-46"><span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-26.gif"/></noscript></span></span>. Only <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-27.gif"/></noscript></span></span> has a non-zero expectation; when <em>p</em>≠0.5, the individual and family mean genotype will be correlated. This regression term is simply <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-28.gif"/></noscript></span></span>.</p><p id="p-47">Therefore:</p><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-29.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span><p id="p-48">and consequently:</p><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/2/341/embed/mml-math-30.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span><p id="p-49">Therefore, the expected effect sizes when using EBVs or residuals are biased downward, with the bias being a function of the heritability, the quantity of familial information, and the frequency of the major gene.</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008706/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-51"><em>Communicating editor: B. S. Yandell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-30"><span class="received-label">Received </span>September 30, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-12-13"><span class="accepted-label">Accepted </span>December 13, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Ekine <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.1" data-doi="10.1534/genetics.107.075614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">GRAMMAR: a fast and simple method for genome-wide pedigree-based quantitative trait loci association analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075614%26rft_id%253Dinfo%253Apmid%252F17660554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvNTc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8yLzM0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.2" data-doi="10.1093/bioinformatics/btm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaacs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">GenABEL: an R library for genome-wide association analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm108%26rft_id%253Dinfo%253Apmid%252F17384015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTAvMTI5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMi8zNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-other" id="cit-4.2.341.3" data-doi="10.1371/journal.pone.0055951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luther</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leeb</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide association study to Detect QTL for commercially important traits in Swiss large white boars.</span> <span class="cit-source">PLoS ONE</span> <span class="cit-vol">8</span>: <span class="cit-fpage">e55951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De55951%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0055951%26rft_id%253Dinfo%253Apmid%252F23393604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0055951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23393604&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.4" data-doi="10.1016/j.meatsci.2013.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Čepica</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zambonelli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Association mapping of quantitative trait loci for carcass and meat quality traits at the central part of chromosome 2 in Italian large white pigs.</span> <abbr class="cit-jnl-abbrev">Meat Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMeat%2BSci.%26rft.volume%253D95%26rft.spage%253D368%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meatsci.2013.05.002%26rft_id%253Dinfo%253Apmid%252F23747631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meatsci.2013.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23747631&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.2.341.5"><div class="cit-metadata unstructured">Crooks, L., G. Sahana, D. J. De Koning, M. Lund, and Ö. Carlborg, 2009 Comparison of analyses of the QTLMAS XII common dataset. II: genome-wide association and fine mapping, <em>BMC Proceedings Vol. 3: S2.</em></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.6" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.7" data-doi="10.1159/000085224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijsman</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">MCMC multilocus lod scores: application of a new approach.</span> <abbr class="cit-jnl-abbrev">Hum. Hered.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bheredity%26rft.stitle%253DHum%2BHered%26rft.aulast%253DGeorge%26rft.auinit1%253DA.%2BW.%26rft.volume%253D59%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DMCMC%2Bmultilocus%2Blod%2Bscores%253A%2Bapplication%2Bof%2Ba%2Bnew%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000085224%26rft_id%253Dinfo%253Apmid%252F15838179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000085224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15838179&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229033000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-4.2.341.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml User Guide Release 3.0. 1ES</span>. <span class="cit-publ-name">VSN International LTD</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.9" data-doi="10.1111/j.1365-294X.2011.05076.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickering</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beraldi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping identifies the genetic basis of discrete and quantitative variation in sexual weaponry in a wild sheep population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2555</span>–<span class="cit-lpage">2566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DJohnston%26rft.auinit1%253DS.%2BE.%26rft.volume%253D20%26rft.issue%253D12%26rft.spage%253D2555%26rft.epage%253D2566%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bidentifies%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bdiscrete%2Band%2Bquantitative%2Bvariation%2Bin%2Bsexual%2Bweaponry%2Bin%2Ba%2Bwild%2Bsheep%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05076.x%26rft_id%253Dinfo%253Apmid%252F21651634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05076.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21651634&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291402600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.10" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.11" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzIvMzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.12" data-doi="10.1038/ng.2410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svishcheva</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axenovich</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belonogova</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rapid variance components–based method for whole-genome association analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSvishcheva%26rft.auinit1%253DG.%2BR.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1166%26rft.epage%253D1170%26rft.atitle%253DRapid%2Bvariance%2Bcomponents-based%2Bmethod%2Bfor%2Bwhole-genome%2Bassociation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2410%26rft_id%253Dinfo%253Apmid%252F22983301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22983301&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.13" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.2.341.14" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F2%2F341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/2/341.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/2/325" title="Horizontal Gene Transfer Can Rescue Prokaryotes from Muller’s Ratchet: Benefit of DNA from Dead Cells and Population Subdivision" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/2/349" title="Global Analysis of Serine/Threonine and Tyrosine Protein Phosphatase Catalytic Subunit Genes in Neurospora crassa Reveals Interplay Between Phosphatases and the p38 Mitogen-Activated Protein Kinase" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/2"><span>Volume 4 Issue 2, February 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/2.cover-source.jpg?itok=nk9yht63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/2/341.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-39rYzWMFB_d8MeBafZkdUC3GUjWB7veBSQMEfcZ0wWc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/2/341" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/2/341" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/2/341" class="active" data-icon-position="" data-hide-link-title="0">Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459634" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tpA68PyDGwXtPQrfcsql_BylHp1cDMD4OVoUkyGCrJI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/2/341.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/2/341" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zxVuRF3IY8mWayClJiPbCazyJKEYHpTbslQ3ZOmnTSA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/2/341" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/2/341.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459634" id="node459634--2" data-pisa="ggg;4/2/341" data-pisa-master="ggg;g3.113.008706" data-apath="/ggg/4/2/341.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chinyere C. Ekine</span>, <span class="highwire-citation-author" data-delta="1">Suzanne J. Rowe</span>, <span class="highwire-citation-author" data-delta="2">Stephen C. Bishop</span> and <span class="highwire-citation-author" data-delta="3">Dirk-Jan de Koning</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">341-347;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008706 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459634--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chinyere C. Ekine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chinyere%2BC.%2BEkine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ekine%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChinyere%2BC.%2BEkine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Suzanne J. Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BJ.%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuzanne%2BJ.%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen C. Bishop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BC.%2BBishop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bishop%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BC.%2BBishop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dirk-Jan de Koning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk-Jan%2Bde%2BKoning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Koning%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADirk-Jan%2Bde%2BKoning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dj.de-koning@slu.se" class="" data-icon-position="" data-hide-link-title="0">dj.de-koning@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/2/341" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/2/341&amp;title=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/2/341&amp;title=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/2/341&amp;title=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/2/341&amp;text=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/2/341&amp;title=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/2/341&amp;t=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/2/341&amp;title=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/2/341&amp;title=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459634/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459634" id="node459634--3" data-pisa="ggg;4/2/341" data-pisa-master="ggg;g3.113.008706" data-apath="/ggg/4/2/341.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Why Breeding Values Estimated Using Familial Data Should Not Be Used for Genome-Wide Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chinyere C. Ekine</span>, <span class="highwire-citation-author" data-delta="1">Suzanne J. Rowe</span>, <span class="highwire-citation-author" data-delta="2">Stephen C. Bishop</span> and <span class="highwire-citation-author" data-delta="3">Dirk-Jan de Koning</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">341-347;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008706 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459634--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chinyere C. Ekine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chinyere%2BC.%2BEkine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ekine%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChinyere%2BC.%2BEkine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Suzanne J. Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BJ.%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuzanne%2BJ.%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen C. Bishop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BC.%2BBishop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bishop%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BC.%2BBishop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dirk-Jan de Koning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk-Jan%2Bde%2BKoning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Koning%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADirk-Jan%2Bde%2BKoning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dj.de-koning@slu.se" class="" data-icon-position="" data-hide-link-title="0">dj.de-koning@slu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=341&amp;rft.epage=347&amp;rft.atitle=Why%20Breeding%20Values%20Estimated%20Using%20Familial%20Data%20Should%20Not%20Be%20Used%20for%20Genome-Wide%20Association%20Studies&amp;rft.volume=4&amp;rft.issue=2&amp;rft.date=2014-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ekine%2C+Chinyere+C.&amp;rft.au=Rowe%2C+Suzanne+J.&amp;rft.au=Bishop%2C+Stephen+C.&amp;rft.au=de+Koning%2C+Dirk-Jan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459634/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459634/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459634/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459634/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459634/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459634/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459634/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459634/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459634/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459634/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459634/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459634/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/2/341" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/2/341.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/2/341.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/2/341.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/2/341.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
